

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/5/309/359388","@type":"ScholarlyArticle","name":"Characteristics and Prediction of RNA Editing Sites in Transcripts of the Moss  Takakia lepidozioides  Chloroplast","datePublished":"2008-07-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2008-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn016","keywords":["bioinformatics","chloroplast","computational biology","plant organelle","singlet and doublet propensities","<italic>Takakia lepidozioides</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/5/309/359388","author":[{"name":"Yura, Kei","affiliation":"1  Graduate School of Humanities and Sciences ,  Ochanomizu University ,  2-1-1 Otsuka, Bunkyo, Tokyo 112-8610 ,  Japan","@type":"Person"},{"name":"Miyata, Yuki","affiliation":"2  Center for Gene Research ,  Nagoya University, Chikusa ,  Nagoya 464-8602 ,  Japan","@type":"Person"},{"name":"Arikawa, Tomotsugu","affiliation":"3  Department of Biology ,  Keio University ,  4-1-1 Hiyoshi, Kohoku, Yokohama 223-8521 ,  Japan","@type":"Person"},{"name":"Higuchi, Masanobu","affiliation":"4  Department of Botany ,  National Museum of Nature and Science ,  4-1-1 Amakubo, Tsukuba 305-0005 ,  Japan","@type":"Person"},{"name":"Sugita, Mamoru","affiliation":"2  Center for Gene Research ,  Nagoya University, Chikusa ,  Nagoya 464-8602 ,  Japan","@type":"Person"}],"description":"Abstract.  RNA editing in land plant organelles is a process primarily involving the conversion of cytidine to uridine in pre-mRNAs. The process is required for","pageStart":"309","pageEnd":"321","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Characteristics and Prediction of RNA Editing Sites in Transcripts of the Moss  Takakia lepidozioides  Chloroplast","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Characteristics and Prediction of RNA Editing Sites in Transcripts of the Moss  Takakia lepidozioides  Chloroplast" />
    <meta property="og:description" content="Abstract.  RNA editing in land plant organelles is a process primarily involving the conversion of cytidine to uridine in pre-mRNAs. The process is required for" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/5/309/359388" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-07-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yura, Kei" /><meta name="citation_author_institution" content="Graduate School of Humanities and Sciences, Ochanomizu University, 2-1-1 Otsuka, Bunkyo, Tokyo 112-8610, Japan" /><meta name="citation_author" content="Miyata, Yuki" /><meta name="citation_author_institution" content="Center for Gene Research, Nagoya University, Chikusa, Nagoya 464-8602, Japan" /><meta name="citation_author" content="Arikawa, Tomotsugu" /><meta name="citation_author_institution" content="Department of Biology, Keio University, 4-1-1 Hiyoshi, Kohoku, Yokohama 223-8521, Japan" /><meta name="citation_author" content="Higuchi, Masanobu" /><meta name="citation_author_institution" content="Department of Botany, National Museum of Nature and Science, 4-1-1 Amakubo, Tsukuba 305-0005, Japan" /><meta name="citation_author" content="Sugita, Mamoru" /><meta name="citation_author_institution" content="Center for Gene Research, Nagoya University, Chikusa, Nagoya 464-8602, Japan" /><meta name="citation_title" content="Characteristics and Prediction of RNA Editing Sites in Transcripts of the Moss Takakia lepidozioides Chloroplast" /><meta name="citation_firstpage" content="309" /><meta name="citation_lastpage" content="321" /><meta name="citation_doi" content="10.1093/dnares/dsn016" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2008/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Sense from nonsense: how the genetic information of chloroplasts is altered by RNA editing; Biochimie;  citation_year=2000;  citation_volume=82;  citation_pages=549-557; " /><meta name="citation_reference" content="citation_title=RNA editing in plant mitochondria; Nature;  citation_year=1989;  citation_volume=341;  citation_pages=662-666; " /><meta name="citation_reference" content="citation_title=Editing of a chloroplast mRNA by creation of an initiation codon; Nature;  citation_year=1991;  citation_volume=353;  citation_pages=178-180; " /><meta name="citation_reference" content="citation_title=Evidence for RNA editing in mitochondria of all major groups of land plants except the Bryophyta; Proc. Natl. Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=629-633; " /><meta name="citation_reference" content="citation_title=Extensive RNA editing of U to C in addition to C to U substitution in the rbcL transcripts of hornwort chloroplasts and the origin of RNA editing in green plants; Nucleic Acids Res.;  citation_year=1996;  citation_volume=24;  citation_pages=1008-1014; " /><meta name="citation_reference" content="citation_title=Occurrence of plastid RNA editing in all major lineages of land plants; Proc. Natl. Acad. Sci. USA;  citation_year=1997;  citation_volume=94;  citation_pages=6285-6290; " /><meta name="citation_reference" content="citation_title=Widespread and extensive editing of mitochondrial mRNAs in dinoflagellates; J. Mol. Biol.;  citation_year=2002;  citation_volume=320;  citation_pages=727-739; " /><meta name="citation_reference" content="citation_title=RNA editing in hornwort chloroplasts makes more than half the genes functional; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=2417-2423; " /><meta name="citation_reference" content="citation_title=RNA editing in Arabidopsis mitochondria effects 441 C to U changes in ORFs; Proc. Natl. Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=15324-15329; " /><meta name="citation_reference" content="citation_title=Creation of a novel protein-coding region at the RNA level in black pine chloroplasts: the pattern of RNA editing in the gymnosperm chloroplast is different from that in angiosperms; Proc. Natl. Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=8766-8770; " /><meta name="citation_reference" content="citation_title=RNA editing site recognition in higher plant mitochondria; J. Hered.;  citation_year=1999;  citation_volume=90;  citation_pages=338-344; " /><meta name="citation_reference" content="citation_title=RNA editing in wheat mitochondria proceeds by a deamination mechanism; FEBS Lett.;  citation_year=1995;  citation_volume=373;  citation_pages=56-60; " /><meta name="citation_reference" content="citation_title=Evidence for a site-specific cytidine deamination reaction involved in C to U RNA editing of plant mitochondria; J. Biol. Chem.;  citation_year=1995;  citation_volume=270;  citation_pages=18227-18233; " /><meta name="citation_reference" content="citation_title=Site-specific factor involved in the editing of the psbL mRNA in tobacco plastids; EMBO J.;  citation_year=1995;  citation_volume=14;  citation_pages=2951-2957; " /><meta name="citation_reference" content="citation_title=In vivo dissection of cis-acting determinants for plastid RNA editing; EMBO J.;  citation_year=1996;  citation_volume=15;  citation_pages=5052-5059; " /><meta name="citation_reference" content="citation_title=Identification of critical nucleotide positions for plastid RNA editing site recognition; RNA;  citation_year=1997;  citation_volume=3;  citation_pages=1194-1200; " /><meta name="citation_reference" content="citation_title=Sequences directing C to U editing of the plastid psbL mRNA are located within a 22 nucleotide segment spanning the editing site; EMBO J.;  citation_year=1996;  citation_volume=15;  citation_pages=5958-5964; " /><meta name="citation_reference" content="citation_title=Transfer of plastid RNA-editing activity to novel sites suggests a critical role for spacing in editing-site recognition; Proc. Natl. Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=4856-4861; " /><meta name="citation_reference" content="citation_title=A single alternation 20 nt 5′ to an editing target inhibits chloroplast RNA editing in vivo; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=1507-1513; " /><meta name="citation_reference" content="citation_title=Heterologous, splicing-dependent RNA editing in chloroplasts: allotetrapolidy provides trans-factors; EMBO J.;  citation_year=2001;  citation_volume=20;  citation_pages=4874-4883; " /><meta name="citation_reference" content="citation_title=A site-specific factor interacts directly with its cognate RNA editing site in chloroplast transcripts; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=48-52; " /><meta name="citation_reference" content="citation_title=A simple in vitro RNA editing assay for chloroplast transcripts using fluorescent dideoxynucleotides: distinct types of sequence elements required for editing of ndh transcripts; Plant J.;  citation_year=2006;  citation_volume=47;  citation_pages=802-810; " /><meta name="citation_reference" content="citation_title=The evolution of chloroplast RNA editing; Mol. Biol., Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1912-1921; " /><meta name="citation_reference" content="citation_title=Expression of the wheat mitochondrial nad3-rps12 transcription unit: correlation between editing and mRNA maturation; Plant Cell;  citation_year=1991;  citation_volume=3;  citation_pages=1109-1120; " /><meta name="citation_reference" content="citation_title=Editing of plastid RNA in Arabidopsis thaliana ecotypes; Plant J.;  citation_year=2005;  citation_volume=43;  citation_pages=708-715; " /><meta name="citation_reference" content="citation_title=RNA editing sites in tobacco chloroplast transcripts: editing as a possible regulator of chloroplast RNA polymerase activity; Mol. Gen. Genet.;  citation_year=1999;  citation_volume=262;  citation_pages=462-467; " /><meta name="citation_reference" content="citation_title=A pentatricopeptide repeat protein is essential for RNA editing in chloroplasts; Nature;  citation_year=2005;  citation_volume=433;  citation_pages=326-330; " /><meta name="citation_reference" content="citation_title=A pentatricopeptide repeat protein is a site recognition factor in chloroplast RNA editing; J. Biol. Chem.;  citation_year=2006;  citation_volume=281;  citation_pages=37661-37667; " /><meta name="citation_reference" content="citation_title=Conserved domain structure of pentatricopeptide repeat proteins involved in chloroplast RNA editing; Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=8178-8183; " /><meta name="citation_reference" content="citation_title=On the Expansion of the pentatricopeptide repeat gene family in plants; Mol. Biol. Evol.;  citation_year=2008;  citation_volume=25;  citation_pages=1120-1128; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of Arabidopsis pentatricopeptide repeat proteins reveals their essential role in organelle biogenesis; Plant Cell;  citation_year=2004;  citation_volume=16;  citation_pages=2089-2103; " /><meta name="citation_reference" content="citation_title=A hypothesis on the identification of the editing enzyme in plant organelles; FEBS Lett.;  citation_year=2007;  citation_volume=581;  citation_pages=4132-4138; " /><meta name="citation_reference" content="citation_title=Simple statistical models predict C-to-U edited sites in plant mitochondrial RNA; BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=132" /><meta name="citation_reference" content="citation_title=PREP-Mt: predictive RNA editor for plant mitochondrial genes; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=96" /><meta name="citation_reference" content="citation_title=Genetic algorithm learning as a robust approach to RNA editing site prediction; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=145" /><meta name="citation_reference" content="citation_title=Prediction of C-to-U RNA editing sites in higher plant mitochondria using only nucleotide sequence features; Biochem. Biophys. Res. Commun.;  citation_year=2007;  citation_volume=358;  citation_pages=336-341; " /><meta name="citation_reference" content="citation_title=Identification of RNA editing sites in chloroplast transcripts of Phalaenopsis aphrodite and comparative analysis with those of other seed plants; Plant Cell Physiol.;  citation_year=2007;  citation_volume=48;  citation_pages=362-368; " /><meta name="citation_reference" content="citation_title=Extensive RNA editing in transcripts from the psbB operon and rpoA gene of plastids from the enigmatic moss Takakia lepidozioides; Biosci. Biotechnol. Biochem.;  citation_year=2006;  citation_volume=70;  citation_pages=2268-2274; " /><meta name="citation_reference" content="citation_title=RNA editing in the anticodon of tRNALeu (CAA) occurs before group I intron splicing in plastids of a moss Takakia lepidozioides S. Hatt. &amp; Inoue; Plant Biol.;  citation_year=2008;  citation_volume=10;  citation_pages=250-255; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-425; " /><meta name="citation_reference" content="citation_title=Putative mechanism of natural transformation as deduced from genome data; DNA Res.;  citation_year=1999;  citation_volume=6;  citation_pages=75-82; " /><meta name="citation_reference" content="citation_title=Amino acid residue doublet propensity in the protein-RNA interface and its application to RNA interface prediction; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=6450-6460; " /><meta name="citation_reference" content="citation_title=Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment; Science;  citation_year=1993;  citation_volume=262;  citation_pages=208-214; " /><meta name="citation_reference" content="citation_title=RNA editing in higher plant mitochondria: Analysis of biochemistry and specificity; Biochimie;  citation_year=1995;  citation_volume=77;  citation_pages=79-86; " /><meta name="citation_reference" content="citation_title=A systematic search for RNA editing sites in pea chloroplasts: an editing event causes diversification from the evolutionarily conserved amino acid sequence; Plant Cell Physiol.;  citation_year=2004;  citation_volume=45;  citation_pages=1615-1622; " /><meta name="citation_reference" content="citation_title=A rapid high-throughput method for the detection and quantification of RNA editing based on high-resolution melting of amplicons; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=e114" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/5/309/1028712/dsn016.pdf" /><meta name="description" content="Abstract.  RNA editing in land plant organelles is a process primarily involving the conversion of cytidine to uridine in pre-mRNAs. The process is required for" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/5/309/359388" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/5/309/359388" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/5/309/359388");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn016';
    </script>
<title>
	Characteristics and Prediction of RNA Editing Sites in Transcripts of the Moss Takakia lepidozioides Chloroplast | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./359388" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcNnPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZdXJhLCBLZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSHVtYW5pdGllcyBhbmQgU2NpZW5jZXMsIE9jaGFub21penUgVW5pdmVyc2l0eSwgMi0xLTEgT3RzdWthLCBCdW5reW8sIFRva3lvIDExMi04NjEwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1peWF0YSwgWXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBmb3IgR2VuZSBSZXNlYXJjaCwgTmFnb3lhIFVuaXZlcnNpdHksIENoaWt1c2EsIE5hZ295YSA0NjQtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcmlrYXdhLCBUb21vdHN1Z3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIEtlaW8gVW5pdmVyc2l0eSwgNC0xLTEgSGl5b3NoaSwgS29ob2t1LCBZb2tvaGFtYSAyMjMtODUyMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIaWd1Y2hpLCBNYXNhbm9idSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQm90YW55LCBOYXRpb25hbCBNdXNldW0gb2YgTmF0dXJlIGFuZCBTY2llbmNlLCA0LTEtMSBBbWFrdWJvLCBUc3VrdWJhIDMwNS0wMDA1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1Z2l0YSwgTWFtb3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBHZW5lIFJlc2VhcmNoLCBOYWdveWEgVW5pdmVyc2l0eSwgQ2hpa3VzYSwgTmFnb3lhIDQ2NC04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ2hhcmFjdGVyaXN0aWNzIGFuZCBQcmVkaWN0aW9uIG9mIFJOQSBFZGl0aW5nIFNpdGVzIGluIFRyYW5zY3JpcHRzIG9mIHRoZSBNb3NzIFRha2FraWEgbGVwaWRvemlvaWRlcyBDaGxvcm9wbGFzdCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZW5zZSBmcm9tIG5vbnNlbnNlOiBob3cgdGhlIGdlbmV0aWMgaW5mb3JtYXRpb24gb2YgY2hsb3JvcGxhc3RzIGlzIGFsdGVyZWQgYnkgUk5BIGVkaXRpbmc7IEJpb2NoaW1pZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPTU0OS01NTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBlZGl0aW5nIGluIHBsYW50IG1pdG9jaG9uZHJpYTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTM0MTsgIGNpdGF0aW9uX3BhZ2VzPTY2Mi02NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVkaXRpbmcgb2YgYSBjaGxvcm9wbGFzdCBtUk5BIGJ5IGNyZWF0aW9uIG9mIGFuIGluaXRpYXRpb24gY29kb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTM7ICBjaXRhdGlvbl9wYWdlcz0xNzgtMTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgUk5BIGVkaXRpbmcgaW4gbWl0b2Nob25kcmlhIG9mIGFsbCBtYWpvciBncm91cHMgb2YgbGFuZCBwbGFudHMgZXhjZXB0IHRoZSBCcnlvcGh5dGE7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9NjI5LTYzMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0ZW5zaXZlIFJOQSBlZGl0aW5nIG9mIFUgdG8gQyBpbiBhZGRpdGlvbiB0byBDIHRvIFUgc3Vic3RpdHV0aW9uIGluIHRoZSByYmNMIHRyYW5zY3JpcHRzIG9mIGhvcm53b3J0IGNobG9yb3BsYXN0cyBhbmQgdGhlIG9yaWdpbiBvZiBSTkEgZWRpdGluZyBpbiBncmVlbiBwbGFudHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEwMDgtMTAxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T2NjdXJyZW5jZSBvZiBwbGFzdGlkIFJOQSBlZGl0aW5nIGluIGFsbCBtYWpvciBsaW5lYWdlcyBvZiBsYW5kIHBsYW50czsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz02Mjg1LTYyOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdpZGVzcHJlYWQgYW5kIGV4dGVuc2l2ZSBlZGl0aW5nIG9mIG1pdG9jaG9uZHJpYWwgbVJOQXMgaW4gZGlub2ZsYWdlbGxhdGVzOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMyMDsgIGNpdGF0aW9uX3BhZ2VzPTcyNy03Mzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBlZGl0aW5nIGluIGhvcm53b3J0IGNobG9yb3BsYXN0cyBtYWtlcyBtb3JlIHRoYW4gaGFsZiB0aGUgZ2VuZXMgZnVuY3Rpb25hbDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjQxNy0yNDIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgZWRpdGluZyBpbiBBcmFiaWRvcHNpcyBtaXRvY2hvbmRyaWEgZWZmZWN0cyA0NDEgQyB0byBVIGNoYW5nZXMgaW4gT1JGczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz0xNTMyNC0xNTMyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3JlYXRpb24gb2YgYSBub3ZlbCBwcm90ZWluLWNvZGluZyByZWdpb24gYXQgdGhlIFJOQSBsZXZlbCBpbiBibGFjayBwaW5lIGNobG9yb3BsYXN0czogdGhlIHBhdHRlcm4gb2YgUk5BIGVkaXRpbmcgaW4gdGhlIGd5bW5vc3Blcm0gY2hsb3JvcGxhc3QgaXMgZGlmZmVyZW50IGZyb20gdGhhdCBpbiBhbmdpb3NwZXJtczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz04NzY2LTg3NzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBlZGl0aW5nIHNpdGUgcmVjb2duaXRpb24gaW4gaGlnaGVyIHBsYW50IG1pdG9jaG9uZHJpYTsgSi4gSGVyZWQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9MzM4LTM0NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BIGVkaXRpbmcgaW4gd2hlYXQgbWl0b2Nob25kcmlhIHByb2NlZWRzIGJ5IGEgZGVhbWluYXRpb24gbWVjaGFuaXNtOyBGRUJTIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTM3MzsgIGNpdGF0aW9uX3BhZ2VzPTU2LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgYSBzaXRlLXNwZWNpZmljIGN5dGlkaW5lIGRlYW1pbmF0aW9uIHJlYWN0aW9uIGludm9sdmVkIGluIEMgdG8gVSBSTkEgZWRpdGluZyBvZiBwbGFudCBtaXRvY2hvbmRyaWE7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI3MDsgIGNpdGF0aW9uX3BhZ2VzPTE4MjI3LTE4MjMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaXRlLXNwZWNpZmljIGZhY3RvciBpbnZvbHZlZCBpbiB0aGUgZWRpdGluZyBvZiB0aGUgcHNiTCBtUk5BIGluIHRvYmFjY28gcGxhc3RpZHM7IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yOTUxLTI5NTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHZpdm8gZGlzc2VjdGlvbiBvZiBjaXMtYWN0aW5nIGRldGVybWluYW50cyBmb3IgcGxhc3RpZCBSTkEgZWRpdGluZzsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTUwNTItNTA1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgY3JpdGljYWwgbnVjbGVvdGlkZSBwb3NpdGlvbnMgZm9yIHBsYXN0aWQgUk5BIGVkaXRpbmcgc2l0ZSByZWNvZ25pdGlvbjsgUk5BOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMTk0LTEyMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlcyBkaXJlY3RpbmcgQyB0byBVIGVkaXRpbmcgb2YgdGhlIHBsYXN0aWQgcHNiTCBtUk5BIGFyZSBsb2NhdGVkIHdpdGhpbiBhIDIyIG51Y2xlb3RpZGUgc2VnbWVudCBzcGFubmluZyB0aGUgZWRpdGluZyBzaXRlOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NTk1OC01OTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2ZlciBvZiBwbGFzdGlkIFJOQS1lZGl0aW5nIGFjdGl2aXR5IHRvIG5vdmVsIHNpdGVzIHN1Z2dlc3RzIGEgY3JpdGljYWwgcm9sZSBmb3Igc3BhY2luZyBpbiBlZGl0aW5nLXNpdGUgcmVjb2duaXRpb247IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9NDg1Ni00ODYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbmdsZSBhbHRlcm5hdGlvbiAyMCBudCA14oCyIHRvIGFuIGVkaXRpbmcgdGFyZ2V0IGluaGliaXRzIGNobG9yb3BsYXN0IFJOQSBlZGl0aW5nIGluIHZpdm87IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTE1MDctMTUxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGV0ZXJvbG9nb3VzLCBzcGxpY2luZy1kZXBlbmRlbnQgUk5BIGVkaXRpbmcgaW4gY2hsb3JvcGxhc3RzOiBhbGxvdGV0cmFwb2xpZHkgcHJvdmlkZXMgdHJhbnMtZmFjdG9yczsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTQ4NzQtNDg4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaXRlLXNwZWNpZmljIGZhY3RvciBpbnRlcmFjdHMgZGlyZWN0bHkgd2l0aCBpdHMgY29nbmF0ZSBSTkEgZWRpdGluZyBzaXRlIGluIGNobG9yb3BsYXN0IHRyYW5zY3JpcHRzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz00OC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUgaW4gdml0cm8gUk5BIGVkaXRpbmcgYXNzYXkgZm9yIGNobG9yb3BsYXN0IHRyYW5zY3JpcHRzIHVzaW5nIGZsdW9yZXNjZW50IGRpZGVveHludWNsZW90aWRlczogZGlzdGluY3QgdHlwZXMgb2Ygc2VxdWVuY2UgZWxlbWVudHMgcmVxdWlyZWQgZm9yIGVkaXRpbmcgb2YgbmRoIHRyYW5zY3JpcHRzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTgwMi04MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb24gb2YgY2hsb3JvcGxhc3QgUk5BIGVkaXRpbmc7IE1vbC4gQmlvbC4sIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTkxMi0xOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIG9mIHRoZSB3aGVhdCBtaXRvY2hvbmRyaWFsIG5hZDMtcnBzMTIgdHJhbnNjcmlwdGlvbiB1bml0OiBjb3JyZWxhdGlvbiBiZXR3ZWVuIGVkaXRpbmcgYW5kIG1STkEgbWF0dXJhdGlvbjsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTEwOS0xMTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZGl0aW5nIG9mIHBsYXN0aWQgUk5BIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGVjb3R5cGVzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTcwOC03MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBlZGl0aW5nIHNpdGVzIGluIHRvYmFjY28gY2hsb3JvcGxhc3QgdHJhbnNjcmlwdHM6IGVkaXRpbmcgYXMgYSBwb3NzaWJsZSByZWd1bGF0b3Igb2YgY2hsb3JvcGxhc3QgUk5BIHBvbHltZXJhc2UgYWN0aXZpdHk7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjYyOyAgY2l0YXRpb25fcGFnZXM9NDYyLTQ2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgcHJvdGVpbiBpcyBlc3NlbnRpYWwgZm9yIFJOQSBlZGl0aW5nIGluIGNobG9yb3BsYXN0czsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzMzsgIGNpdGF0aW9uX3BhZ2VzPTMyNi0zMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcGVudGF0cmljb3BlcHRpZGUgcmVwZWF0IHByb3RlaW4gaXMgYSBzaXRlIHJlY29nbml0aW9uIGZhY3RvciBpbiBjaGxvcm9wbGFzdCBSTkEgZWRpdGluZzsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjgxOyAgY2l0YXRpb25fcGFnZXM9Mzc2NjEtMzc2Njc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZlZCBkb21haW4gc3RydWN0dXJlIG9mIHBlbnRhdHJpY29wZXB0aWRlIHJlcGVhdCBwcm90ZWlucyBpbnZvbHZlZCBpbiBjaGxvcm9wbGFzdCBSTkEgZWRpdGluZzsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9ODE3OC04MTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgRXhwYW5zaW9uIG9mIHRoZSBwZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgZ2VuZSBmYW1pbHkgaW4gcGxhbnRzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTEyMC0xMTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBBcmFiaWRvcHNpcyBwZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgcHJvdGVpbnMgcmV2ZWFscyB0aGVpciBlc3NlbnRpYWwgcm9sZSBpbiBvcmdhbmVsbGUgYmlvZ2VuZXNpczsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIwODktMjEwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoeXBvdGhlc2lzIG9uIHRoZSBpZGVudGlmaWNhdGlvbiBvZiB0aGUgZWRpdGluZyBlbnp5bWUgaW4gcGxhbnQgb3JnYW5lbGxlczsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01ODE7ICBjaXRhdGlvbl9wYWdlcz00MTMyLTQxMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBzdGF0aXN0aWNhbCBtb2RlbHMgcHJlZGljdCBDLXRvLVUgZWRpdGVkIHNpdGVzIGluIHBsYW50IG1pdG9jaG9uZHJpYWwgUk5BOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBSRVAtTXQ6IHByZWRpY3RpdmUgUk5BIGVkaXRvciBmb3IgcGxhbnQgbWl0b2Nob25kcmlhbCBnZW5lczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz05NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYWxnb3JpdGhtIGxlYXJuaW5nIGFzIGEgcm9idXN0IGFwcHJvYWNoIHRvIFJOQSBlZGl0aW5nIHNpdGUgcHJlZGljdGlvbjsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIEMtdG8tVSBSTkEgZWRpdGluZyBzaXRlcyBpbiBoaWdoZXIgcGxhbnQgbWl0b2Nob25kcmlhIHVzaW5nIG9ubHkgbnVjbGVvdGlkZSBzZXF1ZW5jZSBmZWF0dXJlczsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1ODsgIGNpdGF0aW9uX3BhZ2VzPTMzNi0zNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFJOQSBlZGl0aW5nIHNpdGVzIGluIGNobG9yb3BsYXN0IHRyYW5zY3JpcHRzIG9mIFBoYWxhZW5vcHNpcyBhcGhyb2RpdGUgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIHdpdGggdGhvc2Ugb2Ygb3RoZXIgc2VlZCBwbGFudHM7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0zNjItMzY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgUk5BIGVkaXRpbmcgaW4gdHJhbnNjcmlwdHMgZnJvbSB0aGUgcHNiQiBvcGVyb24gYW5kIHJwb0EgZ2VuZSBvZiBwbGFzdGlkcyBmcm9tIHRoZSBlbmlnbWF0aWMgbW9zcyBUYWtha2lhIGxlcGlkb3ppb2lkZXM7IEJpb3NjaS4gQmlvdGVjaG5vbC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0yMjY4LTIyNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBlZGl0aW5nIGluIHRoZSBhbnRpY29kb24gb2YgdFJOQUxldSAoQ0FBKSBvY2N1cnMgYmVmb3JlIGdyb3VwIEkgaW50cm9uIHNwbGljaW5nIGluIHBsYXN0aWRzIG9mIGEgbW9zcyBUYWtha2lhIGxlcGlkb3ppb2lkZXMgUy4gSGF0dC4gJmFtcDsgSW5vdWU7IFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MjUwLTI1NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG5laWdoYm9yLWpvaW5pbmcgbWV0aG9kOiBhIG5ldyBtZXRob2QgZm9yIHJlY29uc3RydWN0aW5nIHBoeWxvZ2VuZXRpYyB0cmVlczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDA2LTQyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHV0YXRpdmUgbWVjaGFuaXNtIG9mIG5hdHVyYWwgdHJhbnNmb3JtYXRpb24gYXMgZGVkdWNlZCBmcm9tIGdlbm9tZSBkYXRhOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtaW5vIGFjaWQgcmVzaWR1ZSBkb3VibGV0IHByb3BlbnNpdHkgaW4gdGhlIHByb3RlaW4tUk5BIGludGVyZmFjZSBhbmQgaXRzIGFwcGxpY2F0aW9uIHRvIFJOQSBpbnRlcmZhY2UgcHJlZGljdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NjQ1MC02NDYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpbmcgc3VidGxlIHNlcXVlbmNlIHNpZ25hbHM6IGEgR2liYnMgc2FtcGxpbmcgc3RyYXRlZ3kgZm9yIG11bHRpcGxlIGFsaWdubWVudDsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjI7ICBjaXRhdGlvbl9wYWdlcz0yMDgtMjE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgZWRpdGluZyBpbiBoaWdoZXIgcGxhbnQgbWl0b2Nob25kcmlhOiBBbmFseXNpcyBvZiBiaW9jaGVtaXN0cnkgYW5kIHNwZWNpZmljaXR5OyBCaW9jaGltaWU7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz03OS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzeXN0ZW1hdGljIHNlYXJjaCBmb3IgUk5BIGVkaXRpbmcgc2l0ZXMgaW4gcGVhIGNobG9yb3BsYXN0czogYW4gZWRpdGluZyBldmVudCBjYXVzZXMgZGl2ZXJzaWZpY2F0aW9uIGZyb20gdGhlIGV2b2x1dGlvbmFyaWx5IGNvbnNlcnZlZCBhbWlubyBhY2lkIHNlcXVlbmNlOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9MTYxNS0xNjIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJhcGlkIGhpZ2gtdGhyb3VnaHB1dCBtZXRob2QgZm9yIHRoZSBkZXRlY3Rpb24gYW5kIHF1YW50aWZpY2F0aW9uIG9mIFJOQSBlZGl0aW5nIGJhc2VkIG9uIGhpZ2gtcmVzb2x1dGlvbiBtZWx0aW5nIG9mIGFtcGxpY29uczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9ZTExNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNS81LzMwOS8xMDI4NzEyL2RzbjAxNi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgUk5BIGVkaXRpbmcgaW4gbGFuZCBwbGFudCBvcmdhbmVsbGVzIGlzIGEgcHJvY2VzcyBwcmltYXJpbHkgaW52b2x2aW5nIHRoZSBjb252ZXJzaW9uIG9mIGN5dGlkaW5lIHRvIHVyaWRpbmUgaW4gcHJlLW1STkFzLiBUaGUgcHJvY2VzcyBpcyByZXF1aXJlZCBmb3IiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1OTM4OCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTlflJH5OdAY/OiATu0vFf9SG7wJA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f309%2f359388" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f309%2f359388" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13976" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&amp;Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/5">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5179952">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5179952">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5179954">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5179954">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5179959">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5179959">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5179978">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5179978">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5180014">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5180014">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5180016">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5180016">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5180018">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5180018">Acknowledgement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5180020">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5180020">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/5/297/359537">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/5/323/359995">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Characteristics and Prediction of RNA Editing Sites in Transcripts of the Moss <em>Takakia lepidozioides</em> Chloroplast
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kei Yura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Humanities and Sciences</div>, <div class="insititution">Ochanomizu University</div>, <div class="addr-line">2-1-1 Otsuka, Bunkyo, Tokyo 112-8610</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel/Fax. +81 3-5978-5514. E-mail: <a href="mailto:yura.kei@ocha.ac.jp" target="_blank">yura.kei@ocha.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kei+Yura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yura K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yura K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kei Yura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Miyata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Center for Gene Research</div>, <div class="insititution">Nagoya University, Chikusa</div>, <div class="addr-line">Nagoya 464-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Miyata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyata Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miyata Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Miyata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomotsugu Arikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Biology</div>, <div class="insititution">Keio University</div>, <div class="addr-line">4-1-1 Hiyoshi, Kohoku, Yokohama 223-8521</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomotsugu+Arikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arikawa T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arikawa T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tomotsugu Arikawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masanobu Higuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Botany</div>, <div class="insititution">National Museum of Nature and Science</div>, <div class="addr-line">4-1-1 Amakubo, Tsukuba 305-0005</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masanobu+Higuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Higuchi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Higuchi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masanobu Higuchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mamoru Sugita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Center for Gene Research</div>, <div class="insititution">Nagoya University, Chikusa</div>, <div class="addr-line">Nagoya 464-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mamoru+Sugita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugita M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugita M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mamoru Sugita</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Kenta Nakai</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 309–321, <a href='https://doi.org/10.1093/dnares/dsn016'>https://doi.org/10.1093/dnares/dsn016</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 July 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 May 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 June 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="359388" href="/dnaresearch/article-pdf/15/5/309/1028712/dsn016.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kei Yura, Yuki Miyata, Tomotsugu Arikawa, Masanobu Higuchi, Mamoru Sugita;  Characteristics and Prediction of RNA Editing Sites in Transcripts of the Moss <em>Takakia lepidozioides</em> Chloroplast, <em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 309–321, <a href='https://doi.org/10.1093/dnares/dsn016'>https://doi.org/10.1093/dnares/dsn016</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/359388?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/359388?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/359388?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/359388?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/359388?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/359388?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/359388?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Characteristics%20and%20Prediction%20of%20RNA%20Editing%20Sites%20in%20Transcripts%20of%20the%20Moss%20%26lt%3Bem%26gt%3BTakakia%20lepidozioides%26lt%3B%2Fem%26gt%3B%20Chloroplast&amp;publicationDate=2008-07-23&amp;volumeNum=15&amp;issueNum=5&amp;author=Yura%2C%20Kei%3B%20Miyata%2C%20Yuki&amp;startPage=309&amp;endPage=321&amp;contentId=10.1093%2Fdnares%2Fdsn016&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f309%2f359388" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5179952 id="5179952" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>RNA editing in land plant organelles is a process primarily involving the conversion of cytidine to uridine in pre-mRNAs. The process is required for gene expression in plant organelles, because this conversion alters the encoded amino acid residues and improves the sequence identity to homologous proteins. A recent study uncovered that proteins encoded in the nuclear genome are essential for editing site recognition in chloroplasts; the mechanisms by which this recognition occurs remain unclear. To understand these mechanisms, we determined the genomic and cDNA sequences of moss <em>Takakia lepidozioides</em> chloroplast genes, then computationally analyzed the sequences within −30 to +10 nucleotides of RNA editing sites (neighbor sequences) likely to be recognized by <em>trans-</em>factors. As the <em>T</em>. <em>lepidozioides</em> chloroplast has many RNA editing sites, the analysis of these sequences provides a unique opportunity to perform statistical analyses of chloroplast RNA editing sites. We divided the 302 obtained neighbor sequences into eight groups based on sequence similarity to identify group-specific patterns. The patterns were then applied to predict novel RNA editing sites in <em>T</em>. <em>lepidozioides</em> transcripts; ∼60% of these predicted sites are true editing sites. The success of this prediction algorithm suggests that the obtained patterns are indicative of key sites recognized by <em>trans-</em>factors around editing sites of <em>T</em>. <em>lepidozioides</em> chloroplast genes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="bioinformatics">bioinformatics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="chloroplast">chloroplast</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;computational biology&quot;">computational biology</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;plant organelle&quot;">plant organelle</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;singlet and doublet propensities&quot;">singlet and doublet propensities</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Takakia lepidozioides&quot;"><em>Takakia lepidozioides</em></a></div>                            <h2 scrollto-destination=5179954 id="5179954" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Pre-mRNAs from land plant organelles frequently undergo single nucleotide conversions of cytidine residues to uridine.<sup><a reveal-id="DSN016C1" data-open="DSN016C1" class="link link-ref link-reveal xref-bibr">1</a></sup> This process, called RNA editing, was first discovered in wheat mitochondrion mRNA encoding <em>coxII</em><sup><a reveal-id="DSN016C2" data-open="DSN016C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and maize chloroplast mRNA encoding <em>rpl2</em>.<sup><a reveal-id="DSN016C3" data-open="DSN016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Many reports of RNA editing sites followed these discoveries.<sup><a reveal-id="DSN016C4  DSN016C8" data-open="DSN016C4  DSN016C8" class="link link-ref link-reveal xref-bibr">4–8</a></sup> In <em>Arabidopsis thaliana</em>, 441 nucleotides within mitochondrial mRNAs have been determined to be edited,<sup><a reveal-id="DSN016C9" data-open="DSN016C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and there are at least 26 edited sites in chloroplast transcripts from black pine.<sup><a reveal-id="DSN016C10" data-open="DSN016C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The majority of these RNA edited sites are found in protein coding regions, typically within the first or second letter of a codon; therefore, RNA editing usually alters the amino-acid sequence of the encoded protein in plant organelles. Such alterations in amino-acid sequence frequently improve sequence identity to other homologous proteins, suggesting that RNA editing is a process to repair ‘damaged’ codons.<sup><a reveal-id="DSN016C1" data-open="DSN016C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The editing process, therefore, must be highly specific.<sup><a reveal-id="DSN016C11" data-open="DSN016C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>The editing process is divided into two distinct steps: site recognition and nucleotide modification. The modification process occurs via deamination of the base,<sup><a reveal-id="DSN016C12" data-open="DSN016C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSN016C13" data-open="DSN016C13" class="link link-ref link-reveal xref-bibr">13</a></sup> although the factor(s) mediating this process in plant organelles have not yet been identified. The recognition process for both mitochondrial and chloroplast RNA editing has remained a mystery. <em>Cis</em>-elements upstream of the editing sites have been identified in multiple mRNAs.<sup><a reveal-id="DSN016C14  DSN016C20" data-open="DSN016C14  DSN016C20" class="link link-ref link-reveal xref-bibr">14–20</a></sup> Although rearrangements of nucleotides downstream of one of the editing sites in the maize <em>rps12</em> mitochondrial mRNA did not affect the editing, similar rearrangements at another editing site within this sequence altered editing,<sup><a reveal-id="DSN016C11" data-open="DSN016C11" class="link link-ref link-reveal xref-bibr">11</a></sup> suggesting that each editing site has different <em>cis</em>-elements at different positions. Miyamoto et al.<sup><a reveal-id="DSN016C21" data-open="DSN016C21" class="link link-ref link-reveal xref-bibr">21</a></sup> identified three distinct <em>trans</em>-factor proteins that bound <em>cis</em>-elements at three different editing sites, suggesting that each RNA editing site is recognized by unique proteins. In vitro assays examining RNA editing in tobacco chloroplasts identified an RNA editing site in <em>ndhB</em> that requires a short upstream region for proper editing and one in <em>ndhF</em> that requires both upstream and downstream sequences surrounding the editing site and an additional 5′ distal sequence.<sup><a reveal-id="DSN016C22" data-open="DSN016C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The <em>cis</em>-elements found by those experiments were typically located within 15–30 nucleotides of the 5′ nucleotides of the editing sites.<sup><a reveal-id="DSN016C21" data-open="DSN016C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN016C23" data-open="DSN016C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Several reports indicated nucleotide sequence similarity upstream of several different RNA editing sites,<sup><a reveal-id="DSN016C24" data-open="DSN016C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSN016C25" data-open="DSN016C25" class="link link-ref link-reveal xref-bibr">25</a></sup> but there was no obvious consensus sequence motif for all the upstream sequences.<sup><a reveal-id="DSN016C9" data-open="DSN016C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSN016C26" data-open="DSN016C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Recently, a genetic approach identified CRR4 and CRR21, members of the pentatricopeptide repeat (PPR) proteins, as essential <em>trans</em>-factors in chloroplasts for RNA editing site recognition.<sup><a reveal-id="DSN016C27  DSN016C29" data-open="DSN016C27  DSN016C29" class="link link-ref link-reveal xref-bibr">27–29</a></sup> CRR4 specifically bound the 25 nucleotides upstream and 10 nucleotides downstream of the first editing site in <em>ndhD</em> mRNA.<sup><a reveal-id="DSN016C28" data-open="DSN016C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The length of the mRNA segment protected by CRR4 binding is similar to the length suggested to be necessary for editing site recognition. <em>Arabidopsis thaliana</em> has a huge number of members of the PPR protein family,<sup><a reveal-id="DSN016C30" data-open="DSN016C30" class="link link-ref link-reveal xref-bibr">30</a></sup> proteins suggested to function in organelle biogenesis.<sup><a reveal-id="DSN016C31" data-open="DSN016C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Current advances in understanding the mechanisms of RNA editing in chloroplasts, which likely apply to mitochondrion as well, can be summarized as follows: (i) PPR proteins recognize several RNA editing sites, (ii) PPR proteins bind a region spanning from ∼30 nucleotides upstream to 10 nucleotides downstream of the editing site and (iii) the same protein may also convert the target cytidine to uridine.<sup><a reveal-id="DSN016C32" data-open="DSN016C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>Further understanding of RNA editing site recognition mechanisms in chloroplast transcripts can be achieved by computational analyses of nucleotide sequences aimed at identifying target nucleotides upstream and downstream of editing sites. For this type of analyses, multiple RNA editing sites with non-homologous nucleotide sequences are required. As the nucleotide sequences in homologous sites of homologous sequences are evolutionarily related, this conservation hampers the detection of functionally important nucleotide sequences essential in RNA editing site recognition. Although substantial known non-homologous RNA editing sites exist in mitochondrial mRNAs to allow these statistical calculations to be performed,<sup><a reveal-id="DSN016C33  DSN016C36" data-open="DSN016C33  DSN016C36" class="link link-ref link-reveal xref-bibr">33–36</a></sup> those methods have not yet been applied to chloroplast RNA editing sites from a single species due to the paucity of known RNA editing sites. The maximum number of known C-to-U RNA editing sites in a chloroplast was 44 in the moth orchid <em>Phalaenopsis aphrodite</em>.<sup><a reveal-id="DSN016C37" data-open="DSN016C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p><p>Transcripts from chloroplasts of the moss <em>Takakia lepidozioides</em> are thought to contain a large number of RNA editing sites,<sup><a reveal-id="DSN016C38" data-open="DSN016C38" class="link link-ref link-reveal xref-bibr">38</a></sup> which may provide the opportunity to perform classification and motif analyses of RNA editing site surrounding sequences (−30 to +10 nucleotide). Here, we sequenced the genomic and mRNA transcripts from <em>T</em>. <em>lepidozioides</em>. We then performed statistical analyses of RNA editing site sequences and identified statistically significant patterns in the sequences. We used these patterns to predict novel RNA editing sites and experimentally verified these predictions.</p>                            <h2 scrollto-destination=5179959 id="5179959" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5179960 id="5179960" class="section-title" data-legacy-id=s2a>2.1. Amplification of plastid DNA fragments and analysis of cDNA</h3>
<p>We extracted total cellular DNA from <em>T</em>. <em>lepidozioides</em> S. Hatt. &amp; Inoue (Takakiaceae, Bryophyta) as described previously.<sup><a reveal-id="DSN016C38" data-open="DSN016C38" class="link link-ref link-reveal xref-bibr">38</a></sup> We used the resulting sequences as a template for polymerase chain reaction (PCR) using appropriate primers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) with an initial denaturation at 95°C for 3 min, followed by 30 cycles of denaturation at 94°C for 1 min, annealing at 55°C for 1 min and extension at 72°C for 1 min, with a final extension step at 72°C for 5 min. Amplified DNA fragments were cloned into pGEM-T Easy plasmids (Promega, Madison, WI, USA). RNA extraction, first-strand cDNA synthesis and amplification of cDNA were performed as described.<sup><a reveal-id="DSN016C39" data-open="DSN016C39" class="link link-ref link-reveal xref-bibr">39</a></sup> cDNA was amplified by PCR using appropriate primers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) and subjected to direct sequencing. cDNAs encoding <em>accD</em>, <em>ndhC</em>, <em>ndhJ</em>, <em>petA</em>, <em>petL</em>, <em>petG</em>, <em>psaJ</em>, <em>rpl33</em>, <em>rps18</em>, <em>rpl20</em>, <em>5′-rps12</em> and <em>clpP</em> were cloned into pGEM-T Easy plasmids; at least two independent cDNA clones were sequenced for each cDNA.</p>                            <h3 scrollto-destination=5179962 id="5179962" class="section-title" data-legacy-id=s2b>2.2. DNA sequencing and identification of RNA editing sites</h3>
<p>DNA sequencing was performed as described<sup><a reveal-id="DSN016C39" data-open="DSN016C39" class="link link-ref link-reveal xref-bibr">39</a></sup> using M13 universal primers and internal sequence primers designed from the sequences determined in this study (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). RNA editing sites were analyzed by comparing each cDNA sequence to that of genomic DNA. Nucleotide sequence data were deposited in the DDBJ/EMBL/GenBank database under the accession numbers AB299142, AB367138 (plastid DNA), AB299143–AB299154 and AB367139–AB367144 (cDNA).</p>                            <h3 scrollto-destination=5179964 id="5179964" class="section-title" data-legacy-id=s2c>2.3. Classification of neighbor nucleotide sequences of RNA editing sites</h3>
<p>We collected sequences from each RNA editing site from 30 nucleotides upstream to 10 nucleotides downstream (41 nucleotides in total), which we named the neighbor sequence. After aligning these neighbor sequences without any gaps, we calculated the sequence identities. We used ungapped alignments here to avoid any dependency of the results on the value of gap penalty. Using dissimilarity, defined as 1.0-identity, as a distance among those sequences, we built a dendrogram using the neighbor-joining method.<sup><a reveal-id="DSN016C40" data-open="DSN016C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The dendrogram was expected to show only those sequence similarities derived from similarities in the recognition mechanism. On the basis of the obtained dendrograms, we divided the neighbor sequences into several groups at the first major branches from the root.</p>                            <h3 scrollto-destination=5179966 id="5179966" class="section-title" data-legacy-id=s2d>2.4. Singlet and doublet propensities for each group</h3>
<p>Here, we used the method of deducing functionally and structurally important residue position pairs from sets of functionally related sequences, originally proposed by Yura et al.<sup><a reveal-id="DSN016C41" data-open="DSN016C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The method used to calculate the propensities for residue positions was based on Kim et al.<sup><a reveal-id="DSN016C42" data-open="DSN016C42" class="link link-ref link-reveal xref-bibr">42</a></sup> As we used cDNA to treat the nucleotide data, uridine is hereafter denoted as thymine.</p><div class="block-child-p">The singlet propensity for each group was calculated as follows. In a group with <em>N</em> nucleotide sequences, the number of a certain nucleotide <em>x</em> (<em>x</em> = 1, 2, 3 or 4, which corresponds to A, T, G or C, respectively) at position <em>i</em> (<em>i</em> = −30, −29, −28, … , +10) was denoted as <em>n</em><sub><em>i</em></sub><sup><em>x</em></sup>. The frequency of finding nucleotide <em>x</em> at position <em>i</em> (<em>f</em><sub><em>i</em></sub><sup><em>x</em></sup>) was defined as <a id="DSN016M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN016M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn01606.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01606.gif?Expires=2147483647&amp;Signature=38evF4Hd0qXBllLICH4ivsbTKSY4lpjZ9aTEjCeAuQRGCLDsBFAxHy5mdguAiytqFw16cycfUko0L9zPB-9Q~ar5-LEd3OB~hsRWXJi8ta8VxzuJ-3VYPvClND~ZraZv9VP29J-1sM1GwPecqdj3qoVVMNhrKE0BKPp5W3ac3mnba3~6oC0hsSxbbyXaqg3hxH6kNhjrNMna3AMzKfO8~axvaD5qHbo9QrQKLj1rOfxCOUWLZ6Wz~caKPY-~fUC20g9K1A96-Tecsfdy~HSqRagM2U30T9KmjTL-QPkNKGr44Fz-dME83Z8FtnJwbE0ssHQPdFqLa4O10TzTvA30PA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn01606.gif" /></a></div></div><div class="label">1</div></div> The informative value was the deviation of <em>f</em><sub><em>i</em></sub><sup><em>x</em></sup> from the expected frequency of nucleotide <em>x</em> at position <em>i</em>, which was calculated as <a id="DSN016M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN016M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn01607.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01607.gif?Expires=2147483647&amp;Signature=vMASiYZ2M6EKva0ndT1J7yDU38yqcRgZh-SRH3oBMGrzULRklmtk8DTdf7QLY9TYnWt4EhIBhTToQfGjMI0P~ybiKvfmQ89D-dFHjWzmVV8YFOSAI1GRQwCF0A6UtJWPXfzyD4w8N~Hdhlmr83eOr7bROPgwUsfQiXTI00HjpqIKDsLpmCVXqXKSPR048yK3lhmo0l4BTdbmV5NvGrUQbp8y6dNoytL5JthGLvpx4JGgo2Ha~DQ8vNxga8-ol2TkRMTQQC4X4T5bHG3ShDtwRCyFfbkfa~ATI499lwEAokc2oXe8c1yu-JIvTYc-Z421DmlaB22nQjmCRhIxKTwXjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn01607.gif" /></a></div></div><div class="label">2</div></div> We assumed that the expected frequency did not depend on position <em>i</em> and defined the singlet propensity of nucleotide <em>x</em> at position <em>i</em> (<em>P</em><sub><em>i</em></sub><sup><em>x</em></sup>) as <a id="DSN016M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN016M3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn01608.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01608.gif?Expires=2147483647&amp;Signature=S1ODGDxb0g7DR0WfqwAemUsxIhKNvtX7-EP4UBq3v4qwagCij-olmRrNn2EC5f0d9SAyyG1P8hIN9ZdKO54b-bOQJlqQtBOjvGth1GE5ZpVjdYjWFYrWfNMxI5mU3t3iixCr1yz1HX7egnOTT~GlQLZ4Cho~r4XlMRj-ylM03Us~htacOmW~cJf8~kgSixCKKnL~wfgTq8aePwCaRUJhYygmCaHqbVqoaoacziCYf5Oo2MdHri6SUrS7lRa88cI59U0ouKwdqMmkUxMLbvOFR9v4BpzlbZn7xSJu38SBbxyiHLfOEIUV1zNRYhls75MDDPpiGI6SjT2IavprQRRQFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn01608.gif" /></a></div></div><div class="label">3</div></div> A singlet propensity <em>P</em><sub><em>i</em></sub><sup><em>x</em></sup> &gt; 1.0 indicated that nucleotide <em>x</em> was favored at position <em>i</em>, whereas values &lt;1.0 implied the nucleotide was disfavored.</div><div class="block-child-p">The doublet propensity indicates the preference for a pair of nucleotides at two different, but linked, positions within the neighbor sequence. In a group with <em>N</em> nucleotide sequences, the number of the pair of nucleotides <em>x</em> and <em>y</em> (<em>x</em>, <em>y</em> = 1, 2, 3, 4) at positions <em>i</em> and <em>j</em> (<em>i</em>, <em>j</em> = −30, −29, −28, … , +10) was denoted as <em>n</em><sub><em>i</em></sub><sub>,</sub><sub><em>j</em></sub><sup><em>x</em></sup><sup>,</sup><sup><em>y</em></sup>. The frequency of finding a pair of nucleotides <em>x</em> and <em>y</em> at positions <em>i</em> and <em>j</em>, respectively, was <a id="DSN016M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN016M4"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn01609.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01609.gif?Expires=2147483647&amp;Signature=gt9VyD8gp6916nE48ce3K52ccGlTeMTqnjmylPtiP4mKDdmYFy5WBf3O4~le~xakGoynw2kCHumBfeQfiIJFyyE~GjUfDRrpG8hveu4SrqsVbRZ6HQilbiE4esexEQUlI85tFwBuS6gwyqvWnriGYAiQTOUlcvF5ZuDSYHfaDeRByFfB2iJcJYBkiTq9mXCOKFiSeyrxR6caQ8tPVqe7UHB7iPrJGjlfoDnv5HAqJgZU3fu0j5Ck7-BbBh-1wL5gporzNhNTEAruHJ7z0CxL5WcxwoNjW1mSv9sQqbGFnywdeszf6RmFBZleOFHD33VtNeuSX0v6qnzJfHHimNy5lw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn01609.gif" /></a></div></div><div class="label">4</div></div> The informative value was the deviation of <em>f</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> from the expected frequency, which was calculated for nucleotide <em>x</em> at position <em>i</em> with nucleotide <em>y</em> at position <em>j</em> as <a id="DSN016M5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN016M5"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn01610.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01610.gif?Expires=2147483647&amp;Signature=Q5TpkQ~Kj7S6of7kdmpwN2w7JkzgNYbfRy6OlMPcM9KWEr2OPIixPomt0kcf2GMiIXpbJzdfzQujy4x33LqIYmLb-F0fK79ZHSz3RMrSij5VPRYX5q~W476a-O7jNFFL8fDV1PDQ9-fQzae3oJmhO3~FysDMzTE3dpfJn3InjVHKpw8bGe4lk0UdGvvN-pcu-JUaH6oto1I86LqyRHeYAzcZryvVEDmP0px3L9avcB4K3MxEHpdm3Xl7TlXdjFq4zhUaaZtKhC0rnxI8TJMKxje99XCBnCz-jZTGYWJtGjCJD1G0Q84quhhPgQ0sRfTTKLyqD8CuuVC6nNZNSNJF-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn01610.gif" /></a></div></div><div class="label">5</div></div> We defined the doublet propensity as <a id="DSN016M6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN016M6"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn01611.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01611.gif?Expires=2147483647&amp;Signature=nWrj7kq0d-cmOBKfYpeQVqrZLbYPXAvRmYL0mBfssE0Lhu07zhWsE3jwwCDwZwhfBG6AwT-URPBNt30uRQpIvY-0d5-M4Kn4MPKJiK-Qm4AuA16h4pBQSkRxU6lbWWzRYc-IfvvIuPz~kECsjFg4tmRe1pTSVbcNEkN5EK3ssHQDdD7Kj8xMXW7wc6AuiasDf38H8hrAkCih~XhDZtK7Xo6ySoFMrXQkmeUaXnYre4i5L3btyHnqN41aZgSmC54poVcxXOK7bLrz8N3z8ydELGg8l83ChJqEcqKKWBocs1nzV7wxZRBek4R80o2dQSNPCaxdPq4R9~VzfJ7JFiQoIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn01611.gif" /></a></div></div><div class="label">6</div></div> The doublet propensity <em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> represents the coupling constant for nucleotides <em>x</em> and <em>y</em> at positions <em>i</em> and <em>j</em>, respectively. When <em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> was &gt;1.0, the pair of nucleotides <em>x</em> and <em>y</em> was favored, whereas values &lt;1.0 indicated that this pairing was disfavored. When <em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> was equal to 1.0, no preference for a pair of nucleotides <em>x</em> and <em>y</em> at positions <em>i</em> and <em>j</em> exists.</div><p>When Equations <a href="#DSN016M1  DSN016M6" class="sectionLink xref-sec">(1)–(6)</a> were applied to the neighbor sequences of RNA editing sites from <em>T</em>. <em>lepidozioides</em> chloroplasts, the paucity of events in the count of <em>n</em><sub><em>i</em></sub><sup><em>x</em></sup> and <em>n</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> hampered our ability to obtain valid values for <em>P</em><sub><em>i</em></sub><sup><em>x</em></sup> and <em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup>. To minimize possible statistical error caused by a paucity of data, we employed pseudo-counts for <em>n</em><sub><em>i</em></sub><sup><em>x</em></sup> and <em>n</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup>.<sup><a reveal-id="DSN016C43" data-open="DSN016C43" class="link link-ref link-reveal xref-bibr">43</a></sup> These pseudo-counts should reflect the <em>a priori</em> expectations of the occurrence of the nucleotides <em>x</em> and <em>y</em>; we used a background frequency to define these <em>a priori</em> expectations. Therefore, in the real-data calculations, <em>n</em><sub><em>i</em></sub><sup><em>x</em></sup> was given by <em>n</em><sub><em>i</em></sub><sup><em>x</em></sup> + <em>b</em><sup><em>x</em></sup><em>N</em>, and <em>n</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> was given by <em>n</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> + <em>b</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup><em>N</em>.</p><p>The limited number of data reduced the statistical reliability of the calculated propensities. To estimate the standard deviation of <em>P</em><sub><em>i</em></sub><sup><em>x</em></sup> and <em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup>, we used a bootstrap procedure in which we constructed bootstrap datasets based on 1000 resamplings, then calculated bootstrap replications of <em>P</em><sub><em>i</em></sub><sup><em>x</em></sup> and <em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup>. We estimated standard deviations from these replications. We assessed the reliability of <em>P</em><sub><em>i,j</em></sub><sup><em>x</em>,<em>y</em></sup> by the 5% significance Kolmogorov–Smirnov test to examine if the values derived from the replications followed a Gaussian distribution and by the 15% bootstrap percentile method to test if the values obtained using the entire data set were within a 15% deviation of the average values calculated from the replication.<sup><a reveal-id="DSN016C42" data-open="DSN016C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                            <h3 scrollto-destination=5179975 id="5179975" class="section-title" data-legacy-id=s2e>2.5. Application of propensities to predicting new RNA editing sites from a region covering petL to clpP</h3>
<div class="block-child-p">The propensity values <em>P</em><sub><em>i</em></sub><sup><em>x</em></sup> and <em>P</em><sub><em>i,j</em></sub><sup><em>x,y</em></sup> represent the odds of a certain nucleotide sequence being present in the RNA editing site over a random sequence for nucleotides <em>x</em> and <em>y</em> at positions <em>i</em> and <em>j</em>, respectively. From the values, it is possible to calculate the odds of a given nucleotide sequence being an RNA editing site neighbor sequence over a random sequence by <a id="DSN016M7" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN016M7"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn01612.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01612.gif?Expires=2147483647&amp;Signature=4UenfLBxOFE93ZrroT4GbRoIFTHonk4fYUVXcutpgwWSO3Big95lUJOa6Dh2ScSa3X2E68qucWBUkINcrG5Y-c5J5BZEA6-tFLLri3OYigtBV-fGTBKQ8uj2ZoK-I3KK6Uhn3GtmVhP5gkYpfwFpeICnO4bpuu7Rpg-EyXMJ6JpJjbiPOzNiKwN0Cn5NW6QgbCcZTuq7c4G8pz96nn0nB070f8YSA7zVabj11SRsWg5q0GPhNfL8ktpWeF2arsOHptg92yyp4K~7he2YO3AnOMplcK3Vg5BPGCJSxqAmuersx2s4bCclNijxD2Dp843TIbB6oTyuL3rcjJfGwQFq3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn01612.gif" /></a></div></div><div class="label">7</div></div> where <em>A</em> is a given 41-nucleotide sequence in which the 31st residue was a cytidine and <em>A</em><sub><em>i</em></sub> is the type of nucleotide at the <em>i</em>th position. In the real calculations, we used the logarithmic value, defining the score for sequence <em>A</em> as <a id="DSN016M8" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN016M8"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn01613.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01613.gif?Expires=2147483647&amp;Signature=a6VkSZGYj41YRA0s8l1RvAWbQ0H8nh-rO1GkwpGWJzt-0OVuImsy0pWW-Jgc8ykZYCjdvNqeZvZm73IGH98I-Gnpz1ZZncuh0-4AO7IUXlfEfejQjHAfZyUnBLHeoby71w~7DH1kBouE516GnqMS3FrKKhTDJ2gA0EZm~ae0ANm58M6woEnRq3wNizGrdyMoOolDN476vw~QWotPdTJb-25VYyjma01SNOGnHicmtgepWSUQnx64hbJpxWb8RqU77AlkDR8E3ZZKj90OB1W2FAWB7AYrnJLrfEuwPlhDL5Rvfb5dVU6LQGLS6ps7KAFcK3oLAD9l4h1JgJYzuFo7MQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn01613.gif" /></a></div></div><div class="label">8</div></div> In the calculation, only statistically significant log<sub>2</sub><em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> values were used; non-significant log<sub>2</sub><em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> values were set to 0. <em>P</em><sub><em>i</em></sub><sup><em>x</em></sup> and <em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup> values for <em>i</em>, <em>j</em> = 0, candidate editing sites, were not used throughout these calculations, because position 0 was cytidine by definition. Greater values of <em>S</em><sub><em>A</em></sub> indicate a higher likelihood that the 31st position of the sequence is edited. We determined the threshold values for <em>S</em><sub><em>A</em></sub> based on the prediction applied to sequences used to obtain <em>P</em><sub><em>i</em></sub><sup><em>x</em></sup> and <em>P</em><sub><em>i</em>,<em>j</em></sub><sup><em>x</em>,<em>y</em></sup>.</div><p>We applied the prediction to newly sequenced transcripts from <em>T</em>. <em>lepidozioides</em> chloroplasts and experimentally verified in a blind test if the predicted RNA editing sites were actually edited.</p>                            <h2 scrollto-destination=5179978 id="5179978" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5179979 id="5179979" class="section-title" data-legacy-id=s3a>3.1. Identification of RNA editing sites in T. lepidozioides chloroplasts</h3>
<p>We previously identified 132 RNA editing sites in transcripts from the <em>psbB</em> operon and <em>rpoA</em> gene<sup><a reveal-id="DSN016C38" data-open="DSN016C38" class="link link-ref link-reveal xref-bibr">38</a></sup> as well as one site in the tRNA<sup>Leu</sup> anticodon<sup><a reveal-id="DSN016C39" data-open="DSN016C39" class="link link-ref link-reveal xref-bibr">39</a></sup> of <em>T</em>. <em>lepidozioides</em> chloroplasts. To identify additional RNA editing sites, we determined the nucleotide sequences of a 27 kb plastid DNA region from <em>rps4</em> to <em>psbE</em> (Fig. <a reveal-id="DSN016F1" data-open="DSN016F1" class="link link-reveal link-table xref-fig">1</a>) and the cognate cDNAs. To identify RNA editing sites in <em>rbcL</em>, we compared the cDNA sequences in the database (AB427089) to the sequenced plastid DNA. This comparison of plastid DNA and cDNA sequences identified 170 novel RNA editing sites in a region between <em>rps4</em> and <em>psbE</em>, including one site in the tRNA<sup>Leu</sup> anticodon (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). All sites contained C-to-U conversions. In total, there were 302 RNA editing sites, 132 previously identified and 170 newly recognized, available for computational analysis.</p>                        <a id="5179981" scrollto-destination="5179981"></a>
<div id="DSN016F1" data-id="DSN016F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01601.jpeg?Expires=2147483647&Signature=E84cJ8LT7b0T0K98bA5wRInkzZvttkQ4IZYF9pyWG1vyffqKakkMFQy4DmDS-zpCyfh5RvfNQHMiqBYt52zA-9O1Z1m7b~6-AjL67Qt1iw~A7xXXi-QBuEVPnjVWiZ7q92kU5ZH2x9M00doXDBGWEIe19XIW1Mu3xtVvCzShfQL-jOsEfQZf2wrxmQdPFEwnYe7TvFFNHM~99wm8XIk-7YLVUn3m-Oiqaf9A5i-YY9JuSt1Yrsbdg6BHE-00yquuMW0KQo7MZi9oDLU053V2qIVVm8hMQE-OAInm2p59U0bOi4wfGChaCOH~Y6LIjvuPFAjlug6FmU5ymjRvBx1zDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene arrangement of the region spanning rps4 to rps11 in the T. lepidozioides plastid genome. The filled boxes indicate the translated regions for each gene, and the open boxes represent introns. The genes over the thick horizontal bar were transcribed from left to right, whereas those below the bar were transcribed in the opposite direction. Fragments 1–19, shown under the bar, were amplified by PCR. Sequences for the dotted fragments under the bar were taken from Genbank/EMBL/DDBJ." path-from-xml="dsn01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5179981/dsn01601.jpeg" path-from-xml="dsn01601.jpeg" target="_blank">View large</a><a section="5179981" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01601.jpeg?Expires=2147483647&Signature=oOAYAptfsIsGhkSrqsGG~csFBQED6tMv2I6yrEa~MfwmYB5DQMhhQfFAErFItEdWU7cazF1RS0tSwuFDtv2GUvTCDbEj9-r4EXWlh1auqoa5mF2AEdgU61DZd1xCtLwOl6RZPB2KS5wfj4t3BvXtkhBYIGIyDGe0B-700tKAawDivmGUnvvTCMZCRLoz7GxUudxEs7OeCdJa~62yg9DZN8G6dteBvVCm3VLkhoNy1V9OjxlfPZEikqWxIIas9yWhO69tNAX1ZuCwmdejPFNILCTzLWez47xyyFHeZysmisaAIxMDsPBjcU5LtAZNNKu1Sg4GXK~mzF1ZDmDZtS1~rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5179981&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene arrangement of the region spanning <em>rps4</em> to <em>rps11</em> in the <em>T</em>. <em>lepidozioides</em> plastid genome. The filled boxes indicate the translated regions for each gene, and the open boxes represent introns. The genes over the thick horizontal bar were transcribed from left to right, whereas those below the bar were transcribed in the opposite direction. Fragments 1–19, shown under the bar, were amplified by PCR. Sequences for the dotted fragments under the bar were taken from Genbank/EMBL/DDBJ.</p></div></div><div content-id="DSN016F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01601.jpeg?Expires=2147483647&Signature=E84cJ8LT7b0T0K98bA5wRInkzZvttkQ4IZYF9pyWG1vyffqKakkMFQy4DmDS-zpCyfh5RvfNQHMiqBYt52zA-9O1Z1m7b~6-AjL67Qt1iw~A7xXXi-QBuEVPnjVWiZ7q92kU5ZH2x9M00doXDBGWEIe19XIW1Mu3xtVvCzShfQL-jOsEfQZf2wrxmQdPFEwnYe7TvFFNHM~99wm8XIk-7YLVUn3m-Oiqaf9A5i-YY9JuSt1Yrsbdg6BHE-00yquuMW0KQo7MZi9oDLU053V2qIVVm8hMQE-OAInm2p59U0bOi4wfGChaCOH~Y6LIjvuPFAjlug6FmU5ymjRvBx1zDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene arrangement of the region spanning rps4 to rps11 in the T. lepidozioides plastid genome. The filled boxes indicate the translated regions for each gene, and the open boxes represent introns. The genes over the thick horizontal bar were transcribed from left to right, whereas those below the bar were transcribed in the opposite direction. Fragments 1–19, shown under the bar, were amplified by PCR. Sequences for the dotted fragments under the bar were taken from Genbank/EMBL/DDBJ." path-from-xml="dsn01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5179981/dsn01601.jpeg" path-from-xml="dsn01601.jpeg" target="_blank">View large</a><a section="5179981" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01601.jpeg?Expires=2147483647&Signature=oOAYAptfsIsGhkSrqsGG~csFBQED6tMv2I6yrEa~MfwmYB5DQMhhQfFAErFItEdWU7cazF1RS0tSwuFDtv2GUvTCDbEj9-r4EXWlh1auqoa5mF2AEdgU61DZd1xCtLwOl6RZPB2KS5wfj4t3BvXtkhBYIGIyDGe0B-700tKAawDivmGUnvvTCMZCRLoz7GxUudxEs7OeCdJa~62yg9DZN8G6dteBvVCm3VLkhoNy1V9OjxlfPZEikqWxIIas9yWhO69tNAX1ZuCwmdejPFNILCTzLWez47xyyFHeZysmisaAIxMDsPBjcU5LtAZNNKu1Sg4GXK~mzF1ZDmDZtS1~rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5179981&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene arrangement of the region spanning <em>rps4</em> to <em>rps11</em> in the <em>T</em>. <em>lepidozioides</em> plastid genome. The filled boxes indicate the translated regions for each gene, and the open boxes represent introns. The genes over the thick horizontal bar were transcribed from left to right, whereas those below the bar were transcribed in the opposite direction. Fragments 1–19, shown under the bar, were amplified by PCR. Sequences for the dotted fragments under the bar were taken from Genbank/EMBL/DDBJ.</p></div></div>                            <h3 scrollto-destination=5179982 id="5179982" class="section-title" data-legacy-id=s3b>3.2. Classification of 302 nucleotide sequences with RNA editing site</h3>
<p>We classified the 302 41-nucleotide sequences with the RNA editing site at the 31st position (neighbor sequences) from <em>T</em>. <em>lepidozioides</em> chloroplast (Fig. <a reveal-id="DSN016F2" data-open="DSN016F2" class="link link-reveal link-table xref-fig">2</a>) in a dendrogram based on sequence identity. Grouping of the sequences, visualized by background color, was based on branching of the dendrogram. The threshold for grouping of the sequences was set at the second branching point of the dendrogram from the root. The 302 initial sequences were divided into eight groups, named G-1 to G-8. The largest group (G-1 plus G-2) was further divided into two groups, as the sequence identities within this group were very low. The average number of sequences in each group was ∼38, ranging from 21 to 57. Within each group, the sequences had a sequence identity with the other sequences in that group of at least 40%.</p>                        <a id="5179984" scrollto-destination="5179984"></a>
<div id="DSN016F2" data-id="DSN016F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01602.jpeg?Expires=2147483647&Signature=ZtZDZfWLbk2EY7MlJupcroWjTj14qMjF3EUh4WoWJgx6RL~u0TAtMEs~bvO4n8F0kZDl1ZD-n91rXfwF4juAvyy52GyG~1vcg3K9DYR4syKJWSgmd3bKDVmdF8C7SWyZ2C9h3-lmUsIvlNHLEAtqoimkBTMa5XrXRalFxpiNv1lNbmUXfU51LechTVVM7xLOzhMcQIYjA-tUWfMqGv9RkwoU9jhq-aimBj5t1iEW3u-wheOTbQl~mlwsvdRmLhypq2GRJSIaHu99rB~R30-5K6G9P49fxzZGEsNGCcojM5Q-6Zd0u7f2lKZWO295qgPLOAMKySLHwC1nZ2Mf~faXxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification of neighbor sequences for RNA editing sites. We drew the classification dendrogram for 41-nt sequences based on sequence identity. The sequences were grouped at the second or third branching points. Each group, named from G-1 to G-8, was depicted by a background color. Each sequence was named based on the sequenced region and number of nucleotides. Note that the figure does not express the phylogenetic relation of the sequences." path-from-xml="dsn01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5179984/dsn01602.jpeg" path-from-xml="dsn01602.jpeg" target="_blank">View large</a><a section="5179984" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01602.jpeg?Expires=2147483647&Signature=QLqyTgVIn1gQajIVsA4m7gSHQZrCpqNpl3IFhqc1Mk5Lv3r8tbDkCBjm0akeezUcBhCwy71nfNP4IXnU-w7PDiL99KtDr3fxfcmhk0ArRkDiKxgMAIPxZHe7IhW0X94XRvu95jNnRirSrt1PfK2YlXm5AykvimMncRFRUot31WiWqSxuFo1YlzofXwGUcv4wUCk6qN04WaBk56dWnm1A-SMxlOIwoNIW~-KfNSVUsL5j6lS3A7liAaJnHJjF6uBbKc546nuUOBoQE5q1nNFYMd6XzVwJLYy4Z~jWW7cgNPs0VJV3XEAHml9r~44Vgx7oJOi22Rv~4xT7rax-rcpAUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5179984&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification of neighbor sequences for RNA editing sites. We drew the classification dendrogram for 41-nt sequences based on sequence identity. The sequences were grouped at the second or third branching points. Each group, named from G-1 to G-8, was depicted by a background color. Each sequence was named based on the sequenced region and number of nucleotides. Note that the figure does not express the phylogenetic relation of the sequences.</p></div></div><div content-id="DSN016F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01602.jpeg?Expires=2147483647&Signature=ZtZDZfWLbk2EY7MlJupcroWjTj14qMjF3EUh4WoWJgx6RL~u0TAtMEs~bvO4n8F0kZDl1ZD-n91rXfwF4juAvyy52GyG~1vcg3K9DYR4syKJWSgmd3bKDVmdF8C7SWyZ2C9h3-lmUsIvlNHLEAtqoimkBTMa5XrXRalFxpiNv1lNbmUXfU51LechTVVM7xLOzhMcQIYjA-tUWfMqGv9RkwoU9jhq-aimBj5t1iEW3u-wheOTbQl~mlwsvdRmLhypq2GRJSIaHu99rB~R30-5K6G9P49fxzZGEsNGCcojM5Q-6Zd0u7f2lKZWO295qgPLOAMKySLHwC1nZ2Mf~faXxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification of neighbor sequences for RNA editing sites. We drew the classification dendrogram for 41-nt sequences based on sequence identity. The sequences were grouped at the second or third branching points. Each group, named from G-1 to G-8, was depicted by a background color. Each sequence was named based on the sequenced region and number of nucleotides. Note that the figure does not express the phylogenetic relation of the sequences." path-from-xml="dsn01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5179984/dsn01602.jpeg" path-from-xml="dsn01602.jpeg" target="_blank">View large</a><a section="5179984" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01602.jpeg?Expires=2147483647&Signature=QLqyTgVIn1gQajIVsA4m7gSHQZrCpqNpl3IFhqc1Mk5Lv3r8tbDkCBjm0akeezUcBhCwy71nfNP4IXnU-w7PDiL99KtDr3fxfcmhk0ArRkDiKxgMAIPxZHe7IhW0X94XRvu95jNnRirSrt1PfK2YlXm5AykvimMncRFRUot31WiWqSxuFo1YlzofXwGUcv4wUCk6qN04WaBk56dWnm1A-SMxlOIwoNIW~-KfNSVUsL5j6lS3A7liAaJnHJjF6uBbKc546nuUOBoQE5q1nNFYMd6XzVwJLYy4Z~jWW7cgNPs0VJV3XEAHml9r~44Vgx7oJOi22Rv~4xT7rax-rcpAUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5179984&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification of neighbor sequences for RNA editing sites. We drew the classification dendrogram for 41-nt sequences based on sequence identity. The sequences were grouped at the second or third branching points. Each group, named from G-1 to G-8, was depicted by a background color. Each sequence was named based on the sequenced region and number of nucleotides. Note that the figure does not express the phylogenetic relation of the sequences.</p></div></div><p>Previous reports have been unable to identify common motifs in the neighbor sequences of RNA editing sites. The dendrogram in Fig. <a reveal-id="DSN016F2" data-open="DSN016F2" class="link link-reveal link-table xref-fig">2</a> demonstrates that although none of the neighbor sequences for RNA editing sites are similar, the sequences can be grouped into at least eight different patterns. These groups likely correspond to the neighbor sequence of RNA editing sites recognized by different <em>trans</em>-factors. Unique or similar <em>trans</em>-factors may recognize the neighbor sequences of each group; the classification derived from the dendrogram suggests that each <em>trans</em>-factor recognizes different neighbor sequences. To investigate this possibility, we analyzed the singlet and doublet propensities for each group and compared similarities in propensities within a group and differences between groups.</p>                            <h3 scrollto-destination=5179986 id="5179986" class="section-title" data-legacy-id=s3c>3.3. Singlet and doublet propensities of each group</h3>
<p>Singlet propensities for each group, shown in Fig. <a reveal-id="DSN016F3" data-open="DSN016F3" class="link link-reveal link-table xref-fig">3</a>A, are expressed as a preference for individual nucleotides at each position in the neighbor sequence color-coded from red to blue on a log<sub>2</sub> scale, with the exception of the RNA editing site (position 0). When statistically significant support could not be obtained for the singlet value of a nucleotide at that site, the log<sub>2</sub> scaled singlet value was set as 0 and colored white. A deep red box indicates that the nucleotide appeared at that position twice as frequently as expected from a random distribution; a deep blue box indicates that the nucleotide appeared half as frequently as expected.</p>                        <a id="5179988" scrollto-destination="5179988"></a>
<div id="DSN016F3" data-id="DSN016F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01603.jpeg?Expires=2147483647&Signature=Jp7bG3e08Ob1ONXNkq6~au2auZM4Z4Hg2kyaeGxKnzR6CpnXP4r8T2viF1QNBT9xYbg~ZZg8-ylzjadiRgDUjTO6FdvbqwRG-nqkxZcL6RDRqjSoPTpKpsdb8O5bn-7tDfazVLixQw4XSfdc2-0TD5b1sa3KICx6Lfz3Pu0JbimulTPlBqy1WurbXqXqR1PIjueW3UC01N2V2kLkLhQN-ynFJGfAZZgZwMoZrl0MQWuWf--7z3Q8RhaeXdn8~k7du7z9myiCqVz~5Mwq4hc39snSFa8hdawfpZ2Nv4kaXHAndxsIzfjbSyWlmqFN19NP4Vetpe80liqheUA8loEL9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Singlet propensities of eight groups at each position for every nucleotide. The preference was color-coded from red to blue on a log2 scale, with the exception of the RNA editing site (position 0). When the value was not statistically significant, the singlet value was set to 0 and colored as white. The number of sequences in each group is noted at the bottom of each diagram. (B) All 53 neighbor sequences in group G-2. On the basis of this alignment, we calculated the singlet propensities for G-2 in A. Strong tendencies for specific nucleotides, found in A, at positions −10, −8, −5, −2 and +2 as well as the RNA editing site are boxed. At positions −10 and +2, G is favored, whereas T is favored at −2. At −8, G is avoided, and at −5, C is avoided. C at 0 is edited to U." path-from-xml="dsn01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5179988/dsn01603.jpeg" path-from-xml="dsn01603.jpeg" target="_blank">View large</a><a section="5179988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01603.jpeg?Expires=2147483647&Signature=dWacyAvkf5RfcVFVevZx~-rtcPqqegUyN7xgtz0-iqQFznQ0J501LMpub9bR2SnXdmZ1bkRsFMFJLUz-SNZMuR4WoUUGE6YCFz71plrrV5WU5zmn96xbCmsqjN0t-dsSrXaic-ZwBEk0tuQGjS7xA9J998m5i23oy7~q0N44ywQJhKSlMhcbmg5Fr9F0kR3ZnOakLMKPG08851FW4V9Mqq0hTGGnmsH8INmxrater3zCaeH86RSF5c91euJMnMhuH6frnUI2XtIzHJFjnKuvT9IH8pQQQFi8VjgSNVpGKczHq9az69GQerfeNJBdYQ84tWiCQN6WVs2rW2f2BcDQuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5179988&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>) Singlet propensities of eight groups at each position for every nucleotide. The preference was color-coded from red to blue on a log<sub>2</sub> scale, with the exception of the RNA editing site (position 0). When the value was not statistically significant, the singlet value was set to 0 and colored as white. The number of sequences in each group is noted at the bottom of each diagram. (<strong>B</strong>) All 53 neighbor sequences in group G-2. On the basis of this alignment, we calculated the singlet propensities for G-2 in <strong>A.</strong> Strong tendencies for specific nucleotides, found in <strong>A,</strong> at positions −10, −8, −5, −2 and +2 as well as the RNA editing site are boxed. At positions −10 and +2, G is favored, whereas T is favored at −2. At −8, G is avoided, and at −5, C is avoided. C at 0 is edited to U.</p></div></div><div content-id="DSN016F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01603.jpeg?Expires=2147483647&Signature=Jp7bG3e08Ob1ONXNkq6~au2auZM4Z4Hg2kyaeGxKnzR6CpnXP4r8T2viF1QNBT9xYbg~ZZg8-ylzjadiRgDUjTO6FdvbqwRG-nqkxZcL6RDRqjSoPTpKpsdb8O5bn-7tDfazVLixQw4XSfdc2-0TD5b1sa3KICx6Lfz3Pu0JbimulTPlBqy1WurbXqXqR1PIjueW3UC01N2V2kLkLhQN-ynFJGfAZZgZwMoZrl0MQWuWf--7z3Q8RhaeXdn8~k7du7z9myiCqVz~5Mwq4hc39snSFa8hdawfpZ2Nv4kaXHAndxsIzfjbSyWlmqFN19NP4Vetpe80liqheUA8loEL9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Singlet propensities of eight groups at each position for every nucleotide. The preference was color-coded from red to blue on a log2 scale, with the exception of the RNA editing site (position 0). When the value was not statistically significant, the singlet value was set to 0 and colored as white. The number of sequences in each group is noted at the bottom of each diagram. (B) All 53 neighbor sequences in group G-2. On the basis of this alignment, we calculated the singlet propensities for G-2 in A. Strong tendencies for specific nucleotides, found in A, at positions −10, −8, −5, −2 and +2 as well as the RNA editing site are boxed. At positions −10 and +2, G is favored, whereas T is favored at −2. At −8, G is avoided, and at −5, C is avoided. C at 0 is edited to U." path-from-xml="dsn01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5179988/dsn01603.jpeg" path-from-xml="dsn01603.jpeg" target="_blank">View large</a><a section="5179988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01603.jpeg?Expires=2147483647&Signature=dWacyAvkf5RfcVFVevZx~-rtcPqqegUyN7xgtz0-iqQFznQ0J501LMpub9bR2SnXdmZ1bkRsFMFJLUz-SNZMuR4WoUUGE6YCFz71plrrV5WU5zmn96xbCmsqjN0t-dsSrXaic-ZwBEk0tuQGjS7xA9J998m5i23oy7~q0N44ywQJhKSlMhcbmg5Fr9F0kR3ZnOakLMKPG08851FW4V9Mqq0hTGGnmsH8INmxrater3zCaeH86RSF5c91euJMnMhuH6frnUI2XtIzHJFjnKuvT9IH8pQQQFi8VjgSNVpGKczHq9az69GQerfeNJBdYQ84tWiCQN6WVs2rW2f2BcDQuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5179988&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>) Singlet propensities of eight groups at each position for every nucleotide. The preference was color-coded from red to blue on a log<sub>2</sub> scale, with the exception of the RNA editing site (position 0). When the value was not statistically significant, the singlet value was set to 0 and colored as white. The number of sequences in each group is noted at the bottom of each diagram. (<strong>B</strong>) All 53 neighbor sequences in group G-2. On the basis of this alignment, we calculated the singlet propensities for G-2 in <strong>A.</strong> Strong tendencies for specific nucleotides, found in <strong>A,</strong> at positions −10, −8, −5, −2 and +2 as well as the RNA editing site are boxed. At positions −10 and +2, G is favored, whereas T is favored at −2. At −8, G is avoided, and at −5, C is avoided. C at 0 is edited to U.</p></div></div><p>Throughout the eight groups, several characteristics repeatedly observed from the singlet propensity values. At position −1, purine nucleotides were strongly avoided (deep blue) in all groups with favoring of pyrimidines, especially T.<sup><a reveal-id="DSN016C1" data-open="DSN016C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSN016C23" data-open="DSN016C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSN016C44" data-open="DSN016C44" class="link link-ref link-reveal xref-bibr">44</a></sup> At position 1, C is unfavorable.<sup><a reveal-id="DSN016C1" data-open="DSN016C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSN016C23" data-open="DSN016C23" class="link link-ref link-reveal xref-bibr">23</a></sup> With these two exceptions, we could not identify characteristics common to all the groups. Each group, however, has unique characteristics. In G-1, T at −26 and G at −4 and +8 were strongly favored, with G at −17, C at −14 and G at −5 being strongly disfavored. In G-2, G at −10, T at −2 and G at +2 were favored, and G at −8, C at −5 and G at −2 were disfavored. In G-3, G was favored at −25, whereas C at −23 and G at −21 and −1 were disfavored. In G-4, G at −13, A at −11 and G at −4 were favored, and G at −21, C at −18, C at −5, A/T at −4, G at −2 and A/G at −1 were all disfavored. In G-5, A at −26, C at −13, A at −12, G at −5 and A/G at −1 were disfavored. In G-6, G was favored at +1, but disfavored at −25 and −3. In G-7, G was favored at −30, and C at −28 and G at −5 were disfavored. In G-8, G was favored at −24, and G at −19, C at −12, G at −11, A at −7 and A at −1 were disfavored. The skewed distribution of nucleotides at each position may work as a marker for each RNA editing site facilitating the binding of site recognition <em>trans</em>-factors. G-1, G-2 and G-6 had strong markers both upstream and downstream of the editing site. The markers for G-2 sequences were located close to the RNA editing site, whereas the markers for G-3 were located remotely from the RNA editing sites. In contrast, the markers for G-6 and G-7 were sparsely located throughout the neighbor sequence. Those markers for G-2 are shown in Fig. <a reveal-id="DSN016F3" data-open="DSN016F3" class="link link-reveal link-table xref-fig">3</a>B. Strong singlet propensities demonstrate conservation, as shown in red boxes, but do not necessarily mean perfect conservation. At position −8, singlet propensities describe that G is disfavored, appearing only once in the alignment. At position −5, C also appeared only once.</p><p>RNA editing sites are frequently found at the second letter of a codon; the distribution of singlet propensity may reflect codon usage. If so, then the distribution pattern should have a period of three nucleotides in all places within the sequence. The patterns we observed, however, were free from periodicity and do not reflect the pattern of codons (Fig. <a reveal-id="DSN016F3" data-open="DSN016F3" class="link link-reveal link-table xref-fig">3</a>A).</p><p>As seen for the singlet propensities, strong favoring or disfavoring typically described G and C, suggesting the existence of base pair patterns in the neighbor sequences of RNA editing sites. We therefore analyzed the doublet propensities for each group (Fig. <a reveal-id="DSN016F4" data-open="DSN016F4" class="link link-reveal link-table xref-fig">4</a>). For each pair position, there are 16 (=4 × 4) possible combinations of nucleotides. When no statistical significance could be achieved for the doublet propensity value of a given nucleotide pair position, the log<sub>2</sub> scaled doublet propensity value was set to 0. We calculated the square sum of all log<sub>2</sub>-scaled positive and negative doublet propensities, which are color-coded in the upper and lower triangular portions of the matrix, respectively. All of the groups had pairs of positions with high positive values; high negative values were rarely seen in any of the groups. High values in the far off-diagonal portions of the matrix suggested that there would be long-range correlations in the neighbor sequences of RNA editing sites. An intriguing point is that high positive values were found at pairs of nucleotides located upstream and downstream of the RNA editing site.</p>                        <a id="5179992" scrollto-destination="5179992"></a>
<div id="DSN016F4" data-id="DSN016F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01604.jpeg?Expires=2147483647&Signature=E7ddWOD8Ip2kPunLfzyk3-UE9Jl3f8ESVwupJl-I6Oo2ZODk5RmOQN8JrgLIDDYvrQbXn-UKT6DUHlza9ZOOjfRddaor3DAienHSd0~Bcl8UaNdUh~rakV-rFZRQ1cEFH7qWm9hoAX8OkZrYJef9ROz-c~yNb-A2b5SajLVKaBkfJpUsfokaPOj6VfLcZt6ZVxxs8tjs2~t1hcDJkAzd0wLPwrztZb281gf~z~vT1eU4lCOQ795Bk8RwileFlw5asgLRlmMIZKRUbD7FnICbRNslusr5FUcPcoMHNA5b6HX4F6doPWEjk3B3qwKY6Yo2J28HEOVa8vtcOh48F7NUfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The magnitude of doublet propensities for the eight groups in a log2 scale at each pair of positions. At each position pair, there are 16 possible combinations of nucleotides. Each combination has a value for doublet propensity. When the value was not statistically significant, it was set to 0. A square sum of positive doublet propensity at each position was color-coded from white to red in the upper triangular portion of the matrix. A square sum of negative doublet propensity at each position was color-coded from white to blue in the lower triangular portion of the matrix. The number of statistically significant values for doublet propensity is noted at the top of each matrix. The maximum number of doublet propensities should be 12 480 (=40 nt × 39 nt/2 × 16 pairs)." path-from-xml="dsn01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5179992/dsn01604.jpeg" path-from-xml="dsn01604.jpeg" target="_blank">View large</a><a section="5179992" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01604.jpeg?Expires=2147483647&Signature=J0xSKU8xOCzKQ6r0AC7Ubofh-HCgpPxr~eJ4IygOE5nPY0avoTzGcFhsiHdllxd03SuNlkxKv61x6Y7LUjehfW-UPgksNAGGWmE22UPO3zz60SzTm5t0Ma6sp8y41s8xiUx9s3ryyAoXQGRC-PYXnH-LNp1kLhcpflbk5Z2NFMVixymTWxI6dA9hP2~W3L5B0GeuwlFmPqgoTIVSy88FvfxliunpLDOh2njUhcOYOnDKs3ey5sBA4qL8Muf4fww1af52QDL6geK9MpSDF9zF9QXHrFQxRUOKtXlk~epCjI62X8nDWlK7DMZl-RdMmYxhHVPq5UVPdgs1QQ4ln~o-aQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5179992&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The magnitude of doublet propensities for the eight groups in a log<sub>2</sub> scale at each pair of positions. At each position pair, there are 16 possible combinations of nucleotides. Each combination has a value for doublet propensity. When the value was not statistically significant, it was set to 0. A square sum of positive doublet propensity at each position was color-coded from white to red in the upper triangular portion of the matrix. A square sum of negative doublet propensity at each position was color-coded from white to blue in the lower triangular portion of the matrix. The number of statistically significant values for doublet propensity is noted at the top of each matrix. The maximum number of doublet propensities should be 12 480 (=40 nt × 39 nt/2 × 16 pairs).</p></div></div><div content-id="DSN016F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01604.jpeg?Expires=2147483647&Signature=E7ddWOD8Ip2kPunLfzyk3-UE9Jl3f8ESVwupJl-I6Oo2ZODk5RmOQN8JrgLIDDYvrQbXn-UKT6DUHlza9ZOOjfRddaor3DAienHSd0~Bcl8UaNdUh~rakV-rFZRQ1cEFH7qWm9hoAX8OkZrYJef9ROz-c~yNb-A2b5SajLVKaBkfJpUsfokaPOj6VfLcZt6ZVxxs8tjs2~t1hcDJkAzd0wLPwrztZb281gf~z~vT1eU4lCOQ795Bk8RwileFlw5asgLRlmMIZKRUbD7FnICbRNslusr5FUcPcoMHNA5b6HX4F6doPWEjk3B3qwKY6Yo2J28HEOVa8vtcOh48F7NUfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The magnitude of doublet propensities for the eight groups in a log2 scale at each pair of positions. At each position pair, there are 16 possible combinations of nucleotides. Each combination has a value for doublet propensity. When the value was not statistically significant, it was set to 0. A square sum of positive doublet propensity at each position was color-coded from white to red in the upper triangular portion of the matrix. A square sum of negative doublet propensity at each position was color-coded from white to blue in the lower triangular portion of the matrix. The number of statistically significant values for doublet propensity is noted at the top of each matrix. The maximum number of doublet propensities should be 12 480 (=40 nt × 39 nt/2 × 16 pairs)." path-from-xml="dsn01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5179992/dsn01604.jpeg" path-from-xml="dsn01604.jpeg" target="_blank">View large</a><a section="5179992" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01604.jpeg?Expires=2147483647&Signature=J0xSKU8xOCzKQ6r0AC7Ubofh-HCgpPxr~eJ4IygOE5nPY0avoTzGcFhsiHdllxd03SuNlkxKv61x6Y7LUjehfW-UPgksNAGGWmE22UPO3zz60SzTm5t0Ma6sp8y41s8xiUx9s3ryyAoXQGRC-PYXnH-LNp1kLhcpflbk5Z2NFMVixymTWxI6dA9hP2~W3L5B0GeuwlFmPqgoTIVSy88FvfxliunpLDOh2njUhcOYOnDKs3ey5sBA4qL8Muf4fww1af52QDL6geK9MpSDF9zF9QXHrFQxRUOKtXlk~epCjI62X8nDWlK7DMZl-RdMmYxhHVPq5UVPdgs1QQ4ln~o-aQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5179992&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The magnitude of doublet propensities for the eight groups in a log<sub>2</sub> scale at each pair of positions. At each position pair, there are 16 possible combinations of nucleotides. Each combination has a value for doublet propensity. When the value was not statistically significant, it was set to 0. A square sum of positive doublet propensity at each position was color-coded from white to red in the upper triangular portion of the matrix. A square sum of negative doublet propensity at each position was color-coded from white to blue in the lower triangular portion of the matrix. The number of statistically significant values for doublet propensity is noted at the top of each matrix. The maximum number of doublet propensities should be 12 480 (=40 nt × 39 nt/2 × 16 pairs).</p></div></div><p>There were multiple specific nucleotide pairs with high doublet propensities (Table <a reveal-id="DSN016TB1" data-open="DSN016TB1" class="link link-reveal link-table xref-fig">1</a>). For the 65 pairs with positions of log<sub>2</sub> doublet propensities &gt;0.9, the appearance of the nucleotide pair is ∼1.87 times more frequent than random occurrence. Of the 65 cases, only 18 cases (28%) exhibited a potential for Watson–Crick base pairing. The remaining 47 cases were unable to form Watson–Crick base pairs. Twenty-nine pairs (45%) were formed by the same nucleotide, suggesting that the pair preferences observed for that nucleotide is not due to base pairing, but to other unknown causes, possibly restrictions stemming from the interactions between these positions and a <em>trans</em>-factor.</p>                        <a id="5179994" scrollto-destination="5179994"></a>
<div content-id="DSN016TB1" class="table-modal"><div class="table-wrap"><div id="DSN016TB1" class="table-wrap-title" data-id="DSN016TB1"><div class="label -label">Table 1</div><div class="caption"><p>Log<sub>2</sub> doublet propensities &gt;0.9 in each group</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Group </th><th>nt1* </th><th>nt2* </th><th>Pair* </th><th colspan="2">Log<sub>2</sub> doublet propensity </th></tr></thead><tbody><tr><td>G-1 </td><td>−30 </td><td>9 </td><td>(C–G) </td><td>0.94 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−25 </td><td>−24 </td><td>(G–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td> </td><td>−22 </td><td>(G–G) </td><td>1.02 </td><td> </td></tr><tr><td> </td><td>−22 </td><td>−21 </td><td>(C–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−10 </td><td>5 </td><td>(G–C) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>6 </td><td>(C–C) </td><td>0.90 </td><td> </td></tr><tr><td> </td><td>−3 </td><td>−2 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td>G-2 </td><td>−25 </td><td>−10 </td><td>(C–C) </td><td>0.94 </td><td> </td></tr><tr><td>G-3 </td><td>−16 </td><td>5 </td><td>(G–G) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−10 </td><td>−5 </td><td>(C–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>6 </td><td>8 </td><td>(G–G) </td><td>0.93 </td><td> </td></tr><tr><td>G-4 </td><td>−28 </td><td>−25 </td><td>(C–G) </td><td>1.07 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−27 </td><td>4 </td><td>(C–A) </td><td>0.99 </td><td> </td></tr><tr><td> </td><td> </td><td>5 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−26 </td><td>10 </td><td>(C–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−25 </td><td>−9 </td><td>(G–A) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−5 </td><td>(C–C) </td><td>0.99 </td><td> </td></tr><tr><td> </td><td>−19 </td><td>4 </td><td>(A–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td> </td><td>5 </td><td>(A–A) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>5 </td><td>(A–A) </td><td>1.05 </td><td> </td></tr><tr><td> </td><td>−9 </td><td>2 </td><td>(A–C) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−8 </td><td>5 </td><td>(G–G) </td><td>0.94 </td><td> </td></tr><tr><td> </td><td>−6 </td><td>9 </td><td>(C–C) </td><td>0.94 </td><td> </td></tr><tr><td> </td><td>−2 </td><td>5 </td><td>(A–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>1 </td><td>10 </td><td>(A–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>4 </td><td>5 </td><td>(C–A) </td><td>0.94 </td><td> </td></tr><tr><td>G-5 </td><td>−27 </td><td>5 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−26 </td><td>−16 </td><td>(C–C) </td><td>0.98 </td><td> </td></tr><tr><td> </td><td>−24 </td><td>−16 </td><td>(G–G) </td><td>0.98 </td><td> </td></tr><tr><td> </td><td>−22 </td><td>2 </td><td>(A–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−20 </td><td>(C–C) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td>−16 </td><td>−14 </td><td>(C–C) </td><td>1.05 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>−8 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−13 </td><td>−9 </td><td>(G–A) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>3 </td><td>8 </td><td>(A–C) </td><td>1.02 </td><td> </td></tr><tr><td>G-6 </td><td>−29 </td><td>−13 </td><td>(G–G) </td><td>0.90 </td><td> </td></tr><tr><td> </td><td>−23 </td><td>−15 </td><td>(C–G) </td><td>0.94 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−22 </td><td>−6 </td><td>(C–G) </td><td>0.93 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−20 </td><td>−14 </td><td>(C–C) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td> </td><td>−4 </td><td>(C–T) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−18 </td><td>6 </td><td>(C–C) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>−13 </td><td>3 </td><td>(C–C) </td><td>1.21 </td><td> </td></tr><tr><td> </td><td>−8 </td><td>−3 </td><td>(G–C) </td><td>0.98 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−3 </td><td>4 </td><td>(C–A) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td> </td><td>7 </td><td>(C–A) </td><td>1.06 </td><td> </td></tr><tr><td>G-7 </td><td>−27 </td><td>−15 </td><td>(G–G) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td> </td><td>−9 </td><td>(C–G) </td><td>0.95 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−18 </td><td>9 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td>G-8 </td><td>−27 </td><td>−6 </td><td>(A–C) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>−26 </td><td>−15 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−24 </td><td>−17 </td><td>(G–C) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>3 </td><td>(G–C) </td><td>1.00 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−23 </td><td>−8 </td><td>(G–G) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−2 </td><td>(G–C) </td><td>0.91 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−17 </td><td>−16 </td><td>(G–C) </td><td>0.93 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>3 </td><td>(G–G) </td><td>1.06 </td><td> </td></tr><tr><td> </td><td> </td><td>8 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>2 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−14 </td><td>4 </td><td>(G–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>−13 </td><td>2 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td> </td><td>4 </td><td>(G–G) </td><td>1.03 </td><td> </td></tr><tr><td> </td><td>−6 </td><td>8 </td><td>(C–G) </td><td>1.06 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−2 </td><td>8 </td><td>(T–G) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>2 </td><td>5 </td><td>(C–A) </td><td>0.93 </td><td> </td></tr><tr><td> </td><td> </td><td>8 </td><td>(G–G) </td><td>1.02 </td><td> </td></tr></tbody></table></div><div content-id="DSN016TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Group </th><th>nt1* </th><th>nt2* </th><th>Pair* </th><th colspan="2">Log<sub>2</sub> doublet propensity </th></tr></thead><tbody><tr><td>G-1 </td><td>−30 </td><td>9 </td><td>(C–G) </td><td>0.94 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−25 </td><td>−24 </td><td>(G–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td> </td><td>−22 </td><td>(G–G) </td><td>1.02 </td><td> </td></tr><tr><td> </td><td>−22 </td><td>−21 </td><td>(C–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−10 </td><td>5 </td><td>(G–C) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>6 </td><td>(C–C) </td><td>0.90 </td><td> </td></tr><tr><td> </td><td>−3 </td><td>−2 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td>G-2 </td><td>−25 </td><td>−10 </td><td>(C–C) </td><td>0.94 </td><td> </td></tr><tr><td>G-3 </td><td>−16 </td><td>5 </td><td>(G–G) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−10 </td><td>−5 </td><td>(C–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>6 </td><td>8 </td><td>(G–G) </td><td>0.93 </td><td> </td></tr><tr><td>G-4 </td><td>−28 </td><td>−25 </td><td>(C–G) </td><td>1.07 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−27 </td><td>4 </td><td>(C–A) </td><td>0.99 </td><td> </td></tr><tr><td> </td><td> </td><td>5 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−26 </td><td>10 </td><td>(C–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−25 </td><td>−9 </td><td>(G–A) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−5 </td><td>(C–C) </td><td>0.99 </td><td> </td></tr><tr><td> </td><td>−19 </td><td>4 </td><td>(A–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td> </td><td>5 </td><td>(A–A) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>5 </td><td>(A–A) </td><td>1.05 </td><td> </td></tr><tr><td> </td><td>−9 </td><td>2 </td><td>(A–C) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−8 </td><td>5 </td><td>(G–G) </td><td>0.94 </td><td> </td></tr><tr><td> </td><td>−6 </td><td>9 </td><td>(C–C) </td><td>0.94 </td><td> </td></tr><tr><td> </td><td>−2 </td><td>5 </td><td>(A–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>1 </td><td>10 </td><td>(A–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>4 </td><td>5 </td><td>(C–A) </td><td>0.94 </td><td> </td></tr><tr><td>G-5 </td><td>−27 </td><td>5 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−26 </td><td>−16 </td><td>(C–C) </td><td>0.98 </td><td> </td></tr><tr><td> </td><td>−24 </td><td>−16 </td><td>(G–G) </td><td>0.98 </td><td> </td></tr><tr><td> </td><td>−22 </td><td>2 </td><td>(A–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−20 </td><td>(C–C) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td>−16 </td><td>−14 </td><td>(C–C) </td><td>1.05 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>−8 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−13 </td><td>−9 </td><td>(G–A) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>3 </td><td>8 </td><td>(A–C) </td><td>1.02 </td><td> </td></tr><tr><td>G-6 </td><td>−29 </td><td>−13 </td><td>(G–G) </td><td>0.90 </td><td> </td></tr><tr><td> </td><td>−23 </td><td>−15 </td><td>(C–G) </td><td>0.94 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−22 </td><td>−6 </td><td>(C–G) </td><td>0.93 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−20 </td><td>−14 </td><td>(C–C) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td> </td><td>−4 </td><td>(C–T) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−18 </td><td>6 </td><td>(C–C) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>−13 </td><td>3 </td><td>(C–C) </td><td>1.21 </td><td> </td></tr><tr><td> </td><td>−8 </td><td>−3 </td><td>(G–C) </td><td>0.98 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−3 </td><td>4 </td><td>(C–A) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td> </td><td>7 </td><td>(C–A) </td><td>1.06 </td><td> </td></tr><tr><td>G-7 </td><td>−27 </td><td>−15 </td><td>(G–G) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td> </td><td>−9 </td><td>(C–G) </td><td>0.95 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−18 </td><td>9 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td>G-8 </td><td>−27 </td><td>−6 </td><td>(A–C) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>−26 </td><td>−15 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−24 </td><td>−17 </td><td>(G–C) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>3 </td><td>(G–C) </td><td>1.00 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−23 </td><td>−8 </td><td>(G–G) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−2 </td><td>(G–C) </td><td>0.91 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−17 </td><td>−16 </td><td>(G–C) </td><td>0.93 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>3 </td><td>(G–G) </td><td>1.06 </td><td> </td></tr><tr><td> </td><td> </td><td>8 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>2 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−14 </td><td>4 </td><td>(G–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>−13 </td><td>2 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td> </td><td>4 </td><td>(G–G) </td><td>1.03 </td><td> </td></tr><tr><td> </td><td>−6 </td><td>8 </td><td>(C–G) </td><td>1.06 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−2 </td><td>8 </td><td>(T–G) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>2 </td><td>5 </td><td>(C–A) </td><td>0.93 </td><td> </td></tr><tr><td> </td><td> </td><td>8 </td><td>(G–G) </td><td>1.02 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*The statistically significant nucleotide pair in the parentheses appears at positions nt1 and nt2, respectively. We used the following rule of numbering nucleotide positions; position of the edited C was defined as 0, the nucleotides located on the 3′ side of the edited C had positive numbers and the nucleotides located on the 5′ side had negative numbers based on the distance in nucleotides from the C.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5179994" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN016TB1" class="table-wrap-title" data-id="DSN016TB1"><div class="label -label">Table 1</div><div class="caption"><p>Log<sub>2</sub> doublet propensities &gt;0.9 in each group</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Group </th><th>nt1* </th><th>nt2* </th><th>Pair* </th><th colspan="2">Log<sub>2</sub> doublet propensity </th></tr></thead><tbody><tr><td>G-1 </td><td>−30 </td><td>9 </td><td>(C–G) </td><td>0.94 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−25 </td><td>−24 </td><td>(G–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td> </td><td>−22 </td><td>(G–G) </td><td>1.02 </td><td> </td></tr><tr><td> </td><td>−22 </td><td>−21 </td><td>(C–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−10 </td><td>5 </td><td>(G–C) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>6 </td><td>(C–C) </td><td>0.90 </td><td> </td></tr><tr><td> </td><td>−3 </td><td>−2 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td>G-2 </td><td>−25 </td><td>−10 </td><td>(C–C) </td><td>0.94 </td><td> </td></tr><tr><td>G-3 </td><td>−16 </td><td>5 </td><td>(G–G) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−10 </td><td>−5 </td><td>(C–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>6 </td><td>8 </td><td>(G–G) </td><td>0.93 </td><td> </td></tr><tr><td>G-4 </td><td>−28 </td><td>−25 </td><td>(C–G) </td><td>1.07 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−27 </td><td>4 </td><td>(C–A) </td><td>0.99 </td><td> </td></tr><tr><td> </td><td> </td><td>5 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−26 </td><td>10 </td><td>(C–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−25 </td><td>−9 </td><td>(G–A) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−5 </td><td>(C–C) </td><td>0.99 </td><td> </td></tr><tr><td> </td><td>−19 </td><td>4 </td><td>(A–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td> </td><td>5 </td><td>(A–A) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>5 </td><td>(A–A) </td><td>1.05 </td><td> </td></tr><tr><td> </td><td>−9 </td><td>2 </td><td>(A–C) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−8 </td><td>5 </td><td>(G–G) </td><td>0.94 </td><td> </td></tr><tr><td> </td><td>−6 </td><td>9 </td><td>(C–C) </td><td>0.94 </td><td> </td></tr><tr><td> </td><td>−2 </td><td>5 </td><td>(A–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>1 </td><td>10 </td><td>(A–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>4 </td><td>5 </td><td>(C–A) </td><td>0.94 </td><td> </td></tr><tr><td>G-5 </td><td>−27 </td><td>5 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−26 </td><td>−16 </td><td>(C–C) </td><td>0.98 </td><td> </td></tr><tr><td> </td><td>−24 </td><td>−16 </td><td>(G–G) </td><td>0.98 </td><td> </td></tr><tr><td> </td><td>−22 </td><td>2 </td><td>(A–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−20 </td><td>(C–C) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td>−16 </td><td>−14 </td><td>(C–C) </td><td>1.05 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>−8 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−13 </td><td>−9 </td><td>(G–A) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>3 </td><td>8 </td><td>(A–C) </td><td>1.02 </td><td> </td></tr><tr><td>G-6 </td><td>−29 </td><td>−13 </td><td>(G–G) </td><td>0.90 </td><td> </td></tr><tr><td> </td><td>−23 </td><td>−15 </td><td>(C–G) </td><td>0.94 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−22 </td><td>−6 </td><td>(C–G) </td><td>0.93 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−20 </td><td>−14 </td><td>(C–C) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td> </td><td>−4 </td><td>(C–T) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−18 </td><td>6 </td><td>(C–C) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>−13 </td><td>3 </td><td>(C–C) </td><td>1.21 </td><td> </td></tr><tr><td> </td><td>−8 </td><td>−3 </td><td>(G–C) </td><td>0.98 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−3 </td><td>4 </td><td>(C–A) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td> </td><td>7 </td><td>(C–A) </td><td>1.06 </td><td> </td></tr><tr><td>G-7 </td><td>−27 </td><td>−15 </td><td>(G–G) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td> </td><td>−9 </td><td>(C–G) </td><td>0.95 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−18 </td><td>9 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td>G-8 </td><td>−27 </td><td>−6 </td><td>(A–C) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>−26 </td><td>−15 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−24 </td><td>−17 </td><td>(G–C) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>3 </td><td>(G–C) </td><td>1.00 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−23 </td><td>−8 </td><td>(G–G) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−2 </td><td>(G–C) </td><td>0.91 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−17 </td><td>−16 </td><td>(G–C) </td><td>0.93 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>3 </td><td>(G–G) </td><td>1.06 </td><td> </td></tr><tr><td> </td><td> </td><td>8 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>2 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−14 </td><td>4 </td><td>(G–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>−13 </td><td>2 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td> </td><td>4 </td><td>(G–G) </td><td>1.03 </td><td> </td></tr><tr><td> </td><td>−6 </td><td>8 </td><td>(C–G) </td><td>1.06 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−2 </td><td>8 </td><td>(T–G) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>2 </td><td>5 </td><td>(C–A) </td><td>0.93 </td><td> </td></tr><tr><td> </td><td> </td><td>8 </td><td>(G–G) </td><td>1.02 </td><td> </td></tr></tbody></table></div><div content-id="DSN016TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Group </th><th>nt1* </th><th>nt2* </th><th>Pair* </th><th colspan="2">Log<sub>2</sub> doublet propensity </th></tr></thead><tbody><tr><td>G-1 </td><td>−30 </td><td>9 </td><td>(C–G) </td><td>0.94 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−25 </td><td>−24 </td><td>(G–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td> </td><td>−22 </td><td>(G–G) </td><td>1.02 </td><td> </td></tr><tr><td> </td><td>−22 </td><td>−21 </td><td>(C–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−10 </td><td>5 </td><td>(G–C) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>6 </td><td>(C–C) </td><td>0.90 </td><td> </td></tr><tr><td> </td><td>−3 </td><td>−2 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td>G-2 </td><td>−25 </td><td>−10 </td><td>(C–C) </td><td>0.94 </td><td> </td></tr><tr><td>G-3 </td><td>−16 </td><td>5 </td><td>(G–G) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−10 </td><td>−5 </td><td>(C–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>6 </td><td>8 </td><td>(G–G) </td><td>0.93 </td><td> </td></tr><tr><td>G-4 </td><td>−28 </td><td>−25 </td><td>(C–G) </td><td>1.07 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−27 </td><td>4 </td><td>(C–A) </td><td>0.99 </td><td> </td></tr><tr><td> </td><td> </td><td>5 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−26 </td><td>10 </td><td>(C–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−25 </td><td>−9 </td><td>(G–A) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−5 </td><td>(C–C) </td><td>0.99 </td><td> </td></tr><tr><td> </td><td>−19 </td><td>4 </td><td>(A–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td> </td><td>5 </td><td>(A–A) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>5 </td><td>(A–A) </td><td>1.05 </td><td> </td></tr><tr><td> </td><td>−9 </td><td>2 </td><td>(A–C) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−8 </td><td>5 </td><td>(G–G) </td><td>0.94 </td><td> </td></tr><tr><td> </td><td>−6 </td><td>9 </td><td>(C–C) </td><td>0.94 </td><td> </td></tr><tr><td> </td><td>−2 </td><td>5 </td><td>(A–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>1 </td><td>10 </td><td>(A–C) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>4 </td><td>5 </td><td>(C–A) </td><td>0.94 </td><td> </td></tr><tr><td>G-5 </td><td>−27 </td><td>5 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−26 </td><td>−16 </td><td>(C–C) </td><td>0.98 </td><td> </td></tr><tr><td> </td><td>−24 </td><td>−16 </td><td>(G–G) </td><td>0.98 </td><td> </td></tr><tr><td> </td><td>−22 </td><td>2 </td><td>(A–C) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−20 </td><td>(C–C) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td>−16 </td><td>−14 </td><td>(C–C) </td><td>1.05 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>−8 </td><td>(C–G) </td><td>0.90 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−13 </td><td>−9 </td><td>(G–A) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>3 </td><td>8 </td><td>(A–C) </td><td>1.02 </td><td> </td></tr><tr><td>G-6 </td><td>−29 </td><td>−13 </td><td>(G–G) </td><td>0.90 </td><td> </td></tr><tr><td> </td><td>−23 </td><td>−15 </td><td>(C–G) </td><td>0.94 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−22 </td><td>−6 </td><td>(C–G) </td><td>0.93 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−20 </td><td>−14 </td><td>(C–C) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td> </td><td>−4 </td><td>(C–T) </td><td>0.95 </td><td> </td></tr><tr><td> </td><td>−18 </td><td>6 </td><td>(C–C) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>−13 </td><td>3 </td><td>(C–C) </td><td>1.21 </td><td> </td></tr><tr><td> </td><td>−8 </td><td>−3 </td><td>(G–C) </td><td>0.98 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−3 </td><td>4 </td><td>(C–A) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td> </td><td>7 </td><td>(C–A) </td><td>1.06 </td><td> </td></tr><tr><td>G-7 </td><td>−27 </td><td>−15 </td><td>(G–G) </td><td>0.96 </td><td> </td></tr><tr><td> </td><td> </td><td>−9 </td><td>(C–G) </td><td>0.95 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−18 </td><td>9 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td>G-8 </td><td>−27 </td><td>−6 </td><td>(A–C) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>−26 </td><td>−15 </td><td>(C–G) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−24 </td><td>−17 </td><td>(G–C) </td><td>0.92 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>3 </td><td>(G–C) </td><td>1.00 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−23 </td><td>−8 </td><td>(G–G) </td><td>1.01 </td><td> </td></tr><tr><td> </td><td>−21 </td><td>−2 </td><td>(G–C) </td><td>0.91 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−17 </td><td>−16 </td><td>(G–C) </td><td>0.93 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td> </td><td>3 </td><td>(G–G) </td><td>1.06 </td><td> </td></tr><tr><td> </td><td> </td><td>8 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−15 </td><td>2 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td>−14 </td><td>4 </td><td>(G–G) </td><td>0.92 </td><td> </td></tr><tr><td> </td><td>−13 </td><td>2 </td><td>(G–G) </td><td>0.91 </td><td> </td></tr><tr><td> </td><td> </td><td>4 </td><td>(G–G) </td><td>1.03 </td><td> </td></tr><tr><td> </td><td>−6 </td><td>8 </td><td>(C–G) </td><td>1.06 </td><td>Watson–Crick pair </td></tr><tr><td> </td><td>−2 </td><td>8 </td><td>(T–G) </td><td>0.97 </td><td> </td></tr><tr><td> </td><td>2 </td><td>5 </td><td>(C–A) </td><td>0.93 </td><td> </td></tr><tr><td> </td><td> </td><td>8 </td><td>(G–G) </td><td>1.02 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*The statistically significant nucleotide pair in the parentheses appears at positions nt1 and nt2, respectively. We used the following rule of numbering nucleotide positions; position of the edited C was defined as 0, the nucleotides located on the 3′ side of the edited C had positive numbers and the nucleotides located on the 5′ side had negative numbers based on the distance in nucleotides from the C.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5179994" target="_blank">View Large</a></div></div><p>One focus in the investigation of the editing mechanism has been the role of RNA secondary structures in RNA editing site recognition. Yu et al.<sup><a reveal-id="DSN016C44" data-open="DSN016C44" class="link link-ref link-reveal xref-bibr">44</a></sup> could not identify any RNA secondary structure motifs in the sequences surrounding mitochondrial RNA editing sites. Mulligan et al.<sup><a reveal-id="DSN016C11" data-open="DSN016C11" class="link link-ref link-reveal xref-bibr">11</a></sup> confirmed these findings by demonstrating that the RNA editing sites in the mitochondria <em>rps12</em> transcript did not have conserved RNA secondary structures. In contrast, Cummings and Myers<sup><a reveal-id="DSN016C33" data-open="DSN016C33" class="link link-ref link-reveal xref-bibr">33</a></sup> analyzed RNA editing sites in mitochondria from three different species, discussing the possibility of secondary structure involvement in editing site recognition based on their free-energy calculations for RNA folding. Our study suggests that standard Watson–Crick pairs are not involved in the recognition processes.</p>                            <h3 scrollto-destination=5179996 id="5179996" class="section-title" data-legacy-id=s3d>3.4. Application of the propensities to prediction of RNA editing site</h3>
<p>Although the prediction of RNA editing sites in mitochondrial transcripts has been attempted by several groups,<sup><a reveal-id="DSN016C33  DSN016C36" data-open="DSN016C33  DSN016C36" class="link link-ref link-reveal xref-bibr">33–36</a></sup> prediction of RNA editing sites in chloroplast transcripts has not been tried, as there are significantly fewer known RNA editing sites in chloroplast transcripts.</p><p>One of the prediction methods that is widely used to identify RNA editing sites in chloroplast transcripts requires initial translation of the RNA into the amino-acid sequence followed by alignment of the homologous sequences. As RNA editing improves the amino-acid sequence identity of the edited sequence for homologous sequences, if a well-conserved position in the amino-acid sequence from the chloroplast gene is not conserved and C-to-U conversion within the codon can amend this difference, then the nucleotide is a candidate for an RNA editing site. This rule of thumb is practical, but cannot be used to detect silent RNA editing events that convert nucleotides at the third position in a codon and do not alter the encoded amino acid. In addition, a subset of editing events result in diversity of amino-acid sequences; this method also fails to identify this type of event.<sup><a reveal-id="DSN016C45" data-open="DSN016C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Furthermore, the method does not provide any understanding of the mechanism of RNA editing. Here, we apply the singlet and doublet propensities to neighbor sequences of RNA editing sites, as explained in Equation <a href="#DSN016M8" class="sectionLink xref-sec">(8)</a>, to assess if this score can be a good predictor for the editing sites.</p><p>We applied singlet and doublet propensities of group G-2 to the whole cDNAs (open reading frames) for the <em>rps4–psbE</em> and <em>clpP–rpoA</em> sequences (Fig. <a reveal-id="DSN016F1" data-open="DSN016F1" class="link link-reveal link-table xref-fig">1</a>). Coding sequences were read with a moving window 41-nucleotide in length. When the 31st position was a C, we calculated the score based on Equation <a href="#DSN016M8" class="sectionLink xref-sec">(8)</a> and plotted in the histogram (Fig. <a reveal-id="DSN016F5" data-open="DSN016F5" class="link link-reveal link-table xref-fig">5</a>). Red regions in the figure indicate a count of edited sequences in G-2, whereas the cyan regions indicate a count of non-edited sequences. As we deduced the propensities from the edited sequences in G-2, it is trivial that the red regions were clustered in high-score regions. It is not trivial, however, that non-edited sequences were rarely found in high-score regions and that a peak was found in low-score regions. Similar distribution patterns were observed for the other groups (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). On the basis of this prediction using propensities of G-2, sequences with scores &gt;130 were certainly neighbor sequences of RNA edited sites and sequences with score &gt;100 were neighbor sequences with a likelihood of ∼60%. If the threshold for editing site prediction was set at score 100, then the predicted sites reflected truly edited sites with an accuracy of 60% (true positive), with 40% that were not edited (false-positive) and 15% of true RNA editing sites were missed (false-negative). Similar conclusions were found for the other groups (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                        <a id="5180001" scrollto-destination="5180001"></a>
<div id="DSN016F5" data-id="DSN016F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01605.jpeg?Expires=2147483647&Signature=qFLh~lroXA5FvVpDzV56jlozEn~79EofLDVvZKi8sJA42oAdKGd8~JlxQbTjHDdKQESYq1TwdiLZG9GoPmo1m7Va8GY82WPnKzdphQlm3j-IiQDXE5CLA1vH7tVMB2TJbVEddNrstXmdvBCL61B7fe~Msfhmjdecgr40zOI-IQzWUf2w49zSd7mZEpOQmY7kxTtyO4gaMPH7L~5ou6rIuC1QN1p3OPV4YGOvoQDVHZYdBHPl5L-4zlH0RPvFuB2oCQaST0rUGZXPtr~ZdcBkgzpNXrrlMV~VNWBEu3zO6D7Gj7aop6b29qh9G7fJEGt8Jk6iTcnMsAhfudrj52lgAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction score histogram for group G-2. The horizontal axis is the score in a bin of scale 10, and the vertical axis is the count of the prediction score in the bin. A red bar indicates the score for the edited C, and a bar in cyan indicates the score for a non-edited C. The prediction was performed for all transcripts (coding regions) between clpP and rpoA and between rps4 and psbE in the T. lepidozioides chloroplast genome. A bar for scores &gt;70 has two numbers separated by a slash on the top. The number on the left is a count of the edited C, and the number on the right is the whole count in the bar." path-from-xml="dsn01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5180001/dsn01605.jpeg" path-from-xml="dsn01605.jpeg" target="_blank">View large</a><a section="5180001" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01605.jpeg?Expires=2147483647&Signature=Zf6jKF5KsA--JYkcA8g8deolb-K2UVs6f4npzhUdzHZryJtD6VgRjeyaIJW5Q6V625yLinnJoAJrCm0CmZSgOqzSbeIvlwVxEnfY9jtKBzUrpnGS8UbHgAbXlx3BOoCjFQtywklAfvUIesD-50kcP79n4JBrt~hP8340C9eK2PpawuoYIqwPHLPxGrkIgg8ptXl0av5E~04eidSD9K2drQq1UVrh2ZpxFTHHmZv63i7gmGVJFPV0muBnzSnOH7eInM20yOGDOgIQYUOYAKsCG1oMBqdpn7x7rSh~~WNKCfOwjVPU8iFxgsRcXi1CuZyzC2SFqVeh2zI7pyevhFdItA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5180001&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction score histogram for group G-2. The horizontal axis is the score in a bin of scale 10, and the vertical axis is the count of the prediction score in the bin. A red bar indicates the score for the edited C, and a bar in cyan indicates the score for a non-edited C. The prediction was performed for all transcripts (coding regions) between <em>clpP</em> and <em>rpoA</em> and between <em>rps4</em> and <em>psbE</em> in the <em>T</em>. <em>lepidozioides</em> chloroplast genome. A bar for scores &gt;70 has two numbers separated by a slash on the top. The number on the left is a count of the edited C, and the number on the right is the whole count in the bar.</p></div></div><div content-id="DSN016F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/m_dsn01605.jpeg?Expires=2147483647&Signature=qFLh~lroXA5FvVpDzV56jlozEn~79EofLDVvZKi8sJA42oAdKGd8~JlxQbTjHDdKQESYq1TwdiLZG9GoPmo1m7Va8GY82WPnKzdphQlm3j-IiQDXE5CLA1vH7tVMB2TJbVEddNrstXmdvBCL61B7fe~Msfhmjdecgr40zOI-IQzWUf2w49zSd7mZEpOQmY7kxTtyO4gaMPH7L~5ou6rIuC1QN1p3OPV4YGOvoQDVHZYdBHPl5L-4zlH0RPvFuB2oCQaST0rUGZXPtr~ZdcBkgzpNXrrlMV~VNWBEu3zO6D7Gj7aop6b29qh9G7fJEGt8Jk6iTcnMsAhfudrj52lgAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction score histogram for group G-2. The horizontal axis is the score in a bin of scale 10, and the vertical axis is the count of the prediction score in the bin. A red bar indicates the score for the edited C, and a bar in cyan indicates the score for a non-edited C. The prediction was performed for all transcripts (coding regions) between clpP and rpoA and between rps4 and psbE in the T. lepidozioides chloroplast genome. A bar for scores &gt;70 has two numbers separated by a slash on the top. The number on the left is a count of the edited C, and the number on the right is the whole count in the bar." path-from-xml="dsn01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5180001/dsn01605.jpeg" path-from-xml="dsn01605.jpeg" target="_blank">View large</a><a section="5180001" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn01605.jpeg?Expires=2147483647&Signature=Zf6jKF5KsA--JYkcA8g8deolb-K2UVs6f4npzhUdzHZryJtD6VgRjeyaIJW5Q6V625yLinnJoAJrCm0CmZSgOqzSbeIvlwVxEnfY9jtKBzUrpnGS8UbHgAbXlx3BOoCjFQtywklAfvUIesD-50kcP79n4JBrt~hP8340C9eK2PpawuoYIqwPHLPxGrkIgg8ptXl0av5E~04eidSD9K2drQq1UVrh2ZpxFTHHmZv63i7gmGVJFPV0muBnzSnOH7eInM20yOGDOgIQYUOYAKsCG1oMBqdpn7x7rSh~~WNKCfOwjVPU8iFxgsRcXi1CuZyzC2SFqVeh2zI7pyevhFdItA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5180001&ar=359388&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction score histogram for group G-2. The horizontal axis is the score in a bin of scale 10, and the vertical axis is the count of the prediction score in the bin. A red bar indicates the score for the edited C, and a bar in cyan indicates the score for a non-edited C. The prediction was performed for all transcripts (coding regions) between <em>clpP</em> and <em>rpoA</em> and between <em>rps4</em> and <em>psbE</em> in the <em>T</em>. <em>lepidozioides</em> chloroplast genome. A bar for scores &gt;70 has two numbers separated by a slash on the top. The number on the left is a count of the edited C, and the number on the right is the whole count in the bar.</p></div></div><p>The current prediction method measures the similarity of given nucleotide sequences with the propensities of the set of known sequences. In this way, the method is similar to homology search methods. Most conventional homology search methods, however, do not account for the similarity of residue pairs; this method attempted to address this issue by introducing doublet propensity into the score. When the prediction score was dissected into terms derived from the singlet and doublet propensities, singlet propensities contributed 4–6%, whereas doublet propensities provided 94–96% of the score. The higher proportion contributed by the doublet propensities in the score makes this method significantly different from conventional homology search methods. In a 41-nucleotide sequence, there are 40 singlet propensities (the C at 31st position is not included in the score) and 780 (=40 × 39/2) doublet propensities, which is part of the reason why doublet propensities contribute to the score greater than singlet propensities.</p>                            <h3 scrollto-destination=5180004 id="5180004" class="section-title" data-legacy-id=s3e>3.5. Application of the prediction to the newly sequenced portion of the T. lepidozioides chloroplast genome</h3>
<p>The most rigorous test of the prediction method is a blind test, in which we predict a result and verify the prediction experimentally. After sequencing the region between <em>petL</em> and <em>clpP</em>, we (i) predicted RNA editing sites in the transcript from that region and (ii) experimentally detected RNA editing sites after the prediction. We sequenced cDNA transcripts from <em>petL</em> to <em>clpP</em> (Fig. <a reveal-id="DSN016F1" data-open="DSN016F1" class="link link-reveal link-table xref-fig">1</a>), identifying 42 C-to-U conversions in this region (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p><p>We expected from the assessment of the prediction method that a score &gt;100 has an accuracy of ∼60%. Seven of the 12 predicted sites (∼58%) were actually edited. Higher scores had a greater accuracy of prediction (Table <a reveal-id="DSN016TB2" data-open="DSN016TB2" class="link link-reveal link-table xref-fig">2</a>). None of those predicted sequences were identical to the sequences obtained with the singlet and doublet propensities, which makes this result quite important. This blind test clearly demonstrated that a high score using our prediction method provides a high likelihood that the sequence contains an RNA editing site at the 31st position. In Table <a reveal-id="DSN016TB2" data-open="DSN016TB2" class="link link-reveal link-table xref-fig">2</a>, there were two silent RNA editing sites that did not change the encoded amino-acid residues. Those two predictions can only be achieved using our method. The success of the prediction method above confirms that the characteristics described above (Figs <a reveal-id="DSN016F3" data-open="DSN016F3" class="link link-reveal link-table xref-fig">3</a>A and <a reveal-id="DSN016F4" data-open="DSN016F4" class="link link-reveal link-table xref-fig">4</a>) reflect tangible nucleotide patterns recognized by <em>trans</em>-factors.</p>                        <a id="5180007" scrollto-destination="5180007"></a>
<div content-id="DSN016TB2" class="table-modal"><div class="table-wrap"><div id="DSN016TB2" class="table-wrap-title" data-id="DSN016TB2"><div class="label -label">Table 2</div><div class="caption"><p>Prediction and verification of RNA editing sites (score &gt; 101) in newly sequenced <em>T</em>. <em>lepidozioides</em> genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Predicted edited site with −30 to +10 nucleotides </th><th>Gene </th><th>Position* </th><th>Score </th><th>Codon* </th><th>Unedited → edited </th></tr></thead><tbody><tr><td><span class="monospace">TTGTTGATTTCGTAGCTACAGAAACTGCTT<strong>C</strong>AAATTCTCCA</span> </td><td><em>clpP</em> </td><td>−6243 </td><td>120.00 </td><td>199 </td><td>S(tCa) → L(tTa) </td></tr><tr><td><span class="monospace">AACCTGTTATTGTCACTAGGATTCCTTCCA<strong>C</strong>GCTTACTACA</span> </td><td><em>petL</em> </td><td>1897 </td><td>112.01 </td><td>1 </td><td>T(aCg) → M(aTg) </td></tr><tr><td><span class="monospace">TATTTGCTGCTTCAATTTCAGCTTTAGTTT<strong>C</strong>ATTTATTGGC</span> </td><td><em>petL</em> </td><td>1957 </td><td>109.97 </td><td>21 </td><td>S(tCa) → L(tTa) </td></tr><tr><td><span class="monospace">AATGATCCATCAACCTGCTAGTTCCTACTA<strong>C</strong>GATGGACAAG</span> </td><td><em>clpP</em> </td><td>−6440 </td><td>108.17 </td><td> </td><td>false positive </td></tr><tr><td><span class="monospace">AAGTTTATGGCATTGTTGATTTCGTAGCTA<strong>C</strong>AGAAACTGCT</span> </td><td><em>clpP</em> </td><td>−6255 </td><td>107.40 </td><td>195 </td><td>T(aCa) → I(aTa) </td></tr><tr><td><span class="monospace">CTCCCGGTGGAGCAGTATTAGCAGGAATAT<strong>C</strong>TGTTTATGAT</span> </td><td><em>clpP</em> </td><td>−7143 </td><td>106.91 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">GGATTTCAAGGAGCTCATTCAAAACTATTT<strong>C</strong>GAACTGCTAA</span> </td><td><em>rpl20</em> </td><td>−4681 </td><td>106.11 </td><td>33 </td><td>R(Cga) → *(Tga) </td></tr><tr><td><span class="monospace">CAATGCAAGATGTAAAAACATATCTTTCTA<strong>C</strong>AGCACCTGTG</span> </td><td><em>psaJ</em> </td><td>3317 </td><td>106.02 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">AAAATGGTGAAAAAATAAAATTAGGTGTTT<strong>C</strong>TAGATATACT</span> </td><td><em>rpl33</em> </td><td>3834 </td><td>105.03 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">ATTTCGTAGCTACAGAAACTGCTTCAAATT<strong>C</strong>TCCAACGAAT</span> </td><td><em>clpP</em> </td><td>−6237 </td><td>103.90 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">TTTAGGCTCCCCGGAGAAGAAGATGCTGTT<strong>C</strong>GGATCGACGT</span> </td><td><em>clpP</em> </td><td>−8036 </td><td>103.15 </td><td>27 </td><td>L(ctC) → L(ctT) </td></tr><tr><td><span class="monospace">AGAAGCAAAAGTTTATGGCATTGTTGATTT<strong>C</strong>GTAGCTACAG</span> </td><td><em>clpP</em> </td><td>−6263 </td><td>101.19 </td><td>192 </td><td>F(ttC) → F(ttT) </td></tr></tbody></table></div><div content-id="DSN016TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Predicted edited site with −30 to +10 nucleotides </th><th>Gene </th><th>Position* </th><th>Score </th><th>Codon* </th><th>Unedited → edited </th></tr></thead><tbody><tr><td><span class="monospace">TTGTTGATTTCGTAGCTACAGAAACTGCTT<strong>C</strong>AAATTCTCCA</span> </td><td><em>clpP</em> </td><td>−6243 </td><td>120.00 </td><td>199 </td><td>S(tCa) → L(tTa) </td></tr><tr><td><span class="monospace">AACCTGTTATTGTCACTAGGATTCCTTCCA<strong>C</strong>GCTTACTACA</span> </td><td><em>petL</em> </td><td>1897 </td><td>112.01 </td><td>1 </td><td>T(aCg) → M(aTg) </td></tr><tr><td><span class="monospace">TATTTGCTGCTTCAATTTCAGCTTTAGTTT<strong>C</strong>ATTTATTGGC</span> </td><td><em>petL</em> </td><td>1957 </td><td>109.97 </td><td>21 </td><td>S(tCa) → L(tTa) </td></tr><tr><td><span class="monospace">AATGATCCATCAACCTGCTAGTTCCTACTA<strong>C</strong>GATGGACAAG</span> </td><td><em>clpP</em> </td><td>−6440 </td><td>108.17 </td><td> </td><td>false positive </td></tr><tr><td><span class="monospace">AAGTTTATGGCATTGTTGATTTCGTAGCTA<strong>C</strong>AGAAACTGCT</span> </td><td><em>clpP</em> </td><td>−6255 </td><td>107.40 </td><td>195 </td><td>T(aCa) → I(aTa) </td></tr><tr><td><span class="monospace">CTCCCGGTGGAGCAGTATTAGCAGGAATAT<strong>C</strong>TGTTTATGAT</span> </td><td><em>clpP</em> </td><td>−7143 </td><td>106.91 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">GGATTTCAAGGAGCTCATTCAAAACTATTT<strong>C</strong>GAACTGCTAA</span> </td><td><em>rpl20</em> </td><td>−4681 </td><td>106.11 </td><td>33 </td><td>R(Cga) → *(Tga) </td></tr><tr><td><span class="monospace">CAATGCAAGATGTAAAAACATATCTTTCTA<strong>C</strong>AGCACCTGTG</span> </td><td><em>psaJ</em> </td><td>3317 </td><td>106.02 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">AAAATGGTGAAAAAATAAAATTAGGTGTTT<strong>C</strong>TAGATATACT</span> </td><td><em>rpl33</em> </td><td>3834 </td><td>105.03 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">ATTTCGTAGCTACAGAAACTGCTTCAAATT<strong>C</strong>TCCAACGAAT</span> </td><td><em>clpP</em> </td><td>−6237 </td><td>103.90 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">TTTAGGCTCCCCGGAGAAGAAGATGCTGTT<strong>C</strong>GGATCGACGT</span> </td><td><em>clpP</em> </td><td>−8036 </td><td>103.15 </td><td>27 </td><td>L(ctC) → L(ctT) </td></tr><tr><td><span class="monospace">AGAAGCAAAAGTTTATGGCATTGTTGATTT<strong>C</strong>GTAGCTACAG</span> </td><td><em>clpP</em> </td><td>−6263 </td><td>101.19 </td><td>192 </td><td>F(ttC) → F(ttT) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Position is the nucleotide number of the predicted edited site (bold) in the sequenced region, whereas the minus number indicates that the editing site is predicted to be on the opposite strand of the deposited sequence. The codon number and conversion in the codon were shown for seven sites.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5180007" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN016TB2" class="table-wrap-title" data-id="DSN016TB2"><div class="label -label">Table 2</div><div class="caption"><p>Prediction and verification of RNA editing sites (score &gt; 101) in newly sequenced <em>T</em>. <em>lepidozioides</em> genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Predicted edited site with −30 to +10 nucleotides </th><th>Gene </th><th>Position* </th><th>Score </th><th>Codon* </th><th>Unedited → edited </th></tr></thead><tbody><tr><td><span class="monospace">TTGTTGATTTCGTAGCTACAGAAACTGCTT<strong>C</strong>AAATTCTCCA</span> </td><td><em>clpP</em> </td><td>−6243 </td><td>120.00 </td><td>199 </td><td>S(tCa) → L(tTa) </td></tr><tr><td><span class="monospace">AACCTGTTATTGTCACTAGGATTCCTTCCA<strong>C</strong>GCTTACTACA</span> </td><td><em>petL</em> </td><td>1897 </td><td>112.01 </td><td>1 </td><td>T(aCg) → M(aTg) </td></tr><tr><td><span class="monospace">TATTTGCTGCTTCAATTTCAGCTTTAGTTT<strong>C</strong>ATTTATTGGC</span> </td><td><em>petL</em> </td><td>1957 </td><td>109.97 </td><td>21 </td><td>S(tCa) → L(tTa) </td></tr><tr><td><span class="monospace">AATGATCCATCAACCTGCTAGTTCCTACTA<strong>C</strong>GATGGACAAG</span> </td><td><em>clpP</em> </td><td>−6440 </td><td>108.17 </td><td> </td><td>false positive </td></tr><tr><td><span class="monospace">AAGTTTATGGCATTGTTGATTTCGTAGCTA<strong>C</strong>AGAAACTGCT</span> </td><td><em>clpP</em> </td><td>−6255 </td><td>107.40 </td><td>195 </td><td>T(aCa) → I(aTa) </td></tr><tr><td><span class="monospace">CTCCCGGTGGAGCAGTATTAGCAGGAATAT<strong>C</strong>TGTTTATGAT</span> </td><td><em>clpP</em> </td><td>−7143 </td><td>106.91 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">GGATTTCAAGGAGCTCATTCAAAACTATTT<strong>C</strong>GAACTGCTAA</span> </td><td><em>rpl20</em> </td><td>−4681 </td><td>106.11 </td><td>33 </td><td>R(Cga) → *(Tga) </td></tr><tr><td><span class="monospace">CAATGCAAGATGTAAAAACATATCTTTCTA<strong>C</strong>AGCACCTGTG</span> </td><td><em>psaJ</em> </td><td>3317 </td><td>106.02 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">AAAATGGTGAAAAAATAAAATTAGGTGTTT<strong>C</strong>TAGATATACT</span> </td><td><em>rpl33</em> </td><td>3834 </td><td>105.03 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">ATTTCGTAGCTACAGAAACTGCTTCAAATT<strong>C</strong>TCCAACGAAT</span> </td><td><em>clpP</em> </td><td>−6237 </td><td>103.90 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">TTTAGGCTCCCCGGAGAAGAAGATGCTGTT<strong>C</strong>GGATCGACGT</span> </td><td><em>clpP</em> </td><td>−8036 </td><td>103.15 </td><td>27 </td><td>L(ctC) → L(ctT) </td></tr><tr><td><span class="monospace">AGAAGCAAAAGTTTATGGCATTGTTGATTT<strong>C</strong>GTAGCTACAG</span> </td><td><em>clpP</em> </td><td>−6263 </td><td>101.19 </td><td>192 </td><td>F(ttC) → F(ttT) </td></tr></tbody></table></div><div content-id="DSN016TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Predicted edited site with −30 to +10 nucleotides </th><th>Gene </th><th>Position* </th><th>Score </th><th>Codon* </th><th>Unedited → edited </th></tr></thead><tbody><tr><td><span class="monospace">TTGTTGATTTCGTAGCTACAGAAACTGCTT<strong>C</strong>AAATTCTCCA</span> </td><td><em>clpP</em> </td><td>−6243 </td><td>120.00 </td><td>199 </td><td>S(tCa) → L(tTa) </td></tr><tr><td><span class="monospace">AACCTGTTATTGTCACTAGGATTCCTTCCA<strong>C</strong>GCTTACTACA</span> </td><td><em>petL</em> </td><td>1897 </td><td>112.01 </td><td>1 </td><td>T(aCg) → M(aTg) </td></tr><tr><td><span class="monospace">TATTTGCTGCTTCAATTTCAGCTTTAGTTT<strong>C</strong>ATTTATTGGC</span> </td><td><em>petL</em> </td><td>1957 </td><td>109.97 </td><td>21 </td><td>S(tCa) → L(tTa) </td></tr><tr><td><span class="monospace">AATGATCCATCAACCTGCTAGTTCCTACTA<strong>C</strong>GATGGACAAG</span> </td><td><em>clpP</em> </td><td>−6440 </td><td>108.17 </td><td> </td><td>false positive </td></tr><tr><td><span class="monospace">AAGTTTATGGCATTGTTGATTTCGTAGCTA<strong>C</strong>AGAAACTGCT</span> </td><td><em>clpP</em> </td><td>−6255 </td><td>107.40 </td><td>195 </td><td>T(aCa) → I(aTa) </td></tr><tr><td><span class="monospace">CTCCCGGTGGAGCAGTATTAGCAGGAATAT<strong>C</strong>TGTTTATGAT</span> </td><td><em>clpP</em> </td><td>−7143 </td><td>106.91 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">GGATTTCAAGGAGCTCATTCAAAACTATTT<strong>C</strong>GAACTGCTAA</span> </td><td><em>rpl20</em> </td><td>−4681 </td><td>106.11 </td><td>33 </td><td>R(Cga) → *(Tga) </td></tr><tr><td><span class="monospace">CAATGCAAGATGTAAAAACATATCTTTCTA<strong>C</strong>AGCACCTGTG</span> </td><td><em>psaJ</em> </td><td>3317 </td><td>106.02 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">AAAATGGTGAAAAAATAAAATTAGGTGTTT<strong>C</strong>TAGATATACT</span> </td><td><em>rpl33</em> </td><td>3834 </td><td>105.03 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">ATTTCGTAGCTACAGAAACTGCTTCAAATT<strong>C</strong>TCCAACGAAT</span> </td><td><em>clpP</em> </td><td>−6237 </td><td>103.90 </td><td> </td><td>False positive </td></tr><tr><td><span class="monospace">TTTAGGCTCCCCGGAGAAGAAGATGCTGTT<strong>C</strong>GGATCGACGT</span> </td><td><em>clpP</em> </td><td>−8036 </td><td>103.15 </td><td>27 </td><td>L(ctC) → L(ctT) </td></tr><tr><td><span class="monospace">AGAAGCAAAAGTTTATGGCATTGTTGATTT<strong>C</strong>GTAGCTACAG</span> </td><td><em>clpP</em> </td><td>−6263 </td><td>101.19 </td><td>192 </td><td>F(ttC) → F(ttT) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Position is the nucleotide number of the predicted edited site (bold) in the sequenced region, whereas the minus number indicates that the editing site is predicted to be on the opposite strand of the deposited sequence. The codon number and conversion in the codon were shown for seven sites.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5180007" target="_blank">View Large</a></div></div><p>Experimental identification of RNA editing sites resulted in the identification of additional sites not listed in Table <a reveal-id="DSN016TB2" data-open="DSN016TB2" class="link link-reveal link-table xref-fig">2</a> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Our prediction method gave low scores to those sites. We assume that the RNA editing sites with low scores in our prediction are recognized by <em>trans</em>-factors independent from the ones that recognize G-1 to G-8 sequences (Fig. <a reveal-id="DSN016F2" data-open="DSN016F2" class="link link-reveal link-table xref-fig">2</a>). The current prediction method is based on our classification of the RNA editing site neighbor sequences that may reflect similarities in the <em>trans</em>-factors that recognize those sequences. Therefore, an RNA editing site recognized by a different <em>trans</em>-factor cannot be predicted by this method. This may mean, however, that RNA editing sites recognized by homologous <em>trans</em>-factors, even in other species, can be predicted by the current propensities.</p>                            <h3 scrollto-destination=5180009 id="5180009" class="section-title" data-legacy-id=s3f>3.6. Application of the prediction to the Araidopsis thaliana chloroplast genome</h3>
<p>To test if our method could identify RNA editing sites in other species, we applied our prediction method to the <em>A</em>. <em>thaliana</em> chloroplast genome. RNA editing sites in transcripts of <em>A</em>. <em>thaliana</em> chloroplasts have been extensively analyzed by Tillich et al.,<sup><a reveal-id="DSN016C25" data-open="DSN016C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and Chateigner-Boutin and Small<sup><a reveal-id="DSN016C46" data-open="DSN016C46" class="link link-ref link-reveal xref-bibr">46</a></sup> have recently extended these studies. There are 32 known RNA editing sites in protein-coding regions. Our method determined scores &gt;101 for 254 Cs in <em>A</em>. <em>thaliana</em> chloroplast transcripts (GenBank ID: AP000423). Six out of the verified 32 RNA editing sites were included in the 254 Cs. They are editing sites from <em>psbF</em>, <em>clpP</em>, <em>rpoA</em>, <em>rpl23</em> and <em>ndhB</em>. There are 248 (=254−6) predicted RNA editing sites that have not been experimentally verified and 26 (=32−6) verified RNA editing sites that were missed by our prediction. The large number of predicted sites may contain sites that can be verified as true RNA editing sites, of which the top seven are listed in Table <a reveal-id="DSN016TB3" data-open="DSN016TB3" class="link link-reveal link-table xref-fig">3</a>. If those predicted sites are false-positives, then the high score was obtained because of lack of information in the prediction method, the existence of <em>trans</em>-factors in <em>T</em>. <em>lepidozioides</em> that evolved to recognize different sequences or the disappearance of <em>trans</em>-factors during the course of evolution. The inability to predict the RNA editing sites in <em>A</em>. <em>thaliana</em> may be due to the existence of <em>trans</em>-factors for RNA editing site recognition that do not exist in <em>T</em>. <em>lepidozioides</em>.</p>                        <a id="5180011" scrollto-destination="5180011"></a>
<div content-id="DSN016TB3" class="table-modal"><div class="table-wrap"><div id="DSN016TB3" class="table-wrap-title" data-id="DSN016TB3"><div class="label -label">Table 3</div><div class="caption"><p>Predicted RNA editing site (score &gt; 130) for <em>A</em>. <em>thaliana</em> chloroplast genome transcripts (AP000423)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>predicted edited site with -30 to +10 nucleotides </th><th>gene </th><th>position* </th><th>score </th></tr></thead><tbody><tr><td><span class="monospace">ATACTGCTCTAGTTGCTGGTTGGGCTGGTT<strong>C</strong>GATGGCTCTA</span> </td><td>psbB </td><td>72477 </td><td>146.96 </td></tr><tr><td><span class="monospace">GACTATAACGAATCCGGGTCTTTGGAGTTA<strong>C</strong>GAAGGGGTAG</span> </td><td>psbB </td><td>72649 </td><td>145.06 </td></tr><tr><td><span class="monospace">AAGCAGGGGCTGCGGTAGCTGCTGAATCTT<strong>C</strong>TACTGGTACA</span> </td><td>rbcL </td><td>55142 </td><td>144.09 </td></tr><tr><td><span class="monospace">GCAGGATTCATCATGACAGGAGCTTTTGCT<strong>C</strong>ATGGAGCTAT</span> </td><td>psaB </td><td>−38415 </td><td>133.73 </td></tr><tr><td><span class="monospace">TATCTGGTGGAGATCATATTCACGCGGGTA<strong>C</strong>AGTAGTAGGT</span> </td><td>rbcL </td><td>55946 </td><td>132.84 </td></tr><tr><td><span class="monospace">TACCTATCTCAATAAAGTTTATGATTGGTT<strong>C</strong>GAAGAACGTC</span> </td><td>petB </td><td>75668 </td><td>131.15 </td></tr><tr><td><span class="monospace">CTGGAGTTCCACCTGAAGAAGCAGGGGCTG<strong>C</strong>GGTAGCTGCT</span> </td><td>rbcL </td><td>55124 </td><td>130.87 </td></tr></tbody></table></div><div content-id="DSN016TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>predicted edited site with -30 to +10 nucleotides </th><th>gene </th><th>position* </th><th>score </th></tr></thead><tbody><tr><td><span class="monospace">ATACTGCTCTAGTTGCTGGTTGGGCTGGTT<strong>C</strong>GATGGCTCTA</span> </td><td>psbB </td><td>72477 </td><td>146.96 </td></tr><tr><td><span class="monospace">GACTATAACGAATCCGGGTCTTTGGAGTTA<strong>C</strong>GAAGGGGTAG</span> </td><td>psbB </td><td>72649 </td><td>145.06 </td></tr><tr><td><span class="monospace">AAGCAGGGGCTGCGGTAGCTGCTGAATCTT<strong>C</strong>TACTGGTACA</span> </td><td>rbcL </td><td>55142 </td><td>144.09 </td></tr><tr><td><span class="monospace">GCAGGATTCATCATGACAGGAGCTTTTGCT<strong>C</strong>ATGGAGCTAT</span> </td><td>psaB </td><td>−38415 </td><td>133.73 </td></tr><tr><td><span class="monospace">TATCTGGTGGAGATCATATTCACGCGGGTA<strong>C</strong>AGTAGTAGGT</span> </td><td>rbcL </td><td>55946 </td><td>132.84 </td></tr><tr><td><span class="monospace">TACCTATCTCAATAAAGTTTATGATTGGTT<strong>C</strong>GAAGAACGTC</span> </td><td>petB </td><td>75668 </td><td>131.15 </td></tr><tr><td><span class="monospace">CTGGAGTTCCACCTGAAGAAGCAGGGGCTG<strong>C</strong>GGTAGCTGCT</span> </td><td>rbcL </td><td>55124 </td><td>130.87 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Position is the nucleotide number of the predicted edited site (bold) in the sequence and the minus number indicates that the editing site is predicted to be on the opposite strand.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5180011" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN016TB3" class="table-wrap-title" data-id="DSN016TB3"><div class="label -label">Table 3</div><div class="caption"><p>Predicted RNA editing site (score &gt; 130) for <em>A</em>. <em>thaliana</em> chloroplast genome transcripts (AP000423)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>predicted edited site with -30 to +10 nucleotides </th><th>gene </th><th>position* </th><th>score </th></tr></thead><tbody><tr><td><span class="monospace">ATACTGCTCTAGTTGCTGGTTGGGCTGGTT<strong>C</strong>GATGGCTCTA</span> </td><td>psbB </td><td>72477 </td><td>146.96 </td></tr><tr><td><span class="monospace">GACTATAACGAATCCGGGTCTTTGGAGTTA<strong>C</strong>GAAGGGGTAG</span> </td><td>psbB </td><td>72649 </td><td>145.06 </td></tr><tr><td><span class="monospace">AAGCAGGGGCTGCGGTAGCTGCTGAATCTT<strong>C</strong>TACTGGTACA</span> </td><td>rbcL </td><td>55142 </td><td>144.09 </td></tr><tr><td><span class="monospace">GCAGGATTCATCATGACAGGAGCTTTTGCT<strong>C</strong>ATGGAGCTAT</span> </td><td>psaB </td><td>−38415 </td><td>133.73 </td></tr><tr><td><span class="monospace">TATCTGGTGGAGATCATATTCACGCGGGTA<strong>C</strong>AGTAGTAGGT</span> </td><td>rbcL </td><td>55946 </td><td>132.84 </td></tr><tr><td><span class="monospace">TACCTATCTCAATAAAGTTTATGATTGGTT<strong>C</strong>GAAGAACGTC</span> </td><td>petB </td><td>75668 </td><td>131.15 </td></tr><tr><td><span class="monospace">CTGGAGTTCCACCTGAAGAAGCAGGGGCTG<strong>C</strong>GGTAGCTGCT</span> </td><td>rbcL </td><td>55124 </td><td>130.87 </td></tr></tbody></table></div><div content-id="DSN016TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>predicted edited site with -30 to +10 nucleotides </th><th>gene </th><th>position* </th><th>score </th></tr></thead><tbody><tr><td><span class="monospace">ATACTGCTCTAGTTGCTGGTTGGGCTGGTT<strong>C</strong>GATGGCTCTA</span> </td><td>psbB </td><td>72477 </td><td>146.96 </td></tr><tr><td><span class="monospace">GACTATAACGAATCCGGGTCTTTGGAGTTA<strong>C</strong>GAAGGGGTAG</span> </td><td>psbB </td><td>72649 </td><td>145.06 </td></tr><tr><td><span class="monospace">AAGCAGGGGCTGCGGTAGCTGCTGAATCTT<strong>C</strong>TACTGGTACA</span> </td><td>rbcL </td><td>55142 </td><td>144.09 </td></tr><tr><td><span class="monospace">GCAGGATTCATCATGACAGGAGCTTTTGCT<strong>C</strong>ATGGAGCTAT</span> </td><td>psaB </td><td>−38415 </td><td>133.73 </td></tr><tr><td><span class="monospace">TATCTGGTGGAGATCATATTCACGCGGGTA<strong>C</strong>AGTAGTAGGT</span> </td><td>rbcL </td><td>55946 </td><td>132.84 </td></tr><tr><td><span class="monospace">TACCTATCTCAATAAAGTTTATGATTGGTT<strong>C</strong>GAAGAACGTC</span> </td><td>petB </td><td>75668 </td><td>131.15 </td></tr><tr><td><span class="monospace">CTGGAGTTCCACCTGAAGAAGCAGGGGCTG<strong>C</strong>GGTAGCTGCT</span> </td><td>rbcL </td><td>55124 </td><td>130.87 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Position is the nucleotide number of the predicted edited site (bold) in the sequence and the minus number indicates that the editing site is predicted to be on the opposite strand.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5180011" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5180012 id="5180012" class="section-title" data-legacy-id=s3g>3.7. Conclusion</h3>
<p>We classified the neighbor sequences of <em>T</em>. <em>lepidozioides</em> chloroplast RNA editing sites, taking advantage of the fact that chloroplast transcripts have multiple RNA editing sites. The neighbor sequences of RNA editing sites can be classified into eight groups using distinct singlet and doublet propensities. The differences in propensities among groups are hypothesized to result from recognition by different <em>trans</em>-factors for each group. The propensities predicted RNA editing sites using the nucleotide sequence alone and worked to predict sites in the genome of the same species. Our prediction was also able to identify silent RNA editing sites. Application of the prediction to chloroplast transcripts of a different species worked with a lower efficiency, yet the validity of these result needs to be experimentally tested. The prediction method is available at <a class="link link-uri" href="http://cib.cf.ocha.ac.jp/~yura/RNAE/">http://cib.cf.ocha.ac.jp/~yura/RNAE/</a>.</p>                            <h2 scrollto-destination=5180014 id="5180014" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5180016 id="5180016" class="section-title" data-legacy-id=s5>Funding</h2>
<p>K.Y. was supported by a Grant-in-Aid for Scientific Research (C) from the Japan Society for the Promotion of Science (JSPS) KAKENHI (19570157), Y.M. by the JSPS Research Fellowship for Young Scientists, T.A. by a Grant-in-Aid for Young Scientists (B) from the Ministry of Education, Culture, Sports, Science and Technology (MEXT) of Japan (17770073), M.H. by a Grant-in-Aid for Scientific Research (C) from JSPS KAKENHI (13640707) and M.S. by a Grant-in-Aid for Scientific Research (C) from JSPS KAKENHI (2057003).</p>                            <h2 scrollto-destination=5180018 id="5180018" class="backacknowledgements-title" >Acknowledgement</h2>
<p>The authors thank Kaori Maruyama and Chieko Sugita (Nagoya University) for DNA sequencing.</p>                            <h2 scrollto-destination=5180020 id="5180020" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN016C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bock</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Sense from nonsense: how the genetic information of chloroplasts is altered by RNA editing</div>, <div class="source">Biochimie</div> , <div class="year">2000</div>, vol. <div class="volume">82</div> (pg. <div class="fpage">549</div>-<div class="lpage">557</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sense%20from%20nonsense%3A%20how%20the%20genetic%20information%20of%20chloroplasts%20is%20altered%20by%20RNA%20editing&amp;author=R.%20Bock&amp;publication_year=2000&amp;journal=Biochimie&amp;volume=82&amp;pages=549-557" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0300-9084(00)00610-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0300-9084(00)00610-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0300-9084(00)00610-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10946106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Covello</div> <div class="given-names">P. S.</div></div>,  <div class="name"><div class="surname">Gray</div> <div class="given-names">M. W.</div></div>. </div><div class="article-title">RNA editing in plant mitochondria</div>, <div class="source">Nature</div> , <div class="year">1989</div>, vol. <div class="volume">341</div> (pg. <div class="fpage">662</div>-<div class="lpage">666</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20editing%20in%20plant%20mitochondria&amp;author=P.%20S.%20Covello&amp;author=M.%20W.%20Gray&amp;publication_year=1989&amp;journal=Nature&amp;volume=341&amp;pages=662-666" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/341662a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F341662a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F341662a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2552326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoch</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Maier</div> <div class="given-names">R. M.</div></div>,  <div class="name"><div class="surname">Appel</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Igloi</div> <div class="given-names">G. L.</div></div>,  <div class="name"><div class="surname">Kössel</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Editing of a chloroplast mRNA by creation of an initiation codon</div>, <div class="source">Nature</div> , <div class="year">1991</div>, vol. <div class="volume">353</div> (pg. <div class="fpage">178</div>-<div class="lpage">180</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Editing%20of%20a%20chloroplast%20mRNA%20by%20creation%20of%20an%20initiation%20codon&amp;author=B.%20Hoch&amp;author=R.%20M.%20Maier&amp;author=K.%20Appel&amp;author=G.%20L.%20Igloi&amp;author=H.%20K%C3%B6ssel&amp;publication_year=1991&amp;journal=Nature&amp;volume=353&amp;pages=178-180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/353178a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F353178a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F353178a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1653905" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiesel</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Combettes</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Brennicke</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Evidence for RNA editing in mitochondria of all major groups of land plants except the Bryophyta</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">629</div>-<div class="lpage">633</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20RNA%20editing%20in%20mitochondria%20of%20all%20major%20groups%20of%20land%20plants%20except%20the%20Bryophyta&amp;author=R.%20Hiesel&amp;author=B.%20Combettes&amp;author=A.%20Brennicke&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=629-633" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.2.629" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.2.629" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.2.629"> </span></div> </div></div></div></div></div><div content-id="DSN016C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoshinaga</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Iinuma</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Masuzawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ueda</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Extensive RNA editing of U to C in addition to C to U substitution in the <em>rbcL</em> transcripts of hornwort chloroplasts and the origin of RNA editing in green plants</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1996</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1008</div>-<div class="lpage">1014</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20RNA%20editing%20of%20U%20to%20C%20in%20addition%20to%20C%20to%20U%20substitution%20in%20the%20rbcL%20transcripts%20of%20hornwort%20chloroplasts%20and%20the%20origin%20of%20RNA%20editing%20in%20green%20plants&amp;author=K.%20Yoshinaga&amp;author=H.%20Iinuma&amp;author=T.%20Masuzawa&amp;author=K.%20Ueda&amp;publication_year=1996&amp;journal=Nucleic%20Acids%20Res.&amp;volume=24&amp;pages=1008-1014" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/24.6.1008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F24.6.1008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F24.6.1008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8604330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Freyer</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kiefer-Meyer</div> <div class="given-names">M-C.</div></div>,  <div class="name"><div class="surname">Kössel</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Occurrence of plastid RNA editing in all major lineages of land plants</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1997</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">6285</div>-<div class="lpage">6290</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Occurrence%20of%20plastid%20RNA%20editing%20in%20all%20major%20lineages%20of%20land%20plants&amp;author=R.%20Freyer&amp;author=M-C.%20Kiefer-Meyer&amp;author=H.%20K%C3%B6ssel&amp;publication_year=1997&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=94&amp;pages=6285-6290" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.12.6285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.12.6285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.12.6285"> </span></div> </div></div></div></div></div><div content-id="DSN016C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Spencer</div> <div class="given-names">D. F.</div></div>,  <div class="name"><div class="surname">Norman</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Gray</div> <div class="given-names">M. W.</div></div>. </div><div class="article-title">Widespread and extensive editing of mitochondrial mRNAs in dinoflagellates</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">320</div> (pg. <div class="fpage">727</div>-<div class="lpage">739</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20and%20extensive%20editing%20of%20mitochondrial%20mRNAs%20in%20dinoflagellates&amp;author=S.%20Lin&amp;author=H.%20Zhang&amp;author=D.%20F.%20Spencer&amp;author=J.%20E.%20Norman&amp;author=M.%20W.%20Gray&amp;publication_year=2002&amp;journal=J.%20Mol.%20Biol.&amp;volume=320&amp;pages=727-739" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(02)00468-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(02)00468-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(02)00468-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12095251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kugita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fujikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yoshinaga</div> <div class="given-names">K.</div></div>. </div><div class="article-title">RNA editing in hornwort chloroplasts makes more than half the genes functional</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">2417</div>-<div class="lpage">2423</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20editing%20in%20hornwort%20chloroplasts%20makes%20more%20than%20half%20the%20genes%20functional&amp;author=M.%20Kugita&amp;author=Y.%20Yamamoto&amp;author=T.%20Fujikawa&amp;author=T.%20Matsumoto&amp;author=K.%20Yoshinaga&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=2417-2423" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12711687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giegé</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Brennicke</div> <div class="given-names">A.</div></div>. </div><div class="article-title">RNA editing in <em>Arabidopsis</em> mitochondria effects 441 C to U changes in ORFs</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">15324</div>-<div class="lpage">15329</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20editing%20in%20Arabidopsis%20mitochondria%20effects%20441%20C%20to%20U%20changes%20in%20ORFs&amp;author=P.%20Gieg%C3%A9&amp;author=A.%20Brennicke&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=15324-15329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.26.15324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.26.15324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.26.15324"> </span></div> </div></div></div></div></div><div content-id="DSN016C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wakasugi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hirose</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Horihata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tsudzuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kössel</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sugiura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Creation of a novel protein-coding region at the RNA level in black pine chloroplasts: the pattern of RNA editing in the gymnosperm chloroplast is different from that in angiosperms</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1996</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">8766</div>-<div class="lpage">8770</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Creation%20of%20a%20novel%20protein-coding%20region%20at%20the%20RNA%20level%20in%20black%20pine%20chloroplasts%3A%20the%20pattern%20of%20RNA%20editing%20in%20the%20gymnosperm%20chloroplast%20is%20different%20from%20that%20in%20angiosperms&amp;author=T.%20Wakasugi&amp;author=T.%20Hirose&amp;author=M.%20Horihata&amp;author=T.%20Tsudzuki&amp;author=H.%20K%C3%B6ssel&amp;author=M.%20Sugiura&amp;publication_year=1996&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=93&amp;pages=8766-8770" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.16.8766" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.16.8766" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.16.8766"> </span></div> </div></div></div></div></div><div content-id="DSN016C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mulligan</div> <div class="given-names">R. M.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Shanahan</div> <div class="given-names">M. T.</div></div>. </div><div class="article-title">RNA editing site recognition in higher plant mitochondria</div>, <div class="source">J. Hered.</div> , <div class="year">1999</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">338</div>-<div class="lpage">344</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20editing%20site%20recognition%20in%20higher%20plant%20mitochondria&amp;author=R.%20M.%20Mulligan&amp;author=M.%20A.%20Williams&amp;author=M.%20T.%20Shanahan&amp;publication_year=1999&amp;journal=J.%20Hered.&amp;volume=90&amp;pages=338-344" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/90.3.338" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F90.3.338" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F90.3.338"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10355119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blanc</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Litvak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Araya</div> <div class="given-names">A.</div></div>. </div><div class="article-title">RNA editing in wheat mitochondria proceeds by a deamination mechanism</div>, <div class="source">FEBS Lett.</div> , <div class="year">1995</div>, vol. <div class="volume">373</div> (pg. <div class="fpage">56</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20editing%20in%20wheat%20mitochondria%20proceeds%20by%20a%20deamination%20mechanism&amp;author=V.%20Blanc&amp;author=S.%20Litvak&amp;author=A.%20Araya&amp;publication_year=1995&amp;journal=FEBS%20Lett.&amp;volume=373&amp;pages=56-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(95)00991-H" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(95)00991-H" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(95)00991-H"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7589434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Schuster</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Evidence for a site-specific cytidine deamination reaction involved in C to U RNA editing of plant mitochondria</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">18227</div>-<div class="lpage">18233</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20a%20site-specific%20cytidine%20deamination%20reaction%20involved%20in%20C%20to%20U%20RNA%20editing%20of%20plant%20mitochondria&amp;author=W.%20Yu&amp;author=W.%20Schuster&amp;publication_year=1995&amp;journal=J.%20Biol.%20Chem.&amp;volume=270&amp;pages=18227-18233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.270.31.18227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.270.31.18227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.270.31.18227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7629140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chaudhuri</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Carrer</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Maliga</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Site-specific factor involved in the editing of the <em>psbL</em> mRNA in tobacco plastids</div>, <div class="source">EMBO J.</div> , <div class="year">1995</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2951</div>-<div class="lpage">2957</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Site-specific%20factor%20involved%20in%20the%20editing%20of%20the%20psbL%20mRNA%20in%20tobacco%20plastids&amp;author=S.%20Chaudhuri&amp;author=H.%20Carrer&amp;author=P.%20Maliga&amp;publication_year=1995&amp;journal=EMBO%20J.&amp;volume=14&amp;pages=2951-2957" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7796820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bock</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hermann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kössel</div> <div class="given-names">H.</div></div>. </div><div class="article-title"><em>In vivo</em> dissection of <em>cis</em>-acting determinants for plastid RNA editing</div>, <div class="source">EMBO J.</div> , <div class="year">1996</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">5052</div>-<div class="lpage">5059</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20dissection%20of%20cis-acting%20determinants%20for%20plastid%20RNA%20editing&amp;author=R.%20Bock&amp;author=M.%20Hermann&amp;author=H.%20K%C3%B6ssel&amp;publication_year=1996&amp;journal=EMBO%20J.&amp;volume=15&amp;pages=5052-5059" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8890178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bock</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hermann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fuchs</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Identification of critical nucleotide positions for plastid RNA editing site recognition</div>, <div class="source">RNA</div> , <div class="year">1997</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1194</div>-<div class="lpage">1200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20critical%20nucleotide%20positions%20for%20plastid%20RNA%20editing%20site%20recognition&amp;author=R.%20Bock&amp;author=M.%20Hermann&amp;author=M.%20Fuchs&amp;publication_year=1997&amp;journal=RNA&amp;volume=3&amp;pages=1194-1200" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9326494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chaudhuri</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Maliga</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Sequences directing C to U editing of the plastid <em>psbL</em> mRNA are located within a 22 nucleotide segment spanning the editing site</div>, <div class="source">EMBO J.</div> , <div class="year">1996</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">5958</div>-<div class="lpage">5964</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequences%20directing%20C%20to%20U%20editing%20of%20the%20plastid%20psbL%20mRNA%20are%20located%20within%20a%2022%20nucleotide%20segment%20spanning%20the%20editing%20site&amp;author=S.%20Chaudhuri&amp;author=P.%20Maliga&amp;publication_year=1996&amp;journal=EMBO%20J.&amp;volume=15&amp;pages=5958-5964" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8918473" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hermann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bock</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Transfer of plastid RNA-editing activity to novel sites suggests a critical role for spacing in editing-site recognition</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">4856</div>-<div class="lpage">4861</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20of%20plastid%20RNA-editing%20activity%20to%20novel%20sites%20suggests%20a%20critical%20role%20for%20spacing%20in%20editing-site%20recognition&amp;author=M.%20Hermann&amp;author=R.%20Bock&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=4856-4861" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.9.4856" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.9.4856" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.9.4856"> </span></div> </div></div></div></div></div><div content-id="DSN016C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reed</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">Peeters</div> <div class="given-names">N. M.</div></div>,  <div class="name"><div class="surname">Hanson</div> <div class="given-names">M. R.</div></div>. </div><div class="article-title">A single alternation 20 nt 5′ to an editing target inhibits chloroplast RNA editing <em>in vivo</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">1507</div>-<div class="lpage">1513</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20single%20alternation%2020%20nt%205%E2%80%B2%20to%20an%20editing%20target%20inhibits%20chloroplast%20RNA%20editing%20in%20vivo&amp;author=M.%20L.%20Reed&amp;author=N.%20M.%20Peeters&amp;author=M.%20R.%20Hanson&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=1507-1513" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.7.1507" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.7.1507" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.7.1507"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11266552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmitz-Linneweber</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tillich</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hermann</div> <div class="given-names">R. G.</div></div>,  <div class="name"><div class="surname">Maier</div> <div class="given-names">R. M.</div></div>. </div><div class="article-title">Heterologous, splicing-dependent RNA editing in chloroplasts: allotetrapolidy provides <em>trans</em>-factors</div>, <div class="source">EMBO J.</div> , <div class="year">2001</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">4874</div>-<div class="lpage">4883</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterologous%2C%20splicing-dependent%20RNA%20editing%20in%20chloroplasts%3A%20allotetrapolidy%20provides%20trans-factors&amp;author=C.%20Schmitz-Linneweber&amp;author=M.%20Tillich&amp;author=R.%20G.%20Hermann&amp;author=R.%20M.%20Maier&amp;publication_year=2001&amp;journal=EMBO%20J.&amp;volume=20&amp;pages=4874-4883" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/20.17.4874" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F20.17.4874" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F20.17.4874"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11532951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Obokata</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sugiura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A site-specific factor interacts directly with its cognate RNA editing site in chloroplast transcripts</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">48</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20site-specific%20factor%20interacts%20directly%20with%20its%20cognate%20RNA%20editing%20site%20in%20chloroplast%20transcripts&amp;author=T.%20Miyamoto&amp;author=J.%20Obokata&amp;author=M.%20Sugiura&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=48-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307163101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307163101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307163101"> </span></div> </div></div></div></div></div><div content-id="DSN016C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yukawa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wakasugi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugiura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A simple <em>in vitro</em> RNA editing assay for chloroplast transcripts using fluorescent dideoxynucleotides: distinct types of sequence elements required for editing of <em>ndh</em> transcripts</div>, <div class="source">Plant J.</div> , <div class="year">2006</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">802</div>-<div class="lpage">810</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20in%20vitro%20RNA%20editing%20assay%20for%20chloroplast%20transcripts%20using%20fluorescent%20dideoxynucleotides%3A%20distinct%20types%20of%20sequence%20elements%20required%20for%20editing%20of%20ndh%20transcripts&amp;author=T.%20Sasaki&amp;author=Y.%20Yukawa&amp;author=T.%20Wakasugi&amp;author=K.%20Yamada&amp;author=M.%20Sugiura&amp;publication_year=2006&amp;journal=Plant%20J.&amp;volume=47&amp;pages=802-810" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2006.02825.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2006.02825.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2006.02825.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16856984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tillich</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lehwark</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Morton</div> <div class="given-names">B. R.</div></div>,  <div class="name"><div class="surname">Maier</div> <div class="given-names">U. G.</div></div>. </div><div class="article-title">The evolution of chloroplast RNA editing</div>, <div class="source">Mol. Biol., Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1912</div>-<div class="lpage">1921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20chloroplast%20RNA%20editing&amp;author=M.%20Tillich&amp;author=P.%20Lehwark&amp;author=B.%20R.%20Morton&amp;author=U.%20G.%20Maier&amp;publication_year=2006&amp;journal=Mol.%20Biol.%2C%20Evol.&amp;volume=23&amp;pages=1912-1921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl054"> </span></div> </div></div></div></div></div><div content-id="DSN016C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gualberto</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Bonnard</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lamattina</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Grienenberger</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">Expression of the wheat mitochondrial <em>nad3</em>-<em>rps12</em> transcription unit: correlation between editing and mRNA maturation</div>, <div class="source">Plant Cell</div> , <div class="year">1991</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1109</div>-<div class="lpage">1120</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20the%20wheat%20mitochondrial%20nad3-rps12%20transcription%20unit%3A%20correlation%20between%20editing%20and%20mRNA%20maturation&amp;author=J.%20M.%20Gualberto&amp;author=G.%20Bonnard&amp;author=L.%20Lamattina&amp;author=J.%20M.%20Grienenberger&amp;publication_year=1991&amp;journal=Plant%20Cell&amp;volume=3&amp;pages=1109-1120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.3.10.1109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.3.10.1109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.3.10.1109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1726558" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tillich</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Funk</div> <div class="given-names">H. T.</div></div>,  <div class="name"><div class="surname">Schmitz-Linneweber</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Poltnigg</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sabater</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Maier</div> <div class="given-names">R. M.</div></div>. </div><div class="article-title">Editing of plastid RNA in <em>Arabidopsis thaliana</em> ecotypes</div>, <div class="source">Plant J.</div> , <div class="year">2005</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">708</div>-<div class="lpage">715</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Editing%20of%20plastid%20RNA%20in%20Arabidopsis%20thaliana%20ecotypes&amp;author=M.%20Tillich&amp;author=H.%20T.%20Funk&amp;author=C.%20Schmitz-Linneweber&amp;author=P.%20Poltnigg&amp;author=B.%20Sabater&amp;author=M.%20Martin&amp;author=R.%20M.%20Maier&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=43&amp;pages=708-715" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.43.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.43.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.43.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16115067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirose</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kusumegi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tsudzuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugiura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">RNA editing sites in tobacco chloroplast transcripts: editing as a possible regulator of chloroplast RNA polymerase activity</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">262</div> (pg. <div class="fpage">462</div>-<div class="lpage">467</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20editing%20sites%20in%20tobacco%20chloroplast%20transcripts%3A%20editing%20as%20a%20possible%20regulator%20of%20chloroplast%20RNA%20polymerase%20activity&amp;author=T.%20Hirose&amp;author=T.%20Kusumegi&amp;author=T.%20Tsudzuki&amp;author=M.%20Sugiura&amp;publication_year=1999&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=262&amp;pages=462-467" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380051106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380051106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380051106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10589833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kotera</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Tasaka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Shikanai</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A pentatricopeptide repeat protein is essential for RNA editing in chloroplasts</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">433</div> (pg. <div class="fpage">326</div>-<div class="lpage">330</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20pentatricopeptide%20repeat%20protein%20is%20essential%20for%20RNA%20editing%20in%20chloroplasts&amp;author=E.%20Kotera&amp;author=M.%20Tasaka&amp;author=T.%20Shikanai&amp;publication_year=2005&amp;journal=Nature&amp;volume=433&amp;pages=326-330" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03229" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03229"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15662426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okuda</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Shikanai</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A pentatricopeptide repeat protein is a site recognition factor in chloroplast RNA editing</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2006</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">37661</div>-<div class="lpage">37667</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20pentatricopeptide%20repeat%20protein%20is%20a%20site%20recognition%20factor%20in%20chloroplast%20RNA%20editing&amp;author=K.%20Okuda&amp;author=T.%20Nakamura&amp;author=M.%20Sugita&amp;author=T.%20Shimizu&amp;author=T.%20Shikanai&amp;publication_year=2006&amp;journal=J.%20Biol.%20Chem.&amp;volume=281&amp;pages=37661-37667" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M608184200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M608184200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M608184200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17015439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okuda</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Myouga</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Motohashi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shikanai</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Conserved domain structure of pentatricopeptide repeat proteins involved in chloroplast RNA editing</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">8178</div>-<div class="lpage">8183</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20domain%20structure%20of%20pentatricopeptide%20repeat%20proteins%20involved%20in%20chloroplast%20RNA%20editing&amp;author=K.%20Okuda&amp;author=F.%20Myouga&amp;author=R.%20Motohashi&amp;author=K.%20Shinozaki&amp;author=T.%20Shikanai&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=8178-8183" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0700865104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0700865104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0700865104"> </span></div> </div></div></div></div></div><div content-id="DSN016C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">O'Toole</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hattori</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Andres</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Iida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lurin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schmitz-Linneweber</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Sugita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Small</div> <div class="given-names">I.</div></div>. </div><div class="article-title">On the Expansion of the pentatricopeptide repeat gene family in plants</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2008</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1120</div>-<div class="lpage">1128</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20Expansion%20of%20the%20pentatricopeptide%20repeat%20gene%20family%20in%20plants&amp;author=N.%20O'Toole&amp;author=M.%20Hattori&amp;author=C.%20Andres&amp;author=K.%20Iida&amp;author=C.%20Lurin&amp;author=C.%20Schmitz-Linneweber&amp;author=M.%20Sugita&amp;author=I.%20Small&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=25&amp;pages=1120-1128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18343892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lurin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Andrés</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Aubourg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bellaoui</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bitton</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bruyère</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Caboche</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Debast</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gualberto</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hoffmann</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lecharny</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Le Ret</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Martin-Magniette</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">Mireau</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Peeters</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Renou</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Szurek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Taconnat</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Small</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Genome-wide analysis of <em>Arabidopsis</em> pentatricopeptide repeat proteins reveals their essential role in organelle biogenesis</div>, <div class="source">Plant Cell</div> , <div class="year">2004</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">2089</div>-<div class="lpage">2103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20Arabidopsis%20pentatricopeptide%20repeat%20proteins%20reveals%20their%20essential%20role%20in%20organelle%20biogenesis&amp;author=C.%20Lurin&amp;author=C.%20Andr%C3%A9s&amp;author=S.%20Aubourg&amp;author=M.%20Bellaoui&amp;author=F.%20Bitton&amp;author=C.%20Bruy%C3%A8re&amp;author=M.%20Caboche&amp;author=C.%20Debast&amp;author=J.%20Gualberto&amp;author=B.%20Hoffmann&amp;author=A.%20Lecharny&amp;author=M.%20Le%20Ret&amp;author=M.%20L.%20Martin-Magniette&amp;author=H.%20Mireau&amp;author=N.%20Peeters&amp;author=J.%20P.%20Renou&amp;author=B.%20Szurek&amp;author=L.%20Taconnat&amp;author=I.%20Small&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=2089-2103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.104.022236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.104.022236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.104.022236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15269332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salone</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Rüdinger</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Polsakiewicz</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hoffmann</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Groth-Malonek</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Szurek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Small</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Knoop</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Lurin</div> <div class="given-names">C.</div></div>. </div><div class="article-title">A hypothesis on the identification of the editing enzyme in plant organelles</div>, <div class="source">FEBS Lett.</div> , <div class="year">2007</div>, vol. <div class="volume">581</div> (pg. <div class="fpage">4132</div>-<div class="lpage">4138</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hypothesis%20on%20the%20identification%20of%20the%20editing%20enzyme%20in%20plant%20organelles&amp;author=V.%20Salone&amp;author=M.%20R%C3%BCdinger&amp;author=M.%20Polsakiewicz&amp;author=B.%20Hoffmann&amp;author=M.%20Groth-Malonek&amp;author=B.%20Szurek&amp;author=I.%20Small&amp;author=V.%20Knoop&amp;author=C.%20Lurin&amp;publication_year=2007&amp;journal=FEBS%20Lett.&amp;volume=581&amp;pages=4132-4138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2007.07.075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2007.07.075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2007.07.075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17707818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cummings</div> <div class="given-names">M. P.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">D. S.</div></div>. </div><div class="article-title">Simple statistical models predict C-to-U edited sites in plant mitochondrial RNA</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">132</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20statistical%20models%20predict%20C-to-U%20edited%20sites%20in%20plant%20mitochondrial%20RNA&amp;author=M.%20P.%20Cummings&amp;author=D.%20S.%20Myers&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-132" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-132" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-5-132"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15373947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mower</div> <div class="given-names">J. P.</div></div>. </div><div class="article-title">PREP-Mt: predictive RNA editor for plant mitochondrial genes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">96</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PREP-Mt%3A%20predictive%20RNA%20editor%20for%20plant%20mitochondrial%20genes&amp;author=J.%20P.%20Mower&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-96" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-96" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-96"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15826309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gopal</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Genetic algorithm learning as a robust approach to RNA editing site prediction</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">145</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20algorithm%20learning%20as%20a%20robust%20approach%20to%20RNA%20editing%20site%20prediction&amp;author=J.%20Thompson&amp;author=S.%20Gopal&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=145" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-145" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-145"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16542417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Prediction of C-to-U RNA editing sites in higher plant mitochondria using only nucleotide sequence features</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2007</div>, vol. <div class="volume">358</div> (pg. <div class="fpage">336</div>-<div class="lpage">341</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20C-to-U%20RNA%20editing%20sites%20in%20higher%20plant%20mitochondria%20using%20only%20nucleotide%20sequence%20features&amp;author=P.%20Du&amp;author=T.%20He&amp;author=Y.%20Li&amp;publication_year=2007&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=358&amp;pages=336-341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2007.04.130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2007.04.130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2007.04.130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17482141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeng</div> <div class="given-names">W. H.</div></div>,  <div class="name"><div class="surname">Liao</div> <div class="given-names">S. C.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">C. C.</div></div>. </div><div class="article-title">Identification of RNA editing sites in chloroplast transcripts of <em>Phalaenopsis aphrodite</em> and comparative analysis with those of other seed plants</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">362</div>-<div class="lpage">368</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20RNA%20editing%20sites%20in%20chloroplast%20transcripts%20of%20Phalaenopsis%20aphrodite%20and%20comparative%20analysis%20with%20those%20of%20other%20seed%20plants&amp;author=W.%20H.%20Zeng&amp;author=S.%20C.%20Liao&amp;author=C.%20C.%20Chang&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol.&amp;volume=48&amp;pages=362-368" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcl058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcl058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcl058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17169923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Miyata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Maruyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugiura</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Arikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Higuchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Extensive RNA editing in transcripts from the <em>psbB</em> operon and <em>rpoA</em> gene of plastids from the enigmatic moss <em>Takakia lepidozioides</em></div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2006</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">2268</div>-<div class="lpage">2274</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20RNA%20editing%20in%20transcripts%20from%20the%20psbB%20operon%20and%20rpoA%20gene%20of%20plastids%20from%20the%20enigmatic%20moss%20Takakia%20lepidozioides&amp;author=M.%20Sugita&amp;author=Y.%20Miyata&amp;author=K.%20Maruyama&amp;author=C.%20Sugiura&amp;author=T.%20Arikawa&amp;author=M.%20Higuchi&amp;publication_year=2006&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=70&amp;pages=2268-2274" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.60204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.60204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.60204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16960353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sugita</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Maruyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugita</div> <div class="given-names">M.</div></div>. </div><div class="article-title">RNA editing in the anticodon of tRNA<sup>Leu</sup> (CAA) occurs before group I intron splicing in plastids of a moss <em>Takakia lepidozioides</em> S. Hatt. &amp; Inoue</div>, <div class="source">Plant Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">250</div>-<div class="lpage">255</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20editing%20in%20the%20anticodon%20of%20tRNALeu%20(CAA)%20occurs%20before%20group%20I%20intron%20splicing%20in%20plastids%20of%20a%20moss%20Takakia%20lepidozioides%20S.%20Hatt.%20%26%20Inoue&amp;author=Y.%20Miyata&amp;author=C.%20Sugita&amp;author=K.%20Maruyama&amp;author=M.%20Sugita&amp;publication_year=2008&amp;journal=Plant%20Biol.&amp;volume=10&amp;pages=250-255" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1438-8677.2007.00027.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1438-8677.2007.00027.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1438-8677.2007.00027.x"> </span></div> </div></div></div></div></div><div content-id="DSN016C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">406</div>-<div class="lpage">425</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-425" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Toh</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Go</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Putative mechanism of natural transformation as deduced from genome data</div>, <div class="source">DNA Res.</div> , <div class="year">1999</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">75</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Putative%20mechanism%20of%20natural%20transformation%20as%20deduced%20from%20genome%20data&amp;author=K.%20Yura&amp;author=H.%20Toh&amp;author=M.%20Go&amp;publication_year=1999&amp;journal=DNA%20Res.&amp;volume=6&amp;pages=75-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/6.2.75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F6.2.75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F6.2.75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10382965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">T. P. O.</div></div>,  <div class="name"><div class="surname">Yura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Go</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Amino acid residue doublet propensity in the protein-RNA interface and its application to RNA interface prediction</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">6450</div>-<div class="lpage">6460</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amino%20acid%20residue%20doublet%20propensity%20in%20the%20protein-RNA%20interface%20and%20its%20application%20to%20RNA%20interface%20prediction&amp;author=T.%20P.%20O.%20Kim&amp;author=K.%20Yura&amp;author=N.%20Go&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=6450-6460" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl819" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl819" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl819"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawrence</div> <div class="given-names">C. E.</div></div>,  <div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Boguski</div> <div class="given-names">M. S.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J. S.</div></div>,  <div class="name"><div class="surname">Neuwald</div> <div class="given-names">A. F.</div></div>,  <div class="name"><div class="surname">Wootton</div> <div class="given-names">J. C.</div></div>. </div><div class="article-title">Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment</div>, <div class="source">Science</div> , <div class="year">1993</div>, vol. <div class="volume">262</div> (pg. <div class="fpage">208</div>-<div class="lpage">214</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20subtle%20sequence%20signals%3A%20a%20Gibbs%20sampling%20strategy%20for%20multiple%20alignment&amp;author=C.%20E.%20Lawrence&amp;author=S.%20F.%20Altschul&amp;author=M.%20S.%20Boguski&amp;author=J.%20S.%20Liu&amp;author=A.%20F.%20Neuwald&amp;author=J.%20C.%20Wootton&amp;publication_year=1993&amp;journal=Science&amp;volume=262&amp;pages=208-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.8211139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.8211139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.8211139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8211139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Fester</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bock</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Schuser</div> <div class="given-names">W.</div></div>. </div><div class="article-title">RNA editing in higher plant mitochondria: Analysis of biochemistry and specificity</div>, <div class="source">Biochimie</div> , <div class="year">1995</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">79</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20editing%20in%20higher%20plant%20mitochondria%3A%20Analysis%20of%20biochemistry%20and%20specificity&amp;author=W.%20Yu&amp;author=T.%20Fester&amp;author=H.%20Bock&amp;author=W.%20Schuser&amp;publication_year=1995&amp;journal=Biochimie&amp;volume=77&amp;pages=79-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0300-9084(96)88108-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0300-9084(96)88108-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0300-9084(96)88108-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7599280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Inada</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yukawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tsudzuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugiura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A systematic search for RNA editing sites in pea chloroplasts: an editing event causes diversification from the evolutionarily conserved amino acid sequence</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">1615</div>-<div class="lpage">1622</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20systematic%20search%20for%20RNA%20editing%20sites%20in%20pea%20chloroplasts%3A%20an%20editing%20event%20causes%20diversification%20from%20the%20evolutionarily%20conserved%20amino%20acid%20sequence&amp;author=M.%20Inada&amp;author=T.%20Sasaki&amp;author=M.%20Yukawa&amp;author=T.%20Tsudzuki&amp;author=M.%20Sugiura&amp;publication_year=2004&amp;journal=Plant%20Cell%20Physiol.&amp;volume=45&amp;pages=1615-1622" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pch191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpch191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpch191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15574837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN016C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN016C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chateigner-Boutin</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Small</div> <div class="given-names">I.</div></div>. </div><div class="article-title">A rapid high-throughput method for the detection and quantification of RNA editing based on high-resolution melting of amplicons</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> pg. <div class="fpage">e114</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20high-throughput%20method%20for%20the%20detection%20and%20quantification%20of%20RNA%20editing%20based%20on%20high-resolution%20melting%20of%20amplicons&amp;author=A.%20L.%20Chateigner-Boutin&amp;author=I.%20Small&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=e114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm640" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm640" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm640"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17726051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=359388&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn016/2/dsn016-dsn016supp.pdf?Expires=2147483647&amp;Signature=j-PrKhZLuxAjcl33-NVT~ymRUrAVAAy7QHPrsRezPlt7mkqp1Ainosq~NdamuYbQQ4V2l48d3qRYTj4t0tQ9j4eqKHDtU~i2rhG9XAfYUVWYB~wwkuC8AbfU9w6IpJ38dXfumGv0~Pxe3JZhZ5c6E-VDee~Hfb4TU8MQ5w9dJdI-BUPmUfGS2vQerqdUBaSNX2e8elwrdBBZjb~FtzpcupHioaj3ULZEIAbI1novb~rZas0MDjxPx1dwNtzI-AvnpF-~BKUBVRM45OHfJPTqyRVsmL2FUrA7gWo9WQZexvz3eL0Yuxw-aydS9QtsnFpDPFJlXvwUKde~WlTRQq8zCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">351</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn016"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="359388"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f5%2f309%2f359388">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260843400005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F5%2F309%2F359388" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/359388'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Characteristics and Prediction of RNA Editing Sites in Transcripts of the Moss Takakia lepidozioides Chloroplast" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn016");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f309%2f359388">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

