

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1841,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/2/183/403825","@type":"ScholarlyArticle","name":"Large-Scale Development of Gene-Associated Single-Nucleotide Polymorphism Markers for Molluscan Population Genomic, Comparative Genomic, and Genome-Wide Association Studies","datePublished":"2013-11-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2014-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst048","keywords":["mollusca","single-nucleotide polymorphism (SNP)","transcriptome","high resolution melting (HRM)","genome-wide association (GWAS)"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/2/183/403825","author":[{"name":"Jiao, Wenqian","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Fu, Xiaoteng","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Li, Jinqin","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Li, Ling","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Feng, Liying","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Lv, Jia","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Zhang, Lu","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Wang, Xiaojian","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Li, Yangping","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Hou, Rui","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Zhang, Lingling","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Hu, Xiaoli","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Wang, Shi","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Bao, Zhenmin","affiliation":"Key Laboratory of Marine Genetics and Breeding ,  College of Marine Life Sciences, Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"}],"description":"Abstract.  Mollusca is the second most diverse group of animals in the world. Despite their perceived importance, omics-level studies have seldom been applied t","pageStart":"183","pageEnd":"193","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Large-Scale Development of Gene-Associated Single-Nucleotide Polymorphism Markers for Molluscan Population Genomic, Comparative Genomic, and Genome-Wide Association Studies","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Large-Scale Development of Gene-Associated Single-Nucleotide Polymorphism Markers for Molluscan Population Genomic, Comparative Genomic, and Genome-Wide Association Studies" />
    <meta property="og:description" content="Abstract.  Mollusca is the second most diverse group of animals in the world. Despite their perceived importance, omics-level studies have seldom been applied t" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/2/183/403825" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-11-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Jiao, Wenqian" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Fu, Xiaoteng" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Li, Jinqin" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Li, Ling" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Feng, Liying" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Lv, Jia" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Zhang, Lu" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Wang, Xiaojian" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Li, Yangping" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Hou, Rui" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Zhang, Lingling" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Hu, Xiaoli" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Wang, Shi" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Bao, Zhenmin" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_title" content="Large-Scale Development of Gene-Associated Single-Nucleotide Polymorphism Markers for Molluscan Population Genomic, Comparative Genomic, and Genome-Wide Association Studies" /><meta name="citation_firstpage" content="183" /><meta name="citation_lastpage" content="193" /><meta name="citation_doi" content="10.1093/dnares/dst048" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2014/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Relationships of higher molluscan taxa; citation_publisher=University of California Press, Berkeley; Phylogeny and Evolution of the Mollusca;  citation_year=2008;  citation_pages=19-32; " /><meta name="citation_reference" content="citation_title=Molluscanmemory of injury: evolutionary insights intochronic pain and neurological disorders; Brain Behav. Evol.;  citation_year=2009;  citation_volume=74;  citation_pages=206-18; " /><meta name="citation_reference" content="citation_title=Effects of past, present, and future ocean carbon dioxide concentrations on the growth and survival of larval shellfish; Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=17246-51; " /><meta name="citation_reference" content="citation_title=The oyster genome reveals stress adaptation and complexity of shell formation; Nature;  citation_year=2011;  citation_volume=490;  citation_pages=49-54; " /><meta name="citation_reference" content="citation_title=Draft genome of the pearl oyster Pinctada fucata: a platform for understanding bivalve biology; DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=117-30; " /><meta name="citation_reference" content="citation_title=Applications of next generation sequencing in molecular ecology of non-model organisms; Heredity;  citation_year=2011;  citation_volume=107;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Transcriptome characterization of the South African Abalone Haliotis midae using sequencing-by-synthesis; BMC Res. Notes;  citation_year=2011;  citation_volume=4;  citation_pages=59" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing and de novo analysis for Yesso scallop (Patinopecten yessoensis) using 454 GS Flx; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e21560" /><meta name="citation_reference" content="citation_title=Resolving the evolutionary relationship of molluscs with phylogenomic tools; Nature;  citation_year=2011;  citation_volume=480;  citation_pages=364-7; " /><meta name="citation_reference" content="citation_title=De novo sequencing and transcriptome analysis of the central nervous system of mollusk Lymnaea stagnalis by deep RAN sequencing; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e42546" /><meta name="citation_reference" content="citation_title=Transcriptome analysis of the Octopus vulgaris central nervous system; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e40320" /><meta name="citation_reference" content="citation_title=High-resolution DNA melting analysis for simple and efficient molecular diagnostics; Pharmacogenomics;  citation_year=2007;  citation_volume=8;  citation_pages=597-608; " /><meta name="citation_reference" content="citation_title=Mutation scanning using high-resolution melting; Biochem. Soc. Trans.;  citation_year=2009;  citation_volume=37;  citation_pages=433-7; " /><meta name="citation_reference" content="citation_title=Construction of a high-resolution genetic linkage map and comparative genome analysis for the reef-building coral Acropora millepora; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R126" /><meta name="citation_reference" content="citation_title=An efficient method for developing SNP markers based on EST data combined with high resolution melting (HRM) analysis; BMC Res. Notes;  citation_year=2010;  citation_volume=3;  citation_pages=51" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of Zhikong scallop (Chlamys farreri) and comparative transcriptomic analysis with Yesso scallop (Patinopecten yessoensis); PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e63927" /><meta name="citation_reference" content="citation_title=Development of 44 gene-based SNP markers in Zhikong scallop, Chlamys farreri; Conserv. Genet. Resour.;  citation_year=2011;  citation_volume=3;  citation_pages=659-63; " /><meta name="citation_reference" content="citation_title=Characterization of 38 EST-derived SNP markers in Zhikong scallop (Chlamys farreri) and their cross-species utility in Yesso scallop (Patinopecten yessoensis); Conserv. Genet. Resour.;  citation_year=2012;  citation_volume=4;  citation_pages=747-53; " /><meta name="citation_reference" content="citation_title=CAP3, a DNA sequence assembly program; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=QualitySNP, a pipeline for detecting single nucleotide polymorphisms and insertions/deletions in EST data from diploid and polyploidy species; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=438" /><meta name="citation_reference" content="citation_title=Fine-scale population genetic structure of Zhikong scallop (Chlamys farreri), do local marine currents drive geographical differentiation?; Mar. Biotechnol.;  citation_year=2009;  citation_volume=11;  citation_pages=223-35; " /><meta name="citation_reference" content="citation_title=Population genetic analysis of co-dominant and dominant markers and quantitative traits; Belg. J. Bot.;  citation_year=1997;  citation_volume=129;  citation_pages=157" /><meta name="citation_reference" content="citation_title=GENEPOP (Version 1.2), population genetics software for exact tests and ecumenicism; J. Hered.;  citation_year=1995;  citation_volume=86;  citation_pages=248-9; " /><meta name="citation_reference" content="citation_title=Blast2GO, A universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=KAAS, an automatic genome annotation and pathway reconstruction server; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W182-185; " /><meta name="citation_reference" content="citation_title=Population genomics, linkage disequilibrium holds the key; Curr. Biol.;  citation_year=2001;  citation_volume=11;  citation_pages=R576-9; " /><meta name="citation_reference" content="citation_title=Molecular phylogenetics of the Pectinidae (Mollusca, Bivalvia) and effect of increased taxon sampling and outgroup selection on tree topology; Mol. Phylogenet. Evol.;  citation_year=2008;  citation_volume=48;  citation_pages=1178-88; " /><meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=376-80; " /><meta name="citation_reference" content="citation_title=Accuracy and quanlity of massively parallel DNA pyrosequencing; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R143" /><meta name="citation_reference" content="citation_title=Analysis of 454 sequencing error rate, error sources, and artifact recombination for detection of low-frequency drug resistance mutations in HIV-1 DNA; Retrovirology;  citation_year=2013;  citation_volume=10;  citation_pages=18" /><meta name="citation_reference" content="citation_title=Next-generation transcriptome assembly; Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=671-82; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphisms and their relationship to codon usage bias in the Pacific oyster Crassostrea gigas; Gene;  citation_year=2007;  citation_volume=406;  citation_pages=13-22; " /><meta name="citation_reference" content="citation_title=Construction of microsatellite-based linkage maps and identification of size-related quantitative trait loci for Zhikong scallop (Chlamys farreri); Anim. Genet.;  citation_year=2009;  citation_volume=40;  citation_pages=821-31; " /><meta name="citation_reference" content="citation_title=Null alleles in microsatellite markers; Biodiversity Sci.;  citation_year=2013;  citation_volume=21;  citation_pages=117-26; " /><meta name="citation_reference" content="citation_title=Using population genomics to detect selection in natural populations, key concepts and methodological considerations; Int. J. Plant Sci.;  citation_year=2010;  citation_volume=171;  citation_pages=1059-71; " /><meta name="citation_reference" content="citation_publisher=Oxford University Press, Oxford; Calcium as a cellular regulator; " /><meta name="citation_reference" content="citation_title=Novel SNPs of the bovine NUCB2 gene and their association with growth traits in three native Chinese cattle breeds; Mol. Biol. Rep.;  citation_year=2009;  citation_volume=37;  citation_pages=541-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/2/183/1128002/dst048.pdf" /><meta name="description" content="Abstract.  Mollusca is the second most diverse group of animals in the world. Despite their perceived importance, omics-level studies have seldom been applied t" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/2/183/403825" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/2/183/403825" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/2/183/403825");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst048';
    </script>
<title>
	Large-Scale Development of Gene-Associated Single-Nucleotide Polymorphism Markers for Molluscan Population Genomic, Comparative Genomic, and Genome-Wide Association Studies | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./403825" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdNoPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKaWFvLCBXZW5xaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnUsIFhpYW90ZW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIEppbnFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE1hcmluZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgTWFyaW5lIExpZmUgU2NpZW5jZXMsIE9jZWFuIFVuaXZlcnNpdHkgb2YgQ2hpbmEsIDUgWXVzaGFuIFJvYWQsIFFpbmdkYW8gMjY2MDAzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBMaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRmVuZywgTGl5aW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTHYsIEppYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE1hcmluZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgTWFyaW5lIExpZmUgU2NpZW5jZXMsIE9jZWFuIFVuaXZlcnNpdHkgb2YgQ2hpbmEsIDUgWXVzaGFuIFJvYWQsIFFpbmdkYW8gMjY2MDAzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBMdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE1hcmluZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgTWFyaW5lIExpZmUgU2NpZW5jZXMsIE9jZWFuIFVuaXZlcnNpdHkgb2YgQ2hpbmEsIDUgWXVzaGFuIFJvYWQsIFFpbmdkYW8gMjY2MDAzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIFhpYW9qaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIFlhbmdwaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSG91LCBSdWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBNYXJpbmUgR2VuZXRpY3MgYW5kIEJyZWVkaW5nLCBDb2xsZWdlIG9mIE1hcmluZSBMaWZlIFNjaWVuY2VzLCBPY2VhbiBVbml2ZXJzaXR5IG9mIENoaW5hLCA1IFl1c2hhbiBSb2FkLCBRaW5nZGFvIDI2NjAwMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgTGluZ2xpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBNYXJpbmUgR2VuZXRpY3MgYW5kIEJyZWVkaW5nLCBDb2xsZWdlIG9mIE1hcmluZSBMaWZlIFNjaWVuY2VzLCBPY2VhbiBVbml2ZXJzaXR5IG9mIENoaW5hLCA1IFl1c2hhbiBSb2FkLCBRaW5nZGFvIDI2NjAwMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdSwgWGlhb2xpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgU2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmFvLCBaaGVubWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJMYXJnZS1TY2FsZSBEZXZlbG9wbWVudCBvZiBHZW5lLUFzc29jaWF0ZWQgU2luZ2xlLU51Y2xlb3RpZGUgUG9seW1vcnBoaXNtIE1hcmtlcnMgZm9yIE1vbGx1c2NhbiBQb3B1bGF0aW9uIEdlbm9taWMsIENvbXBhcmF0aXZlIEdlbm9taWMsIGFuZCBHZW5vbWUtV2lkZSBBc3NvY2lhdGlvbiBTdHVkaWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTgzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxOTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwNDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlbGF0aW9uc2hpcHMgb2YgaGlnaGVyIG1vbGx1c2NhbiB0YXhhOyBjaXRhdGlvbl9wdWJsaXNoZXI9VW5pdmVyc2l0eSBvZiBDYWxpZm9ybmlhIFByZXNzLCBCZXJrZWxleTsgUGh5bG9nZW55IGFuZCBFdm9sdXRpb24gb2YgdGhlIE1vbGx1c2NhOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fcGFnZXM9MTktMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGx1c2Nhbm1lbW9yeSBvZiBpbmp1cnk6IGV2b2x1dGlvbmFyeSBpbnNpZ2h0cyBpbnRvY2hyb25pYyBwYWluIGFuZCBuZXVyb2xvZ2ljYWwgZGlzb3JkZXJzOyBCcmFpbiBCZWhhdi4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz0yMDYtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgcGFzdCwgcHJlc2VudCwgYW5kIGZ1dHVyZSBvY2VhbiBjYXJib24gZGlveGlkZSBjb25jZW50cmF0aW9ucyBvbiB0aGUgZ3Jvd3RoIGFuZCBzdXJ2aXZhbCBvZiBsYXJ2YWwgc2hlbGxmaXNoOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0xNzI0Ni01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG95c3RlciBnZW5vbWUgcmV2ZWFscyBzdHJlc3MgYWRhcHRhdGlvbiBhbmQgY29tcGxleGl0eSBvZiBzaGVsbCBmb3JtYXRpb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00OTA7ICBjaXRhdGlvbl9wYWdlcz00OS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHJhZnQgZ2Vub21lIG9mIHRoZSBwZWFybCBveXN0ZXIgUGluY3RhZGEgZnVjYXRhOiBhIHBsYXRmb3JtIGZvciB1bmRlcnN0YW5kaW5nIGJpdmFsdmUgYmlvbG9neTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMTctMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9ucyBvZiBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyBpbiBtb2xlY3VsYXIgZWNvbG9neSBvZiBub24tbW9kZWwgb3JnYW5pc21zOyBIZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0xLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIFNvdXRoIEFmcmljYW4gQWJhbG9uZSBIYWxpb3RpcyBtaWRhZSB1c2luZyBzZXF1ZW5jaW5nLWJ5LXN5bnRoZXNpczsgQk1DIFJlcy4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIGFuZCBkZSBub3ZvIGFuYWx5c2lzIGZvciBZZXNzbyBzY2FsbG9wIChQYXRpbm9wZWN0ZW4geWVzc29lbnNpcykgdXNpbmcgNDU0IEdTIEZseDsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyMTU2MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlc29sdmluZyB0aGUgZXZvbHV0aW9uYXJ5IHJlbGF0aW9uc2hpcCBvZiBtb2xsdXNjcyB3aXRoIHBoeWxvZ2Vub21pYyB0b29sczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ4MDsgIGNpdGF0aW9uX3BhZ2VzPTM2NC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIHNlcXVlbmNpbmcgYW5kIHRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2YgdGhlIGNlbnRyYWwgbmVydm91cyBzeXN0ZW0gb2YgbW9sbHVzayBMeW1uYWVhIHN0YWduYWxpcyBieSBkZWVwIFJBTiBzZXF1ZW5jaW5nOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQyNTQ2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBhbmFseXNpcyBvZiB0aGUgT2N0b3B1cyB2dWxnYXJpcyBjZW50cmFsIG5lcnZvdXMgc3lzdGVtOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQwMzIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIEROQSBtZWx0aW5nIGFuYWx5c2lzIGZvciBzaW1wbGUgYW5kIGVmZmljaWVudCBtb2xlY3VsYXIgZGlhZ25vc3RpY3M7IFBoYXJtYWNvZ2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTU5Ny02MDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGF0aW9uIHNjYW5uaW5nIHVzaW5nIGhpZ2gtcmVzb2x1dGlvbiBtZWx0aW5nOyBCaW9jaGVtLiBTb2MuIFRyYW5zLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTQzMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBoaWdoLXJlc29sdXRpb24gZ2VuZXRpYyBsaW5rYWdlIG1hcCBhbmQgY29tcGFyYXRpdmUgZ2Vub21lIGFuYWx5c2lzIGZvciB0aGUgcmVlZi1idWlsZGluZyBjb3JhbCBBY3JvcG9yYSBtaWxsZXBvcmE7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVIxMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBlZmZpY2llbnQgbWV0aG9kIGZvciBkZXZlbG9waW5nIFNOUCBtYXJrZXJzIGJhc2VkIG9uIEVTVCBkYXRhIGNvbWJpbmVkIHdpdGggaGlnaCByZXNvbHV0aW9uIG1lbHRpbmcgKEhSTSkgYW5hbHlzaXM7IEJNQyBSZXMuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz01MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyBvZiBaaGlrb25nIHNjYWxsb3AgKENobGFteXMgZmFycmVyaSkgYW5kIGNvbXBhcmF0aXZlIHRyYW5zY3JpcHRvbWljIGFuYWx5c2lzIHdpdGggWWVzc28gc2NhbGxvcCAoUGF0aW5vcGVjdGVuIHllc3NvZW5zaXMpOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTYzOTI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgNDQgZ2VuZS1iYXNlZCBTTlAgbWFya2VycyBpbiBaaGlrb25nIHNjYWxsb3AsIENobGFteXMgZmFycmVyaTsgQ29uc2Vydi4gR2VuZXQuIFJlc291ci47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTY1OS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiAzOCBFU1QtZGVyaXZlZCBTTlAgbWFya2VycyBpbiBaaGlrb25nIHNjYWxsb3AgKENobGFteXMgZmFycmVyaSkgYW5kIHRoZWlyIGNyb3NzLXNwZWNpZXMgdXRpbGl0eSBpbiBZZXNzbyBzY2FsbG9wIChQYXRpbm9wZWN0ZW4geWVzc29lbnNpcyk7IENvbnNlcnYuIEdlbmV0LiBSZXNvdXIuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz03NDctNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNBUDMsIGEgRE5BIHNlcXVlbmNlIGFzc2VtYmx5IHByb2dyYW07IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz04NjgtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YWxpdHlTTlAsIGEgcGlwZWxpbmUgZm9yIGRldGVjdGluZyBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGFuZCBpbnNlcnRpb25zL2RlbGV0aW9ucyBpbiBFU1QgZGF0YSBmcm9tIGRpcGxvaWQgYW5kIHBvbHlwbG9pZHkgc3BlY2llczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5lLXNjYWxlIHBvcHVsYXRpb24gZ2VuZXRpYyBzdHJ1Y3R1cmUgb2YgWmhpa29uZyBzY2FsbG9wIChDaGxhbXlzIGZhcnJlcmkpLCBkbyBsb2NhbCBtYXJpbmUgY3VycmVudHMgZHJpdmUgZ2VvZ3JhcGhpY2FsIGRpZmZlcmVudGlhdGlvbj87IE1hci4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yMjMtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gZ2VuZXRpYyBhbmFseXNpcyBvZiBjby1kb21pbmFudCBhbmQgZG9taW5hbnQgbWFya2VycyBhbmQgcXVhbnRpdGF0aXZlIHRyYWl0czsgQmVsZy4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz0xNTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HRU5FUE9QIChWZXJzaW9uIDEuMiksIHBvcHVsYXRpb24gZ2VuZXRpY3Mgc29mdHdhcmUgZm9yIGV4YWN0IHRlc3RzIGFuZCBlY3VtZW5pY2lzbTsgSi4gSGVyZWQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTg2OyAgY2l0YXRpb25fcGFnZXM9MjQ4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJsYXN0MkdPLCBBIHVuaXZlcnNhbCB0b29sIGZvciBhbm5vdGF0aW9uLCB2aXN1YWxpemF0aW9uIGFuZCBhbmFseXNpcyBpbiBmdW5jdGlvbmFsIGdlbm9taWNzIHJlc2VhcmNoOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM2NzQtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S0FBUywgYW4gYXV0b21hdGljIGdlbm9tZSBhbm5vdGF0aW9uIGFuZCBwYXRod2F5IHJlY29uc3RydWN0aW9uIHNlcnZlcjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9VzE4Mi0xODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gZ2Vub21pY3MsIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaG9sZHMgdGhlIGtleTsgQ3Vyci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1SNTc2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBwaHlsb2dlbmV0aWNzIG9mIHRoZSBQZWN0aW5pZGFlIChNb2xsdXNjYSwgQml2YWx2aWEpIGFuZCBlZmZlY3Qgb2YgaW5jcmVhc2VkIHRheG9uIHNhbXBsaW5nIGFuZCBvdXRncm91cCBzZWxlY3Rpb24gb24gdHJlZSB0b3BvbG9neTsgTW9sLiBQaHlsb2dlbmV0LiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTExNzgtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIGluIG1pY3JvZmFicmljYXRlZCBoaWdoLWRlbnNpdHkgcGljb2xpdHJlIHJlYWN0b3JzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM3OyAgY2l0YXRpb25fcGFnZXM9Mzc2LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmFjeSBhbmQgcXVhbmxpdHkgb2YgbWFzc2l2ZWx5IHBhcmFsbGVsIEROQSBweXJvc2VxdWVuY2luZzsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1SMTQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgNDU0IHNlcXVlbmNpbmcgZXJyb3IgcmF0ZSwgZXJyb3Igc291cmNlcywgYW5kIGFydGlmYWN0IHJlY29tYmluYXRpb24gZm9yIGRldGVjdGlvbiBvZiBsb3ctZnJlcXVlbmN5IGRydWcgcmVzaXN0YW5jZSBtdXRhdGlvbnMgaW4gSElWLTEgRE5BOyBSZXRyb3Zpcm9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0LWdlbmVyYXRpb24gdHJhbnNjcmlwdG9tZSBhc3NlbWJseTsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY3MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBhbmQgdGhlaXIgcmVsYXRpb25zaGlwIHRvIGNvZG9uIHVzYWdlIGJpYXMgaW4gdGhlIFBhY2lmaWMgb3lzdGVyIENyYXNzb3N0cmVhIGdpZ2FzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQwNjsgIGNpdGF0aW9uX3BhZ2VzPTEzLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgbWljcm9zYXRlbGxpdGUtYmFzZWQgbGlua2FnZSBtYXBzIGFuZCBpZGVudGlmaWNhdGlvbiBvZiBzaXplLXJlbGF0ZWQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIFpoaWtvbmcgc2NhbGxvcCAoQ2hsYW15cyBmYXJyZXJpKTsgQW5pbS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ODIxLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWxsIGFsbGVsZXMgaW4gbWljcm9zYXRlbGxpdGUgbWFya2VyczsgQmlvZGl2ZXJzaXR5IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMTctMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIHBvcHVsYXRpb24gZ2Vub21pY3MgdG8gZGV0ZWN0IHNlbGVjdGlvbiBpbiBuYXR1cmFsIHBvcHVsYXRpb25zLCBrZXkgY29uY2VwdHMgYW5kIG1ldGhvZG9sb2dpY2FsIGNvbnNpZGVyYXRpb25zOyBJbnQuIEouIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTcxOyAgY2l0YXRpb25fcGFnZXM9MTA1OS03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPU94Zm9yZCBVbml2ZXJzaXR5IFByZXNzLCBPeGZvcmQ7IENhbGNpdW0gYXMgYSBjZWxsdWxhciByZWd1bGF0b3I7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIFNOUHMgb2YgdGhlIGJvdmluZSBOVUNCMiBnZW5lIGFuZCB0aGVpciBhc3NvY2lhdGlvbiB3aXRoIGdyb3d0aCB0cmFpdHMgaW4gdGhyZWUgbmF0aXZlIENoaW5lc2UgY2F0dGxlIGJyZWVkczsgTW9sLiBCaW9sLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9NTQxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS8yLzE4My8xMTI4MDAyL2RzdDA0OC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTW9sbHVzY2EgaXMgdGhlIHNlY29uZCBtb3N0IGRpdmVyc2UgZ3JvdXAgb2YgYW5pbWFscyBpbiB0aGUgd29ybGQuIERlc3BpdGUgdGhlaXIgcGVyY2VpdmVkIGltcG9ydGFuY2UsIG9taWNzLWxldmVsIHN0dWRpZXMgaGF2ZSBzZWxkb20gYmVlbiBhcHBsaWVkIHQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQwMzgyNSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRp0/TnDRfIL5BpXBQxwfT0VbE03Q==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f183%2f403825" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f183%2f403825" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15712" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&amp;Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/2">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5966900">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5966900">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5966905">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5966905">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5966917">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5966917">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5966939">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5966939">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5966984">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5966984">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5966998">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5966998">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5967000">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5967000">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5967002">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5967002">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5967004">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5967004">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/2/169/404005">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/2/195/404182">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Large-Scale Development of Gene-Associated Single-Nucleotide Polymorphism Markers for Molluscan Population Genomic, Comparative Genomic, and Genome-Wide Association Studies
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wenqian Jiao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wenqian+Jiao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiao W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jiao W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wenqian Jiao</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoteng Fu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaoteng+Fu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fu X&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Xiaoteng Fu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jinqin Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jinqin+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jinqin Li</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ling Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ling+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ling Li</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Liying Feng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Liying+Feng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Feng L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Feng L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Liying Feng</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jia Lv</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jia+Lv">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lv J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lv J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jia Lv</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lu Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lu+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Lu Zhang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaojian Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaojian+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang X&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Xiaojian Wang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yangping Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yangping+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yangping Li</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rui Hou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rui+Hou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hou R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hou R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Rui Hou</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lingling Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lingling+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang L&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Lingling Zhang</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoli Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaoli+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hu X&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Xiaoli Hu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shi Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 532-82031969. E-mail: <a href="mailto:swang@ouc.edu.cn" target="_blank">swang@ouc.edu.cn</a> (S.W.); Tel. +86 532-82031960. E-mail: <a href="mailto:zmbao@ouc.edu.cn" target="_blank">zmbao@ouc.edu.cn</a> (Z.B.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shi+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shi Wang</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhenmin Bao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Marine Genetics and Breeding</div>, <div class="insititution">College of Marine Life Sciences, Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 532-82031969. E-mail: <a href="mailto:swang@ouc.edu.cn" target="_blank">swang@ouc.edu.cn</a> (S.W.); Tel. +86 532-82031960. E-mail: <a href="mailto:zmbao@ouc.edu.cn" target="_blank">zmbao@ouc.edu.cn</a> (Z.B.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhenmin+Bao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bao Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bao Z&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Zhenmin Bao</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 183–193, <a href='https://doi.org/10.1093/dnares/dst048'>https://doi.org/10.1093/dnares/dst048</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 November 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 April 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 October 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="403825" href="/dnaresearch/article-pdf/21/2/183/1128002/dst048.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Wenqian Jiao, Xiaoteng Fu, Jinqin Li, Ling Li, Liying Feng, Jia Lv, Lu Zhang, Xiaojian Wang, Yangping Li, Rui Hou, Lingling Zhang, Xiaoli Hu, Shi Wang, Zhenmin Bao;  Large-Scale Development of Gene-Associated Single-Nucleotide Polymorphism Markers for Molluscan Population Genomic, Comparative Genomic, and Genome-Wide Association Studies, <em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 183–193, <a href='https://doi.org/10.1093/dnares/dst048'>https://doi.org/10.1093/dnares/dst048</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/403825?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/403825?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/403825?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/403825?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/403825?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/403825?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/403825?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Large-Scale%20Development%20of%20Gene-Associated%20Single-Nucleotide%20Polymorphism%20Markers%20for%20Molluscan%20Population%20Genomic%2C%20Comparative%20Genomic%2C%20and%20Genome-Wide%20Association%20Studies&amp;publicationDate=2013-11-25&amp;volumeNum=21&amp;issueNum=2&amp;author=Jiao%2C%20Wenqian%3B%20Fu%2C%20Xiaoteng&amp;startPage=183&amp;endPage=193&amp;contentId=10.1093%2Fdnares%2Fdst048&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f183%2f403825" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5966900 id="5966900" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Mollusca is the second most diverse group of animals in the world. Despite their perceived importance, omics-level studies have seldom been applied to this group of animals largely due to a paucity of genomic resources. Here, we report the first large-scale gene-associated marker development and evaluation for a bivalve mollusc, <em>Chlamys farreri</em>. More than 21,000 putative single-nucleotide polymorphisms (SNPs) were identified from the <em>C. farreri</em> transcriptome. Primers and probes were designed and synthesized for 4500 SNPs, and 1492 polymorphic markers were successfully developed using a high-resolution melting genotyping platform. These markers are particularly suitable for population genomic analysis due to high polymorphism within and across populations, a low frequency of null alleles, and conformation to neutral expectations. Unexpectedly, high cross-species transferability was observed, suggesting that the transferable SNPs may largely represent ancestral genetic variations that have been preserved differentially among subfamilies of Pectinidae. Gene annotations were available for 73% of the markers, and 65% could be anchored to the recently released Pacific oyster genome. Large-scale association analysis revealed key candidate genes responsible for scallop growth regulation, and provided markers for further genetic improvement of <em>C. farreri</em> in breeding programmes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="mollusca">mollusca</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single-nucleotide polymorphism (SNP)&quot;">single-nucleotide polymorphism (SNP)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high resolution melting (HRM)&quot;">high resolution melting (HRM)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome-wide association (GWAS)&quot;">genome-wide association (GWAS)</a></div>                            <h2 scrollto-destination=5966905 id="5966905" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Mollusca is the second most speciose animal phylum, containing more than 1 00 000 extant species distributed across eight major lineages.<sup><a reveal-id="DST048C1" data-open="DST048C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Molluscs play vital roles in the structure and functioning of aquatic and terrestrial ecosystems. Many molluscs are important fishery and aquaculture species, and some also serve as models for studying neurobiology, biomineralization, and adaptive evolution in response to climate change.<sup><a reveal-id="DST048C2" data-open="DST048C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST048C3" data-open="DST048C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Despite their perceived importance, systematic studies of molluscan biology and evolution remain very limited largely due to a paucity of genomic resources.</p><p>Recent advances in next-generation sequencing (NGS) technologies (e.g. Roche's 454 and Illumina's Solexa) now allow for rapid generation of extensive genomic resources at affordable cost for any organism, thus opening up opportunities for conducting omics-level analyses in molluscs. NGS-based genome sequencing has recently been performed for two bivalve molluscs, i.e. Pacific oyster (<em>Crassostrea gigas</em>)<sup><a reveal-id="DST048C4" data-open="DST048C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and pearl oyster (<em>Pinctada fucata</em>),<sup><a reveal-id="DST048C5" data-open="DST048C5" class="link link-ref link-reveal xref-bibr">5</a></sup> providing the first insights into molluscan genome architecture and the genetic basis of stress adaptation, shell formation, and pearl biosynthesis. However, whole-genome <em>de novo</em> sequencing is currently still costly even using NGS platforms and remains out of reach for most laboratories focusing on non-model organisms. As an attractive alternative, transcriptome sequencing represents a cost-effective approach to rapidly expand gene resources, and it has been widely applied in many non-model organisms.<sup><a reveal-id="DST048C6" data-open="DST048C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The extensive gene resources generated by transcriptome sequencing are very useful not only for transcriptome-wide gene characterization and expression profiling, but also for large-scale single-nucleotide polymorphism (SNP) discovery. Such ‘functional’ SNPs are particularly valuable for quantitative genetic and evolutionary studies, because they have a great potential for quickly identifying causal genes responsible for either complex traits or adaptive evolution.</p><p>Genome-wide scans for detecting quantitative trait loci (QTL) for traits of interest or genetic determinants for local adaptation usually require a large number of genetic markers. SNPs have attracted significant attention, as they represent the most abundant class of genetic variations in eukaryotic genomes and have already become the marker of choice in large-scale genotyping applications, such as high-resolution linkage and association mapping, genomic selection, and comparative genome analysis. However, SNP markers have been insufficiently developed for molluscs in comparison with well-studied model organisms, such as mouse, nematode worm, and zebrafish. Even for the molluscs that are well characterized at the molecular level (e.g. Pacific oyster), only hundreds of markers are available. Transcriptome sequencing has recently been conducted in many molluscs,<sup><a reveal-id="DST048C7  DST048C11" data-open="DST048C7  DST048C11" class="link link-ref link-reveal xref-bibr">7–11</a></sup> providing extensive resources for large-scale gene-associated SNP mining. High-throughput SNP screening and marker development rely on an efficient genotyping platform. High resolution melting (HRM) has proved to be an extremely powerful technique for rapid profiling of genetic variation within PCR amplicons.<sup><a reveal-id="DST048C12" data-open="DST048C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST048C13" data-open="DST048C13" class="link link-ref link-reveal xref-bibr">13</a></sup> HRM has several advantages over other genotyping methods, such as its simplicity, low cost, high sensitivity, and specificity, and has been widely applied to many non-model species for marker development at medium-to-large scale.<sup><a reveal-id="DST048C14" data-open="DST048C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST048C15" data-open="DST048C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>Our group has recently released a large amount of transcriptome data via 454 sequencing for a bivalve mollusc, <em>Chlamys farreri</em> (Jones et Preston 1904).<sup><a reveal-id="DST048C16" data-open="DST048C16" class="link link-ref link-reveal xref-bibr">16</a></sup><em>Chlamys farreri</em> is naturally distributed along the seacoasts of China, Japan, and Korea and is also an important aquaculture species in China. To date, &lt;100 SNP markers have been developed for this species,<sup><a reveal-id="DST048C17" data-open="DST048C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST048C18" data-open="DST048C18" class="link link-ref link-reveal xref-bibr">18</a></sup> which is not sufficient for large-scale genomic analyses, such as high-resolution linkage and QTL mapping, association mapping, and comparative genome analysis. Here, we conducted the first large-scale gene-associated SNP marker development for <em>C. farreri</em>. The 1492 SNP markers developed in this study were further evaluated for their usefulness in population genomic, comparative genomic, and association analyses.</p>                            <h2 scrollto-destination=5966917 id="5966917" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=5966919 id="5966919" class="section-title" data-legacy-id=s2a>2.1. Transcriptome sequences, assembly, and SNP mining</h3>
<p>The <em>C. farreri</em> transcriptome sequences (SRA accession no. SRA030509) were produced by 454 sequencing of cDNA libraries prepared from diverse developmental stages and adult tissues. The details of sample collection, library preparation, and 454 sequencing have been described in a previous study.<sup><a reveal-id="DST048C16" data-open="DST048C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Briefly, for larval samples, approximately 1000 parents were used for artificial fertilization, while adult tissues were collected from 30 individuals. To reduce the risk of identifying artificial SNPs arising from sequencing errors or misassembly of paralogous sequences, 454 reads were reassembled using the CAP3 programme<sup><a reveal-id="DST048C19" data-open="DST048C19" class="link link-ref link-reveal xref-bibr">19</a></sup> under very stringent assembly criteria (overlap setting: 100 bp and 95% similarity). Using the QualitySNP programme,<sup><a reveal-id="DST048C20" data-open="DST048C20" class="link link-ref link-reveal xref-bibr">20</a></sup> putative SNPs were identified in the assembled contigs that were covered by at least four reads and had at least two reads for each allele.</p>                            <h3 scrollto-destination=5966923 id="5966923" class="section-title" data-legacy-id=s2b>2.2. SNP marker development based on the HRM genotyping platform</h3>
<p>SNP markers were developed using a cost-effective HRM method<sup><a reveal-id="DST048C14" data-open="DST048C14" class="link link-ref link-reveal xref-bibr">14</a></sup> that used two PCR primers and one unlabelled probe. SNP markers were named as follows: C followed by a contig ID, and then S followed by the SNP position (bp) within the contig. HRM primers and probes were designed following the rules described by Wang <em>et al.</em><sup><a reveal-id="DST048C14" data-open="DST048C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Primers and probes were synthesized (Sangon Biotech) and evaluated using six <em>C. farreri</em> individuals. PCR amplifications were performed in a 10-µl volume composed of 20 ng genomic DNA, 0.1 µM forward primer, 0.5 µM reverse primer, 1.5 mM MgCl<sub>2</sub>, 0.2 mM dNTPs (Invitrogen), 1× LCGreen Plus (Idaho Technology), 1× PCR buffer, and 0.5 U Taq DNA polymerase (TaKaRa). Thermal cycling began with an initial denaturing step at 95°C for 5 min, followed by 60 cycles of 95°C for 40 s, 63°C for 40 s, and 72°C for 40 s with a final extension at 72°C for 5 min. The corresponding probe was added to each PCR to a final concentration of 3 µM, and the reaction mixture was denatured at 95°C for 10 min and then slowly cooled to 4°C. HRM genotyping was performed on a Light-Scanner instrument (Idaho Technology) with continuous melting curve acquisition (10 acquisitions per °C) during a 0.1°C/s ramp from 40 to 95°C. All primer and probe sequences of the developed SNP markers are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=5966926 id="5966926" class="section-title" data-legacy-id=s2c>2.3. Population genetic analysis</h3>
<p>The developed SNP markers were evaluated using four wild, geographical populations. In total, 54 <em>C. farreri</em> individuals were used for this evaluation, of which 24 were collected from the Jiaonan (JN) population, 12 from the Changdao (CD) population, 12 from the Rongcheng (RC) population, and 6 from the Dalian (DL) population. The collection details of these samples were described in a previous study.<sup><a reveal-id="DST048C21" data-open="DST048C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Marker polymorphisms were evaluated within and among populations. For each marker, allele frequency, observed heterozygosity (<em>H</em><sub>o</sub>), and expected heterozygosity (<em>H</em><sub>e</sub>), along with tests for neutrality, Hardy–Weinberg equilibrium, and linkage disequilibrium, were calculated using POPGENE<sup><a reveal-id="DST048C22" data-open="DST048C22" class="link link-ref link-reveal xref-bibr">22</a></sup> or GENEPOP 4.0 program.<sup><a reveal-id="DST048C23" data-open="DST048C23" class="link link-ref link-reveal xref-bibr">23</a></sup> For Hardy–Weinberg equilibrium and linkage disequilibrium tests, Bonferroni correction was also applied to account for multiple testing.</p>                            <h3 scrollto-destination=5966931 id="5966931" class="section-title" data-legacy-id=s2d>2.4. Test for cross-species transferability</h3>
<p>In total, 34 markers targeting 24 synonymous and 10 non-synonymous SNPs were selected for evaluating cross-species transferability in five scallop species from three Pectinidae subfamilies, i.e. Chlamydinae (<em>Chlamys nobilis</em> and <em>Patinopecten yessoensis</em>), Pectininae (<em>Amusium pleuronectes</em>), and Aequipectini (<em>Argopecten irradians</em> and <em>Argopecten purpuratus</em>). All species except <em>A. purpuratus</em> were purchased from local markets in China. <em>Argopecten purpuratus</em> samples were kindly provided by Dr Chunde Wang (Qingdao Agriculture University, China). For each species, eight individuals were used to evaluate polymorphisms.</p>                            <h3 scrollto-destination=5966934 id="5966934" class="section-title" data-legacy-id=s2e>2.5. Marker annotation and gene comparison with Pacific oyster</h3>
<p>To provide functional annotations for the SNP markers, relevant contig sequences were compared against the Swiss-Prot and Nr protein databases using BlastX, with an <em>e</em>-value threshold of 1<em>e</em>−5. Gene names were assigned to each marker based on the best hit. The BlastX results were imported into the Blast2GO software<sup><a reveal-id="DST048C24" data-open="DST048C24" class="link link-ref link-reveal xref-bibr">24</a></sup> for gene ontology (GO) analysis. GO terms were assigned to query sequences based on three ontology classifications, i.e. biological process, molecular function, and cellular component. To gain an overview of gene pathways, KEGG analysis was also performed using the KEGG Automatic Annotation Server.<sup><a reveal-id="DST048C25" data-open="DST048C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The bi-directional best hit method was used to obtain KEGG orthology assignments. The contig sequences containing the SNP markers were also compared against the oyster protein database using BlastX, with an <em>e</em>-value threshold of 1<em>e</em>−4.</p>                            <h3 scrollto-destination=5966936 id="5966936" class="section-title" data-legacy-id=s2f>2.6. Large-scale marker associations with growth traits</h3>
<p>Large-scale marker associations with growth traits were conducted on an elite variety of <em>C. farreri</em> named the ‘Penglai-Red’ scallop (Aquacultural Variety Registration Number of the Ministry of Agriculture of China: GS02-001-2005), which was developed by our group and has been under continuous artificial selection for red shell colour and fast growth for multiple generations. A large breeding population was established in May 2010 by artificial fertilization of more than 3000 sexually mature ‘Penglai-Red’ scallops at the hatchery of the Xunshan Aquatic Group Corporation (Shandong Province, China). After rearing for 2 months, half the juvenile scallops were moved to another hatchery that is ∼250 km from the original hatchery. In May 2012, approximately 1000 2-year-old scallops were randomly sampled from each hatchery, constituting two target populations for the association analyses in this study. To reduce the genotyping cost, a selective genotyping strategy was adopted. For each population, 40 individuals with large body sizes and 40 ones with small body sizes were chosen for genotyping with the 1492 SNP markers. For an initial scan of the whole marker set, a DNA pool was established for each group by mixing equal amount of genomic DNA prepared from individuals within each group. The resultant DNA pools were subject to HRM analysis to search for allele frequency differences between groups. Markers showing large between-group differences in allele frequencies were subject to further validation by genotyping all individuals in both groups; statistical significance was determined using a <em>χ</em><sup>2</sup> test.</p>                            <h2 scrollto-destination=5966939 id="5966939" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5966942 id="5966942" class="section-title" data-legacy-id=s3a>3.1. Transcriptome assembly and SNP mining</h3>
<p>In total, 1 099 254 clean reads were used for transcriptome assembly. The final assembly consisted of 50 741 contigs and 406 825 singletons. Contig sizes ranged from 60 to 6063 bp, with an average of 541 bp. The average read and sequencing depth coverage across all contigs were 13.6 and 7.5, respectively. The relatively low sequencing depth limits our ability to detect less common or rare SNPs; nevertheless, this transcriptomic resource enabled us to identify a large number of candidate SNPs for marker development. A total of 21 813 putative SNPs were identified from 18 780 contigs. The distribution of SNP coverage is shown in Fig. <a reveal-id="DST048F1" data-open="DST048F1" class="link link-reveal link-table xref-fig">1</a>. The overall SNP frequency was one SNP per 1258 bp. Of these SNPs, 14 641(67%) were transitions, whereas 7172 (33%) were transversions. A total of 7338 SNPs were identified from contigs for which gene annotation information was available. </p>                        <a id="5966944" scrollto-destination="5966944"></a>
<div id="DST048F1" data-id="DST048F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04801.png?Expires=2147483647&Signature=IKk07dkUkkrJ1P4olE1fp90M97C0iSy6AEdTYiS2Zv8vumZ3jQZ29AAygCEyXj1-j7-QracP8GwQYgxG9dmC5pnCTkoSlswJG6U0F~dw3BlPpMMgbMC3XEs5JWEyIeoCbFFvGZIP9RDeNnNQxvYPurpsS~OjStcT~RFUmJTHbyFONQOdxporW6QwL9AP9~agXsqaiZ9v0ZpoHj-nHftjdMsyVUJR394grMwLy-pet63CHxMDM5CJGuPprwBKcmg9VItQ1uFtEPfju9gz3-6t09YwTMLKbZyWo91EVKEBjLHXM0xqoETTuBr3UKBoHj3PLlTN2CuH1~xIOdjYLLeIFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of sequencing coverage for the SNPs detected in the C. farreri transcriptome." path-from-xml="dst04801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966944/dst04801.tiff" path-from-xml="dst04801.tiff" target="_blank">View large</a><a section="5966944" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04801.png?Expires=2147483647&Signature=2SxKsEWnV4DBL3jbxUp36xB4dMN~7ym~79a-YeKoFFb1RKxgcOJp4jI7k9eK5HKxZmVSiH1fQWaSsgzbHRp0yCkEV7tdo8IIxzEMO9NEo~yyP0WpQeXBBYa8lhMnhK78513UtNPx826MZ15hX4oDq-8IDA1mxc0Xzpw~fJ~MP-2ey6bacDaZCFBk8AGTcPGHKF-QsYgMv8maD1Ke-g8XQpZ6m04iTVeds~7xvG0O8bNxiCBdeOoy~sssvdD3OXQRG6VrlG5kjROh5jcZdiYyrvqCKG3BV3d-niLUKv-pqsau7R4GRA6dmkVvIVgwtCwu0IjjTYmDBKZ0F9GARfPZtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966944&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of sequencing coverage for the SNPs detected in the <em>C. farreri</em> transcriptome.</p></div></div><div content-id="DST048F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04801.png?Expires=2147483647&Signature=IKk07dkUkkrJ1P4olE1fp90M97C0iSy6AEdTYiS2Zv8vumZ3jQZ29AAygCEyXj1-j7-QracP8GwQYgxG9dmC5pnCTkoSlswJG6U0F~dw3BlPpMMgbMC3XEs5JWEyIeoCbFFvGZIP9RDeNnNQxvYPurpsS~OjStcT~RFUmJTHbyFONQOdxporW6QwL9AP9~agXsqaiZ9v0ZpoHj-nHftjdMsyVUJR394grMwLy-pet63CHxMDM5CJGuPprwBKcmg9VItQ1uFtEPfju9gz3-6t09YwTMLKbZyWo91EVKEBjLHXM0xqoETTuBr3UKBoHj3PLlTN2CuH1~xIOdjYLLeIFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of sequencing coverage for the SNPs detected in the C. farreri transcriptome." path-from-xml="dst04801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966944/dst04801.tiff" path-from-xml="dst04801.tiff" target="_blank">View large</a><a section="5966944" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04801.png?Expires=2147483647&Signature=2SxKsEWnV4DBL3jbxUp36xB4dMN~7ym~79a-YeKoFFb1RKxgcOJp4jI7k9eK5HKxZmVSiH1fQWaSsgzbHRp0yCkEV7tdo8IIxzEMO9NEo~yyP0WpQeXBBYa8lhMnhK78513UtNPx826MZ15hX4oDq-8IDA1mxc0Xzpw~fJ~MP-2ey6bacDaZCFBk8AGTcPGHKF-QsYgMv8maD1Ke-g8XQpZ6m04iTVeds~7xvG0O8bNxiCBdeOoy~sssvdD3OXQRG6VrlG5kjROh5jcZdiYyrvqCKG3BV3d-niLUKv-pqsau7R4GRA6dmkVvIVgwtCwu0IjjTYmDBKZ0F9GARfPZtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966944&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of sequencing coverage for the SNPs detected in the <em>C. farreri</em> transcriptome.</p></div></div>                            <h3 scrollto-destination=5966947 id="5966947" class="section-title" data-legacy-id=s3b>3.2. Marker development based on the HRM genotyping platform</h3>
<p>The entire marker development and evaluation procedure are depicted in Fig. <a reveal-id="DST048F2" data-open="DST048F2" class="link link-reveal link-table xref-fig">2</a>. All putative SNPs were evaluated for HRM primer and probe design, which led to a successful set of primers and probes for 4500 SNPs with a wide range of minor allele frequencies (MAFs; Table <a reveal-id="DST048TB1" data-open="DST048TB1" class="link link-reveal link-table xref-fig">1</a>). All primers and probes were synthesized and evaluated using six <em>C. farreri</em> individuals. A total of 3042 primer pairs produced strong single bands with expected sizes. Although the expected amplicon length was restricted to ∼100 bp during primer design, 616 primer pairs still produced bands larger than the expected sizes, indicating potential introns in the vicinity of those SNPs. Longer amplicons can greatly diminish the sensitivity of HRM analysis, so these SNPs were excluded from further consideration. Two hundred and forty-seven primer pairs produced more than one band possibly due to non-specific amplifications. The remaining primer pairs resulted in poor amplifications, i.e. very weak or no PCR product, and were excluded from further consideration. Probe testing was subsequently performed for primer pairs with successful amplifications. In total, 2231 probes generated well-recognizable melting curve profiles with a distinct peak for each allele. Of these, 1492 loci were polymorphic across all assayed individuals, whereas 739 loci were monomorphic possibly due to the small number of assayed individuals. The SNP validation rate was correlated with the MAF in the initial discovery panel. A higher MAF tended to lead to a higher validation rate (Table <a reveal-id="DST048TB1" data-open="DST048TB1" class="link link-reveal link-table xref-fig">1</a>). For example, at the MAF interval of 0.4–0.5, 75% of markers were polymorphic. In contrast, only 42% were polymorphic at the MAF interval of 0–0.1, and the low validation rate may be largely attribute to the relatively low quality of predicted SNPs (i.e. SNPs that were difficult to distinguish from sequencing errors). Information pertaining to all 1492 SNP loci has been submitted to the dbSNP database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/snp/">http://www.ncbi.nlm.nih.gov/snp/</a>) under accession numbers rs831881546–rs831883035 and rs831883061–rs831883062. </p>                        <a id="5966950" scrollto-destination="5966950"></a>
<div content-id="DST048TB1" class="table-modal"><div class="table-wrap"><div id="DST048TB1" class="table-wrap-title" data-id="DST048TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Overview of the efficiency of HRM-based SNP marker development in <em>C. farreri</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Minor allele frequency ranges </th><th>No. of primers designed </th><th>No. of primer validated (% of designed) </th><th>No. of probe validated (% of designed) </th><th>No. of polymorphic loci (% of designed) </th></tr></thead><tbody><tr><td>0.4–0.5 </td><td>1538 </td><td>1056 (69%) </td><td>761 (49%) </td><td>571 (37%) </td></tr><tr><td>0.3–0.4 </td><td>1422 </td><td>954 (67%) </td><td>701 (49%) </td><td>506 (36%) </td></tr><tr><td>0.2–0.3 </td><td>832 </td><td>559 (67%) </td><td>422 (51%) </td><td>266 (32%) </td></tr><tr><td>0.1–0.2 </td><td>636 </td><td>418 (66%) </td><td>309 (49%) </td><td>133 (21%) </td></tr><tr><td>0–0.1 </td><td>72 </td><td>55 (76%) </td><td>38 (53%) </td><td>16 (22%) </td></tr></tbody></table></div><div content-id="DST048TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Minor allele frequency ranges </th><th>No. of primers designed </th><th>No. of primer validated (% of designed) </th><th>No. of probe validated (% of designed) </th><th>No. of polymorphic loci (% of designed) </th></tr></thead><tbody><tr><td>0.4–0.5 </td><td>1538 </td><td>1056 (69%) </td><td>761 (49%) </td><td>571 (37%) </td></tr><tr><td>0.3–0.4 </td><td>1422 </td><td>954 (67%) </td><td>701 (49%) </td><td>506 (36%) </td></tr><tr><td>0.2–0.3 </td><td>832 </td><td>559 (67%) </td><td>422 (51%) </td><td>266 (32%) </td></tr><tr><td>0.1–0.2 </td><td>636 </td><td>418 (66%) </td><td>309 (49%) </td><td>133 (21%) </td></tr><tr><td>0–0.1 </td><td>72 </td><td>55 (76%) </td><td>38 (53%) </td><td>16 (22%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5966950" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST048TB1" class="table-wrap-title" data-id="DST048TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Overview of the efficiency of HRM-based SNP marker development in <em>C. farreri</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Minor allele frequency ranges </th><th>No. of primers designed </th><th>No. of primer validated (% of designed) </th><th>No. of probe validated (% of designed) </th><th>No. of polymorphic loci (% of designed) </th></tr></thead><tbody><tr><td>0.4–0.5 </td><td>1538 </td><td>1056 (69%) </td><td>761 (49%) </td><td>571 (37%) </td></tr><tr><td>0.3–0.4 </td><td>1422 </td><td>954 (67%) </td><td>701 (49%) </td><td>506 (36%) </td></tr><tr><td>0.2–0.3 </td><td>832 </td><td>559 (67%) </td><td>422 (51%) </td><td>266 (32%) </td></tr><tr><td>0.1–0.2 </td><td>636 </td><td>418 (66%) </td><td>309 (49%) </td><td>133 (21%) </td></tr><tr><td>0–0.1 </td><td>72 </td><td>55 (76%) </td><td>38 (53%) </td><td>16 (22%) </td></tr></tbody></table></div><div content-id="DST048TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Minor allele frequency ranges </th><th>No. of primers designed </th><th>No. of primer validated (% of designed) </th><th>No. of probe validated (% of designed) </th><th>No. of polymorphic loci (% of designed) </th></tr></thead><tbody><tr><td>0.4–0.5 </td><td>1538 </td><td>1056 (69%) </td><td>761 (49%) </td><td>571 (37%) </td></tr><tr><td>0.3–0.4 </td><td>1422 </td><td>954 (67%) </td><td>701 (49%) </td><td>506 (36%) </td></tr><tr><td>0.2–0.3 </td><td>832 </td><td>559 (67%) </td><td>422 (51%) </td><td>266 (32%) </td></tr><tr><td>0.1–0.2 </td><td>636 </td><td>418 (66%) </td><td>309 (49%) </td><td>133 (21%) </td></tr><tr><td>0–0.1 </td><td>72 </td><td>55 (76%) </td><td>38 (53%) </td><td>16 (22%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5966950" target="_blank">View Large</a></div></div>                        <a id="5966951" scrollto-destination="5966951"></a>
<div id="DST048F2" data-id="DST048F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04802.png?Expires=2147483647&Signature=4tBbEPbQMltujgGmgud2Lr-ld8oWqgPRgMo~UuREsNxiVTKpyoKIyfoELmGnfDXv0gsvGnGqZqdYakZqxS5ypEk6AApDGKVImnxvqxXU9FFOH9JwqpNuPWl5wR2Gjco0EcAhhZz4rZCNeYBtFZC39BpYnDw8DjFN76DMb6KmTV7lw-knFBJVVRZ~1tXkYoD69NnA70olXpZPsFdmBTSPgUx7rGnPiiulcqnJypO3AmcrgH0~N1JRWF96pUYRBIaZqUyJGsKVs5wNdXjNQMKH-Cm-TXxjzQbKaRbBO-eeOaNYqGBxSfPkjs2-ul3pkpUz8lww3pUKV3QxMaZ-OACR1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic workflow describing SNP marker development and evaluation in C. farreri." path-from-xml="dst04802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966951/dst04802.tiff" path-from-xml="dst04802.tiff" target="_blank">View large</a><a section="5966951" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04802.png?Expires=2147483647&Signature=Pu9nhdMM1UktqpcezAbHH1r4S3U0hdPyTACf~M-dJO7RVnV8YJOsMyk9XZ1qPEdDovJWq60-iOl57~d4kUdg8OKRkYi28MSYxTwORrd~8AqgHQL0DgSZZINEPAzv7oiOIHY0AJ2Ji2LTaXrgWxmNCTZhDdciraJHHAWkVol0HdT-DkdanzNgMghRGy-GslLaNURpchuROFtJ-aeDtPLsU8xsv~OXBBS60ADBtliXxTqlhfdtiXqO0WcYGHbEL3rGArEMOkFmyX2DbPHE6BD0Hc~5YFiXD085HFCzGaPk9uYUCayUOGL~jyUFseVCvsA5MQQ9qAdEQXCqZwE-HCFz~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966951&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic workflow describing SNP marker development and evaluation in <em>C. farreri</em>.</p></div></div><div content-id="DST048F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04802.png?Expires=2147483647&Signature=4tBbEPbQMltujgGmgud2Lr-ld8oWqgPRgMo~UuREsNxiVTKpyoKIyfoELmGnfDXv0gsvGnGqZqdYakZqxS5ypEk6AApDGKVImnxvqxXU9FFOH9JwqpNuPWl5wR2Gjco0EcAhhZz4rZCNeYBtFZC39BpYnDw8DjFN76DMb6KmTV7lw-knFBJVVRZ~1tXkYoD69NnA70olXpZPsFdmBTSPgUx7rGnPiiulcqnJypO3AmcrgH0~N1JRWF96pUYRBIaZqUyJGsKVs5wNdXjNQMKH-Cm-TXxjzQbKaRbBO-eeOaNYqGBxSfPkjs2-ul3pkpUz8lww3pUKV3QxMaZ-OACR1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic workflow describing SNP marker development and evaluation in C. farreri." path-from-xml="dst04802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966951/dst04802.tiff" path-from-xml="dst04802.tiff" target="_blank">View large</a><a section="5966951" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04802.png?Expires=2147483647&Signature=Pu9nhdMM1UktqpcezAbHH1r4S3U0hdPyTACf~M-dJO7RVnV8YJOsMyk9XZ1qPEdDovJWq60-iOl57~d4kUdg8OKRkYi28MSYxTwORrd~8AqgHQL0DgSZZINEPAzv7oiOIHY0AJ2Ji2LTaXrgWxmNCTZhDdciraJHHAWkVol0HdT-DkdanzNgMghRGy-GslLaNURpchuROFtJ-aeDtPLsU8xsv~OXBBS60ADBtliXxTqlhfdtiXqO0WcYGHbEL3rGArEMOkFmyX2DbPHE6BD0Hc~5YFiXD085HFCzGaPk9uYUCayUOGL~jyUFseVCvsA5MQQ9qAdEQXCqZwE-HCFz~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966951&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic workflow describing SNP marker development and evaluation in <em>C. farreri</em>.</p></div></div>                            <h3 scrollto-destination=5966953 id="5966953" class="section-title" data-legacy-id=s3c>3.3. Evaluation of SNP markers for population genomic analysis</h3>
<p>The suitability of the 1492 polymorphic markers for population genomic analysis was evaluated using four geographical populations. PCR success rates across all individuals ranged from 48 to 100% with an average of 93%, suggesting that null alleles are indeed present at very low frequencies in these populations as expected for gene-derived markers. All the markers showed polymorphisms across the four populations and within each population. Marker polymorphism was 100% for the JN population, 96% for the CD population, 97% for the RC population, and 88% for the DL population. Population genetic parameters were estimated for the JN population (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), because this population had a relatively large sample size compared with the other populations, and therefore, genetic parameters could be estimated more reliably. For all markers, MAF ranged from 0.02 to 0.50. The <em>H</em><sub>o</sub> and <em>H</em><sub>e</sub> were 0.34 and 0.37, respectively; these values are relatively low compared with the previous estimates for the same population using microsatellite markers.<sup><a reveal-id="DST048C21" data-open="DST048C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In total, 1422 markers passed the neutrality test, suggesting that these markers are not under strong selection and are suitable for population genomic analyses that usually require neutral markers. The majority of markers were in Hardy–Weinberg equilibrium, and only 81 markers showed significant departures after Bonferroni correction. Significant linkage disequilibrium was detected in 102 marker pairs. These markers are valuable for detecting recent admixture and migration events in natural populations.<sup><a reveal-id="DST048C26" data-open="DST048C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=5966957 id="5966957" class="section-title" data-legacy-id=s3d>3.4. Cross-species transferability test</h3>
<p>To assess the cross-species transferability of the developed SNP markers, 34 markers targeting 24 synonymous and 10 non-synonymous SNPs were selected for HRM genotyping in five scallop species deriving from three Pectinidae subfamilies. A total of 23 were successfully amplified in at least one species, of which 16 were polymorphic in at least one species. The highest transferability (up to four species) was observed for marker C10745S115_CG (Table <a reveal-id="DST048TB2" data-open="DST048TB2" class="link link-reveal link-table xref-fig">2</a>). Unexpectedly, the total number of transferable markers was quite similar (6–7 markers) for the three Pectinidae subfamilies, which does not reflect the difference in their phylogenetic relationships to <em>C. farreri</em> (i.e. <em>C. farreri</em> belongs to Chlamydinae, which is more closely related to Pectininae than to Aequipectini<sup><a reveal-id="DST048C27" data-open="DST048C27" class="link link-ref link-reveal xref-bibr">27</a></sup>). The markers that could be transferred to at least three species were all synonymous SNPs; while all but one of the non-synonymous SNPs were transferable only to a single species. </p>                        <a id="5966961" scrollto-destination="5966961"></a>
<div content-id="DST048TB2" class="table-modal"><div class="table-wrap"><div id="DST048TB2" class="table-wrap-title" data-id="DST048TB2"><div class="label -label">Table 2.</div><div class="caption"><p>The cross-species transferability of 16 <em>C. farreri</em> SNP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Marker name </th><th colspan="5">Species name<hr /> </th></tr><tr><th><em>Chlamys nobilis</em> </th><th><em>Patinopecten yessoensis</em> </th><th><em>Amusium pleuronectes</em> </th><th><em>Argopecten irradians</em> </th><th><em>Argopecten purpuratus</em> </th></tr></thead><tbody><tr><td>C10745S115_CG </td><td>√ </td><td>√ </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C12208S460_TC </td><td>√ </td><td>√ </td><td> </td><td>√ </td><td> </td></tr><tr><td>C49010S592_AG </td><td>√ </td><td>√ </td><td>√ </td><td> </td><td> </td></tr><tr><td>C49829S586_AC </td><td>√ </td><td> </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C30426S483_TC </td><td>√ </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C47021S446_CA </td><td> </td><td>√ </td><td> </td><td> </td><td>√ </td></tr><tr><td>C34305S164_CG </td><td> </td><td> </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C15644S268_AG </td><td>√ </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>C13134S327_TC </td><td> </td><td> </td><td>√ </td><td> </td><td> </td></tr><tr><td>C37392S686_GA </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C20740S340_TC </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C11312S559_GA </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C20815S313_AT </td><td> </td><td> </td><td>√ </td><td> </td><td> </td></tr><tr><td>C19522S376_TG </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C30287S1286_TG </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C15676S348_AG </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr></tbody></table></div><div content-id="DST048TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Marker name </th><th colspan="5">Species name<hr /> </th></tr><tr><th><em>Chlamys nobilis</em> </th><th><em>Patinopecten yessoensis</em> </th><th><em>Amusium pleuronectes</em> </th><th><em>Argopecten irradians</em> </th><th><em>Argopecten purpuratus</em> </th></tr></thead><tbody><tr><td>C10745S115_CG </td><td>√ </td><td>√ </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C12208S460_TC </td><td>√ </td><td>√ </td><td> </td><td>√ </td><td> </td></tr><tr><td>C49010S592_AG </td><td>√ </td><td>√ </td><td>√ </td><td> </td><td> </td></tr><tr><td>C49829S586_AC </td><td>√ </td><td> </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C30426S483_TC </td><td>√ </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C47021S446_CA </td><td> </td><td>√ </td><td> </td><td> </td><td>√ </td></tr><tr><td>C34305S164_CG </td><td> </td><td> </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C15644S268_AG </td><td>√ </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>C13134S327_TC </td><td> </td><td> </td><td>√ </td><td> </td><td> </td></tr><tr><td>C37392S686_GA </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C20740S340_TC </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C11312S559_GA </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C20815S313_AT </td><td> </td><td> </td><td>√ </td><td> </td><td> </td></tr><tr><td>C19522S376_TG </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C30287S1286_TG </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C15676S348_AG </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Non-synonymous SNP makers are indicated in grey.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5966961" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST048TB2" class="table-wrap-title" data-id="DST048TB2"><div class="label -label">Table 2.</div><div class="caption"><p>The cross-species transferability of 16 <em>C. farreri</em> SNP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Marker name </th><th colspan="5">Species name<hr /> </th></tr><tr><th><em>Chlamys nobilis</em> </th><th><em>Patinopecten yessoensis</em> </th><th><em>Amusium pleuronectes</em> </th><th><em>Argopecten irradians</em> </th><th><em>Argopecten purpuratus</em> </th></tr></thead><tbody><tr><td>C10745S115_CG </td><td>√ </td><td>√ </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C12208S460_TC </td><td>√ </td><td>√ </td><td> </td><td>√ </td><td> </td></tr><tr><td>C49010S592_AG </td><td>√ </td><td>√ </td><td>√ </td><td> </td><td> </td></tr><tr><td>C49829S586_AC </td><td>√ </td><td> </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C30426S483_TC </td><td>√ </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C47021S446_CA </td><td> </td><td>√ </td><td> </td><td> </td><td>√ </td></tr><tr><td>C34305S164_CG </td><td> </td><td> </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C15644S268_AG </td><td>√ </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>C13134S327_TC </td><td> </td><td> </td><td>√ </td><td> </td><td> </td></tr><tr><td>C37392S686_GA </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C20740S340_TC </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C11312S559_GA </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C20815S313_AT </td><td> </td><td> </td><td>√ </td><td> </td><td> </td></tr><tr><td>C19522S376_TG </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C30287S1286_TG </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C15676S348_AG </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr></tbody></table></div><div content-id="DST048TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Marker name </th><th colspan="5">Species name<hr /> </th></tr><tr><th><em>Chlamys nobilis</em> </th><th><em>Patinopecten yessoensis</em> </th><th><em>Amusium pleuronectes</em> </th><th><em>Argopecten irradians</em> </th><th><em>Argopecten purpuratus</em> </th></tr></thead><tbody><tr><td>C10745S115_CG </td><td>√ </td><td>√ </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C12208S460_TC </td><td>√ </td><td>√ </td><td> </td><td>√ </td><td> </td></tr><tr><td>C49010S592_AG </td><td>√ </td><td>√ </td><td>√ </td><td> </td><td> </td></tr><tr><td>C49829S586_AC </td><td>√ </td><td> </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C30426S483_TC </td><td>√ </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C47021S446_CA </td><td> </td><td>√ </td><td> </td><td> </td><td>√ </td></tr><tr><td>C34305S164_CG </td><td> </td><td> </td><td>√ </td><td> </td><td>√ </td></tr><tr><td>C15644S268_AG </td><td>√ </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>C13134S327_TC </td><td> </td><td> </td><td>√ </td><td> </td><td> </td></tr><tr><td>C37392S686_GA </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C20740S340_TC </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C11312S559_GA </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C20815S313_AT </td><td> </td><td> </td><td>√ </td><td> </td><td> </td></tr><tr><td>C19522S376_TG </td><td> </td><td> </td><td> </td><td> </td><td>√ </td></tr><tr><td>C30287S1286_TG </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr><tr><td>C15676S348_AG </td><td> </td><td> </td><td> </td><td>√ </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Non-synonymous SNP makers are indicated in grey.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5966961" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5966963 id="5966963" class="section-title" data-legacy-id=s3e>3.5. Functional annotation and gene comparison with Pacific oyster</h3>
<p>Gene annotation was performed for the 1492 markers using BlastX comparison against the Swiss-Prot and Nr protein databases. A total of 1095 (73.4%) markers had significant matches to known proteins in these databases, corresponding to 953 unique accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). GO analysis revealed that one or more GO terms could be assigned to 466 markers for a total of 923 GO assignments. The annotated markers were involved in diverse biological processes and functions, and their GO composition largely resembled that summarized for the total set of contigs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). KEGG pathway analysis revealed that 420 markers were involved in 123 different pathways (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). In particular, 21 markers were involved in the immune system, and these markers are worthy of further evaluation to determine whether any are associated with disease/pathogen resistance.</p><p>To evaluate the utility of the 1492 SNP markers for future comparative analysis with the oyster genome, the contig sequences of these markers were compared against the oyster protein database. A total of 963 <em>C. farreri</em> contigs containing 1034 SNP markers showed significant sequence homology to oyster proteins (<em>e</em>-value of &lt; 1<em>e</em>−4; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). One example showing the correspondence between eight SNP markers and oyster scaffold419 is presented in Fig. <a reveal-id="DST048F3" data-open="DST048F3" class="link link-reveal link-table xref-fig">3</a>. These oyster proteins are distributed on 514 genomic scaffolds (genome assembly v9; Table <a reveal-id="DST048TB3" data-open="DST048TB3" class="link link-reveal link-table xref-fig">3</a>). The lengths of these scaffolds ranged from 2 to 1965 kb. A total of 235 scaffolds were matched by at least two <em>C. farreri</em> markers, and 39 scaffolds were matched by at least five markers (Table <a reveal-id="DST048TB3" data-open="DST048TB3" class="link link-reveal link-table xref-fig">3</a>). These markers and matching related oyster scaffolds provide an important basis for further genome comparison between the two species. </p>                        <a id="5966968" scrollto-destination="5966968"></a>
<div content-id="DST048TB3" class="table-modal"><div class="table-wrap"><div id="DST048TB3" class="table-wrap-title" data-id="DST048TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of the oyster scaffolds matched by <em>C. farreri</em> SNP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Total marker no. matched to each scaffold </th><th>No. of matched oyster scaffolds </th><th>Length of matched scaffolds (kb) </th></tr></thead><tbody><tr><td>12 </td><td>1 </td><td>1620 </td></tr><tr><td>10 </td><td>1 </td><td>1193 </td></tr><tr><td>9 </td><td>1 </td><td>1726 </td></tr><tr><td>8 </td><td>7 </td><td>649–1965 </td></tr><tr><td>7 </td><td>4 </td><td>969–1120 </td></tr><tr><td>6 </td><td>10 </td><td>650–1697 </td></tr><tr><td>5 </td><td>15 </td><td>538–1855 </td></tr><tr><td>4 </td><td>25 </td><td>63–1372 </td></tr><tr><td>3 </td><td>63 </td><td>58–1861 </td></tr><tr><td>2 </td><td>108 </td><td>31–1727 </td></tr><tr><td>1 </td><td>279 </td><td>2–1100 </td></tr></tbody></table></div><div content-id="DST048TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Total marker no. matched to each scaffold </th><th>No. of matched oyster scaffolds </th><th>Length of matched scaffolds (kb) </th></tr></thead><tbody><tr><td>12 </td><td>1 </td><td>1620 </td></tr><tr><td>10 </td><td>1 </td><td>1193 </td></tr><tr><td>9 </td><td>1 </td><td>1726 </td></tr><tr><td>8 </td><td>7 </td><td>649–1965 </td></tr><tr><td>7 </td><td>4 </td><td>969–1120 </td></tr><tr><td>6 </td><td>10 </td><td>650–1697 </td></tr><tr><td>5 </td><td>15 </td><td>538–1855 </td></tr><tr><td>4 </td><td>25 </td><td>63–1372 </td></tr><tr><td>3 </td><td>63 </td><td>58–1861 </td></tr><tr><td>2 </td><td>108 </td><td>31–1727 </td></tr><tr><td>1 </td><td>279 </td><td>2–1100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5966968" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST048TB3" class="table-wrap-title" data-id="DST048TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of the oyster scaffolds matched by <em>C. farreri</em> SNP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Total marker no. matched to each scaffold </th><th>No. of matched oyster scaffolds </th><th>Length of matched scaffolds (kb) </th></tr></thead><tbody><tr><td>12 </td><td>1 </td><td>1620 </td></tr><tr><td>10 </td><td>1 </td><td>1193 </td></tr><tr><td>9 </td><td>1 </td><td>1726 </td></tr><tr><td>8 </td><td>7 </td><td>649–1965 </td></tr><tr><td>7 </td><td>4 </td><td>969–1120 </td></tr><tr><td>6 </td><td>10 </td><td>650–1697 </td></tr><tr><td>5 </td><td>15 </td><td>538–1855 </td></tr><tr><td>4 </td><td>25 </td><td>63–1372 </td></tr><tr><td>3 </td><td>63 </td><td>58–1861 </td></tr><tr><td>2 </td><td>108 </td><td>31–1727 </td></tr><tr><td>1 </td><td>279 </td><td>2–1100 </td></tr></tbody></table></div><div content-id="DST048TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Total marker no. matched to each scaffold </th><th>No. of matched oyster scaffolds </th><th>Length of matched scaffolds (kb) </th></tr></thead><tbody><tr><td>12 </td><td>1 </td><td>1620 </td></tr><tr><td>10 </td><td>1 </td><td>1193 </td></tr><tr><td>9 </td><td>1 </td><td>1726 </td></tr><tr><td>8 </td><td>7 </td><td>649–1965 </td></tr><tr><td>7 </td><td>4 </td><td>969–1120 </td></tr><tr><td>6 </td><td>10 </td><td>650–1697 </td></tr><tr><td>5 </td><td>15 </td><td>538–1855 </td></tr><tr><td>4 </td><td>25 </td><td>63–1372 </td></tr><tr><td>3 </td><td>63 </td><td>58–1861 </td></tr><tr><td>2 </td><td>108 </td><td>31–1727 </td></tr><tr><td>1 </td><td>279 </td><td>2–1100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5966968" target="_blank">View Large</a></div></div>                        <a id="5966970" scrollto-destination="5966970"></a>
<div id="DST048F3" data-id="DST048F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04803.png?Expires=2147483647&Signature=V19RzGZC~Ooj84bK1UyPruWyn-VxfJ9yAuzSSqsJ1lfAa6N9Pgw2qqBdkNmyQ5Jxt9tLc2uOc~7p1aEzoMj4rhRZlGv1jkWEf3HJoWUsBxOaokqbm-REitqJGDdybVaGGZqQUq7hcs2IuNdaofQ6H~jgHsjQnB~MwRL7m96xhgy6xEtsBcji1HASG268AcyoFKVUsIurBX-kDj9BLWwxzgWHQm-zUsqC0t34XYRoQ74o4HaHKShKRoNh5ckofMGs268WARJ5aB8utKaPB8rmyvyItLyeJdey3lFOpF11yYq47R08KK1usfgc6n~HV2E3yS1apgyFjzlSoy9ztSZucQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the correspondence between eight C. farreri SNP markers and Pacific oyster scaffold419 (genome assembly v9). Oyster proteins and the related C. farreri SNPs are indicated below the scaffold." path-from-xml="dst04803.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966970/dst04803.tiff" path-from-xml="dst04803.tiff" target="_blank">View large</a><a section="5966970" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04803.png?Expires=2147483647&Signature=jIvxCI5tpjQdnMgRow-fbsE08ry3xn6cfPxDS5LLvCSONusfjeFfpWL~Ip~zwJkUR9MOLZK2fwW7vjYws62HmG5ds3DbklOhVsjEzm0CQGYgIb8RKu-p9NXdjU7faUYLJkzdTyqY2kmcN3xfx9S4j~6-ge1ruItksbHCIaM4QRCFki6k-mtnfhIbVYY9-a5uDJBQeGHFf1TQEw5UDNfxjorq3TDVlXxV7WRQMMBRN9Nd4rQ567A4n-izb8uhgq45r0q-ncS1m~ZDuOWP5UGBwuXC2W50tHBFMEcd20YQHRcKIx~ZTn28qVZBgA7oYxv6nXOA8gk759JKLhXqzhOBCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966970&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04803.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the correspondence between eight <em>C. farreri</em> SNP markers and Pacific oyster scaffold419 (genome assembly v9). Oyster proteins and the related <em>C. farreri</em> SNPs are indicated below the scaffold.</p></div></div><div content-id="DST048F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04803.png?Expires=2147483647&Signature=V19RzGZC~Ooj84bK1UyPruWyn-VxfJ9yAuzSSqsJ1lfAa6N9Pgw2qqBdkNmyQ5Jxt9tLc2uOc~7p1aEzoMj4rhRZlGv1jkWEf3HJoWUsBxOaokqbm-REitqJGDdybVaGGZqQUq7hcs2IuNdaofQ6H~jgHsjQnB~MwRL7m96xhgy6xEtsBcji1HASG268AcyoFKVUsIurBX-kDj9BLWwxzgWHQm-zUsqC0t34XYRoQ74o4HaHKShKRoNh5ckofMGs268WARJ5aB8utKaPB8rmyvyItLyeJdey3lFOpF11yYq47R08KK1usfgc6n~HV2E3yS1apgyFjzlSoy9ztSZucQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the correspondence between eight C. farreri SNP markers and Pacific oyster scaffold419 (genome assembly v9). Oyster proteins and the related C. farreri SNPs are indicated below the scaffold." path-from-xml="dst04803.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966970/dst04803.tiff" path-from-xml="dst04803.tiff" target="_blank">View large</a><a section="5966970" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04803.png?Expires=2147483647&Signature=jIvxCI5tpjQdnMgRow-fbsE08ry3xn6cfPxDS5LLvCSONusfjeFfpWL~Ip~zwJkUR9MOLZK2fwW7vjYws62HmG5ds3DbklOhVsjEzm0CQGYgIb8RKu-p9NXdjU7faUYLJkzdTyqY2kmcN3xfx9S4j~6-ge1ruItksbHCIaM4QRCFki6k-mtnfhIbVYY9-a5uDJBQeGHFf1TQEw5UDNfxjorq3TDVlXxV7WRQMMBRN9Nd4rQ567A4n-izb8uhgq45r0q-ncS1m~ZDuOWP5UGBwuXC2W50tHBFMEcd20YQHRcKIx~ZTn28qVZBgA7oYxv6nXOA8gk759JKLhXqzhOBCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966970&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04803.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the correspondence between eight <em>C. farreri</em> SNP markers and Pacific oyster scaffold419 (genome assembly v9). Oyster proteins and the related <em>C. farreri</em> SNPs are indicated below the scaffold.</p></div></div>                            <h3 scrollto-destination=5966973 id="5966973" class="section-title" data-legacy-id=s3f>3.6. Large-scale marker associations with growth traits</h3>
<p>Based on the developed marker set, we further conducted a large-scale association analysis of growth traits using two <em>C. farreri</em> populations that were derived from a large breeding population of an elite <em>C. farreri</em> variety (‘Penglai-Red’ scallop), but were reared in two different geographic locations. Two populations were used in the association analysis to ensure identification of SNP loci linked with the genetic variance of traits but not the environmental effects. To reduce the genotyping cost, a selective genotyping approach was adopted, in which only two groups of individuals sampled from the two tails of the trait distribution were used for HRM analysis (Fig. <a reveal-id="DST048F4" data-open="DST048F4" class="link link-reveal link-table xref-fig">4</a>). An initial scan of the whole marker set on DNA pools revealed that five markers showed prominent allele frequency difference between the two groups in both populations (Fig. <a reveal-id="DST048F5" data-open="DST048F5" class="link link-reveal link-table xref-fig">5</a>). Statistical significance of the observed allele frequency difference was further confirmed for the five markers by genotyping each individual within each group for both populations (Table <a reveal-id="DST048TB4" data-open="DST048TB4" class="link link-reveal link-table xref-fig">4</a>). After Bonferroni correction, four markers remained significant in at least one population. Gene annotation information was available for only one marker, C7493S233_CT (<em>SDF4</em>, 45 kDa calcium-binding protein, 7<em>e</em>−19). </p>                        <a id="5966977" scrollto-destination="5966977"></a>
<div content-id="DST048TB4" class="table-modal"><div class="table-wrap"><div id="DST048TB4" class="table-wrap-title" data-id="DST048TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Comparison of the allele and genotype frequency of five growth-related markers between LBS and SBS groups sampled from two <em>C. farreri</em> populations.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Maker name </th><th colspan="5">LBS group (population 1/population 2)<hr /> </th><th colspan="5">SBS group (population 1/population 2)<hr /> </th><th rowspan="2">Allele frequency <em>P</em>-value </th><th rowspan="2">Genotype frequency <em>P</em>-value </th></tr><tr><th colspan="2">Allele frequency<hr /> </th><th colspan="3">Genotype frequency<hr /> </th><th colspan="2">Allele frequency<hr /> </th><th colspan="3">Genotype frequency<hr /> </th></tr></thead><tbody><tr><td>C10973S307_GA </td><td>0.51/0.29 (A) </td><td>0.49/0.71 (G) </td><td>0.36/0.07 (AA) </td><td>0.31/0.44 (GA) </td><td>0.33/0.49 (GG) </td><td>0.86/0.74 (A) </td><td>0.14/0.26 (G) </td><td>0.76/0.68 (AA) </td><td>0.18/0.12 (GA) </td><td>0.05/0.20 (GG) </td><td>8.86<em>e</em>−06<sup>a</sup>/1.10 <em>e</em>−07<sup>a</sup> </td><td>5.00 <em>e</em>−04/5.68 <em>e</em>−07<sup>a</sup> </td></tr><tr><td>C6068S140_CA </td><td>0.49/0.41 (A) </td><td>0.51/0.59 (C) </td><td>0.33/0.24 (AA) </td><td>0.31/0.33 (CA) </td><td>0.36/0.43 (CC) </td><td>0.81/0.80 (A) </td><td>0.19/0.20 (C) </td><td>0.70/0.76 (AA) </td><td>0.22/0.08 (CA) </td><td>0.08/0.16 (CC) </td><td>3.98 <em>e</em>−03/1.12 <em>e</em>−06<sup>a</sup> </td><td>2.24 <em>e</em>−03/5.27 <em>e</em>−05 </td></tr><tr><td>C13555S183_TG </td><td>0.45/0.32 (G) </td><td>0.55/0.68 (T) </td><td>0.28/0.13 (GG) </td><td>0.34/0.38 (TG) </td><td>0.38/0.49 (TT) </td><td>0.77/0.74 (G) </td><td>0.23/0.26 (T) </td><td>0.69/0.64 (GG) </td><td>0.17/0.18 (TG) </td><td>0.14/0.18 (TT) </td><td>6.10 <em>e</em>−05/2.55 <em>e</em>−07<sup>a</sup> </td><td>1.70 <em>e</em>−03/1.20 <em>e</em>−05<sup>a</sup> </td></tr><tr><td>C17803S726_GA </td><td>0.55/0.76 (A) </td><td>0.45/0.24 (G) </td><td>0.40/0.72 (AA) </td><td>0.30/0.08 (GA) </td><td>0.30/0.20 (GG) </td><td>0.25/0.47 (A) </td><td>0.75/0.53 (G) </td><td>0.10/0.36 (AA) </td><td>0.29/0.23 (GA) </td><td>0.61/0.41 (GG) </td><td>1.44 <em>e</em>−04/2.94 <em>e</em>−04 </td><td>5.01 <em>e</em>−03/2.15 <em>e</em>−04 </td></tr><tr><td>C7493S233_CT </td><td>0.96/0.85 (C) </td><td>0.04/0.15 (T) </td><td>0.94/0.77 (CC) </td><td>0.03/0.15 (CT) </td><td>0.03/0.08 (TT) </td><td>0.71/0.53 (C) </td><td>0.29/0.47 (T) </td><td>0.59/0.31 (CC) </td><td>0.23/0.43 (CT) </td><td>0.18/0.26 (TT) </td><td>1.73 <em>e</em>−05<sup>a</sup>/2.75 <em>e</em>−05<sup>a</sup> </td><td>8.39 <em>e</em>−04/4.33 <em>e</em>−04 </td></tr></tbody></table></div><div content-id="DST048TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Maker name </th><th colspan="5">LBS group (population 1/population 2)<hr /> </th><th colspan="5">SBS group (population 1/population 2)<hr /> </th><th rowspan="2">Allele frequency <em>P</em>-value </th><th rowspan="2">Genotype frequency <em>P</em>-value </th></tr><tr><th colspan="2">Allele frequency<hr /> </th><th colspan="3">Genotype frequency<hr /> </th><th colspan="2">Allele frequency<hr /> </th><th colspan="3">Genotype frequency<hr /> </th></tr></thead><tbody><tr><td>C10973S307_GA </td><td>0.51/0.29 (A) </td><td>0.49/0.71 (G) </td><td>0.36/0.07 (AA) </td><td>0.31/0.44 (GA) </td><td>0.33/0.49 (GG) </td><td>0.86/0.74 (A) </td><td>0.14/0.26 (G) </td><td>0.76/0.68 (AA) </td><td>0.18/0.12 (GA) </td><td>0.05/0.20 (GG) </td><td>8.86<em>e</em>−06<sup>a</sup>/1.10 <em>e</em>−07<sup>a</sup> </td><td>5.00 <em>e</em>−04/5.68 <em>e</em>−07<sup>a</sup> </td></tr><tr><td>C6068S140_CA </td><td>0.49/0.41 (A) </td><td>0.51/0.59 (C) </td><td>0.33/0.24 (AA) </td><td>0.31/0.33 (CA) </td><td>0.36/0.43 (CC) </td><td>0.81/0.80 (A) </td><td>0.19/0.20 (C) </td><td>0.70/0.76 (AA) </td><td>0.22/0.08 (CA) </td><td>0.08/0.16 (CC) </td><td>3.98 <em>e</em>−03/1.12 <em>e</em>−06<sup>a</sup> </td><td>2.24 <em>e</em>−03/5.27 <em>e</em>−05 </td></tr><tr><td>C13555S183_TG </td><td>0.45/0.32 (G) </td><td>0.55/0.68 (T) </td><td>0.28/0.13 (GG) </td><td>0.34/0.38 (TG) </td><td>0.38/0.49 (TT) </td><td>0.77/0.74 (G) </td><td>0.23/0.26 (T) </td><td>0.69/0.64 (GG) </td><td>0.17/0.18 (TG) </td><td>0.14/0.18 (TT) </td><td>6.10 <em>e</em>−05/2.55 <em>e</em>−07<sup>a</sup> </td><td>1.70 <em>e</em>−03/1.20 <em>e</em>−05<sup>a</sup> </td></tr><tr><td>C17803S726_GA </td><td>0.55/0.76 (A) </td><td>0.45/0.24 (G) </td><td>0.40/0.72 (AA) </td><td>0.30/0.08 (GA) </td><td>0.30/0.20 (GG) </td><td>0.25/0.47 (A) </td><td>0.75/0.53 (G) </td><td>0.10/0.36 (AA) </td><td>0.29/0.23 (GA) </td><td>0.61/0.41 (GG) </td><td>1.44 <em>e</em>−04/2.94 <em>e</em>−04 </td><td>5.01 <em>e</em>−03/2.15 <em>e</em>−04 </td></tr><tr><td>C7493S233_CT </td><td>0.96/0.85 (C) </td><td>0.04/0.15 (T) </td><td>0.94/0.77 (CC) </td><td>0.03/0.15 (CT) </td><td>0.03/0.08 (TT) </td><td>0.71/0.53 (C) </td><td>0.29/0.47 (T) </td><td>0.59/0.31 (CC) </td><td>0.23/0.43 (CT) </td><td>0.18/0.26 (TT) </td><td>1.73 <em>e</em>−05<sup>a</sup>/2.75 <em>e</em>−05<sup>a</sup> </td><td>8.39 <em>e</em>−04/4.33 <em>e</em>−04 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Significant after Bonferroni correction.</p></div></div><div content-id=""><div class="fn"><p>For each marker, alleles or genotypes are indicated in parenthesis below their frequencies.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5966977" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST048TB4" class="table-wrap-title" data-id="DST048TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Comparison of the allele and genotype frequency of five growth-related markers between LBS and SBS groups sampled from two <em>C. farreri</em> populations.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Maker name </th><th colspan="5">LBS group (population 1/population 2)<hr /> </th><th colspan="5">SBS group (population 1/population 2)<hr /> </th><th rowspan="2">Allele frequency <em>P</em>-value </th><th rowspan="2">Genotype frequency <em>P</em>-value </th></tr><tr><th colspan="2">Allele frequency<hr /> </th><th colspan="3">Genotype frequency<hr /> </th><th colspan="2">Allele frequency<hr /> </th><th colspan="3">Genotype frequency<hr /> </th></tr></thead><tbody><tr><td>C10973S307_GA </td><td>0.51/0.29 (A) </td><td>0.49/0.71 (G) </td><td>0.36/0.07 (AA) </td><td>0.31/0.44 (GA) </td><td>0.33/0.49 (GG) </td><td>0.86/0.74 (A) </td><td>0.14/0.26 (G) </td><td>0.76/0.68 (AA) </td><td>0.18/0.12 (GA) </td><td>0.05/0.20 (GG) </td><td>8.86<em>e</em>−06<sup>a</sup>/1.10 <em>e</em>−07<sup>a</sup> </td><td>5.00 <em>e</em>−04/5.68 <em>e</em>−07<sup>a</sup> </td></tr><tr><td>C6068S140_CA </td><td>0.49/0.41 (A) </td><td>0.51/0.59 (C) </td><td>0.33/0.24 (AA) </td><td>0.31/0.33 (CA) </td><td>0.36/0.43 (CC) </td><td>0.81/0.80 (A) </td><td>0.19/0.20 (C) </td><td>0.70/0.76 (AA) </td><td>0.22/0.08 (CA) </td><td>0.08/0.16 (CC) </td><td>3.98 <em>e</em>−03/1.12 <em>e</em>−06<sup>a</sup> </td><td>2.24 <em>e</em>−03/5.27 <em>e</em>−05 </td></tr><tr><td>C13555S183_TG </td><td>0.45/0.32 (G) </td><td>0.55/0.68 (T) </td><td>0.28/0.13 (GG) </td><td>0.34/0.38 (TG) </td><td>0.38/0.49 (TT) </td><td>0.77/0.74 (G) </td><td>0.23/0.26 (T) </td><td>0.69/0.64 (GG) </td><td>0.17/0.18 (TG) </td><td>0.14/0.18 (TT) </td><td>6.10 <em>e</em>−05/2.55 <em>e</em>−07<sup>a</sup> </td><td>1.70 <em>e</em>−03/1.20 <em>e</em>−05<sup>a</sup> </td></tr><tr><td>C17803S726_GA </td><td>0.55/0.76 (A) </td><td>0.45/0.24 (G) </td><td>0.40/0.72 (AA) </td><td>0.30/0.08 (GA) </td><td>0.30/0.20 (GG) </td><td>0.25/0.47 (A) </td><td>0.75/0.53 (G) </td><td>0.10/0.36 (AA) </td><td>0.29/0.23 (GA) </td><td>0.61/0.41 (GG) </td><td>1.44 <em>e</em>−04/2.94 <em>e</em>−04 </td><td>5.01 <em>e</em>−03/2.15 <em>e</em>−04 </td></tr><tr><td>C7493S233_CT </td><td>0.96/0.85 (C) </td><td>0.04/0.15 (T) </td><td>0.94/0.77 (CC) </td><td>0.03/0.15 (CT) </td><td>0.03/0.08 (TT) </td><td>0.71/0.53 (C) </td><td>0.29/0.47 (T) </td><td>0.59/0.31 (CC) </td><td>0.23/0.43 (CT) </td><td>0.18/0.26 (TT) </td><td>1.73 <em>e</em>−05<sup>a</sup>/2.75 <em>e</em>−05<sup>a</sup> </td><td>8.39 <em>e</em>−04/4.33 <em>e</em>−04 </td></tr></tbody></table></div><div content-id="DST048TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Maker name </th><th colspan="5">LBS group (population 1/population 2)<hr /> </th><th colspan="5">SBS group (population 1/population 2)<hr /> </th><th rowspan="2">Allele frequency <em>P</em>-value </th><th rowspan="2">Genotype frequency <em>P</em>-value </th></tr><tr><th colspan="2">Allele frequency<hr /> </th><th colspan="3">Genotype frequency<hr /> </th><th colspan="2">Allele frequency<hr /> </th><th colspan="3">Genotype frequency<hr /> </th></tr></thead><tbody><tr><td>C10973S307_GA </td><td>0.51/0.29 (A) </td><td>0.49/0.71 (G) </td><td>0.36/0.07 (AA) </td><td>0.31/0.44 (GA) </td><td>0.33/0.49 (GG) </td><td>0.86/0.74 (A) </td><td>0.14/0.26 (G) </td><td>0.76/0.68 (AA) </td><td>0.18/0.12 (GA) </td><td>0.05/0.20 (GG) </td><td>8.86<em>e</em>−06<sup>a</sup>/1.10 <em>e</em>−07<sup>a</sup> </td><td>5.00 <em>e</em>−04/5.68 <em>e</em>−07<sup>a</sup> </td></tr><tr><td>C6068S140_CA </td><td>0.49/0.41 (A) </td><td>0.51/0.59 (C) </td><td>0.33/0.24 (AA) </td><td>0.31/0.33 (CA) </td><td>0.36/0.43 (CC) </td><td>0.81/0.80 (A) </td><td>0.19/0.20 (C) </td><td>0.70/0.76 (AA) </td><td>0.22/0.08 (CA) </td><td>0.08/0.16 (CC) </td><td>3.98 <em>e</em>−03/1.12 <em>e</em>−06<sup>a</sup> </td><td>2.24 <em>e</em>−03/5.27 <em>e</em>−05 </td></tr><tr><td>C13555S183_TG </td><td>0.45/0.32 (G) </td><td>0.55/0.68 (T) </td><td>0.28/0.13 (GG) </td><td>0.34/0.38 (TG) </td><td>0.38/0.49 (TT) </td><td>0.77/0.74 (G) </td><td>0.23/0.26 (T) </td><td>0.69/0.64 (GG) </td><td>0.17/0.18 (TG) </td><td>0.14/0.18 (TT) </td><td>6.10 <em>e</em>−05/2.55 <em>e</em>−07<sup>a</sup> </td><td>1.70 <em>e</em>−03/1.20 <em>e</em>−05<sup>a</sup> </td></tr><tr><td>C17803S726_GA </td><td>0.55/0.76 (A) </td><td>0.45/0.24 (G) </td><td>0.40/0.72 (AA) </td><td>0.30/0.08 (GA) </td><td>0.30/0.20 (GG) </td><td>0.25/0.47 (A) </td><td>0.75/0.53 (G) </td><td>0.10/0.36 (AA) </td><td>0.29/0.23 (GA) </td><td>0.61/0.41 (GG) </td><td>1.44 <em>e</em>−04/2.94 <em>e</em>−04 </td><td>5.01 <em>e</em>−03/2.15 <em>e</em>−04 </td></tr><tr><td>C7493S233_CT </td><td>0.96/0.85 (C) </td><td>0.04/0.15 (T) </td><td>0.94/0.77 (CC) </td><td>0.03/0.15 (CT) </td><td>0.03/0.08 (TT) </td><td>0.71/0.53 (C) </td><td>0.29/0.47 (T) </td><td>0.59/0.31 (CC) </td><td>0.23/0.43 (CT) </td><td>0.18/0.26 (TT) </td><td>1.73 <em>e</em>−05<sup>a</sup>/2.75 <em>e</em>−05<sup>a</sup> </td><td>8.39 <em>e</em>−04/4.33 <em>e</em>−04 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Significant after Bonferroni correction.</p></div></div><div content-id=""><div class="fn"><p>For each marker, alleles or genotypes are indicated in parenthesis below their frequencies.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5966977" target="_blank">View Large</a></div></div>                        <a id="5966979" scrollto-destination="5966979"></a>
<div id="DST048F4" data-id="DST048F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04804.png?Expires=2147483647&Signature=pViz3wXZVaruZfPMd7K-ycfAHjzrdR2YDEblaXlLZjKamQKdAl3JEuoq1aBhY45ZfZIQgY9epty2nr6hQV9x~ukE-SgFJU5U7BQKb7PeTHrgDiPynNoB9uZ65ruTQZ1JdMGVEQyHus1nPRYt9tFp6Z6KPjKvBHeoqTkrD6jrwxpvHnkaQb56saAyUVTAXpqHzgNSUHQpPpmCBUATamIryRWE0vHVlBrH6ieDS2Yw9xaIZlu67j2MCDUcDqhL2QS0ICToq0iwr9gZJVi4B17koolNV7uVw3ScfHuL58WvR8~R6rffDoCaR4kI8OZfytdpr4VYwQdKtRgcoZzjh5GE9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The shell-height distribution of approximately 1000 C. farreri individuals collected from each of two C. farreri populations. For the large body size (LBS) groups, the average trait value was 75.59 ± 2.92 mm for population a and 72.02 ± 2.57 mm for population b, whereas for small body size (SBS) groups, it was 49.93 ± 3.84 and 53.04 ± 3.41 mm for populations a and b, respectively. For each population, 40 individuals were sampled from each group for association analysis." path-from-xml="dst04804.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966979/dst04804.tiff" path-from-xml="dst04804.tiff" target="_blank">View large</a><a section="5966979" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04804.png?Expires=2147483647&Signature=ZVH2gQXeJmXxdRFaRQNmXJytFjF8OI5lfzfASEfMgk5z0CiFj1GHM1GcjtP3rnWE8K0jgFsdffeDvUnXf~wKvtrVX~Tz5l8AWqgN6HbL8H8e9cyrRI2OEAYEPCFqm45Pt52i7njWnfqlh5~~KBPE~jFLa31ob0PFwuRibsot5~d0CmO3SmKCtexjM4tQe0SbY-YDPOgCM3t03pawFHa05Q7hdgtf55ZjaPACjUagBn4cJw3cB1v5342wP6dq0Dq1f7MU1PJ~2g-0k0Pg8dTWaVs3hc0z1O95ciDi2YfU1BgRUgOpyJgK~K7CJpTharpddHPsBfyHLyexwH9BfeE56w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966979&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04804.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The shell-height distribution of approximately 1000 <em>C. farreri</em> individuals collected from each of two <em>C. farreri</em> populations. For the large body size (LBS) groups, the average trait value was 75.59 ± 2.92 mm for population a and 72.02 ± 2.57 mm for population b, whereas for small body size (SBS) groups, it was 49.93 ± 3.84 and 53.04 ± 3.41 mm for populations a and b, respectively. For each population, 40 individuals were sampled from each group for association analysis.</p></div></div><div content-id="DST048F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04804.png?Expires=2147483647&Signature=pViz3wXZVaruZfPMd7K-ycfAHjzrdR2YDEblaXlLZjKamQKdAl3JEuoq1aBhY45ZfZIQgY9epty2nr6hQV9x~ukE-SgFJU5U7BQKb7PeTHrgDiPynNoB9uZ65ruTQZ1JdMGVEQyHus1nPRYt9tFp6Z6KPjKvBHeoqTkrD6jrwxpvHnkaQb56saAyUVTAXpqHzgNSUHQpPpmCBUATamIryRWE0vHVlBrH6ieDS2Yw9xaIZlu67j2MCDUcDqhL2QS0ICToq0iwr9gZJVi4B17koolNV7uVw3ScfHuL58WvR8~R6rffDoCaR4kI8OZfytdpr4VYwQdKtRgcoZzjh5GE9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The shell-height distribution of approximately 1000 C. farreri individuals collected from each of two C. farreri populations. For the large body size (LBS) groups, the average trait value was 75.59 ± 2.92 mm for population a and 72.02 ± 2.57 mm for population b, whereas for small body size (SBS) groups, it was 49.93 ± 3.84 and 53.04 ± 3.41 mm for populations a and b, respectively. For each population, 40 individuals were sampled from each group for association analysis." path-from-xml="dst04804.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966979/dst04804.tiff" path-from-xml="dst04804.tiff" target="_blank">View large</a><a section="5966979" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04804.png?Expires=2147483647&Signature=ZVH2gQXeJmXxdRFaRQNmXJytFjF8OI5lfzfASEfMgk5z0CiFj1GHM1GcjtP3rnWE8K0jgFsdffeDvUnXf~wKvtrVX~Tz5l8AWqgN6HbL8H8e9cyrRI2OEAYEPCFqm45Pt52i7njWnfqlh5~~KBPE~jFLa31ob0PFwuRibsot5~d0CmO3SmKCtexjM4tQe0SbY-YDPOgCM3t03pawFHa05Q7hdgtf55ZjaPACjUagBn4cJw3cB1v5342wP6dq0Dq1f7MU1PJ~2g-0k0Pg8dTWaVs3hc0z1O95ciDi2YfU1BgRUgOpyJgK~K7CJpTharpddHPsBfyHLyexwH9BfeE56w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966979&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04804.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The shell-height distribution of approximately 1000 <em>C. farreri</em> individuals collected from each of two <em>C. farreri</em> populations. For the large body size (LBS) groups, the average trait value was 75.59 ± 2.92 mm for population a and 72.02 ± 2.57 mm for population b, whereas for small body size (SBS) groups, it was 49.93 ± 3.84 and 53.04 ± 3.41 mm for populations a and b, respectively. For each population, 40 individuals were sampled from each group for association analysis.</p></div></div>                        <a id="5966982" scrollto-destination="5966982"></a>
<div id="DST048F5" data-id="DST048F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04805.png?Expires=2147483647&Signature=Q3BmcNVDsdmHfoVHCMml9f0iu5dS6LsDU1Nu~X2bQDB5I2g8znSbafQVxG3xUXXY2FnWasdXi4L0jriMW9xEheFkmXepTFjWjKM0YpbJXGof-rSJ8V6m3G00-MrQDF7Um~KDEafN2nyd6iwQ~y3c0GZlp3DJqzlip0nVnTOD0ZQ~jASB4bPN20Dp9qAjYtCxMMcLVwcHL7~tY3tsU9L4s24I4WAmsfMETU-uqnhY~~ZjW7aW0KO97qUxNLtqoB~yCEqy35ykCR7qZbBK-Zly3AEjv~Tg~Qxsy8~4IfoqnwvFR5uUsL4tBZdPyjtGF5E9lbw6r89fmRXjKCYTZPR5Ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example HRM profiles for two growth-related markers, C10973S307_GA and C7493S233_CT, identified by genotyping DNA pools prepared from both the LBS and SBS groups. Clear differences in allele frequencies were observed between groups." path-from-xml="dst04805.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966982/dst04805.tiff" path-from-xml="dst04805.tiff" target="_blank">View large</a><a section="5966982" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04805.png?Expires=2147483647&Signature=I8uS~LEwEhQA-KzeZ6lQHt7l7Y34RggYez8tI~5vUGjmEiK4SfN0Pb~i27cO8nbA5a~JZQMinXoAAT1Pl2ODX2LjpQStDZI1UoZ4lEUxAfoBOzjsVDNhBbBMuNYAT09k3hFxDmoLnrJusKpiqMSJwYWlw4iXWjaOeDmmCxLGP8wGVM4rDQkeO8Oh2P26vxgUAgDYLLDGwntpHXHstrwEsA37beQndedyOzwJQc8LMWkNIA9mrxEdVLgjUhTnyV2epWYbKphpcLOsIygZ5i8-RFbGsDZT-pQ6v3iGi94lMytQeBhRaXmdEmejrYNV1l1oPkAs5~57kLXjLJE0-nVUPw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966982&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04805.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example HRM profiles for two growth-related markers, C10973S307_GA and C7493S233_CT, identified by genotyping DNA pools prepared from both the LBS and SBS groups. Clear differences in allele frequencies were observed between groups.</p></div></div><div content-id="DST048F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/m_dst04805.png?Expires=2147483647&Signature=Q3BmcNVDsdmHfoVHCMml9f0iu5dS6LsDU1Nu~X2bQDB5I2g8znSbafQVxG3xUXXY2FnWasdXi4L0jriMW9xEheFkmXepTFjWjKM0YpbJXGof-rSJ8V6m3G00-MrQDF7Um~KDEafN2nyd6iwQ~y3c0GZlp3DJqzlip0nVnTOD0ZQ~jASB4bPN20Dp9qAjYtCxMMcLVwcHL7~tY3tsU9L4s24I4WAmsfMETU-uqnhY~~ZjW7aW0KO97qUxNLtqoB~yCEqy35ykCR7qZbBK-Zly3AEjv~Tg~Qxsy8~4IfoqnwvFR5uUsL4tBZdPyjtGF5E9lbw6r89fmRXjKCYTZPR5Ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example HRM profiles for two growth-related markers, C10973S307_GA and C7493S233_CT, identified by genotyping DNA pools prepared from both the LBS and SBS groups. Clear differences in allele frequencies were observed between groups." path-from-xml="dst04805.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5966982/dst04805.tiff" path-from-xml="dst04805.tiff" target="_blank">View large</a><a section="5966982" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst04805.png?Expires=2147483647&Signature=I8uS~LEwEhQA-KzeZ6lQHt7l7Y34RggYez8tI~5vUGjmEiK4SfN0Pb~i27cO8nbA5a~JZQMinXoAAT1Pl2ODX2LjpQStDZI1UoZ4lEUxAfoBOzjsVDNhBbBMuNYAT09k3hFxDmoLnrJusKpiqMSJwYWlw4iXWjaOeDmmCxLGP8wGVM4rDQkeO8Oh2P26vxgUAgDYLLDGwntpHXHstrwEsA37beQndedyOzwJQc8LMWkNIA9mrxEdVLgjUhTnyV2epWYbKphpcLOsIygZ5i8-RFbGsDZT-pQ6v3iGi94lMytQeBhRaXmdEmejrYNV1l1oPkAs5~57kLXjLJE0-nVUPw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5966982&ar=403825&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04805.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example HRM profiles for two growth-related markers, C10973S307_GA and C7493S233_CT, identified by genotyping DNA pools prepared from both the LBS and SBS groups. Clear differences in allele frequencies were observed between groups.</p></div></div>                            <h2 scrollto-destination=5966984 id="5966984" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5966986 id="5966986" class="section-title" data-legacy-id=s4a>4.1. Large-scale marker development from NGS-generated transcriptomic resources</h3>
<p>NGS technologies are now frequently being used to generate extensive genomic resources for non-model organisms. Although a large number of SNPs can be readily discovered from these resources, large-scale marker development and evaluation are seldom performed for such <em>in silico</em> predicted SNPs. In this study, we conducted the first transcriptome-wide marker development from more than 21,000 putative SNPs for a bivalve mollusc, which currently represents the largest gene-associated marker collection for the phylum Mollusca. Although stringent criteria were adopted for transcriptome assembly, the marker conversion rate remained relatively low (∼33%), but was still comparable with those reported in recent studies using the same genotyping platform.<sup><a reveal-id="DST048C14  DST048C16" data-open="DST048C14  DST048C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> The inefficient marker conversion can largely be attributed to low passing rates in the primer and probe tests, i.e. 32.4% failed the primer test and 18% failed the probe test. This issue may be directly related to a few well-known drawbacks of NGS platforms. For example, NGS platforms usually generate a substantial higher rate of sequencing errors than traditional Sanger-based methods. In particular, the 454 sequencing platform is prone to introducing higher indel errors than other platforms, especially when stretches of homopolymeric bases are present.<sup><a reveal-id="DST048C28  DST048C30" data-open="DST048C28  DST048C30" class="link link-ref link-reveal xref-bibr">28–30</a></sup> Sequencing errors, when occurring at positions where a primer or probe binds, may hamper the amplification efficiency and confound subsequent HRM analysis. In addition, sequencing errors are also a major source of falsely predicted SNPs. As revealed by this study, there is a trend for SNPs with higher MAF to have higher validation rates. This result suggests that SNPs with high MAF (e.g. 40–50%) are less likely to be affected by sequencing errors and thus, should be given high priority during marker development. Furthermore, NGS platforms usually produce very short reads (e.g. 35–100 bp); accurate <em>de novo</em> assembly of such short reads poses a significant informatics challenge.<sup><a reveal-id="DST048C31" data-open="DST048C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Assembly artefacts can arise when reads of different genomic origins are misassembled together due to high similarity of sequence context. Such artefacts may result in PCR failure when primers are targeting two genomic regions far from each other. One interesting finding is that introns were present at a substantially high frequency, even though we restricted the expected amplicon length to ∼100 bp during primer design. Approximately 14% of primer pairs produced amplicons with a larger than expected size, indicating that introns were present in the vicinity of SNPs. It should be noted that this percentage most likely underestimates the actual occurrence of introns, because very large introns would more likely result in PCR failure rather than successful amplicons of larger than expected size. The aforementioned issues cannot be easily amended without the aid of a high-quality reference genome. However, with the rapid development of sequencing technologies (i.e. much longer reads and higher accuracy) and more draft genomes being generated for non-model species, there is much hope that these issues can be resolved.</p>                            <h3 scrollto-destination=5966988 id="5966988" class="section-title" data-legacy-id=s4b>4.2. Genic versus non-genic markers in molluscan population studies</h3>
<p>Currently, the genetic markers available for molluscs were developed predominantly from anonymous genomic sequences. It is well known that molluscan genomes are typically highly heterozygous.<sup><a reveal-id="DST048C4" data-open="DST048C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST048C5" data-open="DST048C5" class="link link-ref link-reveal xref-bibr">5</a></sup> For example, the Pacific oyster has an average SNP density of one SNP per 60 bp in coding regions and one per 40 bp in non-coding regions.<sup><a reveal-id="DST048C32" data-open="DST048C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Such high genome heterozygosity can result in a high frequency of null alleles, i.e. alleles fail to amplify due to random mutations in primer-binding regions. Extremely high proportions of null alleles have been observed for the microsatellite markers developed in <em>C. farreri</em> (56%).<sup><a reveal-id="DST048C33" data-open="DST048C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Application of these markers in population genetic studies can severely distort the estimation of population structure and differentiation, parentage analysis, and assignment tests,<sup><a reveal-id="DST048C34" data-open="DST048C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and may result in misleading conclusions or inferences. In contrast, genetic markers developed from transcriptomic sequences can, to some extent, alleviate this problem, because transcribed regions are usually more conserved than non-transcribed regions. In this study, the average PCR success rate across all assayed individuals was 93%, suggesting that null alleles were indeed present at very low frequencies, as expected for gene-derived markers.</p><p>Remarkably, high population transferability was observed for the developed SNP markers, and the vast majority of which were also neutral markers. Our study, therefore, provides a valuable set of SNP markers that are suitable for population genomic studies. A major benefit of utilizing gene-associated markers is that it is possible to quickly identifying causal genes under natural selection, which usually involves identifying outlier SNP markers showing significantly increased or decreased differentiation among populations compared with neutral expectations.<sup><a reveal-id="DST048C35" data-open="DST048C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Scaling up the number of available gene-associated SNPs to thousands of markers extends the genome coverage, thereby increasing the probability of identifying outlier loci that are in tight linkage disequilibrium with loci under selection.</p>                            <h3 scrollto-destination=5966991 id="5966991" class="section-title" data-legacy-id=s4c>4.3. Cross-species transferability and comparative genomics</h3>
<p>Gene-associated markers generally exhibit higher cross-species transferability in closely related species than markers developed from anonymous genomic sequences. However, to what extent SNP markers developed from one species are transferable to others within the Pectinidae family remains unknown. In this study, about half of the tested markers were polymorphic in at least one of the five scallop species, with the most polymorphic marker transferable to up to four species, indicating the high cross-species transferability of the developed SNP markers. However, marker transferability across the three Pectinidae subfamilies showed no correlation with their phylogenetic relationships to <em>C. farreri</em>. This is somewhat unexpected, suggesting that these transferable SNPs may largely represent ancestral genetic variations that have been preserved differentially among Pectinidae subfamilies. Our study also revealed that SNPs with high transferability (≥3 species) were all synonymous SNPs. This makes sense because synonymous SNPs do not alter the encoded amino acids and thus, are less likely to be removed by purifying selection. Once a polymorphism had arisen, it would be preserved for a long period of time during the evolution of the Pectinidae family.</p><p>The high cross-species transferability together with the high annotation rate (∼73%) of the developed markers hold great promise for conducting comparative genome analysis among molluscs. The recently released Pacific oyster genome represents the best assembled genome currently available for bivalve molluscs.<sup><a reveal-id="DST048C4" data-open="DST048C4" class="link link-ref link-reveal xref-bibr">4</a></sup> It provides a valuable genomic resource not only for oyster genetic and breeding studies, but also for comparative genome analysis among molluscs. One of our ongoing projects is to construct a high-density linkage map for <em>C. farreri</em> based on the markers developed here. In the present study, we made an initial assessment of the usefulness of the developed markers (if they were included in a future linkage map) for comparative genome analysis between the two bivalve species. Of the annotated markers, 69% could be anchored onto the Pacific oyster genome, and 250 markers could be linked to very large oyster scaffolds that are longer than 500 kb and matched by at least five markers. Therefore, once the high-density map is constructed, these markers would help identify conserved genomic regions (i.e. synteny blocks) between the two species.</p>                            <h3 scrollto-destination=5966994 id="5966994" class="section-title" data-legacy-id=s4d>4.4. Large-scale marker association analysis of growth traits</h3>
<p>Identification of major genes responsible for growth traits is highly desirable in scallop breeding programmes for the purpose of genetic improvement. Gene-associated markers are valuable tools for fulfilling such task, because they have great potential for quickly identifying causal genes underlying the trait of interest. The extensive marker set generated by this study provides an unprecedented opportunity to conduct a large-scale association analysis of growth traits in <em>C. farreri</em>. Using a selective genotyping approach, five growth-related markers were identified and confirmed in two <em>C. farreri</em> populations, where allele frequencies were associated with variation in growth trait. Unfortunately, four of these markers are currently uncharacterized, and therefore, their putative roles in growth regulation cannot be inferred. The marker C7493S233_CT was the only marker with annotation information (<em>SDF4</em>, 45 kDa calcium-binding protein). This gene belongs to the calcium-binding protein family, a large group of proteins that can regulate a variety of cellular processes including cell division, differentiation, motility, and apoptosis.<sup><a reveal-id="DST048C36" data-open="DST048C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Mutations in calcium-binding proteins have been associated with variation in growth traits, as demonstrated in Chinese cattle.<sup><a reveal-id="DST048C37" data-open="DST048C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Currently, it is largely unknown which genes or genetic loci are involved in scallop growth regulation. Our study offers the first report of candidate genes responsible for scallop growth regulation and provides markers for further genetic improvement of <em>C. farreri</em> in breeding programmes.</p>                            <h3 scrollto-destination=5966996 id="5966996" class="section-title" data-legacy-id=s4e>4.5. Conclusions</h3>
<p>We developed for the first time large-scale gene-associated SNP markers for a bivalve mollusc, which currently represents the largest gene-associated marker collection in the phylum Mollusca. The properties of high polymorphism within and across populations, low frequency of null alleles, neutrality, and high cross-species transferability make these markers highly valuable for population genomic, comparative genomic, and genome-wide association studies.</p>                            <h2 scrollto-destination=5966998 id="5966998" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5967000 id="5967000" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was financially supported and provided by the National Natural Science Foundation of China (31130054), National Basic Research Program of China (2010CB126406), National High Technology Research and Development Program of China (2012AA10A401, 2012AA10A402, and 2012AA10A405) and Natural Science Foundation for Distinguished Young Scholars of Shandong Province (JQ201308).</p>                            <h2 scrollto-destination=5967002 id="5967002" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are very grateful to Xunshan Aquatic Group Corporation (Shandong Province, China) for the help in maintaining and sampling scallop materials.</p>                            <h2 scrollto-destination=5967004 id="5967004" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST048C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haszprunar</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Schander</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Halanych</div> <div class="given-names">K.M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Ponder</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Lindberg</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">Relationships of higher molluscan taxa</div>, <div class="source">Phylogeny and Evolution of the Mollusca</div> , <div class="year">2008</div><div class="publisher-loc">Berkeley</div><div class="publisher-name">University of California Press</div>(pg. <div class="fpage">19</div>-<div class="lpage">32</div>)</div></div></div></div><div content-id="DST048C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walters</div> <div class="given-names">E.T.</div></div>,  <div class="name"><div class="surname">Moroz</div> <div class="given-names">L.L.</div></div>. </div><div class="article-title">Molluscanmemory of injury: evolutionary insights intochronic pain and neurological disorders</div>, <div class="source">Brain Behav. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">206</div>-<div class="lpage">18</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1159/000258667" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1159/000258667</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molluscanmemory%20of%20injury%3A%20evolutionary%20insights%20intochronic%20pain%20and%20neurological%20disorders&amp;author=E.T.%20Walters&amp;author=L.L.%20Moroz&amp;publication_year=2009&amp;journal=Brain%20Behav.%20Evol.&amp;volume=74&amp;pages=206-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000258667" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000258667" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000258667"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20029184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Talmage</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Gobler</div> <div class="given-names">C.J.</div></div>. </div><div class="article-title">Effects of past, present, and future ocean carbon dioxide concentrations on the growth and survival of larval shellfish</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">17246</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0913804107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0913804107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20past%2C%20present%2C%20and%20future%20ocean%20carbon%20dioxide%20concentrations%20on%20the%20growth%20and%20survival%20of%20larval%20shellfish&amp;author=S.C.%20Talmage&amp;author=C.J.%20Gobler&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=17246-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0913804107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0913804107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0913804107"> </span></div> </div></div></div></div></div><div content-id="DST048C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">The oyster genome reveals stress adaptation and complexity of shell formation</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">490</div> (pg. <div class="fpage">49</div>-<div class="lpage">54</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature11413" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature11413</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20oyster%20genome%20reveals%20stress%20adaptation%20and%20complexity%20of%20shell%20formation&amp;author=G.%20Zhang&amp;author=X.%20Fang&amp;author=X.%20Guo&amp;publication_year=2011&amp;journal=Nature&amp;volume=490&amp;pages=49-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11413" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11413" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11413"> </span></div> </div></div></div></div></div><div content-id="DST048C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takeuchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kawashima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Koyanagi</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Draft genome of the pearl oyster <em>Pinctada fucata</em>: a platform for understanding bivalve biology</div>, <div class="source">DNA Res.</div> , <div class="year">2012</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">117</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dss005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dss005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20of%20the%20pearl%20oyster%20Pinctada%20fucata%3A%20a%20platform%20for%20understanding%20bivalve%20biology&amp;author=T.%20Takeuchi&amp;author=T.%20Kawashima&amp;author=R.%20Koyanagi&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=117-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22315334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ekblom</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Galindo</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Applications of next generation sequencing in molecular ecology of non-model organisms</div>, <div class="source">Heredity</div> , <div class="year">2011</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">1</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/hdy.2010.152" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/hdy.2010.152</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20next%20generation%20sequencing%20in%20molecular%20ecology%20of%20non-model%20organisms&amp;author=R.%20Ekblom&amp;author=J.%20Galindo&amp;publication_year=2011&amp;journal=Heredity&amp;volume=107&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2010.152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2010.152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2010.152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21139633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Franchini</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">van der Merwe</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Roodt-Wilding</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Transcriptome characterization of the South African Abalone <em>Haliotis midae</em> using sequencing-by-synthesis</div>, <div class="source">BMC Res. Notes</div> , <div class="year">2011</div>, vol. <div class="volume">4</div> pg. <div class="fpage">59</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1756-0500-4-59" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1756-0500-4-59</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20characterization%20of%20the%20South%20African%20Abalone%20Haliotis%20midae%20using%20sequencing-by-synthesis&amp;author=P.%20Franchini&amp;author=M.%20van%20der%20Merwe&amp;author=R.%20Roodt-Wilding&amp;publication_year=2011&amp;journal=BMC%20Res.%20Notes&amp;volume=4&amp;pages=59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-4-59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-4-59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-4-59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21396099" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hou</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Transcriptome sequencing and <em>de novo</em> analysis for Yesso scallop (<em>Patinopecten yessoensis</em>) using 454 GS Flx</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e21560</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0021560" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0021560</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20and%20de%20novo%20analysis%20for%20Yesso%20scallop%20(Patinopecten%20yessoensis)%20using%20454%20GS%20Flx&amp;author=R.%20Hou&amp;author=Z.%20Bao&amp;author=S.%20Wang&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e21560" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0021560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0021560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0021560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21720557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">N.G.</div></div>,  <div class="name"><div class="surname">Goetz</div> <div class="given-names">F.E.</div></div>, et al. </div><div class="article-title">Resolving the evolutionary relationship of molluscs with phylogenomic tools</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">480</div> (pg. <div class="fpage">364</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature10526" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature10526</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resolving%20the%20evolutionary%20relationship%20of%20molluscs%20with%20phylogenomic%20tools&amp;author=S.A.%20Smith&amp;author=N.G.%20Wilson&amp;author=F.E.%20Goetz&amp;publication_year=2011&amp;journal=Nature&amp;volume=480&amp;pages=364-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10526" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10526" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10526"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22031330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sadamoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Okada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kenmoku</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Toyota</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Asakawa</div> <div class="given-names">Y.</div></div>. </div><div class="article-title"><em>De novo</em> sequencing and transcriptome analysis of the central nervous system of mollusk <em>Lymnaea stagnalis</em> by deep RAN sequencing</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e42546</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0042546" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0042546</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20sequencing%20and%20transcriptome%20analysis%20of%20the%20central%20nervous%20system%20of%20mollusk%20Lymnaea%20stagnalis%20by%20deep%20RAN%20sequencing&amp;author=H.%20Sadamoto&amp;author=H.%20Takahashi&amp;author=T.%20Okada&amp;author=H.%20Kenmoku&amp;author=M.%20Toyota&amp;author=Y.%20Asakawa&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e42546" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0042546" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0042546" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0042546"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22870333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Mao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Transcriptome analysis of the <em>Octopus vulgaris</em> central nervous system</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e40320</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0040320" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0040320</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20the%20Octopus%20vulgaris%20central%20nervous%20system&amp;author=X.%20Zhang&amp;author=Y.%20Mao&amp;author=Z.%20Huang&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e40320" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0040320" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0040320" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0040320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22768275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reed</div> <div class="given-names">G.H.</div></div>,  <div class="name"><div class="surname">Kent</div> <div class="given-names">J.O.</div></div>,  <div class="name"><div class="surname">Wittwer</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">High-resolution DNA melting analysis for simple and efficient molecular diagnostics</div>, <div class="source">Pharmacogenomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">597</div>-<div class="lpage">608</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2217/14622416.8.6.597" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2217/14622416.8.6.597</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20DNA%20melting%20analysis%20for%20simple%20and%20efficient%20molecular%20diagnostics&amp;author=G.H.%20Reed&amp;author=J.O.%20Kent&amp;author=C.T.%20Wittwer&amp;publication_year=2007&amp;journal=Pharmacogenomics&amp;volume=8&amp;pages=597-608" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2217/14622416.8.6.597" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2217%2F14622416.8.6.597" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2217%2F14622416.8.6.597"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17559349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Taylor</div> <div class="given-names">C.F.</div></div>. </div><div class="article-title">Mutation scanning using high-resolution melting</div>, <div class="source">Biochem. Soc. Trans.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">433</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1042/BST0370433" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1042/BST0370433</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20scanning%20using%20high-resolution%20melting&amp;author=C.F.%20Taylor&amp;publication_year=2009&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=37&amp;pages=433-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST0370433" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST0370433" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST0370433"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19290876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Meyer</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Matz</div> <div class="given-names">M.V.</div></div>. </div><div class="article-title">Construction of a high-resolution genetic linkage map and comparative genome analysis for the reef-building coral <em>Acropora millepora</em></div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R126</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2009-10-11-r126" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2009-10-11-r126</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-resolution%20genetic%20linkage%20map%20and%20comparative%20genome%20analysis%20for%20the%20reef-building%20coral%20Acropora%20millepora&amp;author=S.%20Wang&amp;author=L.%20Zhang&amp;author=E.%20Meyer&amp;author=M.V.%20Matz&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R126" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-11-r126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-11-r126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-11-r126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19900279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ujino-Ihara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Taguchi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Moriguchi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsumura</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">An efficient method for developing SNP markers based on EST data combined with high resolution melting (HRM) analysis</div>, <div class="source">BMC Res. Notes</div> , <div class="year">2010</div>, vol. <div class="volume">3</div> pg. <div class="fpage">51</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1756-0500-3-51" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1756-0500-3-51</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20method%20for%20developing%20SNP%20markers%20based%20on%20EST%20data%20combined%20with%20high%20resolution%20melting%20(HRM)%20analysis&amp;author=T.%20Ujino-Ihara&amp;author=Y.%20Taguchi&amp;author=Y.%20Moriguchi&amp;author=Y.%20Tsumura&amp;publication_year=2010&amp;journal=BMC%20Res.%20Notes&amp;volume=3&amp;pages=51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-3-51" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-3-51" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-3-51"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20193087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hou</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Transcriptome sequencing of Zhikong scallop (<em>Chlamys farreri</em>) and comparative transcriptomic analysis with Yesso scallop (<em>Patinopecten yessoensis</em>)</div>, <div class="source">PLoS One</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e63927</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0063927" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0063927</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20Zhikong%20scallop%20(Chlamys%20farreri)%20and%20comparative%20transcriptomic%20analysis%20with%20Yesso%20scallop%20(Patinopecten%20yessoensis)&amp;author=S.%20Wang&amp;author=R.%20Hou&amp;author=Z.%20Bao&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e63927" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0063927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0063927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0063927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23667690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Development of 44 gene-based SNP markers in Zhikong scallop, <em>Chlamys farreri</em></div>, <div class="source">Conserv. Genet. Resour.</div> , <div class="year">2011</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">659</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s12686-011-9427-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s12686-011-9427-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%2044%20gene-based%20SNP%20markers%20in%20Zhikong%20scallop%2C%20Chlamys%20farreri&amp;author=G.%20Jiang&amp;author=J.%20Li&amp;author=L.%20Li&amp;author=L.%20Zhang&amp;author=Z.%20Bao&amp;publication_year=2011&amp;journal=Conserv.%20Genet.%20Resour.&amp;volume=3&amp;pages=659-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12686-011-9427-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12686-011-9427-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12686-011-9427-2"> </span></div> </div></div></div></div></div><div content-id="DST048C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Characterization of 38 EST-derived SNP markers in Zhikong scallop (<em>Chlamys farreri</em>) and their cross-species utility in Yesso scallop (<em>Patinopecten yessoensis</em>)</div>, <div class="source">Conserv. Genet. Resour.</div> , <div class="year">2012</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">747</div>-<div class="lpage">53</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s12686-012-9636-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s12686-012-9636-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%2038%20EST-derived%20SNP%20markers%20in%20Zhikong%20scallop%20(Chlamys%20farreri)%20and%20their%20cross-species%20utility%20in%20Yesso%20scallop%20(Patinopecten%20yessoensis)&amp;author=X.%20Wang&amp;author=X.%20Hu&amp;author=J.%20Li&amp;publication_year=2012&amp;journal=Conserv.%20Genet.%20Resour.&amp;volume=4&amp;pages=747-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12686-012-9636-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12686-012-9636-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12686-012-9636-3"> </span></div> </div></div></div></div></div><div content-id="DST048C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3, a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.9.9.868" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.9.9.868</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%2C%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Vosman</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">van der Linden</div> <div class="given-names">C.G.</div></div>,  <div class="name"><div class="surname">Leunissen</div> <div class="given-names">J.A.M.</div></div>. </div><div class="article-title">QualitySNP, a pipeline for detecting single nucleotide polymorphisms and insertions/deletions in EST data from diploid and polyploidy species</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">438</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-7-438" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-7-438</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QualitySNP%2C%20a%20pipeline%20for%20detecting%20single%20nucleotide%20polymorphisms%20and%20insertions%2Fdeletions%20in%20EST%20data%20from%20diploid%20and%20polyploidy%20species&amp;author=J.F.%20Tang&amp;author=B.%20Vosman&amp;author=R.E.%20Voorrips&amp;author=C.G.%20van%20der%20Linden&amp;author=J.A.M.%20Leunissen&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=438" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-438" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-438" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-438"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17029635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Fine-scale population genetic structure of Zhikong scallop (<em>Chlamys farreri</em>), do local marine currents drive geographical differentiation?</div>, <div class="source">Mar. Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">223</div>-<div class="lpage">35</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10126-008-9138-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10126-008-9138-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine-scale%20population%20genetic%20structure%20of%20Zhikong%20scallop%20(Chlamys%20farreri)%2C%20do%20local%20marine%20currents%20drive%20geographical%20differentiation%3F&amp;author=A.%20Zhan&amp;author=J.%20Hu&amp;author=X.%20Hu&amp;publication_year=2009&amp;journal=Mar.%20Biotechnol.&amp;volume=11&amp;pages=223-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-008-9138-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-008-9138-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-008-9138-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18766401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yeh</div> <div class="given-names">F.C.</div></div>,  <div class="name"><div class="surname">Boyle</div> <div class="given-names">T.J.B.</div></div>. </div><div class="article-title">Population genetic analysis of co-dominant and dominant markers and quantitative traits</div>, <div class="source">Belg. J. Bot.</div> , <div class="year">1997</div>, vol. <div class="volume">129</div> pg. <div class="fpage">157</div> </div></div></div></div><div content-id="DST048C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raymond</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rousset</div> <div class="given-names">F.</div></div>. </div><div class="article-title">GENEPOP (Version 1.2), population genetics software for exact tests and ecumenicism</div>, <div class="source">J. Hered.</div> , <div class="year">1995</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">248</div>-<div class="lpage">9</div>)</div></div></div></div><div content-id="DST048C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Götz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">García-Gómez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Talón</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Blast2GO, A universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bti610" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bti610</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%2C%20A%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=J.M.%20Garc%C3%ADa-G%C3%B3mez&amp;author=J.%20Terol&amp;author=M.%20Tal%C3%B3n&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moriya</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Okuda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yoshizawa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">KAAS, an automatic genome annotation and pathway reconstruction server</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W182</div>-<div class="lpage">185</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm321" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm321</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KAAS%2C%20an%20automatic%20genome%20annotation%20and%20pathway%20reconstruction%20server&amp;author=Y.%20Moriya&amp;author=M.%20Itoh&amp;author=S.%20Okuda&amp;author=A.%20Yoshizawa&amp;author=M.%20Kanehisa&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W182-185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goldstein</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Weale</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Population genomics, linkage disequilibrium holds the key</div>, <div class="source">Curr. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">R576</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0960-9822(01)00348-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0960-9822(01)00348-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20genomics%2C%20linkage%20disequilibrium%20holds%20the%20key&amp;author=D.B.%20Goldstein&amp;author=M.E.%20Weale&amp;publication_year=2001&amp;journal=Curr.%20Biol.&amp;volume=11&amp;pages=R576-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(01)00348-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(01)00348-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(01)00348-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11509259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Puslednik</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Serb</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Molecular phylogenetics of the Pectinidae (Mollusca, Bivalvia) and effect of increased taxon sampling and outgroup selection on tree topology</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">2008</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">1178</div>-<div class="lpage">88</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.ympev.2008.05.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.ympev.2008.05.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20phylogenetics%20of%20the%20Pectinidae%20(Mollusca%2C%20Bivalvia)%20and%20effect%20of%20increased%20taxon%20sampling%20and%20outgroup%20selection%20on%20tree%20topology&amp;author=L.%20Puslednik&amp;author=J.M.%20Serb&amp;publication_year=2008&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=48&amp;pages=1178-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2008.05.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2008.05.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2008.05.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18579415" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Margulies</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Egholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Altman</div> <div class="given-names">W.E.</div></div>, et al. </div><div class="article-title">Genome sequencing in microfabricated high-density picolitre reactors</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">376</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;author=M.%20Egholm&amp;author=W.E.%20Altman&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huse</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Huber</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Morrison</div> <div class="given-names">H.G.</div></div>,  <div class="name"><div class="surname">Sogin</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Welch</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Accuracy and quanlity of massively parallel DNA pyrosequencing</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R143</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2007-8-7-r143" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2007-8-7-r143</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accuracy%20and%20quanlity%20of%20massively%20parallel%20DNA%20pyrosequencing&amp;author=S.M.%20Huse&amp;author=J.A.%20Huber&amp;author=H.G.%20Morrison&amp;author=M.L.%20Sogin&amp;author=D.M.%20Welch&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R143" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-7-r143" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-7-r143" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-7-r143"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17659080" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shao</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Boltz</div> <div class="given-names">V.F.</div></div>,  <div class="name"><div class="surname">Spindler</div> <div class="given-names">J.E.</div></div>, et al. </div><div class="article-title">Analysis of 454 sequencing error rate, error sources, and artifact recombination for detection of low-frequency drug resistance mutations in HIV-1 DNA</div>, <div class="source">Retrovirology</div> , <div class="year">2013</div>, vol. <div class="volume">10</div> pg. <div class="fpage">18</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1742-4690-10-18" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1742-4690-10-18</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20454%20sequencing%20error%20rate%2C%20error%20sources%2C%20and%20artifact%20recombination%20for%20detection%20of%20low-frequency%20drug%20resistance%20mutations%20in%20HIV-1%20DNA&amp;author=W.%20Shao&amp;author=V.F.%20Boltz&amp;author=J.E.%20Spindler&amp;publication_year=2013&amp;journal=Retrovirology&amp;volume=10&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1742-4690-10-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1742-4690-10-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1742-4690-10-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23402264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Next-generation transcriptome assembly</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">671</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg3068" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg3068</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20transcriptome%20assembly&amp;author=J.A.%20Martin&amp;author=Z.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=671-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21897427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sauvage</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bierne</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Lapegue</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Boudry</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Single nucleotide polymorphisms and their relationship to codon usage bias in the Pacific oyster <em>Crassostrea gigas</em></div>, <div class="source">Gene</div> , <div class="year">2007</div>, vol. <div class="volume">406</div> (pg. <div class="fpage">13</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gene.2007.05.011" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gene.2007.05.011</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphisms%20and%20their%20relationship%20to%20codon%20usage%20bias%20in%20the%20Pacific%20oyster%20Crassostrea%20gigas&amp;author=C.%20Sauvage&amp;author=N.%20Bierne&amp;author=S.%20Lapegue&amp;author=P.%20Boudry&amp;publication_year=2007&amp;journal=Gene&amp;volume=406&amp;pages=13-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2007.05.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.05.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2007.05.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17616269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Construction of microsatellite-based linkage maps and identification of size-related quantitative trait loci for Zhikong scallop (<em>Chlamys farreri</em>)</div>, <div class="source">Anim. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">821</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-2052.2009.01920.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-2052.2009.01920.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20microsatellite-based%20linkage%20maps%20and%20identification%20of%20size-related%20quantitative%20trait%20loci%20for%20Zhikong%20scallop%20(Chlamys%20farreri)&amp;author=A.%20Zhan&amp;author=J.%20Hu&amp;author=X.%20Hu&amp;publication_year=2009&amp;journal=Anim.%20Genet.&amp;volume=40&amp;pages=821-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2009.40.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2009.40.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2009.40.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19515089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Uchiyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Han</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Null alleles in microsatellite markers</div>, <div class="source">Biodiversity Sci.</div> , <div class="year">2013</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">117</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Null%20alleles%20in%20microsatellite%20markers&amp;author=Y.%20Wen&amp;author=K.%20Uchiyama&amp;author=W.%20Han&amp;publication_year=2013&amp;journal=Biodiversity%20Sci.&amp;volume=21&amp;pages=117-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3724/SP.J.1003.2013.10133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3724%2FSP.J.1003.2013.10133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3724%2FSP.J.1003.2013.10133"> </span></div> </div></div></div></div></div><div content-id="DST048C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Phillips</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Cresko</div> <div class="given-names">W.A.</div></div>. </div><div class="article-title">Using population genomics to detect selection in natural populations, key concepts and methodological considerations</div>, <div class="source">Int. J. Plant Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">1059</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1086/656306" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1086/656306</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20population%20genomics%20to%20detect%20selection%20in%20natural%20populations%2C%20key%20concepts%20and%20methodological%20considerations&amp;author=P.A.%20Hohenlohe&amp;author=P.C.%20Phillips&amp;author=W.A.%20Cresko&amp;publication_year=2010&amp;journal=Int.%20J.%20Plant%20Sci.&amp;volume=171&amp;pages=1059-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/656306" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F656306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F656306"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21218185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST048C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carafoli</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Klee</div> <div class="given-names">C.B.</div></div>. </div>, <div class="source">Calcium as a cellular regulator</div> <div class="publisher-loc">Oxford</div><div class="publisher-name">Oxford University Press</div></div></div></div></div><div content-id="DST048C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST048C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lei</div> <div class="given-names">C.Z.</div></div>. </div><div class="article-title">Novel SNPs of the bovine <em>NUCB2</em> gene and their association with growth traits in three native Chinese cattle breeds</div>, <div class="source">Mol. Biol. Rep.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">541</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11033-009-9732-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11033-009-9732-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20SNPs%20of%20the%20bovine%20NUCB2%20gene%20and%20their%20association%20with%20growth%20traits%20in%20three%20native%20Chinese%20cattle%20breeds&amp;author=F.%20Li&amp;author=H.%20Chen&amp;author=C.Z.%20Lei&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Rep.&amp;volume=37&amp;pages=541-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11033-009-9732-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11033-009-9732-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11033-009-9732-y"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Shoji Tsuji</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=403825&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst048-dst048supp_table2.pdf?Expires=2147483647&amp;Signature=HlCjquFAg~fbWuem6ySrBMCivLRur4JAsqRdr4fCKtA9SuRN5jvXXdnCC9POCVT3j6HdXGRhbCpjY2UuAKguEeN1~nlAzPEMJ1BeyJ3EdXr8Uo7cAviu1If1YP0I-rF5maAzsD4m~rMC65K2TLHYzK6zayZfnRtgJFV4kx8yzcuLpUDNldx1VUs1J~VpAqRowINQbjwieLSKbgdyC-vTdo0gqT1AI89lI0kxTvBreIBxJbiUbfhxo902P-s3XbwdUiJqJMMoIn~NdOeaZ0H2vQQl-QloPXJiAHNe4eeHkNrviqLuqcYQdiQ5Y2sDC7nN-WjlxzIuxQ2kH0oY0yKKgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst048-dst048supp_table1.xls?Expires=2147483647&amp;Signature=mbsRNOcfiSOSf8d0mCgDva61S8GHHoCoCaaimlJY1wQ3v5-LetZ9tbIreSrcjZN40xjNR3hTEApCpQp~UmeC4h5RtO2EAZdkpLZpW6Tu-kgo91VzA0YxHYVqn6QDEGkBmOB4CKEYr9NuBmLL--7IiF0OB281yiISXhBgnWHYSAwwGJlOVPgpUXAHHDcdAcNsQa2Ol5f8JhSYsQWJ1ZZ7d7jXl7CqcWXWQpGSZn~LjkrEBxA1tyeEz74h1qIQw6pYcEVS6rSNpBznln9E1-jmtKu~YcIEC6LIlU7etRKebLKV61SVqKcZmXp6cwaT3lVbE7SSQHJUQl0lwMxSZM933Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst048/2/dst048-dst048supp.doc?Expires=2147483647&amp;Signature=L3xtFfUmLcnqxIuHfVL3j9IlXCMp4uMu~tHW~0wZ2~gGfMmEBJtcgwkmgQkWt6AkFAWAWLPQsVDtQxmaDSV1LX4J-xHWYfAyihTPxDWFqYtHu525HHBqCbBNd4jvcIfH4CdP-k2IX6vpyQFssKL1j7olKZhXb0Gb1spL0iSKaEjZukZ3mmivU1U-ADVaIVcLy1xqfJ3JPnYwjL8PnsBnm9CYgl0MvArno3Fdp03CdIreOZIOXVf-FoWi0FCl-jHtlNHOZwdcIEDHmQJL6VP57UDYQ3O1bBbYgXUMk~bEZJBJmng5yqz91j04McHjhv4n5YTFja2uxBvRqKc~9TN6LA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">271</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst048"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="403825"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f2%2f183%2f403825">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897500007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F2%2F183%2F403825" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/403825'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Large-Scale Development of Gene-Associated Single-Nucleotide Polymorphism Markers for Molluscan Population Genomic, Comparative Genomic, and Genome-Wide Association Studies" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst048");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f183%2f403825">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

