

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/5/357/628389","@type":"ScholarlyArticle","name":"High-Throughput SNP Discovery and Genotyping for Constructing a Saturated Linkage Map of Chickpea ( Cicer arietinum  L.)","datePublished":"2012-08-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2012-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss018","keywords":["chickpea","SNP","linkage map","genotyping","NGS"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/5/357/628389","author":[{"name":"Gaur, Rashmi","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Azam, Sarwar","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Jeena, Ganga","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Khan, Aamir Waseem","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Choudhary, Shalu","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Jain, Mukesh","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Yadav, Gitanjali","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Tyagi, Akhilesh K.","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Chattopadhyay, Debasis","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"},{"name":"Bhatia, Sabhyata","affiliation":"National Institute of Plant Genome Research ,  Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067 ,  India","@type":"Person"}],"description":"Abstract.  The present study reports the large-scale discovery of genome-wide single-nucleotide polymorphisms (SNPs) in chickpea, identified mainly through the ","pageStart":"357","pageEnd":"373","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"High-Throughput SNP Discovery and Genotyping for Constructing a Saturated Linkage Map of Chickpea ( Cicer arietinum  L.)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-Throughput SNP Discovery and Genotyping for Constructing a Saturated Linkage Map of Chickpea ( Cicer arietinum  L.)" />
    <meta property="og:description" content="Abstract.  The present study reports the large-scale discovery of genome-wide single-nucleotide polymorphisms (SNPs) in chickpea, identified mainly through the " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/5/357/628389" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-08-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Gaur, Rashmi" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Azam, Sarwar" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Jeena, Ganga" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Khan, Aamir Waseem" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Choudhary, Shalu" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Jain, Mukesh" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Yadav, Gitanjali" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Tyagi, Akhilesh K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Chattopadhyay, Debasis" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_author" content="Bhatia, Sabhyata" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067, India" /><meta name="citation_title" content="High-Throughput SNP Discovery and Genotyping for Constructing a Saturated Linkage Map of Chickpea (Cicer arietinum L.)" /><meta name="citation_firstpage" content="357" /><meta name="citation_lastpage" content="373" /><meta name="citation_doi" content="10.1093/dnares/dss018" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2012/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Informativity assessment for biallelic single nucleotide polymorphisms; Electrophoresis;  citation_year=1999;  citation_volume=20;  citation_pages=1676-81; " /><meta name="citation_reference" content="citation_title=Comparison of microsatellites, single-nucleotide polymorphisms (SNPs) and composite markers derived from SNPs in linkage analysis; BMC Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=S29" /><meta name="citation_reference" content="citation_title=An estimate of unique DNA sequence heterozygosity in the human genome; Hum. Genet.;  citation_year=1985;  citation_volume=69;  citation_pages=201-5; " /><meta name="citation_reference" content="citation_title=Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome; Science;  citation_year=1998;  citation_volume=280;  citation_pages=1077-82; " /><meta name="citation_reference" content="citation_title=Comparative SNP diversity among four Eucalyptus species for genes from secondary metabolite biosynthetic pathways; BMC Genomics;  citation_year=2009;  citation_volume=24;  citation_pages=452" /><meta name="citation_reference" content="citation_title=Extensive chromosome homoeology among Brassiceae species were revealed by comparative genetic mapping with high-density EST-based SNP markers in radish (Raphanus sativus L.); DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=401-11; " /><meta name="citation_reference" content="citation_title=Genome-wide DNA polymorphisms in elite indica rice inbreds discovered by whole-genome sequencing; Plant Biotech. J.;  citation_year=2012;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Large-scale discovery of gene-enriched SNPs; Plant Genome;  citation_year=2009;  citation_volume=2;  citation_pages=121-33; " /><meta name="citation_reference" content="citation_title=A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis; Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=685-96; " /><meta name="citation_reference" content="citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines; Nature;  citation_year=2010;  citation_volume=465;  citation_pages=627-31; " /><meta name="citation_reference" content="citation_title=A dense single-nucleotide polymorphism-based genetic linkage map of grapevine (Vitis vinifera L.) anchoring Pinot Noir bacterial artificial chromosome contigs; Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=2637-50; " /><meta name="citation_reference" content="citation_title=Enhancing genetic mapping of complex genomes through the design of highly-multiplexed SNP arrays: application to the large and unsequenced genomes of white spruce and black spruce; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=21" /><meta name="citation_reference" content="citation_title=A consensus genetic map of cowpea [Vigna unguiculata (L) Walp.] and synteny based on EST-derived SNPs; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=18159-64; " /><meta name="citation_reference" content="citation_title=Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=468" /><meta name="citation_reference" content="citation_title=Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (Triticum aestivum L.); Plant Biotech. J.;  citation_year=2011;  citation_volume=9;  citation_pages=1086-99; " /><meta name="citation_reference" content="citation_title=Development and implementation of a highly multiplexed SNP array for genetic mapping in maritime pine and comparative mapping with loblolly pine; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=368" /><meta name="citation_reference" content="citation_title=SNP discovery via 454 transcriptome sequencing; Plant Journal;  citation_year=2007;  citation_volume=51;  citation_pages=910-8; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery through deep resequencing of a reduced representation library to anchor and orient scaffolds in the soybean whole genome sequence; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=38" /><meta name="citation_reference" content="citation_title=Genome-wide SNP discovery in tetraploid alfalfa using 454 sequencing and high resolution melting analysis; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=350" /><meta name="citation_reference" content="citation_title=High-throughput SNP genotyping in the highly heterozygous genome of Eucalyptus: assay success, polymorphism and transferability across species; BMC Plant Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=65" /><meta name="citation_reference" content="citation_title=Discovery of genome-wide DNA polymorphisms in a landrace cultivar of japonica rice by whole-genome sequencing; Plant Cell Physiol.;  citation_year=2011;  citation_volume=52;  citation_pages=274-82; " /><meta name="citation_reference" content="citation_title=SNP discovery and development of a high-density genotyping array for sunflower; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e29814" /><meta name="citation_reference" content="citation_title=Highly parallel SNP genotyping; Cold Spring Harb. Symp. Quant. Biol.;  citation_year=2003;  citation_volume=68;  citation_pages=69-78; " /><meta name="citation_reference" content="citation_title=BeadArray technology: enabling an accurate, cost-effective approach to high-throughput genotyping; Biotechniques;  citation_year=2002;  citation_volume=32;  citation_pages=S56" /><meta name="citation_reference" content="citation_title=Highly parallel genomic assays; Nat. Rev. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=632-44; " /><meta name="citation_reference" content="citation_title=SNPstream UHT: ultra-high throughput SNP genotyping for pharmacogenomics and drug discovery; Biotechniques;  citation_year=2002;  citation_volume=32;  citation_pages=S70-7; " /><meta name="citation_reference" content="citation_title=Parallel genotyping of over 10,000 SNPs using a one-primer assay on a high-density oligonucleotide array; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=414-25; " /><meta name="citation_reference" content="citation_title=Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays; Nat. Methods;  citation_year=2004;  citation_volume=1;  citation_pages=109-11; " /><meta name="citation_reference" content="citation_title=A genome-wide SNP panel for mapping and association studies in the rat; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=95" /><meta name="citation_reference" content="citation_title=Toward genome-wide SNP genotyping; Nat. Rev. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=S5-10; " /><meta name="citation_reference" content="citation_title=High-throughput genotyping with the GoldenGate assay in the complex genome of soybean; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=945-52; " /><meta name="citation_reference" content="citation_title=Development and implementation of high-throughput SNP genotyping in barley; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=582" /><meta name="citation_reference" content="citation_title=High-throughput SNP genotyping with the GoldenGate assay in maize; Mol. Breed.;  citation_year=2010;  citation_volume=25;  citation_pages=441-51; " /><meta name="citation_reference" content="citation_title=High-throughput genotyping and mapping of single nucleotide polymorphisms in loblolly pine (Pinus taeda L.); Tree Genet. Genomes;  citation_year=2009;  citation_volume=5;  citation_pages=225-34; " /><meta name="citation_reference" content="citation_title=Rapid genomic characterization of the genus Vitis; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e8219" /><meta name="citation_reference" content="citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=961-7; " /><meta name="citation_reference" content="citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=159-62; " /><meta name="citation_reference" content="citation_title=Whole-genome nucleotide diversity, recombination, and linkage disequilibrium in the model legume Medicago truncatula; Proc. Natl Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=E864-70; " /><meta name="citation_reference" content="citation_title=Genome comparisons reveal a dominant mechanism of chromosome number reduction in grasses and accelerated genome evolution in Triticeae; Proc. Natl Acad. Sci., USA;  citation_year=2009;  citation_volume=106;  citation_pages=15780-5; " /><meta name="citation_reference" content="citation_title=SNP discovery and linkage map construction in cultivated tomato; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=381-91; " /><meta name="citation_reference" content="citation_title=Comparison of a high-density genetic linkage map to genome features in the model grass Brachypodium distachyon; Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=455-64; " /><meta name="citation_reference" content="citation_title=Development of single nucleotide polymorphism (SNP) markers for use in commercial maize (Zea mays L.) germplasm; Mol. Breed.;  citation_year=2009;  citation_volume=24;  citation_pages=165-76; " /><meta name="citation_reference" content="citation_title=A high resolution genetic map anchoring scaffolds of the sequenced watermelon genome; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e29453" /><meta name="citation_reference" content="citation_title=Nuclear DNA content of some important plant species; Plant Mol. Rep.;  citation_year=1991;  citation_volume=9;  citation_pages=208-18; " /><meta name="citation_reference" content="citation_title=Integration of novel SSR and gene-based SNP marker loci in the chickpea genetic map and establishment of new anchor points with Medicago truncatula genome; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=1415-41; " /><meta name="citation_reference" content="citation_title=Advancing the STMS genomic resources for defining new locations on the intraspecific genetic linkage map of chickpea (Cicer arietinum L.); BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=117" /><meta name="citation_reference" content="citation_title=Development and use of genic molecular markers (GMMs) for construction of a transcript map of chickpea (Cicer arietinum L.); Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=1577-89; " /><meta name="citation_reference" content="citation_title=EST-derived genic molecular markers: development and utilization for generating an advanced transcript map of chickpea; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=1449-62; " /><meta name="citation_reference" content="citation_title=Development of ESTs from chickpea roots and their use in diversity analysis of the Cicer genus; BMC Plant Biol.;  citation_year=2005;  citation_volume=5;  citation_pages=16" /><meta name="citation_reference" content="citation_title=Isolation and analysis of candidate Ascochyta blight defense genes in chickpea, Part I. Generation and analysis of an expressed sequence tag (EST) library; Physiol. Mol. Plant. Pathol.;  citation_year=2005;  citation_volume=66;  citation_pages=192-200; " /><meta name="citation_reference" content="citation_title=A comprehensive resource of drought and salinity responsive ESTs for gene discovery and marker development in chickpea (Cicer arietinum L.); BMC Genomics;  citation_year=2009;  citation_volume=15;  citation_pages=523" /><meta name="citation_reference" content="citation_title=Comparative analysis of genotype dependent expressed sequence tags and stress-responsive transcriptome of chickpea wilt illustrate predicted and unexpected genes and novel regulators of plant immunity; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=415" /><meta name="citation_reference" content="citation_title=De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=53-63; " /><meta name="citation_reference" content="citation_title=Gene discovery and tissue-specific transcriptome analysis in chickpea with massively parallel pyrosequencing and web resource development; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=1661-78; " /><meta name="citation_reference" content="citation_title=Large-scale transcriptome analysis in chickpea (Cicer arietinum L.), an orphan legume crop of the semi-arid tropics of Asia and Africa; Plant Biotech. J.;  citation_year=2011;  citation_volume=9;  citation_pages=922-31; " /><meta name="citation_reference" content="citation_title=Sequence-tagged microsatellite markers for chickpea (Cicer arietinum L.); Genome;  citation_year=1999;  citation_volume=42;  citation_pages=210-7; " /><meta name="citation_reference" content="citation_title=Characterization and mapping of sequence-tagged microsatellite sites in the chickpea (Cicer arietinum L.) genome; Mol. Gen. Genet.;  citation_year=1999;  citation_volume=262;  citation_pages=90-101; " /><meta name="citation_reference" content="citation_title=A linkage map of chickpea (Cicer arietinum L.) genome based on recombinant inbred lines from a C. arietinum times C. reticulatum cross: localization of resistance genes for Fusarium wilt races 4 and 5; Theor. Appl. Genet.;  citation_year=2000;  citation_volume=101;  citation_pages=1155-63; " /><meta name="citation_reference" content="citation_title=Construction of BAC and BIBAC libraries and their applications for generation of SSR markers for genome analysis of chickpea, Cicer arietinum L.; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=492-510; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of sequence tagged microsatellite sites markers in chickpea (Cicer arietinum L.); Mol. Ecol. Notes;  citation_year=2003;  citation_volume=3;  citation_pages=428-30; " /><meta name="citation_reference" content="citation_title=Development of microsatellite markers and analysis of intraspecific genetic variability in chickpea (Cicer arietinum L.); Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=1416-28; " /><meta name="citation_reference" content="citation_title=Development of sequence-tagged microsatellites site markers for chickpea (Cicer arietinum L.); Mol. Ecol. Notes;  citation_year=2006;  citation_volume=6;  citation_pages=93-5; " /><meta name="citation_reference" content="citation_title=Development of chickpea EST-SSR markers and analysis of allelic variation across related species; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=591-608; " /><meta name="citation_reference" content="citation_title=A consensus genetic map of chickpea (Cicer arietinum L.) based on 10 mapping populations; Euphytica;  citation_year=2010;  citation_volume=175;  citation_pages=175-89; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory, Cold Spring Harbor; Molecular Biology of Plants: A Laboratory Course Manual;  citation_year=1985; " /><meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=376-80; " /><meta name="citation_reference" content="citation_title=JoinMap version 4.0: Software for the calculation of genetic linkage maps in experimental population; citation_text=Kyazma B.V., Wageningen, Netherlands;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=The estimation of map distances from recombination values; Ann. Eugenics;  citation_year=1994;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=Detecting polymorphic regions in the Arabidopsis thaliana genome with resequencing microarrays; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=918-929; " /><meta name="citation_reference" content="citation_title=SNP discovery by high-throughput sequencing in soybean; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=469" /><meta name="citation_reference" content="citation_title=Fine definition of the pedigree haplotypes of closely related rice cultivars by means of genome-wide discovery of single-nucleotide polymorphisms; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=267" /><meta name="citation_reference" content="citation_title=Global analysis of allele-specific expression in Arabidopsis thaliana; Genetics;  citation_year=2009;  citation_volume=182;  citation_pages=943-54; " /><meta name="citation_reference" content="citation_title=Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=105-11; " /><meta name="citation_reference" content="citation_title=Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=1053-9; " /><meta name="citation_reference" content="citation_title=Recent history of artificial outcrossing facilitates whole-genome association mapping in elite inbred crop varieties; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=18656-61; " /><meta name="citation_reference" content="citation_title=An application of high-throughput SNP genotyping for barley genome mapping and characterization of recombinant chromosome substitution lines; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=119;  citation_pages=613-9; " /><meta name="citation_reference" content="citation_title=SSR-based linkage map with new markers using an intraspecific population of common wheat; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=1042-51; " /><meta name="citation_reference" content="citation_title=A high density barley microsatellite consensus map with 775 SSR loci; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=1091-113; " /><meta name="citation_reference" content="citation_title=A high density genetic map of tobacco (Nicotiana tabacum L.) obtained from large scale microsatellite marker development; Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=219-30; " /><meta name="citation_reference" content="citation_title=An interspecific linkage map of SSR and intronic polymorphism markers in tomato; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=731-9; " /><meta name="citation_reference" content="citation_title=Development of SSR and gene-targeted markers for construction of a framework linkage map of Catharanthus roseus; Ann. Bot. (Lond.);  citation_year=2011;  citation_volume=108;  citation_pages=321-36; " /><meta name="citation_reference" content="citation_title=High-density integrated linkage map based on SSR markers in soybean; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=213-25; " /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla; Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=The genome of the cucumber, Cucumis sativus L.; Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=1275-81; " /><meta name="citation_reference" content="citation_title=The genome of the domesticated apple (Malus x domestica Borkh.); Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=833-9; " /><meta name="citation_reference" content="citation_title=Estimating genome conservation between crop and model legume species; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=15289-94; " /><meta name="citation_reference" content="citation_title=An analysis of synteny of Arachis with Lotus and Medicago sheds new light on the structure, stability and evolution of legume genomes; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=45" /><meta name="citation_reference" content="citation_title=Saturation of an intra-gene pool linkage map: towards a unified consensus linkage map for fine mapping and synteny analysis in common bean; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e28135" /><meta name="citation_reference" content="citation_title=A SNP and SSR based genetic map of Asparagus bean (Vigna unguiculata ssp. sesquipedalis) and comparison with the broader species; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e15952" /><meta name="citation_reference" content="citation_title=Cross-species amplification of Medicago truncatula microsatellites across three major pulse crops; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=1210-7; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/5/357/1750511/dss018.pdf" /><meta name="description" content="Abstract.  The present study reports the large-scale discovery of genome-wide single-nucleotide polymorphisms (SNPs) in chickpea, identified mainly through the " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/5/357/628389" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/5/357/628389" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/5/357/628389");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss018';
    </script>
<title>
	High-Throughput SNP Discovery and Genotyping for Constructing a Saturated Linkage Map of Chickpea (Cicer arietinum L.) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./628389" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BebEATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2F1ciwgUmFzaG1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgUE8gQm94IDEwNTMxLCBOZXcgRGVsaGkgMTEwMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkF6YW0sIFNhcndhciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2gsIEFydW5hIEFzYWYgQWxpIE1hcmcsIFBPIEJveCAxMDUzMSwgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKZWVuYSwgR2FuZ2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBQTyBCb3ggMTA1MzEsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2hhbiwgQWFtaXIgV2FzZWVtIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgUE8gQm94IDEwNTMxLCBOZXcgRGVsaGkgMTEwMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNob3VkaGFyeSwgU2hhbHUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBQTyBCb3ggMTA1MzEsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmFpbiwgTXVrZXNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgUE8gQm94IDEwNTMxLCBOZXcgRGVsaGkgMTEwMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhZGF2LCBHaXRhbmphbGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBQTyBCb3ggMTA1MzEsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVHlhZ2ksIEFraGlsZXNoIEsuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgUE8gQm94IDEwNTMxLCBOZXcgRGVsaGkgMTEwMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoYXR0b3BhZGh5YXksIERlYmFzaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBQTyBCb3ggMTA1MzEsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmhhdGlhLCBTYWJoeWF0YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2gsIEFydW5hIEFzYWYgQWxpIE1hcmcsIFBPIEJveCAxMDUzMSwgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkhpZ2gtVGhyb3VnaHB1dCBTTlAgRGlzY292ZXJ5IGFuZCBHZW5vdHlwaW5nIGZvciBDb25zdHJ1Y3RpbmcgYSBTYXR1cmF0ZWQgTGlua2FnZSBNYXAgb2YgQ2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLikiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM3MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEyLzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mb3JtYXRpdml0eSBhc3Nlc3NtZW50IGZvciBiaWFsbGVsaWMgc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtczsgRWxlY3Ryb3Bob3Jlc2lzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTY3Ni04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBtaWNyb3NhdGVsbGl0ZXMsIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbXMgKFNOUHMpIGFuZCBjb21wb3NpdGUgbWFya2VycyBkZXJpdmVkIGZyb20gU05QcyBpbiBsaW5rYWdlIGFuYWx5c2lzOyBCTUMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1TMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBlc3RpbWF0ZSBvZiB1bmlxdWUgRE5BIHNlcXVlbmNlIGhldGVyb3p5Z29zaXR5IGluIHRoZSBodW1hbiBnZW5vbWU7IEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MjAxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIGlkZW50aWZpY2F0aW9uLCBtYXBwaW5nLCBhbmQgZ2Vub3R5cGluZyBvZiBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGluIHRoZSBodW1hbiBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjgwOyAgY2l0YXRpb25fcGFnZXM9MTA3Ny04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgU05QIGRpdmVyc2l0eSBhbW9uZyBmb3VyIEV1Y2FseXB0dXMgc3BlY2llcyBmb3IgZ2VuZXMgZnJvbSBzZWNvbmRhcnkgbWV0YWJvbGl0ZSBiaW9zeW50aGV0aWMgcGF0aHdheXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTQ1MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBjaHJvbW9zb21lIGhvbW9lb2xvZ3kgYW1vbmcgQnJhc3NpY2VhZSBzcGVjaWVzIHdlcmUgcmV2ZWFsZWQgYnkgY29tcGFyYXRpdmUgZ2VuZXRpYyBtYXBwaW5nIHdpdGggaGlnaC1kZW5zaXR5IEVTVC1iYXNlZCBTTlAgbWFya2VycyBpbiByYWRpc2ggKFJhcGhhbnVzIHNhdGl2dXMgTC4pOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQwMS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgRE5BIHBvbHltb3JwaGlzbXMgaW4gZWxpdGUgaW5kaWNhIHJpY2UgaW5icmVkcyBkaXNjb3ZlcmVkIGJ5IHdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nOyBQbGFudCBCaW90ZWNoLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3BhZ2VzPTEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIGRpc2NvdmVyeSBvZiBnZW5lLWVucmljaGVkIFNOUHM7IFBsYW50IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTIxLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNveWJlYW4gdHJhbnNjcmlwdCBtYXA6IGdlbmUgZGlzdHJpYnV0aW9uLCBoYXBsb3R5cGUgYW5kIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbSBhbmFseXNpczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25fcGFnZXM9Njg1LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkeSBvZiAxMDcgcGhlbm90eXBlcyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBpbmJyZWQgbGluZXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjU7ICBjaXRhdGlvbl9wYWdlcz02MjctMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZGVuc2Ugc2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtLWJhc2VkIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgZ3JhcGV2aW5lIChWaXRpcyB2aW5pZmVyYSBMLikgYW5jaG9yaW5nIFBpbm90IE5vaXIgYmFjdGVyaWFsIGFydGlmaWNpYWwgY2hyb21vc29tZSBjb250aWdzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzY7ICBjaXRhdGlvbl9wYWdlcz0yNjM3LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmhhbmNpbmcgZ2VuZXRpYyBtYXBwaW5nIG9mIGNvbXBsZXggZ2Vub21lcyB0aHJvdWdoIHRoZSBkZXNpZ24gb2YgaGlnaGx5LW11bHRpcGxleGVkIFNOUCBhcnJheXM6IGFwcGxpY2F0aW9uIHRvIHRoZSBsYXJnZSBhbmQgdW5zZXF1ZW5jZWQgZ2Vub21lcyBvZiB3aGl0ZSBzcHJ1Y2UgYW5kIGJsYWNrIHNwcnVjZTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29uc2Vuc3VzIGdlbmV0aWMgbWFwIG9mIGNvd3BlYSBbVmlnbmEgdW5ndWljdWxhdGEgKEwpIFdhbHAuXSBhbmQgc3ludGVueSBiYXNlZCBvbiBFU1QtZGVyaXZlZCBTTlBzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTE4MTU5LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdobHktbXVsdGlwbGV4ZWQgU05QIGdlbm90eXBpbmcgZm9yIGdlbmV0aWMgbWFwcGluZyBhbmQgZ2VybXBsYXNtIGRpdmVyc2l0eSBzdHVkaWVzIGluIHBlYTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdC1zcGVjaWZpYywgc2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGRpc2NvdmVyeSBhbmQgbGlua2FnZSBhbmFseXNpcyBpbiBoZXhhcGxvaWQgYnJlYWQgd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtIEwuKTsgUGxhbnQgQmlvdGVjaC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEwODYtOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBpbXBsZW1lbnRhdGlvbiBvZiBhIGhpZ2hseSBtdWx0aXBsZXhlZCBTTlAgYXJyYXkgZm9yIGdlbmV0aWMgbWFwcGluZyBpbiBtYXJpdGltZSBwaW5lIGFuZCBjb21wYXJhdGl2ZSBtYXBwaW5nIHdpdGggbG9ibG9sbHkgcGluZTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSB2aWEgNDU0IHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzsgUGxhbnQgSm91cm5hbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTkxMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgU05QIGRpc2NvdmVyeSB0aHJvdWdoIGRlZXAgcmVzZXF1ZW5jaW5nIG9mIGEgcmVkdWNlZCByZXByZXNlbnRhdGlvbiBsaWJyYXJ5IHRvIGFuY2hvciBhbmQgb3JpZW50IHNjYWZmb2xkcyBpbiB0aGUgc295YmVhbiB3aG9sZSBnZW5vbWUgc2VxdWVuY2U7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgU05QIGRpc2NvdmVyeSBpbiB0ZXRyYXBsb2lkIGFsZmFsZmEgdXNpbmcgNDU0IHNlcXVlbmNpbmcgYW5kIGhpZ2ggcmVzb2x1dGlvbiBtZWx0aW5nIGFuYWx5c2lzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zNTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgU05QIGdlbm90eXBpbmcgaW4gdGhlIGhpZ2hseSBoZXRlcm96eWdvdXMgZ2Vub21lIG9mIEV1Y2FseXB0dXM6IGFzc2F5IHN1Y2Nlc3MsIHBvbHltb3JwaGlzbSBhbmQgdHJhbnNmZXJhYmlsaXR5IGFjcm9zcyBzcGVjaWVzOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz02NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBvZiBnZW5vbWUtd2lkZSBETkEgcG9seW1vcnBoaXNtcyBpbiBhIGxhbmRyYWNlIGN1bHRpdmFyIG9mIGphcG9uaWNhIHJpY2UgYnkgd2hvbGUtZ2Vub21lIHNlcXVlbmNpbmc7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz0yNzQtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBkaXNjb3ZlcnkgYW5kIGRldmVsb3BtZW50IG9mIGEgaGlnaC1kZW5zaXR5IGdlbm90eXBpbmcgYXJyYXkgZm9yIHN1bmZsb3dlcjsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUyOTgxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseSBwYXJhbGxlbCBTTlAgZ2Vub3R5cGluZzsgQ29sZCBTcHJpbmcgSGFyYi4gU3ltcC4gUXVhbnQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9NjktNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJlYWRBcnJheSB0ZWNobm9sb2d5OiBlbmFibGluZyBhbiBhY2N1cmF0ZSwgY29zdC1lZmZlY3RpdmUgYXBwcm9hY2ggdG8gaGlnaC10aHJvdWdocHV0IGdlbm90eXBpbmc7IEJpb3RlY2huaXF1ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1TNTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdobHkgcGFyYWxsZWwgZ2Vub21pYyBhc3NheXM7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTYzMi00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05Qc3RyZWFtIFVIVDogdWx0cmEtaGlnaCB0aHJvdWdocHV0IFNOUCBnZW5vdHlwaW5nIGZvciBwaGFybWFjb2dlbm9taWNzIGFuZCBkcnVnIGRpc2NvdmVyeTsgQmlvdGVjaG5pcXVlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPVM3MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJhbGxlbCBnZW5vdHlwaW5nIG9mIG92ZXIgMTAsMDAwIFNOUHMgdXNpbmcgYSBvbmUtcHJpbWVyIGFzc2F5IG9uIGEgaGlnaC1kZW5zaXR5IG9saWdvbnVjbGVvdGlkZSBhcnJheTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00MTQtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm90eXBpbmcgb3ZlciAxMDAsMDAwIFNOUHMgb24gYSBwYWlyIG9mIG9saWdvbnVjbGVvdGlkZSBhcnJheXM7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTA5LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZS13aWRlIFNOUCBwYW5lbCBmb3IgbWFwcGluZyBhbmQgYXNzb2NpYXRpb24gc3R1ZGllcyBpbiB0aGUgcmF0OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkIGdlbm9tZS13aWRlIFNOUCBnZW5vdHlwaW5nOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9UzUtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBnZW5vdHlwaW5nIHdpdGggdGhlIEdvbGRlbkdhdGUgYXNzYXkgaW4gdGhlIGNvbXBsZXggZ2Vub21lIG9mIHNveWJlYW47IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTE2OyAgY2l0YXRpb25fcGFnZXM9OTQ1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgaW1wbGVtZW50YXRpb24gb2YgaGlnaC10aHJvdWdocHV0IFNOUCBnZW5vdHlwaW5nIGluIGJhcmxleTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTgyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBnZW5vdHlwaW5nIHdpdGggdGhlIEdvbGRlbkdhdGUgYXNzYXkgaW4gbWFpemU7IE1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NDQxLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgZ2Vub3R5cGluZyBhbmQgbWFwcGluZyBvZiBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGluIGxvYmxvbGx5IHBpbmUgKFBpbnVzIHRhZWRhIEwuKTsgVHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjI1LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBnZW5vbWljIGNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIGdlbnVzIFZpdGlzOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTgyMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIG9mIDE0IGFncm9ub21pYyB0cmFpdHMgaW4gcmljZSBsYW5kcmFjZXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTYxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IG9mIGxlYWYgYXJjaGl0ZWN0dXJlIGluIHRoZSBtYWl6ZSBuZXN0ZWQgYXNzb2NpYXRpb24gbWFwcGluZyBwb3B1bGF0aW9uOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTE1OS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIG51Y2xlb3RpZGUgZGl2ZXJzaXR5LCByZWNvbWJpbmF0aW9uLCBhbmQgbGlua2FnZSBkaXNlcXVpbGlicml1bSBpbiB0aGUgbW9kZWwgbGVndW1lIE1lZGljYWdvIHRydW5jYXR1bGE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9RTg2NC03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGNvbXBhcmlzb25zIHJldmVhbCBhIGRvbWluYW50IG1lY2hhbmlzbSBvZiBjaHJvbW9zb21lIG51bWJlciByZWR1Y3Rpb24gaW4gZ3Jhc3NlcyBhbmQgYWNjZWxlcmF0ZWQgZ2Vub21lIGV2b2x1dGlvbiBpbiBUcml0aWNlYWU7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiwgVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTE1NzgwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBkaXNjb3ZlcnkgYW5kIGxpbmthZ2UgbWFwIGNvbnN0cnVjdGlvbiBpbiBjdWx0aXZhdGVkIHRvbWF0bzsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0zODEtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgYSBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBsaW5rYWdlIG1hcCB0byBnZW5vbWUgZmVhdHVyZXMgaW4gdGhlIG1vZGVsIGdyYXNzIEJyYWNoeXBvZGl1bSBkaXN0YWNoeW9uOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTQ1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIChTTlApIG1hcmtlcnMgZm9yIHVzZSBpbiBjb21tZXJjaWFsIG1haXplIChaZWEgbWF5cyBMLikgZ2VybXBsYXNtOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE2NS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoIHJlc29sdXRpb24gZ2VuZXRpYyBtYXAgYW5jaG9yaW5nIHNjYWZmb2xkcyBvZiB0aGUgc2VxdWVuY2VkIHdhdGVybWVsb24gZ2Vub21lOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTI5NDUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBETkEgY29udGVudCBvZiBzb21lIGltcG9ydGFudCBwbGFudCBzcGVjaWVzOyBQbGFudCBNb2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTIwOC0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpb24gb2Ygbm92ZWwgU1NSIGFuZCBnZW5lLWJhc2VkIFNOUCBtYXJrZXIgbG9jaSBpbiB0aGUgY2hpY2twZWEgZ2VuZXRpYyBtYXAgYW5kIGVzdGFibGlzaG1lbnQgb2YgbmV3IGFuY2hvciBwb2ludHMgd2l0aCBNZWRpY2FnbyB0cnVuY2F0dWxhIGdlbm9tZTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz0xNDE1LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNpbmcgdGhlIFNUTVMgZ2Vub21pYyByZXNvdXJjZXMgZm9yIGRlZmluaW5nIG5ldyBsb2NhdGlvbnMgb24gdGhlIGludHJhc3BlY2lmaWMgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIHVzZSBvZiBnZW5pYyBtb2xlY3VsYXIgbWFya2VycyAoR01NcykgZm9yIGNvbnN0cnVjdGlvbiBvZiBhIHRyYW5zY3JpcHQgbWFwIG9mIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyMjsgIGNpdGF0aW9uX3BhZ2VzPTE1NzctODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVTVC1kZXJpdmVkIGdlbmljIG1vbGVjdWxhciBtYXJrZXJzOiBkZXZlbG9wbWVudCBhbmQgdXRpbGl6YXRpb24gZm9yIGdlbmVyYXRpbmcgYW4gYWR2YW5jZWQgdHJhbnNjcmlwdCBtYXAgb2YgY2hpY2twZWE7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9MTQ0OS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgRVNUcyBmcm9tIGNoaWNrcGVhIHJvb3RzIGFuZCB0aGVpciB1c2UgaW4gZGl2ZXJzaXR5IGFuYWx5c2lzIG9mIHRoZSBDaWNlciBnZW51czsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgYW5hbHlzaXMgb2YgY2FuZGlkYXRlIEFzY29jaHl0YSBibGlnaHQgZGVmZW5zZSBnZW5lcyBpbiBjaGlja3BlYSwgUGFydCBJLiBHZW5lcmF0aW9uIGFuZCBhbmFseXNpcyBvZiBhbiBleHByZXNzZWQgc2VxdWVuY2UgdGFnIChFU1QpIGxpYnJhcnk7IFBoeXNpb2wuIE1vbC4gUGxhbnQuIFBhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjY7ICBjaXRhdGlvbl9wYWdlcz0xOTItMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXByZWhlbnNpdmUgcmVzb3VyY2Ugb2YgZHJvdWdodCBhbmQgc2FsaW5pdHkgcmVzcG9uc2l2ZSBFU1RzIGZvciBnZW5lIGRpc2NvdmVyeSBhbmQgbWFya2VyIGRldmVsb3BtZW50IGluIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz01MjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiBnZW5vdHlwZSBkZXBlbmRlbnQgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3MgYW5kIHN0cmVzcy1yZXNwb25zaXZlIHRyYW5zY3JpcHRvbWUgb2YgY2hpY2twZWEgd2lsdCBpbGx1c3RyYXRlIHByZWRpY3RlZCBhbmQgdW5leHBlY3RlZCBnZW5lcyBhbmQgbm92ZWwgcmVndWxhdG9ycyBvZiBwbGFudCBpbW11bml0eTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NDE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBhc3NlbWJseSBvZiBjaGlja3BlYSB0cmFuc2NyaXB0b21lIHVzaW5nIHNob3J0IHJlYWRzIGZvciBnZW5lIGRpc2NvdmVyeSBhbmQgbWFya2VyIGlkZW50aWZpY2F0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTUzLTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGRpc2NvdmVyeSBhbmQgdGlzc3VlLXNwZWNpZmljIHRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgaW4gY2hpY2twZWEgd2l0aCBtYXNzaXZlbHkgcGFyYWxsZWwgcHlyb3NlcXVlbmNpbmcgYW5kIHdlYiByZXNvdXJjZSBkZXZlbG9wbWVudDsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTU2OyAgY2l0YXRpb25fcGFnZXM9MTY2MS03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgdHJhbnNjcmlwdG9tZSBhbmFseXNpcyBpbiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKSwgYW4gb3JwaGFuIGxlZ3VtZSBjcm9wIG9mIHRoZSBzZW1pLWFyaWQgdHJvcGljcyBvZiBBc2lhIGFuZCBBZnJpY2E7IFBsYW50IEJpb3RlY2guIEouOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz05MjItMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlLXRhZ2dlZCBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGZvciBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MjEwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gYW5kIG1hcHBpbmcgb2Ygc2VxdWVuY2UtdGFnZ2VkIG1pY3Jvc2F0ZWxsaXRlIHNpdGVzIGluIHRoZSBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKSBnZW5vbWU7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjYyOyAgY2l0YXRpb25fcGFnZXM9OTAtMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGxpbmthZ2UgbWFwIG9mIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pIGdlbm9tZSBiYXNlZCBvbiByZWNvbWJpbmFudCBpbmJyZWQgbGluZXMgZnJvbSBhIEMuIGFyaWV0aW51bSB0aW1lcyBDLiByZXRpY3VsYXR1bSBjcm9zczogbG9jYWxpemF0aW9uIG9mIHJlc2lzdGFuY2UgZ2VuZXMgZm9yIEZ1c2FyaXVtIHdpbHQgcmFjZXMgNCBhbmQgNTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xMTU1LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgQkFDIGFuZCBCSUJBQyBsaWJyYXJpZXMgYW5kIHRoZWlyIGFwcGxpY2F0aW9ucyBmb3IgZ2VuZXJhdGlvbiBvZiBTU1IgbWFya2VycyBmb3IgZ2Vub21lIGFuYWx5c2lzIG9mIGNoaWNrcGVhLCBDaWNlciBhcmlldGludW0gTC47IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9NDkyLTUxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHNlcXVlbmNlIHRhZ2dlZCBtaWNyb3NhdGVsbGl0ZSBzaXRlcyBtYXJrZXJzIGluIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyBNb2wuIEVjb2wuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz00MjgtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgYW5kIGFuYWx5c2lzIG9mIGludHJhc3BlY2lmaWMgZ2VuZXRpYyB2YXJpYWJpbGl0eSBpbiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz0xNDE2LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBzZXF1ZW5jZS10YWdnZWQgbWljcm9zYXRlbGxpdGVzIHNpdGUgbWFya2VycyBmb3IgY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLik7IE1vbC4gRWNvbC4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTkzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGNoaWNrcGVhIEVTVC1TU1IgbWFya2VycyBhbmQgYW5hbHlzaXMgb2YgYWxsZWxpYyB2YXJpYXRpb24gYWNyb3NzIHJlbGF0ZWQgc3BlY2llczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTg7ICBjaXRhdGlvbl9wYWdlcz01OTEtNjA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbnNlbnN1cyBnZW5ldGljIG1hcCBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKSBiYXNlZCBvbiAxMCBtYXBwaW5nIHBvcHVsYXRpb25zOyBFdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc1OyAgY2l0YXRpb25fcGFnZXM9MTc1LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29sZCBTcHJpbmcgSGFyYm9yIExhYm9yYXRvcnksIENvbGQgU3ByaW5nIEhhcmJvcjsgTW9sZWN1bGFyIEJpb2xvZ3kgb2YgUGxhbnRzOiBBIExhYm9yYXRvcnkgQ291cnNlIE1hbnVhbDsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNpbmcgaW4gbWljcm9mYWJyaWNhdGVkIGhpZ2gtZGVuc2l0eSBwaWNvbGl0cmUgcmVhY3RvcnM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00Mzc7ICBjaXRhdGlvbl9wYWdlcz0zNzYtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUpvaW5NYXAgdmVyc2lvbiA0LjA6IFNvZnR3YXJlIGZvciB0aGUgY2FsY3VsYXRpb24gb2YgZ2VuZXRpYyBsaW5rYWdlIG1hcHMgaW4gZXhwZXJpbWVudGFsIHBvcHVsYXRpb247IGNpdGF0aW9uX3RleHQ9S3lhem1hIEIuVi4sIFdhZ2VuaW5nZW4sIE5ldGhlcmxhbmRzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXN0aW1hdGlvbiBvZiBtYXAgZGlzdGFuY2VzIGZyb20gcmVjb21iaW5hdGlvbiB2YWx1ZXM7IEFubi4gRXVnZW5pY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNzItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIHBvbHltb3JwaGljIHJlZ2lvbnMgaW4gdGhlIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGdlbm9tZSB3aXRoIHJlc2VxdWVuY2luZyBtaWNyb2FycmF5czsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz05MTgtOTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlAgZGlzY292ZXJ5IGJ5IGhpZ2gtdGhyb3VnaHB1dCBzZXF1ZW5jaW5nIGluIHNveWJlYW47IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQ2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUgZGVmaW5pdGlvbiBvZiB0aGUgcGVkaWdyZWUgaGFwbG90eXBlcyBvZiBjbG9zZWx5IHJlbGF0ZWQgcmljZSBjdWx0aXZhcnMgYnkgbWVhbnMgb2YgZ2Vub21lLXdpZGUgZGlzY292ZXJ5IG9mIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTI2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBhbmFseXNpcyBvZiBhbGxlbGUtc3BlY2lmaWMgZXhwcmVzc2lvbiBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTgyOyAgY2l0YXRpb25fcGFnZXM9OTQzLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNlcXVlbmNpbmcgNTAgYWNjZXNzaW9ucyBvZiBjdWx0aXZhdGVkIGFuZCB3aWxkIHJpY2UgeWllbGRzIG1hcmtlcnMgZm9yIGlkZW50aWZ5aW5nIGFncm9ub21pY2FsbHkgaW1wb3J0YW50IGdlbmVzOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNlcXVlbmNpbmcgb2YgMzEgd2lsZCBhbmQgY3VsdGl2YXRlZCBzb3liZWFuIGdlbm9tZXMgaWRlbnRpZmllcyBwYXR0ZXJucyBvZiBnZW5ldGljIGRpdmVyc2l0eSBhbmQgc2VsZWN0aW9uOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTEwNTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjZW50IGhpc3Rvcnkgb2YgYXJ0aWZpY2lhbCBvdXRjcm9zc2luZyBmYWNpbGl0YXRlcyB3aG9sZS1nZW5vbWUgYXNzb2NpYXRpb24gbWFwcGluZyBpbiBlbGl0ZSBpbmJyZWQgY3JvcCB2YXJpZXRpZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTg2NTYtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFwcGxpY2F0aW9uIG9mIGhpZ2gtdGhyb3VnaHB1dCBTTlAgZ2Vub3R5cGluZyBmb3IgYmFybGV5IGdlbm9tZSBtYXBwaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHJlY29tYmluYW50IGNocm9tb3NvbWUgc3Vic3RpdHV0aW9uIGxpbmVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExOTsgIGNpdGF0aW9uX3BhZ2VzPTYxMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TU1ItYmFzZWQgbGlua2FnZSBtYXAgd2l0aCBuZXcgbWFya2VycyB1c2luZyBhbiBpbnRyYXNwZWNpZmljIHBvcHVsYXRpb24gb2YgY29tbW9uIHdoZWF0OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTEwNDItNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaCBkZW5zaXR5IGJhcmxleSBtaWNyb3NhdGVsbGl0ZSBjb25zZW5zdXMgbWFwIHdpdGggNzc1IFNTUiBsb2NpOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExNDsgIGNpdGF0aW9uX3BhZ2VzPTEwOTEtMTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2ggZGVuc2l0eSBnZW5ldGljIG1hcCBvZiB0b2JhY2NvIChOaWNvdGlhbmEgdGFiYWN1bSBMLikgb2J0YWluZWQgZnJvbSBsYXJnZSBzY2FsZSBtaWNyb3NhdGVsbGl0ZSBtYXJrZXIgZGV2ZWxvcG1lbnQ7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9MjE5LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlcnNwZWNpZmljIGxpbmthZ2UgbWFwIG9mIFNTUiBhbmQgaW50cm9uaWMgcG9seW1vcnBoaXNtIG1hcmtlcnMgaW4gdG9tYXRvOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBTU1IgYW5kIGdlbmUtdGFyZ2V0ZWQgbWFya2VycyBmb3IgY29uc3RydWN0aW9uIG9mIGEgZnJhbWV3b3JrIGxpbmthZ2UgbWFwIG9mIENhdGhhcmFudGh1cyByb3NldXM7IEFubi4gQm90LiAoTG9uZC4pOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTMyMS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IGludGVncmF0ZWQgbGlua2FnZSBtYXAgYmFzZWQgb24gU1NSIG1hcmtlcnMgaW4gc295YmVhbjsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yMTMtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBncmFwZXZpbmUgZ2Vub21lIHNlcXVlbmNlIHN1Z2dlc3RzIGFuY2VzdHJhbCBoZXhhcGxvaWRpemF0aW9uIGluIG1ham9yIGFuZ2lvc3Blcm0gcGh5bGE7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDk7ICBjaXRhdGlvbl9wYWdlcz00NjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgY3VjdW1iZXIsIEN1Y3VtaXMgc2F0aXZ1cyBMLjsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz0xMjc1LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBkb21lc3RpY2F0ZWQgYXBwbGUgKE1hbHVzIHggZG9tZXN0aWNhIEJvcmtoLik7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9ODMzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgZ2Vub21lIGNvbnNlcnZhdGlvbiBiZXR3ZWVuIGNyb3AgYW5kIG1vZGVsIGxlZ3VtZSBzcGVjaWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE1Mjg5LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbmFseXNpcyBvZiBzeW50ZW55IG9mIEFyYWNoaXMgd2l0aCBMb3R1cyBhbmQgTWVkaWNhZ28gc2hlZHMgbmV3IGxpZ2h0IG9uIHRoZSBzdHJ1Y3R1cmUsIHN0YWJpbGl0eSBhbmQgZXZvbHV0aW9uIG9mIGxlZ3VtZSBnZW5vbWVzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz00NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhdHVyYXRpb24gb2YgYW4gaW50cmEtZ2VuZSBwb29sIGxpbmthZ2UgbWFwOiB0b3dhcmRzIGEgdW5pZmllZCBjb25zZW5zdXMgbGlua2FnZSBtYXAgZm9yIGZpbmUgbWFwcGluZyBhbmQgc3ludGVueSBhbmFseXNpcyBpbiBjb21tb24gYmVhbjsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyODEzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgU05QIGFuZCBTU1IgYmFzZWQgZ2VuZXRpYyBtYXAgb2YgQXNwYXJhZ3VzIGJlYW4gKFZpZ25hIHVuZ3VpY3VsYXRhIHNzcC4gc2VzcXVpcGVkYWxpcykgYW5kIGNvbXBhcmlzb24gd2l0aCB0aGUgYnJvYWRlciBzcGVjaWVzOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE1OTUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3Jvc3Mtc3BlY2llcyBhbXBsaWZpY2F0aW9uIG9mIE1lZGljYWdvIHRydW5jYXR1bGEgbWljcm9zYXRlbGxpdGVzIGFjcm9zcyB0aHJlZSBtYWpvciBwdWxzZSBjcm9wczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0xMjEwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS81LzM1Ny8xNzUwNTExL2RzczAxOC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIHByZXNlbnQgc3R1ZHkgcmVwb3J0cyB0aGUgbGFyZ2Utc2NhbGUgZGlzY292ZXJ5IG9mIGdlbm9tZS13aWRlIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbXMgKFNOUHMpIGluIGNoaWNrcGVhLCBpZGVudGlmaWVkIG1haW5seSB0aHJvdWdoIHRoZSAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTYyODM4OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSp0KM6lfi8L4tN/TSgRTkcdJ1EiQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f357%2f628389" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f357%2f628389" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=25687" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&amp;Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/5">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="10617334">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617334">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10617336">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617336">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10617341">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617341">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10617358">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617358">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10617376">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617376">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10617384">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617384">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10617386">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617386">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10617388">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617388">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10617390">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10617390">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/19/5/375/628653">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-Throughput SNP Discovery and Genotyping for Constructing a Saturated Linkage Map of Chickpea (<em>Cicer arietinum</em> L.)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rashmi Gaur</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rashmi+Gaur">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gaur R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gaur R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rashmi Gaur</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sarwar Azam</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sarwar+Azam">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Azam S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Azam S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sarwar Azam</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ganga Jeena</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ganga+Jeena">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jeena G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jeena G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ganga Jeena</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aamir Waseem Khan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aamir+Waseem+Khan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Khan A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Khan A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aamir Waseem Khan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shalu Choudhary</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shalu+Choudhary">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choudhary S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choudhary S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shalu Choudhary</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mukesh Jain</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mukesh+Jain">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jain M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jain M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mukesh Jain</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gitanjali Yadav</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gitanjali+Yadav">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yadav G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yadav G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gitanjali Yadav</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akhilesh K. Tyagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akhilesh+K.+Tyagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tyagi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tyagi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akhilesh K. Tyagi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Debasis Chattopadhyay</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Debasis+Chattopadhyay">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chattopadhyay D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chattopadhyay D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Debasis Chattopadhyay</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sabhyata Bhatia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research</div>, <div class="addr-line">Aruna Asaf Ali Marg, PO Box 10531, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 11-26735159. Fax. +91 11-26741658. E-mail: <a href="mailto:sabhyata_bhatia@nipgr.res.in" target="_blank">sabhyata_bhatia@nipgr.res.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sabhyata+Bhatia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bhatia S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bhatia S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sabhyata Bhatia</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 5, 1 October 2012, Pages 357–373, <a href='https://doi.org/10.1093/dnares/dss018'>https://doi.org/10.1093/dnares/dss018</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 August 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 April 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 July 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="628389" href="/dnaresearch/article-pdf/19/5/357/1750511/dss018.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Rashmi Gaur, Sarwar Azam, Ganga Jeena, Aamir Waseem Khan, Shalu Choudhary, Mukesh Jain, Gitanjali Yadav, Akhilesh K. Tyagi, Debasis Chattopadhyay, Sabhyata Bhatia;  High-Throughput SNP Discovery and Genotyping for Constructing a Saturated Linkage Map of Chickpea (<em>Cicer arietinum</em> L.), <em>DNA Research</em>, Volume 19, Issue 5, 1 October 2012, Pages 357–373, <a href='https://doi.org/10.1093/dnares/dss018'>https://doi.org/10.1093/dnares/dss018</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/628389?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/628389?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/628389?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/628389?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/628389?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/628389?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/628389?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-Throughput%20SNP%20Discovery%20and%20Genotyping%20for%20Constructing%20a%20Saturated%20Linkage%20Map%20of%20Chickpea%20(%26lt%3Bem%26gt%3BCicer%20arietinum%26lt%3B%2Fem%26gt%3B%20L.)&amp;publicationDate=2012-08-03&amp;volumeNum=19&amp;issueNum=5&amp;author=Gaur%2C%20Rashmi%3B%20Azam%2C%20Sarwar&amp;startPage=357&amp;endPage=373&amp;contentId=10.1093%2Fdnares%2Fdss018&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f357%2f628389" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=10617334 id="10617334" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The present study reports the large-scale discovery of genome-wide single-nucleotide polymorphisms (SNPs) in chickpea, identified mainly through the next generation sequencing of two genotypes, i.e. <em>Cicer arietinum</em> ICC4958 and its wild progenitor <em>C. reticulatum</em> PI489777, parents of an inter-specific reference mapping population of chickpea. Development and validation of a high-throughput SNP genotyping assay based on Illumina's GoldenGate Genotyping Technology and its application in building a high-resolution genetic linkage map of chickpea is described for the first time. In this study, 1022 SNPs were identified, of which 768 high-confidence SNPs were selected for designing the custom Oligo Pool All (CpOPA-I) for genotyping. Of these, 697 SNPs could be successfully used for genotyping, demonstrating a high success rate of 90.75%. Genotyping data of the 697 SNPs were compiled along with those of 368 co-dominant markers mapped in an earlier study, and a saturated genetic linkage map of chickpea was constructed. One thousand and sixty-three markers were mapped onto eight linkage groups spanning 1808.7 cM (centiMorgans) with an average inter-marker distance of 1.70 cM, thereby representing one of the most advanced maps of chickpea. The map was used for the synteny analysis of chickpea, which revealed a higher degree of synteny with the phylogenetically close <em>Medicago</em> than with soybean. The first set of validated SNPs and map resources developed in this study will not only facilitate QTL mapping, genome-wide association analysis and comparative mapping in legumes but also help anchor scaffolds arising out of the whole-genome sequencing of chickpea.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="chickpea">chickpea</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage map&quot;">linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genotyping">genotyping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="NGS">NGS</a></div>                            <h2 scrollto-destination=10617336 id="10617336" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Discovery of the DNA sequence polymorphism is a prerequisite to generating genetic markers for various applications of modern genomics research such as map-based cloning, marker-assisted breeding, association mapping and understanding genome structure and function. Among the sequence polymorphisms, single-nucleotide polymorphisms (SNPs) represent the most abundant type of variation present in DNA. SNPs are mostly biallelic,<sup><a reveal-id="DSS018C1" data-open="DSS018C1" class="link link-ref link-reveal xref-bibr">1</a></sup> co-dominantly inherited, sequence tagged and occur at high density within genomes.<sup><a reveal-id="DSS018C2" data-open="DSS018C2" class="link link-ref link-reveal xref-bibr">2</a></sup> They are thus amenable to the development of genetic molecular markers at low cost, which can provide sufficiently dense genome coverage for the dissection of complex traits. SNPs are known to occur at high frequencies of ∼1 per 500–1000 bp in humans<sup><a reveal-id="DSS018C3" data-open="DSS018C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS018C4" data-open="DSS018C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and in plant genomes also, where their frequency appears to vary significantly, e.g. 1 SNP per 16 bp in eucalyptus,<sup><a reveal-id="DSS018C5" data-open="DSS018C5" class="link link-ref link-reveal xref-bibr">5</a></sup> 1 per 107 bp in radish,<sup><a reveal-id="DSS018C6" data-open="DSS018C6" class="link link-ref link-reveal xref-bibr">6</a></sup> 1 per 147 bp in rice,<sup><a reveal-id="DSS018C7" data-open="DSS018C7" class="link link-ref link-reveal xref-bibr">7</a></sup> 1 per 200 bp in maize,<sup><a reveal-id="DSS018C8" data-open="DSS018C8" class="link link-ref link-reveal xref-bibr">8</a></sup> 1 per 370 bp in soybean genes<sup><a reveal-id="DSS018C9" data-open="DSS018C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and 1 per 500 bp in <em>Arabidopsis</em>.<sup><a reveal-id="DSS018C10" data-open="DSS018C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>Recently, technological advancements have accelerated the genome-wide SNP discovery not only in model species, but also in crop plants. Initial efforts relied on the discovery of large numbers of SNPs from EST databases such as in grapevine,<sup><a reveal-id="DSS018C11" data-open="DSS018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> spruce,<sup><a reveal-id="DSS018C12" data-open="DSS018C12" class="link link-ref link-reveal xref-bibr">12</a></sup> cowpea,<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> pea,<sup><a reveal-id="DSS018C14" data-open="DSS018C14" class="link link-ref link-reveal xref-bibr">14</a></sup> wheat<sup><a reveal-id="DSS018C15" data-open="DSS018C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and pine.<sup><a reveal-id="DSS018C16" data-open="DSS018C16" class="link link-ref link-reveal xref-bibr">16</a></sup> However, more recently, with the development of the next generation sequencing (NGS) platforms which enable the sequencing of millions of bases at deep coverage, thousands of SNPs have been identified in many species such as maize,<sup><a reveal-id="DSS018C17" data-open="DSS018C17" class="link link-ref link-reveal xref-bibr">17</a></sup> soybean,<sup><a reveal-id="DSS018C18" data-open="DSS018C18" class="link link-ref link-reveal xref-bibr">18</a></sup><em>Medicago</em>,<sup><a reveal-id="DSS018C19" data-open="DSS018C19" class="link link-ref link-reveal xref-bibr">19</a></sup><em>Eucalyptus</em>,<sup><a reveal-id="DSS018C20" data-open="DSS018C20" class="link link-ref link-reveal xref-bibr">20</a></sup> rice<sup><a reveal-id="DSS018C21" data-open="DSS018C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and sunflower.<sup><a reveal-id="DSS018C22" data-open="DSS018C22" class="link link-ref link-reveal xref-bibr">22</a></sup> To utilize the millions of available SNPs, various high-throughput SNP genotyping platforms were simultaneously developed such as the GoldenGate Genotyping Technology (GGGT; Illumina, San Diego, CA, USA),<sup><a reveal-id="DSS018C23" data-open="DSS018C23" class="link link-ref link-reveal xref-bibr">23</a></sup> BeadChip-based Infinium assay (Illumina),<sup><a reveal-id="DSS018C24" data-open="DSS018C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSS018C25" data-open="DSS018C25" class="link link-ref link-reveal xref-bibr">25</a></sup> SNPStream (Beckman Coulter, USA),<sup><a reveal-id="DSS018C26" data-open="DSS018C26" class="link link-ref link-reveal xref-bibr">26</a></sup> GeneChip (Affymetrix, USA)<sup><a reveal-id="DSS018C27" data-open="DSS018C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS018C28" data-open="DSS018C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and competitive allele-specific PCR, KASPar (KBio science, UK)<sup><a reveal-id="DSS018C29" data-open="DSS018C29" class="link link-ref link-reveal xref-bibr">29</a></sup> that allow large-scale genotyping of SNPs in parallel in a large set of individuals.<sup><a reveal-id="DSS018C30" data-open="DSS018C30" class="link link-ref link-reveal xref-bibr">30</a></sup> These approaches vary in terms of sensitivity, reproducibility, accuracy, capability of multiplexing and throughput. Among them, one of the most versatile SNP genotyping platforms is the Illumina GGGT, which is capable of multiplexing from 96 to 1536 SNPs per assay in a single reaction over a 3-day period.<sup><a reveal-id="DSS018C23" data-open="DSS018C23" class="link link-ref link-reveal xref-bibr">23</a></sup> This technology has persistently been reported as highly reliable, with high SNP conversion rates and has fostered genetic research in several major as well as in orphan plants, especially self-pollinating crops that have been plagued by a narrow genetic base with nucleotide diversities from 0.2 to 0.5%,<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS018C31  DSS018C33" data-open="DSS018C31  DSS018C33" class="link link-ref link-reveal xref-bibr">31–33</a></sup> as well as the cross-pollinated species with higher sequence diversities of ≥2%.<sup><a reveal-id="DSS018C12" data-open="DSS018C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSS018C34" data-open="DSS018C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSS018C35" data-open="DSS018C35" class="link link-ref link-reveal xref-bibr">35</a></sup> High-throughput SNP genotyping has enabled various applications in plant genomics especially genome-wide association studies and linkage disequilibrium studies,<sup><a reveal-id="DSS018C10" data-open="DSS018C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS018C36  DSS018C38" data-open="DSS018C36  DSS018C38" class="link link-ref link-reveal xref-bibr">36–38</a></sup> synteny-based comparative genomics<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS018C39" data-open="DSS018C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and high-resolution genetic mapping.<sup><a reveal-id="DSS018C40" data-open="DSS018C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSS018C41" data-open="DSS018C41" class="link link-ref link-reveal xref-bibr">41</a></sup> A high-density genetic linkage map is one of the most important genomic tools to accelerate marker-assisted breeding. Recent studies have reported the successful utilization of high-throughput GGGT for SNP genotyping to build genetic linkage maps in legumes such as cowpea,<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> soybean<sup><a reveal-id="DSS018C18" data-open="DSS018C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS018C31" data-open="DSS018C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and pea,<sup><a reveal-id="DSS018C14" data-open="DSS018C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and even in plants with highly repetitive or polyploid genomes like barley,<sup><a reveal-id="DSS018C32" data-open="DSS018C32" class="link link-ref link-reveal xref-bibr">32</a></sup> pine,<sup><a reveal-id="DSS018C34" data-open="DSS018C34" class="link link-ref link-reveal xref-bibr">34</a></sup> grass<sup><a reveal-id="DSS018C41" data-open="DSS018C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and maize.<sup><a reveal-id="DSS018C42" data-open="DSS018C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Production of high-density SNP-based maps have facilitated the fine mapping and cloning of agronomically important genes and also anchoring and orienting the scaffolds generated by whole genome sequence assembly data.<sup><a reveal-id="DSS018C18" data-open="DSS018C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS018C41" data-open="DSS018C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSS018C43" data-open="DSS018C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p><p>Chickpea (<em>C. arietinum</em> L.), a diploid (2<em>n</em> = 2<em>x</em> = 16), annual, self-pollinated crop, with a genome size of 740 Mb,<sup><a reveal-id="DSS018C44" data-open="DSS018C44" class="link link-ref link-reveal xref-bibr">44</a></sup> represents the world's third most important legume crop that is mainly grown in the arid and semi-arid regions of Asia and Africa. It serves as a key source of protein in human nutrition and also plays an important role in the maintenance of soil fertility owing to its ability to fix atmospheric nitrogen. Due to its economic importance, there has been a recent spurt in chickpea genomics research and a large number of genomic resources, such as molecular markers and linkage maps,<sup><a reveal-id="DSS018C45  DSS018C48" data-open="DSS018C45  DSS018C48" class="link link-ref link-reveal xref-bibr">45–48</a></sup> ESTs<sup><a reveal-id="DSS018C49  DSS018C52" data-open="DSS018C49  DSS018C52" class="link link-ref link-reveal xref-bibr">49–52</a></sup> and NGS-based transcriptomes,<sup><a reveal-id="DSS018C53  DSS018C55" data-open="DSS018C53  DSS018C55" class="link link-ref link-reveal xref-bibr">53–55</a></sup> have become available. Currently, there are ∼2000 co-dominant molecular markers available, which include ∼980 genomic simple sequence repeats (gSSRs),<sup><a reveal-id="DSS018C45" data-open="DSS018C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSS018C46" data-open="DSS018C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS018C56  DSS018C62" data-open="DSS018C56  DSS018C62" class="link link-ref link-reveal xref-bibr">56–62</a></sup> and EST-derived markers including 361 EST-SSRs, 238 Intron-Targeted Primers (ITPs), 109 Expressed Sequence Tag Polymorphisms (ESTPs) and 294 Cleaved Amplified Polymorphic Sites/SNPs.<sup><a reveal-id="DSS018C45" data-open="DSS018C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSS018C47  DSS018C49" data-open="DSS018C47  DSS018C49" class="link link-ref link-reveal xref-bibr">47–49</a>,<a reveal-id="DSS018C51" data-open="DSS018C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSS018C63" data-open="DSS018C63" class="link link-ref link-reveal xref-bibr">63</a></sup> Using these co-dominant markers, linkage maps that have been generated define ∼500 mapped positions.<sup><a reveal-id="DSS018C45  DSS018C48" data-open="DSS018C45  DSS018C48" class="link link-ref link-reveal xref-bibr">45–48</a>,<a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DSS018C64" data-open="DSS018C64" class="link link-ref link-reveal xref-bibr">64</a></sup> Moreover, from the recent chickpea transcriptome sequencing, even though ∼5000 molecular markers have been reported,<sup><a reveal-id="DSS018C54" data-open="DSS018C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSS018C55" data-open="DSS018C55" class="link link-ref link-reveal xref-bibr">55</a></sup> the validation and genotyping of these molecular markers is yet to be undertaken. Despite this increased availability of genomic resources, the large-scale discovery and utilization of SNPs, which serve as the most potential markers for providing sufficiently dense genetic maps, has not been carried out in chickpea. Therefore, in the present study, the large-scale identification of genome-wide SNPs was undertaken from the NGS of two parents of a mapping population (<em>C. arietinum</em> cv. ICC4958 and <em>C. reticulatum</em> PI489777). Conversion of SNPs into successful genotyping assays based on the Illumina GoldenGate technology was demonstrated for the first time in chickpea. Moreover, the SNP markers were successfully mapped in the backdrop of previously reported co-dominant markers to generate one of the most comprehensive and dense genetic maps of chickpea.</p>                            <h2 scrollto-destination=10617341 id="10617341" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=10617343 id="10617343" class="section-title" data-legacy-id=s2a>2.1. Plant material and DNA isolation</h3>
<p>For SNP discovery, the two genotypes <em>C. arietinum</em> ICC4958 (fusarium wilt resistant and drought tolerant) and <em>C. reticulatum</em> PI489777 (wild annual species, fusarium wilt susceptible), which are parents of a mapping population, were sequenced. Nuclear DNA from these two genotypes was isolated using the protocol of Malmberg <em>et al</em>.<sup><a reveal-id="DSS018C65" data-open="DSS018C65" class="link link-ref link-reveal xref-bibr">65</a></sup> and used for sequencing. For SNP genotyping and linkage map generation, the internationally accepted reference mapping population consisting of 129 RILs (recombinant inbred lines) arising from an inter-specific cross between the two genotypes mentioned above was utilized. Genomic DNA was isolated from fresh young leaves of the two mapping parents and the 129 RILs using the GenElute™ plant genomic DNA Miniprep kit (Sigma). The DNA quality was checked by electrophoresis on 0.8% agarose gels. For SNP genotyping, the DNA was quantified using Quant-iT™ Pico Green<sup>®</sup> dsDNA Kit (Invitrogen) and the fluorescence was measured with the Microtiter plate reader (Varioscan from Thermo Scientific). Samples were adjusted to 50 ng/μl using Tris-EDTA buffer. The GoldenGate genotyping assay was performed using 250 ng of DNA from each of the RILs.</p>                            <h3 scrollto-destination=10617345 id="10617345" class="section-title" data-legacy-id=s2b>2.2. NGS and SNP discovery</h3>
<p>SNP discovery was based on the high-throughput next generation whole genome sequencing of the two genotypes, i.e. <em>C. arietinum</em> cv. ICC4958 and its wild relative <em>C. reticulatum</em> cv. PI489777. Sequencing of <em>C. arietinum</em> ICC4958 was carried out primarily by the 454/Roche GS FLX Titanium platform, for which DNA sample preparation and construction of whole-genome shotgun (WGS) and matepair libraries were performed as described by the manufacturer,<sup><a reveal-id="DSS018C66" data-open="DSS018C66" class="link link-ref link-reveal xref-bibr">66</a></sup> with some modifications. Briefly, the nuclear DNA was nebulized and size-selected for a 300–900-bp fragment size using Agencourt AMPure SPRI beads and ligated with specific adapters A and B at each end, which resulted in generating a single-stranded or a double-stranded library. One single-stranded library using the standard library preparation kit and two double-stranded libraries using a rapid library preparation kit with an average insert size of 705 bases were used for sequencing. The libraries were amplified by emPCR.<sup><a reveal-id="DSS018C66" data-open="DSS018C66" class="link link-ref link-reveal xref-bibr">66</a></sup> DNA molecules that contain a single A and B adapters at each end were enriched with capture beads coated with capture DNA. Sequencing was performed using GS FLX Titanium Sequencing kits following the manufacturer's protocol (Roche Applied Sciences, Mannheim, Germany). The generated Roche 454 reads were submitted to the Short Read Archive (SRA) database of NCBI (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra">www.ncbi.nlm.nih.gov/sra</a>) and assigned the accession number SRA053228.1. These reads were assembled using the <em>de novo</em> assembly tool Newbler (GS <em>de novo</em> assembler, Roche Applied Sciences) version 2.3 to obtain the reference sequence. Further, two genomic DNA libraries of 3 kb average insert size, of both the parents (<em>C. arietinum</em> ICC4958 and <em>C. reticulatum</em> PI489777) were also prepared by the SOLiD Opti Mate-paired library kit following the manufacturer's protocol and sequenced using the SOLiD 4.0 platform (Applied Biosystems, Foster City, CA, USA). These reads were assigned the accession number SRA053197.5 by the SRA of NCBI. SNPs were identified by aligning and mapping the SOLiD reads of ICC4958 and PI489777 onto the Roche assembly using the tool HAPS (<a class="link link-uri" href="http://solidsoftwaretools.com/gf/project/haps/">http://solidsoftwaretools.com/gf/project/haps/</a>). For the selection of SNPs the following criteria were used: (i) non-reference alleles with a minimum throughput of 5X and score 0.6 were selected, (ii) the non-reference alleles with any variant (SNPs or indels) within 60 bases upstream or downstream of the consensus reference sequences were discarded and (iii) one SNP from each contig was chosen to ensure genome-wide distribution.</p><p>The second approach for SNP discovery was amplicon resequencing. This involved the identification of SNPs from chickpea ESTs generated earlier in the laboratory and is described in Choudhary <em>et al</em>.<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> A set of 222 EST-derived primers were used to PCR amplify and resequence genomic DNA of the two parental lines (<em>C. arietinum</em> cv. ICC4958 and <em>C. reticulatum</em> cv. PI489777) of the reference mapping population.</p>                            <h3 scrollto-destination=10617349 id="10617349" class="section-title" data-legacy-id=s2c>2.3. Development of the GoldenGate SNP genotyping assays</h3>
<p>SNP genotyping was carried out using the Illumina GGGT. For developing the GGGT assay, the identified SNPs along with the 60 bp sequence flanking it on either side were submitted to Illumina for processing by Illumina's Array Design Tool (ADT) in order to obtain a designability rank score for each SNP ranging from 0 to 1. This score serves as a reliability metric for testing whether or not a particular SNP will convert into a successful working GoldenGate assay where SNPs with the scores of 0.6 or higher possess a high probability of converting into a successful genotyping assay. Thus, SNPs having the highest ADT scores were selected for designing the custom Oligo Pool All (CpOPA) containing the allele-specific and locus-specific oligos for use in the Illumina GoldenGate assay and the first chickpea 768-OPA was generated and designated CpOPA-I. The list of SNPs and their flanking regions that constituted CpOPA-I are presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Genotyping of SNPs was performed using Illumina's BeadArray Express Reader according to the standard manufacturer's protocol,<sup><a reveal-id="DSS018C23" data-open="DSS018C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSS018C31" data-open="DSS018C31" class="link link-ref link-reveal xref-bibr">31</a></sup> with 250 ng (50 ng/μl) of genomic DNA from each individual RIL of the mapping population. The automatic allele calling for each locus was inferred with the GenomeStudio Software V2011.1 (Illumina). The fluorescence images of array matrix carrying Cy3- and Cy5-labelled beads were generated with the two-channel scanner. A genotype that is homozygous for one SNP allele will display a signal in either the Cy3 or the Cy5 channel, whereas a genotype that is heterozygous will display a signal in both channels. The intensity data were loaded in GenomeStudio, and cluster positions were assigned to all genotypes at each SNP and these were checked manually for errors and rescored while designating homozygous and heterozygous clusters. Further, the call rates and the GenTrain and GenCall score generated using the GenomeStudio software were analysed for SNP reliability. The GenTrain and GenCall50 score of ≥0.4 and the call rate of ≥95% were used as the Illumina recommended thresholds for declaring a reliable SNP.</p>                            <h3 scrollto-destination=10617352 id="10617352" class="section-title" data-legacy-id=s2d>2.4. Construction of the genetic linkage map</h3>
<p>The inter-specific reference mapping population consisting of 129 RILs described above was used for genotyping and map generation. For linkage analysis, the GoldenGate genotyping data of the successful SNPs were utilized. In addition to this, the genotyping data of 368 polymorphic markers (52 EST-SSRs, 51 ITPs, 25 ESTPs, 2 MtESTs, 130 gSSRs and 108 STMS; Table <a reveal-id="DSS018TB1" data-open="DSS018TB1" class="link link-reveal link-table xref-fig">1</a>) recently mapped by Choudhary <em>et al</em>.<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> using the same reference mapping population were also combined with the SNP genotyping data for anchoring and integrating the previously mapped co-dominant markers. This complete marker data set was employed for linkage analysis using Joinmap ver. 4.0.<sup><a reveal-id="DSS018C67" data-open="DSS018C67" class="link link-ref link-reveal xref-bibr">67</a></sup> A <em>χ</em><sup>2</sup> test (<em>P</em> &lt; 0.05) was performed for the conformity to the expected Mendelian segregation ratio of 1:1. Markers showing segregation distortion were also integrated into the map. Markers were ordered with the regression mapping algorithm and were classified into LGs using the grouping module at LOD thresholds of 8.0–10.0 at an increment of 0.5. Linkage groups were determined at LOD 9.5 with a recombination frequency smaller than 0.49 and a maximum threshold value of 5 for the jump. The best marker order was determined using the ‘Ripple’ function (value of 1). Recombination frequencies were converted to map distances in centiMorgans (cM) using the Kosambi mapping function.<sup><a reveal-id="DSS018C68" data-open="DSS018C68" class="link link-ref link-reveal xref-bibr">68</a></sup></p>                        <a id="10617354" scrollto-destination="10617354"></a>
<div content-id="DSS018TB1" class="table-modal"><div class="table-wrap"><div id="DSS018TB1" class="table-wrap-title" data-id="DSS018TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Markers utilized for construction of the inter-specific linkage map of chickpea (<em>C. arietinum</em> ICC4958 × <em>C. reticulatum</em> PI489777)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Markers analysed </th><th>Polymorphic markers used for mapping (%) </th><th>Markers mapped </th></tr></thead><tbody><tr><td>SNPs (from this study) </td><td>768 </td><td>697 (90.75) </td><td>696 </td></tr><tr><td>EST-SSRs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>185 </td><td>52 (28.1) </td><td>51 </td></tr><tr><td>ITPs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>151 </td><td>51 (33.8) </td><td>51 </td></tr><tr><td>ESTPs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>109 </td><td>25 (22.9) </td><td>25 </td></tr><tr><td>MtESTs<sup><a reveal-id="DSS018C90" data-open="DSS018C90" class="link link-ref link-reveal xref-bibr">90</a></sup> </td><td>15 </td><td>2 (13.3) </td><td>2 </td></tr><tr><td>gSSRs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>310 </td><td>130 (41.9) </td><td>130 </td></tr><tr><td>STMS markers<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> </td><td>108 </td><td>108 </td><td>108 </td></tr><tr><td>Total </td><td>1646 </td><td>1065 (64.7) </td><td>1063 </td></tr></tbody></table></div><div content-id="DSS018TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Markers analysed </th><th>Polymorphic markers used for mapping (%) </th><th>Markers mapped </th></tr></thead><tbody><tr><td>SNPs (from this study) </td><td>768 </td><td>697 (90.75) </td><td>696 </td></tr><tr><td>EST-SSRs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>185 </td><td>52 (28.1) </td><td>51 </td></tr><tr><td>ITPs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>151 </td><td>51 (33.8) </td><td>51 </td></tr><tr><td>ESTPs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>109 </td><td>25 (22.9) </td><td>25 </td></tr><tr><td>MtESTs<sup><a reveal-id="DSS018C90" data-open="DSS018C90" class="link link-ref link-reveal xref-bibr">90</a></sup> </td><td>15 </td><td>2 (13.3) </td><td>2 </td></tr><tr><td>gSSRs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>310 </td><td>130 (41.9) </td><td>130 </td></tr><tr><td>STMS markers<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> </td><td>108 </td><td>108 </td><td>108 </td></tr><tr><td>Total </td><td>1646 </td><td>1065 (64.7) </td><td>1063 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10617354" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS018TB1" class="table-wrap-title" data-id="DSS018TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Markers utilized for construction of the inter-specific linkage map of chickpea (<em>C. arietinum</em> ICC4958 × <em>C. reticulatum</em> PI489777)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Markers analysed </th><th>Polymorphic markers used for mapping (%) </th><th>Markers mapped </th></tr></thead><tbody><tr><td>SNPs (from this study) </td><td>768 </td><td>697 (90.75) </td><td>696 </td></tr><tr><td>EST-SSRs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>185 </td><td>52 (28.1) </td><td>51 </td></tr><tr><td>ITPs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>151 </td><td>51 (33.8) </td><td>51 </td></tr><tr><td>ESTPs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>109 </td><td>25 (22.9) </td><td>25 </td></tr><tr><td>MtESTs<sup><a reveal-id="DSS018C90" data-open="DSS018C90" class="link link-ref link-reveal xref-bibr">90</a></sup> </td><td>15 </td><td>2 (13.3) </td><td>2 </td></tr><tr><td>gSSRs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>310 </td><td>130 (41.9) </td><td>130 </td></tr><tr><td>STMS markers<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> </td><td>108 </td><td>108 </td><td>108 </td></tr><tr><td>Total </td><td>1646 </td><td>1065 (64.7) </td><td>1063 </td></tr></tbody></table></div><div content-id="DSS018TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Markers analysed </th><th>Polymorphic markers used for mapping (%) </th><th>Markers mapped </th></tr></thead><tbody><tr><td>SNPs (from this study) </td><td>768 </td><td>697 (90.75) </td><td>696 </td></tr><tr><td>EST-SSRs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>185 </td><td>52 (28.1) </td><td>51 </td></tr><tr><td>ITPs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>151 </td><td>51 (33.8) </td><td>51 </td></tr><tr><td>ESTPs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>109 </td><td>25 (22.9) </td><td>25 </td></tr><tr><td>MtESTs<sup><a reveal-id="DSS018C90" data-open="DSS018C90" class="link link-ref link-reveal xref-bibr">90</a></sup> </td><td>15 </td><td>2 (13.3) </td><td>2 </td></tr><tr><td>gSSRs<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> </td><td>310 </td><td>130 (41.9) </td><td>130 </td></tr><tr><td>STMS markers<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> </td><td>108 </td><td>108 </td><td>108 </td></tr><tr><td>Total </td><td>1646 </td><td>1065 (64.7) </td><td>1063 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10617354" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=10617356 id="10617356" class="section-title" data-legacy-id=s2e>2.5. Synteny</h3>
<p>Synteny analysis of chickpea was carried out using genomic (gSSRs and SNPs) and genic markers (EST-SSRs, ITPs and ESTPs) from the current chickpea map. Marker sequences were aligned against the chromosome-based assembly of <em>Medicago</em> HapMap Mt 3.5 database [<em>Medicago truncatula</em> HapMap Project 2010 (<a class="link link-uri" href="http://www.medicagohapmap.org/downloads.php">http://www.medicagohapmap.org/downloads.php</a>)] as well as with <em>Glycine max</em>, JGI Glyma1 [Soybean Genome Project (<a class="link link-uri" href="http://www.phytozome.net/soybean">http://www.phytozome.net/soybean</a>)] using local Blastn with an <em>E</em>-value of &lt;1e−05. Synteny was visualized using MapChart 2.2 (Voorrips 2002). All the Blastn individual hits corresponding to markers from the chickpea LGs (CaLG1-8) were parsed into the Microsoft Excel and the whole genome dot-blots were developed by using the physical location (Mb) of <em>M. truncatula</em> and <em>G. max</em> sequences and the genetic location (cM) of chickpea sequences.</p>                            <h2 scrollto-destination=10617358 id="10617358" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=10617359 id="10617359" class="section-title" data-legacy-id=s3a>3.1. SNP identification</h3>
<p>SNPs were identified from the whole genome sequencing of the two parents of the chickpea reference mapping population namely <em>C. arietinum</em> ICC4958 and its wild progenitor <em>C. reticulatum</em> PI489777. Nuclear genomes of the parents were sequenced to various depths using the NGS platforms, i.e. the Roche 454 GSFLX Titanium and the Applied Biosystem's SOLiD platforms and compared for SNP identification. WGS libraries (three) of <em>C. arietinum</em> cv. ICC4958 nuclear DNA were used for 15 runs generating a total of 20.50 million filtered reads with 7.674 Gb high-quality (Phred score 20) bases. All the filtered reads generated by the 454 GS FLX platform were assembled by the <em>de novo</em> assembly tool Newbler (GS <em>de novo</em> assembler, Roche Applied Sciences) version 2.3 to obtain the primary assembly. The assembly parameters used were seed step 12, minimum overlap 40, minimum overlap identity 95% and seed length 16, with default quality filter, adapter and primer trimming of the input reads. The assembly yielded a total length of 350 724 940 bases in 160 883 large contigs (≥500 bases) of average size 2180 bp and the N50 value of 3541 bp with 98.66% of the bases having quality of Q40 or more. Of these, 1000 largest contigs ranging in size from 36 to 13 kb were chosen as the reference sequence for drawing out the single-nucleotide changes. Further, 3 kb paired end libraries of both the parents (<em>C. arietinum</em> ICC4958 and <em>C. reticulatum</em> PI489777) were sequenced using the SOLiD (Applied Biosystems) platform to generate 28 Gb (37.8X) and 12 Gb (16.2X) high-quality paired short reads (50 bases), respectively. SNPs in <em>C. reticulatum</em> PI489777 were identified using the SOLiD reads. First, the SOLiD reads of ICC4958 were aligned on the previously described ICC4958 reference sequence by the Bioscope module of the tool HAPS (<a class="link link-uri" href="http://solidsoftwaretools.com/gf/project/haps/">http://solidsoftwaretools.com/gf/project/haps/</a>). The mapping output file of only the properly paired reads was then utilized by the SST scaffolding module of HAPS to produce a SOLiD read-based consensus sequence of ICC4958. Next, the SOLiD reads of <em>C. reticulatum</em> PI489777 were mapped onto the consensus sequence for SNP identification. Stringent criteria were used for SNP calling and only those SNPs were selected that had identical SNPs of 5X coverage and 0.6 confidence score and whose flanking regions were suitable for designing allele-specific and locus-specific primers for the Illumina GoldenGate Assay. These included only those SNPs which did not have other SNPs or indels in the 60-bp regions flanking it. Hence, a set of 920 candidate SNP loci were selected from the same number of contigs and were designated as CaSNPs. Additionally, by amplicon resequencing of 222 chickpea EST loci from <em>C. arietinum</em> and <em>C. reticulatum</em>, 102 EST-derived SNPs (designated as ESNPs) were identified for genotyping as described in Choudhary <em>et al</em>.<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Thus, from both the approaches, a total of 1022 SNPs (920 CaSNPs + 102 ESNPs) were identified for genotyping using the GoldenGate assay. All the 1022 SNPs and their flanking sequences are available in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Moreover, the 1022 identified SNPs were classified as transitions (C/T and G/A) or transversions (C/G, T/A, A/C and G/T) based on nucleotide substitutions. In this analysis, higher rates of transition (almost double) in comparison with the rate of transversions were found, since 650 (63.65%) transitions were found in comparison with 372 (36.4%) transversions. Among the transitions, the [C/T] type was slightly more prevalent (334, 51.4%) than [G/A] that accounted for 316 (48.6%) of single base changes, whereas among transversions, T/A scored relatively higher (123) in comparison with other transversions.</p>                            <h3 scrollto-destination=10617361 id="10617361" class="section-title" data-legacy-id=s3b>3.2. SNP genotyping</h3>
<p>The 1022 SNPs were submitted to Illumina for the Oligo Pool All (OPA) design for use in the GoldenGate assay. Each of the individual SNPs was assigned an ADT designability score by Illumina, ranging from 0 to 1, where a score &lt;0.4 predicted a low success rate, 0.4 to 0.6 a moderate success rate and &gt;0.6 a high success rate for the conversion of an SNP into a successful GoldenGate assay. In all, 944 SNPs (851 CaSNPs and 93 ESNPs) were assigned designability scores ≥0.6, thereby resulting in the possibility of conversion of 92.37% of the predicted SNPs into successful GoldenGate assays. However, for convenience of customizing the array design, the 768 SNPs with the highest scores (≥0.76) were selected for inclusion in the first chickpea 768-OPA, named as CpOPA-I (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 1</a>). This CpOPA-I assay was utilized for genotyping of the 129 RIL individuals along with the parental lines of the reference mapping population.</p><p>The genotyping data of the 768 SNPs across the 129 RILs were analysed using the GenomeStudio software (Illumina), which clusters and calls the data automatically, allowing visualization of the data directly for downstream analysis. For each SNP, the genotyping data representation included three main clusters corresponding to AA homozygote, AB heterozygote and BB homozygote. As in the present study, an F9-F10 RIL mapping population was used, which is expected to contribute very few heterozygotes, most of the SNP markers produced two main clusters representing the two homozygous genotypes, with sometimes a small additional cluster corresponding to heterozygous genotypes (Fig. <a reveal-id="DSS018F1" data-open="DSS018F1" class="link link-reveal link-table xref-fig">1</a>A). A few data points were sometimes ambiguously located outside these clusters (indicated by arrows in Fig. <a reveal-id="DSS018F1" data-open="DSS018F1" class="link link-reveal link-table xref-fig">1</a>A) and represented those for which no calls were generated and were therefore scored as missing data. In our genotyping data set, the average level of heterozygosity was 6.5%, expected for a RIL population, whereas the level of missing data per marker averaged at 5.6%. </p>                        <a id="10617364" scrollto-destination="10617364"></a>
<div id="DSS018F1" data-id="DSS018F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/m_dss01801.jpeg?Expires=2147483647&Signature=RD0KLMqkoNySDiOa9ai6hQIJEL4K69MXPjiXINDQsQ83XwKs-K2mgF9SXVqa~HOtBPPLf9VQjj-pf5A0FdrGmjr2NtO6tqiYWl1H1Kxj9oqClbGIqoM6AsLbgAlP1aK1WAjqRhxMGmRPQQBISlzRCHN4-mfg8RH0x8yd-aSSj0SvVYk9barJjGsxwD7tJVwhSXQwquk5KMD7paFTCbfuTKBUMk-gFe4eMqtgM5jmOO~AiDYF6krB9quI~YTpZLrV0DpERaURxkv5gfNCBVcMtcY80-eDEEHbIU5ZEk1iYI7bfT2UJqhZcgEIiZXY38oAjWF-i48LUxxwTU92fmfv1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative clustering patterns generated by the Illumina GoldenGate SNP Genotyping assay. For a given SNP marker, genotypes are called for each sample (dots) by their normalized signal intensity (Norm R, y-axis), i.e. sum of intensities of two fluorescent signals, and allele frequency (Norm theta, x-axis) relative to a cluster position (shaded area). The data point colour codes represent: red, AA (homozygous); blue, BB (homozygous); purple, AB (heterozygous); black, no call (missing data). (A) High-quality SNPs (e.g. CaSNP116 and CaSNP290) showing well-separated clusters of homozygous alleles (red and blue) and heterozygotes (purple). Some data points located between or in the border of these clusters (marked by an arrow) are unsuccessfully genotyped samples for which no calls were generated and considered as missing data. (B) SNPs which were considered as false or monomorphic (failed to detect an SNP in the parents and the mapping population) that grouped into a single cluster (e.g. CaSNP917 and ESNP46). (C) Technically unsatisfactory SNPs (e.g. CaSNP61 and CaSNP226) represented by insufficient allele cluster separation." path-from-xml="dss01801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10617364/dss01801.jpeg" path-from-xml="dss01801.jpeg" target="_blank">View large</a><a section="10617364" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss01801.jpeg?Expires=2147483647&Signature=T8ewhFVHptQvMrgyyC0HRYSgAF-6J9eD5N-aRPwh8rYqcYEvoqPO1kKuvLBHrr3Qpr-tMkcvO4KlwUWYQkIukWow0Ngtqhc5S9CJx42vwY1yPOZK6quTz~G7H~S7X0Rj-tyKGItumyAkDoyLPdMEjwyp~pCVrpV4ojYBP0hHidUgKovSuNEtcZjHWgBew7GJrRHg9JXy2h~HGF2uimw9ccMagm7AXQ6YOTaICpXL-~bzyBjze76Rd63AJ9-iim8HIS8A5krSNt7IbJjOzFY1rH3uamGgItQdSKKA7MhGoZiCkT2azYXVe1X6~~bo1u2i5ePmVVi9e9-6Oz4NKmAyhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10617364&ar=628389&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative clustering patterns generated by the Illumina GoldenGate SNP Genotyping assay. For a given SNP marker, genotypes are called for each sample (dots) by their normalized signal intensity (Norm R, <em>y</em>-axis), i.e. sum of intensities of two fluorescent signals, and allele frequency (Norm theta, <em>x</em>-axis) relative to a cluster position (shaded area). The data point colour codes represent: red, AA (homozygous); blue, BB (homozygous); purple, AB (heterozygous); black, no call (missing data). (A) High-quality SNPs (e.g. CaSNP116 and CaSNP290) showing well-separated clusters of homozygous alleles (red and blue) and heterozygotes (purple). Some data points located between or in the border of these clusters (marked by an arrow) are unsuccessfully genotyped samples for which no calls were generated and considered as missing data. (B) SNPs which were considered as false or monomorphic (failed to detect an SNP in the parents and the mapping population) that grouped into a single cluster (e.g. CaSNP917 and ESNP46). (C) Technically unsatisfactory SNPs (e.g. CaSNP61 and CaSNP226) represented by insufficient allele cluster separation.</p></div></div><div content-id="DSS018F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/m_dss01801.jpeg?Expires=2147483647&Signature=RD0KLMqkoNySDiOa9ai6hQIJEL4K69MXPjiXINDQsQ83XwKs-K2mgF9SXVqa~HOtBPPLf9VQjj-pf5A0FdrGmjr2NtO6tqiYWl1H1Kxj9oqClbGIqoM6AsLbgAlP1aK1WAjqRhxMGmRPQQBISlzRCHN4-mfg8RH0x8yd-aSSj0SvVYk9barJjGsxwD7tJVwhSXQwquk5KMD7paFTCbfuTKBUMk-gFe4eMqtgM5jmOO~AiDYF6krB9quI~YTpZLrV0DpERaURxkv5gfNCBVcMtcY80-eDEEHbIU5ZEk1iYI7bfT2UJqhZcgEIiZXY38oAjWF-i48LUxxwTU92fmfv1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative clustering patterns generated by the Illumina GoldenGate SNP Genotyping assay. For a given SNP marker, genotypes are called for each sample (dots) by their normalized signal intensity (Norm R, y-axis), i.e. sum of intensities of two fluorescent signals, and allele frequency (Norm theta, x-axis) relative to a cluster position (shaded area). The data point colour codes represent: red, AA (homozygous); blue, BB (homozygous); purple, AB (heterozygous); black, no call (missing data). (A) High-quality SNPs (e.g. CaSNP116 and CaSNP290) showing well-separated clusters of homozygous alleles (red and blue) and heterozygotes (purple). Some data points located between or in the border of these clusters (marked by an arrow) are unsuccessfully genotyped samples for which no calls were generated and considered as missing data. (B) SNPs which were considered as false or monomorphic (failed to detect an SNP in the parents and the mapping population) that grouped into a single cluster (e.g. CaSNP917 and ESNP46). (C) Technically unsatisfactory SNPs (e.g. CaSNP61 and CaSNP226) represented by insufficient allele cluster separation." path-from-xml="dss01801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10617364/dss01801.jpeg" path-from-xml="dss01801.jpeg" target="_blank">View large</a><a section="10617364" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss01801.jpeg?Expires=2147483647&Signature=T8ewhFVHptQvMrgyyC0HRYSgAF-6J9eD5N-aRPwh8rYqcYEvoqPO1kKuvLBHrr3Qpr-tMkcvO4KlwUWYQkIukWow0Ngtqhc5S9CJx42vwY1yPOZK6quTz~G7H~S7X0Rj-tyKGItumyAkDoyLPdMEjwyp~pCVrpV4ojYBP0hHidUgKovSuNEtcZjHWgBew7GJrRHg9JXy2h~HGF2uimw9ccMagm7AXQ6YOTaICpXL-~bzyBjze76Rd63AJ9-iim8HIS8A5krSNt7IbJjOzFY1rH3uamGgItQdSKKA7MhGoZiCkT2azYXVe1X6~~bo1u2i5ePmVVi9e9-6Oz4NKmAyhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10617364&ar=628389&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative clustering patterns generated by the Illumina GoldenGate SNP Genotyping assay. For a given SNP marker, genotypes are called for each sample (dots) by their normalized signal intensity (Norm R, <em>y</em>-axis), i.e. sum of intensities of two fluorescent signals, and allele frequency (Norm theta, <em>x</em>-axis) relative to a cluster position (shaded area). The data point colour codes represent: red, AA (homozygous); blue, BB (homozygous); purple, AB (heterozygous); black, no call (missing data). (A) High-quality SNPs (e.g. CaSNP116 and CaSNP290) showing well-separated clusters of homozygous alleles (red and blue) and heterozygotes (purple). Some data points located between or in the border of these clusters (marked by an arrow) are unsuccessfully genotyped samples for which no calls were generated and considered as missing data. (B) SNPs which were considered as false or monomorphic (failed to detect an SNP in the parents and the mapping population) that grouped into a single cluster (e.g. CaSNP917 and ESNP46). (C) Technically unsatisfactory SNPs (e.g. CaSNP61 and CaSNP226) represented by insufficient allele cluster separation.</p></div></div><p>In the present study, of the 768 assays, 724 SNPs generated clearly defined, well-separated clusters (as in Fig. <a reveal-id="DSS018F1" data-open="DSS018F1" class="link link-reveal link-table xref-fig">1</a>A), whereas the remaining 44 SNPs did not produce clear clustering patterns and were excluded from the linkage analysis. These consisted of 28 false or monomorphic SNPs (failed to detect an SNP in the parents and the mapping population) that grouped into a single cluster (Fig. <a reveal-id="DSS018F1" data-open="DSS018F1" class="link link-reveal link-table xref-fig">1</a>B), and 16 SNPs that were classified as technically unsuccessful, since they had the GenTrain and GenCall50 score of &lt;0.40, ‘no-call’ frequencies &gt;5% and were represented by insufficient allele cluster separation (Fig. <a reveal-id="DSS018F1" data-open="DSS018F1" class="link link-reveal link-table xref-fig">1</a>C). Further, of the 724 SNP assays that showed clear clustering, 27 were found to be heterozygous in at least one of the parents and hence were excluded from linkage analysis. Therefore in all, 697 SNPs (645 CaSNPs and 52 ESNPs) out of 768 SNPs were found to be successful, demonstrating a success rate of 90.75% for the Illumina GGGT in chickpea.</p>                            <h3 scrollto-destination=10617366 id="10617366" class="section-title" data-legacy-id=s3c>3.3. SNP mapping</h3>
<p>For construction of a dense genetic map of chickpea, the genotyping data of 697 polymorphic SNPs obtained from 129 RILs of the reference inter-specific mapping population (<em>C. arietinum</em> ICC4958 x <em>C. reticulatum</em> PI489777) were utilized. Further, to generate an integrated and advanced genetic map of chickpea, the genotyping data of the 697 SNPs were compiled along with the genotyping data of 368 co-dominant markers published previously by our group.<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> These markers included 238 gSSRs (genomic SSRs), 52 EST-SSRs, 51 ITPs (Intron Targeted Polymorphism), 25 ESTPs and 2 MtESTs (Table <a reveal-id="DSS018TB1" data-open="DSS018TB1" class="link link-reveal link-table xref-fig">1</a>), which served as a framework for anchoring the SNP loci. Hence, data of 1065 markers (697 SNPs and 368 previously mapped co-dominant markers) were compiled together for linkage analysis and map generation using JoinMap ver. 4.0.<sup><a reveal-id="DSS018C67" data-open="DSS018C67" class="link link-ref link-reveal xref-bibr">67</a></sup> The resulting integrated linkage map of chickpea defined map positions of 1063 markers distributed over 8 linkage groups (Fig. <a reveal-id="DSS018F2" data-open="DSS018F2" class="link link-reveal link-table xref-fig">2</a>). The current map spanned 1808.7 cM with an average inter-marker distance of 1.7 cM. The map contained an average of 1.43 map positions per Mb of genome (1063 map positions/740 Mb) and had one marker per 696 kb (740 Mb/1063) when considering the chickpea genome to be 740 Mb.<sup><a reveal-id="DSS018C44" data-open="DSS018C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The LGs were numbered (1 to 8) based on the common marker positions shared between corresponding LGs from previous studies.<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> The genetic length of the LGs ranged from 186.8 cM (LG2) to 269.5 cM (LG1) (Table <a reveal-id="DSS018TB2" data-open="DSS018TB2" class="link link-reveal link-table xref-fig">2</a>). LG3 was the most saturated, having 205 markers with an average marker density of 0.95 cM, whereas LG8 had the least number of markers (only 84) (Table <a reveal-id="DSS018TB2" data-open="DSS018TB2" class="link link-reveal link-table xref-fig">2</a>). On an average, one linkage group contained 132.9 markers that spanned an average of 226 cM. All categories of markers including genic, genomic and SNPs were found on each of the LGs. However, the number of SNP markers in each of the LGs was maximum and varied in the range 65–75% except in LG7, which had only 38% SNP markers. The markers were unevenly and non-randomly distributed in the LGs and 25 clusters of markers (≥5 markers within 1 cM distance) were identified of which 20 clusters comprise only SNP markers, whereas a single cluster on LG3 contained only STMS markers. Moreover, nine gaps ranging from 20 to 35 cM were observed in four LGs (LG1, LG5, LG7 and LG8) located near the distal ends. The <em>χ</em><sup>2</sup> test performed resulted in segregation distortion for 42% (448) of the marker loci. However, 446 of these markers were finally integrated into the map so that the loss of genetic information related to these markers was minimized. Moreover, the distorted markers were found to be widely distributed throughout all the LGs even though the ratios varied from one LG to another. For example in LG1 and LG4, &lt;25% markers showed distortion, whereas in others &gt;30% of the mapped loci were distorted. The overall segregation distortion of 42% observed in this study was similar to the 41.3 and 38% reported earlier for the same mapping population.<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup></p>                        <a id="10617368" scrollto-destination="10617368"></a>
<div content-id="DSS018TB2" class="table-modal"><div class="table-wrap"><div id="DSS018TB2" class="table-wrap-title" data-id="DSS018TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Distribution of markers on the eight linkage groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LGs </th><th>Length (cM) </th><th>Number of mapped markers </th><th>Average marker density (cM) </th><th>SNP markers (% of total) </th><th>Markers showing distortion (%) </th><th>Other markers </th></tr></thead><tbody><tr><td>1 </td><td>269.5 </td><td>119 </td><td>2.26 </td><td>89 (74.8) </td><td>20.2 </td><td>30 </td></tr><tr><td>2 </td><td>186.8 </td><td>138 </td><td>1.35 </td><td>87 (63.0) </td><td>56.5 </td><td>51 </td></tr><tr><td>3 </td><td>196.6 </td><td>205 </td><td>0.95 </td><td>134 (65.4) </td><td>45.4 </td><td>71 </td></tr><tr><td>4 </td><td>200.4 </td><td>168 </td><td>1.19 </td><td>120 (71.4) </td><td>9.5 </td><td>48 </td></tr><tr><td>5 </td><td>259.3 </td><td>141 </td><td>1.83 </td><td>96 (68.0) </td><td>82.3 </td><td>45 </td></tr><tr><td>6 </td><td>220.5 </td><td>108 </td><td>2.04 </td><td>67 (62.0) </td><td>50.9 </td><td>41 </td></tr><tr><td>7 </td><td>263.0 </td><td>100 </td><td>2.63 </td><td>38 (38.0) </td><td>38.0 </td><td>62 </td></tr><tr><td>8 </td><td>212.6 </td><td>84 </td><td>2.53 </td><td>65 (77.4) </td><td>30.9 </td><td>19 </td></tr><tr><td>Total </td><td>1808.7 </td><td>1063 </td><td>1.70 </td><td>696 (65.48) </td><td>42.0 </td><td>367 </td></tr></tbody></table></div><div content-id="DSS018TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>LGs </th><th>Length (cM) </th><th>Number of mapped markers </th><th>Average marker density (cM) </th><th>SNP markers (% of total) </th><th>Markers showing distortion (%) </th><th>Other markers </th></tr></thead><tbody><tr><td>1 </td><td>269.5 </td><td>119 </td><td>2.26 </td><td>89 (74.8) </td><td>20.2 </td><td>30 </td></tr><tr><td>2 </td><td>186.8 </td><td>138 </td><td>1.35 </td><td>87 (63.0) </td><td>56.5 </td><td>51 </td></tr><tr><td>3 </td><td>196.6 </td><td>205 </td><td>0.95 </td><td>134 (65.4) </td><td>45.4 </td><td>71 </td></tr><tr><td>4 </td><td>200.4 </td><td>168 </td><td>1.19 </td><td>120 (71.4) </td><td>9.5 </td><td>48 </td></tr><tr><td>5 </td><td>259.3 </td><td>141 </td><td>1.83 </td><td>96 (68.0) </td><td>82.3 </td><td>45 </td></tr><tr><td>6 </td><td>220.5 </td><td>108 </td><td>2.04 </td><td>67 (62.0) </td><td>50.9 </td><td>41 </td></tr><tr><td>7 </td><td>263.0 </td><td>100 </td><td>2.63 </td><td>38 (38.0) </td><td>38.0 </td><td>62 </td></tr><tr><td>8 </td><td>212.6 </td><td>84 </td><td>2.53 </td><td>65 (77.4) </td><td>30.9 </td><td>19 </td></tr><tr><td>Total </td><td>1808.7 </td><td>1063 </td><td>1.70 </td><td>696 (65.48) </td><td>42.0 </td><td>367 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10617368" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS018TB2" class="table-wrap-title" data-id="DSS018TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Distribution of markers on the eight linkage groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LGs </th><th>Length (cM) </th><th>Number of mapped markers </th><th>Average marker density (cM) </th><th>SNP markers (% of total) </th><th>Markers showing distortion (%) </th><th>Other markers </th></tr></thead><tbody><tr><td>1 </td><td>269.5 </td><td>119 </td><td>2.26 </td><td>89 (74.8) </td><td>20.2 </td><td>30 </td></tr><tr><td>2 </td><td>186.8 </td><td>138 </td><td>1.35 </td><td>87 (63.0) </td><td>56.5 </td><td>51 </td></tr><tr><td>3 </td><td>196.6 </td><td>205 </td><td>0.95 </td><td>134 (65.4) </td><td>45.4 </td><td>71 </td></tr><tr><td>4 </td><td>200.4 </td><td>168 </td><td>1.19 </td><td>120 (71.4) </td><td>9.5 </td><td>48 </td></tr><tr><td>5 </td><td>259.3 </td><td>141 </td><td>1.83 </td><td>96 (68.0) </td><td>82.3 </td><td>45 </td></tr><tr><td>6 </td><td>220.5 </td><td>108 </td><td>2.04 </td><td>67 (62.0) </td><td>50.9 </td><td>41 </td></tr><tr><td>7 </td><td>263.0 </td><td>100 </td><td>2.63 </td><td>38 (38.0) </td><td>38.0 </td><td>62 </td></tr><tr><td>8 </td><td>212.6 </td><td>84 </td><td>2.53 </td><td>65 (77.4) </td><td>30.9 </td><td>19 </td></tr><tr><td>Total </td><td>1808.7 </td><td>1063 </td><td>1.70 </td><td>696 (65.48) </td><td>42.0 </td><td>367 </td></tr></tbody></table></div><div content-id="DSS018TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>LGs </th><th>Length (cM) </th><th>Number of mapped markers </th><th>Average marker density (cM) </th><th>SNP markers (% of total) </th><th>Markers showing distortion (%) </th><th>Other markers </th></tr></thead><tbody><tr><td>1 </td><td>269.5 </td><td>119 </td><td>2.26 </td><td>89 (74.8) </td><td>20.2 </td><td>30 </td></tr><tr><td>2 </td><td>186.8 </td><td>138 </td><td>1.35 </td><td>87 (63.0) </td><td>56.5 </td><td>51 </td></tr><tr><td>3 </td><td>196.6 </td><td>205 </td><td>0.95 </td><td>134 (65.4) </td><td>45.4 </td><td>71 </td></tr><tr><td>4 </td><td>200.4 </td><td>168 </td><td>1.19 </td><td>120 (71.4) </td><td>9.5 </td><td>48 </td></tr><tr><td>5 </td><td>259.3 </td><td>141 </td><td>1.83 </td><td>96 (68.0) </td><td>82.3 </td><td>45 </td></tr><tr><td>6 </td><td>220.5 </td><td>108 </td><td>2.04 </td><td>67 (62.0) </td><td>50.9 </td><td>41 </td></tr><tr><td>7 </td><td>263.0 </td><td>100 </td><td>2.63 </td><td>38 (38.0) </td><td>38.0 </td><td>62 </td></tr><tr><td>8 </td><td>212.6 </td><td>84 </td><td>2.53 </td><td>65 (77.4) </td><td>30.9 </td><td>19 </td></tr><tr><td>Total </td><td>1808.7 </td><td>1063 </td><td>1.70 </td><td>696 (65.48) </td><td>42.0 </td><td>367 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10617368" target="_blank">View Large</a></div></div>                        <a id="10617369" scrollto-destination="10617369"></a>
<div id="DSS018F2" data-id="DSS018F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/m_dss01802a.jpeg?Expires=2147483647&amp;Signature=4f1Iq6C2CAxxLaKgBXn-oxoYfpL5lyi-vEGRfpMFzLiQI8T43Ku-103tjoyduXxMw-o9rxkuqpz~Z-DXEKwHHpaIQpVpYRyt3kYv4pRRQi0z8efNwUZBmEkJSzV67dHLg-u33bxNOqjJjMFebQViQOUDnkcBNV65w-qOLWqgpaWFYvCVe3xO-qeCJTVeWw3hrtKGmvZY~XDJdVrVK3VX4O90DsGJlcmEEtqsLSypshTgGzbydW4qbx9SpSBo9CwVK7x57QCjzPC23KMoDgLXMgFv0RpPYKSAPiTzzigmrwTNHd6t7b9run8scqPYYxrHfhxOhPfSymcPEPSBL7UUDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inter-specific linkage map of chickpea. The inter-specific linkage map of chickpea based on RILs of C. arietinum (ICC4958) × C. reticulatum (PI489777) was generated using JoinMap version 4.0. The name of the linkage groups is mentioned at the top of each LG. Distances of the loci (cM) are shown to the left and the names of loci are shown to the right side of the linkage groups. SNP markers are represented in black, while previously published markers are shown in colour: red, genic markers; green, genomic markers; blue, STMS markers of Winter et al.58 used for anchoring." path-from-xml="dss01802a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10617369/dss01802a.jpeg" path-from-xml="dss01802a.jpeg" target="_blank">View large</a><a section="10617369" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss01802a.jpeg?Expires=2147483647&amp;Signature=todHK58F3lc8S-OQnxB-abqUycSWpiWyjNfF4N0ABRpvNlsbtxC5SYx9DbwPViOr-NpA8MCDAryHydb3LXzDBmb49QsJhj3UnZWvcdBEX7WBpKEmwqcj3WAuOEhDchv-oN~YQqz9C2BRG~QbSZgsy~HfTvf5eD5yGb9eKIWXlRRYI0Gx9e6Xg8-vQrECE-2ySiCCZ5M3zJBfU6GK5t8SAmuGVGsYzzGz2BZrWq-c1ombySIxiFTUzLnCk9EgAJMSVocUoO3E2a-CxAT6S~a55SBSp8jtMeZkC421O10lFsEj6pn9m5PQ9vIfCsfqw5YUQW22jU86mhmrWz6Ki1RBFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=10617369&amp;ar=628389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss01802a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/m_dss01802b.jpeg?Expires=2147483647&amp;Signature=Qg5tYDAuTcKKELBbrR52i8GZiCKuxEUWNBR4JbgVUOGonIAzxqZ99xKGpGhZNzAvjO~B63xK45vEHksHMTkiX~NxOvaW9UyZT1zZBQ-DIsSZ8somY~-2LlIyl7nSPzWSAXmOqF2E2FSRTfdEN8cRaXXrovSIJOmzacIeHqEIBJKsHsqZJLy2YO165nJ~gcMG-U2ys22yCWsK-SKGWbzOXqbrtz88BGMy8PNkJkm2ef4D~8-lIArDg3tn5YlWU0I-5HkY4jF3aiWVcbFMt55Li~QkGnymL-SHYPjXA2QjsA3l33ZIMhIxul6rxiJoGfIc~qxxq7Ai3nE1zdx4JhB3xQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inter-specific linkage map of chickpea. The inter-specific linkage map of chickpea based on RILs of C. arietinum (ICC4958) × C. reticulatum (PI489777) was generated using JoinMap version 4.0. The name of the linkage groups is mentioned at the top of each LG. Distances of the loci (cM) are shown to the left and the names of loci are shown to the right side of the linkage groups. SNP markers are represented in black, while previously published markers are shown in colour: red, genic markers; green, genomic markers; blue, STMS markers of Winter et al.58 used for anchoring." path-from-xml="dss01802b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10617369/dss01802b.jpeg" path-from-xml="dss01802b.jpeg" target="_blank">View large</a><a section="10617369" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss01802b.jpeg?Expires=2147483647&amp;Signature=DwmzNerSkS-SfObXMa~MGgTrqOoK4GiyWev8ms-z6SigoyDofiiAIWZf-V8ndEiFCiw8XibbkS1u42DcTRYY7q4cj2Q7uqjnbqaQUdJqR0dGGqfcKGXGySA9hDxpPTPhs401uaxdUFPXnH20VZljUwAbJBWqKCatv5sjzgJtjR8uVH-J398JnpTrxuerHvI8ocHM6FDtmKlbwbRvKcnb7f1UzEjbtAXN76FJJ7u5RYet5xILmBzz3pnOWohlFBFAd-ynjJS70XNapaWIH9wlPBYu7HUr2VYt1ovFzj~U8L9-N5SIsumitkrV8Vob0LivDh2ye9Hjic9jB47wboUFGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=10617369&amp;ar=628389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss01802b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Inter-specific linkage map of chickpea. The inter-specific linkage map of chickpea based on RILs of <em>C. arietinum</em> (ICC4958) × <em>C. reticulatum</em> (PI489777) was generated using JoinMap version 4.0. The name of the linkage groups is mentioned at the top of each LG. Distances of the loci (cM) are shown to the left and the names of loci are shown to the right side of the linkage groups. SNP markers are represented in black, while previously published markers are shown in colour: red, genic markers; green, genomic markers; blue, STMS markers of Winter <em>et al</em>.<sup><a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> used for anchoring.</p></div></div><div content-id="DSS018F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/m_dss01802a.jpeg?Expires=2147483647&amp;Signature=4f1Iq6C2CAxxLaKgBXn-oxoYfpL5lyi-vEGRfpMFzLiQI8T43Ku-103tjoyduXxMw-o9rxkuqpz~Z-DXEKwHHpaIQpVpYRyt3kYv4pRRQi0z8efNwUZBmEkJSzV67dHLg-u33bxNOqjJjMFebQViQOUDnkcBNV65w-qOLWqgpaWFYvCVe3xO-qeCJTVeWw3hrtKGmvZY~XDJdVrVK3VX4O90DsGJlcmEEtqsLSypshTgGzbydW4qbx9SpSBo9CwVK7x57QCjzPC23KMoDgLXMgFv0RpPYKSAPiTzzigmrwTNHd6t7b9run8scqPYYxrHfhxOhPfSymcPEPSBL7UUDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inter-specific linkage map of chickpea. The inter-specific linkage map of chickpea based on RILs of C. arietinum (ICC4958) × C. reticulatum (PI489777) was generated using JoinMap version 4.0. The name of the linkage groups is mentioned at the top of each LG. Distances of the loci (cM) are shown to the left and the names of loci are shown to the right side of the linkage groups. SNP markers are represented in black, while previously published markers are shown in colour: red, genic markers; green, genomic markers; blue, STMS markers of Winter et al.58 used for anchoring." path-from-xml="dss01802a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10617369/dss01802a.jpeg" path-from-xml="dss01802a.jpeg" target="_blank">View large</a><a section="10617369" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss01802a.jpeg?Expires=2147483647&amp;Signature=todHK58F3lc8S-OQnxB-abqUycSWpiWyjNfF4N0ABRpvNlsbtxC5SYx9DbwPViOr-NpA8MCDAryHydb3LXzDBmb49QsJhj3UnZWvcdBEX7WBpKEmwqcj3WAuOEhDchv-oN~YQqz9C2BRG~QbSZgsy~HfTvf5eD5yGb9eKIWXlRRYI0Gx9e6Xg8-vQrECE-2ySiCCZ5M3zJBfU6GK5t8SAmuGVGsYzzGz2BZrWq-c1ombySIxiFTUzLnCk9EgAJMSVocUoO3E2a-CxAT6S~a55SBSp8jtMeZkC421O10lFsEj6pn9m5PQ9vIfCsfqw5YUQW22jU86mhmrWz6Ki1RBFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=10617369&amp;ar=628389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss01802a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/m_dss01802b.jpeg?Expires=2147483647&amp;Signature=Qg5tYDAuTcKKELBbrR52i8GZiCKuxEUWNBR4JbgVUOGonIAzxqZ99xKGpGhZNzAvjO~B63xK45vEHksHMTkiX~NxOvaW9UyZT1zZBQ-DIsSZ8somY~-2LlIyl7nSPzWSAXmOqF2E2FSRTfdEN8cRaXXrovSIJOmzacIeHqEIBJKsHsqZJLy2YO165nJ~gcMG-U2ys22yCWsK-SKGWbzOXqbrtz88BGMy8PNkJkm2ef4D~8-lIArDg3tn5YlWU0I-5HkY4jF3aiWVcbFMt55Li~QkGnymL-SHYPjXA2QjsA3l33ZIMhIxul6rxiJoGfIc~qxxq7Ai3nE1zdx4JhB3xQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inter-specific linkage map of chickpea. The inter-specific linkage map of chickpea based on RILs of C. arietinum (ICC4958) × C. reticulatum (PI489777) was generated using JoinMap version 4.0. The name of the linkage groups is mentioned at the top of each LG. Distances of the loci (cM) are shown to the left and the names of loci are shown to the right side of the linkage groups. SNP markers are represented in black, while previously published markers are shown in colour: red, genic markers; green, genomic markers; blue, STMS markers of Winter et al.58 used for anchoring." path-from-xml="dss01802b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10617369/dss01802b.jpeg" path-from-xml="dss01802b.jpeg" target="_blank">View large</a><a section="10617369" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss01802b.jpeg?Expires=2147483647&amp;Signature=DwmzNerSkS-SfObXMa~MGgTrqOoK4GiyWev8ms-z6SigoyDofiiAIWZf-V8ndEiFCiw8XibbkS1u42DcTRYY7q4cj2Q7uqjnbqaQUdJqR0dGGqfcKGXGySA9hDxpPTPhs401uaxdUFPXnH20VZljUwAbJBWqKCatv5sjzgJtjR8uVH-J398JnpTrxuerHvI8ocHM6FDtmKlbwbRvKcnb7f1UzEjbtAXN76FJJ7u5RYet5xILmBzz3pnOWohlFBFAd-ynjJS70XNapaWIH9wlPBYu7HUr2VYt1ovFzj~U8L9-N5SIsumitkrV8Vob0LivDh2ye9Hjic9jB47wboUFGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=10617369&amp;ar=628389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss01802b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Inter-specific linkage map of chickpea. The inter-specific linkage map of chickpea based on RILs of <em>C. arietinum</em> (ICC4958) × <em>C. reticulatum</em> (PI489777) was generated using JoinMap version 4.0. The name of the linkage groups is mentioned at the top of each LG. Distances of the loci (cM) are shown to the left and the names of loci are shown to the right side of the linkage groups. SNP markers are represented in black, while previously published markers are shown in colour: red, genic markers; green, genomic markers; blue, STMS markers of Winter <em>et al</em>.<sup><a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> used for anchoring.</p></div></div>                            <h3 scrollto-destination=10617370 id="10617370" class="section-title" data-legacy-id=s3d>3.4. Synteny between chickpea and other legumes</h3>
<p>To identify the syntenic relationships between chickpea and other legume genomes, namely <em>M. truncatula</em> and <em>G. max</em>, the sequence information available for 917 of 1063 mapped loci on the current linkage map of chickpea was utilized (Table <a reveal-id="DSS018TB3" data-open="DSS018TB3" class="link link-reveal link-table xref-fig">3</a>). These markers included 696 SNPs, 51 EST-SSRs, 94 gSSRs, 51 ITPs and 25 ESTPs for which locus sequences were available. The details about the markers on each of the chickpea linkage groups (CaLGs) that showed hits with chromosomes of <em>G. max</em> and <em>M. truncatula</em> are described in Table <a reveal-id="DSS018TB3" data-open="DSS018TB3" class="link link-reveal link-table xref-fig">3</a>. </p>                        <a id="10617372" scrollto-destination="10617372"></a>
<div content-id="DSS018TB3" class="table-modal"><div class="table-wrap"><div id="DSS018TB3" class="table-wrap-title" data-id="DSS018TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Details of CaLGs, number of markers and synteny with <em>M. truncatula</em> and <em>G. max</em> chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chickpea LGs </th><th>Markers </th><th>Markers selected for blast </th><th>Mt chromosome (chickpea orthologs) </th><th>Gm chromosome (chickpea orthologs) </th></tr></thead><tbody><tr><td>CaLG1 </td><td>119 </td><td>107 </td><td>Mt2(35) Mt4(3) Mt1(2) Mt5(2) Mt7(2) Mt3(1) Mt6(1) Mt8(1) </td><td>Gm15(19) Gm13(17) Gm9(4) Gm8(3) Gm17(3) Gm3(2) Gm10(2) Gm11(2) Gm20(2) Gm1(1) Gm4(1) Gm5(1) Gm6(1) Gm7(1) Gm14(1) Gm19(1) </td></tr><tr><td>CaLG2 </td><td>138 </td><td>115 </td><td>Mt4(19) Mt5(9) Mt3(5) Mt6(4) Mt2(2) Mt7(2) Mt8(2) </td><td>Gm12(10) Gm11(7) Gm6(4) Gm9(4) Gm15(4) Gm18(4) Gm1(3) Gm5(3) Gm13(3) Gm20(3) Gm4(2) Gm7(2) Gm10(2) Gm14(2) Gm19(2) Gm3(1) </td></tr><tr><td>CaLG3 </td><td>205 </td><td>179 </td><td>Mt7(35) Mt5(23) Mt3(7) Mt2(6) Mt6(6) Mt8(4) Mt4(2) Mt1(3) </td><td>Gm19(29) Gm3(26) Gm1(12) Gm9(8) Gm10(7) Gm6(2) Gm7(6) Gm11(6) Gm13(5) Gm2(4) Gm5(4) Gm18(4) Gm20(4) Gm4(3) Gm6(2) Gm8(2) Gm12(2) Gm15(2) Gm16(2) Gm14(1) Gm17(1) </td></tr><tr><td>CaLG4 </td><td>168 </td><td>152 </td><td>Mt1(31) Mt5(5) Mt3(3) Mt4(3) Mt7(2) Mt6(1) </td><td>Gm10(18) Gm20(11) Gm17(9) Gm1(7) Gm14(7) Gm8(5) Gm11(5) Gm12(5) Gm3(4) Gm9(4) Gm18(4) Gm5(3) Gm6(2) Gm7(2) Gm13(2) Gm15(2) Gm2(1) Gm4(1) Gm19(1) </td></tr><tr><td>CaLG5 </td><td>141 </td><td>119 </td><td>Mt3(34) Mt1(5) Mt7(5) Mt2(4) Mt5(4) Mt4(2) Mt6(1) Mt8(1) </td><td>Gm6(19) Gm4(18) Gm20(7) Gm14(6) Gm18(6) Gm13(5) Gm11(3)Gm1(2) Gm2(2) Gm8(2) Gm17(2)Gm3(1) Gm5(1) Gm7(1) Gm19(1) </td></tr><tr><td>CaLG6 </td><td>108 </td><td>86 </td><td>Mt8(15) Mt5(7) Mt7(5) Mt2(4) Mt3(4) Mt4(3) Mt6(3) Mt1(2) </td><td>Gm8(11) Gm5(9) Gm13(4) Gm18(4) Gm20(4) Gm6(3) Gm7(3) Gm11(3) Gm1(1) Gm2(1) Gm3(1) Gm10(1) Gm12(1) Gm14(1) Gm16(1) Gm17(1) Gm19(1) </td></tr><tr><td>CaLG7 </td><td>100 </td><td>81 </td><td>Mt8(13) Mt3(6) Mt2(5) Mt5(4) Mt7(4) Mt1(3) Mt4(3) Mt6(1) </td><td>Gm7(6) Gm14(6) Gm7(6) Gm14(6) Gm5(4) Gm6(4) Gm16(4) Gm17(4) Gm1(3) Gm2(3) Gm8(3) Gm11(3) Gm13(3) Gm18(3) Gm4(2) Gm9(2) Gm19(2) Gm12(1) Gm15(1) Gm20(1) </td></tr><tr><td>CaLG8 </td><td>84 </td><td>78 </td><td>Mt4(22) Mt2(4) Mt6(4) Mt1(3) Mt3(3) Mt7(3) Mt8(3) Mt5(2) </td><td>Gm17(20) Gm7(8) Gm13(8) Gm5(6) Gm6(6) Gm4(5) Gm9(4) Gm15(4) Gm10(3) Gm14(3) Gm18(3) Gm19(3) Gm3(2) Gm11(2) Gm12(2) Gm16(2) Gm20(2) Gm8(1) </td></tr><tr><td>Total </td><td>1063 </td><td>917 </td><td>404 </td><td>609 </td></tr></tbody></table></div><div content-id="DSS018TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chickpea LGs </th><th>Markers </th><th>Markers selected for blast </th><th>Mt chromosome (chickpea orthologs) </th><th>Gm chromosome (chickpea orthologs) </th></tr></thead><tbody><tr><td>CaLG1 </td><td>119 </td><td>107 </td><td>Mt2(35) Mt4(3) Mt1(2) Mt5(2) Mt7(2) Mt3(1) Mt6(1) Mt8(1) </td><td>Gm15(19) Gm13(17) Gm9(4) Gm8(3) Gm17(3) Gm3(2) Gm10(2) Gm11(2) Gm20(2) Gm1(1) Gm4(1) Gm5(1) Gm6(1) Gm7(1) Gm14(1) Gm19(1) </td></tr><tr><td>CaLG2 </td><td>138 </td><td>115 </td><td>Mt4(19) Mt5(9) Mt3(5) Mt6(4) Mt2(2) Mt7(2) Mt8(2) </td><td>Gm12(10) Gm11(7) Gm6(4) Gm9(4) Gm15(4) Gm18(4) Gm1(3) Gm5(3) Gm13(3) Gm20(3) Gm4(2) Gm7(2) Gm10(2) Gm14(2) Gm19(2) Gm3(1) </td></tr><tr><td>CaLG3 </td><td>205 </td><td>179 </td><td>Mt7(35) Mt5(23) Mt3(7) Mt2(6) Mt6(6) Mt8(4) Mt4(2) Mt1(3) </td><td>Gm19(29) Gm3(26) Gm1(12) Gm9(8) Gm10(7) Gm6(2) Gm7(6) Gm11(6) Gm13(5) Gm2(4) Gm5(4) Gm18(4) Gm20(4) Gm4(3) Gm6(2) Gm8(2) Gm12(2) Gm15(2) Gm16(2) Gm14(1) Gm17(1) </td></tr><tr><td>CaLG4 </td><td>168 </td><td>152 </td><td>Mt1(31) Mt5(5) Mt3(3) Mt4(3) Mt7(2) Mt6(1) </td><td>Gm10(18) Gm20(11) Gm17(9) Gm1(7) Gm14(7) Gm8(5) Gm11(5) Gm12(5) Gm3(4) Gm9(4) Gm18(4) Gm5(3) Gm6(2) Gm7(2) Gm13(2) Gm15(2) Gm2(1) Gm4(1) Gm19(1) </td></tr><tr><td>CaLG5 </td><td>141 </td><td>119 </td><td>Mt3(34) Mt1(5) Mt7(5) Mt2(4) Mt5(4) Mt4(2) Mt6(1) Mt8(1) </td><td>Gm6(19) Gm4(18) Gm20(7) Gm14(6) Gm18(6) Gm13(5) Gm11(3)Gm1(2) Gm2(2) Gm8(2) Gm17(2)Gm3(1) Gm5(1) Gm7(1) Gm19(1) </td></tr><tr><td>CaLG6 </td><td>108 </td><td>86 </td><td>Mt8(15) Mt5(7) Mt7(5) Mt2(4) Mt3(4) Mt4(3) Mt6(3) Mt1(2) </td><td>Gm8(11) Gm5(9) Gm13(4) Gm18(4) Gm20(4) Gm6(3) Gm7(3) Gm11(3) Gm1(1) Gm2(1) Gm3(1) Gm10(1) Gm12(1) Gm14(1) Gm16(1) Gm17(1) Gm19(1) </td></tr><tr><td>CaLG7 </td><td>100 </td><td>81 </td><td>Mt8(13) Mt3(6) Mt2(5) Mt5(4) Mt7(4) Mt1(3) Mt4(3) Mt6(1) </td><td>Gm7(6) Gm14(6) Gm7(6) Gm14(6) Gm5(4) Gm6(4) Gm16(4) Gm17(4) Gm1(3) Gm2(3) Gm8(3) Gm11(3) Gm13(3) Gm18(3) Gm4(2) Gm9(2) Gm19(2) Gm12(1) Gm15(1) Gm20(1) </td></tr><tr><td>CaLG8 </td><td>84 </td><td>78 </td><td>Mt4(22) Mt2(4) Mt6(4) Mt1(3) Mt3(3) Mt7(3) Mt8(3) Mt5(2) </td><td>Gm17(20) Gm7(8) Gm13(8) Gm5(6) Gm6(6) Gm4(5) Gm9(4) Gm15(4) Gm10(3) Gm14(3) Gm18(3) Gm19(3) Gm3(2) Gm11(2) Gm12(2) Gm16(2) Gm20(2) Gm8(1) </td></tr><tr><td>Total </td><td>1063 </td><td>917 </td><td>404 </td><td>609 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10617372" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS018TB3" class="table-wrap-title" data-id="DSS018TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Details of CaLGs, number of markers and synteny with <em>M. truncatula</em> and <em>G. max</em> chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chickpea LGs </th><th>Markers </th><th>Markers selected for blast </th><th>Mt chromosome (chickpea orthologs) </th><th>Gm chromosome (chickpea orthologs) </th></tr></thead><tbody><tr><td>CaLG1 </td><td>119 </td><td>107 </td><td>Mt2(35) Mt4(3) Mt1(2) Mt5(2) Mt7(2) Mt3(1) Mt6(1) Mt8(1) </td><td>Gm15(19) Gm13(17) Gm9(4) Gm8(3) Gm17(3) Gm3(2) Gm10(2) Gm11(2) Gm20(2) Gm1(1) Gm4(1) Gm5(1) Gm6(1) Gm7(1) Gm14(1) Gm19(1) </td></tr><tr><td>CaLG2 </td><td>138 </td><td>115 </td><td>Mt4(19) Mt5(9) Mt3(5) Mt6(4) Mt2(2) Mt7(2) Mt8(2) </td><td>Gm12(10) Gm11(7) Gm6(4) Gm9(4) Gm15(4) Gm18(4) Gm1(3) Gm5(3) Gm13(3) Gm20(3) Gm4(2) Gm7(2) Gm10(2) Gm14(2) Gm19(2) Gm3(1) </td></tr><tr><td>CaLG3 </td><td>205 </td><td>179 </td><td>Mt7(35) Mt5(23) Mt3(7) Mt2(6) Mt6(6) Mt8(4) Mt4(2) Mt1(3) </td><td>Gm19(29) Gm3(26) Gm1(12) Gm9(8) Gm10(7) Gm6(2) Gm7(6) Gm11(6) Gm13(5) Gm2(4) Gm5(4) Gm18(4) Gm20(4) Gm4(3) Gm6(2) Gm8(2) Gm12(2) Gm15(2) Gm16(2) Gm14(1) Gm17(1) </td></tr><tr><td>CaLG4 </td><td>168 </td><td>152 </td><td>Mt1(31) Mt5(5) Mt3(3) Mt4(3) Mt7(2) Mt6(1) </td><td>Gm10(18) Gm20(11) Gm17(9) Gm1(7) Gm14(7) Gm8(5) Gm11(5) Gm12(5) Gm3(4) Gm9(4) Gm18(4) Gm5(3) Gm6(2) Gm7(2) Gm13(2) Gm15(2) Gm2(1) Gm4(1) Gm19(1) </td></tr><tr><td>CaLG5 </td><td>141 </td><td>119 </td><td>Mt3(34) Mt1(5) Mt7(5) Mt2(4) Mt5(4) Mt4(2) Mt6(1) Mt8(1) </td><td>Gm6(19) Gm4(18) Gm20(7) Gm14(6) Gm18(6) Gm13(5) Gm11(3)Gm1(2) Gm2(2) Gm8(2) Gm17(2)Gm3(1) Gm5(1) Gm7(1) Gm19(1) </td></tr><tr><td>CaLG6 </td><td>108 </td><td>86 </td><td>Mt8(15) Mt5(7) Mt7(5) Mt2(4) Mt3(4) Mt4(3) Mt6(3) Mt1(2) </td><td>Gm8(11) Gm5(9) Gm13(4) Gm18(4) Gm20(4) Gm6(3) Gm7(3) Gm11(3) Gm1(1) Gm2(1) Gm3(1) Gm10(1) Gm12(1) Gm14(1) Gm16(1) Gm17(1) Gm19(1) </td></tr><tr><td>CaLG7 </td><td>100 </td><td>81 </td><td>Mt8(13) Mt3(6) Mt2(5) Mt5(4) Mt7(4) Mt1(3) Mt4(3) Mt6(1) </td><td>Gm7(6) Gm14(6) Gm7(6) Gm14(6) Gm5(4) Gm6(4) Gm16(4) Gm17(4) Gm1(3) Gm2(3) Gm8(3) Gm11(3) Gm13(3) Gm18(3) Gm4(2) Gm9(2) Gm19(2) Gm12(1) Gm15(1) Gm20(1) </td></tr><tr><td>CaLG8 </td><td>84 </td><td>78 </td><td>Mt4(22) Mt2(4) Mt6(4) Mt1(3) Mt3(3) Mt7(3) Mt8(3) Mt5(2) </td><td>Gm17(20) Gm7(8) Gm13(8) Gm5(6) Gm6(6) Gm4(5) Gm9(4) Gm15(4) Gm10(3) Gm14(3) Gm18(3) Gm19(3) Gm3(2) Gm11(2) Gm12(2) Gm16(2) Gm20(2) Gm8(1) </td></tr><tr><td>Total </td><td>1063 </td><td>917 </td><td>404 </td><td>609 </td></tr></tbody></table></div><div content-id="DSS018TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chickpea LGs </th><th>Markers </th><th>Markers selected for blast </th><th>Mt chromosome (chickpea orthologs) </th><th>Gm chromosome (chickpea orthologs) </th></tr></thead><tbody><tr><td>CaLG1 </td><td>119 </td><td>107 </td><td>Mt2(35) Mt4(3) Mt1(2) Mt5(2) Mt7(2) Mt3(1) Mt6(1) Mt8(1) </td><td>Gm15(19) Gm13(17) Gm9(4) Gm8(3) Gm17(3) Gm3(2) Gm10(2) Gm11(2) Gm20(2) Gm1(1) Gm4(1) Gm5(1) Gm6(1) Gm7(1) Gm14(1) Gm19(1) </td></tr><tr><td>CaLG2 </td><td>138 </td><td>115 </td><td>Mt4(19) Mt5(9) Mt3(5) Mt6(4) Mt2(2) Mt7(2) Mt8(2) </td><td>Gm12(10) Gm11(7) Gm6(4) Gm9(4) Gm15(4) Gm18(4) Gm1(3) Gm5(3) Gm13(3) Gm20(3) Gm4(2) Gm7(2) Gm10(2) Gm14(2) Gm19(2) Gm3(1) </td></tr><tr><td>CaLG3 </td><td>205 </td><td>179 </td><td>Mt7(35) Mt5(23) Mt3(7) Mt2(6) Mt6(6) Mt8(4) Mt4(2) Mt1(3) </td><td>Gm19(29) Gm3(26) Gm1(12) Gm9(8) Gm10(7) Gm6(2) Gm7(6) Gm11(6) Gm13(5) Gm2(4) Gm5(4) Gm18(4) Gm20(4) Gm4(3) Gm6(2) Gm8(2) Gm12(2) Gm15(2) Gm16(2) Gm14(1) Gm17(1) </td></tr><tr><td>CaLG4 </td><td>168 </td><td>152 </td><td>Mt1(31) Mt5(5) Mt3(3) Mt4(3) Mt7(2) Mt6(1) </td><td>Gm10(18) Gm20(11) Gm17(9) Gm1(7) Gm14(7) Gm8(5) Gm11(5) Gm12(5) Gm3(4) Gm9(4) Gm18(4) Gm5(3) Gm6(2) Gm7(2) Gm13(2) Gm15(2) Gm2(1) Gm4(1) Gm19(1) </td></tr><tr><td>CaLG5 </td><td>141 </td><td>119 </td><td>Mt3(34) Mt1(5) Mt7(5) Mt2(4) Mt5(4) Mt4(2) Mt6(1) Mt8(1) </td><td>Gm6(19) Gm4(18) Gm20(7) Gm14(6) Gm18(6) Gm13(5) Gm11(3)Gm1(2) Gm2(2) Gm8(2) Gm17(2)Gm3(1) Gm5(1) Gm7(1) Gm19(1) </td></tr><tr><td>CaLG6 </td><td>108 </td><td>86 </td><td>Mt8(15) Mt5(7) Mt7(5) Mt2(4) Mt3(4) Mt4(3) Mt6(3) Mt1(2) </td><td>Gm8(11) Gm5(9) Gm13(4) Gm18(4) Gm20(4) Gm6(3) Gm7(3) Gm11(3) Gm1(1) Gm2(1) Gm3(1) Gm10(1) Gm12(1) Gm14(1) Gm16(1) Gm17(1) Gm19(1) </td></tr><tr><td>CaLG7 </td><td>100 </td><td>81 </td><td>Mt8(13) Mt3(6) Mt2(5) Mt5(4) Mt7(4) Mt1(3) Mt4(3) Mt6(1) </td><td>Gm7(6) Gm14(6) Gm7(6) Gm14(6) Gm5(4) Gm6(4) Gm16(4) Gm17(4) Gm1(3) Gm2(3) Gm8(3) Gm11(3) Gm13(3) Gm18(3) Gm4(2) Gm9(2) Gm19(2) Gm12(1) Gm15(1) Gm20(1) </td></tr><tr><td>CaLG8 </td><td>84 </td><td>78 </td><td>Mt4(22) Mt2(4) Mt6(4) Mt1(3) Mt3(3) Mt7(3) Mt8(3) Mt5(2) </td><td>Gm17(20) Gm7(8) Gm13(8) Gm5(6) Gm6(6) Gm4(5) Gm9(4) Gm15(4) Gm10(3) Gm14(3) Gm18(3) Gm19(3) Gm3(2) Gm11(2) Gm12(2) Gm16(2) Gm20(2) Gm8(1) </td></tr><tr><td>Total </td><td>1063 </td><td>917 </td><td>404 </td><td>609 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10617372" target="_blank">View Large</a></div></div><p><em>Chickpea-Medicago synteny</em><em>.</em> The chickpea-<em>Medicago</em> synteny was observed for 315 (34.35%) of 917 markers for which 404 significant matches were obtained on different chromosomes of <em>Medicago</em>. Synteny blocks were observed between chickpea and most <em>Medicago</em> chromosomes except Mt6 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), indicating an evolutionary relationship between the two species. Many of the CaLGs were found to have syntenic blocks with more than one LG of <em>Medicago</em>. CaLG3 showed maximum synteny, since 68 markers showed 86 hits with <em>Medicago</em> chromosomes (Table <a reveal-id="DSS018TB3" data-open="DSS018TB3" class="link link-reveal link-table xref-fig">3</a>), majority of which (35 and 23) were located on Mt7 and Mt5, respectively (Fig. <a reveal-id="DSS018F3" data-open="DSS018F3" class="link link-reveal link-table xref-fig">3</a>). Similarly, synteny blocks from CaLG2 were observed on Mt4, Mt5 and Mt6 (Fig. <a reveal-id="DSS018F3" data-open="DSS018F3" class="link link-reveal link-table xref-fig">3</a>). Moreover, CaLG1, CaLG4 and CaLG5 showed distinct synteny with Mt2, Mt1 and Mt3 chromosomes, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs. S1 and S2</a>). Additionally, CaLG6 and CaLG7 were seen to be highly syntenic with Mt8, whereas CaLG8 was found to be syntenic with Mt4 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs. S1 and S2</a>). The number of markers that found hits in <em>Medicago</em> in comparison with soybean was more; however, very limited collinearity was observed between the chickpea-<em>Medicago</em> syntenic blocks. </p>                        <a id="10617374" scrollto-destination="10617374"></a>
<div id="DSS018F3" data-id="DSS018F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/m_dss01803.jpeg?Expires=2147483647&Signature=LD7JWaTmFUxhg9cWrT5dBVoycb8COZATzfROId7FTThoruI0f2mrptKzyscUePCdyq2xNjxh-F93BkXbuq9JaTnHgc2UmgjXsUgwunlomOVnjaT2wU7XkTMn8eHo7K8OMTvpLIqbMNjXqNs9LnUo7C0aG6kRyJbwdmQ~sgXNQzlB-uGGU0GMSp-Pja0wX5bD0pOetvv2~5KfnRoa4pobTTWQimVopcBPKEOR5Ue2xT9p64eNLCcwAizzQCSRRmXawS49iyb~JBbSccur8tPKYOtWb-22p4X~ftbJL4jMh9NtTq0Pa8iaxkf2t08yk1cT1DbWMCqmX3YIM2p2VfSSNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Syntenic relationships of chickpea LGs: CaLG2 with Mt4, Mt5 and Mt6; CaLG3 with Mt5 and Mt7; CaLG1 with Gm9, Gm13 and Gm15; CaLG3 with Gm1, Gm3 and Gm19. Coloured regions represent syntenic blocks observed in chromosomes of Medicago and soybean." path-from-xml="dss01803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10617374/dss01803.jpeg" path-from-xml="dss01803.jpeg" target="_blank">View large</a><a section="10617374" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss01803.jpeg?Expires=2147483647&Signature=A1XFr540fnBsoWNAmZ4sYVunhyjiBxsi90H3IiDE-qluAf5dmz29vgGUuMGqO5QoK7bHC2WBvIx17zF0-ECJ7hc6KeRRT-1E6L1VZOkM~-E4sqSoTf4bR95c8cq0-VdMNNu925HxkEvDl~xyvETAFR28JOzUxfMZJQVIOnfjOg2IzZdgg8tuG7Rj2LV0C2~A~0kAx25Pqg9LwXf-3bAtJ8vGmx1kSgUni04OOpIV3pOs-1Y3WG1VknUt5nLbXy1zvBrSVFrsullZSpZrW75BszWGdSzG67aRAV81Win2HOh6ABqQxWavomeiOfJWcskmt4PYME6eTZuZ-FuQRIFTLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10617374&ar=628389&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Syntenic relationships of chickpea LGs: CaLG2 with Mt4, Mt5 and Mt6; CaLG3 with Mt5 and Mt7; CaLG1 with Gm9, Gm13 and Gm15; CaLG3 with Gm1, Gm3 and Gm19. Coloured regions represent syntenic blocks observed in chromosomes of <em>Medicago</em> and soybean.</p></div></div><div content-id="DSS018F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/m_dss01803.jpeg?Expires=2147483647&Signature=LD7JWaTmFUxhg9cWrT5dBVoycb8COZATzfROId7FTThoruI0f2mrptKzyscUePCdyq2xNjxh-F93BkXbuq9JaTnHgc2UmgjXsUgwunlomOVnjaT2wU7XkTMn8eHo7K8OMTvpLIqbMNjXqNs9LnUo7C0aG6kRyJbwdmQ~sgXNQzlB-uGGU0GMSp-Pja0wX5bD0pOetvv2~5KfnRoa4pobTTWQimVopcBPKEOR5Ue2xT9p64eNLCcwAizzQCSRRmXawS49iyb~JBbSccur8tPKYOtWb-22p4X~ftbJL4jMh9NtTq0Pa8iaxkf2t08yk1cT1DbWMCqmX3YIM2p2VfSSNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Syntenic relationships of chickpea LGs: CaLG2 with Mt4, Mt5 and Mt6; CaLG3 with Mt5 and Mt7; CaLG1 with Gm9, Gm13 and Gm15; CaLG3 with Gm1, Gm3 and Gm19. Coloured regions represent syntenic blocks observed in chromosomes of Medicago and soybean." path-from-xml="dss01803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10617374/dss01803.jpeg" path-from-xml="dss01803.jpeg" target="_blank">View large</a><a section="10617374" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss01803.jpeg?Expires=2147483647&Signature=A1XFr540fnBsoWNAmZ4sYVunhyjiBxsi90H3IiDE-qluAf5dmz29vgGUuMGqO5QoK7bHC2WBvIx17zF0-ECJ7hc6KeRRT-1E6L1VZOkM~-E4sqSoTf4bR95c8cq0-VdMNNu925HxkEvDl~xyvETAFR28JOzUxfMZJQVIOnfjOg2IzZdgg8tuG7Rj2LV0C2~A~0kAx25Pqg9LwXf-3bAtJ8vGmx1kSgUni04OOpIV3pOs-1Y3WG1VknUt5nLbXy1zvBrSVFrsullZSpZrW75BszWGdSzG67aRAV81Win2HOh6ABqQxWavomeiOfJWcskmt4PYME6eTZuZ-FuQRIFTLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10617374&ar=628389&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Syntenic relationships of chickpea LGs: CaLG2 with Mt4, Mt5 and Mt6; CaLG3 with Mt5 and Mt7; CaLG1 with Gm9, Gm13 and Gm15; CaLG3 with Gm1, Gm3 and Gm19. Coloured regions represent syntenic blocks observed in chromosomes of <em>Medicago</em> and soybean.</p></div></div><p><em>Chickpea-soybean synteny</em>. In chickpea-soybean synteny analysis, 272 (30%) markers found a larger number of hits (609) in the soybean genome (Table <a reveal-id="DSS018TB3" data-open="DSS018TB3" class="link link-reveal link-table xref-fig">3</a>) in comparison with <em>Medicago</em>. However, regions with macrosynteny, microsynteny and collinearity were observed. All CaLGs showed syntenic blocks with Gm chromosomes except CaLG7 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In the case of soybean also (as with <em>Medicago</em>), CaLG3 (54 markers) found maximum hits (129) in the soybean genome, of which majority were found on Gm19, Gm3 and Gm1 (Table <a reveal-id="DSS018TB3" data-open="DSS018TB3" class="link link-reveal link-table xref-fig">3</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). All CaLGs (except CaLG7) showed multiple synteny blocks with the Gm chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). CaLG5 showed clear correspondence with regions in Gm4 and Gm6 in which the marker order was almost collinear. High collinearity was also observed between CaLG1 with Gm13, Gm15 and Gm9 (Fig. <a reveal-id="DSS018F3" data-open="DSS018F3" class="link link-reveal link-table xref-fig">3</a>), followed by LG4 with Gm17. Similarly, collinear regions were also identified between CaLG6 and Gm5 and Gm8 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                            <h2 scrollto-destination=10617376 id="10617376" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The present study of the chickpea genome reports the large-scale identification of SNPs from the NGS of two contrasting genotypes, which were parents of an RIL mapping population. The development and validation of the high-throughput GoldenGate SNP genotyping assay was demonstrated for the first time in chickpea with a very high success rate of 90.75% (since 697 of 768 SNPs were genotyped). Further, these SNPs were assigned map locations in the backdrop of other co-dominant markers to generate one of the most dense genetic linkage maps of chickpea, thereby clearly establishing the quality of the data and the utility of this approach. This study served to expand the repertoire of genomic resources in chickpea by providing the first set of 696 validated and mapped SNP markers, which are the most preferred next generation markers for molecular breeding, association studies and comparative genomics. Recently, there has been a spurt in the large-scale discovery of SNPs in plants, especially after the public availability of EST databases and more recently with the continuously expanding NGS of genomes and transcriptomes. SNP identification in plants range from a few thousand such as 1114 in wheat,<sup><a reveal-id="DSS018C15" data-open="DSS018C15" class="link link-ref link-reveal xref-bibr">15</a></sup> 2700 in <em>Arabidopsis</em>,<sup><a reveal-id="DSS018C69" data-open="DSS018C69" class="link link-ref link-reveal xref-bibr">69</a></sup> 5000 in tomato,<sup><a reveal-id="DSS018C40" data-open="DSS018C40" class="link link-ref link-reveal xref-bibr">40</a></sup> 6626 in pine,<sup><a reveal-id="DSS018C16" data-open="DSS018C16" class="link link-ref link-reveal xref-bibr">16</a></sup> 10 000 in cowpea,<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> 9194–36 000 in maize,<sup><a reveal-id="DSS018C17" data-open="DSS018C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS018C42" data-open="DSS018C42" class="link link-ref link-reveal xref-bibr">42</a></sup> 22 000 in barley,<sup><a reveal-id="DSS018C32" data-open="DSS018C32" class="link link-ref link-reveal xref-bibr">32</a></sup> 25 000–39 022 in soybean,<sup><a reveal-id="DSS018C18" data-open="DSS018C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS018C70" data-open="DSS018C70" class="link link-ref link-reveal xref-bibr">70</a></sup> 40 000 in <em>Medicago</em><sup><a reveal-id="DSS018C19" data-open="DSS018C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and 67 051 in rice,<sup><a reveal-id="DSS018C71" data-open="DSS018C71" class="link link-ref link-reveal xref-bibr">71</a></sup> to millions in <em>Arabidopsis</em> (2.5 million),<sup><a reveal-id="DSS018C72" data-open="DSS018C72" class="link link-ref link-reveal xref-bibr">72</a></sup> rice (2.4–6.5 million),<sup><a reveal-id="DSS018C7" data-open="DSS018C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSS018C73" data-open="DSS018C73" class="link link-ref link-reveal xref-bibr">73</a></sup><em>Medicago</em> (3 million)<sup><a reveal-id="DSS018C38" data-open="DSS018C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and soybean (6.5 million).<sup><a reveal-id="DSS018C74" data-open="DSS018C74" class="link link-ref link-reveal xref-bibr">74</a></sup> The thousands of SNPs generated are being extensively used for various applications such as the analysis of genetic diversity,<sup><a reveal-id="DSS018C22" data-open="DSS018C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSS018C74" data-open="DSS018C74" class="link link-ref link-reveal xref-bibr">74</a></sup> map construction<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS018C15" data-open="DSS018C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS018C32" data-open="DSS018C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSS018C40" data-open="DSS018C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and orienting the scaffolds in whole genome sequencing projects.<sup><a reveal-id="DSS018C18" data-open="DSS018C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS018C43" data-open="DSS018C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p><p>SNP discovery from <em>de novo</em> sequencing of genomes is a challenge especially in the absence of reference genomes. In the case of chickpea, since no reference genome was available, in the present study, SNPs were deduced by sequence comparisons between chickpea and its wild relative, <em>C. reticulatum</em>, with the aim of producing a highly useful set of molecular markers, which would be directly used for constructing a dense linkage map of chickpea. Consensus calling was done by <em>de novo</em> assembly of the Roche/454 long reads of <em>C. arietinum</em> ICC4958 to generate the reference sequence, onto which the ICC4958 SOLiD short reads were mapped to generate a consensus sequence. Next, SNP calling was done by mapping the SOLiD reads of the contrasting genotype, <em>C. reticulatum</em> PI489777, onto the consensus sequence. Moreover, in order to custom design a genotyping array consisting of SNPs well-distributed throughout the genome, a predetermined number of SNPs were selected from the largest sized contigs. This array proved useful for establishing the efficacy of the SNP genotyping strategy for high-resolution mapping, which is mandatory for MAS, association analysis and anchoring the chickpea genome.</p><p>Many factors may be responsible for the overall success of SNP genotyping. Firstly, SNP prediction from NGS data should be based on stringent parameters. When calling variants (especially from short reads), the quantity of reads and the accuracy of mapping the reads should be precise. This was ensured in our data since we used stringent criteria for SNP calling such as base quality ≥20, minimum read depth of 5X in both genotypes and frequency score &gt;0.6. Secondly, the nucleotides flanking the query SNP also influence the rate of success of SNP development.<sup><a reveal-id="DSS018C16" data-open="DSS018C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSS018C23" data-open="DSS018C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In our study also, care was taken to select only those SNPs that had no variations in the 60 bases flanking it. Moreover, the Illumina ADT which provides designability scores, checks for the presence of repetitive and palindromic sequences, GC content and sequence polymorphisms in the region flanking the query SNP. High designability scores were obtained for the 920 predicted chickpea SNPs of which 851 (92.5%) had an ADT designability score ≥0.6 indicating a high success rate for the conversion of an NGS-derived SNP into a GGGT assay. Even after combining with the 102 ESNPs, designability scores ≥0.6 were obtained for 92.37% of the predicted SNPs. From these, since only 768 SNPs had to be selected for the OPA design, those having scores ≥0.76 (much higher than the threshold values) were selected for the 768-CpOPA-I. Inclusion of SNPs with such high designability scores accounted for the overall success of the genotyping assay as has also been demonstrated in pine.<sup><a reveal-id="DSS018C16" data-open="DSS018C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Moreover, 48 CaSNPs predicted from the NGS data were experimentally validated by resequencing the alleles of the two parental varieties and a success rate of 87.5% was obtained (data not shown).</p><p>In the present study, we also demonstrated that the high-throughput GoldenGate technology could be successfully used for genotyping predicted SNPs by conversion into working assays. This high-throughput platform of Illumina, which ensures a high level of multiplexing and reliability, has been suitably used in plants,<sup><a reveal-id="DSS018C12" data-open="DSS018C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS018C16" data-open="DSS018C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSS018C32" data-open="DSS018C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSS018C33" data-open="DSS018C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSS018C75" data-open="DSS018C75" class="link link-ref link-reveal xref-bibr">75</a>,<a reveal-id="DSS018C76" data-open="DSS018C76" class="link link-ref link-reveal xref-bibr">76</a></sup> where large numbers of SNPs need to be surveyed. In the present chickpea OPA, the success rate of 90.75% was obtained, which was comparable with the success rates of 89–92% reported in the previous studies in other crops such as barley,<sup><a reveal-id="DSS018C75" data-open="DSS018C75" class="link link-ref link-reveal xref-bibr">75</a></sup> soybean,<sup><a reveal-id="DSS018C31" data-open="DSS018C31" class="link link-ref link-reveal xref-bibr">31</a></sup> cowpea,<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> maize<sup><a reveal-id="DSS018C33" data-open="DSS018C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and eucalyptus,<sup><a reveal-id="DSS018C20" data-open="DSS018C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and was higher than those reported in spruce<sup><a reveal-id="DSS018C12" data-open="DSS018C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and pine.<sup><a reveal-id="DSS018C16" data-open="DSS018C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSS018C34" data-open="DSS018C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The SNP success rate may be assessed in various ways and different SNP reliability thresholds may be used. The SNP success rate of 90.7% reported for chickpea was based on the percentage of successfully genotyped SNPs. However, if the criteria of the call rate (&gt;95%) were used, the success rate of 99.8% was achieved. The success rate was 93.0% when considering the GenTrain and GenCall50 score of ≥0.4 as thresholds, as described in Grattapaglia <em>et al</em>.<sup><a reveal-id="DSS018C20" data-open="DSS018C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p><p>We present here the first SNP-based linkage map of chickpea constructed using 697 SNPs in combination with 368 previously mapped co-dominant markers including a large number of SSRs (Fig. <a reveal-id="DSS018F2" data-open="DSS018F2" class="link link-reveal link-table xref-fig">2</a>). In the past few years, SSR markers have been the most preferred markers for constructing linkage maps due to their hypervariability, co-dominant nature and reproducibility, and SSR-based maps have been constructed in many plants such as wheat,<sup><a reveal-id="DSS018C77" data-open="DSS018C77" class="link link-ref link-reveal xref-bibr">77</a></sup> barley,<sup><a reveal-id="DSS018C78" data-open="DSS018C78" class="link link-ref link-reveal xref-bibr">78</a></sup> tobacco,<sup><a reveal-id="DSS018C79" data-open="DSS018C79" class="link link-ref link-reveal xref-bibr">79</a></sup> tomato<sup><a reveal-id="DSS018C80" data-open="DSS018C80" class="link link-ref link-reveal xref-bibr">80</a></sup> and <em>Catharanthus</em>,<sup><a reveal-id="DSS018C81" data-open="DSS018C81" class="link link-ref link-reveal xref-bibr">81</a></sup> and legumes like chickpea<sup><a reveal-id="DSS018C46" data-open="DSS018C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and soybean.<sup><a reveal-id="DSS018C82" data-open="DSS018C82" class="link link-ref link-reveal xref-bibr">82</a></sup> However, SSRs were found to have limitations, since they were not sufficiently dense to provide the genomic coverage necessary for the dissection of complex traits. Moreover, automation in high-throughput genotyping of SSRs was not very common. Therefore, SNPs are now proposed as the most useful markers, especially for the construction of highly saturated maps, which may have very wide and novel applications in plant genomics. Hence, modern sequencing technologies are being used to generate thousands to millions of SNPs in many plant species as mentioned previously. However, when it comes to the validation of SNPs and their use in genotyping and map generation, there are only a few examples of SNP-based high-density maps. The map of chickpea generated in this study contained 697 SNPs—much higher than the 357 in pine,<sup><a reveal-id="DSS018C16" data-open="DSS018C16" class="link link-ref link-reveal xref-bibr">16</a></sup> 480 in wheat<sup><a reveal-id="DSS018C15" data-open="DSS018C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and 558 in grass,<sup><a reveal-id="DSS018C41" data-open="DSS018C41" class="link link-ref link-reveal xref-bibr">41</a></sup> but lower than the 793 in tomato,<sup><a reveal-id="DSS018C40" data-open="DSS018C40" class="link link-ref link-reveal xref-bibr">40</a></sup> 928 in cowpea,<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> 1397 in soybean<sup><a reveal-id="DSS018C31" data-open="DSS018C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and 2943 in barley.<sup><a reveal-id="DSS018C32" data-open="DSS018C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In addition to the SNPs, 368 previously mapped co-dominant markers were positioned in this chickpea map thereby generating one of the most advanced integrated maps of chickpea comprising of 1063 markers with an average marker density of 1.7 cM. This map was more saturated in comparison to other published maps of chickpea such as those of Winter <em>et al</em>.<sup><a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> (2077.9 cM: 303 markers: AMD of 6.8 cM), Nayak <em>et al</em>.<sup><a reveal-id="DSS018C45" data-open="DSS018C45" class="link link-ref link-reveal xref-bibr">45</a></sup> (2602.1 cM: 521 markers: AMD of 4.99 cM), Gujaria <em>et al</em>.<sup><a reveal-id="DSS018C47" data-open="DSS018C47" class="link link-ref link-reveal xref-bibr">47</a></sup> (766.56 cM: 300 markers: AMD of 2.552 cM) and Choudhary <em>et al</em>.<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> (1497.7 cM; 406 markers; AMD of 3.68). A distinguishing feature of this map was that it was constructed based on a single reference population (not on integration of maps from different populations), utilizing actual segregation data of each of the genomic and genic markers listed in Table <a reveal-id="DSS018TB1" data-open="DSS018TB1" class="link link-reveal link-table xref-fig">1</a>. Consensus maps are generated in order to increase the number of mapped markers, but may compromise on the accuracy of the marker order. Hence our map may be considered as more accurate regarding the marker order on the eight linkage groups. This was confirmed as considerable conservation of marker distribution existed between the LGs of the current map and our previous map<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> as well as the anchor map of Winter <em>et al</em>.<sup><a reveal-id="DSS018C58" data-open="DSS018C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Moreover, this map was a major advancement over previous maps based on co-dominant markers,<sup><a reveal-id="DSS018C45" data-open="DSS018C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSS018C47" data-open="DSS018C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> as the mapped markers were more than doubled in the current map. Additionally, nine genic markers (four EST-SSRs, two ITPs and three ESTPs), which remained unmapped in our previous inter-specific map, were also mapped onto LG2, LG3, LG5, LG6, LG7 and LG8 in the current map. The average marker density in each of the LGs in the current map was also found to be increased (0.95–2.65) in comparison with the density reported earlier (1.77–8.01).<sup><a reveal-id="DSS018C48" data-open="DSS018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Therefore, it was envisaged that this highly advanced map would not only be useful in chickpea breeding but would particularly aid and complement the efforts of orienting scaffolds of chickpea whole genome sequence being generated during its <em>de novo</em> sequencing at the institute. High-resolution genetic maps are critical for the assembly of sequence scaffolds into pseudomolecules corresponding to chromosomes of organisms and have been used for anchoring and orienting scaffolds in the soybean,<sup><a reveal-id="DSS018C18" data-open="DSS018C18" class="link link-ref link-reveal xref-bibr">18</a></sup> watermelon,<sup><a reveal-id="DSS018C43" data-open="DSS018C43" class="link link-ref link-reveal xref-bibr">43</a></sup> grape,<sup><a reveal-id="DSS018C83" data-open="DSS018C83" class="link link-ref link-reveal xref-bibr">83</a></sup> cucumber<sup><a reveal-id="DSS018C84" data-open="DSS018C84" class="link link-ref link-reveal xref-bibr">84</a></sup> and apple<sup><a reveal-id="DSS018C85" data-open="DSS018C85" class="link link-ref link-reveal xref-bibr">85</a></sup> whole genome sequencing.</p><p>Genomic synteny can facilitate the transfer of genetic information especially between orphan and model crops. In legumes, map-based synteny has been shown in several studies such as cowpea,<sup><a reveal-id="DSS018C13" data-open="DSS018C13" class="link link-ref link-reveal xref-bibr">13</a></sup><em>Medicago</em>,<sup><a reveal-id="DSS018C86" data-open="DSS018C86" class="link link-ref link-reveal xref-bibr">86</a></sup> peanut,<sup><a reveal-id="DSS018C87" data-open="DSS018C87" class="link link-ref link-reveal xref-bibr">87</a></sup> common bean<sup><a reveal-id="DSS018C88" data-open="DSS018C88" class="link link-ref link-reveal xref-bibr">88</a></sup> and asparagus bean.<sup><a reveal-id="DSS018C89" data-open="DSS018C89" class="link link-ref link-reveal xref-bibr">89</a></sup> However, due to the lack of genomic resources, information on genome organization and comparative analysis was limited in chickpea. In this study, comparative analysis of chickpea using genic and genomic marker sequences with the model legume <em>Medicago</em> and soybean was done, which revealed higher levels of synteny with <em>Medicago</em> in comparison with soybean. This result indicated that the degree of synteny declines with increasing phylogenetic distance, as chickpea and <em>Medicago</em> both belong to the Galegoid clade, whereas soybean belongs to the Phaseoloid clade. Chromosomal rearrangements seemed to have occurred since most of the chickpea LGs were chimeric to two or more Gm chromosomal segments. Moreover, some segments of chickpea such as those on CaLG3 were found to have blocks on two Gm chromosomes (Gm3 and Gm19), thereby indicating the phenomenon of genome duplication.</p><p>In conclusion, the present study illustrates the application of NGS technology for the large-scale identification of SNPs from species without reference genomes such as chickpea. It also demonstrates that the high-throughput genotyping technologies such as the Illumina GoldenGate assay can be successfully used for genotyping and subsequently mapping the SNPs and integrating them with other co-dominant markers for the construction of a dense genetic map of chickpea. The availability of such large numbers of SNP markers, advanced genotyping technologies and the high-density marker map would serve as a foundation not only for orienting and anchoring of scaffolds arising out of whole genome sequencing projects but also for association mapping and synteny-based genomics in chickpea and other legumes.</p>                            <h2 scrollto-destination=10617384 id="10617384" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournal.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=10617386 id="10617386" class="section-title" data-legacy-id=s6>Funding</h2>
<p>We gratefully acknowledge the funding support by the Department of Biotechnology (DBT), Government of India, through their research grant (BT/PR12919/AGR/02/676/2009) for this work.</p>                            <h2 scrollto-destination=10617388 id="10617388" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are thankful to Dr Fred Muehlbauer, Washington State University, USA, for providing the inter-specific chickpea mapping population and genotypic data of published STMS markers.</p>                            <h2 scrollto-destination=10617390 id="10617390" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS018C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krawczak</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Informativity assessment for biallelic single nucleotide polymorphisms</div>, <div class="source">Electrophoresis</div> , <div class="year">1999</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1676</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Informativity%20assessment%20for%20biallelic%20single%20nucleotide%20polymorphisms&amp;author=M.%20Krawczak&amp;publication_year=1999&amp;journal=Electrophoresis&amp;volume=20&amp;pages=1676-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1522-2683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1522-2683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1522-2683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10435431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xing</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schumacher</div> <div class="given-names">F.R.</div></div>,  <div class="name"><div class="surname">Xing</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Elston</div> <div class="given-names">R.C.</div></div>. </div><div class="article-title">Comparison of microsatellites, single-nucleotide polymorphisms (SNPs) and composite markers derived from SNPs in linkage analysis</div>, <div class="source">BMC Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">S29</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20microsatellites%2C%20single-nucleotide%20polymorphisms%20(SNPs)%20and%20composite%20markers%20derived%20from%20SNPs%20in%20linkage%20analysis&amp;author=C.%20Xing&amp;author=F.R.%20Schumacher&amp;author=G.%20Xing&amp;author=Q.%20Lu&amp;author=T.%20Wang&amp;author=R.C.%20Elston&amp;publication_year=2005&amp;journal=BMC%20Genet.&amp;volume=6&amp;pages=S29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-6-S1-S29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-6-S1-S29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-6-S1-S29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16451638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cooper</div> <div class="given-names">D.N.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Cooke</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Niemann</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schmidtke</div> <div class="given-names">J.</div></div>. </div><div class="article-title">An estimate of unique DNA sequence heterozygosity in the human genome</div>, <div class="source">Hum. Genet.</div> , <div class="year">1985</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">201</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20estimate%20of%20unique%20DNA%20sequence%20heterozygosity%20in%20the%20human%20genome&amp;author=D.N.%20Cooper&amp;author=B.A.%20Smith&amp;author=H.J.%20Cooke&amp;author=S.%20Niemann&amp;author=J.%20Schmidtke&amp;publication_year=1985&amp;journal=Hum.%20Genet.&amp;volume=69&amp;pages=201-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00293024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00293024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00293024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2984104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">J.-B.</div></div>,  <div class="name"><div class="surname">Siao</div> <div class="given-names">C.-J.</div></div>, et al. </div><div class="article-title">Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome</div>, <div class="source">Science</div> , <div class="year">1998</div>, vol. <div class="volume">280</div> (pg. <div class="fpage">1077</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20identification%2C%20mapping%2C%20and%20genotyping%20of%20single-nucleotide%20polymorphisms%20in%20the%20human%20genome&amp;author=D.G.%20Wang&amp;author=J.-B.%20Fan&amp;author=C.-J.%20Siao&amp;publication_year=1998&amp;journal=Science&amp;volume=280&amp;pages=1077-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.280.5366.1077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.280.5366.1077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.280.5366.1077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9582121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kulheim</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Yeoh</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Maintz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Foley</div> <div class="given-names">W.J.</div></div>,  <div class="name"><div class="surname">Moran</div> <div class="given-names">G.F.</div></div>. </div><div class="article-title">Comparative SNP diversity among four <em>Eucalyptus</em> species for genes from secondary metabolite biosynthetic pathways</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">24</div> pg. <div class="fpage">452</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20SNP%20diversity%20among%20four%20Eucalyptus%20species%20for%20genes%20from%20secondary%20metabolite%20biosynthetic%20pathways&amp;author=C.%20Kulheim&amp;author=S.H.%20Yeoh&amp;author=J.%20Maintz&amp;author=W.J.%20Foley&amp;author=G.F.%20Moran&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=24&amp;pages=452" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-452" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-452" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-452"> </span></div> </div></div></div></div></div><div content-id="DSS018C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Extensive chromosome homoeology among <em>Brassiceae</em> species were revealed by comparative genetic mapping with high-density EST-based SNP markers in radish (<em>Raphanus sativus</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">401</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20chromosome%20homoeology%20among%20Brassiceae%20species%20were%20revealed%20by%20comparative%20genetic%20mapping%20with%20high-density%20EST-based%20SNP%20markers%20in%20radish%20(Raphanus%20sativus%20L.)&amp;author=F.%20Li&amp;author=Y.%20Hasegawa&amp;author=M.%20Saito&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=401-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Subbaiyan</div> <div class="given-names">G.K.</div></div>,  <div class="name"><div class="surname">Waters</div> <div class="given-names">D.L.E.</div></div>,  <div class="name"><div class="surname">Katiyar</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Sadananda</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Vaddadi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Henry</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Genome-wide DNA polymorphisms in elite <em>indica</em> rice inbreds discovered by whole-genome sequencing</div>, <div class="source">Plant Biotech. J.</div> , <div class="year">2012</div>(pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1467-7652.2011.00676.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1467-7652.2011.00676.x</a></div></div></div></div></div><div content-id="DSS018C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gore</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Wright</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Ersoz</div> <div class="given-names">E.S.</div></div>, et al. </div><div class="article-title">Large-scale discovery of gene-enriched SNPs</div>, <div class="source">Plant Genome</div> , <div class="year">2009</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">121</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20discovery%20of%20gene-enriched%20SNPs&amp;author=M.A.%20Gore&amp;author=M.H.%20Wright&amp;author=E.S.%20Ersoz&amp;publication_year=2009&amp;journal=Plant%20Genome&amp;volume=2&amp;pages=121-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2009.01.0002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2009.01.0002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2009.01.0002"> </span></div> </div></div></div></div></div><div content-id="DSS018C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">I.-Y.</div></div>,  <div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Matukumalli</div> <div class="given-names">L.K.</div></div>, et al. </div><div class="article-title">A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis</div>, <div class="source">Genetics</div> , <div class="year">2007</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">685</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20soybean%20transcript%20map%3A%20gene%20distribution%2C%20haplotype%20and%20single-nucleotide%20polymorphism%20analysis&amp;author=I.-Y.%20Choi&amp;author=D.L.%20Hyten&amp;author=L.K.%20Matukumalli&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=685-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.070821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.070821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.070821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17339218" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Atwell</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Vilhjalmsson</div> <div class="given-names">B.J.</div></div>, et al. </div><div class="article-title">Genome-wide association study of 107 phenotypes in <em>Arabidopsis thaliana</em> inbred lines</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">465</div> (pg. <div class="fpage">627</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20107%20phenotypes%20in%20Arabidopsis%20thaliana%20inbred%20lines&amp;author=S.%20Atwell&amp;author=Y.S.%20Huang&amp;author=B.J.%20Vilhjalmsson&amp;publication_year=2010&amp;journal=Nature&amp;volume=465&amp;pages=627-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08800" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08800" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08800"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20336072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Malacarne</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Coppola</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">A dense single-nucleotide polymorphism-based genetic linkage map of grapevine (<em>Vitis vinifera</em> L.) anchoring Pinot Noir bacterial artificial chromosome contigs</div>, <div class="source">Genetics</div> , <div class="year">2007</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">2637</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dense%20single-nucleotide%20polymorphism-based%20genetic%20linkage%20map%20of%20grapevine%20(Vitis%20vinifera%20L.)%20anchoring%20Pinot%20Noir%20bacterial%20artificial%20chromosome%20contigs&amp;author=M.%20Troggio&amp;author=G.%20Malacarne&amp;author=G.%20Coppola&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=2637-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.067462" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.067462" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.067462"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17603124" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pavy</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Pelgas</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Beauseigle</div> <div class="given-names">S.P.</div></div>, et al. </div><div class="article-title">Enhancing genetic mapping of complex genomes through the design of highly-multiplexed SNP arrays: application to the large and unsequenced genomes of white spruce and black spruce</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">21</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhancing%20genetic%20mapping%20of%20complex%20genomes%20through%20the%20design%20of%20highly-multiplexed%20SNP%20arrays%3A%20application%20to%20the%20large%20and%20unsequenced%20genomes%20of%20white%20spruce%20and%20black%20spruce&amp;author=N.%20Pavy&amp;author=B.%20Pelgas&amp;author=S.P.%20Beauseigle&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18205909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muchero</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Diop</div> <div class="given-names">N.N.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">P.R.</div></div>, et al. </div><div class="article-title">A consensus genetic map of cowpea [<em>Vigna unguiculata</em> (L) Walp.] and synteny based on EST-derived SNPs</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">18159</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20genetic%20map%20of%20cowpea%20%5BVigna%20unguiculata%20(L)%20Walp.%5D%20and%20synteny%20based%20on%20EST-derived%20SNPs&amp;author=W.%20Muchero&amp;author=N.N.%20Diop&amp;author=P.R.%20Bhat&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=18159-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0905886106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0905886106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0905886106"> </span></div> </div></div></div></div></div><div content-id="DSS018C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deulvot</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Charrel</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Marty</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">468</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly-multiplexed%20SNP%20genotyping%20for%20genetic%20mapping%20and%20germplasm%20diversity%20studies%20in%20pea&amp;author=C.%20Deulvot&amp;author=H.%20Charrel&amp;author=A.%20Marty&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20701750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Allen</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Barker</div> <div class="given-names">G.L.A.</div></div>,  <div class="name"><div class="surname">Berry</div> <div class="given-names">S.T.</div></div>, et al. </div><div class="article-title">Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">Plant Biotech. J.</div> , <div class="year">2011</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1086</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript-specific%2C%20single-nucleotide%20polymorphism%20discovery%20and%20linkage%20analysis%20in%20hexaploid%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=A.M.%20Allen&amp;author=G.L.A.%20Barker&amp;author=S.T.%20Berry&amp;publication_year=2011&amp;journal=Plant%20Biotech.%20J.&amp;volume=9&amp;pages=1086-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2011.9.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2011.9.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2011.9.issue-9"> </span></div> </div></div></div></div></div><div content-id="DSS018C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chancerel</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Lepoittevin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Le Provost</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Development and implementation of a highly multiplexed SNP array for genetic mapping in maritime pine and comparative mapping with loblolly pine</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">368</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20implementation%20of%20a%20highly%20multiplexed%20SNP%20array%20for%20genetic%20mapping%20in%20maritime%20pine%20and%20comparative%20mapping%20with%20loblolly%20pine&amp;author=E.%20Chancerel&amp;author=C.%20Lepoittevin&amp;author=G.%20Le%20Provost&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=368" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-368" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-368" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-368"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21767361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barbazuk</div> <div class="given-names">W.B.</div></div>,  <div class="name"><div class="surname">Emrich</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.D.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>. </div><div class="article-title">SNP discovery via 454 transcriptome sequencing</div>, <div class="source">Plant Journal</div> , <div class="year">2007</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">910</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20via%20454%20transcriptome%20sequencing&amp;author=W.B.%20Barbazuk&amp;author=S.J.%20Emrich&amp;author=H.D.%20Chen&amp;author=L.%20Li&amp;author=P.S.%20Schnable&amp;publication_year=2007&amp;journal=Plant%20Journal&amp;volume=51&amp;pages=910-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2007.03193.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03193.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2007.03193.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17662031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">High-throughput SNP discovery through deep resequencing of a reduced representation library to anchor and orient scaffolds in the soybean whole genome sequence</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">38</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20through%20deep%20resequencing%20of%20a%20reduced%20representation%20library%20to%20anchor%20and%20orient%20scaffolds%20in%20the%20soybean%20whole%20genome%20sequence&amp;author=D.L.%20Hyten&amp;author=S.B.%20Cannon&amp;author=Q.%20Song&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-38"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20078886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Torres-Jerez</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Genome-wide SNP discovery in tetraploid alfalfa using 454 sequencing and high resolution melting analysis</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">350</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20discovery%20in%20tetraploid%20alfalfa%20using%20454%20sequencing%20and%20high%20resolution%20melting%20analysis&amp;author=Y.%20Han&amp;author=Y.%20Kang&amp;author=I.%20Torres-Jerez&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=350" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-350" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-350" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-350"> </span></div> </div></div></div></div></div><div content-id="DSS018C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grattapaglia</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Silva-Junior</div> <div class="given-names">O.B.</div></div>,  <div class="name"><div class="surname">Kirst</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lima</div> <div class="given-names">B.M.</div></div>,  <div class="name"><div class="surname">Faria</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Pappas</div> <div class="given-names">G.J.</div></div>. </div><div class="article-title">High-throughput SNP genotyping in the highly heterozygous genome of <em>Eucalyptus</em>: assay success, polymorphism and transferability across species</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">65</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20genotyping%20in%20the%20highly%20heterozygous%20genome%20of%20Eucalyptus%3A%20assay%20success%2C%20polymorphism%20and%20transferability%20across%20species&amp;author=D.%20Grattapaglia&amp;author=O.B.%20Silva-Junior&amp;author=M.%20Kirst&amp;author=B.M.%20Lima&amp;author=D.A.%20Faria&amp;author=G.J.%20Pappas&amp;publication_year=2011&amp;journal=BMC%20Plant%20Biol.&amp;volume=11&amp;pages=65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-65" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-65" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-65"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21492434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arai-Kichise</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shiwa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nagasaki</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Discovery of genome-wide DNA polymorphisms in a landrace cultivar of japonica rice by whole-genome sequencing</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">274</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20genome-wide%20DNA%20polymorphisms%20in%20a%20landrace%20cultivar%20of%20japonica%20rice%20by%20whole-genome%20sequencing&amp;author=Y.%20Arai-Kichise&amp;author=Y.%20Shiwa&amp;author=H.%20Nagasaki&amp;publication_year=2011&amp;journal=Plant%20Cell%20Physiol.&amp;volume=52&amp;pages=274-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcr003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcr003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcr003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21258067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bachlava</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">SNP discovery and development of a high-density genotyping array for sunflower</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e29814</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20development%20of%20a%20high-density%20genotyping%20array%20for%20sunflower&amp;author=E.%20Bachlava&amp;author=C.A.%20Taylor&amp;author=S.%20Tang&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e29814" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0029814" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0029814" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0029814"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22238659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fan</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Oliphant</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Highly parallel SNP genotyping</div>, <div class="source">Cold Spring Harb. Symp. Quant. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">69</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20parallel%20SNP%20genotyping&amp;author=J.B.%20Fan&amp;author=A.%20Oliphant&amp;author=R.%20Shen&amp;publication_year=2003&amp;journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol.&amp;volume=68&amp;pages=69-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/sqb.2003.68.69" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fsqb.2003.68.69" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fsqb.2003.68.69"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15338605" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oliphant</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Barker</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Stuelpnagel</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Chee</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">BeadArray technology: enabling an accurate, cost-effective approach to high-throughput genotyping</div>, <div class="source">Biotechniques</div> , <div class="year">2002</div>, vol. <div class="volume">32</div> pg. <div class="fpage">S56</div> </div></div></div></div><div content-id="DSS018C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fan</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Chee</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Gunderson</div> <div class="given-names">K.L.</div></div>. </div><div class="article-title">Highly parallel genomic assays</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">632</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20parallel%20genomic%20assays&amp;author=J.B.%20Fan&amp;author=M.S.%20Chee&amp;author=K.L.%20Gunderson&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=7&amp;pages=632-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1901" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1901" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1901"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16847463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bell</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Chaturvedi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gelfand</div> <div class="given-names">C.A.</div></div>, et al. </div><div class="article-title">SNPstream UHT: ultra-high throughput SNP genotyping for pharmacogenomics and drug discovery</div>, <div class="source">Biotechniques</div> , <div class="year">2002</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">S70</div>-<div class="lpage">7</div>)</div></div></div></div><div content-id="DSS018C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuzaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Loi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Parallel genotyping of over 10,000 SNPs using a one-primer assay on a high-density oligonucleotide array</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">414</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parallel%20genotyping%20of%20over%2010%2C000%20SNPs%20using%20a%20one-primer%20assay%20on%20a%20high-density%20oligonucleotide%20array&amp;author=H.%20Matsuzaki&amp;author=H.%20Loi&amp;author=S.%20Dong&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=414-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2014904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2014904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2014904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14993208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuzaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Loi</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays</div>, <div class="source">Nat. Methods</div> , <div class="year">2004</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">109</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping%20over%20100%2C000%20SNPs%20on%20a%20pair%20of%20oligonucleotide%20arrays&amp;author=H.%20Matsuzaki&amp;author=S.%20Dong&amp;author=H.%20Loi&amp;publication_year=2004&amp;journal=Nat.%20Methods&amp;volume=1&amp;pages=109-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth718" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth718" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth718"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15782172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nijman</div> <div class="given-names">I.J.</div></div>,  <div class="name"><div class="surname">Kuipers</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Verheul</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Guryev</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Cuppen</div> <div class="given-names">E.</div></div>. </div><div class="article-title">A genome-wide SNP panel for mapping and association studies in the rat</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">95</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20SNP%20panel%20for%20mapping%20and%20association%20studies%20in%20the%20rat&amp;author=I.J.%20Nijman&amp;author=S.%20Kuipers&amp;author=M.%20Verheul&amp;author=V.%20Guryev&amp;author=E.%20Cuppen&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-95" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-95" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-95"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18298839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Syvanen</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">Toward genome-wide SNP genotyping</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">S5</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20genome-wide%20SNP%20genotyping&amp;author=A.C.%20Syvanen&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=37&amp;pages=S5-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1558" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1558" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1558"> </span></div> </div></div></div></div></div><div content-id="DSS018C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">I.Y.</div></div>,  <div class="name"><div class="surname">Yoon</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Cregan</div> <div class="given-names">P.B.</div></div>. </div><div class="article-title">High-throughput genotyping with the GoldenGate assay in the complex genome of soybean</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">945</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20with%20the%20GoldenGate%20assay%20in%20the%20complex%20genome%20of%20soybean&amp;author=D.L.%20Hyten&amp;author=Q.%20Song&amp;author=I.Y.%20Choi&amp;author=M.S.%20Yoon&amp;author=P.B.%20Cregan&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=945-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0726-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0726-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0726-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">P.R.</div></div>,  <div class="name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Development and implementation of high-throughput SNP genotyping in barley</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">582</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20implementation%20of%20high-throughput%20SNP%20genotyping%20in%20barley&amp;author=T.J.%20Close&amp;author=P.R.%20Bhat&amp;author=S.%20Lonardi&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19961604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yan</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.H.</div></div>,  <div class="name"><div class="surname">Shah</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">High-throughput SNP genotyping with the GoldenGate assay in maize</div>, <div class="source">Mol. Breed.</div> , <div class="year">2010</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">441</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20genotyping%20with%20the%20GoldenGate%20assay%20in%20maize&amp;author=J.B.%20Yan&amp;author=X.H.%20Yang&amp;author=T.%20Shah&amp;publication_year=2010&amp;journal=Mol.%20Breed.&amp;volume=25&amp;pages=441-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9343-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9343-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9343-2"> </span></div> </div></div></div></div></div><div content-id="DSS018C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eckert</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pande</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ersoz</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">High-throughput genotyping and mapping of single nucleotide polymorphisms in loblolly pine (<em>Pinus taeda</em> L.)</div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">225</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20and%20mapping%20of%20single%20nucleotide%20polymorphisms%20in%20loblolly%20pine%20(Pinus%20taeda%20L.)&amp;author=A.%20Eckert&amp;author=B.%20Pande&amp;author=E.%20Ersoz&amp;publication_year=2009&amp;journal=Tree%20Genet.%20Genomes&amp;volume=5&amp;pages=225-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-008-0183-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-008-0183-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-008-0183-8"> </span></div> </div></div></div></div></div><div content-id="DSS018C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Myles</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chia</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Hurwitz</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Rapid genomic characterization of the genus <em>Vitis</em></div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e8219</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20genomic%20characterization%20of%20the%20genus%20Vitis&amp;author=S.%20Myles&amp;author=J.M.%20Chia&amp;author=B.%20Hurwitz&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e8219" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0008219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0008219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0008219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20084295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Sang</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide association studies of 14 agronomic traits in rice landraces</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">961</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&amp;author=X.%20Huang&amp;author=X.%20Wei&amp;author=T.%20Sang&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=961-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P.J.</div></div>, et al. </div><div class="article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">159</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20leaf%20architecture%20in%20the%20maize%20nested%20association%20mapping%20population&amp;author=F.%20Tian&amp;author=P.J.%20Bradbury&amp;author=P.J.%20Brown&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=159-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.746" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.746" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.746"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Branca</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Paape</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Whole-genome nucleotide diversity, recombination, and linkage disequilibrium in the model legume <em>Medicago truncatula</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2011</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">E864</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20nucleotide%20diversity%2C%20recombination%2C%20and%20linkage%20disequilibrium%20in%20the%20model%20legume%20Medicago%20truncatula&amp;author=A.%20Branca&amp;author=T.D.%20Paape&amp;author=P.%20Zhou&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=E864-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1104032108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1104032108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1104032108"> </span></div> </div></div></div></div></div><div content-id="DSS018C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luo</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Deal</div> <div class="given-names">K.R.</div></div>,  <div class="name"><div class="surname">Akhunov</div> <div class="given-names">E.D.</div></div>, et al. </div><div class="article-title">Genome comparisons reveal a dominant mechanism of chromosome number reduction in grasses and accelerated genome evolution in Triticeae</div>, <div class="source">Proc. Natl Acad. Sci., USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">15780</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20comparisons%20reveal%20a%20dominant%20mechanism%20of%20chromosome%20number%20reduction%20in%20grasses%20and%20accelerated%20genome%20evolution%20in%20Triticeae&amp;author=M.C.%20Luo&amp;author=K.R.%20Deal&amp;author=E.D.%20Akhunov&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%2C%20USA&amp;volume=106&amp;pages=15780-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0908195106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0908195106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0908195106"> </span></div> </div></div></div></div></div><div content-id="DSS018C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">SNP discovery and linkage map construction in cultivated tomato</div>, <div class="source">DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">381</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20linkage%20map%20construction%20in%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=S.%20Isobe&amp;author=H.%20Hirakawa&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=381-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Garvin</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">You</div> <div class="given-names">F.M.</div></div>, et al. </div><div class="article-title">Comparison of a high-density genetic linkage map to genome features in the model grass <em>Brachypodium distachyon</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">455</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20a%20high-density%20genetic%20linkage%20map%20to%20genome%20features%20in%20the%20model%20grass%20Brachypodium%20distachyon&amp;author=N.%20Huo&amp;author=D.F.%20Garvin&amp;author=F.M.%20You&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=455-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1598-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1598-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1598-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21597976" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Chu</div> <div class="given-names">W.C.</div></div>,  <div class="name"><div class="surname">Ayele</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Development of single nucleotide polymorphism (SNP) markers for use in commercial maize (<em>Zea mays</em> L.) germplasm</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">165</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20single%20nucleotide%20polymorphism%20(SNP)%20markers%20for%20use%20in%20commercial%20maize%20(Zea%20mays%20L.)%20germplasm&amp;author=E.%20Jones&amp;author=W.C.%20Chu&amp;author=M.%20Ayele&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=24&amp;pages=165-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9281-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9281-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9281-z"> </span></div> </div></div></div></div></div><div content-id="DSS018C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ren</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kou</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">A high resolution genetic map anchoring scaffolds of the sequenced watermelon genome</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e29453</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20resolution%20genetic%20map%20anchoring%20scaffolds%20of%20the%20sequenced%20watermelon%20genome&amp;author=Y.%20Ren&amp;author=H.%20Zhao&amp;author=Q.%20Kou&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e29453" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0029453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0029453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0029453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22247776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arumuganathan</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Earle</div> <div class="given-names">E.D.</div></div>. </div><div class="article-title">Nuclear DNA content of some important plant species</div>, <div class="source">Plant Mol. Rep.</div> , <div class="year">1991</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">208</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20of%20some%20important%20plant%20species&amp;author=K.%20Arumuganathan&amp;author=E.D.%20Earle&amp;publication_year=1991&amp;journal=Plant%20Mol.%20Rep.&amp;volume=9&amp;pages=208-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02672069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02672069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02672069"> </span></div> </div></div></div></div></div><div content-id="DSS018C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nayak</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Varghese</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Integration of novel SSR and gene-based SNP marker loci in the chickpea genetic map and establishment of new anchor points with <em>Medicago truncatula</em> genome</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">1415</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20novel%20SSR%20and%20gene-based%20SNP%20marker%20loci%20in%20the%20chickpea%20genetic%20map%20and%20establishment%20of%20new%20anchor%20points%20with%20Medicago%20truncatula%20genome&amp;author=S.N.%20Nayak&amp;author=H.%20Zhu&amp;author=N.%20Varghese&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=1415-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1265-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1265-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1265-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20098978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gaur</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Advancing the STMS genomic resources for defining new locations on the intraspecific genetic linkage map of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">117</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advancing%20the%20STMS%20genomic%20resources%20for%20defining%20new%20locations%20on%20the%20intraspecific%20genetic%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Gaur&amp;author=N.K.%20Sethy&amp;author=S.%20Choudhary&amp;author=B.%20Shokeen&amp;author=V.%20Gupta&amp;author=S.%20Bhatia&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21329497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gujaria</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dauthal</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Development and use of genic molecular markers (GMMs) for construction of a transcript map of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">1577</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20use%20of%20genic%20molecular%20markers%20(GMMs)%20for%20construction%20of%20a%20transcript%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=N.%20Gujaria&amp;author=A.%20Kumar&amp;author=P.%20Dauthal&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=1577-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1556-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1556-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1556-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21384113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gaur</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">EST-derived genic molecular markers: development and utilization for generating an advanced transcript map of chickpea</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">1449</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST-derived%20genic%20molecular%20markers%3A%20development%20and%20utilization%20for%20generating%20an%20advanced%20transcript%20map%20of%20chickpea&amp;author=S.%20Choudhary&amp;author=R.%20Gaur&amp;author=S.%20Gupta&amp;author=S.%20Bhatia&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=1449-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1800-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1800-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1800-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22301907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Buhariwalla</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">Jayashree</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Eshwar</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Crouch</div> <div class="given-names">J.H.</div></div>. </div><div class="article-title">Development of ESTs from chickpea roots and their use in diversity analysis of the <em>Cicer</em> genus</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">5</div> pg. <div class="fpage">16</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20ESTs%20from%20chickpea%20roots%20and%20their%20use%20in%20diversity%20analysis%20of%20the%20Cicer%20genus&amp;author=H.K.%20Buhariwalla&amp;author=B.%20Jayashree&amp;author=K.%20Eshwar&amp;author=J.H.%20Crouch&amp;publication_year=2005&amp;journal=BMC%20Plant%20Biol.&amp;volume=5&amp;pages=16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-5-16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-5-16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-5-16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16107212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coram</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Pang</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Isolation and analysis of candidate <em>Ascochyta blight</em> defense genes in chickpea, Part I. Generation and analysis of an expressed sequence tag (EST) library</div>, <div class="source">Physiol. Mol. Plant. Pathol.</div> , <div class="year">2005</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">192</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20analysis%20of%20candidate%20Ascochyta%20blight%20defense%20genes%20in%20chickpea%2C%20Part%20I.%20Generation%20and%20analysis%20of%20an%20expressed%20sequence%20tag%20(EST)%20library&amp;author=T.%20Coram&amp;author=E.%20Pang&amp;publication_year=2005&amp;journal=Physiol.%20Mol.%20Plant.%20Pathol.&amp;volume=66&amp;pages=192-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pmpp.2005.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pmpp.2005.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pmpp.2005.08.003"> </span></div> </div></div></div></div></div><div content-id="DSS018C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Hiremath</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Lekha</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">A comprehensive resource of drought and salinity responsive ESTs for gene discovery and marker development in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">15</div> pg. <div class="fpage">523</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20resource%20of%20drought%20and%20salinity%20responsive%20ESTs%20for%20gene%20discovery%20and%20marker%20development%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.K.%20Varshney&amp;author=P.J.%20Hiremath&amp;author=P.%20Lekha&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=523" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-523"> </span></div> </div></div></div></div></div><div content-id="DSS018C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashraf</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ghai</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Barman</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Comparative analysis of genotype dependent expressed sequence tags and stress-responsive transcriptome of chickpea wilt illustrate predicted and unexpected genes and novel regulators of plant immunity</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">415</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20genotype%20dependent%20expressed%20sequence%20tags%20and%20stress-responsive%20transcriptome%20of%20chickpea%20wilt%20illustrate%20predicted%20and%20unexpected%20genes%20and%20novel%20regulators%20of%20plant%20immunity&amp;author=N.%20Ashraf&amp;author=D.%20Ghai&amp;author=P.%20Barman&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=415" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-415" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-415" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-415"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19732460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">53</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20chickpea%20transcriptome%20using%20short%20reads%20for%20gene%20discovery%20and%20marker%20identification&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=53-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Jhanwar</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Gene discovery and tissue-specific transcriptome analysis in chickpea with massively parallel pyrosequencing and web resource development</div>, <div class="source">Plant Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1661</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20discovery%20and%20tissue-specific%20transcriptome%20analysis%20in%20chickpea%20with%20massively%20parallel%20pyrosequencing%20and%20web%20resource%20development&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=S.%20Jhanwar&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=1661-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.178616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.178616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.178616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiremath</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Farmer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>, et al. </div><div class="article-title">Large-scale transcriptome analysis in chickpea (<em>Cicer arietinum</em> L.), an orphan legume crop of the semi-arid tropics of Asia and Africa</div>, <div class="source">Plant Biotech. J.</div> , <div class="year">2011</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">922</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20transcriptome%20analysis%20in%20chickpea%20(Cicer%20arietinum%20L.)%2C%20an%20orphan%20legume%20crop%20of%20the%20semi-arid%20tropics%20of%20Asia%20and%20Africa&amp;author=P.J.%20Hiremath&amp;author=A.%20Farmer&amp;author=S.B.%20Cannon&amp;publication_year=2011&amp;journal=Plant%20Biotech.%20J.&amp;volume=9&amp;pages=922-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2011.00625.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2011.00625.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2011.00625.x"> </span></div> </div></div></div></div></div><div content-id="DSS018C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hüttel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Weising</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Choumane</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Weigand</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kahl</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Sequence-tagged microsatellite markers for chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Genome</div> , <div class="year">1999</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">210</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-tagged%20microsatellite%20markers%20for%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=B.%20H%C3%BCttel&amp;author=P.%20Winter&amp;author=K.%20Weising&amp;author=W.%20Choumane&amp;author=F.%20Weigand&amp;author=G.%20Kahl&amp;publication_year=1999&amp;journal=Genome&amp;volume=42&amp;pages=210-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-42-2-210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-42-2-210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-42-2-210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10231957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Pfaff</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Udupa</div> <div class="given-names">S.M.</div></div>, et al. </div><div class="article-title">Characterization and mapping of sequence-tagged microsatellite sites in the chickpea (<em>Cicer arietinum</em> L.) genome</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">262</div> (pg. <div class="fpage">90</div>-<div class="lpage">101</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20mapping%20of%20sequence-tagged%20microsatellite%20sites%20in%20the%20chickpea%20(Cicer%20arietinum%20L.)%20genome&amp;author=P.%20Winter&amp;author=T.%20Pfaff&amp;author=S.M.%20Udupa&amp;publication_year=1999&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=262&amp;pages=90-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380051063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380051063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380051063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10503540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Benko-Iseppon</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Hüttel</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">A linkage map of chickpea (<em>Cicer arietinum</em> L.) genome based on recombinant inbred lines from a <em>C. arietinum</em> times <em>C. reticulatum</em> cross: localization of resistance genes for <em>Fusarium wilt</em> races 4 and 5</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">1155</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)%20genome%20based%20on%20recombinant%20inbred%20lines%20from%20a%20C.%20arietinum%20times%20C.%20reticulatum%20cross%3A%20localization%20of%20resistance%20genes%20for%20Fusarium%20wilt%20races%204%20and%205&amp;author=P.%20Winter&amp;author=A.M.%20Benko-Iseppon&amp;author=B.%20H%C3%BCttel&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=101&amp;pages=1155-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051592" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051592" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051592"> </span></div> </div></div></div></div></div><div content-id="DSS018C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lichtenzveig</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Scheuring</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Dodge</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">H.B.</div></div>. </div><div class="article-title">Construction of BAC and BIBAC libraries and their applications for generation of SSR markers for genome analysis of chickpea, <em>Cicer arietinum</em> L<em>.</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">492</div>-<div class="lpage">510</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20BAC%20and%20BIBAC%20libraries%20and%20their%20applications%20for%20generation%20of%20SSR%20markers%20for%20genome%20analysis%20of%20chickpea%2C%20Cicer%20arietinum%20L.&amp;author=J.%20Lichtenzveig&amp;author=C.%20Scheuring&amp;author=J.%20Dodge&amp;author=S.%20Abbo&amp;author=H.B.%20Zhang&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=492-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1857-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1857-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1857-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15712010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Isolation and characterization of sequence tagged microsatellite sites markers in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2003</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">428</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20sequence%20tagged%20microsatellite%20sites%20markers%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=N.K.%20Sethy&amp;author=B.%20Shokeen&amp;author=S.%20Bhatia&amp;publication_year=2003&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=3&amp;pages=428-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2003.00472.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2003.00472.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2003.00472.x"> </span></div> </div></div></div></div></div><div content-id="DSS018C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of microsatellite markers and analysis of intraspecific genetic variability in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">1416</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20microsatellite%20markers%20and%20analysis%20of%20intraspecific%20genetic%20variability%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=N.K.%20Sethy&amp;author=B.%20Shokeen&amp;author=K.J.%20Edwards&amp;author=S.%20Bhatia&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=1416-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0243-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0243-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0243-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16534564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of sequence-tagged microsatellites site markers for chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">93</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20sequence-tagged%20microsatellites%20site%20markers%20for%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=S.%20Choudhary&amp;author=N.K.%20Sethy&amp;author=B.%20Shokeen&amp;author=S.%20Bhatia&amp;publication_year=2006&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=6&amp;pages=93-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2006.6.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2006.6.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2006.6.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSS018C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of chickpea EST-SSR markers and analysis of allelic variation across related species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">591</div>-<div class="lpage">608</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20chickpea%20EST-SSR%20markers%20and%20analysis%20of%20allelic%20variation%20across%20related%20species&amp;author=S.%20Choudhary&amp;author=N.K.%20Sethy&amp;author=B.%20Shokeen&amp;author=S.%20Bhatia&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=591-608" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0923-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0923-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0923-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19020854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Millan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jüngling</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">A consensus genetic map of chickpea (<em>Cicer arietinum</em> L.) based on 10 mapping populations</div>, <div class="source">Euphytica</div> , <div class="year">2010</div>, vol. <div class="volume">175</div> (pg. <div class="fpage">175</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20genetic%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)%20based%20on%2010%20mapping%20populations&amp;author=T.%20Millan&amp;author=P.%20Winter&amp;author=R.%20J%C3%BCngling&amp;publication_year=2010&amp;journal=Euphytica&amp;volume=175&amp;pages=175-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-010-0157-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-010-0157-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-010-0157-4"> </span></div> </div></div></div></div></div><div content-id="DSS018C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Malmberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Messing</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sussex</div> <div class="given-names">I.</div></div>. </div>, <div class="source">Molecular Biology of Plants: A Laboratory Course Manual</div> , <div class="year">1985</div><div class="publisher-loc">Cold Spring Harbor</div><div class="publisher-name">Cold Spring Harbor Laboratory</div></div></div></div></div><div content-id="DSS018C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Margulies</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Egholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Altman</div> <div class="given-names">W.E.</div></div>, et al. </div><div class="article-title">Genome sequencing in microfabricated high-density picolitre reactors</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">376</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;author=M.%20Egholm&amp;author=W.E.%20Altman&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.</div></div>. </div><div class="article-title">JoinMap version 4.0: Software for the calculation of genetic linkage maps in experimental population</div>, <div class="year">2006</div> <div class="comment">Kyazma B.V., Wageningen, Netherlands</div></div></div></div></div><div content-id="DSS018C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D.</div></div>. </div><div class="article-title">The estimation of map distances from recombination values</div>, <div class="source">Ann. Eugenics</div> , <div class="year">1994</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">172</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distances%20from%20recombination%20values&amp;author=D.%20Kosambi&amp;publication_year=1994&amp;journal=Ann.%20Eugenics&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="DSS018C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeller</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Bohlen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Weigel</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Rätsch</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Detecting polymorphic regions in the <em>Arabidopsis thaliana</em> genome with resequencing microarrays</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">918</div>-<div class="lpage">929</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20polymorphic%20regions%20in%20the%20Arabidopsis%20thaliana%20genome%20with%20resequencing%20microarrays&amp;author=G.%20Zeller&amp;author=R.M.%20Clark&amp;author=K.%20Schneeberger&amp;author=A.%20Bohlen&amp;author=D.%20Weigel&amp;author=G.%20R%C3%A4tsch&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=918-929" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.070169.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.070169.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.070169.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18323538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Joshi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vuong</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Nguyen</div> <div class="given-names">H.T.</div></div>. </div><div class="article-title">SNP discovery by high-throughput sequencing in soybean</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">469</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20by%20high-throughput%20sequencing%20in%20soybean&amp;author=X.%20Wu&amp;author=C.%20Ren&amp;author=T.%20Joshi&amp;author=T.%20Vuong&amp;author=D.%20Xu&amp;author=H.T.%20Nguyen&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=469" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-469" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-469" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-469"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20701770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nagasaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yonemaru</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Fine definition of the pedigree haplotypes of closely related rice cultivars by means of genome-wide discovery of single-nucleotide polymorphisms</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">267</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20definition%20of%20the%20pedigree%20haplotypes%20of%20closely%20related%20rice%20cultivars%20by%20means%20of%20genome-wide%20discovery%20of%20single-nucleotide%20polymorphisms&amp;author=T.%20Yamamoto&amp;author=H.%20Nagasaki&amp;author=J.%20Yonemaru&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20423466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Borevitz</div> <div class="given-names">J.O.</div></div>. </div><div class="article-title">Global analysis of allele-specific expression in <em>Arabidopsis thaliana</em></div>, <div class="source">Genetics</div> , <div class="year">2009</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">943</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20allele-specific%20expression%20in%20Arabidopsis%20thaliana&amp;author=X.%20Zhang&amp;author=J.O.%20Borevitz&amp;publication_year=2009&amp;journal=Genetics&amp;volume=182&amp;pages=943-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.103499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.103499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.103499"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19474198" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Ge</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">105</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%2050%20accessions%20of%20cultivated%20and%20wild%20rice%20yields%20markers%20for%20identifying%20agronomically%20important%20genes&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;author=S.%20Ge&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2050"> </span></div> </div></div></div></div></div><div content-id="DSS018C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lam</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">1053</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%20of%2031%20wild%20and%20cultivated%20soybean%20genomes%20identifies%20patterns%20of%20genetic%20diversity%20and%20selection&amp;author=H.M.%20Lam&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=1053-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.715" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.715" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.715"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rostoks</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ramsay</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">MacKenzie</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Recent history of artificial outcrossing facilitates whole-genome association mapping in elite inbred crop varieties</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">18656</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20history%20of%20artificial%20outcrossing%20facilitates%20whole-genome%20association%20mapping%20in%20elite%20inbred%20crop%20varieties&amp;author=N.%20Rostoks&amp;author=L.%20Ramsay&amp;author=K.%20MacKenzie&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=18656-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0606133103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0606133103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0606133103"> </span></div> </div></div></div></div></div><div content-id="DSS018C76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C76" class="ref-content "><div class="label -label">76</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Takeda</div> <div class="given-names">K.</div></div>. </div><div class="article-title">An application of high-throughput SNP genotyping for barley genome mapping and characterization of recombinant chromosome substitution lines</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">613</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20application%20of%20high-throughput%20SNP%20genotyping%20for%20barley%20genome%20mapping%20and%20characterization%20of%20recombinant%20chromosome%20substitution%20lines&amp;author=K.%20Sato&amp;author=K.%20Takeda&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=119&amp;pages=613-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1071-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1071-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1071-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19488734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C77" class="ref-content "><div class="label -label">77</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Torado</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Koike</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">SSR-based linkage map with new markers using an intraspecific population of common wheat</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">1042</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SSR-based%20linkage%20map%20with%20new%20markers%20using%20an%20intraspecific%20population%20of%20common%20wheat&amp;author=A.%20Torado&amp;author=M.%20Koike&amp;author=K.%20Mochida&amp;author=Y.%20Ogihara&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=1042-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0206-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0206-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0206-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16450184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C78" class="ref-content "><div class="label -label">78</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Marcel</div> <div class="given-names">T.C.</div></div>,  <div class="name"><div class="surname">Ramsay</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">A high density barley microsatellite consensus map with 775 SSR loci</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">1091</div>-<div class="lpage">113</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20density%20barley%20microsatellite%20consensus%20map%20with%20775%20SSR%20loci&amp;author=R.K.%20Varshney&amp;author=T.C.%20Marcel&amp;author=L.%20Ramsay&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=1091-113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0503-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0503-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0503-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17345060" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C79" class="ref-content "><div class="label -label">79</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bindler</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bakaher</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">A high density genetic map of tobacco (<em>Nicotiana tabacum</em> L.) obtained from large scale microsatellite marker development</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">219</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20density%20genetic%20map%20of%20tobacco%20(Nicotiana%20tabacum%20L.)%20obtained%20from%20large%20scale%20microsatellite%20marker%20development&amp;author=G.%20Bindler&amp;author=J.%20Plieske&amp;author=N.%20Bakaher&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=219-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1578-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1578-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1578-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21461649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C80" class="ref-content "><div class="label -label">80</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">An interspecific linkage map of SSR and intronic polymorphism markers in tomato</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20interspecific%20linkage%20map%20of%20SSR%20and%20intronic%20polymorphism%20markers%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=E.%20Asamizu&amp;author=H.%20Fukuoka&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1344-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1344-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1344-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20431859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C81" class="ref-content "><div class="label -label">81</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of SSR and gene-targeted markers for construction of a framework linkage map of <em>Catharanthus roseus</em></div>, <div class="source">Ann. Bot. (Lond.)</div> , <div class="year">2011</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">321</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20SSR%20and%20gene-targeted%20markers%20for%20construction%20of%20a%20framework%20linkage%20map%20of%20Catharanthus%20roseus&amp;author=B.%20Shokeen&amp;author=S.%20Choudhary&amp;author=N.K.%20Sethy&amp;author=S.%20Bhatia&amp;publication_year=2011&amp;journal=Ann.%20Bot.%20(Lond.)&amp;volume=108&amp;pages=321-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcr162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcr162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcr162"> </span></div> </div></div></div></div></div><div content-id="DSS018C82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C82" class="ref-content "><div class="label -label">82</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hwang</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sayama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">High-density integrated linkage map based on SSR markers in soybean</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">213</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20integrated%20linkage%20map%20based%20on%20SSR%20markers%20in%20soybean&amp;author=T.%20Hwang&amp;author=T.%20Sayama&amp;author=M.%20Takahashi&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=213-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19531560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C83" class="ref-content "><div class="label -label">83</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaillon</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Noel</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">449</div> (pg. <div class="fpage">463</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20grapevine%20genome%20sequence%20suggests%20ancestral%20hexaploidization%20in%20major%20angiosperm%20phyla&amp;author=O.%20Jaillon&amp;author=J.M.%20Aury&amp;author=B.%20Noel&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=463-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C84" class="ref-content "><div class="label -label">84</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">The genome of the cucumber, <em>Cucumis sativus</em> L.</div>, <div class="source">Nat. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">1275</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20cucumber%2C%20Cucumis%20sativus%20L.&amp;author=S.%20Huang&amp;author=R.%20Li&amp;author=Z.%20Zhang&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=1275-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19881527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C85" class="ref-content "><div class="label -label">85</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Velasco</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zharkikh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Affourtit</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The genome of the domesticated apple (<em>Malus</em> x <em>domestica Borkh</em>.)</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">833</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20domesticated%20apple%20(Malus%20x%20domestica%20Borkh.)&amp;author=R.%20Velasco&amp;author=A.%20Zharkikh&amp;author=J.%20Affourtit&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=833-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20802477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C86" class="ref-content "><div class="label -label">86</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">Mun</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">D.J.</div></div>, et al. </div><div class="article-title">Estimating genome conservation between crop and model legume species</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">15289</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20genome%20conservation%20between%20crop%20and%20model%20legume%20species&amp;author=H.K.%20Choi&amp;author=J.H.%20Mun&amp;author=D.J.%20Kim&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=15289-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0402251101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0402251101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0402251101"> </span></div> </div></div></div></div></div><div content-id="DSS018C87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C87" class="ref-content "><div class="label -label">87</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bertioli</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Moretzsohn</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">L.H.</div></div>, et al. </div><div class="article-title">An analysis of synteny of <em>Arachis</em> with <em>Lotus</em> and <em>Medicago</em> sheds new light on the structure, stability and evolution of legume genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">45</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20analysis%20of%20synteny%20of%20Arachis%20with%20Lotus%20and%20Medicago%20sheds%20new%20light%20on%20the%20structure%2C%20stability%20and%20evolution%20of%20legume%20genomes&amp;author=D.J.%20Bertioli&amp;author=M.C.%20Moretzsohn&amp;author=L.H.%20Madsen&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19166586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C88" class="ref-content "><div class="label -label">88</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galeano</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Fernandez</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Franco-Herrera</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Saturation of an intra-gene pool linkage map: towards a unified consensus linkage map for fine mapping and synteny analysis in common bean</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e28135</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saturation%20of%20an%20intra-gene%20pool%20linkage%20map%3A%20towards%20a%20unified%20consensus%20linkage%20map%20for%20fine%20mapping%20and%20synteny%20analysis%20in%20common%20bean&amp;author=C.H.%20Galeano&amp;author=A.C.%20Fernandez&amp;author=N.%20Franco-Herrera&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e28135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0028135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0028135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0028135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C89" class="ref-content "><div class="label -label">89</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">A SNP and SSR based genetic map of Asparagus bean (<em>Vigna unguiculata</em> ssp. <em>sesquipedalis</em>) and comparison with the broader species</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e15952</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20SNP%20and%20SSR%20based%20genetic%20map%20of%20Asparagus%20bean%20(Vigna%20unguiculata%20ssp.%20sesquipedalis)%20and%20comparison%20with%20the%20broader%20species&amp;author=P.%20Xu&amp;author=X.%20Wu&amp;author=B.%20Wang&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e15952" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0015952" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0015952" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0015952"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21253606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS018C90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS018C90" class="ref-content "><div class="label -label">90</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gutierrez</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Vaz-Patto</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Huguet</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Cubero</div> <div class="given-names">J.I.</div></div>,  <div class="name"><div class="surname">Moreno</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Torres</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Cross-species amplification of <em>Medicago truncatula</em> microsatellites across three major pulse crops</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">1210</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-species%20amplification%20of%20Medicago%20truncatula%20microsatellites%20across%20three%20major%20pulse%20crops&amp;author=M.V.%20Gutierrez&amp;author=M.C.%20Vaz-Patto&amp;author=T.%20Huguet&amp;author=J.I.%20Cubero&amp;author=M.T.%20Moreno&amp;author=A.M.%20Torres&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=1210-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-1951-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-1951-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-1951-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15806349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=628389&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss018-dss018supp_table1.xls?Expires=2147483647&amp;Signature=hVxbLMbRXGbnETO3uCcRgbPfQZKX4HygMy-elKTDybMrRVnjZCRGfYyd2vu-uVGY9b5LvOoIvs7z0yw7W8CCIHSvbi4Ta~na~XvfiA6TpXeaCTzZ2ITedTgIo1t2bnFpHM5gHvYNR6AHQaeIDEFcqKQUTqD9waNP35bZ7hS0y90QOtsznDCHD4YqEOXjlqH82rVpMdmRtpqRfShDhXqEEdmnL9LxDsMzAmA2n5CZfH7W~Len~bjooPBOF8dInxfGTQoaA9Dg3lk8omseT5d~UE4Z6EVf9eInUUKmRlnr3Mfq8gBq4Ot4uIdfxkNG~3RUhZMeQn~i9~do1zMfShFbEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss018-dss018supp_fig3.ppt?Expires=2147483647&amp;Signature=dlitDnrU9CiFJ-vUjDkrYUTsQug6oh4hSkTh0oHnOwnz75zIkVm2AMdlTTh9iEgUEUKSLgSFcEtSCs4xacoXO6Ks9lXJBbXc~zn8jzawIpojt4hTy-WgyFsNKdFwadR9iAoyv679tPKao35yr~HOpo3QKy2oub-upgjsBMemxxUq3BU03PPuIFVF5~wEUpn1tu11TkQesv7E9LWVtUrAF~Y0EK9yDFAmfx0Llfh3KCgCyZdEHeEcJ-~DSivYm0T~4ESnSxIqcWFSIl3FGf-kTzYopFve0RfF83Rvt2yDhKSYajmq5fagRcw1JTWNAmJTHwerglfY~po5BQvDO151Jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss018-dss018supp_fig2.ppt?Expires=2147483647&amp;Signature=Uk8DRZoaDPiR7pVZsoEbzNMp~DYTh7QmIiWSUHHYnJWDHuo7CsbYBxfL2~hpz9Z6wdDCb5OkWyWyfLAD80gCjrvkhxUG8FZiXj7bhXJE-f2DsNKNHM2N6078bgnvQx4BOpV-7JavEulfRf7NCXy8PPIcuZXpk2~hnx96Wc2pl0STKEmMxg4fUGMIcIUtLrkCHOKk7v-IzyaNUA7b5jlOmBtzhezQiIRlzQ7X~UNfyYdGYhwHZR~t2V8S95CMXtCac-a5h9jeO0P6EXf9mgsvnXLePr-O8qcn42BnQqKg5oTsCdkArSl35tJibkrhUCT4wUDe09GU8Bg15fId9~FYsw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss018/2/dss018-dss018supp_fig1.ppt?Expires=2147483647&amp;Signature=qREyUEg8ttWgYQuDrrtxqwpEYRM1CmfWPsY4lXNWpKwIBJ3dpwu8xayIURez5-1Cv5VP0YmF7Q5azAeWRCTXI3dKX3oqoA5DnAd33iVRepoupGnp3IY-NGIiNXgk2a-OmQpJaE8eGfX~rwnEytrHfIyCnqF-Jqx8zcmwPCxVBIWfCRnXWxFV0MK4I5aGUHES52B610tBod2j6se~e8u95LVoJ2dJ3Slj775EVLiJ2sv4IF9DDNeKj-n8al-spCxsOBhHSYT~ctWk9NcvGBSoGtIkJ3nlnG4P8xiA~wyFodZtaSg3noCm31N4ES6HVO4E-vUUdsWIEdIjIf8e4n4s4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">544</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000310822200001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">82</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss018"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="628389"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000310822200001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f5%2f357%2f628389">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000310822200001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (82)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F5%2F357%2F628389" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/628389'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-Throughput SNP Discovery and Genotyping for Constructing a Saturated Linkage Map of Chickpea (Cicer arietinum L.)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss018");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f357%2f628389">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

