

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/3/155/353741","@type":"ScholarlyArticle","name":"A Novel Construction of Genome Space with Biological Geometry","datePublished":"2010-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2010-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq008","keywords":["genome space","graphical representation","moment vector","classification","phylogeny"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/3/155/353741","author":[{"name":"Yu, Chenglong","affiliation":"1  Department of Mathematics ,  The Chinese University of Hong Kong ,  Shatin ,  Hong Kong","@type":"Person"},{"name":"Liang, Qian","affiliation":"2  Department of Mathematics, Statistics and Computer Science ,  University of Illinois at Chicago ,  Chicago, IL 60607-7045 ,  USA","@type":"Person"},{"name":"Yin, Changchuan","affiliation":"2  Department of Mathematics, Statistics and Computer Science ,  University of Illinois at Chicago ,  Chicago, IL 60607-7045 ,  USA","@type":"Person"},{"name":"He, Rong L.","affiliation":"3  Department of Biological Sciences ,  Chicago State University ,  Chicago, IL 60628 ,  USA","@type":"Person"},{"name":"Yau, Stephen S.-T.","affiliation":"2  Department of Mathematics, Statistics and Computer Science ,  University of Illinois at Chicago ,  Chicago, IL 60607-7045 ,  USA","@type":"Person"}],"description":"Abstract.  A genome space is a moduli space of genomes. In this space, each point corresponds to a genome. The natural distance between two points in the genome","pageStart":"155","pageEnd":"168","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A Novel Construction of Genome Space with Biological Geometry","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Novel Construction of Genome Space with Biological Geometry" />
    <meta property="og:description" content="Abstract.  A genome space is a moduli space of genomes. In this space, each point corresponds to a genome. The natural distance between two points in the genome" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/3/155/353741" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-04-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yu, Chenglong" /><meta name="citation_author_institution" content="Department of Mathematics, The Chinese University of Hong Kong, Shatin, Hong Kong" /><meta name="citation_author" content="Liang, Qian" /><meta name="citation_author_institution" content="Department of Mathematics, Statistics and Computer Science, University of Illinois at Chicago, Chicago, IL 60607-7045, USA" /><meta name="citation_author" content="Yin, Changchuan" /><meta name="citation_author_institution" content="Department of Mathematics, Statistics and Computer Science, University of Illinois at Chicago, Chicago, IL 60607-7045, USA" /><meta name="citation_author" content="He, Rong L." /><meta name="citation_author_institution" content="Department of Biological Sciences, Chicago State University, Chicago, IL 60628, USA" /><meta name="citation_author" content="Yau, Stephen S.-T." /><meta name="citation_author_institution" content="Department of Mathematics, Statistics and Computer Science, University of Illinois at Chicago, Chicago, IL 60607-7045, USA" /><meta name="citation_title" content="A Novel Construction of Genome Space with Biological Geometry" /><meta name="citation_firstpage" content="155" /><meta name="citation_lastpage" content="168" /><meta name="citation_doi" content="10.1093/dnares/dsq008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2010/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Big tree from little genomes: mitochondrial gene order as a phylogenetic tool; Curr. Opin. Genet. Dev.;  citation_year=1998;  citation_volume=8;  citation_pages=668-74; " /><meta name="citation_reference" content="citation_title=Genome phylogeny based on gene content; Nat. Genet.;  citation_year=1999;  citation_volume=21;  citation_pages=108-10; " /><meta name="citation_reference" content="citation_title=Exact and approximation algorithms for the inversion distance; Algorithmica;  citation_year=1995;  citation_volume=13;  citation_pages=180-210; " /><meta name="citation_reference" content="citation_title=Transforming cabbage into turnip; Proceedings of the 27th ACM Symposium on Theory of Computing;  citation_year=1995;  citation_pages=178-89; " /><meta name="citation_reference" content="citation_title=Counting on comparative maps; Trends Genet.;  citation_year=1998;  citation_volume=14;  citation_pages=495-501; " /><meta name="citation_reference" content="citation_title=The emerging paradigm and open problems in comparative genomics; Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=265-6; " /><meta name="citation_reference" content="citation_title=Novel DNA sequence representation; Nature;  citation_year=1985;  citation_volume=314;  citation_pages=585-6; " /><meta name="citation_reference" content="citation_title=Simpler DNA sequence representations; Nature;  citation_year=1985;  citation_volume=316;  citation_pages=219" /><meta name="citation_reference" content="citation_title=DNA sequence representation without degeneracy; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3078-80; " /><meta name="citation_reference" content="citation_title=A protein map and its application; DNA Cell Biol.;  citation_year=2008;  citation_volume=27;  citation_pages=241-50; " /><meta name="citation_reference" content="citation_publisher=Hindustan Publishing Corporation, India; Basic Algebra;  citation_year=1974;  citation_volume=1;  citation_pages=135" /><meta name="citation_reference" content="citation_title=Evolution of protein molecules in mammalian protein metabolism; citation_publisher=Academic Press, New York; Mammalian Protein Metabolism;  citation_year=1969;  citation_pages=21-132; " /><meta name="citation_reference" content="citation_title=A simple model for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences; J. Mol. Evol.;  citation_year=1980;  citation_volume=16;  citation_pages=111-20; " /><meta name="citation_reference" content="citation_title=Statistical analysis of ominoid molecular evolution; Stat. Sci.;  citation_year=1987;  citation_volume=2;  citation_pages=191-210; " /><meta name="citation_reference" content="citation_title=Reconstructing evolutionary trees from DNA and protein sequences: paralinear distances; Proc. Natl Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=1455-9; " /><meta name="citation_reference" content="citation_title=A method for deducing branching sequences in phylogeny; Evolution;  citation_year=1965;  citation_volume=19;  citation_pages=311-26; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis: models and estimation procedures; Evolution;  citation_year=1967;  citation_volume=21;  citation_pages=550-70; " /><meta name="citation_reference" content="citation_title=Toward defining the course of evolution: minimum change for a specific tree topology; Syst. Zool.;  citation_year=1971;  citation_volume=35;  citation_pages=406-16; " /><meta name="citation_reference" content="citation_title=Maximum likelihood and minimum-steps methods for estimating evolutionary trees from data on discrete characters; Syst. Zool.;  citation_year=1973;  citation_volume=22;  citation_pages=240-9; " /><meta name="citation_reference" content="citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach; J. Mol. Evol.;  citation_year=1981;  citation_volume=17;  citation_pages=368-76; " /><meta name="citation_reference" content="citation_title=A hidden Markov model approach to variation among sites in rate of evolution; Mol. Bio. Evol.;  citation_year=1996;  citation_volume=13;  citation_pages=93-104; " /><meta name="citation_reference" content="citation_title=Mitochondrial DNA sequences of primates: tempo and mode of evolution; J. Mol. Evol.;  citation_year=1982;  citation_volume=18;  citation_pages=225-39; " /><meta name="citation_reference" content="citation_title=MEGA: A biologist-centric software for evolutionary analysis of DNA and protein sequences; Brief. Bioinform.;  citation_year=2008;  citation_volume=9;  citation_pages=299-306; " /><meta name="citation_reference" content="citation_title=Evolution of base-substitution gradients in primate mitochondrial genomes; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=665-73; " /><meta name="citation_reference" content="citation_title=The causes and consequences of HIV evolution; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=52-61; " /><meta name="citation_reference" content="citation_title=Origin of HIV-1 in the chimpanzee Pan troglodytes troglodytes; Nature;  citation_year=1999;  citation_volume=397;  citation_pages=436-41; " /><meta name="citation_reference" content="citation_title=Identification of a new human coronavirus; Nat. Med.;  citation_year=2004;  citation_volume=10;  citation_pages=368-73; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of viruses related to the SARS coronavirus from animals in southern China; Science;  citation_year=2003;  citation_volume=302;  citation_pages=276-8; " /><meta name="citation_reference" content="citation_title=SARS-CoV infection in a restaurant from palm civet; Emerg. Infect. Dis.;  citation_year=2005;  citation_volume=11;  citation_pages=1860-5; " /><meta name="citation_reference" content="citation_title=Characterization and complete genome sequence of a novel coronavirus, coronavirus HKU1, from patients with pneumonia; J. Virol.;  citation_year=2005;  citation_volume=79;  citation_pages=884-95; " /><meta name="citation_reference" content="citation_title=Chromosome rearrangements in evolution: From gene order to genome sequence and back; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=11188-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/3/155/1011724/dsq008.pdf" /><meta name="description" content="Abstract.  A genome space is a moduli space of genomes. In this space, each point corresponds to a genome. The natural distance between two points in the genome" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/3/155/353741" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/3/155/353741" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/3/155/353741");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq008';
    </script>
<title>
	Novel Construction of Genome Space with Biological Geometry | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353741" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfxCPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZdSwgQ2hlbmdsb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNYXRoZW1hdGljcywgVGhlIENoaW5lc2UgVW5pdmVyc2l0eSBvZiBIb25nIEtvbmcsIFNoYXRpbiwgSG9uZyBLb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGlhbmcsIFFpYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1hdGhlbWF0aWNzLCBTdGF0aXN0aWNzIGFuZCBDb21wdXRlciBTY2llbmNlLCBVbml2ZXJzaXR5IG9mIElsbGlub2lzIGF0IENoaWNhZ28sIENoaWNhZ28sIElMIDYwNjA3LTcwNDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllpbiwgQ2hhbmdjaHVhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWF0aGVtYXRpY3MsIFN0YXRpc3RpY3MgYW5kIENvbXB1dGVyIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgSWxsaW5vaXMgYXQgQ2hpY2FnbywgQ2hpY2FnbywgSUwgNjA2MDctNzA0NSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGUsIFJvbmcgTC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIENoaWNhZ28gU3RhdGUgVW5pdmVyc2l0eSwgQ2hpY2FnbywgSUwgNjA2MjgsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhdSwgU3RlcGhlbiBTLi1ULiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWF0aGVtYXRpY3MsIFN0YXRpc3RpY3MgYW5kIENvbXB1dGVyIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgSWxsaW5vaXMgYXQgQ2hpY2FnbywgQ2hpY2FnbywgSUwgNjA2MDctNzA0NSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBIE5vdmVsIENvbnN0cnVjdGlvbiBvZiBHZW5vbWUgU3BhY2Ugd2l0aCBCaW9sb2dpY2FsIEdlb21ldHJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTU1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxNjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3EwMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMC8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpZyB0cmVlIGZyb20gbGl0dGxlIGdlbm9tZXM6IG1pdG9jaG9uZHJpYWwgZ2VuZSBvcmRlciBhcyBhIHBoeWxvZ2VuZXRpYyB0b29sOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NjY4LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgcGh5bG9nZW55IGJhc2VkIG9uIGdlbmUgY29udGVudDsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMDgtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4YWN0IGFuZCBhcHByb3hpbWF0aW9uIGFsZ29yaXRobXMgZm9yIHRoZSBpbnZlcnNpb24gZGlzdGFuY2U7IEFsZ29yaXRobWljYTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTE4MC0yMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zZm9ybWluZyBjYWJiYWdlIGludG8gdHVybmlwOyBQcm9jZWVkaW5ncyBvZiB0aGUgMjd0aCBBQ00gU3ltcG9zaXVtIG9uIFRoZW9yeSBvZiBDb21wdXRpbmc7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvdW50aW5nIG9uIGNvbXBhcmF0aXZlIG1hcHM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00OTUtNTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZW1lcmdpbmcgcGFyYWRpZ20gYW5kIG9wZW4gcHJvYmxlbXMgaW4gY29tcGFyYXRpdmUgZ2Vub21pY3M7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjY1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIEROQSBzZXF1ZW5jZSByZXByZXNlbnRhdGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTMxNDsgIGNpdGF0aW9uX3BhZ2VzPTU4NS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1wbGVyIEROQSBzZXF1ZW5jZSByZXByZXNlbnRhdGlvbnM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTY7ICBjaXRhdGlvbl9wYWdlcz0yMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgc2VxdWVuY2UgcmVwcmVzZW50YXRpb24gd2l0aG91dCBkZWdlbmVyYWN5OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMDc4LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb3RlaW4gbWFwIGFuZCBpdHMgYXBwbGljYXRpb247IEROQSBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjQxLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9SGluZHVzdGFuIFB1Ymxpc2hpbmcgQ29ycG9yYXRpb24sIEluZGlhOyBCYXNpYyBBbGdlYnJhOyAgY2l0YXRpb25feWVhcj0xOTc0OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgcHJvdGVpbiBtb2xlY3VsZXMgaW4gbWFtbWFsaWFuIHByb3RlaW4gbWV0YWJvbGlzbTsgY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCBOZXcgWW9yazsgTWFtbWFsaWFuIFByb3RlaW4gTWV0YWJvbGlzbTsgIGNpdGF0aW9uX3llYXI9MTk2OTsgIGNpdGF0aW9uX3BhZ2VzPTIxLTEzMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUgbW9kZWwgZm9yIGVzdGltYXRpbmcgZXZvbHV0aW9uYXJ5IHJhdGVzIG9mIGJhc2Ugc3Vic3RpdHV0aW9ucyB0aHJvdWdoIGNvbXBhcmF0aXZlIHN0dWRpZXMgb2YgbnVjbGVvdGlkZSBzZXF1ZW5jZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsIGFuYWx5c2lzIG9mIG9taW5vaWQgbW9sZWN1bGFyIGV2b2x1dGlvbjsgU3RhdC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTkxLTIxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb25zdHJ1Y3RpbmcgZXZvbHV0aW9uYXJ5IHRyZWVzIGZyb20gRE5BIGFuZCBwcm90ZWluIHNlcXVlbmNlczogcGFyYWxpbmVhciBkaXN0YW5jZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xNDU1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWV0aG9kIGZvciBkZWR1Y2luZyBicmFuY2hpbmcgc2VxdWVuY2VzIGluIHBoeWxvZ2VueTsgRXZvbHV0aW9uOyAgY2l0YXRpb25feWVhcj0xOTY1OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MzExLTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgYW5hbHlzaXM6IG1vZGVscyBhbmQgZXN0aW1hdGlvbiBwcm9jZWR1cmVzOyBFdm9sdXRpb247ICBjaXRhdGlvbl95ZWFyPTE5Njc7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz01NTAtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZCBkZWZpbmluZyB0aGUgY291cnNlIG9mIGV2b2x1dGlvbjogbWluaW11bSBjaGFuZ2UgZm9yIGEgc3BlY2lmaWMgdHJlZSB0b3BvbG9neTsgU3lzdC4gWm9vbC47ICBjaXRhdGlvbl95ZWFyPTE5NzE7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz00MDYtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1heGltdW0gbGlrZWxpaG9vZCBhbmQgbWluaW11bS1zdGVwcyBtZXRob2RzIGZvciBlc3RpbWF0aW5nIGV2b2x1dGlvbmFyeSB0cmVlcyBmcm9tIGRhdGEgb24gZGlzY3JldGUgY2hhcmFjdGVyczsgU3lzdC4gWm9vbC47ICBjaXRhdGlvbl95ZWFyPTE5NzM7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yNDAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IHRyZWVzIGZyb20gRE5BIHNlcXVlbmNlczogYSBtYXhpbXVtIGxpa2VsaWhvb2QgYXBwcm9hY2g7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0zNjgtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlkZGVuIE1hcmtvdiBtb2RlbCBhcHByb2FjaCB0byB2YXJpYXRpb24gYW1vbmcgc2l0ZXMgaW4gcmF0ZSBvZiBldm9sdXRpb247IE1vbC4gQmlvLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTkzLTEwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWl0b2Nob25kcmlhbCBETkEgc2VxdWVuY2VzIG9mIHByaW1hdGVzOiB0ZW1wbyBhbmQgbW9kZSBvZiBldm9sdXRpb247IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0yMjUtMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E6IEEgYmlvbG9naXN0LWNlbnRyaWMgc29mdHdhcmUgZm9yIGV2b2x1dGlvbmFyeSBhbmFseXNpcyBvZiBETkEgYW5kIHByb3RlaW4gc2VxdWVuY2VzOyBCcmllZi4gQmlvaW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9Mjk5LTMwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGJhc2Utc3Vic3RpdHV0aW9uIGdyYWRpZW50cyBpbiBwcmltYXRlIG1pdG9jaG9uZHJpYWwgZ2Vub21lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz02NjUtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjYXVzZXMgYW5kIGNvbnNlcXVlbmNlcyBvZiBISVYgZXZvbHV0aW9uOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01Mi02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIG9mIEhJVi0xIGluIHRoZSBjaGltcGFuemVlIFBhbiB0cm9nbG9keXRlcyB0cm9nbG9keXRlczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTM5NzsgIGNpdGF0aW9uX3BhZ2VzPTQzNi00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBuZXcgaHVtYW4gY29yb25hdmlydXM7IE5hdC4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTM2OC03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHZpcnVzZXMgcmVsYXRlZCB0byB0aGUgU0FSUyBjb3JvbmF2aXJ1cyBmcm9tIGFuaW1hbHMgaW4gc291dGhlcm4gQ2hpbmE7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzAyOyAgY2l0YXRpb25fcGFnZXM9Mjc2LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNBUlMtQ29WIGluZmVjdGlvbiBpbiBhIHJlc3RhdXJhbnQgZnJvbSBwYWxtIGNpdmV0OyBFbWVyZy4gSW5mZWN0LiBEaXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTg2MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIGFuZCBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgYSBub3ZlbCBjb3JvbmF2aXJ1cywgY29yb25hdmlydXMgSEtVMSwgZnJvbSBwYXRpZW50cyB3aXRoIHBuZXVtb25pYTsgSi4gVmlyb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9ODg0LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21lIHJlYXJyYW5nZW1lbnRzIGluIGV2b2x1dGlvbjogRnJvbSBnZW5lIG9yZGVyIHRvIGdlbm9tZSBzZXF1ZW5jZSBhbmQgYmFjazsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0xMTE4OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTcvMy8xNTUvMTAxMTcyNC9kc3EwMDgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEEgZ2Vub21lIHNwYWNlIGlzIGEgbW9kdWxpIHNwYWNlIG9mIGdlbm9tZXMuIEluIHRoaXMgc3BhY2UsIGVhY2ggcG9pbnQgY29ycmVzcG9uZHMgdG8gYSBnZW5vbWUuIFRoZSBuYXR1cmFsIGRpc3RhbmNlIGJldHdlZW4gdHdvIHBvaW50cyBpbiB0aGUgZ2Vub21lIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTM3NDEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk2EM8huBukxHGfUetOwJqz7dEHMM=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f155%2f353741" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f155%2f353741" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13687" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&amp;Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/3">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5037193">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5037193">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5037195">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5037195">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5037199">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5037199">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5037230">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5037230">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5037267">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5037267">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5037280">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5037280">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5037283">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5037283">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/3/139/354144">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/3/169/353612">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A Novel Construction of Genome Space with Biological Geometry
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chenglong Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Mathematics</div>, <div class="insititution">The Chinese University of Hong Kong</div>, <div class="addr-line">Shatin</div>, <div class="country">Hong Kong</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chenglong+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yu C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chenglong Yu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qian Liang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Mathematics, Statistics and Computer Science</div>, <div class="insititution">University of Illinois at Chicago</div>, <div class="addr-line">Chicago, IL 60607-7045</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qian+Liang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liang Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liang Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qian Liang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Changchuan Yin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Mathematics, Statistics and Computer Science</div>, <div class="insititution">University of Illinois at Chicago</div>, <div class="addr-line">Chicago, IL 60607-7045</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Changchuan+Yin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yin C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yin C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Changchuan Yin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rong L. He</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">Chicago State University</div>, <div class="addr-line">Chicago, IL 60628</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rong+L.+He">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=He R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;He R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rong L. He</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stephen S.-T. Yau</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Mathematics, Statistics and Computer Science</div>, <div class="insititution">University of Illinois at Chicago</div>, <div class="addr-line">Chicago, IL 60607-7045</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel./Fax. +1 312-996-3065. E-mail: <a href="mailto:yau@uic.edu" target="_blank">yau@uic.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stephen+S.-T.+Yau">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yau S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yau S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stephen S.-T. Yau</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 3, 1 June 2010, Pages 155–168, <a href='https://doi.org/10.1093/dnares/dsq008'>https://doi.org/10.1093/dnares/dsq008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 April 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 December 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 March 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353741" href="/dnaresearch/article-pdf/17/3/155/1011724/dsq008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Chenglong Yu, Qian Liang, Changchuan Yin, Rong L. He, Stephen S.-T. Yau;  A Novel Construction of Genome Space with Biological Geometry, <em>DNA Research</em>, Volume 17, Issue 3, 1 June 2010, Pages 155–168, <a href='https://doi.org/10.1093/dnares/dsq008'>https://doi.org/10.1093/dnares/dsq008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353741?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353741?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353741?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353741?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353741?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353741?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353741?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20Novel%20Construction%20of%20Genome%20Space%20with%20Biological%20Geometry&amp;publicationDate=2010-04-01&amp;volumeNum=17&amp;issueNum=3&amp;author=Yu%2C%20Chenglong%3B%20Liang%2C%20Qian&amp;startPage=155&amp;endPage=168&amp;contentId=10.1093%2Fdnares%2Fdsq008&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f155%2f353741" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5037193 id="5037193" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>A genome space is a moduli space of genomes. In this space, each point corresponds to a genome. The natural distance between two points in the genome space reflects the biological distance between these two genomes. Currently, there is no method to represent genomes by a point in a space without losing biological information. Here, we propose a new graphical representation for DNA sequences. The breakthrough of the subject is that we can construct the moment vectors from DNA sequences using this new graphical method and prove that the correspondence between moment vectors and DNA sequences is one-to-one. Using these moment vectors, we have constructed a novel genome space as a subspace in <em>R</em><sup><em>N</em></sup>. It allows us to show that the SARS-CoV is most closely related to a coronavirus from the palm civet not from a bird as initially suspected, and the newly discovered human coronavirus HCoV-HKU1 is more closely related to SARS than to any other known member of group 2 coronavirus. Furthermore, we reconstructed the phylogenetic tree for 34 lentiviruses (including human immunodeficiency virus) based on their whole genome sequences. Our genome space will provide a new powerful tool for analyzing the classification of genomes and their phylogenetic relationships.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome space&quot;">genome space</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;graphical representation&quot;">graphical representation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;moment vector&quot;">moment vector</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="classification">classification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogeny">phylogeny</a></div>                            <h2 scrollto-destination=5037195 id="5037195" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Comparative genomics at the sequence level has existed for ∼20 years, since the time genome sequencing started in earnest. Already there have been many proposals to compare genomes. Boore and Brown<sup><a reveal-id="DSQ008C1" data-open="DSQ008C1" class="link link-ref link-reveal xref-bibr">1</a></sup> used gene order to study the evolutionary relationships of metazoan mitochondrial genomes. Snel <em>et al</em>.<sup><a reveal-id="DSQ008C2" data-open="DSQ008C2" class="link link-ref link-reveal xref-bibr">2</a></sup> later constructed the phylogenetic tree for completely sequenced prokaryotic genomes based on gene content. In Snel <em>et al</em>.'s method, the similarity between two genomes is defined as the number of genes that they have in common divided by their total number of genes. However, such techniques are time-consuming as they must first identify all the genes in one genome. These approaches, together with G + C content, edit distance, and reversal and rearrangement distances,<sup><a reveal-id="DSQ008C3  DSQ008C5" data-open="DSQ008C3  DSQ008C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> compare genomes using only partial genomic information. Thus, these results are usually controversial because single-gene sequences generally do not contain enough information to construct an evolutionary history of organisms.</p><p>In Koonin's editorial<sup><a reveal-id="DSQ008C6" data-open="DSQ008C6" class="link link-ref link-reveal xref-bibr">6</a></sup> about the emerging paradigm and open problems in comparative genomics, he pointed out ‘… but within these superfamilies, there is nothing like a straight one-to-one correspondence between genomes, and in distant genomes, most of the members may not be orthologous’. This sentence enlightened us to get a novel idea of comparing genomes. We can construct a genome space. In this space, each point corresponds to a genome uniquely. The natural distance between two points in the genome space reflects the biological distance between these two genomes. Currently, there is no method to represent genomes by a point in a space without losing biological information.</p><p>We introduce graphical representation of DNA sequence to construct the genome space. The graphical representation of DNA sequence provides a simple way of viewing, sorting, and comparing various gene structures. Thus, it is an attractive and promising research direction. The first important method in this direction is due to Hamori.<sup><a reveal-id="DSQ008C7" data-open="DSQ008C7" class="link link-ref link-reveal xref-bibr">7</a></sup> He used a three-dimensional curve to represent a DNA sequence. Gates<sup><a reveal-id="DSQ008C8" data-open="DSQ008C8" class="link link-ref link-reveal xref-bibr">8</a></sup> later constructed a two-dimensional graphical representation that is simpler than the Hamori curve. However, Gates' graphical representation has high degeneracy. Recently, we reported a new two-dimensional graphical representation of gene sequences<sup><a reveal-id="DSQ008C9" data-open="DSQ008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> which has no circuit or degeneracy, so that the correspondence between gene sequences and gene graphs is one-to-one. In this way, the original DNA sequence can be recovered from its graph mathematically without loss of biological information. In this paper, we make a minor modification of our previous method and obtain a new graphical representation approach for DNA sequences. The breakthrough of the subject is that we can construct the moment vectors from DNA sequences using this new graphical method, and we can prove that the correspondence between moment vectors and DNA sequences is one-to-one. The novelty and uniqueness of our approach is that by using these moment vectors of DNA sequences, we have constructed a genome space as a subspace in Euclidean space. Each genome sequence can be represented as a point in this space. Therefore, this genome space can be used to make comparative analysis to study the clustering and phylogenetic relationship among genomes. The biological (evolutionary) distance between two genomes can be obtained through the Euclidean distance among the corresponding points in the genome space.</p>                            <h2 scrollto-destination=5037199 id="5037199" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5037200 id="5037200" class="section-title" data-legacy-id=s2a>2.1. Graphical representation of DNA sequence</h3>
<div class="block-child-p">We constructed a new DNA sequence graph in two quadrants of the Cartesian coordinate system, with pyrimidines (C and T) in the first quadrant and purines (A and G) in the fourth quadrant. As shown in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>, the vectors corresponding to the four nucleotides G, A, T, and C are as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq008ueq1.gif?Expires=2147483647&Signature=dmtaJnlW6K5sugmn4RzjuP3PwwiIfDD1mH73MSSOUn3hQeTeqI7yVFDf6Iyci83tqX2Q-m6nJxkfhgvaD4gRHngSKSnXx7i5C4ilFDrDfwL4bhMpJVZfAjpAVrLSlBopDyrNNO-dTl9v9q4QDV4iOlXXdxQMC-LPjxBGg~ZW4zrFnYhUq7mCnnD0irUq-c4D-455NfonGVWDJlXEI8qIsAC70AM7kLhh8i-3g94kEq2nVa~irWjoLm8MToLIB4PLBxWgGGv9dBCKSYKpykhPr9I-jfpbNvukU47EknxJ28pv1Jo3uF1DxNI1t7SyL8KOnXugnWAesKuF61AEd5iQBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsq008ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ueq1.gif?Expires=2147483647&Signature=mn7zYFBbjo7A9gT3-LtyRc0rKB9dBRGGxNzB~fb9LK8dngbgvg7oLzgoZcoEU--gcXppJE2kq9Djbx0ZyNdw2n189B0fA2lbjxofWD1vgIHhCRfnnw4zr3-dyYiEYVN4SC4-WTyJJ0N9ElyN7tAkWNAyeuhpM5vK701ilJGKNd3Ku0T9wXSIXS6jiKNLy16tlDdS4AOtDitdHuvjcQmgFJ3R3p3GdKfQikjQlW6o-pambVPquDSgdZ~CF-vkmVun3Ldgpt9OGzzXyYcYv0pGe8GFKoF5pCuQ7fRi-uLPGvX6mscGMl8U1UfxAnyLjgM5tQyvLqBMG~FWZTgk3hnACw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq008ueq1.gif" /></a></div></div></div></div>                        <a id="5037202" scrollto-destination="5037202"></a>
<div id="DSQ008F1" data-id="DSQ008F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00801.jpeg?Expires=2147483647&Signature=s2D6muq3DotWRj3ui2ckN07PPDMFwKIJIbtNlh1MFzWwLrT5jEU5dqmMkbVCovLOtSfqI5RCYaQXBJk3cRf51pVfiPB2nVvKNCoq3~aOZ3liudV~A0Myc9GvCSLKy37hRkKTDFgufSS3S3Hehz90hEsWlX5vLdpus9u8k5YnJtg0fNI8HMAmOTmAZdsaJVhJU85AAGx7PIGbdc0mjVZIB1lR6pyCx3P0Dgc8de3aFDZ1UzYue336g5-g58bu-79W08GZpr8msgLoH4C3avjtezkr7OZz-gmMib5k2U9bAB~04fRRamX6cbhU6XavVkm3sy6jIjfBaerGq1zuliWCqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nucleotide vector system based on G(1, −2/3), A(1, −1/3), T(1, 1/3), and C(1, 2/3)." path-from-xml="dsq00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037202/dsq00801.jpeg" path-from-xml="dsq00801.jpeg" target="_blank">View large</a><a section="5037202" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00801.jpeg?Expires=2147483647&Signature=LWhxOxJjQOvGIyNNZ~-OwqeUahdMzJxW3FYMVvT9NrAcdGz5hN8KUQWUXjeM~J2csO3wp37VtcDz~H9zZlzqHxH1EWTGguMQg0JVvxwqXrcvWU4XAToLANSXj4gEUebWcMrZeJ0slEleR4~7lQs2MvY5kMh9LREc55cUqgVFbnhp848C46rYdjrYwuy62PHLRW3ozmbzjf46KVm0pGd6aqrAgR0gKNkPL7i0mz6~ew2-pe3HEFqDzUSw3WXsnrSk~uyW5ODwXVWgrCGODp1f6k3nXB-YkoaXSFOLwS6pBPdTaDEhNZy4mSB7rrbSWHl5oN~h4p0sv9Vb480Cc2uQog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037202&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nucleotide vector system based on G(1, −2/3), A(1, −1/3), T(1, 1/3), and C(1, 2/3).</p></div></div><div content-id="DSQ008F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00801.jpeg?Expires=2147483647&Signature=s2D6muq3DotWRj3ui2ckN07PPDMFwKIJIbtNlh1MFzWwLrT5jEU5dqmMkbVCovLOtSfqI5RCYaQXBJk3cRf51pVfiPB2nVvKNCoq3~aOZ3liudV~A0Myc9GvCSLKy37hRkKTDFgufSS3S3Hehz90hEsWlX5vLdpus9u8k5YnJtg0fNI8HMAmOTmAZdsaJVhJU85AAGx7PIGbdc0mjVZIB1lR6pyCx3P0Dgc8de3aFDZ1UzYue336g5-g58bu-79W08GZpr8msgLoH4C3avjtezkr7OZz-gmMib5k2U9bAB~04fRRamX6cbhU6XavVkm3sy6jIjfBaerGq1zuliWCqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nucleotide vector system based on G(1, −2/3), A(1, −1/3), T(1, 1/3), and C(1, 2/3)." path-from-xml="dsq00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037202/dsq00801.jpeg" path-from-xml="dsq00801.jpeg" target="_blank">View large</a><a section="5037202" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00801.jpeg?Expires=2147483647&Signature=LWhxOxJjQOvGIyNNZ~-OwqeUahdMzJxW3FYMVvT9NrAcdGz5hN8KUQWUXjeM~J2csO3wp37VtcDz~H9zZlzqHxH1EWTGguMQg0JVvxwqXrcvWU4XAToLANSXj4gEUebWcMrZeJ0slEleR4~7lQs2MvY5kMh9LREc55cUqgVFbnhp848C46rYdjrYwuy62PHLRW3ozmbzjf46KVm0pGd6aqrAgR0gKNkPL7i0mz6~ew2-pe3HEFqDzUSw3WXsnrSk~uyW5ODwXVWgrCGODp1f6k3nXB-YkoaXSFOLwS6pBPdTaDEhNZy4mSB7rrbSWHl5oN~h4p0sv9Vb480Cc2uQog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037202&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nucleotide vector system based on G(1, −2/3), A(1, −1/3), T(1, 1/3), and C(1, 2/3).</p></div></div><p>Here, we should emphasize that the specific ordering of the four nucleotides in the Cartesian coordinate system is related to the GC content of genomes (as it is explained in the ‘Discussion’ section). Points in the graphical representation are obtained by the sum of vectors representing nucleotides in the sequence. In Fig. <a reveal-id="DSQ008F2" data-open="DSQ008F2" class="link link-reveal link-table xref-fig">2</a>, we give the graphical representation of whole mitochondrial genome sequences of human, common chimpanzee, Norway rat, and hedgehog, which are based on the vector system shown in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>. Since human and chimpanzee belong to the same order (primates), their mitochondrial genome graphical representations are very similar visually. These four DNA sequence graphical curves have no circuits or degeneracy and the correspondence between the sequence and the graphical curve can be mathematically proved to be one-to-one.<sup><a reveal-id="DSQ008C9" data-open="DSQ008C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p>                        <a id="5037204" scrollto-destination="5037204"></a>
<div id="DSQ008F2" data-id="DSQ008F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00802.jpeg?Expires=2147483647&Signature=TJjwnnAj1N6nm0xXa0QkUC7UAEhc2va96q~YPIEJUGnxRMmdReULbtl24hM4T0oOyGTW797FJKfxm4Tj02O5pC~VLRh~Wv8kFAZNbx-9UkQcPXnR5AwndTJaRhp7XPBhCPWZQ2KlZqTcKFC~Mi5MCqPLDkFWn~hGCFx6Znpz7pGc2lj~qjIfYuX3oSpz3DybW8ZmSjE-JwZzj6MbZFV20BBJ8hXWDHS86F~u-C5LYV7ZVs-9Jho0USzx7W0klIkzuXUlqNKt~KOJkt3bbN7JMvxrZsYf0JP6t7IZ-tzl5Z2xSZ4vRtjs16aE-eBiqzXlzy7JJX42Ty62-hIb4ScQlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical representation of DNA sequences. Graphical representations of whole mitochondrial genome sequences of four species (human, common chimpanzee, Norway rat, and hedgehog) based on the vector system shown in Fig. 1. X-value stands for the number of nucleotides in the DNA sequence. Y-value is the cumulative y-values of nucleotides G, A, T, and C in Fig. 1. Because the genome lengths of these four species are similar (around 17 000 nt), the X-values of the end points of their graphical curves are very similar. For the Y-values of the end points, human and common chimpanzee are very close (more than 1600), but mouse is below 1500 and hedgehog is below 1000." path-from-xml="dsq00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037204/dsq00802.jpeg" path-from-xml="dsq00802.jpeg" target="_blank">View large</a><a section="5037204" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00802.jpeg?Expires=2147483647&Signature=icq-KxU3Wgm3kJL0bAtDPuidduaAEBqy5rdfR45nt47NEuGXpnyVhHMwW3D3upsWX4XbvwmOFLHJLa-bZBh4vIL7ZW8Wj2H9fjRGVHHEB72dU-iQDMMLXP9yDSlAv2xRpYo-o2frak2CjsP85eKakzPTN883wqyoKqkcwLNtOD~Qf0o3SVVzF0wlznq8~p5R4MZW~tBs6feYGGt6LXpFaLnXhY~HTTgfXSUAsbTa4Iiq8M1EnrKWyKhfl5Qsl4LKW-JckkEMgPto8WGMlZT1SjHH2JHcxDFfhTeA00vnJCwGJif0~vEcYIrIlQkd8hg0SlgLRHmbm~yW5pYUFwGqJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037204&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical representation of DNA sequences. Graphical representations of whole mitochondrial genome sequences of four species (human, common chimpanzee, Norway rat, and hedgehog) based on the vector system shown in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>. <em>X</em>-value stands for the number of nucleotides in the DNA sequence. <em>Y</em>-value is the cumulative <em>y</em>-values of nucleotides G, A, T, and C in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>. Because the genome lengths of these four species are similar (around 17 000 nt), the <em>X</em>-values of the end points of their graphical curves are very similar. For the Y-values of the end points, human and common chimpanzee are very close (more than 1600), but mouse is below 1500 and hedgehog is below 1000.</p></div></div><div content-id="DSQ008F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00802.jpeg?Expires=2147483647&Signature=TJjwnnAj1N6nm0xXa0QkUC7UAEhc2va96q~YPIEJUGnxRMmdReULbtl24hM4T0oOyGTW797FJKfxm4Tj02O5pC~VLRh~Wv8kFAZNbx-9UkQcPXnR5AwndTJaRhp7XPBhCPWZQ2KlZqTcKFC~Mi5MCqPLDkFWn~hGCFx6Znpz7pGc2lj~qjIfYuX3oSpz3DybW8ZmSjE-JwZzj6MbZFV20BBJ8hXWDHS86F~u-C5LYV7ZVs-9Jho0USzx7W0klIkzuXUlqNKt~KOJkt3bbN7JMvxrZsYf0JP6t7IZ-tzl5Z2xSZ4vRtjs16aE-eBiqzXlzy7JJX42Ty62-hIb4ScQlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical representation of DNA sequences. Graphical representations of whole mitochondrial genome sequences of four species (human, common chimpanzee, Norway rat, and hedgehog) based on the vector system shown in Fig. 1. X-value stands for the number of nucleotides in the DNA sequence. Y-value is the cumulative y-values of nucleotides G, A, T, and C in Fig. 1. Because the genome lengths of these four species are similar (around 17 000 nt), the X-values of the end points of their graphical curves are very similar. For the Y-values of the end points, human and common chimpanzee are very close (more than 1600), but mouse is below 1500 and hedgehog is below 1000." path-from-xml="dsq00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037204/dsq00802.jpeg" path-from-xml="dsq00802.jpeg" target="_blank">View large</a><a section="5037204" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00802.jpeg?Expires=2147483647&Signature=icq-KxU3Wgm3kJL0bAtDPuidduaAEBqy5rdfR45nt47NEuGXpnyVhHMwW3D3upsWX4XbvwmOFLHJLa-bZBh4vIL7ZW8Wj2H9fjRGVHHEB72dU-iQDMMLXP9yDSlAv2xRpYo-o2frak2CjsP85eKakzPTN883wqyoKqkcwLNtOD~Qf0o3SVVzF0wlznq8~p5R4MZW~tBs6feYGGt6LXpFaLnXhY~HTTgfXSUAsbTa4Iiq8M1EnrKWyKhfl5Qsl4LKW-JckkEMgPto8WGMlZT1SjHH2JHcxDFfhTeA00vnJCwGJif0~vEcYIrIlQkd8hg0SlgLRHmbm~yW5pYUFwGqJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037204&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical representation of DNA sequences. Graphical representations of whole mitochondrial genome sequences of four species (human, common chimpanzee, Norway rat, and hedgehog) based on the vector system shown in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>. <em>X</em>-value stands for the number of nucleotides in the DNA sequence. <em>Y</em>-value is the cumulative <em>y</em>-values of nucleotides G, A, T, and C in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>. Because the genome lengths of these four species are similar (around 17 000 nt), the <em>X</em>-values of the end points of their graphical curves are very similar. For the Y-values of the end points, human and common chimpanzee are very close (more than 1600), but mouse is below 1500 and hedgehog is below 1000.</p></div></div>                            <h3 scrollto-destination=5037205 id="5037205" class="section-title" data-legacy-id=s2b>2.2. Moment vector of DNA sequence</h3>
<div class="block-child-p">Motivated by our previous work for protein sequence analysis,<sup><a reveal-id="DSQ008C10" data-open="DSQ008C10" class="link link-ref link-reveal xref-bibr">10</a></sup> we use moment vectors to characterize a DNA graphical curve. Given the graphical curve of a DNA sequence, which can be represented by a sequence of points (1, <em>y</em><sub>1</sub>), (2, <em>y</em><sub>2</sub>), … , (<em>n</em>, <em>y</em><sub><em>n</em></sub>), we can compute a sequence of numbers 1 − <em>y</em><sub>1</sub>, 2 − <em>y</em><sub>2</sub>, … , <em>n</em> − <em>y</em><sub><em>n</em></sub>. Conversely, if we know the sequence of numbers 1 − <em>y</em><sub>1</sub>, 2 − <em>y</em><sub>2</sub>, … , <em>n</em> − <em>y</em><sub><em>n</em></sub>, we can recover the graph (1, <em>y</em><sub>1</sub>), (2, <em>y</em><sub>2</sub>), … , (<em>n</em>, <em>y</em><sub><em>n</em></sub>). Therefore, we want to find a sequence of numbers, each of which uses the global information of the sequence of numbers 1 − <em>y</em><sub>1</sub>, 2 − <em>y</em><sub>2</sub>, … , <em>n</em> − <em>y</em><sub><em>n</em></sub> in such a way that this new sequence of numbers determines and is determined by the sequence of numbers 1 − <em>y</em><sub>1</sub>, 2 − <em>y</em><sub>2</sub>, … , <em>n</em> − <em>y</em><sub><em>n</em></sub>. For this purpose, we consider the moments which are defined as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq008ueq2.gif?Expires=2147483647&Signature=Pw9p-09O1UpPJYAbgrwQuqzRydqJo5ko9BoXpAt1z59meDhIPBf9SrRv9Rxm5Ya6JVBUHvtlCQ6Zzz2RNFXdXQeLR2g41zLqxYXBlcVu9jS96JwSgovdh-WWTgtGIp3pbcuql~U6RDWW2qWh~pDyuJ4y5YJC9o2h~37c5vuzVRLh5TJhBKOADZd3SqK~sIkUBDGseYxEUnXeh9eqqKrnsVagiFyw7mGRbONA-stIvPMl7UGuP4VpVhF7cjo9hgDk2E3DTVtAEUdFjhIUqlt-l~DpECfGEDS5NJkDDPwc4g048SnUp5t8SKc85BbMabQYYrihOQar-AYqGD4ymNBYqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsq008ueq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ueq2.gif?Expires=2147483647&Signature=YF8emDPm8yssLO1DiMpF0xx3STc7zBzYMK8Q9JuQE-KZGZ9DF29PEXODXQqH6onag~sV8Ai2zvxSRSATB0goZ32fEyQbouXkJFt2w8jl98uY7vqb0P2ajBWXg~aqo6AOIA46AGjPmfoN1Zonw5yqU6G9SDwIzTdeGoMeeJEkUu19l96bTkIAeOQTIm7J9w-UQay-ziiJ7x~gUIYLnTJfAG6ol~Mi19bjLtSqe6PuuTa457SjUKJ2nhn0zvs9hOIfoQG5yxckHpoWBhyXQ70LZfdoZM~V~WeTIofNC78BSqFLoBsIzFtozijmYlSzbe9e7NKzBka9XVroymXifiKemQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq008ueq2.gif" /></a></div></div></div> where <em>n</em> is the number of nucleotides contained in a DNA sequence, and (<em>x</em><sub><em>i</em></sub>, <em>y</em><sub><em>i</em></sub>) represents the position of the <em>i</em>th nucleotide in the DNA graphical curve. According to this definition, each DNA sequence has an <em>n</em>-dimensional moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>n</em></sub>) associated with it.</div><p>The crucial point in this paper is that the correspondence between a DNA sequence and its moment vector obtained from its sequence graph is one-to-one. To obtain this conclusion, we need to prove the following theorem.</p><div class="statement"><p><strong>Theorem</strong></p><p>Consider the set of DNA sequences having the same number (<em>n</em>) of nucleotides. Then, the correspondence between a DNA sequence and its <em>n</em>-dimensional moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>n</em></sub>) is one-to-one.</p></div><div class="statement"><p><em>Proof</em></p><p>We have demonstrated that the correspondence between a DNA sequence and its graphical curve is one-to-one.<sup><a reveal-id="DSQ008C9" data-open="DSQ008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In order to prove the theorem, we will need to prove that the correspondence between a DNA graphical curve and its moment vector is one-to-one.</p><p>By the definition, one DNA sequence graph has an <em>n</em>-dimensional moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>n</em></sub>). Hence, we need to demonstrate that from any given DNA moment vector, we can recover the DNA curve, which means all (<em>x</em><sub><em>i</em></sub>, <em>y</em><sub><em>i</em></sub>) (<em>i</em> = 1, 2, … , <em>n</em>) can be recovered from any given DNA moment vector.</p><div class="block-child-p"><em>x</em><sub><em>i</em></sub> is the <em>x</em>-coordinate value of <em>i</em>th nucleotide on a DNA graph. Based on our assignment, <em>x</em><sub><em>i</em></sub> should be equal to <em>i. y</em><sub><em>i</em></sub> is the <em>y</em>-coordinate value of <em>i</em>th nucleotide on a DNA graph. The next step is to obtain <em>y</em><sub><em>i</em></sub> from moment vector. Let <em>z</em><sub><em>i</em></sub> = <em>x</em><sub><em>i</em></sub> − <em>y</em><sub><em>i</em></sub>, then the moments can be simplified as:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsq008ueq3.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ueq3.gif?Expires=2147483647&amp;Signature=n1zHdzk1avytQsUPkxfvJVNNqR2q5inqZCfSfBL9b6wYXMg1WoPvhe5gZXvYBPbychwYln7BW4hUgGnbIxhxC5O7zdrjxaflcPoNvvmb3LKToJAw5-zqEMKvjo5SORKH9a5j4pVQBRh1wcYLmucD0JAxW4JbhrtJInvPNugjpNiHAKrPtd01qZx6X0aHrdEJb1T~7St51nIGFSeRCLPi~IiKvYl-XXO-O4H8rG2hPQs3ofG8qj~u6pg-P1HE2uYwFn-pKxjTYPXFNwICwU-ZlvVx6a9~UJFq93phGm96EuqPI24F0rxDandEE4hGChDJlnMgLOT4XNN~XRWXf8bWwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq008ueq3.gif" /></a></div></div></div> To solve for <em>z</em><sub><em>i</em></sub>, let δ<sub><em>j</em></sub> = <em>M</em><sub><em>j</em></sub><em>n</em><sup><em>j</em></sup>, then the δ<sub><em>j</em></sub> can be obtained by <em>M</em><sub><em>j</em></sub> and <em>n</em>. Clearly, δ<sub><em>j</em></sub> and <em>z</em><sub><em>i</em></sub> have the below relation:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsq008ueq4.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ueq4.gif?Expires=2147483647&amp;Signature=jTuXDXKp1nafWsM9Dp-NyrTiEkVtzAV-2YiFVx6KtxiunDsvrKNs4vrdjSzNKQ9XGndfXct7ClrjeudJjZmUs8C0jdctk-G3gHG-4PhTUSoR4M38VaOYHjK3ev6tAIw3HB4jwRc6s7XImZgr~spSm9bqAA0iJVUp3P-ccHyg7yqw6f6ypiTHIs4ongC-I66SnW8mvgzD1goS76JTIinX22Z5ee255k0RrtBEhdDA0CgT0Ifl82cH6q7UdBJ94-YgxHP0a3~czqmsTMbLKGBUuw0zRkHkxUlbg0WmDWfSuwEY4lsOzy-9--6Ci0-ZUzCkVeDC1PDf2QQUMgEmdUTmBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq008ueq4.gif" /></a></div></div></div> These <em>z</em><sub>1</sub>, <em>z</em><sub>2</sub>, … , <em>z</em><sub><em>n</em></sub> are roots of the polynomial<span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq1.gif?Expires=2147483647&amp;Signature=LrO3h3AbP5Gn26LuK1n9uKo3cFupwmAvDDxhGp6TwjVFEkR4P3bykezeCJZONxCv7blJ36mfaWpyN8f2wAX-XULOj4cIoon6euKwGGwtFa4KN3z-Qubyx4UkuHfEBELZBjcgGKSvunbnOJ0Bhbp740RTxgYOo2VHCaCtvgfuC3h87PhaOKWXVvnOaa5FmwR~5WW194whhTodMEf2O1fjclvCI9pz2PAB5ZuLGkDnVegq6qQzCwMmZq8CnvcUlDeJcJjEGc~i9oTzqLGMRNFP~IpbpUGrqcFsMvaKTqaK9SC5e19G1BCypuzmj-mAcWkG7zH01rTURvWpga1VaLptaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq1.gif" /></span><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq1a.gif?Expires=2147483647&amp;Signature=NzpVZOQiX85m-bGQboC3XwP9a4flgezR~4MutDrepnvwwWt7iHQHl0fzTV4SCAqXTXBQ8enxCDvZQ2VlKNIYZR4XRhWjYz6c09oR2gsneQtawK7IA1ToAFePrQUiwCsWPsCB82358dxcT2TmwMX38zGBF4GVOpLvtduGWBIv58S~h2eRha7VXuyV1uOHZqofrD9AgK5BybxfYRpZjoieTEGiO-9aBEWdgualhNVdeAHTNe1ShMEXIy4-8xAEBHPlBn5RuthWk7RHd9dFpsUnpKxUvmZQt4KNpS0nCu6knQ0XUyZIpy7EkZJsp-xe-7Av-Zx6si147nL-DNvrBTtUew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq1a.gif" /></span></div><div class="block-child-p">Let <em>s</em><sub><em>d</em></sub> (<em>d</em> = 1, 2, … , <em>n</em>) be the elementary symmetric polynomials in <em>z</em><sub>1</sub>, <em>z</em><sub>2</sub>, … , <em>z</em><sub><em>n</em></sub>, i.e. <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq2.gif?Expires=2147483647&amp;Signature=SJTfDNmK83ZsfxNjtNoFSytLdVHvVqvKaNRiNzdPyuEFroYPXJdDBJ17UWxiCeKwqhml9oq0vT9ZAgeYhl3DzrYO0V9WS1oD3MnXRbZHd7W13HkuaZ0pU3GkV-9GFVvMIeUlv1pLFKZsGJmi1gyb0YaeZbbRWtxLzXzjk-B0vRY~CVg~yGCXUSIKyDU74SB8cx0xjqtvoB89f4-SudL0-850kdsNH0p2bpOgK8pG-oxAgDP63BoUDydt20IVPbJGG532AU1sHWnbNaUeptlNFXLh00ItSx1v5X~LSXzmAejb-oPLQti8ATsgqV40RKEj0uv-myPEy2o9aXPzqymKyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq2.gif" />⁠</span>, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq3.gif?Expires=2147483647&amp;Signature=mP9idb8Fm15D2-ydyVaMplge7PQuizWRAFe0oS8Laaf4N5oe~cS6wxjJOaGVZ5JKL4lOl2X4nMIV9zeopWY7pa7V1d19J4L9iaA5cmMTHalKHjaKWcbJWAFytMYv1Z0Q6GHnkKAyaAqUriL68x5T18V2wIqOYDBRqFkwMuGD-vU1azzeG7vUMcmKi8fIpQT25XA7czg~CKM9lu2RycXFzVFK8Hut0zAirw0U~poQEMFlUK~r-qgfe4ssdy2iyg8V-YCELWGA82nwRk5loy6arKqF2al23eOYwKGYR2NfF-Y~QOe6r3y6Dj-CY7Le3sJdMlFiGZxc42B5VJnA3X87xQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq3.gif" />⁠</span>, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq4.gif?Expires=2147483647&amp;Signature=Ybidc~SiIWz-vHo2iaFiX8gMUHSf8u4kekoc4pJgvf0FkruqC4hggPi9rxW2XtuPLuUC9m8u5gMPzg3WD-7AXcPn4H8jjDCbOmOaqY1G7grrgudOs2SJ7krSjbJ8P0ofWkCOGe03vfH8zMzOe3Yk5gcMpI8GTsbGQff105lztQLb-xXm9usWiMdZF2KjvdWMzdJWwVk4tMBNbSyeh~ZXnkzzcdIjHiG3I4WGtB-MZiZVGjTcJttFp~4d6KbxdPc2yNv-Yyetmm2SBGYlkEKXqhQgf~yqIxXJdrWNVtxZlC18wZrbMYAwkSaOWxWSG8LC-8MKBKDlsZKG2-jyKeyoDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq4.gif" />⁠</span>, … , <em>s</em><sub><em>n</em></sub> = <em>z</em><sub>1</sub><em>z</em><sub>2</sub>···<em>z</em><sub><em>n</em></sub>, then <em>s</em><sub>1</sub> = −<em>a</em><sub><em>n</em></sub><sub>−1</sub>, <em>s</em><sub>2</sub> = <em>a</em><sub><em>n</em></sub><sub>−2</sub>, … , <em>s</em><sub><em>n</em></sub> = (−1)<sup><em>n</em></sup><em>a</em><sub>0</sub>. By using Newton's famous identities<sup><a reveal-id="DSQ008C11" data-open="DSQ008C11" class="link link-ref link-reveal xref-bibr">11</a></sup>: <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq5.gif?Expires=2147483647&amp;Signature=Mp972khDXm0TQtNKCFlxR9iPSBWNd6jIyneXvoIsWF8wMOUpu6fQ7ZxwCnruzp7DpEHrxea2r8UprmTZJXQZbxV8771CO-FnO22TgBAmNBrkD1ggzeRvpqwSOVhUqZblEO3PtEZqZNVNWKWwKhR6edPPtq9Vla00thcLRa~9mr7rwlnNHnu7-1zWQnaX2dqwPwq4VSZ6-akQOOhWtYV5G30qvfbpvm9AVuXTe4H0wdFj4Im6rLXPlBE901bBHFUBtsstdcPcviU3MMBalldm2TQxgWPzq974Vrj2J3nCFAbw9pSQOV41uNEKK8qq9F6b7OqcB1QGx5mWBolWNG8ZMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq5.gif" /></span><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq5a.gif?Expires=2147483647&amp;Signature=WVgErt2NcWeBX7DoTkBCOX5R2DoKJQe4I~aXRnWwhbGGaotiGkVk3W17wNPe~~wuc-aBL5lO1KUNdz9V~c6eLAKl16zdlf~rFdFmaJTlk5yA0~z-eRwoPpyJXQ~THvbSZ0fQlE~c8vyjPlyzxzZBmCHD7MmzhGGCTwmBALXvyKFjP-khC1R507HCEodLO9oi1pbSzEn2MaDpfx-f~6P~MjmdzVwAi4XmH2~W7ZeCToHEMSYUQii1FzQO0SXZo4T~U2Ma1XpLXXQzKEeg6ApkQL2figwEgwFuZnJ-Umgs3NLtn-roihSMsvDeye09v7tLBCwO6Myt11FNy8mBxDwydQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq5a.gif" /></span> where <em>d</em> = 1, 2, … , <em>n</em>, <em>a</em><sub><em>i</em></sub> can be obtained by δ<sub><em>j</em></sub> as shown below:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsq008ueq5.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ueq5.gif?Expires=2147483647&amp;Signature=gELdg35RN0rJ0rP4ix2sF-feZZmudEmaoNszRJM108FieWm1qMGKLMAB2chZSHEkmgh45OgTSogIMsJkDdOqJhhMBsVRb~sIsYxYv8fvcevkvBt7R5fLiV4pbT035InZ6838AxCVvdoDKsLOWop3n9AefOhSE1sUuandUl~WAdlRurFIg7wTG3PIEsUguEeeizqMaS7f4~47~19p1hGokDVYekSm3aTUsLnArTA~DCP-GnJ6btrFBxSsTfyYOwvR7r8K6XNZ9dwUoXexvMqMoLLsiB0z8X3KPHi8e2F-781RFXbg1OHUYCH1CGlP0NxiuH4rTUJU1Gzcvzoal7Dljg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq008ueq5.gif" /></a></div></div></div> As a result, the coefficients of the polynomial<span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq6.gif?Expires=2147483647&amp;Signature=Kp9YIgbf0R4~IztHzPgaXPl2Hi0h4N8h8Ufs4Kwdv5XoGkfuXWE~6Z0o92vfbFL~2JKq1HA31vzpS6v7eTNt5s-8fDi3ZFP58h9HGkPmFuWY145ofdqnGR14sYj7zRDa2iifh-kq7WEU6XfzgyIL~7QTM5cap20I5XZo0R8vP5AVDjz7Z9LB~hWqj-LFDsTt~B7jlekXJFWq5xNERjwZyw-oK~2ydZ4rlCx82dRncFXWK9dzr-p7kDapYxgc~MaYIU~ccIuMRIT0144ieiLTgayMPGavpCLAQnN6VZxaVoMKWrpQSK-3NvgIgWquMMS2-XrbJPjxJTKyEBbRY4lIdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq6.gif" /></span><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq008ileq6a.gif?Expires=2147483647&amp;Signature=D596fPn9hbAtj4L-EB8XONeqn3bD8Gma-oWepRDlgf5FaOy55WD4eyY4O1nKZ8rgPmqb8ZBR-jlA1HSYlmUky3DmzM8nVJabxN0dXxfW7dYs~st60yG9yZii8tHHC9pg1-ZeNO0gLROe~ZgZE9Ha8X6n9tsIm9FywM9m-tfRJJjPCPcNI8V8hnHcvoPV~v8LOX99L8IVc6E6jsUL4y7~LUEmV66iN3MruOYa4pjMMAlW0PnP-c2hpijshFjilsBgAzA7NKfLayxpAYapwnBDd3mKNmnCqU7LGJxBV1z7d47QEYUb05vzmkPa5P-O69bXLstfmrY~l5GrwS43EvmMcw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsq008ileq6a.gif" /></span> can be confirmed, and the set of all roots can be obtained. Next, we need to identify each root <em>z</em><sub>1</sub>,<em>z</em><sub>2</sub>, … , <em>z</em><sub><em>n</em></sub>.</div><p>Because the <em>y</em>-coordinate values of four nucleotides are between –1 and 1, <em>x</em><sub><em>i</em></sub> − <em>y</em><sub><em>i</em></sub> ≥ 0. As we have defined that the position of <em>k</em>th nucleotide on a graph is (<em>x</em><sub><em>k</em></sub>,<em>y</em><sub><em>k</em></sub>) or (<em>k</em>,<em>y</em><sub><em>k</em></sub>), the position of (<em>k</em> + 1)th nucleotide on a graph (<em>x</em><sub><em>k</em></sub><sub>+1</sub>, <em>y</em><sub><em>k</em></sub><sub>+1</sub>) can be represented as (<em>k</em> + 1, <em>y</em><sub><em>k</em></sub> + <em>u</em><sub><em>k</em></sub><sub>+1</sub>), where <em>u</em><sub><em>k</em></sub><sub>+1</sub> may be any of <em>y</em>-coordinate value of the four nucleotides. Thus, <em>z</em><sub><em>k</em></sub><sub>+1</sub> = <em>x</em><sub><em>k</em></sub><sub>+1</sub> − <em>y</em><sub><em>k</em></sub><sub>+1</sub> = (<em>k</em> + 1) − (<em>y</em><sub><em>k</em></sub> + <em>u</em><sub><em>k</em></sub><sub>+1</sub>) = (<em>k</em> − <em>y</em><sub><em>k</em></sub>) + (1 − <em>u</em><sub><em>k</em></sub><sub>+1</sub>) <em>≥</em>(<em>k</em> − <em>y</em><sub><em>k</em></sub>). Because <em>z</em><sub><em>k</em></sub> = <em>x</em><sub><em>k</em></sub> − <em>y</em><sub><em>k</em></sub> = <em>k</em> − <em>y</em><sub><em>k</em></sub>, <em>z</em><sub><em>k</em></sub><sub>+1</sub><em>≥ z</em><sub><em>k</em></sub>. As a consequence, <em>z</em><sub><em>i</em></sub> is increasing and each root can be identified by this property, which means each value of <em>y</em><sub><em>i</em></sub> can be obtained. With all (<em>x</em><sub><em>i</em></sub>,<em>y</em><sub><em>i</em></sub>), a DNA graph can be recovered.</p><p>Therefore, we have successfully proved that the correspondence between a DNA sequence and its moment vector obtained from its sequence graph is one-to-one.</p></div>                            <h3 scrollto-destination=5037218 id="5037218" class="section-title" data-legacy-id=s2c>2.3. Construction of genome space</h3>
<p>We have already obtained a good numerical characterization, moment vector, to represent a DNA sequence. Now, we will use this tool to construct a genome space. Here, we emphasize that the structure of genomes is complicated. It may be single-stranded or double-stranded and in a linear or circular structure. Thus, we should consider the different structures when constructing the genome space.</p><p>For the simplest genome structures, linear single-strand forms, we can treat them as linear DNA sequences. That is, every genome corresponds to a general DNA sequence. Thus, we can utilize our moment vector to construct the genome space. In order to use whole genome information to make comparative analysis among genomes, we can use the first <em>N</em> components (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>N</em></sub>) of the moment vector of a genome sequence graph to represent a genome as a point in <em>N</em>-dimensional space. Thus, we obtain an <em>N</em>-dimensional genome space as a subspace in <em>R</em><sup><em>N</em></sup>. Using the Euclidean distance between two points as an index for comparison, we can perform phylogenetic and clustering analysis for genome sequences in this genome space.</p><p>For the circular single-strand genomes, the construction of genome space is more complicated because we do not know which point is the start point in this circular DNA sequence. In this case, we treat every point as the start point in this circular sequence of length <em>n</em>, and then we get <em>n</em> linear single-strand genomes. For every linear single-strand genome sequence, we can compute its <em>n</em>-dimensional moment vector. Then, we take average by <em>n</em> for these <em>n n</em>-dimensional moment vectors to get a normalized moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>n</em></sub>). For circular single-strand genomes, we use the first <em>N</em> components (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>N</em></sub>) of this normalized moment vector to represent a genome as a point in an <em>N</em>-dimensional space. Thus, we obtain an <em>N</em>-dimensional genome space as a subspace in <em>R</em><sup><em>N</em></sup>.</p><p>For the double-stranded genomes, we need to point out that the moment vector of reverse complementary sequence is not the same as the original sequence. Generally, when meeting the double-stranded genomes, we treat them as two single-stranded genomes. We use the above method (linear or circular) to get two <em>n</em>-dimensional moment vectors for these two single-stranded sequences, and then take average to get a general moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>n</em></sub>). By using the first <em>N</em> components (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>N</em></sub>) of this general moment vector to represent a genome as a point in <em>N</em>-dimensional space. Thus, we obtain an <em>N</em>-dimensional genome space as a subspace in <em>R</em><sup><em>N</em></sup>. Here, we need to point out that the two strands of some genomes (e.g. mitochondrial genomes, some bacterial genomes) are differentiated by their nucleotide content, which are called the heavy strand and the light strand, respectively. The two strands have different masses because one has a higher proportion of heavier nucleic acids and its complement a lower proportion. In this case, we just treat them as the single-stranded (by using the heavy strand) genomes to make the genome space.</p>                            <h2 scrollto-destination=5037230 id="5037230" class="section-title" data-legacy-id=s3>3. Results</h2>
<p>To verify that the biological distance obtained in this way truly incorporates biological utility, we apply our new genome space to the phylogenetic analysis of organisms. Most existing methods for phylogenetic inference using biological sequences can be divided into two groups. The algorithms in the first group utilize various distance measures<sup><a reveal-id="DSQ008C12  DSQ008C15" data-open="DSQ008C12  DSQ008C15" class="link link-ref link-reveal xref-bibr">12–15</a></sup> which are based on different models of nucleotide substitution or amino acid replacement, and then transform the distance matrix into a tree. In the second group of approaches, instead of building a tree, the tree that can best explain the observed sequences under the evolutionary assumption is found by evaluating of different topologies. This category includes parsimony<sup><a reveal-id="DSQ008C16  DSQ008C18" data-open="DSQ008C16  DSQ008C18" class="link link-ref link-reveal xref-bibr">16–18</a></sup> and maximum likelihood methods.<sup><a reveal-id="DSQ008C19  DSQ008C21" data-open="DSQ008C19  DSQ008C21" class="link link-ref link-reveal xref-bibr">19–21</a></sup> All these methods require a multiple alignment of the sequences and assume some sort of evolutionary model, which require human intervention. Thus, the results are usually controversial. However, our genome space does not need sequence alignment and any evolutionary model. It is totally automatically generated and avoids computation repetition.</p><p>First, we consider the phylogeny of mammals. Mitochondrial DNA is not highly conserved and has a rapid mutation rate, thus it is very useful for studying the evolutionary relationships of organisms.<sup><a reveal-id="DSQ008C22" data-open="DSQ008C22" class="link link-ref link-reveal xref-bibr">22</a></sup> We extracted 35 complete mammalian mitochondrial genome sequences from the GenBank, each of which has length of more than 16 000 nucleotides. Moreover, they have double-stranded and circular structures. As mentioned in the previous section, because we have already known the gene content of both strands of these genomes, we just treat them as the single-stranded (by using the heavy strand) circular genomes. For this case, we treat every point as the start point in this circular sequence of length <em>n</em>, and then we get <em>n</em> linear single-strand genomes. For every linear single-strand genome sequence, by using the nucleotide vector system shown in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>, we can compute its <em>n</em>-dimensional moment vector. Then, we take average by <em>n</em> for these <em>n n</em>-dimensional moment vectors to get a normalized moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>n</em></sub>). Here, we use the first 60 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>60</sub>) to characterize these 35 genome graphical curves and obtained 35 points in 60-dimensional genome space. By computing the Euclidean distances between these points, we got the distance matrix for these 35 organisms. The phylogenetic tree for them shown in Fig. <a reveal-id="DSQ008F3" data-open="DSQ008F3" class="link link-reveal link-table xref-fig">3</a> is generated using UPGMA program in the MEGA 4 package.<sup><a reveal-id="DSQ008C23" data-open="DSQ008C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The last 10 mammals are grouped into a cluster because they are primates, and the phylogenetic relationship among them coincides with those found by Raina <em>et al</em>.<sup><a reveal-id="DSQ008C24" data-open="DSQ008C24" class="link link-ref link-reveal xref-bibr">24</a></sup> We also found that Norway rat, vole, and squirrel are grouped into a cluster for the reason that they are rodent species.</p>                        <a id="5037238" scrollto-destination="5037238"></a>
<div id="DSQ008F3" data-id="DSQ008F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00803.jpeg?Expires=2147483647&Signature=QY~7iDmktNhrZ8brxIMB8~e3NGZtpvpf~3fgbLShK6tkIxJ2XJ6mPAjqZaQl9YKYfq5hJTtW8vBbZZo3-vWLd3-f9Z7VKS1hhjguP30hHefkW8orO1eAvJKGBt3xchNFl8H5OmWtoyDitw-eiepmlfkU~fgAAAAZW5poUZfNAEYDfPj2Y6rkGKYmcQwV2Oi-n-O1h6LOZFCZFseNtIIa-ZfcpNdewmrIEHOV4OGNojubWhPZo6ZTTCk74WnB9gnmbl7kmo8YPLldlnZjis9yenikSfYLm546XVQStzDlqV1WADxsu2B8Q9NRX5rQKSQqfolkOoIowx8BaO6YH79n-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of the mitochondrial genome sequences of 35 mammal species. This tree was reconstructed by the first 60 components of the moment vector (M1, M2, … , M60). The accession numbers of these 35 species in the GenBank are as follows: human, V00662; pigmy chimpanzee, D38116; common chimpanzee, D38113; gorilla, D38114; gibbon, X99256; baboon, Y18001; vervet monkey, AY863426; ape, NC_002764; Bornean orangutan, D38115; Sumatran orangutan, NC_002083; cat, U20753; dog, U96639; pig, AJ002189; sheep, AF010406; goat, AF533441; cow, V00654; buffalo, AY488491; wolf, EU442884; tiger, EF551003; leopard, EF551002; Indian rhinoceros, X97336; white rhinoceros, Y07726; harbor seal, X63726; gray seal, X72004; African elephant, AJ224821; Asiatic elephant, DQ316068; black bear, DQ402478; brown bear, AF303110; polar bear, AF303111; giant panda, EF212882; rabbit, AJ001588; hedgehog, X88898; Norway rat, X14848; vole, AF348082; squirrel, AJ238588." path-from-xml="dsq00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037238/dsq00803.jpeg" path-from-xml="dsq00803.jpeg" target="_blank">View large</a><a section="5037238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00803.jpeg?Expires=2147483647&Signature=zBdQ8DQQxqghwtYw5gwb0FF1dALOhqXHGw6W5sjUbkKUOJYAL1OAuqfz3qrn7j-LVFJ4g7EmFp7Gz0fPPGSpgG8yYFQXDlYnfJYgvK8lnzette2tPaSRr9mkmXaa5cz85D2geyUJ8pCoFwXOZ6bxmfLvnNMhhDuS57Pslwi53PmNxTfi-zzhbxyH92brrTbkvwaY7M5epUvUYlIxPQ~WyAj-cKr--oQ0Vr9BU7PlLMwazgF-Ig9Ku1knWDyRfovRtxWFT5U72j9OgBewlo6rfpNUANsoBO6CA61HRuXYZl5-Vhq6XcPhbjTSPgzsDjkyXRycnxsXdV3b6RH5PbUFag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037238&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of the mitochondrial genome sequences of 35 mammal species. This tree was reconstructed by the first 60 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>60</sub>). The accession numbers of these 35 species in the GenBank are as follows: human, V00662; pigmy chimpanzee, D38116; common chimpanzee, D38113; gorilla, D38114; gibbon, X99256; baboon, Y18001; vervet monkey, AY863426; ape, NC_002764; Bornean orangutan, D38115; Sumatran orangutan, NC_002083; cat, U20753; dog, U96639; pig, AJ002189; sheep, AF010406; goat, AF533441; cow, V00654; buffalo, AY488491; wolf, EU442884; tiger, EF551003; leopard, EF551002; Indian rhinoceros, X97336; white rhinoceros, Y07726; harbor seal, X63726; gray seal, X72004; African elephant, AJ224821; Asiatic elephant, DQ316068; black bear, DQ402478; brown bear, AF303110; polar bear, AF303111; giant panda, EF212882; rabbit, AJ001588; hedgehog, X88898; Norway rat, X14848; vole, AF348082; squirrel, AJ238588.</p></div></div><div content-id="DSQ008F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00803.jpeg?Expires=2147483647&Signature=QY~7iDmktNhrZ8brxIMB8~e3NGZtpvpf~3fgbLShK6tkIxJ2XJ6mPAjqZaQl9YKYfq5hJTtW8vBbZZo3-vWLd3-f9Z7VKS1hhjguP30hHefkW8orO1eAvJKGBt3xchNFl8H5OmWtoyDitw-eiepmlfkU~fgAAAAZW5poUZfNAEYDfPj2Y6rkGKYmcQwV2Oi-n-O1h6LOZFCZFseNtIIa-ZfcpNdewmrIEHOV4OGNojubWhPZo6ZTTCk74WnB9gnmbl7kmo8YPLldlnZjis9yenikSfYLm546XVQStzDlqV1WADxsu2B8Q9NRX5rQKSQqfolkOoIowx8BaO6YH79n-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of the mitochondrial genome sequences of 35 mammal species. This tree was reconstructed by the first 60 components of the moment vector (M1, M2, … , M60). The accession numbers of these 35 species in the GenBank are as follows: human, V00662; pigmy chimpanzee, D38116; common chimpanzee, D38113; gorilla, D38114; gibbon, X99256; baboon, Y18001; vervet monkey, AY863426; ape, NC_002764; Bornean orangutan, D38115; Sumatran orangutan, NC_002083; cat, U20753; dog, U96639; pig, AJ002189; sheep, AF010406; goat, AF533441; cow, V00654; buffalo, AY488491; wolf, EU442884; tiger, EF551003; leopard, EF551002; Indian rhinoceros, X97336; white rhinoceros, Y07726; harbor seal, X63726; gray seal, X72004; African elephant, AJ224821; Asiatic elephant, DQ316068; black bear, DQ402478; brown bear, AF303110; polar bear, AF303111; giant panda, EF212882; rabbit, AJ001588; hedgehog, X88898; Norway rat, X14848; vole, AF348082; squirrel, AJ238588." path-from-xml="dsq00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037238/dsq00803.jpeg" path-from-xml="dsq00803.jpeg" target="_blank">View large</a><a section="5037238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00803.jpeg?Expires=2147483647&Signature=zBdQ8DQQxqghwtYw5gwb0FF1dALOhqXHGw6W5sjUbkKUOJYAL1OAuqfz3qrn7j-LVFJ4g7EmFp7Gz0fPPGSpgG8yYFQXDlYnfJYgvK8lnzette2tPaSRr9mkmXaa5cz85D2geyUJ8pCoFwXOZ6bxmfLvnNMhhDuS57Pslwi53PmNxTfi-zzhbxyH92brrTbkvwaY7M5epUvUYlIxPQ~WyAj-cKr--oQ0Vr9BU7PlLMwazgF-Ig9Ku1knWDyRfovRtxWFT5U72j9OgBewlo6rfpNUANsoBO6CA61HRuXYZl5-Vhq6XcPhbjTSPgzsDjkyXRycnxsXdV3b6RH5PbUFag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037238&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of the mitochondrial genome sequences of 35 mammal species. This tree was reconstructed by the first 60 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>60</sub>). The accession numbers of these 35 species in the GenBank are as follows: human, V00662; pigmy chimpanzee, D38116; common chimpanzee, D38113; gorilla, D38114; gibbon, X99256; baboon, Y18001; vervet monkey, AY863426; ape, NC_002764; Bornean orangutan, D38115; Sumatran orangutan, NC_002083; cat, U20753; dog, U96639; pig, AJ002189; sheep, AF010406; goat, AF533441; cow, V00654; buffalo, AY488491; wolf, EU442884; tiger, EF551003; leopard, EF551002; Indian rhinoceros, X97336; white rhinoceros, Y07726; harbor seal, X63726; gray seal, X72004; African elephant, AJ224821; Asiatic elephant, DQ316068; black bear, DQ402478; brown bear, AF303110; polar bear, AF303111; giant panda, EF212882; rabbit, AJ001588; hedgehog, X88898; Norway rat, X14848; vole, AF348082; squirrel, AJ238588.</p></div></div><p>In order to further illustrate the efficiency of our genome space, we then focus on the origins and evolution of human immunodeficiency virus (HIV). HIV is a lentivirus that can lead to acquired immune deficiency syndrome (AIDS), a condition in humans in which the immune system begins to fail, leading to life-threatening opportunistic infections. To develop the anti-HIV drugs and vaccines, the research into the origins and evolution of this virus has become very important. Rambaut <em>et al</em>.<sup><a reveal-id="DSQ008C25" data-open="DSQ008C25" class="link link-ref link-reveal xref-bibr">25</a></sup> reconstructed the phylogenetic tree of the primate lentiviruses including HIV-1, HIV-2, and the simian immunodeficiency viruses (SIVs). It was discovered that the two human viruses are related to different SIVs and therefore have different evolutionary origins. However, in Rambaut <em>et al</em>.'s paper, the tree was constructed using the maximum likelihood method on an alignment of the nucleotide sequences of polymerase gene in these lentiviruses. Generally, a single-gene sequence does not possess enough information to construct an evolutionary history of organisms, thus we use our new method to reconstruct the phylogenetic tree based on the whole genome sequences.</p><p>The genomes of lentiviruses are single-stranded linear RNA. RNA has the base uracil (U) rather than thymine (T) that is present in DNA. In fact, these RNA genome sequences downloaded from GenBank have already been transformed into DNA sequences (change U by T). Thus, we treat them as linear DNA sequences. Using the nucleotide vector system shown in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>, the sequence graphs of the genomes of the 33 lentiviruses were obtained. Here, we use the first 12 components of the moment vector to characterize these 33 genome graphical curves, and thus we obtained 33 twelve-dimensional vectors. These 33 vectors can be viewed as 33 points in a 12-dimensional genome space. By computing the Euclidean distance between these points, we reconstructed the phylogenetic tree of these primate lentiviruses (Fig. <a reveal-id="DSQ008F4" data-open="DSQ008F4" class="link link-reveal link-table xref-fig">4</a>) using UPGMA program in the MEGA 4 package.<sup><a reveal-id="DSQ008C23" data-open="DSQ008C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The figure illustrates that both the HIV-1 and HIV-2 lineages fall within that of the SIVs which are isolated from other primates, thus they represent the independent cross-species transmission events. In agreement with Rambaut <em>et al</em>.'s result, our phylogenetic tree shows that HIV-1 group N is a new HIV-1 type between the HIV-1 groups M and O (somewhat closer to the M group). It also indicates that HIV-1 groups M, N, and O are closely related to SIVs from chimpanzees (<em>Pan troglodytes troglodytes</em> and <em>Pan troglodytes schweinfurthii</em>) because there is a mixing of the HIV-1 and SIV lineages. Moreover, our result suggests that <em>P. t. troglodytes</em> (SIV<sub>cpz2</sub> and SIV<sub>cpz3</sub> in Fig. <a reveal-id="DSQ008F4" data-open="DSQ008F4" class="link link-reveal link-table xref-fig">4</a>) is the primary reservoir for HIV-1 group O, which is not clear in Rambaut <em>et al</em>.'s evolutionary tree. In Rambaut <em>et al</em>.'s evolutionary tree, the position of HIV-1 group O is just between <em>P. t. troglodytes</em> and <em>P. t. schweinfurthii</em>, but not closer to either of them. Our suggestion coincides with those found by Gao <em>et al</em>.<sup><a reveal-id="DSQ008C26" data-open="DSQ008C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In Gao <em>et al</em>.'s work, the authors concluded that the HIV-1 group O is closely related to the just one of these SIVcpz lineages, found in <em>P. t. troglodytes</em>. For HIV-2, we find that the SIVs that are closely related to HIV-2 are not only SIVsm (sooty mangabey monkey) and SIVmac (macaque) as shown by Rambaut <em>et al</em>.'s tree, but also SIVsun (sun-tailed monkey).</p>                        <a id="5037247" scrollto-destination="5037247"></a>
<div id="DSQ008F4" data-id="DSQ008F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00804.jpeg?Expires=2147483647&Signature=GPm~BpsrM4KE13ivSZGJJtlpTxrWyRZiobdVZH1x4fjiPT9XBvAVJvsXRwem7f2ECdZTUFqVpRyDP8POmYy-HeNaWdgxYNkdRFthrFMAeJEpHGzaZsa6JH3beSJuImQVCDFGNse~uAhZvmmI6MtFLb2KTK1dvDTTJWoDnhLgbcueLhb9XsDPZ1-2t1qdap8fsUNigoM6koGWVANaKxvo7Z0djOPhA2KdZWmz7X71KQkmCJnGRR9HxeHUeCMFn3EH3xmB-dp2sLk7IqJgZ4hiGC8crw-5waBJqmMTjz-1YkAeUsUzk5CDRCQ7DlYdjnEfrh7lEyxpeHQ-gTVs3uikoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolutionary tree of the primate lentiviruses. This tree was reconstructed by using the first 12 components of the moment vector (M1, M2, … , M12). The subtypes of HIV-1 and HIV-2 are shown. The virus abbreviations, their primate hosts and genome accession numbers in the GenBank are as follows: HIV-1, group M: A, AF004885; B, A04321; C, AF443079; D, K03454; F, AY173957; G, AY772535; H, AF190127; group N: DQ017382; group O: a, AY169802; b, AY169803; HIV-2: A1, AF082339; A2, M30502; B1, L07625; B2, X61240; SIVcpz1-3, chimpanzee (Pan troglodytes troglodytes): AY169968, AJ271369, DQ373063; SIVcpz4-5, chimpanzee (Pan troglodytes schweinfurthii): DQ374657, DQ374658; SIVdrl, drill: AY159321; SIVgsn, greater spot-nosed monkey: AF468659; SIVlhoest, L'Hoest monkey: AF188114; SIVmac, macaque: D01065; SIVmnd1-2, mandrill: M27470, AY159322; SIVmon, Campbell's mona monkey: AY340701; SIVrcm, red-capped monkey: AF382829; SIVsab, Sabaeus monkey: U04005; SIVsm, sooty mangabey monkey: U72748; SIVsun, sun-tailed monkey: AF131870; SIVsyk, Sykes' monkey: L06042; SIVtan, tantalus monkey: U58991; SIVver, vervet monkey: M29975." path-from-xml="dsq00804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037247/dsq00804.jpeg" path-from-xml="dsq00804.jpeg" target="_blank">View large</a><a section="5037247" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00804.jpeg?Expires=2147483647&Signature=RjpVbETwRW9mqo~WbCkA7cDRP8O7hkv4P-b-tVKOnC3x3VLtfb-1ZXJXjVXEg4CJ-vfS9WLpAx-OXYdJRhUFOs8RzU2SGPgnI-OgluZtSC2ypl1HncQh8uya8bql3mx8ojMyuB~gOK1Qb5EyM71~XBFEVkitw4U5qElyzSpUxMLUjv3eYTMOMmPcg2jZ87z3L6sRVXjd0ukXSwmyQgtqA-BKmNhKdtOaXpA-URsk8beGJb3Dd1~AvgXxs7YNwWAspW~GEYE86DiIUgcat3tm59uAKHPAhTb-L4Z~2nExw8QSI7hnCzJm1GYFV5BHZvfrFgA2vWyyiEaO9fI2Ya-11A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037247&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary tree of the primate lentiviruses. This tree was reconstructed by using the first 12 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>12</sub>). The subtypes of HIV-1 and HIV-2 are shown. The virus abbreviations, their primate hosts and genome accession numbers in the GenBank are as follows: HIV-1, group M: A, AF004885; B, A04321; C, AF443079; D, K03454; F, AY173957; G, AY772535; H, AF190127; group N: DQ017382; group O: a, AY169802; b, AY169803; HIV-2: A1, AF082339; A2, M30502; B1, L07625; B2, X61240; SIVcpz1-3, chimpanzee (<em>Pan troglodytes troglodytes</em>): AY169968, AJ271369, DQ373063; SIVcpz4-5, chimpanzee (<em>Pan troglodytes schweinfurthii</em>): DQ374657, DQ374658; SIVdrl, drill: AY159321; SIVgsn, greater spot-nosed monkey: AF468659; SIVlhoest, L'Hoest monkey: AF188114; SIVmac, macaque: D01065; SIVmnd1-2, mandrill: M27470, AY159322; SIVmon, Campbell's mona monkey: AY340701; SIVrcm, red-capped monkey: AF382829; SIVsab, Sabaeus monkey: U04005; SIVsm, sooty mangabey monkey: U72748; SIVsun, sun-tailed monkey: AF131870; SIVsyk, Sykes' monkey: L06042; SIVtan, tantalus monkey: U58991; SIVver, vervet monkey: M29975.</p></div></div><div content-id="DSQ008F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00804.jpeg?Expires=2147483647&Signature=GPm~BpsrM4KE13ivSZGJJtlpTxrWyRZiobdVZH1x4fjiPT9XBvAVJvsXRwem7f2ECdZTUFqVpRyDP8POmYy-HeNaWdgxYNkdRFthrFMAeJEpHGzaZsa6JH3beSJuImQVCDFGNse~uAhZvmmI6MtFLb2KTK1dvDTTJWoDnhLgbcueLhb9XsDPZ1-2t1qdap8fsUNigoM6koGWVANaKxvo7Z0djOPhA2KdZWmz7X71KQkmCJnGRR9HxeHUeCMFn3EH3xmB-dp2sLk7IqJgZ4hiGC8crw-5waBJqmMTjz-1YkAeUsUzk5CDRCQ7DlYdjnEfrh7lEyxpeHQ-gTVs3uikoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolutionary tree of the primate lentiviruses. This tree was reconstructed by using the first 12 components of the moment vector (M1, M2, … , M12). The subtypes of HIV-1 and HIV-2 are shown. The virus abbreviations, their primate hosts and genome accession numbers in the GenBank are as follows: HIV-1, group M: A, AF004885; B, A04321; C, AF443079; D, K03454; F, AY173957; G, AY772535; H, AF190127; group N: DQ017382; group O: a, AY169802; b, AY169803; HIV-2: A1, AF082339; A2, M30502; B1, L07625; B2, X61240; SIVcpz1-3, chimpanzee (Pan troglodytes troglodytes): AY169968, AJ271369, DQ373063; SIVcpz4-5, chimpanzee (Pan troglodytes schweinfurthii): DQ374657, DQ374658; SIVdrl, drill: AY159321; SIVgsn, greater spot-nosed monkey: AF468659; SIVlhoest, L'Hoest monkey: AF188114; SIVmac, macaque: D01065; SIVmnd1-2, mandrill: M27470, AY159322; SIVmon, Campbell's mona monkey: AY340701; SIVrcm, red-capped monkey: AF382829; SIVsab, Sabaeus monkey: U04005; SIVsm, sooty mangabey monkey: U72748; SIVsun, sun-tailed monkey: AF131870; SIVsyk, Sykes' monkey: L06042; SIVtan, tantalus monkey: U58991; SIVver, vervet monkey: M29975." path-from-xml="dsq00804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037247/dsq00804.jpeg" path-from-xml="dsq00804.jpeg" target="_blank">View large</a><a section="5037247" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00804.jpeg?Expires=2147483647&Signature=RjpVbETwRW9mqo~WbCkA7cDRP8O7hkv4P-b-tVKOnC3x3VLtfb-1ZXJXjVXEg4CJ-vfS9WLpAx-OXYdJRhUFOs8RzU2SGPgnI-OgluZtSC2ypl1HncQh8uya8bql3mx8ojMyuB~gOK1Qb5EyM71~XBFEVkitw4U5qElyzSpUxMLUjv3eYTMOMmPcg2jZ87z3L6sRVXjd0ukXSwmyQgtqA-BKmNhKdtOaXpA-URsk8beGJb3Dd1~AvgXxs7YNwWAspW~GEYE86DiIUgcat3tm59uAKHPAhTb-L4Z~2nExw8QSI7hnCzJm1GYFV5BHZvfrFgA2vWyyiEaO9fI2Ya-11A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037247&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary tree of the primate lentiviruses. This tree was reconstructed by using the first 12 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>12</sub>). The subtypes of HIV-1 and HIV-2 are shown. The virus abbreviations, their primate hosts and genome accession numbers in the GenBank are as follows: HIV-1, group M: A, AF004885; B, A04321; C, AF443079; D, K03454; F, AY173957; G, AY772535; H, AF190127; group N: DQ017382; group O: a, AY169802; b, AY169803; HIV-2: A1, AF082339; A2, M30502; B1, L07625; B2, X61240; SIVcpz1-3, chimpanzee (<em>Pan troglodytes troglodytes</em>): AY169968, AJ271369, DQ373063; SIVcpz4-5, chimpanzee (<em>Pan troglodytes schweinfurthii</em>): DQ374657, DQ374658; SIVdrl, drill: AY159321; SIVgsn, greater spot-nosed monkey: AF468659; SIVlhoest, L'Hoest monkey: AF188114; SIVmac, macaque: D01065; SIVmnd1-2, mandrill: M27470, AY159322; SIVmon, Campbell's mona monkey: AY340701; SIVrcm, red-capped monkey: AF382829; SIVsab, Sabaeus monkey: U04005; SIVsm, sooty mangabey monkey: U72748; SIVsun, sun-tailed monkey: AF131870; SIVsyk, Sykes' monkey: L06042; SIVtan, tantalus monkey: U58991; SIVver, vervet monkey: M29975.</p></div></div><p>In addition, we apply our genome space to another field of virology: the taxonomy of coronavirus. To study the classification and phylogeny of coronaviruses clearly, we apply our genome space to a large set of 30 complete coronavirus genomes from GenBank, including the two newly sequenced human coronaviruses, HCoV-NL63 and HCoV-HKU1, along with four genomes from <em>Flaviviridae</em> and <em>Togaviridae</em> which are not coronaviruses (outgroups). The coronavirus genomes are also single-stranded linear RNA. So, similar to the above lentivirus case, we treat these coronavirus genomes as linear DNA sequences. Their abbreviation, accession number, description, and classification are shown in Table <a reveal-id="DSQ008TB1" data-open="DSQ008TB1" class="link link-reveal link-table xref-fig">1</a>. First, we used our two-dimensional genome space (actually, it is a two-dimensional plane with the first two moments <em>M</em><sub>1</sub> and <em>M</em><sub>2</sub> being <em>x</em>-axis and <em>y</em>-axis) to characterize these ‘34’ virus genomes and calculated ‘34’ points in Fig. <a reveal-id="DSQ008F5" data-open="DSQ008F5" class="link link-reveal link-table xref-fig">5</a>A. Four groups, group 1, group 2, group 3, and outgroups, can be seen in this figure as four distinct clusters. To study the classification for coronavirus clearly, we expanded it and obtained Fig. <a reveal-id="DSQ008F5" data-open="DSQ008F5" class="link link-reveal link-table xref-fig">5</a>B. </p>                        <a id="5037254" scrollto-destination="5037254"></a>
<div content-id="DSQ008TB1" class="table-modal"><div class="table-wrap"><div id="DSQ008TB1" class="table-wrap-title" data-id="DSQ008TB1"><div class="label -label">Table 1</div><div class="caption"><p>Thirty coronavirus genomes and four outgroup genomes from the GenBank</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession No. </th><th>Abbreviation </th><th>Length (nt) </th><th>Description/Classification </th></tr></thead><tbody><tr><td>AF304460 </td><td>HCoV-229E </td><td>27 317 </td><td>Human coronavirus 229E, complete genome/Group 1 </td></tr><tr><td>AY391777 </td><td>HCoV-OC43 </td><td>30 738 </td><td>Human coronavirus OC43, complete genome/Group 2 </td></tr><tr><td>AF353511 </td><td>PEDV </td><td>28 033 </td><td>Porcine epidemic diarrhea virus strain, complete genome/Group 1 </td></tr><tr><td>U00735 </td><td>BCoVM </td><td>31 032 </td><td>Bovine coronavirus strain Mebus, complete genome/Group 2 </td></tr><tr><td>AF391542 </td><td>BCoVL </td><td>31 028 </td><td>Bovine coronavirus isolate BCoV-LUN, complete genome/Group 2 </td></tr><tr><td>AF220295 </td><td>BCoVQ </td><td>31 100 </td><td>Bovine coronavirus strain Quebec, complete genome/Group 2 </td></tr><tr><td>NC_003045 </td><td>BCoV </td><td>31 028 </td><td>Bovine coronavirus, complete genome/Group 2 </td></tr><tr><td>AF208067 </td><td>MHVM </td><td>31 233 </td><td>Murine hepatitis virus strain ML-10, complete genome/Group 2 </td></tr><tr><td>AF201929 </td><td>MHV2 </td><td>31 276 </td><td>Murine hepatitis virus strain 2, complete genome/Group 2 </td></tr><tr><td>AF208066 </td><td>MHVP </td><td>31 112 </td><td>Murine hepatitis virus strain Penn 97-1, complete genome/Group 2 </td></tr><tr><td>NC_001846 </td><td>MHV </td><td>31 357 </td><td>Murine hepatitis virus, complete genome/Group 2 </td></tr><tr><td>NC_001451 </td><td>IBV </td><td>27 608 </td><td>Avian infectious bronchitis virus, complete genome/Group 3 </td></tr><tr><td>EU095850 </td><td>TCoV </td><td>27 657 </td><td>Turkey coronavirus isolate MG10, complete genome/Group 3 </td></tr><tr><td>AY278488 </td><td>BJ01 </td><td>29 725 </td><td>SARS coronavirus BJ01, complete genome/Group 4 </td></tr><tr><td>AY278741 </td><td>Urbani </td><td>29 727 </td><td>SARS coronavirus Urbani, complete genome/Group 4 </td></tr><tr><td>AY278491 </td><td>HKU-39849 </td><td>29 742 </td><td>SARS coronavirus HKU-39849, complete genome/Group 4 </td></tr><tr><td>AY278554 </td><td>CUHK-W1 </td><td>29 736 </td><td>SARS coronavirus CUHK-W1, complete genome/Group 4 </td></tr><tr><td>AY282752 </td><td>CUHK-Su10 </td><td>29 736 </td><td>SARS coronavirus CUHK-Su10, complete genome/Group 4 </td></tr><tr><td>AY283794 </td><td>SIN2500 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2500, complete genome/Group 4 </td></tr><tr><td>AY283795 </td><td>SIN2677 </td><td>29 705 </td><td>SARS coronavirus isolate SIN2677, complete genome/Group 4 </td></tr><tr><td>AY283796 </td><td>SIN2679 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2679, complete genome/Group 4 </td></tr><tr><td>AY283797 </td><td>SIN2748 </td><td>29 706 </td><td>SARS coronavirus isolate SIN2748, complete genome/Group 4 </td></tr><tr><td>AY283798 </td><td>SIN2774 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2774, complete genome/Group 4 </td></tr><tr><td>AY291451 </td><td>TW1 </td><td>29 729 </td><td>SARS coronavirus TW1, complete genome/Group 4 </td></tr><tr><td>NC_004718 </td><td>TOR2 </td><td>29 751 </td><td>SARS coronavirus TOR2, complete genome/Group 4 </td></tr><tr><td>AY297028 </td><td>ZJ01 </td><td>29 715 </td><td>SARS coronavirus ZJ01, complete genome/Group 4 </td></tr><tr><td>AY572034 </td><td>Civet007 </td><td>29 540 </td><td>SARS coronavirus civet007, complete genome/Group 4 </td></tr><tr><td>AY572035 </td><td>Civet010 </td><td>29 518 </td><td>SARS coronavirus civet010, complete genome/Group 4 </td></tr><tr><td>NC_005831 </td><td>HCoV-NL63 </td><td>27 553 </td><td>Human coronavirus NL63, complete genome/Group 1 </td></tr><tr><td>NC_006577 </td><td>HCoV-HKU1 </td><td>29 926 </td><td>Human coronavirus HKU1, complete genome/Group 5 </td></tr><tr><td>NC_001564 </td><td>CellF </td><td>10 695 </td><td>Cell fusing agent virus, complete genome/<em>Flaviviridae</em> (outgroup) </td></tr><tr><td>NC_004102 </td><td>HepaCF </td><td>9646 </td><td>Hepatitis C virus, complete genome/<em>Flaviviridae</em> (outgroup) </td></tr><tr><td>NC_001512 </td><td>NyongT </td><td>11 835 </td><td>O'nyong-nyong virus, complete genome/<em>Togaviridae</em> (outgroup) </td></tr><tr><td>NC_001544 </td><td>RossT </td><td>11 657 </td><td>Ross River virus, complete genome/<em>Togaviridae</em> (outgroup) </td></tr></tbody></table></div><div content-id="DSQ008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession No. </th><th>Abbreviation </th><th>Length (nt) </th><th>Description/Classification </th></tr></thead><tbody><tr><td>AF304460 </td><td>HCoV-229E </td><td>27 317 </td><td>Human coronavirus 229E, complete genome/Group 1 </td></tr><tr><td>AY391777 </td><td>HCoV-OC43 </td><td>30 738 </td><td>Human coronavirus OC43, complete genome/Group 2 </td></tr><tr><td>AF353511 </td><td>PEDV </td><td>28 033 </td><td>Porcine epidemic diarrhea virus strain, complete genome/Group 1 </td></tr><tr><td>U00735 </td><td>BCoVM </td><td>31 032 </td><td>Bovine coronavirus strain Mebus, complete genome/Group 2 </td></tr><tr><td>AF391542 </td><td>BCoVL </td><td>31 028 </td><td>Bovine coronavirus isolate BCoV-LUN, complete genome/Group 2 </td></tr><tr><td>AF220295 </td><td>BCoVQ </td><td>31 100 </td><td>Bovine coronavirus strain Quebec, complete genome/Group 2 </td></tr><tr><td>NC_003045 </td><td>BCoV </td><td>31 028 </td><td>Bovine coronavirus, complete genome/Group 2 </td></tr><tr><td>AF208067 </td><td>MHVM </td><td>31 233 </td><td>Murine hepatitis virus strain ML-10, complete genome/Group 2 </td></tr><tr><td>AF201929 </td><td>MHV2 </td><td>31 276 </td><td>Murine hepatitis virus strain 2, complete genome/Group 2 </td></tr><tr><td>AF208066 </td><td>MHVP </td><td>31 112 </td><td>Murine hepatitis virus strain Penn 97-1, complete genome/Group 2 </td></tr><tr><td>NC_001846 </td><td>MHV </td><td>31 357 </td><td>Murine hepatitis virus, complete genome/Group 2 </td></tr><tr><td>NC_001451 </td><td>IBV </td><td>27 608 </td><td>Avian infectious bronchitis virus, complete genome/Group 3 </td></tr><tr><td>EU095850 </td><td>TCoV </td><td>27 657 </td><td>Turkey coronavirus isolate MG10, complete genome/Group 3 </td></tr><tr><td>AY278488 </td><td>BJ01 </td><td>29 725 </td><td>SARS coronavirus BJ01, complete genome/Group 4 </td></tr><tr><td>AY278741 </td><td>Urbani </td><td>29 727 </td><td>SARS coronavirus Urbani, complete genome/Group 4 </td></tr><tr><td>AY278491 </td><td>HKU-39849 </td><td>29 742 </td><td>SARS coronavirus HKU-39849, complete genome/Group 4 </td></tr><tr><td>AY278554 </td><td>CUHK-W1 </td><td>29 736 </td><td>SARS coronavirus CUHK-W1, complete genome/Group 4 </td></tr><tr><td>AY282752 </td><td>CUHK-Su10 </td><td>29 736 </td><td>SARS coronavirus CUHK-Su10, complete genome/Group 4 </td></tr><tr><td>AY283794 </td><td>SIN2500 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2500, complete genome/Group 4 </td></tr><tr><td>AY283795 </td><td>SIN2677 </td><td>29 705 </td><td>SARS coronavirus isolate SIN2677, complete genome/Group 4 </td></tr><tr><td>AY283796 </td><td>SIN2679 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2679, complete genome/Group 4 </td></tr><tr><td>AY283797 </td><td>SIN2748 </td><td>29 706 </td><td>SARS coronavirus isolate SIN2748, complete genome/Group 4 </td></tr><tr><td>AY283798 </td><td>SIN2774 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2774, complete genome/Group 4 </td></tr><tr><td>AY291451 </td><td>TW1 </td><td>29 729 </td><td>SARS coronavirus TW1, complete genome/Group 4 </td></tr><tr><td>NC_004718 </td><td>TOR2 </td><td>29 751 </td><td>SARS coronavirus TOR2, complete genome/Group 4 </td></tr><tr><td>AY297028 </td><td>ZJ01 </td><td>29 715 </td><td>SARS coronavirus ZJ01, complete genome/Group 4 </td></tr><tr><td>AY572034 </td><td>Civet007 </td><td>29 540 </td><td>SARS coronavirus civet007, complete genome/Group 4 </td></tr><tr><td>AY572035 </td><td>Civet010 </td><td>29 518 </td><td>SARS coronavirus civet010, complete genome/Group 4 </td></tr><tr><td>NC_005831 </td><td>HCoV-NL63 </td><td>27 553 </td><td>Human coronavirus NL63, complete genome/Group 1 </td></tr><tr><td>NC_006577 </td><td>HCoV-HKU1 </td><td>29 926 </td><td>Human coronavirus HKU1, complete genome/Group 5 </td></tr><tr><td>NC_001564 </td><td>CellF </td><td>10 695 </td><td>Cell fusing agent virus, complete genome/<em>Flaviviridae</em> (outgroup) </td></tr><tr><td>NC_004102 </td><td>HepaCF </td><td>9646 </td><td>Hepatitis C virus, complete genome/<em>Flaviviridae</em> (outgroup) </td></tr><tr><td>NC_001512 </td><td>NyongT </td><td>11 835 </td><td>O'nyong-nyong virus, complete genome/<em>Togaviridae</em> (outgroup) </td></tr><tr><td>NC_001544 </td><td>RossT </td><td>11 657 </td><td>Ross River virus, complete genome/<em>Togaviridae</em> (outgroup) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5037254" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ008TB1" class="table-wrap-title" data-id="DSQ008TB1"><div class="label -label">Table 1</div><div class="caption"><p>Thirty coronavirus genomes and four outgroup genomes from the GenBank</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession No. </th><th>Abbreviation </th><th>Length (nt) </th><th>Description/Classification </th></tr></thead><tbody><tr><td>AF304460 </td><td>HCoV-229E </td><td>27 317 </td><td>Human coronavirus 229E, complete genome/Group 1 </td></tr><tr><td>AY391777 </td><td>HCoV-OC43 </td><td>30 738 </td><td>Human coronavirus OC43, complete genome/Group 2 </td></tr><tr><td>AF353511 </td><td>PEDV </td><td>28 033 </td><td>Porcine epidemic diarrhea virus strain, complete genome/Group 1 </td></tr><tr><td>U00735 </td><td>BCoVM </td><td>31 032 </td><td>Bovine coronavirus strain Mebus, complete genome/Group 2 </td></tr><tr><td>AF391542 </td><td>BCoVL </td><td>31 028 </td><td>Bovine coronavirus isolate BCoV-LUN, complete genome/Group 2 </td></tr><tr><td>AF220295 </td><td>BCoVQ </td><td>31 100 </td><td>Bovine coronavirus strain Quebec, complete genome/Group 2 </td></tr><tr><td>NC_003045 </td><td>BCoV </td><td>31 028 </td><td>Bovine coronavirus, complete genome/Group 2 </td></tr><tr><td>AF208067 </td><td>MHVM </td><td>31 233 </td><td>Murine hepatitis virus strain ML-10, complete genome/Group 2 </td></tr><tr><td>AF201929 </td><td>MHV2 </td><td>31 276 </td><td>Murine hepatitis virus strain 2, complete genome/Group 2 </td></tr><tr><td>AF208066 </td><td>MHVP </td><td>31 112 </td><td>Murine hepatitis virus strain Penn 97-1, complete genome/Group 2 </td></tr><tr><td>NC_001846 </td><td>MHV </td><td>31 357 </td><td>Murine hepatitis virus, complete genome/Group 2 </td></tr><tr><td>NC_001451 </td><td>IBV </td><td>27 608 </td><td>Avian infectious bronchitis virus, complete genome/Group 3 </td></tr><tr><td>EU095850 </td><td>TCoV </td><td>27 657 </td><td>Turkey coronavirus isolate MG10, complete genome/Group 3 </td></tr><tr><td>AY278488 </td><td>BJ01 </td><td>29 725 </td><td>SARS coronavirus BJ01, complete genome/Group 4 </td></tr><tr><td>AY278741 </td><td>Urbani </td><td>29 727 </td><td>SARS coronavirus Urbani, complete genome/Group 4 </td></tr><tr><td>AY278491 </td><td>HKU-39849 </td><td>29 742 </td><td>SARS coronavirus HKU-39849, complete genome/Group 4 </td></tr><tr><td>AY278554 </td><td>CUHK-W1 </td><td>29 736 </td><td>SARS coronavirus CUHK-W1, complete genome/Group 4 </td></tr><tr><td>AY282752 </td><td>CUHK-Su10 </td><td>29 736 </td><td>SARS coronavirus CUHK-Su10, complete genome/Group 4 </td></tr><tr><td>AY283794 </td><td>SIN2500 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2500, complete genome/Group 4 </td></tr><tr><td>AY283795 </td><td>SIN2677 </td><td>29 705 </td><td>SARS coronavirus isolate SIN2677, complete genome/Group 4 </td></tr><tr><td>AY283796 </td><td>SIN2679 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2679, complete genome/Group 4 </td></tr><tr><td>AY283797 </td><td>SIN2748 </td><td>29 706 </td><td>SARS coronavirus isolate SIN2748, complete genome/Group 4 </td></tr><tr><td>AY283798 </td><td>SIN2774 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2774, complete genome/Group 4 </td></tr><tr><td>AY291451 </td><td>TW1 </td><td>29 729 </td><td>SARS coronavirus TW1, complete genome/Group 4 </td></tr><tr><td>NC_004718 </td><td>TOR2 </td><td>29 751 </td><td>SARS coronavirus TOR2, complete genome/Group 4 </td></tr><tr><td>AY297028 </td><td>ZJ01 </td><td>29 715 </td><td>SARS coronavirus ZJ01, complete genome/Group 4 </td></tr><tr><td>AY572034 </td><td>Civet007 </td><td>29 540 </td><td>SARS coronavirus civet007, complete genome/Group 4 </td></tr><tr><td>AY572035 </td><td>Civet010 </td><td>29 518 </td><td>SARS coronavirus civet010, complete genome/Group 4 </td></tr><tr><td>NC_005831 </td><td>HCoV-NL63 </td><td>27 553 </td><td>Human coronavirus NL63, complete genome/Group 1 </td></tr><tr><td>NC_006577 </td><td>HCoV-HKU1 </td><td>29 926 </td><td>Human coronavirus HKU1, complete genome/Group 5 </td></tr><tr><td>NC_001564 </td><td>CellF </td><td>10 695 </td><td>Cell fusing agent virus, complete genome/<em>Flaviviridae</em> (outgroup) </td></tr><tr><td>NC_004102 </td><td>HepaCF </td><td>9646 </td><td>Hepatitis C virus, complete genome/<em>Flaviviridae</em> (outgroup) </td></tr><tr><td>NC_001512 </td><td>NyongT </td><td>11 835 </td><td>O'nyong-nyong virus, complete genome/<em>Togaviridae</em> (outgroup) </td></tr><tr><td>NC_001544 </td><td>RossT </td><td>11 657 </td><td>Ross River virus, complete genome/<em>Togaviridae</em> (outgroup) </td></tr></tbody></table></div><div content-id="DSQ008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession No. </th><th>Abbreviation </th><th>Length (nt) </th><th>Description/Classification </th></tr></thead><tbody><tr><td>AF304460 </td><td>HCoV-229E </td><td>27 317 </td><td>Human coronavirus 229E, complete genome/Group 1 </td></tr><tr><td>AY391777 </td><td>HCoV-OC43 </td><td>30 738 </td><td>Human coronavirus OC43, complete genome/Group 2 </td></tr><tr><td>AF353511 </td><td>PEDV </td><td>28 033 </td><td>Porcine epidemic diarrhea virus strain, complete genome/Group 1 </td></tr><tr><td>U00735 </td><td>BCoVM </td><td>31 032 </td><td>Bovine coronavirus strain Mebus, complete genome/Group 2 </td></tr><tr><td>AF391542 </td><td>BCoVL </td><td>31 028 </td><td>Bovine coronavirus isolate BCoV-LUN, complete genome/Group 2 </td></tr><tr><td>AF220295 </td><td>BCoVQ </td><td>31 100 </td><td>Bovine coronavirus strain Quebec, complete genome/Group 2 </td></tr><tr><td>NC_003045 </td><td>BCoV </td><td>31 028 </td><td>Bovine coronavirus, complete genome/Group 2 </td></tr><tr><td>AF208067 </td><td>MHVM </td><td>31 233 </td><td>Murine hepatitis virus strain ML-10, complete genome/Group 2 </td></tr><tr><td>AF201929 </td><td>MHV2 </td><td>31 276 </td><td>Murine hepatitis virus strain 2, complete genome/Group 2 </td></tr><tr><td>AF208066 </td><td>MHVP </td><td>31 112 </td><td>Murine hepatitis virus strain Penn 97-1, complete genome/Group 2 </td></tr><tr><td>NC_001846 </td><td>MHV </td><td>31 357 </td><td>Murine hepatitis virus, complete genome/Group 2 </td></tr><tr><td>NC_001451 </td><td>IBV </td><td>27 608 </td><td>Avian infectious bronchitis virus, complete genome/Group 3 </td></tr><tr><td>EU095850 </td><td>TCoV </td><td>27 657 </td><td>Turkey coronavirus isolate MG10, complete genome/Group 3 </td></tr><tr><td>AY278488 </td><td>BJ01 </td><td>29 725 </td><td>SARS coronavirus BJ01, complete genome/Group 4 </td></tr><tr><td>AY278741 </td><td>Urbani </td><td>29 727 </td><td>SARS coronavirus Urbani, complete genome/Group 4 </td></tr><tr><td>AY278491 </td><td>HKU-39849 </td><td>29 742 </td><td>SARS coronavirus HKU-39849, complete genome/Group 4 </td></tr><tr><td>AY278554 </td><td>CUHK-W1 </td><td>29 736 </td><td>SARS coronavirus CUHK-W1, complete genome/Group 4 </td></tr><tr><td>AY282752 </td><td>CUHK-Su10 </td><td>29 736 </td><td>SARS coronavirus CUHK-Su10, complete genome/Group 4 </td></tr><tr><td>AY283794 </td><td>SIN2500 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2500, complete genome/Group 4 </td></tr><tr><td>AY283795 </td><td>SIN2677 </td><td>29 705 </td><td>SARS coronavirus isolate SIN2677, complete genome/Group 4 </td></tr><tr><td>AY283796 </td><td>SIN2679 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2679, complete genome/Group 4 </td></tr><tr><td>AY283797 </td><td>SIN2748 </td><td>29 706 </td><td>SARS coronavirus isolate SIN2748, complete genome/Group 4 </td></tr><tr><td>AY283798 </td><td>SIN2774 </td><td>29 711 </td><td>SARS coronavirus isolate SIN2774, complete genome/Group 4 </td></tr><tr><td>AY291451 </td><td>TW1 </td><td>29 729 </td><td>SARS coronavirus TW1, complete genome/Group 4 </td></tr><tr><td>NC_004718 </td><td>TOR2 </td><td>29 751 </td><td>SARS coronavirus TOR2, complete genome/Group 4 </td></tr><tr><td>AY297028 </td><td>ZJ01 </td><td>29 715 </td><td>SARS coronavirus ZJ01, complete genome/Group 4 </td></tr><tr><td>AY572034 </td><td>Civet007 </td><td>29 540 </td><td>SARS coronavirus civet007, complete genome/Group 4 </td></tr><tr><td>AY572035 </td><td>Civet010 </td><td>29 518 </td><td>SARS coronavirus civet010, complete genome/Group 4 </td></tr><tr><td>NC_005831 </td><td>HCoV-NL63 </td><td>27 553 </td><td>Human coronavirus NL63, complete genome/Group 1 </td></tr><tr><td>NC_006577 </td><td>HCoV-HKU1 </td><td>29 926 </td><td>Human coronavirus HKU1, complete genome/Group 5 </td></tr><tr><td>NC_001564 </td><td>CellF </td><td>10 695 </td><td>Cell fusing agent virus, complete genome/<em>Flaviviridae</em> (outgroup) </td></tr><tr><td>NC_004102 </td><td>HepaCF </td><td>9646 </td><td>Hepatitis C virus, complete genome/<em>Flaviviridae</em> (outgroup) </td></tr><tr><td>NC_001512 </td><td>NyongT </td><td>11 835 </td><td>O'nyong-nyong virus, complete genome/<em>Togaviridae</em> (outgroup) </td></tr><tr><td>NC_001544 </td><td>RossT </td><td>11 657 </td><td>Ross River virus, complete genome/<em>Togaviridae</em> (outgroup) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5037254" target="_blank">View Large</a></div></div>                        <a id="5037256" scrollto-destination="5037256"></a>
<div id="DSQ008F5" data-id="DSQ008F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00805.jpeg?Expires=2147483647&Signature=vJTlbygSLY9P4FK4UG5d9loSVKvkiRlNYWYW6cJ-RX~-evOG9rFjWxwr0xRyNMLTNZf8G-Y6Aa9Nfb5dh21hMiixB~4miDhk530HEB3m3yCq3eG4bjNKXTb2m4yVq1mdW3F~tahlMqAqWxydIp90j7I7t3jHLqekaeelH-onypycSPM-rvB9Vp27zdlpgNmJ4eeKRuDzBMR-dBAPWclXXYbNd2rP9b59~Go8Ascgoq8v-wEMKsPDBVWu6c6odQddfWwYEMaEcdrsOGeSS1ie-3p7j5Wd6Xo~X~3NP9Z7y99crN8KK~urQX1wuy0jAEtqrv6T3CJ-HY~aQKt1Xk~Pkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two-dimensional genome space. (A) We used the first two components of the moment vector (M1, M2) to characterize 34 virus genomes from Table 1 and obtained 34 two-dimensional vectors. These 34 vectors can be directly viewed as 34 points in a two-dimensional genome space with M1 and M2 being x-axis and y-axis, respectively. (B) Thirty coronavirus genomes from Table 1 are viewed as 30 points in a two-dimensional genome space." path-from-xml="dsq00805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037256/dsq00805.jpeg" path-from-xml="dsq00805.jpeg" target="_blank">View large</a><a section="5037256" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00805.jpeg?Expires=2147483647&Signature=sNaAiHVxg23cB1fcwv6RcL6kw1zU-x-MPwNDCLAWpEFC9WPjRyI3Wf1ler0KIJdPQAIrujM1gRxu2zWVkaRD-QlJSWYuuG1gw6LtR18zlK0Jx6JDl-pa2Zwi1iuhcfI8f-sz~uXodS~ZVfW4847Mc3O~cRlQ61hsHQwnBDAM6d25FOxTYE8eeO~6OXZ0A~czDy2X9-MSHtQpv56k1JhQ7Dft2rg3AzqMa8aRTEUld7rxbWAVAvHZ0emnIDIRl3R3O35Ksw4Hmoud6WG9ntA2prlVbu1f2MxVEzGHCcJR-D4z1Kbvzn0GjhhzcaGzp-8rTDmO5AlDKHYN5ILUNBtWsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037256&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two-dimensional genome space. (A) We used the first two components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>) to characterize 34 virus genomes from Table <a reveal-id="DSQ008TB1" data-open="DSQ008TB1" class="link link-reveal link-table xref-fig">1</a> and obtained 34 two-dimensional vectors. These 34 vectors can be directly viewed as 34 points in a two-dimensional genome space with <em>M</em><sub>1</sub> and <em>M</em><sub>2</sub> being <em>x</em>-axis and <em>y</em>-axis, respectively. (B) Thirty coronavirus genomes from Table <a reveal-id="DSQ008TB1" data-open="DSQ008TB1" class="link link-reveal link-table xref-fig">1</a> are viewed as 30 points in a two-dimensional genome space.</p></div></div><div content-id="DSQ008F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00805.jpeg?Expires=2147483647&Signature=vJTlbygSLY9P4FK4UG5d9loSVKvkiRlNYWYW6cJ-RX~-evOG9rFjWxwr0xRyNMLTNZf8G-Y6Aa9Nfb5dh21hMiixB~4miDhk530HEB3m3yCq3eG4bjNKXTb2m4yVq1mdW3F~tahlMqAqWxydIp90j7I7t3jHLqekaeelH-onypycSPM-rvB9Vp27zdlpgNmJ4eeKRuDzBMR-dBAPWclXXYbNd2rP9b59~Go8Ascgoq8v-wEMKsPDBVWu6c6odQddfWwYEMaEcdrsOGeSS1ie-3p7j5Wd6Xo~X~3NP9Z7y99crN8KK~urQX1wuy0jAEtqrv6T3CJ-HY~aQKt1Xk~Pkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two-dimensional genome space. (A) We used the first two components of the moment vector (M1, M2) to characterize 34 virus genomes from Table 1 and obtained 34 two-dimensional vectors. These 34 vectors can be directly viewed as 34 points in a two-dimensional genome space with M1 and M2 being x-axis and y-axis, respectively. (B) Thirty coronavirus genomes from Table 1 are viewed as 30 points in a two-dimensional genome space." path-from-xml="dsq00805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037256/dsq00805.jpeg" path-from-xml="dsq00805.jpeg" target="_blank">View large</a><a section="5037256" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00805.jpeg?Expires=2147483647&Signature=sNaAiHVxg23cB1fcwv6RcL6kw1zU-x-MPwNDCLAWpEFC9WPjRyI3Wf1ler0KIJdPQAIrujM1gRxu2zWVkaRD-QlJSWYuuG1gw6LtR18zlK0Jx6JDl-pa2Zwi1iuhcfI8f-sz~uXodS~ZVfW4847Mc3O~cRlQ61hsHQwnBDAM6d25FOxTYE8eeO~6OXZ0A~czDy2X9-MSHtQpv56k1JhQ7Dft2rg3AzqMa8aRTEUld7rxbWAVAvHZ0emnIDIRl3R3O35Ksw4Hmoud6WG9ntA2prlVbu1f2MxVEzGHCcJR-D4z1Kbvzn0GjhhzcaGzp-8rTDmO5AlDKHYN5ILUNBtWsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037256&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two-dimensional genome space. (A) We used the first two components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>) to characterize 34 virus genomes from Table <a reveal-id="DSQ008TB1" data-open="DSQ008TB1" class="link link-reveal link-table xref-fig">1</a> and obtained 34 two-dimensional vectors. These 34 vectors can be directly viewed as 34 points in a two-dimensional genome space with <em>M</em><sub>1</sub> and <em>M</em><sub>2</sub> being <em>x</em>-axis and <em>y</em>-axis, respectively. (B) Thirty coronavirus genomes from Table <a reveal-id="DSQ008TB1" data-open="DSQ008TB1" class="link link-reveal link-table xref-fig">1</a> are viewed as 30 points in a two-dimensional genome space.</p></div></div><p>Based on genotypic and serological characterization, coronaviruses are divided into three distinct groups, with human coronavirus 229E (HCoV-229E) being a group 1 coronavirus and human coronavirus OC43 (HCoV-OC43) being a group 2 coronavirus. There is one additional branch, the group 3 coronaviruses, which are found exclusively in birds. These three distinct groups of coronaviruses clearly form three distinct clusters in our two-dimensional genome space. We also find that the newly discovered human coronavirus NL63 is very close to the human coronavirus 229E, and thus it is classified into group 1. This result is the same as the identification by van der Hoek <em>et al</em>.<sup><a reveal-id="DSQ008C27" data-open="DSQ008C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Our two-dimensional genome space reveals that for human SARS-CoV, the most closely related coronavirus is from a small infected mammal, the palm civet (not a bird as initially suspected). This result coincides with those found by Guan <em>et al</em>.<sup><a reveal-id="DSQ008C28" data-open="DSQ008C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and Wang <em>et al</em>.<sup><a reveal-id="DSQ008C29" data-open="DSQ008C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In Guan <em>et al</em>.'s work, the authors also suggested that SARS viruses were isolated from Himalayan palm civets found in a live-animal market in Guangdong, China. Similarly, in Wang <em>et al</em>.'s work, the researchers found six palm civets at the restaurant were positive for SARS-associated coronavirus. They think that SARS cases at the restaurant were the result of recent interspecies transfer from the putative palm civet reservoir and not the result of continued circulation of SARS-CoV in the human population.</p><p>For another newly discovered human coronavirus HCoV-HKU1, Woo <em>et al</em>.<sup><a reveal-id="DSQ008C30" data-open="DSQ008C30" class="link link-ref link-reveal xref-bibr">30</a></sup> classified it into group 2 because HCoV-HKU1 contains certain features that are characteristic of group 2 coronaviruses. However, these authors also stated that the proteins of HCoV-HKU1 formed distinct branches in the phylogenetic trees, indicating that HCoV-HKU1 is a distinct member within the group and is not very closely related to any other known member of group 2 coronaviruses. In our two-dimensional genome space, we found that HCoV-HKU1 is an individual coronavirus between the SARS group (which we suggest as group 4) and the traditional group 2, thus we propose that HCoV-HKU1 belongs to a new group 5. By computing the distances between the genomes in Fig. <a reveal-id="DSQ008F5" data-open="DSQ008F5" class="link link-reveal link-table xref-fig">5</a>A, we also reconstructed the phylogenetic tree of these coronaviruses (Fig. <a reveal-id="DSQ008F6" data-open="DSQ008F6" class="link link-reveal link-table xref-fig">6</a>) using UPGMA program in the MEGA 4 package.<sup><a reveal-id="DSQ008C23" data-open="DSQ008C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Thus, the evolutionary relationship of these genomes is clearer.</p>                        <a id="5037263" scrollto-destination="5037263"></a>
<div id="DSQ008F6" data-id="DSQ008F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00806.jpeg?Expires=2147483647&Signature=q2l7wJVWd975z0jLxVtWRPmmJ4KA9ZyGA1JFa4nfSoUVvuNUgLzExwvuDoPosgluRDkj7yTiiCh5RUQSqgN60dniB47z45DPRZMWxNdy-Ob2I8rRf5v9kKtGNqLdaUAhvUfPxtLUt~cZ7RhMPQEmqUlNSqnh1i6SZ0X-ic3H4sSyb8NDlOGOur9lOUjWHVxgTSF5LSmMbPJz1wYaw7MjpSG3AbZRs8eh28yPreYbzetTLXu8gM3cTebrJCoR1dfJTc5xOrdiaafXb-JNiCZfrVpz3XyS7Lm8ZVNPJqvHD9QI22YKP04RZbDa3WEYat8nmT~LvkshMjhFSxL5~qxjYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of the coronavirus genomes. This tree was reconstructed by the first two components of the moment vector (M1, M2)." path-from-xml="dsq00806.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037263/dsq00806.jpeg" path-from-xml="dsq00806.jpeg" target="_blank">View large</a><a section="5037263" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00806.jpeg?Expires=2147483647&Signature=IBCSnet5GNn7w-l~~rPQenV6dJ4Of7G2iyT5S5jphdG1-PGzHVYMXaW-z37SFBXOgjCxqJ5PuB1WdAzSLvvwcUhqhdf~zuLNfD6yKuU4I2zCdpPAQ7op3XHtopJdxGMohn0hynVhfoVeMUyzMhkYMdPEXTrb89r8K-rES2PR4LB-vNVudOCU39bpEgXEBHYmSa0qSuO0fftaejzhLM6cWjEFbj2eoq1xbC9RtmZ5jLHzgxHXs24SIzzOLnRMW~TisfXVHZuM5zv11eRWQIm84HZFO76TJYw3dCN~NEJUBjUCL4UcRXHR4JHvuH~Um6r0P5ls--XTDbYVEbqiJZeG0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037263&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00806.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of the coronavirus genomes. This tree was reconstructed by the first two components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>).</p></div></div><div content-id="DSQ008F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00806.jpeg?Expires=2147483647&Signature=q2l7wJVWd975z0jLxVtWRPmmJ4KA9ZyGA1JFa4nfSoUVvuNUgLzExwvuDoPosgluRDkj7yTiiCh5RUQSqgN60dniB47z45DPRZMWxNdy-Ob2I8rRf5v9kKtGNqLdaUAhvUfPxtLUt~cZ7RhMPQEmqUlNSqnh1i6SZ0X-ic3H4sSyb8NDlOGOur9lOUjWHVxgTSF5LSmMbPJz1wYaw7MjpSG3AbZRs8eh28yPreYbzetTLXu8gM3cTebrJCoR1dfJTc5xOrdiaafXb-JNiCZfrVpz3XyS7Lm8ZVNPJqvHD9QI22YKP04RZbDa3WEYat8nmT~LvkshMjhFSxL5~qxjYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of the coronavirus genomes. This tree was reconstructed by the first two components of the moment vector (M1, M2)." path-from-xml="dsq00806.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037263/dsq00806.jpeg" path-from-xml="dsq00806.jpeg" target="_blank">View large</a><a section="5037263" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00806.jpeg?Expires=2147483647&Signature=IBCSnet5GNn7w-l~~rPQenV6dJ4Of7G2iyT5S5jphdG1-PGzHVYMXaW-z37SFBXOgjCxqJ5PuB1WdAzSLvvwcUhqhdf~zuLNfD6yKuU4I2zCdpPAQ7op3XHtopJdxGMohn0hynVhfoVeMUyzMhkYMdPEXTrb89r8K-rES2PR4LB-vNVudOCU39bpEgXEBHYmSa0qSuO0fftaejzhLM6cWjEFbj2eoq1xbC9RtmZ5jLHzgxHXs24SIzzOLnRMW~TisfXVHZuM5zv11eRWQIm84HZFO76TJYw3dCN~NEJUBjUCL4UcRXHR4JHvuH~Um6r0P5ls--XTDbYVEbqiJZeG0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037263&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00806.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of the coronavirus genomes. This tree was reconstructed by the first two components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>).</p></div></div>                            <h2 scrollto-destination=5037267 id="5037267" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>It should be pointed out that the construction of our genome space depends on four parameters (the <em>y</em>-coordinates of the A, C, T, and G in Fig. <a reveal-id="DSQ008F1" data-open="DSQ008F1" class="link link-reveal link-table xref-fig">1</a>). If we change these four parameters, we shall have different embedding of the genome space. Because the GC content of DNA molecule is found to be variable with different organisms, GC content should be considered when we assign the <em>y</em>-coordinate values of nucleotide vectors. The genomes of three groups in this paper have low GC content. Most of them have 30–50% GC content, thus we assign larger <em>y</em>-coordinate absolute values for G and C. However, the <em>y</em>-coordinate values of the four nucleotides must be between −1 and 1 to assure that the correspondence between a DNA sequence and its corresponding moment vector is one-to-one. Therefore, in order to obtain a universal genome space for all species, further studies will be needed to determine the universal <em>y</em>-coordinate values.</p><p>In the study of mammalian mitochondrion, lentiviruses (including HIV), and coronavirus genomes, we used 60, 12, and 2 moments to construct the genome space, respectively. Here, we should emphasize that we do not need to calculate all the moments to determine the biological information of genomes. Remember that in the central limit theorem in probability and statistics, the limiting process is Gaussian. For Gaussian, the first two moments determine the density function. Thus, we just use the first <em>N</em> moments to get the results, where <em>N</em> is much less than <em>n</em> (the length of genome). Thus, for coronavirus genomes, we only used the first two components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>) because these two moments have allowed us to obtain the stable classified result—when higher moments are included the relationship of being close or farther away remains unchanged. To make this point clearer, we also use the first 20 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>20</sub>) to construct the 20-dimmensional genome space. By computing the Euclidean distances between these points in this genome space, we reconstructed the phylogenetic tree of these coronaviruses (Fig. <a reveal-id="DSQ008F7" data-open="DSQ008F7" class="link link-reveal link-table xref-fig">7</a>). Comparing Figs <a reveal-id="DSQ008F6" data-open="DSQ008F6" class="link link-reveal link-table xref-fig">6</a> and <a reveal-id="DSQ008F7" data-open="DSQ008F7" class="link link-reveal link-table xref-fig">7</a>, we found that the classification relationship of these genomes are the same—group 1, group 2, group 3, group 4, group 5, and outgroups can still be seen in the two trees as six distinct clusters. This means that when using the higher moments, the relationship of being close or farther away remains unchanged. In other word, two moments are already enough to give the right classifying relationship for these genomes. For the same reason, we used the first 60 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>60</sub>) and the first 12 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>12</sub>) to generate the genome space for mammalian mitochondrion and lentiviruses genomes and obtained the stable phylogenetic analysis result.</p>                        <a id="5037272" scrollto-destination="5037272"></a>
<div id="DSQ008F7" data-id="DSQ008F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00807.jpeg?Expires=2147483647&Signature=lfQDKgMfmR6h46XASLBiD-n8bShHBv28jSPUF1zpyEMv1XPW50nLhMfT4ZqSJt1rtm-WBW6rmNehBeJ-8xi0e4Dd2ViWWcXIgH9CfCgKm7lTU6~Awc0L9FCYl3lbC0btwSBtkNoOmM3ArQQTOVIhU2WaCWm-wuIszBteqkxH3U-wdvV54Sy41fKY3rgJquxGpL3kCnNMb-CSDA1LPkNXzgCXVzwItV5XepjD2WiNFXJ1nqAEmAQkBXf0vGlPdoZYZbo9-lRsglGy86UuGZ4B~JSBsfq0Yf1splYfh5lgo-udYgU3rxxXVshIS1MX2zxjVVo90FcH1eKeUWNVJVTNtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of the coronavirus genomes. This tree was reconstructed by the first 20 components of the moment vector (M1, M2, … , M20)." path-from-xml="dsq00807.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037272/dsq00807.jpeg" path-from-xml="dsq00807.jpeg" target="_blank">View large</a><a section="5037272" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00807.jpeg?Expires=2147483647&Signature=Z7PIqqNxUi4cbqtV8kUNzEcQuJBkiP~kjujqXOxRrIR6c0Ff1zBzmGKXeWe0rbrDsoXOUQipxeszAWiHyLWCf6Ik9qAkmESDhvzN2k3NO8gRETrAtVj-qsH9g~z8~7P40lmDkpbmojIVIkG3LJ3WecfRCWJP2oxdh~-12Ggn0Dvr2ZCiJh5ExnlYpnye8F3BCFnijN3RFFM0FfGZx5oEP670GLgf-9UiOKPjiYtoebG5VhqzRDvhZ7y1KA2OEm-Df9~osTPI3ks9gL7MN~UQEapz~tZrtae2pBzIt77iNxiqtbJrtkYMF9u0wWYb5auKiqore3nX2-kH3M6NhS4nYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037272&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00807.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of the coronavirus genomes. This tree was reconstructed by the first 20 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>20</sub>).</p></div></div><div content-id="DSQ008F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/m_dsq00807.jpeg?Expires=2147483647&Signature=lfQDKgMfmR6h46XASLBiD-n8bShHBv28jSPUF1zpyEMv1XPW50nLhMfT4ZqSJt1rtm-WBW6rmNehBeJ-8xi0e4Dd2ViWWcXIgH9CfCgKm7lTU6~Awc0L9FCYl3lbC0btwSBtkNoOmM3ArQQTOVIhU2WaCWm-wuIszBteqkxH3U-wdvV54Sy41fKY3rgJquxGpL3kCnNMb-CSDA1LPkNXzgCXVzwItV5XepjD2WiNFXJ1nqAEmAQkBXf0vGlPdoZYZbo9-lRsglGy86UuGZ4B~JSBsfq0Yf1splYfh5lgo-udYgU3rxxXVshIS1MX2zxjVVo90FcH1eKeUWNVJVTNtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of the coronavirus genomes. This tree was reconstructed by the first 20 components of the moment vector (M1, M2, … , M20)." path-from-xml="dsq00807.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5037272/dsq00807.jpeg" path-from-xml="dsq00807.jpeg" target="_blank">View large</a><a section="5037272" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq008/2/dsq00807.jpeg?Expires=2147483647&Signature=Z7PIqqNxUi4cbqtV8kUNzEcQuJBkiP~kjujqXOxRrIR6c0Ff1zBzmGKXeWe0rbrDsoXOUQipxeszAWiHyLWCf6Ik9qAkmESDhvzN2k3NO8gRETrAtVj-qsH9g~z8~7P40lmDkpbmojIVIkG3LJ3WecfRCWJP2oxdh~-12Ggn0Dvr2ZCiJh5ExnlYpnye8F3BCFnijN3RFFM0FfGZx5oEP670GLgf-9UiOKPjiYtoebG5VhqzRDvhZ7y1KA2OEm-Df9~osTPI3ks9gL7MN~UQEapz~tZrtae2pBzIt77iNxiqtbJrtkYMF9u0wWYb5auKiqore3nX2-kH3M6NhS4nYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5037272&ar=353741&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00807.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of the coronavirus genomes. This tree was reconstructed by the first 20 components of the moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub>20</sub>).</p></div></div><p>Gene rearrangements play important roles in evolution. The order of genes and transcription regions are changed during evolution by gene rearrangements such as DNA inversions and transpositions,<sup><a reveal-id="DSQ008C31" data-open="DSQ008C31" class="link link-ref link-reveal xref-bibr">31</a></sup> which do not affect the gene content of the chromosomes. For example, inversions of large genomic fragments are often observed even between closely related species. The phylogenetic analysis based on the gene order is challenging as it requires detailed complex gene order data in genomes and intensive computation. In order to test whether our genome space is stable for genomic rearrangement, we consider the following simulated experiment. We choose the human mitochondrion genome and then invert its two genes ATPase 6 and Cytochrome oxidase III to get a simulated genome, which we called human AC genome. We can treat this new genome as the result of the inversion of genes from the human mitochondrion genome. Next, we randomly generate a genome sequence which has the same length and nucleotide content as the human mitochondrion genome, which we called random genome. Thus, we have three genomes of the same length and nucleotide content: human, human AC, and random. In addition, we also choose the common chimpanzee mitochondrion genome as comparison. By using the 60-dimmensional genome space, we calculate the distance among these four genomes and get a distance matrix in Table <a reveal-id="DSQ008TB2" data-open="DSQ008TB2" class="link link-reveal link-table xref-fig">2</a>. In Table <a reveal-id="DSQ008TB2" data-open="DSQ008TB2" class="link link-reveal link-table xref-fig">2</a>, we found that the distance between human and human AC is very small. This means that even some genomic rearrangement happened in a genome, the new produced genome still has very close distance from the original genome. Thus, in our genome space, the original genome and its genomic rearrangement genome represent different points but have very close distance. Large-scale genomic rearrangements in closely related organisms make the genome sequences very different from the original genome sequences. Although the moment vectors of the original and rearranged DNA sequences are different, if we treat double DNA genomes as two single-stranded genomes to get two <em>n</em>-dimensional moment vectors for these two single-stranded sequences and take the average to get a general moment vector (<em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>, … , <em>M</em><sub><em>n</em></sub>), the computed distances among related species by gene rearrangements will be small. The genome spaces constructed by the moment vector have advantages over the gene order phylogenetic analysis since the genome spaces do not depend on gene orders and are capable of using all gene families. We will apply the moment vector method on the detailed phylogenetic analysis using the large-scale genomic rearrangement data. </p>                        <a id="5037277" scrollto-destination="5037277"></a>
<div content-id="DSQ008TB2" class="table-modal"><div class="table-wrap"><div id="DSQ008TB2" class="table-wrap-title" data-id="DSQ008TB2"><div class="label -label">Table 2</div><div class="caption"><p>The distance matrix of four genomes in the 60-dimmensional genome space</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Human </th><th>Human AC </th><th>Random </th><th>Chimpanzee </th></tr></thead><tbody><tr><td>Human </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Human AC </td><td>1.778773 </td><td> </td><td> </td><td> </td></tr><tr><td>Random </td><td>223.018945 </td><td>224.777237 </td><td> </td><td> </td></tr><tr><td>Chimpanzee </td><td>20.171072 </td><td>19.112686 </td><td>237.476133 </td><td> </td></tr></tbody></table></div><div content-id="DSQ008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Human </th><th>Human AC </th><th>Random </th><th>Chimpanzee </th></tr></thead><tbody><tr><td>Human </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Human AC </td><td>1.778773 </td><td> </td><td> </td><td> </td></tr><tr><td>Random </td><td>223.018945 </td><td>224.777237 </td><td> </td><td> </td></tr><tr><td>Chimpanzee </td><td>20.171072 </td><td>19.112686 </td><td>237.476133 </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5037277" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ008TB2" class="table-wrap-title" data-id="DSQ008TB2"><div class="label -label">Table 2</div><div class="caption"><p>The distance matrix of four genomes in the 60-dimmensional genome space</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Human </th><th>Human AC </th><th>Random </th><th>Chimpanzee </th></tr></thead><tbody><tr><td>Human </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Human AC </td><td>1.778773 </td><td> </td><td> </td><td> </td></tr><tr><td>Random </td><td>223.018945 </td><td>224.777237 </td><td> </td><td> </td></tr><tr><td>Chimpanzee </td><td>20.171072 </td><td>19.112686 </td><td>237.476133 </td><td> </td></tr></tbody></table></div><div content-id="DSQ008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Human </th><th>Human AC </th><th>Random </th><th>Chimpanzee </th></tr></thead><tbody><tr><td>Human </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Human AC </td><td>1.778773 </td><td> </td><td> </td><td> </td></tr><tr><td>Random </td><td>223.018945 </td><td>224.777237 </td><td> </td><td> </td></tr><tr><td>Chimpanzee </td><td>20.171072 </td><td>19.112686 </td><td>237.476133 </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5037277" target="_blank">View Large</a></div></div><p>In this paper, we report a two-dimensional graphical representation for DNA sequences. A moment vector system to represent a DNA sequence is introduced, and the correspondence between a DNA sequence and its moment vector is mathematically proven to be one-to-one. With this moment vector system, each genome sequence can be represented as a point in a Euclidean space, and the genome space is constructed as a subspace of this Euclidean space. Genomes with close evolutionary relationship and similar properties plot close together in this genome space. Thus, it will provide a new powerful tool for analyzing the classification of genomes and their phylogenetic relationships. Our method is easier and quicker in handling whole or partial genomes than multiple alignment methods. There are two major advantages to our method. (i) Once a genome space has been constructed, it can be stored in a database. There is no need to reconstruct the genome space for any subsequent application, whereas in multiple alignment methods, realignment is needed for add-on new sequences. (ii) One can have global comparison of all genomes simultaneously, which no other existing method can achieve. Furthermore, in our method, the results in two-dimensional genome space can be displayed and viewed graphically; this is user-friendly and allows even non-expert to understand the relationship among different genomes via viewing the graph of genome space.</p>                            <h2 scrollto-destination=5037280 id="5037280" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We gratefully acknowledge Professor Yuen-Ling Chan from University of Chicago, Department of Biochemistry and Molecular Biology who read our paper and gave some constructive comments. We also thank Dr Max Benson for critically reading and editing the manuscript.</p>                            <h2 scrollto-destination=5037283 id="5037283" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ008C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boore</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">W.M.</div></div>. </div><div class="article-title">Big tree from little genomes: mitochondrial gene order as a phylogenetic tool</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">668</div>-<div class="lpage">74</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0959-437X(98)80035-X" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0959-437X(98)80035-X</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Big%20tree%20from%20little%20genomes%3A%20mitochondrial%20gene%20order%20as%20a%20phylogenetic%20tool&amp;author=J.L.%20Boore&amp;author=W.M.%20Brown&amp;publication_year=1998&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=8&amp;pages=668-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(98)80035-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(98)80035-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(98)80035-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9914213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Huynen</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Genome phylogeny based on gene content</div>, <div class="source">Nat. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">108</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/5052" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/5052</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20phylogeny%20based%20on%20gene%20content&amp;author=B.%20Snel&amp;author=P.%20Bork&amp;author=M.A.%20Huynen&amp;publication_year=1999&amp;journal=Nat.%20Genet.&amp;volume=21&amp;pages=108-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/5052" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F5052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F5052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9916801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kececioglu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sankoff</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Exact and approximation algorithms for the inversion distance</div>, <div class="source">Algorithmica</div> , <div class="year">1995</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">180</div>-<div class="lpage">210</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF01188586" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF01188586</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exact%20and%20approximation%20algorithms%20for%20the%20inversion%20distance&amp;author=J.%20Kececioglu&amp;author=D.%20Sankoff&amp;publication_year=1995&amp;journal=Algorithmica&amp;volume=13&amp;pages=180-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01188586" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01188586" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01188586"> </span></div> </div></div></div></div></div><div content-id="DSQ008C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hannenhalli</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pevzner</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Transforming cabbage into turnip</div>, <div class="source">Proceedings of the 27th ACM Symposium on Theory of Computing</div> , <div class="year">1995</div>(pg. <div class="fpage">178</div>-<div class="lpage">89</div>)</div></div></div></div><div content-id="DSQ008C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nadeau</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Sankoff</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Counting on comparative maps</div>, <div class="source">Trends Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">495</div>-<div class="lpage">501</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0168-9525(98)01607-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0168-9525(98)01607-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Counting%20on%20comparative%20maps&amp;author=J.H.%20Nadeau&amp;author=D.%20Sankoff&amp;publication_year=1998&amp;journal=Trends%20Genet.&amp;volume=14&amp;pages=495-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(98)01607-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(98)01607-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(98)01607-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9865155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">The emerging paradigm and open problems in comparative genomics</div>, <div class="source">Bioinformatics</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">265</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/15.4.265" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/15.4.265</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20emerging%20paradigm%20and%20open%20problems%20in%20comparative%20genomics&amp;author=E.V.%20Koonin&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=265-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.4.265" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.4.265" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.4.265"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10438229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamori</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Novel DNA sequence representation</div>, <div class="source">Nature</div> , <div class="year">1985</div>, vol. <div class="volume">314</div> (pg. <div class="fpage">585</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/314585a0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/314585a0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20DNA%20sequence%20representation&amp;author=E.%20Hamori&amp;publication_year=1985&amp;journal=Nature&amp;volume=314&amp;pages=585-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/314585a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F314585a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F314585a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3990794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gates</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Simpler DNA sequence representations</div>, <div class="source">Nature</div> , <div class="year">1985</div>, vol. <div class="volume">316</div> pg. <div class="fpage">219</div> <div class="pub-id-doi"><a href="https://doi.org/10.1038/316219a0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/316219a0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simpler%20DNA%20sequence%20representations&amp;author=M.A.%20Gates&amp;publication_year=1985&amp;journal=Nature&amp;volume=316&amp;pages=219" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/316219a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F316219a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F316219a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3927167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yau</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Niknejad</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ho</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">DNA sequence representation without degeneracy</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3078</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkg432" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkg432</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20sequence%20representation%20without%20degeneracy&amp;author=S.S.%20Yau&amp;author=J.%20Wang&amp;author=A.%20Niknejad&amp;author=C.%20Lu&amp;author=N.%20Jin&amp;author=Y.%20Ho&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3078-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg432" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg432" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg432"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12799435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yau</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A protein map and its application</div>, <div class="source">DNA Cell Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">241</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1089/dna.2007.0676" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1089/dna.2007.0676</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20protein%20map%20and%20its%20application&amp;author=S.S.%20Yau&amp;author=C.%20Yu&amp;author=R.%20He&amp;publication_year=2008&amp;journal=DNA%20Cell%20Biol.&amp;volume=27&amp;pages=241-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/dna.2007.0676" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fdna.2007.0676" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fdna.2007.0676"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18348704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jacobson</div> <div class="given-names">N.</div></div>. </div>, <div class="source">Basic Algebra</div> , <div class="year">1974</div>, vol. <div class="volume">1</div> <div class="publisher-loc">India</div><div class="publisher-name">Hindustan Publishing Corporation</div>pg. <div class="fpage">135</div> </div></div></div></div><div content-id="DSQ008C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jukes</div> <div class="given-names">T.H.</div></div>,  <div class="name"><div class="surname">Cantor</div> <div class="given-names">C.R.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Munro</div> <div class="given-names">H.N.</div></div>,  <div class="name"><div class="surname">Allison</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">Evolution of protein molecules in mammalian protein metabolism</div>, <div class="source">Mammalian Protein Metabolism</div> , <div class="year">1969</div><div class="publisher-loc">New York</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">21</div>-<div class="lpage">132</div>)</div></div></div></div><div content-id="DSQ008C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A simple model for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1980</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">111</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF01731581" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF01731581</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20model%20for%20estimating%20evolutionary%20rates%20of%20base%20substitutions%20through%20comparative%20studies%20of%20nucleotide%20sequences&amp;author=M.%20Kimura&amp;publication_year=1980&amp;journal=J.%20Mol.%20Evol.&amp;volume=16&amp;pages=111-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01731581" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01731581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01731581"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7463489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barry</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hartigan</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Statistical analysis of ominoid molecular evolution</div>, <div class="source">Stat. Sci.</div> , <div class="year">1987</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">191</div>-<div class="lpage">210</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1214/ss/1177013353" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1214/ss/1177013353</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20analysis%20of%20ominoid%20molecular%20evolution&amp;author=D.%20Barry&amp;author=J.A.%20Hartigan&amp;publication_year=1987&amp;journal=Stat.%20Sci.&amp;volume=2&amp;pages=191-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1214/ss/1177013353" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1214%2Fss%2F1177013353" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1214%2Fss%2F1177013353"> </span></div> </div></div></div></div></div><div content-id="DSQ008C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lake</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Reconstructing evolutionary trees from DNA and protein sequences: paralinear distances</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">1455</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.91.4.1455" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.91.4.1455</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstructing%20evolutionary%20trees%20from%20DNA%20and%20protein%20sequences%3A%20paralinear%20distances&amp;author=J.A.%20Lake&amp;publication_year=1994&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=1455-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.4.1455" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.4.1455" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.4.1455"> </span></div> </div></div></div></div></div><div content-id="DSQ008C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Camin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sokal</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A method for deducing branching sequences in phylogeny</div>, <div class="source">Evolution</div> , <div class="year">1965</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">311</div>-<div class="lpage">26</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2307/2406441" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2307/2406441</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20method%20for%20deducing%20branching%20sequences%20in%20phylogeny&amp;author=J.%20Camin&amp;author=R.%20Sokal&amp;publication_year=1965&amp;journal=Evolution&amp;volume=19&amp;pages=311-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2406441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2406441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2406441"> </span></div> </div></div></div></div></div><div content-id="DSQ008C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cavalli-Sforza</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">A.W.F.</div></div>. </div><div class="article-title">Phylogenetic analysis: models and estimation procedures</div>, <div class="source">Evolution</div> , <div class="year">1967</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">550</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2307/2406616" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2307/2406616</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%3A%20models%20and%20estimation%20procedures&amp;author=L.L.%20Cavalli-Sforza&amp;author=A.W.F.%20Edwards&amp;publication_year=1967&amp;journal=Evolution&amp;volume=21&amp;pages=550-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2406616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2406616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2406616"> </span></div> </div></div></div></div></div><div content-id="DSQ008C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fitch</div> <div class="given-names">W.M.</div></div>. </div><div class="article-title">Toward defining the course of evolution: minimum change for a specific tree topology</div>, <div class="source">Syst. Zool.</div> , <div class="year">1971</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">406</div>-<div class="lpage">16</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2307/2412116" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2307/2412116</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20defining%20the%20course%20of%20evolution%3A%20minimum%20change%20for%20a%20specific%20tree%20topology&amp;author=W.M.%20Fitch&amp;publication_year=1971&amp;journal=Syst.%20Zool.&amp;volume=35&amp;pages=406-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2412116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2412116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2412116"> </span></div> </div></div></div></div></div><div content-id="DSQ008C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Maximum likelihood and minimum-steps methods for estimating evolutionary trees from data on discrete characters</div>, <div class="source">Syst. Zool.</div> , <div class="year">1973</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">240</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2307/2412304" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2307/2412304</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maximum%20likelihood%20and%20minimum-steps%20methods%20for%20estimating%20evolutionary%20trees%20from%20data%20on%20discrete%20characters&amp;author=J.%20Felsenstein&amp;publication_year=1973&amp;journal=Syst.%20Zool.&amp;volume=22&amp;pages=240-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2412304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2412304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2412304"> </span></div> </div></div></div></div></div><div content-id="DSQ008C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Evolutionary trees from DNA sequences: a maximum likelihood approach</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1981</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">368</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF01734359" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF01734359</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20trees%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;author=J.%20Felsenstein&amp;publication_year=1981&amp;journal=J.%20Mol.%20Evol.&amp;volume=17&amp;pages=368-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01734359" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01734359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01734359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7288891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Churchill</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">A hidden Markov model approach to variation among sites in rate of evolution</div>, <div class="source">Mol. Bio. Evol.</div> , <div class="year">1996</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">93</div>-<div class="lpage">104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hidden%20Markov%20model%20approach%20to%20variation%20among%20sites%20in%20rate%20of%20evolution&amp;author=J.%20Felsenstein&amp;author=G.A.%20Churchill&amp;publication_year=1996&amp;journal=Mol.%20Bio.%20Evol.&amp;volume=13&amp;pages=93-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025575" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025575" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025575"> </span></div> </div></div></div></div></div><div content-id="DSQ008C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brown</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Prager</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">Mitochondrial DNA sequences of primates: tempo and mode of evolution</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1982</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">225</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF01734101" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF01734101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20sequences%20of%20primates%3A%20tempo%20and%20mode%20of%20evolution&amp;author=W.M.%20Brown&amp;author=E.M.%20Prager&amp;author=A.%20Wang&amp;author=A.C.%20Wilson&amp;publication_year=1982&amp;journal=J.%20Mol.%20Evol.&amp;volume=18&amp;pages=225-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01734101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01734101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01734101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6284948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>. </div><div class="article-title">MEGA: A biologist-centric software for evolutionary analysis of DNA and protein sequences</div>, <div class="source">Brief. Bioinform.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">299</div>-<div class="lpage">306</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bib/bbn017" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bib/bbn017</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA%3A%20A%20biologist-centric%20software%20for%20evolutionary%20analysis%20of%20DNA%20and%20protein%20sequences&amp;author=S.%20Kumar&amp;author=M.%20Nei&amp;author=J.%20Dudley&amp;author=K.%20Tamura&amp;publication_year=2008&amp;journal=Brief.%20Bioinform.&amp;volume=9&amp;pages=299-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbn017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18417537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raina</div> <div class="given-names">S.Z.</div></div>,  <div class="name"><div class="surname">Faith</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Disotell</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Seligmann</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">C.B.</div></div>,  <div class="name"><div class="surname">Pollock</div> <div class="given-names">D.D.</div></div>. </div><div class="article-title">Evolution of base-substitution gradients in primate mitochondrial genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">665</div>-<div class="lpage">73</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.3128605" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.3128605</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20base-substitution%20gradients%20in%20primate%20mitochondrial%20genomes&amp;author=S.Z.%20Raina&amp;author=J.J.%20Faith&amp;author=T.R.%20Disotell&amp;author=H.%20Seligmann&amp;author=C.B.%20Stewart&amp;author=D.D.%20Pollock&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=665-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3128605" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3128605" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3128605"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15867428" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rambaut</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Posada</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Crandall</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Holmes</div> <div class="given-names">E.C.</div></div>. </div><div class="article-title">The causes and consequences of HIV evolution</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">52</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg1246" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg1246</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20causes%20and%20consequences%20of%20HIV%20evolution&amp;author=A.%20Rambaut&amp;author=D.%20Posada&amp;author=K.A.%20Crandall&amp;author=E.C.%20Holmes&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=52-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1246" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1246" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1246"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14708016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bailes</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Robertson</div> <div class="given-names">D.L.</div></div>, et al. </div><div class="article-title">Origin of HIV-1 in the chimpanzee <em>Pan troglodytes troglodytes</em></div>, <div class="source">Nature</div> , <div class="year">1999</div>, vol. <div class="volume">397</div> (pg. <div class="fpage">436</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/17130" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/17130</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20HIV-1%20in%20the%20chimpanzee%20Pan%20troglodytes%20troglodytes&amp;author=F.%20Gao&amp;author=E.%20Bailes&amp;author=D.L.%20Robertson&amp;publication_year=1999&amp;journal=Nature&amp;volume=397&amp;pages=436-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/17130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F17130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F17130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9989410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van der Hoek</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Pyrc</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jebbink</div> <div class="given-names">M.F.</div></div>, et al. </div><div class="article-title">Identification of a new human coronavirus</div>, <div class="source">Nat. Med.</div> , <div class="year">2004</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">368</div>-<div class="lpage">73</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nm1024" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nm1024</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20new%20human%20coronavirus&amp;author=L.%20van%20der%20Hoek&amp;author=K.%20Pyrc&amp;author=M.F.%20Jebbink&amp;publication_year=2004&amp;journal=Nat.%20Med.&amp;volume=10&amp;pages=368-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm1024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm1024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm1024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034574" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">Y.Q.</div></div>, et al. </div><div class="article-title">Isolation and characterization of viruses related to the SARS coronavirus from animals in southern China</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">276</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1087139" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1087139</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20viruses%20related%20to%20the%20SARS%20coronavirus%20from%20animals%20in%20southern%20China&amp;author=Y.%20Guan&amp;author=B.J.%20Zheng&amp;author=Y.Q.%20He&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=276-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1087139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1087139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1087139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12958366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">SARS-CoV infection in a restaurant from palm civet</div>, <div class="source">Emerg. Infect. Dis.</div> , <div class="year">2005</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1860</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SARS-CoV%20infection%20in%20a%20restaurant%20from%20palm%20civet&amp;author=M.%20Wang&amp;author=M.%20Yan&amp;author=H.%20Xu&amp;publication_year=2005&amp;journal=Emerg.%20Infect.%20Dis.&amp;volume=11&amp;pages=1860-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3201/eid1112.041293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3201%2Feid1112.041293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3201%2Feid1112.041293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16485471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Woo</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Lau</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Chu</div> <div class="given-names">C.M.</div></div>, et al. </div><div class="article-title">Characterization and complete genome sequence of a novel coronavirus, coronavirus HKU1, from patients with pneumonia</div>, <div class="source">J. Virol.</div> , <div class="year">2005</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">884</div>-<div class="lpage">95</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/JVI.79.2.884-895.2005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/JVI.79.2.884-895.2005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20complete%20genome%20sequence%20of%20a%20novel%20coronavirus%2C%20coronavirus%20HKU1%2C%20from%20patients%20with%20pneumonia&amp;author=P.C.%20Woo&amp;author=S.K.%20Lau&amp;author=C.M.%20Chu&amp;publication_year=2005&amp;journal=J.%20Virol.&amp;volume=79&amp;pages=884-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JVI.79.2.884-895.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJVI.79.2.884-895.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJVI.79.2.884-895.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15613317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ008C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ008C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sankoff</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Nadeau</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Chromosome rearrangements in evolution: From gene order to genome sequence and back</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">11188</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.2035002100" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.2035002100</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20rearrangements%20in%20evolution%3A%20From%20gene%20order%20to%20genome%20sequence%20and%20back&amp;author=D.%20Sankoff&amp;author=H.J.%20Nadeau&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=11188-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.2035002100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.2035002100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.2035002100"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353741&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">147</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000279411300003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">43</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353741"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000279411300003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f3%2f155%2f353741">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000279411300003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (43)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F3%2F155%2F353741" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353741'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Novel Construction of Genome Space with Biological Geometry" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f155%2f353741">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

