

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/3/235/3061403","@type":"ScholarlyArticle","name":"Integrated miRNA and mRNA expression profiling reveals the response regulators of a susceptible tomato cultivar to early blight disease","datePublished":"2017-03-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2017-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx003","keywords":["tomato","<italic>Alternaria solani</italic>","miRNA","transcriptome","next generation sequencing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/3/235/3061403","author":[{"name":"Sarkar, Deepti","affiliation":"1 Division of Plant Biology, Bose Institute, Kolkata 700054, India","@type":"Person"},{"name":"Maji, Ranjan Kumar","affiliation":"2 Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India","@type":"Person"},{"name":"Dey, Sayani","affiliation":"1 Division of Plant Biology, Bose Institute, Kolkata 700054, India","@type":"Person"},{"name":"Sarkar, Arijita","affiliation":"2 Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India","@type":"Person"},{"name":"Ghosh, Zhumur","affiliation":"2 Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India","@type":"Person"},{"name":"Kundu, Pallob","affiliation":"1 Division of Plant Biology, Bose Institute, Kolkata 700054, India","@type":"Person"}],"description":"Abstract.  Early blight, caused by the fungus Alternaria solani, is a devastating foliar disease of tomatoes, causes massive yield loss each year worldwide. Mol","pageStart":"235","pageEnd":"250","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Integrated miRNA and mRNA expression profiling reveals the response regulators of a susceptible tomato cultivar to early blight disease","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Integrated miRNA and mRNA expression profiling reveals the response regulators of a susceptible tomato cultivar to early blight disease" />
    <meta property="og:description" content="Abstract.  Early blight, caused by the fungus Alternaria solani, is a devastating foliar disease of tomatoes, causes massive yield loss each year worldwide. Mol" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/3/235/3061403" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-03-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sarkar, Deepti" /><meta name="citation_author_institution" content="Division of Plant Biology, Bose Institute, Kolkata 700054, India" /><meta name="citation_author" content="Maji, Ranjan Kumar" /><meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India" /><meta name="citation_author" content="Dey, Sayani" /><meta name="citation_author_institution" content="Division of Plant Biology, Bose Institute, Kolkata 700054, India" /><meta name="citation_author" content="Sarkar, Arijita" /><meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India" /><meta name="citation_author" content="Ghosh, Zhumur" /><meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India" /><meta name="citation_author" content="Kundu, Pallob" /><meta name="citation_author_institution" content="Division of Plant Biology, Bose Institute, Kolkata 700054, India" /><meta name="citation_title" content="Integrated miRNA and mRNA expression profiling reveals the response regulators of a susceptible tomato cultivar to early blight disease" /><meta name="citation_firstpage" content="235" /><meta name="citation_lastpage" content="250" /><meta name="citation_doi" content="10.1093/dnares/dsx003" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2017/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=New species of wild tomatoes (Solanum Section Lycopersicon: Solanaceae) from Northern Peru;  citation_journal_title=Syst. Bot;  citation_year=2005;  citation_volume=30;  citation_pages=424-34; " /><meta name="citation_reference" content="citation_title=Assessment of early blight (Alternaria solani) resistance in tomato using a droplet inoculation method;  citation_journal_title=J. Gen. Plant Pathol;  citation_year=2007;  citation_volume=73;  citation_pages=96-103; " /><meta name="citation_reference" content="citation_title=MicroRNA-directed regulation of Arabidopsis AUXIN RESPONSE FACTOR17 is essential for proper development and modulates expression of early auxin response genes;  citation_journal_title=Plant cell;  citation_year=2005;  citation_volume=17;  citation_pages=1360-75; " /><meta name="citation_reference" content="citation_title=Characterization of miRNAs associated with Botrytis cinerea infection of tomato leaves;  citation_journal_title=BMC Plant Biol;  citation_year=2015;  citation_volume=15;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Diverse set of microRNAs are responsive to powdery mildew infection and heat stress in wheat (Triticum aestivum L.);  citation_journal_title=BMC Plant Biol;  citation_year=2010;  citation_volume=10;  citation_pages=123" /><meta name="citation_reference" content="citation_title=Identification of MiRNA from eggplant (Solanum melongena L.) by small RNA deep sequencing and their response to Verticillium dahliae infection;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e72840" /><meta name="citation_reference" content="citation_title=Arabidopsis defense against Botrytis cinerea: chronology and regulation deciphered by high-resolution temporal transcriptomic analysis;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=3530-57; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis of the compatible interaction of tomato with Verticillium dahliae using RNA-sequencing;  citation_journal_title=Front. Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=428" /><meta name="citation_reference" content="citation_title=Fusarium oxysporum f.sp. radicis-lycopersici induces distinct transcriptome reprogramming in resistant and susceptible isogenic tomato lines;  citation_journal_title=BMC Plant Biol;  citation_year=2016;  citation_volume=16;  citation_pages=53" /><meta name="citation_reference" content="citation_title=Microarray analyses during early stage of the tomato/Alternaria solani interaction;  citation_journal_title=Genom. Data;  citation_year=2015;  citation_volume=6;  citation_pages=170-2; " /><meta name="citation_reference" content="citation_title=Integrative analysis of the microRNAome and transcriptome illuminates the response of susceptible rice plants to rice stripe virus;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0146946" /><meta name="citation_reference" content="citation_title=Integration of small RNAs, degradome and transcriptome sequencing in hyperaccumulator Sedum alfredii uncovers a complex regulatory network and provides insights into cadmium phytoremediation;  citation_journal_title=Plant Biotechnol. J;  citation_year=2016;  citation_volume=14;  citation_pages=1470-83; " /><meta name="citation_reference" content="citation_title=Integrated analysis of miRNA and mRNA expression profiles in response to Cd exposure in rice seedlings;  citation_journal_title=BMC Genom;  citation_year=2014;  citation_volume=15;  citation_pages=835" /><meta name="citation_reference" content="citation_title=Salicylic acid biosynthesis and metabolism;  citation_journal_title=The Arabidopsis Book/American Society of Plant Biologists;  citation_year=2011;  citation_volume=9;  citation_pages=e0156" /><meta name="citation_reference" content="citation_title=Derepression of ethylene-stabilized transcription factors (EIN3/EIL1) mediates jasmonate and ethylene signaling synergy in Arabidopsis;  citation_journal_title=Proc. Natl Acad. Sci. U. S. A;  citation_year=2011;  citation_volume=108;  citation_pages=12539-44; " /><meta name="citation_reference" content="citation_title=Cross-talk of nitric oxide and reactive oxygen species in plant programed cell death;  citation_journal_title=Front. Plant Sci;  citation_year=2013;  citation_volume=4;  citation_pages=314" /><meta name="citation_reference" content="citation_title=Prediction and characterization of Tomato leaf curl New Delhi virus (ToLCNDV) responsive novel microRNAs in Solanum lycopersicum;  citation_journal_title=Virus Res;  citation_year=2015;  citation_volume=195;  citation_pages=183-95; " /><meta name="citation_reference" content="citation_title=MicroRNAs suppress NB domain genes in tomato that confer resistance to Fusarium oxysporum;  citation_journal_title=PLoS Pathogens;  citation_year=2014;  citation_volume=10;  citation_pages=e1004464" /><meta name="citation_reference" content="citation_title=Classification and comparison of small RNAs from plants;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2013;  citation_volume=64;  citation_pages=137-59; " /><meta name="citation_reference" content="citation_title=Identification of chilling stress-responsive tomato microRNAs and their target genes by high-throughput sequencing and degradome analysis;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1130" /><meta name="citation_reference" content="citation_title=Small RNA and degradome deep sequencing reveals drought-and tissue-specific micrornas and their important roles in drought-sensitive and drought-tolerant tomato genotypes;  citation_journal_title=Plant Biotechnol. J;  citation_year=2016;  citation_volume=14;  citation_pages=1727-46; " /><meta name="citation_reference" content="citation_title=The advance of tomato disease-related microRNAs;  citation_journal_title=Plant Cell Rep;  citation_year=2015;  citation_volume=34;  citation_pages=1089-97; " /><meta name="citation_reference" content="citation_title=Analysis of tomato microRNAs expression profile induced by Cucumovirus and Tobamovirus infections;  citation_journal_title=J. Nanosci. Nanotechnol;  citation_year=2012;  citation_volume=12;  citation_pages=143-50; " /><meta name="citation_reference" content="citation_title=High-throughput sequencing reveals differential expression of miRNAs in tomato inoculated with Phytophthora infestans;  citation_journal_title=Planta;  citation_year=2015;  citation_volume=241;  citation_pages=1405-16; " /><meta name="citation_reference" content="citation_title=Identification of microRNAs and their targets in tomato infected with Cucumber mosaic virus based on deep sequencing;  citation_journal_title=Planta;  citation_year=2014;  citation_volume=240;  citation_pages=1335-52; " /><meta name="citation_reference" content="citation_title=Identification of novel soybean microRNAs involved in abiotic and biotic stresses;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=307" /><meta name="citation_reference" content="citation_title=Biotic stress globally downregulates photosynthesis genes;  citation_journal_title=Plant Cell Environ;  citation_year=2010;  citation_volume=33;  citation_pages=1597-613; " /><meta name="citation_reference" content="citation_title=Glutathione transferases as mediators of signaling pathways involved in cell proliferation and cell death;  citation_journal_title=Cell Death Different;  citation_year=2010;  citation_volume=17;  citation_pages=1373-80; " /><meta name="citation_reference" content="citation_title=Phenylpropanoid biosynthesis;  citation_journal_title=Mol. Plant;  citation_year=2010;  citation_volume=3;  citation_pages=2-20; " /><meta name="citation_reference" content="citation_title=Identification of microRNA targets in tomato fruit development using high-throughput sequencing and degradome analysis;  citation_journal_title=J. Exp. Bot;  citation_year=2013;  citation_volume=64;  citation_pages=1863-78; " /><meta name="citation_reference" content="citation_title=A microRNA superfamily regulates nucleotide binding site-leucine-rich repeats and other mRNAs;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=859-74; " /><meta name="citation_reference" content="citation_title=MicroRNAS and their regulatory roles in plants;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2006;  citation_volume=57;  citation_pages=19-53; " /><meta name="citation_reference" content="citation_title=Massive analysis of rice small RNAs: mechanistic implications of regulated microRNAs and variants for differential target RNA cleavage;  citation_journal_title=Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=4185-207; " /><meta name="citation_reference" content="citation_title=MicroRNA-mediated regulation of gene expression in the response of rice plants to fungal elicitors;  citation_journal_title=RNA Biol;  citation_year=2015;  citation_volume=12;  citation_pages=847-63; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/3/235/17743989/dsx003.pdf" /><meta name="description" content="Abstract.  Early blight, caused by the fungus Alternaria solani, is a devastating foliar disease of tomatoes, causes massive yield loss each year worldwide. Mol" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/3/235/3061403" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/3/235/3061403" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/3/235/3061403");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx003';
    </script>
<title>
	Integrated miRNA and mRNA expression profiling reveals the response regulators of a susceptible tomato cultivar to early blight disease | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3061403" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZdZPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXJrYXIsIERlZXB0aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIFBsYW50IEJpb2xvZ3ksIEJvc2UgSW5zdGl0dXRlLCBLb2xrYXRhIDcwMDA1NCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYWppLCBSYW5qYW4gS3VtYXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgb2YgRXhjZWxsZW5jZSBpbiBCaW9pbmZvcm1hdGljcywgQm9zZSBJbnN0aXR1dGUsIEtvbGthdGEsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGV5LCBTYXlhbmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBQbGFudCBCaW9sb2d5LCBCb3NlIEluc3RpdHV0ZSwgS29sa2F0YSA3MDAwNTQsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2Fya2FyLCBBcmlqaXRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIG9mIEV4Y2VsbGVuY2UgaW4gQmlvaW5mb3JtYXRpY3MsIEJvc2UgSW5zdGl0dXRlLCBLb2xrYXRhLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikdob3NoLCBaaHVtdXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgb2YgRXhjZWxsZW5jZSBpbiBCaW9pbmZvcm1hdGljcywgQm9zZSBJbnN0aXR1dGUsIEtvbGthdGEsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VuZHUsIFBhbGxvYiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIFBsYW50IEJpb2xvZ3ksIEJvc2UgSW5zdGl0dXRlLCBLb2xrYXRhIDcwMDA1NCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkludGVncmF0ZWQgbWlSTkEgYW5kIG1STkEgZXhwcmVzc2lvbiBwcm9maWxpbmcgcmV2ZWFscyB0aGUgcmVzcG9uc2UgcmVndWxhdG9ycyBvZiBhIHN1c2NlcHRpYmxlIHRvbWF0byBjdWx0aXZhciB0byBlYXJseSBibGlnaHQgZGlzZWFzZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgc3BlY2llcyBvZiB3aWxkIHRvbWF0b2VzIChTb2xhbnVtIFNlY3Rpb24gTHljb3BlcnNpY29uOiBTb2xhbmFjZWFlKSBmcm9tIE5vcnRoZXJuIFBlcnU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTQyNC0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiBlYXJseSBibGlnaHQgKEFsdGVybmFyaWEgc29sYW5pKSByZXNpc3RhbmNlIGluIHRvbWF0byB1c2luZyBhIGRyb3BsZXQgaW5vY3VsYXRpb24gbWV0aG9kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBHZW4uIFBsYW50IFBhdGhvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIGNpdGF0aW9uX3BhZ2VzPTk2LTEwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9STkEtZGlyZWN0ZWQgcmVndWxhdGlvbiBvZiBBcmFiaWRvcHNpcyBBVVhJTiBSRVNQT05TRSBGQUNUT1IxNyBpcyBlc3NlbnRpYWwgZm9yIHByb3BlciBkZXZlbG9wbWVudCBhbmQgbW9kdWxhdGVzIGV4cHJlc3Npb24gb2YgZWFybHkgYXV4aW4gcmVzcG9uc2UgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IGNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMzYwLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIG1pUk5BcyBhc3NvY2lhdGVkIHdpdGggQm90cnl0aXMgY2luZXJlYSBpbmZlY3Rpb24gb2YgdG9tYXRvIGxlYXZlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzZSBzZXQgb2YgbWljcm9STkFzIGFyZSByZXNwb25zaXZlIHRvIHBvd2RlcnkgbWlsZGV3IGluZmVjdGlvbiBhbmQgaGVhdCBzdHJlc3MgaW4gd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBNaVJOQSBmcm9tIGVnZ3BsYW50IChTb2xhbnVtIG1lbG9uZ2VuYSBMLikgYnkgc21hbGwgUk5BIGRlZXAgc2VxdWVuY2luZyBhbmQgdGhlaXIgcmVzcG9uc2UgdG8gVmVydGljaWxsaXVtIGRhaGxpYWUgaW5mZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTcyODQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJhYmlkb3BzaXMgZGVmZW5zZSBhZ2FpbnN0IEJvdHJ5dGlzIGNpbmVyZWE6IGNocm9ub2xvZ3kgYW5kIHJlZ3VsYXRpb24gZGVjaXBoZXJlZCBieSBoaWdoLXJlc29sdXRpb24gdGVtcG9yYWwgdHJhbnNjcmlwdG9taWMgYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0zNTMwLTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuYWx5c2lzIG9mIHRoZSBjb21wYXRpYmxlIGludGVyYWN0aW9uIG9mIHRvbWF0byB3aXRoIFZlcnRpY2lsbGl1bSBkYWhsaWFlIHVzaW5nIFJOQS1zZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz00MjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdXNhcml1bSBveHlzcG9ydW0gZi5zcC4gcmFkaWNpcy1seWNvcGVyc2ljaSBpbmR1Y2VzIGRpc3RpbmN0IHRyYW5zY3JpcHRvbWUgcmVwcm9ncmFtbWluZyBpbiByZXNpc3RhbnQgYW5kIHN1c2NlcHRpYmxlIGlzb2dlbmljIHRvbWF0byBsaW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYXJyYXkgYW5hbHlzZXMgZHVyaW5nIGVhcmx5IHN0YWdlIG9mIHRoZSB0b21hdG8vQWx0ZXJuYXJpYSBzb2xhbmkgaW50ZXJhY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tLiBEYXRhOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNzAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgYW5hbHlzaXMgb2YgdGhlIG1pY3JvUk5Bb21lIGFuZCB0cmFuc2NyaXB0b21lIGlsbHVtaW5hdGVzIHRoZSByZXNwb25zZSBvZiBzdXNjZXB0aWJsZSByaWNlIHBsYW50cyB0byByaWNlIHN0cmlwZSB2aXJ1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1lMDE0Njk0NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aW9uIG9mIHNtYWxsIFJOQXMsIGRlZ3JhZG9tZSBhbmQgdHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIGluIGh5cGVyYWNjdW11bGF0b3IgU2VkdW0gYWxmcmVkaWkgdW5jb3ZlcnMgYSBjb21wbGV4IHJlZ3VsYXRvcnkgbmV0d29yayBhbmQgcHJvdmlkZXMgaW5zaWdodHMgaW50byBjYWRtaXVtIHBoeXRvcmVtZWRpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNDcwLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGVkIGFuYWx5c2lzIG9mIG1pUk5BIGFuZCBtUk5BIGV4cHJlc3Npb24gcHJvZmlsZXMgaW4gcmVzcG9uc2UgdG8gQ2QgZXhwb3N1cmUgaW4gcmljZSBzZWVkbGluZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTgzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbGljeWxpYyBhY2lkIGJpb3N5bnRoZXNpcyBhbmQgbWV0YWJvbGlzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlIEFyYWJpZG9wc2lzIEJvb2svQW1lcmljYW4gU29jaWV0eSBvZiBQbGFudCBCaW9sb2dpc3RzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMDE1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlcmVwcmVzc2lvbiBvZiBldGh5bGVuZS1zdGFiaWxpemVkIHRyYW5zY3JpcHRpb24gZmFjdG9ycyAoRUlOMy9FSUwxKSBtZWRpYXRlcyBqYXNtb25hdGUgYW5kIGV0aHlsZW5lIHNpZ25hbGluZyBzeW5lcmd5IGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xMjUzOS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3Jvc3MtdGFsayBvZiBuaXRyaWMgb3hpZGUgYW5kIHJlYWN0aXZlIG94eWdlbiBzcGVjaWVzIGluIHBsYW50IHByb2dyYW1lZCBjZWxsIGRlYXRoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0zMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIFRvbWF0byBsZWFmIGN1cmwgTmV3IERlbGhpIHZpcnVzIChUb0xDTkRWKSByZXNwb25zaXZlIG5vdmVsIG1pY3JvUk5BcyBpbiBTb2xhbnVtIGx5Y29wZXJzaWN1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VmlydXMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE5NTsgIGNpdGF0aW9uX3BhZ2VzPTE4My05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9STkFzIHN1cHByZXNzIE5CIGRvbWFpbiBnZW5lcyBpbiB0b21hdG8gdGhhdCBjb25mZXIgcmVzaXN0YW5jZSB0byBGdXNhcml1bSBveHlzcG9ydW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgUGF0aG9nZW5zOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDQ0NjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbGFzc2lmaWNhdGlvbiBhbmQgY29tcGFyaXNvbiBvZiBzbWFsbCBSTkFzIGZyb20gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz0xMzctNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGNoaWxsaW5nIHN0cmVzcy1yZXNwb25zaXZlIHRvbWF0byBtaWNyb1JOQXMgYW5kIHRoZWlyIHRhcmdldCBnZW5lcyBieSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyBhbmQgZGVncmFkb21lIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMTMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U21hbGwgUk5BIGFuZCBkZWdyYWRvbWUgZGVlcCBzZXF1ZW5jaW5nIHJldmVhbHMgZHJvdWdodC1hbmQgdGlzc3VlLXNwZWNpZmljIG1pY3Jvcm5hcyBhbmQgdGhlaXIgaW1wb3J0YW50IHJvbGVzIGluIGRyb3VnaHQtc2Vuc2l0aXZlIGFuZCBkcm91Z2h0LXRvbGVyYW50IHRvbWF0byBnZW5vdHlwZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNzI3LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYWR2YW5jZSBvZiB0b21hdG8gZGlzZWFzZS1yZWxhdGVkIG1pY3JvUk5BczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xMDg5LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0b21hdG8gbWljcm9STkFzIGV4cHJlc3Npb24gcHJvZmlsZSBpbmR1Y2VkIGJ5IEN1Y3Vtb3ZpcnVzIGFuZCBUb2JhbW92aXJ1cyBpbmZlY3Rpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBOYW5vc2NpLiBOYW5vdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE0My01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgcmV2ZWFscyBkaWZmZXJlbnRpYWwgZXhwcmVzc2lvbiBvZiBtaVJOQXMgaW4gdG9tYXRvIGlub2N1bGF0ZWQgd2l0aCBQaHl0b3BodGhvcmEgaW5mZXN0YW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudGE7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjQxOyAgY2l0YXRpb25fcGFnZXM9MTQwNS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgbWljcm9STkFzIGFuZCB0aGVpciB0YXJnZXRzIGluIHRvbWF0byBpbmZlY3RlZCB3aXRoIEN1Y3VtYmVyIG1vc2FpYyB2aXJ1cyBiYXNlZCBvbiBkZWVwIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50YTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDA7ICBjaXRhdGlvbl9wYWdlcz0xMzM1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBub3ZlbCBzb3liZWFuIG1pY3JvUk5BcyBpbnZvbHZlZCBpbiBhYmlvdGljIGFuZCBiaW90aWMgc3RyZXNzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb3RpYyBzdHJlc3MgZ2xvYmFsbHkgZG93bnJlZ3VsYXRlcyBwaG90b3N5bnRoZXNpcyBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBFbnZpcm9uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTU5Ny02MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsdXRhdGhpb25lIHRyYW5zZmVyYXNlcyBhcyBtZWRpYXRvcnMgb2Ygc2lnbmFsaW5nIHBhdGh3YXlzIGludm9sdmVkIGluIGNlbGwgcHJvbGlmZXJhdGlvbiBhbmQgY2VsbCBkZWF0aDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbCBEZWF0aCBEaWZmZXJlbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMzczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaGVueWxwcm9wYW5vaWQgYmlvc3ludGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBtaWNyb1JOQSB0YXJnZXRzIGluIHRvbWF0byBmcnVpdCBkZXZlbG9wbWVudCB1c2luZyBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyBhbmQgZGVncmFkb21lIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFeHAuIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTE4NjMtNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9STkEgc3VwZXJmYW1pbHkgcmVndWxhdGVzIG51Y2xlb3RpZGUgYmluZGluZyBzaXRlLWxldWNpbmUtcmljaCByZXBlYXRzIGFuZCBvdGhlciBtUk5BczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTg1OS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9STkFTIGFuZCB0aGVpciByZWd1bGF0b3J5IHJvbGVzIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MTktNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hc3NpdmUgYW5hbHlzaXMgb2YgcmljZSBzbWFsbCBSTkFzOiBtZWNoYW5pc3RpYyBpbXBsaWNhdGlvbnMgb2YgcmVndWxhdGVkIG1pY3JvUk5BcyBhbmQgdmFyaWFudHMgZm9yIGRpZmZlcmVudGlhbCB0YXJnZXQgUk5BIGNsZWF2YWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDE4NS0yMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvUk5BLW1lZGlhdGVkIHJlZ3VsYXRpb24gb2YgZ2VuZSBleHByZXNzaW9uIGluIHRoZSByZXNwb25zZSBvZiByaWNlIHBsYW50cyB0byBmdW5nYWwgZWxpY2l0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkEgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTg0Ny02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzMvMjM1LzE3NzQzOTg5L2RzeDAwMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRWFybHkgYmxpZ2h0LCBjYXVzZWQgYnkgdGhlIGZ1bmd1cyBBbHRlcm5hcmlhIHNvbGFuaSwgaXMgYSBkZXZhc3RhdGluZyBmb2xpYXIgZGlzZWFzZSBvZiB0b21hdG9lcywgY2F1c2VzIG1hc3NpdmUgeWllbGQgbG9zcyBlYWNoIHllYXIgd29ybGR3aWRlLiBNb2wiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTMwNjE0MDMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkHZj/cYERIy+2xqLklgDMpCCQjok=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f235%2f3061403" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f235%2f3061403" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&amp;Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/3">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="88473739">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473739">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473741">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473741">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473747">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473747">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473783">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473783">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473837">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473837">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473860">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473860">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473862">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473862">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473870">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473870">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/3/221/3052238">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/3/251/3058507">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Integrated miRNA and mRNA expression profiling reveals the response regulators of a susceptible tomato cultivar to early blight disease
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deepti Sarkar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Division of Plant Biology, Bose Institute, Kolkata 700054, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Deepti+Sarkar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sarkar D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sarkar D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Deepti Sarkar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ranjan Kumar Maji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ranjan+Kumar+Maji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maji R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Maji R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ranjan Kumar Maji</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sayani Dey</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Division of Plant Biology, Bose Institute, Kolkata 700054, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sayani+Dey">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dey S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dey S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sayani Dey</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arijita Sarkar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Arijita+Sarkar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sarkar A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sarkar A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Arijita Sarkar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhumur Ghosh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Centre of Excellence in Bioinformatics, Bose Institute, Kolkata, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhumur+Ghosh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ghosh Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ghosh Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhumur Ghosh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pallob Kundu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Division of Plant Biology, Bose Institute, Kolkata 700054, India</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx003-cor1" data-open="dsx003-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +033 2569 3298. Fax +033 2355 3886. Email: <a href="mailto:pkundu@jcbose.ac.in" target="_blank">pkundu@jcbose.ac.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pallob+Kundu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kundu P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kundu P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Pallob Kundu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx003-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 235–250, <a href='https://doi.org/10.1093/dnares/dsx003'>https://doi.org/10.1093/dnares/dsx003</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 March 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">04 August 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 January 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3061403" href="/dnaresearch/article-pdf/24/3/235/17743989/dsx003.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Deepti Sarkar, Ranjan Kumar Maji, Sayani Dey, Arijita Sarkar, Zhumur Ghosh, Pallob Kundu;  Integrated miRNA and mRNA expression profiling reveals the response regulators of a susceptible tomato cultivar to early blight disease, <em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 235–250, <a href='https://doi.org/10.1093/dnares/dsx003'>https://doi.org/10.1093/dnares/dsx003</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3061403?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3061403?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3061403?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3061403?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3061403?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3061403?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3061403?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Integrated%20miRNA%20and%20mRNA%20expression%20profiling%20reveals%20the%20response%20regulators%20of%20a%20susceptible%20tomato%20cultivar%20to%20early%20blight%20disease&amp;publicationDate=2017-03-03&amp;volumeNum=24&amp;issueNum=3&amp;author=Sarkar%2C%20Deepti%3B%20Maji%2C%20Ranjan%20Kumar&amp;startPage=235&amp;endPage=250&amp;contentId=10.1093%2Fdnares%2Fdsx003&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f235%2f3061403" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=88473739 id="88473739" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Early blight, caused by the fungus <em>Alternaria solani</em>, is a devastating foliar disease of tomatoes, causes massive yield loss each year worldwide. Molecular basis of the compatible host–pathogen interaction was elusive. We adopted next generation sequencing approach to decipher miRNAs and mRNAs that are differentially expressed during <em>Alternaria</em>-stress in tomato. Some of the interesting findings were also validated by alternative techniques. Our analysis revealed 181 known-miRNAs, belonging to 121 miRNA families, of which 67 miRNAs showed at least 2-fold change in expression level with the majority being downregulated. Concomitantly, 5,450 mRNAs were significantly regulated in the same diseased tissues. Differentially expressed genes were most significantly associated with response to stimulus process, photosynthesis, biosynthesis of secondary metabolites, plant–pathogen interaction and plant hormone signal transduction pathways. GO term enrichment-based categorization of gene-functions further supported this observation, as terms related to pathogen perception, disease signal transduction, cellular metabolic processes including oxidoreductase and kinase activity were over represented. In addition, we have discovered 102 miRNA–mRNA pairs which were regulated antagonistically, and careful study of the targeted mRNAs depicted that multiple transcription factors, nucleotide-binding site leucine-rich repeats, receptor-like proteins and enzymes related to cellular ROS management were profoundly affected. These studies have identified key regulators of <em>Alternaria</em>-stress response in tomato and the subset of genes that are likely to be post-transcriptionally silenced during the infection. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="tomato">tomato</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Alternaria solani&quot;"><em>Alternaria solani</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="miRNA">miRNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next generation sequencing&quot;">next generation sequencing</a></div>                            <h2 scrollto-destination=88473741 id="88473741" class="section-title" >1. Introduction</h2>
<p>Early blight (EB) is one of the most devastating diseases of tomato (<em>Solanum lycopersicum</em>) caused by <em>Alternaria solani</em> (Ellis &amp; Martin) Sorauer, a necrotrophic fungus. The disease largely affects the older leaves, stem and fruits of tomato plants.<a reveal-id="dsx003-B1" data-open="dsx003-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Cultivars highly resistant to EB are not known.<a reveal-id="dsx003-B1" data-open="dsx003-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Progress in breeding for EB resistance has been, however, limited by the lack of effective resistance genes in cultivated tomato, and by polygenic inheritance of the resistance.<a reveal-id="dsx003-B2" data-open="dsx003-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Thus, there is an urgent need for identifying new sources of resistance or host factors that are altered during the pathogenesis process and could be modulated for coordinating the resistance response.</p><p>Plants try to defend themselves against phytopathogens by coordinating defence-oriented transcriptional reprogramming of the affected cells. Several signalling and transcription factors are the key elements in this process. Expressions of some of these genes are regulated by miRNAs.<a reveal-id="dsx003-B3" data-open="dsx003-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> It is expected that during tomato–<em>Alternaria</em> interaction expression of a set of miRNAs and mRNAs will be altered, however, such studies are lacking. Moreover, limited information is available about molecular signalling involved in tomato–necrotrophic pathogen interaction.</p><p>Recently, the next generation sequencing (NGS) technology has been used to obtain comprehensive sequencing data for the detection and study of alteration of the expression levels of miRNAs or mRNAs in various plant species under different stress conditions including fungus–stress.<a reveal-id="dsx003-B4 dsx003-B5 dsx003-B6 dsx003-B7 dsx003-B8 dsx003-B9" data-open="dsx003-B4 dsx003-B5 dsx003-B6 dsx003-B7 dsx003-B8 dsx003-B9" class="link link-ref link-reveal xref-bibr"><sup>4–9</sup></a> The identified miRNAs primarily controlled the expression of genes encoding disease-resistance proteins, serine/threonine protein kinases and transcription factors. In a recent publication, microarray-based profiling of tomato mRNA during the early stage of <em>A</em><em>.</em><em>solani</em> infection has been reported,<a reveal-id="dsx003-B10" data-open="dsx003-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> however, a detailed analysis is not available.</p><p>For the proper understanding of the regulatory action of miRNA, an integrated analysis of miRNA and mRNA transcriptome is a requisite. Using plant as the model system, only three such studies with other pathogen or treatment have been reported; these include rice–rice stripe virus (RSV) interaction, rice and <em>Sedum alfredii</em>-cadmium treatment.<a reveal-id="dsx003-B11 dsx003-B12 dsx003-B13" data-open="dsx003-B11 dsx003-B12 dsx003-B13" class="link link-ref link-reveal xref-bibr"><sup>11–13</sup></a> Interestingly, rice–RSV study revealed expression of 22 up-regulated miRNAs targeted 24 repressed mRNAs which are linked to disease resistance pathways.</p><p>The best way of bestowing disease resistance to a plant is to potentiate its natural resistance mechanism. As mentioned earlier, transcriptional reprogramming and regulation of a set of gene/regulator would determine a plant’s response towards the pathogen. Deliberate alteration of expression of these factors potentially should control a plant’s response towards a pathogen. To appreciate fully the factors needed to be altered, the regulators of host processes during the pathogenesis and disease manifestation have to be unravelled. In this regard, a comparative analysis of control vs infected susceptible-plant is advantageous over studying the tolerant plant because multiple modes of tolerance are feasible; however, it is more likely that susceptibility response would follow a unique path. We hypothesized that several miRNAs have the crucial regulatory role in the transcriptional reprogramming process during <em>Alternaria</em> infection, and set out to uncover by genome-wide analysis of miRNA–mRNA relationships specifically regulated during the stress in tomato. To the best of our knowledge, this is the first report of integrating miRNA and mRNA transcriptome analysis data to endorse miRNA-mediated regulation of gene expression during tomato–<em>A</em><em>.</em><em>solani</em> interaction. This combined analysis facilitated in gaining significant insights into the regulatory network of susceptible response against an agronomically important necrotrophic pathogen of tomato.</p>                            <h2 scrollto-destination=88473747 id="88473747" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=88473748 id="88473748" class="section-title" >2.1. Plant material and treatment condition</h3>
<p>Tomato (<em>S</em><em>.</em><em>lycopersicum</em> L.) cultivar Pusa Ruby (Indian Agricultural Research Institute) was used in this study and is known to be susceptible to EB. Plants were grown in pots containing Soilrite (Keltech, India) at 25°C and under natural light with addition of Hoagland solution at regular interval. <em>A</em><em>.</em><em>solani</em> (obtained from IIVR, India) was cultured in potato dextrose agar plate at 25°C for 5 days in dark. Leaves of one and a half month old plants were deliberately inoculated with a drop (∼10 µl, 10<sup>6</sup> CFU/ml water) of mycelial and spore suspension of <em>A. solani</em> on the dorsal surface of leaf and were kept in humid chamber until symptoms appeared (∼3 days) or as mentioned in individual experiments. Control or mock inoculated plants received only a drop of water as inoculum and maintained under similar condition.</p>                            <h3 scrollto-destination=88473750 id="88473750" class="section-title" >2.2. Biochemical assays</h3>
                            <h4 scrollto-destination=88473751 id="88473751" class="section-title" >2.2.1. Callose detection</h4>
<p>Plant leaflets cut from their petiolule were immersed in alcoholic lactophenol (1:1:1:1 ratio of phenol:glycerol:lactic acid:water and 2 volumes of absolute ethanol) and warmed gently to remove the chlorophyll. The dechlorophyllized leaves were rinsed with water and stained with 0.01% toluidine blue solution in 150-mM disodium phosphate buffer (pH 9.5) for 30 min in dark. Leaves were then stained with 0.01% aniline blue for another 30 min in dark, and samples were observed under fluorescent microscope (Carl Zeiss Axioskop 40 FL) with an excitation and emission wavelengths of 470 and 515 nm, respectively.</p>                            <h4 scrollto-destination=88473753 id="88473753" class="section-title" >2.2.2. Nitric oxide detection</h4>
<p>Leaf sections (∼0.5 cm<sup>2</sup>) encompassing the fungal inoculation site were used. The leaf cuttings were gently vacuum-infiltrated with 10 mM DAF-2DA dissolved in 10 mM Tris–HCl (pH 7.5) at room temperature for 1 h. Fluorescence of the tissues was monitored under a fluorescent microscope (excitation/emission are 495/515 nm).</p>                            <h4 scrollto-destination=88473755 id="88473755" class="section-title" >2.2.3. Detection of ROS accumulation</h4>
<p>Detection of intracellular ROS, particularly hydrogen peroxide, using 3,3′-diaminobenzidine (DAB) was performed. Leaflets cut at their petiolule were incubated in DAB staining solution (DAB at 1 mg/ml of 10 mM Na<sub>2</sub>HPO<sub>4</sub>), in dark, overnight at 28°C with constant agitation. DAB staining solution was replaced with bleaching solution (ethanol:acetic acid:glycerol = 3:1:1) and incubated in boiling water for 10–15 min to get rid of chlorophyll, the process was repeated till all of the chlorophyll was bleached away, and then the leaflets were visualized under white light and photographed.</p>                            <h3 scrollto-destination=88473757 id="88473757" class="section-title" >2.3. Scanning electron microscopy (SEM)</h3>
<p>Infected and control tomato leaves were cut into ∼1-cm<sup>2</sup> size pieces and fixed with 3% glutaraldehyde in 0.2-M sodium phosphate buffer (pH 7.4) overnight at 4°C. After rinsing the samples in same buffer, they were dehydrated in increasing concentration of ethanol (50, 70, 85 and 95%) for 30 min in each solution and were kept in absolute ethanol until attaching on SEM stubs. The prepared specimens were directly examined in a SEM (FEI Quanta 200 FEG MKII) at 5–10 kV.</p>                            <h3 scrollto-destination=88473759 id="88473759" class="section-title" >2.4. RNA extraction and cDNA synthesis</h3>
<p>Visible fungal mass and dead tissues from the <em>Alternaria</em>-infected leaves were gently removed and samples (∼300 mg) were collected only from the infected zone. Total RNA was prepared from both infected and mock-inoculated (control) leaves by TRIzol reagent (Invitrogen) following manufacturer’s protocol. RNA was resolved in a 0.8% denaturing formaldehyde gel to check its integrity. Only high-quality RNA was used for downstream processing. Typically, 3 µg of total RNA was reverse transcribed to cDNA using RevertAid Reverse Transcriptase (ThermoFischer scientific) and random hexamer primer at 42°C for 60 min. cDNA was stored at −20°C if not immediately used.</p>                            <h3 scrollto-destination=88473761 id="88473761" class="section-title" >2.5. Real time PCR analysis</h3>
<p>Only 2.5% of the cDNA reaction mix was used as template in each 20 μl reaction containing SYBR green PCR mix (Thermo Scientific) and gene specific primers (0.2 µM each). PCR and analysis were performed using Applied Biosystems 7500 FAST machine. For each sample of cDNA three replicates were analysed. Data were normalized based on the expression level of EF1α transcripts. Amplification of specific products were confirmed by obtaining a single peak in melt curve analysis, first derivative of fluorescence (d<em>F</em>/d<em>T</em>) vs temperature plot and by agarose gel electrophoresis (1.5% agarose-TAE gel) of PCR end-products. Relative expression levels of transcripts were determined using the 2<sup>−</sup><sup>ΔΔ</sup><sup><em>cT</em></sup> method. All experiments were repeated with three independent RNA preparations.</p>                            <h3 scrollto-destination=88473763 id="88473763" class="section-title" >2.6. Small RNA library preparation and sequencing</h3>
<p>Total RNA for NGS was isolated following the method as mentioned above. Integrity of RNA (RIN &gt;7) was further assessed using Bioanalyzer 2100 (Agilent Technologies) on an RNA 6000 Nanochip. These RNAs were also subjected to RT-PCR based analysis for detection of fungal RNA contamination using <em>A. solani</em>-specific primers for <em>AltA1</em>, <em>GAPDH</em> genes and <em>ITS</em> sequence (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). Small RNA libraries for sequencing were constructed using 1 µg of total RNA, according to the Illumina TruSeq Small RNA library preparation protocol (Illumina, USA). In brief, after adapter ligation at both ends the ligated products were reverse transcribed with Superscript II Reverse transcriptase (RT) by priming with adapter-specific RT-primers, PCR amplified and 140–190-bp size fragments were gel purified. The purified high-quality cDNA library was sequenced (Genotypic Pvt. Ltd., India) on an Illumina Genome AnalyserIIx (GAIIx).</p>                            <h3 scrollto-destination=88473765 id="88473765" class="section-title" >2.7. Small RNA analysis</h3>
<p>High-quality reads (Q-phred ≥30) obtained from four different samples (Controls 1 and 2, Infected 1 and 2, GEO accession: GSE75922) were subjected to adapter removal and size fractionation to collect only 18–25 nt reads. Probable fungal miRNA contaminants were discarded. Remaining reads were Blast searched against annotated plant miRNAs gathered from miRBase (release 21), Tomato Functional Genomics Database (TFGD) and Tomato genomics resources database (TGRD) to obtain the known miRNAs. Alignments with no gap and ≤2 mismatches between the query sequences and known miRNAs (subject) were only considered. The identified miRNAs were grouped into families based on sequence similarity. The reads which did not match to any known miRNAs were further processed to discover novel miRNAs using Mireap v0.2 software. Deseq was used to obtain the differentially expressed miRNAs. Outline of the analysis pipeline and a detailed protocol are presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S1 and S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Document S1</a></span>.</p>                            <h3 scrollto-destination=88473767 id="88473767" class="section-title" >2.8. Preparation of DNA oligonucleotide probes</h3>
<p>About 10 pmol of DNA oligo was used for end labelling with 15 U of T4 PNK (NEB) and 40 µCi γ-32P-ATP (BRIT, India) in a total volume of 15 µl according to enzyme manufactures (NEB) protocol. The labelled probes were then purified using illustra<sup>TM</sup> MicroSpin G-25 column (GE Health care), and specific activity was determined after scintillation counting of the purified probe. Probes with specific activity of about 20 × 10<sup>7</sup> cpm/µg oligo was routinely obtained and used in subsequent applications.</p>                            <h3 scrollto-destination=88473769 id="88473769" class="section-title" >2.9. Northern blot hybridization analysis of miRNAs</h3>
<p>Total RNA, 40 μg per sample was resolved in a 7-M urea/10% polyacrylamide/TBE gel at 80 V and transferred to nylon membrane Hybond<sup>TM</sup>–XL (GE healthcare) using Semi-Dry Transfer Cell (TE 77 PWR, GE Healthcare Amersham™) at 150 V for 1 h. RNA was fixed using UV cross linker (UVC 500 UV Crosslinkers, GE Healthcare Amersham™) at 15,000 J/cm<sup>2</sup> for 2 min. The membrane was incubated for 30 min in pre-hybridization buffer (50% Deionized formamide, 5× SSPE buffer, 5× Denhardt’s solution, 0.5% SDS, supplemented with 0.02 mg/ml denatured Salmon sperm DNA) at 42°C, and then hybridization with γ-32P labelled DNA oligo (∼20 × 10<sup>6</sup> cpm) in the same prehybridization buffer was carried out in a heat seal bag by incubating overnight at 42°C with occasional shaking. Membrane was washed using a standard protocol, exposed to phosphor screen and image was captured at 100 µm pixel in Typhoon Trio+ (GE healthcare).</p>                            <h3 scrollto-destination=88473771 id="88473771" class="section-title" >2.10. Detection of miRNA by polyadenylation-mediated PCR</h3>
<p>Small RNA population from total RNA was enriched using PureLink® miRNA Isolation Kit (Invitrogen) according to manufacturer’s protocol. Quality was examined by estimating the integrity of tRNA band loaded in denaturing 7% urea PAGE gel. 500 ng of small RNA was polyadenylated, and reverse transcribed to cDNA according to the protocol of NCode™ VILO™ miRNA cDNA Synthesis Kit (Invitrogen). Desired miRNA was amplified from the cDNA pool by using mature miRNA-specific forward primer and universal qPCR reverse primer provided in the kit. Relative intensity of the U6 snRNA-specific product band or tRNA band in the same amount of small RNA samples that were resolved in a urea-PAGE gel was monitored as normalizing factor.</p>                            <h3 scrollto-destination=88473773 id="88473773" class="section-title" >2.11. mRNA library construction, Illumina sequencing and transcriptome data analysis</h3>
<p>Total RNA from Control 1 and Infected 1 samples were also used for mRNA library preparation and sequencing (Illumina Genome AnalyserIIx) according to the Illumina TruSeq RNA library protocol (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Document 1</a></span>). The paired end raw reads were quality checked using SeqQC (a proprietary tool developed in Genotypic Technology Pvt. Ltd, India) and processed by Genotypic in-house script for adapters and low-quality bases trimming towards 3′-end. Sequence files are deposited in GEO database under the accession number GSE75923. Tophat-2.0.72 and Cufflinks-2.0.14 tools were used for assembling transcripts and estimating their abundances. Data were normalized as fragments mapped per kilobase of exon per million reads mapped. Cuffdiff (included in the cufflinks package) was used to find significant changes in transcript expression. Outline of the analysis pipeline is presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>. Transcripts having log<sub>2</sub> Fold Change (log<sub>2</sub>FC) at least ±1 and <em>P</em> ≤  0.05 were used for further analysis.</p>                            <h3 scrollto-destination=88473775 id="88473775" class="section-title" >2.12. Gene Ontology (GO) and the KEGG pathway annotation of <em>Alternaria</em> responsive transcripts</h3>
<p>The differentially expressed mRNAs were subjected to GO terms and pathway enrichment analyses. The GO terms were extracted for the regulated genes from Sol Genomics Network (SGN) <a class="link link-uri" href="http://ITAG2.4.go.csv" target="_blank">ITAG2.4.go.csv</a>. GO term enrichment was done using Cytoscape plugin: BiNGO of Cytoscape v3.2.1 software with Benjamini and Hochberg false discovery rate (FDR) setting of ≤0.05. BlastKOALA was used for <em>K</em> number assignment to the differentially expressed genes (DEGs) using SSEARCH computation against a non-redundant set of KEGG genes, so that the gene could be put under a pathway. Pathway enrichment was done in a similar manner as for GO using Cytoscape v3.2.1 using the same parameters.</p>                            <h3 scrollto-destination=88473777 id="88473777" class="section-title" >2.13. Prediction of miRNA targets</h3>
<p>Two different softwares were used to predict the putative targets of differentially expressed known and novel miRNA candidates. The putative target sites of miRNA candidates were identified by aligning the miRNA sequences with tomato unigene library, using the plant small RNA Target Analysis Server, psRNATarget, with default parameters except a more relaxed cut-off threshold (5.0) of maximum expectation for higher prediction coverage. In addition to it, we used target prediction function of TFGD with default parameters to predict the putative targets. Expression levels of targets were taken from list of differentially expressed mRNAs identified in our mRNA transcriptome data analysis. GO terms were assigned to these targets using the similar method that has been utilized for assigning terms to DEGs as described in previous section.</p>                            <h3 scrollto-destination=88473779 id="88473779" class="section-title" >2.14. Target cleavage validation by using modified 5′RLM RACE</h3>
<p>Total RNA from tomato leaves were isolated, an RNA adapter was directly ligated to 3 μg of RNA using T4 RNA ligase (NEB). RNA was reverse transcribed using reverse transcriptase (Fermentas) with random hexamers. To amplify RACE products, first PCR was done with 1% of the RT reaction. Initial PCR reactions were done with an adaptor primer and complementary gene specific primers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). PCR cycles were 95°C for 30 s, 58°C for 30 s and 72°C for 45 s for 35 cycles. The nested PCRs were performed on 2.5 μl of initial reactions with the nested adaptor primer and complementary gene-specific internal primers. The resulting PCR products were gel purified, cloned in pGEM-T Easy vector (Promega) and sequenced.</p>                            <h3 scrollto-destination=88473781 id="88473781" class="section-title" >2.15. Statistics</h3>
<p>Each RNA sample for NGS was obtained from multiple leaves collected from different plants. Except radioactive-based assays, in all other experiments at least three different plants of same age group were used. Three different RNA preparations were used for real-time PCR analysis; reactions were also done in triplicates. Q-PCR results are expressed as means ± standard error (SE). Two-tail Student’s <em>t</em>-test was used and <em>P</em> &lt; 0.05 was considered significant for RT-PCR-based differential gene expression analysis.</p>                            <h2 scrollto-destination=88473783 id="88473783" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=88473784 id="88473784" class="section-title" >3.1. Plant infection and disease characterization</h3>
<p>Leaves of tomato plants were inoculated with <em>A</em><em>.</em><em>solani</em> using droplet inoculation method, and chlorosis around the inoculums appeared within 24 h post-inoculation (hpi now onwards) followed by the development of typical blight symptoms at 72 hpi (<a reveal-id="dsx003-F1" data-open="dsx003-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Leaves were succumbed to the disease within 120 hpi. To confirm the disease establishment and to decide on the time of sampling for RNA preparation we did light microscopic, SEM and multiple biochemical assays, followed by analysis of the expression level of stress-related marker genes. Microscopic studies have confirmed fungal invasion through stomatal openings (<a reveal-id="dsx003-F1" data-open="dsx003-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). Assays for detection of callose, nitric oxide (NO) and reactive oxygen species (ROS) accretion indicated that all three showed maximum accretion from 48 to 96 hpi with a peak at 72 hpi (<a reveal-id="dsx003-F1" data-open="dsx003-F1" class="link link-reveal link-table xref-fig">Fig. 1C–E</a>). Further analysis of expression level of pathogenesis-related PR1b and Chitinase genes (<a reveal-id="dsx003-F1" data-open="dsx003-F1" class="link link-reveal link-table xref-fig">Fig. 1F</a>) also showed significant up-regulation occurred during this time. Positive results in biochemical assays for callose deposition, ROS and NO generation together with up-regulated expression of PR1b and chitinase confirmed disease establishment and significant progression within 72 hpi. Hence, RNA was isolated from tomato leaves infected for ∼72 h for further analyses.</p>                        <a id="88473786" scrollto-destination="88473786"></a>
<div id="dsx003-F1" data-id="dsx003-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f1.png?Expires=2147483647&Signature=3isVHcJLHbq4TURLglQ-dy-DNtU2~q85uLao7mFW2Yiu8EeED9E-S9dDMm9L88NBpktm-M5Y3gwSf5WuXxhLccwgL~hBtGA41VYlpUhazugLcj-fLP8oHQz2LkC1Sjy5UXehiihNJ169eCuMK1VYYZeVtnI-NN4BvueF~uNGiVBf-Bt-z24VJdjw5~0wRPOtwgV96zGniA2h0XoD5bYO1I4kJ~MaeoM1GUixiqP9pj-Mq6fNp3DI7303m0WAooAZZXs~CjlrkjccThFT6-32Nv41ZQVr5sUAemGfE1yYtCe1URj6NC95gfy85vUE2OdnY58rEgrwNuBZXCOoa1q1Ig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Symptoms in deliberately infected tomato leaves, and characterization of the disease progression. (A) Manifestation of different stages of disease development during Alternaria infection. (B) Trypan blue staining displaying hyphal invasion (red arrow) through stomata (yellow arrow) into the leaf (left panel), right panel shows SEM of infected leaf surface confirming Alternaria hyphal (red arrow) invasion through stomatal opening (yellow arrow). (C) Detection of callose deposition in infected region. Control (mock inoculated, left panel) and infected (right panel) leaf fluorescence images after staining for the detection of callose deposition, showing accumulation of callose indicated by green fluorescence (black arrow) around hyphal mat, as a response to early plant defence. (D) Visualization of nitric oxide (yellow arrow) production at infection site (white arrow) by diaminofluorescein diacetate (DAF-2DA) staining. (E) Assays for detection of time-dependent accumulation of ROS at infection site. Brown coloration around the infection site resulted from DAB staining confirming ROS generation. Production of maximum ROS at 72 hpi is apparent. (F) Quantitative real-time PCR data showing expression level of selected marker genes for biotic stress. Both PR1 and chitinase transcript levels were upregulated compared with the control. Expression level of EF1α was used for normalization. *Significantly changed." path-from-xml="dsx003f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473786/dsx003f1.tif" path-from-xml="dsx003f1.tif" target="_blank">View large</a><a section="88473786" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f1.png?Expires=2147483647&Signature=mtp8EC6ZuFHI4lupGO6PiaWqnbZdlhl6UEE1iwYISyuSns90OExGWAEyUqh1Wj84X5UGFF2wLgfd3IVIpbhsxsV4H0DhXBF6ISELmaevxdw-OdDggN3-0Hz0WVpxi3jj0oEJQbMo-syaS~GD8yAqv1ofNFmph5EEr2LDkiBWwEhPghhR5qY8P~2134sEiwioE6jIj8nenaLFEsHaDbYwQBuUygUnxMeWS9TWD71EyR1Z~~lCtRj9Poe6zTien5xi7tabnVhLxzfV2t-w2QgKYH~PfsLbBTsZgQExVqs78s~qHfeYbG~KYSU~M05fXm~XQaqeJN~YnYWAwLHAG8gekw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473786&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Symptoms in deliberately infected tomato leaves, and characterization of the disease progression. (A) Manifestation of different stages of disease development during <em>Alternaria</em> infection. (B) Trypan blue staining displaying hyphal invasion (red arrow) through stomata (yellow arrow) into the leaf (left panel), right panel shows SEM of infected leaf surface confirming <em>Alternaria</em> hyphal (red arrow) invasion through stomatal opening (yellow arrow). (C) Detection of callose deposition in infected region. Control (mock inoculated, left panel) and infected (right panel) leaf fluorescence images after staining for the detection of callose deposition, showing accumulation of callose indicated by green fluorescence (black arrow) around hyphal mat, as a response to early plant defence. (D) Visualization of nitric oxide (yellow arrow) production at infection site (white arrow) by diaminofluorescein diacetate (DAF-2DA) staining. (E) Assays for detection of time-dependent accumulation of ROS at infection site. Brown coloration around the infection site resulted from DAB staining confirming ROS generation. Production of maximum ROS at 72 hpi is apparent. (F) Quantitative real-time PCR data showing expression level of selected marker genes for biotic stress. Both PR1 and chitinase transcript levels were upregulated compared with the control. Expression level of EF1α was used for normalization. *Significantly changed.</p></div></div><div content-id="dsx003-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f1.png?Expires=2147483647&Signature=3isVHcJLHbq4TURLglQ-dy-DNtU2~q85uLao7mFW2Yiu8EeED9E-S9dDMm9L88NBpktm-M5Y3gwSf5WuXxhLccwgL~hBtGA41VYlpUhazugLcj-fLP8oHQz2LkC1Sjy5UXehiihNJ169eCuMK1VYYZeVtnI-NN4BvueF~uNGiVBf-Bt-z24VJdjw5~0wRPOtwgV96zGniA2h0XoD5bYO1I4kJ~MaeoM1GUixiqP9pj-Mq6fNp3DI7303m0WAooAZZXs~CjlrkjccThFT6-32Nv41ZQVr5sUAemGfE1yYtCe1URj6NC95gfy85vUE2OdnY58rEgrwNuBZXCOoa1q1Ig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Symptoms in deliberately infected tomato leaves, and characterization of the disease progression. (A) Manifestation of different stages of disease development during Alternaria infection. (B) Trypan blue staining displaying hyphal invasion (red arrow) through stomata (yellow arrow) into the leaf (left panel), right panel shows SEM of infected leaf surface confirming Alternaria hyphal (red arrow) invasion through stomatal opening (yellow arrow). (C) Detection of callose deposition in infected region. Control (mock inoculated, left panel) and infected (right panel) leaf fluorescence images after staining for the detection of callose deposition, showing accumulation of callose indicated by green fluorescence (black arrow) around hyphal mat, as a response to early plant defence. (D) Visualization of nitric oxide (yellow arrow) production at infection site (white arrow) by diaminofluorescein diacetate (DAF-2DA) staining. (E) Assays for detection of time-dependent accumulation of ROS at infection site. Brown coloration around the infection site resulted from DAB staining confirming ROS generation. Production of maximum ROS at 72 hpi is apparent. (F) Quantitative real-time PCR data showing expression level of selected marker genes for biotic stress. Both PR1 and chitinase transcript levels were upregulated compared with the control. Expression level of EF1α was used for normalization. *Significantly changed." path-from-xml="dsx003f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473786/dsx003f1.tif" path-from-xml="dsx003f1.tif" target="_blank">View large</a><a section="88473786" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f1.png?Expires=2147483647&Signature=mtp8EC6ZuFHI4lupGO6PiaWqnbZdlhl6UEE1iwYISyuSns90OExGWAEyUqh1Wj84X5UGFF2wLgfd3IVIpbhsxsV4H0DhXBF6ISELmaevxdw-OdDggN3-0Hz0WVpxi3jj0oEJQbMo-syaS~GD8yAqv1ofNFmph5EEr2LDkiBWwEhPghhR5qY8P~2134sEiwioE6jIj8nenaLFEsHaDbYwQBuUygUnxMeWS9TWD71EyR1Z~~lCtRj9Poe6zTien5xi7tabnVhLxzfV2t-w2QgKYH~PfsLbBTsZgQExVqs78s~qHfeYbG~KYSU~M05fXm~XQaqeJN~YnYWAwLHAG8gekw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473786&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Symptoms in deliberately infected tomato leaves, and characterization of the disease progression. (A) Manifestation of different stages of disease development during <em>Alternaria</em> infection. (B) Trypan blue staining displaying hyphal invasion (red arrow) through stomata (yellow arrow) into the leaf (left panel), right panel shows SEM of infected leaf surface confirming <em>Alternaria</em> hyphal (red arrow) invasion through stomatal opening (yellow arrow). (C) Detection of callose deposition in infected region. Control (mock inoculated, left panel) and infected (right panel) leaf fluorescence images after staining for the detection of callose deposition, showing accumulation of callose indicated by green fluorescence (black arrow) around hyphal mat, as a response to early plant defence. (D) Visualization of nitric oxide (yellow arrow) production at infection site (white arrow) by diaminofluorescein diacetate (DAF-2DA) staining. (E) Assays for detection of time-dependent accumulation of ROS at infection site. Brown coloration around the infection site resulted from DAB staining confirming ROS generation. Production of maximum ROS at 72 hpi is apparent. (F) Quantitative real-time PCR data showing expression level of selected marker genes for biotic stress. Both PR1 and chitinase transcript levels were upregulated compared with the control. Expression level of EF1α was used for normalization. *Significantly changed.</p></div></div>                            <h3 scrollto-destination=88473787 id="88473787" class="section-title" >3.2. Differential expression of diverse sRNA population is associated with <em>Alternaria</em>-infection in tomato</h3>
<p>Only high-quality RNAs devoid of detectable fungal RNA contamination were used for library construction. Four small RNA libraries were constructed and sequenced using NGS system to investigate the expression of small RNAs in tomato upon <em>Alternaria</em> stress. A total of 18.5, 11, 13.4 and 9.8 million reads ranging in size from 16 to 36 nt were retrieved from Control 1, Control 2, Infected 1 and Infected 2 libraries, respectively. After removing low-quality reads and adapter trimming, the remaining reads were size filtered from 18 to 25 nt to enrich the sample with reads corresponding to the size of typical miRNAs. These filtered sequences represented 1,009,952 (Control 1), 1,163,133 (Control 2), 636,784 (Infected 1) and 218,018 (Infected 2) unique sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p><p>The small RNA length distribution among the four libraries showed maximum reads in the size range of 21–24 nt (65.86 and 75.17% for total and unique reads, respectively, in the average of four samples) which corresponds to typical dicer-derived products (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). In total reads 24 nt sRNAs dominated the sRNA transcriptome in a control plant which is consistent with the typical size distribution of sRNAs found in other plants.<a reveal-id="dsx003-B4" data-open="dsx003-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx003-B5" data-open="dsx003-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> However, in infected plants, total reads of 21 nt sRNAs increased (average 16.4 vs. 23.5% in control vs. infected), and the 24-nt population decreased markedly (average 24.8 vs. 14.5% in control vs. infected) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>A). Among the unique reads, the majority of sRNAs were 24 nt in length (39.08%, average of four samples) followed by 23 nt, however, 21 and 22-nt long reads were comparatively equally distributed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>B). Distribution of 24 vs. 21 nt unique reads in control vs. infected plants also followed the similar pattern as in total reads. In addition, the 21 and 22 nt class sRNAs showed the highest redundancy, whereas the 24-nt class showed the lowest redundancy (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>A and B).</p>                            <h3 scrollto-destination=88473790 id="88473790" class="section-title" >3.3. Both conserved and species-specific microRNAs are <em>Alternaria</em>-stress responsive in tomato</h3>
                            <h4 scrollto-destination=88473791 id="88473791" class="section-title" >3.3.1. Profiling of conserved and species-specific miRNAs</h4>
<p>The cleaned and size fractionated reads acquired above were used in BlastN search against a collection of all the annotated plant miRNAs, obtained from multiple databases comprising miRBase (Release 21), TFGD and TGRD. On the basis of sequence similarity (minimum alignment length 18, maximum two mismatches and no gap), our analysis revealed 181 known miRNAs, which belonged to 121 miRNA families. Most of the identified miRNAs belonged to Solanaceae-specific families (75/121 families). Others (24/121 families) were conserved miRNAs for a plant, while few were reported from a specific plant species only (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p><p>We analysed the identified families and found that they included a varying number of miRNA members. All miRNA families with at least five raw read counts for a member in at least one sample have been listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. Among the conserved miRNAs detected, the miR156 family was the largest, having seven members that were distinguished by the specific differences in their nucleotide sequences, followed by other abundant miRNAs such as miR166, miR171and miR482 families comprising of six members each. MiR167 included five members and miR319, miR396, miR9471 and miR393 contained four members each. Of the remaining 112 miRNA families, 2 families included 3 members, 19 families contained 2 members each and rest 91 miRNA families were each represented by a single member only (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p><p>The frequencies of miRNA reads in a family varied from 1 (miR5658, miR4414, miR7717, etc.) to 1,01,449 (miR166), indicating that miRNA families are expressed in a widely variable frequency within the tissue. Of all the identified miRNA families, 10 families were represented by more than 1,000 normalized reads in the tomato dataset (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In addition to miR166, miR159 (50,644 reads) and miR398 (13,132 reads) were among the major abundant families in the library. On average in a control tomato plant about 6.62% of total miRNAs are represented by highly abundant family members. On the contrary, less than 10 reads were detected for species specific and many solanaceae-specific miRNAs such as miR9472, miR8032, miR5083 and miR1873, etc. (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Analysis of the sequenced data also showed that the relative abundance of members of the miRNA families varied greatly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>), suggesting differential expression from different miRNA genes of the same family in tomato contributed to the final number of a mature miRNA produced in the tissue.</p>                            <h4 scrollto-destination=88473795 id="88473795" class="section-title" >3.3.2. Differential expression analysis of conserved and species-specific miRNAs</h4>
<p>miRNAs with at least five raw read counts from two equivalent libraries were selected for further analysis to determine differentially expressed miRNAs between <em>Alternaria</em>-treated and mock-treated samples using DEseq. Out of 181 identified miRNAs, 100 were differentially expressed. Next, to identify miRNAs that were notably changed in expression level after infection, we applied another selection criterion of log<sub>2</sub>FC to be at least ±1. This stringent selection yielded 67 differentially expressed miRNAs, and that the majority of them (44 out of 67) were down-regulated (<a reveal-id="dsx003-F2" data-open="dsx003-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Among the differentially regulated miRNAs, sly-miR9524, sly-miR397 and sly-miR9513 were highly down-regulated with a log<sub>2</sub>FC values of −4.99 (<em>P</em> = 0.04), −4.39 (<em>P</em> = 0.01) and −4.08 (<em>P</em> = 0.02), respectively, and sly-miR482a, sly-miR8175 and sly-miR6300 were highly up-regulated with log<sub>2</sub>FC values of 3.13 (<em>P</em> = 0.04), 3.28 (<em>P</em> =0.05) and 2.65 (<em>P</em>= 0.01), respectively (<a reveal-id="dsx003-F2" data-open="dsx003-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>).</p>                        <a id="88473797" scrollto-destination="88473797"></a>
<div id="dsx003-F2" data-id="dsx003-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f2.png?Expires=2147483647&Signature=wNo1WHdUAS4hEUgE7dRYGihu7RE-i7DIHK9IV0RLSsPoO8QKLoYHfhJMIBl9mFC82rd4NJXbBLZnP0t9tFf5wED9AJ4PwY63W~s~XnI3aiFQOm8jTDttXC5xAQA~zpU~35LbLJQNSb4tJAxEr3mPOlwDLHFEjB5bYsiS3D5DSJB4n7TkszDzRlto4WBRzuACVP9Fej9bgHcZk7oNupfZWmf5jPccrvNRzXrKP-26tWTKxWE~XaBXQPLaR6VIipm3YeCFqCCdLmbMO3hFJ9cXU-zSYhSjRyMMFu9T70hRQb-~zG8ve862ib38TTkXTd9IZsAIz-BSBdqu-l-vV69vFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Differentially expressed miRNAs between Alternaria infected and mock-treated tomato plants. (A) Scatter plot of miRNA expression pattern. Log2 value of read count of mock treated and Alternaria-infected samples are plotted. Green dots (grey dots in print) represent down-regulated, red circles (grey circles in print) indicate upregulated and black dots represent unchanged miRNAs. (B) Bar plot showing expression pattern of highly regulated miRNAs having log2 fold change at least ±2. Majority of the miRNAs were found to be down-regulated." path-from-xml="dsx003f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473797/dsx003f2.tif" path-from-xml="dsx003f2.tif" target="_blank">View large</a><a section="88473797" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f2.png?Expires=2147483647&Signature=zHcBfbg43KpL6rm-aKg35KH8jMJ5AucU3Wp3EiQTWrV8YaHRyMRaiTINBjBx49mnqST7vhcomECbLJioE2vTy4FJDwhti6c03NXOoMuoCt1Num~opl1f2sSxjtYbJy8HiHsF6MaO3o7wfEYECUuhB5cAdW8210lwMb~R5sZ7hmG-ScrU-4GDkS9AaB0WyOYfpaZJFppRUBeU9W7y3hFHEBGkg8e7OncaGG71-RreaQtAMmZUW34xIKu7jb2o0fat076tt3eh-h1lMH0eZlClXF6Czyc5C3U8hE70JJHOLViAXxtYw~WDjbotI-07MmK3JXf3rId-t2kTxVQM4NtMOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473797&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differentially expressed miRNAs between <em>Alternaria</em> infected and mock-treated tomato plants. (A) Scatter plot of miRNA expression pattern. Log<sub>2</sub> value of read count of mock treated and <em>Alternaria</em>-infected samples are plotted. Green dots (grey dots in print) represent down-regulated, red circles (grey circles in print) indicate upregulated and black dots represent unchanged miRNAs. (B) Bar plot showing expression pattern of highly regulated miRNAs having log<sub>2</sub> fold change at least ±2. Majority of the miRNAs were found to be down-regulated.</p></div></div><div content-id="dsx003-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f2.png?Expires=2147483647&Signature=wNo1WHdUAS4hEUgE7dRYGihu7RE-i7DIHK9IV0RLSsPoO8QKLoYHfhJMIBl9mFC82rd4NJXbBLZnP0t9tFf5wED9AJ4PwY63W~s~XnI3aiFQOm8jTDttXC5xAQA~zpU~35LbLJQNSb4tJAxEr3mPOlwDLHFEjB5bYsiS3D5DSJB4n7TkszDzRlto4WBRzuACVP9Fej9bgHcZk7oNupfZWmf5jPccrvNRzXrKP-26tWTKxWE~XaBXQPLaR6VIipm3YeCFqCCdLmbMO3hFJ9cXU-zSYhSjRyMMFu9T70hRQb-~zG8ve862ib38TTkXTd9IZsAIz-BSBdqu-l-vV69vFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Differentially expressed miRNAs between Alternaria infected and mock-treated tomato plants. (A) Scatter plot of miRNA expression pattern. Log2 value of read count of mock treated and Alternaria-infected samples are plotted. Green dots (grey dots in print) represent down-regulated, red circles (grey circles in print) indicate upregulated and black dots represent unchanged miRNAs. (B) Bar plot showing expression pattern of highly regulated miRNAs having log2 fold change at least ±2. Majority of the miRNAs were found to be down-regulated." path-from-xml="dsx003f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473797/dsx003f2.tif" path-from-xml="dsx003f2.tif" target="_blank">View large</a><a section="88473797" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f2.png?Expires=2147483647&Signature=zHcBfbg43KpL6rm-aKg35KH8jMJ5AucU3Wp3EiQTWrV8YaHRyMRaiTINBjBx49mnqST7vhcomECbLJioE2vTy4FJDwhti6c03NXOoMuoCt1Num~opl1f2sSxjtYbJy8HiHsF6MaO3o7wfEYECUuhB5cAdW8210lwMb~R5sZ7hmG-ScrU-4GDkS9AaB0WyOYfpaZJFppRUBeU9W7y3hFHEBGkg8e7OncaGG71-RreaQtAMmZUW34xIKu7jb2o0fat076tt3eh-h1lMH0eZlClXF6Czyc5C3U8hE70JJHOLViAXxtYw~WDjbotI-07MmK3JXf3rId-t2kTxVQM4NtMOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473797&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differentially expressed miRNAs between <em>Alternaria</em> infected and mock-treated tomato plants. (A) Scatter plot of miRNA expression pattern. Log<sub>2</sub> value of read count of mock treated and <em>Alternaria</em>-infected samples are plotted. Green dots (grey dots in print) represent down-regulated, red circles (grey circles in print) indicate upregulated and black dots represent unchanged miRNAs. (B) Bar plot showing expression pattern of highly regulated miRNAs having log<sub>2</sub> fold change at least ±2. Majority of the miRNAs were found to be down-regulated.</p></div></div>                            <h3 scrollto-destination=88473798 id="88473798" class="section-title" >3.4. Several novel and previously uncharacterized miRNAs are also differentially expressed during <em>Alternaria</em>-infection</h3>
                            <h4 scrollto-destination=88473799 id="88473799" class="section-title" >3.4.1. Profiling of novel miRNAs</h4>
<p>We next inquired whether this cultivar of tomato also expressed any novel or previously uncharacterized miRNAs. Approximately 2,582,051 unannotated sequences (accumulated from four samples), not matching to known miRNAs or any other ncRNA such as tRNA, rRNA, snRNA, snoRNA and repeats, were aligned to the tomato genome (Sol Genomics Network). The reads aligned to tomato genome were used as input in the miRNA prediction software Mireap to obtain the putative novel miRNAs with their predicted hairpin precursors. To further increase the confidence of claiming the identified sequences as novel miRNAs, we applied the three Ambros criteria with slight modification. In addition, a sequence with minimum five read counts in at least one sample was taken for novel miRNA analysis. We, thus, recognized 60 potential novel miRNA candidates from all 4 samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). To further confirm precursor stem loop structure of these predicted novel miRNAs, the putative precursor sequence was subjected to folding analysis using RNAfold software. All these novel miRNA candidates’ precursor sequences were folded into a hairpin-like structure apparently similar to other known miRNAs (<a reveal-id="dsx003-F3" data-open="dsx003-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). Moreover, the miRNA* of eight miRNAs were readily detected in our dataset. The precursor sequence of these new miRNA candidates varied from 63 to 101 nt in length and the MFE of precursor hairpins ranged from −25 to −50 kcal/mol. Of the 60 new miRNAs 52 are situated in the intergenic region, and 8 are located in the intronic regions of genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). The mature novel miRNAs could be found in both 5′ and 3′ arms of the predicted precursors and their read number varied from 1 to 20,631. However, some of these candidates could not be detected in all samples.</p>                        <a id="88473801" scrollto-destination="88473801"></a>
<div id="dsx003-F3" data-id="dsx003-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f3.png?Expires=2147483647&Signature=OSUYHeAZ1jIdRcskAxryIA~X9~z6FInIdEUBzoYo~uT29OPWWD0XENz4MR69ak8Qy6d2R2L6MvfE8KuJYXmoKdPln9DmnelIFeh~agRdXYYik~XU~SeBkCLuxYrwPep3-8fo5ogiwv7qF2om54QWZCKSoxyRtiEIQvsFd1nqpEOZV9dOCWDV8~zKITHHqRKcDvUIjhN4K~qETOySmyMjd9kXrOXVgFVU1~M0CkpXBft6iBgQHpA83ZwT6Z-c2qI7z6Uz5RkoVW7wN98sLB-rJn9jUENdWt3WLFR0LGetacIBgJLIxKSB-oHbr-H7~nwTh71zN5ltiqdG0URMTmuGOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Predicted precursor structure and validation of expression of novel miRNAs. (A) Predicted secondary structures of the precursor of selected novel miRNAs as determined using RNAfold. Mature miRNA and miRNA star sequences—both detected by sequencing—are denoted in red (dark grey in print) and green (light grey in print), respectively. (B) Image showing expression pattern of three novel miRNAs as detected by northern blot hybridization analysis. (C) Poly A tailing mediated amplification and detection of selected novel miRNAs. (D) Semi-quantitative RT-PCR analysis for validation of expression of precursor sequence of two of the novel miRNAs. RT-dependent PCR samples are loaded in +RT lanes and −RT lanes contained samples of PCR performed with mock cDNA." path-from-xml="dsx003f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473801/dsx003f3.tif" path-from-xml="dsx003f3.tif" target="_blank">View large</a><a section="88473801" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f3.png?Expires=2147483647&Signature=QIQCHI08~nV5bfXXmlkhwPGcJ5GIwjnKQroCu0Vo-zynR0FUcVeIrN3X1HSb04U2jgrcLNhSI-oaKn8T7TBqdLphs3BSvcEy67WM8mlM-IcNWzriWlPfaBfn95MuFF0CJhhniM6WGucevREICIt3fdlzvyvDzXJhKAK9gQqvQ29ICVyDqnQ3T5wphVtEuS4LbvejLNroYTfl0udbeeTgJPRVupLuasFIBOcvdfIyI74Qtmzzl42v5nKwmwJODbSeKfnxs3DCP~Zl4ZZUBCyEe7bOaX~l63zq0RGnzFxQbm1TViHSdDvhCXGIeVyWvOdkZHt0Z979MJFMjErxRuMhJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473801&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Predicted precursor structure and validation of expression of novel miRNAs. (A) Predicted secondary structures of the precursor of selected novel miRNAs as determined using RNAfold. Mature miRNA and miRNA star sequences—both detected by sequencing—are denoted in red (dark grey in print) and green (light grey in print), respectively. (B) Image showing expression pattern of three novel miRNAs as detected by northern blot hybridization analysis. (C) Poly A tailing mediated amplification and detection of selected novel miRNAs. (D) Semi-quantitative RT-PCR analysis for validation of expression of precursor sequence of two of the novel miRNAs. RT-dependent PCR samples are loaded in +RT lanes and −RT lanes contained samples of PCR performed with mock cDNA.</p></div></div><div content-id="dsx003-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f3.png?Expires=2147483647&Signature=OSUYHeAZ1jIdRcskAxryIA~X9~z6FInIdEUBzoYo~uT29OPWWD0XENz4MR69ak8Qy6d2R2L6MvfE8KuJYXmoKdPln9DmnelIFeh~agRdXYYik~XU~SeBkCLuxYrwPep3-8fo5ogiwv7qF2om54QWZCKSoxyRtiEIQvsFd1nqpEOZV9dOCWDV8~zKITHHqRKcDvUIjhN4K~qETOySmyMjd9kXrOXVgFVU1~M0CkpXBft6iBgQHpA83ZwT6Z-c2qI7z6Uz5RkoVW7wN98sLB-rJn9jUENdWt3WLFR0LGetacIBgJLIxKSB-oHbr-H7~nwTh71zN5ltiqdG0URMTmuGOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Predicted precursor structure and validation of expression of novel miRNAs. (A) Predicted secondary structures of the precursor of selected novel miRNAs as determined using RNAfold. Mature miRNA and miRNA star sequences—both detected by sequencing—are denoted in red (dark grey in print) and green (light grey in print), respectively. (B) Image showing expression pattern of three novel miRNAs as detected by northern blot hybridization analysis. (C) Poly A tailing mediated amplification and detection of selected novel miRNAs. (D) Semi-quantitative RT-PCR analysis for validation of expression of precursor sequence of two of the novel miRNAs. RT-dependent PCR samples are loaded in +RT lanes and −RT lanes contained samples of PCR performed with mock cDNA." path-from-xml="dsx003f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473801/dsx003f3.tif" path-from-xml="dsx003f3.tif" target="_blank">View large</a><a section="88473801" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f3.png?Expires=2147483647&Signature=QIQCHI08~nV5bfXXmlkhwPGcJ5GIwjnKQroCu0Vo-zynR0FUcVeIrN3X1HSb04U2jgrcLNhSI-oaKn8T7TBqdLphs3BSvcEy67WM8mlM-IcNWzriWlPfaBfn95MuFF0CJhhniM6WGucevREICIt3fdlzvyvDzXJhKAK9gQqvQ29ICVyDqnQ3T5wphVtEuS4LbvejLNroYTfl0udbeeTgJPRVupLuasFIBOcvdfIyI74Qtmzzl42v5nKwmwJODbSeKfnxs3DCP~Zl4ZZUBCyEe7bOaX~l63zq0RGnzFxQbm1TViHSdDvhCXGIeVyWvOdkZHt0Z979MJFMjErxRuMhJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473801&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Predicted precursor structure and validation of expression of novel miRNAs. (A) Predicted secondary structures of the precursor of selected novel miRNAs as determined using RNAfold. Mature miRNA and miRNA star sequences—both detected by sequencing—are denoted in red (dark grey in print) and green (light grey in print), respectively. (B) Image showing expression pattern of three novel miRNAs as detected by northern blot hybridization analysis. (C) Poly A tailing mediated amplification and detection of selected novel miRNAs. (D) Semi-quantitative RT-PCR analysis for validation of expression of precursor sequence of two of the novel miRNAs. RT-dependent PCR samples are loaded in +RT lanes and −RT lanes contained samples of PCR performed with mock cDNA.</p></div></div>                            <h4 scrollto-destination=88473802 id="88473802" class="section-title" >3.4.2. Differential expression analysis of novel miRNAs</h4>
<p>Differential expression analysis using DESeq was performed to determine the differential expression level of predicted novel miRNAs. Among the differentially expressed novel miRNAs, 52 miRNAs had log<sub>2</sub>FC at least ±1. Among these, 39 and 13 were down and up-regulated, respectively, and only 7 novel miRNAs were found to be significantly (<em>P</em> ≤ 0.05) differentially expressed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Both known and novel miRNAs expression data showed a majority of the differentially expressed miRNAs to be down-regulated as a response to <em>Alternaria-</em>stress.</p>                            <h3 scrollto-destination=88473804 id="88473804" class="section-title" >3.5. Differential expressions of both known and novel miRNAs could be validated using alternative techniques</h3>
                            <h4 scrollto-destination=88473805 id="88473805" class="section-title" >3.5.1. Validation of expression of known miRNAs</h4>
<p>To validate the expression pattern of selected miRNAs, having at least 10 reads in any sample, we performed Poly A tailing mediated miRNA-amplification and northern blot analysis (<a reveal-id="dsx003-F4" data-open="dsx003-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The highly up-regulated miRNAs, sly-miR482b, sly-miR168, sly-miR6300, sly-miR156f, sly-miR403-3p and sly-miR9519, and highly down-regulated miRNAs, sly-miR408b-3p, sly-miR398a-3p, sly-miR397 and sly-miR9513 were selected for testing along with sly-miR156a/b, sly-miR159, sly-miR4376, sly-miR166 and sly-miR6024 which were not significantly differentially expressed.</p>                        <a id="88473807" scrollto-destination="88473807"></a>
<div id="dsx003-F4" data-id="dsx003-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f4.png?Expires=2147483647&Signature=Q99TxGPsDfyDFMYcoNnKnUDHP-Dxpn0dGrGEEEy6qmGULrPKVSrLflCP28ATAee2Vz83Sg0eEw9uD8OgSwN9uxcwlwezoLHDe4O5b0oM~hN~xp7mIwFDspzPzQEdrHGAQX4THN86d67sNOREXC0nrIXl7EyPoq7fsSXG3gmorKAu1VcsFt6dUhbs5ycPsBOszp4Rd00imNdIskxRlmZMeBVs2G~8fYNfomDpVrSb0kZ6S~qhEx5ZbGrb6SEaEnLdx~BY8q2B2sgmyTwT5mwYu2kdGCHw8JKCnvXSzB~YBDeCWBUeA1q3MPdsrj2QWTg8xV5Lm8LMuutF6w1FhlJwCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of expression level of miRNAs using alternative approaches. (A) PAGE photograph of poly A tailing mediated RT-PCR analysis of selected known miRNAs. U6 snRNA expression was monitored as normalization factor. (B) Images of northern blot hybridization analysis of selected known miRNAs. (C) Quantification of band intensity obtained (using ImageQuant ID GEL quantification software) in northern blot analysis (n = 2). Data presented as log2 fold change (normalized intensity of infected/control)." path-from-xml="dsx003f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473807/dsx003f4.tif" path-from-xml="dsx003f4.tif" target="_blank">View large</a><a section="88473807" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f4.png?Expires=2147483647&Signature=gX52XFJjtLDRiA6EitgDNjMUgtQCjj3~eCMxOzi5Z5uu5XM-8m8CdYZ1vxSmZNIa9c5h3bQkws4pT3Bbjl4IiPNJVEH3hKuf4wuLo6XkCiqBfA4ISRLN8bvASiDokZEQyo5fratpTAUC7KxV7wcHONTW~LGia7K1MeYPSH70Jj48xR8NTAPTclAQ~Ate0jVrVr6h5I0M0koaA214yndxDQPxa7fDfWt2DbsWP07~Gwjm-RlxHpaZIg6U1FZycyxb7OZw6km5HWk1x~panI1qmOgANtGq27t7yCIcoUAeKitLnGhGBR0Lj9yffU7UocfzoSjwEbzt5OUpzmTPJbq0pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473807&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of expression level of miRNAs using alternative approaches. (A) PAGE photograph of poly A tailing mediated RT-PCR analysis of selected known miRNAs. U6 snRNA expression was monitored as normalization factor. (B) Images of northern blot hybridization analysis of selected known miRNAs. (C) Quantification of band intensity obtained (using ImageQuant ID GEL quantification software) in northern blot analysis (<em>n</em> = 2). Data presented as log<sub>2</sub> fold change (normalized intensity of infected/control).</p></div></div><div content-id="dsx003-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f4.png?Expires=2147483647&Signature=Q99TxGPsDfyDFMYcoNnKnUDHP-Dxpn0dGrGEEEy6qmGULrPKVSrLflCP28ATAee2Vz83Sg0eEw9uD8OgSwN9uxcwlwezoLHDe4O5b0oM~hN~xp7mIwFDspzPzQEdrHGAQX4THN86d67sNOREXC0nrIXl7EyPoq7fsSXG3gmorKAu1VcsFt6dUhbs5ycPsBOszp4Rd00imNdIskxRlmZMeBVs2G~8fYNfomDpVrSb0kZ6S~qhEx5ZbGrb6SEaEnLdx~BY8q2B2sgmyTwT5mwYu2kdGCHw8JKCnvXSzB~YBDeCWBUeA1q3MPdsrj2QWTg8xV5Lm8LMuutF6w1FhlJwCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of expression level of miRNAs using alternative approaches. (A) PAGE photograph of poly A tailing mediated RT-PCR analysis of selected known miRNAs. U6 snRNA expression was monitored as normalization factor. (B) Images of northern blot hybridization analysis of selected known miRNAs. (C) Quantification of band intensity obtained (using ImageQuant ID GEL quantification software) in northern blot analysis (n = 2). Data presented as log2 fold change (normalized intensity of infected/control)." path-from-xml="dsx003f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473807/dsx003f4.tif" path-from-xml="dsx003f4.tif" target="_blank">View large</a><a section="88473807" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f4.png?Expires=2147483647&Signature=gX52XFJjtLDRiA6EitgDNjMUgtQCjj3~eCMxOzi5Z5uu5XM-8m8CdYZ1vxSmZNIa9c5h3bQkws4pT3Bbjl4IiPNJVEH3hKuf4wuLo6XkCiqBfA4ISRLN8bvASiDokZEQyo5fratpTAUC7KxV7wcHONTW~LGia7K1MeYPSH70Jj48xR8NTAPTclAQ~Ate0jVrVr6h5I0M0koaA214yndxDQPxa7fDfWt2DbsWP07~Gwjm-RlxHpaZIg6U1FZycyxb7OZw6km5HWk1x~panI1qmOgANtGq27t7yCIcoUAeKitLnGhGBR0Lj9yffU7UocfzoSjwEbzt5OUpzmTPJbq0pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473807&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of expression level of miRNAs using alternative approaches. (A) PAGE photograph of poly A tailing mediated RT-PCR analysis of selected known miRNAs. U6 snRNA expression was monitored as normalization factor. (B) Images of northern blot hybridization analysis of selected known miRNAs. (C) Quantification of band intensity obtained (using ImageQuant ID GEL quantification software) in northern blot analysis (<em>n</em> = 2). Data presented as log<sub>2</sub> fold change (normalized intensity of infected/control).</p></div></div><p>PolyA tailing semiquantitative PCR (<em>n</em> = 3) analysis showed that out of 11 miRNAs tested 8 were differentially expressed (<a reveal-id="dsx003-F4" data-open="dsx003-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Northern blot analyses (<em>n</em> = 2) (<a reveal-id="dsx003-F4" data-open="dsx003-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>) indicated that expression of most of the miRNAs tested (10/12 miRNAs) matched with the trend of regulation observed in NGS analysis, however, the extent of regulation differed (<a reveal-id="dsx003-F4" data-open="dsx003-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Noticeable difference in expressions of sly-miR156a/b, sly-miR397, sly-miR482b, sly-miR159, sly-miR168 and sly-miR156f were also observed. Two conserved miRNAs, miR159 and miR156a/b, which did not fulfil log<sub>2</sub>FC criteria (at least ±1) in NGS analysis showed considerable alteration in expression in Northern blot analysis.</p><p>Next, we investigated whether the expression level of certain miRNAs was also dependent on the infection progression. Samples from different time points post infection (8–72 hpi) were collected for Northern blot analysis. Visible difference in expression of miRNAs tested was apparent only at 72 hpi (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Expressions of miR168 and miR403 were upregulated, whereas the expression of miR397 was downregulated and the changes observed at 72 hpi are similar to NGS as well as to poly-A tailing mediated RT-PCR validation analyses. Thus the expression of these miRNAs was regulated only during the development of blight symptom.</p>                            <h4 scrollto-destination=88473810 id="88473810" class="section-title" >3.5.2. Validation of expression of novel miRNAs</h4>
<p>We selected four novel miRNAs (<a reveal-id="dsx003-F3" data-open="dsx003-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>) for validation of their expression. These miRNAs had at least 10 raw read counts, and miRNA* sequence of three of these miRNAs (sly-miRPR11, sly-miRPR26 and sly-miRPR29) could be detected in the NGS data. Using Northern blot analysis (<em>n</em> = 2), sly-miRPR1, sly-miRPR26 and sly-miRPR29 could be detected in both the samples (<a reveal-id="dsx003-F3" data-open="dsx003-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The novel miRNAs, sly-miRPR11, sly-miRPR26 and sly-miRPR29 with detectable star sequences, were also subjected to PolyA tailing mediated RT-PCR analysis for further validation. All three were readily detected in both the samples and sly-miRPR29 exhibited significant change in expression level during the infection (<a reveal-id="dsx003-F3" data-open="dsx003-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>) which is similar to the pattern observed in NGS analysis. Also, expression of probable precursor sequences of two of these novel miRNAs (sly-miRPR11 and sly-miRPR29) was confirmed by RT-PCR analysis performed with RNAs from both control and infected plants (<a reveal-id="dsx003-F3" data-open="dsx003-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>).</p>                            <h3 scrollto-destination=88473812 id="88473812" class="section-title" >3.6. Genes belonging to stress-related pathways are targeted by differentially expressed miRNAs</h3>
<p>miRNAs mediate their function in the cell by triggering cleavage or translational inhibition of specific target mRNAs, with which it has sequence complementarity. We searched two online databases psRNATarget and TFGD, to find predicted targets of 67 differentially expressed known miRNAs. The databases predicted 7,025 putative targets cumulatively (6,576 miRNA–mRNA pairs in psRNATarget and 1,055 pairs in TFGD, 606 common between the two). The discrepancy in the number of targets predicted by two softwares was due to the difference in the scoring matrix of the two databases. To better understand the group of genes targeted by the differentially expressed miRNAs during <em>Alternaria</em> stress, GO-based functional annotation and enrichment analysis of the predicted targets were performed utilizing Hypergeometric test applying Benjamini &amp; Hochberg FDR correction value ≤0.05, with all predicted targets. Under the molecular function category of the GO classification, genes showing kinase activity (GO:0016301), receptor activity (GO:0004872), transferase activity (GO:0016740), transporter activity (GO:0005215), ATPase activity (GO:0016887), catalytic activity (GO:0003824), oxidoreductase activity (GO:0016724), signal transducer activity (GO:0004871) were highly enriched as targets of the differentially expressed miRNAs. Under the biological process category genes involved in cell death (GO:0008219), RNA metabolic process (GO:0016070), tRNA processing (GO:0008033) were targeted by miRNAs and only thylakoid membrane related genes were found to be enriched for the cellular component category as targets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). In further analysis of the 606 target mRNAs simultaneously predicted by both software, no specific pathway was enriched, however, ‘biosynthesis of secondary metabolites’, ‘biosynthesis of amino acids’ and ‘carbon metabolism’ pathways are highlighted, as more genes of these pathways seem to be targeted. It is worth mentioning that some amino acids are precursors of phytohormones and it has been noted earlier that hormone biosynthesis/signalling pathways were affected during the infection.</p>                            <h3 scrollto-destination=88473814 id="88473814" class="section-title" >3.7. mRNA expression profiles in tomato in response to <em>Alternaria</em> stress</h3>
<p>In order to detect the alteration in expression of genes during <em>Alternaria-</em>stress RNA-seq analysis was performed for two tomato cDNA libraries, Control 1 and Infected 1, constructed with the same RNA preparation that has been used for small RNA transcriptomics. Sequencing generated 75.92 and 78.34 million reads, and after removing the low-quality reads (&lt;20 Phred score) and trimming off the adapter sequences, 70.27 and 72.56 million clean reads remained for Control 1 and Infected 1 samples, respectively. Mean read length obtained was 99 bp (maximum and minimum were 100 and 50 bp). The sequenced reads were aligned to the tomato reference genome (Sol Genomics Network), which showed 81.23 and 80.85% matched sequences for Control 1 and Infected 1 libraries, respectively. Next, these sequences were assigned to transcripts and a total of 23,011 and 22,997 assembled transcripts (identified by their respective gene ID) from Control 1 and Infected 1 samples, respectively, were obtained (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Finally, differential gene expression analysis revealed 5,450 genes showed at least 2-fold change in expression level (<em>P</em> ≤ 0.05). This analysis also revealed that relatively more genes were up-regulated (52.53%) compared with 47.46% genes which were down-regulated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). These DEGs were subjected to further analysis to identify the pathways they were representing.</p>                            <h3 scrollto-destination=88473816 id="88473816" class="section-title" >3.8. Validation of mRNA transcriptome analysis data</h3>
<p>To validate the NGS expression data, we randomly selected 18 differentially expressed mRNAs (log<sub>2</sub>FC at least ±1, <em>P</em> ≤ 0.05) and validated their relative expression levels with qRT-PCR, using gene-specific primers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). Data presented in <a reveal-id="dsx003-F5" data-open="dsx003-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a> show that 17 mRNAs were differentially expressed and the expression pattern matched with NGS data analysis, however, the extent of differential expression differed.</p>                        <a id="88473818" scrollto-destination="88473818"></a>
<div id="dsx003-F5" data-id="dsx003-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f5.png?Expires=2147483647&Signature=IL5nx3ZZ5gsGIlxF557LBJ5Q8SkXF1e0yh3CxrnbtEgjbeUlcZ3NLGzvz~I8AGzWMe3hEZYmHP4pGQCdpngWVkc~rhz6lsxgwIpU2j9LSOxD4Jdfi4e8Qjs24SlOpl9tQcn37VgbQh7hoXG2zfLFHHbBXvyzYTqObJ12DVpPx22GtlGP63aarxBt5l0DnWNUt01Bf88A670F-HhNyWtY7KpuPpxU3pEWR8UT~c0drxy2QwETHMgVYz82s8zMQe89hDuTdXGoK-VKOW4XrXifzRSXfHLGNHyCM7a~UnQ6Ntb2f3XeiGT8apWAI4UKHgMxkY17zOHnFST~S6xIB0VgvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation of expression level of selected genes. (A) Quantitative RT-PCR analysis data of randomly selected Alternaria-responsive 18 mRNAs. (B) Pearson correlation scatter plot of comparisons of differential expression level as measured by NGS analysis (log2FC) and experimentally validated data (log2FC) for mRNAs and miRNAs. ‘r’ indicates the Pearson correlation coefficient." path-from-xml="dsx003f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473818/dsx003f5.tif" path-from-xml="dsx003f5.tif" target="_blank">View large</a><a section="88473818" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f5.png?Expires=2147483647&Signature=H-mdC3qthlpCp8eI4U~d0oM7pOmLkHbb59xV4xIwBgPmm9RnNZYHPT7OknZIjWIdcckWyaawFKsk5rIxfqaYEHyHEMvNBgyKG29sZlE4YWhZLJl-BLmf2QdxyI8VSSP5tDhlxICioYQldpXNRR-pZkm2nwqV5xKakybmZvO22cxPoHQTjM85BVkE1G17pY4LfnVCsxUwYcb5an3H6TGofwWqXD39yFJGCFnRS6zK6VG~32URrAgoZJIVFKyVvTp5iuOg-yLXJm5WjLDm9U68XWcXlRDKMvTBnFo4drhY-gcbPkXF5QVltYym9MyABCMbdeLhIp8jnu5r4N1cxDEY~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473818&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation of expression level of selected genes. (A) Quantitative RT-PCR analysis data of randomly selected <em>Alternaria</em>-responsive 18 mRNAs. (B) Pearson correlation scatter plot of comparisons of differential expression level as measured by NGS analysis (log<sub>2</sub>FC) and experimentally validated data (log<sub>2</sub>FC) for mRNAs and miRNAs. ‘<em>r</em>’ indicates the Pearson correlation coefficient.</p></div></div><div content-id="dsx003-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f5.png?Expires=2147483647&Signature=IL5nx3ZZ5gsGIlxF557LBJ5Q8SkXF1e0yh3CxrnbtEgjbeUlcZ3NLGzvz~I8AGzWMe3hEZYmHP4pGQCdpngWVkc~rhz6lsxgwIpU2j9LSOxD4Jdfi4e8Qjs24SlOpl9tQcn37VgbQh7hoXG2zfLFHHbBXvyzYTqObJ12DVpPx22GtlGP63aarxBt5l0DnWNUt01Bf88A670F-HhNyWtY7KpuPpxU3pEWR8UT~c0drxy2QwETHMgVYz82s8zMQe89hDuTdXGoK-VKOW4XrXifzRSXfHLGNHyCM7a~UnQ6Ntb2f3XeiGT8apWAI4UKHgMxkY17zOHnFST~S6xIB0VgvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation of expression level of selected genes. (A) Quantitative RT-PCR analysis data of randomly selected Alternaria-responsive 18 mRNAs. (B) Pearson correlation scatter plot of comparisons of differential expression level as measured by NGS analysis (log2FC) and experimentally validated data (log2FC) for mRNAs and miRNAs. ‘r’ indicates the Pearson correlation coefficient." path-from-xml="dsx003f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473818/dsx003f5.tif" path-from-xml="dsx003f5.tif" target="_blank">View large</a><a section="88473818" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f5.png?Expires=2147483647&Signature=H-mdC3qthlpCp8eI4U~d0oM7pOmLkHbb59xV4xIwBgPmm9RnNZYHPT7OknZIjWIdcckWyaawFKsk5rIxfqaYEHyHEMvNBgyKG29sZlE4YWhZLJl-BLmf2QdxyI8VSSP5tDhlxICioYQldpXNRR-pZkm2nwqV5xKakybmZvO22cxPoHQTjM85BVkE1G17pY4LfnVCsxUwYcb5an3H6TGofwWqXD39yFJGCFnRS6zK6VG~32URrAgoZJIVFKyVvTp5iuOg-yLXJm5WjLDm9U68XWcXlRDKMvTBnFo4drhY-gcbPkXF5QVltYym9MyABCMbdeLhIp8jnu5r4N1cxDEY~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473818&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation of expression level of selected genes. (A) Quantitative RT-PCR analysis data of randomly selected <em>Alternaria</em>-responsive 18 mRNAs. (B) Pearson correlation scatter plot of comparisons of differential expression level as measured by NGS analysis (log<sub>2</sub>FC) and experimentally validated data (log<sub>2</sub>FC) for mRNAs and miRNAs. ‘<em>r</em>’ indicates the Pearson correlation coefficient.</p></div></div>                            <h3 scrollto-destination=88473819 id="88473819" class="section-title" >3.9. Correlation of NGS analysis and validation data</h3>
<p>The intensity of bands in miRNA Northern blots was determined using ImageQuant ID GEL quantification software, from which normalized fold change in expression of miRNAs was derived. Fold change values of mRNA expression from quantitative RT-PCR analysis was also collected. Correlation between the relative expression level detected by these validated data and NGS sequencing data were then calculated. We obtained a highly significant Pearson correlation coefficient of <em>r</em> = 0.84 (<a reveal-id="dsx003-F5" data-open="dsx003-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>), which strongly supported that our NGS transcriptome data were valid.</p>                            <h3 scrollto-destination=88473821 id="88473821" class="section-title" >3.10. Functional and pathway annotation of <em>Alternaria</em>–stress responsive transcripts</h3>
<p>To better understand the function of the DEGs (<em>P</em> ≤ 0.05), GO terms and pathway enrichment analyses (KEGG analysis) were performed. We found that 3,154 genes were involved with 17 major molecular functions, 62 genes were part of 3 different cellular components and 57 genes participated in 1 biological process (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>A). The GO terms predominantly enriched midst the regulated genes were related to molecular function including ‘catalytic activity’ (GO:0003824), ‘transferase activity’ (GO:0016740) ‘kinase activity’ (GO:0016301), ‘oxidoreductase activity’ (GO:0016491) represented by 1382, 539, 265 and 272 DEGs. Among the ‘kinase activity’ group majority belonged to ‘protein kinase activity’ (GO:0004672, 211 genes), and ‘transferase activity’ group had majority genes belonged to ‘transferring phosphorous-containing groups’ (GO:0016772, 283 genes). Other GO terms that were enriched including ‘transmembrane receptor protein kinase activity’ (GO:0019199), ‘transmembrane receptor activity’ (GO:0004888), ‘receptor activity’ (GO:0004872), ‘chitinase activity’ (GO:0004568), ‘lipoxygenase activity’ (GO:0016165), ‘oxygen binding’ (GO:0019825), ‘antioxidant activity’ (GO:0016209) are related to pathways involved in a stress response. Genes associated with ‘extracellular region’ (GO:0005576), ‘photosystem’ (GO:0009521) and ‘thylakoid’ (GO:0009579) were most significant (represented by 30, 18 and 9 genes, respectively) among the cellular component GO terms. In the GO biological process class, there was a higher number of DEGs annotated to ‘response to stimulus’ (GO:0050896, 63 genes), more specifically ‘response to biotic stimulus’ (GO:0009607, 14 genes).</p><p>To elucidate the exact biological processes the DEGs may participate in during <em>Alternaria</em> stress, we analysed the allotment of a DEG to a particular KEGG pathway. We were able to allot 5,080 genes in 334 different pathways. Next, we performed enrichment analysis, which revealed 24 important pathways were significantly (<em>P</em> ≤ 0.05) enriched in response to <em>Alternaria</em> stress (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>B). It was noticeable that genes associated with photosynthesis particularly photosynthesis–antenna proteins (PATH:ko00196) were significantly enriched among the DEGs, suggesting plant photosynthesis is affected during infection owing to the chlorosis and blight progression during the disease. In addition, genes involved in the biosynthesis of secondary metabolites (PATH: ko01110) and phenylpropanoids (PATH:ko00940), unsaturated fatty acids, zeatin, flavonoid, cutin suberin and wax, stilbenoid, diarylheptanoid and gingerol and steroids were enriched. Moreover, genes for metabolism of phenylalanine, porphyrin and chlorophyll, nitrogen, linoleic and alpha–linoleic acid and starch and sucrose were enriched. Also, DEGs involved in valine, leucine and isoleucine degradation, or apoptosis were enriched. And lastly, plant–pathogen interaction (PATH: ko04626) and plant hormone signal transduction (PATH: ko04075) pathways were enriched, signifying, substantial regulation of genes contributing in the modulation of plant hormone signalling and plant defence occurred during the infection.</p>                            <h3 scrollto-destination=88473824 id="88473824" class="section-title" >3.11. Differential expression of pathogenesis/disease development-related genes occur during <em>Alternaria</em>-stress</h3>
<p>A close inspection of genes belonging to a particular pathway that were differentially regulated during the infection revealed that genes, (i) involved in plant hormone biosynthesis/signal transduction for auxin, ethylene (ET), salicylic acid (SA), jasmonic acid (JA) and abscisic acid (ABA) were regulated, (ii) for ROS or NO generation/scavenging were significantly regulated and (iii) which may have direct role in stress response such as RLKs, NB–LRRs and WRKY transcription factors were also significantly regulated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>).</p><p>Usually, necrotrophic pathogens are benefitted from SA-mediated hypersensitive response due to added cell death, and in a resistance response, SA pathway is down regulated. Isochorismate synthase is a major gene in SA biosynthetic process, which is usually upregulated during a resistance response to a necrotrophic pathogen infection and systemic acquired resistance, however, it was downregulated (log<sub>2</sub>FC = −2.5, although not <em>P</em> significant) in our compatible interaction study. Concomitantly, several Phenylalanine ammonia–lyase (PAL) genes, of the SA–biosynthesis secondary pathway, were significantly upregulated together with the expression of pathogenesis-related PR-1, and PR-5 (log2FC= +6.19 and log2FC = +5.49, respectively) genes. These data support the view that SA-mediated defence pathway was suppressed in this compatible interaction. Although a role of PAL genes in this process is unclear, probably SA-mediated cell death would still occur as these genes’ expression was increased, which might benefit the fungus during this susceptible interaction.<a reveal-id="dsx003-B14" data-open="dsx003-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p><p>During necrotrophic pathogen infection, the plant’s defence response consists of synergistic activation of ET and JA pathways.<a reveal-id="dsx003-B15" data-open="dsx003-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> In line with this, the majority of the genes associated with these pathways were noticeably upregulated. Out of 106 genes associated with the ET pathway that were recognized, 76 were upregulated including 34 genes responsible for ET biosynthesis (1-aminocyclopropane-1-carboxylate synthase (ACS), 1-aminocyclopropane-1-carboxylate oxidase (ACO) and 43 genes (out of 62) responsible for ET signal transduction (ethylene-responsive transcription factors, ERFs). Moreover, 16 genes (among 23 genes detected) involved in jasmonic acid biosynthesis (Allene oxide cyclase and lipoxygenase) and signal transduction (Jasmonate ZIM domain, JAZ) were also found to be upregulated. Suggesting both ET and JA pathways were also activated as the host was continuously trying to resist the pathogen (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>).</p><p>The role of Auxin in a stress response has recently been appreciated, and the hormone’s role in <em>Alternaria</em>–stress response is found to be complicated. Auxin conjugating genes such as GH3.8 were both up and down regulated. On the other hand, a majority of the repressors of auxin signalling pathway including Aux/IAA1, 2, 5, 7, 9, 10 and 11 were significantly downregulated, indicating the onset of Auxin signalling also occurred during the infection. We have also noticed that one of the marker genes for auxin signalling pathway, proteinase inhibitor II (Pin2), was markedly upregulated (log<sub>2</sub>FC = +10.03), supporting the view of activation of auxin signalling pathway during <em>Alternaria</em>-infection in tomato. However, 75% of the auxin effector genes SAUR were down-regulated. Therefore, a detailed analysis is required to comprehend the exact role of auxin in this process.</p><p>In plant defence response, ROS and NO are thought to regulate programmed cell death through the establishment of the hypersensitive reaction.<a reveal-id="dsx003-B16" data-open="dsx003-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Genes related to ROS generation/scavenging systems were highly upregulated including respiratory burst oxidase (<em>RBOH</em>), peroxidise (<em>POD</em>), glutathione peroxidase (<em>GPX</em>), catalase (<em>CAT</em>), nitric oxide reductase, glutathione s transferase, ascorbate peroxidase. In addition, two classes of detoxifying genes UDP-glycosyltransferase (<em>UDPG</em>) (36/66 genes) and ABC transporters (17/25 genes) were upregulated many folds, probably to detoxify toxins generated by <em>A</em><em>.</em><em>solani</em>. Regarding ABA signalling, Out of seven ABA receptors showing differential expression, six depicted highly elevated expression, suggesting, ABA-signalling was also activated.</p><p>The sensors of Ca<sup>2+</sup> flux in plants in response to stress, including 16 calmodulin-like protein (CAM), calcium-dependent protein kinase (CDPKs), 2 CBL-interacting protein kinase (CIPK) were upregulated many folds, however, membrane proteins like sodium/calcium exchanger (NCX), Calcium/proton exchanger are down-regulated, implying calcium accumulation is favoured within the cell which may serve as a second messenger in host to activate downstream gene regulation.</p><p>There were 183 DEGs that were different kinds of transcription factors (TFs), such as the WRKY family (i.e. WRKY16, 23, 29, 30, 37, 27, 73), ERF family (i.e. ERF2, 1b,1a, 2a,7,10), NAC, basic leucine zipper (bZIP) family, GRAS family, basic helix loop helix (bHLH) family and heat stress transcription factor (HSF), and master regulator myeloblastosis protein (MYB) family. Among these DEGs almost all the WRKY transcription factors were found to be upregulated (35/36 genes), ERF (43/63), NAC (25/38), MYB (36/54) and HSF (5/6) followed the same trend. However, the majority of the genes belonging to the bHLH group of transcription factor were found to be down-regulated (23/31).</p><p>Plant transmembrane receptor-like kinases (RLKs), receptor-like proteins (RLPs) and intracellular nucleotide-binding site leucine-rich repeat (NB-LRR) genes are surveyors of pathogen-associated molecular patterns and effector molecules for initiating stress signalling. Very interestingly, 219/326 significantly differentially regulated such genes were upregulated, indicating active signalling was prevalent inside the infected tissues. It will be intriguing to decipher the role of highly differentially regulated disease resistance related NB-LRR genes in the EB disease biology.</p>                            <h3 scrollto-destination=88473833 id="88473833" class="section-title" >3.12. Combined expression analysis of miRNAs and cognate target mRNAs confirms pathogenesis/disease development-related genes are majorly targeted during <em>Alternaria</em>–tomato compatible interaction</h3>
<p>We integrated our miRNA and mRNA sequencing data to get relative expression level of miRNA–mRNA pairs. To impose more stringency in the analysis we have considered only 606 common targets, among these 450 could be detected in our transcriptome analysis. We have further filtered these possible targets based on predetermined cut-off value of log<sub>2</sub>FC at least ±1 (<em>P</em> ≤ 0.05) in the transcriptome dataset and obtained 202 target genes. 102 (50.4%) among these showed significant opposite expression level compared with the expression of cognate miRNA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). The oppositely regulated targets included (i) signalling/sensor molecules such as calmodulin binding protein (Solyc02g079040) and 14 NB-LRR disease resistance genes, 13 of them were cc–nbs–lrr type, (ii) transcription factors including MYB, Squamosa promoter binding like (SBP), AP2-ethylene-responsive and GRAS family, (iii) Nucleoredoxin 2 (Solyc05g005460), a thioredoxin-disulfide reductase and a Glutaredoxin family protein (Solyc08g082590) which might have role in oxidation-reduction process, (iv) multiple RLKs and RLPs and (v) genes related to ubiquitination process such as U-box (Solyc05g008230) and F-box proteins (Solyc01g106510) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). However, it must be noted here that we are considering only genes which are likely to be cleaved by miRNAs, apart from this, translational inhibition of gene expression by miRNAs is also important, which could not be accounted in the present analysis.</p><p>Further, filtering of the targets using more stringent target prediction criteria, such that the target and the miRNA have almost perfect complementarity throughout its length, we obtained a list of 15 stringent miRNA–mRNA pairs showing opposite expression pattern during <em>Alternaria</em>–stress (<a reveal-id="dsx003-F6" data-open="dsx003-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>). Tomato degradome analysis data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>) confirmed nine of these interactions, and our 5′-RLM RACE analysis also confirmed some of these interactions during stress (<a reveal-id="dsx003-F6" data-open="dsx003-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>). The majority of these targets has roles in stress-related gene or pathway regulation, suggesting miRNA-mediated regulation of transcriptome is an essential process in <em>Alternaria</em>–stress response in tomato.</p>                        <a id="88473836" scrollto-destination="88473836"></a>
<div id="dsx003-F6" data-id="dsx003-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f6.png?Expires=2147483647&Signature=ekoM2~Hu~prbOb3BAkXtjgXpSUUQJR6~OgCcisTifI6kOQrAGQ6EssDlsWsq-qMTiRVuQnPMiF-t2gW96bhVfigCwd6q8gIeR6-L-XB3oT0NybSjbS5TeaXrBimzWTgMbec9Heb~F9KW6fRt65bnO4iE3aNUqkALK9QBbfsGb756mSppvARA2FoNAWXd3o96cvgOHkjdIGch9q~n7wDF5-gpqsu2gjf6QqosvQ1elQLZs0okLpJeN61QIP8wAkT82VYTGbnwQVIs1~bGr97lygqiqt7KVSHj5enQVjGSz21Ci1kkKBJjR2z7HU0OIPK8CX86Tu9-OjkubtmXgfYWsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scheme showing some of the key Alternaria-responsive miRNAs and their targets in tomato. (A) Regulated miRNAs with their corresponding targets which are inversely correlated in our NGS analysis. The arrows indicate positive regulation and the hammer head represents negative regulation. (B) miRNA cleavage sites in some mRNAs as determined by modified 5′-RLM RACE. Arrows indicate the position of the cleavage." path-from-xml="dsx003f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473836/dsx003f6.tif" path-from-xml="dsx003f6.tif" target="_blank">View large</a><a section="88473836" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f6.png?Expires=2147483647&Signature=sNSt9qDc97FEhWfF90lzMNQQq9Xum3Mp0mSy3pnoLDB3iMirlMZpyHXECSmYD-iQVudMNQE9sRi53kqn7tWtF40ZsYVcq66YGZEeQdAjyuMOvP51cpDueYZz4FDGu5KapYUdbO-VtJjIkRyl3bALXqO4Jc6VYHm~OdoklkK80IAtU~YQgvrf0uL-H5ZdRwc1~lAKh~ii~AlGjSllXe4YskyBfghHpKlFVLUuHjJdS4fIcypHe2a95hX-gmwvIdBHq608mhFkkTLMdRMY29MPCbrFuEc17qmqIdo3biLosFaMlLE704Dg7Z8O6hQQOr3Pj6nMnch4FdZSxfBSFvaIjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473836&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scheme showing some of the key <em>Alternaria</em>-responsive miRNAs and their targets in tomato. (A) Regulated miRNAs with their corresponding targets which are inversely correlated in our NGS analysis. The arrows indicate positive regulation and the hammer head represents negative regulation. (B) miRNA cleavage sites in some mRNAs as determined by modified 5′-RLM RACE. Arrows indicate the position of the cleavage.</p></div></div><div content-id="dsx003-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/m_dsx003f6.png?Expires=2147483647&Signature=ekoM2~Hu~prbOb3BAkXtjgXpSUUQJR6~OgCcisTifI6kOQrAGQ6EssDlsWsq-qMTiRVuQnPMiF-t2gW96bhVfigCwd6q8gIeR6-L-XB3oT0NybSjbS5TeaXrBimzWTgMbec9Heb~F9KW6fRt65bnO4iE3aNUqkALK9QBbfsGb756mSppvARA2FoNAWXd3o96cvgOHkjdIGch9q~n7wDF5-gpqsu2gjf6QqosvQ1elQLZs0okLpJeN61QIP8wAkT82VYTGbnwQVIs1~bGr97lygqiqt7KVSHj5enQVjGSz21Ci1kkKBJjR2z7HU0OIPK8CX86Tu9-OjkubtmXgfYWsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scheme showing some of the key Alternaria-responsive miRNAs and their targets in tomato. (A) Regulated miRNAs with their corresponding targets which are inversely correlated in our NGS analysis. The arrows indicate positive regulation and the hammer head represents negative regulation. (B) miRNA cleavage sites in some mRNAs as determined by modified 5′-RLM RACE. Arrows indicate the position of the cleavage." path-from-xml="dsx003f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473836/dsx003f6.tif" path-from-xml="dsx003f6.tif" target="_blank">View large</a><a section="88473836" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003f6.png?Expires=2147483647&Signature=sNSt9qDc97FEhWfF90lzMNQQq9Xum3Mp0mSy3pnoLDB3iMirlMZpyHXECSmYD-iQVudMNQE9sRi53kqn7tWtF40ZsYVcq66YGZEeQdAjyuMOvP51cpDueYZz4FDGu5KapYUdbO-VtJjIkRyl3bALXqO4Jc6VYHm~OdoklkK80IAtU~YQgvrf0uL-H5ZdRwc1~lAKh~ii~AlGjSllXe4YskyBfghHpKlFVLUuHjJdS4fIcypHe2a95hX-gmwvIdBHq608mhFkkTLMdRMY29MPCbrFuEc17qmqIdo3biLosFaMlLE704Dg7Z8O6hQQOr3Pj6nMnch4FdZSxfBSFvaIjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473836&ar=3061403&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx003f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scheme showing some of the key <em>Alternaria</em>-responsive miRNAs and their targets in tomato. (A) Regulated miRNAs with their corresponding targets which are inversely correlated in our NGS analysis. The arrows indicate positive regulation and the hammer head represents negative regulation. (B) miRNA cleavage sites in some mRNAs as determined by modified 5′-RLM RACE. Arrows indicate the position of the cleavage.</p></div></div>                            <h2 scrollto-destination=88473837 id="88473837" class="section-title" >4. Discussion</h2>
                            <h3 scrollto-destination=88473838 id="88473838" class="section-title" >4.1. miRNAs in <em>Alternaria</em>-disease biology</h3>
<p>Recent publications have demonstrated that miRNAs are involved in plant defence. The advent of NGS technique has expanded the list of miRNAs discovered in plant and their differential expression in different plants exposed to various biotic stresses. These studies also include analysis of tomato miRNA regulation during biotrophic pathogen interaction, such as tomato–cucumber mosaic virus and tomato-ToLCNDV<a reveal-id="dsx003-B17" data-open="dsx003-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>; and necrotrophic interactions, such as tomato-<em>Botrytis cineria</em><a reveal-id="dsx003-B4" data-open="dsx003-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and tomato-<em>Fusarium</em>.<a reveal-id="dsx003-B18" data-open="dsx003-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Essential role of tomato miRNAs targeting NB-domain protein in conferring resistance to <em>Fusarium</em> fungal attack<a reveal-id="dsx003-B18" data-open="dsx003-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> has been established. However, which miRNAs and genes are precisely regulated during <em>Alternaria-</em>stress in tomato was not known. Hence in this study, we adopted high throughput sequencing approaches to performing a combined expression profiling analysis of miRNAs and mRNAs in the same sample to provide an enhanced understanding of potential miRNA regulatory network in tomato subjected to <em>Alternaria</em>-stress. In fact, an extensive search of databases only yielded one reference related to whole transcriptome analysis during plant–pathogen interaction, describing small RNA and mRNA transcriptome during rice–RSV interaction.<a reveal-id="dsx003-B11" data-open="dsx003-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Thus, comprehensive study of the miRNA and mRNA expression profiles and correlating miRNA–mRNA expression in tomato inoculated with and without <em>A</em><em>.</em><em>solani</em> is well timed.</p><p>We initiated this study with the identification of the optimum stage of disease development which was selected for isolating total RNA. We considered following factors, (i) the disease should be progressed sufficiently, (ii) the tissue should not be completely blighted as these tissues will contain more dead cells and degraded RNA and (iii) the plant should show prominent biochemical or molecular response towards the disease, as it will indicate activation of signalling components. Our observations prompted us to use leaves of stage 3 (∼72 hpi) of the disease (<a reveal-id="dsx003-F1" data-open="dsx003-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). The tissues indeed showed heightened expression of marker genes, PR1b and Chitinase, for biotic stress response (<a reveal-id="dsx003-F1" data-open="dsx003-F1" class="link link-reveal link-table xref-fig">Fig. 1F</a>), confirming appropriate diseased samples were used for comparison purpose. Concurrently, we had ensured that (i) high-quality of RNA were used, (ii) no fungal RNA contaminated our samples and (iii) only high-quality sequence reads were analysed with sufficient stringency wherever required.</p><p>The recent publications on tomato miRNA transcriptome analysis also described a number of conserved and novel miRNAs, however, it is difficult to combine all these information to make a comprehensive database for comparison purpose. In the miRBase, only a few miRNAs are listed as originated from tomato. Thus to eliminate the probability of excluding any significant miRNA from our analysis, we have gathered information about all plant-specific miRNAs available in the miRBase, and tomato miRNAs in TFGD and TGRD databases for making a comprehensive list of miRNAs. This list was used for the identification of miRNAs in our dataset. A perfect match was named according to the name of the matching miRNA found in any of these databases; however, a miRBase match received the priority during naming. An unmatched but otherwise fulfilling pre-set criteria for identification of miRNAs was called as a novel miRNA, and its corresponding sRNA number was also looked for in the TFGD database. This arrangement has avoided unnecessary new naming of a miRNA with known sequence information and eased the tracking of already available information regarding the miRNAs identified here.</p><p>Overall analysis of small RNA data revealed two interesting facts, (i) during the infection the total reads number of 21 nt sRNAs increased and 24-nt population decreased markedly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) and (ii) tomato genome encodes more types of Solanaceae/species-specific miRNAs compared with conserved miRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The heterochromatic siRNAs are 24-nt long, having a possible role in DNA methylation whereas miRNAs or secondary siRNAs are of 21 nt.<a reveal-id="dsx003-B19" data-open="dsx003-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Thus it will be interesting to investigate if the infection has caused reduced methylation at some specific loci leading to the expression of <em>A</em><em>.</em><em>solani</em> responsive genes.</p><p>We have meticulously analysed the expression of each member of a miRNA family. Members to a family were assigned based on the sequence similarity among miRNAs and assumed minor mismatches might have aroused due to the expression from different precursor sequences, and different products that might have generated due to the irregular processing of the same precursor are placed in a same family-member group. We believe this is the best method for identifying a family and its members. However, we have also noticed that the numbers of designated family members differed from the numbers reported in previous studies,<a reveal-id="dsx003-B20" data-open="dsx003-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx003-B21" data-open="dsx003-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> although some of the abundant expresser families (miR166, miR168, miR156, miR 6027) were also identified in these reports.</p><p>The expression level of different members of the same miRNA family showed noticeable variation among them (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Interestingly, upon infection the expression pattern of isoforms also altered within a family (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Visible changes were noticed for miR167, miR171, miR393, miR319, miR397 and miR166 among the conserved miRNA families. In line with this observation, members of the miR166 family were found to have different expression pattern in wheat in response to powdery mildew infection and heat stress.<a reveal-id="dsx003-B5" data-open="dsx003-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a></p><p>Differential miRNA expression analysis indicated that both conserved and Solanaceae-specific miRNAs were differentially regulated during <em>Alternaria</em> stress, which is analogous to observations with other plant–pathogen interaction studies.<a reveal-id="dsx003-B4" data-open="dsx003-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx003-B18" data-open="dsx003-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx003-B22" data-open="dsx003-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsx003-B23" data-open="dsx003-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Our stringent analysis demonstrated that majority of miRNAs were down-regulated upon <em>Alternaria-</em>stress. Similar, down-regulation of the majority of miRNAs were observed in tomato–<em>Phytophthora infestans</em>,<a reveal-id="dsx003-B24" data-open="dsx003-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> brinjal–<em>Verticillium dahliae</em><a reveal-id="dsx003-B6" data-open="dsx003-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> interactions. However, during <em>Botrytis cinerea</em>, CMV<a reveal-id="dsx003-B25" data-open="dsx003-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and <em>Fusarium oxysporum</em><a reveal-id="dsx003-B18" data-open="dsx003-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> stresses tomato plant responded by up-regulating majority of miRNAs. Hence, it seems the pattern of regulating the expression of miRNAs during stress is dependent on a specific plant and pathogen interaction, and thus it is difficult to generalize a rule of regulation of miRNAs during stress.</p><p>Some recent studies conducted using different fungus and plants showed the importance of specific miRNAs in making a plant susceptible or resistant against a particular pathogen.<a reveal-id="dsx003-B4" data-open="dsx003-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx003-B18" data-open="dsx003-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> miR166, miR169 and miR397 were reported to be down-regulated in susceptible soybean variety upon <em>Phakopspora pachyrhizi</em> infection.<a reveal-id="dsx003-B26" data-open="dsx003-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Our data showed that during tomato–<em>Alternaria</em> interaction miR169 and miR397 were down-regulated many fold but expression of miR166 was not regulated significantly. A close inspection of the miRNAs regulated as an effect of necrotrophic fungi <em>Botrytis cinerea</em><a reveal-id="dsx003-B4" data-open="dsx003-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and <em>A</em><em>.</em><em>solani</em> (this study) in tomato we found that 13 miRNAs were commonly regulated by both the stresses in tomato. Eight miRNAs (sly-miR156d-3p, sly-miR6300, sly-miR156f, sly-miR482a, sly-miR482b, sly-miR160a and sly-miR168a/b were up-regulated and sly-miR396a-3p was down-regulated) showed similar trend of regulation among the two stresses, however, five miRNAs (sly-miR171e, sly-miR169b/d/e, sly-miR172a/b, sly-miR390a-5p and sly-miR399a) showed opposite regulation. A similar comparison of regulated miRNAs between biotrophic pathogen CMV-stress<a reveal-id="dsx003-B25" data-open="dsx003-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and <em>Alternaria-</em>stress in tomato showed 11 miRNAs to be differentially regulated in common. Among them, seven miRNAs showed the similar trend in regulation (sly-miR167c, sly-miR6024b, sly-miR168a/b were up-regulated, and sly-miR166k, sly-miR397, sly-miR408b-3p and miR395a/b were down-regulated in both the cases). However, remaining four miRNAs (sly-miR393a-3p, sly-miR390a-3p, sly-miR398a-3p and miR408b-5p) showed opposite regulation. Thus, it seems there are certain common response genes that are probably regulated by miRNAs during a biotic stress; however, the specific response is governed by regulation of a specific set of miRNAs, reinstating our previously stated notion.</p><p>In addition to known miRNAs, 60 novel putative miRNAs were also discovered (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). The novel miRNAs are in general low expressing as compared with known miRNAs. Although we were able to validate the expression of four arbitrarily chosen putative novel miRNAs using alternative techniques, further experimentations are essential to declare them as real miRNAs. However, the precursors of at least two of these novel miRNAs (sly-miRPR11 and sly-miRPR29) were detectable and showed variable expression in infected tissues, thus it is safe to conclude that these two miRNAs are biologically active.</p>                            <h3 scrollto-destination=88473848 id="88473848" class="section-title" >4.2. Transcriptomics of EB diseased leaves</h3>
<p>Profiling of differentially expressed transcripts during <em>Alternaria</em>-stress, and subsequent functional and pathway enrichment analyses (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>A and B) provided a comprehensive view of response regulators during the compatible interaction. We have also delved into more profiling details to find out (i) which GO terms are represented by up and downregulated genes, and (ii) the expression status of genes of an enriched pathway. These analyses provided an enhanced perspective to the overall expression data.</p><p>Our data suggest that enriched GO-terms ‘chlorophyll binding’ and ‘tetrapyrrole binding’ were represented by down-regulated genes in the molecular function category. Under the cellular component category, the majority of the terms enriched were against down-regulated genes, which also included terms related to photosynthesis, more specifically photosystem reaction centre I and thylakoid. Moreover, pathway annotation also showed mostly down-regulated genes enriched the terms related to photosynthesis process, especially photosynthesis–antenna proteins, porphyrin and chlorophyll metabolism, starch and sucrose metabolism and carbon fixation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>B), further supporting photosynthesis was severely affected during <em>Alternaria</em> stress, which could be explained by chlorosis and blight symptom owing to the disease. Other plants are also known to be affected by down-regulation of photosynthesis during a pathogen attack,<a reveal-id="dsx003-B7" data-open="dsx003-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx003-B27" data-open="dsx003-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> however, this could help the plant to cope the stress.</p><p>A biotic stress is perceived by the plant at the membrane, followed by initiation of a signalling cascade in which RLK, RLP and NB-LRR proteins play crucial roles, and subsequently, a set of stress-management-related genes are expressed. Our analysis suggested that multiple such proteins were differentially regulated. In line with this, the major categories that are GO-enriched and represented by mostly up-regulated genes belonged to ‘transferase activity’, ‘kinase activity’ and ‘catalytic activity’; and especially ‘phosphotransferase activity’ and ‘protein kinase activity’. This indicates, events like phosphorylation and de-phosphorylation increased in the cell during the stress, leading to the activation of various signalling intermediates thus altering signalling pathways. KO terms for plant pathogen interaction pathway have also been found to be enriched and represented by upregulated genes. These results prove that an active signalling cascade was primed during the stress. Accordingly, under biological process category terms related to response to stimuli and specifically biotic stimuli were enriched.</p><p>The diseased plant responds by activation of genes that help in scavenging the toxins produced inside the tissues or by producing secondary metabolites that might check the pathogen growth to cope the stress. An important category which was enriched among the up-regulated genes was ‘glutathione transferase (GST) activity’. GSTs are known for their ability to detoxify toxins, protect a plant from oxidative stress and regulating cell proliferation and death by interfering with MAPK pathway.<a reveal-id="dsx003-B28" data-open="dsx003-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><em>Alternaria</em>-infection is associated with oxidative burst (<a reveal-id="dsx003-F1" data-open="dsx003-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), which could be induced by the necrotrophic pathogen to kill the cells, and cellular scavenging enzymes such as GSTs thus are activated to mitigate the stress. Upregulated genes were also involved in secondary metabolite biosynthesis pathways including phenylalanine and phenylpropanoid biosynthesis. Phenylpropanoid metabolism pathway generates diverse kind of secondary metabolites which can protect plants against pathogens. Thus it can be inferred, an increase in the expression of genes responsible for phenylalanine metabolism is providing enough precursor (phenylalanine) for phenylpropanoid metabolism, which in turn forms basic units of many secondary metabolite biosynthesis.<a reveal-id="dsx003-B29" data-open="dsx003-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> In summary, genes involved in three crucial pathways such as activation of, stress-signal transduction, cellular detoxification machinery and metabolic pathways including biosynthesis of secondary metabolites were altered upon <em>Alternaria</em> infection in tomato.</p>                            <h3 scrollto-destination=88473853 id="88473853" class="section-title" >4.3. Integrating miRNA–mRNA transcriptomics</h3>
<p>Within a cell, if the level of a miRNA increases, decreased level of the cognate target mRNA indicates increased cleavage activity of the miRNA, on the other hand, a decrease in the level of miRNA results in increased level of target mRNAs. Hence, a transcriptome-wide combined analysis of miRNA and mRNA expression level was carried out to identify the miRNA–mRNA pairs which were regulated during tomato–<em>A</em><em>.</em><em>solani</em> compatible interaction. This has enabled us to understand which pathways and biological processes of the cell were most likely regulated by miRNAs during the stress.</p><p>We used considerably differentially expressed miRNAs to identify putative targets using two separate target prediction databases. A comparison between the GO-based functional enrichment of these predicted targets with the differentially expressed transcripts (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S10 and S12A</a></span>) revealed the biological activities that were likely regulated by miRNAs during the stress. For example in both the analyses, (i) majority of the enriched genes were categorized under ‘molecular function’; (ii) activities like ‘catalytic’ and ‘transferase’ were most enriched, and (iii) ‘kinase activity’, ‘molecular transducer’, ‘signal transducer’ and ‘receptor activities’ were also substantially reflected. Such observations indicate that genes representing the above-mentioned ontologies are likely targeted by miRNAs. However, genes showing ‘oxidoreductase’, ‘UDP-glycosyltransferase’ and ‘oxygen binding’ activity were enriched only in regulated transcripts, indicating although genes involved in above processes were differentially expressed during <em>Alternaria</em> stress but this regulation was not due to the miRNA-mediated cleavage of transcripts. Similarly, ‘response to stimulus’ especially ‘response to biotic stimulus’ was enriched in the ‘biological process’ category only amid the regulated transcripts. On the contrary, ‘cell death’ and ‘RNA and ncRNA metabolism’ and ‘processing’ terms were enriched under the same category for the predicted targets only. These observations suggesting only a few specific biological processes were regulated by miRNAs during the stress.</p><p>Under the ‘cellular component’, genes associated to thylakoid were common between the two analyses, but no genes related to photosynthesis was enriched among the predicted targets although photosynthetic pathway was found to be affected as a whole during <em>Alternaria</em> stress, implying photosynthetic genes are majorly regulated transcriptionally and may not be significantly affected by miRNA-mediated silencing pathway.</p><p>During the infection ROS is overproduced (<a reveal-id="dsx003-F1" data-open="dsx003-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), thus we set out to detect whether genes related to ROS generation/scavenging pathway were indeed targets of miRNAs. Our analysis indicated that genes involved in oxidation-reduction processes and detoxification during oxidative stress were targeted by miRNAs and their expression was inversely correlated to that of targeting miRNAs, these include stu-miR398a-3p-Nucleoredoxin 2 (Solyc05g005460), sly-miR9518-glutaredoxin family protein (Solyc08g082590), sly-miR408b-3p-blue copper protein–plastocyanin (Solyc01g104400), sly-miR9474-5p-nudix hydrolase 2 (Solyc05g016690) and sly-miR169b-UDP-glucosyltransferase (Solyc06g062290).</p><p>We were able to validate some of the predicted targets by analysing recently published tomato degradome NGS data<a reveal-id="dsx003-B21" data-open="dsx003-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsx003-B30" data-open="dsx003-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and 5′-RACE analysis (<a reveal-id="dsx003-F6" data-open="dsx003-F6" class="link link-reveal link-table xref-fig">Fig. 6b</a>). Among 606 common predicted targets, we found 44 mRNAs to be cleaved at the exact position as predicted in our analysis. Next, a closer inspection of transcriptomics data showed that 38 miRNAs formed 102 miRNA–mRNA interaction pairs and showed opposite regulation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). It was noticed that majority of the 102 oppositely expressed targets with respect to their cognate miRNA, could be clustered into specific groups including transcription factors, NB–LRRs and ubiquitination related proteins. NB–LRR–miRNA interactions have been well documented<a reveal-id="dsx003-B18" data-open="dsx003-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx003-B31" data-open="dsx003-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and F-box proteins are known targets of miR393 and miR394.<a reveal-id="dsx003-B32" data-open="dsx003-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> As late infection inflicts massive cell death, it is imminent that cellular general metabolic processes will be largely affected. We have shown earlier that several metabolism-related GO-terms were enriched, and miRNA–mRNA paired expression analysis revealed some genes involved in the basic metabolism of cell like Phosphofructokinase family protein, Citrate synthase, Fructose-bisphosphate aldolase, etc. were also targeted and expression was negatively affected (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>).</p><p>Further stringent analysis by imposing more selective criteria has led to the identification of specific genes that are most likely targeted by miRNAs (<a reveal-id="dsx003-F6" data-open="dsx003-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Many of these interactions are also validated in independent studies<a reveal-id="dsx003-B18" data-open="dsx003-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx003-B21" data-open="dsx003-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsx003-B30" data-open="dsx003-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsx003-B33" data-open="dsx003-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsx003-B34" data-open="dsx003-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>, <a reveal-id="dsx003-F6" data-open="dsx003-F6" class="link link-reveal link-table xref-fig">Fig. 6b</a>). Hence, in this study key genes and miRNA regulators responsible for mediating <em>Alternaria</em>–stress response in tomato has been identified, and we have confirmed <em>Alternaria</em>-stress-related regulation of miRNA expression as one of the important regulatory events that occur to reprogramme gene expression cascade to cope the stress.</p>                            <h2 scrollto-destination=88473860 id="88473860" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=88473862 id="88473862" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This study was supported by a DBT grant to P.K. Authors acknowledge Bose Institute CIF for technical helps and are thankful to Dr. Sujoy Saha, IIVR, India, and Dr. Subrata Dutta, BCKV, India for assistance with pathogen studies. Sudhriti Ghosh Dostidar has helped in some data analysis.</p>                            <h3 scrollto-destination=88473864 id="88473864" class="section-title" >Accession numbers</h3>
<p>GEO accession: GSE75922 and GSE75923.</p>                            <h3 scrollto-destination=88473866 id="88473866" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=88473868 id="88473868" class="section-title" >Funding</h3>
<p>This study was supported by a grant, BT/PR12942/AGR/36/652/2009, from Department of Biotechnology, Government of India to PK.</p>                            <h2 scrollto-destination=88473870 id="88473870" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx003-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peralta</div> <div class="given-names">I.E.</div></div>, <div class="name string-name"><div class="surname">Knapp</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Spooner</div> <div class="given-names">D.M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">New species of wild tomatoes (Solanum Section Lycopersicon: Solanaceae) from Northern Peru</div>, <div class="source">Syst. Bot</div> ., <div class="volume">30</div>, <div class="fpage">424</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20species%20of%20wild%20tomatoes%20(Solanum%20Section%20Lycopersicon%3A%20Solanaceae)%20from%20Northern%20Peru&amp;author=I.E.%20Peralta&amp;author=S.%20Knapp&amp;author=D.M.%20Spooner&amp;publication_year=2005&amp;journal=Syst.%20Bot&amp;volume=30&amp;pages=424-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1600/0363644054223657" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1600%2F0363644054223657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1600%2F0363644054223657"> </span></div> </div></div></div></div></div><div content-id="dsx003-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chaerani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Groenwold</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Stam</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Voorrips</div> <div class="given-names">E.R.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Assessment of early blight (<em>Alternaria solani</em>) resistance in tomato using a droplet inoculation method</div>. <div class="source">J. Gen. Plant Pathol</div> ., <div class="volume">73</div>, <div class="fpage">96</div>–<div class="lpage">103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20early%20blight%20(Alternaria%20solani)%20resistance%20in%20tomato%20using%20a%20droplet%20inoculation%20method&amp;author=R.%20Chaerani&amp;author=R.%20Groenwold&amp;author=P.%20Stam&amp;author=E.R.%20Voorrips&amp;publication_year=2007&amp;journal=J.%20Gen.%20Plant%20Pathol&amp;volume=73&amp;pages=96-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10327-006-0337-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10327-006-0337-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10327-006-0337-1"> </span></div> </div></div></div></div></div><div content-id="dsx003-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mallory</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">D.P.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">B.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">MicroRNA-directed regulation of Arabidopsis AUXIN RESPONSE FACTOR17 is essential for proper development and modulates expression of early auxin response genes</div>. <div class="source">Plant cell</div> , <div class="volume">17</div>, <div class="fpage">1360</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNA-directed%20regulation%20of%20Arabidopsis%20AUXIN%20RESPONSE%20FACTOR17%20is%20essential%20for%20proper%20development%20and%20modulates%20expression%20of%20early%20auxin%20response%20genes&amp;author=A.C.%20Mallory&amp;author=D.P.%20Bartel&amp;author=B.%20Bartel&amp;publication_year=2005&amp;journal=Plant%20cell&amp;volume=17&amp;pages=1360-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.031716" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.031716" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.031716"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15829600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jin</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">F.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Characterization of miRNAs associated with <em>Botrytis cinerea</em> infection of tomato leaves</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">15</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20miRNAs%20associated%20with%20Botrytis%20cinerea%20infection%20of%20tomato%20leaves&amp;author=W.%20Jin&amp;author=F.%20Wu&amp;publication_year=2015&amp;journal=BMC%20Plant%20Biol&amp;volume=15&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-014-0410-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-014-0410-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-014-0410-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25592487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yao</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Diverse set of microRNAs are responsive to powdery mildew infection and heat stress in wheat (<em>Triticum aestivum</em> L.)</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">10</div>, <div class="fpage">123</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diverse%20set%20of%20microRNAs%20are%20responsive%20to%20powdery%20mildew%20infection%20and%20heat%20stress%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=M.%20Xin&amp;author=Y.%20Wang&amp;author=Y.%20Yao&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol&amp;volume=10&amp;pages=123" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20573268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Jue</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Q.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Identification of MiRNA from eggplant (<em>Solanum melongena</em> L.) by small RNA deep sequencing and their response to <em>Verticillium dahliae</em> infection</div>. <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e72840</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20MiRNA%20from%20eggplant%20(Solanum%20melongena%20L.)%20by%20small%20RNA%20deep%20sequencing%20and%20their%20response%20to%20Verticillium%20dahliae%20infection&amp;author=L.%20Yang&amp;author=D.%20Jue&amp;author=W.%20Li&amp;author=R.%20Zhang&amp;author=M.%20Chen&amp;author=Q.%20Yang&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e72840" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0072840" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0072840" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0072840"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24015279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Windram</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Madhou</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">McHattie</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Arabidopsis defense against <em>Botrytis cinerea</em>: chronology and regulation deciphered by high-resolution temporal transcriptomic analysis</div>. <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">3530</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20defense%20against%20Botrytis%20cinerea%3A%20chronology%20and%20regulation%20deciphered%20by%20high-resolution%20temporal%20transcriptomic%20analysis&amp;author=O.%20Windram&amp;author=P.%20Madhou&amp;author=S.%20McHattie&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=3530-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.112.102046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.102046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.112.102046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23023172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tan</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Transcriptome analysis of the compatible interaction of tomato with <em>Verticillium dahliae</em> using RNA-sequencing</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">428</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20the%20compatible%20interaction%20of%20tomato%20with%20Verticillium%20dahliae%20using%20RNA-sequencing&amp;author=G.%20Tan&amp;author=K.%20Liu&amp;author=J.%20Kang&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=428" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26106404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Manzo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ferriello</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Puopolo</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title"><em>Fusarium oxysporum</em> f.sp. radicis-lycopersici induces distinct transcriptome reprogramming in resistant and susceptible isogenic tomato lines</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fusarium%20oxysporum%20f.sp.%20radicis-lycopersici%20induces%20distinct%20transcriptome%20reprogramming%20in%20resistant%20and%20susceptible%20isogenic%20tomato%20lines&amp;author=D.%20Manzo&amp;author=F.%20Ferriello&amp;author=G.%20Puopolo&amp;publication_year=2016&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-016-0740-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-016-0740-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-016-0740-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26920134" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Upadhyay</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Rai</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sinha</div> <div class="given-names">B.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Microarray analyses during early stage of the tomato/<em>Alternaria solani</em> interaction</div>. <div class="source">Genom. Data</div> , <div class="volume">6</div>, <div class="fpage">170</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20analyses%20during%20early%20stage%20of%20the%20tomato%2FAlternaria%20solani%20interaction&amp;author=P.%20Upadhyay&amp;author=A.%20Rai&amp;author=R.%20Kumar&amp;author=M.%20Singh&amp;author=B.%20Sinha&amp;publication_year=2015&amp;journal=Genom.%20Data&amp;volume=6&amp;pages=170-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gdata.2015.09.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gdata.2015.09.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gdata.2015.09.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26697364" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Integrative analysis of the microRNAome and transcriptome illuminates the response of susceptible rice plants to rice stripe virus</div>. <div class="source">PLoS One</div> , <div class="volume">11</div>, <div class="fpage">e0146946</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20analysis%20of%20the%20microRNAome%20and%20transcriptome%20illuminates%20the%20response%20of%20susceptible%20rice%20plants%20to%20rice%20stripe%20virus&amp;author=J.%20Yang&amp;author=F.%20Zhang&amp;author=J.%20Li&amp;author=J.P.%20Chen&amp;author=H.M.%20Zhang&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0146946" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0146946" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0146946" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0146946"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26799317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yin</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Integration of small RNAs, degradome and transcriptome sequencing in hyperaccumulator <em>Sedum alfredii</em> uncovers a complex regulatory network and provides insights into cadmium phytoremediation</div>. <div class="source">Plant Biotechnol. J</div> ., <div class="volume">14</div>, <div class="fpage">1470</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20small%20RNAs%2C%20degradome%20and%20transcriptome%20sequencing%20in%20hyperaccumulator%20Sedum%20alfredii%20uncovers%20a%20complex%20regulatory%20network%20and%20provides%20insights%20into%20cadmium%20phytoremediation&amp;author=X.%20Han&amp;author=H.%20Yin&amp;author=X.%20Song&amp;publication_year=2016&amp;journal=Plant%20Biotechnol.%20J&amp;volume=14&amp;pages=1470-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.12512" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.12512" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.12512"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26801211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Integrated analysis of miRNA and mRNA expression profiles in response to Cd exposure in rice seedlings</div>. <div class="source">BMC Genom</div> ., <div class="volume">15</div>, <div class="fpage">835</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20analysis%20of%20miRNA%20and%20mRNA%20expression%20profiles%20in%20response%20to%20Cd%20exposure%20in%20rice%20seedlings&amp;author=M.%20Tang&amp;author=D.%20Mao&amp;author=L.%20Xu&amp;author=D.%20Li&amp;author=S.%20Song&amp;author=C.%20Chen&amp;publication_year=2014&amp;journal=BMC%20Genom&amp;volume=15&amp;pages=835" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-835" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-835" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-835"> </span></div> </div></div></div></div></div><div content-id="dsx003-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dempsey</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Vlot</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Wildermuth</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Klessig</div> <div class="given-names">D.F.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Salicylic acid biosynthesis and metabolism</div>. <div class="source">The Arabidopsis Book/American Society of Plant Biologists</div> , <div class="volume">9</div>, <div class="fpage">e0156</div>.</div></div></div></div><div content-id="dsx003-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">An</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Derepression of ethylene-stabilized transcription factors (EIN3/EIL1) mediates jasmonate and ethylene signaling synergy in Arabidopsis</div>. <div class="source">Proc. Natl Acad. Sci. U. S. A</div> ., <div class="volume">108</div>, <div class="fpage">12539</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Derepression%20of%20ethylene-stabilized%20transcription%20factors%20(EIN3%2FEIL1)%20mediates%20jasmonate%20and%20ethylene%20signaling%20synergy%20in%20Arabidopsis&amp;author=Z.%20Zhu&amp;author=F.%20An&amp;author=Y.%20Feng&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=108&amp;pages=12539-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1103959108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1103959108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1103959108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21737749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Loake</div> <div class="given-names">G.J.</div></div>, <div class="name string-name"><div class="surname">Chu</div> <div class="given-names">C.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Cross-talk of nitric oxide and reactive oxygen species in plant programed cell death</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">4</div>, <div class="fpage">314</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-talk%20of%20nitric%20oxide%20and%20reactive%20oxygen%20species%20in%20plant%20programed%20cell%20death&amp;author=Y.%20Wang&amp;author=G.J.%20Loake&amp;author=C.%20Chu&amp;publication_year=2013&amp;journal=Front.%20Plant%20Sci&amp;volume=4&amp;pages=314" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23967004" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pradhan</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Naqvi</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Saraf</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mukherjee</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Dey</div> <div class="given-names">N.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Prediction and characterization of Tomato leaf curl New Delhi virus (ToLCNDV) responsive novel microRNAs in <em>Solanum lycopersicum</em></div>. <div class="source">Virus Res</div> ., <div class="volume">195</div>, <div class="fpage">183</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20and%20characterization%20of%20Tomato%20leaf%20curl%20New%20Delhi%20virus%20(ToLCNDV)%20responsive%20novel%20microRNAs%20in%20Solanum%20lycopersicum&amp;author=B.%20Pradhan&amp;author=A.R.%20Naqvi&amp;author=S.%20Saraf&amp;author=S.K.%20Mukherjee&amp;author=N.%20Dey&amp;publication_year=2015&amp;journal=Virus%20Res&amp;volume=195&amp;pages=183-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.virusres.2014.09.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.virusres.2014.09.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.virusres.2014.09.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25218481" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ouyang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Atamian</div> <div class="given-names">H.S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">MicroRNAs suppress NB domain genes in tomato that confer resistance to <em>Fusarium oxysporum</em></div>. <div class="source">PLoS Pathogens</div> , <div class="volume">10</div>, <div class="fpage">e1004464</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNAs%20suppress%20NB%20domain%20genes%20in%20tomato%20that%20confer%20resistance%20to%20Fusarium%20oxysporum&amp;author=S.%20Ouyang&amp;author=G.%20Park&amp;author=H.S.%20Atamian&amp;publication_year=2014&amp;journal=PLoS%20Pathogens&amp;volume=10&amp;pages=e1004464" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1004464" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1004464" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1004464"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25330340" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Axtell</div> <div class="given-names">M.J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Classification and comparison of small RNAs from plants</div>. <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">64</div>, <div class="fpage">137</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classification%20and%20comparison%20of%20small%20RNAs%20from%20plants&amp;author=M.J.%20Axtell&amp;publication_year=2013&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=64&amp;pages=137-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-050312-120043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-050312-120043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-050312-120043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23330790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Identification of chilling stress-responsive tomato microRNAs and their target genes by high-throughput sequencing and degradome analysis</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">1130</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20chilling%20stress-responsive%20tomato%20microRNAs%20and%20their%20target%20genes%20by%20high-throughput%20sequencing%20and%20degradome%20analysis&amp;author=X.%20Cao&amp;author=Z.%20Wu&amp;author=F.%20Jiang&amp;author=R.%20Zhou&amp;author=Z.%20Yang&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=1130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-1130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-1130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-1130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25519760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Candar-Cakir</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Arican</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">B.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Small RNA and degradome deep sequencing reveals drought-and tissue-specific micrornas and their important roles in drought-sensitive and drought-tolerant tomato genotypes</div>. <div class="source">Plant Biotechnol. J</div> ., <div class="volume">14</div>, <div class="fpage">1727</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20RNA%20and%20degradome%20deep%20sequencing%20reveals%20drought-and%20tissue-specific%20micrornas%20and%20their%20important%20roles%20in%20drought-sensitive%20and%20drought-tolerant%20tomato%20genotypes&amp;author=B.%20Candar-Cakir&amp;author=E.%20Arican&amp;author=B.%20Zhang&amp;publication_year=2016&amp;journal=Plant%20Biotechnol.%20J&amp;volume=14&amp;pages=1727-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2016.14.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2016.14.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2016.14.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26857916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Luan</div> <div class="given-names">Y.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The advance of tomato disease-related microRNAs</div>. <div class="source">Plant Cell Rep</div> ., <div class="volume">34</div>, <div class="fpage">1089</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20advance%20of%20tomato%20disease-related%20microRNAs&amp;author=W.%20Wang&amp;author=Y.%20Luan&amp;publication_year=2015&amp;journal=Plant%20Cell%20Rep&amp;volume=34&amp;pages=1089-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-015-1782-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-015-1782-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-015-1782-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25773761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liao</div> <div class="given-names">Q.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Analysis of tomato microRNAs expression profile induced by Cucumovirus and Tobamovirus infections</div>. <div class="source">J. Nanosci. Nanotechnol</div> ., <div class="volume">12</div>, <div class="fpage">143</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20tomato%20microRNAs%20expression%20profile%20induced%20by%20Cucumovirus%20and%20Tobamovirus%20infections&amp;author=J.%20Chen&amp;author=J.%20Feng&amp;author=Q.%20Liao&amp;publication_year=2012&amp;journal=J.%20Nanosci.%20Nanotechnol&amp;volume=12&amp;pages=143-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1166/jnn.2012.5112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1166%2Fjnn.2012.5112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1166%2Fjnn.2012.5112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22523958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Cui</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhai</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Meng</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">High-throughput sequencing reveals differential expression of miRNAs in tomato inoculated with <em>Phytophthora infestans</em></div>. <div class="source">Planta</div> , <div class="volume">241</div>, <div class="fpage">1405</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20sequencing%20reveals%20differential%20expression%20of%20miRNAs%20in%20tomato%20inoculated%20with%20Phytophthora%20infestans&amp;author=Y.%20Luan&amp;author=J.%20Cui&amp;author=J.%20Zhai&amp;author=J.%20Li&amp;author=L.%20Han&amp;author=J.%20Meng&amp;publication_year=2015&amp;journal=Planta&amp;volume=241&amp;pages=1405-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-015-2267-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-015-2267-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-015-2267-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25697288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Identification of microRNAs and their targets in tomato infected with <em>Cucumber mosaic</em> virus based on deep sequencing</div>. <div class="source">Planta</div> , <div class="volume">240</div>, <div class="fpage">1335</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20microRNAs%20and%20their%20targets%20in%20tomato%20infected%20with%20Cucumber%20mosaic%20virus%20based%20on%20deep%20sequencing&amp;author=J.%20Feng&amp;author=S.%20Liu&amp;author=M.%20Wang&amp;author=Q.%20Lang&amp;author=C.%20Jin&amp;publication_year=2014&amp;journal=Planta&amp;volume=240&amp;pages=1335-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-014-2158-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-014-2158-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-014-2158-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25204630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kulcheski</div> <div class="given-names">F.R.</div></div>, <div class="name string-name"><div class="surname">de Oliveira</div> <div class="given-names">L.F.</div></div>, <div class="name string-name"><div class="surname">Molina</div> <div class="given-names">L.G.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Identification of novel soybean microRNAs involved in abiotic and biotic stresses</div>. <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">307</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20novel%20soybean%20microRNAs%20involved%20in%20abiotic%20and%20biotic%20stresses&amp;author=F.R.%20Kulcheski&amp;author=L.F.%20de%20Oliveira&amp;author=L.G.%20Molina&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-307" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-307"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21663675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bilgin</div> <div class="given-names">D.D.</div></div>, <div class="name string-name"><div class="surname">Zavala</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Clough</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Ort</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">DeLucia</div> <div class="given-names">E.H.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Biotic stress globally downregulates photosynthesis genes</div>. <div class="source">Plant Cell Environ</div> ., <div class="volume">33</div>, <div class="fpage">1597</div>–<div class="lpage">613</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biotic%20stress%20globally%20downregulates%20photosynthesis%20genes&amp;author=D.D.%20Bilgin&amp;author=J.A.%20Zavala&amp;author=J.%20Zhu&amp;author=S.J.%20Clough&amp;author=D.R.%20Ort&amp;author=E.H.%20DeLucia&amp;publication_year=2010&amp;journal=Plant%20Cell%20Environ&amp;volume=33&amp;pages=1597-613" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2010.33.issue-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2010.33.issue-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2010.33.issue-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20444224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laborde</div> <div class="given-names">E.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Glutathione transferases as mediators of signaling pathways involved in cell proliferation and cell death</div>. <div class="source">Cell Death Different</div> ., <div class="volume">17</div>, <div class="fpage">1373</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glutathione%20transferases%20as%20mediators%20of%20signaling%20pathways%20involved%20in%20cell%20proliferation%20and%20cell%20death&amp;author=E.%20Laborde&amp;publication_year=2010&amp;journal=Cell%20Death%20Different&amp;volume=17&amp;pages=1373-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/cdd.2010.80" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fcdd.2010.80" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fcdd.2010.80"> </span></div> </div></div></div></div></div><div content-id="dsx003-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vogt</div> <div class="given-names">T.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Phenylpropanoid biosynthesis</div>. <div class="source">Mol. Plant</div> , <div class="volume">3</div>, <div class="fpage">2</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenylpropanoid%20biosynthesis&amp;author=T.%20Vogt&amp;publication_year=2010&amp;journal=Mol.%20Plant&amp;volume=3&amp;pages=2-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/mp/ssp106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmp%2Fssp106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmp%2Fssp106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20035037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Karlova</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">van Haarst</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Maliepaard</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Identification of microRNA targets in tomato fruit development using high-throughput sequencing and degradome analysis</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">64</div>, <div class="fpage">1863</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20microRNA%20targets%20in%20tomato%20fruit%20development%20using%20high-throughput%20sequencing%20and%20degradome%20analysis&amp;author=R.%20Karlova&amp;author=J.C.%20van%20Haarst&amp;author=C.%20Maliepaard&amp;publication_year=2013&amp;journal=J.%20Exp.%20Bot&amp;volume=64&amp;pages=1863-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/ert049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Fert049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Fert049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23487304" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shivaprasad</div> <div class="given-names">P.V.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">H.M.</div></div>, <div class="name string-name"><div class="surname">Patel</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Bond</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Santos</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Baulcombe</div> <div class="given-names">D.C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A microRNA superfamily regulates nucleotide binding site-leucine-rich repeats and other mRNAs</div>. <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">859</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microRNA%20superfamily%20regulates%20nucleotide%20binding%20site-leucine-rich%20repeats%20and%20other%20mRNAs&amp;author=P.V.%20Shivaprasad&amp;author=H.M.%20Chen&amp;author=K.%20Patel&amp;author=D.M.%20Bond&amp;author=B.A.%20Santos&amp;author=D.C.%20Baulcombe&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=859-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.095380" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.095380" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.095380"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22408077" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones-Rhoades</div> <div class="given-names">M.W.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">D.P.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">B.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">MicroRNAS and their regulatory roles in plants</div>. <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">57</div>, <div class="fpage">19</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNAS%20and%20their%20regulatory%20roles%20in%20plants&amp;author=M.W.%20Jones-Rhoades&amp;author=D.P.%20Bartel&amp;author=B.%20Bartel&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=57&amp;pages=19-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105218" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.57.032905.105218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.57.032905.105218"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16669754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jeong</div> <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhai</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Massive analysis of rice small RNAs: mechanistic implications of regulated microRNAs and variants for differential target RNA cleavage</div>. <div class="source">Plant Cell</div> , <div class="volume">23</div>, <div class="fpage">4185</div>–<div class="lpage">207</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massive%20analysis%20of%20rice%20small%20RNAs%3A%20mechanistic%20implications%20of%20regulated%20microRNAs%20and%20variants%20for%20differential%20target%20RNA%20cleavage&amp;author=D.H.%20Jeong&amp;author=S.%20Park&amp;author=J.%20Zhai&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=4185-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.089045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.089045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.089045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22158467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx003-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx003-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baldrich</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Campo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">T.T.</div></div>, <div class="name string-name"><div class="surname">Hsing</div> <div class="given-names">Y.I.</div></div>, <div class="name string-name"><div class="surname">San Segundo</div> <div class="given-names">B.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">MicroRNA-mediated regulation of gene expression in the response of rice plants to fungal elicitors</div>. <div class="source">RNA Biol</div> ., <div class="volume">12</div>, <div class="fpage">847</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNA-mediated%20regulation%20of%20gene%20expression%20in%20the%20response%20of%20rice%20plants%20to%20fungal%20elicitors&amp;author=P.%20Baldrich&amp;author=S.%20Campo&amp;author=M.T.%20Wu&amp;author=T.T.%20Liu&amp;author=Y.I.%20Hsing&amp;author=B.%20San%20Segundo&amp;publication_year=2015&amp;journal=RNA%20Biol&amp;volume=12&amp;pages=847-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/15476286.2015.1050577" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F15476286.2015.1050577" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F15476286.2015.1050577"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26083154" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3061403&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx003/1/dsx003_Supp.zip?Expires=2147483647&amp;Signature=kDDceQJTGeGA-Ztf0bmt91HO0XS~O7OLKlVS6TX3jGjKtpGEKtSFC9PHgRlQgidm-fDu1XT57v2SvM2L5acIYb9c6Com6ZetVYIqzPRg~30WB~sZCSFZeFTlYnHBl5cLE9lcJSM9uy1b32fT8NmrtLXph92-9ksFd4W6Xz8s9r~n6zSYVxTtHrUIzOy534enKrmV6yvvpjooqy0F-JMldCPQtg5IQ8YTTHB8LmyMm08EVEajeo2x~kbPYOTiL~M1HoC1uQ4aX6E8b1AHn6RZOEahUg8yxinvPzKs3CGug5KsPFt06lEXc73~h-ivQ3yebqtDlpRR0wK2xUOyKw2~Lg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,184</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx003"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3061403"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f3%2f235%2f3061403">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000404439600002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F3%2F235%2F3061403" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3061403'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Integrated miRNA and mRNA expression profiling reveals the response regulators of a susceptible tomato cultivar to early blight disease" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx003");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f235%2f3061403">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

