

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":973,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/2/173/4633668","@type":"ScholarlyArticle","name":"Draft genome analysis provides insights into the fiber yield, crude protein biosynthesis, and vegetative growth of domesticated ramie ( Boehmeria nivea  L. Gaud)","datePublished":"2017-11-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2018-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx047","keywords":["ramie","draft genome","high-density genetic map","positively selected gene"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/2/173/4633668","author":[{"name":"Liu, Chan","affiliation":"Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China","@type":"Person"},{"name":"Zeng, Liangbin","affiliation":"Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China","@type":"Person"},{"name":"Zhu, Siyuan","affiliation":"Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China","@type":"Person"},{"name":"Wu, Lingqing","affiliation":"Novogene Bioinformatics Institute, Beijing, China","@type":"Person"},{"name":"Wang, Yanzhou","affiliation":"Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China","@type":"Person"},{"name":"Tang, Shouwei","affiliation":"Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China","@type":"Person"},{"name":"Wang, Hongwu","affiliation":"Xianning Agriculture Academy of sciences, Hubei, China","@type":"Person"},{"name":"Zheng, Xia","affiliation":"Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China","@type":"Person"},{"name":"Zhao, Jian","affiliation":"Novogene Bioinformatics Institute, Beijing, China","@type":"Person"},{"name":"Chen, Xiaorong","affiliation":"Yichun Institute of Agricultural Sciences, Jiangxi, China","@type":"Person"},{"name":"Dai, Qiuzhong","affiliation":"Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China","@type":"Person"},{"name":"Liu, Touming","affiliation":"Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China","@type":"Person"}],"description":"Abstract.  Plentiful bast fiber, a high crude protein content, and vigorous vegetative growth make ramie a popular fiber and forage crop. Here, we report the dr","pageStart":"173","pageEnd":"181","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Draft genome analysis provides insights into the fiber yield, crude protein biosynthesis, and vegetative growth of domesticated ramie ( Boehmeria nivea  L. Gaud)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Draft genome analysis provides insights into the fiber yield, crude protein biosynthesis, and vegetative growth of domesticated ramie ( Boehmeria nivea  L. Gaud)" />
    <meta property="og:description" content="Abstract.  Plentiful bast fiber, a high crude protein content, and vigorous vegetative growth make ramie a popular fiber and forage crop. Here, we report the dr" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/2/173/4633668" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-11-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Liu, Chan" /><meta name="citation_author_institution" content="Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China" /><meta name="citation_author" content="Zeng, Liangbin" /><meta name="citation_author_institution" content="Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China" /><meta name="citation_author" content="Zhu, Siyuan" /><meta name="citation_author_institution" content="Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China" /><meta name="citation_author" content="Wu, Lingqing" /><meta name="citation_author_institution" content="Novogene Bioinformatics Institute, Beijing, China" /><meta name="citation_author" content="Wang, Yanzhou" /><meta name="citation_author_institution" content="Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China" /><meta name="citation_author" content="Tang, Shouwei" /><meta name="citation_author_institution" content="Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China" /><meta name="citation_author" content="Wang, Hongwu" /><meta name="citation_author_institution" content="Xianning Agriculture Academy of sciences, Hubei, China" /><meta name="citation_author" content="Zheng, Xia" /><meta name="citation_author_institution" content="Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China" /><meta name="citation_author" content="Zhao, Jian" /><meta name="citation_author_institution" content="Novogene Bioinformatics Institute, Beijing, China" /><meta name="citation_author" content="Chen, Xiaorong" /><meta name="citation_author_institution" content="Yichun Institute of Agricultural Sciences, Jiangxi, China" /><meta name="citation_author" content="Dai, Qiuzhong" /><meta name="citation_author_institution" content="Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China" /><meta name="citation_author" content="Liu, Touming" /><meta name="citation_author_institution" content="Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China" /><meta name="citation_title" content="Draft genome analysis provides insights into the fiber yield, crude protein biosynthesis, and vegetative growth of domesticated ramie (Boehmeria nivea L. Gaud)" /><meta name="citation_firstpage" content="173" /><meta name="citation_lastpage" content="181" /><meta name="citation_doi" content="10.1093/dnares/dsx047" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2018/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Study on selection and breeding of new feed ramie variety Zhongsizhu no.1;  citation_journal_title=Plant Fibers Prod;  citation_year=2005;  citation_volume=27;  citation_pages=1-4; " /><meta name="citation_reference" content="citation_title=De novo assembly and characterization of transcriptome using Illumina paired-end sequencing and identification of CesA gene in ramie (Boehmeria nivea L. Gaud);  citation_journal_title=BMC Genomic;  citation_year=2013;  citation_volume=14;  citation_pages=125" /><meta name="citation_reference" content="citation_title=Transcriptome profiling and identification of transcription factors in ramie (Boehmerianivea L. Gaud) in response to PEG treatment, using illumina paired-end sequencing technology;  citation_journal_title=Int. J. Mol. Sci;  citation_year=2015;  citation_volume=16;  citation_pages=3493-511; " /><meta name="citation_reference" content="citation_title=Transcriptome comparison reveals the patterns of selection in domesticated and wild ramie (Boehmerianivea L. Gaud);  citation_journal_title=Plant Mol. Biol;  citation_year=2014;  citation_volume=86;  citation_pages=85-92; " /><meta name="citation_reference" content="citation_title=QTL mapping for fiber yield-related traits by constructing the first genetic linkage map in ramie (Boehmerianivea L. Gaud);  citation_journal_title=Mol. Breeding;  citation_year=2014;  citation_volume=34;  citation_pages=883-92; " /><meta name="citation_reference" content="citation_title=Assembly: a resource for assembled genomes at NCBI;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D73-80; " /><meta name="citation_reference" content="citation_title=Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e24" /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=The Jujube genome provides insights into genome evolution and the domestication of sweetness/acidity taste in fruit trees;  citation_journal_title=PLoS Genet;  citation_year=2016;  citation_volume=12;  citation_pages=e1006433" /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1061-7; " /><meta name="citation_reference" content="citation_title=Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources;  citation_journal_title=BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=62" /><meta name="citation_reference" content="citation_title=The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000;  citation_journal_title=Nucleic Acids Res;  citation_year=2000;  citation_volume=28;  citation_pages=45-8; " /><meta name="citation_reference" content="citation_title=KEGG: kyoto encyclopedia of genes and genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2000;  citation_volume=28;  citation_pages=27-30; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology;  citation_journal_title=Nat. Genet;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=InterPro and InterProScan: tools for protein sequence classification and comparison;  citation_journal_title=Methods Mol. Biol;  citation_year=2007;  citation_volume=396;  citation_pages=59-70; " /><meta name="citation_reference" content="citation_title=WEGO: a web tool for plotting GO annotations;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W293-7; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of the mulberry tree Morus notabilis;  citation_journal_title=Nat. Commun;  citation_year=2013;  citation_volume=4;  citation_pages=2445" /><meta name="citation_reference" content="citation_title=The complex jujube genome provides insights into fruit tree biology;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=5315" /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla; citation_author=The French–Italian Public Consortium for Grapevine Genome Characterization;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=The draft genome of watermelon (Citrullus lanatus) and resequencing of 20 diverse accessions;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=51-8; " /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=The draft genome of a diploid cotton Gossypium raimondii;  citation_journal_title=Nat. Genet;  citation_year=2012;  citation_volume=44;  citation_pages=1098-103; " /><meta name="citation_reference" content="citation_title=A draft sequence of the rice genome (Oryza sativa L. ssp. indica);  citation_journal_title=Science;  citation_year=2002;  citation_volume=296;  citation_pages=79-92; " /><meta name="citation_reference" content="citation_title=The genome of flax (Linum usitatissimum) assembled de novofrom short shotgun sequence reads;  citation_journal_title=Plant J;  citation_year=2012;  citation_volume=72;  citation_pages=461-73; " /><meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes;  citation_journal_title=Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; " /><meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=1792-7; " /><meta name="citation_reference" content="citation_title=A rapid bootstrap algorithm for the RAxML Web servers;  citation_journal_title=Syst. Biol.;  citation_year=2008;  citation_volume=57;  citation_pages=758-71; " /><meta name="citation_reference" content="citation_title=HMMER web server: interactive sequence similarity searching;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W29-37; " /><meta name="citation_reference" content="TreeSoft: TreeBeST ( http://treesoft.sourceforge.net/treebest.shtm ). " /><meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level;  citation_journal_title=Mol. Biol. Evol;  citation_year=2005;  citation_volume=22;  citation_pages=2472-9; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=Gene ontology analysis for RNA-seq: accounting for selection bias;  citation_journal_title=Genome Biol;  citation_year=2010;  citation_volume=11;  citation_pages=R14" /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing;  citation_journal_title=J. Roy. Stat. Soc. B;  citation_year=1995;  citation_volume=57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Genetic dissection of maize plant architecture with an ultra-high density bin map based on recombinant inbred lines;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=178" /><meta name="citation_reference" content="citation_title=Construction of a high-density genetic map of Ziziphus jujube Mill. using genotyping by sequencing technology;  citation_journal_title=Tree Genetics Genomes;  citation_year=2016;  citation_volume=12;  citation_pages=76" /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species;  citation_journal_title=Genet. Res;  citation_year=2011;  citation_volume=93;  citation_pages=343-9; " /><meta name="citation_reference" content="citation_title=The estimation of map distance from recombination values;  citation_journal_title=Ann. Eugenics;  citation_year=1943;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=Identification of 32 full-length NAC transcription factors in ramie (Boehmeria nivea L. Gaud) and characterization of the expression pattern of these genes;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2014;  citation_volume=289;  citation_pages=675-84; " /><meta name="citation_reference" content="citation_title=Development and characterization of 1, 827 expressed sequence tag-derived simple sequence repeat markers for ramie (Boehmeria nivea L. Gaud);  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e60346" /><meta name="citation_reference" content="citation_title=Identification of quantitative trait loci for flowering time traits in ramie (Boehmeria nivea L. Gaud);  citation_journal_title=Euphytica;  citation_year=2016;  citation_volume=210;  citation_pages=367-74; " /><meta name="citation_reference" content="citation_title=Transcriptome profiling using pyrosequencing shows genes associated with bast fiber development in ramie (Boehmerianivea L.);  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=919" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing and profiling of expressed genes in phloem and xylem of ramie (Boehmerianivea L. Gaud);  citation_journal_title=PLoS One.;  citation_year=2014;  citation_volume=9;  citation_pages=e110623" /><meta name="citation_reference" content="citation_title=A phylogenetic classification of the land plants toaccompany APG III;  citation_journal_title=Bot. J. Linn. Soc;  citation_year=2009;  citation_volume=161;  citation_pages=122-7; " /><meta name="citation_reference" content="citation_title=Prime Time for Cellulose;  citation_journal_title=Science;  citation_year=2002;  citation_volume=295;  citation_pages=59-60; " /><meta name="citation_reference" content="citation_title=Cellulose Biosynthesis in Plants: from Genes to Rosettes;  citation_journal_title=Plant Cell Physiol;  citation_year=2002;  citation_volume=43;  citation_pages=1407-20; " /><meta name="citation_reference" content="citation_title=Higher plants contain homologs of the bacterial celA genes encoding the catalytic subunit of cellulose synthase;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=12637-42; " /><meta name="citation_reference" content="citation_title=An update on the nomenclature for the cellulose synthase genes in Populus;  citation_journal_title=Trends Plant Sci;  citation_year=2009;  citation_volume=14;  citation_pages=1360-85; " /><meta name="citation_reference" content="citation_title=The CesA gene family of barley quantitative analysis of transcripts reveals two groups of co-expressed genes;  citation_journal_title=Plant Physiol;  citation_year=2004;  citation_volume=134;  citation_pages=224-36; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of homeologous cellulose synthase catalytic subunit 2 genes from allotetraploid cotton (Gossypium hirsutum L.);  citation_journal_title=Gene;  citation_year=2012;  citation_volume=494;  citation_pages=181-9; " /><meta name="citation_reference" content="citation_title=The cellulose synthase superfamily;  citation_journal_title=Plant Physiol;  citation_year=2000;  citation_volume=124;  citation_pages=495-8; " /><meta name="citation_reference" content="citation_title=Three distinct rice cellulose synthase catalytic subunit genes required for cellulose synthesis in the secondary wall;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=133;  citation_pages=73-83; " /><meta name="citation_reference" content="citation_title=cDNA cloning and expression of two cellulose synthase genes from Boehmeria nivea;  citation_journal_title=Acta. Agronomica Sinca;  citation_year=2014;  citation_volume=40;  citation_pages=1925-35; " /><meta name="citation_reference" content="Tian Z., Yi R., Chen J., Guo Q., Zhang X.  2008 , Cloning and expression of cellulose synthase gene in ramie [ Boehmerianivea (Linn.) Gaud.], Acta. Agronomica Sinca, 34 , 76–83. " /><meta name="citation_reference" content="citation_title=Walls are thin 1 (WAT1), an Arabidopsis homolog of Medicago truncatula NODULIN21, is a tonoplast-localized protein required for secondary wall formation in fibers;  citation_journal_title=Plant J;  citation_year=2010;  citation_volume=63;  citation_pages=469-83; " /><meta name="citation_reference" content="citation_publisher=ASA, CSSA, SSSA, Madison, Wisconsin;  citation_title=Physiology and Determination of Crop Yield;  citation_year=1994;  citation_pages=285-302; " /><meta name="citation_reference" content="citation_title=Nitrate: nutrient and signal for plant growth;  citation_journal_title=Plant Cell;  citation_year=1995;  citation_volume=7;  citation_pages=859-68; " /><meta name="citation_reference" content="citation_title=The molecular-genetics of nitrogen assimilation into amino acids in higher plants;  citation_journal_title=Annu. Rev. Plant Physiol. Plant Mol. Biol;  citation_year=1996;  citation_volume=47;  citation_pages=569-93; " /><meta name="citation_reference" content="citation_title=Engineering nitrogen use efficiency with alanine aminotransferase;  citation_journal_title=Can. J. Bot;  citation_year=2007;  citation_volume=85;  citation_pages=252-62; " /><meta name="citation_reference" content="citation_title=Identification and expression analysis of glutamine synthetase genes in ramie (Boehmeria nivea L. Gaud);  citation_journal_title=Open Life Sci;  citation_year=2015;  citation_volume=10;  citation_pages=299-309; " /><meta name="citation_reference" content="citation_title=Characterization of a glutamine synthetase gene BnGS1-2 from ramie (Boehmeria nivea L. Gaud) and biochemical assays of BnGS1-2-over-expressing transgenic tobacco;  citation_journal_title=Acta Physiol. Plant;  citation_year=2015;  citation_volume=37;  citation_pages=1742" /><meta name="citation_reference" content="citation_title=Members of BTB gene family of scaffold proteins suppress nitrate uptake and nitrogen use efficiency;  citation_journal_title=Plant Physiol;  citation_year=2016;  citation_volume=171;  citation_pages=1523-32; " /><meta name="citation_reference" content="citation_title=Hormonal control of nitrogen acquisition: roles of auxin, abscisic acid, and cytokinin;  citation_journal_title=J. Exp. Bot;  citation_year=2011;  citation_volume=62;  citation_pages=1399-409; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/2/173/24662196/dsx047.pdf" /><meta name="description" content="Abstract.  Plentiful bast fiber, a high crude protein content, and vigorous vegetative growth make ramie a popular fiber and forage crop. Here, we report the dr" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/2/173/4633668" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/2/173/4633668" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/2/173/4633668");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsx047';
    </script>
<title>
	Draft genome analysis provides insights into the fiber yield, crude protein biosynthesis, and vegetative growth of domesticated ramie (Boehmeria nivea L. Gaud) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4633668" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfiZATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBDaGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJhc3QgRmliZXIgQ3JvcHMsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIENoYW5nc2hhLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlplbmcsIExpYW5nYmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJhc3QgRmliZXIgQ3JvcHMsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIENoYW5nc2hhLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpodSwgU2l5dWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJhc3QgRmliZXIgQ3JvcHMsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIENoYW5nc2hhLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ild1LCBMaW5ncWluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5vdm9nZW5lIEJpb2luZm9ybWF0aWNzIEluc3RpdHV0ZSwgQmVpamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBZYW56aG91IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJhc3QgRmliZXIgQ3JvcHMsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIENoYW5nc2hhLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbmcsIFNob3V3ZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmFzdCBGaWJlciBDcm9wcywgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgQ2hhbmdzaGEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgSG9uZ3d1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iWGlhbm5pbmcgQWdyaWN1bHR1cmUgQWNhZGVteSBvZiBzY2llbmNlcywgSHViZWksIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhlbmcsIFhpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCYXN0IEZpYmVyIENyb3BzLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBDaGFuZ3NoYSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFvLCBKaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTm92b2dlbmUgQmlvaW5mb3JtYXRpY3MgSW5zdGl0dXRlLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIFhpYW9yb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iWWljaHVuIEluc3RpdHV0ZSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEppYW5neGksIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGFpLCBRaXV6aG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCYXN0IEZpYmVyIENyb3BzLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBDaGFuZ3NoYSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaXUsIFRvdW1pbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmFzdCBGaWJlciBDcm9wcywgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgQ2hhbmdzaGEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEcmFmdCBnZW5vbWUgYW5hbHlzaXMgcHJvdmlkZXMgaW5zaWdodHMgaW50byB0aGUgZmliZXIgeWllbGQsIGNydWRlIHByb3RlaW4gYmlvc3ludGhlc2lzLCBhbmQgdmVnZXRhdGl2ZSBncm93dGggb2YgZG9tZXN0aWNhdGVkIHJhbWllIChCb2VobWVyaWEgbml2ZWEgTC4gR2F1ZCkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxNzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE4MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDA0NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3R1ZHkgb24gc2VsZWN0aW9uIGFuZCBicmVlZGluZyBvZiBuZXcgZmVlZCByYW1pZSB2YXJpZXR5IFpob25nc2l6aHUgbm8uMTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgRmliZXJzIFByb2Q7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgdHJhbnNjcmlwdG9tZSB1c2luZyBJbGx1bWluYSBwYWlyZWQtZW5kIHNlcXVlbmNpbmcgYW5kIGlkZW50aWZpY2F0aW9uIG9mIENlc0EgZ2VuZSBpbiByYW1pZSAoQm9laG1lcmlhIG5pdmVhIEwuIEdhdWQpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pYzsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgcHJvZmlsaW5nIGFuZCBpZGVudGlmaWNhdGlvbiBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgaW4gcmFtaWUgKEJvZWhtZXJpYW5pdmVhIEwuIEdhdWQpIGluIHJlc3BvbnNlIHRvIFBFRyB0cmVhdG1lbnQsIHVzaW5nIGlsbHVtaW5hIHBhaXJlZC1lbmQgc2VxdWVuY2luZyB0ZWNobm9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIE1vbC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MzQ5My01MTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgY29tcGFyaXNvbiByZXZlYWxzIHRoZSBwYXR0ZXJucyBvZiBzZWxlY3Rpb24gaW4gZG9tZXN0aWNhdGVkIGFuZCB3aWxkIHJhbWllIChCb2VobWVyaWFuaXZlYSBMLiBHYXVkKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTg2OyAgY2l0YXRpb25fcGFnZXM9ODUtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTCBtYXBwaW5nIGZvciBmaWJlciB5aWVsZC1yZWxhdGVkIHRyYWl0cyBieSBjb25zdHJ1Y3RpbmcgdGhlIGZpcnN0IGdlbmV0aWMgbGlua2FnZSBtYXAgaW4gcmFtaWUgKEJvZWhtZXJpYW5pdmVhIEwuIEdhdWQpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkaW5nOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9ODgzLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NlbWJseTogYSByZXNvdXJjZSBmb3IgYXNzZW1ibGVkIGdlbm9tZXMgYXQgTkNCSTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz1ENzMtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlsbHVtaW5hIG1hdGUtcGFpcmVkIEROQSBzZXF1ZW5jaW5nLWxpYnJhcnkgcHJlcGFyYXRpb24gdXNpbmcgQ3JlLUxveCByZWNvbWJpbmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPWUyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFzdCwgbG9jay1mcmVlIGFwcHJvYWNoIGZvciBlZmZpY2llbnQgcGFyYWxsZWwgY291bnRpbmcgb2Ygb2NjdXJyZW5jZXMgb2Ygay1tZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTc2NC03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEp1anViZSBnZW5vbWUgcHJvdmlkZXMgaW5zaWdodHMgaW50byBnZW5vbWUgZXZvbHV0aW9uIGFuZCB0aGUgZG9tZXN0aWNhdGlvbiBvZiBzd2VldG5lc3MvYWNpZGl0eSB0YXN0ZSBpbiBmcnVpdCB0cmVlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA2NDMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNFR01BOiBhIHBpcGVsaW5lIHRvIGFjY3VyYXRlbHkgYW5ub3RhdGUgY29yZSBnZW5lcyBpbiBldWthcnlvdGljIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTA2MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHByZWRpY3Rpb24gaW4gZXVrYXJ5b3RlcyB3aXRoIGEgZ2VuZXJhbGl6ZWQgaGlkZGVuIE1hcmtvdiBtb2RlbCB0aGF0IHVzZXMgaGludHMgZnJvbSBleHRlcm5hbCBzb3VyY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNXSVNTLVBST1QgcHJvdGVpbiBzZXF1ZW5jZSBkYXRhYmFzZSBhbmQgaXRzIHN1cHBsZW1lbnQgVHJFTUJMIGluIDIwMDA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NDUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S0VHRzoga3lvdG8gZW5jeWNsb3BlZGlhIG9mIGdlbmVzIGFuZCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG9udG9sb2d5OiB0b29sIGZvciB0aGUgdW5pZmljYXRpb24gb2YgYmlvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvIGFuZCBJbnRlclByb1NjYW46IHRvb2xzIGZvciBwcm90ZWluIHNlcXVlbmNlIGNsYXNzaWZpY2F0aW9uIGFuZCBjb21wYXJpc29uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NZXRob2RzIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOTY7ICBjaXRhdGlvbl9wYWdlcz01OS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V0VHTzogYSB3ZWIgdG9vbCBmb3IgcGxvdHRpbmcgR08gYW5ub3RhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9VzI5My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBmbG93ZXJpbmcgcGxhbnQgQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDg7ICBjaXRhdGlvbl9wYWdlcz03OTYtODE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIG11bGJlcnJ5IHRyZWUgTW9ydXMgbm90YWJpbGlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjQ0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV4IGp1anViZSBnZW5vbWUgcHJvdmlkZXMgaW5zaWdodHMgaW50byBmcnVpdCB0cmVlIGJpb2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01MzE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdyYXBldmluZSBnZW5vbWUgc2VxdWVuY2Ugc3VnZ2VzdHMgYW5jZXN0cmFsIGhleGFwbG9pZGl6YXRpb24gaW4gbWFqb3IgYW5naW9zcGVybSBwaHlsYTsgY2l0YXRpb25fYXV0aG9yPVRoZSBGcmVuY2jigJNJdGFsaWFuIFB1YmxpYyBDb25zb3J0aXVtIGZvciBHcmFwZXZpbmUgR2Vub21lIENoYXJhY3Rlcml6YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDk7ICBjaXRhdGlvbl9wYWdlcz00NjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRyYWZ0IGdlbm9tZSBvZiB3YXRlcm1lbG9uIChDaXRydWxsdXMgbGFuYXR1cykgYW5kIHJlc2VxdWVuY2luZyBvZiAyMCBkaXZlcnNlIGFjY2Vzc2lvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz01MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIHBhbGFlb3BvbHlwbG9pZCBzb3liZWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDYzOyAgY2l0YXRpb25fcGFnZXM9MTc4LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHJhZnQgZ2Vub21lIG9mIGEgZGlwbG9pZCBjb3R0b24gR29zc3lwaXVtIHJhaW1vbmRpaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTEwOTgtMTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRyYWZ0IHNlcXVlbmNlIG9mIHRoZSByaWNlIGdlbm9tZSAoT3J5emEgc2F0aXZhIEwuIHNzcC4gaW5kaWNhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz03OS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBmbGF4IChMaW51bSB1c2l0YXRpc3NpbXVtKSBhc3NlbWJsZWQgZGUgbm92b2Zyb20gc2hvcnQgc2hvdGd1biBzZXF1ZW5jZSByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03MjsgIGNpdGF0aW9uX3BhZ2VzPTQ2MS03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3J0aG9NQ0w6IGlkZW50aWZpY2F0aW9uIG9mIG9ydGhvbG9nIGdyb3VwcyBmb3IgZXVrYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIxNzgtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1VU0NMRTogbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHdpdGggaGlnaCBhY2N1cmFjeSBhbmQgaGlnaCB0aHJvdWdocHV0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTE3OTItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByYXBpZCBib290c3RyYXAgYWxnb3JpdGhtIGZvciB0aGUgUkF4TUwgV2ViIHNlcnZlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9NzU4LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ITU1FUiB3ZWIgc2VydmVyOiBpbnRlcmFjdGl2ZSBzZXF1ZW5jZSBzaW1pbGFyaXR5IHNlYXJjaGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XMjktMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlRyZWVTb2Z0OiBUcmVlQmVTVCAoIGh0dHA6Ly90cmVlc29mdC5zb3VyY2Vmb3JnZS5uZXQvdHJlZWJlc3Quc2h0bSApLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIG9mIGFuIGltcHJvdmVkIGJyYW5jaC1zaXRlIGxpa2VsaWhvb2QgbWV0aG9kIGZvciBkZXRlY3RpbmcgcG9zaXRpdmUgc2VsZWN0aW9uIGF0IHRoZSBtb2xlY3VsYXIgbGV2ZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTI0NzItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFNTCA0OiBwaHlsb2dlbmV0aWMgYW5hbHlzaXMgYnkgbWF4aW11bSBsaWtlbGlob29kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTg2LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG9udG9sb2d5IGFuYWx5c2lzIGZvciBSTkEtc2VxOiBhY2NvdW50aW5nIGZvciBzZWxlY3Rpb24gYmlhczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1SMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sbGluZyB0aGUgZmFsc2UgZGlzY292ZXJ5IHJhdGU6IGEgcHJhY3RpY2FsIGFuZCBwb3dlcmZ1bCBhcHByb2FjaCB0byBtdWx0aXBsZSB0ZXN0aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBSb3kuIFN0YXQuIFNvYy4gQjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT01NzsgIGNpdGF0aW9uX3BhZ2VzPTI4OS0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGlzc2VjdGlvbiBvZiBtYWl6ZSBwbGFudCBhcmNoaXRlY3R1cmUgd2l0aCBhbiB1bHRyYS1oaWdoIGRlbnNpdHkgYmluIG1hcCBiYXNlZCBvbiByZWNvbWJpbmFudCBpbmJyZWQgbGluZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE3OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGhpZ2gtZGVuc2l0eSBnZW5ldGljIG1hcCBvZiBaaXppcGh1cyBqdWp1YmUgTWlsbC4gdXNpbmcgZ2Vub3R5cGluZyBieSBzZXF1ZW5jaW5nIHRlY2hub2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXRpY3MgR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIGFsaWdubWVudC9tYXAgZm9ybWF0IGFuZCBTQU10b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcG9pbnQgbWF4aW11bSBsaWtlbGlob29kIG1hcHBpbmcgaW4gYSBmdWxsLXNpYiBmYW1pbHkgb2YgYW4gb3V0YnJlZWRpbmcgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTM0My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXN0aW1hdGlvbiBvZiBtYXAgZGlzdGFuY2UgZnJvbSByZWNvbWJpbmF0aW9uIHZhbHVlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBFdWdlbmljczsgIGNpdGF0aW9uX3llYXI9MTk0MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE3Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiAzMiBmdWxsLWxlbmd0aCBOQUMgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGluIHJhbWllIChCb2VobWVyaWEgbml2ZWEgTC4gR2F1ZCkgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIGV4cHJlc3Npb24gcGF0dGVybiBvZiB0aGVzZSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9Mjg5OyAgY2l0YXRpb25fcGFnZXM9Njc1LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiAxLCA4MjcgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZy1kZXJpdmVkIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VycyBmb3IgcmFtaWUgKEJvZWhtZXJpYSBuaXZlYSBMLiBHYXVkKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2MDM0NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGZvciBmbG93ZXJpbmcgdGltZSB0cmFpdHMgaW4gcmFtaWUgKEJvZWhtZXJpYSBuaXZlYSBMLiBHYXVkKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIxMDsgIGNpdGF0aW9uX3BhZ2VzPTM2Ny03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBwcm9maWxpbmcgdXNpbmcgcHlyb3NlcXVlbmNpbmcgc2hvd3MgZ2VuZXMgYXNzb2NpYXRlZCB3aXRoIGJhc3QgZmliZXIgZGV2ZWxvcG1lbnQgaW4gcmFtaWUgKEJvZWhtZXJpYW5pdmVhIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9OTE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIGFuZCBwcm9maWxpbmcgb2YgZXhwcmVzc2VkIGdlbmVzIGluIHBobG9lbSBhbmQgeHlsZW0gb2YgcmFtaWUgKEJvZWhtZXJpYW5pdmVhIEwuIEdhdWQpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZS47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMTA2MjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBoeWxvZ2VuZXRpYyBjbGFzc2lmaWNhdGlvbiBvZiB0aGUgbGFuZCBwbGFudHMgdG9hY2NvbXBhbnkgQVBHIElJSTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qm90LiBKLiBMaW5uLiBTb2M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9MTIyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1lIFRpbWUgZm9yIENlbGx1bG9zZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTU7ICBjaXRhdGlvbl9wYWdlcz01OS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbHVsb3NlIEJpb3N5bnRoZXNpcyBpbiBQbGFudHM6IGZyb20gR2VuZXMgdG8gUm9zZXR0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTE0MDctMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hlciBwbGFudHMgY29udGFpbiBob21vbG9ncyBvZiB0aGUgYmFjdGVyaWFsIGNlbEEgZ2VuZXMgZW5jb2RpbmcgdGhlIGNhdGFseXRpYyBzdWJ1bml0IG9mIGNlbGx1bG9zZSBzeW50aGFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz0xMjYzNy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gdXBkYXRlIG9uIHRoZSBub21lbmNsYXR1cmUgZm9yIHRoZSBjZWxsdWxvc2Ugc3ludGhhc2UgZ2VuZXMgaW4gUG9wdWx1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEzNjAtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDZXNBIGdlbmUgZmFtaWx5IG9mIGJhcmxleSBxdWFudGl0YXRpdmUgYW5hbHlzaXMgb2YgdHJhbnNjcmlwdHMgcmV2ZWFscyB0d28gZ3JvdXBzIG9mIGNvLWV4cHJlc3NlZCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz0yMjQtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb25pbmcgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgaG9tZW9sb2dvdXMgY2VsbHVsb3NlIHN5bnRoYXNlIGNhdGFseXRpYyBzdWJ1bml0IDIgZ2VuZXMgZnJvbSBhbGxvdGV0cmFwbG9pZCBjb3R0b24gKEdvc3N5cGl1bSBoaXJzdXR1bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDk0OyAgY2l0YXRpb25fcGFnZXM9MTgxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjZWxsdWxvc2Ugc3ludGhhc2Ugc3VwZXJmYW1pbHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9NDk1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRocmVlIGRpc3RpbmN0IHJpY2UgY2VsbHVsb3NlIHN5bnRoYXNlIGNhdGFseXRpYyBzdWJ1bml0IGdlbmVzIHJlcXVpcmVkIGZvciBjZWxsdWxvc2Ugc3ludGhlc2lzIGluIHRoZSBzZWNvbmRhcnkgd2FsbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzM7ICBjaXRhdGlvbl9wYWdlcz03My04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Y0ROQSBjbG9uaW5nIGFuZCBleHByZXNzaW9uIG9mIHR3byBjZWxsdWxvc2Ugc3ludGhhc2UgZ2VuZXMgZnJvbSBCb2VobWVyaWEgbml2ZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFjdGEuIEFncm9ub21pY2EgU2luY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0xOTI1LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJUaWFuIFouLCBZaSBSLiwgQ2hlbiBKLiwgR3VvIFEuLCBaaGFuZyBYLiAgMjAwOCAsIENsb25pbmcgYW5kIGV4cHJlc3Npb24gb2YgY2VsbHVsb3NlIHN5bnRoYXNlIGdlbmUgaW4gcmFtaWUgWyBCb2VobWVyaWFuaXZlYSAoTGlubi4pIEdhdWQuXSwgQWN0YS4gQWdyb25vbWljYSBTaW5jYSwgMzQgLCA3NuKAkzgzLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XYWxscyBhcmUgdGhpbiAxIChXQVQxKSwgYW4gQXJhYmlkb3BzaXMgaG9tb2xvZyBvZiBNZWRpY2FnbyB0cnVuY2F0dWxhIE5PRFVMSU4yMSwgaXMgYSB0b25vcGxhc3QtbG9jYWxpemVkIHByb3RlaW4gcmVxdWlyZWQgZm9yIHNlY29uZGFyeSB3YWxsIGZvcm1hdGlvbiBpbiBmaWJlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz00NjktODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1BU0EsIENTU0EsIFNTU0EsIE1hZGlzb24sIFdpc2NvbnNpbjsgIGNpdGF0aW9uX3RpdGxlPVBoeXNpb2xvZ3kgYW5kIERldGVybWluYXRpb24gb2YgQ3JvcCBZaWVsZDsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3BhZ2VzPTI4NS0zMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5pdHJhdGU6IG51dHJpZW50IGFuZCBzaWduYWwgZm9yIHBsYW50IGdyb3d0aDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ODU5LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyLWdlbmV0aWNzIG9mIG5pdHJvZ2VuIGFzc2ltaWxhdGlvbiBpbnRvIGFtaW5vIGFjaWRzIGluIGhpZ2hlciBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gUGxhbnQgUGh5c2lvbC4gUGxhbnQgTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NTY5LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmdpbmVlcmluZyBuaXRyb2dlbiB1c2UgZWZmaWNpZW5jeSB3aXRoIGFsYW5pbmUgYW1pbm90cmFuc2ZlcmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2FuLiBKLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0yNTItNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIGdsdXRhbWluZSBzeW50aGV0YXNlIGdlbmVzIGluIHJhbWllIChCb2VobWVyaWEgbml2ZWEgTC4gR2F1ZCk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU9wZW4gTGlmZSBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIGEgZ2x1dGFtaW5lIHN5bnRoZXRhc2UgZ2VuZSBCbkdTMS0yIGZyb20gcmFtaWUgKEJvZWhtZXJpYSBuaXZlYSBMLiBHYXVkKSBhbmQgYmlvY2hlbWljYWwgYXNzYXlzIG9mIEJuR1MxLTItb3Zlci1leHByZXNzaW5nIHRyYW5zZ2VuaWMgdG9iYWNjbzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWN0YSBQaHlzaW9sLiBQbGFudDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTE3NDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZW1iZXJzIG9mIEJUQiBnZW5lIGZhbWlseSBvZiBzY2FmZm9sZCBwcm90ZWlucyBzdXBwcmVzcyBuaXRyYXRlIHVwdGFrZSBhbmQgbml0cm9nZW4gdXNlIGVmZmljaWVuY3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTcxOyAgY2l0YXRpb25fcGFnZXM9MTUyMy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9ybW9uYWwgY29udHJvbCBvZiBuaXRyb2dlbiBhY3F1aXNpdGlvbjogcm9sZXMgb2YgYXV4aW4sIGFic2Npc2ljIGFjaWQsIGFuZCBjeXRva2luaW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9MTM5OS00MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS8yLzE3My8yNDY2MjE5Ni9kc3gwNDcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFBsZW50aWZ1bCBiYXN0IGZpYmVyLCBhIGhpZ2ggY3J1ZGUgcHJvdGVpbiBjb250ZW50LCBhbmQgdmlnb3JvdXMgdmVnZXRhdGl2ZSBncm93dGggbWFrZSByYW1pZSBhIHBvcHVsYXIgZmliZXIgYW5kIGZvcmFnZSBjcm9wLiBIZXJlLCB3ZSByZXBvcnQgdGhlIGRyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NjMzNjY4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZNAXV6gpqyBkCqP9KX3VCygj4qoF" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f173%2f4633668" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f173%2f4633668" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=108668" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&amp;Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/2">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="115733631">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733631">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733633">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733633">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733638">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733638">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733659">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733659">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733675">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733675">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733687">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733687">5. Accession numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733689">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733689">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733691">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733691">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733693">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733693">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733695">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733695">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/2/161/4633661">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/2/183/4665695">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Draft genome analysis provides insights into the fiber yield, crude protein biosynthesis, and vegetative growth of domesticated ramie (<em>Boehmeria nivea</em> L. Gaud)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chan Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chan+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Chan Liu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Liangbin Zeng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Liangbin+Zeng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zeng L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zeng L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Liangbin Zeng</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Siyuan Zhu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Siyuan+Zhu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhu S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Siyuan Zhu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lingqing Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Novogene Bioinformatics Institute, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lingqing+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Lingqing Wu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yanzhou Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yanzhou+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yanzhou Wang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shouwei Tang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shouwei+Tang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tang S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shouwei Tang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hongwu Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Xianning Agriculture Academy of sciences, Hubei, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hongwu+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hongwu Wang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xia Zheng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xia+Zheng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zheng X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zheng X&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Xia Zheng</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jian Zhao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Novogene Bioinformatics Institute, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jian+Zhao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhao J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhao J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jian Zhao</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaorong Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Yichun Institute of Agricultural Sciences, Jiangxi, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaorong+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen X&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Xiaorong Chen</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qiuzhong Dai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qiuzhong+Dai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dai Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dai Q&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Qiuzhong Dai</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Touming Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx047-cor1" data-open="dsx047-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. 86 731 88998578. Fax. 86 731 88998528. Email: <a href="mailto:liutouming@caas.cn" target="_blank">liutouming@caas.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Touming+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Touming Liu</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx047-FM1"><div class="fn"><div class="footnote-content">Chan Liu and Liangbin Zeng contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx047-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 173–181, <a href='https://doi.org/10.1093/dnares/dsx047'>https://doi.org/10.1093/dnares/dsx047</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 November 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 June 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 November 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4633668" href="/dnaresearch/article-pdf/25/2/173/24662196/dsx047.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Chan Liu, Liangbin Zeng, Siyuan Zhu, Lingqing Wu, Yanzhou Wang, Shouwei Tang, Hongwu Wang, Xia Zheng, Jian Zhao, Xiaorong Chen, Qiuzhong Dai, Touming Liu;  Draft genome analysis provides insights into the fiber yield, crude protein biosynthesis, and vegetative growth of domesticated ramie (<em>Boehmeria nivea</em> L. Gaud), <em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 173–181, <a href='https://doi.org/10.1093/dnares/dsx047'>https://doi.org/10.1093/dnares/dsx047</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4633668?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4633668?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4633668?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4633668?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4633668?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4633668?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4633668?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Draft%20genome%20analysis%20provides%20insights%20into%20the%20fiber%20yield%2C%20crude%20protein%20biosynthesis%2C%20and%20vegetative%20growth%20of%20domesticated%20ramie%20(%26lt%3Bem%26gt%3BBoehmeria%20nivea%26lt%3B%2Fem%26gt%3B%20L.%20Gaud)&amp;publicationDate=2017-11-15&amp;volumeNum=25&amp;issueNum=2&amp;author=Liu%2C%20Chan%3B%20Zeng%2C%20Liangbin&amp;startPage=173&amp;endPage=181&amp;contentId=10.1093%2Fdnares%2Fdsx047&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f173%2f4633668" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=115733631 id="115733631" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Plentiful bast fiber, a high crude protein content, and vigorous vegetative growth make ramie a popular fiber and forage crop. Here, we report the draft genome of ramie, along with a genomic comparison and evolutionary analysis. The draft genome contained a sequence of approximately 335.6 Mb with 42,463 predicted genes. A high-density genetic map with 4,338 single nucleotide polymorphisms (SNPs) was developed and used to anchor the genome sequence, thus, creating an integrated genetic and physical map containing a 58.2-Mb genome sequence and 4,304 molecular markers. A genomic comparison identified 1,075 unique gene families in ramie, containing 4,082 genes. Among these unique genes, five were cellulose synthase genes that were specifically expressed in stem bark, and 3 encoded a <em>WAT</em>1-related protein, suggesting that they are probably related to high bast fiber yield. An evolutionary analysis detected 106 positively selected genes, 22 of which were related to nitrogen metabolism, indicating that they are probably responsible for the crude protein content and vegetative growth of domesticated varieties. This study is the first to characterize the genome and develop a high-density genetic map of ramie and provides a basis for the genetic and molecular study of this crop.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="ramie">ramie</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;draft genome&quot;">draft genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-density genetic map&quot;">high-density genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;positively selected gene&quot;">positively selected gene</a></div>                            <h2 scrollto-destination=115733633 id="115733633" class="section-title" >1. Introduction</h2>
<p>Ramie (<em>Boehmeria nivea</em> L. Gaud), popularly known as China grass, is a perennial herbaceous plant that belongs to the family Urticaceae. Ramie is one of the most important natural fiber crops, and the fiber extracted from its stem bast layer has several advantages over the fibers of other crops, such as long strands, a smooth texture, and high tensile strength. In addition, ramie is widely used as a livestock forage crop in southern China because it contains a high crude protein content in leaves and young stems (more than 20% of dry weight).<a reveal-id="dsx047-B1" data-open="dsx047-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Because of its fiber and nutritional feed properties, ramie has been widely planted in China, India, and other Asian and Pacific Rim countries.<a reveal-id="dsx047-B2" data-open="dsx047-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> In China, ramie has been cultivated for more than 4,700 yrs.</p><p>Both the fiber and forage of ramie are vegetative organs; therefore, high vegetative yields are required. In order to meet the demands of production, cultivated ramie has been domesticated to have vigorous vegetative growth. Generally, domesticated ramie can be harvested three times per year in China, and up to six times per year in well-watered cultivated environments.<a reveal-id="dsx047-B3" data-open="dsx047-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Therefore, plentiful bast fiber, a high crude protein content, and vigorous vegetative growth are three notable features of ramie, and understanding these traits in domesticated ramie would increase our knowledge of the evolution of this crop. Recently, by comparing the transcriptomes of wild and cultivated ramie, two positively selected genes that function in gibberellin metabolism and a <em>Walls are thin</em> 1 (<em>WAT</em>1)-related protein, respectively, were identified, which are probably involved in vegetative growth and bast fiber biosynthesis, respectively.<a reveal-id="dsx047-B4" data-open="dsx047-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> However, because of the absence of ramie genome information, these important traits in domesticated ramie are still poorly understood.</p><p>Ramie is a diploid species with 28 chromosomes (2<em>n</em> = 28), and its nuclear genome has been estimated to be approximately 716 Mb by flow cytometer analysis,<a reveal-id="dsx047-B5" data-open="dsx047-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> which suggests that the nuclear genome is relatively simple and small. Therefore, it is feasible to perform whole-genome sequencing for this economically important crop. The nuclear genome of ramie has remained uncharacterized until now. Next-generation sequencing (NGS) technologies are powerful and cost-effective methods of high-throughput sequence determination, which has made <em>de novo</em> genome sequencing feasible in non-model species. Over 80 plant species have had their complete genomes sequenced using NGS.<a reveal-id="dsx047-B6" data-open="dsx047-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> These genomic resources can greatly accelerate our understanding of the molecular bases of traits and the evolutionary histories of species.</p><p>To the best of our knowledge, ramie draft genome sequences were determined by NGS technology for the first time in the present study. Furthermore, in order to understand the traits in domesticated ramie, comparison and evolutionary analyses of the genome were performed. A high-density genetic map was constructed using a genotyping-by-sequencing (GBS) technique, which was used to anchor the draft genome sequences into the corresponding chromosome, thus developing an integrated genetic and physical map. This integrated map will be an exceptionally valuable tool for many genetic and genomic applications, such as fine-scale quantitative trait locus (QTL) mapping and cloning. </p>                            <h2 scrollto-destination=115733638 id="115733638" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=115733639 id="115733639" class="section-title" >2.1. Plant materials and DNA extraction</h3>
<p>An F<sub>2</sub> agamous line (FAL) population consisting of 134 lines derived from two ramie varieties, Qingdaye (QDY) and Zhongzhu 1 (ZZ1), was developed based on the strategy described in our previous study,<a reveal-id="dsx047-B5" data-open="dsx047-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and was used for the development of the genetic map. All 134 FALs and two parents were planted at the experimental farm of the Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China. Young leaf samples from each line were collected, immediately frozen in liquid nitrogen (N), and preserved at −80°C until DNA extraction. Total genomic DNA was extracted from the fresh leaves using a DNA extraction kit (Tiangen, Beijing, China).</p>                            <h3 scrollto-destination=115733641 id="115733641" class="section-title" >2.2. Genome sequencing, data filtering, and assembly</h3>
<p>The genome of one ZZ1 parent was sequenced using a whole-genome shotgun strategy<strong>.</strong> Total genomic DNA was used to construct sequencing libraries. Short insert paired-end libraries (230 bp and 500 bp) were generated using a NEBNext<sup>®</sup> Ultra™ DNA Library Prep Kit for Illumina (NEB, MA, USA) according to the manufacturer's instructions. Large-insert (2 kb and 5 kb) DNA sequencing libraries were prepared through circularization by Cre-Lox recombination.<a reveal-id="dsx047-B7" data-open="dsx047-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Sequencing for all four libraries was performed using the Illumina sequencing platform (HiSeq<sup>TM</sup> 2500) according to the manufacturer’s instructions (Illumina, San Diego, CA, USA).</p><p>The raw reads obtained were subjected to the following quality control. Firstly, we discarded all reads with adapter contamination. Secondly, we ruled out low-quality reads with more than 10% ambiguous ‘N’ sequences. Thirdly, we removed reads that had over 50% of bases with quality scores of less than 5. The size, heterozygous ratio, and repeat ratio of the genome were estimated based on the <em>k</em>-mer frequency of the sequence reads (<em>k</em> = 17).<a reveal-id="dsx047-B8" data-open="dsx047-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p><p>The clean, filtered reads were used for the <em>de novo</em> assembly of contigs, using a modified version of SOAPdenovo software that was developed specifically for the <em>de novo</em> assembly of the highly heterozygous jujube genome by Huang et al.,<a reveal-id="dsx047-B9" data-open="dsx047-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and the <em>k</em> value in <em>K</em>-mer was set at 41. The assembled genome has been deposited in DDBJ/ENA/GenBank under the accession NHTU00000000. Gaps on the scaffolds of each line were closed with reads using GapCloser 1.12 with the default parameters set (<a class="link link-uri" href="http://soap.genomics.org.cn/soapdenovo.html" target="_blank">http://soap.genomics.org.cn/soapdenovo.html</a>). Burrows-Wheeler Aligner (BWA) software and Core Eukaryotic Genes Mapping Approach (CEGMA) were used to evaluate the quality of the assembled genome.<a reveal-id="dsx047-B10" data-open="dsx047-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx047-B11" data-open="dsx047-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p>                            <h3 scrollto-destination=115733645 id="115733645" class="section-title" >2.3. Gene prediction and annotation</h3>
<p>Genes in the assembled ramie genome were predicted by Augustus<a reveal-id="dsx047-B12" data-open="dsx047-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> with a training set of <em>Arabidopsis thaliana</em> with the default parameters set. We also aligned the protein sequences of <em>A. thaliana</em>, <em>Morus notabilis</em>, <em>Ziziphus jujuba</em>, and <em>Vitis vinifera</em> to the ramie ZZ1 genome using TBLASTN with an E-value cutoff of 1e-3. Genes with complete encoding sequences were screened based on three standards: (1) the presence of start and stop codons; (2) the location of start or stop codons at the region far from the end of the scaffold with more than 10 bp; and (3) a coverage of sequence alignment between the encoded protein and its homolog in other species of more than 80%. All of the predicted genes were functionally annotated by comparing their protein sequences against the SwissProt,<a reveal-id="dsx047-B13" data-open="dsx047-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Kyoto Encyclopedia of Genes and Genomes (KEGG),<a reveal-id="dsx047-B14" data-open="dsx047-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Gene Ontology (GO),<a reveal-id="dsx047-B15" data-open="dsx047-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and TrEMBL<sup>13</sup> databases using BLASTP with an E-value threshold of 1e-4. Protein domains were identified by comparing the proteins to the InterPro database using InterProScan.<a reveal-id="dsx047-B16" data-open="dsx047-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> A KEGG functional classification was performed by assigning the ramie genes to a pathway based on the KEGG database<a reveal-id="dsx047-B14" data-open="dsx047-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> (<a class="link link-uri" href="http://www.genome.jp/kegg" target="_blank">www.genome.jp/kegg</a>), and WEGO software<a reveal-id="dsx047-B17" data-open="dsx047-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> (<a class="link link-uri" href="http://wego.genomics.org.cn/cgi-bin/wego/index.pl" target="_blank">http://wego.genomics.org.cn/cgi-bin/wego/index.pl</a>) was used to perform a GO functional classification of the genes, and to view the distribution of gene functions.</p>                            <h3 scrollto-destination=115733647 id="115733647" class="section-title" >2.4. Genomic comparison and evolutionary analysis</h3>
<p>To better understand the evolutionary placement of ramie in the plant kingdom, we compared the ramie genome with those of other species and constructed their phylogenetic trees. Protein sequences from 10 plant species—<em>A. thaliana</em> (Ath),<a reveal-id="dsx047-B18" data-open="dsx047-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><em>M. notabilis</em> (Mno),<a reveal-id="dsx047-B19" data-open="dsx047-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><em>Z. jujuba</em> (Zju),<a reveal-id="dsx047-B20" data-open="dsx047-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><em>V. vinifera</em> (Vvi),<a reveal-id="dsx047-B21" data-open="dsx047-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><em>Citrullus lanatus</em> (Cla),<a reveal-id="dsx047-B22" data-open="dsx047-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><em>Glycine max</em> (Gma),<a reveal-id="dsx047-B23" data-open="dsx047-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><em> Gossypium raimondii</em> (Gra),<a reveal-id="dsx047-B24" data-open="dsx047-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><em>Oryza sativa</em> (Osa),<a reveal-id="dsx047-B25" data-open="dsx047-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><em>Linum usitatissimum</em> (Lus),<a reveal-id="dsx047-B26" data-open="dsx047-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and <em>B. nivea</em> (Bni) were extracted to build the gene families. For alternatively spliced isoforms, only the longest proteins were used in the analysis. The protein sequences were compared by an all-to-all BLASTP search with an E-value cutoff of 1e-5, and OrthoMCL<a reveal-id="dsx047-B27" data-open="dsx047-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> was then used to cluster genes from these species into families with the parameter ‘-inflation 1.5.’ The proteins in single-copy gene families underwent multiple sequence alignments by MUSCLE with the default parameters set,<a reveal-id="dsx047-B28" data-open="dsx047-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> which generated a super-alignment matrix. The super-alignment matrix of protein sequences was used to construct a phylogenetic tree using RAxML<a reveal-id="dsx047-B29" data-open="dsx047-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> with the maximum likelihood algorithm. The MCMCTree program was used to estimate the time of species divergence (<a class="link link-uri" href="http://abacus.gene.ucl.ac.uk/software/paml.html" target="_blank">http://abacus.gene.ucl.ac.uk/software/paml.html</a>) using the following settings: burn-in = 5,000,000, sample number = 1,000,000, and sample frequency = 50. In addition, two fiber biosynthesis-related gene families, i.e. <em>cellulose synthase</em> (<em>CesA</em>) and <em>WAT</em>1-like genes, were collected and compared from five species (Bni, Mno, Zju, Cla, and Lus). The Hmmersearch program<a reveal-id="dsx047-B30" data-open="dsx047-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> was used to search the members of families of the five species, based on the conserved domain of PF03552.9: Cellulose_synt (<em>CesA</em>) and PF00892.15: EamA (<em>WAT</em>1-like genes), with the default parameters set. CesA and <em>WAT</em>1-like proteins from five investigated species underwent multiple sequence alignments by MUSCLE<a reveal-id="dsx047-B28" data-open="dsx047-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> with the default parameters set, respectively. Based on the result of alignment, the phylogenetic tree was constructed with TreeBeST<a reveal-id="dsx047-B31" data-open="dsx047-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> using the neighbor-joining (NJ) method, and a bootstrap test was conducted with 100 replications.</p>                            <h3 scrollto-destination=115733649 id="115733649" class="section-title" >2.5. Identification of genes subjected to positive selection</h3>
<p>Positive selection is a process that favors the retention of beneficial mutations as well as the rapid diversification of affected genes. Positive selection at the DNA sequence level is often tested by estimating the ratio of non-synonymous to synonymous nucleotide substitutions between homologous protein-coding gene sequences, and a ratio significantly higher than 1 is considered positive selection.<a reveal-id="dsx047-B4" data-open="dsx047-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> A branch-site method<a reveal-id="dsx047-B32" data-open="dsx047-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> has been developed to test positive selection on individual codons along specific lineages. Using this method, branches of the phylogenetic tree are divided <em>a priori</em> into foreground and background lineages, and the branch under test for positive selection is defined as the foreground branch, while all other branches on the tree are the background branches. Likelihood ratio tests (LRTs) based on branch-site model A<a reveal-id="dsx047-B33" data-open="dsx047-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> ascertain the selective patterns of the genes and thus determine what genes underwent positive selection. Therefore, in this study, the branch-site likelihood method was chosen for identifying positively selected genes in ramie. Briefly, orthologous gene families among ramie and four other plant species (Mno, Zju, Cla, and Gma) were obtained through a multi-reciprocal blast search. The proteins in single-copy gene families underwent multiple sequence alignments using MUSCLE, with the default parameters set.<a reveal-id="dsx047-B28" data-open="dsx047-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Positively selected genes in the single-copy gene families were detected by LRTs based on branch-site models<a reveal-id="dsx047-B32" data-open="dsx047-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> of phylogenetic analysis by maximum likelihood (PAML),<a reveal-id="dsx047-B33" data-open="dsx047-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> with ramie as the foreground branch, and Mno, Zju, Cla, and Gma as the background branch. <em>P</em>-values were computed using the χ<sup>2</sup> statistic and corrected for multiple testing by the false discovery rate method. In order to avoid false positives in the test, genes with a low alignment quality were filtered using the following criteria: (1) a gene length of over 300 bp; (2) the number of positively selected sites ≥ 2; and (3) no gap within three amino acids around the positively selected site in the five species. Enrichment of the GO functional categories was analyzed using GOseq, which is based on the Wallenius noncentral hypergeometric distribution.<a reveal-id="dsx047-B34" data-open="dsx047-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Q values were used to determine the <em>P</em>-value threshold in multiple tests and analyses,<a reveal-id="dsx047-B35" data-open="dsx047-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and GO categories with Q &lt; 0.05 were considered significantly enriched.</p>                            <h3 scrollto-destination=115733651 id="115733651" class="section-title" >2.6. Genotyping by high-throughput sequencing in the FAL population</h3>
<p>A GBS strategy was used to develop single nucleotide polymorphism (SNP) markers. A GBS pre-design for restriction enzyme selection was performed by predicting enzyme combinations and evaluating the sizes of digested fragments according to the ramie ZZ1 genome. <em>Mse</em>I and <em>Eco</em>RI were selected to construct the GBS libraries, which were constructed as follows. Firstly, for each line, 0.1–1 μg of genomic DNA was incubated at 37 °C with <em>Mse</em>I (New England Biolabs, MA, USA; NEB), T4 DNA ligase (NEB), ATP (NEB), and <em>Mse</em>I Y adapter N containing barcodes, and then heat-inactivated at 65 °C. <em>Eco</em>RI (NEB) enzymes were then added to the <em>Mse</em>I digestions to further digest the fragments at 37 °C. Then, for each sample, the digested fragments with ligations were purified with Agencourt AMPure XP (Beckman Coulter, IN, USA) and subjected to polymerase chain reaction (PCR) amplification using Phusion<sup>®</sup> Master Mix (NEB) after adding universal primer and index primer. The PCR products of each sample were purified using Agencourt AMPure XP and pooled, then run on a 2% agarose gel. Fragments of 400–450 bp (with indexes and adaptors) were cut from the gel and purified with a gel extraction kit (QIAGEN, Valencia, CA, USA). The purified products were further cleaned with Agencourt AMPure XP prior to sequencing. Finally, paired-end sequencing was performed on the selected tags using an Illumina 2500 platform by Novogene Bioinformatics Institute, Beijing, China.</p>                            <h3 scrollto-destination=115733653 id="115733653" class="section-title" >2.7. Sequence data grouping, SNP identification, and linkage map construction</h3>
<p>The sequence data from each FAL were sorted according to the barcodes in the raw reads. To ensure that reads were reliable in the following analysis, the raw data (raw reads) were filtered according to the criteria that were described by previous studies.<a reveal-id="dsx047-B36" data-open="dsx047-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsx047-B37" data-open="dsx047-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> In brief, the filtering process included three stringent steps: (1) reads with ≥10% unidentified nucleotides (Ns) were removed; (2) reads with &gt;50% of bases having a Phred quality &lt;5 were removed; and (3) reads with &gt;10 nt aligned to the adapter were removed, allowing ≤10% mismatches. Next, the clean reads filtered from each individual were aligned against the assembled ramie genome using BWA software<a reveal-id="dsx047-B10" data-open="dsx047-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> (settings: mem –t − 4 –k 32 –MR), and the alignment files were converted to bam files using SAMtools software<a reveal-id="dsx047-B38" data-open="dsx047-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> (settings: –bS –t). If multiple read pairs had identical external coordinates, only the pair with the highest mapping quality was retained.</p><p>SNP identification was performed for the two parents and 134 FAL progenies using SAMtools software.<a reveal-id="dsx047-B39" data-open="dsx047-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> To ensure SNP quality, only bases with a Phred quality ≥20 were used to identify SNPs by setting the SAMtools parameter as ‘-Q 20.’ Because the two parents were heterozygous, polymorphic markers between them were detected and classified into eight segregation patterns (ef × eg, nn × np, ab × cc, aa × bb, ab × cd, lm × ll, hk × hk, and cc × ab) according to the CP model in JoinMap 4.0 software.<a reveal-id="dsx047-B39" data-open="dsx047-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The numbers of SNPs and types of transitions or transversions were counted. Then, a Perl script was used to filter SNPs that had more than two genotypes, and polymorphic markers with a segregation pattern of aa × bb were reserved for use.</p><p>Markers containing abnormal bases or exhibiting significantly distorted segregation (<em>P</em> &lt; 0.001) or non-integrity (missing data in over 30% of progenies) were filtered by JoinMap 4.0. A regression algorithm, three-time circulation sequence, and Kosambi mapping function<a reveal-id="dsx047-B40" data-open="dsx047-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> were used for marker distance calculation. The linkage map was drawn using Perl SVG.</p>                            <h3 scrollto-destination=115733657 id="115733657" class="section-title" >2.8. Anchoring sequenced scaffolds to the genetic map</h3>
<p>SNP markers localized on the genetic map were used to anchor sequenced scaffolds of ZZ1 by the following method. Firstly, all of the markers were aligned against the assembled <em>B. nivea</em> scaffolds using BLASTN with an E-value threshold of 1e-15. Markers that were mapped to more than one scaffold were discarded. Then, the scaffold order on the chromosome was sorted by the mean value of markers on the same scaffold. In order to determine the orientation of the scaffold on the chromosome, we compared the physical order of the markers with their linkage map order. If the markers in a scaffold that had the same order between the physical and linkage map order were greater, the scaffold was considered in positive orientation. If the markers that had the same order between the physical and linkage map order were less, the orientation of the scaffold was negative and the sequence was reverse-complemented. Scaffolds in which the orientation was unknown remained in positive orientation by default. Finally, a visualized map of linkage groups that correlated with anchored scaffolds was constructed and exported to create an integrated genetic and physical map.</p>                            <h2 scrollto-destination=115733659 id="115733659" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=115733660 id="115733660" class="section-title" >3.1. Draft genome sequencing and assembly</h3>
<p>A total of 212.8 and 91.2 million raw reads with a total size of 38.1 Gb were obtained in the short and large insert libraries, respectively. After filtering, 37.4 Gb clean sequences were used to <em>de novo</em> assemble the ramie genome, which resulted in 344,363 contigs and 293,578 scaffolds, with total sizes of 300,445,202 bp and 335,575,945 bp, respectively (<a reveal-id="dsx047-T1" data-open="dsx047-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The maximum lengths of the contigs and scaffolds were 75,542 bp and 1,596,017 bp, respectively, and the N50 lengths of the contigs and scaffolds were 3,700 bp and 42,283 bp, respectively (<a reveal-id="dsx047-T1" data-open="dsx047-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The GC content of the assembled ZZ1 genome was 36.9% (<a reveal-id="dsx047-T1" data-open="dsx047-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The K-mer analysis revealed that the average depth of K-mer was 44 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), suggesting that the estimated genome size was 448 Mb, and the heterozygous and repeat ratios in the ZZ1 genome were 1.48% and 58.89%, respectively. Therefore, the sequencing depth of the genome was approximately 85 folds, and the assembled scaffold covered 74.9% of the estimated genome. <div content-id="dsx047-T1" class="table-modal"><div class="table-wrap"><div id="dsx047-T1" data-id="dsx047-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Statistics of genome assembly for ramie</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Contig </th><th>Scaffold </th></tr></thead><tbody><tr><td>Number of sequences </td><td>344,363 </td><td>293,578 </td></tr><tr><td>Total length (bp) </td><td>300, 445,202 </td><td>335,575,945 </td></tr><tr><td>Number of sequences with a length of more than 2 kb </td><td>30, 733 </td><td>30,733 </td></tr><tr><td>Maximum length (bp) </td><td>75,542 </td><td>1,596, 017 </td></tr><tr><td>N50 length (bp) </td><td>3,700 </td><td>42,283 </td></tr><tr><td>GC content (%) </td><td> </td><td>36.9 </td></tr></tbody></table></div><div content-id="dsx047-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Contig </th><th>Scaffold </th></tr></thead><tbody><tr><td>Number of sequences </td><td>344,363 </td><td>293,578 </td></tr><tr><td>Total length (bp) </td><td>300, 445,202 </td><td>335,575,945 </td></tr><tr><td>Number of sequences with a length of more than 2 kb </td><td>30, 733 </td><td>30,733 </td></tr><tr><td>Maximum length (bp) </td><td>75,542 </td><td>1,596, 017 </td></tr><tr><td>N50 length (bp) </td><td>3,700 </td><td>42,283 </td></tr><tr><td>GC content (%) </td><td> </td><td>36.9 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx047-T1" data-id="dsx047-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Statistics of genome assembly for ramie</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Contig </th><th>Scaffold </th></tr></thead><tbody><tr><td>Number of sequences </td><td>344,363 </td><td>293,578 </td></tr><tr><td>Total length (bp) </td><td>300, 445,202 </td><td>335,575,945 </td></tr><tr><td>Number of sequences with a length of more than 2 kb </td><td>30, 733 </td><td>30,733 </td></tr><tr><td>Maximum length (bp) </td><td>75,542 </td><td>1,596, 017 </td></tr><tr><td>N50 length (bp) </td><td>3,700 </td><td>42,283 </td></tr><tr><td>GC content (%) </td><td> </td><td>36.9 </td></tr></tbody></table></div><div content-id="dsx047-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Contig </th><th>Scaffold </th></tr></thead><tbody><tr><td>Number of sequences </td><td>344,363 </td><td>293,578 </td></tr><tr><td>Total length (bp) </td><td>300, 445,202 </td><td>335,575,945 </td></tr><tr><td>Number of sequences with a length of more than 2 kb </td><td>30, 733 </td><td>30,733 </td></tr><tr><td>Maximum length (bp) </td><td>75,542 </td><td>1,596, 017 </td></tr><tr><td>N50 length (bp) </td><td>3,700 </td><td>42,283 </td></tr><tr><td>GC content (%) </td><td> </td><td>36.9 </td></tr></tbody></table></div></div></p><p>In order to evaluate the quality of the assembled genome, all of the sequence reads were re-aligned into the assembled genome. We found that 87.1% of the reads could be mapped to the genome, and covered 97.7% of the assembled genome. A total of 993, 187 SNPs were identified in the ZZ1 genome, 32, 062 of which (accounting for 0.012% of the genome sequence) were homologous, suggesting that the base of the genome had a highly accurate ratio. CEGMA indicated that among 248 core eukaryotic genes, 242 (97.6%) were covered by the assembled genome, 219 of which were covered completely and 23 were partially matched.</p>                            <h3 scrollto-destination=115733663 id="115733663" class="section-title" >3.2. Gene prediction and annotation in the assembled genome</h3>
<p>A total of 42,463 genes were predicted in the ramie genome, 12,664 of which were complete and the others were partial (<a reveal-id="dsx047-F1" data-open="dsx047-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Their coding sequences can be downloaded from <a class="link link-uri" href="https://figshare.com/articles/The_coding_sequences_of_42_463_genes_predicted_from_the_ramie_genome/5436268" target="_blank">https://figshare.com/articles/The_coding_sequences_of_42_463_genes_predicted_from_the_ramie_genome/5436268</a>. Among the predicted genes, 26, 965, 27, 442, 29, 825, and 27, 379 encoded proteins that had significant homologs with proteins in Mno, Zju, Vvi, and Ath, respectively (<a reveal-id="dsx047-F1" data-open="dsx047-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>). A functional annotation was performed for the predicted genes, which revealed that of the 42, 463 genes, 28, 088, 29, 333, 29, 204, 40, 766, and 19,562 exhibited significant similarities to known proteins in the InterPro, KEGG, SwissProt, TrEMBL, and GO databases, respectively (<a reveal-id="dsx047-F1" data-open="dsx047-F1" class="link link-reveal link-table xref-fig">Fig. 1c</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In total, 40,922 (96.4%) genes were functionally annotated in at least one of these five databases. A GO functional classification of the predicted genes assigned 7, 836, 3, 653, and 13,257 genes to the GO classes of biological process, cellular component, and molecular function, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). A KEGG classification revealed that these genes were assigned to 161 pathways, and the RNA degradation pathway enriched the most genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). </p>                        <a id="115733665" scrollto-destination="115733665"></a>
<div id="dsx047-F1" data-id="dsx047-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/m_dsx047f1.png?Expires=2147483647&Signature=tSbxpeVQwGFeeyToHufXnasbDBVC1LuA4Iww0Nfo-neEdaDNLYcXzxCcEu8w8YjYaabnDXIB2ehgyAnE6FttM1DBbss2vnCJIBIFLTB7Ye2AoMX6O9jG6n78DhBAA2e-SsLi5n2qhL3lvC1fpujQ~gV-YW9HuvqjxTJ1i9Z2d6LtvPpRD-icmg~zkAEsQ3-0scXysfisXLuBJ~FgJxaTdGMkZRVu~2Sj75q~jjhUQCfvfEwhHI-PxI31~Yder5l6feksESbElx9WSs7dcHombY0i5C3WdTpQfJ3LmQ9ueCB2dJ-Ui2lmwEGTR5oSPHKW1AYz-nRy5O47IV60m6kMgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of genes predicted and annotated. (a) Pie chart showing that there were 12,664 and 29,799 predicted genes with complete and partial encoding sequences, respectively. (b) Histogram showing the number of genes predicted that encode proteins that are homologous to the proteins in Mno, Zju, Vvi and Ath. (c) Number of genes that were functionally annotated in five public databases." path-from-xml="dsx047f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733665/dsx047f1.tif" path-from-xml="dsx047f1.tif" target="_blank">View large</a><a section="115733665" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047f1.png?Expires=2147483647&Signature=wRzFkcpA6bMZsxQ~psgG7WTt2r-bBLpgpmpz3QFudBfLdD4sBo7H5d~6T909FKySYkDAfwvD11piZZTJHJ~jz5FNVY7JobSLq07pfzOkaeQMya09tTw5Jwnao2Snd5Bp1c8gXHYyXwmuOOR~VSX-4-pM433gnJXmakl1sPLedLoFH-VnJT2i7TvUnFguLzRL1Pz1haASUZvz2iawvQa38x2zecCg8I~A-1xF7LdbmZbuJL-Qi4s1zGkZ-jJE-q6RcpiErr7mDDl1kFsCbVvWotfEPF5QYYIWpu2ofz4uItAmLFpZWCcfwyvzH7T-dyhhd5uzPLGn1j4uvKXSgqnUvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733665&ar=4633668&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx047f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of genes predicted and annotated. (a) Pie chart showing that there were 12,664 and 29,799 predicted genes with complete and partial encoding sequences, respectively. (b) Histogram showing the number of genes predicted that encode proteins that are homologous to the proteins in Mno, Zju, Vvi and Ath. (c) Number of genes that were functionally annotated in five public databases.</p></div></div><div content-id="dsx047-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/m_dsx047f1.png?Expires=2147483647&Signature=tSbxpeVQwGFeeyToHufXnasbDBVC1LuA4Iww0Nfo-neEdaDNLYcXzxCcEu8w8YjYaabnDXIB2ehgyAnE6FttM1DBbss2vnCJIBIFLTB7Ye2AoMX6O9jG6n78DhBAA2e-SsLi5n2qhL3lvC1fpujQ~gV-YW9HuvqjxTJ1i9Z2d6LtvPpRD-icmg~zkAEsQ3-0scXysfisXLuBJ~FgJxaTdGMkZRVu~2Sj75q~jjhUQCfvfEwhHI-PxI31~Yder5l6feksESbElx9WSs7dcHombY0i5C3WdTpQfJ3LmQ9ueCB2dJ-Ui2lmwEGTR5oSPHKW1AYz-nRy5O47IV60m6kMgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of genes predicted and annotated. (a) Pie chart showing that there were 12,664 and 29,799 predicted genes with complete and partial encoding sequences, respectively. (b) Histogram showing the number of genes predicted that encode proteins that are homologous to the proteins in Mno, Zju, Vvi and Ath. (c) Number of genes that were functionally annotated in five public databases." path-from-xml="dsx047f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733665/dsx047f1.tif" path-from-xml="dsx047f1.tif" target="_blank">View large</a><a section="115733665" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047f1.png?Expires=2147483647&Signature=wRzFkcpA6bMZsxQ~psgG7WTt2r-bBLpgpmpz3QFudBfLdD4sBo7H5d~6T909FKySYkDAfwvD11piZZTJHJ~jz5FNVY7JobSLq07pfzOkaeQMya09tTw5Jwnao2Snd5Bp1c8gXHYyXwmuOOR~VSX-4-pM433gnJXmakl1sPLedLoFH-VnJT2i7TvUnFguLzRL1Pz1haASUZvz2iawvQa38x2zecCg8I~A-1xF7LdbmZbuJL-Qi4s1zGkZ-jJE-q6RcpiErr7mDDl1kFsCbVvWotfEPF5QYYIWpu2ofz4uItAmLFpZWCcfwyvzH7T-dyhhd5uzPLGn1j4uvKXSgqnUvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733665&ar=4633668&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx047f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of genes predicted and annotated. (a) Pie chart showing that there were 12,664 and 29,799 predicted genes with complete and partial encoding sequences, respectively. (b) Histogram showing the number of genes predicted that encode proteins that are homologous to the proteins in Mno, Zju, Vvi and Ath. (c) Number of genes that were functionally annotated in five public databases.</p></div></div>                            <h3 scrollto-destination=115733666 id="115733666" class="section-title" >3.3. Developing an integrated genetic and physical map of ramie</h3>
<p>In order to develop an integrated genetic and physical map, a high-density SNP marker genetic linkage map with a total length of 1942.9 cM was constructed (<a reveal-id="dsx047-T2" data-open="dsx047-T2" class="link link-reveal link-table xref-fig">Table 2</a>). A total of 4, 338 markers were included in this map (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>), 2,965 of which were bin markers (<a reveal-id="dsx047-T2" data-open="dsx047-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Thereafter, the ramie genome assembly was anchored into the high-density genetic map to construct an integrated genetic and physical map. Finally, in the integrated map, 924 scaffolds of about 58.2 Mb were anchored to the corresponding chromosome, and among 4,338 markers in the genetic map, 4,304 (99.2%) were mapped to the genome (<a reveal-id="dsx047-T2" data-open="dsx047-T2" class="link link-reveal link-table xref-fig">Table 2</a>, <a reveal-id="dsx047-F2" data-open="dsx047-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). <div content-id="dsx047-T2" class="table-modal"><div class="table-wrap"><div id="dsx047-T2" data-id="dsx047-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>The summary of genetic-physical integrative map of ramie</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of SNP markers </th><th>Number of Bin markers </th><th>Length (cM) </th><th>Average gap of Bin marker (cM) </th><th>Number of scaffolds </th><th>Length (bp) </th><th>NMAG </th><th>RMAG (%) </th></tr></thead><tbody><tr><td>Chr01 </td><td>434 </td><td>289 </td><td>117.6 </td><td>0.407 </td><td>98 </td><td>4,395, 037 </td><td>430 </td><td>99.1 </td></tr><tr><td>Chr02 </td><td>387 </td><td>255 </td><td>122.2 </td><td>0.479 </td><td>87 </td><td>4,215,206 </td><td>386 </td><td>99.7 </td></tr><tr><td>Chr03 </td><td>99 </td><td>86 </td><td>98.3 </td><td>1.143 </td><td>5 </td><td>2,000,201 </td><td>99 </td><td>100.0 </td></tr><tr><td>Chr04 </td><td>462 </td><td>302 </td><td>213.6 </td><td>0.707 </td><td>107 </td><td>5,075,558 </td><td>460 </td><td>99.6 </td></tr><tr><td>Chr05 </td><td>275 </td><td>160 </td><td>116.8 </td><td>0.730 </td><td>71 </td><td>3,035,352 </td><td>271 </td><td>98.5 </td></tr><tr><td>Chr06 </td><td>339 </td><td>211 </td><td>140.7 </td><td>0.667 </td><td>62 </td><td>2,889, 068 </td><td>339 </td><td>100.0 </td></tr><tr><td>Chr07 </td><td>254 </td><td>189 </td><td>150.2 </td><td>0.795 </td><td>73 </td><td>3,567,719 </td><td>244 </td><td>96.1 </td></tr><tr><td>Chr08 </td><td>265 </td><td>196 </td><td>148.7 </td><td>0.759 </td><td>79 </td><td>3,713,277 </td><td>262 </td><td>98.9 </td></tr><tr><td>Chr09 </td><td>247 </td><td>189 </td><td>106.9 </td><td>0.566 </td><td>72 </td><td>3,649, 974 </td><td>244 </td><td>98.8 </td></tr><tr><td>Chr10 </td><td>211 </td><td>161 </td><td>108.2 </td><td>0.672 </td><td>51 </td><td>8,666,598 </td><td>211 </td><td>100.0 </td></tr><tr><td>Chr11 </td><td>177 </td><td>150 </td><td>105.1 </td><td>0.701 </td><td>35 </td><td>3,375,825 </td><td>177 </td><td>100.0 </td></tr><tr><td>Chr12 </td><td>322 </td><td>244 </td><td>132.9 </td><td>0.545 </td><td>56 </td><td>7,107, 184 </td><td>322 </td><td>100.0 </td></tr><tr><td>Chr13 </td><td>298 </td><td>201 </td><td>144.4 </td><td>0.718 </td><td>98 </td><td>2, 971,211 </td><td>291 </td><td>97.7 </td></tr><tr><td>Chr14 </td><td>568 </td><td>332 </td><td>237.3 </td><td>0.715 </td><td>30 </td><td>3,528,490 </td><td>568 </td><td>100.0 </td></tr><tr><td>All </td><td>4338 </td><td>2965 </td><td>1942.9 </td><td>0.655 </td><td>924 </td><td>58,190,700 </td><td>4, 304 </td><td>99.2 </td></tr></tbody></table></div><div content-id="dsx047-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of SNP markers </th><th>Number of Bin markers </th><th>Length (cM) </th><th>Average gap of Bin marker (cM) </th><th>Number of scaffolds </th><th>Length (bp) </th><th>NMAG </th><th>RMAG (%) </th></tr></thead><tbody><tr><td>Chr01 </td><td>434 </td><td>289 </td><td>117.6 </td><td>0.407 </td><td>98 </td><td>4,395, 037 </td><td>430 </td><td>99.1 </td></tr><tr><td>Chr02 </td><td>387 </td><td>255 </td><td>122.2 </td><td>0.479 </td><td>87 </td><td>4,215,206 </td><td>386 </td><td>99.7 </td></tr><tr><td>Chr03 </td><td>99 </td><td>86 </td><td>98.3 </td><td>1.143 </td><td>5 </td><td>2,000,201 </td><td>99 </td><td>100.0 </td></tr><tr><td>Chr04 </td><td>462 </td><td>302 </td><td>213.6 </td><td>0.707 </td><td>107 </td><td>5,075,558 </td><td>460 </td><td>99.6 </td></tr><tr><td>Chr05 </td><td>275 </td><td>160 </td><td>116.8 </td><td>0.730 </td><td>71 </td><td>3,035,352 </td><td>271 </td><td>98.5 </td></tr><tr><td>Chr06 </td><td>339 </td><td>211 </td><td>140.7 </td><td>0.667 </td><td>62 </td><td>2,889, 068 </td><td>339 </td><td>100.0 </td></tr><tr><td>Chr07 </td><td>254 </td><td>189 </td><td>150.2 </td><td>0.795 </td><td>73 </td><td>3,567,719 </td><td>244 </td><td>96.1 </td></tr><tr><td>Chr08 </td><td>265 </td><td>196 </td><td>148.7 </td><td>0.759 </td><td>79 </td><td>3,713,277 </td><td>262 </td><td>98.9 </td></tr><tr><td>Chr09 </td><td>247 </td><td>189 </td><td>106.9 </td><td>0.566 </td><td>72 </td><td>3,649, 974 </td><td>244 </td><td>98.8 </td></tr><tr><td>Chr10 </td><td>211 </td><td>161 </td><td>108.2 </td><td>0.672 </td><td>51 </td><td>8,666,598 </td><td>211 </td><td>100.0 </td></tr><tr><td>Chr11 </td><td>177 </td><td>150 </td><td>105.1 </td><td>0.701 </td><td>35 </td><td>3,375,825 </td><td>177 </td><td>100.0 </td></tr><tr><td>Chr12 </td><td>322 </td><td>244 </td><td>132.9 </td><td>0.545 </td><td>56 </td><td>7,107, 184 </td><td>322 </td><td>100.0 </td></tr><tr><td>Chr13 </td><td>298 </td><td>201 </td><td>144.4 </td><td>0.718 </td><td>98 </td><td>2, 971,211 </td><td>291 </td><td>97.7 </td></tr><tr><td>Chr14 </td><td>568 </td><td>332 </td><td>237.3 </td><td>0.715 </td><td>30 </td><td>3,528,490 </td><td>568 </td><td>100.0 </td></tr><tr><td>All </td><td>4338 </td><td>2965 </td><td>1942.9 </td><td>0.655 </td><td>924 </td><td>58,190,700 </td><td>4, 304 </td><td>99.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>NMAG: number of the Bin markers anchored to genome; RMAG: ratio of the Bin markers anchored to genome.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx047-T2" data-id="dsx047-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>The summary of genetic-physical integrative map of ramie</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of SNP markers </th><th>Number of Bin markers </th><th>Length (cM) </th><th>Average gap of Bin marker (cM) </th><th>Number of scaffolds </th><th>Length (bp) </th><th>NMAG </th><th>RMAG (%) </th></tr></thead><tbody><tr><td>Chr01 </td><td>434 </td><td>289 </td><td>117.6 </td><td>0.407 </td><td>98 </td><td>4,395, 037 </td><td>430 </td><td>99.1 </td></tr><tr><td>Chr02 </td><td>387 </td><td>255 </td><td>122.2 </td><td>0.479 </td><td>87 </td><td>4,215,206 </td><td>386 </td><td>99.7 </td></tr><tr><td>Chr03 </td><td>99 </td><td>86 </td><td>98.3 </td><td>1.143 </td><td>5 </td><td>2,000,201 </td><td>99 </td><td>100.0 </td></tr><tr><td>Chr04 </td><td>462 </td><td>302 </td><td>213.6 </td><td>0.707 </td><td>107 </td><td>5,075,558 </td><td>460 </td><td>99.6 </td></tr><tr><td>Chr05 </td><td>275 </td><td>160 </td><td>116.8 </td><td>0.730 </td><td>71 </td><td>3,035,352 </td><td>271 </td><td>98.5 </td></tr><tr><td>Chr06 </td><td>339 </td><td>211 </td><td>140.7 </td><td>0.667 </td><td>62 </td><td>2,889, 068 </td><td>339 </td><td>100.0 </td></tr><tr><td>Chr07 </td><td>254 </td><td>189 </td><td>150.2 </td><td>0.795 </td><td>73 </td><td>3,567,719 </td><td>244 </td><td>96.1 </td></tr><tr><td>Chr08 </td><td>265 </td><td>196 </td><td>148.7 </td><td>0.759 </td><td>79 </td><td>3,713,277 </td><td>262 </td><td>98.9 </td></tr><tr><td>Chr09 </td><td>247 </td><td>189 </td><td>106.9 </td><td>0.566 </td><td>72 </td><td>3,649, 974 </td><td>244 </td><td>98.8 </td></tr><tr><td>Chr10 </td><td>211 </td><td>161 </td><td>108.2 </td><td>0.672 </td><td>51 </td><td>8,666,598 </td><td>211 </td><td>100.0 </td></tr><tr><td>Chr11 </td><td>177 </td><td>150 </td><td>105.1 </td><td>0.701 </td><td>35 </td><td>3,375,825 </td><td>177 </td><td>100.0 </td></tr><tr><td>Chr12 </td><td>322 </td><td>244 </td><td>132.9 </td><td>0.545 </td><td>56 </td><td>7,107, 184 </td><td>322 </td><td>100.0 </td></tr><tr><td>Chr13 </td><td>298 </td><td>201 </td><td>144.4 </td><td>0.718 </td><td>98 </td><td>2, 971,211 </td><td>291 </td><td>97.7 </td></tr><tr><td>Chr14 </td><td>568 </td><td>332 </td><td>237.3 </td><td>0.715 </td><td>30 </td><td>3,528,490 </td><td>568 </td><td>100.0 </td></tr><tr><td>All </td><td>4338 </td><td>2965 </td><td>1942.9 </td><td>0.655 </td><td>924 </td><td>58,190,700 </td><td>4, 304 </td><td>99.2 </td></tr></tbody></table></div><div content-id="dsx047-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of SNP markers </th><th>Number of Bin markers </th><th>Length (cM) </th><th>Average gap of Bin marker (cM) </th><th>Number of scaffolds </th><th>Length (bp) </th><th>NMAG </th><th>RMAG (%) </th></tr></thead><tbody><tr><td>Chr01 </td><td>434 </td><td>289 </td><td>117.6 </td><td>0.407 </td><td>98 </td><td>4,395, 037 </td><td>430 </td><td>99.1 </td></tr><tr><td>Chr02 </td><td>387 </td><td>255 </td><td>122.2 </td><td>0.479 </td><td>87 </td><td>4,215,206 </td><td>386 </td><td>99.7 </td></tr><tr><td>Chr03 </td><td>99 </td><td>86 </td><td>98.3 </td><td>1.143 </td><td>5 </td><td>2,000,201 </td><td>99 </td><td>100.0 </td></tr><tr><td>Chr04 </td><td>462 </td><td>302 </td><td>213.6 </td><td>0.707 </td><td>107 </td><td>5,075,558 </td><td>460 </td><td>99.6 </td></tr><tr><td>Chr05 </td><td>275 </td><td>160 </td><td>116.8 </td><td>0.730 </td><td>71 </td><td>3,035,352 </td><td>271 </td><td>98.5 </td></tr><tr><td>Chr06 </td><td>339 </td><td>211 </td><td>140.7 </td><td>0.667 </td><td>62 </td><td>2,889, 068 </td><td>339 </td><td>100.0 </td></tr><tr><td>Chr07 </td><td>254 </td><td>189 </td><td>150.2 </td><td>0.795 </td><td>73 </td><td>3,567,719 </td><td>244 </td><td>96.1 </td></tr><tr><td>Chr08 </td><td>265 </td><td>196 </td><td>148.7 </td><td>0.759 </td><td>79 </td><td>3,713,277 </td><td>262 </td><td>98.9 </td></tr><tr><td>Chr09 </td><td>247 </td><td>189 </td><td>106.9 </td><td>0.566 </td><td>72 </td><td>3,649, 974 </td><td>244 </td><td>98.8 </td></tr><tr><td>Chr10 </td><td>211 </td><td>161 </td><td>108.2 </td><td>0.672 </td><td>51 </td><td>8,666,598 </td><td>211 </td><td>100.0 </td></tr><tr><td>Chr11 </td><td>177 </td><td>150 </td><td>105.1 </td><td>0.701 </td><td>35 </td><td>3,375,825 </td><td>177 </td><td>100.0 </td></tr><tr><td>Chr12 </td><td>322 </td><td>244 </td><td>132.9 </td><td>0.545 </td><td>56 </td><td>7,107, 184 </td><td>322 </td><td>100.0 </td></tr><tr><td>Chr13 </td><td>298 </td><td>201 </td><td>144.4 </td><td>0.718 </td><td>98 </td><td>2, 971,211 </td><td>291 </td><td>97.7 </td></tr><tr><td>Chr14 </td><td>568 </td><td>332 </td><td>237.3 </td><td>0.715 </td><td>30 </td><td>3,528,490 </td><td>568 </td><td>100.0 </td></tr><tr><td>All </td><td>4338 </td><td>2965 </td><td>1942.9 </td><td>0.655 </td><td>924 </td><td>58,190,700 </td><td>4, 304 </td><td>99.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>NMAG: number of the Bin markers anchored to genome; RMAG: ratio of the Bin markers anchored to genome.</p></div></div></div></div></p>                        <a id="115733668" scrollto-destination="115733668"></a>
<div id="dsx047-F2" data-id="dsx047-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/m_dsx047f2.png?Expires=2147483647&Signature=tavIc42THMur0UyCvqB8MVTcsyILSB2hUeHYtDAbe9I0hjYUk3CnU-evuU-0vieCl8z0-oFy400Z2hmq5ljuoyNkPA2qNzO0QcXYx-RE7MDtX5W6f8kBjdJXjrVB2Fyz2crdt8O7fLc9dw1QOBiTY4PuLWBv0y2dgmnRDm7iNF5SomOs23dG1W8kfmkIOzsFcIRIryC7KG9G1JktDUtZg8zfPa535iiFLLr8n4e8ll7T2D1UaEhZ-sbXXucZiSazji-1i3lN0ldS3YjqofbG8kjElNuUNjdfB5AH72zMiyqQuCPL7bvN3mrKy7p5fWvPqSTP4BLhmxizjqzZIZwHMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic and physical integrated map. A total of 924 scaffolds of about 58.2 Mb, with 4,304 single nucleotide polymorphism markers, are on the map." path-from-xml="dsx047f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733668/dsx047f2.tif" path-from-xml="dsx047f2.tif" target="_blank">View large</a><a section="115733668" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047f2.png?Expires=2147483647&Signature=Ne7Oi~4AxvRwSypQPcqraZkegDcDObM5ujfwa4QH57yzJ9MYeVbpGInIryladK9hzM6JpvM7x2D5WpstEz5LibRD96-G2yATk7BfolvwGaukm4EqWWpJjT5~xVafIqSYS-Va1QJ4d7jxxmMTtTcaanrkX7a-bDfnagFbkg5ShUWoQOC-wENnQalhNrUgrBvlLYB4hzDHHI8RroEdfnPnjzDlGbuqEns9qsBoONAw6aM-tFIUOpX7fk7OdRvaQNG~gsVj4ZZbRJnLFAosVfnGg8LHiWIrVLHBqk9XO2mlWbmwPGbyaeWT3wzT6pXQ9Cqkzr0i~yTeYnydZSxeCyG05A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733668&ar=4633668&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx047f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic and physical integrated map. A total of 924 scaffolds of about 58.2 Mb, with 4,304 single nucleotide polymorphism markers, are on the map.</p></div></div><div content-id="dsx047-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/m_dsx047f2.png?Expires=2147483647&Signature=tavIc42THMur0UyCvqB8MVTcsyILSB2hUeHYtDAbe9I0hjYUk3CnU-evuU-0vieCl8z0-oFy400Z2hmq5ljuoyNkPA2qNzO0QcXYx-RE7MDtX5W6f8kBjdJXjrVB2Fyz2crdt8O7fLc9dw1QOBiTY4PuLWBv0y2dgmnRDm7iNF5SomOs23dG1W8kfmkIOzsFcIRIryC7KG9G1JktDUtZg8zfPa535iiFLLr8n4e8ll7T2D1UaEhZ-sbXXucZiSazji-1i3lN0ldS3YjqofbG8kjElNuUNjdfB5AH72zMiyqQuCPL7bvN3mrKy7p5fWvPqSTP4BLhmxizjqzZIZwHMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic and physical integrated map. A total of 924 scaffolds of about 58.2 Mb, with 4,304 single nucleotide polymorphism markers, are on the map." path-from-xml="dsx047f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733668/dsx047f2.tif" path-from-xml="dsx047f2.tif" target="_blank">View large</a><a section="115733668" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047f2.png?Expires=2147483647&Signature=Ne7Oi~4AxvRwSypQPcqraZkegDcDObM5ujfwa4QH57yzJ9MYeVbpGInIryladK9hzM6JpvM7x2D5WpstEz5LibRD96-G2yATk7BfolvwGaukm4EqWWpJjT5~xVafIqSYS-Va1QJ4d7jxxmMTtTcaanrkX7a-bDfnagFbkg5ShUWoQOC-wENnQalhNrUgrBvlLYB4hzDHHI8RroEdfnPnjzDlGbuqEns9qsBoONAw6aM-tFIUOpX7fk7OdRvaQNG~gsVj4ZZbRJnLFAosVfnGg8LHiWIrVLHBqk9XO2mlWbmwPGbyaeWT3wzT6pXQ9Cqkzr0i~yTeYnydZSxeCyG05A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733668&ar=4633668&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx047f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic and physical integrated map. A total of 924 scaffolds of about 58.2 Mb, with 4,304 single nucleotide polymorphism markers, are on the map.</p></div></div>                            <h3 scrollto-destination=115733669 id="115733669" class="section-title" >3.4. Genomic comparison and evolutionary analysis</h3>
<p>A total of 26,366 orthologous families of genes were identified among the 10 species investigated. Among these orthologous families, 6,977 appeared in all 10 species. In addition, an orthologous analysis of ramie and four other species (Mno, Zju, Cla, and Lus) identified 8,717 common families in the five species, and 1,075 families contained 4,082 genes that appeared only in ramie (<a reveal-id="dsx047-F3" data-open="dsx047-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). These unique genes were markedly enriched in 32 GO terms, including the term ‘cellulose synthase (UDP-forming) activity’ (Q = 0.045) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Among these 4,082 genes, 5 and 10, respectively, were annotated as putative CesA and <em>WAT</em>1-related proteins. Analyzing the domain of proteins encoded by these 15 genes revealed that all five putative CesA proteins and 3 of 10 putative <em>WAT</em>1-related proteins had a corresponding conserved domain, suggesting that these five and three genes might be authentic <em>CesA</em> and <em>WAT</em>1-related genes, respectively, and that they are probably related to fiber yield in ramie. Subsequently, a phylogenetic analysis of the CesA and <em>WAT</em>1-related proteins of the five species (Bni, Mno, Zju, Cla, and Lus) revealed that the five CesA and three <em>WAT</em>1-related proteins were assigned into a branch into which none of the proteins of the other species fell (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S5 and S6</a></span>), confirming that the five CesA and three <em>WAT</em>1-related genes are unique to ramie. Among the 6,977 orthologs that appeared in the 10 species, 160 were single-copy in each species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>) and were more conservative, which can result in a more exact estimation of the evolutionary relationships between species. Therefore, these 160 single-copy orthologs were used for the phylogenetic analysis of 10 species, which resulted in the construction of a phylogenetic tree (<a reveal-id="dsx047-F4" data-open="dsx047-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). According to the phylogenetic tree, ramie and Mno were placed in a monophyletic clade that diverged about 51.4 million years ago, suggesting that ramie has the closest relationship with Mno among the species investigated (<a reveal-id="dsx047-F4" data-open="dsx047-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). </p>                        <a id="115733671" scrollto-destination="115733671"></a>
<div id="dsx047-F3" data-id="dsx047-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/m_dsx047f3.png?Expires=2147483647&Signature=FjsrdcXjV~U9h2gfD0dF02PW4anpypsIithhe5F~zaT0E42s6th1bH6DXKWvCfXe99UNmShCx52FMB7izI6SC12XwRHuoTfuMqxiilX7dIhJxatZb2omw1LtfQqOtS4OQpbeM6sDaIcmdOpdE3WL0LXI9L6BIdspkRPz5-zLOe8ODyw7q~kegutJ6ZnxrNicSKA6yp9C9B9JDW4LkcdYoSctMyypSwY0LjtHQ-MgZBfp0EV25Qlqm0AQQKbTtiGCiJT8IgdMshsredP1tGu~kyE6lLPMyWJ8~iqolnzgAMNCqL9eED2M3YWs9XYK0VJ~MSTvPdw04CYQDKr~ke5kiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram of orthologous gene families among ramie and four other species. There are 1,075 gene families that are unique to ramie." path-from-xml="dsx047f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733671/dsx047f3.tif" path-from-xml="dsx047f3.tif" target="_blank">View large</a><a section="115733671" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047f3.png?Expires=2147483647&Signature=ikoDBt43hrHlfrG8jGK14kIpY-CBuUQVVOscavGAsEXQudrQ6BppgHSK42yyW8uDh2A4obQzAsKsPD1oQgxZ8-l0yRbCI3sDA0ktPqANbOjhkiZ3WzH9mE6rnxDPQKKPdeneWf~BTvFsMLDXSEhjCrrd64jAP9tSXtXsJ~~k9LTXERobYf~vmR7Q3q5lqlaYmL705YilccjQtGPUcGfMzcoN5tSeH9LNBsPyx30gR0y5-rXfsFYyfmwlsNeIlD5eNk3p77xZNeObe1ea4DykAhkVKL9sV4Zo48Fg6Dr8IPt81jrJgKzHJJ7V8UYCTpu~MW1NNV0cYMQwgGjJljkmaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733671&ar=4633668&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx047f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram of orthologous gene families among ramie and four other species. There are 1,075 gene families that are unique to ramie.</p></div></div><div content-id="dsx047-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/m_dsx047f3.png?Expires=2147483647&Signature=FjsrdcXjV~U9h2gfD0dF02PW4anpypsIithhe5F~zaT0E42s6th1bH6DXKWvCfXe99UNmShCx52FMB7izI6SC12XwRHuoTfuMqxiilX7dIhJxatZb2omw1LtfQqOtS4OQpbeM6sDaIcmdOpdE3WL0LXI9L6BIdspkRPz5-zLOe8ODyw7q~kegutJ6ZnxrNicSKA6yp9C9B9JDW4LkcdYoSctMyypSwY0LjtHQ-MgZBfp0EV25Qlqm0AQQKbTtiGCiJT8IgdMshsredP1tGu~kyE6lLPMyWJ8~iqolnzgAMNCqL9eED2M3YWs9XYK0VJ~MSTvPdw04CYQDKr~ke5kiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram of orthologous gene families among ramie and four other species. There are 1,075 gene families that are unique to ramie." path-from-xml="dsx047f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733671/dsx047f3.tif" path-from-xml="dsx047f3.tif" target="_blank">View large</a><a section="115733671" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047f3.png?Expires=2147483647&Signature=ikoDBt43hrHlfrG8jGK14kIpY-CBuUQVVOscavGAsEXQudrQ6BppgHSK42yyW8uDh2A4obQzAsKsPD1oQgxZ8-l0yRbCI3sDA0ktPqANbOjhkiZ3WzH9mE6rnxDPQKKPdeneWf~BTvFsMLDXSEhjCrrd64jAP9tSXtXsJ~~k9LTXERobYf~vmR7Q3q5lqlaYmL705YilccjQtGPUcGfMzcoN5tSeH9LNBsPyx30gR0y5-rXfsFYyfmwlsNeIlD5eNk3p77xZNeObe1ea4DykAhkVKL9sV4Zo48Fg6Dr8IPt81jrJgKzHJJ7V8UYCTpu~MW1NNV0cYMQwgGjJljkmaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733671&ar=4633668&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx047f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram of orthologous gene families among ramie and four other species. There are 1,075 gene families that are unique to ramie.</p></div></div>                        <a id="115733672" scrollto-destination="115733672"></a>
<div id="dsx047-F4" data-id="dsx047-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/m_dsx047f4.png?Expires=2147483647&Signature=fD2QToHe2~dNtYGTukJOTPhoE4RL5GIyE-~bVf-9K~WPFlT~Xkmrk0WKEQpg26~VQDaKTWXIxJtUoY6Rx7upvuekE0cfDJrAfUf~kgCPgeFskuAQ~-G4bEyzq8s-ZA3MWZM55RP6nbkrO4UxWrse-~kIZNezmjpzVzIXr9R2-g3MjkRdqqmO9HP50FZ7TPJxGOcKGPBpQwhnd-aFV35uUfk4Tz4SEpXnyGyMKG1gqSfdsirBXXhBtmXkoATXaZKkd4kEZXjVQIezVIHObN41bjQhIlX4CnHn6HenHvUunEbFwmBVnvjsC0JYVKkzizbATWxIWp0yNtQ3kN43uUG4xQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of ramie and nine other species, constructed based on their genomes. According to the tree, ramie has a relatively close relationship with Morus notabilis, and they diverged about 51.4 million years ago." path-from-xml="dsx047f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733672/dsx047f4.tif" path-from-xml="dsx047f4.tif" target="_blank">View large</a><a section="115733672" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047f4.png?Expires=2147483647&Signature=qRpOpXb~zv6OPlKVhhIAVc-ZOjYvUaefdyzonVwzUhvghdHHykX9BXKdNrvJ8qeBS5znuajkUfh5tLPfuVnA7CQk8NP6tz9P9qkvFerOntA5ELbRwIKtBjHL2IhaaoaSq0wwBEM2kauBCQ8UMWrVXvodDHCKtTtgKUBd2UgMxWnruFF25f4a94esWIEp9BMtm8HZf4oLlmBKVtFFDuqlt0kn7O0DWAMOlH0lN22fYcyPmmK3WtUb7rg5KSb7uOO78H5anPMx0lAnm6Jxum~vAkgV~RRhH~Cf8I39PgE6WpI8qF6L6B4KKB0HtCjo2vi9juQFv~kIw9CS-YXjkaxNCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733672&ar=4633668&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx047f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of ramie and nine other species, constructed based on their genomes. According to the tree, ramie has a relatively close relationship with <em>Morus notabilis</em>, and they diverged about 51.4 million years ago.</p></div></div><div content-id="dsx047-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/m_dsx047f4.png?Expires=2147483647&Signature=fD2QToHe2~dNtYGTukJOTPhoE4RL5GIyE-~bVf-9K~WPFlT~Xkmrk0WKEQpg26~VQDaKTWXIxJtUoY6Rx7upvuekE0cfDJrAfUf~kgCPgeFskuAQ~-G4bEyzq8s-ZA3MWZM55RP6nbkrO4UxWrse-~kIZNezmjpzVzIXr9R2-g3MjkRdqqmO9HP50FZ7TPJxGOcKGPBpQwhnd-aFV35uUfk4Tz4SEpXnyGyMKG1gqSfdsirBXXhBtmXkoATXaZKkd4kEZXjVQIezVIHObN41bjQhIlX4CnHn6HenHvUunEbFwmBVnvjsC0JYVKkzizbATWxIWp0yNtQ3kN43uUG4xQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of ramie and nine other species, constructed based on their genomes. According to the tree, ramie has a relatively close relationship with Morus notabilis, and they diverged about 51.4 million years ago." path-from-xml="dsx047f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733672/dsx047f4.tif" path-from-xml="dsx047f4.tif" target="_blank">View large</a><a section="115733672" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047f4.png?Expires=2147483647&Signature=qRpOpXb~zv6OPlKVhhIAVc-ZOjYvUaefdyzonVwzUhvghdHHykX9BXKdNrvJ8qeBS5znuajkUfh5tLPfuVnA7CQk8NP6tz9P9qkvFerOntA5ELbRwIKtBjHL2IhaaoaSq0wwBEM2kauBCQ8UMWrVXvodDHCKtTtgKUBd2UgMxWnruFF25f4a94esWIEp9BMtm8HZf4oLlmBKVtFFDuqlt0kn7O0DWAMOlH0lN22fYcyPmmK3WtUb7rg5KSb7uOO78H5anPMx0lAnm6Jxum~vAkgV~RRhH~Cf8I39PgE6WpI8qF6L6B4KKB0HtCjo2vi9juQFv~kIw9CS-YXjkaxNCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733672&ar=4633668&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx047f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of ramie and nine other species, constructed based on their genomes. According to the tree, ramie has a relatively close relationship with <em>Morus notabilis</em>, and they diverged about 51.4 million years ago.</p></div></div>                            <h3 scrollto-destination=115733673 id="115733673" class="section-title" >3.5. Positively selected genes</h3>
<p>A total of 106 positively selected genes were identified in ramie (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Of these, six functioned as transcription factors, including two NAC genes. These 106 genes were markedly enriched in 21 GO terms, including one interesting term, “N compound metabolism”, which was enriched by 22 genes (Q = 0.001) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The positive selection of N metabolism-related genes could be responsible for crude protein biosynthesis and vegetative growth in domesticated ramie.</p>                            <h2 scrollto-destination=115733675 id="115733675" class="section-title" >4. Discussion</h2>
                            <h3 scrollto-destination=115733676 id="115733676" class="section-title" >4.1. New genetic and genomic resources</h3>
<p>Ramie is an important natural fiber in the textile industry because of its ability to retain shape, reduce wrinkling, and introduce a silky luster to the appearance of a fabric. There are also some flaws in ramie fiber, including confined elasticity and elongation potential, and the stiff and brittle nature of the cloth. Therefore, a variety with better quality fiber would be a desirable outcome of ramie breeding. However, trait improvement in ramie has been hindered by a shortage of genetic and genomic resources. Although a considerable number of ramie simple sequence repeat (SSR) markers have been developed,<a reveal-id="dsx047-B41" data-open="dsx047-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsx047-B42" data-open="dsx047-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> only one expanded and low-density molecular marker genetic map has been produced, and few QTLs have been detected for agronomic traits.<a reveal-id="dsx047-B5" data-open="dsx047-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx047-B43" data-open="dsx047-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> There are probably still many QTLs missing, as shown by the QTL regions without markers observed in our low-resolution map. Therefore, a high-resolution ramie genetic map is needed. However, because the screening of population genotypes by SSR markers is time-consuming and labor-intensive, it is difficult to construct a high-resolution genetic map using SSR markers, even with sufficient markers. Moreover, few genes have been functionally characterized in ramie, and its genomic information is entirely unknown. Recently, the ramie transcriptome has been <em>de novo</em> assembled and characterized in several studies, and large numbers of expressed tag sequences (ESTs) were identified.<a reveal-id="dsx047-B2" data-open="dsx047-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx047-B4" data-open="dsx047-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx047-B44" data-open="dsx047-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx047-B45" data-open="dsx047-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> However, because these ESTs were assembled using short read sequences, they contained many redundant sequences, and about 80% partial sequences.<a reveal-id="dsx047-B4" data-open="dsx047-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> As a result, these sequences have not been used to uncover the mechanism of trait formation, or to improve traits.</p><p>In the present study, a draft ramie genome was <em>de novo</em> assembled for the first time. Although the draft genome only covered about 74.9% of the estimated genome and had only an approximately 42.3-kb N50 scaffold, 97.6% of the core eukaryotic genes were covered in the draft genome. In other words, this genome assembly covered about 97% of ramie genes, suggesting that it could be used for further research in functional genomics. This genome is the first to be determined in the family Urticaceae, which comprises 45 genera with more than 700 species, and will be important for understanding the evolutionary placement of Urticaceae in the plant kingdom. Our results indicate that ramie has a close relationship with Mno, a species in the family Moraceae, suggesting that the evolutionary placement of Urticaceae as analyzed by genomic comparison is in agreement with the angiosperm phylogeny Group III (APG III) system.<a reveal-id="dsx047-B46" data-open="dsx047-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> In addition, 42,463 genes were predicted, 96.3% of which were functionally annotated. Therefore, the number of annotated genes identified in the genome is far more than in the transcriptome.<a reveal-id="dsx047-B2" data-open="dsx047-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx047-B4" data-open="dsx047-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Large numbers of annotated genes were identified, which will not only be useful for the characterization of the functions of ramie trait-related genes but also provide a basis for uncovering the mechanism of trait formation.</p><p>Furthermore, we developed the first high-resolution genetic map of ramie, which is not only an important tool for QTL identification but can also be used for assembling genome sequences into the chromosome. By anchoring the genome sequence to the chromosome, an integrated genetic and physical map was constructed. This will make genome utilization easier, particularly in fine-scale QTL mapping and cloning. The size of the assembled genome was about 335 Mb, and the high-density SNP map contained 2,965 bin markers. Therefore, the average physical distance between two bin markers was approximately 151 kb. However, because only four libraries with different fragment sizes were used for performing genomic analysis, the N50 length of scaffolds in the assembled genome was only about 42 kb. In other words, in the genome assembled, there were large numbers of sequence fragments that were smaller than the average physical distance between two bin markers on the map, which resulted in only 924 scaffolds of about 58.2 Mb (17.4%) that were assigned to the chromosomal location. Although 82.6% of the genome sequences had uncertain chromosomal locations, these sequences would still be useful in genetic and molecular studies of ramie, such as gene discovery, marker development, etc. Indeed, many species whose genome sequences have not been assigned to a chromosome have been widely used.<a reveal-id="dsx047-B19" data-open="dsx047-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx047-B26" data-open="dsx047-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>                            <h3 scrollto-destination=115733680 id="115733680" class="section-title" >4.2. Unique fiber biosynthesis-related genes are probably responsible for high fiber yield in ramie</h3>
<p>Unlike cotton fiber, which is an epidermal seed fiber, ramie fiber is a vegetative bast organ involved in sugar metabolism and cellulose synthesis; therefore, it has value in biological research into organogenesis and evolution. A major component of ramie fiber is cellulose, which is a chain of glucose residues. Cellulose biosynthesis can be divided into three steps: initiation of the sugar chain, elongation, and termination of the sugar chain.<a reveal-id="dsx047-B47" data-open="dsx047-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The initiation and elongation of cellulose is performed by CesA, which is a rosette-shaped enzyme complex in the plant cell plasma membrane,<a reveal-id="dsx047-B48" data-open="dsx047-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> and catalytic subunits of CesAs are central catalysts involved in the generation of plant cell wall cellulose.<a reveal-id="dsx047-B49" data-open="dsx047-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsx047-B50" data-open="dsx047-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><em>CesA</em> genes have been extensively studied in plants such as <em>Arabidopsis</em>, rice, cotton, and barley.<a reveal-id="dsx047-B51 dsx047-B52 dsx047-B53 dsx047-B54" data-open="dsx047-B51 dsx047-B52 dsx047-B53 dsx047-B54" class="link link-ref link-reveal xref-bibr"><sup>51–54</sup></a> Many <em>CesA</em> genes have also been reported in ramie.<a reveal-id="dsx047-B2" data-open="dsx047-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx047-B55" data-open="dsx047-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsx047-B56" data-open="dsx047-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> However, the mechanism by which ramie was domesticated as a crop with plentiful fiber in its stem bark is unknown. In this study, 16 putative <em>CesA</em> genes were identified in ramie (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Based on an orthologous analysis among ramie and four other species (Mno, Zju, Cla, and Lus), five were found to be unique to ramie, which was validated by phylogenetic analysis. Moreover, the GO term ‘cellulose synthase (UDP-forming) activity’ was markedly enriched by these unique ramie genes. By comparing the sequence of five unique <em>CesA</em> ramie genes with the transcriptome reported in our previous study, we found that all five genes have ESTs (Unigene12687, CL3101.Contig2, Unigene306, Unigene10248, and Unigene26554), suggesting that these predicted genes are authentic. Interestingly, all five genes exhibited relatively higher expression levels in stem bark than in other organs, which indicates that they are probably involved in bast fiber biosynthesis.<a reveal-id="dsx047-B2" data-open="dsx047-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Therefore, these unique <em>CesA</em> genes are probably responsible for high fiber yield in ramie.</p><p><em>WAT</em>1 encodes a tonoplast-localized protein involved in secondary wall formation in fibers in <em>Arabidopsis</em>.<a reveal-id="dsx047-B57" data-open="dsx047-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> A transcriptomic comparison of wild and cultivated ramie showed that the <em>WAT1</em>-related gene underwent positive selection, suggesting that it is probably related to the fiber yield of ramie, and that positive selection for this gene is probably responsible for the improvement of fiber yield in domesticated ramie.<a reveal-id="dsx047-B4" data-open="dsx047-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> In this study, 13 putative ramie <em>WAT</em>1-related genes were identified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). In addition, genome comparison identified 10 putative <em>WAT</em>1-related genes that were only found in ramie among the five species investigated. However, of these 10 genes, only three (Bni_g24740, Bni_g33701, and Bni_g35790) had the conserved domain of the <em>WAT</em>1-related protein. The absence of a conserved domain in the other seven residual genes was probably due to either an annotation error or an incomplete gene sequence. Therefore, three <em>WAT</em>1-related genes were included in the phylogenetic analysis, which confirmed their uniqueness in ramie among the five species investigated, and they are probably responsible for producing plentiful fiber in the ramie stem bast.</p>                            <h3 scrollto-destination=115733683 id="115733683" class="section-title" >4.3. Positive selection of N metabolism-related genes is probably responsible for vegetative growth and crude protein biosynthesis in domesticated ramie</h3>
<p>N is necessary for optimum growth and development in crop plants. Of the nutrient elements required by plants, N most often limits crop yield.<a reveal-id="dsx047-B58" data-open="dsx047-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> In plants, N absorbed from the soil is converted to organic N compounds such as proteins, chlorophyll, and nucleic acids, which play numerous key roles in plant biochemistry and are essential components of enzymes, cell walls, and a vast array of other cellular components.<a reveal-id="dsx047-B59" data-open="dsx047-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> Consequently, N compound metabolism is important for plant growth, and a deficiency in the supply of N would have a profound effect on plant survival and yield.</p><p>Primary N utilization during vegetative growth involves three steps, i.e. uptake, translocation, and assimilation.<a reveal-id="dsx047-B60" data-open="dsx047-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> N assimilation is a complex metabolic process, and six key enzymes, nitrate reductase (NR), nitrite reductase (NiR), glutamine synthetase (GS), glutamate synthase (GOGAT), asparagine synthetase (Asp), and asparagine aminotransferase (AspAT), are involved.<a reveal-id="dsx047-B61" data-open="dsx047-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> There are three main steps involved in the metabolism of N assimilation. Firstly, when the N is absorbed and transferred into the cell, it is first reduced to ammonium by NR and NiR. Then, the ammonium enters the GS and GOGAT cycle where it is converted to glutamine and glutamate. Finally, the glutamate amino group can be transferred to different amino acids by Asp and AspAT,<a reveal-id="dsx047-B61" data-open="dsx047-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> and these amino acids are used to synthesize N compounds.</p><p>Ramie exhibits vigorous vegetative growth, along with a high crude protein content in its tissues. Recently, three N metabolism-related GS genes were identified in ramie,<a reveal-id="dsx047-B62" data-open="dsx047-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> and one of these, <em>BnGS</em>1<em>-</em>2, was overexpressed in transgenic tobacco, which caused significant increases in fresh weight, dry weight, and plant height, along with a higher soluble protein content and higher total N content.<a reveal-id="dsx047-B63" data-open="dsx047-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> Therefore, these previous studies further validated that, in ramie, N metabolism-related genes play important roles in vegetative growth and crude protein biosynthesis. A total of 22 positively selected genes were found to enrich the GO term ‘cellular nitrogen compound metabolic process’, so they may be N metabolism-related genes. Among the 22 genes, one encodes the Asp protein, which is a key enzyme in N-assimilation.<a reveal-id="dsx047-B61" data-open="dsx047-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> In addition, three genes encode transcription factor proteins, and seven are involved in DNA/RNA activity. Previous studies have found that N metabolism is regulated by the <em>BTB</em> gene<a reveal-id="dsx047-B64" data-open="dsx047-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> and is affected by abscisic acid (ABA) and auxin;<a reveal-id="dsx047-B65" data-open="dsx047-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> here, we identified three positively selected genes that encode an ABA and an auxin-related protein, and a BTB/POZ domain protein. The other eight genes enriched in this GO term encode an uncharacterized protein. The rapid diversification of these genes is probably responsible for the high crude protein content and vigorous vegetative growth of domesticated ramie.</p>                            <h2 scrollto-destination=115733687 id="115733687" class="section-title" >5. Accession numbers</h2>
<p>The sequence reads were deposited in the NCBI-short read archive (SRA) database under the accession number SRX2843758. The WGS accession number of genome sequence assembled in GenBank database was NHTU00000000.</p>                            <h2 scrollto-destination=115733689 id="115733689" class="section-title" >Funding</h2>
<p>This work was supported by grants from the National Natural Science Foundation of China (31571725), the Agricultural Science and Technology Innovation Program of China (CAAS-ASTIP-IBFC), and the National Modern Agro-industry Technology Research System of China (nycytx-19-E16).</p>                            <h2 scrollto-destination=115733691 id="115733691" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=115733693 id="115733693" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=115733695 id="115733695" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx047-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xiong</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Study on selection and breeding of new feed ramie variety Zhongsizhu no.1</div>, <div class="source">Plant Fibers Prod</div> ., <div class="volume">27</div>, <div class="fpage">1</div>–<div class="lpage">4</div>.</div></div></div></div><div content-id="dsx047-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title"><em>De novo</em> assembly and characterization of transcriptome using Illumina paired-end sequencing and identification of CesA gene in ramie (<em>Boehmeria nivea</em> L. Gaud)</div>, <div class="source">BMC Genomic</div> ., <div class="volume">14</div>, <div class="fpage">125</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20characterization%20of%20transcriptome%20using%20Illumina%20paired-end%20sequencing%20and%20identification%20of%20CesA%20gene%20in%20ramie%20(Boehmeria%20nivea%20L.%20Gaud)&amp;author=T.%20Liu&amp;author=S.%20Zhu&amp;author=Q.%20Tang&amp;author=P.%20Chen&amp;author=Y.%20Yu&amp;author=S.%20Tang&amp;publication_year=2013&amp;journal=BMC%20Genomic&amp;volume=14&amp;pages=125" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-125"> </span></div> </div></div></div></div></div><div content-id="dsx047-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">An</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Transcriptome profiling and identification of transcription factors in ramie (<em>Boehmerianivea</em> L. Gaud) in response to PEG treatment, using illumina paired-end sequencing technology</div>, <div class="source">Int. J. Mol. Sci</div> ., <div class="volume">16</div>, <div class="fpage">3493</div>–<div class="lpage">511</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20profiling%20and%20identification%20of%20transcription%20factors%20in%20ramie%20(Boehmerianivea%20L.%20Gaud)%20in%20response%20to%20PEG%20treatment%2C%20using%20illumina%20paired-end%20sequencing%20technology&amp;author=X.%20An&amp;author=J.%20Chen&amp;author=J.%20Zhang&amp;publication_year=2015&amp;journal=Int.%20J.%20Mol.%20Sci&amp;volume=16&amp;pages=3493-511" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms16023493" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms16023493" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms16023493"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25658800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">X.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Transcriptome comparison reveals the patterns of selection in domesticated and wild ramie (<em>Boehmerianivea</em> L. Gaud)</div>, <div class="source">Plant Mol. Biol</div> ., <div class="volume">86</div>, <div class="fpage">85</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20comparison%20reveals%20the%20patterns%20of%20selection%20in%20domesticated%20and%20wild%20ramie%20(Boehmerianivea%20L.%20Gaud)&amp;author=T.%20Liu&amp;author=S.%20Tang&amp;author=S.%20Zhu&amp;author=Q.%20Tang&amp;author=X.%20Zheng&amp;publication_year=2014&amp;journal=Plant%20Mol.%20Biol&amp;volume=86&amp;pages=85-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-014-0214-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-014-0214-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-014-0214-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24934879" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Q.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">QTL mapping for fiber yield-related traits by constructing the first genetic linkage map in ramie (<em>Boehmerianivea</em> L. Gaud)</div>, <div class="source">Mol. Breeding</div> , <div class="volume">34</div>, <div class="fpage">883</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20for%20fiber%20yield-related%20traits%20by%20constructing%20the%20first%20genetic%20linkage%20map%20in%20ramie%20(Boehmerianivea%20L.%20Gaud)&amp;author=T.%20Liu&amp;author=S.%20Tang&amp;author=S.%20Zhu&amp;author=Q.%20Tang&amp;publication_year=2014&amp;journal=Mol.%20Breeding&amp;volume=34&amp;pages=883-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-014-0082-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-014-0082-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-014-0082-7"> </span></div> </div></div></div></div></div><div content-id="dsx047-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kitts</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Church</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Thibaud-Nissen</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Assembly: a resource for assembled genomes at NCBI</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D73</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembly%3A%20a%20resource%20for%20assembled%20genomes%20at%20NCBI&amp;author=P.%20Kitts&amp;author=D.%20Church&amp;author=F.%20Thibaud-Nissen&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D73-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26578580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Nieuwerburgh</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ledesma</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Illumina%20mate-paired%20DNA%20sequencing-library%20preparation%20using%20Cre-Lox%20recombination&amp;author=F.%20Van%20Nieuwerburgh&amp;author=R.%20Thompson&amp;author=J.%20Ledesma&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127871" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marcais</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Marcais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The Jujube genome provides insights into genome evolution and the domestication of sweetness/acidity taste in fruit trees</div>, <div class="source">PLoS Genet</div> ., <div class="volume">12</div>, <div class="fpage">e1006433</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Jujube%20genome%20provides%20insights%20into%20genome%20evolution%20and%20the%20domestication%20of%20sweetness%2Facidity%20taste%20in%20fruit%20trees&amp;author=J.%20Huang&amp;author=C.%20Zhang&amp;author=X.%20Zhao&amp;publication_year=2016&amp;journal=PLoS%20Genet&amp;volume=12&amp;pages=e1006433" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1006433" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1006433" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1006433"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28005948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parra</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=I.%20Korf&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1061-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schoffmann</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">7</div>, <div class="fpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20in%20eukaryotes%20with%20a%20generalized%20hidden%20Markov%20model%20that%20uses%20hints%20from%20external%20sources&amp;author=M.%20Stanke&amp;author=O.%20Schoffmann&amp;author=B.%20Morgenstern&amp;author=S.%20Waack&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-62" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-62" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-62"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16469098" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">28</div>, <div class="fpage">45</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20SWISS-PROT%20protein%20sequence%20database%20and%20its%20supplement%20TrEMBL%20in%202000&amp;author=A.%20Bairoch&amp;author=R.%20Apweiler&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;pages=45-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">S.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">KEGG: kyoto encyclopedia of genes and genomes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">28</div>, <div class="fpage">27</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;pages=27-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ashburner</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ball</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Blake</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2000</div>, 
<div class="article-title">Gene ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet</div> ., <div class="volume">25</div>, <div class="fpage">25</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology&amp;author=M.%20Ashburner&amp;author=C.%20Ball&amp;author=J.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mulder</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">InterPro and InterProScan: tools for protein sequence classification and comparison</div>, <div class="source">Methods Mol. Biol</div> ., <div class="volume">396</div>, <div class="fpage">59</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterPro%20and%20InterProScan%3A%20tools%20for%20protein%20sequence%20classification%20and%20comparison&amp;author=N.%20Mulder&amp;author=R.%20Apweiler&amp;publication_year=2007&amp;journal=Methods%20Mol.%20Biol&amp;volume=396&amp;pages=59-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-1-59745-515-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-59745-515-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-1-59745-515-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18025686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">WEGO: a web tool for plotting GO annotations</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W293</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WEGO%3A%20a%20web%20tool%20for%20plotting%20GO%20annotations&amp;author=J.%20Ye&amp;author=L.%20Fang&amp;author=H.%20Zheng&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W293-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kaul</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Koo</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jenkins</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2000</div>, 
<div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="volume">408</div>, <div class="fpage">796</div>–<div class="lpage">815</div>.,<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20genome%20sequence%20of%20the%20flowering%20plant%20Arabidopsis%20thaliana&amp;author=S.%20Kaul&amp;author=H.%20Koo&amp;author=J.%20Jenkins&amp;publication_year=2000&amp;journal=Nature&amp;volume=408&amp;pages=796-815" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">He</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Qi</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Draft genome sequence of the mulberry tree <em>Morus notabilis</em></div>, <div class="source">Nat. Commun</div> ., <div class="volume">4</div>, <div class="fpage">2445</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20the%20mulberry%20tree%20Morus%20notabilis&amp;author=N.%20He&amp;author=C.%20Zhang&amp;author=X.%20Qi&amp;publication_year=2013&amp;journal=Nat.%20Commun&amp;volume=4&amp;pages=2445" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms3445" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms3445" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms3445"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24048436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">in Zhao</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Cai</div> <div class="given-names">Q.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The complex jujube genome provides insights into fruit tree biology</div>, <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">5315</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complex%20jujube%20genome%20provides%20insights%20into%20fruit%20tree%20biology&amp;author=M.%20Liu&amp;author=J.%20in%20Zhao&amp;author=Q.%20Cai&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=5315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms6315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms6315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms6315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25350882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="collab">The French–Italian Public Consortium for Grapevine Genome Characterization</div>, <div class="year">2007</div>, 
<div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>, <div class="source">Nature</div> , <div class="volume">449</div>, <div class="fpage">463</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The draft genome of watermelon (<em>Citrullus lanatus</em>) and resequencing of 20 diverse accessions</div>, <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">51</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20watermelon%20(Citrullus%20lanatus)%20and%20resequencing%20of%2020%20diverse%20accessions&amp;author=S.%20Guo&amp;author=J.%20Zhang&amp;author=H.%20Sun&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=51-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2470" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2470" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2470"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23179023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jeremy</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Steven</div> <div class="given-names">B. C.</div></div>, <div class="name string-name"><div class="surname">Jessica</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , <div class="volume">463</div>, <div class="fpage">178</div>–<div class="lpage">83</div>.</div></div></div></div><div content-id="dsx047-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The draft genome of a diploid cotton <em>Gossypium raimondii</em></div>, <div class="source">Nat. Genet</div> ., <div class="volume">44</div>, <div class="fpage">1098</div>–<div class="lpage">103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20a%20diploid%20cotton%20Gossypium%20raimondii&amp;author=K.%20Wang&amp;author=Z.%20Wang&amp;author=F.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Genet&amp;volume=44&amp;pages=1098-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22922876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">A draft sequence of the rice genome (<em>Oryza sativa</em> L. ssp. <em>indica</em>)</div>, <div class="source">Science</div> , <div class="volume">296</div>, <div class="fpage">79</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20sequence%20of%20the%20rice%20genome%20(Oryza%20sativa%20L.%20ssp.%20indica)&amp;author=J.%20Yu&amp;author=S.%20Hu&amp;author=J.%20Wang&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=79-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1068037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1068037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1068037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11935017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Hobson</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Galindo</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The genome of flax (<em>Linum usitatissimum</em>) assembled <em>de novo</em>from short shotgun sequence reads</div>, <div class="source">Plant J</div> ., <div class="volume">72</div>, <div class="fpage">461</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20flax%20(Linum%20usitatissimum)%20assembled%20de%20novofrom%20short%20shotgun%20sequence%20reads&amp;author=Z.%20Wang&amp;author=N.%20Hobson&amp;author=L.%20Galindo&amp;publication_year=2012&amp;journal=Plant%20J&amp;volume=72&amp;pages=461-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2012.72.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2012.72.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2012.72.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22757964" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Stoeckert</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Roos</div> <div class="given-names">D.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</div>, <div class="source">Genome Res.</div> , <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=C.%20Stoeckert&amp;author=D.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1224503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edgar</div> <div class="given-names">R. C.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">1792</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=R.%20C.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=1792-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hoover</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Rougemont</div> <div class="given-names">J.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">A rapid bootstrap algorithm for the RAxML Web servers</div>, <div class="source">Syst. Biol.</div> , <div class="volume">57</div>, <div class="fpage">758</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20bootstrap%20algorithm%20for%20the%20RAxML%20Web%20servers&amp;author=A.%20Stamatakis&amp;author=P.%20Hoover&amp;author=J.%20Rougemont&amp;publication_year=2008&amp;journal=Syst.%20Biol.&amp;volume=57&amp;pages=758-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150802429642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150802429642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10635150802429642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18853362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">HMMER web server: interactive sequence similarity searching</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">W29</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HMMER%20web%20server%3A%20interactive%20sequence%20similarity%20searching&amp;author=R.%20Finn&amp;author=J.%20Clements&amp;author=S.%20Eddy&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W29-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21593126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">TreeSoft: TreeBeST (<a class="link link-uri" href="http://treesoft.sourceforge.net/treebest.shtm" target="_blank">http://treesoft.sourceforge.net/treebest.shtm</a>).</div></div></div></div><div content-id="dsx047-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">22</div>, <div class="fpage">2472</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20an%20improved%20branch-site%20likelihood%20method%20for%20detecting%20positive%20selection%20at%20the%20molecular%20level&amp;author=J.%20Zhang&amp;author=R.%20Nielsen&amp;author=Z.%20Yang&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol&amp;volume=22&amp;pages=2472-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16107592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Young</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wakefield</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Smyth</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Oshlack</div> <div class="given-names">A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Gene ontology analysis for RNA-seq: accounting for selection bias</div>, <div class="source">Genome Biol</div> ., <div class="volume">11</div>, <div class="fpage">R14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%20analysis%20for%20RNA-seq%3A%20accounting%20for%20selection%20bias&amp;author=M.%20Young&amp;author=M.%20Wakefield&amp;author=G.%20Smyth&amp;author=A.%20Oshlack&amp;publication_year=2010&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-2-r14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-2-r14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-2-r14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20132535" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</div>, <div class="source">J. Roy. Stat. Soc. B</div> , <div class="volume">57</div>, <div class="fpage">289</div>–<div class="lpage">300</div>.</div></div></div></div><div content-id="dsx047-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Genetic dissection of maize plant architecture with an ultra-high density bin map based on recombinant inbred lines</div>, <div class="source">BMC Genomics</div> , <div class="volume">17</div>, <div class="fpage">178</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20maize%20plant%20architecture%20with%20an%20ultra-high%20density%20bin%20map%20based%20on%20recombinant%20inbred%20lines&amp;author=Z.%20Zhou&amp;author=C.%20Zhang&amp;author=Y.%20Zhou&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=178" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-2555-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-2555-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-2555-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26940065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Construction of a high-density genetic map of <em>Ziziphus jujube</em> Mill. using genotyping by sequencing technology</div>, <div class="source">Tree Genetics Genomes</div> , <div class="volume">12</div>, <div class="fpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20genetic%20map%20of%20Ziziphus%20jujube%20Mill.%20using%20genotyping%20by%20sequencing%20technology&amp;author=Z.%20Zhang&amp;author=T.%20Wei&amp;author=Y.%20Zhong&amp;author=X.%20Li&amp;author=J.%20Huang&amp;publication_year=2016&amp;journal=Tree%20Genetics%20Genomes&amp;volume=12&amp;pages=76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-016-1032-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-016-1032-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-016-1032-9"> </span></div> </div></div></div></div></div><div content-id="dsx047-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Ooijen</div> <div class="given-names">J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species</div>, <div class="source">Genet. Res</div> ., <div class="volume">93</div>, <div class="fpage">343</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multipoint%20maximum%20likelihood%20mapping%20in%20a%20full-sib%20family%20of%20an%20outbreeding%20species&amp;author=J.%20Van%20Ooijen&amp;publication_year=2011&amp;journal=Genet.%20Res&amp;volume=93&amp;pages=343-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672311000279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672311000279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672311000279"> </span></div> </div></div></div></div></div><div content-id="dsx047-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kosambi</div> <div class="given-names">D.</div></div></div> <div class="year">1943</div>, 
<div class="article-title">The estimation of map distance from recombination values</div>, <div class="source">Ann. Eugenics</div> , <div class="volume">12</div>, <div class="fpage">172</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distance%20from%20recombination%20values&amp;author=D.%20Kosambi&amp;publication_year=1943&amp;journal=Ann.%20Eugenics&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="dsx047-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Identification of 32 full-length NAC transcription factors in ramie (<em>Boehmeria nivea</em> L. Gaud) and characterization of the expression pattern of these genes</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">289</div>, <div class="fpage">675</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%2032%20full-length%20NAC%20transcription%20factors%20in%20ramie%20(Boehmeria%20nivea%20L.%20Gaud)%20and%20characterization%20of%20the%20expression%20pattern%20of%20these%20genes&amp;author=T.%20Liu&amp;author=S.%20Zhu&amp;author=Q.%20Tang&amp;author=S.%20Tang&amp;publication_year=2014&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=289&amp;pages=675-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-014-0842-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-014-0842-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-014-0842-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24691727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Development and characterization of 1, 827 expressed sequence tag-derived simple sequence repeat markers for ramie (<em>Boehmeria nivea</em> L. Gaud)</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e60346</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%201%2C%20827%20expressed%20sequence%20tag-derived%20simple%20sequence%20repeat%20markers%20for%20ramie%20(Boehmeria%20nivea%20L.%20Gaud)&amp;author=T.%20Liu&amp;author=S.%20Zhu&amp;author=L.%20Fu&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e60346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0060346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0060346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0060346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23565230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Dai</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Identification of quantitative trait loci for flowering time traits in ramie (<em>Boehmeria nivea</em> L. Gaud)</div>, <div class="source">Euphytica</div> , <div class="volume">210</div>, <div class="fpage">367</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20quantitative%20trait%20loci%20for%20flowering%20time%20traits%20in%20ramie%20(Boehmeria%20nivea%20L.%20Gaud)&amp;author=S.%20Zhu&amp;author=X.%20Zheng&amp;author=Q.%20Dai&amp;author=S.%20Tang&amp;author=T.%20Liu&amp;publication_year=2016&amp;journal=Euphytica&amp;volume=210&amp;pages=367-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-016-1692-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-016-1692-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-016-1692-4"> </span></div> </div></div></div></div></div><div content-id="dsx047-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Pei</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Dai</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Transcriptome profiling using pyrosequencing shows genes associated with bast fiber development in ramie (<em>Boehmerianivea</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">919</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20profiling%20using%20pyrosequencing%20shows%20genes%20associated%20with%20bast%20fiber%20development%20in%20ramie%20(Boehmerianivea%20L.)&amp;author=J.%20Chen&amp;author=Z.%20Pei&amp;author=L.%20Dai&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=919" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25339420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">Q.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Transcriptome sequencing and profiling of expressed genes in phloem and xylem of ramie (<em>Boehmerianivea</em> L. Gaud)</div>, <div class="source">PLoS One.</div> , <div class="volume">9</div>, <div class="fpage">e110623</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20and%20profiling%20of%20expressed%20genes%20in%20phloem%20and%20xylem%20of%20ramie%20(Boehmerianivea%20L.%20Gaud)&amp;author=J.%20Chen&amp;author=F.%20Liu&amp;author=Y.%20Tang&amp;author=Y.%20Yuan&amp;author=Q.%20Guo&amp;publication_year=2014&amp;journal=PLoS%20One.&amp;volume=9&amp;pages=e110623" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0110623" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0110623" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0110623"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25354139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chase</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Reveal</div> <div class="given-names">J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">A phylogenetic classification of the land plants toaccompany APG III</div>, <div class="source">Bot. J. Linn. Soc</div> ., <div class="volume">161</div>, <div class="fpage">122</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylogenetic%20classification%20of%20the%20land%20plants%20toaccompany%20APG%20III&amp;author=M.%20Chase&amp;author=J.%20Reveal&amp;publication_year=2009&amp;journal=Bot.%20J.%20Linn.%20Soc&amp;volume=161&amp;pages=122-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/(ISSN)1095-8339" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F(ISSN)1095-8339" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2F(ISSN)1095-8339"> </span></div> </div></div></div></div></div><div content-id="dsx047-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Read</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bacic</div> <div class="given-names">T.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Prime Time for Cellulose</div>, <div class="source">Science</div> , <div class="volume">295</div>, <div class="fpage">59</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prime%20Time%20for%20Cellulose&amp;author=S.%20Read&amp;author=T.%20Bacic&amp;publication_year=2002&amp;journal=Science&amp;volume=295&amp;pages=59-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1068155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1068155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1068155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11778033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doblin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kurek</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Jacob-Wilk</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Delmer</div> <div class="given-names">D.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Cellulose Biosynthesis in Plants: from Genes to Rosettes</div>, <div class="source">Plant Cell Physiol</div> ., <div class="volume">43</div>, <div class="fpage">1407</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cellulose%20Biosynthesis%20in%20Plants%3A%20from%20Genes%20to%20Rosettes&amp;author=M.%20Doblin&amp;author=I.%20Kurek&amp;author=D.%20Jacob-Wilk&amp;author=D.%20Delmer&amp;publication_year=2002&amp;journal=Plant%20Cell%20Physiol&amp;volume=43&amp;pages=1407-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcf164" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcf164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcf164"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12514238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pear</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kawagoe</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Schreckengost</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Delmer</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Stalker</div> <div class="given-names">D.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Higher plants contain homologs of the bacterial celA genes encoding the catalytic subunit of cellulose synthase</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">93</div>, <div class="fpage">12637</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Higher%20plants%20contain%20homologs%20of%20the%20bacterial%20celA%20genes%20encoding%20the%20catalytic%20subunit%20of%20cellulose%20synthase&amp;author=J.%20Pear&amp;author=Y.%20Kawagoe&amp;author=W.%20Schreckengost&amp;author=D.%20Delmer&amp;author=D.%20Stalker&amp;publication_year=1996&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=93&amp;pages=12637-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.22.12637" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.22.12637" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.22.12637"> </span></div> </div></div></div></div></div><div content-id="dsx047-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Thammannagowda</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bulone</div> <div class="given-names">V.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">An update on the nomenclature for the cellulose synthase genes in Populus</div>, <div class="source">Trends Plant Sci</div> ., <div class="volume">14</div>, <div class="fpage">1360</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20update%20on%20the%20nomenclature%20for%20the%20cellulose%20synthase%20genes%20in%20Populus&amp;author=M.%20Kumar&amp;author=S.%20Thammannagowda&amp;author=V.%20Bulone&amp;publication_year=2009&amp;journal=Trends%20Plant%20Sci&amp;volume=14&amp;pages=1360-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2009.02.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2009.02.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2009.02.004"> </span></div> </div></div></div></div></div><div content-id="dsx047-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Burton</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Shirley</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">King</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Harvey</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fincher</div> <div class="given-names">G.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The CesA gene family of barley quantitative analysis of transcripts reveals two groups of co-expressed genes</div>, <div class="source">Plant Physiol</div> ., <div class="volume">134</div>, <div class="fpage">224</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20CesA%20gene%20family%20of%20barley%20quantitative%20analysis%20of%20transcripts%20reveals%20two%20groups%20of%20co-expressed%20genes&amp;author=R.%20Burton&amp;author=N.%20Shirley&amp;author=B.%20King&amp;author=A.%20Harvey&amp;author=G.%20Fincher&amp;publication_year=2004&amp;journal=Plant%20Physiol&amp;volume=134&amp;pages=224-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.032904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.032904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.032904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14701917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Triplett</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Cloning and characterization of homeologous cellulose synthase catalytic subunit 2 genes from allotetraploid cotton (<em>Gossypium hirsutum</em> L.)</div>, <div class="source">Gene</div> , <div class="volume">494</div>, <div class="fpage">181</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20homeologous%20cellulose%20synthase%20catalytic%20subunit%202%20genes%20from%20allotetraploid%20cotton%20(Gossypium%20hirsutum%20L.)&amp;author=H.%20Kim&amp;author=B.%20Triplett&amp;author=H.%20Zhang&amp;publication_year=2012&amp;journal=Gene&amp;volume=494&amp;pages=181-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2011.12.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2011.12.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2011.12.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22200568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Richmond</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Somerville</div> <div class="given-names">C.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The cellulose synthase superfamily</div>, <div class="source">Plant Physiol</div> ., <div class="volume">124</div>, <div class="fpage">495</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cellulose%20synthase%20superfamily&amp;author=T.%20Richmond&amp;author=C.%20Somerville&amp;publication_year=2000&amp;journal=Plant%20Physiol&amp;volume=124&amp;pages=495-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.124.2.495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.124.2.495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.124.2.495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11027699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Murata</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yamazaki</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Onosato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Miyao</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hirochika</div> <div class="given-names">H.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Three distinct rice cellulose synthase catalytic subunit genes required for cellulose synthesis in the secondary wall</div>, <div class="source">Plant Physiol</div> ., <div class="volume">133</div>, <div class="fpage">73</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20distinct%20rice%20cellulose%20synthase%20catalytic%20subunit%20genes%20required%20for%20cellulose%20synthesis%20in%20the%20secondary%20wall&amp;author=K.%20Tanaka&amp;author=K.%20Murata&amp;author=M.%20Yamazaki&amp;author=K.%20Onosato&amp;author=A.%20Miyao&amp;author=H.%20Hirochika&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=133&amp;pages=73-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.022442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.022442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.022442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12970476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Peng</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">cDNA cloning and expression of two cellulose synthase genes from <em>Boehmeria nivea</em></div>, <div class="source">Acta. Agronomica Sinca</div> , <div class="volume">40</div>, <div class="fpage">1925</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=cDNA%20cloning%20and%20expression%20of%20two%20cellulose%20synthase%20genes%20from%20Boehmeria%20nivea&amp;author=Y.%20Liu&amp;author=J.%20Chen&amp;author=Y.%20Peng&amp;publication_year=2014&amp;journal=Acta.%20Agronomica%20Sinca&amp;volume=40&amp;pages=1925-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3724/SP.J.1006.2014.01925" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3724%2FSP.J.1006.2014.01925" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3724%2FSP.J.1006.2014.01925"> </span></div> </div></div></div></div></div><div content-id="dsx047-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tian</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Yi</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div> <div class="year">2008</div>, Cloning and expression of cellulose synthase gene in ramie [<em>Boehmerianivea</em> (Linn.) Gaud.], Acta. Agronomica Sinca, <strong>34</strong>, 76–83.</div></div></div></div><div content-id="dsx047-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ranocha</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Denance</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Vanholme</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title"><em>Walls are thin 1</em> (<em>WAT1</em>), an Arabidopsis homolog of <em>Medicago truncatula NODULIN21</em>, is a tonoplast-localized protein required for secondary wall formation in fibers</div>, <div class="source">Plant J</div> ., <div class="volume">63</div>, <div class="fpage">469</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Walls%20are%20thin%201%20(WAT1)%2C%20an%20Arabidopsis%20homolog%20of%20Medicago%20truncatula%20NODULIN21%2C%20is%20a%20tonoplast-localized%20protein%20required%20for%20secondary%20wall%20formation%20in%20fibers&amp;author=P.%20Ranocha&amp;author=N.%20Denance&amp;author=R.%20Vanholme&amp;publication_year=2010&amp;journal=Plant%20J&amp;volume=63&amp;pages=469-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2010.63.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2010.63.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2010.63.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20497379" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harper</div> <div class="given-names">J.</div></div></div> <div class="year">1994</div>, Nitrogen metabolism. In: <div class="person-group"><div class="name string-name"><div class="surname">Boote</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Bennett</div> <div class="given-names">J</div></div>, <div class="name string-name"><div class="surname">Sinclair</div> <div class="given-names">T</div></div>, <div class="name string-name"><div class="surname">Paulsen</div> <div class="given-names">G</div></div></div> (Eds.), <div class="source">Physiology and Determination of Crop Yield</div> . 
<div class="publisher-name">ASA, CSSA, SSSA</div>, 
<div class="publisher-loc">Madison, Wisconsin</div>, pp. <div class="fpage">285</div>–<div class="lpage">302</div>.</div></div></div></div><div content-id="dsx047-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Crawford</div> <div class="given-names">N.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Nitrate: nutrient and signal for plant growth</div>, <div class="source">Plant Cell</div> , <div class="volume">7</div>, <div class="fpage">859</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nitrate%3A%20nutrient%20and%20signal%20for%20plant%20growth&amp;author=N.%20Crawford&amp;publication_year=1995&amp;journal=Plant%20Cell&amp;volume=7&amp;pages=859-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.7.7.859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.7.7.859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.7.7.859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7640524" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lam</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Coschigano</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Oliveira</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Melo-Oliveira</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Coruzzi</div> <div class="given-names">G.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">The molecular-genetics of nitrogen assimilation into amino acids in higher plants</div>, <div class="source">Annu. Rev. Plant Physiol. Plant Mol. Biol</div> ., <div class="volume">47</div>, <div class="fpage">569</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular-genetics%20of%20nitrogen%20assimilation%20into%20amino%20acids%20in%20higher%20plants&amp;author=H.%20Lam&amp;author=K.%20Coschigano&amp;author=I.%20Oliveira&amp;author=R.%20Melo-Oliveira&amp;author=G.%20Coruzzi&amp;publication_year=1996&amp;journal=Annu.%20Rev.%20Plant%20Physiol.%20Plant%20Mol.%20Biol&amp;volume=47&amp;pages=569-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.47.1.569" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.47.1.569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.47.1.569"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15012301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Good</div> <div class="given-names">A. G.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">De Pauw</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Engineering nitrogen use efficiency with alanine aminotransferase</div>, <div class="source">Can. J. Bot</div> ., <div class="volume">85</div>, <div class="fpage">252</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20nitrogen%20use%20efficiency%20with%20alanine%20aminotransferase&amp;author=A.%20G.%20Good&amp;author=S.%20J.%20Johnson&amp;author=M.%20De%20Pauw&amp;publication_year=2007&amp;journal=Can.%20J.%20Bot&amp;volume=85&amp;pages=252-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/B07-019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FB07-019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FB07-019"> </span></div> </div></div></div></div></div><div content-id="dsx047-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Xiong</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Identification and expression analysis of glutamine synthetase genes in ramie (<em>Boehmeria nivea</em> L. Gaud)</div>, <div class="source">Open Life Sci</div> ., <div class="volume">10</div>, <div class="fpage">299</div>–<div class="lpage">309</div>.</div></div></div></div><div content-id="dsx047-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Characterization of a glutamine synthetase gene <em>BnGS1-2</em> from ramie (<em>Boehmeria nivea</em> L. Gaud) and biochemical assays of <em>BnGS1-2</em>-over-expressing transgenic tobacco</div>, <div class="source">Acta Physiol. Plant</div> ., <div class="volume">37</div>, <div class="fpage">1742</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20glutamine%20synthetase%20gene%20BnGS1-2%20from%20ramie%20(Boehmeria%20nivea%20L.%20Gaud)%20and%20biochemical%20assays%20of%20BnGS1-2-over-expressing%20transgenic%20tobacco&amp;author=J.%20Zheng&amp;author=C.%20Yu&amp;author=P.%20Chen&amp;publication_year=2015&amp;journal=Acta%20Physiol.%20Plant&amp;volume=37&amp;pages=1742" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11738-014-1742-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11738-014-1742-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11738-014-1742-2"> </span></div> </div></div></div></div></div><div content-id="dsx047-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Araus</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Vidal</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Puelma</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Members of BTB gene family of scaffold proteins suppress nitrate uptake and nitrogen use efficiency</div>, <div class="source">Plant Physiol</div> ., <div class="volume">171</div>, <div class="fpage">1523</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Members%20of%20BTB%20gene%20family%20of%20scaffold%20proteins%20suppress%20nitrate%20uptake%20and%20nitrogen%20use%20efficiency&amp;author=V.%20Araus&amp;author=E.%20Vidal&amp;author=T.%20Puelma&amp;publication_year=2016&amp;journal=Plant%20Physiol&amp;volume=171&amp;pages=1523-32" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27208309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx047-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx047-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kiba</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kudo</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kojima</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sakakibara</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Hormonal control of nitrogen acquisition: roles of auxin, abscisic acid, and cytokinin</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">62</div>, <div class="fpage">1399</div>–<div class="lpage">409</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hormonal%20control%20of%20nitrogen%20acquisition%3A%20roles%20of%20auxin%2C%20abscisic%20acid%2C%20and%20cytokinin&amp;author=T.%20Kiba&amp;author=T.%20Kudo&amp;author=M.%20Kojima&amp;author=H.%20Sakakibara&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot&amp;volume=62&amp;pages=1399-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erq410" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferq410" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferq410"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21196475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4633668&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx047/1/dsx047_supp.zip?Expires=2147483647&amp;Signature=x-VwrtOprZAZYq-9ej01jXxv~SsFR7VZx3a~1MDd1SLAgfWp-SB6RXRMyAvpTiiBbGkj-8wjH843-nlzScHLWygSJN5lvAN45QH6L3ItdO8rO-p7am77vWPnPJoPgP2Lf~LLdN8FOznWjtqMQkzivXJzg7OQSqFPVC3FYJk5BReXVB9HEFqwYjiT9hS3FslgZ8XuQpy84ixXe4fG-yWpoe~0WTuMI8exy1mye91a4FrilYwUHGk9SaAJGZrQ4XTmlC9yDJxM2DfMFOypcfhOQFn15YrxYClfdvMmze2MmXpjEKo989OMlF6c5nPH2SVsrsYXYZDqLhLhwrphrOIbcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">907</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000430686200006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx047"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4633668"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000430686200006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f2%2f173%2f4633668">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000430686200006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F2%2F173%2F4633668" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4633668'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Draft genome analysis provides insights into the fiber yield, crude protein biosynthesis, and vegetative growth of domesticated ramie (Boehmeria nivea L. Gaud)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx047");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f173%2f4633668">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

