

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/4/343/4850982","@type":"ScholarlyArticle","name":"Identification of the DNA methyltransferases establishing the methylome of the cyanobacterium  Synechocystis  sp. PCC 6803","datePublished":"2018-02-12","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2018-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy006","keywords":["cyanobacteria","DNA methyltransferase","mutant","photosynthetic pigment","phylogenetics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/4/343/4850982","author":[{"name":"Hagemann, Martin","affiliation":"Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany","@type":"Person"},{"name":"Gärtner, Katrin","affiliation":"Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany","@type":"Person"},{"name":"Scharnagl, Matthias","affiliation":"Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany","@type":"Person"},{"name":"Bolay, Paul","affiliation":"Faculty of Biology, University of Freiburg, Freiburg, Germany","@type":"Person"},{"name":"Lott, Steffen C","affiliation":"Faculty of Biology, University of Freiburg, Freiburg, Germany","@type":"Person"},{"name":"Fuss, Janina","affiliation":"Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany","@type":"Person"},{"name":"Huettel, Bruno","affiliation":"Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany","@type":"Person"},{"name":"Reinhardt, Richard","affiliation":"Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany","@type":"Person"},{"name":"Klähn, Stephan","affiliation":"Faculty of Biology, University of Freiburg, Freiburg, Germany","@type":"Person"},{"name":"Hess, Wolfgang R","affiliation":"Faculty of Biology, University of Freiburg, Freiburg, Germany  Freiburg Institute for Advanced Studies, University of Freiburg, Germany","@type":"Person"}],"description":"Abstract.  DNA methylation in bacteria is important for defense against foreign DNA, but is also involved in DNA repair, replication, chromosome partitioning, a","pageStart":"343","pageEnd":"352","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Identification of the DNA methyltransferases establishing the methylome of the cyanobacterium  Synechocystis  sp. PCC 6803","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identification of the DNA methyltransferases establishing the methylome of the cyanobacterium  Synechocystis  sp. PCC 6803" />
    <meta property="og:description" content="Abstract.  DNA methylation in bacteria is important for defense against foreign DNA, but is also involved in DNA repair, replication, chromosome partitioning, a" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/4/343/4850982" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-02-12" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hagemann, Martin" /><meta name="citation_author_institution" content="Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany" /><meta name="citation_author" content="Gärtner, Katrin" /><meta name="citation_author_institution" content="Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany" /><meta name="citation_author" content="Scharnagl, Matthias" /><meta name="citation_author_institution" content="Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany" /><meta name="citation_author" content="Bolay, Paul" /><meta name="citation_author_institution" content="Faculty of Biology, University of Freiburg, Freiburg, Germany" /><meta name="citation_author" content="Lott, Steffen C" /><meta name="citation_author_institution" content="Faculty of Biology, University of Freiburg, Freiburg, Germany" /><meta name="citation_author" content="Fuss, Janina" /><meta name="citation_author_institution" content="Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany" /><meta name="citation_author" content="Huettel, Bruno" /><meta name="citation_author_institution" content="Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany" /><meta name="citation_author" content="Reinhardt, Richard" /><meta name="citation_author_institution" content="Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany" /><meta name="citation_author" content="Klähn, Stephan" /><meta name="citation_author_institution" content="Faculty of Biology, University of Freiburg, Freiburg, Germany" /><meta name="citation_author" content="Hess, Wolfgang R" /><meta name="citation_author_institution" content="Faculty of Biology, University of Freiburg, Freiburg, Germany" /><meta name="citation_author_institution" content="Freiburg Institute for Advanced Studies, University of Freiburg, Germany" /><meta name="citation_title" content="Identification of the DNA methyltransferases establishing the methylome of the cyanobacterium Synechocystis sp. PCC 6803" /><meta name="citation_firstpage" content="343" /><meta name="citation_lastpage" content="352" /><meta name="citation_doi" content="10.1093/dnares/dsy006" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2018/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Mechanisms of epigenetic memory;  citation_journal_title=Trends Genet;  citation_year=2014;  citation_volume=30;  citation_pages=230-6; " /><meta name="citation_reference" content="citation_title=Methylome evolution in plants;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=264" /><meta name="citation_reference" content="citation_title=Methylation of DNA in prokaryotes;  citation_journal_title=Birkhäuser: Basel;  citation_year=1993; " /><meta name="citation_reference" content="citation_title=The biology of restriction and anti-restriction;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=2005;  citation_volume=8;  citation_pages=466-72; " /><meta name="citation_reference" content="citation_title=Rare bases in animal DNA;  citation_journal_title=Nature;  citation_year=1970;  citation_volume=225;  citation_pages=948-9; " /><meta name="citation_reference" content="citation_title=DNA methylation on N6-adenine in C. elegans;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=161;  citation_pages=868-78; " /><meta name="citation_reference" content="citation_title=N6-methyldeoxyadenosine marks active transcription start sites in Chlamydomonas;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=161;  citation_pages=879-92; " /><meta name="citation_reference" content="citation_title=N6-methyladenine DNA modification in Drosophila;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=161;  citation_pages=893-906; " /><meta name="citation_reference" content="citation_title=Bacterial DNA methylation and methylomes. In: Jeltsch, A., Jurkowska, R. Z. (eds) DNA Methyltransferases – Role and Function;  citation_journal_title=Adv. Exp. Med. Biol;  citation_year=2016;  citation_volume=945;  citation_pages=35-61; " /><meta name="citation_reference" content="citation_title=DNA adenine methyltransferase (Dam) controls the expression of the cytotoxic enterotoxin (act) gene of Aeromonas hydrophila via tRNA modifying enzyme-glucose-inhibited division protein (GidA);  citation_journal_title=Gene;  citation_year=2012;  citation_volume=498;  citation_pages=280-7; " /><meta name="citation_reference" content="citation_title=Methylation patterns in pap regulatory DNA control pyelonephritis-associated pili phase variation in E. coli;  citation_journal_title=Cell;  citation_year=1994;  citation_volume=76;  citation_pages=577-88; " /><meta name="citation_reference" content="citation_title=Differential binding of Lrp to two sets of pap DNA binding sites mediated by Pap I regulates Pap phase variation in Escherichia coli;  citation_journal_title=EMBO J;  citation_year=1995;  citation_volume=14;  citation_pages=5785-97; " /><meta name="citation_reference" content="citation_title=Conservation of Dcm-mediated cytosine DNA methylation in Escherichia coli;  citation_journal_title=FEMS Microbiol. Lett;  citation_year=2012;  citation_volume=328;  citation_pages=78-85; " /><meta name="citation_reference" content="citation_title=DNA binding of the cell cycle transcriptional regulator GcrA depends on N6-adenosine methylation in Caulobacter crescentus and other alphaproteobacteria;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003541" /><meta name="citation_reference" content="citation_title=Comprehensive methylome characterization of Mycoplasma genitalium and Mycoplasma pneumoniae at single-base resolution;  citation_journal_title=PLoS Genet;  citation_year=2012;  citation_volume=9;  citation_pages=e1003191" /><meta name="citation_reference" content="citation_title=The epigenomic landscape of prokaryotes;  citation_journal_title=PLoS Genet;  citation_year=2016;  citation_volume=12;  citation_pages=e1005854" /><meta name="citation_reference" content="citation_title=Systems and synthetic biology for the biotechnological application of cyanobacteria;  citation_journal_title=Curr. Opin. Biotechnol;  citation_year=2018;  citation_volume=49;  citation_pages=94-9; " /><meta name="citation_reference" content="citation_title=DNA methyltransferases of the cyanobacterium Anabaena PCC 7120;  citation_journal_title=Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=1491-506; " /><meta name="citation_reference" content="citation_title=Unique modification of adenine in genomic DNA of the marine cyanobacterium Trichodesmium sp. strain NIBB 1067;  citation_journal_title=J. Bacteriol;  citation_year=1991;  citation_volume=173;  citation_pages=7059-62; " /><meta name="citation_reference" content="citation_title=The cyanobacterium Synechocystis sp. strain PCC 6803 expresses a DNA methyltransferase specific for the recognition sequence of the restriction endonuclease PvuI;  citation_journal_title=J. Bacteriol;  citation_year=1998;  citation_volume=180;  citation_pages=4116-22; " /><meta name="citation_reference" content="citation_title=Premethylation of foreign DNA improves integrative transformation efficiency in Synechocystis sp. strain PCC 6803;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2015;  citation_volume=81;  citation_pages=8500-6; " /><meta name="citation_reference" content="citation_title=Microevolution in cyanobacteria: re-sequencing a motile substrain of Synechocystis sp. PCC 6803;  citation_journal_title=DNA Res;  citation_year=2012;  citation_volume=19;  citation_pages=435-48; " /><meta name="citation_reference" content="citation_title=Generic assignments, strain histories and properties of pure cultures of cyanobacteria;  citation_journal_title=J. Gen. Microbiol;  citation_year=1979;  citation_volume=111;  citation_pages=1-16; " /><meta name="citation_reference" content="citation_title=Preparation of genomic DNA from bacteria;  citation_journal_title=Curr. Prot. Molec. Biol;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications. Krueger F1, Andrews SR;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=1571-2; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbour Laboratory Press, Cold Spring Harbor, NY;  citation_title=Molecular cloning;  citation_year=1989; " /><meta name="citation_reference" content="citation_title=Sequence analysis of the genome of the unicellular cyanobacterium Synechocystis sp. strain PCC 6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions;  citation_journal_title=DNA Res;  citation_year=1996;  citation_volume=3;  citation_pages=109-36; " /><meta name="citation_reference" content="citation_title=Vectors for the complementation analysis of cyanobacterial mutants;  citation_journal_title=Russ. J. Genet;  citation_year=1999;  citation_volume=35;  citation_pages=228-32; " /><meta name="citation_reference" content="citation_title=Protein measurement with the folin phenol reagent;  citation_journal_title=J. Biol. Chem;  citation_year=1951;  citation_volume=193;  citation_pages=265-75; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucl. Acid. Res;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Protein database searches using compositionally adjusted substitution matrices;  citation_journal_title=FEBS J;  citation_year=2005;  citation_volume=272;  citation_pages=5101-9; " /><meta name="citation_reference" content="citation_title=ClustalW and ClustalX version 2;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods;  citation_journal_title=Mol. Biol. Evol;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=The methylomes of six bacteria;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=11450-114362; " /><meta name="citation_reference" content="citation_title=Bisulfite sequencing protocol displays both 5-methylcytosine and N4-methylcytosine;  citation_journal_title=Anal. Biochem;  citation_year=1999;  citation_volume=271;  citation_pages=116-9; " /><meta name="citation_reference" content="citation_title=REBASE—a database for DNA restriction and modification: enzymes, genes and genomes;  citation_journal_title=Nucl. Acid. Res;  citation_year=2010;  citation_volume=38 (Database issue);  citation_pages=D234-6; " /><meta name="citation_reference" content="citation_title=Gene structure and expression of MboI restriction-modification system;  citation_journal_title=Nucleic Acids Res;  citation_year=1993;  citation_volume=21;  citation_pages=2309-13; " /><meta name="citation_reference" content="citation_title=Cloning and sequence analysis of the genes coding for Eco57I type IV restriction-modification enzymes;  citation_journal_title=Nucl. Acid. Res;  citation_year=1992;  citation_volume=20;  citation_pages=6051-6; " /><meta name="citation_reference" content="citation_title=CyanoBase: the cyanobacteria genome database update 2010;  citation_journal_title=Nucl. Acid. Res;  citation_year=2010;  citation_volume=38 (Database issue);  citation_pages=D379-81; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the primary transcriptome of Synechocystis sp. PCC 6803;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=527-39; " /><meta name="citation_reference" content="citation_title=Highly iterated palindromic sequences (HIPs) and their relationship to DNA methyltransferases;  citation_journal_title=Life (Basel);  citation_year=2015;  citation_volume=5;  citation_pages=921-48; " /><meta name="citation_reference" content="citation_title=Singular over-representation of an octameric palindrome, HIP1, in DNA from many cyanobacteria;  citation_journal_title=Nucl. Acid. Res;  citation_year=1995;  citation_volume=23;  citation_pages=729-35; " /><meta name="citation_reference" content="citation_title=HIP1 propagates in cyanobacterial DNA via nucleotide substitutions but promotes excision at similar frequencies in Escherichia coli and Synechococcus PCC 7942;  citation_journal_title=Mol. Microbiol;  citation_year=1997;  citation_volume=24;  citation_pages=181-9; " /><meta name="citation_reference" content="citation_title=Insensitivity of chloroplast gene expression to DNA methylation;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2009;  citation_volume=282;  citation_pages=17-24; " /><meta name="citation_reference" content="citation_title=DNA methyltransferases and epigenetic regulation in bacteria;  citation_journal_title=FEMS Microbiol. Rev;  citation_year=2016;  citation_volume=40;  citation_pages=575-91; " /><meta name="citation_reference" content="citation_title=DNA adenine methylation is required to replicate both Vibrio cholera chromosomes once per cell cycle;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1000939" /><meta name="citation_reference" content="citation_title=Distinct replication requirements for the two Vibrio cholera chromosomes;  citation_journal_title=Cell;  citation_year=2003;  citation_volume=114;  citation_pages=521-30; " /><meta name="citation_reference" content="citation_title=Fuse or die: how to survive the loss of Dam in Vibrio cholerae;  citation_journal_title=Mol. Microbiol;  citation_year=2014;  citation_volume=91;  citation_pages=665-72; " /><meta name="citation_reference" content="citation_title=Transcriptional analysis and mutation of a dnaA-like gene in Synechocystis sp. strain PCC 6803;  citation_journal_title=J. Bacteriol;  citation_year=1998;  citation_volume=180;  citation_pages=4946-9; " /><meta name="citation_reference" content="citation_title=Diversification of DnaA dependency for DNA replication in cyanobacterial evolution;  citation_journal_title=Isme J;  citation_year=2016;  citation_volume=10;  citation_pages=1113-21; " /><meta name="citation_reference" content="citation_title=Structure-guided analyses reveals nine sequence motifs conserved among DNA amino-methyltransferases, and suggests a catalytic mechanism for these enzymes;  citation_journal_title=J. Mol. Biol;  citation_year=1995;  citation_volume=253;  citation_pages=618-32; " /><meta name="citation_reference" content="citation_title=DNAPlotter: circular and linear interactive genome visualization;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=119-20; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/4/343/25501664/dsy006.pdf" /><meta name="description" content="Abstract.  DNA methylation in bacteria is important for defense against foreign DNA, but is also involved in DNA repair, replication, chromosome partitioning, a" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/4/343/4850982" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/4/343/4850982" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/4/343/4850982");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy006';
    </script>
<title>
	Identification of the DNA methyltransferases establishing the methylome of the cyanobacterium Synechocystis sp. PCC 6803 | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4850982" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYaBATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGFnZW1hbm4sIE1hcnRpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9zY2llbmNlcywgUGxhbnQgUGh5c2lvbG9neSwgVW5pdmVyc2l0eSBvZiBSb3N0b2NrLCBSb3N0b2NrLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR8OkcnRuZXIsIEthdHJpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9zY2llbmNlcywgUGxhbnQgUGh5c2lvbG9neSwgVW5pdmVyc2l0eSBvZiBSb3N0b2NrLCBSb3N0b2NrLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2NoYXJuYWdsLCBNYXR0aGlhcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9zY2llbmNlcywgUGxhbnQgUGh5c2lvbG9neSwgVW5pdmVyc2l0eSBvZiBSb3N0b2NrLCBSb3N0b2NrLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQm9sYXksIFBhdWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgRnJlaWJ1cmcsIEZyZWlidXJnLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTG90dCwgU3RlZmZlbiBDIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIEZyZWlidXJnLCBGcmVpYnVyZywgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1c3MsIEphbmluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1heCBQbGFuY2stR2Vub21lLUNlbnRyZSBDb2xvZ25lLCBNYXggUGxhbmNrIEluc3RpdHV0ZSBmb3IgUGxhbnQgQnJlZWRpbmcgUmVzZWFyY2gsIEvDtmxuLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHVldHRlbCwgQnJ1bm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYXggUGxhbmNrLUdlbm9tZS1DZW50cmUgQ29sb2duZSwgTWF4IFBsYW5jayBJbnN0aXR1dGUgZm9yIFBsYW50IEJyZWVkaW5nIFJlc2VhcmNoLCBLw7ZsbiwgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJlaW5oYXJkdCwgUmljaGFyZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1heCBQbGFuY2stR2Vub21lLUNlbnRyZSBDb2xvZ25lLCBNYXggUGxhbmNrIEluc3RpdHV0ZSBmb3IgUGxhbnQgQnJlZWRpbmcgUmVzZWFyY2gsIEvDtmxuLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2zDpGhuLCBTdGVwaGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIEZyZWlidXJnLCBGcmVpYnVyZywgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhlc3MsIFdvbGZnYW5nIFIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgRnJlaWJ1cmcsIEZyZWlidXJnLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnJlaWJ1cmcgSW5zdGl0dXRlIGZvciBBZHZhbmNlZCBTdHVkaWVzLCBVbml2ZXJzaXR5IG9mIEZyZWlidXJnLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJJZGVudGlmaWNhdGlvbiBvZiB0aGUgRE5BIG1ldGh5bHRyYW5zZmVyYXNlcyBlc3RhYmxpc2hpbmcgdGhlIG1ldGh5bG9tZSBvZiB0aGUgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpcyBzcC4gUENDIDY4MDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM1MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeTAwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyBvZiBlcGlnZW5ldGljIG1lbW9yeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjMwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGh5bG9tZSBldm9sdXRpb24gaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTI2NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGh5bGF0aW9uIG9mIEROQSBpbiBwcm9rYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qmlya2jDpHVzZXI6IEJhc2VsOyAgY2l0YXRpb25feWVhcj0xOTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmlvbG9neSBvZiByZXN0cmljdGlvbiBhbmQgYW50aS1yZXN0cmljdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00NjYtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcmUgYmFzZXMgaW4gYW5pbWFsIEROQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTcwOyAgY2l0YXRpb25fdm9sdW1lPTIyNTsgIGNpdGF0aW9uX3BhZ2VzPTk0OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gb24gTjYtYWRlbmluZSBpbiBDLiBlbGVnYW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTg2OC03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TjYtbWV0aHlsZGVveHlhZGVub3NpbmUgbWFya3MgYWN0aXZlIHRyYW5zY3JpcHRpb24gc3RhcnQgc2l0ZXMgaW4gQ2hsYW15ZG9tb25hczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjE7ICBjaXRhdGlvbl9wYWdlcz04NzktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU42LW1ldGh5bGFkZW5pbmUgRE5BIG1vZGlmaWNhdGlvbiBpbiBEcm9zb3BoaWxhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTg5My05MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhbCBETkEgbWV0aHlsYXRpb24gYW5kIG1ldGh5bG9tZXMuIEluOiBKZWx0c2NoLCBBLiwgSnVya293c2thLCBSLiBaLiAoZWRzKSBETkEgTWV0aHlsdHJhbnNmZXJhc2VzIOKAkyBSb2xlIGFuZCBGdW5jdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWR2LiBFeHAuIE1lZC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT05NDU7ICBjaXRhdGlvbl9wYWdlcz0zNS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGFkZW5pbmUgbWV0aHlsdHJhbnNmZXJhc2UgKERhbSkgY29udHJvbHMgdGhlIGV4cHJlc3Npb24gb2YgdGhlIGN5dG90b3hpYyBlbnRlcm90b3hpbiAoYWN0KSBnZW5lIG9mIEFlcm9tb25hcyBoeWRyb3BoaWxhIHZpYSB0Uk5BIG1vZGlmeWluZyBlbnp5bWUtZ2x1Y29zZS1pbmhpYml0ZWQgZGl2aXNpb24gcHJvdGVpbiAoR2lkQSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDk4OyAgY2l0YXRpb25fcGFnZXM9MjgwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGh5bGF0aW9uIHBhdHRlcm5zIGluIHBhcCByZWd1bGF0b3J5IEROQSBjb250cm9sIHB5ZWxvbmVwaHJpdGlzLWFzc29jaWF0ZWQgcGlsaSBwaGFzZSB2YXJpYXRpb24gaW4gRS4gY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT03NjsgIGNpdGF0aW9uX3BhZ2VzPTU3Ny04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGJpbmRpbmcgb2YgTHJwIHRvIHR3byBzZXRzIG9mIHBhcCBETkEgYmluZGluZyBzaXRlcyBtZWRpYXRlZCBieSBQYXAgSSByZWd1bGF0ZXMgUGFwIHBoYXNlIHZhcmlhdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FTUJPIEo7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01Nzg1LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gb2YgRGNtLW1lZGlhdGVkIGN5dG9zaW5lIEROQSBtZXRoeWxhdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gTGV0dDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMjg7ICBjaXRhdGlvbl9wYWdlcz03OC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGJpbmRpbmcgb2YgdGhlIGNlbGwgY3ljbGUgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvciBHY3JBIGRlcGVuZHMgb24gTjYtYWRlbm9zaW5lIG1ldGh5bGF0aW9uIGluIENhdWxvYmFjdGVyIGNyZXNjZW50dXMgYW5kIG90aGVyIGFscGhhcHJvdGVvYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzNTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSBtZXRoeWxvbWUgY2hhcmFjdGVyaXphdGlvbiBvZiBNeWNvcGxhc21hIGdlbml0YWxpdW0gYW5kIE15Y29wbGFzbWEgcG5ldW1vbmlhZSBhdCBzaW5nbGUtYmFzZSByZXNvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzE5MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlcGlnZW5vbWljIGxhbmRzY2FwZSBvZiBwcm9rYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA1ODU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtcyBhbmQgc3ludGhldGljIGJpb2xvZ3kgZm9yIHRoZSBiaW90ZWNobm9sb2dpY2FsIGFwcGxpY2F0aW9uIG9mIGN5YW5vYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTg7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz05NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsdHJhbnNmZXJhc2VzIG9mIHRoZSBjeWFub2JhY3Rlcml1bSBBbmFiYWVuYSBQQ0MgNzEyMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xNDkxLTUwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pcXVlIG1vZGlmaWNhdGlvbiBvZiBhZGVuaW5lIGluIGdlbm9taWMgRE5BIG9mIHRoZSBtYXJpbmUgY3lhbm9iYWN0ZXJpdW0gVHJpY2hvZGVzbWl1bSBzcC4gc3RyYWluIE5JQkIgMTA2NzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTE3MzsgIGNpdGF0aW9uX3BhZ2VzPTcwNTktNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjeWFub2JhY3Rlcml1bSBTeW5lY2hvY3lzdGlzIHNwLiBzdHJhaW4gUENDIDY4MDMgZXhwcmVzc2VzIGEgRE5BIG1ldGh5bHRyYW5zZmVyYXNlIHNwZWNpZmljIGZvciB0aGUgcmVjb2duaXRpb24gc2VxdWVuY2Ugb2YgdGhlIHJlc3RyaWN0aW9uIGVuZG9udWNsZWFzZSBQdnVJOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTgwOyAgY2l0YXRpb25fcGFnZXM9NDExNi0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlbWV0aHlsYXRpb24gb2YgZm9yZWlnbiBETkEgaW1wcm92ZXMgaW50ZWdyYXRpdmUgdHJhbnNmb3JtYXRpb24gZWZmaWNpZW5jeSBpbiBTeW5lY2hvY3lzdGlzIHNwLiBzdHJhaW4gUENDIDY4MDM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTg1MDAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9ldm9sdXRpb24gaW4gY3lhbm9iYWN0ZXJpYTogcmUtc2VxdWVuY2luZyBhIG1vdGlsZSBzdWJzdHJhaW4gb2YgU3luZWNob2N5c3RpcyBzcC4gUENDIDY4MDM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz00MzUtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVyaWMgYXNzaWdubWVudHMsIHN0cmFpbiBoaXN0b3JpZXMgYW5kIHByb3BlcnRpZXMgb2YgcHVyZSBjdWx0dXJlcyBvZiBjeWFub2JhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBHZW4uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk3OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0xLTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVwYXJhdGlvbiBvZiBnZW5vbWljIEROQSBmcm9tIGJhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBQcm90LiBNb2xlYy4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpbW1vbWF0aWM6IGEgZmxleGlibGUgdHJpbW1lciBmb3IgSWxsdW1pbmEgc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yMTE0LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaXNtYXJrOiBhIGZsZXhpYmxlIGFsaWduZXIgYW5kIG1ldGh5bGF0aW9uIGNhbGxlciBmb3IgQmlzdWxmaXRlLVNlcSBhcHBsaWNhdGlvbnMuIEtydWVnZXIgRjEsIEFuZHJld3MgU1I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTU3MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGdhcHBlZC1yZWFkIGFsaWdubWVudCB3aXRoIEJvd3RpZSAyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29sZCBTcHJpbmcgSGFyYm91ciBMYWJvcmF0b3J5IFByZXNzLCBDb2xkIFNwcmluZyBIYXJib3IsIE5ZOyAgY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmc7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgb2YgdGhlIHVuaWNlbGx1bGFyIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgc3AuIHN0cmFpbiBQQ0MgNjgwMy4gSUkuIFNlcXVlbmNlIGRldGVybWluYXRpb24gb2YgdGhlIGVudGlyZSBnZW5vbWUgYW5kIGFzc2lnbm1lbnQgb2YgcG90ZW50aWFsIHByb3RlaW4tY29kaW5nIHJlZ2lvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEwOS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVjdG9ycyBmb3IgdGhlIGNvbXBsZW1lbnRhdGlvbiBhbmFseXNpcyBvZiBjeWFub2JhY3RlcmlhbCBtdXRhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1SdXNzLiBKLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTIyOC0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBtZWFzdXJlbWVudCB3aXRoIHRoZSBmb2xpbiBwaGVub2wgcmVhZ2VudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk1MTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz0yNjUtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkLiBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBkYXRhYmFzZSBzZWFyY2hlcyB1c2luZyBjb21wb3NpdGlvbmFsbHkgYWRqdXN0ZWQgc3Vic3RpdHV0aW9uIG1hdHJpY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIEo7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjcyOyAgY2l0YXRpb25fcGFnZXM9NTEwMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGFsVyBhbmQgQ2x1c3RhbFggdmVyc2lvbiAyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5NDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTU6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdXNpbmcgbWF4aW11bSBsaWtlbGlob29kLCBldm9sdXRpb25hcnkgZGlzdGFuY2UsIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSBtZXRob2RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yNzMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtZXRoeWxvbWVzIG9mIHNpeCBiYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0xMTQ1MC0xMTQzNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpc3VsZml0ZSBzZXF1ZW5jaW5nIHByb3RvY29sIGRpc3BsYXlzIGJvdGggNS1tZXRoeWxjeXRvc2luZSBhbmQgTjQtbWV0aHlsY3l0b3NpbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFuYWwuIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjcxOyAgY2l0YXRpb25fcGFnZXM9MTE2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJFQkFTReKAlGEgZGF0YWJhc2UgZm9yIEROQSByZXN0cmljdGlvbiBhbmQgbW9kaWZpY2F0aW9uOiBlbnp5bWVzLCBnZW5lcyBhbmQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZC4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4IChEYXRhYmFzZSBpc3N1ZSk7ICBjaXRhdGlvbl9wYWdlcz1EMjM0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgc3RydWN0dXJlIGFuZCBleHByZXNzaW9uIG9mIE1ib0kgcmVzdHJpY3Rpb24tbW9kaWZpY2F0aW9uIHN5c3RlbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yMzA5LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nIGFuZCBzZXF1ZW5jZSBhbmFseXNpcyBvZiB0aGUgZ2VuZXMgY29kaW5nIGZvciBFY281N0kgdHlwZSBJViByZXN0cmljdGlvbi1tb2RpZmljYXRpb24gZW56eW1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZC4gUmVzOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NjA1MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DeWFub0Jhc2U6IHRoZSBjeWFub2JhY3RlcmlhIGdlbm9tZSBkYXRhYmFzZSB1cGRhdGUgMjAxMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZC4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4IChEYXRhYmFzZSBpc3N1ZSk7ICBjaXRhdGlvbl9wYWdlcz1EMzc5LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgcHJpbWFyeSB0cmFuc2NyaXB0b21lIG9mIFN5bmVjaG9jeXN0aXMgc3AuIFBDQyA2ODAzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTI3LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdobHkgaXRlcmF0ZWQgcGFsaW5kcm9taWMgc2VxdWVuY2VzIChISVBzKSBhbmQgdGhlaXIgcmVsYXRpb25zaGlwIHRvIEROQSBtZXRoeWx0cmFuc2ZlcmFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUxpZmUgKEJhc2VsKTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9OTIxLTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5ndWxhciBvdmVyLXJlcHJlc2VudGF0aW9uIG9mIGFuIG9jdGFtZXJpYyBwYWxpbmRyb21lLCBISVAxLCBpbiBETkEgZnJvbSBtYW55IGN5YW5vYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWQuIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTcyOS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SElQMSBwcm9wYWdhdGVzIGluIGN5YW5vYmFjdGVyaWFsIEROQSB2aWEgbnVjbGVvdGlkZSBzdWJzdGl0dXRpb25zIGJ1dCBwcm9tb3RlcyBleGNpc2lvbiBhdCBzaW1pbGFyIGZyZXF1ZW5jaWVzIGluIEVzY2hlcmljaGlhIGNvbGkgYW5kIFN5bmVjaG9jb2NjdXMgUENDIDc5NDI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTgxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2Vuc2l0aXZpdHkgb2YgY2hsb3JvcGxhc3QgZ2VuZSBleHByZXNzaW9uIHRvIEROQSBtZXRoeWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjgyOyAgY2l0YXRpb25fcGFnZXM9MTctMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWx0cmFuc2ZlcmFzZXMgYW5kIGVwaWdlbmV0aWMgcmVndWxhdGlvbiBpbiBiYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBNaWNyb2Jpb2wuIFJldjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTU3NS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGFkZW5pbmUgbWV0aHlsYXRpb24gaXMgcmVxdWlyZWQgdG8gcmVwbGljYXRlIGJvdGggVmlicmlvIGNob2xlcmEgY2hyb21vc29tZXMgb25jZSBwZXIgY2VsbCBjeWNsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA5MzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0aW5jdCByZXBsaWNhdGlvbiByZXF1aXJlbWVudHMgZm9yIHRoZSB0d28gVmlicmlvIGNob2xlcmEgY2hyb21vc29tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTE0OyAgY2l0YXRpb25fcGFnZXM9NTIxLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdXNlIG9yIGRpZTogaG93IHRvIHN1cnZpdmUgdGhlIGxvc3Mgb2YgRGFtIGluIFZpYnJpbyBjaG9sZXJhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz02NjUtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBhbmFseXNpcyBhbmQgbXV0YXRpb24gb2YgYSBkbmFBLWxpa2UgZ2VuZSBpbiBTeW5lY2hvY3lzdGlzIHNwLiBzdHJhaW4gUENDIDY4MDM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xODA7ICBjaXRhdGlvbl9wYWdlcz00OTQ2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2lmaWNhdGlvbiBvZiBEbmFBIGRlcGVuZGVuY3kgZm9yIEROQSByZXBsaWNhdGlvbiBpbiBjeWFub2JhY3RlcmlhbCBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlzbWUgSjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTExMTMtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyZS1ndWlkZWQgYW5hbHlzZXMgcmV2ZWFscyBuaW5lIHNlcXVlbmNlIG1vdGlmcyBjb25zZXJ2ZWQgYW1vbmcgRE5BIGFtaW5vLW1ldGh5bHRyYW5zZmVyYXNlcywgYW5kIHN1Z2dlc3RzIGEgY2F0YWx5dGljIG1lY2hhbmlzbSBmb3IgdGhlc2UgZW56eW1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI1MzsgIGNpdGF0aW9uX3BhZ2VzPTYxOC0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BUGxvdHRlcjogY2lyY3VsYXIgYW5kIGxpbmVhciBpbnRlcmFjdGl2ZSBnZW5vbWUgdmlzdWFsaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTktMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS80LzM0My8yNTUwMTY2NC9kc3kwMDYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEROQSBtZXRoeWxhdGlvbiBpbiBiYWN0ZXJpYSBpcyBpbXBvcnRhbnQgZm9yIGRlZmVuc2UgYWdhaW5zdCBmb3JlaWduIEROQSwgYnV0IGlzIGFsc28gaW52b2x2ZWQgaW4gRE5BIHJlcGFpciwgcmVwbGljYXRpb24sIGNocm9tb3NvbWUgcGFydGl0aW9uaW5nLCBhIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODUwOTgyJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZCi0+Obkho7seP9DMjMBOKmrlTL/" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f343%2f4850982" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f343%2f4850982" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109685" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&amp;Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/4">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="120121995">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121995">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121997">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121997">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122001">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122001">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122015">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122015">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122046">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122046">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122054">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122054">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122062">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122062">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/25/4/353/4922629">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identification of the DNA methyltransferases establishing the methylome of the cyanobacterium <em>Synechocystis</em> sp. PCC 6803
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin Hagemann</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy006-cor1" data-open="dsy006-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +49 381 4986110. Fax. +49 381 4986112. Email: <a href="mailto:martin.hagemann@uni-rostock.de" target="_blank">martin.hagemann@uni-rostock.de</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+Hagemann">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hagemann M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hagemann M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Martin Hagemann</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katrin G&#228;rtner</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katrin+G%c3%a4rtner">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=G&#228;rtner K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;G&#228;rtner K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Katrin Gärtner</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthias Scharnagl</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biosciences, Plant Physiology, University of Rostock, Rostock, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthias+Scharnagl">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Scharnagl M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Scharnagl M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Matthias Scharnagl</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paul Bolay</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Faculty of Biology, University of Freiburg, Freiburg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Paul+Bolay">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bolay P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bolay P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Paul Bolay</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Steffen C Lott</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Faculty of Biology, University of Freiburg, Freiburg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Steffen+C+Lott">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lott S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lott S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Steffen C Lott</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Janina Fuss</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Janina+Fuss">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fuss J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fuss J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Janina Fuss</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bruno Huettel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bruno+Huettel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huettel B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huettel B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Bruno Huettel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Richard Reinhardt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Max Planck-Genome-Centre Cologne, Max Planck Institute for Plant Breeding Research, Köln, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Richard+Reinhardt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Reinhardt R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Reinhardt R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Richard Reinhardt</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stephan Kl&#228;hn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Faculty of Biology, University of Freiburg, Freiburg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stephan+Kl%c3%a4hn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kl&#228;hn S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kl&#228;hn S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stephan Klähn</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wolfgang R Hess</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Faculty of Biology, University of Freiburg, Freiburg, Germany</div><div class="aff">Freiburg Institute for Advanced Studies, University of Freiburg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wolfgang+R+Hess">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hess W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hess W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wolfgang R Hess</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy006-FM1"><div class="fn"><div class="footnote-content">Present address: Helmholtz Centre for Environmental Research - UFZ, D-04318 Leipzig, Germany.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy006-FM2"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 343–352, <a href='https://doi.org/10.1093/dnares/dsy006'>https://doi.org/10.1093/dnares/dsy006</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">12 February 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 September 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 January 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4850982" href="/dnaresearch/article-pdf/25/4/343/25501664/dsy006.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Martin Hagemann, Katrin Gärtner, Matthias Scharnagl, Paul Bolay, Steffen C Lott, Janina Fuss, Bruno Huettel, Richard Reinhardt, Stephan Klähn, Wolfgang R Hess;  Identification of the DNA methyltransferases establishing the methylome of the cyanobacterium <em>Synechocystis</em> sp. PCC 6803, <em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 343–352, <a href='https://doi.org/10.1093/dnares/dsy006'>https://doi.org/10.1093/dnares/dsy006</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4850982?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4850982?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4850982?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4850982?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4850982?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4850982?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4850982?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identification%20of%20the%20DNA%20methyltransferases%20establishing%20the%20methylome%20of%20the%20cyanobacterium%20%26lt%3Bem%26gt%3BSynechocystis%26lt%3B%2Fem%26gt%3B%20sp.%20PCC%206803&amp;publicationDate=2018-02-12&amp;volumeNum=25&amp;issueNum=4&amp;author=Hagemann%2C%20Martin%3B%20G%C3%A4rtner%2C%20Katrin&amp;startPage=343&amp;endPage=352&amp;contentId=10.1093%2Fdnares%2Fdsy006&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f343%2f4850982" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=120121995 id="120121995" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>DNA methylation in bacteria is important for defense against foreign DNA, but is also involved in DNA repair, replication, chromosome partitioning, and regulatory processes. Thus, characterization of the underlying DNA methyltransferases in genetically tractable bacteria is of paramount importance. Here, we characterized the methylome and orphan methyltransferases in the model cyanobacterium <em>Synechocystis</em> sp. PCC 6803. Single molecule real-time (SMRT) sequencing revealed four DNA methylation recognition sequences in addition to the previously known motif <sup>m5</sup>CGATCG, which is recognized by M.Ssp6803I. For three of the new recognition sequences, we identified the responsible methyltransferases. M.Ssp6803II, encoded by the <em>sll0729</em> gene, modifies GG<sup>m4</sup>CC, M.Ssp6803III, encoded by <em>slr1803</em>, represents the cyanobacterial <em>dam</em>-like methyltransferase modifying G<sup>m6</sup>ATC, and M.Ssp6803V, encoded by <em>slr6095</em> on plasmid pSYSX, transfers methyl groups to the bipartite motif GG<sup>m6</sup>AN<sub>7</sub>TTGG/CCA<sup>m6</sup>AN<sub>7</sub>TCC. The remaining methylation recognition sequence GA<sup>m6</sup>AGGC is probably recognized by methyltransferase M.Ssp6803IV encoded by <em>slr6050.</em> M.Ssp6803III and M.Ssp6803IV were essential for the viability of <em>Synechocystis</em>, while the strains lacking M.Ssp6803I and M.Ssp6803V showed growth similar to the wild type. In contrast, growth was strongly diminished of the Δ<em>sll0729</em> mutant lacking M.Ssp6803II. These data provide the basis for systematic studies on the molecular mechanisms impacted by these methyltransferases.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="cyanobacteria">cyanobacteria</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methyltransferase&quot;">DNA methyltransferase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mutant">mutant</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;photosynthetic pigment&quot;">photosynthetic pigment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogenetics">phylogenetics</a></div>                            <h2 scrollto-destination=120121997 id="120121997" class="section-title" >1. Introduction</h2>
<p>DNA serves as the universal carrier of information in living cells. In addition to the genetic information encoded in the nucleotide sequence, an epigenetic level of information is encoded in DNA modifications, which play a fundamental role in the differentiation and development of eukaryotic cells, in cancer development and prevention, aging, and acclimation to cellular and environmental stimuli.<a reveal-id="dsy006-B1" data-open="dsy006-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy006-B2" data-open="dsy006-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> DNA modification is carried out by methyltransferases, which transfer methyl groups from the universal substrate S-adenosyl-methionine (AdoMet) to their respective recognition sequences. Among bacteria, epigenetic modifications have been most commonly associated with the genome-wide methylation of specific DNA sequences, which are linked to restriction-modification (RM)-based defense mechanisms against foreign DNA, such as the DNA from phages.<a reveal-id="dsy006-B3" data-open="dsy006-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy006-B4" data-open="dsy006-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> C5-cytosine methylation is regarded as the most common DNA modification in eukaryotes,<a reveal-id="dsy006-B5" data-open="dsy006-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> but N6-adenine methylation has also been reported.<a reveal-id="dsy006-B6 dsy006-B7 dsy006-B8" data-open="dsy006-B6 dsy006-B7 dsy006-B8" class="link link-ref link-reveal xref-bibr"><sup>6–8</sup></a> Prokaryotic DNA methyltransferases typically catalyze N6-adenine, N4- or C5-cytosine methylation.</p><p>In addition to the methyltransferases in RM systems, many prokaryotic genomes harbour orphan DNA methyltransferases that act independently of the RM systems. Most of these solitary or orphan methyltransferases are not well characterized, but functions in the regulation of gene expression, DNA replication, repair, and others have been suggested.<a reveal-id="dsy006-B9" data-open="dsy006-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> One of the most widespread and best characterized orphan methyltransferases is the <em>E. coli</em> Dam enzyme, an N6-adenine-specific methyltransferase modifying the target sequence GATC. Dam methylation plays an important role in DNA repair and replication (reviewed in [<a reveal-id="dsy006-B9" data-open="dsy006-B9" class="link link-ref link-reveal xref-bibr">9</a>]) but also in the regulation of gene expression<a reveal-id="dsy006-B10" data-open="dsy006-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> or phase variation of uropathogenic <em>E. coli</em>.<a reveal-id="dsy006-B11" data-open="dsy006-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsy006-B12" data-open="dsy006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Dcm, which mediates cytosine DNA methylation, is another widespread orphan methyltransferase activity found in 162 strains of <em>E. coli</em>.<a reveal-id="dsy006-B13" data-open="dsy006-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The molecular details of how DNA methylation regulates gene expression and subsequently the cell cycle have been elucidated in the bacterial model system <em>Caulobacter crescentus</em>.<a reveal-id="dsy006-B14" data-open="dsy006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p><p>Single molecule real-time (SMRT) sequencing permits the parallel estimation of the methylation status of specific nucleotides. It was first used for the direct methylome profiling of <em>Mycoplasma pneumoniae</em>.<a reveal-id="dsy006-B15" data-open="dsy006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> More recently, SMRT sequencing has been applied to characterize the methylomes of 230 prokaryotic strains, revealing that the majority of them contain extensive and variable DNA methylation patterns.<a reveal-id="dsy006-B16" data-open="dsy006-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Cyanobacteria, which are the only prokaryotes that perform oxygenic photosynthesis and are increasingly used as cell factories in green biotechnology,<a reveal-id="dsy006-B17" data-open="dsy006-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> have been only scarcely characterized with regard to epigenetic modifications. In <em>Anabaena</em> sp. PCC 7120, four different orphan methyltransferases were detected in addition to the enzymes of the endogenous RM systems.<a reveal-id="dsy006-B18" data-open="dsy006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> A high degree of adenine methylation was reported for the marine cyanobacterium <em>Trichodesmium</em> sp. NIBB1067.<a reveal-id="dsy006-B19" data-open="dsy006-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> In the present study, we analyzed the methylome of the model cyanobacterium <em>Synechocystis</em> sp. PCC 6803 (hereafter <em>Synechocystis</em>), which seems to be virtually free of endogenous restriction endonucleases.<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Nevertheless, the chromosomal DNA of <em>Synechocystis</em> was found to be methylated, and the genome contains several orphan methyltransferase genes. Only the cytosine-specific orphan methyltransferase M.Ssp6803I, encoded by gene <em>slr0214</em>, was previously analyzed.<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The M.Ssp6803I-dependent modification of the <sup>m5</sup>CGATCG motif improved the integration efficiency of external DNA into the <em>Synechocystis</em> chromosome.<a reveal-id="dsy006-B21" data-open="dsy006-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Here, we present the first methylome analysis of <em>Synechocystis</em>, which identified five DNA methylation recognition sequences and the corresponding methyltransferases.</p>                            <h2 scrollto-destination=120122001 id="120122001" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=120122002 id="120122002" class="section-title" >2.1. Strains and culture conditions</h3>
<p><em>Synechocystis</em> sp. 6803 substrain PCC-M<a reveal-id="dsy006-B22" data-open="dsy006-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> was used in all experiments. The axenic strain was maintained on agar plates supplemented with BG11 mineral medium<a reveal-id="dsy006-B23" data-open="dsy006-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> at 30°C under constant illumination. Transformants were initially selected on media containing 10 µg ml<sup>−1</sup> kanamycin (Km; Sigma), while the segregation of clones and cultivation of mutants was performed at 50 µg ml<sup>−1 </sup>Km. For physiological characterization, axenic cultures of the different strains (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were grown photoautotrophically in BG11 medium, either under slight shaking in Erlenmeyer flasks at 50 µmol photons s<sup>−1 </sup>m<sup>−2</sup>, or under bubbling with CO<sub>2</sub>-enriched air (5% [v/v]) in batch cultures at 29°C under continuous illumination of 180 µmol photons s<sup>−1 </sup>m<sup>−2</sup> (warm light, Osram L58 W32/3). Contamination by heterotrophic bacteria was evaluated by microscopy or spreading of 0.2 mL culture on LB plates. The <em>E. coli</em> strains TG1, TOP10, and DH5α were used for routine DNA manipulations. <em>E. coli</em> was cultured in LB medium at 37°C. Growth was followed by measurements of the optical density at 750 nm (OD<sub>750</sub>) for <em>Synechocystis</em> and at 500 nm (OD<sub>500</sub>) for <em>E. coli</em>.</p>                            <h3 scrollto-destination=120122004 id="120122004" class="section-title" >2.2. Methylome analysis</h3>
<p>For SMRT sequencing, high quality genomic DNA was isolated using the CTAB protocol.<a reveal-id="dsy006-B24" data-open="dsy006-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Libraries were prepared according to the large SMRTbell gDNA protocol (Pacific Biosciences) with 10 kb insert size. Genomic DNA was sequenced with a PacBio RS II platform. Base modifications were analyzed using the program RS_Modification_Detection.1 from Pacific Biosciences (v. 2.3.0). For bisulfite sequencing, ∼200 ng of DNA were bisulfite treated with the Zymo Gold kit (Zymo Research) and libraries constructed using the Ovation Ultra-Low Methyl-Seq library kit (NuGEN) following manufacturer’s instructions, followed by sequencing on the Illumina HiSeq2500 system yielding 2 559 017 raw reads. The sequences were quality filtered and adapter trimmed using Trimmomatic v0.36<a reveal-id="dsy006-B25" data-open="dsy006-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>) and FastQC v0.67 (<a class="link link-uri" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a> (1 February 2018, date last accessed)) leaving 2 552 913 reads for further analysis. For mapping to the <em>Synechocystis</em> chromosome and quantitative evaluation we used Bismark (v0.17 with default options<a reveal-id="dsy006-B26" data-open="dsy006-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a>) in conjunction with Bowtie 2.<a reveal-id="dsy006-B27" data-open="dsy006-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> All SMRT and Illumina sequencing raw data are available from the National Center for Biotechnology Information at <a class="link link-uri" href="https://www.ncbi.nlm.nih.gov/biosample/8378604" target="_blank">https://www.ncbi.nlm.nih.gov/biosample/8378604</a> (1 February 2018, date last accessed) (BioProject ID: PRJNA430784, BioSample: SAMN08378604, SRA: SRS2844079).</p>                            <h3 scrollto-destination=120122006 id="120122006" class="section-title" >2.3. DNA manipulations</h3>
<p>The isolation of total DNA from <em>Synechocystis</em> was performed as described previously.<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> All other DNA techniques, such as plasmid isolation, transformation of <em>E. coli</em>, ligations and restriction analysis (restriction enzymes were obtained from Promega and New England Biolabs) followed standard methods.<a reveal-id="dsy006-B28" data-open="dsy006-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> For the restriction analyses using chromosomal DNA from <em>Synechocystis</em>, the restriction endonucleases were used in a 10-fold excess and were incubated for at least 16 h at 37°C to ensure complete digestion. Synthetic primers were deduced from the complete genome sequence of <em>Synechocystis</em><a reveal-id="dsy006-B29" data-open="dsy006-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> for the specific amplification of putative DNA-specific methyltransferase-coding genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Interposon mutagenesis was used to generate mutants defective in these genes. For this purpose, DNA fragments containing their encoding sequences were amplified by PCR and cloned into pGEM-T (Promega). The <em>aph</em>II gene, conferring Km resistance from pUC4K (Pharmacia), was introduced into selected restriction sites (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, <a reveal-id="dsy006-F2" data-open="dsy006-F2" class="link link-reveal link-table xref-fig">Figs 2</a>, <a reveal-id="dsy006-F3" data-open="dsy006-F3" class="link link-reveal link-table xref-fig">3</a>, and <a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">5</a>), and verified constructs were transferred into <em>Synechocystis</em> as described.<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a></p>                            <h3 scrollto-destination=120122008 id="120122008" class="section-title" >2.4. Generation of complementation strains</h3>
<p>For the ectopic expression of <em>sll0729</em> and <em>ssl1378</em>, their coding sequences were fused to the regulatory sequences of the <em>ziaA</em> gene. The upstream region of <em>ziaA</em>, including P<em>ziaA</em>, its 5′-UTR, and the upstream <em>ziaR</em>-gene, as well as <em>sll0729</em> or <em>ssl1378</em>, were amplified by PCR (for oligonucleotides, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The PCR products were digested with <em>Cla</em>I, followed by heat inactivation. The digested products were ligated and re-amplified with PCR, and the resulting <em>PziaA::sll0729</em> and <em>PziaA::ssl1378</em> fusions were cloned into the self-replicating, broad-host range vector pVZ322 via <em>Xba</em>I/<em>Xho</em>I cleavage sites. The resulting plasmids were introduced into Δ<em>sll0729</em> via conjugal transfer, as described previously.<a reveal-id="dsy006-B30" data-open="dsy006-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a></p>                            <h3 scrollto-destination=120122010 id="120122010" class="section-title" >2.5. Generation of recombinant proteins and methyltransferase assay</h3>
<p>For overexpression and purification of the putative DNA-specific methyltransferases, the open reading frames (ORFs) were amplified from chromosomal DNA by PCR using primers for the directed in-frame cloning with the N-terminal His-tag into vectors of the pBAD/His A, B, C series (Invitrogen). Correct insertions were verified by sequencing. For overexpression, recombinant cells of the methyltransferase-defective <em>E. coli</em> strain TOP10 were cultured at 30°C in LB medium. The expression of the recombinant protein was induced at OD<sub>500</sub> = 1.0 by addition of arabinose (0.002% final concentration). The proteins were extracted from <em>E. coli</em> by sonication, and the fusion proteins were purified on a Ni-NTA matrix (ProBond<sup>TM</sup> Resin, novex, life technologies) after elution with an imidazole gradient. The resulting protein fractions were evaluated using SDS gels containing 12% acrylamide. The recombinant proteins were detected by immuno-blotting with an antibody specific for the N-terminal His-tag (Invitrogen).</p><p>The DNA-specific methyltransferase activity was assayed by incubation of non-methylated chromosomal DNA of <em>Micrococcus lysodeikticus</em> (Sigma) with [<sup>3</sup>H]-AdoMet (Amersham), as described.<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The protein content was estimated according to reference.<a reveal-id="dsy006-B31" data-open="dsy006-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a></p>                            <h3 scrollto-destination=120122013 id="120122013" class="section-title" >2.6. Phylogenetic analysis</h3>
<p>The phylogenetic comparison included up to 20 of the most similar proteins found in BlastP searches<a reveal-id="dsy006-B32" data-open="dsy006-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsy006-B33" data-open="dsy006-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> against Genbank with the amino acid sequence of M.Ssp6803II (<a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/Sll0729" target="_blank">Sll0729</a>). In addition, the characterized DNA methyltransferases M.Ssp6803I (<a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/Slr0214" target="_blank">Slr0214</a>), M.Ssp6803III (<a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/Slr1803" target="_blank">Slr1803</a>) and some closely related proteins, as well as functionally characterized methyltransferases, were included. The protein sequences were aligned using ClustalW<a reveal-id="dsy006-B34" data-open="dsy006-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> embedded in the BioEdit software package. Extended sequence parts were manually removed from the N-terminal and C-terminal ends. The phylogenetic tree was calculated in MEGA5 using the neighbour joining method.<a reveal-id="dsy006-B35" data-open="dsy006-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p>                            <h2 scrollto-destination=120122015 id="120122015" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=120122016 id="120122016" class="section-title" >3.1. The <em>Synechocystis</em> methylome</h3>
<p>SMRT and bisulfite sequencing were used to analyze the <em>Synechocystis</em> methylome. The <sup>m</sup>CGATCG motif was previously characterized but the nature of the methylation could only be inferred based on protein similarity.<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Bisulfite sequencing permits the direct detection of 5-methylcytosines, which was found for this motif, yielding <sup>m5</sup>CGATCG (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). In addition, SMRT sequencing indicated that the <em>Synechocystis</em> genome harbours at least four further methylation recognition sequences, which were found on the chromosome with different frequencies: GG<sup>m</sup>CC, G<sup>m6</sup>ATC, GA<sup>m6</sup>AGGC, and GG<sup>m6</sup>AN<sub>7</sub>TTGG/CCA<sup>m6</sup>AN<sub>7</sub>TCC (<a reveal-id="dsy006-F1" data-open="dsy006-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, <a reveal-id="dsy006-T1" data-open="dsy006-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The SMRT sequencing results showed the methylation of GG<sup>m</sup>CC at the first C but failed to identify the precise modification. While SMRT sequencing can detect <sup>m6</sup>A with high accuracy and sensitivity, it is less sensitive towards <sup>m4</sup>C and performs badly on <sup>m5</sup>C methylation.<a reveal-id="dsy006-B36" data-open="dsy006-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> However, standard bisulfite sequencing protocols may also be used to map <sup>m4</sup>C, because <sup>m4</sup>C is partially resistant to bisulfite-mediated deamination.<a reveal-id="dsy006-B37" data-open="dsy006-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Indeed, we observed a GG<sup>m</sup>CC methylation frequency of ∼50% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), matching previous records on the efficiency of bisulfite treatment for the detection of <sup>m4</sup>C.<a reveal-id="dsy006-B37" data-open="dsy006-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> In contrast, the methylation at C5 position of <sup>m5</sup>CGATCG motifs was detected to 100% by bisulfite sequencing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) but was not observed by the SMRT method. The fact that the GGCC methylation was detected both by SMRT and by bisulfite sequencing suggests that it is at the N4 position (GG<sup>m4</sup>CC). The bisulfite sequencing data were used for a global analysis of the methylation of CGATCG and GGCC sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). This analysis revealed 80–85% methylated <sup>m5</sup>CGATCG sites. About 10% of the sites seem to be not completely or unmethylated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). In the case of the GGCC methylation site, bisulfite sequencing revealed 90% of all sites being GG<sup>m4</sup>CC (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2C</a></span>). The great majority of modifications of GGCC and CGATCG sites is consistent with previous restriction analysis, where DNA isolated from <em>Synechocystis</em> was resistant to treatment with methylation-sensitive restriction enzymes <em>Hae</em>III (recognition sequence GGCC), <em>Pvu</em>I (recognition sequence CGATCG), and <em>Mbo</em>I (recognition sequence GATC).<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Similarly, the DNA of the marine cyanobacterium <em>Trichodesmium</em> sp. NIBB1067 was also modified at GATC sites leading to stimulation or inhibition of methylation-dependent restriction enzymes.<a reveal-id="dsy006-B19" data-open="dsy006-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><div content-id="dsy006-T1" class="table-modal"><div class="table-wrap"><div id="dsy006-T1" data-id="dsy006-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Methyltransferases in <em>Synechocystis</em> sp. PCC 6803</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Methyl-transferase </th><th>Encoding gene </th><th>Localized on </th><th>Recognition sequence </th><th>Sites on chromosome </th><th>Reference </th></tr></thead><tbody><tr><td>M.Ssp6803I </td><td><em>slr0214</em> </td><td>Chromosome </td><td><sup>m5</sup>CGATCG </td><td>38 512 </td><td>This study, Scharnagl <em>et al.</em><a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> </td></tr><tr><td>M.Ssp6803II </td><td><em>sll0729</em> </td><td>Chromosome </td><td>GG<sup>m4</sup>CC </td><td>29 852 </td><td>This study </td></tr><tr><td>M.Ssp6803III </td><td><em>slr1803</em> </td><td>Chromosome </td><td>G<sup>m6</sup>ATC </td><td>10 236 </td><td>This study </td></tr><tr><td>M.Ssp6803IV </td><td><em>slr6050</em> </td><td>Plasmid pSYSX </td><td>GA<sup>m6</sup>AGGC </td><td>1317 </td><td>This study, REBASE<a reveal-id="dsy006-B38" data-open="dsy006-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> </td></tr><tr><td>M.Ssp6803V </td><td><em>slr6095</em> </td><td>Plasmid pSYSX </td><td>GG<sup>m6</sup>AN<sub>7</sub>TTGG/ CCA<sup>m6</sup>AN<sub>7</sub>TCC </td><td>1181 </td><td>This study </td></tr><tr><td>M.Ssp6803VI </td><td><em>ssl8010/sll8009</em> </td><td>Plasmid pSYSG </td><td>None </td><td> </td><td>This study </td></tr></tbody></table></div><div content-id="dsy006-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Methyl-transferase </th><th>Encoding gene </th><th>Localized on </th><th>Recognition sequence </th><th>Sites on chromosome </th><th>Reference </th></tr></thead><tbody><tr><td>M.Ssp6803I </td><td><em>slr0214</em> </td><td>Chromosome </td><td><sup>m5</sup>CGATCG </td><td>38 512 </td><td>This study, Scharnagl <em>et al.</em><a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> </td></tr><tr><td>M.Ssp6803II </td><td><em>sll0729</em> </td><td>Chromosome </td><td>GG<sup>m4</sup>CC </td><td>29 852 </td><td>This study </td></tr><tr><td>M.Ssp6803III </td><td><em>slr1803</em> </td><td>Chromosome </td><td>G<sup>m6</sup>ATC </td><td>10 236 </td><td>This study </td></tr><tr><td>M.Ssp6803IV </td><td><em>slr6050</em> </td><td>Plasmid pSYSX </td><td>GA<sup>m6</sup>AGGC </td><td>1317 </td><td>This study, REBASE<a reveal-id="dsy006-B38" data-open="dsy006-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> </td></tr><tr><td>M.Ssp6803V </td><td><em>slr6095</em> </td><td>Plasmid pSYSX </td><td>GG<sup>m6</sup>AN<sub>7</sub>TTGG/ CCA<sup>m6</sup>AN<sub>7</sub>TCC </td><td>1181 </td><td>This study </td></tr><tr><td>M.Ssp6803VI </td><td><em>ssl8010/sll8009</em> </td><td>Plasmid pSYSG </td><td>None </td><td> </td><td>This study </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy006-T1" data-id="dsy006-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Methyltransferases in <em>Synechocystis</em> sp. PCC 6803</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Methyl-transferase </th><th>Encoding gene </th><th>Localized on </th><th>Recognition sequence </th><th>Sites on chromosome </th><th>Reference </th></tr></thead><tbody><tr><td>M.Ssp6803I </td><td><em>slr0214</em> </td><td>Chromosome </td><td><sup>m5</sup>CGATCG </td><td>38 512 </td><td>This study, Scharnagl <em>et al.</em><a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> </td></tr><tr><td>M.Ssp6803II </td><td><em>sll0729</em> </td><td>Chromosome </td><td>GG<sup>m4</sup>CC </td><td>29 852 </td><td>This study </td></tr><tr><td>M.Ssp6803III </td><td><em>slr1803</em> </td><td>Chromosome </td><td>G<sup>m6</sup>ATC </td><td>10 236 </td><td>This study </td></tr><tr><td>M.Ssp6803IV </td><td><em>slr6050</em> </td><td>Plasmid pSYSX </td><td>GA<sup>m6</sup>AGGC </td><td>1317 </td><td>This study, REBASE<a reveal-id="dsy006-B38" data-open="dsy006-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> </td></tr><tr><td>M.Ssp6803V </td><td><em>slr6095</em> </td><td>Plasmid pSYSX </td><td>GG<sup>m6</sup>AN<sub>7</sub>TTGG/ CCA<sup>m6</sup>AN<sub>7</sub>TCC </td><td>1181 </td><td>This study </td></tr><tr><td>M.Ssp6803VI </td><td><em>ssl8010/sll8009</em> </td><td>Plasmid pSYSG </td><td>None </td><td> </td><td>This study </td></tr></tbody></table></div><div content-id="dsy006-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Methyl-transferase </th><th>Encoding gene </th><th>Localized on </th><th>Recognition sequence </th><th>Sites on chromosome </th><th>Reference </th></tr></thead><tbody><tr><td>M.Ssp6803I </td><td><em>slr0214</em> </td><td>Chromosome </td><td><sup>m5</sup>CGATCG </td><td>38 512 </td><td>This study, Scharnagl <em>et al.</em><a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> </td></tr><tr><td>M.Ssp6803II </td><td><em>sll0729</em> </td><td>Chromosome </td><td>GG<sup>m4</sup>CC </td><td>29 852 </td><td>This study </td></tr><tr><td>M.Ssp6803III </td><td><em>slr1803</em> </td><td>Chromosome </td><td>G<sup>m6</sup>ATC </td><td>10 236 </td><td>This study </td></tr><tr><td>M.Ssp6803IV </td><td><em>slr6050</em> </td><td>Plasmid pSYSX </td><td>GA<sup>m6</sup>AGGC </td><td>1317 </td><td>This study, REBASE<a reveal-id="dsy006-B38" data-open="dsy006-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> </td></tr><tr><td>M.Ssp6803V </td><td><em>slr6095</em> </td><td>Plasmid pSYSX </td><td>GG<sup>m6</sup>AN<sub>7</sub>TTGG/ CCA<sup>m6</sup>AN<sub>7</sub>TCC </td><td>1181 </td><td>This study </td></tr><tr><td>M.Ssp6803VI </td><td><em>ssl8010/sll8009</em> </td><td>Plasmid pSYSG </td><td>None </td><td> </td><td>This study </td></tr></tbody></table></div></div></p>                        <a id="120122018" scrollto-destination="120122018"></a>
<div id="dsy006-F1" data-id="dsy006-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f1.png?Expires=2147483647&Signature=N9sh61JzLKEaI8T3~cmk2h49PMAP3lT6jV-yrON2EtLc9ECHeVj9bHeyywI-4wHJqALaL~EUyqvn~X-UcRLcRJkcEIhqv9F1Mq~-iOurAY30urKYDg-YalFDMEwDUCHK8O38p-uZ2bbr60062HswCyH~mcNplKmJQrgcVov5EVWyiee7lsLGiQQYnvuCV19eBZiHe~D6Wa6ajyt16s9hFvYado4ie02c8j8L7gx~MuN~sPraieyNj-MUR2nwjQ0UQcTL~hIHZyJKQiG8nEJZV5sMyvnJ~FBH3~V7gLWJO3gGbJdZyiCMkRIzci1L3IiCNL71pvCrXBwa05z95JP3fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the methylome of Synechocystis sp. PCC 6803. The methylome of Synechocystis sp. strain PCC 6803 comprises five different methylation motifs, which were detected using SMRT (Pacific Biosciences) and bisulfite sequencing. The genome plot shows the distribution of the recognition sequences on both strands of the chromosome and was made by using DNAPlotter.54" path-from-xml="dsy006f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122018/dsy006f1.tif" path-from-xml="dsy006f1.tif" target="_blank">View large</a><a section="120122018" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f1.png?Expires=2147483647&Signature=Lq251RhZfCKOxGSeiqJlac3DMa~J0PjuLKcQydH64KLIJhnlxuZ73ek9mn20I8IchOLpCWC~2DGCC6T5h93haQeDWkDjiqzSqe4UuxMUAR-vvXyaLPGn6W5zezWKJj6Dxk09QvqNgjlHRSodqT8yz-Pv4S8Ol5CI2PYIs~D3s9WQqV6Vh4wljMH-vJRlzfBYr5kqzk543O2ursYXIVAwVNI14vTHaFjDQ4w8opJVZPN5Tt5RHXT7Rk2QUmX~MdNyR0lLDTO4fFTdTQQvjigh5scAABli0ovyC62OjAtpPW6UwBsimeQX7Lf9a2Og9QRicDubNX-uvTvPmp5j8z0C-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122018&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the methylome of <em>Synechocystis</em> sp. PCC 6803. The methylome of <em>Synechocystis</em> sp. strain PCC 6803 comprises five different methylation motifs, which were detected using SMRT (Pacific Biosciences) and bisulfite sequencing. The genome plot shows the distribution of the recognition sequences on both strands of the chromosome and was made by using DNAPlotter.<a reveal-id="dsy006-B54" data-open="dsy006-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a></p></div></div><div content-id="dsy006-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f1.png?Expires=2147483647&Signature=N9sh61JzLKEaI8T3~cmk2h49PMAP3lT6jV-yrON2EtLc9ECHeVj9bHeyywI-4wHJqALaL~EUyqvn~X-UcRLcRJkcEIhqv9F1Mq~-iOurAY30urKYDg-YalFDMEwDUCHK8O38p-uZ2bbr60062HswCyH~mcNplKmJQrgcVov5EVWyiee7lsLGiQQYnvuCV19eBZiHe~D6Wa6ajyt16s9hFvYado4ie02c8j8L7gx~MuN~sPraieyNj-MUR2nwjQ0UQcTL~hIHZyJKQiG8nEJZV5sMyvnJ~FBH3~V7gLWJO3gGbJdZyiCMkRIzci1L3IiCNL71pvCrXBwa05z95JP3fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the methylome of Synechocystis sp. PCC 6803. The methylome of Synechocystis sp. strain PCC 6803 comprises five different methylation motifs, which were detected using SMRT (Pacific Biosciences) and bisulfite sequencing. The genome plot shows the distribution of the recognition sequences on both strands of the chromosome and was made by using DNAPlotter.54" path-from-xml="dsy006f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122018/dsy006f1.tif" path-from-xml="dsy006f1.tif" target="_blank">View large</a><a section="120122018" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f1.png?Expires=2147483647&Signature=Lq251RhZfCKOxGSeiqJlac3DMa~J0PjuLKcQydH64KLIJhnlxuZ73ek9mn20I8IchOLpCWC~2DGCC6T5h93haQeDWkDjiqzSqe4UuxMUAR-vvXyaLPGn6W5zezWKJj6Dxk09QvqNgjlHRSodqT8yz-Pv4S8Ol5CI2PYIs~D3s9WQqV6Vh4wljMH-vJRlzfBYr5kqzk543O2ursYXIVAwVNI14vTHaFjDQ4w8opJVZPN5Tt5RHXT7Rk2QUmX~MdNyR0lLDTO4fFTdTQQvjigh5scAABli0ovyC62OjAtpPW6UwBsimeQX7Lf9a2Og9QRicDubNX-uvTvPmp5j8z0C-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122018&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the methylome of <em>Synechocystis</em> sp. PCC 6803. The methylome of <em>Synechocystis</em> sp. strain PCC 6803 comprises five different methylation motifs, which were detected using SMRT (Pacific Biosciences) and bisulfite sequencing. The genome plot shows the distribution of the recognition sequences on both strands of the chromosome and was made by using DNAPlotter.<a reveal-id="dsy006-B54" data-open="dsy006-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a></p></div></div><p>The Restriction Enzyme Database (REBASE<sup>®</sup><a reveal-id="dsy006-B38" data-open="dsy006-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a>) was searched for <em>Synechocystis</em> genes encoding putative DNA methyltransferases. In addition to <em>slr0214</em>, which encodes M.Ssp6803I modifying <sup>m5</sup>CGATCG,<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> the database analysis revealed five further ORFs with significant similarities to characterized methyltransferase genes, which could be responsible for the observed methylation pattern. In particular, these are <em>sll0729</em> and <em>slr1803</em>, which are present on the chromosome, and <em>slr6050</em>, <em>slr6095</em> and <em>sll8009</em> (see <a reveal-id="dsy006-T1" data-open="dsy006-T1" class="link link-reveal link-table xref-fig">Table 1</a> for an overview on methyltransferase nomenclature and corresponding genes), which are found on the plasmids pSYSX and pSYSG, respectively.</p>                            <h3 scrollto-destination=120122020 id="120122020" class="section-title" >3.2. The <em>Synechocystis</em> DNA methyltransferases</h3>
                            <h4 scrollto-destination=120122021 id="120122021" class="section-title" >3.2.1. The GG<sup>m4</sup>CC motif is methylated by M.Ssp6803II, which is encoded by <em>sll0729</em></h4>
<p>According to the presence of a Dam and a D12 class N6-adenine-specific DNA methyltransferase domain (COG0338 and pfam02086), the gene <em>sll0729</em> likely encodes an adenine-specific methyltransferase. However, phylogenetic analyses showed separate clustering of this methyltransferase from characterized adenine-specific methyltransferases of heterotrophic bacteria such as Dam of <em>E. coli</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Using the BlastP algorithm<a reveal-id="dsy006-B32" data-open="dsy006-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsy006-B33" data-open="dsy006-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and the NCBI nr database (January 2017), the closest homolog among cyanobacteria was found in the genome of <em>Fischerella</em> sp. JSC-11 (identity of 73%, BlastP e value of 9e<sup>−148</sup>), and it should be mentioned that 234 other proteins of high similarity exist in cyanobacteria (<em>e</em>-value ≤ 5e<sup>−15</sup>). The closest homolog beyond cyanobacteria exists in the genome of the archaeon <em>Methanosarcina mazei</em> (identity of 53%, BlastP <em>e</em> value of 9e<sup>−112</sup>) and the bacterial strain <em>Spirochaetes bacterium</em> GWB_1_27_13 (identity of 60%, BlastP <em>e</em> value of 9e<sup>−111</sup>), respectively.</p><p>To study the function of Sll0729, a mutant was generated in which the <em>aph</em>II gene, conferring Km resistance, was inserted into <em>sll0729</em>, leading to the deletion of a <em>Bcl</em>I-<em>Eco</em>RI fragment containing most of its coding sequence (<a reveal-id="dsy006-F2" data-open="dsy006-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). Genotypic analysis revealed that a completely segregated Δ<em>sll0729</em> mutant was obtained since only the fragment enlarged by the expected size of the inserted <em>aph</em>II gene was amplified by PCR, while a WT-sized fragment was not produced with Δ<em>sll0729</em> mutant DNA as template (<a reveal-id="dsy006-F2" data-open="dsy006-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). SMRT sequencing of the Δ<em>sll0729</em> mutant revealed a lack of GG<sup>m4</sup>CC methylation but no other differences compared to the WT DNA methylation status, indicating that <em>sll0729</em> encodes for a cytosine instead of an adenine-specific methyltransferase. Similarly, bisulfite sequencing showed no methylation of GGCC in the Δ<em>sll0729</em> mutant, all methylation signals with mutant DNA were below the threshold differentiating valid signals from background and experimental error (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S2B and C</a></span>). To verify this observation experimentally, chromosomal DNA from Δ<em>sll0729</em> mutant cells was incubated with various methylation-sensitive restriction enzymes to evaluate changes in the methylation pattern. Enzymes known to be unable to cut modified GG<sup>m4</sup>CC motifs, such as <em>Hae</em>III, <em>Eae</em>I, and <em>Apa</em>I, could cut mutant DNA but not WT DNA (<a reveal-id="dsy006-F2" data-open="dsy006-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). This result was supported by over-expression of <em>sll0729</em> in <em>E. coli</em>. Plasmid DNA from <em>E. coli</em> clones expressing <em>sll0729</em> were protected against the action of <em>Hae</em>III (<a reveal-id="dsy006-F2" data-open="dsy006-F2" class="link link-reveal link-table xref-fig">Fig. 2D</a>). Moreover, recombinant Sll0729 protein was purified by affinity chromatography via a fused His-tag and was found to catalyze significant DNA-specific methyltransferase activity in an <em>in vitro</em> enzyme assay (<a reveal-id="dsy006-F4" data-open="dsy006-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). </p>                        <a id="120122024" scrollto-destination="120122024"></a>
<div id="dsy006-F2" data-id="dsy006-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f2.png?Expires=2147483647&Signature=qDbYupwforFYYvZLzI1V-Mjp0m1Rwtn90oKtkdm~NlrKfzl649lOA3fynM7AQfvjNWvxlX~HqXkdHtiA-y8rLm3JYjC6CkDYqqyyXAX3~NyJ-vr2SdwMWm2~ofF2h9p9lnF8zm7C5UrBLJKaXcO45A3ieN7dTyVxMHKSUn72MOsHiiFJyp7Ocp7dH2P~nNBoU9FwejkUJKGfmAzbr-II6Wg-LeaaoPq80l7pVWkF1Oa7Ypjhyx0qnOYlAA3pbv0ZLoRd-OSHHwuir0DFj8kY7FeYt2XtcbcaHfjIaSnS2YcZ6ExC2WS-8k3b3PglUR-HhjlHGqk6DB6T3LpCbbZWig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional verification of M.Ssp6803II encoded by sll0729. (A) Construction strategy for the generation of the Δsll0729 mutant defective in the ORF of M.Ssp6803II. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Characterization of Δsll0729 mutant genotype by PCR using gene specific primers and as templates DNA from wild type (WT) and mutant cells (Mu). (C) Separation of fragments generated during a restriction analysis with GGCC-specific, methylation-sensitive restriction endonucleases (HaeIII, EaeI, and ApaI) and GATC-specific enzyme (Sau3A) of chromosomal DNA of the wild type (WT) and Δsll0729 mutant (Mu) by agarose gel electrophoresis. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by HindIII and EcoRI). Please note that GGCmC is cleaved by HaeIII, whereas GGmCC is resistant to cleavage (NEB). (D) Restriction analysis of plasmids, which were isolated from cells of the DNA-methylation negative E. coli strain TOP10 over-expressing M.Ssp6803II (Sll0729), using methylation-sensitive enzyme (HaeIII) and HindIII as control. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by HindIII and EcoRI)." path-from-xml="dsy006f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122024/dsy006f2.tif" path-from-xml="dsy006f2.tif" target="_blank">View large</a><a section="120122024" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f2.png?Expires=2147483647&Signature=IQx-iZXilQc4xObtRfEn1cJGFZtjI0dF5h7h55ObXtSgZ4B06~mdRRuh98QNmCBwdV6VyX9hcUtC7WPgehgRVIb~dtY05fCAM1y5ydJ-CfFVee2eSivrZ2UtgTtDPVJncYsTE-koSmMhIuzGMBqfiJCC7LRna9AOBIsSxUSA47PgQkjOaqX9xrjlNlblFeiuUN6JvfwDBFB2MiQUtad8XyvyZpfg9N3WH~u3po-O2WlTNEX~amWFI~eiRl4pofpZDnh0qjFbVo-O6PFnLZKWsu-3ELeJO24X0f1Bl5LVkIdziTS6P~kZI8~M8NjtOnwjiSEiHQ39Nd3vSGYPDgs9HQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122024&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional verification of M.Ssp6803II encoded by <em>sll0729</em>. (A) Construction strategy for the generation of the Δ<em>sll0729</em> mutant defective in the ORF of M.Ssp6803II. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Characterization of Δ<em>sll0729</em> mutant genotype by PCR using gene specific primers and as templates DNA from wild type (WT) and mutant cells (Mu). (C) Separation of fragments generated during a restriction analysis with GGCC-specific, methylation-sensitive restriction endonucleases (<em>Hae</em>III, <em>Eae</em>I, and <em>Apa</em>I) and GATC-specific enzyme (<em>Sau</em>3A) of chromosomal DNA of the wild type (WT) and Δ<em>sll0729</em> mutant (Mu) by agarose gel electrophoresis. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by <em>Hin</em>dIII and <em>Eco</em>RI). Please note that GGC<sup>m</sup>C is cleaved by <em>Hae</em>III, whereas GG<sup>m</sup>CC is resistant to cleavage (NEB). (D) Restriction analysis of plasmids, which were isolated from cells of the DNA-methylation negative <em>E. coli</em> strain TOP10 over-expressing M.Ssp6803II (Sll0729), using methylation-sensitive enzyme (<em>Hae</em>III) and <em>Hin</em>dIII as control. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by <em>Hin</em>dIII and <em>Eco</em>RI).</p></div></div><div content-id="dsy006-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f2.png?Expires=2147483647&Signature=qDbYupwforFYYvZLzI1V-Mjp0m1Rwtn90oKtkdm~NlrKfzl649lOA3fynM7AQfvjNWvxlX~HqXkdHtiA-y8rLm3JYjC6CkDYqqyyXAX3~NyJ-vr2SdwMWm2~ofF2h9p9lnF8zm7C5UrBLJKaXcO45A3ieN7dTyVxMHKSUn72MOsHiiFJyp7Ocp7dH2P~nNBoU9FwejkUJKGfmAzbr-II6Wg-LeaaoPq80l7pVWkF1Oa7Ypjhyx0qnOYlAA3pbv0ZLoRd-OSHHwuir0DFj8kY7FeYt2XtcbcaHfjIaSnS2YcZ6ExC2WS-8k3b3PglUR-HhjlHGqk6DB6T3LpCbbZWig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional verification of M.Ssp6803II encoded by sll0729. (A) Construction strategy for the generation of the Δsll0729 mutant defective in the ORF of M.Ssp6803II. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Characterization of Δsll0729 mutant genotype by PCR using gene specific primers and as templates DNA from wild type (WT) and mutant cells (Mu). (C) Separation of fragments generated during a restriction analysis with GGCC-specific, methylation-sensitive restriction endonucleases (HaeIII, EaeI, and ApaI) and GATC-specific enzyme (Sau3A) of chromosomal DNA of the wild type (WT) and Δsll0729 mutant (Mu) by agarose gel electrophoresis. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by HindIII and EcoRI). Please note that GGCmC is cleaved by HaeIII, whereas GGmCC is resistant to cleavage (NEB). (D) Restriction analysis of plasmids, which were isolated from cells of the DNA-methylation negative E. coli strain TOP10 over-expressing M.Ssp6803II (Sll0729), using methylation-sensitive enzyme (HaeIII) and HindIII as control. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by HindIII and EcoRI)." path-from-xml="dsy006f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122024/dsy006f2.tif" path-from-xml="dsy006f2.tif" target="_blank">View large</a><a section="120122024" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f2.png?Expires=2147483647&Signature=IQx-iZXilQc4xObtRfEn1cJGFZtjI0dF5h7h55ObXtSgZ4B06~mdRRuh98QNmCBwdV6VyX9hcUtC7WPgehgRVIb~dtY05fCAM1y5ydJ-CfFVee2eSivrZ2UtgTtDPVJncYsTE-koSmMhIuzGMBqfiJCC7LRna9AOBIsSxUSA47PgQkjOaqX9xrjlNlblFeiuUN6JvfwDBFB2MiQUtad8XyvyZpfg9N3WH~u3po-O2WlTNEX~amWFI~eiRl4pofpZDnh0qjFbVo-O6PFnLZKWsu-3ELeJO24X0f1Bl5LVkIdziTS6P~kZI8~M8NjtOnwjiSEiHQ39Nd3vSGYPDgs9HQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122024&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional verification of M.Ssp6803II encoded by <em>sll0729</em>. (A) Construction strategy for the generation of the Δ<em>sll0729</em> mutant defective in the ORF of M.Ssp6803II. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Characterization of Δ<em>sll0729</em> mutant genotype by PCR using gene specific primers and as templates DNA from wild type (WT) and mutant cells (Mu). (C) Separation of fragments generated during a restriction analysis with GGCC-specific, methylation-sensitive restriction endonucleases (<em>Hae</em>III, <em>Eae</em>I, and <em>Apa</em>I) and GATC-specific enzyme (<em>Sau</em>3A) of chromosomal DNA of the wild type (WT) and Δ<em>sll0729</em> mutant (Mu) by agarose gel electrophoresis. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by <em>Hin</em>dIII and <em>Eco</em>RI). Please note that GGC<sup>m</sup>C is cleaved by <em>Hae</em>III, whereas GG<sup>m</sup>CC is resistant to cleavage (NEB). (D) Restriction analysis of plasmids, which were isolated from cells of the DNA-methylation negative <em>E. coli</em> strain TOP10 over-expressing M.Ssp6803II (Sll0729), using methylation-sensitive enzyme (<em>Hae</em>III) and <em>Hin</em>dIII as control. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by <em>Hin</em>dIII and <em>Eco</em>RI).</p></div></div><p>Altogether, these results indicate that <em>sll0729</em> encodes a cytosine-specific DNA-methyltransferase responsible for modifying the core sequence 5′-GG<sup>m4</sup>CC-3′. Thus, following the established nomenclature of REBASE<sup>®</sup>, this DNA-specific <em>Synechocystis</em> methyltransferase was named M.Ssp6803II (<a reveal-id="dsy006-T1" data-open="dsy006-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>                            <h4 scrollto-destination=120122026 id="120122026" class="section-title" >3.2.2. The G<sup>m6</sup>ATC motif is methylated by M.Ssp6803III, which is encoded by <em>slr1803</em></h4>
<p>The protein sequence of the methyltransferase encoded by <em>slr1803</em> shows significant sequence similarities to Dam-like enzymes from many heterotrophic bacteria and forms one cluster with these enzymes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Furthermore, protein domain prediction at the NCBI Blast server showed that it also possesses a D12-class N6-adenine-specific DNA methyltransferase domain. The highest sequence similarity was found with the homolog from the cyanobacterium <em>Halothece</em> sp. PCC 7418 (identity of 57%, BlastP <em>e</em>-value of 1e<sup>−121</sup>). We found 290 highly similar proteins in other cyanobacteria (<em>e</em>-value ≤ 5e<sup>−15</sup>). In addition, closely related proteins also exist in other Bacteria such as <em>Chloroflexi bacterium</em> RBG_16_57_9 (identity of 58%, BlastP <em>e</em>-value of 5e<sup>−124</sup>) and in Archaea, such as <em>Methanolobus tindarius</em> DSM2278 (identity of 54%, BlastP <em>e</em>-value of 2e<sup>−112</sup>). Among biochemically characterized enzymes, M.MboI from <em>Moraxella bovis</em><a reveal-id="dsy006-B39" data-open="dsy006-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> was identified as the closest relative (identity of 41%, BlastP <em>e</em>-value of 9e<sup>−63</sup>). A closely related enzyme was also reported from the filamentous cyanobacterium <em>Anabaena</em> (<em>Nostoc</em>) sp. PCC 7120.<a reveal-id="dsy006-B18" data-open="dsy006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> These features strongly qualify Slr1803 as a candidate modifier of G<sup>m6</sup>ATC methylation motif, making <em>Synechocystis</em> DNA resistant against <em>Mbo</em>I restriction.</p><p>To verify the function of this putative N6-adenine methyltransferase, a deletion mutant of <em>slr1803</em> was generated; in this mutant, the internal <em>Hin</em>cII fragment of the ORF was replaced by an <em>aph</em>II gene (<a reveal-id="dsy006-F3" data-open="dsy006-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). In addition to the mutated fragment, which is enlarged by the expected size of the inserted <em>aph</em>II gene, PCR analysis also detected the WT-sized PCR fragment from Δ<em>slr1803</em> mutant DNA (<a reveal-id="dsy006-F3" data-open="dsy006-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The non-segregated status of the Δ<em>slr1803</em> mutant could not be improved by cultivation at higher Km concentrations for many generations. This result indicates that the <em>slr1803</em> gene is essential for the viability of <em>Synechocystis</em> under our laboratory conditions. Consistent with the non-segregated genotype of the Δ<em>slr1803</em> mutant, we observed no change in the methylation of the <em>Synechocystis</em> DNA, because <em>Mbo</em>I, which is Dam-methylation sensitive, did not cut DNA isolated from the mutant (<a reveal-id="dsy006-F3" data-open="dsy006-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). </p>                        <a id="120122029" scrollto-destination="120122029"></a>
<div id="dsy006-F3" data-id="dsy006-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f3.png?Expires=2147483647&Signature=0QVXZ88mfvh3vzt3OUx94rIjbZHTpfaGsSTq1vS8AlVtEHjllL4wiuiKvN4UzILNIIQyo~UqqeLlFhrnngC5nAQM5nSFvzwVINxal07velXdQcZle5Pn~W4tKMQqS5ceoAVKWcm1qzSMQJChw8xZfrgsEahSOa-GEhL3FHx3cjb4UPw7aiVikOamkshDwaIQ9VOR-qV0--Z2fhB9j43A~4aQE-9S3RAF1nD3p5X3GizS8u7keAXJ9i6wZ6TaGoSWr8TUaHPouTj8WSRwKrG8jKnm2v4l9wnqUV~IetBeYJDygMhxxy8dlos7vSO2dJ6WaPVVm0TdqFI-kj7LFmoxaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional verification of M.Ssp6803III encoded by slr1803. (A) Construction strategy for the generation of the Δslr1803 mutant defective in the ORF of M.Ssp6803III. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Characterization of Δslr1803 mutant genotype by PCR using gene-specific primers and as templates DNA from wild-type (WT) and mutant cells (Mu). (C) Separation of fragments generated during a restriction analysis with the N6-adenine methylation-sensitive restriction endonuclease (MboI) and enzymes (Sau3A, DpnI, and AvaI), which are not sensitive to N6-adenine methylation, of chromosomal DNA of the wild-type (WT) and Δslr1803 mutant (Mu) by agarose gel electrophoresis. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by HindIII and EcoRI). (D) Restriction analysis of plasmids, which were isolated from cells of the DNA-methylation negative E. coli strain TOP10 over-expressing M.Ssp6803III (Slr1803), using N6-adenine methylation-sensitive enzyme (MboI) and N6-adenine methylation-insensitive enzyme (Sau3A). (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by HindIII and EcoRI)." path-from-xml="dsy006f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122029/dsy006f3.tif" path-from-xml="dsy006f3.tif" target="_blank">View large</a><a section="120122029" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f3.png?Expires=2147483647&Signature=jNpkgBk7DfIfgYoXyym6V9iULIo36KWLHkVSkhM4rULUkK0YaRHgQ8Z3ev-OqtnfWsUKDpS9EjQqvJv4TqZT~Y6nxfzoG~TY9tUGLCaaNyHE3r7JDvnMDLKtXJJ5SALSKHM-MYTYeMihmJdyiolurs~Kb0GYlPIG5mnVPj4MBoYgtIXntGLaItMCtkeMaASybC8Z8dkW0DeuUVTIGRTLxRDlLR1QgrVgTvY6GYAMcse321m8~KOWqrp~vE5KFr5FE~i9ZdiBAfSbpNDYFYhyQQQUYQGxgmKw5reiOYk9OULQjpbLBQF1FCjgwA5On64cNA7qbTclGhsA5o-D2ga9Nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122029&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional verification of M.Ssp6803III encoded by <em>slr1803</em>. (A) Construction strategy for the generation of the Δ<em>slr1803</em> mutant defective in the ORF of M.Ssp6803III. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Characterization of Δ<em>slr1803</em> mutant genotype by PCR using gene-specific primers and as templates DNA from wild-type (WT) and mutant cells (Mu). (C) Separation of fragments generated during a restriction analysis with the N6-adenine methylation-sensitive restriction endonuclease (<em>Mbo</em>I) and enzymes (<em>Sau</em>3A, <em>Dpn</em>I, and <em>Ava</em>I), which are not sensitive to N6-adenine methylation, of chromosomal DNA of the wild-type (WT) and Δ<em>slr1803</em> mutant (Mu) by agarose gel electrophoresis. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by <em>Hin</em>dIII and <em>Eco</em>RI). (D) Restriction analysis of plasmids, which were isolated from cells of the DNA-methylation negative <em>E. coli</em> strain TOP10 over-expressing M.Ssp6803III (Slr1803), using N6-adenine methylation-sensitive enzyme (<em>Mbo</em>I) and N6-adenine methylation-insensitive enzyme (<em>Sau</em>3A). (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by <em>Hin</em>dIII and <em>Eco</em>RI).</p></div></div><div content-id="dsy006-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f3.png?Expires=2147483647&Signature=0QVXZ88mfvh3vzt3OUx94rIjbZHTpfaGsSTq1vS8AlVtEHjllL4wiuiKvN4UzILNIIQyo~UqqeLlFhrnngC5nAQM5nSFvzwVINxal07velXdQcZle5Pn~W4tKMQqS5ceoAVKWcm1qzSMQJChw8xZfrgsEahSOa-GEhL3FHx3cjb4UPw7aiVikOamkshDwaIQ9VOR-qV0--Z2fhB9j43A~4aQE-9S3RAF1nD3p5X3GizS8u7keAXJ9i6wZ6TaGoSWr8TUaHPouTj8WSRwKrG8jKnm2v4l9wnqUV~IetBeYJDygMhxxy8dlos7vSO2dJ6WaPVVm0TdqFI-kj7LFmoxaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional verification of M.Ssp6803III encoded by slr1803. (A) Construction strategy for the generation of the Δslr1803 mutant defective in the ORF of M.Ssp6803III. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Characterization of Δslr1803 mutant genotype by PCR using gene-specific primers and as templates DNA from wild-type (WT) and mutant cells (Mu). (C) Separation of fragments generated during a restriction analysis with the N6-adenine methylation-sensitive restriction endonuclease (MboI) and enzymes (Sau3A, DpnI, and AvaI), which are not sensitive to N6-adenine methylation, of chromosomal DNA of the wild-type (WT) and Δslr1803 mutant (Mu) by agarose gel electrophoresis. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by HindIII and EcoRI). (D) Restriction analysis of plasmids, which were isolated from cells of the DNA-methylation negative E. coli strain TOP10 over-expressing M.Ssp6803III (Slr1803), using N6-adenine methylation-sensitive enzyme (MboI) and N6-adenine methylation-insensitive enzyme (Sau3A). (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by HindIII and EcoRI)." path-from-xml="dsy006f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122029/dsy006f3.tif" path-from-xml="dsy006f3.tif" target="_blank">View large</a><a section="120122029" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f3.png?Expires=2147483647&Signature=jNpkgBk7DfIfgYoXyym6V9iULIo36KWLHkVSkhM4rULUkK0YaRHgQ8Z3ev-OqtnfWsUKDpS9EjQqvJv4TqZT~Y6nxfzoG~TY9tUGLCaaNyHE3r7JDvnMDLKtXJJ5SALSKHM-MYTYeMihmJdyiolurs~Kb0GYlPIG5mnVPj4MBoYgtIXntGLaItMCtkeMaASybC8Z8dkW0DeuUVTIGRTLxRDlLR1QgrVgTvY6GYAMcse321m8~KOWqrp~vE5KFr5FE~i9ZdiBAfSbpNDYFYhyQQQUYQGxgmKw5reiOYk9OULQjpbLBQF1FCjgwA5On64cNA7qbTclGhsA5o-D2ga9Nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122029&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional verification of M.Ssp6803III encoded by <em>slr1803</em>. (A) Construction strategy for the generation of the Δ<em>slr1803</em> mutant defective in the ORF of M.Ssp6803III. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Characterization of Δ<em>slr1803</em> mutant genotype by PCR using gene-specific primers and as templates DNA from wild-type (WT) and mutant cells (Mu). (C) Separation of fragments generated during a restriction analysis with the N6-adenine methylation-sensitive restriction endonuclease (<em>Mbo</em>I) and enzymes (<em>Sau</em>3A, <em>Dpn</em>I, and <em>Ava</em>I), which are not sensitive to N6-adenine methylation, of chromosomal DNA of the wild-type (WT) and Δ<em>slr1803</em> mutant (Mu) by agarose gel electrophoresis. (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by <em>Hin</em>dIII and <em>Eco</em>RI). (D) Restriction analysis of plasmids, which were isolated from cells of the DNA-methylation negative <em>E. coli</em> strain TOP10 over-expressing M.Ssp6803III (Slr1803), using N6-adenine methylation-sensitive enzyme (<em>Mbo</em>I) and N6-adenine methylation-insensitive enzyme (<em>Sau</em>3A). (n.c., uncut control DNA; M, DNA size marker λ-DNA cut by <em>Hin</em>dIII and <em>Eco</em>RI).</p></div></div><p>Since the methylation specificity of Slr1803 could not be verified using the Δ<em>slr1803</em> mutant, the ORF was overexpressed in <em>E. coli</em>. Small amounts of recombinant protein of the expected size were found in crude extracts and could be isolated via the fused His-tag. Plasmid DNA from <em>E. coli</em> clones expressing <em>slr1803</em> was protected against the action of <em>Mbo</em>I (<a reveal-id="dsy006-F3" data-open="dsy006-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>) but could be still cut with <em>Sau</em>3A, which is not affected by N6-adenine methylation. Purified recombinant Slr1803 protein also showed significant DNA-specific methyltransferase activity in an <em>in vitro</em> enzyme assay (<a reveal-id="dsy006-F4" data-open="dsy006-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). </p>                        <a id="120122031" scrollto-destination="120122031"></a>
<div id="dsy006-F4" data-id="dsy006-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f4.png?Expires=2147483647&Signature=Ld-E-NHt6v-uQp1WZEbLIPzRQu0veZCnpW4h~bWJeRU868bkUkIogDYfb6Dk0tGMDFgtsmqPcqqClPzLkncbwUXQnVJn9l1bu9mcWn0y04pFvdG-M9yH20ubOJmF9rJIjKboEzQKkDMypIO6ZnUAVLs01EGZ2OMWwTOXCC5vVbQ5anp6tvd6v-1uelnE-AY5ErBaPOk~FY2qbJ37RnYSX4ZCSKStTMPUpxyGu~dUI2dtAdpskUyMh3P-EWUP41XOt2-TFx9dv4zBxwlm-x5t1mlPjI7d-n0Cs7IsmNw7eF7tv-aqUPwyRoxlWhBg6j3klj9X6WFhrDwVgnOMt7XC4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="In vitro methylation activities of M.Ssp6803II and M.Ssp6803III of Synechocystis sp. PCC 6803. Total DNA-specific methyltransferase activities towards non-methylated Micrococcus DNA was estimated with purified recombinant M.Ssp6803II (Sll0729) and M.Ssp6803III (Slr1803) using the NEB4-restriction buffer (C, control; incubation of DNA without added recombinant protein)." path-from-xml="dsy006f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122031/dsy006f4.tif" path-from-xml="dsy006f4.tif" target="_blank">View large</a><a section="120122031" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f4.png?Expires=2147483647&Signature=fJC7PZkvCyHgSp5liU7hDqQCXLopGtIdVHVTTUGC4OE~9Lln6ZBn8B885c~fxKntFtQFsX-kgj2Z0~tTx6UFr2BOhHdrELQRy9IN7llOgC6ARlqk1cVzL-nPGo~Oc0jemI9wvj7KlSefIGp2c~g~O8FEtBZcvvgIaGHm3NX2h-Od665ZQDyGMBISAthZGgUEPlo8eboV5qt3JF20x4Z0hV~2CLe~N13aMmH0FweqZ4yP2iOZsM6rcrFv9JR1pXRq71pe2ERTUoAHCG6z--Ld32w7zFlFiHfCpvA~YcfO7lf2stjId1xVAOvYBDrLkZvylJGplMSJmpcqJSujJdCzSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122031&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>In vitro</em> methylation activities of M.Ssp6803II and M.Ssp6803III of <em>Synechocystis</em> sp. PCC 6803. Total DNA-specific methyltransferase activities towards non-methylated <em>Micrococcus</em> DNA was estimated with purified recombinant M.Ssp6803II (Sll0729) and M.Ssp6803III (Slr1803) using the NEB4-restriction buffer (C, control; incubation of DNA without added recombinant protein).</p></div></div><div content-id="dsy006-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f4.png?Expires=2147483647&Signature=Ld-E-NHt6v-uQp1WZEbLIPzRQu0veZCnpW4h~bWJeRU868bkUkIogDYfb6Dk0tGMDFgtsmqPcqqClPzLkncbwUXQnVJn9l1bu9mcWn0y04pFvdG-M9yH20ubOJmF9rJIjKboEzQKkDMypIO6ZnUAVLs01EGZ2OMWwTOXCC5vVbQ5anp6tvd6v-1uelnE-AY5ErBaPOk~FY2qbJ37RnYSX4ZCSKStTMPUpxyGu~dUI2dtAdpskUyMh3P-EWUP41XOt2-TFx9dv4zBxwlm-x5t1mlPjI7d-n0Cs7IsmNw7eF7tv-aqUPwyRoxlWhBg6j3klj9X6WFhrDwVgnOMt7XC4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="In vitro methylation activities of M.Ssp6803II and M.Ssp6803III of Synechocystis sp. PCC 6803. Total DNA-specific methyltransferase activities towards non-methylated Micrococcus DNA was estimated with purified recombinant M.Ssp6803II (Sll0729) and M.Ssp6803III (Slr1803) using the NEB4-restriction buffer (C, control; incubation of DNA without added recombinant protein)." path-from-xml="dsy006f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122031/dsy006f4.tif" path-from-xml="dsy006f4.tif" target="_blank">View large</a><a section="120122031" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f4.png?Expires=2147483647&Signature=fJC7PZkvCyHgSp5liU7hDqQCXLopGtIdVHVTTUGC4OE~9Lln6ZBn8B885c~fxKntFtQFsX-kgj2Z0~tTx6UFr2BOhHdrELQRy9IN7llOgC6ARlqk1cVzL-nPGo~Oc0jemI9wvj7KlSefIGp2c~g~O8FEtBZcvvgIaGHm3NX2h-Od665ZQDyGMBISAthZGgUEPlo8eboV5qt3JF20x4Z0hV~2CLe~N13aMmH0FweqZ4yP2iOZsM6rcrFv9JR1pXRq71pe2ERTUoAHCG6z--Ld32w7zFlFiHfCpvA~YcfO7lf2stjId1xVAOvYBDrLkZvylJGplMSJmpcqJSujJdCzSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122031&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>In vitro</em> methylation activities of M.Ssp6803II and M.Ssp6803III of <em>Synechocystis</em> sp. PCC 6803. Total DNA-specific methyltransferase activities towards non-methylated <em>Micrococcus</em> DNA was estimated with purified recombinant M.Ssp6803II (Sll0729) and M.Ssp6803III (Slr1803) using the NEB4-restriction buffer (C, control; incubation of DNA without added recombinant protein).</p></div></div><p>These results clearly indicate that <em>slr1803</em> encodes an N6-adenine-specific DNA methyltransferase modifying the core sequence 5′-G<sup>m6</sup>ATC-3′. Thus, this <em>Synechocystis</em> DNA-specific methyltransferase was named M.Ssp6803III (<a reveal-id="dsy006-T1" data-open="dsy006-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>                            <h4 scrollto-destination=120122033 id="120122033" class="section-title" >3.2.3. Two DNA methyltransferases are localized on the plasmid pSYSX</h4>
<p>The methylome of <em>Synechocystis</em> includes two additional methylation motifs, GA<sup>m6</sup>AGGC and GG<sup>m6</sup>AN<sub>7</sub>TTGG/CCA<sup>m6</sup>AN<sub>7</sub>TCC that are not modified by the methyltransferases encoded on the chromosome. BlastP searches using proteins encoded on the <em>Synechocystis</em> plasmids revealed three additional genes for putative DNA-specific methyltransferases.</p><p>On the pSYSX plasmid, we found <em>slr6050</em> annotated to encode a hypothetical protein of 1100 amino acids. BlastP searches revealed that very similar proteins exist in many different bacteria but only in four other cyanobacteria (<em>Microcystis aeruginosa</em> spp. PCC 9701 and 9806, <em>Synechococcus</em> sp. PCC 73109, and <em>Prochlorothrix hollandica</em>). To verify the function of this putative N6-adenine methyltransferase, a deletion mutant of <em>slr6050</em> was generated; in this mutant, the internal <em>Cla</em>I fragment of the ORF was replaced by an <em>aph</em>II gene (<a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). Since the deleted <em>slr6050</em> fragment had approximately the same size as the inserted <em>aphII</em> gene, the PCR reaction using flanking primers (slr6050fw and slr6050rev, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) did not allow to judge whether or not the mutant was fully segregated. Alternatively, we used a primer pair (slr6050_i_fw and slr6050_i_rev, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), which binds to sequences located inside the deleted <em>Cla</em>I fragment. This PCR-detected DNA of same size using DNA from WT as well as the <em>slr6050</em> mutant (<a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>), which clearly indicated that the mutant was not fully segregated. The <em>aphII</em> gene was detected with primers aphII_fw and aphII_rev only with DNA of the mutant (<a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). The non-segregated status of the Δ<em>slr6050</em> mutant could not be improved by cultivation at higher Km concentrations for many generations. This result indicates that the <em>slr6050</em> gene is essential for the viability of <em>Synechocystis</em> under our laboratory conditions. </p>                        <a id="120122036" scrollto-destination="120122036"></a>
<div id="dsy006-F5" data-id="dsy006-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f5.png?Expires=2147483647&Signature=kJZbnu6hbSRxPG8hsvlOS5CfrXeikaIB99g2agQ17GbJn4CYmNynQm-pV-6fz2SuDklF0SqGz8ZoVuXlZkRh8toTYBD9QkKNoLGYG5BjvkoHUKE4SFzA6MBAh5DO2hauxux47DVJXrCKWhg5rLORppYMfttqtnu33OYQ5ycj0LWzvt9EaqxmP47twPAnQv4UkGtMa4o4yCmaPXTB1xoombYvogvqeqU8yx2UDKdRWS3Dkdsc~EguYuSPsu9xDSYA-eWDjlT3lQaBZtrH1VJuazB37gEvUhJcI72CEIAh9TnbXd-Kx8Qx6VIaJ8fNFqLgNCGh4EpctWjEL7n~n6a0yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mutation of plasmid-located methyltransferase genes. (A) Construction strategy for the generation of the Δslr6050 mutant defective in the ORF of M.Ssp6803IV. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Separation of PCR fragments using primers slr6050_i_fw and Slr6050_i_rev and DNA from wild-type (WT) or mutant cells (Mu) verifying the non-segregated genotype. (C) Separation of PCR fragments using primers aphIIfw and aphIIrev verifying the insertion of the Km-resistance cartridge. (D) Construction strategy for the generation of the Δslr6095 mutant defective in the ORF of M.Ssp6803V. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (E) Characterization of the fully segregated Δslr6095 mutant genotype by PCR using gene-specific primers and as template DNA from WT or Mu cells. (F) Construction strategy for the generation of the Δsll8009 mutant. Thin arrows indicate primer binding sites used for genotype verification. (G) Characterization of the Δsll8009 mutant genotype by PCR using gene-specific primers and WT or Mu cell template DNA." path-from-xml="dsy006f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122036/dsy006f5.tif" path-from-xml="dsy006f5.tif" target="_blank">View large</a><a section="120122036" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f5.png?Expires=2147483647&Signature=B05cw73AVWEbkrBTnWkp67-i8WuhaUlcfJqgd1bsYYa4fRGD6xahGnfdCmruhgGSy7NzM5lbfrkeeZfN--5y05yeWrE~I9Hmoq3rf5gbs8hHQaACBNOoXPas1MTeqivwo4t7NO8eY-Uu~SvIMwF2MIK~dOMPtjneWjXncS1k4d-oIGYW0mDaUV1yq~u2SNXq5exax0eSDJ4Ujn6gOgA71u0~zROqv6VprkMOw3ja2AP~7CfiEfYgStsSm4zKgnDL9Ts2hvZphrGxnj4OJb7Xw1JUyfklDMfhqyj0jEnx4cRbR2dcqLEJcj-MoOMmmNe-5CowxTiimeBv69Cxv4bcIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122036&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mutation of plasmid-located methyltransferase genes. (A) Construction strategy for the generation of the Δ<em>slr6050</em> mutant defective in the ORF of M.Ssp6803IV. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Separation of PCR fragments using primers slr6050_i_fw and Slr6050_i_rev and DNA from wild-type (WT) or mutant cells (Mu) verifying the non-segregated genotype. (C) Separation of PCR fragments using primers aphIIfw and aphIIrev verifying the insertion of the Km-resistance cartridge. (D) Construction strategy for the generation of the Δ<em>slr6095</em> mutant defective in the ORF of M.Ssp6803V. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (E) Characterization of the fully segregated Δ<em>slr6095</em> mutant genotype by PCR using gene-specific primers and as template DNA from WT or Mu cells. (F) Construction strategy for the generation of the Δ<em>sll8009</em> mutant. Thin arrows indicate primer binding sites used for genotype verification. (G) Characterization of the Δ<em>sll8009</em> mutant genotype by PCR using gene-specific primers and WT or Mu cell template DNA.</p></div></div><div content-id="dsy006-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f5.png?Expires=2147483647&Signature=kJZbnu6hbSRxPG8hsvlOS5CfrXeikaIB99g2agQ17GbJn4CYmNynQm-pV-6fz2SuDklF0SqGz8ZoVuXlZkRh8toTYBD9QkKNoLGYG5BjvkoHUKE4SFzA6MBAh5DO2hauxux47DVJXrCKWhg5rLORppYMfttqtnu33OYQ5ycj0LWzvt9EaqxmP47twPAnQv4UkGtMa4o4yCmaPXTB1xoombYvogvqeqU8yx2UDKdRWS3Dkdsc~EguYuSPsu9xDSYA-eWDjlT3lQaBZtrH1VJuazB37gEvUhJcI72CEIAh9TnbXd-Kx8Qx6VIaJ8fNFqLgNCGh4EpctWjEL7n~n6a0yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mutation of plasmid-located methyltransferase genes. (A) Construction strategy for the generation of the Δslr6050 mutant defective in the ORF of M.Ssp6803IV. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Separation of PCR fragments using primers slr6050_i_fw and Slr6050_i_rev and DNA from wild-type (WT) or mutant cells (Mu) verifying the non-segregated genotype. (C) Separation of PCR fragments using primers aphIIfw and aphIIrev verifying the insertion of the Km-resistance cartridge. (D) Construction strategy for the generation of the Δslr6095 mutant defective in the ORF of M.Ssp6803V. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (E) Characterization of the fully segregated Δslr6095 mutant genotype by PCR using gene-specific primers and as template DNA from WT or Mu cells. (F) Construction strategy for the generation of the Δsll8009 mutant. Thin arrows indicate primer binding sites used for genotype verification. (G) Characterization of the Δsll8009 mutant genotype by PCR using gene-specific primers and WT or Mu cell template DNA." path-from-xml="dsy006f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122036/dsy006f5.tif" path-from-xml="dsy006f5.tif" target="_blank">View large</a><a section="120122036" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f5.png?Expires=2147483647&Signature=B05cw73AVWEbkrBTnWkp67-i8WuhaUlcfJqgd1bsYYa4fRGD6xahGnfdCmruhgGSy7NzM5lbfrkeeZfN--5y05yeWrE~I9Hmoq3rf5gbs8hHQaACBNOoXPas1MTeqivwo4t7NO8eY-Uu~SvIMwF2MIK~dOMPtjneWjXncS1k4d-oIGYW0mDaUV1yq~u2SNXq5exax0eSDJ4Ujn6gOgA71u0~zROqv6VprkMOw3ja2AP~7CfiEfYgStsSm4zKgnDL9Ts2hvZphrGxnj4OJb7Xw1JUyfklDMfhqyj0jEnx4cRbR2dcqLEJcj-MoOMmmNe-5CowxTiimeBv69Cxv4bcIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122036&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mutation of plasmid-located methyltransferase genes. (A) Construction strategy for the generation of the Δ<em>slr6050</em> mutant defective in the ORF of M.Ssp6803IV. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (B) Separation of PCR fragments using primers slr6050_i_fw and Slr6050_i_rev and DNA from wild-type (WT) or mutant cells (Mu) verifying the non-segregated genotype. (C) Separation of PCR fragments using primers aphIIfw and aphIIrev verifying the insertion of the Km-resistance cartridge. (D) Construction strategy for the generation of the Δ<em>slr6095</em> mutant defective in the ORF of M.Ssp6803V. Thin arrows indicate primer binding sites, which were used to verify the genotype of the mutants. (E) Characterization of the fully segregated Δ<em>slr6095</em> mutant genotype by PCR using gene-specific primers and as template DNA from WT or Mu cells. (F) Construction strategy for the generation of the Δ<em>sll8009</em> mutant. Thin arrows indicate primer binding sites used for genotype verification. (G) Characterization of the Δ<em>sll8009</em> mutant genotype by PCR using gene-specific primers and WT or Mu cell template DNA.</p></div></div><p>Comparison of Slr6050 against the Pfam database returned a single hit, the Eco57I bifunctional RM methylase, a type IV RM enzyme.<a reveal-id="dsy006-B40" data-open="dsy006-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> The Eco57I domain for AdoMet-dependent enzymes (pfam07669) is clearly present in the sequence of Slr6050 and weak similarity to the HSDR_N restriction endonuclease domain could also be detected. Eco57I is sensitive to the methylation of CTGA<sup>m6</sup>AG or CTTC<sup>m6</sup>AG<a reveal-id="dsy006-B40" data-open="dsy006-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> making Slr6050 a likely candidate for the modification of GA<sup>m6</sup>AGGC, which was detected by SMRT sequencing of <em>Synechocystis</em> DNA (<a reveal-id="dsy006-F1" data-open="dsy006-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Unfortunately, all our attempts to obtain recombinant Slr6050 protein after expression of a codon-optimized <em>slr6050</em> gene in <em>E. coli</em> failed; hence, we could not directly verify its function as DNA-specific methyltransferase and the proposed specificity.</p><p>Therefore, we only tentatively annotate this putative DNA-specific methyltransferase to be responsible for the modification of GA<sup>m6</sup>AGGC site of <em>Synechocystis</em> and name it M.Ssp6803IV (<a reveal-id="dsy006-T1" data-open="dsy006-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p><p>Another methyltransferase candidate was found on plasmid pSYSX (<em>slr6095</em>), which shows similarities to DNA-specific methyltransferases of type I RM systems. Thus, this enzyme could be responsible for the modification of the remaining motif GG<sup>m6</sup>AN<sub>7</sub>TTGG/CCA<sup>m6</sup>AN<sub>7</sub>TCC. BlastP searches against the NCBI database revealed that 487 highly similar proteins are harboured in other cyanobacteria (<em>e-</em>value ≤5e<sup>−15</sup>). To study the function of Slr6095, a mutant was generated in which the <em>aph</em>II gene, conferring Km resistance, was inserted into the coding sequence of <em>slr6095</em> at the single <em>Acc</em>III site (<a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). Genotypic analysis revealed that a completely segregated Δ<em>slr6095</em> mutant was obtained since only the fragment enlarged by the expected size of the inserted <em>aph</em>II gene was amplified by PCR, while a WT-sized fragment was not produced with Δ<em>slr6095</em> mutant DNA as template (<a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">Fig. 5E</a>). SMRT sequencing of the Δ<em>slr6095</em> mutant revealed a lack of GG<sup>m6</sup>AN<sub>7</sub>TTGG/CCA<sup>m6</sup>AN<sub>7</sub>TCC methylation but no other differences compared to the WT DNA methylation status, indicating that <em>slr6095</em> encodes the corresponding type I adenine-specific methyltransferase. Accordingly, this putative DNA-specific methyltransferase of <em>Synechocystis</em> was named M.Ssp6803V (<a reveal-id="dsy006-T1" data-open="dsy006-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>                            <h4 scrollto-destination=120122040 id="120122040" class="section-title" >3.2.4. The DNA methyltransferases on plasmid pSYSG is not active</h4>
<p>Finally, the genes <em>sll8009</em> (M subunit), <em>ssl8010</em>, <em>sll8006</em> (S subunit), and <em>sll8049</em> (R subunit) on the pSYSG plasmid are annotated in CyanoBase<a reveal-id="dsy006-B41" data-open="dsy006-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> to encode all subunits required for a complete RM system. Similar proteins are encoded in about 50 other cyanobacterial genomes. However, after protein sequence comparisons we noticed that <em>sll8009</em> appears to encode a methyltransferase that is N-terminally truncated by 156 amino acid residues. The missing sequence is present in <em>ssl8010</em> and the sequence between <em>ssl8010</em> and <em>sll8009.</em> Because both genes are in the same reading frame, a point mutation leading to a TAA stop codon might have led to its inactivation, consistent with a frameshift mutation in <em>sll8049</em>. The entire region comprising <em>sll8009</em> and <em>ssl8010</em> was amplified and re-sequenced. This analysis confirmed the DNA sequence displayed in CyanoBase<a reveal-id="dsy006-B41" data-open="dsy006-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and the truncated nature of Sll8009. Nevertheless, to study the function of the putative methyltransferase Sll8009, a mutant was generated in which the <em>aph</em>II gene, conferring Km resistance, was inserted into a deletion of an internal <em>Nhe</em>I fragment of the coding sequence of <em>sll8009</em> (<a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">Fig. 5F</a>). Genotypic analysis revealed that a completely segregated Δ<em>sll8009</em> mutant was obtained since only the fragment enlarged by the expected size of the inserted <em>aph</em>II gene was amplified by PCR, while a WT-sized fragment was not produced with Δ<em>sll8009</em> mutant DNA as template (<a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">Fig. 5G</a>). However, SMRT sequencing of the Δ<em>sll8009</em> mutant revealed no differences compared to the WT DNA methylation status, indicating that <em>sll8009</em> is not involved in the methylation of <em>Synechocystis</em> DNA, because it most likely encodes an enzymatically non-active methyltransferase.</p>                            <h3 scrollto-destination=120122042 id="120122042" class="section-title" >3.3. Physiological characterization of mutants defective in DNA methyltransferases</h3>
<p>To gain insight into possible physiological functions of DNA methylation in <em>Synechocystis</em>, mutants defective in these methyltransferase-encoding genes were studied. No clear phenotypical alterations, in comparison to WT, were observed for the Δ<em>slr0214</em>, Δ<em>slr6095</em> and the partially segregated Δ<em>slr1803</em> and Δ<em>slr6050</em> mutants. Because completely segregated Δ<em>slr1803</em> and Δ<em>slr6050</em> mutants could not be obtained (<a reveal-id="dsy006-F3" data-open="dsy006-F3" class="link link-reveal link-table xref-fig">Figs 3B</a> and <a reveal-id="dsy006-F5" data-open="dsy006-F5" class="link link-reveal link-table xref-fig">5</a>), M.Ssp6803III and M.Ssp6803IV play essential roles for cell viability. Interestingly, cultivation under identical conditions indicated a very severe growth deficiency of mutant Δ<em>sll0729</em>, whereas the mutant Δ<em>slr0214</em> grew like WT under these conditions. Moreover, a bluish appearance due to changed pigment composition was characteristic for this mutant compared to WT and mutant Δ<em>slr0214</em> over the entire cultivation time (<a reveal-id="dsy006-F6" data-open="dsy006-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Whole-cell absorbance spectra clearly indicated that the Δ<em>sll0729</em> cells contained a reduced amount of chlorophyll a (represented by the peaks at 440 and 680 nm), while the content of phycocyanin (represented by the peak at 630 nm) was not changed (<a reveal-id="dsy006-F6" data-open="dsy006-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a>). The decreased chlorophyll content most probably reflects reduced photosynthetic capacity, which corresponds to the diminished growth of mutant Δ<em>sll0729.</em></p>                        <a id="120122044" scrollto-destination="120122044"></a>
<div id="dsy006-F6" data-id="dsy006-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f6.png?Expires=2147483647&Signature=5J4OhgBX3j4hyBHLZ3SG9zxqwAUJ1gpUEeij9pQ-e6VD052n9OliMgSWQQNRRia369Xr6XT6R0mj56FG48MJgQtuQvBiB3aThbpRS4Hw~RpaYtLc-hTXVEaHxRZFKVaR8jnNwqdm8A4I7nmI1v2zb24P1OOcXo~kZKmru8PYpexxEi2WZ52EhyM-PbuFTUgzp8umuMXWxzu-dTXCmlq6jhsrCTTtSgrBf80HVLNKoNLqqrz2yPnKKZF2nKquX65gOS0kMjibjzFVh9Ux5-nayOfLpRvwFZPQsltSGrY2Bpc3pSCKWd63afxzVJkBQ8EEF1DHO6a-qAQHCrHsYOwMoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phenotype profiling of mutants Δslr0214 and Δsll0729 defective in M.Ssp6803I and M.Ssp6803II. (A) Growth curves of the WT and the mutants Δslr0214 and Δsll0729. (B) Phenotypic appearance of liquid cultures. (C) Whole cell absorbance spectra of the WT and the mutant Δsll0729." path-from-xml="dsy006f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122044/dsy006f6.tif" path-from-xml="dsy006f6.tif" target="_blank">View large</a><a section="120122044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f6.png?Expires=2147483647&Signature=tnT02GKTVcvWBQONCw~mMOIUWqA2DmTxXVr8ycqhF0p1uo~GDCDFCQHrZQvDSNHVIwtbxOlfW8DCgYOQB2PfQeoSH1WcTM81VZceVeTzGsckRhFp~04ZKSaxj36guAMxyXZaXBYt0I8EVilu7ChzIGyAd6iEC4SCIj-quD1zBUIHFy~n25mFEy6d8KDlNhbS1zywft1BWJokwWxlO~SdNkR58lXHqClWRFftcN5OR5Rl3b5gCRvVWeA09-9v1P9rzgkskv-L5azCBYYL6ivUFd40R4vT9q1Jzzcirozev0pvvyO21Vx2ZifYfp5bxnbPnndeR~gNq0T~s3kTsxnYiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122044&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phenotype profiling of mutants Δ<em>slr0214</em> and Δ<em>sll0729</em> defective in M.Ssp6803I and M.Ssp6803II. (A) Growth curves of the WT and the mutants Δ<em>slr0214</em> and Δ<em>sll0729</em>. (B) Phenotypic appearance of liquid cultures. (C) Whole cell absorbance spectra of the WT and the mutant Δ<em>sll0729</em>.</p></div></div><div content-id="dsy006-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/m_dsy006f6.png?Expires=2147483647&Signature=5J4OhgBX3j4hyBHLZ3SG9zxqwAUJ1gpUEeij9pQ-e6VD052n9OliMgSWQQNRRia369Xr6XT6R0mj56FG48MJgQtuQvBiB3aThbpRS4Hw~RpaYtLc-hTXVEaHxRZFKVaR8jnNwqdm8A4I7nmI1v2zb24P1OOcXo~kZKmru8PYpexxEi2WZ52EhyM-PbuFTUgzp8umuMXWxzu-dTXCmlq6jhsrCTTtSgrBf80HVLNKoNLqqrz2yPnKKZF2nKquX65gOS0kMjibjzFVh9Ux5-nayOfLpRvwFZPQsltSGrY2Bpc3pSCKWd63afxzVJkBQ8EEF1DHO6a-qAQHCrHsYOwMoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phenotype profiling of mutants Δslr0214 and Δsll0729 defective in M.Ssp6803I and M.Ssp6803II. (A) Growth curves of the WT and the mutants Δslr0214 and Δsll0729. (B) Phenotypic appearance of liquid cultures. (C) Whole cell absorbance spectra of the WT and the mutant Δsll0729." path-from-xml="dsy006f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122044/dsy006f6.tif" path-from-xml="dsy006f6.tif" target="_blank">View large</a><a section="120122044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006f6.png?Expires=2147483647&Signature=tnT02GKTVcvWBQONCw~mMOIUWqA2DmTxXVr8ycqhF0p1uo~GDCDFCQHrZQvDSNHVIwtbxOlfW8DCgYOQB2PfQeoSH1WcTM81VZceVeTzGsckRhFp~04ZKSaxj36guAMxyXZaXBYt0I8EVilu7ChzIGyAd6iEC4SCIj-quD1zBUIHFy~n25mFEy6d8KDlNhbS1zywft1BWJokwWxlO~SdNkR58lXHqClWRFftcN5OR5Rl3b5gCRvVWeA09-9v1P9rzgkskv-L5azCBYYL6ivUFd40R4vT9q1Jzzcirozev0pvvyO21Vx2ZifYfp5bxnbPnndeR~gNq0T~s3kTsxnYiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122044&ar=4850982&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy006f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phenotype profiling of mutants Δ<em>slr0214</em> and Δ<em>sll0729</em> defective in M.Ssp6803I and M.Ssp6803II. (A) Growth curves of the WT and the mutants Δ<em>slr0214</em> and Δ<em>sll0729</em>. (B) Phenotypic appearance of liquid cultures. (C) Whole cell absorbance spectra of the WT and the mutant Δ<em>sll0729</em>.</p></div></div><p>According to transcriptomic data available for <em>Synechocystis</em>,<a reveal-id="dsy006-B42" data-open="dsy006-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> the <em>sll0729</em> gene potentially comprises an operon with two adjacent genes (<a reveal-id="dsy006-F2" data-open="dsy006-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), the upstream located <em>sll0728</em> (<em>accA</em>) gene encoding the acetyl-CoA carboxylase alpha subunit and the downstream-located <em>ssl1378</em> gene encoding a small hypothetical protein. To rule out polar effects on the expression of the downstream gene, we generated complementation strains in which <em>sll0729</em> or <em>ssl1378</em> were ectopically expressed. The expression of intact <em>sll0729</em> fully reversed the phenotype back to WT-like growth and pigmentation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), whereas Δ<em>sll0729</em> mutant cells expressing <em>ssl1378</em> did not change the phenotype compared to the original Δ<em>sll0729</em> mutant. These experiments clearly show that defects in the DNA-specific methyltransferase M.Ssp6803II encoded by <em>sll0729</em> result in strong physiological defects; hence, this enzyme seems to play an important role in <em>Synechocystis</em>.</p>                            <h2 scrollto-destination=120122046 id="120122046" class="section-title" >4. Discussion</h2>
<p><em>Synechocystis</em> possesses at least five different methylation activities toward specific DNA sequences, which were detected using SMRT and bisulfite sequencing. The three DNA methyltransferases encoded on the chromosome seem to belong to the type II methyltransferase group since they modify bases in short palindromic sequences (M.Ssp6803I-III), whereas M.Ssp6803V is a type I enzyme that modifies a larger bi-partite motif (see <a reveal-id="dsy006-F1" data-open="dsy006-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). The M.Ssp6803IV is not modifying a palindromic sequence and, based on its similarity to Eco571 RM enzymes, qualifies as a type IV enzyme.<a reveal-id="dsy006-B40" data-open="dsy006-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> The occurrence of five methylated motifs and five methyltransferase-encoding genes is similar to other bacteria. A recent study of the epigenetic landscape of prokaryotes revealed that only a few genomes are not methylated, whereas others contain multiple different motifs, up to 19.<a reveal-id="dsy006-B16" data-open="dsy006-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> This study included the cyanobacteria <em>Leptolyngbya</em> sp. PCC 6406 and <em>Mastigocladopsis repens</em> PCC 10914, with 12 and 2 modified motifs, respectively. A survey of palindromic sequences and their putative modifying methyltransferases identified several types among cyanobacteria.<a reveal-id="dsy006-B43" data-open="dsy006-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Particularly widespread among cyanobacteria are the palindromic sequences GCGATCGC (highly iterated palindrome 1, HIP1),<a reveal-id="dsy006-B44" data-open="dsy006-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsy006-B45" data-open="dsy006-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> GGCC, and GATC. These three sequences contain the motifs for the DNA methyltransferases M.Ssp6803I-III, which are encoded on the chromosome of <em>Synechocystis</em> (this work and [<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr">20</a>]), as well as related enzymes in <em>Anabaena</em> (<em>Nostoc</em>) sp. PCC 7120.<a reveal-id="dsy006-B43" data-open="dsy006-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> It is very likely that the proteins of high similarities encoded in many other cyanobacterial genomes show identical methylation specificities.</p><p>The previously identified C5-cytosine-specific enzyme M.Ssp6803I modifies the core motif within the HIP1 sequences.<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Despite the frequent occurrence of methylated HIP1 sequences, their methylation by M.Ssp6803I seems to have no significant impact on the physiology of <em>Synechocystis</em> under laboratory conditions, because in previous work only slight growth retardation was observed for the corresponding <em>Synechocystis</em> mutant<a reveal-id="dsy006-B20" data-open="dsy006-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and the mutant defective in the ortholog of <em>Anabaena</em> (<em>Nostoc</em>) sp. PCC 7120.<a reveal-id="dsy006-B18" data-open="dsy006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Similarly, the expression of the gene for M.Ssp6803I in tobacco chloroplasts led to the methylation of the plastome DNA, but the transplastomic lines showed no alterations in plastid gene expression and were phenotypically indistinguishable from wild-type plants.<a reveal-id="dsy006-B46" data-open="dsy006-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> The minor effects of the mutation of the gene for M.Ssp6803I on cell viability and gene expression contrast its widespread occurrence among cyanobacteria. However, the close correlation between the presence of this methyltransferase and the occurrence of HIP1 sequences has led to a model wherein M.Ssp6803I could be involved in the methylation-directed mismatch repair of DNA, which is potentially of high importance for cyanobacteria exposed to strong light intensities, including UV.<a reveal-id="dsy006-B43" data-open="dsy006-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a></p><p>The enzyme M.Ssp6803III modifies adenine in the sequence 5′-GATC-3′, which is an internal part of the HIP1 sequence but often also occurs separately. Correspondingly, genes for methyltransferases modifying G<sup>m5</sup>CGATCGC and G<sup>m6</sup>ATC are usually co-occurring in the genomes of cyanobacteria.<a reveal-id="dsy006-B43" data-open="dsy006-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The activity of the Dam-like enzyme M.ssp6803III was clearly proven by the inhibition of <em>Mbo</em>I cleavage in <em>E. coli</em> cells expressing this <em>Synechocystis</em> gene. Its activity is also sufficiently high to modify virtually all GATC sites in the <em>Synechocystis</em> DNA, which is completely resistant against <em>Mbo</em>I treatments. Similar results were obtained when <em>slr1803</em> was expressed in tobacco plastids.<a reveal-id="dsy006-B46" data-open="dsy006-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> The M.Ssp6803III (Slr1803) appears homologous to related Dam-like enzymes from many bacteria (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). It is well known that Dam methylation has many physiological functions such as in the initiation of DNA replication, nucleoid segregation, post-replicative DNA mismatch repair, and gene expression regulation.<a reveal-id="dsy006-B9" data-open="dsy006-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy006-B47" data-open="dsy006-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The Dam-like protein M.Ssp6803III plays an essential role in <em>Synechocystis</em> since our attempts to generate a null mutant were not successful and led only to a partial gene replacement. Similar results were reported for the ortholog in <em>Anabaena</em> (<em>Nostoc</em>) sp. PCC 7120.<a reveal-id="dsy006-B18" data-open="dsy006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> In contrast, <em>dam</em> mutants were obtained for <em>E. coli</em> that remained viable under standard conditions but showed an increased mutation rate (reviewed in [<a reveal-id="dsy006-B47" data-open="dsy006-B47" class="link link-ref link-reveal xref-bibr">47</a>]), whereas Dam methylation is essential for viability in <em>Vibrio cholerae</em>.<a reveal-id="dsy006-B48" data-open="dsy006-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> The different dependence of these bacteria on Dam could be explained by differences in the mode of chromosomal replication. In chromosome II of <em>V. cholera</em>, the origin for DNA replication (<em>oriC</em>) is different from that of <em>E. coli</em>. It replicates in a DnaA-independent manner but was found to strictly depend on the methylation by Dam.<a reveal-id="dsy006-B48 dsy006-B49 dsy006-B50" data-open="dsy006-B48 dsy006-B49 dsy006-B50" class="link link-ref link-reveal xref-bibr"><sup>48–50</sup></a> The molecular details of DNA replication in <em>Synechocystis</em> are less well understood, but it has been shown that DnaA is not essential for the initiation of DNA replication.<a reveal-id="dsy006-B51" data-open="dsy006-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsy006-B52" data-open="dsy006-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Thus, it might be possible that the Dam-dependent DNA methylation is essential for the mode of DNA replication in <em>Synechocystis</em> similar to the case of <em>V. cholerae</em>.</p><p>The DNA-methyltransferase M.Ssp6803II modifies the <em>Hae</em>III recognition sequence 5′-GGCC-3′. Its recognition site was verified by screening <em>Hae</em>III-resistant plasmids in a <em>Synechocystis</em> gene library, by mutation and overexpression of the ORF <em>sll</em>0729. Moreover, bisulfite sequencing revealed that M.Ssp6803II is specific for N4-cytosine leading to 5′-GG<sup>m4</sup>CC-3′ and modifies at least 90% of the recognition sequence. However, the protein shows structural features that are different from the well-conserved C5-cytosine-specific DNA methyltransferases, including M.HaeIII. Instead of cytosine-specific enzymes, the most similar proteins all belong to the group of N6-adenine-specific DNA methyltransferases, which is documented by the phylogenetic analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span><strong>)</strong>. Structural and sequence comparisons of cytosine- and adenine-specific enzymes revealed that most of the conserved motifs are shared by both enzyme classes, only the organization of these conserved motifs and minor sequence differences within seem to determine whether an enzyme is specific for cytosine or adenine.<a reveal-id="dsy006-B53" data-open="dsy006-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Correspondingly, a sequence alignment of M.Ssp6803II with several previously characterized Dam-like sequences revealed distinct sequence differences between the cytosine-specific and the adenine-specific enzymes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p><p>The DmtB enzyme in <em>Anabaena</em> (<em>Nostoc</em>) sp. PCC 7120 shows similar functional and structural features to M.Ssp6803II, which included the N4-methylation of the first cytosine leading to the inhibition of <em>Hae</em>III restriction activity.<a reveal-id="dsy006-B18" data-open="dsy006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The deletion of M.Ssp6803II (Δ<em>sll0729</em> mutant) led to a strong phenotype and the mutant could only be maintained at conditions permitting slow growth. Hence, the modification of the <em>Hae</em>III recognition sequence is important for the performance of <em>Synechocystis</em> under conditions promoting high growth rates. However, further experiments are needed to identify the primary cause of this strong phenotypic alteration. We hypothesize that the absence of GGCC methylation could either have a broad impact on gene expression or the coordination of DNA replication with cell propagation.</p><p>Moreover, we analyzed three additional DNA methyltransferases in <em>Synechocystis</em>, two of which modify sequence motifs that have not been previously detected among cyanobacteria. Albeit lacking genetic and biochemical evidence, we conclude that M.Ssp6803IV, which is encoded by the <em>slr6050</em> gene on the plasmid pSYSX, is likely responsible for the modification of GA<sup>m6</sup>AGGC motifs. This assumption is supported by the REBASE database since a search using this recognition sequence revealed the Slr6050 protein as M.Ssp6803IV and by its similarity to Eco57I with a closely related recognition sequence CTGA <sup>m6</sup>AG.<a reveal-id="dsy006-B40" data-open="dsy006-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Moreover, it is also supported by the elimination of other possible candidate enzymes, since with M.Ssp6803V we identified the enzyme modifying the only remaining, bipartite motif GGAN<sub>7</sub>TTGG/CCAAN<sub>7</sub>TCC. M.Ssp6803V is encoded by the <em>slr6095</em> gene on plasmid pSYSX. It is annotated as part of a type I RM system.<a reveal-id="dsy006-B41" data-open="dsy006-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Nevertheless, the restriction system appears not to be active in <em>Synechocystis</em>, since we obtained a mutant lacking the respective methylation. Furthermore, we characterized the <em>sll8009</em> gene, which is also annotated to encode a methyltransferase of a type I RM system.<a reveal-id="dsy006-B41" data-open="dsy006-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> However, a closer inspection of this DNA locus indicated that the gene encodes a truncated, inactive protein. Correspondingly, the mutation of <em>sll8009</em> has no impact on DNA methylation of <em>Synechocystis</em>. Taking this into account, M.Ssp6803IV is the only remaining DNA methyltransferase for GA<sup>m6</sup>AGGC modification.</p><p>Altogether, this study provides the first comprehensive methylome analysis of the cyanobacterial model strain <em>Synechocystis</em> sp. PCC 6803. Moreover, it can be regarded as the groundwork for systematic analyses of the possible impact and molecular mechanisms linking methyltransferase activities and particular phenotypes in cyanobacteria. Particularly, the detected ∼10% unmethylated CGATCG and GGCC sites in the bisulfite sequencing analyses constitute a solid basis for further detailed analyses of their functional relevance.</p>                            <h2 scrollto-destination=120122054 id="120122054" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Richard J. Roberts (New England Biolabs) for helpful discussion. Klaudia Michl and Viktoria Reimann are acknowledged for technical assistance.</p>                            <h3 scrollto-destination=120122056 id="120122056" class="section-title" >Accession numbers</h3>
<p>All bisulfite and SMRT sequencing raw data were uploaded to the databases at the National Center for Biotechnology Information (BioProject ID: PRJNA430784, BioSample: SAMN08378604, and SRA: SRS2844079).</p>                            <h3 scrollto-destination=120122058 id="120122058" class="section-title" >Funding</h3>
<p>This study was funded by the German Research Foundation (Deutsche Forschungsgemeinschaft) via a joint grant to MH (HA2002/17-1) and WRH (HE 2544/10-1).</p>                            <h3 scrollto-destination=120122060 id="120122060" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=120122062 id="120122062" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy006-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">D'Urso</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Brickner</div> <div class="given-names">J. H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Mechanisms of epigenetic memory</div>, <div class="source">Trends Genet</div> ., <div class="volume">30</div>, <div class="fpage">230</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20epigenetic%20memory&amp;author=A.%20D'Urso&amp;author=J.%20H.%20Brickner&amp;publication_year=2014&amp;journal=Trends%20Genet&amp;volume=30&amp;pages=230-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2014.04.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2014.04.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2014.04.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24780085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vidalis</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Živković</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Wardenaar</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Roquis</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tellier</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Johannes</div> <div class="given-names">F.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Methylome evolution in plants</div>, <div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">264</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methylome%20evolution%20in%20plants&amp;author=A.%20Vidalis&amp;author=D.%20%C5%BDivkovi%C4%87&amp;author=R.%20Wardenaar&amp;author=D.%20Roquis&amp;author=A.%20Tellier&amp;author=F.%20Johannes&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=264" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-016-1127-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-1127-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-016-1127-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27998290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Noyer-Weidner</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Trautner</div> <div class="given-names">T. A.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Methylation of DNA in prokaryotes</div>, <div class="source">Birkhäuser: Basel</div> .</div></div></div></div><div content-id="dsy006-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tock</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Dryden</div> <div class="given-names">D. T.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">The biology of restriction and anti-restriction</div>, <div class="source">Curr. Opin. Microbiol</div> ., <div class="volume">8</div>, <div class="fpage">466</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biology%20of%20restriction%20and%20anti-restriction&amp;author=M.%20R.%20Tock&amp;author=D.%20T.%20Dryden&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=8&amp;pages=466-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2005.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2005.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2005.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15979932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vanyushin</div> <div class="given-names">B. F.</div></div>, <div class="name string-name"><div class="surname">Tkacheva</div> <div class="given-names">S. G.</div></div>, <div class="name string-name"><div class="surname">Belozersky</div> <div class="given-names">A. N.</div></div></div> <div class="year">1970</div>, 
<div class="article-title">Rare bases in animal DNA</div>, <div class="source">Nature</div> , <div class="volume">225</div>, <div class="fpage">948</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rare%20bases%20in%20animal%20DNA&amp;author=B.%20F.%20Vanyushin&amp;author=S.%20G.%20Tkacheva&amp;author=A.%20N.%20Belozersky&amp;publication_year=1970&amp;journal=Nature&amp;volume=225&amp;pages=948-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/225948a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F225948a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F225948a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4391887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Greer</div> <div class="given-names">E. L.</div></div>, <div class="name string-name"><div class="surname">Blanco</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">DNA methylation on N6-adenine in <em>C. elegans</em></div>, <div class="source">Cell</div> , <div class="volume">161</div>, <div class="fpage">868</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20on%20N6-adenine%20in%20C.%20elegans&amp;author=E.%20L.%20Greer&amp;author=M.%20A.%20Blanco&amp;author=L.%20Gu&amp;publication_year=2015&amp;journal=Cell&amp;volume=161&amp;pages=868-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.04.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.04.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.04.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25936839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">G. Z.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">N6-methyldeoxyadenosine marks active transcription start sites in <em>Chlamydomonas</em></div>, <div class="source">Cell</div> , <div class="volume">161</div>, <div class="fpage">879</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=N6-methyldeoxyadenosine%20marks%20active%20transcription%20start%20sites%20in%20Chlamydomonas&amp;author=Y.%20Fu&amp;author=G.%20Z.%20Luo&amp;author=K.%20Chen&amp;publication_year=2015&amp;journal=Cell&amp;volume=161&amp;pages=879-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.04.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.04.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.04.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25936837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">N6-methyladenine DNA modification in <em>Drosophila</em></div>, <div class="source">Cell</div> , <div class="volume">161</div>, <div class="fpage">893</div>–<div class="lpage">906</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=N6-methyladenine%20DNA%20modification%20in%20Drosophila&amp;author=G.%20Zhang&amp;author=H.%20Huang&amp;author=D.%20Liu&amp;publication_year=2015&amp;journal=Cell&amp;volume=161&amp;pages=893-906" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.04.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.04.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.04.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25936838" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Casadesús</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Bacterial DNA methylation and methylomes. In: Jeltsch, A., Jurkowska, R. Z. (eds) DNA Methyltransferases – Role and Function</div>, <div class="source">Adv. Exp. Med. Biol</div> ., <div class="volume">945</div>, <div class="fpage">35</div>–<div class="lpage">61</div>. Vol. Springer International, Switzerland, p.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20DNA%20methylation%20and%20methylomes.%20In%3A%20Jeltsch%2C%20A.%2C%20Jurkowska%2C%20R.%20Z.%20(eds)%20DNA%20Methyltransferases%20%E2%80%93%20Role%20and%20Function&amp;author=J.%20Casades%C3%BAs&amp;publication_year=2016&amp;journal=Adv.%20Exp.%20Med.%20Biol&amp;volume=945&amp;pages=35-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-3-319-43624-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-319-43624-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-3-319-43624-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27826834" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Erova</div> <div class="given-names">T. E.</div></div>, <div class="name string-name"><div class="surname">Kosykh</div> <div class="given-names">V. G.</div></div>, <div class="name string-name"><div class="surname">Sha</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chopra</div> <div class="given-names">A. K.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">DNA adenine methyltransferase (Dam) controls the expression of the cytotoxic enterotoxin (<em>act</em>) gene of <em>Aeromonas hydrophila</em> via tRNA modifying enzyme-glucose-inhibited division protein (GidA)</div>, <div class="source">Gene</div> , <div class="volume">498</div>, <div class="fpage">280</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20adenine%20methyltransferase%20(Dam)%20controls%20the%20expression%20of%20the%20cytotoxic%20enterotoxin%20(act)%20gene%20of%20Aeromonas%20hydrophila%20via%20tRNA%20modifying%20enzyme-glucose-inhibited%20division%20protein%20(GidA)&amp;author=T.%20E.%20Erova&amp;author=V.%20G.%20Kosykh&amp;author=J.%20Sha&amp;author=A.%20K.%20Chopra&amp;publication_year=2012&amp;journal=Gene&amp;volume=498&amp;pages=280-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2012.02.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2012.02.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2012.02.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22391092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Braaten</div> <div class="given-names">B. A.</div></div>, <div class="name string-name"><div class="surname">Nou</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Kaltenbach</div> <div class="given-names">L. S.</div></div>, <div class="name string-name"><div class="surname">Low</div> <div class="given-names">D. A.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Methylation patterns in <em>pap</em> regulatory DNA control pyelonephritis-associated pili phase variation in <em>E. coli</em></div>, <div class="source">Cell</div> , <div class="volume">76</div>, <div class="fpage">577</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methylation%20patterns%20in%20pap%20regulatory%20DNA%20control%20pyelonephritis-associated%20pili%20phase%20variation%20in%20E.%20coli&amp;author=B.%20A.%20Braaten&amp;author=X.%20Nou&amp;author=L.%20S.%20Kaltenbach&amp;author=D.%20A.%20Low&amp;publication_year=1994&amp;journal=Cell&amp;volume=76&amp;pages=577-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(94)90120-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(94)90120-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(94)90120-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7906204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nou</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Braaten</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Kaltenbach</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Low</div> <div class="given-names">D. A.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Differential binding of Lrp to two sets of <em>pap</em> DNA binding sites mediated by Pap I regulates Pap phase variation in <em>Escherichia coli</em></div>, <div class="source">EMBO J</div> ., <div class="volume">14</div>, <div class="fpage">5785</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20binding%20of%20Lrp%20to%20two%20sets%20of%20pap%20DNA%20binding%20sites%20mediated%20by%20Pap%20I%20regulates%20Pap%20phase%20variation%20in%20Escherichia%20coli&amp;author=X.%20Nou&amp;author=B.%20Braaten&amp;author=L.%20Kaltenbach&amp;author=D.%20A.%20Low&amp;publication_year=1995&amp;journal=EMBO%20J&amp;volume=14&amp;pages=5785-97" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8846772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Militello</div> <div class="given-names">K. T.</div></div>, <div class="name string-name"><div class="surname">Simon</div> <div class="given-names">R. D.</div></div>, <div class="name string-name"><div class="surname">Qureshi</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Conservation of Dcm-mediated cytosine DNA methylation in <em>Escherichia coli</em></div>, <div class="source">FEMS Microbiol. Lett</div> ., <div class="volume">328</div>, <div class="fpage">78</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20Dcm-mediated%20cytosine%20DNA%20methylation%20in%20Escherichia%20coli&amp;author=K.%20T.%20Militello&amp;author=R.%20D.%20Simon&amp;author=M.%20Qureshi&amp;publication_year=2012&amp;journal=FEMS%20Microbiol.%20Lett&amp;volume=328&amp;pages=78-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2012.328.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2012.328.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2012.328.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22150247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fioravanti</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fumeaux</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Mohapatra</div> <div class="given-names">S. S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">DNA binding of the cell cycle transcriptional regulator GcrA depends on N6-adenosine methylation in <em>Caulobacter crescentus</em> and other alphaproteobacteria</div>, <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003541</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20binding%20of%20the%20cell%20cycle%20transcriptional%20regulator%20GcrA%20depends%20on%20N6-adenosine%20methylation%20in%20Caulobacter%20crescentus%20and%20other%20alphaproteobacteria&amp;author=A.%20Fioravanti&amp;author=C.%20Fumeaux&amp;author=S.%20S.%20Mohapatra&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003541" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003541" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003541" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003541"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23737758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lluch-Senar</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Luong</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Lloréns-Rico</div> <div class="given-names">V.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Comprehensive methylome characterization of <em>Mycoplasma genitalium</em> and <em>Mycoplasma pneumoniae</em> at single-base resolution</div>, <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003191</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20methylome%20characterization%20of%20Mycoplasma%20genitalium%20and%20Mycoplasma%20pneumoniae%20at%20single-base%20resolution&amp;author=M.%20Lluch-Senar&amp;author=K.%20Luong&amp;author=V.%20Llor%C3%A9ns-Rico&amp;publication_year=2012&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003191" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003191"> </span></div> </div></div></div></div></div><div content-id="dsy006-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blow</div> <div class="given-names">M. J.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">T. A.</div></div>, <div class="name string-name"><div class="surname">Daum</div> <div class="given-names">C. G.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The epigenomic landscape of prokaryotes</div>, <div class="source">PLoS Genet</div> ., <div class="volume">12</div>, <div class="fpage">e1005854</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20epigenomic%20landscape%20of%20prokaryotes&amp;author=M.%20J.%20Blow&amp;author=T.%20A.%20Clark&amp;author=C.%20G.%20Daum&amp;publication_year=2016&amp;journal=PLoS%20Genet&amp;volume=12&amp;pages=e1005854" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005854" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005854" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005854"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26870957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hagemann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W. R.</div></div></div> <div class="year">2018</div>, 
<div class="article-title">Systems and synthetic biology for the biotechnological application of cyanobacteria</div>, <div class="source">Curr. Opin. Biotechnol</div> ., <div class="volume">49</div>, <div class="fpage">94</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systems%20and%20synthetic%20biology%20for%20the%20biotechnological%20application%20of%20cyanobacteria&amp;author=M.%20Hagemann&amp;author=W.%20R.%20Hess&amp;publication_year=2018&amp;journal=Curr.%20Opin.%20Biotechnol&amp;volume=49&amp;pages=94-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.copbio.2017.07.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.copbio.2017.07.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.copbio.2017.07.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28843192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matveyev</div> <div class="given-names">A. V.</div></div>, <div class="name string-name"><div class="surname">Young</div> <div class="given-names">K. T.</div></div>, <div class="name string-name"><div class="surname">Meng</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Elhai</div> <div class="given-names">J.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">DNA methyltransferases of the cyanobacterium <em>Anabaena</em> PCC 7120</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">29</div>, <div class="fpage">1491</div>–<div class="lpage">506</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methyltransferases%20of%20the%20cyanobacterium%20Anabaena%20PCC%207120&amp;author=A.%20V.%20Matveyev&amp;author=K.%20T.%20Young&amp;author=A.%20Meng&amp;author=J.%20Elhai&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=1491-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.7.1491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.7.1491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.7.1491"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11266551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zehr</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Ohki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Landry</div> <div class="given-names">D.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Unique modification of adenine in genomic DNA of the marine cyanobacterium <em>Trichodesmium</em> sp. strain NIBB 1067</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">173</div>, <div class="fpage">7059</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unique%20modification%20of%20adenine%20in%20genomic%20DNA%20of%20the%20marine%20cyanobacterium%20Trichodesmium%20sp.%20strain%20NIBB%201067&amp;author=J.%20P.%20Zehr&amp;author=K.%20Ohki&amp;author=Y.%20Fujita&amp;author=D.%20Landry&amp;publication_year=1991&amp;journal=J.%20Bacteriol&amp;volume=173&amp;pages=7059-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.173.21.7059-7062.1991" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.173.21.7059-7062.1991" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.173.21.7059-7062.1991"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1657876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Scharnagl</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Richter</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hagemann</div> <div class="given-names">M.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">The cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803 expresses a DNA methyltransferase specific for the recognition sequence of the restriction endonuclease <em>Pvu</em>I</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">180</div>, <div class="fpage">4116</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC%206803%20expresses%20a%20DNA%20methyltransferase%20specific%20for%20the%20recognition%20sequence%20of%20the%20restriction%20endonuclease%20PvuI&amp;author=M.%20Scharnagl&amp;author=S.%20Richter&amp;author=M.%20Hagemann&amp;publication_year=1998&amp;journal=J.%20Bacteriol&amp;volume=180&amp;pages=4116-22" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9696758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Meldrum</div> <div class="given-names">D. R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Premethylation of foreign DNA improves integrative transformation efficiency in <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">81</div>, <div class="fpage">8500</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Premethylation%20of%20foreign%20DNA%20improves%20integrative%20transformation%20efficiency%20in%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=B.%20Wang&amp;author=J.%20Yu&amp;author=W.%20Zhang&amp;author=D.%20R.%20Meldrum&amp;publication_year=2015&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=81&amp;pages=8500-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02575-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02575-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02575-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26452551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trautmann</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Voss</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wilde</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Al-Babili</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W. R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Microevolution in cyanobacteria: re-sequencing a motile substrain of <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">DNA Res</div> ., <div class="volume">19</div>, <div class="fpage">435</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microevolution%20in%20cyanobacteria%3A%20re-sequencing%20a%20motile%20substrain%20of%20Synechocystis%20sp.%20PCC%206803&amp;author=D.%20Trautmann&amp;author=B.%20Voss&amp;author=A.%20Wilde&amp;author=S.%20Al-Babili&amp;author=W.%20R.%20Hess&amp;publication_year=2012&amp;journal=DNA%20Res&amp;volume=19&amp;pages=435-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23069868" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rippka</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Deruelles</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Waterbury</div> <div class="given-names">J. B.</div></div>, <div class="name string-name"><div class="surname">Herdman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stanier</div> <div class="given-names">R. Y.</div></div></div> <div class="year">1979</div>, 
<div class="article-title">Generic assignments, strain histories and properties of pure cultures of cyanobacteria</div>, <div class="source">J. Gen. Microbiol</div> ., <div class="volume">111</div>, <div class="fpage">1</div>–<div class="lpage">16</div>.</div></div></div></div><div content-id="dsy006-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">K.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Preparation of genomic DNA from bacteria</div>, <div class="source">Curr. Prot. Molec. Biol</div> ., 00:1:2.4:2.4.1-2.4.5; <a class="link link-uri" href="http://dx.doi.org/10.1002/0471142727.mb0204s56" target="_blank">http://dx.doi.org/10.1002/0471142727.mb0204s56</a> (1 February 2018, date last accessed).</div></div></div></div><div content-id="dsy006-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krueger</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Andrews</div> <div class="given-names">S. R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications. Krueger F1, Andrews SR</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">1571</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bismark%3A%20a%20flexible%20aligner%20and%20methylation%20caller%20for%20Bisulfite-Seq%20applications.%20Krueger%20F1%2C%20Andrews%20SR&amp;author=F.%20Krueger&amp;author=S.%20R.%20Andrews&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=1571-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21493656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fritsch</div> <div class="given-names">E. F.</div></div>, <div class="name string-name"><div class="surname">Maniatis</div> <div class="given-names">T.</div></div></div> <div class="year">1989</div>, <div class="source">Molecular cloning</div> . 
<div class="publisher-name">Cold Spring Harbour Laboratory Press</div>: 
<div class="publisher-loc">Cold Spring Harbor, NY</div>.</div></div></div></div><div content-id="dsy006-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kotani</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">1996</div>, 
<div class="article-title">Sequence analysis of the genome of the unicellular cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions</div>, <div class="source">DNA Res</div> ., <div class="volume">3</div>, <div class="fpage">109</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20the%20genome%20of%20the%20unicellular%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC%206803.%20II.%20Sequence%20determination%20of%20the%20entire%20genome%20and%20assignment%20of%20potential%20protein-coding%20regions&amp;author=T.%20Kaneko&amp;author=S.%20Sato&amp;author=H.%20Kotani&amp;publication_year=1996&amp;journal=DNA%20Res&amp;volume=3&amp;pages=109-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/3.3.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F3.3.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F3.3.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8905231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zinchenko</div> <div class="given-names">V. V.</div></div>, <div class="name string-name"><div class="surname">Piven</div> <div class="given-names">I. V.</div></div>, <div class="name string-name"><div class="surname">Melnik</div> <div class="given-names">V. A.</div></div>, <div class="name string-name"><div class="surname">Shestakov</div> <div class="given-names">S. V.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Vectors for the complementation analysis of cyanobacterial mutants</div>, <div class="source">Russ. J. Genet</div> ., <div class="volume">35</div>, <div class="fpage">228</div>–<div class="lpage">32</div>.</div></div></div></div><div content-id="dsy006-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lowry</div> <div class="given-names">O. H.</div></div>, <div class="name string-name"><div class="surname">Rosenbrough</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Farr</div> <div class="given-names">A. L.</div></div>, <div class="name string-name"><div class="surname">Randall</div> <div class="given-names">R. J.</div></div></div> <div class="year">1951</div>, 
<div class="article-title">Protein measurement with the folin phenol reagent</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">193</div>, <div class="fpage">265</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20measurement%20with%20the%20folin%20phenol%20reagent&amp;author=O.%20H.%20Lowry&amp;author=N.%20J.%20Rosenbrough&amp;author=A.%20L.%20Farr&amp;author=R.%20J.%20Randall&amp;publication_year=1951&amp;journal=J.%20Biol.%20Chem&amp;volume=193&amp;pages=265-75" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14907713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>, <div class="name string-name"><div class="surname">Schäffer</div> <div class="given-names">A. A.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">1997</div>, 
<div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucl. Acid. Res</div> ., <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Sch%C3%A4ffer&amp;author=J.%20Zhang&amp;publication_year=1997&amp;journal=Nucl.%20Acid.%20Res&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div> </div></div></div></div></div><div content-id="dsy006-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>, <div class="name string-name"><div class="surname">Wootton</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Gertz</div> <div class="given-names">E. M.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Protein database searches using compositionally adjusted substitution matrices</div>, <div class="source">FEBS J</div> ., <div class="volume">272</div>, <div class="fpage">5101</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20database%20searches%20using%20compositionally%20adjusted%20substitution%20matrices&amp;author=S.%20F.%20Altschul&amp;author=J.%20C.%20Wootton&amp;author=E.%20M.%20Gertz&amp;publication_year=2005&amp;journal=FEBS%20J&amp;volume=272&amp;pages=5101-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ejb.2005.272.issue-20" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fejb.2005.272.issue-20" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fejb.2005.272.issue-20"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16218944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larkin</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">N. P.</div></div>, <div class="name string-name"><div class="surname">Chenna</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">ClustalW and ClustalX version 2</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2947</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ClustalW%20and%20ClustalX%20version%202&amp;author=M.%20A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.%20P.%20Brown&amp;author=R.%20Chenna&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">28</div>, <div class="fpage">2731</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Murray</div> <div class="given-names">I. A.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">T. A.</div></div>, <div class="name string-name"><div class="surname">Morgan</div> <div class="given-names">R. D.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The methylomes of six bacteria</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">11450</div>–<div class="lpage">114362</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20methylomes%20of%20six%20bacteria&amp;author=I.%20A.%20Murray&amp;author=T.%20A.%20Clark&amp;author=R.%20D.%20Morgan&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=11450-114362" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23034806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vilkaitis</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Klimašauskas</div> <div class="given-names">S.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Bisulfite sequencing protocol displays both 5-methylcytosine and N4-methylcytosine</div>, <div class="source">Anal. Biochem</div> ., <div class="volume">271</div>, <div class="fpage">116</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bisulfite%20sequencing%20protocol%20displays%20both%205-methylcytosine%20and%20N4-methylcytosine&amp;author=G.%20Vilkaitis&amp;author=S.%20Klima%C5%A1auskas&amp;publication_year=1999&amp;journal=Anal.%20Biochem&amp;volume=271&amp;pages=116-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/abio.1999.4116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fabio.1999.4116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fabio.1999.4116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10361019" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Vincze</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Posfai</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Macelis</div> <div class="given-names">D.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">REBASE—a database for DNA restriction and modification: enzymes, genes and genomes</div>, <div class="source">Nucl. Acid. Res</div> ., <div class="volume">38 (Database issue)</div>, <div class="fpage">D234</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=REBASE%E2%80%94a%20database%20for%20DNA%20restriction%20and%20modification%3A%20enzymes%2C%20genes%20and%20genomes&amp;author=R.%20J.%20Roberts&amp;author=T.%20Vincze&amp;author=J.%20Posfai&amp;author=D.%20Macelis&amp;publication_year=2010&amp;journal=Nucl.%20Acid.%20Res&amp;volume=38 (Database issue)&amp;pages=D234-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp874" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp874" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp874"> </span></div> </div></div></div></div></div><div content-id="dsy006-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ueno</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ito</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kimizuka</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Kotani</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nakajima</div> <div class="given-names">K.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Gene structure and expression of <em>Mbo</em>I restriction-modification system</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">21</div>, <div class="fpage">2309</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20structure%20and%20expression%20of%20MboI%20restriction-modification%20system&amp;author=T.%20Ueno&amp;author=H.%20Ito&amp;author=F.%20Kimizuka&amp;author=H.%20Kotani&amp;author=K.%20Nakajima&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res&amp;volume=21&amp;pages=2309-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.10.2309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.10.2309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.10.2309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8506128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Janulaitis</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vaisvila</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Timinskas</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Klimasauskas</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Butkus</div> <div class="given-names">V.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Cloning and sequence analysis of the genes coding for Eco57I type IV restriction-modification enzymes</div>, <div class="source">Nucl. Acid. Res</div> ., <div class="volume">20</div>, <div class="fpage">6051</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20sequence%20analysis%20of%20the%20genes%20coding%20for%20Eco57I%20type%20IV%20restriction-modification%20enzymes&amp;author=A.%20Janulaitis&amp;author=R.%20Vaisvila&amp;author=A.%20Timinskas&amp;author=S.%20Klimasauskas&amp;author=V.%20Butkus&amp;publication_year=1992&amp;journal=Nucl.%20Acid.%20Res&amp;volume=20&amp;pages=6051-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.22.6051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.22.6051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.22.6051"> </span></div> </div></div></div></div></div><div content-id="dsy006-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nakao</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Okamoto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kohara</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fujishiro</div> <div class="given-names">T.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">CyanoBase: the cyanobacteria genome database update 2010</div>, <div class="source">Nucl. Acid. Res</div> ., <div class="volume">38 (Database issue)</div>, <div class="fpage">D379</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CyanoBase%3A%20the%20cyanobacteria%20genome%20database%20update%202010&amp;author=M.%20Nakao&amp;author=S.%20Okamoto&amp;author=M.%20Kohara&amp;author=T.%20Fujishiro&amp;publication_year=2010&amp;journal=Nucl.%20Acid.%20Res&amp;volume=38 (Database issue)&amp;pages=D379-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp915" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp915" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp915"> </span></div> </div></div></div></div></div><div content-id="dsy006-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kopf</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Klähn</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Scholz</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Matthiessen</div> <div class="given-names">J. K.</div></div>, <div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W. R.</div></div>, <div class="name string-name"><div class="surname">Voß</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Comparative analysis of the primary transcriptome of <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">527</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20primary%20transcriptome%20of%20Synechocystis%20sp.%20PCC%206803&amp;author=M.%20Kopf&amp;author=S.%20Kl%C3%A4hn&amp;author=I.%20Scholz&amp;author=J.%20K.%20Matthiessen&amp;author=W.%20R.%20Hess&amp;author=B.%20Vo%C3%9F&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=527-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24935866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elhai</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Highly iterated palindromic sequences (HIPs) and their relationship to DNA methyltransferases</div>, <div class="source">Life (Basel)</div> , <div class="volume">5</div>, <div class="fpage">921</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20iterated%20palindromic%20sequences%20(HIPs)%20and%20their%20relationship%20to%20DNA%20methyltransferases&amp;author=J.%20Elhai&amp;publication_year=2015&amp;journal=Life%20(Basel)&amp;volume=5&amp;pages=921-48" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25789551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">P. J.</div></div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bleasby</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Whitton</div> <div class="given-names">B. A.</div></div>, <div class="name string-name"><div class="surname">Morby</div> <div class="given-names">A. P.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Singular over-representation of an octameric palindrome, HIP1, in DNA from many cyanobacteria</div>, <div class="source">Nucl. Acid. Res</div> ., <div class="volume">23</div>, <div class="fpage">729</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Singular%20over-representation%20of%20an%20octameric%20palindrome%2C%20HIP1%2C%20in%20DNA%20from%20many%20cyanobacteria&amp;author=N.%20J.%20Robinson&amp;author=P.%20J.%20Robinson&amp;author=A.%20Gupta&amp;author=A.%20J.%20Bleasby&amp;author=B.%20A.%20Whitton&amp;author=A.%20P.%20Morby&amp;publication_year=1995&amp;journal=Nucl.%20Acid.%20Res&amp;volume=23&amp;pages=729-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.5.729" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.5.729" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.5.729"> </span></div> </div></div></div></div></div><div content-id="dsy006-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">P. J.</div></div>, <div class="name string-name"><div class="surname">Cranenburgh</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Head</div> <div class="given-names">I. M.</div></div>, <div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">N. J.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">HIP1 propagates in cyanobacterial DNA via nucleotide substitutions but promotes excision at similar frequencies in <em>Escherichia coli</em> and <em>Synechococcus</em> PCC 7942</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">24</div>, <div class="fpage">181</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HIP1%20propagates%20in%20cyanobacterial%20DNA%20via%20nucleotide%20substitutions%20but%20promotes%20excision%20at%20similar%20frequencies%20in%20Escherichia%20coli%20and%20Synechococcus%20PCC%207942&amp;author=P.%20J.%20Robinson&amp;author=R.%20M.%20Cranenburgh&amp;author=I.%20M.%20Head&amp;author=N.%20J.%20Robinson&amp;publication_year=1997&amp;journal=Mol.%20Microbiol&amp;volume=24&amp;pages=181-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.3391695.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1997.3391695.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1997.3391695.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9140975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ahlert</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Stegemann</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kahlau</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ruf</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bock</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Insensitivity of chloroplast gene expression to DNA methylation</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">282</div>, <div class="fpage">17</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insensitivity%20of%20chloroplast%20gene%20expression%20to%20DNA%20methylation&amp;author=D.%20Ahlert&amp;author=S.%20Stegemann&amp;author=S.%20Kahlau&amp;author=S.%20Ruf&amp;author=R.%20Bock&amp;publication_year=2009&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=282&amp;pages=17-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-009-0440-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-009-0440-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-009-0440-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19290543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Adhikari</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Curtis</div> <div class="given-names">P. D.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">DNA methyltransferases and epigenetic regulation in bacteria</div>, <div class="source">FEMS Microbiol. Rev</div> ., <div class="volume">40</div>, <div class="fpage">575</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methyltransferases%20and%20epigenetic%20regulation%20in%20bacteria&amp;author=S.%20Adhikari&amp;author=P.%20D.%20Curtis&amp;publication_year=2016&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=40&amp;pages=575-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/femsre/fuw023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ffemsre%2Ffuw023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ffemsre%2Ffuw023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27476077" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Demarre</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Chattoraj</div> <div class="given-names">D. K.</div></div>, <div class="name string-name"><div class="surname">Burkholder</div> <div class="given-names">W. F.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">DNA adenine methylation is required to replicate both <em>Vibrio cholera</em> chromosomes once per cell cycle</div>, <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1000939</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20adenine%20methylation%20is%20required%20to%20replicate%20both%20Vibrio%20cholera%20chromosomes%20once%20per%20cell%20cycle&amp;author=G.%20Demarre&amp;author=D.%20K.%20Chattoraj&amp;author=W.%20F.%20Burkholder&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1000939" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000939" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000939" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000939"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20463886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Egan</div> <div class="given-names">E. S.</div></div>, <div class="name string-name"><div class="surname">Waldor</div> <div class="given-names">M. K.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Distinct replication requirements for the two <em>Vibrio cholera</em> chromosomes</div>, <div class="source">Cell</div> , <div class="volume">114</div>, <div class="fpage">521</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20replication%20requirements%20for%20the%20two%20Vibrio%20cholera%20chromosomes&amp;author=E.%20S.%20Egan&amp;author=M.%20K.%20Waldor&amp;publication_year=2003&amp;journal=Cell&amp;volume=114&amp;pages=521-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(03)00611-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(03)00611-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(03)00611-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12941279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Val</div> <div class="given-names">M.-E.</div></div>, <div class="name string-name"><div class="surname">Kennedy</div> <div class="given-names">S. P.</div></div>, <div class="name string-name"><div class="surname">Soler-Bistué</div> <div class="given-names">A. J.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Fuse or die: how to survive the loss of Dam in <em>Vibrio cholera</em>e</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">91</div>, <div class="fpage">665</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fuse%20or%20die%3A%20how%20to%20survive%20the%20loss%20of%20Dam%20in%20Vibrio%20cholerae&amp;author=M.-E.%20Val&amp;author=S.%20P.%20Kennedy&amp;author=A.%20J.%20Soler-Bistu%C3%A9&amp;publication_year=2014&amp;journal=Mol.%20Microbiol&amp;volume=91&amp;pages=665-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2014.91.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2014.91.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2014.91.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24308271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Richter</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hagemann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Messer</div> <div class="given-names">W.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Transcriptional analysis and mutation of a <em>dnaA</em>-like gene in <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">180</div>, <div class="fpage">4946</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20analysis%20and%20mutation%20of%20a%20dnaA-like%20gene%20in%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=S.%20Richter&amp;author=M.%20Hagemann&amp;author=W.%20Messer&amp;publication_year=1998&amp;journal=J.%20Bacteriol&amp;volume=180&amp;pages=4946-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9733700" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohbayashi</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ehira</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kanesaki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Chibazakura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yoshikawa</div> <div class="given-names">H.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Diversification of DnaA dependency for DNA replication in cyanobacterial evolution</div>, <div class="source">Isme J</div> ., <div class="volume">10</div>, <div class="fpage">1113</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20DnaA%20dependency%20for%20DNA%20replication%20in%20cyanobacterial%20evolution&amp;author=R.%20Ohbayashi&amp;author=S.%20Watanabe&amp;author=S.%20Ehira&amp;author=Y.%20Kanesaki&amp;author=T.%20Chibazakura&amp;author=H.%20Yoshikawa&amp;publication_year=2016&amp;journal=Isme%20J&amp;volume=10&amp;pages=1113-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2015.194" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2015.194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2015.194"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26517699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Malone</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Blumenthal</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">X.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Structure-guided analyses reveals nine sequence motifs conserved among DNA amino-methyltransferases, and suggests a catalytic mechanism for these enzymes</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">253</div>, <div class="fpage">618</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure-guided%20analyses%20reveals%20nine%20sequence%20motifs%20conserved%20among%20DNA%20amino-methyltransferases%2C%20and%20suggests%20a%20catalytic%20mechanism%20for%20these%20enzymes&amp;author=T.%20Malone&amp;author=R.%20M.%20Blumenthal&amp;author=X.%20Cheng&amp;publication_year=1995&amp;journal=J.%20Mol.%20Biol&amp;volume=253&amp;pages=618-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1995.0577" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1995.0577" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1995.0577"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7473738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy006-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy006-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carver</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Thomson</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Bleasby</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Berriman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">DNAPlotter: circular and linear interactive genome visualization</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">119</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNAPlotter%3A%20circular%20and%20linear%20interactive%20genome%20visualization&amp;author=T.%20Carver&amp;author=N.%20Thomson&amp;author=A.%20Bleasby&amp;author=M.%20Berriman&amp;author=J.%20Parkhill&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=119-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn578" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn578" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn578"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18990721" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4850982&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy006/1/dsy006_supplementary_material_revised2.pdf?Expires=2147483647&amp;Signature=p3MF0dMboVfJ2J0aQKF3NlQUgaCs-WViSP-r3VY3BKgV7stz6xodzogL0dfFZmqlsRd9eiGcCCZmMAkuaFn4uYoDnnv8sd3YXQHN9BXlXwkU7JhiN7MUTS68dwpqom-MjLtMACC2uQzCYQixhnn16ENqRDIa4UQzuaeOHhNbgE28gpEmM2Iq6oM-fUnyRC-RDwRnHjuhT2sJJj0hsB3apOWIzuSkHmJQ9UJd3iWC07dnhrXd5jJDEWHGwgi~a7ZzcZyQE-7qUdyb30ssq1lp6eGJ4T2UiI7~k9XzIkNZr8wDsq84LDQvRO2NP4oox83-Poti7RIfk7hjN24XorF0VQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,402</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy006"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4850982"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f4%2f343%2f4850982">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F4%2F343%2F4850982" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4850982'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identification of the DNA methyltransferases establishing the methylome of the cyanobacterium Synechocystis sp. PCC 6803" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy006");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f343%2f4850982">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

