

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/3/165/444512","@type":"ScholarlyArticle","name":"Draft Genome Sequencing and Comparative Analysis of  Aspergillus sojae  NBRC4239","datePublished":"2011-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2011-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr009","keywords":["<italic>Aspergillus sojae</italic>","<italic>Aspergillus oryzae</italic>","comparative genomics","genome sequencing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/3/165/444512","author":[{"name":"Sato, Atsushi","affiliation":"1  Research and Development Division ,  Kikkoman Corporation ,  399 Noda, Noda City, Chiba 278-0037 ,  Japan    2  Noda Institute for Scientific Research ,  399 Noda, Noda, Chiba 278-0037 ,  Japan","@type":"Person"},{"name":"Oshima, Kenshiro","affiliation":"3  Department of Computational Biology, Graduate School of Frontier Sciences ,  University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561 ,  Japan","@type":"Person"},{"name":"Noguchi, Hideki","affiliation":"4  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  4259 Nagatsutacho, Midoriku, Yokohama 226-8501 ,  Japan","@type":"Person"},{"name":"Ogawa, Masahiro","affiliation":"1  Research and Development Division ,  Kikkoman Corporation ,  399 Noda, Noda City, Chiba 278-0037 ,  Japan","@type":"Person"},{"name":"Takahashi, Tadashi","affiliation":"1  Research and Development Division ,  Kikkoman Corporation ,  399 Noda, Noda City, Chiba 278-0037 ,  Japan","@type":"Person"},{"name":"Oguma, Tetsuya","affiliation":"1  Research and Development Division ,  Kikkoman Corporation ,  399 Noda, Noda City, Chiba 278-0037 ,  Japan","@type":"Person"},{"name":"Koyama, Yasuji","affiliation":"2  Noda Institute for Scientific Research ,  399 Noda, Noda, Chiba 278-0037 ,  Japan","@type":"Person"},{"name":"Itoh, Takehiko","affiliation":"4  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  4259 Nagatsutacho, Midoriku, Yokohama 226-8501 ,  Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"3  Department of Computational Biology, Graduate School of Frontier Sciences ,  University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561 ,  Japan","@type":"Person"},{"name":"Hanya, Yoshiki","affiliation":"1  Research and Development Division ,  Kikkoman Corporation ,  399 Noda, Noda City, Chiba 278-0037 ,  Japan","@type":"Person"}],"description":"Abstract.  We conducted genome sequencing of the filamentous fungus Aspergillus sojae NBRC4239 isolated from the koji used to prepare Japanese soy sauce. We use","pageStart":"165","pageEnd":"176","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Draft Genome Sequencing and Comparative Analysis of  Aspergillus sojae  NBRC4239","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Draft Genome Sequencing and Comparative Analysis of  Aspergillus sojae  NBRC4239" />
    <meta property="og:description" content="Abstract.  We conducted genome sequencing of the filamentous fungus Aspergillus sojae NBRC4239 isolated from the koji used to prepare Japanese soy sauce. We use" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/3/165/444512" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sato, Atsushi" /><meta name="citation_author_institution" content="Research and Development Division, Kikkoman Corporation, 399 Noda, Noda City, Chiba 278-0037, Japan" /><meta name="citation_author_institution" content="Noda Institute for Scientific Research, 399 Noda, Noda, Chiba 278-0037, Japan" /><meta name="citation_author" content="Oshima, Kenshiro" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Noguchi, Hideki" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 4259 Nagatsutacho, Midoriku, Yokohama 226-8501, Japan" /><meta name="citation_author" content="Ogawa, Masahiro" /><meta name="citation_author_institution" content="Research and Development Division, Kikkoman Corporation, 399 Noda, Noda City, Chiba 278-0037, Japan" /><meta name="citation_author" content="Takahashi, Tadashi" /><meta name="citation_author_institution" content="Research and Development Division, Kikkoman Corporation, 399 Noda, Noda City, Chiba 278-0037, Japan" /><meta name="citation_author" content="Oguma, Tetsuya" /><meta name="citation_author_institution" content="Research and Development Division, Kikkoman Corporation, 399 Noda, Noda City, Chiba 278-0037, Japan" /><meta name="citation_author" content="Koyama, Yasuji" /><meta name="citation_author_institution" content="Noda Institute for Scientific Research, 399 Noda, Noda, Chiba 278-0037, Japan" /><meta name="citation_author" content="Itoh, Takehiko" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 4259 Nagatsutacho, Midoriku, Yokohama 226-8501, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Hanya, Yoshiki" /><meta name="citation_author_institution" content="Research and Development Division, Kikkoman Corporation, 399 Noda, Noda City, Chiba 278-0037, Japan" /><meta name="citation_title" content="Draft Genome Sequencing and Comparative Analysis of Aspergillus sojae NBRC4239" /><meta name="citation_firstpage" content="165" /><meta name="citation_lastpage" content="176" /><meta name="citation_doi" content="10.1093/dnares/dsr009" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2011/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Molecular strategy for identification in Aspergillus section Flavi; FEMS Microbiol. Lett.;  citation_year=2010;  citation_volume=304;  citation_pages=157-68; " /><meta name="citation_reference" content="citation_title=Cryptic speciation and recombination in the aflatoxin-producing fungus Aspergillus flavus; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=388-93; " /><meta name="citation_reference" content="citation_title=Analysis of expressed sequence tags from the fungus Aspergillus oryzae cultured under different conditions.; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=47-57; " /><meta name="citation_reference" content="citation_title=Understanding nonaflatoxigenicity of Aspergillus sojae: a windfall of aflatoxin biosynthesis research; Appl. Microbiol. Biotechnol.;  citation_year=2007;  citation_volume=76;  citation_pages=977-84; " /><meta name="citation_reference" content="citation_title=Genomics of Aspergillus oryzae; Biosci. Biotechnol. Biochem.;  citation_year=2007;  citation_volume=71;  citation_pages=646-70; " /><meta name="citation_reference" content="citation_title=Genome sequencing and analysis of Aspergillus oryzae; Nature;  citation_year=2005;  citation_volume=438;  citation_pages=1157-61; " /><meta name="citation_reference" content="citation_title=Genomics of industrial Aspergilli and comparison with toxigenic relatives; Food Addit. Contam. Part A Chem. Anal. Control Expo Risk Assess.;  citation_year=2008;  citation_volume=25;  citation_pages=1147-51; " /><meta name="citation_reference" content="citation_title=Genomics of Aspergillus oryzae: learning from the history of Koji mold and exploration of its future; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=173-83; " /><meta name="citation_reference" content="citation_title=TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=2878-79; " /><meta name="citation_reference" content="citation_title=Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=62" /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=ii215-25; " /><meta name="citation_reference" content="citation_title=Gene finding in novel genomes; BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=59" /><meta name="citation_reference" content="citation_title=GeneMark: web software for gene finding in prokaryotes, eukaryotes and viruses; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W451-4; " /><meta name="citation_reference" content="citation_title=GeneWise and Genomewise; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=988-5; " /><meta name="citation_reference" content="citation_title=Evigan: a hidden variable model for integrating gene evidence for eukaryotic gene predictions; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=597-605; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-4; " /><meta name="citation_reference" content="citation_title=The Aspergillus Genome Database, a curated comparative genomics resource for gene, protein and sequence information for the Aspergillus research community; , Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=D420-27; " /><meta name="citation_reference" content="citation_title=Profile hidden Markov models; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=755-63; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D281-88; " /><meta name="citation_reference" content="citation_title=MEROPS: the peptidase database; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=D227-33; " /><meta name="citation_reference" content="citation_title=The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools; Nucleic Acids Res.;  citation_year=1997;  citation_volume=24;  citation_pages=4876-82; " /><meta name="citation_reference" content="citation_title=TREEVIEW: an application to display phylogenetic trees on personal computers; Appl. Biosci.;  citation_year=1996;  citation_volume=12;  citation_pages=357-8; " /><meta name="citation_reference" content="citation_title=The Carbohydrate-Active EnZymes database (CAZy): an expert resource for glycogenomics; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D233-38; " /><meta name="citation_reference" content="citation_title=SMURF: genomic mapping of fungal secondary metabolite clusters; Fungal Genet. Biol.;  citation_year=2010;  citation_volume=47;  citation_pages=736-41; " /><meta name="citation_reference" content="citation_title=Clustered genes involved in cyclopiazonic acid production are next to the aflatoxin biosynthesis gene cluster; Fungal Genet. Biol.;  citation_year=2009;  citation_volume=46;  citation_pages=176-82; " /><meta name="citation_reference" content="citation_title=Identification of two aflatrem biosynthesis gene loci in Aspergillus flavus and metabolic engineering of Penicillium paxilli to elucidate their function; Appl. Environ. Microbiol.;  citation_year=2009;  citation_volume=75;  citation_pages=7469-81; " /><meta name="citation_reference" content="citation_title=Three alpha-amylase genes of Aspergillus oryzae exhibit identical intron–exon organization; Mol. Microbiol.;  citation_year=1989;  citation_volume=3;  citation_pages=3-14; " /><meta name="citation_reference" content="citation_title=DNA transposons: nature and applications in genomics; Curr. Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=115-8; " /><meta name="citation_reference" content="citation_title=Mode of AmyR binding to the CGGN8AGG sequence in the Aspergillus oryzae taaG2 promoter; Biosci. Biotechnol. Biochem.;  citation_year=2004;  citation_volume=68;  citation_pages=1906-11; " /><meta name="citation_reference" content="citation_title=Carbon repression in Aspergilli; FEMS Microbiol. Lett.;  citation_year=1997;  citation_volume=151;  citation_pages=103-4; " /><meta name="citation_reference" content="citation_title=Mobile elements in Caenorhabditis elegans and other nematodes; citation_publisher=American Society of Microbiology, Washington, DC,; Mobile DNA;  citation_year=1989;  citation_pages=537-56; " /><meta name="citation_reference" content="citation_title=Ant-1, an active transposon from the fungus Aspergillus niger; Mol. Gen. Genet.;  citation_year=1995;  citation_volume=249;  citation_pages=432-8; " /><meta name="citation_reference" content="citation_title=Identification of a novel polyketide synthase-nonribosomal peptide synthetase (PKS-NRPS) gene required for the biosynthesis of cyclopiazonin acid in Aspergillus oryzae; Fungal Genet. Biol.;  citation_year=2008;  citation_volume=45;  citation_pages=1608-15; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/3/165/1216725/dsr009.pdf" /><meta name="description" content="Abstract.  We conducted genome sequencing of the filamentous fungus Aspergillus sojae NBRC4239 isolated from the koji used to prepare Japanese soy sauce. We use" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/3/165/444512" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/3/165/444512" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/3/165/444512");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr009';
    </script>
<title>
	Draft Genome Sequencing and Comparative Analysis of Aspergillus sojae NBRC4239 | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./444512" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd5XPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBBdHN1c2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggYW5kIERldmVsb3BtZW50IERpdmlzaW9uLCBLaWtrb21hbiBDb3Jwb3JhdGlvbiwgMzk5IE5vZGEsIE5vZGEgQ2l0eSwgQ2hpYmEgMjc4LTAwMzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTm9kYSBJbnN0aXR1dGUgZm9yIFNjaWVudGlmaWMgUmVzZWFyY2gsIDM5OSBOb2RhLCBOb2RhLCBDaGliYSAyNzgtMDAzNywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpbWEsIEtlbnNoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5vZ3VjaGksIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBJbmZvcm1hdGlvbiwgR3JhZHVhdGUgU2Nob29sIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCA0MjU5IE5hZ2F0c3V0YWNobywgTWlkb3Jpa3UsIFlva29oYW1hIDIyNi04NTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9nYXdhLCBNYXNhaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlc2VhcmNoIGFuZCBEZXZlbG9wbWVudCBEaXZpc2lvbiwgS2lra29tYW4gQ29ycG9yYXRpb24sIDM5OSBOb2RhLCBOb2RhIENpdHksIENoaWJhIDI3OC0wMDM3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRha2FoYXNoaSwgVGFkYXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlc2VhcmNoIGFuZCBEZXZlbG9wbWVudCBEaXZpc2lvbiwgS2lra29tYW4gQ29ycG9yYXRpb24sIDM5OSBOb2RhLCBOb2RhIENpdHksIENoaWJhIDI3OC0wMDM3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9ndW1hLCBUZXRzdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggYW5kIERldmVsb3BtZW50IERpdmlzaW9uLCBLaWtrb21hbiBDb3Jwb3JhdGlvbiwgMzk5IE5vZGEsIE5vZGEgQ2l0eSwgQ2hpYmEgMjc4LTAwMzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS295YW1hLCBZYXN1amkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOb2RhIEluc3RpdHV0ZSBmb3IgU2NpZW50aWZpYyBSZXNlYXJjaCwgMzk5IE5vZGEsIE5vZGEsIENoaWJhIDI3OC0wMDM3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikl0b2gsIFRha2VoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIEluZm9ybWF0aW9uLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgVG9reW8gSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3ksIDQyNTkgTmFnYXRzdXRhY2hvLCBNaWRvcmlrdSwgWW9rb2hhbWEgMjI2LTg1MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF0dG9yaSwgTWFzYWhpcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIFRva3lvLCA1LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGFueWEsIFlvc2hpa2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBhbmQgRGV2ZWxvcG1lbnQgRGl2aXNpb24sIEtpa2tvbWFuIENvcnBvcmF0aW9uLCAzOTkgTm9kYSwgTm9kYSBDaXR5LCBDaGliYSAyNzgtMDAzNywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRyYWZ0IEdlbm9tZSBTZXF1ZW5jaW5nIGFuZCBDb21wYXJhdGl2ZSBBbmFseXNpcyBvZiBBc3BlcmdpbGx1cyBzb2phZSBOQlJDNDIzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNyMDA5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTEvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgc3RyYXRlZ3kgZm9yIGlkZW50aWZpY2F0aW9uIGluIEFzcGVyZ2lsbHVzIHNlY3Rpb24gRmxhdmk7IEZFTVMgTWljcm9iaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDQ7ICBjaXRhdGlvbl9wYWdlcz0xNTctNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyeXB0aWMgc3BlY2lhdGlvbiBhbmQgcmVjb21iaW5hdGlvbiBpbiB0aGUgYWZsYXRveGluLXByb2R1Y2luZyBmdW5ndXMgQXNwZXJnaWxsdXMgZmxhdnVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9Mzg4LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBleHByZXNzZWQgc2VxdWVuY2UgdGFncyBmcm9tIHRoZSBmdW5ndXMgQXNwZXJnaWxsdXMgb3J5emFlIGN1bHR1cmVkIHVuZGVyIGRpZmZlcmVudCBjb25kaXRpb25zLjsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00Ny01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5kZXJzdGFuZGluZyBub25hZmxhdG94aWdlbmljaXR5IG9mIEFzcGVyZ2lsbHVzIHNvamFlOiBhIHdpbmRmYWxsIG9mIGFmbGF0b3hpbiBiaW9zeW50aGVzaXMgcmVzZWFyY2g7IEFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzY7ICBjaXRhdGlvbl9wYWdlcz05NzctODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIG9mIEFzcGVyZ2lsbHVzIG9yeXphZTsgQmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTY0Ni03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNpbmcgYW5kIGFuYWx5c2lzIG9mIEFzcGVyZ2lsbHVzIG9yeXphZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzODsgIGNpdGF0aW9uX3BhZ2VzPTExNTctNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIG9mIGluZHVzdHJpYWwgQXNwZXJnaWxsaSBhbmQgY29tcGFyaXNvbiB3aXRoIHRveGlnZW5pYyByZWxhdGl2ZXM7IEZvb2QgQWRkaXQuIENvbnRhbS4gUGFydCBBIENoZW0uIEFuYWwuIENvbnRyb2wgRXhwbyBSaXNrIEFzc2Vzcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTQ3LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBBc3BlcmdpbGx1cyBvcnl6YWU6IGxlYXJuaW5nIGZyb20gdGhlIGhpc3Rvcnkgb2YgS29qaSBtb2xkIGFuZCBleHBsb3JhdGlvbiBvZiBpdHMgZnV0dXJlOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3My04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGlnclNjYW4gYW5kIEdsaW1tZXJITU06IHR3byBvcGVuIHNvdXJjZSBhYiBpbml0aW8gZXVrYXJ5b3RpYyBnZW5lLWZpbmRlcnM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9Mjg3OC03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBwcmVkaWN0aW9uIGluIGV1a2FyeW90ZXMgd2l0aCBhIGdlbmVyYWxpemVkIGhpZGRlbiBNYXJrb3YgbW9kZWwgdGhhdCB1c2VzIGhpbnRzIGZyb20gZXh0ZXJuYWwgc291cmNlczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz02MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgcHJlZGljdGlvbiB3aXRoIGEgaGlkZGVuIE1hcmtvdiBtb2RlbCBhbmQgYSBuZXcgaW50cm9uIHN1Ym1vZGVsOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPWlpMjE1LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGZpbmRpbmcgaW4gbm92ZWwgZ2Vub21lczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVNYXJrOiB3ZWIgc29mdHdhcmUgZm9yIGdlbmUgZmluZGluZyBpbiBwcm9rYXJ5b3RlcywgZXVrYXJ5b3RlcyBhbmQgdmlydXNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9VzQ1MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lV2lzZSBhbmQgR2Vub21ld2lzZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz05ODgtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZ2FuOiBhIGhpZGRlbiB2YXJpYWJsZSBtb2RlbCBmb3IgaW50ZWdyYXRpbmcgZ2VuZSBldmlkZW5jZSBmb3IgZXVrYXJ5b3RpYyBnZW5lIHByZWRpY3Rpb25zOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTU5Ny02MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQXNjYW4tU0U6IGEgcHJvZ3JhbSBmb3IgaW1wcm92ZWQgZGV0ZWN0aW9uIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBpbiBnZW5vbWljIHNlcXVlbmNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz05NTUtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEFzcGVyZ2lsbHVzIEdlbm9tZSBEYXRhYmFzZSwgYSBjdXJhdGVkIGNvbXBhcmF0aXZlIGdlbm9taWNzIHJlc291cmNlIGZvciBnZW5lLCBwcm90ZWluIGFuZCBzZXF1ZW5jZSBpbmZvcm1hdGlvbiBmb3IgdGhlIEFzcGVyZ2lsbHVzIHJlc2VhcmNoIGNvbW11bml0eTsgLCBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1ENDIwLTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9maWxlIGhpZGRlbiBNYXJrb3YgbW9kZWxzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTc1NS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFBmYW0gcHJvdGVpbiBmYW1pbGllcyBkYXRhYmFzZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDI4MS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVST1BTOiB0aGUgcGVwdGlkYXNlIGRhdGFiYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1EMjI3LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ2x1c3RhbFggd2luZG93cyBpbnRlcmZhY2U6IGZsZXhpYmxlIHN0cmF0ZWdpZXMgZm9yIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBhaWRlZCBieSBxdWFsaXR5IGFuYWx5c2lzIHRvb2xzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz00ODc2LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UUkVFVklFVzogYW4gYXBwbGljYXRpb24gdG8gZGlzcGxheSBwaHlsb2dlbmV0aWMgdHJlZXMgb24gcGVyc29uYWwgY29tcHV0ZXJzOyBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzU3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDYXJib2h5ZHJhdGUtQWN0aXZlIEVuWnltZXMgZGF0YWJhc2UgKENBWnkpOiBhbiBleHBlcnQgcmVzb3VyY2UgZm9yIGdseWNvZ2Vub21pY3M7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPUQyMzMtMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNNVVJGOiBnZW5vbWljIG1hcHBpbmcgb2YgZnVuZ2FsIHNlY29uZGFyeSBtZXRhYm9saXRlIGNsdXN0ZXJzOyBGdW5nYWwgR2VuZXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NzM2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyZWQgZ2VuZXMgaW52b2x2ZWQgaW4gY3ljbG9waWF6b25pYyBhY2lkIHByb2R1Y3Rpb24gYXJlIG5leHQgdG8gdGhlIGFmbGF0b3hpbiBiaW9zeW50aGVzaXMgZ2VuZSBjbHVzdGVyOyBGdW5nYWwgR2VuZXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTc2LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiB0d28gYWZsYXRyZW0gYmlvc3ludGhlc2lzIGdlbmUgbG9jaSBpbiBBc3BlcmdpbGx1cyBmbGF2dXMgYW5kIG1ldGFib2xpYyBlbmdpbmVlcmluZyBvZiBQZW5pY2lsbGl1bSBwYXhpbGxpIHRvIGVsdWNpZGF0ZSB0aGVpciBmdW5jdGlvbjsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03NTsgIGNpdGF0aW9uX3BhZ2VzPTc0NjktODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRocmVlIGFscGhhLWFteWxhc2UgZ2VuZXMgb2YgQXNwZXJnaWxsdXMgb3J5emFlIGV4aGliaXQgaWRlbnRpY2FsIGludHJvbuKAk2V4b24gb3JnYW5pemF0aW9uOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTMtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSB0cmFuc3Bvc29uczogbmF0dXJlIGFuZCBhcHBsaWNhdGlvbnMgaW4gZ2Vub21pY3M7IEN1cnIuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTE1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZGUgb2YgQW15UiBiaW5kaW5nIHRvIHRoZSBDR0dOOEFHRyBzZXF1ZW5jZSBpbiB0aGUgQXNwZXJnaWxsdXMgb3J5emFlIHRhYUcyIHByb21vdGVyOyBCaW9zY2kuIEJpb3RlY2hub2wuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9MTkwNi0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FyYm9uIHJlcHJlc3Npb24gaW4gQXNwZXJnaWxsaTsgRkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTEwMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGUgZWxlbWVudHMgaW4gQ2Flbm9yaGFiZGl0aXMgZWxlZ2FucyBhbmQgb3RoZXIgbmVtYXRvZGVzOyBjaXRhdGlvbl9wdWJsaXNoZXI9QW1lcmljYW4gU29jaWV0eSBvZiBNaWNyb2Jpb2xvZ3ksIFdhc2hpbmd0b24sIERDLDsgTW9iaWxlIEROQTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3BhZ2VzPTUzNy01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50LTEsIGFuIGFjdGl2ZSB0cmFuc3Bvc29uIGZyb20gdGhlIGZ1bmd1cyBBc3BlcmdpbGx1cyBuaWdlcjsgTW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDk7ICBjaXRhdGlvbl9wYWdlcz00MzItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBub3ZlbCBwb2x5a2V0aWRlIHN5bnRoYXNlLW5vbnJpYm9zb21hbCBwZXB0aWRlIHN5bnRoZXRhc2UgKFBLUy1OUlBTKSBnZW5lIHJlcXVpcmVkIGZvciB0aGUgYmlvc3ludGhlc2lzIG9mIGN5Y2xvcGlhem9uaW4gYWNpZCBpbiBBc3BlcmdpbGx1cyBvcnl6YWU7IEZ1bmdhbCBHZW5ldC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0xNjA4LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTgvMy8xNjUvMTIxNjcyNS9kc3IwMDkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdlIGNvbmR1Y3RlZCBnZW5vbWUgc2VxdWVuY2luZyBvZiB0aGUgZmlsYW1lbnRvdXMgZnVuZ3VzIEFzcGVyZ2lsbHVzIHNvamFlIE5CUkM0MjM5IGlzb2xhdGVkIGZyb20gdGhlIGtvamkgdXNlZCB0byBwcmVwYXJlIEphcGFuZXNlIHNveSBzYXVjZS4gV2UgdXNlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NDQ1MTImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkqmsWYG+5wOcJAyuATBQEw+DQhEk=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f165%2f444512" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f165%2f444512" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17382" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&amp;Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/3">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6645707">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6645707">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6645709">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6645709">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6645716">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6645716">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6645754">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6645754">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6645811">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6645811">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6645822">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6645822">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6645830">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6645830">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/3/153/444270">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/3/177/444849">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Draft Genome Sequencing and Comparative Analysis of <em>Aspergillus sojae</em> NBRC4239
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Research and Development Division</div>, <div class="insititution">Kikkoman Corporation</div>, <div class="addr-line">399 Noda, Noda City, Chiba 278-0037</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Noda Institute for Scientific Research</div>, <div class="addr-line">399 Noda, Noda, Chiba 278-0037</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsushi Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenshiro Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenshiro+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenshiro Oshima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Noguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">4259 Nagatsutacho, Midoriku, Yokohama 226-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Noguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Noguchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Noguchi H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Noguchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahiro Ogawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Research and Development Division</div>, <div class="insititution">Kikkoman Corporation</div>, <div class="addr-line">399 Noda, Noda City, Chiba 278-0037</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahiro+Ogawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogawa M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masahiro Ogawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadashi Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Research and Development Division</div>, <div class="insititution">Kikkoman Corporation</div>, <div class="addr-line">399 Noda, Noda City, Chiba 278-0037</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadashi+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tadashi Takahashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Oguma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Research and Development Division</div>, <div class="insititution">Kikkoman Corporation</div>, <div class="addr-line">399 Noda, Noda City, Chiba 278-0037</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Oguma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oguma T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oguma T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tetsuya Oguma</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuji Koyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Noda Institute for Scientific Research</div>, <div class="addr-line">399 Noda, Noda, Chiba 278-0037</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuji+Koyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koyama Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koyama Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasuji Koyama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">4259 Nagatsutacho, Midoriku, Yokohama 226-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takehiko Itoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiki Hanya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Research and Development Division</div>, <div class="insititution">Kikkoman Corporation</div>, <div class="addr-line">399 Noda, Noda City, Chiba 278-0037</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 4-7123-5515. Fax. +81 4-7123-5959. E-mail: <a href="mailto:yhanya@mail.kikkoman.co.jp" target="_blank">yhanya@mail.kikkoman.co.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiki+Hanya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hanya Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hanya Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiki Hanya</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 3, 1 June 2011, Pages 165–176, <a href='https://doi.org/10.1093/dnares/dsr009'>https://doi.org/10.1093/dnares/dsr009</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 December 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">13 April 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="444512" href="/dnaresearch/article-pdf/18/3/165/1216725/dsr009.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Atsushi Sato, Kenshiro Oshima, Hideki Noguchi, Masahiro Ogawa, Tadashi Takahashi, Tetsuya Oguma, Yasuji Koyama, Takehiko Itoh, Masahira Hattori, Yoshiki Hanya;  Draft Genome Sequencing and Comparative Analysis of <em>Aspergillus sojae</em> NBRC4239, <em>DNA Research</em>, Volume 18, Issue 3, 1 June 2011, Pages 165–176, <a href='https://doi.org/10.1093/dnares/dsr009'>https://doi.org/10.1093/dnares/dsr009</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/444512?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/444512?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/444512?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/444512?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/444512?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/444512?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/444512?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Draft%20Genome%20Sequencing%20and%20Comparative%20Analysis%20of%20%26lt%3Bem%26gt%3BAspergillus%20sojae%26lt%3B%2Fem%26gt%3B%20NBRC4239&amp;publicationDate=2011-01-01&amp;volumeNum=18&amp;issueNum=3&amp;author=Sato%2C%20Atsushi%3B%20Oshima%2C%20Kenshiro&amp;startPage=165&amp;endPage=176&amp;contentId=10.1093%2Fdnares%2Fdsr009&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f165%2f444512" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6645707 id="6645707" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We conducted genome sequencing of the filamentous fungus <em>Aspergillus sojae</em> NBRC4239 isolated from the <em>koji</em> used to prepare Japanese soy sauce. We used the 454 pyrosequencing technology and investigated the genome with respect to enzymes and secondary metabolites in comparison with other <em>Aspergilli</em> sequenced. Assembly of 454 reads generated a non-redundant sequence of 39.5-Mb possessing 13 033 putative genes and 65 scaffolds composed of 557 contigs. Of the 2847 open reading frames with Pfam domain scores of &gt;150 found in <em>A. sojae</em> NBRC4239, 81.7% had a high degree of similarity with the genes of <em>A. oryzae</em>. Comparative analysis identified serine carboxypeptidase and aspartic protease genes unique to <em>A. sojae</em> NBRC4239. While <em>A. oryzae</em> possessed three copies of α-amyalse gene, <em>A. sojae</em> NBRC4239 possessed only a single copy. Comparison of 56 gene clusters for secondary metabolites between <em>A. sojae</em> NBRC4239 and <em>A. oryzae</em> revealed that 24 clusters were conserved, whereas 32 clusters differed between them that included a deletion of 18 508 bp containing <em>mfs1</em>, <em>mao1</em>, <em>dmaT</em>, and <em>pks-nrps</em> for the cyclopiazonic acid (CPA) biosynthesis, explaining the no productivity of CPA in <em>A. sojae</em>. The <em>A. sojae</em> NBRC4239 genome data will be useful to characterize functional features of the <em>koji</em> moulds used in Japanese industries.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Aspergillus sojae&quot;"><em>Aspergillus sojae</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Aspergillus oryzae&quot;"><em>Aspergillus oryzae</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a></div>                            <h2 scrollto-destination=6645709 id="6645709" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Koji</em> moulds are widely used in the production of traditional fermented foods and beverages such as Japanese <em>miso</em>, soy sauce, and <em>sake</em>. Two typical <em>koji</em> moulds, <em>Aspergillus sojae</em> and <em>A. oryzae</em>, are used. During the fermentation process, <em>koji</em> moulds act by breaking down the ingredients. Each species of <em>koji</em> moulds reacts differently to the ingredients used and must therefore be selected based on the desired product. For example, <em>A. sojae</em> is selected to produce <em>miso</em> and soy sauce due to its high proteolytic ability, and <em>A. oryzae</em> is used widely in <em>sake</em>, <em>miso</em>, and soy sauce production for its high amylolytic ability. Among <em>Aspergillus</em> strains deposited in the RIKEN Bioresource Center Japan Collection of Microorganisms (<a class="link link-uri" href="http://www.jcm.riken.jp/JCM/JCM_DB.shtm">http://www.jcm.riken.jp/JCM/JCM_DB.shtm</a>), 15 strains out of 53 in <em>A. oryzae</em> strains were derived from <em>sake koji</em>, 6 from <em>miso</em>, and 17 from soy sauce <em>koji</em>, while 16 strains out of 20 strains of <em>A. sojae</em> were derived from soy sauce <em>koji</em>.</p><p>For taxonomic classification of <em>Aspergillus</em> species, molecular strategies have been developed to discriminate several <em>Aspergillus</em> species.<sup><a reveal-id="DSR009C1" data-open="DSR009C1" class="link link-ref link-reveal xref-bibr">1</a></sup><em>Aspergillus</em><em>oryzae</em> and <em>A. sojae</em> are classified in the <em>Aspergillus</em> section <em>Flavi</em>, which also includes plant pathogen <em>A. flavus</em> and <em>A. parasiticus</em> that produces aflatoxins known to be carcinogenic substances. The analysis based on restriction-site polymorphisms of genes coding for 11 proteins and sequences of five of those genes suggested that <em>A. oryzae</em> is a species derived from <em>A. flavus</em> through human handling.<sup><a reveal-id="DSR009C2" data-open="DSR009C2" class="link link-ref link-reveal xref-bibr">2</a></sup> From a viewpoint of evolutionarily close relation of <em>A. oryzae</em> and <em>A. sojae</em> with pathogenic <em>Aspergillus</em> species, it is vital to distinguish between aflatoxin productive and non-productive moulds and to select the latter for industrial use. It has been reported that <em>A. oryzae</em> does not produce these substances from expressed sequence tag (EST) analysis of <em>A. oryzae</em> RIB40, in which many of the aflatoxin biosynthesis gene clusters were found to be unexpressed.<sup><a reveal-id="DSR009C3" data-open="DSR009C3" class="link link-ref link-reveal xref-bibr">3</a></sup> For <em>A. sojae,</em> a termination point mutation in <em>aflR</em> which controls transcription of aflatoxin biosynthesis gene clusters and lack of the polyketide synthase (PKS) gene are correlated with its aflatoxin non-productivity.<sup><a reveal-id="DSR009C4" data-open="DSR009C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>The genome of <em>A. oryzae</em> RIB40 was recently completely sequenced and the comprehensive analysis showed that this strain possesses 134 protease genes including many paralogous genes and multiple copies of α-amylase and α-glucosidase genes.<sup><a reveal-id="DSR009C5" data-open="DSR009C5" class="link link-ref link-reveal xref-bibr">5</a></sup> These genetic features may account for its high proteolytic and amylolytic abilities in <em>A. oryzae</em> RIB40.</p><p>Many of the moulds classified in <em>Aspergillus</em> section <em>Flavi</em> are known to produce various secondary metabolites. The <em>A. oryzae</em> RIB40 genome encodes genes for numerous secondary metabolites other than aflatoxins,<sup><a reveal-id="DSR009C6" data-open="DSR009C6" class="link link-ref link-reveal xref-bibr">6</a></sup> although EST and microarray analysis of <em>A. oryzae</em> RIB40 suggested that it has almost no productivity of secondary metabolites.<sup><a reveal-id="DSR009C7" data-open="DSR009C7" class="link link-ref link-reveal xref-bibr">7</a></sup> These features of <em>A. oryzae</em> on the basis of quality, productivity, and safety may be one of the reasons for that <em>A. oryzae</em> strains have gradually been selected as industrially useful strains.<sup><a reveal-id="DSR009C8" data-open="DSR009C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>Though similar studies in <em>A. sojae</em> have not been conducted so much as <em>A. oryzae</em>, it is thought to be a domesticated strain selectively bred from natural strains as well as <em>A. oryzae</em> above.</p><p>However, the whole genetic information of <em>A. sojae</em> is insufficient to investigate the functional features important for its industrial use including the protease and amylase activities as well as safety. Therefore, we conducted the whole-genome sequencing of the practical strain <em>A. sojae</em> NBRC4239 isolated from Japanese soy sauce <em>koji</em> by using the next generation sequencer 454 pyrosequencer. The genetic information of <em>A. sojae</em> NBRC4239, combined with that of <em>A. oryzae</em>, will synergistically provide the knowledge for deep understanding of the biological nature of industrially important <em>koji</em> mould and for its further development of usefulness in food science field.</p>                            <h2 scrollto-destination=6645716 id="6645716" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6645717 id="6645717" class="section-title" data-legacy-id=s2a>2.1. Strain and DNA preparation</h3>
<p><em>Aspergillus sojae</em> NBRC4239 was obtained from NBRC (<a class="link link-uri" href="http://www.nbrc.nite.go.jp/">http://www.nbrc.nite.go.jp/</a>). This strain is a practical strain isolated from Japanese soy sauce <em>koji</em>.</p><p><em>Aspergillus sojae</em> NBRC4239 was incubated in PD liquid media (1% peptone, 2% dextrin, 0.5% KH<sub>2</sub>PO<sub>4</sub>, 0.1% NaNO<sub>3</sub>, 0.05% MgSO<sub>4</sub>, and 0.1% casamino acids, pH 6.0) on a shaker at 150 rpm at 30°C for 24 h. After collection on a mortar, mould was frozen in liquid nitrogen and then crushed with a pestle. The genome was extracted from this mould using a Wizard Genomic DNA Purification Kit (Promega Corporation, USA) and purified using a DNeasy Blood &amp; Tissue Kit (QIAGEN Sciences, USA), according to the respective manufacturer's protocols.</p>                            <h3 scrollto-destination=6645720 id="6645720" class="section-title" data-legacy-id=s2b>2.2. Genome sequencing and data assembly</h3>
<p>For GS FLX Titanium fragment sequencing, 500 ng of genomic DNA was sheared into DNA fragments ranging from 300 to 800 bp by nebulization. After both ends of the DNA fragments were repaired and phosphorylated, two types of adaptors (A and B) were ligated to the DNA fragments. Next, the DNA fragments carrying the 5′-biotin of adaptor B from the ligation mixture were immobilized onto magnetic streptavidin-coated beads. The single-stranded template DNA (ssDNA) molecules carrying Adaptor A at 5′-end and Adaptor B at 3′-end were isolated by alkaline denaturation. These purified ssDNAs were then hybridized to DNA capture beads and clonally amplified by an emulsion polymerase chain reaction (PCR) method. After denaturation of the amplified double-stranded DNAs on the capture beads, these beads with single-stranded molecules were spread onto each well of a pico titre plate. For GS FLX Titanium Paired-end sequencing, 15 μg of genomic DNA was sheared into DNA fragments ranging from ∼8 kb by fragmentation. After the ends of the DNA fragments were repaired and internal adaptors were ligated to the DNA fragments, each DNA fragment was circularized and self-ligated. The circular DNA was sheared into DNA fragments ranging from 300 to 800 bp by nebulization. The DNA fragments carrying the 5′-biotin of internal adaptor from the ligation mixture were immobilized onto magnetic streptavidin-coated beads. Paired-end sequencing was carried out similar to the method described above. Two sequencing runs in total were carried out. The GS FLX sequence data were assembled using Newbler assembly software.</p>                            <h3 scrollto-destination=6645722 id="6645722" class="section-title" data-legacy-id=s2c>2.3. Gene prediction and annotation</h3>
<p>GlimmerHMM,<sup><a reveal-id="DSR009C9" data-open="DSR009C9" class="link link-ref link-reveal xref-bibr">9</a></sup> AUGUSTUS,<sup><a reveal-id="DSR009C10" data-open="DSR009C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSR009C11" data-open="DSR009C11" class="link link-ref link-reveal xref-bibr">11</a></sup> SNAP,<sup><a reveal-id="DSR009C12" data-open="DSR009C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and GeneMark + ES<sup><a reveal-id="DSR009C13" data-open="DSR009C13" class="link link-ref link-reveal xref-bibr">13</a></sup> were used as <em>ab initio</em> predictors, and Genewise<sup><a reveal-id="DSR009C14" data-open="DSR009C14" class="link link-ref link-reveal xref-bibr">14</a></sup> was used as the evidence-based predictor. The <em>ab initio</em> GlimmerHMM, AUGUSTUS, and SNAP parameters were trained on all <em>A. oryzae</em> RIB40 gene models, while GeneMark + ES performed an iterative self-training procedure. The amino acid sequence of <em>A. oryzae</em> RIB40 was used for alignment by Genewise. GFF files obtained from these prediction programmes were incorporated by Evigan<sup><a reveal-id="DSR009C15" data-open="DSR009C15" class="link link-ref link-reveal xref-bibr">15</a></sup> to produce prediction results. Out of the predicted open reading frames (ORFs), those with more than 100 amino acid residues were selected as predictor genes.</p><p>Amino acid sequences of the predictor genes were matched with non-redundant protein database (nr, NCBI) by BLAST<sup><a reveal-id="DSR009C16" data-open="DSR009C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and were annotated based on identity. tRNA were predicted by tRNAscan-SE.<sup><a reveal-id="DSR009C17" data-open="DSR009C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p>                            <h3 scrollto-destination=6645725 id="6645725" class="section-title" data-legacy-id=s2d>2.4. Comparative genomics</h3>
<p>Nucleotide and amino acid sequences of <em>A. oryzae</em> RIB40 were obtained from DOGAN (<a class="link link-uri" href="http://www.nbrc.nite.go.jp/dogan/">http://www.nbrc.nite.go.jp/dogan/</a>). Nucleotide and amino acid sequences of <em>A. flavus</em> NRRL3357, <em>A. fumigatus</em> Af293, and <em>A. nidulans</em> FGSC A4 were obtained from the <em>Aspergillus</em> Genome Database (AspGD).<sup><a reveal-id="DSR009C18" data-open="DSR009C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Putative domains were predicted with the HMMER<sup><a reveal-id="DSR009C19" data-open="DSR009C19" class="link link-ref link-reveal xref-bibr">19</a></sup> programme hmmscan using the hidden Markov models from the pfam database.<sup><a reveal-id="DSR009C20" data-open="DSR009C20" class="link link-ref link-reveal xref-bibr">20</a></sup> For <em>A. oryzae</em> RIB40, <em>A. flavus</em> NRRL3357, <em>A. fumigatus</em> Af293, and <em>A. nidulans</em> FGSC A4, ORFs with Pfam domain scores of &gt;150 were subject to comparison.</p>                            <h4 scrollto-destination=6645733 id="6645733" class="section-title" data-legacy-id=s2d1>2.4.1. Protease</h4>
<p>A domain list for protease was created by matching amino acid sequences registered in MEROPS<sup><a reveal-id="DSR009C21" data-open="DSR009C21" class="link link-ref link-reveal xref-bibr">21</a></sup> with the HMMER from the Pfam database. Based on this list, amino acid sequences with protease domain scores &gt;150 were compared between <em>A. sojae</em> NBRC4239 and <em>A. oryzae</em> RIB40. For phylogenetic analysis, multiple alignments were carried out by ClustalX<sup><a reveal-id="DSR009C22" data-open="DSR009C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and phylogenic trees were drawn with TreeView.<sup><a reveal-id="DSR009C23" data-open="DSR009C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                            <h4 scrollto-destination=6645745 id="6645745" class="section-title" data-legacy-id=s2d2>2.4.2. Amylolytic enzymes</h4>
<p>It is known that out of the glycoside hydrolases, families 13, 15, and 31 are involved in amylolysis. Entries of glycoside hydrolase belonging to these three families in relation to <em>A. oryzae</em> RIB40 were extracted from the Carbohydrate Active Enzyme (CAZy) database.<sup><a reveal-id="DSR009C24" data-open="DSR009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> From these data, amylolytic enzymes possessed by <em>A. sojae</em> NBRC4239 strains were predicted. Furthermore, for these predicted amylolytic enzymes, checks were made to confirm the presence of active centre residues (data not shown). Alignment of nucleotide and amino acid sequences were carried out by GENETYX, and BLAST was used for homology searches. PCR primer sequences used for partial nucleotide sequence checks are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h4 scrollto-destination=6645750 id="6645750" class="section-title" data-legacy-id=s2d3>2.4.3. Secondary metabolism</h4>
<p>Sequences of secondary metabolite gene clusters in <em>A. oryzae</em> RIB40 were obtained from the Secondary Metabolite Unknown Region Finder (SMURF) database.<sup><a reveal-id="DSR009C25" data-open="DSR009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Cyclopiazonic acid (CPA) biosynthesis gene cluster sequences of <em>A. flavus</em> were obtained from Broad Institute (<a class="link link-uri" href="http://www.broadinstitute.org/annotation/genome/Aspergillus_group">http://www.broadinstitute.org/annotation/genome/Aspergillus_group</a>) with reference to reports by Chang <em>et al</em>.<sup><a reveal-id="DSR009C26" data-open="DSR009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Sequences near the aflatrem gene clusters in <em>A. flavus</em> and <em>A. oryzae</em> were obtained from reports by Nicholson <em>et al</em>.,<sup><a reveal-id="DSR009C27" data-open="DSR009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> where the sequences for <em>A. flavus</em> NRRL3357 and <em>A. oryzae</em> RIB40 were obtained from Broad Institute and DOGAN, respectively. For <em>A. flavus</em> NRRL6541, sequences ATM1 (AY559849.2 GI:161621808) and ATM2 (AM921700.1 GI:162286818) entered in GenBank (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genbank">http://www.ncbi.nlm.nih.gov/genbank</a>) were used. Harr plots were generated by <em>In silico</em> Molecular Cloning Series IMC, genomics edition (In Silico Biology, Inc.). PCR primer sequences used for partial nucleotide sequence checks are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p>                            <h3 scrollto-destination=6645752 id="6645752" class="section-title" data-legacy-id=s2e>2.5. Accession numbers</h3>
<p>Nucleotide sequence data were entered into the DDBJ/EMBL/GenBank DNA databases. Accession numbers for the 65 scaffold sequences are DF093557–DF093585 and for the 1034 contig sequences are BACA01000001– BACA01001034.</p>                            <h2 scrollto-destination=6645754 id="6645754" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=6645755 id="6645755" class="section-title" data-legacy-id=s3a>3.1. Sequencing and assembly</h3>
<p>We obtained 1034 contigs (&gt;100 bp) and 65 scaffolds by assembling the reads obtained from sequencing (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The 65 scaffolds are composed of 557 contigs, thus 477 contigs did not make up scaffolds. Out of the 1034 contigs, 707 were &gt;500 bp. Total length of the contigs and scaffolds each exceeded 39 Mb. As the genome size reported for <em>A. oryzae</em> RIB40 is 37.6 Mb,<sup><a reveal-id="DSR009C5" data-open="DSR009C5" class="link link-ref link-reveal xref-bibr">5</a></sup> the genome of <em>A. sojae</em> NBRC4239 was predicted to exceed that of <em>A. oryzae</em> RIB40.</p>                            <h3 scrollto-destination=6645757 id="6645757" class="section-title" data-legacy-id=s3b>3.2. Gene prediction and annotation</h3>
<p>ORF prediction was carried out on the 65 scaffolds and the 477 contigs that did not make up scaffolds. As a result, we obtained 13 033 ORFs with amino acid residues &gt;100. Also, 275 tRNAs were predicted by tRNAscan-SE. Table <a reveal-id="DSR009TB1" data-open="DSR009TB1" class="link link-reveal link-table xref-fig">1</a> showed the comparison of ORFs of <em>A. oryzae</em> RIB40 and <em>A. sojae</em> NBRC4239. </p>                        <a id="6645759" scrollto-destination="6645759"></a>
<div content-id="DSR009TB1" class="table-modal"><div class="table-wrap"><div id="DSR009TB1" class="table-wrap-title" data-id="DSR009TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of ORFs of <em>A. oryzae</em> RIB40 and <em>A. sojae</em> NBRC4239</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>A. oryzae</em> RIB40 </th><th><em>A. sojae</em> NBRC4239 </th></tr></thead><tbody><tr><td>Size of assembly (MB) </td><td>37.6 </td><td>39.5 </td></tr><tr><td>GC content (%) </td><td>48.2 </td><td>48.1 </td></tr><tr><td>tRNA genes </td><td>270 </td><td>275 </td></tr><tr><td>Number of ORFs </td><td>12, 074 </td><td>13, 033 </td></tr><tr><td>Average ORF size </td><td>449.8 </td><td>455.9 </td></tr><tr><td>Min ORF size </td><td>101 </td><td>101 </td></tr><tr><td>Max ORF size </td><td>6,886 </td><td>7,566 </td></tr></tbody></table></div><div content-id="DSR009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>A. oryzae</em> RIB40 </th><th><em>A. sojae</em> NBRC4239 </th></tr></thead><tbody><tr><td>Size of assembly (MB) </td><td>37.6 </td><td>39.5 </td></tr><tr><td>GC content (%) </td><td>48.2 </td><td>48.1 </td></tr><tr><td>tRNA genes </td><td>270 </td><td>275 </td></tr><tr><td>Number of ORFs </td><td>12, 074 </td><td>13, 033 </td></tr><tr><td>Average ORF size </td><td>449.8 </td><td>455.9 </td></tr><tr><td>Min ORF size </td><td>101 </td><td>101 </td></tr><tr><td>Max ORF size </td><td>6,886 </td><td>7,566 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The methods of the sequencing and the ORF prediction procedure were different in <em>A. oryzae</em> and <em>A. sojae</em>, and the value 101, the minimum size, just indicated the artificial value for cutoff.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6645759" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR009TB1" class="table-wrap-title" data-id="DSR009TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of ORFs of <em>A. oryzae</em> RIB40 and <em>A. sojae</em> NBRC4239</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>A. oryzae</em> RIB40 </th><th><em>A. sojae</em> NBRC4239 </th></tr></thead><tbody><tr><td>Size of assembly (MB) </td><td>37.6 </td><td>39.5 </td></tr><tr><td>GC content (%) </td><td>48.2 </td><td>48.1 </td></tr><tr><td>tRNA genes </td><td>270 </td><td>275 </td></tr><tr><td>Number of ORFs </td><td>12, 074 </td><td>13, 033 </td></tr><tr><td>Average ORF size </td><td>449.8 </td><td>455.9 </td></tr><tr><td>Min ORF size </td><td>101 </td><td>101 </td></tr><tr><td>Max ORF size </td><td>6,886 </td><td>7,566 </td></tr></tbody></table></div><div content-id="DSR009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>A. oryzae</em> RIB40 </th><th><em>A. sojae</em> NBRC4239 </th></tr></thead><tbody><tr><td>Size of assembly (MB) </td><td>37.6 </td><td>39.5 </td></tr><tr><td>GC content (%) </td><td>48.2 </td><td>48.1 </td></tr><tr><td>tRNA genes </td><td>270 </td><td>275 </td></tr><tr><td>Number of ORFs </td><td>12, 074 </td><td>13, 033 </td></tr><tr><td>Average ORF size </td><td>449.8 </td><td>455.9 </td></tr><tr><td>Min ORF size </td><td>101 </td><td>101 </td></tr><tr><td>Max ORF size </td><td>6,886 </td><td>7,566 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The methods of the sequencing and the ORF prediction procedure were different in <em>A. oryzae</em> and <em>A. sojae</em>, and the value 101, the minimum size, just indicated the artificial value for cutoff.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6645759" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6645760 id="6645760" class="section-title" data-legacy-id=s3c>3.3. Comparative genomics</h3>
<p>The proportion of ORFs with Pfam domain scores of &gt;150 was 2847/13 033 (21.8%) for <em>A. sojae</em> NBRC4239, 2868/12 074 (23.8%) for <em>A. oryzae</em> RIB40, 2880/12 604 (22.9%) for <em>A. flavus</em> NRRL3357, 2517/9887 (25.5%) for <em>A. fumigatus</em> Af293, and 2641/11 272 (23.4%) for <em>A. nidulans</em> FGSC A4.</p><p>ORFs with Pfam domain scores of  &gt;150 were compared between <em>A. sojae</em> NBRC4239 and <em>A. oryzae</em> RIB40 by BLASTP, and it was found that 2326/2847 (81.7%) had &gt;90% identity (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1a</a>). Next, 397/2847 ORFs from <em>A. sojae</em> NBRC4239 were found to have &lt;70% identity with those from <em>A. oryzae</em> RIB40. Of those, 134 ORFs had &gt;90% identity and 192 ORFs had &lt;70% identity in the nucleotide sequence to the corresponding regions in <em>A. oryzae</em> RIB40 by tBLASTN. Therefore, the 134 ORFs found in <em>A. sojae</em> NBRC4239 may have been missed in the gene prediction of <em>A. oryzae</em> RIB40, and the orthologous ORFs to the 192 ORFs may be absent in <em>A. oryzae</em> RIB40. These 192 ORFs in <em>A. sojae</em> NBRC4239 were matched with <em>A. flavus</em> NRRL3357 genes with domain scores &gt;150 using BLASTP. As a result, 22 had &gt;90% identity and 170 had &lt;70% identity. These results indicated that the 22 ORFs are present in both <em>A. sojae</em> NBRC4239 and <em>A. flavus</em> NRRL3357 but absent in <em>A. oryzae</em> RIB40. The 170 ORFs with &lt;70% identity were matched with nr of the NCBI database by BLASTP, and 132 ORFs were found to have &lt;70% identity. Thus, these 132 ORFs may be unique to <em>A. sojae</em> NBRC4239 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1b</a>).</p>                            <h4 scrollto-destination=6645763 id="6645763" class="section-title" data-legacy-id=s3c1>3.3.1. Protease</h4>
<p>Out of the 2847 ORFs of <em>A. sojae</em> NBRC4239 and 2868 of <em>A. oryzae</em> RIB40, 83 ORFs in <em>A. oryzae</em> RIB40 and 76 in <em>A. sojae</em> NBRC4239 had protease domain scores &gt;150; thus, <em>A. sojae</em> NBRC4239 had seven fewer ORFs. The total number of predicted protease genes in <em>A. oryzae</em> RIB40 was considerably less than the reported 134,<sup><a reveal-id="DSR009C5" data-open="DSR009C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and this difference is likely due to the strict domain score set at &gt;150.</p><p>ORFs with domain scores &gt;150 from <em>A. sojae</em> NBRC4239 and <em>A. oryzae</em> RIB40 were sorted and compared by domains. The number of proteases with specific domains was not very different in either species. Four types of proteases in <em>A. sojae</em> NBRC4239 had one more gene than those in <em>A. oryzae</em> RIB40, respectively. Eleven types of proteases in <em>A. sojae</em> NBRC4239 had one gene less than those in <em>A. oryzae</em> RIB40, respectively. In both <em>A. oryzae</em> RIB40 and <em>A. sojae</em> NBRC4239, serine carboxypeptidases were most abundant, followed by aspartic proteases. <em>Aspergillus sojae</em> NBRC4239 had 13 serine carboxypeptidases, which was one more than <em>A. oryzae</em> RIB40, and it had seven aspartic proteases which was two less than <em>A. oryzae</em> RIB40.</p><p>Under the strict condition of protease domain scores &gt;150, we found no significant difference in the number of protease genes between the two species. However, by using a less strict condition, a difference in the number of protease genes may be observed between <em>A. sojae</em> and <em>A. oryzae</em>.</p>                            <h5 scrollto-destination=6645767 id="6645767" class="section-title" data-legacy-id=s3c1a>3.3.1.1. Serine carboxypeptidase</h5>
<p>Phylogenic tree for serine carboxypeptidases based on sequences with domain scores &gt;150 was constructed and then compared for <em>A. sojae</em> NBRC4239, <em>A. oryzae</em> RIB40, <em>A. flavus</em> NRRL3357, <em>A. fumigatus</em> Af293, and <em>A. nidulans</em> FGSC A4 (Fig. <a reveal-id="DSR009F1" data-open="DSR009F1" class="link link-reveal link-table xref-fig">1</a>). We found that <em>A. sojae</em> NBRC4239 possesses a serine carboxypeptidase gene (scaffold00048.369) that has low sequence similarity with the other four species. This gene was also included in the 132 ORFs that had &lt;70% identity against nr (refer to the ‘Comparative genomics’ section). The similarity search of scaffold00048.369 against the ORFs of <em>A. oryzae</em> RIB40 by BLASTP identified a gene AO090103000026 with the closest match of 56% identity. AO090103000026 was annotated as a serine carboxypeptidase in <em>A. oryzae</em> RIB40 (Fig. <a reveal-id="DSR009F1" data-open="DSR009F1" class="link link-reveal link-table xref-fig">1</a>). In addition, Scaffold00048.369 was found to have the highest similarity to carboxypeptidase S1 in <em>Neosartorya fischeri</em> NRRL181 with 58% identity by searching against the nr NCBI database by BLASTP. From these results, scaffold00048.369 is likely to be a serine carboxypeptidase unique to <em>A. sojae</em> NBRC4239. We confirmed the expression of scaffold00048.369 by RT–PCR for mRNA isolated from <em>A. sojae</em> NBRC4239 incubated in wheat bran media. The sequencing of the RT–PCR product revealed a sequence identical to the predicted ORF, indicating that our ORF prediction for scaffold00048.36 was correct, and this gene is expressed in wheat bran media (data not shown). </p>                        <a id="6645769" scrollto-destination="6645769"></a>
<div id="DSR009F1" data-id="DSR009F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00901.jpeg?Expires=2147483647&Signature=ZibGZsit8J8VCHnnh6DyJWlWdSV-jTr2Cc2AxML2lz-tEK~l6G8jX4opP5BtuvBIf3iLihOByJ6esRsPaPgQGQCLnyVxoYScdn-7y6rWGXT9uo5-uHgjA6dd7Qh2G0ZV4013Lx6xowtyKplCj7JAUKb~JgycmFiBoGF57Ssgqn6oy5DFPL8SU6TvQDkhCwNhiprMdppkfHWp8g8yYZ0LOHHWtPUW3TAxT5vAT-PhQ~bFXey16NNqe~QovAtHaefB8-uMTIomHQt2Auw56VOHwkL0iQUMPaLd6FHdT86zJd9Tb6MsPHKsC~kKdPXMTKC1CyOCsbGhR-vlst8E1sQiLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenic analysis of serine carboxypeptidase in five Aspergillus species. ORFs of serine carboxypeptidase with Pfam domain scores of &gt;150 were analysed by ClustalW and were drawn by TreeView for five Aspergillus species. Filled circles, A. sojae; filled squares, A. oryzae; filled diamonds, A. flavus; triangle, A. nidulans; inverted triangles, A. fumigatus. Blue lines indicate the genes in common with all five Aspergillus species. Green lines indicate the genes in common with three species in Aspergillus section Flavi. Red shows the gene unique to A. sojae. Oranges show extra homologues in A. oryzae reported in Machida et al.6" path-from-xml="dsr00901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645769/dsr00901.jpeg" path-from-xml="dsr00901.jpeg" target="_blank">View large</a><a section="6645769" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00901.jpeg?Expires=2147483647&Signature=vT94VTgiKiNT5-jxv18B2N0Vsf1wgJdEtQ1jfHBBO1EUYRu6k4a4CAAmuDkdNUhaIeRycFU0-3vS1ULPsAi1X7fy-ui~WsaHg07Z3cLtdqfz7BGOGGiQ0VelELQd-xV9BDFIo1UkIxz2fwkFFL-snK5arS-JaYEL6zX-nzGnOwjbtuUGwYlOZ9Fjq5JMhGob3YUSo8RFLZGNiF-C0oDuxpWASTu8ldD9c2sGbginUj3jUggFjin0lS85SkdbXrFKwBKtobJmMA9XUt4XrK6mZ4N-4cGyZn85Q2s7eUvVtVu0Y8q4O40jqsSFwdGGH-hZjsGVuhLK1q35oVCJDTyvGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645769&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenic analysis of serine carboxypeptidase in five <em>Aspergillus</em> species. ORFs of serine carboxypeptidase with Pfam domain scores of &gt;150 were analysed by ClustalW and were drawn by TreeView for five <em>Aspergillus</em> species. Filled circles, <em>A. sojae</em>; filled squares, <em>A. oryzae</em>; filled diamonds, <em>A. flavus</em>; triangle, <em>A. nidulans</em>; inverted triangles, <em>A. fumigatus.</em> Blue lines indicate the genes in common with all five <em>Aspergillus</em> species. Green lines indicate the genes in common with three species in <em>Aspergillus</em> section <em>Flavi</em>. Red shows the gene unique to <em>A. sojae</em>. Oranges show extra homologues in <em>A. oryzae</em> reported in Machida <em>et al</em>.<sup><a reveal-id="DSR009C6" data-open="DSR009C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div><div content-id="DSR009F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00901.jpeg?Expires=2147483647&Signature=ZibGZsit8J8VCHnnh6DyJWlWdSV-jTr2Cc2AxML2lz-tEK~l6G8jX4opP5BtuvBIf3iLihOByJ6esRsPaPgQGQCLnyVxoYScdn-7y6rWGXT9uo5-uHgjA6dd7Qh2G0ZV4013Lx6xowtyKplCj7JAUKb~JgycmFiBoGF57Ssgqn6oy5DFPL8SU6TvQDkhCwNhiprMdppkfHWp8g8yYZ0LOHHWtPUW3TAxT5vAT-PhQ~bFXey16NNqe~QovAtHaefB8-uMTIomHQt2Auw56VOHwkL0iQUMPaLd6FHdT86zJd9Tb6MsPHKsC~kKdPXMTKC1CyOCsbGhR-vlst8E1sQiLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenic analysis of serine carboxypeptidase in five Aspergillus species. ORFs of serine carboxypeptidase with Pfam domain scores of &gt;150 were analysed by ClustalW and were drawn by TreeView for five Aspergillus species. Filled circles, A. sojae; filled squares, A. oryzae; filled diamonds, A. flavus; triangle, A. nidulans; inverted triangles, A. fumigatus. Blue lines indicate the genes in common with all five Aspergillus species. Green lines indicate the genes in common with three species in Aspergillus section Flavi. Red shows the gene unique to A. sojae. Oranges show extra homologues in A. oryzae reported in Machida et al.6" path-from-xml="dsr00901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645769/dsr00901.jpeg" path-from-xml="dsr00901.jpeg" target="_blank">View large</a><a section="6645769" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00901.jpeg?Expires=2147483647&Signature=vT94VTgiKiNT5-jxv18B2N0Vsf1wgJdEtQ1jfHBBO1EUYRu6k4a4CAAmuDkdNUhaIeRycFU0-3vS1ULPsAi1X7fy-ui~WsaHg07Z3cLtdqfz7BGOGGiQ0VelELQd-xV9BDFIo1UkIxz2fwkFFL-snK5arS-JaYEL6zX-nzGnOwjbtuUGwYlOZ9Fjq5JMhGob3YUSo8RFLZGNiF-C0oDuxpWASTu8ldD9c2sGbginUj3jUggFjin0lS85SkdbXrFKwBKtobJmMA9XUt4XrK6mZ4N-4cGyZn85Q2s7eUvVtVu0Y8q4O40jqsSFwdGGH-hZjsGVuhLK1q35oVCJDTyvGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645769&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenic analysis of serine carboxypeptidase in five <em>Aspergillus</em> species. ORFs of serine carboxypeptidase with Pfam domain scores of &gt;150 were analysed by ClustalW and were drawn by TreeView for five <em>Aspergillus</em> species. Filled circles, <em>A. sojae</em>; filled squares, <em>A. oryzae</em>; filled diamonds, <em>A. flavus</em>; triangle, <em>A. nidulans</em>; inverted triangles, <em>A. fumigatus.</em> Blue lines indicate the genes in common with all five <em>Aspergillus</em> species. Green lines indicate the genes in common with three species in <em>Aspergillus</em> section <em>Flavi</em>. Red shows the gene unique to <em>A. sojae</em>. Oranges show extra homologues in <em>A. oryzae</em> reported in Machida <em>et al</em>.<sup><a reveal-id="DSR009C6" data-open="DSR009C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div><p>Phylogenic analysis showed that serine carboxypeptidase genes are classified into five clusters that are in common with the five <em>Aspergillus</em> species (C1–5), and seven clusters that are in common only with three species of <em>Aspergillus</em> section <em>Flavi</em> (F1–7). Three of the five common clusters contained serine carboxypeptidase genes that are unique to <em>Aspergillus</em> section <em>Flavi</em>, in addition to the putative orthologous genes. The serine carboxypeptidases unique to <em>A. oryzae</em> RIB40 previously reported<sup><a reveal-id="DSR009C6" data-open="DSR009C6" class="link link-ref link-reveal xref-bibr">6</a></sup> are therefore considered to be in common with the <em>Aspergillus</em> section <em>Flavi</em>. It is conceivable that <em>A. sojae</em> and <em>A. oryzae</em> have been used widely in <em>miso</em> and soy sauce fermentation because of their possession of highly similar protease genes.</p>                            <h5 scrollto-destination=6645771 id="6645771" class="section-title" data-legacy-id=s3c1b>3.3.1.2. Aspartic protease</h5>
<p>Phylogenic tree for aspartic proteases based on sequences with domain scores of &gt;150 was constructed and then compared for <em>A. sojae</em> NBRC4239, <em>A. oryzae</em> RIB40, <em>A. flavus</em> NRRL3357, <em>A. fumigatus</em> Af293, and <em>A. nidulans</em> FGSC A4 (Fig. <a reveal-id="DSR009F2" data-open="DSR009F2" class="link link-reveal link-table xref-fig">2</a>). We found that <em>A. sojae</em> NBRC4239 possesses an aspartic protease gene (scaffold00063.1451) that showed low in sequence similarity with those in the other four species. This gene was also included in the 132 ORFs that had &lt;70% identity against nr (refer to the ‘Comparative genomics’ section). Scaffold00063.1451 was searched against the ORFs of <em>A. oryzae</em> RIB40 by BLASTP, and AO090701000002 was found to be the closest gene with 33% identity. AO090701000002 was annotated as an aspartic protease in <em>A. oryzae</em> RIB40 (Fig. <a reveal-id="DSR009F2" data-open="DSR009F2" class="link link-reveal link-table xref-fig">2</a>). Similarity search of scaffold00063.1451 identified yapsin of <em>Penicillium marneffei</em> ATCC18224 with the highest similarity of 46% identity against the nr NCBI database by BLASTP. From these results, scaffold00063.1451 is likely to be an aspartic protease unique to <em>A. sojae</em> NBRC4239. We confirmed the expression of scaffold00063.1451 by RT–PCR followed by sequencing, as described in the ‘Serine carboxypeptidase' section, indicating that our ORF prediction for scaffold00063.1451 was correct, and this gene is expressed in wheat bran media (data not shown). </p>                        <a id="6645773" scrollto-destination="6645773"></a>
<div id="DSR009F2" data-id="DSR009F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00902.jpeg?Expires=2147483647&Signature=xv88EgBZERCW1ylrQnxebLoQE5nglq0aH5Y7FzETzf0WKjZw62VIicTuegV-6mP5kBQVW7NQeslfCobV6fqbu-PYtVccRZVNs~4llrHU7KOxEXb-~oZaTg7wW3gszhyOzeUDaDYiYgCKfPVcjEPoGnzGdRutAjcODOBWhgz3QdXr1Kj6w~NUvZBLe8KaIgsVqhFrT~Vyae7ozWjflNPaeBn6JdRMN6WTdUOanjNQUSTDIxessbIzt4GTmvC8hBvTAEWy3-aqSFnc1O0X2liPkb1WJ296G3z7QWjGhfRlSgXuJ7hj0CIrstqcqWm97T6RFpBwhXid4AiUK86PZ7usBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenic analysis of aspartic protease in five Aspergillus species. ORFs of aspartic protease with Pfam domain scores of &gt;150 were analysed by ClustalW and were drawn by TreeView for 5 Aspergillus species. Filled circles, A. sojae; filled squares, A. oryzae; filled diamonds, A. flavus; triangle, A. nidulans; inverted triangles, A. fumigatus. Blue lines indicate the genes in common with all five Aspergillus species. Green line indicates the genes in common with three species in Aspergillus section Flavi. Red shows the gene unique to A. sojae. Oranges show extra homologues in A. oryzae reported in Machida et al.6" path-from-xml="dsr00902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645773/dsr00902.jpeg" path-from-xml="dsr00902.jpeg" target="_blank">View large</a><a section="6645773" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00902.jpeg?Expires=2147483647&Signature=sshVh3RlMdKhYTHONgg-psq71-eRWEij6qa9YLrYcM3JaHJgWPaK3rKicelPysoIUjSJLxWdTUI2N06mcLD8dhn4626Nx3OmML7GL~2kPyyvrcTJysBESkEgBGcsbG3m6Xt8yRKc6pUnhO1bGlxl8BOKo0mudyPk-i6n9L2Y1ddMwMo3cFlc-94Q6zDcTLFeUsUkRQ8kqfIrkK6inzCGn3evu9bA3kY9rcI0FvvAHtbEyV8jtc1l-aW0K9-0ZCqf2hjt~AFkmPVpdDoFF~0OZaDq31bvbl4cq01VttYMaUMfg4xJ86B7co3RzuogET-jXwpUaYCjOF1koP55xrgJNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645773&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenic analysis of aspartic protease in five <em>Aspergillus</em> species. ORFs of aspartic protease with Pfam domain scores of &gt;150 were analysed by ClustalW and were drawn by TreeView for 5 <em>Aspergillus</em> species. Filled circles, <em>A. sojae</em>; filled squares, <em>A. oryzae</em>; filled diamonds, <em>A. flavus</em>; triangle, <em>A. nidulans</em>; inverted triangles, <em>A. fumigatus</em><em>.</em> Blue lines indicate the genes in common with all five <em>Aspergillus</em> species. Green line indicates the genes in common with three species in <em>Aspergillus</em> section <em>Flavi</em>. Red shows the gene unique to <em>A. sojae</em>. Oranges show extra homologues in <em>A. oryzae</em> reported in Machida <em>et al</em>.<sup><a reveal-id="DSR009C6" data-open="DSR009C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div><div content-id="DSR009F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00902.jpeg?Expires=2147483647&Signature=xv88EgBZERCW1ylrQnxebLoQE5nglq0aH5Y7FzETzf0WKjZw62VIicTuegV-6mP5kBQVW7NQeslfCobV6fqbu-PYtVccRZVNs~4llrHU7KOxEXb-~oZaTg7wW3gszhyOzeUDaDYiYgCKfPVcjEPoGnzGdRutAjcODOBWhgz3QdXr1Kj6w~NUvZBLe8KaIgsVqhFrT~Vyae7ozWjflNPaeBn6JdRMN6WTdUOanjNQUSTDIxessbIzt4GTmvC8hBvTAEWy3-aqSFnc1O0X2liPkb1WJ296G3z7QWjGhfRlSgXuJ7hj0CIrstqcqWm97T6RFpBwhXid4AiUK86PZ7usBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenic analysis of aspartic protease in five Aspergillus species. ORFs of aspartic protease with Pfam domain scores of &gt;150 were analysed by ClustalW and were drawn by TreeView for 5 Aspergillus species. Filled circles, A. sojae; filled squares, A. oryzae; filled diamonds, A. flavus; triangle, A. nidulans; inverted triangles, A. fumigatus. Blue lines indicate the genes in common with all five Aspergillus species. Green line indicates the genes in common with three species in Aspergillus section Flavi. Red shows the gene unique to A. sojae. Oranges show extra homologues in A. oryzae reported in Machida et al.6" path-from-xml="dsr00902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645773/dsr00902.jpeg" path-from-xml="dsr00902.jpeg" target="_blank">View large</a><a section="6645773" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00902.jpeg?Expires=2147483647&Signature=sshVh3RlMdKhYTHONgg-psq71-eRWEij6qa9YLrYcM3JaHJgWPaK3rKicelPysoIUjSJLxWdTUI2N06mcLD8dhn4626Nx3OmML7GL~2kPyyvrcTJysBESkEgBGcsbG3m6Xt8yRKc6pUnhO1bGlxl8BOKo0mudyPk-i6n9L2Y1ddMwMo3cFlc-94Q6zDcTLFeUsUkRQ8kqfIrkK6inzCGn3evu9bA3kY9rcI0FvvAHtbEyV8jtc1l-aW0K9-0ZCqf2hjt~AFkmPVpdDoFF~0OZaDq31bvbl4cq01VttYMaUMfg4xJ86B7co3RzuogET-jXwpUaYCjOF1koP55xrgJNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645773&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenic analysis of aspartic protease in five <em>Aspergillus</em> species. ORFs of aspartic protease with Pfam domain scores of &gt;150 were analysed by ClustalW and were drawn by TreeView for 5 <em>Aspergillus</em> species. Filled circles, <em>A. sojae</em>; filled squares, <em>A. oryzae</em>; filled diamonds, <em>A. flavus</em>; triangle, <em>A. nidulans</em>; inverted triangles, <em>A. fumigatus</em><em>.</em> Blue lines indicate the genes in common with all five <em>Aspergillus</em> species. Green line indicates the genes in common with three species in <em>Aspergillus</em> section <em>Flavi</em>. Red shows the gene unique to <em>A. sojae</em>. Oranges show extra homologues in <em>A. oryzae</em> reported in Machida <em>et al</em>.<sup><a reveal-id="DSR009C6" data-open="DSR009C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div><p>Phylogenic analysis showed that aspartic protease genes are classified into four clusters that are in common with the five <em>Aspergillus</em> species (C1–4). Only one cluster was in common with only the three species belonging to <em>Aspergillus</em> section <em>Flavi</em> (F1), and other three clusters were shared with two of the three species. These data suggested that aspartic protease genes are less conserved among <em>Aspergillus</em> section <em>Flavi</em>, in contrast to serine carboxypeptidase genes.</p>                            <h4 scrollto-destination=6645775 id="6645775" class="section-title" data-legacy-id=s3c2>3.3.2. Amylolytic enzymes</h4>
                            <h5 scrollto-destination=6645776 id="6645776" class="section-title" data-legacy-id=s3c2a>3.3.2.1. Amylolytic enzymes in A. sojae NBRC4239 and A. oryzae RIB40</h5>
<p>It is known in general that <em>A. sojae</em> has lower amylolytic activity compared with <em>A. oryzae.</em> We studied the genes for amylolytic enzymes in <em>A. oryzae</em> RIB40 and <em>A. sojae</em> NBRC4239 to analyse this difference. First, we compared the number of glycoside hydrolases belonging to Family 13, 15, and 31 in both strains, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). We found no difference in gene numbers of glycoside hydrolases between <em>A. sojae</em> and <em>A. oryzae</em> for Family 31 including α-glucosidase (EC.3.2.1.20), and for Family 15 including glucoamylase (EC.3.2.1.3). Thus, there is unlikely to be a difference in these enzymatic activities between the two <em>Aspergillus</em> strains. In contrast, we found that <em>A. sojae</em> has two copies less glycoside hydrolases in Family 13 including α-amylase (EC.3.2.1.1) than those in <em>A. oryzae.</em> Missing of the two genes in <em>A. sojae</em> was due to the copy number variation between the two strains; <em>A. sojae</em> only has one copy of <em>amyB</em> compared with the three copies (AO090023000944: <em>amy1</em>, AO090120000196: <em>amy2</em>, and AO090003001210: <em>amy3</em>) in <em>A. oryzae</em>.<sup><a reveal-id="DSR009C28" data-open="DSR009C28" class="link link-ref link-reveal xref-bibr">28</a></sup> In <em>A. oryzae</em>, <em>amyB</em> codes for so-called Taka-amylase, which is important for amylolysis. Therefore, a decreased copy number of <em>amyB</em> orthologues in <em>A. sojae</em> likely accounts for the lower amylolytic ability of <em>A. sojae</em> than that of <em>A. oryzae</em>.</p>                            <h5 scrollto-destination=6645778 id="6645778" class="section-title" data-legacy-id=s3c2b>3.3.2.2. α-Amylase genes and their flanking regions</h5>
<p>The above-mentioned three α-amylase genes and their flanking 20-kb regions of <em>A. oryzae</em> were further compared with the corresponding regions in <em>A. sojae</em> NBRC4239 scaffolds. We investigated whether the difference in α-amylase gene copy numbers results from the difference in genomic structures. The results are shown in Fig. <a reveal-id="DSR009F3" data-open="DSR009F3" class="link link-reveal link-table xref-fig">3</a>. In the <em>A. sojae amy1</em> region, the 12.5-kb sequence including 2.2 kb of <em>amy1</em> ORF, and its promoter and terminator regions were absent. Instead of the 12.5-kb region, a unique 2.9-kb sequence excluding <em>amy1</em> was present. This was also the case for <em>amy2</em>, where the 12.4-kb region including <em>amy2</em> ORF, and its promoter and terminator regions were absent, but a sequence unique as observed in the <em>amy1</em> region was not present in <em>amy2</em>. Furthermore, a 7.2-kb region was also absent and an inverted region was observed near the missing <em>amy2</em> regions in <em>A. sojae</em>. These results indicated that this region of the <em>A. sojae</em> genome was structurally rearranged. In the <em>amy3</em> region, we found that <em>amy3</em> structural genes and its terminator regions were conserved between <em>A. oryzae</em> and <em>A. sojae</em>. However, a 1.9-kb insertion sequence was found at 0.53 kb upstream of the translation initiation site in the <em>A. oryzae amy3</em> promoter. </p>                        <a id="6645780" scrollto-destination="6645780"></a>
<div id="DSR009F3" data-id="DSR009F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00903.jpeg?Expires=2147483647&Signature=0ttGMA1XPPn0u1Rgt5T3W8Oa6n8rWRlrcL1eVBPdG~wcf~2RMuNPB-S~pLtl~l9ZZYE~vPKxineFql5gbAhMLi7yMKgaardDbe2c8x9UIVYz-rUh9NifP4DTMMj~2YJOpeB1dl6Ao3KlFdDckaFOsQ~yP1Qg1scTmfBICi0rXFuHQ7GmKNZ~yr26GOhKyFnliLtVGHXfpeonab-CFYLnEinLNTfOAsvHWabivPqzzCeZoV7x1ryFYjbeWJ6SbLisYo2O6zSuPgZFp1J1RVxVzTsGYR07eehPxL78peOBXN6aRMHeDDI~Ej4Wx2gfZQ8unKbOX0Z8A3SboNHOUN4-Dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of α-amylase genes and their flanking regions. Map of α-amylase and flanking regions. Grey box, conserved regions; shaded box, regions unique to A. sojae; white box, regions unique to A. oryzae ; black box, inversed regions; black arrow, α-amylase gene ORF; black triangle, PCR primer-binding site." path-from-xml="dsr00903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645780/dsr00903.jpeg" path-from-xml="dsr00903.jpeg" target="_blank">View large</a><a section="6645780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00903.jpeg?Expires=2147483647&Signature=TeEDFi7ND6KQx-PKJRaC8I7ruMatzNvHYQblkFTWO9Sh6KZLWpbHjEzvT9cwWLMqCeyw3DkigsVDY9voob4puS-bxPlIEn1lFFnMG0KevNyb3h7RQyU9EInPdTQTnYqYtInjFgb-erQq7hKnlHAqXXUv1M~f39N7Ahb20tPAhGJamp~eBTzzhgSxFsQCRvgIHIrEctCaSSb59uk6lG7ZOHlA1D9QYGPVzDUFicSBxsAEsPA9zfnq-w~WOsc~VlqsyMldaukpHlSsCKfgVlFwMiOUYV7rORCNFMpPmQIl9bv~sfBrkoF9ksDNvkQOZuWJcFTC5FpJCEUp5LtQE4ennw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645780&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of α-amylase genes and their flanking regions. Map of α-amylase and flanking regions. Grey box, conserved regions; shaded box, regions unique to <em>A. sojae</em>; white box, regions unique to <em>A. oryzae</em> ; black box, inversed regions; black arrow, α-amylase gene ORF; black triangle, PCR primer-binding site.</p></div></div><div content-id="DSR009F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00903.jpeg?Expires=2147483647&Signature=0ttGMA1XPPn0u1Rgt5T3W8Oa6n8rWRlrcL1eVBPdG~wcf~2RMuNPB-S~pLtl~l9ZZYE~vPKxineFql5gbAhMLi7yMKgaardDbe2c8x9UIVYz-rUh9NifP4DTMMj~2YJOpeB1dl6Ao3KlFdDckaFOsQ~yP1Qg1scTmfBICi0rXFuHQ7GmKNZ~yr26GOhKyFnliLtVGHXfpeonab-CFYLnEinLNTfOAsvHWabivPqzzCeZoV7x1ryFYjbeWJ6SbLisYo2O6zSuPgZFp1J1RVxVzTsGYR07eehPxL78peOBXN6aRMHeDDI~Ej4Wx2gfZQ8unKbOX0Z8A3SboNHOUN4-Dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of α-amylase genes and their flanking regions. Map of α-amylase and flanking regions. Grey box, conserved regions; shaded box, regions unique to A. sojae; white box, regions unique to A. oryzae ; black box, inversed regions; black arrow, α-amylase gene ORF; black triangle, PCR primer-binding site." path-from-xml="dsr00903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645780/dsr00903.jpeg" path-from-xml="dsr00903.jpeg" target="_blank">View large</a><a section="6645780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00903.jpeg?Expires=2147483647&Signature=TeEDFi7ND6KQx-PKJRaC8I7ruMatzNvHYQblkFTWO9Sh6KZLWpbHjEzvT9cwWLMqCeyw3DkigsVDY9voob4puS-bxPlIEn1lFFnMG0KevNyb3h7RQyU9EInPdTQTnYqYtInjFgb-erQq7hKnlHAqXXUv1M~f39N7Ahb20tPAhGJamp~eBTzzhgSxFsQCRvgIHIrEctCaSSb59uk6lG7ZOHlA1D9QYGPVzDUFicSBxsAEsPA9zfnq-w~WOsc~VlqsyMldaukpHlSsCKfgVlFwMiOUYV7rORCNFMpPmQIl9bv~sfBrkoF9ksDNvkQOZuWJcFTC5FpJCEUp5LtQE4ennw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645780&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of α-amylase genes and their flanking regions. Map of α-amylase and flanking regions. Grey box, conserved regions; shaded box, regions unique to <em>A. sojae</em>; white box, regions unique to <em>A. oryzae</em> ; black box, inversed regions; black arrow, α-amylase gene ORF; black triangle, PCR primer-binding site.</p></div></div><p>We therefore confirmed that the genomic structures of the α-amylase regions predicted from genome analysis were correct for this strain by PCR (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). These results indicate that the difference in copy numbers of α-amylase genes between <em>A. sojae</em> NBRC4239 and <em>A. oryzae</em> RIB40 is a result of a rearrangement in genomic structure.</p>                            <h5 scrollto-destination=6645782 id="6645782" class="section-title" data-legacy-id=s3c2c>3.3.2.3. Analysis of transposons surrounding the α-amylase genes</h5>
<p>We investigated transposons existing near the α-amylase genes in <em>A. oryzae</em> and <em>A. sojae</em> (Fig. <a reveal-id="DSR009F4" data-open="DSR009F4" class="link link-reveal link-table xref-fig">4</a>A). We found that the ∼1.9-kb insertion sequence locating upstream the <em>A. oryzae amy3</em> promoter is a transposon <em>Tao1</em> (DDBJ/EMBL/GenBank accession number: AB021710.1). This transposon was flanked by inverted repeat sequences characteristic to ClassII DNA transposons (Fig. <a reveal-id="DSR009F4" data-open="DSR009F4" class="link link-reveal link-table xref-fig">4</a>A).<sup><a reveal-id="DSR009C29" data-open="DSR009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The <em>Tao1</em> insertion site at the <em>A. oryzae amy3</em> promoter region was found to correspond to the ‘TA’ sequence in the −533 to −532 region upstream the <em>A. sojae amy3</em> promoter. This is consistent with that ClassII transposons tend to be preferentially integrated at a TA sequence, resulting in target site TA duplication on both flanking of the integrated transposon.<sup><a reveal-id="DSR009C29" data-open="DSR009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The <em>Tao1</em> is located at the further upstream of the amylase transcription factor AmyR recognition site<sup><a reveal-id="DSR009C30" data-open="DSR009C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and the CreA recognition sites<sup><a reveal-id="DSR009C31" data-open="DSR009C31" class="link link-ref link-reveal xref-bibr">31</a></sup> involved in carbon catabolite repression (Fig. <a reveal-id="DSR009F4" data-open="DSR009F4" class="link link-reveal link-table xref-fig">4</a>B). It is not clear whether <em>Tao1</em> insertion affects the expression of the <em>A. oryzae amy3</em> gene or not, and further study will be needed to clarify the effect of <em>Tao1</em> insertion. </p>                        <a id="6645784" scrollto-destination="6645784"></a>
<div id="DSR009F4" data-id="DSR009F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00904.jpeg?Expires=2147483647&Signature=X~PhJNwawKBP~tHn6yRYi5XRsX7nlGP9jGOhotTAkvsUYLoFUOrGAod2v0bx33D34XD9GdlIqRC9Z-ve~qbSkYx37L88SQV6dXxmzRAtwsxCX9VLLlPqp~bEJN89XWIsVga7Mm6WGzopa6bVNjcZA9utbB2K2D2LOsfhVLagIr8qU0Rsj0p0~60SM0DEUB~LQqkBkbcDsHG3W0ggQDwEpHxQEx--w8IOGTZN0BJXpxoEzf9XEEZ7Hf39rZY54MvKIOFA6WQWXShk1Hs4fwNxc2wkXxssHEcs0XvUSdNY55vi-T0t8lMDxudXGq1XIAVouHhd20aLNp6BVfUDDm2X-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of transposons surrounding α-amylase genes in A. oryzae RIB40 and A. sojae NBRC4239. (A) Expected insertion site of A. oryzae Tao1 transposon. (B) Map of transposons surrounding A. oryzae and A. sojae α-amylase genes." path-from-xml="dsr00904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645784/dsr00904.jpeg" path-from-xml="dsr00904.jpeg" target="_blank">View large</a><a section="6645784" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00904.jpeg?Expires=2147483647&Signature=rB4-gCe7Roi0LpqMZ32xNh~ulyl56Y3fl14TFaxQcva7aULlM5nrlpNiaF2UPF3n87IHISMNPVxVHWnWvhU9rX2UEVG550KNaobNorRSwMrJPP8Uq5YifKW0JKEWX9DhZ8s6lhcSHSVGBAk4Pjer4U5YQsR3T48KN9aQJwhcLwg2QDEps46Xyp-gFxFe0IAOXgLSmllkfpGLAT-2XPZ4I-od-p13~8KXvd9j5kiqT6lmxE~Ob6cYYo3zHtSSPWFJlWplrA07a3Xyq3xFhLT~XZk8tp0J6CU25JKqvQUM6NaUiun25JY6dhNyzmUrQhYw3LD5HjUnqc7bpovmc~vRiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645784&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of transposons surrounding α-amylase genes in <em>A. oryzae</em> RIB40 and <em>A. sojae</em> NBRC4239. (<strong>A</strong>) Expected insertion site of <em>A. oryzae Tao1</em> transposon. (<strong>B</strong>) Map of transposons surrounding <em>A. oryzae</em> and <em>A. sojae α</em>-amylase genes.</p></div></div><div content-id="DSR009F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00904.jpeg?Expires=2147483647&Signature=X~PhJNwawKBP~tHn6yRYi5XRsX7nlGP9jGOhotTAkvsUYLoFUOrGAod2v0bx33D34XD9GdlIqRC9Z-ve~qbSkYx37L88SQV6dXxmzRAtwsxCX9VLLlPqp~bEJN89XWIsVga7Mm6WGzopa6bVNjcZA9utbB2K2D2LOsfhVLagIr8qU0Rsj0p0~60SM0DEUB~LQqkBkbcDsHG3W0ggQDwEpHxQEx--w8IOGTZN0BJXpxoEzf9XEEZ7Hf39rZY54MvKIOFA6WQWXShk1Hs4fwNxc2wkXxssHEcs0XvUSdNY55vi-T0t8lMDxudXGq1XIAVouHhd20aLNp6BVfUDDm2X-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of transposons surrounding α-amylase genes in A. oryzae RIB40 and A. sojae NBRC4239. (A) Expected insertion site of A. oryzae Tao1 transposon. (B) Map of transposons surrounding A. oryzae and A. sojae α-amylase genes." path-from-xml="dsr00904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645784/dsr00904.jpeg" path-from-xml="dsr00904.jpeg" target="_blank">View large</a><a section="6645784" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00904.jpeg?Expires=2147483647&Signature=rB4-gCe7Roi0LpqMZ32xNh~ulyl56Y3fl14TFaxQcva7aULlM5nrlpNiaF2UPF3n87IHISMNPVxVHWnWvhU9rX2UEVG550KNaobNorRSwMrJPP8Uq5YifKW0JKEWX9DhZ8s6lhcSHSVGBAk4Pjer4U5YQsR3T48KN9aQJwhcLwg2QDEps46Xyp-gFxFe0IAOXgLSmllkfpGLAT-2XPZ4I-od-p13~8KXvd9j5kiqT6lmxE~Ob6cYYo3zHtSSPWFJlWplrA07a3Xyq3xFhLT~XZk8tp0J6CU25JKqvQUM6NaUiun25JY6dhNyzmUrQhYw3LD5HjUnqc7bpovmc~vRiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645784&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of transposons surrounding α-amylase genes in <em>A. oryzae</em> RIB40 and <em>A. sojae</em> NBRC4239. (<strong>A</strong>) Expected insertion site of <em>A. oryzae Tao1</em> transposon. (<strong>B</strong>) Map of transposons surrounding <em>A. oryzae</em> and <em>A. sojae α</em>-amylase genes.</p></div></div><p>The <em>Tao1</em> transposon was also present within the promoters of <em>amy1</em> and <em>amy2</em> of <em>A. oryzae.</em> The insertion sites were identical to <em>amy3</em> insertion (Fig. <a reveal-id="DSR009F4" data-open="DSR009F4" class="link link-reveal link-table xref-fig">4</a>A). However, <em>Tao1</em> inserted in the <em>amy1</em> and <em>amy2</em> promoters were largely truncated and only partial sequences of 575 bp at the 3′-end of the total 1.9 kb sequence were left. The truncation of <em>Tao1</em> in <em>amy1</em> and <em>amy2</em> have occurred after triplication of the single <em>amy</em> gene having <em>Tao1</em> transposon, which inserted in the <em>A. oryzae</em> lineage after divergence of <em>A. sojae</em> from the common ancestor.</p><p>We also found a 981-bp ORF similar to <em>Ant1</em> transposase near <em>amy1</em> and <em>amy2</em> of <em>A. oryzae. Ant1</em> transposon is a member of ClassII DNA transposon classified in the <em>Tc1</em>/<em>mariner</em> group,<sup><a reveal-id="DSR009C29" data-open="DSR009C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSR009C32" data-open="DSR009C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and <em>Ant1</em> in <em>A. niger</em> is reported to have transfer activity.<sup><a reveal-id="DSR009C33" data-open="DSR009C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The corresponding <em>Ant1</em> transposase homologues were not found in the downstream regions of <em>amy3</em> of <em>A. oryzae</em> or <em>amyA</em> of <em>A. sojae</em>.</p><p>As shown above, several transposons, such as <em>Tao1</em> and <em>Ant1</em> transposase homologue, were found near the three α-amylase genes of <em>A. oryzae,</em> but were absent around <em>amyA</em> of <em>A. sojae.</em> The mechanism for the multiplication of α-amylase gene in <em>A. oryzae</em> is unclear, but these transposons might have a crucial role for the amylase gene multiplication in <em>A. oryzae</em>. The difference in α-amylase gene copy numbers might result in the difference in amylolytic activity between the two strains. This is likely to be a major factor for why <em>A. oryzae</em> became widely used in industry, such as in fermentation of <em>sake</em>, soy sauce, and <em>miso</em>, whereas <em>A. sojae</em> became used solely for soy sauce fermentation.</p>                            <h4 scrollto-destination=6645789 id="6645789" class="section-title" data-legacy-id=s3c3>3.3.3. Analysis of secondary metabolism-related genes</h4>
                            <h5 scrollto-destination=6645790 id="6645790" class="section-title" data-legacy-id=s3c3a>3.3.3.1. Comparison with secondary metabolite gene clusters in A. oryzae</h5>
<p>The 56 secondary metabolite cluster sequences were predicted using SMURF in the <em>A. oryzae</em> RIB40 genome.<sup><a reveal-id="DSR009C25" data-open="DSR009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> We analysed these clusters for the <em>A. sojae</em> NBRC4239 genome by Harr plots. Out of the 56 predicted secondary metabolite clusters, 24 clusters were found to be almost identical and the remaining 32 clusters differed from those in <em>A. oryzae</em> RIB40 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). <em>Aspergillus sojae</em> NBRC4239 had no sequence homologous to Cluster 51 located on the end of chromosome 5 in <em>A. oryzae.</em> In addition, large portions of Clusters 38 (non-ribosomal peptide synthetase: NRPS) and 47 (NRPS) were missing in <em>A. sojae</em> NBRC4239 (Fig. <a reveal-id="DSR009F5" data-open="DSR009F5" class="link link-reveal link-table xref-fig">5</a>). For Cluster 38, <em>A. sojae</em> NBRC4239 had a replaced 363-bp sequence and a sequence containing the 5′ portion of 460 bp and the 3′ portion of 2.9 kb generated by a deletion of 23 188 bp from the 26 571 bp (SC026: 1081838–1108408) predicted in <em>A. oryzae</em> RIB40 (Fig. <a reveal-id="DSR009F5" data-open="DSR009F5" class="link link-reveal link-table xref-fig">5</a>A). For Cluster 47, the 3′ portion of 14 kb in 30 338 bp (SC102: 1249352–1279689) predicted in <em>A. oryzae</em> RIB40 was replaced by unrelated 17-kb sequence (Fig. <a reveal-id="DSR009F5" data-open="DSR009F5" class="link link-reveal link-table xref-fig">5</a>B). Also, small deletions and insertions were observed in the remaining 29 clusters (data not shown). </p>                        <a id="6645793" scrollto-destination="6645793"></a>
<div id="DSR009F5" data-id="DSR009F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00905.jpeg?Expires=2147483647&Signature=db~DTUSpeqj2AaCMmkjZuJFs5N8T8QLhBixJMygL-YYravFnSBOjzPvPlnCZQrv~~7UOnACzzkJMyNlS6wOK2raAq3ggBG8E8ju-sfzcLUNdUuoF~tEfSbNGzgnOwZ3QXYcWwB1P5W~OiFs4a0JwjVkBcxwhbmNE5KrbzGQFo1cQ~fwX67teUT4LtNG1o~eDuIOh7mS2BQLhVYWrE-UqyrZWtHQ6Zb2iHLSDgRg-xkKReZ3sx7TWynpwTYVURVmH6E7ilx1tpYNLQHyqj9IIn4QLaqu5wmKa7jbeaQM5yxEe43UjAUQcHI~EI8DNudZp1juyRhrWvew8tk6gqRvY9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of A. oryzae predicted secondary metabolite clusters with A. sojae. Secondary metabolite Clusters 38 (A) and 47 (B) which were predicted from A. oryzae RIB40 by SMURF were compared with sequences in A. sojaeNBRC4239 by Harr plots." path-from-xml="dsr00905.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645793/dsr00905.jpeg" path-from-xml="dsr00905.jpeg" target="_blank">View large</a><a section="6645793" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00905.jpeg?Expires=2147483647&Signature=opjUBH-DrvKkFNpwOUUNxEDagDsdWf354Nw~Iem4NefSRIOmRy20j~~P6HTBT1mlY-AYCFzEgRdCRc0aj1kk9FuUlWPLIoBtBG-UxRvzCmKIZatdeZQ-Na848ok2bg7W4NH4kZAcRnVqbjIQkLX-HvcQk8WtEOGJAXp4ZpV3AO9WqPYbo74OanjjhGYnylbPbbgpk8zS352k4UREy-AU-ECxrGjK4ukaG35Wacz6SU1wvZzXH0T61tmbe3Cbx09hoYp41kiL-j-OoVGhU9xN60~cJElnKttb-rIKhCTPtvrmMHojixWPp0VgkQ8Abe~wsSqaePeEvVuApdxxdLIN2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645793&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00905.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of <em>A. oryzae</em> predicted secondary metabolite clusters with <em>A. sojae.</em> Secondary metabolite Clusters 38 (<strong>A</strong>) and 47 (<strong>B</strong>) which were predicted from <em>A. oryzae</em> RIB40 by SMURF were compared with sequences in <em>A. sojae</em>NBRC4239 by Harr plots.</p></div></div><div content-id="DSR009F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00905.jpeg?Expires=2147483647&Signature=db~DTUSpeqj2AaCMmkjZuJFs5N8T8QLhBixJMygL-YYravFnSBOjzPvPlnCZQrv~~7UOnACzzkJMyNlS6wOK2raAq3ggBG8E8ju-sfzcLUNdUuoF~tEfSbNGzgnOwZ3QXYcWwB1P5W~OiFs4a0JwjVkBcxwhbmNE5KrbzGQFo1cQ~fwX67teUT4LtNG1o~eDuIOh7mS2BQLhVYWrE-UqyrZWtHQ6Zb2iHLSDgRg-xkKReZ3sx7TWynpwTYVURVmH6E7ilx1tpYNLQHyqj9IIn4QLaqu5wmKa7jbeaQM5yxEe43UjAUQcHI~EI8DNudZp1juyRhrWvew8tk6gqRvY9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of A. oryzae predicted secondary metabolite clusters with A. sojae. Secondary metabolite Clusters 38 (A) and 47 (B) which were predicted from A. oryzae RIB40 by SMURF were compared with sequences in A. sojaeNBRC4239 by Harr plots." path-from-xml="dsr00905.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645793/dsr00905.jpeg" path-from-xml="dsr00905.jpeg" target="_blank">View large</a><a section="6645793" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00905.jpeg?Expires=2147483647&Signature=opjUBH-DrvKkFNpwOUUNxEDagDsdWf354Nw~Iem4NefSRIOmRy20j~~P6HTBT1mlY-AYCFzEgRdCRc0aj1kk9FuUlWPLIoBtBG-UxRvzCmKIZatdeZQ-Na848ok2bg7W4NH4kZAcRnVqbjIQkLX-HvcQk8WtEOGJAXp4ZpV3AO9WqPYbo74OanjjhGYnylbPbbgpk8zS352k4UREy-AU-ECxrGjK4ukaG35Wacz6SU1wvZzXH0T61tmbe3Cbx09hoYp41kiL-j-OoVGhU9xN60~cJElnKttb-rIKhCTPtvrmMHojixWPp0VgkQ8Abe~wsSqaePeEvVuApdxxdLIN2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645793&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00905.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of <em>A. oryzae</em> predicted secondary metabolite clusters with <em>A. sojae.</em> Secondary metabolite Clusters 38 (<strong>A</strong>) and 47 (<strong>B</strong>) which were predicted from <em>A. oryzae</em> RIB40 by SMURF were compared with sequences in <em>A. sojae</em>NBRC4239 by Harr plots.</p></div></div><p>Unlike Cluster 51 (PKS) located near the end of chromosome 5 (SC113: 1828103-1841684) in <em>A. oryzae</em> RIB40, missing of the equivalent cluster to Cluster 51 in <em>A. sojae</em> NBRC4239 may be partly explained by the instability of the region near the chromosome end in <em>A. sojae</em> NBRC4239. On the other hand, Clusters 38 and 47 in <em>A. oryzae</em> RIB40, of which equivalent clusters that had large deleted portions in <em>A. sojae</em> NBRC4239 are located near the centre of arm of chromosome 3 (SC26: 1081838–1108408 of 2 324 132 bp) and near the centromere of chromosome 4 (SC102: 1249352–1279689 of 1 779 707 bp), respectively. Therefore, the reasons for the depletion of these secondary metabolite gene clusters may be different from that for missing of the equivalent cluster to Cluster 51 at the chromosomal end.</p>                            <h5 scrollto-destination=6645796 id="6645796" class="section-title" data-legacy-id=s3c3b>3.3.3.2. Analysis of CPA gene cluster regions</h5>
<p>Gene cluster regions for CPA biosynthesis in <em>A. flavus</em> and <em>A. oryzae</em> were analysed for genomes of <em>A. sojae</em> NBRC4239, <em>A. flavus</em> NRRL3357, and <em>A. oryzae</em> RIB40 by BLASTN.<sup><a reveal-id="DSR009C34" data-open="DSR009C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The results are shown in Fig. <a reveal-id="DSR009F6" data-open="DSR009F6" class="link link-reveal link-table xref-fig">6</a>. In <em>A. flavus</em> and <em>A. oryzae</em>, CPA clusters were found at the end of chromosome 3, next to the aflatoxin biosynthesis gene clusters (Fig. <a reveal-id="DSR009F6" data-open="DSR009F6" class="link link-reveal link-table xref-fig">6</a>). Genes <em>mfs1</em>, <em>mao1</em>, <em>dmaT</em>, <em>Pks-nrps</em>, and <em>cttr1</em> shown in the figure are considered to be involved in CPA biosynthesis. Since the large portion of <em>Pks-nrps</em> at the telomere side is deleted, CPA cannot be synthesized in <em>A. oryzae</em> RIB40.<sup><a reveal-id="DSR009C34" data-open="DSR009C34" class="link link-ref link-reveal xref-bibr">34</a></sup> On the other hand, 18 508 bp of the CPA biosynthesis cluster region was found to be deleted in <em>A. sojae</em> NBRC4239, thus most of the <em>mfs1</em>, <em>mao1</em>, <em>dmaT</em>, and <em>Pks-nrps</em> sequences were missing (Fig. <a reveal-id="DSR009F6" data-open="DSR009F6" class="link link-reveal link-table xref-fig">6</a>). Furthermore, a 7653-bp sequence including <em>ord1</em>, <em>ord2</em>, and <em>ord3</em> is present 25 kb distant from the CPA cluster toward the telomere side in <em>A. flavus</em>. This sequence was found to be inserted inversely next to the missing CPA gene cluster in <em>A. sojae</em>. </p>                        <a id="6645799" scrollto-destination="6645799"></a>
<div id="DSR009F6" data-id="DSR009F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00906.jpeg?Expires=2147483647&Signature=ZutNt2uLqa3YxuS26FskUsHbt-736ezyK62KEQazc50zRSu2JToiLsmOLYygcn8sRwK8zJIDaAFiPSinr9FYHHR5-HOeQPlx13jLIS88NCHZGsgyXotC6q5BGTWC-Jg~B~Cju2JBp5CEXKFipt6UngS9RI51A37qCoyAmtjxq6NgpPqxhzwZb2hagawA93HFouUDkJFKDNNcIBztCb1Qc~jkQ73~P2eX7dkwA-Owp32a72ZC5WOKMKH5ySeeUucMANHZyUL4ZeD8p58hulk78-Re7ueUXFgXmcYnWgMx~wH~sgruQuTc7elvSBBJn6deYtIJp7x9HYfwozHkXnSYRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of CPA biosynthesis gene cluster regions between A. sojae NBRC4239, A. flavus NRRL3357, and A. oryzae RIB40. Nucleotide sequences of CPA biosynthesis gene cluster regions were compared between A. sojae NBRC4239, A. flavus NRRL3357, and A. oryzae RIB40 by BLASTN and are presented diagrammatically." path-from-xml="dsr00906.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645799/dsr00906.jpeg" path-from-xml="dsr00906.jpeg" target="_blank">View large</a><a section="6645799" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00906.jpeg?Expires=2147483647&Signature=PiYPepfcnMDIufHaZakHwtz~TExP3QYn-qqUM5Kb5hFUwDjtxy8I7o6W7jKwAP-h56kripGTH3xrSM2-DeDRs8H0OSIlv73bmA92pPm0N1d2lM~QQmvb0On-jdEXi4g7GLT~cGOmk~mujp3onovAttaVpX-NR6~s-LYskehd8DH7EPzrQHSxR8J8nb8HhLkhxZnnaxVcEgF447AbHB-~Nc~gXina7kjCeb-QkKjd9DCsWcJJEVwLh2mzQcG9Dp2PZlIOo5mHSUvN1509fur-kHGTepM~YS9hDwVt3h3WQCwbxZMEZeuzhgdhgVAXALLzcQeo-L2YHGKk0AB50qW88w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645799&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00906.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of CPA biosynthesis gene cluster regions between <em>A. sojae</em> NBRC4239, <em>A. flavus</em> NRRL3357, and <em>A. oryzae</em> RIB40. Nucleotide sequences of CPA biosynthesis gene cluster regions were compared between <em>A. sojae</em> NBRC4239, <em>A. flavus</em> NRRL3357, and <em>A. oryzae</em> RIB40 by BLASTN and are presented diagrammatically.</p></div></div><div content-id="DSR009F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00906.jpeg?Expires=2147483647&Signature=ZutNt2uLqa3YxuS26FskUsHbt-736ezyK62KEQazc50zRSu2JToiLsmOLYygcn8sRwK8zJIDaAFiPSinr9FYHHR5-HOeQPlx13jLIS88NCHZGsgyXotC6q5BGTWC-Jg~B~Cju2JBp5CEXKFipt6UngS9RI51A37qCoyAmtjxq6NgpPqxhzwZb2hagawA93HFouUDkJFKDNNcIBztCb1Qc~jkQ73~P2eX7dkwA-Owp32a72ZC5WOKMKH5ySeeUucMANHZyUL4ZeD8p58hulk78-Re7ueUXFgXmcYnWgMx~wH~sgruQuTc7elvSBBJn6deYtIJp7x9HYfwozHkXnSYRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of CPA biosynthesis gene cluster regions between A. sojae NBRC4239, A. flavus NRRL3357, and A. oryzae RIB40. Nucleotide sequences of CPA biosynthesis gene cluster regions were compared between A. sojae NBRC4239, A. flavus NRRL3357, and A. oryzae RIB40 by BLASTN and are presented diagrammatically." path-from-xml="dsr00906.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645799/dsr00906.jpeg" path-from-xml="dsr00906.jpeg" target="_blank">View large</a><a section="6645799" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00906.jpeg?Expires=2147483647&Signature=PiYPepfcnMDIufHaZakHwtz~TExP3QYn-qqUM5Kb5hFUwDjtxy8I7o6W7jKwAP-h56kripGTH3xrSM2-DeDRs8H0OSIlv73bmA92pPm0N1d2lM~QQmvb0On-jdEXi4g7GLT~cGOmk~mujp3onovAttaVpX-NR6~s-LYskehd8DH7EPzrQHSxR8J8nb8HhLkhxZnnaxVcEgF447AbHB-~Nc~gXina7kjCeb-QkKjd9DCsWcJJEVwLh2mzQcG9Dp2PZlIOo5mHSUvN1509fur-kHGTepM~YS9hDwVt3h3WQCwbxZMEZeuzhgdhgVAXALLzcQeo-L2YHGKk0AB50qW88w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645799&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00906.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of CPA biosynthesis gene cluster regions between <em>A. sojae</em> NBRC4239, <em>A. flavus</em> NRRL3357, and <em>A. oryzae</em> RIB40. Nucleotide sequences of CPA biosynthesis gene cluster regions were compared between <em>A. sojae</em> NBRC4239, <em>A. flavus</em> NRRL3357, and <em>A. oryzae</em> RIB40 by BLASTN and are presented diagrammatically.</p></div></div><p>PCR was carried out to confirm the missing <em>A. sojae</em> CPA biosynthesis gene cluster region and the inverted insert (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). These results confirmed the 18.5-kb deletion and the inverted 7.6-kb insertion found in the <em>A. sojae</em> NBRC4239 genome.</p><p>In addition to the finding of complete deletion of <em>mfs1</em>, <em>mao1</em>, and <em>dmaT</em>, the present analysis also found the deletion of a promoter and the half of the ORF containing the ketoacyl synthase (KR) domain and the acyltransferase (AT) domain for <em>Pks-nrps</em> in <em>A. sojae</em>. The genes <em>mao1</em>, <em>dmaT</em>, and <em>Pks-nrps</em> are essential for CPA biosynthesis in <em>A. flavus</em>.<sup><a reveal-id="DSR009C26" data-open="DSR009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Therefore, the present data lead to the conclusion that <em>A. sojae</em> is unable to produce CPA, which also verifies the safety of <em>A. sojae</em> for use in industry.</p>                            <h5 scrollto-destination=6645803 id="6645803" class="section-title" data-legacy-id=s3c3c>3.3.3.3. Analysis of aflatrem biosynthesis gene cluster regions</h5>
<p>The <em>A. sojae</em> genome was analysed for the aflatrem biosynthesis gene cluster found in <em>A. flavus</em>.<sup><a reveal-id="DSR009C27" data-open="DSR009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Aflatrem biosynthesis genes in <em>A. flavus</em> are known to consist of genes required to synthesize the intermediate paspaline (<em>atmG</em>, a<em>tmC</em>, a<em>tmM</em>, and a<em>tmB</em>) as well as genes required to convert paspaline to aflatrem (<em>atmP</em>, <em>atmQ</em>, and <em>atmD</em>), which are encoded at two separate loci ATM1 and ATM2 in <em>A. flavus</em>, respectively.<sup><a reveal-id="DSR009C27" data-open="DSR009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> ATM1 (34 816 bp) and ATM2 (25 256 bp) were analysed for the <em>A. oryzae</em> and <em>A. sojae</em> genomes by BLASTN. We found almost identical sequences to ATM1 and ATM2 with a few base substitutions in <em>A. oryzae</em>. In contrast, five gaps with &gt;100 bp were found in the corresponding region to ATM1 locus in <em>A. sojae</em> (Fig. <a reveal-id="DSR009F7" data-open="DSR009F7" class="link link-reveal link-table xref-fig">7</a>A). In Gap 4, a 1140-bp sequence in <em>A. flavus</em> was replaced by unrelated 6138-bp sequence in <em>A. sojae</em> (Fig. <a reveal-id="DSR009F7" data-open="DSR009F7" class="link link-reveal link-table xref-fig">7</a>A). Nine gaps with &gt;100 bp were also observed in the corresponding region to ATM2 locus in <em>A. sojae</em> (Fig. <a reveal-id="DSR009F7" data-open="DSR009F7" class="link link-reveal link-table xref-fig">7</a>B). In Gap 7, a 2761-bp sequence in <em>A. flavus</em> was replaced by unrelated 11 493-bp sequence in <em>A. sojae</em> (Fig. <a reveal-id="DSR009F7" data-open="DSR009F7" class="link link-reveal link-table xref-fig">7</a>B). All these gaps found in <em>A. sojae</em> were present in the non-coding regions. A frameshift due to single base insertion in exon 7 of <em>atmQ</em> was reported to account for the non-productivity of aflatrem in <em>A. oryzae</em>,<sup><a reveal-id="DSR009C27" data-open="DSR009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> but such mutation was not found in <em>A. sojae</em> in this study. The presence of insertions in <em>A. sojae</em> NBRC4239 was confirmed by PCR (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). </p>                        <a id="6645806" scrollto-destination="6645806"></a>
<div id="DSR009F7" data-id="DSR009F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00907.jpeg?Expires=2147483647&Signature=VI4WJ7EjHvC~Px9NhufjW1hSRcOven5hGc~ftd46EgWuX630KrrMYuRkbgl5xc9A4L33RaDKN~7QTR6AVpamDQM8kECWggnzKBHE2Mn724S6ton47M~8-hGPcsAD5sPiEOHzpBYV7yWnPxpFD1t7Zqhwr2hUqrffb1dIuvCdXi8LYzLaEubB2StlCKUzBJ~6ew9UnPVKF0RDcITJ6um55n2faKDiHF9gUJSovRbi8GdzDPC1hckx3JNkPMZlzFTktJtkzH2BEn0A9onEZ~lZDqcZIhDx5SvEUDuqyfc7WE~O0TOFPI8j7iqlVAzj0ivnp3sgtelixaVcsIHWQslFNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of aflatrem cluster regions in A. sojae NBRC4239 and A. flavus. Diagram of aflatrem cluster sequence comparison between A. flavus NRRL3357 and A. sojae NBRC4239 strains. (A) ATM1 region. (B) ATM2 region." path-from-xml="dsr00907.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645806/dsr00907.jpeg" path-from-xml="dsr00907.jpeg" target="_blank">View large</a><a section="6645806" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00907.jpeg?Expires=2147483647&Signature=jig49ShmepP0KXESKKNNHPc9br8gSHY2htcEMxlnCBQDIouVAE8OLqkk4U9XvMPCTFlk0H245gPy~4aOcNxWkAA3JawXZN6ynQY6GIMuwBIhlU4ZVxhfQbXp1LgcaYimODEItZUPUmFfbyYCMs8UIy8MaSeaVhTIo1TpdbXjp2jWVW376Lse~idbkS3YaBELYjv21kxNyV-iUZpLXJn09iVOm8WlBQZ-Jo~d9NFxm-sSJHQkqqtRS5kwUijCfGC4YyjRkhdWUmIdZ-tZhZPw2MFma9s1YzggU~q72qUKn61DStafrnvTvTe0JnVe7E8mMFEDouQrdR0yxzT2Icb3RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645806&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00907.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of aflatrem cluster regions in <em>A. sojae</em> NBRC4239 and <em>A. flavus</em>. Diagram of aflatrem cluster sequence comparison between <em>A. flavus</em> NRRL3357 and <em>A. sojae</em> NBRC4239 strains. (<strong>A</strong>) ATM1 region. (<strong>B</strong>) ATM2 region.</p></div></div><div content-id="DSR009F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/m_dsr00907.jpeg?Expires=2147483647&Signature=VI4WJ7EjHvC~Px9NhufjW1hSRcOven5hGc~ftd46EgWuX630KrrMYuRkbgl5xc9A4L33RaDKN~7QTR6AVpamDQM8kECWggnzKBHE2Mn724S6ton47M~8-hGPcsAD5sPiEOHzpBYV7yWnPxpFD1t7Zqhwr2hUqrffb1dIuvCdXi8LYzLaEubB2StlCKUzBJ~6ew9UnPVKF0RDcITJ6um55n2faKDiHF9gUJSovRbi8GdzDPC1hckx3JNkPMZlzFTktJtkzH2BEn0A9onEZ~lZDqcZIhDx5SvEUDuqyfc7WE~O0TOFPI8j7iqlVAzj0ivnp3sgtelixaVcsIHWQslFNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of aflatrem cluster regions in A. sojae NBRC4239 and A. flavus. Diagram of aflatrem cluster sequence comparison between A. flavus NRRL3357 and A. sojae NBRC4239 strains. (A) ATM1 region. (B) ATM2 region." path-from-xml="dsr00907.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6645806/dsr00907.jpeg" path-from-xml="dsr00907.jpeg" target="_blank">View large</a><a section="6645806" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr00907.jpeg?Expires=2147483647&Signature=jig49ShmepP0KXESKKNNHPc9br8gSHY2htcEMxlnCBQDIouVAE8OLqkk4U9XvMPCTFlk0H245gPy~4aOcNxWkAA3JawXZN6ynQY6GIMuwBIhlU4ZVxhfQbXp1LgcaYimODEItZUPUmFfbyYCMs8UIy8MaSeaVhTIo1TpdbXjp2jWVW376Lse~idbkS3YaBELYjv21kxNyV-iUZpLXJn09iVOm8WlBQZ-Jo~d9NFxm-sSJHQkqqtRS5kwUijCfGC4YyjRkhdWUmIdZ-tZhZPw2MFma9s1YzggU~q72qUKn61DStafrnvTvTe0JnVe7E8mMFEDouQrdR0yxzT2Icb3RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6645806&ar=444512&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00907.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of aflatrem cluster regions in <em>A. sojae</em> NBRC4239 and <em>A. flavus</em>. Diagram of aflatrem cluster sequence comparison between <em>A. flavus</em> NRRL3357 and <em>A. sojae</em> NBRC4239 strains. (<strong>A</strong>) ATM1 region. (<strong>B</strong>) ATM2 region.</p></div></div><p>In this study, we showed that <em>A. sojae</em> NBRC4239 had many differences in ATM1 and ATM2 loci including deletions and insertion of unrelated sequences in comparison with those in <em>A. oryzae and A. flavus</em>, where both loci are well conserved. In addition to these differences, more than 10 gaps &lt;100 bp were also observed in the corresponding loci in <em>A. sojae</em> NBRC4239 (data not shown). As described above, all the 14 gaps were present in the non-coding regions in <em>A. sojae</em>. To date, production of aflatrem in <em>A. sojae</em> has not been reported but the present study did not provide the evidence for aflatrem non-productivity from the sequence information. Further analysis will be needed to solve this discrepancy on the aflatrem production in <em>A. sojae</em>.</p>                            <h2 scrollto-destination=6645811 id="6645811" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6645822 id="6645822" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank S.W. Kim, M. Kikuchi, K. Furuya, C. Shindo, H. Inaba, E. Iioka, Y. Takayama, W. Suda, and Y. Hattori from the University of Tokyo, S. Hara, K. Matsushima, K. Ito, and Y. Shinohara from Noda Institute for Scientific Research, and T. Yoshimura, A. Komura, J. Manaka, and M. Yuzuki from Kikkoman Corporation for technical assistance and fruitful discussion.</p>                            <h2 scrollto-destination=6645830 id="6645830" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR009C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Godet</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Munaut</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Molecular strategy for identification in <em>Aspergillus</em> section <em>Flavi</em></div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2010</div>, vol. <div class="volume">304</div> (pg. <div class="fpage">157</div>-<div class="lpage">68</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1574-6968.2009.01890.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1574-6968.2009.01890.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20strategy%20for%20identification%20in%20Aspergillus%20section%20Flavi&amp;author=M.%20Godet&amp;author=F.%20Munaut&amp;publication_year=2010&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=304&amp;pages=157-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2010.304.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2010.304.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2010.304.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20377644" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Geiser</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Pitt</div> <div class="given-names">J.I.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">Cryptic speciation and recombination in the aflatoxin-producing fungus <em>Aspergillus flavus</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">388</div>-<div class="lpage">93</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.95.1.388" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.95.1.388</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cryptic%20speciation%20and%20recombination%20in%20the%20aflatoxin-producing%20fungus%20Aspergillus%20flavus&amp;author=D.M.%20Geiser&amp;author=J.I.%20Pitt&amp;author=J.W.%20Taylor&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=388-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.1.388" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.1.388" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.1.388"> </span></div> </div></div></div></div></div><div content-id="DSR009C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akao</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Analysis of expressed sequence tags from the fungus <em>Aspergillus oryzae</em> cultured under different conditions.</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">47</div>-<div class="lpage">57</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsm008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsm008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20expressed%20sequence%20tags%20from%20the%20fungus%20Aspergillus%20oryzae%20cultured%20under%20different%20conditions.&amp;author=T.%20Akao&amp;author=M.%20Sano&amp;author=O.%20Yamada&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=47-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chang</div> <div class="given-names">P.K.</div></div>,  <div class="name"><div class="surname">Matsushima</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Understanding nonaflatoxigenicity of <em>Aspergillus sojae</em>: a windfall of aflatoxin biosynthesis research</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">76</div> (pg. <div class="fpage">977</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00253-007-1116-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00253-007-1116-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Understanding%20nonaflatoxigenicity%20of%20Aspergillus%20sojae%3A%20a%20windfall%20of%20aflatoxin%20biosynthesis%20research&amp;author=P.K.%20Chang&amp;author=K.%20Matsushima&amp;author=T.%20Takahashi&amp;publication_year=2007&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=76&amp;pages=977-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-007-1116-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-007-1116-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-007-1116-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17665189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asai</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genomics of <em>Aspergillus oryzae</em></div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2007</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">646</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1271/bbb.60550" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1271/bbb.60550</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20Aspergillus%20oryzae&amp;author=T.%20Kobayashi&amp;author=K.%20Abe&amp;author=K.%20Asai&amp;publication_year=2007&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=71&amp;pages=646-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.60550" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.60550" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.60550"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17341818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Machida</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Asai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome sequencing and analysis of <em>Aspergillus oryzae</em></div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">438</div> (pg. <div class="fpage">1157</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature04300" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature04300</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20and%20analysis%20of%20Aspergillus%20oryzae&amp;author=M.%20Machida&amp;author=K.%20Asai&amp;author=M.%20Sano&amp;publication_year=2005&amp;journal=Nature&amp;volume=438&amp;pages=1157-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16372010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Machida</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Terabayashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genomics of industrial <em>Aspergilli</em> and comparison with toxigenic relatives</div>, <div class="source">Food Addit. Contam. Part A Chem. Anal. Control Expo Risk Assess.</div> , <div class="year">2008</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1147</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20industrial%20Aspergilli%20and%20comparison%20with%20toxigenic%20relatives&amp;author=M.%20Machida&amp;author=Y.%20Terabayashi&amp;author=M.%20Sano&amp;publication_year=2008&amp;journal=Food%20Addit.%20Contam.%20Part%20A%20Chem.%20Anal.%20Control%20Expo%20Risk%20Assess.&amp;volume=25&amp;pages=1147-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/02652030802273114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F02652030802273114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F02652030802273114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18798040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Machida</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Gomi</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genomics of <em>Aspergillus oryzae</em>: learning from the history of Koji mold and exploration of its future</div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">173</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn020" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn020</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20Aspergillus%20oryzae%3A%20learning%20from%20the%20history%20of%20Koji%20mold%20and%20exploration%20of%20its%20future&amp;author=M.%20Machida&amp;author=O.%20Yamada&amp;author=K.%20Gomi&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=173-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18820080" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Majoros</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Pertea</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2878</div>-<div class="lpage">79</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bth315" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bth315</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TigrScan%20and%20GlimmerHMM%3A%20two%20open%20source%20ab%20initio%20eukaryotic%20gene-finders&amp;author=W.H.%20Majoros&amp;author=M.%20Pertea&amp;author=S.L.%20Salzberg&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=2878-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15145805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schöffmann</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Waack</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">62</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-7-62" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-7-62</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20in%20eukaryotes%20with%20a%20generalized%20hidden%20Markov%20model%20that%20uses%20hints%20from%20external%20sources&amp;author=M.%20Stanke&amp;author=O.%20Sch%C3%B6ffmann&amp;author=B.%20Morgenstern&amp;author=S.%20Waack&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-62" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-62" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-62"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16469098" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Waack</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Gene prediction with a hidden Markov model and a new intron submodel</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">ii215</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=ii215-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg1080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg1080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg1080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korf</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Gene finding in novel genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">59</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-5-59" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-5-59</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20finding%20in%20novel%20genomes&amp;author=I.%20Korf&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-5-59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15144565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Besemer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GeneMark: web software for gene finding in prokaryotes, eukaryotes and viruses</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">W451</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gki487" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gki487</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMark%3A%20web%20software%20for%20gene%20finding%20in%20prokaryotes%2C%20eukaryotes%20and%20viruses&amp;author=J.%20Besemer&amp;author=M.%20Borodovsky&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W451-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki487" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki487" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki487"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Clamp</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">GeneWise and Genomewise</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">988</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.1865504" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.1865504</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneWise%20and%20Genomewise&amp;author=E.%20Birney&amp;author=M.%20Clamp&amp;author=R.%20Durbin&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=988-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1865504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1865504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1865504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15123596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Mackey</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Roos</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Pereira</div> <div class="given-names">F.C.N.</div></div>. </div><div class="article-title">Evigan: a hidden variable model for integrating gene evidence for eukaryotic gene predictions</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">597</div>-<div class="lpage">605</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btn004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btn004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evigan%3A%20a%20hidden%20variable%20model%20for%20integrating%20gene%20evidence%20for%20eukaryotic%20gene%20predictions&amp;author=Q.%20Liu&amp;author=A.J.%20Mackey&amp;author=D.S.%20Roos&amp;author=F.C.N.%20Pereira&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=597-605" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18187439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/25.17.3389" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/25.17.3389</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/25.5.955" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/25.5.955</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arnaud</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Chibucos</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Costanzo</div> <div class="given-names">M.C.</div></div>, et al. </div><div class="article-title">The <em>Aspergillus</em> Genome Database, a curated comparative genomics resource for gene, protein and sequence information for the <em>Aspergillus</em> research community</div>, <div class="source">, Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">D420</div>-<div class="lpage">27</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkp751" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkp751</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Aspergillus%20Genome%20Database%2C%20a%20curated%20comparative%20genomics%20resource%20for%20gene%2C%20protein%20and%20sequence%20information%20for%20the%20Aspergillus%20research%20community&amp;author=M.B.%20Arnaud&amp;author=M.C.%20Chibucos&amp;author=M.C.%20Costanzo&amp;publication_year=2010&amp;journal=%2C%20Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=D420-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp751" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp751" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp751"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19773420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Profile hidden Markov models</div>, <div class="source">Bioinformatics</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">755</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/14.9.755" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/14.9.755</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Profile%20hidden%20Markov%20models&amp;author=S.R.%20Eddy&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=755-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.755" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.9.755" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.9.755"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9918945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Tate</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D281</div>-<div class="lpage">88</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm960" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm960</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=J.%20Tate&amp;author=J.%20Mistry&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D281-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm960" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm960" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm960"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039703" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rawlings</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Barrett</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>. </div><div class="article-title">MEROPS: the peptidase database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">D227</div>-<div class="lpage">33</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkp971" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkp971</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEROPS%3A%20the%20peptidase%20database&amp;author=N.D.%20Rawlings&amp;author=A.J.%20Barrett&amp;author=A.%20Bateman&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=D227-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp971" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp971" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp971"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19892822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Plewniak</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Jeanmougin</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>. </div><div class="article-title">The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">4876</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/25.24.4876" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/25.24.4876</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ClustalX%20windows%20interface%3A%20flexible%20strategies%20for%20multiple%20sequence%20alignment%20aided%20by%20quality%20analysis%20tools&amp;author=J.D.%20Thompson&amp;author=T.J.%20Gibson&amp;author=F.%20Plewniak&amp;author=F.%20Jeanmougin&amp;author=D.G.%20Higgins&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=24&amp;pages=4876-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.24.4876" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.24.4876"> </span></div> </div></div></div></div></div><div content-id="DSR009C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Page</div> <div class="given-names">R.</div></div>. </div><div class="article-title">TREEVIEW: an application to display phylogenetic trees on personal computers</div>, <div class="source">Appl. Biosci.</div> , <div class="year">1996</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">357</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSR009C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cantarel</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Coutinho</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Rancurel</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bernard</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lombard</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Henrissat</div> <div class="given-names">B.</div></div>. </div><div class="article-title">The Carbohydrate-Active EnZymes database (CAZy): an expert resource for glycogenomics</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D233</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn663" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn663</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Carbohydrate-Active%20EnZymes%20database%20(CAZy)%3A%20an%20expert%20resource%20for%20glycogenomics&amp;author=B.L.%20Cantarel&amp;author=P.M.%20Coutinho&amp;author=C.%20Rancurel&amp;author=T.%20Bernard&amp;author=V.%20Lombard&amp;author=B.%20Henrissat&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D233-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn663" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn663" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn663"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18838391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Khaldi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Seifuddin</div> <div class="given-names">F.T.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">SMURF: genomic mapping of fungal secondary metabolite clusters</div>, <div class="source">Fungal Genet. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">736</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.fgb.2010.06.003" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.fgb.2010.06.003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMURF%3A%20genomic%20mapping%20of%20fungal%20secondary%20metabolite%20clusters&amp;author=N.%20Khaldi&amp;author=F.T.%20Seifuddin&amp;author=G.%20Turner&amp;publication_year=2010&amp;journal=Fungal%20Genet.%20Biol.&amp;volume=47&amp;pages=736-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2010.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2010.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2010.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20554054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chang</div> <div class="given-names">P.K.</div></div>,  <div class="name"><div class="surname">Horn</div> <div class="given-names">B.W.</div></div>,  <div class="name"><div class="surname">Dorner</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">Clustered genes involved in cyclopiazonic acid production are next to the aflatoxin biosynthesis gene cluster</div>, <div class="source">Fungal Genet. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">176</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.fgb.2008.11.002" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.fgb.2008.11.002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustered%20genes%20involved%20in%20cyclopiazonic%20acid%20production%20are%20next%20to%20the%20aflatoxin%20biosynthesis%20gene%20cluster&amp;author=P.K.%20Chang&amp;author=B.W.%20Horn&amp;author=J.W.%20Dorner&amp;publication_year=2009&amp;journal=Fungal%20Genet.%20Biol.&amp;volume=46&amp;pages=176-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2008.11.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2008.11.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2008.11.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19038354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nicholson</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Koulman</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Monahan</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Pritchard</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Payne</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Scott</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Identification of two aflatrem biosynthesis gene loci in <em>Aspergillus flavus</em> and metabolic engineering of <em>Penicillium paxilli</em> to elucidate their function</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">7469</div>-<div class="lpage">81</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/AEM.02146-08" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/AEM.02146-08</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20two%20aflatrem%20biosynthesis%20gene%20loci%20in%20Aspergillus%20flavus%20and%20metabolic%20engineering%20of%20Penicillium%20paxilli%20to%20elucidate%20their%20function&amp;author=M.J.%20Nicholson&amp;author=A.%20Koulman&amp;author=B.J.%20Monahan&amp;author=B.L.%20Pritchard&amp;author=G.A.%20Payne&amp;author=B.%20Scott&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=75&amp;pages=7469-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02146-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02146-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02146-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19801473" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wirsel</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lachmund</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wildhardt</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ruttkowski</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Three alpha-amylase genes of <em>Aspergillus oryzae</em> exhibit identical intron–exon organization</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1989</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">3</div>-<div class="lpage">14</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-2958.1989.tb00097.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-2958.1989.tb00097.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20alpha-amylase%20genes%20of%20Aspergillus%20oryzae%20exhibit%20identical%20intron%E2%80%93exon%20organization&amp;author=S.%20Wirsel&amp;author=A.%20Lachmund&amp;author=G.%20Wildhardt&amp;author=E.%20Ruttkowski&amp;publication_year=1989&amp;journal=Mol.%20Microbiol.&amp;volume=3&amp;pages=3-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1989.3.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1989.3.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1989.3.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2785629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muñoz-López</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">García-Pérez</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">DNA transposons: nature and applications in genomics</div>, <div class="source">Curr. Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">115</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2174/138920210790886871" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2174/138920210790886871</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20transposons%3A%20nature%20and%20applications%20in%20genomics&amp;author=M.%20Mu%C3%B1oz-L%C3%B3pez&amp;author=J.L.%20Garc%C3%ADa-P%C3%A9rez&amp;publication_year=2010&amp;journal=Curr.%20Genomics&amp;volume=11&amp;pages=115-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2174/138920210790886871" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2174%2F138920210790886871" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2174%2F138920210790886871"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20885819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tani</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tsukagoshi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Mode of AmyR binding to the CGGN<sub>8</sub>AGG sequence in the <em>Aspergillus oryzae taaG2</em> promoter</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2004</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">1906</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1271/bbb.68.1906" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1271/bbb.68.1906</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mode%20of%20AmyR%20binding%20to%20the%20CGGN8AGG%20sequence%20in%20the%20Aspergillus%20oryzae%20taaG2%20promoter&amp;author=T.%20Ito&amp;author=S.%20Tani&amp;author=T.%20Itoh&amp;author=N.%20Tsukagoshi&amp;author=M.%20Kato&amp;author=T.%20Kobayashi&amp;publication_year=2004&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=68&amp;pages=1906-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.68.1906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.68.1906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.68.1906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15388966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruijter</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Visser</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Carbon repression in <em>Aspergilli</em></div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">1997</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">103</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1574-6968.1997.tb12557.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1574-6968.1997.tb12557.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Carbon%20repression%20in%20Aspergilli&amp;author=G.J.%20Ruijter&amp;author=J.%20Visser&amp;publication_year=1997&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=151&amp;pages=103-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6968.1997.tb12557.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6968.1997.tb12557.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6968.1997.tb12557.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9228741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moerman</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Waterston</div> <div class="given-names">R.H.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Berg</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Howe</div> <div class="given-names">M.M.</div></div>. </div><div class="article-title">Mobile elements in <em>Caenorhabditis elegans</em> and other nematodes</div>, <div class="source">Mobile DNA</div> , <div class="year">1989</div><div class="publisher-loc">Washington, DC,</div><div class="publisher-name">American Society of Microbiology</div>(pg. <div class="fpage">537</div>-<div class="lpage">56</div>)</div></div></div></div><div content-id="DSR009C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glayzer</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Archer</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">R.P.</div></div>. </div><div class="article-title"><em>Ant-1</em>, an active transposon from the fungus <em>Aspergillus niger</em></div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">249</div> (pg. <div class="fpage">432</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ant-1%2C%20an%20active%20transposon%20from%20the%20fungus%20Aspergillus%20niger&amp;author=D.%20Glayzer&amp;author=I.%20Roberts&amp;author=D.B.%20Archer&amp;author=R.P.%20Oliver&amp;publication_year=1995&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=249&amp;pages=432-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00287105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00287105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00287105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8552048" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR009C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR009C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tokuoka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Seshime</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fujii</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kitamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Koyama</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Identification of a novel polyketide synthase-nonribosomal peptide synthetase (PKS-NRPS) gene required for the biosynthesis of cyclopiazonin acid in <em>Aspergillus oryzae</em></div>, <div class="source">Fungal Genet. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">1608</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.fgb.2008.09.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.fgb.2008.09.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20novel%20polyketide%20synthase-nonribosomal%20peptide%20synthetase%20(PKS-NRPS)%20gene%20required%20for%20the%20biosynthesis%20of%20cyclopiazonin%20acid%20in%20Aspergillus%20oryzae&amp;author=M.%20Tokuoka&amp;author=Y.%20Seshime&amp;author=I.%20Fujii&amp;author=K.%20Kitamoto&amp;author=T.%20Takahashi&amp;author=Y.%20Koyama&amp;publication_year=2008&amp;journal=Fungal%20Genet.%20Biol.&amp;volume=45&amp;pages=1608-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2008.09.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2008.09.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2008.09.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18854220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=444512&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_table5.png?Expires=2147483647&amp;Signature=Tr3EUqVwdlR~zM80cwEh4NNoyFGEsHy1c6l8s3u8oijccIvSWk0UDtOTeZxf9rjbuZejn61YoB7yiqOwwA0wsu3s2-Ceek1trHKsttveX0nLRYbaWTETsLNrsdPhwcwl~jzs6~4vvxmbiDTzUPvUjxPQ3V3ZhS8JKKK6G3hqcZQHkN7~3icMEvQ4v5z7CVpplHzYD4sXyDqXTzJok3yiY-sxRBctcV4ugGBnQQTJHPla-9-o66lAKGbVk~vEUWnnJUJLZxuhD06s0nargi4ntz2DUu2acKU5JlwMtyp-EY-j0Ae6-Ced~X4A9ZEccL8xhjWmBwAY7EJqOgasjpNH6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_table4.png?Expires=2147483647&amp;Signature=T2qjn2k77IPxKp28AvcqgeO-0f1gdA3CoB6wWc1ZKWKw0Z4nllZTsLzAX2RfwTSo14NNVDfA919K4RujHxKuxkiG1jcd8TS2HB~TOg1w8znM1SlsKN~wwyjyLs45~6du45hPskR5qDhWvJtRDbQBWnNV8XNf90GJ1WCpMRmExPxyYQNCTlKzzc9n1hdhM0Sq3qNyKJTpQbAF~43TKEOmZC~CARMZ2EFShoso0WsK~rrsxlOrs6FJT1pD8d~FhxqYtJeTWCzdHZY7iuKA6tDEruj2VF0n24l29wSh2~5oV3r4IXNLpWvL4kfrGWOoaSOuoCEeliNsOKrD-HCc-8TTDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_table3.png?Expires=2147483647&amp;Signature=HfeDYKeu~Em0TKdRxMK92CPfc1y8SPeslKyst8fUs-Sh5QgyXeDWIlZ3gWGAQJqt-H4XoV01TiDa9OH2ZWJNqM-1LciBbkGf57C~WsWduJF1ttHdyv2F4SJg7migUoZWUqQGc46oqpQi2aq~rX5WJ6HAy4xjbtjt6KTQABclfqMT~pgV6oTwe~cYMx1jl3GP6xExT3YJ0tN~SjYxPHNEwU3OWxLV94XZOIKJdVSUG1kHWFWORuNI1W8wzoBnx6ppR7NByBToCYg5Ga5P2juYgqTRXFukK-6WUflYuVbq71FIqdHazQa1ecoXLw14F4XI2DQAx2~Y3exgY7iaiH3JNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_table2.png?Expires=2147483647&amp;Signature=Tko8atnhSX79RoGS6d1O~tLJuN12WvzCSYfvy0K~EYLUH-RWXgIbckNHEnDHtjujzt~ZuF141nFXpEH8dFu0aDv-drgNoZTHliEwSCJF-YqnWZpFs0OPFbp4KzuD5W9ioLBtqrjZ~yobUjruu6OGG-rZKXYCly2k8FUzlFfvKsyTMIHFGSwTGYH4IT8QOaKPLAHwWcLpy1Qy1pCN1MjD4w3b73mdwcYJ-8cTLdNoJ3JO~rk3H-9oq5l4M0RuZcSHn3g2YGfn0Hu7gAtspSJMZIT6zt~zpzVe-9woG3fMyD7hJp0h8LyM45qqBGgwARQLxqjzrAg4qh2qGzrVVarJkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_table1.png?Expires=2147483647&amp;Signature=CTh~JHCzhZuwBk4kou81dc9pKJU-B~kaJWIk7Iw87hqx9A~OmDDGLe2X0Al-~QA~TBlw2RRGa2Z8lhZ8yljNBO-0OFhGab~kl8KSf4gty1741uTphT7AnjZneL7UGaw7EushEZKCBEig1PoCmV-3V4LnSrJ-JxEuX5-BlM2W5c1jXqzrjmWd66xJ5yoQGx5SHViZFaJhYDy2YdOhLleuNoFc-2kwEBvlLTb-cv-sqji9k0ZGiW5eIs-O1IwIwq9EDEP2SnwMqJhl5o19U2fgK4lho9zhcJ0IHPcpBagGp88Ft0vVl-bBq12WQXY2aqiLO0KTvXqhkhhEKXppXzD1zA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_fig4.png?Expires=2147483647&amp;Signature=x0LDjhHwOoSCcOYC1JAwMe1Bx~PPhwaC8r9rGlEjSjTkkv9cr0k3xmkXMXcyovZ6RWwVpLiyF0kXOlpVQpaGo6eSuN~Ip~CdRc1Xlp~bv~IlaNRhtdjgBCd5yt41qz5VGtLxIAd72HNn5oJliZomx7mjXn31ALG14jWoWBPRo2qQO8eF-9Uo6p2-a152eWx-ODMDImSjF4-4115hQ7O5m0gKijN8jg8ik0Euf7l9jDAJHoTLxSYU~eTznOr45AIRCwkaVAP95kzIzh7JI2aN667KlH6qPinqVz1R7D52J2Tce5nr4fff4hPNKRhXSZXA7bAeA9m2UVHklz9bJjmVaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_fig3.png?Expires=2147483647&amp;Signature=FDg94IIPSXsSRqEbpFHxBln03MgAYUlWLuXzOZkMUMElaaESecwKhkpq~T1ok0RvVTrELHK2tVQqcVSHCaCDpxuK4kGDf7ILU-SW3dA5TsyWP6PPgx8uG2q1oNLdI8ul2jZczvuUUecSULt~jaoSGjVeN6klmwqdd~N4fcZLNGfXRPCybGa3mOgQSivGU8nm-ihhx99sm5lbOF5hPw-SoO6e3T18ekMJ5Vh~XTDkY2SaFv2lfQdbFnNaY7DXuVOJeVj7kcBayeObDmrexyMDOf3XT3owvDUwxlLfwkHlrOUa0D-5cY18b0GhJECzpxOMzSRkRe5O1BIlRThb0o~41A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_fig2.png?Expires=2147483647&amp;Signature=RRoDUfjFJYRphSws3TWfXVhinNPUhcMS2fA8u3Es23VzLKCdixW5PwPit8gZZseYaOZCoBVCvcvPe0TgFKKfx0UmpKzC8Mh8HARj675-~9N8abfdiudAC5OGZui2ji24mdTuXgndg8OeT~hb0ZGWA-Rx3y6goEde58jp9b7n5nefOgP8Gga6KReQhx5c2i1TXMuohJDGm--49In-4NBqVXZyxsrgm3bVZl4-eIpkoPDIDiG2r37wYW4LxpuzwqbBf3RZWppReitoULjXBmV39VuzDsRBmkR3gA9Hkr0uIn4wBT8J3AArppT0b7ngX8SnL1kSW~mXP4CO432aKTbTWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp_fig1.png?Expires=2147483647&amp;Signature=2jrDiCfJumkG1U5O3avwZlZtN4y6hsM4cNXVE~liwpGmJkdXWxlZBBcnPSXxpjr0UIKCHwalEAU9bG3-YmWtn9XFj-Zf26AioyT8wsktU7ev2A0xB3iOzgacKecdMLYw7FDyP9s5xtMDLDRxTwAqFF6x15BPsEYc0147LWjxH42sSAkqAET1te4zK7dPUzqG~fRDU74die1HfymtsgzEn0PtAufIebSKhvH2bTX3Z7IGOoTLDW~VFXIgxTjJhqdO2OZ9GmHTH2ciAYQe6fAp~A3irKI7UD8rcUCXLw~48Q3MwFTrtxAl5Q~bZXuBAKVxLvIttpyFvdyaRKhrpEegpw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr009/2/dsr009-dsr009supp.doc?Expires=2147483647&amp;Signature=G3WuM2Ah6u-EzcbrDJfoa~eAw2yckSTqKp9iKiG5C6dr1CiAwAQketcEg1yfzzsvvfxmN~-qvg~caPOJVN5m7yWfVlkamWiv1HYCnKsh90GZ2tF4yPWmWA7m3of8bV914wOEvB2Vy84zbxOCMUyNEbX3yEsS0zg~FdNpthwIdGNJ4Kh~X7qFwCul7Y~vEUkvw3MfhOf0Yux10q92OPedwcrTJv~aciqtfIbW0SDFhRt5FmjPH9NY4ckTYUoS8k7cd058TttNMkBXbqDJCjmWOdvSmNBrYX1XtQex0TfOzuNozDbj4EFWcQ-l7m-ctbq4C2kXELtCLCYYEWS13golTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">320</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000292070400003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">27</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr009"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="444512"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000292070400003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f3%2f165%2f444512">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000292070400003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (27)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F3%2F165%2F444512" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/444512'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Draft Genome Sequencing and Comparative Analysis of Aspergillus sojae NBRC4239" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr009");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f165%2f444512">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

