

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/4/141/466067","@type":"ScholarlyArticle","name":"Reverse Polarization in Amino acid and Nucleotide Substitution Patterns Between Human–Mouse Orthologs of Two Compositional Extrema","datePublished":"2007-09-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm015","keywords":["high-GC orthologs","low-GC orthologs","amino acid replacement matrix","nucleotide replacement matrix","sequence divergence"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/4/141/466067","author":[{"name":"Bag, Sumit K.","affiliation":"1  Bioinformatics Centre ,  Indian Institute of Chemical Biology ,  Kolkata 700 032 ,  India","@type":"Person"},{"name":"Paul, Sandip","affiliation":"1  Bioinformatics Centre ,  Indian Institute of Chemical Biology ,  Kolkata 700 032 ,  India","@type":"Person"},{"name":"Ghosh, Subhagata","affiliation":"2  Structural Biology and Bioinformatics Division ,  Indian Institute of Chemical Biology ,  4, Raja S. C. Mullick Road, Kolkata 700 032 ,  India","@type":"Person"},{"name":"Dutta, Chitra","affiliation":"1  Bioinformatics Centre ,  Indian Institute of Chemical Biology ,  Kolkata 700 032 ,  India    2  Structural Biology and Bioinformatics Division ,  Indian Institute of Chemical Biology ,  4, Raja S. C. Mullick Road, Kolkata 700 032 ,  India","@type":"Person"}],"description":"Abstract.  Genome-wide analysis of sequence divergence patterns in 12 024 human–mouse orthologous pairs reveals, for the first time, that the trends in nucleoti","pageStart":"141","pageEnd":"154","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Reverse Polarization in Amino acid and Nucleotide Substitution Patterns Between Human–Mouse Orthologs of Two Compositional Extrema","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Reverse Polarization in Amino acid and Nucleotide Substitution Patterns Between Human–Mouse Orthologs of Two Compositional Extrema" />
    <meta property="og:description" content="Abstract.  Genome-wide analysis of sequence divergence patterns in 12 024 human–mouse orthologous pairs reveals, for the first time, that the trends in nucleoti" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/4/141/466067" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-09-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Bag, Sumit K." /><meta name="citation_author_institution" content="Bioinformatics Centre, Indian Institute of Chemical Biology, Kolkata 700 032, India" /><meta name="citation_author" content="Paul, Sandip" /><meta name="citation_author_institution" content="Bioinformatics Centre, Indian Institute of Chemical Biology, Kolkata 700 032, India" /><meta name="citation_author" content="Ghosh, Subhagata" /><meta name="citation_author_institution" content="Structural Biology and Bioinformatics Division, Indian Institute of Chemical Biology, 4, Raja S. C. Mullick Road, Kolkata 700 032, India" /><meta name="citation_author" content="Dutta, Chitra" /><meta name="citation_author_institution" content="Bioinformatics Centre, Indian Institute of Chemical Biology, Kolkata 700 032, India" /><meta name="citation_author_institution" content="Structural Biology and Bioinformatics Division, Indian Institute of Chemical Biology, 4, Raja S. C. Mullick Road, Kolkata 700 032, India" /><meta name="citation_title" content="Reverse Polarization in Amino acid and Nucleotide Substitution Patterns Between Human–Mouse Orthologs of Two Compositional Extrema" /><meta name="citation_firstpage" content="141" /><meta name="citation_lastpage" content="154" /><meta name="citation_doi" content="10.1093/dnares/dsm015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Isochores and the evolutionary genomics of vertebrates; Gene;  citation_year=2000;  citation_volume=241;  citation_pages=3-17; " /><meta name="citation_reference" content="citation_title=The evolution of isochores; Nat. Rev. Genet.;  citation_year=2001;  citation_volume=2;  citation_pages=549-555; " /><meta name="citation_reference" content="citation_title=An analysis of the bovine genome by Cs2SO4-Ag density gradient centrifugation; J. Mol. Biol.;  citation_year=1973;  citation_volume=80;  citation_pages=177-197; " /><meta name="citation_reference" content="citation_title=Warm-blooded isochore structure in Nile crocodile and turtle; Mol. Biol. Evol.;  citation_year=1999;  citation_volume=16;  citation_pages=1521-1527; " /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=The human transcriptome map: clustering of highly expressed genes in chromosomal domains; Science;  citation_year=2001;  citation_volume=291;  citation_pages=1289-1292; " /><meta name="citation_reference" content="citation_title=Local rates of recombination are positively correlated with GC content in the human genome; Mol. Biol. Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=1139-1142; " /><meta name="citation_reference" content="citation_title=A high-resolution recombination map of the human genome; Nat. Genet.;  citation_year=2002;  citation_volume=31;  citation_pages=241-247; " /><meta name="citation_reference" content="citation_title=A unification of mosaic structures in the human genome; Hum. Mol. Genet.;  citation_year=2003;  citation_volume=12;  citation_pages=2411-2415; " /><meta name="citation_reference" content="citation_title=The distribution of genes in the human genome; Gene;  citation_year=1991;  citation_volume=100;  citation_pages=181-187; " /><meta name="citation_reference" content="citation_title=Correlations between isochores and chromosomal bands in the human genome; Proc. Natl. Acad. Sci. USA;  citation_year=1993;  citation_volume=90;  citation_pages=11929-11933; " /><meta name="citation_reference" content="citation_title=Initial sequencing and comparative analysis of the mouse genome; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=520-562; " /><meta name="citation_reference" content="citation_title=The Wilhelmine E. Key 2001 Invitational Lecture. Estimation of divergence times for a few mammalian and several primate species; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=157-164; " /><meta name="citation_reference" content="citation_title=Molecular dating: ape bones agree with chicken entrails; Trends Genet.;  citation_year=2005;  citation_volume=21;  citation_pages=89-92; " /><meta name="citation_reference" content="citation_title=Distinct changes of genomic biases in nucleotide substitution at the time of Mammalian radiation; Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=1887-1896; " /><meta name="citation_reference" content="citation_title=Vanishing GC-rich isochores in mammalian genomes; Genetics;  citation_year=2002;  citation_volume=162;  citation_pages=1837-1847; " /><meta name="citation_reference" content="citation_title=Deterministic mutation rate variation in the human genome; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1350-1356; " /><meta name="citation_reference" content="citation_title=Compositional evolution of noncoding DNA in the human and chimpanzee genomes; Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=278-286; " /><meta name="citation_reference" content="citation_title=Inaccurate reconstruction of ancestral GC levels creates a ‘vanishing isochores’ effect; Mol. Phylogenet. Evol.;  citation_year=2004;  citation_volume=31;  citation_pages=788-793; " /><meta name="citation_reference" content="citation_title=Are GC-rich isochores vanishing in mammals?; Gene;  citation_year=2006;  citation_volume=385;  citation_pages=50-56; " /><meta name="citation_reference" content="citation_title=SPEED: a molecular-evolution-based database of mammalian orthologous groups; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2835-2837; " /><meta name="citation_reference" content="CodonW;  citation_year=1997;  citation_volume=v. 1.4.2;  citation_pages=1.4.2" /><meta name="citation_reference" content="citation_title=The compositional properties of human genes; J. Mol. Evol.;  citation_year=1991;  citation_volume=32;  citation_pages=493-503; " /><meta name="citation_reference" content="citation_title=The mosaic genome of warm-blooded vertebrates; Science;  citation_year=1985;  citation_volume=228;  citation_pages=953-958; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-4680; " /><meta name="citation_reference" content="citation_title=The codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications; Nucleic Acids Res.;  citation_year=1987;  citation_volume=15;  citation_pages=1281-1295; " /><meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions; Mol. Biol. Evol.;  citation_year=1986;  citation_volume=3;  citation_pages=418-426; " /><meta name="citation_reference" content="citation_title=Nucleotide bias causes a genomewide bias in the amino acid composition of proteins; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=1581-1588; " /><meta name="citation_reference" content="citation_title=Mutational bias affects protein evolution in flowering plants; Mol. Biol. Evol.;  citation_year=2004;  citation_volume=21;  citation_pages=90-96; " /><meta name="citation_reference" content="citation_title=Negative correlation of G + C content at silent substitution sites between orthologous human and mouse protein-coding sequences; DNA Res.;  citation_year=2006;  citation_volume=13;  citation_pages=135-140; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/4/141/17462542/dsm015.pdf" /><meta name="description" content="Abstract.  Genome-wide analysis of sequence divergence patterns in 12 024 human–mouse orthologous pairs reveals, for the first time, that the trends in nucleoti" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/4/141/466067" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/4/141/466067" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/4/141/466067");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm015';
    </script>
<title>
	Reverse Polarization in Amino acid and Nucleotide Substitution Patterns Between Human–Mouse Orthologs of Two Compositional Extrema | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./466067" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeVCPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYWcsIFN1bWl0IEsuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgQ2VudHJlLCBJbmRpYW4gSW5zdGl0dXRlIG9mIENoZW1pY2FsIEJpb2xvZ3ksIEtvbGthdGEgNzAwIDAzMiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXVsLCBTYW5kaXAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9pbmZvcm1hdGljcyBDZW50cmUsIEluZGlhbiBJbnN0aXR1dGUgb2YgQ2hlbWljYWwgQmlvbG9neSwgS29sa2F0YSA3MDAgMDMyLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikdob3NoLCBTdWJoYWdhdGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdHJ1Y3R1cmFsIEJpb2xvZ3kgYW5kIEJpb2luZm9ybWF0aWNzIERpdmlzaW9uLCBJbmRpYW4gSW5zdGl0dXRlIG9mIENoZW1pY2FsIEJpb2xvZ3ksIDQsIFJhamEgUy4gQy4gTXVsbGljayBSb2FkLCBLb2xrYXRhIDcwMCAwMzIsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRHV0dGEsIENoaXRyYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIENlbnRyZSwgSW5kaWFuIEluc3RpdHV0ZSBvZiBDaGVtaWNhbCBCaW9sb2d5LCBLb2xrYXRhIDcwMCAwMzIsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RydWN0dXJhbCBCaW9sb2d5IGFuZCBCaW9pbmZvcm1hdGljcyBEaXZpc2lvbiwgSW5kaWFuIEluc3RpdHV0ZSBvZiBDaGVtaWNhbCBCaW9sb2d5LCA0LCBSYWphIFMuIEMuIE11bGxpY2sgUm9hZCwgS29sa2F0YSA3MDAgMDMyLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iUmV2ZXJzZSBQb2xhcml6YXRpb24gaW4gQW1pbm8gYWNpZCBhbmQgTnVjbGVvdGlkZSBTdWJzdGl0dXRpb24gUGF0dGVybnMgQmV0d2VlbiBIdW1hbuKAk01vdXNlIE9ydGhvbG9ncyBvZiBUd28gQ29tcG9zaXRpb25hbCBFeHRyZW1hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxNTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc20wMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNy8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2Nob3JlcyBhbmQgdGhlIGV2b2x1dGlvbmFyeSBnZW5vbWljcyBvZiB2ZXJ0ZWJyYXRlczsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDE7ICBjaXRhdGlvbl9wYWdlcz0zLTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIGlzb2Nob3JlczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NTQ5LTU1NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYW5hbHlzaXMgb2YgdGhlIGJvdmluZSBnZW5vbWUgYnkgQ3MyU080LUFnIGRlbnNpdHkgZ3JhZGllbnQgY2VudHJpZnVnYXRpb247IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5NzM7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0xNzctMTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XYXJtLWJsb29kZWQgaXNvY2hvcmUgc3RydWN0dXJlIGluIE5pbGUgY3JvY29kaWxlIGFuZCB0dXJ0bGU7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xNTIxLTE1Mjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluaXRpYWwgc2VxdWVuY2luZyBhbmQgYW5hbHlzaXMgb2YgdGhlIGh1bWFuIGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQwOTsgIGNpdGF0aW9uX3BhZ2VzPTg2MC05MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBodW1hbiB0cmFuc2NyaXB0b21lIG1hcDogY2x1c3RlcmluZyBvZiBoaWdobHkgZXhwcmVzc2VkIGdlbmVzIGluIGNocm9tb3NvbWFsIGRvbWFpbnM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjkxOyAgY2l0YXRpb25fcGFnZXM9MTI4OS0xMjkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb2NhbCByYXRlcyBvZiByZWNvbWJpbmF0aW9uIGFyZSBwb3NpdGl2ZWx5IGNvcnJlbGF0ZWQgd2l0aCBHQyBjb250ZW50IGluIHRoZSBodW1hbiBnZW5vbWU7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xMTM5LTExNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaC1yZXNvbHV0aW9uIHJlY29tYmluYXRpb24gbWFwIG9mIHRoZSBodW1hbiBnZW5vbWU7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjQxLTI0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB1bmlmaWNhdGlvbiBvZiBtb3NhaWMgc3RydWN0dXJlcyBpbiB0aGUgaHVtYW4gZ2Vub21lOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjQxMS0yNDE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZGlzdHJpYnV0aW9uIG9mIGdlbmVzIGluIHRoZSBodW1hbiBnZW5vbWU7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MTgxLTE4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb25zIGJldHdlZW4gaXNvY2hvcmVzIGFuZCBjaHJvbW9zb21hbCBiYW5kcyBpbiB0aGUgaHVtYW4gZ2Vub21lOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPTExOTI5LTExOTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBtb3VzZSBnZW5vbWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MjA7ICBjaXRhdGlvbl9wYWdlcz01MjAtNTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgV2lsaGVsbWluZSBFLiBLZXkgMjAwMSBJbnZpdGF0aW9uYWwgTGVjdHVyZS4gRXN0aW1hdGlvbiBvZiBkaXZlcmdlbmNlIHRpbWVzIGZvciBhIGZldyBtYW1tYWxpYW4gYW5kIHNldmVyYWwgcHJpbWF0ZSBzcGVjaWVzOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz0xNTctMTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgZGF0aW5nOiBhcGUgYm9uZXMgYWdyZWUgd2l0aCBjaGlja2VuIGVudHJhaWxzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9ODktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3RpbmN0IGNoYW5nZXMgb2YgZ2Vub21pYyBiaWFzZXMgaW4gbnVjbGVvdGlkZSBzdWJzdGl0dXRpb24gYXQgdGhlIHRpbWUgb2YgTWFtbWFsaWFuIHJhZGlhdGlvbjsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE4ODctMTg5NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFuaXNoaW5nIEdDLXJpY2ggaXNvY2hvcmVzIGluIG1hbW1hbGlhbiBnZW5vbWVzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjI7ICBjaXRhdGlvbl9wYWdlcz0xODM3LTE4NDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVybWluaXN0aWMgbXV0YXRpb24gcmF0ZSB2YXJpYXRpb24gaW4gdGhlIGh1bWFuIGdlbm9tZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMzUwLTEzNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBvc2l0aW9uYWwgZXZvbHV0aW9uIG9mIG5vbmNvZGluZyBETkEgaW4gdGhlIGh1bWFuIGFuZCBjaGltcGFuemVlIGdlbm9tZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yNzgtMjg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmFjY3VyYXRlIHJlY29uc3RydWN0aW9uIG9mIGFuY2VzdHJhbCBHQyBsZXZlbHMgY3JlYXRlcyBhIOKAmHZhbmlzaGluZyBpc29jaG9yZXPigJkgZWZmZWN0OyBNb2wuIFBoeWxvZ2VuZXQuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9Nzg4LTc5MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJlIEdDLXJpY2ggaXNvY2hvcmVzIHZhbmlzaGluZyBpbiBtYW1tYWxzPzsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODU7ICBjaXRhdGlvbl9wYWdlcz01MC01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1BFRUQ6IGEgbW9sZWN1bGFyLWV2b2x1dGlvbi1iYXNlZCBkYXRhYmFzZSBvZiBtYW1tYWxpYW4gb3J0aG9sb2dvdXMgZ3JvdXBzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTI4MzUtMjgzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQ29kb25XOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPXYuIDEuNC4yOyAgY2l0YXRpb25fcGFnZXM9MS40LjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcG9zaXRpb25hbCBwcm9wZXJ0aWVzIG9mIGh1bWFuIGdlbmVzOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NDkzLTUwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vc2FpYyBnZW5vbWUgb2Ygd2FybS1ibG9vZGVkIHZlcnRlYnJhdGVzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTIyODsgIGNpdGF0aW9uX3BhZ2VzPTk1My05NTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNUQUwgVzogaW1wcm92aW5nIHRoZSBzZW5zaXRpdml0eSBvZiBwcm9ncmVzc2l2ZSBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgdGhyb3VnaCBzZXF1ZW5jZSB3ZWlnaHRpbmcsIHBvc2l0aW9uLXNwZWNpZmljIGdhcCBwZW5hbHRpZXMgYW5kIHdlaWdodCBtYXRyaXggY2hvaWNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00NjczLTQ2ODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb2RvbiBBZGFwdGF0aW9uIEluZGV44oCUYSBtZWFzdXJlIG9mIGRpcmVjdGlvbmFsIHN5bm9ueW1vdXMgY29kb24gdXNhZ2UgYmlhcywgYW5kIGl0cyBwb3RlbnRpYWwgYXBwbGljYXRpb25zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMjgxLTEyOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBtZXRob2RzIGZvciBlc3RpbWF0aW5nIHRoZSBudW1iZXJzIG9mIHN5bm9ueW1vdXMgYW5kIG5vbnN5bm9ueW1vdXMgbnVjbGVvdGlkZSBzdWJzdGl0dXRpb25zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz00MTgtNDI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW90aWRlIGJpYXMgY2F1c2VzIGEgZ2Vub21ld2lkZSBiaWFzIGluIHRoZSBhbWlubyBhY2lkIGNvbXBvc2l0aW9uIG9mIHByb3RlaW5zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTU4MS0xNTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbmFsIGJpYXMgYWZmZWN0cyBwcm90ZWluIGV2b2x1dGlvbiBpbiBmbG93ZXJpbmcgcGxhbnRzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9OTAtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5lZ2F0aXZlIGNvcnJlbGF0aW9uIG9mIEcgKyBDIGNvbnRlbnQgYXQgc2lsZW50IHN1YnN0aXR1dGlvbiBzaXRlcyBiZXR3ZWVuIG9ydGhvbG9nb3VzIGh1bWFuIGFuZCBtb3VzZSBwcm90ZWluLWNvZGluZyBzZXF1ZW5jZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTM1LTE0MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE0LzQvMTQxLzE3NDYyNTQyL2RzbTAxNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgR2Vub21lLXdpZGUgYW5hbHlzaXMgb2Ygc2VxdWVuY2UgZGl2ZXJnZW5jZSBwYXR0ZXJucyBpbiAxMiAwMjQgaHVtYW7igJNtb3VzZSBvcnRob2xvZ291cyBwYWlycyByZXZlYWxzLCBmb3IgdGhlIGZpcnN0IHRpbWUsIHRoYXQgdGhlIHRyZW5kcyBpbiBudWNsZW90aSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDY2MDY3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZKDfW/fmVcRsc1iFU7GwF5VsfTHX" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f141%2f466067" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f141%2f466067" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18275" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&amp;Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/4">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87194663">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194663">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194665">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194665">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194668">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194668">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194689">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194689">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194712">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194712">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194720">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194720">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194722">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194722">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194724">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194724">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/14/4/155/466553">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Reverse Polarization in Amino acid and Nucleotide Substitution Patterns Between Human–Mouse Orthologs of Two Compositional Extrema
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sumit K. Bag</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Bioinformatics Centre</div>, <div class="insititution">Indian Institute of Chemical Biology</div>, <div class="addr-line">Kolkata 700 032</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sumit+K.+Bag">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bag S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bag S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sumit K. Bag</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sandip Paul</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Bioinformatics Centre</div>, <div class="insititution">Indian Institute of Chemical Biology</div>, <div class="addr-line">Kolkata 700 032</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sandip+Paul">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Paul S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Paul S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sandip Paul</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Subhagata Ghosh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Structural Biology and Bioinformatics Division</div>, <div class="insititution">Indian Institute of Chemical Biology</div>, <div class="addr-line">4, Raja S. C. Mullick Road, Kolkata 700 032</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Subhagata+Ghosh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ghosh S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ghosh S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Subhagata Ghosh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chitra Dutta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Bioinformatics Centre</div>, <div class="insititution">Indian Institute of Chemical Biology</div>, <div class="addr-line">Kolkata 700 032</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Structural Biology and Bioinformatics Division</div>, <div class="insititution">Indian Institute of Chemical Biology</div>, <div class="addr-line">4, Raja S. C. Mullick Road, Kolkata 700 032</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 33-2473-3491. Fax. <div class="fax">+91 33-2473-0284</div>. E-mail: <a href="mailto:cdutta@iicb.res.in" target="_blank">cdutta@iicb.res.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chitra+Dutta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dutta C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dutta C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chitra Dutta</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 4, 1 January 2007, Pages 141–154, <a href='https://doi.org/10.1093/dnares/dsm015'>https://doi.org/10.1093/dnares/dsm015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 September 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 June 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 June 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="466067" href="/dnaresearch/article-pdf/14/4/141/17462542/dsm015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Sumit K. Bag, Sandip Paul, Subhagata Ghosh, Chitra Dutta;  Reverse Polarization in Amino acid and Nucleotide Substitution Patterns Between Human–Mouse Orthologs of Two Compositional Extrema, <em>DNA Research</em>, Volume 14, Issue 4, 1 January 2007, Pages 141–154, <a href='https://doi.org/10.1093/dnares/dsm015'>https://doi.org/10.1093/dnares/dsm015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/466067?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/466067?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/466067?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/466067?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/466067?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/466067?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/466067?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Reverse%20Polarization%20in%20Amino%20acid%20and%20Nucleotide%20Substitution%20Patterns%20Between%20Human%E2%80%93Mouse%20Orthologs%20of%20Two%20Compositional%20Extrema&amp;publicationDate=2007-09-25&amp;volumeNum=14&amp;issueNum=4&amp;author=Bag%2C%20Sumit%20K.%3B%20Paul%2C%20Sandip&amp;startPage=141&amp;endPage=154&amp;contentId=10.1093%2Fdnares%2Fdsm015&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f141%2f466067" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87194663 id="87194663" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Genome-wide analysis of sequence divergence patterns in 12 024 human–mouse orthologous pairs reveals, for the first time, that the trends in nucleotide and amino acid substitutions in orthologs of high and low GC composition are highly asymmetric and polarized to opposite directions. The entire dataset has been divided into three groups on the basis of the GC content at third codon sites of human genes: high, medium, and low. High-GC orthologs exhibit significant bias in favor of the replacements, Thr → Ala, Ser → Ala, Val → Ala, Lys → Arg, Asn → Ser, Ile → Val etc., from mouse to human, whereas in low-GC orthologs, the reverse trends prevail. In general, in the high-GC group, residues encoded by A/U-rich codons of mouse proteins tend to be replaced by the residues encoded by relatively G/C-rich codons in their human orthologs, whereas the opposite trend is observed among the low-GC orthologous pairs. The medium-GC group shares some trends with high-GC group and some with low-GC group. The only significant trend common in all groups of orthologs, irrespective of their GC bias, is (Asp)<sub>Mouse</sub> → (Glu)<sub>Human</sub> replacement. At the nucleotide level, high-GC orthologs have undergone a large excess of (A/T)<sub>Mouse</sub> → (G/C)<sub>Human</sub> substitutions over (G/C)<sub>Mouse</sub> → (A/T)<sub>Human</sub> at each codon position, whereas for low-GC orthologs, the reverse is true.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-GC orthologs&quot;">high-GC orthologs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;low-GC orthologs&quot;">low-GC orthologs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;amino acid replacement matrix&quot;">amino acid replacement matrix</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;nucleotide replacement matrix&quot;">nucleotide replacement matrix</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence divergence&quot;">sequence divergence</a></div>                            <h2 scrollto-destination=87194665 id="87194665" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Mammalian genomes are highly heterogeneous in base composition. These are composed of long stretches of DNA with distinct GC composition, commonly known as the isochore structures<sup><a reveal-id="DSM015C1  DSM015C4" data-open="DSM015C1  DSM015C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> or GC-content domains.<sup><a reveal-id="DSM015C5" data-open="DSM015C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The local GC composition correlates with a number of important genomic features such as gene density, gene length, patterns of gene expression, repeat element distribution, recombination rate etc.<sup><a reveal-id="DSM015C6  DSM015C11" data-open="DSM015C6  DSM015C11" class="link link-ref link-reveal xref-bibr">6–11</a></sup> Evolutionary stability of the GC-content distribution has been demonstrated for mice and humans on a genome-wide level.<sup><a reveal-id="DSM015C12" data-open="DSM015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The GC-rich sequences from one genome were demonstrated to be GC rich in the other genome and vice versa. Finding such one-to-one correspondence between the local GC distribution patterns in mouse and human was, however, not trivial. Since the divergence of the rodent and primate lineages at around 84–121 million years ago,<sup><a reveal-id="DSM015C13" data-open="DSM015C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSM015C14" data-open="DSM015C14" class="link link-ref link-reveal xref-bibr">14</a></sup> multiple substitutions might have occurred at the same sites of a pair of mouse–human orthologs independently in two lineages and if there had not been a strong directionality of the selection process(es) prevailing over the random mutation and fixation, such multiple substitutions should have randomized the local GC distribution patterns in two genomes. Invariance of the overall patterns of GC distribution along the chromosomes of mouse and human, therefore, suggests that there might be some well-defined trends in the nucleotide and/or amino acid substitution patterns across these two species. The present study was designed to determine such trends, if any.</p><p>A number of efforts have been made earlier to determine the evolutionary trends in mammalian genomes, but no definite conclusion could be reached. On the basis of the analysis of orthologous gene sequences from closely related species, it has been proposed that GC-rich regions of primate and cetartiodactyl genomes are becoming GC poorer, i.e. GC-rich isochores are now vanishing in these lineages.<sup><a reveal-id="DSM015C15  DSM015C18" data-open="DSM015C15  DSM015C18" class="link link-ref link-reveal xref-bibr">15–18</a></sup> Alvarez-Valin <em>et al.</em>,<sup><a reveal-id="DSM015C19" data-open="DSM015C19" class="link link-ref link-reveal xref-bibr">19</a></sup> however, described the ‘vanishing isochores’ effect as an artifact created due to inaccurate reconstruction of ancestral GC levels in such studies,<sup><a reveal-id="DSM015C15" data-open="DSM015C15" class="link link-ref link-reveal xref-bibr">15</a></sup> offering an evidence for an AT substitution bias within the repetitive elements of mammals. On the contrary, the maximum parsimony analysis conducted by Gu and Li<sup><a reveal-id="DSM015C20" data-open="DSM015C20" class="link link-ref link-reveal xref-bibr">20</a></sup> advocated for recent enrichment of the GC content of GC-rich genes in some genomes, e.g. the rabbit. Therefore, the direction(s) of evolution of mammalian genes is a matter of conjecture. Did mammalian genes of varying GC bias follow distinct evolutionary trajectories, and if yes, to what extent could they influence the evolution of encoded proteins? In an attempt to address these questions, the present study carried out a genome-scale analysis of the trends in nucleotide and amino acid substitutions between human and mouse orthologous pairs of varying GC content. The analysis showed that indeed there exist definite trends not only in nucleotide, but also in amino acid substitution patterns between mouse and human orthologous pairs, and that these trends are, in general, highly asymmetric and polarized to the reverse directions in high-GC and low-GC sets of orthologs in such a way that in course of evolution, the compositional heterogeneity has been significantly enhanced in coding regions in human compared with that in mouse.</p>                            <h2 scrollto-destination=87194668 id="87194668" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87194669 id="87194669" class="section-title" data-legacy-id=s2a>2.1. Sequence retrieval</h3>
<p>Nucleotide sequences of 12 405 pairs of orthologous coding regions of human and mouse were extracted from the Searchable Prototype Experimental Evolutionary Database (SPEED)<sup><a reveal-id="DSM015C21" data-open="DSM015C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (<a class="link link-uri" href="http://www.bioinfobase.umkc.edu/speed/">http://www.bioinfobase.umkc.edu/speed/</a>), using an in-house program developed in Perl. To minimize the sampling errors, a total of 174 sequences, which were shorter than 100 codons in either organism, were excluded from the analysis. The remaining sequences were subjected to a codon integrity check using a freely available program, CodonW<sup><a reveal-id="DSM015C22" data-open="DSM015C22" class="link link-ref link-reveal xref-bibr">22</a></sup> (<a class="link link-uri" href="http://www.molbiol.ox.ac.uk/cu/">http://www.molbiol.ox.ac.uk/cu/</a>), and the dataset was further screened for removing redundant sequences. The final dataset of human mouse orthologs contains 12 024 nonredundant sequences. We generated corresponding nonredundant protein sequence using C program developed in-house.</p>                            <h3 scrollto-destination=87194671 id="87194671" class="section-title" data-legacy-id=s2b>2.2. Classification of orthologs in three compositional groups</h3>
<p>The pairs of orthologous sequences under study exhibited significant correlations, not only between the overall GC contents, but also between the GC contents at third codon sites (GC<sub>3</sub>) as shown in Fig. <a reveal-id="DSM015F1" data-open="DSM015F1" class="link link-reveal link-table xref-fig">1</a>A and B. These orthologs were then classified into three groups according to the GC<sub>3</sub> contents of the human genes: the low-GC group with (GC<sub>3</sub>)<sub>Human</sub> &lt; 50%, the medium-GC group with 50% ≤ (GC<sub>3</sub>)<sub>Human</sub> ≤ 70%, and the high-GC group with (GC<sub>3</sub>)<sub>Human</sub> &gt; 70%. The numbers of pairs of orthologous genes in these three groups were comparable with one another (3896, 3960, and 4168 numbers in high-, medium-, and low-GC groups, respectively). The sequences in these three groups were used to examine the trends in amino acid and nucleotide substitution patterns.</p>                        <a id="87194673" scrollto-destination="87194673"></a>
<div id="DSM015F1" data-id="DSM015F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/m_dsm01501.jpeg?Expires=2147483647&Signature=cEzOhWnuH2kFjWXKOJmcPKGakaveFQOQs0zjk5uTeTwXJlwJRtVkaW-UghAHEPTZZwba2heKc10UUz6WEV9IH33dLGnL~rfOQnjLgfK3xvy0EbnNFDiEU4WnNptxPH2QctMu-QSuDsstrSXPcAmmGsxeulwzZL3GDI1wg4rZxRmApci9jv4Ool0-buJIVaus4BkUFSWis10lEBsXyj0CmFosZJ4mdk7VXOtAHu1YWI5EeVNACl1VEximJLNjDFVjGUe1jUUZkbLYpD1mF6LOjihRhbazS8iYA7NMqMYmu0fiohXZO~8O8IWBE2lX3pkkYQAss2mGSbSPZcEd-LJNjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of (A) overall GC content (%) and (B) GC content at third codon sites (%) of 12 024 orthologous genes of human and mouse with their correlation coefficient values." path-from-xml="dsm01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194673/dsm01501.jpeg" path-from-xml="dsm01501.jpeg" target="_blank">View large</a><a section="87194673" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/dsm01501.jpeg?Expires=2147483647&Signature=szN1WdqAT4OD4u~u8kKwDnJyCqgrvq0ExHk-dx8A5Sm~74xqkg7aEHydZBKmg5L-DYuisq7KHuDB7z92BbXJhWIqBvSSIQwJYJpyUwv~49cg4ZEoO7zYdQuYlY4pVZy9nNIok3HYtdR0mWA~Kg8A~sd8SegSkoDdXpmTawyS1szkzXTCxI6SuDfbT6Zj0XYAXu5DRzMQEKV9LKXsiGmxT0I5AW50O-7qRj06Eu6CYuEbpBIehKl5QNxponIQipQ9aDLNs2W98DoHfFEw~CtPwPULfUdhJWdoili96EM8gZR3Ocv12yjfyLVdWXeOflLrHFrI5QREo9gmFaIKTY12Bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194673&ar=466067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of (<strong>A</strong>) overall GC content (%) and (<strong>B</strong>) GC content at third codon sites (%) of 12 024 orthologous genes of human and mouse with their correlation coefficient values.</p></div></div><div content-id="DSM015F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/m_dsm01501.jpeg?Expires=2147483647&Signature=cEzOhWnuH2kFjWXKOJmcPKGakaveFQOQs0zjk5uTeTwXJlwJRtVkaW-UghAHEPTZZwba2heKc10UUz6WEV9IH33dLGnL~rfOQnjLgfK3xvy0EbnNFDiEU4WnNptxPH2QctMu-QSuDsstrSXPcAmmGsxeulwzZL3GDI1wg4rZxRmApci9jv4Ool0-buJIVaus4BkUFSWis10lEBsXyj0CmFosZJ4mdk7VXOtAHu1YWI5EeVNACl1VEximJLNjDFVjGUe1jUUZkbLYpD1mF6LOjihRhbazS8iYA7NMqMYmu0fiohXZO~8O8IWBE2lX3pkkYQAss2mGSbSPZcEd-LJNjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of (A) overall GC content (%) and (B) GC content at third codon sites (%) of 12 024 orthologous genes of human and mouse with their correlation coefficient values." path-from-xml="dsm01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194673/dsm01501.jpeg" path-from-xml="dsm01501.jpeg" target="_blank">View large</a><a section="87194673" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/dsm01501.jpeg?Expires=2147483647&Signature=szN1WdqAT4OD4u~u8kKwDnJyCqgrvq0ExHk-dx8A5Sm~74xqkg7aEHydZBKmg5L-DYuisq7KHuDB7z92BbXJhWIqBvSSIQwJYJpyUwv~49cg4ZEoO7zYdQuYlY4pVZy9nNIok3HYtdR0mWA~Kg8A~sd8SegSkoDdXpmTawyS1szkzXTCxI6SuDfbT6Zj0XYAXu5DRzMQEKV9LKXsiGmxT0I5AW50O-7qRj06Eu6CYuEbpBIehKl5QNxponIQipQ9aDLNs2W98DoHfFEw~CtPwPULfUdhJWdoili96EM8gZR3Ocv12yjfyLVdWXeOflLrHFrI5QREo9gmFaIKTY12Bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194673&ar=466067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of (<strong>A</strong>) overall GC content (%) and (<strong>B</strong>) GC content at third codon sites (%) of 12 024 orthologous genes of human and mouse with their correlation coefficient values.</p></div></div><p>The total dataset was also classified into another three groups on the basis of GC<sub>3</sub> contents of the mouse genes: the new low-GC group with (GC<sub>3</sub>)<sub>Mouse</sub>&lt; 50%, the new medium-GC group with 50% ≤ (GC<sub>3</sub>)<sub>Mouse</sub> ≤ 70%, and the new high-GC group with (GC<sub>3</sub>)<sub>Mouse</sub> &gt; 70%. The entire study carried out with the high-, medium-, and low-GC groups were re-checked with these new high-, new medium-, and new low-GC groups of orthologs.</p><p>We classified the datasets on the basis of the GC<sub>3</sub> of coding sequences rather than the overall GC, because the GC<sub>3</sub> contents of mammalian genes are known to exhibit strong correlation with the GC content of the genomic region, where the genes are located.<sup><a reveal-id="DSM015C23" data-open="DSM015C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSM015C24" data-open="DSM015C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h3 scrollto-destination=87194676 id="87194676" class="section-title" data-legacy-id=s2c>2.3. Analysis of amino acid substitution patterns: evaluation of amino acid replacement matrix (AARM) for different groups of orthologs</h3>
<p>The alignments of orthologous sequences of three groups were created separately using the pairwise alignment program ClustalW<sup><a reveal-id="DSM015C25" data-open="DSM015C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and only the gap-free aligned regions of length &gt;100 residues were considered to avoid any spurious short-alignment regions. The numbers of pairs of aligned orthologous genes in three groups of sequences with gap-free regions of length &gt;100 residues were less than the previous set (3659, 2669, and 3291 numbers in high-, medium-, and low-GC groups, respectively). Amino acid replacements were calculated for all gap-free alignment regions of &gt;100 residues and also for fully aligned sequences including gaps. The replacement data are represented in a 20 × 20 matrix, designated as amino acid replacement matrix (AARM), as shown in Tables <a reveal-id="DSM015TB1" data-open="DSM015TB1" class="link link-reveal link-table xref-fig">1</a>, <a reveal-id="DSM015TB2" data-open="DSM015TB2" class="link link-reveal link-table xref-fig">2</a>, and <a reveal-id="DSM015TB3" data-open="DSM015TB3" class="link link-reveal link-table xref-fig">3</a> for gap-free alignment regions of &gt;100 residues. (To avoid confusion with the standard amino acid substitution matrices like PAM or BLOSUM, we have used the term ‘Replacement’ matrix.) The elements of AARM represent the ratio between the number of forward replacements and the number of backward replacements for any specific pairs of residues, i.e. the value of any element <em>R<sub>ij</sub></em> of the AARM represents the ratio of the total number of [<em>i</em>]<sub>Mouse</sub> → [<em>j</em>]<sub>Human</sub> replacements to the number of [<em>j</em>]<sub>Mouse</sub> → [<em>i</em>]<sub>Human</sub> replacements. If <em>R<sub>ij</sub></em> &gt; 1, then there will be an overall gain in the amino acid residue <em>j</em> at the cost of the amino acid residue <em>i</em> in human compared with that in mouse. If <em>R<sub>ij</sub></em> &lt; 1, the reverse will be true. The actual number of forward and backward replacements for all possible pairs of amino acid residues for high-, medium-, and low-GC groups are given in Supplementary Table S1a–c. Other than diagonal positions of the matrices (representing the identical substitutions), all other elements represent the nonidentical substitutions. The replacement values for the gap-free alignment regions were not changed significantly from the result obtained by alignment of full sequences including gaps. In order to test whether there were any significant intra-group variations in the replacement values, subsets of 500 pairs of sequences were taken sequentially from start to end and also randomly from the entire dataset of a specific group of orthologs (i.e. high-/medium-/low-GC group) and 20 × 20 AARM was determined for each subset of sequence pairs. Comparison of the replacement values obtained from different subsets of any particular group was then carried out, and no significant variations in substitution values were found for individual residue pairs within a group. All these computations were done using Substitution Pattern Analysis Software Tool (SPAST), a program in C, developed in-house.</p>                        <a id="87194678" scrollto-destination="87194678"></a>
<div content-id="DSM015TB1" class="table-modal"><div class="table-wrap"><div id="DSM015TB1" class="table-wrap-title" data-id="DSM015TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Amino Acid Replacement Matrix (AARM) for high-GC group in human mouse orthologs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td><strong>Mouse</strong> </td><td>Gly </td><td>– </td><td>0.93 </td><td><strong>0.84</strong> </td><td>1.02 </td><td>0.62 </td><td>0.35 </td><td>0.84 </td><td><strong>0.40</strong> </td><td><strong>0.47</strong> </td><td><strong>0.47</strong> </td><td><strong>0.67</strong> </td><td><strong>0.55</strong> </td><td>0.76 </td><td>0.86 </td><td><strong>0.72</strong> </td><td>0.76 </td><td><strong>0.77</strong> </td><td><strong>0.73</strong> </td><td>0.69 </td><td>0.83 </td></tr><tr><td> </td><td>Ala </td><td>1.08 </td><td>– </td><td>0.85 </td><td>1.04 </td><td>0.69 </td><td>0.40 </td><td><strong>0.62</strong> </td><td><strong>0.47</strong> </td><td><strong>0.49</strong> </td><td><strong>0.55</strong> </td><td><strong>0.59</strong> </td><td><strong>0.58</strong> </td><td>0.67 </td><td>1.00 </td><td><strong>0.70</strong> </td><td><strong><em>0.78</em></strong> </td><td><strong>0.71</strong> </td><td><strong>0.64</strong> </td><td>1.03 </td><td>0.79 </td></tr><tr><td> </td><td>Arg </td><td><strong>1.19</strong> </td><td>1.17 </td><td>– </td><td>0.87 </td><td><strong><em>0.45</em></strong> </td><td><strong>0.47</strong> </td><td>0.76 </td><td>0.72 </td><td><strong>0.56</strong> </td><td><strong>0.74</strong> </td><td><strong><em>0.85</em></strong> </td><td>1.00 </td><td>0.82 </td><td>0.87 </td><td>1.06 </td><td><strong>0.77</strong> </td><td>0.86 </td><td>0.94 </td><td><strong>0.71</strong> </td><td><strong>0.72</strong> </td></tr><tr><td> </td><td>Pro </td><td>0.98 </td><td>0.96 </td><td>1.15 </td><td>– </td><td><strong>0.44</strong> </td><td>0.48 </td><td>0.66 </td><td><strong><em>0.49</em></strong> </td><td>0.81 </td><td>0.81 </td><td><strong>0.58</strong> </td><td><strong>0.66</strong> </td><td>0.59 </td><td>0.74 </td><td><strong>0.63</strong> </td><td><strong>0.79</strong> </td><td>1.00 </td><td>0.82 </td><td><strong>0.72</strong> </td><td><strong>0.82</strong> </td></tr><tr><td> </td><td>Phe </td><td>1.62 </td><td>1.44 </td><td><strong><em>2.21</em></strong> </td><td><strong>2.25</strong> </td><td>– </td><td>1.13 </td><td>0.73 </td><td>0.82 </td><td>1.00 </td><td>0.67 </td><td>1.07 </td><td>0.67 </td><td>1.22 </td><td>1.50 </td><td>1.20 </td><td><strong>1.27</strong> </td><td>1.14 </td><td>0.78 </td><td>1.82 </td><td>1.06 </td></tr><tr><td> </td><td>Tyr </td><td>2.82 </td><td>2.50 </td><td><strong>2.12</strong> </td><td>2.09 </td><td>0.88 </td><td>– </td><td>1.00 </td><td>0.85 </td><td>1.13 </td><td>1.71 </td><td>1.31 </td><td>1.53 </td><td>1.25 </td><td>1.33 </td><td>0.70 </td><td>1.08 </td><td>1.83 </td><td>1.68 </td><td><strong>1.50</strong> </td><td>1.61 </td></tr><tr><td> </td><td>Met </td><td>1.20 </td><td><strong>1.60</strong> </td><td>1.32 </td><td>1.51 </td><td>1.38 </td><td>1.00 </td><td>– </td><td><strong>0.81</strong> </td><td>0.81 </td><td>0.99 </td><td>0.89 </td><td>0.90 </td><td>0.62 </td><td>1.00 </td><td><strong>1.21</strong> </td><td><strong>1.45</strong> </td><td><strong><em>1.83</em></strong> </td><td>1.07 </td><td>0.5 </td><td>1.25 </td></tr><tr><td> </td><td>Ile </td><td><strong>2.51</strong> </td><td><strong>2.12</strong> </td><td>1.39 </td><td><strong><em>2.02</em></strong> </td><td>1.22 </td><td>1.18 </td><td><strong>1.24</strong> </td><td>– </td><td>1.38 </td><td>1.07 </td><td><strong><em>1.42</em></strong> </td><td><strong>1.40</strong> </td><td>1.00 </td><td>0.33 </td><td><strong>1.22</strong> </td><td><strong>1.48</strong> </td><td>1.47 </td><td>0.65 </td><td>0.91 </td><td>1.64 </td></tr><tr><td> </td><td>Asn </td><td><strong>2.13</strong> </td><td><strong>2.03</strong> </td><td><strong>1.77</strong> </td><td>1.24 </td><td>1.00 </td><td>0.89 </td><td>1.23 </td><td>0.73 </td><td>– </td><td>1.12 </td><td><strong>1.34</strong> </td><td>1.11 </td><td>1.76 </td><td>– </td><td>1.24 </td><td>1.10 </td><td>1.42 </td><td><strong>1.26</strong> </td><td><strong>1.30</strong> </td><td>1.48 </td></tr><tr><td> </td><td>Lys </td><td><strong>2.15</strong> </td><td><strong>1.81</strong> </td><td><strong>1.36</strong> </td><td>1.24 </td><td>1.50 </td><td>0.58 </td><td>1.01 </td><td>0.94 </td><td>0.89 </td><td>– </td><td>1.39 </td><td>1.00 </td><td>0.80 </td><td><strong><em>3.67</em></strong> </td><td>1.76 </td><td>1.30 </td><td><strong>1.60</strong> </td><td>1.22 </td><td>1.61 </td><td><strong>1.47</strong> </td></tr><tr><td> </td><td>Ser </td><td><strong>1.49</strong> </td><td><strong>1.70</strong> </td><td><strong><em>1.18</em></strong> </td><td><strong>1.72</strong> </td><td>0.94 </td><td>0.76 </td><td>1.13 </td><td><strong><em>0.71</em></strong> </td><td><strong>0.75</strong> </td><td>0.72 </td><td>– </td><td>1.03 </td><td>0.93 </td><td>1.10 </td><td>1.00 </td><td>1.01 </td><td>1.36 </td><td>1.06 </td><td>1.15 </td><td>1.31 </td></tr><tr><td> </td><td>Thr </td><td><strong>1.83</strong> </td><td><strong>1.71</strong> </td><td>1.00 </td><td><strong>1.52</strong> </td><td>1.50 </td><td>0.65 </td><td>1.11 </td><td><strong>0.72</strong> </td><td>0.90 </td><td>1.00 </td><td>0.97 </td><td>– </td><td>1.16 </td><td>0.89 </td><td>1.08 </td><td>1.06 </td><td>1.37 </td><td>1.09 </td><td>0.91 </td><td>0.99 </td></tr><tr><td> </td><td>Cys </td><td>1.31 </td><td>1.50 </td><td>1.22 </td><td>1.69 </td><td>0.82 </td><td>0.80 </td><td>1.60 </td><td>1.00 </td><td>0.57 </td><td>1.25 </td><td>1.07 </td><td>0.86 </td><td>– </td><td>1.17 </td><td>1.04 </td><td>0.98 </td><td>0.75 </td><td>1.17 </td><td>1.01 </td><td>0.63 </td></tr><tr><td> </td><td>Trp </td><td>1.16 </td><td>1.00 </td><td>1.15 </td><td>1.35 </td><td>0.67 </td><td>0.75 </td><td>1.00 </td><td>3.00 </td><td>– </td><td><strong><em>0.27</em></strong> </td><td>0.91 </td><td>1.12 </td><td>0.85 </td><td>– </td><td>1.05 </td><td>0.94 </td><td>1.00 </td><td>0.33 </td><td>0.59 </td><td>0.76 </td></tr><tr><td> </td><td>Val </td><td><strong>1.40</strong> </td><td><strong>1.43</strong> </td><td>0.94 </td><td><strong>1.60</strong> </td><td>0.84 </td><td>1.43 </td><td><strong>0.82</strong> </td><td><strong>0.82</strong> </td><td>0.8 </td><td>0.57 </td><td>1.00 </td><td>0.93 </td><td>0.96 </td><td>0.95 </td><td>– </td><td>1.08 </td><td>0.81 </td><td>0.86 </td><td>1.17 </td><td>1.07 </td></tr><tr><td> </td><td>Leu </td><td>1.31 </td><td><strong><em>1.28</em></strong> </td><td><strong>1.31</strong> </td><td><strong>1.26</strong> </td><td><strong>0.79</strong> </td><td>0.92 </td><td><strong>0.69</strong> </td><td><strong>0.67</strong> </td><td>0.91 </td><td>0.77 </td><td>0.99 </td><td>0.94 </td><td>1.02 </td><td>1.07 </td><td>0.92 </td><td>– </td><td>1.51 </td><td>1.04 </td><td>1.16 </td><td>0.88 </td></tr><tr><td> </td><td>Glu </td><td><strong>1.30</strong> </td><td><strong>1.41</strong> </td><td>1.16 </td><td>1.00 </td><td>0.88 </td><td>0.55 </td><td><strong><em>0.55</em></strong> </td><td>0.68 </td><td>0.71 </td><td><strong>0.62</strong> </td><td>0.74 </td><td>0.73 </td><td>1.33 </td><td>1.00 </td><td>1.23 </td><td>0.66 </td><td>– </td><td><strong>0.85</strong> </td><td>0.79 </td><td><strong>0.86</strong> </td></tr><tr><td> </td><td>Asp </td><td><strong>1.38</strong> </td><td><strong>1.57</strong> </td><td>1.06 </td><td>1.22 </td><td>1.29 </td><td>0.59 </td><td>0.93 </td><td>1.53 </td><td><strong>0.80</strong> </td><td>0.82 </td><td>0.95 </td><td>0.92 </td><td>0.86 </td><td>3.00 </td><td>1.16 </td><td>0.96 </td><td><strong>1.17</strong> </td><td>– </td><td>0.94 </td><td>1.26 </td></tr><tr><td> </td><td>His </td><td>1.45 </td><td>0.97 </td><td><strong>1.41</strong> </td><td><strong>1.38</strong> </td><td>0.55 </td><td><strong>0.67</strong> </td><td>2.00 </td><td>1.10 </td><td><strong>0.77</strong> </td><td>0.62 </td><td>0.87 </td><td>1.10 </td><td>0.99 </td><td>1.70 </td><td>0.85 </td><td>0.86 </td><td>1.27 </td><td>1.06 </td><td>– </td><td>1.02 </td></tr><tr><td> </td><td>Gln </td><td>1.20 </td><td>1.27 </td><td><strong>1.38</strong> </td><td><strong>1.21</strong> </td><td>0.94 </td><td>0.62 </td><td>0.80 </td><td>0.61 </td><td>0.68 </td><td><strong>0.68</strong> </td><td>0.76 </td><td>1.01 </td><td>1.58 </td><td>1.31 </td><td>0.93 </td><td>1.13 </td><td><strong>1.17</strong> </td><td>0.79 </td><td>0.98 </td><td>– </td></tr></tbody></table></div><div content-id="DSM015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td><strong>Mouse</strong> </td><td>Gly </td><td>– </td><td>0.93 </td><td><strong>0.84</strong> </td><td>1.02 </td><td>0.62 </td><td>0.35 </td><td>0.84 </td><td><strong>0.40</strong> </td><td><strong>0.47</strong> </td><td><strong>0.47</strong> </td><td><strong>0.67</strong> </td><td><strong>0.55</strong> </td><td>0.76 </td><td>0.86 </td><td><strong>0.72</strong> </td><td>0.76 </td><td><strong>0.77</strong> </td><td><strong>0.73</strong> </td><td>0.69 </td><td>0.83 </td></tr><tr><td> </td><td>Ala </td><td>1.08 </td><td>– </td><td>0.85 </td><td>1.04 </td><td>0.69 </td><td>0.40 </td><td><strong>0.62</strong> </td><td><strong>0.47</strong> </td><td><strong>0.49</strong> </td><td><strong>0.55</strong> </td><td><strong>0.59</strong> </td><td><strong>0.58</strong> </td><td>0.67 </td><td>1.00 </td><td><strong>0.70</strong> </td><td><strong><em>0.78</em></strong> </td><td><strong>0.71</strong> </td><td><strong>0.64</strong> </td><td>1.03 </td><td>0.79 </td></tr><tr><td> </td><td>Arg </td><td><strong>1.19</strong> </td><td>1.17 </td><td>– </td><td>0.87 </td><td><strong><em>0.45</em></strong> </td><td><strong>0.47</strong> </td><td>0.76 </td><td>0.72 </td><td><strong>0.56</strong> </td><td><strong>0.74</strong> </td><td><strong><em>0.85</em></strong> </td><td>1.00 </td><td>0.82 </td><td>0.87 </td><td>1.06 </td><td><strong>0.77</strong> </td><td>0.86 </td><td>0.94 </td><td><strong>0.71</strong> </td><td><strong>0.72</strong> </td></tr><tr><td> </td><td>Pro </td><td>0.98 </td><td>0.96 </td><td>1.15 </td><td>– </td><td><strong>0.44</strong> </td><td>0.48 </td><td>0.66 </td><td><strong><em>0.49</em></strong> </td><td>0.81 </td><td>0.81 </td><td><strong>0.58</strong> </td><td><strong>0.66</strong> </td><td>0.59 </td><td>0.74 </td><td><strong>0.63</strong> </td><td><strong>0.79</strong> </td><td>1.00 </td><td>0.82 </td><td><strong>0.72</strong> </td><td><strong>0.82</strong> </td></tr><tr><td> </td><td>Phe </td><td>1.62 </td><td>1.44 </td><td><strong><em>2.21</em></strong> </td><td><strong>2.25</strong> </td><td>– </td><td>1.13 </td><td>0.73 </td><td>0.82 </td><td>1.00 </td><td>0.67 </td><td>1.07 </td><td>0.67 </td><td>1.22 </td><td>1.50 </td><td>1.20 </td><td><strong>1.27</strong> </td><td>1.14 </td><td>0.78 </td><td>1.82 </td><td>1.06 </td></tr><tr><td> </td><td>Tyr </td><td>2.82 </td><td>2.50 </td><td><strong>2.12</strong> </td><td>2.09 </td><td>0.88 </td><td>– </td><td>1.00 </td><td>0.85 </td><td>1.13 </td><td>1.71 </td><td>1.31 </td><td>1.53 </td><td>1.25 </td><td>1.33 </td><td>0.70 </td><td>1.08 </td><td>1.83 </td><td>1.68 </td><td><strong>1.50</strong> </td><td>1.61 </td></tr><tr><td> </td><td>Met </td><td>1.20 </td><td><strong>1.60</strong> </td><td>1.32 </td><td>1.51 </td><td>1.38 </td><td>1.00 </td><td>– </td><td><strong>0.81</strong> </td><td>0.81 </td><td>0.99 </td><td>0.89 </td><td>0.90 </td><td>0.62 </td><td>1.00 </td><td><strong>1.21</strong> </td><td><strong>1.45</strong> </td><td><strong><em>1.83</em></strong> </td><td>1.07 </td><td>0.5 </td><td>1.25 </td></tr><tr><td> </td><td>Ile </td><td><strong>2.51</strong> </td><td><strong>2.12</strong> </td><td>1.39 </td><td><strong><em>2.02</em></strong> </td><td>1.22 </td><td>1.18 </td><td><strong>1.24</strong> </td><td>– </td><td>1.38 </td><td>1.07 </td><td><strong><em>1.42</em></strong> </td><td><strong>1.40</strong> </td><td>1.00 </td><td>0.33 </td><td><strong>1.22</strong> </td><td><strong>1.48</strong> </td><td>1.47 </td><td>0.65 </td><td>0.91 </td><td>1.64 </td></tr><tr><td> </td><td>Asn </td><td><strong>2.13</strong> </td><td><strong>2.03</strong> </td><td><strong>1.77</strong> </td><td>1.24 </td><td>1.00 </td><td>0.89 </td><td>1.23 </td><td>0.73 </td><td>– </td><td>1.12 </td><td><strong>1.34</strong> </td><td>1.11 </td><td>1.76 </td><td>– </td><td>1.24 </td><td>1.10 </td><td>1.42 </td><td><strong>1.26</strong> </td><td><strong>1.30</strong> </td><td>1.48 </td></tr><tr><td> </td><td>Lys </td><td><strong>2.15</strong> </td><td><strong>1.81</strong> </td><td><strong>1.36</strong> </td><td>1.24 </td><td>1.50 </td><td>0.58 </td><td>1.01 </td><td>0.94 </td><td>0.89 </td><td>– </td><td>1.39 </td><td>1.00 </td><td>0.80 </td><td><strong><em>3.67</em></strong> </td><td>1.76 </td><td>1.30 </td><td><strong>1.60</strong> </td><td>1.22 </td><td>1.61 </td><td><strong>1.47</strong> </td></tr><tr><td> </td><td>Ser </td><td><strong>1.49</strong> </td><td><strong>1.70</strong> </td><td><strong><em>1.18</em></strong> </td><td><strong>1.72</strong> </td><td>0.94 </td><td>0.76 </td><td>1.13 </td><td><strong><em>0.71</em></strong> </td><td><strong>0.75</strong> </td><td>0.72 </td><td>– </td><td>1.03 </td><td>0.93 </td><td>1.10 </td><td>1.00 </td><td>1.01 </td><td>1.36 </td><td>1.06 </td><td>1.15 </td><td>1.31 </td></tr><tr><td> </td><td>Thr </td><td><strong>1.83</strong> </td><td><strong>1.71</strong> </td><td>1.00 </td><td><strong>1.52</strong> </td><td>1.50 </td><td>0.65 </td><td>1.11 </td><td><strong>0.72</strong> </td><td>0.90 </td><td>1.00 </td><td>0.97 </td><td>– </td><td>1.16 </td><td>0.89 </td><td>1.08 </td><td>1.06 </td><td>1.37 </td><td>1.09 </td><td>0.91 </td><td>0.99 </td></tr><tr><td> </td><td>Cys </td><td>1.31 </td><td>1.50 </td><td>1.22 </td><td>1.69 </td><td>0.82 </td><td>0.80 </td><td>1.60 </td><td>1.00 </td><td>0.57 </td><td>1.25 </td><td>1.07 </td><td>0.86 </td><td>– </td><td>1.17 </td><td>1.04 </td><td>0.98 </td><td>0.75 </td><td>1.17 </td><td>1.01 </td><td>0.63 </td></tr><tr><td> </td><td>Trp </td><td>1.16 </td><td>1.00 </td><td>1.15 </td><td>1.35 </td><td>0.67 </td><td>0.75 </td><td>1.00 </td><td>3.00 </td><td>– </td><td><strong><em>0.27</em></strong> </td><td>0.91 </td><td>1.12 </td><td>0.85 </td><td>– </td><td>1.05 </td><td>0.94 </td><td>1.00 </td><td>0.33 </td><td>0.59 </td><td>0.76 </td></tr><tr><td> </td><td>Val </td><td><strong>1.40</strong> </td><td><strong>1.43</strong> </td><td>0.94 </td><td><strong>1.60</strong> </td><td>0.84 </td><td>1.43 </td><td><strong>0.82</strong> </td><td><strong>0.82</strong> </td><td>0.8 </td><td>0.57 </td><td>1.00 </td><td>0.93 </td><td>0.96 </td><td>0.95 </td><td>– </td><td>1.08 </td><td>0.81 </td><td>0.86 </td><td>1.17 </td><td>1.07 </td></tr><tr><td> </td><td>Leu </td><td>1.31 </td><td><strong><em>1.28</em></strong> </td><td><strong>1.31</strong> </td><td><strong>1.26</strong> </td><td><strong>0.79</strong> </td><td>0.92 </td><td><strong>0.69</strong> </td><td><strong>0.67</strong> </td><td>0.91 </td><td>0.77 </td><td>0.99 </td><td>0.94 </td><td>1.02 </td><td>1.07 </td><td>0.92 </td><td>– </td><td>1.51 </td><td>1.04 </td><td>1.16 </td><td>0.88 </td></tr><tr><td> </td><td>Glu </td><td><strong>1.30</strong> </td><td><strong>1.41</strong> </td><td>1.16 </td><td>1.00 </td><td>0.88 </td><td>0.55 </td><td><strong><em>0.55</em></strong> </td><td>0.68 </td><td>0.71 </td><td><strong>0.62</strong> </td><td>0.74 </td><td>0.73 </td><td>1.33 </td><td>1.00 </td><td>1.23 </td><td>0.66 </td><td>– </td><td><strong>0.85</strong> </td><td>0.79 </td><td><strong>0.86</strong> </td></tr><tr><td> </td><td>Asp </td><td><strong>1.38</strong> </td><td><strong>1.57</strong> </td><td>1.06 </td><td>1.22 </td><td>1.29 </td><td>0.59 </td><td>0.93 </td><td>1.53 </td><td><strong>0.80</strong> </td><td>0.82 </td><td>0.95 </td><td>0.92 </td><td>0.86 </td><td>3.00 </td><td>1.16 </td><td>0.96 </td><td><strong>1.17</strong> </td><td>– </td><td>0.94 </td><td>1.26 </td></tr><tr><td> </td><td>His </td><td>1.45 </td><td>0.97 </td><td><strong>1.41</strong> </td><td><strong>1.38</strong> </td><td>0.55 </td><td><strong>0.67</strong> </td><td>2.00 </td><td>1.10 </td><td><strong>0.77</strong> </td><td>0.62 </td><td>0.87 </td><td>1.10 </td><td>0.99 </td><td>1.70 </td><td>0.85 </td><td>0.86 </td><td>1.27 </td><td>1.06 </td><td>– </td><td>1.02 </td></tr><tr><td> </td><td>Gln </td><td>1.20 </td><td>1.27 </td><td><strong>1.38</strong> </td><td><strong>1.21</strong> </td><td>0.94 </td><td>0.62 </td><td>0.80 </td><td>0.61 </td><td>0.68 </td><td><strong>0.68</strong> </td><td>0.76 </td><td>1.01 </td><td>1.58 </td><td>1.31 </td><td>0.93 </td><td>1.13 </td><td><strong>1.17</strong> </td><td>0.79 </td><td>0.98 </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each element <em>R<sub>ij</sub></em> in the AARM represents the ratio of number of replacements of the residue <em>i</em> by the residue <em>j</em> in the forward direction (mouse → human) to that in the reverse direction. This means that if <em>R<sub>ij</sub></em> &gt;1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements and if <em>R<sub>ij</sub></em> &lt;1, the reverse is true. Bold and Bold-italics ratios signifies the directional bias at <em>p</em> &lt; 0.0001 and 0.001 respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194678" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM015TB1" class="table-wrap-title" data-id="DSM015TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Amino Acid Replacement Matrix (AARM) for high-GC group in human mouse orthologs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td><strong>Mouse</strong> </td><td>Gly </td><td>– </td><td>0.93 </td><td><strong>0.84</strong> </td><td>1.02 </td><td>0.62 </td><td>0.35 </td><td>0.84 </td><td><strong>0.40</strong> </td><td><strong>0.47</strong> </td><td><strong>0.47</strong> </td><td><strong>0.67</strong> </td><td><strong>0.55</strong> </td><td>0.76 </td><td>0.86 </td><td><strong>0.72</strong> </td><td>0.76 </td><td><strong>0.77</strong> </td><td><strong>0.73</strong> </td><td>0.69 </td><td>0.83 </td></tr><tr><td> </td><td>Ala </td><td>1.08 </td><td>– </td><td>0.85 </td><td>1.04 </td><td>0.69 </td><td>0.40 </td><td><strong>0.62</strong> </td><td><strong>0.47</strong> </td><td><strong>0.49</strong> </td><td><strong>0.55</strong> </td><td><strong>0.59</strong> </td><td><strong>0.58</strong> </td><td>0.67 </td><td>1.00 </td><td><strong>0.70</strong> </td><td><strong><em>0.78</em></strong> </td><td><strong>0.71</strong> </td><td><strong>0.64</strong> </td><td>1.03 </td><td>0.79 </td></tr><tr><td> </td><td>Arg </td><td><strong>1.19</strong> </td><td>1.17 </td><td>– </td><td>0.87 </td><td><strong><em>0.45</em></strong> </td><td><strong>0.47</strong> </td><td>0.76 </td><td>0.72 </td><td><strong>0.56</strong> </td><td><strong>0.74</strong> </td><td><strong><em>0.85</em></strong> </td><td>1.00 </td><td>0.82 </td><td>0.87 </td><td>1.06 </td><td><strong>0.77</strong> </td><td>0.86 </td><td>0.94 </td><td><strong>0.71</strong> </td><td><strong>0.72</strong> </td></tr><tr><td> </td><td>Pro </td><td>0.98 </td><td>0.96 </td><td>1.15 </td><td>– </td><td><strong>0.44</strong> </td><td>0.48 </td><td>0.66 </td><td><strong><em>0.49</em></strong> </td><td>0.81 </td><td>0.81 </td><td><strong>0.58</strong> </td><td><strong>0.66</strong> </td><td>0.59 </td><td>0.74 </td><td><strong>0.63</strong> </td><td><strong>0.79</strong> </td><td>1.00 </td><td>0.82 </td><td><strong>0.72</strong> </td><td><strong>0.82</strong> </td></tr><tr><td> </td><td>Phe </td><td>1.62 </td><td>1.44 </td><td><strong><em>2.21</em></strong> </td><td><strong>2.25</strong> </td><td>– </td><td>1.13 </td><td>0.73 </td><td>0.82 </td><td>1.00 </td><td>0.67 </td><td>1.07 </td><td>0.67 </td><td>1.22 </td><td>1.50 </td><td>1.20 </td><td><strong>1.27</strong> </td><td>1.14 </td><td>0.78 </td><td>1.82 </td><td>1.06 </td></tr><tr><td> </td><td>Tyr </td><td>2.82 </td><td>2.50 </td><td><strong>2.12</strong> </td><td>2.09 </td><td>0.88 </td><td>– </td><td>1.00 </td><td>0.85 </td><td>1.13 </td><td>1.71 </td><td>1.31 </td><td>1.53 </td><td>1.25 </td><td>1.33 </td><td>0.70 </td><td>1.08 </td><td>1.83 </td><td>1.68 </td><td><strong>1.50</strong> </td><td>1.61 </td></tr><tr><td> </td><td>Met </td><td>1.20 </td><td><strong>1.60</strong> </td><td>1.32 </td><td>1.51 </td><td>1.38 </td><td>1.00 </td><td>– </td><td><strong>0.81</strong> </td><td>0.81 </td><td>0.99 </td><td>0.89 </td><td>0.90 </td><td>0.62 </td><td>1.00 </td><td><strong>1.21</strong> </td><td><strong>1.45</strong> </td><td><strong><em>1.83</em></strong> </td><td>1.07 </td><td>0.5 </td><td>1.25 </td></tr><tr><td> </td><td>Ile </td><td><strong>2.51</strong> </td><td><strong>2.12</strong> </td><td>1.39 </td><td><strong><em>2.02</em></strong> </td><td>1.22 </td><td>1.18 </td><td><strong>1.24</strong> </td><td>– </td><td>1.38 </td><td>1.07 </td><td><strong><em>1.42</em></strong> </td><td><strong>1.40</strong> </td><td>1.00 </td><td>0.33 </td><td><strong>1.22</strong> </td><td><strong>1.48</strong> </td><td>1.47 </td><td>0.65 </td><td>0.91 </td><td>1.64 </td></tr><tr><td> </td><td>Asn </td><td><strong>2.13</strong> </td><td><strong>2.03</strong> </td><td><strong>1.77</strong> </td><td>1.24 </td><td>1.00 </td><td>0.89 </td><td>1.23 </td><td>0.73 </td><td>– </td><td>1.12 </td><td><strong>1.34</strong> </td><td>1.11 </td><td>1.76 </td><td>– </td><td>1.24 </td><td>1.10 </td><td>1.42 </td><td><strong>1.26</strong> </td><td><strong>1.30</strong> </td><td>1.48 </td></tr><tr><td> </td><td>Lys </td><td><strong>2.15</strong> </td><td><strong>1.81</strong> </td><td><strong>1.36</strong> </td><td>1.24 </td><td>1.50 </td><td>0.58 </td><td>1.01 </td><td>0.94 </td><td>0.89 </td><td>– </td><td>1.39 </td><td>1.00 </td><td>0.80 </td><td><strong><em>3.67</em></strong> </td><td>1.76 </td><td>1.30 </td><td><strong>1.60</strong> </td><td>1.22 </td><td>1.61 </td><td><strong>1.47</strong> </td></tr><tr><td> </td><td>Ser </td><td><strong>1.49</strong> </td><td><strong>1.70</strong> </td><td><strong><em>1.18</em></strong> </td><td><strong>1.72</strong> </td><td>0.94 </td><td>0.76 </td><td>1.13 </td><td><strong><em>0.71</em></strong> </td><td><strong>0.75</strong> </td><td>0.72 </td><td>– </td><td>1.03 </td><td>0.93 </td><td>1.10 </td><td>1.00 </td><td>1.01 </td><td>1.36 </td><td>1.06 </td><td>1.15 </td><td>1.31 </td></tr><tr><td> </td><td>Thr </td><td><strong>1.83</strong> </td><td><strong>1.71</strong> </td><td>1.00 </td><td><strong>1.52</strong> </td><td>1.50 </td><td>0.65 </td><td>1.11 </td><td><strong>0.72</strong> </td><td>0.90 </td><td>1.00 </td><td>0.97 </td><td>– </td><td>1.16 </td><td>0.89 </td><td>1.08 </td><td>1.06 </td><td>1.37 </td><td>1.09 </td><td>0.91 </td><td>0.99 </td></tr><tr><td> </td><td>Cys </td><td>1.31 </td><td>1.50 </td><td>1.22 </td><td>1.69 </td><td>0.82 </td><td>0.80 </td><td>1.60 </td><td>1.00 </td><td>0.57 </td><td>1.25 </td><td>1.07 </td><td>0.86 </td><td>– </td><td>1.17 </td><td>1.04 </td><td>0.98 </td><td>0.75 </td><td>1.17 </td><td>1.01 </td><td>0.63 </td></tr><tr><td> </td><td>Trp </td><td>1.16 </td><td>1.00 </td><td>1.15 </td><td>1.35 </td><td>0.67 </td><td>0.75 </td><td>1.00 </td><td>3.00 </td><td>– </td><td><strong><em>0.27</em></strong> </td><td>0.91 </td><td>1.12 </td><td>0.85 </td><td>– </td><td>1.05 </td><td>0.94 </td><td>1.00 </td><td>0.33 </td><td>0.59 </td><td>0.76 </td></tr><tr><td> </td><td>Val </td><td><strong>1.40</strong> </td><td><strong>1.43</strong> </td><td>0.94 </td><td><strong>1.60</strong> </td><td>0.84 </td><td>1.43 </td><td><strong>0.82</strong> </td><td><strong>0.82</strong> </td><td>0.8 </td><td>0.57 </td><td>1.00 </td><td>0.93 </td><td>0.96 </td><td>0.95 </td><td>– </td><td>1.08 </td><td>0.81 </td><td>0.86 </td><td>1.17 </td><td>1.07 </td></tr><tr><td> </td><td>Leu </td><td>1.31 </td><td><strong><em>1.28</em></strong> </td><td><strong>1.31</strong> </td><td><strong>1.26</strong> </td><td><strong>0.79</strong> </td><td>0.92 </td><td><strong>0.69</strong> </td><td><strong>0.67</strong> </td><td>0.91 </td><td>0.77 </td><td>0.99 </td><td>0.94 </td><td>1.02 </td><td>1.07 </td><td>0.92 </td><td>– </td><td>1.51 </td><td>1.04 </td><td>1.16 </td><td>0.88 </td></tr><tr><td> </td><td>Glu </td><td><strong>1.30</strong> </td><td><strong>1.41</strong> </td><td>1.16 </td><td>1.00 </td><td>0.88 </td><td>0.55 </td><td><strong><em>0.55</em></strong> </td><td>0.68 </td><td>0.71 </td><td><strong>0.62</strong> </td><td>0.74 </td><td>0.73 </td><td>1.33 </td><td>1.00 </td><td>1.23 </td><td>0.66 </td><td>– </td><td><strong>0.85</strong> </td><td>0.79 </td><td><strong>0.86</strong> </td></tr><tr><td> </td><td>Asp </td><td><strong>1.38</strong> </td><td><strong>1.57</strong> </td><td>1.06 </td><td>1.22 </td><td>1.29 </td><td>0.59 </td><td>0.93 </td><td>1.53 </td><td><strong>0.80</strong> </td><td>0.82 </td><td>0.95 </td><td>0.92 </td><td>0.86 </td><td>3.00 </td><td>1.16 </td><td>0.96 </td><td><strong>1.17</strong> </td><td>– </td><td>0.94 </td><td>1.26 </td></tr><tr><td> </td><td>His </td><td>1.45 </td><td>0.97 </td><td><strong>1.41</strong> </td><td><strong>1.38</strong> </td><td>0.55 </td><td><strong>0.67</strong> </td><td>2.00 </td><td>1.10 </td><td><strong>0.77</strong> </td><td>0.62 </td><td>0.87 </td><td>1.10 </td><td>0.99 </td><td>1.70 </td><td>0.85 </td><td>0.86 </td><td>1.27 </td><td>1.06 </td><td>– </td><td>1.02 </td></tr><tr><td> </td><td>Gln </td><td>1.20 </td><td>1.27 </td><td><strong>1.38</strong> </td><td><strong>1.21</strong> </td><td>0.94 </td><td>0.62 </td><td>0.80 </td><td>0.61 </td><td>0.68 </td><td><strong>0.68</strong> </td><td>0.76 </td><td>1.01 </td><td>1.58 </td><td>1.31 </td><td>0.93 </td><td>1.13 </td><td><strong>1.17</strong> </td><td>0.79 </td><td>0.98 </td><td>– </td></tr></tbody></table></div><div content-id="DSM015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td><strong>Mouse</strong> </td><td>Gly </td><td>– </td><td>0.93 </td><td><strong>0.84</strong> </td><td>1.02 </td><td>0.62 </td><td>0.35 </td><td>0.84 </td><td><strong>0.40</strong> </td><td><strong>0.47</strong> </td><td><strong>0.47</strong> </td><td><strong>0.67</strong> </td><td><strong>0.55</strong> </td><td>0.76 </td><td>0.86 </td><td><strong>0.72</strong> </td><td>0.76 </td><td><strong>0.77</strong> </td><td><strong>0.73</strong> </td><td>0.69 </td><td>0.83 </td></tr><tr><td> </td><td>Ala </td><td>1.08 </td><td>– </td><td>0.85 </td><td>1.04 </td><td>0.69 </td><td>0.40 </td><td><strong>0.62</strong> </td><td><strong>0.47</strong> </td><td><strong>0.49</strong> </td><td><strong>0.55</strong> </td><td><strong>0.59</strong> </td><td><strong>0.58</strong> </td><td>0.67 </td><td>1.00 </td><td><strong>0.70</strong> </td><td><strong><em>0.78</em></strong> </td><td><strong>0.71</strong> </td><td><strong>0.64</strong> </td><td>1.03 </td><td>0.79 </td></tr><tr><td> </td><td>Arg </td><td><strong>1.19</strong> </td><td>1.17 </td><td>– </td><td>0.87 </td><td><strong><em>0.45</em></strong> </td><td><strong>0.47</strong> </td><td>0.76 </td><td>0.72 </td><td><strong>0.56</strong> </td><td><strong>0.74</strong> </td><td><strong><em>0.85</em></strong> </td><td>1.00 </td><td>0.82 </td><td>0.87 </td><td>1.06 </td><td><strong>0.77</strong> </td><td>0.86 </td><td>0.94 </td><td><strong>0.71</strong> </td><td><strong>0.72</strong> </td></tr><tr><td> </td><td>Pro </td><td>0.98 </td><td>0.96 </td><td>1.15 </td><td>– </td><td><strong>0.44</strong> </td><td>0.48 </td><td>0.66 </td><td><strong><em>0.49</em></strong> </td><td>0.81 </td><td>0.81 </td><td><strong>0.58</strong> </td><td><strong>0.66</strong> </td><td>0.59 </td><td>0.74 </td><td><strong>0.63</strong> </td><td><strong>0.79</strong> </td><td>1.00 </td><td>0.82 </td><td><strong>0.72</strong> </td><td><strong>0.82</strong> </td></tr><tr><td> </td><td>Phe </td><td>1.62 </td><td>1.44 </td><td><strong><em>2.21</em></strong> </td><td><strong>2.25</strong> </td><td>– </td><td>1.13 </td><td>0.73 </td><td>0.82 </td><td>1.00 </td><td>0.67 </td><td>1.07 </td><td>0.67 </td><td>1.22 </td><td>1.50 </td><td>1.20 </td><td><strong>1.27</strong> </td><td>1.14 </td><td>0.78 </td><td>1.82 </td><td>1.06 </td></tr><tr><td> </td><td>Tyr </td><td>2.82 </td><td>2.50 </td><td><strong>2.12</strong> </td><td>2.09 </td><td>0.88 </td><td>– </td><td>1.00 </td><td>0.85 </td><td>1.13 </td><td>1.71 </td><td>1.31 </td><td>1.53 </td><td>1.25 </td><td>1.33 </td><td>0.70 </td><td>1.08 </td><td>1.83 </td><td>1.68 </td><td><strong>1.50</strong> </td><td>1.61 </td></tr><tr><td> </td><td>Met </td><td>1.20 </td><td><strong>1.60</strong> </td><td>1.32 </td><td>1.51 </td><td>1.38 </td><td>1.00 </td><td>– </td><td><strong>0.81</strong> </td><td>0.81 </td><td>0.99 </td><td>0.89 </td><td>0.90 </td><td>0.62 </td><td>1.00 </td><td><strong>1.21</strong> </td><td><strong>1.45</strong> </td><td><strong><em>1.83</em></strong> </td><td>1.07 </td><td>0.5 </td><td>1.25 </td></tr><tr><td> </td><td>Ile </td><td><strong>2.51</strong> </td><td><strong>2.12</strong> </td><td>1.39 </td><td><strong><em>2.02</em></strong> </td><td>1.22 </td><td>1.18 </td><td><strong>1.24</strong> </td><td>– </td><td>1.38 </td><td>1.07 </td><td><strong><em>1.42</em></strong> </td><td><strong>1.40</strong> </td><td>1.00 </td><td>0.33 </td><td><strong>1.22</strong> </td><td><strong>1.48</strong> </td><td>1.47 </td><td>0.65 </td><td>0.91 </td><td>1.64 </td></tr><tr><td> </td><td>Asn </td><td><strong>2.13</strong> </td><td><strong>2.03</strong> </td><td><strong>1.77</strong> </td><td>1.24 </td><td>1.00 </td><td>0.89 </td><td>1.23 </td><td>0.73 </td><td>– </td><td>1.12 </td><td><strong>1.34</strong> </td><td>1.11 </td><td>1.76 </td><td>– </td><td>1.24 </td><td>1.10 </td><td>1.42 </td><td><strong>1.26</strong> </td><td><strong>1.30</strong> </td><td>1.48 </td></tr><tr><td> </td><td>Lys </td><td><strong>2.15</strong> </td><td><strong>1.81</strong> </td><td><strong>1.36</strong> </td><td>1.24 </td><td>1.50 </td><td>0.58 </td><td>1.01 </td><td>0.94 </td><td>0.89 </td><td>– </td><td>1.39 </td><td>1.00 </td><td>0.80 </td><td><strong><em>3.67</em></strong> </td><td>1.76 </td><td>1.30 </td><td><strong>1.60</strong> </td><td>1.22 </td><td>1.61 </td><td><strong>1.47</strong> </td></tr><tr><td> </td><td>Ser </td><td><strong>1.49</strong> </td><td><strong>1.70</strong> </td><td><strong><em>1.18</em></strong> </td><td><strong>1.72</strong> </td><td>0.94 </td><td>0.76 </td><td>1.13 </td><td><strong><em>0.71</em></strong> </td><td><strong>0.75</strong> </td><td>0.72 </td><td>– </td><td>1.03 </td><td>0.93 </td><td>1.10 </td><td>1.00 </td><td>1.01 </td><td>1.36 </td><td>1.06 </td><td>1.15 </td><td>1.31 </td></tr><tr><td> </td><td>Thr </td><td><strong>1.83</strong> </td><td><strong>1.71</strong> </td><td>1.00 </td><td><strong>1.52</strong> </td><td>1.50 </td><td>0.65 </td><td>1.11 </td><td><strong>0.72</strong> </td><td>0.90 </td><td>1.00 </td><td>0.97 </td><td>– </td><td>1.16 </td><td>0.89 </td><td>1.08 </td><td>1.06 </td><td>1.37 </td><td>1.09 </td><td>0.91 </td><td>0.99 </td></tr><tr><td> </td><td>Cys </td><td>1.31 </td><td>1.50 </td><td>1.22 </td><td>1.69 </td><td>0.82 </td><td>0.80 </td><td>1.60 </td><td>1.00 </td><td>0.57 </td><td>1.25 </td><td>1.07 </td><td>0.86 </td><td>– </td><td>1.17 </td><td>1.04 </td><td>0.98 </td><td>0.75 </td><td>1.17 </td><td>1.01 </td><td>0.63 </td></tr><tr><td> </td><td>Trp </td><td>1.16 </td><td>1.00 </td><td>1.15 </td><td>1.35 </td><td>0.67 </td><td>0.75 </td><td>1.00 </td><td>3.00 </td><td>– </td><td><strong><em>0.27</em></strong> </td><td>0.91 </td><td>1.12 </td><td>0.85 </td><td>– </td><td>1.05 </td><td>0.94 </td><td>1.00 </td><td>0.33 </td><td>0.59 </td><td>0.76 </td></tr><tr><td> </td><td>Val </td><td><strong>1.40</strong> </td><td><strong>1.43</strong> </td><td>0.94 </td><td><strong>1.60</strong> </td><td>0.84 </td><td>1.43 </td><td><strong>0.82</strong> </td><td><strong>0.82</strong> </td><td>0.8 </td><td>0.57 </td><td>1.00 </td><td>0.93 </td><td>0.96 </td><td>0.95 </td><td>– </td><td>1.08 </td><td>0.81 </td><td>0.86 </td><td>1.17 </td><td>1.07 </td></tr><tr><td> </td><td>Leu </td><td>1.31 </td><td><strong><em>1.28</em></strong> </td><td><strong>1.31</strong> </td><td><strong>1.26</strong> </td><td><strong>0.79</strong> </td><td>0.92 </td><td><strong>0.69</strong> </td><td><strong>0.67</strong> </td><td>0.91 </td><td>0.77 </td><td>0.99 </td><td>0.94 </td><td>1.02 </td><td>1.07 </td><td>0.92 </td><td>– </td><td>1.51 </td><td>1.04 </td><td>1.16 </td><td>0.88 </td></tr><tr><td> </td><td>Glu </td><td><strong>1.30</strong> </td><td><strong>1.41</strong> </td><td>1.16 </td><td>1.00 </td><td>0.88 </td><td>0.55 </td><td><strong><em>0.55</em></strong> </td><td>0.68 </td><td>0.71 </td><td><strong>0.62</strong> </td><td>0.74 </td><td>0.73 </td><td>1.33 </td><td>1.00 </td><td>1.23 </td><td>0.66 </td><td>– </td><td><strong>0.85</strong> </td><td>0.79 </td><td><strong>0.86</strong> </td></tr><tr><td> </td><td>Asp </td><td><strong>1.38</strong> </td><td><strong>1.57</strong> </td><td>1.06 </td><td>1.22 </td><td>1.29 </td><td>0.59 </td><td>0.93 </td><td>1.53 </td><td><strong>0.80</strong> </td><td>0.82 </td><td>0.95 </td><td>0.92 </td><td>0.86 </td><td>3.00 </td><td>1.16 </td><td>0.96 </td><td><strong>1.17</strong> </td><td>– </td><td>0.94 </td><td>1.26 </td></tr><tr><td> </td><td>His </td><td>1.45 </td><td>0.97 </td><td><strong>1.41</strong> </td><td><strong>1.38</strong> </td><td>0.55 </td><td><strong>0.67</strong> </td><td>2.00 </td><td>1.10 </td><td><strong>0.77</strong> </td><td>0.62 </td><td>0.87 </td><td>1.10 </td><td>0.99 </td><td>1.70 </td><td>0.85 </td><td>0.86 </td><td>1.27 </td><td>1.06 </td><td>– </td><td>1.02 </td></tr><tr><td> </td><td>Gln </td><td>1.20 </td><td>1.27 </td><td><strong>1.38</strong> </td><td><strong>1.21</strong> </td><td>0.94 </td><td>0.62 </td><td>0.80 </td><td>0.61 </td><td>0.68 </td><td><strong>0.68</strong> </td><td>0.76 </td><td>1.01 </td><td>1.58 </td><td>1.31 </td><td>0.93 </td><td>1.13 </td><td><strong>1.17</strong> </td><td>0.79 </td><td>0.98 </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each element <em>R<sub>ij</sub></em> in the AARM represents the ratio of number of replacements of the residue <em>i</em> by the residue <em>j</em> in the forward direction (mouse → human) to that in the reverse direction. This means that if <em>R<sub>ij</sub></em> &gt;1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements and if <em>R<sub>ij</sub></em> &lt;1, the reverse is true. Bold and Bold-italics ratios signifies the directional bias at <em>p</em> &lt; 0.0001 and 0.001 respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194678" target="_blank">View Large</a></div></div>                        <a id="87194679" scrollto-destination="87194679"></a>
<div content-id="DSM015TB2" class="table-modal"><div class="table-wrap"><div id="DSM015TB2" class="table-wrap-title" data-id="DSM015TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Amino Acid Replacement Matrix (AARM) for medium-GC group in human mouse orthologs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td>Mouse </td><td>Gly </td><td>– </td><td>0.92 </td><td>0.90 </td><td>1.09 </td><td>0.93 </td><td>0.9 </td><td>1.04 </td><td>0.76 </td><td>0.84 </td><td>0.83 </td><td>0.92 </td><td><strong><em>0.73</em></strong> </td><td>0.89 </td><td>0.90 </td><td>1.10 </td><td>1.20 </td><td>0.97 </td><td>0.88 </td><td>0.69 </td><td>0.80 </td></tr><tr><td> </td><td><strong>Ala</strong> </td><td>1.09 </td><td>– </td><td>1.19 </td><td>1.08 </td><td>1.19 </td><td>1.00 </td><td>0.9 </td><td>0.85 </td><td>0.97 </td><td>1.05 </td><td><strong>0.85</strong> </td><td><strong>0.85</strong> </td><td>0.8 </td><td>0.59 </td><td>0.95 </td><td>0.96 </td><td>0.90 </td><td><strong>0.74</strong> </td><td>1.06 </td><td>1.21 </td></tr><tr><td> </td><td>Arg </td><td>1.11 </td><td>0.84 </td><td>– </td><td>1.02 </td><td>0.91 </td><td>0.84 </td><td>0.85 </td><td>0.76 </td><td>1.04 </td><td><strong><em>1.09</em></strong> </td><td>1.10 </td><td>1.07 </td><td>1.07 </td><td>1.23 </td><td>0.91 </td><td>0.92 </td><td>1.12 </td><td>0.98 </td><td>0.95 </td><td><strong>0.89</strong> </td></tr><tr><td> </td><td>Pro </td><td>0.92 </td><td>0.93 </td><td>0.98 </td><td>– </td><td>0.76 </td><td>0.61 </td><td>1.21 </td><td>0.82 </td><td>0.79 </td><td>1.03 </td><td><strong>0.82</strong> </td><td>0.97 </td><td>0.62 </td><td>1.75 </td><td>0.77 </td><td><strong>0.85</strong> </td><td>1.05 </td><td>0.78 </td><td>1.11 </td><td>1.15 </td></tr><tr><td> </td><td>Phe </td><td>1.08 </td><td>0.84 </td><td>1.10 </td><td>1.32 </td><td>– </td><td><strong><em>1.19</em></strong> </td><td>0.72 </td><td>0.95 </td><td>1.41 </td><td>1.67 </td><td>0.98 </td><td>0.93 </td><td>0.97 </td><td>1.38 </td><td>0.82 </td><td>1.01 </td><td>1.80 </td><td>0.93 </td><td>1.62 </td><td>1.15 </td></tr><tr><td> </td><td>Tyr </td><td>1.11 </td><td>1.00 </td><td>1.19 </td><td>1.64 </td><td><strong><em>0.84</em></strong> </td><td>– </td><td>1.25 </td><td>0.78 </td><td>0.88 </td><td>0.94 </td><td>0.94 </td><td>1.12 </td><td>0.91 </td><td>1.32 </td><td>1.04 </td><td>0.84 </td><td>1.12 </td><td>0.86 </td><td>1.12 </td><td>0.88 </td></tr><tr><td> </td><td>Met </td><td>0.97 </td><td>1.11 </td><td>1.17 </td><td>0.82 </td><td>1.38 </td><td>0.80 </td><td>– </td><td>1.09 </td><td>0.96 </td><td>0.86 </td><td>0.77 </td><td><strong><em>0.86</em></strong> </td><td>1.11 </td><td>0.75 </td><td>0.98 </td><td>1.11 </td><td>0.91 </td><td>0.65 </td><td>0.93 </td><td>1.46 </td></tr><tr><td> </td><td>Ile </td><td>1.32 </td><td>1.18 </td><td>1.32 </td><td>1.21 </td><td>1.05 </td><td>1.29 </td><td>0.92 </td><td>– </td><td>1.13 </td><td>1.02 </td><td>1.03 </td><td>0.95 </td><td>1.28 </td><td>0.25 </td><td><strong>0.85</strong> </td><td>0.92 </td><td>1.33 </td><td>0.88 </td><td>0.83 </td><td>1.29 </td></tr><tr><td> </td><td>Asn </td><td>1.19 </td><td>1.04 </td><td>0.96 </td><td>1.27 </td><td>0.71 </td><td>1.13 </td><td>1.04 </td><td>0.88 </td><td>– </td><td>1.00 </td><td><strong>0.89</strong> </td><td>0.88 </td><td>0.76 </td><td>1.33 </td><td>0.89 </td><td>0.92 </td><td>1.07 </td><td>1.01 </td><td>0.98 </td><td>0.72 </td></tr><tr><td> </td><td>Lys </td><td>1.20 </td><td>0.95 </td><td><strong><em>0.92</em></strong> </td><td>0.97 </td><td>0.60 </td><td>1.06 </td><td>1.16 </td><td>0.98 </td><td>1.00 </td><td>– </td><td>0.89 </td><td>0.84 </td><td>0.73 </td><td>1.94 </td><td>1.05 </td><td>1.35 </td><td><strong><em>1.13</em></strong> </td><td>0.87 </td><td>0.81 </td><td>1.04 </td></tr><tr><td> </td><td>Ser </td><td>1.09 </td><td><strong>1.18</strong> </td><td>0.91 </td><td><strong>1.22</strong> </td><td>1.02 </td><td>1.06 </td><td>1.30 </td><td>0.97 </td><td><strong>1.13</strong> </td><td>1.13 </td><td>– </td><td><strong>1.17</strong> </td><td>0.91 </td><td>1.10 </td><td>1.08 </td><td>1.05 </td><td>1.14 </td><td>1.04 </td><td>1.05 </td><td>1.22 </td></tr><tr><td> </td><td>Thr </td><td><strong><em>1.38</em></strong> </td><td><strong>1.17</strong> </td><td>0.93 </td><td>1.03 </td><td>1.08 </td><td>0.89 </td><td><strong><em>1.16</em></strong> </td><td>1.06 </td><td>1.14 </td><td>1.20 </td><td><strong>0.85</strong> </td><td>– </td><td>0.86 </td><td>0.57 </td><td>1.06 </td><td>0.80 </td><td>1.05 </td><td>0.93 </td><td>0.80 </td><td>1.15 </td></tr><tr><td> </td><td>Cys </td><td>1.12 </td><td>1.24 </td><td>0.93 </td><td>1.62 </td><td>1.03 </td><td>1.10 </td><td>0.90 </td><td>0.78 </td><td>1.31 </td><td>1.36 </td><td>1.09 </td><td>1.16 </td><td>– </td><td>0.93 </td><td>0.84 </td><td>1.18 </td><td>1.00 </td><td>0.76 </td><td>0.93 </td><td>1.00 </td></tr><tr><td> </td><td>Trp </td><td>1.11 </td><td>1.70 </td><td>0.81 </td><td>0.57 </td><td>0.72 </td><td>0.76 </td><td>1.33 </td><td>4.00 </td><td>0.75 </td><td>0.52 </td><td>0.91 </td><td>1.75 </td><td>1.08 </td><td>– </td><td>1.30 </td><td>0.89 </td><td>0.76 </td><td>0.75 </td><td>0.55 </td><td>0.83 </td></tr><tr><td> </td><td>Val </td><td>0.91 </td><td>1.06 </td><td>1.10 </td><td>1.30 </td><td>1.21 </td><td>0.96 </td><td>1.02 </td><td><strong>1.17</strong> </td><td>1.12 </td><td>0.96 </td><td>0.92 </td><td>0.94 </td><td>1.19 </td><td>0.77 </td><td>– </td><td>1.05 </td><td>0.91 </td><td>0.72 </td><td>0.90 </td><td>1.02 </td></tr><tr><td> </td><td>Leu </td><td>0.83 </td><td>1.04 </td><td>1.08 </td><td><strong>1.18</strong> </td><td>0.99 </td><td>1.19 </td><td>0.90 </td><td>1.09 </td><td>1.09 </td><td>0.74 </td><td>0.95 </td><td>1.25 </td><td>0.84 </td><td>1.12 </td><td>0.95 </td><td>– </td><td>0.98 </td><td>1.39 </td><td>1.10 </td><td>0.94 </td></tr><tr><td> </td><td>Glu </td><td>1.03 </td><td>1.11 </td><td>0.89 </td><td>0.95 </td><td>0.56 </td><td>0.89 </td><td>1.10 </td><td>0.75 </td><td>0.93 </td><td><strong><em>0.88</em></strong> </td><td>0.87 </td><td>0.95 </td><td>1.00 </td><td>1.32 </td><td>1.10 </td><td>1.02 </td><td>– </td><td><strong>0.86</strong> </td><td>0.69 </td><td>0.91 </td></tr><tr><td> </td><td>Asp </td><td>1.14 </td><td><strong>1.35</strong> </td><td>1.03 </td><td>1.29 </td><td>1.07 </td><td>1.17 </td><td>1.55 </td><td>1.14 </td><td>0.99 </td><td>1.15 </td><td>0.96 </td><td>1.08 </td><td>1.32 </td><td>1.33 </td><td>1.38 </td><td>0.72 </td><td><strong>1.16</strong> </td><td>– </td><td>0.81 </td><td>0.83 </td></tr><tr><td> </td><td>His </td><td>1.45 </td><td>0.95 </td><td>1.06 </td><td>0.90 </td><td>0.62 </td><td>0.90 </td><td>1.08 </td><td>1.20 </td><td>1.02 </td><td>1.23 </td><td>0.95 </td><td>1.25 </td><td>1.07 </td><td>1.83 </td><td>1.11 </td><td>0.91 </td><td>1.46 </td><td>1.23 </td><td>– </td><td>1.06 </td></tr><tr><td> </td><td>Gln </td><td>1.25 </td><td>0.83 </td><td><strong>1.13</strong> </td><td>0.87 </td><td>0.87 </td><td>1.13 </td><td>0.69 </td><td>0.78 </td><td>1.38 </td><td>0.96 </td><td>0.82 </td><td>0.87 </td><td>1.00 </td><td>1.21 </td><td>0.98 </td><td>1.06 </td><td>1.09 </td><td>1.21 </td><td>0.94 </td><td>– </td></tr></tbody></table></div><div content-id="DSM015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td>Mouse </td><td>Gly </td><td>– </td><td>0.92 </td><td>0.90 </td><td>1.09 </td><td>0.93 </td><td>0.9 </td><td>1.04 </td><td>0.76 </td><td>0.84 </td><td>0.83 </td><td>0.92 </td><td><strong><em>0.73</em></strong> </td><td>0.89 </td><td>0.90 </td><td>1.10 </td><td>1.20 </td><td>0.97 </td><td>0.88 </td><td>0.69 </td><td>0.80 </td></tr><tr><td> </td><td><strong>Ala</strong> </td><td>1.09 </td><td>– </td><td>1.19 </td><td>1.08 </td><td>1.19 </td><td>1.00 </td><td>0.9 </td><td>0.85 </td><td>0.97 </td><td>1.05 </td><td><strong>0.85</strong> </td><td><strong>0.85</strong> </td><td>0.8 </td><td>0.59 </td><td>0.95 </td><td>0.96 </td><td>0.90 </td><td><strong>0.74</strong> </td><td>1.06 </td><td>1.21 </td></tr><tr><td> </td><td>Arg </td><td>1.11 </td><td>0.84 </td><td>– </td><td>1.02 </td><td>0.91 </td><td>0.84 </td><td>0.85 </td><td>0.76 </td><td>1.04 </td><td><strong><em>1.09</em></strong> </td><td>1.10 </td><td>1.07 </td><td>1.07 </td><td>1.23 </td><td>0.91 </td><td>0.92 </td><td>1.12 </td><td>0.98 </td><td>0.95 </td><td><strong>0.89</strong> </td></tr><tr><td> </td><td>Pro </td><td>0.92 </td><td>0.93 </td><td>0.98 </td><td>– </td><td>0.76 </td><td>0.61 </td><td>1.21 </td><td>0.82 </td><td>0.79 </td><td>1.03 </td><td><strong>0.82</strong> </td><td>0.97 </td><td>0.62 </td><td>1.75 </td><td>0.77 </td><td><strong>0.85</strong> </td><td>1.05 </td><td>0.78 </td><td>1.11 </td><td>1.15 </td></tr><tr><td> </td><td>Phe </td><td>1.08 </td><td>0.84 </td><td>1.10 </td><td>1.32 </td><td>– </td><td><strong><em>1.19</em></strong> </td><td>0.72 </td><td>0.95 </td><td>1.41 </td><td>1.67 </td><td>0.98 </td><td>0.93 </td><td>0.97 </td><td>1.38 </td><td>0.82 </td><td>1.01 </td><td>1.80 </td><td>0.93 </td><td>1.62 </td><td>1.15 </td></tr><tr><td> </td><td>Tyr </td><td>1.11 </td><td>1.00 </td><td>1.19 </td><td>1.64 </td><td><strong><em>0.84</em></strong> </td><td>– </td><td>1.25 </td><td>0.78 </td><td>0.88 </td><td>0.94 </td><td>0.94 </td><td>1.12 </td><td>0.91 </td><td>1.32 </td><td>1.04 </td><td>0.84 </td><td>1.12 </td><td>0.86 </td><td>1.12 </td><td>0.88 </td></tr><tr><td> </td><td>Met </td><td>0.97 </td><td>1.11 </td><td>1.17 </td><td>0.82 </td><td>1.38 </td><td>0.80 </td><td>– </td><td>1.09 </td><td>0.96 </td><td>0.86 </td><td>0.77 </td><td><strong><em>0.86</em></strong> </td><td>1.11 </td><td>0.75 </td><td>0.98 </td><td>1.11 </td><td>0.91 </td><td>0.65 </td><td>0.93 </td><td>1.46 </td></tr><tr><td> </td><td>Ile </td><td>1.32 </td><td>1.18 </td><td>1.32 </td><td>1.21 </td><td>1.05 </td><td>1.29 </td><td>0.92 </td><td>– </td><td>1.13 </td><td>1.02 </td><td>1.03 </td><td>0.95 </td><td>1.28 </td><td>0.25 </td><td><strong>0.85</strong> </td><td>0.92 </td><td>1.33 </td><td>0.88 </td><td>0.83 </td><td>1.29 </td></tr><tr><td> </td><td>Asn </td><td>1.19 </td><td>1.04 </td><td>0.96 </td><td>1.27 </td><td>0.71 </td><td>1.13 </td><td>1.04 </td><td>0.88 </td><td>– </td><td>1.00 </td><td><strong>0.89</strong> </td><td>0.88 </td><td>0.76 </td><td>1.33 </td><td>0.89 </td><td>0.92 </td><td>1.07 </td><td>1.01 </td><td>0.98 </td><td>0.72 </td></tr><tr><td> </td><td>Lys </td><td>1.20 </td><td>0.95 </td><td><strong><em>0.92</em></strong> </td><td>0.97 </td><td>0.60 </td><td>1.06 </td><td>1.16 </td><td>0.98 </td><td>1.00 </td><td>– </td><td>0.89 </td><td>0.84 </td><td>0.73 </td><td>1.94 </td><td>1.05 </td><td>1.35 </td><td><strong><em>1.13</em></strong> </td><td>0.87 </td><td>0.81 </td><td>1.04 </td></tr><tr><td> </td><td>Ser </td><td>1.09 </td><td><strong>1.18</strong> </td><td>0.91 </td><td><strong>1.22</strong> </td><td>1.02 </td><td>1.06 </td><td>1.30 </td><td>0.97 </td><td><strong>1.13</strong> </td><td>1.13 </td><td>– </td><td><strong>1.17</strong> </td><td>0.91 </td><td>1.10 </td><td>1.08 </td><td>1.05 </td><td>1.14 </td><td>1.04 </td><td>1.05 </td><td>1.22 </td></tr><tr><td> </td><td>Thr </td><td><strong><em>1.38</em></strong> </td><td><strong>1.17</strong> </td><td>0.93 </td><td>1.03 </td><td>1.08 </td><td>0.89 </td><td><strong><em>1.16</em></strong> </td><td>1.06 </td><td>1.14 </td><td>1.20 </td><td><strong>0.85</strong> </td><td>– </td><td>0.86 </td><td>0.57 </td><td>1.06 </td><td>0.80 </td><td>1.05 </td><td>0.93 </td><td>0.80 </td><td>1.15 </td></tr><tr><td> </td><td>Cys </td><td>1.12 </td><td>1.24 </td><td>0.93 </td><td>1.62 </td><td>1.03 </td><td>1.10 </td><td>0.90 </td><td>0.78 </td><td>1.31 </td><td>1.36 </td><td>1.09 </td><td>1.16 </td><td>– </td><td>0.93 </td><td>0.84 </td><td>1.18 </td><td>1.00 </td><td>0.76 </td><td>0.93 </td><td>1.00 </td></tr><tr><td> </td><td>Trp </td><td>1.11 </td><td>1.70 </td><td>0.81 </td><td>0.57 </td><td>0.72 </td><td>0.76 </td><td>1.33 </td><td>4.00 </td><td>0.75 </td><td>0.52 </td><td>0.91 </td><td>1.75 </td><td>1.08 </td><td>– </td><td>1.30 </td><td>0.89 </td><td>0.76 </td><td>0.75 </td><td>0.55 </td><td>0.83 </td></tr><tr><td> </td><td>Val </td><td>0.91 </td><td>1.06 </td><td>1.10 </td><td>1.30 </td><td>1.21 </td><td>0.96 </td><td>1.02 </td><td><strong>1.17</strong> </td><td>1.12 </td><td>0.96 </td><td>0.92 </td><td>0.94 </td><td>1.19 </td><td>0.77 </td><td>– </td><td>1.05 </td><td>0.91 </td><td>0.72 </td><td>0.90 </td><td>1.02 </td></tr><tr><td> </td><td>Leu </td><td>0.83 </td><td>1.04 </td><td>1.08 </td><td><strong>1.18</strong> </td><td>0.99 </td><td>1.19 </td><td>0.90 </td><td>1.09 </td><td>1.09 </td><td>0.74 </td><td>0.95 </td><td>1.25 </td><td>0.84 </td><td>1.12 </td><td>0.95 </td><td>– </td><td>0.98 </td><td>1.39 </td><td>1.10 </td><td>0.94 </td></tr><tr><td> </td><td>Glu </td><td>1.03 </td><td>1.11 </td><td>0.89 </td><td>0.95 </td><td>0.56 </td><td>0.89 </td><td>1.10 </td><td>0.75 </td><td>0.93 </td><td><strong><em>0.88</em></strong> </td><td>0.87 </td><td>0.95 </td><td>1.00 </td><td>1.32 </td><td>1.10 </td><td>1.02 </td><td>– </td><td><strong>0.86</strong> </td><td>0.69 </td><td>0.91 </td></tr><tr><td> </td><td>Asp </td><td>1.14 </td><td><strong>1.35</strong> </td><td>1.03 </td><td>1.29 </td><td>1.07 </td><td>1.17 </td><td>1.55 </td><td>1.14 </td><td>0.99 </td><td>1.15 </td><td>0.96 </td><td>1.08 </td><td>1.32 </td><td>1.33 </td><td>1.38 </td><td>0.72 </td><td><strong>1.16</strong> </td><td>– </td><td>0.81 </td><td>0.83 </td></tr><tr><td> </td><td>His </td><td>1.45 </td><td>0.95 </td><td>1.06 </td><td>0.90 </td><td>0.62 </td><td>0.90 </td><td>1.08 </td><td>1.20 </td><td>1.02 </td><td>1.23 </td><td>0.95 </td><td>1.25 </td><td>1.07 </td><td>1.83 </td><td>1.11 </td><td>0.91 </td><td>1.46 </td><td>1.23 </td><td>– </td><td>1.06 </td></tr><tr><td> </td><td>Gln </td><td>1.25 </td><td>0.83 </td><td><strong>1.13</strong> </td><td>0.87 </td><td>0.87 </td><td>1.13 </td><td>0.69 </td><td>0.78 </td><td>1.38 </td><td>0.96 </td><td>0.82 </td><td>0.87 </td><td>1.00 </td><td>1.21 </td><td>0.98 </td><td>1.06 </td><td>1.09 </td><td>1.21 </td><td>0.94 </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each element <em>R<sub>ij</sub></em> in the AARM represents the ratio of number of replacements of the residue <em>i</em> by the residue <em>j</em> in the forward direction (mouse → human) to that in the reverse direction. This means that if <em>R<sub>ij</sub></em> &gt;1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements and if <em>R<sub>ij</sub></em> &lt;1, the reverse is true. Bold and Bold-italics ratios signifies the directional bias at <em>p</em> &lt; 0.0001 and 0.001 respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194679" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM015TB2" class="table-wrap-title" data-id="DSM015TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Amino Acid Replacement Matrix (AARM) for medium-GC group in human mouse orthologs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td>Mouse </td><td>Gly </td><td>– </td><td>0.92 </td><td>0.90 </td><td>1.09 </td><td>0.93 </td><td>0.9 </td><td>1.04 </td><td>0.76 </td><td>0.84 </td><td>0.83 </td><td>0.92 </td><td><strong><em>0.73</em></strong> </td><td>0.89 </td><td>0.90 </td><td>1.10 </td><td>1.20 </td><td>0.97 </td><td>0.88 </td><td>0.69 </td><td>0.80 </td></tr><tr><td> </td><td><strong>Ala</strong> </td><td>1.09 </td><td>– </td><td>1.19 </td><td>1.08 </td><td>1.19 </td><td>1.00 </td><td>0.9 </td><td>0.85 </td><td>0.97 </td><td>1.05 </td><td><strong>0.85</strong> </td><td><strong>0.85</strong> </td><td>0.8 </td><td>0.59 </td><td>0.95 </td><td>0.96 </td><td>0.90 </td><td><strong>0.74</strong> </td><td>1.06 </td><td>1.21 </td></tr><tr><td> </td><td>Arg </td><td>1.11 </td><td>0.84 </td><td>– </td><td>1.02 </td><td>0.91 </td><td>0.84 </td><td>0.85 </td><td>0.76 </td><td>1.04 </td><td><strong><em>1.09</em></strong> </td><td>1.10 </td><td>1.07 </td><td>1.07 </td><td>1.23 </td><td>0.91 </td><td>0.92 </td><td>1.12 </td><td>0.98 </td><td>0.95 </td><td><strong>0.89</strong> </td></tr><tr><td> </td><td>Pro </td><td>0.92 </td><td>0.93 </td><td>0.98 </td><td>– </td><td>0.76 </td><td>0.61 </td><td>1.21 </td><td>0.82 </td><td>0.79 </td><td>1.03 </td><td><strong>0.82</strong> </td><td>0.97 </td><td>0.62 </td><td>1.75 </td><td>0.77 </td><td><strong>0.85</strong> </td><td>1.05 </td><td>0.78 </td><td>1.11 </td><td>1.15 </td></tr><tr><td> </td><td>Phe </td><td>1.08 </td><td>0.84 </td><td>1.10 </td><td>1.32 </td><td>– </td><td><strong><em>1.19</em></strong> </td><td>0.72 </td><td>0.95 </td><td>1.41 </td><td>1.67 </td><td>0.98 </td><td>0.93 </td><td>0.97 </td><td>1.38 </td><td>0.82 </td><td>1.01 </td><td>1.80 </td><td>0.93 </td><td>1.62 </td><td>1.15 </td></tr><tr><td> </td><td>Tyr </td><td>1.11 </td><td>1.00 </td><td>1.19 </td><td>1.64 </td><td><strong><em>0.84</em></strong> </td><td>– </td><td>1.25 </td><td>0.78 </td><td>0.88 </td><td>0.94 </td><td>0.94 </td><td>1.12 </td><td>0.91 </td><td>1.32 </td><td>1.04 </td><td>0.84 </td><td>1.12 </td><td>0.86 </td><td>1.12 </td><td>0.88 </td></tr><tr><td> </td><td>Met </td><td>0.97 </td><td>1.11 </td><td>1.17 </td><td>0.82 </td><td>1.38 </td><td>0.80 </td><td>– </td><td>1.09 </td><td>0.96 </td><td>0.86 </td><td>0.77 </td><td><strong><em>0.86</em></strong> </td><td>1.11 </td><td>0.75 </td><td>0.98 </td><td>1.11 </td><td>0.91 </td><td>0.65 </td><td>0.93 </td><td>1.46 </td></tr><tr><td> </td><td>Ile </td><td>1.32 </td><td>1.18 </td><td>1.32 </td><td>1.21 </td><td>1.05 </td><td>1.29 </td><td>0.92 </td><td>– </td><td>1.13 </td><td>1.02 </td><td>1.03 </td><td>0.95 </td><td>1.28 </td><td>0.25 </td><td><strong>0.85</strong> </td><td>0.92 </td><td>1.33 </td><td>0.88 </td><td>0.83 </td><td>1.29 </td></tr><tr><td> </td><td>Asn </td><td>1.19 </td><td>1.04 </td><td>0.96 </td><td>1.27 </td><td>0.71 </td><td>1.13 </td><td>1.04 </td><td>0.88 </td><td>– </td><td>1.00 </td><td><strong>0.89</strong> </td><td>0.88 </td><td>0.76 </td><td>1.33 </td><td>0.89 </td><td>0.92 </td><td>1.07 </td><td>1.01 </td><td>0.98 </td><td>0.72 </td></tr><tr><td> </td><td>Lys </td><td>1.20 </td><td>0.95 </td><td><strong><em>0.92</em></strong> </td><td>0.97 </td><td>0.60 </td><td>1.06 </td><td>1.16 </td><td>0.98 </td><td>1.00 </td><td>– </td><td>0.89 </td><td>0.84 </td><td>0.73 </td><td>1.94 </td><td>1.05 </td><td>1.35 </td><td><strong><em>1.13</em></strong> </td><td>0.87 </td><td>0.81 </td><td>1.04 </td></tr><tr><td> </td><td>Ser </td><td>1.09 </td><td><strong>1.18</strong> </td><td>0.91 </td><td><strong>1.22</strong> </td><td>1.02 </td><td>1.06 </td><td>1.30 </td><td>0.97 </td><td><strong>1.13</strong> </td><td>1.13 </td><td>– </td><td><strong>1.17</strong> </td><td>0.91 </td><td>1.10 </td><td>1.08 </td><td>1.05 </td><td>1.14 </td><td>1.04 </td><td>1.05 </td><td>1.22 </td></tr><tr><td> </td><td>Thr </td><td><strong><em>1.38</em></strong> </td><td><strong>1.17</strong> </td><td>0.93 </td><td>1.03 </td><td>1.08 </td><td>0.89 </td><td><strong><em>1.16</em></strong> </td><td>1.06 </td><td>1.14 </td><td>1.20 </td><td><strong>0.85</strong> </td><td>– </td><td>0.86 </td><td>0.57 </td><td>1.06 </td><td>0.80 </td><td>1.05 </td><td>0.93 </td><td>0.80 </td><td>1.15 </td></tr><tr><td> </td><td>Cys </td><td>1.12 </td><td>1.24 </td><td>0.93 </td><td>1.62 </td><td>1.03 </td><td>1.10 </td><td>0.90 </td><td>0.78 </td><td>1.31 </td><td>1.36 </td><td>1.09 </td><td>1.16 </td><td>– </td><td>0.93 </td><td>0.84 </td><td>1.18 </td><td>1.00 </td><td>0.76 </td><td>0.93 </td><td>1.00 </td></tr><tr><td> </td><td>Trp </td><td>1.11 </td><td>1.70 </td><td>0.81 </td><td>0.57 </td><td>0.72 </td><td>0.76 </td><td>1.33 </td><td>4.00 </td><td>0.75 </td><td>0.52 </td><td>0.91 </td><td>1.75 </td><td>1.08 </td><td>– </td><td>1.30 </td><td>0.89 </td><td>0.76 </td><td>0.75 </td><td>0.55 </td><td>0.83 </td></tr><tr><td> </td><td>Val </td><td>0.91 </td><td>1.06 </td><td>1.10 </td><td>1.30 </td><td>1.21 </td><td>0.96 </td><td>1.02 </td><td><strong>1.17</strong> </td><td>1.12 </td><td>0.96 </td><td>0.92 </td><td>0.94 </td><td>1.19 </td><td>0.77 </td><td>– </td><td>1.05 </td><td>0.91 </td><td>0.72 </td><td>0.90 </td><td>1.02 </td></tr><tr><td> </td><td>Leu </td><td>0.83 </td><td>1.04 </td><td>1.08 </td><td><strong>1.18</strong> </td><td>0.99 </td><td>1.19 </td><td>0.90 </td><td>1.09 </td><td>1.09 </td><td>0.74 </td><td>0.95 </td><td>1.25 </td><td>0.84 </td><td>1.12 </td><td>0.95 </td><td>– </td><td>0.98 </td><td>1.39 </td><td>1.10 </td><td>0.94 </td></tr><tr><td> </td><td>Glu </td><td>1.03 </td><td>1.11 </td><td>0.89 </td><td>0.95 </td><td>0.56 </td><td>0.89 </td><td>1.10 </td><td>0.75 </td><td>0.93 </td><td><strong><em>0.88</em></strong> </td><td>0.87 </td><td>0.95 </td><td>1.00 </td><td>1.32 </td><td>1.10 </td><td>1.02 </td><td>– </td><td><strong>0.86</strong> </td><td>0.69 </td><td>0.91 </td></tr><tr><td> </td><td>Asp </td><td>1.14 </td><td><strong>1.35</strong> </td><td>1.03 </td><td>1.29 </td><td>1.07 </td><td>1.17 </td><td>1.55 </td><td>1.14 </td><td>0.99 </td><td>1.15 </td><td>0.96 </td><td>1.08 </td><td>1.32 </td><td>1.33 </td><td>1.38 </td><td>0.72 </td><td><strong>1.16</strong> </td><td>– </td><td>0.81 </td><td>0.83 </td></tr><tr><td> </td><td>His </td><td>1.45 </td><td>0.95 </td><td>1.06 </td><td>0.90 </td><td>0.62 </td><td>0.90 </td><td>1.08 </td><td>1.20 </td><td>1.02 </td><td>1.23 </td><td>0.95 </td><td>1.25 </td><td>1.07 </td><td>1.83 </td><td>1.11 </td><td>0.91 </td><td>1.46 </td><td>1.23 </td><td>– </td><td>1.06 </td></tr><tr><td> </td><td>Gln </td><td>1.25 </td><td>0.83 </td><td><strong>1.13</strong> </td><td>0.87 </td><td>0.87 </td><td>1.13 </td><td>0.69 </td><td>0.78 </td><td>1.38 </td><td>0.96 </td><td>0.82 </td><td>0.87 </td><td>1.00 </td><td>1.21 </td><td>0.98 </td><td>1.06 </td><td>1.09 </td><td>1.21 </td><td>0.94 </td><td>– </td></tr></tbody></table></div><div content-id="DSM015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td>Mouse </td><td>Gly </td><td>– </td><td>0.92 </td><td>0.90 </td><td>1.09 </td><td>0.93 </td><td>0.9 </td><td>1.04 </td><td>0.76 </td><td>0.84 </td><td>0.83 </td><td>0.92 </td><td><strong><em>0.73</em></strong> </td><td>0.89 </td><td>0.90 </td><td>1.10 </td><td>1.20 </td><td>0.97 </td><td>0.88 </td><td>0.69 </td><td>0.80 </td></tr><tr><td> </td><td><strong>Ala</strong> </td><td>1.09 </td><td>– </td><td>1.19 </td><td>1.08 </td><td>1.19 </td><td>1.00 </td><td>0.9 </td><td>0.85 </td><td>0.97 </td><td>1.05 </td><td><strong>0.85</strong> </td><td><strong>0.85</strong> </td><td>0.8 </td><td>0.59 </td><td>0.95 </td><td>0.96 </td><td>0.90 </td><td><strong>0.74</strong> </td><td>1.06 </td><td>1.21 </td></tr><tr><td> </td><td>Arg </td><td>1.11 </td><td>0.84 </td><td>– </td><td>1.02 </td><td>0.91 </td><td>0.84 </td><td>0.85 </td><td>0.76 </td><td>1.04 </td><td><strong><em>1.09</em></strong> </td><td>1.10 </td><td>1.07 </td><td>1.07 </td><td>1.23 </td><td>0.91 </td><td>0.92 </td><td>1.12 </td><td>0.98 </td><td>0.95 </td><td><strong>0.89</strong> </td></tr><tr><td> </td><td>Pro </td><td>0.92 </td><td>0.93 </td><td>0.98 </td><td>– </td><td>0.76 </td><td>0.61 </td><td>1.21 </td><td>0.82 </td><td>0.79 </td><td>1.03 </td><td><strong>0.82</strong> </td><td>0.97 </td><td>0.62 </td><td>1.75 </td><td>0.77 </td><td><strong>0.85</strong> </td><td>1.05 </td><td>0.78 </td><td>1.11 </td><td>1.15 </td></tr><tr><td> </td><td>Phe </td><td>1.08 </td><td>0.84 </td><td>1.10 </td><td>1.32 </td><td>– </td><td><strong><em>1.19</em></strong> </td><td>0.72 </td><td>0.95 </td><td>1.41 </td><td>1.67 </td><td>0.98 </td><td>0.93 </td><td>0.97 </td><td>1.38 </td><td>0.82 </td><td>1.01 </td><td>1.80 </td><td>0.93 </td><td>1.62 </td><td>1.15 </td></tr><tr><td> </td><td>Tyr </td><td>1.11 </td><td>1.00 </td><td>1.19 </td><td>1.64 </td><td><strong><em>0.84</em></strong> </td><td>– </td><td>1.25 </td><td>0.78 </td><td>0.88 </td><td>0.94 </td><td>0.94 </td><td>1.12 </td><td>0.91 </td><td>1.32 </td><td>1.04 </td><td>0.84 </td><td>1.12 </td><td>0.86 </td><td>1.12 </td><td>0.88 </td></tr><tr><td> </td><td>Met </td><td>0.97 </td><td>1.11 </td><td>1.17 </td><td>0.82 </td><td>1.38 </td><td>0.80 </td><td>– </td><td>1.09 </td><td>0.96 </td><td>0.86 </td><td>0.77 </td><td><strong><em>0.86</em></strong> </td><td>1.11 </td><td>0.75 </td><td>0.98 </td><td>1.11 </td><td>0.91 </td><td>0.65 </td><td>0.93 </td><td>1.46 </td></tr><tr><td> </td><td>Ile </td><td>1.32 </td><td>1.18 </td><td>1.32 </td><td>1.21 </td><td>1.05 </td><td>1.29 </td><td>0.92 </td><td>– </td><td>1.13 </td><td>1.02 </td><td>1.03 </td><td>0.95 </td><td>1.28 </td><td>0.25 </td><td><strong>0.85</strong> </td><td>0.92 </td><td>1.33 </td><td>0.88 </td><td>0.83 </td><td>1.29 </td></tr><tr><td> </td><td>Asn </td><td>1.19 </td><td>1.04 </td><td>0.96 </td><td>1.27 </td><td>0.71 </td><td>1.13 </td><td>1.04 </td><td>0.88 </td><td>– </td><td>1.00 </td><td><strong>0.89</strong> </td><td>0.88 </td><td>0.76 </td><td>1.33 </td><td>0.89 </td><td>0.92 </td><td>1.07 </td><td>1.01 </td><td>0.98 </td><td>0.72 </td></tr><tr><td> </td><td>Lys </td><td>1.20 </td><td>0.95 </td><td><strong><em>0.92</em></strong> </td><td>0.97 </td><td>0.60 </td><td>1.06 </td><td>1.16 </td><td>0.98 </td><td>1.00 </td><td>– </td><td>0.89 </td><td>0.84 </td><td>0.73 </td><td>1.94 </td><td>1.05 </td><td>1.35 </td><td><strong><em>1.13</em></strong> </td><td>0.87 </td><td>0.81 </td><td>1.04 </td></tr><tr><td> </td><td>Ser </td><td>1.09 </td><td><strong>1.18</strong> </td><td>0.91 </td><td><strong>1.22</strong> </td><td>1.02 </td><td>1.06 </td><td>1.30 </td><td>0.97 </td><td><strong>1.13</strong> </td><td>1.13 </td><td>– </td><td><strong>1.17</strong> </td><td>0.91 </td><td>1.10 </td><td>1.08 </td><td>1.05 </td><td>1.14 </td><td>1.04 </td><td>1.05 </td><td>1.22 </td></tr><tr><td> </td><td>Thr </td><td><strong><em>1.38</em></strong> </td><td><strong>1.17</strong> </td><td>0.93 </td><td>1.03 </td><td>1.08 </td><td>0.89 </td><td><strong><em>1.16</em></strong> </td><td>1.06 </td><td>1.14 </td><td>1.20 </td><td><strong>0.85</strong> </td><td>– </td><td>0.86 </td><td>0.57 </td><td>1.06 </td><td>0.80 </td><td>1.05 </td><td>0.93 </td><td>0.80 </td><td>1.15 </td></tr><tr><td> </td><td>Cys </td><td>1.12 </td><td>1.24 </td><td>0.93 </td><td>1.62 </td><td>1.03 </td><td>1.10 </td><td>0.90 </td><td>0.78 </td><td>1.31 </td><td>1.36 </td><td>1.09 </td><td>1.16 </td><td>– </td><td>0.93 </td><td>0.84 </td><td>1.18 </td><td>1.00 </td><td>0.76 </td><td>0.93 </td><td>1.00 </td></tr><tr><td> </td><td>Trp </td><td>1.11 </td><td>1.70 </td><td>0.81 </td><td>0.57 </td><td>0.72 </td><td>0.76 </td><td>1.33 </td><td>4.00 </td><td>0.75 </td><td>0.52 </td><td>0.91 </td><td>1.75 </td><td>1.08 </td><td>– </td><td>1.30 </td><td>0.89 </td><td>0.76 </td><td>0.75 </td><td>0.55 </td><td>0.83 </td></tr><tr><td> </td><td>Val </td><td>0.91 </td><td>1.06 </td><td>1.10 </td><td>1.30 </td><td>1.21 </td><td>0.96 </td><td>1.02 </td><td><strong>1.17</strong> </td><td>1.12 </td><td>0.96 </td><td>0.92 </td><td>0.94 </td><td>1.19 </td><td>0.77 </td><td>– </td><td>1.05 </td><td>0.91 </td><td>0.72 </td><td>0.90 </td><td>1.02 </td></tr><tr><td> </td><td>Leu </td><td>0.83 </td><td>1.04 </td><td>1.08 </td><td><strong>1.18</strong> </td><td>0.99 </td><td>1.19 </td><td>0.90 </td><td>1.09 </td><td>1.09 </td><td>0.74 </td><td>0.95 </td><td>1.25 </td><td>0.84 </td><td>1.12 </td><td>0.95 </td><td>– </td><td>0.98 </td><td>1.39 </td><td>1.10 </td><td>0.94 </td></tr><tr><td> </td><td>Glu </td><td>1.03 </td><td>1.11 </td><td>0.89 </td><td>0.95 </td><td>0.56 </td><td>0.89 </td><td>1.10 </td><td>0.75 </td><td>0.93 </td><td><strong><em>0.88</em></strong> </td><td>0.87 </td><td>0.95 </td><td>1.00 </td><td>1.32 </td><td>1.10 </td><td>1.02 </td><td>– </td><td><strong>0.86</strong> </td><td>0.69 </td><td>0.91 </td></tr><tr><td> </td><td>Asp </td><td>1.14 </td><td><strong>1.35</strong> </td><td>1.03 </td><td>1.29 </td><td>1.07 </td><td>1.17 </td><td>1.55 </td><td>1.14 </td><td>0.99 </td><td>1.15 </td><td>0.96 </td><td>1.08 </td><td>1.32 </td><td>1.33 </td><td>1.38 </td><td>0.72 </td><td><strong>1.16</strong> </td><td>– </td><td>0.81 </td><td>0.83 </td></tr><tr><td> </td><td>His </td><td>1.45 </td><td>0.95 </td><td>1.06 </td><td>0.90 </td><td>0.62 </td><td>0.90 </td><td>1.08 </td><td>1.20 </td><td>1.02 </td><td>1.23 </td><td>0.95 </td><td>1.25 </td><td>1.07 </td><td>1.83 </td><td>1.11 </td><td>0.91 </td><td>1.46 </td><td>1.23 </td><td>– </td><td>1.06 </td></tr><tr><td> </td><td>Gln </td><td>1.25 </td><td>0.83 </td><td><strong>1.13</strong> </td><td>0.87 </td><td>0.87 </td><td>1.13 </td><td>0.69 </td><td>0.78 </td><td>1.38 </td><td>0.96 </td><td>0.82 </td><td>0.87 </td><td>1.00 </td><td>1.21 </td><td>0.98 </td><td>1.06 </td><td>1.09 </td><td>1.21 </td><td>0.94 </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each element <em>R<sub>ij</sub></em> in the AARM represents the ratio of number of replacements of the residue <em>i</em> by the residue <em>j</em> in the forward direction (mouse → human) to that in the reverse direction. This means that if <em>R<sub>ij</sub></em> &gt;1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements and if <em>R<sub>ij</sub></em> &lt;1, the reverse is true. Bold and Bold-italics ratios signifies the directional bias at <em>p</em> &lt; 0.0001 and 0.001 respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194679" target="_blank">View Large</a></div></div>                        <a id="87194680" scrollto-destination="87194680"></a>
<div content-id="DSM015TB3" class="table-modal"><div class="table-wrap"><div id="DSM015TB3" class="table-wrap-title" data-id="DSM015TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Amino Acid Replacement Matrix (AARM) for low-GC group in human mouse orthologs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td>Mouse </td><td>Gly </td><td>– </td><td><strong><em>0.88</em></strong> </td><td>0.98 </td><td>0.91 </td><td>2.10 </td><td>1.50 </td><td>1.60 </td><td><strong>2.08</strong> </td><td><strong>1.77</strong> </td><td>1.22 </td><td><strong>1.15</strong> </td><td>1.09 </td><td>1.32 </td><td>0.97 </td><td><strong>1.80</strong> </td><td>1.56 </td><td><strong>1.29</strong> </td><td><strong>1.35</strong> </td><td>1.23 </td><td>1.08 </td></tr><tr><td> </td><td>Ala </td><td><strong><em>1.13</em></strong> </td><td>– </td><td>0.97 </td><td><strong>1.24</strong> </td><td>1.61 </td><td>1.32 </td><td><strong><em>1.36</em></strong> </td><td><strong>1.67</strong> </td><td><strong>1.97</strong> </td><td><strong>1.97</strong> </td><td><strong>1.24</strong> </td><td><strong>1.25</strong> </td><td>0.97 </td><td>3.00 </td><td><strong>1.28</strong> </td><td><strong>1.37</strong> </td><td><strong>1.31</strong> </td><td><strong><em>1.24</em></strong> </td><td>1.62 </td><td>1.45 </td></tr><tr><td> </td><td>Arg </td><td>1.02 </td><td>1.03 </td><td>– </td><td>0.97 </td><td>1.32 </td><td>1.38 </td><td><strong><em>1.45</em></strong> </td><td><strong><em>1.50</em></strong> </td><td><strong>1.89</strong> </td><td><strong>1.63</strong> </td><td><strong><em>1.18</em></strong> </td><td>1.14 </td><td>1.11 </td><td>0.97 </td><td>1.04 </td><td>1.23 </td><td><strong>1.40</strong> </td><td><strong><em>1.69</em></strong> </td><td><strong>1.41</strong> </td><td>1.09 </td></tr><tr><td> </td><td>Pro </td><td>1.10 </td><td><strong>0.81</strong> </td><td>1.03 </td><td>– </td><td>1.34 </td><td><strong>2.43</strong> </td><td>1.40 </td><td><strong>1.64</strong> </td><td><strong>2.63</strong> </td><td><strong>2.00</strong> </td><td><strong>1.26</strong> </td><td><strong>1.51</strong> </td><td>0.77 </td><td>1.07 </td><td>1.10 </td><td><strong>1.15</strong> </td><td>1.45 </td><td>1.32 </td><td><strong>1.56</strong> </td><td><strong>1.69</strong> </td></tr><tr><td> </td><td>Phe </td><td>0.48 </td><td>0.62 </td><td>0.76 </td><td>0.74 </td><td>– </td><td><strong>1.25</strong> </td><td>0.68 </td><td>0.95 </td><td>1.21 </td><td>1.54 </td><td><strong>0.76</strong> </td><td>0.75 </td><td><strong>0.60</strong> </td><td>0.68 </td><td><strong>0.64</strong> </td><td><strong>0.78</strong> </td><td>0.76 </td><td>0.50 </td><td>0.92 </td><td>0.85 </td></tr><tr><td> </td><td>Tyr </td><td>0.67 </td><td>0.76 </td><td>0.72 </td><td><strong>0.41</strong> </td><td><strong>0.80</strong> </td><td>– </td><td>0.73 </td><td>1.12 </td><td>0.86 </td><td>0.93 </td><td><strong>0.67</strong> </td><td>0.70 </td><td><strong>0.74</strong> </td><td>1.38 </td><td>0.76 </td><td>0.91 </td><td>1.00 </td><td>0.80 </td><td><strong>0.77</strong> </td><td>0.83 </td></tr><tr><td> </td><td>Met </td><td>0.62 </td><td><strong><em>0.73</em></strong> </td><td><strong><em>0.69</em></strong> </td><td>0.71 </td><td>1.46 </td><td>1.38 </td><td>– </td><td><strong>1.48</strong> </td><td>1.23 </td><td>1.05 </td><td>0.83 </td><td>0.88 </td><td>0.50 </td><td>0.38 </td><td><strong>0.81</strong> </td><td><strong>0.84</strong> </td><td>0.60 </td><td>0.96 </td><td>0.87 </td><td>1.04 </td></tr><tr><td> </td><td>Ile </td><td><strong>0.48</strong> </td><td><strong>0.60</strong> </td><td><strong><em>0.67</em></strong> </td><td><strong>0.61</strong> </td><td>1.05 </td><td>0.89 </td><td><strong>0.68</strong> </td><td>– </td><td>1.17 </td><td>1.00 </td><td><strong>0.65</strong> </td><td><strong>0.66</strong> </td><td>0.50 </td><td>0.89 </td><td><strong>0.67</strong> </td><td><strong>0.68</strong> </td><td>0.65 </td><td>0.70 </td><td>0.96 </td><td>0.57 </td></tr><tr><td> </td><td>Asn </td><td><strong>0.57</strong> </td><td><strong>0.51</strong> </td><td><strong>0.53</strong> </td><td><strong>0.38</strong> </td><td>0.83 </td><td>1.16 </td><td>0.81 </td><td>0.86 </td><td>– </td><td>1.04 </td><td><strong>0.57</strong> </td><td><strong>0.60</strong> </td><td><strong><em>0.54</em></strong> </td><td>0.14 </td><td>0.72 </td><td>0.68 </td><td>0.90 </td><td><strong>0.81</strong> </td><td><strong>0.57</strong> </td><td>0.74 </td></tr><tr><td> </td><td>Lys </td><td>0.82 </td><td><strong>0.51</strong> </td><td><strong>0.61</strong> </td><td><strong>0.50</strong> </td><td>0.65 </td><td>1.08 </td><td>0.95 </td><td>1.00 </td><td>0.96 </td><td>– </td><td><strong>0.62</strong> </td><td><strong>0.62</strong> </td><td>0.75 </td><td>0.65 </td><td>0.77 </td><td>0.70 </td><td>0.93 </td><td>0.89 </td><td><strong>0.57</strong> </td><td><strong>0.73</strong> </td></tr><tr><td> </td><td>Ser </td><td><strong>0.87</strong> </td><td><strong>0.80</strong> </td><td><strong><em>0.85</em></strong> </td><td><strong>0.79</strong> </td><td><strong>1.32</strong> </td><td><strong>1.48</strong> </td><td>1.20 </td><td><strong>1.53</strong> </td><td><strong>1.74</strong> </td><td><strong>1.60</strong> </td><td>– </td><td><strong>1.26</strong> </td><td><strong>0.76</strong> </td><td>0.66 </td><td>1.04 </td><td><strong><em>1.15</em></strong> </td><td>1.33 </td><td><strong>1.39</strong> </td><td>0.99 </td><td>1.10 </td></tr><tr><td> </td><td>Thr </td><td>0.91 </td><td><strong>0.80</strong> </td><td>0.88 </td><td><strong>0.66</strong> </td><td>1.33 </td><td>1.43 </td><td>1.13 </td><td><strong>1.51</strong> </td><td><strong>1.67</strong> </td><td><strong>1.60</strong> </td><td><strong>0.79</strong> </td><td>– </td><td>1.00 </td><td>1.8 </td><td>0.91 </td><td>0.91 </td><td>1.19 </td><td>1.08 </td><td>0.91 </td><td>1.05 </td></tr><tr><td> </td><td>Cys </td><td>0.76 </td><td>1.03 </td><td>0.90 </td><td>1.30 </td><td><strong>1.67</strong> </td><td><strong>1.36</strong> </td><td>2.00 </td><td>2.00 </td><td><strong><em>1.84</em></strong> </td><td>1.33 </td><td><strong>1.31</strong> </td><td>1.00 </td><td>– </td><td>1.00 </td><td>0.52 </td><td>1.11 </td><td>0.57 </td><td>1.44 </td><td>0.94 </td><td>1.31 </td></tr><tr><td> </td><td>Trp </td><td>1.03 </td><td>0.33 </td><td>1.03 </td><td>0.93 </td><td>1.48 </td><td>0.73 </td><td>2.67 </td><td>1.12 </td><td>7.00 </td><td>1.54 </td><td>1.52 </td><td>0.56 </td><td>1.00 </td><td>– </td><td>1.22 </td><td>1.16 </td><td>1.19 </td><td>0.8 </td><td>0.94 </td><td>1.11 </td></tr><tr><td> </td><td>Val </td><td><strong>0.55</strong> </td><td><strong>0.78</strong> </td><td>0.96 </td><td>0.91 </td><td><strong>1.56</strong> </td><td>1.32 </td><td><strong>1.23</strong> </td><td><strong>1.48</strong> </td><td>1.39 </td><td>1.31 </td><td>0.96 </td><td>1.10 </td><td>1.91 </td><td>0.82 </td><td>– </td><td><strong>1.13</strong> </td><td><strong><em>0.79</em></strong> </td><td>0.89 </td><td>1.16 </td><td>0.97 </td></tr><tr><td> </td><td>Leu </td><td>0.64 </td><td><strong>0.73</strong> </td><td>0.82 </td><td><strong>0.87</strong> </td><td><strong>1.28</strong> </td><td>1.09 </td><td><strong>1.19</strong> </td><td><strong>1.48</strong> </td><td>1.47 </td><td>1.42 </td><td><strong><em>0.87</em></strong> </td><td>1.09 </td><td>0.9 </td><td>0.86 </td><td><strong>0.89</strong> </td><td>– </td><td>0.92 </td><td>0.58 </td><td>1.13 </td><td>0.92 </td></tr><tr><td> </td><td>Glu </td><td><strong>0.78</strong> </td><td><strong>0.76</strong> </td><td><strong>0.71</strong> </td><td>0.69 </td><td>1.31 </td><td>1.00 </td><td>1.67 </td><td>1.53 </td><td>1.12 </td><td>1.08 </td><td>0.75 </td><td>0.84 </td><td>1.75 </td><td>0.84 </td><td><strong><em>1.27</em></strong> </td><td>1.09 </td><td>– </td><td><strong>0.84</strong> </td><td>1.06 </td><td>0.95 </td></tr><tr><td> </td><td>Asp </td><td><strong>0.74</strong> </td><td><strong><em>0.81</em></strong> </td><td><strong><em>0.59</em></strong> </td><td>0.76 </td><td>2.00 </td><td>1.25 </td><td>1.04 </td><td>1.43 </td><td><strong>1.24</strong> </td><td>1.12 </td><td><strong>0.72</strong> </td><td>0.93 </td><td>0.69 </td><td>1.25 </td><td>1.12 </td><td>1.72 </td><td><strong>1.19</strong> </td><td>– </td><td><strong><em>0.78</em></strong> </td><td>1.00 </td></tr><tr><td> </td><td>His </td><td>0.81 </td><td>0.62 </td><td><strong>0.71</strong> </td><td><strong>0.64</strong> </td><td>1.09 </td><td><strong>1.30</strong> </td><td>1.15 </td><td>1.04 </td><td><strong>1.76</strong> </td><td><strong>1.75</strong> </td><td>1.01 </td><td>1.10 </td><td>1.07 </td><td>1.06 </td><td>0.86 </td><td>0.89 </td><td>0.94 </td><td><strong><em>1.28</em></strong> </td><td>– </td><td>1.09 </td></tr><tr><td> </td><td>Gln </td><td>0.93 </td><td>0.69 </td><td>0.92 </td><td><strong>0.59</strong> </td><td>1.170 </td><td>1.20 </td><td>0.96 </td><td>1.77 </td><td>1.36 </td><td><strong>1.37</strong> </td><td>0.91 </td><td>0.95 </td><td>0.77 </td><td>0.90 </td><td>1.03 </td><td>1.08 </td><td>1.05 </td><td>1.00 </td><td>0.91 </td><td>– </td></tr></tbody></table></div><div content-id="DSM015TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td>Mouse </td><td>Gly </td><td>– </td><td><strong><em>0.88</em></strong> </td><td>0.98 </td><td>0.91 </td><td>2.10 </td><td>1.50 </td><td>1.60 </td><td><strong>2.08</strong> </td><td><strong>1.77</strong> </td><td>1.22 </td><td><strong>1.15</strong> </td><td>1.09 </td><td>1.32 </td><td>0.97 </td><td><strong>1.80</strong> </td><td>1.56 </td><td><strong>1.29</strong> </td><td><strong>1.35</strong> </td><td>1.23 </td><td>1.08 </td></tr><tr><td> </td><td>Ala </td><td><strong><em>1.13</em></strong> </td><td>– </td><td>0.97 </td><td><strong>1.24</strong> </td><td>1.61 </td><td>1.32 </td><td><strong><em>1.36</em></strong> </td><td><strong>1.67</strong> </td><td><strong>1.97</strong> </td><td><strong>1.97</strong> </td><td><strong>1.24</strong> </td><td><strong>1.25</strong> </td><td>0.97 </td><td>3.00 </td><td><strong>1.28</strong> </td><td><strong>1.37</strong> </td><td><strong>1.31</strong> </td><td><strong><em>1.24</em></strong> </td><td>1.62 </td><td>1.45 </td></tr><tr><td> </td><td>Arg </td><td>1.02 </td><td>1.03 </td><td>– </td><td>0.97 </td><td>1.32 </td><td>1.38 </td><td><strong><em>1.45</em></strong> </td><td><strong><em>1.50</em></strong> </td><td><strong>1.89</strong> </td><td><strong>1.63</strong> </td><td><strong><em>1.18</em></strong> </td><td>1.14 </td><td>1.11 </td><td>0.97 </td><td>1.04 </td><td>1.23 </td><td><strong>1.40</strong> </td><td><strong><em>1.69</em></strong> </td><td><strong>1.41</strong> </td><td>1.09 </td></tr><tr><td> </td><td>Pro </td><td>1.10 </td><td><strong>0.81</strong> </td><td>1.03 </td><td>– </td><td>1.34 </td><td><strong>2.43</strong> </td><td>1.40 </td><td><strong>1.64</strong> </td><td><strong>2.63</strong> </td><td><strong>2.00</strong> </td><td><strong>1.26</strong> </td><td><strong>1.51</strong> </td><td>0.77 </td><td>1.07 </td><td>1.10 </td><td><strong>1.15</strong> </td><td>1.45 </td><td>1.32 </td><td><strong>1.56</strong> </td><td><strong>1.69</strong> </td></tr><tr><td> </td><td>Phe </td><td>0.48 </td><td>0.62 </td><td>0.76 </td><td>0.74 </td><td>– </td><td><strong>1.25</strong> </td><td>0.68 </td><td>0.95 </td><td>1.21 </td><td>1.54 </td><td><strong>0.76</strong> </td><td>0.75 </td><td><strong>0.60</strong> </td><td>0.68 </td><td><strong>0.64</strong> </td><td><strong>0.78</strong> </td><td>0.76 </td><td>0.50 </td><td>0.92 </td><td>0.85 </td></tr><tr><td> </td><td>Tyr </td><td>0.67 </td><td>0.76 </td><td>0.72 </td><td><strong>0.41</strong> </td><td><strong>0.80</strong> </td><td>– </td><td>0.73 </td><td>1.12 </td><td>0.86 </td><td>0.93 </td><td><strong>0.67</strong> </td><td>0.70 </td><td><strong>0.74</strong> </td><td>1.38 </td><td>0.76 </td><td>0.91 </td><td>1.00 </td><td>0.80 </td><td><strong>0.77</strong> </td><td>0.83 </td></tr><tr><td> </td><td>Met </td><td>0.62 </td><td><strong><em>0.73</em></strong> </td><td><strong><em>0.69</em></strong> </td><td>0.71 </td><td>1.46 </td><td>1.38 </td><td>– </td><td><strong>1.48</strong> </td><td>1.23 </td><td>1.05 </td><td>0.83 </td><td>0.88 </td><td>0.50 </td><td>0.38 </td><td><strong>0.81</strong> </td><td><strong>0.84</strong> </td><td>0.60 </td><td>0.96 </td><td>0.87 </td><td>1.04 </td></tr><tr><td> </td><td>Ile </td><td><strong>0.48</strong> </td><td><strong>0.60</strong> </td><td><strong><em>0.67</em></strong> </td><td><strong>0.61</strong> </td><td>1.05 </td><td>0.89 </td><td><strong>0.68</strong> </td><td>– </td><td>1.17 </td><td>1.00 </td><td><strong>0.65</strong> </td><td><strong>0.66</strong> </td><td>0.50 </td><td>0.89 </td><td><strong>0.67</strong> </td><td><strong>0.68</strong> </td><td>0.65 </td><td>0.70 </td><td>0.96 </td><td>0.57 </td></tr><tr><td> </td><td>Asn </td><td><strong>0.57</strong> </td><td><strong>0.51</strong> </td><td><strong>0.53</strong> </td><td><strong>0.38</strong> </td><td>0.83 </td><td>1.16 </td><td>0.81 </td><td>0.86 </td><td>– </td><td>1.04 </td><td><strong>0.57</strong> </td><td><strong>0.60</strong> </td><td><strong><em>0.54</em></strong> </td><td>0.14 </td><td>0.72 </td><td>0.68 </td><td>0.90 </td><td><strong>0.81</strong> </td><td><strong>0.57</strong> </td><td>0.74 </td></tr><tr><td> </td><td>Lys </td><td>0.82 </td><td><strong>0.51</strong> </td><td><strong>0.61</strong> </td><td><strong>0.50</strong> </td><td>0.65 </td><td>1.08 </td><td>0.95 </td><td>1.00 </td><td>0.96 </td><td>– </td><td><strong>0.62</strong> </td><td><strong>0.62</strong> </td><td>0.75 </td><td>0.65 </td><td>0.77 </td><td>0.70 </td><td>0.93 </td><td>0.89 </td><td><strong>0.57</strong> </td><td><strong>0.73</strong> </td></tr><tr><td> </td><td>Ser </td><td><strong>0.87</strong> </td><td><strong>0.80</strong> </td><td><strong><em>0.85</em></strong> </td><td><strong>0.79</strong> </td><td><strong>1.32</strong> </td><td><strong>1.48</strong> </td><td>1.20 </td><td><strong>1.53</strong> </td><td><strong>1.74</strong> </td><td><strong>1.60</strong> </td><td>– </td><td><strong>1.26</strong> </td><td><strong>0.76</strong> </td><td>0.66 </td><td>1.04 </td><td><strong><em>1.15</em></strong> </td><td>1.33 </td><td><strong>1.39</strong> </td><td>0.99 </td><td>1.10 </td></tr><tr><td> </td><td>Thr </td><td>0.91 </td><td><strong>0.80</strong> </td><td>0.88 </td><td><strong>0.66</strong> </td><td>1.33 </td><td>1.43 </td><td>1.13 </td><td><strong>1.51</strong> </td><td><strong>1.67</strong> </td><td><strong>1.60</strong> </td><td><strong>0.79</strong> </td><td>– </td><td>1.00 </td><td>1.8 </td><td>0.91 </td><td>0.91 </td><td>1.19 </td><td>1.08 </td><td>0.91 </td><td>1.05 </td></tr><tr><td> </td><td>Cys </td><td>0.76 </td><td>1.03 </td><td>0.90 </td><td>1.30 </td><td><strong>1.67</strong> </td><td><strong>1.36</strong> </td><td>2.00 </td><td>2.00 </td><td><strong><em>1.84</em></strong> </td><td>1.33 </td><td><strong>1.31</strong> </td><td>1.00 </td><td>– </td><td>1.00 </td><td>0.52 </td><td>1.11 </td><td>0.57 </td><td>1.44 </td><td>0.94 </td><td>1.31 </td></tr><tr><td> </td><td>Trp </td><td>1.03 </td><td>0.33 </td><td>1.03 </td><td>0.93 </td><td>1.48 </td><td>0.73 </td><td>2.67 </td><td>1.12 </td><td>7.00 </td><td>1.54 </td><td>1.52 </td><td>0.56 </td><td>1.00 </td><td>– </td><td>1.22 </td><td>1.16 </td><td>1.19 </td><td>0.8 </td><td>0.94 </td><td>1.11 </td></tr><tr><td> </td><td>Val </td><td><strong>0.55</strong> </td><td><strong>0.78</strong> </td><td>0.96 </td><td>0.91 </td><td><strong>1.56</strong> </td><td>1.32 </td><td><strong>1.23</strong> </td><td><strong>1.48</strong> </td><td>1.39 </td><td>1.31 </td><td>0.96 </td><td>1.10 </td><td>1.91 </td><td>0.82 </td><td>– </td><td><strong>1.13</strong> </td><td><strong><em>0.79</em></strong> </td><td>0.89 </td><td>1.16 </td><td>0.97 </td></tr><tr><td> </td><td>Leu </td><td>0.64 </td><td><strong>0.73</strong> </td><td>0.82 </td><td><strong>0.87</strong> </td><td><strong>1.28</strong> </td><td>1.09 </td><td><strong>1.19</strong> </td><td><strong>1.48</strong> </td><td>1.47 </td><td>1.42 </td><td><strong><em>0.87</em></strong> </td><td>1.09 </td><td>0.9 </td><td>0.86 </td><td><strong>0.89</strong> </td><td>– </td><td>0.92 </td><td>0.58 </td><td>1.13 </td><td>0.92 </td></tr><tr><td> </td><td>Glu </td><td><strong>0.78</strong> </td><td><strong>0.76</strong> </td><td><strong>0.71</strong> </td><td>0.69 </td><td>1.31 </td><td>1.00 </td><td>1.67 </td><td>1.53 </td><td>1.12 </td><td>1.08 </td><td>0.75 </td><td>0.84 </td><td>1.75 </td><td>0.84 </td><td><strong><em>1.27</em></strong> </td><td>1.09 </td><td>– </td><td><strong>0.84</strong> </td><td>1.06 </td><td>0.95 </td></tr><tr><td> </td><td>Asp </td><td><strong>0.74</strong> </td><td><strong><em>0.81</em></strong> </td><td><strong><em>0.59</em></strong> </td><td>0.76 </td><td>2.00 </td><td>1.25 </td><td>1.04 </td><td>1.43 </td><td><strong>1.24</strong> </td><td>1.12 </td><td><strong>0.72</strong> </td><td>0.93 </td><td>0.69 </td><td>1.25 </td><td>1.12 </td><td>1.72 </td><td><strong>1.19</strong> </td><td>– </td><td><strong><em>0.78</em></strong> </td><td>1.00 </td></tr><tr><td> </td><td>His </td><td>0.81 </td><td>0.62 </td><td><strong>0.71</strong> </td><td><strong>0.64</strong> </td><td>1.09 </td><td><strong>1.30</strong> </td><td>1.15 </td><td>1.04 </td><td><strong>1.76</strong> </td><td><strong>1.75</strong> </td><td>1.01 </td><td>1.10 </td><td>1.07 </td><td>1.06 </td><td>0.86 </td><td>0.89 </td><td>0.94 </td><td><strong><em>1.28</em></strong> </td><td>– </td><td>1.09 </td></tr><tr><td> </td><td>Gln </td><td>0.93 </td><td>0.69 </td><td>0.92 </td><td><strong>0.59</strong> </td><td>1.170 </td><td>1.20 </td><td>0.96 </td><td>1.77 </td><td>1.36 </td><td><strong>1.37</strong> </td><td>0.91 </td><td>0.95 </td><td>0.77 </td><td>0.90 </td><td>1.03 </td><td>1.08 </td><td>1.05 </td><td>1.00 </td><td>0.91 </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each element <em>R<sub>ij</sub></em> in the AARM represents the ratio of number of replacements of the residue <em>i</em> by the residue <em>j</em> in the forward direction (mouse → human) to that in the reverse direction. This means that if <em>R<sub>ij</sub></em> &gt;1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements and if <em>R<sub>ij</sub></em> &lt;1, the reverse is true. Bold and Bold-italics ratios signifies the directional bias at <em>p</em> &lt; 0.0001 and &lt; 0.001 respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194680" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM015TB3" class="table-wrap-title" data-id="DSM015TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Amino Acid Replacement Matrix (AARM) for low-GC group in human mouse orthologs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td>Mouse </td><td>Gly </td><td>– </td><td><strong><em>0.88</em></strong> </td><td>0.98 </td><td>0.91 </td><td>2.10 </td><td>1.50 </td><td>1.60 </td><td><strong>2.08</strong> </td><td><strong>1.77</strong> </td><td>1.22 </td><td><strong>1.15</strong> </td><td>1.09 </td><td>1.32 </td><td>0.97 </td><td><strong>1.80</strong> </td><td>1.56 </td><td><strong>1.29</strong> </td><td><strong>1.35</strong> </td><td>1.23 </td><td>1.08 </td></tr><tr><td> </td><td>Ala </td><td><strong><em>1.13</em></strong> </td><td>– </td><td>0.97 </td><td><strong>1.24</strong> </td><td>1.61 </td><td>1.32 </td><td><strong><em>1.36</em></strong> </td><td><strong>1.67</strong> </td><td><strong>1.97</strong> </td><td><strong>1.97</strong> </td><td><strong>1.24</strong> </td><td><strong>1.25</strong> </td><td>0.97 </td><td>3.00 </td><td><strong>1.28</strong> </td><td><strong>1.37</strong> </td><td><strong>1.31</strong> </td><td><strong><em>1.24</em></strong> </td><td>1.62 </td><td>1.45 </td></tr><tr><td> </td><td>Arg </td><td>1.02 </td><td>1.03 </td><td>– </td><td>0.97 </td><td>1.32 </td><td>1.38 </td><td><strong><em>1.45</em></strong> </td><td><strong><em>1.50</em></strong> </td><td><strong>1.89</strong> </td><td><strong>1.63</strong> </td><td><strong><em>1.18</em></strong> </td><td>1.14 </td><td>1.11 </td><td>0.97 </td><td>1.04 </td><td>1.23 </td><td><strong>1.40</strong> </td><td><strong><em>1.69</em></strong> </td><td><strong>1.41</strong> </td><td>1.09 </td></tr><tr><td> </td><td>Pro </td><td>1.10 </td><td><strong>0.81</strong> </td><td>1.03 </td><td>– </td><td>1.34 </td><td><strong>2.43</strong> </td><td>1.40 </td><td><strong>1.64</strong> </td><td><strong>2.63</strong> </td><td><strong>2.00</strong> </td><td><strong>1.26</strong> </td><td><strong>1.51</strong> </td><td>0.77 </td><td>1.07 </td><td>1.10 </td><td><strong>1.15</strong> </td><td>1.45 </td><td>1.32 </td><td><strong>1.56</strong> </td><td><strong>1.69</strong> </td></tr><tr><td> </td><td>Phe </td><td>0.48 </td><td>0.62 </td><td>0.76 </td><td>0.74 </td><td>– </td><td><strong>1.25</strong> </td><td>0.68 </td><td>0.95 </td><td>1.21 </td><td>1.54 </td><td><strong>0.76</strong> </td><td>0.75 </td><td><strong>0.60</strong> </td><td>0.68 </td><td><strong>0.64</strong> </td><td><strong>0.78</strong> </td><td>0.76 </td><td>0.50 </td><td>0.92 </td><td>0.85 </td></tr><tr><td> </td><td>Tyr </td><td>0.67 </td><td>0.76 </td><td>0.72 </td><td><strong>0.41</strong> </td><td><strong>0.80</strong> </td><td>– </td><td>0.73 </td><td>1.12 </td><td>0.86 </td><td>0.93 </td><td><strong>0.67</strong> </td><td>0.70 </td><td><strong>0.74</strong> </td><td>1.38 </td><td>0.76 </td><td>0.91 </td><td>1.00 </td><td>0.80 </td><td><strong>0.77</strong> </td><td>0.83 </td></tr><tr><td> </td><td>Met </td><td>0.62 </td><td><strong><em>0.73</em></strong> </td><td><strong><em>0.69</em></strong> </td><td>0.71 </td><td>1.46 </td><td>1.38 </td><td>– </td><td><strong>1.48</strong> </td><td>1.23 </td><td>1.05 </td><td>0.83 </td><td>0.88 </td><td>0.50 </td><td>0.38 </td><td><strong>0.81</strong> </td><td><strong>0.84</strong> </td><td>0.60 </td><td>0.96 </td><td>0.87 </td><td>1.04 </td></tr><tr><td> </td><td>Ile </td><td><strong>0.48</strong> </td><td><strong>0.60</strong> </td><td><strong><em>0.67</em></strong> </td><td><strong>0.61</strong> </td><td>1.05 </td><td>0.89 </td><td><strong>0.68</strong> </td><td>– </td><td>1.17 </td><td>1.00 </td><td><strong>0.65</strong> </td><td><strong>0.66</strong> </td><td>0.50 </td><td>0.89 </td><td><strong>0.67</strong> </td><td><strong>0.68</strong> </td><td>0.65 </td><td>0.70 </td><td>0.96 </td><td>0.57 </td></tr><tr><td> </td><td>Asn </td><td><strong>0.57</strong> </td><td><strong>0.51</strong> </td><td><strong>0.53</strong> </td><td><strong>0.38</strong> </td><td>0.83 </td><td>1.16 </td><td>0.81 </td><td>0.86 </td><td>– </td><td>1.04 </td><td><strong>0.57</strong> </td><td><strong>0.60</strong> </td><td><strong><em>0.54</em></strong> </td><td>0.14 </td><td>0.72 </td><td>0.68 </td><td>0.90 </td><td><strong>0.81</strong> </td><td><strong>0.57</strong> </td><td>0.74 </td></tr><tr><td> </td><td>Lys </td><td>0.82 </td><td><strong>0.51</strong> </td><td><strong>0.61</strong> </td><td><strong>0.50</strong> </td><td>0.65 </td><td>1.08 </td><td>0.95 </td><td>1.00 </td><td>0.96 </td><td>– </td><td><strong>0.62</strong> </td><td><strong>0.62</strong> </td><td>0.75 </td><td>0.65 </td><td>0.77 </td><td>0.70 </td><td>0.93 </td><td>0.89 </td><td><strong>0.57</strong> </td><td><strong>0.73</strong> </td></tr><tr><td> </td><td>Ser </td><td><strong>0.87</strong> </td><td><strong>0.80</strong> </td><td><strong><em>0.85</em></strong> </td><td><strong>0.79</strong> </td><td><strong>1.32</strong> </td><td><strong>1.48</strong> </td><td>1.20 </td><td><strong>1.53</strong> </td><td><strong>1.74</strong> </td><td><strong>1.60</strong> </td><td>– </td><td><strong>1.26</strong> </td><td><strong>0.76</strong> </td><td>0.66 </td><td>1.04 </td><td><strong><em>1.15</em></strong> </td><td>1.33 </td><td><strong>1.39</strong> </td><td>0.99 </td><td>1.10 </td></tr><tr><td> </td><td>Thr </td><td>0.91 </td><td><strong>0.80</strong> </td><td>0.88 </td><td><strong>0.66</strong> </td><td>1.33 </td><td>1.43 </td><td>1.13 </td><td><strong>1.51</strong> </td><td><strong>1.67</strong> </td><td><strong>1.60</strong> </td><td><strong>0.79</strong> </td><td>– </td><td>1.00 </td><td>1.8 </td><td>0.91 </td><td>0.91 </td><td>1.19 </td><td>1.08 </td><td>0.91 </td><td>1.05 </td></tr><tr><td> </td><td>Cys </td><td>0.76 </td><td>1.03 </td><td>0.90 </td><td>1.30 </td><td><strong>1.67</strong> </td><td><strong>1.36</strong> </td><td>2.00 </td><td>2.00 </td><td><strong><em>1.84</em></strong> </td><td>1.33 </td><td><strong>1.31</strong> </td><td>1.00 </td><td>– </td><td>1.00 </td><td>0.52 </td><td>1.11 </td><td>0.57 </td><td>1.44 </td><td>0.94 </td><td>1.31 </td></tr><tr><td> </td><td>Trp </td><td>1.03 </td><td>0.33 </td><td>1.03 </td><td>0.93 </td><td>1.48 </td><td>0.73 </td><td>2.67 </td><td>1.12 </td><td>7.00 </td><td>1.54 </td><td>1.52 </td><td>0.56 </td><td>1.00 </td><td>– </td><td>1.22 </td><td>1.16 </td><td>1.19 </td><td>0.8 </td><td>0.94 </td><td>1.11 </td></tr><tr><td> </td><td>Val </td><td><strong>0.55</strong> </td><td><strong>0.78</strong> </td><td>0.96 </td><td>0.91 </td><td><strong>1.56</strong> </td><td>1.32 </td><td><strong>1.23</strong> </td><td><strong>1.48</strong> </td><td>1.39 </td><td>1.31 </td><td>0.96 </td><td>1.10 </td><td>1.91 </td><td>0.82 </td><td>– </td><td><strong>1.13</strong> </td><td><strong><em>0.79</em></strong> </td><td>0.89 </td><td>1.16 </td><td>0.97 </td></tr><tr><td> </td><td>Leu </td><td>0.64 </td><td><strong>0.73</strong> </td><td>0.82 </td><td><strong>0.87</strong> </td><td><strong>1.28</strong> </td><td>1.09 </td><td><strong>1.19</strong> </td><td><strong>1.48</strong> </td><td>1.47 </td><td>1.42 </td><td><strong><em>0.87</em></strong> </td><td>1.09 </td><td>0.9 </td><td>0.86 </td><td><strong>0.89</strong> </td><td>– </td><td>0.92 </td><td>0.58 </td><td>1.13 </td><td>0.92 </td></tr><tr><td> </td><td>Glu </td><td><strong>0.78</strong> </td><td><strong>0.76</strong> </td><td><strong>0.71</strong> </td><td>0.69 </td><td>1.31 </td><td>1.00 </td><td>1.67 </td><td>1.53 </td><td>1.12 </td><td>1.08 </td><td>0.75 </td><td>0.84 </td><td>1.75 </td><td>0.84 </td><td><strong><em>1.27</em></strong> </td><td>1.09 </td><td>– </td><td><strong>0.84</strong> </td><td>1.06 </td><td>0.95 </td></tr><tr><td> </td><td>Asp </td><td><strong>0.74</strong> </td><td><strong><em>0.81</em></strong> </td><td><strong><em>0.59</em></strong> </td><td>0.76 </td><td>2.00 </td><td>1.25 </td><td>1.04 </td><td>1.43 </td><td><strong>1.24</strong> </td><td>1.12 </td><td><strong>0.72</strong> </td><td>0.93 </td><td>0.69 </td><td>1.25 </td><td>1.12 </td><td>1.72 </td><td><strong>1.19</strong> </td><td>– </td><td><strong><em>0.78</em></strong> </td><td>1.00 </td></tr><tr><td> </td><td>His </td><td>0.81 </td><td>0.62 </td><td><strong>0.71</strong> </td><td><strong>0.64</strong> </td><td>1.09 </td><td><strong>1.30</strong> </td><td>1.15 </td><td>1.04 </td><td><strong>1.76</strong> </td><td><strong>1.75</strong> </td><td>1.01 </td><td>1.10 </td><td>1.07 </td><td>1.06 </td><td>0.86 </td><td>0.89 </td><td>0.94 </td><td><strong><em>1.28</em></strong> </td><td>– </td><td>1.09 </td></tr><tr><td> </td><td>Gln </td><td>0.93 </td><td>0.69 </td><td>0.92 </td><td><strong>0.59</strong> </td><td>1.170 </td><td>1.20 </td><td>0.96 </td><td>1.77 </td><td>1.36 </td><td><strong>1.37</strong> </td><td>0.91 </td><td>0.95 </td><td>0.77 </td><td>0.90 </td><td>1.03 </td><td>1.08 </td><td>1.05 </td><td>1.00 </td><td>0.91 </td><td>– </td></tr></tbody></table></div><div content-id="DSM015TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="20">Human </th></tr><tr><th> </th><th> </th><th>Gly </th><th>Ala </th><th>Arg </th><th>Pro </th><th>Phe </th><th>Tyr </th><th>Met </th><th>Ile </th><th>Asn </th><th>Lys </th><th>Ser </th><th>Thr </th><th>Cys </th><th>Trp </th><th>Val </th><th>Leu </th><th>Glu </th><th>Asp </th><th>His </th><th>Gln </th></tr></thead><tbody><tr><td>Mouse </td><td>Gly </td><td>– </td><td><strong><em>0.88</em></strong> </td><td>0.98 </td><td>0.91 </td><td>2.10 </td><td>1.50 </td><td>1.60 </td><td><strong>2.08</strong> </td><td><strong>1.77</strong> </td><td>1.22 </td><td><strong>1.15</strong> </td><td>1.09 </td><td>1.32 </td><td>0.97 </td><td><strong>1.80</strong> </td><td>1.56 </td><td><strong>1.29</strong> </td><td><strong>1.35</strong> </td><td>1.23 </td><td>1.08 </td></tr><tr><td> </td><td>Ala </td><td><strong><em>1.13</em></strong> </td><td>– </td><td>0.97 </td><td><strong>1.24</strong> </td><td>1.61 </td><td>1.32 </td><td><strong><em>1.36</em></strong> </td><td><strong>1.67</strong> </td><td><strong>1.97</strong> </td><td><strong>1.97</strong> </td><td><strong>1.24</strong> </td><td><strong>1.25</strong> </td><td>0.97 </td><td>3.00 </td><td><strong>1.28</strong> </td><td><strong>1.37</strong> </td><td><strong>1.31</strong> </td><td><strong><em>1.24</em></strong> </td><td>1.62 </td><td>1.45 </td></tr><tr><td> </td><td>Arg </td><td>1.02 </td><td>1.03 </td><td>– </td><td>0.97 </td><td>1.32 </td><td>1.38 </td><td><strong><em>1.45</em></strong> </td><td><strong><em>1.50</em></strong> </td><td><strong>1.89</strong> </td><td><strong>1.63</strong> </td><td><strong><em>1.18</em></strong> </td><td>1.14 </td><td>1.11 </td><td>0.97 </td><td>1.04 </td><td>1.23 </td><td><strong>1.40</strong> </td><td><strong><em>1.69</em></strong> </td><td><strong>1.41</strong> </td><td>1.09 </td></tr><tr><td> </td><td>Pro </td><td>1.10 </td><td><strong>0.81</strong> </td><td>1.03 </td><td>– </td><td>1.34 </td><td><strong>2.43</strong> </td><td>1.40 </td><td><strong>1.64</strong> </td><td><strong>2.63</strong> </td><td><strong>2.00</strong> </td><td><strong>1.26</strong> </td><td><strong>1.51</strong> </td><td>0.77 </td><td>1.07 </td><td>1.10 </td><td><strong>1.15</strong> </td><td>1.45 </td><td>1.32 </td><td><strong>1.56</strong> </td><td><strong>1.69</strong> </td></tr><tr><td> </td><td>Phe </td><td>0.48 </td><td>0.62 </td><td>0.76 </td><td>0.74 </td><td>– </td><td><strong>1.25</strong> </td><td>0.68 </td><td>0.95 </td><td>1.21 </td><td>1.54 </td><td><strong>0.76</strong> </td><td>0.75 </td><td><strong>0.60</strong> </td><td>0.68 </td><td><strong>0.64</strong> </td><td><strong>0.78</strong> </td><td>0.76 </td><td>0.50 </td><td>0.92 </td><td>0.85 </td></tr><tr><td> </td><td>Tyr </td><td>0.67 </td><td>0.76 </td><td>0.72 </td><td><strong>0.41</strong> </td><td><strong>0.80</strong> </td><td>– </td><td>0.73 </td><td>1.12 </td><td>0.86 </td><td>0.93 </td><td><strong>0.67</strong> </td><td>0.70 </td><td><strong>0.74</strong> </td><td>1.38 </td><td>0.76 </td><td>0.91 </td><td>1.00 </td><td>0.80 </td><td><strong>0.77</strong> </td><td>0.83 </td></tr><tr><td> </td><td>Met </td><td>0.62 </td><td><strong><em>0.73</em></strong> </td><td><strong><em>0.69</em></strong> </td><td>0.71 </td><td>1.46 </td><td>1.38 </td><td>– </td><td><strong>1.48</strong> </td><td>1.23 </td><td>1.05 </td><td>0.83 </td><td>0.88 </td><td>0.50 </td><td>0.38 </td><td><strong>0.81</strong> </td><td><strong>0.84</strong> </td><td>0.60 </td><td>0.96 </td><td>0.87 </td><td>1.04 </td></tr><tr><td> </td><td>Ile </td><td><strong>0.48</strong> </td><td><strong>0.60</strong> </td><td><strong><em>0.67</em></strong> </td><td><strong>0.61</strong> </td><td>1.05 </td><td>0.89 </td><td><strong>0.68</strong> </td><td>– </td><td>1.17 </td><td>1.00 </td><td><strong>0.65</strong> </td><td><strong>0.66</strong> </td><td>0.50 </td><td>0.89 </td><td><strong>0.67</strong> </td><td><strong>0.68</strong> </td><td>0.65 </td><td>0.70 </td><td>0.96 </td><td>0.57 </td></tr><tr><td> </td><td>Asn </td><td><strong>0.57</strong> </td><td><strong>0.51</strong> </td><td><strong>0.53</strong> </td><td><strong>0.38</strong> </td><td>0.83 </td><td>1.16 </td><td>0.81 </td><td>0.86 </td><td>– </td><td>1.04 </td><td><strong>0.57</strong> </td><td><strong>0.60</strong> </td><td><strong><em>0.54</em></strong> </td><td>0.14 </td><td>0.72 </td><td>0.68 </td><td>0.90 </td><td><strong>0.81</strong> </td><td><strong>0.57</strong> </td><td>0.74 </td></tr><tr><td> </td><td>Lys </td><td>0.82 </td><td><strong>0.51</strong> </td><td><strong>0.61</strong> </td><td><strong>0.50</strong> </td><td>0.65 </td><td>1.08 </td><td>0.95 </td><td>1.00 </td><td>0.96 </td><td>– </td><td><strong>0.62</strong> </td><td><strong>0.62</strong> </td><td>0.75 </td><td>0.65 </td><td>0.77 </td><td>0.70 </td><td>0.93 </td><td>0.89 </td><td><strong>0.57</strong> </td><td><strong>0.73</strong> </td></tr><tr><td> </td><td>Ser </td><td><strong>0.87</strong> </td><td><strong>0.80</strong> </td><td><strong><em>0.85</em></strong> </td><td><strong>0.79</strong> </td><td><strong>1.32</strong> </td><td><strong>1.48</strong> </td><td>1.20 </td><td><strong>1.53</strong> </td><td><strong>1.74</strong> </td><td><strong>1.60</strong> </td><td>– </td><td><strong>1.26</strong> </td><td><strong>0.76</strong> </td><td>0.66 </td><td>1.04 </td><td><strong><em>1.15</em></strong> </td><td>1.33 </td><td><strong>1.39</strong> </td><td>0.99 </td><td>1.10 </td></tr><tr><td> </td><td>Thr </td><td>0.91 </td><td><strong>0.80</strong> </td><td>0.88 </td><td><strong>0.66</strong> </td><td>1.33 </td><td>1.43 </td><td>1.13 </td><td><strong>1.51</strong> </td><td><strong>1.67</strong> </td><td><strong>1.60</strong> </td><td><strong>0.79</strong> </td><td>– </td><td>1.00 </td><td>1.8 </td><td>0.91 </td><td>0.91 </td><td>1.19 </td><td>1.08 </td><td>0.91 </td><td>1.05 </td></tr><tr><td> </td><td>Cys </td><td>0.76 </td><td>1.03 </td><td>0.90 </td><td>1.30 </td><td><strong>1.67</strong> </td><td><strong>1.36</strong> </td><td>2.00 </td><td>2.00 </td><td><strong><em>1.84</em></strong> </td><td>1.33 </td><td><strong>1.31</strong> </td><td>1.00 </td><td>– </td><td>1.00 </td><td>0.52 </td><td>1.11 </td><td>0.57 </td><td>1.44 </td><td>0.94 </td><td>1.31 </td></tr><tr><td> </td><td>Trp </td><td>1.03 </td><td>0.33 </td><td>1.03 </td><td>0.93 </td><td>1.48 </td><td>0.73 </td><td>2.67 </td><td>1.12 </td><td>7.00 </td><td>1.54 </td><td>1.52 </td><td>0.56 </td><td>1.00 </td><td>– </td><td>1.22 </td><td>1.16 </td><td>1.19 </td><td>0.8 </td><td>0.94 </td><td>1.11 </td></tr><tr><td> </td><td>Val </td><td><strong>0.55</strong> </td><td><strong>0.78</strong> </td><td>0.96 </td><td>0.91 </td><td><strong>1.56</strong> </td><td>1.32 </td><td><strong>1.23</strong> </td><td><strong>1.48</strong> </td><td>1.39 </td><td>1.31 </td><td>0.96 </td><td>1.10 </td><td>1.91 </td><td>0.82 </td><td>– </td><td><strong>1.13</strong> </td><td><strong><em>0.79</em></strong> </td><td>0.89 </td><td>1.16 </td><td>0.97 </td></tr><tr><td> </td><td>Leu </td><td>0.64 </td><td><strong>0.73</strong> </td><td>0.82 </td><td><strong>0.87</strong> </td><td><strong>1.28</strong> </td><td>1.09 </td><td><strong>1.19</strong> </td><td><strong>1.48</strong> </td><td>1.47 </td><td>1.42 </td><td><strong><em>0.87</em></strong> </td><td>1.09 </td><td>0.9 </td><td>0.86 </td><td><strong>0.89</strong> </td><td>– </td><td>0.92 </td><td>0.58 </td><td>1.13 </td><td>0.92 </td></tr><tr><td> </td><td>Glu </td><td><strong>0.78</strong> </td><td><strong>0.76</strong> </td><td><strong>0.71</strong> </td><td>0.69 </td><td>1.31 </td><td>1.00 </td><td>1.67 </td><td>1.53 </td><td>1.12 </td><td>1.08 </td><td>0.75 </td><td>0.84 </td><td>1.75 </td><td>0.84 </td><td><strong><em>1.27</em></strong> </td><td>1.09 </td><td>– </td><td><strong>0.84</strong> </td><td>1.06 </td><td>0.95 </td></tr><tr><td> </td><td>Asp </td><td><strong>0.74</strong> </td><td><strong><em>0.81</em></strong> </td><td><strong><em>0.59</em></strong> </td><td>0.76 </td><td>2.00 </td><td>1.25 </td><td>1.04 </td><td>1.43 </td><td><strong>1.24</strong> </td><td>1.12 </td><td><strong>0.72</strong> </td><td>0.93 </td><td>0.69 </td><td>1.25 </td><td>1.12 </td><td>1.72 </td><td><strong>1.19</strong> </td><td>– </td><td><strong><em>0.78</em></strong> </td><td>1.00 </td></tr><tr><td> </td><td>His </td><td>0.81 </td><td>0.62 </td><td><strong>0.71</strong> </td><td><strong>0.64</strong> </td><td>1.09 </td><td><strong>1.30</strong> </td><td>1.15 </td><td>1.04 </td><td><strong>1.76</strong> </td><td><strong>1.75</strong> </td><td>1.01 </td><td>1.10 </td><td>1.07 </td><td>1.06 </td><td>0.86 </td><td>0.89 </td><td>0.94 </td><td><strong><em>1.28</em></strong> </td><td>– </td><td>1.09 </td></tr><tr><td> </td><td>Gln </td><td>0.93 </td><td>0.69 </td><td>0.92 </td><td><strong>0.59</strong> </td><td>1.170 </td><td>1.20 </td><td>0.96 </td><td>1.77 </td><td>1.36 </td><td><strong>1.37</strong> </td><td>0.91 </td><td>0.95 </td><td>0.77 </td><td>0.90 </td><td>1.03 </td><td>1.08 </td><td>1.05 </td><td>1.00 </td><td>0.91 </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each element <em>R<sub>ij</sub></em> in the AARM represents the ratio of number of replacements of the residue <em>i</em> by the residue <em>j</em> in the forward direction (mouse → human) to that in the reverse direction. This means that if <em>R<sub>ij</sub></em> &gt;1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements and if <em>R<sub>ij</sub></em> &lt;1, the reverse is true. Bold and Bold-italics ratios signifies the directional bias at <em>p</em> &lt; 0.0001 and &lt; 0.001 respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194680" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87194681 id="87194681" class="section-title" data-legacy-id=s2d>2.4. Analysis of nucleotide substitution patterns: evaluation of nucleotide replacement matrix (NRM) for three codon sites of different groups of orthologs</h3>
<p>We created the nucleotide sequence alignments on the basis of amino acid alignments and calculated the nucleotide replacements in the form of 4 × 4 NRM, individually for three codon positions for three different groups of orthologs under study. The elements r<sub>ij</sub> of NRM represent the ratio of the number of forward replacements to that of backward replacements for any specific pairs of nucleotides. Comparison of the nucleotide replacement values obtained from different subsets of 500 orthologous sequences (taken sequentially from start to end and also randomly) of any particular group was then carried out and no significant variations in replacement values were found for individual nucleotide pairs within a group.</p>                            <h3 scrollto-destination=87194683 id="87194683" class="section-title" data-legacy-id=s2e>2.5. Tests for statistical significance of different elements (<em>R<sub>ij</sub>/r<sub>ij</sub></em>) of AARM and NRM</h3>
<p>For a given pair of amino acids or nucleotides, the mouse to human replacement was taken as the forward direction and human to mouse as the reverse direction, and each <em>R<sub>ij</sub></em> in AARM or <em>r<sub>ij</sub></em> in NRM represents the ratio of number of replacements of the residue <em>i</em> by the residue <em>j</em> in the forward direction (mouse to human) to that in the reverse direction (human to mouse). This means that if <em>R<sub>ij</sub></em> (or <em>r<sub>ij</sub></em>) &gt; 1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements, and if <em>R<sub>ij</sub></em> (or <em>r<sub>ij</sub></em>) &lt; 1, the reverse is true.</p><p>For each pair of replacements, the ratio of forward to reverse replacements was expected to be 1:1 under unbiased conditions. To test this hypothesis, the observed and expected numbers (based on a 1:1 ratio) were recorded for each pair of a particular group. In all cases, the Chi-square test was used to assess the significance of the directional bias, if any, at <em>p</em> = 0.001 and 0.0001. For each pair of replacements, the first and second rows of the 2 × 2 contingency table represented the number of replacements from one particular residue (say, <em>i</em>) to another (say, <em>j</em>) of the pair and the total count of the remaining replacements (say, <em>k</em>) from the residue <em>i</em> (where <em>k</em> ≠ <em>j</em>), respectively. The procedure was repeated also for orthologous replacements of 500 sequences taken sequentially from start to end and randomly. The significant (at <em>p</em> &lt; 0.001 or 0.0001) trends for the whole dataset are also consistent with sequences taken sequentially from start to end and randomly.</p>                            <h3 scrollto-destination=87194686 id="87194686" class="section-title" data-legacy-id=s2f>2.6. Correspondence analysis (COA) on relative synonymous codon usage (RSCU) and estimation of synonymous and nonsynonymous substitution</h3>
<p>Correspondence analysis on RSCU<sup><a reveal-id="DSM015C26" data-open="DSM015C26" class="link link-ref link-reveal xref-bibr">26</a></sup> was performed using the CodonW 1.4.2<sup><a reveal-id="DSM015C22" data-open="DSM015C22" class="link link-ref link-reveal xref-bibr">22</a></sup> program to identify the major factors influencing the variation in synonymous codon usage in three groups of orthologous sets. These analyses generate a series of orthogonal axes to identify trends that explain the variation within a dataset, with each subsequent axis explaining a decreasing amount of the variation.</p><p>To examine the nucleotide substitution patterns, we estimated the number of synonymous substitutions per synonymous site, <em>d</em><sub>S</sub>, and the number of nonsynonymous substitutions per nonsynonymous site, <em>d</em><sub>N</sub>, of randomly chosen 500 pairs of ten sets of orthologs in each three groups using the MEGA program (version 2.1), as described by Nei and Gojobori.<sup><a reveal-id="DSM015C27" data-open="DSM015C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The values of <em>d</em><sub>S</sub>, <em>d</em><sub>N</sub>, and <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> of three orthologous groups were compared by <em>t</em>-test.</p>                            <h2 scrollto-destination=87194689 id="87194689" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87194690 id="87194690" class="section-title" data-legacy-id=s3a>3.1. Specific trends in amino acid substitution patterns between mouse and human orthologs</h3>
<p>In order to investigate whether the mouse and human proteins of high-, medium-, and low-GC composition followed the same or different evolutionary trajectories since the divergence of the two species, trends in amino acid substitution between the human–mouse orthologous pairs were studied individually in three groups of genes, using the program SPAST developed in-house. Tables <a reveal-id="DSM015TB1" data-open="DSM015TB1" class="link link-reveal link-table xref-fig">1</a>, <a reveal-id="DSM015TB2" data-open="DSM015TB2" class="link link-reveal link-table xref-fig">2</a>, and <a reveal-id="DSM015TB3" data-open="DSM015TB3" class="link link-reveal link-table xref-fig">3</a> represents the AARMs for aligned regions of orthologous pairs (gap-free regions of length &gt;100 residues at a stretch) in high-, medium-, and low-GC groups, respectively. As already mentioned, the mouse to human replacements was taken by convention as the forward direction and human to mouse as the reverse direction.</p><p>For each group of orthologs, some specific amino acid pairs exhibit significant bias in the replacement patterns. For instance, in high-GC group, the value of <em>R</em><sub>IG</sub>, i.e. the ratio of [Ile]<sub>Mouse</sub> → [Gly]<sub>Human</sub> to [Gly]<sub>Mouse</sub> → [Ile]<sub>Human</sub>, is 2.51 (<em>p</em> &lt; 0.0001), implying that the frequency of replacement of Ile in mouse sequence with Gly in human is &gt;2.5-fold higher than that in reverse direction, i.e. the frequency of replacement of Gly of mouse sequence with Ile in human. On the contrary, in low-GC group, <em>R</em><sub>IG</sub> = 0.48, indicating that in low-GC orthologs, the frequency of substitution of Ile of mouse sequence by Gly in human is more than two-fold lower than the frequency of the reverse substitution. For the medium-GC group, the value of <em>R</em><sub>IG</sub> is not statistically significant, suggesting that the frequencies of substitution of Ile by Gly and of Gly by Ile are comparable in cases of medium-GC orthologs of mouse and human.</p><p>As <em>R<sub>ij</sub></em> = 1/<em>R<sub>ji</sub></em>, out of the 380 off-diagonal elements of an AARM (Tables <a reveal-id="DSM015TB1" data-open="DSM015TB1" class="link link-reveal link-table xref-fig">1</a>, <a reveal-id="DSM015TB2" data-open="DSM015TB2" class="link link-reveal link-table xref-fig">2</a>, and <a reveal-id="DSM015TB3" data-open="DSM015TB3" class="link link-reveal link-table xref-fig">3</a>), only 190 are independent. Out of these 190 AARM elements, 53 are significantly biased in a specific direction for high-GC group (46 at <em>p</em> &lt; 0.0001 and seven at <em>p</em> &lt; 0.001), whereas for low-GC group, 67 elements are found to be significantly biased (56 and 11 at <em>p</em> &lt; 0.0001 and 0.001, respectively). In the medium-GC group, only 15 AARM elements are statistically significant (ten at <em>p</em> &lt; 0.0001 and five at <em>p</em> &lt; 0.001). Some of them are shared with the high-GC group and some with the low-GC group.</p>                            <h3 scrollto-destination=87194694 id="87194694" class="section-title" data-legacy-id=s3b>3.2. Significant trends in amino acid substitution in high-GC and low-GC groups are, in general, opposite to one another</h3>
<p>A careful examination of Tables <a reveal-id="DSM015TB1" data-open="DSM015TB1" class="link link-reveal link-table xref-fig">1</a>, <a reveal-id="DSM015TB2" data-open="DSM015TB2" class="link link-reveal link-table xref-fig">2</a>, and <a reveal-id="DSM015TB3" data-open="DSM015TB3" class="link link-reveal link-table xref-fig">3</a> reveals that when <em>i</em> represents a residue encoded by A/U-rich codons and <em>j</em> represents a residue encoded by relatively G/C-rich codons, the AARM element, <em>R<sub>ij</sub></em><sub>,</sub> in most cases (but not in all), is &gt;1 in the high-GC group, &lt;1 in the low-GC group, and nearly equal to ∼1 in the medium-GC group. Reverse situation occurs, in general, when <em>i</em> represents a residue encoded by G/C-rich codons and <em>j</em> by relatively A/U-rich codons. For instance, for <em>i</em> = Ala (A) (encoded by GCN), <em>R</em><sub>A<em>j</em></sub> is significantly &lt;1 in high-GC group and significantly &gt;1 in low-GC group for <em>j</em> = Ile, Asn, Lys, Ser, Thr, Val, Glu etc (encoded, respectively, by AUH, AAY, AAR, UCN/AGY, ACN, GUN, and GAR). On the contrary, for <em>i</em> = Asn (N) (encoded by AAY), <em>R</em><sub>N<em>j</em></sub> &gt; 1 for high-GC group and &lt;1 for low-GC group, when <em>j</em> = Gly or Ala or Arg (encoded by GGN, GCN, and CGN/AGR, respectively). There are altogether 33 AARM elements, which are polarized to the opposite directions (&gt;1 and &lt;1) in high- and low-GC groups and are found to be statistically significant in both groups.</p><p>Table <a reveal-id="DSM015TB4" data-open="DSM015TB4" class="link link-reveal link-table xref-fig">4</a> provides the lists of the 15 amino acid pairs having the largest differences in total number of forward (mouse to human) and backward (human to mouse) substitutions between them for three different groups of orthologs under study. There are eight pairs of residues (marked with ±) that appear among the top 15 trends in both high- and low-GC groups, but with opposite directionality (Table <a reveal-id="DSM015TB4" data-open="DSM015TB4" class="link link-reveal link-table xref-fig">4</a>). There are four other pairs of residues among the top 15 of the high-GC group (marked with +), which exhibit significant, but opposite, bias in the low-GC group (Table <a reveal-id="DSM015TB4" data-open="DSM015TB4" class="link link-reveal link-table xref-fig">4</a>), but did not come among the top 15 in the later group. Similarly, there are also four pairs (marked with –) among the top 15 of the low-GC group, which are opposite and significant, but not among the top 15 in the high-GC group. Thus, the trends in amino acid substitutions between the mouse and human orthologs follow reverse directionality, in general, in the high- and low-GC groups. Among the top 15 trends in the medium-GC groups, some are common in directionality with high-GC group and some with the low-GC group.</p>                        <a id="87194697" scrollto-destination="87194697"></a>
<div content-id="DSM015TB4" class="table-modal"><div class="table-wrap"><div id="DSM015TB4" class="table-wrap-title" data-id="DSM015TB4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Top 15 amino acid pairs of three orthologous groups according to differences in number of forward (mouse to human) and reverse (human to mouse) replacements in AARM</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pair </th><th>Forward no. </th><th>Reverse no. </th><th>Difference </th><th>Ratio </th><th>Codon changes </th><th>Trends </th></tr></thead><tbody><tr><td>High-GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Thr → Ala* </td><td>6873 </td><td>4020 </td><td>2853 </td><td>1.71 </td><td>ACN → GCN </td><td>± </td></tr><tr><td>Ser → Ala* </td><td>4160 </td><td>2447 </td><td>1713 </td><td>1.70 </td><td>UCN/AGY → GCN </td><td>± </td></tr><tr><td>Ser → Pro* </td><td>3603 </td><td>2097 </td><td>1506 </td><td>1.72 </td><td>UCN/AGY → CCN </td><td>± </td></tr><tr><td>Val → Ala* </td><td>4484 </td><td>3144 </td><td>1340 </td><td>1.43 </td><td>GUN → GCN </td><td>± </td></tr><tr><td>Ser → Gly* </td><td>3749 </td><td>2516 </td><td>1233 </td><td>1.49 </td><td>UCN/AGY → GGN </td><td>+ </td></tr><tr><td>Gln → Arg* </td><td>3389 </td><td>2448 </td><td>941 </td><td>1.38 </td><td>CAR → CGN/AGR </td><td> </td></tr><tr><td>Lys → Arg* </td><td>3412 </td><td>2518 </td><td>894 </td><td>1.36 </td><td>AAR → CGN/AGR </td><td>± </td></tr><tr><td>Ile → Val* </td><td>4572 </td><td>3756 </td><td>816 </td><td>1.22 </td><td>AUH → GUN </td><td>± </td></tr><tr><td>Asn → Ser* </td><td>2878 </td><td>2145 </td><td>733 </td><td>1.34 </td><td>AAY → UCN/AGY </td><td>± </td></tr><tr><td>Asp → Glu* </td><td>3799 </td><td>3237 </td><td>562 </td><td>1.17 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>His → Arg* </td><td>1899 </td><td>1343 </td><td>556 </td><td>1.41 </td><td>CAY → CGN/AGR </td><td>+ </td></tr><tr><td>Met → Leu* </td><td>1753 </td><td>1211 </td><td>542 </td><td>1.45 </td><td>AUG → UUR/CUN </td><td>+ </td></tr><tr><td>Ile → Leu* </td><td>1500 </td><td>1012 </td><td>488 </td><td>1.48 </td><td>AUN → UUR/CUN </td><td>± </td></tr><tr><td>Lys → Glu* </td><td>1257 </td><td>785 </td><td>472 </td><td>1.60 </td><td>AAR → GAR </td><td> </td></tr><tr><td>Leu → Pro* </td><td>2060 </td><td>1634 </td><td>426 </td><td>1.26 </td><td>YYR/CUN → CCN </td><td>+ </td></tr><tr><td>Medium-GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Val → Ile* </td><td>5839 </td><td>4982 </td><td>857 </td><td>1.17 </td><td>GUN → AUH </td><td>□ </td></tr><tr><td>Thr → Ala* </td><td>5566 </td><td>4753 </td><td>813 </td><td>1.17 </td><td>ACN → GCN </td><td>○ </td></tr><tr><td>Ser → Pro* </td><td>3804 </td><td>3113 </td><td>691 </td><td>1.22 </td><td>UCN/AGY → CCN </td><td>○ </td></tr><tr><td>Asp → Glu* </td><td>4443 </td><td>3833 </td><td>610 </td><td>1.16 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>Ser → Thr* </td><td>3652 </td><td>3113 </td><td>539 </td><td>1.17 </td><td>UCN/AGY → ACN </td><td>□ </td></tr><tr><td>Ser → Ala* </td><td>3507 </td><td>2972 </td><td>535 </td><td>1.18 </td><td>UCN/AGY → GCN </td><td>○ </td></tr><tr><td>Ser → Asn* </td><td>3626 </td><td>3213 </td><td>413 </td><td>1.13 </td><td>UCN/AGY → AAY </td><td>□ </td></tr><tr><td>Arg → Lys** </td><td>4017 </td><td>3701 </td><td>316 </td><td>1.08 </td><td>CGN/AGR → AAR </td><td>□ </td></tr><tr><td>Leu → Pro* </td><td>1999 </td><td>1691 </td><td>308 </td><td>1.18 </td><td>UUR/CUN → CCN </td><td>○ </td></tr><tr><td>Gln → Arg* </td><td>2676 </td><td>2372 </td><td>304 </td><td>1.13 </td><td>CAR → CGN/AGR </td><td>○ </td></tr><tr><td>Ser → Gly </td><td>2782 </td><td>2560 </td><td>222 </td><td>1.09 </td><td>UCN/AGY → GGN </td><td>○ </td></tr><tr><td>Val → Ala </td><td>3759 </td><td>3561 </td><td>198 </td><td>1.06 </td><td>GUN → GCN </td><td>○ </td></tr><tr><td>Lys → Glu** </td><td>1514 </td><td>1336 </td><td>178 </td><td>1.13 </td><td>AAR → GAR </td><td>○ </td></tr><tr><td>Asp → Ala* </td><td>661 </td><td>488 </td><td>173 </td><td>1.35 </td><td>GSN → GCN </td><td> </td></tr><tr><td>Met → Leu </td><td>1648 </td><td>1483 </td><td>165 </td><td>1.11 </td><td>AUG → UUR/CUN </td><td>○ </td></tr><tr><td>Low GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Val → Ile* </td><td>9402 </td><td>6335 </td><td>3067 </td><td>1.48 </td><td>GUN → AUH </td><td>± </td></tr><tr><td>Ser → Asn* </td><td>6508 </td><td>3733 </td><td>2775 </td><td>1.74 </td><td>UCN/AGY → AAY </td><td>± </td></tr><tr><td>Arg → Lys* </td><td>6603 </td><td>4059 </td><td>2544 </td><td>1.63 </td><td>CGN/AGR → AAR </td><td>± </td></tr><tr><td>Ala → Thr* </td><td>6393 </td><td>5117 </td><td>1276 </td><td>1.25 </td><td>GCN → ACN </td><td>± </td></tr><tr><td>Leu → Ile* </td><td>3003 </td><td>2035 </td><td>968 </td><td>1.48 </td><td>UUR/CUN → AUH </td><td>± </td></tr><tr><td>Asp → Glu* </td><td>5952 </td><td>5014 </td><td>938 </td><td>1.19 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>Ser → Thr* </td><td>4498 </td><td>3563 </td><td>935 </td><td>1.26 </td><td>UCN/AGY → ACN </td><td> </td></tr><tr><td>Ala → Val* </td><td>4193 </td><td>3264 </td><td>929 </td><td>1.28 </td><td>GCN → GUN </td><td>± </td></tr><tr><td>Thr → Ile* </td><td>2612 </td><td>1733 </td><td>879 </td><td>1.51 </td><td>CAN → AUH </td><td>– </td></tr><tr><td>Pro → Ser* </td><td>4178 </td><td>3304 </td><td>874 </td><td>1.26 </td><td>CCN → UCN/AGY </td><td>± </td></tr><tr><td>Ala → Ser* </td><td>3975 </td><td>3195 </td><td>780 </td><td>1.24 </td><td>GCN → UCN/AGY </td><td>± </td></tr><tr><td>Leu → Phe* </td><td>2990 </td><td>2330 </td><td>660 </td><td>1.28 </td><td>UUR/CUN → UUY </td><td>– </td></tr><tr><td>Thr → Asn* </td><td>1630 </td><td>975 </td><td>655 </td><td>1.67 </td><td>CAN → AAY </td><td> </td></tr><tr><td>Met → Ile* </td><td>1900 </td><td>1288 </td><td>612 </td><td>1.48 </td><td>AUG → AUH </td><td>– </td></tr><tr><td>His → Asn* </td><td>1258 </td><td>715 </td><td>543 </td><td>1.76 </td><td>CAY → AAY </td><td>– </td></tr></tbody></table></div><div content-id="DSM015TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Pair </th><th>Forward no. </th><th>Reverse no. </th><th>Difference </th><th>Ratio </th><th>Codon changes </th><th>Trends </th></tr></thead><tbody><tr><td>High-GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Thr → Ala* </td><td>6873 </td><td>4020 </td><td>2853 </td><td>1.71 </td><td>ACN → GCN </td><td>± </td></tr><tr><td>Ser → Ala* </td><td>4160 </td><td>2447 </td><td>1713 </td><td>1.70 </td><td>UCN/AGY → GCN </td><td>± </td></tr><tr><td>Ser → Pro* </td><td>3603 </td><td>2097 </td><td>1506 </td><td>1.72 </td><td>UCN/AGY → CCN </td><td>± </td></tr><tr><td>Val → Ala* </td><td>4484 </td><td>3144 </td><td>1340 </td><td>1.43 </td><td>GUN → GCN </td><td>± </td></tr><tr><td>Ser → Gly* </td><td>3749 </td><td>2516 </td><td>1233 </td><td>1.49 </td><td>UCN/AGY → GGN </td><td>+ </td></tr><tr><td>Gln → Arg* </td><td>3389 </td><td>2448 </td><td>941 </td><td>1.38 </td><td>CAR → CGN/AGR </td><td> </td></tr><tr><td>Lys → Arg* </td><td>3412 </td><td>2518 </td><td>894 </td><td>1.36 </td><td>AAR → CGN/AGR </td><td>± </td></tr><tr><td>Ile → Val* </td><td>4572 </td><td>3756 </td><td>816 </td><td>1.22 </td><td>AUH → GUN </td><td>± </td></tr><tr><td>Asn → Ser* </td><td>2878 </td><td>2145 </td><td>733 </td><td>1.34 </td><td>AAY → UCN/AGY </td><td>± </td></tr><tr><td>Asp → Glu* </td><td>3799 </td><td>3237 </td><td>562 </td><td>1.17 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>His → Arg* </td><td>1899 </td><td>1343 </td><td>556 </td><td>1.41 </td><td>CAY → CGN/AGR </td><td>+ </td></tr><tr><td>Met → Leu* </td><td>1753 </td><td>1211 </td><td>542 </td><td>1.45 </td><td>AUG → UUR/CUN </td><td>+ </td></tr><tr><td>Ile → Leu* </td><td>1500 </td><td>1012 </td><td>488 </td><td>1.48 </td><td>AUN → UUR/CUN </td><td>± </td></tr><tr><td>Lys → Glu* </td><td>1257 </td><td>785 </td><td>472 </td><td>1.60 </td><td>AAR → GAR </td><td> </td></tr><tr><td>Leu → Pro* </td><td>2060 </td><td>1634 </td><td>426 </td><td>1.26 </td><td>YYR/CUN → CCN </td><td>+ </td></tr><tr><td>Medium-GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Val → Ile* </td><td>5839 </td><td>4982 </td><td>857 </td><td>1.17 </td><td>GUN → AUH </td><td>□ </td></tr><tr><td>Thr → Ala* </td><td>5566 </td><td>4753 </td><td>813 </td><td>1.17 </td><td>ACN → GCN </td><td>○ </td></tr><tr><td>Ser → Pro* </td><td>3804 </td><td>3113 </td><td>691 </td><td>1.22 </td><td>UCN/AGY → CCN </td><td>○ </td></tr><tr><td>Asp → Glu* </td><td>4443 </td><td>3833 </td><td>610 </td><td>1.16 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>Ser → Thr* </td><td>3652 </td><td>3113 </td><td>539 </td><td>1.17 </td><td>UCN/AGY → ACN </td><td>□ </td></tr><tr><td>Ser → Ala* </td><td>3507 </td><td>2972 </td><td>535 </td><td>1.18 </td><td>UCN/AGY → GCN </td><td>○ </td></tr><tr><td>Ser → Asn* </td><td>3626 </td><td>3213 </td><td>413 </td><td>1.13 </td><td>UCN/AGY → AAY </td><td>□ </td></tr><tr><td>Arg → Lys** </td><td>4017 </td><td>3701 </td><td>316 </td><td>1.08 </td><td>CGN/AGR → AAR </td><td>□ </td></tr><tr><td>Leu → Pro* </td><td>1999 </td><td>1691 </td><td>308 </td><td>1.18 </td><td>UUR/CUN → CCN </td><td>○ </td></tr><tr><td>Gln → Arg* </td><td>2676 </td><td>2372 </td><td>304 </td><td>1.13 </td><td>CAR → CGN/AGR </td><td>○ </td></tr><tr><td>Ser → Gly </td><td>2782 </td><td>2560 </td><td>222 </td><td>1.09 </td><td>UCN/AGY → GGN </td><td>○ </td></tr><tr><td>Val → Ala </td><td>3759 </td><td>3561 </td><td>198 </td><td>1.06 </td><td>GUN → GCN </td><td>○ </td></tr><tr><td>Lys → Glu** </td><td>1514 </td><td>1336 </td><td>178 </td><td>1.13 </td><td>AAR → GAR </td><td>○ </td></tr><tr><td>Asp → Ala* </td><td>661 </td><td>488 </td><td>173 </td><td>1.35 </td><td>GSN → GCN </td><td> </td></tr><tr><td>Met → Leu </td><td>1648 </td><td>1483 </td><td>165 </td><td>1.11 </td><td>AUG → UUR/CUN </td><td>○ </td></tr><tr><td>Low GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Val → Ile* </td><td>9402 </td><td>6335 </td><td>3067 </td><td>1.48 </td><td>GUN → AUH </td><td>± </td></tr><tr><td>Ser → Asn* </td><td>6508 </td><td>3733 </td><td>2775 </td><td>1.74 </td><td>UCN/AGY → AAY </td><td>± </td></tr><tr><td>Arg → Lys* </td><td>6603 </td><td>4059 </td><td>2544 </td><td>1.63 </td><td>CGN/AGR → AAR </td><td>± </td></tr><tr><td>Ala → Thr* </td><td>6393 </td><td>5117 </td><td>1276 </td><td>1.25 </td><td>GCN → ACN </td><td>± </td></tr><tr><td>Leu → Ile* </td><td>3003 </td><td>2035 </td><td>968 </td><td>1.48 </td><td>UUR/CUN → AUH </td><td>± </td></tr><tr><td>Asp → Glu* </td><td>5952 </td><td>5014 </td><td>938 </td><td>1.19 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>Ser → Thr* </td><td>4498 </td><td>3563 </td><td>935 </td><td>1.26 </td><td>UCN/AGY → ACN </td><td> </td></tr><tr><td>Ala → Val* </td><td>4193 </td><td>3264 </td><td>929 </td><td>1.28 </td><td>GCN → GUN </td><td>± </td></tr><tr><td>Thr → Ile* </td><td>2612 </td><td>1733 </td><td>879 </td><td>1.51 </td><td>CAN → AUH </td><td>– </td></tr><tr><td>Pro → Ser* </td><td>4178 </td><td>3304 </td><td>874 </td><td>1.26 </td><td>CCN → UCN/AGY </td><td>± </td></tr><tr><td>Ala → Ser* </td><td>3975 </td><td>3195 </td><td>780 </td><td>1.24 </td><td>GCN → UCN/AGY </td><td>± </td></tr><tr><td>Leu → Phe* </td><td>2990 </td><td>2330 </td><td>660 </td><td>1.28 </td><td>UUR/CUN → UUY </td><td>– </td></tr><tr><td>Thr → Asn* </td><td>1630 </td><td>975 </td><td>655 </td><td>1.67 </td><td>CAN → AAY </td><td> </td></tr><tr><td>Met → Ile* </td><td>1900 </td><td>1288 </td><td>612 </td><td>1.48 </td><td>AUG → AUH </td><td>– </td></tr><tr><td>His → Asn* </td><td>1258 </td><td>715 </td><td>543 </td><td>1.76 </td><td>CAY → AAY </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*<sup>,</sup>**Replacements of pairs are significant at <em>p</em> &lt; 0.0001 and 0.001, respectively.</p></div></div><div content-id=""><div class="fn"><p>Symbols used in codons: R, A/G; Y, C/T; S, G/C; W, A/T; H, A/C/T; N, any nucleotide. Symbols used in trends: opposite trends among top 15 amino acid pairs of both high- and low-GC groups (±); opposite trends in high- and low-GC group, but not present in top 15 of low-GC group (+); opposite trends in high- and low-GC group, but not present in top 15 of high-GC group (–); same trend in high-, medium-, and low-GC groups (•); same trend in high- and medium-GC groups (○); same trend in low- and medium-GC groups (□).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194697" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM015TB4" class="table-wrap-title" data-id="DSM015TB4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Top 15 amino acid pairs of three orthologous groups according to differences in number of forward (mouse to human) and reverse (human to mouse) replacements in AARM</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pair </th><th>Forward no. </th><th>Reverse no. </th><th>Difference </th><th>Ratio </th><th>Codon changes </th><th>Trends </th></tr></thead><tbody><tr><td>High-GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Thr → Ala* </td><td>6873 </td><td>4020 </td><td>2853 </td><td>1.71 </td><td>ACN → GCN </td><td>± </td></tr><tr><td>Ser → Ala* </td><td>4160 </td><td>2447 </td><td>1713 </td><td>1.70 </td><td>UCN/AGY → GCN </td><td>± </td></tr><tr><td>Ser → Pro* </td><td>3603 </td><td>2097 </td><td>1506 </td><td>1.72 </td><td>UCN/AGY → CCN </td><td>± </td></tr><tr><td>Val → Ala* </td><td>4484 </td><td>3144 </td><td>1340 </td><td>1.43 </td><td>GUN → GCN </td><td>± </td></tr><tr><td>Ser → Gly* </td><td>3749 </td><td>2516 </td><td>1233 </td><td>1.49 </td><td>UCN/AGY → GGN </td><td>+ </td></tr><tr><td>Gln → Arg* </td><td>3389 </td><td>2448 </td><td>941 </td><td>1.38 </td><td>CAR → CGN/AGR </td><td> </td></tr><tr><td>Lys → Arg* </td><td>3412 </td><td>2518 </td><td>894 </td><td>1.36 </td><td>AAR → CGN/AGR </td><td>± </td></tr><tr><td>Ile → Val* </td><td>4572 </td><td>3756 </td><td>816 </td><td>1.22 </td><td>AUH → GUN </td><td>± </td></tr><tr><td>Asn → Ser* </td><td>2878 </td><td>2145 </td><td>733 </td><td>1.34 </td><td>AAY → UCN/AGY </td><td>± </td></tr><tr><td>Asp → Glu* </td><td>3799 </td><td>3237 </td><td>562 </td><td>1.17 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>His → Arg* </td><td>1899 </td><td>1343 </td><td>556 </td><td>1.41 </td><td>CAY → CGN/AGR </td><td>+ </td></tr><tr><td>Met → Leu* </td><td>1753 </td><td>1211 </td><td>542 </td><td>1.45 </td><td>AUG → UUR/CUN </td><td>+ </td></tr><tr><td>Ile → Leu* </td><td>1500 </td><td>1012 </td><td>488 </td><td>1.48 </td><td>AUN → UUR/CUN </td><td>± </td></tr><tr><td>Lys → Glu* </td><td>1257 </td><td>785 </td><td>472 </td><td>1.60 </td><td>AAR → GAR </td><td> </td></tr><tr><td>Leu → Pro* </td><td>2060 </td><td>1634 </td><td>426 </td><td>1.26 </td><td>YYR/CUN → CCN </td><td>+ </td></tr><tr><td>Medium-GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Val → Ile* </td><td>5839 </td><td>4982 </td><td>857 </td><td>1.17 </td><td>GUN → AUH </td><td>□ </td></tr><tr><td>Thr → Ala* </td><td>5566 </td><td>4753 </td><td>813 </td><td>1.17 </td><td>ACN → GCN </td><td>○ </td></tr><tr><td>Ser → Pro* </td><td>3804 </td><td>3113 </td><td>691 </td><td>1.22 </td><td>UCN/AGY → CCN </td><td>○ </td></tr><tr><td>Asp → Glu* </td><td>4443 </td><td>3833 </td><td>610 </td><td>1.16 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>Ser → Thr* </td><td>3652 </td><td>3113 </td><td>539 </td><td>1.17 </td><td>UCN/AGY → ACN </td><td>□ </td></tr><tr><td>Ser → Ala* </td><td>3507 </td><td>2972 </td><td>535 </td><td>1.18 </td><td>UCN/AGY → GCN </td><td>○ </td></tr><tr><td>Ser → Asn* </td><td>3626 </td><td>3213 </td><td>413 </td><td>1.13 </td><td>UCN/AGY → AAY </td><td>□ </td></tr><tr><td>Arg → Lys** </td><td>4017 </td><td>3701 </td><td>316 </td><td>1.08 </td><td>CGN/AGR → AAR </td><td>□ </td></tr><tr><td>Leu → Pro* </td><td>1999 </td><td>1691 </td><td>308 </td><td>1.18 </td><td>UUR/CUN → CCN </td><td>○ </td></tr><tr><td>Gln → Arg* </td><td>2676 </td><td>2372 </td><td>304 </td><td>1.13 </td><td>CAR → CGN/AGR </td><td>○ </td></tr><tr><td>Ser → Gly </td><td>2782 </td><td>2560 </td><td>222 </td><td>1.09 </td><td>UCN/AGY → GGN </td><td>○ </td></tr><tr><td>Val → Ala </td><td>3759 </td><td>3561 </td><td>198 </td><td>1.06 </td><td>GUN → GCN </td><td>○ </td></tr><tr><td>Lys → Glu** </td><td>1514 </td><td>1336 </td><td>178 </td><td>1.13 </td><td>AAR → GAR </td><td>○ </td></tr><tr><td>Asp → Ala* </td><td>661 </td><td>488 </td><td>173 </td><td>1.35 </td><td>GSN → GCN </td><td> </td></tr><tr><td>Met → Leu </td><td>1648 </td><td>1483 </td><td>165 </td><td>1.11 </td><td>AUG → UUR/CUN </td><td>○ </td></tr><tr><td>Low GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Val → Ile* </td><td>9402 </td><td>6335 </td><td>3067 </td><td>1.48 </td><td>GUN → AUH </td><td>± </td></tr><tr><td>Ser → Asn* </td><td>6508 </td><td>3733 </td><td>2775 </td><td>1.74 </td><td>UCN/AGY → AAY </td><td>± </td></tr><tr><td>Arg → Lys* </td><td>6603 </td><td>4059 </td><td>2544 </td><td>1.63 </td><td>CGN/AGR → AAR </td><td>± </td></tr><tr><td>Ala → Thr* </td><td>6393 </td><td>5117 </td><td>1276 </td><td>1.25 </td><td>GCN → ACN </td><td>± </td></tr><tr><td>Leu → Ile* </td><td>3003 </td><td>2035 </td><td>968 </td><td>1.48 </td><td>UUR/CUN → AUH </td><td>± </td></tr><tr><td>Asp → Glu* </td><td>5952 </td><td>5014 </td><td>938 </td><td>1.19 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>Ser → Thr* </td><td>4498 </td><td>3563 </td><td>935 </td><td>1.26 </td><td>UCN/AGY → ACN </td><td> </td></tr><tr><td>Ala → Val* </td><td>4193 </td><td>3264 </td><td>929 </td><td>1.28 </td><td>GCN → GUN </td><td>± </td></tr><tr><td>Thr → Ile* </td><td>2612 </td><td>1733 </td><td>879 </td><td>1.51 </td><td>CAN → AUH </td><td>– </td></tr><tr><td>Pro → Ser* </td><td>4178 </td><td>3304 </td><td>874 </td><td>1.26 </td><td>CCN → UCN/AGY </td><td>± </td></tr><tr><td>Ala → Ser* </td><td>3975 </td><td>3195 </td><td>780 </td><td>1.24 </td><td>GCN → UCN/AGY </td><td>± </td></tr><tr><td>Leu → Phe* </td><td>2990 </td><td>2330 </td><td>660 </td><td>1.28 </td><td>UUR/CUN → UUY </td><td>– </td></tr><tr><td>Thr → Asn* </td><td>1630 </td><td>975 </td><td>655 </td><td>1.67 </td><td>CAN → AAY </td><td> </td></tr><tr><td>Met → Ile* </td><td>1900 </td><td>1288 </td><td>612 </td><td>1.48 </td><td>AUG → AUH </td><td>– </td></tr><tr><td>His → Asn* </td><td>1258 </td><td>715 </td><td>543 </td><td>1.76 </td><td>CAY → AAY </td><td>– </td></tr></tbody></table></div><div content-id="DSM015TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Pair </th><th>Forward no. </th><th>Reverse no. </th><th>Difference </th><th>Ratio </th><th>Codon changes </th><th>Trends </th></tr></thead><tbody><tr><td>High-GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Thr → Ala* </td><td>6873 </td><td>4020 </td><td>2853 </td><td>1.71 </td><td>ACN → GCN </td><td>± </td></tr><tr><td>Ser → Ala* </td><td>4160 </td><td>2447 </td><td>1713 </td><td>1.70 </td><td>UCN/AGY → GCN </td><td>± </td></tr><tr><td>Ser → Pro* </td><td>3603 </td><td>2097 </td><td>1506 </td><td>1.72 </td><td>UCN/AGY → CCN </td><td>± </td></tr><tr><td>Val → Ala* </td><td>4484 </td><td>3144 </td><td>1340 </td><td>1.43 </td><td>GUN → GCN </td><td>± </td></tr><tr><td>Ser → Gly* </td><td>3749 </td><td>2516 </td><td>1233 </td><td>1.49 </td><td>UCN/AGY → GGN </td><td>+ </td></tr><tr><td>Gln → Arg* </td><td>3389 </td><td>2448 </td><td>941 </td><td>1.38 </td><td>CAR → CGN/AGR </td><td> </td></tr><tr><td>Lys → Arg* </td><td>3412 </td><td>2518 </td><td>894 </td><td>1.36 </td><td>AAR → CGN/AGR </td><td>± </td></tr><tr><td>Ile → Val* </td><td>4572 </td><td>3756 </td><td>816 </td><td>1.22 </td><td>AUH → GUN </td><td>± </td></tr><tr><td>Asn → Ser* </td><td>2878 </td><td>2145 </td><td>733 </td><td>1.34 </td><td>AAY → UCN/AGY </td><td>± </td></tr><tr><td>Asp → Glu* </td><td>3799 </td><td>3237 </td><td>562 </td><td>1.17 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>His → Arg* </td><td>1899 </td><td>1343 </td><td>556 </td><td>1.41 </td><td>CAY → CGN/AGR </td><td>+ </td></tr><tr><td>Met → Leu* </td><td>1753 </td><td>1211 </td><td>542 </td><td>1.45 </td><td>AUG → UUR/CUN </td><td>+ </td></tr><tr><td>Ile → Leu* </td><td>1500 </td><td>1012 </td><td>488 </td><td>1.48 </td><td>AUN → UUR/CUN </td><td>± </td></tr><tr><td>Lys → Glu* </td><td>1257 </td><td>785 </td><td>472 </td><td>1.60 </td><td>AAR → GAR </td><td> </td></tr><tr><td>Leu → Pro* </td><td>2060 </td><td>1634 </td><td>426 </td><td>1.26 </td><td>YYR/CUN → CCN </td><td>+ </td></tr><tr><td>Medium-GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Val → Ile* </td><td>5839 </td><td>4982 </td><td>857 </td><td>1.17 </td><td>GUN → AUH </td><td>□ </td></tr><tr><td>Thr → Ala* </td><td>5566 </td><td>4753 </td><td>813 </td><td>1.17 </td><td>ACN → GCN </td><td>○ </td></tr><tr><td>Ser → Pro* </td><td>3804 </td><td>3113 </td><td>691 </td><td>1.22 </td><td>UCN/AGY → CCN </td><td>○ </td></tr><tr><td>Asp → Glu* </td><td>4443 </td><td>3833 </td><td>610 </td><td>1.16 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>Ser → Thr* </td><td>3652 </td><td>3113 </td><td>539 </td><td>1.17 </td><td>UCN/AGY → ACN </td><td>□ </td></tr><tr><td>Ser → Ala* </td><td>3507 </td><td>2972 </td><td>535 </td><td>1.18 </td><td>UCN/AGY → GCN </td><td>○ </td></tr><tr><td>Ser → Asn* </td><td>3626 </td><td>3213 </td><td>413 </td><td>1.13 </td><td>UCN/AGY → AAY </td><td>□ </td></tr><tr><td>Arg → Lys** </td><td>4017 </td><td>3701 </td><td>316 </td><td>1.08 </td><td>CGN/AGR → AAR </td><td>□ </td></tr><tr><td>Leu → Pro* </td><td>1999 </td><td>1691 </td><td>308 </td><td>1.18 </td><td>UUR/CUN → CCN </td><td>○ </td></tr><tr><td>Gln → Arg* </td><td>2676 </td><td>2372 </td><td>304 </td><td>1.13 </td><td>CAR → CGN/AGR </td><td>○ </td></tr><tr><td>Ser → Gly </td><td>2782 </td><td>2560 </td><td>222 </td><td>1.09 </td><td>UCN/AGY → GGN </td><td>○ </td></tr><tr><td>Val → Ala </td><td>3759 </td><td>3561 </td><td>198 </td><td>1.06 </td><td>GUN → GCN </td><td>○ </td></tr><tr><td>Lys → Glu** </td><td>1514 </td><td>1336 </td><td>178 </td><td>1.13 </td><td>AAR → GAR </td><td>○ </td></tr><tr><td>Asp → Ala* </td><td>661 </td><td>488 </td><td>173 </td><td>1.35 </td><td>GSN → GCN </td><td> </td></tr><tr><td>Met → Leu </td><td>1648 </td><td>1483 </td><td>165 </td><td>1.11 </td><td>AUG → UUR/CUN </td><td>○ </td></tr><tr><td>Low GC group </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Val → Ile* </td><td>9402 </td><td>6335 </td><td>3067 </td><td>1.48 </td><td>GUN → AUH </td><td>± </td></tr><tr><td>Ser → Asn* </td><td>6508 </td><td>3733 </td><td>2775 </td><td>1.74 </td><td>UCN/AGY → AAY </td><td>± </td></tr><tr><td>Arg → Lys* </td><td>6603 </td><td>4059 </td><td>2544 </td><td>1.63 </td><td>CGN/AGR → AAR </td><td>± </td></tr><tr><td>Ala → Thr* </td><td>6393 </td><td>5117 </td><td>1276 </td><td>1.25 </td><td>GCN → ACN </td><td>± </td></tr><tr><td>Leu → Ile* </td><td>3003 </td><td>2035 </td><td>968 </td><td>1.48 </td><td>UUR/CUN → AUH </td><td>± </td></tr><tr><td>Asp → Glu* </td><td>5952 </td><td>5014 </td><td>938 </td><td>1.19 </td><td>GAY → GAR </td><td>• </td></tr><tr><td>Ser → Thr* </td><td>4498 </td><td>3563 </td><td>935 </td><td>1.26 </td><td>UCN/AGY → ACN </td><td> </td></tr><tr><td>Ala → Val* </td><td>4193 </td><td>3264 </td><td>929 </td><td>1.28 </td><td>GCN → GUN </td><td>± </td></tr><tr><td>Thr → Ile* </td><td>2612 </td><td>1733 </td><td>879 </td><td>1.51 </td><td>CAN → AUH </td><td>– </td></tr><tr><td>Pro → Ser* </td><td>4178 </td><td>3304 </td><td>874 </td><td>1.26 </td><td>CCN → UCN/AGY </td><td>± </td></tr><tr><td>Ala → Ser* </td><td>3975 </td><td>3195 </td><td>780 </td><td>1.24 </td><td>GCN → UCN/AGY </td><td>± </td></tr><tr><td>Leu → Phe* </td><td>2990 </td><td>2330 </td><td>660 </td><td>1.28 </td><td>UUR/CUN → UUY </td><td>– </td></tr><tr><td>Thr → Asn* </td><td>1630 </td><td>975 </td><td>655 </td><td>1.67 </td><td>CAN → AAY </td><td> </td></tr><tr><td>Met → Ile* </td><td>1900 </td><td>1288 </td><td>612 </td><td>1.48 </td><td>AUG → AUH </td><td>– </td></tr><tr><td>His → Asn* </td><td>1258 </td><td>715 </td><td>543 </td><td>1.76 </td><td>CAY → AAY </td><td>– </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*<sup>,</sup>**Replacements of pairs are significant at <em>p</em> &lt; 0.0001 and 0.001, respectively.</p></div></div><div content-id=""><div class="fn"><p>Symbols used in codons: R, A/G; Y, C/T; S, G/C; W, A/T; H, A/C/T; N, any nucleotide. Symbols used in trends: opposite trends among top 15 amino acid pairs of both high- and low-GC groups (±); opposite trends in high- and low-GC group, but not present in top 15 of low-GC group (+); opposite trends in high- and low-GC group, but not present in top 15 of high-GC group (–); same trend in high-, medium-, and low-GC groups (•); same trend in high- and medium-GC groups (○); same trend in low- and medium-GC groups (□).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194697" target="_blank">View Large</a></div></div><p>In the high-GC group, although amino acids of mouse proteins encoded by A/U-rich codons tend to be replaced by the amino acids encoded by G/C-rich codons in their human orthologs, not all amino acid residues encoded by A/U-rich codons exhibit equal bias in replacement patterns. There are six residues, <em>viz</em>. Phe, Tyr, Met, Ile, Asn, and Lys, which are encoded by A/U-rich codons and four residues, <em>viz.</em> Gly, Ala, Arg, and Pro, encoded by G/C-rich codons. As can be seen from Tables <a reveal-id="DSM015TB1" data-open="DSM015TB1" class="link link-reveal link-table xref-fig">1</a>, <a reveal-id="DSM015TB2" data-open="DSM015TB2" class="link link-reveal link-table xref-fig">2</a>, and <a reveal-id="DSM015TB3" data-open="DSM015TB3" class="link link-reveal link-table xref-fig">3</a>, among the amino acid residues encoded by A/U-rich codons, Ile, Asn, and Lys have a more number of significantly biased replacement ratios (AARM elements) both in the high-GC group and in the low-GC group. Replacement ratios of Phe and Tyr, though follow the general trend, are not statistically significant in most cases. Rather, some other residues like Ser, Thr, Val, Leu etc., which are not necessarily encoded by A/U codons, exhibit significant bias in the replacement ratios (Tables <a reveal-id="DSM015TB1" data-open="DSM015TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSM015TB3" data-open="DSM015TB3" class="link link-reveal link-table xref-fig">3</a>). Similarly, among Gly, Ala, Arg, and Pro, the former two have more number of significant <em>R<sub>ij</sub></em> values. Previous analysis of many prokaryotic genomes<sup><a reveal-id="DSM015C28" data-open="DSM015C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and high-GC rice genes with their <em>Arabidopsis</em> homologs<sup><a reveal-id="DSM015C29" data-open="DSM015C29" class="link link-ref link-reveal xref-bibr">29</a></sup> showed that proteins encoded by GC-rich sequences are characterized by increased levels of Gly, Ala, Arg, and Pro residues and a corresponding decrease in Phe, Tyr, Met, Ile, Asn, and Lys residues. It is, therefore, intriguing to examine to what extent the overall usage of the residues Gly/Ala/Arg/Pro and that of Phe/Tyr/Met/Ile/Asn/Lys vary within the mouse and human orthologs of high-, medium- and low-GC groups. Our analysis indicates that the mouse and human orthologs of three groups are indeed characterized by distinct usage profile of these residues (Supplementary Fig. S1). In the high-GC group, the human orthologs have higher usage of Gly, Ala, Arg, and Pro and lower usages of Phe, Tyr, Met, Ile, Asn, and Lys compared with their mouse orthologs, but the differences are not as pronounced as shown previously for homologous gene pairs from the rice and <em>Arabidopsis</em>, having large difference in GC content.<sup><a reveal-id="DSM015C29" data-open="DSM015C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In the low-GC group, the reverse is true, whereas in the medium-GC group, there is no significant difference between mouse and human orthologous pairs in the usage of these two groups of residues.</p>                            <h3 scrollto-destination=87194699 id="87194699" class="section-title" data-legacy-id=s3c>3.3. (Asp)<sub>Mouse</sub> → (Glu)<sub>Human</sub> trend in all groups of orthologs irrespective of their GC content</h3>
<p>There is only one replacement ratio <em>R</em><sub>DE</sub>, which exhibits same directionality and almost the same value in all three AARMs (Tables <a reveal-id="DSM015TB1" data-open="DSM015TB1" class="link link-reveal link-table xref-fig">1</a>, <a reveal-id="DSM015TB2" data-open="DSM015TB2" class="link link-reveal link-table xref-fig">2</a>, and <a reveal-id="DSM015TB3" data-open="DSM015TB3" class="link link-reveal link-table xref-fig">3</a>), indicating that in all groups of orthologs, the frequency of (Asp)<sub>Mouse</sub>→ (Glu)<sub>Human</sub> replacements is slightly higher than the replacement in the opposite direction. As the Asp → Glu replacement is among the top 15 trends in substitution in all three groups (Table <a reveal-id="DSM015TB4" data-open="DSM015TB4" class="link link-reveal link-table xref-fig">4</a>), it is one of the most common trends in amino acid replacement in mouse–human orthologs. These observations suggest that irrespective of the GC content of the encoding genes, there has been a consistent increase in glutamic acid in human proteins at the cost of aspartic acid compared with their mouse orthologs. The structural and/or functional implications of this unique evolutionary trend is, however, not clear. There are two other substitution trends, Ser → Thr and Phe → Tyr, which also exhibit same directionality in all three groups under study, but the replacement values are not statistically significant for the high-GC group.</p>                            <h3 scrollto-destination=87194701 id="87194701" class="section-title" data-legacy-id=s3d>3.4. High-GC orthologs are biased towards (A/T)<sub>Mouse</sub> → (G/C)<sub>Human</sub> replacements, whereas in low-GC orthologs, (G/C)<sub>Mouse</sub> → (A/T)<sub>Human</sub> replacements prevail</h3>
<p>As already emphasized, the major trends in amino acid replacements between mouse and human orthologs (Tables <a reveal-id="DSM015TB1  DSM015TB4" data-open="DSM015TB1  DSM015TB4" class="link link-reveal link-table xref-fig">1–4</a>) indicate that in the high-GC group, the amino acid residues encoded by relatively GC-rich codons tend to increase in human proteins compared with mouse orthologs, and in the low-GC group, the reverse trends prevail. In the medium-GC-group, however, there is no such specific directionality in codon substitution patterns. These observations have prompted us to examine the trends in nucleotide substitution patterns individually in three codon positions in three groups of orthologs. As can be seen from the NRMs shown in Table <a reveal-id="DSM015TB5" data-open="DSM015TB5" class="link link-reveal link-table xref-fig">5</a>, in the high-GC dataset, <em>r<sub>ij</sub></em>, is significantly greater than 1, when <em>i</em> = A or T and <em>j</em> = G or C. On the contrary, <em>r<sub>ij</sub></em> is significantly less than 1, when <em>i</em> = G or C and <em>j</em> = A or T. These trends are valid in all three codon positions, although the deviation of <em>r<sub>ij</sub></em> from 1 (for a particular set of <em>m</em> and <em>n</em>) is highest in third codon positions, followed by the first and second codon positions. Therefore, in the high-GC group, there has been an excess of (A/T)<sub>Mouse</sub> → (G/C)<sub>Human</sub> replacements over (G/C)<sub>Mouse</sub> → (A/T)<sub>Human</sub> at each codon position individually. For the low-GC group, the reverse situation has been encountered (Table <a reveal-id="DSM015TB5" data-open="DSM015TB5" class="link link-reveal link-table xref-fig">5</a>), i.e. there is a tendency for G and C in mouse genes to be replaced by A or T in their human orthologs, the bias being maximal at the third codon positions. For the medium-GC group, however, no significant difference between (A/T)<sub>Mouse</sub> → (G/C)<sub>Human</sub> and (G/C)<sub>Mouse</sub> → (A/T)<sub>Human</sub> replacements could be observed at the first and second codon sites, whereas for the third codon sites, the (A)<sub>Mouse</sub> → (G)<sub>Human</sub> and (T)<sub>Mouse</sub> → (C)<sub>Human</sub> replacements dominate over the reverse replacements. These observations imply that for the high-GC group, either the GC content tends to increase in human genes relative to mouse or tends to decrease in mouse genes relative to human, whereas for low-GC group, either there is a trend in relative decrease in GC content in human compared with mouse or there is a trend in relative increase in GC content in mouse compared with human. This suggests that with time, there is a relative increase in compositional heterogeneity within human genes compared with that within mouse genes or decrease in compositional heterogeneity within mouse genes compared with that within human genes.</p>                        <a id="87194703" scrollto-destination="87194703"></a>
<div content-id="DSM015TB5" class="table-modal"><div class="table-wrap"><div id="DSM015TB5" class="table-wrap-title" data-id="DSM015TB5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Nucleotide replacement matrices at (NRMs) three codon positions for human mouse orthologs of high-, medium-, and low-GC groups under study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="13">Human </th></tr><tr><th> </th><th>Group </th><th> </th><th colspan="4">First codon position </th><th colspan="4">Second codon position </th><th colspan="4">Third codon position </th></tr></thead><tbody><tr><td>Mouse </td><td> </td><td> </td><td>A </td><td>T </td><td>G </td><td>C </td><td>A </td><td>T </td><td>G </td><td>C </td><td>A </td><td>T </td><td>G </td><td>C </td></tr><tr><td> </td><td>High-GC group </td><td>A </td><td>― </td><td>1.00 </td><td><strong>1.13</strong> </td><td><strong>1.16</strong> </td><td>― </td><td>1.01 </td><td><strong>1.08</strong> </td><td><strong>1.06</strong> </td><td>― </td><td>1.00 </td><td><strong>1.86</strong> </td><td><strong>1.71</strong> </td></tr><tr><td> </td><td> </td><td>T </td><td>1.00 </td><td>― </td><td><strong>1.11</strong> </td><td><strong>1.19</strong> </td><td>0.99 </td><td>― </td><td><strong>1.06</strong> </td><td><strong>1.06</strong> </td><td>1.00 </td><td>― </td><td><strong>1.69</strong> </td><td><strong>1.86</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td><strong>0.88</strong> </td><td><strong>0.90</strong> </td><td>― </td><td><strong>1.01</strong> </td><td><strong>0.92</strong> </td><td><strong>0.95</strong> </td><td>― </td><td>0.98 </td><td><strong>0.54</strong> </td><td><strong>0.59</strong> </td><td>― </td><td><strong>1.01</strong> </td></tr><tr><td> </td><td> </td><td>C </td><td><strong>0.86</strong> </td><td><strong>0.84</strong> </td><td><strong>0.99</strong> </td><td>― </td><td><strong>0.94</strong> </td><td><strong>0.95</strong> </td><td>1.02 </td><td>― </td><td><strong>0.58</strong> </td><td><strong>0.54</strong> </td><td><strong>0.99</strong> </td><td>― </td></tr><tr><td> </td><td>Medium-GC group </td><td>A </td><td>― </td><td>1.01 </td><td>1.02 </td><td><strong><em>1.03</em></strong> </td><td>― </td><td>0.99 </td><td>1.01 </td><td>1.01 </td><td>― </td><td>1.00 </td><td><strong>1.04</strong> </td><td>1.01 </td></tr><tr><td> </td><td> </td><td>T </td><td>0.99 </td><td>― </td><td><strong><em>1.03</em></strong> </td><td>1.02 </td><td>1.01 </td><td>― </td><td>1.02 </td><td>1.02 </td><td>1.00 </td><td>― </td><td>1.02 </td><td><strong>1.03</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td>0.98 </td><td><strong><em>0.97</em></strong> </td><td>― </td><td>1.00 </td><td>0.99 </td><td>0.98 </td><td>― </td><td>1.00 </td><td><strong>0.97</strong> </td><td>0.98 </td><td>― </td><td>1.00 </td></tr><tr><td> </td><td> </td><td>C </td><td><strong><em>0.97</em></strong> </td><td>0.98 </td><td>1.00 </td><td>― </td><td>0.99 </td><td>0.98 </td><td>1.00 </td><td>― </td><td>0.99 </td><td><strong>0.97</strong> </td><td>1.00 </td><td>― </td></tr><tr><td> </td><td>Low-GC group </td><td>A </td><td>― </td><td>1.01 </td><td><strong>0.93</strong> </td><td><strong>0.90</strong> </td><td>― </td><td>0.99 </td><td><strong>0.90</strong> </td><td><strong>0.94</strong> </td><td>― </td><td>0.99 </td><td><strong>0.64</strong> </td><td><strong>0.65</strong> </td></tr><tr><td> </td><td> </td><td>T </td><td>0.99 </td><td>― </td><td><strong>0.93</strong> </td><td><strong>0.83</strong> </td><td>1.01 </td><td>― </td><td><strong>0.94</strong> </td><td><strong>0.95</strong> </td><td>1.01 </td><td>― </td><td><strong>0.70</strong> </td><td><strong>0.62</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td><strong>1.08</strong> </td><td><strong>1.07</strong> </td><td>― </td><td><strong>0.97</strong> </td><td><strong>1.11</strong> </td><td><strong>1.06</strong> </td><td>― </td><td>1.02 </td><td><strong>1.56</strong> </td><td><strong>1.42</strong> </td><td>― </td><td><strong>0.96</strong> </td></tr><tr><td> </td><td> </td><td>C </td><td><strong>1.11</strong> </td><td><strong>1.20</strong> </td><td><strong>1.03</strong> </td><td>― </td><td><strong>1.07</strong> </td><td><strong>1.06</strong> </td><td>0.98 </td><td>― </td><td><strong>1.53</strong> </td><td><strong>1.62</strong> </td><td><strong>1.04</strong> </td><td>― </td></tr></tbody></table></div><div content-id="DSM015TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="13">Human </th></tr><tr><th> </th><th>Group </th><th> </th><th colspan="4">First codon position </th><th colspan="4">Second codon position </th><th colspan="4">Third codon position </th></tr></thead><tbody><tr><td>Mouse </td><td> </td><td> </td><td>A </td><td>T </td><td>G </td><td>C </td><td>A </td><td>T </td><td>G </td><td>C </td><td>A </td><td>T </td><td>G </td><td>C </td></tr><tr><td> </td><td>High-GC group </td><td>A </td><td>― </td><td>1.00 </td><td><strong>1.13</strong> </td><td><strong>1.16</strong> </td><td>― </td><td>1.01 </td><td><strong>1.08</strong> </td><td><strong>1.06</strong> </td><td>― </td><td>1.00 </td><td><strong>1.86</strong> </td><td><strong>1.71</strong> </td></tr><tr><td> </td><td> </td><td>T </td><td>1.00 </td><td>― </td><td><strong>1.11</strong> </td><td><strong>1.19</strong> </td><td>0.99 </td><td>― </td><td><strong>1.06</strong> </td><td><strong>1.06</strong> </td><td>1.00 </td><td>― </td><td><strong>1.69</strong> </td><td><strong>1.86</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td><strong>0.88</strong> </td><td><strong>0.90</strong> </td><td>― </td><td><strong>1.01</strong> </td><td><strong>0.92</strong> </td><td><strong>0.95</strong> </td><td>― </td><td>0.98 </td><td><strong>0.54</strong> </td><td><strong>0.59</strong> </td><td>― </td><td><strong>1.01</strong> </td></tr><tr><td> </td><td> </td><td>C </td><td><strong>0.86</strong> </td><td><strong>0.84</strong> </td><td><strong>0.99</strong> </td><td>― </td><td><strong>0.94</strong> </td><td><strong>0.95</strong> </td><td>1.02 </td><td>― </td><td><strong>0.58</strong> </td><td><strong>0.54</strong> </td><td><strong>0.99</strong> </td><td>― </td></tr><tr><td> </td><td>Medium-GC group </td><td>A </td><td>― </td><td>1.01 </td><td>1.02 </td><td><strong><em>1.03</em></strong> </td><td>― </td><td>0.99 </td><td>1.01 </td><td>1.01 </td><td>― </td><td>1.00 </td><td><strong>1.04</strong> </td><td>1.01 </td></tr><tr><td> </td><td> </td><td>T </td><td>0.99 </td><td>― </td><td><strong><em>1.03</em></strong> </td><td>1.02 </td><td>1.01 </td><td>― </td><td>1.02 </td><td>1.02 </td><td>1.00 </td><td>― </td><td>1.02 </td><td><strong>1.03</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td>0.98 </td><td><strong><em>0.97</em></strong> </td><td>― </td><td>1.00 </td><td>0.99 </td><td>0.98 </td><td>― </td><td>1.00 </td><td><strong>0.97</strong> </td><td>0.98 </td><td>― </td><td>1.00 </td></tr><tr><td> </td><td> </td><td>C </td><td><strong><em>0.97</em></strong> </td><td>0.98 </td><td>1.00 </td><td>― </td><td>0.99 </td><td>0.98 </td><td>1.00 </td><td>― </td><td>0.99 </td><td><strong>0.97</strong> </td><td>1.00 </td><td>― </td></tr><tr><td> </td><td>Low-GC group </td><td>A </td><td>― </td><td>1.01 </td><td><strong>0.93</strong> </td><td><strong>0.90</strong> </td><td>― </td><td>0.99 </td><td><strong>0.90</strong> </td><td><strong>0.94</strong> </td><td>― </td><td>0.99 </td><td><strong>0.64</strong> </td><td><strong>0.65</strong> </td></tr><tr><td> </td><td> </td><td>T </td><td>0.99 </td><td>― </td><td><strong>0.93</strong> </td><td><strong>0.83</strong> </td><td>1.01 </td><td>― </td><td><strong>0.94</strong> </td><td><strong>0.95</strong> </td><td>1.01 </td><td>― </td><td><strong>0.70</strong> </td><td><strong>0.62</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td><strong>1.08</strong> </td><td><strong>1.07</strong> </td><td>― </td><td><strong>0.97</strong> </td><td><strong>1.11</strong> </td><td><strong>1.06</strong> </td><td>― </td><td>1.02 </td><td><strong>1.56</strong> </td><td><strong>1.42</strong> </td><td>― </td><td><strong>0.96</strong> </td></tr><tr><td> </td><td> </td><td>C </td><td><strong>1.11</strong> </td><td><strong>1.20</strong> </td><td><strong>1.03</strong> </td><td>― </td><td><strong>1.07</strong> </td><td><strong>1.06</strong> </td><td>0.98 </td><td>― </td><td><strong>1.53</strong> </td><td><strong>1.62</strong> </td><td><strong>1.04</strong> </td><td>― </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each element <em>r<sub>ij</sub></em> in the NRM represents the ratio of number of replacements of the nucleotide <em>i</em> by the nucleotide <em>j</em> in the forward direction (mouse to human) to that in the reverse direction. This means that if <em>r<sub>ij</sub></em> &gt; 1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements and if <em>r<sub>ij</sub></em> &lt; 1, the reverse is true. Bold and bold-italics ratios signifies the directional bias at <em>p</em> &lt; 0.0001 and 0.001, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194703" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM015TB5" class="table-wrap-title" data-id="DSM015TB5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Nucleotide replacement matrices at (NRMs) three codon positions for human mouse orthologs of high-, medium-, and low-GC groups under study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="13">Human </th></tr><tr><th> </th><th>Group </th><th> </th><th colspan="4">First codon position </th><th colspan="4">Second codon position </th><th colspan="4">Third codon position </th></tr></thead><tbody><tr><td>Mouse </td><td> </td><td> </td><td>A </td><td>T </td><td>G </td><td>C </td><td>A </td><td>T </td><td>G </td><td>C </td><td>A </td><td>T </td><td>G </td><td>C </td></tr><tr><td> </td><td>High-GC group </td><td>A </td><td>― </td><td>1.00 </td><td><strong>1.13</strong> </td><td><strong>1.16</strong> </td><td>― </td><td>1.01 </td><td><strong>1.08</strong> </td><td><strong>1.06</strong> </td><td>― </td><td>1.00 </td><td><strong>1.86</strong> </td><td><strong>1.71</strong> </td></tr><tr><td> </td><td> </td><td>T </td><td>1.00 </td><td>― </td><td><strong>1.11</strong> </td><td><strong>1.19</strong> </td><td>0.99 </td><td>― </td><td><strong>1.06</strong> </td><td><strong>1.06</strong> </td><td>1.00 </td><td>― </td><td><strong>1.69</strong> </td><td><strong>1.86</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td><strong>0.88</strong> </td><td><strong>0.90</strong> </td><td>― </td><td><strong>1.01</strong> </td><td><strong>0.92</strong> </td><td><strong>0.95</strong> </td><td>― </td><td>0.98 </td><td><strong>0.54</strong> </td><td><strong>0.59</strong> </td><td>― </td><td><strong>1.01</strong> </td></tr><tr><td> </td><td> </td><td>C </td><td><strong>0.86</strong> </td><td><strong>0.84</strong> </td><td><strong>0.99</strong> </td><td>― </td><td><strong>0.94</strong> </td><td><strong>0.95</strong> </td><td>1.02 </td><td>― </td><td><strong>0.58</strong> </td><td><strong>0.54</strong> </td><td><strong>0.99</strong> </td><td>― </td></tr><tr><td> </td><td>Medium-GC group </td><td>A </td><td>― </td><td>1.01 </td><td>1.02 </td><td><strong><em>1.03</em></strong> </td><td>― </td><td>0.99 </td><td>1.01 </td><td>1.01 </td><td>― </td><td>1.00 </td><td><strong>1.04</strong> </td><td>1.01 </td></tr><tr><td> </td><td> </td><td>T </td><td>0.99 </td><td>― </td><td><strong><em>1.03</em></strong> </td><td>1.02 </td><td>1.01 </td><td>― </td><td>1.02 </td><td>1.02 </td><td>1.00 </td><td>― </td><td>1.02 </td><td><strong>1.03</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td>0.98 </td><td><strong><em>0.97</em></strong> </td><td>― </td><td>1.00 </td><td>0.99 </td><td>0.98 </td><td>― </td><td>1.00 </td><td><strong>0.97</strong> </td><td>0.98 </td><td>― </td><td>1.00 </td></tr><tr><td> </td><td> </td><td>C </td><td><strong><em>0.97</em></strong> </td><td>0.98 </td><td>1.00 </td><td>― </td><td>0.99 </td><td>0.98 </td><td>1.00 </td><td>― </td><td>0.99 </td><td><strong>0.97</strong> </td><td>1.00 </td><td>― </td></tr><tr><td> </td><td>Low-GC group </td><td>A </td><td>― </td><td>1.01 </td><td><strong>0.93</strong> </td><td><strong>0.90</strong> </td><td>― </td><td>0.99 </td><td><strong>0.90</strong> </td><td><strong>0.94</strong> </td><td>― </td><td>0.99 </td><td><strong>0.64</strong> </td><td><strong>0.65</strong> </td></tr><tr><td> </td><td> </td><td>T </td><td>0.99 </td><td>― </td><td><strong>0.93</strong> </td><td><strong>0.83</strong> </td><td>1.01 </td><td>― </td><td><strong>0.94</strong> </td><td><strong>0.95</strong> </td><td>1.01 </td><td>― </td><td><strong>0.70</strong> </td><td><strong>0.62</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td><strong>1.08</strong> </td><td><strong>1.07</strong> </td><td>― </td><td><strong>0.97</strong> </td><td><strong>1.11</strong> </td><td><strong>1.06</strong> </td><td>― </td><td>1.02 </td><td><strong>1.56</strong> </td><td><strong>1.42</strong> </td><td>― </td><td><strong>0.96</strong> </td></tr><tr><td> </td><td> </td><td>C </td><td><strong>1.11</strong> </td><td><strong>1.20</strong> </td><td><strong>1.03</strong> </td><td>― </td><td><strong>1.07</strong> </td><td><strong>1.06</strong> </td><td>0.98 </td><td>― </td><td><strong>1.53</strong> </td><td><strong>1.62</strong> </td><td><strong>1.04</strong> </td><td>― </td></tr></tbody></table></div><div content-id="DSM015TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="13">Human </th></tr><tr><th> </th><th>Group </th><th> </th><th colspan="4">First codon position </th><th colspan="4">Second codon position </th><th colspan="4">Third codon position </th></tr></thead><tbody><tr><td>Mouse </td><td> </td><td> </td><td>A </td><td>T </td><td>G </td><td>C </td><td>A </td><td>T </td><td>G </td><td>C </td><td>A </td><td>T </td><td>G </td><td>C </td></tr><tr><td> </td><td>High-GC group </td><td>A </td><td>― </td><td>1.00 </td><td><strong>1.13</strong> </td><td><strong>1.16</strong> </td><td>― </td><td>1.01 </td><td><strong>1.08</strong> </td><td><strong>1.06</strong> </td><td>― </td><td>1.00 </td><td><strong>1.86</strong> </td><td><strong>1.71</strong> </td></tr><tr><td> </td><td> </td><td>T </td><td>1.00 </td><td>― </td><td><strong>1.11</strong> </td><td><strong>1.19</strong> </td><td>0.99 </td><td>― </td><td><strong>1.06</strong> </td><td><strong>1.06</strong> </td><td>1.00 </td><td>― </td><td><strong>1.69</strong> </td><td><strong>1.86</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td><strong>0.88</strong> </td><td><strong>0.90</strong> </td><td>― </td><td><strong>1.01</strong> </td><td><strong>0.92</strong> </td><td><strong>0.95</strong> </td><td>― </td><td>0.98 </td><td><strong>0.54</strong> </td><td><strong>0.59</strong> </td><td>― </td><td><strong>1.01</strong> </td></tr><tr><td> </td><td> </td><td>C </td><td><strong>0.86</strong> </td><td><strong>0.84</strong> </td><td><strong>0.99</strong> </td><td>― </td><td><strong>0.94</strong> </td><td><strong>0.95</strong> </td><td>1.02 </td><td>― </td><td><strong>0.58</strong> </td><td><strong>0.54</strong> </td><td><strong>0.99</strong> </td><td>― </td></tr><tr><td> </td><td>Medium-GC group </td><td>A </td><td>― </td><td>1.01 </td><td>1.02 </td><td><strong><em>1.03</em></strong> </td><td>― </td><td>0.99 </td><td>1.01 </td><td>1.01 </td><td>― </td><td>1.00 </td><td><strong>1.04</strong> </td><td>1.01 </td></tr><tr><td> </td><td> </td><td>T </td><td>0.99 </td><td>― </td><td><strong><em>1.03</em></strong> </td><td>1.02 </td><td>1.01 </td><td>― </td><td>1.02 </td><td>1.02 </td><td>1.00 </td><td>― </td><td>1.02 </td><td><strong>1.03</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td>0.98 </td><td><strong><em>0.97</em></strong> </td><td>― </td><td>1.00 </td><td>0.99 </td><td>0.98 </td><td>― </td><td>1.00 </td><td><strong>0.97</strong> </td><td>0.98 </td><td>― </td><td>1.00 </td></tr><tr><td> </td><td> </td><td>C </td><td><strong><em>0.97</em></strong> </td><td>0.98 </td><td>1.00 </td><td>― </td><td>0.99 </td><td>0.98 </td><td>1.00 </td><td>― </td><td>0.99 </td><td><strong>0.97</strong> </td><td>1.00 </td><td>― </td></tr><tr><td> </td><td>Low-GC group </td><td>A </td><td>― </td><td>1.01 </td><td><strong>0.93</strong> </td><td><strong>0.90</strong> </td><td>― </td><td>0.99 </td><td><strong>0.90</strong> </td><td><strong>0.94</strong> </td><td>― </td><td>0.99 </td><td><strong>0.64</strong> </td><td><strong>0.65</strong> </td></tr><tr><td> </td><td> </td><td>T </td><td>0.99 </td><td>― </td><td><strong>0.93</strong> </td><td><strong>0.83</strong> </td><td>1.01 </td><td>― </td><td><strong>0.94</strong> </td><td><strong>0.95</strong> </td><td>1.01 </td><td>― </td><td><strong>0.70</strong> </td><td><strong>0.62</strong> </td></tr><tr><td> </td><td> </td><td>G </td><td><strong>1.08</strong> </td><td><strong>1.07</strong> </td><td>― </td><td><strong>0.97</strong> </td><td><strong>1.11</strong> </td><td><strong>1.06</strong> </td><td>― </td><td>1.02 </td><td><strong>1.56</strong> </td><td><strong>1.42</strong> </td><td>― </td><td><strong>0.96</strong> </td></tr><tr><td> </td><td> </td><td>C </td><td><strong>1.11</strong> </td><td><strong>1.20</strong> </td><td><strong>1.03</strong> </td><td>― </td><td><strong>1.07</strong> </td><td><strong>1.06</strong> </td><td>0.98 </td><td>― </td><td><strong>1.53</strong> </td><td><strong>1.62</strong> </td><td><strong>1.04</strong> </td><td>― </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each element <em>r<sub>ij</sub></em> in the NRM represents the ratio of number of replacements of the nucleotide <em>i</em> by the nucleotide <em>j</em> in the forward direction (mouse to human) to that in the reverse direction. This means that if <em>r<sub>ij</sub></em> &gt; 1, the number of (<em>i</em>)<sub>Mouse</sub> → (<em>j</em>)<sub>Human</sub> replacements is higher than the number of (<em>j</em>)<sub>Mouse</sub> → (<em>i</em>)<sub>Human</sub> replacements and if <em>r<sub>ij</sub></em> &lt; 1, the reverse is true. Bold and bold-italics ratios signifies the directional bias at <em>p</em> &lt; 0.0001 and 0.001, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194703" target="_blank">View Large</a></div></div><p>Our next task was to see to what extent the observed trends in nucleotide substitution patterns have affected the relative GC divergence between mouse and human orthologs. To this end, the number of genes was plotted against their GC<sub>12</sub> and GC<sub>3</sub> values both for mouse and for human in all three groups of orthologs (Fig. <a reveal-id="DSM015F2" data-open="DSM015F2" class="link link-reveal link-table xref-fig">2</a>) using STATISTICA (version 6.0). In all cases, normal distributions were obtained (Fig. <a reveal-id="DSM015F2" data-open="DSM015F2" class="link link-reveal link-table xref-fig">2</a>). In high-GC group, both GC<sub>12</sub> and GC<sub>3</sub> distributions in human are skewed towards right (increasing GC contents) compared with mouse (Fig. <a reveal-id="DSM015F2" data-open="DSM015F2" class="link link-reveal link-table xref-fig">2</a>A and B), but for low-GC group, the reverse is true (Fig. <a reveal-id="DSM015F2" data-open="DSM015F2" class="link link-reveal link-table xref-fig">2</a>E and F). The extent of inter-species divergence in GC distribution is much more apparent in case of third codon positions (Fig. <a reveal-id="DSM015F2" data-open="DSM015F2" class="link link-reveal link-table xref-fig">2</a>B, D, and F) compared with the first and second positions (Fig. <a reveal-id="DSM015F2" data-open="DSM015F2" class="link link-reveal link-table xref-fig">2</a>A, C, and E). For medium-GC orthologs, medians of both GC<sub>12</sub> and GC<sub>3</sub> distributions are almost same in both species under study (Fig. <a reveal-id="DSM015F2" data-open="DSM015F2" class="link link-reveal link-table xref-fig">2</a>C and D). These observations imply that the intra-species divergence in base composition is higher in case of human genes than that in their mouse orthologs such that among the GC-rich pairs of orthologs, human coding sequences are usually higher in GC content than their mouse counterparts, but among the GC-poor orthologous pairs, human coding sequences are, in general, lower in GC content than the respective mouse sequences.</p>                        <a id="87194705" scrollto-destination="87194705"></a>
<div id="DSM015F2" data-id="DSM015F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/m_dsm01502.jpeg?Expires=2147483647&Signature=QomtmEckgh5LURJIsZTSrnl-AJsydUcTku4hMWvD4Nro65cpDMzJfKjsYsM7fzmhT~JYcbTTvCNevlQ5v5waK4BTR8u0lOVXB4OoUGZJ1Ou0K~iMeffRnGfhLVPdK1BdqEKJ5BG7NL27t-TsiIZyhMrQOFAM~TJJVfcAFHLi1DxuIVaaVbYa7qxJ1pDBawl7QqvCUY7XsVAns9vfzwYwB7uz-RJFMAiO5dB3FCchZMVRAEUD6rX9tQQoGuTwK3l8PlxcKy2tLXGucCDWrq-TK7KVNUa7g1pXtC~mNGvZ8PwVYQgycXnOGxIpRiq3~-vymnLBzc7D6zqztUSUoBwqFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Left panel: distribution of GC content at first and second codon positions among human (blue) and mouse (red) orthologous genes for (A) high-, (C) medium-, and (E) low-GC group with their normal distributions. Right panel: distribution of GC content at third codon position among human and mouse orthologous genes for (B) high-, (D) medium-, and (F) low-GC group with their normal distributions." path-from-xml="dsm01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194705/dsm01502.jpeg" path-from-xml="dsm01502.jpeg" target="_blank">View large</a><a section="87194705" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/dsm01502.jpeg?Expires=2147483647&Signature=j6T5bNbCpQGPqfsaGTfNFMemO5OqxIR60GiJ0Lm8YxvFB1H7U7mVqNYUX2X903DkobxCz~z3MTiOac1pLsIo2j6qfOmgUjt9Y9aOTnQy6r2~lqtKheNL2PmEMloIyj0zConCkKrrnHlwDMN-ea2g9AUOYIEFaPaxtvTLo1uqxyTP1NZIRL5MQ5T~YDQUmRQ0mXsvQ08bIw~IyrY-V-u6qasi0JP~qMYqLaIPqtW83bAbyVdzMQCyeEPa0avT~mbGDlhpmBDubTTaMTZyjDh9rfSqFU7b~8HkGlhiTsiNeWZNZ~0jd6haYP9iNi6W-1DIbiuoe7Pi6uAMMWY0CsK3lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194705&ar=466067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Left panel: distribution of GC content at first and second codon positions among human (blue) and mouse (red) orthologous genes for (<strong>A</strong>) high-, (<strong>C</strong>) medium-, and (<strong>E</strong>) low-GC group with their normal distributions. Right panel: distribution of GC content at third codon position among human and mouse orthologous genes for (<strong>B</strong>) high-, (<strong>D</strong>) medium-, and (<strong>F</strong>) low-GC group with their normal distributions.</p></div></div><div content-id="DSM015F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/m_dsm01502.jpeg?Expires=2147483647&Signature=QomtmEckgh5LURJIsZTSrnl-AJsydUcTku4hMWvD4Nro65cpDMzJfKjsYsM7fzmhT~JYcbTTvCNevlQ5v5waK4BTR8u0lOVXB4OoUGZJ1Ou0K~iMeffRnGfhLVPdK1BdqEKJ5BG7NL27t-TsiIZyhMrQOFAM~TJJVfcAFHLi1DxuIVaaVbYa7qxJ1pDBawl7QqvCUY7XsVAns9vfzwYwB7uz-RJFMAiO5dB3FCchZMVRAEUD6rX9tQQoGuTwK3l8PlxcKy2tLXGucCDWrq-TK7KVNUa7g1pXtC~mNGvZ8PwVYQgycXnOGxIpRiq3~-vymnLBzc7D6zqztUSUoBwqFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Left panel: distribution of GC content at first and second codon positions among human (blue) and mouse (red) orthologous genes for (A) high-, (C) medium-, and (E) low-GC group with their normal distributions. Right panel: distribution of GC content at third codon position among human and mouse orthologous genes for (B) high-, (D) medium-, and (F) low-GC group with their normal distributions." path-from-xml="dsm01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194705/dsm01502.jpeg" path-from-xml="dsm01502.jpeg" target="_blank">View large</a><a section="87194705" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/dsm01502.jpeg?Expires=2147483647&Signature=j6T5bNbCpQGPqfsaGTfNFMemO5OqxIR60GiJ0Lm8YxvFB1H7U7mVqNYUX2X903DkobxCz~z3MTiOac1pLsIo2j6qfOmgUjt9Y9aOTnQy6r2~lqtKheNL2PmEMloIyj0zConCkKrrnHlwDMN-ea2g9AUOYIEFaPaxtvTLo1uqxyTP1NZIRL5MQ5T~YDQUmRQ0mXsvQ08bIw~IyrY-V-u6qasi0JP~qMYqLaIPqtW83bAbyVdzMQCyeEPa0avT~mbGDlhpmBDubTTaMTZyjDh9rfSqFU7b~8HkGlhiTsiNeWZNZ~0jd6haYP9iNi6W-1DIbiuoe7Pi6uAMMWY0CsK3lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194705&ar=466067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Left panel: distribution of GC content at first and second codon positions among human (blue) and mouse (red) orthologous genes for (<strong>A</strong>) high-, (<strong>C</strong>) medium-, and (<strong>E</strong>) low-GC group with their normal distributions. Right panel: distribution of GC content at third codon position among human and mouse orthologous genes for (<strong>B</strong>) high-, (<strong>D</strong>) medium-, and (<strong>F</strong>) low-GC group with their normal distributions.</p></div></div>                            <h3 scrollto-destination=87194706 id="87194706" class="section-title" data-legacy-id=s3e>3.5. Multivariate analysis of synonymous codon usage confirms opposite trends in high-GC and low-GC groups of orthologs</h3>
<p>The skewness of GC<sub>3</sub> in human genes towards increasing GC in high-GC group and decreasing GC in low-GC group compared with mouse orthologs (Fig. <a reveal-id="DSM015F2" data-open="DSM015F2" class="link link-reveal link-table xref-fig">2</a>B, D, and F) has also been reflected in the COA of RSCU. Fig. <a reveal-id="DSM015F3" data-open="DSM015F3" class="link link-reveal link-table xref-fig">3</a>A–C represents axis-1 versus axis-2 plot of the COA on RSCU of genes in three different groups. In all cases, axis-1 exhibits strong negative correlation with GC content at synonymous substitution sites (GC<sub>3S</sub>). The distribution of human and mouse genes along axis-1 confirms that in high-GC group (Fig. <a reveal-id="DSM015F3" data-open="DSM015F3" class="link link-reveal link-table xref-fig">3</a>A), human genes exhibit higher usage of G or C ending synonymous codons compared with their mouse orthologs, whereas for low-GC group, the reverse trend dominates (Fig. <a reveal-id="DSM015F3" data-open="DSM015F3" class="link link-reveal link-table xref-fig">3</a>C). For medium-GC group, as expected, usage of G/C-ending codons is comparable in mouse and human (Fig. <a reveal-id="DSM015F3" data-open="DSM015F3" class="link link-reveal link-table xref-fig">3</a>B).</p>                        <a id="87194708" scrollto-destination="87194708"></a>
<div id="DSM015F3" data-id="DSM015F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/m_dsm01503.jpeg?Expires=2147483647&Signature=u1PiKa5XYwJMKfmXion~QFzbIlqDqddWOGSCDucIxr~-1j~89caShrmw9W5uLen-ub~guENGB4TsqhUI03fmSXhOLW38LM7JDLakRVtQI~GD1HAxF3WTirg-~12ikjYXhvhWYmii5BCi~OwKBp~rFNh90FkR-HYtbnbS15WyPSOhkPilf34M6~FURpdk2xJWIPiXyDJgYEGpStfxSiFacp74fOI~OhTYni~pBTuYxDkZOYd9ro5S9nXH4ypqPaIC3aBvSPdcPCf1nXU3UTALX2NoG-hstT-iRpX9qBj~AUWhU0iYRgKC8rI5vh8qzcRhtbxJsQ7cPvS3lZ0hLnn3rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positions of orthologous gene pairs between human and mouse along the first two identical principal axes generated by COA on RSCU values of (A) 3896 genes from high-GC group, (B) 3960 genes from medium-GC group, and (C) 4168 genes from low-GC group. The filled quadrangle and open quadrangle represent genes from mouse and human orthologous genes, respectively." path-from-xml="dsm01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194708/dsm01503.jpeg" path-from-xml="dsm01503.jpeg" target="_blank">View large</a><a section="87194708" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/dsm01503.jpeg?Expires=2147483647&Signature=xbdIkNKFTmHH-6qmYamRNGpOhDlWX8T7OEZW1Dn44ciU0NTva1fFmVUfr-sGs-WdY5DBAKyiDOLvIcme2YD7HAOMmEoN2YNsmIYz8su1fVyyIbMKWFMN0uU8MXwQlq0ImtAvRdMYwV~tH-nijNAwWxWIbUma3GiypGZi5xpFzTPcE6YZ-Xw2TerZzNLIN28Hqkw9amI6YTWBpR7obQiUiEQeDzzM7k4D7B1l~UjIPD8Lop4YZicP~jLKbG-3tFlveAF3t014FHuKfo7UjAaZIuRAamoFcPJkPRMqtllrUg2rDUWaKPGNn3-OJVTQXzdT3V1QFTbn-GoRQfdYU0HctA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194708&ar=466067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positions of orthologous gene pairs between human and mouse along the first two identical principal axes generated by COA on RSCU values of (<strong>A</strong>) 3896 genes from high-GC group, (<strong>B</strong>) 3960 genes from medium-GC group, and (<strong>C</strong>) 4168 genes from low-GC group. The filled quadrangle and open quadrangle represent genes from mouse and human orthologous genes, respectively.</p></div></div><div content-id="DSM015F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/m_dsm01503.jpeg?Expires=2147483647&Signature=u1PiKa5XYwJMKfmXion~QFzbIlqDqddWOGSCDucIxr~-1j~89caShrmw9W5uLen-ub~guENGB4TsqhUI03fmSXhOLW38LM7JDLakRVtQI~GD1HAxF3WTirg-~12ikjYXhvhWYmii5BCi~OwKBp~rFNh90FkR-HYtbnbS15WyPSOhkPilf34M6~FURpdk2xJWIPiXyDJgYEGpStfxSiFacp74fOI~OhTYni~pBTuYxDkZOYd9ro5S9nXH4ypqPaIC3aBvSPdcPCf1nXU3UTALX2NoG-hstT-iRpX9qBj~AUWhU0iYRgKC8rI5vh8qzcRhtbxJsQ7cPvS3lZ0hLnn3rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positions of orthologous gene pairs between human and mouse along the first two identical principal axes generated by COA on RSCU values of (A) 3896 genes from high-GC group, (B) 3960 genes from medium-GC group, and (C) 4168 genes from low-GC group. The filled quadrangle and open quadrangle represent genes from mouse and human orthologous genes, respectively." path-from-xml="dsm01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194708/dsm01503.jpeg" path-from-xml="dsm01503.jpeg" target="_blank">View large</a><a section="87194708" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/dsm01503.jpeg?Expires=2147483647&Signature=xbdIkNKFTmHH-6qmYamRNGpOhDlWX8T7OEZW1Dn44ciU0NTva1fFmVUfr-sGs-WdY5DBAKyiDOLvIcme2YD7HAOMmEoN2YNsmIYz8su1fVyyIbMKWFMN0uU8MXwQlq0ImtAvRdMYwV~tH-nijNAwWxWIbUma3GiypGZi5xpFzTPcE6YZ-Xw2TerZzNLIN28Hqkw9amI6YTWBpR7obQiUiEQeDzzM7k4D7B1l~UjIPD8Lop4YZicP~jLKbG-3tFlveAF3t014FHuKfo7UjAaZIuRAamoFcPJkPRMqtllrUg2rDUWaKPGNn3-OJVTQXzdT3V1QFTbn-GoRQfdYU0HctA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194708&ar=466067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positions of orthologous gene pairs between human and mouse along the first two identical principal axes generated by COA on RSCU values of (<strong>A</strong>) 3896 genes from high-GC group, (<strong>B</strong>) 3960 genes from medium-GC group, and (<strong>C</strong>) 4168 genes from low-GC group. The filled quadrangle and open quadrangle represent genes from mouse and human orthologous genes, respectively.</p></div></div><p>It is worth mentioning that the GC contents of the synonymous substitution sites in the mouse and human orthologous pairs exhibit negative correlations in all three groups (supplement-I). These observations are in accordance with the previous report by Takahashi and Nakashima.<sup><a reveal-id="DSM015C30" data-open="DSM015C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p>                            <h3 scrollto-destination=87194710 id="87194710" class="section-title" data-legacy-id=s3f>3.6. Rate of synonymous and nonsynonymous substitutions are same in all three groups of orthologs</h3>
<p>In order to examine whether the rate of nucleotide substitution between mouse and human orthologs varies in three different groups, the number of synonymous substitutions per synonymous site, <em>d</em><sub>S</sub>, and the number of nonsynonymous substitutions per nonsynonymous site, <em>d</em><sub>N</sub>, were estimated for randomly selected 500 pairs of the orthologs from each group. The value of <em>d</em><sub>S</sub> remains almost same in all three groups (data not shown). The value of <em>d</em><sub>N</sub> apparently seems to be lower in the medium-GC group, but its difference with its values for the other two groups is not statistically significant. These observations indicate that although the trends in nucleotide substitutions are polarized to the opposite directions in the high- and low-GC groups of orthologs, the rates of synonymous or nonsynonymous substitutions did not vary with the GC bias of the genes.</p>                            <h2 scrollto-destination=87194712 id="87194712" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Since the divergence of the rodent and primate lineages, multiple substitutions might have occurred at the same site of a pair of mouse–human orthologs independently in two lineages. Had there not been a strong directionality of selection process(es) prevailing over the random mutational events, such multiple hits should have obscured the true pattern of substitution, if any, between such orthologous pairs. However, the present study has revealed that the nucleotide and amino acid substitution patterns in mouse–human orthologs have followed definite trends that are highly asymmetric and polarized to opposite directions in high- and low-GC groups, suggesting that indeed there has been a definite directionality in gene/protein evolution towards increasing compositional divergence in human protein-coding regions compared with that in mouse protein-coding regions or towards decreasing compositional divergence in mouse protein-coding regions compared with that in human protein-coding regions. It is true that the GC content shows evolutionary stability between mouse and human, i.e. orthologs have similar GC contents in two species, but among the high-GC orthologs, human proteins are slightly higher in GC content than their mouse orthologs, whereas among the low-GC orthologs, human proteins are slightly higher in AT content than their mouse counterparts.</p><p>A question may be raised at this point: why, of all mammalian species, only mouse and human were chosen as the species of study in the present report. The reason is as follows: initially we intended to analyze the sequence divergence patterns between the orthologous coding regions of human, chimpanzee, and rhesus monkey. However, the numbers of nonsynonymous substitutions between two orthologs of any two primate species were often too low to reveal any significant statistical trend. Therefore, we have decided to analyze the trends in substitution patterns between a rodent and a primate species, mouse and human have been chosen as the representative species of the two lineages.</p><p>As already mentioned in Section 2, the trends reported here are robust enough to be valid for any subset of the total datasets of orthologous sequences. Any trend in amino acid/nucleotide replacement between the pairs of orthologs of a particular dataset remains invariant, in general, when a subset of sequences are chosen randomly from that particular dataset. This indicates that same trends are usually followed individually by each pair of orthologs in a particular group (high-, medium-, or low-GC group).</p><p>The trends in amino acid and nucleotide replacement patterns also remained same when the orthologous sequences were classified in high-, medium-, and low-GC groups on the basis of the GC<sub>3</sub> content of mouse genes instead of human genes. The same previous directionality was observed for high- or low-GC groups, i.e. GC content either increase in human genes relative to mouse or decrease in mouse genes relative to human for the high-GC group, whereas for low-GC group, either there is relative decrease in GC content in human genes compared with mouse gene or relative increase in GC content in mouse genes compared with human gene. This was, however, expected as the two genome sequences exhibit a one-to-one correspondence in their local GC content.</p><p>The only significant trend common in all three groups of orthologs is (Asp)<sub>Mouse</sub> → (Glu)<sub>Human</sub>. Surprisingly, the value of <em>R</em><sub>DE</sub> is almost same in all three groups and the trend has also been exhibited by the subsets chosen randomly from the whole dataset of any particular compositional group. This indicates that this trend, in general, does not alter with the compositional bias or functional characteristics of the genes. In accordance with this, average frequency of Glu (7.01% for mouse and 7.11% for human) is significantly higher in human (<em>p</em> &lt; 10<sup>−5</sup>) and that of Asp (4.90% for mouse and 4.81% for human) is significantly higher in mouse (<em>p</em> &lt; 10<sup>−5</sup>). The structural consequence of this trend is, however, not clear.</p><p>No significant differences could be observed between the synonymous or nonsynonymous substitution rates in three groups of orthologs under study. This suggests that although the directionality of evolution in orthologs of two extreme GC compositions is oppositely polarized, the rate at which they evolve is almost same in both cases.</p><p>In a nutshell, the present study indicates that in comparison with mouse, the coding regions of the human genome have experienced an expansion, not shrinkage, in intra-species heterogeneity in local GC content. This observation, however, does not warrant the relative expansion of the human GC islands as a whole, since it would depend not only on the evolutionary trends of the coding region, but also on those of the noncoding regions. One should also remember that a relative increase in GC heterogeneity in human orthologs compared with mouse orthologs not necessarily implies an absolute increase in GC heterogeneity in human coding regions with evolution. In absolute sense, both human and mouse might have evolved towards decreasing compositional heterogeneity, the rate of decrease in heterogeneity being less in human than in mouse, or alternatively, both the species might be evolving towards increasing intra-species inhomogeneity, the rate of increase being higher in human relative to mouse.</p>                            <h2 scrollto-destination=87194720 id="87194720" class="section-title" data-legacy-id=s5>Funding</h2>
<p>Council of Scientific and Industrial Research (Project No. CMM 0017 to C.D and S.G); Department of Biotechnology, Government of India (BT/BI/04/055-2001 to S.K.B and S.P).</p>                            <h2 scrollto-destination=87194722 id="87194722" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Dr. A. Pan, Indian Association for the Cultivation of Science, Kolkata, India, for critical reading of the manuscript.</p>                            <h2 scrollto-destination=87194724 id="87194724" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM015C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernardi</div> <div class="given-names">G</div></div>. </div><div class="article-title">Isochores and the evolutionary genomics of vertebrates</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">241</div> (pg. <div class="fpage">3</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isochores%20and%20the%20evolutionary%20genomics%20of%20vertebrates&amp;author=G%20Bernardi&amp;publication_year=2000&amp;journal=Gene&amp;volume=241&amp;pages=3-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00485-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00485-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00485-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10607893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A</div></div>,  <div class="name"><div class="surname">Hurst</div> <div class="given-names">L. D</div></div>. </div><div class="article-title">The evolution of isochores</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">549</div>-<div class="lpage">555</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20isochores&amp;author=A%20Eyre-Walker&amp;author=L.%20D%20Hurst&amp;publication_year=2001&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=2&amp;pages=549-555" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35080577" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35080577" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35080577"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11433361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Filipski</div> <div class="given-names">J</div></div>,  <div class="name"><div class="surname">Thiery</div> <div class="given-names">J. P</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G</div></div>. </div><div class="article-title">An analysis of the bovine genome by Cs2SO4-Ag density gradient centrifugation</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1973</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">177</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20analysis%20of%20the%20bovine%20genome%20by%20Cs2SO4-Ag%20density%20gradient%20centrifugation&amp;author=J%20Filipski&amp;author=J.%20P%20Thiery&amp;author=G%20Bernardi&amp;publication_year=1973&amp;journal=J.%20Mol.%20Biol.&amp;volume=80&amp;pages=177-197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(73)90240-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(73)90240-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(73)90240-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4798988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hughes</div> <div class="given-names">S</div></div>,  <div class="name"><div class="surname">Zelus</div> <div class="given-names">D</div></div>,  <div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D</div></div>. </div><div class="article-title">Warm-blooded isochore structure in Nile crocodile and turtle</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1521</div>-<div class="lpage">1527</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Warm-blooded%20isochore%20structure%20in%20Nile%20crocodile%20and%20turtle&amp;author=S%20Hughes&amp;author=D%20Zelus&amp;author=D%20Mouchiroud&amp;publication_year=1999&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=16&amp;pages=1521-1527" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026064" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026064" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026064"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10555283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E. S</div></div>,  <div class="name"><div class="surname">Linton</div> <div class="given-names">L. M</div></div>,  <div class="name"><div class="surname">Birren</div> <div class="given-names">B</div></div>, et al. </div><div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">860</div>-<div class="lpage">921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.%20S%20Lander&amp;author=L.%20M%20Linton&amp;author=B%20Birren&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Caron</div> <div class="given-names">H</div></div>,  <div class="name"><div class="surname">van Schaik</div> <div class="given-names">B</div></div>,  <div class="name"><div class="surname">van der Mee</div> <div class="given-names">M</div></div>, et al. </div><div class="article-title">The human transcriptome map: clustering of highly expressed genes in chromosomal domains</div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">291</div> (pg. <div class="fpage">1289</div>-<div class="lpage">1292</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20transcriptome%20map%3A%20clustering%20of%20highly%20expressed%20genes%20in%20chromosomal%20domains&amp;author=H%20Caron&amp;author=B%20van%20Schaik&amp;author=M%20van%20der%20Mee&amp;publication_year=2001&amp;journal=Science&amp;volume=291&amp;pages=1289-1292" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1056794" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1056794" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1056794"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11181992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fullerton</div> <div class="given-names">S. M</div></div>,  <div class="name"><div class="surname">Bernardo Carvalho</div> <div class="given-names">A</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">A. G</div></div>. </div><div class="article-title">Local rates of recombination are positively correlated with GC content in the human genome</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1139</div>-<div class="lpage">1142</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Local%20rates%20of%20recombination%20are%20positively%20correlated%20with%20GC%20content%20in%20the%20human%20genome&amp;author=S.%20M%20Fullerton&amp;author=A%20Bernardo%20Carvalho&amp;author=A.%20G%20Clark&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=18&amp;pages=1139-1142" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003886" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003886" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003886"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11371603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kong</div> <div class="given-names">A</div></div>,  <div class="name"><div class="surname">Gudbjartsson</div> <div class="given-names">D. F</div></div>,  <div class="name"><div class="surname">Sainz</div> <div class="given-names">J</div></div>, et al. </div><div class="article-title">A high-resolution recombination map of the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">241</div>-<div class="lpage">247</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%20recombination%20map%20of%20the%20human%20genome&amp;author=A%20Kong&amp;author=D.%20F%20Gudbjartsson&amp;author=J%20Sainz&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=31&amp;pages=241-247" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12053178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lercher</div> <div class="given-names">M. J</div></div>,  <div class="name"><div class="surname">Urrutia</div> <div class="given-names">A. O</div></div>,  <div class="name"><div class="surname">Pavlicek</div> <div class="given-names">A</div></div>,  <div class="name"><div class="surname">Hurst</div> <div class="given-names">L. D</div></div>. </div><div class="article-title">A unification of mosaic structures in the human genome</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">2411</div>-<div class="lpage">2415</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unification%20of%20mosaic%20structures%20in%20the%20human%20genome&amp;author=M.%20J%20Lercher&amp;author=A.%20O%20Urrutia&amp;author=A%20Pavlicek&amp;author=L.%20D%20Hurst&amp;publication_year=2003&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=12&amp;pages=2411-2415" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddg251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddg251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddg251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12915446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D</div></div>,  <div class="name"><div class="surname">D'Onofrio</div> <div class="given-names">G</div></div>,  <div class="name"><div class="surname">Aissani</div> <div class="given-names">B</div></div>,  <div class="name"><div class="surname">Macaya</div> <div class="given-names">G</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G</div></div>. </div><div class="article-title">The distribution of genes in the human genome</div>, <div class="source">Gene</div> , <div class="year">1991</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">181</div>-<div class="lpage">187</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20distribution%20of%20genes%20in%20the%20human%20genome&amp;author=D%20Mouchiroud&amp;author=G%20D'Onofrio&amp;author=B%20Aissani&amp;author=G%20Macaya&amp;author=C%20Gautier&amp;author=G%20Bernardi&amp;publication_year=1991&amp;journal=Gene&amp;volume=100&amp;pages=181-187" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(91)90364-H" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(91)90364-H" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(91)90364-H"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2055469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saccone</div> <div class="given-names">S</div></div>,  <div class="name"><div class="surname">De Sario</div> <div class="given-names">A</div></div>,  <div class="name"><div class="surname">Wiegant</div> <div class="given-names">J</div></div>,  <div class="name"><div class="surname">Raap</div> <div class="given-names">A. K</div></div>,  <div class="name"><div class="surname">Della Valle</div> <div class="given-names">G</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G</div></div>. </div><div class="article-title">Correlations between isochores and chromosomal bands in the human genome</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1993</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">11929</div>-<div class="lpage">11933</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlations%20between%20isochores%20and%20chromosomal%20bands%20in%20the%20human%20genome&amp;author=S%20Saccone&amp;author=A%20De%20Sario&amp;author=J%20Wiegant&amp;author=A.%20K%20Raap&amp;author=G%20Della%20Valle&amp;author=G%20Bernardi&amp;publication_year=1993&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=90&amp;pages=11929-11933" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.90.24.11929" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.90.24.11929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.90.24.11929"> </span></div> </div></div></div></div></div><div content-id="DSM015C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waterston</div> <div class="given-names">R. H</div></div>,  <div class="name"><div class="surname">Lindblad-Toh</div> <div class="given-names">K</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E</div></div>, et al. </div><div class="article-title">Initial sequencing and comparative analysis of the mouse genome</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">520</div>-<div class="lpage">562</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20comparative%20analysis%20of%20the%20mouse%20genome&amp;author=R.%20H%20Waterston&amp;author=K%20Lindblad-Toh&amp;author=E%20Birney&amp;publication_year=2002&amp;journal=Nature&amp;volume=420&amp;pages=520-562" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>,  <div class="name"><div class="surname">Glazko</div> <div class="given-names">G. V</div></div>. </div><div class="article-title">The Wilhelmine E. Key 2001 Invitational Lecture. Estimation of divergence times for a few mammalian and several primate species</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">157</div>-<div class="lpage">164</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Wilhelmine%20E.%20Key%202001%20Invitational%20Lecture.%20Estimation%20of%20divergence%20times%20for%20a%20few%20mammalian%20and%20several%20primate%20species&amp;author=M%20Nei&amp;author=G.%20V%20Glazko&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=157-164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.3.157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.3.157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.3.157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12195029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glazko</div> <div class="given-names">G. V</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E. V</div></div>,  <div class="name"><div class="surname">Rogozin</div> <div class="given-names">I. B</div></div>. </div><div class="article-title">Molecular dating: ape bones agree with chicken entrails</div>, <div class="source">Trends Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">89</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20dating%3A%20ape%20bones%20agree%20with%20chicken%20entrails&amp;author=G.%20V%20Glazko&amp;author=E.%20V%20Koonin&amp;author=I.%20B%20Rogozin&amp;publication_year=2005&amp;journal=Trends%20Genet.&amp;volume=21&amp;pages=89-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2004.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15661354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arndt</div> <div class="given-names">P. F</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D. A</div></div>,  <div class="name"><div class="surname">Hwa</div> <div class="given-names">T</div></div>. </div><div class="article-title">Distinct changes of genomic biases in nucleotide substitution at the time of Mammalian radiation</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1887</div>-<div class="lpage">1896</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20changes%20of%20genomic%20biases%20in%20nucleotide%20substitution%20at%20the%20time%20of%20Mammalian%20radiation&amp;author=P.%20F%20Arndt&amp;author=D.%20A%20Petrov&amp;author=T%20Hwa&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=1887-1896" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12885958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duret</div> <div class="given-names">L</div></div>,  <div class="name"><div class="surname">Semon</div> <div class="given-names">M</div></div>,  <div class="name"><div class="surname">Piganeau</div> <div class="given-names">G</div></div>,  <div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D</div></div>,  <div class="name"><div class="surname">Galtier</div> <div class="given-names">N</div></div>. </div><div class="article-title">Vanishing GC-rich isochores in mammalian genomes</div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">162</div> (pg. <div class="fpage">1837</div>-<div class="lpage">1847</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vanishing%20GC-rich%20isochores%20in%20mammalian%20genomes&amp;author=L%20Duret&amp;author=M%20Semon&amp;author=G%20Piganeau&amp;author=D%20Mouchiroud&amp;author=N%20Galtier&amp;publication_year=2002&amp;journal=Genetics&amp;volume=162&amp;pages=1837-1847" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12524353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">N. G</div></div>,  <div class="name"><div class="surname">Webster</div> <div class="given-names">M. T</div></div>,  <div class="name"><div class="surname">Ellegren</div> <div class="given-names">H</div></div>. </div><div class="article-title">Deterministic mutation rate variation in the human genome</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1350</div>-<div class="lpage">1356</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deterministic%20mutation%20rate%20variation%20in%20the%20human%20genome&amp;author=N.%20G%20Smith&amp;author=M.%20T%20Webster&amp;author=H%20Ellegren&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1350-1356" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.220502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.220502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.220502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12213772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Webster</div> <div class="given-names">M. T</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">N. G</div></div>,  <div class="name"><div class="surname">Ellegren</div> <div class="given-names">H</div></div>. </div><div class="article-title">Compositional evolution of noncoding DNA in the human and chimpanzee genomes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">278</div>-<div class="lpage">286</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20evolution%20of%20noncoding%20DNA%20in%20the%20human%20and%20chimpanzee%20genomes&amp;author=M.%20T%20Webster&amp;author=N.%20G%20Smith&amp;author=H%20Ellegren&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=278-286" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12598695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alvarez-Valin</div> <div class="given-names">F</div></div>,  <div class="name"><div class="surname">Clay</div> <div class="given-names">O</div></div>,  <div class="name"><div class="surname">Cruveiller</div> <div class="given-names">S</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G</div></div>. </div><div class="article-title">Inaccurate reconstruction of ancestral GC levels creates a ‘vanishing isochores’ effect</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">788</div>-<div class="lpage">793</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inaccurate%20reconstruction%20of%20ancestral%20GC%20levels%20creates%20a%20%E2%80%98vanishing%20isochores%E2%80%99%20effect&amp;author=F%20Alvarez-Valin&amp;author=O%20Clay&amp;author=S%20Cruveiller&amp;author=G%20Bernardi&amp;publication_year=2004&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=31&amp;pages=788-793" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2004.01.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2004.01.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2004.01.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15062812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gu</div> <div class="given-names">J</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W. H</div></div>. </div><div class="article-title">Are GC-rich isochores vanishing in mammals?</div>, <div class="source">Gene</div> , <div class="year">2006</div>, vol. <div class="volume">385</div> (pg. <div class="fpage">50</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Are%20GC-rich%20isochores%20vanishing%20in%20mammals%3F&amp;author=J%20Gu&amp;author=W.%20H%20Li&amp;publication_year=2006&amp;journal=Gene&amp;volume=385&amp;pages=50-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.03.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.03.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.03.026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16987615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vallender</div> <div class="given-names">E. J</div></div>,  <div class="name"><div class="surname">Paschall</div> <div class="given-names">J. E</div></div>,  <div class="name"><div class="surname">Malcom</div> <div class="given-names">C. M</div></div>,  <div class="name"><div class="surname">Lahn</div> <div class="given-names">B. T</div></div>,  <div class="name"><div class="surname">Wyckoff</div> <div class="given-names">G. J</div></div>. </div><div class="article-title">SPEED: a molecular-evolution-based database of mammalian orthologous groups</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">2835</div>-<div class="lpage">2837</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPEED%3A%20a%20molecular-evolution-based%20database%20of%20mammalian%20orthologous%20groups&amp;author=E.%20J%20Vallender&amp;author=J.%20E%20Paschall&amp;author=C.%20M%20Malcom&amp;author=B.%20T%20Lahn&amp;author=G.%20J%20Wyckoff&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2835-2837" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl471" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16966361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Penden</div> <div class="given-names">J</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M</div></div>. </div>, <div class="source">CodonW</div> , <div class="year">1997</div>, vol. <div class="volume">v. 1.4.2</div> pg. <div class="fpage">1.4.2</div> </div></div></div></div><div content-id="DSM015C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aissani</div> <div class="given-names">B</div></div>,  <div class="name"><div class="surname">D'Onofrio</div> <div class="given-names">G</div></div>,  <div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D</div></div>,  <div class="name"><div class="surname">Gardiner</div> <div class="given-names">K</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G</div></div>. </div><div class="article-title">The compositional properties of human genes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1991</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">493</div>-<div class="lpage">503</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20compositional%20properties%20of%20human%20genes&amp;author=B%20Aissani&amp;author=G%20D'Onofrio&amp;author=D%20Mouchiroud&amp;author=K%20Gardiner&amp;author=C%20Gautier&amp;author=G%20Bernardi&amp;publication_year=1991&amp;journal=J.%20Mol.%20Evol.&amp;volume=32&amp;pages=493-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02102651" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02102651" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02102651"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1908020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernardi</div> <div class="given-names">G</div></div>,  <div class="name"><div class="surname">Olofsson</div> <div class="given-names">B</div></div>,  <div class="name"><div class="surname">Filipski</div> <div class="given-names">J</div></div>, et al. </div><div class="article-title">The mosaic genome of warm-blooded vertebrates</div>, <div class="source">Science</div> , <div class="year">1985</div>, vol. <div class="volume">228</div> (pg. <div class="fpage">953</div>-<div class="lpage">958</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mosaic%20genome%20of%20warm-blooded%20vertebrates&amp;author=G%20Bernardi&amp;author=B%20Olofsson&amp;author=J%20Filipski&amp;publication_year=1985&amp;journal=Science&amp;volume=228&amp;pages=953-958" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.4001930" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.4001930" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.4001930"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4001930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J. D</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D. G</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T. J</div></div>. </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4673</div>-<div class="lpage">4680</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.%20D%20Thompson&amp;author=D.%20G%20Higgins&amp;author=T.%20J%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-4680" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W. H</div></div>. </div><div class="article-title">The codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1987</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1281</div>-<div class="lpage">1295</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20Adaptation%20Index%E2%80%94a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications&amp;author=P.%20M%20Sharp&amp;author=W.%20H%20Li&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res.&amp;volume=15&amp;pages=1281-1295" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3547335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>,  <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T</div></div>. </div><div class="article-title">Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1986</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">418</div>-<div class="lpage">426</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20methods%20for%20estimating%20the%20numbers%20of%20synonymous%20and%20nonsynonymous%20nucleotide%20substitutions&amp;author=M%20Nei&amp;author=T%20Gojobori&amp;publication_year=1986&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=3&amp;pages=418-426" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3444411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singer</div> <div class="given-names">G. A</div></div>,  <div class="name"><div class="surname">Hickey</div> <div class="given-names">D. A</div></div>. </div><div class="article-title">Nucleotide bias causes a genomewide bias in the amino acid composition of proteins</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1581</div>-<div class="lpage">1588</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20bias%20causes%20a%20genomewide%20bias%20in%20the%20amino%20acid%20composition%20of%20proteins&amp;author=G.%20A%20Singer&amp;author=D.%20A%20Hickey&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=1581-1588" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11070046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H. C</div></div>,  <div class="name"><div class="surname">Singer</div> <div class="given-names">G. A</div></div>,  <div class="name"><div class="surname">Hickey</div> <div class="given-names">D. A</div></div>. </div><div class="article-title">Mutational bias affects protein evolution in flowering plants</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">90</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20bias%20affects%20protein%20evolution%20in%20flowering%20plants&amp;author=H.%20C%20Wang&amp;author=G.%20A%20Singer&amp;author=D.%20A%20Hickey&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=21&amp;pages=90-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14595101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM015C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM015C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">N</div></div>,  <div class="name"><div class="surname">Nakashima</div> <div class="given-names">H</div></div>. </div><div class="article-title">Negative correlation of G + C content at silent substitution sites between orthologous human and mouse protein-coding sequences</div>, <div class="source">DNA Res.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">135</div>-<div class="lpage">140</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Negative%20correlation%20of%20G%20%2B%20C%20content%20at%20silent%20substitution%20sites%20between%20orthologous%20human%20and%20mouse%20protein-coding%20sequences&amp;author=N%20Takahashi&amp;author=H%20Nakashima&amp;publication_year=2006&amp;journal=DNA%20Res.&amp;volume=13&amp;pages=135-140" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsl007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsl007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsl007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17046958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=466067&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm015/1/dsm015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=H8~C-w~lBsbgcHuUR8fBwLpCiummwVywX8cv-9XIf3gXwV6x1LyM6ESkEYsikjIo4LOpx5ZiKSSyRXcbxUmslEH2opHUbfAWhG5ITEDf5Mnnw3I3oq1071iydiFZsKS6YSkvQodpmFz5FQk~Kl-4uovDuo69IEiLdRK4sUUVmVjz4Zr~VQw3BaSg6zqdrHGIYghINXwnqCtO8kO93lsT5lsRRj8tmCB0zD31EFPeqqduGNPAFeEsdbpl7jxZaorOx4k9QjpQoVpeQdy99HXbxFXJo8qKAb1ymzOxWxKF2K5IaEEO~wao65Mxt8asAG976dOiibR--PYpbIwaBfxfiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">137</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000250863200001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="466067"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000250863200001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f4%2f141%2f466067">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000250863200001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F4%2F141%2F466067" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/466067'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Reverse Polarization in Amino acid and Nucleotide Substitution Patterns Between Human–Mouse Orthologs of Two Compositional Extrema" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f141%2f466067">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

