

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/4/253/501301","@type":"ScholarlyArticle","name":"Transcriptome Analysis of  Sarracenia , an Insectivorous Plant","datePublished":"2011-06-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2011-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr014","keywords":["insectivorous","substitutions rates","transcriptome","<italic>Sarracenia</italic>","duplication"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/4/253/501301","author":[{"name":"Srivastava, Anuj","affiliation":"1  Institute of Bioinformatics ,  University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"},{"name":"Rogers, Willie L.","affiliation":"2  Department of Plant Biology ,  University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"},{"name":"Breton, Catherine M.","affiliation":"3  Center for Applied Genetic Technologies ,  University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"},{"name":"Cai, Liming","affiliation":"1  Institute of Bioinformatics ,  University of Georgia ,  Athens, GA 30602 ,  USA    4  Department of Computer Sciences ,  University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"},{"name":"Malmberg, Russell L.","affiliation":"1  Institute of Bioinformatics ,  University of Georgia ,  Athens, GA 30602 ,  USA    2  Department of Plant Biology ,  University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"}],"description":"Abstract.  Sarracenia species (pitcher plants) are carnivorous plants which obtain a portion of their nutrients from insects captured in the pitchers. To invest","pageStart":"253","pageEnd":"261","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Transcriptome Analysis of  Sarracenia , an Insectivorous Plant","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Transcriptome Analysis of  Sarracenia , an Insectivorous Plant" />
    <meta property="og:description" content="Abstract.  Sarracenia species (pitcher plants) are carnivorous plants which obtain a portion of their nutrients from insects captured in the pitchers. To invest" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/4/253/501301" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-06-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Srivastava, Anuj" /><meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author" content="Rogers, Willie L." /><meta name="citation_author_institution" content="Department of Plant Biology, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author" content="Breton, Catherine M." /><meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author" content="Cai, Liming" /><meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author_institution" content="Department of Computer Sciences, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author" content="Malmberg, Russell L." /><meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author_institution" content="Department of Plant Biology, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_title" content="Transcriptome Analysis of Sarracenia, an Insectivorous Plant" /><meta name="citation_firstpage" content="253" /><meta name="citation_lastpage" content="261" /><meta name="citation_doi" content="10.1093/dnares/dsr014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2011/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Murray, London; Insectivorous Plants;  citation_year=1875; " /><meta name="citation_reference" content="citation_title=The absorption of nutrients and allied phenomena in the pitchers of the Sarraceniaceae; J. Franklin Inst.;  citation_year=1920;  citation_volume=189;  citation_pages=147-84; " /><meta name="citation_reference" content="citation_title=Signal transduction in the carnivorous plant Sarracenia purpurea—regulation of secretory hydrolase expression during development and in response to resources; Plant Physiol.;  citation_year=1997;  citation_volume=115;  citation_pages=1461-71; " /><meta name="citation_reference" content="citation_title=Food-web models predict species abundances in response to habitat change; Plos Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=1869-73; " /><meta name="citation_reference" content="citation_title=Notes on the pollination of Sarracenia-Flava L (Sarraceniaceae) in the Piedmont Province of North-Carolina; Rhodora;  citation_year=1983;  citation_volume=85;  citation_pages=405-20; " /><meta name="citation_reference" content="citation_title=Limits to reproductive success of Sarracenia purpurea (Sarraceniaceae); Am. J. Botany;  citation_year=2006;  citation_volume=93;  citation_pages=1660-6; " /><meta name="citation_reference" content="citation_title=The molecular ecologist's guide to expressed sequence tags; Mol. Ecol.;  citation_year=2007;  citation_volume=16;  citation_pages=907-24; " /><meta name="citation_reference" content="citation_title=Sequencing breakthroughs for genomic ecology and evolutionary biology; Mol. Ecol. Res.;  citation_year=2008;  citation_volume=8;  citation_pages=3-17; " /><meta name="citation_reference" content="citation_title=Harnessing genomics for evolutionary insights; Trends Ecol. Evol.;  citation_year=2009;  citation_volume=24;  citation_pages=192-200; " /><meta name="citation_reference" content="citation_title=Genetic basis of fitness differences in natural populations; Nature;  citation_year=2008;  citation_volume=452;  citation_pages=169-75; " /><meta name="citation_reference" content="citation_title=Pyrobayes: an improved base caller for SNP discovery in pyrosequences; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=179-81; " /><meta name="citation_reference" content="citation_title=Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1147-59; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes; Plant Cell;  citation_year=2004;  citation_volume=16;  citation_pages=1667-78; " /><meta name="citation_reference" content="citation_title=Clustal W and clustal X version 2.0; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W609-12; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology; citation_author=The Gene Ontology Consortium; Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=Phylogenetic and functional assessment of orthologs inference projects and methods; citation_text=5, e1000262. doi:10.1371/journal.pcbi.1000262; PLoS Comput. Biol.;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Comparative evolutionary analysis of chalcone synthase and alcohol dehydrogenase loci in Arabidopsis, Arabis, and related genera (Brassicaceae); Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=1483-98; " /><meta name="citation_reference" content="citation_title=A survey of tricolpate (eudicot) phylogenetic relationships; Am. J. Botany;  citation_year=2004;  citation_volume=91;  citation_pages=1627-44; " /><meta name="citation_reference" content="citation_title=The origin and diversification of angiosperms; Am. J. Botany;  citation_year=2004;  citation_volume=91;  citation_pages=1614-26; " /><meta name="citation_reference" content="citation_title=The functional RNA Database 3.0: databases to support mining and annotation of functional RNAs; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D89-92; " /><meta name="citation_reference" content="citation_title=Signal transduction in the carnivorous plant Sarracenia purpurea. Regulation of secretory hydrolase expression during development and in response to resources; Plant Physiol.;  citation_year=1997;  citation_volume=115;  citation_pages=1461-71; " /><meta name="citation_reference" content="citation_title=The development of the pitcher plant Sarracenia purpurea into a potentially valuable recombinant protein production system; Biotechnol. Mol. Biol. Rev.;  citation_year=2009;  citation_volume=3;  citation_pages=105-10; " /><meta name="citation_reference" content="citation_title=The evolutionary ecology of carnivorous plants; Adv. Ecol. Res.;  citation_year=2003;  citation_volume=33;  citation_pages=1-74; " /><meta name="citation_reference" content="citation_title=Development and characterization of microsatellite markers in Sarracenia L. (pitcher plant) species; Conserv. Genet Resou.;  citation_year=2010;  citation_volume=2;  citation_pages=75-9; " /><meta name="citation_reference" content="citation_title=The frequency of polyploid speciation in vascular plants; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=13875-9; " /><meta name="citation_reference" content="citation_title=Statistical methods for detecting molecular adaptation; Trends Ecol. Evol.;  citation_year=2000;  citation_volume=15;  citation_pages=496-503; " /><meta name="citation_reference" content="citation_title=Natural selection on protein-coding genes in the human genome; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=1153-7; " /><meta name="citation_reference" content="citation_title=Evolutionary genomics and the reach of selection; J. Biol.;  citation_year=2009;  citation_volume=8;  citation_pages=12" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/4/253/1349675/dsr014.pdf" /><meta name="description" content="Abstract.  Sarracenia species (pitcher plants) are carnivorous plants which obtain a portion of their nutrients from insects captured in the pitchers. To invest" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/4/253/501301" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/4/253/501301" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/4/253/501301");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr014';
    </script>
<title>
	Transcriptome Analysis of Sarracenia, an Insectivorous Plant | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./501301" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcZKPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTcml2YXN0YXZhLCBBbnVqIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzLCBVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIEF0aGVucywgR0EgMzA2MDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJvZ2VycywgV2lsbGllIEwuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIEF0aGVucywgR0EgMzA2MDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJyZXRvbiwgQ2F0aGVyaW5lIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBBcHBsaWVkIEdlbmV0aWMgVGVjaG5vbG9naWVzLCBVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIEF0aGVucywgR0EgMzA2MDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNhaSwgTGltaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzLCBVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIEF0aGVucywgR0EgMzA2MDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0ZXIgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgR2VvcmdpYSwgQXRoZW5zLCBHQSAzMDYwMiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFsbWJlcmcsIFJ1c3NlbGwgTC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIFVuaXZlcnNpdHkgb2YgR2VvcmdpYSwgQXRoZW5zLCBHQSAzMDYwMiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIEF0aGVucywgR0EgMzA2MDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVHJhbnNjcmlwdG9tZSBBbmFseXNpcyBvZiBTYXJyYWNlbmlhLCBhbiBJbnNlY3Rpdm9yb3VzIFBsYW50IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMS8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1NdXJyYXksIExvbmRvbjsgSW5zZWN0aXZvcm91cyBQbGFudHM7ICBjaXRhdGlvbl95ZWFyPTE4NzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhYnNvcnB0aW9uIG9mIG51dHJpZW50cyBhbmQgYWxsaWVkIHBoZW5vbWVuYSBpbiB0aGUgcGl0Y2hlcnMgb2YgdGhlIFNhcnJhY2VuaWFjZWFlOyBKLiBGcmFua2xpbiBJbnN0LjsgIGNpdGF0aW9uX3llYXI9MTkyMDsgIGNpdGF0aW9uX3ZvbHVtZT0xODk7ICBjaXRhdGlvbl9wYWdlcz0xNDctODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpZ25hbCB0cmFuc2R1Y3Rpb24gaW4gdGhlIGNhcm5pdm9yb3VzIHBsYW50IFNhcnJhY2VuaWEgcHVycHVyZWHigJRyZWd1bGF0aW9uIG9mIHNlY3JldG9yeSBoeWRyb2xhc2UgZXhwcmVzc2lvbiBkdXJpbmcgZGV2ZWxvcG1lbnQgYW5kIGluIHJlc3BvbnNlIHRvIHJlc291cmNlczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9MTQ2MS03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9vZC13ZWIgbW9kZWxzIHByZWRpY3Qgc3BlY2llcyBhYnVuZGFuY2VzIGluIHJlc3BvbnNlIHRvIGhhYml0YXQgY2hhbmdlOyBQbG9zIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xODY5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3RlcyBvbiB0aGUgcG9sbGluYXRpb24gb2YgU2FycmFjZW5pYS1GbGF2YSBMIChTYXJyYWNlbmlhY2VhZSkgaW4gdGhlIFBpZWRtb250IFByb3ZpbmNlIG9mIE5vcnRoLUNhcm9saW5hOyBSaG9kb3JhOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9NDA1LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW1pdHMgdG8gcmVwcm9kdWN0aXZlIHN1Y2Nlc3Mgb2YgU2FycmFjZW5pYSBwdXJwdXJlYSAoU2FycmFjZW5pYWNlYWUpOyBBbS4gSi4gQm90YW55OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9MTY2MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyIGVjb2xvZ2lzdCdzIGd1aWRlIHRvIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzOyBNb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9OTA3LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIGJyZWFrdGhyb3VnaHMgZm9yIGdlbm9taWMgZWNvbG9neSBhbmQgZXZvbHV0aW9uYXJ5IGJpb2xvZ3k7IE1vbC4gRWNvbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9My0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGFybmVzc2luZyBnZW5vbWljcyBmb3IgZXZvbHV0aW9uYXJ5IGluc2lnaHRzOyBUcmVuZHMgRWNvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xOTItMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGJhc2lzIG9mIGZpdG5lc3MgZGlmZmVyZW5jZXMgaW4gbmF0dXJhbCBwb3B1bGF0aW9uczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MjsgIGNpdGF0aW9uX3BhZ2VzPTE2OS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHlyb2JheWVzOiBhbiBpbXByb3ZlZCBiYXNlIGNhbGxlciBmb3IgU05QIGRpc2NvdmVyeSBpbiBweXJvc2VxdWVuY2VzOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE3OS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgdGhlIG1pcmFFU1QgYXNzZW1ibGVyIGZvciByZWxpYWJsZSBhbmQgYXV0b21hdGVkIG1STkEgdHJhbnNjcmlwdCBhc3NlbWJseSBhbmQgU05QIGRldGVjdGlvbiBpbiBzZXF1ZW5jZWQgRVNUczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMTQ3LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CbGFzdDJHTzogYSB1bml2ZXJzYWwgdG9vbCBmb3IgYW5ub3RhdGlvbiwgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgaW4gZnVuY3Rpb25hbCBnZW5vbWljcyByZXNlYXJjaDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNjc0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVTigJR0aGUgQkxBU1QtbGlrZSBhbGlnbm1lbnQgdG9vbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NTYtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2lkZXNwcmVhZCBwYWxlb3BvbHlwbG9pZHkgaW4gbW9kZWwgcGxhbnQgc3BlY2llcyBpbmZlcnJlZCBmcm9tIGFnZSBkaXN0cmlidXRpb25zIG9mIGR1cGxpY2F0ZSBnZW5lczsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE2NjctNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0YWwgVyBhbmQgY2x1c3RhbCBYIHZlcnNpb24gMi4wOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5NDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFMMk5BTDogcm9idXN0IGNvbnZlcnNpb24gb2YgcHJvdGVpbiBzZXF1ZW5jZSBhbGlnbm1lbnRzIGludG8gdGhlIGNvcnJlc3BvbmRpbmcgY29kb24gYWxpZ25tZW50czsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9VzYwOS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFNTCA0OiBwaHlsb2dlbmV0aWMgYW5hbHlzaXMgYnkgbWF4aW11bSBsaWtlbGlob29kOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTU4Ni05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBvbnRvbG9neTogdG9vbCBmb3IgdGhlIHVuaWZpY2F0aW9uIG9mIGJpb2xvZ3k7IGNpdGF0aW9uX2F1dGhvcj1UaGUgR2VuZSBPbnRvbG9neSBDb25zb3J0aXVtOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBhbmQgZnVuY3Rpb25hbCBhc3Nlc3NtZW50IG9mIG9ydGhvbG9ncyBpbmZlcmVuY2UgcHJvamVjdHMgYW5kIG1ldGhvZHM7IGNpdGF0aW9uX3RleHQ9NSwgZTEwMDAyNjIuIGRvaToxMC4xMzcxL2pvdXJuYWwucGNiaS4xMDAwMjYyOyBQTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGV2b2x1dGlvbmFyeSBhbmFseXNpcyBvZiBjaGFsY29uZSBzeW50aGFzZSBhbmQgYWxjb2hvbCBkZWh5ZHJvZ2VuYXNlIGxvY2kgaW4gQXJhYmlkb3BzaXMsIEFyYWJpcywgYW5kIHJlbGF0ZWQgZ2VuZXJhIChCcmFzc2ljYWNlYWUpOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTQ4My05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdXJ2ZXkgb2YgdHJpY29scGF0ZSAoZXVkaWNvdCkgcGh5bG9nZW5ldGljIHJlbGF0aW9uc2hpcHM7IEFtLiBKLiBCb3Rhbnk7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xNjI3LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JpZ2luIGFuZCBkaXZlcnNpZmljYXRpb24gb2YgYW5naW9zcGVybXM7IEFtLiBKLiBCb3Rhbnk7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xNjE0LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZnVuY3Rpb25hbCBSTkEgRGF0YWJhc2UgMy4wOiBkYXRhYmFzZXMgdG8gc3VwcG9ydCBtaW5pbmcgYW5kIGFubm90YXRpb24gb2YgZnVuY3Rpb25hbCBSTkFzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1EODktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpZ25hbCB0cmFuc2R1Y3Rpb24gaW4gdGhlIGNhcm5pdm9yb3VzIHBsYW50IFNhcnJhY2VuaWEgcHVycHVyZWEuIFJlZ3VsYXRpb24gb2Ygc2VjcmV0b3J5IGh5ZHJvbGFzZSBleHByZXNzaW9uIGR1cmluZyBkZXZlbG9wbWVudCBhbmQgaW4gcmVzcG9uc2UgdG8gcmVzb3VyY2VzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz0xNDYxLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZGV2ZWxvcG1lbnQgb2YgdGhlIHBpdGNoZXIgcGxhbnQgU2FycmFjZW5pYSBwdXJwdXJlYSBpbnRvIGEgcG90ZW50aWFsbHkgdmFsdWFibGUgcmVjb21iaW5hbnQgcHJvdGVpbiBwcm9kdWN0aW9uIHN5c3RlbTsgQmlvdGVjaG5vbC4gTW9sLiBCaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMDUtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb25hcnkgZWNvbG9neSBvZiBjYXJuaXZvcm91cyBwbGFudHM7IEFkdi4gRWNvbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTEtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgaW4gU2FycmFjZW5pYSBMLiAocGl0Y2hlciBwbGFudCkgc3BlY2llczsgQ29uc2Vydi4gR2VuZXQgUmVzb3UuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz03NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZnJlcXVlbmN5IG9mIHBvbHlwbG9pZCBzcGVjaWF0aW9uIGluIHZhc2N1bGFyIHBsYW50czsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xMzg3NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGF0aXN0aWNhbCBtZXRob2RzIGZvciBkZXRlY3RpbmcgbW9sZWN1bGFyIGFkYXB0YXRpb247IFRyZW5kcyBFY29sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTQ5Ni01MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5hdHVyYWwgc2VsZWN0aW9uIG9uIHByb3RlaW4tY29kaW5nIGdlbmVzIGluIHRoZSBodW1hbiBnZW5vbWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00Mzc7ICBjaXRhdGlvbl9wYWdlcz0xMTUzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBnZW5vbWljcyBhbmQgdGhlIHJlYWNoIG9mIHNlbGVjdGlvbjsgSi4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzQvMjUzLzEzNDk2NzUvZHNyMDE0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBTYXJyYWNlbmlhIHNwZWNpZXMgKHBpdGNoZXIgcGxhbnRzKSBhcmUgY2Fybml2b3JvdXMgcGxhbnRzIHdoaWNoIG9idGFpbiBhIHBvcnRpb24gb2YgdGhlaXIgbnV0cmllbnRzIGZyb20gaW5zZWN0cyBjYXB0dXJlZCBpbiB0aGUgcGl0Y2hlcnMuIFRvIGludmVzdCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTAxMzAxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJddhmrwgyQ7rgBiEuizi65s0kA+" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f253%2f501301" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f253%2f501301" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19820" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&amp;Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/4">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7730591">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7730591">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7730598">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7730598">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7730610">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7730610">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7730648">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7730648">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7730673">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7730673">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7730682">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7730682">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7730684">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7730684">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7730686">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7730686">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/4/233/501892">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/4/263/501627">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Transcriptome Analysis of <em>Sarracenia</em>, an Insectivorous Plant
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anuj Srivastava</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Malmberg Laboratory, Plant Biology Department, Miller Plant Sciences Bldg., University of Georgia, Athens, GA 30602-7271, USA. Tel. +1 706-542-1869. Fax. +1 706-542-1805. Email: <a href="mailto:anuj@uga.edu" target="_blank">anuj@uga.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anuj+Srivastava">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Srivastava A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Srivastava A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Anuj Srivastava</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Willie L. Rogers</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Plant Biology</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Willie+L.+Rogers">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rogers W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rogers W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Willie L. Rogers</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Catherine M. Breton</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Center for Applied Genetic Technologies</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Catherine+M.+Breton">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Breton C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Breton C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Catherine M. Breton</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Liming Cai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Computer Sciences</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Liming+Cai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cai L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cai L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Liming Cai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Russell L. Malmberg</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Plant Biology</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Russell+L.+Malmberg">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Malmberg R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Malmberg R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Russell L. Malmberg</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 253–261, <a href='https://doi.org/10.1093/dnares/dsr014'>https://doi.org/10.1093/dnares/dsr014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 June 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">01 February 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">15 May 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="501301" href="/dnaresearch/article-pdf/18/4/253/1349675/dsr014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Anuj Srivastava, Willie L. Rogers, Catherine M. Breton, Liming Cai, Russell L. Malmberg;  Transcriptome Analysis of <em>Sarracenia</em>, an Insectivorous Plant, <em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 253–261, <a href='https://doi.org/10.1093/dnares/dsr014'>https://doi.org/10.1093/dnares/dsr014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/501301?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/501301?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/501301?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/501301?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/501301?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/501301?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/501301?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Transcriptome%20Analysis%20of%20%26lt%3Bem%26gt%3BSarracenia%26lt%3B%2Fem%26gt%3B%2C%20an%20Insectivorous%20Plant&amp;publicationDate=2011-06-15&amp;volumeNum=18&amp;issueNum=4&amp;author=Srivastava%2C%20Anuj%3B%20Rogers%2C%20Willie%20L.&amp;startPage=253&amp;endPage=261&amp;contentId=10.1093%2Fdnares%2Fdsr014&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f253%2f501301" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7730591 id="7730591" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Sarracenia</em> species (pitcher plants) are carnivorous plants which obtain a portion of their nutrients from insects captured in the pitchers. To investigate these plants, we sequenced the transcriptome of two species, <em>Sarracenia psittacina</em> and <em>Sarracenia purpurea</em>, using Roche 454 pyrosequencing technology. We obtained 46 275 and 36 681 contigs by <em>de novo</em> assembly methods for <em>S. psittacina</em> and <em>S. purpurea</em>, respectively, and further identified 16 163 orthologous contigs between them. Estimation of synonymous substitution rates between orthologous and paralogous contigs indicates the events of genome duplication and speciation within the <em>Sarracenia</em> genus both occurred ∼2 million years ago. The ratios of synonymous and non-synonymous substitution rates indicated that 491 contigs have been under positive selection (<em>K</em><sub>a</sub><em>/K</em><sub>s</sub> &gt; 1). Significant proportions of these contigs were involved in functions related to binding activity. We also found that the greatest sequence similarity for both of these species was to <em>Vitis vinifera</em>, which is most consistent with a non-current classification of the order Ericales as an asterid. This study has provided new insights into pitcher plants and will contribute greatly to future research on this genus and its distinctive ecological adaptations.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="insectivorous">insectivorous</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;substitutions rates&quot;">substitutions rates</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Sarracenia"><em>Sarracenia</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="duplication">duplication</a></div>                            <h2 scrollto-destination=7730598 id="7730598" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Carnivorous plants fascinate both scientists<sup><a reveal-id="DSR014C1" data-open="DSR014C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and the general public (Audrey Jr. in: Corman, 1960). One carnivorous plant genus is <em>Sarracenia</em> (pitcher plants) which typically grow in highly acidic, nutrient poor soils that are water saturated for at least part of the year, such as bogs, grassy savannas, fens, and similar wetlands. They obtain a portion of their nutrients from prey captured in their pitchers—highly modified tubular leaves. <em>Sarracenia</em> species may digest their prey directly with secreted proteases, phosphatases, and nucleases.<sup><a reveal-id="DSR014C2" data-open="DSR014C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSR014C3" data-open="DSR014C3" class="link link-ref link-reveal xref-bibr">3</a></sup> However, one of our focal species, <em>Sarracenia purpurea</em>, hosts a complex food web of bacteria, protozoa, and arthropods that mineralize the prey and release nutrients that are taken up by the plant.<sup><a reveal-id="DSR014C4" data-open="DSR014C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The other focal species, <em>Sarracenia psittacina</em>, does not host such a food web. Despite the interspecific variability in pitcher form, the flowers of the different species are morphologically quite similar, and they are pollinated by a range of generalist bees and sarcophagid flies;<sup><a reveal-id="DSR014C5" data-open="DSR014C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR014C6" data-open="DSR014C6" class="link link-ref link-reveal xref-bibr">6</a></sup> interspecific hybrids with intermediate morphologies are common in nature and are fertile. A number of the taxa within the genus are considered endangered. As a genus, <em>Sarracenia</em> provides a wealth of opportunities for ecological and evolutionary studies.</p><p>Here, we present a comprehensive analysis of the transcriptome (cDNA) of two <em>Sarracenia</em> species (<em>S. psittacina</em> and <em>S. purpurea</em>) obtained by 454 sequencing technology (454 GS FLX). Transcriptome sequencing represents the subset of genes from the genome that are functionally active in a selected tissue and species of interest. In non-model organisms lacking genomic resources, such as sequenced genome, transcriptome study is an effective way to study the gene expression and address comparative genomic-level questions.<sup><a reveal-id="DSR014C7" data-open="DSR014C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR014C8" data-open="DSR014C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Moreover, massive parallelized sequencing technologies have made transcriptome studies, one of the most cost-effective methods for gene discovery,<sup><a reveal-id="DSR014C7" data-open="DSR014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> even more robust and efficient. 454 sequencing was selected as its sufficiently long-sequence reads can help compensate for the lack of a reference genome during <em>de novo</em> sequence assembly.<sup><a reveal-id="DSR014C9" data-open="DSR014C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>As this is the first set of sequence data for any pitcher plants, we addressed a number of questions in this study ranging from: identifying the events of genome duplication, determining the level of orthology between two species, estimating the substitution rates between the orthologous contigs, and investigating the contigs which show signatures of diversifying natural selection (a non-neutral rate of synonymous and non-synonymous substitutions between sequence pairs). These comparative genomics methods along with genetic mapping, expression profiling, and candidate gene approaches are part of investigating the genetic basis of phenotypic variation.<sup><a reveal-id="DSR014C10" data-open="DSR014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> We also performed a functional annotation, through gene ontology analysis, of all contigs in order to detect any pattern (biological process, molecular function, and cellular component) which may be unique or predominant to pitcher plants.</p>                            <h2 scrollto-destination=7730610 id="7730610" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7730614 id="7730614" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p>Two species of <em>Sarracenia</em> were chosen based on morphology and differences in insect trapping/digestion. <em>Sarracenia purpurea</em> has the widest natural range among all species within the genus. It is found from Mississippi eastward up the entire US east coast. It is also found in all the Great Lake states and throughout the majority of Canada. Natural populations of <em>S. psittacina</em> can be found in all Gulf coast states except Texas and it also has populations in Georgia. Fresh juvenile leaf samples were taken from greenhouse maintained plant stocks. Samples came from identical or very similar cultivars generated from rhizome propagation.</p>                            <h3 scrollto-destination=7730622 id="7730622" class="section-title" data-legacy-id=s2b>2.2. RNA extraction</h3>
<p>After multiple extraction attempts using various kits and other wet lab techniques, a successful protocol was found using the Spectrum Plant Total RNA Kit (Sigma #STRN50-1KT). Only the youngest unopened leaves were used as older tissues yield negligible RNA amounts. Certain steps call for the use of RNase-free water; this was created by adding 0.1 ml of diethylpyrocarbonate to 100 ml of water, incubating at 37°C for 12 h and then autoclaving for 15 min to remove trace amounts of the chemical. All instruments and surfaces were cleaned thoroughly, treated with an RNase deactivating solution (100 mM sodium hydroxide plus 0.1% sodium lauryl sulphate) and wiped dry. The mortar and pestles were frozen with liquid nitrogen for about 20 s prior to the start of tissue and maintained at a subzero temperature throughout tissue grinding to prevent RNA degradation. A single total RNA prep of 100 mg of young leaf tissue yielded on average 4–8 µg of high-quality RNA. The extraction was repeated 40–50 times. We obtained ∼0.25 mg of total RNA for each species.</p>                            <h3 scrollto-destination=7730631 id="7730631" class="section-title" data-legacy-id=s2c>2.3. mRNA isolation and cDNA generation</h3>
<p>The Oligotec mRNA Kit (Qiagen # 70022) was used to purify mRNA from 0.25 mg extracted total RNA, according to the manufacturer's recommendation. During the elution steps, 25 µl of OEB buffer was used in the initial step and 25 µl in the follow-up step for the maximum mRNA concentration in the smallest volume possible. mRNA quality was checked with a Bioanalyzer (Agilent, Inc.) and a NanoDrop 2000 (Thermo Scientific). For cDNA generation, the Evrogen cDNA synthesis kit (SK001) was used with a modification to the kit's 3′ primer. We used ∼26 ng of mRNA for each species.</p><p>Normalization of the cDNA was performed using Evrogen Normalization Trimmer Kit (NK001) in order to minimize the repetition of transcripts. This normalization protocol is based on denaturing-reassociation of cDNA, followed by digestion with a duplex-specific nuclease. The single-stranded cDNA fraction was amplified twice by sequential PCR reactions according to the manufacturer's protocol. A Qiagen MiniElut kit (#28006) was used to purify the normalized cDNA and sterile water was used in the final elution in order to prevent the likely interference of TE with 454 processing. Normalized cDNA in 100 µl sterile water was submitted to the Georgia Genomic Facility (<a class="link link-uri" href="http://www.dna.uga.edu">www.dna.uga.edu</a>) for 454 sequencing in a Genome Sequencer TM (GS) Titanium FLX instrument (Roche Diagnostics) employing a standard protocol. <em>Sarracenia psittacina</em> and <em>Sarracenia purpurea</em> cDNAs were submitted at a concentration of 49 and 45 ng/µl, respectively.</p>                            <h3 scrollto-destination=7730638 id="7730638" class="section-title" data-legacy-id=s2d>2.4. Sequence assembly, contig annotation, and ortholog/paralog identification</h3>
<p>We called the bases from the sequence data using <em>pyrobayes</em><sup><a reveal-id="DSR014C11" data-open="DSR014C11" class="link link-ref link-reveal xref-bibr">11</a></sup> from the 454 sequencer generated sff files. Vector and other contaminants were removed using <em>seqclean</em> (<a class="link link-uri" href="http://compbio.dfci.harvard.edu">http://compbio.dfci.harvard.edu</a>); assembly of the cleaned reads was performed by <em>MIRA.</em><sup><a reveal-id="DSR014C12" data-open="DSR014C12" class="link link-ref link-reveal xref-bibr">12</a></sup><em>Blast2Go</em> (B2G)<sup><a reveal-id="DSR014C13" data-open="DSR014C13" class="link link-ref link-reveal xref-bibr">13</a></sup> was used at the default criterion to functionally annotated the contigs. In order to localize the contigs, we obtained <em>Vitis</em> genomic sequences (since this genome had the top hits in blastx searches) and gff files from the phytozome project (<a class="link link-uri" href="http://www.phytozome.net/">http://www.phytozome.net/</a>) and then mapped both the assemblies against the <em>Vitis</em> genome using <em>blat</em> (step size = 11, minScore = 40).<sup><a reveal-id="DSR014C14" data-open="DSR014C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Orthologous sequences between the two <em>Sarracenia</em> species were predicted using the reciprocal blast (blastn)<sup><a reveal-id="DSR014C15" data-open="DSR014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> hit method at <em>e</em>-value 1<em>e</em>− 20. This stringent <em>e</em>-value cut-off leads to a higher identification of orthologous as opposed to paralogous sequences. Paralogous sequences were identified by doing all vs all blast (blastn) within the species. Sequences producing a significant alignment over 300 bp and 40% identity were defined as paralogs.<sup><a reveal-id="DSR014C16" data-open="DSR014C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>                            <h3 scrollto-destination=7730644 id="7730644" class="section-title" data-legacy-id=s2e>2.5. Detection of genome duplication and speciation events</h3>
<p>The approach used in the detection of genome duplication and speciation event was adapted from Blanc and Wolfe.<sup><a reveal-id="DSR014C16" data-open="DSR014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Identified paralogous pairs were organized into gene families using single-linkage clustering. Afterwards, synonymous substitution rates (<em>K</em><sub>s</sub>) were estimated for all possible pairs within a gene family. One potential drawback of measuring the substitution rates from transcriptome data is that multiple entries of the same gene can be present which leads to redundant <em>K</em><sub>s</sub> measures. To minimize this false peak in the <em>K</em><sub>s</sub> distribution plot, we discarded one of the sequence from paralogous pairs with <em>K</em><sub>s</sub> = 0 (assuming that no synonymous substitutions between sequences means they belong to the same gene) and all other <em>K</em><sub>s</sub> values involving that particular sequence. We corrected for multiple <em>K</em><sub>s</sub> comparisons from gene families which contain non-overlapping incomplete sequences by a simple clustering method, as described in Blanc and Wolfe.<sup><a reveal-id="DSR014C16" data-open="DSR014C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>                            <h3 scrollto-destination=7730646 id="7730646" class="section-title" data-legacy-id=s2f>2.6. Estimation of substitution rates</h3>
<p>To estimate the synonymous (<em>K</em><sub>s</sub>) and non-synonymous substitutions (<em>K</em><sub>a</sub>) rates between paralogous and orthologous sequences pairs, we first aligned the sequence pairs using <em>tblastx</em>;<sup><a reveal-id="DSR014C15" data-open="DSR014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> sequences producing significant alignments were extracted using their aligned coordinates and further analysed by translation and then amino acid sequence alignment was performed by <em>Clustalw.</em><sup><a reveal-id="DSR014C17" data-open="DSR014C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Only the longest uninterrupted reading frame was used for the analysis. Corresponding codon alignments were produced using <em>PAL2NAL</em><sup><a reveal-id="DSR014C18" data-open="DSR014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and finally rates were estimated using a maximum likelihood method implemented in the <em>CODEML</em> program of the <em>PAML</em> package Version 4.1.<sup><a reveal-id="DSR014C19" data-open="DSR014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Pair-wise maximum likelihood analyses were performed in runmode-2. In order to minimize the statistical artefacts which could arise due to short alignments and a saturation of <em>K</em><sub>s</sub>, we further discarded those alignments which are less than 30aa in length and which had <em>K</em><sub>s</sub> &gt;2. The <em>K</em><sub>s</sub> frequency in each interval size of 0.01 within the range [0, 2.0] was plotted.</p>                            <h2 scrollto-destination=7730648 id="7730648" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7730649 id="7730649" class="section-title" data-legacy-id=s3a>3.1. Sequencing and assembly</h3>
<p>The amount of raw sequences obtained was 123 and 88 Mb with a mean raw read length was 249 and 258 bp for <em>S. psittacina</em> and <em>S. purpurea</em>, respectively. Raw sequences were cleaned and assembled into contigs. Since the <em>Sarracenia</em> genome sequence was not available, a <em>de novo</em> assembly of the cDNA sequences was performed. We obtained 46 275 and 36 681 contigs with an N50 value of 479 and 485 for <em>S. psittacina and S. purpurea</em>, respectively. The number of assembled contigs and the mean average coverage per contig were found to be correlated with the number of cleaned reads. The complete assembly statistics are shown in Table <a reveal-id="DSR014TB1" data-open="DSR014TB1" class="link link-reveal link-table xref-fig">1</a> and assemblies are available under <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S1</a>. </p>                        <a id="7730651" scrollto-destination="7730651"></a>
<div content-id="DSR014TB1" class="table-modal"><div class="table-wrap"><div id="DSR014TB1" class="table-wrap-title" data-id="DSR014TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly statistics for two <em>Sarracenia</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Ind<sup>a</sup> </th><th>Plate<sup>b</sup> </th><th>Cleaned reads </th><th>Cleaned bases (Mb) </th><th>Contigs </th><th>Singletons </th><th>Mean avg coverage per contig </th><th>Mean GC per contig (%) </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>8 </td><td>1/2 </td><td>392 346 </td><td>102 </td><td>46 275 </td><td>587 </td><td>3.40 </td><td>40.84 </td></tr><tr><td><em>S. purpurea</em> </td><td>4 </td><td>1/2 </td><td>282 150 </td><td>75 </td><td>36 681 </td><td>234 </td><td>3.05 </td><td>41.14 </td></tr></tbody></table></div><div content-id="DSR014TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Ind<sup>a</sup> </th><th>Plate<sup>b</sup> </th><th>Cleaned reads </th><th>Cleaned bases (Mb) </th><th>Contigs </th><th>Singletons </th><th>Mean avg coverage per contig </th><th>Mean GC per contig (%) </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>8 </td><td>1/2 </td><td>392 346 </td><td>102 </td><td>46 275 </td><td>587 </td><td>3.40 </td><td>40.84 </td></tr><tr><td><em>S. purpurea</em> </td><td>4 </td><td>1/2 </td><td>282 150 </td><td>75 </td><td>36 681 </td><td>234 </td><td>3.05 </td><td>41.14 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of individual pooled prior to sequencing.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>One plate represents a full Roche 454 run.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7730651" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR014TB1" class="table-wrap-title" data-id="DSR014TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly statistics for two <em>Sarracenia</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Ind<sup>a</sup> </th><th>Plate<sup>b</sup> </th><th>Cleaned reads </th><th>Cleaned bases (Mb) </th><th>Contigs </th><th>Singletons </th><th>Mean avg coverage per contig </th><th>Mean GC per contig (%) </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>8 </td><td>1/2 </td><td>392 346 </td><td>102 </td><td>46 275 </td><td>587 </td><td>3.40 </td><td>40.84 </td></tr><tr><td><em>S. purpurea</em> </td><td>4 </td><td>1/2 </td><td>282 150 </td><td>75 </td><td>36 681 </td><td>234 </td><td>3.05 </td><td>41.14 </td></tr></tbody></table></div><div content-id="DSR014TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Ind<sup>a</sup> </th><th>Plate<sup>b</sup> </th><th>Cleaned reads </th><th>Cleaned bases (Mb) </th><th>Contigs </th><th>Singletons </th><th>Mean avg coverage per contig </th><th>Mean GC per contig (%) </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>8 </td><td>1/2 </td><td>392 346 </td><td>102 </td><td>46 275 </td><td>587 </td><td>3.40 </td><td>40.84 </td></tr><tr><td><em>S. purpurea</em> </td><td>4 </td><td>1/2 </td><td>282 150 </td><td>75 </td><td>36 681 </td><td>234 </td><td>3.05 </td><td>41.14 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of individual pooled prior to sequencing.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>One plate represents a full Roche 454 run.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7730651" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7730652 id="7730652" class="section-title" data-legacy-id=s3b>3.2. Functional annotation of contigs</h3>
<p>We used B2G to functionally annotate the contigs. The B2G annotation has three steps which involve using blast against the public or private databases, mapping against GO resources, and annotating to generate reliable functional assignments. From our data, 20 920 (45.2%) of the <em>S. psittacina</em> and 17 821 (48.6%) of <em>S. purpurea</em> cDNA sequences were shown to have significant matches to currently known proteins in the NCBI non-redundant protein database. The B2G blast hit bar plot (Fig. <a reveal-id="DSR014F1" data-open="DSR014F1" class="link link-reveal link-table xref-fig">1</a>) shows <em>Vitis</em>, <em>Ricinus</em>, and <em>Populus</em> as the top three species with greatest number of hits for both species<em>.</em> Contigs with the significant blast matches were functionally annotated. We found GO resource assignments for 19.92% and 21.19% of contigs for <em>S. psittacina</em> and <em>S. purpurea</em>, respectively. A summary of B2G mapping is given in Table <a reveal-id="DSR014TB2" data-open="DSR014TB2" class="link link-reveal link-table xref-fig">2</a>. </p>                        <a id="7730654" scrollto-destination="7730654"></a>
<div content-id="DSR014TB2" class="table-modal"><div class="table-wrap"><div id="DSR014TB2" class="table-wrap-title" data-id="DSR014TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics for two <em>Sarracenia</em> species of Blast2GO assignment</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of contigs </th><th>Number of blast hits </th><th>Number of GO mapped </th><th>Number of GO terms<sup>a</sup> </th><th>Number of functionally annotated </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>46 275 </td><td>20 920 </td><td>9222 </td><td>39 500 </td><td>7208 </td></tr><tr><td><em>S. purpurea</em> </td><td>36 681 </td><td>17 821 </td><td>7773 </td><td>33 093 </td><td>6060 </td></tr></tbody></table></div><div content-id="DSR014TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of contigs </th><th>Number of blast hits </th><th>Number of GO mapped </th><th>Number of GO terms<sup>a</sup> </th><th>Number of functionally annotated </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>46 275 </td><td>20 920 </td><td>9222 </td><td>39 500 </td><td>7208 </td></tr><tr><td><em>S. purpurea</em> </td><td>36 681 </td><td>17 821 </td><td>7773 </td><td>33 093 </td><td>6060 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Can be multiple per contigs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7730654" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR014TB2" class="table-wrap-title" data-id="DSR014TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics for two <em>Sarracenia</em> species of Blast2GO assignment</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of contigs </th><th>Number of blast hits </th><th>Number of GO mapped </th><th>Number of GO terms<sup>a</sup> </th><th>Number of functionally annotated </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>46 275 </td><td>20 920 </td><td>9222 </td><td>39 500 </td><td>7208 </td></tr><tr><td><em>S. purpurea</em> </td><td>36 681 </td><td>17 821 </td><td>7773 </td><td>33 093 </td><td>6060 </td></tr></tbody></table></div><div content-id="DSR014TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of contigs </th><th>Number of blast hits </th><th>Number of GO mapped </th><th>Number of GO terms<sup>a</sup> </th><th>Number of functionally annotated </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>46 275 </td><td>20 920 </td><td>9222 </td><td>39 500 </td><td>7208 </td></tr><tr><td><em>S. purpurea</em> </td><td>36 681 </td><td>17 821 </td><td>7773 </td><td>33 093 </td><td>6060 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Can be multiple per contigs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7730654" target="_blank">View Large</a></div></div>                        <a id="7730655" scrollto-destination="7730655"></a>
<div id="DSR014F1" data-id="DSR014F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01401.jpeg?Expires=2147483647&Signature=U7b06AjBRJCJdtJGT6AYwVl0kh29UyICoxCOqy~kdapoDu~bD~q2GoRFq~k~8Gm8s8qzpA6MoUDIblYLElbptfpoG9Z~Aws4JAsTSJJQvmnmRlcdvX-LFzBkweI1ysPP7HAdfRJypaFZrYPjmDNURqqpqyoQHN8OURsV2t1jNkmjX-SruCj~W7UHRwjp12yjuiPciHaRwCxmNqVyIoR3XQGyGxyWGHsx9NyaMIWdugMNpPsJAf1E1YaGUqo16B3UMqq4gAqq4cM4CAMK-ZYTuoH76aUgGqztm417gJH6gCY~1WZMMAQk0uImvbn5zwM9clC88~4bgm4yeYIpSS~HhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A bar plot showing the hits (blastx top hit) to previously sequenced species (displaying only top five species) for S. psittacina and S. purpurea contigs." path-from-xml="dsr01401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730655/dsr01401.jpeg" path-from-xml="dsr01401.jpeg" target="_blank">View large</a><a section="7730655" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01401.jpeg?Expires=2147483647&Signature=kJtZWGuynxxCkGxnXFMOIqplYlAIMI2himMYNEFAUYj3HWUMsE2MmHVMgtjEt6kas-acwYfV4mYUNuEGxkZ2SsEdIN6gyh824dhID2I2l11Xr0fsiEYRIeQwgLpeo7f10G63SWRXgDp0RXaP9-ZgEe0zgcn62~2Oh7RlYnkkgUOaA-fUBSUKxVqBCGR0l5szFEmJkEWoKUBOZsUnI4G0XGJEvHVXABM8HgJBktTCgEO1Q6OhI4JlGXmQ5xdMtwBUplDfTVAPGAJqMc0JBlKxSJkczlJF0cqAaz7xEiwCgRWlkH530M0YcJFilA~Pmxlqvwr-Iz9NLh3TZRwQ58H66g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730655&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A bar plot showing the hits (blastx top hit) to previously sequenced species (displaying only top five species) for <em>S. psittacina and S. purpurea</em> contigs.</p></div></div><div content-id="DSR014F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01401.jpeg?Expires=2147483647&Signature=U7b06AjBRJCJdtJGT6AYwVl0kh29UyICoxCOqy~kdapoDu~bD~q2GoRFq~k~8Gm8s8qzpA6MoUDIblYLElbptfpoG9Z~Aws4JAsTSJJQvmnmRlcdvX-LFzBkweI1ysPP7HAdfRJypaFZrYPjmDNURqqpqyoQHN8OURsV2t1jNkmjX-SruCj~W7UHRwjp12yjuiPciHaRwCxmNqVyIoR3XQGyGxyWGHsx9NyaMIWdugMNpPsJAf1E1YaGUqo16B3UMqq4gAqq4cM4CAMK-ZYTuoH76aUgGqztm417gJH6gCY~1WZMMAQk0uImvbn5zwM9clC88~4bgm4yeYIpSS~HhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A bar plot showing the hits (blastx top hit) to previously sequenced species (displaying only top five species) for S. psittacina and S. purpurea contigs." path-from-xml="dsr01401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730655/dsr01401.jpeg" path-from-xml="dsr01401.jpeg" target="_blank">View large</a><a section="7730655" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01401.jpeg?Expires=2147483647&Signature=kJtZWGuynxxCkGxnXFMOIqplYlAIMI2himMYNEFAUYj3HWUMsE2MmHVMgtjEt6kas-acwYfV4mYUNuEGxkZ2SsEdIN6gyh824dhID2I2l11Xr0fsiEYRIeQwgLpeo7f10G63SWRXgDp0RXaP9-ZgEe0zgcn62~2Oh7RlYnkkgUOaA-fUBSUKxVqBCGR0l5szFEmJkEWoKUBOZsUnI4G0XGJEvHVXABM8HgJBktTCgEO1Q6OhI4JlGXmQ5xdMtwBUplDfTVAPGAJqMc0JBlKxSJkczlJF0cqAaz7xEiwCgRWlkH530M0YcJFilA~Pmxlqvwr-Iz9NLh3TZRwQ58H66g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730655&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A bar plot showing the hits (blastx top hit) to previously sequenced species (displaying only top five species) for <em>S. psittacina and S. purpurea</em> contigs.</p></div></div><p>The first major GO division, ‘biological process’, associates contigs with the biological objective to which it contributes.<sup><a reveal-id="DSR014C20" data-open="DSR014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Within it, 11 major categories were identified and found to be similarly distributed in both species. The two most abundant categories were: (i) ‘cellular and metabolic processes’, to which 75% of both species’ contigs were associated (<em>S. psittacina:</em> 7543 sequences and <em>S. purpurea</em>: 6285 sequences) and (ii) ‘biological regulation’, to which 8% of contigs were dedicated (<em>S. psittacina</em>: 802 sequences and <em>S. purpurea</em>: 684 sequences) (Fig. <a reveal-id="DSR014F2" data-open="DSR014F2" class="link link-reveal link-table xref-fig">2</a>A). </p>                        <a id="7730657" scrollto-destination="7730657"></a>
<div id="DSR014F2" data-id="DSR014F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01402.jpeg?Expires=2147483647&Signature=4E28XJ3u7aQVo-zWqT2hWqG3TvGr3MMpXhsrqOuH4yWRwoIvh2EnoMW1MJPiloIyjwzm2PsjlkEzQtA85NznfASqZ8IP-dxT0ZZEXApJIB6T8lYsP7uIQdvpLr95qfGfTzHYawxyx7BLoCuj~3cl-1nf7OrPHE1IcpESMhZ4sytasnxX22hppYGtTVIpVP06oYIwlnDtVuBrd3xJiJVAfThp2b~FwPC83gN6S23gQPf5aazOAPaxPidazWYZCvEYXYCNDvV2zJGrQHsrU8Ntg1yak8KaCV8bpCryyVz2RxdVCvxte3dTZ3hzm0ft5Tk9iByFD1459A1u4MEwAeZaZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A bar plot showing the Blast2GO functional assignments in three GO categories: (A) biological process, (B) molecular function, and (C) cellular component for S. psittacina and S. purpurea contigs." path-from-xml="dsr01402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730657/dsr01402.jpeg" path-from-xml="dsr01402.jpeg" target="_blank">View large</a><a section="7730657" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01402.jpeg?Expires=2147483647&Signature=tvO93pV3yfWpuaFRW1w3Bzra7vRTPOSNhCI7fZlT~eR0nYzhJCymOkawzw-Uudg5Y29X53m7SL5Rlyb8le3FoT17V~TN5ey~cTX6dv9OkoL~ElxE3a8L~ZtKSmc8EZmxoXueY432EiCv7BhJ0t~ho7feFlrle9pA-x4AtYU4ee2SOBCcaQm-NxBqxwntbPcOt-5KXQ6Or1qYlwU4DK6hUmiiSS~f0BBAMHVowTp~Cu3yuK4bMuDcUcaaVLUlvFpz-GRBNY3v3UIrnmiE~sdTSqp3qCKpKItvVpznXPWAEuPkYVR-vwSD4kVSZBAEwj5FHat5xRrMZOEmALXZ0cwYUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730657&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A bar plot showing the Blast2GO functional assignments in three GO categories: (A) biological process, (B) molecular function, and (C) cellular component for <em>S. psittacina</em> and <em>S. purpurea</em> contigs.</p></div></div><div content-id="DSR014F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01402.jpeg?Expires=2147483647&Signature=4E28XJ3u7aQVo-zWqT2hWqG3TvGr3MMpXhsrqOuH4yWRwoIvh2EnoMW1MJPiloIyjwzm2PsjlkEzQtA85NznfASqZ8IP-dxT0ZZEXApJIB6T8lYsP7uIQdvpLr95qfGfTzHYawxyx7BLoCuj~3cl-1nf7OrPHE1IcpESMhZ4sytasnxX22hppYGtTVIpVP06oYIwlnDtVuBrd3xJiJVAfThp2b~FwPC83gN6S23gQPf5aazOAPaxPidazWYZCvEYXYCNDvV2zJGrQHsrU8Ntg1yak8KaCV8bpCryyVz2RxdVCvxte3dTZ3hzm0ft5Tk9iByFD1459A1u4MEwAeZaZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A bar plot showing the Blast2GO functional assignments in three GO categories: (A) biological process, (B) molecular function, and (C) cellular component for S. psittacina and S. purpurea contigs." path-from-xml="dsr01402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730657/dsr01402.jpeg" path-from-xml="dsr01402.jpeg" target="_blank">View large</a><a section="7730657" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01402.jpeg?Expires=2147483647&Signature=tvO93pV3yfWpuaFRW1w3Bzra7vRTPOSNhCI7fZlT~eR0nYzhJCymOkawzw-Uudg5Y29X53m7SL5Rlyb8le3FoT17V~TN5ey~cTX6dv9OkoL~ElxE3a8L~ZtKSmc8EZmxoXueY432EiCv7BhJ0t~ho7feFlrle9pA-x4AtYU4ee2SOBCcaQm-NxBqxwntbPcOt-5KXQ6Or1qYlwU4DK6hUmiiSS~f0BBAMHVowTp~Cu3yuK4bMuDcUcaaVLUlvFpz-GRBNY3v3UIrnmiE~sdTSqp3qCKpKItvVpznXPWAEuPkYVR-vwSD4kVSZBAEwj5FHat5xRrMZOEmALXZ0cwYUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730657&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A bar plot showing the Blast2GO functional assignments in three GO categories: (A) biological process, (B) molecular function, and (C) cellular component for <em>S. psittacina</em> and <em>S. purpurea</em> contigs.</p></div></div><p>The second major GO division, ‘molecular function’, links genes to their biochemical activity.<sup><a reveal-id="DSR014C20" data-open="DSR014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The contig coverage was again found to be similar for both species. Most of the contigs in the molecular function division were dedicated to binding functions and catalytic activity (82% of both species; <em>S. psittacina</em>: 7367 sequences and <em>S. purpurea</em>: 6170 sequences) (Fig. <a reveal-id="DSR014F2" data-open="DSR014F2" class="link link-reveal link-table xref-fig">2</a>B).</p><p>The last GO division is ‘cellular component’, which refers to sub-cellular location where gene product is active.<sup><a reveal-id="DSR014C20" data-open="DSR014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In this, eight major categories were identified and again a similar type of coverage was found for both species. Gene products were mainly expressed intracellularly (52% for both species; <em>S. psittacina</em>: 3033 sequences; <em>S. purpurea</em>: 2562 sequences) or in the membrane-bound/non-membrane-bound organelle (29% for both species; <em>S. psittacina</em>: 1730 sequences; <em>S. purpurea</em>: 1386) (Fig. <a reveal-id="DSR014F2" data-open="DSR014F2" class="link link-reveal link-table xref-fig">2</a>C). The complete B2G results are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>.</p>                            <h3 scrollto-destination=7730660 id="7730660" class="section-title" data-legacy-id=s3c>3.3. Localization of the contigs with respect to genic features</h3>
<p>We used the <em>Vitis</em> genome as a reference in order to locate the contigs with respect to genomic sequence. <em>Vitis</em> was the species whose sequences were found as the top hit for sequences from both <em>Sarracenia</em> species in the blast (blastx) search. Blat was used to map the assemblies against the <em>Vitis</em> genome. Sequences uniquely mapped to a particular feature (5′UTR, 3′UTR, CDS, 1 kb up, 1 kb down, and intergenic) were plotted as a bar plot (Fig. <a reveal-id="DSR014F3" data-open="DSR014F3" class="link link-reveal link-table xref-fig">3</a>). A total of 8501 and 7224 contigs were uniquely mapped to features under consideration for <em>S. psittacina</em> and <em>S. purpurea</em>, respectively. Based on the <em>Vitis</em> genome annotation, nearly 60% of the mapped contigs were found to be in CDS regions and 33% of the contigs were associated with the putative intergenic region (shown as NA in the figure). Only a very tiny fraction of the contigs were mapped to 5′UTR/3′UTR and regions 1 kb upstream/1 kb downstream of them. </p>                        <a id="7730662" scrollto-destination="7730662"></a>
<div id="DSR014F3" data-id="DSR014F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01403.jpeg?Expires=2147483647&Signature=Py20lBq9Gf0yfbdYjMGom9iOVgC0lUYbJw6JBLTLOMs3saqFghzLdlmW3Tq28uNJBot00ekfjEDyUbc8VAC~VJRMH182hqadfFalJxrkTo2s6YKNo62MclDyGWcthRbbGfb~Xoq398WAI6AKniUBU3HH7XPhRy7Gd5-UmQcm8jMbXTiMXOzBVCLtuZQTW4TOZalhReWWn5eFbAe7Cz3EUSt6b4Y~XE9pevvd19ax8Pyh5HL5ZpuEvEg~NsPPuOsnAo2A6UOtb1qhk~LEcluvlwH~gVTIUTzNGqQtmX1070kySfG2J7JNsSKexgecqcgH2OmXBlK0tOKvpQgNKt-4Nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A bar plot displaying the proportion of contigs mapped to a particular region of Vitis genome in two species." path-from-xml="dsr01403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730662/dsr01403.jpeg" path-from-xml="dsr01403.jpeg" target="_blank">View large</a><a section="7730662" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01403.jpeg?Expires=2147483647&Signature=MBBK5uHHbgLlafvO2pUgj4dI-PeyaTxy~9thecMSZX6GdfdzpUoS2F6Fc2atijcQFwhEwviCOvH7erC02Aa0YhAqKciemC6N49~jAIRvO27-SB1UWRimniAyTm4y4KSAjg-684UYdOcVkRri0JlX6UnqLFXDG6jBPVJcyuI4uINYnap-MLMH3aF1H3u5qI7yaRJ7khriWr8WevEWteVa1b5-z-MvVMUtxPvt~KIvTMH5VH8sMA8f9w5SuqKvWh1d4tnqAEgE0woEbQHmQDBqUkvg32mij4A7q2FDiGO6e7YsbG8wm6oa-OxQL6Q9k62~7eXzMxa36Q~C6jzU5iC1wQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730662&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A bar plot displaying the proportion of contigs mapped to a particular region of <em>Vitis</em> genome in two species.</p></div></div><div content-id="DSR014F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01403.jpeg?Expires=2147483647&Signature=Py20lBq9Gf0yfbdYjMGom9iOVgC0lUYbJw6JBLTLOMs3saqFghzLdlmW3Tq28uNJBot00ekfjEDyUbc8VAC~VJRMH182hqadfFalJxrkTo2s6YKNo62MclDyGWcthRbbGfb~Xoq398WAI6AKniUBU3HH7XPhRy7Gd5-UmQcm8jMbXTiMXOzBVCLtuZQTW4TOZalhReWWn5eFbAe7Cz3EUSt6b4Y~XE9pevvd19ax8Pyh5HL5ZpuEvEg~NsPPuOsnAo2A6UOtb1qhk~LEcluvlwH~gVTIUTzNGqQtmX1070kySfG2J7JNsSKexgecqcgH2OmXBlK0tOKvpQgNKt-4Nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A bar plot displaying the proportion of contigs mapped to a particular region of Vitis genome in two species." path-from-xml="dsr01403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730662/dsr01403.jpeg" path-from-xml="dsr01403.jpeg" target="_blank">View large</a><a section="7730662" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01403.jpeg?Expires=2147483647&Signature=MBBK5uHHbgLlafvO2pUgj4dI-PeyaTxy~9thecMSZX6GdfdzpUoS2F6Fc2atijcQFwhEwviCOvH7erC02Aa0YhAqKciemC6N49~jAIRvO27-SB1UWRimniAyTm4y4KSAjg-684UYdOcVkRri0JlX6UnqLFXDG6jBPVJcyuI4uINYnap-MLMH3aF1H3u5qI7yaRJ7khriWr8WevEWteVa1b5-z-MvVMUtxPvt~KIvTMH5VH8sMA8f9w5SuqKvWh1d4tnqAEgE0woEbQHmQDBqUkvg32mij4A7q2FDiGO6e7YsbG8wm6oa-OxQL6Q9k62~7eXzMxa36Q~C6jzU5iC1wQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730662&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A bar plot displaying the proportion of contigs mapped to a particular region of <em>Vitis</em> genome in two species.</p></div></div>                            <h3 scrollto-destination=7730663 id="7730663" class="section-title" data-legacy-id=s3d>3.4. Orthologous and paralogous contigs</h3>
<p>We identified 16 163 pairs of orthologous contigs by the reciprocal best blast hit approach; this approach has been found superior to more sophisticated orthology detection algorithms.<sup><a reveal-id="DSR014C21" data-open="DSR014C21" class="link link-ref link-reveal xref-bibr">21</a></sup> We used a stringent <em>e</em>-value cutoff (1<em>e</em> −20) in order to separate the paralogous sequences from the orthologous sequences. A total of 8706 orthologous contigs matched to open reading frames of known or putative proteins. The Venn diagram indicating the orthologous and unique contigs is shown in Fig. <a reveal-id="DSR014F4" data-open="DSR014F4" class="link link-reveal link-table xref-fig">4</a>. We also identified 18 296 and 9565 paralogs pairs by all vs all blast (blastn); these were organized into 2555 and 1708 gene families by a single linkage clustering method for <em>S. psittacina</em> and <em>S. purpurea</em>, respectively. </p>                        <a id="7730665" scrollto-destination="7730665"></a>
<div id="DSR014F4" data-id="DSR014F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01404.jpeg?Expires=2147483647&Signature=WqR0X3eCH2EIZS6sqEZ3JeUicGloBLqFgKjCcIb0ZOIMJeTpXwYf0u3I67Or5166LAQvwZ8Pa1gsiFXqswzREyDl2BYNxytJRb3Q24qI24HBLDtB2424X4gWcljd6F677l6W5PEV9YgW70cD~xTRIJoh0cLtDLaehSDQhOb9bNvmlzC6Q0WkLYCoA2jS2vUcA0bKgWs53ABzh-fCbZsbmBC2y-LhDJE5N6dx1x6ktQj7zvm759-5qesu8LGDuFvYB9jCloP6VxTiBnp8HU-BppYSekKWh2ldy9XgWezCGYQ7YRtoZ7lCqaf3d9vzFpIaESnluZzC1sgTnd3rQ7oyDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A Venn diagram showing the count of orthologous and unique contigs between two species." path-from-xml="dsr01404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730665/dsr01404.jpeg" path-from-xml="dsr01404.jpeg" target="_blank">View large</a><a section="7730665" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01404.jpeg?Expires=2147483647&Signature=4oPQEM8nj-yLzbUHRdioRlyMOspJKVFrHFJxWiBoIsBiwUatLBelZsjYQRx7ECYC2Bk~-xFpBu6gunGc1reyAQL-FS4JOEh2PU5W2xKBro5AYBxJhH-v0CbaO9X37OSxLz2O3rV5b~xLHoqJOMpLglGb0FJO0T4Z8XoCKC3AAELWPvrr7bRwZcBSqfh6vF3scOtVe0QLWImGv~yVfYjCCfSXZHRhmiJMVx3ytztu7m3AbY9Q8gKfgua0TBX-X3xuGuhlbCeLWCCYXZiY~QCxOcs4wcW4l2RCCKCZ229lEyIZOFSjcV6V-8JR9mkuN-113WFb6ZLZH35FT~m5UTiygw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730665&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A Venn diagram showing the count of orthologous and unique contigs between two species.</p></div></div><div content-id="DSR014F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01404.jpeg?Expires=2147483647&Signature=WqR0X3eCH2EIZS6sqEZ3JeUicGloBLqFgKjCcIb0ZOIMJeTpXwYf0u3I67Or5166LAQvwZ8Pa1gsiFXqswzREyDl2BYNxytJRb3Q24qI24HBLDtB2424X4gWcljd6F677l6W5PEV9YgW70cD~xTRIJoh0cLtDLaehSDQhOb9bNvmlzC6Q0WkLYCoA2jS2vUcA0bKgWs53ABzh-fCbZsbmBC2y-LhDJE5N6dx1x6ktQj7zvm759-5qesu8LGDuFvYB9jCloP6VxTiBnp8HU-BppYSekKWh2ldy9XgWezCGYQ7YRtoZ7lCqaf3d9vzFpIaESnluZzC1sgTnd3rQ7oyDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A Venn diagram showing the count of orthologous and unique contigs between two species." path-from-xml="dsr01404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730665/dsr01404.jpeg" path-from-xml="dsr01404.jpeg" target="_blank">View large</a><a section="7730665" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01404.jpeg?Expires=2147483647&Signature=4oPQEM8nj-yLzbUHRdioRlyMOspJKVFrHFJxWiBoIsBiwUatLBelZsjYQRx7ECYC2Bk~-xFpBu6gunGc1reyAQL-FS4JOEh2PU5W2xKBro5AYBxJhH-v0CbaO9X37OSxLz2O3rV5b~xLHoqJOMpLglGb0FJO0T4Z8XoCKC3AAELWPvrr7bRwZcBSqfh6vF3scOtVe0QLWImGv~yVfYjCCfSXZHRhmiJMVx3ytztu7m3AbY9Q8gKfgua0TBX-X3xuGuhlbCeLWCCYXZiY~QCxOcs4wcW4l2RCCKCZ229lEyIZOFSjcV6V-8JR9mkuN-113WFb6ZLZH35FT~m5UTiygw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730665&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A Venn diagram showing the count of orthologous and unique contigs between two species.</p></div></div>                            <h3 scrollto-destination=7730666 id="7730666" class="section-title" data-legacy-id=s3e>3.5. Estimation of K<sub>a</sub><em>and</em> K<sub>s</sub></h3>
<p>We calculated the <em>K</em><sub>a</sub> and <em>K</em><sub>s</sub> values for 10 715 orthologous contigs and their ratio for 4810 contigs (a <em>K</em><sub>s</sub> = 0 made the ratio incalculable for some of the contigs). Similarly, we estimated the <em>K</em><sub>s</sub> value for each of the paralogous pairs. We further studied substitution rates and GO categorization within the orthologous contigs only. Out of these 4810 contigs, we were able to find the functional annotation for 3444 contigs. We identified 491 contigs which are under diversifying selection <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ≥ 1 (Fig. <a reveal-id="DSR014F5" data-open="DSR014F5" class="link link-reveal link-table xref-fig">5</a>). Most of these contigs were found to be involved in the molecular function related to nucleotide binding. The complete <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> result for 3444 contigs is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. </p>                        <a id="7730668" scrollto-destination="7730668"></a>
<div id="DSR014F5" data-id="DSR014F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01405.jpeg?Expires=2147483647&Signature=MXdZd8W9~KT7oZkhoh2YukRwOHEFtt7BhiVIb5nhjYVLvJBIZvFQOubVBuo59JdhXDS3G2ILyyhWyfDICvQSYBE0lwDp3twAygnEdB95Gq7oULLLTdR5z28fBy1P0GiY67R6WFwwJ6-0F9lljkhpotHEf68yiRmvfG4wgndd1VRPP3vI298OYrwIqXLALArmpKWpZEAFH-cglp~UAsO2D0KyibMOm3VoHitWptsehrVhwAZvpNR-LDgsAZa1neb0xBHT6QPUi087aSNzRBeJzgfpT7RVkLN0lg8Lg-d9Oy~ZnRLxAfNJfXZsRsxx2G86~5oWgbHRsFeJ-3UeSeO1Vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A scatter plot of the Ka/Ks ratio for 491 orthologous contigs under diversifying selection. Contigs with Ka/Ks &gt; 1 fall above the light grey line and &gt;2 values fall above the black line." path-from-xml="dsr01405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730668/dsr01405.jpeg" path-from-xml="dsr01405.jpeg" target="_blank">View large</a><a section="7730668" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01405.jpeg?Expires=2147483647&Signature=snHH15MTppbgmCv9GE~KNd8vthmMIYarlrI-WqbigN99b5-LYlHvUseBrrwjkZQ-jOADMHfLN0zrYO9C8sV0iY0eoAr~noFWq-eNxywU3f0wfnMesY5KEq8o3BGCWzxDyUtjpa1QavcRNEaZdIcKz3rP5~Y7XmMDQ9ozodeaEJllB~9Cw~AVlytBPPRg~wABfb9qJonHfOqbGeQE7KYW99QIKdF8Lr8cZqKgEyVvdPjG23l3pgwD~Dvvk4bEQHOhRBxP5PLGPDnsdddCezhmFWdYicRxw65WaO2m1bQy1iffa3wOFCfQKkUjkw6ahInT7vzvUQ5pWxpevKjajr5eWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730668&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A scatter plot of the <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio for 491 orthologous contigs under diversifying selection. Contigs with <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> &gt; 1 fall above the light grey line and &gt;2 values fall above the black line.</p></div></div><div content-id="DSR014F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01405.jpeg?Expires=2147483647&Signature=MXdZd8W9~KT7oZkhoh2YukRwOHEFtt7BhiVIb5nhjYVLvJBIZvFQOubVBuo59JdhXDS3G2ILyyhWyfDICvQSYBE0lwDp3twAygnEdB95Gq7oULLLTdR5z28fBy1P0GiY67R6WFwwJ6-0F9lljkhpotHEf68yiRmvfG4wgndd1VRPP3vI298OYrwIqXLALArmpKWpZEAFH-cglp~UAsO2D0KyibMOm3VoHitWptsehrVhwAZvpNR-LDgsAZa1neb0xBHT6QPUi087aSNzRBeJzgfpT7RVkLN0lg8Lg-d9Oy~ZnRLxAfNJfXZsRsxx2G86~5oWgbHRsFeJ-3UeSeO1Vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A scatter plot of the Ka/Ks ratio for 491 orthologous contigs under diversifying selection. Contigs with Ka/Ks &gt; 1 fall above the light grey line and &gt;2 values fall above the black line." path-from-xml="dsr01405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730668/dsr01405.jpeg" path-from-xml="dsr01405.jpeg" target="_blank">View large</a><a section="7730668" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01405.jpeg?Expires=2147483647&Signature=snHH15MTppbgmCv9GE~KNd8vthmMIYarlrI-WqbigN99b5-LYlHvUseBrrwjkZQ-jOADMHfLN0zrYO9C8sV0iY0eoAr~noFWq-eNxywU3f0wfnMesY5KEq8o3BGCWzxDyUtjpa1QavcRNEaZdIcKz3rP5~Y7XmMDQ9ozodeaEJllB~9Cw~AVlytBPPRg~wABfb9qJonHfOqbGeQE7KYW99QIKdF8Lr8cZqKgEyVvdPjG23l3pgwD~Dvvk4bEQHOhRBxP5PLGPDnsdddCezhmFWdYicRxw65WaO2m1bQy1iffa3wOFCfQKkUjkw6ahInT7vzvUQ5pWxpevKjajr5eWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730668&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A scatter plot of the <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio for 491 orthologous contigs under diversifying selection. Contigs with <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> &gt; 1 fall above the light grey line and &gt;2 values fall above the black line.</p></div></div>                            <h3 scrollto-destination=7730669 id="7730669" class="section-title" data-legacy-id=s3f>3.6. Genome duplication and speciation</h3>
<p>From the estimation of synonymous substitution rates, we were able to find signatures of genome duplication within both <em>S. psittacina</em> and <em>S. purpurea</em>. The <em>K</em><sub>s</sub> value distribution for both species is shown in Fig. <a reveal-id="DSR014F6" data-open="DSR014F6" class="link link-reveal link-table xref-fig">6</a>A. The secondary peak in the paralogous <em>K</em><sub>s</sub> distribution plot indicates a genome duplication event.<sup><a reveal-id="DSR014C16" data-open="DSR014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The estimated <em>K</em><sub>s</sub> value between orthologous pairs was also plotted along with paralogous (Fig. <a reveal-id="DSR014F6" data-open="DSR014F6" class="link link-reveal link-table xref-fig">6</a>B). The secondary peak in the orthologous <em>K</em><sub>s</sub> value distribution indicates the speciation events. The number of sequences involved in the genome duplication events and gene family statistics are shown in Table <a reveal-id="DSR014TB3" data-open="DSR014TB3" class="link link-reveal link-table xref-fig">3</a>. Considering a clock-like rates of synonymous substitution of 1.5 × 10<sup>−8</sup> substitutions/synonymous site/year for dicots,<sup><a reveal-id="DSR014C22" data-open="DSR014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> we estimated the age of these events and found that they are fairly recent [∼2 million years (myr) for both duplication and speciation]. The purpose of these estimates is just to give an idea of the time scale involved because they are certainly highly approximate. </p>                        <a id="7730671" scrollto-destination="7730671"></a>
<div content-id="DSR014TB3" class="table-modal"><div class="table-wrap"><div id="DSR014TB3" class="table-wrap-title" data-id="DSR014TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of sequences and paralogs found for <em>S. psittacina</em> and <em>S. purpurea</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of contigs </th><th>Number of paralogs pairs </th><th>Percentage of paralogs </th><th>Gene families </th><th>Gene family size </th><th>Duplication event with mean <em>K</em><sub>s</sub> </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>46 275 </td><td>18 296 </td><td>39.53 </td><td>2555 </td><td>3.53 </td><td>2635 </td></tr><tr><td><em>S. purpurea</em> </td><td>36 681 </td><td>9565 </td><td>26.07 </td><td>1708 </td><td>3.46 </td><td>1417 </td></tr></tbody></table></div><div content-id="DSR014TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of contigs </th><th>Number of paralogs pairs </th><th>Percentage of paralogs </th><th>Gene families </th><th>Gene family size </th><th>Duplication event with mean <em>K</em><sub>s</sub> </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>46 275 </td><td>18 296 </td><td>39.53 </td><td>2555 </td><td>3.53 </td><td>2635 </td></tr><tr><td><em>S. purpurea</em> </td><td>36 681 </td><td>9565 </td><td>26.07 </td><td>1708 </td><td>3.46 </td><td>1417 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7730671" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR014TB3" class="table-wrap-title" data-id="DSR014TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of sequences and paralogs found for <em>S. psittacina</em> and <em>S. purpurea</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of contigs </th><th>Number of paralogs pairs </th><th>Percentage of paralogs </th><th>Gene families </th><th>Gene family size </th><th>Duplication event with mean <em>K</em><sub>s</sub> </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>46 275 </td><td>18 296 </td><td>39.53 </td><td>2555 </td><td>3.53 </td><td>2635 </td></tr><tr><td><em>S. purpurea</em> </td><td>36 681 </td><td>9565 </td><td>26.07 </td><td>1708 </td><td>3.46 </td><td>1417 </td></tr></tbody></table></div><div content-id="DSR014TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Number of contigs </th><th>Number of paralogs pairs </th><th>Percentage of paralogs </th><th>Gene families </th><th>Gene family size </th><th>Duplication event with mean <em>K</em><sub>s</sub> </th></tr></thead><tbody><tr><td><em>S. psittacina</em> </td><td>46 275 </td><td>18 296 </td><td>39.53 </td><td>2555 </td><td>3.53 </td><td>2635 </td></tr><tr><td><em>S. purpurea</em> </td><td>36 681 </td><td>9565 </td><td>26.07 </td><td>1708 </td><td>3.46 </td><td>1417 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7730671" target="_blank">View Large</a></div></div>                        <a id="7730672" scrollto-destination="7730672"></a>
<div id="DSR014F6" data-id="DSR014F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01406.jpeg?Expires=2147483647&Signature=iXY2kHwqlofEwmFo7aq2Iwy7AGZvuZJ2vdfcFj8IFG330~pVJGDc8WFRewo1osa99-3wbblG46j-VHNljoi3ya7PaOi2NxpRoEhcE7A-H0M83zub9yQ0CHjmlf0GztmS30Qzlsyt57lciEY1bqfvZutUFJLPdFYnzaT5EfuhrYoUMLz9f5jF-MO4tJlKLuvdOC90XOKW5s1cNW1icqoD-wnOUPFaBx4aPdu1dsh5~5MFCfdVGFk0IF8GKjXCQ39xfPjovoW59FLDBUtSBbj9ciL~3XIeT~BYosfHJLfJ-RuWDEyHYiCQkXhTJQ5i-Up1P~jc-h34KYM3m07Qvvgcgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Ks value distribution between the two Sarracenia species for the identification of the genome duplication event and the speciation event. (A) The secondary peak (Ks = 0.03) in the paralogous Ks distribution indicates the genome duplication. (B) The secondary peak (Ks = 0.03) in the orthologous Ks distribution give the indication of speciation event." path-from-xml="dsr01406.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730672/dsr01406.jpeg" path-from-xml="dsr01406.jpeg" target="_blank">View large</a><a section="7730672" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01406.jpeg?Expires=2147483647&Signature=upbd9Q7VGOHgRGgBL9P34WGIvIBFm3s1EoNUBdsYiZ7X9u~aZLseo5voMZ3Fuj7D8F-J69gzJgvxxgUXkgyc7USrrSLMtLlpRU5gto78VF~tIR9dWh7mmVnX2StYvwsS8NVzZPcavLmTJyjpujMDSn3x95fjCOlhIsPFg97eaiLTyzuCqUi-geAdBh0JR8nPZtciZIRy-ssHhmBs4PGz-GqbbkPLK3dThcKJ6DARbj34O9ViLbbTLhIRzemvhJp4i1cLs26w6T7Lyw6qFH~hb1PozytuOPOuiE8qJFS8pImcESQY0YPRgGdU84xwmC6VXvpBFVwIBHPj7JtpSMsOtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730672&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01406.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>K</em><sub>s</sub> value distribution between the two <em>Sarracenia</em> species for the identification of the genome duplication event and the speciation event. (A) The secondary peak (<em>K</em><sub>s</sub> = 0.03) in the paralogous <em>K</em><sub>s</sub> distribution indicates the genome duplication. (B) The secondary peak (<em>K</em><sub>s</sub> = 0.03) in the orthologous <em>K</em><sub>s</sub> distribution give the indication of speciation event.</p></div></div><div content-id="DSR014F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/m_dsr01406.jpeg?Expires=2147483647&Signature=iXY2kHwqlofEwmFo7aq2Iwy7AGZvuZJ2vdfcFj8IFG330~pVJGDc8WFRewo1osa99-3wbblG46j-VHNljoi3ya7PaOi2NxpRoEhcE7A-H0M83zub9yQ0CHjmlf0GztmS30Qzlsyt57lciEY1bqfvZutUFJLPdFYnzaT5EfuhrYoUMLz9f5jF-MO4tJlKLuvdOC90XOKW5s1cNW1icqoD-wnOUPFaBx4aPdu1dsh5~5MFCfdVGFk0IF8GKjXCQ39xfPjovoW59FLDBUtSBbj9ciL~3XIeT~BYosfHJLfJ-RuWDEyHYiCQkXhTJQ5i-Up1P~jc-h34KYM3m07Qvvgcgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Ks value distribution between the two Sarracenia species for the identification of the genome duplication event and the speciation event. (A) The secondary peak (Ks = 0.03) in the paralogous Ks distribution indicates the genome duplication. (B) The secondary peak (Ks = 0.03) in the orthologous Ks distribution give the indication of speciation event." path-from-xml="dsr01406.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7730672/dsr01406.jpeg" path-from-xml="dsr01406.jpeg" target="_blank">View large</a><a section="7730672" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr01406.jpeg?Expires=2147483647&Signature=upbd9Q7VGOHgRGgBL9P34WGIvIBFm3s1EoNUBdsYiZ7X9u~aZLseo5voMZ3Fuj7D8F-J69gzJgvxxgUXkgyc7USrrSLMtLlpRU5gto78VF~tIR9dWh7mmVnX2StYvwsS8NVzZPcavLmTJyjpujMDSn3x95fjCOlhIsPFg97eaiLTyzuCqUi-geAdBh0JR8nPZtciZIRy-ssHhmBs4PGz-GqbbkPLK3dThcKJ6DARbj34O9ViLbbTLhIRzemvhJp4i1cLs26w6T7Lyw6qFH~hb1PozytuOPOuiE8qJFS8pImcESQY0YPRgGdU84xwmC6VXvpBFVwIBHPj7JtpSMsOtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7730672&ar=501301&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01406.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>K</em><sub>s</sub> value distribution between the two <em>Sarracenia</em> species for the identification of the genome duplication event and the speciation event. (A) The secondary peak (<em>K</em><sub>s</sub> = 0.03) in the paralogous <em>K</em><sub>s</sub> distribution indicates the genome duplication. (B) The secondary peak (<em>K</em><sub>s</sub> = 0.03) in the orthologous <em>K</em><sub>s</sub> distribution give the indication of speciation event.</p></div></div>                            <h2 scrollto-destination=7730673 id="7730673" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We generated the 46 275 and 36 681 contigs by pyrosequencing for <em>S. psittacina</em> and <em>S. purpurea</em>, respectively, using young pitchers as the starting material. We normalized our cDNA library in order to maximize coverage of transcripts and prevent biases due to highly expressed transcripts. Based upon the sequence similarity searches, we found <em>Vitis</em> to have the greatest number of hits to contigs from both species. This is an interesting result since <em>Sarracenia</em> is currently considered to belong to the order Ericales which is a part of clade asterids, whereas <em>Vitis</em> belongs to clade rosids (<a class="link link-uri" href="http://www.mobot.org/mobot/research/apweb/">http://www.mobot.org/mobot/research/apweb/</a>). The expectation might have been that an extensively sequenced asterid such as <em>Lycopersicon</em> would have had the greatest number of blast hits, rather than a rosid. To test the statistical significance of this result, we divided the species with the top 20 number of blast (blastx) hits into two parts based upon their phylogenetic position in asterids and rosids and performed a <em>t</em>-test of significance (two-sample <em>t</em>-test for unequal variances, one-tail <em>t</em>-statistics <em>t</em>(6) = 1.943180274, <em>P</em>∼0.05 for the <em>S. psittacina</em> sequences, with a similar result for the <em>S. purpurea</em> sequences). The <em>t</em>-test shows that the observed difference is significant. Relationships among familial clades in the order Ericales have been considered as problematic<sup><a reveal-id="DSR014C23" data-open="DSR014C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and Ericales was placed in the clade dilleniidae in the previous classifications.<sup><a reveal-id="DSR014C24" data-open="DSR014C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Our results are thus consistent with the original placement of the Ericales, as being closer to rosids than asterids.</p><p>Since a genomic sequence is not available for any pitcher plants, we used the <em>Vitis</em> genome as a reference for contig localization. About 60% of contigs aligned uniquely to the protein coding regions of <em>Vitis</em> genome in both species. The nearly equal coverage in 5′UTR and 3′UTR regions showed the success of the protocol in full-length cDNA construction. We also found a number of contigs belong to intergenic regions based upon the <em>Vitis</em> genome annotation. As the <em>Vitis</em> genome annotation is still not finished and many more genes have yet to be identified, it is possible that contigs currently localized in intergenic regions might be as-yet unidentified protein-coding genes, or they might be non-coding RNA genes or alternatively spliced exons. To test this, we took the contigs belonging to intergenic regions and checked whether they had similarity to any known or hypothetical protein. Out of 1773 putative intergenic sequences of <em>S. psittacina</em>, 1280 sequences had shown similarity (≥1<em>e</em> −5) to known/hypothetical proteins. We also obtained ncRNA sequences for <em>Arabidopsis</em> and <em>Oryza</em> from ncRNA database<sup><a reveal-id="DSR014C25" data-open="DSR014C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and mapped our intergenic sequences against them. Only few sequences had showed similarity to known ncRNA. Similar patterns were obtained from <em>S. purpurea</em>. The detailed mapping results are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>.</p><p>The paired pattern in functional annotation for all three GO divisions reflects that our library and 454 sequencing covered the transcriptome of the two species equally well. We can speculate about the significance of the GO annotations relative to the pitcher plant insectivorous adaptations. In the biological process and molecular function division, an abundance of genes were found to be related with metabolic process and catalytic activity and inside the metabolic process and catalytic activity, a number of genes were found related to macromolecule metabolic process and hydrolase activity (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S1A, B and S2A, B</a>). The hydrolytic enzymes (protease, RNase, nuclease, phosphatase) may be required for the digestion of prey.<sup><a reveal-id="DSR014C26" data-open="DSR014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The pitchers of pitcher plants may contain water with microorganisms; a high level of hydrolytic enzymes in the pitcher plant transcriptome may favour the hypothesis that pitcher plants do not rely solely on microorganisms to digest their insect prey.<sup><a reveal-id="DSR014C27" data-open="DSR014C27" class="link link-ref link-reveal xref-bibr">27</a></sup> More detailed information about <em>Sarracenia</em>, its prey-digestion system and its microbial food web, can be found in a review by Ellison <em>et al</em>.<sup><a reveal-id="DSR014C28" data-open="DSR014C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>Previously, we estimated the genome sizes (25% more nuclear DNA than maize) of these two species<sup><a reveal-id="DSR014C29" data-open="DSR014C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and based on that, we expected that the species might be polyploid. To test our hypothesis, we estimated the substitution rate for all the paralogous contigs showing a significant sequence similarity at protein level and plotted the <em>K</em><sub>s</sub> value distribution histogram. We were able to detect a moderate signature of a duplication event within this data set. Time estimates suggest that these events were fairly recent. The lack of clear secondary peaks can be attributed to the fact that signal of this event, provided it is relatively recent, is not dissociable from the initial peak. The orthologous contig comparison secondary peak is in the same time frame as the ortholog/paralog duplication peak. Possibly speciation within the genus <em>Sarracenia</em> might have occurred after the duplication event, which is a well-recognized pattern of plant evolution.<sup><a reveal-id="DSR014C30" data-open="DSR014C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>A substitution rate within the orthologous contigs (identified by reciprocal blast) found 491 contigs having the <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio ≥1. This ratio is considered to be a good measure of selective pressure acting at the sequence level<sup><a reveal-id="DSR014C31" data-open="DSR014C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSR014C32" data-open="DSR014C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and has been used in different studies to identify the contigs under positive/adaptive evolution (<em>K</em><sub>a</sub><em>/K</em><sub>s</sub> &gt; 1) or under negative/purifying selection (<em>K</em><sub>a</sub><em>/K</em><sub>s</sub> &lt; 1).<sup><a reveal-id="DSR014C33" data-open="DSR014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The majority of contigs under diversifying selection were found to be related to binding activity in the molecular function category of GO assignment (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>In summary, our analysis showed a high degree of similarity in sequence existed between the pitcher plants, with <em>Vitis</em> as the model species outside the genus with the greatest sequence similarity. Functional annotation of all the contigs showed the major categories of genes that exist, and the substitution rate estimation identified the signatures of genome duplication and rapidly evolving genes in the pitcher plants. We believe that these sequences and analysis will greatly aid the research community working on insectivorous plants and their ecology.</p>                            <h3 scrollto-destination=7730680 id="7730680" class="section-title" data-legacy-id=s4a>4.1. Availability</h3>
<p>The data from the experiments described in this work are available from the NCBI Sequence Read Archive at <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a> under the accessions SRP006675 and SRP006677.</p>                            <h2 scrollto-destination=7730682 id="7730682" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7730684 id="7730684" class="section-title" data-legacy-id=s6>Funding</h2>
<p>We gratefully acknowledge support from NSF grant IIS 0916250 to L.C. and from the University of Georgia Franklin College of Arts &amp; Science's research fund.</p>                            <h2 scrollto-destination=7730686 id="7730686" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR014C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Darwin</div> <div class="given-names">C.</div></div>. </div>, <div class="source">Insectivorous Plants</div> , <div class="year">1875</div><div class="publisher-loc">London</div><div class="publisher-name">Murray</div></div></div></div></div><div content-id="DSR014C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hepburn</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">John</div> <div class="given-names">E.Q.S.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">F.M.</div></div>. </div><div class="article-title">The absorption of nutrients and allied phenomena in the pitchers of the Sarraceniaceae</div>, <div class="source">J. Franklin Inst.</div> , <div class="year">1920</div>, vol. <div class="volume">189</div> (pg. <div class="fpage">147</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20absorption%20of%20nutrients%20and%20allied%20phenomena%20in%20the%20pitchers%20of%20the%20Sarraceniaceae&amp;author=J.S.%20Hepburn&amp;author=E.Q.S.%20John&amp;author=F.M.%20Jones&amp;publication_year=1920&amp;journal=J.%20Franklin%20Inst.&amp;volume=189&amp;pages=147-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0016-0032(20)91579-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0016-0032(20)91579-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0016-0032(20)91579-X"> </span></div> </div></div></div></div></div><div content-id="DSR014C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gallie</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Signal transduction in the carnivorous plant <em>Sarracenia purpurea</em>—regulation of secretory hydrolase expression during development and in response to resources</div>, <div class="source">Plant Physiol.</div> , <div class="year">1997</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">1461</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signal%20transduction%20in%20the%20carnivorous%20plant%20Sarracenia%20purpurea%E2%80%94regulation%20of%20secretory%20hydrolase%20expression%20during%20development%20and%20in%20response%20to%20resources&amp;author=D.R.%20Gallie&amp;author=S.C.%20Chang&amp;publication_year=1997&amp;journal=Plant%20Physiol.&amp;volume=115&amp;pages=1461-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.115.4.1461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.115.4.1461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.115.4.1461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9414556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gotelli</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Ellison</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Food-web models predict species abundances in response to habitat change</div>, <div class="source">Plos Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">1869</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Food-web%20models%20predict%20species%20abundances%20in%20response%20to%20habitat%20change&amp;author=N.J.%20Gotelli&amp;author=A.M.%20Ellison&amp;publication_year=2006&amp;journal=Plos%20Biol.&amp;volume=4&amp;pages=1869-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040324"> </span></div> </div></div></div></div></div><div content-id="DSR014C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnell</div> <div class="given-names">D.E.</div></div>. </div><div class="article-title">Notes on the pollination of Sarracenia-Flava L (Sarraceniaceae) in the Piedmont Province of North-Carolina</div>, <div class="source">Rhodora</div> , <div class="year">1983</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">405</div>-<div class="lpage">20</div>)</div></div></div></div><div content-id="DSR014C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ne'eman</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ne'eman</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ellison</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Limits to reproductive success of <em>Sarracenia purpurea</em> (Sarraceniaceae)</div>, <div class="source">Am. J. Botany</div> , <div class="year">2006</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">1660</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Limits%20to%20reproductive%20success%20of%20Sarracenia%20purpurea%20(Sarraceniaceae)&amp;author=G.%20Ne'eman&amp;author=R.%20Ne'eman&amp;author=A.M.%20Ellison&amp;publication_year=2006&amp;journal=Am.%20J.%20Botany&amp;volume=93&amp;pages=1660-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.93.11.1660" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.93.11.1660" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.93.11.1660"> </span></div> </div></div></div></div></div><div content-id="DSR014C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bouck</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Vision</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The molecular ecologist's guide to expressed sequence tags</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2007</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">907</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20ecologist's%20guide%20to%20expressed%20sequence%20tags&amp;author=A.%20Bouck&amp;author=T.%20Vision&amp;publication_year=2007&amp;journal=Mol.%20Ecol.&amp;volume=16&amp;pages=907-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-294X.2006.03195.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2006.03195.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-294X.2006.03195.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17305850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Sequencing breakthroughs for genomic ecology and evolutionary biology</div>, <div class="source">Mol. Ecol. Res.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">3</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20breakthroughs%20for%20genomic%20ecology%20and%20evolutionary%20biology&amp;author=M.E.%20Hudson&amp;publication_year=2008&amp;journal=Mol.%20Ecol.%20Res.&amp;volume=8&amp;pages=3-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1471-8286.2007.02019.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1471-8286.2007.02019.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1471-8286.2007.02019.x"> </span></div> </div></div></div></div></div><div content-id="DSR014C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rokas</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Abbot</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Harnessing genomics for evolutionary insights</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">192</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Harnessing%20genomics%20for%20evolutionary%20insights&amp;author=A.%20Rokas&amp;author=P.%20Abbot&amp;publication_year=2009&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=24&amp;pages=192-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2008.11.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2008.11.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2008.11.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19201503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sheldon</div> <div class="given-names">B.C.</div></div>. </div><div class="article-title">Genetic basis of fitness differences in natural populations</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">452</div> (pg. <div class="fpage">169</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20basis%20of%20fitness%20differences%20in%20natural%20populations&amp;author=H.%20Ellegren&amp;author=B.C.%20Sheldon&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=169-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06737" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06737" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06737"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18337813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quinlan</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Stromberg</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Marth</div> <div class="given-names">G.T.</div></div>. </div><div class="article-title">Pyrobayes: an improved base caller for SNP discovery in pyrosequences</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">179</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pyrobayes%3A%20an%20improved%20base%20caller%20for%20SNP%20discovery%20in%20pyrosequences&amp;author=A.R.%20Quinlan&amp;author=D.A.%20Stewart&amp;author=M.P.%20Stromberg&amp;author=G.T.%20Marth&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=179-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1172" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1172"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18193056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chevreux</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pfisterer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Drescher</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1147</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20the%20miraEST%20assembler%20for%20reliable%20and%20automated%20mRNA%20transcript%20assembly%20and%20SNP%20detection%20in%20sequenced%20ESTs&amp;author=B.%20Chevreux&amp;author=T.%20Pfisterer&amp;author=B.%20Drescher&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1147-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1917404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1917404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1917404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15140833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Götz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Talon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT—the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blanc</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>. </div><div class="article-title">Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes</div>, <div class="source">Plant Cell</div> , <div class="year">2004</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1667</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20paleopolyploidy%20in%20model%20plant%20species%20inferred%20from%20age%20distributions%20of%20duplicate%20genes&amp;author=G.%20Blanc&amp;author=K.H.%20Wolfe&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=1667-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.021345" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.021345" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.021345"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15208399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larkin</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">N.P.</div></div>, et al. </div><div class="article-title">Clustal W and clustal X version 2.0</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2947</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20clustal%20X%20version%202.0&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suyama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Torrents</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">W609</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAL2NAL%3A%20robust%20conversion%20of%20protein%20sequence%20alignments%20into%20the%20corresponding%20codon%20alignments&amp;author=M.%20Suyama&amp;author=D.%20Torrents&amp;author=P.%20Bork&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W609-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1586</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="collab">The Gene Ontology Consortium</div><div class="article-title">Gene ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">25</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altenhoff</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Dessimoz</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Phylogenetic and functional assessment of orthologs inference projects and methods</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2009</div> <div class="comment"><strong>5</strong>, e1000262. doi:10.1371/journal.pcbi.1000262</div></div></div></div></div><div content-id="DSR014C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koch</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Haubold</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Mitchell-Olds</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Comparative evolutionary analysis of chalcone synthase and alcohol dehydrogenase loci in Arabidopsis, Arabis, and related genera (Brassicaceae)</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1483</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20evolutionary%20analysis%20of%20chalcone%20synthase%20and%20alcohol%20dehydrogenase%20loci%20in%20Arabidopsis%2C%20Arabis%2C%20and%20related%20genera%20(Brassicaceae)&amp;author=M.A.%20Koch&amp;author=B.%20Haubold&amp;author=T.%20Mitchell-Olds&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=1483-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026248" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026248"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11018155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Judd</div> <div class="given-names">S.W.</div></div>,  <div class="name"><div class="surname">Olmstead</div> <div class="given-names">G.R.</div></div>. </div><div class="article-title">A survey of tricolpate (eudicot) phylogenetic relationships</div>, <div class="source">Am. J. Botany</div> , <div class="year">2004</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">1627</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20tricolpate%20(eudicot)%20phylogenetic%20relationships&amp;author=S.W.%20Judd&amp;author=G.R.%20Olmstead&amp;publication_year=2004&amp;journal=Am.%20J.%20Botany&amp;volume=91&amp;pages=1627-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.91.10.1627" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.91.10.1627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.91.10.1627"> </span></div> </div></div></div></div></div><div content-id="DSR014C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Soltis</div> <div class="given-names">S.P.</div></div>,  <div class="name"><div class="surname">Soltis</div> <div class="given-names">E.D.</div></div>. </div><div class="article-title">The origin and diversification of angiosperms</div>, <div class="source">Am. J. Botany</div> , <div class="year">2004</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">1614</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20and%20diversification%20of%20angiosperms&amp;author=S.P.%20Soltis&amp;author=E.D.%20Soltis&amp;publication_year=2004&amp;journal=Am.%20J.%20Botany&amp;volume=91&amp;pages=1614-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.91.10.1614" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.91.10.1614" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.91.10.1614"> </span></div> </div></div></div></div></div><div content-id="DSR014C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mituyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hattori</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">The functional RNA Database 3.0: databases to support mining and annotation of functional RNAs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D89</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20functional%20RNA%20Database%203.0%3A%20databases%20to%20support%20mining%20and%20annotation%20of%20functional%20RNAs&amp;author=T.%20Mituyama&amp;author=K.%20Yamada&amp;author=E.%20Hattori&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D89-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn805" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn805" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn805"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gallie</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Signal transduction in the carnivorous plant <em>Sarracenia purpurea</em>. Regulation of secretory hydrolase expression during development and in response to resources</div>, <div class="source">Plant Physiol.</div> , <div class="year">1997</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">1461</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signal%20transduction%20in%20the%20carnivorous%20plant%20Sarracenia%20purpurea.%20Regulation%20of%20secretory%20hydrolase%20expression%20during%20development%20and%20in%20response%20to%20resources&amp;author=D.R.%20Gallie&amp;author=S.C.%20Chang&amp;publication_year=1997&amp;journal=Plant%20Physiol.&amp;volume=115&amp;pages=1461-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.115.4.1461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.115.4.1461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.115.4.1461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9414556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosa</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Malek</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Qin</div> <div class="given-names">W.</div></div>. </div><div class="article-title">The development of the pitcher plant <em>Sarracenia purpurea</em> into a potentially valuable recombinant protein production system</div>, <div class="source">Biotechnol. Mol. Biol. Rev.</div> , <div class="year">2009</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">105</div>-<div class="lpage">10</div>)</div></div></div></div><div content-id="DSR014C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellison</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Gotelli</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Brewer</div> <div class="given-names">J.S.</div></div>, et al. </div><div class="article-title">The evolutionary ecology of carnivorous plants</div>, <div class="source">Adv. Ecol. Res.</div> , <div class="year">2003</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1</div>-<div class="lpage">74</div>)</div></div></div></div><div content-id="DSR014C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rogers</div> <div class="given-names">L.W.</div></div>,  <div class="name"><div class="surname">Cruse-Sanders</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Determann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Malmberg</div> <div class="given-names">L.R.</div></div>. </div><div class="article-title">Development and characterization of microsatellite markers in Sarracenia L. (pitcher plant) species</div>, <div class="source">Conserv. Genet Resou.</div> , <div class="year">2010</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">75</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20microsatellite%20markers%20in%20Sarracenia%20L.%20(pitcher%20plant)%20species&amp;author=L.W.%20Rogers&amp;author=M.J.%20Cruse-Sanders&amp;author=R.%20Determann&amp;author=L.R.%20Malmberg&amp;publication_year=2010&amp;journal=Conserv.%20Genet%20Resou.&amp;volume=2&amp;pages=75-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12686-009-9165-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12686-009-9165-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12686-009-9165-x"> </span></div> </div></div></div></div></div><div content-id="DSR014C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wood</div> <div class="given-names">T.E.</div></div>,  <div class="name"><div class="surname">Takebayashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Barker</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Mayrose</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Greenspoon</div> <div class="given-names">P.B.</div></div>,  <div class="name"><div class="surname">Rieseberg</div> <div class="given-names">L.H.</div></div>. </div><div class="article-title">The frequency of polyploid speciation in vascular plants</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">13875</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20frequency%20of%20polyploid%20speciation%20in%20vascular%20plants&amp;author=T.E.%20Wood&amp;author=N.%20Takebayashi&amp;author=M.S.%20Barker&amp;author=I.%20Mayrose&amp;author=P.B.%20Greenspoon&amp;author=L.H.%20Rieseberg&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=13875-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0811575106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0811575106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0811575106"> </span></div> </div></div></div></div></div><div content-id="DSR014C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Bielawski</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Statistical methods for detecting molecular adaptation</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">496</div>-<div class="lpage">503</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20methods%20for%20detecting%20molecular%20adaptation&amp;author=Z.%20Yang&amp;author=J.P.%20Bielawski&amp;publication_year=2000&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=15&amp;pages=496-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0169-5347(00)01994-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(00)01994-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0169-5347(00)01994-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11114436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bustamante</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Fledel-Alon</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Williamson</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Natural selection on protein-coding genes in the human genome</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">1153</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20selection%20on%20protein-coding%20genes%20in%20the%20human%20genome&amp;author=C.D.%20Bustamante&amp;author=A.%20Fledel-Alon&amp;author=S.%20Williamson&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=1153-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16237444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR014C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR014C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div>. </div><div class="article-title">Evolutionary genomics and the reach of selection</div>, <div class="source">J. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">8</div> pg. <div class="fpage">12</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20genomics%20and%20the%20reach%20of%20selection&amp;author=L.D.%20Hurst&amp;publication_year=2009&amp;journal=J.%20Biol.&amp;volume=8&amp;pages=12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/jbiol113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fjbiol113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fjbiol113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19291255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=501301&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr014-dsr014supp_table4.xlsx?Expires=2147483647&amp;Signature=X-u~p5h0CThrJCmkdN65awSUuETO-~IIOY6RAynh0wyhrJSFSUoq7qX7lAvH4uZVysiLdEsLzd1lQHolRj2NiNbTxn2se-h3K7dQB~vArjnwwIP0qErHVN1NfVTzi0dTp50AX2vDc2D-e2ClT2SeNYhwjKGCtDOvHZ6HOwwuzI8DXI24ulu6z72xQPaYhZbvrdu6IV9Avk9sCKZK~VCGkxoSED7uyz4reVgiJLUQv8B283JqTVyMDzqr1X-eTR~wVkZJXdNl7EI4uLU4t5GBv5~-FqAaLJxd24Or1f6sPt7Xn7iS6br0DsVZV8l116xznkEhJ5WrxRS8-wsjtV2OTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr014-dsr014supp_table3.xls?Expires=2147483647&amp;Signature=RM~fVNNbUQ4w7AOChvj0QqtdE1eEXNUcLII~lC2qn-YEVTHvTwyPS~BQ57IWVRf2yBzhRKv0tcKDDOX81fezuoDFQj12YhThCPMarh8w10t2JcqtuMm6mIau5beEFpFfrwZjQt6A6OoFl-sdkRIxquRqjy4rbN2QriCEmdwENS4~NjmnbiOB381OPQr6bmbuxhY~m~xVobATQValtFjaOhV~~QUx7xxDwr8xtxgiGLQKjP2G6cAn-drV0owXJishOwriBwxXntTbsvpIaQpl7sr6pUbfzf-gH9SzE73KujuUIRrAyh~9Q7yROzQ~aWRqtcC4sqTq4RifKeZ2vVg0SA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr014-dsr014supp_table2.xls?Expires=2147483647&amp;Signature=VoLfCJ5GDjaScRDBr4Tm2gqc1Q8dqIP08jAKP66Xl-7HNxUxWiGgldzkjQlC-AfIt2jLuO5QtZjVRuGr9LZjEYtQPBo3T7JrI3sWW~qAQjGYP-nsVECAu~eOe2yqkyx~yIDiiQdraMCj-~ivCocIup~lYhtMqcpXau7T5fAhOkm9tGcOAfY7yohC69H27syc8k6y5xfVPvxbYEkiL0Gh6sPO0TgwLoq5wdIK0s3puh82X4yceS8j05daUT4-3-DDciidLO2jW0dO2xzfCLWvQQD0p0Zu-rgS0lk-Ay4swlYbTXaaclyGQVh3AcAndN57LKJ3reyUEw7oX89E237G-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr014-dsr014supp_table1.xls?Expires=2147483647&amp;Signature=Dc10JZHvz0QqJIV-J62QBhB1p4PFH8lxiWVJyj8d41qtLmHVJ4tdKJvTuHbyI2GTWV5ZlEjhgHWzd7x259xzDukoIY5l6DjGm2fDd-nsT04gtc0KwOYXYnXOsUJ~ur-pAHEtGvrjga~btXhvit1SbVJLCn4HgnyOD5J2B1ssl4ImBnzkoRSiVvXebQyQTGIBIpF8cBZFfkT6laFBfOOdBU1L1N2JPZUorxlMuGoFw7Uxz9FCS1J7TockkEI85fKwhlkMLZvPPMkqNePRtEYicUqU1lbK2iUWDmt4gUX7tIBYBajiTLyIgzW1lcqx6JufDpo27Shb9reDjd8r79Mbiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr014-dsr014supp_fig2B.png?Expires=2147483647&amp;Signature=rNYEYrjvcBUk3kdsu0k8YXKmzblD0mPJ~95jNU1BBY~ZzQbTyVZDA148mjwuTSg~cMXiPl7NnzrfEIBUIba7Schu1PRXQt~x4WilawFFY9RHuURktEaPsOxLAcFT0f9P2OfgJFdOYmSa6HebOSLDK7G2F-sm~WTjKvNNMzlPvQWirv89y9WqBAT1bKPtT~WKvDozy10BeceBbgK-e8GWyp50iH1JhcsoK9yMSFOoB4cZCW71V-FclG8pJ2hn2FLThtQOqGPVwYb0gdEuoLef1hyMxbYpd3xLSx-RPfB381GV5lnwMBJe3G9tqXSKt8Gv5OLGEnp41s4KKJtpkaBooQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr014-dsr014supp_fig2A.png?Expires=2147483647&amp;Signature=jcbwEjWSQuVXtvdq4mChnAvUEOKXbI-9kdjDUifaDiTPi4GRKZHBaA64QrMbzSgWm6TibNdiUjkYejUMLfkahpQJtqwvS~u8UDol2p1n5G1Is02Vcc-d5iTEl5F~v~-yo-YPRFb0GZJz5p6N1WG~3~cu8QAP03ro7lZqViPw2JpEwChaLf~lMHko~BvfRRt4nRd0erMAAUvBxRISgX0cS450Or8UcMie1hhR-t8qQPeiv1swmPVb0-4lwWqJP9S2x6zwnb5qy3AWvjkHYEV8rqjonNsxFU~-ncTmwWdilwb2CdPdf5J2KxTXY1O8j6d-haY0MHYtPEc28u3MdZKl6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr014-dsr014supp_fig1B.png?Expires=2147483647&amp;Signature=nfTXUw1TRghh3EsF9tS8f-fEYQEnYjQCYZkcoOK49Xuvq-0S9xS0JRQfhiMQLzBZU4j-b4KIa4mhXRbBYvwm2ugeMFdhoEv0HBayXw9QlgWG7fG6cLTxCOoBQCLZRmXx0i79HSRKt6pxbfwOAyAEo9G42zQlGWPlJCbJO2Da2S8uZAQLoWnhxWdnenXbWQkUcxWcTUnxcqqZ9Af4Fp8sl30xkvijWRczFlKdTW9gpcNTDpvUkCe9djUXtt1hVtrFTmZYahPY5yGXALELSKuOEkzRbf-pBw88rwL2TcEPfcdN6VxICG8~LYcDpohELljB9IqlBmXkkT2yV6RQUdKOsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr014/2/dsr014-dsr014supp_fig1A.png?Expires=2147483647&amp;Signature=sB4-kAwBMQ~jbpNGhraJjoyvyxuhtWSn550Az8OKiEXYyRtxLWSA40hQRN4jO3BnHgQkyHvVIedXbZK4w1S6fwOzkSpF5tLSQjigYXsva8ShivKjkIngKkduIOdVXJ9Q1xtNfDxZaQcJqaS7ejNpRmzHrKTy3W99B6oMujmxw3syKbESEepeuz2AgMMyd01vhM9-kemM09P1kuu8RuB67CIypBDDd~yX-J6-XGiHrVeWBKkAa9kGvfD6H9oBvBPkW-DHTAawHqZWRmIlCEVlRkdlzLc7WcScC9GH2nOTRqd85HIvRf2fFXGO1GKiufSgnNNCiv~VnmIGpEetpRxMHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - png file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">816</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">22</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="501301"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f4%2f253%2f501301">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000294936100005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (22)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F4%2F253%2F501301" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/501301'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Transcriptome Analysis of Sarracenia, an Insectivorous Plant" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f253%2f501301">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

