

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/4/407/446188","@type":"ScholarlyArticle","name":"Evolutionary Origin of Higher-Order Repeat Structure in Alpha-Satellite DNA of Primate Centromeres","datePublished":"2014-02-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2014-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu005","keywords":["alpha-satellite DNA","higher-order repeat","centromere","hominoids","evolutionary origin"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/4/407/446188","author":[{"name":"Koga, Akihiko","affiliation":"1  Primate Research Institute, Kyoto University ,  Inuyama City 484-8506 ,  Japan","@type":"Person"},{"name":"Hirai, Yuriko","affiliation":"1  Primate Research Institute, Kyoto University ,  Inuyama City 484-8506 ,  Japan","@type":"Person"},{"name":"Terada, Shoko","affiliation":"1  Primate Research Institute, Kyoto University ,  Inuyama City 484-8506 ,  Japan","@type":"Person"},{"name":"Jahan, Israt","affiliation":"1  Primate Research Institute, Kyoto University ,  Inuyama City 484-8506 ,  Japan","@type":"Person"},{"name":"Baicharoen, Sudarath","affiliation":"2  Bureau of Conservation Research and Education ,  Zoological Park Organization ,  Bangkok 10300 ,  Thailand","@type":"Person"},{"name":"Arsaithamkul, Visit","affiliation":"2  Bureau of Conservation Research and Education ,  Zoological Park Organization ,  Bangkok 10300 ,  Thailand","@type":"Person"},{"name":"Hirai, Hirohisa","affiliation":"1  Primate Research Institute, Kyoto University ,  Inuyama City 484-8506 ,  Japan","@type":"Person"}],"description":"Abstract.  Alpha-satellite DNA (AS) is a main DNA component of primate centromeres, consisting of tandemly repeated units of ∼170 bp. The AS of humans contains ","pageStart":"407","pageEnd":"415","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Evolutionary Origin of Higher-Order Repeat Structure in Alpha-Satellite DNA of Primate Centromeres","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Evolutionary Origin of Higher-Order Repeat Structure in Alpha-Satellite DNA of Primate Centromeres" />
    <meta property="og:description" content="Abstract.  Alpha-satellite DNA (AS) is a main DNA component of primate centromeres, consisting of tandemly repeated units of ∼170 bp. The AS of humans contains " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/4/407/446188" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-02-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Koga, Akihiko" /><meta name="citation_author_institution" content="Primate Research Institute, Kyoto University, Inuyama City 484-8506, Japan" /><meta name="citation_author" content="Hirai, Yuriko" /><meta name="citation_author_institution" content="Primate Research Institute, Kyoto University, Inuyama City 484-8506, Japan" /><meta name="citation_author" content="Terada, Shoko" /><meta name="citation_author_institution" content="Primate Research Institute, Kyoto University, Inuyama City 484-8506, Japan" /><meta name="citation_author" content="Jahan, Israt" /><meta name="citation_author_institution" content="Primate Research Institute, Kyoto University, Inuyama City 484-8506, Japan" /><meta name="citation_author" content="Baicharoen, Sudarath" /><meta name="citation_author_institution" content="Bureau of Conservation Research and Education, Zoological Park Organization, Bangkok 10300, Thailand" /><meta name="citation_author" content="Arsaithamkul, Visit" /><meta name="citation_author_institution" content="Bureau of Conservation Research and Education, Zoological Park Organization, Bangkok 10300, Thailand" /><meta name="citation_author" content="Hirai, Hirohisa" /><meta name="citation_author_institution" content="Primate Research Institute, Kyoto University, Inuyama City 484-8506, Japan" /><meta name="citation_title" content="Evolutionary Origin of Higher-Order Repeat Structure in Alpha-Satellite DNA of Primate Centromeres" /><meta name="citation_firstpage" content="407" /><meta name="citation_lastpage" content="415" /><meta name="citation_doi" content="10.1093/dnares/dsu005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2014/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Evolution of alpha satellite; Curr. Opin. Genet. Dev.;  citation_year=1991;  citation_volume=1;  citation_pages=509-14; " /><meta name="citation_reference" content="citation_title=The evolutionary dynamics of alpha-satellite; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=88-96; " /><meta name="citation_reference" content="citation_title=Chromosome-specific organization of human alpha satellite DNA; Am. J. Hum. Genet.;  citation_year=1985;  citation_volume=37;  citation_pages=524-32; " /><meta name="citation_reference" content="citation_title=Alpha-satellite DNA of primates: old and new families; Chromosoma;  citation_year=2001;  citation_volume=110;  citation_pages=253-66; " /><meta name="citation_reference" content="citation_title=Analysis of the centromeric regions of the human genome assembly; Trends Genet.;  citation_year=2004;  citation_volume=20;  citation_pages=529-33; " /><meta name="citation_reference" content="citation_title=Co-localization of centromere activity, proteins and topoisomerase II within a subdomain of the major human X α-satellite array; EMBO J.;  citation_year=2002;  citation_volume=21;  citation_pages=5269-80; " /><meta name="citation_reference" content="citation_title=Orangutan alpha-satellite monomers are closely related to the human consensus sequence; Mamm. Genome;  citation_year=1998;  citation_volume=9;  citation_pages=440-7; " /><meta name="citation_reference" content="citation_title=Concerted evolution of alpha satellite DNA: evidence for species specificity and a general lack of sequence conservation among alphoid sequences of higher primates; Chromosoma;  citation_year=1989;  citation_volume=98;  citation_pages=273-9; " /><meta name="citation_reference" content="citation_title=Higher-order repeat structure in alpha satellite DNA is an attribute of hominoids rather than hominids; J. Hum. Genet.;  citation_year=2013; " /><meta name="citation_reference" content="citation_title=Repetitive sequences originating from the centromere constitute large-scale heterochromatin in the telomere region in the siamang, a small ape; Heredity;  citation_year=2012;  citation_volume=109;  citation_pages=180-7; " /><meta name="citation_reference" content="citation_title=A subterminal satellite located adjacent to telomeres in chimpanzees is absent from the human genome; Nat. Genet.;  citation_year=1994;  citation_volume=6;  citation_pages=52-6; " /><meta name="citation_reference" content="citation_title=Evolution of subterminal satellite (StSat) repeats in hominids; Genetica;  citation_year=2011;  citation_volume=139;  citation_pages=167-75; " /><meta name="citation_reference" content="citation_title=Finishing the euchromatic sequence of the human genome; citation_author=International Human Genome Sequencing Consortium; Nature;  citation_year=2004;  citation_volume=431;  citation_pages=931-45; " /><meta name="citation_reference" content="citation_title=An assessment of the sequence gaps: unfinished business in a finished human genome; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=345-54; " /><meta name="citation_reference" content="citation_title=Asian primate classification; Int. J. Primatol.;  citation_year=2004;  citation_volume=25;  citation_pages=97-164; " /><meta name="citation_reference" content="citation_title=Mitochondrial evidence for multiple radiations in the evolutionary history of small apes; BMC Evol. Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=74" /><meta name="citation_reference" content="citation_title=Two types of alpha satellite DNA in distinct chromosomal locations in Azara's owl monkey; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=235-40; " /><meta name="citation_reference" content="citation_title=Heterochromatin blocks constituting the entire short arms of acrocentric chromosomes of Azara's owl monkey: formation processes inferred from chromosomal locations; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=461-70; " /><meta name="citation_reference" content="citation_title=Unidirectional digestion with exonuclease III creates targeted breakpoints for DNA sequencing; Gene;  citation_year=1984;  citation_volume=28;  citation_pages=351-9; " /><meta name="citation_reference" content="citation_title=In situ hybridization analysis of gibbon chromosomes suggests that amplification of alpha satellite DNA in the telomere region is confined to two of the four genera; Genome;  citation_year=2012;  citation_volume=55;  citation_pages=809-12; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=Definition of a new alpha satellite suprachromosomal family characterized by monomeric organization; Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=2209-15; " /><meta name="citation_reference" content="citation_title=New insights into centromere organization and evolution from the white-cheeked gibbon and marmoset; Mol. Biol. Evol.;  citation_year=2009;  citation_volume=26;  citation_pages=1889-900; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/4/407/1220536/dsu005.pdf" /><meta name="description" content="Abstract.  Alpha-satellite DNA (AS) is a main DNA component of primate centromeres, consisting of tandemly repeated units of ∼170 bp. The AS of humans contains " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/4/407/446188" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/4/407/446188" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/4/407/446188");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu005';
    </script>
<title>
	Evolutionary Origin of Higher-Order Repeat Structure in Alpha-Satellite DNA of Primate Centromeres | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./446188" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfM8PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2dhLCBBa2loaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUHJpbWF0ZSBSZXNlYXJjaCBJbnN0aXR1dGUsIEt5b3RvIFVuaXZlcnNpdHksIEludXlhbWEgQ2l0eSA0ODQtODUwNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIaXJhaSwgWXVyaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUHJpbWF0ZSBSZXNlYXJjaCBJbnN0aXR1dGUsIEt5b3RvIFVuaXZlcnNpdHksIEludXlhbWEgQ2l0eSA0ODQtODUwNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUZXJhZGEsIFNob2tvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUHJpbWF0ZSBSZXNlYXJjaCBJbnN0aXR1dGUsIEt5b3RvIFVuaXZlcnNpdHksIEludXlhbWEgQ2l0eSA0ODQtODUwNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKYWhhbiwgSXNyYXQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQcmltYXRlIFJlc2VhcmNoIEluc3RpdHV0ZSwgS3lvdG8gVW5pdmVyc2l0eSwgSW51eWFtYSBDaXR5IDQ4NC04NTA2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJhaWNoYXJvZW4sIFN1ZGFyYXRoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQnVyZWF1IG9mIENvbnNlcnZhdGlvbiBSZXNlYXJjaCBhbmQgRWR1Y2F0aW9uLCBab29sb2dpY2FsIFBhcmsgT3JnYW5pemF0aW9uLCBCYW5na29rIDEwMzAwLCBUaGFpbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFyc2FpdGhhbWt1bCwgVmlzaXQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCdXJlYXUgb2YgQ29uc2VydmF0aW9uIFJlc2VhcmNoIGFuZCBFZHVjYXRpb24sIFpvb2xvZ2ljYWwgUGFyayBPcmdhbml6YXRpb24sIEJhbmdrb2sgMTAzMDAsIFRoYWlsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWksIEhpcm9oaXNhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUHJpbWF0ZSBSZXNlYXJjaCBJbnN0aXR1dGUsIEt5b3RvIFVuaXZlcnNpdHksIEludXlhbWEgQ2l0eSA0ODQtODUwNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkV2b2x1dGlvbmFyeSBPcmlnaW4gb2YgSGlnaGVyLU9yZGVyIFJlcGVhdCBTdHJ1Y3R1cmUgaW4gQWxwaGEtU2F0ZWxsaXRlIEROQSBvZiBQcmltYXRlIENlbnRyb21lcmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0MTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBhbHBoYSBzYXRlbGxpdGU7IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz01MDktMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb25hcnkgZHluYW1pY3Mgb2YgYWxwaGEtc2F0ZWxsaXRlOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTg4LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21lLXNwZWNpZmljIG9yZ2FuaXphdGlvbiBvZiBodW1hbiBhbHBoYSBzYXRlbGxpdGUgRE5BOyBBbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz01MjQtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFscGhhLXNhdGVsbGl0ZSBETkEgb2YgcHJpbWF0ZXM6IG9sZCBhbmQgbmV3IGZhbWlsaWVzOyBDaHJvbW9zb21hOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTI1My02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGNlbnRyb21lcmljIHJlZ2lvbnMgb2YgdGhlIGh1bWFuIGdlbm9tZSBhc3NlbWJseTsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTUyOS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q28tbG9jYWxpemF0aW9uIG9mIGNlbnRyb21lcmUgYWN0aXZpdHksIHByb3RlaW5zIGFuZCB0b3BvaXNvbWVyYXNlIElJIHdpdGhpbiBhIHN1YmRvbWFpbiBvZiB0aGUgbWFqb3IgaHVtYW4gWCDOsS1zYXRlbGxpdGUgYXJyYXk7IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz01MjY5LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmFuZ3V0YW4gYWxwaGEtc2F0ZWxsaXRlIG1vbm9tZXJzIGFyZSBjbG9zZWx5IHJlbGF0ZWQgdG8gdGhlIGh1bWFuIGNvbnNlbnN1cyBzZXF1ZW5jZTsgTWFtbS4gR2Vub21lOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00NDAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uY2VydGVkIGV2b2x1dGlvbiBvZiBhbHBoYSBzYXRlbGxpdGUgRE5BOiBldmlkZW5jZSBmb3Igc3BlY2llcyBzcGVjaWZpY2l0eSBhbmQgYSBnZW5lcmFsIGxhY2sgb2Ygc2VxdWVuY2UgY29uc2VydmF0aW9uIGFtb25nIGFscGhvaWQgc2VxdWVuY2VzIG9mIGhpZ2hlciBwcmltYXRlczsgQ2hyb21vc29tYTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTI3My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoZXItb3JkZXIgcmVwZWF0IHN0cnVjdHVyZSBpbiBhbHBoYSBzYXRlbGxpdGUgRE5BIGlzIGFuIGF0dHJpYnV0ZSBvZiBob21pbm9pZHMgcmF0aGVyIHRoYW4gaG9taW5pZHM7IEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBldGl0aXZlIHNlcXVlbmNlcyBvcmlnaW5hdGluZyBmcm9tIHRoZSBjZW50cm9tZXJlIGNvbnN0aXR1dGUgbGFyZ2Utc2NhbGUgaGV0ZXJvY2hyb21hdGluIGluIHRoZSB0ZWxvbWVyZSByZWdpb24gaW4gdGhlIHNpYW1hbmcsIGEgc21hbGwgYXBlOyBIZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xODAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdWJ0ZXJtaW5hbCBzYXRlbGxpdGUgbG9jYXRlZCBhZGphY2VudCB0byB0ZWxvbWVyZXMgaW4gY2hpbXBhbnplZXMgaXMgYWJzZW50IGZyb20gdGhlIGh1bWFuIGdlbm9tZTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTUyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBzdWJ0ZXJtaW5hbCBzYXRlbGxpdGUgKFN0U2F0KSByZXBlYXRzIGluIGhvbWluaWRzOyBHZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzk7ICBjaXRhdGlvbl9wYWdlcz0xNjctNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmlzaGluZyB0aGUgZXVjaHJvbWF0aWMgc2VxdWVuY2Ugb2YgdGhlIGh1bWFuIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUludGVybmF0aW9uYWwgSHVtYW4gR2Vub21lIFNlcXVlbmNpbmcgQ29uc29ydGl1bTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQzMTsgIGNpdGF0aW9uX3BhZ2VzPTkzMS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYXNzZXNzbWVudCBvZiB0aGUgc2VxdWVuY2UgZ2FwczogdW5maW5pc2hlZCBidXNpbmVzcyBpbiBhIGZpbmlzaGVkIGh1bWFuIGdlbm9tZTsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzQ1LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc2lhbiBwcmltYXRlIGNsYXNzaWZpY2F0aW9uOyBJbnQuIEouIFByaW1hdG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTk3LTE2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWl0b2Nob25kcmlhbCBldmlkZW5jZSBmb3IgbXVsdGlwbGUgcmFkaWF0aW9ucyBpbiB0aGUgZXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2Ygc21hbGwgYXBlczsgQk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gdHlwZXMgb2YgYWxwaGEgc2F0ZWxsaXRlIEROQSBpbiBkaXN0aW5jdCBjaHJvbW9zb21hbCBsb2NhdGlvbnMgaW4gQXphcmEncyBvd2wgbW9ua2V5OyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTIzNS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGV0ZXJvY2hyb21hdGluIGJsb2NrcyBjb25zdGl0dXRpbmcgdGhlIGVudGlyZSBzaG9ydCBhcm1zIG9mIGFjcm9jZW50cmljIGNocm9tb3NvbWVzIG9mIEF6YXJhJ3Mgb3dsIG1vbmtleTogZm9ybWF0aW9uIHByb2Nlc3NlcyBpbmZlcnJlZCBmcm9tIGNocm9tb3NvbWFsIGxvY2F0aW9uczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz00NjEtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaWRpcmVjdGlvbmFsIGRpZ2VzdGlvbiB3aXRoIGV4b251Y2xlYXNlIElJSSBjcmVhdGVzIHRhcmdldGVkIGJyZWFrcG9pbnRzIGZvciBETkEgc2VxdWVuY2luZzsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk4NDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTM1MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiBzaXR1IGh5YnJpZGl6YXRpb24gYW5hbHlzaXMgb2YgZ2liYm9uIGNocm9tb3NvbWVzIHN1Z2dlc3RzIHRoYXQgYW1wbGlmaWNhdGlvbiBvZiBhbHBoYSBzYXRlbGxpdGUgRE5BIGluIHRoZSB0ZWxvbWVyZSByZWdpb24gaXMgY29uZmluZWQgdG8gdHdvIG9mIHRoZSBmb3VyIGdlbmVyYTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9ODA5LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNTogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB1c2luZyBtYXhpbXVtIGxpa2VsaWhvb2QsIGV2b2x1dGlvbmFyeSBkaXN0YW5jZSwgYW5kIG1heGltdW0gcGFyc2ltb255IG1ldGhvZHM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yNzMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmluaXRpb24gb2YgYSBuZXcgYWxwaGEgc2F0ZWxsaXRlIHN1cHJhY2hyb21vc29tYWwgZmFtaWx5IGNoYXJhY3Rlcml6ZWQgYnkgbW9ub21lcmljIG9yZ2FuaXphdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjIwOS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV3IGluc2lnaHRzIGludG8gY2VudHJvbWVyZSBvcmdhbml6YXRpb24gYW5kIGV2b2x1dGlvbiBmcm9tIHRoZSB3aGl0ZS1jaGVla2VkIGdpYmJvbiBhbmQgbWFybW9zZXQ7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xODg5LTkwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzQvNDA3LzEyMjA1MzYvZHN1MDA1LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBbHBoYS1zYXRlbGxpdGUgRE5BIChBUykgaXMgYSBtYWluIEROQSBjb21wb25lbnQgb2YgcHJpbWF0ZSBjZW50cm9tZXJlcywgY29uc2lzdGluZyBvZiB0YW5kZW1seSByZXBlYXRlZCB1bml0cyBvZiDiiLwxNzAgYnAuIFRoZSBBUyBvZiBodW1hbnMgY29udGFpbnMgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NDYxODgmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkXIqIHuMjVG+xV2CQ8rmjYE/gPDs=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f407%2f446188" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f407%2f446188" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17419" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&amp;Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/4">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6676323">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676323">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6676325">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676325">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6676330">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676330">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6676339">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676339">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6676377">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676377">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6676386">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676386">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6676388">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676388">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6676390">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676390">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6676392">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6676392">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/4/397/446522">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/4/417/446835">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Evolutionary Origin of Higher-Order Repeat Structure in Alpha-Satellite DNA of Primate Centromeres
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akihiko Koga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Primate Research Institute, Kyoto University</div>, <div class="addr-line">Inuyama City 484-8506</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div> To whom correspondence should be addressed. Tel. +81 568-63-0567. Fax. +81 568-63-0085. Email: <a href="mailto:koga.akihiko.5n@kyoto-u.ac.jp" target="_blank">koga.akihiko.5n@kyoto-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akihiko+Koga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koga A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koga A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akihiko Koga</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuriko Hirai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Primate Research Institute, Kyoto University</div>, <div class="addr-line">Inuyama City 484-8506</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuriko+Hirai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirai Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirai Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuriko Hirai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shoko Terada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Primate Research Institute, Kyoto University</div>, <div class="addr-line">Inuyama City 484-8506</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shoko+Terada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Terada S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Terada S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shoko Terada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Israt Jahan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Primate Research Institute, Kyoto University</div>, <div class="addr-line">Inuyama City 484-8506</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Israt+Jahan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jahan I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jahan I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Israt Jahan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sudarath Baicharoen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bureau of Conservation Research and Education</div>, <div class="insititution">Zoological Park Organization</div>, <div class="addr-line">Bangkok 10300</div>, <div class="country">Thailand</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sudarath+Baicharoen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Baicharoen S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Baicharoen S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sudarath Baicharoen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Visit Arsaithamkul</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bureau of Conservation Research and Education</div>, <div class="insititution">Zoological Park Organization</div>, <div class="addr-line">Bangkok 10300</div>, <div class="country">Thailand</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Visit+Arsaithamkul">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arsaithamkul V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arsaithamkul V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Visit Arsaithamkul</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirohisa Hirai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Primate Research Institute, Kyoto University</div>, <div class="addr-line">Inuyama City 484-8506</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirohisa+Hirai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirai H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirai H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hirohisa Hirai</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Mitsuo Oshimura</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 407–415, <a href='https://doi.org/10.1093/dnares/dsu005'>https://doi.org/10.1093/dnares/dsu005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 February 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 October 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">27 January 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="446188" href="/dnaresearch/article-pdf/21/4/407/1220536/dsu005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Akihiko Koga, Yuriko Hirai, Shoko Terada, Israt Jahan, Sudarath Baicharoen, Visit Arsaithamkul, Hirohisa Hirai;  Evolutionary Origin of Higher-Order Repeat Structure in Alpha-Satellite DNA of Primate Centromeres, <em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 407–415, <a href='https://doi.org/10.1093/dnares/dsu005'>https://doi.org/10.1093/dnares/dsu005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/446188?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/446188?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/446188?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/446188?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/446188?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/446188?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/446188?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Evolutionary%20Origin%20of%20Higher-Order%20Repeat%20Structure%20in%20Alpha-Satellite%20DNA%20of%20Primate%20Centromeres&amp;publicationDate=2014-02-27&amp;volumeNum=21&amp;issueNum=4&amp;author=Koga%2C%20Akihiko%3B%20Hirai%2C%20Yuriko&amp;startPage=407&amp;endPage=415&amp;contentId=10.1093%2Fdnares%2Fdsu005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f407%2f446188" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6676323 id="6676323" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alpha-satellite DNA (AS) is a main DNA component of primate centromeres, consisting of tandemly repeated units of ∼170 bp. The AS of humans contains sequences organized into higher-order repeat (HOR) structures, in which a block of multiple repeat units forms a larger repeat unit and the larger units are repeated tandemly. The presence of HOR in AS is widely thought to be unique to hominids (family Hominidae; humans and great apes). Recently, we have identified an HOR-containing AS in the siamang, which is a small ape species belonging to the genus <em>Symphalangus</em> in the family Hylobatidae. This result supports the view that HOR in AS is an attribute of hominoids (superfamily Hominoidea) rather than hominids. A single example is, however, not sufficient for discussion of the evolutionary origin of HOR-containing AS. In the present study, we developed an efficient method for detecting signs of large-scale HOR and demonstrated HOR of AS in all the three other genera. Thus, AS organized into HOR occurs widely in hominoids. Our results indicate that (i) HOR-containing AS was present in the last common ancestor of hominoids or (ii) HOR-containing AS emerged independently in most or all basal branches of hominoids. We have also confirmed HOR occurrence in centromeric AS in the Hylobatidae family, which remained unclear in our previous study because of the existence of AS in subtelomeric regions, in addition to centromeres, of siamang chromosomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alpha-satellite DNA&quot;">alpha-satellite DNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;higher-order repeat&quot;">higher-order repeat</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="centromere">centromere</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="hominoids">hominoids</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;evolutionary origin&quot;">evolutionary origin</a></div>                            <h2 scrollto-destination=6676325 id="6676325" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The centromere is the part of the chromosome to which spindle fibres attach via the kinetochore, and it serves as the forefront of chromosome migration during cell division. The main DNA components of centromeres are usually tandemly repeated sequences, and the most abundant sequence in primate centromeres is alpha-satellite DNA (AS), which has basic repeat units of ∼170 bp in length.<sup><a reveal-id="DSU005C1" data-open="DSU005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU005C2" data-open="DSU005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Human AS has been classified into two types according to the organization and nucleotide sequence properties of its repeat units.<sup><a reveal-id="DSU005C3" data-open="DSU005C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU005C4" data-open="DSU005C4" class="link link-ref link-reveal xref-bibr">4</a></sup> One is monomeric AS, which is a simple repetition of the basic repeat units. The other is higher-order AS, whose sequence is organized into a higher-order repeat (HOR) structure, in which a block of multiple basic repeat units forms a larger repeat unit. In a typical HOR structure, any combination of two basic repeat units within the larger repeat unit shows nucleotide sequence identities of 70–90%, whereas the identity often exceeds 95% between a basic repeat unit and its counterpart (located at the same position) in another larger unit. Higher-order AS is known to be present in all normal human centromeres,<sup><a reveal-id="DSU005C4" data-open="DSU005C4" class="link link-ref link-reveal xref-bibr">4</a></sup> to occupy central portions of centromeres (whereas monomeric AS is found mostly in peripheral regions),<sup><a reveal-id="DSU005C5" data-open="DSU005C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and to be more important than monomeric AS in centromere function.<sup><a reveal-id="DSU005C3" data-open="DSU005C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU005C4" data-open="DSU005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSU005C6" data-open="DSU005C6" class="link link-ref link-reveal xref-bibr">6</a></sup> It is unclear, however, what aspects of higher-order AS are associated with these facts. One approach to identifying these associations would be to clarify the history of higher-order AS, including its evolutionary origin and expansion. The present study addresses the evolutionary origin of higher-order AS.</p><p>Humans (genus <em>Homo</em>) are included in hominids (family Hominidae), along with chimpanzees (genus <em>Pan</em>), gorillas (genus <em>Gorilla</em>), and other great apes. Small apes (gibbons) are not included in hominids and form another taxon (family Hylobatidae). Collectively, organisms included in these two families are called hominoids (superfamily Hominoidea). All hominids examined to date carry higher-order AS.<sup><a reveal-id="DSU005C7" data-open="DSU005C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSU005C8" data-open="DSU005C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In small apes, however, AS of this type had not been found until we recently reported one in the siamang (<em>Sympha-langus syndactylus</em>).<sup><a reveal-id="DSU005C9" data-open="DSU005C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>Although the HOR structure observed in siamang AS was clear (HOR units consisting of four or six basic repeat units), there remained two unresolved questions: (i) whether HOR structures are common among small apes or are limited to the siamang and (ii) whether AS residing in the centromere (without migrating to another chromosomal position) exhibits HOR structures or if HOR is limited to non-centromeric AS in small apes. The second question arose because of the fact that the siamang carries AS in terminal regions of chromosomes as large blocks of constitutive heterochromatin as well as centromeres.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The function and formation processes of these heterochromatin blocks are not yet clear. Similar structures have also been observed in chromosomes of chimpanzees and gorillas. However, their DNA component is not AS, but is an unrelated tandem repeat sequence (called the StSat sequence) comprising 32-bp repeat units.<sup><a reveal-id="DSU005C11" data-open="DSU005C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSU005C12" data-open="DSU005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In the present study, we obtained results that can be used to answer the two questions described above. Our results indicate that higher-order AS widely occurs in hominoids.</p><p>Structures of tandem repeat sequences are difficult to accurately determine using data from whole-genome shotgun sequencing or next-generation sequencing, because of difficulty in assembling sequence reads into contigs. Primer walking on a cloned DNA fragment is also inadequate because of the presence of multiple primer annealing sites in the clone. In fact, centromere regions are left as large gaps even in the human genome database.<sup><a reveal-id="DSU005C13" data-open="DSU005C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSU005C14" data-open="DSU005C14" class="link link-ref link-reveal xref-bibr">14</a></sup> To overcome this problem, we used a traditional method consisting of cloning long genomic DNA fragments, preparation of deletion clones, and sequencing end regions of these clones. There was, however, another problem to be solved before sequencing analyses: which clones should be sequenced? Our idea was to detect signs of HOR structures by comparing sequences of two different regions within a clone. For this purpose, we used the transposition reaction of a bacterial transposon, which provided AS sequences with unique sites for annealing by sequencing primers. Below, we describe our methods, results, and their significance with regard to the evolutionary origin of higher-order AS.</p>                            <h2 scrollto-destination=6676330 id="6676330" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6676331 id="6676331" class="section-title" data-legacy-id=s2a>2.1. Animals for collection of genomic DNA</h3>
<p>In the latest taxonomy of small apes, the family Hylobatidae consists of four genera: <em>Hoolock</em>, <em>Hylobates</em>, <em>Nomascus</em>, and <em>Symphalangus</em>.<sup><a reveal-id="DSU005C15" data-open="DSU005C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSU005C16" data-open="DSU005C16" class="link link-ref link-reveal xref-bibr">16</a></sup> A species of the genus <em>Symphalangus</em> (<em>S. syndactylus</em>) has been examined in a previous study.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In the present study, we focused on the other three genera. The species and the origin of the individuals that served as DNA sources were as follows: the Western hoolock gibbon (<em>Hoolock hoolock</em>), an adult female (named Faisal) bred at Bangabandhu Sheikh Mujib Safari Park, Bangladesh; the agile gibbon (<em>Hylobates agilis</em>), an adult male (named Raja) bred at the Primate Research Institute of Kyoto University; and the Southern white-cheeked gibbon (<em>Nomascus siki</em>), an adult female (identification number 001064376) bred at Khao Kheow Forest and Wildlife Park, Thailand.</p>                            <h3 scrollto-destination=6676333 id="6676333" class="section-title" data-legacy-id=s2b>2.2. Preparation of genomic libraries</h3>
<p>We collected genomic DNA from white blood cells by following a standard method. A genomic library of each species was constructed, following a previously described protocol.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU005C17" data-open="DSU005C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The vector was an 8.1-kb fosmid pCC1FOS, and the inserts were 40- to 44-kb genomic DNA fragments that were generated by mechanical shearing and isolated by gel electrophoresis and subsequent recovery from a gel piece. Identification of AS-containing sequences from these libraries is described in the Results section together with strategies and results.</p>                            <h3 scrollto-destination=6676335 id="6676335" class="section-title" data-legacy-id=s2c>2.3. DNA sequencing</h3>
<p>We performed subcloning of a part of the fosmid clones, preparation of a series of deletion clones from the subclone plasmids and sequenced the deletion clones by using the M13 universal primer. The details of our strategy are described in the Results section along with the results. The sequencing method used was Sanger's method, which involved using an Applied Biosystems 3730<em>xl</em> DNA Analyzer.</p>                            <h3 scrollto-destination=6676337 id="6676337" class="section-title" data-legacy-id=s2d>2.4. Fluorescent <em>in situ</em> hybridization analysis of chromosomes</h3>
<p>We performed fluorescent <em>in situ</em> hybridization (FISH) analysis following the procedures described previously.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Specific conditions are explained in each case.</p>                            <h2 scrollto-destination=6676339 id="6676339" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6676340 id="6676340" class="section-title" data-legacy-id=s3a>3.1. Collection of genomic DNA clones containing repetitive sequences</h3>
<p>We screened the genomic libraries of the three gibbon species for repetitive sequences by the genomic hybridization method described previously.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU005C17" data-open="DSU005C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In this screening, the target of the hybridization was fosmid clones randomly selected from the library and fixed on nylon membranes. The probe was genomic DNA (fragmented by mechanical shearing in advance) of the same species as that of the target DNA. For example, fragmented genomic DNA of <em>Hoolock hoolock</em> was chemically labelled and used as probe in the hybridization assay for <em>Hoolock hoolock</em> fosmid clones. If a fosmid carries repetitive DNA (such as AS) as its insert, the same sequence occupies a large fraction of the probe DNA. A large number of labelled DNA fragments anneal to this fosmid, and the fosmid produces an intense signal. If a fosmid clone carries a single-copy region, the same sequence is present at a low frequency in the labelled DNA, and the fosmid displays a weak (or even undetectable) signal. A big advantage of this method is that any repetitive sequences are identified irrespective of their nucleotide sequences (even if the sequences are unknown), with more chances of identification (stronger signals and larger numbers of signals) for sequences of higher copy numbers.<sup><a reveal-id="DSU005C18" data-open="DSU005C18" class="link link-ref link-reveal xref-bibr">18</a></sup> For each species, we screened 384 fosmid clones (four membranes corresponding to four 96-well plates) and picked up 12 clones exhibiting the highest level of signal intensities.</p>                            <h3 scrollto-destination=6676342 id="6676342" class="section-title" data-legacy-id=s3b>3.2. Selection of AS-containing clones</h3>
<p>We sequenced one end of the clones obtained by genomic hybridization and selected clones whose sequence reads showed tandem repeat structures with unit sizes between 100 and 200 bp (the size of the basic repeat units of AS is uniform among hominoids at ∼170 bp and is included in this range). Of the 12 <em>Hoolock</em>, 12 <em>Hylobates</em>, and 12 <em>Nomascus</em> clones, 4, 9, and 6 clones, respectively, met this criterion. Sequence reads from all of these clones exhibited &gt;70% nucleotide identity with the consensus sequence of human AS and with that of siamang AS.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p>                            <h3 scrollto-destination=6676344 id="6676344" class="section-title" data-legacy-id=s3c>3.3. Detection of signs of HOR</h3>
<p>We first tried to detect HOR signs by comparing the sequences of two terminal regions (∼900 bp each), which we had successfully carried out in the case of <em>Symphalangus</em> in our previous study.<sup><a reveal-id="DSU005C9" data-open="DSU005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> This test resulted in an HOR sign in one (HylFos15) of the clones from <em>Hylobates</em> (data not shown). The test, however, did not give a positive result in any of the four clones from <em>Hoolock</em> or six clones from <em>Nomascus</em>, suggesting that, even if these clones contain an HOR structure, its scale (the number of basic repeat units constituting an HOR unit) is larger than that found in <em>Symphalangus</em>. We developed another method to obtain longer sequences, with the expectation that longer sequences would increase the possibility of detecting signs. The strategy used for this purpose is shown in Fig. <a reveal-id="DSU005F1" data-open="DSU005F1" class="link link-reveal link-table xref-fig">1</a>. We prepared a clone of the Tn5 transposon that carried the kanamycin-resistance gene. We mixed it with an AS-containing genomic DNA clone and induced the transposition reaction by supplying the Tn5 transposase enzyme. The fosmid vector (pCC1FOS) for the genomic DNA clone carried the chloramphenicol-resistance gene. After the transposition reaction, the reaction mixture was introduced into bacterial cells by electroporation, and single colonies that survived on LB/agar plates containing chloramphenicol (5 µg/ml) and kanamycin (5 µg/ml) were selected. Fosmid DNA extracted from these colonies mostly carried a single Tn5 insertion at different positions. We sequenced two of these clones by using sequencing primers that annealed to the left or right terminal regions of Tn5 and were oriented outward. </p>                        <a id="6676346" scrollto-destination="6676346"></a>
<div id="DSU005F1" data-id="DSU005F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00501.png?Expires=2147483647&Signature=JfwHkqtf8auugUglTLQP-PT0MS9vc7B4FDtRfIWbPz216Yr4aPxp7JE1aU~yT-1YYJZUiRroksmi3yOusTcOhyjiHh4I6cGm4LsOh6bGlwMOGH6AH7w1FmjANX8vxPYbYgv4j9LO8hRFU8MnPa5JLzc~SUIzJkr89dksNOUSN2dvncnVz2~Vtk8Gox-PABGy2tyajOuhYdc~lb6AS4snWcsOPqL-90Iq3WmRlu77V~ilOfW6tcnYOu~sGPdx~qNEM5JP9WKBuhFr~n4HWB~ZQuEX2vB02p1eDvlk9LnrkimbHSp-EE3KG69EnMMWKsgE2sctay71fS0pi1FdpkE3dQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Strategy for the detection of HOR signs. Transposition of the Tn5 transposon to DNA molecules of an AS-containing fosmid clone was induced, and two clones carrying Tn5 at different positions were selected. Each clone was sequenced using two primers that annealed to Tn5 and oriented outward. The sequence reads (∼900 bp, including the tail region of a low reliability) were combined, and a longer sequence (∼1800 bp) was obtained. These sequences were compared between the two clones by the dot-matrix analysis. When a line spanning more than two basic units was found, it was regarded as a sign of HOR." path-from-xml="dsu00501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676346/dsu00501.tiff" path-from-xml="dsu00501.tiff" target="_blank">View large</a><a section="6676346" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00501.png?Expires=2147483647&Signature=LpE1X~wEtUfW8XmvcDbefI7ro-Tfhxm1HG2Yf2X3pah6yhMM2HiqMrVP7XiuenRTMa9CC1BkR6eXyf4hvYaAJkTaaQ4FT~eqnKBXeCvNqSDX~dXeAthcOA66jsOsdKHC4CHPOewVPTVYE0un0C8Lh1yHu8jHGSMkdrJgaFNlF9q5bPqvBzDBCeKKHp6~cIktmK5NvS3p1zEk~jkKQZzNRsMJhm8n~itotrWe4VPpUWqkhmN2~Kf7qIk5sqm9DVzTymiM7WzBqO~2RefAsUPpbbMXjLUufXUE7CWbgCcgPiii7mRgAIjKDRQO877I~2u8GhyYGTPkzhCV9OYOs4yXtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676346&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Strategy for the detection of HOR signs. Transposition of the Tn5 transposon to DNA molecules of an AS-containing fosmid clone was induced, and two clones carrying Tn5 at different positions were selected. Each clone was sequenced using two primers that annealed to Tn5 and oriented outward. The sequence reads (∼900 bp, including the tail region of a low reliability) were combined, and a longer sequence (∼1800 bp) was obtained. These sequences were compared between the two clones by the dot-matrix analysis. When a line spanning more than two basic units was found, it was regarded as a sign of HOR.</p></div></div><div content-id="DSU005F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00501.png?Expires=2147483647&Signature=JfwHkqtf8auugUglTLQP-PT0MS9vc7B4FDtRfIWbPz216Yr4aPxp7JE1aU~yT-1YYJZUiRroksmi3yOusTcOhyjiHh4I6cGm4LsOh6bGlwMOGH6AH7w1FmjANX8vxPYbYgv4j9LO8hRFU8MnPa5JLzc~SUIzJkr89dksNOUSN2dvncnVz2~Vtk8Gox-PABGy2tyajOuhYdc~lb6AS4snWcsOPqL-90Iq3WmRlu77V~ilOfW6tcnYOu~sGPdx~qNEM5JP9WKBuhFr~n4HWB~ZQuEX2vB02p1eDvlk9LnrkimbHSp-EE3KG69EnMMWKsgE2sctay71fS0pi1FdpkE3dQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Strategy for the detection of HOR signs. Transposition of the Tn5 transposon to DNA molecules of an AS-containing fosmid clone was induced, and two clones carrying Tn5 at different positions were selected. Each clone was sequenced using two primers that annealed to Tn5 and oriented outward. The sequence reads (∼900 bp, including the tail region of a low reliability) were combined, and a longer sequence (∼1800 bp) was obtained. These sequences were compared between the two clones by the dot-matrix analysis. When a line spanning more than two basic units was found, it was regarded as a sign of HOR." path-from-xml="dsu00501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676346/dsu00501.tiff" path-from-xml="dsu00501.tiff" target="_blank">View large</a><a section="6676346" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00501.png?Expires=2147483647&Signature=LpE1X~wEtUfW8XmvcDbefI7ro-Tfhxm1HG2Yf2X3pah6yhMM2HiqMrVP7XiuenRTMa9CC1BkR6eXyf4hvYaAJkTaaQ4FT~eqnKBXeCvNqSDX~dXeAthcOA66jsOsdKHC4CHPOewVPTVYE0un0C8Lh1yHu8jHGSMkdrJgaFNlF9q5bPqvBzDBCeKKHp6~cIktmK5NvS3p1zEk~jkKQZzNRsMJhm8n~itotrWe4VPpUWqkhmN2~Kf7qIk5sqm9DVzTymiM7WzBqO~2RefAsUPpbbMXjLUufXUE7CWbgCcgPiii7mRgAIjKDRQO877I~2u8GhyYGTPkzhCV9OYOs4yXtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676346&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Strategy for the detection of HOR signs. Transposition of the Tn5 transposon to DNA molecules of an AS-containing fosmid clone was induced, and two clones carrying Tn5 at different positions were selected. Each clone was sequenced using two primers that annealed to Tn5 and oriented outward. The sequence reads (∼900 bp, including the tail region of a low reliability) were combined, and a longer sequence (∼1800 bp) was obtained. These sequences were compared between the two clones by the dot-matrix analysis. When a line spanning more than two basic units was found, it was regarded as a sign of HOR.</p></div></div><p>Each sequence read contained data for &gt;855 (5 × 171) nucleotides. Although the tail region of the sequence read (distant by &gt;700 nucleotides from the sequencing primer) had quite a high possibility of containing sequencing errors, this was not a problem for our purpose because the data were used for detecting signs of HOR, but sequencing errors masked such signs. By combining the sequence reads for both sides of a single Tn5 insertion, we could obtain nucleotide sequence information for a region of &gt;1710 (10 × 171) nucleotides, and it contained sequence information for at least nine consecutive repeat units.</p><p>In each clone, we compared two sets of such sequences by dot-matrix analysis. When a line spanning more than two repeat units was found in the dot matrix, we regarded it as a sign of an HOR. The detection efficiency of this method depends on the scale of the HOR to be revealed, namely, how many (designated as <em>n</em> hereafter) basic repeat units constitute the HOR. For HORs with <em>n</em> ≤ 16, our method is expected to be powerful enough to detect a sign with only two sets of sequence data. For <em>n</em> ≥ 17, the probability of detecting a sign with two sets of sequence data is 15/<em>n</em>. The value of the probability is as large as 0.5 for <em>n</em> = 30 and 0.375 for <em>n</em> = 40. If we use sequence data of additional insertion points, we can increase the probability.</p><p>We examined the AS-containing fosmid clones by the transposition assay and subsequent sequencing analysis. Of the four <em>Hoolock</em> and six <em>Nomascus</em> clones, one and three clones, respectively, exhibited signs of HOR. Figure <a reveal-id="DSU005F2" data-open="DSU005F2" class="link link-reveal link-table xref-fig">2</a> shows examples of the results of this dot-matrix analysis. We selected one clone each (clones HooFos09 of <em>Hoolock</em> and NomFos1B3 of <em>Nomascus</em>) and moved on to determining the long sequences of the AS arrays contained in these two clones, and also the clone HylFos15 from <em>Hylobates</em>. </p>                        <a id="6676350" scrollto-destination="6676350"></a>
<div id="DSU005F2" data-id="DSU005F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00502.png?Expires=2147483647&Signature=EUEMQeEXYKR~tZOP3PY3rvdd13QaEmT5IeJOAVX4ilqVcj9yQA9QUG3arDRYZf8839IsxdxiYwrWfApcgtMBy4-CNhLgd7l40AZw13GM2vzxSyW6gWZeVIHZsZWKJnnbeaetNM4NkaDpCd0BcqaNYS4sNpXmmzL0Vs8TeppV1jjvyEfigi0A1lKObKs7rBYOMEkzhFKMVV8Nh1S4gIxzfot6y6QReiOro5TGOK6k4FNn8lkLPEgvm2kjy-K1JpYjwA2bHpEaGVx9XEPj6ifONlEFpzhDUxPxrzYFMRfmoblK3eJlatQhp6Ol3y8keDwUdMSWffTJFIfN8nfUK~jlBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot-matrix analysis of clone partial sequences for signs of HOR. The criterion in the dot-matrix analysis was that a 19-nucleotide match should exist over a window of 20 nucleotides. Examples of the results are shown. HooFos09 and NomFos1B3 exhibited a line spanning more than two basic repeat units, and HooFos03 and NomFos3F5 did not." path-from-xml="dsu00502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676350/dsu00502.tiff" path-from-xml="dsu00502.tiff" target="_blank">View large</a><a section="6676350" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00502.png?Expires=2147483647&Signature=hNta1enzSlgb~TEy7iY6wQZun4eqphJBT08ESWUt-pScE-xfddxrq4ZDgQPjQLWVYHgJteBdogsLVImORxpTQ1YdeNedSMT1vkqupgR7e2PWJxUp8VB3t0cYx5Nok6Khyd198soha5x96YH8nDgapnXfnj-pDODnqghVZYAd4JF3bz3EPza730KTdatM97SS8WqSKfn8GPhWxBFZ3XpNFT3rnGZ8m~W1X4ppj7JBo04Tg2geXEc9VCUGMAjSphZw~iaRgoStnyk628Rqj5BRhADileUBGWd8m4YIjA~7Df0TWU1eJMGHjCdFSiXnRzmn8WYjVzU6kfzwSt5i0W9UPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676350&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot-matrix analysis of clone partial sequences for signs of HOR. The criterion in the dot-matrix analysis was that a 19-nucleotide match should exist over a window of 20 nucleotides. Examples of the results are shown. HooFos09 and NomFos1B3 exhibited a line spanning more than two basic repeat units, and HooFos03 and NomFos3F5 did not.</p></div></div><div content-id="DSU005F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00502.png?Expires=2147483647&Signature=EUEMQeEXYKR~tZOP3PY3rvdd13QaEmT5IeJOAVX4ilqVcj9yQA9QUG3arDRYZf8839IsxdxiYwrWfApcgtMBy4-CNhLgd7l40AZw13GM2vzxSyW6gWZeVIHZsZWKJnnbeaetNM4NkaDpCd0BcqaNYS4sNpXmmzL0Vs8TeppV1jjvyEfigi0A1lKObKs7rBYOMEkzhFKMVV8Nh1S4gIxzfot6y6QReiOro5TGOK6k4FNn8lkLPEgvm2kjy-K1JpYjwA2bHpEaGVx9XEPj6ifONlEFpzhDUxPxrzYFMRfmoblK3eJlatQhp6Ol3y8keDwUdMSWffTJFIfN8nfUK~jlBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot-matrix analysis of clone partial sequences for signs of HOR. The criterion in the dot-matrix analysis was that a 19-nucleotide match should exist over a window of 20 nucleotides. Examples of the results are shown. HooFos09 and NomFos1B3 exhibited a line spanning more than two basic repeat units, and HooFos03 and NomFos3F5 did not." path-from-xml="dsu00502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676350/dsu00502.tiff" path-from-xml="dsu00502.tiff" target="_blank">View large</a><a section="6676350" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00502.png?Expires=2147483647&Signature=hNta1enzSlgb~TEy7iY6wQZun4eqphJBT08ESWUt-pScE-xfddxrq4ZDgQPjQLWVYHgJteBdogsLVImORxpTQ1YdeNedSMT1vkqupgR7e2PWJxUp8VB3t0cYx5Nok6Khyd198soha5x96YH8nDgapnXfnj-pDODnqghVZYAd4JF3bz3EPza730KTdatM97SS8WqSKfn8GPhWxBFZ3XpNFT3rnGZ8m~W1X4ppj7JBo04Tg2geXEc9VCUGMAjSphZw~iaRgoStnyk628Rqj5BRhADileUBGWd8m4YIjA~7Df0TWU1eJMGHjCdFSiXnRzmn8WYjVzU6kfzwSt5i0W9UPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676350&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot-matrix analysis of clone partial sequences for signs of HOR. The criterion in the dot-matrix analysis was that a 19-nucleotide match should exist over a window of 20 nucleotides. Examples of the results are shown. HooFos09 and NomFos1B3 exhibited a line spanning more than two basic repeat units, and HooFos03 and NomFos3F5 did not.</p></div></div>                            <h3 scrollto-destination=6676351 id="6676351" class="section-title" data-legacy-id=s3d>3.4. Transfer of fragments to plasmid</h3>
<p>As explained earlier, sequencing strategies relying on shotgun segmentation or primer walking are not adequate for tandemly repeated sequences, such as AS. The strategy we employed was preparation of a series of deletion clones by using exonuclease III and Mung bean nuclease.<sup><a reveal-id="DSU005C19" data-open="DSU005C19" class="link link-ref link-reveal xref-bibr">19</a></sup> By digesting the DNA fragment with an appropriate combination of restriction endonucleases (<em>Kpn</em>I whose 3′-protruding cutting site is resistant to the exonuclease, and <em>Bam</em>HI or <em>Sal</em>I whose 5′-protruding cutting sites are sensitive), the deletion starts at one end of the fragment and continues to various degrees. The maximum fragment size for which these treatments work smoothly is ∼16 kb. The size of the fosmid clones was 40–44 kb. Therefore, it was necessary to transfer part of the fosmid clones into a plasmid vector. For delimiting the part to be analysed, PCR was not adequate because of the repetitive nature of the object sequence.</p><p>Restriction endonuclease <em>Not</em>I recognizes an eight-nucleotide segment (GCGGCCGC), which rarely appears even in the insert size of the fosmid vector. The vector carried two <em>Not</em>I recognition sites located closely to its cloning site, and we manipulated the Tn5 transposon in advance to have an <em>Not</em>I site in it. These <em>Not</em>I sites enabled us to determine the position of the Tn5 insertion by cutting Tn5-carrying fosmid DNAs and running an electrophoresis gel. We checked 12 Tn5-carrying clones originating from the HooFos09, HylFos15, and NomFos1B3 clones, and selected one in which Tn5 had been inserted at a position 10–16 kb distant from one end of its AS array. We then subcloned the <em>Not</em>I-delimited DNA fragment into the <em>Not</em>I site of an ampicillin-resistant plasmid (pBluescript II SK+), and subsequently transferred the fragment to a chloramphenicol-resistant plasmid (originating from pHSG399) to which we had, in advance, added some restriction enzyme sites necessary for the preparation of deletion clones. We used the ampicillin-resistant plasmid because transfer of a long fragment from a chloramphenicol-resistant vector to another chloramphenicol-resistant vector becomes easier by using an intermediate vector carrying a resistance gene to a different antibiotic (ampicillin in this case). The plasmid clones carrying the AS fragments from the HooFos09, HylFos15, and NomFos1B3 clones are designated HooCam09, HylCam15, and NomCam1B3, respectively.</p>                            <h3 scrollto-destination=6676354 id="6676354" class="section-title" data-legacy-id=s3e>3.5. Sequencing analysis</h3>
<p>We prepared deletion clones of various lengths by treating HooCam09, HylCam15, and NomCam1B3 with exonuclease III and then with Mung bean nuclease. We checked the insert size of these deletion clones, arranged them in the order of longest to shortest, and selected 55, 28, and 41 clones, respectively, to be used as sources of partial sequences. The criterion for this selection was any two adjacent clones that did not differ in insert size by &gt;0.5 kb. Figure <a reveal-id="DSU005F3" data-open="DSU005F3" class="link link-reveal link-table xref-fig">3</a> shows an example of gel electrophoresis in which clones of 5.3–0.5 kb were placed according to their sizes. We sequenced the selected clones by using the M13 universal primer. We then manually assembled the obtained sequence reads into a contig sequence. </p>                        <a id="6676356" scrollto-destination="6676356"></a>
<div id="DSU005F3" data-id="DSU005F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00503.png?Expires=2147483647&Signature=Rqq3Js~mux7ppzR5GUIjE01dw4tqNOBZ7U2K3aUu88oeOl5k-7rJ28kLkKq~SKNvG~IcrqNtjBweZVqgl5jTUappHREuv6tZwt2UWlriGO2Sbio7RRRdyCx8vj6OeMDsQf9ywHASTuVur5oEFaFGRb-tB79JK7xTs0~CGDdvzeRmwNepSnu0SMgCNNNxj8707yW3EyAM8jju8z0XWEt26-UMb-QPqpc4C0nWqn0VINqhvrU6d~EwaFU1SCreuYoQshKSCSXoXVwiySKpgGOA2M2gMIH55Pg23qI4mF-zkEI7zDNSQ~j85XM6Pjgm1xv9yyVauS2uk3o79QNMxv~~qA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Confirmation of deletion clone insert size. Deletion clones of various sizes were prepared from the HooCam09 and NomCam1B3 clones. Out of these pools, clones of sizes adequate for sequencing analysis were selected. This photograph is an example of gel electrophoresis in which deletion clones of insert sizes of 5.3–0.5 kb, originating from HooCam09, are contained. The 2.1-kb vector plasmid carried two cutting sites for restriction endonuclease XhoI just outside of the cloning site, and the insert fragments in the original clones did not have a site for this enzyme. Each deletion clone was digested with XhoI and divided into two fragments: the 2.1-kb plasmid vector and its insert fragment." path-from-xml="dsu00503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676356/dsu00503.tiff" path-from-xml="dsu00503.tiff" target="_blank">View large</a><a section="6676356" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00503.png?Expires=2147483647&Signature=XzvYS4BWEnsFar6jNM2Z5tsVhbEEkPgpVWWUmWe9Tvc12N5jGY0aUUl~RrBLBWgYhhmMcsA1oknR~xVE3IP-4mbe-kT9UXYQeXYLMavW~K5Al-RZkbUkbKSPhGNFQRHsWY8z6D4ORNo5yNiZ2y6SX~CMBaQoaw3IujNiJVOz4HMAU~kdKDPLhB~pYkL9SwDbwky1gL9qeweSGkua75fMoATlzSb27ZDzIOsfc4GfK6WKkC~F1fdn92BaApcBNh3YkgnNwP8dTXaiwanXMhdPD6Neto-71PAf8wND7Wtaro61PmNWH4ua0imNS22CfxA1w6R9AmsfxacIIsfBHhRKeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676356&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Confirmation of deletion clone insert size. Deletion clones of various sizes were prepared from the HooCam09 and NomCam1B3 clones. Out of these pools, clones of sizes adequate for sequencing analysis were selected. This photograph is an example of gel electrophoresis in which deletion clones of insert sizes of 5.3–0.5 kb, originating from HooCam09, are contained. The 2.1-kb vector plasmid carried two cutting sites for restriction endonuclease <em>Xho</em>I just outside of the cloning site, and the insert fragments in the original clones did not have a site for this enzyme. Each deletion clone was digested with <em>Xho</em>I and divided into two fragments: the 2.1-kb plasmid vector and its insert fragment.</p></div></div><div content-id="DSU005F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00503.png?Expires=2147483647&Signature=Rqq3Js~mux7ppzR5GUIjE01dw4tqNOBZ7U2K3aUu88oeOl5k-7rJ28kLkKq~SKNvG~IcrqNtjBweZVqgl5jTUappHREuv6tZwt2UWlriGO2Sbio7RRRdyCx8vj6OeMDsQf9ywHASTuVur5oEFaFGRb-tB79JK7xTs0~CGDdvzeRmwNepSnu0SMgCNNNxj8707yW3EyAM8jju8z0XWEt26-UMb-QPqpc4C0nWqn0VINqhvrU6d~EwaFU1SCreuYoQshKSCSXoXVwiySKpgGOA2M2gMIH55Pg23qI4mF-zkEI7zDNSQ~j85XM6Pjgm1xv9yyVauS2uk3o79QNMxv~~qA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Confirmation of deletion clone insert size. Deletion clones of various sizes were prepared from the HooCam09 and NomCam1B3 clones. Out of these pools, clones of sizes adequate for sequencing analysis were selected. This photograph is an example of gel electrophoresis in which deletion clones of insert sizes of 5.3–0.5 kb, originating from HooCam09, are contained. The 2.1-kb vector plasmid carried two cutting sites for restriction endonuclease XhoI just outside of the cloning site, and the insert fragments in the original clones did not have a site for this enzyme. Each deletion clone was digested with XhoI and divided into two fragments: the 2.1-kb plasmid vector and its insert fragment." path-from-xml="dsu00503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676356/dsu00503.tiff" path-from-xml="dsu00503.tiff" target="_blank">View large</a><a section="6676356" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00503.png?Expires=2147483647&Signature=XzvYS4BWEnsFar6jNM2Z5tsVhbEEkPgpVWWUmWe9Tvc12N5jGY0aUUl~RrBLBWgYhhmMcsA1oknR~xVE3IP-4mbe-kT9UXYQeXYLMavW~K5Al-RZkbUkbKSPhGNFQRHsWY8z6D4ORNo5yNiZ2y6SX~CMBaQoaw3IujNiJVOz4HMAU~kdKDPLhB~pYkL9SwDbwky1gL9qeweSGkua75fMoATlzSb27ZDzIOsfc4GfK6WKkC~F1fdn92BaApcBNh3YkgnNwP8dTXaiwanXMhdPD6Neto-71PAf8wND7Wtaro61PmNWH4ua0imNS22CfxA1w6R9AmsfxacIIsfBHhRKeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676356&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Confirmation of deletion clone insert size. Deletion clones of various sizes were prepared from the HooCam09 and NomCam1B3 clones. Out of these pools, clones of sizes adequate for sequencing analysis were selected. This photograph is an example of gel electrophoresis in which deletion clones of insert sizes of 5.3–0.5 kb, originating from HooCam09, are contained. The 2.1-kb vector plasmid carried two cutting sites for restriction endonuclease <em>Xho</em>I just outside of the cloning site, and the insert fragments in the original clones did not have a site for this enzyme. Each deletion clone was digested with <em>Xho</em>I and divided into two fragments: the 2.1-kb plasmid vector and its insert fragment.</p></div></div>                            <h3 scrollto-destination=6676357 id="6676357" class="section-title" data-legacy-id=s3f>3.6. Evidence for HORs</h3>
<p>We obtained a 16 429-bp contig sequence for the HooCam09 clone, a 7828-bp contig sequence for the HylCam15 clone, and a 12 093-bp contig sequence for the NomCam1B3 clone. These were deposited in DDBJ with accession numbers AB861929, AB900117, and AB846956, respectively. We defined the boundaries and orientation of the basic repeat unit, so that they would be the same as those used in our previous studies.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU005C20" data-open="DSU005C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The three clones contained 95, 44, and 70 basic repeat units, respectively.</p><p>We made pairwise comparisons between basic repeat units, and the essence of the obtained results is shown as matrices in Fig. <a reveal-id="DSU005F4" data-open="DSU005F4" class="link link-reveal link-table xref-fig">4</a>. In the matrices, red and yellow cells indicate identities of &gt;95 and 90–95%, respectively. Matrices containing the nucleotide identity of each cell are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S1–S3</a>. In HooCam09 and NomCam1B3, clear step-like lines consisting of red cells appeared with a constant interval throughout the sequence. The interval was 31 basic repeat units for HooCam09 and 9 basic repeat units for NomCam1B3. HylCam15 showed step-like lines exhibiting two types of intervals (8-unit interval for units 5–12 and 13-unit interval for units 13–44). Thus, it is evident that the AS sequences contained in these clones are organized into HOR structures. </p>                        <a id="6676360" scrollto-destination="6676360"></a>
<div id="DSU005F4" data-id="DSU005F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00504.png?Expires=2147483647&Signature=kNsfLjg0ZZnIfPhH~z1ULUUwIPRfY4-9dFsy9xnYwRu~nUjPT4aSH8qzQFxsrBNEkeIBVsAmoxxLgQKNDYBoFjyHHH94v~o4d6jvDStEy-KOGzNWvedAhwefbBNjlbkQhi9kOef~NcHB7skIMRCjSPTXIGTGBfMDxl5QkAAw5z-yW3rr2frXqkMSyygxiWHcgxANMqNvfoAtEtBV9bfJb2C9CtvAkSSMFzi3tQafizYEKsi2eu-bNA6MplUpcbcYU455TUpBJBiWdFUVB8rPZbemcwyrgwETRl2wrwQ92ypd-Nl5ISyVgZS27cMf0YKAHFBjKE9UhjE3-3Akn0SfyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pairwise comparisons of basic repeat unit sequences. The comparisons were made using the MEGA5 program21 under default settings. The horizontal and vertical axes of each matrix represent consecutive basic repeat units contained in the AS contig sequence. Cells showing nucleotide identities of 90–95 and &gt;95% are indicated by yellow and red, respectively. The same matrices containing the identity values in the cells are shown in Supplementary Figs S1–S3." path-from-xml="dsu00504.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676360/dsu00504.tiff" path-from-xml="dsu00504.tiff" target="_blank">View large</a><a section="6676360" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00504.png?Expires=2147483647&Signature=CHPIRkshuQdPGE3Tl-aBaGjIOQxOfEduLfQaV7bNA4oRpTy8XJCKKOHGSmZG7S6xLAxGEqPSB3H-RxzqKNzO6nBJVhwBjw-wLqXKfAPtR9q7JassY1vPIeDcEOYt2RiQm843o99jyxV0k-8BiLj~QKbghg9796IYbsZHWLsQvp7UQQolaMfqFCEad-1KWxWHzoR8yhtTmKseaNzU5WVK2IxiRVk4q-zzWdU9yE~OaO2gQhStG4dccN2T3KCyZsPonTcTeK~~BHJsHOK5MtRLaXjk76B0Q3EzkdtWjXorgaCQR9Cfcuu6ZKLy~n1qOhBx2PT6YX3FZiwG0kQ-e-DSgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676360&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00504.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pairwise comparisons of basic repeat unit sequences. The comparisons were made using the MEGA5 program<sup><a reveal-id="DSU005C21" data-open="DSU005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> under default settings. The horizontal and vertical axes of each matrix represent consecutive basic repeat units contained in the AS contig sequence. Cells showing nucleotide identities of 90–95 and &gt;95% are indicated by yellow and red, respectively. The same matrices containing the identity values in the cells are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S1–S3</a>.</p></div></div><div content-id="DSU005F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00504.png?Expires=2147483647&Signature=kNsfLjg0ZZnIfPhH~z1ULUUwIPRfY4-9dFsy9xnYwRu~nUjPT4aSH8qzQFxsrBNEkeIBVsAmoxxLgQKNDYBoFjyHHH94v~o4d6jvDStEy-KOGzNWvedAhwefbBNjlbkQhi9kOef~NcHB7skIMRCjSPTXIGTGBfMDxl5QkAAw5z-yW3rr2frXqkMSyygxiWHcgxANMqNvfoAtEtBV9bfJb2C9CtvAkSSMFzi3tQafizYEKsi2eu-bNA6MplUpcbcYU455TUpBJBiWdFUVB8rPZbemcwyrgwETRl2wrwQ92ypd-Nl5ISyVgZS27cMf0YKAHFBjKE9UhjE3-3Akn0SfyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pairwise comparisons of basic repeat unit sequences. The comparisons were made using the MEGA5 program21 under default settings. The horizontal and vertical axes of each matrix represent consecutive basic repeat units contained in the AS contig sequence. Cells showing nucleotide identities of 90–95 and &gt;95% are indicated by yellow and red, respectively. The same matrices containing the identity values in the cells are shown in Supplementary Figs S1–S3." path-from-xml="dsu00504.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676360/dsu00504.tiff" path-from-xml="dsu00504.tiff" target="_blank">View large</a><a section="6676360" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00504.png?Expires=2147483647&Signature=CHPIRkshuQdPGE3Tl-aBaGjIOQxOfEduLfQaV7bNA4oRpTy8XJCKKOHGSmZG7S6xLAxGEqPSB3H-RxzqKNzO6nBJVhwBjw-wLqXKfAPtR9q7JassY1vPIeDcEOYt2RiQm843o99jyxV0k-8BiLj~QKbghg9796IYbsZHWLsQvp7UQQolaMfqFCEad-1KWxWHzoR8yhtTmKseaNzU5WVK2IxiRVk4q-zzWdU9yE~OaO2gQhStG4dccN2T3KCyZsPonTcTeK~~BHJsHOK5MtRLaXjk76B0Q3EzkdtWjXorgaCQR9Cfcuu6ZKLy~n1qOhBx2PT6YX3FZiwG0kQ-e-DSgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676360&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00504.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pairwise comparisons of basic repeat unit sequences. The comparisons were made using the MEGA5 program<sup><a reveal-id="DSU005C21" data-open="DSU005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> under default settings. The horizontal and vertical axes of each matrix represent consecutive basic repeat units contained in the AS contig sequence. Cells showing nucleotide identities of 90–95 and &gt;95% are indicated by yellow and red, respectively. The same matrices containing the identity values in the cells are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S1–S3</a>.</p></div></div><p>From the matrices shown in Fig. <a reveal-id="DSU005F4" data-open="DSU005F4" class="link link-reveal link-table xref-fig">4</a>, we picked up cells that represented pairs of basic units located at identical positions in different HOR units (e.g. the second basic repeat unit in the third HOR unit and the second basic repeat unit in the fourth HOR unit). In the HooCam09 clone, the average pairwise nucleotide identity among these pairs was 99.8%. The average value among the pairs of the other relationships was 89.2%. In the NomCam1B3 clone, the values were 98.6 and 88.0%, respectively. In the analysis of the HylCam15 clone, in which the 13th–44th basic repeat units (exhibiting a 13-unit interval) were used, we obtained the values of 97.0 and 86.5%, respectively. These situations are similar to those observed in human AS sequences: 95–99 and 70–90%, respectively.<sup><a reveal-id="DSU005C22" data-open="DSU005C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h3 scrollto-destination=6676362 id="6676362" class="section-title" data-legacy-id=s3g>3.7. Chromosomal locations</h3>
<p>We conducted FISH analysis of chromosomes, by using HooCam09, HylCam15, and NomCam1B3 as probes, to identify chromosomal locations (Fig. <a reveal-id="DSU005F5" data-open="DSU005F5" class="link link-reveal link-table xref-fig">5</a>). HooCam09 exhibited strong signals on the centromere region of chromosome 4 of <em>Hoolock hoolock</em> chromosome spreads. HylCam15 produced signals in the centromere regions of all <em>Hylobates agilis</em> chromosomes. NomCam1B3 showed clear signals on the centromere regions and terminal regions of most chromosomes of <em>Nomascus siki</em>. These results are consistent with the FISH results we obtained previously by using a clone of siamang AS as a probe.<sup><a reveal-id="DSU005C20" data-open="DSU005C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                        <a id="6676364" scrollto-destination="6676364"></a>
<div id="DSU005F5" data-id="DSU005F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00505.png?Expires=2147483647&Signature=V99LIJcNsY8~19HcSt5IUtTUGC3ZeDMYJQBhjv4oLVOgu1~mBuiWaRRnRj6C~HE-AfCHxQEm2WojqdXYdMF2lpgstPHerWHFpkTYzLXgz3MFmvGstHkqzOJrZIoAf2WldGWUMlmR2rhIwLbotPvHe5GmNJpNSmBr8B~Av9AEcwTSU-VSOCVywe0SCpLwuyF3Zmr81wdTDP1XNATrx9QOqJ75zX24FK4PdmAZXHniSFgoCNX2szYHgphqFPr5mKGn8ss6KaXQidRIBS0X7jgPT7dk3~LCzWTyx2c8eqQs8yDr-SdHxsffwGmrQKepdFedV-YYMmDr7MBIZKwIHvpNwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FISH analysis of chromosomes for locations of AS. The left panels show FISH images obtained using a fluorescence scanner, and the right panels are images of the same chromosome spreads stained with 4′,6-diamidino-2-phenylindole (DAPI). In the left panels, red signals indicate hybridization of the probe DNA to chromosome DNA. The combination of chromosome spread and probe is shown on each left panel. The bar in the panels represents 10 µm. All experimental procedures were the same as those described in our previous work.10" path-from-xml="dsu00505.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676364/dsu00505.tiff" path-from-xml="dsu00505.tiff" target="_blank">View large</a><a section="6676364" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00505.png?Expires=2147483647&Signature=ule786RjwgusRPK0mc-x8NupwhMSEybuBjAUoUZ7ou5kj4IMGqVdgG-NI9wmiyxQX8EdHmbPC07QQ0NZOT~zod6u7JVN0OAk6t2CZiTRjk0BNQMuAM47Y0agXrs2Dp5jWH-oF3nsmxchMYKzWI5jpsEWbPemQx3cmN6tmuqOeUT1PYP8SV2EHok9Jg75M3FaIq9WOyvYazKghcrrWJ2TwIIdHM6ejAai1ACyuU9GPxqjexfDmygWePOPTU1jXYStnnp0j~A60qKLL61gc1L46WG9z4S3MP4AALPp5dMFs75YCnRtuPUVCnBuPdqBSP4BPFcnD7FXYZFsLJFwv37wMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676364&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00505.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>FISH analysis of chromosomes for locations of AS. The left panels show FISH images obtained using a fluorescence scanner, and the right panels are images of the same chromosome spreads stained with 4′,6-diamidino-2-phenylindole (DAPI). In the left panels, red signals indicate hybridization of the probe DNA to chromosome DNA. The combination of chromosome spread and probe is shown on each left panel. The bar in the panels represents 10 µm. All experimental procedures were the same as those described in our previous work.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p></div></div><div content-id="DSU005F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00505.png?Expires=2147483647&Signature=V99LIJcNsY8~19HcSt5IUtTUGC3ZeDMYJQBhjv4oLVOgu1~mBuiWaRRnRj6C~HE-AfCHxQEm2WojqdXYdMF2lpgstPHerWHFpkTYzLXgz3MFmvGstHkqzOJrZIoAf2WldGWUMlmR2rhIwLbotPvHe5GmNJpNSmBr8B~Av9AEcwTSU-VSOCVywe0SCpLwuyF3Zmr81wdTDP1XNATrx9QOqJ75zX24FK4PdmAZXHniSFgoCNX2szYHgphqFPr5mKGn8ss6KaXQidRIBS0X7jgPT7dk3~LCzWTyx2c8eqQs8yDr-SdHxsffwGmrQKepdFedV-YYMmDr7MBIZKwIHvpNwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FISH analysis of chromosomes for locations of AS. The left panels show FISH images obtained using a fluorescence scanner, and the right panels are images of the same chromosome spreads stained with 4′,6-diamidino-2-phenylindole (DAPI). In the left panels, red signals indicate hybridization of the probe DNA to chromosome DNA. The combination of chromosome spread and probe is shown on each left panel. The bar in the panels represents 10 µm. All experimental procedures were the same as those described in our previous work.10" path-from-xml="dsu00505.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676364/dsu00505.tiff" path-from-xml="dsu00505.tiff" target="_blank">View large</a><a section="6676364" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00505.png?Expires=2147483647&Signature=ule786RjwgusRPK0mc-x8NupwhMSEybuBjAUoUZ7ou5kj4IMGqVdgG-NI9wmiyxQX8EdHmbPC07QQ0NZOT~zod6u7JVN0OAk6t2CZiTRjk0BNQMuAM47Y0agXrs2Dp5jWH-oF3nsmxchMYKzWI5jpsEWbPemQx3cmN6tmuqOeUT1PYP8SV2EHok9Jg75M3FaIq9WOyvYazKghcrrWJ2TwIIdHM6ejAai1ACyuU9GPxqjexfDmygWePOPTU1jXYStnnp0j~A60qKLL61gc1L46WG9z4S3MP4AALPp5dMFs75YCnRtuPUVCnBuPdqBSP4BPFcnD7FXYZFsLJFwv37wMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676364&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00505.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>FISH analysis of chromosomes for locations of AS. The left panels show FISH images obtained using a fluorescence scanner, and the right panels are images of the same chromosome spreads stained with 4′,6-diamidino-2-phenylindole (DAPI). In the left panels, red signals indicate hybridization of the probe DNA to chromosome DNA. The combination of chromosome spread and probe is shown on each left panel. The bar in the panels represents 10 µm. All experimental procedures were the same as those described in our previous work.<sup><a reveal-id="DSU005C10" data-open="DSU005C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p></div></div>                            <h3 scrollto-destination=6676368 id="6676368" class="section-title" data-legacy-id=s3h>3.8. Comparison of AS sequences among hominoid species</h3>
<p>We made within-species consensus sequences of AS repeat units of <em>Hoolock hoolock</em>, <em>Hylobates agilis</em>, and <em>Nomascus siki</em>. Furthermore, from references, we obtained consensus sequence for <em>Nomascus leucogenys</em>, <em>S. syndactylus</em>, human, and orangutan. We then made a neighbour-joining phylogenetic tree of these consensus sequences, which is shown in Fig. <a reveal-id="DSU005F6" data-open="DSU005F6" class="link link-reveal link-table xref-fig">6</a>. The five gibbon species representing four genera formed a cluster distinct from that of the two hominid species, and the two <em>Nomascus</em> species formed an internal cluster within the gibbon cluster. </p>                        <a id="6676373" scrollto-destination="6676373"></a>
<div id="DSU005F6" data-id="DSU005F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00506.png?Expires=2147483647&Signature=FqNjm2H5gmZCJPM3eIdY9H5QZXK4uA8Tg1fBDo45wlpl5qG8CTji90HvkaR4qvpMwWhg~ALqXTgLKEqvYUXBUM0IBMYAKxlX160SRxXzIhdnQkqvKAy9I-phTNI0PSKct385zTvREOP8C7qrJ77LZy9nDAU~ai5KnNwh1T3iP3DVrMg1-D2A6hmHNbEXKZqAS5-uYvMGpYOnH6se9M34LA8xWmnw8cz-yCunBpEd7JZhGvPBF1ew6p7zQLrNgwHbwEjaMxeZmf9fMKN2p8gF8VEqIT56b106KSxLo9J973839SuWnBwdlYqTv6yTm-b4w-gITTcK0VqeqEN1ziEteQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Neighbour-joining phylogenetic tree of within-species consensus sequences. The consensus sequences of AS of Hoolock hoolock, Hylobates agilis and Nomascus siki were obtained by sending the contig sequences, which we determined in the present study, to the Consensus Maker program (version 2.0.0; http://www.hiv.lanl.gov/content/sequence/CONSENSUS/consensus.html). The consensus sequences obtained are included in the respective DDBJ files. The consensus sequences for Nomascus leucogenys,23Symphalangus syndactylus,9 human22, and orangutan7 AS were obtained from the respective references. The neighbour-joining phylogenetic tree of these within-species consensus sequences was made using the ClustalW program contained in MEGA.21." path-from-xml="dsu00506.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676373/dsu00506.tiff" path-from-xml="dsu00506.tiff" target="_blank">View large</a><a section="6676373" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00506.png?Expires=2147483647&Signature=HIEWMmNGX8XV0r9MQT98a08ZnO2BdnseVRHcroGvocwVggHddtdjHvEDL9UATVFbes7vIy~-m11pFVNLU1lSZCDk43ijgGFGo2Yuctj6pCEXVi0oxgn4vcgWzEiKrS2qvkITMEGNIO84ACv-pB8ayp9XnozdeEBfspluitFuddx5-EbbLGa3HhcBouyETL0g9q6Yp~tzqjpNT4Nb9ncivq1W-jh3p~PXMnt0qoqHMp6fZYllnE7eNrKqHUzsa7d0sO1jRWjHBjiCgZkkBSeh2JVNGZKpOwXVQznRAd203UbUQq-UK6dXISGQiWLziPnjk9HbokRYhnhQbwj-zdxu-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676373&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00506.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Neighbour-joining phylogenetic tree of within-species consensus sequences. The consensus sequences of AS of <em>Hoolock hoolock</em>, <em>Hylobates agilis</em> and <em>Nomascus siki</em> were obtained by sending the contig sequences, which we determined in the present study, to the Consensus Maker program (version 2.0.0; <a class="link link-uri" href="http://www.hiv.lanl.gov/content/sequence/CONSENSUS/consensus.html">http://www.hiv.lanl.gov/content/sequence/CONSENSUS/consensus.html</a>). The consensus sequences obtained are included in the respective DDBJ files. The consensus sequences for <em>Nomascus leucogenys</em>,<sup><a reveal-id="DSU005C23" data-open="DSU005C23" class="link link-ref link-reveal xref-bibr">23</a></sup><em>Symphalangus syndactylus</em>,<sup><a reveal-id="DSU005C9" data-open="DSU005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> human<sup><a reveal-id="DSU005C22" data-open="DSU005C22" class="link link-ref link-reveal xref-bibr">22</a></sup>, and orangutan<sup><a reveal-id="DSU005C7" data-open="DSU005C7" class="link link-ref link-reveal xref-bibr">7</a></sup> AS were obtained from the respective references. The neighbour-joining phylogenetic tree of these within-species consensus sequences was made using the ClustalW program contained in MEGA.<sup><a reveal-id="DSU005C21" data-open="DSU005C21" class="link link-ref link-reveal xref-bibr">21</a>.</sup></p></div></div><div content-id="DSU005F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/m_dsu00506.png?Expires=2147483647&Signature=FqNjm2H5gmZCJPM3eIdY9H5QZXK4uA8Tg1fBDo45wlpl5qG8CTji90HvkaR4qvpMwWhg~ALqXTgLKEqvYUXBUM0IBMYAKxlX160SRxXzIhdnQkqvKAy9I-phTNI0PSKct385zTvREOP8C7qrJ77LZy9nDAU~ai5KnNwh1T3iP3DVrMg1-D2A6hmHNbEXKZqAS5-uYvMGpYOnH6se9M34LA8xWmnw8cz-yCunBpEd7JZhGvPBF1ew6p7zQLrNgwHbwEjaMxeZmf9fMKN2p8gF8VEqIT56b106KSxLo9J973839SuWnBwdlYqTv6yTm-b4w-gITTcK0VqeqEN1ziEteQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Neighbour-joining phylogenetic tree of within-species consensus sequences. The consensus sequences of AS of Hoolock hoolock, Hylobates agilis and Nomascus siki were obtained by sending the contig sequences, which we determined in the present study, to the Consensus Maker program (version 2.0.0; http://www.hiv.lanl.gov/content/sequence/CONSENSUS/consensus.html). The consensus sequences obtained are included in the respective DDBJ files. The consensus sequences for Nomascus leucogenys,23Symphalangus syndactylus,9 human22, and orangutan7 AS were obtained from the respective references. The neighbour-joining phylogenetic tree of these within-species consensus sequences was made using the ClustalW program contained in MEGA.21." path-from-xml="dsu00506.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6676373/dsu00506.tiff" path-from-xml="dsu00506.tiff" target="_blank">View large</a><a section="6676373" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu00506.png?Expires=2147483647&Signature=HIEWMmNGX8XV0r9MQT98a08ZnO2BdnseVRHcroGvocwVggHddtdjHvEDL9UATVFbes7vIy~-m11pFVNLU1lSZCDk43ijgGFGo2Yuctj6pCEXVi0oxgn4vcgWzEiKrS2qvkITMEGNIO84ACv-pB8ayp9XnozdeEBfspluitFuddx5-EbbLGa3HhcBouyETL0g9q6Yp~tzqjpNT4Nb9ncivq1W-jh3p~PXMnt0qoqHMp6fZYllnE7eNrKqHUzsa7d0sO1jRWjHBjiCgZkkBSeh2JVNGZKpOwXVQznRAd203UbUQq-UK6dXISGQiWLziPnjk9HbokRYhnhQbwj-zdxu-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6676373&ar=446188&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00506.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Neighbour-joining phylogenetic tree of within-species consensus sequences. The consensus sequences of AS of <em>Hoolock hoolock</em>, <em>Hylobates agilis</em> and <em>Nomascus siki</em> were obtained by sending the contig sequences, which we determined in the present study, to the Consensus Maker program (version 2.0.0; <a class="link link-uri" href="http://www.hiv.lanl.gov/content/sequence/CONSENSUS/consensus.html">http://www.hiv.lanl.gov/content/sequence/CONSENSUS/consensus.html</a>). The consensus sequences obtained are included in the respective DDBJ files. The consensus sequences for <em>Nomascus leucogenys</em>,<sup><a reveal-id="DSU005C23" data-open="DSU005C23" class="link link-ref link-reveal xref-bibr">23</a></sup><em>Symphalangus syndactylus</em>,<sup><a reveal-id="DSU005C9" data-open="DSU005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> human<sup><a reveal-id="DSU005C22" data-open="DSU005C22" class="link link-ref link-reveal xref-bibr">22</a></sup>, and orangutan<sup><a reveal-id="DSU005C7" data-open="DSU005C7" class="link link-ref link-reveal xref-bibr">7</a></sup> AS were obtained from the respective references. The neighbour-joining phylogenetic tree of these within-species consensus sequences was made using the ClustalW program contained in MEGA.<sup><a reveal-id="DSU005C21" data-open="DSU005C21" class="link link-ref link-reveal xref-bibr">21</a>.</sup></p></div></div>                            <h2 scrollto-destination=6676377 id="6676377" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In addition to humans, higher-order AS has also been observed in all great ape species examined to date, although it had not been found in small apes (except for our recent report) or other primate taxa more distant from humans. Based on this distribution in primates, it is widely postulated that higher-order AS arose in the common ancestor of hominids.</p><p>We have demonstrated the presence of high-order AS in all of the four genera of small apes. Thus, higher-order AS occurs widely in hominoids. The species of the <em>Symphalangus</em> genus that we previously examined was shown to have higher-order AS containing four or six basic repeat units in its HOR units. In the present study, we found clear HOR structures in the AS of species of <em>Hoolock</em> and <em>Nomascus</em>, in which 31 and 9 basic repeat units formed HOR units, respectively. HOR structures were also found in a species of <em>Hylobates</em>, although two types of HORs (8 and 13 basic repeat units) coexisted in a single clone.</p><p><em>Nomascus leucogenys</em> (Northern white-cheeked gibbon) has been extensively examined in the past for HOR structures by computational analysis of its genome sequence database and additional FISH experiments, but no positive evidence was obtained.<sup><a reveal-id="DSU005C23" data-open="DSU005C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In the present study, we have identified HOR in AS of <em>N. siki</em> (Southern white-cheeked gibbon). These two species are so close phylogenetically that they have been regarded as subspecies of one species until about a decade ago (<em>N. leucogenys leucogenys</em> and <em>N. leucogenys siki</em>, respectively). The most probable reason for this difference of results would be the difference in the method used; we relied on traditional sequencing of long clones preceded by detection of HOR signs. Another possible factor is the difference in the gibbon species. Satellite DNA is known to undergo rapid fluctuation in copy number. The HOR-containing AS we found may have been amplified in a small number of generations in the lineage leading to <em>N. siki</em> (or the individual we used), or its copy number may have been reduced rapidly in the lineage for <em>N. leucogenys</em> (or the individual used by those authors).</p><p>Our strategy included a screening step of AS-containing fosmid clones for those exhibiting HOR signs, in which small parts of candidate clones were sequenced. We examined four <em>Hoolock</em> and six <em>Nomascus</em> clones by the method newly developed in the present study, and one and three clones, respectively, exhibited an HOR sign. Considering that our library screening did not involve a factor to favour HOR-containing AS clones, the HOR structure does not appear to be rare in AS sequences contained in their genomes. The lack of a sign does not necessarily mean that the AS clones examined have no HOR structure. In our method, as we described in the Results section, the probability of finding a sign in a single fosmid clone is 15/<em>n</em>, where <em>n</em> is the number of basic repeat units constituting the HOR unit. If <em>n</em> is as large as 100, it would be required to increase the number of fosmid clones to be examined or the number of Tn5 insertions in individual clones. We did not attempt this in the present study because we wanted to develop another, more efficient method.</p><p>Another significance of our results is that we found HOR structures in AS residing in the centromere. This question remained from our previous study,<sup><a reveal-id="DSU005C9" data-open="DSU005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> because <em>Symphalangus syndactylus</em> carries AS in terminal regions of chromosomes in addition to in centromeres. The species of genus <em>Nomascus</em> examined so far also contain AS in terminal regions.<sup><a reveal-id="DSU005C20" data-open="DSU005C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSU005C23" data-open="DSU005C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In contrast, such structures are not found in <em>Hoolock hoolock</em> and <em>Hylobates agilis</em>,<sup><a reveal-id="DSU005C20" data-open="DSU005C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and, therefore, the possibility that the fosmid clones we sequenced originate from chromosomal portions other than the centromere would be negligible.</p><p>The currently dominant view about the evolutionary origin of HOR-containing AS is that it arose in the common ancestor of hominids (family Hominidae). The evidence of its widespread occurrence in the other family (Hylobatidae) of hominoids requires reconsideration of the currently dominant view. Likely explanations for the history of HOR-containing AS are as follows: (i) HOR-containing AS was present in the last common ancestor of hominoids and (ii) HOR-containing AS emerged in most or all basal branches of hominoids. As a more parsimonious explanation, (i) would be preferred.</p><p>The neighbour-joining phylogenetic tree of AS sequences is consistent with the widely accepted phylogenetic relationship of the host species. This neighbour-joining phylogenetic tree is, however, not powerful enough to extract information concerning the history of HOR structures, because the source sequences of the gibbon species (except for the case of <em>Nomascus leucogenys</em>) are limited to higher-order AS, while the consensus sequences of the hominid species represent both higher-order AS and monomeric AS.</p><p>The purpose of the present study was to show evidence, if any, for HOR-containing AS in multiple gibbon genera, which was attained with lines clearly appearing in constant intervals in the pairwise comparison tables. It is now of interest whether AS sequences of gibbons exhibit within-species variation in the HOR structure, and, if so, how large the variation is. In hominids, the within-species HOR structure variation is larger in humans than in great ape species, and the reason for the higher level of variation in humans is not known. Further studies by identifying different types of HOR structures in gibbons are expected to contribute to the understanding of factors leading to the high level of variation of AS structure in humans, and factors that are associated with the more important role of higher-order AS than monomeric AS in centromere function.</p>                            <h2 scrollto-destination=6676386 id="6676386" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6676388 id="6676388" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported by Grants-in-Aid (23657165 and 23470098 to A.K. and 22247037 to H.H.) from the Japan Society for the Promotion of Science.</p>                            <h2 scrollto-destination=6676390 id="6676390" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Hiroshi Masumoto and Masanaru Takai for helpful discussions.</p>                            <h2 scrollto-destination=6676392 id="6676392" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU005C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div>. </div><div class="article-title">Evolution of alpha satellite</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">1991</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">509</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20alpha%20satellite&amp;author=H.F.%20Willard&amp;publication_year=1991&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=1&amp;pages=509-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(05)80200-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(05)80200-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(05)80200-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1822282" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rudd</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Wray</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div>. </div><div class="article-title">The evolutionary dynamics of alpha-satellite</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">88</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20dynamics%20of%20alpha-satellite&amp;author=M.K.%20Rudd&amp;author=G.A.%20Wray&amp;author=H.F.%20Willard&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=88-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3810906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3810906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3810906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16344556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div>. </div><div class="article-title">Chromosome-specific organization of human alpha satellite DNA</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">1985</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">524</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome-specific%20organization%20of%20human%20alpha%20satellite%20DNA&amp;author=H.F.%20Willard&amp;publication_year=1985&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=37&amp;pages=524-32" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2988334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alexandrov</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kazakov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tumeneva</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Shepelev</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Yurov</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Alpha-satellite DNA of primates: old and new families</div>, <div class="source">Chromosoma</div> , <div class="year">2001</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">253</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alpha-satellite%20DNA%20of%20primates%3A%20old%20and%20new%20families&amp;author=I.%20Alexandrov&amp;author=A.%20Kazakov&amp;author=I.%20Tumeneva&amp;author=V.%20Shepelev&amp;author=Y.%20Yurov&amp;publication_year=2001&amp;journal=Chromosoma&amp;volume=110&amp;pages=253-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004120100146" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004120100146" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004120100146"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11534817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rudd</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div>. </div><div class="article-title">Analysis of the centromeric regions of the human genome assembly</div>, <div class="source">Trends Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">529</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20centromeric%20regions%20of%20the%20human%20genome%20assembly&amp;author=M.K.%20Rudd&amp;author=H.F.%20Willard&amp;publication_year=2004&amp;journal=Trends%20Genet.&amp;volume=20&amp;pages=529-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.08.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.08.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2004.08.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15475110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Spence</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Critcher</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ebersole</div> <div class="given-names">T.A.</div></div>, et al. </div><div class="article-title">Co-localization of centromere activity, proteins and topoisomerase II within a subdomain of the major human X α-satellite array</div>, <div class="source">EMBO J.</div> , <div class="year">2002</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">5269</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Co-localization%20of%20centromere%20activity%2C%20proteins%20and%20topoisomerase%20II%20within%20a%20subdomain%20of%20the%20major%20human%20X%20%CE%B1-satellite%20array&amp;author=J.M.%20Spence&amp;author=R.%20Critcher&amp;author=T.A.%20Ebersole&amp;publication_year=2002&amp;journal=EMBO%20J.&amp;volume=21&amp;pages=5269-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/cdf511" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2Fcdf511" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2Fcdf511"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12356743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haaf</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div>. </div><div class="article-title">Orangutan alpha-satellite monomers are closely related to the human consensus sequence</div>, <div class="source">Mamm. Genome</div> , <div class="year">1998</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">440</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Orangutan%20alpha-satellite%20monomers%20are%20closely%20related%20to%20the%20human%20consensus%20sequence&amp;author=T.%20Haaf&amp;author=H.F.%20Willard&amp;publication_year=1998&amp;journal=Mamm.%20Genome&amp;volume=9&amp;pages=440-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s003359900793" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs003359900793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs003359900793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9585431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waye</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Willard</div> <div class="given-names">H.F.</div></div>. </div><div class="article-title">Concerted evolution of alpha satellite DNA: evidence for species specificity and a general lack of sequence conservation among alphoid sequences of higher primates</div>, <div class="source">Chromosoma</div> , <div class="year">1989</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">273</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Concerted%20evolution%20of%20alpha%20satellite%20DNA%3A%20evidence%20for%20species%20specificity%20and%20a%20general%20lack%20of%20sequence%20conservation%20among%20alphoid%20sequences%20of%20higher%20primates&amp;author=J.S.%20Waye&amp;author=H.F.%20Willard&amp;publication_year=1989&amp;journal=Chromosoma&amp;volume=98&amp;pages=273-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00327313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00327313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00327313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2515043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Terada</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Koga</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Higher-order repeat structure in alpha satellite DNA is an attribute of hominoids rather than hominids</div>, <div class="source">J. Hum. Genet.</div> , <div class="year">2013</div><div class="pub-id-doi"><a href="https://doi.org/10.1038/jhg.2013.87" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/jhg.2013.87</a></div></div></div></div></div><div content-id="DSU005C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koga</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Repetitive sequences originating from the centromere constitute large-scale heterochromatin in the telomere region in the siamang, a small ape</div>, <div class="source">Heredity</div> , <div class="year">2012</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">180</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repetitive%20sequences%20originating%20from%20the%20centromere%20constitute%20large-scale%20heterochromatin%20in%20the%20telomere%20region%20in%20the%20siamang%2C%20a%20small%20ape&amp;author=A.%20Koga&amp;author=Y.%20Hirai&amp;author=T.%20Hara&amp;author=H.%20Hirai&amp;publication_year=2012&amp;journal=Heredity&amp;volume=109&amp;pages=180-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2012.28" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2012.28" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2012.28"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22669075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Royle</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Baird</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Jeffreys</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">A subterminal satellite located adjacent to telomeres in chimpanzees is absent from the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="year">1994</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">52</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20subterminal%20satellite%20located%20adjacent%20to%20telomeres%20in%20chimpanzees%20is%20absent%20from%20the%20human%20genome&amp;author=N.J.%20Royle&amp;author=D.M.%20Baird&amp;author=A.J.%20Jeffreys&amp;publication_year=1994&amp;journal=Nat.%20Genet.&amp;volume=6&amp;pages=52-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0194-52" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0194-52" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0194-52"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8136835" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koga</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Notohara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Evolution of subterminal satellite (StSat) repeats in hominids</div>, <div class="source">Genetica</div> , <div class="year">2011</div>, vol. <div class="volume">139</div> (pg. <div class="fpage">167</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20subterminal%20satellite%20(StSat)%20repeats%20in%20hominids&amp;author=A.%20Koga&amp;author=M.%20Notohara&amp;author=H.%20Hirai&amp;publication_year=2011&amp;journal=Genetica&amp;volume=139&amp;pages=167-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-010-9534-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-010-9534-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-010-9534-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21136140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="collab">International Human Genome Sequencing Consortium</div><div class="article-title">Finishing the euchromatic sequence of the human genome</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">431</div> (pg. <div class="fpage">931</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15496913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">She</div> <div class="given-names">X.</div></div>. </div><div class="article-title">An assessment of the sequence gaps: unfinished business in a finished human genome</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">345</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20assessment%20of%20the%20sequence%20gaps%3A%20unfinished%20business%20in%20a%20finished%20human%20genome&amp;author=E.E.%20Eichler&amp;author=R.A.%20Clark&amp;author=X.%20She&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=345-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1322" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1322" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1322"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15143317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brandon-Jones</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Eudey</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Geissmann</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Asian primate classification</div>, <div class="source">Int. J. Primatol.</div> , <div class="year">2004</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">97</div>-<div class="lpage">164</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asian%20primate%20classification&amp;author=D.%20Brandon-Jones&amp;author=A.A.%20Eudey&amp;author=T.%20Geissmann&amp;publication_year=2004&amp;journal=Int.%20J.%20Primatol.&amp;volume=25&amp;pages=97-164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:IJOP.0000014647.18720.32" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:IJOP.0000014647.18720.32" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:IJOP.0000014647.18720.32"> </span></div> </div></div></div></div></div><div content-id="DSU005C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thinh</div> <div class="given-names">V.N.</div></div>,  <div class="name"><div class="surname">Mootnick</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Geissmann</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Mitochondrial evidence for multiple radiations in the evolutionary history of small apes</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">74</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20evidence%20for%20multiple%20radiations%20in%20the%20evolutionary%20history%20of%20small%20apes&amp;author=V.N.%20Thinh&amp;author=A.R.%20Mootnick&amp;author=T.%20Geissmann&amp;publication_year=2010&amp;journal=BMC%20Evol.%20Biol.&amp;volume=10&amp;pages=74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-10-74" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-74" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-10-74"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20226039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prakhongcheep</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Srikulnath</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Koga</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Two types of alpha satellite DNA in distinct chromosomal locations in Azara's owl monkey</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">235</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20types%20of%20alpha%20satellite%20DNA%20in%20distinct%20chromosomal%20locations%20in%20Azara's%20owl%20monkey&amp;author=O.%20Prakhongcheep&amp;author=Y.%20Hirai&amp;author=T.%20Hara&amp;author=K.%20Srikulnath&amp;author=H.%20Hirai&amp;author=A.%20Koga&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=235-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23477842" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prakhongcheep</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Chaiprasertsri</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Terada</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Heterochromatin blocks constituting the entire short arms of acrocentric chromosomes of Azara's owl monkey: formation processes inferred from chromosomal locations</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">461</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterochromatin%20blocks%20constituting%20the%20entire%20short%20arms%20of%20acrocentric%20chromosomes%20of%20Azara's%20owl%20monkey%3A%20formation%20processes%20inferred%20from%20chromosomal%20locations&amp;author=O.%20Prakhongcheep&amp;author=N.%20Chaiprasertsri&amp;author=S.%20Terada&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=461-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23761219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Unidirectional digestion with exonuclease III creates targeted breakpoints for DNA sequencing</div>, <div class="source">Gene</div> , <div class="year">1984</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">351</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unidirectional%20digestion%20with%20exonuclease%20III%20creates%20targeted%20breakpoints%20for%20DNA%20sequencing&amp;author=S.%20Henikoff&amp;publication_year=1984&amp;journal=Gene&amp;volume=28&amp;pages=351-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(84)90153-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(84)90153-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(84)90153-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6235151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baicharoen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Arsaithamkul</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Koga</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">H.</div></div>. </div><div class="article-title">In situ hybridization analysis of gibbon chromosomes suggests that amplification of alpha satellite DNA in the telomere region is confined to two of the four genera</div>, <div class="source">Genome</div> , <div class="year">2012</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">809</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20situ%20hybridization%20analysis%20of%20gibbon%20chromosomes%20suggests%20that%20amplification%20of%20alpha%20satellite%20DNA%20in%20the%20telomere%20region%20is%20confined%20to%20two%20of%20the%20four%20genera&amp;author=S.%20Baicharoen&amp;author=V.%20Arsaithamkul&amp;author=Y.%20Hirai&amp;author=T.%20Hara&amp;author=A.%20Koga&amp;author=H.%20Hirai&amp;publication_year=2012&amp;journal=Genome&amp;volume=55&amp;pages=809-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-2012-0123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-2012-0123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-2012-0123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23199575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alexandrov</div> <div class="given-names">I.A.</div></div>,  <div class="name"><div class="surname">Medvedev</div> <div class="given-names">L.I.</div></div>,  <div class="name"><div class="surname">Mashkova</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Kisselev</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Romanova</div> <div class="given-names">L.Y.</div></div>,  <div class="name"><div class="surname">Yurov</div> <div class="given-names">Y.B.</div></div>. </div><div class="article-title">Definition of a new alpha satellite suprachromosomal family characterized by monomeric organization</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1993</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">2209</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Definition%20of%20a%20new%20alpha%20satellite%20suprachromosomal%20family%20characterized%20by%20monomeric%20organization&amp;author=I.A.%20Alexandrov&amp;author=L.I.%20Medvedev&amp;author=T.D.%20Mashkova&amp;author=L.L.%20Kisselev&amp;author=L.Y.%20Romanova&amp;author=Y.B.%20Yurov&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=2209-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.9.2209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.9.2209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.9.2209"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8502563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU005C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU005C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cellamare</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Catacchio</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Alkan</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">New insights into centromere organization and evolution from the white-cheeked gibbon and marmoset</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1889</div>-<div class="lpage">900</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20insights%20into%20centromere%20organization%20and%20evolution%20from%20the%20white-cheeked%20gibbon%20and%20marmoset&amp;author=A.%20Cellamare&amp;author=C.R.%20Catacchio&amp;author=C.%20Alkan&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=26&amp;pages=1889-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19429672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=446188&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu005-dsu005supp_fig3.pdf?Expires=2147483647&amp;Signature=pYsVhvhhB2a0d1JoG4xsCZ8VuVPUzOL2~t0rv0nHP9JCaJPqi5YIN9VpBB-D6P1khTPdq54FH-pm2k9BveY6kuJZKJ~yPRZeg4Epe2~WjqCIHp1dKPnNKpqL7GtO-zQ7JYuR~Q-UEBDANxm2CquduKyLrEW3pP9XLXKwdFA5L6FVqnKUgFV94F1tY0LD9fMZYjk~YsqIUjYh8PEZuK199pzIFvh25VRgc~WV8~TtErbRDDt~EjYcxcLgqhQtKs7TXo7e8Bo3fadkFb1NGZVAWKNFcCnkn10AILvqjDPQJhowPhv0Hc0X5InwczHYeHbAJ1mw9UI5Cspa1CyAYlseog__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu005-dsu005supp_fig2.pdf?Expires=2147483647&amp;Signature=WZC8UTm~KAyknwp13tRHSGRI8ERofzlyU4MyDjZV8zJsaEWyEV8xac6pglqOe15ynU9cz7a9YRE0H~syZ3~aJTwkxBkwJTlGaIB2ig3rvr866PthE2-ZRkz0FTMeSWqC7iZX3qTIaDXRGGiOpS5i86pxBeu7flyAEwXEiUiCTczAdn7oAwbpITOpL9c6PQzzn67zVsxXCKepB9aroGgd7nFfvrF9nT7Px-RmnN6HorJPhbrfpCg9~2pXagp7krKCtKJxkmcOKWR8JRI44eN4Zffp7XB1op1eUz98g0lUSaJohF8ubwSCbCMQmB3lRYVkGsccLK9tsKOPQCjS0NFsQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu005-dsu005supp_fig1.pdf?Expires=2147483647&amp;Signature=e1066EOjBazw6uTadohkIZHlTNyv~jCUUwoJOyIOdGpHtCri8ZxPu6xPTaQNnQit1jzuS5-bf0cgGZjcrN8GDtV8iyHdyJ~jXv~bVL0FWJsDbN9JGwVhGF8D2CsdeUj0rQAAEGiLgxwX3C-ek5OOKLiyLaXraYhLXsPtmXBq8IdORuZoSuaweaag9rpgkVXDpvsFctVdwy0qR~PP091H4LzoAZ3eYbEIuZ7MdW4haxDhlN23MSS5~s4zsSB~f1coTqVm5xvwXC2HgwfMmq~S7crAgIWTwcy2i-MPI22NWJZWznKSBnNDI4g~EbhQU1LnsfzuYfKaWk-qSPvpoM72XA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu005/2/dsu005-dsu005supp.docx?Expires=2147483647&amp;Signature=R-~7qsg1Y5~-hpWyzNnQSNLWvDmUzB3j-IaWy65~c~RoDlk2VOSNpzGfC2PsuYT4cUQDXyUKp5U8pDKx01rbcL8OG5tLmqevRMeCen1Frh5KvNATpy5aqxl6G8Zr0HLeyLRnHfYozSTPHceGRh6a~eZxvIIv2mj0kJynaEvpspJGVcp0gpZcfNsQy7TpgJS4JLp0Y3Evop40HBt1rdG4n6nlUq6gDpSW3KrWariy03FXrnmS8PH2wtAeXEmIoSa6IMmrQlZPy5flA6BCsO8RxMqYFJQbgL9vX2ghco59bMICksgDuxeyPFBjGt8U14EWxCcIu8ecjGmx~2EtVOrsCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - docx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">296</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="446188"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f4%2f407%2f446188">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344243900006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F4%2F407%2F446188" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/446188'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Evolutionary Origin of Higher-Order Repeat Structure in Alpha-Satellite DNA of Primate Centromeres" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f407%2f446188">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

