

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/6/711/2754555","@type":"ScholarlyArticle","name":"Genomic Analysis Reveals Distinct Concentration-Dependent Evolutionary Trajectories for Antibiotic Resistance in  Escherichia coli ","datePublished":"2014-10-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2014-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu032","keywords":["antibiotic resistance","aminoglycosides","evolution"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/6/711/2754555","author":[{"name":"Mogre, Aalap","affiliation":"1  National Centre for Biological Sciences ,  Tata Institute of Fundamental Research, GKVK  , Bellary Road, Bangalore, Karnataka 560065 ,  India","@type":"Person"},{"name":"Sengupta, Titas","affiliation":"1  National Centre for Biological Sciences ,  Tata Institute of Fundamental Research, GKVK  , Bellary Road, Bangalore, Karnataka 560065 ,  India","@type":"Person"},{"name":"Veetil, Reshma T.","affiliation":"1  National Centre for Biological Sciences ,  Tata Institute of Fundamental Research, GKVK  , Bellary Road, Bangalore, Karnataka 560065 ,  India","@type":"Person"},{"name":"Ravi, Preethi","affiliation":"1  National Centre for Biological Sciences ,  Tata Institute of Fundamental Research, GKVK  , Bellary Road, Bangalore, Karnataka 560065 ,  India","@type":"Person"},{"name":"Seshasayee, Aswin Sai Narain","affiliation":"1  National Centre for Biological Sciences ,  Tata Institute of Fundamental Research, GKVK  , Bellary Road, Bangalore, Karnataka 560065 ,  India","@type":"Person"}],"description":"Evolution of bacteria under sublethal concentrations of antibiotics represents a trade-off between growth and resistance to the antibiotic. To understand this t","pageStart":"711","pageEnd":"726","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genomic Analysis Reveals Distinct Concentration-Dependent Evolutionary Trajectories for Antibiotic Resistance in  Escherichia coli ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genomic Analysis Reveals Distinct Concentration-Dependent Evolutionary Trajectories for Antibiotic Resistance in  Escherichia coli " />
    <meta property="og:description" content="Evolution of bacteria under sublethal concentrations of antibiotics represents a trade-off between growth and resistance to the antibiotic. To understand this t" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/6/711/2754555" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-10-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Mogre, Aalap" /><meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research, GKVK, Bellary Road, Bangalore, Karnataka 560065, India" /><meta name="citation_author" content="Sengupta, Titas" /><meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research, GKVK, Bellary Road, Bangalore, Karnataka 560065, India" /><meta name="citation_author" content="Veetil, Reshma T." /><meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research, GKVK, Bellary Road, Bangalore, Karnataka 560065, India" /><meta name="citation_author" content="Ravi, Preethi" /><meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research, GKVK, Bellary Road, Bangalore, Karnataka 560065, India" /><meta name="citation_author" content="Seshasayee, Aswin Sai Narain" /><meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research, GKVK, Bellary Road, Bangalore, Karnataka 560065, India" /><meta name="citation_title" content="Genomic Analysis Reveals Distinct Concentration-Dependent Evolutionary Trajectories for Antibiotic Resistance in Escherichia coli" /><meta name="citation_firstpage" content="711" /><meta name="citation_lastpage" content="726" /><meta name="citation_doi" content="10.1093/dnares/dsu032" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2014/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=New insights into bacterial adaptation through in vivo and in silico experimental evolution;  citation_journal_title=Nat. Rev. Microbiol.;  citation_year=2012;  citation_volume=10;  citation_pages=352-65; " /><meta name="citation_reference" content="citation_title=Origins and evolution of antibiotic resistance;  citation_journal_title=Microbiol. Mol. Biol. Rev.;  citation_year=2010;  citation_volume=74;  citation_pages=417-33; " /><meta name="citation_reference" content="citation_title=Intrinsic antibiotic resistance: mechanisms, origins, challenges and solutions;  citation_journal_title=Int. J. Med. Microbiol.;  citation_year=2013;  citation_volume=303;  citation_pages=287-92; " /><meta name="citation_reference" content="citation_title=The role of natural environments in the evolution of resistance traits in pathogenic bacteria;  citation_journal_title=Proc. Biol. Sci.;  citation_year=2009;  citation_volume=276;  citation_pages=2521-30; " /><meta name="citation_reference" content="citation_title=A global view of antibiotic resistance;  citation_journal_title=FEMS Microbiol. Rev.;  citation_year=2009;  citation_volume=33;  citation_pages=44-65; " /><meta name="citation_reference" content="citation_title=The genomic enzymology of antibiotic resistance;  citation_journal_title=Ann. Rev. Genet.;  citation_year=2010;  citation_volume=44;  citation_pages=25-51; " /><meta name="citation_reference" content="citation_title=Antibiotics as signalling molecules;  citation_journal_title=Philos. Trans. R. Soc. Lond. B. Biol. Sci.;  citation_year=2007;  citation_volume=362;  citation_pages=1195-200; " /><meta name="citation_reference" content="citation_title=Compensatory evolution in rifampin-resistant Escherichia coli;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=156;  citation_pages=1471-81; " /><meta name="citation_reference" content="citation_title=Molecular basis and mechanisms of drug resistance in Mycobacterium tuberculosis: classical and new drugs;  citation_journal_title=J. Antimicrob. Chemother.;  citation_year=2011;  citation_volume=66;  citation_pages=1417-30; " /><meta name="citation_reference" content="citation_title=Multidrug-resistant tuberculosis and its control;  citation_journal_title=Res. Microbiol.;  citation_year=1993;  citation_volume=144;  citation_pages=104-10; " /><meta name="citation_reference" content="citation_title=Antibiotic resistance and its cost: is it possible to reverse resistance?;  citation_journal_title=Nat. Rev. Microbiol.;  citation_year=2010;  citation_volume=8;  citation_pages=260-71; " /><meta name="citation_reference" content="citation_title=Aminoglycosides: perspectives on mechanisms of action and resistance and strategies to counter resistance;  citation_journal_title=Antimicrob. Agents Chemother.;  citation_year=2000;  citation_volume=44;  citation_pages=3249-56; " /><meta name="citation_reference" content="citation_title=Aminoglycoside antibiotics in the 21st century;  citation_journal_title=ACS Chem. Biol.;  citation_year=2013;  citation_volume=8;  citation_pages=105-15; " /><meta name="citation_reference" content="citation_title=Misreading of ribonucleic acid code words induced by aminoglycoside antibiotics. The effect of drug concentration;  citation_journal_title=J. Biol. Chem.;  citation_year=1968;  citation_volume=243;  citation_pages=3312-6; " /><meta name="citation_reference" content="citation_title=The impact of aminoglycosides on the dynamics of translation elongation;  citation_journal_title=Cell Rep.;  citation_year=2013;  citation_volume=3;  citation_pages=497-508; " /><meta name="citation_reference" content="citation_title=Structural basis for aminoglycoside inhibition of bacterial ribosome recycling;  citation_journal_title=Nat. Struct. Mol. Biol.;  citation_year=2007;  citation_volume=14;  citation_pages=727-32; " /><meta name="citation_reference" content="citation_title=Interaction of kanamycin and related antibiotics with the large subunit of ribosomes and the inhibition of translocation;  citation_journal_title=Biochem. Biophys. Res. Commun.;  citation_year=1978;  citation_volume=84;  citation_pages=358-65; " /><meta name="citation_reference" content="citation_title=Inhibition of ribosomal translocation by aminoglycoside antibiotics;  citation_journal_title=Biochem. Biophys. Res. Commun.;  citation_year=1978;  citation_volume=83;  citation_pages=991-7; " /><meta name="citation_reference" content="citation_title=Post-termination complex disassembly by ribosome recycling factor, a functional tRNA mimic;  citation_journal_title=EMBO J.;  citation_year=2002;  citation_volume=21;  citation_pages=2272-81; " /><meta name="citation_reference" content="citation_title=Translation rates and misreading characteristics of rpsD mutants in Escherichia coli;  citation_journal_title=Mol. Gen. Genet.;  citation_year=1982;  citation_volume=187;  citation_pages=467-72; " /><meta name="citation_reference" content="citation_title=Translational accuracy and the fitness of bacteria;  citation_journal_title=Ann. Rev. Genet.;  citation_year=1992;  citation_volume=26;  citation_pages=29-50; " /><meta name="citation_reference" content="citation_title=A ribosomal ambiguity mutation in the 530 loop of E. coli 16S rRNA;  citation_journal_title=Nucleic Acids Res.;  citation_year=1992;  citation_volume=20;  citation_pages=4221-7; " /><meta name="citation_reference" content="citation_title=Mechanism of bactericidal action of aminoglycosides;  citation_journal_title=Microbiol. Rev.;  citation_year=1987;  citation_volume=51;  citation_pages=341-50; " /><meta name="citation_reference" content="citation_title=Killing by bactericidal antibiotics does not depend on reactive oxygen species;  citation_journal_title=Science;  citation_year=2013;  citation_volume=339;  citation_pages=1213-6; " /><meta name="citation_reference" content="citation_title=How antibiotics kill bacteria: from targets to networks;  citation_journal_title=Nat. Rev. Microbiol.;  citation_year=2010;  citation_volume=8;  citation_pages=423-35; " /><meta name="citation_reference" content="citation_title=Cell death from antibiotics without the involvement of reactive oxygen species;  citation_journal_title=Science;  citation_year=2013;  citation_volume=339;  citation_pages=1210-3; " /><meta name="citation_reference" content="citation_title=Molecular genetics of aminoglycoside resistance genes and familial relationships of the aminoglycoside-modifying enzymes;  citation_journal_title=Microbiol. Rev.;  citation_year=1993;  citation_volume=57;  citation_pages=138-63; " /><meta name="citation_reference" content="citation_title=Basis for prokaryotic specificity of action of aminoglycoside antibiotics;  citation_journal_title=EMBO J.;  citation_year=1999;  citation_volume=18;  citation_pages=3133-8; " /><meta name="citation_reference" content="citation_title=RNA sequence determinants for aminoglycoside binding to an A-site rRNA model oligonucleotide;  citation_journal_title=J. Mol. Biol.;  citation_year=1996;  citation_volume=262;  citation_pages=421-36; " /><meta name="citation_reference" content="citation_title=Mutations conferring aminoglycoside and spectinomycin resistance in Borrelia burgdorferi;  citation_journal_title=Antimicrob. Agents Chemother.;  citation_year=2006;  citation_volume=50;  citation_pages=445-52; " /><meta name="citation_reference" content="citation_title=Aminoglycoside resistance with homogeneous and heterogeneous populations of antibiotic-resistant ribosomes;  citation_journal_title=Antimicrob. Agents Chemother.;  citation_year=2001;  citation_volume=45;  citation_pages=2414-9; " /><meta name="citation_reference" content="citation_title=In vivo selection of conditional-lethal mutations in the gene encoding elongation factor G of Escherichia coli;  citation_journal_title=J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=123-9; " /><meta name="citation_reference" content="citation_title=Bacterial evolution of antibiotic hypersensitivity;  citation_journal_title=Mol. Syst. Biol.;  citation_year=2013;  citation_volume=9;  citation_pages=700" /><meta name="citation_reference" content="citation_title=Mutation as a stress response and the regulation of evolvability;  citation_journal_title=Crit. Rev. Biochem. Mol. Biol.;  citation_year=2007;  citation_volume=42;  citation_pages=399-435; " /><meta name="citation_reference" content="citation_title=SOS response promotes horizontal dissemination of antibiotic resistance genes;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=427;  citation_pages=72-4; " /><meta name="citation_reference" content="citation_title=Selection of resistant bacteria at very low antibiotic concentrations;  citation_journal_title=PLoS Pathog.;  citation_year=2011;  citation_volume=7;  citation_pages=e1002158" /><meta name="citation_reference" content="citation_title=Bacterial charity work leads to population-wide resistance;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=467;  citation_pages=82-5; " /><meta name="citation_reference" content="citation_title=Evolutionary paths to antibiotic resistance under dynamically sustained drug selection;  citation_journal_title=Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=101-5; " /><meta name="citation_reference" content="citation_title=Acceleration of emergence of bacterial antibiotic resistance in connected microenvironments;  citation_journal_title=Science;  citation_year=2011;  citation_volume=333;  citation_pages=1764-7; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinforma. Oxf. Engl.;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools;  citation_journal_title=Bioinforma. Oxf. Engl.;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=VarScan: variant detection in massively parallel sequencing of individual and pooled samples;  citation_journal_title=Bioinforma. Oxf. Engl.;  citation_year=2009;  citation_volume=25;  citation_pages=2283-5; " /><meta name="citation_reference" content="citation_title=Identification of mutations in laboratory-evolved microbes from next-generation sequencing data using breseq;  citation_journal_title=Methods Mol. Biol.;  citation_year=2014;  citation_volume=1151;  citation_pages=165-88; " /><meta name="citation_reference" content="citation_title=One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=6640-5; " /><meta name="citation_reference" content="citation_title=E. coli genome manipulation by P1 transduction;  citation_journal_title=Curr. Protoc. Mol. Biol. Ed. Frederick M Ausubel Al;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=FACS-optimized mutants of the green fluorescent protein (GFP);  citation_journal_title=Gene;  citation_year=1996;  citation_volume=173;  citation_pages=33-8; " /><meta name="citation_reference" content="citation_title=Tight regulation, modulation, and high-level expression by vectors containing the arabinose PBAD promoter;  citation_journal_title=J. Bacteriol.;  citation_year=1995;  citation_volume=177;  citation_pages=4121-30; " /><meta name="citation_reference" content="citation_title=A comprehensive library of fluorescent transcriptional reporters for Escherichia coli;  citation_journal_title=Nat. Methods;  citation_year=2006;  citation_volume=3;  citation_pages=623-8; " /><meta name="citation_reference" content="citation_title=The Cpx stress response confers resistance to some, but not all, bactericidal antibiotics;  citation_journal_title=J. Bacteriol.;  citation_year=2013;  citation_volume=195;  citation_pages=1869-74; " /><meta name="citation_reference" content="citation_title=Fitness benefits in fluoroquinolone-resistant Salmonella Typhi in the absence of antimicrobial pressure;  citation_journal_title=eLife;  citation_year=2013;  citation_volume=2;  citation_pages=e01229" /><meta name="citation_reference" content="citation_title=Molecular dynamics of EF-G during translocation;  citation_journal_title=Proteins;  citation_year=2011;  citation_volume=79;  citation_pages=1478-86; " /><meta name="citation_reference" content="citation_title=A common mechanism of cellular death induced by bactericidal antibiotics;  citation_journal_title=Cell;  citation_year=2007;  citation_volume=130;  citation_pages=797-810; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/6/711/17463174/dsu032.pdf" /><meta name="description" content="Evolution of bacteria under sublethal concentrations of antibiotics represents a trade-off between growth and resistance to the antibiotic. To understand this t" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/6/711/2754555" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/6/711/2754555" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/6/711/2754555");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu032';
    </script>
<title>
	Genomic Analysis Reveals Distinct Concentration-Dependent Evolutionary Trajectories for Antibiotic Resistance in Escherichia coli | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754555" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbF2PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb2dyZSwgQWFsYXAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBDZW50cmUgZm9yIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIFRhdGEgSW5zdGl0dXRlIG9mIEZ1bmRhbWVudGFsIFJlc2VhcmNoLCBHS1ZLLCBCZWxsYXJ5IFJvYWQsIEJhbmdhbG9yZSwgS2FybmF0YWthIDU2MDA2NSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTZW5ndXB0YSwgVGl0YXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBDZW50cmUgZm9yIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIFRhdGEgSW5zdGl0dXRlIG9mIEZ1bmRhbWVudGFsIFJlc2VhcmNoLCBHS1ZLLCBCZWxsYXJ5IFJvYWQsIEJhbmdhbG9yZSwgS2FybmF0YWthIDU2MDA2NSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJWZWV0aWwsIFJlc2htYSBULiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIENlbnRyZSBmb3IgQmlvbG9naWNhbCBTY2llbmNlcywgVGF0YSBJbnN0aXR1dGUgb2YgRnVuZGFtZW50YWwgUmVzZWFyY2gsIEdLVkssIEJlbGxhcnkgUm9hZCwgQmFuZ2Fsb3JlLCBLYXJuYXRha2EgNTYwMDY1LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJhdmksIFByZWV0aGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBDZW50cmUgZm9yIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIFRhdGEgSW5zdGl0dXRlIG9mIEZ1bmRhbWVudGFsIFJlc2VhcmNoLCBHS1ZLLCBCZWxsYXJ5IFJvYWQsIEJhbmdhbG9yZSwgS2FybmF0YWthIDU2MDA2NSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTZXNoYXNheWVlLCBBc3dpbiBTYWkgTmFyYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQ2VudHJlIGZvciBCaW9sb2dpY2FsIFNjaWVuY2VzLCBUYXRhIEluc3RpdHV0ZSBvZiBGdW5kYW1lbnRhbCBSZXNlYXJjaCwgR0tWSywgQmVsbGFyeSBSb2FkLCBCYW5nYWxvcmUsIEthcm5hdGFrYSA1NjAwNjUsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWljIEFuYWx5c2lzIFJldmVhbHMgRGlzdGluY3QgQ29uY2VudHJhdGlvbi1EZXBlbmRlbnQgRXZvbHV0aW9uYXJ5IFRyYWplY3RvcmllcyBmb3IgQW50aWJpb3RpYyBSZXNpc3RhbmNlIGluIEVzY2hlcmljaGlhIGNvbGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI3MTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjcyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdTAzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV3IGluc2lnaHRzIGludG8gYmFjdGVyaWFsIGFkYXB0YXRpb24gdGhyb3VnaCBpbiB2aXZvIGFuZCBpbiBzaWxpY28gZXhwZXJpbWVudGFsIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0zNTItNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yaWdpbnMgYW5kIGV2b2x1dGlvbiBvZiBhbnRpYmlvdGljIHJlc2lzdGFuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbC4gTW9sLiBCaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9NDE3LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyaW5zaWMgYW50aWJpb3RpYyByZXNpc3RhbmNlOiBtZWNoYW5pc21zLCBvcmlnaW5zLCBjaGFsbGVuZ2VzIGFuZCBzb2x1dGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gTWVkLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMwMzsgIGNpdGF0aW9uX3BhZ2VzPTI4Ny05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgbmF0dXJhbCBlbnZpcm9ubWVudHMgaW4gdGhlIGV2b2x1dGlvbiBvZiByZXNpc3RhbmNlIHRyYWl0cyBpbiBwYXRob2dlbmljIGJhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBCaW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3NjsgIGNpdGF0aW9uX3BhZ2VzPTI1MjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2xvYmFsIHZpZXcgb2YgYW50aWJpb3RpYyByZXNpc3RhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTQ0LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21pYyBlbnp5bW9sb2d5IG9mIGFudGliaW90aWMgcmVzaXN0YW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTI1LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRpYmlvdGljcyBhcyBzaWduYWxsaW5nIG1vbGVjdWxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGhpbG9zLiBUcmFucy4gUi4gU29jLiBMb25kLiBCLiBCaW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM2MjsgIGNpdGF0aW9uX3BhZ2VzPTExOTUtMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wZW5zYXRvcnkgZXZvbHV0aW9uIGluIHJpZmFtcGluLXJlc2lzdGFudCBFc2NoZXJpY2hpYSBjb2xpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0xNDcxLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYmFzaXMgYW5kIG1lY2hhbmlzbXMgb2YgZHJ1ZyByZXNpc3RhbmNlIGluIE15Y29iYWN0ZXJpdW0gdHViZXJjdWxvc2lzOiBjbGFzc2ljYWwgYW5kIG5ldyBkcnVnczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW50aW1pY3JvYi4gQ2hlbW90aGVyLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02NjsgIGNpdGF0aW9uX3BhZ2VzPTE0MTctMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpZHJ1Zy1yZXNpc3RhbnQgdHViZXJjdWxvc2lzIGFuZCBpdHMgY29udHJvbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UmVzLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTE0NDsgIGNpdGF0aW9uX3BhZ2VzPTEwNC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aWJpb3RpYyByZXNpc3RhbmNlIGFuZCBpdHMgY29zdDogaXMgaXQgcG9zc2libGUgdG8gcmV2ZXJzZSByZXNpc3RhbmNlPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTI2MC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1pbm9nbHljb3NpZGVzOiBwZXJzcGVjdGl2ZXMgb24gbWVjaGFuaXNtcyBvZiBhY3Rpb24gYW5kIHJlc2lzdGFuY2UgYW5kIHN0cmF0ZWdpZXMgdG8gY291bnRlciByZXNpc3RhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbnRpbWljcm9iLiBBZ2VudHMgQ2hlbW90aGVyLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTMyNDktNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtaW5vZ2x5Y29zaWRlIGFudGliaW90aWNzIGluIHRoZSAyMXN0IGNlbnR1cnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFDUyBDaGVtLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTA1LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXNyZWFkaW5nIG9mIHJpYm9udWNsZWljIGFjaWQgY29kZSB3b3JkcyBpbmR1Y2VkIGJ5IGFtaW5vZ2x5Y29zaWRlIGFudGliaW90aWNzLiBUaGUgZWZmZWN0IG9mIGRydWcgY29uY2VudHJhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5Njg7ICBjaXRhdGlvbl92b2x1bWU9MjQzOyAgY2l0YXRpb25fcGFnZXM9MzMxMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW1wYWN0IG9mIGFtaW5vZ2x5Y29zaWRlcyBvbiB0aGUgZHluYW1pY3Mgb2YgdHJhbnNsYXRpb24gZWxvbmdhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbCBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz00OTctNTA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGJhc2lzIGZvciBhbWlub2dseWNvc2lkZSBpbmhpYml0aW9uIG9mIGJhY3RlcmlhbCByaWJvc29tZSByZWN5Y2xpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gU3RydWN0LiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NzI3LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmFjdGlvbiBvZiBrYW5hbXljaW4gYW5kIHJlbGF0ZWQgYW50aWJpb3RpY3Mgd2l0aCB0aGUgbGFyZ2Ugc3VidW5pdCBvZiByaWJvc29tZXMgYW5kIHRoZSBpbmhpYml0aW9uIG9mIHRyYW5zbG9jYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MTk3ODsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTM1OC02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5oaWJpdGlvbiBvZiByaWJvc29tYWwgdHJhbnNsb2NhdGlvbiBieSBhbWlub2dseWNvc2lkZSBhbnRpYmlvdGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0xOTc4OyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9OTkxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvc3QtdGVybWluYXRpb24gY29tcGxleCBkaXNhc3NlbWJseSBieSByaWJvc29tZSByZWN5Y2xpbmcgZmFjdG9yLCBhIGZ1bmN0aW9uYWwgdFJOQSBtaW1pYzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIyNzItODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zbGF0aW9uIHJhdGVzIGFuZCBtaXNyZWFkaW5nIGNoYXJhY3RlcmlzdGljcyBvZiBycHNEIG11dGFudHMgaW4gRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz00NjctNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zbGF0aW9uYWwgYWNjdXJhY3kgYW5kIHRoZSBmaXRuZXNzIG9mIGJhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MjktNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmlib3NvbWFsIGFtYmlndWl0eSBtdXRhdGlvbiBpbiB0aGUgNTMwIGxvb3Agb2YgRS4gY29saSAxNlMgclJOQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDIyMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc20gb2YgYmFjdGVyaWNpZGFsIGFjdGlvbiBvZiBhbWlub2dseWNvc2lkZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTM0MS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S2lsbGluZyBieSBiYWN0ZXJpY2lkYWwgYW50aWJpb3RpY3MgZG9lcyBub3QgZGVwZW5kIG9uIHJlYWN0aXZlIG94eWdlbiBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMzOTsgIGNpdGF0aW9uX3BhZ2VzPTEyMTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG93IGFudGliaW90aWNzIGtpbGwgYmFjdGVyaWE6IGZyb20gdGFyZ2V0cyB0byBuZXR3b3JrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTQyMy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbCBkZWF0aCBmcm9tIGFudGliaW90aWNzIHdpdGhvdXQgdGhlIGludm9sdmVtZW50IG9mIHJlYWN0aXZlIG94eWdlbiBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMzOTsgIGNpdGF0aW9uX3BhZ2VzPTEyMTAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGdlbmV0aWNzIG9mIGFtaW5vZ2x5Y29zaWRlIHJlc2lzdGFuY2UgZ2VuZXMgYW5kIGZhbWlsaWFsIHJlbGF0aW9uc2hpcHMgb2YgdGhlIGFtaW5vZ2x5Y29zaWRlLW1vZGlmeWluZyBlbnp5bWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0xMzgtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2lzIGZvciBwcm9rYXJ5b3RpYyBzcGVjaWZpY2l0eSBvZiBhY3Rpb24gb2YgYW1pbm9nbHljb3NpZGUgYW50aWJpb3RpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0zMTMzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBzZXF1ZW5jZSBkZXRlcm1pbmFudHMgZm9yIGFtaW5vZ2x5Y29zaWRlIGJpbmRpbmcgdG8gYW4gQS1zaXRlIHJSTkEgbW9kZWwgb2xpZ29udWNsZW90aWRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI2MjsgIGNpdGF0aW9uX3BhZ2VzPTQyMS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25zIGNvbmZlcnJpbmcgYW1pbm9nbHljb3NpZGUgYW5kIHNwZWN0aW5vbXljaW4gcmVzaXN0YW5jZSBpbiBCb3JyZWxpYSBidXJnZG9yZmVyaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW50aW1pY3JvYi4gQWdlbnRzIENoZW1vdGhlci47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz00NDUtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtaW5vZ2x5Y29zaWRlIHJlc2lzdGFuY2Ugd2l0aCBob21vZ2VuZW91cyBhbmQgaGV0ZXJvZ2VuZW91cyBwb3B1bGF0aW9ucyBvZiBhbnRpYmlvdGljLXJlc2lzdGFudCByaWJvc29tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFudGltaWNyb2IuIEFnZW50cyBDaGVtb3RoZXIuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9MjQxNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiB2aXZvIHNlbGVjdGlvbiBvZiBjb25kaXRpb25hbC1sZXRoYWwgbXV0YXRpb25zIGluIHRoZSBnZW5lIGVuY29kaW5nIGVsb25nYXRpb24gZmFjdG9yIEcgb2YgRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzY7ICBjaXRhdGlvbl9wYWdlcz0xMjMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGV2b2x1dGlvbiBvZiBhbnRpYmlvdGljIGh5cGVyc2Vuc2l0aXZpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTcwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGF0aW9uIGFzIGEgc3RyZXNzIHJlc3BvbnNlIGFuZCB0aGUgcmVndWxhdGlvbiBvZiBldm9sdmFiaWxpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyaXQuIFJldi4gQmlvY2hlbS4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTM5OS00MzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNPUyByZXNwb25zZSBwcm9tb3RlcyBob3Jpem9udGFsIGRpc3NlbWluYXRpb24gb2YgYW50aWJpb3RpYyByZXNpc3RhbmNlIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDI3OyAgY2l0YXRpb25fcGFnZXM9NzItNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW9uIG9mIHJlc2lzdGFudCBiYWN0ZXJpYSBhdCB2ZXJ5IGxvdyBhbnRpYmlvdGljIGNvbmNlbnRyYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIFBhdGhvZy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyMTU4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGNoYXJpdHkgd29yayBsZWFkcyB0byBwb3B1bGF0aW9uLXdpZGUgcmVzaXN0YW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NzsgIGNpdGF0aW9uX3BhZ2VzPTgyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBwYXRocyB0byBhbnRpYmlvdGljIHJlc2lzdGFuY2UgdW5kZXIgZHluYW1pY2FsbHkgc3VzdGFpbmVkIGRydWcgc2VsZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTEwMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2NlbGVyYXRpb24gb2YgZW1lcmdlbmNlIG9mIGJhY3RlcmlhbCBhbnRpYmlvdGljIHJlc2lzdGFuY2UgaW4gY29ubmVjdGVkIG1pY3JvZW52aXJvbm1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTMzMzsgIGNpdGF0aW9uX3BhZ2VzPTE3NjQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hLiBPeGYuIEVuZ2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTc1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNlcXVlbmNlIEFsaWdubWVudC9NYXAgZm9ybWF0IGFuZCBTQU10b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYS4gT3hmLiBFbmdsLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyU2NhbjogdmFyaWFudCBkZXRlY3Rpb24gaW4gbWFzc2l2ZWx5IHBhcmFsbGVsIHNlcXVlbmNpbmcgb2YgaW5kaXZpZHVhbCBhbmQgcG9vbGVkIHNhbXBsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWEuIE94Zi4gRW5nbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMjgzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIG11dGF0aW9ucyBpbiBsYWJvcmF0b3J5LWV2b2x2ZWQgbWljcm9iZXMgZnJvbSBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhIHVzaW5nIGJyZXNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExNTE7ICBjaXRhdGlvbl9wYWdlcz0xNjUtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uZS1zdGVwIGluYWN0aXZhdGlvbiBvZiBjaHJvbW9zb21hbCBnZW5lcyBpbiBFc2NoZXJpY2hpYSBjb2xpIEstMTIgdXNpbmcgUENSIHByb2R1Y3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTk3OyAgY2l0YXRpb25fcGFnZXM9NjY0MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FLiBjb2xpIGdlbm9tZSBtYW5pcHVsYXRpb24gYnkgUDEgdHJhbnNkdWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBQcm90b2MuIE1vbC4gQmlvbC4gRWQuIEZyZWRlcmljayBNIEF1c3ViZWwgQWw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZBQ1Mtb3B0aW1pemVkIG11dGFudHMgb2YgdGhlIGdyZWVuIGZsdW9yZXNjZW50IHByb3RlaW4gKEdGUCk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTczOyAgY2l0YXRpb25fcGFnZXM9MzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGlnaHQgcmVndWxhdGlvbiwgbW9kdWxhdGlvbiwgYW5kIGhpZ2gtbGV2ZWwgZXhwcmVzc2lvbiBieSB2ZWN0b3JzIGNvbnRhaW5pbmcgdGhlIGFyYWJpbm9zZSBQQkFEIHByb21vdGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE3NzsgIGNpdGF0aW9uX3BhZ2VzPTQxMjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSBsaWJyYXJ5IG9mIGZsdW9yZXNjZW50IHRyYW5zY3JpcHRpb25hbCByZXBvcnRlcnMgZm9yIEVzY2hlcmljaGlhIGNvbGk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NjIzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDcHggc3RyZXNzIHJlc3BvbnNlIGNvbmZlcnMgcmVzaXN0YW5jZSB0byBzb21lLCBidXQgbm90IGFsbCwgYmFjdGVyaWNpZGFsIGFudGliaW90aWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE5NTsgIGNpdGF0aW9uX3BhZ2VzPTE4NjktNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpdG5lc3MgYmVuZWZpdHMgaW4gZmx1b3JvcXVpbm9sb25lLXJlc2lzdGFudCBTYWxtb25lbGxhIFR5cGhpIGluIHRoZSBhYnNlbmNlIG9mIGFudGltaWNyb2JpYWwgcHJlc3N1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPWVMaWZlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lMDEyMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgZHluYW1pY3Mgb2YgRUYtRyBkdXJpbmcgdHJhbnNsb2NhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvdGVpbnM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Nzk7ICBjaXRhdGlvbl9wYWdlcz0xNDc4LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbW1vbiBtZWNoYW5pc20gb2YgY2VsbHVsYXIgZGVhdGggaW5kdWNlZCBieSBiYWN0ZXJpY2lkYWwgYW50aWJpb3RpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTMwOyAgY2l0YXRpb25fcGFnZXM9Nzk3LTgxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzYvNzExLzE3NDYzMTc0L2RzdTAzMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkV2b2x1dGlvbiBvZiBiYWN0ZXJpYSB1bmRlciBzdWJsZXRoYWwgY29uY2VudHJhdGlvbnMgb2YgYW50aWJpb3RpY3MgcmVwcmVzZW50cyBhIHRyYWRlLW9mZiBiZXR3ZWVuIGdyb3d0aCBhbmQgcmVzaXN0YW5jZSB0byB0aGUgYW50aWJpb3RpYy4gVG8gdW5kZXJzdGFuZCB0aGlzIHQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI3NTQ1NTUmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkw6YhUZvD0iqxIzQm1UAhQLPpWRY=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f711%2f2754555" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f711%2f2754555" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=22047" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&amp;Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/6">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199751">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199751">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199758">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199758">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199779">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199779">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199800">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199800">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199811">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199811">5. Accession numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199813">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199813">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199815">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199815">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199817">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199817">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/6/695/2754554">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/6/727/2754556">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genomic Analysis Reveals Distinct Concentration-Dependent Evolutionary Trajectories for Antibiotic Resistance in <em>Escherichia coli</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aalap Mogre</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Centre for Biological Sciences</div>, <div class="insititution">Tata Institute of Fundamental Research, GKVK</div><div class="addr-line">, Bellary Road, Bangalore, Karnataka 560065</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 80-6717-6502. Fax. +91 80-2363-6662. Email: <a href="mailto:aswin@ncbs.res.in" target="_blank">aswin@ncbs.res.in</a> (A.S.N.S); <a href="mailto:aalapbm@ncbs.res.in" target="_blank">aalapbm@ncbs.res.in</a> (A.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aalap+Mogre">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mogre A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mogre A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aalap Mogre</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Titas Sengupta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Centre for Biological Sciences</div>, <div class="insititution">Tata Institute of Fundamental Research, GKVK</div><div class="addr-line">, Bellary Road, Bangalore, Karnataka 560065</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Titas+Sengupta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sengupta T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sengupta T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Titas Sengupta</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Reshma T. Veetil</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Centre for Biological Sciences</div>, <div class="insititution">Tata Institute of Fundamental Research, GKVK</div><div class="addr-line">, Bellary Road, Bangalore, Karnataka 560065</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Reshma+T.+Veetil">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Veetil R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Veetil R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Reshma T. Veetil</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Preethi Ravi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Centre for Biological Sciences</div>, <div class="insititution">Tata Institute of Fundamental Research, GKVK</div><div class="addr-line">, Bellary Road, Bangalore, Karnataka 560065</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Preethi+Ravi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ravi P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ravi P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Preethi Ravi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aswin Sai Narain Seshasayee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Centre for Biological Sciences</div>, <div class="insititution">Tata Institute of Fundamental Research, GKVK</div><div class="addr-line">, Bellary Road, Bangalore, Karnataka 560065</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 80-6717-6502. Fax. +91 80-2363-6662. Email: <a href="mailto:aswin@ncbs.res.in" target="_blank">aswin@ncbs.res.in</a> (A.S.N.S); <a href="mailto:aalapbm@ncbs.res.in" target="_blank">aalapbm@ncbs.res.in</a> (A.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aswin+Sai+Narain+Seshasayee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seshasayee A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Seshasayee A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aswin Sai Narain Seshasayee</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 711–726, <a href='https://doi.org/10.1093/dnares/dsu032'>https://doi.org/10.1093/dnares/dsu032</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 October 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 May 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">07 September 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754555" href="/dnaresearch/article-pdf/21/6/711/17463174/dsu032.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Aalap Mogre, Titas Sengupta, Reshma T. Veetil, Preethi Ravi, Aswin Sai Narain Seshasayee;  Genomic Analysis Reveals Distinct Concentration-Dependent Evolutionary Trajectories for Antibiotic Resistance in <em>Escherichia coli</em>, <em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 711–726, <a href='https://doi.org/10.1093/dnares/dsu032'>https://doi.org/10.1093/dnares/dsu032</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754555?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754555?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754555?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754555?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754555?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754555?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754555?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genomic%20Analysis%20Reveals%20Distinct%20Concentration-Dependent%20Evolutionary%20Trajectories%20for%20Antibiotic%20Resistance%20in%20%26lt%3Bem%26gt%3BEscherichia%20coli%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2014-10-03&amp;volumeNum=21&amp;issueNum=6&amp;author=Mogre%2C%20Aalap%3B%20Sengupta%2C%20Titas&amp;startPage=711&amp;endPage=726&amp;contentId=10.1093%2Fdnares%2Fdsu032&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f711%2f2754555" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>Evolution of bacteria under sublethal concentrations of antibiotics represents a trade-off between growth and resistance to the antibiotic. To understand this trade-off, we performed <em>in vitro</em> evolution of laboratory <em>Escherichia coli</em> under sublethal concentrations of the aminoglycoside kanamycin over short time durations. We report that fixation of less costly kanamycin-resistant mutants occurred earlier in populations growing at lower sublethal concentration of the antibiotic, compared with those growing at higher sublethal concentrations; in the latter, resistant mutants with a significant growth defect persisted longer. Using deep sequencing, we identified kanamycin resistance-conferring mutations, which were costly or not in terms of growth in the absence of the antibiotic. Multiple mutations in the C-terminal end of domain IV of the translation elongation factor EF-G provided low-cost resistance to kanamycin. Despite targeting the same or adjacent residues of the protein, these mutants differed from each other in the levels of resistance they provided. Analysis of one of these mutations showed that it has little defect in growth or in synthesis of green fluorescent protein (GFP) from an inducible plasmid in the absence of the antibiotic. A second class of mutations, recovered only during evolution in higher sublethal concentrations of the antibiotic, deleted the C-terminal end of the ATP synthase shaft. This mutation confers basal-level resistance to kanamycin while showing a strong growth defect in the absence of the antibiotic. In conclusion, the early dynamics of the development of resistance to an aminoglycoside antibiotic is dependent on the levels of stress (concentration) imposed by the antibiotic, with the evolution of less costly variants only a matter of time.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;antibiotic resistance&quot;">antibiotic resistance</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="aminoglycosides">aminoglycosides</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a></div>                            <h2 scrollto-destination=87199751 id="87199751" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Bacteria have an enormous potential to adapt and evolve,<sup><a reveal-id="DSU032C1" data-open="DSU032C1" class="link link-ref link-reveal xref-bibr">1</a></sup> which is exemplified by the emergence of pathogens which are resistant to antibiotics. In this light, and with the diminishing pipeline of novel drugs, a ‘return to the pre-antibiotic era’ is almost certain unless we explore new means to counter resistance.<sup><a reveal-id="DSU032C2" data-open="DSU032C2" class="link link-ref link-reveal xref-bibr">2</a></sup> An important step in this process is achieving a greater understanding of how antibiotics kill cells and how resistant cells counter their action.</p><p>Mechanisms of antibiotic resistance include horizontal acquisition of genes for enzymes that modify the antibiotic or its target, or transporters that export them out of the cell.<sup><a reveal-id="DSU032C3" data-open="DSU032C3" class="link link-ref link-reveal xref-bibr">3</a></sup> For example, β-lactamases are enzymes that hydrolyze antibiotics such as penicillin. In various enteric pathogens, the transport system of AcrAB-TolC can export multiple drugs. Many antibiotics are prevalent in the natural environment, outside their role as drugs. This implies that there is a large pool of antibiotic resistance determinants in the environment.<sup><a reveal-id="DSU032C4" data-open="DSU032C4" class="link link-ref link-reveal xref-bibr">4</a>–<a reveal-id="DSU032C7" data-open="DSU032C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Additionally, mutations in the genome, which alter the target of the antibiotic or otherwise affect processes involved in antibiotic action, can also confer resistance. For example, treatment of <em>Escherichia coli</em> with the antibiotic rifampicin readily selects for mutations in its target RNA polymerase, leading to resistance.<sup><a reveal-id="DSU032C8" data-open="DSU032C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Point mutations on the genome are in fact the primary means of antibiotic resistance in <em>Mycobacterium tuberculosis</em>.<sup><a reveal-id="DSU032C9" data-open="DSU032C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU032C10" data-open="DSU032C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, since antibiotics target well-conserved and essential cellular processes, many resistance-conferring mutations in antibiotic targets might have adverse effects on fitness in the absence of the antibiotic.<sup><a reveal-id="DSU032C11" data-open="DSU032C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>Aminoglycosides are a class of ribosome-targeting bactericidal antibiotics that bind the 16S rRNA of the 30S ribosomal subunit and induce mistranslation as well as translational blockade.<sup><a reveal-id="DSU032C12" data-open="DSU032C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Aminoglycosides have a streptamine or a 2-deoxystreptamine (2-DOS) ring. The commonly used antibiotic streptomycin belongs to the former class. The latter group of aminoglycosides includes antibiotics such as kanamycin, gentamicin, neomycin and paromomycin.<sup><a reveal-id="DSU032C13" data-open="DSU032C13" class="link link-ref link-reveal xref-bibr">13</a></sup> 2-DOS aminoglycosides typically bind to the ribosomal decoding site in the 16S rRNA and cause a conformational change, which leads to the recognition of near-cognate codons and therefore miscoding.<sup><a reveal-id="DSU032C14" data-open="DSU032C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU032C15" data-open="DSU032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> This interaction, along with the binding of the antibiotic to the 23S rRNA in the large subunit,<sup><a reveal-id="DSU032C16" data-open="DSU032C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU032C17" data-open="DSU032C17" class="link link-ref link-reveal xref-bibr">17</a></sup> also causes translocation blockade<sup><a reveal-id="DSU032C15" data-open="DSU032C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSU032C18" data-open="DSU032C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and inhibition of ribosome recycling.<sup><a reveal-id="DSU032C16" data-open="DSU032C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU032C19" data-open="DSU032C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Of these mechanisms, translational miscoding might play only a minor role in cell killing, going by the evidence that strains with ribosomal mutations known to reduce translation fidelity are viable.<sup><a reveal-id="DSU032C20" data-open="DSU032C20" class="link link-ref link-reveal xref-bibr">20</a>–<a reveal-id="DSU032C22" data-open="DSU032C22" class="link link-ref link-reveal xref-bibr">22</a></sup> This probably lays greater emphasis on the translation blockade that these antibiotics cause<sup><a reveal-id="DSU032C23" data-open="DSU032C23" class="link link-ref link-reveal xref-bibr">23</a></sup> through their effects on ribosome translocation and recycling. Though various other non-intuitive mechanisms for cell death such as oxidative stress have been proposed, these remain controversial.<sup><a reveal-id="DSU032C24" data-open="DSU032C24" class="link link-ref link-reveal xref-bibr">24</a>–<a reveal-id="DSU032C26" data-open="DSU032C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p><p>A common method by which bacteria acquire resistance to aminoglycosides is via enzymes that modify the antibiotic, such as aminoglycoside acetyltransferases and phosphotransferases.<sup><a reveal-id="DSU032C27" data-open="DSU032C27" class="link link-ref link-reveal xref-bibr">27</a></sup> These genes are generally used as selectable markers in routine laboratory work. In addition, mutations targeting the aminoglycoside binding site in the 16S rRNA can confer resistance.<sup><a reveal-id="DSU032C28" data-open="DSU032C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSU032C29" data-open="DSU032C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Spontaneous emergence of such mutants in certain bacteria confers two orders of magnitude-fold resistance to the antibiotic.<sup><a reveal-id="DSU032C30" data-open="DSU032C30" class="link link-ref link-reveal xref-bibr">30</a></sup> However, such mutations are not accessible to <em>E. coli</em> due to the presence of multiple copies of the 16S rRNA gene,<sup><a reveal-id="DSU032C12" data-open="DSU032C12" class="link link-ref link-reveal xref-bibr">12</a></sup> a property common to many fast-growing bacteria. Even in the event that a single copy of the gene gets mutated, sensitivity to aminoglycosides is dominant in a heterogeneous ribosomal population.<sup><a reveal-id="DSU032C31" data-open="DSU032C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Previous studies on mutations conferring resistance to the aminoglycoside kanamycin in <em>E. coli</em> have led to the isolation of two elongation factor-G (EF-G) mutations. However, besides showing a temperature-sensitive phenotype, these bear a heavy cost to the cell (by affecting translation) in the absence of the antibiotic,<sup><a reveal-id="DSU032C32" data-open="DSU032C32" class="link link-ref link-reveal xref-bibr">32</a></sup> a property common to resistance mutations for many classes of antibiotics.<sup><a reveal-id="DSU032C11" data-open="DSU032C11" class="link link-ref link-reveal xref-bibr">11</a></sup> A more recent large-scale screen for aminoglycoside resistance suggested that reduction of proton motive force (PMF) on the membrane confers resistance, while making the cell more sensitive to other antibiotics that require PMF for efflux.<sup><a reveal-id="DSU032C33" data-open="DSU032C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p><p>Sublethal concentrations of antibiotics might play important physiological roles in bacteria, by acting as signalling molecules affecting gene expression.<sup><a reveal-id="DSU032C7" data-open="DSU032C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Besides, these can also promote development of resistance, either by induction of specific mutator pathways<sup><a reveal-id="DSU032C34" data-open="DSU032C34" class="link link-ref link-reveal xref-bibr">34</a></sup> or—in the natural environment—by promotion of horizontal gene transfer.<sup><a reveal-id="DSU032C35" data-open="DSU032C35" class="link link-ref link-reveal xref-bibr">35</a></sup> More fundamentally, under low antibiotic concentrations, growth of a subpopulation of cells that do not see the antibiotic, or the uniformly slower growth of the entire cell population, provides a substrate for selection to act. For example, antibiotics at concentrations as low as 1/100th the killing concentration leads to the selection of resistant bacteria.<sup><a reveal-id="DSU032C36" data-open="DSU032C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Sublethal concentrations of antibiotics are important in a clinical context, because factors such as incorrect antibiotic dosage or non-compliance of the patient with a prescribed dose regime could lead to low levels of the antibiotic in the body. Therefore, it is important to understand the effect of antibiotic dosage in the development of resistance, and the effects of the ensuing resistance mechanisms on fitness in the absence of the antibiotic.</p><p>The development of next-generation deep-sequencing technologies has made it relatively easy to track the emergence of variants in a population, including in the context of antibiotic resistance.<sup><a reveal-id="DSU032C37" data-open="DSU032C37" class="link link-ref link-reveal xref-bibr">37</a>–<a reveal-id="DSU032C39" data-open="DSU032C39" class="link link-ref link-reveal xref-bibr">39</a></sup> In this study, we use laboratory evolution of <em>E. coli</em> in batch cultures, followed by deep sequencing, to interrogate the emergence of resistance to an aminoglycoside antibiotic at two different sublethal concentrations. We show that resistance mechanisms with little consequence to cell fitness in the absence of the antibiotic emerge rapidly, and that the early dynamics of development of resistance might be dependent on the concentration of the antibiotic.</p>                            <h2 scrollto-destination=87199758 id="87199758" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87199759 id="87199759" class="section-title" data-legacy-id=s2a>2.1. Strains, culture media, growth curves and optical density</h3>
<p>Non-pathogenic <em>E. coli</em> MG1655 was used for the evolution experiments. Growth curves were generated in flasks or 96-well plates in Luria Bertani (LB; Hi-Media, India) broth using 1:100 dilution of overnight culture and incubation at 37°C with shaking at 200 rpm. Optical density (OD) measurements were carried out at 600 nm (OD<sub>600</sub>) using either a UV-visible spectrophotometer (SP-8001, Metertech) when growth curves were generated in flasks or using a plate reader (Infinite F200pro, Tecan) when generated in 96-well plates.</p>                            <h3 scrollto-destination=87199761 id="87199761" class="section-title" data-legacy-id=s2b>2.2. Minimum inhibitory concentration determination</h3>
<p>The minimum inhibitory concentration (MIC) was determined by a modification of the broth dilution technique. Antibiotics were obtained from Sigma-Aldrich (Kanamycin sulfate K1377, Streptomycin sulfate salt S6501, Gentamicin sulfate salt G1264, Neomycin trisulfate salt hydrate N1876, Paromomycin sulfate salt P5057, Hygromycin B H7772, Apramycin sulfate salt A2024). Dilutions of the antibiotic stock were made in sterile distilled water to which equal volume of 2× concentrated LB broth was added. Amount of stock taken was calculated with respect to final diluted volume. Overnight grown culture was added to the broth to achieve 1:100 dilution. After 24-h incubation at 37°C with shaking at 200 rpm, the MIC was inferred as the lowest concentration at which OD<sub>600</sub> falls below 0.05 with respect to the blank. Eight independently grown replicates were so tested for their MIC.</p>                            <h3 scrollto-destination=87199763 id="87199763" class="section-title" data-legacy-id=s2c>2.3. Evolution in kanamycin</h3>
<p>For evolution experiments in sublethal kanamycin concentrations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), the sublethal window was defined on the basis of an MIC of kanamycin for <em>wild type</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). We selected two concentrations for the evolution experiment: 4 µg/ml (4-kan, ∼25% MIC) and 8 µg/ml (8-kan, ∼50% MIC); growth parameters like length of lag period, growth rates and stationary-phase OD<sub>600</sub> were significantly different at these two concentrations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Evolution experiments were carried out in a 100-ml LB broth in 250 ml conical flasks with incubation at 37°C and shaking at 200 r.p.m. Two replicate populations in the case of 0-kan and 4-kan, and four replicate populations in the case of 8-kan, were passaged by batch culture into fresh 100-ml LB broth (while maintaining kanamycin concentration) using 1:100 dilution every 24 h, under identical shaking and incubation conditions. For the 4-kan populations, the MICs of the two replicate populations were followed with each passage, and populations were collected by spinning down the culture for genomic DNA (gDNA) extraction and deep sequencing. Glycerol stocks were also maintained for each passage. For the 8-kan populations, the MICs of colonies derived from these populations on plain LB agar were followed with each passage. A few colonies were selected and grown in LB broth for collecting cells for gDNA extraction and deep sequencing; glycerol stocks for these were also prepared.</p>                            <h3 scrollto-destination=87199765 id="87199765" class="section-title" data-legacy-id=s2d>2.4. Population structure analysis of evolving bacterial cultures</h3>
<p>As described in the previous section, we grew <em>E. coli</em> batch cultures in 0-kan (plain LB), 4-kan and 8-kan up to one passage. Cultures at the 0 h (passage 0), 24 h (passage 0) and 24 h (passage 1) were subjected to serial 10-fold dilution and plated on plain LB agar plates. The resulting colonies were randomly picked and tested for resistance by growth in two different concentrations of kanamycin: 4-kan (moderately-lethal) and 20-kan (lethal), and tested for defects in growth in plain LB (0-kan). Only 24 h OD<sub>600</sub> was measured. This experiment was performed on around 88 isolates obtained from each of the three evolving populations (0-kan, 4-kan and 20-kan) at the three indicated time points. We did not maintain replicate populations; instead, we repeated the experiment multiple times. Results of two different repeats of the experiment that had considerable variation are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p>                            <h3 scrollto-destination=87199767 id="87199767" class="section-title" data-legacy-id=s2e>2.5. Genomic DNA extraction</h3>
<p>Extraction of gDNA from either experimental populations or individual isolates from such populations was done using the Sigma-Aldrich GenElute Bacterial Genomic DNA Kit (NA2120) using the manufacturer's protocol. gDNA integrity was checked on agarose gel, and quality and concentration were checked using NanoDrop UV-Vis Spectrophotometer (Thermo Scientific).</p>                            <h3 scrollto-destination=87199769 id="87199769" class="section-title" data-legacy-id=s2f>2.6. Sequencing and determination of mutations</h3>
<p>Library preparation was carried out using the Truseq DNA library sample prep kit V2, and sequencing was carried out using TruSeq SBS Kit v3-HS (50 Cycles) at Centre for Cellular and Molecular Platforms (C-CAMP) on Illumina HiSeq 1000 platform according to manufacturer's instructions. Samples were sequenced to ∼300× coverage. FASTX quality assessed reads were mapped to the reference genome (NC_000913.2) using BWA.<sup><a reveal-id="DSU032C40" data-open="DSU032C40" class="link link-ref link-reveal xref-bibr">40</a></sup> SAMTOOLS<sup><a reveal-id="DSU032C41" data-open="DSU032C41" class="link link-ref link-reveal xref-bibr">41</a></sup> was used to process the BWA output aligned sam files to generate a pileup file, and VARSCAN<sup><a reveal-id="DSU032C42" data-open="DSU032C42" class="link link-ref link-reveal xref-bibr">42</a></sup> was used to extract the list of single nucleotide polymorphisms (SNPs) and indels from the pileup. Percentage of reads supporting a SNP was obtained by dividing the number of reads containing the SNP by the total number of reads mapping to that locus, multiplied by 100. In the case of 4-kan populations, mutations were called when percentage of reads supporting the mutant allele was above 20% in at least one of the sequenced time points during the course of the evolution experiment. This cut-off was set to 50% for isolate sequencing in the case of 8-kan populations; we note here that though a low cut-off was set, mutations identified in the 8-kan clones were generally supported by ∼97% of reads. Once the list of mutations was generated using these cut-offs, their abundances across all time points were recalculated from the pileup files using a perl script. Only good-quality bases (quality &gt; phred score 30) were considered for this analysis, and bases at the start and end of reads that mapped to the mutation position were excluded. Additionally, BRESEQ<sup><a reveal-id="DSU032C43" data-open="DSU032C43" class="link link-ref link-reveal xref-bibr">43</a></sup> was used to verify the mutations. Longer deletions in the genes encoding subunits of the ATP synthase complex were inferred by BRESEQ. Sanger sequencing was done to confirm mutations.</p>                            <h3 scrollto-destination=87199771 id="87199771" class="section-title" data-legacy-id=s2g>2.7. Phage transduction</h3>
<p>To move the FusA<sup>P610T</sup> mutation into the <em>wild type</em> background for characterization, phage transduction using the P1 phage was used. The kanamycin resistance cassette, amplified from pKD13, was inserted in a featureless region near the <em>fusA</em> gene (without deleting a single nucleotide) using the one-step inactivation procedure of Datsenko and Wanner.<sup><a reveal-id="DSU032C44" data-open="DSU032C44" class="link link-ref link-reveal xref-bibr">44</a></sup> (A higher concentration of kanamycin, i.e. 80 µg/ml, was used to limit growth of the resistant parent strain.) This strain was used as donor to prepare P1 phage lysate that was subsequently used to transduce <em>wild type</em> as per the Court lab protocol.<sup><a reveal-id="DSU032C45" data-open="DSU032C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Selection for transductants was done on LB agar plates containing kanamycin. FLP recombinase provided by pCP20 was used to remove the kanamycin resistance cassette which is flanked by FRT sites (leaving behind a FRT site in the process).</p>                            <h3 scrollto-destination=87199773 id="87199773" class="section-title" data-legacy-id=s2h>2.8. Growth simulation</h3>
<p>Discrete growth simulations were used to assess the effect of different growth rate regimes on the persistence of sick variants. Each simulation was seeded with <em>N</em> objects—each object representing a bacterial cell—of three types as follows. Most of these objects were <em>wild type</em>. A small number of objects represented mutants of two types, one with a higher growth rate than the other (3-fold difference in growth rates), with both mutants displaying a higher rate of growth than the <em>wild type</em> (with the <em>wild type</em> growing three times slower than the slow-growing mutant). This set-up reflects the situation wherein two resistant mutants—one with a growth defect and the other not—are competing with each other and the <em>wild type</em> for growth in the presence of sublethal concentrations of an antibiotic. The simulation was seeded with five times as many sick mutants as fit ones. Each growth iteration would allow an object to replicate or not—the probability that an object replicates would depend on the growth rate. The simulations were stopped when the total number of objects crossed a pre-defined threshold. This reflects only exponential growth. Introduction of a subsampling step after a certain number of iterations, to simulate serial passages, does not alter these results.</p>                            <h3 scrollto-destination=87199775 id="87199775" class="section-title" data-legacy-id=s2i>2.9. Competition assays</h3>
<p>To distinguish between two competing MG1655 strains, a knock-out of <em>lacZ</em> gene was generated in <em>wild type</em> MG1655 background using the one-step inactivation procedure of Datsenko and Wanner.<sup><a reveal-id="DSU032C44" data-open="DSU032C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The colonies of a Δ<em>lacZ</em> strain can be distinguished from those of <em>lacZ<sup>+</sup></em> colonies on McConkey's agar (HiMedia, India). The <em>lacZ</em> deletion was made in both <em>wild type</em> and <em>wild-type</em>-FRT backgrounds. For competition, <em>wild type</em>/<em>wild-type</em>-FRT (Δ<em>lacZ</em>) cultures were co-inoculated in LB along with FusA<sup>P610T</sup>-FRT (<em>lacZ<sup>+</sup></em>) in 1:1 proportion, and the counts of both colony types were followed on LB agar. As control, mock competitions were carried out with the two (otherwise competing) cultures grown separately. They were mixed together in 1:1 proportion immediately prior to performing viable count on the differentiating medium.</p>                            <h3 scrollto-destination=87199777 id="87199777" class="section-title" data-legacy-id=s2j>2.10. GFP induction</h3>
<p>Hou <em>et al.</em><sup><a reveal-id="DSU032C32" data-open="DSU032C32" class="link link-ref link-reveal xref-bibr">32</a></sup> assayed translation defects in their kanamycin-resistant <em>fusA</em> mutants by looking at the kinetics of induction of β-galactosidase in these strains versus the <em>wild type</em>. Similarly, we looked at the kinetics of induction of green fluorescent protein (GFP) to check for translation defects in FusA<sup>P610T</sup>. GFP<sup>mut2 <a reveal-id="DSU032C46" data-open="DSU032C46" class="link link-ref link-reveal xref-bibr">46</a></sup> was cloned from pMS201 into pBAD18<sup><a reveal-id="DSU032C47" data-open="DSU032C47" class="link link-ref link-reveal xref-bibr">47</a></sup> within <em>Eco</em>RI and <em>Hin</em>dIII restriction sites to get pBAD18::<em>gfpmut2</em>. Overnight cultures of both WT-FRT and FusA<sup>P610T</sup>-FRT were grown in LB broth in a 96-well black plate (Sigma-Aldrich, costar 3603) until OD<sub>600</sub> reached ∼0.5. Subsequently to induce GFP<sup>mut2</sup>, 1 or 10 mM arabinose (HiMedia, India) was added to the wells that were to be tested for induction. Also, kanamycin was added to the wells that were to be tested for induction in the presence of the antibiotic, to a final concentration of 10 μg/ml. GFP<sup>mut2</sup> fluorescence was measured at 520 nm by excitation at 485 nm using a plate reader (Infinite F200pro Tecan). Promoter activities<sup><a reveal-id="DSU032C48" data-open="DSU032C48" class="link link-ref link-reveal xref-bibr">48</a></sup> were calculated as follows: OD and GFP fluorescence intensity measurements were blank subtracted separately. GFP intensity was then normalized to OD. Promoter activity was calculated as d<em>F</em>/d<em>t</em>, where d<em>F</em> is change in OD-normalized GFP fluorescence in time interval d<em>t</em>. Finally, promoter activities thus calculated were normalized to fall in the range of 0–1.</p>                            <h2 scrollto-destination=87199779 id="87199779" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87199780 id="87199780" class="section-title" data-legacy-id=s3a>3.1. Growth of E. coli in sublethal concentrations
of kanamycin</h3>
<p>To explore the landscape of mutations leading to resistance of <em>E. coli</em> to aminoglycosides, we grew the bacteria—in a batch culture—in two sublethal concentrations of kanamycin (4 µg/ml, ∼20–25% of the MIC; 8 µg/ml, ∼40–50% of the MIC; see ‘Materials and Methods’; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), a 2-DOS aminoglycoside antibiotic commonly used in the laboratory. Here we refer to the first batch culture of bacteria in antibiotic-containing medium as P0. A sample of bacteria obtained after ∼24 h of growth in P0 was serially passaged under identical conditions, and these serial passages will be referred to as P1, P2, etc.</p><div class="block-child-p">The P0 growth curve of <em>E. coli</em> grown in 4 µg/ml kanamycin (4-kan) is characterized by a long lag phase (6–7 h) followed by an exponential phase defined by low growth rate (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>A and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> for statistics). The lag phase period increased considerably to at least 11 h in 8 µg/ml kanamycin (8-kan) (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>C and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>); the cultures typically reached a final OD after 24 h that was less than what was observed for the 4-kan cultures (compare Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>A and C, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). <div data-id="DSU032F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03201.jpeg?Expires=2147483647&amp;Signature=RCCc2k4M8pzVtwbQtrKTyTNN7QtOhjO--j7yYN-BXiDJ7HoGpAKNUcMedCzXAUo33gaVieLMQ-i5tl3yG9ygdPqqBvApOSeDNnIGDzelR4XjlE04C7so50CaTfwpWYafVjrEr6uTp~gmyicNS7X45yEtVyvnfWKXpe87PAq2tMdJHb7U9TF69R0Pq5s3i5i1PCKj050IPswA6qcLyB8eylr1bG7Nn8QbbnXXb4hqqoVFiwDybXMRtHhvZM1irndWZw7SMVKmeBKfI5L0KMUUXEEDyJHck6zZnrOO4BvadGkCo8xgg6KZG91SYD~pawhI3go5gSv7SdG3sF3xpxl8Tg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Resistance developed as a result of growth in sublethal kanamycin concentration. (A) Growth of two replicate populations of Escherichia coli MG1655 in ∼25% of the lethal level of kanamycin (4 µg/ml: 4-kan 1 and 2; for additional growth curves, see Supplementary Data). Control populations in plain LB are 0-kan 1 and 2. Populations at various time points from the 4-kan curves were sequenced while only mid-exponential- and stationary-phase populations from the 0-kan curves were sequenced. (B) MICs of evolving populations. P0 indicates the first round of growth in kanamycin (i.e. the growth curves in A) while P1–P10 indicate serial transfers by 100-fold dilution. Black bars indicate MIC levels of control populations passaged in plain LB (0-kan). Red bars indicate populations evolving in 4-kan, with data available for multiple time points in P0. Green bars indicate populations grown in 4-kan in P0 but subsequently transferred into plain LB for the remaining passages P1–P10 (4-kan → 0-kan). The maintenance of increased MIC levels suggests the presence of resistant mutations and not a temporary stress response. The average MIC level of the two biological replicates (from three technical replicates) is plotted with the total length of the positive and negative error bars representing the difference in MICs between the two replicates (range). (C) Growth of four replicate populations of E. coli MG1655 in ∼50% lethal levels of kanamycin (8 µg/ml: 8-kan 1-4; more growth curves can be found in Supplementary Data). (D) Isolates derived from (four replicates of) the P0 8-kan and the P1 8-kan populations were tested for their MIC (red bars) and growth in plain LB (black bars, taken at 7 h. after inoculation). Many isolates in P0 were sick, i.e. had a lower OD600 in plain LB but could resist kanamycin. The incidence of these sick isolates decreases after P1, and fit resistant forms seem to dominate. For comparison, kanamycin MICs (red bars) and OD600 (black bars) of multiple isolates, from P0 and P1 control wild-type populations grown in plain LB, are shown in the right panels." path-from-xml="dsu03201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199782/dsu03201.jpeg" path-from-xml="dsu03201.jpeg" target="_blank">View large</a><a section="87199782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03201.jpeg?Expires=2147483647&amp;Signature=gQaMVpbkcKJvrfQCANVs~XzS1E3vpqtyPA3HhP4sKu0PHukOOrdZW79Cv8nYTBX~PW9jANubQf9--S5-5GAPwmusqWSIjt2abuaAKiiTEXYw7jTFXOWpyq5i36pylu1TqrRXsaFx-Z6EDIegqeP6~r0Pdhh9Zi2SSI~kJLnOjSqtKOuH9oMtqaf4NWuOSjgRJlqiG1O9-LNxDXgwDEutMWHnGCiJyZad~7dqDLnCNfaJr2Lcd3kFTlmgjxMKbVQd4cYVWBqIe7578CbPjMFRKPK~oUBjuM3iiVQJDmeU0AC0XoZvtz7Ihxt0Wp0KRbKAlsrOfUvX81hv1UnFeMR-rQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199782&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Resistance developed as a result of growth in sublethal kanamycin concentration. (A) Growth of two replicate populations of <em>Escherichia coli</em> MG1655 in ∼25% of the lethal level of kanamycin (4 µg/ml: 4-kan 1 and 2; for additional growth curves, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Control populations in plain LB are 0-kan 1 and 2. Populations at various time points from the 4-kan curves were sequenced while only mid-exponential- and stationary-phase populations from the 0-kan curves were sequenced. (B) MICs of evolving populations. P0 indicates the first round of growth in kanamycin (i.e. the growth curves in A) while P1–P10 indicate serial transfers by 100-fold dilution. Black bars indicate MIC levels of control populations passaged in plain LB (0-kan). Red bars indicate populations evolving in 4-kan, with data available for multiple time points in P0. Green bars indicate populations grown in 4-kan in P0 but subsequently transferred into plain LB for the remaining passages P1–P10 (4-kan → 0-kan). The maintenance of increased MIC levels suggests the presence of resistant mutations and not a temporary stress response. The average MIC level of the two biological replicates (from three technical replicates) is plotted with the total length of the positive and negative error bars representing the difference in MICs between the two replicates (range). (C) Growth of four replicate populations of <em>E. coli</em> MG1655 in ∼50% lethal levels of kanamycin (8 µg/ml: 8-kan 1-4; more growth curves can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). (D) Isolates derived from (four replicates of) the P0 8-kan and the P1 8-kan populations were tested for their MIC (red bars) and growth in plain LB (black bars, taken at 7 h. after inoculation). Many isolates in P0 were sick, i.e. had a lower OD<sub>600</sub> in plain LB but could resist kanamycin. The incidence of these sick isolates decreases after P1, and fit resistant forms seem to dominate. For comparison, kanamycin MICs (red bars) and OD<sub>600</sub> (black bars) of multiple isolates, from P0 and P1 control <em>wild-type</em> populations grown in plain LB, are shown in the right panels.</p></div></div><div content-id="DSU032F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03201.jpeg?Expires=2147483647&amp;Signature=RCCc2k4M8pzVtwbQtrKTyTNN7QtOhjO--j7yYN-BXiDJ7HoGpAKNUcMedCzXAUo33gaVieLMQ-i5tl3yG9ygdPqqBvApOSeDNnIGDzelR4XjlE04C7so50CaTfwpWYafVjrEr6uTp~gmyicNS7X45yEtVyvnfWKXpe87PAq2tMdJHb7U9TF69R0Pq5s3i5i1PCKj050IPswA6qcLyB8eylr1bG7Nn8QbbnXXb4hqqoVFiwDybXMRtHhvZM1irndWZw7SMVKmeBKfI5L0KMUUXEEDyJHck6zZnrOO4BvadGkCo8xgg6KZG91SYD~pawhI3go5gSv7SdG3sF3xpxl8Tg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Resistance developed as a result of growth in sublethal kanamycin concentration. (A) Growth of two replicate populations of Escherichia coli MG1655 in ∼25% of the lethal level of kanamycin (4 µg/ml: 4-kan 1 and 2; for additional growth curves, see Supplementary Data). Control populations in plain LB are 0-kan 1 and 2. Populations at various time points from the 4-kan curves were sequenced while only mid-exponential- and stationary-phase populations from the 0-kan curves were sequenced. (B) MICs of evolving populations. P0 indicates the first round of growth in kanamycin (i.e. the growth curves in A) while P1–P10 indicate serial transfers by 100-fold dilution. Black bars indicate MIC levels of control populations passaged in plain LB (0-kan). Red bars indicate populations evolving in 4-kan, with data available for multiple time points in P0. Green bars indicate populations grown in 4-kan in P0 but subsequently transferred into plain LB for the remaining passages P1–P10 (4-kan → 0-kan). The maintenance of increased MIC levels suggests the presence of resistant mutations and not a temporary stress response. The average MIC level of the two biological replicates (from three technical replicates) is plotted with the total length of the positive and negative error bars representing the difference in MICs between the two replicates (range). (C) Growth of four replicate populations of E. coli MG1655 in ∼50% lethal levels of kanamycin (8 µg/ml: 8-kan 1-4; more growth curves can be found in Supplementary Data). (D) Isolates derived from (four replicates of) the P0 8-kan and the P1 8-kan populations were tested for their MIC (red bars) and growth in plain LB (black bars, taken at 7 h. after inoculation). Many isolates in P0 were sick, i.e. had a lower OD600 in plain LB but could resist kanamycin. The incidence of these sick isolates decreases after P1, and fit resistant forms seem to dominate. For comparison, kanamycin MICs (red bars) and OD600 (black bars) of multiple isolates, from P0 and P1 control wild-type populations grown in plain LB, are shown in the right panels." path-from-xml="dsu03201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199782/dsu03201.jpeg" path-from-xml="dsu03201.jpeg" target="_blank">View large</a><a section="87199782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03201.jpeg?Expires=2147483647&amp;Signature=gQaMVpbkcKJvrfQCANVs~XzS1E3vpqtyPA3HhP4sKu0PHukOOrdZW79Cv8nYTBX~PW9jANubQf9--S5-5GAPwmusqWSIjt2abuaAKiiTEXYw7jTFXOWpyq5i36pylu1TqrRXsaFx-Z6EDIegqeP6~r0Pdhh9Zi2SSI~kJLnOjSqtKOuH9oMtqaf4NWuOSjgRJlqiG1O9-LNxDXgwDEutMWHnGCiJyZad~7dqDLnCNfaJr2Lcd3kFTlmgjxMKbVQd4cYVWBqIe7578CbPjMFRKPK~oUBjuM3iiVQJDmeU0AC0XoZvtz7Ihxt0Wp0KRbKAlsrOfUvX81hv1UnFeMR-rQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199782&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Resistance developed as a result of growth in sublethal kanamycin concentration. (A) Growth of two replicate populations of <em>Escherichia coli</em> MG1655 in ∼25% of the lethal level of kanamycin (4 µg/ml: 4-kan 1 and 2; for additional growth curves, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Control populations in plain LB are 0-kan 1 and 2. Populations at various time points from the 4-kan curves were sequenced while only mid-exponential- and stationary-phase populations from the 0-kan curves were sequenced. (B) MICs of evolving populations. P0 indicates the first round of growth in kanamycin (i.e. the growth curves in A) while P1–P10 indicate serial transfers by 100-fold dilution. Black bars indicate MIC levels of control populations passaged in plain LB (0-kan). Red bars indicate populations evolving in 4-kan, with data available for multiple time points in P0. Green bars indicate populations grown in 4-kan in P0 but subsequently transferred into plain LB for the remaining passages P1–P10 (4-kan → 0-kan). The maintenance of increased MIC levels suggests the presence of resistant mutations and not a temporary stress response. The average MIC level of the two biological replicates (from three technical replicates) is plotted with the total length of the positive and negative error bars representing the difference in MICs between the two replicates (range). (C) Growth of four replicate populations of <em>E. coli</em> MG1655 in ∼50% lethal levels of kanamycin (8 µg/ml: 8-kan 1-4; more growth curves can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). (D) Isolates derived from (four replicates of) the P0 8-kan and the P1 8-kan populations were tested for their MIC (red bars) and growth in plain LB (black bars, taken at 7 h. after inoculation). Many isolates in P0 were sick, i.e. had a lower OD<sub>600</sub> in plain LB but could resist kanamycin. The incidence of these sick isolates decreases after P1, and fit resistant forms seem to dominate. For comparison, kanamycin MICs (red bars) and OD<sub>600</sub> (black bars) of multiple isolates, from P0 and P1 control <em>wild-type</em> populations grown in plain LB, are shown in the right panels.</p></div></div></div><p>We studied the kanamycin resistance of the bacterial population grown after P0 in 4-kan or 8-kan. For the 4-kan populations, MIC assays were performed on a sample drawn at 24 h (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>B). For the 8-kan cells, these assays were performed on individual isolates derived from the 24-h-old P0 culture (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>D). This difference in approach was adopted, because replicate 8-kan cultures grown in flasks for these experiments appeared to show more variable growth characteristics than the 4-kan cultures (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>A and C); note however that this cannot be statistically supported by data from a larger number of cultures grown in plates (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The 4-kan populations displayed an increase in MIC of ∼2-fold compared with the initial inoculum (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>B, red bars). Among the clones derived from the 8-kan populations, a few were only slightly different from the <em>wild type</em>, whereas others displayed an increase of ∼2- to 5-fold (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>D, P0 8-kan). As a control, bacteria grown in plain LB showed no increase in kanamycin resistance (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>B—black bars; for <em>wild type</em> control colonies refer to the right side panels in Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>D).</p><p>After the first 24 h culture, the bacteria were diluted into fresh medium containing the same concentration of the antibiotic (i.e. P1). The MIC of the 4-kan population was determined again after entry into stationary phase and found to be ∼4-fold higher than a control population that had not been exposed to the antibiotic (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>B, red bars). Further serial passages in 4-kan did not lead to substantial changes in the MIC of the population, suggesting that a stable variant had emerged. In the 8-kan populations, after dilution into fresh medium, most resultant isolates displayed &gt;4- to 5-fold increase in MIC over the <em>wild type</em> (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>D, 8-kan P1 colonies). We do not attempt a direct, time-anchored comparison of MICs between the 4-kan and 8-kan cultures here, since the experiment for the 8-kan populations included a plating step to isolate colonies, which was not performed for the 4-kan cultures (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> for time-anchored population MICs for a separate set of experiments).</p><div class="block-child-p">What are the characteristics of the bacterial population resulting from the above-described growth in the antibiotic, in terms of its ability to grow in the presence or absence of kanamycin? For this, we first grew cells—produced by P0 cultures in antibiotic-free, 4-kan or 8-kan media—on LB agar plates. Individual colonies so obtained were then checked for their ability to grow in the absence of kanamycin (plain LB, 0-kan), moderately sublethal levels of kanamycin (4-kan) and lethal levels of kanamycin (20-kan). We note here that this experimental strategy is limited in the sense that the growth experiments performed to assess fitness also contribute to further evolution and selection. However, these steps were performed in the same manner for 4-kan and 8-kan populations, and therefore, the results thus obtained enable a direct comparison between the two antibiotic concentrations. Control colonies derived from all the populations immediately after inoculation displayed <em>wild-type</em> characteristics: good growth in 0-kan, poor growth at 4-kan and no growth at 20-kan (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>, 0 h.). Similarly, colonies derived from populations not treated with kanamycin retained this response throughout the course of the experiment (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Most colonies derived from 4-kan populations grew well in LB and in 4-kan; in addition, they were more resistant to 20-kan than the <em>wild type</em> (Fig. <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>A, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). In contrast, a proportion of colonies derived from 8-kan populations, though more resistant to the antibiotic than the <em>wild type</em>, showed severe growth defects (Fig. <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>B, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The proportion of colonies with a growth defect varied between trials (Fig. <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>B, compare group-1 and -2 colonies in 8-kan 1 and 2 at 24 h, between the two replicates). After another passage in the same concentration of the antibiotic (P1), both 4-kan and 8-kan cultures produced antibiotic-resistant colonies with little growth defect (Fig. <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>A and 2B, group-3 colonies increase at P1). <div data-id="DSU032F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03202.jpeg?Expires=2147483647&amp;Signature=OgryhSgFuaBfi6vosiOMimIKkB9frCnOzTmnev9PEg2C5GFy-dkC-8H2SwfvzHK1JdYwDV78bc8FP~fO1ooFGxmjtjNsK3DZg2LWvSAC-EFh-YZ7HhGARMPfFafgtrdP4me9il59jjSJ3cJJx42kb3ZEL4pI4fehz-oohxG3OBL1JkQ03BbDQnVJYTjl~0XaqZ81QP62fGQZHrWMRoCjkFzHfq9XXNeEx9wSq~XUik7h58rspt-jgQZ2otu8lbvMCxGfifC~Q4jCnlHoxWDVfXuVLJm-sl1EieT7XrYk0edevT7zZwc~4HNeRUBTV1bvbhGlLi2KxGkv0OU~0EhLsw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population structure of evolving populations. (A) E. coli populations grown in 4-kan are more homogeneous in composition and comprised mostly of fit variants at 24 h. Isolates were classified into several groups based on the stationary-phase OD600 reached in 4-kan (tolerance to kanamycin) and 0-kan (growth in the absence of kanamycin; see inset in B). Plotted are counts of different types of isolates. Wild-type-like isolates (black bars) have been arbitrarily defined for the analysis as non-tolerant but fit isolates (falling in the OD range indicated in the inset of B) and predominate at the start of the experiment (0 h.). Group 1 (dark grey) isolates are tolerant but sick variants. Group 2 (grey) and 3 (light grey) are tolerant and fit variants (see inset of Figure 1B for definition of groups based on stationary-phase OD600 in plain LB and 4-kan and Supplementary Data for all plots related to this figure as well as plots for 0-kan control populations in which the wild-type-like colonies predominate throughout the course of the experiment). The counts change with time during the course of the batch evolution experiment (0 h, 24 h and P1). Results of the first replicate are displayed in the left panel and the second replicate in the right panel. (B) Populations grown in 8-kan are composed of multiple subpopulations, with the sick variants dominating at 24 h. Thus, tolerant but sick variants predominate in the first batch growth in 8-kan, and subsequent passaging results in an increase in number of the fit variants. (C) 1,000 independent growth simulations were performed for three growth regimes. This plot shows the distributions of the total population size (a reflection of the number of iterations or time) at which the ratio of the sick mutant to the fit mutant falls below 10%. Distributions are represented by box plots. The box represents the inter-quartile range and the line within the box indicates the median value. The maximum and the minimum value (1.5 times the inter-quartile range), excluding the outliers, is represented by the whiskers. The points represent the outliers. Probabilities of division per iteration are as follows. For the slow growth regime (green), µwt = 0.01, µsick = 0.03 and µfit = 0.1; for the intermediate growth regime (blue), µwt = 0.03, µsick = 0.1 and µfit = 0.3; for the fast growth regime (red), µwt = 0.1, µsick = 0.3 and µfit = 1. (D) A scatter plot of the ratio of the number of sick to fit mutants as a function of the population size (equivalent to the number of iterations or time) for an example simulation from (C)." path-from-xml="dsu03202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199785/dsu03202.jpeg" path-from-xml="dsu03202.jpeg" target="_blank">View large</a><a section="87199785" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03202.jpeg?Expires=2147483647&amp;Signature=bo2jLDhww2l6bHpdiyLFv0atzjA~Xf01Fx0AAl3dsCIu8hN6DP5Z64SyyafUhs49rfq0PBKQ1loZ9FFhdlo~Y5Yi6RC~9nMtYSHsndi4E3eg6yA0VDtMgR9Le2o6MJxFT-e1hLFyw3vY9rvZgiTlvSeJLBtkvSjVnSvCf7TcckV83owtabfOJnRhhQwxoYGpD9WZ5YaSSGBvK5XErOi3OAKnafTFfPiCOqgjtp~Gg97YE2sw1R0TiHFohZ5iWYdF0FSAlmgk3t6JisPKmt7fdiOfuXV4sudWkg1vajkNmPwtMgGrqbeDq18LKW~WNPOuiGltQ-290S4W05bRneCKuw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199785&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population structure of evolving populations. (A) <em>E. coli</em> populations grown in 4-kan are more homogeneous in composition and comprised mostly of fit variants at 24 h. Isolates were classified into several groups based on the stationary-phase OD<sub>600</sub> reached in 4-kan (tolerance to kanamycin) and 0-kan (growth in the absence of kanamycin; see inset in B). Plotted are counts of different types of isolates. <em>Wild-type-</em>like isolates (black bars) have been arbitrarily defined for the analysis as non-tolerant but fit isolates (falling in the OD range indicated in the inset of B) and predominate at the start of the experiment (0 h.). Group 1 (dark grey) isolates are tolerant but sick variants. Group 2 (grey) and 3 (light grey) are tolerant and fit variants (see inset of Figure <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>B for definition of groups based on stationary-phase OD<sub>600</sub> in plain LB and 4-kan and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> for all plots related to this figure as well as plots for 0-kan control populations in which the <em>wild-type</em>-like colonies predominate throughout the course of the experiment). The counts change with time during the course of the batch evolution experiment (0 h, 24 h and P1). Results of the first replicate are displayed in the left panel and the second replicate in the right panel. (B) Populations grown in 8-kan are composed of multiple subpopulations, with the sick variants dominating at 24 h. Thus, tolerant but sick variants predominate in the first batch growth in 8-kan, and subsequent passaging results in an increase in number of the fit variants. (C) 1,000 independent growth simulations were performed for three growth regimes. This plot shows the distributions of the total population size (a reflection of the number of iterations or time) at which the ratio of the sick mutant to the fit mutant falls below 10%. Distributions are represented by box plots. The box represents the inter-quartile range and the line within the box indicates the median value. The maximum and the minimum value (1.5 times the inter-quartile range), excluding the outliers, is represented by the whiskers. The points represent the outliers. Probabilities of division per iteration are as follows. For the slow growth regime (green), µ<sub>wt</sub> = 0.01, µ<sub>sick</sub> = 0.03 and µ<sub>fit</sub> = 0.1; for the intermediate growth regime (blue), µ<sub>wt</sub> = 0.03, µ<sub>sick</sub> = 0.1 and µ<sub>fit</sub> = 0.3; for the fast growth regime (red), µ<sub>wt</sub> = 0.1, µ<sub>sick</sub> = 0.3 and µ<sub>fit</sub> = 1. (D) A scatter plot of the ratio of the number of sick to fit mutants as a function of the population size (equivalent to the number of iterations or time) for an example simulation from (C).</p></div></div><div content-id="DSU032F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03202.jpeg?Expires=2147483647&amp;Signature=OgryhSgFuaBfi6vosiOMimIKkB9frCnOzTmnev9PEg2C5GFy-dkC-8H2SwfvzHK1JdYwDV78bc8FP~fO1ooFGxmjtjNsK3DZg2LWvSAC-EFh-YZ7HhGARMPfFafgtrdP4me9il59jjSJ3cJJx42kb3ZEL4pI4fehz-oohxG3OBL1JkQ03BbDQnVJYTjl~0XaqZ81QP62fGQZHrWMRoCjkFzHfq9XXNeEx9wSq~XUik7h58rspt-jgQZ2otu8lbvMCxGfifC~Q4jCnlHoxWDVfXuVLJm-sl1EieT7XrYk0edevT7zZwc~4HNeRUBTV1bvbhGlLi2KxGkv0OU~0EhLsw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population structure of evolving populations. (A) E. coli populations grown in 4-kan are more homogeneous in composition and comprised mostly of fit variants at 24 h. Isolates were classified into several groups based on the stationary-phase OD600 reached in 4-kan (tolerance to kanamycin) and 0-kan (growth in the absence of kanamycin; see inset in B). Plotted are counts of different types of isolates. Wild-type-like isolates (black bars) have been arbitrarily defined for the analysis as non-tolerant but fit isolates (falling in the OD range indicated in the inset of B) and predominate at the start of the experiment (0 h.). Group 1 (dark grey) isolates are tolerant but sick variants. Group 2 (grey) and 3 (light grey) are tolerant and fit variants (see inset of Figure 1B for definition of groups based on stationary-phase OD600 in plain LB and 4-kan and Supplementary Data for all plots related to this figure as well as plots for 0-kan control populations in which the wild-type-like colonies predominate throughout the course of the experiment). The counts change with time during the course of the batch evolution experiment (0 h, 24 h and P1). Results of the first replicate are displayed in the left panel and the second replicate in the right panel. (B) Populations grown in 8-kan are composed of multiple subpopulations, with the sick variants dominating at 24 h. Thus, tolerant but sick variants predominate in the first batch growth in 8-kan, and subsequent passaging results in an increase in number of the fit variants. (C) 1,000 independent growth simulations were performed for three growth regimes. This plot shows the distributions of the total population size (a reflection of the number of iterations or time) at which the ratio of the sick mutant to the fit mutant falls below 10%. Distributions are represented by box plots. The box represents the inter-quartile range and the line within the box indicates the median value. The maximum and the minimum value (1.5 times the inter-quartile range), excluding the outliers, is represented by the whiskers. The points represent the outliers. Probabilities of division per iteration are as follows. For the slow growth regime (green), µwt = 0.01, µsick = 0.03 and µfit = 0.1; for the intermediate growth regime (blue), µwt = 0.03, µsick = 0.1 and µfit = 0.3; for the fast growth regime (red), µwt = 0.1, µsick = 0.3 and µfit = 1. (D) A scatter plot of the ratio of the number of sick to fit mutants as a function of the population size (equivalent to the number of iterations or time) for an example simulation from (C)." path-from-xml="dsu03202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199785/dsu03202.jpeg" path-from-xml="dsu03202.jpeg" target="_blank">View large</a><a section="87199785" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03202.jpeg?Expires=2147483647&amp;Signature=bo2jLDhww2l6bHpdiyLFv0atzjA~Xf01Fx0AAl3dsCIu8hN6DP5Z64SyyafUhs49rfq0PBKQ1loZ9FFhdlo~Y5Yi6RC~9nMtYSHsndi4E3eg6yA0VDtMgR9Le2o6MJxFT-e1hLFyw3vY9rvZgiTlvSeJLBtkvSjVnSvCf7TcckV83owtabfOJnRhhQwxoYGpD9WZ5YaSSGBvK5XErOi3OAKnafTFfPiCOqgjtp~Gg97YE2sw1R0TiHFohZ5iWYdF0FSAlmgk3t6JisPKmt7fdiOfuXV4sudWkg1vajkNmPwtMgGrqbeDq18LKW~WNPOuiGltQ-290S4W05bRneCKuw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199785&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population structure of evolving populations. (A) <em>E. coli</em> populations grown in 4-kan are more homogeneous in composition and comprised mostly of fit variants at 24 h. Isolates were classified into several groups based on the stationary-phase OD<sub>600</sub> reached in 4-kan (tolerance to kanamycin) and 0-kan (growth in the absence of kanamycin; see inset in B). Plotted are counts of different types of isolates. <em>Wild-type-</em>like isolates (black bars) have been arbitrarily defined for the analysis as non-tolerant but fit isolates (falling in the OD range indicated in the inset of B) and predominate at the start of the experiment (0 h.). Group 1 (dark grey) isolates are tolerant but sick variants. Group 2 (grey) and 3 (light grey) are tolerant and fit variants (see inset of Figure <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>B for definition of groups based on stationary-phase OD<sub>600</sub> in plain LB and 4-kan and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> for all plots related to this figure as well as plots for 0-kan control populations in which the <em>wild-type</em>-like colonies predominate throughout the course of the experiment). The counts change with time during the course of the batch evolution experiment (0 h, 24 h and P1). Results of the first replicate are displayed in the left panel and the second replicate in the right panel. (B) Populations grown in 8-kan are composed of multiple subpopulations, with the sick variants dominating at 24 h. Thus, tolerant but sick variants predominate in the first batch growth in 8-kan, and subsequent passaging results in an increase in number of the fit variants. (C) 1,000 independent growth simulations were performed for three growth regimes. This plot shows the distributions of the total population size (a reflection of the number of iterations or time) at which the ratio of the sick mutant to the fit mutant falls below 10%. Distributions are represented by box plots. The box represents the inter-quartile range and the line within the box indicates the median value. The maximum and the minimum value (1.5 times the inter-quartile range), excluding the outliers, is represented by the whiskers. The points represent the outliers. Probabilities of division per iteration are as follows. For the slow growth regime (green), µ<sub>wt</sub> = 0.01, µ<sub>sick</sub> = 0.03 and µ<sub>fit</sub> = 0.1; for the intermediate growth regime (blue), µ<sub>wt</sub> = 0.03, µ<sub>sick</sub> = 0.1 and µ<sub>fit</sub> = 0.3; for the fast growth regime (red), µ<sub>wt</sub> = 0.1, µ<sub>sick</sub> = 0.3 and µ<sub>fit</sub> = 1. (D) A scatter plot of the ratio of the number of sick to fit mutants as a function of the population size (equivalent to the number of iterations or time) for an example simulation from (C).</p></div></div></div><p>In summary, growth in 4-kan quickly results in a population of bacteria that are not only resistant to the antibiotic, but also display little defect in growth in the absence of the antibiotic. On the other hand, many sick but resistant mutants persist longer during growth in 8-kan. We suspect that the difference between the two antibiotic regimes in the trade-off between growth and antibiotic resistance might explain our observations. The lower toxicity, at lower antibiotic concentration, leads to higher growth rates. This could select for variants with low fitness defects; resistant, but sick bacteria might be competed out rapidly. On the other hand, the greater lethality imposed by the higher antibiotic concentration allows the persistence of sick bacteria for a longer time. To test this further, we performed <em>in silico</em> stochastic growth simulations in which two types of antibiotic-resistant mutant bacteria—one with high growth and the other with low growth—were competed against each other and with a very slow-growing, antibiotic-sensitive <em>wild type</em> (see Methods and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). These simulations suggest that the slow-growing mutant will be out-competed more rapidly in an environment supporting higher growth rates in general (e.g. 4-kan) than one where overall growth rates are low (8-kan) (Fig. <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>C and 2D). Our analysis of colonies had also suggested tentatively that the manner in which stationary-phase OD varies with the concentration of the antibiotic might differ across populations: while many of the fast-growing colonies show a sharp decline in growth from 4-kan to 20-kan, the slow-growing variants typically show less drastic decline in growth in 20-kan compared with 4-kan (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). This will only accentuate the trends observed in our simulations.</p>                            <h3 scrollto-destination=87199787 id="87199787" class="section-title" data-legacy-id=s3b>3.2. Representative low-cost mutations are in the translation elongation factor EF-G</h3>
<div class="block-child-p">To characterize the mutational trajectories taken by <em>E. coli</em> towards increased resistance in 4-kan, we performed deep sequencing of the (meta-)genomes of the bacterial populations at various time points during growth in the antibiotic. Two biological replicate cultures were analysed. We performed gDNA sequencing at high temporal resolution for the P0 culture (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>A). As mentioned earlier, over the course of this culture, the MIC increased ∼2-fold from the basal level. As a control, growth was monitored in an antibiotic-free medium and gDNA sequenced from mid-exponential- and stationary-phase populations. For subsequent dilutions into antibiotic-containing media (P1, P2, etc.), sequencing was performed after entry into stationary phase. <div data-id="DSU032F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03203.jpeg?Expires=2147483647&amp;Signature=X2O2APgPl6zqp7XZjAA8iD7VWmPgxvslihR0ExmeCV4IY0UopNLVd32CQenuUh1fmgV1gPQgFbH~YRF8i73N90ifxRXobUUkntBDHwhRtx7qdcvi4vaN2AZisUPKh6L~or~CSOLL~eQqHqA1mGEIgAtPlswkanwgM6-u27YTBMVRjUBCTMpZZndf0p0K9xuEEJs6EhMNoukcaGmHD-Sus90Se1obdSoDclemxe0YYj9liPeGcGN~1uAxGcukklHUSabDLVF-98twD-5QXiocxFtvEJv9siy4CfPYjj4meSv9Q~CmyevjANS36GTlQFz-SRzOUTBjG9pxww9GmNPCmg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FusAP610T confers low-cost resistance to kanamycin. (A) Heat maps indicating the proportion of reads supporting a given mutation (red—100% of reads support the mutation; black—close to 0%, i.e. very low; grey—0%, i.e. not detectable). Each column represents a time point, and the mutations are represented by rows (AP0-5: passages in 4-kan; 15–23 h: time points in AP0; ME, S and P1-5: mid-exponential- and stationary-phase time points in passage 0 and passages 1 through 5 in 0-kan; replicates are indicated below the heat map). Standard single letter amino acid representations have been used to denote substituted reside at the indicated position in superscripts. Intergenic region (IGR) mutations are represented by a ‘−/−’ between the genes flanking the IGR and the nucleotides found modified are indicated with a ‘&gt;’. For simplicity, mutations found only in 4-kan populations are shown. However, since AcrER334L was also present in the control, the mutation is included separately below. While the mutation in fusA reaches (true, i.e. 100%) fixation in 4-kan populations by P3, it is also detected in the initial round of growth (P0) albeit at very low levels. AcrER334L fixes along with FusAP610T in the second replicate but remains in the first replicate at an intermediate level. This maintenance can be explained only if FusAP610T arose again in this background (see Supplementary Data: Sanger-sequenced isolates suggest this scenario). A mutation in cpxA is also seen in AP0 in both replicates, but it is lost in subsequent passages as the mutation in fusA picks up. (B) The FusAP610T mutation alone confers ∼3-fold resistance to kanamycin. This panel shows box plots (as defined in description of Figure 2C) of MICs (n = 8) for the wild-type strain into which the FusAP610T mutation was transferred by phage transduction (FusAP610T-FRT; named so as the FRT site remains near the gene after the process). Controls wild type (WT), wild type with the FRT site at the same locus (WT-FRT) and the (back-swap) control in which the FusAP610T mutation was replaced with the wild-type allele by phage transduction are also shown along with a representative 4-kan population isolate in which the FusAP610T mutation was confirmed by Sanger sequencing. (C) Growth curves of wild type (WT) and FusAP610T-FRT in plain LB (0-kan) and (D) in LB + 4 µg/ml kanamycin (4-kan). In the absence of kanamycin, the FusAP610T strain grows almost as well as wild type. (E) The FusAP610T-FRT strain competes well against the WT-FRT strain in plain LB. Bold colours represent the competition experiment (see Methods), whereas the light colours represent the mock competition experiment (both strains were grown separately and mixed prior to plating). Plotted are log10 colony forming units (cfu)/ml versus the time point {error bars represent relative error in y, [0.434 * (dy/y)], from pooled multiple counts from two independent replicates; dy: standard deviation}. (F) Promoter activities (normalized to fall between 0 and 1) for GFPmut2 induction are similar in the wild type and the FusAP610T mutant, suggesting the absence of any defects in translation in this mutant. GFPmut2 induction from the PBAD promoter in WT-FRT and FusAP610T-FRT for two different concentrations of arabinose is shown (1 mM: 1 Ara; 10 mM: 10 Ara and uninduced control: UI). Induction was done in plain LB; period of peak promoter activity is represented by a box (error bars indicate standard deviation for four replicates). For induction in presence of kanamycin, see Supplementary Data." path-from-xml="dsu03203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199788/dsu03203.jpeg" path-from-xml="dsu03203.jpeg" target="_blank">View large</a><a section="87199788" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03203.jpeg?Expires=2147483647&amp;Signature=Nd-raH7tHixNCms6-kUd5b2adVZ3WqQiNrSdvgsKt4N0WYDus0VSAQ2FVAGcm4mF5DEK6Vd7t0buU3ooul8UKiA-nXfSuDYE22REtVOzn5sb1YVpkSElT3VN3yEwG-omK3bYvxmOYiUSjU7yh0HcoW5ZnA~slMLKTYSq9Rrx~UL~X8tii7cZIydtZDnhx9Ryon4mM2Ker6wPp59MEdtEYRB5nCy8iz~nBYG8lbtNHdhQiJh-HpFfsVAjiSJsz60YNJdLKq11QBPDQS~DKE1-dF8JUzyiDCoaHfZv~eH8Wk~Arkc7~dWN8lCbExY2rNZKvYd~VZkSyI8gWRXAa2WBzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199788&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>FusA<sup>P610T</sup> confers low-cost resistance to kanamycin. (A) Heat maps indicating the proportion of reads supporting a given mutation (red—100% of reads support the mutation; black—close to 0%, i.e. very low; grey—0%, i.e. not detectable). Each column represents a time point, and the mutations are represented by rows (AP0-5: passages in 4-kan; 15–23 h: time points in AP0; ME, S and P1-5: mid-exponential- and stationary-phase time points in passage 0 and passages 1 through 5 in 0-kan; replicates are indicated below the heat map). Standard single letter amino acid representations have been used to denote substituted reside at the indicated position in superscripts. Intergenic region (IGR) mutations are represented by a ‘−/−’ between the genes flanking the IGR and the nucleotides found modified are indicated with a ‘&gt;’. For simplicity, mutations found only in 4-kan populations are shown. However, since AcrE<sup>R334L</sup> was also present in the control, the mutation is included separately below. While the mutation in <em>fusA</em> reaches (true, i.e. 100%) fixation in 4-kan populations by P3, it is also detected in the initial round of growth (P0) albeit at very low levels. AcrE<sup>R334L</sup> fixes along with FusA<sup>P610T</sup> in the second replicate but remains in the first replicate at an intermediate level. This maintenance can be explained only if FusA<sup>P610T</sup> arose again in this background (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>: Sanger-sequenced isolates suggest this scenario). A mutation in <em>cpxA</em> is also seen in AP0 in both replicates, but it is lost in subsequent passages as the mutation in <em>fusA</em> picks up. (B) The FusA<sup>P610T</sup> mutation alone confers ∼3-fold resistance to kanamycin. This panel shows box plots (as defined in description of Figure <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>C) of MICs (<em>n</em> = 8) for the <em>wild-type</em> strain into which the FusA<sup>P610T</sup> mutation was transferred by phage transduction (FusA<sup>P610T</sup>-FRT; named so as the FRT site remains near the gene after the process). Controls <em>wild type</em> (WT), <em>wild type</em> with the FRT site at the same locus (WT-FRT) and the (back-swap) control in which the FusA<sup>P610T</sup> mutation was replaced with the <em>wild-type</em> allele by phage transduction are also shown along with a representative 4-kan population isolate in which the FusA<sup>P610T</sup> mutation was confirmed by Sanger sequencing. (C) Growth curves of <em>wild type</em> (WT) and FusA<sup>P610T</sup>-FRT in plain LB (0-kan) and (D) in LB + 4 µg/ml kanamycin (4-kan). In the absence of kanamycin, the FusA<sup>P610T</sup> strain grows almost as well as <em>wild type</em>. (E) The FusA<sup>P610T</sup>-FRT strain competes well against the WT-FRT strain in plain LB. Bold colours represent the competition experiment (see Methods), whereas the light colours represent the mock competition experiment (both strains were grown separately and mixed prior to plating). Plotted are log<sub>10</sub> colony forming units (cfu)/ml versus the time point {error bars represent relative error in <em>y</em>, [0.434 * (d<em>y</em>/<em>y</em>)], from pooled multiple counts from two independent replicates; d<em>y</em>: standard deviation}. (F) Promoter activities (normalized to fall between 0 and 1) for GFP<sup>mut2</sup> induction are similar in the <em>wild type</em> and the FusA<sup>P610T</sup> mutant, suggesting the absence of any defects in translation in this mutant. GFP<sup>mut2</sup> induction from the P<sub>BAD</sub> promoter in WT-FRT and FusA<sup>P610T</sup>-FRT for two different concentrations of arabinose is shown (1 mM: 1 Ara; 10 mM: 10 Ara and uninduced control: UI). Induction was done in plain LB; period of peak promoter activity is represented by a box (error bars indicate standard deviation for four replicates). For induction in presence of kanamycin, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div><div content-id="DSU032F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03203.jpeg?Expires=2147483647&amp;Signature=X2O2APgPl6zqp7XZjAA8iD7VWmPgxvslihR0ExmeCV4IY0UopNLVd32CQenuUh1fmgV1gPQgFbH~YRF8i73N90ifxRXobUUkntBDHwhRtx7qdcvi4vaN2AZisUPKh6L~or~CSOLL~eQqHqA1mGEIgAtPlswkanwgM6-u27YTBMVRjUBCTMpZZndf0p0K9xuEEJs6EhMNoukcaGmHD-Sus90Se1obdSoDclemxe0YYj9liPeGcGN~1uAxGcukklHUSabDLVF-98twD-5QXiocxFtvEJv9siy4CfPYjj4meSv9Q~CmyevjANS36GTlQFz-SRzOUTBjG9pxww9GmNPCmg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FusAP610T confers low-cost resistance to kanamycin. (A) Heat maps indicating the proportion of reads supporting a given mutation (red—100% of reads support the mutation; black—close to 0%, i.e. very low; grey—0%, i.e. not detectable). Each column represents a time point, and the mutations are represented by rows (AP0-5: passages in 4-kan; 15–23 h: time points in AP0; ME, S and P1-5: mid-exponential- and stationary-phase time points in passage 0 and passages 1 through 5 in 0-kan; replicates are indicated below the heat map). Standard single letter amino acid representations have been used to denote substituted reside at the indicated position in superscripts. Intergenic region (IGR) mutations are represented by a ‘−/−’ between the genes flanking the IGR and the nucleotides found modified are indicated with a ‘&gt;’. For simplicity, mutations found only in 4-kan populations are shown. However, since AcrER334L was also present in the control, the mutation is included separately below. While the mutation in fusA reaches (true, i.e. 100%) fixation in 4-kan populations by P3, it is also detected in the initial round of growth (P0) albeit at very low levels. AcrER334L fixes along with FusAP610T in the second replicate but remains in the first replicate at an intermediate level. This maintenance can be explained only if FusAP610T arose again in this background (see Supplementary Data: Sanger-sequenced isolates suggest this scenario). A mutation in cpxA is also seen in AP0 in both replicates, but it is lost in subsequent passages as the mutation in fusA picks up. (B) The FusAP610T mutation alone confers ∼3-fold resistance to kanamycin. This panel shows box plots (as defined in description of Figure 2C) of MICs (n = 8) for the wild-type strain into which the FusAP610T mutation was transferred by phage transduction (FusAP610T-FRT; named so as the FRT site remains near the gene after the process). Controls wild type (WT), wild type with the FRT site at the same locus (WT-FRT) and the (back-swap) control in which the FusAP610T mutation was replaced with the wild-type allele by phage transduction are also shown along with a representative 4-kan population isolate in which the FusAP610T mutation was confirmed by Sanger sequencing. (C) Growth curves of wild type (WT) and FusAP610T-FRT in plain LB (0-kan) and (D) in LB + 4 µg/ml kanamycin (4-kan). In the absence of kanamycin, the FusAP610T strain grows almost as well as wild type. (E) The FusAP610T-FRT strain competes well against the WT-FRT strain in plain LB. Bold colours represent the competition experiment (see Methods), whereas the light colours represent the mock competition experiment (both strains were grown separately and mixed prior to plating). Plotted are log10 colony forming units (cfu)/ml versus the time point {error bars represent relative error in y, [0.434 * (dy/y)], from pooled multiple counts from two independent replicates; dy: standard deviation}. (F) Promoter activities (normalized to fall between 0 and 1) for GFPmut2 induction are similar in the wild type and the FusAP610T mutant, suggesting the absence of any defects in translation in this mutant. GFPmut2 induction from the PBAD promoter in WT-FRT and FusAP610T-FRT for two different concentrations of arabinose is shown (1 mM: 1 Ara; 10 mM: 10 Ara and uninduced control: UI). Induction was done in plain LB; period of peak promoter activity is represented by a box (error bars indicate standard deviation for four replicates). For induction in presence of kanamycin, see Supplementary Data." path-from-xml="dsu03203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199788/dsu03203.jpeg" path-from-xml="dsu03203.jpeg" target="_blank">View large</a><a section="87199788" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03203.jpeg?Expires=2147483647&amp;Signature=Nd-raH7tHixNCms6-kUd5b2adVZ3WqQiNrSdvgsKt4N0WYDus0VSAQ2FVAGcm4mF5DEK6Vd7t0buU3ooul8UKiA-nXfSuDYE22REtVOzn5sb1YVpkSElT3VN3yEwG-omK3bYvxmOYiUSjU7yh0HcoW5ZnA~slMLKTYSq9Rrx~UL~X8tii7cZIydtZDnhx9Ryon4mM2Ker6wPp59MEdtEYRB5nCy8iz~nBYG8lbtNHdhQiJh-HpFfsVAjiSJsz60YNJdLKq11QBPDQS~DKE1-dF8JUzyiDCoaHfZv~eH8Wk~Arkc7~dWN8lCbExY2rNZKvYd~VZkSyI8gWRXAa2WBzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199788&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>FusA<sup>P610T</sup> confers low-cost resistance to kanamycin. (A) Heat maps indicating the proportion of reads supporting a given mutation (red—100% of reads support the mutation; black—close to 0%, i.e. very low; grey—0%, i.e. not detectable). Each column represents a time point, and the mutations are represented by rows (AP0-5: passages in 4-kan; 15–23 h: time points in AP0; ME, S and P1-5: mid-exponential- and stationary-phase time points in passage 0 and passages 1 through 5 in 0-kan; replicates are indicated below the heat map). Standard single letter amino acid representations have been used to denote substituted reside at the indicated position in superscripts. Intergenic region (IGR) mutations are represented by a ‘−/−’ between the genes flanking the IGR and the nucleotides found modified are indicated with a ‘&gt;’. For simplicity, mutations found only in 4-kan populations are shown. However, since AcrE<sup>R334L</sup> was also present in the control, the mutation is included separately below. While the mutation in <em>fusA</em> reaches (true, i.e. 100%) fixation in 4-kan populations by P3, it is also detected in the initial round of growth (P0) albeit at very low levels. AcrE<sup>R334L</sup> fixes along with FusA<sup>P610T</sup> in the second replicate but remains in the first replicate at an intermediate level. This maintenance can be explained only if FusA<sup>P610T</sup> arose again in this background (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>: Sanger-sequenced isolates suggest this scenario). A mutation in <em>cpxA</em> is also seen in AP0 in both replicates, but it is lost in subsequent passages as the mutation in <em>fusA</em> picks up. (B) The FusA<sup>P610T</sup> mutation alone confers ∼3-fold resistance to kanamycin. This panel shows box plots (as defined in description of Figure <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>C) of MICs (<em>n</em> = 8) for the <em>wild-type</em> strain into which the FusA<sup>P610T</sup> mutation was transferred by phage transduction (FusA<sup>P610T</sup>-FRT; named so as the FRT site remains near the gene after the process). Controls <em>wild type</em> (WT), <em>wild type</em> with the FRT site at the same locus (WT-FRT) and the (back-swap) control in which the FusA<sup>P610T</sup> mutation was replaced with the <em>wild-type</em> allele by phage transduction are also shown along with a representative 4-kan population isolate in which the FusA<sup>P610T</sup> mutation was confirmed by Sanger sequencing. (C) Growth curves of <em>wild type</em> (WT) and FusA<sup>P610T</sup>-FRT in plain LB (0-kan) and (D) in LB + 4 µg/ml kanamycin (4-kan). In the absence of kanamycin, the FusA<sup>P610T</sup> strain grows almost as well as <em>wild type</em>. (E) The FusA<sup>P610T</sup>-FRT strain competes well against the WT-FRT strain in plain LB. Bold colours represent the competition experiment (see Methods), whereas the light colours represent the mock competition experiment (both strains were grown separately and mixed prior to plating). Plotted are log<sub>10</sub> colony forming units (cfu)/ml versus the time point {error bars represent relative error in <em>y</em>, [0.434 * (d<em>y</em>/<em>y</em>)], from pooled multiple counts from two independent replicates; d<em>y</em>: standard deviation}. (F) Promoter activities (normalized to fall between 0 and 1) for GFP<sup>mut2</sup> induction are similar in the <em>wild type</em> and the FusA<sup>P610T</sup> mutant, suggesting the absence of any defects in translation in this mutant. GFP<sup>mut2</sup> induction from the P<sub>BAD</sub> promoter in WT-FRT and FusA<sup>P610T</sup>-FRT for two different concentrations of arabinose is shown (1 mM: 1 Ara; 10 mM: 10 Ara and uninduced control: UI). Induction was done in plain LB; period of peak promoter activity is represented by a box (error bars indicate standard deviation for four replicates). For induction in presence of kanamycin, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div><p>To catalogue the constituting members of the heterogeneous population resulting from growth of <em>E. coli</em> in 8-kan, we first plated 24-h-old 8-kan populations (P0 and P1) on LB agar plates. We selected multiple isolates, representing the range of MIC and growth rates observed, from four biological replicates for gDNA sequencing (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>D). From each of P0 and P1, we sequenced 11 isolates, alongside one isolate from a control population grown in plain LB.</p><p>The only mutation that had gone to fixation in the two replicate 4-kan cultures was one in <em>fusA</em> (FusA<sup>P610T</sup>), the gene encoding the translation elongation factor EF-G, which was found in the population obtained after P1 in 4-kan. The P0 culture in 4-kan had produced a genetically heterogeneous population of variants, with none supported by &gt;31% of reads (e.g. AcrE<sup>R334L</sup> mutation, also seen in <em>E. coli</em> not exposed to the antibiotic and probably not associated with kanamycin resistance as shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>, and CpxA<sup>G124C</sup> in a sensory histidine kinase with known links to antibiotic resistance<sup><a reveal-id="DSU032C49" data-open="DSU032C49" class="link link-ref link-reveal xref-bibr">49</a></sup>; Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>A, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). All these mutations were supported by high-quality Illumina sequence reads and gave no reason for suspecting artefactual data: no enrichment for being seen in sequences mapping to one strand and not particularly located at the ends of reads. Here we also note that we do not see any evidence for ribosomal gene amplifications (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). In contrast, at the end of P1, FusA<sup>P610T</sup> was supported by &gt;90% of sequencing reads. In retrospect, we noticed that the FusA<sup>P610T</sup>mutation was present at low levels (∼1%) in sequencing reads from the P0 culture in 4-kan, showing that it had started to develop during the earliest stages of the experiment, going to fixation at some point in P1. This mutation was commonly present alongside that in <em>acrE</em> and less frequently with a mutation in the intergenic region between <em>rho</em> and <em>wecA</em>, as determined by Sanger sequencing of PCR amplicons from single colonies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The population (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>B), and various clones isolated from it (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), showed ∼4-fold higher MIC for kanamycin, with no difference between clones carrying only the FusA<sup>P610T</sup> mutation and those carrying the <em>rho</em>-<em>wecA</em> mutation alongside (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The FusA<sup>P610T</sup> mutation was stable: it was found in genome sequences obtained after 5 passages (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>A), and we did not observe any change in MIC even after 10 serial passages in an antibiotic-free medium (Fig. <a reveal-id="DSU032F1" data-open="DSU032F1" class="link link-reveal link-table xref-fig">1</a>B, green bars).</p><div class="block-child-p">We identified two other FusA mutations in the 8-kan cultures, in six (out of 11) colonies from the P0 culture and eight (out of 11) colonies from P1 (Fig. <a reveal-id="DSU032F4" data-open="DSU032F4" class="link link-reveal link-table xref-fig">4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The most common mutation was FusA<sup>A608E</sup>, located close to the FusA<sup>P610T</sup> mutation found in our 4-kan cultures. The FusA<sup>A608E</sup> mutation was found alongside a stop mutation in <em>yhgA</em> (YhgA<sup>W136*</sup>); given the obvious importance of <em>fusA</em> mutations to kanamycin resistance, it is likely that the <em>yhgA</em> mutation is not contributing and is probably a hitch-hiking candidate. All the FusA<sup>A608E</sup>-containing clones had a mutation in <em>acrE</em> (AcrE<sup>R334L</sup>), which was also present in the no-antibiotic control isolate sequenced. Yet another EF-G mutation was found in a single colony obtained from P1: this mutation affected the same residue as that in the 4-kan cultures, with P610 mutated to a leucine (FusA<sup>P610L</sup>) instead of a threonine seen in the 4-kan population. <div data-id="DSU032F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03204.jpeg?Expires=2147483647&amp;Signature=zeUE-JQ0itRzXiGeNTSc9kadYmW2JeBfytt5m2jQT~IBT7gLoZxRse4KtQN-pi4SFXsH2aUL815KcAfJR25wV6b5pFxqOMo69uaeUB4NWy009fjAfqgeZb562VYJqb4RBgL3Z7BRrWpPh~RZ7vARj-zcm11qit1Ddsr~f5WCEW8~lYutm24zxO0WIQzks70th5KkZxSgIZEsrb6T-MO~b4nB74FLIQ-Dya0t8nj50T2zg7rrSGLZeeDw9A5XezGYXNjaNh6yeOY5V7yQVkYIKOdoqyKr4qxpGgESMcV6SrB4mCg5bQjOBARlKL9UcHs6p2uErI025IEBt~4Xu1SlxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Deep sequencing of isolates reveal two groups of mutations in the 8-kan evolving populations. Heat maps with the presence/absence of mutations in the isolates. Each column represents an isolate, and the mutations are represented by rows. Passage number is indicated below. The control isolate is the first column in either passages (indicated by WT). Both 0%, i.e. not detectable, and values close to 0% are represented with black for ease of visualization; for isolates, any low percentage SNPs can be ignored. The isolates can be easily put into two groups: those with a point mutation in fusA and those with deletion(s) in subunits of the ATP synthase complex. In P1, a constellation of other mutations can be seen. These second site mutations occur only in the FusAA608E background. Also, another fusA variant (FusAP610L) emerges in this passage. Standard single letter amino acid representations have been used to denote substituted residue at the indicated position in superscripts. ‘*’ indicates a STOP mutation. Δ indicates a deletion of nucleotides indicated by the number beside the symbol. Intergenic region (IGR) mutations are represented by a ‘−/−’ between the genes flanking the IGR and the nucleotides found modified are indicated with a ‘&gt;’. Insertion of ‘two cytosines’ in the gene gatC is represented as .-CC." path-from-xml="dsu03204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199791/dsu03204.jpeg" path-from-xml="dsu03204.jpeg" target="_blank">View large</a><a section="87199791" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03204.jpeg?Expires=2147483647&amp;Signature=fJX2tLg2juHS9iGOOeBvJ9X3B7bQlY2NEgI~ikESYZ27nlfVjt1Fr4rYKH1c~0JORIdE1CuPpc9df4N8uatTraHGvGthMEZ43qJg1w-y9YZd7JhqYj2XIPD5CRPz60rRfKZg09tjDHZij7zcEBFQk53kkFZpAHJ6rsbrZcSXcdO82LEbUyMoQARrLWeQnsHOSLbJkjqvqcx8j-r7OPZdwzi1vyWi5PDL7Nw1FFle0qJBJIsrDcIhGyBKTjAe71TiSeLyCB2MCZxcWlWE0LsB7YKb3oKieF1t7Ol50BMvQDupimEQwG8LF7erhxCE2TOQyLi2Z~NQJ0TdFRU6iiy0bQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199791&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Deep sequencing of isolates reveal two groups of mutations in the 8-kan evolving populations. Heat maps with the presence/absence of mutations in the isolates. Each column represents an isolate, and the mutations are represented by rows. Passage number is indicated below. The control isolate is the first column in either passages (indicated by WT). Both 0%, i.e. not detectable, and values close to 0% are represented with black for ease of visualization; for isolates, any low percentage SNPs can be ignored. The isolates can be easily put into two groups: those with a point mutation in <em>fusA</em> and those with deletion(s) in subunits of the ATP synthase complex. In P1, a constellation of other mutations can be seen. These second site mutations occur only in the FusA<sup>A608E</sup> background. Also, another <em>fusA</em> variant (FusA<sup>P610L</sup>) emerges in this passage. Standard single letter amino acid representations have been used to denote substituted residue at the indicated position in superscripts. ‘*’ indicates a STOP mutation. Δ indicates a deletion of nucleotides indicated by the number beside the symbol. Intergenic region (IGR) mutations are represented by a ‘−/−’ between the genes flanking the IGR and the nucleotides found modified are indicated with a ‘&gt;’. Insertion of ‘two cytosines’ in the gene <em>gatC</em> is represented as .-CC.</p></div></div><div content-id="DSU032F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03204.jpeg?Expires=2147483647&amp;Signature=zeUE-JQ0itRzXiGeNTSc9kadYmW2JeBfytt5m2jQT~IBT7gLoZxRse4KtQN-pi4SFXsH2aUL815KcAfJR25wV6b5pFxqOMo69uaeUB4NWy009fjAfqgeZb562VYJqb4RBgL3Z7BRrWpPh~RZ7vARj-zcm11qit1Ddsr~f5WCEW8~lYutm24zxO0WIQzks70th5KkZxSgIZEsrb6T-MO~b4nB74FLIQ-Dya0t8nj50T2zg7rrSGLZeeDw9A5XezGYXNjaNh6yeOY5V7yQVkYIKOdoqyKr4qxpGgESMcV6SrB4mCg5bQjOBARlKL9UcHs6p2uErI025IEBt~4Xu1SlxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Deep sequencing of isolates reveal two groups of mutations in the 8-kan evolving populations. Heat maps with the presence/absence of mutations in the isolates. Each column represents an isolate, and the mutations are represented by rows. Passage number is indicated below. The control isolate is the first column in either passages (indicated by WT). Both 0%, i.e. not detectable, and values close to 0% are represented with black for ease of visualization; for isolates, any low percentage SNPs can be ignored. The isolates can be easily put into two groups: those with a point mutation in fusA and those with deletion(s) in subunits of the ATP synthase complex. In P1, a constellation of other mutations can be seen. These second site mutations occur only in the FusAA608E background. Also, another fusA variant (FusAP610L) emerges in this passage. Standard single letter amino acid representations have been used to denote substituted residue at the indicated position in superscripts. ‘*’ indicates a STOP mutation. Δ indicates a deletion of nucleotides indicated by the number beside the symbol. Intergenic region (IGR) mutations are represented by a ‘−/−’ between the genes flanking the IGR and the nucleotides found modified are indicated with a ‘&gt;’. Insertion of ‘two cytosines’ in the gene gatC is represented as .-CC." path-from-xml="dsu03204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199791/dsu03204.jpeg" path-from-xml="dsu03204.jpeg" target="_blank">View large</a><a section="87199791" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03204.jpeg?Expires=2147483647&amp;Signature=fJX2tLg2juHS9iGOOeBvJ9X3B7bQlY2NEgI~ikESYZ27nlfVjt1Fr4rYKH1c~0JORIdE1CuPpc9df4N8uatTraHGvGthMEZ43qJg1w-y9YZd7JhqYj2XIPD5CRPz60rRfKZg09tjDHZij7zcEBFQk53kkFZpAHJ6rsbrZcSXcdO82LEbUyMoQARrLWeQnsHOSLbJkjqvqcx8j-r7OPZdwzi1vyWi5PDL7Nw1FFle0qJBJIsrDcIhGyBKTjAe71TiSeLyCB2MCZxcWlWE0LsB7YKb3oKieF1t7Ol50BMvQDupimEQwG8LF7erhxCE2TOQyLi2Z~NQJ0TdFRU6iiy0bQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199791&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Deep sequencing of isolates reveal two groups of mutations in the 8-kan evolving populations. Heat maps with the presence/absence of mutations in the isolates. Each column represents an isolate, and the mutations are represented by rows. Passage number is indicated below. The control isolate is the first column in either passages (indicated by WT). Both 0%, i.e. not detectable, and values close to 0% are represented with black for ease of visualization; for isolates, any low percentage SNPs can be ignored. The isolates can be easily put into two groups: those with a point mutation in <em>fusA</em> and those with deletion(s) in subunits of the ATP synthase complex. In P1, a constellation of other mutations can be seen. These second site mutations occur only in the FusA<sup>A608E</sup> background. Also, another <em>fusA</em> variant (FusA<sup>P610L</sup>) emerges in this passage. Standard single letter amino acid representations have been used to denote substituted residue at the indicated position in superscripts. ‘*’ indicates a STOP mutation. Δ indicates a deletion of nucleotides indicated by the number beside the symbol. Intergenic region (IGR) mutations are represented by a ‘−/−’ between the genes flanking the IGR and the nucleotides found modified are indicated with a ‘&gt;’. Insertion of ‘two cytosines’ in the gene <em>gatC</em> is represented as .-CC.</p></div></div></div><div class="block-child-p">The FusA<sup>P610L</sup> mutation grew almost as well in 8-kan (Fig. <a reveal-id="DSU032F5" data-open="DSU032F5" class="link link-reveal link-table xref-fig">5</a>A) as did the <em>wild type</em> in 0-kan (Fig. <a reveal-id="DSU032F5" data-open="DSU032F5" class="link link-reveal link-table xref-fig">5</a>B) and FusA<sup>P610T</sup> in 4-kan (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>C). However, the FusA<sup>P610T</sup> mutation performed relatively poorly in 8-kan (Fig. <a reveal-id="DSU032F5" data-open="DSU032F5" class="link link-reveal link-table xref-fig">5</a>A), suggesting that its effect is more restricted to lower kanamycin concentrations. The difference between the two mutations at the same site may also be reflected in the higher MIC of the FusA<sup>P610L</sup> variant (Fig. <a reveal-id="DSU032F5" data-open="DSU032F5" class="link link-reveal link-table xref-fig">5</a>C). This isolate had also an intergenic mutation at another locus (<em>ybfQ</em> -/- <em>ybfL</em>); at this point, we do not know whether this adds to the resistance phenotype of FusA<sup>P610L</sup>. We also noticed that FusA<sup>P610L</sup> grew much better than FusA<sup>A608E</sup> in 8-kan (Fig. <a reveal-id="DSU032F5" data-open="DSU032F5" class="link link-reveal link-table xref-fig">5</a>A), despite the two being obtained from the same culture conditions and carrying mutations in adjacent residues. Consistent with this observation, FusA<sup>A608E</sup>-containing isolates were embellished with additional mutations in P1, but not in P0 (Fig. <a reveal-id="DSU032F4" data-open="DSU032F4" class="link link-reveal link-table xref-fig">4</a>—P1). These, admittedly curious, mutations include RpoD<sup>L261Q</sup>, CpxA<sup>F218Y</sup>, TopA<sup>S180L</sup> and CyaA<sup>N600Y</sup>. It is notable that the FusA<sup>A608E</sup> + CpxA<sup>F218Y</sup>/TopA<sup>S180L</sup>/CyaA<sup>N600Y</sup>/RpoD<sup>L261Q</sup> double mutants showed higher growth than the FusA<sup>A608E</sup> single mutation alone in 8-kan (Fig. <a reveal-id="DSU032F5" data-open="DSU032F5" class="link link-reveal link-table xref-fig">5</a>A), though the single and the double mutations did not differ from each other in growth in the absence of the antibiotic (Fig. <a reveal-id="DSU032F5" data-open="DSU032F5" class="link link-reveal link-table xref-fig">5</a>B). And the MIC of the double mutants was higher when FusA<sup>A608E</sup> was present in combination with RpoD<sup>L261Q</sup> and CyaA<sup>N600Y</sup>; with the others, the increase was not so evident (Fig. <a reveal-id="DSU032F5" data-open="DSU032F5" class="link link-reveal link-table xref-fig">5</a>C). These might suggest genotype-dependent fine tuning of adaptation to the selected concentration of the antibiotic. <div data-id="DSU032F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03205.jpeg?Expires=2147483647&Signature=ijSKf147QXh4cPpYAFHwN0JvMfcenMepUGMzqT4mmCD7HxF8WxEv29wimRpNz61wprfPYCKi7eFPqerfdJRl3BFXEuqb-Dp~Z4SUYElqkdU8Dn9kPdF4QCdUK0loNlWtPreWezZ5Frynr9vZ~cgVQ6qcCjUcn1pMck~c0ei-gT-lQGTuwWDu3Baw9uJIC3pHPv8PBG~S~edZL3hU65EU4LNWsRlX7HQtnM4-dnTTAQlbniHTxvyH~Si8xOg19~vW1xFVhdg0c6Q7GuoaOI2cW~prDVQhkak6RqOzWzG-NteztmY4Urnu8Q~Mq016XYrkC-8be6Nw7knNB1RWE95SLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple EF-G mutations confer kanamycin resistance which is increased by second site mutations. (A) Growth curves of the EF-G mutants in 8-kan and (B) plain LB (error bars represent standard deviation from eight replicates; legend from B is applicable throughout the figure). The stationary-phase OD600 for the isolates containing the FusAA608E alone, and in combination with gatC .-CC (cytosine insertions) mutation, is lower in 8-kan than for any of the other mutants. This suggests that the second site mutations in RpoD, CpxA, TopA and CyaA may improve FusAA608E strain's ability to grow at low kanamycin concentrations. Of note, the FusAP610T mutation isolated from 4-kan populations does not grow as well as the other fusA mutant strains in 8-kan. (C) Kanamycin MICs (n = 8; box plots as defined in description of Figure 2C) for the isolates with the mutations in EF-G (fusA). The FusAP610L mutation might provide higher resistance to kanamycin than the FusAP610T mutation (P &lt; 10−5; two-sample independent t-test). However, the FusAA608E mutation alone or in combination with gatC .-CC (cytosine insertions) or TopAS180L confers resistance levels similar to the FusAP610T mutation (P = 0.289, 0.817 and 0.151, respectively). In combination with CyaAN600Y, CpxAF218Y,and RpoDL261Q, the FusAA608E mutation does better and has higher resistance than either FusAP610T or FusAA608E alone (P = 0.006, 0.0003 and &lt;10−5, respectively). The emergence of some of these mutations in P1, although a resistant mutation is already present, may be explained by this observation." path-from-xml="dsu03205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199792/dsu03205.jpeg" path-from-xml="dsu03205.jpeg" target="_blank">View large</a><a section="87199792" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03205.jpeg?Expires=2147483647&Signature=hG1TC5o3XayCtE6vJY7KrPVRUK4iCCT4jRP7D2Gy1vmBnoWmfIIMInCsJLZ~569jvTTiNwiR3qDQbmjIu6i9ZGpXxxeGhyLBDLHCxMtzycAwxpKIfyWwiUSmsCtRY1Xyw2RYtoVdlG4c-gVsXf2~cOkkh44B7NYxtq0DrB2I80eTfMos1szaoX0kEa2Zf~41qMoPzAA4bW0Nblc2yo8OazCJeBuacapjOgyw6khmnt7eUqZL5O6oWVEPsgL256QLDsD9Vz0IXBIhKqwm8uehL2BXourE5weZMKdtaMXwwTUKxfk22Oya9cYtB6w9toA669igixhYPXDPg~BaxWI5Pw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199792&ar=2754555&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple EF-G mutations confer kanamycin resistance which is increased by second site mutations. (A) Growth curves of the EF-G mutants in 8-kan and (B) plain LB (error bars represent standard deviation from eight replicates; legend from B is applicable throughout the figure). The stationary-phase OD<sub>600</sub> for the isolates containing the FusA<sup>A608E</sup> alone, and in combination with <em>gatC</em> .-CC (cytosine insertions) mutation, is lower in 8-kan than for any of the other mutants. This suggests that the second site mutations in RpoD, CpxA, TopA and CyaA may improve FusA<sup>A608E</sup> strain's ability to grow at low kanamycin concentrations. Of note, the FusA<sup>P610T</sup> mutation isolated from 4-kan populations does not grow as well as the other <em>fusA</em> mutant strains in 8-kan. (C) Kanamycin MICs (<em>n</em> = 8; box plots as defined in description of Figure <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>C) for the isolates with the mutations in EF-G (<em>fusA</em>). The FusA<sup>P610L</sup> mutation might provide higher resistance to kanamycin than the FusA<sup>P610T</sup> mutation (<em>P</em> &lt; 10<sup>−5</sup>; two-sample independent <em>t</em>-test). However, the FusA<sup>A608E</sup> mutation alone or in combination with <em>gatC</em> .-CC (cytosine insertions) or TopA<sup>S180L</sup> confers resistance levels similar to the FusA<sup>P610T</sup> mutation (<em>P</em> = 0.289, 0.817 and 0.151, respectively). In combination with CyaA<sup>N600Y</sup>, CpxA<sup>F218Y</sup>,and RpoD<sup>L261Q</sup>, the FusA<sup>A608E</sup> mutation does better and has higher resistance than either FusA<sup>P610T</sup> or FusA<sup>A608E</sup> alone (<em>P</em> = 0.006, 0.0003 and &lt;10<sup>−5</sup>, respectively). The emergence of some of these mutations in P1, although a resistant mutation is already present, may be explained by this observation.</p></div></div><div content-id="DSU032F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03205.jpeg?Expires=2147483647&Signature=ijSKf147QXh4cPpYAFHwN0JvMfcenMepUGMzqT4mmCD7HxF8WxEv29wimRpNz61wprfPYCKi7eFPqerfdJRl3BFXEuqb-Dp~Z4SUYElqkdU8Dn9kPdF4QCdUK0loNlWtPreWezZ5Frynr9vZ~cgVQ6qcCjUcn1pMck~c0ei-gT-lQGTuwWDu3Baw9uJIC3pHPv8PBG~S~edZL3hU65EU4LNWsRlX7HQtnM4-dnTTAQlbniHTxvyH~Si8xOg19~vW1xFVhdg0c6Q7GuoaOI2cW~prDVQhkak6RqOzWzG-NteztmY4Urnu8Q~Mq016XYrkC-8be6Nw7knNB1RWE95SLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple EF-G mutations confer kanamycin resistance which is increased by second site mutations. (A) Growth curves of the EF-G mutants in 8-kan and (B) plain LB (error bars represent standard deviation from eight replicates; legend from B is applicable throughout the figure). The stationary-phase OD600 for the isolates containing the FusAA608E alone, and in combination with gatC .-CC (cytosine insertions) mutation, is lower in 8-kan than for any of the other mutants. This suggests that the second site mutations in RpoD, CpxA, TopA and CyaA may improve FusAA608E strain's ability to grow at low kanamycin concentrations. Of note, the FusAP610T mutation isolated from 4-kan populations does not grow as well as the other fusA mutant strains in 8-kan. (C) Kanamycin MICs (n = 8; box plots as defined in description of Figure 2C) for the isolates with the mutations in EF-G (fusA). The FusAP610L mutation might provide higher resistance to kanamycin than the FusAP610T mutation (P &lt; 10−5; two-sample independent t-test). However, the FusAA608E mutation alone or in combination with gatC .-CC (cytosine insertions) or TopAS180L confers resistance levels similar to the FusAP610T mutation (P = 0.289, 0.817 and 0.151, respectively). In combination with CyaAN600Y, CpxAF218Y,and RpoDL261Q, the FusAA608E mutation does better and has higher resistance than either FusAP610T or FusAA608E alone (P = 0.006, 0.0003 and &lt;10−5, respectively). The emergence of some of these mutations in P1, although a resistant mutation is already present, may be explained by this observation." path-from-xml="dsu03205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199792/dsu03205.jpeg" path-from-xml="dsu03205.jpeg" target="_blank">View large</a><a section="87199792" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03205.jpeg?Expires=2147483647&Signature=hG1TC5o3XayCtE6vJY7KrPVRUK4iCCT4jRP7D2Gy1vmBnoWmfIIMInCsJLZ~569jvTTiNwiR3qDQbmjIu6i9ZGpXxxeGhyLBDLHCxMtzycAwxpKIfyWwiUSmsCtRY1Xyw2RYtoVdlG4c-gVsXf2~cOkkh44B7NYxtq0DrB2I80eTfMos1szaoX0kEa2Zf~41qMoPzAA4bW0Nblc2yo8OazCJeBuacapjOgyw6khmnt7eUqZL5O6oWVEPsgL256QLDsD9Vz0IXBIhKqwm8uehL2BXourE5weZMKdtaMXwwTUKxfk22Oya9cYtB6w9toA669igixhYPXDPg~BaxWI5Pw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199792&ar=2754555&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple EF-G mutations confer kanamycin resistance which is increased by second site mutations. (A) Growth curves of the EF-G mutants in 8-kan and (B) plain LB (error bars represent standard deviation from eight replicates; legend from B is applicable throughout the figure). The stationary-phase OD<sub>600</sub> for the isolates containing the FusA<sup>A608E</sup> alone, and in combination with <em>gatC</em> .-CC (cytosine insertions) mutation, is lower in 8-kan than for any of the other mutants. This suggests that the second site mutations in RpoD, CpxA, TopA and CyaA may improve FusA<sup>A608E</sup> strain's ability to grow at low kanamycin concentrations. Of note, the FusA<sup>P610T</sup> mutation isolated from 4-kan populations does not grow as well as the other <em>fusA</em> mutant strains in 8-kan. (C) Kanamycin MICs (<em>n</em> = 8; box plots as defined in description of Figure <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>C) for the isolates with the mutations in EF-G (<em>fusA</em>). The FusA<sup>P610L</sup> mutation might provide higher resistance to kanamycin than the FusA<sup>P610T</sup> mutation (<em>P</em> &lt; 10<sup>−5</sup>; two-sample independent <em>t</em>-test). However, the FusA<sup>A608E</sup> mutation alone or in combination with <em>gatC</em> .-CC (cytosine insertions) or TopA<sup>S180L</sup> confers resistance levels similar to the FusA<sup>P610T</sup> mutation (<em>P</em> = 0.289, 0.817 and 0.151, respectively). In combination with CyaA<sup>N600Y</sup>, CpxA<sup>F218Y</sup>,and RpoD<sup>L261Q</sup>, the FusA<sup>A608E</sup> mutation does better and has higher resistance than either FusA<sup>P610T</sup> or FusA<sup>A608E</sup> alone (<em>P</em> = 0.006, 0.0003 and &lt;10<sup>−5</sup>, respectively). The emergence of some of these mutations in P1, although a resistant mutation is already present, may be explained by this observation.</p></div></div></div><p>The above analysis show that several mutations in <em>fusA</em> arise in cultures grown in kanamycin and spread through the population rapidly. All the three mutations are present at the carboxy-end of Domain 4 of EF-G, the tip of which interacts with the decoding site of the ribosome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). To specifically investigate the contribution of FusA<sup>P610T</sup> to kanamycin resistance and its cost in terms of growth in the absence of the antibiotic, we used phage transduction to move this mutation into a parental <em>wild-type</em> strain. This strain now had the FusA<sup>P610T</sup> mutation in addition to an FRT site downstream of <em>fusA</em> (between the <em>rpsG</em>-<em>fusA</em>-<em>tufA</em> transcription unit and <em>chiA</em>). Genome sequencing of this clone showed that no other mutation had been introduced during transduction (data not shown). This strain is henceforth referred to as the <em>fusA<sup>mut</sup>::FRT</em> strain. The characteristics of <em>fusA<sup>mut</sup>::FRT</em> strain were compared with a <em>wild type</em> carrying the FRT site in the same region (<em>wt::FRT</em>) and with the parental <em>wild type</em>. We measured the MIC of the <em>fusA<sup>mut</sup>::FRT</em> strain and found it to be comparable with those of the clones carrying the mutation (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>B, compare FA-FRT and Isolate 1). The <em>wt::FRT</em> showed no resistance, similar to the <em>wild type</em> without the FRT site. The <em>wt::FRT</em>, as well as the parental <em>wild type</em>, showed little or slow growth in solid or liquid media containing 4-kan (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>C). However, the <em>fusA<sup>mut</sup>::FRT</em> strain grew as well in 4-kan (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>C) as the <em>wt::FRT</em> and the parental <em>wild type</em> in non-antibiotic medium (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>D). Finally, the <em>fusA<sup>mut</sup>::FRT</em> strain, when replaced with the <em>wt::FRT</em> allele by phage transduction, lost its resistance to kanamycin (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>B). These show that the FusA<sup>P610T</sup> mutation contributes almost entirely to the resistance of the 4-kan evolved population.</p><p>The following set of growth experiments allowed us to confirm that the FusA<sup>P610T</sup> mutation does not result in a perceivable growth defect in the absence of the antibiotic: <em>fusA<sup>mut</sup>::FRT</em> could grow almost as well as <em>wt::FRT</em> in LB without kanamycin (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>D); the <em>fusA<sup>mut</sup>::FRT</em> strain grew well and conferred kanamycin resistance even at 42°C (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), showing the absence of a temperature-sensitive phenotype in contrast to previously isolated FusA mutants<sup><a reveal-id="DSU032C32" data-open="DSU032C32" class="link link-ref link-reveal xref-bibr">32</a></sup>; the <em>fusA<sup>mut</sup>::FRT</em> strain competed well with <em>wt::FRT</em> strain in competition assays (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>E and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> for competitions with <em>wild type</em> without the FRT site). Finally, the <em>fusA<sup>mut</sup>::FRT</em> strain did not seem to have defects in translation as seen from kinetics of induction of GFP<sup>mut2</sup> from a tightly controlled arabinose-inducible promoter (pBAD18::<em>gfpmut2</em>) (Fig. <a reveal-id="DSU032F3" data-open="DSU032F3" class="link link-reveal link-table xref-fig">3</a>F). For induction in the presence of kanamycin, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p><p>Hou <em>et al.</em>,<sup><a reveal-id="DSU032C32" data-open="DSU032C32" class="link link-ref link-reveal xref-bibr">32</a></sup> in their report of a screen for temperature-sensitive mutations in EF-G, document two mutations: <em>fusA100</em> (FusA<sup>G502D</sup>) and <em>fusA101</em> (FusA<sup>Q495P</sup>). The screen was performed by selection of resistant colonies at 30°C on LB agar containing lethal concentrations of kanamycin. These two strains seemed to incur a heavy cost manifested by growth defects even at non-restrictive temperatures and had defects in translation as seen from kinetics of induction of a reporter gene (β-gal in their assay). In contrast, our study, by performing selection experiments at sublethal kanamycin concentration, has identified a group of mutations with little detectable defect in growth or induction of translation.</p>                            <h3 scrollto-destination=87199796 id="87199796" class="section-title" data-legacy-id=s3c>3.3. Mutations in ATP synthase confer costly, basal level resistance to kanamycin</h3>
<p>Whole genome sequencing indicated that isolates from the 8-kan population (5 from P0 and 3 from P1), which did not contain a mutation in EF-G, contained deletions in genes encoding subunits of ATP synthase (Fig. <a reveal-id="DSU032F4" data-open="DSU032F4" class="link link-reveal link-table xref-fig">4</a>). The deletions in the genes encoding subunits of ATP synthase were of different lengths: deletions of 108 bp (three isolates), 28 bp (1 isolate) and 1 bp (1 isolate) were seen in <em>atpG</em>, encoding the γ-subunit of the ATP synthase complex; and a deletion of 15 bp (1 isolate) was seen in <em>atpA</em> encoding the α-subunit. The isolate with the 15 bp deletion in <em>atpA</em> also had the 1 bp deletion in <em>atpG</em>. A recent large-scale study of aminoglycoside resistance mutations had identified mutations not only in the ATP synthase, but also in other members of the oxidative phosphorylation pathway.<sup><a reveal-id="DSU032C33" data-open="DSU032C33" class="link link-ref link-reveal xref-bibr">33</a></sup> This suggests that inactivation of the ATP synthase complex and oxidative phosphorylation must be a general mechanism towards resisting kanamycin. Whereas the three types of ATP synthase mutations described above were found in P0, only the 108 bp deletion in <em>atpG</em> could be recovered in P1 (Fig. <a reveal-id="DSU032F4" data-open="DSU032F4" class="link link-reveal link-table xref-fig">4</a>).</p><div class="block-child-p">We analysed the isolates in which the ATP synthase mutations were present in further detail. All the three mutations in <em>atpG</em> are expected to <em>delete</em> the C-terminal end of the γ-subunit of the F0 component of the ATP synthase (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Since most of these strains are fully sequenced and do not possess any other mutations apart from the AcrE<sup>R334L</sup> that does not confer kanamycin resistance, we could work directly with these strains. Surprisingly, only the isolates with the 108 bp deletion in <em>atpG</em> seem to be around 1.5- to 2-fold resistant to kanamycin (Fig. <a reveal-id="DSU032F6" data-open="DSU032F6" class="link link-reveal link-table xref-fig">6</a>A) and grow well in 8-kan (Fig. <a reveal-id="DSU032F6" data-open="DSU032F6" class="link link-reveal link-table xref-fig">6</a>B), while the other deletion strains do not. Regardless of the resistance phenotype, all of these strains have incurred a severe cost as a result of the deletions as seen from their growth in plain LB (Fig. <a reveal-id="DSU032F6" data-open="DSU032F6" class="link link-reveal link-table xref-fig">6</a>C). Consistent with this, there is a relative decline in the abundance of ATP synthase mutants in P1 compared with P0, with a concomitant increase in the abundance of the less costly mutations in FusA. <div data-id="DSU032F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03206.jpeg?Expires=2147483647&amp;Signature=I24J7K-qFqCr8T-K1VGY~N6v6bXLLSI5yImt-sgXDToj1sGyh6dNnB1r0Hj3kQzJzlr55Wh7SdR~exV-1-0cxCFMd5-gUzxFjp~vNGTBH7wZu6NSClwhe-JSn3sRvi0EzOuXqAgijq8dvLYMrgGLDYsij9x6dR0u3loXrbrUUZE~pF9sQHEWR20-L0luEWWmRVPg9g4xq6hpBWXiqDTf~tMExEJWWGZgeobriajNU7RKOsBHTp6IqXDYNZvzDG7d9it4~UtkkbacKIL~V0lX4YFX0IKh6DE-OgxkydGmsLrzgeDq3B3PywF7uCkeKHXIC~VbsuI-ipo9XTYQDksz7A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Deletions in subunits of the ATP synthase gene are costly mutations that confer kanamycin resistance. (A) Kanamycin MICs (n = 8, box plots as defined in description of Figure 2C) for the isolates with the deletions (two-sample independent t-tests for wild type against isolates yield P values &lt; 10−5 for all except isolates L: P = 0.0095 and X: P = 0.000879). Thus, these deletions increase resistance to kanamycin only slightly. (B) Growth curves of the ATP synthase mutants in 8-kan and (C) 0-kan (error bars represent standard deviation from eight replicates; legend from B is applicable throughout the figure). In plain LB, the ATP synthase mutants grow to a much lesser extent than wild type, suggesting a huge cost factor associated with these deletion mutations. The strains with the 108 bp deletion in atpG are able to grow in 8-kan to approximately the same extent as in the absence of kanamycin. Oddly though, isolates H (atpGΔ28) and L (atpGΔ1, atpAΔ15), although possessing a similar mutation (refer Supplementary Data), do not grow well in 8-kan. This is consistent with these strains not being picked up in the P1 (Fig. 4), although limited sampling could also result in this." path-from-xml="dsu03206.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199798/dsu03206.jpeg" path-from-xml="dsu03206.jpeg" target="_blank">View large</a><a section="87199798" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03206.jpeg?Expires=2147483647&amp;Signature=GiudFuY8pomiOY04m236RMpd2-r9JeXvHIkhsJBo93pCMf2SPEOOrlJ~HA7x~rIXgSPR4MSJRnSU~8ADFC1CDGudknDDEXIypba2yWHyqtEk-CeY7oaST2cY6t739PHo4XCY-rNQ3pvXca383ZPgFT9eJNfOWQZkdM2ArJfnucuOGIb-XrgeV0EwXHUsdcxDoFXdxAFGQCgnSFKcUcAXKe702HfQnFZ657NYiJVE0RqjrPpGyQy481ebu~QwbBAmshwP32nUJRAl5SimOEs1zvBxeGcYikGcNSxCQYYgB-7pOW0pvIy5h~wSZZSpYMwle79HY6b8PtcPNjMHFqtkqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199798&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03206.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Deletions in subunits of the ATP synthase gene are costly mutations that confer kanamycin resistance. (A) Kanamycin MICs (<em>n</em> = 8, box plots as defined in description of Figure <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>C) for the isolates with the deletions (two-sample independent <em>t</em>-tests for <em>wild type</em> against isolates yield <em>P</em> values &lt; 10<sup>−5</sup> for all except isolates L: <em>P</em> = 0.0095 and X: <em>P</em> = 0.000879). Thus, these deletions increase resistance to kanamycin only slightly. (B) Growth curves of the ATP synthase mutants in 8-kan and (C) 0-kan (error bars represent standard deviation from eight replicates; legend from B is applicable throughout the figure). In plain LB, the ATP synthase mutants grow to a much lesser extent than <em>wild type</em>, suggesting a huge cost factor associated with these deletion mutations. The strains with the 108 bp deletion in <em>atpG</em> are able to grow in 8-kan to approximately the same extent as in the absence of kanamycin. Oddly though, isolates H (<em>atpG</em>Δ28) and L (<em>atpG</em>Δ1, <em>atpA</em>Δ15), although possessing a similar mutation (refer <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), do not grow well in 8-kan. This is consistent with these strains not being picked up in the P1 (Fig. <a reveal-id="DSU032F4" data-open="DSU032F4" class="link link-reveal link-table xref-fig">4</a>), although limited sampling could also result in this.</p></div></div><div content-id="DSU032F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/m_dsu03206.jpeg?Expires=2147483647&amp;Signature=I24J7K-qFqCr8T-K1VGY~N6v6bXLLSI5yImt-sgXDToj1sGyh6dNnB1r0Hj3kQzJzlr55Wh7SdR~exV-1-0cxCFMd5-gUzxFjp~vNGTBH7wZu6NSClwhe-JSn3sRvi0EzOuXqAgijq8dvLYMrgGLDYsij9x6dR0u3loXrbrUUZE~pF9sQHEWR20-L0luEWWmRVPg9g4xq6hpBWXiqDTf~tMExEJWWGZgeobriajNU7RKOsBHTp6IqXDYNZvzDG7d9it4~UtkkbacKIL~V0lX4YFX0IKh6DE-OgxkydGmsLrzgeDq3B3PywF7uCkeKHXIC~VbsuI-ipo9XTYQDksz7A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Deletions in subunits of the ATP synthase gene are costly mutations that confer kanamycin resistance. (A) Kanamycin MICs (n = 8, box plots as defined in description of Figure 2C) for the isolates with the deletions (two-sample independent t-tests for wild type against isolates yield P values &lt; 10−5 for all except isolates L: P = 0.0095 and X: P = 0.000879). Thus, these deletions increase resistance to kanamycin only slightly. (B) Growth curves of the ATP synthase mutants in 8-kan and (C) 0-kan (error bars represent standard deviation from eight replicates; legend from B is applicable throughout the figure). In plain LB, the ATP synthase mutants grow to a much lesser extent than wild type, suggesting a huge cost factor associated with these deletion mutations. The strains with the 108 bp deletion in atpG are able to grow in 8-kan to approximately the same extent as in the absence of kanamycin. Oddly though, isolates H (atpGΔ28) and L (atpGΔ1, atpAΔ15), although possessing a similar mutation (refer Supplementary Data), do not grow well in 8-kan. This is consistent with these strains not being picked up in the P1 (Fig. 4), although limited sampling could also result in this." path-from-xml="dsu03206.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199798/dsu03206.jpeg" path-from-xml="dsu03206.jpeg" target="_blank">View large</a><a section="87199798" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu03206.jpeg?Expires=2147483647&amp;Signature=GiudFuY8pomiOY04m236RMpd2-r9JeXvHIkhsJBo93pCMf2SPEOOrlJ~HA7x~rIXgSPR4MSJRnSU~8ADFC1CDGudknDDEXIypba2yWHyqtEk-CeY7oaST2cY6t739PHo4XCY-rNQ3pvXca383ZPgFT9eJNfOWQZkdM2ArJfnucuOGIb-XrgeV0EwXHUsdcxDoFXdxAFGQCgnSFKcUcAXKe702HfQnFZ657NYiJVE0RqjrPpGyQy481ebu~QwbBAmshwP32nUJRAl5SimOEs1zvBxeGcYikGcNSxCQYYgB-7pOW0pvIy5h~wSZZSpYMwle79HY6b8PtcPNjMHFqtkqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199798&amp;ar=2754555&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03206.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Deletions in subunits of the ATP synthase gene are costly mutations that confer kanamycin resistance. (A) Kanamycin MICs (<em>n</em> = 8, box plots as defined in description of Figure <a reveal-id="DSU032F2" data-open="DSU032F2" class="link link-reveal link-table xref-fig">2</a>C) for the isolates with the deletions (two-sample independent <em>t</em>-tests for <em>wild type</em> against isolates yield <em>P</em> values &lt; 10<sup>−5</sup> for all except isolates L: <em>P</em> = 0.0095 and X: <em>P</em> = 0.000879). Thus, these deletions increase resistance to kanamycin only slightly. (B) Growth curves of the ATP synthase mutants in 8-kan and (C) 0-kan (error bars represent standard deviation from eight replicates; legend from B is applicable throughout the figure). In plain LB, the ATP synthase mutants grow to a much lesser extent than <em>wild type</em>, suggesting a huge cost factor associated with these deletion mutations. The strains with the 108 bp deletion in <em>atpG</em> are able to grow in 8-kan to approximately the same extent as in the absence of kanamycin. Oddly though, isolates H (<em>atpG</em>Δ28) and L (<em>atpG</em>Δ1, <em>atpA</em>Δ15), although possessing a similar mutation (refer <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), do not grow well in 8-kan. This is consistent with these strains not being picked up in the P1 (Fig. <a reveal-id="DSU032F4" data-open="DSU032F4" class="link link-reveal link-table xref-fig">4</a>), although limited sampling could also result in this.</p></div></div></div><p>Taken together, genome-sequencing experiments show that EF-G variants represent low-cost, kanamycin-resistant mutations. Several mutations, mapping to the same or adjacent residues in EF-G, confer resistance to the antibiotic. On the other hand, mutations in ATP synthase confer resistance to kanamycin, but also display severe growth defects.</p>                            <h2 scrollto-destination=87199800 id="87199800" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Antibiotic resistance is a major problem in health today. In particular, development of resistance under sub-inhibitory concentrations of an antibiotic is of interest. Bacteria are believed to be routinely exposed to sublethal concentrations of various antibiotics in their natural environments over long periods of time. In a clinical setting, improper adherence to a dose regime could lead to low levels of antibiotics in the body. Mutations conferring resistance to antibiotics target essential genes which may tolerate mutations poorly. Therefore, they represent a balance between the level of resistance they confer, as well as a possible loss of fitness—in the absence of the antibiotic. In addition, the accessibility of a resistance-conferring mutation also affects the kinetics of the evolution of resistance.</p><p>In this study, we observe that resistance of laboratory <em>E. coli</em> to two sublethal concentrations of the aminoglycoside kanamycin eventually produces a mutation that provides sufficient resistance to the antibiotic, while being of low cost as measured by growth or fitness in the absence of the antibiotic. This is only to be expected given enough time for selection to act as long as a less costly mutation (or a combination of mutations) for resistance is available to the bacterium. In a recent, intriguing report, an antibiotic resistance-conferring mutation was found in fact to enhance fitness of <em>Salmonella</em>.<sup><a reveal-id="DSU032C50" data-open="DSU032C50" class="link link-ref link-reveal xref-bibr">50</a></sup></p><p>In cases where the levels of resistance conferred by a mutation are negatively correlated with its fitness, one would expect resistance to be persistently associated with a growth defect. Our experimental conditions presumably do not impose such a trade-off, though this needs to be systematically evaluated. We observe however that the selection under high sublethal concentrations of kanamycin results in the longer persistence of growth-defective variants in the population. This is consistent with the prior isolation of two defective, but resistant mutations by selection under lethal kanamycin concentrations.<sup><a reveal-id="DSU032C32" data-open="DSU032C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>A representative low-cost mutation we have isolated is in the translation elongation factor EF-G. It is surprising that at least one of the mutations here displays little defect in growth or in inducing translation of GFP <em>in cellulo</em>. Further, the mutated residue P610, which we have investigated in some detail here, and A608 are both highly conserved as observed from multiple sequence alignments of EF-G from &gt;2,000 bacterial genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Analysis of this residue in the context of EF-G structure shows that it is located at the C-terminal end of ‘Domain IV’ (DIV; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), whose tip interacts with the ribosomal decoding site. Mutation from a Proline to any other residue (Threonine and Leucine in our study; Glutamate in a recent screen for aminoglycoside resistance<sup><a reveal-id="DSU032C33" data-open="DSU032C33" class="link link-ref link-reveal xref-bibr">33</a></sup>) might increase the flexibility in movement of DIV, a feature that appears to play important roles during translocation.<sup><a reveal-id="DSU032C51" data-open="DSU032C51" class="link link-ref link-reveal xref-bibr">51</a></sup></p><p>The levels of resistance conferred by the P610T mutation to a panel of aminoglycosides (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), in the context of a recent single-molecule study<sup><a reveal-id="DSU032C15" data-open="DSU032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> investigating the contribution of translocation inhibition to the activities of these antibiotics, suggest that its contribution to resistance is consistent with an effect on translocation. Aminoglycosides also affect ribosome recycling<sup><a reveal-id="DSU032C16" data-open="DSU032C16" class="link link-ref link-reveal xref-bibr">16</a></sup>; however, the mutation we describe is unable to restore a temperature-sensitive phenotype of a mutation in the ribosome release factor (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). In summary, a mutation at the C-terminal end of DIV of EF-G confers low-cost resistance to 2-DOS aminoglycoside antibiotics in a manner probably consistent with an effect on ribosome translocation. We note here that, as described in the ‘Results’, not all EF-G mutations—despite targeting the same site or adjacent sites—are equal.</p><p>At the other end of the fitness scale is the set of mutations we have identified in the ATP synthase. Most of these mutations are in the gamma component of the F0 subunit. The gamma polypeptide is the shaft that couples the proton gradient-driven rotation of the F1 subunit to the catalytic region on the F0 subunit</p><p>All the mutations in the gamma polypeptide, either as frame-shifts or as in-frame deletions, remove the C-terminal ∼40 residues of the polypeptide (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). This end of the polypeptide cosily nestles in a region surrounded by the catalytic components of the ATP synthase (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). It is thus reasonable to expect that these mutations abolish ATP synthase activity. Though <em>E. coli</em> should be able to synthesize ATP via substrate-level phosphorylation, oxidative phosphorylation culminating in ATP synthase activity is the major source of ATP. Thus, as expected, these mutations in ATP synthase result in severe growth defects. How inactivation of the ATP synthase can confer resistance to aminoglycosides is open to debate. To our knowledge, there are two relevant models in the literature. A recently published large-scale screen for aminoglycoside resistance identified members of the oxidative phosphorylation pathway as a major class of targets of mutation.<sup><a reveal-id="DSU032C33" data-open="DSU032C33" class="link link-ref link-reveal xref-bibr">33</a></sup> These researchers report that a reduction in the PMF, as a consequence of these mutations, increases aminoglycoside resistance by decreasing the uptake of these antibiotics. On the flip side, these mutations increase sensitivity to other classes of antibiotics, which require PMF for efflux. A second model is related to the generation of reactive oxygen species. It has been suggested that many antibiotics kill their target cells by producing reactive oxygen.<sup><a reveal-id="DSU032C52" data-open="DSU032C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Inhibition of ATP synthase activity would eventually result in the cessation of oxidative phosphorylation, thus reducing reactive oxygen production. However, this model remains controversial and the jury is out to decide.<sup><a reveal-id="DSU032C24" data-open="DSU032C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU032C26" data-open="DSU032C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p><p>Besides representing two extremes in the fitness landscape, the two mutations described here also represent two levels of ‘accessibility’. The space of mutations for ATP synthase inactivation is potentially large and, therefore, easily achieved. On the other hand, a mutation in EF-G, which specifically affects a certain function, resulting in low-cost aminoglycoside resistance is likely to be relatively rare. Despite this, the fact that the mutation in EF-G is more commonly observed in our experiments is a reflection of the considerable over-compensation of the difference in accessibility by that in fitness, both in the presence and in the absence of the antibiotic.</p><p>Mutations in EF-G may also indicate fine tuning of adaptation to narrow windows of antibiotic concentration: a mutation selected in a low sublethal concentration of the antibiotic resists a higher concentration poorly, though one selected in the latter performs better at this concentration despite the two mutations targeting the same amino acid residue (P to T versus P to L). In contrast, mutations in ATP synthase may confer a basal level of resistance that is robust to a broader range of antibiotic concentrations. This is suggested by a difference, between mutations in EF-G and those in ATP synthase, in the manner in which their growth decreases with increasing concentration of kanamycin. To visualize this, we first obtain 0-kan normalized MIC curves in which growth in (any) concentration of kanamycin, as measured by OD<sub>600</sub> after 24 h, is normalized with respect to growth in the absence of kanamycin. Based on this, we find that the decline in growth with increasing antibiotic concentration is steeper for EF-G mutations than for ATP synthase mutations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>).</p><p>In summary, our work points to the relevance of sublethal concentrations of an antibiotic to the rapid development of low-cost and resistant mutants, while also describing two classes of mutations—one obviously costly and the other less so—that confer resistance to a subset of aminoglycoside antibiotics.</p>                            <h2 scrollto-destination=87199811 id="87199811" class="section-title" data-legacy-id=s5>5. Accession numbers</h2>
<p>Genomic deep-sequencing data can be accessed from the NCBI Sequence Read Archive (accession number: SRP040449).</p>                            <h2 scrollto-destination=87199813 id="87199813" class="section-title" data-legacy-id=s7>Funding</h2>
<p>The work was supported by the National Centre for Biological Sciences core funding. Funding to pay the Open Access publication charges for this article was provided by the National Centre for Biological Sciences, Tata Institute of Fundamental Research.</p>                            <h2 scrollto-destination=87199815 id="87199815" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Prof. Umesh Varshney (IISc, Bangalore), Srinivas A. (IISc, Bangalore), Prof. Joseph Puglisi (Stanford University School of Medicine), Prof. Venkatraman Ramakrishnan (MRC Laboratory of Molecular Biology), Prof. J. Gowrishankar (CDFD, Hyderabad), Prof. Ramaswamy Subramaniam and Dr Shachi Gosavi for discussion and comments and Prof. U. Varshney for gift of <em>E. coli</em> strain carrying temperature-sensitive RRF. We thank Krishnamohan Atmakuri (THSTI, Gurgaon) for comments on the manuscript. We declare that this does not imply that these scientists endorse the findings of this paper. We thank Meenakshi Iyer and Deepti Chandraprakash for help with some growth experiments. We acknowledge C-CAMP for providing next-generation sequencing services. We thank the Editor and the anonymous referees for their comments on the manuscript.</p>                            <h2 scrollto-destination=87199817 id="87199817" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU032C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hindré</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Knibbe</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Beslon</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schneider</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="article-title">New insights into bacterial adaptation through in vivo and in silico experimental evolution</div>
          <div class="source">Nat. Rev. Microbiol.</div> 
          <div class="year">2012</div>
          <div class="volume">10</div>
          <div class="fpage">352</div>
          <div class="lpage">65</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20insights%20into%20bacterial%20adaptation%20through%20in%20vivo%20and%20in%20silico%20experimental%20evolution&amp;author=T.%20Hindr%C3%A9&amp;author=C.%20Knibbe&amp;author=G.%20Beslon&amp;author=D.%20Schneider&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=10&amp;pages=352-65" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22450379" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Davies</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Davies</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="article-title">Origins and evolution of antibiotic resistance</div>
          <div class="source">Microbiol. Mol. Biol. Rev.</div> 
          <div class="year">2010</div>
          <div class="volume">74</div>
          <div class="fpage">417</div>
          <div class="lpage">33</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%20and%20evolution%20of%20antibiotic%20resistance&amp;author=J.%20Davies&amp;author=D.%20Davies&amp;publication_year=2010&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=74&amp;pages=417-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00016-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00016-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00016-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20805405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cox</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wright</div>
               <div class="given-names">G.D.</div>
            </div>
          </div>
          <div class="article-title">Intrinsic antibiotic resistance: mechanisms, origins, challenges and solutions</div>
          <div class="source">Int. J. Med. Microbiol.</div> 
          <div class="year">2013</div>
          <div class="volume">303</div>
          <div class="fpage">287</div>
          <div class="lpage">92</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intrinsic%20antibiotic%20resistance%3A%20mechanisms%2C%20origins%2C%20challenges%20and%20solutions&amp;author=G.%20Cox&amp;author=G.D.%20Wright&amp;publication_year=2013&amp;journal=Int.%20J.%20Med.%20Microbiol.&amp;volume=303&amp;pages=287-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijmm.2013.02.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijmm.2013.02.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijmm.2013.02.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23499305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martinez</div>
               <div class="given-names">J.L.</div>
            </div>
          </div>
          <div class="article-title">The role of natural environments in the evolution of resistance traits in pathogenic bacteria</div>
          <div class="source">Proc. Biol. Sci.</div> 
          <div class="year">2009</div>
          <div class="volume">276</div>
          <div class="fpage">2521</div>
          <div class="lpage">30</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20natural%20environments%20in%20the%20evolution%20of%20resistance%20traits%20in%20pathogenic%20bacteria&amp;author=J.L.%20Martinez&amp;publication_year=2009&amp;journal=Proc.%20Biol.%20Sci.&amp;volume=276&amp;pages=2521-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.2009.0320" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2009.0320" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frspb.2009.0320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19364732" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martinez</div>
               <div class="given-names">J.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fajardo</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Garmendia</div>
               <div class="given-names">L.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">A global view of antibiotic resistance</div>
          <div class="source">FEMS Microbiol. Rev.</div> 
          <div class="year">2009</div>
          <div class="volume">33</div>
          <div class="fpage">44</div>
          <div class="lpage">65</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20global%20view%20of%20antibiotic%20resistance&amp;author=J.L.%20Martinez&amp;author=A.%20Fajardo&amp;author=L.%20Garmendia&amp;publication_year=2009&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=33&amp;pages=44-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2008.00142.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2008.00142.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2008.00142.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19054120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morar</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wright</div>
               <div class="given-names">G.D.</div>
            </div>
          </div>
          <div class="article-title">The genomic enzymology of antibiotic resistance</div>
          <div class="source">Ann. Rev. Genet.</div> 
          <div class="year">2010</div>
          <div class="volume">44</div>
          <div class="fpage">25</div>
          <div class="lpage">51</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomic%20enzymology%20of%20antibiotic%20resistance&amp;author=M.%20Morar&amp;author=G.D.%20Wright&amp;publication_year=2010&amp;journal=Ann.%20Rev.%20Genet.&amp;volume=44&amp;pages=25-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-102209-163517" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-102209-163517" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-102209-163517"> </span></div> </div></div></div></div></div><div content-id="DSU032C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yim</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">H.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Davies</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Antibiotics as signalling molecules</div>
          <div class="source">Philos. Trans. R. Soc. Lond. B. Biol. Sci.</div> 
          <div class="year">2007</div>
          <div class="volume">362</div>
          <div class="fpage">1195</div>
          <div class="lpage">200</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antibiotics%20as%20signalling%20molecules&amp;author=G.%20Yim&amp;author=H.H.%20Wang&amp;author=J.%20Davies&amp;publication_year=2007&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B.%20Biol.%20Sci.&amp;volume=362&amp;pages=1195-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2007.2044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2007.2044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2007.2044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17360275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Reynolds</div>
               <div class="given-names">M.G.</div>
            </div>
          </div>
          <div class="article-title">Compensatory evolution in rifampin-resistant Escherichia coli</div>
          <div class="source">Genetics</div> 
          <div class="year">2000</div>
          <div class="volume">156</div>
          <div class="fpage">1471</div>
          <div class="lpage">81</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compensatory%20evolution%20in%20rifampin-resistant%20Escherichia%20coli&amp;author=M.G.%20Reynolds&amp;publication_year=2000&amp;journal=Genetics&amp;volume=156&amp;pages=1471-81" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11102350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Almeida Da Silva</div>
               <div class="given-names">P.E.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Palomino</div>
               <div class="given-names">J.C.</div>
            </div>
          </div>
          <div class="article-title">Molecular basis and mechanisms of drug resistance in <em>Mycobacterium tuberculosis</em>: classical and new drugs</div>
          <div class="source">J. Antimicrob. Chemother.</div> 
          <div class="year">2011</div>
          <div class="volume">66</div>
          <div class="fpage">1417</div>
          <div class="lpage">30</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20basis%20and%20mechanisms%20of%20drug%20resistance%20in%20Mycobacterium%20tuberculosis%3A%20classical%20and%20new%20drugs&amp;author=P.E.A.%20Almeida%20Da%20Silva&amp;author=J.C.%20Palomino&amp;publication_year=2011&amp;journal=J.%20Antimicrob.%20Chemother.&amp;volume=66&amp;pages=1417-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jac/dkr173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjac%2Fdkr173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjac%2Fdkr173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21558086" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kochi</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vareldzis</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Styblo</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="article-title">Multidrug-resistant tuberculosis and its control</div>
          <div class="source">Res. Microbiol.</div> 
          <div class="year">1993</div>
          <div class="volume">144</div>
          <div class="fpage">104</div>
          <div class="lpage">10</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multidrug-resistant%20tuberculosis%20and%20its%20control&amp;author=A.%20Kochi&amp;author=B.%20Vareldzis&amp;author=K.%20Styblo&amp;publication_year=1993&amp;journal=Res.%20Microbiol.&amp;volume=144&amp;pages=104-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0923-2508(93)90023-U" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0923-2508(93)90023-U" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0923-2508(93)90023-U"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8337467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Andersson</div>
               <div class="given-names">D.I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hughes</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="article-title">Antibiotic resistance and its cost: is it possible to reverse resistance?</div>
          <div class="source">Nat. Rev. Microbiol.</div> 
          <div class="year">2010</div>
          <div class="volume">8</div>
          <div class="fpage">260</div>
          <div class="lpage">71</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antibiotic%20resistance%20and%20its%20cost%3A%20is%20it%20possible%20to%20reverse%20resistance%3F&amp;author=D.I.%20Andersson&amp;author=D.%20Hughes&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=8&amp;pages=260-71" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20208551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kotra</div>
               <div class="given-names">L.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Haddad</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mobashery</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">Aminoglycosides: perspectives on mechanisms of action and resistance and strategies to counter resistance</div>
          <div class="source">Antimicrob. Agents Chemother.</div> 
          <div class="year">2000</div>
          <div class="volume">44</div>
          <div class="fpage">3249</div>
          <div class="lpage">56</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aminoglycosides%3A%20perspectives%20on%20mechanisms%20of%20action%20and%20resistance%20and%20strategies%20to%20counter%20resistance&amp;author=L.P.%20Kotra&amp;author=J.%20Haddad&amp;author=S.%20Mobashery&amp;publication_year=2000&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;volume=44&amp;pages=3249-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.44.12.3249-3256.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.44.12.3249-3256.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.44.12.3249-3256.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11083623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Becker</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cooper</div>
               <div class="given-names">M.A.</div>
            </div>
          </div>
          <div class="article-title">Aminoglycoside antibiotics in the 21st century</div>
          <div class="source">ACS Chem. Biol.</div> 
          <div class="year">2013</div>
          <div class="volume">8</div>
          <div class="fpage">105</div>
          <div class="lpage">15</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aminoglycoside%20antibiotics%20in%20the%2021st%20century&amp;author=B.%20Becker&amp;author=M.A.%20Cooper&amp;publication_year=2013&amp;journal=ACS%20Chem.%20Biol.&amp;volume=8&amp;pages=105-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/cb3005116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fcb3005116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fcb3005116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23110460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Davies</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Davis</div>
               <div class="given-names">B.D.</div>
            </div>
          </div>
          <div class="article-title">Misreading of ribonucleic acid code words induced by aminoglycoside antibiotics. The effect of drug concentration</div>
          <div class="source">J. Biol. Chem.</div> 
          <div class="year">1968</div>
          <div class="volume">243</div>
          <div class="fpage">3312</div>
          <div class="lpage">6</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Misreading%20of%20ribonucleic%20acid%20code%20words%20induced%20by%20aminoglycoside%20antibiotics.%20The%20effect%20of%20drug%20concentration&amp;author=J.%20Davies&amp;author=B.D.%20Davis&amp;publication_year=1968&amp;journal=J.%20Biol.%20Chem.&amp;volume=243&amp;pages=3312-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5656371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsai</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Uemura</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Johansson</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">The impact of aminoglycosides on the dynamics of translation elongation</div>
          <div class="source">Cell Rep.</div> 
          <div class="year">2013</div>
          <div class="volume">3</div>
          <div class="fpage">497</div>
          <div class="lpage">508</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20aminoglycosides%20on%20the%20dynamics%20of%20translation%20elongation&amp;author=A.%20Tsai&amp;author=S.%20Uemura&amp;author=M.%20Johansson&amp;publication_year=2013&amp;journal=Cell%20Rep.&amp;volume=3&amp;pages=497-508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2013.01.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2013.01.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2013.01.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23416053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Borovinskaya</div>
               <div class="given-names">M.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pai</div>
               <div class="given-names">R.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">W.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Structural basis for aminoglycoside inhibition of bacterial ribosome recycling</div>
          <div class="source">Nat. Struct. Mol. Biol.</div> 
          <div class="year">2007</div>
          <div class="volume">14</div>
          <div class="fpage">727</div>
          <div class="lpage">32</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20aminoglycoside%20inhibition%20of%20bacterial%20ribosome%20recycling&amp;author=M.A.%20Borovinskaya&amp;author=R.D.%20Pai&amp;author=W.%20Zhang&amp;publication_year=2007&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=14&amp;pages=727-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb1271" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb1271" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb1271"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17660832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Misumi</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nishimura</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Komai</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tanaka</div>
               <div class="given-names">N.</div>
            </div>
          </div>
          <div class="article-title">Interaction of kanamycin and related antibiotics with the large subunit of ribosomes and the inhibition of translocation</div>
          <div class="source">Biochem. Biophys. Res. Commun.</div> 
          <div class="year">1978</div>
          <div class="volume">84</div>
          <div class="fpage">358</div>
          <div class="lpage">65</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interaction%20of%20kanamycin%20and%20related%20antibiotics%20with%20the%20large%20subunit%20of%20ribosomes%20and%20the%20inhibition%20of%20translocation&amp;author=M.%20Misumi&amp;author=T.%20Nishimura&amp;author=T.%20Komai&amp;author=N.%20Tanaka&amp;publication_year=1978&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=84&amp;pages=358-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0006-291X(78)90178-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0006-291X(78)90178-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0006-291X(78)90178-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/363127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cabañas</div>
               <div class="given-names">M.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vázquez</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Modolell</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Inhibition of ribosomal translocation by aminoglycoside antibiotics</div>
          <div class="source">Biochem. Biophys. Res. Commun.</div> 
          <div class="year">1978</div>
          <div class="volume">83</div>
          <div class="fpage">991</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inhibition%20of%20ribosomal%20translocation%20by%20aminoglycoside%20antibiotics&amp;author=M.J.%20Caba%C3%B1as&amp;author=D.%20V%C3%A1zquez&amp;author=J.%20Modolell&amp;publication_year=1978&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=83&amp;pages=991-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0006-291X(78)91493-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0006-291X(78)91493-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0006-291X(78)91493-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/361042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hirokawa</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kiel</div>
               <div class="given-names">M.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Muto</div>
               <div class="given-names">A.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Post-termination complex disassembly by ribosome recycling factor, a functional tRNA mimic</div>
          <div class="source">EMBO J.</div> 
          <div class="year">2002</div>
          <div class="volume">21</div>
          <div class="fpage">2272</div>
          <div class="lpage">81</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Post-termination%20complex%20disassembly%20by%20ribosome%20recycling%20factor%2C%20a%20functional%20tRNA%20mimic&amp;author=G.%20Hirokawa&amp;author=M.C.%20Kiel&amp;author=A.%20Muto&amp;publication_year=2002&amp;journal=EMBO%20J.&amp;volume=21&amp;pages=2272-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/21.9.2272" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F21.9.2272" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F21.9.2272"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11980724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Andersson</div>
               <div class="given-names">D.I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bohman</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Isaksson</div>
               <div class="given-names">L.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kurland</div>
               <div class="given-names">C.G.</div>
            </div>
          </div>
          <div class="article-title">Translation rates and misreading characteristics of rpsD mutants in <em>Escherichia coli</em></div>
          <div class="source">Mol. Gen. Genet.</div> 
          <div class="year">1982</div>
          <div class="volume">187</div>
          <div class="fpage">467</div>
          <div class="lpage">72</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20rates%20and%20misreading%20characteristics%20of%20rpsD%20mutants%20in%20Escherichia%20coli&amp;author=D.I.%20Andersson&amp;author=K.%20Bohman&amp;author=L.A.%20Isaksson&amp;author=C.G.%20Kurland&amp;publication_year=1982&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=187&amp;pages=467-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00332630" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00332630" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00332630"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6757661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kurland</div>
               <div class="given-names">C.G.</div>
            </div>
          </div>
          <div class="article-title">Translational accuracy and the fitness of bacteria</div>
          <div class="source">Ann. Rev. Genet.</div> 
          <div class="year">1992</div>
          <div class="volume">26</div>
          <div class="fpage">29</div>
          <div class="lpage">50</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20accuracy%20and%20the%20fitness%20of%20bacteria&amp;author=C.G.%20Kurland&amp;publication_year=1992&amp;journal=Ann.%20Rev.%20Genet.&amp;volume=26&amp;pages=29-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.26.120192.000333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.26.120192.000333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.26.120192.000333"> </span></div> </div></div></div></div></div><div content-id="DSU032C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">O'Connor</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Göringer</div>
               <div class="given-names">H.U.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dahlberg</div>
               <div class="given-names">A.E.</div>
            </div>
          </div>
          <div class="article-title">A ribosomal ambiguity mutation in the 530 loop of E. coli 16S rRNA</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">1992</div>
          <div class="volume">20</div>
          <div class="fpage">4221</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20ribosomal%20ambiguity%20mutation%20in%20the%20530%20loop%20of%20E.%20coli%2016S%20rRNA&amp;author=M.%20O'Connor&amp;author=H.U.%20G%C3%B6ringer&amp;author=A.E.%20Dahlberg&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res.&amp;volume=20&amp;pages=4221-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.16.4221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.16.4221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.16.4221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1380697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Davis</div>
               <div class="given-names">B.D.</div>
            </div>
          </div>
          <div class="article-title">Mechanism of bactericidal action of aminoglycosides</div>
          <div class="source">Microbiol. Rev.</div> 
          <div class="year">1987</div>
          <div class="volume">51</div>
          <div class="fpage">341</div>
          <div class="lpage">50</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20bactericidal%20action%20of%20aminoglycosides&amp;author=B.D.%20Davis&amp;publication_year=1987&amp;journal=Microbiol.%20Rev.&amp;volume=51&amp;pages=341-50" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3312985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Keren</div>
               <div class="given-names">I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Inocencio</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mulcahy</div>
               <div class="given-names">L.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lewis</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="article-title">Killing by bactericidal antibiotics does not depend on reactive oxygen species</div>
          <div class="source">Science</div> 
          <div class="year">2013</div>
          <div class="volume">339</div>
          <div class="fpage">1213</div>
          <div class="lpage">6</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Killing%20by%20bactericidal%20antibiotics%20does%20not%20depend%20on%20reactive%20oxygen%20species&amp;author=I.%20Keren&amp;author=Y.%20Wu&amp;author=J.%20Inocencio&amp;author=L.R.%20Mulcahy&amp;author=K.%20Lewis&amp;publication_year=2013&amp;journal=Science&amp;volume=339&amp;pages=1213-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1232688" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1232688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1232688"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23471410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kohanski</div>
               <div class="given-names">M.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dwyer</div>
               <div class="given-names">D.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Collins</div>
               <div class="given-names">J.J.</div>
            </div>
          </div>
          <div class="article-title">How antibiotics kill bacteria: from targets to networks</div>
          <div class="source">Nat. Rev. Microbiol.</div> 
          <div class="year">2010</div>
          <div class="volume">8</div>
          <div class="fpage">423</div>
          <div class="lpage">35</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20antibiotics%20kill%20bacteria%3A%20from%20targets%20to%20networks&amp;author=M.A.%20Kohanski&amp;author=D.J.%20Dwyer&amp;author=J.J.%20Collins&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=8&amp;pages=423-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro2333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro2333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro2333"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20440275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Imlay</div>
               <div class="given-names">J.A.</div>
            </div>
          </div>
          <div class="article-title">Cell death from antibiotics without the involvement of reactive oxygen species</div>
          <div class="source">Science</div> 
          <div class="year">2013</div>
          <div class="volume">339</div>
          <div class="fpage">1210</div>
          <div class="lpage">3</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20death%20from%20antibiotics%20without%20the%20involvement%20of%20reactive%20oxygen%20species&amp;author=Y.%20Liu&amp;author=J.A.%20Imlay&amp;publication_year=2013&amp;journal=Science&amp;volume=339&amp;pages=1210-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1232751" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1232751" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1232751"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23471409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shaw</div>
               <div class="given-names">K.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rather</div>
               <div class="given-names">P.N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hare</div>
               <div class="given-names">R.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Miller</div>
               <div class="given-names">G.H.</div>
            </div>
          </div>
          <div class="article-title">Molecular genetics of aminoglycoside resistance genes and familial relationships of the aminoglycoside-modifying enzymes</div>
          <div class="source">Microbiol. Rev.</div> 
          <div class="year">1993</div>
          <div class="volume">57</div>
          <div class="fpage">138</div>
          <div class="lpage">63</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20genetics%20of%20aminoglycoside%20resistance%20genes%20and%20familial%20relationships%20of%20the%20aminoglycoside-modifying%20enzymes&amp;author=K.J.%20Shaw&amp;author=P.N.%20Rather&amp;author=R.S.%20Hare&amp;author=G.H.%20Miller&amp;publication_year=1993&amp;journal=Microbiol.%20Rev.&amp;volume=57&amp;pages=138-63" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8385262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Recht</div>
               <div class="given-names">M.I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Douthwaite</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Puglisi</div>
               <div class="given-names">J.D.</div>
            </div>
          </div>
          <div class="article-title">Basis for prokaryotic specificity of action of aminoglycoside antibiotics</div>
          <div class="source">EMBO J.</div> 
          <div class="year">1999</div>
          <div class="volume">18</div>
          <div class="fpage">3133</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basis%20for%20prokaryotic%20specificity%20of%20action%20of%20aminoglycoside%20antibiotics&amp;author=M.I.%20Recht&amp;author=S.%20Douthwaite&amp;author=J.D.%20Puglisi&amp;publication_year=1999&amp;journal=EMBO%20J.&amp;volume=18&amp;pages=3133-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/18.11.3133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F18.11.3133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F18.11.3133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10357824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Recht</div>
               <div class="given-names">M.I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fourmy</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Blanchard</div>
               <div class="given-names">S.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dahlquist</div>
               <div class="given-names">K.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Puglisi</div>
               <div class="given-names">J.D.</div>
            </div>
          </div>
          <div class="article-title">RNA sequence determinants for aminoglycoside binding to an A-site rRNA model oligonucleotide</div>
          <div class="source">J. Mol. Biol.</div> 
          <div class="year">1996</div>
          <div class="volume">262</div>
          <div class="fpage">421</div>
          <div class="lpage">36</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20sequence%20determinants%20for%20aminoglycoside%20binding%20to%20an%20A-site%20rRNA%20model%20oligonucleotide&amp;author=M.I.%20Recht&amp;author=D.%20Fourmy&amp;author=S.C.%20Blanchard&amp;author=K.D.%20Dahlquist&amp;author=J.D.%20Puglisi&amp;publication_year=1996&amp;journal=J.%20Mol.%20Biol.&amp;volume=262&amp;pages=421-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1996.0526" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0526" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1996.0526"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8893854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Criswell</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tobiason</div>
               <div class="given-names">V.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lodmell</div>
               <div class="given-names">J.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Samuels</div>
               <div class="given-names">D.S.</div>
            </div>
          </div>
          <div class="article-title">Mutations conferring aminoglycoside and spectinomycin resistance in Borrelia burgdorferi</div>
          <div class="source">Antimicrob. Agents Chemother.</div> 
          <div class="year">2006</div>
          <div class="volume">50</div>
          <div class="fpage">445</div>
          <div class="lpage">52</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutations%20conferring%20aminoglycoside%20and%20spectinomycin%20resistance%20in%20Borrelia%20burgdorferi&amp;author=D.%20Criswell&amp;author=V.L.%20Tobiason&amp;author=J.S.%20Lodmell&amp;author=D.S.%20Samuels&amp;publication_year=2006&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;volume=50&amp;pages=445-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.50.2.445-452.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.50.2.445-452.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.50.2.445-452.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16436695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Recht</div>
               <div class="given-names">M.I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Puglisi</div>
               <div class="given-names">J.D.</div>
            </div>
          </div>
          <div class="article-title">Aminoglycoside resistance with homogeneous and heterogeneous populations of antibiotic-resistant ribosomes</div>
          <div class="source">Antimicrob. Agents Chemother.</div> 
          <div class="year">2001</div>
          <div class="volume">45</div>
          <div class="fpage">2414</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aminoglycoside%20resistance%20with%20homogeneous%20and%20heterogeneous%20populations%20of%20antibiotic-resistant%20ribosomes&amp;author=M.I.%20Recht&amp;author=J.D.%20Puglisi&amp;publication_year=2001&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;volume=45&amp;pages=2414-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.45.9.2414-2419.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.45.9.2414-2419.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.45.9.2414-2419.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11502507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hou</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lin</div>
               <div class="given-names">Y.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sharer</div>
               <div class="given-names">J.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">March</div>
               <div class="given-names">P.E.</div>
            </div>
          </div>
          <div class="article-title">In vivo selection of conditional-lethal mutations in the gene encoding elongation factor G of Escherichia coli</div>
          <div class="source">J. Bacteriol.</div> 
          <div class="year">1994</div>
          <div class="volume">176</div>
          <div class="fpage">123</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20selection%20of%20conditional-lethal%20mutations%20in%20the%20gene%20encoding%20elongation%20factor%20G%20of%20Escherichia%20coli&amp;author=Y.%20Hou&amp;author=Y.P.%20Lin&amp;author=J.D.%20Sharer&amp;author=P.E.%20March&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=123-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.176.1.123-129.1994" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.176.1.123-129.1994" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.176.1.123-129.1994"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8282687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lázár</div>
               <div class="given-names">V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pal Singh</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Spohn</div>
               <div class="given-names">R.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Bacterial evolution of antibiotic hypersensitivity</div>
          <div class="source">Mol. Syst. Biol.</div> 
          <div class="year">2013</div>
          <div class="volume">9</div>
          <div class="fpage">700</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20evolution%20of%20antibiotic%20hypersensitivity&amp;author=V.%20L%C3%A1z%C3%A1r&amp;author=G.%20Pal%20Singh&amp;author=R.%20Spohn&amp;publication_year=2013&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=9&amp;pages=700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2013.57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2013.57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2013.57"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24169403" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Galhardo</div>
               <div class="given-names">R.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hastings</div>
               <div class="given-names">P.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rosenberg</div>
               <div class="given-names">S.M.</div>
            </div>
          </div>
          <div class="article-title">Mutation as a stress response and the regulation of evolvability</div>
          <div class="source">Crit. Rev. Biochem. Mol. Biol.</div> 
          <div class="year">2007</div>
          <div class="volume">42</div>
          <div class="fpage">399</div>
          <div class="lpage">435</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20as%20a%20stress%20response%20and%20the%20regulation%20of%20evolvability&amp;author=R.S.%20Galhardo&amp;author=P.J.%20Hastings&amp;author=S.M.%20Rosenberg&amp;publication_year=2007&amp;journal=Crit.%20Rev.%20Biochem.%20Mol.%20Biol.&amp;volume=42&amp;pages=399-435" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10409230701648502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10409230701648502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10409230701648502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17917874" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Beaber</div>
               <div class="given-names">J.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hochhut</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Waldor</div>
               <div class="given-names">M.K.</div>
            </div>
          </div>
          <div class="article-title">SOS response promotes horizontal dissemination of antibiotic resistance genes</div>
          <div class="source">Nature</div> 
          <div class="year">2004</div>
          <div class="volume">427</div>
          <div class="fpage">72</div>
          <div class="lpage">4</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOS%20response%20promotes%20horizontal%20dissemination%20of%20antibiotic%20resistance%20genes&amp;author=J.W.%20Beaber&amp;author=B.%20Hochhut&amp;author=M.K.%20Waldor&amp;publication_year=2004&amp;journal=Nature&amp;volume=427&amp;pages=72-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14688795" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gullberg</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cao</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Berg</div>
               <div class="given-names">O.G.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Selection of resistant bacteria at very low antibiotic concentrations</div>
          <div class="source">PLoS Pathog.</div> 
          <div class="year">2011</div>
          <div class="volume">7</div>
          <div class="fpage">e1002158</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20of%20resistant%20bacteria%20at%20very%20low%20antibiotic%20concentrations&amp;author=E.%20Gullberg&amp;author=S.%20Cao&amp;author=O.G.%20Berg&amp;publication_year=2011&amp;journal=PLoS%20Pathog.&amp;volume=7&amp;pages=e1002158" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1002158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1002158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1002158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21811410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">H.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Molla</div>
               <div class="given-names">M.N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cantor</div>
               <div class="given-names">C.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Collins</div>
               <div class="given-names">J.J.</div>
            </div>
          </div>
          <div class="article-title">Bacterial charity work leads to population-wide resistance</div>
          <div class="source">Nature</div> 
          <div class="year">2010</div>
          <div class="volume">467</div>
          <div class="fpage">82</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20charity%20work%20leads%20to%20population-wide%20resistance&amp;author=H.H.%20Lee&amp;author=M.N.%20Molla&amp;author=C.R.%20Cantor&amp;author=J.J.%20Collins&amp;publication_year=2010&amp;journal=Nature&amp;volume=467&amp;pages=82-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09354" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09354" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09354"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20811456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Toprak</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Veres</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Michel</div>
               <div class="given-names">J.-B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Chait</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hartl</div>
               <div class="given-names">D.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kishony</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">Evolutionary paths to antibiotic resistance under dynamically sustained drug selection</div>
          <div class="source">Nat. Genet.</div> 
          <div class="year">2012</div>
          <div class="volume">44</div>
          <div class="fpage">101</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20paths%20to%20antibiotic%20resistance%20under%20dynamically%20sustained%20drug%20selection&amp;author=E.%20Toprak&amp;author=A.%20Veres&amp;author=J.-B.%20Michel&amp;author=R.%20Chait&amp;author=D.L.%20Hartl&amp;author=R.%20Kishony&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=101-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.1034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.1034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.1034"> </span></div> </div></div></div></div></div><div content-id="DSU032C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Q.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lambert</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Liao</div>
               <div class="given-names">D.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Acceleration of emergence of bacterial antibiotic resistance in connected microenvironments</div>
          <div class="source">Science</div> 
          <div class="year">2011</div>
          <div class="volume">333</div>
          <div class="fpage">1764</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acceleration%20of%20emergence%20of%20bacterial%20antibiotic%20resistance%20in%20connected%20microenvironments&amp;author=Q.%20Zhang&amp;author=G.%20Lambert&amp;author=D.%20Liao&amp;publication_year=2011&amp;journal=Science&amp;volume=333&amp;pages=1764-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1208747" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1208747" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1208747"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21940899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Durbin</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>
          <div class="source">Bioinforma. Oxf. Engl.</div> 
          <div class="year">2009</div>
          <div class="volume">25</div>
          <div class="fpage">1754</div>
          <div class="lpage">60</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinforma.%20Oxf.%20Engl.&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div> </div></div></div></div></div><div content-id="DSU032C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Handsaker</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wysoker</div>
               <div class="given-names">A.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">The Sequence Alignment/Map format and SAMtools</div>
          <div class="source">Bioinforma. Oxf. Engl.</div> 
          <div class="year">2009</div>
          <div class="volume">25</div>
          <div class="fpage">2078</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinforma.%20Oxf.%20Engl.&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div> </div></div></div></div></div><div content-id="DSU032C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Koboldt</div>
               <div class="given-names">D.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wylie</div>
               <div class="given-names">T.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">VarScan: variant detection in massively parallel sequencing of individual and pooled samples</div>
          <div class="source">Bioinforma. Oxf. Engl.</div> 
          <div class="year">2009</div>
          <div class="volume">25</div>
          <div class="fpage">2283</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=VarScan%3A%20variant%20detection%20in%20massively%20parallel%20sequencing%20of%20individual%20and%20pooled%20samples&amp;author=D.C.%20Koboldt&amp;author=K.%20Chen&amp;author=T.%20Wylie&amp;publication_year=2009&amp;journal=Bioinforma.%20Oxf.%20Engl.&amp;volume=25&amp;pages=2283-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp373" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp373" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp373"> </span></div> </div></div></div></div></div><div content-id="DSU032C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Deatherage</div>
               <div class="given-names">D.E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Barrick</div>
               <div class="given-names">J.E.</div>
            </div>
          </div>
          <div class="article-title">Identification of mutations in laboratory-evolved microbes from next-generation sequencing data using breseq</div>
          <div class="source">Methods Mol. Biol.</div> 
          <div class="year">2014</div>
          <div class="volume">1151</div>
          <div class="fpage">165</div>
          <div class="lpage">88</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20mutations%20in%20laboratory-evolved%20microbes%20from%20next-generation%20sequencing%20data%20using%20breseq&amp;author=D.E.%20Deatherage&amp;author=J.E.%20Barrick&amp;publication_year=2014&amp;journal=Methods%20Mol.%20Biol.&amp;volume=1151&amp;pages=165-88" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24838886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Datsenko</div>
               <div class="given-names">K.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wanner</div>
               <div class="given-names">B.L.</div>
            </div>
          </div>
          <div class="article-title">One-step inactivation of chromosomal genes in <em>Escherichia coli</em> K-12 using PCR products</div>
          <div class="source">Proc. Natl Acad. Sci. USA</div> 
          <div class="year">2000</div>
          <div class="volume">97</div>
          <div class="fpage">6640</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=One-step%20inactivation%20of%20chromosomal%20genes%20in%20Escherichia%20coli%20K-12%20using%20PCR%20products&amp;author=K.A.%20Datsenko&amp;author=B.L.%20Wanner&amp;publication_year=2000&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=6640-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.120163297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.120163297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.120163297"> </span></div> </div></div></div></div></div><div content-id="DSU032C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thomason</div>
               <div class="given-names">L.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Costantino</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Court</div>
               <div class="given-names">D.L.</div>
            </div>
          </div>
          <div class="article-title">
            <em>E. coli</em> genome manipulation by P1 transduction</div>
          <div class="source">Curr. Protoc. Mol. Biol. Ed. Frederick M Ausubel Al</div> 
          <div class="year">2007</div>
          <div class="comment">Chapter 1, Unit 1.17.1–1.17.8</div>
        </div></div></div></div><div content-id="DSU032C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cormack</div>
               <div class="given-names">B.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Valdivia</div>
               <div class="given-names">R.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Falkow</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">FACS-optimized mutants of the green fluorescent protein (GFP)</div>
          <div class="source">Gene</div> 
          <div class="year">1996</div>
          <div class="volume">173</div>
          <div class="fpage">33</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FACS-optimized%20mutants%20of%20the%20green%20fluorescent%20protein%20(GFP)&amp;author=B.P.%20Cormack&amp;author=R.H.%20Valdivia&amp;author=S.%20Falkow&amp;publication_year=1996&amp;journal=Gene&amp;volume=173&amp;pages=33-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(95)00685-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(95)00685-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(95)00685-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8707053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guzman</div>
               <div class="given-names">L.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Belin</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Carson</div>
               <div class="given-names">M.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Beckwith</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Tight regulation, modulation, and high-level expression by vectors containing the arabinose PBAD promoter</div>
          <div class="source">J. Bacteriol.</div> 
          <div class="year">1995</div>
          <div class="volume">177</div>
          <div class="fpage">4121</div>
          <div class="lpage">30</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tight%20regulation%2C%20modulation%2C%20and%20high-level%20expression%20by%20vectors%20containing%20the%20arabinose%20PBAD%20promoter&amp;author=L.M.%20Guzman&amp;author=D.%20Belin&amp;author=M.J.%20Carson&amp;author=J.%20Beckwith&amp;publication_year=1995&amp;journal=J.%20Bacteriol.&amp;volume=177&amp;pages=4121-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.177.14.4121-4130.1995" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.177.14.4121-4130.1995" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.177.14.4121-4130.1995"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7608087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zaslaver</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bren</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ronen</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">A comprehensive library of fluorescent transcriptional reporters for Escherichia coli</div>
          <div class="source">Nat. Methods</div> 
          <div class="year">2006</div>
          <div class="volume">3</div>
          <div class="fpage">623</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20library%20of%20fluorescent%20transcriptional%20reporters%20for%20Escherichia%20coli&amp;author=A.%20Zaslaver&amp;author=A.%20Bren&amp;author=M.%20Ronen&amp;publication_year=2006&amp;journal=Nat.%20Methods&amp;volume=3&amp;pages=623-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16862137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mahoney</div>
               <div class="given-names">T.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Silhavy</div>
               <div class="given-names">T.J.</div>
            </div>
          </div>
          <div class="article-title">The Cpx stress response confers resistance to some, but not all, bactericidal antibiotics</div>
          <div class="source">J. Bacteriol.</div> 
          <div class="year">2013</div>
          <div class="volume">195</div>
          <div class="fpage">1869</div>
          <div class="lpage">74</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Cpx%20stress%20response%20confers%20resistance%20to%20some%2C%20but%20not%20all%2C%20bactericidal%20antibiotics&amp;author=T.F.%20Mahoney&amp;author=T.J.%20Silhavy&amp;publication_year=2013&amp;journal=J.%20Bacteriol.&amp;volume=195&amp;pages=1869-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.02197-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.02197-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.02197-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23335416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Baker</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Duy</div>
               <div class="given-names">P.T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nga</div>
               <div class="given-names">T.V.T.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Fitness benefits in fluoroquinolone-resistant Salmonella Typhi in the absence of antimicrobial pressure</div>
          <div class="source">eLife</div> 
          <div class="year">2013</div>
          <div class="volume">2</div>
          <div class="fpage">e01229</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fitness%20benefits%20in%20fluoroquinolone-resistant%20Salmonella%20Typhi%20in%20the%20absence%20of%20antimicrobial%20pressure&amp;author=S.%20Baker&amp;author=P.T.%20Duy&amp;author=T.V.T.%20Nga&amp;publication_year=2013&amp;journal=eLife&amp;volume=2&amp;pages=e01229" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.01229" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.01229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.01229"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24327559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Trabuco</div>
               <div class="given-names">L.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schulten</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Frank</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Molecular dynamics of EF-G during translocation</div>
          <div class="source">Proteins</div> 
          <div class="year">2011</div>
          <div class="volume">79</div>
          <div class="fpage">1478</div>
          <div class="lpage">86</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20dynamics%20of%20EF-G%20during%20translocation&amp;author=W.%20Li&amp;author=L.G.%20Trabuco&amp;author=K.%20Schulten&amp;author=J.%20Frank&amp;publication_year=2011&amp;journal=Proteins&amp;volume=79&amp;pages=1478-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/prot.v79.5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fprot.v79.5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fprot.v79.5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21365677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU032C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU032C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kohanski</div>
               <div class="given-names">M.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dwyer</div>
               <div class="given-names">D.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hayete</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lawrence</div>
               <div class="given-names">C.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Collins</div>
               <div class="given-names">J.J.</div>
            </div>
          </div>
          <div class="article-title">A common mechanism of cellular death induced by bactericidal antibiotics</div>
          <div class="source">Cell</div> 
          <div class="year">2007</div>
          <div class="volume">130</div>
          <div class="fpage">797</div>
          <div class="lpage">810</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20common%20mechanism%20of%20cellular%20death%20induced%20by%20bactericidal%20antibiotics&amp;author=M.A.%20Kohanski&amp;author=D.J.%20Dwyer&amp;author=B.%20Hayete&amp;author=C.A.%20Lawrence&amp;author=J.J.%20Collins&amp;publication_year=2007&amp;journal=Cell&amp;volume=130&amp;pages=797-810" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.06.049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.06.049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.06.049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17803904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754555&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu032/2/dsu032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=pclAW4nFRGX9RHcV8MuWQaz6V73pI3KDZkSiRMoDQTVdnYa3PIQ3bYdU2NGQsPYRO~FtQYHZuVyoYchZwweZAL3dOVUTIFkiYAx-O4ljupjg3VNLMiKREtIDsRwxC8e~C4aYgu3tLUhAkdSfFHR168ydbu-W4FddajqhymwQX6B18DiHII8YX6TB43ssXFbunE5G~K~dy3JsLQTTiBGNBUYVnRgixX91EjXwaQkP337Z6KapT7pOuYq6G0I1qs8uPj~vF0qug3XpC9KhBfs2zfdgBu90LqOxLzANz2Nwle2krpSxZk9LLs~5odpsTwDCND6Yq-pxkvIYeVdsAWfRxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">699</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100012&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu032"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754555"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100012&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f6%2f711%2f2754555">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000347101100012&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F6%2F711%2F2754555" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754555'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genomic Analysis Reveals Distinct Concentration-Dependent Evolutionary Trajectories for Antibiotic Resistance in Escherichia coli" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu032");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f711%2f2754555">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

