

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/2/131/419166","@type":"ScholarlyArticle","name":"Transcriptome Analysis of Silver Carp ( Hypophthalmichthys molitrix ) by Paired-End RNA Sequencing","datePublished":"2012-01-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2012-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr046","keywords":["silver carp","transcriptome","paired-end","RNA sequencing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/2/131/419166","author":[{"name":"Fu, Beide","affiliation":"1  The Key Laboratory of Aquatic Biodiversity and Conservation of Chinese Academy of Sciences ,  Institute of Hydrobiology, Chinese Academy of Sciences ,  Wuhan 430072 ,  PR China    2  Graduate University of the Chinese Academy of Sciences ,  Beijing 100039 ,  PR China","@type":"Person"},{"name":"He, Shunping","affiliation":"1  The Key Laboratory of Aquatic Biodiversity and Conservation of Chinese Academy of Sciences ,  Institute of Hydrobiology, Chinese Academy of Sciences ,  Wuhan 430072 ,  PR China","@type":"Person"}],"description":"Abstract.  The silver carp (Hypophthalmichthys molitrix) is among the most intensively pond-cultured fish species and is used in the wild to counteract water bl","pageStart":"131","pageEnd":"142","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Transcriptome Analysis of Silver Carp ( Hypophthalmichthys molitrix ) by Paired-End RNA Sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Transcriptome Analysis of Silver Carp ( Hypophthalmichthys molitrix ) by Paired-End RNA Sequencing" />
    <meta property="og:description" content="Abstract.  The silver carp (Hypophthalmichthys molitrix) is among the most intensively pond-cultured fish species and is used in the wild to counteract water bl" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/2/131/419166" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-01-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Fu, Beide" /><meta name="citation_author_institution" content="The Key Laboratory of Aquatic Biodiversity and Conservation of Chinese Academy of Sciences, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan 430072, PR China" /><meta name="citation_author_institution" content="Graduate University of the Chinese Academy of Sciences, Beijing 100039, PR China" /><meta name="citation_author" content="He, Shunping" /><meta name="citation_author_institution" content="The Key Laboratory of Aquatic Biodiversity and Conservation of Chinese Academy of Sciences, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan 430072, PR China" /><meta name="citation_title" content="Transcriptome Analysis of Silver Carp (Hypophthalmichthys molitrix) by Paired-End RNA Sequencing" /><meta name="citation_firstpage" content="131" /><meta name="citation_lastpage" content="142" /><meta name="citation_doi" content="10.1093/dnares/dsr046" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2012/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genetics of global gene expression; Nat. Rev.;  citation_year=2006;  citation_volume=7;  citation_pages=862-72; " /><meta name="citation_reference" content="citation_title=Next-generation DNA sequencing; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=1135-45; " /><meta name="citation_reference" content="citation_title=Next-generation DNA sequencing of paired-end tags (PET) for transcriptome and genome analyses; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=521-32; " /><meta name="citation_reference" content="citation_title=Integrative analysis of the melanoma transcriptome; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=413-27; " /><meta name="citation_reference" content="citation_title=Deep RNA sequencing at single base-pair resolution reveals high complexity of the rice transcriptome; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=646-54; " /><meta name="citation_reference" content="citation_title=Function annotation of the rice transcriptome at single-nucleotide resolution by RNA-seq; Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1238-49; " /><meta name="citation_reference" content="citation_title=Massively parallel sequencing of the polyadenylated transcriptome of C. elegans; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=657-66; " /><meta name="citation_reference" content="citation_title=A single-base resolution map of an archaeal transcriptome; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=133-41; " /><meta name="citation_reference" content="citation_title=Stem cell transcriptome profiling via massive-scale mRNA sequencing; Nat Methods;  citation_year=2008;  citation_volume=5;  citation_pages=613-9; " /><meta name="citation_reference" content="citation_title=Highly integrated single-base resolution maps of the epigenome in Arabidopsis; Cell;  citation_year=2008;  citation_volume=133;  citation_pages=523-36; " /><meta name="citation_reference" content="citation_title=Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing; Nat. Genetics;  citation_year=2008;  citation_volume=40;  citation_pages=1413-5; " /><meta name="citation_reference" content="citation_title=A global view of gene activity and alternative splicing by deep sequencing of the human transcriptome; Science;  citation_year=2008;  citation_volume=321;  citation_pages=956-60; " /><meta name="citation_reference" content="citation_title=RNA deep sequencing of the Atlantic cod transcriptome; Comp Biochem Physiol;  citation_year=2010;  citation_volume=6;  citation_pages=18-22; " /><meta name="citation_reference" content="citation_title=Deep sequencing-based transcriptome profiling analysis of bacteria-challenged Lateolabrax japonicus reveals insight into the immune-relevant genes in marine fish; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=472" /><meta name="citation_reference" content="citation_text=John Wiley: Hoboken, N.J.; Fishes of the World;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Whole-genome shotgun assembly and analysis of the genome of Fugu rubripes; Science;  citation_year=2002;  citation_volume=297;  citation_pages=1301-10; " /><meta name="citation_reference" content="citation_title=Genome duplication in the teleost fish Tetraodon nigroviridis reveals the early vertebrate proto-karyotype; Nature;  citation_year=2004;  citation_volume=431;  citation_pages=946-57; " /><meta name="citation_reference" content="citation_title=The medaka draft genome and insights into vertebrate genome evolution; Nature;  citation_year=2007;  citation_volume=447;  citation_pages=714-9; " /><meta name="citation_reference" content="citation_title=The genome sequence of Atlantic cod reveals a unique immune system; Nature;  citation_year=2011;  citation_volume=477;  citation_pages=207-10; " /><meta name="citation_reference" content="citation_title=Sequencing and characterization of the guppy (Poecilia reticulata) transcriptome; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=202" /><meta name="citation_reference" content="citation_title=Sequencing, de novo annotation and analysis of the first Anguilla anguilla transcriptome: EeelBase opens new perspectives for the study of the critically endangered European eel; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=635" /><meta name="citation_reference" content="citation_title=Molecular evidence for the monophyly of East Asian groups of Cyprinidae (Teleostei: Cypriniformes) derived from the nuclear recombination activating gene 2 sequences; Mol. Phylogenet. Evol.;  citation_year=2007;  citation_volume=42;  citation_pages=157-70; " /><meta name="citation_reference" content="citation_author=Bureau of Fishery; citation_publisher=China Agriculture Press, Beijing; China Fishery Statistical Yearbook;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Practical success of biomanipulation using filter-feeding Fish to control cyanobacteria blooms: a synthesis of decades of research and application in a subtropical hypereutrophic lake; Scientific World J.;  citation_year=2001;  citation_volume=1;  citation_pages=337-56; " /><meta name="citation_reference" content="citation_title=Large enclosures experimental study on algal control by silver carp and bighead; Chinese Environ. Sci.;  citation_year=2009;  citation_volume=29;  citation_pages=1190-5; " /><meta name="citation_reference" content="citation_title=Phylogenomic analysis resolves the formerly intractable adaptive diversification of the endemic clade of east Asian Cyprinidae (Cypriniformes); PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e13508" /><meta name="citation_reference" content="citation_title=Optimization of de novo transcriptome assembly from next-generation sequencing data; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1432-40; " /><meta name="citation_reference" content="citation_title=ABySS: a parallel assembler for short read sequence data; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1117-23; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1658-9; " /><meta name="citation_reference" content="citation_title=Ensembl 2011; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=D800-6; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=High-throughput functional annotation and data mining with the Blast2GO suite; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=3420-35; " /><meta name="citation_reference" content="citation_title=Exploiting EST databases for the development and characterization of gene-derived SSR-markers in barley (Hordeum vulgare L.); Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=411-22; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=SOAP: short oligonucleotide alignment program; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=713-4; " /><meta name="citation_reference" content="citation_title=In situ study on the control of toxic microcystis blooms using phytoplanktivorous fish in the subtropical Lake Taihu of China: a large fish pen experiment; Aquaculture;  citation_year=2007;  citation_volume=265;  citation_pages=127-38; " /><meta name="citation_reference" content="citation_title=In situ studies on the bioaccumulation of microcystins in the phytoplanktivorous silver carp (Hypophthalmichthys molitrix) stocked in Lake Taihu with dense toxic microcystis blooms; Aquaculture;  citation_year=2006;  citation_volume=261;  citation_pages=1026-38; " /><meta name="citation_reference" content="citation_title=Sequential ultrastructural and biochemical changes induced in vivo by the hepatotoxic microcystins in liver of the phytoplanktivorous silver carp Hypophthalmichthys molitrix; Comp. Biochem. Physiol. C Toxicol. Pharmacol.;  citation_year=2007;  citation_volume=146;  citation_pages=357-67; " /><meta name="citation_reference" content="citation_title=Benchmarking next-generation transcriptome sequencing for functional and evolutionary genomics; Mol. Biol. Evol.;  citation_year=2009;  citation_volume=26;  citation_pages=2731-44; " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=Assembling millions of short DNA sequences using SSAKE; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=500-1; " /><meta name="citation_reference" content="citation_title=Next-generation sequencing transforms today's biology; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=16-8; " /><meta name="citation_reference" content="citation_title=Analysis of muscle and ovary transcriptome of Sus scrofa: assembly, annotation and marker discovery; DNA Res;  citation_year=2011;  citation_volume=18;  citation_pages=343-51; " /><meta name="citation_reference" content="citation_title=Characterization of the rainbow trout transcriptome using Sanger and 454-pyrosequencing approaches; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=564" /><meta name="citation_reference" content="citation_title=Identification of an enzymatically formed glutathione conjugate of the cyanobacterial hepatotoxin microcystin-LR: the first step of detoxication; Biochim. Biophys. Acta;  citation_year=1998;  citation_volume=1425;  citation_pages=527-33; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map for silver carp (Hypophthalmichthys molitrix); Anim. Genet;  citation_year=2010;  citation_volume=41;  citation_pages=523-30; " /><meta name="citation_reference" content="citation_title=Development of silver carp (Hypophthalmichthys molitrix) and bighead carp (Aristichthys nobilis) genetic maps using microsatellite and AFLP markers and a pseudo-testcross strategy; Anim. Genet.;  citation_year=2007;  citation_volume=38;  citation_pages=364-70; " /><meta name="citation_reference" content="citation_title=Defining the transcriptome assembly and its use for genome dynamics and transcriptome profiling studies in pigeonpea (Cajanus cajan L.); DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=153-64; " /><meta name="citation_reference" content="citation_title=Development of genic-SSR markers by deep transcriptome sequencing in pigeonpea [Cajanus cajan (L.) Millspaugh]; BMC Plant Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=17" /><meta name="citation_reference" content="citation_title=QTL mapping of ten agronomic traits on the soybean (Glycine max L. Merr.) genetic map and their association with EST markers; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=1131-9; " /><meta name="citation_reference" content="citation_title=The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics; PLoS Biol.;  citation_year=2003;  citation_volume=1;  citation_pages=E45" /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/2/131/1162479/dsr046.pdf" /><meta name="description" content="Abstract.  The silver carp (Hypophthalmichthys molitrix) is among the most intensively pond-cultured fish species and is used in the wild to counteract water bl" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/2/131/419166" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/2/131/419166" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/2/131/419166");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr046';
    </script>
<title>
	Transcriptome Analysis of Silver Carp (Hypophthalmichthys molitrix) by Paired-End RNA Sequencing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./419166" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbBuPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdSwgQmVpZGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgS2V5IExhYm9yYXRvcnkgb2YgQXF1YXRpYyBCaW9kaXZlcnNpdHkgYW5kIENvbnNlcnZhdGlvbiBvZiBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEluc3RpdHV0ZSBvZiBIeWRyb2Jpb2xvZ3ksIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgV3VoYW4gNDMwMDcyLCBQUiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFVuaXZlcnNpdHkgb2YgdGhlIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgQmVpamluZyAxMDAwMzksIFBSIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGUsIFNodW5waW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGhlIEtleSBMYWJvcmF0b3J5IG9mIEFxdWF0aWMgQmlvZGl2ZXJzaXR5IGFuZCBDb25zZXJ2YXRpb24gb2YgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBJbnN0aXR1dGUgb2YgSHlkcm9iaW9sb2d5LCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIFd1aGFuIDQzMDA3MiwgUFIgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlRyYW5zY3JpcHRvbWUgQW5hbHlzaXMgb2YgU2lsdmVyIENhcnAgKEh5cG9waHRoYWxtaWNodGh5cyBtb2xpdHJpeCkgYnkgUGFpcmVkLUVuZCBSTkEgU2VxdWVuY2luZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNyMDQ2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljcyBvZiBnbG9iYWwgZ2VuZSBleHByZXNzaW9uOyBOYXQuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTg2Mi03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTEzNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nIG9mIHBhaXJlZC1lbmQgdGFncyAoUEVUKSBmb3IgdHJhbnNjcmlwdG9tZSBhbmQgZ2Vub21lIGFuYWx5c2VzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTUyMS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgYW5hbHlzaXMgb2YgdGhlIG1lbGFub21hIHRyYW5zY3JpcHRvbWU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDEzLTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWVwIFJOQSBzZXF1ZW5jaW5nIGF0IHNpbmdsZSBiYXNlLXBhaXIgcmVzb2x1dGlvbiByZXZlYWxzIGhpZ2ggY29tcGxleGl0eSBvZiB0aGUgcmljZSB0cmFuc2NyaXB0b21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTY0Ni01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb24gYW5ub3RhdGlvbiBvZiB0aGUgcmljZSB0cmFuc2NyaXB0b21lIGF0IHNpbmdsZS1udWNsZW90aWRlIHJlc29sdXRpb24gYnkgUk5BLXNlcTsgR2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEyMzgtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hc3NpdmVseSBwYXJhbGxlbCBzZXF1ZW5jaW5nIG9mIHRoZSBwb2x5YWRlbnlsYXRlZCB0cmFuc2NyaXB0b21lIG9mIEMuIGVsZWdhbnM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NjU3LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbmdsZS1iYXNlIHJlc29sdXRpb24gbWFwIG9mIGFuIGFyY2hhZWFsIHRyYW5zY3JpcHRvbWU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGVtIGNlbGwgdHJhbnNjcmlwdG9tZSBwcm9maWxpbmcgdmlhIG1hc3NpdmUtc2NhbGUgbVJOQSBzZXF1ZW5jaW5nOyBOYXQgTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NjEzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseSBpbnRlZ3JhdGVkIHNpbmdsZS1iYXNlIHJlc29sdXRpb24gbWFwcyBvZiB0aGUgZXBpZ2Vub21lIGluIEFyYWJpZG9wc2lzOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTUyMy0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVlcCBzdXJ2ZXlpbmcgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgY29tcGxleGl0eSBpbiB0aGUgaHVtYW4gdHJhbnNjcmlwdG9tZSBieSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZzsgTmF0LiBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTE0MTMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnbG9iYWwgdmlldyBvZiBnZW5lIGFjdGl2aXR5IGFuZCBhbHRlcm5hdGl2ZSBzcGxpY2luZyBieSBkZWVwIHNlcXVlbmNpbmcgb2YgdGhlIGh1bWFuIHRyYW5zY3JpcHRvbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzIxOyAgY2l0YXRpb25fcGFnZXM9OTU2LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgZGVlcCBzZXF1ZW5jaW5nIG9mIHRoZSBBdGxhbnRpYyBjb2QgdHJhbnNjcmlwdG9tZTsgQ29tcCBCaW9jaGVtIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE4LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWVwIHNlcXVlbmNpbmctYmFzZWQgdHJhbnNjcmlwdG9tZSBwcm9maWxpbmcgYW5hbHlzaXMgb2YgYmFjdGVyaWEtY2hhbGxlbmdlZCBMYXRlb2xhYnJheCBqYXBvbmljdXMgcmV2ZWFscyBpbnNpZ2h0IGludG8gdGhlIGltbXVuZS1yZWxldmFudCBnZW5lcyBpbiBtYXJpbmUgZmlzaDsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGV4dD1Kb2huIFdpbGV5OiBIb2Jva2VuLCBOLkouOyBGaXNoZXMgb2YgdGhlIFdvcmxkOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2hvdGd1biBhc3NlbWJseSBhbmQgYW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBvZiBGdWd1IHJ1YnJpcGVzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI5NzsgIGNpdGF0aW9uX3BhZ2VzPTEzMDEtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBkdXBsaWNhdGlvbiBpbiB0aGUgdGVsZW9zdCBmaXNoIFRldHJhb2RvbiBuaWdyb3ZpcmlkaXMgcmV2ZWFscyB0aGUgZWFybHkgdmVydGVicmF0ZSBwcm90by1rYXJ5b3R5cGU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00MzE7ICBjaXRhdGlvbl9wYWdlcz05NDYtNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtZWRha2EgZHJhZnQgZ2Vub21lIGFuZCBpbnNpZ2h0cyBpbnRvIHZlcnRlYnJhdGUgZ2Vub21lIGV2b2x1dGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0NzsgIGNpdGF0aW9uX3BhZ2VzPTcxNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlIG9mIEF0bGFudGljIGNvZCByZXZlYWxzIGEgdW5pcXVlIGltbXVuZSBzeXN0ZW07IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00Nzc7ICBjaXRhdGlvbl9wYWdlcz0yMDctMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIGd1cHB5IChQb2VjaWxpYSByZXRpY3VsYXRhKSB0cmFuc2NyaXB0b21lOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nLCBkZSBub3ZvIGFubm90YXRpb24gYW5kIGFuYWx5c2lzIG9mIHRoZSBmaXJzdCBBbmd1aWxsYSBhbmd1aWxsYSB0cmFuc2NyaXB0b21lOiBFZWVsQmFzZSBvcGVucyBuZXcgcGVyc3BlY3RpdmVzIGZvciB0aGUgc3R1ZHkgb2YgdGhlIGNyaXRpY2FsbHkgZW5kYW5nZXJlZCBFdXJvcGVhbiBlZWw7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTYzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBldmlkZW5jZSBmb3IgdGhlIG1vbm9waHlseSBvZiBFYXN0IEFzaWFuIGdyb3VwcyBvZiBDeXByaW5pZGFlIChUZWxlb3N0ZWk6IEN5cHJpbmlmb3JtZXMpIGRlcml2ZWQgZnJvbSB0aGUgbnVjbGVhciByZWNvbWJpbmF0aW9uIGFjdGl2YXRpbmcgZ2VuZSAyIHNlcXVlbmNlczsgTW9sLiBQaHlsb2dlbmV0LiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTE1Ny03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPUJ1cmVhdSBvZiBGaXNoZXJ5OyBjaXRhdGlvbl9wdWJsaXNoZXI9Q2hpbmEgQWdyaWN1bHR1cmUgUHJlc3MsIEJlaWppbmc7IENoaW5hIEZpc2hlcnkgU3RhdGlzdGljYWwgWWVhcmJvb2s7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByYWN0aWNhbCBzdWNjZXNzIG9mIGJpb21hbmlwdWxhdGlvbiB1c2luZyBmaWx0ZXItZmVlZGluZyBGaXNoIHRvIGNvbnRyb2wgY3lhbm9iYWN0ZXJpYSBibG9vbXM6IGEgc3ludGhlc2lzIG9mIGRlY2FkZXMgb2YgcmVzZWFyY2ggYW5kIGFwcGxpY2F0aW9uIGluIGEgc3VidHJvcGljYWwgaHlwZXJldXRyb3BoaWMgbGFrZTsgU2NpZW50aWZpYyBXb3JsZCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MzM3LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZSBlbmNsb3N1cmVzIGV4cGVyaW1lbnRhbCBzdHVkeSBvbiBhbGdhbCBjb250cm9sIGJ5IHNpbHZlciBjYXJwIGFuZCBiaWdoZWFkOyBDaGluZXNlIEVudmlyb24uIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xMTkwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2Vub21pYyBhbmFseXNpcyByZXNvbHZlcyB0aGUgZm9ybWVybHkgaW50cmFjdGFibGUgYWRhcHRpdmUgZGl2ZXJzaWZpY2F0aW9uIG9mIHRoZSBlbmRlbWljIGNsYWRlIG9mIGVhc3QgQXNpYW4gQ3lwcmluaWRhZSAoQ3lwcmluaWZvcm1lcyk7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTM1MDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWl6YXRpb24gb2YgZGUgbm92byB0cmFuc2NyaXB0b21lIGFzc2VtYmx5IGZyb20gbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgZGF0YTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNDMyLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BQnlTUzogYSBwYXJhbGxlbCBhc3NlbWJsZXIgZm9yIHNob3J0IHJlYWQgc2VxdWVuY2UgZGF0YTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMTE3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TY2FmZm9sZGluZyBwcmUtYXNzZW1ibGVkIGNvbnRpZ3MgdXNpbmcgU1NQQUNFOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZC1oaXQ6IGEgZmFzdCBwcm9ncmFtIGZvciBjbHVzdGVyaW5nIGFuZCBjb21wYXJpbmcgbGFyZ2Ugc2V0cyBvZiBwcm90ZWluIG9yIG51Y2xlb3RpZGUgc2VxdWVuY2VzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE2NTgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5zZW1ibCAyMDExOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1EODAwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IGZ1bmN0aW9uYWwgYW5ub3RhdGlvbiBhbmQgZGF0YSBtaW5pbmcgd2l0aCB0aGUgQmxhc3QyR08gc3VpdGU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTM0MjAtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvaXRpbmcgRVNUIGRhdGFiYXNlcyBmb3IgdGhlIGRldmVsb3BtZW50IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGdlbmUtZGVyaXZlZCBTU1ItbWFya2VycyBpbiBiYXJsZXkgKEhvcmRldW0gdnVsZ2FyZSBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9NDExLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQU1MIDQ6IHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBieSBtYXhpbXVtIGxpa2VsaWhvb2Q7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTg2LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQOiBzaG9ydCBvbGlnb251Y2xlb3RpZGUgYWxpZ25tZW50IHByb2dyYW07IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NzEzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHNpdHUgc3R1ZHkgb24gdGhlIGNvbnRyb2wgb2YgdG94aWMgbWljcm9jeXN0aXMgYmxvb21zIHVzaW5nIHBoeXRvcGxhbmt0aXZvcm91cyBmaXNoIGluIHRoZSBzdWJ0cm9waWNhbCBMYWtlIFRhaWh1IG9mIENoaW5hOiBhIGxhcmdlIGZpc2ggcGVuIGV4cGVyaW1lbnQ7IEFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI2NTsgIGNpdGF0aW9uX3BhZ2VzPTEyNy0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gc2l0dSBzdHVkaWVzIG9uIHRoZSBiaW9hY2N1bXVsYXRpb24gb2YgbWljcm9jeXN0aW5zIGluIHRoZSBwaHl0b3BsYW5rdGl2b3JvdXMgc2lsdmVyIGNhcnAgKEh5cG9waHRoYWxtaWNodGh5cyBtb2xpdHJpeCkgc3RvY2tlZCBpbiBMYWtlIFRhaWh1IHdpdGggZGVuc2UgdG94aWMgbWljcm9jeXN0aXMgYmxvb21zOyBBcXVhY3VsdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjE7ICBjaXRhdGlvbl9wYWdlcz0xMDI2LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW50aWFsIHVsdHJhc3RydWN0dXJhbCBhbmQgYmlvY2hlbWljYWwgY2hhbmdlcyBpbmR1Y2VkIGluIHZpdm8gYnkgdGhlIGhlcGF0b3RveGljIG1pY3JvY3lzdGlucyBpbiBsaXZlciBvZiB0aGUgcGh5dG9wbGFua3Rpdm9yb3VzIHNpbHZlciBjYXJwIEh5cG9waHRoYWxtaWNodGh5cyBtb2xpdHJpeDsgQ29tcC4gQmlvY2hlbS4gUGh5c2lvbC4gQyBUb3hpY29sLiBQaGFybWFjb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0NjsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmVuY2htYXJraW5nIG5leHQtZ2VuZXJhdGlvbiB0cmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgZm9yIGZ1bmN0aW9uYWwgYW5kIGV2b2x1dGlvbmFyeSBnZW5vbWljczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTI3MzEtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlbHZldDogYWxnb3JpdGhtcyBmb3IgZGUgbm92byBzaG9ydCByZWFkIGFzc2VtYmx5IHVzaW5nIGRlIEJydWlqbiBncmFwaHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9ODIxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2VtYmxpbmcgbWlsbGlvbnMgb2Ygc2hvcnQgRE5BIHNlcXVlbmNlcyB1c2luZyBTU0FLRTsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz01MDAtMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdHJhbnNmb3JtcyB0b2RheSdzIGJpb2xvZ3k7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgbXVzY2xlIGFuZCBvdmFyeSB0cmFuc2NyaXB0b21lIG9mIFN1cyBzY3JvZmE6IGFzc2VtYmx5LCBhbm5vdGF0aW9uIGFuZCBtYXJrZXIgZGlzY292ZXJ5OyBETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzQzLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSByYWluYm93IHRyb3V0IHRyYW5zY3JpcHRvbWUgdXNpbmcgU2FuZ2VyIGFuZCA0NTQtcHlyb3NlcXVlbmNpbmcgYXBwcm9hY2hlczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTY0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYW4gZW56eW1hdGljYWxseSBmb3JtZWQgZ2x1dGF0aGlvbmUgY29uanVnYXRlIG9mIHRoZSBjeWFub2JhY3RlcmlhbCBoZXBhdG90b3hpbiBtaWNyb2N5c3Rpbi1MUjogdGhlIGZpcnN0IHN0ZXAgb2YgZGV0b3hpY2F0aW9uOyBCaW9jaGltLiBCaW9waHlzLiBBY3RhOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE0MjU7ICBjaXRhdGlvbl9wYWdlcz01MjctMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGdlbmV0aWMgbGlua2FnZSBtYXAgZm9yIHNpbHZlciBjYXJwIChIeXBvcGh0aGFsbWljaHRoeXMgbW9saXRyaXgpOyBBbmltLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTUyMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2Ygc2lsdmVyIGNhcnAgKEh5cG9waHRoYWxtaWNodGh5cyBtb2xpdHJpeCkgYW5kIGJpZ2hlYWQgY2FycCAoQXJpc3RpY2h0aHlzIG5vYmlsaXMpIGdlbmV0aWMgbWFwcyB1c2luZyBtaWNyb3NhdGVsbGl0ZSBhbmQgQUZMUCBtYXJrZXJzIGFuZCBhIHBzZXVkby10ZXN0Y3Jvc3Mgc3RyYXRlZ3k7IEFuaW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTM2NC03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVmaW5pbmcgdGhlIHRyYW5zY3JpcHRvbWUgYXNzZW1ibHkgYW5kIGl0cyB1c2UgZm9yIGdlbm9tZSBkeW5hbWljcyBhbmQgdHJhbnNjcmlwdG9tZSBwcm9maWxpbmcgc3R1ZGllcyBpbiBwaWdlb25wZWEgKENhamFudXMgY2FqYW4gTC4pOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE1My02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgZ2VuaWMtU1NSIG1hcmtlcnMgYnkgZGVlcCB0cmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgaW4gcGlnZW9ucGVhIFtDYWphbnVzIGNhamFuIChMLikgTWlsbHNwYXVnaF07IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMIG1hcHBpbmcgb2YgdGVuIGFncm9ub21pYyB0cmFpdHMgb24gdGhlIHNveWJlYW4gKEdseWNpbmUgbWF4IEwuIE1lcnIuKSBnZW5ldGljIG1hcCBhbmQgdGhlaXIgYXNzb2NpYXRpb24gd2l0aCBFU1QgbWFya2VyczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xMTMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgQ2Flbm9yaGFiZGl0aXMgYnJpZ2dzYWU6IGEgcGxhdGZvcm0gZm9yIGNvbXBhcmF0aXZlIGdlbm9taWNzOyBQTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz1FNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGFuYWx5c2lzIG9mIHRoZSBodW1hbiBnZW5vbWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTkvMi8xMzEvMTE2MjQ3OS9kc3IwNDYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBzaWx2ZXIgY2FycCAoSHlwb3BodGhhbG1pY2h0aHlzIG1vbGl0cml4KSBpcyBhbW9uZyB0aGUgbW9zdCBpbnRlbnNpdmVseSBwb25kLWN1bHR1cmVkIGZpc2ggc3BlY2llcyBhbmQgaXMgdXNlZCBpbiB0aGUgd2lsZCB0byBjb3VudGVyYWN0IHdhdGVyIGJsIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00MTkxNjYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk5UFWwBinKQr6BOONT8KKdZuCx7c=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f131%2f419166" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f131%2f419166" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16354" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&amp;Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/2">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6206181">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6206181">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6206183">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6206183">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6206187">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6206187">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6206203">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6206203">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6206253">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6206253">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6206262">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6206262">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6206264">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6206264">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6206266">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6206266">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/2/117/421175">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/2/143/419384">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Transcriptome Analysis of Silver Carp (<em>Hypophthalmichthys molitrix</em>) by Paired-End RNA Sequencing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Beide Fu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Key Laboratory of Aquatic Biodiversity and Conservation of Chinese Academy of Sciences</div>, <div class="insititution">Institute of Hydrobiology, Chinese Academy of Sciences</div>, <div class="addr-line">Wuhan 430072</div>, <div class="country">PR China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Graduate University of the Chinese Academy of Sciences</div>, <div class="addr-line">Beijing 100039</div>, <div class="country">PR China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Beide+Fu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fu B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fu B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Beide Fu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shunping He</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Key Laboratory of Aquatic Biodiversity and Conservation of Chinese Academy of Sciences</div>, <div class="insititution">Institute of Hydrobiology, Chinese Academy of Sciences</div>, <div class="addr-line">Wuhan 430072</div>, <div class="country">PR China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 27-68780430. Fax. +86 27-68780430. E-mail: <a href="mailto:clad@ihb.ac.cn" target="_blank">clad@ihb.ac.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shunping+He">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=He S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;He S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shunping He</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Masahira Hattori</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 131–142, <a href='https://doi.org/10.1093/dnares/dsr046'>https://doi.org/10.1093/dnares/dsr046</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 January 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 October 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 December 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="419166" href="/dnaresearch/article-pdf/19/2/131/1162479/dsr046.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Beide Fu, Shunping He;  Transcriptome Analysis of Silver Carp (<em>Hypophthalmichthys molitrix</em>) by Paired-End RNA Sequencing, <em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 131–142, <a href='https://doi.org/10.1093/dnares/dsr046'>https://doi.org/10.1093/dnares/dsr046</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/419166?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/419166?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/419166?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/419166?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/419166?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/419166?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/419166?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Transcriptome%20Analysis%20of%20Silver%20Carp%20(%26lt%3Bem%26gt%3BHypophthalmichthys%20molitrix%26lt%3B%2Fem%26gt%3B)%20by%20Paired-End%20RNA%20Sequencing&amp;publicationDate=2012-01-24&amp;volumeNum=19&amp;issueNum=2&amp;author=Fu%2C%20Beide%3B%20He%2C%20Shunping&amp;startPage=131&amp;endPage=142&amp;contentId=10.1093%2Fdnares%2Fdsr046&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f131%2f419166" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6206181 id="6206181" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The silver carp (<em>Hypophthalmichthys molitrix</em>) is among the most intensively pond-cultured fish species and is used in the wild to counteract water bloom in China. However, little genomic information is available for this species, especially regarding its ability to grow rapidly in water, even water contaminated with high concentrations of poisonous microcystin. In this study, we performed <em>de novo</em> transcriptome assembly and analysis of the 17.10 million short-read sequences produced by the Illumina paired-end sequencing technology. Using an improved multiple k-mer contig assembly method coupled with further scaffolding, 85 759 sequences were obtained. There were 23 044 sequences annotated with 3423 gene ontology terms for 104 196 term occurrences and the three corresponding organizing principles. A total of 38 200 assembled sequences were involved in 218 predicted Kyoto Encyclopedia of Genes and Genomes metabolic pathways. We also recovered 41 of 44 genes involved in the biosynthesis of glutathione. Of these, five genes were identified as experienced positive selection between silver carp and zebrafish, as determined by the likelihood ratio test. This report is the first annotated review of the silver carp transcriptome. These data will be of interest to researchers investigating the evolution and biological processes of the silver carp. This work also provides an archive for future studies of recent speciation and evolution of Cyprinidae fishes and can be used in comparative studies of other fishes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;silver carp&quot;">silver carp</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="paired-end">paired-end</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RNA sequencing&quot;">RNA sequencing</a></div>                            <h2 scrollto-destination=6206183 id="6206183" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The transcriptome is made up of the subset of genes active in a selected tissue and species. Understanding the dynamics of the transcriptome is essential for interpreting phenotypic variation caused by combinations of genotypic and environmental factors.<sup><a reveal-id="DSR046C1" data-open="DSR046C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Massively parallel sequencing of RNA<sup><a reveal-id="DSR046C2" data-open="DSR046C2" class="link link-ref link-reveal xref-bibr">2</a></sup> (RNA-Seq) has offered the opportunity to characterize the transcriptome with unprecedented sensitivity and depth. It has already revolutionized the way we study the transcriptome. The latest paired-end sequencing of RNA-Seq techniques have further improved the efficiency of DNA sequencing and expanded short read lengths, permitting a deeper understanding of the transcriptome.<sup><a reveal-id="DSR046C3" data-open="DSR046C3" class="link link-ref link-reveal xref-bibr">3</a></sup> RNA-Seq is independent of prior knowledge and does not require design work, thus reducing the required staff, cost and time and providing the unprecedented opportunity to conduct low-cost transcriptome studies at lower cost for non-model organisms. The RNA-Seq technology has been applied to many model organisms<sup><a reveal-id="DSR046C4  DSR046C12" data-open="DSR046C4  DSR046C12" class="link link-ref link-reveal xref-bibr">4–12</a></sup> for the discovery of splice variants, RNA editing sites and new microRNAs, but fewer studies were conducted in non-model fish organisms.<sup><a reveal-id="DSR046C13" data-open="DSR046C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR046C14" data-open="DSR046C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>The Actinopterygii, in terms of numbers, are the dominant class of vertebrate, comprising nearly 96% of the 26 000 species of fish.<sup><a reveal-id="DSR046C15" data-open="DSR046C15" class="link link-ref link-reveal xref-bibr">15</a></sup> However, the genomic information of this group is very rare: only six genomes<sup><a reveal-id="DSR046C16  DSR046C19" data-open="DSR046C16  DSR046C19" class="link link-ref link-reveal xref-bibr">16–19</a></sup> and several transcriptomes<sup><a reveal-id="DSR046C20" data-open="DSR046C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSR046C21" data-open="DSR046C21" class="link link-ref link-reveal xref-bibr">21</a></sup> are available. This has hindered research into these valuable species. Cyprinidae is the largest family of freshwater fish in the Actinopterygii.<sup><a reveal-id="DSR046C15" data-open="DSR046C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The endemic clade of East Asian Cyprinidae displays a tremendous diversity of phenotypic and ecological traits in this area. This clade is an ideal model system for the study of rapid radiations and evolutionary adaptation over short periods of time.<sup><a reveal-id="DSR046C22" data-open="DSR046C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Silver carp (<em>Hypophthalmichthys molitrix</em>) of the family Cyprinidae are among the most intensively pond-cultured species in China. As one of famous four major Chinese carps, breeding production reached 3 million tons in China in 2009.<sup><a reveal-id="DSR046C23" data-open="DSR046C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Aside from their great importance to the fishery economy, silver carp have been found useful in counteracting cyanobacteria blooms in China.<sup><a reveal-id="DSR046C24" data-open="DSR046C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR046C25" data-open="DSR046C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Silver carp are also a good model for the study of speciation because of its split with bighead carp (<em>Hypophthalmichthys nobilis</em>), which occurred only ∼3 Mya.<sup><a reveal-id="DSR046C26" data-open="DSR046C26" class="link link-ref link-reveal xref-bibr">26</a></sup> However, lack of genomic resources like genome sequence, transcriptome sequences and molecular markers has made the study of silver carp breeding, the mechanism of its ability to counteract water bloom and evolutionary analysis a difficult task.</p><p>When no genome sequence is available, transcriptome sequencing is an effective way to obtain large numbers of molecular makers and identify transcripts involved in specific biological processes. In this study, we present the first silver carp transcriptome using massively parallel mRNA sequencing. We perform Illumina sequencing of the heart, liver, brain, spleen and kidney tissues to characterize the <em>H. molitrix</em> transcriptome. A database (Silver Carp Base) is under construction and we expect that it will provide the first picture of the transcriptome of this species. The database will be updated in the future if additional data become available.</p>                            <h2 scrollto-destination=6206187 id="6206187" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6206188 id="6206188" class="section-title" data-legacy-id=s2a>2.1. Ethics statement</h3>
<p>All experimental protocols were approved by the ethics committee of Institute of HydroBiology, Chinese Academy of Sciences.</p>                            <h3 scrollto-destination=6206190 id="6206190" class="section-title" data-legacy-id=s2b>2.2. Organ collection and RNA isolation</h3>
<p>A wild silver carp was collected from the middle reach of the Yangtze River. To obtain the whole transcriptome, RNA from five organs (heart, liver, brain, spleen and kidneys) was extracted using TRIzol reagent (Invitrogen, Carlsbad, CA, USA). After the quality examination by the way of electrophoresis and a BioPhotometer plus 6132 (Eppendorf, Germany), RNAs from different organs were mixed together at equivalent concentrations. Total RNA extraction was in accordance with the manufacturer's protocol and it was treated with RNase-free DNase I (New England Biolabs) for 30 min at 37°C to remove the residual DNA.</p>                            <h3 scrollto-destination=6206192 id="6206192" class="section-title" data-legacy-id=s2c>2.3. cDNA library preparation and sequencing</h3>
<p>Beads with oligo(dT) were used to purify poly(A) mRNA from total RNA. Then, the mRNA was fragmented using a RNA fragmentation kit (Ambion). First-strand cDNA was synthesized using random hexamer-primer and reverse transcriptase (Invitrogen), and second-strand cDNA was synthesized next. Then the paired-end cDNA library was prepared in accordance with Illumina's protocols with an insert size of 200 bp and sequenced for 75 bp. The Illumina GA processing pipeline v0.2.2.6 was used to analyze the image and for base calling.</p>                            <h3 scrollto-destination=6206194 id="6206194" class="section-title" data-legacy-id=s2d>2.4. De novo assembly of silver carp transcriptome</h3>
<p>As no optimal k-mer length is appropriate for all de novo transcriptome assemblies, the multiple k-mer method was used to obtain longer silver carp mRNA sequences, which are very useful in subsequent analysis steps. Our method is based on the modified ‘additive Multi-<em>k</em>’ method described by Yann Surget-Groba<sup><a reveal-id="DSR046C27" data-open="DSR046C27" class="link link-ref link-reveal xref-bibr">27</a></sup> After removing reads with the sequencing adapter and reads of low quality, paired-end reads were subjected to de novo assembly using ABySS<sup><a reveal-id="DSR046C28" data-open="DSR046C28" class="link link-ref link-reveal xref-bibr">28</a></sup> with k-mer lengths of 58, 54, 52, 50, 48, 46, 44, 42, 40, 38 and 34. The unused reads at higher k-mer lengths were not discarded before running the assembly for a lower k-mer length. The output data set of each k-mer length was subjected to SSPACE<sup><a reveal-id="DSR046C29" data-open="DSR046C29" class="link link-ref link-reveal xref-bibr">29</a></sup> for scaffolding, respectively. When pooling all the results together, some contigs and scaffolds appeared in two or more assemblies, causing redundancy. These were removed using CD-HIT-EST.<sup><a reveal-id="DSR046C30" data-open="DSR046C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The longest possible contigs and scaffolds were retained. At last, the STM<sup>+</sup> method<sup><a reveal-id="DSR046C27" data-open="DSR046C27" class="link link-ref link-reveal xref-bibr">27</a></sup> was used to perform translation mapping scaffolding with the <em>Danio rerio</em> proteome<sup><a reveal-id="DSR046C31" data-open="DSR046C31" class="link link-ref link-reveal xref-bibr">31</a></sup> serving as a reference.</p>                            <h3 scrollto-destination=6206196 id="6206196" class="section-title" data-legacy-id=s2e>2.5. Sequence annotation</h3>
<p>The assembled sequences were blasted against the NCBI Nr (non-redundant) protein database and Swiss-prot database using BLASTX<sup><a reveal-id="DSR046C32" data-open="DSR046C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and an <em>E</em>-value of 1e−5. To shorten the search time, searches were limited to the first 10 significant hits for each query. Gene names were assigned to each sequence according to its best BLAST hit (highest score).</p><p>The Blast2GO suit<sup><a reveal-id="DSR046C33" data-open="DSR046C33" class="link link-ref link-reveal xref-bibr">33</a></sup> was used for functional annotation of assembled sequences applying the function for the mapping of gene ontology (GO) terms to sequences with BLAST hits obtained from hits with <em>E</em>-value &lt; 1e−5, annotation cut-off &gt; 55 and a GO weight &gt; 5 were used for annotation. Assembled sequences were thus assigned to primary and sub-GO functional categories.</p>                            <h3 scrollto-destination=6206199 id="6206199" class="section-title" data-legacy-id=s2f>2.6. Simple sequence repeat markers discovery</h3>
<p>A microsatellite program (MISA)<sup><a reveal-id="DSR046C34" data-open="DSR046C34" class="link link-ref link-reveal xref-bibr">34</a></sup> (<a class="link link-uri" href="http://pgrc.ipkgatersleben.de/misa/">http://pgrc.ipkgatersleben.de/misa/</a>) was used to identify and localize microsatellite motifs. We searched for all types of simple sequence repeats (SSRs) from mononucleotide to hexanucleotides using the following parameters: at least 10 repeats for mono-, 6 repeats for di- and 5 repeats for tri-, tetra-, penta- and hexanucleotide for simple repeats. Both perfect (i.e. SSRs contain a single repeat motif like such as ‘ATC’) and compound (i.e. composed of two or more motifs separated by &lt;100 bp) SSRs were identified.</p>                            <h3 scrollto-destination=6206201 id="6206201" class="section-title" data-legacy-id=s2g>2.7. Positive selection</h3>
<p>All the 44 sequences involved in the biosynthesis of glutathione (GSH) were downloaded from Kyoto Encyclopedia of Genes and Genomes (KEGG) and used as a query to blast against the 85 759 sequences assembled. Only the reciprocal BLAST best-hit result sequence was kept to form the sequence pair with its corresponding query. To determine whether the sequence pair underwent positive selection, a likelihood ratio test (LRT) was performed for the Nsite 7 and Nsite 8 of codeml in PAML was used.<sup><a reveal-id="DSR046C35" data-open="DSR046C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                            <h2 scrollto-destination=6206203 id="6206203" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6206204 id="6206204" class="section-title" data-legacy-id=s3a>3.1. De novo assembly with multiple k-mer lengths and sequence validation</h3>
<p>We prepared the mixed cDNAs from the heart, liver, brain, spleen and kidneys of silver carp at equivalent concentration. One lane of Illumina Genome Analyzer was performed and ∼17.10 million 75 bp paired-end reads were obtained. After cleaning the low-quality reads, we used a modified version of a previous published procedure<sup><a reveal-id="DSR046C27" data-open="DSR046C27" class="link link-ref link-reveal xref-bibr">27</a></sup> (see Materials and methods) to assemble the reads for non-redundant consensus. The bioinformatics workflow is depicted in the flowchart shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 1S</a>. Short read data have been deposited in NCBI's Short Read Archive at <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a> under the accession SRP008133.</p><p>To assemble the paired-end reads into contigs, we used ABySS<sup><a reveal-id="DSR046C28" data-open="DSR046C28" class="link link-ref link-reveal xref-bibr">28</a></sup> with different k-mer lengths (Table <a reveal-id="DSR046TB1" data-open="DSR046TB1" class="link link-reveal link-table xref-fig">1</a>). Although paired-end information has been used in ABySS, a great improvement was found after scaffolding the contigs with SSPACE<sup><a reveal-id="DSR046C29" data-open="DSR046C29" class="link link-ref link-reveal xref-bibr">29</a></sup> (Table <a reveal-id="DSR046TB2" data-open="DSR046TB2" class="link link-reveal link-table xref-fig">2</a>). After pooling all the scaffolds obtained from multiple k-mers together, 3 930 925 sequences were collected. Using CD-HIT-EST,<sup><a reveal-id="DSR046C30" data-open="DSR046C30" class="link link-ref link-reveal xref-bibr">30</a></sup> scaffolds were assembled into clusters that were analyzed for consensus. Finally, 85 796 sequences ranging from 200 to 13 880 bp were collected. The length distribution of all the sequences is shown in Fig. <a reveal-id="DSR046F1" data-open="DSR046F1" class="link link-reveal link-table xref-fig">1</a>. </p>                        <a id="6206207" scrollto-destination="6206207"></a>
<div content-id="DSR046TB1" class="table-modal"><div class="table-wrap"><div id="DSR046TB1" class="table-wrap-title" data-id="DSR046TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary statistics of the assemblies used to assess the performances of the Mulit-K <em>de novo</em> assembly method</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>k-mer </th><th>Contig &gt; 100 </th><th>N50 </th><th>Max length </th><th>Total length (Mb) </th><th>Average contig size </th></tr></thead><tbody><tr><td>single K </td><td>58 </td><td>3328 </td><td>65 </td><td>3324 </td><td>2.076 </td><td>87 </td></tr><tr><td> </td><td>54 </td><td>22 397 </td><td>159 </td><td>5597 </td><td>8.197 </td><td>127 </td></tr><tr><td> </td><td>52 </td><td>37 207 </td><td>233 </td><td>6087 </td><td>12.602 </td><td>140 </td></tr><tr><td> </td><td>50 </td><td>51 041 </td><td>239 </td><td>8297 </td><td>18.059 </td><td>142 </td></tr><tr><td> </td><td>48 </td><td>61 097 </td><td>241 </td><td>8045 </td><td>23.245 </td><td>144 </td></tr><tr><td> </td><td>46 </td><td>69 717 </td><td>242 </td><td>8358 </td><td>28.235 </td><td>145 </td></tr><tr><td> </td><td>44 </td><td>77 038 </td><td>239 </td><td>11 062 </td><td>33.133 </td><td>142 </td></tr><tr><td> </td><td>42 </td><td>82 806 </td><td>236 </td><td>10 004 </td><td>37.633 </td><td>139 </td></tr><tr><td> </td><td>40 </td><td>87 673 </td><td>233 </td><td>7322 </td><td>41.928 </td><td>135 </td></tr><tr><td> </td><td>38 </td><td>91 694 </td><td>228 </td><td>10 092 </td><td>45.964 </td><td>130 </td></tr><tr><td> </td><td>34 </td><td>97 153 </td><td>220 </td><td>13 873 </td><td>53.206 </td><td>119 </td></tr><tr><td>multi K </td><td> </td><td>118 764 </td><td>257 </td><td>13 880 </td><td>58.075 </td><td>159 </td></tr></tbody></table></div><div content-id="DSR046TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>k-mer </th><th>Contig &gt; 100 </th><th>N50 </th><th>Max length </th><th>Total length (Mb) </th><th>Average contig size </th></tr></thead><tbody><tr><td>single K </td><td>58 </td><td>3328 </td><td>65 </td><td>3324 </td><td>2.076 </td><td>87 </td></tr><tr><td> </td><td>54 </td><td>22 397 </td><td>159 </td><td>5597 </td><td>8.197 </td><td>127 </td></tr><tr><td> </td><td>52 </td><td>37 207 </td><td>233 </td><td>6087 </td><td>12.602 </td><td>140 </td></tr><tr><td> </td><td>50 </td><td>51 041 </td><td>239 </td><td>8297 </td><td>18.059 </td><td>142 </td></tr><tr><td> </td><td>48 </td><td>61 097 </td><td>241 </td><td>8045 </td><td>23.245 </td><td>144 </td></tr><tr><td> </td><td>46 </td><td>69 717 </td><td>242 </td><td>8358 </td><td>28.235 </td><td>145 </td></tr><tr><td> </td><td>44 </td><td>77 038 </td><td>239 </td><td>11 062 </td><td>33.133 </td><td>142 </td></tr><tr><td> </td><td>42 </td><td>82 806 </td><td>236 </td><td>10 004 </td><td>37.633 </td><td>139 </td></tr><tr><td> </td><td>40 </td><td>87 673 </td><td>233 </td><td>7322 </td><td>41.928 </td><td>135 </td></tr><tr><td> </td><td>38 </td><td>91 694 </td><td>228 </td><td>10 092 </td><td>45.964 </td><td>130 </td></tr><tr><td> </td><td>34 </td><td>97 153 </td><td>220 </td><td>13 873 </td><td>53.206 </td><td>119 </td></tr><tr><td>multi K </td><td> </td><td>118 764 </td><td>257 </td><td>13 880 </td><td>58.075 </td><td>159 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>These statistics correspond to the set of contig &gt; 100 bp. k-mer, required length of identical overlap match between two reads by ABySS; N50, contig length–weighted median; max length, length of the longest contig; (Total length) summed length of all contig &gt; 100 bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206207" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR046TB1" class="table-wrap-title" data-id="DSR046TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary statistics of the assemblies used to assess the performances of the Mulit-K <em>de novo</em> assembly method</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>k-mer </th><th>Contig &gt; 100 </th><th>N50 </th><th>Max length </th><th>Total length (Mb) </th><th>Average contig size </th></tr></thead><tbody><tr><td>single K </td><td>58 </td><td>3328 </td><td>65 </td><td>3324 </td><td>2.076 </td><td>87 </td></tr><tr><td> </td><td>54 </td><td>22 397 </td><td>159 </td><td>5597 </td><td>8.197 </td><td>127 </td></tr><tr><td> </td><td>52 </td><td>37 207 </td><td>233 </td><td>6087 </td><td>12.602 </td><td>140 </td></tr><tr><td> </td><td>50 </td><td>51 041 </td><td>239 </td><td>8297 </td><td>18.059 </td><td>142 </td></tr><tr><td> </td><td>48 </td><td>61 097 </td><td>241 </td><td>8045 </td><td>23.245 </td><td>144 </td></tr><tr><td> </td><td>46 </td><td>69 717 </td><td>242 </td><td>8358 </td><td>28.235 </td><td>145 </td></tr><tr><td> </td><td>44 </td><td>77 038 </td><td>239 </td><td>11 062 </td><td>33.133 </td><td>142 </td></tr><tr><td> </td><td>42 </td><td>82 806 </td><td>236 </td><td>10 004 </td><td>37.633 </td><td>139 </td></tr><tr><td> </td><td>40 </td><td>87 673 </td><td>233 </td><td>7322 </td><td>41.928 </td><td>135 </td></tr><tr><td> </td><td>38 </td><td>91 694 </td><td>228 </td><td>10 092 </td><td>45.964 </td><td>130 </td></tr><tr><td> </td><td>34 </td><td>97 153 </td><td>220 </td><td>13 873 </td><td>53.206 </td><td>119 </td></tr><tr><td>multi K </td><td> </td><td>118 764 </td><td>257 </td><td>13 880 </td><td>58.075 </td><td>159 </td></tr></tbody></table></div><div content-id="DSR046TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>k-mer </th><th>Contig &gt; 100 </th><th>N50 </th><th>Max length </th><th>Total length (Mb) </th><th>Average contig size </th></tr></thead><tbody><tr><td>single K </td><td>58 </td><td>3328 </td><td>65 </td><td>3324 </td><td>2.076 </td><td>87 </td></tr><tr><td> </td><td>54 </td><td>22 397 </td><td>159 </td><td>5597 </td><td>8.197 </td><td>127 </td></tr><tr><td> </td><td>52 </td><td>37 207 </td><td>233 </td><td>6087 </td><td>12.602 </td><td>140 </td></tr><tr><td> </td><td>50 </td><td>51 041 </td><td>239 </td><td>8297 </td><td>18.059 </td><td>142 </td></tr><tr><td> </td><td>48 </td><td>61 097 </td><td>241 </td><td>8045 </td><td>23.245 </td><td>144 </td></tr><tr><td> </td><td>46 </td><td>69 717 </td><td>242 </td><td>8358 </td><td>28.235 </td><td>145 </td></tr><tr><td> </td><td>44 </td><td>77 038 </td><td>239 </td><td>11 062 </td><td>33.133 </td><td>142 </td></tr><tr><td> </td><td>42 </td><td>82 806 </td><td>236 </td><td>10 004 </td><td>37.633 </td><td>139 </td></tr><tr><td> </td><td>40 </td><td>87 673 </td><td>233 </td><td>7322 </td><td>41.928 </td><td>135 </td></tr><tr><td> </td><td>38 </td><td>91 694 </td><td>228 </td><td>10 092 </td><td>45.964 </td><td>130 </td></tr><tr><td> </td><td>34 </td><td>97 153 </td><td>220 </td><td>13 873 </td><td>53.206 </td><td>119 </td></tr><tr><td>multi K </td><td> </td><td>118 764 </td><td>257 </td><td>13 880 </td><td>58.075 </td><td>159 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>These statistics correspond to the set of contig &gt; 100 bp. k-mer, required length of identical overlap match between two reads by ABySS; N50, contig length–weighted median; max length, length of the longest contig; (Total length) summed length of all contig &gt; 100 bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206207" target="_blank">View Large</a></div></div>                        <a id="6206208" scrollto-destination="6206208"></a>
<div content-id="DSR046TB2" class="table-modal"><div class="table-wrap"><div id="DSR046TB2" class="table-wrap-title" data-id="DSR046TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics of the scaffolds produced by SSPACE</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>k-mer </th><th>scaffold &gt; 100 </th><th>N50 </th><th>Max length </th><th>Total length (Mb) </th><th>Average scaffold size </th></tr></thead><tbody><tr><td>58 </td><td>2805 </td><td>65 </td><td>4835 </td><td>2.074 </td><td>92 </td></tr><tr><td>54 </td><td>19 184 </td><td>241 </td><td>7069 </td><td>8.165 </td><td>135 </td></tr><tr><td>52 </td><td>31 314 </td><td>279 </td><td>11 041 </td><td>12.561 </td><td>151 </td></tr><tr><td>50 </td><td>41 815 </td><td>301 </td><td>11 669 </td><td>18.033 </td><td>155 </td></tr><tr><td>48 </td><td>49 814 </td><td>325 </td><td>12 403 </td><td>23.239 </td><td>159 </td></tr><tr><td>46 </td><td>57 241 </td><td>332 </td><td>10 964 </td><td>28.259 </td><td>160 </td></tr><tr><td>44 </td><td>63 799 </td><td>324 </td><td>11 062 </td><td>33.196 </td><td>156 </td></tr><tr><td>42 </td><td>69 856 </td><td>314 </td><td>10 950 </td><td>37.804 </td><td>152 </td></tr><tr><td>40 </td><td>74 827 </td><td>302 </td><td>12 140 </td><td>42.046 </td><td>146 </td></tr><tr><td>38 </td><td>79 408 </td><td>291 </td><td>11 339 </td><td>46.097 </td><td>139 </td></tr><tr><td>34 </td><td>87 408 </td><td>270 </td><td>13 880 </td><td>53.831 </td><td>127 </td></tr></tbody></table></div><div content-id="DSR046TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>k-mer </th><th>scaffold &gt; 100 </th><th>N50 </th><th>Max length </th><th>Total length (Mb) </th><th>Average scaffold size </th></tr></thead><tbody><tr><td>58 </td><td>2805 </td><td>65 </td><td>4835 </td><td>2.074 </td><td>92 </td></tr><tr><td>54 </td><td>19 184 </td><td>241 </td><td>7069 </td><td>8.165 </td><td>135 </td></tr><tr><td>52 </td><td>31 314 </td><td>279 </td><td>11 041 </td><td>12.561 </td><td>151 </td></tr><tr><td>50 </td><td>41 815 </td><td>301 </td><td>11 669 </td><td>18.033 </td><td>155 </td></tr><tr><td>48 </td><td>49 814 </td><td>325 </td><td>12 403 </td><td>23.239 </td><td>159 </td></tr><tr><td>46 </td><td>57 241 </td><td>332 </td><td>10 964 </td><td>28.259 </td><td>160 </td></tr><tr><td>44 </td><td>63 799 </td><td>324 </td><td>11 062 </td><td>33.196 </td><td>156 </td></tr><tr><td>42 </td><td>69 856 </td><td>314 </td><td>10 950 </td><td>37.804 </td><td>152 </td></tr><tr><td>40 </td><td>74 827 </td><td>302 </td><td>12 140 </td><td>42.046 </td><td>146 </td></tr><tr><td>38 </td><td>79 408 </td><td>291 </td><td>11 339 </td><td>46.097 </td><td>139 </td></tr><tr><td>34 </td><td>87 408 </td><td>270 </td><td>13 880 </td><td>53.831 </td><td>127 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>These statistics correspond to the set of scaffold &gt; 100 bp. k-mer, required length of identical overlap match between two reads by ABySS; N50, scaffold length–weighted median; max length, length of the longest scaffold; total length, summed length of all scaffold &gt; 100 bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206208" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR046TB2" class="table-wrap-title" data-id="DSR046TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics of the scaffolds produced by SSPACE</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>k-mer </th><th>scaffold &gt; 100 </th><th>N50 </th><th>Max length </th><th>Total length (Mb) </th><th>Average scaffold size </th></tr></thead><tbody><tr><td>58 </td><td>2805 </td><td>65 </td><td>4835 </td><td>2.074 </td><td>92 </td></tr><tr><td>54 </td><td>19 184 </td><td>241 </td><td>7069 </td><td>8.165 </td><td>135 </td></tr><tr><td>52 </td><td>31 314 </td><td>279 </td><td>11 041 </td><td>12.561 </td><td>151 </td></tr><tr><td>50 </td><td>41 815 </td><td>301 </td><td>11 669 </td><td>18.033 </td><td>155 </td></tr><tr><td>48 </td><td>49 814 </td><td>325 </td><td>12 403 </td><td>23.239 </td><td>159 </td></tr><tr><td>46 </td><td>57 241 </td><td>332 </td><td>10 964 </td><td>28.259 </td><td>160 </td></tr><tr><td>44 </td><td>63 799 </td><td>324 </td><td>11 062 </td><td>33.196 </td><td>156 </td></tr><tr><td>42 </td><td>69 856 </td><td>314 </td><td>10 950 </td><td>37.804 </td><td>152 </td></tr><tr><td>40 </td><td>74 827 </td><td>302 </td><td>12 140 </td><td>42.046 </td><td>146 </td></tr><tr><td>38 </td><td>79 408 </td><td>291 </td><td>11 339 </td><td>46.097 </td><td>139 </td></tr><tr><td>34 </td><td>87 408 </td><td>270 </td><td>13 880 </td><td>53.831 </td><td>127 </td></tr></tbody></table></div><div content-id="DSR046TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>k-mer </th><th>scaffold &gt; 100 </th><th>N50 </th><th>Max length </th><th>Total length (Mb) </th><th>Average scaffold size </th></tr></thead><tbody><tr><td>58 </td><td>2805 </td><td>65 </td><td>4835 </td><td>2.074 </td><td>92 </td></tr><tr><td>54 </td><td>19 184 </td><td>241 </td><td>7069 </td><td>8.165 </td><td>135 </td></tr><tr><td>52 </td><td>31 314 </td><td>279 </td><td>11 041 </td><td>12.561 </td><td>151 </td></tr><tr><td>50 </td><td>41 815 </td><td>301 </td><td>11 669 </td><td>18.033 </td><td>155 </td></tr><tr><td>48 </td><td>49 814 </td><td>325 </td><td>12 403 </td><td>23.239 </td><td>159 </td></tr><tr><td>46 </td><td>57 241 </td><td>332 </td><td>10 964 </td><td>28.259 </td><td>160 </td></tr><tr><td>44 </td><td>63 799 </td><td>324 </td><td>11 062 </td><td>33.196 </td><td>156 </td></tr><tr><td>42 </td><td>69 856 </td><td>314 </td><td>10 950 </td><td>37.804 </td><td>152 </td></tr><tr><td>40 </td><td>74 827 </td><td>302 </td><td>12 140 </td><td>42.046 </td><td>146 </td></tr><tr><td>38 </td><td>79 408 </td><td>291 </td><td>11 339 </td><td>46.097 </td><td>139 </td></tr><tr><td>34 </td><td>87 408 </td><td>270 </td><td>13 880 </td><td>53.831 </td><td>127 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>These statistics correspond to the set of scaffold &gt; 100 bp. k-mer, required length of identical overlap match between two reads by ABySS; N50, scaffold length–weighted median; max length, length of the longest scaffold; total length, summed length of all scaffold &gt; 100 bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206208" target="_blank">View Large</a></div></div>                        <a id="6206209" scrollto-destination="6206209"></a>
<div id="DSR046F1" data-id="DSR046F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/m_dsr04601.jpeg?Expires=2147483647&Signature=lgI~lWv7PtBEot4mzMW00RfibnyIy4Gq1~uz-RwilruuF5vKnyw45subHkY3D4xascPDI62I6g-5lNVt7ULsP3A1MWCG8SxkhVC4QUgEpNuKzPKYwHcuqE3Gq8IGR6u2ORLbei3n~fsX9VUERl~0yAlguOTUQBe09hbVed2btVTjheYiWNoF8SiVXJXyPwBodvIpN45Xvq0KNoppD4OFjSYpCSK-al0GmewC0CDb-Ur-0PncWcrEuabrJKoq2w8N1Ja5zWr84m9NB1uYDrKGnv7dvikBKmyaL0R6qZDRHmWY~0-ULV6cwjqHHz99scwgZMgx2MQcJ9jpU7zGh0SJmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length distributions of scaffolds assembled by a multiple k-mer method." path-from-xml="dsr04601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6206209/dsr04601.jpeg" path-from-xml="dsr04601.jpeg" target="_blank">View large</a><a section="6206209" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr04601.jpeg?Expires=2147483647&Signature=tYdEIm~sc0QViJ4wgjq~h6gppONM83MEZUd9oKmVyzbWKmTMrpzhiwoCjTxAiMy-iWCygoagw7jhli1s~DaeIHJUatcZYEczEDex7C4WGGu-~wyteMFMqWSMUUNxIlkEG1X5mK3~GIuqrZagC9HSLIyQlr9yzSC9AiqR0VEuPimSJEeNquggpVNcEuauMBNfkUcZc13ypqrzqt~44z97746fYbjLQ4mk~EVqvQyKeSpFoP4X1qxw4UQPte9Dh16O5d1D6tLzdqDtNZIE-XOD6PFnwTHSJVN2C5dsXZua~QjzxuhjJDFphYOhljIW7M0hqR0mNAovUrsbwQgnfYKKgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6206209&ar=419166&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length distributions of scaffolds assembled by a multiple k-mer method.</p></div></div><div content-id="DSR046F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/m_dsr04601.jpeg?Expires=2147483647&Signature=lgI~lWv7PtBEot4mzMW00RfibnyIy4Gq1~uz-RwilruuF5vKnyw45subHkY3D4xascPDI62I6g-5lNVt7ULsP3A1MWCG8SxkhVC4QUgEpNuKzPKYwHcuqE3Gq8IGR6u2ORLbei3n~fsX9VUERl~0yAlguOTUQBe09hbVed2btVTjheYiWNoF8SiVXJXyPwBodvIpN45Xvq0KNoppD4OFjSYpCSK-al0GmewC0CDb-Ur-0PncWcrEuabrJKoq2w8N1Ja5zWr84m9NB1uYDrKGnv7dvikBKmyaL0R6qZDRHmWY~0-ULV6cwjqHHz99scwgZMgx2MQcJ9jpU7zGh0SJmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length distributions of scaffolds assembled by a multiple k-mer method." path-from-xml="dsr04601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6206209/dsr04601.jpeg" path-from-xml="dsr04601.jpeg" target="_blank">View large</a><a section="6206209" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr04601.jpeg?Expires=2147483647&Signature=tYdEIm~sc0QViJ4wgjq~h6gppONM83MEZUd9oKmVyzbWKmTMrpzhiwoCjTxAiMy-iWCygoagw7jhli1s~DaeIHJUatcZYEczEDex7C4WGGu-~wyteMFMqWSMUUNxIlkEG1X5mK3~GIuqrZagC9HSLIyQlr9yzSC9AiqR0VEuPimSJEeNquggpVNcEuauMBNfkUcZc13ypqrzqt~44z97746fYbjLQ4mk~EVqvQyKeSpFoP4X1qxw4UQPte9Dh16O5d1D6tLzdqDtNZIE-XOD6PFnwTHSJVN2C5dsXZua~QjzxuhjJDFphYOhljIW7M0hqR0mNAovUrsbwQgnfYKKgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6206209&ar=419166&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length distributions of scaffolds assembled by a multiple k-mer method.</p></div></div><p>To determine the expression level of the transcripts, we mapped the raw reads to the assembled sequences with SOAP<sup><a reveal-id="DSR046C36" data-open="DSR046C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and the RPKM value (Reads Per Kilobase of exon model per Million mapped reads) of all the transcripts are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Figure <a reveal-id="DSR046F2" data-open="DSR046F2" class="link link-reveal link-table xref-fig">2</a> depicts the relationship of RPKM versus the transcript size. Transcript length increased with coverage depth and reached an asymptote approximately at an average coverage of ∼50. </p>                        <a id="6206211" scrollto-destination="6206211"></a>
<div id="DSR046F2" data-id="DSR046F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/m_dsr04602.jpeg?Expires=2147483647&Signature=kr4h79tC5GD9iTnZgKNclJuv9CZmwx09N-uwYcIeXvxTmkGwrtKTFPT8VQ5ESyktlDM7Y97YHXGdETX7fc8JDbLx29iYTvSH6ksalm6Fi-X20Nf1i2Ngnb55AH8rOiOEO8XkvW58PHy60rFyzBzNKuZFXhELVicJrwt-uNYlgTHttmbJl7aT3Ou9MxsTexnoDlxqgqICjTcT~-6RVoYPvB6bH1BmNVmYDO8wP30gchwoW7CxWH9ov1JhBOu1mHhaFWkU03OKScdLVx~uFObNu70rzEd6wAWtwif9Ws7sfUMnRqXO5dzv5Yuoo68IEaxOPOuq9IkfJQcukqAgEg9bcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The relationship of RPKM versus the transcript size. RPKM, Reads Per Kilobase of exon model per Million mapped reads." path-from-xml="dsr04602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6206211/dsr04602.jpeg" path-from-xml="dsr04602.jpeg" target="_blank">View large</a><a section="6206211" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr04602.jpeg?Expires=2147483647&Signature=aWeJAkc1kkzXADZjnmSAqU9iL1n4Gl2I4BbqHOp3v3EdnPqMrnss-jZt56BFePSenAwScMaeRuiEjA23nKb5gBsfg1fcknTUDNUz5MLXQP4skvhCJA85TwGhuYOaWcrTeySiErfBFFRiEh-YOpGPPPcTQBJKddQj3a1keIg9BlHfDojK2~TIkQxYLWBFmdW1caoHW5D~0wrZEEMr6WvGeXiLCn~5CNmGOErIVq9MbQ5iRkFcg4zpfnWczzjc99mYH5ziygYFgJb6xPFFaaQWhG2LiYIgve0YDDNz5eQpEr4pZgMKOKbUy9urR-O-WBgSsQWlh9Q4zqPXnpDltUCYbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6206211&ar=419166&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The relationship of RPKM versus the transcript size. RPKM, Reads Per Kilobase of exon model per Million mapped reads.</p></div></div><div content-id="DSR046F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/m_dsr04602.jpeg?Expires=2147483647&Signature=kr4h79tC5GD9iTnZgKNclJuv9CZmwx09N-uwYcIeXvxTmkGwrtKTFPT8VQ5ESyktlDM7Y97YHXGdETX7fc8JDbLx29iYTvSH6ksalm6Fi-X20Nf1i2Ngnb55AH8rOiOEO8XkvW58PHy60rFyzBzNKuZFXhELVicJrwt-uNYlgTHttmbJl7aT3Ou9MxsTexnoDlxqgqICjTcT~-6RVoYPvB6bH1BmNVmYDO8wP30gchwoW7CxWH9ov1JhBOu1mHhaFWkU03OKScdLVx~uFObNu70rzEd6wAWtwif9Ws7sfUMnRqXO5dzv5Yuoo68IEaxOPOuq9IkfJQcukqAgEg9bcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The relationship of RPKM versus the transcript size. RPKM, Reads Per Kilobase of exon model per Million mapped reads." path-from-xml="dsr04602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6206211/dsr04602.jpeg" path-from-xml="dsr04602.jpeg" target="_blank">View large</a><a section="6206211" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr04602.jpeg?Expires=2147483647&Signature=aWeJAkc1kkzXADZjnmSAqU9iL1n4Gl2I4BbqHOp3v3EdnPqMrnss-jZt56BFePSenAwScMaeRuiEjA23nKb5gBsfg1fcknTUDNUz5MLXQP4skvhCJA85TwGhuYOaWcrTeySiErfBFFRiEh-YOpGPPPcTQBJKddQj3a1keIg9BlHfDojK2~TIkQxYLWBFmdW1caoHW5D~0wrZEEMr6WvGeXiLCn~5CNmGOErIVq9MbQ5iRkFcg4zpfnWczzjc99mYH5ziygYFgJb6xPFFaaQWhG2LiYIgve0YDDNz5eQpEr4pZgMKOKbUy9urR-O-WBgSsQWlh9Q4zqPXnpDltUCYbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6206211&ar=419166&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The relationship of RPKM versus the transcript size. RPKM, Reads Per Kilobase of exon model per Million mapped reads.</p></div></div><p>Until now, no general criteria have been proposed as standards for evaluation of the quality of transcriptome assembly. We used three substantial factors to assess how well the assembled sequences represent the actual transcriptome population: (i) gene coverage, (ii) transcript sequence quality and (iii) completeness.</p><p>The transcriptome gene coverage was judged by comparison with the sequence information available for silver carp. All 13 mitochondrial protein-coding genes and 203 of 217 proteins in the NCBI database were present in our assembled scaffolds. We compared our assembled scaffolds with the zebrafish transcriptome (ENSEMBL Zv61) and found that 40 509 of 41 759 (85.9%) zebrafish transcripts have matches in assembled scaffolds. At the same time, 19 893 reciprocal best-hit blast matches with the zebrafish transcriptome were identified using <em>E</em>-value 1e−5.</p><p>Transcriptome quality was assessed by comparing the mitochondrial protein-coding genes found in assembled sequences to mitochondrion sequence in GenBank (NC_010156). A total of 10 185 nucleotide identities were observed out of 10 522 (96.8%) total nucleotide length of contig to coding mitochondrial sequences BLAST matches, suggesting very good transcriptome sequence quality. The observed 3.2% sequence difference might be due to the high intraspecific genetic variability.</p><p>Finally, in terms of sequence completeness, the relative number of full-length sequences in the 19 893 reciprocal best-hit blast matches to zebrafish transcriptome was estimated. A sequence was considered full length if it contained the complete 5′- and 3′-UTR of the mRNA. In this study, we used a less stringent but broadly adopted definition, considering a sequence to be full length if it comprised at least the complete coding sequence (CDS).<sup><a reveal-id="DSR046C21" data-open="DSR046C21" class="link link-ref link-reveal xref-bibr">21</a></sup> We mapped the 19 893 sequences to their corresponding CDS in the zebrafish transcript, and if the CDS was fully covered by assembled sequence, we thought the sequence as full-length sequence. Under the criteria given above, 1937 sequences (9.7%) were validated as full length. One thousand, six hundred and thirty-five sequences (8.2%) covered more than 95% of the zebrafish CDS and 2394 sequences (12.0%) covered more than 75% of the zebrafish CDS. For a pseudo-stop codon usually appears on a chimeric or truncated transcript, we translated the nucleotide sequences to protein sequences to verify the completeness of the transcripts. One thousand, three hundred and seventy-seven sequences were validated as full length and 2109 sequences covered more than 95% of the zebrafish CDS.</p><p>In addition to the computing methods given above, Reverse transcription polymerase chain reaction (RT-PCR) assay was used to validate the quality of the assembled transcriptome. Primers for 22 transcripts with different expression levels (RPKM ranged from 336 to 10 507) were designed and all the cDNAs were amplified. Out of the 22 pairs of primers, 10 pairs were silver-specific transcripts which did not have an NCBI Nr BLAST hit (see Sequence annotation). The primer information and RT–PCR results are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2 and Fig. S2</a>.</p>                            <h3 scrollto-destination=6206221 id="6206221" class="section-title" data-legacy-id=s3b>3.2. Sequence annotation</h3>
<p>Several complementary methods were used to annotate the assembled sequences. First, the assembled sequences were searched against the Nr protein databases using BLASTX with an <em>E</em>-value of 1e−5. Of the 85 796 assembled sequences, 54 198 (63.2%) had significant matches (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Most of the sequences with top-hit blast result from zebrafish (44 999 sequences; 83.0%). In addition, 18 536 (34.2%) sequences matched predicted proteins, 81 (0.1%) with unknown proteins.</p><p>Second, silver carp sequences that had matches in Nr databases were given GO annotations with the Uniprot database. Of these, 23 044 were assigned to one or more 3423 GO terms, for a total of 104 196 term occurrences. As many as 17 451 sequences were found to be involved in biological process and could be divided into cellular process (13 382 sequences with percentage of 76.7%), metabolic process (10 846; 62.2%), biological regulation (5032; 28.9%), multicellular organismal process (4386; 25.1%), pigmentation (4296; 24.6%), localization (4162; 23.8%), developmental process (4088; 23.4%), establishment of localization (3448; 19.8%), cellular component organization (2793; 16.0%) and response to stimulus (2275; 13.0%). Other type of functions occurred at &lt;10% each (Fig. <a reveal-id="DSR046F3" data-open="DSR046F3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="6206228" scrollto-destination="6206228"></a>
<div id="DSR046F3" data-id="DSR046F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/m_dsr04603.jpeg?Expires=2147483647&Signature=RQbfv1A07r5e8ziE3utW99rnVfPnPQ4UD5VrEeNC72OTAhzc9-686dffesfq-Qso5EyE~DrJOfuNgzsueY8O00Iz1ky93ksTeJpWE8C4NV8eMP2ZE~zo5taFSAuNRJCGvQN8wXzJI7XLjlUdIRetPdmB15Q8-hX0kxzPiNn9dSpEBM1yM~e9tM1vfn6s0aMkEOUWJ~qwZsRUd6sfVt7CyCnJYq0gfHL0QhYLFhya1PYCbW1sqQCYBWwup2DxdeN9B2XIzENO2n8JdN847VFqza834JtbjxHRN75jAjs7OqaCEYnC2adGXIKQs2R6wBAHFoZ9Lt-q5Zay1W9xWYZCGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of silver carp transcriptome and comparison with zebrafish transcriptome. (A) GO: biological process. (B) GO: molecular function. (C) GO: cellular component. In some cases, one transcript may have multiple functions. Grey, silver carp; black, zebrafish." path-from-xml="dsr04603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6206228/dsr04603.jpeg" path-from-xml="dsr04603.jpeg" target="_blank">View large</a><a section="6206228" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr04603.jpeg?Expires=2147483647&Signature=LUhYrYS2WOWGdbvu-hVxpCN5NUD5-pV7GhYbRRA1ux3r0nVXG0ZZDfc2pXCtILPkKTv3VwCZApqdDllCWIwBaM4UXISvCkTsng3IRMkvMes~hLvUnA8k~Si5g1Bx3~EBZQR1TZ8IfhhWShvZPOFUx0zXsErsvavZ5qsUZpBFRha9EB1i-roFUNzAsrx0K9lsWMT5-UEmUoN-JRGW2ms6kHdp1BJNhCYXiB2CCmm49tIsrexh~VqXYd~rCFwCwHjX0OR5rXfyc8v3hqvyh5lHdLZv2w7LkLGAGw1so3Oa0vaj7cCW2J5d~zwFaXwnYVbLRSJ2Ml1ia0RH17WM8gI13Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6206228&ar=419166&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of silver carp transcriptome and comparison with zebrafish transcriptome. (A) GO: biological process. (B) GO: molecular function. (C) GO: cellular component. In some cases, one transcript may have multiple functions. Grey, silver carp; black, zebrafish.</p></div></div><div content-id="DSR046F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/m_dsr04603.jpeg?Expires=2147483647&Signature=RQbfv1A07r5e8ziE3utW99rnVfPnPQ4UD5VrEeNC72OTAhzc9-686dffesfq-Qso5EyE~DrJOfuNgzsueY8O00Iz1ky93ksTeJpWE8C4NV8eMP2ZE~zo5taFSAuNRJCGvQN8wXzJI7XLjlUdIRetPdmB15Q8-hX0kxzPiNn9dSpEBM1yM~e9tM1vfn6s0aMkEOUWJ~qwZsRUd6sfVt7CyCnJYq0gfHL0QhYLFhya1PYCbW1sqQCYBWwup2DxdeN9B2XIzENO2n8JdN847VFqza834JtbjxHRN75jAjs7OqaCEYnC2adGXIKQs2R6wBAHFoZ9Lt-q5Zay1W9xWYZCGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of silver carp transcriptome and comparison with zebrafish transcriptome. (A) GO: biological process. (B) GO: molecular function. (C) GO: cellular component. In some cases, one transcript may have multiple functions. Grey, silver carp; black, zebrafish." path-from-xml="dsr04603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6206228/dsr04603.jpeg" path-from-xml="dsr04603.jpeg" target="_blank">View large</a><a section="6206228" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr04603.jpeg?Expires=2147483647&Signature=LUhYrYS2WOWGdbvu-hVxpCN5NUD5-pV7GhYbRRA1ux3r0nVXG0ZZDfc2pXCtILPkKTv3VwCZApqdDllCWIwBaM4UXISvCkTsng3IRMkvMes~hLvUnA8k~Si5g1Bx3~EBZQR1TZ8IfhhWShvZPOFUx0zXsErsvavZ5qsUZpBFRha9EB1i-roFUNzAsrx0K9lsWMT5-UEmUoN-JRGW2ms6kHdp1BJNhCYXiB2CCmm49tIsrexh~VqXYd~rCFwCwHjX0OR5rXfyc8v3hqvyh5lHdLZv2w7LkLGAGw1so3Oa0vaj7cCW2J5d~zwFaXwnYVbLRSJ2Ml1ia0RH17WM8gI13Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6206228&ar=419166&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of silver carp transcriptome and comparison with zebrafish transcriptome. (A) GO: biological process. (B) GO: molecular function. (C) GO: cellular component. In some cases, one transcript may have multiple functions. Grey, silver carp; black, zebrafish.</p></div></div><p>GO analysis have also shown that 15 799 sequences were associated with a cellular component, including cell (15 114; 95.6%), cell part (15 114; 95.6%), organelle (8265; 52.3%), organelle part (3624; 22.9%) and macromolecular complex (3042; 19.2%). Moreover, 17 837 sequences showed potential molecular function, such as binding (13 027; 73%), catalytic activity (8473; 47.5%), molecular transducer activity (1648; 9.2%) and transporter activity (1406; 7.8%). The detailed information about the functional classification is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>.</p><p>Representation of GO categories in the silver carp transcriptome set was found to be similar to that of the zebrafish GO database, but there were a few differences in each of the three main GO categories (Fig. <a reveal-id="DSR046F3" data-open="DSR046F3" class="link link-reveal link-table xref-fig">3</a>). After correcting for multiple tests, we found that 30 of 37 comparisons were significantly over or underrepresented in comparison to the zebrafish records. For example, among the biological processes, pigmentation (GO: 0043473) was underrepresented in the silver carp transcriptome, while localization (GO: 0051179) and response to stimulus (GO: 0050896) were overrepresented.</p><p>Meanwhile, annotation of the 85 759 sequences using Clusters of Orthologous Groups of protein (COG) databases yielded good results for 14 840 putative proteins. The COG-annotated putative proteins ranged functionally into at least 25 molecular families, including biochemical metabolism, signal transduction, cellular structure and immune defense, in accordance with the categories observed in GO annotation (Fig. <a reveal-id="DSR046F4" data-open="DSR046F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="6206238" scrollto-destination="6206238"></a>
<div id="DSR046F4" data-id="DSR046F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/m_dsr04604.jpeg?Expires=2147483647&Signature=ZRBF7rhFTxpJYkb4sZfzrGh0NNCZ4YZuZv3yo3oHXb46alTsCoWfF12BfhzGZ4ZBBXcKjCTSg~65H1H-RyOsObbnOV1yIvZJ1PzvkCd9R~EuaRDXiDl4MMAE9Tj6OkK5mfd4vOdIplG4M4DJejk8j~cAdeqQmt6EkG9DciztwdBNEIaUPwxH4Lxky~pRBSJf83mgeS1KzoWdE~-Pxp5jNah6AoIgImoR-AUBgiglLK8H3WcekN9G5K8fTJR0gebZiLE5bUaMEggKV8Sbz870vItYjTq2RTmPWegc0G~CqgZSAY673jQhdgt56dsS83KmAF0SefI4akgFi3VTtOC66w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="COG annotations of putative proteins. All putative proteins were aligned to COG database and can be classified functionally into at least 25 molecular families." path-from-xml="dsr04604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6206238/dsr04604.jpeg" path-from-xml="dsr04604.jpeg" target="_blank">View large</a><a section="6206238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr04604.jpeg?Expires=2147483647&Signature=oVzSw9JSSiu1O-N-LdTDNrDkxT5aixUZQtBLzFWx8qLjf3NUH4zZQpriJmBcRsuIqZn03am5cAzMFHPpppc3LxHaLVZtc4yOynzxlKUx0eiP9jWiCCTKJpUX8ijnbPKosMrJJUmTzyCtftZ5BZGJ73oFDFek7IpGUSaOUGT9Ar7213UMh~Ms1rowcfTNek-oB70BHhRTjnW~Z1OtFLriX82I78JlLaAXRPK-F2-v1pUTcuDVzia7-XmJOpPe52haBSdGmo7CZVyDL2CZQkiNRSp4tptqwS7Uw3yUuWHp1x0hCIi97kLtmXXrziOsQYg228tfIphTNXZipJA-M1NYmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6206238&ar=419166&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>COG annotations of putative proteins. All putative proteins were aligned to COG database and can be classified functionally into at least 25 molecular families.</p></div></div><div content-id="DSR046F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/m_dsr04604.jpeg?Expires=2147483647&Signature=ZRBF7rhFTxpJYkb4sZfzrGh0NNCZ4YZuZv3yo3oHXb46alTsCoWfF12BfhzGZ4ZBBXcKjCTSg~65H1H-RyOsObbnOV1yIvZJ1PzvkCd9R~EuaRDXiDl4MMAE9Tj6OkK5mfd4vOdIplG4M4DJejk8j~cAdeqQmt6EkG9DciztwdBNEIaUPwxH4Lxky~pRBSJf83mgeS1KzoWdE~-Pxp5jNah6AoIgImoR-AUBgiglLK8H3WcekN9G5K8fTJR0gebZiLE5bUaMEggKV8Sbz870vItYjTq2RTmPWegc0G~CqgZSAY673jQhdgt56dsS83KmAF0SefI4akgFi3VTtOC66w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="COG annotations of putative proteins. All putative proteins were aligned to COG database and can be classified functionally into at least 25 molecular families." path-from-xml="dsr04604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6206238/dsr04604.jpeg" path-from-xml="dsr04604.jpeg" target="_blank">View large</a><a section="6206238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr04604.jpeg?Expires=2147483647&Signature=oVzSw9JSSiu1O-N-LdTDNrDkxT5aixUZQtBLzFWx8qLjf3NUH4zZQpriJmBcRsuIqZn03am5cAzMFHPpppc3LxHaLVZtc4yOynzxlKUx0eiP9jWiCCTKJpUX8ijnbPKosMrJJUmTzyCtftZ5BZGJ73oFDFek7IpGUSaOUGT9Ar7213UMh~Ms1rowcfTNek-oB70BHhRTjnW~Z1OtFLriX82I78JlLaAXRPK-F2-v1pUTcuDVzia7-XmJOpPe52haBSdGmo7CZVyDL2CZQkiNRSp4tptqwS7Uw3yUuWHp1x0hCIi97kLtmXXrziOsQYg228tfIphTNXZipJA-M1NYmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6206238&ar=419166&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>COG annotations of putative proteins. All putative proteins were aligned to COG database and can be classified functionally into at least 25 molecular families.</p></div></div>                            <h3 scrollto-destination=6206243 id="6206243" class="section-title" data-legacy-id=s3c>3.3. Metabolic pathways by KEGG analysis</h3>
<p>A total of 38 200 assembled sequences were found to be involved in 218 predicted KEGG metabolic pathways. The number of sequences ranged from 3 to 4510 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The top 20 pathways with the greatest number of sequences are shown in Table <a reveal-id="DSR046TB3" data-open="DSR046TB3" class="link link-reveal link-table xref-fig">3</a>, and the greatest number of transcripts was found in the metabolic pathways. The top 10 metabolic pathways were: purine metabolism (789), pyrimidine metabolism (473), oxidative phosphorylation (436), inositol phosphate metabolism (435), glycerophospholipid metabolism (371), riboflavin metabolism (347), glycolysis/gluconeogenesis (341), lysine degradation (337), pyruvate metabolism (239) and starch and sucrose metabolism (218) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). </p>                        <a id="6206245" scrollto-destination="6206245"></a>
<div content-id="DSR046TB3" class="table-modal"><div class="table-wrap"><div id="DSR046TB3" class="table-wrap-title" data-id="DSR046TB3"><div class="label -label">Table 3.</div><div class="caption"><p>The top 20 pathways with highest sequence numbers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Num </th><th>Pathway </th><th>All genes with pathway annotation (38 200) </th><th>Pathway ID </th></tr></thead><tbody><tr><td>1 </td><td>Metabolic pathways </td><td>4510 (11.81%) </td><td>ko01100 </td></tr><tr><td>2 </td><td>Pathways in cancer </td><td>1790 (4.69%) </td><td>ko05200 </td></tr><tr><td>3 </td><td>Regulation of actin cytoskeleton </td><td>1634 (4.28%) </td><td>ko04810 </td></tr><tr><td>4 </td><td>Focal adhesion </td><td>1518 (3.97%) </td><td>ko04510 </td></tr><tr><td>5 </td><td>MAPK signaling pathway </td><td>1463 (3.83%) </td><td>ko04010 </td></tr><tr><td>6 </td><td>Endocytosis </td><td>1345 (3.52%) </td><td>ko04144 </td></tr><tr><td>7 </td><td>Tight junction </td><td>1256 (3.29%) </td><td>ko04530 </td></tr><tr><td>8 </td><td>Adherens junction </td><td>1073 (2.81%) </td><td>ko04520 </td></tr><tr><td>9 </td><td>Phagosome </td><td>1034 (2.71%) </td><td>ko04145 </td></tr><tr><td>10 </td><td>Dilated cardiomyopathy </td><td>1027 (2.69%) </td><td>ko05414 </td></tr><tr><td>11 </td><td>Vascular smooth muscle contraction </td><td>1014 (2.65%) </td><td>ko04270 </td></tr><tr><td>12 </td><td>Complement and coagulation cascades </td><td>1005 (2.63%) </td><td>ko04610 </td></tr><tr><td>13 </td><td>Hypertrophic cardiomyopathy (HCM) </td><td>957 (2.51%) </td><td>ko05410 </td></tr><tr><td>14 </td><td>Chemokine signaling pathway </td><td>955 (2.5%) </td><td>ko04062 </td></tr><tr><td>15 </td><td>Calcium signaling pathway </td><td>942 (2.47%) </td><td>ko04020 </td></tr><tr><td>16 </td><td>Axon guidance </td><td>939 (2.46%) </td><td>ko04360 </td></tr><tr><td>17 </td><td>Insulin signaling pathway </td><td>912 (2.39%) </td><td>ko04910 </td></tr><tr><td>18 </td><td>Huntington's disease </td><td>907 (2.37%) </td><td>ko05016 </td></tr><tr><td>19 </td><td>Leukocyte transendothelial migration </td><td>869 (2.27%) </td><td>ko04670 </td></tr><tr><td>20 </td><td>Protein processing in endoplasmic reticulum </td><td>864 (2.26%) </td><td>ko04141 </td></tr></tbody></table></div><div content-id="DSR046TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Num </th><th>Pathway </th><th>All genes with pathway annotation (38 200) </th><th>Pathway ID </th></tr></thead><tbody><tr><td>1 </td><td>Metabolic pathways </td><td>4510 (11.81%) </td><td>ko01100 </td></tr><tr><td>2 </td><td>Pathways in cancer </td><td>1790 (4.69%) </td><td>ko05200 </td></tr><tr><td>3 </td><td>Regulation of actin cytoskeleton </td><td>1634 (4.28%) </td><td>ko04810 </td></tr><tr><td>4 </td><td>Focal adhesion </td><td>1518 (3.97%) </td><td>ko04510 </td></tr><tr><td>5 </td><td>MAPK signaling pathway </td><td>1463 (3.83%) </td><td>ko04010 </td></tr><tr><td>6 </td><td>Endocytosis </td><td>1345 (3.52%) </td><td>ko04144 </td></tr><tr><td>7 </td><td>Tight junction </td><td>1256 (3.29%) </td><td>ko04530 </td></tr><tr><td>8 </td><td>Adherens junction </td><td>1073 (2.81%) </td><td>ko04520 </td></tr><tr><td>9 </td><td>Phagosome </td><td>1034 (2.71%) </td><td>ko04145 </td></tr><tr><td>10 </td><td>Dilated cardiomyopathy </td><td>1027 (2.69%) </td><td>ko05414 </td></tr><tr><td>11 </td><td>Vascular smooth muscle contraction </td><td>1014 (2.65%) </td><td>ko04270 </td></tr><tr><td>12 </td><td>Complement and coagulation cascades </td><td>1005 (2.63%) </td><td>ko04610 </td></tr><tr><td>13 </td><td>Hypertrophic cardiomyopathy (HCM) </td><td>957 (2.51%) </td><td>ko05410 </td></tr><tr><td>14 </td><td>Chemokine signaling pathway </td><td>955 (2.5%) </td><td>ko04062 </td></tr><tr><td>15 </td><td>Calcium signaling pathway </td><td>942 (2.47%) </td><td>ko04020 </td></tr><tr><td>16 </td><td>Axon guidance </td><td>939 (2.46%) </td><td>ko04360 </td></tr><tr><td>17 </td><td>Insulin signaling pathway </td><td>912 (2.39%) </td><td>ko04910 </td></tr><tr><td>18 </td><td>Huntington's disease </td><td>907 (2.37%) </td><td>ko05016 </td></tr><tr><td>19 </td><td>Leukocyte transendothelial migration </td><td>869 (2.27%) </td><td>ko04670 </td></tr><tr><td>20 </td><td>Protein processing in endoplasmic reticulum </td><td>864 (2.26%) </td><td>ko04141 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206245" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR046TB3" class="table-wrap-title" data-id="DSR046TB3"><div class="label -label">Table 3.</div><div class="caption"><p>The top 20 pathways with highest sequence numbers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Num </th><th>Pathway </th><th>All genes with pathway annotation (38 200) </th><th>Pathway ID </th></tr></thead><tbody><tr><td>1 </td><td>Metabolic pathways </td><td>4510 (11.81%) </td><td>ko01100 </td></tr><tr><td>2 </td><td>Pathways in cancer </td><td>1790 (4.69%) </td><td>ko05200 </td></tr><tr><td>3 </td><td>Regulation of actin cytoskeleton </td><td>1634 (4.28%) </td><td>ko04810 </td></tr><tr><td>4 </td><td>Focal adhesion </td><td>1518 (3.97%) </td><td>ko04510 </td></tr><tr><td>5 </td><td>MAPK signaling pathway </td><td>1463 (3.83%) </td><td>ko04010 </td></tr><tr><td>6 </td><td>Endocytosis </td><td>1345 (3.52%) </td><td>ko04144 </td></tr><tr><td>7 </td><td>Tight junction </td><td>1256 (3.29%) </td><td>ko04530 </td></tr><tr><td>8 </td><td>Adherens junction </td><td>1073 (2.81%) </td><td>ko04520 </td></tr><tr><td>9 </td><td>Phagosome </td><td>1034 (2.71%) </td><td>ko04145 </td></tr><tr><td>10 </td><td>Dilated cardiomyopathy </td><td>1027 (2.69%) </td><td>ko05414 </td></tr><tr><td>11 </td><td>Vascular smooth muscle contraction </td><td>1014 (2.65%) </td><td>ko04270 </td></tr><tr><td>12 </td><td>Complement and coagulation cascades </td><td>1005 (2.63%) </td><td>ko04610 </td></tr><tr><td>13 </td><td>Hypertrophic cardiomyopathy (HCM) </td><td>957 (2.51%) </td><td>ko05410 </td></tr><tr><td>14 </td><td>Chemokine signaling pathway </td><td>955 (2.5%) </td><td>ko04062 </td></tr><tr><td>15 </td><td>Calcium signaling pathway </td><td>942 (2.47%) </td><td>ko04020 </td></tr><tr><td>16 </td><td>Axon guidance </td><td>939 (2.46%) </td><td>ko04360 </td></tr><tr><td>17 </td><td>Insulin signaling pathway </td><td>912 (2.39%) </td><td>ko04910 </td></tr><tr><td>18 </td><td>Huntington's disease </td><td>907 (2.37%) </td><td>ko05016 </td></tr><tr><td>19 </td><td>Leukocyte transendothelial migration </td><td>869 (2.27%) </td><td>ko04670 </td></tr><tr><td>20 </td><td>Protein processing in endoplasmic reticulum </td><td>864 (2.26%) </td><td>ko04141 </td></tr></tbody></table></div><div content-id="DSR046TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Num </th><th>Pathway </th><th>All genes with pathway annotation (38 200) </th><th>Pathway ID </th></tr></thead><tbody><tr><td>1 </td><td>Metabolic pathways </td><td>4510 (11.81%) </td><td>ko01100 </td></tr><tr><td>2 </td><td>Pathways in cancer </td><td>1790 (4.69%) </td><td>ko05200 </td></tr><tr><td>3 </td><td>Regulation of actin cytoskeleton </td><td>1634 (4.28%) </td><td>ko04810 </td></tr><tr><td>4 </td><td>Focal adhesion </td><td>1518 (3.97%) </td><td>ko04510 </td></tr><tr><td>5 </td><td>MAPK signaling pathway </td><td>1463 (3.83%) </td><td>ko04010 </td></tr><tr><td>6 </td><td>Endocytosis </td><td>1345 (3.52%) </td><td>ko04144 </td></tr><tr><td>7 </td><td>Tight junction </td><td>1256 (3.29%) </td><td>ko04530 </td></tr><tr><td>8 </td><td>Adherens junction </td><td>1073 (2.81%) </td><td>ko04520 </td></tr><tr><td>9 </td><td>Phagosome </td><td>1034 (2.71%) </td><td>ko04145 </td></tr><tr><td>10 </td><td>Dilated cardiomyopathy </td><td>1027 (2.69%) </td><td>ko05414 </td></tr><tr><td>11 </td><td>Vascular smooth muscle contraction </td><td>1014 (2.65%) </td><td>ko04270 </td></tr><tr><td>12 </td><td>Complement and coagulation cascades </td><td>1005 (2.63%) </td><td>ko04610 </td></tr><tr><td>13 </td><td>Hypertrophic cardiomyopathy (HCM) </td><td>957 (2.51%) </td><td>ko05410 </td></tr><tr><td>14 </td><td>Chemokine signaling pathway </td><td>955 (2.5%) </td><td>ko04062 </td></tr><tr><td>15 </td><td>Calcium signaling pathway </td><td>942 (2.47%) </td><td>ko04020 </td></tr><tr><td>16 </td><td>Axon guidance </td><td>939 (2.46%) </td><td>ko04360 </td></tr><tr><td>17 </td><td>Insulin signaling pathway </td><td>912 (2.39%) </td><td>ko04910 </td></tr><tr><td>18 </td><td>Huntington's disease </td><td>907 (2.37%) </td><td>ko05016 </td></tr><tr><td>19 </td><td>Leukocyte transendothelial migration </td><td>869 (2.27%) </td><td>ko04670 </td></tr><tr><td>20 </td><td>Protein processing in endoplasmic reticulum </td><td>864 (2.26%) </td><td>ko04141 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206245" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6206246 id="6206246" class="section-title" data-legacy-id=s3d>3.4. Positive selection of genes involved in GSH synthesis</h3>
<p>Microcystins (MCs) are cyclic non-ribosomal peptides produced by cyanobacteria. They are cyanotoxins and can be very toxic to fishes and other animals, including humans. With the increasing frequency of water bloom outbreaks in many countries, the task of eliminating them has become both more urgent and more difficult. Recently, the silver carp and bighead carp have been used to counteract cyanobacteria in many lakes in China.<sup><a reveal-id="DSR046C24" data-open="DSR046C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR046C37" data-open="DSR046C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Despite the hepatotoxicity of MC, the body weights of silver carp increase very fast in bodies of water that are full of MCs.<sup><a reveal-id="DSR046C38" data-open="DSR046C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The high tolerance of silver carp to MCs might be due to the high basic GSH level in the liver or an increased GSH synthesis.<sup><a reveal-id="DSR046C39" data-open="DSR046C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>To fully understand the mechanism behind the high tolerance of silver cap to MCs, we evaluated whether the genes involved in glutathione synthesizing were under positive selection in silver carp. From the KEGG databases, 44 genes involved in glutathione synthesis in zebrafish were obtained with the full CDS region (PATHWAY dre00480). After searching against the whole transcriptome sequences, we found that most of the zebrafish CDS had been recovered (Table <a reveal-id="DSR046TB4" data-open="DSR046TB4" class="link link-reveal link-table xref-fig">4</a>). Sequence pairs were constructed by the zebrafish CDS, and its corresponding best-hit blast scaffolds of silver carp were thereafter tested for whether they had experienced positive selection. For these 44 genes, the average number of codons was found to be 356 (range, 137–966). The F3 × 4 model of codon frequencies was used and models M7 and M8 were used to determine which pairs of sequences were under positive selection. The log-likelihood value under M8 was much higher than its corresponding value under M7, indicating that model M8 is more suitable to the sequence pair compared with model M7. LRT shows that five sequence pairs were found to have <em>P</em>-values &lt;0.05. They are thought to have experienced positive selection between silver carp and zebrafish (Table <a reveal-id="DSR046TB5" data-open="DSR046TB5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="6206249" scrollto-destination="6206249"></a>
<div content-id="DSR046TB4" class="table-modal"><div class="table-wrap"><div id="DSR046TB4" class="table-wrap-title" data-id="DSR046TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Sequences recovered in the glutathione synthesizing pathway</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene id </th><th>Description </th><th>Length </th><th>Matched </th></tr></thead><tbody><tr><td>dre:100002145 </td><td>Gamma-glutamyltranspeptidase </td><td>2082 </td><td>0 </td></tr><tr><td>dre:100006589 </td><td>Isocitrate dehydrogenase 1 (NADP+) </td><td>1290 </td><td>1254 </td></tr><tr><td>dre:100124622 </td><td>Glutathione <em>S</em>-transferase </td><td>672 </td><td>514 </td></tr><tr><td>dre:100330864 </td><td>Ribonucleoside-diphosphate reductase subunit M2-like </td><td>1161 </td><td>1057 </td></tr><tr><td>dre:100333757 </td><td>Gamma-glutamyltransferase 5-like </td><td>1521 </td><td>1162 </td></tr><tr><td>dre:114426 </td><td>Ornithine decarboxylase </td><td>1386 </td><td>1370 </td></tr><tr><td>dre:30733 </td><td>Ribonucleotide reductase M2 polypeptide </td><td>1161 </td><td>1057 </td></tr><tr><td>dre:30740 </td><td>Ribonucleotide reductase M1 polypeptide </td><td>2385 </td><td>2385 </td></tr><tr><td>dre:322533 </td><td>Alanyl (membrane) aminopeptidase b </td><td>2898 </td><td>1238 </td></tr><tr><td>dre:324366 </td><td>Glutathione <em>S</em>-transferase M </td><td>660 </td><td>658 </td></tr><tr><td>dre:324900 </td><td>Protein-disulfide reductase (glutathione) </td><td>519 </td><td>515 </td></tr><tr><td>dre:326857 </td><td>Glutamate-cysteine ligase, catalytic subunit </td><td>1896 </td><td>1896 </td></tr><tr><td>dre:333974 </td><td>Glutamate-cysteine ligase, modifier subunit </td><td>822 </td><td>736 </td></tr><tr><td>dre:352926 </td><td>Glutathione peroxidase 1a </td><td>576 </td><td>565 </td></tr><tr><td>dre:352928 </td><td>Glutathione peroxidase 4a </td><td>561 </td><td>561 </td></tr><tr><td>dre:352929 </td><td>Glutathione peroxidase 4b </td><td>576 </td><td>576 </td></tr><tr><td>dre:386951 </td><td>Isocitrate dehydrogenase 2 (NADP+), mitochondrial </td><td>1350 </td><td>1350 </td></tr><tr><td>dre:394009 </td><td>Spermidine synthase </td><td>870 </td><td>749 </td></tr><tr><td>dre:406278 </td><td>Gamma-glutamylcyclotransferase </td><td>663 </td><td>600 </td></tr><tr><td>dre:406703 </td><td>Glutathione <em>S</em>-transferase, alpha-like </td><td>672 </td><td>571 </td></tr><tr><td>dre:406736 </td><td>Glutathione <em>S</em>-transferase </td><td>453 </td><td>425 </td></tr><tr><td>dre:406762 </td><td>Phosphogluconate hydrogenase </td><td>1536 </td><td>1418 </td></tr><tr><td>dre:431762 </td><td>Glutathione <em>S</em>-transferase </td><td>459 </td><td>451 </td></tr><tr><td>dre:436833 </td><td>Glutathione <em>S</em>-transferase kappa 1 </td><td>690 </td><td>680 </td></tr><tr><td>dre:436894 </td><td>Glutathione <em>S</em>-transferase </td><td>723 </td><td>722 </td></tr><tr><td>dre:449784 </td><td>Microsomal glutathione <em>S</em>-transferase 1 </td><td>465 </td><td>244 </td></tr><tr><td>dre:450084 </td><td>Glutathione synthetase </td><td>1428 </td><td>1385 </td></tr><tr><td>dre:552981 </td><td>Glutathione peroxidase 7 </td><td>561 </td><td>0 </td></tr><tr><td>dre:553169 </td><td>Glutathione <em>S</em>-transferase pi 2 </td><td>627 </td><td>625 </td></tr><tr><td>dre:553575 </td><td>Glutathione reductase (NADPH) </td><td>1278 </td><td>1257 </td></tr><tr><td>dre:555478 </td><td>Aminopeptidase N </td><td>2883 </td><td>763 </td></tr><tr><td>dre:562854 </td><td>Leucine aminopeptidase 3 </td><td>1554 </td><td>1320 </td></tr><tr><td>dre:563972 </td><td>Glutathione <em>S</em>-transferase theta 1a </td><td>729 </td><td>729 </td></tr><tr><td>dre:566746 </td><td>Gamma-glutamyltranspeptidase </td><td>1773 </td><td>82 </td></tr><tr><td>dre:567275 </td><td>Glutathione <em>S</em>-transferase </td><td>423 </td><td>423 </td></tr><tr><td>dre:568744 </td><td>Glutathione <em>S</em>-transferase M3 </td><td>660 </td><td>658 </td></tr><tr><td>dre:569014 </td><td>Gamma-glutamyltranspeptidase 1-like </td><td>1725 </td><td>281 </td></tr><tr><td>dre:570579 </td><td>Glucose-6-phosphate dehydrogenase </td><td>1572 </td><td>1572 </td></tr><tr><td>dre:571365 </td><td>Glutathione <em>S</em>-transferase </td><td>660 </td><td>658 </td></tr><tr><td>dre:723997 </td><td>Microsomal glutathione <em>S</em>-transferase 2 </td><td>411 </td><td>0 </td></tr><tr><td>dre:79381 </td><td>Glutathione <em>S</em>-transferase pi </td><td>627 </td><td>626 </td></tr><tr><td>dre:798788 </td><td>Glutathione peroxidase 3 </td><td>669 </td><td>529 </td></tr><tr><td>dre:799288 </td><td>Glutathione <em>S</em>-transferase </td><td>672 </td><td>512 </td></tr><tr><td>dre:80872 </td><td>Spermine synthase </td><td>1083 </td><td>1057 </td></tr></tbody></table></div><div content-id="DSR046TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene id </th><th>Description </th><th>Length </th><th>Matched </th></tr></thead><tbody><tr><td>dre:100002145 </td><td>Gamma-glutamyltranspeptidase </td><td>2082 </td><td>0 </td></tr><tr><td>dre:100006589 </td><td>Isocitrate dehydrogenase 1 (NADP+) </td><td>1290 </td><td>1254 </td></tr><tr><td>dre:100124622 </td><td>Glutathione <em>S</em>-transferase </td><td>672 </td><td>514 </td></tr><tr><td>dre:100330864 </td><td>Ribonucleoside-diphosphate reductase subunit M2-like </td><td>1161 </td><td>1057 </td></tr><tr><td>dre:100333757 </td><td>Gamma-glutamyltransferase 5-like </td><td>1521 </td><td>1162 </td></tr><tr><td>dre:114426 </td><td>Ornithine decarboxylase </td><td>1386 </td><td>1370 </td></tr><tr><td>dre:30733 </td><td>Ribonucleotide reductase M2 polypeptide </td><td>1161 </td><td>1057 </td></tr><tr><td>dre:30740 </td><td>Ribonucleotide reductase M1 polypeptide </td><td>2385 </td><td>2385 </td></tr><tr><td>dre:322533 </td><td>Alanyl (membrane) aminopeptidase b </td><td>2898 </td><td>1238 </td></tr><tr><td>dre:324366 </td><td>Glutathione <em>S</em>-transferase M </td><td>660 </td><td>658 </td></tr><tr><td>dre:324900 </td><td>Protein-disulfide reductase (glutathione) </td><td>519 </td><td>515 </td></tr><tr><td>dre:326857 </td><td>Glutamate-cysteine ligase, catalytic subunit </td><td>1896 </td><td>1896 </td></tr><tr><td>dre:333974 </td><td>Glutamate-cysteine ligase, modifier subunit </td><td>822 </td><td>736 </td></tr><tr><td>dre:352926 </td><td>Glutathione peroxidase 1a </td><td>576 </td><td>565 </td></tr><tr><td>dre:352928 </td><td>Glutathione peroxidase 4a </td><td>561 </td><td>561 </td></tr><tr><td>dre:352929 </td><td>Glutathione peroxidase 4b </td><td>576 </td><td>576 </td></tr><tr><td>dre:386951 </td><td>Isocitrate dehydrogenase 2 (NADP+), mitochondrial </td><td>1350 </td><td>1350 </td></tr><tr><td>dre:394009 </td><td>Spermidine synthase </td><td>870 </td><td>749 </td></tr><tr><td>dre:406278 </td><td>Gamma-glutamylcyclotransferase </td><td>663 </td><td>600 </td></tr><tr><td>dre:406703 </td><td>Glutathione <em>S</em>-transferase, alpha-like </td><td>672 </td><td>571 </td></tr><tr><td>dre:406736 </td><td>Glutathione <em>S</em>-transferase </td><td>453 </td><td>425 </td></tr><tr><td>dre:406762 </td><td>Phosphogluconate hydrogenase </td><td>1536 </td><td>1418 </td></tr><tr><td>dre:431762 </td><td>Glutathione <em>S</em>-transferase </td><td>459 </td><td>451 </td></tr><tr><td>dre:436833 </td><td>Glutathione <em>S</em>-transferase kappa 1 </td><td>690 </td><td>680 </td></tr><tr><td>dre:436894 </td><td>Glutathione <em>S</em>-transferase </td><td>723 </td><td>722 </td></tr><tr><td>dre:449784 </td><td>Microsomal glutathione <em>S</em>-transferase 1 </td><td>465 </td><td>244 </td></tr><tr><td>dre:450084 </td><td>Glutathione synthetase </td><td>1428 </td><td>1385 </td></tr><tr><td>dre:552981 </td><td>Glutathione peroxidase 7 </td><td>561 </td><td>0 </td></tr><tr><td>dre:553169 </td><td>Glutathione <em>S</em>-transferase pi 2 </td><td>627 </td><td>625 </td></tr><tr><td>dre:553575 </td><td>Glutathione reductase (NADPH) </td><td>1278 </td><td>1257 </td></tr><tr><td>dre:555478 </td><td>Aminopeptidase N </td><td>2883 </td><td>763 </td></tr><tr><td>dre:562854 </td><td>Leucine aminopeptidase 3 </td><td>1554 </td><td>1320 </td></tr><tr><td>dre:563972 </td><td>Glutathione <em>S</em>-transferase theta 1a </td><td>729 </td><td>729 </td></tr><tr><td>dre:566746 </td><td>Gamma-glutamyltranspeptidase </td><td>1773 </td><td>82 </td></tr><tr><td>dre:567275 </td><td>Glutathione <em>S</em>-transferase </td><td>423 </td><td>423 </td></tr><tr><td>dre:568744 </td><td>Glutathione <em>S</em>-transferase M3 </td><td>660 </td><td>658 </td></tr><tr><td>dre:569014 </td><td>Gamma-glutamyltranspeptidase 1-like </td><td>1725 </td><td>281 </td></tr><tr><td>dre:570579 </td><td>Glucose-6-phosphate dehydrogenase </td><td>1572 </td><td>1572 </td></tr><tr><td>dre:571365 </td><td>Glutathione <em>S</em>-transferase </td><td>660 </td><td>658 </td></tr><tr><td>dre:723997 </td><td>Microsomal glutathione <em>S</em>-transferase 2 </td><td>411 </td><td>0 </td></tr><tr><td>dre:79381 </td><td>Glutathione <em>S</em>-transferase pi </td><td>627 </td><td>626 </td></tr><tr><td>dre:798788 </td><td>Glutathione peroxidase 3 </td><td>669 </td><td>529 </td></tr><tr><td>dre:799288 </td><td>Glutathione <em>S</em>-transferase </td><td>672 </td><td>512 </td></tr><tr><td>dre:80872 </td><td>Spermine synthase </td><td>1083 </td><td>1057 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206249" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR046TB4" class="table-wrap-title" data-id="DSR046TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Sequences recovered in the glutathione synthesizing pathway</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene id </th><th>Description </th><th>Length </th><th>Matched </th></tr></thead><tbody><tr><td>dre:100002145 </td><td>Gamma-glutamyltranspeptidase </td><td>2082 </td><td>0 </td></tr><tr><td>dre:100006589 </td><td>Isocitrate dehydrogenase 1 (NADP+) </td><td>1290 </td><td>1254 </td></tr><tr><td>dre:100124622 </td><td>Glutathione <em>S</em>-transferase </td><td>672 </td><td>514 </td></tr><tr><td>dre:100330864 </td><td>Ribonucleoside-diphosphate reductase subunit M2-like </td><td>1161 </td><td>1057 </td></tr><tr><td>dre:100333757 </td><td>Gamma-glutamyltransferase 5-like </td><td>1521 </td><td>1162 </td></tr><tr><td>dre:114426 </td><td>Ornithine decarboxylase </td><td>1386 </td><td>1370 </td></tr><tr><td>dre:30733 </td><td>Ribonucleotide reductase M2 polypeptide </td><td>1161 </td><td>1057 </td></tr><tr><td>dre:30740 </td><td>Ribonucleotide reductase M1 polypeptide </td><td>2385 </td><td>2385 </td></tr><tr><td>dre:322533 </td><td>Alanyl (membrane) aminopeptidase b </td><td>2898 </td><td>1238 </td></tr><tr><td>dre:324366 </td><td>Glutathione <em>S</em>-transferase M </td><td>660 </td><td>658 </td></tr><tr><td>dre:324900 </td><td>Protein-disulfide reductase (glutathione) </td><td>519 </td><td>515 </td></tr><tr><td>dre:326857 </td><td>Glutamate-cysteine ligase, catalytic subunit </td><td>1896 </td><td>1896 </td></tr><tr><td>dre:333974 </td><td>Glutamate-cysteine ligase, modifier subunit </td><td>822 </td><td>736 </td></tr><tr><td>dre:352926 </td><td>Glutathione peroxidase 1a </td><td>576 </td><td>565 </td></tr><tr><td>dre:352928 </td><td>Glutathione peroxidase 4a </td><td>561 </td><td>561 </td></tr><tr><td>dre:352929 </td><td>Glutathione peroxidase 4b </td><td>576 </td><td>576 </td></tr><tr><td>dre:386951 </td><td>Isocitrate dehydrogenase 2 (NADP+), mitochondrial </td><td>1350 </td><td>1350 </td></tr><tr><td>dre:394009 </td><td>Spermidine synthase </td><td>870 </td><td>749 </td></tr><tr><td>dre:406278 </td><td>Gamma-glutamylcyclotransferase </td><td>663 </td><td>600 </td></tr><tr><td>dre:406703 </td><td>Glutathione <em>S</em>-transferase, alpha-like </td><td>672 </td><td>571 </td></tr><tr><td>dre:406736 </td><td>Glutathione <em>S</em>-transferase </td><td>453 </td><td>425 </td></tr><tr><td>dre:406762 </td><td>Phosphogluconate hydrogenase </td><td>1536 </td><td>1418 </td></tr><tr><td>dre:431762 </td><td>Glutathione <em>S</em>-transferase </td><td>459 </td><td>451 </td></tr><tr><td>dre:436833 </td><td>Glutathione <em>S</em>-transferase kappa 1 </td><td>690 </td><td>680 </td></tr><tr><td>dre:436894 </td><td>Glutathione <em>S</em>-transferase </td><td>723 </td><td>722 </td></tr><tr><td>dre:449784 </td><td>Microsomal glutathione <em>S</em>-transferase 1 </td><td>465 </td><td>244 </td></tr><tr><td>dre:450084 </td><td>Glutathione synthetase </td><td>1428 </td><td>1385 </td></tr><tr><td>dre:552981 </td><td>Glutathione peroxidase 7 </td><td>561 </td><td>0 </td></tr><tr><td>dre:553169 </td><td>Glutathione <em>S</em>-transferase pi 2 </td><td>627 </td><td>625 </td></tr><tr><td>dre:553575 </td><td>Glutathione reductase (NADPH) </td><td>1278 </td><td>1257 </td></tr><tr><td>dre:555478 </td><td>Aminopeptidase N </td><td>2883 </td><td>763 </td></tr><tr><td>dre:562854 </td><td>Leucine aminopeptidase 3 </td><td>1554 </td><td>1320 </td></tr><tr><td>dre:563972 </td><td>Glutathione <em>S</em>-transferase theta 1a </td><td>729 </td><td>729 </td></tr><tr><td>dre:566746 </td><td>Gamma-glutamyltranspeptidase </td><td>1773 </td><td>82 </td></tr><tr><td>dre:567275 </td><td>Glutathione <em>S</em>-transferase </td><td>423 </td><td>423 </td></tr><tr><td>dre:568744 </td><td>Glutathione <em>S</em>-transferase M3 </td><td>660 </td><td>658 </td></tr><tr><td>dre:569014 </td><td>Gamma-glutamyltranspeptidase 1-like </td><td>1725 </td><td>281 </td></tr><tr><td>dre:570579 </td><td>Glucose-6-phosphate dehydrogenase </td><td>1572 </td><td>1572 </td></tr><tr><td>dre:571365 </td><td>Glutathione <em>S</em>-transferase </td><td>660 </td><td>658 </td></tr><tr><td>dre:723997 </td><td>Microsomal glutathione <em>S</em>-transferase 2 </td><td>411 </td><td>0 </td></tr><tr><td>dre:79381 </td><td>Glutathione <em>S</em>-transferase pi </td><td>627 </td><td>626 </td></tr><tr><td>dre:798788 </td><td>Glutathione peroxidase 3 </td><td>669 </td><td>529 </td></tr><tr><td>dre:799288 </td><td>Glutathione <em>S</em>-transferase </td><td>672 </td><td>512 </td></tr><tr><td>dre:80872 </td><td>Spermine synthase </td><td>1083 </td><td>1057 </td></tr></tbody></table></div><div content-id="DSR046TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene id </th><th>Description </th><th>Length </th><th>Matched </th></tr></thead><tbody><tr><td>dre:100002145 </td><td>Gamma-glutamyltranspeptidase </td><td>2082 </td><td>0 </td></tr><tr><td>dre:100006589 </td><td>Isocitrate dehydrogenase 1 (NADP+) </td><td>1290 </td><td>1254 </td></tr><tr><td>dre:100124622 </td><td>Glutathione <em>S</em>-transferase </td><td>672 </td><td>514 </td></tr><tr><td>dre:100330864 </td><td>Ribonucleoside-diphosphate reductase subunit M2-like </td><td>1161 </td><td>1057 </td></tr><tr><td>dre:100333757 </td><td>Gamma-glutamyltransferase 5-like </td><td>1521 </td><td>1162 </td></tr><tr><td>dre:114426 </td><td>Ornithine decarboxylase </td><td>1386 </td><td>1370 </td></tr><tr><td>dre:30733 </td><td>Ribonucleotide reductase M2 polypeptide </td><td>1161 </td><td>1057 </td></tr><tr><td>dre:30740 </td><td>Ribonucleotide reductase M1 polypeptide </td><td>2385 </td><td>2385 </td></tr><tr><td>dre:322533 </td><td>Alanyl (membrane) aminopeptidase b </td><td>2898 </td><td>1238 </td></tr><tr><td>dre:324366 </td><td>Glutathione <em>S</em>-transferase M </td><td>660 </td><td>658 </td></tr><tr><td>dre:324900 </td><td>Protein-disulfide reductase (glutathione) </td><td>519 </td><td>515 </td></tr><tr><td>dre:326857 </td><td>Glutamate-cysteine ligase, catalytic subunit </td><td>1896 </td><td>1896 </td></tr><tr><td>dre:333974 </td><td>Glutamate-cysteine ligase, modifier subunit </td><td>822 </td><td>736 </td></tr><tr><td>dre:352926 </td><td>Glutathione peroxidase 1a </td><td>576 </td><td>565 </td></tr><tr><td>dre:352928 </td><td>Glutathione peroxidase 4a </td><td>561 </td><td>561 </td></tr><tr><td>dre:352929 </td><td>Glutathione peroxidase 4b </td><td>576 </td><td>576 </td></tr><tr><td>dre:386951 </td><td>Isocitrate dehydrogenase 2 (NADP+), mitochondrial </td><td>1350 </td><td>1350 </td></tr><tr><td>dre:394009 </td><td>Spermidine synthase </td><td>870 </td><td>749 </td></tr><tr><td>dre:406278 </td><td>Gamma-glutamylcyclotransferase </td><td>663 </td><td>600 </td></tr><tr><td>dre:406703 </td><td>Glutathione <em>S</em>-transferase, alpha-like </td><td>672 </td><td>571 </td></tr><tr><td>dre:406736 </td><td>Glutathione <em>S</em>-transferase </td><td>453 </td><td>425 </td></tr><tr><td>dre:406762 </td><td>Phosphogluconate hydrogenase </td><td>1536 </td><td>1418 </td></tr><tr><td>dre:431762 </td><td>Glutathione <em>S</em>-transferase </td><td>459 </td><td>451 </td></tr><tr><td>dre:436833 </td><td>Glutathione <em>S</em>-transferase kappa 1 </td><td>690 </td><td>680 </td></tr><tr><td>dre:436894 </td><td>Glutathione <em>S</em>-transferase </td><td>723 </td><td>722 </td></tr><tr><td>dre:449784 </td><td>Microsomal glutathione <em>S</em>-transferase 1 </td><td>465 </td><td>244 </td></tr><tr><td>dre:450084 </td><td>Glutathione synthetase </td><td>1428 </td><td>1385 </td></tr><tr><td>dre:552981 </td><td>Glutathione peroxidase 7 </td><td>561 </td><td>0 </td></tr><tr><td>dre:553169 </td><td>Glutathione <em>S</em>-transferase pi 2 </td><td>627 </td><td>625 </td></tr><tr><td>dre:553575 </td><td>Glutathione reductase (NADPH) </td><td>1278 </td><td>1257 </td></tr><tr><td>dre:555478 </td><td>Aminopeptidase N </td><td>2883 </td><td>763 </td></tr><tr><td>dre:562854 </td><td>Leucine aminopeptidase 3 </td><td>1554 </td><td>1320 </td></tr><tr><td>dre:563972 </td><td>Glutathione <em>S</em>-transferase theta 1a </td><td>729 </td><td>729 </td></tr><tr><td>dre:566746 </td><td>Gamma-glutamyltranspeptidase </td><td>1773 </td><td>82 </td></tr><tr><td>dre:567275 </td><td>Glutathione <em>S</em>-transferase </td><td>423 </td><td>423 </td></tr><tr><td>dre:568744 </td><td>Glutathione <em>S</em>-transferase M3 </td><td>660 </td><td>658 </td></tr><tr><td>dre:569014 </td><td>Gamma-glutamyltranspeptidase 1-like </td><td>1725 </td><td>281 </td></tr><tr><td>dre:570579 </td><td>Glucose-6-phosphate dehydrogenase </td><td>1572 </td><td>1572 </td></tr><tr><td>dre:571365 </td><td>Glutathione <em>S</em>-transferase </td><td>660 </td><td>658 </td></tr><tr><td>dre:723997 </td><td>Microsomal glutathione <em>S</em>-transferase 2 </td><td>411 </td><td>0 </td></tr><tr><td>dre:79381 </td><td>Glutathione <em>S</em>-transferase pi </td><td>627 </td><td>626 </td></tr><tr><td>dre:798788 </td><td>Glutathione peroxidase 3 </td><td>669 </td><td>529 </td></tr><tr><td>dre:799288 </td><td>Glutathione <em>S</em>-transferase </td><td>672 </td><td>512 </td></tr><tr><td>dre:80872 </td><td>Spermine synthase </td><td>1083 </td><td>1057 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206249" target="_blank">View Large</a></div></div>                        <a id="6206250" scrollto-destination="6206250"></a>
<div content-id="DSR046TB5" class="table-modal"><div class="table-wrap"><div id="DSR046TB5" class="table-wrap-title" data-id="DSR046TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Genes determined to be under positive selection</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene id </th><th>Model </th><th>Log likelihood </th><th>d<em>N</em>/d<em>S</em> </th><th>Estimates of parameters </th><th>Sites under selection (<em>P</em> &gt; 0.95) </th></tr></thead><tbody><tr><td rowspan="2">dre_322533 </td><td>M7(beta) </td><td>−4530.079761 </td><td>0.3750 </td><td><em>p</em> = 0.00500 and <em>q</em> = 0.00810 </td><td rowspan="2">164,167,256 </td></tr><tr><td>M8(beta and ω) </td><td>−4519.934204 </td><td>0.7805 </td><td><em>p</em><sub>0</sub> = 0.93767, <em>p</em> = 0.04957, <em>q</em> = 0.14698 and <em>w</em> = 8.71914 </td></tr><tr><td>dre_406703 </td><td>M7(beta) </td><td>−1299.159488 </td><td>0.1663 </td><td><em>p</em> = 0.13888 and <em>q</em> = 0.68894 </td><td rowspan="2">No </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1295.384925 </td><td>16.1965 </td><td><em>p</em><sub>0</sub>=0.94248, <em>p</em>=9.04140, <em>q</em> = 99.00000 and <em>w</em> = 280.21751 </td></tr><tr><td>dre_563972 </td><td>M7(beta) </td><td>−1399.109283 </td><td>0.3750 </td><td><em>p</em> = 0.00542 and <em>q</em> = 0.00912 </td><td rowspan="2">224,226,227,233 </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1390.997418 </td><td>2.5343 </td><td><em>p</em><sub>0</sub> = 0.91793, <em>p</em> = 9.32406, <em>q</em> = 32.38965 and <em>w</em> = 28.38710 </td></tr><tr><td>dre_79381 </td><td>M7(beta) </td><td>−1105.063900 </td><td>0.1428 </td><td><em>p</em> = 0.03221 and <em>q</em> = 0.18627 </td><td rowspan="2">129,174 </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1100.372249 </td><td>14.0364 </td><td><em>p</em><sub>0</sub> = 0.97094, <em>p</em> = 7.50265, <em>q</em> = 99.00000 and <em>w</em> = 480.64973 </td></tr><tr><td>dre_799288 </td><td>M7(beta) </td><td>−1334.096144 </td><td>0.1833 </td><td><em>p</em> = 0.12802 and <em>q</em> = 0.56863 </td><td rowspan="2">No </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1328.432914 </td><td>13.0557 </td><td><em>p</em><sub>0</sub> = 0.92145, <em>p</em> = 9.04080, <em>q</em> = 99.00000 and <em>w</em> = 165.23531 </td></tr></tbody></table></div><div content-id="DSR046TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene id </th><th>Model </th><th>Log likelihood </th><th>d<em>N</em>/d<em>S</em> </th><th>Estimates of parameters </th><th>Sites under selection (<em>P</em> &gt; 0.95) </th></tr></thead><tbody><tr><td rowspan="2">dre_322533 </td><td>M7(beta) </td><td>−4530.079761 </td><td>0.3750 </td><td><em>p</em> = 0.00500 and <em>q</em> = 0.00810 </td><td rowspan="2">164,167,256 </td></tr><tr><td>M8(beta and ω) </td><td>−4519.934204 </td><td>0.7805 </td><td><em>p</em><sub>0</sub> = 0.93767, <em>p</em> = 0.04957, <em>q</em> = 0.14698 and <em>w</em> = 8.71914 </td></tr><tr><td>dre_406703 </td><td>M7(beta) </td><td>−1299.159488 </td><td>0.1663 </td><td><em>p</em> = 0.13888 and <em>q</em> = 0.68894 </td><td rowspan="2">No </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1295.384925 </td><td>16.1965 </td><td><em>p</em><sub>0</sub>=0.94248, <em>p</em>=9.04140, <em>q</em> = 99.00000 and <em>w</em> = 280.21751 </td></tr><tr><td>dre_563972 </td><td>M7(beta) </td><td>−1399.109283 </td><td>0.3750 </td><td><em>p</em> = 0.00542 and <em>q</em> = 0.00912 </td><td rowspan="2">224,226,227,233 </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1390.997418 </td><td>2.5343 </td><td><em>p</em><sub>0</sub> = 0.91793, <em>p</em> = 9.32406, <em>q</em> = 32.38965 and <em>w</em> = 28.38710 </td></tr><tr><td>dre_79381 </td><td>M7(beta) </td><td>−1105.063900 </td><td>0.1428 </td><td><em>p</em> = 0.03221 and <em>q</em> = 0.18627 </td><td rowspan="2">129,174 </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1100.372249 </td><td>14.0364 </td><td><em>p</em><sub>0</sub> = 0.97094, <em>p</em> = 7.50265, <em>q</em> = 99.00000 and <em>w</em> = 480.64973 </td></tr><tr><td>dre_799288 </td><td>M7(beta) </td><td>−1334.096144 </td><td>0.1833 </td><td><em>p</em> = 0.12802 and <em>q</em> = 0.56863 </td><td rowspan="2">No </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1328.432914 </td><td>13.0557 </td><td><em>p</em><sub>0</sub> = 0.92145, <em>p</em> = 9.04080, <em>q</em> = 99.00000 and <em>w</em> = 165.23531 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206250" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR046TB5" class="table-wrap-title" data-id="DSR046TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Genes determined to be under positive selection</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene id </th><th>Model </th><th>Log likelihood </th><th>d<em>N</em>/d<em>S</em> </th><th>Estimates of parameters </th><th>Sites under selection (<em>P</em> &gt; 0.95) </th></tr></thead><tbody><tr><td rowspan="2">dre_322533 </td><td>M7(beta) </td><td>−4530.079761 </td><td>0.3750 </td><td><em>p</em> = 0.00500 and <em>q</em> = 0.00810 </td><td rowspan="2">164,167,256 </td></tr><tr><td>M8(beta and ω) </td><td>−4519.934204 </td><td>0.7805 </td><td><em>p</em><sub>0</sub> = 0.93767, <em>p</em> = 0.04957, <em>q</em> = 0.14698 and <em>w</em> = 8.71914 </td></tr><tr><td>dre_406703 </td><td>M7(beta) </td><td>−1299.159488 </td><td>0.1663 </td><td><em>p</em> = 0.13888 and <em>q</em> = 0.68894 </td><td rowspan="2">No </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1295.384925 </td><td>16.1965 </td><td><em>p</em><sub>0</sub>=0.94248, <em>p</em>=9.04140, <em>q</em> = 99.00000 and <em>w</em> = 280.21751 </td></tr><tr><td>dre_563972 </td><td>M7(beta) </td><td>−1399.109283 </td><td>0.3750 </td><td><em>p</em> = 0.00542 and <em>q</em> = 0.00912 </td><td rowspan="2">224,226,227,233 </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1390.997418 </td><td>2.5343 </td><td><em>p</em><sub>0</sub> = 0.91793, <em>p</em> = 9.32406, <em>q</em> = 32.38965 and <em>w</em> = 28.38710 </td></tr><tr><td>dre_79381 </td><td>M7(beta) </td><td>−1105.063900 </td><td>0.1428 </td><td><em>p</em> = 0.03221 and <em>q</em> = 0.18627 </td><td rowspan="2">129,174 </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1100.372249 </td><td>14.0364 </td><td><em>p</em><sub>0</sub> = 0.97094, <em>p</em> = 7.50265, <em>q</em> = 99.00000 and <em>w</em> = 480.64973 </td></tr><tr><td>dre_799288 </td><td>M7(beta) </td><td>−1334.096144 </td><td>0.1833 </td><td><em>p</em> = 0.12802 and <em>q</em> = 0.56863 </td><td rowspan="2">No </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1328.432914 </td><td>13.0557 </td><td><em>p</em><sub>0</sub> = 0.92145, <em>p</em> = 9.04080, <em>q</em> = 99.00000 and <em>w</em> = 165.23531 </td></tr></tbody></table></div><div content-id="DSR046TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene id </th><th>Model </th><th>Log likelihood </th><th>d<em>N</em>/d<em>S</em> </th><th>Estimates of parameters </th><th>Sites under selection (<em>P</em> &gt; 0.95) </th></tr></thead><tbody><tr><td rowspan="2">dre_322533 </td><td>M7(beta) </td><td>−4530.079761 </td><td>0.3750 </td><td><em>p</em> = 0.00500 and <em>q</em> = 0.00810 </td><td rowspan="2">164,167,256 </td></tr><tr><td>M8(beta and ω) </td><td>−4519.934204 </td><td>0.7805 </td><td><em>p</em><sub>0</sub> = 0.93767, <em>p</em> = 0.04957, <em>q</em> = 0.14698 and <em>w</em> = 8.71914 </td></tr><tr><td>dre_406703 </td><td>M7(beta) </td><td>−1299.159488 </td><td>0.1663 </td><td><em>p</em> = 0.13888 and <em>q</em> = 0.68894 </td><td rowspan="2">No </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1295.384925 </td><td>16.1965 </td><td><em>p</em><sub>0</sub>=0.94248, <em>p</em>=9.04140, <em>q</em> = 99.00000 and <em>w</em> = 280.21751 </td></tr><tr><td>dre_563972 </td><td>M7(beta) </td><td>−1399.109283 </td><td>0.3750 </td><td><em>p</em> = 0.00542 and <em>q</em> = 0.00912 </td><td rowspan="2">224,226,227,233 </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1390.997418 </td><td>2.5343 </td><td><em>p</em><sub>0</sub> = 0.91793, <em>p</em> = 9.32406, <em>q</em> = 32.38965 and <em>w</em> = 28.38710 </td></tr><tr><td>dre_79381 </td><td>M7(beta) </td><td>−1105.063900 </td><td>0.1428 </td><td><em>p</em> = 0.03221 and <em>q</em> = 0.18627 </td><td rowspan="2">129,174 </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1100.372249 </td><td>14.0364 </td><td><em>p</em><sub>0</sub> = 0.97094, <em>p</em> = 7.50265, <em>q</em> = 99.00000 and <em>w</em> = 480.64973 </td></tr><tr><td>dre_799288 </td><td>M7(beta) </td><td>−1334.096144 </td><td>0.1833 </td><td><em>p</em> = 0.12802 and <em>q</em> = 0.56863 </td><td rowspan="2">No </td></tr><tr><td> </td><td>M8(beta &amp; ω) </td><td>−1328.432914 </td><td>13.0557 </td><td><em>p</em><sub>0</sub> = 0.92145, <em>p</em> = 9.04080, <em>q</em> = 99.00000 and <em>w</em> = 165.23531 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6206250" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6206251 id="6206251" class="section-title" data-legacy-id=s3e>3.5. Identification of SSR or microsatellites</h3>
<p>Because SSRs or microsatellite markers are used for many animal breeding applications, the 85 796 sequences were analyzed for identification of SSR markers. We obtained 13 327 SSR markers in 9636 sequences with the MISA.<sup><a reveal-id="DSR046C34" data-open="DSR046C34" class="link link-ref link-reveal xref-bibr">34</a></sup> In terms of abundance, mononucleotide repeats were found to be most abundant (7693, 57.3%) followed by dinucleotide repeats (3733, 28.0%) and trinucleotide repeats (1538, 11.5%). Other type of repeat units occurred at &lt;2% each. SSR markers were divided into two groups, perfect SSR markers (only one single repeat motif such as ‘AGC’) and compound SSR markers (composed of two or more SSR markers separated by &lt;100 bp). A total of 1206 (9.0%) compound SSR markers were identified. After excluding the mono-nucleotide repeats, the frequency of an SSR motif was calculated. Among the dinucleotide repeat motifs, AC/GT was the most abundant, with 69.2%; trinucleotide repeat motifs were rich in ATC/GAT, with 27.8%, and tetranucleotide repeat motifs were AGAT/ATCT, with 21.1%.</p>                            <h2 scrollto-destination=6206253 id="6206253" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The transcriptome is the complete repertoire of expressed RNA transcripts in the cell and its characterization is essential to understanding the functional complexity of the genome. Using the next-generation sequencing technology, we were able to sequence and annotate the transcriptome of silver carp. This is most comprehensive study of silver carp transcriptome data to date. The transcriptome sequences obtained by this study are useful to the understanding of the genetic makeup of the silver carp transcriptome, which until now has been very limited. The Illumina sequencing yielded 17.10 million paired-end reads for silver carp. The 85 769 sequences produced here may be useful for further research into silver carp functional genomics. The obtained overall GC content of the silver carp transcriptome was 39.2%, which was lower than the GC content of cDNA library of zebrafish (Ensembl 61).<sup><a reveal-id="DSR046C31" data-open="DSR046C31" class="link link-ref link-reveal xref-bibr">31</a></sup> However, when we removed the assembled sequences that contained gaps, the GC content rose to 45.5%, which was similar to that of the zebrafish cDNA library (46.2%).</p><p>Due to the lack of a complete genome sequence, the quality of transcriptome analysis of non-model species must rely largely on the contigs and scaffolds assembled from the raw reads. After reassembling the transcriptomes of two mosquitoes with known genomes using a de novo assembler, Gibbons et al.<sup><a reveal-id="DSR046C40" data-open="DSR046C40" class="link link-ref link-reveal xref-bibr">40</a></sup> found that short reads can be used to assemble transcriptomes of non-model organisms. Although the development of the short-read assembler<sup><a reveal-id="DSR046C28" data-open="DSR046C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR046C41" data-open="DSR046C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSR046C42" data-open="DSR046C42" class="link link-ref link-reveal xref-bibr">42</a></sup> has rendered research facilities capable of dealing with more and more reads, de novo assembly of transcriptomes without known reference genome using short reads is still difficult for transcripts with highly variable coverage.<sup><a reveal-id="DSR046C43" data-open="DSR046C43" class="link link-ref link-reveal xref-bibr">43</a></sup> So, a higher k-mer length will theoretically generate a more contiguous assembly of highly expressed RNAs while poorly expressed transcripts will be more easily obtained if a lower k-mer length is used.<sup><a reveal-id="DSR046C41" data-open="DSR046C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Therefore, an approach for de novo assembly of the transcriptome using various k-mer lengths is highly desirable and has been proven useful.<sup><a reveal-id="DSR046C27" data-open="DSR046C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The final assembly statistics indicate that the multiple k-mer method used in this study outperforms all other single k-mer methods (Table <a reveal-id="DSR046TB1" data-open="DSR046TB1" class="link link-reveal link-table xref-fig">1</a>). In the single k-mer assembly, the average length and N50 were highest when the k-mer was set to 46, which we found to be best in all single k-mer assemblies. However, the number of contigs &gt;100 bp and total length were twice that of the best single k-mer ABySS assembly. This marked increase was accompanied by a higher N50 and average contig size, indicating a substantial improvement in contiguity.</p><p>Multiple k-mer methods assembled the Illumina reads into contigs, but the location information in paired-end reads were not used at all. In this study, we improved upon the methods described by Yann et al.<sup><a reveal-id="DSR046C27" data-open="DSR046C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Results proved that using SSPACE to scaffold the contigs produced by each k-mer could produce longer sequences. The statistics before and after scaffolding (Tables <a reveal-id="DSR046TB1" data-open="DSR046TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSR046TB2" data-open="DSR046TB2" class="link link-reveal link-table xref-fig">2</a>) indicate that a higher average length of sequence can be obtained by joining the two contigs originating from the two ends of a DNA fragment. The max length and the average length of the sequences after scaffolding were further extended. To assess the quality of assembled transcriptome, we used both computing and experimenting assays to validate the transcripts generated. The RT–PCR results confirmed that our method is reliable for the recovery of both highly and poorly expressed transcripts.</p><p>Both gene annotation and KEGG pathway analyses are useful for us to predict potential genes and their functions at a whole-transcriptome level. In the silver carp transcriptome, as discovered by this study, the predominant gene clusters are involved in the structural formation of the cell, cell part and organelle of a cellular component, the binding and catalytic activity of molecular function, metabolic process and cellular processes of biological processes. Similar results were found in <em>Sus scrofa</em>,<sup><a reveal-id="DSR046C44" data-open="DSR046C44" class="link link-ref link-reveal xref-bibr">44</a></sup> European eel<sup><a reveal-id="DSR046C21" data-open="DSR046C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and rainbow trout.<sup><a reveal-id="DSR046C45" data-open="DSR046C45" class="link link-ref link-reveal xref-bibr">45</a></sup> However, in Chickpea transcriptome, sequences were found to be mainly involved in the protein metabolism of biological process, in chloroplasts, in the transferase activity of molecular function. This suggests remarkable difference between animal and plants. KEGG analysis showed that more than 44.5% of transcripts to be enrichment factors involved in 218 known metabolic or signaling pathways, including cell adherence, migration, apoptosis and immune-related processes. The KEGG pathway analysis and gene annotation may be useful for further investigation of gene function in future. Although there are differences between our silver carp transcriptome and available database for zebrafish in GO annotations, concordance in the overall patterns suggests that our library were widely sampled and provided a good representation.</p><p>One previous study reported that GSH can conjugate with MC on its sulfhydryl, which is the first step in the detoxification of a cyanobacterial toxin in aquatic organisms.<sup><a reveal-id="DSR046C46" data-open="DSR046C46" class="link link-ref link-reveal xref-bibr">46</a></sup> The glutathione S-transferase (GST) gene plays important roles both in the biosynthesis of GST and catalysis of the reaction between GSH and MC. M8 assumes 11 site classes: 10 classes for the beta distribution and 1 class for the positively selected site. Therefore, it is suitable to detect positive selection in sequence pairs. Although the value of d<em>N</em>/d<em>S</em> observed in this model might not be precise, the LRT is most likely reliable. Positive selection pressure on <em>GST</em> gene might be the result of the adaptation of silver carp to the eutrophied bodies of water in the middle and lower reaches of the Yangtze River.</p><p>Genetic markers are of great importance to the understanding genetic variation and to the identification of genes and quantitative trait locus for traits of interested in molecular breeding applications. Until now, only a small number of genetic markers have been available for silver carp.<sup><a reveal-id="DSR046C47" data-open="DSR046C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSR046C48" data-open="DSR046C48" class="link link-ref link-reveal xref-bibr">48</a></sup> One of the main reasons for this is the lack of genome sequence information. Alternatively, transcriptomes have been used for the discovery of genetic markers.<sup><a reveal-id="DSR046C44" data-open="DSR046C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSR046C49" data-open="DSR046C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSR046C50" data-open="DSR046C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Although markers developed from transcriptomes are less polymorphic, they have been found to be very useful in trait mapping<sup><a reveal-id="DSR046C51" data-open="DSR046C51" class="link link-ref link-reveal xref-bibr">51</a></sup> and comparative genomics studies.<sup><a reveal-id="DSR046C52" data-open="DSR046C52" class="link link-ref link-reveal xref-bibr">52</a></sup></p><p>It has been reported that SSRs comprise 3% of the human genome, with the largest proportion of them being dinucleotide repeats (0.5%).<sup><a reveal-id="DSR046C53" data-open="DSR046C53" class="link link-ref link-reveal xref-bibr">53</a></sup> In this study, 28% of the 13 327 silver carp SSRs were found to be dinucleotide repeats, followed by trinucleotide repeats (11.5%). The most common dinucleotide repeats were AC and AG, in contrast to those found in the human genome (AC and AT). The same difference was also found in trinucleotide repeats, with ATC and AGG being most common in silver carp and AAT and AAC being most common in human.<sup><a reveal-id="DSR046C53" data-open="DSR046C53" class="link link-ref link-reveal xref-bibr">53</a></sup></p><p>In conclusion, we have determined the transcriptome of silver carp through use of high-throughput Illumina paired-end sequencing. Our study obtained 85 759 scaffolds and demonstrated some important features of silver carp transcriptome, such as gene annotation and KEGG pathway analysis, as shown by cross-transcriptome analysis. In addition, we identified reliable genetic markers for 13 324 SSRs. We also found that five genes identified as under positive selection between silver carp and zebrafish. This study will be helpful for improvement of the understanding of the recent speciation and adaption of Cyprinidae and provides useful resources and markers for future functional genomic research.</p>                            <h2 scrollto-destination=6206262 id="6206262" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6206264 id="6206264" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This research was supported by the grants from National Basic Research Program of China (973 Program 2010CB126302), National Natural Science Foundation of China (31090254 and U1036603) and Chinese Academy of Sciences (KSCX2-EW-Q-12).</p>                            <h2 scrollto-destination=6206266 id="6206266" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR046C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rockman</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Kruglyak</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Genetics of global gene expression</div>, <div class="source">Nat. Rev.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">862</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20global%20gene%20expression&amp;author=M.V.%20Rockman&amp;author=L.%20Kruglyak&amp;publication_year=2006&amp;journal=Nat.%20Rev.&amp;volume=7&amp;pages=862-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1964" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1964" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1964"> </span></div> </div></div></div></div></div><div content-id="DSR046C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Next-generation DNA sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1135</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20DNA%20sequencing&amp;author=J.%20Shendure&amp;author=H.%20Ji&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=1135-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18846087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fullwood</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">E.T.</div></div>,  <div class="name"><div class="surname">Ruan</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Next-generation DNA sequencing of paired-end tags (PET) for transcriptome and genome analyses</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">521</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20DNA%20sequencing%20of%20paired-end%20tags%20(PET)%20for%20transcriptome%20and%20genome%20analyses&amp;author=M.J.%20Fullwood&amp;author=C.L.%20Wei&amp;author=E.T.%20Liu&amp;author=Y.%20Ruan&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=521-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074906.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074906.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074906.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19339662" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berger</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Levin</div> <div class="given-names">J.Z.</div></div>,  <div class="name"><div class="surname">Vijayendran</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Integrative analysis of the melanoma transcriptome</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">413</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20analysis%20of%20the%20melanoma%20transcriptome&amp;author=M.F.%20Berger&amp;author=J.Z.%20Levin&amp;author=K.%20Vijayendran&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=413-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.103697.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.103697.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.103697.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20179022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Deep RNA sequencing at single base-pair resolution reveals high complexity of the rice transcriptome</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">646</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20RNA%20sequencing%20at%20single%20base-pair%20resolution%20reveals%20high%20complexity%20of%20the%20rice%20transcriptome&amp;author=G.%20Zhang&amp;author=G.%20Guo&amp;author=X.%20Hu&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=646-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.100677.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.100677.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.100677.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20305017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Function annotation of the rice transcriptome at single-nucleotide resolution by RNA-seq</div>, <div class="source">Genome Res</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1238</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Function%20annotation%20of%20the%20rice%20transcriptome%20at%20single-nucleotide%20resolution%20by%20RNA-seq&amp;author=T.%20Lu&amp;author=G.%20Lu&amp;author=D.%20Fan&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1238-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.106120.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.106120.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.106120.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20627892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hillier</div> <div class="given-names">L.W.</div></div>,  <div class="name"><div class="surname">Reinke</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hirst</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Marra</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Waterston</div> <div class="given-names">R.H.</div></div>. </div><div class="article-title">Massively parallel sequencing of the polyadenylated transcriptome of <em>C. elegans</em></div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">657</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massively%20parallel%20sequencing%20of%20the%20polyadenylated%20transcriptome%20of%20C.%20elegans&amp;author=L.W.%20Hillier&amp;author=V.%20Reinke&amp;author=P.%20Green&amp;author=M.%20Hirst&amp;author=M.A.%20Marra&amp;author=R.H.%20Waterston&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=657-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.088112.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.088112.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.088112.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19181841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wurtzel</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Sapra</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Simmons</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A single-base resolution map of an archaeal transcriptome</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">133</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20single-base%20resolution%20map%20of%20an%20archaeal%20transcriptome&amp;author=O.%20Wurtzel&amp;author=R.%20Sapra&amp;author=F.%20Chen&amp;author=Y.%20Zhu&amp;author=B.A.%20Simmons&amp;author=R.%20Sorek&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=133-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.100396.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.100396.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.100396.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19884261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cloonan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Forrest</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Kolle</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Stem cell transcriptome profiling via massive-scale mRNA sequencing</div>, <div class="source">Nat Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">613</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stem%20cell%20transcriptome%20profiling%20via%20massive-scale%20mRNA%20sequencing&amp;author=N.%20Cloonan&amp;author=A.R.%20Forrest&amp;author=G.%20Kolle&amp;publication_year=2008&amp;journal=Nat%20Methods&amp;volume=5&amp;pages=613-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18516046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lister</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">O'Malley</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Tonti-Filippini</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Highly integrated single-base resolution maps of the epigenome in <em>Arabidopsis</em></div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">523</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20integrated%20single-base%20resolution%20maps%20of%20the%20epigenome%20in%20Arabidopsis&amp;author=R.%20Lister&amp;author=R.C.%20O'Malley&amp;author=J.%20Tonti-Filippini&amp;publication_year=2008&amp;journal=Cell&amp;volume=133&amp;pages=523-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.03.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.03.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.03.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18423832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Shai</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Frey</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing</div>, <div class="source">Nat. Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">1413</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20surveying%20of%20alternative%20splicing%20complexity%20in%20the%20human%20transcriptome%20by%20high-throughput%20sequencing&amp;author=Q.%20Pan&amp;author=O.%20Shai&amp;author=L.J.%20Lee&amp;author=J.%20Frey&amp;author=B.J.%20Blencowe&amp;publication_year=2008&amp;journal=Nat.%20Genetics&amp;volume=40&amp;pages=1413-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.259"> </span></div> </div></div></div></div></div><div content-id="DSR046C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sultan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schulz</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Richard</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">A global view of gene activity and alternative splicing by deep sequencing of the human transcriptome</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">321</div> (pg. <div class="fpage">956</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20global%20view%20of%20gene%20activity%20and%20alternative%20splicing%20by%20deep%20sequencing%20of%20the%20human%20transcriptome&amp;author=M.%20Sultan&amp;author=M.H.%20Schulz&amp;author=H.%20Richard&amp;publication_year=2008&amp;journal=Science&amp;volume=321&amp;pages=956-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1160342" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1160342" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1160342"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18599741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johansen</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Karlsen</div> <div class="given-names">B.O.</div></div>,  <div class="name"><div class="surname">Furmanek</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">RNA deep sequencing of the Atlantic cod transcriptome</div>, <div class="source">Comp Biochem Physiol</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">18</div>-<div class="lpage">22</div>)</div></div></div></div><div content-id="DSR046C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xiang</div> <div class="given-names">L.X.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.W.</div></div>,  <div class="name"><div class="surname">Shao</div> <div class="given-names">J.Z.</div></div>. </div><div class="article-title">Deep sequencing-based transcriptome profiling analysis of bacteria-challenged <em>Lateolabrax japonicus</em> reveals insight into the immune-relevant genes in marine fish</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">472</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20sequencing-based%20transcriptome%20profiling%20analysis%20of%20bacteria-challenged%20Lateolabrax%20japonicus%20reveals%20insight%20into%20the%20immune-relevant%20genes%20in%20marine%20fish&amp;author=L.X.%20Xiang&amp;author=D.%20He&amp;author=W.R.%20Dong&amp;author=Y.W.%20Zhang&amp;author=J.Z.%20Shao&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=472" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-472" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-472" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-472"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20707909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">J.S.</div></div>. </div>, <div class="source">Fishes of the World</div> , <div class="year">2006</div> <div class="comment">John Wiley: Hoboken, N.J.</div></div></div></div></div><div content-id="DSR046C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aparicio</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chapman</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Stupka</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Whole-genome shotgun assembly and analysis of the genome of <em>Fugu rubripes</em></div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">297</div> (pg. <div class="fpage">1301</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20shotgun%20assembly%20and%20analysis%20of%20the%20genome%20of%20Fugu%20rubripes&amp;author=S.%20Aparicio&amp;author=J.%20Chapman&amp;author=E.%20Stupka&amp;publication_year=2002&amp;journal=Science&amp;volume=297&amp;pages=1301-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1072104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1072104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1072104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12142439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaillon</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Brunet</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Genome duplication in the teleost fish <em>Tetraodon nigroviridis</em> reveals the early vertebrate proto-karyotype</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">431</div> (pg. <div class="fpage">946</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20duplication%20in%20the%20teleost%20fish%20Tetraodon%20nigroviridis%20reveals%20the%20early%20vertebrate%20proto-karyotype&amp;author=O.%20Jaillon&amp;author=J.M.%20Aury&amp;author=F.%20Brunet&amp;publication_year=2004&amp;journal=Nature&amp;volume=431&amp;pages=946-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15496914" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kasahara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Naruse</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The medaka draft genome and insights into vertebrate genome evolution</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">447</div> (pg. <div class="fpage">714</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20medaka%20draft%20genome%20and%20insights%20into%20vertebrate%20genome%20evolution&amp;author=M.%20Kasahara&amp;author=K.%20Naruse&amp;author=S.%20Sasaki&amp;publication_year=2007&amp;journal=Nature&amp;volume=447&amp;pages=714-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05846" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05846" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05846"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17554307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Star</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Nederbragt</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Jentoft</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome sequence of Atlantic cod reveals a unique immune system</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">477</div> (pg. <div class="fpage">207</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Atlantic%20cod%20reveals%20a%20unique%20immune%20system&amp;author=B.%20Star&amp;author=A.J.%20Nederbragt&amp;author=S.%20Jentoft&amp;publication_year=2011&amp;journal=Nature&amp;volume=477&amp;pages=207-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10342" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10342" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10342"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21832995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fraser</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Weadick</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Janowitz</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Rodd</div> <div class="given-names">F.H.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">K.A.</div></div>. </div><div class="article-title">Sequencing and characterization of the guppy (<em>Poecilia reticulata</em>) transcriptome</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">202</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20characterization%20of%20the%20guppy%20(Poecilia%20reticulata)%20transcriptome&amp;author=B.A.%20Fraser&amp;author=C.J.%20Weadick&amp;author=I.%20Janowitz&amp;author=F.H.%20Rodd&amp;author=K.A.%20Hughes&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21507250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coppe</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pujolar</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Maes</div> <div class="given-names">G.E.</div></div>, et al. </div><div class="article-title">Sequencing, de novo annotation and analysis of the first <em>Anguilla anguilla</em> transcriptome: EeelBase opens new perspectives for the study of the critically endangered European eel</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">635</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%2C%20de%20novo%20annotation%20and%20analysis%20of%20the%20first%20Anguilla%20anguilla%20transcriptome%3A%20EeelBase%20opens%20new%20perspectives%20for%20the%20study%20of%20the%20critically%20endangered%20European%20eel&amp;author=A.%20Coppe&amp;author=J.M.%20Pujolar&amp;author=G.E.%20Maes&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=635" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21080939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Molecular evidence for the monophyly of East Asian groups of Cyprinidae (Teleostei: Cypriniformes) derived from the nuclear recombination activating gene 2 sequences</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">157</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evidence%20for%20the%20monophyly%20of%20East%20Asian%20groups%20of%20Cyprinidae%20(Teleostei%3A%20Cypriniformes)%20derived%20from%20the%20nuclear%20recombination%20activating%20gene%202%20sequences&amp;author=X.%20Wang&amp;author=J.%20Li&amp;author=S.%20He&amp;publication_year=2007&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=42&amp;pages=157-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2006.06.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2006.06.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2006.06.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16919973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="collab">Bureau of Fishery</div><div class="collab">Ministry of Agriculture</div>, <div class="source">China Fishery Statistical Yearbook</div> , <div class="year">2010</div><div class="publisher-loc">Beijing</div><div class="publisher-name">China Agriculture Press</div></div></div></div></div><div content-id="DSR046C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Practical success of biomanipulation using filter-feeding Fish to control cyanobacteria blooms: a synthesis of decades of research and application in a subtropical hypereutrophic lake</div>, <div class="source">Scientific World J.</div> , <div class="year">2001</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">337</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Practical%20success%20of%20biomanipulation%20using%20filter-feeding%20Fish%20to%20control%20cyanobacteria%20blooms%3A%20a%20synthesis%20of%20decades%20of%20research%20and%20application%20in%20a%20subtropical%20hypereutrophic%20lake&amp;author=P.%20Xie&amp;author=J.%20Liu&amp;publication_year=2001&amp;journal=Scientific%20World%20J.&amp;volume=1&amp;pages=337-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1100/tsw.2001.67" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1100%2Ftsw.2001.67" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1100%2Ftsw.2001.67"> </span></div> </div></div></div></div></div><div content-id="DSR046C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Song</div> <div class="given-names">W.Q.</div></div>,  <div class="name"><div class="surname">Libing</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Jinxin</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Large enclosures experimental study on algal control by silver carp and bighead</div>, <div class="source">Chinese Environ. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">1190</div>-<div class="lpage">5</div>)</div></div></div></div><div content-id="DSR046C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tao</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Zou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Gan</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Mayden</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Phylogenomic analysis resolves the formerly intractable adaptive diversification of the endemic clade of east Asian Cyprinidae (Cypriniformes)</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e13508</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic%20analysis%20resolves%20the%20formerly%20intractable%20adaptive%20diversification%20of%20the%20endemic%20clade%20of%20east%20Asian%20Cyprinidae%20(Cypriniformes)&amp;author=W.%20Tao&amp;author=M.%20Zou&amp;author=X.%20Wang&amp;author=X.%20Gan&amp;author=R.%20L.%20Mayden&amp;author=S.%20He&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e13508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0013508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0013508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0013508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20976012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yann</div> <div class="given-names">Surget-Groba</div></div>,  <div class="name"><div class="surname">Montoya-Burgos</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">Optimization of de novo transcriptome assembly from next-generation sequencing data</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1432</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimization%20of%20de%20novo%20transcriptome%20assembly%20from%20next-generation%20sequencing%20data&amp;author=Surget-Groba%20Yann&amp;author=J.I.%20Montoya-Burgos&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1432-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.103846.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.103846.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.103846.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20693479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simpson</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jackman</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Schein</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Birol</div> <div class="given-names">I.</div></div>. </div><div class="article-title">ABySS: a parallel assembler for short read sequence data</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1117</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ABySS%3A%20a%20parallel%20assembler%20for%20short%20read%20sequence%20data&amp;author=J.T.%20Simpson&amp;author=K.%20Wong&amp;author=S.D.%20Jackman&amp;author=J.E.%20Schein&amp;author=S.J.%20Jones&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1117-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089532.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089532.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089532.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19251739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Henkel</div> <div class="given-names">C.V.</div></div>,  <div class="name"><div class="surname">Jansen</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">578</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1658</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cd-hit%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences&amp;author=W.%20Li&amp;author=A.%20Godzik&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1658-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flicek</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Amode</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Barrell</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Ensembl 2011</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">D800</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensembl%202011&amp;author=P.%20Flicek&amp;author=M.R.%20Amode&amp;author=D.%20Barrell&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=D800-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1064" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1064" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1064"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21045057" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">High-throughput functional annotation and data mining with the Blast2GO suite</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">3420</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20functional%20annotation%20and%20data%20mining%20with%20the%20Blast2GO%20suite&amp;author=S.%20Gotz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=3420-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18445632" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thiel</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Michalek</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Exploiting EST databases for the development and characterization of gene-derived SSR-markers in barley (<em>Hordeum vulgare</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">411</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploiting%20EST%20databases%20for%20the%20development%20and%20characterization%20of%20gene-derived%20SSR-markers%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;author=T.%20Thiel&amp;author=W.%20Michalek&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=411-22" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12589540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1586</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kristiansen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>. </div><div class="article-title">SOAP: short oligonucleotide alignment program</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">713</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP%3A%20short%20oligonucleotide%20alignment%20program&amp;author=R.%20Li&amp;author=Y.%20Li&amp;author=K.%20Kristiansen&amp;author=J.%20Wang&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=713-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18227114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ke</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">In situ study on the control of toxic microcystis blooms using phytoplanktivorous fish in the subtropical Lake Taihu of China: a large fish pen experiment</div>, <div class="source">Aquaculture</div> , <div class="year">2007</div>, vol. <div class="volume">265</div> (pg. <div class="fpage">127</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20situ%20study%20on%20the%20control%20of%20toxic%20microcystis%20blooms%20using%20phytoplanktivorous%20fish%20in%20the%20subtropical%20Lake%20Taihu%20of%20China%3A%20a%20large%20fish%20pen%20experiment&amp;author=Z.%20Ke&amp;author=P.%20Xie&amp;author=L.%20Guo&amp;author=Y.%20Liu&amp;author=H.%20Yang&amp;publication_year=2007&amp;journal=Aquaculture&amp;volume=265&amp;pages=127-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2007.01.049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2007.01.049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2007.01.049"> </span></div> </div></div></div></div></div><div content-id="DSR046C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Ke</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">In situ studies on the bioaccumulation of microcystins in the phytoplanktivorous silver carp (<em>Hypophthalmichthys molitrix</em>) stocked in Lake Taihu with dense toxic microcystis blooms</div>, <div class="source">Aquaculture</div> , <div class="year">2006</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">1026</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20situ%20studies%20on%20the%20bioaccumulation%20of%20microcystins%20in%20the%20phytoplanktivorous%20silver%20carp%20(Hypophthalmichthys%20molitrix)%20stocked%20in%20Lake%20Taihu%20with%20dense%20toxic%20microcystis%20blooms&amp;author=J.%20Chen&amp;author=P.%20Xie&amp;author=D.%20Zhang&amp;author=Z.%20Ke&amp;author=H.%20Yang&amp;publication_year=2006&amp;journal=Aquaculture&amp;volume=261&amp;pages=1026-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2006.08.028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2006.08.028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2006.08.028"> </span></div> </div></div></div></div></div><div content-id="DSR046C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Qiu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Sequential ultrastructural and biochemical changes induced in vivo by the hepatotoxic microcystins in liver of the phytoplanktivorous silver carp <em>Hypophthalmichthys molitrix</em></div>, <div class="source">Comp. Biochem. Physiol. C Toxicol. Pharmacol.</div> , <div class="year">2007</div>, vol. <div class="volume">146</div> (pg. <div class="fpage">357</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequential%20ultrastructural%20and%20biochemical%20changes%20induced%20in%20vivo%20by%20the%20hepatotoxic%20microcystins%20in%20liver%20of%20the%20phytoplanktivorous%20silver%20carp%20Hypophthalmichthys%20molitrix&amp;author=L.%20Li&amp;author=P.%20Xie&amp;author=S.%20Li&amp;author=T.%20Qiu&amp;author=L.%20Guo&amp;publication_year=2007&amp;journal=Comp.%20Biochem.%20Physiol.%20C%20Toxicol.%20Pharmacol.&amp;volume=146&amp;pages=357-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cbpc.2007.04.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cbpc.2007.04.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cbpc.2007.04.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17574927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gibbons</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Janson</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Hittinger</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Johnston</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Abbot</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rokas</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Benchmarking next-generation transcriptome sequencing for functional and evolutionary genomics</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">2731</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Benchmarking%20next-generation%20transcriptome%20sequencing%20for%20functional%20and%20evolutionary%20genomics&amp;author=J.G.%20Gibbons&amp;author=E.M.%20Janson&amp;author=C.T.%20Hittinger&amp;author=M.%20Johnston&amp;author=P.%20Abbot&amp;author=A.%20Rokas&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=26&amp;pages=2731-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19706727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">821</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Warren</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Sutton</div> <div class="given-names">G.G.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Holt</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Assembling millions of short DNA sequences using SSAKE</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">500</div>-<div class="lpage">1</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembling%20millions%20of%20short%20DNA%20sequences%20using%20SSAKE&amp;author=R.L.%20Warren&amp;author=G.G.%20Sutton&amp;author=S.J.%20Jones&amp;author=R.A.%20Holt&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=500-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl629" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl629" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl629"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17158514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schuster</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Next-generation sequencing transforms today's biology</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">16</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20transforms%20today's%20biology&amp;author=S.C.%20Schuster&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=16-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18165802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nie</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jia</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Analysis of muscle and ovary transcriptome of <em>Sus scrofa</em>: assembly, annotation and marker discovery</div>, <div class="source">DNA Res</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">343</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20muscle%20and%20ovary%20transcriptome%20of%20Sus%20scrofa%3A%20assembly%2C%20annotation%20and%20marker%20discovery&amp;author=Q.%20Nie&amp;author=M.%20Fang&amp;author=X.%20Jia&amp;publication_year=2011&amp;journal=DNA%20Res&amp;volume=18&amp;pages=343-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21729922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salem</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rexroad</div> <div class="given-names">C.E.</div>3rd</div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Thorgaard</div> <div class="given-names">G.H.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Characterization of the rainbow trout transcriptome using Sanger and 454-pyrosequencing approaches</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">564</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20rainbow%20trout%20transcriptome%20using%20Sanger%20and%20454-pyrosequencing%20approaches&amp;author=M.%20Salem&amp;author=C.E.%20Rexroad&amp;author=J.%20Wang&amp;author=G.H.%20Thorgaard&amp;author=J.%20Yao&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=564" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-564" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-564" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-564"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20942956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pflugmacher</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wiegand</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Oberemm</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Identification of an enzymatically formed glutathione conjugate of the cyanobacterial hepatotoxin microcystin-LR: the first step of detoxication</div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="year">1998</div>, vol. <div class="volume">1425</div> (pg. <div class="fpage">527</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20an%20enzymatically%20formed%20glutathione%20conjugate%20of%20the%20cyanobacterial%20hepatotoxin%20microcystin-LR%3A%20the%20first%20step%20of%20detoxication&amp;author=S.%20Pflugmacher&amp;author=C.%20Wiegand&amp;author=A.%20Oberemm&amp;publication_year=1998&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1425&amp;pages=527-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0304-4165(98)00107-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0304-4165(98)00107-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0304-4165(98)00107-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9838216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Zou</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Construction of a genetic linkage map for silver carp (<em>Hypophthalmichthys molitrix</em>)</div>, <div class="source">Anim. Genet</div> , <div class="year">2010</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">523</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20for%20silver%20carp%20(Hypophthalmichthys%20molitrix)&amp;author=L.%20Zhang&amp;author=G.%20Yang&amp;author=S.%20Guo&amp;author=Q.%20Wei&amp;author=G.%20Zou&amp;publication_year=2010&amp;journal=Anim.%20Genet&amp;volume=41&amp;pages=523-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2010.41.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2010.41.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2010.41.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20331598" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Development of silver carp (<em>Hypophthalmichthys molitrix</em>) and bighead carp (<em>Aristichthys nobilis</em>) genetic maps using microsatellite and AFLP markers and a pseudo-testcross strategy</div>, <div class="source">Anim. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">364</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20silver%20carp%20(Hypophthalmichthys%20molitrix)%20and%20bighead%20carp%20(Aristichthys%20nobilis)%20genetic%20maps%20using%20microsatellite%20and%20AFLP%20markers%20and%20a%20pseudo-testcross%20strategy&amp;author=M.%20Liao&amp;author=L.%20Zhang&amp;author=G.%20Yang&amp;publication_year=2007&amp;journal=Anim.%20Genet.&amp;volume=38&amp;pages=364-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2007.38.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2007.38.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2007.38.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17614988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dubey</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Farmer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Defining the transcriptome assembly and its use for genome dynamics and transcriptome profiling studies in pigeonpea (<em>Cajanus cajan</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">153</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20transcriptome%20assembly%20and%20its%20use%20for%20genome%20dynamics%20and%20transcriptome%20profiling%20studies%20in%20pigeonpea%20(Cajanus%20cajan%20L.)&amp;author=A.%20Dubey&amp;author=A.%20Farmer&amp;author=J.%20Schlueter&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=153-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21565938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dutta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kumawat</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">B.P.</div></div>, et al. </div><div class="article-title">Development of genic-SSR markers by deep transcriptome sequencing in pigeonpea [<em>Cajanus cajan</em> (L.) Millspaugh]</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">17</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genic-SSR%20markers%20by%20deep%20transcriptome%20sequencing%20in%20pigeonpea%20%5BCajanus%20cajan%20(L.)%20Millspaugh%5D&amp;author=S.%20Dutta&amp;author=G.%20Kumawat&amp;author=B.P.%20Singh&amp;publication_year=2011&amp;journal=BMC%20Plant%20Biol.&amp;volume=11&amp;pages=17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21251263" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">W.K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.J.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">G.Z.</div></div>, et al. </div><div class="article-title">QTL mapping of ten agronomic traits on the soybean (<em>Glycine max</em> L. Merr.) genetic map and their association with EST markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">1131</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20of%20ten%20agronomic%20traits%20on%20the%20soybean%20(Glycine%20max%20L.%20Merr.)%20genetic%20map%20and%20their%20association%20with%20EST%20markers&amp;author=W.K.%20Zhang&amp;author=Y.J.%20Wang&amp;author=G.Z.%20Luo&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=1131-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1527-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1527-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1527-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15067400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stein</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Blasiar</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">The genome sequence of <em>Caenorhabditis briggsae</em>: a platform for comparative genomics</div>, <div class="source">PLoS Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">1</div> pg. <div class="fpage">E45</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Caenorhabditis%20briggsae%3A%20a%20platform%20for%20comparative%20genomics&amp;author=L.D.%20Stein&amp;author=Z.%20Bao&amp;author=D.%20Blasiar&amp;publication_year=2003&amp;journal=PLoS%20Biol.&amp;volume=1&amp;pages=E45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0000045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0000045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14624247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR046C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR046C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Linton</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Birren</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">860</div>-<div class="lpage">921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.S.%20Lander&amp;author=L.M.%20Linton&amp;author=B.%20Birren&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=419166&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr046-dsr046supp_table5.xls?Expires=2147483647&amp;Signature=R-UXobk-I7FiSl0zCHVQ6vlHDWXXZWwFcIDar5Ts43JOW0B88IxZaRpiOkQdeG-rAz19pnFtFOSjaguyezhNKOjOZsfB9c72ivdngAOIHYPQzUfCaPN5CUwap87Sc7S6yY~vM917~EBncybpNrMRxHrvkmIx3L7KvHtZ5QdetffDHfUN3npYRjf4j8wYwEYMdSMHklkmlYBVyAPom81zx8h0JcKLcyKLoLURbe3F9nvNb1g~kcpOYGKLzSQ0vXP8jeCStBn7M-k5Ebejoi6ZVnW3luKSuEXdNdp0sHjdQRZL15pw79lAK1-86sjwxiJdyaybvWqq-LKxpgfViXBJGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr046-dsr046supp_table4.xls?Expires=2147483647&amp;Signature=dSCYOt4jnCAnKK2uA23HWBvDAlzblLSrY-UghK0IF-T4lvUtKcgERPNyKFhzJAwrvFqU0zmK-HleVNn1PcMYF4YfXI6I81xQ0n1zBqilmemNz5EuLjzb7qG1UeG4b5oP2MSmB4rU2aBEoxxfMoGA6iNA-u76CricMDZoLq3wE~Jx2WQA5qR6~bDDN0crvkDFjEfDHs8anMJO0tK1I2FND7pG0PdxpwW2bbgqp5n7tGmCy1zPfl2AQttXWARNB~gFyB1lo1XIhZkco6oZfinVn2dNdRl14lNxiUJ~~pV48q-Xt2rxNErjgaQNMwyiIkBkJSNjP9B6sBoR3I3~Wd0Ejg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr046-dsr046supp_table3.xls?Expires=2147483647&amp;Signature=2jRz~iTp1S1KewotJYadfQ9a~n8mEPWhN2UC8DONf~DShyDMEs4KNiMMtuA4PSIG9BS8kFXn6XqeTfcRyvm9MoMvrkGxS6yydMGfen-CUOp0jyyABtGQmTUcYnr6MOXmSnpngbXwD-3dFyPKxFuMru-E3WLqP4c9sV7JaEDitcbTLZzeR2OnGaq-YuQsNtKr3X7ST5zDUfykYJXL2TxEdLQMDbCtGWpBlPDkSPHIXkxPdYPx9vn~KF50xP0us9cu9YGxzpBE8eSjSIVWq7wHgvsEUG-PxnxBZoZaIFc2W39tIxctNwRxyi8DIPToKkLDXigJrEz4AqAgdMlK8VnNVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr046-dsr046supp_table2.xls?Expires=2147483647&amp;Signature=wxMAujsE9-ndlHsIXwwiXc5dHO0OtGFz8EFMAgS9mzLouWUpQ1rlIxRE2lqkh6nWMVr~yxMux9oRrifdL9q5A9rf7ILGG0vqXc6PJeOeeJviKJ4BsPcMj9kNIkgFSD72XKA99~JNXz-OtUU5AUiyyAHB5lSfoSTU-MWNPNhoH8vB3b5lWVk55ge8DlZMoYuNmuRYdqDUfjx10KwtNXo4zN5ZYfTMIh6f8CNaAMZkD819yIlFuJG-ahixB1TDQS2FU5Rj2jPdlAivMrvheEJtur5yLFSe5MGD5QqNskOT7mN2S3VHafFpJhtIwqaWrhY~RA7rdgCaISrVSRQuOTebpw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dsr046/2/dsr046-dsr046supp_table1.xls?Expires=2147483647&amp;Signature=1hQepwHmyBEHitS1kXhG4MfzsjJUIuzB4baJWcfp1QZV21ckB-rya35OE3S9TEdFP3nDftE4LJXTx0Pf3dOH-nV9TlneFTm9Wa7uEQsS6x21Lk~HhFaOenHfb32FzuMBe2lIQS~5JV2lwV9u9q7Zg5NDAINUEbpIJGDLIY3Ul~7QC98XJ-gd66SR0o2XHOMgpPo1~V93EI5ecdCPrjeVuWu-FMl8sx~2HrF6UYyVDVpGiQu9uHT09XZmxWjoUL2yv7Cj21m1tjsXtuU4j3zkvSvnwnWSm1KTpWjOeVQo3pmaeTU554yj2BRTPdF0H8xdfJBIfJZBGEy6UkNgFvLOIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">326</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">38</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr046"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="419166"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f2%2f131%2f419166">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000303290100003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (38)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F2%2F131%2F419166" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/419166'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Transcriptome Analysis of Silver Carp (Hypophthalmichthys molitrix) by Paired-End RNA Sequencing" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr046");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f131%2f419166">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

