

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/2/171/333025","@type":"ScholarlyArticle","name":"Survey of genome sequences in a wild sweet potato,  Ipomoea trifida  (H. B. K.) G. Don","datePublished":"2015-03-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2015-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv002","keywords":["<italic>Ipomoea trifida</italic>","genome sequence assembly","core- and line-specific sequences","SNPs","CNVs"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/2/171/333025","author":[{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Okada, Yoshihiro","affiliation":"2  Crop and Agribusiness Research Division, Kyushu Okinawa Agricultural Research Center ,  National Agriculture and Food Research Organization (NARO/KARC) ,  Itoman, Okinawa 901-0336 ,  Japan","@type":"Person"},{"name":"Tabuchi, Hiroaki","affiliation":"3  Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center ,  National Agriculture and Food Research Organization (NARO/KARC) ,  Miyakonojo, Miyazaki 885-0091 ,  Japan","@type":"Person"},{"name":"Shirasawa, Kenta","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tsuruoka, Hisano","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Minami, Chiharu","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakayama, Shinobu","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sasamoto, Shigemi","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kishida, Yoshie","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fujishiro, Tsunakazu","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kato, Midori","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nanri, Keiko","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Komaki, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yoshinaga, Masaru","affiliation":"3  Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center ,  National Agriculture and Food Research Organization (NARO/KARC) ,  Miyakonojo, Miyazaki 885-0091 ,  Japan","@type":"Person"},{"name":"Takahata, Yasuhiro","affiliation":"3  Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center ,  National Agriculture and Food Research Organization (NARO/KARC) ,  Miyakonojo, Miyazaki 885-0091 ,  Japan","@type":"Person"},{"name":"Tanaka, Masaru","affiliation":"3  Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center ,  National Agriculture and Food Research Organization (NARO/KARC) ,  Miyakonojo, Miyazaki 885-0091 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko N.","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Ipomoea trifida (H. B. K.) G. Don. is the most likely diploid ancestor of the hexaploid sweet potato, I. batatas (L.) Lam. To assist in analysis of t","pageStart":"171","pageEnd":"179","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Survey of genome sequences in a wild sweet potato,  Ipomoea trifida  (H. B. K.) G. Don","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Survey of genome sequences in a wild sweet potato,  Ipomoea trifida  (H. B. K.) G. Don" />
    <meta property="og:description" content="Abstract.  Ipomoea trifida (H. B. K.) G. Don. is the most likely diploid ancestor of the hexaploid sweet potato, I. batatas (L.) Lam. To assist in analysis of t" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/2/171/333025" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-03-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Okada, Yoshihiro" /><meta name="citation_author_institution" content="Crop and Agribusiness Research Division, Kyushu Okinawa Agricultural Research Center, National Agriculture and Food Research Organization (NARO/KARC), Itoman, Okinawa 901-0336, Japan" /><meta name="citation_author" content="Tabuchi, Hiroaki" /><meta name="citation_author_institution" content="Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center, National Agriculture and Food Research Organization (NARO/KARC), Miyakonojo, Miyazaki 885-0091, Japan" /><meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tsuruoka, Hisano" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Minami, Chiharu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakayama, Shinobu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sasamoto, Shigemi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kishida, Yoshie" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fujishiro, Tsunakazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kato, Midori" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nanri, Keiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Komaki, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yoshinaga, Masaru" /><meta name="citation_author_institution" content="Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center, National Agriculture and Food Research Organization (NARO/KARC), Miyakonojo, Miyazaki 885-0091, Japan" /><meta name="citation_author" content="Takahata, Yasuhiro" /><meta name="citation_author_institution" content="Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center, National Agriculture and Food Research Organization (NARO/KARC), Miyakonojo, Miyazaki 885-0091, Japan" /><meta name="citation_author" content="Tanaka, Masaru" /><meta name="citation_author_institution" content="Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center, National Agriculture and Food Research Organization (NARO/KARC), Miyakonojo, Miyazaki 885-0091, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko N." /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Survey of genome sequences in a wild sweet potato, Ipomoea trifida (H. B. K.) G. Don" /><meta name="citation_firstpage" content="171" /><meta name="citation_lastpage" content="179" /><meta name="citation_doi" content="10.1093/dnares/dsv002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2015/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A synopsis of Ipomoea (Convolvulaceae) in the Americas; citation_author=Austin  D.F.; citation_author=Huám  Z.;  citation_journal_title=Taxon;  citation_year=1996;  citation_volume=45;  citation_pages=3-38; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content and ploidy levels in the genus Ipomoea; citation_author=Ozias-Akins  P.; citation_author=Jarret  R.L.;  citation_journal_title=J. Amer. Soc. Hort. Sci.;  citation_year=1994;  citation_volume=119;  citation_pages=110-5; " /><meta name="citation_reference" content="citation_title=Incompatibility in the sweet potato. A review; citation_author=Martin  F.W.;  citation_journal_title=Eco. Bot.;  citation_year=1965;  citation_volume=19;  citation_pages=406-15; " /><meta name="citation_reference" content="citation_title=The Ipomoea trifida complex closely related to sweet potato; citation_author=Kobayashi  M.; citation_author=Shideler  S.F.; citation_author=Rincon  H.; citation_publisher=International Potato Center Lima, Peru;  citation_title=Proceedings of the 6th Symposium of the International Society of Tropical Root Crops;  citation_year=1983;  citation_pages=561-8; " /><meta name="citation_reference" content="citation_title=Genomic structure of the sweet potato and hexaploid in Ipomoea trifida (H. B. K.) Don.; citation_author=Shiotani  I.; citation_author=Kawase  T.;  citation_journal_title=Japan. J. Breed.;  citation_year=1989;  citation_volume=39;  citation_pages=57-66; " /><meta name="citation_reference" content="citation_title=Use of Ipomoea trifida germ plasm for sweet potato improvement. 3. Development of 4x interspecific hybrids between Ipomoea batatas (L.) Lam. (2n = 6x = 90) and I. trifida (H. B. K) G. Don. (2n = 2x = 30) as storage-root initiators for wild species; citation_author=Orjeda  G.; citation_author=Freyre  R.; citation_author=Iwanaga  M.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=1991;  citation_volume=83;  citation_pages=159-63; " /><meta name="citation_reference" content="citation_title=Root thickness of diploid Ipomoea trifida (H. B. K.) G. Don and performance of progeny derived from the cross with sweetpotato; citation_author=Komaki  K.; citation_author=Katayama  K.;  citation_journal_title=Breed. Sci.;  citation_year=1999;  citation_volume=49;  citation_pages=123-9; " /><meta name="citation_reference" content="citation_title=Genetic diversity and systematic relationship in sweetpotato (Ipomoea batatas (L.) Lam.) and related species as revealed by RAPD analysis; citation_author=Jarret  R.L.; citation_author=Austin  D.F.;  citation_journal_title=Genet. Resour. Crop Evol.;  citation_year=1994;  citation_volume=41;  citation_pages=165-73; " /><meta name="citation_reference" content="citation_title=Morphological and RAPD pattern variations in sweetpotato and its closely related species; citation_author=Komaki  K.; citation_author=Regmi  H.N.; citation_author=Katayama  K.; citation_author=Tamiya  S.;  citation_journal_title=Breed. Sci.;  citation_year=1998;  citation_volume=48;  citation_pages=281-6; " /><meta name="citation_reference" content="citation_title=Genetic diversity and relationships of sweetpotato and its wild relatives in Ipomoea series Batatas (Convolvulaceae) as revealed by inter-simple sequence repeat (ISSR) and restriction analysis of chloroplast DNA; citation_author=Huang  J.C.; citation_author=Sun  M.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2000;  citation_volume=100;  citation_pages=1050-60; " /><meta name="citation_reference" content="citation_title=Disentangling the origins of cultivated sweet potato (Ipomoea batatas (L.) Lam.); citation_author=Roullier  C.; citation_author=Duputié  A.; citation_author=Wennekes  P.;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e62707" /><meta name="citation_reference" content="citation_title=The origin and evolution of sweet potato (Ipomoea batatas Lam.) and its wild relatives through the cytogenetic approaches; citation_author=Srisuwan  S.; citation_author=Sihachakr  D.; citation_author=Siljak-Yakovlev  S.;  citation_journal_title=Plant Sci.;  citation_year=2006;  citation_volume=171;  citation_pages=424-33; " /><meta name="citation_reference" content="citation_title=Sporophitic self-incompatibility in Ipomoea trifida, a close relative of sweet potato; citation_author=Kowyama  Y.; citation_author=Tsuchiya  T.; citation_author=Kakeda  K.;  citation_journal_title=Ann. Bot.;  citation_year=2000;  citation_volume=85;  citation_pages=191-6; " /><meta name="citation_reference" content="citation_title=Visualization of the S-locus region in Ipomoea trifida: toward positional cloning of self-incompatibility genes; citation_author=Suzuki  G.; citation_author=Tanaka  S.; citation_author=Yamamoto  M.; citation_author=Tomita  R.N.; citation_author=Kowyama  Y.; citation_author=Mukai  Y.;  citation_journal_title=Chromosome Res.;  citation_year=2004;  citation_volume=12;  citation_pages=475-81; " /><meta name="citation_reference" content="citation_title=Physical size of the S locus region defined by genetic recombination and genome sequencing in Ipomoea trifida, Convolvulaceae; citation_author=Rahman  M.H.; citation_author=Tsuchiya  T.; citation_author=Suwabe  K.;  citation_journal_title=Sexual Plant Reprod.;  citation_year=2007;  citation_volume=20;  citation_pages=63-72; " /><meta name="citation_reference" content="citation_title=Expression of stigma- and anther-specific genes located in the S locus region of Ipomoea trifida; citation_author=Rahman  M.H.; citation_author=Uchiyama  M.; citation_author=Kuno  M.;  citation_journal_title=Sexual Plant Reprod.;  citation_year=2007;  citation_volume=20;  citation_pages=73-85; " /><meta name="citation_reference" content="citation_title=An AFLP-based genetic linkage map of Ipomoea trifida (H.B.K.) G. Don., a diploid relative of sweetpotato; citation_author=Nakayama  H.; citation_author=Tanaka  M.; citation_author=Takahata  Y.;  citation_journal_title=I. Batatas (L.) Lam. Trop. Agr. Develop.;  citation_year=2010;  citation_volume=54;  citation_pages=9-16; " /><meta name="citation_reference" content="citation_title=The first 50 plant genomes; citation_author=Michael  T.P.; citation_author=Jackson  S.;  citation_journal_title=Plant Genome;  citation_year=2013;  citation_volume=6;  citation_pages=2" /><meta name="citation_reference" content="citation_title=The 1001 genomes project for Arabidopsis thaliana; citation_author=Weigel  D.; citation_author=Mott  R.;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=107" /><meta name="citation_reference" content="citation_title=Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes; citation_author=Xu  X.; citation_author=Liu  X.; citation_author=Ge  S.;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=105-11; " /><meta name="citation_reference" content="citation_title=Genome-wide genetic changes during modern breeding of maize; citation_author=Jiao  Y.; citation_author=Zhao  H.; citation_author=Ren  L.;  citation_journal_title=Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=812-5; " /><meta name="citation_reference" content="citation_title=Transposable elements and the plant pan-genomes; citation_author=Morgante  M.; citation_author=De Paoli  E.; citation_author=Radovic  S.;  citation_journal_title=Curr. Opin. Plant Biol.;  citation_year=2007;  citation_volume=10;  citation_pages=149-55; " /><meta name="citation_reference" content="citation_title=Genome analysis of multiple pathogenic isolates of Streptococcus agalactiae: implications for the microbial ‘pan-genome; citation_author=Tettelin  H.; citation_author=Masignani  V.; citation_author=Cieslewicz  M.J.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=13950-5; " /><meta name="citation_reference" content="citation_title=Evolution of DNA sequence nonhomologies among maize inbreds; citation_author=Brunner  S.; citation_author=Fengler  K.; citation_author=Morgante  M.; citation_author=Tingey  S.; citation_author=Rafalski  A.;  citation_journal_title=Plant Cell;  citation_year=2005;  citation_volume=17;  citation_pages=343-60; " /><meta name="citation_reference" content="citation_title=De novo assembly of soybean wild relatives for pan-genome analysis of diversity and agronomic traits; citation_author=Li  Y.H.; citation_author=Zhou  G.; citation_author=Ma  J.;  citation_journal_title=Nat. Biotechnol.;  citation_year=2014;  citation_volume=32;  citation_pages=1045-52; " /><meta name="citation_reference" content="citation_title=Structural variations in plant genomes; citation_author=Saxena  R.K.; citation_author=Edwards  D.; citation_author=Varshney  R.K.;  citation_journal_title=Brief Funct. Genomics;  citation_year=2014;  citation_volume=13;  citation_pages=296-307; " /><meta name="citation_reference" content="citation_title=Rapid isolation of high-molecular-weight plant DNA; citation_author=Murray  M.G.; citation_author=Thompson  W.F.;  citation_journal_title=Nucleic Acids Res.;  citation_year=1980;  citation_volume=8;  citation_pages=4321-5; " /><meta name="citation_reference" content="citation_title=Comprehensive structural analysis of the genome of red clover (Trifolium pretense L.); citation_author=Sato  S.; citation_author=Isobe  S.; citation_author=Asamizu  E.;  citation_journal_title=DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=301-64; " /><meta name="citation_reference" content="citation_title=Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination; citation_author=Nieuwerburgh  F.V.; citation_author=Thompson  R.C.; citation_author=Ledesma  J.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=e24" /><meta name="citation_reference" content="citation_title=Quality control and preprocessing of metagenomic datasets; citation_author=Schmieder  R.; citation_author=Edwards  R.;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=863-4; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers; citation_author=Marçais  G.; citation_author=Kingsford  C.;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=De novo assembly of human genomes with massively parallel short read sequencing; citation_author=Li  R.; citation_author=Zhu  H.; citation_author=Ruan  J.;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=265-72; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads; citation_author=Kajitani  R.; citation_author=Toshimoto  K.; citation_author=Noguchi  H.;  citation_journal_title=Genome Res.;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE; citation_author=Boetzer  M.; citation_author=Henkel  C.V.; citation_author=Jansen  H.J.; citation_author=Butler  D.; citation_author=Pirovano  W.;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; citation_author=Altschul  S.F.; citation_author=Gish  W.; citation_author=Miller  W.; citation_author=Myers  E.W.; citation_author=Lipman  D.J.;  citation_journal_title=J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Molecular characterization of a 313-kb genomic region containing the self-incompatibility locus of Ipomoea trifida, a diploid relative of sweet potato; citation_author=Tomita  R.N.; citation_author=Suzuki  G.; citation_author=Yoshida  K.;  citation_journal_title=Breed. Sci.;  citation_year=2004;  citation_volume=54;  citation_pages=165-75; " /><meta name="citation_reference" content="citation_title=Versatile and open software for comparing large genomes; citation_author=Kurtz  S.; citation_author=Phillippy  A.; citation_author=Delcher  A.L.;  citation_journal_title=Genome Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=R12" /><meta name="citation_reference" content="citation_title=Adaptive seeds tame genomic sequence comparison; citation_author=Kielbasa  S.M.; citation_author=Wan  R.; citation_author=Sato  K.; citation_author=Horton  P.; citation_author=Frith  M.C.;  citation_journal_title=Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=487-93; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes; citation_author=Price  A.L.; citation_author=Jones  N.C.; citation_author=Pevzner  P.A.;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of eggplant (Solanum melongena L.): the representative Solanum species indigenous to the Old World; citation_author=Hirakawa  H.; citation_author=Shirasawa  K.; citation_author=Miyatake  K.;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=649-60; " /><meta name="citation_reference" content="citation_title=SciRoKo: a new tool for whole genome microsatellite search and investigation; citation_author=Kofler  R.; citation_author=Schlotterer  C.; citation_author=Lelley  T.;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1683-5; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; citation_author=Lowe  T.M.; citation_author=Eddy  S.R.;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel; citation_author=Stanke  M.; citation_author=Waack  S.;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_issue=(Suppl. 2);  citation_pages=ii215-25; " /><meta name="citation_reference" content="citation_title=Assembling genes from predicted exons in linear time with dynamic programming; citation_author=Guigó  R.;  citation_journal_title=J. Comput. Biol.;  citation_year=1998;  citation_volume=5;  citation_pages=681-702; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier; citation_author=Quevillon  E.; citation_author=Silventoinen  V.; citation_author=Pillai  S.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=New developments in the InterPro database; citation_author=Mulder  N.J.; citation_author=Apweiler  R.; citation_author=Attwood  T.K.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D224-8; " /><meta name="citation_reference" content="citation_title=A new generation of homology search tools based on probabilistic inference; citation_author=Eddy  S.R.;  citation_journal_title=Genome Informatics;  citation_year=2009;  citation_volume=23;  citation_pages=205-11; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology; citation_author=The Gene Ontology Consortium;  citation_journal_title=Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; citation_author=Tatusov  R.L.; citation_author=Fedorova  N.D.; citation_author=Jackson  J.D.;  citation_journal_title=BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes; citation_author=Ogata  H.; citation_author=Goto  S.; citation_author=Sato  K.; citation_author=Fujibuchi  W.; citation_author=Bono  H.; citation_author=Kanehisa  M.;  citation_journal_title=Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=29-34; " /><meta name="citation_reference" content="citation_title=The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools; citation_author=Thompson  J.D.; citation_author=Gibson  T.J.; citation_author=Plewniak  F.; citation_author=Jeanmougin  F.; citation_author=Higgins  D.G.;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=4876-82; " /><meta name="citation_reference" content="citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0; citation_author=Tamura  K.; citation_author=Stecher  G.; citation_author=Peterson  D.; citation_author=Filipski  A.; citation_author=Kumar  S.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=Langmead  B.; citation_author=Salzberg  S.;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=The Sequence alignment/map (SAM) format and SAMtools; citation_author=Li  H.; citation_author=Handsaker  B.; citation_author=Wysoker  A.;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=The Variant Call Format and VCFtools; citation_author=Danecek  P.; citation_author=Auton  A.; citation_author=Abecasis  G.;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2156-8; " /><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3; citation_author=Cingolani  P.; citation_author=Platts  A.; citation_author=Wangle  L.;  citation_journal_title=Fly (Austin);  citation_year=2012;  citation_volume=6;  citation_pages=80-92; " /><meta name="citation_reference" content="citation_title=CNV-seq, a new method to detect copy number variation using high-throughput sequencing; citation_author=Xie  C.; citation_author=Tammi  M.T.;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=12;  citation_pages=80" /><meta name="citation_reference" content="citation_title=Genome structure of the Legume; citation_author=Sato  S.; citation_author=Nakamura  Y.; citation_author=Kaneko  T.;  citation_journal_title=Lotus Japonicas, DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=227-39; " /><meta name="citation_reference" content="citation_title=Genome sequence and analysis of the tuber crop potato; citation_author=The Potato Genome Sequencing Consortium;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=475;  citation_pages=189-95; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution; citation_author=The Tomato Genome Consortium;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Inhibition of the expression of the starch synthase II gene leads to lower pasting temperature in sweetpotato starch; citation_author=Takahata  Y.; citation_author=Tanaka  M.; citation_author=Otani  M.;  citation_journal_title=Plant Cell Rep.;  citation_year=2010;  citation_volume=29;  citation_pages=535-43; " /><meta name="citation_reference" content="citation_title=A comprehensive expression analysis of the starch synthase gene family in rice (Oryza sativa L.); citation_author=Hirose  T.; citation_author=Terao  T.;  citation_journal_title=Planta;  citation_year=2004;  citation_volume=220;  citation_pages=9-16; " /><meta name="citation_reference" content="citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres; citation_author=Peterson  A.; citation_author=Wendel  J.F.; citation_author=Gundlach  H.;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=492;  citation_pages=423-7; " /><meta name="citation_reference" content="citation_title=The banana (Musa acuminata) genome and the evolution of monocotyledonous plants; citation_author=D'Hont  A.; citation_author=Denoeud  F.; citation_author=Aury  J.M.;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=488;  citation_pages=213-7; " /><meta name="citation_reference" content="citation_title=Dissection of the octoploid strawberry genome by deep sequencing of the genomes of Fragaria species; citation_author=Hirakawa  H.; citation_author=Shirasawa  K.; citation_author=Kosugi  S.;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=169-81; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/2/171/952242/dsv002.pdf" /><meta name="description" content="Abstract.  Ipomoea trifida (H. B. K.) G. Don. is the most likely diploid ancestor of the hexaploid sweet potato, I. batatas (L.) Lam. To assist in analysis of t" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/2/171/333025" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/2/171/333025" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/2/171/333025");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv002';
    </script>
<title>
	Survey of genome sequences in a wild sweet potato, Ipomoea trifida (H. B. K.) G. Don | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./333025" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbfRATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2thZGEsIFlvc2hpaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNyb3AgYW5kIEFncmlidXNpbmVzcyBSZXNlYXJjaCBEaXZpc2lvbiwgS3l1c2h1IE9raW5hd2EgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIENlbnRlciwgTmF0aW9uYWwgQWdyaWN1bHR1cmUgYW5kIEZvb2QgUmVzZWFyY2ggT3JnYW5pemF0aW9uIChOQVJPL0tBUkMpLCBJdG9tYW4sIE9raW5hd2EgOTAxLTAzMzYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFidWNoaSwgSGlyb2FraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVwbGFuZCBGYXJtaW5nIFJlc2VhcmNoIERpdmlzaW9uLCBLeXVzaHUgT2tpbmF3YSBBZ3JpY3VsdHVyYWwgUmVzZWFyY2ggQ2VudGVyLCBOYXRpb25hbCBBZ3JpY3VsdHVyZSBhbmQgRm9vZCBSZXNlYXJjaCBPcmdhbml6YXRpb24gKE5BUk8vS0FSQyksIE1peWFrb25vam8sIE1peWF6YWtpIDg4NS0wMDkxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaXJhc2F3YSwgS2VudGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhdGFuYWJlLCBBa2lrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVHN1cnVva2EsIEhpc2FubyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWluYW1pLCBDaGloYXJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgU2hpbm9idSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FzYW1vdG8sIFNoaWdlbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvaGFyYSwgTWl0c3V5byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2lzaGlkYSwgWW9zaGllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppc2hpcm8sIFRzdW5ha2F6dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0bywgTWlkb3JpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYW5yaSwgS2Vpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvbWFraSwgQWtpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Illvc2hpbmFnYSwgTWFzYXJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVXBsYW5kIEZhcm1pbmcgUmVzZWFyY2ggRGl2aXNpb24sIEt5dXNodSBPa2luYXdhIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIsIE5hdGlvbmFsIEFncmljdWx0dXJlIGFuZCBGb29kIFJlc2VhcmNoIE9yZ2FuaXphdGlvbiAoTkFSTy9LQVJDKSwgTWl5YWtvbm9qbywgTWl5YXpha2kgODg1LTAwOTEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYWhhdGEsIFlhc3VoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVXBsYW5kIEZhcm1pbmcgUmVzZWFyY2ggRGl2aXNpb24sIEt5dXNodSBPa2luYXdhIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIsIE5hdGlvbmFsIEFncmljdWx0dXJlIGFuZCBGb29kIFJlc2VhcmNoIE9yZ2FuaXphdGlvbiAoTkFSTy9LQVJDKSwgTWl5YWtvbm9qbywgTWl5YXpha2kgODg1LTAwOTEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuYWthLCBNYXNhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVcGxhbmQgRmFybWluZyBSZXNlYXJjaCBEaXZpc2lvbiwgS3l1c2h1IE9raW5hd2EgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIENlbnRlciwgTmF0aW9uYWwgQWdyaWN1bHR1cmUgYW5kIEZvb2QgUmVzZWFyY2ggT3JnYW5pemF0aW9uIChOQVJPL0tBUkMpLCBNaXlha29ub2pvLCBNaXlhemFraSA4ODUtMDA5MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWJhdGEsIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iklzb2JlLCBTYWNoaWtvIE4uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlN1cnZleSBvZiBnZW5vbWUgc2VxdWVuY2VzIGluIGEgd2lsZCBzd2VldCBwb3RhdG8sIElwb21vZWEgdHJpZmlkYSAoSC4gQi4gSy4pIEcuIERvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN5bm9wc2lzIG9mIElwb21vZWEgKENvbnZvbHZ1bGFjZWFlKSBpbiB0aGUgQW1lcmljYXM7IGNpdGF0aW9uX2F1dGhvcj1BdXN0aW4gIEQuRi47IGNpdGF0aW9uX2F1dGhvcj1IdcOhbSAgWi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRheG9uOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9My0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBETkEgY29udGVudCBhbmQgcGxvaWR5IGxldmVscyBpbiB0aGUgZ2VudXMgSXBvbW9lYTsgY2l0YXRpb25fYXV0aG9yPU96aWFzLUFraW5zICBQLjsgY2l0YXRpb25fYXV0aG9yPUphcnJldCAgUi5MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW1lci4gU29jLiBIb3J0LiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTExOTsgIGNpdGF0aW9uX3BhZ2VzPTExMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNvbXBhdGliaWxpdHkgaW4gdGhlIHN3ZWV0IHBvdGF0by4gQSByZXZpZXc7IGNpdGF0aW9uX2F1dGhvcj1NYXJ0aW4gIEYuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVjby4gQm90LjsgIGNpdGF0aW9uX3llYXI9MTk2NTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQwNi0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIElwb21vZWEgdHJpZmlkYSBjb21wbGV4IGNsb3NlbHkgcmVsYXRlZCB0byBzd2VldCBwb3RhdG87IGNpdGF0aW9uX2F1dGhvcj1Lb2JheWFzaGkgIE0uOyBjaXRhdGlvbl9hdXRob3I9U2hpZGVsZXIgIFMuRi47IGNpdGF0aW9uX2F1dGhvcj1SaW5jb24gIEguOyBjaXRhdGlvbl9wdWJsaXNoZXI9SW50ZXJuYXRpb25hbCBQb3RhdG8gQ2VudGVyIExpbWEsIFBlcnU7ICBjaXRhdGlvbl90aXRsZT1Qcm9jZWVkaW5ncyBvZiB0aGUgNnRoIFN5bXBvc2l1bSBvZiB0aGUgSW50ZXJuYXRpb25hbCBTb2NpZXR5IG9mIFRyb3BpY2FsIFJvb3QgQ3JvcHM7ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl9wYWdlcz01NjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBzdHJ1Y3R1cmUgb2YgdGhlIHN3ZWV0IHBvdGF0byBhbmQgaGV4YXBsb2lkIGluIElwb21vZWEgdHJpZmlkYSAoSC4gQi4gSy4pIERvbi47IGNpdGF0aW9uX2F1dGhvcj1TaGlvdGFuaSAgSS47IGNpdGF0aW9uX2F1dGhvcj1LYXdhc2UgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KYXBhbi4gSi4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NTctNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzZSBvZiBJcG9tb2VhIHRyaWZpZGEgZ2VybSBwbGFzbSBmb3Igc3dlZXQgcG90YXRvIGltcHJvdmVtZW50LiAzLiBEZXZlbG9wbWVudCBvZiA0eCBpbnRlcnNwZWNpZmljIGh5YnJpZHMgYmV0d2VlbiBJcG9tb2VhIGJhdGF0YXMgKEwuKSBMYW0uICgybiA9IDZ4ID0gOTApIGFuZCBJLiB0cmlmaWRhIChILiBCLiBLKSBHLiBEb24uICgybiA9IDJ4ID0gMzApIGFzIHN0b3JhZ2Utcm9vdCBpbml0aWF0b3JzIGZvciB3aWxkIHNwZWNpZXM7IGNpdGF0aW9uX2F1dGhvcj1PcmplZGEgIEcuOyBjaXRhdGlvbl9hdXRob3I9RnJleXJlICBSLjsgY2l0YXRpb25fYXV0aG9yPUl3YW5hZ2EgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9MTU5LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb290IHRoaWNrbmVzcyBvZiBkaXBsb2lkIElwb21vZWEgdHJpZmlkYSAoSC4gQi4gSy4pIEcuIERvbiBhbmQgcGVyZm9ybWFuY2Ugb2YgcHJvZ2VueSBkZXJpdmVkIGZyb20gdGhlIGNyb3NzIHdpdGggc3dlZXRwb3RhdG87IGNpdGF0aW9uX2F1dGhvcj1Lb21ha2kgIEsuOyBjaXRhdGlvbl9hdXRob3I9S2F0YXlhbWEgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmVlZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTEyMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBhbmQgc3lzdGVtYXRpYyByZWxhdGlvbnNoaXAgaW4gc3dlZXRwb3RhdG8gKElwb21vZWEgYmF0YXRhcyAoTC4pIExhbS4pIGFuZCByZWxhdGVkIHNwZWNpZXMgYXMgcmV2ZWFsZWQgYnkgUkFQRCBhbmFseXNpczsgY2l0YXRpb25fYXV0aG9yPUphcnJldCAgUi5MLjsgY2l0YXRpb25fYXV0aG9yPUF1c3RpbiAgRC5GLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIFJlc291ci4gQ3JvcCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTE2NS03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9ycGhvbG9naWNhbCBhbmQgUkFQRCBwYXR0ZXJuIHZhcmlhdGlvbnMgaW4gc3dlZXRwb3RhdG8gYW5kIGl0cyBjbG9zZWx5IHJlbGF0ZWQgc3BlY2llczsgY2l0YXRpb25fYXV0aG9yPUtvbWFraSAgSy47IGNpdGF0aW9uX2F1dGhvcj1SZWdtaSAgSC5OLjsgY2l0YXRpb25fYXV0aG9yPUthdGF5YW1hICBLLjsgY2l0YXRpb25fYXV0aG9yPVRhbWl5YSAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyZWVkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MjgxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IGFuZCByZWxhdGlvbnNoaXBzIG9mIHN3ZWV0cG90YXRvIGFuZCBpdHMgd2lsZCByZWxhdGl2ZXMgaW4gSXBvbW9lYSBzZXJpZXMgQmF0YXRhcyAoQ29udm9sdnVsYWNlYWUpIGFzIHJldmVhbGVkIGJ5IGludGVyLXNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgKElTU1IpIGFuZCByZXN0cmljdGlvbiBhbmFseXNpcyBvZiBjaGxvcm9wbGFzdCBETkE7IGNpdGF0aW9uX2F1dGhvcj1IdWFuZyAgSi5DLjsgY2l0YXRpb25fYXV0aG9yPVN1biAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MTA1MC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzZW50YW5nbGluZyB0aGUgb3JpZ2lucyBvZiBjdWx0aXZhdGVkIHN3ZWV0IHBvdGF0byAoSXBvbW9lYSBiYXRhdGFzIChMLikgTGFtLik7IGNpdGF0aW9uX2F1dGhvcj1Sb3VsbGllciAgQy47IGNpdGF0aW9uX2F1dGhvcj1EdXB1dGnDqSAgQS47IGNpdGF0aW9uX2F1dGhvcj1XZW5uZWtlcyAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNjI3MDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JpZ2luIGFuZCBldm9sdXRpb24gb2Ygc3dlZXQgcG90YXRvIChJcG9tb2VhIGJhdGF0YXMgTGFtLikgYW5kIGl0cyB3aWxkIHJlbGF0aXZlcyB0aHJvdWdoIHRoZSBjeXRvZ2VuZXRpYyBhcHByb2FjaGVzOyBjaXRhdGlvbl9hdXRob3I9U3Jpc3V3YW4gIFMuOyBjaXRhdGlvbl9hdXRob3I9U2loYWNoYWtyICBELjsgY2l0YXRpb25fYXV0aG9yPVNpbGphay1ZYWtvdmxldiAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTcxOyAgY2l0YXRpb25fcGFnZXM9NDI0LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcG9yb3BoaXRpYyBzZWxmLWluY29tcGF0aWJpbGl0eSBpbiBJcG9tb2VhIHRyaWZpZGEsIGEgY2xvc2UgcmVsYXRpdmUgb2Ygc3dlZXQgcG90YXRvOyBjaXRhdGlvbl9hdXRob3I9S293eWFtYSAgWS47IGNpdGF0aW9uX2F1dGhvcj1Uc3VjaGl5YSAgVC47IGNpdGF0aW9uX2F1dGhvcj1LYWtlZGEgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0xOTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmlzdWFsaXphdGlvbiBvZiB0aGUgUy1sb2N1cyByZWdpb24gaW4gSXBvbW9lYSB0cmlmaWRhOiB0b3dhcmQgcG9zaXRpb25hbCBjbG9uaW5nIG9mIHNlbGYtaW5jb21wYXRpYmlsaXR5IGdlbmVzOyBjaXRhdGlvbl9hdXRob3I9U3V6dWtpICBHLjsgY2l0YXRpb25fYXV0aG9yPVRhbmFrYSAgUy47IGNpdGF0aW9uX2F1dGhvcj1ZYW1hbW90byAgTS47IGNpdGF0aW9uX2F1dGhvcj1Ub21pdGEgIFIuTi47IGNpdGF0aW9uX2F1dGhvcj1Lb3d5YW1hICBZLjsgY2l0YXRpb25fYXV0aG9yPU11a2FpICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hyb21vc29tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDc1LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlzaWNhbCBzaXplIG9mIHRoZSBTIGxvY3VzIHJlZ2lvbiBkZWZpbmVkIGJ5IGdlbmV0aWMgcmVjb21iaW5hdGlvbiBhbmQgZ2Vub21lIHNlcXVlbmNpbmcgaW4gSXBvbW9lYSB0cmlmaWRhLCBDb252b2x2dWxhY2VhZTsgY2l0YXRpb25fYXV0aG9yPVJhaG1hbiAgTS5ILjsgY2l0YXRpb25fYXV0aG9yPVRzdWNoaXlhICBULjsgY2l0YXRpb25fYXV0aG9yPVN1d2FiZSAgSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNleHVhbCBQbGFudCBSZXByb2QuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NjMtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gb2Ygc3RpZ21hLSBhbmQgYW50aGVyLXNwZWNpZmljIGdlbmVzIGxvY2F0ZWQgaW4gdGhlIFMgbG9jdXMgcmVnaW9uIG9mIElwb21vZWEgdHJpZmlkYTsgY2l0YXRpb25fYXV0aG9yPVJhaG1hbiAgTS5ILjsgY2l0YXRpb25fYXV0aG9yPVVjaGl5YW1hICBNLjsgY2l0YXRpb25fYXV0aG9yPUt1bm8gIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TZXh1YWwgUGxhbnQgUmVwcm9kLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTczLTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBBRkxQLWJhc2VkIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgSXBvbW9lYSB0cmlmaWRhIChILkIuSy4pIEcuIERvbi4sIGEgZGlwbG9pZCByZWxhdGl2ZSBvZiBzd2VldHBvdGF0bzsgY2l0YXRpb25fYXV0aG9yPU5ha2F5YW1hICBILjsgY2l0YXRpb25fYXV0aG9yPVRhbmFrYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1UYWthaGF0YSAgWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUkuIEJhdGF0YXMgKEwuKSBMYW0uIFRyb3AuIEFnci4gRGV2ZWxvcC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz05LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZmlyc3QgNTAgcGxhbnQgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPU1pY2hhZWwgIFQuUC47IGNpdGF0aW9uX2F1dGhvcj1KYWNrc29uICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIDEwMDEgZ2Vub21lcyBwcm9qZWN0IGZvciBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgY2l0YXRpb25fYXV0aG9yPVdlaWdlbCAgRC47IGNpdGF0aW9uX2F1dGhvcj1Nb3R0ICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzZXF1ZW5jaW5nIDUwIGFjY2Vzc2lvbnMgb2YgY3VsdGl2YXRlZCBhbmQgd2lsZCByaWNlIHlpZWxkcyBtYXJrZXJzIGZvciBpZGVudGlmeWluZyBhZ3Jvbm9taWNhbGx5IGltcG9ydGFudCBnZW5lczsgY2l0YXRpb25fYXV0aG9yPVh1ICBYLjsgY2l0YXRpb25fYXV0aG9yPUxpdSAgWC47IGNpdGF0aW9uX2F1dGhvcj1HZSAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMDUtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGdlbmV0aWMgY2hhbmdlcyBkdXJpbmcgbW9kZXJuIGJyZWVkaW5nIG9mIG1haXplOyBjaXRhdGlvbl9hdXRob3I9SmlhbyAgWS47IGNpdGF0aW9uX2F1dGhvcj1aaGFvICBILjsgY2l0YXRpb25fYXV0aG9yPVJlbiAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9ODEyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9zYWJsZSBlbGVtZW50cyBhbmQgdGhlIHBsYW50IHBhbi1nZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9TW9yZ2FudGUgIE0uOyBjaXRhdGlvbl9hdXRob3I9RGUgUGFvbGkgIEUuOyBjaXRhdGlvbl9hdXRob3I9UmFkb3ZpYyAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTQ5LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5hbHlzaXMgb2YgbXVsdGlwbGUgcGF0aG9nZW5pYyBpc29sYXRlcyBvZiBTdHJlcHRvY29jY3VzIGFnYWxhY3RpYWU6IGltcGxpY2F0aW9ucyBmb3IgdGhlIG1pY3JvYmlhbCDigJhwYW4tZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9VGV0dGVsaW4gIEguOyBjaXRhdGlvbl9hdXRob3I9TWFzaWduYW5pICBWLjsgY2l0YXRpb25fYXV0aG9yPUNpZXNsZXdpY3ogIE0uSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9MTM5NTAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIEROQSBzZXF1ZW5jZSBub25ob21vbG9naWVzIGFtb25nIG1haXplIGluYnJlZHM7IGNpdGF0aW9uX2F1dGhvcj1CcnVubmVyICBTLjsgY2l0YXRpb25fYXV0aG9yPUZlbmdsZXIgIEsuOyBjaXRhdGlvbl9hdXRob3I9TW9yZ2FudGUgIE0uOyBjaXRhdGlvbl9hdXRob3I9VGluZ2V5ICBTLjsgY2l0YXRpb25fYXV0aG9yPVJhZmFsc2tpICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTM0My02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBhc3NlbWJseSBvZiBzb3liZWFuIHdpbGQgcmVsYXRpdmVzIGZvciBwYW4tZ2Vub21lIGFuYWx5c2lzIG9mIGRpdmVyc2l0eSBhbmQgYWdyb25vbWljIHRyYWl0czsgY2l0YXRpb25fYXV0aG9yPUxpICBZLkguOyBjaXRhdGlvbl9hdXRob3I9WmhvdSAgRy47IGNpdGF0aW9uX2F1dGhvcj1NYSAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xMDQ1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIHZhcmlhdGlvbnMgaW4gcGxhbnQgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPVNheGVuYSAgUi5LLjsgY2l0YXRpb25fYXV0aG9yPUVkd2FyZHMgIEQuOyBjaXRhdGlvbl9hdXRob3I9VmFyc2huZXkgIFIuSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyaWVmIEZ1bmN0LiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI5Ni0zMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGlzb2xhdGlvbiBvZiBoaWdoLW1vbGVjdWxhci13ZWlnaHQgcGxhbnQgRE5BOyBjaXRhdGlvbl9hdXRob3I9TXVycmF5ICBNLkcuOyBjaXRhdGlvbl9hdXRob3I9VGhvbXBzb24gIFcuRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDMyMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIHN0cnVjdHVyYWwgYW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBvZiByZWQgY2xvdmVyIChUcmlmb2xpdW0gcHJldGVuc2UgTC4pOyBjaXRhdGlvbl9hdXRob3I9U2F0byAgUy47IGNpdGF0aW9uX2F1dGhvcj1Jc29iZSAgUy47IGNpdGF0aW9uX2F1dGhvcj1Bc2FtaXp1ICBFLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zMDEtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlsbHVtaW5hIG1hdGUtcGFpcmVkIEROQSBzZXF1ZW5jaW5nLWxpYnJhcnkgcHJlcGFyYXRpb24gdXNpbmcgQ3JlLUxveCByZWNvbWJpbmF0aW9uOyBjaXRhdGlvbl9hdXRob3I9TmlldXdlcmJ1cmdoICBGLlYuOyBjaXRhdGlvbl9hdXRob3I9VGhvbXBzb24gIFIuQy47IGNpdGF0aW9uX2F1dGhvcj1MZWRlc21hICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ZTI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbGl0eSBjb250cm9sIGFuZCBwcmVwcm9jZXNzaW5nIG9mIG1ldGFnZW5vbWljIGRhdGFzZXRzOyBjaXRhdGlvbl9hdXRob3I9U2NobWllZGVyICBSLjsgY2l0YXRpb25fYXV0aG9yPUVkd2FyZHMgIFIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTg2My00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZhc3QsIGxvY2stZnJlZSBhcHByb2FjaCBmb3IgZWZmaWNpZW50IHBhcmFsbGVsIGNvdW50aW5nIG9mIG9jY3VycmVuY2VzIG9mIGstbWVyczsgY2l0YXRpb25fYXV0aG9yPU1hcsOnYWlzICBHLjsgY2l0YXRpb25fYXV0aG9yPUtpbmdzZm9yZCAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NzY0LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IG9mIGh1bWFuIGdlbm9tZXMgd2l0aCBtYXNzaXZlbHkgcGFyYWxsZWwgc2hvcnQgcmVhZCBzZXF1ZW5jaW5nOyBjaXRhdGlvbl9hdXRob3I9TGkgIFIuOyBjaXRhdGlvbl9hdXRob3I9Wmh1ICBILjsgY2l0YXRpb25fYXV0aG9yPVJ1YW4gIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI2NS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGRlIG5vdm8gYXNzZW1ibHkgb2YgaGlnaGx5IGhldGVyb3p5Z291cyBnZW5vbWVzIGZyb20gd2hvbGUtZ2Vub21lIHNob3RndW4gc2hvcnQgcmVhZHM7IGNpdGF0aW9uX2F1dGhvcj1LYWppdGFuaSAgUi47IGNpdGF0aW9uX2F1dGhvcj1Ub3NoaW1vdG8gIEsuOyBjaXRhdGlvbl9hdXRob3I9Tm9ndWNoaSAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTM4NC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NhZmZvbGRpbmcgcHJlLWFzc2VtYmxlZCBjb250aWdzIHVzaW5nIFNTUEFDRTsgY2l0YXRpb25fYXV0aG9yPUJvZXR6ZXIgIE0uOyBjaXRhdGlvbl9hdXRob3I9SGVua2VsICBDLlYuOyBjaXRhdGlvbl9hdXRob3I9SmFuc2VuICBILkouOyBjaXRhdGlvbl9hdXRob3I9QnV0bGVyICBELjsgY2l0YXRpb25fYXV0aG9yPVBpcm92YW5vICBXLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyBjaXRhdGlvbl9hdXRob3I9QWx0c2NodWwgIFMuRi47IGNpdGF0aW9uX2F1dGhvcj1HaXNoICBXLjsgY2l0YXRpb25fYXV0aG9yPU1pbGxlciAgVy47IGNpdGF0aW9uX2F1dGhvcj1NeWVycyAgRS5XLjsgY2l0YXRpb25fYXV0aG9yPUxpcG1hbiAgRC5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgMzEzLWtiIGdlbm9taWMgcmVnaW9uIGNvbnRhaW5pbmcgdGhlIHNlbGYtaW5jb21wYXRpYmlsaXR5IGxvY3VzIG9mIElwb21vZWEgdHJpZmlkYSwgYSBkaXBsb2lkIHJlbGF0aXZlIG9mIHN3ZWV0IHBvdGF0bzsgY2l0YXRpb25fYXV0aG9yPVRvbWl0YSAgUi5OLjsgY2l0YXRpb25fYXV0aG9yPVN1enVraSAgRy47IGNpdGF0aW9uX2F1dGhvcj1Zb3NoaWRhICBLLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJlZWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0xNjUtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlcnNhdGlsZSBhbmQgb3BlbiBzb2Z0d2FyZSBmb3IgY29tcGFyaW5nIGxhcmdlIGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1LdXJ0eiAgUy47IGNpdGF0aW9uX2F1dGhvcj1QaGlsbGlwcHkgIEEuOyBjaXRhdGlvbl9hdXRob3I9RGVsY2hlciAgQS5MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1SMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGFwdGl2ZSBzZWVkcyB0YW1lIGdlbm9taWMgc2VxdWVuY2UgY29tcGFyaXNvbjsgY2l0YXRpb25fYXV0aG9yPUtpZWxiYXNhICBTLk0uOyBjaXRhdGlvbl9hdXRob3I9V2FuICBSLjsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIEsuOyBjaXRhdGlvbl9hdXRob3I9SG9ydG9uICBQLjsgY2l0YXRpb25fYXV0aG9yPUZyaXRoICBNLkMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBpZGVudGlmaWNhdGlvbiBvZiByZXBlYXQgZmFtaWxpZXMgaW4gbGFyZ2UgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPVByaWNlICBBLkwuOyBjaXRhdGlvbl9hdXRob3I9Sm9uZXMgIE4uQy47IGNpdGF0aW9uX2F1dGhvcj1QZXZ6bmVyICBQLkEuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgZWdncGxhbnQgKFNvbGFudW0gbWVsb25nZW5hIEwuKTogdGhlIHJlcHJlc2VudGF0aXZlIFNvbGFudW0gc3BlY2llcyBpbmRpZ2Vub3VzIHRvIHRoZSBPbGQgV29ybGQ7IGNpdGF0aW9uX2F1dGhvcj1IaXJha2F3YSAgSC47IGNpdGF0aW9uX2F1dGhvcj1TaGlyYXNhd2EgIEsuOyBjaXRhdGlvbl9hdXRob3I9TWl5YXRha2UgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTY0OS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NpUm9LbzogYSBuZXcgdG9vbCBmb3Igd2hvbGUgZ2Vub21lIG1pY3Jvc2F0ZWxsaXRlIHNlYXJjaCBhbmQgaW52ZXN0aWdhdGlvbjsgY2l0YXRpb25fYXV0aG9yPUtvZmxlciAgUi47IGNpdGF0aW9uX2F1dGhvcj1TY2hsb3R0ZXJlciAgQy47IGNpdGF0aW9uX2F1dGhvcj1MZWxsZXkgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE2ODMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQXNjYW4tU0U6IGEgcHJvZ3JhbSBmb3IgaW1wcm92ZWQgZGV0ZWN0aW9uIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBpbiBnZW5vbWljIHNlcXVlbmNlOyBjaXRhdGlvbl9hdXRob3I9TG93ZSAgVC5NLjsgY2l0YXRpb25fYXV0aG9yPUVkZHkgIFMuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTk1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBwcmVkaWN0aW9uIHdpdGggYSBoaWRkZW4gTWFya292IG1vZGVsIGFuZCBhIG5ldyBpbnRyb24gc3VibW9kZWw7IGNpdGF0aW9uX2F1dGhvcj1TdGFua2UgIE0uOyBjaXRhdGlvbl9hdXRob3I9V2FhY2sgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX2lzc3VlPShTdXBwbC4gMik7ICBjaXRhdGlvbl9wYWdlcz1paTIxNS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZW1ibGluZyBnZW5lcyBmcm9tIHByZWRpY3RlZCBleG9ucyBpbiBsaW5lYXIgdGltZSB3aXRoIGR5bmFtaWMgcHJvZ3JhbW1pbmc7IGNpdGF0aW9uX2F1dGhvcj1HdWlnw7MgIFIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDb21wdXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz02ODEtNzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW46IHByb3RlaW4gZG9tYWlucyBpZGVudGlmaWVyOyBjaXRhdGlvbl9hdXRob3I9UXVldmlsbG9uICBFLjsgY2l0YXRpb25fYXV0aG9yPVNpbHZlbnRvaW5lbiAgVi47IGNpdGF0aW9uX2F1dGhvcj1QaWxsYWkgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz1XMTE2LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgZGV2ZWxvcG1lbnRzIGluIHRoZSBJbnRlclBybyBkYXRhYmFzZTsgY2l0YXRpb25fYXV0aG9yPU11bGRlciAgTi5KLjsgY2l0YXRpb25fYXV0aG9yPUFwd2VpbGVyICBSLjsgY2l0YXRpb25fYXV0aG9yPUF0dHdvb2QgIFQuSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQyMjQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgZ2VuZXJhdGlvbiBvZiBob21vbG9neSBzZWFyY2ggdG9vbHMgYmFzZWQgb24gcHJvYmFiaWxpc3RpYyBpbmZlcmVuY2U7IGNpdGF0aW9uX2F1dGhvcj1FZGR5ICBTLlIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgSW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMDUtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgb250b2xvZ3k6IHRvb2wgZm9yIHRoZSB1bmlmaWNhdGlvbiBvZiBiaW9sb2d5OyBjaXRhdGlvbl9hdXRob3I9VGhlIEdlbmUgT250b2xvZ3kgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ09HIGRhdGFiYXNlOiBhbiB1cGRhdGVkIHZlcnNpb24gaW5jbHVkZXMgZXVrYXJ5b3RlczsgY2l0YXRpb25fYXV0aG9yPVRhdHVzb3YgIFIuTC47IGNpdGF0aW9uX2F1dGhvcj1GZWRvcm92YSAgTi5ELjsgY2l0YXRpb25fYXV0aG9yPUphY2tzb24gIEouRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LRUdHOiBLeW90byBFbmN5Y2xvcGVkaWEgb2YgR2VuZXMgYW5kIEdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1PZ2F0YSAgSC47IGNpdGF0aW9uX2F1dGhvcj1Hb3RvICBTLjsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIEsuOyBjaXRhdGlvbl9hdXRob3I9RnVqaWJ1Y2hpICBXLjsgY2l0YXRpb25fYXV0aG9yPUJvbm8gIEguOyBjaXRhdGlvbl9hdXRob3I9S2FuZWhpc2EgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yOS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENsdXN0YWxYIHdpbmRvd3MgaW50ZXJmYWNlOiBmbGV4aWJsZSBzdHJhdGVnaWVzIGZvciBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYWlkZWQgYnkgcXVhbGl0eSBhbmFseXNpcyB0b29sczsgY2l0YXRpb25fYXV0aG9yPVRob21wc29uICBKLkQuOyBjaXRhdGlvbl9hdXRob3I9R2lic29uICBULkouOyBjaXRhdGlvbl9hdXRob3I9UGxld25pYWsgIEYuOyBjaXRhdGlvbl9hdXRob3I9SmVhbm1vdWdpbiAgRi47IGNpdGF0aW9uX2F1dGhvcj1IaWdnaW5zICBELkcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00ODc2LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNjogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB2ZXJzaW9uIDYuMDsgY2l0YXRpb25fYXV0aG9yPVRhbXVyYSAgSy47IGNpdGF0aW9uX2F1dGhvcj1TdGVjaGVyICBHLjsgY2l0YXRpb25fYXV0aG9yPVBldGVyc29uICBELjsgY2l0YXRpb25fYXV0aG9yPUZpbGlwc2tpICBBLjsgY2l0YXRpb25fYXV0aG9yPUt1bWFyICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI3MjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBnYXBwZWQtcmVhZCBhbGlnbm1lbnQgd2l0aCBCb3d0aWUgMjsgY2l0YXRpb25fYXV0aG9yPUxhbmdtZWFkICBCLjsgY2l0YXRpb25fYXV0aG9yPVNhbHpiZXJnICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNlcXVlbmNlIGFsaWdubWVudC9tYXAgKFNBTSkgZm9ybWF0IGFuZCBTQU10b29sczsgY2l0YXRpb25fYXV0aG9yPUxpICBILjsgY2l0YXRpb25fYXV0aG9yPUhhbmRzYWtlciAgQi47IGNpdGF0aW9uX2F1dGhvcj1XeXNva2VyICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBWYXJpYW50IENhbGwgRm9ybWF0IGFuZCBWQ0Z0b29sczsgY2l0YXRpb25fYXV0aG9yPURhbmVjZWsgIFAuOyBjaXRhdGlvbl9hdXRob3I9QXV0b24gIEEuOyBjaXRhdGlvbl9hdXRob3I9QWJlY2FzaXMgIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTIxNTYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcm9ncmFtIGZvciBhbm5vdGF0aW5nIGFuZCBwcmVkaWN0aW5nIHRoZSBlZmZlY3RzIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMsIFNucEVmZjogU05QcyBpbiB0aGUgZ2Vub21lIG9mIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIHN0cmFpbiB3MTExODsgaXNvLTI7IGlzby0zOyBjaXRhdGlvbl9hdXRob3I9Q2luZ29sYW5pICBQLjsgY2l0YXRpb25fYXV0aG9yPVBsYXR0cyAgQS47IGNpdGF0aW9uX2F1dGhvcj1XYW5nbGUgIEwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GbHkgKEF1c3Rpbik7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTgwLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DTlYtc2VxLCBhIG5ldyBtZXRob2QgdG8gZGV0ZWN0IGNvcHkgbnVtYmVyIHZhcmlhdGlvbiB1c2luZyBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZzsgY2l0YXRpb25fYXV0aG9yPVhpZSAgQy47IGNpdGF0aW9uX2F1dGhvcj1UYW1taSAgTS5ULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJlIG9mIHRoZSBMZWd1bWU7IGNpdGF0aW9uX2F1dGhvcj1TYXRvICBTLjsgY2l0YXRpb25fYXV0aG9yPU5ha2FtdXJhICBZLjsgY2l0YXRpb25fYXV0aG9yPUthbmVrbyAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUxvdHVzIEphcG9uaWNhcywgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMjctMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBhbmQgYW5hbHlzaXMgb2YgdGhlIHR1YmVyIGNyb3AgcG90YXRvOyBjaXRhdGlvbl9hdXRob3I9VGhlIFBvdGF0byBHZW5vbWUgU2VxdWVuY2luZyBDb25zb3J0aXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDc1OyAgY2l0YXRpb25fcGFnZXM9MTg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdG9tYXRvIGdlbm9tZSBzZXF1ZW5jZSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGZsZXNoeSBmcnVpdCBldm9sdXRpb247IGNpdGF0aW9uX2F1dGhvcj1UaGUgVG9tYXRvIEdlbm9tZSBDb25zb3J0aXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg1OyAgY2l0YXRpb25fcGFnZXM9NjM1LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmhpYml0aW9uIG9mIHRoZSBleHByZXNzaW9uIG9mIHRoZSBzdGFyY2ggc3ludGhhc2UgSUkgZ2VuZSBsZWFkcyB0byBsb3dlciBwYXN0aW5nIHRlbXBlcmF0dXJlIGluIHN3ZWV0cG90YXRvIHN0YXJjaDsgY2l0YXRpb25fYXV0aG9yPVRha2FoYXRhICBZLjsgY2l0YXRpb25fYXV0aG9yPVRhbmFrYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1PdGFuaSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTUzNS00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wcmVoZW5zaXZlIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgdGhlIHN0YXJjaCBzeW50aGFzZSBnZW5lIGZhbWlseSBpbiByaWNlIChPcnl6YSBzYXRpdmEgTC4pOyBjaXRhdGlvbl9hdXRob3I9SGlyb3NlICBULjsgY2l0YXRpb25fYXV0aG9yPVRlcmFvICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnRhOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIyMDsgIGNpdGF0aW9uX3BhZ2VzPTktMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGVhdGVkIHBvbHlwbG9pZGl6YXRpb24gb2YgR29zc3lwaXVtIGdlbm9tZXMgYW5kIHRoZSBldm9sdXRpb24gb2Ygc3Bpbm5hYmxlIGNvdHRvbiBmaWJyZXM7IGNpdGF0aW9uX2F1dGhvcj1QZXRlcnNvbiAgQS47IGNpdGF0aW9uX2F1dGhvcj1XZW5kZWwgIEouRi47IGNpdGF0aW9uX2F1dGhvcj1HdW5kbGFjaCAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTI7ICBjaXRhdGlvbl9wYWdlcz00MjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJhbmFuYSAoTXVzYSBhY3VtaW5hdGEpIGdlbm9tZSBhbmQgdGhlIGV2b2x1dGlvbiBvZiBtb25vY290eWxlZG9ub3VzIHBsYW50czsgY2l0YXRpb25fYXV0aG9yPUQnSG9udCAgQS47IGNpdGF0aW9uX2F1dGhvcj1EZW5vZXVkICBGLjsgY2l0YXRpb25fYXV0aG9yPUF1cnkgIEouTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODg7ICBjaXRhdGlvbl9wYWdlcz0yMTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzc2VjdGlvbiBvZiB0aGUgb2N0b3Bsb2lkIHN0cmF3YmVycnkgZ2Vub21lIGJ5IGRlZXAgc2VxdWVuY2luZyBvZiB0aGUgZ2Vub21lcyBvZiBGcmFnYXJpYSBzcGVjaWVzOyBjaXRhdGlvbl9hdXRob3I9SGlyYWthd2EgIEguOyBjaXRhdGlvbl9hdXRob3I9U2hpcmFzYXdhICBLLjsgY2l0YXRpb25fYXV0aG9yPUtvc3VnaSAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTY5LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvMi8xNzEvOTUyMjQyL2RzdjAwMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSXBvbW9lYSB0cmlmaWRhIChILiBCLiBLLikgRy4gRG9uLiBpcyB0aGUgbW9zdCBsaWtlbHkgZGlwbG9pZCBhbmNlc3RvciBvZiB0aGUgaGV4YXBsb2lkIHN3ZWV0IHBvdGF0bywgSS4gYmF0YXRhcyAoTC4pIExhbS4gVG8gYXNzaXN0IGluIGFuYWx5c2lzIG9mIHQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTMzMzAyNSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQSEUKeNOJCcTo+g9Wk4BndVNZPZg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f171%2f333025" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f171%2f333025" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12675" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&amp;Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/2">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4598260">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598260">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598262">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598262">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598269">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598269">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598287">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598287">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598347">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598347">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598362">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598362">5. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598366">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598366">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598369">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598369">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598374">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598374">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4598379">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4598379">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/2/161/332908">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/2/181/333100">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Survey of genome sequences in a wild sweet potato, <em>Ipomoea trifida</em> (H. B. K.) G. Don
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshihiro Okada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Crop and Agribusiness Research Division, Kyushu Okinawa Agricultural Research Center</div>, <div class="insititution">National Agriculture and Food Research Organization (NARO/KARC)</div>, <div class="addr-line">Itoman, Okinawa 901-0336</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshihiro+Okada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okada Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okada Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshihiro Okada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroaki Tabuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center</div>, <div class="insititution">National Agriculture and Food Research Organization (NARO/KARC)</div>, <div class="addr-line">Miyakonojo, Miyazaki 885-0091</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroaki+Tabuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabuchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabuchi H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroaki Tabuchi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisano Tsuruoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisano+Tsuruoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuruoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuruoka H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hisano Tsuruoka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chiharu Minami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chiharu+Minami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minami C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Minami C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Chiharu Minami</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinobu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinobu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shinobu Nakayama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigemi Sasamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigemi+Sasamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shigemi Sasamoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshie Kishida</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsunakazu Fujishiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsunakazu+Fujishiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujishiro T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujishiro T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsunakazu Fujishiro</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Midori Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Midori+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Midori Kato</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keiko Nanri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keiko+Nanri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nanri K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nanri K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Keiko Nanri</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Komaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Komaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Komaki A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Komaki A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akiko Komaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaru Yoshinaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center</div>, <div class="insititution">National Agriculture and Food Research Organization (NARO/KARC)</div>, <div class="addr-line">Miyakonojo, Miyazaki 885-0091</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaru+Yoshinaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoshinaga M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoshinaga M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masaru Yoshinaga</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuhiro Takahata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center</div>, <div class="insititution">National Agriculture and Food Research Organization (NARO/KARC)</div>, <div class="addr-line">Miyakonojo, Miyazaki 885-0091</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuhiro+Takahata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahata Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahata Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yasuhiro Takahata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaru Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Upland Farming Research Division, Kyushu Okinawa Agricultural Research Center</div>, <div class="insititution">National Agriculture and Food Research Organization (NARO/KARC)</div>, <div class="addr-line">Miyakonojo, Miyazaki 885-0091</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaru+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masaru Tanaka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko N. Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3928. Fax. +81 438-52-3934. E-mail: <a href="mailto:sisobe@kazusa.or.jp" target="_blank">sisobe@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+N.+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sachiko N. Isobe</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: NARO Headquarters, Kannondai 3-1-1, Tsukuba, Ibaraki, 305-8517, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN3"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 2, 1 April 2015, Pages 171–179, <a href='https://doi.org/10.1093/dnares/dsv002'>https://doi.org/10.1093/dnares/dsv002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 March 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 November 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">17 February 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="333025" href="/dnaresearch/article-pdf/22/2/171/952242/dsv002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hideki Hirakawa, Yoshihiro Okada, Hiroaki Tabuchi, Kenta Shirasawa, Akiko Watanabe, Hisano Tsuruoka, Chiharu Minami, Shinobu Nakayama, Shigemi Sasamoto, Mitsuyo Kohara, Yoshie Kishida, Tsunakazu Fujishiro, Midori Kato, Keiko Nanri, Akiko Komaki, Masaru Yoshinaga, Yasuhiro Takahata, Masaru Tanaka, Satoshi Tabata, Sachiko N. Isobe;  Survey of genome sequences in a wild sweet potato, <em>Ipomoea trifida</em> (H. B. K.) G. Don, <em>DNA Research</em>, Volume 22, Issue 2, 1 April 2015, Pages 171–179, <a href='https://doi.org/10.1093/dnares/dsv002'>https://doi.org/10.1093/dnares/dsv002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/333025?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/333025?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/333025?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/333025?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/333025?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/333025?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/333025?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Survey%20of%20genome%20sequences%20in%20a%20wild%20sweet%20potato%2C%20%26lt%3Bem%26gt%3BIpomoea%20trifida%26lt%3B%2Fem%26gt%3B%20(H.%20B.%20K.)%20G.%20Don&amp;publicationDate=2015-03-24&amp;volumeNum=22&amp;issueNum=2&amp;author=Hirakawa%2C%20Hideki%3B%20Okada%2C%20Yoshihiro&amp;startPage=171&amp;endPage=179&amp;contentId=10.1093%2Fdnares%2Fdsv002&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f171%2f333025" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4598260 id="4598260" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Ipomoea trifida</em> (H. B. K.) G. Don. is the most likely diploid ancestor of the hexaploid sweet potato, <em>I. batatas</em> (L.) Lam. To assist in analysis of the sweet potato genome, <em>de novo</em> whole-genome sequencing was performed with two lines of <em>I. trifida</em>, namely the selfed line Mx23Hm and the highly heterozygous line 0431-1, using the Illumina HiSeq platform. We classified the sequences thus obtained as either ‘core candidates’ (common to the two lines) or ‘line specific’. The total lengths of the assembled sequences of Mx23Hm (ITR_r1.0) was 513 Mb, while that of 0431-1 (ITRk_r1.0) was 712 Mb. Of the assembled sequences, 240 Mb (Mx23Hm) and 353 Mb (0431-1) were classified into core candidate sequences. A total of 62,407 (62.4 Mb) and 109,449 (87.2 Mb) putative genes were identified, respectively, in the genomes of Mx23Hm and 0431-1, of which 11,823 were derived from core sequences of Mx23Hm, while 28,831 were from the core candidate sequence of 0431-1. There were a total of 1,464,173 single-nucleotide polymorphisms and 16,682 copy number variations (CNVs) in the two assembled genomic sequences (under the condition of log<sub>2</sub> ratio of &gt;1 and CNV size &gt;1,000 bases). The results presented here are expected to contribute to the progress of genomic and genetic studies of <em>I. trifida</em>, as well as studies of the sweet potato and the genus <em>Ipomoea</em> in general.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Ipomoea trifida&quot;"><em>Ipomoea trifida</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequence assembly&quot;">genome sequence assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;core- and line-specific sequences&quot;">core- and line-specific sequences</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNPs">SNPs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="CNVs">CNVs</a></div>                            <h2 scrollto-destination=4598262 id="4598262" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Ipomoea</em> is the largest genus in the family Convolvulaceae, which consists of 600–700 species.<sup><a reveal-id="DSV002C1" data-open="DSV002C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Sweet potato (<em>Ipomoea batatas</em> (L.) Lam) is the only species in the genus <em>Ipomoea</em> that is widely cultivated and consumed as a crop around the world. It has a complex genome structure, with hexaploidy (2<em>N</em> = 6<em>x</em> = 90), and a large genome size (4.8–5.3 pg/2C nucleus).<sup><a reveal-id="DSV002C2" data-open="DSV002C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The complex nature of the sweet potato genome has obstructed genetic studies on agronomically important characteristics such as self- and cross-incompatibility,<sup><a reveal-id="DSV002C3" data-open="DSV002C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and thus, the progress in genetics and genomics in this species lags far behind that in other important crop species.</p><p><em>I. trifida</em> (H. B. K.) G. Don., a wild relative of sweet potato distributed around the Caribbean Sea, forms a polyploidy complex ranging from diploid (2<em>N</em> = 2<em>x</em> = 30) to hexaploid (2<em>N</em> = 6<em>x</em> = 90).<sup><a reveal-id="DSV002C4" data-open="DSV002C4" class="link link-ref link-reveal xref-bibr">4</a></sup><em>I. trifida</em> and sweet potato are closely related, because they are cross-compatible.<sup><a reveal-id="DSV002C5  DSV002C7" data-open="DSV002C5  DSV002C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> Molecular genetic<sup><a reveal-id="DSV002C8  DSV002C11" data-open="DSV002C8  DSV002C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> and cytogenetic<sup><a reveal-id="DSV002C12" data-open="DSV002C12" class="link link-ref link-reveal xref-bibr">12</a></sup> data also support the close relationship of these two species. The discussion of polyploidization process in sweet potato is not complete; however, Shiotani and Kawase<sup><a reveal-id="DSV002C5" data-open="DSV002C5" class="link link-ref link-reveal xref-bibr">5</a></sup> suggested that sweet potato is an autohexaploid derived from diploid <em>I. trifida</em> based on cytogenetic analysis of a series of interspecific hybrids between sweet potato and <em>I. trifida</em>. In addition, molecular genetic data from simple sequence repeats (SSRs), non-coding chloroplast and nuclear ITS sequences implied an autohexaploid origin of sweet potato from an ancestor it shares with <em>I. trifida</em>.<sup><a reveal-id="DSV002C11" data-open="DSV002C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>Because of its cross-compatibility, status as the closest wild relative, and varied polyploidy, <em>I. trifida</em> is considered as a model species of sweet potato and is therefore used for genetic, physiological, and cytological analyses. In particular, the self-incompatibility system has often been studied in <em>I. trifida,</em> with the goal of achieving random crossing in sweet potato breeding in the future.<sup><a reveal-id="DSV002C13  DSV002C16" data-open="DSV002C13  DSV002C16" class="link link-ref link-reveal xref-bibr">13–16</a></sup> However, limited genetic and genomic resources have been developed in <em>I. trifida</em>. An amplified fragment length polymorphism (AFLP)-based linkage map for this species was first developed by Nakayama <em>et al.</em><sup><a reveal-id="DSV002C17" data-open="DSV002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The nucleotide sequences available to date (November 2014) in GenBank include 1,407 expressed sequence tags (ESTs) and 642 nucleotide or genome survey sequences.</p><p>With the advances in next-generation sequencing (NGS) technology, <em>de novo</em> whole-genome sequencing is no longer limited to a few plant species: to date, the whole-genome sequences of &gt;50 plant species have been published.<sup><a reveal-id="DSV002C18" data-open="DSV002C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Under these circumstances, plant scientists are further focussing on variations in genomes, with the goal of understanding the overall genome structure of a variety of germplasms with different characteristics of individual species. Whole-genome re-sequencing of multiple lines has been performed in several plant species, including <em>Arabidopsis thaliana</em>,<sup><a reveal-id="DSV002C19" data-open="DSV002C19" class="link link-ref link-reveal xref-bibr">19</a></sup> rice,<sup><a reveal-id="DSV002C20" data-open="DSV002C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and maize.<sup><a reveal-id="DSV002C21" data-open="DSV002C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>The accumulating information on the genome structures of plant germplasms has led to our interest in the ‘pan-genome concept’,<sup><a reveal-id="DSV002C22" data-open="DSV002C22" class="link link-ref link-reveal xref-bibr">22</a></sup> which was first proposed by Tettelin <em>et al.</em><sup><a reveal-id="DSV002C23" data-open="DSV002C23" class="link link-ref link-reveal xref-bibr">23</a></sup> in reference to the <em>Streptococcus agalactiae</em> genome. The pan-genome consists of a core genome that is present in all strains, and a dispensable genome composed of partially shared and strain-specific DNA sequences. Analyses of plant genomes based on a pan-genome perspective have already been performed in a few plant species to better understand the process of evolution and to accelerate the breeding process.<sup><a reveal-id="DSV002C24" data-open="DSV002C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSV002C25" data-open="DSV002C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In addition, investigation of structural variations (SVs), defined as genomic variations in the size range above 1 kb, using the NGS technology has also become more widespread in plant genomics.<sup><a reveal-id="DSV002C26" data-open="DSV002C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Genome sequencing by NGS can be straightforwardly adapted to validation of SVs, especially copy number variations (CNVs) and presence–absence variations. Detection of SVs throughout the genome, along with base-level variations such as single-nucleotide polymorphisms (SNPs), is expected to contribute to our understanding of phenotypic variation in species.</p><p><em>I. trifida</em> generally exhibits severe self-incompatibility and maintains heterozygosity within an accession. However, self-fertile lines were recently discovered by Kowyama <em>et al.</em><sup><a reveal-id="DSV002C13" data-open="DSV002C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and Nakayama <em>et al.</em><sup><a reveal-id="DSV002C17" data-open="DSV002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Using one such self-fertile line, we developed a single descendant selfed line (S<sub>11</sub>), named Mx23Hm, which is the progeny of a paternal line (Mx23-4) of a population used for construction of the first linkage map in <em>I. trifida</em>.<sup><a reveal-id="DSV002C17" data-open="DSV002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In this study, we performed <em>de novo</em> whole-genome sequencing for Mx23Hm using the Illumina sequencing platform. Whole-genome sequencing was also carried out for another <em>I. trifida</em> line, 0431-1, which exhibits heterozygosity and was used as the maternal line for the first linkage map. The independently assembled genomic sequences of both lines were classified as either ‘core candidates' (common to the two lines) or ‘line specific’. CNVs and SNPs in the two assembled sequences were also investigated to understand genome-wide variation in <em>I. trifida</em>. This is the first report of <em>de novo</em> whole-genome sequencing in the genus <em>Ipomoea</em>, and the results are expected to contribute to genomic and genetic analysis of <em>I. trifida</em>, as well as of sweet potato and the genus <em>Ipomoea</em> in general.</p>                            <h2 scrollto-destination=4598269 id="4598269" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4598270 id="4598270" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>Two lines of diploid <em>I. trifida</em>, Mx23Hm and 0431-1, were subjected to genome sequencing. Mx23Hm is a single descendant selfed line (S<sub>11</sub>) derived from the self-compatible experimental line Mx23-4, which was introduced from Mexico to Japan in 1961 and deposited in the <em>Ipomoea</em> collection of NARO/KARC. 0431-1 is a self-incompatible experimental line obtained by crosses between several diploid <em>I. trifida</em> lines introduced from Mexico and Colombia in 1973 and 1980, respectively. Genomic DNA was extracted from young leaves using the DNeasy Plant Mini Kit (Qiagen, Valencia, CA, USA) or a modified CTAB method.<sup><a reveal-id="DSV002C27" data-open="DSV002C27" class="link link-ref link-reveal xref-bibr">27</a></sup> DNA quantitation and quality checks were performed using a NanoDrop ND1000 spectrophotometer (NanoDrop Technologies, Wilmington, DE, USA) and 0.8% agarose gel electrophoresis, respectively.</p><p>Reduction of heterozygosity in the selfed descendants of Mx23-4 (S<sub>1</sub>, S<sub>7</sub>, and S<sub>10</sub> generation) was monitored using 14 SSR markers that identified heterozygous alleles in the S<sub>1</sub> plants. The 14 SSR markers were selected from 85 sweet potato EST-derived SSR markers developed at the Kazusa DNA Research Institute (unpublished). The primer sequences of the 14 SSR markers are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Genomic DNA was extracted from eight descent lines of each generation. Amplification of SSR markers was performed using a modified touchdown PCR protocol<sup><a reveal-id="DSV002C28" data-open="DSV002C28" class="link link-ref link-reveal xref-bibr">28</a></sup> in 20 μl reaction mixtures containing 20 ng of DNA, 200 μM dNTPs, 1 μM of each primer, 0.5 units of <em>Taq</em> DNA polymerase, and 1× <em>Taq</em> polymerase buffer. Five microlitres of each PCR product was subjected to electrophoresis on a MultiNA MCE-202 system (SHIMADZU Biotech, Tokyo).</p>                            <h3 scrollto-destination=4598273 id="4598273" class="section-title" data-legacy-id=s2b>2.2. Genome sequencing and genome size estimation</h3>
<p>Paired-end (PE) and mate-pair (MP) libraries were prepared using total cellular DNA from Mx23Hm and 0431-1. The PE libraries of both lines and the MP library of Mx23Hm were constructed according to the instruction provided by Illumina (San Diego, CA, USA). A modified protocol proposed by Nieuwerburgh <em>et al.</em><sup><a reveal-id="DSV002C29" data-open="DSV002C29" class="link link-ref link-reveal xref-bibr">29</a></sup> was employed for MP library preparation for 0431-1. The read length was 101 bases, and the expected insert size ranged from 0.3 to 20 kb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Sequence analyses were carried out on an Illumina HiSeq 2000 platform.</p><p>The obtained reads were subjected to quality control as follows. Bases with quality scores &lt;10 were filtered out by PRINSEQ 0.20.4.<sup><a reveal-id="DSV002C30" data-open="DSV002C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Adaptor sequences in the reads were trimmed using fastx_clipper of the FASTX-Toolkit 0.0.13 (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit">http://hannonlab.cshl.edu/fastx_toolkit</a>). After trimming, reads including <em>N</em> nucleotides, or with lengths &lt;100 bases, were excluded. The genome size of <em>I. trifida</em> was estimated based on the k-mer frequency of the sequence reads (<em>k</em> = 17) using Jellyfish ver. 2.1.1.<sup><a reveal-id="DSV002C31" data-open="DSV002C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p>                            <h3 scrollto-destination=4598276 id="4598276" class="section-title" data-legacy-id=s2c>2.3. Sequence assembly</h3>
<p>Two programs, SOAPdenovo2 r223<sup><a reveal-id="DSV002C32" data-open="DSV002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and Platanus ver.1.2.1,<sup><a reveal-id="DSV002C33" data-open="DSV002C33" class="link link-ref link-reveal xref-bibr">33</a></sup> were adopted for assembly of the Illumina PE reads. For SOAPdenovo2 r223, k-mer sizes from 41 to 95 were examined using default parameters, and the optimal k-mer size was selected from the N50 length in each k-mer size. The gaps on the scaffolds in each line were closed with the PE reads using GapCloser 1.10 (<em>P</em> = 31) (<a class="link link-uri" href="http://soap.genomics.org.cn/soapdenovo.html">http://soap.genomics.org.cn/soapdenovo.html</a>). The resultant sequences were then subjected to further scaffolding with the MP reads using SSPACE2.0 with the parameters (−k 3 −x 0).<sup><a reveal-id="DSV002C34" data-open="DSV002C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Potential contaminating sequences in the assembled scaffolds were detected and removed using BLASTN<sup><a reveal-id="DSV002C35" data-open="DSV002C35" class="link link-ref link-reveal xref-bibr">35</a></sup> searches against the chloroplast genome sequence of potato (<em>Solanum tuberosum</em>; accession number: NC_008096.2), the mitochondrial sequence of <em>A. thaliana</em> (accession number: NC_001284.2), bacteria, fungi, and human genome sequences registered in NCBI (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) and vector sequences from UniVec (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/">http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/</a>) with an <em>E</em>-value cut-off of 1E−10 and length coverage of ≥10%. The resultant scaffolds longer than 300 bases in length were selected and designated ITR_r1.0 for Mx23Hm and ITRk_r1.0 for 0431-1.</p><p>Assembly of the Illumina reads using Platanus was carried out for construction of the contigs and scaffolds. The gaps on the scaffolds were closed using a gap-closing program in Platanus. Authenticity of the assembled sequences was confirmed by comparing with the BAC (Bacterial Artificial Chromosome) contigs derived from haplotype S1 self-incompatibility (S-) locus of <em>I. trifida</em><sup><a reveal-id="DSV002C36" data-open="DSV002C36" class="link link-ref link-reveal xref-bibr">36</a></sup> registered in NCBI using a NUCmer program<sup><a reveal-id="DSV002C37" data-open="DSV002C37" class="link link-ref link-reveal xref-bibr">37</a></sup> followed by a two-dimensional dot plot analysis. The accession numbers of these BAC contigs were AY448010.1 (82,054 bases), AY448011.1 (9,363 bases), AY448012.1 (31,590 bases), AY448013.1 (87,724 bases), AY448014.1 (7,191 bases), AY448015.1 (39,512 bases), and AY448016.1 (55,205 bases).<sup><a reveal-id="DSV002C36" data-open="DSV002C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p><p>Core candidate and line-specific sequences were classified using LAST<sup><a reveal-id="DSV002C38" data-open="DSV002C38" class="link link-ref link-reveal xref-bibr">38</a></sup> ver. 490 with a score of 272 corresponding to an <em>E</em>-value cut-off of 1E−100. Repetitive sequences were detected using RepeatScount 1.0.5<sup><a reveal-id="DSV002C39" data-open="DSV002C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and RepeatMasker 3.30 (<a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) as described by Hirakawa <em>et al.</em><sup><a reveal-id="DSV002C40" data-open="DSV002C40" class="link link-ref link-reveal xref-bibr">40</a></sup> SSR motifs were identified using the SciRoKo software<sup><a reveal-id="DSV002C41" data-open="DSV002C41" class="link link-ref link-reveal xref-bibr">41</a></sup> in the ‘MISA’ mode with default parameters. The minimum numbers of SSR repeats for mono-, di-, tri-, tetra-, penta-, and hexa-nucleotides adopted for identification were 14, 7, 5, 4, 4, and 4, respectively.</p>                            <h3 scrollto-destination=4598280 id="4598280" class="section-title" data-legacy-id=s2d>2.4. Gene prediction and annotation</h3>
<p>Transfer RNA genes were predicted using tRNAscan-SE ver. 1.23<sup><a reveal-id="DSV002C42" data-open="DSV002C42" class="link link-ref link-reveal xref-bibr">42</a></sup> with default parameters, and ribosomal RNA (rRNA) genes were predicted by BLAST searches with an E-value cut-off of 1<em>E</em>-10. The <em>A. thaliana</em> 5.8S and 25S rRNAs (accession number: X52320.1) and 18S rRNA (accession number: X16077.1) were used as query sequences.</p><p>Protein-encoding sequences in the assembled genomic sequences were predicted by Augustus 2.7<sup><a reveal-id="DSV002C43" data-open="DSV002C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and geneid<sup><a reveal-id="DSV002C44" data-open="DSV002C44" class="link link-ref link-reveal xref-bibr">44</a></sup> programs using the <em>A. thaliana</em> training set under the default parameters. The training set of tomato genes provided by Augustus 2.7 was also examined for comparison. Reciprocal best-hit analysis was performed to compare the results of the prediction by using <em>A. thaliana</em> and tomato training sets. Genes related to transposable elements (TEs) were detected by BLASTP searches against the NCBI non-redundant (nr) protein database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) with an <em>E</em>-value cut-off of 1E−10, and InterProScan<sup><a reveal-id="DSV002C45" data-open="DSV002C45" class="link link-ref link-reveal xref-bibr">45</a></sup> searches against the InterPro database<sup><a reveal-id="DSV002C46" data-open="DSV002C46" class="link link-ref link-reveal xref-bibr">46</a></sup> with an <em>E</em>-value cut-off of 1.0.</p><p>The putative genes of <em>I. trifida</em> were clustered by using the CD-hit program<sup><a reveal-id="DSV002C47" data-open="DSV002C47" class="link link-ref link-reveal xref-bibr">47</a></sup> with the unigene sets of potato (<a class="link link-uri" href="http://potato.plantbiology.msu.edu/index.shtml">http://potato.plantbiology.msu.edu/index.shtml</a>) (<em>S. tuberosum</em>, PGSC DM v3.4, 56,218 genes), cassava (<a class="link link-uri" href="http://www.phytozome.net/cassava.php">http://www.phytozome.net/cassava.php</a>) (<em>Manihot esculenta</em>, v4.1, 34,151 genes), and <em>A. thaliana</em> (<a class="link link-uri" href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>) (TAIR10, 35,386 genes) with the parameters <em>c</em> = 0.4 and aS = 0.4. Genes in the plant species described above were classified into the plant gene ontology (GO) slim categories,<sup><a reveal-id="DSV002C48" data-open="DSV002C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and the ‘euKaryotic clusters of Orthologous Groups' (KOG) categories<sup><a reveal-id="DSV002C49" data-open="DSV002C49" class="link link-ref link-reveal xref-bibr">49</a></sup> and then mapped onto the Kyoto Encyclopedia of Genes and Genomes (KEGG) reference pathways<sup><a reveal-id="DSV002C50" data-open="DSV002C50" class="link link-ref link-reveal xref-bibr">50</a></sup> as described by Hirakawa <em>et al.</em><sup><a reveal-id="DSV002C39" data-open="DSV002C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Multiple alignment of amino acid sequences was performed for starch synthase homologues predicted in ITR_r1.0 using ClustalX,<sup><a reveal-id="DSV002C51" data-open="DSV002C51" class="link link-ref link-reveal xref-bibr">51</a></sup> and a genetic tree was constructed using the neighbour-joining algorithm of MEGA6.<sup><a reveal-id="DSV002C52" data-open="DSV002C52" class="link link-ref link-reveal xref-bibr">52</a></sup></p>                            <h3 scrollto-destination=4598284 id="4598284" class="section-title" data-legacy-id=s2e>2.5. SNP discovery and CNVs analysis</h3>
<p>The trimmed PE reads for Mx23Hm and 0431-1 described above were independently mapped onto ITR_r1.0 using Bowtie 2 2.2.34 (parameter maxins = 1000).<sup><a reveal-id="DSV002C53" data-open="DSV002C53" class="link link-ref link-reveal xref-bibr">53</a></sup> To eliminate possible alignment errors, the results with the following SAM Flags were employed: 83, 99, 147, and 163 for PE reads that were mapped on the same contig with correct read orientation and insert size, and 65, 81, 97, 113, 129, 145, 161, and 177 for PE reads that were mapped on unique positions of different contigs.</p><p>SNP candidates were called using samtools mpileup ver. 11.19 (parameters: -Duf -d 1000000),<sup><a reveal-id="DSV002C54" data-open="DSV002C54" class="link link-ref link-reveal xref-bibr">54</a></sup> followed by filtration using VCFtools ver. 1.1.19<sup><a reveal-id="DSV002C55" data-open="DSV002C55" class="link link-ref link-reveal xref-bibr">55</a></sup> and in-house Perl scripts according to the following criteria: (i) Bases at the SNP loci in Mx23Hm should exactly match the bases of ITR_r1.0 (to avoid false-positive SNP detection caused by sequencing or assembling error); (ii) all non-insertion or deletion mutations should not be included; (iii) quality scores of the SNP sites should be &gt;200; and (iv) SNPs should not be located in repetitive sequences. SNP effects on the functions of putative genes in ITR_r1.0 were predicted using SnpEff ver. 4.0e (parameters: -no-downstream, -no-upstream).<sup><a reveal-id="DSV002C56" data-open="DSV002C56" class="link link-ref link-reveal xref-bibr">56</a></sup> The <em>I. trifida</em> database for SNP annotation in SnpEff was constructed using the gff file generated by Augustus 3.0.2. CNVs were analysed using CNV-seq ver. 0.2.7 (parameters: -genome-size 239146348).<sup><a reveal-id="DSV002C57" data-open="DSV002C57" class="link link-ref link-reveal xref-bibr">57</a></sup></p>                            <h2 scrollto-destination=4598287 id="4598287" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4598288 id="4598288" class="section-title" data-legacy-id=s3a>3.1. Establishment of a selfed line, Mx23Hm</h3>
<p>Mx23-4, the parental line of Mx23Hm, exhibited less heterozygosity than 0431-1 in a previous study of linkage map construction.<sup><a reveal-id="DSV002C17" data-open="DSV002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> To obtain a highly homozygous descendant of Mx23-4, we performed successive selfing by the single seed descent method until generation S<sub>11</sub>. No heterozygous alleles were observed in any of the S<sub>7</sub> and S<sub>10</sub> descendants at 14 SSR loci that exhibited polymorphism in S<sub>1</sub> generation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Significant differences were not observed among the various generations in regard to morphology or fertility, except slightly smaller corolla size and seed weight in generations S<sub>7</sub>–S<sub>10</sub> (data not shown). One of the S<sub>11</sub> descendants was named Mx23Hm and subjected to genome sequencing, along with the heterozygous line 0431-1.</p>                            <h3 scrollto-destination=4598290 id="4598290" class="section-title" data-legacy-id=s3b>3.2. Genome sequencing and genome size estimation</h3>
<p>The total numbers of Illumina reads obtained from Mx23Hm was 1,322,328,000, while that from 0431-1 was 2,698,276,042 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). After trimming, 9–11% of PE reads and 38–45% of MP reads were excluded from further analysis. The total lengths of trimmed PE and MP reads in Mx23Hm were 64.5 and 33.9 Gb, respectively, and those in 0431-1 were 80.3 and 108.9 Gb, respectively.</p><p>Distributions of the number of distinct k-mers (<em>k</em> = 17) with the given multiplicity values reflected differences in heterozygosity between the two lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). One large and one very small peak were observed in Mx23Hm; the large peak with multiplicity of 105 was employed for genome size estimation. In contrast, two distinct peaks were observed in 0431-1; the higher peak with multiplicity of 62 was considered to represent heterozygous sequences, and the lower peak with multiplicity of 125 was employed for genome size estimation. The estimated genome sizes were 515.8 Mb (Mx23Hm) and 539.9 Mb (0431-1).</p>                            <h3 scrollto-destination=4598293 id="4598293" class="section-title" data-legacy-id=s3c>3.3. Assembly of the Mx23Hm and 0431-1 genomes</h3>
<p>Assembly of the Illumina reads described in the Section 3.2 was carried out using the two computer programs based on different algorithms, SOAPdenovo2 r223 and Platanus ver. 1.2.1, as described in Materials and Methods. The results are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. The total lengths of the assembled sequences generated by SOAPdenovo2 r223 were longer than those generated by Platanus after gap closing (columns G, K versus O, S in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>) and were closer to the estimated genome sizes described above (columns G, K, O and S in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Considering that the purpose of this study is to survey as much genomic regions as possible to assign core candidates and line-specific sequences in the <em>I. trifida</em> genome, SOAPdenovo2 r223 was chosen for the following analyses.</p><p>The Mx23Hm PE reads, which had a total length of 64.5 Gb, were assembled by using SOAPdenovo2 r223 and GapCloser 1.10. The 740,762 generated scaffolds were subjected to further scaffolding by using SSPACE2.0 with the MP reads having expected insert sizes of 3 and 10 kb, followed by the exclusion of contaminated DNA sequences such as those derived from bacterial, fungal, and human genomes. The resultant number of scaffolds was 559,634, totalling 584.7 Mb in length. The sequences shorter than 300 bases were considered insignificant, because they were likely to be derived from repetitive or low-quality sequences, and therefore, such sequences were excluded from further analysis. The remaining 77,400 sequences were designated as ITR_r1.0, the total length of which was 512,990,885 bases, including 175,412,753 Ns. The GC% and N50 length were 35.6% and 42,586 bases, respectively (Table <a reveal-id="DSV002TB1" data-open="DSV002TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). </p>                        <a id="4598296" scrollto-destination="4598296"></a>
<div content-id="DSV002TB1" class="table-modal"><div class="table-wrap"><div id="DSV002TB1" class="table-wrap-title" data-id="DSV002TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the assembled genome sequences for Mx23Hm and 0431-1</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Sequenced line </th><th>Mx23Hm </th><th>0431-1 </th></tr><tr><th>(ITR_r1.0) </th><th>(ITRk_r1.0) </th></tr></thead><tbody><tr><td>Number of sequences </td><td>77,400 </td><td>181,194 </td></tr><tr><td>Total length (bases) </td><td>512,990,885 </td><td>712,155,587 </td></tr><tr><td>Average length (bases) </td><td>6,628 </td><td>3,930 </td></tr><tr><td>Max length (bases) </td><td>910,847 </td><td>1,352,076 </td></tr><tr><td>Min length (bases) </td><td>300 </td><td>300 </td></tr><tr><td>N50 length (bases) </td><td>42,586 </td><td>36,283 </td></tr><tr><td>A </td><td>108,919,552 </td><td>155,339,270 </td></tr><tr><td>T </td><td>108,380,339 </td><td>154,432,148 </td></tr><tr><td>G </td><td>60,024,339 </td><td>86,821,603 </td></tr><tr><td>C </td><td>60,253,902 </td><td>87,276,414 </td></tr><tr><td><em>N</em> </td><td>175,412,753 </td><td>228,286,152 </td></tr><tr><td>Total (ATGC) </td><td>337,578,132 </td><td>483,869,435 </td></tr><tr><td>GC% (GC/ATGC) </td><td>35.6 </td><td>36.0 </td></tr></tbody></table></div><div content-id="DSV002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Sequenced line </th><th>Mx23Hm </th><th>0431-1 </th></tr><tr><th>(ITR_r1.0) </th><th>(ITRk_r1.0) </th></tr></thead><tbody><tr><td>Number of sequences </td><td>77,400 </td><td>181,194 </td></tr><tr><td>Total length (bases) </td><td>512,990,885 </td><td>712,155,587 </td></tr><tr><td>Average length (bases) </td><td>6,628 </td><td>3,930 </td></tr><tr><td>Max length (bases) </td><td>910,847 </td><td>1,352,076 </td></tr><tr><td>Min length (bases) </td><td>300 </td><td>300 </td></tr><tr><td>N50 length (bases) </td><td>42,586 </td><td>36,283 </td></tr><tr><td>A </td><td>108,919,552 </td><td>155,339,270 </td></tr><tr><td>T </td><td>108,380,339 </td><td>154,432,148 </td></tr><tr><td>G </td><td>60,024,339 </td><td>86,821,603 </td></tr><tr><td>C </td><td>60,253,902 </td><td>87,276,414 </td></tr><tr><td><em>N</em> </td><td>175,412,753 </td><td>228,286,152 </td></tr><tr><td>Total (ATGC) </td><td>337,578,132 </td><td>483,869,435 </td></tr><tr><td>GC% (GC/ATGC) </td><td>35.6 </td><td>36.0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4598296" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV002TB1" class="table-wrap-title" data-id="DSV002TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the assembled genome sequences for Mx23Hm and 0431-1</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Sequenced line </th><th>Mx23Hm </th><th>0431-1 </th></tr><tr><th>(ITR_r1.0) </th><th>(ITRk_r1.0) </th></tr></thead><tbody><tr><td>Number of sequences </td><td>77,400 </td><td>181,194 </td></tr><tr><td>Total length (bases) </td><td>512,990,885 </td><td>712,155,587 </td></tr><tr><td>Average length (bases) </td><td>6,628 </td><td>3,930 </td></tr><tr><td>Max length (bases) </td><td>910,847 </td><td>1,352,076 </td></tr><tr><td>Min length (bases) </td><td>300 </td><td>300 </td></tr><tr><td>N50 length (bases) </td><td>42,586 </td><td>36,283 </td></tr><tr><td>A </td><td>108,919,552 </td><td>155,339,270 </td></tr><tr><td>T </td><td>108,380,339 </td><td>154,432,148 </td></tr><tr><td>G </td><td>60,024,339 </td><td>86,821,603 </td></tr><tr><td>C </td><td>60,253,902 </td><td>87,276,414 </td></tr><tr><td><em>N</em> </td><td>175,412,753 </td><td>228,286,152 </td></tr><tr><td>Total (ATGC) </td><td>337,578,132 </td><td>483,869,435 </td></tr><tr><td>GC% (GC/ATGC) </td><td>35.6 </td><td>36.0 </td></tr></tbody></table></div><div content-id="DSV002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Sequenced line </th><th>Mx23Hm </th><th>0431-1 </th></tr><tr><th>(ITR_r1.0) </th><th>(ITRk_r1.0) </th></tr></thead><tbody><tr><td>Number of sequences </td><td>77,400 </td><td>181,194 </td></tr><tr><td>Total length (bases) </td><td>512,990,885 </td><td>712,155,587 </td></tr><tr><td>Average length (bases) </td><td>6,628 </td><td>3,930 </td></tr><tr><td>Max length (bases) </td><td>910,847 </td><td>1,352,076 </td></tr><tr><td>Min length (bases) </td><td>300 </td><td>300 </td></tr><tr><td>N50 length (bases) </td><td>42,586 </td><td>36,283 </td></tr><tr><td>A </td><td>108,919,552 </td><td>155,339,270 </td></tr><tr><td>T </td><td>108,380,339 </td><td>154,432,148 </td></tr><tr><td>G </td><td>60,024,339 </td><td>86,821,603 </td></tr><tr><td>C </td><td>60,253,902 </td><td>87,276,414 </td></tr><tr><td><em>N</em> </td><td>175,412,753 </td><td>228,286,152 </td></tr><tr><td>Total (ATGC) </td><td>337,578,132 </td><td>483,869,435 </td></tr><tr><td>GC% (GC/ATGC) </td><td>35.6 </td><td>36.0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4598296" target="_blank">View Large</a></div></div><p>In 0431-1, a total of 1,578,945 scaffolds generated by the assembly of 80.3 Gb PE reads were subjected to further scaffolding with MP reads, generating 1,578,945 scaffolds (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). After the removal of contaminated sequences and sequences shorter than 300 bases, a total of 181,194 sequences comprising 712,155,587 bases with 228,286,152 Ns were finally obtained, which were collectively designated as ITRk_r1.0 (Table <a reveal-id="DSV002TB1" data-open="DSV002TB1" class="link link-reveal link-table xref-fig">1</a>). The GC% and N50 length were 36.0% and 36,283 bases, respectively.</p><p>The assembled sequences were then classified into ‘core candidates' and ‘line-specific’ sequences using the LAST program. The total lengths of core candidates were 239.6 Mb (Mx23Hm) and 353.5 Mb (0431-1), whereas those of line-specific sequences were 273.4 Mb (Mx23Hm) and 358.6 Mb (0431-1) (Fig. <a reveal-id="DSV002F1" data-open="DSV002F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). In both assembled sequences, most of the <em>N</em> bases were classified as line-specific sequences: the <em>N</em>% of Mx23Hm and 0431-1 was 64.2 and 63.6%, respectively. </p>                        <a id="4598301" scrollto-destination="4598301"></a>
<div id="DSV002F1" data-id="DSV002F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/m_dsv00201.png?Expires=2147483647&Signature=KT7~MPGZkg6XfdR8SKdyfvtkjTuWpnU1HLhyPc6vJL3mnaOlh54xUcOQkafW4RhQJjjr65KParrnB3h64awqv4T2MNfjQJluCjM-l6k9OMZb9NVoBRtIpH6sB8Ed4F9d1Z6m84bQuD0BKdKmvItMeBOeepLmPBjEuLzJP84SgBMwTyoMY-JY1H~GbBBl7rJr090Xy8toHSqgb1XWuCZuN21WK4yxSrW8gKqvB3j-U~sTPk9VRS~f7DgVbYnPDIm3fRokBgVxiWdm9mzGRvddRr44VwGhjpvJs9xVhMmb9orfsjNXVXa-bB3rnuzZIXR3atM4E8QEQWIly1F9F-Qgsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Total lengths of core candidates and line-specific sequences of ITR_r1.0 (Mx23Hm) and ITRk_r1.0 (0431-1)." path-from-xml="dsv00201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4598301/dsv00201.tiff" path-from-xml="dsv00201.tiff" target="_blank">View large</a><a section="4598301" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv00201.png?Expires=2147483647&Signature=Nrzlj~E6ChbyDgI~4-nmGpvnTegMDQ9lxp2SAvwGYwyivWTdNajFm7Sgs-S-AtSIh15VlqKePOJgY4WjkBToCwOYP0umij4rRv234pfwmYMtLQWlgDkbQAdYLwDs5FC-8k5nqBqwnAJGsREBbWafjE51WsGYbdAbCicvG-9yMvPo59gWx6gEQh54OPDQO~RqOoTsCqTFZ~2s4pkZFzZwVC2kXrm7zB5iu8UXLsWroTyc1jPrcFxal93Hr-TLfJmT2OWIybYEWLey4o~kxN9ze6DioMJDH9HiY4mEJ4s58Y11Nqe9kHPz3z0Wai3zCPf7carSSOeX1xm7nAdCho3zzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4598301&ar=333025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Total lengths of core candidates and line-specific sequences of ITR_r1.0 (Mx23Hm) and ITRk_r1.0 (0431-1).</p></div></div><div content-id="DSV002F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/m_dsv00201.png?Expires=2147483647&Signature=KT7~MPGZkg6XfdR8SKdyfvtkjTuWpnU1HLhyPc6vJL3mnaOlh54xUcOQkafW4RhQJjjr65KParrnB3h64awqv4T2MNfjQJluCjM-l6k9OMZb9NVoBRtIpH6sB8Ed4F9d1Z6m84bQuD0BKdKmvItMeBOeepLmPBjEuLzJP84SgBMwTyoMY-JY1H~GbBBl7rJr090Xy8toHSqgb1XWuCZuN21WK4yxSrW8gKqvB3j-U~sTPk9VRS~f7DgVbYnPDIm3fRokBgVxiWdm9mzGRvddRr44VwGhjpvJs9xVhMmb9orfsjNXVXa-bB3rnuzZIXR3atM4E8QEQWIly1F9F-Qgsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Total lengths of core candidates and line-specific sequences of ITR_r1.0 (Mx23Hm) and ITRk_r1.0 (0431-1)." path-from-xml="dsv00201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4598301/dsv00201.tiff" path-from-xml="dsv00201.tiff" target="_blank">View large</a><a section="4598301" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv00201.png?Expires=2147483647&Signature=Nrzlj~E6ChbyDgI~4-nmGpvnTegMDQ9lxp2SAvwGYwyivWTdNajFm7Sgs-S-AtSIh15VlqKePOJgY4WjkBToCwOYP0umij4rRv234pfwmYMtLQWlgDkbQAdYLwDs5FC-8k5nqBqwnAJGsREBbWafjE51WsGYbdAbCicvG-9yMvPo59gWx6gEQh54OPDQO~RqOoTsCqTFZ~2s4pkZFzZwVC2kXrm7zB5iu8UXLsWroTyc1jPrcFxal93Hr-TLfJmT2OWIybYEWLey4o~kxN9ze6DioMJDH9HiY4mEJ4s58Y11Nqe9kHPz3z0Wai3zCPf7carSSOeX1xm7nAdCho3zzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4598301&ar=333025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Total lengths of core candidates and line-specific sequences of ITR_r1.0 (Mx23Hm) and ITRk_r1.0 (0431-1).</p></div></div><p>Authenticity of the assembled genomic sequences was examined by comparison with the registered BAC contig sequences in NCBI derived from the genome of <em>I. trifida</em>. Six out of seven BAC contigs tested showed sequence similarity to the assembled Mx23Hm and 0431-1 genomic sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Despite some inconsistency possibly due to improper assembly or structural changes of the genomes among varieties, long stretches of linear dot plots were observed for these regions, indicating high degree of reliability of the assembled sequences. Multiple linear plots were often observed in 0431-1, which may reflect higher heterozygosity of the genome of this line.</p>                            <h3 scrollto-destination=4598308 id="4598308" class="section-title" data-legacy-id=s3d>3.4. Repetitive sequences</h3>
<p>The total lengths of repetitive sequences were 142.7 Mb (Mx23Hm) and 230.8 Mb (0431-1) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Unique repeats were abundant in both genomes, constituting 76.0% (Mx23Hm) and 78.9% (0431-1) of all repeats. Of the known types of repeats, Class I LTR elements were observed most frequently, followed by low-complexity repeats. Percentages of the repetitive in the assembled sequences (excluding <em>N</em>) sequences accounted for 27.7 and 30.9% of the core candidate, and 80.6 and 96.9% of the line-specific sequences in Mx23Hm and 0431-1, respectively. The repeat masked sequences of the assembled genomic sequences were constructed and designated as ITR_r1.0_masked (Mx23Hm) and ITRk_r1.0_masked (0431-1). The numbers of total bases (A, T, G, and C) of the constructed repeat masked sequences were 192,898,192 bases (Mx23Hm) and 254,160,527 bases (0431-1, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The numbers of total bases in core candidates were 155,039,354 bases (Mx23Hm) and 205,173,148 bases, while those in line specific were 37,858,838 bases (Mx23Hm) and 48,987,379 bases (0431-1).</p><p>The total numbers of SSRs identified in the assembled genome sequences were 108,397 (Mx23Hm) and 95,138 (0431-1) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). The average frequency of SSRs was 20.9/100 kb in Mx23Hm, while that in 0431-1 was 13.1/100 kb. SSRs were more frequently observed in introns than in exons. Di-nucleotide motifs were more frequently observed than tri-nucleotide motifs in Mx23Hm (54,631 di-nucleotides and 34,619 tri-nucleotides), whereas a small difference was observed in the numbers of di- and tri-nucleotide motifs in 0431-1 (36,941 di-nucleotide and 35,489 tri-nucleotide).</p>                            <h3 scrollto-destination=4598311 id="4598311" class="section-title" data-legacy-id=s3e>3.5. RNA-encoding genes</h3>
<p>A total of 2,089 and 3,168 putative genes for transfer RNAs (tRNAs) were identified in the assembled genomic sequences of Mx23Hm and 0431-1, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Larger numbers of tRNA-encoding genes were identified in the core candidate sequences (1,335 in Mx23Hm and 2,006 in 0431-1) than in the line-specific sequences (754 in Mx23Hm and 1,162 in 0431-1). The total numbers of partial rRNA-encoding genes predicted in Mx23Hm and 0431-1 were 476 (5.8S: 12; 18S: 223; and 25S: 241) and 299 (5.8S: 16; 18S: 148; and 25S: 135), respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). The numbers of rRNA-encoding genes identified in the core candidates (65 in Mx23Hm and 76 in 0431-1) were less than those in line-specific sequences (411 in Mx23Hm and 223 in 0431-1).</p>                            <h3 scrollto-destination=4598313 id="4598313" class="section-title" data-legacy-id=s3f>3.6. Prediction of protein-encoding genes and annotation</h3>
<p>Two computer programs, Augustus and geneid, and two gene training sets, one derived from <em>A. thaliana</em> and the other from tomato, were used for <em>de novo</em> gene prediction. Augustus predicted 62,403 protein-encoding genes in the assembled genomic sequences of Mx23Hm using the <em>A. thaliana</em> training set, while geneid predicted 104,524 genes with the same training set (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). If only genes capable of encoding proteins of equal to or longer than 100 amino acid residues were taken into account, however, Augustus and geneid predicted similar number of genes, 50,689 and 50,515, respectively. When these two gene sets were compared, 26,364 genes (52.1% of the total predicted genes) were common to the both programs, and 1,274 (2.5%) and 1,956 (3.9%) genes were specific to Augustus and geneid, respectively, indicating that the two programs predicted similar sets of genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Gene prediction by Augustus using the two different gene training sets resulted in generation of similar number of genes, 62,403 (<em>A. thaliana</em>) and 59,182 (tomato) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>). Putting these results together, we adopted the gene sets predicted by Augustus using the <em>A. thaliana</em> training set for further analyses.</p><p>An N50 length of the 62,403 protein-encoding genes predicted in Mx23Hm was 1,536 bases, and totalled 62,468,431 bases in length (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). A larger number of genes (109,449) was predicted in 0431-1 with a total length of 82,228,566 bases, and an N50 length was 1,230 bases. The predicted genes were classified as intrinsic genes or TEs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13</a>); the percentages of intrinsic and partial genes (76%) and TEs (18–19%) were almost identical between the two assembled genomic sequences. The ratio of full-length intrinsic genes was higher in Mx23Hm, reflecting the higher quality of the assembled sequences. The numbers of the sum of intrinsic and partial genes were 47,511 with a total length of 44,591,152 bases for Mx23Hm, while that for 0431-1 was 83,366 with a total length of 58,192,632 bases (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). Sets of intrinsic and partial genes were designated as ITR_r1.0_cds_ip (Mx23Hm) and ITRk_r1.0_cds_ip (0431-1). The total numbers of predicted genes in core candidate and line-specific sequences were 11,823 and 50,580, respectively, in Mx23Hm, and 28,831 and 80,618, respectively, in 0431-1 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). The percentages of intrinsic and partial genes in core candidate sequences (82–83%) were slightly higher than that in line-specific sequences (74–75%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13</a>).</p><p>The intrinsic and partial genes were subjected to clustering with a total of 57,354 predicted genes containing 35,386 genes in <em>A. thaliana</em> as a dicot model and 56,218 genes in potato and 34,151 genes in cassava as tuber crops by similarity searches using the CD-hit program. Of the putative <em>I. trifida</em> genes, 15,184 (Mx23Hm) and 26,243 (0431-1) could be clustered with predicted genes identified in other species, whereas the remaining 5,849 (Mx23Hm) and 15,020 (0431-1) were not clustered and therefore considered as <em>I. trifida</em>-specific genes (Fig. <a reveal-id="DSV002F2" data-open="DSV002F2" class="link link-reveal link-table xref-fig">2</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S14 and S15</a>). </p>                        <a id="4598317" scrollto-destination="4598317"></a>
<div id="DSV002F2" data-id="DSV002F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/m_dsv00202.png?Expires=2147483647&Signature=dJRtOn2bqpAfJ9DS~V6Okbh1wT3jlyh7T0Fwjwz0tavl2zEWukl1Wl9dNSFfO5x72Ur3mqDyMQpsNR9WmNNfT~srkYWfKAGAmIAJpUPWqI1E60SfS0EKRLgKRsCW5ebGgOhRroCxj0UHvwahHv0u-qDLO1AYU0H8u-A4-tTlW-gQ5wDFX0SYLLbm5ZBUFBdCvnp~MMgzAamWn1ZqE3z0xdOWl9cKCm9wnrnvBPDO~r3DIwiDVGhDzXk2bOnh1leyw5-42Q0JTR6Yk6Pdasz1ku3m~OnAUXeJX-3NnaajRg4tSA5wSI1Ys5QlVUwDte8iDNr1D2i1xn8aMB-3AuoR2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the numbers of gene clusters in Ipomoea trifida and other plant species, i.e. Arabidopsis thaliana, potato (Solanum tuberosum), and cassava (Manihot esculenta). The black and white numbers in parenthesis represent the numbers of non-clustered sequences and sequences clustered with other species, respectively. The green, purple, orange, aqua, and red numbers in parenthesis represent the total numbers of putative genes subjected to clustering." path-from-xml="dsv00202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4598317/dsv00202.tiff" path-from-xml="dsv00202.tiff" target="_blank">View large</a><a section="4598317" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv00202.png?Expires=2147483647&Signature=Ri1-iD4UL4HHMyoTCOUwLO42Y0qKqktBeDt-NozyA-8kP8SksxL8JHaW9B4XcfsZE48E5Nyw0YooPOEjeRWtOchzq8-dnWCqQg3J4Sq-Dg8LG7buBa-cFuTN3cm3yoMQZe~amaf6zyFV7YZKWQKc74RhZVBRDALFMY3cHqdf6HKPf4VOawGn02Gbhh63-Yaf823x-8mihtm594O7yqSlbGAHonbBuudek4vvWJh5YFzGr83zT7ekJNVIYgpzdTYeo09tNdG9MJOWql0SD37boEz66XdShFeh4zxsReeRKO3rWaOi57ttYtZ-YiTbESAlaaYcV9-qcjtcRuTfPY0FJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4598317&ar=333025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the numbers of gene clusters in <em>Ipomoea trifida</em> and other plant species, i.e. <em>Arabidopsis thaliana</em>, potato (<em>Solanum tuberosum</em>), and cassava (<em>Manihot esculenta</em>). The black and white numbers in parenthesis represent the numbers of non-clustered sequences and sequences clustered with other species, respectively. The green, purple, orange, aqua, and red numbers in parenthesis represent the total numbers of putative genes subjected to clustering.</p></div></div><div content-id="DSV002F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/m_dsv00202.png?Expires=2147483647&Signature=dJRtOn2bqpAfJ9DS~V6Okbh1wT3jlyh7T0Fwjwz0tavl2zEWukl1Wl9dNSFfO5x72Ur3mqDyMQpsNR9WmNNfT~srkYWfKAGAmIAJpUPWqI1E60SfS0EKRLgKRsCW5ebGgOhRroCxj0UHvwahHv0u-qDLO1AYU0H8u-A4-tTlW-gQ5wDFX0SYLLbm5ZBUFBdCvnp~MMgzAamWn1ZqE3z0xdOWl9cKCm9wnrnvBPDO~r3DIwiDVGhDzXk2bOnh1leyw5-42Q0JTR6Yk6Pdasz1ku3m~OnAUXeJX-3NnaajRg4tSA5wSI1Ys5QlVUwDte8iDNr1D2i1xn8aMB-3AuoR2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the numbers of gene clusters in Ipomoea trifida and other plant species, i.e. Arabidopsis thaliana, potato (Solanum tuberosum), and cassava (Manihot esculenta). The black and white numbers in parenthesis represent the numbers of non-clustered sequences and sequences clustered with other species, respectively. The green, purple, orange, aqua, and red numbers in parenthesis represent the total numbers of putative genes subjected to clustering." path-from-xml="dsv00202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4598317/dsv00202.tiff" path-from-xml="dsv00202.tiff" target="_blank">View large</a><a section="4598317" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv00202.png?Expires=2147483647&Signature=Ri1-iD4UL4HHMyoTCOUwLO42Y0qKqktBeDt-NozyA-8kP8SksxL8JHaW9B4XcfsZE48E5Nyw0YooPOEjeRWtOchzq8-dnWCqQg3J4Sq-Dg8LG7buBa-cFuTN3cm3yoMQZe~amaf6zyFV7YZKWQKc74RhZVBRDALFMY3cHqdf6HKPf4VOawGn02Gbhh63-Yaf823x-8mihtm594O7yqSlbGAHonbBuudek4vvWJh5YFzGr83zT7ekJNVIYgpzdTYeo09tNdG9MJOWql0SD37boEz66XdShFeh4zxsReeRKO3rWaOi57ttYtZ-YiTbESAlaaYcV9-qcjtcRuTfPY0FJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4598317&ar=333025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the numbers of gene clusters in <em>Ipomoea trifida</em> and other plant species, i.e. <em>Arabidopsis thaliana</em>, potato (<em>Solanum tuberosum</em>), and cassava (<em>Manihot esculenta</em>). The black and white numbers in parenthesis represent the numbers of non-clustered sequences and sequences clustered with other species, respectively. The green, purple, orange, aqua, and red numbers in parenthesis represent the total numbers of putative genes subjected to clustering.</p></div></div><p>The intrinsic genes were classified based on GO slim analysis together with those of <em>A. thaliana</em>, potato, and cassava. The numbers of genes classified into the molecular function (MF), cellular component (CC), and biological process (BP) categories are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>. In parallel, a total of 28,208 (Mx23Hm) and 49,610 (0431-1) putative genes were classified into KOG functional categories along with the predicted genes of <em>A. thaliana</em> (33,630 genes), potato (41,952 genes), and cassava (31,668 genes) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). In the ‘Metabolism’ category, the number of classified genes in 0431-1 was higher than that in Mx23Hm.</p><p>Mapping of the genes for intrinsic proteins in <em>I. trifida</em> onto the KEGG metabolic pathways was carried out along with mapping of the genes in other plant species, including <em>A. thaliana</em> (13,154 genes), potato (14,318 genes), and cassava (14,251 genes). In the pathways categorized ‘1. Metabolism’ in the KEGG database, a total of 2,794 (Mx23Hm) and 2,444 (0431-1) putative genes were mapped onto 1,750 and 1,547 enzymes listed in the KEGG GENES database, which were involved in 138 and 137 metabolic pathways, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S16</a>). The pathways present only in Mx23Hm in <em>I. trifida</em> were ‘mucin type O-glycan biosynthesis' and ‘non-ribosomal peptide structures', whereas ‘biosynthesis of unsaturated fatty acids' was present only in 0431-1.</p>                            <h3 scrollto-destination=4598326 id="4598326" class="section-title" data-legacy-id=s3g>3.7. Genes involved in starch biosynthesis</h3>
<p>Starch biosynthesis is catalyzed by several enzymes including ADP-glucose pyrophosphorylase (AGPase), starch synthase (SS), branching enzyme (BE), and isoamylase (ISA). Similarity searches and phylogenetic analyses of genes encoding these enzymes in ITR_r1.0 (Mx23Hm) identified several potential homologues, which are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S17</a>. For AGPase, seven and three homologues potentially encoding the large and the small subunits, respectively, were detected in Mx23Hm. Two homologues for the large subunit were located at the ends of scaffolds and were likely to be partial. For SS, nine putative homologues were found in Mx23Hm. Phylogenetic analysis using the known isoforms of plant SS suggested that Itr_sc000002.1_g00061.1 and Itr_sc000727.1_g00009.1 are homologues of the genes for soluble <em>starch synthase II</em> (<em>SS II</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). For BE and ISA, three homologues of each were detected in Mx23Hm.</p>                            <h3 scrollto-destination=4598331 id="4598331" class="section-title" data-legacy-id=s3h>3.8. SNP discovery and CNV analysis</h3>
<p>Of the 596 M (Mx23Hm) and 484 M (0431-1) trimmed PE reads, 88.8 and 65.7%, respectively, were successfully mapped onto the assembled genome of Mx23Hm (ITR_r1.0). The mean depth of the mapped reads was 103.2 in Mx23Hm, covering 66.1% of the assembled genome, while the read depth in 0431-1 was 62.0, covering 60.8% of the assembled genome. Inspection of the sequence alignments in both lines detected a total of 7,970,056 variant candidates among the two lines. Of these candidates, 5,086,488 loci were selected according to Criteria 1 and 2 in the Materials and Methods. Further selection by application of Criterion 3 resulted in 2,089,076 candidates as a high-quality SNP set. Finally, 1,464,173 loci, including 596,211 heterozygotes and 867,962 homozygotes, were selected as highly confident SNP candidates by removing the repetitive sequences (Criterion 4). The SNP density was 1 SNP/231 bases in ITR_r1.0 (Ns were excluded), and the transition/transversion ratio was 1.45 (865,368 transitions and 598,805 transversions).</p><p>The effects of SNPs on gene functions were predicted using SnpEff, which groups SNPs into four categories (high-impact, moderate, modifier, and low) based on the positions of SNPs in the genome sequences. Among the 1,464,173 SNP candidates, 3,894 (0.3%) were classified as high-impact SNPs, including ‘splice acceptor and donor variants', ‘loss of the start codon’, or ‘gain/loss of the stop codon’. Another 137,786 loci (9.4%) were classified as ‘moderate effects' (missense variants), 1,037,197 (70.8%) as ‘modifiers' (e.g. variants in intergenic regions and intron), and 257,795 (17.6%) as ‘low-impact’ (e.g. synonymous variants) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). The remaining 27,501 loci (1.9%) were not assigned to any categories.</p><p>CNVs in the genomes of Mx23Hm and 0431-1 were detected using the CNV-seq program. CNVs between Mx23Hm and 0431-1 longer than 50 kb in length were identified on 2,095 scaffolds. The numbers of detected CNVs varied in accordance with the threshold values of the log<sub>2</sub> ratio (log-transformed ratio of the number of mapped reads in 0431-1 to the number in Mx23Hm) and CNV sizes. For example, 54 CNVs with a mean length of 12,444 bases were identified under the stringent condition (log<sub>2</sub> ratio &gt;10 and CNV size &gt;10,000 bases) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). Under a more relaxed condition (i.e. log<sub>2</sub> ratio of &gt;1 and CNV size &gt;1,000 bases), 16,682 CNVs with a mean length of 1,918 bases were detected. Under both stringent and relaxed conditions, the CNVs occupied 671,974 bases (0.3%) and 44,892,563 bases (18.8%), respectively, of the total length of the 2,095 contigs (239,146,348 bases). The mean absolute values of log<sub>2</sub> ratios in the core candidate and line-specific sequences were 0.63 and 1.72, respectively, indicating that CNVs were significantly enriched in the line-specific regions relative to the core candidate regions of the genomes (<em>P</em> &lt; 0.001) (Fig. <a reveal-id="DSV002F3" data-open="DSV002F3" class="link link-reveal link-table xref-fig">3</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>). </p>                        <a id="4598342" scrollto-destination="4598342"></a>
<div id="DSV002F3" data-id="DSV002F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/m_dsv00203.png?Expires=2147483647&Signature=GSTjkXOGrtt6vdpoappjntQTUKNHCVEhm~AOwN6-k~hO184oZrOUSWvDDOmatRmnF1RHRhIxHn5YXJLPc64ykYVr~-cNPnR8F5jBuij8PaZ5hjV8bYgTgovKwStHXrI9vffW3sIx3cRhgwQazNmOP05LVY1LN2JuSW2ZBKTLhHAplAG8mXAhwkQZfBuih8F6N6NkqLQ5It5lVqVKWViaTNdC3mSq2cqA02FBB-HgQmvneaVa0pyeUVkejIot84Ck1aBUGP7joV6MY2X47Pk4qeH6pZEdbw0H2ffOyH5N5pm4wOuEP4PpM8djiZa89TQRv3xzYw1XlJ513ufICSqUbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CNV distributions and corresponding positions of core candidates and line-specific sequences of scaffold Itr_sc000048.1 (1,177,009 bases in length). Red dots show the log2 ratios of CNVs between Mx23Hm and 0431-1. The upper bar represents core candidate (red) and line-specific (blue) sequences in their corresponding positions." path-from-xml="dsv00203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4598342/dsv00203.tiff" path-from-xml="dsv00203.tiff" target="_blank">View large</a><a section="4598342" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv00203.png?Expires=2147483647&Signature=QHNEYBnTryb-ElOTK3UmPKlsOrql69EbE83eoI8i3PvGPBOu09qXA2hY0A-yh-erd4h~Z4Fll6TphfljfhF2IhDhs~8cxojX7h0Zvu-GAxvIjNoMHdra1gQUto1rt8yBN38gKMuQaaHQpaOM1A4RguoftZucpJ6nwDI3wHcY-zvQIeNgfSoFyKQ0jB7gXMMj9HqQyQTV7IFMcUSjKI2LWcySbDuNRzLKCY29iTiRXltXHAnlk~3F4QKdQ0ftx-2G5FbzdmdXUo4Tig5HXlsvmiM08FYpHDnjFUngSq9fieTm0MhDuDNTufV9ne9oqgv8Si11CPenB3CBDRGVf2BYFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4598342&ar=333025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CNV distributions and corresponding positions of core candidates and line-specific sequences of scaffold Itr_sc000048.1 (1,177,009 bases in length). Red dots show the log<sub>2</sub> ratios of CNVs between Mx23Hm and 0431-1. The upper bar represents core candidate (red) and line-specific (blue) sequences in their corresponding positions.</p></div></div><div content-id="DSV002F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/m_dsv00203.png?Expires=2147483647&Signature=GSTjkXOGrtt6vdpoappjntQTUKNHCVEhm~AOwN6-k~hO184oZrOUSWvDDOmatRmnF1RHRhIxHn5YXJLPc64ykYVr~-cNPnR8F5jBuij8PaZ5hjV8bYgTgovKwStHXrI9vffW3sIx3cRhgwQazNmOP05LVY1LN2JuSW2ZBKTLhHAplAG8mXAhwkQZfBuih8F6N6NkqLQ5It5lVqVKWViaTNdC3mSq2cqA02FBB-HgQmvneaVa0pyeUVkejIot84Ck1aBUGP7joV6MY2X47Pk4qeH6pZEdbw0H2ffOyH5N5pm4wOuEP4PpM8djiZa89TQRv3xzYw1XlJ513ufICSqUbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CNV distributions and corresponding positions of core candidates and line-specific sequences of scaffold Itr_sc000048.1 (1,177,009 bases in length). Red dots show the log2 ratios of CNVs between Mx23Hm and 0431-1. The upper bar represents core candidate (red) and line-specific (blue) sequences in their corresponding positions." path-from-xml="dsv00203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4598342/dsv00203.tiff" path-from-xml="dsv00203.tiff" target="_blank">View large</a><a section="4598342" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv00203.png?Expires=2147483647&Signature=QHNEYBnTryb-ElOTK3UmPKlsOrql69EbE83eoI8i3PvGPBOu09qXA2hY0A-yh-erd4h~Z4Fll6TphfljfhF2IhDhs~8cxojX7h0Zvu-GAxvIjNoMHdra1gQUto1rt8yBN38gKMuQaaHQpaOM1A4RguoftZucpJ6nwDI3wHcY-zvQIeNgfSoFyKQ0jB7gXMMj9HqQyQTV7IFMcUSjKI2LWcySbDuNRzLKCY29iTiRXltXHAnlk~3F4QKdQ0ftx-2G5FbzdmdXUo4Tig5HXlsvmiM08FYpHDnjFUngSq9fieTm0MhDuDNTufV9ne9oqgv8Si11CPenB3CBDRGVf2BYFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4598342&ar=333025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CNV distributions and corresponding positions of core candidates and line-specific sequences of scaffold Itr_sc000048.1 (1,177,009 bases in length). Red dots show the log<sub>2</sub> ratios of CNVs between Mx23Hm and 0431-1. The upper bar represents core candidate (red) and line-specific (blue) sequences in their corresponding positions.</p></div></div>                            <h2 scrollto-destination=4598347 id="4598347" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The genome size of <em>I. trifida</em> was estimated to be 515.8–539.9 Mb by analysis of the multiplicity of k-mers of the Illumina sequencing reads. This size was smaller than the predicted values of 737–814 Mb, which were based on a previous report of DNA content in the hexaploid sweet potato (4.8–5.3 pg/2C).<sup><a reveal-id="DSV002C2" data-open="DSV002C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The k-mer analysis also supported that 0431-1 is highly heterozygous, whereas Mx23Hm is homozygous, which is consistent with the results of polymorphic analysis using 14 SSR markers.</p><p>The total length of ITR_r1.0 (513 Mb) for Mx23Hm corresponded well to the genome size estimated by multiplicity of k-mers, whereas the total length of ITRk_r1.0 (712 Mb) for 0431-1 was larger than the estimated value, probably due to unassembled heterozygous sequences. Moreover, the N50 of ITR_r1.0 was longer than that of ITRk_r1.0 despite the fact that fewer MP reads were used. These results may reflect the difficulty in sequence assembly of heterozygous genomes. The N50 value of 36.3–42.6 kb of the assembled sequences in this study is long enough to serve as a basis for a primary survey of the <em>I. trifida</em> genome. However, additional effort would be necessary to reduce the proportion of the <em>N</em> base, which currently occupies 34% (Mx23Hm) and 32% (0431-1) of the assembled genomic sequences.</p><p>When the <em>N</em> bases were excluded from the assembled sequences, the content of the repetitive sequences was calculated to be 42.3–47.7% of the entire <em>I. trifida</em> genomes, which is lower than the contents in other plant species such as <em>Lotus japonicus</em> (56.8%),<sup><a reveal-id="DSV002C58" data-open="DSV002C58" class="link link-ref link-reveal xref-bibr">58</a></sup> potato (64.2%),<sup><a reveal-id="DSV002C59" data-open="DSV002C59" class="link link-ref link-reveal xref-bibr">59</a></sup> and tomato (68.3%).<sup><a reveal-id="DSV002C60" data-open="DSV002C60" class="link link-ref link-reveal xref-bibr">60</a></sup> While the length of the assembled sequences of 0431-1 (712 Mb) was longer than that of Mx23Hm (513 Mb), the number of di-nucleotide SSRs of 0431-1 (36,941) was, for example, fewer than that of Mx23Hm (54,631). The distribution of numbers of repetitions indicated that the lengths of the SSRs identified in 0431-1 were shorter than those in Mx23Hm (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S11</a>). It is likely that the shorter lengths of the scaffolds in 0431-1 hampered the success of detection of SSRs with long lengths.</p><p>The assembled sequences were classified into two categories, ‘core candidate’ and ‘line-specific’ sequences. In this study, we expected to investigate the genome structures of three haplotypes of <em>I. trifida</em>, one from Mx23Hm and two from the highly heterozygous 0431-1, to identify sequence variations. We adopted the term ‘core candidate’ rather than ‘core’ to refer to the conserved regions in the genomes, because three haplotypes may not be sufficient. The total length of core candidate sequences in 0431-1 was longer than those in Mx23Hm. Multiple core sequences in 0431-1 corresponded to single sequences in Mx23Hm (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S12</a>), suggesting that the longer core candidate sequences in 0431-1 reflected the high heterozygosity in the 0431-1 genome. The total lengths of line-specific sequences were 273.4 Mb in Mx23Hm and 358.7 Mb in 0431-1, and 64% of these sequences were occupied by the <em>N</em> bases. When these Ns were excluded from the calculation, the ratios of the line-specific sequences to the whole-genome sequences fell to 29% in Mx23Hm and 27% in 0431-1, being rich in repetitive sequences and rRNA-encoding genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S5 and S9</a>).</p><p>A total of 62,403 and 109,449 protein-encoding genes were predicted in the assembled genomic sequences of Mx23Hm and 0431-1, respectively. It is likely that the larger number of predicted genes in 0431-1 might reflect the high heterozygosity of this line. We focussed on genes involved in starch biosynthesis, namely AGPase, BE, ISA, and SS that catalyze starch biosynthesis, because starch is a nutritionally and industrially important component in the storage roots of sweet potato. AGPase catalyzes the conversion of glucose 1-phosphate to ADP-glucose, which is subsequently used by SS as a substrate for elongation of the α-1,4 glucan chain. BE and ISA play roles in amylopectin biosynthesis though the generation (BE) or degradation (ISA) of α-1,6 branches. For AGPase, a total of 10 homologues were identified in ITR_r1.0. Of these homologues, two of the large subunit were located on the ends of scaffolds and were likely to be partial. For BE and ISA, three homologues of each were detected in ITR_r1.0. These include the sequences for isoforms that have not been reported from sweet potato so far. Phylogenetic analysis suggested that two homologues (Itr_sc000002.1_g00061.1 and Itr_sc000727.1_g00009.1) of <em>SSII</em> genes are present in <em>I. trifida</em>, in contrast to potato and <em>Arabidopsis</em>, in which only a single <em>SSII</em> gene is present. Of these, Itr_sc000727.1_g00009.1 is likely to be an orthologue of the previously identified <em>SSII</em> gene of sweet potato.<sup><a reveal-id="DSV002C61" data-open="DSV002C61" class="link link-ref link-reveal xref-bibr">61</a></sup> In addition, a highly similar sequence of Itr_sc000002.1_g00061.1 was found in the NCBI Transcriptome Shotgun Assembly (TSA) of sweet potato, suggesting the presence of an orthologue in the sweet potato genome. The three <em>SSII</em> isoforms in rice have different tissue specificity.<sup><a reveal-id="DSV002C62" data-open="DSV002C62" class="link link-ref link-reveal xref-bibr">62</a></sup> Therefore, it is possible that the two homologues of <em>SSII</em> in <em>I. trifida</em> and sweet potato have distinct functions in different tissues.</p><p>Genomic variation can be surveyed by SNP and CNV analyses. Among three haplotypes in the two assembled genomes, we discovered a total of 1,464,173 SNP candidates. Of these SNPs, 596,211 were heterozygotes while 867,962 were homozygotes in the 0431-1 genome, suggesting that 59.3% (867,962/1,464,173) of SNP alleles were homozygotes in the heterozygous line. When the Ns were excluded from the estimation, the mean frequencies of SNPs between the two assembled genomes were 1 SNP per 231 bases (Mx23Hm) and 331 bases (0431-1). Of these SNPs, 3,894 were predicted to have high impacts on gene function. In particular, one, two, and one high-impact SNPs were predicted in the putative genes encoding AGPase, SS, and BE, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S17</a>). CNVs were discovered in from 0.3 to 18.8% of the genome regions, depending on the different thresholds. Though the number of CNVs was less than that of SNPs, the discovered CNVs may have greater potential to disrupt gene functions due to gene deletions, frame-shift mutations, and alternating gene expressions.</p><p>This is the first report of a whole-genome survey for a plant belonging to the genus <em>Ipomoea</em>. Within this genus, sweet potato is the most important species in terms of industrial applicability. However, genome analysis of this species has been hindered by its hexaploid characteristics. The sequence information of the diploid genome would therefore be a useful resource for investigation of the genomes of related polyploid species. Whole-genome sequences of the closest wild relatives have been performed prior to genomic characterization of several important polyploid crop species such as cotton,<sup><a reveal-id="DSV002C63" data-open="DSV002C63" class="link link-ref link-reveal xref-bibr">63</a></sup> banana,<sup><a reveal-id="DSV002C64" data-open="DSV002C64" class="link link-ref link-reveal xref-bibr">64</a></sup> and strawberry.<sup><a reveal-id="DSV002C65" data-open="DSV002C65" class="link link-ref link-reveal xref-bibr">65</a></sup> These studies suggested that the construction of pseudomolecules along individual chromosomes of the diploid genomes would be necessary to fully utilize the obtained information. For this purpose, development of high-density linkage maps of the <em>I. trifida</em> genome is the next crucial step.</p>                            <h2 scrollto-destination=4598362 id="4598362" class="section-title" data-legacy-id=s5>5. Data availability</h2>
<p>The genome assembly data, annotations, gene models, and SNPs of <em>I. trifida</em> are available at the Sweetpotato GARDEN (<a class="link link-uri" href="http://sweetpotato-garden.kazusa.or.jp">http://sweetpotato-garden.kazusa.or.jp</a>). The BioProject accession number of <em>I. trifida</em> is PRJDB3230. The WGS (CON) accession numbers of assembled sequences in Mx23Hm and 0431-1 are BBOG01000001-BBOG01163047 (DF850533-DF884990) and BBOH01000001-BBOH01377770 (DF884991-DF933566), respectively. The genome sequence reads obtained by Illumina HiSeq 2000 are available from the DDBJ Sequence Read Archive (DRA) under the accession numbers DRR023905-DRR023907 (Mx23Hm) and DRR023898-DRR023904 (0431-1).</p>                            <h2 scrollto-destination=4598366 id="4598366" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4598369 id="4598369" class="section-title" data-legacy-id=s7>Funding</h2>
<p>The work was supported by the Kazusa DNA Research Institute Foundation and National Agriculture and Food Research Organization. Funding to pay the Open Access publication charges for this article was provided by the Kazusa DNA Research Institute.</p>                            <h2 scrollto-destination=4598374 id="4598374" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work could not have been accomplished without the highly homozygous <em>I. trifida</em> line Mx23Hm established by Dr Hiroki Nakayama (NARO/KARC), who passed away in 2012.</p>                            <h2 scrollto-destination=4598379 id="4598379" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV002C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Austin</div> <div class="given-names">D.F.</div></div> <div class="name"><div class="surname">Huám</div> <div class="given-names">Z.</div></div></div> <div class="year">1996</div>, <div class="article-title">A synopsis of <em>Ipomoea</em> (Convolvulaceae) in the Americas</div>, <div class="source">Taxon</div> , <div class="volume">45</div>, <div class="fpage">3</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20synopsis%20of%20Ipomoea%20(Convolvulaceae)%20in%20the%20Americas&amp;author=D.F.%20Austin&amp;author=Z.%20Hu%C3%A1m&amp;publication_year=1996&amp;journal=Taxon&amp;volume=45&amp;pages=3-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/1222581" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F1222581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F1222581"> </span></div> </div></div></div></div></div><div content-id="DSV002C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ozias-Akins</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Jarret</div> <div class="given-names">R.L.</div></div></div> <div class="year">1994</div>, <div class="article-title">Nuclear DNA content and ploidy levels in the genus <em>Ipomoea</em></div>, <div class="source">J. Amer. Soc. Hort. Sci.</div> , <div class="volume">119</div>, <div class="fpage">110</div>–<div class="lpage">5</div>.</div></div></div></div><div content-id="DSV002C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">F.W.</div></div></div> <div class="year">1965</div>, <div class="article-title">Incompatibility in the sweet potato. A review</div>, <div class="source">Eco. Bot.</div> , <div class="volume">19</div>, <div class="fpage">406</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incompatibility%20in%20the%20sweet%20potato.%20A%20review&amp;author=F.W.%20Martin&amp;publication_year=1965&amp;journal=Eco.%20Bot.&amp;volume=19&amp;pages=406-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02904812" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02904812" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02904812"> </span></div> </div></div></div></div></div><div content-id="DSV002C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">M.</div></div></div> <div class="year">1983</div>, <div class="article-title">The <em>Ipomoea trifida</em> complex closely related to sweet potato</div>. In: <div class="person-group"><div class="name"><div class="surname">Shideler</div> <div class="given-names">S.F.</div></div> <div class="name"><div class="surname">Rincon</div> <div class="given-names">H.</div></div></div> (eds), <div class="source">Proceedings of the 6th Symposium of the International Society of Tropical Root Crops</div> , pp.<div class="fpage">561</div>–<div class="lpage">8</div>. <div class="publisher-name">International Potato Center Lima</div>, <div class="publisher-loc">Peru</div>.</div></div></div></div><div content-id="DSV002C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Shiotani</div> <div class="given-names">I.</div></div> <div class="name"><div class="surname">Kawase</div> <div class="given-names">T.</div></div></div> <div class="year">1989</div>, <div class="article-title">Genomic structure of the sweet potato and hexaploid in <em>Ipomoea trifida</em> (H. B. K.) Don.</div>, <div class="source">Japan. J. Breed.</div> , <div class="volume">39</div>, <div class="fpage">57</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20structure%20of%20the%20sweet%20potato%20and%20hexaploid%20in%20Ipomoea%20trifida%20(H.%20B.%20K.)%20Don.&amp;author=I.%20Shiotani&amp;author=T.%20Kawase&amp;publication_year=1989&amp;journal=Japan.%20J.%20Breed.&amp;volume=39&amp;pages=57-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs1951.39.57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs1951.39.57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs1951.39.57"> </span></div> </div></div></div></div></div><div content-id="DSV002C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Orjeda</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Freyre</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Iwanaga</div> <div class="given-names">M.</div></div></div> <div class="year">1991</div>, <div class="article-title">Use of <em>Ipomoea trifida</em> germ plasm for sweet potato improvement. 3. Development of 4x interspecific hybrids between <em>Ipomoea batatas</em> (L.) Lam. (2n = 6x = 90) and <em>I. trifida</em> (H. B. K) G. Don. (2n = 2x = 30) as storage-root initiators for wild species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">83</div>, <div class="fpage">159</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20Ipomoea%20trifida%20germ%20plasm%20for%20sweet%20potato%20improvement.%203.%20Development%20of%204x%20interspecific%20hybrids%20between%20Ipomoea%20batatas%20(L.)%20Lam.%20(2n%20%3D%206x%20%3D%2090)%20and%20I.%20trifida%20(H.%20B.%20K)%20G.%20Don.%20(2n%20%3D%202x%20%3D%2030)%20as%20storage-root%20initiators%20for%20wild%20species&amp;author=G.%20Orjeda&amp;author=R.%20Freyre&amp;author=M.%20Iwanaga&amp;publication_year=1991&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=83&amp;pages=159-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00226245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00226245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00226245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24202352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Komaki</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Katayama</div> <div class="given-names">K.</div></div></div> <div class="year">1999</div>, <div class="article-title">Root thickness of diploid <em>Ipomoea trifida</em> (H. B. K.) G. Don and performance of progeny derived from the cross with sweetpotato</div>, <div class="source">Breed. Sci.</div> , <div class="volume">49</div>, <div class="fpage">123</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root%20thickness%20of%20diploid%20Ipomoea%20trifida%20(H.%20B.%20K.)%20G.%20Don%20and%20performance%20of%20progeny%20derived%20from%20the%20cross%20with%20sweetpotato&amp;author=K.%20Komaki&amp;author=K.%20Katayama&amp;publication_year=1999&amp;journal=Breed.%20Sci.&amp;volume=49&amp;pages=123-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.49.123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.49.123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.49.123"> </span></div> </div></div></div></div></div><div content-id="DSV002C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Jarret</div> <div class="given-names">R.L.</div></div> <div class="name"><div class="surname">Austin</div> <div class="given-names">D.F.</div></div></div> <div class="year">1994</div>, <div class="article-title">Genetic diversity and systematic relationship in sweetpotato (<em>Ipomoea batatas</em> (L.) Lam.) and related species as revealed by RAPD analysis</div>, <div class="source">Genet. Resour. Crop Evol.</div> , <div class="volume">41</div>, <div class="fpage">165</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20systematic%20relationship%20in%20sweetpotato%20(Ipomoea%20batatas%20(L.)%20Lam.)%20and%20related%20species%20as%20revealed%20by%20RAPD%20analysis&amp;author=R.L.%20Jarret&amp;author=D.F.%20Austin&amp;publication_year=1994&amp;journal=Genet.%20Resour.%20Crop%20Evol.&amp;volume=41&amp;pages=165-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00051633" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00051633" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00051633"> </span></div> </div></div></div></div></div><div content-id="DSV002C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Komaki</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Regmi</div> <div class="given-names">H.N.</div></div> <div class="name"><div class="surname">Katayama</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Tamiya</div> <div class="given-names">S.</div></div></div> <div class="year">1998</div>, <div class="article-title">Morphological and RAPD pattern variations in sweetpotato and its closely related species</div>, <div class="source">Breed. Sci.</div> , <div class="volume">48</div>, <div class="fpage">281</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSV002C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Sun</div> <div class="given-names">M.</div></div></div> <div class="year">2000</div>, <div class="article-title">Genetic diversity and relationships of sweetpotato and its wild relatives in <em>Ipomoea</em> series <em>Batatas</em> (Convolvulaceae) as revealed by inter-simple sequence repeat (ISSR) and restriction analysis of chloroplast DNA</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">100</div>, <div class="fpage">1050</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20relationships%20of%20sweetpotato%20and%20its%20wild%20relatives%20in%20Ipomoea%20series%20Batatas%20(Convolvulaceae)%20as%20revealed%20by%20inter-simple%20sequence%20repeat%20(ISSR)%20and%20restriction%20analysis%20of%20chloroplast%20DNA&amp;author=J.C.%20Huang&amp;author=M.%20Sun&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=100&amp;pages=1050-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051386" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051386" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051386"> </span></div> </div></div></div></div></div><div content-id="DSV002C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Roullier</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Duputié</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Wennekes</div> <div class="given-names">P.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Disentangling the origins of cultivated sweet potato (<em>Ipomoea batatas</em> (L.) Lam.)</div>, <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e62707</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disentangling%20the%20origins%20of%20cultivated%20sweet%20potato%20(Ipomoea%20batatas%20(L.)%20Lam.)&amp;author=C.%20Roullier&amp;author=A.%20Duputi%C3%A9&amp;author=P.%20Wennekes&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e62707" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0062707" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0062707" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0062707"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23723970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Srisuwan</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Sihachakr</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Siljak-Yakovlev</div> <div class="given-names">S.</div></div></div> <div class="year">2006</div>, <div class="article-title">The origin and evolution of sweet potato (<em>Ipomoea batatas</em> Lam.) and its wild relatives through the cytogenetic approaches</div>, <div class="source">Plant Sci.</div> , <div class="volume">171</div>, <div class="fpage">424</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20and%20evolution%20of%20sweet%20potato%20(Ipomoea%20batatas%20Lam.)%20and%20its%20wild%20relatives%20through%20the%20cytogenetic%20approaches&amp;author=S.%20Srisuwan&amp;author=D.%20Sihachakr&amp;author=S.%20Siljak-Yakovlev&amp;publication_year=2006&amp;journal=Plant%20Sci.&amp;volume=171&amp;pages=424-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2006.05.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2006.05.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2006.05.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22980213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kowyama</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Tsuchiya</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Kakeda</div> <div class="given-names">K.</div></div></div> <div class="year">2000</div>, <div class="article-title">Sporophitic self-incompatibility in <em>Ipomoea trifida</em>, a close relative of sweet potato</div>, <div class="source">Ann. Bot.</div> , <div class="volume">85</div>, <div class="fpage">191</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sporophitic%20self-incompatibility%20in%20Ipomoea%20trifida%2C%20a%20close%20relative%20of%20sweet%20potato&amp;author=Y.%20Kowyama&amp;author=T.%20Tsuchiya&amp;author=K.%20Kakeda&amp;publication_year=2000&amp;journal=Ann.%20Bot.&amp;volume=85&amp;pages=191-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/anbo.1999.1036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fanbo.1999.1036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fanbo.1999.1036"> </span></div> </div></div></div></div></div><div content-id="DSV002C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Tanaka</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Tomita</div> <div class="given-names">R.N.</div></div> <div class="name"><div class="surname">Kowyama</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Mukai</div> <div class="given-names">Y.</div></div></div> <div class="year">2004</div>, <div class="article-title">Visualization of the S-locus region in <em>Ipomoea trifida</em>: toward positional cloning of self-incompatibility genes</div>, <div class="source">Chromosome Res.</div> , <div class="volume">12</div>, <div class="fpage">475</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Visualization%20of%20the%20S-locus%20region%20in%20Ipomoea%20trifida%3A%20toward%20positional%20cloning%20of%20self-incompatibility%20genes&amp;author=G.%20Suzuki&amp;author=S.%20Tanaka&amp;author=M.%20Yamamoto&amp;author=R.N.%20Tomita&amp;author=Y.%20Kowyama&amp;author=Y.%20Mukai&amp;publication_year=2004&amp;journal=Chromosome%20Res.&amp;volume=12&amp;pages=475-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:CHRO.0000034749.58877.b7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:CHRO.0000034749.58877.b7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:CHRO.0000034749.58877.b7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15252243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rahman</div> <div class="given-names">M.H.</div></div> <div class="name"><div class="surname">Tsuchiya</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Physical size of the S locus region defined by genetic recombination and genome sequencing in <em>Ipomoea trifida</em>, Convolvulaceae</div>, <div class="source">Sexual Plant Reprod.</div> , <div class="volume">20</div>, <div class="fpage">63</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20size%20of%20the%20S%20locus%20region%20defined%20by%20genetic%20recombination%20and%20genome%20sequencing%20in%20Ipomoea%20trifida%2C%20Convolvulaceae&amp;author=M.H.%20Rahman&amp;author=T.%20Tsuchiya&amp;author=K.%20Suwabe&amp;publication_year=2007&amp;journal=Sexual%20Plant%20Reprod.&amp;volume=20&amp;pages=63-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00497-007-0044-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00497-007-0044-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00497-007-0044-x"> </span></div> </div></div></div></div></div><div content-id="DSV002C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rahman</div> <div class="given-names">M.H.</div></div> <div class="name"><div class="surname">Uchiyama</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Kuno</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Expression of stigma- and anther-specific genes located in the S locus region of <em>Ipomoea trifida</em></div>, <div class="source">Sexual Plant Reprod.</div> , <div class="volume">20</div>, <div class="fpage">73</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20stigma-%20and%20anther-specific%20genes%20located%20in%20the%20S%20locus%20region%20of%20Ipomoea%20trifida&amp;author=M.H.%20Rahman&amp;author=M.%20Uchiyama&amp;author=M.%20Kuno&amp;publication_year=2007&amp;journal=Sexual%20Plant%20Reprod.&amp;volume=20&amp;pages=73-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00497-007-0045-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00497-007-0045-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00497-007-0045-9"> </span></div> </div></div></div></div></div><div content-id="DSV002C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Nakayama</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Tanaka</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Takahata</div> <div class="given-names">Y.</div></div></div> <div class="year">2010</div>, <div class="article-title">An AFLP-based genetic linkage map of <em>Ipomoea trifida</em> (H.B.K.) G. Don., a diploid relative of sweetpotato</div>, <div class="source">I. Batatas <em>(L.)</em> Lam. Trop. Agr. Develop.</div> , <div class="volume">54</div>, <div class="fpage">9</div>–<div class="lpage">16</div>.</div></div></div></div><div content-id="DSV002C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Michael</div> <div class="given-names">T.P.</div></div> <div class="name"><div class="surname">Jackson</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, <div class="article-title">The first 50 plant genomes</div>, <div class="source">Plant Genome</div> , <div class="volume">6</div>, <div class="fpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%2050%20plant%20genomes&amp;author=T.P.%20Michael&amp;author=S.%20Jackson&amp;publication_year=2013&amp;journal=Plant%20Genome&amp;volume=6&amp;pages=2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2013.03.0001in" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2013.03.0001in" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2013.03.0001in"> </span></div> </div></div></div></div></div><div content-id="DSV002C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Weigel</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Mott</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, <div class="article-title">The 1001 genomes project for <em>Arabidopsis thaliana</em></div>, <div class="source">Genome Biol.</div> , <div class="volume">10</div>, <div class="fpage">107</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%201001%20genomes%20project%20for%20Arabidopsis%20thaliana&amp;author=D.%20Weigel&amp;author=R.%20Mott&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-5-107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-5-107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-5-107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19519932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Ge</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">30</div>, <div class="fpage">105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%2050%20accessions%20of%20cultivated%20and%20wild%20rice%20yields%20markers%20for%20identifying%20agronomically%20important%20genes&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;author=S.%20Ge&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2050"> </span></div> </div></div></div></div></div><div content-id="DSV002C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Jiao</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Zhao</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Ren</div> <div class="given-names">L.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Genome-wide genetic changes during modern breeding of maize</div>, <div class="source">Nat. Genet.</div> , <div class="volume">44</div>, <div class="fpage">812</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20changes%20during%20modern%20breeding%20of%20maize&amp;author=Y.%20Jiao&amp;author=H.%20Zhao&amp;author=L.%20Ren&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=812-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2312" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2312" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2312"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">De Paoli</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Radovic</div> <div class="given-names">S.</div></div></div> <div class="year">2007</div>, <div class="article-title">Transposable elements and the plant pan-genomes</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">10</div>, <div class="fpage">149</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%20and%20the%20plant%20pan-genomes&amp;author=M.%20Morgante&amp;author=E.%20De%20Paoli&amp;author=S.%20Radovic&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=10&amp;pages=149-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2007.02.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2007.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2007.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17300983" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tettelin</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Masignani</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Cieslewicz</div> <div class="given-names">M.J.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">Genome analysis of multiple pathogenic isolates of <em>Streptococcus agalactiae</em>: implications for the microbial ‘pan-genome</div>’, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">102</div>, <div class="fpage">13950</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20analysis%20of%20multiple%20pathogenic%20isolates%20of%20Streptococcus%20agalactiae%3A%20implications%20for%20the%20microbial%20%E2%80%98pan-genome&amp;author=H.%20Tettelin&amp;author=V.%20Masignani&amp;author=M.J.%20Cieslewicz&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=13950-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0506758102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0506758102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0506758102"> </span></div> </div></div></div></div></div><div content-id="DSV002C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Brunner</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Fengler</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Tingey</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div></div> <div class="year">2005</div>, <div class="article-title">Evolution of DNA sequence nonhomologies among maize inbreds</div>, <div class="source">Plant Cell</div> , <div class="volume">17</div>, <div class="fpage">343</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20DNA%20sequence%20nonhomologies%20among%20maize%20inbreds&amp;author=S.%20Brunner&amp;author=K.%20Fengler&amp;author=M.%20Morgante&amp;author=S.%20Tingey&amp;author=A.%20Rafalski&amp;publication_year=2005&amp;journal=Plant%20Cell&amp;volume=17&amp;pages=343-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.104.025627" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.104.025627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.104.025627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15659640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.H.</div></div> <div class="name"><div class="surname">Zhou</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">De novo assembly of soybean wild relatives for pan-genome analysis of diversity and agronomic traits</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">32</div>, <div class="fpage">1045</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20soybean%20wild%20relatives%20for%20pan-genome%20analysis%20of%20diversity%20and%20agronomic%20traits&amp;author=Y.H.%20Li&amp;author=G.%20Zhou&amp;author=J.%20Ma&amp;publication_year=2014&amp;journal=Nat.%20Biotechnol.&amp;volume=32&amp;pages=1045-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25218520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div> <div class="name"><div class="surname">Edwards</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div></div> <div class="year">2014</div>, <div class="article-title">Structural variations in plant genomes</div>, <div class="source">Brief Funct. Genomics</div> , <div class="volume">13</div>, <div class="fpage">296</div>–<div class="lpage">307</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20variations%20in%20plant%20genomes&amp;author=R.K.%20Saxena&amp;author=D.%20Edwards&amp;author=R.K.%20Varshney&amp;publication_year=2014&amp;journal=Brief%20Funct.%20Genomics&amp;volume=13&amp;pages=296-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bfgp/elu016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbfgp%2Felu016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbfgp%2Felu016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24907366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Murray</div> <div class="given-names">M.G.</div></div> <div class="name"><div class="surname">Thompson</div> <div class="given-names">W.F.</div></div></div> <div class="year">1980</div>, <div class="article-title">Rapid isolation of high-molecular-weight plant DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">8</div>, <div class="fpage">4321</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20isolation%20of%20high-molecular-weight%20plant%20DNA&amp;author=M.G.%20Murray&amp;author=W.F.%20Thompson&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res.&amp;volume=8&amp;pages=4321-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/8.19.4321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F8.19.4321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F8.19.4321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7433111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">Comprehensive structural analysis of the genome of red clover (<em>Trifolium pretense</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="volume">12</div>, <div class="fpage">301</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20structural%20analysis%20of%20the%20genome%20of%20red%20clover%20(Trifolium%20pretense%20L.)&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;author=E.%20Asamizu&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=301-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Nieuwerburgh</div> <div class="given-names">F.V.</div></div> <div class="name"><div class="surname">Thompson</div> <div class="given-names">R.C.</div></div> <div class="name"><div class="surname">Ledesma</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">e24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Illumina%20mate-paired%20DNA%20sequencing-library%20preparation%20using%20Cre-Lox%20recombination&amp;author=F.V.%20Nieuwerburgh&amp;author=R.C.%20Thompson&amp;author=J.%20Ledesma&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=e24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127871" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Schmieder</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Edwards</div> <div class="given-names">R.</div></div></div> <div class="year">2011</div>, <div class="article-title">Quality control and preprocessing of metagenomic datasets</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">863</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quality%20control%20and%20preprocessing%20of%20metagenomic%20datasets&amp;author=R.%20Schmieder&amp;author=R.%20Edwards&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=863-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21278185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, <div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">De novo assembly of human genomes with massively parallel short read sequencing</div>, <div class="source">Genome Res.</div> , <div class="volume">20</div>, <div class="fpage">265</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20human%20genomes%20with%20massively%20parallel%20short%20read%20sequencing&amp;author=R.%20Li&amp;author=H.%20Zhu&amp;author=J.%20Ruan&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=265-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.097261.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.097261.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.097261.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20019144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res.</div> , <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res.&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Henkel</div> <div class="given-names">C.V.</div></div> <div class="name"><div class="surname">Jansen</div> <div class="given-names">H.J.</div></div> <div class="name"><div class="surname">Butler</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2010</div>, <div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div> <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div> <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div> <div class="year">1990</div>, <div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tomita</div> <div class="given-names">R.N.</div></div> <div class="name"><div class="surname">Suzuki</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>et al.  </div> <div class="year">2004</div>, <div class="article-title">Molecular characterization of a 313-kb genomic region containing the self-incompatibility locus of <em>Ipomoea trifida</em>, a diploid relative of sweet potato</div>, <div class="source">Breed. Sci.</div> , <div class="volume">54</div>, <div class="fpage">165</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20a%20313-kb%20genomic%20region%20containing%20the%20self-incompatibility%20locus%20of%20Ipomoea%20trifida%2C%20a%20diploid%20relative%20of%20sweet%20potato&amp;author=R.N.%20Tomita&amp;author=G.%20Suzuki&amp;author=K.%20Yoshida&amp;publication_year=2004&amp;journal=Breed.%20Sci.&amp;volume=54&amp;pages=165-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.54.165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.54.165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.54.165"> </span></div> </div></div></div></div></div><div content-id="DSV002C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Phillippy</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>et al.  </div> <div class="year">2004</div>, <div class="article-title">Versatile and open software for comparing large genomes</div>, <div class="source">Genome Biol.</div> , <div class="volume">5</div>, <div class="fpage">R12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes&amp;author=S.%20Kurtz&amp;author=A.%20Phillippy&amp;author=A.L.%20Delcher&amp;publication_year=2004&amp;journal=Genome%20Biol.&amp;volume=5&amp;pages=R12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-2-r12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-2-r12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-2-r12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14759262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kielbasa</div> <div class="given-names">S.M.</div></div> <div class="name"><div class="surname">Wan</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Horton</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Frith</div> <div class="given-names">M.C.</div></div></div> <div class="year">2011</div>, <div class="article-title">Adaptive seeds tame genomic sequence comparison</div>, <div class="source">Genome Res.</div> , <div class="volume">21</div>, <div class="fpage">487</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20seeds%20tame%20genomic%20sequence%20comparison&amp;author=S.M.%20Kielbasa&amp;author=R.%20Wan&amp;author=K.%20Sato&amp;author=P.%20Horton&amp;author=M.C.%20Frith&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=487-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.113985.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.113985.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.113985.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21209072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Price</div> <div class="given-names">A.L.</div></div> <div class="name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div> <div class="name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div></div> <div class="year">2005</div>, <div class="article-title">De novo identification of repeat families in large genomes</div>. In: <div class="conf-name">Proceedings of the 13 Annual International conference on Intelligent Systems for Molecular Biology (ISMB-05)</div>, <div class="comment">Detroit, MI, USA. <em>Bioinformatics</em>, <strong>21</strong> (suppl 1), i351–i358</div>.</div></div></div></div><div content-id="DSV002C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Miyatake</div> <div class="given-names">K.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Draft genome sequence of eggplant (<em>Solanum melongena</em> L.): the representative <em>Solanum</em> species indigenous to the Old World</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">649</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20eggplant%20(Solanum%20melongena%20L.)%3A%20the%20representative%20Solanum%20species%20indigenous%20to%20the%20Old%20World&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=K.%20Miyatake&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=649-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25233906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kofler</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Schlotterer</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Lelley</div> <div class="given-names">T.</div></div></div> <div class="year">2007</div>, <div class="article-title">SciRoKo: a new tool for whole genome microsatellite search and investigation</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1683</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SciRoKo%3A%20a%20new%20tool%20for%20whole%20genome%20microsatellite%20search%20and%20investigation&amp;author=R.%20Kofler&amp;author=C.%20Schlotterer&amp;author=T.%20Lelley&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1683-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17463017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div> <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">1997</div>, <div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div> <div class="year">2003</div>, <div class="article-title">Gene prediction with a hidden Markov model and a new intron submodel</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div><div class="issue">(Suppl. 2)</div>, <div class="fpage">ii215</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=ii215-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Guigó</div> <div class="given-names">R.</div></div></div> <div class="year">1998</div>, <div class="article-title">Assembling genes from predicted exons in linear time with dynamic programming</div>, <div class="source">J. Comput. Biol.</div> , <div class="volume">5</div>, <div class="fpage">681</div>–<div class="lpage">702</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembling%20genes%20from%20predicted%20exons%20in%20linear%20time%20with%20dynamic%20programming&amp;author=R.%20Guig%C3%B3&amp;publication_year=1998&amp;journal=J.%20Comput.%20Biol.&amp;volume=5&amp;pages=681-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.1998.5.681" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.1998.5.681" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.1998.5.681"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10072084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">W116</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Mulder</div> <div class="given-names">N.J.</div></div> <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Attwood</div> <div class="given-names">T.K.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">New developments in the InterPro database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">35</div>, <div class="fpage">D224</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20developments%20in%20the%20InterPro%20database&amp;author=N.J.%20Mulder&amp;author=R.%20Apweiler&amp;author=T.K.%20Attwood&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D224-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17202162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">2009</div>, <div class="article-title">A new generation of homology search tools based on probabilistic inference</div>, <div class="source">Genome Informatics</div> , <div class="volume">23</div>, <div class="fpage">205</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20generation%20of%20homology%20search%20tools%20based%20on%20probabilistic%20inference&amp;author=S.R.%20Eddy&amp;publication_year=2009&amp;journal=Genome%20Informatics&amp;volume=23&amp;pages=205-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20180275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="collab">The Gene Ontology Consortium</div>. <div class="year">2000</div>, <div class="article-title">Gene ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet.</div> , <div class="volume">25</div>, <div class="fpage">25</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div><div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=The+Gene+Ontology+Consortium.+2000,+Gene+ontology:+tool+for+the+unification+of+biology,+Nat.+Genet.,+25,+25–9." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV002C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div> <div class="name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div> <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>et al.  </div> <div class="year">2003</div>, <div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">4</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Fujibuchi</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Bono</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div></div> <div class="year">1999</div>, <div class="article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">27</div>, <div class="fpage">29</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20Kyoto%20Encyclopedia%20of%20Genes%20and%20Genomes&amp;author=H.%20Ogata&amp;author=S.%20Goto&amp;author=K.%20Sato&amp;author=W.%20Fujibuchi&amp;author=H.%20Bono&amp;author=M.%20Kanehisa&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=29-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div> <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div> <div class="name"><div class="surname">Plewniak</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Jeanmougin</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div></div> <div class="year">1997</div>, <div class="article-title">The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">4876</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ClustalX%20windows%20interface%3A%20flexible%20strategies%20for%20multiple%20sequence%20alignment%20aided%20by%20quality%20analysis%20tools&amp;author=J.D.%20Thompson&amp;author=T.J.%20Gibson&amp;author=F.%20Plewniak&amp;author=F.%20Jeanmougin&amp;author=D.G.%20Higgins&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=4876-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.24.4876" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.24.4876"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9396791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Filipski</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, <div class="article-title">MEGA6: molecular evolutionary genetics analysis version 6.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">30</div>, <div class="fpage">2725</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20molecular%20evolutionary%20genetics%20analysis%20version%206.0&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.</div></div></div> <div class="year">2012</div>, <div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">The Sequence alignment/map (SAM) format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20alignment%2Fmap%20(SAM)%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Danecek</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Auton</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Abecasis</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, <div class="article-title">The Variant Call Format and VCFtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2156</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Variant%20Call%20Format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Cingolani</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Platts</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Wangle</div> <div class="given-names">L.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3</div>, <div class="source"><em>Fly</em> (Austin)</div> , <div class="volume">6</div>, <div class="fpage">80</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.%20Wangle&amp;publication_year=2012&amp;journal=Fly%20(Austin)&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Ffly.19695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Tammi</div> <div class="given-names">M.T.</div></div></div> <div class="year">2009</div>, <div class="article-title">CNV-seq, a new method to detect copy number variation using high-throughput sequencing</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CNV-seq%2C%20a%20new%20method%20to%20detect%20copy%20number%20variation%20using%20high-throughput%20sequencing&amp;author=C.%20Xie&amp;author=M.T.%20Tammi&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-80" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-80" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-80"> </span></div> </div></div></div></div></div><div content-id="DSV002C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Genome structure of the Legume</div>, <div class="source"><em>Lotus Japonicas</em>, <em>DNA Res</em>.</div> , <div class="volume">15</div>, <div class="fpage">227</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20Legume&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;publication_year=2008&amp;journal=Lotus%20Japonicas%2C%20DNA%20Res.&amp;volume=15&amp;pages=227-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn008"> </span></div> </div></div></div></div></div><div content-id="DSV002C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="collab">The Potato Genome Sequencing Consortium</div>. <div class="year">2011</div>, <div class="article-title">Genome sequence and analysis of the tuber crop potato</div>, <div class="source">Nature</div> , <div class="volume">475</div>, <div class="fpage">189</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743474" class="link link-pub-id" target="_blank">PubMed</a></div> </div><div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=The+Potato+Genome+Sequencing+Consortium.+2011,+Genome+sequence+and+analysis+of+the+tuber+crop+potato,+Nature,+475,+189–95." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV002C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="collab">The Tomato Genome Consortium</div>. <div class="year">2012</div>, <div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>, <div class="source">Nature</div> , <div class="volume">485</div>, <div class="fpage">635</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div><div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=The+Tomato+Genome+Consortium.+2012,+The+tomato+genome+sequence+provides+insights+into+fleshy+fruit+evolution,+Nature,+485,+635–41." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV002C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Takahata</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Tanaka</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Otani</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">Inhibition of the expression of the <em>starch synthase II</em> gene leads to lower pasting temperature in sweetpotato starch</div>, <div class="source">Plant Cell Rep.</div> , <div class="volume">29</div>, <div class="fpage">535</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inhibition%20of%20the%20expression%20of%20the%20starch%20synthase%20II%20gene%20leads%20to%20lower%20pasting%20temperature%20in%20sweetpotato%20starch&amp;author=Y.%20Takahata&amp;author=M.%20Tanaka&amp;author=M.%20Otani&amp;publication_year=2010&amp;journal=Plant%20Cell%20Rep.&amp;volume=29&amp;pages=535-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-010-0842-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-010-0842-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-010-0842-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20306051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hirose</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Terao</div> <div class="given-names">T.</div></div></div> <div class="year">2004</div>, <div class="article-title">A comprehensive expression analysis of the starch synthase gene family in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Planta</div> , <div class="volume">220</div>, <div class="fpage">9</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20expression%20analysis%20of%20the%20starch%20synthase%20gene%20family%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=T.%20Hirose&amp;author=T.%20Terao&amp;publication_year=2004&amp;journal=Planta&amp;volume=220&amp;pages=9-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-004-1314-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-004-1314-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-004-1314-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15232694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Peterson</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div> <div class="name"><div class="surname">Gundlach</div> <div class="given-names">H.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres</div>, <div class="source">Nature</div> , <div class="volume">492</div>, <div class="fpage">423</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repeated%20polyploidization%20of%20Gossypium%20genomes%20and%20the%20evolution%20of%20spinnable%20cotton%20fibres&amp;author=A.%20Peterson&amp;author=J.F.%20Wendel&amp;author=H.%20Gundlach&amp;publication_year=2012&amp;journal=Nature&amp;volume=492&amp;pages=423-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11798" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11798"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23257886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">D'Hont</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Denoeud</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">The banana (<em>Musa acuminata</em>) genome and the evolution of monocotyledonous plants</div>, <div class="source">Nature</div> , <div class="volume">488</div>, <div class="fpage">213</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20banana%20(Musa%20acuminata)%20genome%20and%20the%20evolution%20of%20monocotyledonous%20plants&amp;author=A.%20D'Hont&amp;author=F.%20Denoeud&amp;author=J.M.%20Aury&amp;publication_year=2012&amp;journal=Nature&amp;volume=488&amp;pages=213-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22801500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV002C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV002C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Dissection of the octoploid strawberry genome by deep sequencing of the genomes of <em>Fragaria</em> species</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">169</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissection%20of%20the%20octoploid%20strawberry%20genome%20by%20deep%20sequencing%20of%20the%20genomes%20of%20Fragaria%20species&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=S.%20Kosugi&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24282021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=333025&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv002-dsv002supp_tables.xlsx?Expires=2147483647&amp;Signature=tHhGUFX5HH3w7-YyNik9GYlUe-Z7t6FaviVt1W4rgRwUcoRdl7CBVnSMxEKN2XrOuxGS6Im09kPInf0afUffcRiaZFW1-Dz2Xz8zyouaMQO04G6OSRFNDyNBHmgZQYigKRIkHTtEpvRiM9kJYfO2sj4VhHLkX1wteGTUkZ~tfCZDInInyeJwtD18UAqpAXF5J41XuVUiU3kz1Y4t-WvS9RhBKLWn5u1VnMYNbm9y7vExUijXWDCjg4M8DElklcOMEbyMRVsuxvYNpPRWxER9EWjG3Q62ta-fCs1aVnof1FubI4Wg3dmD3ayHunWG83QwiplHUXlnHoCPap~awaO35Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv002-dsv002supp_figures.pptx?Expires=2147483647&amp;Signature=ibvKr22cZAl8S0pYRq958aPecOZpzgpr9MHY5xtDKfWpIuHB6VUk~v661SWquiH8NVtY1l8UM13EmfIsamoCaLxq~Ya7e4NXVZ0NFjZgb7OFwJCCoAh7p4Y~a0PNgS-Zw~uhDxe4Cv5ayxrae4NELbyQnE3DeQFgKJSRCYIC3CaHH4ARRTza~v50FSJ4t7C4W3Q-A8Uy1TA6PyVljhSclylmq9eEMhYi2jtn6cUdKfWxxJDaibhfGEgQ3njo~yjNrKsblvp1YJNbnysU2Hhs-ik3wjYl5SkoTM8l1tIJIjRbFlYOm8Ay~lkgbKW4L~ETTZsJop4oI2U0sds2LWMXPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pptx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv002/2/dsv002-dsv002supp_fig10.pdf?Expires=2147483647&amp;Signature=rUdVntsd7Zzodv3W9CwkBzZLoziW~4PdIY934tI2AAe-vf0A3HL22CG2pvDCV3o2qvrzWBp51wPaaqIgt7gJi8cEAAmx36IPG99YEwWGTW1KjNqW~ryYjCQddyKxh9H5PQH-991BxNL11MXmetUrbsv96lJjedCjQsY-n-gdItDiW9tZnMvKNSCWaiCeaJWBDPUV4PyS~e4pzNbSRp1CTPtUUgx1dxCwEITQxRnK1pThQvKz5xpk55PBipMXu35Pb1l90lGsnzdX6amx7ntx92-XaKyKh7Ia1zXMI-IJ4STFUbBDOlbWnPgTmN00Zpxifj4d-NBj9kJ43lCvSAvhVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,208</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000353644600005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">32</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv002"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="333025"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000353644600005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f2%2f171%2f333025">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000353644600005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (32)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F2%2F171%2F333025" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/333025'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Survey of genome sequences in a wild sweet potato, Ipomoea trifida (H. B. K.) G. Don" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv002");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f171%2f333025">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

