

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/6/593/420691","@type":"ScholarlyArticle","name":"Genome-Wide Association Studies Using Single Nucleotide Polymorphism Markers Developed by Re-Sequencing of the Genomes of Cultivated Tomato","datePublished":"2013-07-31","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2013-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst033","keywords":["genome-wide association studies","linkage disequilibrium","whole-genome re-sequencing","single nucleotide polymorphism","tomato"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/6/593/420691","author":[{"name":"Shirasawa, Kenta","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fukuoka, Hiroyuki","affiliation":"2  NARO Institute of Vegetable and Tea Sciences ,  360 Kusawa, Ano, Tsu, Mie 514-2392 ,  Japan","@type":"Person"},{"name":"Matsunaga, Hiroshi","affiliation":"2  NARO Institute of Vegetable and Tea Sciences ,  360 Kusawa, Ano, Tsu, Mie 514-2392 ,  Japan","@type":"Person"},{"name":"Kobayashi, Yuhko","affiliation":"3  Life Science Research Center ,  Mie University ,  1577 Kurimamachiya, Tsu, Mie 514-8507 ,  Japan","@type":"Person"},{"name":"Kobayashi, Issei","affiliation":"3  Life Science Research Center ,  Mie University ,  1577 Kurimamachiya, Tsu, Mie 514-8507 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  With the aim of understanding relationship between genetic and phenotypic variations in cultivated tomato, single nucleotide polymorphism (SNP) marke","pageStart":"593","pageEnd":"603","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-Wide Association Studies Using Single Nucleotide Polymorphism Markers Developed by Re-Sequencing of the Genomes of Cultivated Tomato","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-Wide Association Studies Using Single Nucleotide Polymorphism Markers Developed by Re-Sequencing of the Genomes of Cultivated Tomato" />
    <meta property="og:description" content="Abstract.  With the aim of understanding relationship between genetic and phenotypic variations in cultivated tomato, single nucleotide polymorphism (SNP) marke" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/6/593/420691" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-07-31" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fukuoka, Hiroyuki" /><meta name="citation_author_institution" content="NARO Institute of Vegetable and Tea Sciences, 360 Kusawa, Ano, Tsu, Mie 514-2392, Japan" /><meta name="citation_author" content="Matsunaga, Hiroshi" /><meta name="citation_author_institution" content="NARO Institute of Vegetable and Tea Sciences, 360 Kusawa, Ano, Tsu, Mie 514-2392, Japan" /><meta name="citation_author" content="Kobayashi, Yuhko" /><meta name="citation_author_institution" content="Life Science Research Center, Mie University, 1577 Kurimamachiya, Tsu, Mie 514-8507, Japan" /><meta name="citation_author" content="Kobayashi, Issei" /><meta name="citation_author_institution" content="Life Science Research Center, Mie University, 1577 Kurimamachiya, Tsu, Mie 514-8507, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Genome-Wide Association Studies Using Single Nucleotide Polymorphism Markers Developed by Re-Sequencing of the Genomes of Cultivated Tomato" /><meta name="citation_firstpage" content="593" /><meta name="citation_lastpage" content="603" /><meta name="citation_doi" content="10.1093/dnares/dst033" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2013/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Tomato; citation_publisher=Springer, New York; Genome Mapping and Molecular Breeding in Plants;  citation_year=2007;  citation_volume=5;  citation_pages=1-125; " /><meta name="citation_reference" content="citation_title=Chromosomal evolution in the plant family Solanaceae; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=182" /><meta name="citation_reference" content="citation_title=DNA marker applications to molecular genetics and genomics in tomato; Breed. Sci.;  citation_year=2013;  citation_volume=63;  citation_pages=21-30; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism discovery in cultivated tomato via sequencing by synthesis; Plant Genome;  citation_year=2012;  citation_volume=5;  citation_pages=17-29; " /><meta name="citation_reference" content="citation_title=An integrated map of genetic variation from 1092 human genomes; citation_author=The 1000 Genomes Project Consortium; Nature;  citation_year=2012;  citation_volume=491;  citation_pages=56-65; " /><meta name="citation_reference" content="citation_title=Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds; citation_author=The Bovine HapMap Consortium; Science;  citation_year=2009;  citation_volume=324;  citation_pages=528-32; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=956-63; " /><meta name="citation_reference" content="citation_title=Multiple reference genomes and transcriptomes for Arabidopsis thaliana; Nature;  citation_year=2011;  citation_volume=477;  citation_pages=419-23; " /><meta name="citation_reference" content="citation_title=A map of rice genome variation reveals the origin of cultivated rice; Nature;  citation_year=2012;  citation_volume=490;  citation_pages=497-501; " /><meta name="citation_reference" content="citation_title=Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=105-11; " /><meta name="citation_reference" content="citation_title=Toward a saturated linkage map in tomato based on isozymes and random cDNA sequences; Genetics;  citation_year=1986;  citation_volume=112;  citation_pages=887-98; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution; citation_author=The Tomato Genome Consortium; Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP genotyping to infer the effects on gene functions in tomato; DNA Res;  citation_year=2013;  citation_volume=20;  citation_pages=221-33; " /><meta name="citation_reference" content="citation_title=Advances in plant genome sequencing; Plant J.;  citation_year=2012;  citation_volume=70;  citation_pages=177-90; " /><meta name="citation_reference" content="citation_title=Current challenges in de novo plant genome sequencing and assembly; Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=243" /><meta name="citation_reference" content="citation_title=Can genomics boost productivity of orphan crops?; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=1172-6; " /><meta name="citation_reference" content="citation_title=Tomato Chuukanbohon Nou 11â€™, a tomato parental line with a short-internode trait; Bull. Natl. Inst. Veg. Tea Sci.;  citation_year=2007;  citation_volume=6;  citation_pages=65-76; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=SNP2CAPS: a SNP and INDEL analysis tool for CAPS marker development; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=e5" /><meta name="citation_reference" content="citation_title=Web-based primer design for single nucleotide polymorphism analysis; Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=613-5; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; Methods Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=SNP discovery and linkage map construction in cultivated tomato; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=381-91; " /><meta name="citation_reference" content="citation_title=Distribution of SUN, OVATE, LC, and FAS in the tomato germplasm and the relationship to fruit shape diversity; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=275-85; " /><meta name="citation_reference" content="citation_title=GGT 2.0: versatile software for visualization and analysis of genetic data; J. Hered.;  citation_year=2008;  citation_volume=99;  citation_pages=232-6; " /><meta name="citation_reference" content="citation_title=Development of Capsicum EST-SSR markers for species identification and in silico mapping onto the tomato genome sequence; Mol. Breed.;  citation_year=2013;  citation_volume=31;  citation_pages=101-10; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=TASSEL: software for association mapping of complex traits in diverse samples; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2633-5; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=945-59; " /><meta name="citation_reference" content="citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study; Mol. Ecol.;  citation_year=2005;  citation_volume=14;  citation_pages=2611-20; " /><meta name="citation_reference" content="citation_title=Haploview: analysis and visualization of LD and haplotype maps; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=263-5; " /><meta name="citation_reference" content="citation_title=The structure of haplotype blocks in the human genome; Science;  citation_year=2002;  citation_volume=296;  citation_pages=2225-9; " /><meta name="citation_reference" content="citation_title=An interspecific linkage map of SSR and intronic polymorphism markers in tomato; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=731-9; " /><meta name="citation_reference" content="citation_title=The making of a compound inflorescence in tomato and related nightshades; PLoS Biol.;  citation_year=2008;  citation_volume=6;  citation_pages=e288" /><meta name="citation_reference" content="citation_title=The SELF-PRUNING gene of tomato regulates vegetative to reproductive switching of sympodial meristems and is the ortholog of CEN and TFL1; Development;  citation_year=1998;  citation_volume=125;  citation_pages=1979-89; " /><meta name="citation_reference" content="citation_title=Regulatory change in YABBY-like transcription factor led to evolution of extreme fruit size during tomato domestication; Nat. Genet.;  citation_year=2008;  citation_volume=40;  citation_pages=800-4; " /><meta name="citation_reference" content="citation_title=Increase in tomato locule number is controlled by two single-nucleotide polymorphisms located near WUSCHEL; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=2244-54; " /><meta name="citation_reference" content="citation_title=A new class of regulatory genes underlying the cause of pear-shaped tomato fruit; Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=13302-6; " /><meta name="citation_reference" content="citation_title=Uniform ripening encodes a Golden 2-like transcription factor regulating tomato fruit chloroplast development; Science;  citation_year=2012;  citation_volume=336;  citation_pages=1711-5; " /><meta name="citation_reference" content="citation_title=Biochemical and molecular analysis of pink tomatoes: deregulated expression of the gene encoding transcription factor SlMYB12 leads to pink tomato fruit color; Plant Physiol.;  citation_year=2010;  citation_volume=152;  citation_pages=71-84; " /><meta name="citation_reference" content="citation_title=Genetic and biosystematic studies on two new sibling species of Lycopersicon from interandean Peru; Theor. Appl. Genet.;  citation_year=1976;  citation_volume=47;  citation_pages=55-68; " /><meta name="citation_reference" content="citation_title=Mapping of Micro-Tom BAC-end sequences to the reference tomato genome reveals possible genome rearrangements and polymorphisms; Inter. J. Plant Genomics;  citation_year=2012;  citation_volume=2012;  citation_pages=437026" /><meta name="citation_reference" content="citation_title=Development of a large SNP genotyping array and generation of high-density genetic maps in tomato; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e40563" /><meta name="citation_reference" content="citation_title=Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=1053-9; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping in tomato (Solanum lycopersicum) is possible using genome admixture of Solanum lycopersicum var. cerasiforme; G3;  citation_year=2012;  citation_volume=2;  citation_pages=853-64; " /><meta name="citation_reference" content="citation_title=High-density SNP genotyping of tomato (Solanum lycopersicum L.) reveals patterns of genetic variation due to breeding; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e45520" /><meta name="citation_reference" content="citation_title=Genome-wide association studies for agronomical traits in a world wide spring barley collection; BMC Plant Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=16" /><meta name="citation_reference" content="citation_title=Quality of core collections for effective utilisation of genetic resources review, discussion and interpretation; Theor. Appl. Genet.;  citation_year=2013;  citation_volume=126;  citation_pages=289-305; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/6/593/1165572/dst033.pdf" /><meta name="description" content="Abstract.  With the aim of understanding relationship between genetic and phenotypic variations in cultivated tomato, single nucleotide polymorphism (SNP) marke" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/6/593/420691" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/6/593/420691" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/6/593/420691");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst033';
    </script>
<title>
	Genome-Wide Association Studies Using Single Nucleotide Polymorphism Markers Developed by Re-Sequencing of the Genomes of Cultivated Tomato | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./420691" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd1rPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGlyYXNhd2EsIEtlbnRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3Vva2EsIEhpcm95dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTkFSTyBJbnN0aXR1dGUgb2YgVmVnZXRhYmxlIGFuZCBUZWEgU2NpZW5jZXMsIDM2MCBLdXNhd2EsIEFubywgVHN1LCBNaWUgNTE0LTIzOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWF0c3VuYWdhLCBIaXJvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTkFSTyBJbnN0aXR1dGUgb2YgVmVnZXRhYmxlIGFuZCBUZWEgU2NpZW5jZXMsIDM2MCBLdXNhd2EsIEFubywgVHN1LCBNaWUgNTE0LTIzOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29iYXlhc2hpLCBZdWhrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxpZmUgU2NpZW5jZSBSZXNlYXJjaCBDZW50ZXIsIE1pZSBVbml2ZXJzaXR5LCAxNTc3IEt1cmltYW1hY2hpeWEsIFRzdSwgTWllIDUxNC04NTA3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvYmF5YXNoaSwgSXNzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMaWZlIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBNaWUgVW5pdmVyc2l0eSwgMTU3NyBLdXJpbWFtYWNoaXlhLCBUc3UsIE1pZSA1MTQtODUwNywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIaXJha2F3YSwgSGlkZWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iklzb2JlLCBTYWNoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhYmF0YSwgU2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZS1XaWRlIEFzc29jaWF0aW9uIFN0dWRpZXMgVXNpbmcgU2luZ2xlIE51Y2xlb3RpZGUgUG9seW1vcnBoaXNtIE1hcmtlcnMgRGV2ZWxvcGVkIGJ5IFJlLVNlcXVlbmNpbmcgb2YgdGhlIEdlbm9tZXMgb2YgQ3VsdGl2YXRlZCBUb21hdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI1OTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjYwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9tYXRvOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldyBZb3JrOyBHZW5vbWUgTWFwcGluZyBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nIGluIFBsYW50czsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MS0xMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWFsIGV2b2x1dGlvbiBpbiB0aGUgcGxhbnQgZmFtaWx5IFNvbGFuYWNlYWU7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE4MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtYXJrZXIgYXBwbGljYXRpb25zIHRvIG1vbGVjdWxhciBnZW5ldGljcyBhbmQgZ2Vub21pY3MgaW4gdG9tYXRvOyBCcmVlZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTIxLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZGlzY292ZXJ5IGluIGN1bHRpdmF0ZWQgdG9tYXRvIHZpYSBzZXF1ZW5jaW5nIGJ5IHN5bnRoZXNpczsgUGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNy0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZWdyYXRlZCBtYXAgb2YgZ2VuZXRpYyB2YXJpYXRpb24gZnJvbSAxMDkyIGh1bWFuIGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1UaGUgMTAwMCBHZW5vbWVzIFByb2plY3QgQ29uc29ydGl1bTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ5MTsgIGNpdGF0aW9uX3BhZ2VzPTU2LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBzdXJ2ZXkgb2YgU05QIHZhcmlhdGlvbiB1bmNvdmVycyB0aGUgZ2VuZXRpYyBzdHJ1Y3R1cmUgb2YgY2F0dGxlIGJyZWVkczsgY2l0YXRpb25fYXV0aG9yPVRoZSBCb3ZpbmUgSGFwTWFwIENvbnNvcnRpdW07IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI0OyAgY2l0YXRpb25fcGFnZXM9NTI4LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2VxdWVuY2luZyBvZiBtdWx0aXBsZSBBcmFiaWRvcHNpcyB0aGFsaWFuYSBwb3B1bGF0aW9uczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz05NTYtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIHJlZmVyZW5jZSBnZW5vbWVzIGFuZCB0cmFuc2NyaXB0b21lcyBmb3IgQXJhYmlkb3BzaXMgdGhhbGlhbmE7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00Nzc7ICBjaXRhdGlvbl9wYWdlcz00MTktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWFwIG9mIHJpY2UgZ2Vub21lIHZhcmlhdGlvbiByZXZlYWxzIHRoZSBvcmlnaW4gb2YgY3VsdGl2YXRlZCByaWNlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDkwOyAgY2l0YXRpb25fcGFnZXM9NDk3LTUwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzZXF1ZW5jaW5nIDUwIGFjY2Vzc2lvbnMgb2YgY3VsdGl2YXRlZCBhbmQgd2lsZCByaWNlIHlpZWxkcyBtYXJrZXJzIGZvciBpZGVudGlmeWluZyBhZ3Jvbm9taWNhbGx5IGltcG9ydGFudCBnZW5lczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEwNS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkIGEgc2F0dXJhdGVkIGxpbmthZ2UgbWFwIGluIHRvbWF0byBiYXNlZCBvbiBpc296eW1lcyBhbmQgcmFuZG9tIGNETkEgc2VxdWVuY2VzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz04ODctOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0b21hdG8gZ2Vub21lIHNlcXVlbmNlIHByb3ZpZGVzIGluc2lnaHRzIGludG8gZmxlc2h5IGZydWl0IGV2b2x1dGlvbjsgY2l0YXRpb25fYXV0aG9yPVRoZSBUb21hdG8gR2Vub21lIENvbnNvcnRpdW07IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODU7ICBjaXRhdGlvbl9wYWdlcz02MzUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIFNOUCBnZW5vdHlwaW5nIHRvIGluZmVyIHRoZSBlZmZlY3RzIG9uIGdlbmUgZnVuY3Rpb25zIGluIHRvbWF0bzsgRE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTIyMS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWR2YW5jZXMgaW4gcGxhbnQgZ2Vub21lIHNlcXVlbmNpbmc7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9MTc3LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdXJyZW50IGNoYWxsZW5nZXMgaW4gZGUgbm92byBwbGFudCBnZW5vbWUgc2VxdWVuY2luZyBhbmQgYXNzZW1ibHk7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhbiBnZW5vbWljcyBib29zdCBwcm9kdWN0aXZpdHkgb2Ygb3JwaGFuIGNyb3BzPzsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTExNzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9tYXRvIENodXVrYW5ib2hvbiBOb3UgMTHigJksIGEgdG9tYXRvIHBhcmVudGFsIGxpbmUgd2l0aCBhIHNob3J0LWludGVybm9kZSB0cmFpdDsgQnVsbC4gTmF0bC4gSW5zdC4gVmVnLiBUZWEgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NjUtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENPRyBkYXRhYmFzZTogYW4gdXBkYXRlZCB2ZXJzaW9uIGluY2x1ZGVzIGV1a2FyeW90ZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlAyQ0FQUzogYSBTTlAgYW5kIElOREVMIGFuYWx5c2lzIHRvb2wgZm9yIENBUFMgbWFya2VyIGRldmVsb3BtZW50OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1lNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdlYi1iYXNlZCBwcmltZXIgZGVzaWduIGZvciBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gYW5hbHlzaXM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz02MTMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbWVyMyBvbiB0aGUgV1dXIGZvciBnZW5lcmFsIHVzZXJzIGFuZCBmb3IgYmlvbG9naXN0IHByb2dyYW1tZXJzOyBNZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9MzY1LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlAgZGlzY292ZXJ5IGFuZCBsaW5rYWdlIG1hcCBjb25zdHJ1Y3Rpb24gaW4gY3VsdGl2YXRlZCB0b21hdG87IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MzgxLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0cmlidXRpb24gb2YgU1VOLCBPVkFURSwgTEMsIGFuZCBGQVMgaW4gdGhlIHRvbWF0byBnZXJtcGxhc20gYW5kIHRoZSByZWxhdGlvbnNoaXAgdG8gZnJ1aXQgc2hhcGUgZGl2ZXJzaXR5OyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0yNzUtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdHVCAyLjA6IHZlcnNhdGlsZSBzb2Z0d2FyZSBmb3IgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgb2YgZ2VuZXRpYyBkYXRhOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0yMzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgQ2Fwc2ljdW0gRVNULVNTUiBtYXJrZXJzIGZvciBzcGVjaWVzIGlkZW50aWZpY2F0aW9uIGFuZCBpbiBzaWxpY28gbWFwcGluZyBvbnRvIHRoZSB0b21hdG8gZ2Vub21lIHNlcXVlbmNlOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTEwMS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTU6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdXNpbmcgbWF4aW11bSBsaWtlbGlob29kLCBldm9sdXRpb25hcnkgZGlzdGFuY2UsIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSBtZXRob2RzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UQVNTRUw6IHNvZnR3YXJlIGZvciBhc3NvY2lhdGlvbiBtYXBwaW5nIG9mIGNvbXBsZXggdHJhaXRzIGluIGRpdmVyc2Ugc2FtcGxlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yNjMzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmVyZW5jZSBvZiBwb3B1bGF0aW9uIHN0cnVjdHVyZSB1c2luZyBtdWx0aWxvY3VzIGdlbm90eXBlIGRhdGE7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1NTsgIGNpdGF0aW9uX3BhZ2VzPTk0NS01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIHRoZSBudW1iZXIgb2YgY2x1c3RlcnMgb2YgaW5kaXZpZHVhbHMgdXNpbmcgdGhlIHNvZnR3YXJlIFNUUlVDVFVSRTogYSBzaW11bGF0aW9uIHN0dWR5OyBNb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjYxMS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGFwbG92aWV3OiBhbmFseXNpcyBhbmQgdmlzdWFsaXphdGlvbiBvZiBMRCBhbmQgaGFwbG90eXBlIG1hcHM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjYzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzdHJ1Y3R1cmUgb2YgaGFwbG90eXBlIGJsb2NrcyBpbiB0aGUgaHVtYW4gZ2Vub21lOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI5NjsgIGNpdGF0aW9uX3BhZ2VzPTIyMjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZXJzcGVjaWZpYyBsaW5rYWdlIG1hcCBvZiBTU1IgYW5kIGludHJvbmljIHBvbHltb3JwaGlzbSBtYXJrZXJzIGluIHRvbWF0bzsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjE7ICBjaXRhdGlvbl9wYWdlcz03MzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1ha2luZyBvZiBhIGNvbXBvdW5kIGluZmxvcmVzY2VuY2UgaW4gdG9tYXRvIGFuZCByZWxhdGVkIG5pZ2h0c2hhZGVzOyBQTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjg4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNFTEYtUFJVTklORyBnZW5lIG9mIHRvbWF0byByZWd1bGF0ZXMgdmVnZXRhdGl2ZSB0byByZXByb2R1Y3RpdmUgc3dpdGNoaW5nIG9mIHN5bXBvZGlhbCBtZXJpc3RlbXMgYW5kIGlzIHRoZSBvcnRob2xvZyBvZiBDRU4gYW5kIFRGTDE7IERldmVsb3BtZW50OyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTEyNTsgIGNpdGF0aW9uX3BhZ2VzPTE5NzktODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgY2hhbmdlIGluIFlBQkJZLWxpa2UgdHJhbnNjcmlwdGlvbiBmYWN0b3IgbGVkIHRvIGV2b2x1dGlvbiBvZiBleHRyZW1lIGZydWl0IHNpemUgZHVyaW5nIHRvbWF0byBkb21lc3RpY2F0aW9uOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTgwMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNyZWFzZSBpbiB0b21hdG8gbG9jdWxlIG51bWJlciBpcyBjb250cm9sbGVkIGJ5IHR3byBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGxvY2F0ZWQgbmVhciBXVVNDSEVMOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0yMjQ0LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBjbGFzcyBvZiByZWd1bGF0b3J5IGdlbmVzIHVuZGVybHlpbmcgdGhlIGNhdXNlIG9mIHBlYXItc2hhcGVkIHRvbWF0byBmcnVpdDsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xMzMwMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vbmlmb3JtIHJpcGVuaW5nIGVuY29kZXMgYSBHb2xkZW4gMi1saWtlIHRyYW5zY3JpcHRpb24gZmFjdG9yIHJlZ3VsYXRpbmcgdG9tYXRvIGZydWl0IGNobG9yb3BsYXN0IGRldmVsb3BtZW50OyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMzNjsgIGNpdGF0aW9uX3BhZ2VzPTE3MTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvY2hlbWljYWwgYW5kIG1vbGVjdWxhciBhbmFseXNpcyBvZiBwaW5rIHRvbWF0b2VzOiBkZXJlZ3VsYXRlZCBleHByZXNzaW9uIG9mIHRoZSBnZW5lIGVuY29kaW5nIHRyYW5zY3JpcHRpb24gZmFjdG9yIFNsTVlCMTIgbGVhZHMgdG8gcGluayB0b21hdG8gZnJ1aXQgY29sb3I7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1MjsgIGNpdGF0aW9uX3BhZ2VzPTcxLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuZCBiaW9zeXN0ZW1hdGljIHN0dWRpZXMgb24gdHdvIG5ldyBzaWJsaW5nIHNwZWNpZXMgb2YgTHljb3BlcnNpY29uIGZyb20gaW50ZXJhbmRlYW4gUGVydTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk3NjsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTU1LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIG9mIE1pY3JvLVRvbSBCQUMtZW5kIHNlcXVlbmNlcyB0byB0aGUgcmVmZXJlbmNlIHRvbWF0byBnZW5vbWUgcmV2ZWFscyBwb3NzaWJsZSBnZW5vbWUgcmVhcnJhbmdlbWVudHMgYW5kIHBvbHltb3JwaGlzbXM7IEludGVyLiBKLiBQbGFudCBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDEyOyAgY2l0YXRpb25fcGFnZXM9NDM3MDI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgYSBsYXJnZSBTTlAgZ2Vub3R5cGluZyBhcnJheSBhbmQgZ2VuZXJhdGlvbiBvZiBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXBzIGluIHRvbWF0bzsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU0MDU2MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlc2VxdWVuY2luZyBvZiAzMSB3aWxkIGFuZCBjdWx0aXZhdGVkIHNveWJlYW4gZ2Vub21lcyBpZGVudGlmaWVzIHBhdHRlcm5zIG9mIGdlbmV0aWMgZGl2ZXJzaXR5IGFuZCBzZWxlY3Rpb247IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MTA1My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBtYXBwaW5nIGluIHRvbWF0byAoU29sYW51bSBseWNvcGVyc2ljdW0pIGlzIHBvc3NpYmxlIHVzaW5nIGdlbm9tZSBhZG1peHR1cmUgb2YgU29sYW51bSBseWNvcGVyc2ljdW0gdmFyLiBjZXJhc2lmb3JtZTsgRzM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTg1My02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IFNOUCBnZW5vdHlwaW5nIG9mIHRvbWF0byAoU29sYW51bSBseWNvcGVyc2ljdW0gTC4pIHJldmVhbHMgcGF0dGVybnMgb2YgZ2VuZXRpYyB2YXJpYXRpb24gZHVlIHRvIGJyZWVkaW5nOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ1NTIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllcyBmb3IgYWdyb25vbWljYWwgdHJhaXRzIGluIGEgd29ybGQgd2lkZSBzcHJpbmcgYmFybGV5IGNvbGxlY3Rpb247IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbGl0eSBvZiBjb3JlIGNvbGxlY3Rpb25zIGZvciBlZmZlY3RpdmUgdXRpbGlzYXRpb24gb2YgZ2VuZXRpYyByZXNvdXJjZXMgcmV2aWV3LCBkaXNjdXNzaW9uIGFuZCBpbnRlcnByZXRhdGlvbjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz0yODktMzA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvNi81OTMvMTE2NTU3Mi9kc3QwMzMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdpdGggdGhlIGFpbSBvZiB1bmRlcnN0YW5kaW5nIHJlbGF0aW9uc2hpcCBiZXR3ZWVuIGdlbmV0aWMgYW5kIHBoZW5vdHlwaWMgdmFyaWF0aW9ucyBpbiBjdWx0aXZhdGVkIHRvbWF0bywgc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIChTTlApIG1hcmtlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00MjA2OTEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkwlfXl3NuSM5Nn4oyz7t6k/MUz1o=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f593%2f420691" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f593%2f420691" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16367" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&amp;Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/6">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6232609">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232609">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232611">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232611">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232617">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232617">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232637">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232637">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232663">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232663">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232670">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232670">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232672">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232672">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232674">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232674">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232676">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232676">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6232678">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6232678">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/6/583/420514">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/6/605/420907">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide Association Studies Using Single Nucleotide Polymorphism Markers Developed by Re-Sequencing of the Genomes of Cultivated Tomato
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3935. Fax. +81 438-52-3934. E-mail: <a href="mailto:shirasaw@kazusa.or.jp" target="_blank">shirasaw@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Fukuoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">NARO Institute of Vegetable and Tea Sciences</div>, <div class="addr-line">360 Kusawa, Ano, Tsu, Mie 514-2392</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Fukuoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukuoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukuoka H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroyuki Fukuoka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Matsunaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">NARO Institute of Vegetable and Tea Sciences</div>, <div class="addr-line">360 Kusawa, Ano, Tsu, Mie 514-2392</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Matsunaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsunaga H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsunaga H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Matsunaga</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuhko Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Life Science Research Center</div>, <div class="insititution">Mie University</div>, <div class="addr-line">1577 Kurimamachiya, Tsu, Mie 514-8507</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuhko+Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobayashi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kobayashi Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuhko Kobayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Issei Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Life Science Research Center</div>, <div class="insititution">Mie University</div>, <div class="addr-line">1577 Kurimamachiya, Tsu, Mie 514-8507</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Issei+Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobayashi I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kobayashi I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Issei Kobayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 593â€“603, <a href='https://doi.org/10.1093/dnares/dst033'>https://doi.org/10.1093/dnares/dst033</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">31 July 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">04 June 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 July 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="420691" href="/dnaresearch/article-pdf/20/6/593/1165572/dst033.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kenta Shirasawa, Hiroyuki Fukuoka, Hiroshi Matsunaga, Yuhko Kobayashi, Issei Kobayashi, Hideki Hirakawa, Sachiko Isobe, Satoshi Tabata;  Genome-Wide Association Studies Using Single Nucleotide Polymorphism Markers Developed by Re-Sequencing of the Genomes of Cultivated Tomato, <em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 593â€“603, <a href='https://doi.org/10.1093/dnares/dst033'>https://doi.org/10.1093/dnares/dst033</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/420691?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/420691?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/420691?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/420691?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/420691?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/420691?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/420691?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-Wide%20Association%20Studies%20Using%20Single%20Nucleotide%20Polymorphism%20Markers%20Developed%20by%20Re-Sequencing%20of%20the%20Genomes%20of%20Cultivated%20Tomato&amp;publicationDate=2013-07-31&amp;volumeNum=20&amp;issueNum=6&amp;author=Shirasawa%2C%20Kenta%3B%20Fukuoka%2C%20Hiroyuki&amp;startPage=593&amp;endPage=603&amp;contentId=10.1093%2Fdnares%2Fdst033&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f593%2f420691" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6232609 id="6232609" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>With the aim of understanding relationship between genetic and phenotypic variations in cultivated tomato, single nucleotide polymorphism (SNP) markers covering the whole genome of cultivated tomato were developed and genome-wide association studies (GWAS) were performed. The whole genomes of six tomato lines were sequenced with the ABI-5500xl SOLiD sequencer. Sequence reads covering âˆ¼13.7Ã— of the genome for each line were obtained, and mapped onto tomato reference genomes (SL2.40) to detect âˆ¼1.5 million SNP candidates. Of the identified SNPs, 1.5% were considered to confer gene functions. In the subsequent Illumina GoldenGate assay for 1536 SNPs, 1293 SNPs were successfully genotyped, and 1248 showed polymorphisms among 663 tomato accessions. The whole-genome linkage disequilibrium (LD) analysis detected highly biased LD decays between euchromatic (58 kb) and heterochromatic regions (13.8 Mb). Subsequent GWAS identified SNPs that were significantly associated with agronomical traits, with SNP loci located near genes that were previously reported as candidates for these traits. This study demonstrates that attractive loci can be identified by performing GWAS with a large number of SNPs obtained from re-sequencing analysis.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome-wide association studies&quot;">genome-wide association studies</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage disequilibrium&quot;">linkage disequilibrium</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome re-sequencing&quot;">whole-genome re-sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single nucleotide polymorphism&quot;">single nucleotide polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="tomato">tomato</a></div>                            <h2 scrollto-destination=6232611 id="6232611" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Tomato (<em>Solanum lycopersicum</em>), which is considered to be an important crop, originated from South and Central America, and spread to the rest of the world with accompanying morphological diversification.<sup><a reveal-id="DST033C1" data-open="DST033C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The Solanaceae family, to which tomato belongs, includes other important crop species, such as potato (<em>S. tuberosum</em>), eggplant (<em>S. melongena</em>), tobacco (<em>Nicotiana tabacum</em>), and pepper (<em>Capsicum annuum</em>). Comparative genomics within these various genera and species have greatly accelerated understanding of their genome evolution and the genetic mechanisms that confer phenotypic diversity to these species.<sup><a reveal-id="DST033C2" data-open="DST033C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Furthermore, several interspecific genetic linkage maps have been constructed between cultivated tomato and its wild relatives (<em>S. chmielewskii</em>, <em>S. habrochaites</em>, <em>S. pennellii</em>, and <em>S. pimpinellifolium</em>).<sup><a reveal-id="DST033C3" data-open="DST033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> These maps allow identification of the genes responsible for interspecific phenotypic variations, including disease resistance, fruit size and shape, and plant architecture.<sup><a reveal-id="DST033C3" data-open="DST033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> However, few genetic studies have reported intraspecific variations due to its narrow genetic diversity.<sup><a reveal-id="DST033C3" data-open="DST033C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST033C4" data-open="DST033C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>In the field of human and animal genomic and genetic studies, the availability of whole-genome sequence data has resulted in more rapid advances in re-sequencing analysis and genome-wide association studies (GWAS) than in classical genetics and quantitative trait locus (QTL) mapping.<sup><a reveal-id="DST033C5" data-open="DST033C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST033C6" data-open="DST033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In plants such as rice (<em>Oryza sativa</em>) and <em>Arabidopsis thaliana</em>, the initial plant species for which whole-genome sequences were available provided representative targets for such analysis.<sup><a reveal-id="DST033C7  DST033C10" data-open="DST033C7  DST033C10" class="link link-ref link-reveal xref-bibr">7â€“10</a></sup></p><p>Tomato has also been used as a model plant in classical and molecular genetics,<sup><a reveal-id="DST033C11" data-open="DST033C11" class="link link-ref link-reveal xref-bibr">11</a></sup> due to autogamous diploidy (2<em>n</em> = 2<em>x</em> = 24) and a relatively compact genome (âˆ¼950 Mb). Recently, the whole-genome sequence of tomato was published.<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Furthermore, Hirakawa <em>et al.</em><sup><a reveal-id="DST033C13" data-open="DST033C13" class="link link-ref link-reveal xref-bibr">13</a></sup> inferred the functions of 200 SNPs among the transcribed sequences of cultivated tomato lines by determining their positions in predicted genes on the tomato genome. These results are expected to accelerate the understanding of genetic mechanisms that confer phenotypic variations among tomato cultivars.</p><p>Massive parallel sequencing and genotyping methods have contributed to progress in genetics and genomics. Next-generation sequencers (NGSs), such as HiSeq2500 (Illumina), the GS FLX+ system (Roche), 5500xl SOLiD (Life Technologies), and Ion Proton (Life Technologies), have been employed for <em>de novo</em> assembly of genome sequences and re-sequencing analyses of genomes of several organisms.<sup><a reveal-id="DST033C14" data-open="DST033C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST033C15" data-open="DST033C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In such re-sequencing analysis, sequence reads from the whole genome are mapped onto the reference genome to identify nucleotide variations, including single nucleotide polymorphisms (SNPs) and insertions/deletions (indels).<sup><a reveal-id="DST033C14" data-open="DST033C14" class="link link-ref link-reveal xref-bibr">14</a></sup> A large amount of nucleotide sequence data (up to Mb- or Gb-scale), redundantly covering the whole-genome sequence, can be obtained simultaneously by NGS technologies. This allows a huge number of the nucleotide variations to be identified cheaply and within a relatively short period of time. The identified SNPs can be used, for example, for polymorphic analysis of germplasm collections, which, in turn, allows genetic analyses such as QTL mapping, GWAS, and genomic selection.<sup><a reveal-id="DST033C16" data-open="DST033C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Large-scale SNP genotyping is often performed with commercially available array-based platforms, such as Infinium (Illumina), GoldenGate (Illumina), and Axiom Genotyping Solution (Affymetrix).</p><p>Tomato accessions, so-called genetic resources, are stocked in several gene banks, including the Tomato Genetic Resource Center (TGRC), USA (<a class="link link-uri" href="http://tgrc.ucdavis.edu">http://tgrc.ucdavis.edu</a>); the National Institute of Agrobiological Sciences (NIAS) Genebank, Japan (<a class="link link-uri" href="http://www.gene.affrc.go.jp">http://www.gene.affrc.go.jp</a>); and the NARO Institute of Vegetable and Tea Science (NIVTS), Japan (<a class="link link-uri" href="http://www.naro.affrc.go.jp/vegetea">http://www.naro.affrc.go.jp/vegetea</a>). In the NIAS and NIVTS Genebanks, over 1500 tomato lines have been deposited from &gt;50 countries. The morphological traits of each line are recorded when the plants are reproduced, whereas DNA-based genetic variation has not yet been evaluated. By combining massive parallel sequencing and high-throughput genotyping technologies, it is now possible to probe genome-wide genetic diversity in the large number of tomato accessions currently available. In addition, associations between genetic and phenotypic variations can be identified in the genetic resources by using morphological traits recorded in the NIVTS and NIAS Genebanks. These studies would provide useful knowledge for molecular genetic analysis and breeding. In this study, we re-sequenced six tomato lines to discover novel SNPs that could be used to estimate the ratio of the SNPs contributing to the phenotypic variation. The identified candidate SNPs were used for GWAS to predict the loci responsible for agronomically important traits, e.g. fruit size and shape and plant architecture.</p>                            <h2 scrollto-destination=6232617 id="6232617" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=6232618 id="6232618" class="section-title" data-legacy-id=s2a>2.1. Plant materials and DNA isolation</h3>
<p>Six inbred lines, â€˜Ailsa Craigâ€™ (AIC), â€˜Furikomaâ€™ (FRK), â€˜M82â€™ (M82), â€˜Tomato Chuukanbonhon Nou 11â€™ (PL11), â€˜Ponderosaâ€™ (PON), and â€˜Reginaâ€™ (REG), which were selected as representative lines from the clusters in the phylogenetic tree obtained in our previous study,<sup><a reveal-id="DST033C13" data-open="DST033C13" class="link link-ref link-reveal xref-bibr">13</a></sup> were used for whole-genome re-sequencing (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). AIC and PON are greenhouse types, and FRK and M82 are processing types suited for field cultivation. PL11 is a breeding material developed at the NIVTS for a short-internode trait,<sup><a reveal-id="DST033C17" data-open="DST033C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and REG is a dwarf tomato with cherry-type fruits obtained from Sakata Seeds Co., Japan. All materials except for REG are available from the NIVTS, Japan.</p><p>The number of genotyped tomato accessions with SNPs was 663, of which 641, 9, 6, 5, 1, and 1 were derived from the NIVTS, Japan; five private companies (De Ruiter Seeds Co., The Netherlands; Sakata Seeds Co., Japan; Suntory Holdings Ltd., Japan; Takii Seeds Co., Japan; and Vilmorin Seeds Co., France); the TGRC at the University of California, USA; the National BioResource Project (NBRP) at the University of Tsukuba, Japan; Cornell University, USA; and the Institut National de la Recherche Agronomique (INRA), France, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Total genomic DNA was isolated from leaves of a single plant from each line using a DNeasy plant mini kit (Qiagen).</p>                            <h3 scrollto-destination=6232621 id="6232621" class="section-title" data-legacy-id=s2b>2.2. Whole-genome re-sequencing and identification of SNP candidates</h3>
<p>Total genomic DNA from the six lines, such as AIC, FRK, M82, PL11, PON, and REG, was used for whole-genome shotgun sequencing according to the standard protocol (Life Technologies). The nucleotide sequences were determined using the 5500xl SOLiD sequencer (Life Technologies) in the paired-end mode (35 + 75 bases). The data obtained were mapped onto the reference genome sequence of â€˜Heinz 1706â€™ (H1706) ver. SL2.40<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> for SNP discovery using the LifeScope Genomic Analysis software (Life Technologies) with default parameters. When heterozygous SNPs were discovered in any one of six lines, they were manually excluded from the list of SNP candidates.</p><p>The SNP candidates were classified into seven groups according to ITAG2.3 predictions of the gene positions on the tomato genome<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> as follows: intergenic SNPs, SNPs at the donor and acceptor splice sites bordering two bases of introns, intron SNPs, SNPs at untranslated regions (UTRs), synonymous SNPs, missense SNPs, and nonsense SNPs. The functional categories of tomato genes predicted in the ITAG2.3<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> were assigned by BLASTP<sup><a reveal-id="DST033C18" data-open="DST033C18" class="link link-ref link-reveal xref-bibr">18</a></sup> searches against the eukaryotic orthologous groups (KOG) database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/COG">http://www.ncbi.nlm.nih.gov/COG</a>), with E-value cut-off of 1Eâˆ’4.<sup><a reveal-id="DST033C19" data-open="DST033C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>SNP2CAPS<sup><a reveal-id="DST033C20" data-open="DST033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and dCAPS Finder 2.0<sup><a reveal-id="DST033C21" data-open="DST033C21" class="link link-ref link-reveal xref-bibr">21</a></sup> were used for developing cleaved amplified polymorphic sequence (CAPS) and derived CAPS (dCAPS) markers, respectively. Oligonucleotides for the markers were designed using the PRIMER3 software.<sup><a reveal-id="DST033C22" data-open="DST033C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h3 scrollto-destination=6232625 id="6232625" class="section-title" data-legacy-id=s2c>2.3. SNP genotyping</h3>
<p>A total of 1536 SNPs were selected for Illumina GoldenGate SNP genotyping of the 663 tomato accessions. The Illumina GoldenGate assay and subsequent SNP calling were performed as described by Shirasawa <em>et al.</em><sup><a reveal-id="DST033C23" data-open="DST033C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Polymorphic analysis of CAPS and dCAPS markers including <em>FAS</em>, <em>SP</em>, and <em>OVATE</em><sup><a reveal-id="DST033C23" data-open="DST033C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DST033C24" data-open="DST033C24" class="link link-ref link-reveal xref-bibr">24</a></sup> was performed as described by Shirasawa <em>et al.</em><sup><a reveal-id="DST033C23" data-open="DST033C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                            <h3 scrollto-destination=6232627 id="6232627" class="section-title" data-legacy-id=s2d>2.4. Data analysis</h3>
                            <h4 scrollto-destination=6232628 id="6232628" class="section-title" data-legacy-id=s2d1>2.4.1. Clustering of the genetic resources</h4>
<p>The genetic distances and Jaccard's similarity coefficients of all combinations of any two accessions were calculated from the genotypic data using the GGT2 software<sup><a reveal-id="DST033C25" data-open="DST033C25" class="link link-ref link-reveal xref-bibr">25</a></sup> as described by Shirasawa <em>et al</em>.<sup><a reveal-id="DST033C26" data-open="DST033C26" class="link link-ref link-reveal xref-bibr">26</a></sup> A dendrogram of the genetic resources was established using the neighbor-joining method in the MEGA5 software.<sup><a reveal-id="DST033C27" data-open="DST033C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p><p>Principal component analysis (PCA) was also performed to determine the relationship between samples using the TASSEL software,<sup><a reveal-id="DST033C28" data-open="DST033C28" class="link link-ref link-reveal xref-bibr">28</a></sup> in which SNPs with minor allele frequencies (MAFs) of &lt;0.05 were removed and the number of components was limited to three.</p><p>The STRUCTURE software,<sup><a reveal-id="DST033C29" data-open="DST033C29" class="link link-ref link-reveal xref-bibr">29</a></sup> in which SNPs with MAFs of &gt;0.00 were included, was used to assess the genetic relationships of the investigated lines. The degree of admixture in each line was estimated under the conditions of a 100 000 burn-in period and 100 000 Markov Chain Monte Carlo replications. The ideal number of clusters (K) was estimated from the output of 20 independent calculations as described by Evanno <em>et al.</em><sup><a reveal-id="DST033C30" data-open="DST033C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p>                            <h4 scrollto-destination=6232632 id="6232632" class="section-title" data-legacy-id=s2d2>2.4.2. Linkage disequilibrium and haplotyping analysis</h4>
<p>Linkage disequilibriums (LDs) of all SNP pairs on each chromosome were detected using the Haploview software<sup><a reveal-id="DST033C31" data-open="DST033C31" class="link link-ref link-reveal xref-bibr">31</a></sup> with the following parameters: MAF, â‰¥0.05; Hardyâ€“Weinberg <em>P</em>-value cut-off, 0; and percentage of genotyped lines, â‰¥0.75. Haplotypes and tag SNPs were predicted based on the estimated LD blocks according to the definition of Gabriel <em>et al.</em><sup><a reveal-id="DST033C32" data-open="DST033C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p>                            <h4 scrollto-destination=6232634 id="6232634" class="section-title" data-legacy-id=s2d3>2.4.3. Genome-wide association studies</h4>
<p>Associations between genotypes and phenotypes were analysed using the mixed linear model (MLM) using the TASSEL program<sup><a reveal-id="DST033C28" data-open="DST033C28" class="link link-ref link-reveal xref-bibr">28</a></sup> with the following parameters: MAF of â‰¥0.05. In the association analysis, we considered the kinship matrix based on the SNP data in the model of MLM, while population structure was excluded from the model since it could not be detected in the tomato accessions with the STRUCTURE analysis. The thresholds for the association were set to a â€“log<em>P</em> of &gt;5.06 and 4.36 at a significant level of 1 and 5%, respectively, after Bonferroni multiple test correction.</p><p>On NIAS Genebank databases (<a class="link link-uri" href="http://www.gene.affrc.go.jp">http://www.gene.affrc.go.jp</a>), 71 phenotypic traits are registered for 9â€“479 accessions (the numbers of investigated lines differ depending on the traits) as actual measured numeric data, qualitative data, and ranked data. They were investigated in the field and/or under greenhouse conditions over a number of years (1983â€“2011) at multiple locations (seven sites in Japan and Taiwan). The phenotypic data for each accession redundantly recorded in multi-years and locations were averaged, so that the data could be regarded as continuous numerical data for the MLM. Of these, 23 traits that scored in &gt;100 lines genotyped in this study were tested for the GWAS.</p>                            <h2 scrollto-destination=6232637 id="6232637" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6232638 id="6232638" class="section-title" data-legacy-id=s3a>3.1. Whole-genome shotgun re-sequencing of cultivated tomato</h3>
<p>Whole-genome shotgun re-sequencing was performed for the six inbred tomato lines, such as AIC, FRK, M82, PL11, PON, and REG. DNA samples tagged with line-specific index sequences were subjected to sequencing analysis using the 5500xl SOLiD sequencer (Life Technologies) in the paired-end mode (75 + 35 bases) (TableÂ <a reveal-id="DST033TB1" data-open="DST033TB1" class="link link-reveal link-table xref-fig">1</a>). A total of 708.3 million read pairs corresponding to 77.9 Gb DNA were obtained (13.7Ã— mean depth for each line). In the subsequent <em>in silico</em> analysis with the LifeScope Genomic Analysis software (Life Technologies), 53.9% of the obtained sequences covered 93.4% of the reference genome sequence of H1706 ver. SL2.40<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> at 9.2Ã— coverage on average for each line (TableÂ <a reveal-id="DST033TB1" data-open="DST033TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The other 46.1% reads were omitted from the mapping results due to the low quality of the reads and repetitive sequences in the tomato genome. </p>                        <a id="6232640" scrollto-destination="6232640"></a>
<div content-id="DST033TB1" class="table-modal"><div class="table-wrap"><div id="DST033TB1" class="table-wrap-title" data-id="DST033TB1"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Statistics of the re-sequenced genomes in the six tomato lines</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Line nameÂ </th><th>Number of read pairs (reads)Â </th><th>Total sequence length (bp)Â </th><th>Re-sequencing depth<sup>a</sup> (times)Â </th><th>% of genome coverage<sup>b</sup>Â </th><th>Coverage depth (times)Â </th></tr></thead><tbody><tr><td>AICÂ </td><td>104 913 343Â </td><td>11 540 467 730Â </td><td>12.1Â </td><td>93.5Â </td><td>8.3Â </td></tr><tr><td>FRKÂ </td><td>91 995 911Â </td><td>10 119 550 210Â </td><td>10.7Â </td><td>93.2Â </td><td>7.2Â </td></tr><tr><td>M82Â </td><td>107 226 071Â </td><td>11 794 867 810Â </td><td>12.4Â </td><td>93.0Â </td><td>8.3Â </td></tr><tr><td>PL11Â </td><td>121 752 304Â </td><td>13 392 753 440Â </td><td>14.1Â </td><td>93.7Â </td><td>9.7Â </td></tr><tr><td>PONÂ </td><td>94 404 895Â </td><td>10 384 538 450Â </td><td>10.9Â </td><td>93.4Â </td><td>7.7Â </td></tr><tr><td>REGÂ </td><td>188 026 505Â </td><td>20 682 915 550Â </td><td>21.8Â </td><td>93.3Â </td><td>14.1Â </td></tr><tr><td>MeanÂ </td><td>118 053 172Â </td><td>12 985 848 865Â </td><td>13.7Â </td><td>93.4Â </td><td>9.2Â </td></tr></tbody></table></div><div content-id="DST033TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Line nameÂ </th><th>Number of read pairs (reads)Â </th><th>Total sequence length (bp)Â </th><th>Re-sequencing depth<sup>a</sup> (times)Â </th><th>% of genome coverage<sup>b</sup>Â </th><th>Coverage depth (times)Â </th></tr></thead><tbody><tr><td>AICÂ </td><td>104 913 343Â </td><td>11 540 467 730Â </td><td>12.1Â </td><td>93.5Â </td><td>8.3Â </td></tr><tr><td>FRKÂ </td><td>91 995 911Â </td><td>10 119 550 210Â </td><td>10.7Â </td><td>93.2Â </td><td>7.2Â </td></tr><tr><td>M82Â </td><td>107 226 071Â </td><td>11 794 867 810Â </td><td>12.4Â </td><td>93.0Â </td><td>8.3Â </td></tr><tr><td>PL11Â </td><td>121 752 304Â </td><td>13 392 753 440Â </td><td>14.1Â </td><td>93.7Â </td><td>9.7Â </td></tr><tr><td>PONÂ </td><td>94 404 895Â </td><td>10 384 538 450Â </td><td>10.9Â </td><td>93.4Â </td><td>7.7Â </td></tr><tr><td>REGÂ </td><td>188 026 505Â </td><td>20 682 915 550Â </td><td>21.8Â </td><td>93.3Â </td><td>14.1Â </td></tr><tr><td>MeanÂ </td><td>118 053 172Â </td><td>12 985 848 865Â </td><td>13.7Â </td><td>93.4Â </td><td>9.2Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Re-sequencing depth = total sequence length/tomato genome size (950 Mb).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Mapped percentage on the reference genome sequences (SL2.40, 760 Mb) at â‰¥1 coverage.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6232640" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST033TB1" class="table-wrap-title" data-id="DST033TB1"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Statistics of the re-sequenced genomes in the six tomato lines</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Line nameÂ </th><th>Number of read pairs (reads)Â </th><th>Total sequence length (bp)Â </th><th>Re-sequencing depth<sup>a</sup> (times)Â </th><th>% of genome coverage<sup>b</sup>Â </th><th>Coverage depth (times)Â </th></tr></thead><tbody><tr><td>AICÂ </td><td>104 913 343Â </td><td>11 540 467 730Â </td><td>12.1Â </td><td>93.5Â </td><td>8.3Â </td></tr><tr><td>FRKÂ </td><td>91 995 911Â </td><td>10 119 550 210Â </td><td>10.7Â </td><td>93.2Â </td><td>7.2Â </td></tr><tr><td>M82Â </td><td>107 226 071Â </td><td>11 794 867 810Â </td><td>12.4Â </td><td>93.0Â </td><td>8.3Â </td></tr><tr><td>PL11Â </td><td>121 752 304Â </td><td>13 392 753 440Â </td><td>14.1Â </td><td>93.7Â </td><td>9.7Â </td></tr><tr><td>PONÂ </td><td>94 404 895Â </td><td>10 384 538 450Â </td><td>10.9Â </td><td>93.4Â </td><td>7.7Â </td></tr><tr><td>REGÂ </td><td>188 026 505Â </td><td>20 682 915 550Â </td><td>21.8Â </td><td>93.3Â </td><td>14.1Â </td></tr><tr><td>MeanÂ </td><td>118 053 172Â </td><td>12 985 848 865Â </td><td>13.7Â </td><td>93.4Â </td><td>9.2Â </td></tr></tbody></table></div><div content-id="DST033TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Line nameÂ </th><th>Number of read pairs (reads)Â </th><th>Total sequence length (bp)Â </th><th>Re-sequencing depth<sup>a</sup> (times)Â </th><th>% of genome coverage<sup>b</sup>Â </th><th>Coverage depth (times)Â </th></tr></thead><tbody><tr><td>AICÂ </td><td>104 913 343Â </td><td>11 540 467 730Â </td><td>12.1Â </td><td>93.5Â </td><td>8.3Â </td></tr><tr><td>FRKÂ </td><td>91 995 911Â </td><td>10 119 550 210Â </td><td>10.7Â </td><td>93.2Â </td><td>7.2Â </td></tr><tr><td>M82Â </td><td>107 226 071Â </td><td>11 794 867 810Â </td><td>12.4Â </td><td>93.0Â </td><td>8.3Â </td></tr><tr><td>PL11Â </td><td>121 752 304Â </td><td>13 392 753 440Â </td><td>14.1Â </td><td>93.7Â </td><td>9.7Â </td></tr><tr><td>PONÂ </td><td>94 404 895Â </td><td>10 384 538 450Â </td><td>10.9Â </td><td>93.4Â </td><td>7.7Â </td></tr><tr><td>REGÂ </td><td>188 026 505Â </td><td>20 682 915 550Â </td><td>21.8Â </td><td>93.3Â </td><td>14.1Â </td></tr><tr><td>MeanÂ </td><td>118 053 172Â </td><td>12 985 848 865Â </td><td>13.7Â </td><td>93.4Â </td><td>9.2Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Re-sequencing depth = total sequence length/tomato genome size (950 Mb).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Mapped percentage on the reference genome sequences (SL2.40, 760 Mb) at â‰¥1 coverage.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6232640" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6232641 id="6232641" class="section-title" data-legacy-id=s3b>3.2. Identification of SNP candidates and their positions on the tomato genome</h3>
<p>Within the mapped sequence reads, a total of 2 011 984 SNP candidates were discovered between H1706 (SL2.40ch01 to SL2.40ch12) and the re-sequenced lines. Heterozygous and triallelic SNPs were often observed among the identified SNP candidates. They were considered false positives and were excluded from further analysis. As a result, a total of 1 473 798 SNPs, consisting of 836 676 transition and 637 122 transversion mutations, were identified as confident biallelic SNP candidates (Fig.Â <a reveal-id="DST033F1" data-open="DST033F1" class="link link-reveal link-table xref-fig">1</a>, <a class="link link-uri" href="http://www.kazusa.or.jp/tomato">http://www.kazusa.or.jp/tomato</a>), for which accuracy was validated using the GoldenGate assay described below. Among these, 170 173 SNPs were confirmed by their convertibility to CAPS markers (<a class="link link-uri" href="http://www.kazusa.or.jp/tomato">http://www.kazusa.or.jp/tomato</a>), which are considered a useful tool for conventional DNA polymorphic analysis. </p>                        <a id="6232643" scrollto-destination="6232643"></a>
<div id="DST033F1" data-id="DST033F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/m_dst03301.png?Expires=2147483647&Signature=EQLM7c80ctzyD7V5k6eg0j4Wf9jYVEUYxkSylIw9icT-cMYuojKe~wxJ4Nmb~PBkZxyHCNbedC6p0h8KzMnJvdhf6-eXpsL1HIDJwrTT~r-337~sZ84i32FWgGsEshs1hZZ8Kao6LOkV-uWaPAgWrp91HHQ-UhjI6BSHDyyzDuTNYeiXMh9botWAbpyqyeGdabLWGhva1wiJlFD5PjFNnQj36~6dWAXqjQbQb~s5iYzE2Iudk32ZSY9PFzaTur6SgRvr7QPOV1Gwo29f2j5QBlMuXtFv63C36abZk86NhNmH~G9jVQ3MtEobj-keUNTP0Qb0Ds7vxDc4REhuA-Ja~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Density maps for SNPs detected in six tomato lines with respect to the reference tomato genome, SL2.40. The colours in each block represent a continuum of SNP densities: low-to-high SNP densities are represented by green to red. Left-side elliptic bars indicate the tomato chromosomes. Horizontal lines in each chromosome bar show mapped positions of SNPs used for the GoldenGate assay (black for intergenic SNPs, red for SNPs at splice sites and intron SNPs, blue for SNPs at UTRs and synonymous SNPs, and green for missense SNPs and nonsense SNPs). Heterochromatic regions are indicated by vertical lines on the right of the chromosomes. Names of genes identified by map-based cloning in previous studies are shown on the right of the chromosomes." path-from-xml="dst03301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232643/dst03301.tiff" path-from-xml="dst03301.tiff" target="_blank">View large</a><a section="6232643" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst03301.png?Expires=2147483647&Signature=JzJjtn0pRumjEcl8ouRgmMmxcQbgeCMcLE14HMjpEK0m83gYO6p~yUjb6~D2CoggHPdRc9sNcE9s4UqqLOl8dxiNDfnHoCzO3NRXToofIO2zfEGzYohbWvgk0E1rFXMZI-SX8C4ZxgdCACita29a0v0q7PDvqoqfEfFzsVzpEbe8bo44eyxvEHIEo6HBYppAZOiNpcXXODa2edKu--6j3fvZCn3VwRu~JBFANTIemrhd27mH0ovRUwX5TMc29vBwzRCcIpDUtsRQk5e7-MQ9NY4mwdUiZT1YL64RPfVu3COnuUBgkumqzT35cpCrZswFF-hF4~rG8b8BZHinQmymWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232643&ar=420691&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Density maps for SNPs detected in six tomato lines with respect to the reference tomato genome, SL2.40. The colours in each block represent a continuum of SNP densities: low-to-high SNP densities are represented by green to red. Left-side elliptic bars indicate the tomato chromosomes. Horizontal lines in each chromosome bar show mapped positions of SNPs used for the GoldenGate assay (black for intergenic SNPs, red for SNPs at splice sites and intron SNPs, blue for SNPs at UTRs and synonymous SNPs, and green for missense SNPs and nonsense SNPs). Heterochromatic regions are indicated by vertical lines on the right of the chromosomes. Names of genes identified by map-based cloning in previous studies are shown on the right of the chromosomes.</p></div></div><div content-id="DST033F1" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/m_dst03301.png?Expires=2147483647&Signature=EQLM7c80ctzyD7V5k6eg0j4Wf9jYVEUYxkSylIw9icT-cMYuojKe~wxJ4Nmb~PBkZxyHCNbedC6p0h8KzMnJvdhf6-eXpsL1HIDJwrTT~r-337~sZ84i32FWgGsEshs1hZZ8Kao6LOkV-uWaPAgWrp91HHQ-UhjI6BSHDyyzDuTNYeiXMh9botWAbpyqyeGdabLWGhva1wiJlFD5PjFNnQj36~6dWAXqjQbQb~s5iYzE2Iudk32ZSY9PFzaTur6SgRvr7QPOV1Gwo29f2j5QBlMuXtFv63C36abZk86NhNmH~G9jVQ3MtEobj-keUNTP0Qb0Ds7vxDc4REhuA-Ja~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Density maps for SNPs detected in six tomato lines with respect to the reference tomato genome, SL2.40. The colours in each block represent a continuum of SNP densities: low-to-high SNP densities are represented by green to red. Left-side elliptic bars indicate the tomato chromosomes. Horizontal lines in each chromosome bar show mapped positions of SNPs used for the GoldenGate assay (black for intergenic SNPs, red for SNPs at splice sites and intron SNPs, blue for SNPs at UTRs and synonymous SNPs, and green for missense SNPs and nonsense SNPs). Heterochromatic regions are indicated by vertical lines on the right of the chromosomes. Names of genes identified by map-based cloning in previous studies are shown on the right of the chromosomes." path-from-xml="dst03301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232643/dst03301.tiff" path-from-xml="dst03301.tiff" target="_blank">View large</a><a section="6232643" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst03301.png?Expires=2147483647&Signature=JzJjtn0pRumjEcl8ouRgmMmxcQbgeCMcLE14HMjpEK0m83gYO6p~yUjb6~D2CoggHPdRc9sNcE9s4UqqLOl8dxiNDfnHoCzO3NRXToofIO2zfEGzYohbWvgk0E1rFXMZI-SX8C4ZxgdCACita29a0v0q7PDvqoqfEfFzsVzpEbe8bo44eyxvEHIEo6HBYppAZOiNpcXXODa2edKu--6j3fvZCn3VwRu~JBFANTIemrhd27mH0ovRUwX5TMc29vBwzRCcIpDUtsRQk5e7-MQ9NY4mwdUiZT1YL64RPfVu3COnuUBgkumqzT35cpCrZswFF-hF4~rG8b8BZHinQmymWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232643&ar=420691&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Density maps for SNPs detected in six tomato lines with respect to the reference tomato genome, SL2.40. The colours in each block represent a continuum of SNP densities: low-to-high SNP densities are represented by green to red. Left-side elliptic bars indicate the tomato chromosomes. Horizontal lines in each chromosome bar show mapped positions of SNPs used for the GoldenGate assay (black for intergenic SNPs, red for SNPs at splice sites and intron SNPs, blue for SNPs at UTRs and synonymous SNPs, and green for missense SNPs and nonsense SNPs). Heterochromatic regions are indicated by vertical lines on the right of the chromosomes. Names of genes identified by map-based cloning in previous studies are shown on the right of the chromosomes.</p></div></div><p>Different numbers of SNPs with respect to H1706 were observed in each line, e.g. 85 534 in PON, 85 670 in AIC, 120 329 in FRK, 245 730 in PL11, 710 904 in M82, and 1 102 982 in REG (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). SNP density with respect to H1706 was calculated to be, on average, one SNP per 516 bp (0.19%), and ranged from 1 SNP/689 bp (0.15%) in REG to 1 SNP/8884 bp (0.01%) in AIC, assuming a 760 Mb genome size for SL2.40 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The SNPs were unevenly distributed across the genomes, i.e. a remarkably large number of SNPs were observed on Chromosome 11 (Chr11) in PL11; Chr04, Chr05, and Chr11 in M82; and Chr04, Chr05, and Chr12 in REG. At the chromosomal segment level, the numbers of SNPs ranged from 1 (88â€“89 Mb position of Chr01 in M82) to 10 847 (34â€“35 Mb position of Chr05 in REG) using a 1-Mb window scale (Fig.Â <a reveal-id="DST033F1" data-open="DST033F1" class="link link-reveal link-table xref-fig">1</a>).</p><p>The identified SNP candidates were classified into seven groups according to their positions in predicted genes on the tomato genome sequence (see Section 2 for details). Of the 1 473 798 SNP candidates, 998, 279, 110, and 1 were redundantly mapped onto two, three, four, and five gene models, respectively, while the other 1 472 410 SNPs were positioned on a single gene model. As a result, a total of 1 475 688 SNP sites in gene models were targeted for classification. Among them, 1 316 332 (89.2%) were in intergenic spaces, corresponding to DNA sequences located between genes, including UTRs. The other 159 356 SNPs (10.8%) were in genic regions, of which 110 315 (7.5%) and 49 041 (3.3%) were in introns and exons, respectively (TableÂ <a reveal-id="DST033TB2" data-open="DST033TB2" class="link link-reveal link-table xref-fig">2</a>). The number of SNPs potentially affecting gene function was 22 805 (1.5%), including 156 SNPs at splice sites in introns, 558 resulting in nonsense codons, and 22 091 of missense codons. </p>                        <a id="6232646" scrollto-destination="6232646"></a>
<div content-id="DST033TB2" class="table-modal"><div class="table-wrap"><div id="DST033TB2" class="table-wrap-title" data-id="DST033TB2"><div class="label -label">TableÂ 2.</div><div class="caption"><p>The number of SNPs categorized into seven classes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="5">LineÂ </th><th rowspan="5">TotalÂ </th><th rowspan="5">IntergenicÂ </th><th colspan="6">Intragenic<hr />Â </th></tr><tr><th colspan="2">Intron<hr />Â </th><th colspan="4">Exon<hr />Â </th></tr><tr><th rowspan="3">Splice siteÂ </th><th rowspan="3">IntronÂ </th><th rowspan="3">UTRÂ </th><th colspan="3">CDS<hr />Â </th></tr><tr><th rowspan="2">SynonymousÂ </th><th colspan="2">Non-synonymous<hr />Â </th></tr><tr><th>MissenseÂ </th><th>NonsenseÂ </th></tr></thead><tbody><tr><td>AICÂ </td><td>85 721Â </td><td>70 707Â </td><td>26Â </td><td>9477Â </td><td>628Â </td><td>1769Â </td><td>3032Â </td><td>82Â </td></tr><tr><td>FRKÂ </td><td>120 379Â </td><td>104 542Â </td><td>27Â </td><td>10 082Â </td><td>645Â </td><td>1799Â </td><td>3175Â </td><td>109Â </td></tr><tr><td>M82Â </td><td>710 986Â </td><td>672 467Â </td><td>60Â </td><td>25 951Â </td><td>1529Â </td><td>4035Â </td><td>6708Â </td><td>236Â </td></tr><tr><td>PL11Â </td><td>245 805Â </td><td>213 427Â </td><td>34Â </td><td>22 083Â </td><td>1438Â </td><td>3646Â </td><td>5054Â </td><td>123Â </td></tr><tr><td>PONÂ </td><td>85 595Â </td><td>70 205Â </td><td>22Â </td><td>9847Â </td><td>661Â </td><td>1751Â </td><td>3017Â </td><td>92Â </td></tr><tr><td>REGÂ </td><td>1 104 787Â </td><td>984 271Â </td><td>123Â </td><td>83 074Â </td><td>7102Â </td><td>13 157Â </td><td>16 647Â </td><td>413Â </td></tr><tr><td>6 linesÂ </td><td>1 475 688Â </td><td>1 316 332Â </td><td>156Â </td><td>110 159Â </td><td>8982Â </td><td>17 410Â </td><td>22 091Â </td><td>558Â </td></tr></tbody></table></div><div content-id="DST033TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="5">LineÂ </th><th rowspan="5">TotalÂ </th><th rowspan="5">IntergenicÂ </th><th colspan="6">Intragenic<hr />Â </th></tr><tr><th colspan="2">Intron<hr />Â </th><th colspan="4">Exon<hr />Â </th></tr><tr><th rowspan="3">Splice siteÂ </th><th rowspan="3">IntronÂ </th><th rowspan="3">UTRÂ </th><th colspan="3">CDS<hr />Â </th></tr><tr><th rowspan="2">SynonymousÂ </th><th colspan="2">Non-synonymous<hr />Â </th></tr><tr><th>MissenseÂ </th><th>NonsenseÂ </th></tr></thead><tbody><tr><td>AICÂ </td><td>85 721Â </td><td>70 707Â </td><td>26Â </td><td>9477Â </td><td>628Â </td><td>1769Â </td><td>3032Â </td><td>82Â </td></tr><tr><td>FRKÂ </td><td>120 379Â </td><td>104 542Â </td><td>27Â </td><td>10 082Â </td><td>645Â </td><td>1799Â </td><td>3175Â </td><td>109Â </td></tr><tr><td>M82Â </td><td>710 986Â </td><td>672 467Â </td><td>60Â </td><td>25 951Â </td><td>1529Â </td><td>4035Â </td><td>6708Â </td><td>236Â </td></tr><tr><td>PL11Â </td><td>245 805Â </td><td>213 427Â </td><td>34Â </td><td>22 083Â </td><td>1438Â </td><td>3646Â </td><td>5054Â </td><td>123Â </td></tr><tr><td>PONÂ </td><td>85 595Â </td><td>70 205Â </td><td>22Â </td><td>9847Â </td><td>661Â </td><td>1751Â </td><td>3017Â </td><td>92Â </td></tr><tr><td>REGÂ </td><td>1 104 787Â </td><td>984 271Â </td><td>123Â </td><td>83 074Â </td><td>7102Â </td><td>13 157Â </td><td>16 647Â </td><td>413Â </td></tr><tr><td>6 linesÂ </td><td>1 475 688Â </td><td>1 316 332Â </td><td>156Â </td><td>110 159Â </td><td>8982Â </td><td>17 410Â </td><td>22 091Â </td><td>558Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6232646" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST033TB2" class="table-wrap-title" data-id="DST033TB2"><div class="label -label">TableÂ 2.</div><div class="caption"><p>The number of SNPs categorized into seven classes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="5">LineÂ </th><th rowspan="5">TotalÂ </th><th rowspan="5">IntergenicÂ </th><th colspan="6">Intragenic<hr />Â </th></tr><tr><th colspan="2">Intron<hr />Â </th><th colspan="4">Exon<hr />Â </th></tr><tr><th rowspan="3">Splice siteÂ </th><th rowspan="3">IntronÂ </th><th rowspan="3">UTRÂ </th><th colspan="3">CDS<hr />Â </th></tr><tr><th rowspan="2">SynonymousÂ </th><th colspan="2">Non-synonymous<hr />Â </th></tr><tr><th>MissenseÂ </th><th>NonsenseÂ </th></tr></thead><tbody><tr><td>AICÂ </td><td>85 721Â </td><td>70 707Â </td><td>26Â </td><td>9477Â </td><td>628Â </td><td>1769Â </td><td>3032Â </td><td>82Â </td></tr><tr><td>FRKÂ </td><td>120 379Â </td><td>104 542Â </td><td>27Â </td><td>10 082Â </td><td>645Â </td><td>1799Â </td><td>3175Â </td><td>109Â </td></tr><tr><td>M82Â </td><td>710 986Â </td><td>672 467Â </td><td>60Â </td><td>25 951Â </td><td>1529Â </td><td>4035Â </td><td>6708Â </td><td>236Â </td></tr><tr><td>PL11Â </td><td>245 805Â </td><td>213 427Â </td><td>34Â </td><td>22 083Â </td><td>1438Â </td><td>3646Â </td><td>5054Â </td><td>123Â </td></tr><tr><td>PONÂ </td><td>85 595Â </td><td>70 205Â </td><td>22Â </td><td>9847Â </td><td>661Â </td><td>1751Â </td><td>3017Â </td><td>92Â </td></tr><tr><td>REGÂ </td><td>1 104 787Â </td><td>984 271Â </td><td>123Â </td><td>83 074Â </td><td>7102Â </td><td>13 157Â </td><td>16 647Â </td><td>413Â </td></tr><tr><td>6 linesÂ </td><td>1 475 688Â </td><td>1 316 332Â </td><td>156Â </td><td>110 159Â </td><td>8982Â </td><td>17 410Â </td><td>22 091Â </td><td>558Â </td></tr></tbody></table></div><div content-id="DST033TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="5">LineÂ </th><th rowspan="5">TotalÂ </th><th rowspan="5">IntergenicÂ </th><th colspan="6">Intragenic<hr />Â </th></tr><tr><th colspan="2">Intron<hr />Â </th><th colspan="4">Exon<hr />Â </th></tr><tr><th rowspan="3">Splice siteÂ </th><th rowspan="3">IntronÂ </th><th rowspan="3">UTRÂ </th><th colspan="3">CDS<hr />Â </th></tr><tr><th rowspan="2">SynonymousÂ </th><th colspan="2">Non-synonymous<hr />Â </th></tr><tr><th>MissenseÂ </th><th>NonsenseÂ </th></tr></thead><tbody><tr><td>AICÂ </td><td>85 721Â </td><td>70 707Â </td><td>26Â </td><td>9477Â </td><td>628Â </td><td>1769Â </td><td>3032Â </td><td>82Â </td></tr><tr><td>FRKÂ </td><td>120 379Â </td><td>104 542Â </td><td>27Â </td><td>10 082Â </td><td>645Â </td><td>1799Â </td><td>3175Â </td><td>109Â </td></tr><tr><td>M82Â </td><td>710 986Â </td><td>672 467Â </td><td>60Â </td><td>25 951Â </td><td>1529Â </td><td>4035Â </td><td>6708Â </td><td>236Â </td></tr><tr><td>PL11Â </td><td>245 805Â </td><td>213 427Â </td><td>34Â </td><td>22 083Â </td><td>1438Â </td><td>3646Â </td><td>5054Â </td><td>123Â </td></tr><tr><td>PONÂ </td><td>85 595Â </td><td>70 205Â </td><td>22Â </td><td>9847Â </td><td>661Â </td><td>1751Â </td><td>3017Â </td><td>92Â </td></tr><tr><td>REGÂ </td><td>1 104 787Â </td><td>984 271Â </td><td>123Â </td><td>83 074Â </td><td>7102Â </td><td>13 157Â </td><td>16 647Â </td><td>413Â </td></tr><tr><td>6 linesÂ </td><td>1 475 688Â </td><td>1 316 332Â </td><td>156Â </td><td>110 159Â </td><td>8982Â </td><td>17 410Â </td><td>22 091Â </td><td>558Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6232646" target="_blank">View Large</a></div></div><p>The functions of genes having or not having the SNPs were investigated. First, a total of the 34 348 tomato genes predicted in the ITAG2.3<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> were classified into the three groups: 508 genes having nonsense SNPs (Group 1); 9436 genes having nonsynoymous SNPs including nonsense, missense SNPs, and SNPs at splice junctions (Group 2); and 24 404 genes not classified in the Group 2 (Group 3). BLASTP was then used to compare the protein sequences with those in the KOG database.<sup><a reveal-id="DST033C19" data-open="DST033C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The 15 974 predicted genes were classified into KOG categories. The distributions of the categories were similar between the Groups 2 and 3 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In the Group 3, on the other hand, the proportions of the Categories C (energy production and conversion) and T (signal transduction mechanisms) were relatively prominent, while those of the Categories M (cell wall/membrane/envelope biogenesis), O (post-translational modification, protein turnover, and chaperones), U (intracellular trafficking, secretion, and vesicular transport), Y (nuclear structure), and Z (cytoskeleton) were conversely low (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                            <h3 scrollto-destination=6232648 id="6232648" class="section-title" data-legacy-id=s3c>3.3. SNP genotyping of tomato accessions by the Illumina GoldenGate assay</h3>
<p>To select SNPs showing high polymorphism in the accessions, the 1 473 798 SNPs were filtered by the following criteria: (i) a LifeScope score of 0.000000; (ii) a 3:4 SNP segregation ratio in seven plant lines (H1706 and the six re-sequenced lines), or PL11-specific SNPs, or SNPs specific to two lines including PL11 [the later two criteria were set because the PL11 line is considered to be closely related to many modern F<sub>1</sub> hybrid cultivars (Fukuoka, personal communication)]; (iii) SNPs showing different segregation patterns among the seven lines within 3-cM windows covering whole genomes of a total length of 1500 cM;<sup><a reveal-id="DST033C33" data-open="DST033C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and (iv) an Illumina SNP score of &gt;0.6, as determined on the Illumina website (<a class="link link-uri" href="https://icom.illumina.com">https://icom.illumina.com</a>). Using these criteria, 1235 SNPs were selected (Fig.Â <a reveal-id="DST033F1" data-open="DST033F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). An additional 301 SNPs with MAFs of &gt;0.3 and an Illumina SNP score of 1.0 were selected based on data reported in our previous studies.<sup><a reveal-id="DST033C13" data-open="DST033C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST033C23" data-open="DST033C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p><p>A total of 663 tomato accessions (listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were genotyped with the 1536 SNPs using the GoldenGate assay. As a result, 1293 SNPs were successfully genotyped in the 663 accessions, satisfying the criteria of the GenomeStudio Data Analysis software (Illumina). Of the 1293 SNPs, 1248 (96.5%) and 1147 (88.7%) showed segregations within the 663 accessions within the threshold of MAFs of &gt;0 and â‰¥0.05, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The MAF values of the 1248 SNPs were evenly distributed from 0.001 to 0.5, and no significant differences in the distribution of the MAF values of the seven SNP categories were observed (data not shown). The ratios of heterozygous alleles and null alleles were high in the seven F<sub>1</sub> hybrids and three wild species, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The higher ratio in the three wild species could reflect polymorphisms at the probe annealing sites.<sup><a reveal-id="DST033C23" data-open="DST033C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In contrast, few heterozygous or null alleles were observed in the 23 inbred lines.</p>                            <h3 scrollto-destination=6232652 id="6232652" class="section-title" data-legacy-id=s3d>3.4. Clustering analyses of the tomato accessions</h3>
<p>The genetic distances between all combinations of any pairs in the 663 tomato accessions were calculated based on the genotypes of the 1248 SNPs. The genetic distances among the 663 accessions ranged from 0.00 to 0.72, with an average of 0.39. No obvious clusters were observed in the dendrogram of the genetic distances (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3A</a>). To evaluate this result, the genetic relationships between the accessions were determined by PCA, which showed that there were no clusters in the 663 lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3B</a>), because the individual proportions for PC1, 2, and 3 were 0.09, 0.06, and 0.05, respectively. Genetic relationship analysis using the STRUCTURE software indicated that there was no population structure in the accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3C</a>). This is in contrast to the six clusters identified by the delta-K method reported by Evanno <em>et al.</em><sup><a reveal-id="DST033C30" data-open="DST033C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p>                            <h3 scrollto-destination=6232654 id="6232654" class="section-title" data-legacy-id=s3e>3.5. Linkage disequilibrium and haplotype identifications</h3>
<p>Because no clear genetic structure was observed in the 663 accessions, LD across the tomato genome in these lines was investigated (Fig.Â <a reveal-id="DST033F2" data-open="DST033F2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S4 and S5</a>). A total of 123 LD blocks, i.e. chromosome sections showing significant LD (based on the definition of Gabriel <em>et al.</em><sup><a reveal-id="DST033C32" data-open="DST033C32" class="link link-ref link-reveal xref-bibr">32</a></sup>) between each pair of located SNPs, were observed across chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The 123 LD blocks comprised a total of 458 SNPs. The average length of the LD blocks was 3.2 Mb, ranging from 256 bp in Chr10 between solcap_snp_sl_8260 and SL2.40ch10_59989140W to 58.3 Mb in Chr01 between SL2.40ch01_7886746R and SL2.40ch01_66149134Y (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The lengths of LD blocks containing heterochromatic regions (average, 13.8 Mb) were longer than that in euchromatic regions (average, 58 kb) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). </p>                        <a id="6232656" scrollto-destination="6232656"></a>
<div id="DST033F2" data-id="DST033F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/m_dst03302.png?Expires=2147483647&Signature=oOD0vRraRYDsj1tayD5Wc-FirRORt3qpcspR48s7CXNBa9~dqEVBOrjgay689juF7RVqAGL1WGDSBD-h0LrbNEaBW01Tai9NVDQ1QeBHwymB~wbro01Z4MRjUNX8JmDawfoKV1d4shQ3dZ853AvIX38G2pWO12Dq6dfRxOdKPwUNvaVgnkFoSiyzf~Z-uHbip5WAgTIeRs6tYGCeVXRFEpdeeDvZX9kAKZ0GB6PzJqvWW-ga8-lBhawKF4EhFIgC8qoHMhPH-y-Ec5sYkj5ioXLP3LPxetAz-lhU3gpIpOLfEKC6m2ZHI13o2HPZWJpE0n0-XShzJjuC5bzy~rgsOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LD measures; r2 values against physical distance (Mb) between all pairs of SNPs located on the same chromosome." path-from-xml="dst03302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232656/dst03302.tiff" path-from-xml="dst03302.tiff" target="_blank">View large</a><a section="6232656" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst03302.png?Expires=2147483647&Signature=c-gtHWjIAX1xhijQAP7e6ST32pTtNlScZKHGuo9jDgK~Ez87I66nwNjGHO8~sRpN2GGUYH4mbZr7yYvpsvWP29WHg85KjJebCszM6mB2xq5K7OqatMRvDMcchKiTMiSGBMiyChmbwfiDg2h5CWdeR~VyptGZeLUwsvaErzaJTLpzKJK~VCMTlngxtU36I~FAL7VPevA4whF4ZDIq-~uFkTcDVTARsV2M8JVBBNGdDhcbB8I0NwLAeNifZgaKbdRjTRSXGTF3IoSL1eJBjHm8Enxb1mHWMX7intRTktN-OCdrnmwgaWNifltpS~5UZt~akENkWBEG4nK6~GBi2J5UYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232656&ar=420691&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LD measures; <em>r</em><sup>2</sup> values against physical distance (Mb) between all pairs of SNPs located on the same chromosome.</p></div></div><div content-id="DST033F2" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/m_dst03302.png?Expires=2147483647&Signature=oOD0vRraRYDsj1tayD5Wc-FirRORt3qpcspR48s7CXNBa9~dqEVBOrjgay689juF7RVqAGL1WGDSBD-h0LrbNEaBW01Tai9NVDQ1QeBHwymB~wbro01Z4MRjUNX8JmDawfoKV1d4shQ3dZ853AvIX38G2pWO12Dq6dfRxOdKPwUNvaVgnkFoSiyzf~Z-uHbip5WAgTIeRs6tYGCeVXRFEpdeeDvZX9kAKZ0GB6PzJqvWW-ga8-lBhawKF4EhFIgC8qoHMhPH-y-Ec5sYkj5ioXLP3LPxetAz-lhU3gpIpOLfEKC6m2ZHI13o2HPZWJpE0n0-XShzJjuC5bzy~rgsOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LD measures; r2 values against physical distance (Mb) between all pairs of SNPs located on the same chromosome." path-from-xml="dst03302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232656/dst03302.tiff" path-from-xml="dst03302.tiff" target="_blank">View large</a><a section="6232656" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst03302.png?Expires=2147483647&Signature=c-gtHWjIAX1xhijQAP7e6ST32pTtNlScZKHGuo9jDgK~Ez87I66nwNjGHO8~sRpN2GGUYH4mbZr7yYvpsvWP29WHg85KjJebCszM6mB2xq5K7OqatMRvDMcchKiTMiSGBMiyChmbwfiDg2h5CWdeR~VyptGZeLUwsvaErzaJTLpzKJK~VCMTlngxtU36I~FAL7VPevA4whF4ZDIq-~uFkTcDVTARsV2M8JVBBNGdDhcbB8I0NwLAeNifZgaKbdRjTRSXGTF3IoSL1eJBjHm8Enxb1mHWMX7intRTktN-OCdrnmwgaWNifltpS~5UZt~akENkWBEG4nK6~GBi2J5UYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232656&ar=420691&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LD measures; <em>r</em><sup>2</sup> values against physical distance (Mb) between all pairs of SNPs located on the same chromosome.</p></div></div><p>A total of 437 haplotypes were identified in the 123 LD blocks. An LD block had an average of 3.6 haplotypes consisting of an average of 3.7 SNPs (data now shown). Subsequently, 308 tag SNPs, the minimum SNP subset required for distinguishing haplotypes, were selected from the 458 SNPs located in the 123 LD blocks (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p>                            <h3 scrollto-destination=6232658 id="6232658" class="section-title" data-legacy-id=s3f>3.6. GWAS for agronomical traits in genetic resources</h3>
<p>GWAS identified a total of nine SNP loci that were significantly associated with eight morphological traits recorded in the NIVTS and NIAS Genebanks (Fig.Â <a reveal-id="DST033F3" data-open="DST033F3" class="link link-reveal link-table xref-fig">3</a>, TableÂ <a reveal-id="DST033TB3" data-open="DST033TB3" class="link link-reveal link-table xref-fig">3</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). The eight traits were phenotyped by actual measured numeric data, qualitative data, and ranked data, and comprised inflorescence branching (nine ranks), plant habit determinate (indeterminante or determinate), plant height (cm), number of leaves between inflorescences (number of leaves), fruit size (10 ranks), locule number (five ranks), green shoulder on immature fruit (10 ranks), and the colour of the fruit epidermis (colorless or yellow), of which the numbers of scored lines were 476, 478, 457, 111, 479, 474, 452, and 137, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dst033/-/DC1" target="_blank">Fig. S7</a>). </p>                        <a id="6232660" scrollto-destination="6232660"></a>
<div content-id="DST033TB3" class="table-modal"><div class="table-wrap"><div id="DST033TB3" class="table-wrap-title" data-id="DST033TB3"><div class="label -label">TableÂ 3.</div><div class="caption"><p>Effects of associated SNPs on the traits</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TraitÂ </th><th>Associating SNPÂ </th><th>ChromosomeÂ </th><th>PositionÂ </th><th>-Log <em>P</em><sup>a</sup>Â </th><th>Additive effect<sup>b</sup>Â </th><th>Dominant effect<sup>b</sup>Â </th><th>Candidate geneÂ </th></tr></thead><tbody><tr><td rowspan="2">Inflorescence branchingÂ </td><td>SL2.40ch02_41751976YÂ </td><td>SL2.40ch02Â </td><td>41 751 976Â </td><td>4.4*Â </td><td>0.2Â </td><td>0.2Â </td><td><em>S</em> and <em>AN</em>Â </td></tr><tr><td>solcap_snp_sl_39457Â </td><td>SL2.40ch09Â </td><td>4 904 111Â </td><td>5.4**Â </td><td>âˆ’0.3Â </td><td>âˆ’0.4Â </td><td>Â </td></tr><tr><td rowspan="2">No. of leaves between inflorescencesÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>5.4**Â </td><td>âˆ’0.4Â </td><td>0.4Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>7.4</strong>**Â </td><td>âˆ’<strong>0.7</strong>Â </td><td><strong>0.4</strong>Â </td><td>Â </td></tr><tr><td rowspan="2">Plant habit determinateÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>26.2**Â </td><td>0.2Â </td><td>âˆ’0.3Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>28.8</strong>**Â </td><td><strong>0.3</strong>Â </td><td>âˆ’<strong>0.4</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Plant heightÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>11.2**Â </td><td>âˆ’7.7Â </td><td>21.1Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>11.6</strong>**Â </td><td>âˆ’<strong>13.5</strong>Â </td><td><strong>25.6</strong>Â </td><td>Â </td></tr><tr><td>solcap_snp_sl_16654Â </td><td>SL2.40ch09Â </td><td>2 135 101Â </td><td>7.5**Â </td><td>âˆ’1.0Â </td><td>31.6Â </td><td>Â </td></tr><tr><td rowspan="2">Fruit sizeÂ </td><td>SL1_00sc6004_2094360_solcap_snp_sl_44897Â </td><td>SL2.40ch11Â </td><td>52 280 215Â </td><td>4.6*Â </td><td>âˆ’10.9Â </td><td>9.1Â </td><td><em>FAS</em>Â </td></tr><tr><td><strong><em>FAS</em></strong>Â </td><td><strong>SL2.40ch11</strong>Â </td><td><strong>52</strong><strong>252</strong><strong>771</strong>Â </td><td><strong>8.4</strong>**Â </td><td>âˆ’<strong>33.3</strong>Â </td><td><strong>12.2</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Locule numberÂ </td><td>SL2.40ch02_41172086RÂ </td><td>SL2.40ch02Â </td><td>41 172 086Â </td><td>4.9*Â </td><td>âˆ’0.4Â </td><td>âˆ’0.6Â </td><td><em>LC</em> and <em>OVATE</em>Â </td></tr><tr><td>SL1_00sc6004_2094360_solcap_snp_sl_44897Â </td><td>SL2.40ch11Â </td><td>52 280 215Â </td><td>7.6**Â </td><td>âˆ’0.5Â </td><td>âˆ’0.2Â </td><td><em>FAS</em>Â </td></tr><tr><td><strong><em>FAS</em></strong>Â </td><td><strong>SL2.40ch11</strong>Â </td><td><strong>52</strong><strong>252</strong><strong>771</strong>Â </td><td><strong>10.1</strong>**Â </td><td>âˆ’<strong>1.2</strong>Â </td><td><strong>0</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Green shoulderÂ </td><td>SL2.40ch01_89266983YÂ </td><td>SL2.40ch01Â </td><td>89 266 983Â </td><td>4.3*Â </td><td>âˆ’0.3Â </td><td>0.9Â </td><td>Â </td></tr><tr><td>SL2.40ch10_1539862RÂ </td><td>SL2.40ch10Â </td><td>1 539 862Â </td><td>5.5**Â </td><td>âˆ’0.4Â </td><td>1.2Â </td><td><em>U</em>Â </td></tr><tr><td><strong><em>U</em></strong>Â </td><td><strong>SL2.40ch10</strong>Â </td><td><strong>2</strong><strong>292</strong><strong>260</strong>Â </td><td><strong>20.9</strong>**Â </td><td>âˆ’<strong>1.5</strong>Â </td><td><strong>1</strong>Â </td><td>Â </td></tr><tr><td>Colour of fruit epidermisÂ </td><td>SL2.40ch01_71279371YÂ </td><td>SL2.40ch01Â </td><td>71 279 371Â </td><td>9.0**Â </td><td>0.2Â </td><td>Not detectedÂ </td><td><em>Y</em>Â </td></tr></tbody></table></div><div content-id="DST033TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>TraitÂ </th><th>Associating SNPÂ </th><th>ChromosomeÂ </th><th>PositionÂ </th><th>-Log <em>P</em><sup>a</sup>Â </th><th>Additive effect<sup>b</sup>Â </th><th>Dominant effect<sup>b</sup>Â </th><th>Candidate geneÂ </th></tr></thead><tbody><tr><td rowspan="2">Inflorescence branchingÂ </td><td>SL2.40ch02_41751976YÂ </td><td>SL2.40ch02Â </td><td>41 751 976Â </td><td>4.4*Â </td><td>0.2Â </td><td>0.2Â </td><td><em>S</em> and <em>AN</em>Â </td></tr><tr><td>solcap_snp_sl_39457Â </td><td>SL2.40ch09Â </td><td>4 904 111Â </td><td>5.4**Â </td><td>âˆ’0.3Â </td><td>âˆ’0.4Â </td><td>Â </td></tr><tr><td rowspan="2">No. of leaves between inflorescencesÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>5.4**Â </td><td>âˆ’0.4Â </td><td>0.4Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>7.4</strong>**Â </td><td>âˆ’<strong>0.7</strong>Â </td><td><strong>0.4</strong>Â </td><td>Â </td></tr><tr><td rowspan="2">Plant habit determinateÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>26.2**Â </td><td>0.2Â </td><td>âˆ’0.3Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>28.8</strong>**Â </td><td><strong>0.3</strong>Â </td><td>âˆ’<strong>0.4</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Plant heightÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>11.2**Â </td><td>âˆ’7.7Â </td><td>21.1Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>11.6</strong>**Â </td><td>âˆ’<strong>13.5</strong>Â </td><td><strong>25.6</strong>Â </td><td>Â </td></tr><tr><td>solcap_snp_sl_16654Â </td><td>SL2.40ch09Â </td><td>2 135 101Â </td><td>7.5**Â </td><td>âˆ’1.0Â </td><td>31.6Â </td><td>Â </td></tr><tr><td rowspan="2">Fruit sizeÂ </td><td>SL1_00sc6004_2094360_solcap_snp_sl_44897Â </td><td>SL2.40ch11Â </td><td>52 280 215Â </td><td>4.6*Â </td><td>âˆ’10.9Â </td><td>9.1Â </td><td><em>FAS</em>Â </td></tr><tr><td><strong><em>FAS</em></strong>Â </td><td><strong>SL2.40ch11</strong>Â </td><td><strong>52</strong><strong>252</strong><strong>771</strong>Â </td><td><strong>8.4</strong>**Â </td><td>âˆ’<strong>33.3</strong>Â </td><td><strong>12.2</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Locule numberÂ </td><td>SL2.40ch02_41172086RÂ </td><td>SL2.40ch02Â </td><td>41 172 086Â </td><td>4.9*Â </td><td>âˆ’0.4Â </td><td>âˆ’0.6Â </td><td><em>LC</em> and <em>OVATE</em>Â </td></tr><tr><td>SL1_00sc6004_2094360_solcap_snp_sl_44897Â </td><td>SL2.40ch11Â </td><td>52 280 215Â </td><td>7.6**Â </td><td>âˆ’0.5Â </td><td>âˆ’0.2Â </td><td><em>FAS</em>Â </td></tr><tr><td><strong><em>FAS</em></strong>Â </td><td><strong>SL2.40ch11</strong>Â </td><td><strong>52</strong><strong>252</strong><strong>771</strong>Â </td><td><strong>10.1</strong>**Â </td><td>âˆ’<strong>1.2</strong>Â </td><td><strong>0</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Green shoulderÂ </td><td>SL2.40ch01_89266983YÂ </td><td>SL2.40ch01Â </td><td>89 266 983Â </td><td>4.3*Â </td><td>âˆ’0.3Â </td><td>0.9Â </td><td>Â </td></tr><tr><td>SL2.40ch10_1539862RÂ </td><td>SL2.40ch10Â </td><td>1 539 862Â </td><td>5.5**Â </td><td>âˆ’0.4Â </td><td>1.2Â </td><td><em>U</em>Â </td></tr><tr><td><strong><em>U</em></strong>Â </td><td><strong>SL2.40ch10</strong>Â </td><td><strong>2</strong><strong>292</strong><strong>260</strong>Â </td><td><strong>20.9</strong>**Â </td><td>âˆ’<strong>1.5</strong>Â </td><td><strong>1</strong>Â </td><td>Â </td></tr><tr><td>Colour of fruit epidermisÂ </td><td>SL2.40ch01_71279371YÂ </td><td>SL2.40ch01Â </td><td>71 279 371Â </td><td>9.0**Â </td><td>0.2Â </td><td>Not detectedÂ </td><td><em>Y</em>Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Genes associating with the traits in the replicated GWAS are shown by bold.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>** and * indicate the significance level of 1 and 5%, respectively.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Effect of â€˜Heinz 1706â€™ allele.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6232660" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST033TB3" class="table-wrap-title" data-id="DST033TB3"><div class="label -label">TableÂ 3.</div><div class="caption"><p>Effects of associated SNPs on the traits</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TraitÂ </th><th>Associating SNPÂ </th><th>ChromosomeÂ </th><th>PositionÂ </th><th>-Log <em>P</em><sup>a</sup>Â </th><th>Additive effect<sup>b</sup>Â </th><th>Dominant effect<sup>b</sup>Â </th><th>Candidate geneÂ </th></tr></thead><tbody><tr><td rowspan="2">Inflorescence branchingÂ </td><td>SL2.40ch02_41751976YÂ </td><td>SL2.40ch02Â </td><td>41 751 976Â </td><td>4.4*Â </td><td>0.2Â </td><td>0.2Â </td><td><em>S</em> and <em>AN</em>Â </td></tr><tr><td>solcap_snp_sl_39457Â </td><td>SL2.40ch09Â </td><td>4 904 111Â </td><td>5.4**Â </td><td>âˆ’0.3Â </td><td>âˆ’0.4Â </td><td>Â </td></tr><tr><td rowspan="2">No. of leaves between inflorescencesÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>5.4**Â </td><td>âˆ’0.4Â </td><td>0.4Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>7.4</strong>**Â </td><td>âˆ’<strong>0.7</strong>Â </td><td><strong>0.4</strong>Â </td><td>Â </td></tr><tr><td rowspan="2">Plant habit determinateÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>26.2**Â </td><td>0.2Â </td><td>âˆ’0.3Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>28.8</strong>**Â </td><td><strong>0.3</strong>Â </td><td>âˆ’<strong>0.4</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Plant heightÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>11.2**Â </td><td>âˆ’7.7Â </td><td>21.1Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>11.6</strong>**Â </td><td>âˆ’<strong>13.5</strong>Â </td><td><strong>25.6</strong>Â </td><td>Â </td></tr><tr><td>solcap_snp_sl_16654Â </td><td>SL2.40ch09Â </td><td>2 135 101Â </td><td>7.5**Â </td><td>âˆ’1.0Â </td><td>31.6Â </td><td>Â </td></tr><tr><td rowspan="2">Fruit sizeÂ </td><td>SL1_00sc6004_2094360_solcap_snp_sl_44897Â </td><td>SL2.40ch11Â </td><td>52 280 215Â </td><td>4.6*Â </td><td>âˆ’10.9Â </td><td>9.1Â </td><td><em>FAS</em>Â </td></tr><tr><td><strong><em>FAS</em></strong>Â </td><td><strong>SL2.40ch11</strong>Â </td><td><strong>52</strong><strong>252</strong><strong>771</strong>Â </td><td><strong>8.4</strong>**Â </td><td>âˆ’<strong>33.3</strong>Â </td><td><strong>12.2</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Locule numberÂ </td><td>SL2.40ch02_41172086RÂ </td><td>SL2.40ch02Â </td><td>41 172 086Â </td><td>4.9*Â </td><td>âˆ’0.4Â </td><td>âˆ’0.6Â </td><td><em>LC</em> and <em>OVATE</em>Â </td></tr><tr><td>SL1_00sc6004_2094360_solcap_snp_sl_44897Â </td><td>SL2.40ch11Â </td><td>52 280 215Â </td><td>7.6**Â </td><td>âˆ’0.5Â </td><td>âˆ’0.2Â </td><td><em>FAS</em>Â </td></tr><tr><td><strong><em>FAS</em></strong>Â </td><td><strong>SL2.40ch11</strong>Â </td><td><strong>52</strong><strong>252</strong><strong>771</strong>Â </td><td><strong>10.1</strong>**Â </td><td>âˆ’<strong>1.2</strong>Â </td><td><strong>0</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Green shoulderÂ </td><td>SL2.40ch01_89266983YÂ </td><td>SL2.40ch01Â </td><td>89 266 983Â </td><td>4.3*Â </td><td>âˆ’0.3Â </td><td>0.9Â </td><td>Â </td></tr><tr><td>SL2.40ch10_1539862RÂ </td><td>SL2.40ch10Â </td><td>1 539 862Â </td><td>5.5**Â </td><td>âˆ’0.4Â </td><td>1.2Â </td><td><em>U</em>Â </td></tr><tr><td><strong><em>U</em></strong>Â </td><td><strong>SL2.40ch10</strong>Â </td><td><strong>2</strong><strong>292</strong><strong>260</strong>Â </td><td><strong>20.9</strong>**Â </td><td>âˆ’<strong>1.5</strong>Â </td><td><strong>1</strong>Â </td><td>Â </td></tr><tr><td>Colour of fruit epidermisÂ </td><td>SL2.40ch01_71279371YÂ </td><td>SL2.40ch01Â </td><td>71 279 371Â </td><td>9.0**Â </td><td>0.2Â </td><td>Not detectedÂ </td><td><em>Y</em>Â </td></tr></tbody></table></div><div content-id="DST033TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>TraitÂ </th><th>Associating SNPÂ </th><th>ChromosomeÂ </th><th>PositionÂ </th><th>-Log <em>P</em><sup>a</sup>Â </th><th>Additive effect<sup>b</sup>Â </th><th>Dominant effect<sup>b</sup>Â </th><th>Candidate geneÂ </th></tr></thead><tbody><tr><td rowspan="2">Inflorescence branchingÂ </td><td>SL2.40ch02_41751976YÂ </td><td>SL2.40ch02Â </td><td>41 751 976Â </td><td>4.4*Â </td><td>0.2Â </td><td>0.2Â </td><td><em>S</em> and <em>AN</em>Â </td></tr><tr><td>solcap_snp_sl_39457Â </td><td>SL2.40ch09Â </td><td>4 904 111Â </td><td>5.4**Â </td><td>âˆ’0.3Â </td><td>âˆ’0.4Â </td><td>Â </td></tr><tr><td rowspan="2">No. of leaves between inflorescencesÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>5.4**Â </td><td>âˆ’0.4Â </td><td>0.4Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>7.4</strong>**Â </td><td>âˆ’<strong>0.7</strong>Â </td><td><strong>0.4</strong>Â </td><td>Â </td></tr><tr><td rowspan="2">Plant habit determinateÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>26.2**Â </td><td>0.2Â </td><td>âˆ’0.3Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>28.8</strong>**Â </td><td><strong>0.3</strong>Â </td><td>âˆ’<strong>0.4</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Plant heightÂ </td><td>SL2.40ch06_42601581WÂ </td><td>SL2.40ch06Â </td><td>42 601 581Â </td><td>11.2**Â </td><td>âˆ’7.7Â </td><td>21.1Â </td><td><em>SP</em>Â </td></tr><tr><td><strong><em>SP</em></strong>Â </td><td><strong>SL2.40ch06</strong>Â </td><td><strong>42</strong><strong>362</strong><strong>163</strong>Â </td><td><strong>11.6</strong>**Â </td><td>âˆ’<strong>13.5</strong>Â </td><td><strong>25.6</strong>Â </td><td>Â </td></tr><tr><td>solcap_snp_sl_16654Â </td><td>SL2.40ch09Â </td><td>2 135 101Â </td><td>7.5**Â </td><td>âˆ’1.0Â </td><td>31.6Â </td><td>Â </td></tr><tr><td rowspan="2">Fruit sizeÂ </td><td>SL1_00sc6004_2094360_solcap_snp_sl_44897Â </td><td>SL2.40ch11Â </td><td>52 280 215Â </td><td>4.6*Â </td><td>âˆ’10.9Â </td><td>9.1Â </td><td><em>FAS</em>Â </td></tr><tr><td><strong><em>FAS</em></strong>Â </td><td><strong>SL2.40ch11</strong>Â </td><td><strong>52</strong><strong>252</strong><strong>771</strong>Â </td><td><strong>8.4</strong>**Â </td><td>âˆ’<strong>33.3</strong>Â </td><td><strong>12.2</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Locule numberÂ </td><td>SL2.40ch02_41172086RÂ </td><td>SL2.40ch02Â </td><td>41 172 086Â </td><td>4.9*Â </td><td>âˆ’0.4Â </td><td>âˆ’0.6Â </td><td><em>LC</em> and <em>OVATE</em>Â </td></tr><tr><td>SL1_00sc6004_2094360_solcap_snp_sl_44897Â </td><td>SL2.40ch11Â </td><td>52 280 215Â </td><td>7.6**Â </td><td>âˆ’0.5Â </td><td>âˆ’0.2Â </td><td><em>FAS</em>Â </td></tr><tr><td><strong><em>FAS</em></strong>Â </td><td><strong>SL2.40ch11</strong>Â </td><td><strong>52</strong><strong>252</strong><strong>771</strong>Â </td><td><strong>10.1</strong>**Â </td><td>âˆ’<strong>1.2</strong>Â </td><td><strong>0</strong>Â </td><td>Â </td></tr><tr><td rowspan="3">Green shoulderÂ </td><td>SL2.40ch01_89266983YÂ </td><td>SL2.40ch01Â </td><td>89 266 983Â </td><td>4.3*Â </td><td>âˆ’0.3Â </td><td>0.9Â </td><td>Â </td></tr><tr><td>SL2.40ch10_1539862RÂ </td><td>SL2.40ch10Â </td><td>1 539 862Â </td><td>5.5**Â </td><td>âˆ’0.4Â </td><td>1.2Â </td><td><em>U</em>Â </td></tr><tr><td><strong><em>U</em></strong>Â </td><td><strong>SL2.40ch10</strong>Â </td><td><strong>2</strong><strong>292</strong><strong>260</strong>Â </td><td><strong>20.9</strong>**Â </td><td>âˆ’<strong>1.5</strong>Â </td><td><strong>1</strong>Â </td><td>Â </td></tr><tr><td>Colour of fruit epidermisÂ </td><td>SL2.40ch01_71279371YÂ </td><td>SL2.40ch01Â </td><td>71 279 371Â </td><td>9.0**Â </td><td>0.2Â </td><td>Not detectedÂ </td><td><em>Y</em>Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Genes associating with the traits in the replicated GWAS are shown by bold.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>** and * indicate the significance level of 1 and 5%, respectively.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Effect of â€˜Heinz 1706â€™ allele.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6232660" target="_blank">View Large</a></div></div>                        <a id="6232661" scrollto-destination="6232661"></a>
<div id="DST033F3" data-id="DST033F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/m_dst03303.png?Expires=2147483647&Signature=LpgD-~RbBsP2kCl9c-1JDbU0KZNxpoXSGNc6CLD2OORNJDqg~gDPtHhmp6wRZFgf7v7sPFafZiGtYjrMY5R53Bp-ZLwO7kT69GK1OMIJJyJvsTkN4mIuzwAmETi5u4SARc7Af~yO2cm~eHTkq~oCv8t8Z2BogS1PDYhqtTy9OGn-NCyQs3b5~HOyL-ucCL4z5moXJ9X6dV8a35odpwL7D1asc6rWK4SooD8~-n-PJuPIJ0sRE0yID43tpRaMDOfOoBeojQrxvj1Ox8TmThB3xnBs-j4lkUn2mcNNnFB4uVhtxqAlLIgOMVpBsrVVBVVMCl3sR7JW-1oJv0X03eSaoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNPs associated with inflorescence branching identified by GWAS. Distribution of SNPs associated with inflorescence branching. SNPs that associated significantly (-logP of 4.36 at a significant level of 5%) are indicated by arrows." path-from-xml="dst03303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232661/dst03303.tiff" path-from-xml="dst03303.tiff" target="_blank">View large</a><a section="6232661" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst03303.png?Expires=2147483647&Signature=eunIFPTTP-EJ7QA70bKaw0IbAU9PyRSKAaQ83QWhW6f9eldo8ZoBJRu9txq9KLs2G04UeCZ-cL5LVQPcfpvcc8cAKIg74upO6hum2svlpsqOnAcmcUrVjBiqZhAETJ9h9pQQo1zIJjBaHq3TKXEn64K0gXdJ6lwVMGPlPPL3PvOe89eEmPyHoQWbj5C~iICSPzGzqvrgmjuidsJcZb5pkrfVtDPX8QmbTsnINS2cU8mxKmUu8fJuPl~pW1RegvYJHcZ1ciOZOThBCMcIpQZoDNUenrxxtMmK-K2bl4NRl3wf39JYpCXEqIlRVBxyEYz7hfAYQAgBozdXa4ix6o0-7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232661&ar=420691&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNPs associated with inflorescence branching identified by GWAS. Distribution of SNPs associated with inflorescence branching. SNPs that associated significantly (-log<em>P</em> of 4.36 at a significant level of 5%) are indicated by arrows.</p></div></div><div content-id="DST033F3" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/m_dst03303.png?Expires=2147483647&Signature=LpgD-~RbBsP2kCl9c-1JDbU0KZNxpoXSGNc6CLD2OORNJDqg~gDPtHhmp6wRZFgf7v7sPFafZiGtYjrMY5R53Bp-ZLwO7kT69GK1OMIJJyJvsTkN4mIuzwAmETi5u4SARc7Af~yO2cm~eHTkq~oCv8t8Z2BogS1PDYhqtTy9OGn-NCyQs3b5~HOyL-ucCL4z5moXJ9X6dV8a35odpwL7D1asc6rWK4SooD8~-n-PJuPIJ0sRE0yID43tpRaMDOfOoBeojQrxvj1Ox8TmThB3xnBs-j4lkUn2mcNNnFB4uVhtxqAlLIgOMVpBsrVVBVVMCl3sR7JW-1oJv0X03eSaoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNPs associated with inflorescence branching identified by GWAS. Distribution of SNPs associated with inflorescence branching. SNPs that associated significantly (-logP of 4.36 at a significant level of 5%) are indicated by arrows." path-from-xml="dst03303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6232661/dst03303.tiff" path-from-xml="dst03303.tiff" target="_blank">View large</a><a section="6232661" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst03303.png?Expires=2147483647&Signature=eunIFPTTP-EJ7QA70bKaw0IbAU9PyRSKAaQ83QWhW6f9eldo8ZoBJRu9txq9KLs2G04UeCZ-cL5LVQPcfpvcc8cAKIg74upO6hum2svlpsqOnAcmcUrVjBiqZhAETJ9h9pQQo1zIJjBaHq3TKXEn64K0gXdJ6lwVMGPlPPL3PvOe89eEmPyHoQWbj5C~iICSPzGzqvrgmjuidsJcZb5pkrfVtDPX8QmbTsnINS2cU8mxKmUu8fJuPl~pW1RegvYJHcZ1ciOZOThBCMcIpQZoDNUenrxxtMmK-K2bl4NRl3wf39JYpCXEqIlRVBxyEYz7hfAYQAgBozdXa4ix6o0-7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6232661&ar=420691&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNPs associated with inflorescence branching identified by GWAS. Distribution of SNPs associated with inflorescence branching. SNPs that associated significantly (-log<em>P</em> of 4.36 at a significant level of 5%) are indicated by arrows.</p></div></div><p>Among the eight traits, inflorescence branching was associated with two SNP loci, SL2.40ch02_41751976Y and solcap_snp_sl_39457 (Fig.Â <a reveal-id="DST033F3" data-open="DST033F3" class="link link-reveal link-table xref-fig">3</a> and TableÂ <a reveal-id="DST033TB3" data-open="DST033TB3" class="link link-reveal link-table xref-fig">3</a>). The SNP SL2.40ch02_41751976Y not belonging to any LD block was located at a distance of 4.8 and 1.6 Mb from the previously identified <em>S</em> (Solyc02g077390) and <em>AN</em> (Solyc02g081670) genes involved in compound inflorescence,<sup><a reveal-id="DST033C34" data-open="DST033C34" class="link link-ref link-reveal xref-bibr">34</a></sup> respectively. The other seven morphological traits were significantly associated with seven SNP loci (TableÂ <a reveal-id="DST033TB3" data-open="DST033TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). Of the seven SNPs not belonging to any LD block, five were located near previously identified genes responsible for the targeted traits. These were SL2.40ch06_42601581W located at 240 kb from <em>SP</em> (Solyc06g074350),<sup><a reveal-id="DST033C35" data-open="DST033C35" class="link link-ref link-reveal xref-bibr">35</a></sup> which is associated with plant habit determinate, plant height, and the number of leaves between inflorescences; SL1_00sc6004_2094360_solcap_snp_sl_44897, located at 31 kb from <em>FAS</em> (Solyc11g071810),<sup><a reveal-id="DST033C36" data-open="DST033C36" class="link link-ref link-reveal xref-bibr">36</a></sup> which is associated with fruit size; SL1_00sc6004_2094360_solcap_snp_sl_44897, located at 31 kb from <em>FAS</em> (Solyc11g071810);<sup><a reveal-id="DST033C36" data-open="DST033C36" class="link link-ref link-reveal xref-bibr">36</a></sup> SL2.40ch02_41172086R, located at 594 kb from <em>LC</em> (Solyc02g083940 and/or Solyc02g083950),<sup><a reveal-id="DST033C37" data-open="DST033C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and 1.8 Mb from <em>OVATE</em> (Solyc02g085500),<sup><a reveal-id="DST033C38" data-open="DST033C38" class="link link-ref link-reveal xref-bibr">38</a></sup> which are associated with locule number; SL2.40ch10_1539862R, located at 753 kb from U (Solyc10g008160),<sup><a reveal-id="DST033C39" data-open="DST033C39" class="link link-ref link-reveal xref-bibr">39</a></sup> which is associated with green shoulder on immature fruit; and SL2.40ch01_71279371Y, located at 24 kb from <em>Y</em> (Solyc01g079620),<sup><a reveal-id="DST033C40" data-open="DST033C40" class="link link-ref link-reveal xref-bibr">40</a></sup> which is associated with colour of the fruit epidermis. To investigate association between the genes conferring the traits, polymorphic analysis of <em>SP</em>, <em>FAS</em>, <em>LC</em>, <em>OVATE</em>, and <em>U</em> was performed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). The replicated GWAS including the five loci showed that <em>SP</em>, <em>U</em>, and <em>FAS</em> were strongly associated with these traits (TableÂ <a reveal-id="DST033TB3" data-open="DST033TB3" class="link link-reveal link-table xref-fig">3</a>).</p>                            <h2 scrollto-destination=6232663 id="6232663" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The re-sequencing analysis presented here identified a large number of SNP candidates in the cultivated tomato, <em>S. lycopersicum</em>, in which DNA polymorphisms have been difficult to detect.<sup><a reveal-id="DST033C4" data-open="DST033C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST033C23" data-open="DST033C23" class="link link-ref link-reveal xref-bibr">23</a></sup> This has been attributed to its narrow genetic diversity, which was caused by the genetic bottlenecks that occurred during its domestication, cultivation, and breeding.<sup><a reveal-id="DST033C41" data-open="DST033C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The intraspecies SNP density of 0.19% was approximately three times lower than that of 0.6% between <em>S. lycopersicum</em> and <em>S. pimpinellifolium</em>.<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The distribution of the SNPs on the reference sequence of H1706 was not evenly spaced over the genome as reported by Asamizu <em>et al.</em><sup><a reveal-id="DST033C42" data-open="DST033C42" class="link link-ref link-reveal xref-bibr">42</a></sup> (Fig.Â <a reveal-id="DST033F1" data-open="DST033F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). In the H1706 genome, large introgressions are observed in Chr04, 09, 11, and 12, which has implications for the introduction of disease resistance loci into H1706 from <em>S. pimpinellifolium</em>.<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The biased SNP density observed in this study also suggests the presence of introgressions of genome segments from wild relatives in tomato breeding processes for disease resistance.<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p><p>SNPs are abundant sequence alterations that can affect gene function. Among the seven inbred tomato lines (including H1706), 558 nonsense, and 22 091 missense, and 17 410 synonymous SNPs were found in 508 (1.5%), 9285 (26.7%), and 7825 (22.5%) of 34 727 predicted genes, respectively (TableÂ <a reveal-id="DST033TB2" data-open="DST033TB2" class="link link-reveal link-table xref-fig">2</a>). Between <em>S. lycopersicum</em> and <em>S. pimpinellifolium</em>, 3.5, 36.3, and 37.0% of genes contain nonsense, missense, and synonymous mutations.<sup><a reveal-id="DST033C12" data-open="DST033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The ratio of interspecies nonsense mutations to intraspecies nonsense mutations is 2.3, while the ratios of missense mutations and synonymous variations were 1.4 and 1.6, respectively. This result suggests that the alleles of wild relatives possessing SNPs that critically disrupt gene function, i.e. nonsense SNP, have been negatively selected from the gene pool of wild relatives for the purpose of breeding.</p><p>The tomato accessions used in this study included broad genetic diversities (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S2 and S3</a>). Genome-wide LD analysis based on these accessions revealed that the extension of the LD was dependent on the nature of the chromatin (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Similar observations have been reported for interspecific F<sub>2</sub> mapping populations in tomato, which indicated that chromosome recombination in heterochromatin is strongly suppressed compared with that in euchromatin.<sup><a reveal-id="DST033C33" data-open="DST033C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DST033C43" data-open="DST033C43" class="link link-ref link-reveal xref-bibr">43</a></sup> LD analysis of whole genomes have been previously performed not only for tomato, but also for rice, soybean, and <em>Arabidopsis</em>.<sup><a reveal-id="DST033C7  DST033C10" data-open="DST033C7  DST033C10" class="link link-ref link-reveal xref-bibr">7â€“10</a>,<a reveal-id="DST033C13" data-open="DST033C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST033C44  DST033C46" data-open="DST033C44  DST033C46" class="link link-ref link-reveal xref-bibr">44â€“46</a></sup> However, LDs specific to chromatin have not been investigated. It is expected that chromosome recombination over the genome is not appreciably different between the accessions and biparental mapping populations.</p><p>GWAS revealed SNPs that were associated with agronomically important traits (Fig.Â <a reveal-id="DST033F3" data-open="DST033F3" class="link link-reveal link-table xref-fig">3</a>, TableÂ <a reveal-id="DST033TB3" data-open="DST033TB3" class="link link-reveal link-table xref-fig">3</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>), and three genes (<em>FAS</em>, <em>SP</em>, and <em>U</em>) were found to confer trait variations (TableÂ <a reveal-id="DST033TB3" data-open="DST033TB3" class="link link-reveal link-table xref-fig">3</a>). Although such genes have been previously identified by a map-based cloning strategy, with interspecific populations conferring phenotypic variations between cultivated tomato and its wild relatives, the present results indicated that these genes are responsible for phenotypic variations within cultivated tomato. The identified SNPs could be potent selection markers for marker-assisted selection in breeding. However, no significant SNP association was detected for most of the traits registered in the NIVTS and NIAS Genebanks. Two possibilities can be advanced to explain the lack of a significant association. First, the density of the SNPs was insufficient for GWAS. In this study, while 1248 SNPs were employed in GWAS, LD extension in the gene-rich euchromatin region (58 kb) was too short to be covered by the SNP density employed (1 SNP/213 kb in euchromatin; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). This analysis suggests that &gt;3228 and &gt;41 SNPs in eu- and heterochromatin regions, respectively, would be required to obtain high-resolution results from GWAS. Additionally, most of the traits were scored on 1â€“5 or 1â€“10 scales, rather than by performing actual measurements. Since the scale standards may vary between individual investigators, the accuracy is unlikely to be sufficient for GWAS. One of the reasons for the success in identifying the SNP associations with the eight morphological traits might be that the SNPs possessed large effects on phenotypic variations.</p><p>In this study, we demonstrated that genetic resource accessions can be used for GWAS, i.e. there is no need to establish a specific mapping population via labour-intensive methods for performing crosses and advancing generations. In addition, a core collection would be more effective for GWAS, as it would avoid the labour and cost associated with high-density whole-genome genotyping and replicated phenotyping. In barley, GWAS was used to detect SNP associations with agronomical traits in a worldwide collection.<sup><a reveal-id="DST033C47" data-open="DST033C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The establishment of core collections for tomato, whose contents could be changed depending on the purpose,<sup><a reveal-id="DST033C48" data-open="DST033C48" class="link link-ref link-reveal xref-bibr">48</a></sup> would enable the identification of valuable loci for molecular genetics and breeding.</p><p>In conclusion, the usefulness of GWAS was demonstrated by analysing a large SNP data set obtained from the re-sequencing data. This study represents an important step forward in genomics, genetics, and for the breeding of cultivated tomato.</p>                            <h2 scrollto-destination=6232670 id="6232670" class="section-title" data-legacy-id=s5>5. Availability</h2>
<p>Nucleotide sequence data reported are available in the DDBJ Sequence Read Archive (BioProject PRJDB1397) under the accession numbers DRA001017 (AIC), DRA001018 (FRK), DRA001019 (M82), DRA001020 (PL11), DRA001021 (PON), and DRA001022 (REG). Details of the SNPs and genotypes of the investigated genetic resources are available at the Kazusa Tomato Genomics DataBase (KaTomicsDB: <a class="link link-uri" href="http://www.kazusa.or.jp/tomato">http://www.kazusa.or.jp/tomato</a>).</p>                            <h2 scrollto-destination=6232672 id="6232672" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6232674 id="6232674" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation and the Genomics for Agricultural Innovation Foundation (SGE-1001), Ministry of Agriculture, Forestry, and Fisheries of Japan.</p>                            <h2 scrollto-destination=6232676 id="6232676" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank S. Sasamoto, S. Nakayama, Y. Kishida, M. Kohara, and H. Tsuruoka (Kazusa DNA Research Institute) for their technical assistance, and Drs T. Ariizumi (NBRP), S.M. Tam (TGRC), S.D. Tanksley (Cornell University), and C. Rothan (INRA) for providing plant materials.</p>                            <h2 scrollto-destination=6232678 id="6232678" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST033C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Labate</div> <div class="given-names">J.A.</div></div>,Â  <div class="name"><div class="surname">Grandillo</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Fulton</div> <div class="given-names">T.</div></div>,Â et al.Â </div><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">C.</div></div>.Â </div><div class="article-title">Tomato</div>,Â <div class="source">Genome Mapping and Molecular Breeding in Plants</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">5</div>Â <div class="publisher-loc">New York</div><div class="publisher-name">Springer</div>(pg.Â <div class="fpage">1</div>-<div class="lpage">125</div>)</div></div></div></div><div content-id="DST033C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>.Â </div><div class="article-title">Chromosomal evolution in the plant family Solanaceae</div>,Â <div class="source">BMC Genomics</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">11</div>Â pg.Â <div class="fpage">182</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20evolution%20in%20the%20plant%20family%20Solanaceae&amp;author=F.%20Wu&amp;author=S.D.%20Tanksley&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=182" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-182" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-182" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-182"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20236516" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>.Â </div><div class="article-title">DNA marker applications to molecular genetics and genomics in tomato</div>,Â <div class="source">Breed. Sci.</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">63</div>Â (pg.Â <div class="fpage">21</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20marker%20applications%20to%20molecular%20genetics%20and%20genomics%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=H.%20Hirakawa&amp;publication_year=2013&amp;journal=Breed.%20Sci.&amp;volume=63&amp;pages=21-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.63.21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.63.21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.63.21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23641178" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamilton</div> <div class="given-names">J.P.</div></div>,Â  <div class="name"><div class="surname">Sim</div> <div class="given-names">S.C.</div></div>,Â  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Van Deynze</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>,Â  <div class="name"><div class="surname">Francis</div> <div class="given-names">D.M.</div></div>.Â </div><div class="article-title">Single nucleotide polymorphism discovery in cultivated tomato via sequencing by synthesis</div>,Â <div class="source">Plant Genome</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">5</div>Â (pg.Â <div class="fpage">17</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20discovery%20in%20cultivated%20tomato%20via%20sequencing%20by%20synthesis&amp;author=J.P.%20Hamilton&amp;author=S.C.%20Sim&amp;author=K.%20Stoffel&amp;author=A.%20Van%20Deynze&amp;author=C.R.%20Buell&amp;author=D.M.%20Francis&amp;publication_year=2012&amp;journal=Plant%20Genome&amp;volume=5&amp;pages=17-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2011.12.0033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2011.12.0033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2011.12.0033"> </span></div>Â </div></div></div></div></div><div content-id="DST033C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="collab">The 1000 Genomes Project Consortium</div><div class="article-title">An integrated map of genetic variation from 1092 human genomes</div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">491</div>Â (pg.Â <div class="fpage">56</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11632" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11632" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11632"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23128226" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="collab">The Bovine HapMap Consortium</div><div class="article-title">Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds</div>,Â <div class="source">Science</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">324</div>Â (pg.Â <div class="fpage">528</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1167936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1167936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1167936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19390050" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cao</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Ossowski</div> <div class="given-names">S.</div></div>,Â et al.Â </div><div class="article-title">Whole-genome sequencing of multiple <em>Arabidopsis thaliana</em> populations</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">43</div>Â (pg.Â <div class="fpage">956</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20multiple%20Arabidopsis%20thaliana%20populations&amp;author=J.%20Cao&amp;author=K.%20Schneeberger&amp;author=S.%20Ossowski&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=956-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.911" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.911"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21874002" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gan</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Stegle</div> <div class="given-names">O.</div></div>,Â  <div class="name"><div class="surname">Behr</div> <div class="given-names">J.</div></div>,Â et al.Â </div><div class="article-title">Multiple reference genomes and transcriptomes for <em>Arabidopsis thaliana</em></div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">477</div>Â (pg.Â <div class="fpage">419</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20reference%20genomes%20and%20transcriptomes%20for%20Arabidopsis%20thaliana&amp;author=X.%20Gan&amp;author=O.%20Stegle&amp;author=J.%20Behr&amp;publication_year=2011&amp;journal=Nature&amp;volume=477&amp;pages=419-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21874022" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Kurata</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>,Â et al.Â </div><div class="article-title">A map of rice genome variation reveals the origin of cultivated rice</div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">490</div>Â (pg.Â <div class="fpage">497</div>-<div class="lpage">501</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20map%20of%20rice%20genome%20variation%20reveals%20the%20origin%20of%20cultivated%20rice&amp;author=X.%20Huang&amp;author=N.%20Kurata&amp;author=X.%20Wei&amp;publication_year=2012&amp;journal=Nature&amp;volume=490&amp;pages=497-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11532" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11532" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11532"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23034647" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Ge</div> <div class="given-names">S.</div></div>,Â et al.Â </div><div class="article-title">Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes</div>,Â <div class="source">Nat. Biotechnol.</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">30</div>Â (pg.Â <div class="fpage">105</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%2050%20accessions%20of%20cultivated%20and%20wild%20rice%20yields%20markers%20for%20identifying%20agronomically%20important%20genes&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;author=S.%20Ge&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2050"> </span></div>Â </div></div></div></div></div><div content-id="DST033C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernatzky</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>.Â </div><div class="article-title">Toward a saturated linkage map in tomato based on isozymes and random cDNA sequences</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">1986</div>,Â vol.Â <div class="volume">112</div>Â (pg.Â <div class="fpage">887</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20a%20saturated%20linkage%20map%20in%20tomato%20based%20on%20isozymes%20and%20random%20cDNA%20sequences&amp;author=R.%20Bernatzky&amp;author=S.D.%20Tanksley&amp;publication_year=1986&amp;journal=Genetics&amp;volume=112&amp;pages=887-98" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17246322" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="collab">The Tomato Genome Consortium</div><div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">485</div>Â (pg.Â <div class="fpage">635</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Ohyama</div> <div class="given-names">A.</div></div>,Â et al.Â </div><div class="article-title">Genome-wide SNP genotyping to infer the effects on gene functions in tomato</div>,Â <div class="source">DNA Res</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">20</div>Â (pg.Â <div class="fpage">221</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20genotyping%20to%20infer%20the%20effects%20on%20gene%20functions%20in%20tomato&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=A.%20Ohyama&amp;publication_year=2013&amp;journal=DNA%20Res&amp;volume=20&amp;pages=221-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23482505" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamilton</div> <div class="given-names">J.P.</div></div>,Â  <div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>.Â </div><div class="article-title">Advances in plant genome sequencing</div>,Â <div class="source">Plant J.</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">70</div>Â (pg.Â <div class="fpage">177</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20plant%20genome%20sequencing&amp;author=J.P.%20Hamilton&amp;author=C.R.%20Buell&amp;publication_year=2012&amp;journal=Plant%20J.&amp;volume=70&amp;pages=177-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2012.04894.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2012.04894.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2012.04894.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22449051" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schatz</div> <div class="given-names">M.C.</div></div>,Â  <div class="name"><div class="surname">Witkowski</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">McCombie</div> <div class="given-names">W.R.</div></div>.Â </div><div class="article-title">Current challenges in de novo plant genome sequencing and assembly</div>,Â <div class="source">Genome Biol.</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">13</div>Â pg.Â <div class="fpage">243</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Current%20challenges%20in%20de%20novo%20plant%20genome%20sequencing%20and%20assembly&amp;author=M.C.%20Schatz&amp;author=J.%20Witkowski&amp;author=W.R.%20McCombie&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=243" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-4-243" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-4-243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-4-243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22546054" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,Â  <div class="name"><div class="surname">Ribaut</div> <div class="given-names">J.M.</div></div>,Â  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>,Â  <div class="name"><div class="surname">Tuberosa</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Rafalski</div> <div class="given-names">J.A.</div></div>,Â  <div class="name"><div class="surname">Langridge</div> <div class="given-names">P.</div></div>.Â </div><div class="article-title">Can genomics boost productivity of orphan crops?</div>,Â <div class="source">Nat. Biotechnol.</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">30</div>Â (pg.Â <div class="fpage">1172</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Can%20genomics%20boost%20productivity%20of%20orphan%20crops%3F&amp;author=R.K.%20Varshney&amp;author=J.M.%20Ribaut&amp;author=E.S.%20Buckler&amp;author=R.%20Tuberosa&amp;author=J.A.%20Rafalski&amp;author=P.%20Langridge&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=1172-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2440" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2440" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2440"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23222781" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saito</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Matsunaga</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">T.</div></div>,Â et al.Â </div><div class="article-title">Tomato Chuukanbohon Nou 11â€™, a tomato parental line with a short-internode trait</div>,Â <div class="source">Bull. Natl. Inst. Veg. Tea Sci.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">6</div>Â (pg.Â <div class="fpage">65</div>-<div class="lpage">76</div>)</div></div></div></div><div content-id="DST033C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,Â  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,Â  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>,Â et al.Â </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">1997</div>,Â vol.Â <div class="volume">25</div>Â (pg.Â <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>,Â  <div class="name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>,Â  <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>,Â et al.Â </div><div class="article-title">The COG database: an updated version includes eukaryotes</div>,Â <div class="source">BMC Bioinformatics</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">4</div>Â pg.Â <div class="fpage">41</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thiel</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Kota</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Grosse</div> <div class="given-names">I.</div></div>,Â  <div class="name"><div class="surname">Stein</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>.Â </div><div class="article-title">SNP2CAPS: a SNP and INDEL analysis tool for CAPS marker development</div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">32</div>Â pg.Â <div class="fpage">e5</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP2CAPS%3A%20a%20SNP%20and%20INDEL%20analysis%20tool%20for%20CAPS%20marker%20development&amp;author=T.%20Thiel&amp;author=R.%20Kota&amp;author=I.%20Grosse&amp;author=N.%20Stein&amp;author=A.%20Graner&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=e5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gnh006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgnh006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgnh006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704362" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neff</div> <div class="given-names">M.M.</div></div>,Â  <div class="name"><div class="surname">Turk</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Kalishman</div> <div class="given-names">M.</div></div>.Â </div><div class="article-title">Web-based primer design for single nucleotide polymorphism analysis</div>,Â <div class="source">Trends Genet.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">18</div>Â (pg.Â <div class="fpage">613</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Web-based%20primer%20design%20for%20single%20nucleotide%20polymorphism%20analysis&amp;author=M.M.%20Neff&amp;author=E.%20Turk&amp;author=M.%20Kalishman&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=613-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(02)02820-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(02)02820-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(02)02820-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12446140" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>.Â </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>,Â <div class="source">Methods Mol. Biol.</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">132</div>Â (pg.Â <div class="fpage">365</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&amp;author=S.%20Rozen&amp;author=H.%20Skaletsky&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol.&amp;volume=132&amp;pages=365-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>,Â et al.Â </div><div class="article-title">SNP discovery and linkage map construction in cultivated tomato</div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">381</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20linkage%20map%20construction%20in%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=S.%20Isobe&amp;author=H.%20Hirakawa&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=381-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044984" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">RodrÃ­guez</div> <div class="given-names">G.R.</div></div>,Â  <div class="name"><div class="surname">MuÃ±os</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Anderson</div> <div class="given-names">C.</div></div>,Â et al.Â </div><div class="article-title">Distribution of <em>SUN</em>, <em>OVATE</em>, <em>LC</em>, and <em>FAS</em> in the tomato germplasm and the relationship to fruit shape diversity</div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">156</div>Â (pg.Â <div class="fpage">275</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20SUN%2C%20OVATE%2C%20LC%2C%20and%20FAS%20in%20the%20tomato%20germplasm%20and%20the%20relationship%20to%20fruit%20shape%20diversity&amp;author=G.R.%20Rodr%C3%ADguez&amp;author=S.%20Mu%C3%B1os&amp;author=C.%20Anderson&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=275-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.167577" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.167577" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.167577"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21441384" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Berloo</div> <div class="given-names">R.</div></div>.Â </div><div class="article-title">GGT 2.0: versatile software for visualization and analysis of genetic data</div>,Â <div class="source">J. Hered.</div>Â ,Â <div class="year">2008</div>,Â vol.Â <div class="volume">99</div>Â (pg.Â <div class="fpage">232</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GGT%202.0%3A%20versatile%20software%20for%20visualization%20and%20analysis%20of%20genetic%20data&amp;author=R.%20van%20Berloo&amp;publication_year=2008&amp;journal=J.%20Hered.&amp;volume=99&amp;pages=232-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/esm109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fesm109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fesm109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18222930" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Ishii</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Kim</div> <div class="given-names">C.</div></div>,Â et al.Â </div><div class="article-title">Development of <em>Capsicum</em> EST-SSR markers for species identification and <em>in silico</em> mapping onto the tomato genome sequence</div>,Â <div class="source">Mol. Breed.</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">31</div>Â (pg.Â <div class="fpage">101</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20Capsicum%20EST-SSR%20markers%20for%20species%20identification%20and%20in%20silico%20mapping%20onto%20the%20tomato%20genome%20sequence&amp;author=K.%20Shirasawa&amp;author=K.%20Ishii&amp;author=C.%20Kim&amp;publication_year=2013&amp;journal=Mol.%20Breed.&amp;volume=31&amp;pages=101-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-012-9774-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-012-9774-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-012-9774-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23316112" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>.Â </div><div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>,Â <div class="source">Mol. Biol. Evol.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">28</div>Â (pg.Â <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>,Â  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,Â  <div class="name"><div class="surname">Kroon</div> <div class="given-names">D.E.</div></div>,Â  <div class="name"><div class="surname">Casstevens</div> <div class="given-names">T.M.</div></div>,Â  <div class="name"><div class="surname">Ramdoss</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>.Â </div><div class="article-title">TASSEL: software for association mapping of complex traits in diverse samples</div>,Â <div class="source">Bioinformatics</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">23</div>Â (pg.Â <div class="fpage">2633</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.E.%20Kroon&amp;author=T.M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=E.S.%20Buckler&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2633-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586829" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div>,Â  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div>.Â </div><div class="article-title">Inference of population structure using multilocus genotype data</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">155</div>Â (pg.Â <div class="fpage">945</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=945-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Evanno</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Regnaut</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Goudet</div> <div class="given-names">J.</div></div>.Â </div><div class="article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study</div>,Â <div class="source">Mol. Ecol.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">14</div>Â (pg.Â <div class="fpage">2611</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;publication_year=2005&amp;journal=Mol.%20Ecol.&amp;volume=14&amp;pages=2611-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2005.14.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2005.14.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2005.14.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrett</div> <div class="given-names">J.C.</div></div>,Â  <div class="name"><div class="surname">Fry</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Maller</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Daly</div> <div class="given-names">M.J.</div></div>.Â </div><div class="article-title">Haploview: analysis and visualization of LD and haplotype maps</div>,Â <div class="source">Bioinformatics</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">21</div>Â (pg.Â <div class="fpage">263</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Haploview%3A%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&amp;author=J.C.%20Barrett&amp;author=B.%20Fry&amp;author=J.%20Maller&amp;author=M.J.%20Daly&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=263-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth457" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth457" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth457"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15297300" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gabriel</div> <div class="given-names">S.B.</div></div>,Â  <div class="name"><div class="surname">Schaffner</div> <div class="given-names">S.F.</div></div>,Â  <div class="name"><div class="surname">Nguyen</div> <div class="given-names">H.</div></div>,Â et al.Â </div><div class="article-title">The structure of haplotype blocks in the human genome</div>,Â <div class="source">Science</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">296</div>Â (pg.Â <div class="fpage">2225</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&amp;author=S.B.%20Gabriel&amp;author=S.F.%20Schaffner&amp;author=H.%20Nguyen&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=2225-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1069424" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1069424" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1069424"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12029063" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>,Â et al.Â </div><div class="article-title">An interspecific linkage map of SSR and intronic polymorphism markers in tomato</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">121</div>Â (pg.Â <div class="fpage">731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20interspecific%20linkage%20map%20of%20SSR%20and%20intronic%20polymorphism%20markers%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=E.%20Asamizu&amp;author=H.%20Fukuoka&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1344-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1344-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1344-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20431859" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lippman</div> <div class="given-names">Z.B.</div></div>,Â  <div class="name"><div class="surname">Cohen</div> <div class="given-names">O.</div></div>,Â  <div class="name"><div class="surname">Alvarez</div> <div class="given-names">J.P.</div></div>,Â et al.Â </div><div class="article-title">The making of a compound inflorescence in tomato and related nightshades</div>,Â <div class="source">PLoS Biol.</div>Â ,Â <div class="year">2008</div>,Â vol.Â <div class="volume">6</div>Â pg.Â <div class="fpage">e288</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20making%20of%20a%20compound%20inflorescence%20in%20tomato%20and%20related%20nightshades&amp;author=Z.B.%20Lippman&amp;author=O.%20Cohen&amp;author=J.P.%20Alvarez&amp;publication_year=2008&amp;journal=PLoS%20Biol.&amp;volume=6&amp;pages=e288" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0060288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0060288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0060288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19018664" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pnueli</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Carmel-Goren</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Hareven</div> <div class="given-names">D.</div></div>,Â et al.Â </div><div class="article-title">The <em>SELF-PRUNING</em> gene of tomato regulates vegetative to reproductive switching of sympodial meristems and is the ortholog of <em>CEN</em> and <em>TFL1</em></div>,Â <div class="source">Development</div>Â ,Â <div class="year">1998</div>,Â vol.Â <div class="volume">125</div>Â (pg.Â <div class="fpage">1979</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20SELF-PRUNING%20gene%20of%20tomato%20regulates%20vegetative%20to%20reproductive%20switching%20of%20sympodial%20meristems%20and%20is%20the%20ortholog%20of%20CEN%20and%20TFL1&amp;author=L.%20Pnueli&amp;author=L.%20Carmel-Goren&amp;author=D.%20Hareven&amp;publication_year=1998&amp;journal=Development&amp;volume=125&amp;pages=1979-89" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9570763" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cong</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Barrero</div> <div class="given-names">L.S.</div></div>,Â  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>.Â </div><div class="article-title">Regulatory change in YABBY-like transcription factor led to evolution of extreme fruit size during tomato domestication</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2008</div>,Â vol.Â <div class="volume">40</div>Â (pg.Â <div class="fpage">800</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20change%20in%20YABBY-like%20transcription%20factor%20led%20to%20evolution%20of%20extreme%20fruit%20size%20during%20tomato%20domestication&amp;author=B.%20Cong&amp;author=L.S.%20Barrero&amp;author=S.D.%20Tanksley&amp;publication_year=2008&amp;journal=Nat.%20Genet.&amp;volume=40&amp;pages=800-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.144"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18469814" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Munos</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Ranc</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Botton</div> <div class="given-names">E.</div></div>,Â et al.Â </div><div class="article-title">Increase in tomato locule number is controlled by two single-nucleotide polymorphisms located near WUSCHEL</div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">156</div>Â (pg.Â <div class="fpage">2244</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Increase%20in%20tomato%20locule%20number%20is%20controlled%20by%20two%20single-nucleotide%20polymorphisms%20located%20near%20WUSCHEL&amp;author=S.%20Munos&amp;author=N.%20Ranc&amp;author=E.%20Botton&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=2244-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.173997" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.173997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.173997"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21673133" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Van Eck</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Cong</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>.Â </div><div class="article-title">A new class of regulatory genes underlying the cause of pear-shaped tomato fruit</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">99</div>Â (pg.Â <div class="fpage">13302</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20class%20of%20regulatory%20genes%20underlying%20the%20cause%20of%20pear-shaped%20tomato%20fruit&amp;author=J.%20Liu&amp;author=J.%20Van%20Eck&amp;author=B.%20Cong&amp;author=S.D.%20Tanksley&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=13302-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.162485999" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.162485999" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.162485999"> </span></div>Â </div></div></div></div></div><div content-id="DST033C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Powell</div> <div class="given-names">A.L.</div></div>,Â  <div class="name"><div class="surname">Nguyen</div> <div class="given-names">C.V.</div></div>,Â  <div class="name"><div class="surname">Hill</div> <div class="given-names">T.</div></div>,Â et al.Â </div><div class="article-title"><em>Uniform ripening</em> encodes a <em>Golden 2-like</em> transcription factor regulating tomato fruit chloroplast development</div>,Â <div class="source">Science</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">336</div>Â (pg.Â <div class="fpage">1711</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uniform%20ripening%20encodes%20a%20Golden%202-like%20transcription%20factor%20regulating%20tomato%20fruit%20chloroplast%20development&amp;author=A.L.%20Powell&amp;author=C.V.%20Nguyen&amp;author=T.%20Hill&amp;publication_year=2012&amp;journal=Science&amp;volume=336&amp;pages=1711-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1222218" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1222218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1222218"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22745430" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ballester</div> <div class="given-names">A.R.</div></div>,Â  <div class="name"><div class="surname">Molthoff</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">de Vos</div> <div class="given-names">R.</div></div>,Â et al.Â </div><div class="article-title">Biochemical and molecular analysis of pink tomatoes: deregulated expression of the gene encoding transcription factor SlMYB12 leads to pink tomato fruit color</div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">152</div>Â (pg.Â <div class="fpage">71</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biochemical%20and%20molecular%20analysis%20of%20pink%20tomatoes%3A%20deregulated%20expression%20of%20the%20gene%20encoding%20transcription%20factor%20SlMYB12%20leads%20to%20pink%20tomato%20fruit%20color&amp;author=A.R.%20Ballester&amp;author=J.%20Molthoff&amp;author=R.%20de%20Vos&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=152&amp;pages=71-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.147322" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.147322" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.147322"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19906891" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rick</div> <div class="given-names">C.M.</div></div>,Â  <div class="name"><div class="surname">Kesicki</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Fobes</div> <div class="given-names">J.F.</div></div>,Â  <div class="name"><div class="surname">Holle</div> <div class="given-names">M.</div></div>.Â </div><div class="article-title">Genetic and biosystematic studies on two new sibling species of <em>Lycopersicon</em> from interandean Peru</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">1976</div>,Â vol.Â <div class="volume">47</div>Â (pg.Â <div class="fpage">55</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20biosystematic%20studies%20on%20two%20new%20sibling%20species%20of%20Lycopersicon%20from%20interandean%20Peru&amp;author=C.M.%20Rick&amp;author=E.%20Kesicki&amp;author=J.F.%20Fobes&amp;author=M.%20Holle&amp;publication_year=1976&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=47&amp;pages=55-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00281917" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00281917" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00281917"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24414383" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>,Â et al.Â </div><div class="article-title">Mapping of Micro-Tom BAC-end sequences to the reference tomato genome reveals possible genome rearrangements and polymorphisms</div>,Â <div class="source">Inter. J. Plant Genomics</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">2012</div>Â pg.Â <div class="fpage">437026</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20Micro-Tom%20BAC-end%20sequences%20to%20the%20reference%20tomato%20genome%20reveals%20possible%20genome%20rearrangements%20and%20polymorphisms&amp;author=E.%20Asamizu&amp;author=K.%20Shirasawa&amp;author=H.%20Hirakawa&amp;publication_year=2012&amp;journal=Inter.%20J.%20Plant%20Genomics&amp;volume=2012&amp;pages=437026" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1155/2012/437026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2012%2F437026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1155%2F2012%2F437026"> </span></div>Â </div></div></div></div></div><div content-id="DST033C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sim</div> <div class="given-names">S.C.</div></div>,Â  <div class="name"><div class="surname">Durstewitz</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div>,Â et al.Â </div><div class="article-title">Development of a large SNP genotyping array and generation of high-density genetic maps in tomato</div>,Â <div class="source">PLoS One</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">7</div>Â pg.Â <div class="fpage">e40563</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20large%20SNP%20genotyping%20array%20and%20generation%20of%20high-density%20genetic%20maps%20in%20tomato&amp;author=S.C.%20Sim&amp;author=G.%20Durstewitz&amp;author=J.%20Plieske&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e40563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0040563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0040563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0040563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22802968" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lam</div> <div class="given-names">H.M.</div></div>,Â  <div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,Â et al.Â </div><div class="article-title">Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">42</div>Â (pg.Â <div class="fpage">1053</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%20of%2031%20wild%20and%20cultivated%20soybean%20genomes%20identifies%20patterns%20of%20genetic%20diversity%20and%20selection&amp;author=H.M.%20Lam&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=1053-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.715" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.715" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.715"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076406" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ranc</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Munos</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>,Â et al.Â </div><div class="article-title">Genome-wide association mapping in tomato (<em>Solanum lycopersicum</em>) is possible using genome admixture of <em>Solanum lycopersicum</em> var. <em>cerasiforme</em></div>,Â <div class="source">G3</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">2</div>Â (pg.Â <div class="fpage">853</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20in%20tomato%20(Solanum%20lycopersicum)%20is%20possible%20using%20genome%20admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme&amp;author=N.%20Ranc&amp;author=S.%20Munos&amp;author=J.%20Xu&amp;publication_year=2012&amp;journal=G3&amp;volume=2&amp;pages=853-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.112.002667" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.112.002667" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.112.002667"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22908034" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sim</div> <div class="given-names">S.C.</div></div>,Â  <div class="name"><div class="surname">Van Deynze</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>,Â et al.Â </div><div class="article-title">High-density SNP genotyping of tomato (<em>Solanum lycopersicum</em> L.) reveals patterns of genetic variation due to breeding</div>,Â <div class="source">PLoS One</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">7</div>Â pg.Â <div class="fpage">e45520</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20SNP%20genotyping%20of%20tomato%20(Solanum%20lycopersicum%20L.)%20reveals%20patterns%20of%20genetic%20variation%20due%20to%20breeding&amp;author=S.C.%20Sim&amp;author=A.%20Van%20Deynze&amp;author=K.%20Stoffel&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e45520" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0045520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0045520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0045520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23029069" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pasam</div> <div class="given-names">R.K.</div></div>,Â  <div class="name"><div class="surname">Sharma</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Malosetti</div> <div class="given-names">M.</div></div>,Â et al.Â </div><div class="article-title">Genome-wide association studies for agronomical traits in a world wide spring barley collection</div>,Â <div class="source">BMC Plant Biol.</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">12</div>Â pg.Â <div class="fpage">16</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20for%20agronomical%20traits%20in%20a%20world%20wide%20spring%20barley%20collection&amp;author=R.K.%20Pasam&amp;author=R.%20Sharma&amp;author=M.%20Malosetti&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol.&amp;volume=12&amp;pages=16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22284310" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DST033C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST033C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Odong</div> <div class="given-names">T.L.</div></div>,Â  <div class="name"><div class="surname">Jansen</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">van Eeuwijk</div> <div class="given-names">F.A.</div></div>,Â  <div class="name"><div class="surname">van Hintum</div> <div class="given-names">T.J.</div></div>.Â </div><div class="article-title">Quality of core collections for effective utilisation of genetic resources review, discussion and interpretation</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">126</div>Â (pg.Â <div class="fpage">289</div>-<div class="lpage">305</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quality%20of%20core%20collections%20for%20effective%20utilisation%20of%20genetic%20resources%20review%2C%20discussion%20and%20interpretation&amp;author=T.L.%20Odong&amp;author=J.%20Jansen&amp;author=F.A.%20van%20Eeuwijk&amp;author=T.J.%20van%20Hintum&amp;publication_year=2013&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=126&amp;pages=289-305" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1971-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1971-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1971-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22983567" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=420691&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_table5.xls?Expires=2147483647&amp;Signature=1MlZOWF60IREYcT6X2R2agYjO3P9azxDAYKwcKEPsyOm38mfW0hfS4Urub2gZ6W5r~RL~vNQDRN27K-Iw3rtqDClOT324XytzcnAski2Y-2vdCPK06lS6JRv7WlsBhyHiR7gy7knbzAiw9~rquqPC-Y9j9tzvma934On1U004BEjPe50QKFGPzu8fp8oEEN~Cj6273ztAhIqczSjSUjaxwMwPKLWwMPYNI~LzCwicrrGju5dleWor3gIsK2QTDVoX1bVl3ihb9ugwnvCxwz~pGoh8ZSP5Iar8leF5IJ~MKvQxUos9KTzfSlvvSMnwDAOKd710Od-p8bQCiBToc8Ofw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_table4.xls?Expires=2147483647&amp;Signature=Vys7gwLUd7V5aQdErXF3HjUX1LDag5mpJ7eROvpdNXV9CxBxL-Sjd7RUvbyKhF~3fly8rFHcBpH5n2dhjvvq-Q8ze5kf-LWY53iLRA-mCV7esbszoQTGelYhGR5HX~GMFllpPrArrTnaAIkgmomHUYG7W7gh91fRq3ceIJYZNd5kYvNeOTXhNSf4MoJHaj0w~E8rYi8eoQB2O100S6EoaTp1EPYr-oo197Cu-~ZKfkNw6FIFggVQ2fb-jXRnbhtEwRYa-vxdtYu3-v9NnPkMKqSlmP59sI2k~eOv5xfTYpJXk4nF3Ty1dNq3kE5xrAjzQEN~-LIXHgC1Gj6kLWdLdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_table3.xls?Expires=2147483647&amp;Signature=HpQtORWWz9N~0SsHQlGhihyjRlCh40GF6K4uvJ8mIr2gkkurp~0LZsNj3bmY2xQ8Ii738t4HshOt4eGON3FxFWHYR22tXD1BbxXvvYZBCHzk3KHZIlxdtR1Rgv2kf7oKmtqYhVOlE4Lk3gJ26PZNzSizn~5BX9IbnI1M~M8hrcZBfWmi3AESXM~IVoYr7T3W1VNcd2xmuEsUG7hNRuCOB-VHmYY9IzCqH2gDcmOnDNi30xdKys6HJ85CbEkynbG9FBd8kiqw32FFEVyWg4piw2haRcnmBCuuIxkD27So6eweqgT52kYxbmk9mKR3~SEh1a1L9jO~DKL6yIhVmYj6AQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_table2.xls?Expires=2147483647&amp;Signature=SYcHOZ0cd5kZoq4x6t0E4Xo5~jxB9t9qh26q53k6VK2g7KBx5PEtr-4T0h-Dbpqt5fpyFUL1DnrQwKw9m2ZSiyKqjrXdBrYweE787VzMIfhHHz9VYI9lxYSNnAzU832IyjgDfWwFeb-SBCstms6vu6jXmK5QpQ-AYb-WUVxsSkezJEDDStPz~wFfnQuVcfPagJi-dWQmwC~sdKrtCgaAG~6PWivSWvPK7v2Czyo8EeDn4IIiWragAUXLRLKQxfAUdyONK8ZBCc6ZQZ9JwSF0AfWAnSnflyysxjmVycoGVCFMZiuBGC65qMFmeEbVSi0bWGB9TEVWo50Y0AjgOCrbMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_table1.xls?Expires=2147483647&amp;Signature=1FqvrarVBZnYCaXSdQk3OleCXswkyjEQQt78utUqOrs3mA7rKfY9sVy07r2Pun0e8q2gHJJ6pecPYa-PHc47BCNnXdGTfyKkJHBV9LuLYbegSQDg8UkRxgmehODQTczW4GhVfZAZIcPyQqGbtca4aGurvAEO7dRynpmdUPPHUqJaaZNT8-azxQ4yK55al8BBYJqGjtNP7vaTXvjxRh-vunz1GeNfTxLwDxN7~GWLwy4Ns9iWhuhQ3CmGt~F1fI2r8KFcTAklpGaqF-wTUDX7dkKTWosLvJf7pkAre3opzfCnSC1p8i5W8ogb49YX~hu6ErlMVrMxoArmdOh1Zgy-7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_fig7.jpg?Expires=2147483647&amp;Signature=gAH~o4rLx2-CbOoKCoTVKvl7eE254Bdrcoy2uRhlKb~RgD9G0xt4QD07WRX960ZHJh5ktX7DMN8Lh2xapxfHTvMQLWoSDKB4PnXoxZk~xbl-423DGvTfg3tlPrgXhoqjt4dbNZbwpqtSZJ3vwLxHct6qfe5GOM-JS7j~7bHF3bYyAOcdKlo5qhX559D~2YVWYklUbG2ZrUZmZHaeDA9EvGIaTmc6Sx~50nflFN0awezHmg7ruKKPHPvBeEPKtyJjKrwz3Z2c3UjLGhVSDLPKRBCaXMCCwIZ-fdrg9HmFkxkKAmlR2HJDenGUGVj4yRWlRp8OJ0JxpB4abz8rExATOA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_fig6.jpg?Expires=2147483647&amp;Signature=LkGcCM-WLW~DbFKuHHtx3jx~U4wl9MBgbu~ABn3U0V7qzVmlsc8grFqOYRmErReLRFDkhjgEdxJmHsHpKj~bwiuKbQ3DKO7n5psjyN6pQGAiwPj0NFxXnjZJ1NvBbEUmBTf~-9dTt9cYIkx7igapLlpBqxL-6ILp1-kN7wColBnZxC4Ky09fhvaTwniJSA0Cxy6Wzd6ell1U6cvj1Hh9RUN92illYq0PcC9V5OfIm2yhUfeRleQUj-xJIj5NgbLyrDyhs7xqjVGj6cM~-Ng0Edgx4ICCb9NOMtrvhnEc8O6Xq30EIe5IMJAFmXsW3XuCGlGTeqURb51bO0UaujO9ow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_fig5.jpg?Expires=2147483647&amp;Signature=OKL11r0A-blxmpe7KvrFa2yW1kBlX2obCNZq6iUSsEn6I9wTCWD4zluFMrUyOf93queuKEN7xgQzJxKIVsv37v1Ry8wqbUKNbjY6NeZ~0uD4iWO2p5Tw~et6wHx0SsquxrSt38D32X3i2B5QPPyQAm6QVsR~jkSUKdYkfaISpRm-7ddUWylz3QfhEpqQuJZzxHbrXZf-PZ4kdiwNYO-Y2GR-u1EsTfp1GREpg0W5fYs1gkfxHdqGunXuA15-y2evRxkV0f4-Ue9kO8ESiH6sU4N-S0P8RY~SaXdcyBwKJaaQGDkUyffhZYoYpY5bJkJLv0w6KUY3Jgeqn1h6~dHZTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_fig4.jpg?Expires=2147483647&amp;Signature=lfH36wQENhqTanGmBhU8JpXzuoAK-WBib9b2OdyLSrZC5Uk2rhAGZBIQq620jUMvnoGUu40zHSzM5lM-a3I5hXEookzrbizYlzL1eX279~EEMq4B17kiw-n6rMOO-Gh0aXLoKsU0r5FCj7il8gp~F8JoIm0-CG-SGp62jdVzGeS3~ejLMvcTXpG1vwIcFEOYJOmazpVNYkmvj8u5yZ9JljgXOKcuOxnnb4CXyV6f~QLYB4PytNidr8hriTCXRYt8zmrKTDAgKrb3K-NpkP~HUDJzve2vlBHO7Jdexb~etqdMg7r~Ae9gU3QrXW8LxoATj-LaXOJ~PdZNoymAkpPLjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_fig3.jpg?Expires=2147483647&amp;Signature=P9uilgZxcIxyrgIbDhKNMyyGREsle3CKMYmZ6h-ZORRkBZtcN8NKv6md5zWLVABtFaJ7amovr0CPUdWQPkDwkD8KIsgxTn~WQXKkiTMjmpXfrTnw3~fJUBMWeAfUYKY-vP3HLrWLiY-FgXngyCuRoCVicd9VQX5NmLo-wQ6V-C6jtku4IoILKW1GN5siEDXXqOWhRQsIalbj3z1PSvNlVQpr4mpeXLDasXyP~OQ0ijyan8L93gJ6gbA0dvadKyNoYg36q7AaTwRkloGO4CWAngIeKSSIBuDQKP7-QneZBnUOg5uXaR7zrBJY65x48xLw2JHNPTQz6pHCuxj7h6OTxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_fig2.jpg?Expires=2147483647&amp;Signature=UpCUs1FDGnj30-zkSKbvq45~NyZgPRPcPKXytp2IpzU3D8hi4Q4fWXesNDUfHW9et33hc3bCeRo9FJqd9kmyS0v1RsFbQja8K~uqsJORG5zunpu89~AKm9cr-5m3AixPWhUrGgg13Biwrl~eYASgs7J2mQ~2BcCcIDxf37X4lGZmgYUltOoKUVUg64IXPIqBeJ369ztiVyGGt5Rhe1UU1Q6QOyOPyJel5m~YZMSquiccyvCNnXy5N0VrXcPdtJAHU9KqybR7LxzR0713NJ5dSRIGOHBP~JgQBj37w5UJ0BWqYLgRyXasMSRGGynyKA6hEp7zsXqpzSXCg3~rz6So2w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp_fig1.jpg?Expires=2147483647&amp;Signature=yrrGOx~uisrD6S7tA~11CtdMS3JCzESfIagQ8aCHqA2y15GSDXdp6w0c-ziZ4zKNyBJAE2OpDacsRIP0wvMtxSGlUlkAYiAxxeo2rx1OAmVyrWH3oL8lU5Ir8lEORXOC349PZlwkZXCyE-w4SiIhGj4WtrLOsUoeTkOXJ2dyvURSrmoU8~DJY6rqWjajFAMAi7Av3Rw-ssO0ueUwdjoopdAN~7T1iAQ3noLwJNNkRdqRu0uQD~CdHsMCTn58rXd0QJ5V9wJia2AjI1F526vyfTjvNMN9f4-buiB85ysedQHHjj41cQ4hr2RB6Zu7ar7z-Plc8vQCZVzMwg-j9f5rWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst033/2/dst033-dst033supp.doc?Expires=2147483647&amp;Signature=PujlbF9QFSjaBuekck226ihuP-geV67xZjhdjn51gxPEFywAkZoArxdt9ShsM5FA6k1Jzy5eB7UEbbBs~FhrsT2chclt16QvD-gy9yEJYbOZR0ri0zdrmwlehGNhRc18hmGOZ6YeQ-uEO8Vts4SnY4c52jEScsByWdGjaIdcqP9ruKL6~rLSc1Jxu~QhtNRLUIxTYknH0gWTaTqRylcdCNJsnyhowkUJU5tZnFb1x0bhlc31ydoo~wPm-JqfcFy2rzYcXdE2nw5SKnxmu1hk5oQjeUG4C-vjzVb2C59gbHGzYWODX26wUW2hzijBI1Wnb0rXn09gwtjdYwWhaGnR3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">935</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">37</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst033"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="420691"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f6%2f593%2f420691">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000330168100006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (37)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F6%2F593%2F420691" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/420691'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-Wide Association Studies Using Single Nucleotide Polymorphism Markers Developed by Re-Sequencing of the Genomes of Cultivated Tomato" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst033");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f593%2f420691">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

