

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/6/371/364039","@type":"ScholarlyArticle","name":"Identification and Functional Analyses of 11 769 Full-length Human cDNAs Focused on Alternative Splicing","datePublished":"2009-10-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2009-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp022","keywords":["full-length cDNA","alternative splicing","alternative transcription start site","mRNA diversity","tissue-specific expression"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/6/371/364039","author":[{"name":"Wakamatsu, Ai","affiliation":"1  Graduate School of Pharmaceutical Sciences, The University of Tokyo ,  7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033 ,  Japan","@type":"Person"},{"name":"Kimura, Kouichi","affiliation":"2  Central Research Laboratory, Hitachi, Ltd ,  Kokubunji, Tokyo 185-8601 ,  Japan","@type":"Person"},{"name":"Yamamoto, Jun-ichi","affiliation":"3  Reverse Proteomics Research Institute ,  1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044 ,  Japan","@type":"Person"},{"name":"Nishikawa, Tetsuo","affiliation":"3  Reverse Proteomics Research Institute ,  1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044 ,  Japan","@type":"Person"},{"name":"Nomura, Nobuo","affiliation":"4  National Institute of Advanced Industrial Science and Technology ,  2-41-6 Aomi, Koto-ku, Tokyo 135-0064 ,  Japan","@type":"Person"},{"name":"Sugano, Sumio","affiliation":"5  Department of Medical Genome Sciences ,  Graduate School of Frontier Sciences, The University of Tokyo ,  4-6-1 Shiroganedai, Minato-ku, Tokyo 108-8639 ,  Japan","@type":"Person"},{"name":"Isogai, Takao","affiliation":"1  Graduate School of Pharmaceutical Sciences, The University of Tokyo ,  7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033 ,  Japan    3  Reverse Proteomics Research Institute ,  1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044 ,  Japan","@type":"Person"}],"description":"Abstract.  We analyzed diversity of mRNA produced as a result of alternative splicing in order to evaluate gene function. First, we predicted the number of huma","pageStart":"371","pageEnd":"383","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/6/1/m_cover.gif?Expires=2147483647&Signature=sOIaosdT6uXpAtt-bpJN4lGWjn89o2XivgxEg5n1d0b6uc4rutjtPlXH8KDnUug4~icfBszbFG-0HY5pufJ6XhNpk9T-WjxmPJk7r95Av27GD54YDSvnj0wsx2TG9xuLp7aypGeHjTYyfAqWQxmgT4WMKLKZ~bziJjeuuiOZ6JhEQXVX~MAA3Zt1fUOb9CdMTVZj~3ig4~HJVGzIabQIJJAixJWzKelG~tTDiZn3JXJTacFA1vuy9TeJf2GPMdGcE7jlnPZJysftGzu~rqySy8oeattNRvSoC4MzCGEev9vmEBUi4d~QtEtbGuzOY3ZLYzMg7sgtncIpR8sAs8c6ig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Identification and Functional Analyses of 11 769 Full-length Human cDNAs Focused on Alternative Splicing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/6/1/m_cover.gif?Expires=2147483647&Signature=sOIaosdT6uXpAtt-bpJN4lGWjn89o2XivgxEg5n1d0b6uc4rutjtPlXH8KDnUug4~icfBszbFG-0HY5pufJ6XhNpk9T-WjxmPJk7r95Av27GD54YDSvnj0wsx2TG9xuLp7aypGeHjTYyfAqWQxmgT4WMKLKZ~bziJjeuuiOZ6JhEQXVX~MAA3Zt1fUOb9CdMTVZj~3ig4~HJVGzIabQIJJAixJWzKelG~tTDiZn3JXJTacFA1vuy9TeJf2GPMdGcE7jlnPZJysftGzu~rqySy8oeattNRvSoC4MzCGEev9vmEBUi4d~QtEtbGuzOY3ZLYzMg7sgtncIpR8sAs8c6ig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identification and Functional Analyses of 11 769 Full-length Human cDNAs Focused on Alternative Splicing" />
    <meta property="og:description" content="Abstract.  We analyzed diversity of mRNA produced as a result of alternative splicing in order to evaluate gene function. First, we predicted the number of huma" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/6/371/364039" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-10-30" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Wakamatsu, Ai" /><meta name="citation_author_institution" content="Graduate School of Pharmaceutical Sciences, The University of Tokyo, 7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033, Japan" /><meta name="citation_author" content="Kimura, Kouichi" /><meta name="citation_author_institution" content="Central Research Laboratory, Hitachi, Ltd, Kokubunji, Tokyo 185-8601, Japan" /><meta name="citation_author" content="Yamamoto, Jun-ichi" /><meta name="citation_author_institution" content="Reverse Proteomics Research Institute, 1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044, Japan" /><meta name="citation_author" content="Nishikawa, Tetsuo" /><meta name="citation_author_institution" content="Reverse Proteomics Research Institute, 1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044, Japan" /><meta name="citation_author" content="Nomura, Nobuo" /><meta name="citation_author_institution" content="National Institute of Advanced Industrial Science and Technology, 2-41-6 Aomi, Koto-ku, Tokyo 135-0064, Japan" /><meta name="citation_author" content="Sugano, Sumio" /><meta name="citation_author_institution" content="Department of Medical Genome Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 4-6-1 Shiroganedai, Minato-ku, Tokyo 108-8639, Japan" /><meta name="citation_author" content="Isogai, Takao" /><meta name="citation_author_institution" content="Graduate School of Pharmaceutical Sciences, The University of Tokyo, 7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033, Japan" /><meta name="citation_author_institution" content="Reverse Proteomics Research Institute, 1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044, Japan" /><meta name="citation_title" content="Identification and Functional Analyses of 11 769 Full-length Human cDNAs Focused on Alternative Splicing" /><meta name="citation_firstpage" content="371" /><meta name="citation_lastpage" content="383" /><meta name="citation_doi" content="10.1093/dnares/dsp022" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2009/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Finishing the euchromatic sequence of the human genome; citation_author=International Human Genome Sequencing Consortium; Nature;  citation_year=2004;  citation_volume=431;  citation_pages=931-45; " /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=Alternative splicing of pre-mRNA: developmental consequences and mechanisms of regulation; Annu. Rev. Genet.;  citation_year=1998;  citation_volume=32;  citation_pages=279-305; " /><meta name="citation_reference" content="citation_title=Protein diversity from alternative splicing: a challenge for bioinformatics and post-genome biology; Cell;  citation_year=2000;  citation_volume=103;  citation_pages=367-70; " /><meta name="citation_reference" content="citation_title=A genomic view of alternative splicing; Nat. Genet.;  citation_year=2002;  citation_volume=30;  citation_pages=13-9; " /><meta name="citation_reference" content="citation_title=Signals and their transduction pathways regulating alternative splicing: a new dimension of the human genome; Hum. Mol. Genet.;  citation_year=2002;  citation_volume=11;  citation_pages=2409-16; " /><meta name="citation_reference" content="citation_title=The relevance of alternative RNA splicing to pharmacogenomics; Trends Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=346-53; " /><meta name="citation_reference" content="citation_title=Complex controls: the role of alternative promoters in mammalian genomes; Trends Genet.;  citation_year=2003;  citation_volume=19;  citation_pages=640-8; " /><meta name="citation_reference" content="citation_title=Multiple variable first exons: a mechanism for cell- and tissue-specific gene regulation; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=79-89; " /><meta name="citation_reference" content="citation_title=Large exons encoding multiple ectodomains are a characteristic feature of protocadherin genes; Proc. Natl Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=3124-9; " /><meta name="citation_reference" content="citation_title=Differential expression of the UGT1A locus in human liver, biliary, and gastric tissue: identification of UGT1A7 and UGT1A10 transcripts in extrahepatic tissue; Mol. Pharmacol.;  citation_year=1997;  citation_volume=52;  citation_pages=212-20; " /><meta name="citation_reference" content="citation_title=Alternative isoform regulation in human tissue transcriptomes; Nature;  citation_year=2008;  citation_volume=456;  citation_pages=470-6; " /><meta name="citation_reference" content="citation_title=HITS-CLIP yields genome-wide insights into brain alternative RNA processing; Nature;  citation_year=2008;  citation_volume=456;  citation_pages=464-9; " /><meta name="citation_reference" content="citation_title=Complete sequencing and characterization of 21,243 full-length human cDNAs; Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=40-5; " /><meta name="citation_reference" content="citation_title=Signal sequence and keyword trap in silico for selection of full-length human cDNAs encoding secretion or membrane proteins from oligo-capped cDNA libraries; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=117-26; " /><meta name="citation_reference" content="citation_title=Human protein factory for converting the transcriptome into an in vitro-expressed proteome; Nat. Methods.;  citation_year=2008;  citation_volume=5;  citation_pages=1011-7; " /><meta name="citation_reference" content="citation_title=Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=55-65; " /><meta name="citation_reference" content="citation_title=Oligo-capping: a simple method to replace the cap structure of eukaryotic mRNAs with oligoribonucleotides; Gene;  citation_year=1994;  citation_volume=138;  citation_pages=171-4; " /><meta name="citation_reference" content="citation_title=Construction of a full-length enriched and a 5'-end enriched cDNA library using the oligo-capping method; Methods Mol. Biol.;  citation_year=2003;  citation_volume=221;  citation_pages=73-91; " /><meta name="citation_reference" content="citation_title=Construction and characterization of a full length-enriched and a 5'-end-enriched cDNA library; Gene;  citation_year=1997;  citation_volume=200;  citation_pages=149-56; " /><meta name="citation_reference" content="citation_title=Statistical analysis of the 5' untranslated region of human mRNA using ‘Oligo-Capped’ cDNA libraries; Genomics;  citation_year=2000;  citation_volume=64;  citation_pages=286-97; " /><meta name="citation_reference" content="citation_title=Database and analysis system for cDNA clones obtained from full-length enriched cDNA libraries; In. Silico Biol.;  citation_year=2002;  citation_volume=2;  citation_pages=5-18; " /><meta name="citation_reference" content="citation_title=Comparison of sequences of cDNA clones obtained from oligo-capping cDNA libraries with those from unigene; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=255-62; " /><meta name="citation_reference" content="citation_title=Intris: A viewer for cDNA-genome alignments enabling efficient detection of splicing variants and expression profiles; Genome Inform.;  citation_year=2002;  citation_volume=13;  citation_pages=548-50; " /><meta name="citation_reference" content="citation_title=Assessing protein coding region integrity in cDNA sequencing projects; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=384-90; " /><meta name="citation_reference" content="citation_title=The translated region inspector for cDNA sequences; Genome Inform.;  citation_year=2003;  citation_volume=14;  citation_pages=456-7; " /><meta name="citation_reference" content="citation_title=A cDNA evaluation system for highly efficient sequencing of splicing variant cDNAs; Genome Inform.;  citation_year=2003;  citation_volume=14;  citation_pages=430-1; " /><meta name="citation_reference" content="citation_title=Identification of a novel domain of fibroblast growth factor 2 controlling its angiogenic properties; J. Biol. Chem.;  citation_year=2003;  citation_volume=278;  citation_pages=8751-60; " /><meta name="citation_reference" content="citation_title=Identification and characterization of a novel member of the fibroblast growth factor family; Eur. J. Neurosci.;  citation_year=1998;  citation_volume=10;  citation_pages=1911-25; " /><meta name="citation_reference" content="citation_title=The OXR domain defines a conserved family of eukaryotic oxidation resistance proteins; BMC Cell Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=13" /><meta name="citation_reference" content="citation_title=Phospholipase D in cell proliferation and cancer; Mol. Cancer. Res.;  citation_year=2003;  citation_volume=1;  citation_pages=789-800; " /><meta name="citation_reference" content="citation_title=Spred-1 negatively regulates interleukin-3-mediated ERK/mitogen-activated protein (MAP) kinase activation in hematopoietic cells; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=52543-51; " /><meta name="citation_reference" content="citation_title=A novel class of murine semaphorins with homology to thrombospondin is differentially expressed during early embryogenesis; Mech. Dev.;  citation_year=1996;  citation_volume=57;  citation_pages=33-45; " /><meta name="citation_reference" content="citation_title=The structures of the human calcium channel alpha 1 subunit (CACNL1A2) and beta subunit (CACNLB3) genes; Genomics;  citation_year=1995;  citation_volume=27;  citation_pages=312-9; " /><meta name="citation_reference" content="citation_title=Translational regulation of BACE-1 expression in neuronal and non-neuronal cells; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=1808-17; " /><meta name="citation_reference" content="citation_title=Phosphatidylinositol 3-kinase and frabin mediate Cryptosporidium parvum cellular invasion via activation of Cdc42; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=31671-8; " /><meta name="citation_reference" content="citation_title=C6ORF32 is upregulated during muscle cell differentiation and induces the formation of cellular filopodia; Dev. Biol.;  citation_year=2007;  citation_volume=301;  citation_pages=70-81; " /><meta name="citation_reference" content="citation_title=The protein tyrosine phosphatase PTPN4/PTP-MEG1, an enzyme capable of dephosphorylating the TCR ITAMs and regulating NF-kappaB, is dispensable for T cell development and/or T cell effector functions; Mol. Immunol.;  citation_year=2008;  citation_volume=45;  citation_pages=3756-66; " /><meta name="citation_reference" content="citation_title=The cluster of BTN genes in the extended major histocompatibility complex; Genomics;  citation_year=2001;  citation_volume=71;  citation_pages=351-62; " /><meta name="citation_reference" content="citation_title=A new mutational AKTivation in the PI3K pathway; Cancer. Cell;  citation_year=2007;  citation_volume=12;  citation_pages=104-7; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/6/371/1040096/dsp022.pdf" /><meta name="description" content="Abstract.  We analyzed diversity of mRNA produced as a result of alternative splicing in order to evaluate gene function. First, we predicted the number of huma" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/6/371/364039" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/6/371/364039" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/6/371/364039");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp022';
    </script>
<title>
	Identification and Functional Analyses of 11 769 Full-length Human cDNAs Focused on Alternative Splicing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./364039" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZBfPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYWthbWF0c3UsIEFpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIFBoYXJtYWNldXRpY2FsIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNy0zLTEgSG9uZ28sIEJ1bmt5by1rdSwgVG9reW8gMTEzLTAwMzMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltdXJhLCBLb3VpY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJhbCBSZXNlYXJjaCBMYWJvcmF0b3J5LCBIaXRhY2hpLCBMdGQsIEtva3VidW5qaSwgVG9reW8gMTg1LTg2MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYW1vdG8sIEp1bi1pY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmV2ZXJzZSBQcm90ZW9taWNzIFJlc2VhcmNoIEluc3RpdHV0ZSwgMS05LTExIEthamksIENoaXlvZGEta3UsIFRva3lvIDEwMS0wMDQ0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5pc2hpa2F3YSwgVGV0c3VvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmV2ZXJzZSBQcm90ZW9taWNzIFJlc2VhcmNoIEluc3RpdHV0ZSwgMS05LTExIEthamksIENoaXlvZGEta3UsIFRva3lvIDEwMS0wMDQ0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5vbXVyYSwgTm9idW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWR2YW5jZWQgSW5kdXN0cmlhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCAyLTQxLTYgQW9taSwgS290by1rdSwgVG9reW8gMTM1LTAwNjQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3VnYW5vLCBTdW1pbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDQtNi0xIFNoaXJvZ2FuZWRhaSwgTWluYXRvLWt1LCBUb2t5byAxMDgtODYzOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc29nYWksIFRha2FvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIFBoYXJtYWNldXRpY2FsIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNy0zLTEgSG9uZ28sIEJ1bmt5by1rdSwgVG9reW8gMTEzLTAwMzMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmV2ZXJzZSBQcm90ZW9taWNzIFJlc2VhcmNoIEluc3RpdHV0ZSwgMS05LTExIEthamksIENoaXlvZGEta3UsIFRva3lvIDEwMS0wMDQ0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSWRlbnRpZmljYXRpb24gYW5kIEZ1bmN0aW9uYWwgQW5hbHlzZXMgb2YgMTEgNzY5IEZ1bGwtbGVuZ3RoIEh1bWFuIGNETkFzIEZvY3VzZWQgb24gQWx0ZXJuYXRpdmUgU3BsaWNpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM4MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcDAyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA5LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluaXNoaW5nIHRoZSBldWNocm9tYXRpYyBzZXF1ZW5jZSBvZiB0aGUgaHVtYW4gZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBIdW1hbiBHZW5vbWUgU2VxdWVuY2luZyBDb25zb3J0aXVtOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDMxOyAgY2l0YXRpb25fcGFnZXM9OTMxLTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGFuYWx5c2lzIG9mIHRoZSBodW1hbiBnZW5vbWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBvZiBwcmUtbVJOQTogZGV2ZWxvcG1lbnRhbCBjb25zZXF1ZW5jZXMgYW5kIG1lY2hhbmlzbXMgb2YgcmVndWxhdGlvbjsgQW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0yNzktMzA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGRpdmVyc2l0eSBmcm9tIGFsdGVybmF0aXZlIHNwbGljaW5nOiBhIGNoYWxsZW5nZSBmb3IgYmlvaW5mb3JtYXRpY3MgYW5kIHBvc3QtZ2Vub21lIGJpb2xvZ3k7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MzY3LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9taWMgdmlldyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaWduYWxzIGFuZCB0aGVpciB0cmFuc2R1Y3Rpb24gcGF0aHdheXMgcmVndWxhdGluZyBhbHRlcm5hdGl2ZSBzcGxpY2luZzogYSBuZXcgZGltZW5zaW9uIG9mIHRoZSBodW1hbiBnZW5vbWU7IEh1bS4gTW9sLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yNDA5LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVsZXZhbmNlIG9mIGFsdGVybmF0aXZlIFJOQSBzcGxpY2luZyB0byBwaGFybWFjb2dlbm9taWNzOyBUcmVuZHMgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNDYtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXggY29udHJvbHM6IHRoZSByb2xlIG9mIGFsdGVybmF0aXZlIHByb21vdGVycyBpbiBtYW1tYWxpYW4gZ2Vub21lczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTY0MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSB2YXJpYWJsZSBmaXJzdCBleG9uczogYSBtZWNoYW5pc20gZm9yIGNlbGwtIGFuZCB0aXNzdWUtc3BlY2lmaWMgZ2VuZSByZWd1bGF0aW9uOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTc5LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZSBleG9ucyBlbmNvZGluZyBtdWx0aXBsZSBlY3RvZG9tYWlucyBhcmUgYSBjaGFyYWN0ZXJpc3RpYyBmZWF0dXJlIG9mIHByb3RvY2FkaGVyaW4gZ2VuZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0zMTI0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBleHByZXNzaW9uIG9mIHRoZSBVR1QxQSBsb2N1cyBpbiBodW1hbiBsaXZlciwgYmlsaWFyeSwgYW5kIGdhc3RyaWMgdGlzc3VlOiBpZGVudGlmaWNhdGlvbiBvZiBVR1QxQTcgYW5kIFVHVDFBMTAgdHJhbnNjcmlwdHMgaW4gZXh0cmFoZXBhdGljIHRpc3N1ZTsgTW9sLiBQaGFybWFjb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9MjEyLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBpc29mb3JtIHJlZ3VsYXRpb24gaW4gaHVtYW4gdGlzc3VlIHRyYW5zY3JpcHRvbWVzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDU2OyAgY2l0YXRpb25fcGFnZXM9NDcwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhJVFMtQ0xJUCB5aWVsZHMgZ2Vub21lLXdpZGUgaW5zaWdodHMgaW50byBicmFpbiBhbHRlcm5hdGl2ZSBSTkEgcHJvY2Vzc2luZzsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1NjsgIGNpdGF0aW9uX3BhZ2VzPTQ2NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBzZXF1ZW5jaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIDIxLDI0MyBmdWxsLWxlbmd0aCBodW1hbiBjRE5BczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz00MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaWduYWwgc2VxdWVuY2UgYW5kIGtleXdvcmQgdHJhcCBpbiBzaWxpY28gZm9yIHNlbGVjdGlvbiBvZiBmdWxsLWxlbmd0aCBodW1hbiBjRE5BcyBlbmNvZGluZyBzZWNyZXRpb24gb3IgbWVtYnJhbmUgcHJvdGVpbnMgZnJvbSBvbGlnby1jYXBwZWQgY0ROQSBsaWJyYXJpZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTE3LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiBwcm90ZWluIGZhY3RvcnkgZm9yIGNvbnZlcnRpbmcgdGhlIHRyYW5zY3JpcHRvbWUgaW50byBhbiBpbiB2aXRyby1leHByZXNzZWQgcHJvdGVvbWU7IE5hdC4gTWV0aG9kcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaWZpY2F0aW9uIG9mIHRyYW5zY3JpcHRpb25hbCBtb2R1bGF0aW9uOiBsYXJnZS1zY2FsZSBpZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBwdXRhdGl2ZSBhbHRlcm5hdGl2ZSBwcm9tb3RlcnMgb2YgaHVtYW4gZ2VuZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTUtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9saWdvLWNhcHBpbmc6IGEgc2ltcGxlIG1ldGhvZCB0byByZXBsYWNlIHRoZSBjYXAgc3RydWN0dXJlIG9mIGV1a2FyeW90aWMgbVJOQXMgd2l0aCBvbGlnb3JpYm9udWNsZW90aWRlczsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzg7ICBjaXRhdGlvbl9wYWdlcz0xNzEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgZnVsbC1sZW5ndGggZW5yaWNoZWQgYW5kIGEgNSctZW5kIGVucmljaGVkIGNETkEgbGlicmFyeSB1c2luZyB0aGUgb2xpZ28tY2FwcGluZyBtZXRob2Q7IE1ldGhvZHMgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjE7ICBjaXRhdGlvbl9wYWdlcz03My05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgZnVsbCBsZW5ndGgtZW5yaWNoZWQgYW5kIGEgNSctZW5kLWVucmljaGVkIGNETkEgbGlicmFyeTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDA7ICBjaXRhdGlvbl9wYWdlcz0xNDktNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsIGFuYWx5c2lzIG9mIHRoZSA1JyB1bnRyYW5zbGF0ZWQgcmVnaW9uIG9mIGh1bWFuIG1STkEgdXNpbmcg4oCYT2xpZ28tQ2FwcGVk4oCZIGNETkEgbGlicmFyaWVzOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTI4Ni05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGF0YWJhc2UgYW5kIGFuYWx5c2lzIHN5c3RlbSBmb3IgY0ROQSBjbG9uZXMgb2J0YWluZWQgZnJvbSBmdWxsLWxlbmd0aCBlbnJpY2hlZCBjRE5BIGxpYnJhcmllczsgSW4uIFNpbGljbyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NS0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBzZXF1ZW5jZXMgb2YgY0ROQSBjbG9uZXMgb2J0YWluZWQgZnJvbSBvbGlnby1jYXBwaW5nIGNETkEgbGlicmFyaWVzIHdpdGggdGhvc2UgZnJvbSB1bmlnZW5lOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjU1LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyaXM6IEEgdmlld2VyIGZvciBjRE5BLWdlbm9tZSBhbGlnbm1lbnRzIGVuYWJsaW5nIGVmZmljaWVudCBkZXRlY3Rpb24gb2Ygc3BsaWNpbmcgdmFyaWFudHMgYW5kIGV4cHJlc3Npb24gcHJvZmlsZXM7IEdlbm9tZSBJbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NTQ4LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NpbmcgcHJvdGVpbiBjb2RpbmcgcmVnaW9uIGludGVncml0eSBpbiBjRE5BIHNlcXVlbmNpbmcgcHJvamVjdHM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9Mzg0LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdHJhbnNsYXRlZCByZWdpb24gaW5zcGVjdG9yIGZvciBjRE5BIHNlcXVlbmNlczsgR2Vub21lIEluZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00NTYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjRE5BIGV2YWx1YXRpb24gc3lzdGVtIGZvciBoaWdobHkgZWZmaWNpZW50IHNlcXVlbmNpbmcgb2Ygc3BsaWNpbmcgdmFyaWFudCBjRE5BczsgR2Vub21lIEluZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00MzAtMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBub3ZlbCBkb21haW4gb2YgZmlicm9ibGFzdCBncm93dGggZmFjdG9yIDIgY29udHJvbGxpbmcgaXRzIGFuZ2lvZ2VuaWMgcHJvcGVydGllczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9Mjc4OyAgY2l0YXRpb25fcGFnZXM9ODc1MS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSBub3ZlbCBtZW1iZXIgb2YgdGhlIGZpYnJvYmxhc3QgZ3Jvd3RoIGZhY3RvciBmYW1pbHk7IEV1ci4gSi4gTmV1cm9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTkxMS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE9YUiBkb21haW4gZGVmaW5lcyBhIGNvbnNlcnZlZCBmYW1pbHkgb2YgZXVrYXJ5b3RpYyBveGlkYXRpb24gcmVzaXN0YW5jZSBwcm90ZWluczsgQk1DIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhvc3Bob2xpcGFzZSBEIGluIGNlbGwgcHJvbGlmZXJhdGlvbiBhbmQgY2FuY2VyOyBNb2wuIENhbmNlci4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9Nzg5LTgwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3ByZWQtMSBuZWdhdGl2ZWx5IHJlZ3VsYXRlcyBpbnRlcmxldWtpbi0zLW1lZGlhdGVkIEVSSy9taXRvZ2VuLWFjdGl2YXRlZCBwcm90ZWluIChNQVApIGtpbmFzZSBhY3RpdmF0aW9uIGluIGhlbWF0b3BvaWV0aWMgY2VsbHM7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3OTsgIGNpdGF0aW9uX3BhZ2VzPTUyNTQzLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIGNsYXNzIG9mIG11cmluZSBzZW1hcGhvcmlucyB3aXRoIGhvbW9sb2d5IHRvIHRocm9tYm9zcG9uZGluIGlzIGRpZmZlcmVudGlhbGx5IGV4cHJlc3NlZCBkdXJpbmcgZWFybHkgZW1icnlvZ2VuZXNpczsgTWVjaC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT01NzsgIGNpdGF0aW9uX3BhZ2VzPTMzLTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3RydWN0dXJlcyBvZiB0aGUgaHVtYW4gY2FsY2l1bSBjaGFubmVsIGFscGhhIDEgc3VidW5pdCAoQ0FDTkwxQTIpIGFuZCBiZXRhIHN1YnVuaXQgKENBQ05MQjMpIGdlbmVzOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTMxMi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbmFsIHJlZ3VsYXRpb24gb2YgQkFDRS0xIGV4cHJlc3Npb24gaW4gbmV1cm9uYWwgYW5kIG5vbi1uZXVyb25hbCBjZWxsczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTgwOC0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhvc3BoYXRpZHlsaW5vc2l0b2wgMy1raW5hc2UgYW5kIGZyYWJpbiBtZWRpYXRlIENyeXB0b3Nwb3JpZGl1bSBwYXJ2dW0gY2VsbHVsYXIgaW52YXNpb24gdmlhIGFjdGl2YXRpb24gb2YgQ2RjNDI7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3OTsgIGNpdGF0aW9uX3BhZ2VzPTMxNjcxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUM2T1JGMzIgaXMgdXByZWd1bGF0ZWQgZHVyaW5nIG11c2NsZSBjZWxsIGRpZmZlcmVudGlhdGlvbiBhbmQgaW5kdWNlcyB0aGUgZm9ybWF0aW9uIG9mIGNlbGx1bGFyIGZpbG9wb2RpYTsgRGV2LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDE7ICBjaXRhdGlvbl9wYWdlcz03MC04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHByb3RlaW4gdHlyb3NpbmUgcGhvc3BoYXRhc2UgUFRQTjQvUFRQLU1FRzEsIGFuIGVuenltZSBjYXBhYmxlIG9mIGRlcGhvc3Bob3J5bGF0aW5nIHRoZSBUQ1IgSVRBTXMgYW5kIHJlZ3VsYXRpbmcgTkYta2FwcGFCLCBpcyBkaXNwZW5zYWJsZSBmb3IgVCBjZWxsIGRldmVsb3BtZW50IGFuZC9vciBUIGNlbGwgZWZmZWN0b3IgZnVuY3Rpb25zOyBNb2wuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9Mzc1Ni02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNsdXN0ZXIgb2YgQlROIGdlbmVzIGluIHRoZSBleHRlbmRlZCBtYWpvciBoaXN0b2NvbXBhdGliaWxpdHkgY29tcGxleDsgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz0zNTEtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IG11dGF0aW9uYWwgQUtUaXZhdGlvbiBpbiB0aGUgUEkzSyBwYXRod2F5OyBDYW5jZXIuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMDQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE2LzYvMzcxLzEwNDAwOTYvZHNwMDIyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSBhbmFseXplZCBkaXZlcnNpdHkgb2YgbVJOQSBwcm9kdWNlZCBhcyBhIHJlc3VsdCBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiBvcmRlciB0byBldmFsdWF0ZSBnZW5lIGZ1bmN0aW9uLiBGaXJzdCwgd2UgcHJlZGljdGVkIHRoZSBudW1iZXIgb2YgaHVtYSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzY0MDM5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZAL0u6dUQKnAhCV41L0MWWnbuysE" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f6%2f371%2f364039" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f6%2f371%2f364039" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14135" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/6/1/m_cover.gif?Expires=2147483647&amp;Signature=sOIaosdT6uXpAtt-bpJN4lGWjn89o2XivgxEg5n1d0b6uc4rutjtPlXH8KDnUug4~icfBszbFG-0HY5pufJ6XhNpk9T-WjxmPJk7r95Av27GD54YDSvnj0wsx2TG9xuLp7aypGeHjTYyfAqWQxmgT4WMKLKZ~bziJjeuuiOZ6JhEQXVX~MAA3Zt1fUOb9CdMTVZj~3ig4~HJVGzIabQIJJAixJWzKelG~tTDiZn3JXJTacFA1vuy9TeJf2GPMdGcE7jlnPZJysftGzu~rqySy8oeattNRvSoC4MzCGEev9vmEBUi4d~QtEtbGuzOY3ZLYzMg7sgtncIpR8sAs8c6ig__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/6">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5278003">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5278003">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5278009">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5278009">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5278019">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5278019">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5278030">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5278030">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5278060">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5278060">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5278062">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5278062">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5278064">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5278064">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5278066">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5278066">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/6/353/364236">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identification and Functional Analyses of 11 769 Full-length Human cDNAs Focused on Alternative Splicing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ai Wakamatsu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Graduate School of Pharmaceutical Sciences, The University of Tokyo</div>, <div class="addr-line">7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ai+Wakamatsu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wakamatsu A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wakamatsu A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ai Wakamatsu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kouichi Kimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Central Research Laboratory, Hitachi, Ltd</div>, <div class="addr-line">Kokubunji, Tokyo 185-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kouichi+Kimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kimura K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kimura K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kouichi Kimura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun-ichi Yamamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Reverse Proteomics Research Institute</div>, <div class="addr-line">1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun-ichi+Yamamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamamoto J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamamoto J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jun-ichi Yamamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuo Nishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Reverse Proteomics Research Institute</div>, <div class="addr-line">1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuo+Nishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishikawa T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishikawa T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tetsuo Nishikawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nobuo Nomura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">National Institute of Advanced Industrial Science and Technology</div>, <div class="addr-line">2-41-6 Aomi, Koto-ku, Tokyo 135-0064</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nobuo+Nomura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nomura N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nomura N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nobuo Nomura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sumio Sugano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Medical Genome Sciences</div>, <div class="insititution">Graduate School of Frontier Sciences, The University of Tokyo</div>, <div class="addr-line">4-6-1 Shiroganedai, Minato-ku, Tokyo 108-8639</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sumio+Sugano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugano S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugano S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sumio Sugano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takao Isogai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Graduate School of Pharmaceutical Sciences, The University of Tokyo</div>, <div class="addr-line">7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Reverse Proteomics Research Institute</div>, <div class="addr-line">1-9-11 Kaji, Chiyoda-ku, Tokyo 101-0044</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>Corresponding author. E-mail: <a href="mailto:tisogai@mol.f.u-tokyo.ac.jp" target="_blank">tisogai@mol.f.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takao+Isogai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isogai T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isogai T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takao Isogai</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 6, 1 December 2009, Pages 371–383, <a href='https://doi.org/10.1093/dnares/dsp022'>https://doi.org/10.1093/dnares/dsp022</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">30 October 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 August 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 October 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="364039" href="/dnaresearch/article-pdf/16/6/371/1040096/dsp022.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Ai Wakamatsu, Kouichi Kimura, Jun-ichi Yamamoto, Tetsuo Nishikawa, Nobuo Nomura, Sumio Sugano, Takao Isogai;  Identification and Functional Analyses of 11 769 Full-length Human cDNAs Focused on Alternative Splicing, <em>DNA Research</em>, Volume 16, Issue 6, 1 December 2009, Pages 371–383, <a href='https://doi.org/10.1093/dnares/dsp022'>https://doi.org/10.1093/dnares/dsp022</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/364039?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/364039?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/364039?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/364039?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/364039?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/364039?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/364039?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identification%20and%20Functional%20Analyses%20of%2011%20769%20Full-length%20Human%20cDNAs%20Focused%20on%20Alternative%20Splicing&amp;publicationDate=2009-10-30&amp;volumeNum=16&amp;issueNum=6&amp;author=Wakamatsu%2C%20Ai%3B%20Kimura%2C%20Kouichi&amp;startPage=371&amp;endPage=383&amp;contentId=10.1093%2Fdnares%2Fdsp022&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f6%2f371%2f364039" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5278003 id="5278003" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We analyzed diversity of mRNA produced as a result of alternative splicing in order to evaluate gene function. First, we predicted the number of human genes transcribed into protein-coding mRNAs by using the sequence information of full-length cDNAs and 5′-ESTs and obtained 23 241 of such human genes. Next, using these genes, we analyzed the mRNA diversity and consequently sequenced and identified 11 769 human full-length cDNAs whose predicted open reading frames were different from other known full-length cDNAs. Especially, 30% of the cDNAs we identified contained variation in the transcription start site (TSS). Our analysis, which particularly focused on multiple variable first exons (FEVs) formed due to the alternative utilization of TSSs, led to the identification of 261 FEVs expressed in the tissue-specific manner. Quantification of the expression profiles of 13 genes by real-time PCR analysis further confirmed the tissue-specific expression of FEVs, e.g. OXR1 had specific TSS in brain and tumor tissues, and so on. Finally, based on the results of our mRNA diversity analysis, we have created the FLJ Human cDNA Database. From our result, it has been understood mechanisms that one gene produces suitable protein-coding transcripts responding to the situation and the environment.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;full-length cDNA&quot;">full-length cDNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative transcription start site&quot;">alternative transcription start site</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mRNA diversity&quot;">mRNA diversity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tissue-specific expression&quot;">tissue-specific expression</a></div>                            <h2 scrollto-destination=5278009 id="5278009" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>One of the most interesting findings revealed by the Human Genome Project is that the human genome contains only 20 000–25 000 protein-coding genes.<sup><a reveal-id="DSP022C1" data-open="DSP022C1" class="link link-ref link-reveal xref-bibr">1</a></sup> This number is unexpectedly too small. To explain this unexpected result and to understand functions of genes, it is necessary to analyze mRNA diversity.</p><p>Biologically, multiple transcripts can be generated from a single gene by alternative splicing (AS). According to several reports on genome research, AS occurs in 30–60% of human genes.<sup><a reveal-id="DSP022C2  DSP022C5" data-open="DSP022C2  DSP022C5" class="link link-ref link-reveal xref-bibr">2–5</a></sup> It has been reported that AS of a single gene could produce transcripts coding for multiple proteins, each exhibiting different biochemical properties including binding, intracellular localization and regulation of enzymatic activities.<sup><a reveal-id="DSP022C6" data-open="DSP022C6" class="link link-ref link-reveal xref-bibr">6</a></sup> AS is also of interest to the pharmaceutical research because unwanted AS of genes could lead to various genetic diseases and cancers.<sup><a reveal-id="DSP022C7" data-open="DSP022C7" class="link link-ref link-reveal xref-bibr">7</a></sup> We have particularly focused on the analysis of AS patterns that are produce by utilizing alternative transcription start sites (TSSs). Indeed, multiple transcripts were produced from a gene by utilizing variable TSSs.<sup><a reveal-id="DSP022C8" data-open="DSP022C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSP022C9" data-open="DSP022C9" class="link link-ref link-reveal xref-bibr">9</a></sup> For example, the <em>Pcdh</em> gene, which contained variable TSSs, was shown to produce different transcripts;<sup><a reveal-id="DSP022C10" data-open="DSP022C10" class="link link-ref link-reveal xref-bibr">10</a></sup> similarly, UGTs (UDP-glucuronosyltransferases), which contained more than 10 TSSs.<sup><a reveal-id="DSP022C11" data-open="DSP022C11" class="link link-ref link-reveal xref-bibr">11</a></sup> From these findings, it is clear that to elucidate gene function, we have to further our knowledge on and understanding of all transcripts made from each gene, particularly those of the protein-coding transcripts. However, identification of all protein-coding transcripts have so far been difficult due to the fact that a large number of EST data accumulated in the databases are 3'-EST data, which were obtained by sequencing cDNAs from the polyA-end. Thus, even though sequences of a large number of mRNAs are already known, our understanding of these mRNAs remained incomplete because of the fragmentary nature and 3′-end bias of their sequences. Because of the lack of sequence information, it has been difficult to predict TSSs and to identify all the open reading frame (ORF) regions. Although the use of next generation sequencer helped in making advances in analyzing TSSs, it still remains extremely difficult to evaluate diversities of mRNAs transcribed by each gene because of their accumulation of short-length sequences (less than 50 bases) of cDNA clones.<sup><a reveal-id="DSP022C12" data-open="DSP022C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSP022C13" data-open="DSP022C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>We sequenced ∼55 000 human full-length cDNAs, including 11 769 newly identified cDNAs described in this paper, and also obtained ∼1.45 million 5'-end-one-pass sequences (5'-EST).<sup><a reveal-id="DSP022C14  DSP022C17" data-open="DSP022C14  DSP022C17" class="link link-ref link-reveal xref-bibr">14–17</a></sup> We believe that these cDNA sequences are very useful in analyzing the diversity of protein-coding transcripts and would definitely contribute to our understanding of mRNA. First, our cDNA clones were isolated from full-length human cDNA libraries constructed by an optimized oligo-capping method, and therefore by utilizing their sequence information, we were able to identify the TSS with 90% or better accuracy.<sup><a reveal-id="DSP022C14" data-open="DSP022C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSP022C18  DSP022C20" data-open="DSP022C18  DSP022C20" class="link link-ref link-reveal xref-bibr">18–20</a></sup> Thus, we could easily and accurately identify TSSs of even low-expressing genes, for which up until now it required comparison of a large amount of data.<sup><a reveal-id="DSP022C17" data-open="DSP022C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Second, our 5′-EST data contained, on the average, sequence information of ∼500 bases/cDNA clone, which covered two or more exons. Since the average length of the 5'-untranslated region is believed to be 125 bases,<sup><a reveal-id="DSP022C21" data-open="DSP022C21" class="link link-ref link-reveal xref-bibr">21</a></sup> it was possible to predict ORF regions using our 5'-EST data. Finally, the most important point is that all of our resources were obtained from the full-length cDNAs, including the TSS and the polyA site. Moreover, we could obtain various findings on protein expression from our full-length cDNAs.<sup><a reveal-id="DSP022C16" data-open="DSP022C16" class="link link-ref link-reveal xref-bibr">16</a></sup> These findings could not be obtained from sequences of short mRNA fragments. Since AS of genes could potentially create a large number of protein-coding transcripts, analyzing full-length cDNAs might be immensely valuable in understanding gene function.</p><p>Here, we report on our analysis of 11 769 full-length cDNAs, which were identified from our full-length cDNA libraries, and contained ORFs as a result of AS. We also present our analysis on the splice patterns and expression profiles of the identified cDNAs to explore the correlation between the mRNA diversity and gene function. Furthermore, we describe 261 full-length cDNAs with unique TSSs known as multiple variable first exon (FEV) and report on their expression profiles. Finally, we report establishing the FLJ Human cDNA Database based on the results of our analysis of the variable protein-coding transcripts generated from each gene by AS.</p>                            <h2 scrollto-destination=5278019 id="5278019" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5278020 id="5278020" class="section-title" data-legacy-id=s2a>2.1. Construction of full-length cDNA libraries</h3>
<p>Most total RNAs isolated from various tissues and cells were purchased from Clontech and Ambion. Cells were cultured following established protocol, and cytoplasmic total RNAs were extracted from these cultured cells following a standard RNA purification method. The list of total RNAs used in this study was shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. We constructed cDNA libraries from total RNAs by an optimized oligo-capping method (detailed method for the optimized oligo-capping is provided in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Method 1</a>).<sup><a reveal-id="DSP022C18" data-open="DSP022C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSP022C19" data-open="DSP022C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Briefly, total RNAs were treated with bacteria alkaline phosphatase (TaKaRa) and tobacco acid pyrophosphatase. After that, total RNAs were ligated to the oligo-RNA using the RNA ligase (TaKaRa). Oligo-capped polyA(+) RNAs were then isolated oligo-dT columns. The first-strand cDNAs were synthesized using the Superscript II reverse transcriptase (Invitrogen), the synthesized cDNAs were amplified using the Gene Amp XL PCR kit (ABI) and the amplified product was digested with the restriction enzyme SfiI. Fragments longer than 2 kb were selected and purified by agarose gel electrophoresis and cloned into the DraIII-digested pME18SFL3 vector following the standard methods. The 5′-end-one-pass sequences of cloned cDNAs were analyzed using the ABI 377 and 3700 sequencers (ABI). The 5'-end fullness rate of the constructed oligo-capped cDNA libraries was evaluated as described previously,<sup><a reveal-id="DSP022C22" data-open="DSP022C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSP022C23" data-open="DSP022C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and the detailed method for determining the 5'-end fullness rate is provided in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Method 2</a>.</p>                            <h3 scrollto-destination=5278022 id="5278022" class="section-title" data-legacy-id=s2b>2.2. Genome mapping and clustering</h3>
<p>The 5′- and 3′-ends of cDNA sequences and the full-length cDNA sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) were mapped onto the human genome (UCSC hg 18 NCBI Build 36.1). Possible local alignments between the cDNAs and genome sequences were identified by using the NCBI Mega BLAST program (ftp://ftp.ncbi.nih.gov/blast/). For each cDNA, best mapping of the sequence was determined from these local alignments using a dynamic programming technique that optimized the identity, coverage and topology of exons. The joining portions of consecutive local alignments were refined so as to restore the consensus sequence in the canonical splice sites. On the basis of the mapping results clustering of cDNA sequences were performed as follows: two cDNA sequences were grouped into the same cluster if their mapped positions shared at least one base on the genome. In general, each cluster corresponded to a single gene locus.</p>                            <h3 scrollto-destination=5278024 id="5278024" class="section-title" data-legacy-id=s2c>2.3. Identification of alternatively spliced variants of mRNAs</h3>
<p>On the basis of the results of genome mapping and clustering analysis, ESTs that had different regions compared with known full-length cDNAs by AS were selected by Intris, a viewer for cDNA-genome alignments used for analysis of splicing variants and expression profiles.<sup><a reveal-id="DSP022C24" data-open="DSP022C24" class="link link-ref link-reveal xref-bibr">24</a></sup> To exclude the cDNA fragments derived from the immature mRNA and genomic DNA, reliability of mRNA was evaluated by using not only the human EST data but also the data conserved from other animals (Phastcons; obtained from UCSC Genome Browser). We predicted the ORF regions from the 5'-end sequences of full-length cDNAs on selected ESTs by using ATGpr (<a class="link link-uri" href="http://flj.lifesciencedb.jp/top/">http://flj.lifesciencedb.jp/top/</a>).<sup><a reveal-id="DSP022C25" data-open="DSP022C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Next, we excluded those ESTs from the selected analytical targets when the predicted ORF regions of the selected ESTs were the same as the ORF regions of known full-length cDNAs. In addition, even if the predicted ORF regions were different from the ORF regions of known full-length cDNAs, we excluded cDNA clones containing extremely short ORF regions (mostly 60 amino acids or less) compared with the other full-length cDNAs that mapped in the same locus of the human genome. The selected cDNAs were further sequenced by primer walking method using an ABI3700 sequencer (ABI) to obtain information on 500 additional bases, and the ORF regions were predicted again by using the ATGpr.<sup><a reveal-id="DSP022C25" data-open="DSP022C25" class="link link-ref link-reveal xref-bibr">25</a></sup> We also evaluated the predicted ORF regions by using TRis,<sup><a reveal-id="DSP022C26" data-open="DSP022C26" class="link link-ref link-reveal xref-bibr">26</a></sup> translated region inspector, and examined their novelty of amino acid sequences by using ALVISION,<sup><a reveal-id="DSP022C27" data-open="DSP022C27" class="link link-ref link-reveal xref-bibr">27</a></sup> aligns two cDNA sequences that are splicing variants allowing large gaps. When the reliability of the predicted ORF region was insufficient, we excluded it from our list of analytical targets. When the predicted ORF regions of the selected cDNAs were judged reliable and different from those of the known full-length cDNAs, we then sequenced the full-length cDNA clone all the way up to the stop codon. Consequently, we completely sequenced 11 769 of full-length FLJ cDNAs and analyzed their tissue-specific expression. A detailed method for the analysis of the tissue-specific expression of the cDNAs is provided in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Method 3</a>. We have also constructed the FLJ Human cDNA Database (<a class="link link-uri" href="http://flj.lifesciencedb.jp">http://flj.lifesciencedb.jp</a>) that contained these sequence information. A detailed method for the analysis of AS by using the information available in the FLJ Human cDNA Database is provided in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Method 4</a>. Sequences of 11 769 of our full-length cDNAs were also deposited in the DDBJ/GenBank/EMBL databases (AK293122–AK304890).</p>                            <h3 scrollto-destination=5278026 id="5278026" class="section-title" data-legacy-id=s2d>2.4. Functional analysis of full-length cDNAs <em>in silico</em></h3>
<p>Sequences of cDNAs were analyzed for the signal sequences, trans-membrane domains and motifs in the encoded proteins by using Signal P ver. 3.0 (<a class="link link-uri" href="http://www.cbs.dtu.dk/services/SignalP/">http://www.cbs.dtu.dk/services/SignalP/</a>), SOSUI ver. 1.5 (Mitsui Knowledge Industry) and Pfam 19.0 (November 2005; <a class="link link-uri" href="http://pfam.sanger.ac.uk/">http://pfam.sanger.ac.uk/</a>), respectively. We obtained information on motifs showing E-values of e-30 or more from the Pfam analysis, and based on these results, we then categorized each cDNA and the corresponding gene according to its gene ontology (GO) (<a class="link link-uri" href="http://www.geneontology.org/">http://www.geneontology.org/</a>) classification by using InterPro (<a class="link link-uri" href="http://www.ebi.ac.uk/interpro/">http://www.ebi.ac.uk/interpro/</a>).</p>                            <h3 scrollto-destination=5278028 id="5278028" class="section-title" data-legacy-id=s2e>2.5. Quantitative real-time PCR analysis</h3>
<p>Total RNAs derived from various tissues were purchased from Clontech, Ambion and STRATAGENE (listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). From 10 µg of each total RNA, first-strand cDNAs were synthesized using random primers and the Superscript III reverse transcriptase (Invitrogen) following the manufacturer's instructions. Real-time PCR was performed using TaqMan Universal Master Mix (ABI) or SYBR Master Mix (ABI) on an ABI Fast7500 System (ABI) according to the manufacturer's instructions. Approximately 300 ng of template cDNAs was used in each PCR reaction. Probes and primers were designed using the Primer Express3.0 (ABI) (refer to <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a> for the list of primers). The expression levels of genes were normalized with respect to that of the human GAPDH, and expression values of individual genes were calculated by comparing their Ct values to that of the control using the RQ software (ABI). The expression levels of genes were represented in log<sub>10</sub> base. Samples were run in duplicates and the data shown are the average of two experiments.</p>                            <h2 scrollto-destination=5278030 id="5278030" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5278031 id="5278031" class="section-title" data-legacy-id=s3a>3.1. Identification of human genes</h3>
<p>It is known that AS could produce mRNA diversity.<sup><a reveal-id="DSP022C2  DSP022C6" data-open="DSP022C2  DSP022C6" class="link link-ref link-reveal xref-bibr">2–6</a></sup> However, to analyze the mRNA diversity, it is necessary to identify human genes (i.e. the genome loci from where the protein-coding mRNAs are transcribed). We obtained 1.45 million human full-length cDNAs and sequenced their 5'-ends. We previously selected ∼30 000 cDNAs from these full-length cDNAs based on the novelty analysis, and completely sequenced them.<sup><a reveal-id="DSP022C14  DSP022C16" data-open="DSP022C14  DSP022C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> Later, we also selected ∼25 000 cDNAs based on the mRNA diversity and also sequenced them completely. In our quest to identify human genes, we used, for our analysis, the sequence information on these 55 000 full-length human cDNAs including 11 769 cDNAs reported in this paper (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Furthermore, for the analysis, we not only used our own data but also data from 52 000 full-length human cDNA sequences available from the public databases, 30 000 human RefSeq (NCBI Reference Sequences; <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/RefSeq/">http://www.ncbi.nlm.nih.gov/RefSeq/</a>) and 48 000 Ensembl, human gene transcripts (<a class="link link-uri" href="http://www.ensembl.org/index.html">http://www.ensembl.org/index.html</a>). In addition, we used EST sequences obtained by us and from other public databases (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). All the sequence data we collected were mapped onto the human genome and clustered. We then examined reliability of each full-length cDNAs by Intris<sup><a reveal-id="DSP022C24" data-open="DSP022C24" class="link link-ref link-reveal xref-bibr">24</a></sup> using sequences of all full-length cDNAs and ESTs mapped on the same locus of the genome, and based on this analysis, we selected only the reliable cDNAs for the gene identification analysis. We determined the genome locus of each one of the selected reliable cDNA and manually checked them one by one to identify the corresponding gene. As a result, we identified 23 241 human genes from this analysis (Fig. <a reveal-id="DSP022F1" data-open="DSP022F1" class="link link-reveal link-table xref-fig">1</a>A). Each gene cluster was classified into three categories based on the reliability scores. The number of genes in the high reliability category (high category) were 16 754. Sequences of cDNAs belonging to the high-category group were found to be already analyzed because the genome locus was covered by sequence information available from the three types of databases, the human full-length cDNAs, RefSeq and Ensembl. It accounted for 72% of the total number of genes. The number of genes with intermediate reliability (medium category) was 2854. As for the medium-category group, the genome locus was covered by sequence information available from only the human full-length cDNAs or from two out of three of the above-mentioned databases. The number of genes with low reliability (low category) were 3633. As for the low-category group, the gene locus was covered by sequence information available only from the RefSeq or the Ensembl.</p>                        <a id="5278033" scrollto-destination="5278033"></a>
<div id="DSP022F1" data-id="DSP022F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/m_dsp02201.jpeg?Expires=2147483647&Signature=SkNoDwcWWeUhzIBnHhfWWhuc2U4lgxcih-quskfopmvRsi9QRY14Ikrr8cu9sQq88LlqR9RLT1kzdFHImbRC5qwh9Cd2v8iv6RvD0pQnSbmTbE0UJ3efw7IMKQroglwQfF6E45-AOiA5~JhApM4uz9d27esz8WYKq6lFJJI8fqkoATdnxEI8ukwrf4vRdrJtVHeNaK-lwB1Cguv7CShtEF25GhcTpijEcrrOmznoTBRl9TimzeZmKtGaTQPTWxYuTE5y19finZ04szCykWiinrz8FJXIRbAYAywjHJc~d53S-Q4pqjFYejtxoNHBTMrFKggk957MzcRiFsIdjEMHlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Clustering of human cDNA sequences. (A) Estimation of the number of human genes from full-length cDNAs and ESTs. Outline of our gene prediction method from the human full-length cDNAs and ESTs mapped to human genome is schematically shown. For each one of the predicted genes, classification reliability was evaluated manually. (B) Cover rate of FLJ EST sequences and (C) cover rate of FLJ full-length sequenced cDNAs. Results of reliability analysis according to the category based on the cover rates of 1.45 million of ESTs (B) and 55 000 full-length cDNAs (C)." path-from-xml="dsp02201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5278033/dsp02201.jpeg" path-from-xml="dsp02201.jpeg" target="_blank">View large</a><a section="5278033" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/dsp02201.jpeg?Expires=2147483647&Signature=wCdPYFXlVoX34sGAClFkIrUxHQyS7IZbFJ22Ew4U2rwdc4ypBBLde5v-Ml~TEBJCxRL6fqXnJuxUmkryrs-NHxVBPZxndsGVUsh2Xj909HbRYcftwyuUAvShE5I-y6i2k1dl~bKru4lNUtStywLSfeUPCtYmTEj~McFTXHdQypGN9nNZXWPsVSBUeqUIt5Wt6Dio87H9mdKbRrHP8UXwcyWu~K86aHBYjKZlhY0uVJ56SHz8aGGwDBOi7sHEIQEE3lQsq0B6awaHErDd11ZIhBvMG0RjXvG8APtuiNQoYYTkdue2sMA1xAAUpwphtc2ji51j7gvwQXV-3fXoYKWjzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5278033&ar=364039&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Clustering of human cDNA sequences. (A) Estimation of the number of human genes from full-length cDNAs and ESTs. Outline of our gene prediction method from the human full-length cDNAs and ESTs mapped to human genome is schematically shown. For each one of the predicted genes, classification reliability was evaluated manually. (B) Cover rate of FLJ EST sequences and (C) cover rate of FLJ full-length sequenced cDNAs. Results of reliability analysis according to the category based on the cover rates of 1.45 million of ESTs (B) and 55 000 full-length cDNAs (C).</p></div></div><div content-id="DSP022F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/m_dsp02201.jpeg?Expires=2147483647&Signature=SkNoDwcWWeUhzIBnHhfWWhuc2U4lgxcih-quskfopmvRsi9QRY14Ikrr8cu9sQq88LlqR9RLT1kzdFHImbRC5qwh9Cd2v8iv6RvD0pQnSbmTbE0UJ3efw7IMKQroglwQfF6E45-AOiA5~JhApM4uz9d27esz8WYKq6lFJJI8fqkoATdnxEI8ukwrf4vRdrJtVHeNaK-lwB1Cguv7CShtEF25GhcTpijEcrrOmznoTBRl9TimzeZmKtGaTQPTWxYuTE5y19finZ04szCykWiinrz8FJXIRbAYAywjHJc~d53S-Q4pqjFYejtxoNHBTMrFKggk957MzcRiFsIdjEMHlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Clustering of human cDNA sequences. (A) Estimation of the number of human genes from full-length cDNAs and ESTs. Outline of our gene prediction method from the human full-length cDNAs and ESTs mapped to human genome is schematically shown. For each one of the predicted genes, classification reliability was evaluated manually. (B) Cover rate of FLJ EST sequences and (C) cover rate of FLJ full-length sequenced cDNAs. Results of reliability analysis according to the category based on the cover rates of 1.45 million of ESTs (B) and 55 000 full-length cDNAs (C)." path-from-xml="dsp02201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5278033/dsp02201.jpeg" path-from-xml="dsp02201.jpeg" target="_blank">View large</a><a section="5278033" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/dsp02201.jpeg?Expires=2147483647&Signature=wCdPYFXlVoX34sGAClFkIrUxHQyS7IZbFJ22Ew4U2rwdc4ypBBLde5v-Ml~TEBJCxRL6fqXnJuxUmkryrs-NHxVBPZxndsGVUsh2Xj909HbRYcftwyuUAvShE5I-y6i2k1dl~bKru4lNUtStywLSfeUPCtYmTEj~McFTXHdQypGN9nNZXWPsVSBUeqUIt5Wt6Dio87H9mdKbRrHP8UXwcyWu~K86aHBYjKZlhY0uVJ56SHz8aGGwDBOi7sHEIQEE3lQsq0B6awaHErDd11ZIhBvMG0RjXvG8APtuiNQoYYTkdue2sMA1xAAUpwphtc2ji51j7gvwQXV-3fXoYKWjzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5278033&ar=364039&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Clustering of human cDNA sequences. (A) Estimation of the number of human genes from full-length cDNAs and ESTs. Outline of our gene prediction method from the human full-length cDNAs and ESTs mapped to human genome is schematically shown. For each one of the predicted genes, classification reliability was evaluated manually. (B) Cover rate of FLJ EST sequences and (C) cover rate of FLJ full-length sequenced cDNAs. Results of reliability analysis according to the category based on the cover rates of 1.45 million of ESTs (B) and 55 000 full-length cDNAs (C).</p></div></div><p>To further assess these reliabilities, we next calculated the cover rate of genes using our cDNAs. First, the cover rate was calculated using our 1.45 million FLJ ESTs, and we found a positive correlation between these reliabilities and the cover rate of FLJ ESTs (Fig. <a reveal-id="DSP022F1" data-open="DSP022F1" class="link link-reveal link-table xref-fig">1</a>B). Next, we calculated the cover rate of genes using our 55 000 FLJ human full-length cDNA sequences. In this case, we also found a positive correlation between the reliability and the cover rate similar to that was observed for the ESTs (Fig. <a reveal-id="DSP022F1" data-open="DSP022F1" class="link link-reveal link-table xref-fig">1</a>C). Thus, we were able to verify reliability irrespective of whether we used the sequences of our ESTs or full-length cDNAs in the analysis.</p>                            <h3 scrollto-destination=5278035 id="5278035" class="section-title" data-legacy-id=s3b>3.2. Analysis of AS and functional classification of sequenced full-length cDNAs by GO</h3>
<p>We selected 25 000 full-length cDNAs from among the identified genes by focusing our attention on AS and subsequently sequenced them. In addition, from these cDNAs, we selected 11 769 of human full-length cDNAs in which the ORF regions were predicted to be different from the known full-length cDNAs, and then classified them by GO according to their predicted functions. First, ESTs exhibiting a different splicing pattern than the known full-length cDNAs were selected and were completely sequenced. From the sequence analysis, we were able to predict the ORF regions in only 30% of them (results not shown). Interestingly, a number of cDNA, for which we were unable to predict the ORF region, were thought to produced by AS. But, because our target was to be able to predict the function of the gene from the sequence of its transcript, it was necessary to select protein-coding transcripts efficiently. It is difficult to predict the ORF region correctly from the EST sequences lacking the TSS. However, our 5'-EST sequences not only contained the TSS but also contained sequence information on an average of 500 bases from the TSS. Therefore, we were able to correctly predict the ORF regions of our 5'-EST by using ATGpr.<sup><a reveal-id="DSP022C25" data-open="DSP022C25" class="link link-ref link-reveal xref-bibr">25</a></sup> As a result, the number of clones containing unpredictable ORF regions decreased to ∼10%. Moreover, by using the tools such as TRins<sup><a reveal-id="DSP022C26" data-open="DSP022C26" class="link link-ref link-reveal xref-bibr">26</a></sup> for inspecting the translated region and ALVISION<sup><a reveal-id="DSP022C27" data-open="DSP022C27" class="link link-ref link-reveal xref-bibr">27</a></sup> for evaluating the novelty of amino acid sequences, we succeeded in identifying the ORF regions with high accuracy. Consequently, we obtained 11 769 of human full-length cDNAs in which the ORF regions were predicted to be different from the known full-length cDNAs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Ninety-six percent of these cDNAs-encoded proteins which differed in at least 10 amino acids from those encoded by their respective known full-length cDNAs, mainly because we selected them based on their altered ORF regions as a result of AS. These full-length cDNAs covered 7025 of 23 241 genes that we had originally identified.</p><p>Once it was established that human genes could produce multiple protein-coding transcripts, it was important to analyze their putative functions. The GO classification analysis was performed for all 11 769 our full-length cDNAs using Pfam, and their predicted functions, obtained from this analysis, are summarized in Table <a reveal-id="DSP022TB1" data-open="DSP022TB1" class="link link-reveal link-table xref-fig">1</a>. The classification results revealed that a large number of our cDNA clones were listed under the GO molecular function categories ‘nucleotide binding’, ‘nucleic acid binding’, ‘protein binding’, ‘hydrolase activity’, ‘transferase activity’ and ‘oxidoreductase activity’. Because 11 769 of our full-length cDNAs had ORF regions different from those of the known full-length cDNAs, we also analyzed their functions by predicting domains and motifs using Pfam, SOSUI and SignalP (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Consequently, we discovered full-length cDNAs that encoded proteins with altered functional domains and signal sequences as a result of AS.</p>                        <a id="5278038" scrollto-destination="5278038"></a>
<div content-id="DSP022TB1" class="table-modal"><div class="table-wrap"><div id="DSP022TB1" class="table-wrap-title" data-id="DSP022TB1"><div class="label -label">Table 1</div><div class="caption"><p>Functional classification of the 11 769 full-length cDNAs based on the molecular function hierarchy of GO</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Functional categorization (GO: molecular function) </th><th>Number of matched cDNAs </th></tr></thead><tbody><tr><td>Binding </td><td> </td></tr><tr><td> Nucleotide binding </td><td>681 </td></tr><tr><td> Nucleic acid binding </td><td>341 </td></tr><tr><td> Protein binding </td><td>202 </td></tr><tr><td> Ion binding </td><td>149 </td></tr><tr><td> Lipid binding </td><td>28 </td></tr><tr><td> Tetrapyrrole binding </td><td>27 </td></tr><tr><td> Neurotransmitter binding </td><td>24 </td></tr><tr><td> Carbohydrate binding </td><td>22 </td></tr><tr><td> Other bindings </td><td>57 </td></tr><tr><td>Catalytic activity </td><td> </td></tr><tr><td> Hydrolase activity </td><td>506 </td></tr><tr><td> Transferase activity </td><td>479 </td></tr><tr><td> Oxidoreductase activity </td><td>207 </td></tr><tr><td> Ligase activity </td><td>85 </td></tr><tr><td> Lyase activity </td><td>47 </td></tr><tr><td> Helicase activity </td><td>38 </td></tr><tr><td> Isomerase activity </td><td>26 </td></tr><tr><td> Other catalytic activities </td><td>106 </td></tr><tr><td>Enzyme regulator activity </td><td> </td></tr><tr><td> GTPase regulator activity </td><td>45 </td></tr><tr><td> Enzyme inhibitor activity </td><td>44 </td></tr><tr><td> Other enzyme regulator activities </td><td>21 </td></tr><tr><td>Motor activity </td><td> </td></tr><tr><td> Microtubule motor activity </td><td>24 </td></tr><tr><td> Other motor activities </td><td>20 </td></tr><tr><td>Signal transducer activity </td><td> </td></tr><tr><td> Receptor activity </td><td>124 </td></tr><tr><td> Receptor binding </td><td>25 </td></tr><tr><td> Other signal transducer activities </td><td>40 </td></tr><tr><td>Structural molecule activity </td><td> </td></tr><tr><td> Structural constituent of ribosome </td><td>25 </td></tr><tr><td> Other structural molecule activities </td><td>56 </td></tr><tr><td>Transcription regulator activity </td><td> </td></tr><tr><td> Transcription factor activity </td><td>138 </td></tr><tr><td> Other transcription regulator activities </td><td>39 </td></tr><tr><td>Translation regulator activity </td><td> </td></tr><tr><td> Translation factor activity, nucleic acid binding </td><td>25 </td></tr><tr><td>Transporter activity </td><td> </td></tr><tr><td> Ion transporter activity </td><td>169 </td></tr><tr><td> Carrier activity </td><td>90 </td></tr><tr><td> Channel or pore class transporter activity </td><td>79 </td></tr><tr><td> ATPase activity, coupled to movement of substances </td><td>39 </td></tr><tr><td> Other transporter activities </td><td>131 </td></tr><tr><td>Others </td><td>2 </td></tr><tr><td>Molecular function unknown </td><td>45 </td></tr></tbody></table></div><div content-id="DSP022TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Functional categorization (GO: molecular function) </th><th>Number of matched cDNAs </th></tr></thead><tbody><tr><td>Binding </td><td> </td></tr><tr><td> Nucleotide binding </td><td>681 </td></tr><tr><td> Nucleic acid binding </td><td>341 </td></tr><tr><td> Protein binding </td><td>202 </td></tr><tr><td> Ion binding </td><td>149 </td></tr><tr><td> Lipid binding </td><td>28 </td></tr><tr><td> Tetrapyrrole binding </td><td>27 </td></tr><tr><td> Neurotransmitter binding </td><td>24 </td></tr><tr><td> Carbohydrate binding </td><td>22 </td></tr><tr><td> Other bindings </td><td>57 </td></tr><tr><td>Catalytic activity </td><td> </td></tr><tr><td> Hydrolase activity </td><td>506 </td></tr><tr><td> Transferase activity </td><td>479 </td></tr><tr><td> Oxidoreductase activity </td><td>207 </td></tr><tr><td> Ligase activity </td><td>85 </td></tr><tr><td> Lyase activity </td><td>47 </td></tr><tr><td> Helicase activity </td><td>38 </td></tr><tr><td> Isomerase activity </td><td>26 </td></tr><tr><td> Other catalytic activities </td><td>106 </td></tr><tr><td>Enzyme regulator activity </td><td> </td></tr><tr><td> GTPase regulator activity </td><td>45 </td></tr><tr><td> Enzyme inhibitor activity </td><td>44 </td></tr><tr><td> Other enzyme regulator activities </td><td>21 </td></tr><tr><td>Motor activity </td><td> </td></tr><tr><td> Microtubule motor activity </td><td>24 </td></tr><tr><td> Other motor activities </td><td>20 </td></tr><tr><td>Signal transducer activity </td><td> </td></tr><tr><td> Receptor activity </td><td>124 </td></tr><tr><td> Receptor binding </td><td>25 </td></tr><tr><td> Other signal transducer activities </td><td>40 </td></tr><tr><td>Structural molecule activity </td><td> </td></tr><tr><td> Structural constituent of ribosome </td><td>25 </td></tr><tr><td> Other structural molecule activities </td><td>56 </td></tr><tr><td>Transcription regulator activity </td><td> </td></tr><tr><td> Transcription factor activity </td><td>138 </td></tr><tr><td> Other transcription regulator activities </td><td>39 </td></tr><tr><td>Translation regulator activity </td><td> </td></tr><tr><td> Translation factor activity, nucleic acid binding </td><td>25 </td></tr><tr><td>Transporter activity </td><td> </td></tr><tr><td> Ion transporter activity </td><td>169 </td></tr><tr><td> Carrier activity </td><td>90 </td></tr><tr><td> Channel or pore class transporter activity </td><td>79 </td></tr><tr><td> ATPase activity, coupled to movement of substances </td><td>39 </td></tr><tr><td> Other transporter activities </td><td>131 </td></tr><tr><td>Others </td><td>2 </td></tr><tr><td>Molecular function unknown </td><td>45 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>If a protein was predicted to belong to two or more categories, all categories were included for counting.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5278038" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP022TB1" class="table-wrap-title" data-id="DSP022TB1"><div class="label -label">Table 1</div><div class="caption"><p>Functional classification of the 11 769 full-length cDNAs based on the molecular function hierarchy of GO</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Functional categorization (GO: molecular function) </th><th>Number of matched cDNAs </th></tr></thead><tbody><tr><td>Binding </td><td> </td></tr><tr><td> Nucleotide binding </td><td>681 </td></tr><tr><td> Nucleic acid binding </td><td>341 </td></tr><tr><td> Protein binding </td><td>202 </td></tr><tr><td> Ion binding </td><td>149 </td></tr><tr><td> Lipid binding </td><td>28 </td></tr><tr><td> Tetrapyrrole binding </td><td>27 </td></tr><tr><td> Neurotransmitter binding </td><td>24 </td></tr><tr><td> Carbohydrate binding </td><td>22 </td></tr><tr><td> Other bindings </td><td>57 </td></tr><tr><td>Catalytic activity </td><td> </td></tr><tr><td> Hydrolase activity </td><td>506 </td></tr><tr><td> Transferase activity </td><td>479 </td></tr><tr><td> Oxidoreductase activity </td><td>207 </td></tr><tr><td> Ligase activity </td><td>85 </td></tr><tr><td> Lyase activity </td><td>47 </td></tr><tr><td> Helicase activity </td><td>38 </td></tr><tr><td> Isomerase activity </td><td>26 </td></tr><tr><td> Other catalytic activities </td><td>106 </td></tr><tr><td>Enzyme regulator activity </td><td> </td></tr><tr><td> GTPase regulator activity </td><td>45 </td></tr><tr><td> Enzyme inhibitor activity </td><td>44 </td></tr><tr><td> Other enzyme regulator activities </td><td>21 </td></tr><tr><td>Motor activity </td><td> </td></tr><tr><td> Microtubule motor activity </td><td>24 </td></tr><tr><td> Other motor activities </td><td>20 </td></tr><tr><td>Signal transducer activity </td><td> </td></tr><tr><td> Receptor activity </td><td>124 </td></tr><tr><td> Receptor binding </td><td>25 </td></tr><tr><td> Other signal transducer activities </td><td>40 </td></tr><tr><td>Structural molecule activity </td><td> </td></tr><tr><td> Structural constituent of ribosome </td><td>25 </td></tr><tr><td> Other structural molecule activities </td><td>56 </td></tr><tr><td>Transcription regulator activity </td><td> </td></tr><tr><td> Transcription factor activity </td><td>138 </td></tr><tr><td> Other transcription regulator activities </td><td>39 </td></tr><tr><td>Translation regulator activity </td><td> </td></tr><tr><td> Translation factor activity, nucleic acid binding </td><td>25 </td></tr><tr><td>Transporter activity </td><td> </td></tr><tr><td> Ion transporter activity </td><td>169 </td></tr><tr><td> Carrier activity </td><td>90 </td></tr><tr><td> Channel or pore class transporter activity </td><td>79 </td></tr><tr><td> ATPase activity, coupled to movement of substances </td><td>39 </td></tr><tr><td> Other transporter activities </td><td>131 </td></tr><tr><td>Others </td><td>2 </td></tr><tr><td>Molecular function unknown </td><td>45 </td></tr></tbody></table></div><div content-id="DSP022TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Functional categorization (GO: molecular function) </th><th>Number of matched cDNAs </th></tr></thead><tbody><tr><td>Binding </td><td> </td></tr><tr><td> Nucleotide binding </td><td>681 </td></tr><tr><td> Nucleic acid binding </td><td>341 </td></tr><tr><td> Protein binding </td><td>202 </td></tr><tr><td> Ion binding </td><td>149 </td></tr><tr><td> Lipid binding </td><td>28 </td></tr><tr><td> Tetrapyrrole binding </td><td>27 </td></tr><tr><td> Neurotransmitter binding </td><td>24 </td></tr><tr><td> Carbohydrate binding </td><td>22 </td></tr><tr><td> Other bindings </td><td>57 </td></tr><tr><td>Catalytic activity </td><td> </td></tr><tr><td> Hydrolase activity </td><td>506 </td></tr><tr><td> Transferase activity </td><td>479 </td></tr><tr><td> Oxidoreductase activity </td><td>207 </td></tr><tr><td> Ligase activity </td><td>85 </td></tr><tr><td> Lyase activity </td><td>47 </td></tr><tr><td> Helicase activity </td><td>38 </td></tr><tr><td> Isomerase activity </td><td>26 </td></tr><tr><td> Other catalytic activities </td><td>106 </td></tr><tr><td>Enzyme regulator activity </td><td> </td></tr><tr><td> GTPase regulator activity </td><td>45 </td></tr><tr><td> Enzyme inhibitor activity </td><td>44 </td></tr><tr><td> Other enzyme regulator activities </td><td>21 </td></tr><tr><td>Motor activity </td><td> </td></tr><tr><td> Microtubule motor activity </td><td>24 </td></tr><tr><td> Other motor activities </td><td>20 </td></tr><tr><td>Signal transducer activity </td><td> </td></tr><tr><td> Receptor activity </td><td>124 </td></tr><tr><td> Receptor binding </td><td>25 </td></tr><tr><td> Other signal transducer activities </td><td>40 </td></tr><tr><td>Structural molecule activity </td><td> </td></tr><tr><td> Structural constituent of ribosome </td><td>25 </td></tr><tr><td> Other structural molecule activities </td><td>56 </td></tr><tr><td>Transcription regulator activity </td><td> </td></tr><tr><td> Transcription factor activity </td><td>138 </td></tr><tr><td> Other transcription regulator activities </td><td>39 </td></tr><tr><td>Translation regulator activity </td><td> </td></tr><tr><td> Translation factor activity, nucleic acid binding </td><td>25 </td></tr><tr><td>Transporter activity </td><td> </td></tr><tr><td> Ion transporter activity </td><td>169 </td></tr><tr><td> Carrier activity </td><td>90 </td></tr><tr><td> Channel or pore class transporter activity </td><td>79 </td></tr><tr><td> ATPase activity, coupled to movement of substances </td><td>39 </td></tr><tr><td> Other transporter activities </td><td>131 </td></tr><tr><td>Others </td><td>2 </td></tr><tr><td>Molecular function unknown </td><td>45 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>If a protein was predicted to belong to two or more categories, all categories were included for counting.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5278038" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5278039 id="5278039" class="section-title" data-legacy-id=s3c>3.3. Classification of splicing patterns of full-length cDNAs</h3>
<p>Up until now, majority of the ESTs entered in the public databases were 3'-EST. We succeeded in constructing full-length cDNA libraries efficiently by using the optimized oligo-capping method and obtained ∼1.4 million 5'-ESTs of full-length cDNAs constructed by this method.<sup><a reveal-id="DSP022C18" data-open="DSP022C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSP022C19" data-open="DSP022C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Our 5'-EST sequences were especially useful for the analysis of TSSs because 90% or more of our cDNAs contained the TSSs. We analyzed the splicing patterns of the 11 769 cDNAs by using the 5'-EST sequence data (Fig. <a reveal-id="DSP022F2" data-open="DSP022F2" class="link link-reveal link-table xref-fig">2</a>). Results of this analysis revealed that 3403 cDNAs, which correspond to ∼30% of all cDNAs, were transcribed using alternative TSSs (Type A), and thus, the predicted proteins contained new amino acid sequences at their N-terminal ends. In addition, 1962 cDNAs in Type A (designated as Type A1) contained FEV, due to transcripts originating from a TSS that was previously ignored because it was mapped in an intron region of the genome or transcripts originating from a TSS that was mapped upstream from the one that was analyzed before. Taken together, these results led to the discovery of new exons. We analyzed expression profiles of the genes containing multiple TSSs and discovered that the same gene could code for proteins with diverse function in different tissues by the proper utilization of alternative TSS. There were 8277 cDNAs (i.e. ∼70% of all the full-length cDNAs) that were transcribed from the previously identified TSSs, but contained different ORF region because of AS; they were designated as Type B. Because we used our 5'-EST data for the selection, a lot of Type B cDNAs were predicted to contain N-terminal sequences different from those of the known cDNAs, except for a portion of cDNAs which were either selected by PCR or found during sequencing analysis. To assess whether AS or use of alternative TSS could alter the function of the predicted protein, we compared the GO functional categories of the Type A and Type B (Table <a reveal-id="DSP022TB2" data-open="DSP022TB2" class="link link-reveal link-table xref-fig">2</a>). Our results showed that majority of the Type A belonged mainly to the GO molecular function categories of ‘neurotransmitter binding’, ‘enzyme activator activity’, ‘cyclase activity’, ‘ATPase activity, coupled to movement of substances’ and ‘GTPase regulator activity’. Thus, by using our 5'-EST data, a lot of valuable information were obtained regarding the diversity of TSS and amino acid sequences at the N-terminal ends of proteins. However, since only a portion of the full-length cDNAs was selected for this analysis, information on sequence diversity in regions beyond 500 bases from the TSSs were not obtained. We believe that there are additional alternately spliced transcripts which remained to be analyzed in the future studies.</p>                        <a id="5278041" scrollto-destination="5278041"></a>
<div id="DSP022F2" data-id="DSP022F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/m_dsp02202.jpeg?Expires=2147483647&Signature=q3m1uv8U~kj7YNYjdZ2tiMXtn9mEBqaK5QHYpxyHPFj1kwrtaQE1NqH10k~QSjk6NHZ-FeDtEzu1tzN9zGE3AHV1A6BNuCKp1e0o1bGUPdaTBQxK3YJFJH35ObjkjpUgs1lc7tE751EIfLHAECXplv~6qXN79BxsgBTNJHDJ7U-MXf5Usut41eFJeNAfKJn75VbCaAUZp0dsD3DoYLbn0RmPtATHSwZoc2hVyIV2joDTrdB10uqGHC1-p5CYIhDZwDmwSmVRFQdXf0SZEH3Av~pZSLAgjlRpwSL-gKALSvh5ZaNLEbnC~b4MMlclqv3rFxkjoDBzHLqEsL1NJUC64A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classifications of the 11 769 full-length cDNAs based on splicing patterns. The 11 769 human full-length cDNAs were classified according to their TSS utilization. Type A: these cDNAs were derived from transcripts which were generated utilizing a TSS different than the previously analyzed TSS of the gene. Type A1: cDNAs contained a sequence variation known as FEV. Type A2: this class of cDNAs did not have the FEV feature. Type B: these cDNAs were derived from transcripts that were generated utilizing the same TSS as the previously analyzed TSS, but were found to be alternatively spliced. We could not classify 89 cDNAs because they coded for newly identified proteins." path-from-xml="dsp02202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5278041/dsp02202.jpeg" path-from-xml="dsp02202.jpeg" target="_blank">View large</a><a section="5278041" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/dsp02202.jpeg?Expires=2147483647&Signature=0ZYWMSmnn01pDv0jDPCtOdsD0hB3FkxexAi7KfA6nVRvlji-5P7Ebq-tqE9BGC1ycGgtegOJEJ7B5o0vYcPcBUE5UEG8XZI3EBP1VLbwp0oLxVDeyfZSwU24pwHKDEBNDJoMe4KQRfIooqa6ZBwkS8AzgdNiwwhFMzQ5o99nQperBlTAM3-nmMMfAHW2DLi7y~PThYLvYGP0wvOF6y4EUyl9pb6JRwN0B6onHwEYgIPntLa8DYQ7NBBxL1XksPrQ7ai-jNihO2t6iza~syAz0M4OQPCWLGqFe7kjcoT~MMsgHvViCymGu07YsUFF0b0jFZ-laRMvw3IALcyDGFgu9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5278041&ar=364039&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classifications of the 11 769 full-length cDNAs based on splicing patterns. The 11 769 human full-length cDNAs were classified according to their TSS utilization. Type A: these cDNAs were derived from transcripts which were generated utilizing a TSS different than the previously analyzed TSS of the gene. Type A1: cDNAs contained a sequence variation known as FEV. Type A2: this class of cDNAs did not have the FEV feature. Type B: these cDNAs were derived from transcripts that were generated utilizing the same TSS as the previously analyzed TSS, but were found to be alternatively spliced. We could not classify 89 cDNAs because they coded for newly identified proteins.</p></div></div><div content-id="DSP022F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/m_dsp02202.jpeg?Expires=2147483647&Signature=q3m1uv8U~kj7YNYjdZ2tiMXtn9mEBqaK5QHYpxyHPFj1kwrtaQE1NqH10k~QSjk6NHZ-FeDtEzu1tzN9zGE3AHV1A6BNuCKp1e0o1bGUPdaTBQxK3YJFJH35ObjkjpUgs1lc7tE751EIfLHAECXplv~6qXN79BxsgBTNJHDJ7U-MXf5Usut41eFJeNAfKJn75VbCaAUZp0dsD3DoYLbn0RmPtATHSwZoc2hVyIV2joDTrdB10uqGHC1-p5CYIhDZwDmwSmVRFQdXf0SZEH3Av~pZSLAgjlRpwSL-gKALSvh5ZaNLEbnC~b4MMlclqv3rFxkjoDBzHLqEsL1NJUC64A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classifications of the 11 769 full-length cDNAs based on splicing patterns. The 11 769 human full-length cDNAs were classified according to their TSS utilization. Type A: these cDNAs were derived from transcripts which were generated utilizing a TSS different than the previously analyzed TSS of the gene. Type A1: cDNAs contained a sequence variation known as FEV. Type A2: this class of cDNAs did not have the FEV feature. Type B: these cDNAs were derived from transcripts that were generated utilizing the same TSS as the previously analyzed TSS, but were found to be alternatively spliced. We could not classify 89 cDNAs because they coded for newly identified proteins." path-from-xml="dsp02202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5278041/dsp02202.jpeg" path-from-xml="dsp02202.jpeg" target="_blank">View large</a><a section="5278041" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/dsp02202.jpeg?Expires=2147483647&Signature=0ZYWMSmnn01pDv0jDPCtOdsD0hB3FkxexAi7KfA6nVRvlji-5P7Ebq-tqE9BGC1ycGgtegOJEJ7B5o0vYcPcBUE5UEG8XZI3EBP1VLbwp0oLxVDeyfZSwU24pwHKDEBNDJoMe4KQRfIooqa6ZBwkS8AzgdNiwwhFMzQ5o99nQperBlTAM3-nmMMfAHW2DLi7y~PThYLvYGP0wvOF6y4EUyl9pb6JRwN0B6onHwEYgIPntLa8DYQ7NBBxL1XksPrQ7ai-jNihO2t6iza~syAz0M4OQPCWLGqFe7kjcoT~MMsgHvViCymGu07YsUFF0b0jFZ-laRMvw3IALcyDGFgu9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5278041&ar=364039&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classifications of the 11 769 full-length cDNAs based on splicing patterns. The 11 769 human full-length cDNAs were classified according to their TSS utilization. Type A: these cDNAs were derived from transcripts which were generated utilizing a TSS different than the previously analyzed TSS of the gene. Type A1: cDNAs contained a sequence variation known as FEV. Type A2: this class of cDNAs did not have the FEV feature. Type B: these cDNAs were derived from transcripts that were generated utilizing the same TSS as the previously analyzed TSS, but were found to be alternatively spliced. We could not classify 89 cDNAs because they coded for newly identified proteins.</p></div></div>                        <a id="5278042" scrollto-destination="5278042"></a>
<div content-id="DSP022TB2" class="table-modal"><div class="table-wrap"><div id="DSP022TB2" class="table-wrap-title" data-id="DSP022TB2"><div class="label -label">Table 2</div><div class="caption"><p>Functional classification of two types of splicing patterns of 11 769 full-length cDNAs based on GO category analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Functional categorization (GO: molecular function) </th><th colspan="3">Number of matched cDNAs<hr /> </th></tr><tr><th>Type A (%) </th><th>Type B (%) </th><th>Type A + B </th></tr></thead><tbody><tr><td>Binding </td><td> </td><td> </td><td> </td></tr><tr><td> Lipid binding </td><td>4 (14.3) </td><td>24 (85.7) </td><td>28 </td></tr><tr><td> Tetrapyrrole binding </td><td>5 (18.5) </td><td>22 (81.5) </td><td>27 </td></tr><tr><td> Neurotransmitter binding </td><td>12 (50.0)* </td><td>12 (50.0) </td><td>24 </td></tr><tr><td> Carbohydrate binding </td><td>4 (18.2) </td><td>18 (81.8) </td><td>22 </td></tr><tr><td> Cofactor binding </td><td>3 (16.7) </td><td>15 (83.3) </td><td>18 </td></tr><tr><td> Steroid binding </td><td>1 (10.0) </td><td>9 (90.0) </td><td>10 </td></tr><tr><td>Catalytic activity </td><td> </td><td> </td><td> </td></tr><tr><td> Helicase activity </td><td>4 (10.5) </td><td>34 (89.5) </td><td>38 </td></tr><tr><td> Small protein activating enzyme activity </td><td>2 (18.2) </td><td>9 (81.8) </td><td>11 </td></tr><tr><td> Cyclase activity </td><td>6 (54.5)* </td><td>5 (45.5) </td><td>11 </td></tr><tr><td>Enzyme regulator activity </td><td> </td><td> </td><td> </td></tr><tr><td> GTPase regulator activity </td><td>31 (68.9)* </td><td>14 (31.1) </td><td>45 </td></tr><tr><td> Enzyme activator activity </td><td>6 (50.0)* </td><td>6 (50.0) </td><td>12 </td></tr><tr><td>Structural molecule activity </td><td> </td><td> </td><td> </td></tr><tr><td> Structural constituent of ribosome </td><td>1 (4.0) </td><td>24 (96.0) </td><td>25 </td></tr><tr><td>Transporter activity </td><td> </td><td> </td><td> </td></tr><tr><td> ATPase activity, coupled to movement of substances </td><td>23 (59.0)* </td><td>16 (41.0) </td><td>39 </td></tr><tr><td> Electron transporter activity </td><td>2 (13.3) </td><td>13 (86.7) </td><td>15 </td></tr><tr><td>Total </td><td>1344 (32.0) </td><td>2862 (68.0) </td><td>4206 </td></tr></tbody></table></div><div content-id="DSP022TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Functional categorization (GO: molecular function) </th><th colspan="3">Number of matched cDNAs<hr /> </th></tr><tr><th>Type A (%) </th><th>Type B (%) </th><th>Type A + B </th></tr></thead><tbody><tr><td>Binding </td><td> </td><td> </td><td> </td></tr><tr><td> Lipid binding </td><td>4 (14.3) </td><td>24 (85.7) </td><td>28 </td></tr><tr><td> Tetrapyrrole binding </td><td>5 (18.5) </td><td>22 (81.5) </td><td>27 </td></tr><tr><td> Neurotransmitter binding </td><td>12 (50.0)* </td><td>12 (50.0) </td><td>24 </td></tr><tr><td> Carbohydrate binding </td><td>4 (18.2) </td><td>18 (81.8) </td><td>22 </td></tr><tr><td> Cofactor binding </td><td>3 (16.7) </td><td>15 (83.3) </td><td>18 </td></tr><tr><td> Steroid binding </td><td>1 (10.0) </td><td>9 (90.0) </td><td>10 </td></tr><tr><td>Catalytic activity </td><td> </td><td> </td><td> </td></tr><tr><td> Helicase activity </td><td>4 (10.5) </td><td>34 (89.5) </td><td>38 </td></tr><tr><td> Small protein activating enzyme activity </td><td>2 (18.2) </td><td>9 (81.8) </td><td>11 </td></tr><tr><td> Cyclase activity </td><td>6 (54.5)* </td><td>5 (45.5) </td><td>11 </td></tr><tr><td>Enzyme regulator activity </td><td> </td><td> </td><td> </td></tr><tr><td> GTPase regulator activity </td><td>31 (68.9)* </td><td>14 (31.1) </td><td>45 </td></tr><tr><td> Enzyme activator activity </td><td>6 (50.0)* </td><td>6 (50.0) </td><td>12 </td></tr><tr><td>Structural molecule activity </td><td> </td><td> </td><td> </td></tr><tr><td> Structural constituent of ribosome </td><td>1 (4.0) </td><td>24 (96.0) </td><td>25 </td></tr><tr><td>Transporter activity </td><td> </td><td> </td><td> </td></tr><tr><td> ATPase activity, coupled to movement of substances </td><td>23 (59.0)* </td><td>16 (41.0) </td><td>39 </td></tr><tr><td> Electron transporter activity </td><td>2 (13.3) </td><td>13 (86.7) </td><td>15 </td></tr><tr><td>Total </td><td>1344 (32.0) </td><td>2862 (68.0) </td><td>4206 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The ratio of Type A and Type B is 3:7 as shown by total. Total is all the results of classification in the category of molecular function. If a protein was predicted to belong to two or more categories, all categories were included for counting.</p></div></div><div content-id=""><div class="fn"><p>*Functional categories biased to Type A.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5278042" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP022TB2" class="table-wrap-title" data-id="DSP022TB2"><div class="label -label">Table 2</div><div class="caption"><p>Functional classification of two types of splicing patterns of 11 769 full-length cDNAs based on GO category analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Functional categorization (GO: molecular function) </th><th colspan="3">Number of matched cDNAs<hr /> </th></tr><tr><th>Type A (%) </th><th>Type B (%) </th><th>Type A + B </th></tr></thead><tbody><tr><td>Binding </td><td> </td><td> </td><td> </td></tr><tr><td> Lipid binding </td><td>4 (14.3) </td><td>24 (85.7) </td><td>28 </td></tr><tr><td> Tetrapyrrole binding </td><td>5 (18.5) </td><td>22 (81.5) </td><td>27 </td></tr><tr><td> Neurotransmitter binding </td><td>12 (50.0)* </td><td>12 (50.0) </td><td>24 </td></tr><tr><td> Carbohydrate binding </td><td>4 (18.2) </td><td>18 (81.8) </td><td>22 </td></tr><tr><td> Cofactor binding </td><td>3 (16.7) </td><td>15 (83.3) </td><td>18 </td></tr><tr><td> Steroid binding </td><td>1 (10.0) </td><td>9 (90.0) </td><td>10 </td></tr><tr><td>Catalytic activity </td><td> </td><td> </td><td> </td></tr><tr><td> Helicase activity </td><td>4 (10.5) </td><td>34 (89.5) </td><td>38 </td></tr><tr><td> Small protein activating enzyme activity </td><td>2 (18.2) </td><td>9 (81.8) </td><td>11 </td></tr><tr><td> Cyclase activity </td><td>6 (54.5)* </td><td>5 (45.5) </td><td>11 </td></tr><tr><td>Enzyme regulator activity </td><td> </td><td> </td><td> </td></tr><tr><td> GTPase regulator activity </td><td>31 (68.9)* </td><td>14 (31.1) </td><td>45 </td></tr><tr><td> Enzyme activator activity </td><td>6 (50.0)* </td><td>6 (50.0) </td><td>12 </td></tr><tr><td>Structural molecule activity </td><td> </td><td> </td><td> </td></tr><tr><td> Structural constituent of ribosome </td><td>1 (4.0) </td><td>24 (96.0) </td><td>25 </td></tr><tr><td>Transporter activity </td><td> </td><td> </td><td> </td></tr><tr><td> ATPase activity, coupled to movement of substances </td><td>23 (59.0)* </td><td>16 (41.0) </td><td>39 </td></tr><tr><td> Electron transporter activity </td><td>2 (13.3) </td><td>13 (86.7) </td><td>15 </td></tr><tr><td>Total </td><td>1344 (32.0) </td><td>2862 (68.0) </td><td>4206 </td></tr></tbody></table></div><div content-id="DSP022TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Functional categorization (GO: molecular function) </th><th colspan="3">Number of matched cDNAs<hr /> </th></tr><tr><th>Type A (%) </th><th>Type B (%) </th><th>Type A + B </th></tr></thead><tbody><tr><td>Binding </td><td> </td><td> </td><td> </td></tr><tr><td> Lipid binding </td><td>4 (14.3) </td><td>24 (85.7) </td><td>28 </td></tr><tr><td> Tetrapyrrole binding </td><td>5 (18.5) </td><td>22 (81.5) </td><td>27 </td></tr><tr><td> Neurotransmitter binding </td><td>12 (50.0)* </td><td>12 (50.0) </td><td>24 </td></tr><tr><td> Carbohydrate binding </td><td>4 (18.2) </td><td>18 (81.8) </td><td>22 </td></tr><tr><td> Cofactor binding </td><td>3 (16.7) </td><td>15 (83.3) </td><td>18 </td></tr><tr><td> Steroid binding </td><td>1 (10.0) </td><td>9 (90.0) </td><td>10 </td></tr><tr><td>Catalytic activity </td><td> </td><td> </td><td> </td></tr><tr><td> Helicase activity </td><td>4 (10.5) </td><td>34 (89.5) </td><td>38 </td></tr><tr><td> Small protein activating enzyme activity </td><td>2 (18.2) </td><td>9 (81.8) </td><td>11 </td></tr><tr><td> Cyclase activity </td><td>6 (54.5)* </td><td>5 (45.5) </td><td>11 </td></tr><tr><td>Enzyme regulator activity </td><td> </td><td> </td><td> </td></tr><tr><td> GTPase regulator activity </td><td>31 (68.9)* </td><td>14 (31.1) </td><td>45 </td></tr><tr><td> Enzyme activator activity </td><td>6 (50.0)* </td><td>6 (50.0) </td><td>12 </td></tr><tr><td>Structural molecule activity </td><td> </td><td> </td><td> </td></tr><tr><td> Structural constituent of ribosome </td><td>1 (4.0) </td><td>24 (96.0) </td><td>25 </td></tr><tr><td>Transporter activity </td><td> </td><td> </td><td> </td></tr><tr><td> ATPase activity, coupled to movement of substances </td><td>23 (59.0)* </td><td>16 (41.0) </td><td>39 </td></tr><tr><td> Electron transporter activity </td><td>2 (13.3) </td><td>13 (86.7) </td><td>15 </td></tr><tr><td>Total </td><td>1344 (32.0) </td><td>2862 (68.0) </td><td>4206 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The ratio of Type A and Type B is 3:7 as shown by total. Total is all the results of classification in the category of molecular function. If a protein was predicted to belong to two or more categories, all categories were included for counting.</p></div></div><div content-id=""><div class="fn"><p>*Functional categories biased to Type A.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5278042" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5278043 id="5278043" class="section-title" data-legacy-id=s3d>3.4. Analysis of genes showing tissue-specific expression</h3>
<p>We analyzed expression of genes producing multiple protein-coding transcripts by AS and found that many of these transcripts were expressed in specific tissues or cells, suggesting that the genes likely use this diversity according to the need and situation. We next analyzed expression profiles of 10 069 cDNAs, which corresponded to 5542 genes, out of 11 769 full-length cDNAs we identified in this study. As our cDNA libraries were constructed using RNAs derived from more than 100 different types of tissues and cells, we therefore used the 5′-EST data for analyzing gene expression. We next analyzed gene expression profiles of Type A1 cDNAs containing the FEV diversity and found that the FEVs of 261 cDNAs, which correspond to 155 genes, showed specific expression patterns that were different from those already obtained for the genes with alternative TSSs (Table <a reveal-id="DSP022TB3" data-open="DSP022TB3" class="link link-reveal link-table xref-fig">3</a>). Thus, like the genes with alternative TSSs, the expression patterns of the genes with FEVs likely depended on the tissue and condition. Consequently, we found genes producing multiple protein-coding transcripts by AS.</p>                        <a id="5278045" scrollto-destination="5278045"></a>
<div content-id="DSP022TB3" class="table-modal"><div class="table-wrap"><div id="DSP022TB3" class="table-wrap-title" data-id="DSP022TB3"><div class="label -label">Table 3</div><div class="caption"><p>Expressions of a selected list of 261 FEV-containing cDNAs (155 genes)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th></tr></thead><tbody><tr><td>FLJ50079 </td><td>Brain </td><td>NRK </td><td>FLJ52319 </td><td>Trachea </td><td>GNE </td><td>FLJ55043 </td><td>FB, NT </td><td>PDZRN3 </td><td>FLJ57051 </td><td>Brain </td><td>Pld5 </td></tr><tr><td>FLJ50162 </td><td>Brain </td><td>LARGE1 </td><td>FLJ52354 </td><td>Brain, NT </td><td>CHRNB1_pre </td><td>FLJ55050 </td><td>Brain </td><td>EPS15 </td><td>FLJ57068 </td><td>FB </td><td>FGF13 </td></tr><tr><td>FLJ50199 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ52356 </td><td>Testis </td><td>ARMC4 </td><td>FLJ55194 </td><td>Brain </td><td>Unknown </td><td>FLJ57107 </td><td>Brain, NT </td><td>CHRNB1_pre </td></tr><tr><td>FLJ50365 </td><td>Trachea </td><td>CRISPLD1 </td><td>FLJ52358 </td><td>Testis </td><td>TP73 </td><td>FLJ55226 </td><td>FB </td><td>CHST10 </td><td>FLJ57108 </td><td>Brain </td><td>SNAP91 </td></tr><tr><td>FLJ50390 </td><td>Brain </td><td>GRIA1_pre </td><td>FLJ52367 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ55256 </td><td>Synovial </td><td>TFEC </td><td>FLJ57207 </td><td>Im </td><td>Unknown </td></tr><tr><td>FLJ50398 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ52368 </td><td>Testis, Trachea </td><td>ARMC4 </td><td>FLJ55265 </td><td>Im </td><td>Unknown </td><td>FLJ57232 </td><td>Testis </td><td>PRCP_pre </td></tr><tr><td>FLJ50459 </td><td>Brain </td><td>ETV1 </td><td>FLJ52384 </td><td>Im </td><td>PTPN3 </td><td>FLJ55281 </td><td>Heart, Fetal heart </td><td>SLC5A1 </td><td>FLJ57269 </td><td>Brain </td><td>BTBD10 </td></tr><tr><td>FLJ50460 </td><td>Brain </td><td>DLG4 </td><td>FLJ52407 </td><td>Testis </td><td>CRB1_pre </td><td>FLJ55284 </td><td>FB, NT </td><td>MAGI2 </td><td>FLJ57290 </td><td>Trachea </td><td>CRISPLD1 </td></tr><tr><td>FLJ50484 </td><td>Brain </td><td>SLC26A4 </td><td>FLJ52427 </td><td>Brain </td><td>AMPD3 </td><td>FLJ55338 </td><td>FB </td><td>CLASP1 </td><td>FLJ57298 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50494 </td><td>Brain </td><td>ETV1 </td><td>FLJ52435 </td><td>Testis </td><td>MARCH7 </td><td>FLJ55344 </td><td>Brain </td><td>DYSF </td><td>FLJ57302 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50523 </td><td>Brain </td><td>PEX5L </td><td>FLJ52438 </td><td>Brain </td><td>RIMS1 </td><td>FLJ55381 </td><td>FB </td><td>SLC44A5 </td><td>FLJ57330 </td><td>Brain </td><td>APBB1 </td></tr><tr><td>FLJ50526 </td><td>Brain </td><td>PEX5L </td><td>FLJ52453 </td><td>Testis </td><td>AMPD3 </td><td>FLJ55423 </td><td>Placenta </td><td>NRK </td><td>FLJ57521 </td><td>Tu </td><td>PPFIBP2 </td></tr><tr><td>FLJ50533 </td><td>Brain </td><td>SLC6A9 </td><td>FLJ52496 </td><td>Brain </td><td>TSPAN5 </td><td>FLJ55434 </td><td>Testis </td><td>POMGNT1 </td><td>FLJ57884 </td><td>FB </td><td>FGF13 </td></tr><tr><td>FLJ50539 </td><td>Brain, NT </td><td>DCAMKL1 </td><td>FLJ52520 </td><td>FB </td><td>EOMES </td><td>FLJ55460 </td><td>Brain </td><td>SEMA5B_pre </td><td>FLJ57888 </td><td>Brain </td><td>SGCB </td></tr><tr><td>FLJ50557 </td><td>Brain </td><td>MAP7 </td><td>FLJ52731 </td><td>Brain </td><td>SPRED2 </td><td>FLJ55461 </td><td>NT </td><td>KLHL13 </td><td>FLJ57953 </td><td>Brain </td><td>STAU </td></tr><tr><td>FLJ50577 </td><td>FB </td><td>DLG4 </td><td>FLJ52750 </td><td>Brain </td><td>ARHGEF7 </td><td>FLJ55481 </td><td>NT </td><td>RGMA_pre </td><td>FLJ58008 </td><td>Brain </td><td>PPP2R2B </td></tr><tr><td>FLJ50619 </td><td>NT </td><td>ELAVL4 </td><td>FLJ52810 </td><td>Testis </td><td>GABRB3_pre </td><td>FLJ55495 </td><td>Testis </td><td>PCYT2 </td><td>FLJ58099 </td><td>Brain </td><td>CLTCL1 </td></tr><tr><td>FLJ50623 </td><td>Brain, NT </td><td>DCAMKL1 </td><td>FLJ53109 </td><td>Testis </td><td>PPP2R5E </td><td>FLJ55504 </td><td>Testis </td><td>KLHL13 </td><td>FLJ58366 </td><td>Brain </td><td>RIMS1 </td></tr><tr><td>FLJ50641 </td><td>Brain </td><td>ETV1 </td><td>FLJ53114 </td><td>Testis </td><td>NCAM2_pre </td><td>FLJ55514 </td><td>Brain, Tu </td><td>EGFR_pre </td><td>FLJ58368 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50646 </td><td>FB </td><td>DLG4 </td><td>FLJ53167 </td><td>NT </td><td>CUL4B </td><td>FLJ55516 </td><td>Tu </td><td>LIMS1 </td><td>FLJ58494 </td><td>Brain </td><td>Unknown </td></tr><tr><td>FLJ50725 </td><td>Testis </td><td>ATPAF1 </td><td>FLJ53184 </td><td>Brain </td><td>PPFIA2 </td><td>FLJ55607 </td><td>Brain, Trachea </td><td>HDAC9 </td><td>FLJ58753 </td><td>Brain </td><td>ARHGEF3 </td></tr><tr><td>FLJ50745 </td><td>Testis </td><td>CCNA1 </td><td>FLJ53222 </td><td>FB </td><td>MLLT3 </td><td>FLJ55622 </td><td>Testis </td><td>MMRN1_pre </td><td>FLJ58755 </td><td>Brain </td><td>CHN2 </td></tr><tr><td>FLJ50761 </td><td>Brain </td><td>LRIG1_pre </td><td>FLJ53242 </td><td>Testis </td><td>CLASP1 </td><td>FLJ55627 </td><td>Testis </td><td>MOV10L1 </td><td>FLJ58966 </td><td>Im </td><td>RAB37 </td></tr><tr><td>FLJ50773 </td><td>Brain </td><td>CALB1 </td><td>FLJ53247 </td><td>Testis </td><td>IDE </td><td>FLJ55628 </td><td>Testis </td><td>LOXHD1 </td><td>FLJ59303 </td><td>Brain </td><td>DOCK4 </td></tr><tr><td>FLJ50776 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ53252 </td><td>Testis </td><td>CDH2_pre </td><td>FLJ55641 </td><td>Brain, NT </td><td>JARID2 </td><td>FLJ59333 </td><td>Tu </td><td>RARG </td></tr><tr><td>FLJ50810 </td><td>FB, NT </td><td>MAGI2 </td><td>FLJ53320 </td><td>Brain </td><td>DLGAP1 </td><td>FLJ55662 </td><td>Im </td><td>FGR </td><td>FLJ59338 </td><td>Tu </td><td>RARG </td></tr><tr><td>FLJ50844 </td><td>Brain </td><td>WARS2_pre </td><td>FLJ53324 </td><td>Brain </td><td>TJP2 </td><td>FLJ55664 </td><td>Testis </td><td>NTRK3_pre </td><td>FLJ59345 </td><td>Brain </td><td>PPFIA2 </td></tr><tr><td>FLJ50917 </td><td>Testis </td><td>PCCB_pre </td><td>FLJ53330 </td><td>Brain, NT </td><td>EXOC4 </td><td>FLJ55778 </td><td>Brain </td><td>CLASP1 </td><td>FLJ59425 </td><td>Placenta </td><td>SH3KBP1 </td></tr><tr><td>FLJ50956 </td><td>Brain </td><td>RAPGEF4 </td><td>FLJ53518 </td><td>Testis </td><td>POMGNT1 </td><td>FLJ55834 </td><td>Brain, NT </td><td>FGF11 </td><td>FLJ59496 </td><td>Brain </td><td>CHN2 </td></tr><tr><td>FLJ50959 </td><td>Brain </td><td>RAPGEF4 </td><td>FLJ53578 </td><td>Brain </td><td>Rims1 </td><td>FLJ55856 </td><td>Testis </td><td>ARHGEF3 </td><td>FLJ59502 </td><td>Brain </td><td>PPFIA2 </td></tr><tr><td>FLJ50961 </td><td>Brain </td><td>TMEM16C </td><td>FLJ53606 </td><td>NT </td><td>AKT1 </td><td>FLJ55859 </td><td>Testis </td><td>ST7L </td><td>FLJ59511 </td><td>Brain </td><td>GRIA1_pre </td></tr><tr><td>FLJ50989 </td><td>FB </td><td>EOMES </td><td>FLJ53680 </td><td>Testis </td><td>KIF2C </td><td>FLJ55865 </td><td>Im </td><td>SLC43A2 </td><td>FLJ59545 </td><td>Brain </td><td>EML2 </td></tr><tr><td>FLJ51025 </td><td>Kidney </td><td>NOX4 </td><td>FLJ53829 </td><td>Brain </td><td>APBB1 </td><td>FLJ55903 </td><td>FB </td><td>GPR161 </td><td>FLJ59625 </td><td>Brain </td><td>ARHGEF7 </td></tr><tr><td>FLJ51027 </td><td>Kidney </td><td>NOX4 </td><td>FLJ53875 </td><td>Brain </td><td>APBB1 </td><td>FLJ55905 </td><td>Im </td><td>FGD4 </td><td>FLJ59641 </td><td>Testis </td><td>PPFIA2 </td></tr><tr><td>FLJ51073 </td><td>FB </td><td>EOMES </td><td>FLJ53929 </td><td>Im </td><td>PTPN4 </td><td>FLJ55906 </td><td>Testis </td><td>KIFC3 </td><td>FLJ59648 </td><td>Im </td><td>DYSF </td></tr><tr><td>FLJ51155 </td><td>Testis </td><td>Unknown </td><td>FLJ53980 </td><td>Brain </td><td>PPM1F </td><td>FLJ55918 </td><td>Brain </td><td>EML2 </td><td>FLJ59678 </td><td>Brain </td><td>PEX5L </td></tr><tr><td>FLJ51157 </td><td>Testis </td><td>HDAC4 </td><td>FLJ53990 </td><td>Brain </td><td>GABRB3_pre </td><td>FLJ55961 </td><td>Brain </td><td>GRM4_pre </td><td>FLJ59684 </td><td>Brain </td><td>PLEKHG5 </td></tr><tr><td>FLJ51174 </td><td>Im </td><td>HDAC4 </td><td>FLJ53997 </td><td>Brain </td><td>CTNNA2 </td><td>FLJ55997 </td><td>Brain </td><td>CPNE6 </td><td>FLJ59710 </td><td>Brain </td><td>MCF2 </td></tr><tr><td>FLJ51177 </td><td>Im </td><td>HDAC4 </td><td>FLJ53999 </td><td>Brain </td><td>GAB1 </td><td>FLJ56033 </td><td>Testis </td><td>Unknown </td><td>FLJ59717 </td><td>FB </td><td>TBR1 </td></tr><tr><td>FLJ51210 </td><td>Brain </td><td>KIFC3 </td><td>FLJ54008 </td><td>Brain </td><td>TPCN1 </td><td>FLJ56036 </td><td>Tu </td><td>KIFC3 </td><td>FLJ59769 </td><td>Im </td><td>PLEKHG5 </td></tr><tr><td>FLJ51383 </td><td>Testis </td><td>PPP2R5A </td><td>FLJ54011 </td><td>Brain </td><td>PPFIA2 </td><td>FLJ56037 </td><td>Testis, Prostate </td><td>CUL2 </td><td>FLJ59799 </td><td>Testis </td><td>CTNNA2 </td></tr><tr><td>FLJ51528 </td><td>Im </td><td>BTNL8_pre </td><td>FLJ54016 </td><td>Testis </td><td>DIP13B </td><td>FLJ56038 </td><td>Small intestine </td><td>Unknown </td><td>FLJ59802 </td><td>Testis </td><td>ADCY5 </td></tr><tr><td>FLJ51566 </td><td>Brain </td><td>PDK1 </td><td>FLJ54093 </td><td>Brain </td><td>GPHN </td><td>FLJ56044 </td><td>Brain </td><td>OXR1 </td><td>FLJ59806 </td><td>Im </td><td>HDAC4 </td></tr><tr><td>FLJ51606 </td><td>Trachea </td><td>HABP2_pre </td><td>FLJ54100 </td><td>Brain </td><td>CHN2 </td><td>FLJ56093 </td><td>Brain </td><td>PTPRR_pre </td><td>FLJ60503 </td><td>Brain </td><td>LARGE1 </td></tr><tr><td>FLJ51663 </td><td>Testis </td><td>CPS1_pre </td><td>FLJ54331 </td><td>Brain, Osteoclast </td><td>Unknown </td><td>FLJ56095 </td><td>Brain </td><td>KLHL13 </td><td>FLJ60665 </td><td>Tu </td><td>SLC44A5 </td></tr><tr><td>FLJ51675 </td><td>Brain </td><td>ETV1 </td><td>FLJ54394 </td><td>Testis </td><td>CRB1_pre </td><td>FLJ56110 </td><td>FB </td><td>GOLSYN </td><td>FLJ60667 </td><td>Tu </td><td>SLC44A5 </td></tr><tr><td>FLJ51685 </td><td>Testis </td><td>MCF2 </td><td>FLJ54513 </td><td>Testis </td><td>WDR59 </td><td>FLJ56116 </td><td>FB </td><td>APLP1 </td><td>FLJ60693 </td><td>FB </td><td>PHF21B </td></tr><tr><td>FLJ51695 </td><td>Im </td><td>TP74 </td><td>FLJ54541 </td><td>FB </td><td>EXOC4 </td><td>FLJ56136 </td><td>NT </td><td>SLC2A14 </td><td>FLJ60998 </td><td>Testis </td><td>INPP4B </td></tr><tr><td>FLJ51706 </td><td>Testis </td><td>RAPGEF4 </td><td>FLJ54577 </td><td>NT </td><td>HDAC9 </td><td>FLJ56137 </td><td>Im </td><td>Unknown </td><td>FLJ61124 </td><td>Brain </td><td>RAB37 </td></tr><tr><td>FLJ51734 </td><td>Uterus </td><td>TMEM16C </td><td>FLJ54580 </td><td>NT </td><td>HDAC9 </td><td>FLJ56142 </td><td>NT </td><td>AMOTL2 </td><td>FLJ61133 </td><td>FB </td><td>EXOC4 </td></tr><tr><td>FLJ51737 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ54612 </td><td>Brain </td><td>SH3KBP1 </td><td>FLJ56148 </td><td>Brain </td><td>PLEKHG5 </td><td>FLJ61370 </td><td>FB </td><td>SNCAIP </td></tr><tr><td>FLJ51769 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ54642 </td><td>Brain </td><td>APBB1 </td><td>FLJ56167 </td><td>Testis </td><td>KLHL12 </td><td>FLJ61443 </td><td>Testis </td><td>LARGE1 </td></tr><tr><td>FLJ51805 </td><td>Brain </td><td>RIMS2 </td><td>FLJ54658 </td><td>Brain </td><td>LSAMP_pre </td><td>FLJ56226 </td><td>NT </td><td>SNCAIP </td><td>FLJ61560 </td><td>Trachea </td><td>TJP2 </td></tr><tr><td>FLJ51859 </td><td>Brain </td><td>APBB1 </td><td>FLJ54672 </td><td>Brain </td><td>DOCK4 </td><td>FLJ56370 </td><td>Testis, Prostate </td><td>FKBP8 </td><td>FLJ61674 </td><td>Brain </td><td>PEX5L </td></tr><tr><td>FLJ51873 </td><td>Brain, NT </td><td>AGPS_pre </td><td>FLJ54673 </td><td>Brain </td><td>Unknown </td><td>FLJ56376 </td><td>Brain </td><td>MTMR1 </td><td>FLJ61679 </td><td>Brain </td><td>APBB1 </td></tr><tr><td>FLJ51910 </td><td>FB </td><td>GTPBP3 </td><td>FLJ54674 </td><td>Brain </td><td>TPCN1 </td><td>FLJ56411 </td><td>Brain </td><td>GRIA2_pre </td><td>FLJ53199 </td><td>Brain ↓ </td><td>NEDD4L </td></tr><tr><td>FLJ51934 </td><td>Im </td><td>AOAH_pre </td><td>FLJ54690 </td><td>Brain </td><td>BACE1_pre </td><td>FLJ56420 </td><td>Testis </td><td>DNPEP </td><td>FLJ59993 </td><td>Brain ↓ </td><td>RIMS1 </td></tr><tr><td>FLJ51957 </td><td>NT </td><td>ELAVL4 </td><td>FLJ54693 </td><td>Brain </td><td>BACE1_pre </td><td>FLJ56452 </td><td>Brain </td><td>EML2 </td><td>FLJ55591 </td><td>Brain ↓ </td><td>ARHGEF3 </td></tr><tr><td>FLJ51977 </td><td>Brain </td><td>Unknown </td><td>FLJ54702 </td><td>Brain </td><td>DLGAP1 </td><td>FLJ56634 </td><td>Brain </td><td>GRM4_pre </td><td>FLJ56152 </td><td>Brain ↓ </td><td>ARHGEF7 </td></tr><tr><td>FLJ52027 </td><td>Testis </td><td>ATPAF1 </td><td>FLJ54724 </td><td>FB </td><td>DLG2 </td><td>FLJ56895 </td><td>Testis </td><td>EML2 </td><td>FLJ58411 </td><td>FB ↓ </td><td>CACNB3 </td></tr><tr><td>FLJ52034 </td><td>Im </td><td>Unknown </td><td>FLJ54738 </td><td>Brain </td><td>PDZRN3 </td><td>FLJ56912 </td><td>Uterus </td><td>FBLN2_pre </td><td>FLJ58949 </td><td>FB ↓ </td><td>CACNB3 </td></tr><tr><td>FLJ52037 </td><td>Im </td><td>GRAP2 </td><td>FLJ54742 </td><td>Testis </td><td>Slmap </td><td>FLJ56913 </td><td>Placenta, Uterus </td><td>FBLN2 </td><td>FLJ57810 </td><td>Tu ↓ </td><td>A2ML1 </td></tr><tr><td>FLJ52039 </td><td>Im </td><td>GRAP2 </td><td>FLJ54746 </td><td>NT </td><td>PDZRN3 </td><td>FLJ56957 </td><td>Brain </td><td>TMEM16C </td><td>FLJ53545 </td><td>Tu ↓ </td><td>RARG </td></tr><tr><td>FLJ52041 </td><td>Im </td><td>Unknown </td><td>FLJ54751 </td><td>NT </td><td>SUV420H1 </td><td>FLJ56961 </td><td>Brain </td><td>CLTCL1 </td><td> </td><td> </td><td> </td></tr><tr><td>FLJ52042 </td><td>Im </td><td>GRAP2 </td><td>FLJ54906 </td><td>Trachea </td><td>TMC5 </td><td>FLJ56973 </td><td>Brain </td><td>TMEM16C </td><td> </td><td> </td><td> </td></tr><tr><td>FLJ52288 </td><td>Testis </td><td>ARMC4 </td><td>FLJ54987 </td><td>FB </td><td>PHF21B </td><td>FLJ56979 </td><td>Brain </td><td>MYRIP </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSP022TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th></tr></thead><tbody><tr><td>FLJ50079 </td><td>Brain </td><td>NRK </td><td>FLJ52319 </td><td>Trachea </td><td>GNE </td><td>FLJ55043 </td><td>FB, NT </td><td>PDZRN3 </td><td>FLJ57051 </td><td>Brain </td><td>Pld5 </td></tr><tr><td>FLJ50162 </td><td>Brain </td><td>LARGE1 </td><td>FLJ52354 </td><td>Brain, NT </td><td>CHRNB1_pre </td><td>FLJ55050 </td><td>Brain </td><td>EPS15 </td><td>FLJ57068 </td><td>FB </td><td>FGF13 </td></tr><tr><td>FLJ50199 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ52356 </td><td>Testis </td><td>ARMC4 </td><td>FLJ55194 </td><td>Brain </td><td>Unknown </td><td>FLJ57107 </td><td>Brain, NT </td><td>CHRNB1_pre </td></tr><tr><td>FLJ50365 </td><td>Trachea </td><td>CRISPLD1 </td><td>FLJ52358 </td><td>Testis </td><td>TP73 </td><td>FLJ55226 </td><td>FB </td><td>CHST10 </td><td>FLJ57108 </td><td>Brain </td><td>SNAP91 </td></tr><tr><td>FLJ50390 </td><td>Brain </td><td>GRIA1_pre </td><td>FLJ52367 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ55256 </td><td>Synovial </td><td>TFEC </td><td>FLJ57207 </td><td>Im </td><td>Unknown </td></tr><tr><td>FLJ50398 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ52368 </td><td>Testis, Trachea </td><td>ARMC4 </td><td>FLJ55265 </td><td>Im </td><td>Unknown </td><td>FLJ57232 </td><td>Testis </td><td>PRCP_pre </td></tr><tr><td>FLJ50459 </td><td>Brain </td><td>ETV1 </td><td>FLJ52384 </td><td>Im </td><td>PTPN3 </td><td>FLJ55281 </td><td>Heart, Fetal heart </td><td>SLC5A1 </td><td>FLJ57269 </td><td>Brain </td><td>BTBD10 </td></tr><tr><td>FLJ50460 </td><td>Brain </td><td>DLG4 </td><td>FLJ52407 </td><td>Testis </td><td>CRB1_pre </td><td>FLJ55284 </td><td>FB, NT </td><td>MAGI2 </td><td>FLJ57290 </td><td>Trachea </td><td>CRISPLD1 </td></tr><tr><td>FLJ50484 </td><td>Brain </td><td>SLC26A4 </td><td>FLJ52427 </td><td>Brain </td><td>AMPD3 </td><td>FLJ55338 </td><td>FB </td><td>CLASP1 </td><td>FLJ57298 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50494 </td><td>Brain </td><td>ETV1 </td><td>FLJ52435 </td><td>Testis </td><td>MARCH7 </td><td>FLJ55344 </td><td>Brain </td><td>DYSF </td><td>FLJ57302 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50523 </td><td>Brain </td><td>PEX5L </td><td>FLJ52438 </td><td>Brain </td><td>RIMS1 </td><td>FLJ55381 </td><td>FB </td><td>SLC44A5 </td><td>FLJ57330 </td><td>Brain </td><td>APBB1 </td></tr><tr><td>FLJ50526 </td><td>Brain </td><td>PEX5L </td><td>FLJ52453 </td><td>Testis </td><td>AMPD3 </td><td>FLJ55423 </td><td>Placenta </td><td>NRK </td><td>FLJ57521 </td><td>Tu </td><td>PPFIBP2 </td></tr><tr><td>FLJ50533 </td><td>Brain </td><td>SLC6A9 </td><td>FLJ52496 </td><td>Brain </td><td>TSPAN5 </td><td>FLJ55434 </td><td>Testis </td><td>POMGNT1 </td><td>FLJ57884 </td><td>FB </td><td>FGF13 </td></tr><tr><td>FLJ50539 </td><td>Brain, NT </td><td>DCAMKL1 </td><td>FLJ52520 </td><td>FB </td><td>EOMES </td><td>FLJ55460 </td><td>Brain </td><td>SEMA5B_pre </td><td>FLJ57888 </td><td>Brain </td><td>SGCB </td></tr><tr><td>FLJ50557 </td><td>Brain </td><td>MAP7 </td><td>FLJ52731 </td><td>Brain </td><td>SPRED2 </td><td>FLJ55461 </td><td>NT </td><td>KLHL13 </td><td>FLJ57953 </td><td>Brain </td><td>STAU </td></tr><tr><td>FLJ50577 </td><td>FB </td><td>DLG4 </td><td>FLJ52750 </td><td>Brain </td><td>ARHGEF7 </td><td>FLJ55481 </td><td>NT </td><td>RGMA_pre </td><td>FLJ58008 </td><td>Brain </td><td>PPP2R2B </td></tr><tr><td>FLJ50619 </td><td>NT </td><td>ELAVL4 </td><td>FLJ52810 </td><td>Testis </td><td>GABRB3_pre </td><td>FLJ55495 </td><td>Testis </td><td>PCYT2 </td><td>FLJ58099 </td><td>Brain </td><td>CLTCL1 </td></tr><tr><td>FLJ50623 </td><td>Brain, NT </td><td>DCAMKL1 </td><td>FLJ53109 </td><td>Testis </td><td>PPP2R5E </td><td>FLJ55504 </td><td>Testis </td><td>KLHL13 </td><td>FLJ58366 </td><td>Brain </td><td>RIMS1 </td></tr><tr><td>FLJ50641 </td><td>Brain </td><td>ETV1 </td><td>FLJ53114 </td><td>Testis </td><td>NCAM2_pre </td><td>FLJ55514 </td><td>Brain, Tu </td><td>EGFR_pre </td><td>FLJ58368 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50646 </td><td>FB </td><td>DLG4 </td><td>FLJ53167 </td><td>NT </td><td>CUL4B </td><td>FLJ55516 </td><td>Tu </td><td>LIMS1 </td><td>FLJ58494 </td><td>Brain </td><td>Unknown </td></tr><tr><td>FLJ50725 </td><td>Testis </td><td>ATPAF1 </td><td>FLJ53184 </td><td>Brain </td><td>PPFIA2 </td><td>FLJ55607 </td><td>Brain, Trachea </td><td>HDAC9 </td><td>FLJ58753 </td><td>Brain </td><td>ARHGEF3 </td></tr><tr><td>FLJ50745 </td><td>Testis </td><td>CCNA1 </td><td>FLJ53222 </td><td>FB </td><td>MLLT3 </td><td>FLJ55622 </td><td>Testis </td><td>MMRN1_pre </td><td>FLJ58755 </td><td>Brain </td><td>CHN2 </td></tr><tr><td>FLJ50761 </td><td>Brain </td><td>LRIG1_pre </td><td>FLJ53242 </td><td>Testis </td><td>CLASP1 </td><td>FLJ55627 </td><td>Testis </td><td>MOV10L1 </td><td>FLJ58966 </td><td>Im </td><td>RAB37 </td></tr><tr><td>FLJ50773 </td><td>Brain </td><td>CALB1 </td><td>FLJ53247 </td><td>Testis </td><td>IDE </td><td>FLJ55628 </td><td>Testis </td><td>LOXHD1 </td><td>FLJ59303 </td><td>Brain </td><td>DOCK4 </td></tr><tr><td>FLJ50776 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ53252 </td><td>Testis </td><td>CDH2_pre </td><td>FLJ55641 </td><td>Brain, NT </td><td>JARID2 </td><td>FLJ59333 </td><td>Tu </td><td>RARG </td></tr><tr><td>FLJ50810 </td><td>FB, NT </td><td>MAGI2 </td><td>FLJ53320 </td><td>Brain </td><td>DLGAP1 </td><td>FLJ55662 </td><td>Im </td><td>FGR </td><td>FLJ59338 </td><td>Tu </td><td>RARG </td></tr><tr><td>FLJ50844 </td><td>Brain </td><td>WARS2_pre </td><td>FLJ53324 </td><td>Brain </td><td>TJP2 </td><td>FLJ55664 </td><td>Testis </td><td>NTRK3_pre </td><td>FLJ59345 </td><td>Brain </td><td>PPFIA2 </td></tr><tr><td>FLJ50917 </td><td>Testis </td><td>PCCB_pre </td><td>FLJ53330 </td><td>Brain, NT </td><td>EXOC4 </td><td>FLJ55778 </td><td>Brain </td><td>CLASP1 </td><td>FLJ59425 </td><td>Placenta </td><td>SH3KBP1 </td></tr><tr><td>FLJ50956 </td><td>Brain </td><td>RAPGEF4 </td><td>FLJ53518 </td><td>Testis </td><td>POMGNT1 </td><td>FLJ55834 </td><td>Brain, NT </td><td>FGF11 </td><td>FLJ59496 </td><td>Brain </td><td>CHN2 </td></tr><tr><td>FLJ50959 </td><td>Brain </td><td>RAPGEF4 </td><td>FLJ53578 </td><td>Brain </td><td>Rims1 </td><td>FLJ55856 </td><td>Testis </td><td>ARHGEF3 </td><td>FLJ59502 </td><td>Brain </td><td>PPFIA2 </td></tr><tr><td>FLJ50961 </td><td>Brain </td><td>TMEM16C </td><td>FLJ53606 </td><td>NT </td><td>AKT1 </td><td>FLJ55859 </td><td>Testis </td><td>ST7L </td><td>FLJ59511 </td><td>Brain </td><td>GRIA1_pre </td></tr><tr><td>FLJ50989 </td><td>FB </td><td>EOMES </td><td>FLJ53680 </td><td>Testis </td><td>KIF2C </td><td>FLJ55865 </td><td>Im </td><td>SLC43A2 </td><td>FLJ59545 </td><td>Brain </td><td>EML2 </td></tr><tr><td>FLJ51025 </td><td>Kidney </td><td>NOX4 </td><td>FLJ53829 </td><td>Brain </td><td>APBB1 </td><td>FLJ55903 </td><td>FB </td><td>GPR161 </td><td>FLJ59625 </td><td>Brain </td><td>ARHGEF7 </td></tr><tr><td>FLJ51027 </td><td>Kidney </td><td>NOX4 </td><td>FLJ53875 </td><td>Brain </td><td>APBB1 </td><td>FLJ55905 </td><td>Im </td><td>FGD4 </td><td>FLJ59641 </td><td>Testis </td><td>PPFIA2 </td></tr><tr><td>FLJ51073 </td><td>FB </td><td>EOMES </td><td>FLJ53929 </td><td>Im </td><td>PTPN4 </td><td>FLJ55906 </td><td>Testis </td><td>KIFC3 </td><td>FLJ59648 </td><td>Im </td><td>DYSF </td></tr><tr><td>FLJ51155 </td><td>Testis </td><td>Unknown </td><td>FLJ53980 </td><td>Brain </td><td>PPM1F </td><td>FLJ55918 </td><td>Brain </td><td>EML2 </td><td>FLJ59678 </td><td>Brain </td><td>PEX5L </td></tr><tr><td>FLJ51157 </td><td>Testis </td><td>HDAC4 </td><td>FLJ53990 </td><td>Brain </td><td>GABRB3_pre </td><td>FLJ55961 </td><td>Brain </td><td>GRM4_pre </td><td>FLJ59684 </td><td>Brain </td><td>PLEKHG5 </td></tr><tr><td>FLJ51174 </td><td>Im </td><td>HDAC4 </td><td>FLJ53997 </td><td>Brain </td><td>CTNNA2 </td><td>FLJ55997 </td><td>Brain </td><td>CPNE6 </td><td>FLJ59710 </td><td>Brain </td><td>MCF2 </td></tr><tr><td>FLJ51177 </td><td>Im </td><td>HDAC4 </td><td>FLJ53999 </td><td>Brain </td><td>GAB1 </td><td>FLJ56033 </td><td>Testis </td><td>Unknown </td><td>FLJ59717 </td><td>FB </td><td>TBR1 </td></tr><tr><td>FLJ51210 </td><td>Brain </td><td>KIFC3 </td><td>FLJ54008 </td><td>Brain </td><td>TPCN1 </td><td>FLJ56036 </td><td>Tu </td><td>KIFC3 </td><td>FLJ59769 </td><td>Im </td><td>PLEKHG5 </td></tr><tr><td>FLJ51383 </td><td>Testis </td><td>PPP2R5A </td><td>FLJ54011 </td><td>Brain </td><td>PPFIA2 </td><td>FLJ56037 </td><td>Testis, Prostate </td><td>CUL2 </td><td>FLJ59799 </td><td>Testis </td><td>CTNNA2 </td></tr><tr><td>FLJ51528 </td><td>Im </td><td>BTNL8_pre </td><td>FLJ54016 </td><td>Testis </td><td>DIP13B </td><td>FLJ56038 </td><td>Small intestine </td><td>Unknown </td><td>FLJ59802 </td><td>Testis </td><td>ADCY5 </td></tr><tr><td>FLJ51566 </td><td>Brain </td><td>PDK1 </td><td>FLJ54093 </td><td>Brain </td><td>GPHN </td><td>FLJ56044 </td><td>Brain </td><td>OXR1 </td><td>FLJ59806 </td><td>Im </td><td>HDAC4 </td></tr><tr><td>FLJ51606 </td><td>Trachea </td><td>HABP2_pre </td><td>FLJ54100 </td><td>Brain </td><td>CHN2 </td><td>FLJ56093 </td><td>Brain </td><td>PTPRR_pre </td><td>FLJ60503 </td><td>Brain </td><td>LARGE1 </td></tr><tr><td>FLJ51663 </td><td>Testis </td><td>CPS1_pre </td><td>FLJ54331 </td><td>Brain, Osteoclast </td><td>Unknown </td><td>FLJ56095 </td><td>Brain </td><td>KLHL13 </td><td>FLJ60665 </td><td>Tu </td><td>SLC44A5 </td></tr><tr><td>FLJ51675 </td><td>Brain </td><td>ETV1 </td><td>FLJ54394 </td><td>Testis </td><td>CRB1_pre </td><td>FLJ56110 </td><td>FB </td><td>GOLSYN </td><td>FLJ60667 </td><td>Tu </td><td>SLC44A5 </td></tr><tr><td>FLJ51685 </td><td>Testis </td><td>MCF2 </td><td>FLJ54513 </td><td>Testis </td><td>WDR59 </td><td>FLJ56116 </td><td>FB </td><td>APLP1 </td><td>FLJ60693 </td><td>FB </td><td>PHF21B </td></tr><tr><td>FLJ51695 </td><td>Im </td><td>TP74 </td><td>FLJ54541 </td><td>FB </td><td>EXOC4 </td><td>FLJ56136 </td><td>NT </td><td>SLC2A14 </td><td>FLJ60998 </td><td>Testis </td><td>INPP4B </td></tr><tr><td>FLJ51706 </td><td>Testis </td><td>RAPGEF4 </td><td>FLJ54577 </td><td>NT </td><td>HDAC9 </td><td>FLJ56137 </td><td>Im </td><td>Unknown </td><td>FLJ61124 </td><td>Brain </td><td>RAB37 </td></tr><tr><td>FLJ51734 </td><td>Uterus </td><td>TMEM16C </td><td>FLJ54580 </td><td>NT </td><td>HDAC9 </td><td>FLJ56142 </td><td>NT </td><td>AMOTL2 </td><td>FLJ61133 </td><td>FB </td><td>EXOC4 </td></tr><tr><td>FLJ51737 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ54612 </td><td>Brain </td><td>SH3KBP1 </td><td>FLJ56148 </td><td>Brain </td><td>PLEKHG5 </td><td>FLJ61370 </td><td>FB </td><td>SNCAIP </td></tr><tr><td>FLJ51769 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ54642 </td><td>Brain </td><td>APBB1 </td><td>FLJ56167 </td><td>Testis </td><td>KLHL12 </td><td>FLJ61443 </td><td>Testis </td><td>LARGE1 </td></tr><tr><td>FLJ51805 </td><td>Brain </td><td>RIMS2 </td><td>FLJ54658 </td><td>Brain </td><td>LSAMP_pre </td><td>FLJ56226 </td><td>NT </td><td>SNCAIP </td><td>FLJ61560 </td><td>Trachea </td><td>TJP2 </td></tr><tr><td>FLJ51859 </td><td>Brain </td><td>APBB1 </td><td>FLJ54672 </td><td>Brain </td><td>DOCK4 </td><td>FLJ56370 </td><td>Testis, Prostate </td><td>FKBP8 </td><td>FLJ61674 </td><td>Brain </td><td>PEX5L </td></tr><tr><td>FLJ51873 </td><td>Brain, NT </td><td>AGPS_pre </td><td>FLJ54673 </td><td>Brain </td><td>Unknown </td><td>FLJ56376 </td><td>Brain </td><td>MTMR1 </td><td>FLJ61679 </td><td>Brain </td><td>APBB1 </td></tr><tr><td>FLJ51910 </td><td>FB </td><td>GTPBP3 </td><td>FLJ54674 </td><td>Brain </td><td>TPCN1 </td><td>FLJ56411 </td><td>Brain </td><td>GRIA2_pre </td><td>FLJ53199 </td><td>Brain ↓ </td><td>NEDD4L </td></tr><tr><td>FLJ51934 </td><td>Im </td><td>AOAH_pre </td><td>FLJ54690 </td><td>Brain </td><td>BACE1_pre </td><td>FLJ56420 </td><td>Testis </td><td>DNPEP </td><td>FLJ59993 </td><td>Brain ↓ </td><td>RIMS1 </td></tr><tr><td>FLJ51957 </td><td>NT </td><td>ELAVL4 </td><td>FLJ54693 </td><td>Brain </td><td>BACE1_pre </td><td>FLJ56452 </td><td>Brain </td><td>EML2 </td><td>FLJ55591 </td><td>Brain ↓ </td><td>ARHGEF3 </td></tr><tr><td>FLJ51977 </td><td>Brain </td><td>Unknown </td><td>FLJ54702 </td><td>Brain </td><td>DLGAP1 </td><td>FLJ56634 </td><td>Brain </td><td>GRM4_pre </td><td>FLJ56152 </td><td>Brain ↓ </td><td>ARHGEF7 </td></tr><tr><td>FLJ52027 </td><td>Testis </td><td>ATPAF1 </td><td>FLJ54724 </td><td>FB </td><td>DLG2 </td><td>FLJ56895 </td><td>Testis </td><td>EML2 </td><td>FLJ58411 </td><td>FB ↓ </td><td>CACNB3 </td></tr><tr><td>FLJ52034 </td><td>Im </td><td>Unknown </td><td>FLJ54738 </td><td>Brain </td><td>PDZRN3 </td><td>FLJ56912 </td><td>Uterus </td><td>FBLN2_pre </td><td>FLJ58949 </td><td>FB ↓ </td><td>CACNB3 </td></tr><tr><td>FLJ52037 </td><td>Im </td><td>GRAP2 </td><td>FLJ54742 </td><td>Testis </td><td>Slmap </td><td>FLJ56913 </td><td>Placenta, Uterus </td><td>FBLN2 </td><td>FLJ57810 </td><td>Tu ↓ </td><td>A2ML1 </td></tr><tr><td>FLJ52039 </td><td>Im </td><td>GRAP2 </td><td>FLJ54746 </td><td>NT </td><td>PDZRN3 </td><td>FLJ56957 </td><td>Brain </td><td>TMEM16C </td><td>FLJ53545 </td><td>Tu ↓ </td><td>RARG </td></tr><tr><td>FLJ52041 </td><td>Im </td><td>Unknown </td><td>FLJ54751 </td><td>NT </td><td>SUV420H1 </td><td>FLJ56961 </td><td>Brain </td><td>CLTCL1 </td><td> </td><td> </td><td> </td></tr><tr><td>FLJ52042 </td><td>Im </td><td>GRAP2 </td><td>FLJ54906 </td><td>Trachea </td><td>TMC5 </td><td>FLJ56973 </td><td>Brain </td><td>TMEM16C </td><td> </td><td> </td><td> </td></tr><tr><td>FLJ52288 </td><td>Testis </td><td>ARMC4 </td><td>FLJ54987 </td><td>FB </td><td>PHF21B </td><td>FLJ56979 </td><td>Brain </td><td>MYRIP </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>We analyzed expression profiles of the first exons of ∼1.5 million 5'-ESTs constructed by the oligo-capping method. From this analysis, we selected 261 full-length cDNAs based on the expression levels of their FEVs in specific tissues. Expression levels of cDNAs indicated without any label and with a ‘↓’ label were high and low, respectively, in the respective tissues.</p></div></div><div content-id=""><div class="fn"><p>*NT: NT2 cell induced by retinoic acid; FB, fetal brain; Im, immune tissues; Tu, tumor tissues; pre, precursor; unknown, function unknown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5278045" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP022TB3" class="table-wrap-title" data-id="DSP022TB3"><div class="label -label">Table 3</div><div class="caption"><p>Expressions of a selected list of 261 FEV-containing cDNAs (155 genes)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th></tr></thead><tbody><tr><td>FLJ50079 </td><td>Brain </td><td>NRK </td><td>FLJ52319 </td><td>Trachea </td><td>GNE </td><td>FLJ55043 </td><td>FB, NT </td><td>PDZRN3 </td><td>FLJ57051 </td><td>Brain </td><td>Pld5 </td></tr><tr><td>FLJ50162 </td><td>Brain </td><td>LARGE1 </td><td>FLJ52354 </td><td>Brain, NT </td><td>CHRNB1_pre </td><td>FLJ55050 </td><td>Brain </td><td>EPS15 </td><td>FLJ57068 </td><td>FB </td><td>FGF13 </td></tr><tr><td>FLJ50199 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ52356 </td><td>Testis </td><td>ARMC4 </td><td>FLJ55194 </td><td>Brain </td><td>Unknown </td><td>FLJ57107 </td><td>Brain, NT </td><td>CHRNB1_pre </td></tr><tr><td>FLJ50365 </td><td>Trachea </td><td>CRISPLD1 </td><td>FLJ52358 </td><td>Testis </td><td>TP73 </td><td>FLJ55226 </td><td>FB </td><td>CHST10 </td><td>FLJ57108 </td><td>Brain </td><td>SNAP91 </td></tr><tr><td>FLJ50390 </td><td>Brain </td><td>GRIA1_pre </td><td>FLJ52367 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ55256 </td><td>Synovial </td><td>TFEC </td><td>FLJ57207 </td><td>Im </td><td>Unknown </td></tr><tr><td>FLJ50398 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ52368 </td><td>Testis, Trachea </td><td>ARMC4 </td><td>FLJ55265 </td><td>Im </td><td>Unknown </td><td>FLJ57232 </td><td>Testis </td><td>PRCP_pre </td></tr><tr><td>FLJ50459 </td><td>Brain </td><td>ETV1 </td><td>FLJ52384 </td><td>Im </td><td>PTPN3 </td><td>FLJ55281 </td><td>Heart, Fetal heart </td><td>SLC5A1 </td><td>FLJ57269 </td><td>Brain </td><td>BTBD10 </td></tr><tr><td>FLJ50460 </td><td>Brain </td><td>DLG4 </td><td>FLJ52407 </td><td>Testis </td><td>CRB1_pre </td><td>FLJ55284 </td><td>FB, NT </td><td>MAGI2 </td><td>FLJ57290 </td><td>Trachea </td><td>CRISPLD1 </td></tr><tr><td>FLJ50484 </td><td>Brain </td><td>SLC26A4 </td><td>FLJ52427 </td><td>Brain </td><td>AMPD3 </td><td>FLJ55338 </td><td>FB </td><td>CLASP1 </td><td>FLJ57298 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50494 </td><td>Brain </td><td>ETV1 </td><td>FLJ52435 </td><td>Testis </td><td>MARCH7 </td><td>FLJ55344 </td><td>Brain </td><td>DYSF </td><td>FLJ57302 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50523 </td><td>Brain </td><td>PEX5L </td><td>FLJ52438 </td><td>Brain </td><td>RIMS1 </td><td>FLJ55381 </td><td>FB </td><td>SLC44A5 </td><td>FLJ57330 </td><td>Brain </td><td>APBB1 </td></tr><tr><td>FLJ50526 </td><td>Brain </td><td>PEX5L </td><td>FLJ52453 </td><td>Testis </td><td>AMPD3 </td><td>FLJ55423 </td><td>Placenta </td><td>NRK </td><td>FLJ57521 </td><td>Tu </td><td>PPFIBP2 </td></tr><tr><td>FLJ50533 </td><td>Brain </td><td>SLC6A9 </td><td>FLJ52496 </td><td>Brain </td><td>TSPAN5 </td><td>FLJ55434 </td><td>Testis </td><td>POMGNT1 </td><td>FLJ57884 </td><td>FB </td><td>FGF13 </td></tr><tr><td>FLJ50539 </td><td>Brain, NT </td><td>DCAMKL1 </td><td>FLJ52520 </td><td>FB </td><td>EOMES </td><td>FLJ55460 </td><td>Brain </td><td>SEMA5B_pre </td><td>FLJ57888 </td><td>Brain </td><td>SGCB </td></tr><tr><td>FLJ50557 </td><td>Brain </td><td>MAP7 </td><td>FLJ52731 </td><td>Brain </td><td>SPRED2 </td><td>FLJ55461 </td><td>NT </td><td>KLHL13 </td><td>FLJ57953 </td><td>Brain </td><td>STAU </td></tr><tr><td>FLJ50577 </td><td>FB </td><td>DLG4 </td><td>FLJ52750 </td><td>Brain </td><td>ARHGEF7 </td><td>FLJ55481 </td><td>NT </td><td>RGMA_pre </td><td>FLJ58008 </td><td>Brain </td><td>PPP2R2B </td></tr><tr><td>FLJ50619 </td><td>NT </td><td>ELAVL4 </td><td>FLJ52810 </td><td>Testis </td><td>GABRB3_pre </td><td>FLJ55495 </td><td>Testis </td><td>PCYT2 </td><td>FLJ58099 </td><td>Brain </td><td>CLTCL1 </td></tr><tr><td>FLJ50623 </td><td>Brain, NT </td><td>DCAMKL1 </td><td>FLJ53109 </td><td>Testis </td><td>PPP2R5E </td><td>FLJ55504 </td><td>Testis </td><td>KLHL13 </td><td>FLJ58366 </td><td>Brain </td><td>RIMS1 </td></tr><tr><td>FLJ50641 </td><td>Brain </td><td>ETV1 </td><td>FLJ53114 </td><td>Testis </td><td>NCAM2_pre </td><td>FLJ55514 </td><td>Brain, Tu </td><td>EGFR_pre </td><td>FLJ58368 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50646 </td><td>FB </td><td>DLG4 </td><td>FLJ53167 </td><td>NT </td><td>CUL4B </td><td>FLJ55516 </td><td>Tu </td><td>LIMS1 </td><td>FLJ58494 </td><td>Brain </td><td>Unknown </td></tr><tr><td>FLJ50725 </td><td>Testis </td><td>ATPAF1 </td><td>FLJ53184 </td><td>Brain </td><td>PPFIA2 </td><td>FLJ55607 </td><td>Brain, Trachea </td><td>HDAC9 </td><td>FLJ58753 </td><td>Brain </td><td>ARHGEF3 </td></tr><tr><td>FLJ50745 </td><td>Testis </td><td>CCNA1 </td><td>FLJ53222 </td><td>FB </td><td>MLLT3 </td><td>FLJ55622 </td><td>Testis </td><td>MMRN1_pre </td><td>FLJ58755 </td><td>Brain </td><td>CHN2 </td></tr><tr><td>FLJ50761 </td><td>Brain </td><td>LRIG1_pre </td><td>FLJ53242 </td><td>Testis </td><td>CLASP1 </td><td>FLJ55627 </td><td>Testis </td><td>MOV10L1 </td><td>FLJ58966 </td><td>Im </td><td>RAB37 </td></tr><tr><td>FLJ50773 </td><td>Brain </td><td>CALB1 </td><td>FLJ53247 </td><td>Testis </td><td>IDE </td><td>FLJ55628 </td><td>Testis </td><td>LOXHD1 </td><td>FLJ59303 </td><td>Brain </td><td>DOCK4 </td></tr><tr><td>FLJ50776 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ53252 </td><td>Testis </td><td>CDH2_pre </td><td>FLJ55641 </td><td>Brain, NT </td><td>JARID2 </td><td>FLJ59333 </td><td>Tu </td><td>RARG </td></tr><tr><td>FLJ50810 </td><td>FB, NT </td><td>MAGI2 </td><td>FLJ53320 </td><td>Brain </td><td>DLGAP1 </td><td>FLJ55662 </td><td>Im </td><td>FGR </td><td>FLJ59338 </td><td>Tu </td><td>RARG </td></tr><tr><td>FLJ50844 </td><td>Brain </td><td>WARS2_pre </td><td>FLJ53324 </td><td>Brain </td><td>TJP2 </td><td>FLJ55664 </td><td>Testis </td><td>NTRK3_pre </td><td>FLJ59345 </td><td>Brain </td><td>PPFIA2 </td></tr><tr><td>FLJ50917 </td><td>Testis </td><td>PCCB_pre </td><td>FLJ53330 </td><td>Brain, NT </td><td>EXOC4 </td><td>FLJ55778 </td><td>Brain </td><td>CLASP1 </td><td>FLJ59425 </td><td>Placenta </td><td>SH3KBP1 </td></tr><tr><td>FLJ50956 </td><td>Brain </td><td>RAPGEF4 </td><td>FLJ53518 </td><td>Testis </td><td>POMGNT1 </td><td>FLJ55834 </td><td>Brain, NT </td><td>FGF11 </td><td>FLJ59496 </td><td>Brain </td><td>CHN2 </td></tr><tr><td>FLJ50959 </td><td>Brain </td><td>RAPGEF4 </td><td>FLJ53578 </td><td>Brain </td><td>Rims1 </td><td>FLJ55856 </td><td>Testis </td><td>ARHGEF3 </td><td>FLJ59502 </td><td>Brain </td><td>PPFIA2 </td></tr><tr><td>FLJ50961 </td><td>Brain </td><td>TMEM16C </td><td>FLJ53606 </td><td>NT </td><td>AKT1 </td><td>FLJ55859 </td><td>Testis </td><td>ST7L </td><td>FLJ59511 </td><td>Brain </td><td>GRIA1_pre </td></tr><tr><td>FLJ50989 </td><td>FB </td><td>EOMES </td><td>FLJ53680 </td><td>Testis </td><td>KIF2C </td><td>FLJ55865 </td><td>Im </td><td>SLC43A2 </td><td>FLJ59545 </td><td>Brain </td><td>EML2 </td></tr><tr><td>FLJ51025 </td><td>Kidney </td><td>NOX4 </td><td>FLJ53829 </td><td>Brain </td><td>APBB1 </td><td>FLJ55903 </td><td>FB </td><td>GPR161 </td><td>FLJ59625 </td><td>Brain </td><td>ARHGEF7 </td></tr><tr><td>FLJ51027 </td><td>Kidney </td><td>NOX4 </td><td>FLJ53875 </td><td>Brain </td><td>APBB1 </td><td>FLJ55905 </td><td>Im </td><td>FGD4 </td><td>FLJ59641 </td><td>Testis </td><td>PPFIA2 </td></tr><tr><td>FLJ51073 </td><td>FB </td><td>EOMES </td><td>FLJ53929 </td><td>Im </td><td>PTPN4 </td><td>FLJ55906 </td><td>Testis </td><td>KIFC3 </td><td>FLJ59648 </td><td>Im </td><td>DYSF </td></tr><tr><td>FLJ51155 </td><td>Testis </td><td>Unknown </td><td>FLJ53980 </td><td>Brain </td><td>PPM1F </td><td>FLJ55918 </td><td>Brain </td><td>EML2 </td><td>FLJ59678 </td><td>Brain </td><td>PEX5L </td></tr><tr><td>FLJ51157 </td><td>Testis </td><td>HDAC4 </td><td>FLJ53990 </td><td>Brain </td><td>GABRB3_pre </td><td>FLJ55961 </td><td>Brain </td><td>GRM4_pre </td><td>FLJ59684 </td><td>Brain </td><td>PLEKHG5 </td></tr><tr><td>FLJ51174 </td><td>Im </td><td>HDAC4 </td><td>FLJ53997 </td><td>Brain </td><td>CTNNA2 </td><td>FLJ55997 </td><td>Brain </td><td>CPNE6 </td><td>FLJ59710 </td><td>Brain </td><td>MCF2 </td></tr><tr><td>FLJ51177 </td><td>Im </td><td>HDAC4 </td><td>FLJ53999 </td><td>Brain </td><td>GAB1 </td><td>FLJ56033 </td><td>Testis </td><td>Unknown </td><td>FLJ59717 </td><td>FB </td><td>TBR1 </td></tr><tr><td>FLJ51210 </td><td>Brain </td><td>KIFC3 </td><td>FLJ54008 </td><td>Brain </td><td>TPCN1 </td><td>FLJ56036 </td><td>Tu </td><td>KIFC3 </td><td>FLJ59769 </td><td>Im </td><td>PLEKHG5 </td></tr><tr><td>FLJ51383 </td><td>Testis </td><td>PPP2R5A </td><td>FLJ54011 </td><td>Brain </td><td>PPFIA2 </td><td>FLJ56037 </td><td>Testis, Prostate </td><td>CUL2 </td><td>FLJ59799 </td><td>Testis </td><td>CTNNA2 </td></tr><tr><td>FLJ51528 </td><td>Im </td><td>BTNL8_pre </td><td>FLJ54016 </td><td>Testis </td><td>DIP13B </td><td>FLJ56038 </td><td>Small intestine </td><td>Unknown </td><td>FLJ59802 </td><td>Testis </td><td>ADCY5 </td></tr><tr><td>FLJ51566 </td><td>Brain </td><td>PDK1 </td><td>FLJ54093 </td><td>Brain </td><td>GPHN </td><td>FLJ56044 </td><td>Brain </td><td>OXR1 </td><td>FLJ59806 </td><td>Im </td><td>HDAC4 </td></tr><tr><td>FLJ51606 </td><td>Trachea </td><td>HABP2_pre </td><td>FLJ54100 </td><td>Brain </td><td>CHN2 </td><td>FLJ56093 </td><td>Brain </td><td>PTPRR_pre </td><td>FLJ60503 </td><td>Brain </td><td>LARGE1 </td></tr><tr><td>FLJ51663 </td><td>Testis </td><td>CPS1_pre </td><td>FLJ54331 </td><td>Brain, Osteoclast </td><td>Unknown </td><td>FLJ56095 </td><td>Brain </td><td>KLHL13 </td><td>FLJ60665 </td><td>Tu </td><td>SLC44A5 </td></tr><tr><td>FLJ51675 </td><td>Brain </td><td>ETV1 </td><td>FLJ54394 </td><td>Testis </td><td>CRB1_pre </td><td>FLJ56110 </td><td>FB </td><td>GOLSYN </td><td>FLJ60667 </td><td>Tu </td><td>SLC44A5 </td></tr><tr><td>FLJ51685 </td><td>Testis </td><td>MCF2 </td><td>FLJ54513 </td><td>Testis </td><td>WDR59 </td><td>FLJ56116 </td><td>FB </td><td>APLP1 </td><td>FLJ60693 </td><td>FB </td><td>PHF21B </td></tr><tr><td>FLJ51695 </td><td>Im </td><td>TP74 </td><td>FLJ54541 </td><td>FB </td><td>EXOC4 </td><td>FLJ56136 </td><td>NT </td><td>SLC2A14 </td><td>FLJ60998 </td><td>Testis </td><td>INPP4B </td></tr><tr><td>FLJ51706 </td><td>Testis </td><td>RAPGEF4 </td><td>FLJ54577 </td><td>NT </td><td>HDAC9 </td><td>FLJ56137 </td><td>Im </td><td>Unknown </td><td>FLJ61124 </td><td>Brain </td><td>RAB37 </td></tr><tr><td>FLJ51734 </td><td>Uterus </td><td>TMEM16C </td><td>FLJ54580 </td><td>NT </td><td>HDAC9 </td><td>FLJ56142 </td><td>NT </td><td>AMOTL2 </td><td>FLJ61133 </td><td>FB </td><td>EXOC4 </td></tr><tr><td>FLJ51737 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ54612 </td><td>Brain </td><td>SH3KBP1 </td><td>FLJ56148 </td><td>Brain </td><td>PLEKHG5 </td><td>FLJ61370 </td><td>FB </td><td>SNCAIP </td></tr><tr><td>FLJ51769 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ54642 </td><td>Brain </td><td>APBB1 </td><td>FLJ56167 </td><td>Testis </td><td>KLHL12 </td><td>FLJ61443 </td><td>Testis </td><td>LARGE1 </td></tr><tr><td>FLJ51805 </td><td>Brain </td><td>RIMS2 </td><td>FLJ54658 </td><td>Brain </td><td>LSAMP_pre </td><td>FLJ56226 </td><td>NT </td><td>SNCAIP </td><td>FLJ61560 </td><td>Trachea </td><td>TJP2 </td></tr><tr><td>FLJ51859 </td><td>Brain </td><td>APBB1 </td><td>FLJ54672 </td><td>Brain </td><td>DOCK4 </td><td>FLJ56370 </td><td>Testis, Prostate </td><td>FKBP8 </td><td>FLJ61674 </td><td>Brain </td><td>PEX5L </td></tr><tr><td>FLJ51873 </td><td>Brain, NT </td><td>AGPS_pre </td><td>FLJ54673 </td><td>Brain </td><td>Unknown </td><td>FLJ56376 </td><td>Brain </td><td>MTMR1 </td><td>FLJ61679 </td><td>Brain </td><td>APBB1 </td></tr><tr><td>FLJ51910 </td><td>FB </td><td>GTPBP3 </td><td>FLJ54674 </td><td>Brain </td><td>TPCN1 </td><td>FLJ56411 </td><td>Brain </td><td>GRIA2_pre </td><td>FLJ53199 </td><td>Brain ↓ </td><td>NEDD4L </td></tr><tr><td>FLJ51934 </td><td>Im </td><td>AOAH_pre </td><td>FLJ54690 </td><td>Brain </td><td>BACE1_pre </td><td>FLJ56420 </td><td>Testis </td><td>DNPEP </td><td>FLJ59993 </td><td>Brain ↓ </td><td>RIMS1 </td></tr><tr><td>FLJ51957 </td><td>NT </td><td>ELAVL4 </td><td>FLJ54693 </td><td>Brain </td><td>BACE1_pre </td><td>FLJ56452 </td><td>Brain </td><td>EML2 </td><td>FLJ55591 </td><td>Brain ↓ </td><td>ARHGEF3 </td></tr><tr><td>FLJ51977 </td><td>Brain </td><td>Unknown </td><td>FLJ54702 </td><td>Brain </td><td>DLGAP1 </td><td>FLJ56634 </td><td>Brain </td><td>GRM4_pre </td><td>FLJ56152 </td><td>Brain ↓ </td><td>ARHGEF7 </td></tr><tr><td>FLJ52027 </td><td>Testis </td><td>ATPAF1 </td><td>FLJ54724 </td><td>FB </td><td>DLG2 </td><td>FLJ56895 </td><td>Testis </td><td>EML2 </td><td>FLJ58411 </td><td>FB ↓ </td><td>CACNB3 </td></tr><tr><td>FLJ52034 </td><td>Im </td><td>Unknown </td><td>FLJ54738 </td><td>Brain </td><td>PDZRN3 </td><td>FLJ56912 </td><td>Uterus </td><td>FBLN2_pre </td><td>FLJ58949 </td><td>FB ↓ </td><td>CACNB3 </td></tr><tr><td>FLJ52037 </td><td>Im </td><td>GRAP2 </td><td>FLJ54742 </td><td>Testis </td><td>Slmap </td><td>FLJ56913 </td><td>Placenta, Uterus </td><td>FBLN2 </td><td>FLJ57810 </td><td>Tu ↓ </td><td>A2ML1 </td></tr><tr><td>FLJ52039 </td><td>Im </td><td>GRAP2 </td><td>FLJ54746 </td><td>NT </td><td>PDZRN3 </td><td>FLJ56957 </td><td>Brain </td><td>TMEM16C </td><td>FLJ53545 </td><td>Tu ↓ </td><td>RARG </td></tr><tr><td>FLJ52041 </td><td>Im </td><td>Unknown </td><td>FLJ54751 </td><td>NT </td><td>SUV420H1 </td><td>FLJ56961 </td><td>Brain </td><td>CLTCL1 </td><td> </td><td> </td><td> </td></tr><tr><td>FLJ52042 </td><td>Im </td><td>GRAP2 </td><td>FLJ54906 </td><td>Trachea </td><td>TMC5 </td><td>FLJ56973 </td><td>Brain </td><td>TMEM16C </td><td> </td><td> </td><td> </td></tr><tr><td>FLJ52288 </td><td>Testis </td><td>ARMC4 </td><td>FLJ54987 </td><td>FB </td><td>PHF21B </td><td>FLJ56979 </td><td>Brain </td><td>MYRIP </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSP022TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th><th>FLJ ID </th><th>Specific expression </th><th>Gene symbol </th></tr></thead><tbody><tr><td>FLJ50079 </td><td>Brain </td><td>NRK </td><td>FLJ52319 </td><td>Trachea </td><td>GNE </td><td>FLJ55043 </td><td>FB, NT </td><td>PDZRN3 </td><td>FLJ57051 </td><td>Brain </td><td>Pld5 </td></tr><tr><td>FLJ50162 </td><td>Brain </td><td>LARGE1 </td><td>FLJ52354 </td><td>Brain, NT </td><td>CHRNB1_pre </td><td>FLJ55050 </td><td>Brain </td><td>EPS15 </td><td>FLJ57068 </td><td>FB </td><td>FGF13 </td></tr><tr><td>FLJ50199 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ52356 </td><td>Testis </td><td>ARMC4 </td><td>FLJ55194 </td><td>Brain </td><td>Unknown </td><td>FLJ57107 </td><td>Brain, NT </td><td>CHRNB1_pre </td></tr><tr><td>FLJ50365 </td><td>Trachea </td><td>CRISPLD1 </td><td>FLJ52358 </td><td>Testis </td><td>TP73 </td><td>FLJ55226 </td><td>FB </td><td>CHST10 </td><td>FLJ57108 </td><td>Brain </td><td>SNAP91 </td></tr><tr><td>FLJ50390 </td><td>Brain </td><td>GRIA1_pre </td><td>FLJ52367 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ55256 </td><td>Synovial </td><td>TFEC </td><td>FLJ57207 </td><td>Im </td><td>Unknown </td></tr><tr><td>FLJ50398 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ52368 </td><td>Testis, Trachea </td><td>ARMC4 </td><td>FLJ55265 </td><td>Im </td><td>Unknown </td><td>FLJ57232 </td><td>Testis </td><td>PRCP_pre </td></tr><tr><td>FLJ50459 </td><td>Brain </td><td>ETV1 </td><td>FLJ52384 </td><td>Im </td><td>PTPN3 </td><td>FLJ55281 </td><td>Heart, Fetal heart </td><td>SLC5A1 </td><td>FLJ57269 </td><td>Brain </td><td>BTBD10 </td></tr><tr><td>FLJ50460 </td><td>Brain </td><td>DLG4 </td><td>FLJ52407 </td><td>Testis </td><td>CRB1_pre </td><td>FLJ55284 </td><td>FB, NT </td><td>MAGI2 </td><td>FLJ57290 </td><td>Trachea </td><td>CRISPLD1 </td></tr><tr><td>FLJ50484 </td><td>Brain </td><td>SLC26A4 </td><td>FLJ52427 </td><td>Brain </td><td>AMPD3 </td><td>FLJ55338 </td><td>FB </td><td>CLASP1 </td><td>FLJ57298 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50494 </td><td>Brain </td><td>ETV1 </td><td>FLJ52435 </td><td>Testis </td><td>MARCH7 </td><td>FLJ55344 </td><td>Brain </td><td>DYSF </td><td>FLJ57302 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50523 </td><td>Brain </td><td>PEX5L </td><td>FLJ52438 </td><td>Brain </td><td>RIMS1 </td><td>FLJ55381 </td><td>FB </td><td>SLC44A5 </td><td>FLJ57330 </td><td>Brain </td><td>APBB1 </td></tr><tr><td>FLJ50526 </td><td>Brain </td><td>PEX5L </td><td>FLJ52453 </td><td>Testis </td><td>AMPD3 </td><td>FLJ55423 </td><td>Placenta </td><td>NRK </td><td>FLJ57521 </td><td>Tu </td><td>PPFIBP2 </td></tr><tr><td>FLJ50533 </td><td>Brain </td><td>SLC6A9 </td><td>FLJ52496 </td><td>Brain </td><td>TSPAN5 </td><td>FLJ55434 </td><td>Testis </td><td>POMGNT1 </td><td>FLJ57884 </td><td>FB </td><td>FGF13 </td></tr><tr><td>FLJ50539 </td><td>Brain, NT </td><td>DCAMKL1 </td><td>FLJ52520 </td><td>FB </td><td>EOMES </td><td>FLJ55460 </td><td>Brain </td><td>SEMA5B_pre </td><td>FLJ57888 </td><td>Brain </td><td>SGCB </td></tr><tr><td>FLJ50557 </td><td>Brain </td><td>MAP7 </td><td>FLJ52731 </td><td>Brain </td><td>SPRED2 </td><td>FLJ55461 </td><td>NT </td><td>KLHL13 </td><td>FLJ57953 </td><td>Brain </td><td>STAU </td></tr><tr><td>FLJ50577 </td><td>FB </td><td>DLG4 </td><td>FLJ52750 </td><td>Brain </td><td>ARHGEF7 </td><td>FLJ55481 </td><td>NT </td><td>RGMA_pre </td><td>FLJ58008 </td><td>Brain </td><td>PPP2R2B </td></tr><tr><td>FLJ50619 </td><td>NT </td><td>ELAVL4 </td><td>FLJ52810 </td><td>Testis </td><td>GABRB3_pre </td><td>FLJ55495 </td><td>Testis </td><td>PCYT2 </td><td>FLJ58099 </td><td>Brain </td><td>CLTCL1 </td></tr><tr><td>FLJ50623 </td><td>Brain, NT </td><td>DCAMKL1 </td><td>FLJ53109 </td><td>Testis </td><td>PPP2R5E </td><td>FLJ55504 </td><td>Testis </td><td>KLHL13 </td><td>FLJ58366 </td><td>Brain </td><td>RIMS1 </td></tr><tr><td>FLJ50641 </td><td>Brain </td><td>ETV1 </td><td>FLJ53114 </td><td>Testis </td><td>NCAM2_pre </td><td>FLJ55514 </td><td>Brain, Tu </td><td>EGFR_pre </td><td>FLJ58368 </td><td>Brain </td><td>RAPGEF4 </td></tr><tr><td>FLJ50646 </td><td>FB </td><td>DLG4 </td><td>FLJ53167 </td><td>NT </td><td>CUL4B </td><td>FLJ55516 </td><td>Tu </td><td>LIMS1 </td><td>FLJ58494 </td><td>Brain </td><td>Unknown </td></tr><tr><td>FLJ50725 </td><td>Testis </td><td>ATPAF1 </td><td>FLJ53184 </td><td>Brain </td><td>PPFIA2 </td><td>FLJ55607 </td><td>Brain, Trachea </td><td>HDAC9 </td><td>FLJ58753 </td><td>Brain </td><td>ARHGEF3 </td></tr><tr><td>FLJ50745 </td><td>Testis </td><td>CCNA1 </td><td>FLJ53222 </td><td>FB </td><td>MLLT3 </td><td>FLJ55622 </td><td>Testis </td><td>MMRN1_pre </td><td>FLJ58755 </td><td>Brain </td><td>CHN2 </td></tr><tr><td>FLJ50761 </td><td>Brain </td><td>LRIG1_pre </td><td>FLJ53242 </td><td>Testis </td><td>CLASP1 </td><td>FLJ55627 </td><td>Testis </td><td>MOV10L1 </td><td>FLJ58966 </td><td>Im </td><td>RAB37 </td></tr><tr><td>FLJ50773 </td><td>Brain </td><td>CALB1 </td><td>FLJ53247 </td><td>Testis </td><td>IDE </td><td>FLJ55628 </td><td>Testis </td><td>LOXHD1 </td><td>FLJ59303 </td><td>Brain </td><td>DOCK4 </td></tr><tr><td>FLJ50776 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ53252 </td><td>Testis </td><td>CDH2_pre </td><td>FLJ55641 </td><td>Brain, NT </td><td>JARID2 </td><td>FLJ59333 </td><td>Tu </td><td>RARG </td></tr><tr><td>FLJ50810 </td><td>FB, NT </td><td>MAGI2 </td><td>FLJ53320 </td><td>Brain </td><td>DLGAP1 </td><td>FLJ55662 </td><td>Im </td><td>FGR </td><td>FLJ59338 </td><td>Tu </td><td>RARG </td></tr><tr><td>FLJ50844 </td><td>Brain </td><td>WARS2_pre </td><td>FLJ53324 </td><td>Brain </td><td>TJP2 </td><td>FLJ55664 </td><td>Testis </td><td>NTRK3_pre </td><td>FLJ59345 </td><td>Brain </td><td>PPFIA2 </td></tr><tr><td>FLJ50917 </td><td>Testis </td><td>PCCB_pre </td><td>FLJ53330 </td><td>Brain, NT </td><td>EXOC4 </td><td>FLJ55778 </td><td>Brain </td><td>CLASP1 </td><td>FLJ59425 </td><td>Placenta </td><td>SH3KBP1 </td></tr><tr><td>FLJ50956 </td><td>Brain </td><td>RAPGEF4 </td><td>FLJ53518 </td><td>Testis </td><td>POMGNT1 </td><td>FLJ55834 </td><td>Brain, NT </td><td>FGF11 </td><td>FLJ59496 </td><td>Brain </td><td>CHN2 </td></tr><tr><td>FLJ50959 </td><td>Brain </td><td>RAPGEF4 </td><td>FLJ53578 </td><td>Brain </td><td>Rims1 </td><td>FLJ55856 </td><td>Testis </td><td>ARHGEF3 </td><td>FLJ59502 </td><td>Brain </td><td>PPFIA2 </td></tr><tr><td>FLJ50961 </td><td>Brain </td><td>TMEM16C </td><td>FLJ53606 </td><td>NT </td><td>AKT1 </td><td>FLJ55859 </td><td>Testis </td><td>ST7L </td><td>FLJ59511 </td><td>Brain </td><td>GRIA1_pre </td></tr><tr><td>FLJ50989 </td><td>FB </td><td>EOMES </td><td>FLJ53680 </td><td>Testis </td><td>KIF2C </td><td>FLJ55865 </td><td>Im </td><td>SLC43A2 </td><td>FLJ59545 </td><td>Brain </td><td>EML2 </td></tr><tr><td>FLJ51025 </td><td>Kidney </td><td>NOX4 </td><td>FLJ53829 </td><td>Brain </td><td>APBB1 </td><td>FLJ55903 </td><td>FB </td><td>GPR161 </td><td>FLJ59625 </td><td>Brain </td><td>ARHGEF7 </td></tr><tr><td>FLJ51027 </td><td>Kidney </td><td>NOX4 </td><td>FLJ53875 </td><td>Brain </td><td>APBB1 </td><td>FLJ55905 </td><td>Im </td><td>FGD4 </td><td>FLJ59641 </td><td>Testis </td><td>PPFIA2 </td></tr><tr><td>FLJ51073 </td><td>FB </td><td>EOMES </td><td>FLJ53929 </td><td>Im </td><td>PTPN4 </td><td>FLJ55906 </td><td>Testis </td><td>KIFC3 </td><td>FLJ59648 </td><td>Im </td><td>DYSF </td></tr><tr><td>FLJ51155 </td><td>Testis </td><td>Unknown </td><td>FLJ53980 </td><td>Brain </td><td>PPM1F </td><td>FLJ55918 </td><td>Brain </td><td>EML2 </td><td>FLJ59678 </td><td>Brain </td><td>PEX5L </td></tr><tr><td>FLJ51157 </td><td>Testis </td><td>HDAC4 </td><td>FLJ53990 </td><td>Brain </td><td>GABRB3_pre </td><td>FLJ55961 </td><td>Brain </td><td>GRM4_pre </td><td>FLJ59684 </td><td>Brain </td><td>PLEKHG5 </td></tr><tr><td>FLJ51174 </td><td>Im </td><td>HDAC4 </td><td>FLJ53997 </td><td>Brain </td><td>CTNNA2 </td><td>FLJ55997 </td><td>Brain </td><td>CPNE6 </td><td>FLJ59710 </td><td>Brain </td><td>MCF2 </td></tr><tr><td>FLJ51177 </td><td>Im </td><td>HDAC4 </td><td>FLJ53999 </td><td>Brain </td><td>GAB1 </td><td>FLJ56033 </td><td>Testis </td><td>Unknown </td><td>FLJ59717 </td><td>FB </td><td>TBR1 </td></tr><tr><td>FLJ51210 </td><td>Brain </td><td>KIFC3 </td><td>FLJ54008 </td><td>Brain </td><td>TPCN1 </td><td>FLJ56036 </td><td>Tu </td><td>KIFC3 </td><td>FLJ59769 </td><td>Im </td><td>PLEKHG5 </td></tr><tr><td>FLJ51383 </td><td>Testis </td><td>PPP2R5A </td><td>FLJ54011 </td><td>Brain </td><td>PPFIA2 </td><td>FLJ56037 </td><td>Testis, Prostate </td><td>CUL2 </td><td>FLJ59799 </td><td>Testis </td><td>CTNNA2 </td></tr><tr><td>FLJ51528 </td><td>Im </td><td>BTNL8_pre </td><td>FLJ54016 </td><td>Testis </td><td>DIP13B </td><td>FLJ56038 </td><td>Small intestine </td><td>Unknown </td><td>FLJ59802 </td><td>Testis </td><td>ADCY5 </td></tr><tr><td>FLJ51566 </td><td>Brain </td><td>PDK1 </td><td>FLJ54093 </td><td>Brain </td><td>GPHN </td><td>FLJ56044 </td><td>Brain </td><td>OXR1 </td><td>FLJ59806 </td><td>Im </td><td>HDAC4 </td></tr><tr><td>FLJ51606 </td><td>Trachea </td><td>HABP2_pre </td><td>FLJ54100 </td><td>Brain </td><td>CHN2 </td><td>FLJ56093 </td><td>Brain </td><td>PTPRR_pre </td><td>FLJ60503 </td><td>Brain </td><td>LARGE1 </td></tr><tr><td>FLJ51663 </td><td>Testis </td><td>CPS1_pre </td><td>FLJ54331 </td><td>Brain, Osteoclast </td><td>Unknown </td><td>FLJ56095 </td><td>Brain </td><td>KLHL13 </td><td>FLJ60665 </td><td>Tu </td><td>SLC44A5 </td></tr><tr><td>FLJ51675 </td><td>Brain </td><td>ETV1 </td><td>FLJ54394 </td><td>Testis </td><td>CRB1_pre </td><td>FLJ56110 </td><td>FB </td><td>GOLSYN </td><td>FLJ60667 </td><td>Tu </td><td>SLC44A5 </td></tr><tr><td>FLJ51685 </td><td>Testis </td><td>MCF2 </td><td>FLJ54513 </td><td>Testis </td><td>WDR59 </td><td>FLJ56116 </td><td>FB </td><td>APLP1 </td><td>FLJ60693 </td><td>FB </td><td>PHF21B </td></tr><tr><td>FLJ51695 </td><td>Im </td><td>TP74 </td><td>FLJ54541 </td><td>FB </td><td>EXOC4 </td><td>FLJ56136 </td><td>NT </td><td>SLC2A14 </td><td>FLJ60998 </td><td>Testis </td><td>INPP4B </td></tr><tr><td>FLJ51706 </td><td>Testis </td><td>RAPGEF4 </td><td>FLJ54577 </td><td>NT </td><td>HDAC9 </td><td>FLJ56137 </td><td>Im </td><td>Unknown </td><td>FLJ61124 </td><td>Brain </td><td>RAB37 </td></tr><tr><td>FLJ51734 </td><td>Uterus </td><td>TMEM16C </td><td>FLJ54580 </td><td>NT </td><td>HDAC9 </td><td>FLJ56142 </td><td>NT </td><td>AMOTL2 </td><td>FLJ61133 </td><td>FB </td><td>EXOC4 </td></tr><tr><td>FLJ51737 </td><td>Brain </td><td>ARHGEF6 </td><td>FLJ54612 </td><td>Brain </td><td>SH3KBP1 </td><td>FLJ56148 </td><td>Brain </td><td>PLEKHG5 </td><td>FLJ61370 </td><td>FB </td><td>SNCAIP </td></tr><tr><td>FLJ51769 </td><td>Testis </td><td>IQGAP2 </td><td>FLJ54642 </td><td>Brain </td><td>APBB1 </td><td>FLJ56167 </td><td>Testis </td><td>KLHL12 </td><td>FLJ61443 </td><td>Testis </td><td>LARGE1 </td></tr><tr><td>FLJ51805 </td><td>Brain </td><td>RIMS2 </td><td>FLJ54658 </td><td>Brain </td><td>LSAMP_pre </td><td>FLJ56226 </td><td>NT </td><td>SNCAIP </td><td>FLJ61560 </td><td>Trachea </td><td>TJP2 </td></tr><tr><td>FLJ51859 </td><td>Brain </td><td>APBB1 </td><td>FLJ54672 </td><td>Brain </td><td>DOCK4 </td><td>FLJ56370 </td><td>Testis, Prostate </td><td>FKBP8 </td><td>FLJ61674 </td><td>Brain </td><td>PEX5L </td></tr><tr><td>FLJ51873 </td><td>Brain, NT </td><td>AGPS_pre </td><td>FLJ54673 </td><td>Brain </td><td>Unknown </td><td>FLJ56376 </td><td>Brain </td><td>MTMR1 </td><td>FLJ61679 </td><td>Brain </td><td>APBB1 </td></tr><tr><td>FLJ51910 </td><td>FB </td><td>GTPBP3 </td><td>FLJ54674 </td><td>Brain </td><td>TPCN1 </td><td>FLJ56411 </td><td>Brain </td><td>GRIA2_pre </td><td>FLJ53199 </td><td>Brain ↓ </td><td>NEDD4L </td></tr><tr><td>FLJ51934 </td><td>Im </td><td>AOAH_pre </td><td>FLJ54690 </td><td>Brain </td><td>BACE1_pre </td><td>FLJ56420 </td><td>Testis </td><td>DNPEP </td><td>FLJ59993 </td><td>Brain ↓ </td><td>RIMS1 </td></tr><tr><td>FLJ51957 </td><td>NT </td><td>ELAVL4 </td><td>FLJ54693 </td><td>Brain </td><td>BACE1_pre </td><td>FLJ56452 </td><td>Brain </td><td>EML2 </td><td>FLJ55591 </td><td>Brain ↓ </td><td>ARHGEF3 </td></tr><tr><td>FLJ51977 </td><td>Brain </td><td>Unknown </td><td>FLJ54702 </td><td>Brain </td><td>DLGAP1 </td><td>FLJ56634 </td><td>Brain </td><td>GRM4_pre </td><td>FLJ56152 </td><td>Brain ↓ </td><td>ARHGEF7 </td></tr><tr><td>FLJ52027 </td><td>Testis </td><td>ATPAF1 </td><td>FLJ54724 </td><td>FB </td><td>DLG2 </td><td>FLJ56895 </td><td>Testis </td><td>EML2 </td><td>FLJ58411 </td><td>FB ↓ </td><td>CACNB3 </td></tr><tr><td>FLJ52034 </td><td>Im </td><td>Unknown </td><td>FLJ54738 </td><td>Brain </td><td>PDZRN3 </td><td>FLJ56912 </td><td>Uterus </td><td>FBLN2_pre </td><td>FLJ58949 </td><td>FB ↓ </td><td>CACNB3 </td></tr><tr><td>FLJ52037 </td><td>Im </td><td>GRAP2 </td><td>FLJ54742 </td><td>Testis </td><td>Slmap </td><td>FLJ56913 </td><td>Placenta, Uterus </td><td>FBLN2 </td><td>FLJ57810 </td><td>Tu ↓ </td><td>A2ML1 </td></tr><tr><td>FLJ52039 </td><td>Im </td><td>GRAP2 </td><td>FLJ54746 </td><td>NT </td><td>PDZRN3 </td><td>FLJ56957 </td><td>Brain </td><td>TMEM16C </td><td>FLJ53545 </td><td>Tu ↓ </td><td>RARG </td></tr><tr><td>FLJ52041 </td><td>Im </td><td>Unknown </td><td>FLJ54751 </td><td>NT </td><td>SUV420H1 </td><td>FLJ56961 </td><td>Brain </td><td>CLTCL1 </td><td> </td><td> </td><td> </td></tr><tr><td>FLJ52042 </td><td>Im </td><td>GRAP2 </td><td>FLJ54906 </td><td>Trachea </td><td>TMC5 </td><td>FLJ56973 </td><td>Brain </td><td>TMEM16C </td><td> </td><td> </td><td> </td></tr><tr><td>FLJ52288 </td><td>Testis </td><td>ARMC4 </td><td>FLJ54987 </td><td>FB </td><td>PHF21B </td><td>FLJ56979 </td><td>Brain </td><td>MYRIP </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>We analyzed expression profiles of the first exons of ∼1.5 million 5'-ESTs constructed by the oligo-capping method. From this analysis, we selected 261 full-length cDNAs based on the expression levels of their FEVs in specific tissues. Expression levels of cDNAs indicated without any label and with a ‘↓’ label were high and low, respectively, in the respective tissues.</p></div></div><div content-id=""><div class="fn"><p>*NT: NT2 cell induced by retinoic acid; FB, fetal brain; Im, immune tissues; Tu, tumor tissues; pre, precursor; unknown, function unknown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5278045" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5278046 id="5278046" class="section-title" data-legacy-id=s3e>3.5. Analysis of expression patterns of tissue-specific expressed genes</h3>
<p>We quantified tissue-specific expressions of 13 out of 261 selected cDNAs by real-time PCR (Fig. <a reveal-id="DSP022F3" data-open="DSP022F3" class="link link-reveal link-table xref-fig">3</a>). Results of our analysis especially suggested that there was a strong relationship between the tissue-specific expression and diversity of gene function or disease. We compared the expression profile of a specific gene by utilizing the TSS identified in this study with that of the same gene in which a previously identified TSS was utilized for expression. These results are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a> and are discussed below in more detail.</p>                        <a id="5278048" scrollto-destination="5278048"></a>
<div id="DSP022F3" data-id="DSP022F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/m_dsp02203.jpeg?Expires=2147483647&Signature=s-z7Nm36fiOq8rbd-np1XVIvh8OJ4twv0-Tc1WOV88iJB7W5Blbv1PqlC57L8kAHf7I8-UE7rTvPQ5I-M8FaPaBBeoxMM-iRRDkmsgq9qntGB2m~GYrAUED11h2vh9z8n7AMH85fK0Q-RCEoMapwNNqECyKHgCxlpU7XHCthwv33mTu6hGRI8FTj8x6IieIHmJQdVSCXOTurfZ4f~VC9eCBG0dDA6xs6k96E5iuEAUohGGVGmEZNeeAyjCFirU-z22FWHw3qxbSb4Tk-od45R8RFT6aUEcyoncriH9SnADz5UEMC63yJdvkv3au0cRSmsWNll0sZOLSjWGpyMf6IIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quantitative evaluation of selected genes by real-time PCR. Expression levels of the first exon regions of the selected genes were analyzed by real-time PCR. The data were normalized with respect to that of the human GAPDH as described in the Materials and methods section. The expression levels of genes were represented in log10 base. Expression levels of cDNAs labeled ‘$$’ represent the very low expression level or undetected. (A) FGF13, (B) OXR1, (C) C6orf142, (D) PLD5, (E) FGD4, (F) C6orf32. BW, brain, whole; BC, brain, cerebellum; BF, fetal brain; SP, spleen; BM, bone marrow; TH, thymus; OV, ovary; PR, prostate; UT, uterus; MT, mixture of tumor human tissues; MN, control, mixture of normal human tissues; KT, kidney tumor; LT, lung tumor." path-from-xml="dsp02203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5278048/dsp02203.jpeg" path-from-xml="dsp02203.jpeg" target="_blank">View large</a><a section="5278048" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/dsp02203.jpeg?Expires=2147483647&Signature=47uMW3fcUKGT9kixz~aqC96x069zrgWI0KtHB6cd0h1PjaMHHmcnM9iCizTb4EryMfCo0FU90jbRJmdnMb5DGAorDkS6shwC4-488mc3Kit-y30Iu0qmy37yEEjpgEx75EIdgfzTHGVAeoVZYGunphwju1dZtJbrSnkYBinYVq3RfbSAkBCfCD8NL0NkAqKq4y~OTMVG5p14rAywvsMRz6fTOXO-iBBd~zjgr2rQnvHv7YwNR2HYsEnZPlwWj02-LuAdyFeP97croP7ypwPaLxJw-py-V0DuaUmE62dE~VHmyyxGaUB-GmA-sKkQNELnSyHVQQQWQAt1mjB6MiEChw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5278048&ar=364039&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Quantitative evaluation of selected genes by real-time PCR. Expression levels of the first exon regions of the selected genes were analyzed by real-time PCR. The data were normalized with respect to that of the human GAPDH as described in the Materials and methods section. The expression levels of genes were represented in log<sub>10</sub> base. Expression levels of cDNAs labeled ‘$$’ represent the very low expression level or undetected. (A) FGF13, (B) OXR1, (C) C6orf142, (D) PLD5, (E) FGD4, (F) C6orf32. BW, brain, whole; BC, brain, cerebellum; BF, fetal brain; SP, spleen; BM, bone marrow; TH, thymus; OV, ovary; PR, prostate; UT, uterus; MT, mixture of tumor human tissues; MN, control, mixture of normal human tissues; KT, kidney tumor; LT, lung tumor.</p></div></div><div content-id="DSP022F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/m_dsp02203.jpeg?Expires=2147483647&Signature=s-z7Nm36fiOq8rbd-np1XVIvh8OJ4twv0-Tc1WOV88iJB7W5Blbv1PqlC57L8kAHf7I8-UE7rTvPQ5I-M8FaPaBBeoxMM-iRRDkmsgq9qntGB2m~GYrAUED11h2vh9z8n7AMH85fK0Q-RCEoMapwNNqECyKHgCxlpU7XHCthwv33mTu6hGRI8FTj8x6IieIHmJQdVSCXOTurfZ4f~VC9eCBG0dDA6xs6k96E5iuEAUohGGVGmEZNeeAyjCFirU-z22FWHw3qxbSb4Tk-od45R8RFT6aUEcyoncriH9SnADz5UEMC63yJdvkv3au0cRSmsWNll0sZOLSjWGpyMf6IIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quantitative evaluation of selected genes by real-time PCR. Expression levels of the first exon regions of the selected genes were analyzed by real-time PCR. The data were normalized with respect to that of the human GAPDH as described in the Materials and methods section. The expression levels of genes were represented in log10 base. Expression levels of cDNAs labeled ‘$$’ represent the very low expression level or undetected. (A) FGF13, (B) OXR1, (C) C6orf142, (D) PLD5, (E) FGD4, (F) C6orf32. BW, brain, whole; BC, brain, cerebellum; BF, fetal brain; SP, spleen; BM, bone marrow; TH, thymus; OV, ovary; PR, prostate; UT, uterus; MT, mixture of tumor human tissues; MN, control, mixture of normal human tissues; KT, kidney tumor; LT, lung tumor." path-from-xml="dsp02203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5278048/dsp02203.jpeg" path-from-xml="dsp02203.jpeg" target="_blank">View large</a><a section="5278048" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/dsp02203.jpeg?Expires=2147483647&Signature=47uMW3fcUKGT9kixz~aqC96x069zrgWI0KtHB6cd0h1PjaMHHmcnM9iCizTb4EryMfCo0FU90jbRJmdnMb5DGAorDkS6shwC4-488mc3Kit-y30Iu0qmy37yEEjpgEx75EIdgfzTHGVAeoVZYGunphwju1dZtJbrSnkYBinYVq3RfbSAkBCfCD8NL0NkAqKq4y~OTMVG5p14rAywvsMRz6fTOXO-iBBd~zjgr2rQnvHv7YwNR2HYsEnZPlwWj02-LuAdyFeP97croP7ypwPaLxJw-py-V0DuaUmE62dE~VHmyyxGaUB-GmA-sKkQNELnSyHVQQQWQAt1mjB6MiEChw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5278048&ar=364039&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Quantitative evaluation of selected genes by real-time PCR. Expression levels of the first exon regions of the selected genes were analyzed by real-time PCR. The data were normalized with respect to that of the human GAPDH as described in the Materials and methods section. The expression levels of genes were represented in log<sub>10</sub> base. Expression levels of cDNAs labeled ‘$$’ represent the very low expression level or undetected. (A) FGF13, (B) OXR1, (C) C6orf142, (D) PLD5, (E) FGD4, (F) C6orf32. BW, brain, whole; BC, brain, cerebellum; BF, fetal brain; SP, spleen; BM, bone marrow; TH, thymus; OV, ovary; PR, prostate; UT, uterus; MT, mixture of tumor human tissues; MN, control, mixture of normal human tissues; KT, kidney tumor; LT, lung tumor.</p></div></div><p>First example, FGF13 is a gene that belongs to the FGF family and is believed to play roles in cell proliferation and differentiation, and also in neuronal differentiation.<sup><a reveal-id="DSP022C28" data-open="DSP022C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSP022C29" data-open="DSP022C29" class="link link-ref link-reveal xref-bibr">29</a></sup> FLJ57884 and FLJ57068 cDNAs exhibited different ORF regions as a result of FEV and were splicing variants of the known FGF13 cDNA. The TSSs we found in each one of them were located upstream from the TSS of FGF13. Whereas the known TSS of FGF13 was expressed highly in both fetal and adult brains, the TSSs of both FLJ57884 and FLJ57068 cDNAs were highly expressed only in the fetal brain. Moreover, the TSS of our FLJ57068 cDNA was also expressed highly in the kidney cancer (Fig. <a reveal-id="DSP022F3" data-open="DSP022F3" class="link link-reveal link-table xref-fig">3</a>A). Second example, OXR1 is one of the oxidation stress receptivity genes localized in mitochondria.<sup><a reveal-id="DSP022C30" data-open="DSP022C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The TSS of known OXR1 was expressed at equal levels in various tissues. But the TSS we identified in the FLJ56044 cDNA was located upstream from the known TSS of OXR1 and was highly expressed in brain, kidney cancer and lung cancer (Fig. <a reveal-id="DSP022F3" data-open="DSP022F3" class="link link-reveal link-table xref-fig">3</a>B). Thus, these results suggested that these two genes were using different TSSs to regulate their expression levels in the brain. Moreover, our results also suggest that, for both genes, only one of the TSSs was preferentially recognized by the transcription machinery in the cancerous tissue.</p><p>Third example, C6orf142 (chromosome 6 ORF 142) is a gene of an unknown function. The known TSS of C6orf142 was highly expressed in the heart. However, the TSS we identified in the FLJ58494 cDNA, which was located downstream from the previously identified TSS of C6orf142, was highly expressed in both fetal and adult brains (Fig. <a reveal-id="DSP022F3" data-open="DSP022F3" class="link link-reveal link-table xref-fig">3</a>C). Fourth example, PLD5 is one of the phospholipid-splitting enzymes presumably involved in the intracellular signaling.<sup><a reveal-id="DSP022C31" data-open="DSP022C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Although the known TSS of PLD5 was expressed equally in various tissues, the TSS we identified in the FLJ57051 cDNA, which was located downstream of the previously identified TSS of PLD5, was highly expressed in the brain (Fig. <a reveal-id="DSP022F3" data-open="DSP022F3" class="link link-reveal link-table xref-fig">3</a>D). Fifth example, SPRED2 is a Ras inhibitory factor belonging to the Sprouty/Spred family.<sup><a reveal-id="DSP022C32" data-open="DSP022C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The TSS we identified in the FLJ52731 cDNA, which was located downstream from the known TSS of SPRED2, was expressed highly in the brain (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Sixth example, SEMA5B is a nerve guidance factor which is involved in organogenesis, angiogenesis and oncogenesis.<sup><a reveal-id="DSP022C33" data-open="DSP022C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The TSS we identified in the FLJ55460 cDNA, which was located downstream from the known TSS of SEMA5B, also was expressed highly in the brain (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Seventh example, CACNB3 is a calcium channel beta-3 subunit, which is involved in modifying sympathetic nervous system, olfaction and control of blood pressure.<sup><a reveal-id="DSP022C34" data-open="DSP022C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Although the known TSS of CACNB3 was expressed highly in both fetal and adult brains, the newly identified TSSs of FLJ58949 and FLJ58411 cDNAs, both of which were located downstream from the known TSS of CACNB3, were expressed at a low level in the brain (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). These cDNAs exhibited different ORF regions as a result of AS. Eighth example, BACE1 is a peptide hydrolase that cleaves the amyloid precursor protein and is one of the factors involved in Alzheimer's disease.<sup><a reveal-id="DSP022C35" data-open="DSP022C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The known TSS of BACE1 was expressed equally in various tissues. However, the TSS we identified in the FLJ54690 cDNA, which was located downstream from the known TSS of BACE1, was expressed highly in the brain (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Thus, these six genes regulated their expression levels in the brain using a specific TSS in each gene.</p><p>Ninth example, FGD4 is a gene that seemed to be involved in the regulation of the actin in the cytoskeleton and cell shape and also have various roles in proliferation, differentiation, transcriptional regulation and development.<sup><a reveal-id="DSP022C36" data-open="DSP022C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The known TSS of FGD4 was highly expressed in the nervous system tissues such as brain, spinal cord and testis. However, the TSS we identified in the FLJ55905 cDNA, which was located downstream from the known TSS of FGD4, was highly expressed in the immune system tissues such as bone marrow and spleen (Fig. <a reveal-id="DSP022F3" data-open="DSP022F3" class="link link-reveal link-table xref-fig">3</a>E). Tenth example, C6orf32 is a gene of unknown function whose expression level increased during the myoblast differentiation of the embryo.<sup><a reveal-id="DSP022C37" data-open="DSP022C37" class="link link-ref link-reveal xref-bibr">37</a></sup> FLJ56038 and FLJ56137 cDNAs exhibited different ORF regions as a result of FEV and were splicing variants of the known C6orf32 cDNA. The known TSS of C6orf32 was expressed at equal levels in various tissues. However, the TSSs we found in FLJ56038 and FLJ56137 cDNAs were located upstream of the known TSS of C6orf32, and both of these newly identified TSSs were highly expressed in the immune system tissues such as bone marrow, spleen and thymus (Fig. <a reveal-id="DSP022F3" data-open="DSP022F3" class="link link-reveal link-table xref-fig">3</a>F). Eleventh example, PTPN4 is a gene belonging to the PTP (tyrosine escape phosphoric acid enzyme) family that works as a transmitter and controls various cellular processes like cell proliferation, differentiation, mitotic cycle and oncogenesis.<sup><a reveal-id="DSP022C38" data-open="DSP022C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The known TSS of PTPN4 was highly expressed in the brain, but the TSS we identified in the FLJ53929 cDNA, which was located downstream from the known TSS of TPN4, was highly expressed in the immune system tissues such as bone marrow and spleen (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Twelfth example, BTNL8 is one of the butyrophilin-like proteins and seemed to be involved in conferring immunity.<sup><a reveal-id="DSP022C39" data-open="DSP022C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The known TSS of BTNL8 was found to be expressed at equal levels in various tissues. However, the TSS we identified in the FLJ51528 cDNA, which was located downstream from the known TSS of BTNL8, was highly expressed in the lung and thymus (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Thus, it seems that these four genes regulated their expression levels in the immune system tissues by using specific TSSs.</p><p>Thirteenth example, AKT1 is a gene involved in apoptosis and neuronal differentiation and also may have a role in schizophrenia, especially in the neurotransmission system.<sup><a reveal-id="DSP022C40" data-open="DSP022C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The TSS we identified in the FLJ53606 cDNA, which was located downstream from the known TSS of AKT, was highly expressed in the retinoic acid-induced NT2 cells (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Thus, this gene uses a specific TSS during the neuronal differentiation.</p><p>Thus, among the newly identified genes we have analyzed in this study, the TSSs of a number of these genes revealed specific expression patterns. These results suggest that a single gene could use alternative TSS for tissue-specific transcription. We also found a close relationship between the predicted function of a gene and its tissue-specific expression. Thus, our results suggest a strong correlation between the mRNA diversity and function of a gene.</p>                            <h3 scrollto-destination=5278054 id="5278054" class="section-title" data-legacy-id=s3f>3.6. Construction and use of the FLJ Human cDNA Database</h3>
<p>We constructed the FLJ Human cDNA Database ver. 3.0 (<a class="link link-uri" href="http://flj.lifesciencedb.jp">http://flj.lifesciencedb.jp</a>) based on the results of our analysis of variable protein-coding transcripts produced from a gene by AS. A detailed description of our DB is available at the DB website. The DB graphically displays mapping of all the full-length cDNAs in the human genome and their ORF regions and thus provides a lot of useful information on the mRNA diversity. Moreover, the DB not only contain sequence information on full-length human cDNAs but also contain sequence information on a huge number of human ESTs generated using the oligo-capping method, allowing us to obtain useful information on ESTs mapped on the same genome locus. Because the average length of our EST sequences was ∼500 bases, the diversity of mRNAs produced as a result of AS could be efficiently analyzed by using this information. Because we were able to accurately identify TSSs using our 5′-EST data, we believe that they could be used to understand the relationship between the variable utilization of TSSs and biological functions of genes. Moreover, one could analyze the expression profiles of the transcriptional region of genes using the data from our high accuracy 5′-EST sequences, although in some cases the results might be different from those obtained using the 3′-EST data.</p><p>Despite these useful features, our database specializes on 5′-end sequences, and therefore these data are not suitable for predicting AS in the C-terminal end. Then, a lot of AS-related information still remain to be extracted from our 1.4 million cDNA resources as all of them were not sequenced to completion. Because our cDNA resources are mostly full-length cDNAs including the TSS and the polyA site, complete sequencing of these cDNA clones will add to our understanding of the mRNA diversity. In addition, every full-length sequenced FLJ cDNAs is available from the National Institute of Technology and Evaluation (<a class="link link-uri" href="http://www.nite.go.jp/">http://www.nite.go.jp/</a>). We will continue to add new information on our resources to our database, and these resources will be very useful in the analysis of gene functions.</p><p>Because our interest was on the mRNAs with ORF regions different from those of already known mRNAs, we stopped sequencing the cDNA once we found that the predicted ORF region of the transcript was not different from the known mRNA (for instance, where the alternative TSS only existed in the 5′-untranslated region). We, however, found that there is a tissue specificity in the expression patterns of these genes where the variation in TSS existed in the 5′-untranslated region (results not shown). Collectively, these results suggest that depending on the situation and environment, the transcription machinery utilizes alternative TSS to regulate the expression of a transcript, even when the translated protein is same. These results are also included in our DB. We also did not complete sequencing the clones for which we were unable to predict the ORF regions of their mRNAs. However, we have also included these clones in the DB with the belief that one could obtain some new and useful information by analyzing these clones.</p><p>We discovered a lot of genes had mRNA diversity due to, for example, FEVs. We also found a lot of tissue-specific splicing patterns. Especially, in the case of FEVs that we analyzed, genes used different regions of the genome loci as the first exon, which seemed to be dependent on the tissue and its condition. We also discovered genes, the TSSs of which were located further away on the same genome locus of the gene. In these cases, there exists a high possibility that their transcription is controlled by individual transcription factors. As the mechanisms for controlling the transcription are closely related to the function, by understanding these mechanisms one could be able to artificially control the expression of an appropriate transcript in the future.</p><p>In this study, we have identified multiple transcripts producing genes, and we believe that each one of these genes is transcribed into an appropriate transcript according to the need and circumstance. Now, it will be important to know whether there is any correlation between the expression of one of the transcripts produced by a gene and a disease. For example, in the case of transcripts containing FEVs, which we analyzed in detail, only the first exon regions were different from the other previously characterized transcripts. Since the first exon regions of these transcripts are unique, it is possible to distinguish them easily from the other transcripts. It may be possible to control the expression of a specific mRNA from a group of mRNAs transcribed from a gene by targeting the first exon. As we accumulate more information on mRNA diversity of genes using approaches similar to what we have described in this study, we might be able to identify candidate genes as novel targets for the development of drugs with lower side effects.</p>                            <h2 scrollto-destination=5278060 id="5278060" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5278062 id="5278062" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was partly supported by a grant from New Energy and Industrial Technology Developmental Organization (NEDO) project of the Ministry of Economy, Trade and Industry of Japan.</p>                            <h2 scrollto-destination=5278064 id="5278064" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the members of the NEDO functional analysis of protein and research application project for cDNA sequencing and clone stock, especially thanks to M. Yamazaki, K. Watanabe, A. Sugiyama, Y. Ono, T. Takayama (Japan Biological Informatics Consortium) and K. Fujita (National Institute of Technology and Evaluation, Japan). We also thank the members of the NEDO human full-length cDNA sequencing project for EST sequencing.</p>                            <h2 scrollto-destination=5278066 id="5278066" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP022C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="collab">International Human Genome Sequencing Consortium</div><div class="article-title">Finishing the euchromatic sequence of the human genome</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">431</div> (pg. <div class="fpage">931</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15496913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Linton</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Birren</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">860</div>-<div class="lpage">921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.S.%20Lander&amp;author=L.M.%20Linton&amp;author=B.%20Birren&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lopez</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Alternative splicing of pre-mRNA: developmental consequences and mechanisms of regulation</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">279</div>-<div class="lpage">305</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20pre-mRNA%3A%20developmental%20consequences%20and%20mechanisms%20of%20regulation&amp;author=A.J.%20Lopez&amp;publication_year=1998&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=32&amp;pages=279-305" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.32.1.279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.32.1.279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.32.1.279"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9928482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Black</div> <div class="given-names">D.L.</div></div>. </div><div class="article-title">Protein diversity from alternative splicing: a challenge for bioinformatics and post-genome biology</div>, <div class="source">Cell</div> , <div class="year">2000</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">367</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20diversity%20from%20alternative%20splicing%3A%20a%20challenge%20for%20bioinformatics%20and%20post-genome%20biology&amp;author=D.L.%20Black&amp;publication_year=2000&amp;journal=Cell&amp;volume=103&amp;pages=367-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)00128-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)00128-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)00128-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11081623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">A genomic view of alternative splicing</div>, <div class="source">Nat. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">13</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20view%20of%20alternative%20splicing&amp;author=B.%20Modrek&amp;author=C.%20Lee&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=30&amp;pages=13-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0102-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0102-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0102-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11753382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamm</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Signals and their transduction pathways regulating alternative splicing: a new dimension of the human genome</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">2409</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signals%20and%20their%20transduction%20pathways%20regulating%20alternative%20splicing%3A%20a%20new%20dimension%20of%20the%20human%20genome&amp;author=S.%20Stamm&amp;publication_year=2002&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=11&amp;pages=2409-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/11.20.2409" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2F11.20.2409" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2F11.20.2409"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12351576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bracco</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Kearsey</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The relevance of alternative RNA splicing to pharmacogenomics</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">346</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relevance%20of%20alternative%20RNA%20splicing%20to%20pharmacogenomics&amp;author=L.%20Bracco&amp;author=J.%20Kearsey&amp;publication_year=2003&amp;journal=Trends%20Biotechnol.&amp;volume=21&amp;pages=346-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-7799(03)00146-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-7799(03)00146-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-7799(03)00146-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12902171" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Landry</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Mager</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Wilhelm</div> <div class="given-names">B.T.</div></div>. </div><div class="article-title">Complex controls: the role of alternative promoters in mammalian genomes</div>, <div class="source">Trends Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">640</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20controls%3A%20the%20role%20of%20alternative%20promoters%20in%20mammalian%20genomes&amp;author=J.R.%20Landry&amp;author=D.L.%20Mager&amp;author=B.T.%20Wilhelm&amp;publication_year=2003&amp;journal=Trends%20Genet.&amp;volume=19&amp;pages=640-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2003.09.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2003.09.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2003.09.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14585616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Haws</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Q.</div></div>. </div><div class="article-title">Multiple variable first exons: a mechanism for cell- and tissue-specific gene regulation</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">79</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20variable%20first%20exons%3A%20a%20mechanism%20for%20cell-%20and%20tissue-specific%20gene%20regulation&amp;author=T.%20Zhang&amp;author=P.%20Haws&amp;author=Q.%20Wu&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=79-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1225204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1225204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1225204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14672974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Maniatis</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Large exons encoding multiple ectodomains are a characteristic feature of protocadherin genes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">3124</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20exons%20encoding%20multiple%20ectodomains%20are%20a%20characteristic%20feature%20of%20protocadherin%20genes&amp;author=Q.%20Wu&amp;author=T.%20Maniatis&amp;publication_year=2000&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=3124-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.7.3124" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.7.3124" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.7.3124"> </span></div> </div></div></div></div></div><div content-id="DSP022C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strassburg</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">Oldhafer</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Manns</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Tukey</div> <div class="given-names">R.H.</div></div>. </div><div class="article-title">Differential expression of the UGT1A locus in human liver, biliary, and gastric tissue: identification of UGT1A7 and UGT1A10 transcripts in extrahepatic tissue</div>, <div class="source">Mol. Pharmacol.</div> , <div class="year">1997</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">212</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20the%20UGT1A%20locus%20in%20human%20liver%2C%20biliary%2C%20and%20gastric%20tissue%3A%20identification%20of%20UGT1A7%20and%20UGT1A10%20transcripts%20in%20extrahepatic%20tissue&amp;author=C.P.%20Strassburg&amp;author=K.%20Oldhafer&amp;author=M.P.%20Manns&amp;author=R.H.%20Tukey&amp;publication_year=1997&amp;journal=Mol.%20Pharmacol.&amp;volume=52&amp;pages=212-20" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9271343" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">E.T.</div></div>,  <div class="name"><div class="surname">Sandberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Alternative isoform regulation in human tissue transcriptomes</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">456</div> (pg. <div class="fpage">470</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20isoform%20regulation%20in%20human%20tissue%20transcriptomes&amp;author=E.T.%20Wang&amp;author=R.%20Sandberg&amp;author=S.%20Luo&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=470-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Licatalosi</div> <div class="given-names">D.D.</div></div>,  <div class="name"><div class="surname">Mele</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Fak</div> <div class="given-names">J.J.</div></div>, et al. </div><div class="article-title">HITS-CLIP yields genome-wide insights into brain alternative RNA processing</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">456</div> (pg. <div class="fpage">464</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HITS-CLIP%20yields%20genome-wide%20insights%20into%20brain%20alternative%20RNA%20processing&amp;author=D.D.%20Licatalosi&amp;author=A.%20Mele&amp;author=J.J.%20Fak&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=464-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07488" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07488" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07488"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ota</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Complete sequencing and characterization of 21,243 full-length human cDNAs</div>, <div class="source">Nat. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">40</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20sequencing%20and%20characterization%20of%2021%2C243%20full-length%20human%20cDNAs&amp;author=T.%20Ota&amp;author=Y.%20Suzuki&amp;author=T.%20Nishikawa&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=40-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14702039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Otsuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ota</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Signal sequence and keyword trap in silico for selection of full-length human cDNAs encoding secretion or membrane proteins from oligo-capped cDNA libraries</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">117</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signal%20sequence%20and%20keyword%20trap%20in%20silico%20for%20selection%20of%20full-length%20human%20cDNAs%20encoding%20secretion%20or%20membrane%20proteins%20from%20oligo-capped%20cDNA%20libraries&amp;author=T.%20Otsuki&amp;author=T.%20Ota&amp;author=T.%20Nishikawa&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=117-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.2.117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.2.117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.2.117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16303743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goshima</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kawamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fukumoto</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Human protein factory for converting the transcriptome into an in vitro-expressed proteome</div>, <div class="source">Nat. Methods.</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1011</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20protein%20factory%20for%20converting%20the%20transcriptome%20into%20an%20in%20vitro-expressed%20proteome&amp;author=N.%20Goshima&amp;author=Y.%20Kawamura&amp;author=A.%20Fukumoto&amp;publication_year=2008&amp;journal=Nat.%20Methods.&amp;volume=5&amp;pages=1011-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1273" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1273" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1273"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19054851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wakamatsu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">55</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20transcriptional%20modulation%3A%20large-scale%20identification%20and%20characterization%20of%20putative%20alternative%20promoters%20of%20human%20genes&amp;author=K.%20Kimura&amp;author=A.%20Wakamatsu&amp;author=Y.%20Suzuki&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=55-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4039406" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4039406" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4039406"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16344560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maruyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Oligo-capping: a simple method to replace the cap structure of eukaryotic mRNAs with oligoribonucleotides</div>, <div class="source">Gene</div> , <div class="year">1994</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">171</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oligo-capping%3A%20a%20simple%20method%20to%20replace%20the%20cap%20structure%20of%20eukaryotic%20mRNAs%20with%20oligoribonucleotides&amp;author=K.%20Maruyama&amp;author=S.%20Sugano&amp;publication_year=1994&amp;journal=Gene&amp;volume=138&amp;pages=171-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(94)90802-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(94)90802-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(94)90802-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8125298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Construction of a full-length enriched and a 5'-end enriched cDNA library using the oligo-capping method</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">221</div> (pg. <div class="fpage">73</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20full-length%20enriched%20and%20a%205'-end%20enriched%20cDNA%20library%20using%20the%20oligo-capping%20method&amp;author=Y.%20Suzuki&amp;author=S.%20Sugano&amp;publication_year=2003&amp;journal=Methods%20Mol.%20Biol.&amp;volume=221&amp;pages=73-91" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12703735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yoshitomo-Nakagawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Maruyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Suyama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Construction and characterization of a full length-enriched and a 5'-end-enriched cDNA library</div>, <div class="source">Gene</div> , <div class="year">1997</div>, vol. <div class="volume">200</div> (pg. <div class="fpage">149</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20characterization%20of%20a%20full%20length-enriched%20and%20a%205'-end-enriched%20cDNA%20library&amp;author=Y.%20Suzuki&amp;author=K.%20Yoshitomo-Nakagawa&amp;author=K.%20Maruyama&amp;author=A.%20Suyama&amp;author=S.%20Sugano&amp;publication_year=1997&amp;journal=Gene&amp;volume=200&amp;pages=149-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(97)00411-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(97)00411-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(97)00411-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9373149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ishihara</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Statistical analysis of the 5' untranslated region of human mRNA using ‘Oligo-Capped’ cDNA libraries</div>, <div class="source">Genomics</div> , <div class="year">2000</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">286</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20analysis%20of%20the%205'%20untranslated%20region%20of%20human%20mRNA%20using%20%E2%80%98Oligo-Capped%E2%80%99%20cDNA%20libraries&amp;author=Y.%20Suzuki&amp;author=D.%20Ishihara&amp;author=M.%20Sasaki&amp;publication_year=2000&amp;journal=Genomics&amp;volume=64&amp;pages=286-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.2000.6076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.2000.6076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.2000.6076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10756096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ota</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Database and analysis system for cDNA clones obtained from full-length enriched cDNA libraries</div>, <div class="source">In. Silico Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">5</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20and%20analysis%20system%20for%20cDNA%20clones%20obtained%20from%20full-length%20enriched%20cDNA%20libraries&amp;author=T.%20Nishikawa&amp;author=T.%20Ota&amp;author=Y.%20Kawai&amp;publication_year=2002&amp;journal=In.%20Silico%20Biol.&amp;volume=2&amp;pages=5-18" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11808872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ota</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Comparison of sequences of cDNA clones obtained from oligo-capping cDNA libraries with those from unigene</div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">255</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20sequences%20of%20cDNA%20clones%20obtained%20from%20oligo-capping%20cDNA%20libraries%20with%20those%20from%20unigene&amp;author=T.%20Nishikawa&amp;author=T.%20Ota&amp;author=Y.%20Kawai&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=255-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nagai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Isogai</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Intris: A viewer for cDNA-genome alignments enabling efficient detection of splicing variants and expression profiles</div>, <div class="source">Genome Inform.</div> , <div class="year">2002</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">548</div>-<div class="lpage">50</div>)</div></div></div></div><div content-id="DSP022C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salamov</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Swindells</div> <div class="given-names">M.B.</div></div>. </div><div class="article-title">Assessing protein coding region integrity in cDNA sequencing projects</div>, <div class="source">Bioinformatics</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">384</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20protein%20coding%20region%20integrity%20in%20cDNA%20sequencing%20projects&amp;author=A.A.%20Salamov&amp;author=T.%20Nishikawa&amp;author=M.B.%20Swindells&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=384-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.5.384" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.5.384" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.5.384"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9682051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nagai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nomura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Isogai</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The translated region inspector for cDNA sequences</div>, <div class="source">Genome Inform.</div> , <div class="year">2003</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">456</div>-<div class="lpage">7</div>)</div></div></div></div><div content-id="DSP022C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hatano</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Araki</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">A cDNA evaluation system for highly efficient sequencing of splicing variant cDNAs</div>, <div class="source">Genome Inform.</div> , <div class="year">2003</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">430</div>-<div class="lpage">1</div>)</div></div></div></div><div content-id="DSP022C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Facchiano</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Russo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Facchiano</div> <div class="given-names">A.M.</div></div>, et al. </div><div class="article-title">Identification of a novel domain of fibroblast growth factor 2 controlling its angiogenic properties</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">8751</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20novel%20domain%20of%20fibroblast%20growth%20factor%202%20controlling%20its%20angiogenic%20properties&amp;author=A.%20Facchiano&amp;author=K.%20Russo&amp;author=A.M.%20Facchiano&amp;publication_year=2003&amp;journal=J.%20Biol.%20Chem.&amp;volume=278&amp;pages=8751-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M209936200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M209936200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M209936200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12496262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Greene</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.L.</div></div>,  <div class="name"><div class="surname">Yourey</div> <div class="given-names">P.A.</div></div>, et al. </div><div class="article-title">Identification and characterization of a novel member of the fibroblast growth factor family</div>, <div class="source">Eur. J. Neurosci.</div> , <div class="year">1998</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1911</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20a%20novel%20member%20of%20the%20fibroblast%20growth%20factor%20family&amp;author=J.M.%20Greene&amp;author=Y.L.%20Li&amp;author=P.A.%20Yourey&amp;publication_year=1998&amp;journal=Eur.%20J.%20Neurosci.&amp;volume=10&amp;pages=1911-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1460-9568.1998.00211.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1460-9568.1998.00211.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1460-9568.1998.00211.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9751161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Durand</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kolpak</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Farrell</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">The OXR domain defines a conserved family of eukaryotic oxidation resistance proteins</div>, <div class="source">BMC Cell Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">13</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20OXR%20domain%20defines%20a%20conserved%20family%20of%20eukaryotic%20oxidation%20resistance%20proteins&amp;author=M.%20Durand&amp;author=A.%20Kolpak&amp;author=T.%20Farrell&amp;publication_year=2007&amp;journal=BMC%20Cell%20Biol.&amp;volume=8&amp;pages=13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2121-8-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2121-8-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2121-8-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17391516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foster</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Phospholipase D in cell proliferation and cancer</div>, <div class="source">Mol. Cancer. Res.</div> , <div class="year">2003</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">789</div>-<div class="lpage">800</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phospholipase%20D%20in%20cell%20proliferation%20and%20cancer&amp;author=D.A.%20Foster&amp;author=L.%20Xu&amp;publication_year=2003&amp;journal=Mol.%20Cancer.%20Res.&amp;volume=1&amp;pages=789-800" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14517341" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nonami</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Taniguchi</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Spred-1 negatively regulates interleukin-3-mediated ERK/mitogen-activated protein (MAP) kinase activation in hematopoietic cells</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">52543</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spred-1%20negatively%20regulates%20interleukin-3-mediated%20ERK%2Fmitogen-activated%20protein%20(MAP)%20kinase%20activation%20in%20hematopoietic%20cells&amp;author=A.%20Nonami&amp;author=R.%20Kato&amp;author=K.%20Taniguchi&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=52543-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M405189200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M405189200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M405189200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15465815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adams</div> <div class="given-names">R.H.</div></div>,  <div class="name"><div class="surname">Betz</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Puschel</div> <div class="given-names">A.W.</div></div>. </div><div class="article-title">A novel class of murine semaphorins with homology to thrombospondin is differentially expressed during early embryogenesis</div>, <div class="source">Mech. Dev.</div> , <div class="year">1996</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">33</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20class%20of%20murine%20semaphorins%20with%20homology%20to%20thrombospondin%20is%20differentially%20expressed%20during%20early%20embryogenesis&amp;author=R.H.%20Adams&amp;author=H.%20Betz&amp;author=A.W.%20Puschel&amp;publication_year=1996&amp;journal=Mech.%20Dev.&amp;volume=57&amp;pages=33-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0925-4773(96)00525-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0925-4773(96)00525-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0925-4773(96)00525-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8817451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Masuda</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">The structures of the human calcium channel alpha 1 subunit (CACNL1A2) and beta subunit (CACNLB3) genes</div>, <div class="source">Genomics</div> , <div class="year">1995</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">312</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structures%20of%20the%20human%20calcium%20channel%20alpha%201%20subunit%20(CACNL1A2)%20and%20beta%20subunit%20(CACNLB3)%20genes&amp;author=Y.%20Yamada&amp;author=K.%20Masuda&amp;author=Q.%20Li&amp;publication_year=1995&amp;journal=Genomics&amp;volume=27&amp;pages=312-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1995.1048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1995.1048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1995.1048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7557998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">De Pietri Tonelli</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Mihailovich</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Di Cesare</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Codazzi</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Grohovaz</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zacchetti</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Translational regulation of BACE-1 expression in neuronal and non-neuronal cells</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">1808</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20regulation%20of%20BACE-1%20expression%20in%20neuronal%20and%20non-neuronal%20cells&amp;author=D.%20De%20Pietri%20Tonelli&amp;author=M.%20Mihailovich&amp;author=A.%20Di%20Cesare&amp;author=F.%20Codazzi&amp;author=F.%20Grohovaz&amp;author=D.%20Zacchetti&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1808-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh348" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">X.M.</div></div>,  <div class="name"><div class="surname">Splinter</div> <div class="given-names">P.L.</div></div>,  <div class="name"><div class="surname">Tietz</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">B.Q.</div></div>,  <div class="name"><div class="surname">Billadeau</div> <div class="given-names">D.D.</div></div>,  <div class="name"><div class="surname">LaRusso</div> <div class="given-names">N.F.</div></div>. </div><div class="article-title">Phosphatidylinositol 3-kinase and frabin mediate Cryptosporidium parvum cellular invasion via activation of Cdc42</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">31671</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phosphatidylinositol%203-kinase%20and%20frabin%20mediate%20Cryptosporidium%20parvum%20cellular%20invasion%20via%20activation%20of%20Cdc42&amp;author=X.M.%20Chen&amp;author=P.L.%20Splinter&amp;author=P.S.%20Tietz&amp;author=B.Q.%20Huang&amp;author=D.D.%20Billadeau&amp;author=N.F.%20LaRusso&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=31671-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M401592200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M401592200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M401592200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15133042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Molloy</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Cowan</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Gussoni</div> <div class="given-names">E.</div></div>. </div><div class="article-title">C6ORF32 is upregulated during muscle cell differentiation and induces the formation of cellular filopodia</div>, <div class="source">Dev. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">301</div> (pg. <div class="fpage">70</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=C6ORF32%20is%20upregulated%20during%20muscle%20cell%20differentiation%20and%20induces%20the%20formation%20of%20cellular%20filopodia&amp;author=S.%20Yoon&amp;author=M.J.%20Molloy&amp;author=M.P.%20Wu&amp;author=D.B.%20Cowan&amp;author=E.%20Gussoni&amp;publication_year=2007&amp;journal=Dev.%20Biol.&amp;volume=301&amp;pages=70-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2006.11.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ydbio.2006.11.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ydbio.2006.11.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17150207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Young</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Medeiros</div> <div class="given-names">J.J.</div></div>, et al. </div><div class="article-title">The protein tyrosine phosphatase PTPN4/PTP-MEG1, an enzyme capable of dephosphorylating the TCR ITAMs and regulating NF-kappaB, is dispensable for T cell development and/or T cell effector functions</div>, <div class="source">Mol. Immunol.</div> , <div class="year">2008</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">3756</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20protein%20tyrosine%20phosphatase%20PTPN4%2FPTP-MEG1%2C%20an%20enzyme%20capable%20of%20dephosphorylating%20the%20TCR%20ITAMs%20and%20regulating%20NF-kappaB%2C%20is%20dispensable%20for%20T%20cell%20development%20and%2For%20T%20cell%20effector%20functions&amp;author=J.A.%20Young&amp;author=A.M.%20Becker&amp;author=J.J.%20Medeiros&amp;publication_year=2008&amp;journal=Mol.%20Immunol.&amp;volume=45&amp;pages=3756-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molimm.2008.05.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molimm.2008.05.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molimm.2008.05.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18614237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rhodes</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Stammers</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Malcherek</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Beck</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Trowsdale</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The cluster of BTN genes in the extended major histocompatibility complex</div>, <div class="source">Genomics</div> , <div class="year">2001</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">351</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cluster%20of%20BTN%20genes%20in%20the%20extended%20major%20histocompatibility%20complex&amp;author=D.A.%20Rhodes&amp;author=M.%20Stammers&amp;author=G.%20Malcherek&amp;author=S.%20Beck&amp;author=J.%20Trowsdale&amp;publication_year=2001&amp;journal=Genomics&amp;volume=71&amp;pages=351-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.2000.6406" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.2000.6406" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.2000.6406"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11170752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP022C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP022C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brugge</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hung</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Mills</div> <div class="given-names">G.B.</div></div>. </div><div class="article-title">A new mutational AKTivation in the PI3K pathway</div>, <div class="source">Cancer. Cell</div> , <div class="year">2007</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">104</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20mutational%20AKTivation%20in%20the%20PI3K%20pathway&amp;author=J.%20Brugge&amp;author=M.C.%20Hung&amp;author=G.B.%20Mills&amp;publication_year=2007&amp;journal=Cancer.%20Cell&amp;volume=12&amp;pages=104-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ccr.2007.07.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ccr.2007.07.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ccr.2007.07.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17692802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=364039&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/dsp022-dsp022_table_3.xls?Expires=2147483647&amp;Signature=Q1uMl9xpS490XvXNR2X6ac4AQ~6h64DD2LV0T-aZ5ADQtnHYl-ScdsABIGmuM17P-XlWlc0b9GIq-YqT5jCKc-WX7gYRGJVpEVsuRhIBrr0sBNceSMBI4FOQoVzF8285aiUKoFDQAeKsqC1wNgq-MXv0Rats91MvI7VcGLAamt0LOgFCNQEnjYJ5KQB1xAek8h9p0n4ma-j1CLeMzaae26ho3KWR4lm4PrWtlE74gZ~RXBNgyizXJET2D2smA52oLz8ztpoFQifQPK0uUk1f6Lw8tX-wT9qngk5xg2MgSfSbI3cCAe4qqjJc7v1qOz01J88C6Zwff8MAZd5YkIQzFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp022/2/dsp022-dsp022supp.pdf?Expires=2147483647&amp;Signature=U5kICOORzih~74oeNqVE~HinBucvgiq~pOxQ5aET1EaHdQ1HRpOdnYVGNMR0uYB3Sn7VRFeHR5Nxce9pEzk0XtXlukpmkRLHWoQJoZlAXqy14DxfF7tSzbwTmtMwbfcdz5OPRwI-OEiC-giw73HZDJsLUdE8hqn9eQ3WRwwpbZ8HPN-PmkpxN1YVu8Qm6fpz~DJ2rvevOqN5HpepmcnQp54hCgsbR7fAl1nOq~VbeE5JUhjDKPI-nRSPdBmUX7k5AwlAFSl6HGykthei4GWSmVaEDGwxKYo07ZTDtuyq5CrSKGLRTL050jskJPKCYvCmQiuObrwhC0Im9QpaLGlrRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">89</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000272832700005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp022"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="364039"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000272832700005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f6%2f371%2f364039">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000272832700005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F6%2F371%2F364039" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/364039'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identification and Functional Analyses of 11 769 Full-length Human cDNAs Focused on Alternative Splicing" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp022");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f6%2f371%2f364039">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

