

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/1/93/384291","@type":"ScholarlyArticle","name":" De Novo  Assembly and Functional Annotation of the Olive ( Olea europaea ) Transcriptome","datePublished":"2013-01-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2013-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss036","keywords":["<italic>Olea europaea</italic>","454 pyrosequencing","genomics","Sanger sequencing","transcriptome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/1/93/384291","author":[{"name":"Muñoz-Mérida, Antonio","affiliation":"1  Department of Integrated Bioinformatics ,  National Institute for Bioinformatics, University of Málaga ,  Campus de Teatinos, Málaga ,  Spain","@type":"Person"},{"name":"González-Plaza, Juan José","affiliation":"2  Biología Celular, Genética y Fisiología ,  Instituto de Hortofruticultura Subtropical y Mediterránea, Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC) ,  Campus de Teatinos, Málaga E-29071 ,  Spain","@type":"Person"},{"name":"Cañada, Andrés","affiliation":"3  Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO) ,  Melchor Fernández Almagro 3, Madrid E-28029 ,  Spain","@type":"Person"},{"name":"Blanco, Ana María","affiliation":"4  Sistemas Genómicos S.L. ,  Parque Tecnológico de Valencia, Ronda G. Marconi 6, Paterna E-46980 ,  Spain","@type":"Person"},{"name":"García-López, Maria del Carmen","affiliation":"5  Dpto. de Biología Experimental ,  Universidad de Jaén ,  Campus de las Lagunillas s/n. Edif. B-3, Jaén 23071 ,  Spain","@type":"Person"},{"name":"Rodríguez, José Manuel","affiliation":"3  Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO) ,  Melchor Fernández Almagro 3, Madrid E-28029 ,  Spain","@type":"Person"},{"name":"Pedrola, Laia","affiliation":"6  Life Sequencing S.L. ,  Parc Científic Universitat de Valencia ,  9, Catedrático Agustín Escardino Benlloch Edificio N2 BIOTECH, Paterna, Valencia ,  Spain","@type":"Person"},{"name":"Sicardo, M. Dolores","affiliation":"7  Instituto de la Grasa (IG-CSIC) ,  Avenida Padre García Tejero 4, Seville 41012 ,  Spain","@type":"Person"},{"name":"Hernández, M. Luisa","affiliation":"7  Instituto de la Grasa (IG-CSIC) ,  Avenida Padre García Tejero 4, Seville 41012 ,  Spain","@type":"Person"},{"name":"De la Rosa, Raúl","affiliation":"8  IFAPA Centro Alameda del Obispo ,  Avda Menéndez Pidal s/n 14004, Córdoba ,  Spain","@type":"Person"},{"name":"Belaj, Angjelina","affiliation":"8  IFAPA Centro Alameda del Obispo ,  Avda Menéndez Pidal s/n 14004, Córdoba ,  Spain","@type":"Person"},{"name":"Gil-Borja, Mayte","affiliation":"4  Sistemas Genómicos S.L. ,  Parque Tecnológico de Valencia, Ronda G. Marconi 6, Paterna E-46980 ,  Spain","@type":"Person"},{"name":"Luque, Francisco","affiliation":"5  Dpto. de Biología Experimental ,  Universidad de Jaén ,  Campus de las Lagunillas s/n. Edif. B-3, Jaén 23071 ,  Spain","@type":"Person"},{"name":"Martínez-Rivas, José Manuel","affiliation":"7  Instituto de la Grasa (IG-CSIC) ,  Avenida Padre García Tejero 4, Seville 41012 ,  Spain","@type":"Person"},{"name":"Pisano, David G.","affiliation":"3  Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO) ,  Melchor Fernández Almagro 3, Madrid E-28029 ,  Spain","@type":"Person"},{"name":"Trelles, Oswaldo","affiliation":"1  Department of Integrated Bioinformatics ,  National Institute for Bioinformatics, University of Málaga ,  Campus de Teatinos, Málaga ,  Spain","@type":"Person"},{"name":"Valpuesta, Victoriano","affiliation":"9  Instituto de Hortofruticultura Subtropical y Mediterránea, Dpto. Biología Molecular y Bioquímica ,  Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC) ,  Campus de Teatinos, Málaga E-29071 ,  Spain","@type":"Person"},{"name":"Beuzón, Carmen R.","affiliation":"2  Biología Celular, Genética y Fisiología ,  Instituto de Hortofruticultura Subtropical y Mediterránea, Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC) ,  Campus de Teatinos, Málaga E-29071 ,  Spain","@type":"Person"}],"description":"Abstract.  Olive breeding programmes are focused on selecting for traits as short juvenile period, plant architecture suited for mechanical harvest, or oil char","pageStart":"93","pageEnd":"108","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" De Novo  Assembly and Functional Annotation of the Olive ( Olea europaea ) Transcriptome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" De Novo  Assembly and Functional Annotation of the Olive ( Olea europaea ) Transcriptome" />
    <meta property="og:description" content="Abstract.  Olive breeding programmes are focused on selecting for traits as short juvenile period, plant architecture suited for mechanical harvest, or oil char" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/1/93/384291" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-01-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Muñoz-Mérida, Antonio" /><meta name="citation_author_institution" content="Department of Integrated Bioinformatics, National Institute for Bioinformatics, University of Málaga, Campus de Teatinos, Málaga, Spain" /><meta name="citation_author" content="González-Plaza, Juan José" /><meta name="citation_author_institution" content="Biología Celular, Genética y Fisiología, Instituto de Hortofruticultura Subtropical y Mediterránea, Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC), Campus de Teatinos, Málaga E-29071, Spain" /><meta name="citation_author" content="Cañada, Andrés" /><meta name="citation_author_institution" content="Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO), Melchor Fernández Almagro 3, Madrid E-28029, Spain" /><meta name="citation_author" content="Blanco, Ana María" /><meta name="citation_author_institution" content="Sistemas Genómicos S.L., Parque Tecnológico de Valencia, Ronda G. Marconi 6, Paterna E-46980, Spain" /><meta name="citation_author" content="García-López, Maria del Carmen" /><meta name="citation_author_institution" content="Dpto. de Biología Experimental, Universidad de Jaén, Campus de las Lagunillas s/n. Edif. B-3, Jaén 23071, Spain" /><meta name="citation_author" content="Rodríguez, José Manuel" /><meta name="citation_author_institution" content="Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO), Melchor Fernández Almagro 3, Madrid E-28029, Spain" /><meta name="citation_author" content="Pedrola, Laia" /><meta name="citation_author_institution" content="Life Sequencing S.L., Parc Científic Universitat de Valencia, 9, Catedrático Agustín Escardino Benlloch Edificio N2 BIOTECH, Paterna, Valencia, Spain" /><meta name="citation_author" content="Sicardo, M. Dolores" /><meta name="citation_author_institution" content="Instituto de la Grasa (IG-CSIC), Avenida Padre García Tejero 4, Seville 41012, Spain" /><meta name="citation_author" content="Hernández, M. Luisa" /><meta name="citation_author_institution" content="Instituto de la Grasa (IG-CSIC), Avenida Padre García Tejero 4, Seville 41012, Spain" /><meta name="citation_author" content="De la Rosa, Raúl" /><meta name="citation_author_institution" content="IFAPA Centro Alameda del Obispo, Avda Menéndez Pidal s/n 14004, Córdoba, Spain" /><meta name="citation_author" content="Belaj, Angjelina" /><meta name="citation_author_institution" content="IFAPA Centro Alameda del Obispo, Avda Menéndez Pidal s/n 14004, Córdoba, Spain" /><meta name="citation_author" content="Gil-Borja, Mayte" /><meta name="citation_author_institution" content="Sistemas Genómicos S.L., Parque Tecnológico de Valencia, Ronda G. Marconi 6, Paterna E-46980, Spain" /><meta name="citation_author" content="Luque, Francisco" /><meta name="citation_author_institution" content="Dpto. de Biología Experimental, Universidad de Jaén, Campus de las Lagunillas s/n. Edif. B-3, Jaén 23071, Spain" /><meta name="citation_author" content="Martínez-Rivas, José Manuel" /><meta name="citation_author_institution" content="Instituto de la Grasa (IG-CSIC), Avenida Padre García Tejero 4, Seville 41012, Spain" /><meta name="citation_author" content="Pisano, David G." /><meta name="citation_author_institution" content="Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO), Melchor Fernández Almagro 3, Madrid E-28029, Spain" /><meta name="citation_author" content="Trelles, Oswaldo" /><meta name="citation_author_institution" content="Department of Integrated Bioinformatics, National Institute for Bioinformatics, University of Málaga, Campus de Teatinos, Málaga, Spain" /><meta name="citation_author" content="Valpuesta, Victoriano" /><meta name="citation_author_institution" content="Instituto de Hortofruticultura Subtropical y Mediterránea, Dpto. Biología Molecular y Bioquímica, Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC), Campus de Teatinos, Málaga E-29071, Spain" /><meta name="citation_author" content="Beuzón, Carmen R." /><meta name="citation_author_institution" content="Biología Celular, Genética y Fisiología, Instituto de Hortofruticultura Subtropical y Mediterránea, Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC), Campus de Teatinos, Málaga E-29071, Spain" /><meta name="citation_title" content="De Novo Assembly and Functional Annotation of the Olive (Olea europaea) Transcriptome" /><meta name="citation_firstpage" content="93" /><meta name="citation_lastpage" content="108" /><meta name="citation_doi" content="10.1093/dnares/dss036" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2013/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Olive groves: ‘The life and identity of the Mediterranean; Agric. Hum. Values;  citation_year=2003;  citation_volume=20;  citation_pages=87-95; " /><meta name="citation_reference" content="FAOSTAT 2008 home page Columbia URL: http://www.columbia.edu/cgi-bin/cul/resolve?ASL9609 " /><meta name="citation_reference" content="citation_title=A consensus list of microsatellite markers for olive genotyping; Mol. Breed.;  citation_year=2009;  citation_volume=24;  citation_pages=213-31; " /><meta name="citation_reference" content="citation_title=Physiological, biochemical and molecular changes occurring during olive development and ripening; J. Plant Physiol.;  citation_year=2008;  citation_volume=165;  citation_pages=1545-62; " /><meta name="citation_reference" content="citation_title=Isolation of triploid and tetraploid olive (Olea europaea L.) plants from mixoploid cv. ‘Frantoio’ and ‘Leccino’ mutants by in vivo and in vitro selection; Plant Breed.;  citation_year=1996;  citation_volume=115;  citation_pages=23-7; " /><meta name="citation_reference" content="citation_title=A first linkage map of olive (Olea europaea L.) cultivars using RAPD, AFLP, RFLP and SSR markers; Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=1273-82; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content and genome size of trout and human; Cytometry A;  citation_year=2003;  citation_volume=51;  citation_pages=127-9; " /><meta name="citation_reference" content="citation_title=Salinity Tolerance in Olive, in Horticultural Reviews; citation_publisher=John Wiley &amp; Sons, Inc.: Oxford,, UK;  citation_year=2010;  citation_volume=21;  citation_pages=177-214; " /><meta name="citation_reference" content="citation_title=Genetic variability and heritability of some characters in cross-bred olive seedlings; Olivae;  citation_year=1993;  citation_volume=49;  citation_pages=21-34; " /><meta name="citation_reference" content="citation_title=Optimizing early flowering and pre-selection for short juvenile period in olive seedlings; HortScience;  citation_year=2010;  citation_volume=45;  citation_pages=519-22; " /><meta name="citation_reference" content="citation_title=‘Chiquitita’ olive; HortScience;  citation_year=2008;  citation_volume=43;  citation_pages=529-31; " /><meta name="citation_reference" content="citation_title=Fatty acid composition of advanced olive selections obtained by crossbreeding; J. Sci. Food Agric.;  citation_year=2008;  citation_volume=88;  citation_pages=1921-6; " /><meta name="citation_reference" content="citation_title=Comparative 454 pyrosequencing of transcripts from two olive genotypes during fruit development; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=399" /><meta name="citation_reference" content="citation_title=Computational annotation of genes differentially expressed along olive fruit development; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=128" /><meta name="citation_reference" content="citation_title=Generation and analysis of expressed sequence tags from Olea europaea L. Comp; Funct. Genomics;  citation_year=2010;  citation_volume=2010;  citation_pages=757512" /><meta name="citation_reference" content="citation_title=High-throughput sequencing of RNA silencing-associated small RNAs in olive (Olea europaea L.); PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e27916" /><meta name="citation_reference" content="citation_title=A high quality draft consensus sequence of the genome of a heterozygous grapevine variety; PLoS ONE;  citation_year=2007;  citation_volume=2;  citation_pages=e1326" /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla; Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray); Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-604; " /><meta name="citation_reference" content="citation_title=Molecular studies in olive (Olea europaea L.): overview on DNA markers applications and recent advances in genome analysis; Plant Cell Rep.;  citation_year=2011;  citation_volume=30;  citation_pages=449-62; " /><meta name="citation_reference" content="citation_title=Pectin esterase gene family in strawberry fruit: study of FaPE1, a ripening-specific isoform; J. Exp. Bot.;  citation_year=2004;  citation_volume=55;  citation_pages=909" /><meta name="citation_reference" content="citation_title=Molecular cloning and characterization of genes encoding two microsomal oleate desaturases (FAD2) from olive; Phytochemistry;  citation_year=2005;  citation_volume=66;  citation_pages=1417-26; " /><meta name="citation_reference" content="citation_title=TRIzol: A new reagent for optimal single-step isolation of RNA; Focus;  citation_year=1993;  citation_volume=15;  citation_pages=99-102; " /><meta name="citation_reference" content="citation_title=A useful protocol for in situ RT-PCR on plant tissues; Cell. Mol. Biol. Lett.;  citation_year=2002;  citation_volume=7;  citation_pages=7-18; " /><meta name="citation_reference" content="citation_title=Reverse transcriptase template switching: a SMART approach for full-length cDNA library construction; Biotechniques;  citation_year=2001;  citation_volume=30;  citation_pages=892-7; " /><meta name="citation_reference" content="citation_title=Simple cDNA normalization using kamchatka crab duplex specific nuclease; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=e37" /><meta name="citation_reference" content="citation_title=A novel method for SNP detection using a new duplex-specific nuclease from crab hepatopancreas; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1935-42; " /><meta name="citation_reference" content="citation_title=Construction of cDNA library of cotton mutant (Xiangmian-18) library during gland forming stage; Colloids Surf. B Biointerfaces;  citation_year=2007;  citation_volume=60;  citation_pages=258-63; " /><meta name="citation_reference" content="citation_title=Ice-binding proteins from enoki and shiitake mushrooms; Cryobiology;  citation_year=2009;  citation_volume=58;  citation_pages=151-6; " /><meta name="citation_reference" content="citation_title=Rapid transcriptome characterization for a nonmodel organism using 454 pyrosequencing; Mol. Ecol.;  citation_year=2008;  citation_volume=17;  citation_pages=1636-47; " /><meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=376-80; " /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="Oliveros J.C. VENNY. An interactive tool for comparing lists with Venn Diagrams 2007 http://bioinfogp.cnb.csic.es/tools/venny/index.html. " /><meta name="citation_reference" content="citation_title=Comprehensive transcriptome analysis of the highly complex Pisum sativum genome using next generation sequencing; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=227" /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Database of homology-derived protein structures and the structural meaning of sequence alignment; Proteins;  citation_year=1991;  citation_volume=9;  citation_pages=56-68; " /><meta name="citation_reference" content="citation_title=Twilight zone of protein sequence alignments; Protein Eng.;  citation_year=1999;  citation_volume=12;  citation_pages=85-94; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press: Cold Spring Harbor, New York, ; Statistics at the Bench: A Step-By-Step Handbook for Biologists;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; J. Roy. Stat. Soc. B Met.;  citation_year=1995;  citation_volume=57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Libro II. Variabilidad y Selección; citation_publisher=MAPA y Ediciones Mundi-Prensa, Madrid; Variedades de olivo en España, Junta de Andalucía;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=Juvenility and maturity of plants as influenced by their ontogenetical and physiological ageing; Acta Hort. (ISHS);  citation_year=1976;  citation_volume=56;  citation_pages=37-44; " /><meta name="citation_reference" content="citation_title=Juvenility and maturation in conifers: current concepts; Tree Physiol.;  citation_year=1995;  citation_volume=15;  citation_pages=433-8; " /><meta name="citation_reference" content="citation_title=Phase change and the regulation of shoot morphogenesis in plants; Science;  citation_year=1990;  citation_volume=250;  citation_pages=923-30; " /><meta name="citation_reference" content="citation_title=Deductions about the number, organization, and evolution of genes in the tomato genome based on analysis of a large expressed sequence tag collection and selective genomic sequencing; Plant Cell;  citation_year=2002;  citation_volume=14;  citation_pages=1441-56; " /><meta name="citation_reference" content="McCourt R.M. Chapman R.L. Buchheim M. Mishler B.D. Green plants 1996 http://tolweb.org/Green_plants/2382/1996.01.01 in The Tree of Life Web Project http://tolweb.org/ " /><meta name="citation_reference" content="Uniprot taxonomy http://www.uniprot.org/taxonomy/ " /><meta name="citation_reference" content="citation_title=Developing a core collection of olive (Olea europaea L.) based on molecular markers (DArTs, SSRs, SNPs) and agronomic traits; Tree Genet. Genomes;  citation_year=2012;  citation_volume=8;  citation_pages=365-78; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/1/93/1088282/dss036.pdf" /><meta name="description" content="Abstract.  Olive breeding programmes are focused on selecting for traits as short juvenile period, plant architecture suited for mechanical harvest, or oil char" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/1/93/384291" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/1/93/384291" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/1/93/384291");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss036';
    </script>
<title>
	De Novo Assembly and Functional Annotation of the Olive (Olea europaea) Transcriptome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./384291" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeN9PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNdcOxb3otTcOpcmlkYSwgQW50b25pbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgSW50ZWdyYXRlZCBCaW9pbmZvcm1hdGljcywgTmF0aW9uYWwgSW5zdGl0dXRlIGZvciBCaW9pbmZvcm1hdGljcywgVW5pdmVyc2l0eSBvZiBNw6FsYWdhLCBDYW1wdXMgZGUgVGVhdGlub3MsIE3DoWxhZ2EsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR29uesOhbGV6LVBsYXphLCBKdWFuIEpvc8OpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvbG9nw61hIENlbHVsYXIsIEdlbsOpdGljYSB5IEZpc2lvbG9nw61hLCBJbnN0aXR1dG8gZGUgSG9ydG9mcnV0aWN1bHR1cmEgU3VidHJvcGljYWwgeSBNZWRpdGVycsOhbmVhLCBVbml2ZXJzaWRhZCBkZSBNw6FsYWdhLUNvbnNlam8gU3VwZXJpb3IgZGUgSW52ZXN0aWdhY2lvbmVzIENpZW50w61maWNhcyAoSUhTTS1VTUEtQ1NJQyksIENhbXB1cyBkZSBUZWF0aW5vcywgTcOhbGFnYSBFLTI5MDcxLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNhw7FhZGEsIEFuZHLDqXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dG8gTmFjaW9uYWwgZGUgQmlvaW5mb3Jtw6F0aWNhLCBTcGFuaXNoIE5hdGlvbmFsIENhbmNlciBSZXNlYXJjaCBDZW50cmUgKENOSU8pLCBNZWxjaG9yIEZlcm7DoW5kZXogQWxtYWdybyAzLCBNYWRyaWQgRS0yODAyOSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCbGFuY28sIEFuYSBNYXLDrWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTaXN0ZW1hcyBHZW7Ds21pY29zIFMuTC4sIFBhcnF1ZSBUZWNub2zDs2dpY28gZGUgVmFsZW5jaWEsIFJvbmRhIEcuIE1hcmNvbmkgNiwgUGF0ZXJuYSBFLTQ2OTgwLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdhcmPDrWEtTMOzcGV6LCBNYXJpYSBkZWwgQ2FybWVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRHB0by4gZGUgQmlvbG9nw61hIEV4cGVyaW1lbnRhbCwgVW5pdmVyc2lkYWQgZGUgSmHDqW4sIENhbXB1cyBkZSBsYXMgTGFndW5pbGxhcyBzL24uIEVkaWYuIEItMywgSmHDqW4gMjMwNzEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUm9kcsOtZ3VleiwgSm9zw6kgTWFudWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRvIE5hY2lvbmFsIGRlIEJpb2luZm9ybcOhdGljYSwgU3BhbmlzaCBOYXRpb25hbCBDYW5jZXIgUmVzZWFyY2ggQ2VudHJlIChDTklPKSwgTWVsY2hvciBGZXJuw6FuZGV6IEFsbWFncm8gMywgTWFkcmlkIEUtMjgwMjksIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGVkcm9sYSwgTGFpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxpZmUgU2VxdWVuY2luZyBTLkwuLCBQYXJjIENpZW50w61maWMgVW5pdmVyc2l0YXQgZGUgVmFsZW5jaWEsIDksIENhdGVkcsOhdGljbyBBZ3VzdMOtbiBFc2NhcmRpbm8gQmVubGxvY2ggRWRpZmljaW8gTjIgQklPVEVDSCwgUGF0ZXJuYSwgVmFsZW5jaWEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2ljYXJkbywgTS4gRG9sb3JlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0byBkZSBsYSBHcmFzYSAoSUctQ1NJQyksIEF2ZW5pZGEgUGFkcmUgR2FyY8OtYSBUZWplcm8gNCwgU2V2aWxsZSA0MTAxMiwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIZXJuw6FuZGV6LCBNLiBMdWlzYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0byBkZSBsYSBHcmFzYSAoSUctQ1NJQyksIEF2ZW5pZGEgUGFkcmUgR2FyY8OtYSBUZWplcm8gNCwgU2V2aWxsZSA0MTAxMiwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEZSBsYSBSb3NhLCBSYcO6bCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklGQVBBIENlbnRybyBBbGFtZWRhIGRlbCBPYmlzcG8sIEF2ZGEgTWVuw6luZGV6IFBpZGFsIHMvbiAxNDAwNCwgQ8OzcmRvYmEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmVsYWosIEFuZ2plbGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklGQVBBIENlbnRybyBBbGFtZWRhIGRlbCBPYmlzcG8sIEF2ZGEgTWVuw6luZGV6IFBpZGFsIHMvbiAxNDAwNCwgQ8OzcmRvYmEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2lsLUJvcmphLCBNYXl0ZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNpc3RlbWFzIEdlbsOzbWljb3MgUy5MLiwgUGFycXVlIFRlY25vbMOzZ2ljbyBkZSBWYWxlbmNpYSwgUm9uZGEgRy4gTWFyY29uaSA2LCBQYXRlcm5hIEUtNDY5ODAsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTHVxdWUsIEZyYW5jaXNjbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRwdG8uIGRlIEJpb2xvZ8OtYSBFeHBlcmltZW50YWwsIFVuaXZlcnNpZGFkIGRlIEphw6luLCBDYW1wdXMgZGUgbGFzIExhZ3VuaWxsYXMgcy9uLiBFZGlmLiBCLTMsIEphw6luIDIzMDcxLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hcnTDrW5lei1SaXZhcywgSm9zw6kgTWFudWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRvIGRlIGxhIEdyYXNhIChJRy1DU0lDKSwgQXZlbmlkYSBQYWRyZSBHYXJjw61hIFRlamVybyA0LCBTZXZpbGxlIDQxMDEyLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBpc2FubywgRGF2aWQgRy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dG8gTmFjaW9uYWwgZGUgQmlvaW5mb3Jtw6F0aWNhLCBTcGFuaXNoIE5hdGlvbmFsIENhbmNlciBSZXNlYXJjaCBDZW50cmUgKENOSU8pLCBNZWxjaG9yIEZlcm7DoW5kZXogQWxtYWdybyAzLCBNYWRyaWQgRS0yODAyOSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUcmVsbGVzLCBPc3dhbGRvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBJbnRlZ3JhdGVkIEJpb2luZm9ybWF0aWNzLCBOYXRpb25hbCBJbnN0aXR1dGUgZm9yIEJpb2luZm9ybWF0aWNzLCBVbml2ZXJzaXR5IG9mIE3DoWxhZ2EsIENhbXB1cyBkZSBUZWF0aW5vcywgTcOhbGFnYSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJWYWxwdWVzdGEsIFZpY3Rvcmlhbm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dG8gZGUgSG9ydG9mcnV0aWN1bHR1cmEgU3VidHJvcGljYWwgeSBNZWRpdGVycsOhbmVhLCBEcHRvLiBCaW9sb2fDrWEgTW9sZWN1bGFyIHkgQmlvcXXDrW1pY2EsIFVuaXZlcnNpZGFkIGRlIE3DoWxhZ2EtQ29uc2VqbyBTdXBlcmlvciBkZSBJbnZlc3RpZ2FjaW9uZXMgQ2llbnTDrWZpY2FzIChJSFNNLVVNQS1DU0lDKSwgQ2FtcHVzIGRlIFRlYXRpbm9zLCBNw6FsYWdhIEUtMjkwNzEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmV1esOzbiwgQ2FybWVuIFIuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvbG9nw61hIENlbHVsYXIsIEdlbsOpdGljYSB5IEZpc2lvbG9nw61hLCBJbnN0aXR1dG8gZGUgSG9ydG9mcnV0aWN1bHR1cmEgU3VidHJvcGljYWwgeSBNZWRpdGVycsOhbmVhLCBVbml2ZXJzaWRhZCBkZSBNw6FsYWdhLUNvbnNlam8gU3VwZXJpb3IgZGUgSW52ZXN0aWdhY2lvbmVzIENpZW50w61maWNhcyAoSUhTTS1VTUEtQ1NJQyksIENhbXB1cyBkZSBUZWF0aW5vcywgTcOhbGFnYSBFLTI5MDcxLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGUgTm92byBBc3NlbWJseSBhbmQgRnVuY3Rpb25hbCBBbm5vdGF0aW9uIG9mIHRoZSBPbGl2ZSAoT2xlYSBldXJvcGFlYSkgVHJhbnNjcmlwdG9tZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjkzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3MwMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMy8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9saXZlIGdyb3Zlczog4oCYVGhlIGxpZmUgYW5kIGlkZW50aXR5IG9mIHRoZSBNZWRpdGVycmFuZWFuOyBBZ3JpYy4gSHVtLiBWYWx1ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz04Ny05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iRkFPU1RBVCAyMDA4IGhvbWUgcGFnZSBDb2x1bWJpYSBVUkw6IGh0dHA6Ly93d3cuY29sdW1iaWEuZWR1L2NnaS1iaW4vY3VsL3Jlc29sdmU/QVNMOTYwOSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbnNlbnN1cyBsaXN0IG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZm9yIG9saXZlIGdlbm90eXBpbmc7IE1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjEzLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlzaW9sb2dpY2FsLCBiaW9jaGVtaWNhbCBhbmQgbW9sZWN1bGFyIGNoYW5nZXMgb2NjdXJyaW5nIGR1cmluZyBvbGl2ZSBkZXZlbG9wbWVudCBhbmQgcmlwZW5pbmc7IEouIFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE2NTsgIGNpdGF0aW9uX3BhZ2VzPTE1NDUtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBvZiB0cmlwbG9pZCBhbmQgdGV0cmFwbG9pZCBvbGl2ZSAoT2xlYSBldXJvcGFlYSBMLikgcGxhbnRzIGZyb20gbWl4b3Bsb2lkIGN2LiDigJhGcmFudG9pb+KAmSBhbmQg4oCYTGVjY2lub+KAmSBtdXRhbnRzIGJ5IGluIHZpdm8gYW5kIGluIHZpdHJvIHNlbGVjdGlvbjsgUGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTIzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmlyc3QgbGlua2FnZSBtYXAgb2Ygb2xpdmUgKE9sZWEgZXVyb3BhZWEgTC4pIGN1bHRpdmFycyB1c2luZyBSQVBELCBBRkxQLCBSRkxQIGFuZCBTU1IgbWFya2VyczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xMjczLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZWFyIEROQSBjb250ZW50IGFuZCBnZW5vbWUgc2l6ZSBvZiB0cm91dCBhbmQgaHVtYW47IEN5dG9tZXRyeSBBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9MTI3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbGluaXR5IFRvbGVyYW5jZSBpbiBPbGl2ZSwgaW4gSG9ydGljdWx0dXJhbCBSZXZpZXdzOyBjaXRhdGlvbl9wdWJsaXNoZXI9Sm9obiBXaWxleSAmYW1wOyBTb25zLCBJbmMuOiBPeGZvcmQsLCBVSzsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE3Ny0yMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdmFyaWFiaWxpdHkgYW5kIGhlcml0YWJpbGl0eSBvZiBzb21lIGNoYXJhY3RlcnMgaW4gY3Jvc3MtYnJlZCBvbGl2ZSBzZWVkbGluZ3M7IE9saXZhZTsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTIxLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWl6aW5nIGVhcmx5IGZsb3dlcmluZyBhbmQgcHJlLXNlbGVjdGlvbiBmb3Igc2hvcnQganV2ZW5pbGUgcGVyaW9kIGluIG9saXZlIHNlZWRsaW5nczsgSG9ydFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz01MTktMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPeKAmENoaXF1aXRpdGHigJkgb2xpdmU7IEhvcnRTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NTI5LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXR0eSBhY2lkIGNvbXBvc2l0aW9uIG9mIGFkdmFuY2VkIG9saXZlIHNlbGVjdGlvbnMgb2J0YWluZWQgYnkgY3Jvc3NicmVlZGluZzsgSi4gU2NpLiBGb29kIEFncmljLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTE5MjEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgNDU0IHB5cm9zZXF1ZW5jaW5nIG9mIHRyYW5zY3JpcHRzIGZyb20gdHdvIG9saXZlIGdlbm90eXBlcyBkdXJpbmcgZnJ1aXQgZGV2ZWxvcG1lbnQ7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTM5OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgYW5ub3RhdGlvbiBvZiBnZW5lcyBkaWZmZXJlbnRpYWxseSBleHByZXNzZWQgYWxvbmcgb2xpdmUgZnJ1aXQgZGV2ZWxvcG1lbnQ7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBhbmQgYW5hbHlzaXMgb2YgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3MgZnJvbSBPbGVhIGV1cm9wYWVhIEwuIENvbXA7IEZ1bmN0LiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDEwOyAgY2l0YXRpb25fcGFnZXM9NzU3NTEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgb2YgUk5BIHNpbGVuY2luZy1hc3NvY2lhdGVkIHNtYWxsIFJOQXMgaW4gb2xpdmUgKE9sZWEgZXVyb3BhZWEgTC4pOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTI3OTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoIHF1YWxpdHkgZHJhZnQgY29uc2Vuc3VzIHNlcXVlbmNlIG9mIHRoZSBnZW5vbWUgb2YgYSBoZXRlcm96eWdvdXMgZ3JhcGV2aW5lIHZhcmlldHk7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lMTMyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBncmFwZXZpbmUgZ2Vub21lIHNlcXVlbmNlIHN1Z2dlc3RzIGFuY2VzdHJhbCBoZXhhcGxvaWRpemF0aW9uIGluIG1ham9yIGFuZ2lvc3Blcm0gcGh5bGE7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDk7ICBjaXRhdGlvbl9wYWdlcz00NjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBibGFjayBjb3R0b253b29kLCBQb3B1bHVzIHRyaWNob2NhcnBhIChUb3JyLiAmYW1wOyBHcmF5KTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTM7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHN0dWRpZXMgaW4gb2xpdmUgKE9sZWEgZXVyb3BhZWEgTC4pOiBvdmVydmlldyBvbiBETkEgbWFya2VycyBhcHBsaWNhdGlvbnMgYW5kIHJlY2VudCBhZHZhbmNlcyBpbiBnZW5vbWUgYW5hbHlzaXM7IFBsYW50IENlbGwgUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTQ0OS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVjdGluIGVzdGVyYXNlIGdlbmUgZmFtaWx5IGluIHN0cmF3YmVycnkgZnJ1aXQ6IHN0dWR5IG9mIEZhUEUxLCBhIHJpcGVuaW5nLXNwZWNpZmljIGlzb2Zvcm07IEouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTkwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGdlbmVzIGVuY29kaW5nIHR3byBtaWNyb3NvbWFsIG9sZWF0ZSBkZXNhdHVyYXNlcyAoRkFEMikgZnJvbSBvbGl2ZTsgUGh5dG9jaGVtaXN0cnk7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjY7ICBjaXRhdGlvbl9wYWdlcz0xNDE3LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UUkl6b2w6IEEgbmV3IHJlYWdlbnQgZm9yIG9wdGltYWwgc2luZ2xlLXN0ZXAgaXNvbGF0aW9uIG9mIFJOQTsgRm9jdXM7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz05OS0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdXNlZnVsIHByb3RvY29sIGZvciBpbiBzaXR1IFJULVBDUiBvbiBwbGFudCB0aXNzdWVzOyBDZWxsLiBNb2wuIEJpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz03LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXZlcnNlIHRyYW5zY3JpcHRhc2UgdGVtcGxhdGUgc3dpdGNoaW5nOiBhIFNNQVJUIGFwcHJvYWNoIGZvciBmdWxsLWxlbmd0aCBjRE5BIGxpYnJhcnkgY29uc3RydWN0aW9uOyBCaW90ZWNobmlxdWVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9ODkyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBjRE5BIG5vcm1hbGl6YXRpb24gdXNpbmcga2FtY2hhdGthIGNyYWIgZHVwbGV4IHNwZWNpZmljIG51Y2xlYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1lMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIG1ldGhvZCBmb3IgU05QIGRldGVjdGlvbiB1c2luZyBhIG5ldyBkdXBsZXgtc3BlY2lmaWMgbnVjbGVhc2UgZnJvbSBjcmFiIGhlcGF0b3BhbmNyZWFzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE5MzUtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBjRE5BIGxpYnJhcnkgb2YgY290dG9uIG11dGFudCAoWGlhbmdtaWFuLTE4KSBsaWJyYXJ5IGR1cmluZyBnbGFuZCBmb3JtaW5nIHN0YWdlOyBDb2xsb2lkcyBTdXJmLiBCIEJpb2ludGVyZmFjZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0yNTgtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUljZS1iaW5kaW5nIHByb3RlaW5zIGZyb20gZW5va2kgYW5kIHNoaWl0YWtlIG11c2hyb29tczsgQ3J5b2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0xNTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgdHJhbnNjcmlwdG9tZSBjaGFyYWN0ZXJpemF0aW9uIGZvciBhIG5vbm1vZGVsIG9yZ2FuaXNtIHVzaW5nIDQ1NCBweXJvc2VxdWVuY2luZzsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE2MzYtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIGluIG1pY3JvZmFicmljYXRlZCBoaWdoLWRlbnNpdHkgcGljb2xpdHJlIHJlYWN0b3JzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM3OyAgY2l0YXRpb25fcGFnZXM9Mzc2LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DQVAzOiBhIEROQSBzZXF1ZW5jZSBhc3NlbWJseSBwcm9ncmFtOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ODY4LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJPbGl2ZXJvcyBKLkMuIFZFTk5ZLiBBbiBpbnRlcmFjdGl2ZSB0b29sIGZvciBjb21wYXJpbmcgbGlzdHMgd2l0aCBWZW5uIERpYWdyYW1zIDIwMDcgaHR0cDovL2Jpb2luZm9ncC5jbmIuY3NpYy5lcy90b29scy92ZW5ueS9pbmRleC5odG1sLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIHRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2YgdGhlIGhpZ2hseSBjb21wbGV4IFBpc3VtIHNhdGl2dW0gZ2Vub21lIHVzaW5nIG5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yMjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EYXRhYmFzZSBvZiBob21vbG9neS1kZXJpdmVkIHByb3RlaW4gc3RydWN0dXJlcyBhbmQgdGhlIHN0cnVjdHVyYWwgbWVhbmluZyBvZiBzZXF1ZW5jZSBhbGlnbm1lbnQ7IFByb3RlaW5zOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz01Ni02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdpbGlnaHQgem9uZSBvZiBwcm90ZWluIHNlcXVlbmNlIGFsaWdubWVudHM7IFByb3RlaW4gRW5nLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTg1LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CbGFzdDJHTzogYSB1bml2ZXJzYWwgdG9vbCBmb3IgYW5ub3RhdGlvbiwgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgaW4gZnVuY3Rpb25hbCBnZW5vbWljcyByZXNlYXJjaDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNjc0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Db2xkIFNwcmluZyBIYXJib3IgTGFib3JhdG9yeSBQcmVzczogQ29sZCBTcHJpbmcgSGFyYm9yLCBOZXcgWW9yaywgOyBTdGF0aXN0aWNzIGF0IHRoZSBCZW5jaDogQSBTdGVwLUJ5LVN0ZXAgSGFuZGJvb2sgZm9yIEJpb2xvZ2lzdHM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyb2xsaW5nIHRoZSBmYWxzZSBkaXNjb3ZlcnkgcmF0ZTogYSBwcmFjdGljYWwgYW5kIHBvd2VyZnVsIGFwcHJvYWNoIHRvIG11bHRpcGxlIHRlc3Rpbmc7IEouIFJveS4gU3RhdC4gU29jLiBCIE1ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0yODktMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaWJybyBJSS4gVmFyaWFiaWxpZGFkIHkgU2VsZWNjacOzbjsgY2l0YXRpb25fcHVibGlzaGVyPU1BUEEgeSBFZGljaW9uZXMgTXVuZGktUHJlbnNhLCBNYWRyaWQ7IFZhcmllZGFkZXMgZGUgb2xpdm8gZW4gRXNwYcOxYSwgSnVudGEgZGUgQW5kYWx1Y8OtYTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SnV2ZW5pbGl0eSBhbmQgbWF0dXJpdHkgb2YgcGxhbnRzIGFzIGluZmx1ZW5jZWQgYnkgdGhlaXIgb250b2dlbmV0aWNhbCBhbmQgcGh5c2lvbG9naWNhbCBhZ2Vpbmc7IEFjdGEgSG9ydC4gKElTSFMpOyAgY2l0YXRpb25feWVhcj0xOTc2OyAgY2l0YXRpb25fdm9sdW1lPTU2OyAgY2l0YXRpb25fcGFnZXM9MzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUp1dmVuaWxpdHkgYW5kIG1hdHVyYXRpb24gaW4gY29uaWZlcnM6IGN1cnJlbnQgY29uY2VwdHM7IFRyZWUgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz00MzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhhc2UgY2hhbmdlIGFuZCB0aGUgcmVndWxhdGlvbiBvZiBzaG9vdCBtb3JwaG9nZW5lc2lzIGluIHBsYW50czsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTA7ICBjaXRhdGlvbl9wYWdlcz05MjMtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZHVjdGlvbnMgYWJvdXQgdGhlIG51bWJlciwgb3JnYW5pemF0aW9uLCBhbmQgZXZvbHV0aW9uIG9mIGdlbmVzIGluIHRoZSB0b21hdG8gZ2Vub21lIGJhc2VkIG9uIGFuYWx5c2lzIG9mIGEgbGFyZ2UgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZyBjb2xsZWN0aW9uIGFuZCBzZWxlY3RpdmUgZ2Vub21pYyBzZXF1ZW5jaW5nOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTQ0MS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTWNDb3VydCBSLk0uIENoYXBtYW4gUi5MLiBCdWNoaGVpbSBNLiBNaXNobGVyIEIuRC4gR3JlZW4gcGxhbnRzIDE5OTYgaHR0cDovL3RvbHdlYi5vcmcvR3JlZW5fcGxhbnRzLzIzODIvMTk5Ni4wMS4wMSBpbiBUaGUgVHJlZSBvZiBMaWZlIFdlYiBQcm9qZWN0IGh0dHA6Ly90b2x3ZWIub3JnLyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJVbmlwcm90IHRheG9ub215IGh0dHA6Ly93d3cudW5pcHJvdC5vcmcvdGF4b25vbXkvICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BpbmcgYSBjb3JlIGNvbGxlY3Rpb24gb2Ygb2xpdmUgKE9sZWEgZXVyb3BhZWEgTC4pIGJhc2VkIG9uIG1vbGVjdWxhciBtYXJrZXJzIChEQXJUcywgU1NScywgU05QcykgYW5kIGFncm9ub21pYyB0cmFpdHM7IFRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM2NS03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIwLzEvOTMvMTA4ODI4Mi9kc3MwMzYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIE9saXZlIGJyZWVkaW5nIHByb2dyYW1tZXMgYXJlIGZvY3VzZWQgb24gc2VsZWN0aW5nIGZvciB0cmFpdHMgYXMgc2hvcnQganV2ZW5pbGUgcGVyaW9kLCBwbGFudCBhcmNoaXRlY3R1cmUgc3VpdGVkIGZvciBtZWNoYW5pY2FsIGhhcnZlc3QsIG9yIG9pbCBjaGFyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zODQyOTEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkUirEnWAuVYXR8YoYY1gAiUh/lK0=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f93%2f384291" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f93%2f384291" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14916" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&amp;Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/1">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5649560">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5649560">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5649562">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5649562">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5649567">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5649567">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5649586">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5649586">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5649628">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5649628">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5649630">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5649630">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5649632">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5649632">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5649634">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5649634">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/1/79/383877">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"><em>De Novo</em> Assembly and Functional Annotation of the Olive (<em>Olea europaea</em>) Transcriptome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antonio Mu&#241;oz-M&#233;rida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Integrated Bioinformatics</div>, <div class="insititution">National Institute for Bioinformatics, University of Málaga</div>, <div class="addr-line">Campus de Teatinos, Málaga</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antonio+Mu%c3%b1oz-M%c3%a9rida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mu&#241;oz-M&#233;rida A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mu&#241;oz-M&#233;rida A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Antonio Muñoz-Mérida</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Juan Jos&#233; Gonz&#225;lez-Plaza</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Biología Celular, Genética y Fisiología</div>, <div class="insititution">Instituto de Hortofruticultura Subtropical y Mediterránea, Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC)</div>, <div class="addr-line">Campus de Teatinos, Málaga E-29071</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Juan+Jos%c3%a9+Gonz%c3%a1lez-Plaza">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gonz&#225;lez-Plaza J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gonz&#225;lez-Plaza J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Juan José González-Plaza</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andr&#233;s Ca&#241;ada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO)</div>, <div class="addr-line">Melchor Fernández Almagro 3, Madrid E-28029</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andr%c3%a9s+Ca%c3%b1ada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ca&#241;ada A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ca&#241;ada A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Andrés Cañada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ana Mar&#237;a Blanco</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Sistemas Genómicos S.L.</div>, <div class="addr-line">Parque Tecnológico de Valencia, Ronda G. Marconi 6, Paterna E-46980</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ana+Mar%c3%ada+Blanco">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Blanco A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Blanco A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ana María Blanco</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maria del Carmen Garc&#237;a-L&#243;pez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Dpto. de Biología Experimental</div>, <div class="insititution">Universidad de Jaén</div>, <div class="addr-line">Campus de las Lagunillas s/n. Edif. B-3, Jaén 23071</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maria+del+Carmen+Garc%c3%ada-L%c3%b3pez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Garc&#237;a-L&#243;pez M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Garc&#237;a-L&#243;pez M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Maria del Carmen García-López</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jos&#233; Manuel Rodr&#237;guez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO)</div>, <div class="addr-line">Melchor Fernández Almagro 3, Madrid E-28029</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jos%c3%a9+Manuel+Rodr%c3%adguez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rodr&#237;guez J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rodr&#237;guez J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">José Manuel Rodríguez</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Laia Pedrola</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Life Sequencing S.L.</div>, <div class="insititution">Parc Científic Universitat de Valencia</div>, <div class="addr-line">9, Catedrático Agustín Escardino Benlloch Edificio N2 BIOTECH, Paterna, Valencia</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Laia+Pedrola">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pedrola L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pedrola L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Laia Pedrola</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M. Dolores Sicardo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">Instituto de la Grasa (IG-CSIC)</div>, <div class="addr-line">Avenida Padre García Tejero 4, Seville 41012</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.+Dolores+Sicardo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sicardo M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sicardo M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">M. Dolores Sicardo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M. Luisa Hern&#225;ndez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">Instituto de la Grasa (IG-CSIC)</div>, <div class="addr-line">Avenida Padre García Tejero 4, Seville 41012</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.+Luisa+Hern%c3%a1ndez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hern&#225;ndez M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hern&#225;ndez M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">M. Luisa Hernández</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ra&#250;l De la Rosa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="insititution">IFAPA Centro Alameda del Obispo</div>, <div class="addr-line">Avda Menéndez Pidal s/n 14004, Córdoba</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ra%c3%bal+De+la+Rosa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=De la Rosa R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;De la Rosa R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Raúl De la Rosa</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Angjelina Belaj</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="insititution">IFAPA Centro Alameda del Obispo</div>, <div class="addr-line">Avda Menéndez Pidal s/n 14004, Córdoba</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Angjelina+Belaj">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Belaj A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Belaj A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Angjelina Belaj</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mayte Gil-Borja</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Sistemas Genómicos S.L.</div>, <div class="addr-line">Parque Tecnológico de Valencia, Ronda G. Marconi 6, Paterna E-46980</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mayte+Gil-Borja">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gil-Borja M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gil-Borja M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mayte Gil-Borja</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Francisco Luque</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Dpto. de Biología Experimental</div>, <div class="insititution">Universidad de Jaén</div>, <div class="addr-line">Campus de las Lagunillas s/n. Edif. B-3, Jaén 23071</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Francisco+Luque">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Luque F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Luque F&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Francisco Luque</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jos&#233; Manuel Mart&#237;nez-Rivas</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">Instituto de la Grasa (IG-CSIC)</div>, <div class="addr-line">Avenida Padre García Tejero 4, Seville 41012</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jos%c3%a9+Manuel+Mart%c3%adnez-Rivas">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mart&#237;nez-Rivas J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mart&#237;nez-Rivas J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">José Manuel Martínez-Rivas</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David G. Pisano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Instituto Nacional de Bioinformática, Spanish National Cancer Research Centre (CNIO)</div>, <div class="addr-line">Melchor Fernández Almagro 3, Madrid E-28029</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=David+G.+Pisano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pisano D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pisano D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">David G. Pisano</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Oswaldo Trelles</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Integrated Bioinformatics</div>, <div class="insititution">National Institute for Bioinformatics, University of Málaga</div>, <div class="addr-line">Campus de Teatinos, Málaga</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Oswaldo+Trelles">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Trelles O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Trelles O&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Oswaldo Trelles</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Victoriano Valpuesta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div><div class="addr-line">Instituto de Hortofruticultura Subtropical y Mediterránea, Dpto. Biología Molecular y Bioquímica</div>, <div class="insititution">Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC)</div>, <div class="addr-line">Campus de Teatinos, Málaga E-29071</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Victoriano+Valpuesta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Valpuesta V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Valpuesta V&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Victoriano Valpuesta</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Carmen R. Beuz&#243;n</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Biología Celular, Genética y Fisiología</div>, <div class="insititution">Instituto de Hortofruticultura Subtropical y Mediterránea, Universidad de Málaga-Consejo Superior de Investigaciones Científicas (IHSM-UMA-CSIC)</div>, <div class="addr-line">Campus de Teatinos, Málaga E-29071</div>, <div class="country">Spain</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +34 952-131959. Fax. +34 952-132001. Email: <a href="mailto:cbl@uma.es" target="_blank">cbl@uma.es</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Carmen+R.+Beuz%c3%b3n">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Beuz&#243;n C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Beuz&#243;n C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Carmen R. Beuzón</a>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 93–108, <a href='https://doi.org/10.1093/dnares/dss036'>https://doi.org/10.1093/dnares/dss036</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 January 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 July 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 November 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="384291" href="/dnaresearch/article-pdf/20/1/93/1088282/dss036.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Antonio Muñoz-Mérida, Juan José González-Plaza, Andrés Cañada, Ana María Blanco, Maria del Carmen García-López, José Manuel Rodríguez, Laia Pedrola, M. Dolores Sicardo, M. Luisa Hernández, Raúl De la Rosa, Angjelina Belaj, Mayte Gil-Borja, Francisco Luque, José Manuel Martínez-Rivas, David G. Pisano, Oswaldo Trelles, Victoriano Valpuesta, Carmen R. Beuzón;  <em>De Novo</em> Assembly and Functional Annotation of the Olive (<em>Olea europaea</em>) Transcriptome, <em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 93–108, <a href='https://doi.org/10.1093/dnares/dss036'>https://doi.org/10.1093/dnares/dss036</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/384291?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/384291?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/384291?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/384291?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/384291?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/384291?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/384291?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%26lt%3Bem%26gt%3BDe%20Novo%26lt%3B%2Fem%26gt%3B%20Assembly%20and%20Functional%20Annotation%20of%20the%20Olive%20(%26lt%3Bem%26gt%3BOlea%20europaea%26lt%3B%2Fem%26gt%3B)%20Transcriptome&amp;publicationDate=2013-01-07&amp;volumeNum=20&amp;issueNum=1&amp;author=Mu%C3%B1oz-M%C3%A9rida%2C%20Antonio%3B%20Gonz%C3%A1lez-Plaza%2C%20Juan%20Jos%C3%A9&amp;startPage=93&amp;endPage=108&amp;contentId=10.1093%2Fdnares%2Fdss036&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f93%2f384291" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5649560 id="5649560" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Olive breeding programmes are focused on selecting for traits as short juvenile period, plant architecture suited for mechanical harvest, or oil characteristics, including fatty acid composition, phenolic, and volatile compounds to suit new markets. Understanding the molecular basis of these characteristics and improving the efficiency of such breeding programmes require the development of genomic information and tools. However, despite its economic relevance, genomic information on olive or closely related species is still scarce. We have applied Sanger and 454 pyrosequencing technologies to generate close to 2 million reads from 12 cDNA libraries obtained from the Picual, Arbequina, and Lechin de Sevilla cultivars and seedlings from a segregating progeny of a Picual × Arbequina cross. The libraries include fruit mesocarp and seeds at three relevant developmental stages, young stems and leaves, active juvenile and adult buds as well as dormant buds, and juvenile and adult roots. The reads were assembled by library or tissue and then assembled together into 81 020 unigenes with an average size of 496 bases. Here, we report their assembly and their functional annotation.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Olea europaea&quot;"><em>Olea europaea</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;454 pyrosequencing&quot;">454 pyrosequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genomics">genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Sanger sequencing&quot;">Sanger sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a></div>                            <h2 scrollto-destination=5649562 id="5649562" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Olea europaea</em> L. is one of the most important and widespread fruit trees in the Mediterranean basin, with 95% of the 750 million olive trees cultivated worldwide, being Spain and Italy the main growers.<sup><a reveal-id="DSS036C1" data-open="DSS036C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS036C2" data-open="DSS036C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Olive, one of the first plants to be cultivated for oil production, is the second most important oil fruit crop cultivated worldwide, with over 8 million hectares of land devoted to its cultivation.<sup><a reveal-id="DSS036C3" data-open="DSS036C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Most olive production is destined for olive oil, ranking sixth of all vegetable oils produced worldwide.<sup><a reveal-id="DSS036C4" data-open="DSS036C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Virgin olive oil is a natural fruit juice with no additives or preservatives and a unique flavour, increasingly demanded for its potential health benefits, exceptional nutritional properties, and exceptional organoleptic properties. These characteristics, and a high oxidative stability, are all due to a well-balanced fatty acid composition and the presence of minor components such as natural antioxidants and vitamins.</p><p>Olive is a diploid (2<em>n</em> = 46), predominantly allogamous, vegetatively propagated species, with an estimated genome size of 2200 Mb<sup><a reveal-id="DSS036C5  DSS036C7" data-open="DSS036C5  DSS036C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> and a higher tolerance to drought and salt stresses than other fruit trees.<sup><a reveal-id="DSS036C8" data-open="DSS036C8" class="link link-ref link-reveal xref-bibr">8</a></sup> As many woody plants, olive requires several growth seasons from seed germination to first flowering (juvenile phase). This long juvenile phase had severely hampered the generation of new breeding cultivars that could replace ancient ones still under cultivation.<sup><a reveal-id="DSS036C9" data-open="DSS036C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Therefore, new cultivars adapted to the new trends in olive growing, i.e. shorter juvenile periods,<sup><a reveal-id="DSS036C10" data-open="DSS036C10" class="link link-ref link-reveal xref-bibr">10</a></sup> tree architecture suitable for hedgerow orchards and mechanical harvesting,<sup><a reveal-id="DSS036C11" data-open="DSS036C11" class="link link-ref link-reveal xref-bibr">11</a></sup> or increased oil content and organoleptic profiles tailored to different markets,<sup><a reveal-id="DSS036C12" data-open="DSS036C12" class="link link-ref link-reveal xref-bibr">12</a></sup> are needed.<sup><a reveal-id="DSS036C11" data-open="DSS036C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>Understanding the complex molecular basis of fruit and oil quality and that of traits of agronomical importance can be greatly helped by the application of genomic tools. Furthermore, development of new molecular markers to assist selection in breeding programmes looking for cultivars with more desirable phenotypes for any of these traits can be greatly helped through the generation and application of genomic tools. However, despite its economical relevance, the generation of sequence information and genomic tools for either olive or any closely related species has only just started.<sup><a reveal-id="DSS036C13  DSS036C16" data-open="DSS036C13  DSS036C16" class="link link-ref link-reveal xref-bibr">13–16</a></sup> Genomic information in olive is well behind other species of woody plants, such as <em>Vitis vinifera</em><sup><a reveal-id="DSS036C17" data-open="DSS036C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS036C18" data-open="DSS036C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and <em>Populus trichocarpa</em>,<sup><a reveal-id="DSS036C19" data-open="DSS036C19" class="link link-ref link-reveal xref-bibr">19</a></sup> whose complete genome sequences are already available, although the first sequences for these species became available around the same time, or even later, than those from olive.<sup><a reveal-id="DSS036C20" data-open="DSS036C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Without a genome sequence, transcriptome sequencing is an effective approach for gene discovery and identification of transcripts involved in specific biological processes (BP).</p><p>In this study, we report the largest contribution to date to sequence information in this important crop species, with close to 2 M reads generated through the combined use of Sanger sequencing and 454 pyrosequencing technologies, for the identification of expressed sequence tags (ESTs) from different plant tissues and developmental stages. The assembly of the 2 M reads has rendered over 81 020 unigenes, and we present here the result of this assembly and its functional annotation.</p>                            <h2 scrollto-destination=5649567 id="5649567" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5649568 id="5649568" class="section-title" data-legacy-id=s2a>2.1. Plant material for the generation of cDNA libraries and collections</h3>
<p>Four cDNA libraries were generated for Sanger sequencing (Table <a reveal-id="DSS036TB1" data-open="DSS036TB1" class="link link-reveal link-table xref-fig">1</a>). To generate a mesocarp cDNA library (OLmeso), fruit mesocarp was collected from fruits of Lechin de Sevilla, harvested at three relevant stages of maturation: green with lignified endocarp (18 WAF), turning (27 WAF), and purple (34 WAF). For the bud cDNA library (OLmer), buds containing actively growing meristems harvested from juvenile low branches and adult high branches of 10 segregating seedlings from a Picual × Arbequina cross of the UCO-IFAPA Olive Breeding Program of Córdoba, Spain, were taken by mid April 2008. Roots from a single adult tree of Lechin de Sevilla and 10 different 2-month-old trees from the segregating progeny of the Picual × Arbequina cross were also harvested to generate a root cDNA library (OLroot). Young leaves and stems of Lechin de Sevilla were collected and used together with seeds from olive fruits of the Picual × Arbequina progeny at two different maturation stages: green with lignified endocarp (18 WAF) and turning (27 WAF) to generate OLrest. The tissues used to generate the eight cDNA collections used for pyrosequencing are detailed in Table <a reveal-id="DSS036TB1" data-open="DSS036TB1" class="link link-reveal link-table xref-fig">1</a>. </p>                        <a id="5649570" scrollto-destination="5649570"></a>
<div content-id="DSS036TB1" class="table-modal"><div class="table-wrap"><div id="DSS036TB1" class="table-wrap-title" data-id="DSS036TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Description of the olive cDNA libraries and collections generated in this work</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Variety </th><th>Name of the library </th><th>Description </th></tr></thead><tbody><tr><td colspan="3">Sanger sequencing </td></tr><tr><td> Lechín de Sevilla </td><td>OLmeso </td><td>Mesocarp from fruits at 2 stages of maturation: green with lignified endocarp (18 WAF), turning (27 WAF), and purple (34 WAF). </td></tr><tr><td> Picual × Arbequina </td><td>OLmer </td><td>Buds from young and adult branches, from 10 segregating trees. Full RNA was purified from each sample and pooled adding equal amounts of RNA from juvenile and adult tissue from each tree. </td></tr><tr><td> Lechín de Sevilla;  Picual × Arbequina </td><td>OLroot </td><td>One root sample from adult trees of Lechin de Sevilla and 10 root samples from juvenile trees from a segregating progeny of the Picual × Arbequina cross. Full RNA was purified from each sample and pooled adding equal amounts from adult and juvenile roots. </td></tr><tr><td> Picual × Arbequina; Lechín de Sevilla </td><td>OLrest </td><td>Young leaves and stems of Lechin de Sevilla and seeds from fruits at two different maturation stages, green with lignified endocarp (18 WAF) and turning (27WAF) from the segregating progeny of a Picual × Arbequina cross. RNA was purified from each sample and pooled adding equal amounts of RNA from each tissue and stage. </td></tr><tr><td colspan="3">454 Pyrosequencing </td></tr><tr><td> Picual </td><td>MAP </td><td>Active buds. </td></tr><tr><td> Picual </td><td>MIP </td><td>Dormant (inactive) buds. </td></tr><tr><td> Arbequina </td><td>MAA </td><td>Active buds. </td></tr><tr><td> Arbequina </td><td>MIA </td><td>Dormant (inactive) buds. </td></tr><tr><td> Arbequina × Picual </td><td>SGAP </td><td>Seed from green fruits with lignified endocarp (18 WAF). </td></tr><tr><td> Picual </td><td>MGP </td><td>Mesocarp from green fruits with lignified endocarp (18 WAF). </td></tr><tr><td> Arbequina </td><td>MTA </td><td>Mesocarp from fruits at turning stage (27 WAF). </td></tr><tr><td> Picual </td><td>MTP </td><td>Mesocarp from fruits at turning stage (27 WAF). </td></tr></tbody></table></div><div content-id="DSS036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Variety </th><th>Name of the library </th><th>Description </th></tr></thead><tbody><tr><td colspan="3">Sanger sequencing </td></tr><tr><td> Lechín de Sevilla </td><td>OLmeso </td><td>Mesocarp from fruits at 2 stages of maturation: green with lignified endocarp (18 WAF), turning (27 WAF), and purple (34 WAF). </td></tr><tr><td> Picual × Arbequina </td><td>OLmer </td><td>Buds from young and adult branches, from 10 segregating trees. Full RNA was purified from each sample and pooled adding equal amounts of RNA from juvenile and adult tissue from each tree. </td></tr><tr><td> Lechín de Sevilla;  Picual × Arbequina </td><td>OLroot </td><td>One root sample from adult trees of Lechin de Sevilla and 10 root samples from juvenile trees from a segregating progeny of the Picual × Arbequina cross. Full RNA was purified from each sample and pooled adding equal amounts from adult and juvenile roots. </td></tr><tr><td> Picual × Arbequina; Lechín de Sevilla </td><td>OLrest </td><td>Young leaves and stems of Lechin de Sevilla and seeds from fruits at two different maturation stages, green with lignified endocarp (18 WAF) and turning (27WAF) from the segregating progeny of a Picual × Arbequina cross. RNA was purified from each sample and pooled adding equal amounts of RNA from each tissue and stage. </td></tr><tr><td colspan="3">454 Pyrosequencing </td></tr><tr><td> Picual </td><td>MAP </td><td>Active buds. </td></tr><tr><td> Picual </td><td>MIP </td><td>Dormant (inactive) buds. </td></tr><tr><td> Arbequina </td><td>MAA </td><td>Active buds. </td></tr><tr><td> Arbequina </td><td>MIA </td><td>Dormant (inactive) buds. </td></tr><tr><td> Arbequina × Picual </td><td>SGAP </td><td>Seed from green fruits with lignified endocarp (18 WAF). </td></tr><tr><td> Picual </td><td>MGP </td><td>Mesocarp from green fruits with lignified endocarp (18 WAF). </td></tr><tr><td> Arbequina </td><td>MTA </td><td>Mesocarp from fruits at turning stage (27 WAF). </td></tr><tr><td> Picual </td><td>MTP </td><td>Mesocarp from fruits at turning stage (27 WAF). </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5649570" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS036TB1" class="table-wrap-title" data-id="DSS036TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Description of the olive cDNA libraries and collections generated in this work</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Variety </th><th>Name of the library </th><th>Description </th></tr></thead><tbody><tr><td colspan="3">Sanger sequencing </td></tr><tr><td> Lechín de Sevilla </td><td>OLmeso </td><td>Mesocarp from fruits at 2 stages of maturation: green with lignified endocarp (18 WAF), turning (27 WAF), and purple (34 WAF). </td></tr><tr><td> Picual × Arbequina </td><td>OLmer </td><td>Buds from young and adult branches, from 10 segregating trees. Full RNA was purified from each sample and pooled adding equal amounts of RNA from juvenile and adult tissue from each tree. </td></tr><tr><td> Lechín de Sevilla;  Picual × Arbequina </td><td>OLroot </td><td>One root sample from adult trees of Lechin de Sevilla and 10 root samples from juvenile trees from a segregating progeny of the Picual × Arbequina cross. Full RNA was purified from each sample and pooled adding equal amounts from adult and juvenile roots. </td></tr><tr><td> Picual × Arbequina; Lechín de Sevilla </td><td>OLrest </td><td>Young leaves and stems of Lechin de Sevilla and seeds from fruits at two different maturation stages, green with lignified endocarp (18 WAF) and turning (27WAF) from the segregating progeny of a Picual × Arbequina cross. RNA was purified from each sample and pooled adding equal amounts of RNA from each tissue and stage. </td></tr><tr><td colspan="3">454 Pyrosequencing </td></tr><tr><td> Picual </td><td>MAP </td><td>Active buds. </td></tr><tr><td> Picual </td><td>MIP </td><td>Dormant (inactive) buds. </td></tr><tr><td> Arbequina </td><td>MAA </td><td>Active buds. </td></tr><tr><td> Arbequina </td><td>MIA </td><td>Dormant (inactive) buds. </td></tr><tr><td> Arbequina × Picual </td><td>SGAP </td><td>Seed from green fruits with lignified endocarp (18 WAF). </td></tr><tr><td> Picual </td><td>MGP </td><td>Mesocarp from green fruits with lignified endocarp (18 WAF). </td></tr><tr><td> Arbequina </td><td>MTA </td><td>Mesocarp from fruits at turning stage (27 WAF). </td></tr><tr><td> Picual </td><td>MTP </td><td>Mesocarp from fruits at turning stage (27 WAF). </td></tr></tbody></table></div><div content-id="DSS036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Variety </th><th>Name of the library </th><th>Description </th></tr></thead><tbody><tr><td colspan="3">Sanger sequencing </td></tr><tr><td> Lechín de Sevilla </td><td>OLmeso </td><td>Mesocarp from fruits at 2 stages of maturation: green with lignified endocarp (18 WAF), turning (27 WAF), and purple (34 WAF). </td></tr><tr><td> Picual × Arbequina </td><td>OLmer </td><td>Buds from young and adult branches, from 10 segregating trees. Full RNA was purified from each sample and pooled adding equal amounts of RNA from juvenile and adult tissue from each tree. </td></tr><tr><td> Lechín de Sevilla;  Picual × Arbequina </td><td>OLroot </td><td>One root sample from adult trees of Lechin de Sevilla and 10 root samples from juvenile trees from a segregating progeny of the Picual × Arbequina cross. Full RNA was purified from each sample and pooled adding equal amounts from adult and juvenile roots. </td></tr><tr><td> Picual × Arbequina; Lechín de Sevilla </td><td>OLrest </td><td>Young leaves and stems of Lechin de Sevilla and seeds from fruits at two different maturation stages, green with lignified endocarp (18 WAF) and turning (27WAF) from the segregating progeny of a Picual × Arbequina cross. RNA was purified from each sample and pooled adding equal amounts of RNA from each tissue and stage. </td></tr><tr><td colspan="3">454 Pyrosequencing </td></tr><tr><td> Picual </td><td>MAP </td><td>Active buds. </td></tr><tr><td> Picual </td><td>MIP </td><td>Dormant (inactive) buds. </td></tr><tr><td> Arbequina </td><td>MAA </td><td>Active buds. </td></tr><tr><td> Arbequina </td><td>MIA </td><td>Dormant (inactive) buds. </td></tr><tr><td> Arbequina × Picual </td><td>SGAP </td><td>Seed from green fruits with lignified endocarp (18 WAF). </td></tr><tr><td> Picual </td><td>MGP </td><td>Mesocarp from green fruits with lignified endocarp (18 WAF). </td></tr><tr><td> Arbequina </td><td>MTA </td><td>Mesocarp from fruits at turning stage (27 WAF). </td></tr><tr><td> Picual </td><td>MTP </td><td>Mesocarp from fruits at turning stage (27 WAF). </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5649570" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5649571 id="5649571" class="section-title" data-legacy-id=s2b>2.2. RNA extraction, cDNA synthesis, and generation of cDNA libraries</h3>
<p>Total RNA was purified from mesocarp samples following previously described methods<sup><a reveal-id="DSS036C21" data-open="DSS036C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS036C22" data-open="DSS036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> that allow elimination of phenolic compounds and using the TURBO DNA-free kit (Ambion, USA) to remove genomic DNA. Oligotex (Qiagen) was used to obtain mRNA from total RNA to guarantee RNA integrity when mesocarp samples were to be used to generate cDNA libraries. Total RNA extraction from juvenile and adult roots, young leaves and stems, and from seeds of green and turning fruits were carried out using Spectrum Total RNA Kit (Sigma-Aldrich, Spain) after tissue processing with an electric homogenizer. Total RNA from buds was purified using a Trizol-based method (Life Technologies)<sup><a reveal-id="DSS036C23" data-open="DSS036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and treated twice with DNase.<sup><a reveal-id="DSS036C24" data-open="DSS036C24" class="link link-ref link-reveal xref-bibr">24</a></sup> In all cases, RNA concentration was quantified using a spectrophotometer at OD 260 nm, and integrity checked by electrophoresis in agarose gels.</p><p>For the generation of the four cDNA libraries for Sanger sequencing, and prior to cDNA synthesis, equal amounts of RNA from mesocarp from the different stages of maturation (OLmeso), equal amounts of RNA from juvenile and adult roots (OLroot), or buds (OLmer), and equal amounts of young leaves and stems, seeds from green and turning fruits (OLrest) were pooled together. Synthesis of cDNA was carried out using the SMART kit (DB Clontech, USA),<sup><a reveal-id="DSS036C25" data-open="DSS036C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and normalized using the DSN nuclease (TRIMMER-Direct cDNA Normalization Kit from Evrogen, JSC, Russia).<sup><a reveal-id="DSS036C26" data-open="DSS036C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSS036C27" data-open="DSS036C27" class="link link-ref link-reveal xref-bibr">27</a></sup> To confirm that the normalization process was correct, an aliquot was visualized by gel electrophoresis, and the presence of olive genes of known expression was confirmed for some of the tissues by polymerase chain reaction (PCR). cDNA insert preparation was treated with Proteinase K, cleaned, and digested with the appropriate restriction enzymes. cDNA was size fractionated by ChromaSpin-400 columns, ligated into the pDNR-LIB,<sup><a reveal-id="DSS036C28" data-open="DSS036C28" class="link link-ref link-reveal xref-bibr">28</a></sup> (OLroot and OLrest libraries), pAL17.1<sup><a reveal-id="DSS036C29" data-open="DSS036C29" class="link link-ref link-reveal xref-bibr">29</a></sup> (OLmeso library), or pSTBlue-1 (Novagen, Merck KGaA, Darmstadt, Germany) (OLmer library) vectors, previously dephosphorylated, and transformed into <em>Escherichia coli</em>. Libraries were titred and tested by randomly picking and amplifying a small number of clones using universal primers. The sizes of the PCR products ranged between 0.5 and 2 Kb.</p><p>When generating cDNA collections for pyrosequencing, the first strand cDNA synthesis was performed using the ‘RevertAid H Minus First Strand cDNA Synthesis Kit’ (Fermentas), and, the second strand cDNA was synthesized using <em>E. coli</em> DNA polymerase I and RNase H (Fermentas).</p>                            <h3 scrollto-destination=5649575 id="5649575" class="section-title" data-legacy-id=s2c>2.3. Sanger and 454 sequencing</h3>
<p>cDNA libraries were automatically sequenced from the 5′ end using either the T7 primer (OLmeso and OLmer) or the pDNRlib30-50 primer (OLroot and OLrest), and FASTA files were generated for bioinformatic processing. cDNA collections for pyrosequencing were sheared via nebulization into small fragments and sequenced using GS-FLX (MAP, MIP, MIA, and MAA) or GS-FLX Titanium (SGAP, MGP, MTA, and MTP) (Roche, Basel, Switzerland). Half of a picotiter plate was used for cDNA collections: MAP, MIP, MIA, and MAA, with single libraries being sequenced in separated areas of the plate, whereas a full plate was used for SGAP, MGP, MTA, and MTP. An initial quality filtering was carried out before the actual base calling, where 454 adapter sequences were removed. Transmission, storage, and handling of sequencing data were subsequently done using the standard flowgram format (SFF) files for compactness and easiness of usage. SFF files were created after image analysis using the GSRunProcessor Version 2.3 provided by 454 Roche through Lifesequencing S.L., Spain. Calling qualities and sequence extraction from SFF files were carried out using sfftools Version 2.3 also provided by 454 Roche through Lifesequencing.</p>                            <h3 scrollto-destination=5649577 id="5649577" class="section-title" data-legacy-id=s2d>2.4. EST processing protocols</h3>
<p>Most of the EST processing protocols were adapted from those described by Vera and collaborators.<sup><a reveal-id="DSS036C30" data-open="DSS036C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Based on preliminary data using different computational assembly pipelines, including the 454 GS Assembler software, we chose to assemble the 454 sequence data using SeqMan Pro 8.1.1 (DNASTAR Lasergene v8.0 Suite, DNASTAR, USA). During pre-assembly, sequencing ends were trimmed using a medium-stringency criteria, with the following SeqMan parameters: average quality score Q/n threshold of 12, a non-trace window size of 50, and a maximum of 2 Ns. As part of the preprocessing step, an exhaustive comparison against a prokaryotic database (Uniprot database: taxonomic divisions Bacteria and Archaea) was performed to remove all those sequences that potentially originated from external contamination. For sequences obtained using 454 technology, no scanning for vector sequences was carried out. In the actual assembly step, the assembler used a strategy combining quality weighting (when there was not enough coverage depth) and simple majority (when deeper coverage was present) for contig consensus determination. For large 454 sequencing projects, the software provider recommended a set of pre-assembly and assembly parameters; however, because they were optimized for shorter reads than those obtained in our study, we performed several benchmark assemblies under different sets of parameters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>). When final length and coverage of the contigs were considered adequate, the parameters identified as optimal, which were different from those that were vendor recommended, were used to assemble the data obtained from 454 sequencing. We used a minimum consecutive match size of 25 bases, a minimum match percentage of 70%, and a match spacing of 15-mer tags. We established a minimum length of 70 bases for a sequence to be considered for assembly, no gap penalty and a 0.70 gap length penalty, and a maximum mismatch in end bases of 0. No repeat handling was used. Sanger sequencing data was also assembled using SeqMan Pro 8.1.1, using the default parameters recommended, that thus allowed us to perform joint 454-Sanger assemblies. High stringency quality trimming was used to remove low-quality bases at the ends of the sequences. The parameters for pre-assembly and assembly were similar to those used for 454 cDNA collections, but the match percentage was set to 80%, and a match spacing of 150 was required. A minimum of 100 bases was required for a sequence to be assembled (because of Sanger sequencing technology obtaining longer reads). The maximum mismatch in the end bases was also set to 15 to help mitigate assembly problems, if untrimmed vectors or bad base calls are present at the end of the reads. ESTs screening for vector sequences was carried out as follows in Sanger libraries: OLmeso was screened for pAL17.1, OLmer for pSTBlue-1, and OLrest and OLroot for pDNR-LIB. After the pre-assembly (cleaning and trimming) step, most of the raw reads were considerably reduced in size because of the tendency of the Sanger sequencing technology to incorporate traces of vector sequence and lower quality calls at 3′ and 5′ ends. The complete assembly of all sequences was carried out using a sequential combination of two different assemblers. Briefly, the initial sequencing dataset was assembled using Newbler 2.6 (Roche Ltd)<sup><a reveal-id="DSS036C31" data-open="DSS036C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and default parameters for cDNA data (seed step = 12; seed length = 16, seed count = 1, minimum overlap length = 40, and minimum overlap identity = 90%). The resulting dataset of contigs and singletons was used as input for the second assembler, CAP3,<sup><a reveal-id="DSS036C32" data-open="DSS036C32" class="link link-ref link-reveal xref-bibr">32</a></sup> to collapse all related sequences that could not have done so during the first assembly step. The parameters used to perform the CAP3 assembly were a minimum overlap length of 60 bases and a minimum overlap identity of 90%. Unigenes generated from the assembly were further filtered for a minimum size of 200 nt. Unigenes generated were classified according to the tissues from which they included sequencing reads. The lists of unigenes detected in each of the tissues were compared using the software VENNY, to identify those unigenes expressed in a single or two or more tissues. This software displays the results of these comparisons as Venn diagrams (<a class="link link-uri" href="http://bioinfogp.cnb.csic.es/tools/venny/">http://bioinfogp.cnb.csic.es/tools/venny/</a>).<sup><a reveal-id="DSS036C33" data-open="DSS036C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p>                            <h3 scrollto-destination=5649579 id="5649579" class="section-title" data-legacy-id=s2e>2.5. Analysis of transcript and transcriptome coverage</h3>
<p>A mathematical analysis similar to rarefaction analysis and previously applied for similar purposes in pea (<em>Pisum sativum)</em> was used to estimate both transcript and transcriptome coverage.<sup><a reveal-id="DSS036C34" data-open="DSS036C34" class="link link-ref link-reveal xref-bibr">34</a></sup> A read pool was defined using all reads obtained after the cleaning and trimming process from which reads were randomly drawn to generate sets of reads (datasets) of increasing sizes. The number of tomato genes identified using BLASTX (Basic Local Alignment Search Tool 2.2.18)<sup><a reveal-id="DSS036C35" data-open="DSS036C35" class="link link-ref link-reveal xref-bibr">35</a></sup> to compare these different datasets against the tomato ‘ITAG2 protein database (<a class="link link-uri" href="http://solgenomics.net">http://solgenomics.net</a>)’ (e-value ≤ 10<sup>−4</sup>) was recorded for each dataset and plotted against the corresponding sample sizes. Data points, thus, obtained were fitted to a non-linear regression with the model <em>y</em> = <em>ax</em>/(<em>b</em> + <em>x</em>) (SigmaPlot Software, Systat Software Inc., Scientific Software Products). Curves are expected to converge against a fixed value parameter ‘<em>a</em>’ in the model function indicating an upper limit for gene detection, which correlated with a decreasing slope at higher sample sizes indicative of a decreasing potential to detect additional tomato genes with further sampling. This approach was also used to record tomato genes with more than 5, 10, and 100 hits to estimate transcript coverage.</p>                            <h3 scrollto-destination=5649581 id="5649581" class="section-title" data-legacy-id=s2f>2.6. Data availability</h3>
<p>The 454 sequenced read data reported in this manuscript have been deposited in the National Center for Biotechnology Information (NCBI) Sequence Read Archive and are available under the Accession Number (NCBI: SRA059461). The results from the assembly reported in this manuscript have been deposited in the NCBI Transcriptome Shotgun Assembly Archive and are available under the Accession Number (GABQ00000000).</p>                            <h3 scrollto-destination=5649583 id="5649583" class="section-title" data-legacy-id=s2g>2.7. Functional annotation</h3>
<p>Unigenes generated were functionally annotated based on the similarity with UniProt-annotated sequences, in particular using plants taxonomy. The annotation algorithm used, Sma3s (Pérez-Pulido A.J., Muñoz-Mérida A., Viguera E., Claros M.G., Trelles O., Sma3s: a three-step modular sequence annotator. In preparation), was developed at the University of Malaga and briefly consists on a restrictive BLAST<sup><a reveal-id="DSS036C35" data-open="DSS036C35" class="link link-ref link-reveal xref-bibr">35</a></sup> search with three steps of significance that sequentially retrieve annotations from: (i) already existing annotated sequences, (ii) orthologous sequences, and (iii) groups of sequences sharing a statistically significant pattern. Sma3s provides the following biological descriptors associated with each sequence: Gene ontology (GO) terms, Swiss-Prot keywords and pathways, InterPro domains, and IntAct interactions (Uniprot, UniProt Knowledgebase Release 2012_04). In the first step, minimal correspondence was established at a 90% of sequence identity with a sequence overlap of at least 90% of the query sequence length. In the second step, the threshold was established at 75% of per-residue identity and length overlapping of more than 75% of their length. Lastly, the third step uses all BLAST alignments related to each query sequence, which are classified as statistically significant by the Sander's formulae.<sup><a reveal-id="DSS036C36" data-open="DSS036C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSS036C37" data-open="DSS036C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The curve determined by the formulae binds the relationship between sequence similarity and alignment length observed, in sequences sharing structural similarity. Thus, those entries displaying identity length coordinates over the curve correspond to statistically significant hits. This analysis is necessary to distinguish true from false positives, when the level of sequence identity is not sufficiently high to unambiguously distinguish between protein pairs of similar and non-similar structures. The annotations resulting from these hits were reported by the algorithm.</p><p>The annotation generated by sma3s was loaded as a tab-separated file in the Blast2GO suite,<sup><a reveal-id="DSS036C38" data-open="DSS036C38" class="link link-ref link-reveal xref-bibr">38</a></sup> to retrieve the GO structure of our data. A directed acyclic graph (DAG)<sup><a reveal-id="DSS036C38" data-open="DSS036C38" class="link link-ref link-reveal xref-bibr">38</a></sup> was made with a &gt;2 unigenes per GO term as a cutoff to overcome the computing limitations encountered as a result of the large size of our dataset. Statistics of our annotation dataset were saved into a Microsoft Excel datasheet, and charts were generated. Statistical analysis of GO terms enrichment was carried out using the Blast2GO suite<sup><a reveal-id="DSS036C38" data-open="DSS036C38" class="link link-ref link-reveal xref-bibr">38</a></sup> to perform a two-tailed Fisher's exact test,<sup><a reveal-id="DSS036C39" data-open="DSS036C39" class="link link-ref link-reveal xref-bibr">39</a></sup> a non-parametric test for independence that calculates the false discovery rate (FDR) following the method described by Benjamini and Hochberg.<sup><a reveal-id="DSS036C40" data-open="DSS036C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The test was carried out using a term filter value of 0.05.</p>                            <h2 scrollto-destination=5649586 id="5649586" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5649587 id="5649587" class="section-title" data-legacy-id=s3a>3.1. Experimental design</h3>
<p>Our primary interest focuses in three major traits: (i) fruit and oil quality, (ii) juvenility, and (iii) plant architecture. Selection of the adequate tissues, developmental stages, and varieties, from which ESTs would be obtained, is therefore paramount. Thus, cultivars Picual and Arbequina were selected for having very different characteristics regarding fruit and oil organoleptic properties, as well as tree architecture, whereas cultivar Lechin de Sevilla was included in the study for showing intermediate phenotypes for all these traits<sup><a reveal-id="DSS036C41" data-open="DSS036C41" class="link link-ref link-reveal xref-bibr">41</a></sup> (Table <a reveal-id="DSS036TB1" data-open="DSS036TB1" class="link link-reveal link-table xref-fig">1</a>). In addition, they are among the varieties most widely cultivated worldwide.</p><p>To get information about genes involved in determining oil content and composition, we selected mesocarp and seed tissues from fruits of different varieties at stages of maturation that correspond to the relevant phases of fruit development and ripening with respect to oil accumulation (Table <a reveal-id="DSS036TB1" data-open="DSS036TB1" class="link link-reveal link-table xref-fig">1</a>). In particular, we selected mesocarp and seed tissues from green fruits with lignified endocarp as the earliest developmental stage because the lignification of the endocarp marks the start of the triacylglycerol synthesis and oil accumulation in both, the mesocarp, with a major contribution to the final composition of the olive oil and the seed. Unlike the seed, in the mesocarp, the increase in oil content continues during the ripening process that is characterized by a change in colour of the fruit, turning from green to purple and then black. Therefore, we also selected turning and purple mesocarp. In this way, we have generated five cDNA library/ collections for Sanger sequencing or pyrosequencing to obtain ESTs related to oil quality.</p><p>Determining the length of the juvenile period and the plant architecture are processes strongly associated with meristematic activity. During the transition from juvenile to adult, a maturation gradient is developed in plants from the basal parts formed by juvenile apical meristems, to the upper parts formed by adult meristems.<sup><a reveal-id="DSS036C10" data-open="DSS036C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS036C42" data-open="DSS036C42" class="link link-ref link-reveal xref-bibr">42</a>–<a reveal-id="DSS036C44" data-open="DSS036C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Thus, we generated a cDNA library using RNA extracted from actively growing buds and four cDNA collections using RNA extracted from buds containing active or inactive (dormant) buds (Table <a reveal-id="DSS036TB1" data-open="DSS036TB1" class="link link-reveal link-table xref-fig">1</a>). In addition, in woody plants, a minimum distance from the roots to the apical meristems, measured as number of nodes rather than length, is required to get the juvenile-to-adult phase transition, suggesting that juvenility signals coming from the roots are involved in the process. Therefore, we also included in our study a cDNA library using RNA extracted from roots (Table <a reveal-id="DSS036TB1" data-open="DSS036TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>Finally, a fourth cDNA library was generated from a mix of other tissues, to extend the number of representative ESTs from olive and used for Sanger sequencing (Table <a reveal-id="DSS036TB1" data-open="DSS036TB1" class="link link-reveal link-table xref-fig">1</a>). It is noteworthy that whereas the four cDNA libraries generated for Sanger sequencing were normalized to maximize transcriptome coverage, the cDNA collections for 454 pyrosequencing were non-normalized to gain information on gene expression levels.</p>                            <h3 scrollto-destination=5649592 id="5649592" class="section-title" data-legacy-id=s3b>3.2. Sequencing results</h3>
<p>In total, we obtained 1 932 337 reads (1 781 033 after trimming and cleaning), 47 518 (38 183 after trimming and cleaning) of which were generated through Sanger sequencing, with an average length of 847 bp, and 1 884 819 (1 742 850 after trimming and cleaning) generated through 454 pyrosequencing, with an average length of 269 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). It is noteworthy that during the execution of the pyrosequencing project, the GS-FLX Titanium platform became available, which considerably improved the quality of the results. From an average length of 225 bp for the first 660 715 reads corresponding to the four cDNA collections generated from buds, we moved to an average length of 314 bp for the remaining 1 224 102 reads corresponding to the four cDNA collections generated from fruits. The average number of sequences left after the process, as well as their average length for each library and collection, is displayed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. After the cleaning and trimming processes, the remaining 1 781 033 reads (92.1% of the total number of reads) were used for assembly.</p>                            <h3 scrollto-destination=5649594 id="5649594" class="section-title" data-legacy-id=s3c>3.3. Sequencing data assembly</h3>
<p>Figure <a reveal-id="DSS036F1" data-open="DSS036F1" class="link link-reveal link-table xref-fig">1</a> shows the workflow of the process used to generate ESTs from the assembly of individual libraries (OLroot, OLmer, OLrest, and OLmeso) (panel A) and collections (MIP, MIA, MAP, MAA, SGAP, MGP, MTA, and MTP) (panel B). The number of unigenes, defined as the number of contigs (consensus sequence obtained from assembled reads), and the number of singletons (unassembled reads), generated from the assembly process of the cleaned reads obtained for each library and collection, are detailed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. </p>                        <a id="5649596" scrollto-destination="5649596"></a>
<div id="DSS036F1" data-id="DSS036F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03601.jpeg?Expires=2147483647&Signature=VrGCqzK2mLwWs3M2iETZ49ZLaacLQfb54X-9ltxB~mF-xDIGiWqtrxEfQcuH58z7dogAxcq2Xb6yJd9W4y2iSiqxWuxyP65UWNR6hie-0lujr2i0QNQ4R5uOCgq8TIgawaQURs5s4Qm2e9-nCGvwJsH20R-6h0xrWy-WfR4xGq906OF1MHI7yvpDERegWem3UgmETr8lU9K3XSkpvUQ4U0CSEY3VMi3MVPGnQaiN9~-YnC9niMZYFfzk0e96FBodCj3G~PLXkkt2O3UKik1Gy6S2qcFTxHfELSSDvl8KxEYU-IaccPrVbGOg~ZqbkXWYR8aQ2DWn4WtjNkliT0gmFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow followed for the individual assembly of different library/collections and for the assembly of different combinations of library/collections (e.g. assembly of libraries generated from the same tissue). Sections A and B show separate processes for both Sanger and 454 reads assembly. Different assemblies were performed using SeqMan Pro 8.1.1 except when assembling together all the reads generated, which was performed using a two-step assembly process (Newbler 2.6 followed by CAP3) to overcome problems when handling such large amount of data. QC Assembly stands for a quality control step carried out before the actual assembly, where all the reads matching any prokaryotic sequence were removed. QC Blast stands for a quality control step carried out for each assembly consisting of comparing the unigenes generated with themselves using BLAST to guarantee optimal assembly results." path-from-xml="dss03601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649596/dss03601.jpeg" path-from-xml="dss03601.jpeg" target="_blank">View large</a><a section="5649596" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03601.jpeg?Expires=2147483647&Signature=keIF37FBfLN6a7jbfeieFslFjVtzvx73LznohvcQ0YJWmXnrOeKkPStvUkC-VBGlIoDZmEaEygpzmN3cVbj~mny-6kA0orUr4gHcAkPhYT0rgTxXR1lUYeISfpYtehy6L-3ADJVjnciu4gya8JenzoIcvJ-nbDxfRy9sgNynRpLUveotp4CCikAr0zyOSkeTqYN1b73Ntv1nN414YUC3DRmuThWbM4fSrs00R-S4vVMNjSPZEtxG35dYIm~r2OUN-T3OYPrxRgjNwcc9q7BYzLtRf3bGtKqKxoJl7lIMfp57rr0HIfC1iKTk9eL9J3VRb5AaEg7neH2C3B2pfjJVsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649596&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow followed for the individual assembly of different library/collections and for the assembly of different combinations of library/collections (e.g. assembly of libraries generated from the same tissue). Sections A and B show separate processes for both Sanger and 454 reads assembly. Different assemblies were performed using SeqMan Pro 8.1.1 except when assembling together all the reads generated, which was performed using a two-step assembly process (Newbler 2.6 followed by CAP3) to overcome problems when handling such large amount of data. QC Assembly stands for a quality control step carried out before the actual assembly, where all the reads matching any prokaryotic sequence were removed. QC Blast stands for a quality control step carried out for each assembly consisting of comparing the unigenes generated with themselves using BLAST to guarantee optimal assembly results.</p></div></div><div content-id="DSS036F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03601.jpeg?Expires=2147483647&Signature=VrGCqzK2mLwWs3M2iETZ49ZLaacLQfb54X-9ltxB~mF-xDIGiWqtrxEfQcuH58z7dogAxcq2Xb6yJd9W4y2iSiqxWuxyP65UWNR6hie-0lujr2i0QNQ4R5uOCgq8TIgawaQURs5s4Qm2e9-nCGvwJsH20R-6h0xrWy-WfR4xGq906OF1MHI7yvpDERegWem3UgmETr8lU9K3XSkpvUQ4U0CSEY3VMi3MVPGnQaiN9~-YnC9niMZYFfzk0e96FBodCj3G~PLXkkt2O3UKik1Gy6S2qcFTxHfELSSDvl8KxEYU-IaccPrVbGOg~ZqbkXWYR8aQ2DWn4WtjNkliT0gmFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow followed for the individual assembly of different library/collections and for the assembly of different combinations of library/collections (e.g. assembly of libraries generated from the same tissue). Sections A and B show separate processes for both Sanger and 454 reads assembly. Different assemblies were performed using SeqMan Pro 8.1.1 except when assembling together all the reads generated, which was performed using a two-step assembly process (Newbler 2.6 followed by CAP3) to overcome problems when handling such large amount of data. QC Assembly stands for a quality control step carried out before the actual assembly, where all the reads matching any prokaryotic sequence were removed. QC Blast stands for a quality control step carried out for each assembly consisting of comparing the unigenes generated with themselves using BLAST to guarantee optimal assembly results." path-from-xml="dss03601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649596/dss03601.jpeg" path-from-xml="dss03601.jpeg" target="_blank">View large</a><a section="5649596" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03601.jpeg?Expires=2147483647&Signature=keIF37FBfLN6a7jbfeieFslFjVtzvx73LznohvcQ0YJWmXnrOeKkPStvUkC-VBGlIoDZmEaEygpzmN3cVbj~mny-6kA0orUr4gHcAkPhYT0rgTxXR1lUYeISfpYtehy6L-3ADJVjnciu4gya8JenzoIcvJ-nbDxfRy9sgNynRpLUveotp4CCikAr0zyOSkeTqYN1b73Ntv1nN414YUC3DRmuThWbM4fSrs00R-S4vVMNjSPZEtxG35dYIm~r2OUN-T3OYPrxRgjNwcc9q7BYzLtRf3bGtKqKxoJl7lIMfp57rr0HIfC1iKTk9eL9J3VRb5AaEg7neH2C3B2pfjJVsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649596&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow followed for the individual assembly of different library/collections and for the assembly of different combinations of library/collections (e.g. assembly of libraries generated from the same tissue). Sections A and B show separate processes for both Sanger and 454 reads assembly. Different assemblies were performed using SeqMan Pro 8.1.1 except when assembling together all the reads generated, which was performed using a two-step assembly process (Newbler 2.6 followed by CAP3) to overcome problems when handling such large amount of data. QC Assembly stands for a quality control step carried out before the actual assembly, where all the reads matching any prokaryotic sequence were removed. QC Blast stands for a quality control step carried out for each assembly consisting of comparing the unigenes generated with themselves using BLAST to guarantee optimal assembly results.</p></div></div><p>To assess the performance of the 454 assembly, a BLASTN search was carried out using as query the contigs resulting from assembling together the reads generated from the 4 collections sequenced by 454 prior to the GS-FLX Titanium upgrade (MIP, MIA, MAP, and MAA), selected for being those with the shortest average length versus a database with all the unigenes (contigs plus singletons) obtained from the assembly. If the assembly process was optimal, a singleton should not match with any of the contigs, nor a contig must match any other contig. Thus, the matches obtained between the query sequences (contigs) and the total unigenes (contigs + singletons), apart from the hit of each contig with itself, should be as few and as short as possible, demonstrating the uniqueness of each of the contigs and singletons generated through the assembly. Our analysis revealed that only 9% of the contigs (<em>n</em> = 4272) had BLAST hits with 100% identity with other contigs (<em>n</em> = 47.5) or singletons (<em>n</em> = 63.7), but in no case these alignments extended over the entire length of either the BLAST subject or the query, suggesting that they originate from partial similarities, rather than from ineffective assembly. Regions that obtained a perfect match against a different contig or a singleton had an average length of 125 and 114 nt, respectively, that represent 24.3% of the length of the query sequence, in average for matches to contig and 19.7% for matches to singletons. The simplest explanation for these results is that these small matching regions, with 100% identity alignments, are either small sequence motifs highly conserved in different genes or the result of alternative splicing. In either case, the results of our test indicate that the assembly pipeline was capable of distinguishing these short matches from proper matches, assigning the sequences to separate contigs.</p><p>Assembling together the data generated from all the libraries and collections required the use of a more complex strategy, to overcome the limitations in the computer's memory, when handling such a large amount of sequencing data and the potential difficulties associated with sequence variability when assembling different plant varieties in a highly heterozygous species. With the clear aim of reducing the number of sequencing reads from the same gene assembled into different contigs, our chosen strategy was the sequential use of two different assemblers. The assemblers selected are based on two different mathematical models for assembly: ‘Greedy’ and ‘Overlap-Layout-Consensus’ (OLC). The Greedy assembler chosen was Newbler as it is the software provided by the company that performed the 454 sequencing (LifeScience/Roche). The OLC method of our choice was CAP3 that was carried out after the first assembly step, taking as input the output of the Newbler assembly. Thus, we obtained 162 118 unigenes from which 52 781 were contigs and 109 337 singletons. A subsequent filter was performed setting a minimal length of 200 nt. The resulting dataset was, thus, composed of 81 020 unigenes from which 47 520 were contigs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p>                            <h3 scrollto-destination=5649599 id="5649599" class="section-title" data-legacy-id=s3d>3.4. Analysis of the assembled dataset</h3>
<p>The distribution by length of the unigenes generated in the assembly is shown in Fig. <a reveal-id="DSS036F2" data-open="DSS036F2" class="link link-reveal link-table xref-fig">2</a>. Interestingly, only a single peak could be observed for either contigs, or unigenes, indicating that the sequences generated from the two types of technologies have been integrated during the assembly process. The GC content of the olive transcriptome assembled in this study ranged from 15.4 to 76.2% in the different unigenes, with an average of 40.4%, very similar to that of 42.5% reported previously.<sup><a reveal-id="DSS036C13" data-open="DSS036C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p>                        <a id="5649601" scrollto-destination="5649601"></a>
<div id="DSS036F2" data-id="DSS036F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03602.jpeg?Expires=2147483647&Signature=pbK-paktR0NXdKMpsmxaDEZjYfS58Otai0~~eg6mrj6ikhIlAREHBFq3KFku30Hq1ZfcnQbYDSj-Muy9mSclRwKngfqyDHhCHterIpt7dFwyqDMyXGf-Bm4WWQdvmoCmoE-rW5unT7ZwGbDnBq2AqW1UNYsyyZp7dbIRzna9KCgygwsuyFIxAlB4-vJ05BOMtisLvrsWSgONv98qrfRjD1uLnwB6cjCPB9f~O7ZNRnudk8ZDcUDNTYCa3xqiM7Q82e9EFpsZRaKdPEV7YOA1T073vJwRMT88uOkv58xk9-~Sm~XUHtlFSXY6ZyRAiGYoqu35f70RNNeFNIgbY~BMaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of lengths of contigs and unigenes from the fully assembled dataset. Unigenes are represented in green and contigs in red." path-from-xml="dss03602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649601/dss03602.jpeg" path-from-xml="dss03602.jpeg" target="_blank">View large</a><a section="5649601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03602.jpeg?Expires=2147483647&Signature=OjxaMACh~wHdpmVVbMmFSpBHjCBs4AyuTS~27o0WSpz550omaP6uBMsVcVeHygfjctwIK0Dh9~Z7F2jl0mkYL9VeZXG35GMNV5nVxktH1bJmbNWKEwiPsV3TCkngwPDzcITJ1eNdiIcdDS82TvzaRIaGE9GNmlxEwklmIcQCgF1Ia943RrG34xWaf~424J9keVQM5FXTBQpfzxWDdR-HLbIKaNRHaA8rUGg0T4xytYF3kp0YjR5yEMfaWdiH1u~m2jJ599lfHFtU2NlUpBt5X7UHbe6qgLOFCFxZN2sNfN34qXXW4lSBuPbEsOw-vXLhhwmw5xkTcaJjmOzKt7ivdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649601&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of lengths of contigs and unigenes from the fully assembled dataset. Unigenes are represented in green and contigs in red.</p></div></div><div content-id="DSS036F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03602.jpeg?Expires=2147483647&Signature=pbK-paktR0NXdKMpsmxaDEZjYfS58Otai0~~eg6mrj6ikhIlAREHBFq3KFku30Hq1ZfcnQbYDSj-Muy9mSclRwKngfqyDHhCHterIpt7dFwyqDMyXGf-Bm4WWQdvmoCmoE-rW5unT7ZwGbDnBq2AqW1UNYsyyZp7dbIRzna9KCgygwsuyFIxAlB4-vJ05BOMtisLvrsWSgONv98qrfRjD1uLnwB6cjCPB9f~O7ZNRnudk8ZDcUDNTYCa3xqiM7Q82e9EFpsZRaKdPEV7YOA1T073vJwRMT88uOkv58xk9-~Sm~XUHtlFSXY6ZyRAiGYoqu35f70RNNeFNIgbY~BMaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of lengths of contigs and unigenes from the fully assembled dataset. Unigenes are represented in green and contigs in red." path-from-xml="dss03602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649601/dss03602.jpeg" path-from-xml="dss03602.jpeg" target="_blank">View large</a><a section="5649601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03602.jpeg?Expires=2147483647&Signature=OjxaMACh~wHdpmVVbMmFSpBHjCBs4AyuTS~27o0WSpz550omaP6uBMsVcVeHygfjctwIK0Dh9~Z7F2jl0mkYL9VeZXG35GMNV5nVxktH1bJmbNWKEwiPsV3TCkngwPDzcITJ1eNdiIcdDS82TvzaRIaGE9GNmlxEwklmIcQCgF1Ia943RrG34xWaf~424J9keVQM5FXTBQpfzxWDdR-HLbIKaNRHaA8rUGg0T4xytYF3kp0YjR5yEMfaWdiH1u~m2jJ599lfHFtU2NlUpBt5X7UHbe6qgLOFCFxZN2sNfN34qXXW4lSBuPbEsOw-vXLhhwmw5xkTcaJjmOzKt7ivdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649601&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of lengths of contigs and unigenes from the fully assembled dataset. Unigenes are represented in green and contigs in red.</p></div></div><p>Analysis of EST abundance in a contig can provide insight into the gene expression levels, although in our case, this information must be taken with caution because the Sanger-sequenced libraries were normalized. Figure <a reveal-id="DSS036F3" data-open="DSS036F3" class="link link-reveal link-table xref-fig">3</a> shows the distribution of contigs coverage for those originated from the assembly of all the sequences obtained. A wide distribution of coverage among the contigs can be observed in the figure, with the number of ESTs assembled into a given contig ranging between 2 and more than 1000. The largest group was that of the contigs formed by 2–4 reads (47.6% of all contigs) dropping to 19.1% of those containing 5–10 reads, probably owing to the limitations of the assembly process and length of the 454 reads, as well as the presence of normalized libraries among those generated. However, 33.2% of the total contigs were formed by 10–10 000 reads. For these contigs, the number of reads is likely to be a good indication of their level of expression, and this correlation is expected to be stronger as the number of reads forming the contigs increases. </p>                        <a id="5649603" scrollto-destination="5649603"></a>
<div id="DSS036F3" data-id="DSS036F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03603.jpeg?Expires=2147483647&Signature=shgFZZ7GR-gU8SBk-0-lqPrcWqGRXUkEZG68Tg3tevWg5xVyhHwYbf0L~3NFHwV4-EOMLWEUiGB2bFKxVX7iXQPNs2ifalgcauVEYyFlh0mvwGoZ~egnA-x3or5zuMqvxvHDddDSSe5gXf0oHnhEhwVKgfU6AwKvwJ~6~i9VugqKsHoWDdI73sbdJb-rg1vxinAB8GpUZROkyNrJ0Yjo2sZakE8a~C5cpz53~t6nObeJ3XDwfwqDyVaW4IDdWg6ufR4~vpAsJt8kmxlcB~qjPaNd91VEUVcUa6QyNY7c5BryJC7RgfT~BWlvxf8A2VKivIxKEN8S9r7fYYjRmO8i6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the number of reads per contig for the contigs from the assembled dataset." path-from-xml="dss03603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649603/dss03603.jpeg" path-from-xml="dss03603.jpeg" target="_blank">View large</a><a section="5649603" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03603.jpeg?Expires=2147483647&Signature=kyFLU2Q96jgtW7Smni1V0FLw6Vl67fX--nF8vYvjEghNtS0xDULLRbPNTieg~oRaIIURs1YXanO3ufyqgjxMoySw9dxygce9sjMLgEdf64chDbpyTX~hgMZqr3-lMgvubKWJV~PX8cOf4ifmZevsN-fb5WyNy2NGGnF4DiJnq0MGR401CjvUf1ccpdoZGUtfyM6Mkg~aYL7wbtE6L4nwY6fE~U5Vdm6lDOR6jWsqCsNvqzjs9cx5i5zQGNsKVCrIJvd7gWcziID6NigEGXB16-dDm5OAWqTzydeNA9WGtWv6nzcIzSq7JUnQx~Un9CaGV8CdYQ1QvYm9EfncIF5IuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649603&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the number of reads per contig for the contigs from the assembled dataset.</p></div></div><div content-id="DSS036F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03603.jpeg?Expires=2147483647&Signature=shgFZZ7GR-gU8SBk-0-lqPrcWqGRXUkEZG68Tg3tevWg5xVyhHwYbf0L~3NFHwV4-EOMLWEUiGB2bFKxVX7iXQPNs2ifalgcauVEYyFlh0mvwGoZ~egnA-x3or5zuMqvxvHDddDSSe5gXf0oHnhEhwVKgfU6AwKvwJ~6~i9VugqKsHoWDdI73sbdJb-rg1vxinAB8GpUZROkyNrJ0Yjo2sZakE8a~C5cpz53~t6nObeJ3XDwfwqDyVaW4IDdWg6ufR4~vpAsJt8kmxlcB~qjPaNd91VEUVcUa6QyNY7c5BryJC7RgfT~BWlvxf8A2VKivIxKEN8S9r7fYYjRmO8i6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the number of reads per contig for the contigs from the assembled dataset." path-from-xml="dss03603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649603/dss03603.jpeg" path-from-xml="dss03603.jpeg" target="_blank">View large</a><a section="5649603" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03603.jpeg?Expires=2147483647&Signature=kyFLU2Q96jgtW7Smni1V0FLw6Vl67fX--nF8vYvjEghNtS0xDULLRbPNTieg~oRaIIURs1YXanO3ufyqgjxMoySw9dxygce9sjMLgEdf64chDbpyTX~hgMZqr3-lMgvubKWJV~PX8cOf4ifmZevsN-fb5WyNy2NGGnF4DiJnq0MGR401CjvUf1ccpdoZGUtfyM6Mkg~aYL7wbtE6L4nwY6fE~U5Vdm6lDOR6jWsqCsNvqzjs9cx5i5zQGNsKVCrIJvd7gWcziID6NigEGXB16-dDm5OAWqTzydeNA9WGtWv6nzcIzSq7JUnQx~Un9CaGV8CdYQ1QvYm9EfncIF5IuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649603&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the number of reads per contig for the contigs from the assembled dataset.</p></div></div><p>Probably, one of the best assessments of the quality of the assembled database is provided by the number of different transcripts of the transcriptome being tagged by at least one read; in other words, the transcriptome coverage. To estimate the transcriptome coverage, the number of olive transcripts likely to be included in our assembled dataset, we used as a reference the tomato (<em>Solanum lycopersicum</em>) ITAG2 protein database (<a class="link link-uri" href="http://solgenomics.net">http://solgenomics.net</a>), a sequenced<sup><a reveal-id="DSS036C45" data-open="DSS036C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and relatively closely related species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).<sup><a reveal-id="DSS036C46" data-open="DSS036C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS036C47" data-open="DSS036C47" class="link link-ref link-reveal xref-bibr">47</a></sup> We found that 81.2% of the tomato genes were hit by at least one read from our olive dataset. We mathematically estimated transcriptome coverage, following a strategy similar to rarefaction analysis previously used for the same purpose in pea (<em>P. sativum</em>).<sup><a reveal-id="DSS036C34" data-open="DSS036C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Random groups of reads of increasing sizes were tested for identification of tomato genes and fitted to a hyperbolic curve (Fig. <a reveal-id="DSS036F4" data-open="DSS036F4" class="link link-reveal link-table xref-fig">4</a>). Because the slope at the final three points of the curve, corresponding to 60, 80, and 100% of the total reads, was very low (5 × 10<sup>−4</sup>, 3 × 10<sup>−4</sup>, and 2 × 10<sup>−4</sup>, respectively), a further increase in the size of the sample (number of reads obtained) would not result in a significant increase in gene identification. Taking into account the taxonomical distance between the two species, the different representation of tissues in our dataset, and the absence of flowers in this analysis, we are likely close in saturate the olive transcriptomes for mesocarp, meristem, and seed tissues. Thus, even though the final assembly did not collapse and we still have a number of unigenes larger than expected, we are likely to have at least one read from the majority of the transcripts generated in these tissues. </p>                        <a id="5649605" scrollto-destination="5649605"></a>
<div id="DSS036F4" data-id="DSS036F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03604.jpeg?Expires=2147483647&Signature=nKTN8t0isjSQ3zGRxKehxizb383KsWc7X7bDIGUQA6G2S2OIUu6J2WXBPfcoGHH4cj--QmRKkf3FpUWZi3Xaq8z~cnoLzkYkq126b1xt4hvVSZp48CRYBdjx9OjfcxcgCIrBHpZksvd9TfPLla2iuDP9gv2blvT1Cx3nk8vtsGEpqutYJPJCQoBqGzFHJ09vTbR3C-QZ9xqFGrxHUjEkkObJq0cdrAfMScScp0xIgNrHpiX6sC8G6cIqxmB~56bbpg~MUty5XNgwCc8FQ8OW6wsen2uFAieAlghF-xzI~Yl8-6W4roQQ~yZrk~QlmfkNQghPqVxyKcKe6fM1NqwJGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rarefaction analysis of gene representation in the assembled dataset. Different numbers of randomly sampled reads were blasted against the SOLLC protein database, and the number of proteins identified at least 1time, 5, 10 and a 100 times was recorded. The resulting data were modelled with non-linear regression y = (ax)/(b–x)." path-from-xml="dss03604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649605/dss03604.jpeg" path-from-xml="dss03604.jpeg" target="_blank">View large</a><a section="5649605" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03604.jpeg?Expires=2147483647&Signature=X5z5DM8NxWAPqLIt0Ke5XW0qdgibRcpvr5Mjhc50DHoYbBK-7cA9AfIQ2H9HiqmXvc2sFmLffU8MT98Bev0jOM6FGpFihc0rHIizsQ4JuOwLLpzJic3F2li~phIt-nt~XUL0yCUxV1H-OdsX0mRd8Ufln4167hIt8N~K9bCRbXQXw5MbbQpuGIgY~FHecpJ45PYFNbg3sRiF6nkKVhwdzRy1QLbW4LIFe6Fyrr29ORPNSKPgTINf5ewkT2dkhJXA8itkn4hjaa7wk63~s5wzwlNkNoVjlTDBIZU9ymJbsntDz6RVFNaSvkBtLrK4zvftmuzxG2T8VpF0uoxRMnsdbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649605&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rarefaction analysis of gene representation in the assembled dataset. Different numbers of randomly sampled reads were blasted against the SOLLC protein database, and the number of proteins identified at least 1time, 5, 10 and a 100 times was recorded. The resulting data were modelled with non-linear regression <em>y</em> = (<em>ax</em>)/(<em>b</em>–<em>x</em>).</p></div></div><div content-id="DSS036F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03604.jpeg?Expires=2147483647&Signature=nKTN8t0isjSQ3zGRxKehxizb383KsWc7X7bDIGUQA6G2S2OIUu6J2WXBPfcoGHH4cj--QmRKkf3FpUWZi3Xaq8z~cnoLzkYkq126b1xt4hvVSZp48CRYBdjx9OjfcxcgCIrBHpZksvd9TfPLla2iuDP9gv2blvT1Cx3nk8vtsGEpqutYJPJCQoBqGzFHJ09vTbR3C-QZ9xqFGrxHUjEkkObJq0cdrAfMScScp0xIgNrHpiX6sC8G6cIqxmB~56bbpg~MUty5XNgwCc8FQ8OW6wsen2uFAieAlghF-xzI~Yl8-6W4roQQ~yZrk~QlmfkNQghPqVxyKcKe6fM1NqwJGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rarefaction analysis of gene representation in the assembled dataset. Different numbers of randomly sampled reads were blasted against the SOLLC protein database, and the number of proteins identified at least 1time, 5, 10 and a 100 times was recorded. The resulting data were modelled with non-linear regression y = (ax)/(b–x)." path-from-xml="dss03604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649605/dss03604.jpeg" path-from-xml="dss03604.jpeg" target="_blank">View large</a><a section="5649605" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03604.jpeg?Expires=2147483647&Signature=X5z5DM8NxWAPqLIt0Ke5XW0qdgibRcpvr5Mjhc50DHoYbBK-7cA9AfIQ2H9HiqmXvc2sFmLffU8MT98Bev0jOM6FGpFihc0rHIizsQ4JuOwLLpzJic3F2li~phIt-nt~XUL0yCUxV1H-OdsX0mRd8Ufln4167hIt8N~K9bCRbXQXw5MbbQpuGIgY~FHecpJ45PYFNbg3sRiF6nkKVhwdzRy1QLbW4LIFe6Fyrr29ORPNSKPgTINf5ewkT2dkhJXA8itkn4hjaa7wk63~s5wzwlNkNoVjlTDBIZU9ymJbsntDz6RVFNaSvkBtLrK4zvftmuzxG2T8VpF0uoxRMnsdbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649605&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rarefaction analysis of gene representation in the assembled dataset. Different numbers of randomly sampled reads were blasted against the SOLLC protein database, and the number of proteins identified at least 1time, 5, 10 and a 100 times was recorded. The resulting data were modelled with non-linear regression <em>y</em> = (<em>ax</em>)/(<em>b</em>–<em>x</em>).</p></div></div><p>An additional parameter frequently used to assess the goodness of an assembly is the transcript coverage or percentage of the total number of bases of the transcriptome that has been covered. To estimate transcript coverage, we also followed the same strategy previously applied to pea,<sup><a reveal-id="DSS036C34" data-open="DSS036C34" class="link link-ref link-reveal xref-bibr">34</a></sup> which based on the notion of most transcripts requiring more than one read to cover its full length, analysed the number of genes tagged by multiple reads. Reads were again tested in random groups of increasing size, and the number of genes tagged by at least 5, 10, or 100 reads was established (Fig. <a reveal-id="DSS036F4" data-open="DSS036F4" class="link link-reveal link-table xref-fig">4</a>). The lowering slopes of the curves obtained for all three cases (e.g. 6 × 10<sup>−3</sup>, 4 × 10<sup>−3</sup>, and 3 × 10<sup>−3</sup> for genes tagged by more than 100 hits) suggest that although transcript coverage is not complete, it may also be close to saturation.</p><p>We also evaluated the level of redundancy between libraries by analysing the distribution of reads in contigs across the libraries (Fig. <a reveal-id="DSS036F5" data-open="DSS036F5" class="link link-reveal link-table xref-fig">5</a>). We found that 23.1% of the contigs contained reads generated from a single library, 26.9% contained contigs generated from 2, 49.0% of them contained reads that originated from 3 to 9 libraries, whereas only 0.9% contained reads originated from more than 9 to 12 libraries. As expected, even though a large percentage of contigs is formed by a reduced number of reads (Fig. <a reveal-id="DSS036F2" data-open="DSS036F2" class="link link-reveal link-table xref-fig">2</a>), a large percentage of contigs contain reads from many libraries. The drop in the percentage of contigs containing reads originated from 9 to 12 libraries is likely to reflect the difference in size among some of the libraries. We further characterized the contribution of the different libraries to the final assembly by comparing the number of reads from each library that corresponds to singletons, or either belongs to contigs formed by reads shared by different libraries, or to contigs formed by reads from a single library (Fig. <a reveal-id="DSS036F6" data-open="DSS036F6" class="link link-reveal link-table xref-fig">6</a>A). Two libraries, MAP and SGAP, stand out for their seemingly larger specific composition because reads classified into singletons or assembled into library-specific contigs appear overrepresented when compared with other libraries. However, these results could potentially be biased by the differences in the total number of reads existing between libraries. Thus, we also represented the data normalized against the number of reads per library (Fig. <a reveal-id="DSS036F6" data-open="DSS036F6" class="link link-reveal link-table xref-fig">6</a>B). After normalization, the library generated from inactive buds from Arbequina, MIA, displayed an abnormally high level of specificity with the majority of its reads belonging to library-specific contigs. A more detailed look at the composition of this library revealed an overrepresentation of a few contigs (14) formed by an unusually high number of reads per contig (over 1000 to more than 12 000). We believe this to be a technical issue associated with the generation or sequencing of this particular library and, thus, without biological relevance. Therefore, the potential library specificity of MIA contigs should not be considered without additional evaluation. </p>                        <a id="5649608" scrollto-destination="5649608"></a>
<div id="DSS036F5" data-id="DSS036F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03605.jpeg?Expires=2147483647&Signature=WTFP-PoFzk61WZuuoTAKso7jLfMX9T7aSGOHOODxUcK6xWVIh2~sT8z1mJXTYVYHWJUgev196udOpGMTLGx0c03gSqqYgp35z3-a0trsSzLQM77NNZBuddAfreKzgrumpZZaCHgTTVz2kDWYcHgh58z~JJVQL0spWg6sh6IumVuvtjz1U7aiTK6S~rrmxatolYq0yv5asdek3rvnn5D7ch1P2tDMQ-DO4ZBdHIqOQ3E8blAjNF99YAM~7qAjYZzOMHb49GO6q7My8HDKgJvEoBQ4TAuwyBWw5~4oe6oI4W~cDKXqzI5tmHaObmyt2rfKdOHKmPzJD7O025akrn5tfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of number of libraries providing reads per contig. The total number of contigs generated in this work was analysed." path-from-xml="dss03605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649608/dss03605.jpeg" path-from-xml="dss03605.jpeg" target="_blank">View large</a><a section="5649608" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03605.jpeg?Expires=2147483647&Signature=5AVGkg3X~8MVC3NLSvYbLR53k4RLNR~tlGjEDboeAcK5oDSHwpeqKNwPiFFTGK2NzMjSBw4zwxBdkA67n108HXj4ATKIwYTY~~cEa0cjZ6JJeqMKt51ukscUJ8JWxa2jzWo8etoT18gxQeVrSHYj09P3fkM6VvcfnrqJWCStmCM99s3t7DLVs1tKO9E3E9Ci-KGEMjD2Z1Bf7C6HjgGuTSvLjxwt5X89sUIlzxqQrI5w7MTTrh1WOaHM4Zdtodfsi5PA2NhK9RxrhJohdMiOgRto8Kqlg2e5wAJ-PgN-xrAK6VvQciDB~LGybC1iblxNBzk-kCFfX8l6NfU0TmvXGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649608&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of number of libraries providing reads per contig. The total number of contigs generated in this work was analysed.</p></div></div><div content-id="DSS036F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03605.jpeg?Expires=2147483647&Signature=WTFP-PoFzk61WZuuoTAKso7jLfMX9T7aSGOHOODxUcK6xWVIh2~sT8z1mJXTYVYHWJUgev196udOpGMTLGx0c03gSqqYgp35z3-a0trsSzLQM77NNZBuddAfreKzgrumpZZaCHgTTVz2kDWYcHgh58z~JJVQL0spWg6sh6IumVuvtjz1U7aiTK6S~rrmxatolYq0yv5asdek3rvnn5D7ch1P2tDMQ-DO4ZBdHIqOQ3E8blAjNF99YAM~7qAjYZzOMHb49GO6q7My8HDKgJvEoBQ4TAuwyBWw5~4oe6oI4W~cDKXqzI5tmHaObmyt2rfKdOHKmPzJD7O025akrn5tfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of number of libraries providing reads per contig. The total number of contigs generated in this work was analysed." path-from-xml="dss03605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649608/dss03605.jpeg" path-from-xml="dss03605.jpeg" target="_blank">View large</a><a section="5649608" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03605.jpeg?Expires=2147483647&Signature=5AVGkg3X~8MVC3NLSvYbLR53k4RLNR~tlGjEDboeAcK5oDSHwpeqKNwPiFFTGK2NzMjSBw4zwxBdkA67n108HXj4ATKIwYTY~~cEa0cjZ6JJeqMKt51ukscUJ8JWxa2jzWo8etoT18gxQeVrSHYj09P3fkM6VvcfnrqJWCStmCM99s3t7DLVs1tKO9E3E9Ci-KGEMjD2Z1Bf7C6HjgGuTSvLjxwt5X89sUIlzxqQrI5w7MTTrh1WOaHM4Zdtodfsi5PA2NhK9RxrhJohdMiOgRto8Kqlg2e5wAJ-PgN-xrAK6VvQciDB~LGybC1iblxNBzk-kCFfX8l6NfU0TmvXGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649608&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of number of libraries providing reads per contig. The total number of contigs generated in this work was analysed.</p></div></div>                        <a id="5649609" scrollto-destination="5649609"></a>
<div id="DSS036F6" data-id="DSS036F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03606.jpeg?Expires=2147483647&Signature=JtAgnteQKGecwRpN5eoopE9tkXmzrWmSRhFLQonb6XoONVhMpvRTtrL4JIY0dtV5-BPUSPZxz-FPKCTwbqjac1gkItvl5vXZh5ArfR5xlRRtmIOo~Y-deBYqKkEd68VrSPu9x27P9TzlSyW7BqX97LMWuRlW8u-bHttXgbsuobdqD6wyUQul6J45mCDmQbeYrCoZM637IrkbjnvTXYf1wxgmvovpK5nFUjLWeZSqgWhgcYoW3x62LuXZS4ATldCF7lP1xijH8PG0hMWBSXtHySWHd5xdxEeIDNCcntKESGv4Y~sJkgvWNL8-EPhsbXZ8~YzbsB4WvkmbocGO1OobbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of number of reads contributing to the singleton category, or contributing to either library-specific or shared contigs, per library. (A) Numbers indicated correspondence to the actual number of reads of the libraries. (B) Numbers of reads per category have been normalized to the total number of reads per library. The number (A), or percentage (B), of contigs shared by more than one library is represented in green, contigs formed by reads from a single library are shown in blue, and singletons in red." path-from-xml="dss03606.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649609/dss03606.jpeg" path-from-xml="dss03606.jpeg" target="_blank">View large</a><a section="5649609" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03606.jpeg?Expires=2147483647&Signature=CFvCtDcqg71-VemaItzMP3Zt-aFLFPG4RvzQ-crNwqxP6EpUJpuELe~vIKEAJjrGOZW86iXC5vsB-lynE8YJbkaXm4ifHLPVxaN4g6c~EoHqWkXSRmqVHbgB9ijuzFVlzgKYMEZ4B84B9ieyX5hZEQXak0BDrJ55SJGn-dGZV9MqBG00aMBB41prunQGuhr-FSHD38QyfNlEF-ViKjnDDMYeRawqlfLnyF3BK32uHPNg5jX7qPAzJP19Trot9DQfS71yjkOlGWQu0PKDMY10FTGjPXNB6rIlQgiTo91oE8C87AUWRtdXJX~UHsnWxWSWic8IL-PXQFecl3X5VlL6CQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649609&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03606.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of number of reads contributing to the singleton category, or contributing to either library-specific or shared contigs, per library. (A) Numbers indicated correspondence to the actual number of reads of the libraries. (B) Numbers of reads per category have been normalized to the total number of reads per library. The number (A), or percentage (B), of contigs shared by more than one library is represented in green, contigs formed by reads from a single library are shown in blue, and singletons in red.</p></div></div><div content-id="DSS036F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03606.jpeg?Expires=2147483647&Signature=JtAgnteQKGecwRpN5eoopE9tkXmzrWmSRhFLQonb6XoONVhMpvRTtrL4JIY0dtV5-BPUSPZxz-FPKCTwbqjac1gkItvl5vXZh5ArfR5xlRRtmIOo~Y-deBYqKkEd68VrSPu9x27P9TzlSyW7BqX97LMWuRlW8u-bHttXgbsuobdqD6wyUQul6J45mCDmQbeYrCoZM637IrkbjnvTXYf1wxgmvovpK5nFUjLWeZSqgWhgcYoW3x62LuXZS4ATldCF7lP1xijH8PG0hMWBSXtHySWHd5xdxEeIDNCcntKESGv4Y~sJkgvWNL8-EPhsbXZ8~YzbsB4WvkmbocGO1OobbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of number of reads contributing to the singleton category, or contributing to either library-specific or shared contigs, per library. (A) Numbers indicated correspondence to the actual number of reads of the libraries. (B) Numbers of reads per category have been normalized to the total number of reads per library. The number (A), or percentage (B), of contigs shared by more than one library is represented in green, contigs formed by reads from a single library are shown in blue, and singletons in red." path-from-xml="dss03606.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649609/dss03606.jpeg" path-from-xml="dss03606.jpeg" target="_blank">View large</a><a section="5649609" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03606.jpeg?Expires=2147483647&Signature=CFvCtDcqg71-VemaItzMP3Zt-aFLFPG4RvzQ-crNwqxP6EpUJpuELe~vIKEAJjrGOZW86iXC5vsB-lynE8YJbkaXm4ifHLPVxaN4g6c~EoHqWkXSRmqVHbgB9ijuzFVlzgKYMEZ4B84B9ieyX5hZEQXak0BDrJ55SJGn-dGZV9MqBG00aMBB41prunQGuhr-FSHD38QyfNlEF-ViKjnDDMYeRawqlfLnyF3BK32uHPNg5jX7qPAzJP19Trot9DQfS71yjkOlGWQu0PKDMY10FTGjPXNB6rIlQgiTo91oE8C87AUWRtdXJX~UHsnWxWSWic8IL-PXQFecl3X5VlL6CQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649609&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03606.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of number of reads contributing to the singleton category, or contributing to either library-specific or shared contigs, per library. (A) Numbers indicated correspondence to the actual number of reads of the libraries. (B) Numbers of reads per category have been normalized to the total number of reads per library. The number (A), or percentage (B), of contigs shared by more than one library is represented in green, contigs formed by reads from a single library are shown in blue, and singletons in red.</p></div></div><p>Because a large percentage of the contigs are formed by a reduced number of reads, potentially library-specific genes have to be considered with caution. However, when the number of reads per contig is considered in relation to the tissue specificity of each library, many of the contigs obtained show potential specificity. In regards to this, because some of the libraries were constructed using the same type of tissue, we also evaluated the level of redundancy between tissues. The assembled dataset was examined to identify unigenes formed by reads obtained from sequencing libraries generated from the same tissue. We identified a total of 26 430 unigenes that contained reads only sequenced in libraries generated from meristems, 11 352 unigenes formed by reads obtained from fruit mesocarp, 11 182 unigenes containing reads obtained from seeds, and 757 unigenes containing reads obtained from roots (Fig. <a reveal-id="DSS036F7" data-open="DSS036F7" class="link link-reveal link-table xref-fig">7</a>). The reduced number of reads generated from roots, as compared to the other tissues, is likely to limit the identification of potentially specific ESTs. Because OLrest contains reads from seeds, and from stems and leaves, the results obtained from this library were not considered when establishing potential tissue specificity. Potentially tissue-specific ESTs could be of great interest, particularly considering the large number of reads generated from some of the tissues and the results from the rarefaction analysis indicating that we are close in saturating the corresponding transcriptomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> and Fig. <a reveal-id="DSS036F4" data-open="DSS036F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="5649611" scrollto-destination="5649611"></a>
<div id="DSS036F7" data-id="DSS036F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03607.jpeg?Expires=2147483647&Signature=kSaD6A-Q-IFo8tAucWdRg8Sqjp-yxah1mlQr49jwru7b2-yCuYbbmr-bTGfdB11lGv74COX-pN1oETc3WfD2~WHwwYzt~K7vfzWrP3WO3YZRNOWnC3tTcZ6mB6GVyYnbHI14aZecMg0RBPR3oCpKJJDL4k56XXtZ2owua~cn9VG0hlXbzjkQbS9itgncoBVWzoWWPpMXSc-fIpiepmECKlI7oJx9PcooEX2YNy3urVP8N6d32EQvUSH8-y-trR9tnnXxshgjNi3ANjT671MnYt5E5VGN11QiyWIjGLhTQ-6~nFuL2HSAwstuzdn6jgEik7uJqjDPWWcvkiaEVYc8Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing common and specific genes for mesocarp, meristem (buds), seeds, and roots." path-from-xml="dss03607.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649611/dss03607.jpeg" path-from-xml="dss03607.jpeg" target="_blank">View large</a><a section="5649611" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03607.jpeg?Expires=2147483647&Signature=yth2aBPXAvsff0XTvrG1auXIm7ApSdl7T6szKlJwMldc4arD4fdso4WbkLa42v3RJBMrfRPFpFK6gDbqN9LtIZ1-oMxg~MDrie155ItHzPc0rmthtGE-0-19e-N~oItNeZN89gGUqit09b9DzWwUryNyu3mBrGr~VQD1D4oeYQtkntw0bi1EanCD05uVIxDs4A5LqRYCeyo9XbtrG6P6-QK6V6ziGemIYItp5t5oA2Q9D0QkZ0kc5RI-RzzTrV0a0q1kP-~AjHep4uByBTj-fb0vGGg1rFaYvjZKzoSGhZRCKWwcNHUPqAdTaFDlOVWRhQ4nu~RzRrcGpEfx7rdZQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649611&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03607.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing common and specific genes for mesocarp, meristem (buds), seeds, and roots.</p></div></div><div content-id="DSS036F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03607.jpeg?Expires=2147483647&Signature=kSaD6A-Q-IFo8tAucWdRg8Sqjp-yxah1mlQr49jwru7b2-yCuYbbmr-bTGfdB11lGv74COX-pN1oETc3WfD2~WHwwYzt~K7vfzWrP3WO3YZRNOWnC3tTcZ6mB6GVyYnbHI14aZecMg0RBPR3oCpKJJDL4k56XXtZ2owua~cn9VG0hlXbzjkQbS9itgncoBVWzoWWPpMXSc-fIpiepmECKlI7oJx9PcooEX2YNy3urVP8N6d32EQvUSH8-y-trR9tnnXxshgjNi3ANjT671MnYt5E5VGN11QiyWIjGLhTQ-6~nFuL2HSAwstuzdn6jgEik7uJqjDPWWcvkiaEVYc8Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing common and specific genes for mesocarp, meristem (buds), seeds, and roots." path-from-xml="dss03607.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649611/dss03607.jpeg" path-from-xml="dss03607.jpeg" target="_blank">View large</a><a section="5649611" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03607.jpeg?Expires=2147483647&Signature=yth2aBPXAvsff0XTvrG1auXIm7ApSdl7T6szKlJwMldc4arD4fdso4WbkLa42v3RJBMrfRPFpFK6gDbqN9LtIZ1-oMxg~MDrie155ItHzPc0rmthtGE-0-19e-N~oItNeZN89gGUqit09b9DzWwUryNyu3mBrGr~VQD1D4oeYQtkntw0bi1EanCD05uVIxDs4A5LqRYCeyo9XbtrG6P6-QK6V6ziGemIYItp5t5oA2Q9D0QkZ0kc5RI-RzzTrV0a0q1kP-~AjHep4uByBTj-fb0vGGg1rFaYvjZKzoSGhZRCKWwcNHUPqAdTaFDlOVWRhQ4nu~RzRrcGpEfx7rdZQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649611&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03607.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing common and specific genes for mesocarp, meristem (buds), seeds, and roots.</p></div></div>                            <h3 scrollto-destination=5649612 id="5649612" class="section-title" data-legacy-id=s3e>3.5. Functional annotation</h3>
<p>The assembled dataset still includes a considerable number of unassembled reads, a problem common to 454 pyrosequencing projects that may be aggravated by a relative abundance of short repeats within the species,<sup><a reveal-id="DSS036C13" data-open="DSS036C13" class="link link-ref link-reveal xref-bibr">13</a></sup> the occurrence of alternative splicing, or heterozygosity.<sup><a reveal-id="DSS036C48" data-open="DSS036C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Theoretically, sequencing a single cultivar could reduce the limitations that heterozygosity may determine in the efficiency of the assembly. Nonetheless, we obtained similar efficiencies, when using our assembly pipeline in libraries generated from a single versus a mix of varieties (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> and data not shown). Despite these potential limitations, the functional annotation of our assembled dataset retrieved annotations from various databases for over 61.4% of the contigs (29 194 out of the total 47 520) and 27.8% of the singletons (9311 out of the total 33 500). Similar or much lower percentages of gene annotation have been reporte previously for olive ESTs sequencing projects.<sup><a reveal-id="DSS036C13" data-open="DSS036C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS036C15" data-open="DSS036C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In addition, when we used BLAST to further evaluate the full set of unigenes against the plant sequences deposited in Uniprot, we found that 68.6% of the contigs (32 592 out of the total 47 520) and 35.4% of the singletons (11 863 out of the total 33 500) had a hit with a <em>P</em>-value greater than 10<sup>−6</sup>. Although the algorithm used in BLAST comparisons is based on local alignment, the relevance of this results is supported by the fact that more than 50% of the hits obtained for the contigs (16 489 of the 32 592) and over 83% of the hits obtained for the singletons (7770 out of the total 9311) extended over more than 75% of their lengths. In total, 47.5% of the unigenes have a significant hit with plant databases, having found for 63% (27 879) of these, a potential homologue. These results further support the notion of our assembly dataset being a fair representation of the olive transcriptome.</p><p>The distribution of hits obtained against entries for other plants within the NCBI database was used to get a descriptive view of the newly generated dataset. The highest number of hits was obtained against <em>V. vinifera</em> (VITVI), <em>Ricinus communis</em> (RICCO), and <em>P. tricocharpa</em> (POPTR) (Fig. <a reveal-id="DSS036F8" data-open="DSS036F8" class="link link-reveal link-table xref-fig">8</a>A). However, <em>O. europaea</em> (OLEEU) is only the ninth species displaying the highest numbers of hits, just two positions ahead of <em>Arabidopsis thaliana</em> (ARATH), probably owing to the smaller number of olive sequences present in the NCBI database when compared with the other three species. In keeping with this, species phylogenetically closer to olive such as tomato (<em>S. lycopersicum</em> SOLLC) or potato (<em>Solanum tuberosum</em> SOLTU) did display lower number of hits. We include a phylogenetic tree enclosing the species relevant for our study as <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">supplementary material</a> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="5649615" scrollto-destination="5649615"></a>
<div id="DSS036F8" data-id="DSS036F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03608.jpeg?Expires=2147483647&Signature=jaW-qOS48shiOF~B-VTk8yy3NvMlgiuHYNy-acnfZHabSu-MKGPJac6lkEk39RvHBxpgHTd0frQpaJIJ0hBlEbGwjhlTZ-mp5R8Jx7bBaCLYuMB8LQbiSzJ-Mf8NhNFmBZMK~ffNGfezW1YO-Sr3hfTI-4G7UEKe3jBZtfbeXJIr7PD97bS51JLEv7ZgcRFLJALZgjS7v8cnKqRh7QQ5eSdI88EcX8YpZGa1SvT5z8SrDCCX5jAgLgJSkFz0Rklcb7pcUYXMAFTD~7sxMhaBDBG~cDv2FajhE4SuoZXwFsp76Ts-KLFi7FvgGg9dzjPed9ZoroCklhVwouTLXdsNtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Distribution of hits of the complete assembly obtained against sequences available in Uniprot for different plant species. (B) Distribution of hits of the complete assembly obtained against sequences available for other plant species, normalized according to the total number of entries for each species. The database code correspondence to plant species is as follows: VITVI, V. vinifera (Grape); RICCO, R. communis (Castor bean); POPTR, P. trichocarpa (Western balsam poplar) (Populus balsamifera subsp. trichocarpa); TOBAC, Nicotiana tabacum (Common tobacco); ARATH, A. thaliana (Mouse-ear cress); OLEEU, O. europaea (Common olive); SOLLC, S. lycopersicum (Tomato) (Lycopersicon esculentum); ORYSJ, Oryza sativa subsp. japonica (Rice); SOLTU, S. tuberosum (Potato); MEDTR, Medicago truncatula (Barrel medic) (Medicago tribuloides); SOYBN, Glycine max (Soybean) (Glycine hispida); CAPAN, Capsicum annuum (Bell pepper); CATRO, Catharanthus roseus (Madagascar periwinkle) (Vinca rosea); GOSHI, Gossypium hirsutum (Upland cotton) (Gossypium mexicanum); NICBE, Nicotiana benthamiana; PETHY, Petunia hybrida (Petunia); BETVU, Beta vulgaris (Sugar beet); PEA, P. sativum (Garden pea), IPOBA, Ipomoea batatas (Sweet potato) (Convolvulus batatas); PICSI, Picea sitchensis (Sitka spruce) (Pinus sitchensis); SESIN, Sesamum indicum (Oriental sesame) (Sesamum orientale); SOLDE, Solanum demissum (Wild potato); PLAMJ, Plantago major (Common plantain); WHEAT, Triticum aestivum (Wheat); NICLS, Nicotiana langsdorffii × Nicotiana sanderae (Ornamental tobacco); NICPL, Nicotiana plumbaginifolia (Leadwort-leaved tobacco or Tex-Mex tobacco); SALMI, Salvias miltiorrhiza (Chinese sage); PERFR, Perilla frutescens (Beefsteak mint) (Perilla ocymoides); MALDO, Malus domestica (Apple) (Pyrus malus); SORBI, Sorghum bicolor (Sorghum); CUCME, Cucumis melo (Muskmelon); LOTJA, Lotus japonicus; SILLA, Silene latifolia (White campion); HELAN, Helianthus annuus (Common sunflower); and ELAGV, Elaeis guineensis var. tenera (Oil palm)." path-from-xml="dss03608.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649615/dss03608.jpeg" path-from-xml="dss03608.jpeg" target="_blank">View large</a><a section="5649615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03608.jpeg?Expires=2147483647&Signature=zx7nviwCIm2v~PYjYLqUXYCknL6TQDq5MJuFWWqXygHEvC2vHFEz0y07MOKXioPk6ykropQnugNF~d814i2By5fbeECsmgmfGFaxDCJa9zqlgAhbgKBC6m5zhPphSX2QtDKRFy1KRxA~zJIK-FtPTQp6Ft~0u97fuyeMNYD7QpW4hrqQx9Z16lG-ezuAsJxvmD7ohaMG6nQc-hdjO4J32Vc~-Z85bLtLCu8ZawfPp08EuCdroLTnA1tLj1XxHuSHyec-w8l0kqNfmBuevnLKpBsIryGh-w5MGeVWqRJVFjcWuFsbfq~j8-n6fLrceCH0PrLS2vN7Kwb1e2bc~TMGHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649615&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03608.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Distribution of hits of the complete assembly obtained against sequences available in Uniprot for different plant species. (B) Distribution of hits of the complete assembly obtained against sequences available for other plant species, normalized according to the total number of entries for each species. The database code correspondence to plant species is as follows: VITVI, <em>V. vinifera</em> (Grape); RICCO, <em>R. communis</em> (Castor bean); POPTR, <em>P. trichocarpa</em> (Western balsam poplar) (<em>Populus balsamifera</em> subsp. <em>trichocarpa</em>); TOBAC, <em>Nicotiana tabacum</em> (Common tobacco); ARATH, <em>A. thaliana</em> (Mouse-ear cress); OLEEU, <em>O. europaea</em> (Common olive); SOLLC, <em>S. lycopersicum</em> (Tomato) (<em>Lycopersicon esculentum</em>); ORYSJ, <em>Oryza sativa</em> subsp. <em>japonica</em> (Rice); SOLTU, <em>S. tuberosum</em> (Potato); MEDTR, <em>Medicago truncatula</em> (Barrel medic) (<em>Medicago tribuloides</em>); SOYBN, <em>Glycine max</em> (Soybean) (<em>Glycine hispida</em>); CAPAN, <em>Capsicum annuum</em> (Bell pepper); CATRO, <em>Catharanthus roseus</em> (Madagascar periwinkle) (<em>Vinca rosea</em>); GOSHI, <em>Gossypium hirsutum</em> (Upland cotton) (<em>Gossypium mexicanum</em>); NICBE, <em>Nicotiana benthamiana</em>; PETHY, <em>Petunia hybrida</em> (Petunia); BETVU, <em>Beta vulgaris</em> (Sugar beet); PEA, <em>P. sativum</em> (Garden pea), IPOBA, <em>Ipomoea batatas</em> (Sweet potato) (<em>Convolvulus batatas</em>); PICSI, <em>Picea sitchensis</em> (Sitka spruce) (<em>Pinus sitchensis</em>); SESIN, <em>Sesamum indicum</em> (Oriental sesame) (<em>Sesamum orientale</em>); SOLDE, <em>Solanum demissum</em> (Wild potato); PLAMJ, <em>Plantago major</em> (Common plantain); WHEAT, <em>Triticum aestivum</em> (Wheat); NICLS, <em>Nicotiana langsdorffii × Nicotiana sanderae</em> (Ornamental tobacco); NICPL, <em>Nicotiana plumbaginifolia</em> (Leadwort-leaved tobacco or Tex-Mex tobacco); SALMI, <em>Salvias miltiorrhiza</em> (Chinese sage); PERFR, <em>Perilla frutescens</em> (Beefsteak mint) (<em>Perilla ocymoides</em>); MALDO, <em>Malus domestica</em> (Apple) (<em>Pyrus malus</em>); SORBI, <em>Sorghum bicolor</em> (Sorghum); CUCME, <em>Cucumis melo</em> (Muskmelon); LOTJA, <em>Lotus japonicus</em>; SILLA, <em>Silene latifolia</em> (White campion); HELAN, <em>Helianthus annuus</em> (Common sunflower); and ELAGV, <em>Elaeis guineensis var. tenera</em> (Oil palm).</p></div></div><div content-id="DSS036F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03608.jpeg?Expires=2147483647&Signature=jaW-qOS48shiOF~B-VTk8yy3NvMlgiuHYNy-acnfZHabSu-MKGPJac6lkEk39RvHBxpgHTd0frQpaJIJ0hBlEbGwjhlTZ-mp5R8Jx7bBaCLYuMB8LQbiSzJ-Mf8NhNFmBZMK~ffNGfezW1YO-Sr3hfTI-4G7UEKe3jBZtfbeXJIr7PD97bS51JLEv7ZgcRFLJALZgjS7v8cnKqRh7QQ5eSdI88EcX8YpZGa1SvT5z8SrDCCX5jAgLgJSkFz0Rklcb7pcUYXMAFTD~7sxMhaBDBG~cDv2FajhE4SuoZXwFsp76Ts-KLFi7FvgGg9dzjPed9ZoroCklhVwouTLXdsNtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Distribution of hits of the complete assembly obtained against sequences available in Uniprot for different plant species. (B) Distribution of hits of the complete assembly obtained against sequences available for other plant species, normalized according to the total number of entries for each species. The database code correspondence to plant species is as follows: VITVI, V. vinifera (Grape); RICCO, R. communis (Castor bean); POPTR, P. trichocarpa (Western balsam poplar) (Populus balsamifera subsp. trichocarpa); TOBAC, Nicotiana tabacum (Common tobacco); ARATH, A. thaliana (Mouse-ear cress); OLEEU, O. europaea (Common olive); SOLLC, S. lycopersicum (Tomato) (Lycopersicon esculentum); ORYSJ, Oryza sativa subsp. japonica (Rice); SOLTU, S. tuberosum (Potato); MEDTR, Medicago truncatula (Barrel medic) (Medicago tribuloides); SOYBN, Glycine max (Soybean) (Glycine hispida); CAPAN, Capsicum annuum (Bell pepper); CATRO, Catharanthus roseus (Madagascar periwinkle) (Vinca rosea); GOSHI, Gossypium hirsutum (Upland cotton) (Gossypium mexicanum); NICBE, Nicotiana benthamiana; PETHY, Petunia hybrida (Petunia); BETVU, Beta vulgaris (Sugar beet); PEA, P. sativum (Garden pea), IPOBA, Ipomoea batatas (Sweet potato) (Convolvulus batatas); PICSI, Picea sitchensis (Sitka spruce) (Pinus sitchensis); SESIN, Sesamum indicum (Oriental sesame) (Sesamum orientale); SOLDE, Solanum demissum (Wild potato); PLAMJ, Plantago major (Common plantain); WHEAT, Triticum aestivum (Wheat); NICLS, Nicotiana langsdorffii × Nicotiana sanderae (Ornamental tobacco); NICPL, Nicotiana plumbaginifolia (Leadwort-leaved tobacco or Tex-Mex tobacco); SALMI, Salvias miltiorrhiza (Chinese sage); PERFR, Perilla frutescens (Beefsteak mint) (Perilla ocymoides); MALDO, Malus domestica (Apple) (Pyrus malus); SORBI, Sorghum bicolor (Sorghum); CUCME, Cucumis melo (Muskmelon); LOTJA, Lotus japonicus; SILLA, Silene latifolia (White campion); HELAN, Helianthus annuus (Common sunflower); and ELAGV, Elaeis guineensis var. tenera (Oil palm)." path-from-xml="dss03608.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649615/dss03608.jpeg" path-from-xml="dss03608.jpeg" target="_blank">View large</a><a section="5649615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03608.jpeg?Expires=2147483647&Signature=zx7nviwCIm2v~PYjYLqUXYCknL6TQDq5MJuFWWqXygHEvC2vHFEz0y07MOKXioPk6ykropQnugNF~d814i2By5fbeECsmgmfGFaxDCJa9zqlgAhbgKBC6m5zhPphSX2QtDKRFy1KRxA~zJIK-FtPTQp6Ft~0u97fuyeMNYD7QpW4hrqQx9Z16lG-ezuAsJxvmD7ohaMG6nQc-hdjO4J32Vc~-Z85bLtLCu8ZawfPp08EuCdroLTnA1tLj1XxHuSHyec-w8l0kqNfmBuevnLKpBsIryGh-w5MGeVWqRJVFjcWuFsbfq~j8-n6fLrceCH0PrLS2vN7Kwb1e2bc~TMGHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649615&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03608.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Distribution of hits of the complete assembly obtained against sequences available in Uniprot for different plant species. (B) Distribution of hits of the complete assembly obtained against sequences available for other plant species, normalized according to the total number of entries for each species. The database code correspondence to plant species is as follows: VITVI, <em>V. vinifera</em> (Grape); RICCO, <em>R. communis</em> (Castor bean); POPTR, <em>P. trichocarpa</em> (Western balsam poplar) (<em>Populus balsamifera</em> subsp. <em>trichocarpa</em>); TOBAC, <em>Nicotiana tabacum</em> (Common tobacco); ARATH, <em>A. thaliana</em> (Mouse-ear cress); OLEEU, <em>O. europaea</em> (Common olive); SOLLC, <em>S. lycopersicum</em> (Tomato) (<em>Lycopersicon esculentum</em>); ORYSJ, <em>Oryza sativa</em> subsp. <em>japonica</em> (Rice); SOLTU, <em>S. tuberosum</em> (Potato); MEDTR, <em>Medicago truncatula</em> (Barrel medic) (<em>Medicago tribuloides</em>); SOYBN, <em>Glycine max</em> (Soybean) (<em>Glycine hispida</em>); CAPAN, <em>Capsicum annuum</em> (Bell pepper); CATRO, <em>Catharanthus roseus</em> (Madagascar periwinkle) (<em>Vinca rosea</em>); GOSHI, <em>Gossypium hirsutum</em> (Upland cotton) (<em>Gossypium mexicanum</em>); NICBE, <em>Nicotiana benthamiana</em>; PETHY, <em>Petunia hybrida</em> (Petunia); BETVU, <em>Beta vulgaris</em> (Sugar beet); PEA, <em>P. sativum</em> (Garden pea), IPOBA, <em>Ipomoea batatas</em> (Sweet potato) (<em>Convolvulus batatas</em>); PICSI, <em>Picea sitchensis</em> (Sitka spruce) (<em>Pinus sitchensis</em>); SESIN, <em>Sesamum indicum</em> (Oriental sesame) (<em>Sesamum orientale</em>); SOLDE, <em>Solanum demissum</em> (Wild potato); PLAMJ, <em>Plantago major</em> (Common plantain); WHEAT, <em>Triticum aestivum</em> (Wheat); NICLS, <em>Nicotiana langsdorffii × Nicotiana sanderae</em> (Ornamental tobacco); NICPL, <em>Nicotiana plumbaginifolia</em> (Leadwort-leaved tobacco or Tex-Mex tobacco); SALMI, <em>Salvias miltiorrhiza</em> (Chinese sage); PERFR, <em>Perilla frutescens</em> (Beefsteak mint) (<em>Perilla ocymoides</em>); MALDO, <em>Malus domestica</em> (Apple) (<em>Pyrus malus</em>); SORBI, <em>Sorghum bicolor</em> (Sorghum); CUCME, <em>Cucumis melo</em> (Muskmelon); LOTJA, <em>Lotus japonicus</em>; SILLA, <em>Silene latifolia</em> (White campion); HELAN, <em>Helianthus annuus</em> (Common sunflower); and ELAGV, <em>Elaeis guineensis var. tenera</em> (Oil palm).</p></div></div><p>To make up for the differences in the number of protein entries per each plant species, we also plotted the distribution of the number of hits per species normalized for the number of entries from each species within the NCBI database. We established 500 Uniprot entries per species to eliminate distortions frequently observed with very underrepresented species (data not shown). After this process, although many of the species present in the untreated graph (Fig. <a reveal-id="DSS036F8" data-open="DSS036F8" class="link link-reveal link-table xref-fig">8</a>A) were still present, the relative order in which they appeared was considerably altered, with the highest number of hits being obtained against olive (OLEEU) (Fig. <a reveal-id="DSS036F8" data-open="DSS036F8" class="link link-reveal link-table xref-fig">8</a>B). In the normalized graph, the six closest species to our dataset, including olive (OLEEU), correspond to members of the asterids clade (Fig. <a reveal-id="DSS036F8" data-open="DSS036F8" class="link link-reveal link-table xref-fig">8</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In addition, VITVI (<em>V. vinifera</em>) dropped from the first position in the non-normalized graph to eighth and <em>Arabidopsis</em> (ARATH) dropped from 11th to position 23rd, being these new positions more in agreement with the taxonomic relationships between the species (Fig. <a reveal-id="DSS036F8" data-open="DSS036F8" class="link link-reveal link-table xref-fig">8</a>B; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p><p>The distribution of hits against different plant species is frequently used without taking into account the large differences between the total numbers of entries for each species included in the NCBI database. We show here that the results obtained without normalization are clearly influenced by the number of entries and far away from the results expected according to the taxonomical relationships between the species.</p><p>Because one of our main aims was to generate ESTs with information about the metabolism involved in oil production, we also retrieved enzyme IDs from the functional annotation obtained for our dataset. We found the following distribution into each of the six major enzyme classes (<a class="link link-uri" href="http://enzyme.expasy.org/cgi-bin/enzyme/">http://enzyme.expasy.org/cgi-bin/enzyme/</a>): transferases 27% (in 87 unigenes), hydrolases 24% (in 76 unigenes), oxidoreductases 30% (in 95 unigenes), ligases 5% (in 15 unigenes), lyases 7% (in 23 unigenes), and isomerases 7% (in 24 unigenes) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p>                            <h3 scrollto-destination=5649619 id="5649619" class="section-title" data-legacy-id=s3f>3.6. GO annotation</h3>
<p>Blast2GO suite<sup><a reveal-id="DSS036C38" data-open="DSS036C38" class="link link-ref link-reveal xref-bibr">38</a></sup> is often used for analysing functional annotation for purposes of biologically profiling a newly generated sequence dataset beyond the single-gene level. However, because the suite has limitations when handling large datasets, we could only use Blast2GO to retrieve the GO terms previously assigned to our unigene dataset by Sma3s. To facilitate data processing, we established a cutoff for GO terms that were present in only one sequence within the DAG<sup><a reveal-id="DSS036C38" data-open="DSS036C38" class="link link-ref link-reveal xref-bibr">38</a></sup> generated (data not shown). A total of 87 309 GO terms were obtained from a total of 27 494 unigenes for which at least 3 GO were obtained. The distribution of assignments into the GO categories was 14 930 unigenes with at least 3 GO terms from BP, 24 568 unigenes with at least 3 GO terms from molecular function (MF), and 11 481 unigenes with at least 3 GO terms from cellular components (CC). The most abundant high-level GO counts within these categories were BP, the very general categories of metabolic processes associated with 10 471 unigenes, and cellular processes, associated with 11 872 unigenes, and more specific, biological regulation, localization, and response to stimulus with 2832, 2168, and 2246 unigenes, respectively; MF, binding, associated with 16 902 unigenes, and catalytic activity associated with 14 657 unigenes; and CC, the very general cell, associated with 11 182 and more specific organelle, with 5220, membrane with 4789, and macromolecular complex with 2557 (Fig. <a reveal-id="DSS036F9" data-open="DSS036F9" class="link link-reveal link-table xref-fig">9</a>). </p>                        <a id="5649621" scrollto-destination="5649621"></a>
<div id="DSS036F9" data-id="DSS036F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03609.jpeg?Expires=2147483647&Signature=wEfV7ePVcMz06pewqV4NXGGt2~pJQ~8pSd4syJpHcwtz5JGmG7I4pGABNoIx6zs0ti4mPYIVvWnAA2kt2M1k1jq4qUqWW7VFKC3lb4X5Bqp~6ntZbxiVsR4ARSxdm6BNFlHTgOiiSHsUakDenwPcrkAyahW8T3SPqzf54N8Yv71c7uy5~OXwdiL7UiLDgN-Q1LdJI2T6sER858UAkasNk0535bNZmiiyj~~7c5NwHod7JUX42AVHjWb1f58i9cwErldqEEHZvlIUI8QWQBteIgikMiB5vGGzI8JQ63JFlZ4xr6ituJzTW8ewbtFj5rdARlcIjVRzKLN-wtI50v2qUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Level 2 GO term assignment to the olive fully assembled dataset and distribution in categories of BP, MF, and CC." path-from-xml="dss03609.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649621/dss03609.jpeg" path-from-xml="dss03609.jpeg" target="_blank">View large</a><a section="5649621" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03609.jpeg?Expires=2147483647&Signature=b9bLXrr8L9B2P4TlfL4nzOMUXVBPwx4sqLnqCRBgxwjL-P8o9sYMi1zxPo7cFUDQc~9F-Tj2YBAi6Jq5EINqI3xIZ4mJ4XD-8FevH~Nix-Q-nNWcfD05oocR2AK8M4yfdd1ryocqCTGhi40gyhwdHOSaajM3~q3NNGV~W~T-tElWEjVtcWSMsIrrX901V9Zn8bYB~aYi5mJYaiq9-8cI6fYrMsKP9GFDJapMALGs~wb2PaKCvoRetgztkpYQliI7rLr1pQHOnqkUtUbqyLMUoldlUwR-p30ei5vjqYJd2dX6P~TTyHuFum2-ONpJB0ujGoy4h4f4uJa5IW2ywKB-BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649621&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03609.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Level 2 GO term assignment to the olive fully assembled dataset and distribution in categories of BP, MF, and CC.</p></div></div><div content-id="DSS036F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/m_dss03609.jpeg?Expires=2147483647&Signature=wEfV7ePVcMz06pewqV4NXGGt2~pJQ~8pSd4syJpHcwtz5JGmG7I4pGABNoIx6zs0ti4mPYIVvWnAA2kt2M1k1jq4qUqWW7VFKC3lb4X5Bqp~6ntZbxiVsR4ARSxdm6BNFlHTgOiiSHsUakDenwPcrkAyahW8T3SPqzf54N8Yv71c7uy5~OXwdiL7UiLDgN-Q1LdJI2T6sER858UAkasNk0535bNZmiiyj~~7c5NwHod7JUX42AVHjWb1f58i9cwErldqEEHZvlIUI8QWQBteIgikMiB5vGGzI8JQ63JFlZ4xr6ituJzTW8ewbtFj5rdARlcIjVRzKLN-wtI50v2qUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Level 2 GO term assignment to the olive fully assembled dataset and distribution in categories of BP, MF, and CC." path-from-xml="dss03609.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5649621/dss03609.jpeg" path-from-xml="dss03609.jpeg" target="_blank">View large</a><a section="5649621" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss03609.jpeg?Expires=2147483647&Signature=b9bLXrr8L9B2P4TlfL4nzOMUXVBPwx4sqLnqCRBgxwjL-P8o9sYMi1zxPo7cFUDQc~9F-Tj2YBAi6Jq5EINqI3xIZ4mJ4XD-8FevH~Nix-Q-nNWcfD05oocR2AK8M4yfdd1ryocqCTGhi40gyhwdHOSaajM3~q3NNGV~W~T-tElWEjVtcWSMsIrrX901V9Zn8bYB~aYi5mJYaiq9-8cI6fYrMsKP9GFDJapMALGs~wb2PaKCvoRetgztkpYQliI7rLr1pQHOnqkUtUbqyLMUoldlUwR-p30ei5vjqYJd2dX6P~TTyHuFum2-ONpJB0ujGoy4h4f4uJa5IW2ywKB-BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5649621&ar=384291&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03609.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Level 2 GO term assignment to the olive fully assembled dataset and distribution in categories of BP, MF, and CC.</p></div></div><p>Comparisons between the functional annotations associated with tissue-specific genes can provide insights into the expression profile in relation to the nature and biological function of a given tissue. We analysed the functional enrichment of GO terms in meristem-specific genes using Fisher's exact test.<sup><a reveal-id="DSS036C39" data-open="DSS036C39" class="link link-ref link-reveal xref-bibr">39</a></sup> GO terms associated with cellular division and DNA replication, as well as response to stimuli and hormones, were remarkable overrepresented among meristem-specific genes in comparison with the rest of the transcriptome (Table <a reveal-id="DSS036TB2" data-open="DSS036TB2" class="link link-reveal link-table xref-fig">2</a>). Because meristems are actively growing sites, key for plant development and shoot branching, both processes strongly influenced by environmental and endogenous signals, such as hormones, the overrepresented GO terms in this tissue are full in keeping with these activities. Interestingly, a clear underrepresentation of terms associated with ubiquitin-mediated protein degradation and regulation could also be seen at several GO levels (Table <a reveal-id="DSS036TB2" data-open="DSS036TB2" class="link link-reveal link-table xref-fig">2</a>), suggesting that such regulatory mechanism does not have a major role in the regulation of meristem-specific processes. </p>                        <a id="5649623" scrollto-destination="5649623"></a>
<div content-id="DSS036TB2" class="table-modal"><div class="table-wrap"><div id="DSS036TB2" class="table-wrap-title" data-id="DSS036TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Significantly enriched terms (FDR &lt; 0.05) for potentially meristem-specific unigenes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Level </th><th>GO term </th><th>Name </th><th>FDR </th></tr></thead><tbody><tr><td colspan="4">Overrepresented GO terms </td></tr><tr><td colspan="4">Meristematic activity </td></tr><tr><td> 4 </td><td>GO:0048532 </td><td>Anatomical structure arrangement </td><td>7.6 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0009933 </td><td>Meristem structural organization </td><td>7.6 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0048509 </td><td>Regulation of meristem development </td><td>7.0 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">Responses to stimuli </td></tr><tr><td> 4 </td><td>GO:0009725 </td><td>Response to hormone stimulus </td><td>3.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0009719 </td><td>Response to endogenous stimulus </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009755 </td><td>Hormone-mediated signalling pathway </td><td>3.2 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0010033 </td><td>Response to organic substance </td><td>3.5 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0009314 </td><td>Response to radiation </td><td>3.3 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009914 </td><td>Hormone transport </td><td>1.1 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009416 </td><td>Response to light stimulus </td><td>1.7 × 10<sup>−2</sup> </td></tr><tr><td> 5 </td><td>GO:0009733 </td><td>Response to auxin stimulus </td><td>1.4 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0060918 </td><td>Auxin transport </td><td>2.5 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0071365 </td><td>Cellular response to auxin stimulus </td><td>8.3 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0010332 </td><td>Response to gamma radiation </td><td>3.7 × 10<sup>−2</sup> </td></tr><tr><td> 7 </td><td>GO:0009926 </td><td>Auxin polar transport </td><td>2.5 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">DNA replication and cell division </td></tr><tr><td> 3 </td><td>GO:0071554 </td><td>Cell wall organization or biogenesis </td><td>2.3 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0007049 </td><td>Cell cycle </td><td>3.1 × 10<sup>−9</sup> </td></tr><tr><td> 3 </td><td>GO:0051301 </td><td>Cell division </td><td>9.0 × 10<sup>−6</sup> </td></tr><tr><td> 5 </td><td>GO:0042546 </td><td>Cell wall biogenesis </td><td>1.7 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0070882 </td><td>Cellular cell wall organization or biogenesis </td><td>7.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0007059 </td><td>Chromosome segregation </td><td>2.2 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0071103 </td><td>DNA conformation change </td><td>1.3 × 10<sup>−2</sup> </td></tr><tr><td> 8 </td><td>GO:0006323 </td><td>DNA packaging </td><td>3.3 × 10<sup>−4</sup> </td></tr><tr><td> 5 </td><td>GO:0071669 </td><td>Plant-type cell wall organization or biogenesis </td><td>1.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0032993 </td><td>Protein–DNA complex </td><td>8.8 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0022402 </td><td>Cell cycle process </td><td>5.2 × 10<sup>−12</sup> </td></tr><tr><td> 6 </td><td>GO:0030261 </td><td>Chromosome condensation </td><td>2.5 × 10<sup>−6</sup> </td></tr><tr><td> 4 </td><td>GO:0000910 </td><td>Cytokinesis </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0000278 </td><td>Mitotic cell cycle </td><td>2.5 × 10<sup>−9</sup> </td></tr><tr><td> 6 </td><td>GO:0048285 </td><td>Organelle fission </td><td>3.5 × 10<sup>−6</sup> </td></tr><tr><td> 6 </td><td>GO:0009832 </td><td>Plant-type cell wall biogenesis </td><td>3.7 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0007062 </td><td>Sister chromatid cohesion </td><td>2.6 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0033205 </td><td>Cell cycle cytokinesis </td><td>8.5 × 10<sup>−6</sup> </td></tr><tr><td> 5 </td><td>GO:0022403 </td><td>Cell cycle phase </td><td>5.4 × 10<sup>−7</sup> </td></tr><tr><td> 5 </td><td>GO:0032870 </td><td>Cellular response to hormone stimulus </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0032506 </td><td>Cytokinetic process </td><td>1.5 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0000280 </td><td>Nuclear division </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 5 </td><td>GO:0090304 </td><td>Nucleic acid metabolic process </td><td>1.3 × 10<sup>−24</sup> </td></tr><tr><td> 5 </td><td>GO:0051726 </td><td>Regulation of cell cycle </td><td>1.1 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0009834 </td><td>Secondary cell wall biogenesis </td><td>8.0 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0044427 </td><td>Chromosomal part </td><td>9.3 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0000911 </td><td>Cytokinesis by cell plate formation </td><td>8.5 × 10<sup>−6</sup> </td></tr><tr><td> 6 </td><td>GO:0015074 </td><td>DNA integration </td><td>3.2 × 10<sup>−36</sup> </td></tr><tr><td> 6 </td><td>GO:0006259 </td><td>DNA metabolic process </td><td>7.0 × 10<sup>−56</sup> </td></tr><tr><td> 6 </td><td>GO:0034061 </td><td>DNA polymerase activity </td><td>3.5 × 10<sup>−10</sup> </td></tr><tr><td> 6 </td><td>GO:0006310 </td><td>DNA recombination </td><td>3.1 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0006260 </td><td>DNA replication </td><td>8.1 × 10<sup>−21</sup> </td></tr><tr><td> 6 </td><td>GO:0009008 </td><td>DNA-methyltransferase activity </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0051325 </td><td>Interphase </td><td>3.7 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0051329 </td><td>Interphase of mitotic cell cycle </td><td>3.7 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0000279 </td><td>M phase </td><td>6.8 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0000087 </td><td>M phase of mitotic cell cycle </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 6 </td><td>GO:0010564 </td><td>Regulation of cell cycle process </td><td>1.8 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0009934 </td><td>Regulation of meristem structural organization </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0007346 </td><td>Regulation of mitotic cell cycle </td><td>1.2 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0030894 </td><td>Replisome </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0003886 </td><td>DNA (cytosine-5-)-methyltransferase activity </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0003887 </td><td>DNA-directed DNA polymerase activity </td><td>1.5 × 10<sup>−5</sup> </td></tr><tr><td> 7 </td><td>GO:0000086 </td><td>G2/M transition of mitotic cell cycle </td><td>6.1 × 10<sup>−4</sup> </td></tr><tr><td> 7 </td><td>GO:0007067 </td><td>Mitosis </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 7 </td><td>GO:0043601 </td><td>Nuclear replisome </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0017111 </td><td>Nucleoside-triphosphatase activity </td><td>5.2 × 10<sup>−10</sup> </td></tr><tr><td> 8 </td><td>GO:0010389 </td><td>Regulation of G2/M transition of mitotic cell cycle </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 8 </td><td>GO:0003896 </td><td>DNA primase activity </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 8 </td><td>GO:0006261 </td><td>DNA-dependent DNA replication </td><td>6.7 × 10<sup>−17</sup> </td></tr><tr><td> 8 </td><td>GO:0042555 </td><td>MCM complex </td><td>2.1 × 10<sup>−2</sup> </td></tr><tr><td> 8 </td><td>GO:0043596 </td><td>Nuclear replication fork </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 9 </td><td>GO:0003678 </td><td>DNA helicase activity </td><td>1.5 × 10<sup>−3</sup> </td></tr><tr><td> 9 </td><td>GO:0006269 </td><td>DNA replication, synthesis of RNA primer </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 10 </td><td>GO:0043138 </td><td>3′-5′ DNA helicase activity </td><td>2.1 × 10<sup>−3</sup> </td></tr><tr><td> 11 </td><td>GO:0043140 </td><td>ATP-dependent 3′-5′ DNA helicase activity </td><td>2.1 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">Under-represented GO terms </td></tr><tr><td colspan="4">Ubiquitine-associated processes </td></tr><tr><td> 5 </td><td>GO:0000151 </td><td>Ubiquitin ligase complex </td><td>3.6 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0031461 </td><td>Cullin–RING ubiquitin ligase complex </td><td>1.3 × 10<sup>−2</sup> </td></tr><tr><td> 9 </td><td>GO:0006511 </td><td>Ubiquitin-dependent protein catabolic process </td><td>4.8 × 10<sup>−3</sup> </td></tr></tbody></table></div><div content-id="DSS036TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Level </th><th>GO term </th><th>Name </th><th>FDR </th></tr></thead><tbody><tr><td colspan="4">Overrepresented GO terms </td></tr><tr><td colspan="4">Meristematic activity </td></tr><tr><td> 4 </td><td>GO:0048532 </td><td>Anatomical structure arrangement </td><td>7.6 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0009933 </td><td>Meristem structural organization </td><td>7.6 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0048509 </td><td>Regulation of meristem development </td><td>7.0 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">Responses to stimuli </td></tr><tr><td> 4 </td><td>GO:0009725 </td><td>Response to hormone stimulus </td><td>3.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0009719 </td><td>Response to endogenous stimulus </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009755 </td><td>Hormone-mediated signalling pathway </td><td>3.2 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0010033 </td><td>Response to organic substance </td><td>3.5 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0009314 </td><td>Response to radiation </td><td>3.3 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009914 </td><td>Hormone transport </td><td>1.1 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009416 </td><td>Response to light stimulus </td><td>1.7 × 10<sup>−2</sup> </td></tr><tr><td> 5 </td><td>GO:0009733 </td><td>Response to auxin stimulus </td><td>1.4 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0060918 </td><td>Auxin transport </td><td>2.5 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0071365 </td><td>Cellular response to auxin stimulus </td><td>8.3 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0010332 </td><td>Response to gamma radiation </td><td>3.7 × 10<sup>−2</sup> </td></tr><tr><td> 7 </td><td>GO:0009926 </td><td>Auxin polar transport </td><td>2.5 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">DNA replication and cell division </td></tr><tr><td> 3 </td><td>GO:0071554 </td><td>Cell wall organization or biogenesis </td><td>2.3 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0007049 </td><td>Cell cycle </td><td>3.1 × 10<sup>−9</sup> </td></tr><tr><td> 3 </td><td>GO:0051301 </td><td>Cell division </td><td>9.0 × 10<sup>−6</sup> </td></tr><tr><td> 5 </td><td>GO:0042546 </td><td>Cell wall biogenesis </td><td>1.7 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0070882 </td><td>Cellular cell wall organization or biogenesis </td><td>7.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0007059 </td><td>Chromosome segregation </td><td>2.2 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0071103 </td><td>DNA conformation change </td><td>1.3 × 10<sup>−2</sup> </td></tr><tr><td> 8 </td><td>GO:0006323 </td><td>DNA packaging </td><td>3.3 × 10<sup>−4</sup> </td></tr><tr><td> 5 </td><td>GO:0071669 </td><td>Plant-type cell wall organization or biogenesis </td><td>1.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0032993 </td><td>Protein–DNA complex </td><td>8.8 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0022402 </td><td>Cell cycle process </td><td>5.2 × 10<sup>−12</sup> </td></tr><tr><td> 6 </td><td>GO:0030261 </td><td>Chromosome condensation </td><td>2.5 × 10<sup>−6</sup> </td></tr><tr><td> 4 </td><td>GO:0000910 </td><td>Cytokinesis </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0000278 </td><td>Mitotic cell cycle </td><td>2.5 × 10<sup>−9</sup> </td></tr><tr><td> 6 </td><td>GO:0048285 </td><td>Organelle fission </td><td>3.5 × 10<sup>−6</sup> </td></tr><tr><td> 6 </td><td>GO:0009832 </td><td>Plant-type cell wall biogenesis </td><td>3.7 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0007062 </td><td>Sister chromatid cohesion </td><td>2.6 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0033205 </td><td>Cell cycle cytokinesis </td><td>8.5 × 10<sup>−6</sup> </td></tr><tr><td> 5 </td><td>GO:0022403 </td><td>Cell cycle phase </td><td>5.4 × 10<sup>−7</sup> </td></tr><tr><td> 5 </td><td>GO:0032870 </td><td>Cellular response to hormone stimulus </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0032506 </td><td>Cytokinetic process </td><td>1.5 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0000280 </td><td>Nuclear division </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 5 </td><td>GO:0090304 </td><td>Nucleic acid metabolic process </td><td>1.3 × 10<sup>−24</sup> </td></tr><tr><td> 5 </td><td>GO:0051726 </td><td>Regulation of cell cycle </td><td>1.1 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0009834 </td><td>Secondary cell wall biogenesis </td><td>8.0 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0044427 </td><td>Chromosomal part </td><td>9.3 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0000911 </td><td>Cytokinesis by cell plate formation </td><td>8.5 × 10<sup>−6</sup> </td></tr><tr><td> 6 </td><td>GO:0015074 </td><td>DNA integration </td><td>3.2 × 10<sup>−36</sup> </td></tr><tr><td> 6 </td><td>GO:0006259 </td><td>DNA metabolic process </td><td>7.0 × 10<sup>−56</sup> </td></tr><tr><td> 6 </td><td>GO:0034061 </td><td>DNA polymerase activity </td><td>3.5 × 10<sup>−10</sup> </td></tr><tr><td> 6 </td><td>GO:0006310 </td><td>DNA recombination </td><td>3.1 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0006260 </td><td>DNA replication </td><td>8.1 × 10<sup>−21</sup> </td></tr><tr><td> 6 </td><td>GO:0009008 </td><td>DNA-methyltransferase activity </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0051325 </td><td>Interphase </td><td>3.7 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0051329 </td><td>Interphase of mitotic cell cycle </td><td>3.7 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0000279 </td><td>M phase </td><td>6.8 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0000087 </td><td>M phase of mitotic cell cycle </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 6 </td><td>GO:0010564 </td><td>Regulation of cell cycle process </td><td>1.8 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0009934 </td><td>Regulation of meristem structural organization </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0007346 </td><td>Regulation of mitotic cell cycle </td><td>1.2 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0030894 </td><td>Replisome </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0003886 </td><td>DNA (cytosine-5-)-methyltransferase activity </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0003887 </td><td>DNA-directed DNA polymerase activity </td><td>1.5 × 10<sup>−5</sup> </td></tr><tr><td> 7 </td><td>GO:0000086 </td><td>G2/M transition of mitotic cell cycle </td><td>6.1 × 10<sup>−4</sup> </td></tr><tr><td> 7 </td><td>GO:0007067 </td><td>Mitosis </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 7 </td><td>GO:0043601 </td><td>Nuclear replisome </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0017111 </td><td>Nucleoside-triphosphatase activity </td><td>5.2 × 10<sup>−10</sup> </td></tr><tr><td> 8 </td><td>GO:0010389 </td><td>Regulation of G2/M transition of mitotic cell cycle </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 8 </td><td>GO:0003896 </td><td>DNA primase activity </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 8 </td><td>GO:0006261 </td><td>DNA-dependent DNA replication </td><td>6.7 × 10<sup>−17</sup> </td></tr><tr><td> 8 </td><td>GO:0042555 </td><td>MCM complex </td><td>2.1 × 10<sup>−2</sup> </td></tr><tr><td> 8 </td><td>GO:0043596 </td><td>Nuclear replication fork </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 9 </td><td>GO:0003678 </td><td>DNA helicase activity </td><td>1.5 × 10<sup>−3</sup> </td></tr><tr><td> 9 </td><td>GO:0006269 </td><td>DNA replication, synthesis of RNA primer </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 10 </td><td>GO:0043138 </td><td>3′-5′ DNA helicase activity </td><td>2.1 × 10<sup>−3</sup> </td></tr><tr><td> 11 </td><td>GO:0043140 </td><td>ATP-dependent 3′-5′ DNA helicase activity </td><td>2.1 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">Under-represented GO terms </td></tr><tr><td colspan="4">Ubiquitine-associated processes </td></tr><tr><td> 5 </td><td>GO:0000151 </td><td>Ubiquitin ligase complex </td><td>3.6 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0031461 </td><td>Cullin–RING ubiquitin ligase complex </td><td>1.3 × 10<sup>−2</sup> </td></tr><tr><td> 9 </td><td>GO:0006511 </td><td>Ubiquitin-dependent protein catabolic process </td><td>4.8 × 10<sup>−3</sup> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5649623" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS036TB2" class="table-wrap-title" data-id="DSS036TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Significantly enriched terms (FDR &lt; 0.05) for potentially meristem-specific unigenes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Level </th><th>GO term </th><th>Name </th><th>FDR </th></tr></thead><tbody><tr><td colspan="4">Overrepresented GO terms </td></tr><tr><td colspan="4">Meristematic activity </td></tr><tr><td> 4 </td><td>GO:0048532 </td><td>Anatomical structure arrangement </td><td>7.6 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0009933 </td><td>Meristem structural organization </td><td>7.6 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0048509 </td><td>Regulation of meristem development </td><td>7.0 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">Responses to stimuli </td></tr><tr><td> 4 </td><td>GO:0009725 </td><td>Response to hormone stimulus </td><td>3.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0009719 </td><td>Response to endogenous stimulus </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009755 </td><td>Hormone-mediated signalling pathway </td><td>3.2 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0010033 </td><td>Response to organic substance </td><td>3.5 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0009314 </td><td>Response to radiation </td><td>3.3 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009914 </td><td>Hormone transport </td><td>1.1 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009416 </td><td>Response to light stimulus </td><td>1.7 × 10<sup>−2</sup> </td></tr><tr><td> 5 </td><td>GO:0009733 </td><td>Response to auxin stimulus </td><td>1.4 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0060918 </td><td>Auxin transport </td><td>2.5 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0071365 </td><td>Cellular response to auxin stimulus </td><td>8.3 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0010332 </td><td>Response to gamma radiation </td><td>3.7 × 10<sup>−2</sup> </td></tr><tr><td> 7 </td><td>GO:0009926 </td><td>Auxin polar transport </td><td>2.5 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">DNA replication and cell division </td></tr><tr><td> 3 </td><td>GO:0071554 </td><td>Cell wall organization or biogenesis </td><td>2.3 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0007049 </td><td>Cell cycle </td><td>3.1 × 10<sup>−9</sup> </td></tr><tr><td> 3 </td><td>GO:0051301 </td><td>Cell division </td><td>9.0 × 10<sup>−6</sup> </td></tr><tr><td> 5 </td><td>GO:0042546 </td><td>Cell wall biogenesis </td><td>1.7 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0070882 </td><td>Cellular cell wall organization or biogenesis </td><td>7.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0007059 </td><td>Chromosome segregation </td><td>2.2 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0071103 </td><td>DNA conformation change </td><td>1.3 × 10<sup>−2</sup> </td></tr><tr><td> 8 </td><td>GO:0006323 </td><td>DNA packaging </td><td>3.3 × 10<sup>−4</sup> </td></tr><tr><td> 5 </td><td>GO:0071669 </td><td>Plant-type cell wall organization or biogenesis </td><td>1.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0032993 </td><td>Protein–DNA complex </td><td>8.8 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0022402 </td><td>Cell cycle process </td><td>5.2 × 10<sup>−12</sup> </td></tr><tr><td> 6 </td><td>GO:0030261 </td><td>Chromosome condensation </td><td>2.5 × 10<sup>−6</sup> </td></tr><tr><td> 4 </td><td>GO:0000910 </td><td>Cytokinesis </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0000278 </td><td>Mitotic cell cycle </td><td>2.5 × 10<sup>−9</sup> </td></tr><tr><td> 6 </td><td>GO:0048285 </td><td>Organelle fission </td><td>3.5 × 10<sup>−6</sup> </td></tr><tr><td> 6 </td><td>GO:0009832 </td><td>Plant-type cell wall biogenesis </td><td>3.7 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0007062 </td><td>Sister chromatid cohesion </td><td>2.6 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0033205 </td><td>Cell cycle cytokinesis </td><td>8.5 × 10<sup>−6</sup> </td></tr><tr><td> 5 </td><td>GO:0022403 </td><td>Cell cycle phase </td><td>5.4 × 10<sup>−7</sup> </td></tr><tr><td> 5 </td><td>GO:0032870 </td><td>Cellular response to hormone stimulus </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0032506 </td><td>Cytokinetic process </td><td>1.5 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0000280 </td><td>Nuclear division </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 5 </td><td>GO:0090304 </td><td>Nucleic acid metabolic process </td><td>1.3 × 10<sup>−24</sup> </td></tr><tr><td> 5 </td><td>GO:0051726 </td><td>Regulation of cell cycle </td><td>1.1 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0009834 </td><td>Secondary cell wall biogenesis </td><td>8.0 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0044427 </td><td>Chromosomal part </td><td>9.3 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0000911 </td><td>Cytokinesis by cell plate formation </td><td>8.5 × 10<sup>−6</sup> </td></tr><tr><td> 6 </td><td>GO:0015074 </td><td>DNA integration </td><td>3.2 × 10<sup>−36</sup> </td></tr><tr><td> 6 </td><td>GO:0006259 </td><td>DNA metabolic process </td><td>7.0 × 10<sup>−56</sup> </td></tr><tr><td> 6 </td><td>GO:0034061 </td><td>DNA polymerase activity </td><td>3.5 × 10<sup>−10</sup> </td></tr><tr><td> 6 </td><td>GO:0006310 </td><td>DNA recombination </td><td>3.1 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0006260 </td><td>DNA replication </td><td>8.1 × 10<sup>−21</sup> </td></tr><tr><td> 6 </td><td>GO:0009008 </td><td>DNA-methyltransferase activity </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0051325 </td><td>Interphase </td><td>3.7 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0051329 </td><td>Interphase of mitotic cell cycle </td><td>3.7 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0000279 </td><td>M phase </td><td>6.8 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0000087 </td><td>M phase of mitotic cell cycle </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 6 </td><td>GO:0010564 </td><td>Regulation of cell cycle process </td><td>1.8 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0009934 </td><td>Regulation of meristem structural organization </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0007346 </td><td>Regulation of mitotic cell cycle </td><td>1.2 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0030894 </td><td>Replisome </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0003886 </td><td>DNA (cytosine-5-)-methyltransferase activity </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0003887 </td><td>DNA-directed DNA polymerase activity </td><td>1.5 × 10<sup>−5</sup> </td></tr><tr><td> 7 </td><td>GO:0000086 </td><td>G2/M transition of mitotic cell cycle </td><td>6.1 × 10<sup>−4</sup> </td></tr><tr><td> 7 </td><td>GO:0007067 </td><td>Mitosis </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 7 </td><td>GO:0043601 </td><td>Nuclear replisome </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0017111 </td><td>Nucleoside-triphosphatase activity </td><td>5.2 × 10<sup>−10</sup> </td></tr><tr><td> 8 </td><td>GO:0010389 </td><td>Regulation of G2/M transition of mitotic cell cycle </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 8 </td><td>GO:0003896 </td><td>DNA primase activity </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 8 </td><td>GO:0006261 </td><td>DNA-dependent DNA replication </td><td>6.7 × 10<sup>−17</sup> </td></tr><tr><td> 8 </td><td>GO:0042555 </td><td>MCM complex </td><td>2.1 × 10<sup>−2</sup> </td></tr><tr><td> 8 </td><td>GO:0043596 </td><td>Nuclear replication fork </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 9 </td><td>GO:0003678 </td><td>DNA helicase activity </td><td>1.5 × 10<sup>−3</sup> </td></tr><tr><td> 9 </td><td>GO:0006269 </td><td>DNA replication, synthesis of RNA primer </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 10 </td><td>GO:0043138 </td><td>3′-5′ DNA helicase activity </td><td>2.1 × 10<sup>−3</sup> </td></tr><tr><td> 11 </td><td>GO:0043140 </td><td>ATP-dependent 3′-5′ DNA helicase activity </td><td>2.1 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">Under-represented GO terms </td></tr><tr><td colspan="4">Ubiquitine-associated processes </td></tr><tr><td> 5 </td><td>GO:0000151 </td><td>Ubiquitin ligase complex </td><td>3.6 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0031461 </td><td>Cullin–RING ubiquitin ligase complex </td><td>1.3 × 10<sup>−2</sup> </td></tr><tr><td> 9 </td><td>GO:0006511 </td><td>Ubiquitin-dependent protein catabolic process </td><td>4.8 × 10<sup>−3</sup> </td></tr></tbody></table></div><div content-id="DSS036TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Level </th><th>GO term </th><th>Name </th><th>FDR </th></tr></thead><tbody><tr><td colspan="4">Overrepresented GO terms </td></tr><tr><td colspan="4">Meristematic activity </td></tr><tr><td> 4 </td><td>GO:0048532 </td><td>Anatomical structure arrangement </td><td>7.6 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0009933 </td><td>Meristem structural organization </td><td>7.6 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0048509 </td><td>Regulation of meristem development </td><td>7.0 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">Responses to stimuli </td></tr><tr><td> 4 </td><td>GO:0009725 </td><td>Response to hormone stimulus </td><td>3.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0009719 </td><td>Response to endogenous stimulus </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009755 </td><td>Hormone-mediated signalling pathway </td><td>3.2 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0010033 </td><td>Response to organic substance </td><td>3.5 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0009314 </td><td>Response to radiation </td><td>3.3 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009914 </td><td>Hormone transport </td><td>1.1 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0009416 </td><td>Response to light stimulus </td><td>1.7 × 10<sup>−2</sup> </td></tr><tr><td> 5 </td><td>GO:0009733 </td><td>Response to auxin stimulus </td><td>1.4 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0060918 </td><td>Auxin transport </td><td>2.5 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0071365 </td><td>Cellular response to auxin stimulus </td><td>8.3 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0010332 </td><td>Response to gamma radiation </td><td>3.7 × 10<sup>−2</sup> </td></tr><tr><td> 7 </td><td>GO:0009926 </td><td>Auxin polar transport </td><td>2.5 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">DNA replication and cell division </td></tr><tr><td> 3 </td><td>GO:0071554 </td><td>Cell wall organization or biogenesis </td><td>2.3 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0007049 </td><td>Cell cycle </td><td>3.1 × 10<sup>−9</sup> </td></tr><tr><td> 3 </td><td>GO:0051301 </td><td>Cell division </td><td>9.0 × 10<sup>−6</sup> </td></tr><tr><td> 5 </td><td>GO:0042546 </td><td>Cell wall biogenesis </td><td>1.7 × 10<sup>−2</sup> </td></tr><tr><td> 4 </td><td>GO:0070882 </td><td>Cellular cell wall organization or biogenesis </td><td>7.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0007059 </td><td>Chromosome segregation </td><td>2.2 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0071103 </td><td>DNA conformation change </td><td>1.3 × 10<sup>−2</sup> </td></tr><tr><td> 8 </td><td>GO:0006323 </td><td>DNA packaging </td><td>3.3 × 10<sup>−4</sup> </td></tr><tr><td> 5 </td><td>GO:0071669 </td><td>Plant-type cell wall organization or biogenesis </td><td>1.0 × 10<sup>−3</sup> </td></tr><tr><td> 3 </td><td>GO:0032993 </td><td>Protein–DNA complex </td><td>8.8 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0022402 </td><td>Cell cycle process </td><td>5.2 × 10<sup>−12</sup> </td></tr><tr><td> 6 </td><td>GO:0030261 </td><td>Chromosome condensation </td><td>2.5 × 10<sup>−6</sup> </td></tr><tr><td> 4 </td><td>GO:0000910 </td><td>Cytokinesis </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0000278 </td><td>Mitotic cell cycle </td><td>2.5 × 10<sup>−9</sup> </td></tr><tr><td> 6 </td><td>GO:0048285 </td><td>Organelle fission </td><td>3.5 × 10<sup>−6</sup> </td></tr><tr><td> 6 </td><td>GO:0009832 </td><td>Plant-type cell wall biogenesis </td><td>3.7 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0007062 </td><td>Sister chromatid cohesion </td><td>2.6 × 10<sup>−3</sup> </td></tr><tr><td> 5 </td><td>GO:0033205 </td><td>Cell cycle cytokinesis </td><td>8.5 × 10<sup>−6</sup> </td></tr><tr><td> 5 </td><td>GO:0022403 </td><td>Cell cycle phase </td><td>5.4 × 10<sup>−7</sup> </td></tr><tr><td> 5 </td><td>GO:0032870 </td><td>Cellular response to hormone stimulus </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 4 </td><td>GO:0032506 </td><td>Cytokinetic process </td><td>1.5 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0000280 </td><td>Nuclear division </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 5 </td><td>GO:0090304 </td><td>Nucleic acid metabolic process </td><td>1.3 × 10<sup>−24</sup> </td></tr><tr><td> 5 </td><td>GO:0051726 </td><td>Regulation of cell cycle </td><td>1.1 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0009834 </td><td>Secondary cell wall biogenesis </td><td>8.0 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0044427 </td><td>Chromosomal part </td><td>9.3 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0000911 </td><td>Cytokinesis by cell plate formation </td><td>8.5 × 10<sup>−6</sup> </td></tr><tr><td> 6 </td><td>GO:0015074 </td><td>DNA integration </td><td>3.2 × 10<sup>−36</sup> </td></tr><tr><td> 6 </td><td>GO:0006259 </td><td>DNA metabolic process </td><td>7.0 × 10<sup>−56</sup> </td></tr><tr><td> 6 </td><td>GO:0034061 </td><td>DNA polymerase activity </td><td>3.5 × 10<sup>−10</sup> </td></tr><tr><td> 6 </td><td>GO:0006310 </td><td>DNA recombination </td><td>3.1 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0006260 </td><td>DNA replication </td><td>8.1 × 10<sup>−21</sup> </td></tr><tr><td> 6 </td><td>GO:0009008 </td><td>DNA-methyltransferase activity </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0051325 </td><td>Interphase </td><td>3.7 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0051329 </td><td>Interphase of mitotic cell cycle </td><td>3.7 × 10<sup>−4</sup> </td></tr><tr><td> 6 </td><td>GO:0000279 </td><td>M phase </td><td>6.8 × 10<sup>−5</sup> </td></tr><tr><td> 6 </td><td>GO:0000087 </td><td>M phase of mitotic cell cycle </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 6 </td><td>GO:0010564 </td><td>Regulation of cell cycle process </td><td>1.8 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0009934 </td><td>Regulation of meristem structural organization </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0007346 </td><td>Regulation of mitotic cell cycle </td><td>1.2 × 10<sup>−3</sup> </td></tr><tr><td> 6 </td><td>GO:0030894 </td><td>Replisome </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0003886 </td><td>DNA (cytosine-5-)-methyltransferase activity </td><td>5.5 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0003887 </td><td>DNA-directed DNA polymerase activity </td><td>1.5 × 10<sup>−5</sup> </td></tr><tr><td> 7 </td><td>GO:0000086 </td><td>G2/M transition of mitotic cell cycle </td><td>6.1 × 10<sup>−4</sup> </td></tr><tr><td> 7 </td><td>GO:0007067 </td><td>Mitosis </td><td>7.9 × 10<sup>−7</sup> </td></tr><tr><td> 7 </td><td>GO:0043601 </td><td>Nuclear replisome </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 7 </td><td>GO:0017111 </td><td>Nucleoside-triphosphatase activity </td><td>5.2 × 10<sup>−10</sup> </td></tr><tr><td> 8 </td><td>GO:0010389 </td><td>Regulation of G2/M transition of mitotic cell cycle </td><td>2.4 × 10<sup>−3</sup> </td></tr><tr><td> 8 </td><td>GO:0003896 </td><td>DNA primase activity </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 8 </td><td>GO:0006261 </td><td>DNA-dependent DNA replication </td><td>6.7 × 10<sup>−17</sup> </td></tr><tr><td> 8 </td><td>GO:0042555 </td><td>MCM complex </td><td>2.1 × 10<sup>−2</sup> </td></tr><tr><td> 8 </td><td>GO:0043596 </td><td>Nuclear replication fork </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 9 </td><td>GO:0003678 </td><td>DNA helicase activity </td><td>1.5 × 10<sup>−3</sup> </td></tr><tr><td> 9 </td><td>GO:0006269 </td><td>DNA replication, synthesis of RNA primer </td><td>6.2 × 10<sup>−3</sup> </td></tr><tr><td> 10 </td><td>GO:0043138 </td><td>3′-5′ DNA helicase activity </td><td>2.1 × 10<sup>−3</sup> </td></tr><tr><td> 11 </td><td>GO:0043140 </td><td>ATP-dependent 3′-5′ DNA helicase activity </td><td>2.1 × 10<sup>−3</sup> </td></tr><tr><td colspan="4">Under-represented GO terms </td></tr><tr><td colspan="4">Ubiquitine-associated processes </td></tr><tr><td> 5 </td><td>GO:0000151 </td><td>Ubiquitin ligase complex </td><td>3.6 × 10<sup>−2</sup> </td></tr><tr><td> 6 </td><td>GO:0031461 </td><td>Cullin–RING ubiquitin ligase complex </td><td>1.3 × 10<sup>−2</sup> </td></tr><tr><td> 9 </td><td>GO:0006511 </td><td>Ubiquitin-dependent protein catabolic process </td><td>4.8 × 10<sup>−3</sup> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5649623" target="_blank">View Large</a></div></div><p>Unfortunately, no such a clear specific profile could be identified for the other tissues. Few relevant differences could be established with confidence in unigenes potentially specific for roots, probably owing to their significantly smaller number, likely determined by the considerably smaller number of reads obtained from this tissue. However, this underrepresentation of root genes probably allowed us to obtain the clear functional enrichment found in meristems because the likely overlap expected between the functional annotation of both tissues (e.g. abundance of GO terms associated with cell division and DNA replication) could have potentially hindered the statistical validation of such specific enrichment in meristem-specific genes. Interestingly, among the very few overrepresented GO terms associated with root-specific genes were acropetal auxin transport (GO:0010541; FDR 3 × 10<sup>−2</sup>) and basipetal auxin transport (GO:0010540; FDR 3 × 10<sup>−2</sup>).</p><p>A functional overlap can also be expected between mesocarp and seeds regarding GO terms (i.e. abundance of GO terms associated with oil production), and it is probably the reason why our comparative analysis has not rendered clearer functional profiles for these two tissues. However, a number of unigenes from our assembled dataset have the annotation of GO terms associated with oil production. Quantitative analysis on the relative level of expression of unigenes associated with oil production-related GO terms would probably give insight into the gene expression profile in relation to the biological activities within these tissues.</p><p>The functional profiles obtained through GO annotation from the unigenes expressed in meristems are in good agreement with the biological characteristics and activities expected for this tissue, indicating that the transcriptome hereby assembled is a valuable representation of the full transcriptome for this tissue. Thus, the dataset can be highly useful for the identification of interesting candidate genes in traits associated with meristematic activity such as regulation of the juvenility period or tree architecture. Additional work will be necessary to test the potential for the identification of candidate genes involved in oil composition and quality. However, the presence of almost 300 unigenes annotated with the GO level 4 term lipid metabolic process, or lipid biosynthesis (Table <a reveal-id="DSS036TB3" data-open="DSS036TB3" class="link link-reveal link-table xref-fig">3</a>), does support such potential. This report represents a major contribution to the molecular knowledge of the species, opening the possibility of developing molecular tools to the level currently available for other model plant species, such as <em>Populus</em> or apple tree. </p>                        <a id="5649627" scrollto-destination="5649627"></a>
<div content-id="DSS036TB3" class="table-modal"><div class="table-wrap"><div id="DSS036TB3" class="table-wrap-title" data-id="DSS036TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of annotations within the full assembly (AS8) to GO terms associated with oil production</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GO level </th><th>Name </th><th>Number of hits in AS8 </th></tr></thead><tbody><tr><td>4 </td><td>Lipid metabolic process </td><td>250 </td></tr><tr><td>4 </td><td>Lipid biosynthetic process </td><td>25 </td></tr><tr><td>6 </td><td>Fatty acid biosynthetic process </td><td>156 </td></tr><tr><td>6 </td><td>Fatty acid metabolic process </td><td>46 </td></tr></tbody></table></div><div content-id="DSS036TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>GO level </th><th>Name </th><th>Number of hits in AS8 </th></tr></thead><tbody><tr><td>4 </td><td>Lipid metabolic process </td><td>250 </td></tr><tr><td>4 </td><td>Lipid biosynthetic process </td><td>25 </td></tr><tr><td>6 </td><td>Fatty acid biosynthetic process </td><td>156 </td></tr><tr><td>6 </td><td>Fatty acid metabolic process </td><td>46 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5649627" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS036TB3" class="table-wrap-title" data-id="DSS036TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of annotations within the full assembly (AS8) to GO terms associated with oil production</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GO level </th><th>Name </th><th>Number of hits in AS8 </th></tr></thead><tbody><tr><td>4 </td><td>Lipid metabolic process </td><td>250 </td></tr><tr><td>4 </td><td>Lipid biosynthetic process </td><td>25 </td></tr><tr><td>6 </td><td>Fatty acid biosynthetic process </td><td>156 </td></tr><tr><td>6 </td><td>Fatty acid metabolic process </td><td>46 </td></tr></tbody></table></div><div content-id="DSS036TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>GO level </th><th>Name </th><th>Number of hits in AS8 </th></tr></thead><tbody><tr><td>4 </td><td>Lipid metabolic process </td><td>250 </td></tr><tr><td>4 </td><td>Lipid biosynthetic process </td><td>25 </td></tr><tr><td>6 </td><td>Fatty acid biosynthetic process </td><td>156 </td></tr><tr><td>6 </td><td>Fatty acid metabolic process </td><td>46 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5649627" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=5649628 id="5649628" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dss036/-/DC1" target="_blank">.</a></p>                            <h2 scrollto-destination=5649630 id="5649630" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by OLEAGEN Project funded by Fundación Genoma España, Junta de Andalucía through Instituto de Investigación y Formación Agraria y Pesquera (IFAPA) and Corporación Tecnológica de Andalucía (CTA).</p>                            <h2 scrollto-destination=5649632 id="5649632" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are extremely grateful to the cooperative breeding program carried out by the University of Cordoba, Spain and the Instituto de Investigación y Formación Agraria y Pesquera (IFAPA), Junta de Andalucía, Spain, for providing all the samples used in this work. We also wish to thank E.R. Bejarano and M.A. Botella for their helpful discussions.</p>                            <h2 scrollto-destination=5649634 id="5649634" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS036C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loumou</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Giourga</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Olive groves: ‘The life and identity of the Mediterranean</div>, <div class="source">Agric. Hum. Values</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">87</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Olive%20groves%3A%20%E2%80%98The%20life%20and%20identity%20of%20the%20Mediterranean&amp;author=A.%20Loumou&amp;author=C.%20Giourga&amp;publication_year=2003&amp;journal=Agric.%20Hum.%20Values&amp;volume=20&amp;pages=87-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022444005336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022444005336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022444005336"> </span></div> </div></div></div></div></div><div content-id="DSS036C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="collab">FAOSTAT</div>, <div class="year">2008</div> <div class="comment">home page Columbia URL: <a class="link link-uri" href="http://www.columbia.edu/cgi-bin/cul/resolve?ASL9609">http://www.columbia.edu/cgi-bin/cul/resolve?ASL9609</a></div></div></div></div></div><div content-id="DSS036C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baldoni</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cultrera</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Mariotti</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">A consensus list of microsatellite markers for olive genotyping</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">213</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20list%20of%20microsatellite%20markers%20for%20olive%20genotyping&amp;author=L.%20Baldoni&amp;author=N.%20Cultrera&amp;author=R.%20Mariotti&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=24&amp;pages=213-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9285-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9285-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9285-8"> </span></div> </div></div></div></div></div><div content-id="DSS036C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conde</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Delrot</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Geros</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Physiological, biochemical and molecular changes occurring during olive development and ripening</div>, <div class="source">J. Plant Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">1545</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physiological%2C%20biochemical%20and%20molecular%20changes%20occurring%20during%20olive%20development%20and%20ripening&amp;author=C.%20Conde&amp;author=S.%20Delrot&amp;author=H.%20Geros&amp;publication_year=2008&amp;journal=J.%20Plant%20Physiol.&amp;volume=165&amp;pages=1545-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2008.04.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2008.04.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2008.04.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18571766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rugini</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Pannelli</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ceccarelli</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Muganu</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Isolation of triploid and tetraploid olive (<em>Olea europaea</em> L.) plants from mixoploid cv. ‘Frantoio’ and ‘Leccino’ mutants by <em>in vivo</em> and <em>in vitro</em> selection</div>, <div class="source">Plant Breed.</div> , <div class="year">1996</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">23</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20triploid%20and%20tetraploid%20olive%20(Olea%20europaea%20L.)%20plants%20from%20mixoploid%20cv.%20%E2%80%98Frantoio%E2%80%99%20and%20%E2%80%98Leccino%E2%80%99%20mutants%20by%20in%20vivo%20and%20in%20vitro%20selection&amp;author=E.%20Rugini&amp;author=G.%20Pannelli&amp;author=M.%20Ceccarelli&amp;author=M.%20Muganu&amp;publication_year=1996&amp;journal=Plant%20Breed.&amp;volume=115&amp;pages=23-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.1996.115.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.1996.115.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.1996.115.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSS036C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">De la Rosa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Angiolillo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Guerrero</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A first linkage map of olive (<em>Olea europaea</em> L.) cultivars using RAPD, AFLP, RFLP and SSR markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">1273</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20first%20linkage%20map%20of%20olive%20(Olea%20europaea%20L.)%20cultivars%20using%20RAPD%2C%20AFLP%2C%20RFLP%20and%20SSR%20markers&amp;author=R.%20De%20la%20Rosa&amp;author=A.%20Angiolillo&amp;author=C.%20Guerrero&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=1273-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12748779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doležel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bartoš</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Voglmayr</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Greilhuber</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Nuclear DNA content and genome size of trout and human</div>, <div class="source">Cytometry A</div> , <div class="year">2003</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">127</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20and%20genome%20size%20of%20trout%20and%20human&amp;author=J.%20Dole%C5%BEel&amp;author=J.%20Barto%C5%A1&amp;author=H.%20Voglmayr&amp;author=J.%20Greilhuber&amp;author=R.A.%20Thomas&amp;publication_year=2003&amp;journal=Cytometry%20A&amp;volume=51&amp;pages=127-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0320" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0320" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12541287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gucci</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Tattini</div> <div class="given-names">M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Janick</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Salinity Tolerance in Olive, in Horticultural Reviews</div>, <div class="year">2010</div>, vol. <div class="volume">21</div> <div class="publisher-loc">UK</div><div class="publisher-name">John Wiley &amp; Sons, Inc.: Oxford,</div>(pg. <div class="fpage">177</div>-<div class="lpage">214</div>)</div></div></div></div><div content-id="DSS036C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bellini</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Genetic variability and heritability of some characters in cross-bred olive seedlings</div>, <div class="source">Olivae</div> , <div class="year">1993</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">21</div>-<div class="lpage">34</div>)</div></div></div></div><div content-id="DSS036C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moreno-Alias</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Rapoport</div> <div class="given-names">H.F.</div></div>,  <div class="name"><div class="surname">Lopez</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Leon</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">De la Rosa</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Optimizing early flowering and pre-selection for short juvenile period in olive seedlings</div>, <div class="source">HortScience</div> , <div class="year">2010</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">519</div>-<div class="lpage">22</div>)</div></div></div></div><div content-id="DSS036C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rallo</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Barranco</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">De La Rosa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">León</div> <div class="given-names">L.</div></div>. </div><div class="article-title">‘Chiquitita’ olive</div>, <div class="source">HortScience</div> , <div class="year">2008</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">529</div>-<div class="lpage">31</div>)</div></div></div></div><div content-id="DSS036C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">León</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">De la Rosa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gracia</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Barranco</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Rallo</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Fatty acid composition of advanced olive selections obtained by crossbreeding</div>, <div class="source">J. Sci. Food Agric.</div> , <div class="year">2008</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">1921</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fatty%20acid%20composition%20of%20advanced%20olive%20selections%20obtained%20by%20crossbreeding&amp;author=L.%20Le%C3%B3n&amp;author=R.%20De%20la%20Rosa&amp;author=A.%20Gracia&amp;author=D.%20Barranco&amp;author=L.%20Rallo&amp;publication_year=2008&amp;journal=J.%20Sci.%20Food%20Agric.&amp;volume=88&amp;pages=1921-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jsfa.v88:11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjsfa.v88:11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjsfa.v88:11"> </span></div> </div></div></div></div></div><div content-id="DSS036C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alagna</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">D'Agostino</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Torchia</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Comparative 454 pyrosequencing of transcripts from two olive genotypes during fruit development</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">399</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20454%20pyrosequencing%20of%20transcripts%20from%20two%20olive%20genotypes%20during%20fruit%20development&amp;author=F.%20Alagna&amp;author=N.%20D'Agostino&amp;author=L.%20Torchia&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=399" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-399" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-399" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-399"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19709400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galla</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Barcaccia</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ramina</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Computational annotation of genes differentially expressed along olive fruit development</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">128</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20annotation%20of%20genes%20differentially%20expressed%20along%20olive%20fruit%20development&amp;author=G.%20Galla&amp;author=G.%20Barcaccia&amp;author=A.%20Ramina&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-128" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-128"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19852839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ozgenturk</div> <div class="given-names">N.O.</div></div>,  <div class="name"><div class="surname">Oruc</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Sezerman</div> <div class="given-names">U.</div></div>, et al. </div><div class="article-title">Generation and analysis of expressed sequence tags from <em>Olea europaea</em> L. Comp</div>, <div class="source">Funct. Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">2010</div> pg. <div class="fpage">757512</div> </div></div></div></div><div content-id="DSS036C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Donaire</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Pedrola</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">De la Rosa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Llave</div> <div class="given-names">C.</div></div>. </div><div class="article-title">High-throughput sequencing of RNA silencing-associated small RNAs in olive (<em>Olea europaea</em> L.)</div>, <div class="source">PLoS ONE</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e27916</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20sequencing%20of%20RNA%20silencing-associated%20small%20RNAs%20in%20olive%20(Olea%20europaea%20L.)&amp;author=L.%20Donaire&amp;author=L.%20Pedrola&amp;author=R.%20De%20la%20Rosa&amp;author=C.%20Llave&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e27916" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0027916" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0027916" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0027916"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Velasco</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zharkikh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A high quality draft consensus sequence of the genome of a heterozygous grapevine variety</div>, <div class="source">PLoS ONE</div> , <div class="year">2007</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e1326</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20quality%20draft%20consensus%20sequence%20of%20the%20genome%20of%20a%20heterozygous%20grapevine%20variety&amp;author=R.%20Velasco&amp;author=A.%20Zharkikh&amp;author=M.%20Troggio&amp;publication_year=2007&amp;journal=PLoS%20ONE&amp;volume=2&amp;pages=e1326" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0001326" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0001326" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0001326"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18094749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaillon</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Noel</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">449</div> (pg. <div class="fpage">463</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20grapevine%20genome%20sequence%20suggests%20ancestral%20hexaploidization%20in%20major%20angiosperm%20phyla&amp;author=O.%20Jaillon&amp;author=J.M.%20Aury&amp;author=B.%20Noel&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=463-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuskan</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">DiFazio</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">1596</div>-<div class="lpage">604</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.A.%20Tuskan&amp;author=S.%20DiFazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bracci</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Busconi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fogher</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Sebastiani</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Molecular studies in olive (<em>Olea europaea</em> L.): overview on DNA markers applications and recent advances in genome analysis</div>, <div class="source">Plant Cell Rep.</div> , <div class="year">2011</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">449</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20studies%20in%20olive%20(Olea%20europaea%20L.)%3A%20overview%20on%20DNA%20markers%20applications%20and%20recent%20advances%20in%20genome%20analysis&amp;author=T.%20Bracci&amp;author=M.%20Busconi&amp;author=C.%20Fogher&amp;author=L.%20Sebastiani&amp;publication_year=2011&amp;journal=Plant%20Cell%20Rep.&amp;volume=30&amp;pages=449-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-010-0991-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-010-0991-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-010-0991-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21212959" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Castillejo</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">De La Fuente</div> <div class="given-names">J.I.</div></div>,  <div class="name"><div class="surname">Iannetta</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Botella</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Valpuesta</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Pectin esterase gene family in strawberry fruit: study of FaPE1, a ripening-specific isoform</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2004</div>, vol. <div class="volume">55</div> pg. <div class="fpage">909</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pectin%20esterase%20gene%20family%20in%20strawberry%20fruit%3A%20study%20of%20FaPE1%2C%20a%20ripening-specific%20isoform&amp;author=C.%20Castillejo&amp;author=J.I.%20De%20La%20Fuente&amp;author=P.%20Iannetta&amp;author=M.A.%20Botella&amp;author=V.%20Valpuesta&amp;publication_year=2004&amp;journal=J.%20Exp.%20Bot.&amp;volume=55&amp;pages=909" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erh102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferh102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferh102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15020638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hernández</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Mancha</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Martínez-Rivas</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Molecular cloning and characterization of genes encoding two microsomal oleate desaturases (FAD2) from olive</div>, <div class="source">Phytochemistry</div> , <div class="year">2005</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">1417</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20characterization%20of%20genes%20encoding%20two%20microsomal%20oleate%20desaturases%20(FAD2)%20from%20olive&amp;author=M.L.%20Hern%C3%A1ndez&amp;author=M.%20Mancha&amp;author=J.M.%20Mart%C3%ADnez-Rivas&amp;publication_year=2005&amp;journal=Phytochemistry&amp;volume=66&amp;pages=1417-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.phytochem.2005.04.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.phytochem.2005.04.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.phytochem.2005.04.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15896817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simms</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Cizdziel</div> <div class="given-names">P.E.</div></div>,  <div class="name"><div class="surname">Chomczynski</div> <div class="given-names">P.</div></div>. </div><div class="article-title">TRIzol: A new reagent for optimal single-step isolation of RNA</div>, <div class="source">Focus</div> , <div class="year">1993</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">99</div>-<div class="lpage">102</div>)</div></div></div></div><div content-id="DSS036C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Urbańczyk-Wochniak</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Filipecki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Przybecki</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">A useful protocol for in situ RT-PCR on plant tissues</div>, <div class="source">Cell. Mol. Biol. Lett.</div> , <div class="year">2002</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">7</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20useful%20protocol%20for%20in%20situ%20RT-PCR%20on%20plant%20tissues&amp;author=E.%20Urba%C5%84czyk-Wochniak&amp;author=M.%20Filipecki&amp;author=Z.%20Przybecki&amp;publication_year=2002&amp;journal=Cell.%20Mol.%20Biol.%20Lett.&amp;volume=7&amp;pages=7-18" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11944046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Machleder</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Chenchik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Siebert</div> <div class="given-names">P.D.</div></div>. </div><div class="article-title">Reverse transcriptase template switching: a SMART approach for full-length cDNA library construction</div>, <div class="source">Biotechniques</div> , <div class="year">2001</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">892</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reverse%20transcriptase%20template%20switching%3A%20a%20SMART%20approach%20for%20full-length%20cDNA%20library%20construction&amp;author=Y.Y.%20Zhu&amp;author=E.M.%20Machleder&amp;author=A.%20Chenchik&amp;author=R.%20Li&amp;author=P.D.%20Siebert&amp;publication_year=2001&amp;journal=Biotechniques&amp;volume=30&amp;pages=892-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11314272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhulidov</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Bogdanova</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Shcheglov</div> <div class="given-names">A.S.</div></div>, et al. </div><div class="article-title">Simple cDNA normalization using kamchatka crab duplex specific nuclease</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> pg. <div class="fpage">e37</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20cDNA%20normalization%20using%20kamchatka%20crab%20duplex%20specific%20nuclease&amp;author=P.A.%20Zhulidov&amp;author=E.A.%20Bogdanova&amp;author=A.S.%20Shcheglov&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=e37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gnh031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgnh031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgnh031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14973331" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shagin</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Rebrikov</div> <div class="given-names">D.V.</div></div>,  <div class="name"><div class="surname">Kozhemyako</div> <div class="given-names">V.B.</div></div>, et al. </div><div class="article-title">A novel method for SNP detection using a new duplex-specific nuclease from crab hepatopancreas</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1935</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20method%20for%20SNP%20detection%20using%20a%20new%20duplex-specific%20nuclease%20from%20crab%20hepatopancreas&amp;author=D.A.%20Shagin&amp;author=D.V.%20Rebrikov&amp;author=V.B.%20Kozhemyako&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1935-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.547002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.547002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.547002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">Y.F.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">B.C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Construction of cDNA library of cotton mutant (Xiangmian-18) library during gland forming stage</div>, <div class="source">Colloids Surf. B Biointerfaces</div> , <div class="year">2007</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">258</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20cDNA%20library%20of%20cotton%20mutant%20(Xiangmian-18)%20library%20during%20gland%20forming%20stage&amp;author=Y.F.%20Xie&amp;author=B.C.%20Wang&amp;author=B.%20Li&amp;publication_year=2007&amp;journal=Colloids%20Surf.%20B%20Biointerfaces&amp;volume=60&amp;pages=258-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.colsurfb.2007.06.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.colsurfb.2007.06.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.colsurfb.2007.06.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17689935" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raymond</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Janech</div> <div class="given-names">M.G.</div></div>. </div><div class="article-title">Ice-binding proteins from enoki and shiitake mushrooms</div>, <div class="source">Cryobiology</div> , <div class="year">2009</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">151</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ice-binding%20proteins%20from%20enoki%20and%20shiitake%20mushrooms&amp;author=J.A.%20Raymond&amp;author=M.G.%20Janech&amp;publication_year=2009&amp;journal=Cryobiology&amp;volume=58&amp;pages=151-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cryobiol.2008.11.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cryobiol.2008.11.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cryobiol.2008.11.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vera</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wheat</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fescemyer</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Rapid transcriptome characterization for a nonmodel organism using 454 pyrosequencing</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2008</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1636</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20transcriptome%20characterization%20for%20a%20nonmodel%20organism%20using%20454%20pyrosequencing&amp;author=J.%20Vera&amp;author=C.%20Wheat&amp;author=H.%20Fescemyer&amp;publication_year=2008&amp;journal=Mol.%20Ecol.&amp;volume=17&amp;pages=1636-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2008.17.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2008.17.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2008.17.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18266620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Margulies</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Egholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Altman</div> <div class="given-names">W.E.</div></div>, et al. </div><div class="article-title">Genome sequencing in microfabricated high-density picolitre reactors</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">376</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;author=M.%20Egholm&amp;author=W.E.%20Altman&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oliveros</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">VENNY. An interactive tool for comparing lists with Venn Diagrams</div>, <div class="year">2007</div> <div class="comment"><a class="link link-uri" href="http://bioinfogp.cnb.csic.es/tools/venny/index.html">http://bioinfogp.cnb.csic.es/tools/venny/index.html</a>.</div></div></div></div></div><div content-id="DSS036C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Franssen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shrestha</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Brautigam</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bornberg-Bauer</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Weber</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Comprehensive transcriptome analysis of the highly complex <em>Pisum sativum</em> genome using next generation sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">227</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20transcriptome%20analysis%20of%20the%20highly%20complex%20Pisum%20sativum%20genome%20using%20next%20generation%20sequencing&amp;author=S.%20Franssen&amp;author=R.%20Shrestha&amp;author=A.%20Brautigam&amp;author=E.%20Bornberg-Bauer&amp;author=A.%20Weber&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=227" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21569327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sander</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schneider</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Database of homology-derived protein structures and the structural meaning of sequence alignment</div>, <div class="source">Proteins</div> , <div class="year">1991</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">56</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20of%20homology-derived%20protein%20structures%20and%20the%20structural%20meaning%20of%20sequence%20alignment&amp;author=C.%20Sander&amp;author=R.%20Schneider&amp;publication_year=1991&amp;journal=Proteins&amp;volume=9&amp;pages=56-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0134" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0134" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0134"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2017436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rost</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Twilight zone of protein sequence alignments</div>, <div class="source">Protein Eng.</div> , <div class="year">1999</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">85</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Twilight%20zone%20of%20protein%20sequence%20alignments&amp;author=B.%20Rost&amp;publication_year=1999&amp;journal=Protein%20Eng.&amp;volume=12&amp;pages=85-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/protein/12.2.85" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fprotein%2F12.2.85" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fprotein%2F12.2.85"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10195279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Talon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20Gotz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bremer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Doerge</div> <div class="given-names">R.W.</div></div>. </div>, <div class="source">Statistics at the Bench: A Step-By-Step Handbook for Biologists</div> , <div class="year">2010</div><div class="publisher-name">Cold Spring Harbor Laboratory Press: Cold Spring Harbor, New York</div></div></div></div></div><div content-id="DSS036C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</div>, <div class="source">J. Roy. Stat. Soc. B Met.</div> , <div class="year">1995</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">289</div>-<div class="lpage">300</div>)</div></div></div></div><div content-id="DSS036C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tous</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Del Río</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Caballero</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Rallo</div> <div class="given-names">L.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Rallo</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Barranco</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Caballero</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">Libro II. Variabilidad y Selección</div>, <div class="source">Variedades de olivo en España, Junta de Andalucía</div> , <div class="year">2005</div><div class="publisher-loc">Madrid</div><div class="publisher-name">MAPA y Ediciones Mundi-Prensa</div></div></div></div></div><div content-id="DSS036C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fortanier</div> <div class="given-names">E.J.</div></div>,  <div class="name"><div class="surname">Jonkers</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Juvenility and maturity of plants as influenced by their ontogenetical and physiological ageing</div>, <div class="source">Acta Hort. (ISHS)</div> , <div class="year">1976</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">37</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Juvenility%20and%20maturity%20of%20plants%20as%20influenced%20by%20their%20ontogenetical%20and%20physiological%20ageing&amp;author=E.J.%20Fortanier&amp;author=H.%20Jonkers&amp;publication_year=1976&amp;journal=Acta%20Hort.%20(ISHS)&amp;volume=56&amp;pages=37-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.17660/ActaHortic.1976.56.2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.17660%2FActaHortic.1976.56.2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.17660%2FActaHortic.1976.56.2"> </span></div> </div></div></div></div></div><div content-id="DSS036C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Greenwood</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">Juvenility and maturation in conifers: current concepts</div>, <div class="source">Tree Physiol.</div> , <div class="year">1995</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">433</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Juvenility%20and%20maturation%20in%20conifers%3A%20current%20concepts&amp;author=M.S.%20Greenwood&amp;publication_year=1995&amp;journal=Tree%20Physiol.&amp;volume=15&amp;pages=433-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/treephys/15.7-8.433" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ftreephys%2F15.7-8.433" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ftreephys%2F15.7-8.433"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14965924" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Poethig</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Phase change and the regulation of shoot morphogenesis in plants</div>, <div class="source">Science</div> , <div class="year">1990</div>, vol. <div class="volume">250</div> (pg. <div class="fpage">923</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phase%20change%20and%20the%20regulation%20of%20shoot%20morphogenesis%20in%20plants&amp;author=R.%20Poethig&amp;publication_year=1990&amp;journal=Science&amp;volume=250&amp;pages=923-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.250.4983.923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.250.4983.923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.250.4983.923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17746915" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van der Hoeven</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ronning</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Giovannoni</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Deductions about the number, organization, and evolution of genes in the tomato genome based on analysis of a large expressed sequence tag collection and selective genomic sequencing</div>, <div class="source">Plant Cell</div> , <div class="year">2002</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1441</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deductions%20about%20the%20number%2C%20organization%2C%20and%20evolution%20of%20genes%20in%20the%20tomato%20genome%20based%20on%20analysis%20of%20a%20large%20expressed%20sequence%20tag%20collection%20and%20selective%20genomic%20sequencing&amp;author=R.%20Van%20der%20Hoeven&amp;author=C.%20Ronning&amp;author=J.%20Giovannoni&amp;author=G.%20Martin&amp;author=S.%20Tanksley&amp;publication_year=2002&amp;journal=Plant%20Cell&amp;volume=14&amp;pages=1441-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.010478" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.010478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.010478"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12119366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS036C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCourt</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Chapman</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Buchheim</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mishler</div> <div class="given-names">B.D.</div></div>. </div><div class="article-title">Green plants</div>, <div class="year">1996</div> <div class="comment"><a class="link link-uri" href="http://tolweb.org/Green_plants/2382/1996.01.01">http://tolweb.org/Green_plants/2382/1996.01.01</a> in The Tree of Life Web Project <a class="link link-uri" href="http://tolweb.org/">http://tolweb.org/</a></div></div></div></div></div><div content-id="DSS036C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="article-title">Uniprot taxonomy</div> <div class="comment"><a class="link link-uri" href="http://www.uniprot.org/taxonomy/">http://www.uniprot.org/taxonomy/</a></div></div></div></div></div><div content-id="DSS036C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS036C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Belaj</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dominguez-García</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Atienza</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Developing a core collection of olive (<em>Olea europaea</em> L.) based on molecular markers (DArTs, SSRs, SNPs) and agronomic traits</div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">365</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developing%20a%20core%20collection%20of%20olive%20(Olea%20europaea%20L.)%20based%20on%20molecular%20markers%20(DArTs%2C%20SSRs%2C%20SNPs)%20and%20agronomic%20traits&amp;author=A.%20Belaj&amp;author=M.%20Dominguez-Garc%C3%ADa&amp;author=S.%20Atienza&amp;publication_year=2012&amp;journal=Tree%20Genet.%20Genomes&amp;volume=8&amp;pages=365-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-011-0447-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-011-0447-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-011-0447-6"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=384291&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss036/2/dss036-dss036supp.pdf?Expires=2147483647&amp;Signature=vwU3J4KWoSod0Pigm4~h0stP0S728SttVzYWnD2IwEhC23LYcsx-~~sAm5JktGOCuK36KRmNvPCmpIFRa8yiqaCobK3z69smL1GGCtMZcgMKXqosxSwg4QuDlg9g-To8B2R~M4454laS3ozvosb83fSTrWDmGi4SJvHUHt~dARtRD839v-1zjTlLxHLqnrklDKRWVW4-zsr37aKUct1JWQE9r219EUWxl6G2tQFRA-TI4haRH9tuDhORwPN7I8qji5MjUDUAlkXEbjfd4NbXxQaAUCRkSg2E0PRZVmtvsVus0A4dxi5ITjDVKqFqA1PRSibETIntybxqRuD29SKOYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">446</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">41</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss036"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="384291"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f1%2f93%2f384291">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000315666200008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (41)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F1%2F93%2F384291" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/384291'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="De Novo Assembly and Functional Annotation of the Olive (Olea europaea) Transcriptome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss036");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f93%2f384291">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

