

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":828,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/1/3/486680","@type":"ScholarlyArticle","name":"Complexity of the Genomic Diversity in Enterohemorrhagic  Escherichia coli  O157 Revealed by the Combinational Use of the O157 Sakai OligoDNA Microarray and the Whole Genome PCR scanning","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi026","keywords":["<italic>E. coli</italic> O157","genomic diversity","microarray","whole genome PCR scanning"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/1/3/486680","author":[{"name":"Ogura, Yoshitoshi","affiliation":"1  Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki  Miyazaki, Japan    2  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki  Miyazaki, Japan","@type":"Person"},{"name":"Kurokawa, Ken","affiliation":"3  Laboratory of Comparative Genomics, Graduate School of Information Science, Nara Institute of Science and Technology  Nara, Japan","@type":"Person"},{"name":"Ooka, Tadasuke","affiliation":"2  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki  Miyazaki, Japan","@type":"Person"},{"name":"Tashiro, Kousuke","affiliation":"4  Laboratory of Molecular Gene Technics, Department of Genetic Resources Technology, Faculty of Agriculture, Kyushu University  Fukuoka, Japan","@type":"Person"},{"name":"Tobe, Toru","affiliation":"5  Division of Applied Bacteriology, Graduate School of Medicine, Osaka University  Osaka, Japan","@type":"Person"},{"name":"Ohnishi, Makoto","affiliation":"6  Department of Bacteriology, National Institute for Infectious Diseases  Tokyo, Japan","@type":"Person"},{"name":"Nakayama, Keisuke","affiliation":"2  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki  Miyazaki, Japan","@type":"Person"},{"name":"Morimoto, Takuya","affiliation":"1  Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki  Miyazaki, Japan","@type":"Person"},{"name":"Terajima, Jun","affiliation":"6  Department of Bacteriology, National Institute for Infectious Diseases  Tokyo, Japan","@type":"Person"},{"name":"Watanabe, Haruo","affiliation":"6  Department of Bacteriology, National Institute for Infectious Diseases  Tokyo, Japan","@type":"Person"},{"name":"Kuhara, Satoru","affiliation":"4  Laboratory of Molecular Gene Technics, Department of Genetic Resources Technology, Faculty of Agriculture, Kyushu University  Fukuoka, Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"1  Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki  Miyazaki, Japan    2  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki  Miyazaki, Japan","@type":"Person"}],"description":"Abstract.  Escherichia coli O157, an etiological agent of hemorrhagic colitis and hemolytic uremic syndrome, is one of the leading worldwide public health threa","pageStart":"3","pageEnd":"14","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/1/0/m_dnaresearch13_1.cover.gif?Expires=2147483647&Signature=3oqtDw-kLRjSEm4EY64kE0hilMeDCpbVJFwgtb8mTtUDglvHBVQaY830YGZNmb1qrEFH~O6CtaLWlJlx9IkpJCU5tO0ipfevd~K63~l3nIVOLsJGNlRWLVVALxmZpM7jifTyk3mS8HjlLeYcyrmK7-2l0xuICIuGG2ysZijttJeAjFS-W0P13ZzPfM-JSxX78MIbsukCLaoCwNU6iOmwNTXp80TXjLlXYBJ9w6qv2~PrDu6Jyds3GVIt~x2AihwLsjtQZsr-mXf8FcjIZWItTx-0BRBHtUxlj94m8g~yHh2xXWxLXxMf6hjV-5N0NJAtguL2iunnsy2rYudaUR1WRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Complexity of the Genomic Diversity in Enterohemorrhagic  Escherichia coli  O157 Revealed by the Combinational Use of the O157 Sakai OligoDNA Microarray and the Whole Genome PCR scanning","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/1/0/m_dnaresearch13_1.cover.gif?Expires=2147483647&Signature=3oqtDw-kLRjSEm4EY64kE0hilMeDCpbVJFwgtb8mTtUDglvHBVQaY830YGZNmb1qrEFH~O6CtaLWlJlx9IkpJCU5tO0ipfevd~K63~l3nIVOLsJGNlRWLVVALxmZpM7jifTyk3mS8HjlLeYcyrmK7-2l0xuICIuGG2ysZijttJeAjFS-W0P13ZzPfM-JSxX78MIbsukCLaoCwNU6iOmwNTXp80TXjLlXYBJ9w6qv2~PrDu6Jyds3GVIt~x2AihwLsjtQZsr-mXf8FcjIZWItTx-0BRBHtUxlj94m8g~yHh2xXWxLXxMf6hjV-5N0NJAtguL2iunnsy2rYudaUR1WRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Complexity of the Genomic Diversity in Enterohemorrhagic  Escherichia coli  O157 Revealed by the Combinational Use of the O157 Sakai OligoDNA Microarray and the Whole Genome PCR scanning" />
    <meta property="og:description" content="Abstract.  Escherichia coli O157, an etiological agent of hemorrhagic colitis and hemolytic uremic syndrome, is one of the leading worldwide public health threa" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/1/3/486680" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ogura, Yoshitoshi" /><meta name="citation_author_institution" content="1Division of Bioenvironmental Science, Frontier Science Research Center, University of MiyazakiMiyazaki, Japan" /><meta name="citation_author_institution" content="2Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of MiyazakiMiyazaki, Japan" /><meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="3Laboratory of Comparative Genomics, Graduate School of Information Science, Nara Institute of Science and TechnologyNara, Japan" /><meta name="citation_author" content="Ooka, Tadasuke" /><meta name="citation_author_institution" content="2Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of MiyazakiMiyazaki, Japan" /><meta name="citation_author" content="Tashiro, Kousuke" /><meta name="citation_author_institution" content="4Laboratory of Molecular Gene Technics, Department of Genetic Resources Technology, Faculty of Agriculture, Kyushu UniversityFukuoka, Japan" /><meta name="citation_author" content="Tobe, Toru" /><meta name="citation_author_institution" content="5Division of Applied Bacteriology, Graduate School of Medicine, Osaka UniversityOsaka, Japan" /><meta name="citation_author" content="Ohnishi, Makoto" /><meta name="citation_author_institution" content="6Department of Bacteriology, National Institute for Infectious DiseasesTokyo, Japan" /><meta name="citation_author" content="Nakayama, Keisuke" /><meta name="citation_author_institution" content="2Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of MiyazakiMiyazaki, Japan" /><meta name="citation_author" content="Morimoto, Takuya" /><meta name="citation_author_institution" content="1Division of Bioenvironmental Science, Frontier Science Research Center, University of MiyazakiMiyazaki, Japan" /><meta name="citation_author" content="Terajima, Jun" /><meta name="citation_author_institution" content="6Department of Bacteriology, National Institute for Infectious DiseasesTokyo, Japan" /><meta name="citation_author" content="Watanabe, Haruo" /><meta name="citation_author_institution" content="6Department of Bacteriology, National Institute for Infectious DiseasesTokyo, Japan" /><meta name="citation_author" content="Kuhara, Satoru" /><meta name="citation_author_institution" content="4Laboratory of Molecular Gene Technics, Department of Genetic Resources Technology, Faculty of Agriculture, Kyushu UniversityFukuoka, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="1Division of Bioenvironmental Science, Frontier Science Research Center, University of MiyazakiMiyazaki, Japan" /><meta name="citation_author_institution" content="2Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of MiyazakiMiyazaki, Japan" /><meta name="citation_title" content="Complexity of the Genomic Diversity in Enterohemorrhagic Escherichia coli O157 Revealed by the Combinational Use of the O157 Sakai OligoDNA Microarray and the Whole Genome PCR scanning" /><meta name="citation_firstpage" content="3" /><meta name="citation_lastpage" content="14" /><meta name="citation_doi" content="10.1093/dnares/dsi026" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="Johnson J. R. Donnenberg M. S. Evolution of pathogenic Escherichia coli 2002 San Diego, Calif Academic Press 55 77 Escherichia coli : Virulence Mechanisms of a Versatile Pathogen " /><meta name="citation_reference" content="citation_title=Pathogenic Escherichia coli; Nat. Rev. Microbiol.;  citation_year=2004;  citation_volume=2;  citation_pages=123-140; " /><meta name="citation_reference" content="citation_title=Escherichia coli O157:H7; Lancet;  citation_year=1998;  citation_volume=352;  citation_pages=1207-1212; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12; Science;  citation_year=1997;  citation_volume=277;  citation_pages=1453-1474; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of enterohemorrhagic Escherichia coli O157:H7 and genomic comparison with a laboratory strain K-12; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=11-22; " /><meta name="citation_reference" content="citation_title=Diversification of Escherichia coli genomes: are bacteriophages the major contributors?; Trends Microbiol.;  citation_year=2001;  citation_volume=9;  citation_pages=481-485; " /><meta name="citation_reference" content="citation_title=Genome sequence of enterohaemorrhagic Escherichia coli O157:H7; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=529-533; " /><meta name="citation_reference" content="citation_title=Genomic diversity of enterohemorrhagic Escherichia coli O157 revealed by whole genome PCR scanning; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=17043-17048; " /><meta name="citation_reference" content="citation_title=Improved analytical methods for microarray-based genome-composition analysis; citation_text=RESEARCH0065; Genome Biol.;  citation_year=2002;  citation_volume=3; " /><meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=14863-14868; " /><meta name="citation_reference" content="citation_title=Complete nucleotide sequences of 93-kb and 3.3-kb plasmids of an enterohemorrhagic Escherichia coli O157:H7 derived from Sakai outbreak; DNA Res.;  citation_year=1998;  citation_volume=5;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Homologous pairing proteins encoded by the Escherichia coli recE and recT genes; Mol. Microbiol.;  citation_year=1994;  citation_volume=11;  citation_pages=23-30; " /><meta name="citation_reference" content="citation_title=Cloning, expression, and DNA sequence analysis of genes encoding nontypeable Haemophilus influenzae high-molecular-weight surface-exposed proteins related to filamentous hemagglutinin of Bordetella pertussis; Infect. Immun.;  citation_year=1992;  citation_volume=60;  citation_pages=1302-1313; " /><meta name="citation_reference" content="citation_title=Role of TraT protein, an anticomplementary protein produced in Escherichia coli by R100 factor, in serum resistance; J. Immunol.;  citation_year=1992;  citation_volume=148;  citation_pages=827-836; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Neisseria meningitidis serogroup B strain MC58; Science;  citation_year=2000;  citation_volume=287;  citation_pages=1809-1815; " /><meta name="citation_reference" content="citation_title=Identification of a novel genetic locus that is required for in vitro adhesion of a clinical isolate of enterohaemorrhagic Escherichia coli to epithelial cells; Mol. Microbiol.;  citation_year=2000;  citation_volume=35;  citation_pages=275-288; " /><meta name="citation_reference" content="citation_title=A large toxin from pathogenic Escherichia coli strains that inhibits lymphocyte activation; Infect Immun.;  citation_year=2000;  citation_volume=68;  citation_pages=2148-2155; " /><meta name="citation_reference" content="citation_title=Mutation of toxB and a truncated version of the efa-1 gene in Escherichia coli O157:H7 influences the expression and secretion of locus of enterocyte effacement-encoded proteins but not intestinal colonization in calves or sheep; Infect. Immun.;  citation_year=2004;  citation_volume=72;  citation_pages=5402-5411; " /><meta name="citation_reference" content="citation_title=Positive effects of multiple pch genes on expression of the locus of enterocyte effacement genes and adherence of enterohaemorrhagic Escherichia coli O157: H7 to HEp-2 cells; Microbiology;  citation_year=2004;  citation_volume=150;  citation_pages=2357-2571; " /><meta name="citation_reference" content="citation_title=EspFU is a translocated EHEC effector that interacts with Tir and N-WASP and promotes Nck-independent actin assembly; Dev. Cell;  citation_year=2004;  citation_volume=7;  citation_pages=217-228; " /><meta name="citation_reference" content="citation_title=EspJ is a prophage-carried type III effector protein of attaching and effacing pathogens that modulates infection dynamics; Infect. Immun.;  citation_year=2005;  citation_volume=73;  citation_pages=679-686; " /><meta name="citation_reference" content="citation_title=Dissecting virulence: systematic and functional analyses of a pathogenicity island; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=3597-3602; " /><meta name="citation_reference" content="citation_title=TccP is an enterohaemorrhagic Escherichia coli O157:H7 type III effector protein that couples Tir to the actin-cytoskeleton; Cell Microbiol.;  citation_year=2004;  citation_volume=6;  citation_pages=1167-1183; " /><meta name="citation_reference" content="citation_title=Identification and characterization of NleA, a non-LEE-encoded type III translocated virulence factor of enterohaemorrhagic Escherichia coli O157:H7; Mol. Microbiol.;  citation_year=2004;  citation_volume=51;  citation_pages=1233-1249; " /><meta name="citation_reference" content="citation_title=Identification of a novel Citrobacter rodentium type III secreted protein, EspI, and roles of this and other secreted proteins in infection; Infect. Immun.;  citation_year=2004;  citation_volume=72;  citation_pages=2288-2302; " /><meta name="citation_reference" content="citation_title=Evolution of genomic content in the stepwise emergence of Escherichia coli O157:H7; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=1783-1791; " /><meta name="citation_reference" content="citation_title=Genotypic and phenotypic changes in the emergence of Escherichia coli O157:H7; J. Infect. Dis.;  citation_year=1998;  citation_volume=177;  citation_pages=1750-1753; " /><meta name="citation_reference" content="citation_title=Escherichia coli O157:H7 Shiga toxin-encoding bacteriophages: integrations, excisions, truncations, and evolutionary implications; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=3596-3605; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/1/3/17462364/dsi026.pdf" /><meta name="description" content="Abstract.  Escherichia coli O157, an etiological agent of hemorrhagic colitis and hemolytic uremic syndrome, is one of the leading worldwide public health threa" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/1/3/486680" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/1/3/486680" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/1/3/486680");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi026';
    </script>
<title>
	Complexity of the Genomic Diversity in Enterohemorrhagic Escherichia coli O157 Revealed by the Combinational Use of the O157 Sakai OligoDNA Microarray and the Whole Genome PCR scanning | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./486680" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Ba5XPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPZ3VyYSwgWW9zaGl0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFEaXZpc2lvbiBvZiBCaW9lbnZpcm9ubWVudGFsIFNjaWVuY2UsIEZyb250aWVyIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpTWl5YXpha2ksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBGYWN1bHR5IG9mIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpTWl5YXpha2ksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3Vyb2thd2EsIEtlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjNMYWJvcmF0b3J5IG9mIENvbXBhcmF0aXZlIEdlbm9taWNzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neU5hcmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT29rYSwgVGFkYXN1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIyRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5LCBEZXBhcnRtZW50IG9mIEluZmVjdGlvdXMgRGlzZWFzZXMsIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2lNaXlhemFraSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYXNoaXJvLCBLb3VzdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iNExhYm9yYXRvcnkgb2YgTW9sZWN1bGFyIEdlbmUgVGVjaG5pY3MsIERlcGFydG1lbnQgb2YgR2VuZXRpYyBSZXNvdXJjZXMgVGVjaG5vbG9neSwgRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgS3l1c2h1IFVuaXZlcnNpdHlGdWt1b2thLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRvYmUsIFRvcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSI1RGl2aXNpb24gb2YgQXBwbGllZCBCYWN0ZXJpb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpY2luZSwgT3Nha2EgVW5pdmVyc2l0eU9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9obmlzaGksIE1ha290byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjZEZXBhcnRtZW50IG9mIEJhY3RlcmlvbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIGZvciBJbmZlY3Rpb3VzIERpc2Vhc2VzVG9reW8sIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYXlhbWEsIEtlaXN1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIyRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5LCBEZXBhcnRtZW50IG9mIEluZmVjdGlvdXMgRGlzZWFzZXMsIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2lNaXlhemFraSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3JpbW90bywgVGFrdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURpdmlzaW9uIG9mIEJpb2Vudmlyb25tZW50YWwgU2NpZW5jZSwgRnJvbnRpZXIgU2NpZW5jZSBSZXNlYXJjaCBDZW50ZXIsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2lNaXlhemFraSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUZXJhamltYSwgSnVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iNkRlcGFydG1lbnQgb2YgQmFjdGVyaW9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgZm9yIEluZmVjdGlvdXMgRGlzZWFzZXNUb2t5bywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYXRhbmFiZSwgSGFydW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSI2RGVwYXJ0bWVudCBvZiBCYWN0ZXJpb2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBmb3IgSW5mZWN0aW91cyBEaXNlYXNlc1Rva3lvLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1aGFyYSwgU2F0b3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iNExhYm9yYXRvcnkgb2YgTW9sZWN1bGFyIEdlbmUgVGVjaG5pY3MsIERlcGFydG1lbnQgb2YgR2VuZXRpYyBSZXNvdXJjZXMgVGVjaG5vbG9neSwgRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgS3l1c2h1IFVuaXZlcnNpdHlGdWt1b2thLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhheWFzaGksIFRldHN1eWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxRGl2aXNpb24gb2YgQmlvZW52aXJvbm1lbnRhbCBTY2llbmNlLCBGcm9udGllciBTY2llbmNlIFJlc2VhcmNoIENlbnRlciwgVW5pdmVyc2l0eSBvZiBNaXlhemFraU1peWF6YWtpLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJEaXZpc2lvbiBvZiBNaWNyb2Jpb2xvZ3ksIERlcGFydG1lbnQgb2YgSW5mZWN0aW91cyBEaXNlYXNlcywgRmFjdWx0eSBvZiBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBNaXlhemFraU1peWF6YWtpLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29tcGxleGl0eSBvZiB0aGUgR2Vub21pYyBEaXZlcnNpdHkgaW4gRW50ZXJvaGVtb3JyaGFnaWMgRXNjaGVyaWNoaWEgY29saSBPMTU3IFJldmVhbGVkIGJ5IHRoZSBDb21iaW5hdGlvbmFsIFVzZSBvZiB0aGUgTzE1NyBTYWthaSBPbGlnb0ROQSBNaWNyb2FycmF5IGFuZCB0aGUgV2hvbGUgR2Vub21lIFBDUiBzY2FubmluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNpMDI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDYvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJKb2huc29uIEouIFIuIERvbm5lbmJlcmcgTS4gUy4gRXZvbHV0aW9uIG9mIHBhdGhvZ2VuaWMgRXNjaGVyaWNoaWEgY29saSAyMDAyIFNhbiBEaWVnbywgQ2FsaWYgQWNhZGVtaWMgUHJlc3MgNTUgNzcgRXNjaGVyaWNoaWEgY29saSA6IFZpcnVsZW5jZSBNZWNoYW5pc21zIG9mIGEgVmVyc2F0aWxlIFBhdGhvZ2VuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdGhvZ2VuaWMgRXNjaGVyaWNoaWEgY29saTsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEyMy0xNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzY2hlcmljaGlhIGNvbGkgTzE1NzpINzsgTGFuY2V0OyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTM1MjsgIGNpdGF0aW9uX3BhZ2VzPTEyMDctMTIxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBFc2NoZXJpY2hpYSBjb2xpIEstMTI7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9MTQ1My0xNDc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgZW50ZXJvaGVtb3JyaGFnaWMgRXNjaGVyaWNoaWEgY29saSBPMTU3Okg3IGFuZCBnZW5vbWljIGNvbXBhcmlzb24gd2l0aCBhIGxhYm9yYXRvcnkgc3RyYWluIEstMTI7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaWZpY2F0aW9uIG9mIEVzY2hlcmljaGlhIGNvbGkgZ2Vub21lczogYXJlIGJhY3RlcmlvcGhhZ2VzIHRoZSBtYWpvciBjb250cmlidXRvcnM/OyBUcmVuZHMgTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NDgxLTQ4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIGVudGVyb2hhZW1vcnJoYWdpYyBFc2NoZXJpY2hpYSBjb2xpIE8xNTc6SDc7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz01MjktNTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGRpdmVyc2l0eSBvZiBlbnRlcm9oZW1vcnJoYWdpYyBFc2NoZXJpY2hpYSBjb2xpIE8xNTcgcmV2ZWFsZWQgYnkgd2hvbGUgZ2Vub21lIFBDUiBzY2FubmluZzsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTE3MDQzLTE3MDQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlZCBhbmFseXRpY2FsIG1ldGhvZHMgZm9yIG1pY3JvYXJyYXktYmFzZWQgZ2Vub21lLWNvbXBvc2l0aW9uIGFuYWx5c2lzOyBjaXRhdGlvbl90ZXh0PVJFU0VBUkNIMDA2NTsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0ZXIgYW5hbHlzaXMgYW5kIGRpc3BsYXkgb2YgZ2Vub21lLXdpZGUgZXhwcmVzc2lvbiBwYXR0ZXJuczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTE0ODYzLTE0ODY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBudWNsZW90aWRlIHNlcXVlbmNlcyBvZiA5My1rYiBhbmQgMy4zLWtiIHBsYXNtaWRzIG9mIGFuIGVudGVyb2hlbW9ycmhhZ2ljIEVzY2hlcmljaGlhIGNvbGkgTzE1NzpINyBkZXJpdmVkIGZyb20gU2FrYWkgb3V0YnJlYWs7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvbW9sb2dvdXMgcGFpcmluZyBwcm90ZWlucyBlbmNvZGVkIGJ5IHRoZSBFc2NoZXJpY2hpYSBjb2xpIHJlY0UgYW5kIHJlY1QgZ2VuZXM7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTIzLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nLCBleHByZXNzaW9uLCBhbmQgRE5BIHNlcXVlbmNlIGFuYWx5c2lzIG9mIGdlbmVzIGVuY29kaW5nIG5vbnR5cGVhYmxlIEhhZW1vcGhpbHVzIGluZmx1ZW56YWUgaGlnaC1tb2xlY3VsYXItd2VpZ2h0IHN1cmZhY2UtZXhwb3NlZCBwcm90ZWlucyByZWxhdGVkIHRvIGZpbGFtZW50b3VzIGhlbWFnZ2x1dGluaW4gb2YgQm9yZGV0ZWxsYSBwZXJ0dXNzaXM7IEluZmVjdC4gSW1tdW4uOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MTMwMi0xMzEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIFRyYVQgcHJvdGVpbiwgYW4gYW50aWNvbXBsZW1lbnRhcnkgcHJvdGVpbiBwcm9kdWNlZCBpbiBFc2NoZXJpY2hpYSBjb2xpIGJ5IFIxMDAgZmFjdG9yLCBpbiBzZXJ1bSByZXNpc3RhbmNlOyBKLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDg7ICBjaXRhdGlvbl9wYWdlcz04MjctODM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgTmVpc3NlcmlhIG1lbmluZ2l0aWRpcyBzZXJvZ3JvdXAgQiBzdHJhaW4gTUM1ODsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODc7ICBjaXRhdGlvbl9wYWdlcz0xODA5LTE4MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGEgbm92ZWwgZ2VuZXRpYyBsb2N1cyB0aGF0IGlzIHJlcXVpcmVkIGZvciBpbiB2aXRybyBhZGhlc2lvbiBvZiBhIGNsaW5pY2FsIGlzb2xhdGUgb2YgZW50ZXJvaGFlbW9ycmhhZ2ljIEVzY2hlcmljaGlhIGNvbGkgdG8gZXBpdGhlbGlhbCBjZWxsczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9Mjc1LTI4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBsYXJnZSB0b3hpbiBmcm9tIHBhdGhvZ2VuaWMgRXNjaGVyaWNoaWEgY29saSBzdHJhaW5zIHRoYXQgaW5oaWJpdHMgbHltcGhvY3l0ZSBhY3RpdmF0aW9uOyBJbmZlY3QgSW1tdW4uOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9MjE0OC0yMTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbiBvZiB0b3hCIGFuZCBhIHRydW5jYXRlZCB2ZXJzaW9uIG9mIHRoZSBlZmEtMSBnZW5lIGluIEVzY2hlcmljaGlhIGNvbGkgTzE1NzpINyBpbmZsdWVuY2VzIHRoZSBleHByZXNzaW9uIGFuZCBzZWNyZXRpb24gb2YgbG9jdXMgb2YgZW50ZXJvY3l0ZSBlZmZhY2VtZW50LWVuY29kZWQgcHJvdGVpbnMgYnV0IG5vdCBpbnRlc3RpbmFsIGNvbG9uaXphdGlvbiBpbiBjYWx2ZXMgb3Igc2hlZXA7IEluZmVjdC4gSW1tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9NTQwMi01NDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3NpdGl2ZSBlZmZlY3RzIG9mIG11bHRpcGxlIHBjaCBnZW5lcyBvbiBleHByZXNzaW9uIG9mIHRoZSBsb2N1cyBvZiBlbnRlcm9jeXRlIGVmZmFjZW1lbnQgZ2VuZXMgYW5kIGFkaGVyZW5jZSBvZiBlbnRlcm9oYWVtb3JyaGFnaWMgRXNjaGVyaWNoaWEgY29saSBPMTU3OiBINyB0byBIRXAtMiBjZWxsczsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTIzNTctMjU3MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNwRlUgaXMgYSB0cmFuc2xvY2F0ZWQgRUhFQyBlZmZlY3RvciB0aGF0IGludGVyYWN0cyB3aXRoIFRpciBhbmQgTi1XQVNQIGFuZCBwcm9tb3RlcyBOY2staW5kZXBlbmRlbnQgYWN0aW4gYXNzZW1ibHk7IERldi4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjE3LTIyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNwSiBpcyBhIHByb3BoYWdlLWNhcnJpZWQgdHlwZSBJSUkgZWZmZWN0b3IgcHJvdGVpbiBvZiBhdHRhY2hpbmcgYW5kIGVmZmFjaW5nIHBhdGhvZ2VucyB0aGF0IG1vZHVsYXRlcyBpbmZlY3Rpb24gZHluYW1pY3M7IEluZmVjdC4gSW1tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTczOyAgY2l0YXRpb25fcGFnZXM9Njc5LTY4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzc2VjdGluZyB2aXJ1bGVuY2U6IHN5c3RlbWF0aWMgYW5kIGZ1bmN0aW9uYWwgYW5hbHlzZXMgb2YgYSBwYXRob2dlbmljaXR5IGlzbGFuZDsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0zNTk3LTM2MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRjY1AgaXMgYW4gZW50ZXJvaGFlbW9ycmhhZ2ljIEVzY2hlcmljaGlhIGNvbGkgTzE1NzpINyB0eXBlIElJSSBlZmZlY3RvciBwcm90ZWluIHRoYXQgY291cGxlcyBUaXIgdG8gdGhlIGFjdGluLWN5dG9za2VsZXRvbjsgQ2VsbCBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xMTY3LTExODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIE5sZUEsIGEgbm9uLUxFRS1lbmNvZGVkIHR5cGUgSUlJIHRyYW5zbG9jYXRlZCB2aXJ1bGVuY2UgZmFjdG9yIG9mIGVudGVyb2hhZW1vcnJoYWdpYyBFc2NoZXJpY2hpYSBjb2xpIE8xNTc6SDc7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTEyMzMtMTI0OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBub3ZlbCBDaXRyb2JhY3RlciByb2RlbnRpdW0gdHlwZSBJSUkgc2VjcmV0ZWQgcHJvdGVpbiwgRXNwSSwgYW5kIHJvbGVzIG9mIHRoaXMgYW5kIG90aGVyIHNlY3JldGVkIHByb3RlaW5zIGluIGluZmVjdGlvbjsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NzI7ICBjaXRhdGlvbl9wYWdlcz0yMjg4LTIzMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBnZW5vbWljIGNvbnRlbnQgaW4gdGhlIHN0ZXB3aXNlIGVtZXJnZW5jZSBvZiBFc2NoZXJpY2hpYSBjb2xpIE8xNTc6SDc7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg3OyAgY2l0YXRpb25fcGFnZXM9MTc4My0xNzkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwaWMgYW5kIHBoZW5vdHlwaWMgY2hhbmdlcyBpbiB0aGUgZW1lcmdlbmNlIG9mIEVzY2hlcmljaGlhIGNvbGkgTzE1NzpINzsgSi4gSW5mZWN0LiBEaXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE3NzsgIGNpdGF0aW9uX3BhZ2VzPTE3NTAtMTc1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNjaGVyaWNoaWEgY29saSBPMTU3Okg3IFNoaWdhIHRveGluLWVuY29kaW5nIGJhY3RlcmlvcGhhZ2VzOiBpbnRlZ3JhdGlvbnMsIGV4Y2lzaW9ucywgdHJ1bmNhdGlvbnMsIGFuZCBldm9sdXRpb25hcnkgaW1wbGljYXRpb25zOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE4NTsgIGNpdGF0aW9uX3BhZ2VzPTM1OTYtMzYwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzEzLzEvMy8xNzQ2MjM2NC9kc2kwMjYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEVzY2hlcmljaGlhIGNvbGkgTzE1NywgYW4gZXRpb2xvZ2ljYWwgYWdlbnQgb2YgaGVtb3JyaGFnaWMgY29saXRpcyBhbmQgaGVtb2x5dGljIHVyZW1pYyBzeW5kcm9tZSwgaXMgb25lIG9mIHRoZSBsZWFkaW5nIHdvcmxkd2lkZSBwdWJsaWMgaGVhbHRoIHRocmVhIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODY2ODAmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkeb84q0QFa59bEI3AfkBg0YJE/r4=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f1%2f3%2f486680" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f1%2f3%2f486680" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19173" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/1/0/m_dnaresearch13_1.cover.gif?Expires=2147483647&amp;Signature=3oqtDw-kLRjSEm4EY64kE0hilMeDCpbVJFwgtb8mTtUDglvHBVQaY830YGZNmb1qrEFH~O6CtaLWlJlx9IkpJCU5tO0ipfevd~K63~l3nIVOLsJGNlRWLVVALxmZpM7jifTyk3mS8HjlLeYcyrmK7-2l0xuICIuGG2ysZijttJeAjFS-W0P13ZzPfM-JSxX78MIbsukCLaoCwNU6iOmwNTXp80TXjLlXYBJ9w6qv2~PrDu6Jyds3GVIt~x2AihwLsjtQZsr-mXf8FcjIZWItTx-0BRBHtUxlj94m8g~yHh2xXWxLXxMf6hjV-5N0NJAtguL2iunnsy2rYudaUR1WRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/1">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87193498">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193498">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193500">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193500">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193504">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193504">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193516">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193516">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193545">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193545">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193554">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193554">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/13/1/1/486900">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/13/1/15/486747">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Complexity of the Genomic Diversity in Enterohemorrhagic <em>Escherichia coli</em> O157 Revealed by the Combinational Use of the O157 Sakai OligoDNA Microarray and the Whole Genome PCR scanning
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki</div><div class="addr-line">Miyazaki, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki</div><div class="addr-line">Miyazaki, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Laboratory of Comparative Genomics, Graduate School of Information Science, Nara Institute of Science and Technology</div><div class="addr-line">Nara, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadasuke Ooka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki</div><div class="addr-line">Miyazaki, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadasuke+Ooka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ooka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ooka T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tadasuke Ooka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kousuke Tashiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Laboratory of Molecular Gene Technics, Department of Genetic Resources Technology, Faculty of Agriculture, Kyushu University</div><div class="addr-line">Fukuoka, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kousuke+Tashiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tashiro K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tashiro K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kousuke Tashiro</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toru Tobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><sup>5</sup><div class="insititution">Division of Applied Bacteriology, Graduate School of Medicine, Osaka University</div><div class="addr-line">Osaka, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toru+Tobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tobe T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tobe T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Toru Tobe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makoto Ohnishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><sup>6</sup><div class="insititution">Department of Bacteriology, National Institute for Infectious Diseases</div><div class="addr-line">Tokyo, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makoto+Ohnishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohnishi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohnishi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Makoto Ohnishi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keisuke Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki</div><div class="addr-line">Miyazaki, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keisuke+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Keisuke Nakayama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takuya Morimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki</div><div class="addr-line">Miyazaki, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takuya+Morimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morimoto T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morimoto T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takuya Morimoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Terajima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><sup>6</sup><div class="insititution">Department of Bacteriology, National Institute for Infectious Diseases</div><div class="addr-line">Tokyo, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Terajima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Terajima J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Terajima J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jun Terajima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haruo Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><sup>6</sup><div class="insititution">Department of Bacteriology, National Institute for Infectious Diseases</div><div class="addr-line">Tokyo, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haruo+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Haruo Watanabe</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Kuhara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Laboratory of Molecular Gene Technics, Department of Genetic Resources Technology, Faculty of Agriculture, Kyushu University</div><div class="addr-line">Fukuoka, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoru+Kuhara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuhara S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuhara S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoru Kuhara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki</div><div class="addr-line">Miyazaki, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki</div><div class="addr-line">Miyazaki, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-985-85-0871, Fax. +81-985-85-6475, E-mail: <a href="mailto:thayash@med.miyazaki-u.ac.jp" target="_blank">thayash@med.miyazaki-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 1, 1 January 2006, Pages 3–14, <a href='https://doi.org/10.1093/dnares/dsi026'>https://doi.org/10.1093/dnares/dsi026</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 November 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">21 December 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="486680" href="/dnaresearch/article-pdf/13/1/3/17462364/dsi026.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yoshitoshi Ogura, Ken Kurokawa, Tadasuke Ooka, Kousuke Tashiro, Toru Tobe, Makoto Ohnishi, Keisuke Nakayama, Takuya Morimoto, Jun Terajima, Haruo Watanabe, Satoru Kuhara, Tetsuya Hayashi;  Complexity of the Genomic Diversity in Enterohemorrhagic <em>Escherichia coli</em> O157 Revealed by the Combinational Use of the O157 Sakai OligoDNA Microarray and the Whole Genome PCR scanning, <em>DNA Research</em>, Volume 13, Issue 1, 1 January 2006, Pages 3–14, <a href='https://doi.org/10.1093/dnares/dsi026'>https://doi.org/10.1093/dnares/dsi026</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/486680?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/486680?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/486680?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/486680?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/486680?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/486680?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/486680?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Complexity%20of%20the%20Genomic%20Diversity%20in%20Enterohemorrhagic%20%26lt%3Bem%26gt%3BEscherichia%20coli%26lt%3B%2Fem%26gt%3B%20O157%20Revealed%20by%20the%20Combinational%20Use%20of%20the%20O157%20Sakai%20OligoDNA%20Microarray%20and%20the%20Whole%20Genome%20PCR%20scanning&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=1&amp;author=Ogura%2C%20Yoshitoshi%3B%20Kurokawa%2C%20Ken&amp;startPage=3&amp;endPage=14&amp;contentId=10.1093%2Fdnares%2Fdsi026&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f1%2f3%2f486680" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87193498 id="87193498" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Escherichia coli</em> O157, an etiological agent of hemorrhagic colitis and hemolytic uremic syndrome, is one of the leading worldwide public health threats. Genome sequencing of two O157 strains have revealed that the chromosome is comprised of a 4.1 Mb backbone shared by K-12 and a total of 1.4 Mb O157-specific sequences. Most of the large O157-specific sequences are prophages and prophage-like elements, which have carried many virulence genes into the O157 genome. This suggests that bacteriophages have played the key roles in the emergence of O157. The Whole Genome PCR Scanning (WGPScanning) analysis of O157 strains, on the other hand, revealed a high level of genomic diversity in O157. Variation of prophages has also been suggested as a major factor generating such diversity. In this study, we analyzed the gene content of O157 strains, by an oligoDNA microarray, using the same set of strains as examined by the WGPScanning method. Although most of the strains were typical O157 : H7, they differed remarkably in gene composition, particularly in those on prophages, and we identified more than 400 ‘variably absent or present’ genes which included virulence-related genes. This confirms the role of prophages in generating the genomic diversity, and raises a possibility that some level of variation in potential virulence is present among O157 strains. Fine comparison of the two datasets obtained by microarray and WGPScanning provided much further details on the O157 genome diversity than illustrated by each method alone, indicating the usefulness of this combinational approach in the genomic comparison of closely related strains.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;E. coli O157&quot;"><em>E. coli</em> O157</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genomic diversity&quot;">genomic diversity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="microarray">microarray</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole genome PCR scanning&quot;">whole genome PCR scanning</a></div>                            <h2 scrollto-destination=87193500 id="87193500" class="section-title" >1. Introduction</h2>
<p><em>Escherichia coli</em> is a constituent of normal microflora in intestinal tracts, but certain types of <em>E. coli</em> strains are associated with diseases in human and animals.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> Among these pathogenic <em>E. coli</em> strains, enterohemorrhagic <em>E. coli</em> O157 not only causes large outbreaks of hemorrhagic colitis but also numerous small outbreaks and sporadic cases, and is regarded as one of the major worldwide public health concerns.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> The genome sequence determination of an O157 strain, RIMD 0509952 (referred to as O157 Sakai) and the genomic comparison with a nonpathogenic strain, K-12 MG1655,<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> have revealed that the O157 chromosome is comprised of a 4.1 Mb backbone common with K-12 and a total of 1.4 Mb strain-specific sequences. O157 Sakai-specific sequences (referred to as S-loops) are inserted at various sites on the chromosome backbone, and encode more than 1600 of O157 Sakai-specific genes. Of importance is that most of the large S-loops are prophages and prophage-like elements. We have identified 18 prophages (Sp1-18) and 6 prophage-like elements (SpLE1-6). They comprise about two thirds of the O157 Sakai-specific sequences and have carried many virulence-related genes including Shiga toxin (Stx) genes (<em>stx1</em> and <em>stx2</em>) into O157. This suggested that acquisition of these bacteriophages have played the key roles in the emergence of O157.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> Similar findings were also obtained by the genome sequencing analysis of another O157 strain, EDL933.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a></p><p>More recently, we analyzed, based on the genome sequence of O157 Sakai, the whole genome structures of eight O157 strains that displayed diverse <em>Xba</em>I-digestion patterns by a systematic PCR analysis called Whole Genome PCR Scanning (WGPScanning).<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> In this analysis, we amplified the whole genome of each strain using 560 pairs of PCR primers, and searched for genomic segments with any structural differences by comparing each PCR product with that from O157 Sakai. This analysis provided the first genome-wide view on the genomic diversity of O157, and revealed that there is an unexpectedly high degree of diversity among O157 strains. In particular, the variation in prophages and prophage-related elements were remarkable, implying that their variability is the major factor generating such structural diversity. However, additional information on the gene level is required.</p><p>In this study, we constructed an oligoDNA microarray according to the genome sequence of O157 Sakai. We first validated the accuracy of the comparative genomic hybridization (CGH) analysis using this oligoarray by a test experiment with K-12 and O157 Sakai, and then performed a gene composition analysis of O157 using the same set of strains examined by the WGPScanning. We further made a fine comparison of the microarray data with WGPScanning data to understand the genomic diversity of O157 more in detail.</p>                            <h2 scrollto-destination=87193504 id="87193504" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=87193505 id="87193505" class="section-title" >2.1. Bacterial strains, growth condition and DNA extraction</h3>
<p>Eight O157 strains examined in this study were 980938 (referred to as #2), 980706 (#3), 990281 (#4), 980551 (#5), 990570 (#6), 981456 (#7), 982243 (#8) and 981795 (#9). They were human isolates obtained in Japan in 1998 as described previously.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> All were negative for Sorbitol fermentation (SOR<sup>−</sup>) and β-glucuronidase activity (GUD<sup>−</sup>) whose activities were examined after 24 h incubation at 37°C on SIB II agar plates (Kyokuto Pharmaceutical Industrial, Tokyo, Japan) and on ES Colimark agar plate (Eiken Kizai, Tokyo, Japan), respectively. As for strain #2, however, a weak positive reaction for β-glucuronidase was detected after 40 h incubation. O157 Sakai (RIMD 0509952) was used as the reference in all hybridization experiments. This strain was isolated in a large outbreak in Sakai city, Japan, which occurred in 1996, and has been sequenced by our group.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> The strain is available at the American Type Culture Collection (ATCC BAA-460). <em>Escherichia coli</em> K-12 MG1655 was kindly provided by Dr H. Mori (Nara Institute of Science and Technology). Cells were grown to the stationary phase at 37°C in Luria–Bertani medium. Genomic DNA was purified using the Genomic-tip 100/G and Genomic DNA buffer set (Qiagen) according to the manufacturer's instruction.</p>                            <h3 scrollto-destination=87193507 id="87193507" class="section-title" >2.2. Design of the O157 Sakai microarray</h3>
<p>Oligonucleotide probes were prepared for all the protein-coding genes on the O157 Sakai genome (5447 genes in total). Principles for the probe design were 60 mer in length and two probes for each gene. However, O157 Sakai contains 542 repeated genes, most of which are derived from IS elements or 13 lambda-like prophages sharing various length of DNA segments with almost identical sequences. As for these repeated genes, we prepared a single probe for each repeated gene family. Such probes totaled to 151 probes, each having two or more targets of ≥90% sequence identity on the O157 Sakai genome (referred to as ‘multiple hit probes’). As for the singleton genes (4905 genes), we were able to design two different 60 mer probes for 4173 genes, but only one 60 mer probe for 725 genes and one 44 or 45 mer probes for 7 genes. In total, 9620 probes were synthesized (Sigma-Aldrich Japan, Tokyo, Japan), and each probe was spotted onto poly-<span class="small-caps">l</span>-Lysine-coated glass slides (SD10011; Matsunami Glass Ind., Osaka, Japan) using SPBIO (Hitachi Software Engineering, Tokyo, Japan). As the negative controls, we included 20 oligonucleotides prepared from 10 yeast genes. Probe sequences are shown in Supplementary Table 1 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>.</p>                            <h3 scrollto-destination=87193509 id="87193509" class="section-title" >2.3. DNA labeling and hybridization</h3>
<p>Genomic DNAs from test strains and the reference (O157 Sakai) were fluorescently labeled with Cy3 and Cy5, respectively, according to the following protocol. First, 3 µg of genomic DNA was labeled with aminoallyl-modified dUTP (Sigma) using the Bioprime DNA Labeling System (Invitrogen). DNAs were not sheared or digested by restriction enzymes prior to the labeling. The aminoallyl-labeled DNA was purified by Microcon YM-30 (Millipore), dried in a speed-vac and resuspended in 10 µl of 50 mM NaHCO<sub>3</sub>. After adding 10 µl of dimethyl sulfoxide solution containing Cy3 or Cy5 monofunctional reactive dye (Amersham), the sample was incubated at room temperature in the dark for 1 h to allow the dye to couple with DNA. The fluorescently labeled DNA was finally purified by the Qiaquick PCR purification kit (Qiagen) into 30 µl elution buffer provided by the manufacturer.</p><p>The Cy5-labeled and the Cy3-labeled DNA preparations (15 µl for each) were mixed with a 90 µl hybridization solution containing 6.4× SSC, 0.64% SDS and 1.3 mg/ml yeast tRNA. After incubation at 96°C for 2 min, the denatured sample was applied to a microarray slide and incubated on an ArrayBooster hybridization apparatus (Advalytix AG, Brunnthal, Germany) at 50°C for 16 h. The slide was then washed twice in 2× SSC/0.2% SDS at room temperature for 10 min, twice in 0.2× SSC/0.2% SDS at 50°C for 5 min and twice in 0.2× SSC at room temperature for 10 min. Finally, the slide was briefly rinsed with ethanol, dried by centrifugation and scanned with a FLA-8000 scanner (Fuji Photofilm, Tokyo, Japan). The obtained data were analyzed by the ArrayVision 8.0 software (Imaging Research). Each strain was examined twice with the labeled DNAs prepared independently for each hybridyzation.</p>                            <h3 scrollto-destination=87193512 id="87193512" class="section-title" >2.4. Data analysis</h3>
<p>In our experimental condition, signal intensities from the negative controls were almost the same as that of the local background (LBG) in both cannels. Thus, spots with reference signal intensities lower than the LBG plus 5 SD or with some spotting abnormalities were categorized as ‘low quality’ (LQ). Signal intensities of other spots were corrected by subtracting the LBG, and were subjected to the ‘presence (P) or absence (A)’ determination using the array-based genotyping software GACK capable of dynamically generating cutoffs for present/absent (conserved/divergent) gene analysis in each array hybridization based on the signal-ratio distribution of un-normalized data sets.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> Since we performed two independent experiments for each strain, some probes yielded inconsistent results; P/A, P/LQ or A/LQ. In these cases, the probe was categorized as ‘uncertain’ (U), P or A, respectively.</p><p>Finally, the presence or absence of each gene was determined according to the data from each spot. Because most genes were represented by two different probes on our microarray, inconsistent results could also be obtained from the two probes. In such cases, the gene was categorized as P when the judgments of two probes were P/U or P/LQ, as A when they were A/U or A/LQ, and as U when they were P/A or U/LQ. Processed data sets were displayed in the genomic order using the TREEVIEW program.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> Raw signal intensities from each spot and processed data are presented in Supplementary Tables 1 and 2 available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>, respectively.</p><p>The presence or absence of 37 genes, which were always categorized into LQ on the current version of O157 Sakai microarray, were determined by PCR. The PCR amplification was performed using the Ex Taq PCR kit (TAKARA Bio, Japan) and 10 ng of template DNA with 30 cycles of 20 s at 98°C/30 s at 60°C/30 (or 120) s at 72°C. The list of genes analyzed by PCR and the primer sequences are shown in Supplementary Table 3 available at <a class="link link-uri" href="http://www.dnares.oxfordjournals.org" target="_blank"></a>.</p>                            <h2 scrollto-destination=87193516 id="87193516" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=87193517 id="87193517" class="section-title" >3.1. Evaluation of the sensitivity and specificity of CGH analysis using the O157 Sakai oligoDNA microarray</h3>
<p>The array we constructed covered all of the 5447 protein-coding genes on the O157 Sakai genome; 3729 on the chromosome backbone, 1632 on S-loops, and 86 on pO157 and pOSAK1 plasmids.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> Before applying the microarray to the CGH analysis of O157 strains, we evaluated its sensitivity and specificity using K-12 MG1655 as a test strain.</p><p>We first examined the sequence identities of each probe to the K-12 genome sequence by the FASTA program, and classified the probes into two groups; those with ≥90% identity into ‘Conserved in K-12’ (CK) probes and others into ‘Specific to Sakai’ (SS) probes. Each gene was then classified into CK, ‘Partially Conserved in K-12’ (PCK; genes represented by a CK probe and an SS probe), or SS genes according to the probe category (Supplementary Table 1). Next, the data of the microarray analysis were compared with those from the <em>in silico</em> analysis. In this comparison, repeated gene families, represented by ‘multiple hit probes’ having two or more targets of ≥90% sequence identity on the O157 Sakai genome, were analyzed separately from the singleton genes.</p><p>As for the singleton genes, 96.9% of CK genes were correctly judged as ‘present’, and 97.8% of SS genes as ‘absent’ (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). Most of the K-12 genes that gave incorrect results (false negative, false positive and uncertain) contained slightly divergent target sequences with 3–6 base mismatches with CK probes or weak homologies to the SS probes (70–90% identity). On the other hand, the presence/absence determination of the repeated gene families conserved in K-12 was somewhat problematic. While all families with the copy number ratio of &gt;0.5 were judged as ‘present’, many families with the copy number ratios of ≤0.5 were judged as ‘absent’ or ‘uncertain’. This indicates that the repeated genes judged as ‘absent’ by the microarray analysis include those actually absent and those with reduced copy numbers. Thus, in the following CGH analyses of O157 strains, we decided to categorize such repeated genes into ‘uncertain due to the multiple target sequences in O157 Sakai’ [U(M)] and to consider only the repeated genes judged as ‘present’ as having the same or similar (or higher) copy numbers in the test strain.</p>                        <a id="87193521" scrollto-destination="87193521"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Summary of the CGH analysis of K-12 MG1655 using the O157 Sakai oligoDNA microarray</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Classification by the <em>in silico</em> analysis </th><th colspan="5">Number of genes (%)<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th></tr><tr><th> </th><th colspan="5"><hr /> </th></tr><tr><th> </th><th>Present </th><th>Absent </th><th>Uncertain </th><th>LQ<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Total </th></tr></thead><tbody><tr><td colspan="6">Singleton genes [4905 genes] </td></tr><tr><td>    Conserved in K-12 (CK genes) </td><td>3536 (96.9) </td><td>18 (0.5) dSeq in both probes [7/18] </td><td>75 (2.1) dSeq in either probe [67/75] </td><td>22 (0.6) </td><td>3651 </td></tr><tr><td>    Partially conserved in K-12 (PCK gene) </td><td>30 (29.7) wHom in SS probes [30/30] </td><td>9 (8.9) dSeq in CK probes [8/9] </td><td>62 (61.4) </td><td>0 </td><td>101 </td></tr><tr><td>    Sakai-specific (SS genes) </td><td>3 (0.3) wHom in both probes [3/3] </td><td>1128 (97.8) </td><td>9 (0.8) wHom in either probe [7/9] </td><td>13 (1.1) </td><td>1153 </td></tr><tr><td colspan="6">Repeated genes [151 families]<a reveal-id="tf1-3" data-open="tf1-3" class="link link-ref link-reveal xref-fn">c</a> </td></tr><tr><td>    Conserved in K-12 (CK genes) </td><td>12 (52.2) c.n.r. &gt; 0.5 [7/12] c.n.r. = 0.5 [5/12] </td><td>10 (43.5) c.n.r. = 0.5 [7/10] c.n.r. &lt; 0.5 [3/10] </td><td>1 (4.3) c.n.r. = 0.5 [1/1] </td><td>0 </td><td>23 </td></tr><tr><td>    Sakai-specific (SS genes) </td><td>0 </td><td>126 (98.4) </td><td>0 </td><td>2 (1.6) </td><td>128 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Classification by the <em>in silico</em> analysis </th><th colspan="5">Number of genes (%)<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th></tr><tr><th> </th><th colspan="5"><hr /> </th></tr><tr><th> </th><th>Present </th><th>Absent </th><th>Uncertain </th><th>LQ<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Total </th></tr></thead><tbody><tr><td colspan="6">Singleton genes [4905 genes] </td></tr><tr><td>    Conserved in K-12 (CK genes) </td><td>3536 (96.9) </td><td>18 (0.5) dSeq in both probes [7/18] </td><td>75 (2.1) dSeq in either probe [67/75] </td><td>22 (0.6) </td><td>3651 </td></tr><tr><td>    Partially conserved in K-12 (PCK gene) </td><td>30 (29.7) wHom in SS probes [30/30] </td><td>9 (8.9) dSeq in CK probes [8/9] </td><td>62 (61.4) </td><td>0 </td><td>101 </td></tr><tr><td>    Sakai-specific (SS genes) </td><td>3 (0.3) wHom in both probes [3/3] </td><td>1128 (97.8) </td><td>9 (0.8) wHom in either probe [7/9] </td><td>13 (1.1) </td><td>1153 </td></tr><tr><td colspan="6">Repeated genes [151 families]<a reveal-id="tf1-3" data-open="tf1-3" class="link link-ref link-reveal xref-fn">c</a> </td></tr><tr><td>    Conserved in K-12 (CK genes) </td><td>12 (52.2) c.n.r. &gt; 0.5 [7/12] c.n.r. = 0.5 [5/12] </td><td>10 (43.5) c.n.r. = 0.5 [7/10] c.n.r. &lt; 0.5 [3/10] </td><td>1 (4.3) c.n.r. = 0.5 [1/1] </td><td>0 </td><td>23 </td></tr><tr><td>    Sakai-specific (SS genes) </td><td>0 </td><td>126 (98.4) </td><td>0 </td><td>2 (1.6) </td><td>128 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf1-1"><div class="fn"><p><sup>a</sup>‘dSeq’ stands for the divergent sequence with 3–6 base mismatch, ‘wHom’ for the weak homology of 70–90% identiity, and ‘c.n.r.’ for the copy number ratio (MG1655/O157 Sakai).</p></div></div><div content-id="tf1-2"><div class="fn"><p><sup>b</sup>These 37 genes were always categorized into the LQ (low quality) group because of weak signal intensity. Their presence or absence in O157 strains was determind by PCR.</p></div></div><div content-id="tf1-3"><div class="fn"><p><sup>c</sup>Each repeated gene family was represented by a single probe in the O157 Sakai oligoDNA microarray.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193521" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Summary of the CGH analysis of K-12 MG1655 using the O157 Sakai oligoDNA microarray</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Classification by the <em>in silico</em> analysis </th><th colspan="5">Number of genes (%)<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th></tr><tr><th> </th><th colspan="5"><hr /> </th></tr><tr><th> </th><th>Present </th><th>Absent </th><th>Uncertain </th><th>LQ<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Total </th></tr></thead><tbody><tr><td colspan="6">Singleton genes [4905 genes] </td></tr><tr><td>    Conserved in K-12 (CK genes) </td><td>3536 (96.9) </td><td>18 (0.5) dSeq in both probes [7/18] </td><td>75 (2.1) dSeq in either probe [67/75] </td><td>22 (0.6) </td><td>3651 </td></tr><tr><td>    Partially conserved in K-12 (PCK gene) </td><td>30 (29.7) wHom in SS probes [30/30] </td><td>9 (8.9) dSeq in CK probes [8/9] </td><td>62 (61.4) </td><td>0 </td><td>101 </td></tr><tr><td>    Sakai-specific (SS genes) </td><td>3 (0.3) wHom in both probes [3/3] </td><td>1128 (97.8) </td><td>9 (0.8) wHom in either probe [7/9] </td><td>13 (1.1) </td><td>1153 </td></tr><tr><td colspan="6">Repeated genes [151 families]<a reveal-id="tf1-3" data-open="tf1-3" class="link link-ref link-reveal xref-fn">c</a> </td></tr><tr><td>    Conserved in K-12 (CK genes) </td><td>12 (52.2) c.n.r. &gt; 0.5 [7/12] c.n.r. = 0.5 [5/12] </td><td>10 (43.5) c.n.r. = 0.5 [7/10] c.n.r. &lt; 0.5 [3/10] </td><td>1 (4.3) c.n.r. = 0.5 [1/1] </td><td>0 </td><td>23 </td></tr><tr><td>    Sakai-specific (SS genes) </td><td>0 </td><td>126 (98.4) </td><td>0 </td><td>2 (1.6) </td><td>128 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Classification by the <em>in silico</em> analysis </th><th colspan="5">Number of genes (%)<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th></tr><tr><th> </th><th colspan="5"><hr /> </th></tr><tr><th> </th><th>Present </th><th>Absent </th><th>Uncertain </th><th>LQ<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Total </th></tr></thead><tbody><tr><td colspan="6">Singleton genes [4905 genes] </td></tr><tr><td>    Conserved in K-12 (CK genes) </td><td>3536 (96.9) </td><td>18 (0.5) dSeq in both probes [7/18] </td><td>75 (2.1) dSeq in either probe [67/75] </td><td>22 (0.6) </td><td>3651 </td></tr><tr><td>    Partially conserved in K-12 (PCK gene) </td><td>30 (29.7) wHom in SS probes [30/30] </td><td>9 (8.9) dSeq in CK probes [8/9] </td><td>62 (61.4) </td><td>0 </td><td>101 </td></tr><tr><td>    Sakai-specific (SS genes) </td><td>3 (0.3) wHom in both probes [3/3] </td><td>1128 (97.8) </td><td>9 (0.8) wHom in either probe [7/9] </td><td>13 (1.1) </td><td>1153 </td></tr><tr><td colspan="6">Repeated genes [151 families]<a reveal-id="tf1-3" data-open="tf1-3" class="link link-ref link-reveal xref-fn">c</a> </td></tr><tr><td>    Conserved in K-12 (CK genes) </td><td>12 (52.2) c.n.r. &gt; 0.5 [7/12] c.n.r. = 0.5 [5/12] </td><td>10 (43.5) c.n.r. = 0.5 [7/10] c.n.r. &lt; 0.5 [3/10] </td><td>1 (4.3) c.n.r. = 0.5 [1/1] </td><td>0 </td><td>23 </td></tr><tr><td>    Sakai-specific (SS genes) </td><td>0 </td><td>126 (98.4) </td><td>0 </td><td>2 (1.6) </td><td>128 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf1-1"><div class="fn"><p><sup>a</sup>‘dSeq’ stands for the divergent sequence with 3–6 base mismatch, ‘wHom’ for the weak homology of 70–90% identiity, and ‘c.n.r.’ for the copy number ratio (MG1655/O157 Sakai).</p></div></div><div content-id="tf1-2"><div class="fn"><p><sup>b</sup>These 37 genes were always categorized into the LQ (low quality) group because of weak signal intensity. Their presence or absence in O157 strains was determind by PCR.</p></div></div><div content-id="tf1-3"><div class="fn"><p><sup>c</sup>Each repeated gene family was represented by a single probe in the O157 Sakai oligoDNA microarray.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193521" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87193522 id="87193522" class="section-title" >3.2. Overview of the gene content analysis of O157 strains</h3>
<p>Using the O157 Sakai microarray, we analyzed the gene content of eight O157 strains that were previously analyzed by the WGPScanning method.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> The data are summarized in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> and <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Tables 2</a> and <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">3</a>. As shown in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>, all the singleton genes identified in O157 Sakai were shared by at least one strain, but as many as 431 genes displayed variable distributions (or highly divergent sequences). Most of these ‘variably absent or present genes’ (referred to as VAP genes) belonged to the SS genes; 389 of the 1153 SS singleton genes were variably present. As shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, these genes appeared very frequently on prophages and prophage-like elements; among the 640 singleton genes on these genetic elements, 350 (54.7%) displayed variable distributions. In sharp contrast, CK singleton genes exhibited a high level of conservation; only 1% (37 genes) exhibited variable distributions (<a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>). Among these, 20 genes were again located on prophage regions shared by O157 and K-12. In particular, a region of Sp10 (corresponding to a part of the Rac prophage in K-12) contained 11 VAP genes. These included <em>recE</em> and <em>recT</em> genes (ECs1933 and ECs1934) involved in the RecA-independent recombination and double-strand break repair.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a></p>                        <a id="87193524" scrollto-destination="87193524"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/m_dsi026f1a.jpeg?Expires=2147483647&amp;Signature=S~oX-41CkUBx6NkiBY8GAZuKmEs48Wsi6RYkyowKHRPf7B9vhsiqhCM3LvLmWKQm6OTm0~zlBh3ftZHu3OLZ68hzGiuPh~OHxYEs1pAfStJ3Qt~cOg4DhlUfvIXN~XB0FVHXERsD3z6KdNAJvb3uDK~JIj8W4KeuxYWP23Lh0h-JXgTZms73jUZIszHSZbmxFk~uuRDcz5qRF-UPVNQqVggVGU9Mg44pKY6yPBIA5Kur623WeL~YU934536EuGwtxTNzB-eS~2gAByf9xGyVovcCtFOp-~ro-3gASBQf2mD4au6gmPEPrDfDfjiBEk-YMuqt8DuhGLWeYQDOgIdCfw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The summary and data comparison of the microarray analysis and the WGPScanning analysis of the same set of O157 strains. Results from the gene composition analysis of eight O157 strains (strains #2–#9) using the O157 Sakai microarray are shown in the upper parts, and those from the genome structure analysis using the WGPScanning method in the lower parts. In the three rows above the microarray data, genes conserved and partially conserved in K-12 are indicated in green and pink, respectively (the first row), genes on prophages, prophage-like elements and plasmids in red (the second row), and repeated genes represented by multiple hit probes in black (the third row). Genes judged as present and absent in the microarray analysis are indicated in blue and yellow, respectively. Singleton and repeated genes that were classified as ‘uncertain’ are indicated in pink and gray, respectively. Results from the WGPScaning analysis are presented as follows. The segments of the same sizes as those from O157 Sakai are indicated in gray, and those with larger (≥2 kb) and smaller (&lt;2 kb) size reductions in yellow and light yellow, respectively. The segments with larger (≥2 kb) and smaller (&lt;2 kb) size increments are indicated in blue and light blue, respectively, and those that were not amplified in red. Prophages or prophage-like elements of O157 Sakai that were deleted (or translocated or not integrated) in some test strains are left blank, and their alternative integration sites so far identified are also indicated. In this figure, each segment is not drawn to scale but to the gene position in the data presentation of the microarray analysis. The data presented here are the most update version, thus several modifications have been made from the original version published previously.8" path-from-xml="dsi026f1a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193524/dsi026f1a.jpeg" path-from-xml="dsi026f1a.jpeg" target="_blank">View large</a><a section="87193524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/dsi026f1a.jpeg?Expires=2147483647&amp;Signature=S84BJL3X~uAT0L5Rp7ItfvzNRAYGfj7M7rJXWBxkB4H~JfvAyK-PR~MD5sg1KNF-8oNxhSTsQR3wtPouFMZExa1fS9fhywBUk7U5rjvpn1O-OeO0TlY7gg6KwxcrXFARgbn4dgb5f1ioGRaR7sTn8MWmsu8LU0UL2xKNhJhCnafGirP7wI13XuXlrA3GBWZAjtVk3Rf2S827HaaFXeKaYVIq03x6SQ9JLl9SF9u6NwWsSOxY1ckivZ6wyc6SEU8P0DH99DNqQVrULyWtodxXa5q413e5K9lHbtO7XWL3Rj5EDc3YfeIJNxtdUgUO2qVNSLFbM-TT2FuNvoiXN18Pkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193524&amp;ar=486680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi026f1a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/m_dsi026f1b.jpeg?Expires=2147483647&amp;Signature=trIZoGa6yUbhl32bkol4u~9Vwz3XbPZ4Md3e2~VWnwC3mMcbd8ne4JO2VWxzlOaZYg~PlpGDlgYu8StmoOMQH14aoYj78r6q6P7vWhT2y6dPpE1O0ZomJ~kbaTgKYp2IOW2uMA1zv34YXmX~BeSTo4qnPFTp94rI6VMQXYJbHktL2n9Ah13GIHMXrZOUzXzPblebYa0QKUBu0WlvmeXHxRE7kAuLx35fD6AtAG5O74ZqY1UH3qr7CHVd4Gki4LMiN-cemSWc6xkpJrfr625TtvntTsAwGb~BIqjM4RkHExFkC1pHPmJF1DtQ8rYsljEF7nyFQbI7bmpnqth3bDVA0A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The summary and data comparison of the microarray analysis and the WGPScanning analysis of the same set of O157 strains. Results from the gene composition analysis of eight O157 strains (strains #2–#9) using the O157 Sakai microarray are shown in the upper parts, and those from the genome structure analysis using the WGPScanning method in the lower parts. In the three rows above the microarray data, genes conserved and partially conserved in K-12 are indicated in green and pink, respectively (the first row), genes on prophages, prophage-like elements and plasmids in red (the second row), and repeated genes represented by multiple hit probes in black (the third row). Genes judged as present and absent in the microarray analysis are indicated in blue and yellow, respectively. Singleton and repeated genes that were classified as ‘uncertain’ are indicated in pink and gray, respectively. Results from the WGPScaning analysis are presented as follows. The segments of the same sizes as those from O157 Sakai are indicated in gray, and those with larger (≥2 kb) and smaller (&lt;2 kb) size reductions in yellow and light yellow, respectively. The segments with larger (≥2 kb) and smaller (&lt;2 kb) size increments are indicated in blue and light blue, respectively, and those that were not amplified in red. Prophages or prophage-like elements of O157 Sakai that were deleted (or translocated or not integrated) in some test strains are left blank, and their alternative integration sites so far identified are also indicated. In this figure, each segment is not drawn to scale but to the gene position in the data presentation of the microarray analysis. The data presented here are the most update version, thus several modifications have been made from the original version published previously.8" path-from-xml="dsi026f1b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193524/dsi026f1b.jpeg" path-from-xml="dsi026f1b.jpeg" target="_blank">View large</a><a section="87193524" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/dsi026f1b.jpeg?Expires=2147483647&amp;Signature=s6zqQqnHXh486JjTL-F656zrnS5qq8XK~H~njxgqeXE2ktpC1gf5MsNareTjQ0YUhypL9zFqbig3EE0iV8TqTGl4Oh3Zuvnym5VWAht8QU8kpG06vXxu0hbwWN4LcETZJAjzBSZPw~dOBqqmUaAT8Lwvmv6E6FTJCKHENyZapa2vPeuCHirrSdJlyL2Njs~6yFlOdFMWEwLCmbupdcZRhIsZ~qZJvS9lbNw787UkB-Afq-An7xnx-mnd1TaIWEt3441XBPKeT0EYF97gxqjkH0L7yBVnj-G1qUJO3MCQU~O1C5IXzKW2UYUmrF4T9amLJNmmA6aMNNm2tlpcoF6j2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193524&amp;ar=486680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi026f1b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The summary and data comparison of the microarray analysis and the WGPScanning analysis of the same set of O157 strains. Results from the gene composition analysis of eight O157 strains (strains #2–#9) using the O157 Sakai microarray are shown in the upper parts, and those from the genome structure analysis using the WGPScanning method in the lower parts. In the three rows above the microarray data, genes conserved and partially conserved in K-12 are indicated in green and pink, respectively (the first row), genes on prophages, prophage-like elements and plasmids in red (the second row), and repeated genes represented by multiple hit probes in black (the third row). Genes judged as present and absent in the microarray analysis are indicated in blue and yellow, respectively. Singleton and repeated genes that were classified as ‘uncertain’ are indicated in pink and gray, respectively. Results from the WGPScaning analysis are presented as follows. The segments of the same sizes as those from O157 Sakai are indicated in gray, and those with larger (≥2 kb) and smaller (&lt;2 kb) size reductions in yellow and light yellow, respectively. The segments with larger (≥2 kb) and smaller (&lt;2 kb) size increments are indicated in blue and light blue, respectively, and those that were not amplified in red. Prophages or prophage-like elements of O157 Sakai that were deleted (or translocated or not integrated) in some test strains are left blank, and their alternative integration sites so far identified are also indicated. In this figure, each segment is not drawn to scale but to the gene position in the data presentation of the microarray analysis. The data presented here are the most update version, thus several modifications have been made from the original version published previously.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a></p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/m_dsi026f1a.jpeg?Expires=2147483647&amp;Signature=S~oX-41CkUBx6NkiBY8GAZuKmEs48Wsi6RYkyowKHRPf7B9vhsiqhCM3LvLmWKQm6OTm0~zlBh3ftZHu3OLZ68hzGiuPh~OHxYEs1pAfStJ3Qt~cOg4DhlUfvIXN~XB0FVHXERsD3z6KdNAJvb3uDK~JIj8W4KeuxYWP23Lh0h-JXgTZms73jUZIszHSZbmxFk~uuRDcz5qRF-UPVNQqVggVGU9Mg44pKY6yPBIA5Kur623WeL~YU934536EuGwtxTNzB-eS~2gAByf9xGyVovcCtFOp-~ro-3gASBQf2mD4au6gmPEPrDfDfjiBEk-YMuqt8DuhGLWeYQDOgIdCfw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The summary and data comparison of the microarray analysis and the WGPScanning analysis of the same set of O157 strains. Results from the gene composition analysis of eight O157 strains (strains #2–#9) using the O157 Sakai microarray are shown in the upper parts, and those from the genome structure analysis using the WGPScanning method in the lower parts. In the three rows above the microarray data, genes conserved and partially conserved in K-12 are indicated in green and pink, respectively (the first row), genes on prophages, prophage-like elements and plasmids in red (the second row), and repeated genes represented by multiple hit probes in black (the third row). Genes judged as present and absent in the microarray analysis are indicated in blue and yellow, respectively. Singleton and repeated genes that were classified as ‘uncertain’ are indicated in pink and gray, respectively. Results from the WGPScaning analysis are presented as follows. The segments of the same sizes as those from O157 Sakai are indicated in gray, and those with larger (≥2 kb) and smaller (&lt;2 kb) size reductions in yellow and light yellow, respectively. The segments with larger (≥2 kb) and smaller (&lt;2 kb) size increments are indicated in blue and light blue, respectively, and those that were not amplified in red. Prophages or prophage-like elements of O157 Sakai that were deleted (or translocated or not integrated) in some test strains are left blank, and their alternative integration sites so far identified are also indicated. In this figure, each segment is not drawn to scale but to the gene position in the data presentation of the microarray analysis. The data presented here are the most update version, thus several modifications have been made from the original version published previously.8" path-from-xml="dsi026f1a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193524/dsi026f1a.jpeg" path-from-xml="dsi026f1a.jpeg" target="_blank">View large</a><a section="87193524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/dsi026f1a.jpeg?Expires=2147483647&amp;Signature=S84BJL3X~uAT0L5Rp7ItfvzNRAYGfj7M7rJXWBxkB4H~JfvAyK-PR~MD5sg1KNF-8oNxhSTsQR3wtPouFMZExa1fS9fhywBUk7U5rjvpn1O-OeO0TlY7gg6KwxcrXFARgbn4dgb5f1ioGRaR7sTn8MWmsu8LU0UL2xKNhJhCnafGirP7wI13XuXlrA3GBWZAjtVk3Rf2S827HaaFXeKaYVIq03x6SQ9JLl9SF9u6NwWsSOxY1ckivZ6wyc6SEU8P0DH99DNqQVrULyWtodxXa5q413e5K9lHbtO7XWL3Rj5EDc3YfeIJNxtdUgUO2qVNSLFbM-TT2FuNvoiXN18Pkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193524&amp;ar=486680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi026f1a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/m_dsi026f1b.jpeg?Expires=2147483647&amp;Signature=trIZoGa6yUbhl32bkol4u~9Vwz3XbPZ4Md3e2~VWnwC3mMcbd8ne4JO2VWxzlOaZYg~PlpGDlgYu8StmoOMQH14aoYj78r6q6P7vWhT2y6dPpE1O0ZomJ~kbaTgKYp2IOW2uMA1zv34YXmX~BeSTo4qnPFTp94rI6VMQXYJbHktL2n9Ah13GIHMXrZOUzXzPblebYa0QKUBu0WlvmeXHxRE7kAuLx35fD6AtAG5O74ZqY1UH3qr7CHVd4Gki4LMiN-cemSWc6xkpJrfr625TtvntTsAwGb~BIqjM4RkHExFkC1pHPmJF1DtQ8rYsljEF7nyFQbI7bmpnqth3bDVA0A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The summary and data comparison of the microarray analysis and the WGPScanning analysis of the same set of O157 strains. Results from the gene composition analysis of eight O157 strains (strains #2–#9) using the O157 Sakai microarray are shown in the upper parts, and those from the genome structure analysis using the WGPScanning method in the lower parts. In the three rows above the microarray data, genes conserved and partially conserved in K-12 are indicated in green and pink, respectively (the first row), genes on prophages, prophage-like elements and plasmids in red (the second row), and repeated genes represented by multiple hit probes in black (the third row). Genes judged as present and absent in the microarray analysis are indicated in blue and yellow, respectively. Singleton and repeated genes that were classified as ‘uncertain’ are indicated in pink and gray, respectively. Results from the WGPScaning analysis are presented as follows. The segments of the same sizes as those from O157 Sakai are indicated in gray, and those with larger (≥2 kb) and smaller (&lt;2 kb) size reductions in yellow and light yellow, respectively. The segments with larger (≥2 kb) and smaller (&lt;2 kb) size increments are indicated in blue and light blue, respectively, and those that were not amplified in red. Prophages or prophage-like elements of O157 Sakai that were deleted (or translocated or not integrated) in some test strains are left blank, and their alternative integration sites so far identified are also indicated. In this figure, each segment is not drawn to scale but to the gene position in the data presentation of the microarray analysis. The data presented here are the most update version, thus several modifications have been made from the original version published previously.8" path-from-xml="dsi026f1b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193524/dsi026f1b.jpeg" path-from-xml="dsi026f1b.jpeg" target="_blank">View large</a><a section="87193524" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/dsi026f1b.jpeg?Expires=2147483647&amp;Signature=s6zqQqnHXh486JjTL-F656zrnS5qq8XK~H~njxgqeXE2ktpC1gf5MsNareTjQ0YUhypL9zFqbig3EE0iV8TqTGl4Oh3Zuvnym5VWAht8QU8kpG06vXxu0hbwWN4LcETZJAjzBSZPw~dOBqqmUaAT8Lwvmv6E6FTJCKHENyZapa2vPeuCHirrSdJlyL2Njs~6yFlOdFMWEwLCmbupdcZRhIsZ~qZJvS9lbNw787UkB-Afq-An7xnx-mnd1TaIWEt3441XBPKeT0EYF97gxqjkH0L7yBVnj-G1qUJO3MCQU~O1C5IXzKW2UYUmrF4T9amLJNmmA6aMNNm2tlpcoF6j2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193524&amp;ar=486680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi026f1b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The summary and data comparison of the microarray analysis and the WGPScanning analysis of the same set of O157 strains. Results from the gene composition analysis of eight O157 strains (strains #2–#9) using the O157 Sakai microarray are shown in the upper parts, and those from the genome structure analysis using the WGPScanning method in the lower parts. In the three rows above the microarray data, genes conserved and partially conserved in K-12 are indicated in green and pink, respectively (the first row), genes on prophages, prophage-like elements and plasmids in red (the second row), and repeated genes represented by multiple hit probes in black (the third row). Genes judged as present and absent in the microarray analysis are indicated in blue and yellow, respectively. Singleton and repeated genes that were classified as ‘uncertain’ are indicated in pink and gray, respectively. Results from the WGPScaning analysis are presented as follows. The segments of the same sizes as those from O157 Sakai are indicated in gray, and those with larger (≥2 kb) and smaller (&lt;2 kb) size reductions in yellow and light yellow, respectively. The segments with larger (≥2 kb) and smaller (&lt;2 kb) size increments are indicated in blue and light blue, respectively, and those that were not amplified in red. Prophages or prophage-like elements of O157 Sakai that were deleted (or translocated or not integrated) in some test strains are left blank, and their alternative integration sites so far identified are also indicated. In this figure, each segment is not drawn to scale but to the gene position in the data presentation of the microarray analysis. The data presented here are the most update version, thus several modifications have been made from the original version published previously.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a></p></div></div>                        <a id="87193525" scrollto-destination="87193525"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Conservation of O157 Sakai genes in the eight O157 strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="9">Number of absent genes<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Total<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>No. of VAP<a reveal-id="tf2-3" data-open="tf2-3" class="link link-ref link-reveal xref-fn">c</a> genes (%) </th></tr><tr><th> </th><th colspan="9"><hr /> </th><th> </th><th> </th></tr><tr><th> </th><th>8/8 </th><th>7/8 </th><th>6/8 </th><th>5/8 </th><th>4/8 </th><th>3/8 </th><th>2/8 </th><th>1/8 </th><th>0/8 </th><th> </th><th> </th></tr></thead><tbody><tr><td>Conserved in K-12 (CK genes) </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td><td>13 </td><td>18 </td><td>3614 </td><td>3651 </td><td>37 (1.0) </td></tr><tr><td>Partly conserved in K-12 (PCK gene) </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td><td>3 </td><td>96 </td><td>101 </td><td>5 (5.0) </td></tr><tr><td>Sakai-specific (SS genes) </td><td>0 </td><td>63 </td><td>11 </td><td>27 </td><td>52 </td><td>37 </td><td>78 </td><td>121 </td><td>764 </td><td>1153 </td><td>389 (33.7) </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="9">Number of absent genes<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Total<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>No. of VAP<a reveal-id="tf2-3" data-open="tf2-3" class="link link-ref link-reveal xref-fn">c</a> genes (%) </th></tr><tr><th> </th><th colspan="9"><hr /> </th><th> </th><th> </th></tr><tr><th> </th><th>8/8 </th><th>7/8 </th><th>6/8 </th><th>5/8 </th><th>4/8 </th><th>3/8 </th><th>2/8 </th><th>1/8 </th><th>0/8 </th><th> </th><th> </th></tr></thead><tbody><tr><td>Conserved in K-12 (CK genes) </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td><td>13 </td><td>18 </td><td>3614 </td><td>3651 </td><td>37 (1.0) </td></tr><tr><td>Partly conserved in K-12 (PCK gene) </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td><td>3 </td><td>96 </td><td>101 </td><td>5 (5.0) </td></tr><tr><td>Sakai-specific (SS genes) </td><td>0 </td><td>63 </td><td>11 </td><td>27 </td><td>52 </td><td>37 </td><td>78 </td><td>121 </td><td>764 </td><td>1153 </td><td>389 (33.7) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf2-1"><div class="fn"><p><sup>a</sup>Genes that were judged as ‘uncertain’ were not considered.</p></div></div><div content-id="tf2-2"><div class="fn"><p><sup>b</sup>Only the singleton genes were analyzed.</p></div></div><div content-id="tf2-3"><div class="fn"><p><sup>c</sup>Genes that were absent at least in one strain.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193525" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Conservation of O157 Sakai genes in the eight O157 strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="9">Number of absent genes<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Total<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>No. of VAP<a reveal-id="tf2-3" data-open="tf2-3" class="link link-ref link-reveal xref-fn">c</a> genes (%) </th></tr><tr><th> </th><th colspan="9"><hr /> </th><th> </th><th> </th></tr><tr><th> </th><th>8/8 </th><th>7/8 </th><th>6/8 </th><th>5/8 </th><th>4/8 </th><th>3/8 </th><th>2/8 </th><th>1/8 </th><th>0/8 </th><th> </th><th> </th></tr></thead><tbody><tr><td>Conserved in K-12 (CK genes) </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td><td>13 </td><td>18 </td><td>3614 </td><td>3651 </td><td>37 (1.0) </td></tr><tr><td>Partly conserved in K-12 (PCK gene) </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td><td>3 </td><td>96 </td><td>101 </td><td>5 (5.0) </td></tr><tr><td>Sakai-specific (SS genes) </td><td>0 </td><td>63 </td><td>11 </td><td>27 </td><td>52 </td><td>37 </td><td>78 </td><td>121 </td><td>764 </td><td>1153 </td><td>389 (33.7) </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="9">Number of absent genes<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Total<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>No. of VAP<a reveal-id="tf2-3" data-open="tf2-3" class="link link-ref link-reveal xref-fn">c</a> genes (%) </th></tr><tr><th> </th><th colspan="9"><hr /> </th><th> </th><th> </th></tr><tr><th> </th><th>8/8 </th><th>7/8 </th><th>6/8 </th><th>5/8 </th><th>4/8 </th><th>3/8 </th><th>2/8 </th><th>1/8 </th><th>0/8 </th><th> </th><th> </th></tr></thead><tbody><tr><td>Conserved in K-12 (CK genes) </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td><td>13 </td><td>18 </td><td>3614 </td><td>3651 </td><td>37 (1.0) </td></tr><tr><td>Partly conserved in K-12 (PCK gene) </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>1 </td><td>3 </td><td>96 </td><td>101 </td><td>5 (5.0) </td></tr><tr><td>Sakai-specific (SS genes) </td><td>0 </td><td>63 </td><td>11 </td><td>27 </td><td>52 </td><td>37 </td><td>78 </td><td>121 </td><td>764 </td><td>1153 </td><td>389 (33.7) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf2-1"><div class="fn"><p><sup>a</sup>Genes that were judged as ‘uncertain’ were not considered.</p></div></div><div content-id="tf2-2"><div class="fn"><p><sup>b</sup>Only the singleton genes were analyzed.</p></div></div><div content-id="tf2-3"><div class="fn"><p><sup>c</sup>Genes that were absent at least in one strain.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193525" target="_blank">View Large</a></div></div>                        <a id="87193526" scrollto-destination="87193526"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of the CGH analysis of each O157 strain</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="8">Strains </th></tr><tr><th> </th><th colspan="8"><hr /> </th></tr><tr><th> </th><th>#2 </th><th>#3 </th><th>#4 </th><th>#5 </th><th>#6 </th><th>#7 </th><th>#8 </th><th>#9 </th></tr></thead><tbody><tr><td colspan="9">Singleton genes [4905 genes] </td></tr><tr><td>    Present </td><td>4578 </td><td>4771 </td><td>4721 </td><td>4823 </td><td>4712 </td><td>4659 </td><td>4746 </td><td>4783 </td></tr><tr><td>    Absent </td><td>307 </td><td>125 </td><td>141 </td><td>80 </td><td>174 </td><td>222 </td><td>140 </td><td>117 </td></tr><tr><td>    Uncertain </td><td>20 </td><td>9 </td><td>43 </td><td>2 </td><td>19 </td><td>24 </td><td>19 </td><td>5 </td></tr><tr><td colspan="9">Repeated genes [151 families] </td></tr><tr><td>    Present </td><td>111 </td><td>150 </td><td>128 </td><td>146 </td><td>126 </td><td>133 </td><td>143 </td><td>149 </td></tr><tr><td>    Uncertain [M]<a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>40 </td><td>1 </td><td>23 </td><td>5 </td><td>25 </td><td>18 </td><td>8 </td><td>2 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="8">Strains </th></tr><tr><th> </th><th colspan="8"><hr /> </th></tr><tr><th> </th><th>#2 </th><th>#3 </th><th>#4 </th><th>#5 </th><th>#6 </th><th>#7 </th><th>#8 </th><th>#9 </th></tr></thead><tbody><tr><td colspan="9">Singleton genes [4905 genes] </td></tr><tr><td>    Present </td><td>4578 </td><td>4771 </td><td>4721 </td><td>4823 </td><td>4712 </td><td>4659 </td><td>4746 </td><td>4783 </td></tr><tr><td>    Absent </td><td>307 </td><td>125 </td><td>141 </td><td>80 </td><td>174 </td><td>222 </td><td>140 </td><td>117 </td></tr><tr><td>    Uncertain </td><td>20 </td><td>9 </td><td>43 </td><td>2 </td><td>19 </td><td>24 </td><td>19 </td><td>5 </td></tr><tr><td colspan="9">Repeated genes [151 families] </td></tr><tr><td>    Present </td><td>111 </td><td>150 </td><td>128 </td><td>146 </td><td>126 </td><td>133 </td><td>143 </td><td>149 </td></tr><tr><td>    Uncertain [M]<a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>40 </td><td>1 </td><td>23 </td><td>5 </td><td>25 </td><td>18 </td><td>8 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf3-1"><div class="fn"><p><sup>a</sup>They included actually absent gene families and those appeard to be absent due to the reduced copy number.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193526" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of the CGH analysis of each O157 strain</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="8">Strains </th></tr><tr><th> </th><th colspan="8"><hr /> </th></tr><tr><th> </th><th>#2 </th><th>#3 </th><th>#4 </th><th>#5 </th><th>#6 </th><th>#7 </th><th>#8 </th><th>#9 </th></tr></thead><tbody><tr><td colspan="9">Singleton genes [4905 genes] </td></tr><tr><td>    Present </td><td>4578 </td><td>4771 </td><td>4721 </td><td>4823 </td><td>4712 </td><td>4659 </td><td>4746 </td><td>4783 </td></tr><tr><td>    Absent </td><td>307 </td><td>125 </td><td>141 </td><td>80 </td><td>174 </td><td>222 </td><td>140 </td><td>117 </td></tr><tr><td>    Uncertain </td><td>20 </td><td>9 </td><td>43 </td><td>2 </td><td>19 </td><td>24 </td><td>19 </td><td>5 </td></tr><tr><td colspan="9">Repeated genes [151 families] </td></tr><tr><td>    Present </td><td>111 </td><td>150 </td><td>128 </td><td>146 </td><td>126 </td><td>133 </td><td>143 </td><td>149 </td></tr><tr><td>    Uncertain [M]<a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>40 </td><td>1 </td><td>23 </td><td>5 </td><td>25 </td><td>18 </td><td>8 </td><td>2 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="8">Strains </th></tr><tr><th> </th><th colspan="8"><hr /> </th></tr><tr><th> </th><th>#2 </th><th>#3 </th><th>#4 </th><th>#5 </th><th>#6 </th><th>#7 </th><th>#8 </th><th>#9 </th></tr></thead><tbody><tr><td colspan="9">Singleton genes [4905 genes] </td></tr><tr><td>    Present </td><td>4578 </td><td>4771 </td><td>4721 </td><td>4823 </td><td>4712 </td><td>4659 </td><td>4746 </td><td>4783 </td></tr><tr><td>    Absent </td><td>307 </td><td>125 </td><td>141 </td><td>80 </td><td>174 </td><td>222 </td><td>140 </td><td>117 </td></tr><tr><td>    Uncertain </td><td>20 </td><td>9 </td><td>43 </td><td>2 </td><td>19 </td><td>24 </td><td>19 </td><td>5 </td></tr><tr><td colspan="9">Repeated genes [151 families] </td></tr><tr><td>    Present </td><td>111 </td><td>150 </td><td>128 </td><td>146 </td><td>126 </td><td>133 </td><td>143 </td><td>149 </td></tr><tr><td>    Uncertain [M]<a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>40 </td><td>1 </td><td>23 </td><td>5 </td><td>25 </td><td>18 </td><td>8 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf3-1"><div class="fn"><p><sup>a</sup>They included actually absent gene families and those appeard to be absent due to the reduced copy number.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193526" target="_blank">View Large</a></div></div><p>Among the eight strains, strain #2 lacked the largest number of genes (at least 307 genes), and the number of missing singleton genes reduced in the following order; #7, #6, #4, #8, #3, #9 and #5 (<a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>). This pattern well correlated with the level of structural diversification observed in the WGPScanning analysis (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, and also refer to Ref. <a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a>). Most of the repeated gene families identified in O157 Sakai were also conserved (present in the same or similar copy number) in strains #3, #5, #8 and #9. These data suggest that the four strains are more closely related to O157 Sakai in terms of gene content and genome structure.</p>                            <h3 scrollto-destination=87193528 id="87193528" class="section-title" >3.3. O157 Sakai-specific genes with variable distributions in O157</h3>
<p>Classification of the SS genes according to their functions revealed that genes belonging to specific categories contained more VAP genes compared to other categories (<a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>). The abundance of VAP genes in ‘transcription’, ‘replication, recombination, and repair’, ‘genes with unknown or uncharacterized functions’ and ‘unclassified genes’ was remarkable. This appears to be a reflection of the fact that most VAP genes are on prophages or prophage-like elements, which contain many genes for transcriptional regulation, replication, repair and phage-specific functions as well as a number of uncharacterized genes.</p>                        <a id="87193530" scrollto-destination="87193530"></a>
<div content-id="tbl4" class="table-modal"><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Categorization and conservation of SS genes in the eight O157 strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th colspan="3">Number of genes </th></tr><tr><th> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Conserved </th><th>VAP<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Total<a reveal-id="tf4-2" data-open="tf4-2" class="link link-ref link-reveal xref-fn">b</a> </th></tr></thead><tbody><tr><td colspan="4">On the chromosome<a reveal-id="tf4-3" data-open="tf4-3" class="link link-ref link-reveal xref-fn">c</a> </td></tr><tr><td>    Energy production and conversion </td><td>6 </td><td>0 </td><td>6 </td></tr><tr><td>    Amino acid transport and metabolism </td><td>4 </td><td>4 </td><td>8 </td></tr><tr><td>    Carbohydrate transport and metabolism </td><td>37 </td><td>1 </td><td>38 </td></tr><tr><td>    Coenzyme transport and metabolism </td><td>5 </td><td>1 </td><td>6 </td></tr><tr><td>    Lipid transport and metabolism </td><td>15 </td><td>0 </td><td>15 </td></tr><tr><td>    Translation </td><td>1 </td><td>0 </td><td>1 </td></tr><tr><td>    Transcription </td><td>30 </td><td>16 </td><td>46 </td></tr><tr><td>    Replication, recombination and repair </td><td>38 </td><td>31 </td><td>69 </td></tr><tr><td>    Cell wall/membrane biogenesis </td><td>32 </td><td>2 </td><td>34 </td></tr><tr><td>    Cell motility </td><td>45 </td><td>1 </td><td>46 </td></tr><tr><td>    Post-translational modification, protein turnover, chaperones </td><td>3 </td><td>5 </td><td>8 </td></tr><tr><td>    Inorganic ion transport and metabolism </td><td>25 </td><td>4 </td><td>29 </td></tr><tr><td>    Secondary metabolites biosynthesis, transport and catabolism </td><td>7 </td><td>1 </td><td>8 </td></tr><tr><td>    Signal transduction mechanisms </td><td>10 </td><td>2 </td><td>12 </td></tr><tr><td>    Intracellular trafficking and secretion </td><td>16 </td><td>1 </td><td>17 </td></tr><tr><td>    Defense mechanisms </td><td>8 </td><td>1 </td><td>9 </td></tr><tr><td>    General function prediction only </td><td>40 </td><td>22 </td><td>62 </td></tr><tr><td>    Function unknown </td><td>56 </td><td>19 </td><td>75 </td></tr><tr><td>    Not in COGs </td><td>328 </td><td>257 </td><td>585 </td></tr><tr><td>On plasmids </td><td>58 </td><td>21 </td><td>79 </td></tr><tr><td>Total </td><td>764 </td><td>389 </td><td>1153 </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th colspan="3">Number of genes </th></tr><tr><th> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Conserved </th><th>VAP<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Total<a reveal-id="tf4-2" data-open="tf4-2" class="link link-ref link-reveal xref-fn">b</a> </th></tr></thead><tbody><tr><td colspan="4">On the chromosome<a reveal-id="tf4-3" data-open="tf4-3" class="link link-ref link-reveal xref-fn">c</a> </td></tr><tr><td>    Energy production and conversion </td><td>6 </td><td>0 </td><td>6 </td></tr><tr><td>    Amino acid transport and metabolism </td><td>4 </td><td>4 </td><td>8 </td></tr><tr><td>    Carbohydrate transport and metabolism </td><td>37 </td><td>1 </td><td>38 </td></tr><tr><td>    Coenzyme transport and metabolism </td><td>5 </td><td>1 </td><td>6 </td></tr><tr><td>    Lipid transport and metabolism </td><td>15 </td><td>0 </td><td>15 </td></tr><tr><td>    Translation </td><td>1 </td><td>0 </td><td>1 </td></tr><tr><td>    Transcription </td><td>30 </td><td>16 </td><td>46 </td></tr><tr><td>    Replication, recombination and repair </td><td>38 </td><td>31 </td><td>69 </td></tr><tr><td>    Cell wall/membrane biogenesis </td><td>32 </td><td>2 </td><td>34 </td></tr><tr><td>    Cell motility </td><td>45 </td><td>1 </td><td>46 </td></tr><tr><td>    Post-translational modification, protein turnover, chaperones </td><td>3 </td><td>5 </td><td>8 </td></tr><tr><td>    Inorganic ion transport and metabolism </td><td>25 </td><td>4 </td><td>29 </td></tr><tr><td>    Secondary metabolites biosynthesis, transport and catabolism </td><td>7 </td><td>1 </td><td>8 </td></tr><tr><td>    Signal transduction mechanisms </td><td>10 </td><td>2 </td><td>12 </td></tr><tr><td>    Intracellular trafficking and secretion </td><td>16 </td><td>1 </td><td>17 </td></tr><tr><td>    Defense mechanisms </td><td>8 </td><td>1 </td><td>9 </td></tr><tr><td>    General function prediction only </td><td>40 </td><td>22 </td><td>62 </td></tr><tr><td>    Function unknown </td><td>56 </td><td>19 </td><td>75 </td></tr><tr><td>    Not in COGs </td><td>328 </td><td>257 </td><td>585 </td></tr><tr><td>On plasmids </td><td>58 </td><td>21 </td><td>79 </td></tr><tr><td>Total </td><td>764 </td><td>389 </td><td>1153 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf4-1"><div class="fn"><p><sup>a</sup>Genes that were absent at least in one strain.</p></div></div><div content-id="tf4-2"><div class="fn"><p><sup>b</sup>Only the singleton genes were analyzed.</p></div></div><div content-id="tf4-3"><div class="fn"><p><sup>c</sup>Genes on the chromosome were classified according to the COG database.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193530" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Categorization and conservation of SS genes in the eight O157 strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th colspan="3">Number of genes </th></tr><tr><th> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Conserved </th><th>VAP<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Total<a reveal-id="tf4-2" data-open="tf4-2" class="link link-ref link-reveal xref-fn">b</a> </th></tr></thead><tbody><tr><td colspan="4">On the chromosome<a reveal-id="tf4-3" data-open="tf4-3" class="link link-ref link-reveal xref-fn">c</a> </td></tr><tr><td>    Energy production and conversion </td><td>6 </td><td>0 </td><td>6 </td></tr><tr><td>    Amino acid transport and metabolism </td><td>4 </td><td>4 </td><td>8 </td></tr><tr><td>    Carbohydrate transport and metabolism </td><td>37 </td><td>1 </td><td>38 </td></tr><tr><td>    Coenzyme transport and metabolism </td><td>5 </td><td>1 </td><td>6 </td></tr><tr><td>    Lipid transport and metabolism </td><td>15 </td><td>0 </td><td>15 </td></tr><tr><td>    Translation </td><td>1 </td><td>0 </td><td>1 </td></tr><tr><td>    Transcription </td><td>30 </td><td>16 </td><td>46 </td></tr><tr><td>    Replication, recombination and repair </td><td>38 </td><td>31 </td><td>69 </td></tr><tr><td>    Cell wall/membrane biogenesis </td><td>32 </td><td>2 </td><td>34 </td></tr><tr><td>    Cell motility </td><td>45 </td><td>1 </td><td>46 </td></tr><tr><td>    Post-translational modification, protein turnover, chaperones </td><td>3 </td><td>5 </td><td>8 </td></tr><tr><td>    Inorganic ion transport and metabolism </td><td>25 </td><td>4 </td><td>29 </td></tr><tr><td>    Secondary metabolites biosynthesis, transport and catabolism </td><td>7 </td><td>1 </td><td>8 </td></tr><tr><td>    Signal transduction mechanisms </td><td>10 </td><td>2 </td><td>12 </td></tr><tr><td>    Intracellular trafficking and secretion </td><td>16 </td><td>1 </td><td>17 </td></tr><tr><td>    Defense mechanisms </td><td>8 </td><td>1 </td><td>9 </td></tr><tr><td>    General function prediction only </td><td>40 </td><td>22 </td><td>62 </td></tr><tr><td>    Function unknown </td><td>56 </td><td>19 </td><td>75 </td></tr><tr><td>    Not in COGs </td><td>328 </td><td>257 </td><td>585 </td></tr><tr><td>On plasmids </td><td>58 </td><td>21 </td><td>79 </td></tr><tr><td>Total </td><td>764 </td><td>389 </td><td>1153 </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th colspan="3">Number of genes </th></tr><tr><th> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Conserved </th><th>VAP<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Total<a reveal-id="tf4-2" data-open="tf4-2" class="link link-ref link-reveal xref-fn">b</a> </th></tr></thead><tbody><tr><td colspan="4">On the chromosome<a reveal-id="tf4-3" data-open="tf4-3" class="link link-ref link-reveal xref-fn">c</a> </td></tr><tr><td>    Energy production and conversion </td><td>6 </td><td>0 </td><td>6 </td></tr><tr><td>    Amino acid transport and metabolism </td><td>4 </td><td>4 </td><td>8 </td></tr><tr><td>    Carbohydrate transport and metabolism </td><td>37 </td><td>1 </td><td>38 </td></tr><tr><td>    Coenzyme transport and metabolism </td><td>5 </td><td>1 </td><td>6 </td></tr><tr><td>    Lipid transport and metabolism </td><td>15 </td><td>0 </td><td>15 </td></tr><tr><td>    Translation </td><td>1 </td><td>0 </td><td>1 </td></tr><tr><td>    Transcription </td><td>30 </td><td>16 </td><td>46 </td></tr><tr><td>    Replication, recombination and repair </td><td>38 </td><td>31 </td><td>69 </td></tr><tr><td>    Cell wall/membrane biogenesis </td><td>32 </td><td>2 </td><td>34 </td></tr><tr><td>    Cell motility </td><td>45 </td><td>1 </td><td>46 </td></tr><tr><td>    Post-translational modification, protein turnover, chaperones </td><td>3 </td><td>5 </td><td>8 </td></tr><tr><td>    Inorganic ion transport and metabolism </td><td>25 </td><td>4 </td><td>29 </td></tr><tr><td>    Secondary metabolites biosynthesis, transport and catabolism </td><td>7 </td><td>1 </td><td>8 </td></tr><tr><td>    Signal transduction mechanisms </td><td>10 </td><td>2 </td><td>12 </td></tr><tr><td>    Intracellular trafficking and secretion </td><td>16 </td><td>1 </td><td>17 </td></tr><tr><td>    Defense mechanisms </td><td>8 </td><td>1 </td><td>9 </td></tr><tr><td>    General function prediction only </td><td>40 </td><td>22 </td><td>62 </td></tr><tr><td>    Function unknown </td><td>56 </td><td>19 </td><td>75 </td></tr><tr><td>    Not in COGs </td><td>328 </td><td>257 </td><td>585 </td></tr><tr><td>On plasmids </td><td>58 </td><td>21 </td><td>79 </td></tr><tr><td>Total </td><td>764 </td><td>389 </td><td>1153 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf4-1"><div class="fn"><p><sup>a</sup>Genes that were absent at least in one strain.</p></div></div><div content-id="tf4-2"><div class="fn"><p><sup>b</sup>Only the singleton genes were analyzed.</p></div></div><div content-id="tf4-3"><div class="fn"><p><sup>c</sup>Genes on the chromosome were classified according to the COG database.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193530" target="_blank">View Large</a></div></div><p>A notable finding was that 16 virulence-related genes exhibited variable distributions (<a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>). They included genes for Stx1, an HmwA-like protein, a TraT-homologue, a HecB-like protein. HmwA is known to be involved in adhesion in <em>Haemophilus influenzae</em>,<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> TraT in serum resistance in enteric bacteria<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> and HecB in hemolysin activation in <em>Neisseria meningitides.</em><a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> One strain lacked the <em>efa-1</em> gene which is involved in the adherence of non-O157 EHEC to cultured epithelial cells<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> and in the inhibition of lymphocyte proliferation and proinflammatory cytokine synthesis in EPEC (called <em>lifA</em> in EPEC).<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> In O157, the <em>efa-1</em> gene is split into two genes (ECs3860 and 3861), but it has been reported that disruption of ECs3860 (<em>efa-1</em>') results in reduced expression and secretion of LEE (the locus of enterocyte effacement)-encoded proteins and in reduced adherence to cultured cells.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> The <em>pchD</em> and <em>pchE</em> genes encoding PerC-like regulators were also missing in four strains and one strain, respectively. Since the other <em>pch</em> genes (<em>pchA</em>, <em>pchB</em> and <em>pchC</em>) have been shown to regulate the LEE gene expression,<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a><em>pchD</em> and <em>pchE</em> may be involved in the regulation of virulence-related genes as well.</p>                        <a id="87193532" scrollto-destination="87193532"></a>
<div content-id="tbl5" class="table-modal"><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Virulence-related genes that were variably present among the eight O157 strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ORF number </th><th>Location </th><th>Gene product </th><th>Description/function </th><th>References </th><th>Conserved in </th></tr></thead><tbody><tr><td>ECs0350 </td><td>S-loop 23 </td><td>HmwA (<em>Haemophilus influenzae</em>)—like protein </td><td>Adhesion </td><td><a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> </td><td>7/8 </td></tr><tr><td>ECs0743 </td><td>S-loop 51 </td><td>Outer membrane usher protein </td><td>Adhesion (fimbrial synthesis) </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>7/8 </td></tr><tr><td>ECs1236 </td><td>Sp5 </td><td>Lom-like protein </td><td>Adhesion/resistance to host immune response </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>5/8 </td></tr><tr><td>ECs1312 </td><td>SpLE1 </td><td>TraT </td><td>Resistance to host immune response </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>7/8 </td></tr><tr><td>ECs1382 </td><td>SpLE1 </td><td>HecB (<em>Neisseria meningitidis</em>)—like protein </td><td>Toxin activation </td><td><a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> </td><td>5/8 </td></tr><tr><td>ECs1388 </td><td>SpLE1 </td><td>PchD </td><td>PerC-like regulator </td><td><a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> </td><td>4/8 </td></tr><tr><td>ECs1588 </td><td>Sp7 </td><td>PchE </td><td>PerC-like regulator </td><td><a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> </td><td>7/8 </td></tr><tr><td>ECs1812 </td><td>Sp9 </td><td>NleA/EspI </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> </td><td>7/8 </td></tr><tr><td>ECs1814 </td><td>Sp9 </td><td>NleH (<em>Citrobactor rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>4/8 </td></tr><tr><td>ECs1815 </td><td>Sp9 </td><td>NleF (<em>C. rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>4/8 </td></tr><tr><td>ECs1828 </td><td>Sp9 </td><td>NleG (<em>C. rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>7/8 </td></tr><tr><td>ECs2973 </td><td>Sp15 </td><td>Stx1B </td><td>Toxin </td><td><a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> </td><td>6/8 </td></tr><tr><td>ECs2974 </td><td>Sp15 </td><td>Stx1A </td><td>Toxin </td><td><a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> </td><td>6/8 </td></tr><tr><td>ECs3860 </td><td>SpLE3 </td><td>Efa1 (N-terminal half) </td><td>Adhesin and other </td><td><a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> </td><td>7/8 </td></tr><tr><td>ECs3861 </td><td>SpLE3 </td><td>Efa1 (C-terminal half) </td><td>Adhesin and other </td><td><a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> </td><td>7/8 </td></tr><tr><td>pO157_76 </td><td>pO157 </td><td>KatP </td><td>Catalase/peroxidase </td><td><a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> </td><td>6/8 </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>ORF number </th><th>Location </th><th>Gene product </th><th>Description/function </th><th>References </th><th>Conserved in </th></tr></thead><tbody><tr><td>ECs0350 </td><td>S-loop 23 </td><td>HmwA (<em>Haemophilus influenzae</em>)—like protein </td><td>Adhesion </td><td><a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> </td><td>7/8 </td></tr><tr><td>ECs0743 </td><td>S-loop 51 </td><td>Outer membrane usher protein </td><td>Adhesion (fimbrial synthesis) </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>7/8 </td></tr><tr><td>ECs1236 </td><td>Sp5 </td><td>Lom-like protein </td><td>Adhesion/resistance to host immune response </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>5/8 </td></tr><tr><td>ECs1312 </td><td>SpLE1 </td><td>TraT </td><td>Resistance to host immune response </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>7/8 </td></tr><tr><td>ECs1382 </td><td>SpLE1 </td><td>HecB (<em>Neisseria meningitidis</em>)—like protein </td><td>Toxin activation </td><td><a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> </td><td>5/8 </td></tr><tr><td>ECs1388 </td><td>SpLE1 </td><td>PchD </td><td>PerC-like regulator </td><td><a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> </td><td>4/8 </td></tr><tr><td>ECs1588 </td><td>Sp7 </td><td>PchE </td><td>PerC-like regulator </td><td><a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> </td><td>7/8 </td></tr><tr><td>ECs1812 </td><td>Sp9 </td><td>NleA/EspI </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> </td><td>7/8 </td></tr><tr><td>ECs1814 </td><td>Sp9 </td><td>NleH (<em>Citrobactor rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>4/8 </td></tr><tr><td>ECs1815 </td><td>Sp9 </td><td>NleF (<em>C. rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>4/8 </td></tr><tr><td>ECs1828 </td><td>Sp9 </td><td>NleG (<em>C. rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>7/8 </td></tr><tr><td>ECs2973 </td><td>Sp15 </td><td>Stx1B </td><td>Toxin </td><td><a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> </td><td>6/8 </td></tr><tr><td>ECs2974 </td><td>Sp15 </td><td>Stx1A </td><td>Toxin </td><td><a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> </td><td>6/8 </td></tr><tr><td>ECs3860 </td><td>SpLE3 </td><td>Efa1 (N-terminal half) </td><td>Adhesin and other </td><td><a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> </td><td>7/8 </td></tr><tr><td>ECs3861 </td><td>SpLE3 </td><td>Efa1 (C-terminal half) </td><td>Adhesin and other </td><td><a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> </td><td>7/8 </td></tr><tr><td>pO157_76 </td><td>pO157 </td><td>KatP </td><td>Catalase/peroxidase </td><td><a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> </td><td>6/8 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193532" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Virulence-related genes that were variably present among the eight O157 strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ORF number </th><th>Location </th><th>Gene product </th><th>Description/function </th><th>References </th><th>Conserved in </th></tr></thead><tbody><tr><td>ECs0350 </td><td>S-loop 23 </td><td>HmwA (<em>Haemophilus influenzae</em>)—like protein </td><td>Adhesion </td><td><a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> </td><td>7/8 </td></tr><tr><td>ECs0743 </td><td>S-loop 51 </td><td>Outer membrane usher protein </td><td>Adhesion (fimbrial synthesis) </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>7/8 </td></tr><tr><td>ECs1236 </td><td>Sp5 </td><td>Lom-like protein </td><td>Adhesion/resistance to host immune response </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>5/8 </td></tr><tr><td>ECs1312 </td><td>SpLE1 </td><td>TraT </td><td>Resistance to host immune response </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>7/8 </td></tr><tr><td>ECs1382 </td><td>SpLE1 </td><td>HecB (<em>Neisseria meningitidis</em>)—like protein </td><td>Toxin activation </td><td><a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> </td><td>5/8 </td></tr><tr><td>ECs1388 </td><td>SpLE1 </td><td>PchD </td><td>PerC-like regulator </td><td><a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> </td><td>4/8 </td></tr><tr><td>ECs1588 </td><td>Sp7 </td><td>PchE </td><td>PerC-like regulator </td><td><a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> </td><td>7/8 </td></tr><tr><td>ECs1812 </td><td>Sp9 </td><td>NleA/EspI </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> </td><td>7/8 </td></tr><tr><td>ECs1814 </td><td>Sp9 </td><td>NleH (<em>Citrobactor rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>4/8 </td></tr><tr><td>ECs1815 </td><td>Sp9 </td><td>NleF (<em>C. rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>4/8 </td></tr><tr><td>ECs1828 </td><td>Sp9 </td><td>NleG (<em>C. rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>7/8 </td></tr><tr><td>ECs2973 </td><td>Sp15 </td><td>Stx1B </td><td>Toxin </td><td><a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> </td><td>6/8 </td></tr><tr><td>ECs2974 </td><td>Sp15 </td><td>Stx1A </td><td>Toxin </td><td><a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> </td><td>6/8 </td></tr><tr><td>ECs3860 </td><td>SpLE3 </td><td>Efa1 (N-terminal half) </td><td>Adhesin and other </td><td><a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> </td><td>7/8 </td></tr><tr><td>ECs3861 </td><td>SpLE3 </td><td>Efa1 (C-terminal half) </td><td>Adhesin and other </td><td><a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> </td><td>7/8 </td></tr><tr><td>pO157_76 </td><td>pO157 </td><td>KatP </td><td>Catalase/peroxidase </td><td><a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> </td><td>6/8 </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>ORF number </th><th>Location </th><th>Gene product </th><th>Description/function </th><th>References </th><th>Conserved in </th></tr></thead><tbody><tr><td>ECs0350 </td><td>S-loop 23 </td><td>HmwA (<em>Haemophilus influenzae</em>)—like protein </td><td>Adhesion </td><td><a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> </td><td>7/8 </td></tr><tr><td>ECs0743 </td><td>S-loop 51 </td><td>Outer membrane usher protein </td><td>Adhesion (fimbrial synthesis) </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>7/8 </td></tr><tr><td>ECs1236 </td><td>Sp5 </td><td>Lom-like protein </td><td>Adhesion/resistance to host immune response </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>5/8 </td></tr><tr><td>ECs1312 </td><td>SpLE1 </td><td>TraT </td><td>Resistance to host immune response </td><td><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> </td><td>7/8 </td></tr><tr><td>ECs1382 </td><td>SpLE1 </td><td>HecB (<em>Neisseria meningitidis</em>)—like protein </td><td>Toxin activation </td><td><a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> </td><td>5/8 </td></tr><tr><td>ECs1388 </td><td>SpLE1 </td><td>PchD </td><td>PerC-like regulator </td><td><a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> </td><td>4/8 </td></tr><tr><td>ECs1588 </td><td>Sp7 </td><td>PchE </td><td>PerC-like regulator </td><td><a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> </td><td>7/8 </td></tr><tr><td>ECs1812 </td><td>Sp9 </td><td>NleA/EspI </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> </td><td>7/8 </td></tr><tr><td>ECs1814 </td><td>Sp9 </td><td>NleH (<em>Citrobactor rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>4/8 </td></tr><tr><td>ECs1815 </td><td>Sp9 </td><td>NleF (<em>C. rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>4/8 </td></tr><tr><td>ECs1828 </td><td>Sp9 </td><td>NleG (<em>C. rodentium</em>)—like protein </td><td>Non-LEE TTSS effector </td><td><a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> </td><td>7/8 </td></tr><tr><td>ECs2973 </td><td>Sp15 </td><td>Stx1B </td><td>Toxin </td><td><a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> </td><td>6/8 </td></tr><tr><td>ECs2974 </td><td>Sp15 </td><td>Stx1A </td><td>Toxin </td><td><a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> </td><td>6/8 </td></tr><tr><td>ECs3860 </td><td>SpLE3 </td><td>Efa1 (N-terminal half) </td><td>Adhesin and other </td><td><a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> </td><td>7/8 </td></tr><tr><td>ECs3861 </td><td>SpLE3 </td><td>Efa1 (C-terminal half) </td><td>Adhesin and other </td><td><a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> </td><td>7/8 </td></tr><tr><td>pO157_76 </td><td>pO157 </td><td>KatP </td><td>Catalase/peroxidase </td><td><a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> </td><td>6/8 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193532" target="_blank">View Large</a></div></div><p>More importantly, several genes encoding type III secretion system (TTSS) effectors exhibited variable distributions. Recently, several proteins encoded on the non-LEE loci have been identified as effectors secreted by the LEE-encoded TTSS (non-LEE encoded effectors) in EPEC, EHEC and <em>Citrobactor rodentium</em>.<a reveal-id="b20  b25" data-open="b20  b25" class="link link-ref link-reveal xref-bibr">20–25</a> O157 Sakai contains at least 19 non-LEE encoded effectors or their homologues. Among these, <em>nleA</em> (also named <em>espI</em>, ECs1812), an <em>nleF</em>-homologue (ECs1815), an <em>nleG</em>-homologue (ECs1828) and an <em>nleH</em>-homologue (ECs1814) were variably present among the eight O157 strains. Functions of these effectors have not been well elucidated, but it has been shown that NleA/EspI injected into the host cell localizes to the Golgi apparatus.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a></p><p>Many virulence-related genes were also encoded on the pO157 plasmid, but all were conserved except for <em>katP</em> (pO157_76). Of interest was that 18 out of the 83 genes on pO157 were missing in Strain #2 (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Since the pO157 plasmid of #2 is almost the same in size as those from other strains (data not shown), several parts of the plasmid have probably been replaced by different or highly divergent sequences. pOSAK1, a small cryptic plasmid, was present only in two strains.</p>                            <h3 scrollto-destination=87193535 id="87193535" class="section-title" >3.4. Comparison of two datasets obtained by microarray and WGPScanning analyses</h3>
<p>The CGH analysis using microarray provides the information on the gene composition but not their genomic positions, thus translocation events specific to the test strain cannot be detected. It also provides no information on strain-specific insertions. Conversely, WGPScanning detects possible loci where such genetic events have taken place while the presence or absence of each gene cannot be determined. Therefore, in order to know more details on the genomic diversity of O157, we compared the two datasets, one from the present microarray analysis and the other from the previous WGPScanning<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>).</p><p>In the segments that exhibited no size variation in the WGPScanning analysis (indicated in gray in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>), no deletion of genes was detected by microarray except for only five segments. This confirmed that these regions possess the same genomic structures as those in Sakai. Among the five segments where deletions of a gene(s) were observed, four were derived from lambda-like prophage regions, implying that some parts of these prophages have been replaced by DNA fragments with same sizes but with different or highly divergent sequences.</p><p>By the WGPScanning analysis, a total of 35 segments with larger size reduction (≥2 kb smaller than that of O157 Sakai) were identified from the 8 O157 strains (indicated in yellow in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). In most cases (29 segments), deleted genes were identified. The regions with gene deletion, however, were often associated with repeated genes derived from IS elements or lambda-like prophages, and thus precise boundaries of the deletion events were not defined in many cases. In six segments, no apparent gene deletion was detected, indicating that several genes on these segments have probably translocated to other genomic loci.</p><p>Among the 60 segments with smaller size reduction (&lt;2 kb), which were indicated in light yellow in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, we detected deletion of a gene(s) only in seven segments by the microarray analysis. Most of the segments with undetectable deletions (50 out of 53) contained IS elements and/or parts of lambda-like prophages, indicating that small deletions involving these elements frequently occur on O157 genomes. In the remaining three segments, small deletions in the regions that were not represented by probes or some translocations have probably taken place.</p><p>Besides the above-mentioned genomic segments that apparently contain deletions, 20 loci have been identified by the WGPScanning analysis, where prophages or prophage-like elements are integrated in Sakai but deleted (or not integrated) in some other strains (indicated by blank in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). These loci were classified into three groups according to the patterns of gene conservation observed in the microarray analysis. The first group included the Sp7 region of strain #2, Sp13 (P2-like phage) of strains #3 and #9, and Sp18 (Mu-like phage) of strains #2, #3, #5, #6, #7, #8 and #9. In these regions, all or nearly all genes were absent. We concluded that these genetic elements are completely missing in these strains. The second group included the Sp18 region of strain #4 and SpLE1 of strain #6. In both cases, most genes on the elements were conserved, indicating that a prophage or prophage-like element almost identical to Sp18 or SpLE1 is present in other locus. We have already identified alternative integration sites for each element (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). The third group included the Sp5 region of strains #2, #3, #4, #6, #7 and #8, Sp1/Sp2 of strain #8 and Sp7 of strain #3. In these regions, genes were partially conserved, exhibiting mosaic conservation patterns. This indicates that these strains contain, in some other loci, prophages which are significantly diverged but with some structural similarity to Sp5, Sp1/Sp2 or Sp7. Of particular interest was Sp5, the Stx2 phage of O157 Sakai. The Sp5 region was one of the regions with the highest level of variation, and we have already identified the alternative integration sites for the Stx2 phages in these six strains (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). These Stx2 phages are thus different from Sp5 not only in the integration site but also in the genomic structure.</p><p>A total of 83 segments yielded PCR products larger than those of Sakai in the WGPScanning analysis as shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, where the segments with larger increments (≥2 kb) are indicated in blue, and those with smaller increments (&lt;2 kb) in light blue. These segments must contain some insertions. In addition, as many as 120 segments were not amplified by PCR (indicated in red in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>), in which large insertions or some other types of large genomic rearrangements likely have taken place. In these cases, if all genes on a segment were conserved, the segment most likely contains a simple insertion. By comparing the two datasets, 66 such segments were identified. Among these, we have identified seven new phage integration sites including those for Stx1 and Stx2 phages (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). In other cases, more complicated genetic events must have occurred.</p>                            <h3 scrollto-destination=87193542 id="87193542" class="section-title" >3.5. Unusual signal reductions around the IS insertion sites</h3>
<p>During the data comparison of microarray and WGPScanning analysis, we noticed that somewhat unusual signal reductions occurred in the ETT2 region. In strains #2, #4 and #7, the region yielded larger PCR products, but a block of genes were judged as absent in the microarray analysis. Sequencing analyses of these regions, however, indicated that all genes were completely conserved but one copy of IS<em>Ec8</em> was newly inserted in each strain. A closer inspection of the microarray data revealed that, although the IS<em>Ec8</em> insertion sites were different between the strains, the genes judged as absent were all located in the 2–4 kb regions surrounding each IS<em>Ec8</em>, and that their signal ratios were just slightly lower than the cutoff value (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>). This phenomenon was specific to the ETT2 region, and observed even after the labeled DNA was fragmented by sonication. Although we do not know the mechanism underlying this phenomenon, some caution is required to interpret the microarray data when a block of genes exhibit a moderate level of signal reduction.</p>                        <a id="87193544" scrollto-destination="87193544"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/m_dsi026f2.jpeg?Expires=2147483647&Signature=vOv3Z4Y~5Avwt1cKd0axky9SMfYV-8~T9ZB2HrdKxn568tPpzE24kpjM0gJ6xbRIWUP3xYYW6iUfhq9Q2sZN8jSiIIjfLwcUzjP5GheeuSjk8hBHnFsP22D-NM31GXwmAVCnA~R5Si4B3cOSRI8oexoxZEijbvubTI4G2aEc-22XhLylyWtz3BgCouDpJ50MPQsbq1-zg2G79RH7H6N3seOHaWR4lUQiy2oibPYSlWzHRzqeuzuC7~e50ol61xg5hk6rr7OA7sRnXAAhsBKzzaJrFoPQoKp1l-wWRYCyBgIHcN~Tqy8NB-mTWXoYw3g1Od~mN6SF3f2~QrJwyaV93w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Unusual signal reduction associated with insertions of ISEc8. Distribution of signal ratios (relative to O157 Sakai) from the probes representing the ETT2 regions of strains #2, #4, #7 and #8 is shown. Genes in the ETT2 region (ECs3703–3737) are depicted by black arrows, and genes on ISEc8 newly inserted in strains #2, #4 and #7 by gray arrows. Positions of each gene and probe are drawn to the O157 Sakai genome sequence. Horizontal broken lines indicate cutoff lines for the presence or absence determination in each strain. We have confirmed that all the probe sequences exhibiting unusual signal reduction are identical to those of Sakai. As seen in strain #8, no signal reduction was observed in other strains without ISEc8-insertion." path-from-xml="dsi026f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193544/dsi026f2.jpeg" path-from-xml="dsi026f2.jpeg" target="_blank">View large</a><a section="87193544" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/dsi026f2.jpeg?Expires=2147483647&Signature=bgeM0Cw7-e0gjMshbqlyYMD4AfOAkL29pFf8tW3lDdLBWPkijw4YKUsZ8mBe8BtFC1Odzpb1pO0BzN7FfaHudBHGGOs45qfdNhAp7YFG7F-~UaTYAhlLHlPapAo4h9XY53zjEd4HElT~r7MwY-Z7-wSeW3NsJmqse6L3hbrOOkiKErzu-yZwbtzQVo-pNNyQvy92rhBTu-t0I0FN1z-I-plFe4t0Dv0jmHviMc2eepDkPrFL1rpbqimLpGG2x~ngJDeb8Af~rQGDYoikxdXDD5eJNWk9~vyLCXck2RC8ucW79XB7ySFj-~y-yIbcGanJs6A9pfzCAtVlUMOyPPVvfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193544&ar=486680&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi026f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Unusual signal reduction associated with insertions of IS<em>Ec8</em>. Distribution of signal ratios (relative to O157 Sakai) from the probes representing the ETT2 regions of strains #2, #4, #7 and #8 is shown. Genes in the ETT2 region (ECs3703–3737) are depicted by black arrows, and genes on IS<em>Ec8</em> newly inserted in strains #2, #4 and #7 by gray arrows. Positions of each gene and probe are drawn to the O157 Sakai genome sequence. Horizontal broken lines indicate cutoff lines for the presence or absence determination in each strain. We have confirmed that all the probe sequences exhibiting unusual signal reduction are identical to those of Sakai. As seen in strain #8, no signal reduction was observed in other strains without IS<em>Ec8</em>-insertion.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/m_dsi026f2.jpeg?Expires=2147483647&Signature=vOv3Z4Y~5Avwt1cKd0axky9SMfYV-8~T9ZB2HrdKxn568tPpzE24kpjM0gJ6xbRIWUP3xYYW6iUfhq9Q2sZN8jSiIIjfLwcUzjP5GheeuSjk8hBHnFsP22D-NM31GXwmAVCnA~R5Si4B3cOSRI8oexoxZEijbvubTI4G2aEc-22XhLylyWtz3BgCouDpJ50MPQsbq1-zg2G79RH7H6N3seOHaWR4lUQiy2oibPYSlWzHRzqeuzuC7~e50ol61xg5hk6rr7OA7sRnXAAhsBKzzaJrFoPQoKp1l-wWRYCyBgIHcN~Tqy8NB-mTWXoYw3g1Od~mN6SF3f2~QrJwyaV93w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Unusual signal reduction associated with insertions of ISEc8. Distribution of signal ratios (relative to O157 Sakai) from the probes representing the ETT2 regions of strains #2, #4, #7 and #8 is shown. Genes in the ETT2 region (ECs3703–3737) are depicted by black arrows, and genes on ISEc8 newly inserted in strains #2, #4 and #7 by gray arrows. Positions of each gene and probe are drawn to the O157 Sakai genome sequence. Horizontal broken lines indicate cutoff lines for the presence or absence determination in each strain. We have confirmed that all the probe sequences exhibiting unusual signal reduction are identical to those of Sakai. As seen in strain #8, no signal reduction was observed in other strains without ISEc8-insertion." path-from-xml="dsi026f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193544/dsi026f2.jpeg" path-from-xml="dsi026f2.jpeg" target="_blank">View large</a><a section="87193544" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/dsi026f2.jpeg?Expires=2147483647&Signature=bgeM0Cw7-e0gjMshbqlyYMD4AfOAkL29pFf8tW3lDdLBWPkijw4YKUsZ8mBe8BtFC1Odzpb1pO0BzN7FfaHudBHGGOs45qfdNhAp7YFG7F-~UaTYAhlLHlPapAo4h9XY53zjEd4HElT~r7MwY-Z7-wSeW3NsJmqse6L3hbrOOkiKErzu-yZwbtzQVo-pNNyQvy92rhBTu-t0I0FN1z-I-plFe4t0Dv0jmHviMc2eepDkPrFL1rpbqimLpGG2x~ngJDeb8Af~rQGDYoikxdXDD5eJNWk9~vyLCXck2RC8ucW79XB7ySFj-~y-yIbcGanJs6A9pfzCAtVlUMOyPPVvfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193544&ar=486680&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi026f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Unusual signal reduction associated with insertions of IS<em>Ec8</em>. Distribution of signal ratios (relative to O157 Sakai) from the probes representing the ETT2 regions of strains #2, #4, #7 and #8 is shown. Genes in the ETT2 region (ECs3703–3737) are depicted by black arrows, and genes on IS<em>Ec8</em> newly inserted in strains #2, #4 and #7 by gray arrows. Positions of each gene and probe are drawn to the O157 Sakai genome sequence. Horizontal broken lines indicate cutoff lines for the presence or absence determination in each strain. We have confirmed that all the probe sequences exhibiting unusual signal reduction are identical to those of Sakai. As seen in strain #8, no signal reduction was observed in other strains without IS<em>Ec8</em>-insertion.</p></div></div>                            <h2 scrollto-destination=87193545 id="87193545" class="section-title" >4. Discussion</h2>
<p>Presence of an unexpectedly high degree of structural diversity in O157 genomes has been revealed by our previous analysis using the WGPScanning method.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> The results also suggested that the major factor generating such diversity is the variation of prophages and prophage-like elements. However, it remained to be elucidated how the genomic diversity detected by WGPScanning affects the gene content of each strain. In order to know the variation of their gene composition, we constructed an oligoDNA microarray according to the genome sequence of O157 Sakai,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> and performed the CGH analysis of the same set of O157 strains as examined by the WGPScanning.</p><p>By this analysis, we identified 431 VAP genes that were variably present in the 8 O157 strains, and found that most of them were on S-loops, particularly on the prophage regions. This indicates that a remarkable level of variation in the gene repertoire exist among the O157 strains, and confirms the role of bacteriophages as the key players in the genome diversification of O157. Very recently, Wick <em>et al</em>.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> have reported the results of CGH analyses of several types of O157 strains and its close relatives, including typical O157:H7 strains (GUD<sup>−</sup>, SOR<sup>−</sup>), atypical O157:H7 strains (GUD<sup>+</sup>, SOR<sup>−</sup>), O157:H<sup>−</sup> German clones (GUD<sup>+</sup>, SOR<sup>+</sup>) and O55:H7 strains (the closest relatives of O157:H7<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a>). They used a multigenome array containing 50 mer oligonucleotide probes that target 6176 ORFs on the chromosomes of K-12, O157 EDL933 and O157 Sakai. By this analysis, they identified the Sakai and K-12 genes that were gained or lost during the emergence of O157:H7 from its O55:H7-like ancestor. Many of these genes were parts of prophages and prophage-like elements, suggesting complex histories of these elements during the evolution of O157<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a>. What we should emphasize here is, however, that all O157 strains but strain #2 examined in our present study belonged to the typical O157 group. Even if the 89 genes that were absent only in strain #2 were excluded, 350 genes still exhibited variable distribution. This indicates that there is a remarkable variation in gene content even among typical O157 strains.</p><p>Identification of these VAP genes is of medical importance in that they could be used as valuable genetic markers to discriminate O157 strains. For example, a mini-DNA chip featuring these genes would become a useful molecular tool for epidemiological studies of O157. Of more importance may be that the VAP genes include a considerable number of virulence-related genes. Although the variation of <em>stx1</em> and <em>stx2</em> genes among O157 strains is known, additional 14 potentially virulence-related genes were found to display variable distributions. They include four non-LEE TTSS effectors, indicating that, although the LEE locus encoding the TTS machinery is highly conserved, the repertoire of its effectors differs significantly among O157 strains. Some metabolic genes, such as those for urease production (ECs1321–1326) and iron acquisition (ECs1693–1699), which may be indirectly involved in the pathogenesis of O157, were also found to be variably present. These data raise a possibility that some variation in the potential virulence and infectivity exists among O157 strains, and we need to analyze more O157 strains from various sources, including environmental isolates.</p><p>The phylogenetic position of strain #2 in the O157 lineage is somewhat obscure. But, since weak β-glucuronidase activity was detected after long cultivation, the strain may belong to the GUD<sup>+</sup> O157:H7 group. The largest number of VAP genes was detected in this strain, and it shared many, if not all, features that Wick <em>et al</em>.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> described as being specific to GUD<sup>+</sup> strains. For example, several gene blocks, such as ECs4134–4139, ECs1611–1619 and ECs3860–3861, were deleted (or highly divergent) only in this strain. The gene content on the pO157 plasmid of strain #2 also differed significantly from other strains. Unfortunately Wick <em>et al</em>.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> did not examine the pO157 genes, but the difference in pO157 may be another genomic feature that discriminates the GUD<sup>+</sup> group from typical O157 : H7 strains.</p><p>In the present study, we examined the same set of O157 strains as that were analyzed previously by the WGPScanning method. This provided a good opportunity to compare the data obtained by the two methods (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Although the microarray is now widely used as a major tool in comparative genomics, a fine comparison of the two datasets rather highlighted the weakness of microarray and emphasized the usefulness of a combinational approach with WGPScanning. Microarray actually provided valuable genome-wide information on the genome composition in each test strain, but almost no information on strain-specific insertions and translocations. In the case of O157, particular cautions are required in interpreting the microarray data because of the presence of same or very similar prophages as well as many IS elements. Even with such difficulties, all simple insertions as well as many translocations and replacements have been identified by comparing the two datasets. This approach also identified many small deletions that could not be detected by microarray alone. IS elements were frequently associated with such small deletions, suggesting that IS elements may be another key player to diversify O157 genomes. The combinational use of microarray and WGPScanning overcame the weakness of each method alone, and provided us more details and thus a much complex view on the genomic diversity of O157 strains.</p><p>The variation of prophages among O157 strains was not fully illustrated even with this combinational approach. But, as most strikingly shown for the Stx2-transducing phages, the data clearly indicated that prophages of O157 exhibit an extremely high level of strain-to-strain variations. These variations appeared to have been generated not only by simple insertion, deletion or translocation of prophages but also by more complex genetic events involving local recombination and deletion of prophage segments. One such example has been documented by Shaikh and Tarr<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> in the prophages integrated into the <em>yehV</em> locus (corresponding to Sp15, the Stx1-transducing phage of O157 Sakai). Of importance again is that most of the strains examined here were typical O157 : H7 strains. This indicates that such dynamic turnover of prophages is still actively taking place, and as we have proposed previously, O157 strains are functioning as a kind of ‘phage factory’, releasing a variety of new recombinant bacteriophages into the environment.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> This process should be significantly involved in the generation of variable gene contents of O157 strains detected by the CGH analysis.</p><p>This work was supported by Research for the Future Program funded by the Japan Society for the Promotion of Science (JSPS-RFTF00L01411), Grant-in-Aids for Scientific Research on Priority Areas and the 21st Century COE (Centers of Excellence) Program (Life Science) from the Ministry of Education, Culture, Sports, Science and Technology, Japan and a grant from the Yakult Foundation. We thank Dr Taku Ohshima for his valuable advice, Yoshiyuki Maki for his assistance in primer design, Akemi Yoshida and Yumiko Takeshita for their technical assistances, and Yumiko Hayashi for her language assistance.</p>                            <h2 scrollto-destination=87193554 id="87193554" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">J. R.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Donnenberg</div> <div class="given-names">M. S.</div></div>. </div><div class="article-title">Evolution of pathogenic <em>Escherichia coli</em></div>, <div class="year">2002</div><div class="publisher-loc">San Diego, Calif</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">55</div>-<div class="lpage">77</div>) <div class="comment"><em>Escherichia coli</em>: Virulence Mechanisms of a Versatile Pathogen</div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaper</div> <div class="given-names">J. B.</div></div>,  <div class="name"><div class="surname">Nataro</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Mobley</div> <div class="given-names">H. L.</div></div>. </div><div class="article-title">Pathogenic <em>Escherichia coli</em></div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">123</div>-<div class="lpage">140</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pathogenic%20Escherichia%20coli&amp;author=J.%20B.%20Kaper&amp;author=J.%20P.%20Nataro&amp;author=H.%20L.%20Mobley&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=2&amp;pages=123-140" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro818" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro818" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro818"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15040260" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mead</div> <div class="given-names">P. S.</div></div>,  <div class="name"><div class="surname">Griffin</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title"><em>Escherichia coli</em> O157:H7</div>, <div class="source">Lancet</div> , <div class="year">1998</div>, vol. <div class="volume">352</div> (pg. <div class="fpage">1207</div>-<div class="lpage">1212</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20O157%3AH7&amp;author=P.%20S.%20Mead&amp;author=P.%20M.%20Griffin&amp;publication_year=1998&amp;journal=Lancet&amp;volume=352&amp;pages=1207-1212" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0140-6736(98)01267-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0140-6736(98)01267-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0140-6736(98)01267-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9777854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blattner</div> <div class="given-names">F. R.</div></div>,  <div class="name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>3rd</div>,  <div class="name"><div class="surname">Bloch</div> <div class="given-names">C. A.</div></div>, et al. </div><div class="article-title">The complete genome sequence of <em>Escherichia coli</em> K-12</div>, <div class="source">Science</div> , <div class="year">1997</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">1453</div>-<div class="lpage">1474</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12&amp;author=F.%20R.%20Blattner&amp;author=G.%20Plunkett&amp;author=C.%20A.%20Bloch&amp;publication_year=1997&amp;journal=Science&amp;volume=277&amp;pages=1453-1474" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.277.5331.1453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.277.5331.1453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Makino</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Complete genome sequence of enterohemorrhagic <em>Escherichia coli</em> O157:H7 and genomic comparison with a laboratory strain K-12</div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">11</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20enterohemorrhagic%20Escherichia%20coli%20O157%3AH7%20and%20genomic%20comparison%20with%20a%20laboratory%20strain%20K-12&amp;author=T.%20Hayashi&amp;author=K.%20Makino&amp;author=M.%20Ohnishi&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=11-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.1.11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.1.11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.1.11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11258796" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Diversification of <em>Escherichia coli</em> genomes: are bacteriophages the major contributors?</div>, <div class="source">Trends Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">481</div>-<div class="lpage">485</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20Escherichia%20coli%20genomes%3A%20are%20bacteriophages%20the%20major%20contributors%3F&amp;author=M.%20Ohnishi&amp;author=K.%20Kurokawa&amp;author=T.%20Hayashi&amp;publication_year=2001&amp;journal=Trends%20Microbiol.&amp;volume=9&amp;pages=481-485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(01)02173-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(01)02173-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(01)02173-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11597449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perna</div> <div class="given-names">N.T.</div></div>,  <div class="name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>3rd</div>,  <div class="name"><div class="surname">Burland</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Genome sequence of enterohaemorrhagic <em>Escherichia coli</em> O157:H7</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">529</div>-<div class="lpage">533</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20enterohaemorrhagic%20Escherichia%20coli%20O157%3AH7&amp;author=N.T.%20Perna&amp;author=G.%20Plunkett&amp;author=V.%20Burland&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=529-533" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35054089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35054089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35054089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11206551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Terajima</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genomic diversity of enterohemorrhagic <em>Escherichia coli</em> O157 revealed by whole genome PCR scanning</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">17043</div>-<div class="lpage">17048</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20diversity%20of%20enterohemorrhagic%20Escherichia%20coli%20O157%20revealed%20by%20whole%20genome%20PCR%20scanning&amp;author=M.%20Ohnishi&amp;author=J.%20Terajima&amp;author=K.%20Kurokawa&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=17043-17048" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.262441699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.262441699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.262441699"> </span></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">C. C.</div></div>,  <div class="name"><div class="surname">Joyce</div> <div class="given-names">E. A.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Falkow</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Improved analytical methods for microarray-based genome-composition analysis</div>, <div class="source">Genome Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div>  <div class="comment">RESEARCH0065</div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eisen</div> <div class="given-names">M. B.</div></div>,  <div class="name"><div class="surname">Spellman</div> <div class="given-names">P. T.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div>,  <div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Cluster analysis and display of genome-wide expression patterns</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">14863</div>-<div class="lpage">14868</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cluster%20analysis%20and%20display%20of%20genome-wide%20expression%20patterns&amp;author=M.%20B.%20Eisen&amp;author=P.%20T.%20Spellman&amp;author=P.%20O.%20Brown&amp;author=D.%20Botstein&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=14863-14868" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.25.14863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.25.14863"> </span></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Makino</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ishii</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yasunaga</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Complete nucleotide sequences of 93-kb and 3.3-kb plasmids of an enterohemorrhagic <em>Escherichia coli</em> O157:H7 derived from Sakai outbreak</div>, <div class="source">DNA Res.</div> , <div class="year">1998</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20nucleotide%20sequences%20of%2093-kb%20and%203.3-kb%20plasmids%20of%20an%20enterohemorrhagic%20Escherichia%20coli%20O157%3AH7%20derived%20from%20Sakai%20outbreak&amp;author=K.%20Makino&amp;author=K.%20Ishii&amp;author=T.%20Yasunaga&amp;publication_year=1998&amp;journal=DNA%20Res.&amp;volume=5&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/5.1.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F5.1.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F5.1.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9628576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kolodner</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hall</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Luisi-DeLuca</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Homologous pairing proteins encoded by the <em>Escherichia coli</em> recE and recT genes</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1994</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">23</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homologous%20pairing%20proteins%20encoded%20by%20the%20Escherichia%20coli%20recE%20and%20recT%20genes&amp;author=R.%20Kolodner&amp;author=S.%20D.%20Hall&amp;author=C.%20Luisi-DeLuca&amp;publication_year=1994&amp;journal=Mol.%20Microbiol.&amp;volume=11&amp;pages=23-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1994.11.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1994.11.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1994.11.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8145642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barenkamp</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Leininger</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Cloning, expression, and DNA sequence analysis of genes encoding nontypeable <em>Haemophilus influenzae</em> high-molecular-weight surface-exposed proteins related to filamentous hemagglutinin of <em>Bordetella pertussis</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">1992</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">1302</div>-<div class="lpage">1313</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%2C%20expression%2C%20and%20DNA%20sequence%20analysis%20of%20genes%20encoding%20nontypeable%20Haemophilus%20influenzae%20high-molecular-weight%20surface-exposed%20proteins%20related%20to%20filamentous%20hemagglutinin%20of%20Bordetella%20pertussis&amp;author=S.%20J.%20Barenkamp&amp;author=E.%20Leininger&amp;publication_year=1992&amp;journal=Infect.%20Immun.&amp;volume=60&amp;pages=1302-1313" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1548058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pramoonjago</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Role of TraT protein, an anticomplementary protein produced in <em>Escherichia coli</em> by R100 factor, in serum resistance</div>, <div class="source">J. Immunol.</div> , <div class="year">1992</div>, vol. <div class="volume">148</div> (pg. <div class="fpage">827</div>-<div class="lpage">836</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20TraT%20protein%2C%20an%20anticomplementary%20protein%20produced%20in%20Escherichia%20coli%20by%20R100%20factor%2C%20in%20serum%20resistance&amp;author=P.%20Pramoonjago&amp;author=M.%20Kaneko&amp;author=T.%20Kinoshita&amp;publication_year=1992&amp;journal=J.%20Immunol.&amp;volume=148&amp;pages=827-836" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1730875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tettelin</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Saunders</div> <div class="given-names">N. J.</div></div>,  <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Neisseria meningitidis</em> serogroup B strain MC58</div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">287</div> (pg. <div class="fpage">1809</div>-<div class="lpage">1815</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Neisseria%20meningitidis%20serogroup%20B%20strain%20MC58&amp;author=H.%20Tettelin&amp;author=N.%20J.%20Saunders&amp;author=J.%20Heidelberg&amp;publication_year=2000&amp;journal=Science&amp;volume=287&amp;pages=1809-1815" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5459.1809" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5459.1809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5459.1809"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10710307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nicholls</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Grant</div> <div class="given-names">T. H.</div></div>,  <div class="name"><div class="surname">Robins-Browne</div> <div class="given-names">R. M.</div></div>. </div><div class="article-title">Identification of a novel genetic locus that is required for <em>in vitro</em> adhesion of a clinical isolate of enterohaemorrhagic <em>Escherichia coli</em> to epithelial cells</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">275</div>-<div class="lpage">288</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20novel%20genetic%20locus%20that%20is%20required%20for%20in%20vitro%20adhesion%20of%20a%20clinical%20isolate%20of%20enterohaemorrhagic%20Escherichia%20coli%20to%20epithelial%20cells&amp;author=L.%20Nicholls&amp;author=T.%20H.%20Grant&amp;author=R.%20M.%20Robins-Browne&amp;publication_year=2000&amp;journal=Mol.%20Microbiol.&amp;volume=35&amp;pages=275-288" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01690.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2000.01690.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2000.01690.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10652089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klapproth</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Scaletsky</div> <div class="given-names">I. C.</div></div>,  <div class="name"><div class="surname">McNamara</div> <div class="given-names">B. P.</div></div>, et al. </div><div class="article-title">A large toxin from pathogenic <em>Escherichia coli</em> strains that inhibits lymphocyte activation</div>, <div class="source">Infect Immun.</div> , <div class="year">2000</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">2148</div>-<div class="lpage">2155</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large%20toxin%20from%20pathogenic%20Escherichia%20coli%20strains%20that%20inhibits%20lymphocyte%20activation&amp;author=J.%20M.%20Klapproth&amp;author=I.%20C.%20Scaletsky&amp;author=B.%20P.%20McNamara&amp;publication_year=2000&amp;journal=Infect%20Immun.&amp;volume=68&amp;pages=2148-2155" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.68.4.2148-2155.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.68.4.2148-2155.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.68.4.2148-2155.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10722613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stevens</div> <div class="given-names">M. P.</div></div>,  <div class="name"><div class="surname">Roe</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Vlisidou</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Mutation of <em>toxB</em> and a truncated version of the <em>efa-1</em> gene in <em>Escherichia coli</em> O157:H7 influences the expression and secretion of locus of enterocyte effacement-encoded proteins but not intestinal colonization in calves or sheep</div>, <div class="source">Infect. Immun.</div> , <div class="year">2004</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">5402</div>-<div class="lpage">5411</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20of%20toxB%20and%20a%20truncated%20version%20of%20the%20efa-1%20gene%20in%20Escherichia%20coli%20O157%3AH7%20influences%20the%20expression%20and%20secretion%20of%20locus%20of%20enterocyte%20effacement-encoded%20proteins%20but%20not%20intestinal%20colonization%20in%20calves%20or%20sheep&amp;author=M.%20P.%20Stevens&amp;author=A.%20J.%20Roe&amp;author=I.%20Vlisidou&amp;publication_year=2004&amp;journal=Infect.%20Immun.&amp;volume=72&amp;pages=5402-5411" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.72.9.5402-5411.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.72.9.5402-5411.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.72.9.5402-5411.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15322038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iyoda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Positive effects of multiple <em>pch</em> genes on expression of the locus of enterocyte effacement genes and adherence of enterohaemorrhagic <em>Escherichia coli</em> O157: H7 to HEp-2 cells</div>, <div class="source">Microbiology</div> , <div class="year">2004</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">2357</div>-<div class="lpage">2571</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positive%20effects%20of%20multiple%20pch%20genes%20on%20expression%20of%20the%20locus%20of%20enterocyte%20effacement%20genes%20and%20adherence%20of%20enterohaemorrhagic%20Escherichia%20coli%20O157%3A%20H7%20to%20HEp-2%20cells&amp;author=S.%20Iyoda&amp;author=H.%20Watanabe&amp;publication_year=2004&amp;journal=Microbiology&amp;volume=150&amp;pages=2357-2571" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.27100-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.27100-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.27100-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15256577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campellone</div> <div class="given-names">K. G.</div></div>,  <div class="name"><div class="surname">Robbins</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Leong</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">EspFU is a translocated EHEC effector that interacts with Tir and N-WASP and promotes Nck-independent actin assembly</div>, <div class="source">Dev. Cell</div> , <div class="year">2004</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">217</div>-<div class="lpage">228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EspFU%20is%20a%20translocated%20EHEC%20effector%20that%20interacts%20with%20Tir%20and%20N-WASP%20and%20promotes%20Nck-independent%20actin%20assembly&amp;author=K.%20G.%20Campellone&amp;author=D.%20Robbins&amp;author=J.%20M.%20Leong&amp;publication_year=2004&amp;journal=Dev.%20Cell&amp;volume=7&amp;pages=217-228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.devcel.2004.07.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.devcel.2004.07.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.devcel.2004.07.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15296718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dahan</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wiles</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">La Ragione</div> <div class="given-names">R. M.</div></div>, et al. </div><div class="article-title">EspJ is a prophage-carried type III effector protein of attaching and effacing pathogens that modulates infection dynamics</div>, <div class="source">Infect. Immun.</div> , <div class="year">2005</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">679</div>-<div class="lpage">686</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EspJ%20is%20a%20prophage-carried%20type%20III%20effector%20protein%20of%20attaching%20and%20effacing%20pathogens%20that%20modulates%20infection%20dynamics&amp;author=S.%20Dahan&amp;author=S.%20Wiles&amp;author=R.%20M.%20La%20Ragione&amp;publication_year=2005&amp;journal=Infect.%20Immun.&amp;volume=73&amp;pages=679-686" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.73.2.679-686.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.73.2.679-686.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.73.2.679-686.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15664905" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deng</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Puente</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Gruenheid</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Dissecting virulence: systematic and functional analyses of a pathogenicity island</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">3597</div>-<div class="lpage">3602</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissecting%20virulence%3A%20systematic%20and%20functional%20analyses%20of%20a%20pathogenicity%20island&amp;author=W.%20Deng&amp;author=J.%20L.%20Puente&amp;author=S.%20Gruenheid&amp;author=Y.%20Li&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=3597-3602" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0400326101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0400326101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0400326101"> </span></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garmendia</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Phillips</div> <div class="given-names">A. D.</div></div>,  <div class="name"><div class="surname">Carlier</div> <div class="given-names">M. F.</div></div>, et al. </div><div class="article-title">TccP is an enterohaemorrhagic <em>Escherichia coli</em> O157:H7 type III effector protein that couples Tir to the actin-cytoskeleton</div>, <div class="source">Cell Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">1167</div>-<div class="lpage">1183</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TccP%20is%20an%20enterohaemorrhagic%20Escherichia%20coli%20O157%3AH7%20type%20III%20effector%20protein%20that%20couples%20Tir%20to%20the%20actin-cytoskeleton&amp;author=J.%20Garmendia&amp;author=A.%20D.%20Phillips&amp;author=M.%20F.%20Carlier&amp;publication_year=2004&amp;journal=Cell%20Microbiol.&amp;volume=6&amp;pages=1167-1183" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cmi.2004.6.issue-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcmi.2004.6.issue-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcmi.2004.6.issue-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15527496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gruenheid</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sekirov</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">N. A.</div></div>, et al. </div><div class="article-title">Identification and characterization of NleA, a non-LEE-encoded type III translocated virulence factor of enterohaemorrhagic <em>Escherichia coli</em> O157:H7</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">1233</div>-<div class="lpage">1249</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20NleA%2C%20a%20non-LEE-encoded%20type%20III%20translocated%20virulence%20factor%20of%20enterohaemorrhagic%20Escherichia%20coli%20O157%3AH7&amp;author=S.%20Gruenheid&amp;author=I.%20Sekirov&amp;author=N.%20A.%20Thomas&amp;publication_year=2004&amp;journal=Mol.%20Microbiol.&amp;volume=51&amp;pages=1233-1249" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03911.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03911.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03911.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14982621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mundy</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Petrovska</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Smollett</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Identification of a novel <em>Citrobacter rodentium</em> type III secreted protein, EspI, and roles of this and other secreted proteins in infection</div>, <div class="source">Infect. Immun.</div> , <div class="year">2004</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">2288</div>-<div class="lpage">2302</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20novel%20Citrobacter%20rodentium%20type%20III%20secreted%20protein%2C%20EspI%2C%20and%20roles%20of%20this%20and%20other%20secreted%20proteins%20in%20infection&amp;author=R.%20Mundy&amp;author=L.%20Petrovska&amp;author=K.%20Smollett&amp;publication_year=2004&amp;journal=Infect.%20Immun.&amp;volume=72&amp;pages=2288-2302" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.72.4.2288-2302.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.72.4.2288-2302.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.72.4.2288-2302.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15039354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wick</div> <div class="given-names">L. M.</div></div>,  <div class="name"><div class="surname">Qi</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Lacher</div> <div class="given-names">D. W.</div></div>,  <div class="name"><div class="surname">Whittam</div> <div class="given-names">T. S.</div></div>. </div><div class="article-title">Evolution of genomic content in the stepwise emergence of <em>Escherichia coli</em> O157:H7</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">1783</div>-<div class="lpage">1791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20genomic%20content%20in%20the%20stepwise%20emergence%20of%20Escherichia%20coli%20O157%3AH7&amp;author=L.%20M.%20Wick&amp;author=W.%20Qi&amp;author=D.%20W.%20Lacher&amp;author=T.%20S.%20Whittam&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=1783-1791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.5.1783-1791.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.5.1783-1791.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.5.1783-1791.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15716450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feng</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Lampel</div> <div class="given-names">K. A.</div></div>,  <div class="name"><div class="surname">Karch</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Whittam</div> <div class="given-names">T. S.</div></div>. </div><div class="article-title">Genotypic and phenotypic changes in the emergence of <em>Escherichia coli</em> O157:H7</div>, <div class="source">J. Infect. Dis.</div> , <div class="year">1998</div>, vol. <div class="volume">177</div> (pg. <div class="fpage">1750</div>-<div class="lpage">1753</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotypic%20and%20phenotypic%20changes%20in%20the%20emergence%20of%20Escherichia%20coli%20O157%3AH7&amp;author=P.%20Feng&amp;author=K.%20A.%20Lampel&amp;author=H.%20Karch&amp;author=T.%20S.%20Whittam&amp;publication_year=1998&amp;journal=J.%20Infect.%20Dis.&amp;volume=177&amp;pages=1750-1753" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/jid.1998.177.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2Fjid.1998.177.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2Fjid.1998.177.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9607864" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shaikh</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tarr</div> <div class="given-names">P. I.</div></div>. </div><div class="article-title"><em>Escherichia coli</em> O157:H7 Shiga toxin-encoding bacteriophages: integrations, excisions, truncations, and evolutionary implications</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">3596</div>-<div class="lpage">3605</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20O157%3AH7%20Shiga%20toxin-encoding%20bacteriophages%3A%20integrations%2C%20excisions%2C%20truncations%2C%20and%20evolutionary%20implications&amp;author=N.%20Shaikh&amp;author=P.%20I.%20Tarr&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=3596-3605" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.12.3596-3605.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.12.3596-3605.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.12.3596-3605.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12775697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=486680&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/1/10.1093_dnares_dsi026/1/dsi026_Supplementary_Data.zip?Expires=2147483647&amp;Signature=oxbo~OWjVI5CJjgA64aKZLo87ZcD8Tql1nb18~kctbm0l2BnH~WqfKoKe9vfBMdtKWJts4~apowgdesT0F8ZsCrFMOVkjAg1XTM2DXNxj-gic3PniuySFK1oioaYu9SawliOko24GC-M0nrOJQeQ69bX6tsO~3pDmN2xUiAgz1cay6HMyK0gmQEQXk7FRgJRWH3n0G5urVV01oSWBhjAdffZS4JMrpFJGy1Ftglje~6~ikMT6mDL-zAxwIdMQUmLPfBPJzTTsjgOBQAet3sIXZYrFU4ulbIHXfI6PIllEbgsf5hXbLcFUD9vJUzoVRXwId9lodLI1AQE4oPuYmpCqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">147</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000240077000001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">39</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi026"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="486680"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000240077000001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f1%2f3%2f486680">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000240077000001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (39)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F1%2F3%2F486680" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/486680'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Complexity of the Genomic Diversity in Enterohemorrhagic Escherichia coli O157 Revealed by the Combinational Use of the O157 Sakai OligoDNA Microarray and the Whole Genome PCR scanning" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi026");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f1%2f3%2f486680">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

