

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1441,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/3/283/1745776","@type":"ScholarlyArticle","name":"Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map","datePublished":"2016-04-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2016-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw016","keywords":["tetraploid cotton","genetic linkage map","genome structure","evolution","comparative genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/3/283/1745776","author":[{"name":"Li, Ximei","affiliation":"1  National Key Laboratory of Crop Genetic Improvement (Wuhan) ,  Huazhong Agricultural University ,  Wuhan, Hubei 430070 ,  China    2  College of Agronomy and Plant Protection ,  Qingdao Agricultural University/Shandong Key Laboratory of Dryland Farming Technology ,  Qingdao, Shandong ,  China","@type":"Person"},{"name":"Jin, Xin","affiliation":"1  National Key Laboratory of Crop Genetic Improvement (Wuhan) ,  Huazhong Agricultural University ,  Wuhan, Hubei 430070 ,  China","@type":"Person"},{"name":"Wang, Hantao","affiliation":"1  National Key Laboratory of Crop Genetic Improvement (Wuhan) ,  Huazhong Agricultural University ,  Wuhan, Hubei 430070 ,  China","@type":"Person"},{"name":"Zhang, Xianlong","affiliation":"1  National Key Laboratory of Crop Genetic Improvement (Wuhan) ,  Huazhong Agricultural University ,  Wuhan, Hubei 430070 ,  China","@type":"Person"},{"name":"Lin, Zhongxu","affiliation":"1  National Key Laboratory of Crop Genetic Improvement (Wuhan) ,  Huazhong Agricultural University ,  Wuhan, Hubei 430070 ,  China","@type":"Person"}],"description":"Abstract.  A high-density linkage map was constructed using 1,885 newly obtained loci and 3,747 previously published loci, which included 5,152 loci with 4696.0","pageStart":"283","pageEnd":"293","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map" />
    <meta property="og:description" content="Abstract.  A high-density linkage map was constructed using 1,885 newly obtained loci and 3,747 previously published loci, which included 5,152 loci with 4696.0" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/3/283/1745776" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-04-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Ximei" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement (Wuhan), Huazhong Agricultural University, Wuhan, Hubei 430070, China" /><meta name="citation_author_institution" content="College of Agronomy and Plant Protection, Qingdao Agricultural University/Shandong Key Laboratory of Dryland Farming Technology, Qingdao, Shandong, China" /><meta name="citation_author" content="Jin, Xin" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement (Wuhan), Huazhong Agricultural University, Wuhan, Hubei 430070, China" /><meta name="citation_author" content="Wang, Hantao" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement (Wuhan), Huazhong Agricultural University, Wuhan, Hubei 430070, China" /><meta name="citation_author" content="Zhang, Xianlong" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement (Wuhan), Huazhong Agricultural University, Wuhan, Hubei 430070, China" /><meta name="citation_author" content="Lin, Zhongxu" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement (Wuhan), Huazhong Agricultural University, Wuhan, Hubei 430070, China" /><meta name="citation_title" content="Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map" /><meta name="citation_firstpage" content="283" /><meta name="citation_lastpage" content="293" /><meta name="citation_doi" content="10.1093/dnares/dsw016" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2016/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Recent advances in cotton genomics;  citation_journal_title=Int. J. Plant Genomics;  citation_year=2008;  citation_volume=2008;  citation_pages=742304" /><meta name="citation_reference" content="citation_title=A detailed RFLP map of cotton Gossypium hirsutum× Gossypium barbadense: chromosome organization and evolution in a disomic polyploid genome;  citation_journal_title=Genetics;  citation_year=1994;  citation_volume=138;  citation_pages=829-47; " /><meta name="citation_reference" content="citation_title=A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (Gossypium);  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=166;  citation_pages=389-417; " /><meta name="citation_reference" content="citation_title=A preliminary analysis of genome structure and composition in Gossypium hirsutum;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=314" /><meta name="citation_reference" content="citation_title=Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between Gossypium hirsutum and G. barbadense;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=15" /><meta name="citation_reference" content="citation_title=A high-density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome;  citation_journal_title=G3 (Bethesda);  citation_year=2012;  citation_volume=2;  citation_pages=43-58; " /><meta name="citation_reference" content="citation_title=Sequence-based ultra-dense genetic and physical maps reveal structural variations of allopolyploid cotton genomes;  citation_journal_title=Genome Biol.;  citation_year=2015;  citation_volume=16;  citation_pages=108" /><meta name="citation_reference" content="citation_title=The draft genome of a diploid cotton Gossypium raimondii;  citation_journal_title=Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=1098-103; " /><meta name="citation_reference" content="citation_title=Genome sequence of the cultivated cotton Gossypium arboreum;  citation_journal_title=Nat. Genet.;  citation_year=2014;  citation_volume=46;  citation_pages=567-72; " /><meta name="citation_reference" content="citation_title=Genome sequence of cultivated Upland cotton (Gossypium hirsutum TM-1) provides insights into genome evolution;  citation_journal_title=Nat. Biotechnol.;  citation_year=2015;  citation_volume=33;  citation_pages=524-30; " /><meta name="citation_reference" content="citation_title=Sequencing of allotetraploid cotton (Gossypium hirsutum L. acc. TM-1) provides a resource for fiber improvement;  citation_journal_title=Nat. Biotechnol.;  citation_year=2015;  citation_volume=33;  citation_pages=531-7; " /><meta name="citation_reference" content="citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=492;  citation_pages=423-7; " /><meta name="citation_reference" content="citation_title=PIP: a database of potential intron polymorphism markers;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2174-7; " /><meta name="citation_reference" content="citation_title=Sampling nucleotide diversity in cotton;  citation_journal_title=BMC Plant. Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=125" /><meta name="citation_reference" content="citation_title=Linkage map construction and mapping QTL for cotton fibre quality using SRAP, SSR and RAPD;  citation_journal_title=Plant. Breed;  citation_year=2005;  citation_volume=124;  citation_pages=180-7; " /><meta name="citation_reference" content="citation_title=Increasing cotton genome coverage with polymorphic SSRs as revealed by SSCP;  citation_journal_title=Genome;  citation_year=2012;  citation_volume=55;  citation_pages=459-70; " /><meta name="citation_reference" content="citation_title=Construction of integrated genetic linkage maps by means of a new computer package: Join Map;  citation_journal_title=Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=739-44; " /><meta name="citation_reference" content="citation_title=The estimation of map distance from recombination values;  citation_journal_title=Ann. Eugen.;  citation_year=1944;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=An integrative genetic linkage map of winter wheat (Triticum aestivum L.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=1235-42; " /><meta name="citation_reference" content="citation_title=Development and evaluation of intron and insertion-deletion markers for Gossypium barbadense;  citation_journal_title=Plant. Mol. Biol. Rep.;  citation_year=2012;  citation_volume=30;  citation_pages=605-13; " /><meta name="citation_reference" content="citation_title=Unbiased genomic distribution of genes related to cell morphogenesis in cotton by chromosome mapping;  citation_journal_title=Plant Cell Tissue Org. Cult.;  citation_year=2012;  citation_volume=108;  citation_pages=529-34; " /><meta name="citation_reference" content="citation_title=Genetic mapping and characteristics of genes specifically or preferentially expressed during fiber development in cotton;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e54444" /><meta name="citation_reference" content="citation_title=Linkage mapping and expression analysis of miRNAs and their target genes during fiber development in cotton;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=706" /><meta name="citation_reference" content="citation_title=Mining, genetic mapping and expression analysis of EST-derived resistance gene homologs (RGHs) in cotton;  citation_journal_title=BMC Plant. Biol.;  citation_year=2014;  citation_volume=14;  citation_pages=203" /><meta name="citation_reference" content="citation_title=Development of EST-based SNP and InDel markers and their utilization in tetraploid cotton genetic mapping;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1046" /><meta name="citation_reference" content="citation_title=Comparison and development of EST–SSRs from two 454 sequencing libraries of Gossypium barbadense;  citation_journal_title=Euphytica;  citation_year=2014;  citation_volume=198;  citation_pages=277-88; " /><meta name="citation_reference" content="citation_title=Genetic mapping and comparative expression analysis of transcription factors in cotton;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0126150" /><meta name="citation_reference" content="citation_title=A microsatellite-based, gene-rich linkage map reveals genome structure, function and evolution in Gossypium;  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=527-41; " /><meta name="citation_reference" content="citation_title=A high-density integrative linkage map for Gossypium hirsutum;  citation_journal_title=Euphytica;  citation_year=2009;  citation_volume=166;  citation_pages=35-45; " /><meta name="citation_reference" content="citation_title=Toward allotetraploid cotton genome assembly: integration of a high-density molecular genetic linkage map with DNA sequence information;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=539" /><meta name="citation_reference" content="citation_title=Transferability of wheat microsatellites to diploid Triticeae species carrying the A, B and D genomes;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2001;  citation_volume=103;  citation_pages=346-52; " /><meta name="citation_reference" content="citation_title=New SSR markers for use in cotton (Gossypium spp.) improvement;  citation_journal_title=J. Cotton Sci.;  citation_year=2009;  citation_volume=13;  citation_pages=75-157; " /><meta name="citation_reference" content="citation_title=Development and characterization of genomic and expressed SSRs for levant cotton (Gossypium herbaceum L.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=565-76; " /><meta name="citation_reference" content="citation_title=Medicago truncatula EST-SSRs reveal cross-species genetic markers for Medicago spp;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=414-22; " /><meta name="citation_reference" content="citation_title=Genome mapping of white clover (Trifolium repens L.) and comparative analysis within the Trifolieae using cross-species SSR markers;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=1367-78; " /><meta name="citation_reference" content="citation_title=Common features of segregation distortion in plants and animals;  citation_journal_title=Genetica;  citation_year=2003;  citation_volume=117;  citation_pages=27-35; " /><meta name="citation_reference" content="citation_title=New World tetraploid cottons contain old world cytoplasm;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1989;  citation_volume=86;  citation_pages=4132-6; " /><meta name="citation_reference" content="citation_title=Polyploidy and the evolutionary history of cotton;  citation_journal_title=Adv. Agron.;  citation_year=2003;  citation_volume=78;  citation_pages=139-86; " /><meta name="citation_reference" content="citation_title=A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e45739" /><meta name="citation_reference" content="citation_title=Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing;  citation_journal_title=DNA Res.;  citation_year=2015;  citation_volume=22;  citation_pages=147-60; " /><meta name="citation_reference" content="citation_title=Comparative development of fiber in wild and cultivated cotton;  citation_journal_title=Evol. Dev.;  citation_year=2001;  citation_volume=3;  citation_pages=3-17; " /><meta name="citation_reference" content="citation_title=Polyploid formation created unique avenues for response to selection in Gossypium (cotton);  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=4419-24; " /><meta name="citation_reference" content="citation_title=Genetic mapping of new cotton fiber loci using EST-derived microsatellites in an interspecific recombinant inbred line (RIL) cotton population;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2005;  citation_volume=274;  citation_pages=428-41; " /><meta name="citation_reference" content="citation_title=QTL analysis of genotype × environmental interactions affecting cotton fiber quality;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=384-96; " /><meta name="citation_reference" content="citation_title=The genome sequence of Sea-Island cotton (Gossypium barbadense) provides insights into the allopolyploidization and development of superior spinnable fibres;  citation_journal_title=Sci. Rep.;  citation_year=2015;  citation_volume=5;  citation_pages=17662" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/3/283/17463428/dsw016.pdf" /><meta name="description" content="Abstract.  A high-density linkage map was constructed using 1,885 newly obtained loci and 3,747 previously published loci, which included 5,152 loci with 4696.0" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/3/283/1745776" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/3/283/1745776" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/3/283/1745776");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw016';
    </script>
<title>
	Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745776" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BchuPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaSwgWGltZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBLZXkgTGFib3JhdG9yeSBvZiBDcm9wIEdlbmV0aWMgSW1wcm92ZW1lbnQgKFd1aGFuKSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuLCBIdWJlaSA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29sbGVnZSBvZiBBZ3Jvbm9teSBhbmQgUGxhbnQgUHJvdGVjdGlvbiwgUWluZ2RhbyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eS9TaGFuZG9uZyBLZXkgTGFib3JhdG9yeSBvZiBEcnlsYW5kIEZhcm1pbmcgVGVjaG5vbG9neSwgUWluZ2RhbywgU2hhbmRvbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmluLCBYaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBLZXkgTGFib3JhdG9yeSBvZiBDcm9wIEdlbmV0aWMgSW1wcm92ZW1lbnQgKFd1aGFuKSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuLCBIdWJlaSA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgSGFudGFvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgS2V5IExhYm9yYXRvcnkgb2YgQ3JvcCBHZW5ldGljIEltcHJvdmVtZW50IChXdWhhbiksIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiwgSHViZWkgNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBYaWFubG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpYyBJbXByb3ZlbWVudCAoV3VoYW4pLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4sIEh1YmVpIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaW4sIFpob25neHUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBLZXkgTGFib3JhdG9yeSBvZiBDcm9wIEdlbmV0aWMgSW1wcm92ZW1lbnQgKFd1aGFuKSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuLCBIdWJlaSA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJTdHJ1Y3R1cmUsIGV2b2x1dGlvbiwgYW5kIGNvbXBhcmF0aXZlIGdlbm9taWNzIG9mIHRldHJhcGxvaWQgY290dG9uIGJhc2VkIG9uIGEgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbGlua2FnZSBtYXAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyODMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjZW50IGFkdmFuY2VzIGluIGNvdHRvbiBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBQbGFudCBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMDA4OyAgY2l0YXRpb25fcGFnZXM9NzQyMzA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkZXRhaWxlZCBSRkxQIG1hcCBvZiBjb3R0b24gR29zc3lwaXVtIGhpcnN1dHVtw5cgR29zc3lwaXVtIGJhcmJhZGVuc2U6IGNocm9tb3NvbWUgb3JnYW5pemF0aW9uIGFuZCBldm9sdXRpb24gaW4gYSBkaXNvbWljIHBvbHlwbG9pZCBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTEzODsgIGNpdGF0aW9uX3BhZ2VzPTgyOS00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSAzMzQ3LWxvY3VzIGdlbmV0aWMgcmVjb21iaW5hdGlvbiBtYXAgb2Ygc2VxdWVuY2UtdGFnZ2VkIHNpdGVzIHJldmVhbHMgZmVhdHVyZXMgb2YgZ2Vub21lIG9yZ2FuaXphdGlvbiwgdHJhbnNtaXNzaW9uIGFuZCBldm9sdXRpb24gb2YgY290dG9uIChHb3NzeXBpdW0pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz0zODktNDE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByZWxpbWluYXJ5IGFuYWx5c2lzIG9mIGdlbm9tZSBzdHJ1Y3R1cmUgYW5kIGNvbXBvc2l0aW9uIGluIEdvc3N5cGl1bSBoaXJzdXR1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJlIG9mIGNvdHRvbiByZXZlYWxlZCBieSBhIGdlbm9tZS13aWRlIFNTUiBnZW5ldGljIG1hcCBjb25zdHJ1Y3RlZCBmcm9tIGEgQkMxIHBvcHVsYXRpb24gYmV0d2VlbiBHb3NzeXBpdW0gaGlyc3V0dW0gYW5kIEcuIGJhcmJhZGVuc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBhbmQgc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGdlbmV0aWMgbWFwIG9mIHRoZSB0ZXRyYXBsb2lkIGNvdHRvbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIChCZXRoZXNkYSk7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTQzLTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1iYXNlZCB1bHRyYS1kZW5zZSBnZW5ldGljIGFuZCBwaHlzaWNhbCBtYXBzIHJldmVhbCBzdHJ1Y3R1cmFsIHZhcmlhdGlvbnMgb2YgYWxsb3BvbHlwbG9pZCBjb3R0b24gZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRyYWZ0IGdlbm9tZSBvZiBhIGRpcGxvaWQgY290dG9uIEdvc3N5cGl1bSByYWltb25kaWk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MTA5OC0xMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgY3VsdGl2YXRlZCBjb3R0b24gR29zc3lwaXVtIGFyYm9yZXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTU2Ny03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIGN1bHRpdmF0ZWQgVXBsYW5kIGNvdHRvbiAoR29zc3lwaXVtIGhpcnN1dHVtIFRNLTEpIHByb3ZpZGVzIGluc2lnaHRzIGludG8gZ2Vub21lIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTUyNC0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBvZiBhbGxvdGV0cmFwbG9pZCBjb3R0b24gKEdvc3N5cGl1bSBoaXJzdXR1bSBMLiBhY2MuIFRNLTEpIHByb3ZpZGVzIGEgcmVzb3VyY2UgZm9yIGZpYmVyIGltcHJvdmVtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTMxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGVhdGVkIHBvbHlwbG9pZGl6YXRpb24gb2YgR29zc3lwaXVtIGdlbm9tZXMgYW5kIHRoZSBldm9sdXRpb24gb2Ygc3Bpbm5hYmxlIGNvdHRvbiBmaWJyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTI7ICBjaXRhdGlvbl9wYWdlcz00MjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UElQOiBhIGRhdGFiYXNlIG9mIHBvdGVudGlhbCBpbnRyb24gcG9seW1vcnBoaXNtIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjE3NC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYW1wbGluZyBudWNsZW90aWRlIGRpdmVyc2l0eSBpbiBjb3R0b247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmthZ2UgbWFwIGNvbnN0cnVjdGlvbiBhbmQgbWFwcGluZyBRVEwgZm9yIGNvdHRvbiBmaWJyZSBxdWFsaXR5IHVzaW5nIFNSQVAsIFNTUiBhbmQgUkFQRDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQuIEJyZWVkOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyNDsgIGNpdGF0aW9uX3BhZ2VzPTE4MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNyZWFzaW5nIGNvdHRvbiBnZW5vbWUgY292ZXJhZ2Ugd2l0aCBwb2x5bW9ycGhpYyBTU1JzIGFzIHJldmVhbGVkIGJ5IFNTQ1A7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTQ1OS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGludGVncmF0ZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcHMgYnkgbWVhbnMgb2YgYSBuZXcgY29tcHV0ZXIgcGFja2FnZTogSm9pbiBNYXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz03MzktNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlc3RpbWF0aW9uIG9mIG1hcCBkaXN0YW5jZSBmcm9tIHJlY29tYmluYXRpb24gdmFsdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEV1Z2VuLjsgIGNpdGF0aW9uX3llYXI9MTk0NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE3Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGl2ZSBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIHdpbnRlciB3aGVhdCAoVHJpdGljdW0gYWVzdGl2dW0gTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTEyMzUtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBldmFsdWF0aW9uIG9mIGludHJvbiBhbmQgaW5zZXJ0aW9uLWRlbGV0aW9uIG1hcmtlcnMgZm9yIEdvc3N5cGl1bSBiYXJiYWRlbnNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudC4gTW9sLiBCaW9sLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NjA1LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmJpYXNlZCBnZW5vbWljIGRpc3RyaWJ1dGlvbiBvZiBnZW5lcyByZWxhdGVkIHRvIGNlbGwgbW9ycGhvZ2VuZXNpcyBpbiBjb3R0b24gYnkgY2hyb21vc29tZSBtYXBwaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFRpc3N1ZSBPcmcuIEN1bHQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTUyOS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBtYXBwaW5nIGFuZCBjaGFyYWN0ZXJpc3RpY3Mgb2YgZ2VuZXMgc3BlY2lmaWNhbGx5IG9yIHByZWZlcmVudGlhbGx5IGV4cHJlc3NlZCBkdXJpbmcgZmliZXIgZGV2ZWxvcG1lbnQgaW4gY290dG9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTU0NDQ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlua2FnZSBtYXBwaW5nIGFuZCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIG1pUk5BcyBhbmQgdGhlaXIgdGFyZ2V0IGdlbmVzIGR1cmluZyBmaWJlciBkZXZlbG9wbWVudCBpbiBjb3R0b247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTcwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmluZywgZ2VuZXRpYyBtYXBwaW5nIGFuZCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIEVTVC1kZXJpdmVkIHJlc2lzdGFuY2UgZ2VuZSBob21vbG9ncyAoUkdIcykgaW4gY290dG9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgRVNULWJhc2VkIFNOUCBhbmQgSW5EZWwgbWFya2VycyBhbmQgdGhlaXIgdXRpbGl6YXRpb24gaW4gdGV0cmFwbG9pZCBjb3R0b24gZ2VuZXRpYyBtYXBwaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMDQ2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBhbmQgZGV2ZWxvcG1lbnQgb2YgRVNU4oCTU1NScyBmcm9tIHR3byA0NTQgc2VxdWVuY2luZyBsaWJyYXJpZXMgb2YgR29zc3lwaXVtIGJhcmJhZGVuc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTg7ICBjaXRhdGlvbl9wYWdlcz0yNzctODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWFwcGluZyBhbmQgY29tcGFyYXRpdmUgZXhwcmVzc2lvbiBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgaW4gY290dG9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUwMTI2MTUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtaWNyb3NhdGVsbGl0ZS1iYXNlZCwgZ2VuZS1yaWNoIGxpbmthZ2UgbWFwIHJldmVhbHMgZ2Vub21lIHN0cnVjdHVyZSwgZnVuY3Rpb24gYW5kIGV2b2x1dGlvbiBpbiBHb3NzeXBpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3NjsgIGNpdGF0aW9uX3BhZ2VzPTUyNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgaW50ZWdyYXRpdmUgbGlua2FnZSBtYXAgZm9yIEdvc3N5cGl1bSBoaXJzdXR1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2NjsgIGNpdGF0aW9uX3BhZ2VzPTM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmQgYWxsb3RldHJhcGxvaWQgY290dG9uIGdlbm9tZSBhc3NlbWJseTogaW50ZWdyYXRpb24gb2YgYSBoaWdoLWRlbnNpdHkgbW9sZWN1bGFyIGdlbmV0aWMgbGlua2FnZSBtYXAgd2l0aCBETkEgc2VxdWVuY2UgaW5mb3JtYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTUzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zZmVyYWJpbGl0eSBvZiB3aGVhdCBtaWNyb3NhdGVsbGl0ZXMgdG8gZGlwbG9pZCBUcml0aWNlYWUgc3BlY2llcyBjYXJyeWluZyB0aGUgQSwgQiBhbmQgRCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTM0Ni01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV3IFNTUiBtYXJrZXJzIGZvciB1c2UgaW4gY290dG9uIChHb3NzeXBpdW0gc3BwLikgaW1wcm92ZW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENvdHRvbiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NzUtMTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBnZW5vbWljIGFuZCBleHByZXNzZWQgU1NScyBmb3IgbGV2YW50IGNvdHRvbiAoR29zc3lwaXVtIGhlcmJhY2V1bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9NTY1LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWRpY2FnbyB0cnVuY2F0dWxhIEVTVC1TU1JzIHJldmVhbCBjcm9zcy1zcGVjaWVzIGdlbmV0aWMgbWFya2VycyBmb3IgTWVkaWNhZ28gc3BwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTQxNC0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIG1hcHBpbmcgb2Ygd2hpdGUgY2xvdmVyIChUcmlmb2xpdW0gcmVwZW5zIEwuKSBhbmQgY29tcGFyYXRpdmUgYW5hbHlzaXMgd2l0aGluIHRoZSBUcmlmb2xpZWFlIHVzaW5nIGNyb3NzLXNwZWNpZXMgU1NSIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTE0OyAgY2l0YXRpb25fcGFnZXM9MTM2Ny03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbW9uIGZlYXR1cmVzIG9mIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb24gaW4gcGxhbnRzIGFuZCBhbmltYWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz0yNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV3IFdvcmxkIHRldHJhcGxvaWQgY290dG9ucyBjb250YWluIG9sZCB3b3JsZCBjeXRvcGxhc207ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz00MTMyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbHlwbG9pZHkgYW5kIHRoZSBldm9sdXRpb25hcnkgaGlzdG9yeSBvZiBjb3R0b247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFkdi4gQWdyb24uOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9MTM5LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2ggZGVuc2l0eSBjb25zZW5zdXMgZ2VuZXRpYyBtYXAgb2YgdGV0cmFwbG9pZCBjb3R0b24gdGhhdCBpbnRlZ3JhdGVzIG11bHRpcGxlIGNvbXBvbmVudCBtYXBzIHRocm91Z2ggbW9sZWN1bGFyIG1hcmtlciByZWR1bmRhbmN5IGNoZWNrOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ1NzM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5yaWNobWVudCBvZiBhbiBpbnRyYXNwZWNpZmljIGdlbmV0aWMgbWFwIG9mIHVwbGFuZCBjb3R0b24gYnkgZGV2ZWxvcGluZyBtYXJrZXJzIHVzaW5nIHBhcmVudGFsIFJBRCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE0Ny02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZGV2ZWxvcG1lbnQgb2YgZmliZXIgaW4gd2lsZCBhbmQgY3VsdGl2YXRlZCBjb3R0b247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2wuIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTMtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbHlwbG9pZCBmb3JtYXRpb24gY3JlYXRlZCB1bmlxdWUgYXZlbnVlcyBmb3IgcmVzcG9uc2UgdG8gc2VsZWN0aW9uIGluIEdvc3N5cGl1bSAoY290dG9uKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTQ0MTktMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWFwcGluZyBvZiBuZXcgY290dG9uIGZpYmVyIGxvY2kgdXNpbmcgRVNULWRlcml2ZWQgbWljcm9zYXRlbGxpdGVzIGluIGFuIGludGVyc3BlY2lmaWMgcmVjb21iaW5hbnQgaW5icmVkIGxpbmUgKFJJTCkgY290dG9uIHBvcHVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI3NDsgIGNpdGF0aW9uX3BhZ2VzPTQyOC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMIGFuYWx5c2lzIG9mIGdlbm90eXBlIMOXIGVudmlyb25tZW50YWwgaW50ZXJhY3Rpb25zIGFmZmVjdGluZyBjb3R0b24gZmliZXIgcXVhbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0zODQtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgU2VhLUlzbGFuZCBjb3R0b24gKEdvc3N5cGl1bSBiYXJiYWRlbnNlKSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIHRoZSBhbGxvcG9seXBsb2lkaXphdGlvbiBhbmQgZGV2ZWxvcG1lbnQgb2Ygc3VwZXJpb3Igc3Bpbm5hYmxlIGZpYnJlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNzY2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMy8zLzI4My8xNzQ2MzQyOC9kc3cwMTYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEEgaGlnaC1kZW5zaXR5IGxpbmthZ2UgbWFwIHdhcyBjb25zdHJ1Y3RlZCB1c2luZyAxLDg4NSBuZXdseSBvYnRhaW5lZCBsb2NpIGFuZCAzLDc0NyBwcmV2aW91c2x5IHB1Ymxpc2hlZCBsb2NpLCB3aGljaCBpbmNsdWRlZCA1LDE1MiBsb2NpIHdpdGggNDY5Ni4wIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0xNzQ1Nzc2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJ7voggkRPKVqubg5M9d7vcwsOmd" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f283%2f1745776" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f283%2f1745776" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69063" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&amp;Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/3">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87201788">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201788">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201791">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201791">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201807">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201807">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201827">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201827">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201867">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201867">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201887">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201887">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201889">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201889">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/3/271/1745617">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ximei Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Key Laboratory of Crop Genetic Improvement (Wuhan)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan, Hubei 430070</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">College of Agronomy and Plant Protection</div>, <div class="insititution">Qingdao Agricultural University/Shandong Key Laboratory of Dryland Farming Technology</div>, <div class="addr-line">Qingdao, Shandong</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ximei+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ximei Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xin Jin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Key Laboratory of Crop Genetic Improvement (Wuhan)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan, Hubei 430070</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xin+Jin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jin X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jin X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xin Jin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hantao Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Key Laboratory of Crop Genetic Improvement (Wuhan)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan, Hubei 430070</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hantao+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hantao Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xianlong Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Key Laboratory of Crop Genetic Improvement (Wuhan)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan, Hubei 430070</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xianlong+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xianlong Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhongxu Lin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Key Laboratory of Crop Genetic Improvement (Wuhan)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan, Hubei 430070</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 27-87283955. Fax. +86 27-87280016. Email: <a href="mailto:linzhongxu@mail.hzau.edu.cn" target="_blank">linzhongxu@mail.hzau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhongxu+Lin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lin Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lin Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhongxu Lin</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 283–293, <a href='https://doi.org/10.1093/dnares/dsw016'>https://doi.org/10.1093/dnares/dsw016</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 April 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">01 October 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">17 March 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745776" href="/dnaresearch/article-pdf/23/3/283/17463428/dsw016.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Ximei Li, Xin Jin, Hantao Wang, Xianlong Zhang, Zhongxu Lin;  Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map, <em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 283–293, <a href='https://doi.org/10.1093/dnares/dsw016'>https://doi.org/10.1093/dnares/dsw016</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745776?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745776?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745776?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745776?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745776?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745776?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745776?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Structure%2C%20evolution%2C%20and%20comparative%20genomics%20of%20tetraploid%20cotton%20based%20on%20a%20high-density%20genetic%20linkage%20map&amp;publicationDate=2016-04-15&amp;volumeNum=23&amp;issueNum=3&amp;author=Li%2C%20Ximei%3B%20Jin%2C%20Xin&amp;startPage=283&amp;endPage=293&amp;contentId=10.1093%2Fdnares%2Fdsw016&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f283%2f1745776" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87201788 id="87201788" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>A high-density linkage map was constructed using 1,885 newly obtained loci and 3,747 previously published loci, which included 5,152 loci with 4696.03 cM in total length and 0.91 cM in mean distance. Homology analysis in the cotton genome further confirmed the 13 expected homologous chromosome pairs and revealed an obvious inversion on Chr10 or Chr20 and repeated inversions on Chr07 or Chr16. In addition, two reciprocal translocations between Chr02 and Chr03 and between Chr04 and Chr05 were confirmed. Comparative genomics between the tetraploid cotton and the diploid cottons showed that no major structural changes exist between D<sub>T</sub> and D chromosomes but rather between A<sub>T</sub> and A chromosomes. Blast analysis between the tetraploid cotton genome and the mixed genome of two diploid cottons showed that most AD chromosomes, regardless of whether it is from the A<sub>T</sub> or D<sub>T</sub> genome, preferentially matched with the corresponding homologous chromosome in the diploid A genome, and then the corresponding homologous chromosome in the diploid D genome, indicating that the diploid D genome underwent converted evolution by the diploid A genome to form the D<sub>T</sub> genome during polyploidization. In addition, the results reflected that a series of chromosomal translocations occurred among Chr01/Chr15, Chr02/Chr14, Chr03/Chr17, Chr04/Chr22, and Chr05/Chr19.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tetraploid cotton&quot;">tetraploid cotton</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic linkage map&quot;">genetic linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome structure&quot;">genome structure</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a></div>                            <h2 scrollto-destination=87201791 id="87201791" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Cotton (<em>Gossypium</em>) is an important cash crop and the uppermost source of textile fibre, and <em>G. hirsutum</em> and <em>G. barbadense</em> account for 90 and 8% of the world cotton production, respectively.<sup><a reveal-id="DSW016C1" data-open="DSW016C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Thus, genome structure analysis of allotetraploid cotton is important, but the allotetraploid (2<em>n</em> = 4<em>x</em> = 52) species has a large genome size of ∼2,246 Mb,<sup><a reveal-id="DSW016C2" data-open="DSW016C2" class="link link-ref link-reveal xref-bibr">2</a></sup> indicating that it is especially difficult to generate a reference genome sequence. As a result, a high-density genetic linkage map is an optional tool to reveal genome structure and chromosomal architecture of allotetraploid cotton.</p><p>Currently, there are four comparatively dense linkage maps in cotton based on experimental data. Rong et al.<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> (2,584 loci, 4447.9 cM in length with an average distance of 1.72 cM) revealed all 13 expected homologous chromosome pairs and reported that there were no major structural changes between D<sub>T</sub> and D chromosomes, but two reciprocal translocations between A<sub>T</sub> and A chromosomes and several inversions. Guo et al.<sup><a reveal-id="DSW016C4" data-open="DSW016C4" class="link link-ref link-reveal xref-bibr">4</a></sup> (2,247 loci, 3440.4 cM in length with an average distance of 1.58 cM) again reported the translocations that occurred between some chromosomes (Chr02 and Chr03, Chr04 and Chr05), providing a glimpse of cotton genome complexity. Based on a genome-wide simple sequence repeat (SSR) genetic map (2,316 loci, 4418.9 cM in length with an average distance of 1.91 cM) constructed in our laboratory,<sup><a reveal-id="DSW016C5" data-open="DSW016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> 21 segregation distortion regions (SDRs) were found, and 3 segregation distorted chromosomes (Chr02, Chr16, and Chr18) were identified with 99.9% of distorted markers segregating towards the heterozygous allele. A genetic linkage map (2,072 loci, 3379.9 cM in length with an average distance of 1.63 cM) constructed by Yu et al.<sup><a reveal-id="DSW016C6" data-open="DSW016C6" class="link link-ref link-reveal xref-bibr">6</a></sup> showed that the allotetraploid cotton genome produced equivalent recombination frequencies in its two genomes and revealed that the genetically smallest homologous chromosome pair was Chr04 and Chr22, and the largest was Chr05 and Chr19. Just prior to this publication, a sequence-based interspecific genetic map composed of 4,999,048 SNP loci was reported, which contains only 4,049 recombination bins and covers 4,042 cM with an average interbin genetic distance of 1.0 cM.<sup><a reveal-id="DSW016C7" data-open="DSW016C7" class="link link-ref link-reveal xref-bibr">7</a></sup> First, this map played a role in genomic assembly of allotetraploid cotton. Second, some structural variations not only in the A<sub>T</sub> genome but also in the D<sub>T</sub> genome were detected, including 15 first reported simple translocations. Third, centromeric regions of tetraploid cotton were predicted. All of the above genetic maps played fundamental roles in understanding the cotton genome structure and in studying cotton evolutionary genomics.</p><p>With the release of <em>G. raimondii</em> genome sequence, genetic relationship analysis showed that <em>G. raimondii</em> and <em>Theobroma cacao</em> belong to a common subclade.<sup><a reveal-id="DSW016C8" data-open="DSW016C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In addition, the two genomes possess a moderate syntenic relationship, with 463 collinear blocks (with ≥5 genes per block) covering 64.8 and 74.41% of the assembled <em>G. raimondii</em> and <em>T. cacao</em> genomes, respectively.<sup><a reveal-id="DSW016C8" data-open="DSW016C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Subsequently, <em>G. arboreum</em> genome sequence was released,<sup><a reveal-id="DSW016C9" data-open="DSW016C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and a close collinear relationship between <em>G. arboreum</em> and <em>T. cacao</em> was also discovered. Besides, collinearity analysis between <em>G. raimondii</em> and <em>G. arboreum</em> revealed that chromosomes 1, 4–6, and 9–13 were highly collinear, whereas large-scale rearrangements were observed on chromosomes 2 and 3 of <em>G. raimondii</em>, and deletions/insertions were observed on chromosomes 7 and 8 of <em>G. arboreum</em>. In 2015, <em>G. hirsutum</em> genome sequence was released,<sup><a reveal-id="DSW016C10" data-open="DSW016C10" class="link link-ref link-reveal xref-bibr">10</a></sup> which showed a conserved order between the A<sub>T</sub>D<sub>T</sub> genome and the already sequenced diploid genomes.<sup><a reveal-id="DSW016C8" data-open="DSW016C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSW016C9" data-open="DSW016C9" class="link link-ref link-reveal xref-bibr">9</a></sup> However, another version of the A<sub>T</sub>D<sub>T</sub> genome (<em>G. hirsutum</em>)<sup><a reveal-id="DSW016C11" data-open="DSW016C11" class="link link-ref link-reveal xref-bibr">11</a></sup> showed that such collinearity was not obvious with either <em>G. arboreum</em><sup><a reveal-id="DSW016C9" data-open="DSW016C9" class="link link-ref link-reveal xref-bibr">9</a></sup> or <em>G. raimondii.</em><sup><a reveal-id="DSW016C8" data-open="DSW016C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Instead, it is largely conserved with another version of the D-progenitor genome (<em>G. raimondii</em>).<sup><a reveal-id="DSW016C12" data-open="DSW016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In addition, the overall gene order and collinearity are largely conserved between the A<sub>T</sub> and D<sub>T</sub> genomes although at least 9 translocations and 28 inversions were identified.</p><p>In the present study, a high-density genetic linkage map including 5,152 loci for tetraploid cotton was constructed mainly based on SSR markers. With this map, we aimed to reveal the cotton genome structure by homologous chromosome comparisons and to explain the polyploidization of tetraploid cotton by comparing this map with two sequenced progenitor diploid cottons.</p>                            <h2 scrollto-destination=87201807 id="87201807" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87201810 id="87201810" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p><em>Gossypium hirsutum</em> cv. Emian22 and <em>G. barbadense</em> acc. 3–79 were used to detect polymorphisms of all the new markers. Emian22 is a high yield cultivar with moderate fibre quality and no resistance to verticillium wilt, whereas 3–79 is the genetic and cytogenetic standard line for <em>G. barbadense</em> with super fibre quality and high resistance to verticillium wilt. To improve the performance of Emian22 by backcrossing and molecular-assisted selection, a cross between these two materials was performed. Subsequently, the BC<sub>1</sub> population [(Emian22 × 3–79) × Emian22] including 141 progeny was used as the mapping population, which had been used to construct a 2,316 loci map.<sup><a reveal-id="DSW016C5" data-open="DSW016C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p>                            <h3 scrollto-destination=87201814 id="87201814" class="section-title" data-legacy-id=s2b>2.2. Molecular markers</h3>
<p>During map construction in this study, 5,299 new primer pairs were applied, including 4,569 newly published SSRs (2,937 MON-prefixed, 664 NBRI-prefixed, 670 CCRI-prefixed, 200 HAU-prefixed, and 98 NAU-prefixed; <a class="link link-uri" href="http://www.cottongen.org/">http://www.cottongen.org/</a>) and 730 other batches of primers (579 GhirPIP-prefixed,<sup><a reveal-id="DSW016C13" data-open="DSW016C13" class="link link-ref link-reveal xref-bibr">13</a></sup> 115 cg-prefixed,<sup><a reveal-id="DSW016C14" data-open="DSW016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and 36 cot-prefixed<sup><a reveal-id="DSW016C14" data-open="DSW016C14" class="link link-ref link-reveal xref-bibr">14</a></sup>).</p>                            <h3 scrollto-destination=87201818 id="87201818" class="section-title" data-legacy-id=s2c>2.3. Genotyping analysis</h3>
<p>Polymorphism detection of all the new SSRs was performed as previously described.<sup><a reveal-id="DSW016C15" data-open="DSW016C15" class="link link-ref link-reveal xref-bibr">15</a></sup> For the remnant monomorphic SSRs, single-strand conformation polymorphism (SSCP) analysis was applied, which is identical to the method described by Li et al.<sup><a reveal-id="DSW016C16" data-open="DSW016C16" class="link link-ref link-reveal xref-bibr">16</a></sup> For the other three batches of primers (GhirPIP-prefixed, cg-prefixed, and cot-prefixed), SSCP analysis is the only method for genotyping analysis. Subsequently, genotyping of the whole population using polymorphic primers was carried out on the corresponding condition. All DNA fragments were detected with silver staining.</p>                            <h3 scrollto-destination=87201822 id="87201822" class="section-title" data-legacy-id=s2d>2.4. Map construction</h3>
<p>The mapping data for each BC<sub>1</sub> individual were scored according to the definition of JoinMap 3.0.<sup><a reveal-id="DSW016C17" data-open="DSW016C17" class="link link-ref link-reveal xref-bibr">17</a></sup> For each segregating marker, a <em>χ</em><sup>2</sup> analysis was performed to determine whether it is deviated from the expected 1 : 1 segregation ratio. During map construction, the logarithm of odds (LOD) threshold was ≥8.0, and the maximum recombination rate was 0.4. Map distances in centiMorgans (cM) were calculated using the Kosambi mapping function.<sup><a reveal-id="DSW016C18" data-open="DSW016C18" class="link link-ref link-reveal xref-bibr">18</a></sup> In the resulting linkage map, a region with at least three adjacent loci showing significant segregation distortion (<em>P</em> &lt; 0.05) was defined as the SDR.<sup><a reveal-id="DSW016C19" data-open="DSW016C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                            <h3 scrollto-destination=87201824 id="87201824" class="section-title" data-legacy-id=s2e>2.5. Collinearity and comparative genomic analysis</h3>
<p>Based on the high-density genetic linkage map and sequences corresponding to markers, collinearity analysis between homologous chromosome pairs was performed using a BLASTN search with <em>E</em> ≤ 1e<sup>−5</sup>, identity ≥80%, and matched length ≥200 bp. Next, the best hit for each marker was chosen, and all the best hits were illustrated intuitively using online drawing tools (<a class="link link-uri" href="http://circos.ca/">http://circos.ca/</a>). Comparative genomic analysis between tetraploid cotton and diploid cottons was performed using a similar method.</p>                            <h2 scrollto-destination=87201827 id="87201827" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87201830 id="87201830" class="section-title" data-legacy-id=s3a>3.1. Marker information</h3>
<p>To construct a high-density genetic linkage map to meet the demands of cotton genetics and breeding, 200 novel EST-SSRs (HAU3599-HAU3798) were developed in this study. In detail, 3,647 unique sequences were obtained from a normalized adversity cDNA library of <em>G. barbadense</em> acc. Hai7124, and then they were searched against the cotton EST database (posted date: 22 March 2009, with a total of 375,349 sequences), with an <em>E</em>-value cut-off of &lt;1e<sup>−10</sup>. As a result, 255 had no BLAST hits to known sequences. Subsequently, a total of 200 novel EST-SSRs were developed. During map construction, traditional genotyping analysis made 26 (13.00%) polymorphic primer pairs generating 26 polymorphic loci, and SSCP analysis made 14 (8.05%) polymorphic primer pairs generating 16 polymorphic loci from the remnant monomorphic markers. In total, 40 (20.00%) HAU-prefixed EST-SSRs showed polymorphism, and generated 42 polymorphic loci, with an average of 1.05 alleles.</p><p>Among the 670 CCRI-prefixed and 98 NAU-prefixed EST-SSRs, traditional genotyping analysis made 75 (11.19%) and 66 (67.35%) polymorphic EST-SSRs generating 78 and 79 polymorphic loci, respectively. For the remnant monomorphic markers, SSCP analysis was applied. As a result, 67 (11.26%) and 11 (34.38%) were polymorphic with 73 and 11 polymorphic loci, respectively. In total, 142 (21.19%) CCRI-prefixed and 77 (78.57%) NAU-prefixed EST-SSRs were polymorphic, and generated 151 and 90 polymorphic loci, with an average of 1.06 and 1.17 alleles, respectively.</p><p>The total 2,937 MON-prefixed SSRs consist of 2,521 gSSRs and 416 EST-SSRs. After traditional genotyping analysis, 879 (34.87%) gSSRs and 129 (31.01%) EST-SSRs showed polymorphism, and generated 1,005 and 140 polymorphic loci, respectively. Then, the SSCP analysis was conducted with the remnant monomorphic markers. Subsequently, 126 (7.67%) gSSRs and 12 (4.18%) EST-SSRs showed polymorphism, and generated 138 and 13 polymorphic loci, respectively. In total, 1,005 (39.87%) MON-prefixed gSSRs and 141 (33.89%) MON-prefixed EST-SSRs showed polymorphism, and generated 1,143 and 153 polymorphic loci, with an average of 1.14 and 1.09 alleles, respectively. Taking the 2,937 MON-prefixed SSRs as a whole, 1,146 (39.02%) primers were polymorphic, and generated 1,296 polymorphic loci, with an average of 1.13 alleles.</p><p>The total 664 NBRI-prefixed SSRs consist of 263 gSSRs and 401 EST-SSRs. After traditional genotyping analysis, 85 (32.32%) gSSRs and 122 (30.42%) EST-SSRs showed polymorphism, and generated 97 and 136 polymorphic loci, respectively. Then, SSCP analysis was performed with the remnant monomorphic markers. Subsequently, 11 (6.18%) gSSRs and 15 (5.38%) EST-SSRs showed polymorphism, and generated 11 and 17 polymorphic loci, respectively. In total, 96 (36.50%) NBRI-prefixed gSSRs and 137 (34.16%) NBRI-prefixed EST-SSRs showed polymorphism, and generated 108 and 153 polymorphic loci, with an average of 1.13 and 1.12 alleles, respectively. Taking the 664 NBRI-prefixed SSRs as a whole, 233 (35.09%) primers showed polymorphism, and generated 261 polymorphic loci, with an average of 1.12 alleles.</p><p>For the 579 GhirPIP-prefixed, 115 cg-prefixed, and 36 cot-prefixed primer pairs, SSCP analysis was directly applied. As a result, 27 (4.66%) GhirPIP-prefixed, 13 (11.30%) cg-prefixed, and 5 (13.89%) cot-prefixed primer pairs showed polymorphism, and each polymorphic primer pair produced one polymorphic locus.</p>                            <h3 scrollto-destination=87201847 id="87201847" class="section-title" data-legacy-id=s3b>3.2. Map construction and overview</h3>
<p>The 1,885 loci obtained in this study were added to the 3,747 loci updated in our laboratory,<sup><a reveal-id="DSW016C5" data-open="DSW016C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW016C16" data-open="DSW016C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW016C20" data-open="DSW016C20" class="link link-ref link-reveal xref-bibr">20</a>–<a reveal-id="DSW016C27" data-open="DSW016C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and a total of 5,632 loci were used for map construction. After calculation, a linkage map with 5,152 loci was constructed, and it was 4696.03 cM in total length and 0.91 cM in mean distance (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and Table <a reveal-id="DSW016TB1" data-open="DSW016TB1" class="link link-reveal link-table xref-fig">1</a>). The chromosomes were built with LODs ranging from 8.0 to 15.0 (Table <a reveal-id="DSW016TB1" data-open="DSW016TB1" class="link link-reveal link-table xref-fig">1</a>). The A<sub>T</sub> genome contained 2,473 loci with 2359.36 cM in total length and 0.95 cM in mean distance, whereas the D<sub>T</sub> genome contained 2,679 loci with 2336.67 cM in total length and 0.87 cM in mean distance. </p>                        <a id="87201851" scrollto-destination="87201851"></a>
<div content-id="DSW016TB1" class="table-modal"><div class="table-wrap"><div id="DSW016TB1" class="table-wrap-title" data-id="DSW016TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Characteristics of the linkage map constructed from the BC<sub>1</sub> population</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>LOD </th><th>Total loci </th><th>Size (cM) </th><th>Mean distance (cM) </th><th>Largest gap (cM) </th><th>&gt;10 cM makers </th></tr></thead><tbody><tr><td>Chr01 </td><td>10.0 </td><td>143 </td><td>186.87 </td><td>1.31 </td><td>21.22 </td><td>1 </td></tr><tr><td>Chr02 </td><td>12.0 </td><td>130 </td><td>156.03 </td><td>1.20 </td><td>10.67 </td><td>1 </td></tr><tr><td>Chr03 </td><td>15.0 </td><td>169 </td><td>164.93 </td><td>0.98 </td><td>9.49 </td><td>0 </td></tr><tr><td>Chr04 </td><td>12.0 </td><td>122 </td><td>149.82 </td><td>1.23 </td><td>7.85 </td><td>0 </td></tr><tr><td>Chr05 </td><td>13.0 </td><td>285 </td><td>242.76 </td><td>0.85 </td><td>9.16 </td><td>0 </td></tr><tr><td>Chr06 </td><td>9.0 </td><td>165 </td><td>171.43 </td><td>1.04 </td><td>10.41 </td><td>1 </td></tr><tr><td>Chr07 </td><td>14.0 </td><td>163 </td><td>105.78 </td><td>0.65 </td><td>3.42 </td><td>0 </td></tr><tr><td>Chr08 </td><td>11.0 </td><td>213 </td><td>151.02 </td><td>0.71 </td><td>9.38 </td><td>0 </td></tr><tr><td>Chr09 </td><td>10.0 </td><td>180 </td><td>148.83 </td><td>0.83 </td><td>4.15 </td><td>0 </td></tr><tr><td>Chr10 </td><td>10.0 </td><td>180 </td><td>200.94 </td><td>1.12 </td><td>17.93 </td><td>1 </td></tr><tr><td>Chr11 </td><td>13.0 </td><td>271 </td><td>234.77 </td><td>0.87 </td><td>6.09 </td><td>0 </td></tr><tr><td>Chr12 </td><td>14.0 </td><td>241 </td><td>238.05 </td><td>0.99 </td><td>6.26 </td><td>0 </td></tr><tr><td>Chr13 </td><td>10.0 </td><td>211 </td><td>208.14 </td><td>0.99 </td><td>6.56 </td><td>0 </td></tr><tr><td>A<sub>T</sub> genome </td><td> </td><td>2,473 </td><td>2,359.36 </td><td>0.95 </td><td>21.22 </td><td>4 </td></tr><tr><td>Chr14 </td><td>15.0 </td><td>180 </td><td>164.12 </td><td>0.91 </td><td>9.45 </td><td>0 </td></tr><tr><td>Chr15 </td><td>11.0 </td><td>215 </td><td>197.10 </td><td>0.92 </td><td>11.05 </td><td>1 </td></tr><tr><td>Chr16 </td><td>14.0 </td><td>179 </td><td>94.32 </td><td>0.53 </td><td>4.68 </td><td>0 </td></tr><tr><td>Chr17 </td><td>9.0 </td><td>150 </td><td>162.23 </td><td>1.08 </td><td>22.56 </td><td>2 </td></tr><tr><td>Chr18 </td><td>10.0 </td><td>188 </td><td>146.95 </td><td>0.78 </td><td>12.18 </td><td>1 </td></tr><tr><td>Chr19 </td><td>13.0 </td><td>306 </td><td>252.27 </td><td>0.82 </td><td>5.36 </td><td>0 </td></tr><tr><td>Chr20 </td><td>10.0 </td><td>183 </td><td>117.61 </td><td>0.64 </td><td>8.48 </td><td>0 </td></tr><tr><td>Chr21 </td><td>13.0 </td><td>265 </td><td>256.03 </td><td>0.97 </td><td>8.85 </td><td>0 </td></tr><tr><td>Chr22 </td><td>8.0 </td><td>147 </td><td>169.93 </td><td>1.16 </td><td>16.39 </td><td>1 </td></tr><tr><td>Chr23 </td><td>10.0 </td><td>209 </td><td>193.19 </td><td>0.92 </td><td>12.84 </td><td>2 </td></tr><tr><td>Chr24 </td><td>10.0 </td><td>225 </td><td>198.85 </td><td>0.88 </td><td>11.81 </td><td>1 </td></tr><tr><td>Chr25 </td><td>12.0 </td><td>194 </td><td>172.19 </td><td>0.89 </td><td>11.41 </td><td>1 </td></tr><tr><td>Chr26 </td><td>9.0 </td><td>238 </td><td>211.90 </td><td>0.89 </td><td>14.80 </td><td>2 </td></tr><tr><td>D<sub>T</sub> genome </td><td> </td><td>2,679 </td><td>2,336.67 </td><td>0.87 </td><td>22.56 </td><td>11 </td></tr><tr><td>Total </td><td> </td><td>5,152 </td><td>4,696.03 </td><td>0.91 </td><td>22.56 </td><td>15 </td></tr></tbody></table></div><div content-id="DSW016TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>LOD </th><th>Total loci </th><th>Size (cM) </th><th>Mean distance (cM) </th><th>Largest gap (cM) </th><th>&gt;10 cM makers </th></tr></thead><tbody><tr><td>Chr01 </td><td>10.0 </td><td>143 </td><td>186.87 </td><td>1.31 </td><td>21.22 </td><td>1 </td></tr><tr><td>Chr02 </td><td>12.0 </td><td>130 </td><td>156.03 </td><td>1.20 </td><td>10.67 </td><td>1 </td></tr><tr><td>Chr03 </td><td>15.0 </td><td>169 </td><td>164.93 </td><td>0.98 </td><td>9.49 </td><td>0 </td></tr><tr><td>Chr04 </td><td>12.0 </td><td>122 </td><td>149.82 </td><td>1.23 </td><td>7.85 </td><td>0 </td></tr><tr><td>Chr05 </td><td>13.0 </td><td>285 </td><td>242.76 </td><td>0.85 </td><td>9.16 </td><td>0 </td></tr><tr><td>Chr06 </td><td>9.0 </td><td>165 </td><td>171.43 </td><td>1.04 </td><td>10.41 </td><td>1 </td></tr><tr><td>Chr07 </td><td>14.0 </td><td>163 </td><td>105.78 </td><td>0.65 </td><td>3.42 </td><td>0 </td></tr><tr><td>Chr08 </td><td>11.0 </td><td>213 </td><td>151.02 </td><td>0.71 </td><td>9.38 </td><td>0 </td></tr><tr><td>Chr09 </td><td>10.0 </td><td>180 </td><td>148.83 </td><td>0.83 </td><td>4.15 </td><td>0 </td></tr><tr><td>Chr10 </td><td>10.0 </td><td>180 </td><td>200.94 </td><td>1.12 </td><td>17.93 </td><td>1 </td></tr><tr><td>Chr11 </td><td>13.0 </td><td>271 </td><td>234.77 </td><td>0.87 </td><td>6.09 </td><td>0 </td></tr><tr><td>Chr12 </td><td>14.0 </td><td>241 </td><td>238.05 </td><td>0.99 </td><td>6.26 </td><td>0 </td></tr><tr><td>Chr13 </td><td>10.0 </td><td>211 </td><td>208.14 </td><td>0.99 </td><td>6.56 </td><td>0 </td></tr><tr><td>A<sub>T</sub> genome </td><td> </td><td>2,473 </td><td>2,359.36 </td><td>0.95 </td><td>21.22 </td><td>4 </td></tr><tr><td>Chr14 </td><td>15.0 </td><td>180 </td><td>164.12 </td><td>0.91 </td><td>9.45 </td><td>0 </td></tr><tr><td>Chr15 </td><td>11.0 </td><td>215 </td><td>197.10 </td><td>0.92 </td><td>11.05 </td><td>1 </td></tr><tr><td>Chr16 </td><td>14.0 </td><td>179 </td><td>94.32 </td><td>0.53 </td><td>4.68 </td><td>0 </td></tr><tr><td>Chr17 </td><td>9.0 </td><td>150 </td><td>162.23 </td><td>1.08 </td><td>22.56 </td><td>2 </td></tr><tr><td>Chr18 </td><td>10.0 </td><td>188 </td><td>146.95 </td><td>0.78 </td><td>12.18 </td><td>1 </td></tr><tr><td>Chr19 </td><td>13.0 </td><td>306 </td><td>252.27 </td><td>0.82 </td><td>5.36 </td><td>0 </td></tr><tr><td>Chr20 </td><td>10.0 </td><td>183 </td><td>117.61 </td><td>0.64 </td><td>8.48 </td><td>0 </td></tr><tr><td>Chr21 </td><td>13.0 </td><td>265 </td><td>256.03 </td><td>0.97 </td><td>8.85 </td><td>0 </td></tr><tr><td>Chr22 </td><td>8.0 </td><td>147 </td><td>169.93 </td><td>1.16 </td><td>16.39 </td><td>1 </td></tr><tr><td>Chr23 </td><td>10.0 </td><td>209 </td><td>193.19 </td><td>0.92 </td><td>12.84 </td><td>2 </td></tr><tr><td>Chr24 </td><td>10.0 </td><td>225 </td><td>198.85 </td><td>0.88 </td><td>11.81 </td><td>1 </td></tr><tr><td>Chr25 </td><td>12.0 </td><td>194 </td><td>172.19 </td><td>0.89 </td><td>11.41 </td><td>1 </td></tr><tr><td>Chr26 </td><td>9.0 </td><td>238 </td><td>211.90 </td><td>0.89 </td><td>14.80 </td><td>2 </td></tr><tr><td>D<sub>T</sub> genome </td><td> </td><td>2,679 </td><td>2,336.67 </td><td>0.87 </td><td>22.56 </td><td>11 </td></tr><tr><td>Total </td><td> </td><td>5,152 </td><td>4,696.03 </td><td>0.91 </td><td>22.56 </td><td>15 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>LOD, logarithm of odds.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201851" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW016TB1" class="table-wrap-title" data-id="DSW016TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Characteristics of the linkage map constructed from the BC<sub>1</sub> population</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>LOD </th><th>Total loci </th><th>Size (cM) </th><th>Mean distance (cM) </th><th>Largest gap (cM) </th><th>&gt;10 cM makers </th></tr></thead><tbody><tr><td>Chr01 </td><td>10.0 </td><td>143 </td><td>186.87 </td><td>1.31 </td><td>21.22 </td><td>1 </td></tr><tr><td>Chr02 </td><td>12.0 </td><td>130 </td><td>156.03 </td><td>1.20 </td><td>10.67 </td><td>1 </td></tr><tr><td>Chr03 </td><td>15.0 </td><td>169 </td><td>164.93 </td><td>0.98 </td><td>9.49 </td><td>0 </td></tr><tr><td>Chr04 </td><td>12.0 </td><td>122 </td><td>149.82 </td><td>1.23 </td><td>7.85 </td><td>0 </td></tr><tr><td>Chr05 </td><td>13.0 </td><td>285 </td><td>242.76 </td><td>0.85 </td><td>9.16 </td><td>0 </td></tr><tr><td>Chr06 </td><td>9.0 </td><td>165 </td><td>171.43 </td><td>1.04 </td><td>10.41 </td><td>1 </td></tr><tr><td>Chr07 </td><td>14.0 </td><td>163 </td><td>105.78 </td><td>0.65 </td><td>3.42 </td><td>0 </td></tr><tr><td>Chr08 </td><td>11.0 </td><td>213 </td><td>151.02 </td><td>0.71 </td><td>9.38 </td><td>0 </td></tr><tr><td>Chr09 </td><td>10.0 </td><td>180 </td><td>148.83 </td><td>0.83 </td><td>4.15 </td><td>0 </td></tr><tr><td>Chr10 </td><td>10.0 </td><td>180 </td><td>200.94 </td><td>1.12 </td><td>17.93 </td><td>1 </td></tr><tr><td>Chr11 </td><td>13.0 </td><td>271 </td><td>234.77 </td><td>0.87 </td><td>6.09 </td><td>0 </td></tr><tr><td>Chr12 </td><td>14.0 </td><td>241 </td><td>238.05 </td><td>0.99 </td><td>6.26 </td><td>0 </td></tr><tr><td>Chr13 </td><td>10.0 </td><td>211 </td><td>208.14 </td><td>0.99 </td><td>6.56 </td><td>0 </td></tr><tr><td>A<sub>T</sub> genome </td><td> </td><td>2,473 </td><td>2,359.36 </td><td>0.95 </td><td>21.22 </td><td>4 </td></tr><tr><td>Chr14 </td><td>15.0 </td><td>180 </td><td>164.12 </td><td>0.91 </td><td>9.45 </td><td>0 </td></tr><tr><td>Chr15 </td><td>11.0 </td><td>215 </td><td>197.10 </td><td>0.92 </td><td>11.05 </td><td>1 </td></tr><tr><td>Chr16 </td><td>14.0 </td><td>179 </td><td>94.32 </td><td>0.53 </td><td>4.68 </td><td>0 </td></tr><tr><td>Chr17 </td><td>9.0 </td><td>150 </td><td>162.23 </td><td>1.08 </td><td>22.56 </td><td>2 </td></tr><tr><td>Chr18 </td><td>10.0 </td><td>188 </td><td>146.95 </td><td>0.78 </td><td>12.18 </td><td>1 </td></tr><tr><td>Chr19 </td><td>13.0 </td><td>306 </td><td>252.27 </td><td>0.82 </td><td>5.36 </td><td>0 </td></tr><tr><td>Chr20 </td><td>10.0 </td><td>183 </td><td>117.61 </td><td>0.64 </td><td>8.48 </td><td>0 </td></tr><tr><td>Chr21 </td><td>13.0 </td><td>265 </td><td>256.03 </td><td>0.97 </td><td>8.85 </td><td>0 </td></tr><tr><td>Chr22 </td><td>8.0 </td><td>147 </td><td>169.93 </td><td>1.16 </td><td>16.39 </td><td>1 </td></tr><tr><td>Chr23 </td><td>10.0 </td><td>209 </td><td>193.19 </td><td>0.92 </td><td>12.84 </td><td>2 </td></tr><tr><td>Chr24 </td><td>10.0 </td><td>225 </td><td>198.85 </td><td>0.88 </td><td>11.81 </td><td>1 </td></tr><tr><td>Chr25 </td><td>12.0 </td><td>194 </td><td>172.19 </td><td>0.89 </td><td>11.41 </td><td>1 </td></tr><tr><td>Chr26 </td><td>9.0 </td><td>238 </td><td>211.90 </td><td>0.89 </td><td>14.80 </td><td>2 </td></tr><tr><td>D<sub>T</sub> genome </td><td> </td><td>2,679 </td><td>2,336.67 </td><td>0.87 </td><td>22.56 </td><td>11 </td></tr><tr><td>Total </td><td> </td><td>5,152 </td><td>4,696.03 </td><td>0.91 </td><td>22.56 </td><td>15 </td></tr></tbody></table></div><div content-id="DSW016TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>LOD </th><th>Total loci </th><th>Size (cM) </th><th>Mean distance (cM) </th><th>Largest gap (cM) </th><th>&gt;10 cM makers </th></tr></thead><tbody><tr><td>Chr01 </td><td>10.0 </td><td>143 </td><td>186.87 </td><td>1.31 </td><td>21.22 </td><td>1 </td></tr><tr><td>Chr02 </td><td>12.0 </td><td>130 </td><td>156.03 </td><td>1.20 </td><td>10.67 </td><td>1 </td></tr><tr><td>Chr03 </td><td>15.0 </td><td>169 </td><td>164.93 </td><td>0.98 </td><td>9.49 </td><td>0 </td></tr><tr><td>Chr04 </td><td>12.0 </td><td>122 </td><td>149.82 </td><td>1.23 </td><td>7.85 </td><td>0 </td></tr><tr><td>Chr05 </td><td>13.0 </td><td>285 </td><td>242.76 </td><td>0.85 </td><td>9.16 </td><td>0 </td></tr><tr><td>Chr06 </td><td>9.0 </td><td>165 </td><td>171.43 </td><td>1.04 </td><td>10.41 </td><td>1 </td></tr><tr><td>Chr07 </td><td>14.0 </td><td>163 </td><td>105.78 </td><td>0.65 </td><td>3.42 </td><td>0 </td></tr><tr><td>Chr08 </td><td>11.0 </td><td>213 </td><td>151.02 </td><td>0.71 </td><td>9.38 </td><td>0 </td></tr><tr><td>Chr09 </td><td>10.0 </td><td>180 </td><td>148.83 </td><td>0.83 </td><td>4.15 </td><td>0 </td></tr><tr><td>Chr10 </td><td>10.0 </td><td>180 </td><td>200.94 </td><td>1.12 </td><td>17.93 </td><td>1 </td></tr><tr><td>Chr11 </td><td>13.0 </td><td>271 </td><td>234.77 </td><td>0.87 </td><td>6.09 </td><td>0 </td></tr><tr><td>Chr12 </td><td>14.0 </td><td>241 </td><td>238.05 </td><td>0.99 </td><td>6.26 </td><td>0 </td></tr><tr><td>Chr13 </td><td>10.0 </td><td>211 </td><td>208.14 </td><td>0.99 </td><td>6.56 </td><td>0 </td></tr><tr><td>A<sub>T</sub> genome </td><td> </td><td>2,473 </td><td>2,359.36 </td><td>0.95 </td><td>21.22 </td><td>4 </td></tr><tr><td>Chr14 </td><td>15.0 </td><td>180 </td><td>164.12 </td><td>0.91 </td><td>9.45 </td><td>0 </td></tr><tr><td>Chr15 </td><td>11.0 </td><td>215 </td><td>197.10 </td><td>0.92 </td><td>11.05 </td><td>1 </td></tr><tr><td>Chr16 </td><td>14.0 </td><td>179 </td><td>94.32 </td><td>0.53 </td><td>4.68 </td><td>0 </td></tr><tr><td>Chr17 </td><td>9.0 </td><td>150 </td><td>162.23 </td><td>1.08 </td><td>22.56 </td><td>2 </td></tr><tr><td>Chr18 </td><td>10.0 </td><td>188 </td><td>146.95 </td><td>0.78 </td><td>12.18 </td><td>1 </td></tr><tr><td>Chr19 </td><td>13.0 </td><td>306 </td><td>252.27 </td><td>0.82 </td><td>5.36 </td><td>0 </td></tr><tr><td>Chr20 </td><td>10.0 </td><td>183 </td><td>117.61 </td><td>0.64 </td><td>8.48 </td><td>0 </td></tr><tr><td>Chr21 </td><td>13.0 </td><td>265 </td><td>256.03 </td><td>0.97 </td><td>8.85 </td><td>0 </td></tr><tr><td>Chr22 </td><td>8.0 </td><td>147 </td><td>169.93 </td><td>1.16 </td><td>16.39 </td><td>1 </td></tr><tr><td>Chr23 </td><td>10.0 </td><td>209 </td><td>193.19 </td><td>0.92 </td><td>12.84 </td><td>2 </td></tr><tr><td>Chr24 </td><td>10.0 </td><td>225 </td><td>198.85 </td><td>0.88 </td><td>11.81 </td><td>1 </td></tr><tr><td>Chr25 </td><td>12.0 </td><td>194 </td><td>172.19 </td><td>0.89 </td><td>11.41 </td><td>1 </td></tr><tr><td>Chr26 </td><td>9.0 </td><td>238 </td><td>211.90 </td><td>0.89 </td><td>14.80 </td><td>2 </td></tr><tr><td>D<sub>T</sub> genome </td><td> </td><td>2,679 </td><td>2,336.67 </td><td>0.87 </td><td>22.56 </td><td>11 </td></tr><tr><td>Total </td><td> </td><td>5,152 </td><td>4,696.03 </td><td>0.91 </td><td>22.56 </td><td>15 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>LOD, logarithm of odds.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201851" target="_blank">View Large</a></div></div><p>The chromosome with the most loci was Chr19 (306 loci), whereas Chr04 had the fewest loci (122 loci), with average loci on each chromosome of 198. The longest chromosome was Chr21 (256.03 cM), whereas the shortest was Chr16 (94.32 cM), with an average chromosome length of 180.62 cM. The largest average distance between markers was on Chr01 (1.31 cM), and the least was on Chr16 (0.53 cM). The largest gap between markers was 22.56 cM on Chr17, and there were a total of 15 gaps &gt;10 cM with 4 on the A<sub>T</sub> and 11 on the D<sub>T</sub> genome.</p><p>Among the 5,632 polymorphic loci used for map construction, 1,006 loci (17.86%) showed segregation distortion (<em>P</em> &lt; 0.05) and 776 distorted loci, accounting for 15.06% of the mapped loci, were unevenly mapped on cotton chromosomes with 5–125 loci on each chromosome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The most distorted loci were on Chr02 (71), Chr16 (125), and Chr18 (106) (&gt;50% of loci were distorted), accounting for 38.92% of the mapped distorted loci. A total of 62 SDRs were found on 18 cotton chromosomes. More SDRs were found on Chr02 (9), Chr16 (11), and Chr18 (10), which were the chromosomes with the most distorted loci.</p>                            <h3 scrollto-destination=87201854 id="87201854" class="section-title" data-legacy-id=s3c>3.3. Collinearity in the cotton genome</h3>
<p>Based on the 4,807 available marker-derived sequences in the genetic linkage map, homology analysis showed that homologous chromosomes between A<sub>T</sub> and D<sub>T</sub> had the highest homology (Fig. <a reveal-id="DSW016F1" data-open="DSW016F1" class="link link-reveal link-table xref-fig">1</a>). In detail, seven homologous chromosome pairs (Chr01 and Chr15, Chr06 and Chr25, Chr08 and Chr24, Chr09 and Chr23, Chr11 and Chr21, Chr12 and Chr26, and Chr13 and Chr18) showed high collinearity. However, not the whole chromosomes showed collinearity between Chr10 and Chr20 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>), whereas Chr07 and Chr16 had good homology but hardly any collinearity (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>). In addition, although most of Chr02 had good collinearity with Chr14, the rest had comparatively good collinearity with Chr17 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A</a></span>). Additionally, most of Chr03 had good collinearity with Chr17, but the rest had comparatively good collinearity with Chr14 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B</a></span>). Moreover, this also appeared among Chr04, Chr05, Chr19, and Chr22 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3C and D</a></span>). </p>                        <a id="87201856" scrollto-destination="87201856"></a>
<div id="DSW016F1" data-id="DSW016F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01601.png?Expires=2147483647&Signature=GzqvqUmyiEwErSrQquIDO3EkH55Qyi~EmENrM-OrnKNAHHMO6crHSc-Yz0t~qBSWcVEHeMKH9JsEs7jMlx3mdR5tF3ablsFh7ra9R4d2vTjvWXdIjeR8Kk61LaZ84OH3wLp18sGx1cWps8svlCxwCTVuDZthYVZk3p1BCRbpdn-mf7px8y-nRPplgvSbQdQQqOOfXmnsdPmuDvhOjU7ANY5Vc~lXs-YI5UlhP0xFqa6VGWAKxTBaksC3Q4DS3TF1shsV~uz13L8HEgMZvoY092bf685YMKZP4jINkSPxBiBzJrxy6DpQyFsG7artkD2eg9bcdTdllLnQASlcKun4iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Collinearity among 26 chromosomes in tetraploid cotton. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01601.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201856/dsw01601.tiff" path-from-xml="dsw01601.tiff" target="_blank">View large</a><a section="87201856" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01601.png?Expires=2147483647&Signature=yJRx~enB5NG7Qi7p8vnN8I7f~7e7NPpYv7EOJep2RwYBA7WbPkF37dD5n5rwHkAaRhkpv~rdqOrQMwko7MWT89g1~1N~eQcm-m5UOK3cMSjURasqBAR81fN3N4nzlppsoDrNgXGEvldHX6hP1w-9AVLwSI0Jm5Vf64mGweGuwltX9hF03Vrb5pLLt40ZgWvQn4AO9CvORL4KJ02ti0IcvUA0cT1kfqs45Z30EozLC2Ochl3OFSQgPeTs36kwc80f1GxUWZRpxYsUDI9sFW0HE3bPYUl0oXoo5zu3W8zJPHNrB1Geg9HoF6NCdUKdP0XNBxP~YpwQ6vY92PysbO2Uhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201856&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01601.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Collinearity among 26 chromosomes in tetraploid cotton. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW016F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01601.png?Expires=2147483647&Signature=GzqvqUmyiEwErSrQquIDO3EkH55Qyi~EmENrM-OrnKNAHHMO6crHSc-Yz0t~qBSWcVEHeMKH9JsEs7jMlx3mdR5tF3ablsFh7ra9R4d2vTjvWXdIjeR8Kk61LaZ84OH3wLp18sGx1cWps8svlCxwCTVuDZthYVZk3p1BCRbpdn-mf7px8y-nRPplgvSbQdQQqOOfXmnsdPmuDvhOjU7ANY5Vc~lXs-YI5UlhP0xFqa6VGWAKxTBaksC3Q4DS3TF1shsV~uz13L8HEgMZvoY092bf685YMKZP4jINkSPxBiBzJrxy6DpQyFsG7artkD2eg9bcdTdllLnQASlcKun4iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Collinearity among 26 chromosomes in tetraploid cotton. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01601.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201856/dsw01601.tiff" path-from-xml="dsw01601.tiff" target="_blank">View large</a><a section="87201856" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01601.png?Expires=2147483647&Signature=yJRx~enB5NG7Qi7p8vnN8I7f~7e7NPpYv7EOJep2RwYBA7WbPkF37dD5n5rwHkAaRhkpv~rdqOrQMwko7MWT89g1~1N~eQcm-m5UOK3cMSjURasqBAR81fN3N4nzlppsoDrNgXGEvldHX6hP1w-9AVLwSI0Jm5Vf64mGweGuwltX9hF03Vrb5pLLt40ZgWvQn4AO9CvORL4KJ02ti0IcvUA0cT1kfqs45Z30EozLC2Ochl3OFSQgPeTs36kwc80f1GxUWZRpxYsUDI9sFW0HE3bPYUl0oXoo5zu3W8zJPHNrB1Geg9HoF6NCdUKdP0XNBxP~YpwQ6vY92PysbO2Uhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201856&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01601.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Collinearity among 26 chromosomes in tetraploid cotton. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87201857 id="87201857" class="section-title" data-legacy-id=s3d>3.4. Comparative genomics between the tetraploid cotton and diploid cottons</h3>
<p>Using the 4,807 available nucleotide sequences of 5,152 mapped markers and the genome sequence of <em>G. arboreum</em> or <em>G. raimondii</em>, we found by comparative genomics that sequences of 3,534 and 3,584 markers were homologous from the physical map of <em>G. arboreum</em> and <em>G. raimondii</em>, with an alignment proportion of 68.59 and 69.57%, respectively. The 3,534 markers blasted on the <em>G. arboreum</em> genome included 1,717 markers from the A<sub>T</sub> genome with an alignment proportion of 69.43%, and 1,817 markers from the D<sub>T</sub> genome with an alignment proportion of 67.82%. The 3,584 markers blasted on the <em>G. raimondii</em> genome included 1,631 markers from the A<sub>T</sub> genome with an alignment proportion of 65.95% and 1,953 markers from the D<sub>T</sub> genome with an alignment proportion of 72.90%.</p><p>In general, most chromosomes had good homology with their corresponding chromosomes in the diploid A or D genome (Figs <a reveal-id="DSW016F2" data-open="DSW016F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSW016F3" data-open="DSW016F3" class="link link-reveal link-table xref-fig">3</a>, and Table <a reveal-id="DSW016TB2" data-open="DSW016TB2" class="link link-reveal link-table xref-fig">2</a>). Exceptionally, during comparative analysis with the diploid A genome, three homologous chromosome pairs, Chr01/Chr15, Chr02/Chr14, and Chr03/Chr17, also showed good homology with A2, A5, and A7, respectively (Fig. <a reveal-id="DSW016F2" data-open="DSW016F2" class="link link-reveal link-table xref-fig">2</a> and Table <a reveal-id="DSW016TB2" data-open="DSW016TB2" class="link link-reveal link-table xref-fig">2</a>). During comparative analysis with the diploid D genome, Chr02, Chr03, Chr04, and Chr05 also showed good homology with D3, D5, D9, and D12, respectively (Fig. <a reveal-id="DSW016F3" data-open="DSW016F3" class="link link-reveal link-table xref-fig">3</a> and Table <a reveal-id="DSW016TB2" data-open="DSW016TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="87201860" scrollto-destination="87201860"></a>
<div content-id="DSW016TB2" class="table-modal"><div class="table-wrap"><div id="DSW016TB2" class="table-wrap-title" data-id="DSW016TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Chromosome collinearity between tetraploid cotton and diploid cotton</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest/higher homology with <em>G. arboreum</em> </th><th>Chromosome of highest/higher homology with <em>G. raimondii</em> </th><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest/higher homology with <em>G. arboreum</em> </th><th>Chromosome of highest/higher homology with <em>G. raimondii</em> </th></tr></thead><tbody><tr><td>Chr01 (A<sub>T</sub>1) </td><td>A7/A2 </td><td>D2 </td><td>Chr15 (D<sub>T</sub>1) </td><td>A7/A2 </td><td>D2 </td></tr><tr><td>Chr02 (A<sub>T</sub>2) </td><td>A5/A2 </td><td>D3/D5 </td><td>Chr14 (D<sub>T</sub>2) </td><td>A5 </td><td>D5 </td></tr><tr><td>Chr03 (A<sub>T</sub>3) </td><td>A5/A7 </td><td>D5/D3 </td><td>Chr17 (D<sub>T</sub>3) </td><td>A7/A2 </td><td>D3 </td></tr><tr><td>Chr04 (A<sub>T</sub>4) </td><td>A12 </td><td>D12/D9 </td><td>Chr22 (D<sub>T</sub>4) </td><td>A12 </td><td>D12 </td></tr><tr><td>Chr05 (A<sub>T</sub>5) </td><td>A10 </td><td>D9/D12 </td><td>Chr19 (D<sub>T</sub>5) </td><td>A10 </td><td>D9 </td></tr><tr><td>Chr06 (A<sub>T</sub>6) </td><td>A8 </td><td>D10 </td><td>Chr25 (D<sub>T</sub>6) </td><td>A8 </td><td>D10 </td></tr><tr><td>Chr07 (A<sub>T</sub>7) </td><td>A1 </td><td>D1 </td><td>Chr16 (D<sub>T</sub>7) </td><td>A1 </td><td>D1 </td></tr><tr><td>Chr08 (A<sub>T</sub>8) </td><td>A3 </td><td>D4 </td><td>Chr24 (D<sub>T</sub>8) </td><td>A3 </td><td>D4 </td></tr><tr><td>Chr09 (A<sub>T</sub>9) </td><td>A11 </td><td>D6 </td><td>Chr23 (D<sub>T</sub>9) </td><td>A11 </td><td>D6 </td></tr><tr><td>Chr10 (A<sub>T</sub>10) </td><td>A9 </td><td>D11 </td><td>Chr20 (D<sub>T</sub>10) </td><td>A9 </td><td>D11 </td></tr><tr><td>Chr11 (A<sub>T</sub>11) </td><td>A4 </td><td>D7 </td><td>Chr21 (D<sub>T</sub>11) </td><td>A4 </td><td>D7 </td></tr><tr><td>Chr12 (A<sub>T</sub>12) </td><td>A6 </td><td>D8 </td><td>Chr26 (D<sub>T</sub>12) </td><td>A6 </td><td>D8 </td></tr><tr><td>Chr13 (A<sub>T</sub>13) </td><td>A13 </td><td>D13 </td><td>Chr18 (D<sub>T</sub>13) </td><td>A13 </td><td>D13 </td></tr></tbody></table></div><div content-id="DSW016TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest/higher homology with <em>G. arboreum</em> </th><th>Chromosome of highest/higher homology with <em>G. raimondii</em> </th><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest/higher homology with <em>G. arboreum</em> </th><th>Chromosome of highest/higher homology with <em>G. raimondii</em> </th></tr></thead><tbody><tr><td>Chr01 (A<sub>T</sub>1) </td><td>A7/A2 </td><td>D2 </td><td>Chr15 (D<sub>T</sub>1) </td><td>A7/A2 </td><td>D2 </td></tr><tr><td>Chr02 (A<sub>T</sub>2) </td><td>A5/A2 </td><td>D3/D5 </td><td>Chr14 (D<sub>T</sub>2) </td><td>A5 </td><td>D5 </td></tr><tr><td>Chr03 (A<sub>T</sub>3) </td><td>A5/A7 </td><td>D5/D3 </td><td>Chr17 (D<sub>T</sub>3) </td><td>A7/A2 </td><td>D3 </td></tr><tr><td>Chr04 (A<sub>T</sub>4) </td><td>A12 </td><td>D12/D9 </td><td>Chr22 (D<sub>T</sub>4) </td><td>A12 </td><td>D12 </td></tr><tr><td>Chr05 (A<sub>T</sub>5) </td><td>A10 </td><td>D9/D12 </td><td>Chr19 (D<sub>T</sub>5) </td><td>A10 </td><td>D9 </td></tr><tr><td>Chr06 (A<sub>T</sub>6) </td><td>A8 </td><td>D10 </td><td>Chr25 (D<sub>T</sub>6) </td><td>A8 </td><td>D10 </td></tr><tr><td>Chr07 (A<sub>T</sub>7) </td><td>A1 </td><td>D1 </td><td>Chr16 (D<sub>T</sub>7) </td><td>A1 </td><td>D1 </td></tr><tr><td>Chr08 (A<sub>T</sub>8) </td><td>A3 </td><td>D4 </td><td>Chr24 (D<sub>T</sub>8) </td><td>A3 </td><td>D4 </td></tr><tr><td>Chr09 (A<sub>T</sub>9) </td><td>A11 </td><td>D6 </td><td>Chr23 (D<sub>T</sub>9) </td><td>A11 </td><td>D6 </td></tr><tr><td>Chr10 (A<sub>T</sub>10) </td><td>A9 </td><td>D11 </td><td>Chr20 (D<sub>T</sub>10) </td><td>A9 </td><td>D11 </td></tr><tr><td>Chr11 (A<sub>T</sub>11) </td><td>A4 </td><td>D7 </td><td>Chr21 (D<sub>T</sub>11) </td><td>A4 </td><td>D7 </td></tr><tr><td>Chr12 (A<sub>T</sub>12) </td><td>A6 </td><td>D8 </td><td>Chr26 (D<sub>T</sub>12) </td><td>A6 </td><td>D8 </td></tr><tr><td>Chr13 (A<sub>T</sub>13) </td><td>A13 </td><td>D13 </td><td>Chr18 (D<sub>T</sub>13) </td><td>A13 </td><td>D13 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201860" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW016TB2" class="table-wrap-title" data-id="DSW016TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Chromosome collinearity between tetraploid cotton and diploid cotton</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest/higher homology with <em>G. arboreum</em> </th><th>Chromosome of highest/higher homology with <em>G. raimondii</em> </th><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest/higher homology with <em>G. arboreum</em> </th><th>Chromosome of highest/higher homology with <em>G. raimondii</em> </th></tr></thead><tbody><tr><td>Chr01 (A<sub>T</sub>1) </td><td>A7/A2 </td><td>D2 </td><td>Chr15 (D<sub>T</sub>1) </td><td>A7/A2 </td><td>D2 </td></tr><tr><td>Chr02 (A<sub>T</sub>2) </td><td>A5/A2 </td><td>D3/D5 </td><td>Chr14 (D<sub>T</sub>2) </td><td>A5 </td><td>D5 </td></tr><tr><td>Chr03 (A<sub>T</sub>3) </td><td>A5/A7 </td><td>D5/D3 </td><td>Chr17 (D<sub>T</sub>3) </td><td>A7/A2 </td><td>D3 </td></tr><tr><td>Chr04 (A<sub>T</sub>4) </td><td>A12 </td><td>D12/D9 </td><td>Chr22 (D<sub>T</sub>4) </td><td>A12 </td><td>D12 </td></tr><tr><td>Chr05 (A<sub>T</sub>5) </td><td>A10 </td><td>D9/D12 </td><td>Chr19 (D<sub>T</sub>5) </td><td>A10 </td><td>D9 </td></tr><tr><td>Chr06 (A<sub>T</sub>6) </td><td>A8 </td><td>D10 </td><td>Chr25 (D<sub>T</sub>6) </td><td>A8 </td><td>D10 </td></tr><tr><td>Chr07 (A<sub>T</sub>7) </td><td>A1 </td><td>D1 </td><td>Chr16 (D<sub>T</sub>7) </td><td>A1 </td><td>D1 </td></tr><tr><td>Chr08 (A<sub>T</sub>8) </td><td>A3 </td><td>D4 </td><td>Chr24 (D<sub>T</sub>8) </td><td>A3 </td><td>D4 </td></tr><tr><td>Chr09 (A<sub>T</sub>9) </td><td>A11 </td><td>D6 </td><td>Chr23 (D<sub>T</sub>9) </td><td>A11 </td><td>D6 </td></tr><tr><td>Chr10 (A<sub>T</sub>10) </td><td>A9 </td><td>D11 </td><td>Chr20 (D<sub>T</sub>10) </td><td>A9 </td><td>D11 </td></tr><tr><td>Chr11 (A<sub>T</sub>11) </td><td>A4 </td><td>D7 </td><td>Chr21 (D<sub>T</sub>11) </td><td>A4 </td><td>D7 </td></tr><tr><td>Chr12 (A<sub>T</sub>12) </td><td>A6 </td><td>D8 </td><td>Chr26 (D<sub>T</sub>12) </td><td>A6 </td><td>D8 </td></tr><tr><td>Chr13 (A<sub>T</sub>13) </td><td>A13 </td><td>D13 </td><td>Chr18 (D<sub>T</sub>13) </td><td>A13 </td><td>D13 </td></tr></tbody></table></div><div content-id="DSW016TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest/higher homology with <em>G. arboreum</em> </th><th>Chromosome of highest/higher homology with <em>G. raimondii</em> </th><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest/higher homology with <em>G. arboreum</em> </th><th>Chromosome of highest/higher homology with <em>G. raimondii</em> </th></tr></thead><tbody><tr><td>Chr01 (A<sub>T</sub>1) </td><td>A7/A2 </td><td>D2 </td><td>Chr15 (D<sub>T</sub>1) </td><td>A7/A2 </td><td>D2 </td></tr><tr><td>Chr02 (A<sub>T</sub>2) </td><td>A5/A2 </td><td>D3/D5 </td><td>Chr14 (D<sub>T</sub>2) </td><td>A5 </td><td>D5 </td></tr><tr><td>Chr03 (A<sub>T</sub>3) </td><td>A5/A7 </td><td>D5/D3 </td><td>Chr17 (D<sub>T</sub>3) </td><td>A7/A2 </td><td>D3 </td></tr><tr><td>Chr04 (A<sub>T</sub>4) </td><td>A12 </td><td>D12/D9 </td><td>Chr22 (D<sub>T</sub>4) </td><td>A12 </td><td>D12 </td></tr><tr><td>Chr05 (A<sub>T</sub>5) </td><td>A10 </td><td>D9/D12 </td><td>Chr19 (D<sub>T</sub>5) </td><td>A10 </td><td>D9 </td></tr><tr><td>Chr06 (A<sub>T</sub>6) </td><td>A8 </td><td>D10 </td><td>Chr25 (D<sub>T</sub>6) </td><td>A8 </td><td>D10 </td></tr><tr><td>Chr07 (A<sub>T</sub>7) </td><td>A1 </td><td>D1 </td><td>Chr16 (D<sub>T</sub>7) </td><td>A1 </td><td>D1 </td></tr><tr><td>Chr08 (A<sub>T</sub>8) </td><td>A3 </td><td>D4 </td><td>Chr24 (D<sub>T</sub>8) </td><td>A3 </td><td>D4 </td></tr><tr><td>Chr09 (A<sub>T</sub>9) </td><td>A11 </td><td>D6 </td><td>Chr23 (D<sub>T</sub>9) </td><td>A11 </td><td>D6 </td></tr><tr><td>Chr10 (A<sub>T</sub>10) </td><td>A9 </td><td>D11 </td><td>Chr20 (D<sub>T</sub>10) </td><td>A9 </td><td>D11 </td></tr><tr><td>Chr11 (A<sub>T</sub>11) </td><td>A4 </td><td>D7 </td><td>Chr21 (D<sub>T</sub>11) </td><td>A4 </td><td>D7 </td></tr><tr><td>Chr12 (A<sub>T</sub>12) </td><td>A6 </td><td>D8 </td><td>Chr26 (D<sub>T</sub>12) </td><td>A6 </td><td>D8 </td></tr><tr><td>Chr13 (A<sub>T</sub>13) </td><td>A13 </td><td>D13 </td><td>Chr18 (D<sub>T</sub>13) </td><td>A13 </td><td>D13 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201860" target="_blank">View Large</a></div></div>                        <a id="87201861" scrollto-destination="87201861"></a>
<div id="DSW016F2" data-id="DSW016F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01602.png?Expires=2147483647&Signature=cOu7MGe4C020FCG0VLwHAfyXR3Be38~qAmw-1Sv3YBUwiq1xkk7x3GZz-ba-qTCFHUQusDnFV~B5cUUWCmZ-C2kWaYk-sgoBKs3P4DYl-woIRCYFgVS5KuUikFJAWSbLkkHuD2sdKX3wmCMpANRrwDZSzCaX0zX9kerHeUSZ2rUQIz0HD1jME8NAibkcg7pf8YsOOEZmUSq1d5nlObREDl4gmGIstpu5bKO8IVNvyLV17Ak7XszGZwgpmOFA9Axu6g7jhLU94sgDKjyHMaZuuwzCbUhTPekaMnD~UuOFKUqV7gEoeBnTwfMKJ8aQaSXkwynJjb5QPLPxT45jyfz20A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intuitive diagram of affinity between the linkage map and G. arboreum. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01602.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201861/dsw01602.tiff" path-from-xml="dsw01602.tiff" target="_blank">View large</a><a section="87201861" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01602.png?Expires=2147483647&Signature=XYWdx-v3xfY5S3bNzcL~FxxAdy~mGBaYfcZH-RtqejTztjWyEJI0IVyrUvPfo8pYE-JjxhKi5Bv~bbynb3Ka-CcPynnz8kROnZYfqOBf~1Eta9e3uwmMIlGgpor6fIIrvZtmfcZVtVu~Fqo9rUpBJM0AEI4G4mzz9WI2ON1tTB3WmmtaCOMsnLAn8Fn39n9O04FZroeM5aP~vCSUJNxI2tk5eZx3hQbawstbRQSJ6h7nDNd~PLw6bEpB6nDeR4-r824XR5bXV6xHLAMy9XFGTrV7djqzn6HcrRCQKmFrk~nhWSZ0TpnMP~3hhl6lCSzcgpuqUzy9RUmEDTvVEDkKmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201861&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01602.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intuitive diagram of affinity between the linkage map and <em>G. arboreum</em>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW016F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01602.png?Expires=2147483647&Signature=cOu7MGe4C020FCG0VLwHAfyXR3Be38~qAmw-1Sv3YBUwiq1xkk7x3GZz-ba-qTCFHUQusDnFV~B5cUUWCmZ-C2kWaYk-sgoBKs3P4DYl-woIRCYFgVS5KuUikFJAWSbLkkHuD2sdKX3wmCMpANRrwDZSzCaX0zX9kerHeUSZ2rUQIz0HD1jME8NAibkcg7pf8YsOOEZmUSq1d5nlObREDl4gmGIstpu5bKO8IVNvyLV17Ak7XszGZwgpmOFA9Axu6g7jhLU94sgDKjyHMaZuuwzCbUhTPekaMnD~UuOFKUqV7gEoeBnTwfMKJ8aQaSXkwynJjb5QPLPxT45jyfz20A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intuitive diagram of affinity between the linkage map and G. arboreum. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01602.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201861/dsw01602.tiff" path-from-xml="dsw01602.tiff" target="_blank">View large</a><a section="87201861" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01602.png?Expires=2147483647&Signature=XYWdx-v3xfY5S3bNzcL~FxxAdy~mGBaYfcZH-RtqejTztjWyEJI0IVyrUvPfo8pYE-JjxhKi5Bv~bbynb3Ka-CcPynnz8kROnZYfqOBf~1Eta9e3uwmMIlGgpor6fIIrvZtmfcZVtVu~Fqo9rUpBJM0AEI4G4mzz9WI2ON1tTB3WmmtaCOMsnLAn8Fn39n9O04FZroeM5aP~vCSUJNxI2tk5eZx3hQbawstbRQSJ6h7nDNd~PLw6bEpB6nDeR4-r824XR5bXV6xHLAMy9XFGTrV7djqzn6HcrRCQKmFrk~nhWSZ0TpnMP~3hhl6lCSzcgpuqUzy9RUmEDTvVEDkKmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201861&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01602.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intuitive diagram of affinity between the linkage map and <em>G. arboreum</em>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="87201862" scrollto-destination="87201862"></a>
<div id="DSW016F3" data-id="DSW016F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01603.png?Expires=2147483647&Signature=p0lzkCdapzmu5VnBA5pnjGVRhatLJ5XMXCuJM6oyhVsCBynoLbOKLwJFtSNJWpVj03eLVCcY8-dfFtDblmsNr30LQQZ4MqHpKaUTskdKP7~ogTpN1kO7N45TnVLaWCQSs7iO6ILrPp0teAs5DlH6q~fV17i7NlE08lILQPQr-chTCxA2OTNJj9XfQphrVHriFPStgn1AENaB~bHrMhrQjbQSTR22yZtZDt8yT2UndHeKWJu6oGeB2VKteiBI4wNuZgMIlMKpg39YIkZsck2myKyaAI9ECrWA6RyNi05lOUDVg~AHpzm2TiNaKraWt81HcNW4IcuT6Bkqwx0k01An1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intuitive diagram of affinity between the linkage map and G. raimondii. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01603.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201862/dsw01603.tiff" path-from-xml="dsw01603.tiff" target="_blank">View large</a><a section="87201862" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01603.png?Expires=2147483647&Signature=4K1lcKmcnrDE4eTdF2EVn8S3t6VDlDusYxMIpJV3xGwGCjSmhqXxggO7TK0aAQ8yB5XGu5sQPqgikq1T-xmwvG1ChjiMztUulqhw8SvkPL1ftK2PTpgytSp1gAFP0m0~WLfrdpS6BGnFT9P5~8-fQSc04edBjHLfOBGR19mMlCGKKY4QCQ1p32OfKs5bNpAoA7j5bNNebT~fzVvdjBitW55wU5eACNhfw0enqKrxvn6xFOUeWY0KMcN~pS4Yocc5bTRKvK6uacu3hRlaZTDmu9oiwoXwjaEDg01DdoNBhxk1-0GbgqMYj~52xm3O~UDRIZvoyMoGL1~1gj3ROQokCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201862&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01603.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intuitive diagram of affinity between the linkage map and <em>G. raimondii</em>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW016F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01603.png?Expires=2147483647&Signature=p0lzkCdapzmu5VnBA5pnjGVRhatLJ5XMXCuJM6oyhVsCBynoLbOKLwJFtSNJWpVj03eLVCcY8-dfFtDblmsNr30LQQZ4MqHpKaUTskdKP7~ogTpN1kO7N45TnVLaWCQSs7iO6ILrPp0teAs5DlH6q~fV17i7NlE08lILQPQr-chTCxA2OTNJj9XfQphrVHriFPStgn1AENaB~bHrMhrQjbQSTR22yZtZDt8yT2UndHeKWJu6oGeB2VKteiBI4wNuZgMIlMKpg39YIkZsck2myKyaAI9ECrWA6RyNi05lOUDVg~AHpzm2TiNaKraWt81HcNW4IcuT6Bkqwx0k01An1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intuitive diagram of affinity between the linkage map and G. raimondii. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01603.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201862/dsw01603.tiff" path-from-xml="dsw01603.tiff" target="_blank">View large</a><a section="87201862" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01603.png?Expires=2147483647&Signature=4K1lcKmcnrDE4eTdF2EVn8S3t6VDlDusYxMIpJV3xGwGCjSmhqXxggO7TK0aAQ8yB5XGu5sQPqgikq1T-xmwvG1ChjiMztUulqhw8SvkPL1ftK2PTpgytSp1gAFP0m0~WLfrdpS6BGnFT9P5~8-fQSc04edBjHLfOBGR19mMlCGKKY4QCQ1p32OfKs5bNpAoA7j5bNNebT~fzVvdjBitW55wU5eACNhfw0enqKrxvn6xFOUeWY0KMcN~pS4Yocc5bTRKvK6uacu3hRlaZTDmu9oiwoXwjaEDg01DdoNBhxk1-0GbgqMYj~52xm3O~UDRIZvoyMoGL1~1gj3ROQokCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201862&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01603.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intuitive diagram of affinity between the linkage map and <em>G. raimondii</em>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87201863 id="87201863" class="section-title" data-legacy-id=s3e>3.5. Blast analysis between the tetraploid cotton and the mixed genome of two diploid cottons</h3>
<p>A blast analysis of markers on the tetraploid cotton genetic linkage map was conducted by taking the <em>G. arboreum</em> and <em>G. raimondii</em> genomes together. Regardless of whether it is from the A<sub>T</sub> or D<sub>T</sub> genome, all homologous chromosome pairs, except for Chr01/Chr15, Chr02/Chr14, Chr03/Chr17, and Chr05/Chr19, had the highest homology with the corresponding homologous chromosome in the diploid A genome, and had the second highest homology with the corresponding homologous chromosome in the diploid D genome (Fig. <a reveal-id="DSW016F4" data-open="DSW016F4" class="link link-reveal link-table xref-fig">4</a> and Table <a reveal-id="DSW016TB3" data-open="DSW016TB3" class="link link-reveal link-table xref-fig">3</a>). Chr01 was predominantly homologous to A7, followed by A2 and D2; Chr15 had the highest homology with D2, followed by A2 and A7. Chr02 had the highest homology with A5, followed by A2, but very low homology with D5; Chr14 had the highest homology with A5, followed by D5, but very low homology with A2. Chr03 had the highest homology with A5, followed by A7, and low homology with D3; Chr17 had the highest homology with D3, followed by A7, and no homology with A5. Chr04 had the highest homology with A12, followed by D12 and D9; Chr22 had the highest homology with A12, followed by D12. Chr05 had the highest homology with A10, followed by D9 and A12; Chr19 had the highest homology with D9, followed by A10 (Fig. <a reveal-id="DSW016F4" data-open="DSW016F4" class="link link-reveal link-table xref-fig">4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>, and Table <a reveal-id="DSW016TB3" data-open="DSW016TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="87201865" scrollto-destination="87201865"></a>
<div content-id="DSW016TB3" class="table-modal"><div class="table-wrap"><div id="DSW016TB3" class="table-wrap-title" data-id="DSW016TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Blast analysis between the tetraploid cotton and the mixed genome of two diploid cottons</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest homology with the mixed genome of two diploid cottons </th><th>Chromosome of second highest homology with the mixed genome of two diploid cottons </th><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest homology with the mixed genome of two diploid cottons </th><th>Chromosome of second highest homology with the mixed genome of two diploid cottons </th></tr></thead><tbody><tr><td>Chr01 (A<sub>T</sub>1) </td><td>A7 </td><td>A2 </td><td>Chr15 (D<sub>T</sub>1) </td><td>D2 </td><td>A2 </td></tr><tr><td>Chr02 (A<sub>T</sub>2) </td><td>A5 </td><td>A2 </td><td>Chr14 (D<sub>T</sub>2) </td><td>A5 </td><td>D5 </td></tr><tr><td>Chr03 (A<sub>T</sub>3) </td><td>A5 </td><td>A7 </td><td>Chr17 (D<sub>T</sub>3) </td><td>D3 </td><td>A7 </td></tr><tr><td>Chr04 (A<sub>T</sub>4) </td><td>A12 </td><td>D12 </td><td>Chr22 (D<sub>T</sub>4) </td><td>A12 </td><td>D12 </td></tr><tr><td>Chr05 (A<sub>T</sub>5) </td><td>A10 </td><td>D9 </td><td>Chr19 (D<sub>T</sub>5) </td><td>D9 </td><td>A10 </td></tr><tr><td>Chr06 (A<sub>T</sub>6) </td><td>A8 </td><td>D10 </td><td>Chr25 (D<sub>T</sub>6) </td><td>A8 </td><td>D10 </td></tr><tr><td>Chr07 (A<sub>T</sub>7) </td><td>A1 </td><td>D1 </td><td>Chr16 (D<sub>T</sub>7) </td><td>A1 </td><td>D1 </td></tr><tr><td>Chr08 (A<sub>T</sub>8) </td><td>A3 </td><td>D4 </td><td>Chr24 (D<sub>T</sub>8) </td><td>A3 </td><td>D4 </td></tr><tr><td>Chr09 (A<sub>T</sub>9) </td><td>A11 </td><td>D6 </td><td>Chr23 (D<sub>T</sub>9) </td><td>A11 </td><td>D6 </td></tr><tr><td>Chr10 (A<sub>T</sub>10) </td><td>A9 </td><td>D11 </td><td>Chr20 (D<sub>T</sub>10) </td><td>A9 </td><td>D11 </td></tr><tr><td>Chr11 (A<sub>T</sub>11) </td><td>A4 </td><td>D7 </td><td>Chr21 (D<sub>T</sub>11) </td><td>A4 </td><td>D7 </td></tr><tr><td>Chr12 (A<sub>T</sub>12) </td><td>A6 </td><td>D8 </td><td>Chr26 (D<sub>T</sub>12) </td><td>A6 </td><td>D8 </td></tr><tr><td>Chr13 (A<sub>T</sub>13) </td><td>A13 </td><td>D13 </td><td>Chr18 (D<sub>T</sub>13) </td><td>A13 </td><td>D13 </td></tr></tbody></table></div><div content-id="DSW016TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest homology with the mixed genome of two diploid cottons </th><th>Chromosome of second highest homology with the mixed genome of two diploid cottons </th><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest homology with the mixed genome of two diploid cottons </th><th>Chromosome of second highest homology with the mixed genome of two diploid cottons </th></tr></thead><tbody><tr><td>Chr01 (A<sub>T</sub>1) </td><td>A7 </td><td>A2 </td><td>Chr15 (D<sub>T</sub>1) </td><td>D2 </td><td>A2 </td></tr><tr><td>Chr02 (A<sub>T</sub>2) </td><td>A5 </td><td>A2 </td><td>Chr14 (D<sub>T</sub>2) </td><td>A5 </td><td>D5 </td></tr><tr><td>Chr03 (A<sub>T</sub>3) </td><td>A5 </td><td>A7 </td><td>Chr17 (D<sub>T</sub>3) </td><td>D3 </td><td>A7 </td></tr><tr><td>Chr04 (A<sub>T</sub>4) </td><td>A12 </td><td>D12 </td><td>Chr22 (D<sub>T</sub>4) </td><td>A12 </td><td>D12 </td></tr><tr><td>Chr05 (A<sub>T</sub>5) </td><td>A10 </td><td>D9 </td><td>Chr19 (D<sub>T</sub>5) </td><td>D9 </td><td>A10 </td></tr><tr><td>Chr06 (A<sub>T</sub>6) </td><td>A8 </td><td>D10 </td><td>Chr25 (D<sub>T</sub>6) </td><td>A8 </td><td>D10 </td></tr><tr><td>Chr07 (A<sub>T</sub>7) </td><td>A1 </td><td>D1 </td><td>Chr16 (D<sub>T</sub>7) </td><td>A1 </td><td>D1 </td></tr><tr><td>Chr08 (A<sub>T</sub>8) </td><td>A3 </td><td>D4 </td><td>Chr24 (D<sub>T</sub>8) </td><td>A3 </td><td>D4 </td></tr><tr><td>Chr09 (A<sub>T</sub>9) </td><td>A11 </td><td>D6 </td><td>Chr23 (D<sub>T</sub>9) </td><td>A11 </td><td>D6 </td></tr><tr><td>Chr10 (A<sub>T</sub>10) </td><td>A9 </td><td>D11 </td><td>Chr20 (D<sub>T</sub>10) </td><td>A9 </td><td>D11 </td></tr><tr><td>Chr11 (A<sub>T</sub>11) </td><td>A4 </td><td>D7 </td><td>Chr21 (D<sub>T</sub>11) </td><td>A4 </td><td>D7 </td></tr><tr><td>Chr12 (A<sub>T</sub>12) </td><td>A6 </td><td>D8 </td><td>Chr26 (D<sub>T</sub>12) </td><td>A6 </td><td>D8 </td></tr><tr><td>Chr13 (A<sub>T</sub>13) </td><td>A13 </td><td>D13 </td><td>Chr18 (D<sub>T</sub>13) </td><td>A13 </td><td>D13 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201865" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW016TB3" class="table-wrap-title" data-id="DSW016TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Blast analysis between the tetraploid cotton and the mixed genome of two diploid cottons</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest homology with the mixed genome of two diploid cottons </th><th>Chromosome of second highest homology with the mixed genome of two diploid cottons </th><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest homology with the mixed genome of two diploid cottons </th><th>Chromosome of second highest homology with the mixed genome of two diploid cottons </th></tr></thead><tbody><tr><td>Chr01 (A<sub>T</sub>1) </td><td>A7 </td><td>A2 </td><td>Chr15 (D<sub>T</sub>1) </td><td>D2 </td><td>A2 </td></tr><tr><td>Chr02 (A<sub>T</sub>2) </td><td>A5 </td><td>A2 </td><td>Chr14 (D<sub>T</sub>2) </td><td>A5 </td><td>D5 </td></tr><tr><td>Chr03 (A<sub>T</sub>3) </td><td>A5 </td><td>A7 </td><td>Chr17 (D<sub>T</sub>3) </td><td>D3 </td><td>A7 </td></tr><tr><td>Chr04 (A<sub>T</sub>4) </td><td>A12 </td><td>D12 </td><td>Chr22 (D<sub>T</sub>4) </td><td>A12 </td><td>D12 </td></tr><tr><td>Chr05 (A<sub>T</sub>5) </td><td>A10 </td><td>D9 </td><td>Chr19 (D<sub>T</sub>5) </td><td>D9 </td><td>A10 </td></tr><tr><td>Chr06 (A<sub>T</sub>6) </td><td>A8 </td><td>D10 </td><td>Chr25 (D<sub>T</sub>6) </td><td>A8 </td><td>D10 </td></tr><tr><td>Chr07 (A<sub>T</sub>7) </td><td>A1 </td><td>D1 </td><td>Chr16 (D<sub>T</sub>7) </td><td>A1 </td><td>D1 </td></tr><tr><td>Chr08 (A<sub>T</sub>8) </td><td>A3 </td><td>D4 </td><td>Chr24 (D<sub>T</sub>8) </td><td>A3 </td><td>D4 </td></tr><tr><td>Chr09 (A<sub>T</sub>9) </td><td>A11 </td><td>D6 </td><td>Chr23 (D<sub>T</sub>9) </td><td>A11 </td><td>D6 </td></tr><tr><td>Chr10 (A<sub>T</sub>10) </td><td>A9 </td><td>D11 </td><td>Chr20 (D<sub>T</sub>10) </td><td>A9 </td><td>D11 </td></tr><tr><td>Chr11 (A<sub>T</sub>11) </td><td>A4 </td><td>D7 </td><td>Chr21 (D<sub>T</sub>11) </td><td>A4 </td><td>D7 </td></tr><tr><td>Chr12 (A<sub>T</sub>12) </td><td>A6 </td><td>D8 </td><td>Chr26 (D<sub>T</sub>12) </td><td>A6 </td><td>D8 </td></tr><tr><td>Chr13 (A<sub>T</sub>13) </td><td>A13 </td><td>D13 </td><td>Chr18 (D<sub>T</sub>13) </td><td>A13 </td><td>D13 </td></tr></tbody></table></div><div content-id="DSW016TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest homology with the mixed genome of two diploid cottons </th><th>Chromosome of second highest homology with the mixed genome of two diploid cottons </th><th>Chromosome in tetraploid cotton </th><th>Chromosome of highest homology with the mixed genome of two diploid cottons </th><th>Chromosome of second highest homology with the mixed genome of two diploid cottons </th></tr></thead><tbody><tr><td>Chr01 (A<sub>T</sub>1) </td><td>A7 </td><td>A2 </td><td>Chr15 (D<sub>T</sub>1) </td><td>D2 </td><td>A2 </td></tr><tr><td>Chr02 (A<sub>T</sub>2) </td><td>A5 </td><td>A2 </td><td>Chr14 (D<sub>T</sub>2) </td><td>A5 </td><td>D5 </td></tr><tr><td>Chr03 (A<sub>T</sub>3) </td><td>A5 </td><td>A7 </td><td>Chr17 (D<sub>T</sub>3) </td><td>D3 </td><td>A7 </td></tr><tr><td>Chr04 (A<sub>T</sub>4) </td><td>A12 </td><td>D12 </td><td>Chr22 (D<sub>T</sub>4) </td><td>A12 </td><td>D12 </td></tr><tr><td>Chr05 (A<sub>T</sub>5) </td><td>A10 </td><td>D9 </td><td>Chr19 (D<sub>T</sub>5) </td><td>D9 </td><td>A10 </td></tr><tr><td>Chr06 (A<sub>T</sub>6) </td><td>A8 </td><td>D10 </td><td>Chr25 (D<sub>T</sub>6) </td><td>A8 </td><td>D10 </td></tr><tr><td>Chr07 (A<sub>T</sub>7) </td><td>A1 </td><td>D1 </td><td>Chr16 (D<sub>T</sub>7) </td><td>A1 </td><td>D1 </td></tr><tr><td>Chr08 (A<sub>T</sub>8) </td><td>A3 </td><td>D4 </td><td>Chr24 (D<sub>T</sub>8) </td><td>A3 </td><td>D4 </td></tr><tr><td>Chr09 (A<sub>T</sub>9) </td><td>A11 </td><td>D6 </td><td>Chr23 (D<sub>T</sub>9) </td><td>A11 </td><td>D6 </td></tr><tr><td>Chr10 (A<sub>T</sub>10) </td><td>A9 </td><td>D11 </td><td>Chr20 (D<sub>T</sub>10) </td><td>A9 </td><td>D11 </td></tr><tr><td>Chr11 (A<sub>T</sub>11) </td><td>A4 </td><td>D7 </td><td>Chr21 (D<sub>T</sub>11) </td><td>A4 </td><td>D7 </td></tr><tr><td>Chr12 (A<sub>T</sub>12) </td><td>A6 </td><td>D8 </td><td>Chr26 (D<sub>T</sub>12) </td><td>A6 </td><td>D8 </td></tr><tr><td>Chr13 (A<sub>T</sub>13) </td><td>A13 </td><td>D13 </td><td>Chr18 (D<sub>T</sub>13) </td><td>A13 </td><td>D13 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201865" target="_blank">View Large</a></div></div>                        <a id="87201866" scrollto-destination="87201866"></a>
<div id="DSW016F4" data-id="DSW016F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01604.png?Expires=2147483647&Signature=LzCijFZUpOhG0VxT8PYkj86M9HeOfCZPgVCAPCRIa9a~g7p3wsodq2Vhp78wBMELCt5Ds3GXtViiVMM5h1cU~pZpHDf2WcBZ-prxaoip13-BB~MA-2If9fc9lOJg0trXYQwNR2go7AUQ312AZtTQkYi~Gu8l2aM9g1wIHigWVOtOY2dykVd1RFeTVzM7m1Apo0dO8-IxVsXOqt7SBDlmZtNahVnq06AF4-Uv8~JXCtkWR3ujdsCnwVxyrDfCTlnnYncc1S8lLFcrhRH96YELIPPq4zp0Zm4SIEEQsJhIQM6MQTBh7rD1ugt4tX89TcsB~PVvkRbrrjmcs62W1b1u~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intuitive diagram of blast analysis between tetraploid cotton and the mixed genome of two diploid cottons. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01604.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201866/dsw01604.tiff" path-from-xml="dsw01604.tiff" target="_blank">View large</a><a section="87201866" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01604.png?Expires=2147483647&Signature=kNRaQTiHSsAIjsLfYjOwXxXkBC3LJxWs4plMl~Rmt17pxAzv6q-ZC84fk9EdQEKFOo8gq4C1tk4Rg1WCqptpg1k46UbnudIYIh~cKZWyD9LbqppErHcVT3fmNchRuXahjjyaUgKhc8~rTIexgTwahti2IG7~SiS-z7qyY45k2c2TwiX7n78zz0mXpbjOy63TR8c-rxE3Phta0VYLeerfoSSMnjyyCrHOH~pw3~RavlqmBHortGdxgU0Ng9fbnS1-maztfZLzv732m5guxGPJEHW8xbKNKiZhnrM4OaJtN1C7ASRW9jiSTWBWvXPQ-3wxOFcTbCajjsLicgxn~H8xlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201866&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01604.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intuitive diagram of blast analysis between tetraploid cotton and the mixed genome of two diploid cottons. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW016F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01604.png?Expires=2147483647&Signature=LzCijFZUpOhG0VxT8PYkj86M9HeOfCZPgVCAPCRIa9a~g7p3wsodq2Vhp78wBMELCt5Ds3GXtViiVMM5h1cU~pZpHDf2WcBZ-prxaoip13-BB~MA-2If9fc9lOJg0trXYQwNR2go7AUQ312AZtTQkYi~Gu8l2aM9g1wIHigWVOtOY2dykVd1RFeTVzM7m1Apo0dO8-IxVsXOqt7SBDlmZtNahVnq06AF4-Uv8~JXCtkWR3ujdsCnwVxyrDfCTlnnYncc1S8lLFcrhRH96YELIPPq4zp0Zm4SIEEQsJhIQM6MQTBh7rD1ugt4tX89TcsB~PVvkRbrrjmcs62W1b1u~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intuitive diagram of blast analysis between tetraploid cotton and the mixed genome of two diploid cottons. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01604.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201866/dsw01604.tiff" path-from-xml="dsw01604.tiff" target="_blank">View large</a><a section="87201866" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01604.png?Expires=2147483647&Signature=kNRaQTiHSsAIjsLfYjOwXxXkBC3LJxWs4plMl~Rmt17pxAzv6q-ZC84fk9EdQEKFOo8gq4C1tk4Rg1WCqptpg1k46UbnudIYIh~cKZWyD9LbqppErHcVT3fmNchRuXahjjyaUgKhc8~rTIexgTwahti2IG7~SiS-z7qyY45k2c2TwiX7n78zz0mXpbjOy63TR8c-rxE3Phta0VYLeerfoSSMnjyyCrHOH~pw3~RavlqmBHortGdxgU0Ng9fbnS1-maztfZLzv732m5guxGPJEHW8xbKNKiZhnrM4OaJtN1C7ASRW9jiSTWBWvXPQ-3wxOFcTbCajjsLicgxn~H8xlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201866&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01604.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intuitive diagram of blast analysis between tetraploid cotton and the mixed genome of two diploid cottons. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h2 scrollto-destination=87201867 id="87201867" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=87201868 id="87201868" class="section-title" data-legacy-id=s4a>4.1. Different polymorphism ratios of markers</h3>
<p>High-density genetic linkage maps are becoming increasingly important. Because the polymorphism in cotton is low,<sup><a reveal-id="DSW016C28" data-open="DSW016C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSW016C29" data-open="DSW016C29" class="link link-ref link-reveal xref-bibr">29</a></sup> more researchers have paid attention to developing novel markers. Our laboratory also endeavoured to develop more molecular markers to construct high-density linkage maps in cotton to facilitate our cotton genetics and breeding studies. In this study, 200 HAU-prefixed new SSRs were developed from newly released ESTs from an adversity cDNA library of <em>G. barbadense</em> acc. Hai7124, which supplement our previously published markers.</p><p>Among all the SSR markers applied in this study, NAU-prefixed EST-SSRs showed the highest polymorphism ratio (78.57%), mainly owing to the artificial selection of polymorphic SSRs based on published results.<sup><a reveal-id="DSW016C30" data-open="DSW016C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Polymorphisms of HAU (20.00%), CCRI (21.19%), MON (33.89%), and NBRI (34.16%)-prefixed EST-SSRs were universally lower than that of MON (39.87%) and NBRI (36.50%)-prefixed gSSRs, which is in accordance with previous reports that ESTs are more conservative because of more selective pressure.<sup><a reveal-id="DSW016C16" data-open="DSW016C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW016C31" data-open="DSW016C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Polymorphisms of MON- and NBRI-prefixed EST-SSRs were much higher than that of HAU- and CCRI-prefixed EST-SSRs for the following reasons. MON-prefixed SSRs reliably generated two or more amplicons,<sup><a reveal-id="DSW016C32" data-open="DSW016C32" class="link link-ref link-reveal xref-bibr">32</a></sup> which raised the possibility of producing polymorphisms in each primer pair. NBRI-prefixed SSRs were developed from <em>G. herbaceum</em> (AA);<sup><a reveal-id="DSW016C33" data-open="DSW016C33" class="link link-ref link-reveal xref-bibr">33</a></sup> when they were used in tetraploid cottons (AADD), high polymorphism between <em>G. hirsutum</em> and <em>G. barbadense</em> may be detected. Generally speaking, the SSR markers in this study were comparatively highly polymorphic. One cause may be the great difference between the two parents, and the other would be the application of the SSCP method during genotyping analysis.</p><p>GhirPIP-prefixed markers<sup><a reveal-id="DSW016C13" data-open="DSW016C13" class="link link-ref link-reveal xref-bibr">13</a></sup> (containing intron single-nucleotide polymorphisms and intron length polymorphisms with a predicted ratio of 3 : 1), cg-prefixed SNP/InDel markers<sup><a reveal-id="DSW016C14" data-open="DSW016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (developed from 3′ end, 5′ end, or intron sequences), and cot-prefixed SNP/InDel markers<sup><a reveal-id="DSW016C14" data-open="DSW016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (developed from conserved orthologous sets) all showed low polymorphism (4.66, 11.30, and 13.89%, respectively), mainly because the SSCP method applied in this study has its recognized limits in reflecting SNPs/InDels. The reason why GhirPIP-prefixed markers showed such a low polymorphism rate (4.66%) may be that they were developed from the predicted introns of <em>G. hirsutum</em> based on the complete genome sequence of model plants, which existed deviation undoubtedly.</p><p>Thus, together with previous reports that SSR markers are highly reproducible across species,<sup><a reveal-id="DSW016C34" data-open="DSW016C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and could provide more convenient assays of collinearity between different genomes,<sup><a reveal-id="DSW016C35" data-open="DSW016C35" class="link link-ref link-reveal xref-bibr">35</a></sup> SSRs are ideal markers for map construction.</p>                            <h3 scrollto-destination=87201873 id="87201873" class="section-title" data-legacy-id=s4b>4.2. Characteristics of genome structure in tetraploid cotton</h3>
<p>As previously reported, a high-density genetic linkage map could accelerate genome structure analysis. The present map (5,152 loci, 4696.03 cM in length with an average distance of 0.91 cM; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and Table <a reveal-id="DSW016TB1" data-open="DSW016TB1" class="link link-reveal link-table xref-fig">1</a>) revealed that (i) more loci were found on the D<sub>T</sub> genome than on the A<sub>T</sub> genome, consistent with the results of Guo et al.<sup><a reveal-id="DSW016C4" data-open="DSW016C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and Yu et al.,<sup><a reveal-id="DSW016C5" data-open="DSW016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> but inconsistent with those of Rong et al.<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and Yu et al.;<sup><a reveal-id="DSW016C6" data-open="DSW016C6" class="link link-ref link-reveal xref-bibr">6</a></sup> (ii) the D<sub>T</sub> genome was shorter than the A<sub>T</sub> genome, consistent with the results of Rong et al.,<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Yu et al.,<sup><a reveal-id="DSW016C5" data-open="DSW016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and Yu et al.,<sup><a reveal-id="DSW016C6" data-open="DSW016C6" class="link link-ref link-reveal xref-bibr">6</a></sup> but inconsistent with those of Guo et al.;<sup><a reveal-id="DSW016C4" data-open="DSW016C4" class="link link-ref link-reveal xref-bibr">4</a></sup> (iii) the average marker distance of the D<sub>T</sub> genome is shorter than that of the A<sub>T</sub> genome, consistent with the results of Rong et al.,<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Guo et al.,<sup><a reveal-id="DSW016C4" data-open="DSW016C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and Yu et al.,<sup><a reveal-id="DSW016C5" data-open="DSW016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> but inconsistent with those of Yu et al.;<sup><a reveal-id="DSW016C6" data-open="DSW016C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and (iv) there were more gaps (&gt;10 cM) on the D<sub>T</sub> genome than on the A<sub>T</sub> genome, consistent with the results of Yu et al.,<sup><a reveal-id="DSW016C5" data-open="DSW016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> but inconsistent with those of Rong et al.,<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Guo et al.,<sup><a reveal-id="DSW016C4" data-open="DSW016C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and Yu et al.<sup><a reveal-id="DSW016C6" data-open="DSW016C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Variations among the five genetic linkage maps are likely the results of differences in the mapping population types and sizes, as well as in the numbers and sources of molecular markers.</p><p>In this study, 776 of the total 1,006 distorted loci were mapped on the 26 cotton chromosomes, and 62 SDRs were discovered. Further analysis showed that three chromosomes (Chr02, Chr16, and Chr18) had extreme segregation distortion (&gt;50% of loci showing distortion, accounting for 38.92% of the mapped distorted loci, and containing 30 SDRs), which has been identified by our previous study.<sup><a reveal-id="DSW016C5" data-open="DSW016C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Because segregation distortion is increasingly recognized as a potentially powerful evolutionary force,<sup><a reveal-id="DSW016C36" data-open="DSW016C36" class="link link-ref link-reveal xref-bibr">36</a></sup> the present results suggest the prospect for a wider application. In addition, segregation distortion mechanisms of these three chromosomes are being researched using eight reciprocal backcrossing populations, which were financially supported by the National Science Foundation of China (Grant No. 31171593).</p><p>Tetraploid cotton, containing A<sub>T</sub> and D<sub>T</sub> genomes, was formed from an interspecific hybridization event between diploid A and diploid D cotton species that may have evolved from a common ancestor.<sup><a reveal-id="DSW016C37" data-open="DSW016C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSW016C38" data-open="DSW016C38" class="link link-ref link-reveal xref-bibr">38</a></sup> As a result, 13 expected homologous A<sub>T</sub>/D<sub>T</sub> chromosome pairs were further confirmed by the present results that the highest homology or collinearity existed between them (Fig. <a reveal-id="DSW016F1" data-open="DSW016F1" class="link link-reveal link-table xref-fig">1</a>), which is in accordance with previous reports.<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> These results may be useful in identifying candidate genes. For example, if one quantitative trait locus (QTL) was detected on the A<sub>T</sub> chromosome, a candidate gene controlling the same trait could be predicted on the corresponding collinear segment of the D<sub>T</sub> chromosome, so that we can study the functional divergence of homologous genes. However, it is notable that an obvious inversion appeared on Chr10 or Chr20, and repeated inversions appeared on Chr07 or Chr16 (Fig. <a reveal-id="DSW016F1" data-open="DSW016F1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). In addition, two reciprocal translocations between Chr02 and Chr03 and between Chr04 and Chr05 were confirmed in this study (Fig. <a reveal-id="DSW016F1" data-open="DSW016F1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), corresponding to previous reports.<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW016C6" data-open="DSW016C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSW016C28" data-open="DSW016C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSW016C39" data-open="DSW016C39" class="link link-ref link-reveal xref-bibr">39</a></sup> All of the above may result from genome rearrangements during or after the polyploidization process of the two ancestral diploid genomes, indicating complex but linear features of the tetraploid cotton genome.<sup><a reveal-id="DSW016C6" data-open="DSW016C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Another possibility is that the conformation of Chr02, Chr03, Chr04, and Chr05 in the A<sub>T</sub> genome may result from fracture and fusion of ancient chromosomes, since diploid cotton is a paleohexaploid with a radix of seven.<sup><a reveal-id="DSW016C12" data-open="DSW016C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p>                            <h3 scrollto-destination=87201877 id="87201877" class="section-title" data-legacy-id=s4c>4.3. Good collinearity between the A<sub>T</sub>/D<sub>T</sub> genome in tetraploid and diploid A/D genomes</h3>
<p>Comparative genomic analysis between tetraploid cotton and diploid cottons showed that 12 chromosomes in the A<sub>T</sub> genome, except for Chr02 (A<sub>T</sub>2), showed good collinearity with chromosomes in the diploid A genome (Fig. <a reveal-id="DSW016F2" data-open="DSW016F2" class="link link-reveal link-table xref-fig">2</a> and Table <a reveal-id="DSW016TB2" data-open="DSW016TB2" class="link link-reveal link-table xref-fig">2</a>); and 13 chromosomes in the D<sub>T</sub> genome showed good collinearity with chromosomes in the diploid D genome (Fig. <a reveal-id="DSW016F3" data-open="DSW016F3" class="link link-reveal link-table xref-fig">3</a> and Table <a reveal-id="DSW016TB2" data-open="DSW016TB2" class="link link-reveal link-table xref-fig">2</a>), which is consistent with the results of Li et al.<sup><a reveal-id="DSW016C9" data-open="DSW016C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and Wang et al.,<sup><a reveal-id="DSW016C8" data-open="DSW016C8" class="link link-ref link-reveal xref-bibr">8</a></sup> respectively, and is consistent with the report that no major structural changes exist between the D<sub>T</sub> and D chromosomes, but rather between the A<sub>T</sub> and A chromosomes.<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW016C40" data-open="DSW016C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Moreover, homology between the A<sub>T</sub> genome and the diploid D genome (Fig. <a reveal-id="DSW016F3" data-open="DSW016F3" class="link link-reveal link-table xref-fig">3</a> and Table <a reveal-id="DSW016TB2" data-open="DSW016TB2" class="link link-reveal link-table xref-fig">2</a>), as well as between the D<sub>T</sub> genome and the diploid A genome (Fig. <a reveal-id="DSW016F2" data-open="DSW016F2" class="link link-reveal link-table xref-fig">2</a> and Table <a reveal-id="DSW016TB2" data-open="DSW016TB2" class="link link-reveal link-table xref-fig">2</a>), supplied further evidence to define 13 homologous chromosome pairs in tetraploid cotton. In addition, it also provided evidence for collinearity between the diploid A and D genomes, as well as homology between chromosomes from different diploid genomes. The present results are a complete unification of the reports that are entirely based on the genome sequences of two diploid species.<sup><a reveal-id="DSW016C9" data-open="DSW016C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p>                            <h3 scrollto-destination=87201879 id="87201879" class="section-title" data-legacy-id=s4d>4.4. The diploid A genome dominates the tetraploid cotton genome</h3>
<p>It is well known that tetraploid cotton was formed from an interspecific hybridization event between an A-genome species and a D-genome species,<sup><a reveal-id="DSW016C4" data-open="DSW016C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and the diploid A genome is nearly 2-fold larger than the D genome, although they diverged from a common ancestor ∼5–10 million years ago.<sup><a reveal-id="DSW016C38" data-open="DSW016C38" class="link link-ref link-reveal xref-bibr">38</a></sup> However, previous studies,<sup><a reveal-id="DSW016C3" data-open="DSW016C3" class="link link-ref link-reveal xref-bibr">3</a>–<a reveal-id="DSW016C6" data-open="DSW016C6" class="link link-ref link-reveal xref-bibr">6</a></sup> as well as the present study, reported that there is a minor difference between the A<sub>T</sub> and D<sub>T</sub> genomes in tetraploid cotton. Previous reports that the A<sub>T</sub> genome is more stable than the D<sub>T</sub> genome,<sup><a reveal-id="DSW016C28" data-open="DSW016C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and the present fact that regardless of an A<sub>T</sub> or D<sub>T</sub> genome origin, most chromosomes had the highest homology with the corresponding homologous chromosomes in the diploid A genome (Fig. <a reveal-id="DSW016F4" data-open="DSW016F4" class="link link-reveal link-table xref-fig">4</a> and Table <a reveal-id="DSW016TB3" data-open="DSW016TB3" class="link link-reveal link-table xref-fig">3</a>), support the inference that during the hybridization evolution of tetraploid cotton, the D<sub>T</sub> genome underwent invasion of the diploid A genome (Fig. <a reveal-id="DSW016F5" data-open="DSW016F5" class="link link-reveal link-table xref-fig">5</a>). This result may partly explain why the diploid D-genome species do not produce spinnable fibre,<sup><a reveal-id="DSW016C41" data-open="DSW016C41" class="link link-ref link-reveal xref-bibr">41</a></sup> while many QTLs for fibre-related traits have been detected in the D<sub>T</sub> genome of tetraploid cotton.<sup><a reveal-id="DSW016C42" data-open="DSW016C42" class="link link-ref link-reveal xref-bibr">42</a>–<a reveal-id="DSW016C44" data-open="DSW016C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p>                        <a id="87201881" scrollto-destination="87201881"></a>
<div id="DSW016F5" data-id="DSW016F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01605.png?Expires=2147483647&Signature=Ye4EsYJLK5WV0nzZ3H10U8Z71w5CxwVA-UWzosDOZCuFyN1-udjWuSDJlj9cSIIxvLeiTunTaagq5-0LufSUkSEF~3vD0FJWEQXt1poei21rx99MQVFbIEs7a6~Xn-b2tclA6dz-TkJNn4~gx~ylwwYh4Mxm62Kgc8RXPH76iWs~46T5DUrsxTqx577nHmTtktWRNXsBkPKG9P4za94-lQaAogO3u8IUW8OCoNXWIsDsly9zllkWvQ1kC-uz2gVzhjDYSTug9WXCyfwZp8bkpqp3LlL0cYbs5SP4T4o0kGRaMcuP3ZveQHx2F~pA71kfY2Uk1rRmbuBRFtL9UF9LWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic drawing that illustrates the evolution of most chromosomes. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01605.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201881/dsw01605.tiff" path-from-xml="dsw01605.tiff" target="_blank">View large</a><a section="87201881" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01605.png?Expires=2147483647&Signature=tHDavXnISLyLw1FNy7dtHk1-gpgRRxW0hjtT9ENGeoCr7iMfBKNFVxhKk1x80zeClCeEuymnnGwYbT7FByTHG1N4-l9J2FkdFPk9Qh3FSu2MQYGGyRBMRTsf166raKwgG9ogWVQCmPASEnmR6TfwlVwKIjYapfhXjuHvrJR5UI-uPfwXxpfVmMxph7p4PuTlNIe3xRLkNwmmCO-wCZHtUA~JoINlfPciKXhDpmsvWHNUWah7nbVLHO09iRqEw9OIQ3K2qpEGZkpJi-~XlUdLuI4robqrwE7DCDComYkQWxRWuFRTIBnhX5dlyP40hLjtkVlQ1Vtqwy60qpXUZJYa7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201881&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01605.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic drawing that illustrates the evolution of most chromosomes. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW016F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01605.png?Expires=2147483647&Signature=Ye4EsYJLK5WV0nzZ3H10U8Z71w5CxwVA-UWzosDOZCuFyN1-udjWuSDJlj9cSIIxvLeiTunTaagq5-0LufSUkSEF~3vD0FJWEQXt1poei21rx99MQVFbIEs7a6~Xn-b2tclA6dz-TkJNn4~gx~ylwwYh4Mxm62Kgc8RXPH76iWs~46T5DUrsxTqx577nHmTtktWRNXsBkPKG9P4za94-lQaAogO3u8IUW8OCoNXWIsDsly9zllkWvQ1kC-uz2gVzhjDYSTug9WXCyfwZp8bkpqp3LlL0cYbs5SP4T4o0kGRaMcuP3ZveQHx2F~pA71kfY2Uk1rRmbuBRFtL9UF9LWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic drawing that illustrates the evolution of most chromosomes. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01605.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201881/dsw01605.tiff" path-from-xml="dsw01605.tiff" target="_blank">View large</a><a section="87201881" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01605.png?Expires=2147483647&Signature=tHDavXnISLyLw1FNy7dtHk1-gpgRRxW0hjtT9ENGeoCr7iMfBKNFVxhKk1x80zeClCeEuymnnGwYbT7FByTHG1N4-l9J2FkdFPk9Qh3FSu2MQYGGyRBMRTsf166raKwgG9ogWVQCmPASEnmR6TfwlVwKIjYapfhXjuHvrJR5UI-uPfwXxpfVmMxph7p4PuTlNIe3xRLkNwmmCO-wCZHtUA~JoINlfPciKXhDpmsvWHNUWah7nbVLHO09iRqEw9OIQ3K2qpEGZkpJi-~XlUdLuI4robqrwE7DCDComYkQWxRWuFRTIBnhX5dlyP40hLjtkVlQ1Vtqwy60qpXUZJYa7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201881&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01605.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic drawing that illustrates the evolution of most chromosomes. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>If the formation of Chr04/Chr22 and Chr05/Chr19 was similar to other homologous chromosome pairs, their structure should be as shown in Fig. <a reveal-id="DSW016F6" data-open="DSW016F6" class="link link-reveal link-table xref-fig">6</a>A. However, blast analysis between the tetraploid cotton genome and the mixed genome of two diploid cottons showed that Chr04 had the highest homology with A12, followed by D12 and D9 that were equal to each other. Additionally, Chr05 had the highest homology with A10, followed by D9 and A12 that were equal to each other (Fig. <a reveal-id="DSW016F4" data-open="DSW016F4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Thus, it is predicted that Chr04 should have a segment of D9, and Chr05 should have a segment of A12 (as shown in Fig. <a reveal-id="DSW016F6" data-open="DSW016F6" class="link link-reveal link-table xref-fig">6</a>C). This prediction could also be confirmed by comparative genomics between the tetraploid cotton and diploid cottons (Figs <a reveal-id="DSW016F2" data-open="DSW016F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSW016F3" data-open="DSW016F3" class="link link-reveal link-table xref-fig">3</a>, and Table <a reveal-id="DSW016TB2" data-open="DSW016TB2" class="link link-reveal link-table xref-fig">2</a>). In that way, chromosomal translocations must occur during interspecific hybridization events as shown in Fig. <a reveal-id="DSW016F6" data-open="DSW016F6" class="link link-reveal link-table xref-fig">6</a>B. In detail, one chromosome translocation occurred between Chr04 and Chr05, and another one occurred between Chr04 and Chr19. Similarly, considering the results of comparative genomics between the tetraploid cotton and diploid cottons, and the results of blast analysis between the tetraploid cotton genome and the mixed genome of two diploid cottons, a series of chromosomal translocations among Chr01/Chr15, Chr02/Chr14, and Chr03/Chr17 could also be predicted (Fig. <a reveal-id="DSW016F7" data-open="DSW016F7" class="link link-reveal link-table xref-fig">7</a>). All of the above demonstrate that these five homologous chromosome pairs changed in a complex and dramatic manner in that they coevolved during polyploidization, while other chromosomes in the A<sub>T</sub> genome just happened minor variations and those in the D<sub>T</sub> genome just underwent invasion of the corresponding homologous chromosome in the diploid A genome. </p>                        <a id="87201883" scrollto-destination="87201883"></a>
<div id="DSW016F6" data-id="DSW016F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01606.png?Expires=2147483647&Signature=vgeYPMGCjeIu92VnCTJuRhY23o39GOS45toPs0HcknhAWABD9tt93Pq7XvziVnWCZstSOzD-i2xRl4kTM2C9H3ReBhr-HHgFA57WmCCEGv-wy~YUGYHi1LJcaFo5YVT7dAwIHKT6SuI5UydKp9TeLW8se3dfLxUM8qMGh~KEGsgFrZ78EVdFu6E-aTZoCM-yaoq962Rw~szIPsdbrNZKnGghFoV15EetGYhq7SjCAMl7gIbqgwmwPcgsxQqK4tp0Fz8nIREFU3QkkjIPd6WidZbN-rj1WIF~RN3tf35PJkEg-yhjgd8uJ3FtbsPEdp4fZfLtPr-SUxL2tTANYZpwkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic drawing that illustrates the evolution of Chr04/Chr22 and Chr05/Chr19. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01606.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201883/dsw01606.tiff" path-from-xml="dsw01606.tiff" target="_blank">View large</a><a section="87201883" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01606.png?Expires=2147483647&Signature=aItRXy2m4naAnz0DAmeIId8IUNEt29DY1quhS9F16OKyQ0tUXA9TiD8rO9OIEP7hhpYjdYWvPwr5SMVmziVXEDrQEW6PBvDwYZ8jA4M6kZNCm9DLoJZGb2EaeHYuuh4bcA3OhkAAyJnJTQuV083hyti4uS0b-UYRHYd9OW-Bt9tQwWmWIsnbGOgvX1M5slk5izV3iHJK5O2sNdHwkwmiO~znoheFFh8bXJ3ggpU8wZfARjO~QC4iw3yk92hIRCfFBy19PGj~cbrT~Uc5VYaL-J6fK1qbel-uV4vpkOPB39Q9ZM8~o0y9vuS5QE4fpZuuuD1lYFL4g~pzzTeWwqbVNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201883&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01606.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic drawing that illustrates the evolution of Chr04/Chr22 and Chr05/Chr19. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW016F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01606.png?Expires=2147483647&Signature=vgeYPMGCjeIu92VnCTJuRhY23o39GOS45toPs0HcknhAWABD9tt93Pq7XvziVnWCZstSOzD-i2xRl4kTM2C9H3ReBhr-HHgFA57WmCCEGv-wy~YUGYHi1LJcaFo5YVT7dAwIHKT6SuI5UydKp9TeLW8se3dfLxUM8qMGh~KEGsgFrZ78EVdFu6E-aTZoCM-yaoq962Rw~szIPsdbrNZKnGghFoV15EetGYhq7SjCAMl7gIbqgwmwPcgsxQqK4tp0Fz8nIREFU3QkkjIPd6WidZbN-rj1WIF~RN3tf35PJkEg-yhjgd8uJ3FtbsPEdp4fZfLtPr-SUxL2tTANYZpwkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic drawing that illustrates the evolution of Chr04/Chr22 and Chr05/Chr19. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01606.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201883/dsw01606.tiff" path-from-xml="dsw01606.tiff" target="_blank">View large</a><a section="87201883" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01606.png?Expires=2147483647&Signature=aItRXy2m4naAnz0DAmeIId8IUNEt29DY1quhS9F16OKyQ0tUXA9TiD8rO9OIEP7hhpYjdYWvPwr5SMVmziVXEDrQEW6PBvDwYZ8jA4M6kZNCm9DLoJZGb2EaeHYuuh4bcA3OhkAAyJnJTQuV083hyti4uS0b-UYRHYd9OW-Bt9tQwWmWIsnbGOgvX1M5slk5izV3iHJK5O2sNdHwkwmiO~znoheFFh8bXJ3ggpU8wZfARjO~QC4iw3yk92hIRCfFBy19PGj~cbrT~Uc5VYaL-J6fK1qbel-uV4vpkOPB39Q9ZM8~o0y9vuS5QE4fpZuuuD1lYFL4g~pzzTeWwqbVNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201883&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01606.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic drawing that illustrates the evolution of Chr04/Chr22 and Chr05/Chr19. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="87201884" scrollto-destination="87201884"></a>
<div id="DSW016F7" data-id="DSW016F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01607.png?Expires=2147483647&Signature=skovP3l~wxRTQkLSDrr4Cy0jtDV1qQj9KNyj1kt0GteWP12HyVTGjVVtT9CGSGBl8euFegJ-V0-3Zwmkl61XsZgAwjTWqCHef6afgVZt9kGmT736hc0fDZLRf3NlSX~47OV8lMFQgf2ckyguSLn2SUz6BFKQ7kPGE0UtJRCt4-5QX3XXg2VfTJxa1Nw5qqm50iypl6DSDGZqMbwPBzjpPaxmm~ld7kKfG7fjPLjjf2cePLFmFEZaaqrXAEBfRy8hypYlMS0twpr-g7rE-7igEADH82TIWpyoLTYP4kI2vbJjJHVezRQdtXZGLjese7k4dGxXfVWITcGwjp11dNsi5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic drawing that illustrates the evolution of Chr01/Chr15, Chr02/Chr14, and Chr03/Chr17. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01607.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201884/dsw01607.tiff" path-from-xml="dsw01607.tiff" target="_blank">View large</a><a section="87201884" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01607.png?Expires=2147483647&Signature=AtDKfTfGP~lSBTkAVyhafjoHdksY5cncjuHiNOCWiFPHEGUJDLA3jPoFE7uZ9lqph6chgt1SsrUXgWyaYVM-U5lhYIr5RSaGNDt6Keu6yKS~SuKbhRrGFMljgi0N44VGxElndusFLuLGLbxkNHMUNgbBhlPO5pWOflaiWB58xixaK8isg-b1JEMfa9CikD7y7pSvqiKjCU9TbwwCx9Ml0AUzB65Tkz3ohOTbirg8879UycKfLYy6VqiDs1z5TiWIII1dblYHWe083g8kUl5iGQZXFfzYQmO7M85kgaACGHJt6usOwp~vlOwQcmVSRLzp~LOSGhwFmjqB-Pu9KmZxWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201884&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01607.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic drawing that illustrates the evolution of Chr01/Chr15, Chr02/Chr14, and Chr03/Chr17. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW016F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/m_dsw01607.png?Expires=2147483647&Signature=skovP3l~wxRTQkLSDrr4Cy0jtDV1qQj9KNyj1kt0GteWP12HyVTGjVVtT9CGSGBl8euFegJ-V0-3Zwmkl61XsZgAwjTWqCHef6afgVZt9kGmT736hc0fDZLRf3NlSX~47OV8lMFQgf2ckyguSLn2SUz6BFKQ7kPGE0UtJRCt4-5QX3XXg2VfTJxa1Nw5qqm50iypl6DSDGZqMbwPBzjpPaxmm~ld7kKfG7fjPLjjf2cePLFmFEZaaqrXAEBfRy8hypYlMS0twpr-g7rE-7igEADH82TIWpyoLTYP4kI2vbJjJHVezRQdtXZGLjese7k4dGxXfVWITcGwjp11dNsi5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic drawing that illustrates the evolution of Chr01/Chr15, Chr02/Chr14, and Chr03/Chr17. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01607.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201884/dsw01607.tiff" path-from-xml="dsw01607.tiff" target="_blank">View large</a><a section="87201884" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw01607.png?Expires=2147483647&Signature=AtDKfTfGP~lSBTkAVyhafjoHdksY5cncjuHiNOCWiFPHEGUJDLA3jPoFE7uZ9lqph6chgt1SsrUXgWyaYVM-U5lhYIr5RSaGNDt6Keu6yKS~SuKbhRrGFMljgi0N44VGxElndusFLuLGLbxkNHMUNgbBhlPO5pWOflaiWB58xixaK8isg-b1JEMfa9CikD7y7pSvqiKjCU9TbwwCx9Ml0AUzB65Tkz3ohOTbirg8879UycKfLYy6VqiDs1z5TiWIII1dblYHWe083g8kUl5iGQZXFfzYQmO7M85kgaACGHJt6usOwp~vlOwQcmVSRLzp~LOSGhwFmjqB-Pu9KmZxWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201884&ar=1745776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01607.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic drawing that illustrates the evolution of Chr01/Chr15, Chr02/Chr14, and Chr03/Chr17. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87201885 id="87201885" class="section-title" data-legacy-id=s4e>4.5. Applications of the high-density linkage map in tetraploid cotton genomics, genetics, and breeding</h3>
<p>The present high-density genetic linkage map, with an average interval of 0.91 × 400 kb between genetic markers on the basis of a consensus estimate of genome size of ∼2,246Mb, provides a foundation to facilitate genome sequencing and sequence assembly.<sup><a reveal-id="DSW016C45" data-open="DSW016C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Besides, it well revealed the genome structure of tetraploid cotton and provided preliminary hypotheses of tetraploid cotton formation from two diploid cotton species. Until now, cotton breeders have been working to transfer excellent genes controlling fibre quality from <em>G. barbadense</em> to <em>G. hirsutum</em>. And, introgression lines have proved to be an optional way to solve high sterility and crazed segregation of interspecific hybrid progeny. Excellent introgression lines are usually constructed by marker-assisted selection, which is inevitably linked to a genetic linkage map. The present high-density linkage map could also provide markers that are tightly linked with target traits, which is undoubtedly very useful to molecular breeding and genetic improvement.</p>                            <h2 scrollto-destination=87201887 id="87201887" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was financially supported by the National Science Foundation of China (grant no. 31171593) and Genetically Modified Organisms Breeding Major Project of China (no. 2014ZX08009). Funding to pay the Open Access publication charges for this article was provided by the National Science Foundation of China (grant no. 31171593).</p>                            <h2 scrollto-destination=87201889 id="87201889" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW016C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">H.B.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Chee</div> <div class="given-names">P.W.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Recent advances in cotton genomics</div>, <div class="source">Int. J. Plant Genomics</div> , <div class="volume">2008</div>, <div class="fpage">742304</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20advances%20in%20cotton%20genomics&amp;author=H.B.%20Zhang&amp;author=Y.%20Li&amp;author=B.%20Wang&amp;author=P.W.%20Chee&amp;publication_year=2008&amp;journal=Int.%20J.%20Plant%20Genomics&amp;volume=2008&amp;pages=742304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1155/2008/742304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2008%2F742304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1155%2F2008%2F742304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18288253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Reinisch</div>
               <div class="given-names">A.J.</div>
            </div>, <div class="name string-name"><div class="surname">Dong</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Brubaker</div> <div class="given-names">C.L.</div></div>, <div class="name string-name"><div class="surname">Stelly</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div></div>
          <div class="year">1994</div>, <div class="article-title">A detailed RFLP map of cotton <em>Gossypium hirsutum</em>× <em>Gossypium barbadense</em>: chromosome organization and evolution in a disomic polyploid genome</div>, <div class="source">Genetics</div> , <div class="volume">138</div>, <div class="fpage">829</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20detailed%20RFLP%20map%20of%20cotton%20Gossypium%20hirsutum%C3%97%20Gossypium%20barbadense%3A%20chromosome%20organization%20and%20evolution%20in%20a%20disomic%20polyploid%20genome&amp;author=A.J.%20Reinisch&amp;author=J.M.%20Dong&amp;author=C.L.%20Brubaker&amp;author=D.M.%20Stelly&amp;author=J.F.%20Wendel&amp;author=A.H.%20Paterson&amp;publication_year=1994&amp;journal=Genetics&amp;volume=138&amp;pages=829-47" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7851778" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rong</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Abbey</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bowers</div> <div class="given-names">J.E.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (<em>Gossypium</em>)</div>, <div class="source">Genetics</div> , <div class="volume">166</div>, <div class="fpage">389</div>–<div class="lpage">417</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%203347-locus%20genetic%20recombination%20map%20of%20sequence-tagged%20sites%20reveals%20features%20of%20genome%20organization%2C%20transmission%20and%20evolution%20of%20cotton%20(Gossypium)&amp;author=J.%20Rong&amp;author=C.%20Abbey&amp;author=J.E.%20Bowers&amp;publication_year=2004&amp;journal=Genetics&amp;volume=166&amp;pages=389-417" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.166.1.389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.166.1.389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.166.1.389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15020432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guo</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Cai</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">T.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">A preliminary analysis of genome structure and composition in <em>Gossypium hirsutum</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">9</div>, <div class="fpage">314</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20preliminary%20analysis%20of%20genome%20structure%20and%20composition%20in%20Gossypium%20hirsutum&amp;author=W.%20Guo&amp;author=C.%20Cai&amp;author=C.%20Wang&amp;author=L.%20Zhao&amp;author=L.%20Wang&amp;author=T.%20Zhang&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-314" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-314" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-314"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18590573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC<sub>1</sub> population between <em>Gossypium hirsutum</em> and <em>G. barbadense</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20cotton%20revealed%20by%20a%20genome-wide%20SSR%20genetic%20map%20constructed%20from%20a%20BC1%20population%20between%20Gossypium%20hirsutum%20and%20G.%20barbadense&amp;author=Y.%20Yu&amp;author=D.%20Yuan&amp;author=S.%20Liang&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21214949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">J.Z.</div>
            </div>, <div class="name string-name"><div class="surname">Kohel</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">D.D.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">A high-density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome</div>, <div class="source">G3 (Bethesda)</div> , <div class="volume">2</div>, <div class="fpage">43</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20simple%20sequence%20repeat%20and%20single%20nucleotide%20polymorphism%20genetic%20map%20of%20the%20tetraploid%20cotton%20genome&amp;author=J.Z.%20Yu&amp;author=R.J.%20Kohel&amp;author=D.D.%20Fang&amp;publication_year=2012&amp;journal=G3%20(Bethesda)&amp;volume=2&amp;pages=43-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.111.001552" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.111.001552" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.111.001552"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Sequence-based ultra-dense genetic and physical maps reveal structural variations of allopolyploid cotton genomes</div>, <div class="source">Genome Biol.</div> , <div class="volume">16</div>, <div class="fpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-based%20ultra-dense%20genetic%20and%20physical%20maps%20reveal%20structural%20variations%20of%20allopolyploid%20cotton%20genomes&amp;author=S.%20Wang&amp;author=J.%20Chen&amp;author=W.%20Zhang&amp;publication_year=2015&amp;journal=Genome%20Biol.&amp;volume=16&amp;pages=108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0678-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0678-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0678-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26003111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">The draft genome of a diploid cotton <em>Gossypium raimondii</em></div>, <div class="source">Nat. Genet.</div> , <div class="volume">44</div>, <div class="fpage">1098</div>–<div class="lpage">103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20a%20diploid%20cotton%20Gossypium%20raimondii&amp;author=K.%20Wang&amp;author=Z.%20Wang&amp;author=F.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=1098-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22922876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome sequence of the cultivated cotton <em>Gossypium arboreum</em></div>, <div class="source">Nat. Genet.</div> , <div class="volume">46</div>, <div class="fpage">567</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20cultivated%20cotton%20Gossypium%20arboreum&amp;author=F.%20Li&amp;author=G.%20Fan&amp;author=K.%20Wang&amp;publication_year=2014&amp;journal=Nat.%20Genet.&amp;volume=46&amp;pages=567-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24836287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Genome sequence of cultivated Upland cotton (<em>Gossypium hirsutum</em> TM-1) provides insights into genome evolution</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">33</div>, <div class="fpage">524</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20cultivated%20Upland%20cotton%20(Gossypium%20hirsutum%20TM-1)%20provides%20insights%20into%20genome%20evolution&amp;author=F.%20Li&amp;author=G.%20Fan&amp;author=C.%20Lu&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol.&amp;volume=33&amp;pages=524-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3208" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3208" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3208"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25893780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">W.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Sequencing of allotetraploid cotton (<em>Gossypium hirsutum</em> L. acc. TM-1) provides a resource for fiber improvement</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">33</div>, <div class="fpage">531</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20of%20allotetraploid%20cotton%20(Gossypium%20hirsutum%20L.%20acc.%20TM-1)%20provides%20a%20resource%20for%20fiber%20improvement&amp;author=T.%20Zhang&amp;author=Y.%20Hu&amp;author=W.%20Jiang&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol.&amp;volume=33&amp;pages=531-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25893781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paterson</div>
               <div class="given-names">A.H.</div>
            </div>, <div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Gundlach</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres</div>, <div class="source">Nature</div> , <div class="volume">492</div>, <div class="fpage">423</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repeated%20polyploidization%20of%20Gossypium%20genomes%20and%20the%20evolution%20of%20spinnable%20cotton%20fibres&amp;author=A.H.%20Paterson&amp;author=J.F.%20Wendel&amp;author=H.%20Gundlach&amp;publication_year=2012&amp;journal=Nature&amp;volume=492&amp;pages=423-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11798" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11798"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23257886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">W.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">PIP: a database of potential intron polymorphism markers</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2174</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PIP%3A%20a%20database%20of%20potential%20intron%20polymorphism%20markers&amp;author=L.%20Yang&amp;author=G.%20Jin&amp;author=X.%20Zhao&amp;author=Y.%20Zheng&amp;author=Z.%20Xu&amp;author=W.%20Wu&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2174-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm296" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm296" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm296"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17545179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Van Deynze</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Sampling nucleotide diversity in cotton</div>, <div class="source">BMC Plant. Biol.</div> , <div class="volume">9</div>, <div class="fpage">125</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sampling%20nucleotide%20diversity%20in%20cotton&amp;author=A.%20Van%20Deynze&amp;author=K.%20Stoffel&amp;author=M.%20Lee&amp;publication_year=2009&amp;journal=BMC%20Plant.%20Biol.&amp;volume=9&amp;pages=125" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-125"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19840401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lin</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">Linkage map construction and mapping QTL for cotton fibre quality using SRAP, SSR and RAPD</div>, <div class="source">Plant. Breed</div> , <div class="volume">124</div>, <div class="fpage">180</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20map%20construction%20and%20mapping%20QTL%20for%20cotton%20fibre%20quality%20using%20SRAP%2C%20SSR%20and%20RAPD&amp;author=Z.%20Lin&amp;author=D.%20He&amp;author=X.%20Zhang&amp;publication_year=2005&amp;journal=Plant.%20Breed&amp;volume=124&amp;pages=180-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2005.124.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2005.124.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2005.124.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSW016C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Increasing cotton genome coverage with polymorphic SSRs as revealed by SSCP</div>, <div class="source">Genome</div> , <div class="volume">55</div>, <div class="fpage">459</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Increasing%20cotton%20genome%20coverage%20with%20polymorphic%20SSRs%20as%20revealed%20by%20SSCP&amp;author=X.%20Li&amp;author=D.%20Yuan&amp;author=H.%20Wang&amp;publication_year=2012&amp;journal=Genome&amp;volume=55&amp;pages=459-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g2012-032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg2012-032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg2012-032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22670804" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stam</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">1993</div>, <div class="article-title">Construction of integrated genetic linkage maps by means of a new computer package: Join Map</div>, <div class="source">Plant J.</div> , <div class="volume">3</div>, <div class="fpage">739</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20integrated%20genetic%20linkage%20maps%20by%20means%20of%20a%20new%20computer%20package%3A%20Join%20Map&amp;author=P.%20Stam&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=739-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.1993.00739.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.1993.00739.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.1993.00739.x"> </span></div> </div></div></div></div></div><div content-id="DSW016C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kosambi</div>
               <div class="given-names">D.D.</div>
            </div>
          </div>
          <div class="year">1944</div>, <div class="article-title">The estimation of map distance from recombination values</div>, <div class="source">Ann. Eugen.</div> , <div class="volume">12</div>, <div class="fpage">172</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distance%20from%20recombination%20values&amp;author=D.D.%20Kosambi&amp;publication_year=1944&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="DSW016C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paillard</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Schnurbusch</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Winzeler</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">An integrative genetic linkage map of winter wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">107</div>, <div class="fpage">1235</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrative%20genetic%20linkage%20map%20of%20winter%20wheat%20(Triticum%20aestivum%20L.)&amp;author=S.%20Paillard&amp;author=T.%20Schnurbusch&amp;author=M.%20Winzeler&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=1235-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1361-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1361-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1361-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12898031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Ren</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Tu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Development and evaluation of intron and insertion-deletion markers for <em>Gossypium barbadense</em></div>, <div class="source">Plant. Mol. Biol. Rep.</div> , <div class="volume">30</div>, <div class="fpage">605</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20evaluation%20of%20intron%20and%20insertion-deletion%20markers%20for%20Gossypium%20barbadense&amp;author=X.%20Wang&amp;author=G.%20Ren&amp;author=X.%20Li&amp;author=J.%20Tu&amp;author=Z.%20Lin&amp;author=X.%20Zhang&amp;publication_year=2012&amp;journal=Plant.%20Mol.%20Biol.%20Rep.&amp;volume=30&amp;pages=605-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-011-0369-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-011-0369-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-011-0369-3"> </span></div> </div></div></div></div></div><div content-id="DSW016C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Unbiased genomic distribution of genes related to cell morphogenesis in cotton by chromosome mapping</div>, <div class="source">Plant Cell Tissue Org. Cult.</div> , <div class="volume">108</div>, <div class="fpage">529</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unbiased%20genomic%20distribution%20of%20genes%20related%20to%20cell%20morphogenesis%20in%20cotton%20by%20chromosome%20mapping&amp;author=C.%20Liu&amp;author=Z.%20Lin&amp;author=X.%20Zhang&amp;publication_year=2012&amp;journal=Plant%20Cell%20Tissue%20Org.%20Cult.&amp;volume=108&amp;pages=529-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11240-011-0059-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11240-011-0059-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11240-011-0059-8"> </span></div> </div></div></div></div></div><div content-id="DSW016C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Genetic mapping and characteristics of genes specifically or preferentially expressed during fiber development in cotton</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e54444</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20and%20characteristics%20of%20genes%20specifically%20or%20preferentially%20expressed%20during%20fiber%20development%20in%20cotton&amp;author=X.%20Li&amp;author=D.%20Yuan&amp;author=J.%20Zhang&amp;author=Z.%20Lin&amp;author=X.%20Zhang&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e54444" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0054444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0054444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0054444"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23372723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Linkage mapping and expression analysis of miRNAs and their target genes during fiber development in cotton</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">706</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20mapping%20and%20expression%20analysis%20of%20miRNAs%20and%20their%20target%20genes%20during%20fiber%20development%20in%20cotton&amp;author=X.%20Chen&amp;author=W.%20Gao&amp;author=J.%20Zhang&amp;author=X.%20Zhang&amp;author=Z.%20Lin&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=706" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-706" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-706" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-706"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24131852" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ren</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Mining, genetic mapping and expression analysis of EST-derived resistance gene homologs (RGHs) in cotton</div>, <div class="source">BMC Plant. Biol.</div> , <div class="volume">14</div>, <div class="fpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%2C%20genetic%20mapping%20and%20expression%20analysis%20of%20EST-derived%20resistance%20gene%20homologs%20(RGHs)%20in%20cotton&amp;author=G.%20Ren&amp;author=X.%20Li&amp;author=Z.%20Lin&amp;publication_year=2014&amp;journal=BMC%20Plant.%20Biol.&amp;volume=14&amp;pages=203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-014-0203-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-014-0203-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-014-0203-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25064562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">D.D.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Development of EST-based SNP and InDel markers and their utilization in tetraploid cotton genetic mapping</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">1046</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20EST-based%20SNP%20and%20InDel%20markers%20and%20their%20utilization%20in%20tetraploid%20cotton%20genetic%20mapping&amp;author=X.%20Li&amp;author=W.%20Gao&amp;author=H.%20Guo&amp;author=X.%20Zhang&amp;author=D.D.%20Fang&amp;author=Z.%20Lin&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=1046" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-1046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-1046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-1046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25442170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Comparison and development of EST–SSRs from two 454 sequencing libraries of <em>Gossypium barbadense</em></div>, <div class="source">Euphytica</div> , <div class="volume">198</div>, <div class="fpage">277</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20and%20development%20of%20EST%E2%80%93SSRs%20from%20two%20454%20sequencing%20libraries%20of%20Gossypium%20barbadense&amp;author=H.%20Wang&amp;author=X.%20Li&amp;author=W.%20Gao&amp;author=X.%20Jin&amp;author=X.%20Zhang&amp;author=Z.%20Lin&amp;publication_year=2014&amp;journal=Euphytica&amp;volume=198&amp;pages=277-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-014-1104-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-014-1104-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-014-1104-6"> </span></div> </div></div></div></div></div><div content-id="DSW016C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Genetic mapping and comparative expression analysis of transcription factors in cotton</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0126150</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20and%20comparative%20expression%20analysis%20of%20transcription%20factors%20in%20cotton&amp;author=X.%20Chen&amp;author=X.%20Jin&amp;author=X.%20Li&amp;author=Z.%20Lin&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0126150" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0126150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0126150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0126150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25946129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guo</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Cai</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">A microsatellite-based, gene-rich linkage map reveals genome structure, function and evolution in <em>Gossypium</em></div>, <div class="source">Genetics</div> , <div class="volume">176</div>, <div class="fpage">527</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite-based%2C%20gene-rich%20linkage%20map%20reveals%20genome%20structure%2C%20function%20and%20evolution%20in%20Gossypium&amp;author=W.%20Guo&amp;author=C.%20Cai&amp;author=C.%20Wang&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=527-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.070375" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.070375" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.070375"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17409069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lin</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">X.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">A high-density integrative linkage map for <em>Gossypium hirsutum</em></div>, <div class="source">Euphytica</div> , <div class="volume">166</div>, <div class="fpage">35</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20integrative%20linkage%20map%20for%20Gossypium%20hirsutum&amp;author=Z.%20Lin&amp;author=Y.%20Zhang&amp;author=X.%20Zhang&amp;author=X.%20Guo&amp;publication_year=2009&amp;journal=Euphytica&amp;volume=166&amp;pages=35-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-008-9822-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-008-9822-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-008-9822-2"> </span></div> </div></div></div></div></div><div content-id="DSW016C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Yuanda</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Caiping</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Toward allotetraploid cotton genome assembly: integration of a high-density molecular genetic linkage map with DNA sequence information</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">539</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20allotetraploid%20cotton%20genome%20assembly%3A%20integration%20of%20a%20high-density%20molecular%20genetic%20linkage%20map%20with%20DNA%20sequence%20information&amp;author=L.%20Zhao&amp;author=L.%20Yuanda&amp;author=C.%20Caiping&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=539" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-539" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-539" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-539"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23046547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sourdille</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Tavaud</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Charmet</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bernard</div> <div class="given-names">M.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Transferability of wheat microsatellites to diploid Triticeae species carrying the A, B and D genomes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">103</div>, <div class="fpage">346</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transferability%20of%20wheat%20microsatellites%20to%20diploid%20Triticeae%20species%20carrying%20the%20A%2C%20B%20and%20D%20genomes&amp;author=P.%20Sourdille&amp;author=M.%20Tavaud&amp;author=G.%20Charmet&amp;author=M.%20Bernard&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=103&amp;pages=346-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-001-0542-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-001-0542-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-001-0542-4"> </span></div> </div></div></div></div></div><div content-id="DSW016C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xiao</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">D.D.</div></div>, <div class="name string-name"><div class="surname">Stelly</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Cantrell</div> <div class="given-names">R.G.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">New SSR markers for use in cotton (<em>Gossypium</em> spp.) improvement</div>, <div class="source">J. Cotton Sci.</div> , <div class="volume">13</div>, <div class="fpage">75</div>–<div class="lpage">157</div>.</div></div></div></div><div content-id="DSW016C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jena</div>
               <div class="given-names">S.N.</div>
            </div>, <div class="name string-name"><div class="surname">Srivastava</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Rai</div> <div class="given-names">K.M.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Development and characterization of genomic and expressed SSRs for levant cotton (<em>Gossypium herbaceum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">124</div>, <div class="fpage">565</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20genomic%20and%20expressed%20SSRs%20for%20levant%20cotton%20(Gossypium%20herbaceum%20L.)&amp;author=S.N.%20Jena&amp;author=A.%20Srivastava&amp;author=K.M.%20Rai&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=565-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1729-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1729-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1729-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22038488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eujayl</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Sledge</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Medicago truncatula EST-SSRs reveal cross-species genetic markers for <em>Medicago</em> spp</div>., <div class="source">Theor. Appl. Genet.</div> , <div class="volume">108</div>, <div class="fpage">414</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20EST-SSRs%20reveal%20cross-species%20genetic%20markers%20for%20Medicago%20spp&amp;author=I.%20Eujayl&amp;author=M.K.%20Sledge&amp;author=L.%20Wang&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=414-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1450-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1450-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1450-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13679975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Sledge</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Bouton</div> <div class="given-names">J.H.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Genome mapping of white clover (<em>Trifolium repens</em> L.) and comparative analysis within the <em>Trifolieae</em> using cross-species SSR markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">114</div>, <div class="fpage">1367</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20mapping%20of%20white%20clover%20(Trifolium%20repens%20L.)%20and%20comparative%20analysis%20within%20the%20Trifolieae%20using%20cross-species%20SSR%20markers&amp;author=Y.%20Zhang&amp;author=M.K.%20Sledge&amp;author=J.H.%20Bouton&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=1367-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0523-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0523-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0523-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17356868" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Taylor</div>
               <div class="given-names">D.R.</div>
            </div>, <div class="name string-name"><div class="surname">Ingvarsson</div> <div class="given-names">P.K.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Common features of segregation distortion in plants and animals</div>, <div class="source">Genetica</div> , <div class="volume">117</div>, <div class="fpage">27</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20features%20of%20segregation%20distortion%20in%20plants%20and%20animals&amp;author=D.R.%20Taylor&amp;author=P.K.%20Ingvarsson&amp;publication_year=2003&amp;journal=Genetica&amp;volume=117&amp;pages=27-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022308414864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022308414864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022308414864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12656570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wendel</div>
               <div class="given-names">J.F.</div>
            </div>
          </div>
          <div class="year">1989</div>, <div class="article-title">New World tetraploid cottons contain old world cytoplasm</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">86</div>, <div class="fpage">4132</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20World%20tetraploid%20cottons%20contain%20old%20world%20cytoplasm&amp;author=J.F.%20Wendel&amp;publication_year=1989&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=86&amp;pages=4132-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.86.11.4132" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.86.11.4132" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.86.11.4132"> </span></div> </div></div></div></div></div><div content-id="DSW016C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wendel</div>
               <div class="given-names">J.F.</div>
            </div>, <div class="name string-name"><div class="surname">Cronn</div> <div class="given-names">R.C.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Polyploidy and the evolutionary history of cotton</div>, <div class="source">Adv. Agron.</div> , <div class="volume">78</div>, <div class="fpage">139</div>–<div class="lpage">86</div>.</div></div></div></div><div content-id="DSW016C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Blenda</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">D.D.</div></div>, <div class="name string-name"><div class="surname">Rami</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Garsmeur</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Lacape</div> <div class="given-names">J.M.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e45739</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20density%20consensus%20genetic%20map%20of%20tetraploid%20cotton%20that%20integrates%20multiple%20component%20maps%20through%20molecular%20marker%20redundancy%20check&amp;author=A.%20Blenda&amp;author=D.D.%20Fang&amp;author=J.F.%20Rami&amp;author=O.%20Garsmeur&amp;author=F.%20Luo&amp;author=J.M.%20Lacape&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e45739" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0045739" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0045739" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0045739"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23029214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing</div>, <div class="source">DNA Res.</div> , <div class="volume">22</div>, <div class="fpage">147</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enrichment%20of%20an%20intraspecific%20genetic%20map%20of%20upland%20cotton%20by%20developing%20markers%20using%20parental%20RAD%20sequencing&amp;author=H.%20Wang&amp;author=X.%20Jin&amp;author=B.%20Zhang&amp;author=C.%20Shen&amp;author=Z.%20Lin&amp;publication_year=2015&amp;journal=DNA%20Res.&amp;volume=22&amp;pages=147-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu047" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu047" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu047"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25656006" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Applequist</div>
               <div class="given-names">W.L.</div>
            </div>, <div class="name string-name"><div class="surname">Cronn</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Comparative development of fiber in wild and cultivated cotton</div>, <div class="source">Evol. Dev.</div> , <div class="volume">3</div>, <div class="fpage">3</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20development%20of%20fiber%20in%20wild%20and%20cultivated%20cotton&amp;author=W.L.%20Applequist&amp;author=R.%20Cronn&amp;author=J.F.%20Wendel&amp;publication_year=2001&amp;journal=Evol.%20Dev.&amp;volume=3&amp;pages=3-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1525-142x.2001.00079.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1525-142x.2001.00079.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1525-142x.2001.00079.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11256432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jiang</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">El-Zik</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">Polyploid formation created unique avenues for response to selection in <em>Gossypium</em> (cotton)</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">95</div>, <div class="fpage">4419</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polyploid%20formation%20created%20unique%20avenues%20for%20response%20to%20selection%20in%20Gossypium%20(cotton)&amp;author=C.%20Jiang&amp;author=R.J.%20Wright&amp;author=K.M.%20El-Zik&amp;author=A.H.%20Paterson&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=4419-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.8.4419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.8.4419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.8.4419"> </span></div> </div></div></div></div></div><div content-id="DSW016C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Park</div>
               <div class="given-names">Y.H.</div>
            </div>, <div class="name string-name"><div class="surname">Alabady</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Ulloa</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">Genetic mapping of new cotton fiber loci using EST-derived microsatellites in an interspecific recombinant inbred line (RIL) cotton population</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">274</div>, <div class="fpage">428</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20new%20cotton%20fiber%20loci%20using%20EST-derived%20microsatellites%20in%20an%20interspecific%20recombinant%20inbred%20line%20(RIL)%20cotton%20population&amp;author=Y.H.%20Park&amp;author=M.S.%20Alabady&amp;author=M.%20Ulloa&amp;publication_year=2005&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=274&amp;pages=428-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-005-0037-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-005-0037-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-005-0037-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16187061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paterson</div>
               <div class="given-names">A.H.</div>
            </div>, <div class="name string-name"><div class="surname">Saranga</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Menz</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">C.X.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">R.J.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">QTL analysis of genotype × environmental interactions affecting cotton fiber quality</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">106</div>, <div class="fpage">384</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20genotype%20%C3%97%20environmental%20interactions%20affecting%20cotton%20fiber%20quality&amp;author=A.H.%20Paterson&amp;author=Y.%20Saranga&amp;author=M.%20Menz&amp;author=C.X.%20Jiang&amp;author=R.J.%20Wright&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=384-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-1025-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-1025-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-1025-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12589538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW016C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW016C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yuan</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">The genome sequence of Sea-Island cotton (<em>Gossypium barbadense</em>) provides insights into the allopolyploidization and development of superior spinnable fibres</div>, <div class="source">Sci. Rep.</div> , <div class="volume">5</div>, <div class="fpage">17662</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Sea-Island%20cotton%20(Gossypium%20barbadense)%20provides%20insights%20into%20the%20allopolyploidization%20and%20development%20of%20superior%20spinnable%20fibres&amp;author=D.%20Yuan&amp;author=Z.%20Tang&amp;author=M.%20Wang&amp;publication_year=2015&amp;journal=Sci.%20Rep.&amp;volume=5&amp;pages=17662" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep17662" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep17662" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep17662"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26634818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745776&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw016/1/dsw016_Supplementary_Data.zip?Expires=2147483647&amp;Signature=w84~QHHFCiCMqafFb4YiKhi-Dv8JgDxcFwjxbbXoh3jVmKMUok9wnM2ewQsAYgohx~lZUcybFFrCHgsRw9jPC3AlVfWuxP-GsGu0Z7q-QTQHWYYkLYCVVptAhfWzyriDTdTEJxtiAKsEfClb~0HcAjAjszzDv4edYGe2dy2nZOV1UN9QzOy9vVl9Czs~hPPmg48EWlWDKHMeDiwD2fNIy~Vg3hDngkSjFcn~mxP8QcnukhqzPREM7DW2K0r7y93ur7EQwLZy6ie8t0UM-KGjIa3cQxt28nXM2OPRhGT7r1v5N2OSv9rAvESbpBDDByj4GStFWLuU-pBNB8XUCM1YKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">464</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500010&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw016"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745776"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500010&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f3%2f283%2f1745776">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000379741500010&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F3%2F283%2F1745776" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745776'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw016");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f283%2f1745776">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

