

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/4/369/445591","@type":"ScholarlyArticle","name":"Complete Genome Sequence of  Enterococcus mundtii  QU 25, an Efficient  l -(+)-Lactic Acid-Producing Bacterium","datePublished":"2014-02-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2014-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu003","keywords":["<italic>Enterococcus mundtii</italic>","xylose","<sc>l</sc>-lactic acid","mundticin"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/4/369/445591","author":[{"name":"Shiwa, Yuh","affiliation":"1  Genome Research Center ,  NODAI Research Institute, Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156–8502 ,  Japan","@type":"Person"},{"name":"Yanase, Hiroaki","affiliation":"2  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Hirose, Yuu","affiliation":"3  Electronics-Inspired Interdisciplinary Research Institute (EIIRIS) ,  Toyohashi University of Technology ,  Toyohashi 441-8580 ,  Japan","@type":"Person"},{"name":"Satomi, Shohei","affiliation":"4  Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture ,  Graduate School, Kyushu University ,  6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Araya-Kojima, Tomoko","affiliation":"2  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Watanabe, Satoru","affiliation":"2  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Zendo, Takeshi","affiliation":"4  Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture ,  Graduate School, Kyushu University ,  6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Chibazakura, Taku","affiliation":"2  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Shimizu-Kadota, Mariko","affiliation":"2  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan    5  Department of Environmental Science ,  Musashino University ,  3-3-3 Ariake, Koto-ku, Tokyo 135-0063 ,  Japan","@type":"Person"},{"name":"Yoshikawa, Hirofumi","affiliation":"1  Genome Research Center ,  NODAI Research Institute, Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156–8502 ,  Japan    2  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Sonomoto, Kenji","affiliation":"4  Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture ,  Graduate School, Kyushu University ,  6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"}],"description":"Abstract.  Enterococcus mundtii QU 25, a non-dairy bacterial strain of ovine faecal origin, can ferment both cellobiose and xylose to produce l-lactic acid. The","pageStart":"369","pageEnd":"377","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Complete Genome Sequence of  Enterococcus mundtii  QU 25, an Efficient  l -(+)-Lactic Acid-Producing Bacterium","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Complete Genome Sequence of  Enterococcus mundtii  QU 25, an Efficient  l -(+)-Lactic Acid-Producing Bacterium" />
    <meta property="og:description" content="Abstract.  Enterococcus mundtii QU 25, a non-dairy bacterial strain of ovine faecal origin, can ferment both cellobiose and xylose to produce l-lactic acid. The" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/4/369/445591" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-02-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shiwa, Yuh" /><meta name="citation_author_institution" content="Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156–8502, Japan" /><meta name="citation_author" content="Yanase, Hiroaki" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Hirose, Yuu" /><meta name="citation_author_institution" content="Electronics-Inspired Interdisciplinary Research Institute (EIIRIS), Toyohashi University of Technology, Toyohashi 441-8580, Japan" /><meta name="citation_author" content="Satomi, Shohei" /><meta name="citation_author_institution" content="Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture, Graduate School, Kyushu University, 6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Araya-Kojima, Tomoko" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Watanabe, Satoru" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Zendo, Takeshi" /><meta name="citation_author_institution" content="Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture, Graduate School, Kyushu University, 6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Chibazakura, Taku" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Shimizu-Kadota, Mariko" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author_institution" content="Department of Environmental Science, Musashino University, 3-3-3 Ariake, Koto-ku, Tokyo 135-0063, Japan" /><meta name="citation_author" content="Yoshikawa, Hirofumi" /><meta name="citation_author_institution" content="Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156–8502, Japan" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Sonomoto, Kenji" /><meta name="citation_author_institution" content="Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture, Graduate School, Kyushu University, 6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_title" content="Complete Genome Sequence of Enterococcus mundtii QU 25, an Efficient l-(+)-Lactic Acid-Producing Bacterium" /><meta name="citation_firstpage" content="369" /><meta name="citation_lastpage" content="377" /><meta name="citation_doi" content="10.1093/dnares/dsu003" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2014/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Isolation and characterisation of lactic acid bacterium for effective fermentation of cellobiose into optically pure homo l-(+)-lactic acid; Appl. Microbiol. Biotechnol.;  citation_year=2011;  citation_volume=89;  citation_pages=1039-49; " /><meta name="citation_reference" content="citation_title=Efficient homofermentative l-(+)-lactic acid production from xylose by a novel lactic acid bacterium, Enterococcus mundtii QU 25; Appl. Environ. Microbiol.;  citation_year=2011;  citation_volume=77;  citation_pages=1892-5; " /><meta name="citation_reference" content="citation_title=Improved lactic acid productivity by an open repeated batch fermentation system using Enterococcus mundtii QU 25; RSC Adv.;  citation_year=2013;  citation_volume=3;  citation_pages=8437-45; " /><meta name="citation_reference" content="citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data; Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=563-9; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Enterococcus mundtii CRL1656; J. Bacteriol.;  citation_year=2012;  citation_volume=194;  citation_pages=550" /><meta name="citation_reference" content="citation_title=Chemically defined media and auxotrophy of the prolific l-lactic acid producer Lactococcus lactis IO-1; J. Biosci. Bioeng.;  citation_year=2013;  citation_volume=115;  citation_pages=481-4; " /><meta name="citation_reference" content="citation_title=Hybrid error correction and de novo assembly of single-molecule sequencing reads; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=693-700; " /><meta name="citation_reference" content="citation_title=Aggressive assembly of pyrosequencing reads with mates; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=2818-24; " /><meta name="citation_reference" content="citation_title=Scaffolding and validation of bacterial genome assemblies using optical restriction maps; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=1229-35; " /><meta name="citation_reference" content="citation_title=MetaGeneAnnotator: detecting species-specific patterns of ribosomal binding site for precise gene prediction in anonymous prokaryotic and phage genomes; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=387-96; " /><meta name="citation_reference" content="citation_title=Microbial genome annotation pipeline (MiGAP) for diverse users; citation_text=S001-1-2;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D5-D12; " /><meta name="citation_reference" content="citation_title=InterProScan—an integration platform for the signature-recognition methods in InterPro; Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=847-8; " /><meta name="citation_reference" content="citation_title=KAAS: an automatic genome annotation and pathway reconstruction server; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W182-5; " /><meta name="citation_reference" content="citation_title=PHAST: a fast phage search tool; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=W347-52; " /><meta name="citation_reference" content="citation_title=ISsaga is an ensemble of web-based methods for high throughput identification and semi-automatic annotation of insertion sequences in prokaryotic genomes; Genome Biol.;  citation_year=2011;  citation_volume=12;  citation_pages=R30" /><meta name="citation_reference" content="citation_title=CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W52-7; " /><meta name="citation_reference" content="citation_title=High-quality draft genome sequences of 28 Enterococcus sp. isolates; J. Bacteriol.;  citation_year=2010;  citation_volume=192;  citation_pages=2469-70; " /><meta name="citation_reference" content="citation_title=Role of mobile DNA in the evolution of vancomycin-resistant Enterococcus faecalis; Science;  citation_year=2003;  citation_volume=299;  citation_pages=2071-4; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the first complete Enterococcus faecium genome; J. Bacteriol.;  citation_year=2012;  citation_volume=194;  citation_pages=2334-41; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Enterococcus faecium strain TX16 and comparative genomic analysis of Enterococcus faecium genomes; BMC Microbiol.;  citation_year=2012;  citation_volume=12;  citation_pages=135" /><meta name="citation_reference" content="citation_title=Genome sequence of Enterococcus hirae (Streptococcus faecalis) ATCC 9790, a model organism for the study of ion transport, bioenergetics, and copper homeostasis; J. Bacteriol.;  citation_year=2012;  citation_volume=194;  citation_pages=5126-7; " /><meta name="citation_reference" content="citation_title=GenomeMatcher: a graphical user interface for DNA sequence comparison; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=376" /><meta name="citation_reference" content="citation_title=Identification and production of a bacteriocin from Enterococcus mundtii QU 2 isolated from soybean; J. Appl. Microbiol.;  citation_year=2005;  citation_volume=99;  citation_pages=1181-90; " /><meta name="citation_reference" content="citation_title=Biochemical and genetic evidence for production of enterocins A and B by Enterococcus faecium WHE 81; Int. J. Food Microbiol.;  citation_year=2001;  citation_volume=70;  citation_pages=291-301; " /><meta name="citation_reference" content="citation_title=Advantages of single-molecule real-time sequencing in high-GC content genomes; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e68824" /><meta name="citation_reference" content="citation_title=A flexible and efficient template format for circular consensus sequencing and SNP detection; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=e159" /><meta name="citation_reference" content=" citation_edition=2 edition; citation_publisher=Springer Publishing Company, New York; Bergey's Manual of Systematic Bacteriology;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=Mutational bias suggests that replication termination occurs near the dif site, not at Ter Sites; Mol. Microbiol.;  citation_year=2007;  citation_volume=64;  citation_pages=42-56; " /><meta name="citation_reference" content="citation_title=CRISPR interference: RNA-directed adaptive immunity in bacteria and archaea; Nat. Rev. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=181-90; " /><meta name="citation_reference" content="citation_title=Cotransfer of antibiotic resistance genes and a hylEfm-containing virulence plasmid in Enterococcus faecium; Antimicrob. Agents Chemother.;  citation_year=2009;  citation_volume=53;  citation_pages=4240-6; " /><meta name="citation_reference" content="citation_title=Presence of pRI1: a small cryptic mobilizable plasmid isolated from Enterococcus faecium of human and animal origin; Curr. Microbiol.;  citation_year=2009;  citation_volume=58;  citation_pages=95-100; " /><meta name="citation_reference" content="citation_title=Transposon characterization of vancomycin-resistant Enterococcus faecium (VREF) and dissemination of resistance associated with transferable plasmids; J. Antimicrob. Chemother.;  citation_year=2007;  citation_volume=60;  citation_pages=263-8; " /><meta name="citation_reference" content="citation_title=Copper resistance in Enterococcus faecium, mediated by the tcrB gene, is selected by supplementation of pig feed with copper sulfate; Appl. Environ. Microbiol.;  citation_year=2006;  citation_volume=72;  citation_pages=5784-9; " /><meta name="citation_reference" content="citation_title=Toxin-antitoxin systems are ubiquitous and plasmid-encoded in vancomycin-resistant enterococci; Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=311-6; " /><meta name="citation_reference" content="citation_title=Glycopeptide antibiotic resistance; Annu. Rev. Pharmacol. Toxicol.;  citation_year=2002;  citation_volume=42;  citation_pages=381-408; " /><meta name="citation_reference" content="citation_title=Biochemical and genetic characterization of mundticin KS, an antilisterial peptide produced by Enterococcus mundtii NFRI 7393; Appl. Environ. Microbiol.;  citation_year=2002;  citation_volume=68;  citation_pages=3830-40; " /><meta name="citation_reference" content="citation_title=Recent advances in lactic acid production by microbial fermentation processes; Biotechnol. Adv.;  citation_year=2013;  citation_volume=31;  citation_pages=877-902; " /><meta name="citation_reference" content="citation_title=Utilization of d-ribose through d-xylose transporter; FEMS Microbiol. Lett.;  citation_year=1998;  citation_volume=163;  citation_pages=255-61; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Lactococcus lactis IO-1, a lactic acid bacterium that utilizes xylose and produces high levels of l-lactic acid; J. Bacteriol.;  citation_year=2012;  citation_volume=194;  citation_pages=2102-3; " /><meta name="citation_reference" content="citation_title=Carbon catabolite control of the metabolic network in Bacillus subtilis; Biosci. Biotechnol. Biochem.;  citation_year=2009;  citation_volume=73;  citation_pages=245-59; " /><meta name="citation_reference" content="citation_title=Lactate racemization as a rescue pathway for supplying d-lactate to the cell wall biosynthesis machinery in Lactobacillus plantarum; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=6750-61; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/4/369/1218868/dsu003.pdf" /><meta name="description" content="Abstract.  Enterococcus mundtii QU 25, a non-dairy bacterial strain of ovine faecal origin, can ferment both cellobiose and xylose to produce l-lactic acid. The" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/4/369/445591" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/4/369/445591" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/4/369/445591");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu003';
    </script>
<title>
	Complete Genome Sequence of Enterococcus mundtii QU 25, an Efficient l-(+)-Lactic Acid-Producing Bacterium | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./445591" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeZwPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGl3YSwgWXVoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR2Vub21lIFJlc2VhcmNoIENlbnRlciwgTk9EQUkgUmVzZWFyY2ggSW5zdGl0dXRlLCBUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlLCAxLTEtMSBTYWt1cmFnYW9rYSwgU2V0YWdheWEta3UsIFRva3lvIDE1NuKAkzg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuYXNlLCBIaXJvYWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlLCBUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlLCAxLTEtMSBTYWt1cmFnYW9rYSwgU2V0YWdheWEta3UsIFRva3lvIDE1Ni04NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhpcm9zZSwgWXV1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRWxlY3Ryb25pY3MtSW5zcGlyZWQgSW50ZXJkaXNjaXBsaW5hcnkgUmVzZWFyY2ggSW5zdGl0dXRlIChFSUlSSVMpLCBUb3lvaGFzaGkgVW5pdmVyc2l0eSBvZiBUZWNobm9sb2d5LCBUb3lvaGFzaGkgNDQxLTg1ODAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0b21pLCBTaG9oZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIE1pY3JvYmlhbCBUZWNobm9sb2d5LCBEaXZpc2lvbiBvZiBBcHBsaWVkIE1vbGVjdWxhciBNaWNyb2Jpb2xvZ3kgYW5kIEJpb21hc3MgQ2hlbWlzdHJ5LCBEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIEZhY3VsdHkgb2YgQWdyaWN1bHR1cmUsIEdyYWR1YXRlIFNjaG9vbCwgS3l1c2h1IFVuaXZlcnNpdHksIDYtMTAtMSBIYWtvemFraSwgSGlnYXNoaS1rdSwgRnVrdW9rYSA4MTItODU4MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcmF5YS1Lb2ppbWEsIFRvbW9rbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSwgMS0xLTEgU2FrdXJhZ2Fva2EsIFNldGFnYXlhLWt1LCBUb2t5byAxNTYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYXRhbmFiZSwgU2F0b3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlLCBUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlLCAxLTEtMSBTYWt1cmFnYW9rYSwgU2V0YWdheWEta3UsIFRva3lvIDE1Ni04NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlplbmRvLCBUYWtlc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBNaWNyb2JpYWwgVGVjaG5vbG9neSwgRGl2aXNpb24gb2YgQXBwbGllZCBNb2xlY3VsYXIgTWljcm9iaW9sb2d5IGFuZCBCaW9tYXNzIENoZW1pc3RyeSwgRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBHcmFkdWF0ZSBTY2hvb2wsIEt5dXNodSBVbml2ZXJzaXR5LCA2LTEwLTEgSGFrb3pha2ksIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hpYmF6YWt1cmEsIFRha3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2UsIFRva3lvIFVuaXZlcnNpdHkgb2YgQWdyaWN1bHR1cmUsIDEtMS0xIFNha3VyYWdhb2thLCBTZXRhZ2F5YS1rdSwgVG9reW8gMTU2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbWl6dS1LYWRvdGEsIE1hcmlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSwgMS0xLTEgU2FrdXJhZ2Fva2EsIFNldGFnYXlhLWt1LCBUb2t5byAxNTYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEVudmlyb25tZW50YWwgU2NpZW5jZSwgTXVzYXNoaW5vIFVuaXZlcnNpdHksIDMtMy0zIEFyaWFrZSwgS290by1rdSwgVG9reW8gMTM1LTAwNjMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWW9zaGlrYXdhLCBIaXJvZnVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlbm9tZSBSZXNlYXJjaCBDZW50ZXIsIE5PREFJIFJlc2VhcmNoIEluc3RpdHV0ZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSwgMS0xLTEgU2FrdXJhZ2Fva2EsIFNldGFnYXlhLWt1LCBUb2t5byAxNTbigJM4NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSwgMS0xLTEgU2FrdXJhZ2Fva2EsIFNldGFnYXlhLWt1LCBUb2t5byAxNTYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTb25vbW90bywgS2VuamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIE1pY3JvYmlhbCBUZWNobm9sb2d5LCBEaXZpc2lvbiBvZiBBcHBsaWVkIE1vbGVjdWxhciBNaWNyb2Jpb2xvZ3kgYW5kIEJpb21hc3MgQ2hlbWlzdHJ5LCBEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIEZhY3VsdHkgb2YgQWdyaWN1bHR1cmUsIEdyYWR1YXRlIFNjaG9vbCwgS3l1c2h1IFVuaXZlcnNpdHksIDYtMTAtMSBIYWtvemFraSwgSGlnYXNoaS1rdSwgRnVrdW9rYSA4MTItODU4MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNvbXBsZXRlIEdlbm9tZSBTZXF1ZW5jZSBvZiBFbnRlcm9jb2NjdXMgbXVuZHRpaSBRVSAyNSwgYW4gRWZmaWNpZW50IGwtKCspLUxhY3RpYyBBY2lkLVByb2R1Y2luZyBCYWN0ZXJpdW0iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdTAwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpc2F0aW9uIG9mIGxhY3RpYyBhY2lkIGJhY3Rlcml1bSBmb3IgZWZmZWN0aXZlIGZlcm1lbnRhdGlvbiBvZiBjZWxsb2Jpb3NlIGludG8gb3B0aWNhbGx5IHB1cmUgaG9tbyBsLSgrKS1sYWN0aWMgYWNpZDsgQXBwbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTEwMzktNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBob21vZmVybWVudGF0aXZlIGwtKCspLWxhY3RpYyBhY2lkIHByb2R1Y3Rpb24gZnJvbSB4eWxvc2UgYnkgYSBub3ZlbCBsYWN0aWMgYWNpZCBiYWN0ZXJpdW0sIEVudGVyb2NvY2N1cyBtdW5kdGlpIFFVIDI1OyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc3OyAgY2l0YXRpb25fcGFnZXM9MTg5Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlZCBsYWN0aWMgYWNpZCBwcm9kdWN0aXZpdHkgYnkgYW4gb3BlbiByZXBlYXRlZCBiYXRjaCBmZXJtZW50YXRpb24gc3lzdGVtIHVzaW5nIEVudGVyb2NvY2N1cyBtdW5kdGlpIFFVIDI1OyBSU0MgQWR2LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ODQzNy00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9uaHlicmlkLCBmaW5pc2hlZCBtaWNyb2JpYWwgZ2Vub21lIGFzc2VtYmxpZXMgZnJvbSBsb25nLXJlYWQgU01SVCBzZXF1ZW5jaW5nIGRhdGE7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU2My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgRW50ZXJvY29jY3VzIG11bmR0aWkgQ1JMMTY1NjsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTQ7ICBjaXRhdGlvbl9wYWdlcz01NTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGVtaWNhbGx5IGRlZmluZWQgbWVkaWEgYW5kIGF1eG90cm9waHkgb2YgdGhlIHByb2xpZmljIGwtbGFjdGljIGFjaWQgcHJvZHVjZXIgTGFjdG9jb2NjdXMgbGFjdGlzIElPLTE7IEouIEJpb3NjaS4gQmlvZW5nLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz00ODEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHlicmlkIGVycm9yIGNvcnJlY3Rpb24gYW5kIGRlIG5vdm8gYXNzZW1ibHkgb2Ygc2luZ2xlLW1vbGVjdWxlIHNlcXVlbmNpbmcgcmVhZHM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz02OTMtNzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ2dyZXNzaXZlIGFzc2VtYmx5IG9mIHB5cm9zZXF1ZW5jaW5nIHJlYWRzIHdpdGggbWF0ZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjgxOC0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NhZmZvbGRpbmcgYW5kIHZhbGlkYXRpb24gb2YgYmFjdGVyaWFsIGdlbm9tZSBhc3NlbWJsaWVzIHVzaW5nIG9wdGljYWwgcmVzdHJpY3Rpb24gbWFwczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMjI5LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhR2VuZUFubm90YXRvcjogZGV0ZWN0aW5nIHNwZWNpZXMtc3BlY2lmaWMgcGF0dGVybnMgb2Ygcmlib3NvbWFsIGJpbmRpbmcgc2l0ZSBmb3IgcHJlY2lzZSBnZW5lIHByZWRpY3Rpb24gaW4gYW5vbnltb3VzIHByb2thcnlvdGljIGFuZCBwaGFnZSBnZW5vbWVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTM4Ny05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iaWFsIGdlbm9tZSBhbm5vdGF0aW9uIHBpcGVsaW5lIChNaUdBUCkgZm9yIGRpdmVyc2UgdXNlcnM7IGNpdGF0aW9uX3RleHQ9UzAwMS0xLTI7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhdGFiYXNlIHJlc291cmNlcyBvZiB0aGUgTmF0aW9uYWwgQ2VudGVyIGZvciBCaW90ZWNobm9sb2d5IEluZm9ybWF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1ENS1EMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbuKAlGFuIGludGVncmF0aW9uIHBsYXRmb3JtIGZvciB0aGUgc2lnbmF0dXJlLXJlY29nbml0aW9uIG1ldGhvZHMgaW4gSW50ZXJQcm87IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9ODQ3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtBQVM6IGFuIGF1dG9tYXRpYyBnZW5vbWUgYW5ub3RhdGlvbiBhbmQgcGF0aHdheSByZWNvbnN0cnVjdGlvbiBzZXJ2ZXI7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVcxODItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEhBU1Q6IGEgZmFzdCBwaGFnZSBzZWFyY2ggdG9vbDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9VzM0Ny01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SVNzYWdhIGlzIGFuIGVuc2VtYmxlIG9mIHdlYi1iYXNlZCBtZXRob2RzIGZvciBoaWdoIHRocm91Z2hwdXQgaWRlbnRpZmljYXRpb24gYW5kIHNlbWktYXV0b21hdGljIGFubm90YXRpb24gb2YgaW5zZXJ0aW9uIHNlcXVlbmNlcyBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz1SMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUklTUFJGaW5kZXI6IGEgd2ViIHRvb2wgdG8gaWRlbnRpZnkgY2x1c3RlcmVkIHJlZ3VsYXJseSBpbnRlcnNwYWNlZCBzaG9ydCBwYWxpbmRyb21pYyByZXBlYXRzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1XNTItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1xdWFsaXR5IGRyYWZ0IGdlbm9tZSBzZXF1ZW5jZXMgb2YgMjggRW50ZXJvY29jY3VzIHNwLiBpc29sYXRlczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTI7ICBjaXRhdGlvbl9wYWdlcz0yNDY5LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIG1vYmlsZSBETkEgaW4gdGhlIGV2b2x1dGlvbiBvZiB2YW5jb215Y2luLXJlc2lzdGFudCBFbnRlcm9jb2NjdXMgZmFlY2FsaXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9Mjk5OyAgY2l0YXRpb25fcGFnZXM9MjA3MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgZmlyc3QgY29tcGxldGUgRW50ZXJvY29jY3VzIGZhZWNpdW0gZ2Vub21lOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5NDsgIGNpdGF0aW9uX3BhZ2VzPTIzMzQtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBFbnRlcm9jb2NjdXMgZmFlY2l1bSBzdHJhaW4gVFgxNiBhbmQgY29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpcyBvZiBFbnRlcm9jb2NjdXMgZmFlY2l1bSBnZW5vbWVzOyBCTUMgTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiBFbnRlcm9jb2NjdXMgaGlyYWUgKFN0cmVwdG9jb2NjdXMgZmFlY2FsaXMpIEFUQ0MgOTc5MCwgYSBtb2RlbCBvcmdhbmlzbSBmb3IgdGhlIHN0dWR5IG9mIGlvbiB0cmFuc3BvcnQsIGJpb2VuZXJnZXRpY3MsIGFuZCBjb3BwZXIgaG9tZW9zdGFzaXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk0OyAgY2l0YXRpb25fcGFnZXM9NTEyNi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWVNYXRjaGVyOiBhIGdyYXBoaWNhbCB1c2VyIGludGVyZmFjZSBmb3IgRE5BIHNlcXVlbmNlIGNvbXBhcmlzb247IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9Mzc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIHByb2R1Y3Rpb24gb2YgYSBiYWN0ZXJpb2NpbiBmcm9tIEVudGVyb2NvY2N1cyBtdW5kdGlpIFFVIDIgaXNvbGF0ZWQgZnJvbSBzb3liZWFuOyBKLiBBcHBsLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MTE4MS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvY2hlbWljYWwgYW5kIGdlbmV0aWMgZXZpZGVuY2UgZm9yIHByb2R1Y3Rpb24gb2YgZW50ZXJvY2lucyBBIGFuZCBCIGJ5IEVudGVyb2NvY2N1cyBmYWVjaXVtIFdIRSA4MTsgSW50LiBKLiBGb29kIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0yOTEtMzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbnRhZ2VzIG9mIHNpbmdsZS1tb2xlY3VsZSByZWFsLXRpbWUgc2VxdWVuY2luZyBpbiBoaWdoLUdDIGNvbnRlbnQgZ2Vub21lczsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2ODgyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmxleGlibGUgYW5kIGVmZmljaWVudCB0ZW1wbGF0ZSBmb3JtYXQgZm9yIGNpcmN1bGFyIGNvbnNlbnN1cyBzZXF1ZW5jaW5nIGFuZCBTTlAgZGV0ZWN0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1lMTU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX2VkaXRpb249MiBlZGl0aW9uOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIgUHVibGlzaGluZyBDb21wYW55LCBOZXcgWW9yazsgQmVyZ2V5J3MgTWFudWFsIG9mIFN5c3RlbWF0aWMgQmFjdGVyaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbmFsIGJpYXMgc3VnZ2VzdHMgdGhhdCByZXBsaWNhdGlvbiB0ZXJtaW5hdGlvbiBvY2N1cnMgbmVhciB0aGUgZGlmIHNpdGUsIG5vdCBhdCBUZXIgU2l0ZXM7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTQyLTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUklTUFIgaW50ZXJmZXJlbmNlOiBSTkEtZGlyZWN0ZWQgYWRhcHRpdmUgaW1tdW5pdHkgaW4gYmFjdGVyaWEgYW5kIGFyY2hhZWE7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvdHJhbnNmZXIgb2YgYW50aWJpb3RpYyByZXNpc3RhbmNlIGdlbmVzIGFuZCBhIGh5bEVmbS1jb250YWluaW5nIHZpcnVsZW5jZSBwbGFzbWlkIGluIEVudGVyb2NvY2N1cyBmYWVjaXVtOyBBbnRpbWljcm9iLiBBZ2VudHMgQ2hlbW90aGVyLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTQyNDAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlc2VuY2Ugb2YgcFJJMTogYSBzbWFsbCBjcnlwdGljIG1vYmlsaXphYmxlIHBsYXNtaWQgaXNvbGF0ZWQgZnJvbSBFbnRlcm9jb2NjdXMgZmFlY2l1bSBvZiBodW1hbiBhbmQgYW5pbWFsIG9yaWdpbjsgQ3Vyci4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTk1LTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NvbiBjaGFyYWN0ZXJpemF0aW9uIG9mIHZhbmNvbXljaW4tcmVzaXN0YW50IEVudGVyb2NvY2N1cyBmYWVjaXVtIChWUkVGKSBhbmQgZGlzc2VtaW5hdGlvbiBvZiByZXNpc3RhbmNlIGFzc29jaWF0ZWQgd2l0aCB0cmFuc2ZlcmFibGUgcGxhc21pZHM7IEouIEFudGltaWNyb2IuIENoZW1vdGhlci47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0yNjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29wcGVyIHJlc2lzdGFuY2UgaW4gRW50ZXJvY29jY3VzIGZhZWNpdW0sIG1lZGlhdGVkIGJ5IHRoZSB0Y3JCIGdlbmUsIGlzIHNlbGVjdGVkIGJ5IHN1cHBsZW1lbnRhdGlvbiBvZiBwaWcgZmVlZCB3aXRoIGNvcHBlciBzdWxmYXRlOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9NTc4NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3hpbi1hbnRpdG94aW4gc3lzdGVtcyBhcmUgdWJpcXVpdG91cyBhbmQgcGxhc21pZC1lbmNvZGVkIGluIHZhbmNvbXljaW4tcmVzaXN0YW50IGVudGVyb2NvY2NpOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz0zMTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2x5Y29wZXB0aWRlIGFudGliaW90aWMgcmVzaXN0YW5jZTsgQW5udS4gUmV2LiBQaGFybWFjb2wuIFRveGljb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MzgxLTQwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvY2hlbWljYWwgYW5kIGdlbmV0aWMgY2hhcmFjdGVyaXphdGlvbiBvZiBtdW5kdGljaW4gS1MsIGFuIGFudGlsaXN0ZXJpYWwgcGVwdGlkZSBwcm9kdWNlZCBieSBFbnRlcm9jb2NjdXMgbXVuZHRpaSBORlJJIDczOTM7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0zODMwLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnQgYWR2YW5jZXMgaW4gbGFjdGljIGFjaWQgcHJvZHVjdGlvbiBieSBtaWNyb2JpYWwgZmVybWVudGF0aW9uIHByb2Nlc3NlczsgQmlvdGVjaG5vbC4gQWR2LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTg3Ny05MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVV0aWxpemF0aW9uIG9mIGQtcmlib3NlIHRocm91Z2ggZC14eWxvc2UgdHJhbnNwb3J0ZXI7IEZFTVMgTWljcm9iaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjM7ICBjaXRhdGlvbl9wYWdlcz0yNTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBMYWN0b2NvY2N1cyBsYWN0aXMgSU8tMSwgYSBsYWN0aWMgYWNpZCBiYWN0ZXJpdW0gdGhhdCB1dGlsaXplcyB4eWxvc2UgYW5kIHByb2R1Y2VzIGhpZ2ggbGV2ZWxzIG9mIGwtbGFjdGljIGFjaWQ7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk0OyAgY2l0YXRpb25fcGFnZXM9MjEwMi0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXJib24gY2F0YWJvbGl0ZSBjb250cm9sIG9mIHRoZSBtZXRhYm9saWMgbmV0d29yayBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgQmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIGNpdGF0aW9uX3BhZ2VzPTI0NS01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFjdGF0ZSByYWNlbWl6YXRpb24gYXMgYSByZXNjdWUgcGF0aHdheSBmb3Igc3VwcGx5aW5nIGQtbGFjdGF0ZSB0byB0aGUgY2VsbCB3YWxsIGJpb3N5bnRoZXNpcyBtYWNoaW5lcnkgaW4gTGFjdG9iYWNpbGx1cyBwbGFudGFydW07IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg3OyAgY2l0YXRpb25fcGFnZXM9Njc1MC02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzQvMzY5LzEyMTg4NjgvZHN1MDAzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBFbnRlcm9jb2NjdXMgbXVuZHRpaSBRVSAyNSwgYSBub24tZGFpcnkgYmFjdGVyaWFsIHN0cmFpbiBvZiBvdmluZSBmYWVjYWwgb3JpZ2luLCBjYW4gZmVybWVudCBib3RoIGNlbGxvYmlvc2UgYW5kIHh5bG9zZSB0byBwcm9kdWNlIGwtbGFjdGljIGFjaWQuIFRoZSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDQ1NTkxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZC17g7AV32AvZHVQJxXuyTFWVl6j" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f369%2f445591" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f369%2f445591" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17419" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&amp;Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/4">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6663244">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663244">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6663246">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663246">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6663250">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663250">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6663268">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663268">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6663300">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663300">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6663302">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663302">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6663304">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663304">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6663306">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663306">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6663308">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6663308">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/4/355/445136">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/4/379/445765">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Complete Genome Sequence of <em>Enterococcus mundtii</em> QU 25, an Efficient <span class="small-caps">l</span>-(+)-Lactic Acid-Producing Bacterium
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuh Shiwa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Genome Research Center</div>, <div class="insititution">NODAI Research Institute, Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156–8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuh+Shiwa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shiwa Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shiwa Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yuh Shiwa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroaki Yanase</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroaki+Yanase">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yanase H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yanase H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroaki Yanase</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuu Hirose</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Electronics-Inspired Interdisciplinary Research Institute (EIIRIS)</div>, <div class="insititution">Toyohashi University of Technology</div>, <div class="addr-line">Toyohashi 441-8580</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuu+Hirose">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirose Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirose Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yuu Hirose</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shohei Satomi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture</div>, <div class="insititution">Graduate School, Kyushu University</div>, <div class="addr-line">6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shohei+Satomi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Satomi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Satomi S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shohei Satomi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoko Araya-Kojima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoko+Araya-Kojima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Araya-Kojima T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Araya-Kojima T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tomoko Araya-Kojima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoru+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satoru Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Zendo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture</div>, <div class="insititution">Graduate School, Kyushu University</div>, <div class="addr-line">6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeshi+Zendo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zendo T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zendo T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takeshi Zendo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taku Chibazakura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taku+Chibazakura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chibazakura T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chibazakura T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Taku Chibazakura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mariko Shimizu-Kadota</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Environmental Science</div>, <div class="insititution">Musashino University</div>, <div class="addr-line">3-3-3 Ariake, Koto-ku, Tokyo 135-0063</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mariko+Shimizu-Kadota">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu-Kadota M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu-Kadota M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mariko Shimizu-Kadota</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirofumi Yoshikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Genome Research Center</div>, <div class="insititution">NODAI Research Institute, Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156–8502</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirofumi+Yoshikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoshikawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoshikawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hirofumi Yoshikawa</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenji Sonomoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Laboratory of Microbial Technology, Division of Applied Molecular Microbiology and Biomass Chemistry, Department of Bioscience and Biotechnology, Faculty of Agriculture</div>, <div class="insititution">Graduate School, Kyushu University</div>, <div class="addr-line">6-10-1 Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 92-642-3019. Fax. +81 92-642-3019. Email: <a href="mailto:sonomoto@agr.kyushu-u.ac.jp" target="_blank">sonomoto@agr.kyushu-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenji+Sonomoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sonomoto K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sonomoto K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kenji Sonomoto</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Iwate Tohoku Medical Megabank Organization, Iwate Medical University, 2-1-1 Nishitokuta, Yahaba-cho, Shiwa-gun, Iwate 028-3694, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Naotake Ogasawara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 369–377, <a href='https://doi.org/10.1093/dnares/dsu003'>https://doi.org/10.1093/dnares/dsu003</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 February 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 November 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">14 January 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="445591" href="/dnaresearch/article-pdf/21/4/369/1218868/dsu003.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yuh Shiwa, Hiroaki Yanase, Yuu Hirose, Shohei Satomi, Tomoko Araya-Kojima, Satoru Watanabe, Takeshi Zendo, Taku Chibazakura, Mariko Shimizu-Kadota, Hirofumi Yoshikawa, Kenji Sonomoto;  Complete Genome Sequence of <em>Enterococcus mundtii</em> QU 25, an Efficient <span class="small-caps">l</span>-(+)-Lactic Acid-Producing Bacterium, <em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 369–377, <a href='https://doi.org/10.1093/dnares/dsu003'>https://doi.org/10.1093/dnares/dsu003</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/445591?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/445591?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/445591?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/445591?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/445591?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/445591?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/445591?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Complete%20Genome%20Sequence%20of%20%26lt%3Bem%26gt%3BEnterococcus%20mundtii%26lt%3B%2Fem%26gt%3B%20QU%2025%2C%20an%20Efficient%20%26lt%3Bspan%20class%3D%26quot%3Bsmall-caps%26quot%3B%26gt%3Bl%26lt%3B%2Fspan%26gt%3B-(%2B)-Lactic%20Acid-Producing%20Bacterium&amp;publicationDate=2014-02-24&amp;volumeNum=21&amp;issueNum=4&amp;author=Shiwa%2C%20Yuh%3B%20Yanase%2C%20Hiroaki&amp;startPage=369&amp;endPage=377&amp;contentId=10.1093%2Fdnares%2Fdsu003&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f369%2f445591" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6663244 id="6663244" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Enterococcus mundtii</em> QU 25, a non-dairy bacterial strain of ovine faecal origin, can ferment both cellobiose and xylose to produce <span class="small-caps">l</span>-lactic acid. The use of this strain is highly desirable for economical <span class="small-caps">l</span>-lactate production from renewable biomass substrates. Genome sequence determination is necessary for the genetic improvement of this strain. We report the complete genome sequence of strain QU 25, primarily determined using Pacific Biosciences sequencing technology. The <em>E. mundtii</em> QU 25 genome comprises a 3 022 186-bp single circular chromosome (GC content, 38.6%) and five circular plasmids: pQY182, pQY082, pQY039, pQY024, and pQY003. In all, 2900 protein-coding sequences, 63 tRNA genes, and 6 rRNA operons were predicted in the QU 25 chromosome. Plasmid pQY024 harbours genes for mundticin production. We found that strain QU 25 produces a bacteriocin, suggesting that mundticin-encoded genes on plasmid pQY024 were functional. For lactic acid fermentation, two gene clusters were identified—one involved in the initial metabolism of xylose and uptake of pentose and the second containing genes for the pentose phosphate pathway and uptake of related sugars. This is the first complete genome sequence of an <em>E. mundtii</em> strain. The data provide insights into lactate production in this bacterium and its evolution among enterococci.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Enterococcus mundtii&quot;"><em>Enterococcus mundtii</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="xylose">xylose</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;l-lactic acid&quot;"><span class="small-caps">l</span>-lactic acid</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mundticin">mundticin</a></div>                            <h2 scrollto-destination=6663246 id="6663246" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Optically pure lactic acid is necessary for the production of the bioplastic polylactic acid. The use of cellulosic biomass instead of food crops should lower the cost for commercial production of this green plastic. <em>Enterococcus mundtii</em> QU 25 is a non-dairy bacterial strain that was originally isolated from ovine faeces.<sup><a reveal-id="DSU003C1" data-open="DSU003C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Unlike most lactic acid bacteria, strain QU 25 can ferment both cellobiose and xylose to produce <span class="small-caps">l</span>-lactic acid.<sup><a reveal-id="DSU003C1" data-open="DSU003C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU003C2" data-open="DSU003C2" class="link link-ref link-reveal xref-bibr">2</a></sup> This strain metabolizes a mixture of glucose and cellobiose simultaneously without apparent carbon catabolite repression (CCR),<sup><a reveal-id="DSU003C1" data-open="DSU003C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and it produces optically pure <span class="small-caps">l</span>-lactate (≥99.9%) with a yield of 1.41 mol/mol xylose consumed, without by-products such as acetic acid or ethanol.<sup><a reveal-id="DSU003C1" data-open="DSU003C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU003C2" data-open="DSU003C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Moreover, high productivity of <span class="small-caps">l</span>-lactic acid in an open repeated batch fermentation system under non-sterile conditions was demonstrated.<sup><a reveal-id="DSU003C3" data-open="DSU003C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Therefore, the use of strain QU 25 is highly desirable for the economical production of <span class="small-caps">l</span>-lactate from renewable biomass substrates. Furthermore, determination of the genome sequence of this bacterium is necessary to generate optimized, recombinant strains for commercial use.</p><p>Draft genome sequences of microorganisms can be obtained rapidly and cost-effectively by using second-generation sequencing technologies. Owing to its relatively short read length of second-generation platforms (100–700 bp), obtaining a complete genome sequence requires additional costs and time-consuming finishing steps such as scaffolding and gap closing. By comparison, third-generation single molecule real time (SMRT) sequencing technology developed by Pacific Biosciences (PacBio) can produce considerably long sequences (∼23 kb). Complete genome sequencing using only PacBio sequence data was recently reported.<sup><a reveal-id="DSU003C4" data-open="DSU003C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>To date, there have been only two draft genome sequences available for <em>E. mundtii</em> (strains CRL1656<sup><a reveal-id="DSU003C5" data-open="DSU003C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and ATCC 882). Here, we report the complete genome sequence of strain QU 25, whose chromosome sequence was determined using only PacBio sequence data. This is the first complete genome sequence of an <em>E. mundtii</em> strain. The data reveal useful insights on lactic acid fermentation in this bacterium, as well as phylogenetic relationships with other <em>Enterococcus</em> species.</p>                            <h2 scrollto-destination=6663250 id="6663250" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6663251 id="6663251" class="section-title" data-legacy-id=s2a>2.1. Culture conditions and preparation of genomic DNA for sequencing</h3>
<p><em>Enterococcus mundtii</em> QU 25 cells were grown to mid-log phase in GM17.<sup><a reveal-id="DSU003C6" data-open="DSU003C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Cells from 100-ml culture were harvested, suspended in 25 ml of a solution containing 2 g of polyethylene glycol 2000, 62.5 mg of egg white lysozyme, and 5 mM Tris–hydrochloride (pH 8.0), and incubated for 60 min at 37°C. After centrifugation, the cells were resuspended in 12.5 ml of TES buffer (50 mM Tris–hydrochloride [pH 7.6], 20 mM sodium ethylenediaminetetraacetic acid, and 25% sucrose), treated with 8 µg/ml of ribonuclease A for 60 min at 37°C, and then lysed with heat at 60°C for 2 h in the presence of 40 µg/ml of proteinase K and 1.7% sodium dodecyl sulphate. Total DNA was extracted gently from the lysate with PCI mixture (phenol, chloroform, and isoamyl alcohol in a ratio of 25 : 24 : 1, respectively) three times and precipitated with ethanol.</p>                            <h3 scrollto-destination=6663253 id="6663253" class="section-title" data-legacy-id=s2b>2.2. Genome sequencing</h3>
<p>For PacBio RS sequencing, two types of SMRTbell DNA template libraries were created with 1- and 10-kb sheared genomic DNA, and prepared using the standard PacBio RS sample preparation methods with C2 chemistry specific to each insert size. The 10-kb library was sequenced on eight SMRT cells with a 1 × 120-min collection protocol, generating 189 953 post-filtered continuous long reads (CLRs; mean length, 3702 bp; maximum length 20 405 bp; ∼234× coverage). The 1-kb library was sequenced on eight SMRT cells with a 2 × 55-min collection protocol, generating 258 068 post-filtered circular consensus sequences (CCS; mean length, 660 bp; ∼57× coverage).</p><p>For the second-generation Roche/454 sequencing, a 1.6-kb fragment library was sequenced using the GS-FLX+ system. A total of 302 056 reads (mean length, 455 bp; ∼34× coverage) was generated and assembled using Newbler (ver. 2.6). To estimate the plasmid copy number, a 500-bp paired-end library was sequenced using the Illumina Genome Analyzer IIx, generating 76-bp paired-end reads (48 724 736 reads, ∼1234× coverage). The copy number of plasmid per chromosome was estimated based on the coverage ratio between the corresponding contig and the chromosome contig using CLC Genomics workbench 6 (CLC bio, Aarhus, Denmark).</p>                            <h3 scrollto-destination=6663256 id="6663256" class="section-title" data-legacy-id=s2c>2.3. Error correction, assembly, and optical mapping</h3>
<p>The error correction of CLRs was performed using the command pacBioToCA.<sup><a reveal-id="DSU003C7" data-open="DSU003C7" class="link link-ref link-reveal xref-bibr">7</a></sup> CCS (57× length coverage) reads were used for correction. After error correction, we selected reads named PacBio-corrected Reads (PBcRs) for assembly. Assembly was performed using Celera Assembler (ver. 7.0).<sup><a reveal-id="DSU003C8" data-open="DSU003C8" class="link link-ref link-reveal xref-bibr">8</a></sup> To confirm the correctness of the assembly, DNA from QU 25 cells was digested using <em>Nco</em>I and a whole-genome optical map (OpGen, Inc., Gaithersburg, MD, USA) was generated.<sup><a reveal-id="DSU003C9" data-open="DSU003C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p>                            <h3 scrollto-destination=6663258 id="6663258" class="section-title" data-legacy-id=s2d>2.4. Genome analysis</h3>
<p>At first, the genome sequence was automatically annotated using the MiGAP, Microbial Genome Annotation Pipeline (<a class="link link-uri" href="http://www.migap.org">www.migap.org</a>) with the g-MiGAP level. In the pipeline, open reading frames (ORFs) were identified using MetaGeneAnnotator,<sup><a reveal-id="DSU003C10" data-open="DSU003C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and genes for tRNAs and rRNAs were identified by tRNAscan-SE and RNAmmer, respectively.<sup><a reveal-id="DSU003C11" data-open="DSU003C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Predicted ORFs were annotated using BLASTP searches against other <em>Enterococcus</em> genomes, and National Center of Biological Information (NCBI) nr<sup><a reveal-id="DSU003C12" data-open="DSU003C12" class="link link-ref link-reveal xref-bibr">12</a></sup> with an <em>E</em>-value of 1 × 10<sup>−10</sup>. Additional annotation was performed using InterProScan<sup><a reveal-id="DSU003C13" data-open="DSU003C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and KEGG pathway analyses.<sup><a reveal-id="DSU003C14" data-open="DSU003C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Regions containing prophage were predicted by a phage search tool, PHAST (<a class="link link-uri" href="http://phast.wishartlab.com/">http://phast.wishartlab.com/</a>)<sup><a reveal-id="DSU003C15" data-open="DSU003C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and further manual inspection. The insertion sequence (IS) was detected by ISsaga.<sup><a reveal-id="DSU003C16" data-open="DSU003C16" class="link link-ref link-reveal xref-bibr">16</a></sup> CRISPR loci were detected by CRISPRfinder.<sup><a reveal-id="DSU003C17" data-open="DSU003C17" class="link link-ref link-reveal xref-bibr">17</a></sup> For comparative genomes of other <em>Enterococcus</em> spp., a draft genome sequence of <em>E. mundtii</em> ATCC 882 was obtained from the Broad Institute website (<a class="link link-uri" href="https://olive.broadinstitute.org/genomes/ente_mund_atcc882.1">https://olive.broadinstitute.org/genomes/ente_mund_atcc882.1</a>). In addition, complete genome sequences of five <em>Enterococcus</em> spp. with (<em>E. casseliflavus</em> EC20<sup><a reveal-id="DSU003C18" data-open="DSU003C18" class="link link-ref link-reveal xref-bibr">18</a></sup>: NC_021023; <em>E. faecalis</em> V583<sup><a reveal-id="DSU003C19" data-open="DSU003C19" class="link link-ref link-reveal xref-bibr">19</a></sup>: NC_004668, NC_004669, NC_004670, and NC_004671; <em>E. faecium</em> Aus0004<sup><a reveal-id="DSU003C20" data-open="DSU003C20" class="link link-ref link-reveal xref-bibr">20</a></sup>: NC_017022, NC_017023, NC_017024, and NC_017032; <em>E. faecium</em> DO<sup><a reveal-id="DSU003C21" data-open="DSU003C21" class="link link-ref link-reveal xref-bibr">21</a></sup>: NC_017960, NC_017961, NC_017962, and NC_017963; <em>E. hirae</em> ATCC 9790<sup><a reveal-id="DSU003C22" data-open="DSU003C22" class="link link-ref link-reveal xref-bibr">22</a></sup>: NC_015845 and NC_018081) were obtained from the NCBI website for genome comparisons. GenomeMatcher<sup><a reveal-id="DSU003C23" data-open="DSU003C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and In Silico Molecular Cloning Genomics Edition (IMC-GE) software (In Silico Biology, Japan) were also used for intra- and inter-QU 25 genome comparisons.</p>                            <h3 scrollto-destination=6663260 id="6663260" class="section-title" data-legacy-id=s2e>2.5. Vancomycin resistance assay</h3>
<p><em>Enterococcus mundtii</em> QU 25 was propagated in MRS medium (Oxoid, Basingstoke, UK) at 30°C for 12 h as a pre-culture. One hundred microliters of pre-culture were inoculated in 10 ml of MRS medium containing various concentrations of vancomycin (2–100 µg/ml, Sigma, St. Louis, MO, USA). After incubation at 30°C for 24 h, the minimum concentration of vancomycin that resulted in the absence of growth was considered to be the minimum inhibitory concentration (MIC).</p>                            <h3 scrollto-destination=6663262 id="6663262" class="section-title" data-legacy-id=s2f>2.6. Bacteriocin activity assay</h3>
<p><em>Lactobacillus sakei</em> JCM 1157<sup>T</sup> and <em>E. faecalis</em> JCM 5803<sup>T</sup> were employed as indicator strains for bacteriocin activity. Taken together with <em>E. mundtii</em> QU 25, <em>E. mundtii</em> QU 2 (mundticin producer)<sup><a reveal-id="DSU003C24" data-open="DSU003C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and JCM 8731<sup>T</sup> (non-bacteriocin producer) were tested as positive and negative controls, respectively. The three <em>E. mundtii</em> strains were also used as indicator strains for cross- and self-immunity. All strains were propagated in MRS medium at 30°C for 12–18 h before use. Strains QU 25, QU 2, and JCM 8731<sup>T</sup> were cultured in MRS medium at 30°C for 12 h for bacteriocin production. Bacteriocin activity assay was performed by the spot-on-lawn method, as described previously.<sup><a reveal-id="DSU003C25" data-open="DSU003C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Briefly, 10 µl of each cell-free culture supernatant were spotted onto a double-layered agar plate containing 5 ml of Lactobacilli Agar AOAC (BD, Sparks, MD, USA) inoculated with an overnight culture of an indicator strain as an upper layer, and 10 ml of MRS medium supplemented with 1.2% agar as a bottom layer. After overnight incubation at appropriate temperatures for indicator strains, bacterial lawns were checked for inhibition zones.</p>                            <h3 scrollto-destination=6663264 id="6663264" class="section-title" data-legacy-id=s2g>2.7. Catalase and haemolysin activity assays</h3>
<p>Strain QU 25 was tested for catalase activity by two methods. First, cells cultured in MRS liquid medium at 30°C for 12 h were collected by centrifugation and examined for catalase activity by the addition of 3% hydrogen peroxide solution. Secondly, colonies formed on the sheep blood-containing agar plates (Eiken Chemical, Tokyo, Japan) after incubation at 30°C for 24 h were examined by the addition of 3% hydrogen peroxide solution. Generation of oxygen bubbles was considered to be indicative of catalase activity. Colonies formed on blood agar plates were also examined for haemolysin activity. Discolouration or clearing of blood agar in the vicinity of the colonies was regarded as haemolysis.</p>                            <h3 scrollto-destination=6663266 id="6663266" class="section-title" data-legacy-id=s2h>2.8. Nucleotide sequence accession numbers</h3>
<p>The complete genome sequence for <em>E. mundtii</em> QU 25 was deposited in GenBank/DDBJ/EMBL under accession numbers AP013036–AP013041.</p>                            <h2 scrollto-destination=6663268 id="6663268" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=6663269 id="6663269" class="section-title" data-legacy-id=s3a>3.1. Genome summary</h3>
<p>To determine the full genome sequence of <em>E. mundtii</em> QU 25, we combined two sequencing platforms: PacBio and Roche/454. As a result, the complete chromosome sequence of <em>E. mundtii</em> QU 25 was determined using only PacBio RS sequencing. This technology generates two types of sequences: CLRs and CCS reads.<sup><a reveal-id="DSU003C26" data-open="DSU003C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The read length of CLRs can reach up to 23 kb; however, the average base accuracy is only 82.1–84.4%.<sup><a reveal-id="DSU003C7" data-open="DSU003C7" class="link link-ref link-reveal xref-bibr">7</a></sup> CCS reads are consensus sequences obtained from multiple passes on a single sequence with relatively short lengths (∼2 kb) and a low error rate.<sup><a reveal-id="DSU003C27" data-open="DSU003C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The PBcR algorithm can improve read accuracy of CLRs from 80% to over 99.9% by using CCS reads.<sup><a reveal-id="DSU003C7" data-open="DSU003C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Error correction of CLRs was done using high-accuracy sequences of CCS with the PBcR algorithm.<sup><a reveal-id="DSU003C7" data-open="DSU003C7" class="link link-ref link-reveal xref-bibr">7</a></sup> After error correction, the resulting 6806 PBcRs of at least 7-kb length (mean length, 8517 bp; maximum length, 16 733 bp; ∼20× coverage) were assembled, generating five contigs. Comparison between the <em>Nco</em>I digest of the whole-genome optical map and <em>in silico</em>-generated physical maps of contigs showed that the largest contig (3 Mb) was mapped onto the chromosome, confirming the correctness of the assembly (Fig. <a reveal-id="DSU003F1" data-open="DSU003F1" class="link link-reveal link-table xref-fig">1</a>A). Thus, we concluded that the remaining four contigs were plasmids. The presence of the 2.5-kb plasmid, which had been detected as an extrachromosomal element by agarose gel electrophoresis (data not shown), was not included in the five contigs generated by PacBio, due to its assembly threshold of 7-kb read length. Therefore, we adapted one contig produced using Roche/454. </p>                        <a id="6663272" scrollto-destination="6663272"></a>
<div id="DSU003F1" data-id="DSU003F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/m_dsu00301.png?Expires=2147483647&Signature=ZA2XunuH1saSmiyBS8MJDZ~54mYGro2jc2~RVqpIVN8w8s-ZTcb4Wrx5NKPOyrIMjrYv0kYF3rAYZL62GUPvsGHaQowibbTPALHzaHGJbWbdN~jb23rg8eGebJb67noFAjd~WO6ZqocKb0JDWIU5LoUmhKi6Vz0jjGv1GrXay6JGbNzjusKWbn45TahOtRhvp-ibwhepFTpT7dPGm8lLoXdXZ-Rzyoje4-~L1lPcTi9xooIDhFSbyC8PNMS1JsPmOq5b10h7kziYU7yn~SpZoq6a1WON-rqQxaSrJKBdpDLw2R6WQS1wozmncF7SXH-HL~uoxz5kUzphxA7BHu~jBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of the Enterococcus mundtii QU 25 complete genome. (A) Linearized NcoI-digested whole-genome optical map of QU 25 compared with the in silico-derived NcoI-digest map, demonstrating correct chromosome assembly. (B) Genome map of the QU 25 strain. In the outermost circle, three prophages of phiEmqu1, phiEmqu2, and phiEmqu3, replication origin (dnaA), and terminus (dif) are shown. In the second circle, the ORFs transcribed in a clockwise manner are shown as bars. The third circle shows ORFS transcribed in a counter-clockwise manner. The fourth to ninth circles depict the results of ortholog analyses (BLASTP E-value ≤1 × 10−10) with E. mundtii ATCC 882, E. faecium DO, E. faecium Aus0004, E. hirae ATCC 9790, E. casseliflavus EC20, and E. faecalis V583, respectively. The extent of homology relative to QU 25 is depicted using a heat map of arbitrarily chosen bins. The colour scheme and percentage identity for orthologs are as follows: red, orthologs with &gt;90% identity; green, 70–90% identity; blue, 50–70% identity; and black, &lt;50% identity. The 10th circle shows the positions of rRNA operons. The last two (innermost) circles represent G + C content (purple &gt;39.5% average; green &lt;39.5% average; range from 32 to 47%) and G + C skew, both calculated for a 10-kb window with 1-kb stepping." path-from-xml="dsu00301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6663272/dsu00301.tiff" path-from-xml="dsu00301.tiff" target="_blank">View large</a><a section="6663272" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu00301.png?Expires=2147483647&Signature=ShW7ITf~e0M9P1UEC8oLoj225hCrNq93CQvONxvT-j1hY0tYJ18Ge8FtTVyaMLYv-gX3mQHeoLuvoWzDOMdP1oc70i3rYCy-F7ctq5Lds~xYDtwGaFEM1VwnzS7vPK7mWt64JlqqMGCfgycy2pWPl63lOQVADKw0l9pbWJGw7Xo-monaFRMRLIVNdBuLccUGZePz6muj0FTZDxd39VEBbV8oJLyA5AIUfuauJd4xFDuKWIuec6Gnuq2rgLaWRKZ5jIkbipPDx9I3nKp4jx1qlDrjKiSh5e8Y6aThjBDC34SIrdS80jhaUmfoGuqJB5FQvlSukD~E9oUTJcZNpQUflg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6663272&ar=445591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of the <em>Enterococcus mundtii</em> QU 25 complete genome. (A) Linearized <em>Nco</em>I-digested whole-genome optical map of QU 25 compared with the <em>in silico</em>-derived <em>Nco</em>I-digest map, demonstrating correct chromosome assembly. (B) Genome map of the QU 25 strain. In the outermost circle, three prophages of phiEmqu1, phiEmqu2, and phiEmqu3, replication origin (<em>dnaA</em>), and terminus (<em>dif</em>) are shown. In the second circle, the ORFs transcribed in a clockwise manner are shown as bars. The third circle shows ORFS transcribed in a counter-clockwise manner. The fourth to ninth circles depict the results of ortholog analyses (BLASTP <em>E</em>-value ≤1 × 10<sup>−10</sup>) with <em>E. mundtii</em> ATCC 882, <em>E. faecium</em> DO, <em>E. faecium</em> Aus0004, <em>E. hirae</em> ATCC 9790, <em>E. casseliflavus</em> EC20, and <em>E. faecalis</em> V583, respectively. The extent of homology relative to QU 25 is depicted using a heat map of arbitrarily chosen bins. The colour scheme and percentage identity for orthologs are as follows: red, orthologs with &gt;90% identity; green, 70–90% identity; blue, 50–70% identity; and black, &lt;50% identity. The 10th circle shows the positions of rRNA operons. The last two (innermost) circles represent G + C content (purple &gt;39.5% average; green &lt;39.5% average; range from 32 to 47%) and G + C skew, both calculated for a 10-kb window with 1-kb stepping.</p></div></div><div content-id="DSU003F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/m_dsu00301.png?Expires=2147483647&Signature=ZA2XunuH1saSmiyBS8MJDZ~54mYGro2jc2~RVqpIVN8w8s-ZTcb4Wrx5NKPOyrIMjrYv0kYF3rAYZL62GUPvsGHaQowibbTPALHzaHGJbWbdN~jb23rg8eGebJb67noFAjd~WO6ZqocKb0JDWIU5LoUmhKi6Vz0jjGv1GrXay6JGbNzjusKWbn45TahOtRhvp-ibwhepFTpT7dPGm8lLoXdXZ-Rzyoje4-~L1lPcTi9xooIDhFSbyC8PNMS1JsPmOq5b10h7kziYU7yn~SpZoq6a1WON-rqQxaSrJKBdpDLw2R6WQS1wozmncF7SXH-HL~uoxz5kUzphxA7BHu~jBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of the Enterococcus mundtii QU 25 complete genome. (A) Linearized NcoI-digested whole-genome optical map of QU 25 compared with the in silico-derived NcoI-digest map, demonstrating correct chromosome assembly. (B) Genome map of the QU 25 strain. In the outermost circle, three prophages of phiEmqu1, phiEmqu2, and phiEmqu3, replication origin (dnaA), and terminus (dif) are shown. In the second circle, the ORFs transcribed in a clockwise manner are shown as bars. The third circle shows ORFS transcribed in a counter-clockwise manner. The fourth to ninth circles depict the results of ortholog analyses (BLASTP E-value ≤1 × 10−10) with E. mundtii ATCC 882, E. faecium DO, E. faecium Aus0004, E. hirae ATCC 9790, E. casseliflavus EC20, and E. faecalis V583, respectively. The extent of homology relative to QU 25 is depicted using a heat map of arbitrarily chosen bins. The colour scheme and percentage identity for orthologs are as follows: red, orthologs with &gt;90% identity; green, 70–90% identity; blue, 50–70% identity; and black, &lt;50% identity. The 10th circle shows the positions of rRNA operons. The last two (innermost) circles represent G + C content (purple &gt;39.5% average; green &lt;39.5% average; range from 32 to 47%) and G + C skew, both calculated for a 10-kb window with 1-kb stepping." path-from-xml="dsu00301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6663272/dsu00301.tiff" path-from-xml="dsu00301.tiff" target="_blank">View large</a><a section="6663272" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu00301.png?Expires=2147483647&Signature=ShW7ITf~e0M9P1UEC8oLoj225hCrNq93CQvONxvT-j1hY0tYJ18Ge8FtTVyaMLYv-gX3mQHeoLuvoWzDOMdP1oc70i3rYCy-F7ctq5Lds~xYDtwGaFEM1VwnzS7vPK7mWt64JlqqMGCfgycy2pWPl63lOQVADKw0l9pbWJGw7Xo-monaFRMRLIVNdBuLccUGZePz6muj0FTZDxd39VEBbV8oJLyA5AIUfuauJd4xFDuKWIuec6Gnuq2rgLaWRKZ5jIkbipPDx9I3nKp4jx1qlDrjKiSh5e8Y6aThjBDC34SIrdS80jhaUmfoGuqJB5FQvlSukD~E9oUTJcZNpQUflg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6663272&ar=445591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of the <em>Enterococcus mundtii</em> QU 25 complete genome. (A) Linearized <em>Nco</em>I-digested whole-genome optical map of QU 25 compared with the <em>in silico</em>-derived <em>Nco</em>I-digest map, demonstrating correct chromosome assembly. (B) Genome map of the QU 25 strain. In the outermost circle, three prophages of phiEmqu1, phiEmqu2, and phiEmqu3, replication origin (<em>dnaA</em>), and terminus (<em>dif</em>) are shown. In the second circle, the ORFs transcribed in a clockwise manner are shown as bars. The third circle shows ORFS transcribed in a counter-clockwise manner. The fourth to ninth circles depict the results of ortholog analyses (BLASTP <em>E</em>-value ≤1 × 10<sup>−10</sup>) with <em>E. mundtii</em> ATCC 882, <em>E. faecium</em> DO, <em>E. faecium</em> Aus0004, <em>E. hirae</em> ATCC 9790, <em>E. casseliflavus</em> EC20, and <em>E. faecalis</em> V583, respectively. The extent of homology relative to QU 25 is depicted using a heat map of arbitrarily chosen bins. The colour scheme and percentage identity for orthologs are as follows: red, orthologs with &gt;90% identity; green, 70–90% identity; blue, 50–70% identity; and black, &lt;50% identity. The 10th circle shows the positions of rRNA operons. The last two (innermost) circles represent G + C content (purple &gt;39.5% average; green &lt;39.5% average; range from 32 to 47%) and G + C skew, both calculated for a 10-kb window with 1-kb stepping.</p></div></div><p>The genome of QU 25 comprises a single circular chromosome of 3 022 186 bp (GC content, 38.6%) and five circular plasmids: pQY182, pQY082, pQY039, pQY024, and pQY003, with lengths of 181 920, 82 213, 38 528, 23 629, and 2584 bp, and GC contents of 36.2, 35.8, 33.8, 35.3, and 38.9%, respectively (Table <a reveal-id="DSU003TB1" data-open="DSU003TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSU003F1" data-open="DSU003F1" class="link link-reveal link-table xref-fig">1</a>B). Coordinates on the genome were designated as bp starting from the first nucleotide of the start codon of <em>dnaA.</em> We confirmed the likely location of the replication terminus by identifying a single <em>dif</em> sequence (ACTTTGTATAATATATATTATGTAAACT, position 1 449 088 to 1 449 115) that aligns with the shift in GC skew (Fig. <a reveal-id="DSU003F1" data-open="DSU003F1" class="link link-reveal link-table xref-fig">1</a>B). A total of 2900 protein-coding sequences (CDS), 63 tRNA genes, and 6 rRNA operons were predicted in the QU 25 chromosome. </p>                        <a id="6663274" scrollto-destination="6663274"></a>
<div content-id="DSU003TB1" class="table-modal"><div class="table-wrap"><div id="DSU003TB1" class="table-wrap-title" data-id="DSU003TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General features of the <em>Enterococcus mundtii</em> QU 25 genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Features </th><th>Chromosome </th><th>pQY182 </th><th>pQY082 </th><th>pQY039 </th><th>pQY024 </th><th>pQY003 </th></tr></thead><tbody><tr><td>Size (bp) </td><td>3 022 186 </td><td>181 920 </td><td>82 213 </td><td>38 528 </td><td>23 629 </td><td>2584 </td></tr><tr><td>G + C content (%) </td><td>38.6 </td><td>36.2 </td><td>35.8 </td><td>33.8 </td><td>35.3 </td><td>38.9 </td></tr><tr><td>No. of rRNA operons </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>No. of tRNA genes </td><td>63 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>CDS (protein coding) </td><td>2900 </td><td>178 </td><td>82 </td><td>36 </td><td>21 </td><td>4 </td></tr><tr><td>Average of CDS length (bp) </td><td>884 </td><td>813 </td><td>818 </td><td>850 </td><td>903 </td><td>436 </td></tr><tr><td>Estimated copy number of replicon </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>5 </td></tr></tbody></table></div><div content-id="DSU003TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Features </th><th>Chromosome </th><th>pQY182 </th><th>pQY082 </th><th>pQY039 </th><th>pQY024 </th><th>pQY003 </th></tr></thead><tbody><tr><td>Size (bp) </td><td>3 022 186 </td><td>181 920 </td><td>82 213 </td><td>38 528 </td><td>23 629 </td><td>2584 </td></tr><tr><td>G + C content (%) </td><td>38.6 </td><td>36.2 </td><td>35.8 </td><td>33.8 </td><td>35.3 </td><td>38.9 </td></tr><tr><td>No. of rRNA operons </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>No. of tRNA genes </td><td>63 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>CDS (protein coding) </td><td>2900 </td><td>178 </td><td>82 </td><td>36 </td><td>21 </td><td>4 </td></tr><tr><td>Average of CDS length (bp) </td><td>884 </td><td>813 </td><td>818 </td><td>850 </td><td>903 </td><td>436 </td></tr><tr><td>Estimated copy number of replicon </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>5 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6663274" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU003TB1" class="table-wrap-title" data-id="DSU003TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General features of the <em>Enterococcus mundtii</em> QU 25 genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Features </th><th>Chromosome </th><th>pQY182 </th><th>pQY082 </th><th>pQY039 </th><th>pQY024 </th><th>pQY003 </th></tr></thead><tbody><tr><td>Size (bp) </td><td>3 022 186 </td><td>181 920 </td><td>82 213 </td><td>38 528 </td><td>23 629 </td><td>2584 </td></tr><tr><td>G + C content (%) </td><td>38.6 </td><td>36.2 </td><td>35.8 </td><td>33.8 </td><td>35.3 </td><td>38.9 </td></tr><tr><td>No. of rRNA operons </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>No. of tRNA genes </td><td>63 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>CDS (protein coding) </td><td>2900 </td><td>178 </td><td>82 </td><td>36 </td><td>21 </td><td>4 </td></tr><tr><td>Average of CDS length (bp) </td><td>884 </td><td>813 </td><td>818 </td><td>850 </td><td>903 </td><td>436 </td></tr><tr><td>Estimated copy number of replicon </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>5 </td></tr></tbody></table></div><div content-id="DSU003TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Features </th><th>Chromosome </th><th>pQY182 </th><th>pQY082 </th><th>pQY039 </th><th>pQY024 </th><th>pQY003 </th></tr></thead><tbody><tr><td>Size (bp) </td><td>3 022 186 </td><td>181 920 </td><td>82 213 </td><td>38 528 </td><td>23 629 </td><td>2584 </td></tr><tr><td>G + C content (%) </td><td>38.6 </td><td>36.2 </td><td>35.8 </td><td>33.8 </td><td>35.3 </td><td>38.9 </td></tr><tr><td>No. of rRNA operons </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>No. of tRNA genes </td><td>63 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>CDS (protein coding) </td><td>2900 </td><td>178 </td><td>82 </td><td>36 </td><td>21 </td><td>4 </td></tr><tr><td>Average of CDS length (bp) </td><td>884 </td><td>813 </td><td>818 </td><td>850 </td><td>903 </td><td>436 </td></tr><tr><td>Estimated copy number of replicon </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>5 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6663274" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6663275 id="6663275" class="section-title" data-legacy-id=s3b>3.2. Genomic comparisons with other <em>Enterococcus</em> species</h3>
<p>From the phylogeny based on 16S rRNA sequences, <em>E. mundtii</em> was closely related to <em>E. hirae</em> and <em>E. faecium</em>, while <em>E. casseliflavus</em> and <em>E. faecalis</em> were more distantly related to <em>E. mundtii.</em><sup><a reveal-id="DSU003C28" data-open="DSU003C28" class="link link-ref link-reveal xref-bibr">28</a></sup> To investigate the taxonomic position of <em>E. mundtii</em> QU 25 based on genome-wide comparisons, we first carried out ortholog analysis with the draft genome of <em>E. mundtii</em> ATCC 882 and five complete genomes of other <em>Enterococcus</em> spp., including two <em>E. faecium</em> strains (DO and Aus0004), <em>E. hirae</em> ATCC 9790, <em>E. casseliflavus</em> EC20, and <em>E. faecalis</em> V583 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> and Fig. <a reveal-id="DSU003F1" data-open="DSU003F1" class="link link-reveal link-table xref-fig">1</a>B). The results were consistent with the phylogeny based on 16S rRNA. While <em>E. mundtii</em> ATCC 882 showed the highest similarity to QU 25 as anticipated, two other species (<em>E. faecium</em> DO and Aus0004, and <em>E. hirae</em> ATCC 9790) showed moderate degrees of similarity, while <em>E. casseliflavus</em> EC20 and <em>E. faecalis</em> V583 were the least similar. DNA dot plot analysis showed the centre diagonal line between QU 25, the two <em>E. faecium</em> strains, and <em>E. hirae</em> (Fig. <a reveal-id="DSU003F2" data-open="DSU003F2" class="link link-reveal link-table xref-fig">2</a>), indicating that not only each of the genes but also their genome structures (or gene orders) were related. There were gene regions unique to the QU 25 strain (Fig. <a reveal-id="DSU003F1" data-open="DSU003F1" class="link link-reveal link-table xref-fig">1</a>B). Except for prophages, the 9-kb region (positions 1 359 588–1 368 963) includes five hypothetical proteins and a cell wall-anchored protein with a LPXTG motif. </p>                        <a id="6663277" scrollto-destination="6663277"></a>
<div id="DSU003F2" data-id="DSU003F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/m_dsu00302.png?Expires=2147483647&Signature=inOjIjsS710DmVgla6K2jImL7StKgeASDlb2veYQpTCIdkQ~bgs9JzOP3PuWOQ7RkvvkGnZEyWWHmBtWnjO53WDku~lXQUnLlWodGkj~MjP5DN7c8mj458q8EwvxpD~SuxeduVOaX5NzlSzf-S-UNmYy3ihPm~hOdxoYneWmfxwW0mt4xvgngPOhjfiCtFglfKvpQ-u9ha2e6mA9gbco2SstMV6RYIWR4~nL3GwtZJPmdAUiOh~4Cf~Fm0IzgBe93Z9RgovHGguDnpGUEJMgcAtivn8J~5o12HimK9Xr2K9mq8e27TFeXKpAKtevp3ZHyJ3QcoGvw1WawaOnTQ4MbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot plots comparing the chromosomal organization among the six Enterococcus species. Dot plots were generated using BLASTN and GenomeMatcher software. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6663277/dsu00302.tiff" path-from-xml="dsu00302.tiff" target="_blank">View large</a><a section="6663277" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu00302.png?Expires=2147483647&Signature=XWj1Gz2BsCedvFaRlEFASTMpo8kta~4VHfkOzjE3JCvgvtU3pQ~Tcbw4uATbjfEbJgB4xlQSPlUfaSLuejUfX7xw1lVQ8~H-~DcqsQ4WyhJ4xp~MkVqUiPOC-Ag~pxU1T~LFr6iTbT5IUMImKLajY1mWUKdQzOc8l7M0oS7Pzo50oxh24l0Fhd1HRZoUtsEcm48TPY4~krR-PTYFoJ-z~ylUT1kBPJRJ5JNXBnU6BHtFsgGO0Y67Y3~LktaHvQC-ZBY5NIe3yy2gFEZxbIYMnFdRkKlFs5kdjm0dJHoZMyjXsR3bs8m4EoBiOx9QZCayFrwqSmgSswlvIB1nOSP1zA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6663277&ar=445591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot plots comparing the chromosomal organization among the six <em>Enterococcus</em> species. Dot plots were generated using BLASTN and GenomeMatcher software. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU003F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/m_dsu00302.png?Expires=2147483647&Signature=inOjIjsS710DmVgla6K2jImL7StKgeASDlb2veYQpTCIdkQ~bgs9JzOP3PuWOQ7RkvvkGnZEyWWHmBtWnjO53WDku~lXQUnLlWodGkj~MjP5DN7c8mj458q8EwvxpD~SuxeduVOaX5NzlSzf-S-UNmYy3ihPm~hOdxoYneWmfxwW0mt4xvgngPOhjfiCtFglfKvpQ-u9ha2e6mA9gbco2SstMV6RYIWR4~nL3GwtZJPmdAUiOh~4Cf~Fm0IzgBe93Z9RgovHGguDnpGUEJMgcAtivn8J~5o12HimK9Xr2K9mq8e27TFeXKpAKtevp3ZHyJ3QcoGvw1WawaOnTQ4MbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot plots comparing the chromosomal organization among the six Enterococcus species. Dot plots were generated using BLASTN and GenomeMatcher software. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6663277/dsu00302.tiff" path-from-xml="dsu00302.tiff" target="_blank">View large</a><a section="6663277" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu00302.png?Expires=2147483647&Signature=XWj1Gz2BsCedvFaRlEFASTMpo8kta~4VHfkOzjE3JCvgvtU3pQ~Tcbw4uATbjfEbJgB4xlQSPlUfaSLuejUfX7xw1lVQ8~H-~DcqsQ4WyhJ4xp~MkVqUiPOC-Ag~pxU1T~LFr6iTbT5IUMImKLajY1mWUKdQzOc8l7M0oS7Pzo50oxh24l0Fhd1HRZoUtsEcm48TPY4~krR-PTYFoJ-z~ylUT1kBPJRJ5JNXBnU6BHtFsgGO0Y67Y3~LktaHvQC-ZBY5NIe3yy2gFEZxbIYMnFdRkKlFs5kdjm0dJHoZMyjXsR3bs8m4EoBiOx9QZCayFrwqSmgSswlvIB1nOSP1zA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6663277&ar=445591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot plots comparing the chromosomal organization among the six <em>Enterococcus</em> species. Dot plots were generated using BLASTN and GenomeMatcher software. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div>                            <h3 scrollto-destination=6663278 id="6663278" class="section-title" data-legacy-id=s3c>3.3. Mobile genetic elements</h3>
<p>Three chromosomal regions were identified as prophage loci. Their positions are indicated in Fig. <a reveal-id="DSU003F1" data-open="DSU003F1" class="link link-reveal link-table xref-fig">1</a>B and are named phiEmqu1 (38.7 kb; positions 806 547–845 215), phiEmqu2 (47.9 kb; positions 2 327 297–2 375 151), and phiEmqu3 (40.8 kb; positions 2 556 843–2 597 594). Sixty, 70, and 54 phage-related genes were identified in these regions, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Prophages phiEmqu1 and phiEmqu3 contained several putative genes involved in DNA replication. However, no genes for DNA synthesis were found in the largest prophage phiEmqu2, suggesting that it is replication-defective. As shown in Fig. <a reveal-id="DSU003F1" data-open="DSU003F1" class="link link-reveal link-table xref-fig">1</a>B, the locations of <em>ori</em> (<em>dnaA</em>) and <em>ter</em> (<em>dif</em>) were not exactly opposite each other. The <em>dif</em> motif, which is strongly associated with replication termini,<sup><a reveal-id="DSU003C29" data-open="DSU003C29" class="link link-ref link-reveal xref-bibr">29</a></sup> was ∼60 kb off the exact opposite position of <em>ori</em> (<em>dnaA</em>). Phage-mediated replicore imbalance has been observed in <em>E. faecium</em> Aus0004.<sup><a reveal-id="DSU003C20" data-open="DSU003C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Thus, the replicore balance of QU 25 could be disrupted by the insertion of phiEmqu2 and/or phiEmqu3.</p><p>Clustered regularly interspaced short palindromic repeats (CRISPRs) are involved in a recently discovered interference pathway that protects cells from bacteriophages and conjugative plasmids. Approximately 40% of sequenced bacterial genomes and ∼90% of genomes from archaea contain at least one CRISPR locus.<sup><a reveal-id="DSU003C30" data-open="DSU003C30" class="link link-ref link-reveal xref-bibr">30</a></sup> No CRISPR loci were detected in the QU 25 genome. Many insertion sequence elements (ISEs) have been found in enterococci. The relatively closed species, such as <em>E. mundtii</em> ATCC 882, <em>E. faecium</em> DO and Aus0004 strains, and <em>E. hirae</em> ATCC 9790, have 21, 180, 76, and 14 ISEs and transposase-related genes, respectively. At least 13 different ISEs were detected in the QU 25 genome, ranging in copy number from 1 to 5, representing 33 distinct copies and distributed around the chromosome and plasmids (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The most frequently observed ISE type was the ISL3 family.</p>                            <h3 scrollto-destination=6663281 id="6663281" class="section-title" data-legacy-id=s3d>3.4. Plasmids</h3>
<p><em>Enterococcus</em> spp. have been reported to possess a number of plasmids that often confer resistance to antimicrobials and particular heavy metals, and serve to enhance virulence and/or DNA repair mechanisms.<sup><a reveal-id="DSU003C31  DSU003C34" data-open="DSU003C31  DSU003C34" class="link link-ref link-reveal xref-bibr">31–34</a></sup> In strain QU 25, the plasmid copy number per chromosome was estimated by observing the distribution of read coverage of the Illumina sequence read, which indicated one copy of pQY182, pQY082, pQY039, and pQY024, and five copies of pQY003 (Table <a reveal-id="DSU003TB1" data-open="DSU003TB1" class="link link-reveal link-table xref-fig">1</a>). BLASTN analyses showed that these five plasmids were similar to those of <em>E. mundtii</em>, <em>E. faecium</em>, <em>E. hirae</em>, or <em>E. faecalis</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p><p>Plasmid pQY182 harbours genes that encode a two-component regulatory system, a cellulose 1,4-beta-cellobiosidase, a toxin–antitoxin system, and several proteins with DNA repair functions. Plasmid pQY082 harbours duplicated regions of 8.3 kb, which include the IS1675 transposase, <em>ubiD</em> family decarboxylase, two cell surface proteins, and two proteins with unknown functions (EMQU_3088-3094 and EMQU_3155-3160 with 99% similarity). pQY082 also harbours genes that encode several proteins with DNA repair functions and a toxin–antitoxin system. Toxin–antitoxin systems have been frequently reported in <em>E. faecium</em> strains,<sup><a reveal-id="DSU003C35" data-open="DSU003C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and the QU 25 chromosome additionally has at least four such systems. Plasmid pQY039 contains several genes for a DNA damage-inducible protein and a gene encoding a toxin–antitoxin system. Plasmid pQY024 also harbours genes for a DNA damage-inducible protein, a toxin–antitoxin system, and mundticin KS genes (see Discussion later). Plasmid pQY003 only harbours genes with unknown function, except for the replication initiation protein.</p>                            <h3 scrollto-destination=6663284 id="6663284" class="section-title" data-legacy-id=s3e>3.5. Vancomycin resistance</h3>
<p>Because many <em>Enterococcus</em> isolates show vancomycin resistance which has been associated with hospital-acquired infections, we tested the sensitivity of QU 25 to this antibiotic for the safety of industrial use. The results showed that the vancomycin MIC for QU 25 was &gt;2 µg/ml, indicating that this strain is vancomycin-sensitive. Several known genes involved in vancomycin resistance (<em>vanA</em>, <em>vanB</em>, <em>vanX</em>, <em>vanH</em>, <em>vanR</em>, and <em>vanS</em>)<sup><a reveal-id="DSU003C36" data-open="DSU003C36" class="link link-ref link-reveal xref-bibr">36</a></sup> were not detected in the QU 25 genome and also in plasmids.</p>                            <h3 scrollto-destination=6663286 id="6663286" class="section-title" data-legacy-id=s3f>3.6. Bacteriocin activity and self- and cross-immunity</h3>
<p>Bacteriocins are ribosomally synthesized bacterial peptides or proteins that show antimicrobial activity, generally against species that are closely related to bacteriocin producers.<sup><a reveal-id="DSU003C24" data-open="DSU003C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Mundticin is one of the bacteriocins produced by some <em>E. mundtii</em> strains. It is significant to clarify whether QU 25 produces mundticin or not for the resistivity to contamination in a large-scale culture. Three genes, such as <em>munA</em> (mundticin precursor), <em>munB</em> (ATP-binding cassette [ABC] transporter), and <em>munC</em> (mundticin KS immunity protein), are responsible for mundticin production in <em>E. mundtii</em> NERI 7393.<sup><a reveal-id="DSU003C37" data-open="DSU003C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The gene cluster containing these three genes was identified on plasmid pQY024, and <em>munA</em> (EMQU_3203; 100% nucleotide sequence identity), <em>munB</em> (EMQU_3204; 99.56% nucleotide sequence identity), and <em>munC</em> (EMQU_3205; 98.99% nucleotide sequence identity) showed high homology with corresponding genes in strain NERI 7393.</p><p>To examine whether the gene cluster for mundticin synthesis was functional, QU 25 was tested for bacteriocin production. QU 25 and <em>E. mundtii</em> QU 2 showed bacteriocin activity against three indicator strains (<em>L. sakei</em> JCM 1157<sup>T</sup>, <em>E. faecalis</em> JCM 5803<sup>T</sup>, and <em>E. mundtii</em> JCM 8731<sup>T</sup>), none of which showed inhibitory activity (Table <a reveal-id="DSU003TB2" data-open="DSU003TB2" class="link link-reveal link-table xref-fig">2</a>). QU 25 and QU 2 showed no activity against each other (Table <a reveal-id="DSU003TB2" data-open="DSU003TB2" class="link link-reveal link-table xref-fig">2</a>), which indicated that these strains have self- and cross-immunity against their bacteriocins. Bacteriocin-producing strains are known to have immunity (tolerance) to their own bacteriocins and to bacteriocins with similar structures. Collectively, these results strongly suggest that QU 25 produces a bacteriocin with similar characteristics to mundticin produced by QU 2. </p>                        <a id="6663289" scrollto-destination="6663289"></a>
<div content-id="DSU003TB2" class="table-modal"><div class="table-wrap"><div id="DSU003TB2" class="table-wrap-title" data-id="DSU003TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Bacteriocin activity and immunity of <em>Enterococcus mundtii</em> strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Indicator strain </th><th colspan="3">Bacteriocin producer<hr /> </th></tr><tr><th><em>E. mundtii</em> QU 25 </th><th><em>E. mundtii</em> QU 2 </th><th><em>E. mundtii</em> JCM 8731<sup>T</sup> </th></tr></thead><tbody><tr><td><em>E. mundtii</em> QU 25 </td><td>− </td><td>− </td><td>− </td></tr><tr><td><em>E. mundtii</em> QU 2 </td><td>− </td><td>− </td><td>− </td></tr><tr><td><em>E. mundtii</em> JCM 8731<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td><em>E. faecalis</em> JCM 5803<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td><em>L. sakei</em> JCM 1157<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr></tbody></table></div><div content-id="DSU003TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Indicator strain </th><th colspan="3">Bacteriocin producer<hr /> </th></tr><tr><th><em>E. mundtii</em> QU 25 </th><th><em>E. mundtii</em> QU 2 </th><th><em>E. mundtii</em> JCM 8731<sup>T</sup> </th></tr></thead><tbody><tr><td><em>E. mundtii</em> QU 25 </td><td>− </td><td>− </td><td>− </td></tr><tr><td><em>E. mundtii</em> QU 2 </td><td>− </td><td>− </td><td>− </td></tr><tr><td><em>E. mundtii</em> JCM 8731<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td><em>E. faecalis</em> JCM 5803<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td><em>L. sakei</em> JCM 1157<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p>+, growth inhibition of indicator strains; −, no inhibition of indicator strains.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6663289" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU003TB2" class="table-wrap-title" data-id="DSU003TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Bacteriocin activity and immunity of <em>Enterococcus mundtii</em> strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Indicator strain </th><th colspan="3">Bacteriocin producer<hr /> </th></tr><tr><th><em>E. mundtii</em> QU 25 </th><th><em>E. mundtii</em> QU 2 </th><th><em>E. mundtii</em> JCM 8731<sup>T</sup> </th></tr></thead><tbody><tr><td><em>E. mundtii</em> QU 25 </td><td>− </td><td>− </td><td>− </td></tr><tr><td><em>E. mundtii</em> QU 2 </td><td>− </td><td>− </td><td>− </td></tr><tr><td><em>E. mundtii</em> JCM 8731<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td><em>E. faecalis</em> JCM 5803<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td><em>L. sakei</em> JCM 1157<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr></tbody></table></div><div content-id="DSU003TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Indicator strain </th><th colspan="3">Bacteriocin producer<hr /> </th></tr><tr><th><em>E. mundtii</em> QU 25 </th><th><em>E. mundtii</em> QU 2 </th><th><em>E. mundtii</em> JCM 8731<sup>T</sup> </th></tr></thead><tbody><tr><td><em>E. mundtii</em> QU 25 </td><td>− </td><td>− </td><td>− </td></tr><tr><td><em>E. mundtii</em> QU 2 </td><td>− </td><td>− </td><td>− </td></tr><tr><td><em>E. mundtii</em> JCM 8731<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td><em>E. faecalis</em> JCM 5803<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td><em>L. sakei</em> JCM 1157<sup>T</sup> </td><td>+ </td><td>+ </td><td>− </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p>+, growth inhibition of indicator strains; −, no inhibition of indicator strains.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6663289" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6663290 id="6663290" class="section-title" data-legacy-id=s3g>3.7. Haemolysin activity</h3>
<p>Haemolysin is one of the putative enterococcal virulence factors,<sup><a reveal-id="DSU003C20" data-open="DSU003C20" class="link link-ref link-reveal xref-bibr">20</a></sup> so it is important to test the haemolysin activity for the safety of industrial use in a large-scale culture. Four putative haemolysin genes (haemolysin, haemolysin III, haemolysin A, and α-haemolysin) were identified (EMQU_0190 and EMQU_0948, EMQU_0449, EMQU_0841, and EMQU_1982, respectively). We tested haemolysin activity and no changes to the blood agar in the vicinity of the colonies were observed (data not shown), suggesting that these putative haemolysin genes in QU 25 might be inactive or silent under the tested culture conditions.</p>                            <h3 scrollto-destination=6663292 id="6663292" class="section-title" data-legacy-id=s3h>3.8. Metabolic pathways for lactic acid fermentation</h3>
<p>QU 25 was previously reported to have two different pathways for xylose metabolism: the phosphoketolase (PK) pathway and the pentose phosphate (PP)/glycolytic pathway in low xylose concentrations.<sup><a reveal-id="DSU003C2" data-open="DSU003C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSU003C38" data-open="DSU003C38" class="link link-ref link-reveal xref-bibr">38</a></sup> When QU 25 was grown in high xylose concentrations, PK activity was not detected. However, higher transaldolase and transketolase activities were detected<sup><a reveal-id="DSU003C2" data-open="DSU003C2" class="link link-ref link-reveal xref-bibr">2</a></sup>, indicating that strain QU 25 would utilize the PP/glycolytic pathway, not the PK pathway, as the main pathway for lactic acid fermentation.</p><p>Genes for xylose metabolism in the QU 25 chromosome were located in a 22-kb region (positions 2 904 895–2 926 710 bp) in two gene clusters: one involved in the initial metabolism of xylose and uptake of pentose, and the other involved in the PP/glycolytic pathway and uptake of related sugars (Fig. <a reveal-id="DSU003F3" data-open="DSU003F3" class="link link-reveal link-table xref-fig">3</a>). The first gene cluster contained <em>xylR</em> (EMQU_2811; xylose repressor), <em>xylA</em> (EMQU_2810; xylose isomerase), <em>xynB</em> (EMQU_2809; xylan beta-1,4-xylosidase; additionally, there is another <em>xynB</em> gene [EMQU_2642] outside of this cluster), <em>xylB</em> (EMQU_2805; <span class="small-caps">d</span>-xylulose kinase), putative xylose transporter genes annotated as <span class="small-caps">l</span>-arabinose and <span class="small-caps">d</span>-ribose ABC transporter (EMQU_2806-2808), and a hypothetical protein (EMQU_2804), the N- and C-terminal regions of DNA mismatch repair protein (EMQU_2803 and EMQU_2802, respectively, which were thought as pseudogenes), ABC transporter ATP-binding protein (EMQU_2801), and its permease (EMQU_2800). Since pentose transporters have been shown to be promiscuous,<sup><a reveal-id="DSU003C39" data-open="DSU003C39" class="link link-ref link-reveal xref-bibr">39</a></sup><span class="small-caps">d</span>-xylose would likely be transported by these gene products in QU 25. </p>                        <a id="6663295" scrollto-destination="6663295"></a>
<div id="DSU003F3" data-id="DSU003F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/m_dsu00303.png?Expires=2147483647&Signature=yFsAC9DrR0xwYszhw6CHgrcwNmvbZh-G-I3xlK47yCipShTE~PdHpv429W6zq~n0CFaEFKU3NZjyYJQy93E2w0UPfoxdNdxuOjLyABDmZAKo6KwSg3g6i5lH24O7mbm8oNswGgk-zftyQR8k9hczVTsrBp0PHYKLYsSIhURCQjDgogxhY6X~ofrlx9ClPbpsLzFy~hR6wgC3eum8t1kXoRtSmcawb6b84XLUKJ8RgA6iNYjhDv-VkeT-JnRqiTl8OV1jXBSU1HnUTPBorX-Ca9oRxJviq1RTJLPGuq2IzOMbXOWwxYxHvGJ5kjIBvp0sq0DzQeimz~o30ZP7N5YKBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene clusters involved in the initial metabolism of xylose and the PP/glycolytic pathway in E. mundtii QU 25. Numbers below arrows indicate feature code (EMQU_XXXX)." path-from-xml="dsu00303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6663295/dsu00303.tiff" path-from-xml="dsu00303.tiff" target="_blank">View large</a><a section="6663295" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu00303.png?Expires=2147483647&Signature=lpYg0j8iqL7iulKz6cX4DOLmQnfIqbvE3rxC7Z~M5BXeR~DsfKz7bt6i8CsBcEa6TCbQQ2VlhHBCwxT3gEbZ5O-tYHKRjcgX4a2shfcLAeq37FS7WxX53PFbn3Zwq5rt3lHBZpC1wqE0kcZk8IYFun9tpGhLCzXT8LZjqZC3Y44OveSw2I9pwd8iTwm9HyKTgYbDvu8Lu3Fwxj8UNezepxvcmlGsl1zyt2-b8nYA-l-O0zkxJ4KlQQND9LRij0hh1-YQYOGYcu6IKekpzxjiEleO5NkSevRTIcr5LIbrKvZTFkbmOOWeziLOSl-VyAxcqucrjUAykj57Mv4KONI5zA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6663295&ar=445591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene clusters involved in the initial metabolism of xylose and the PP/glycolytic pathway in <em>E. mundtii</em> QU 25. Numbers below arrows indicate feature code (EMQU_XXXX).</p></div></div><div content-id="DSU003F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/m_dsu00303.png?Expires=2147483647&Signature=yFsAC9DrR0xwYszhw6CHgrcwNmvbZh-G-I3xlK47yCipShTE~PdHpv429W6zq~n0CFaEFKU3NZjyYJQy93E2w0UPfoxdNdxuOjLyABDmZAKo6KwSg3g6i5lH24O7mbm8oNswGgk-zftyQR8k9hczVTsrBp0PHYKLYsSIhURCQjDgogxhY6X~ofrlx9ClPbpsLzFy~hR6wgC3eum8t1kXoRtSmcawb6b84XLUKJ8RgA6iNYjhDv-VkeT-JnRqiTl8OV1jXBSU1HnUTPBorX-Ca9oRxJviq1RTJLPGuq2IzOMbXOWwxYxHvGJ5kjIBvp0sq0DzQeimz~o30ZP7N5YKBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene clusters involved in the initial metabolism of xylose and the PP/glycolytic pathway in E. mundtii QU 25. Numbers below arrows indicate feature code (EMQU_XXXX)." path-from-xml="dsu00303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6663295/dsu00303.tiff" path-from-xml="dsu00303.tiff" target="_blank">View large</a><a section="6663295" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu00303.png?Expires=2147483647&Signature=lpYg0j8iqL7iulKz6cX4DOLmQnfIqbvE3rxC7Z~M5BXeR~DsfKz7bt6i8CsBcEa6TCbQQ2VlhHBCwxT3gEbZ5O-tYHKRjcgX4a2shfcLAeq37FS7WxX53PFbn3Zwq5rt3lHBZpC1wqE0kcZk8IYFun9tpGhLCzXT8LZjqZC3Y44OveSw2I9pwd8iTwm9HyKTgYbDvu8Lu3Fwxj8UNezepxvcmlGsl1zyt2-b8nYA-l-O0zkxJ4KlQQND9LRij0hh1-YQYOGYcu6IKekpzxjiEleO5NkSevRTIcr5LIbrKvZTFkbmOOWeziLOSl-VyAxcqucrjUAykj57Mv4KONI5zA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6663295&ar=445591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene clusters involved in the initial metabolism of xylose and the PP/glycolytic pathway in <em>E. mundtii</em> QU 25. Numbers below arrows indicate feature code (EMQU_XXXX).</p></div></div><p>The second gene cluster contains genes for the PP/glycolytic pathway, including a transketolase (EMQU_2812) and a transaldolase (EMQU_2814). Furthermore, this cluster contains an allulose-6-phosphate 3-epimerase gene (EMQU_2813), genes for a fructose-like phosphotransferase system (EMQU_2815-2819), and a putative transcriptional regulator (EMQU_2820). Transketolase is a key enzyme in the PP/glycolytic pathway, and the QU 25 chromosome additionally harbours one transketolase gene (EMQU_1275). Since allulose-6-phosphate 3-epimerase catalyses the conversion of <span class="small-caps">d</span>-allulose-6-phosphate to <span class="small-caps">d</span>-fructose-6-phosphate, this enzyme and the fructose transporters may supply various ketoses to the PP/glycolytic pathway. For genes involved in the PK pathway, PK (EMQU_1837), acetate kinase (EMQU_2620), phosphotransacetylase (EMQU_2119), acetaldehyde dehydrogenase (EMQU_2205), and alcohol dehydrogenase (EMQU_1129, EMQU_1829, and EMQU_2109) were dispersed throughout the chromosome.</p><p>To get insights into the lactic acid fermentation characteristics of strain QU 25, we performed the comparative analysis on the presence of key enzymes related to lactic acid fermentation from xylose using KO (KEGG Orthology) gene assignment (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Two clinical isolates of <em>E. faecium</em> (DO and Aus0004) and <em>E. hirae</em> ATCC 9790 lack genes necessary for the metabolism of xylose (transaldolase, PK, xylulokinase, and xylose isomerase). Another clinical isolate, <em>E. faecalis</em> V583, lacks genes for transketolase, transaldolase, and PK. Thus, it is most unlikely that these strains metabolize xylose. <em>E. casseliflavus</em> EC20 has genes for PK, xylulokinase, and xylose isomerase, so that this strain can metabolize xylose by the PK pathway. It also has two complete genes for transketolase, but lacks transaldolase for the PP/glycolytic pathway. However, Kato <em>et al.</em><sup><a reveal-id="DSU003C40" data-open="DSU003C40" class="link link-ref link-reveal xref-bibr">40</a></sup> reported that <em>Lactococcus lactis</em> IO-1, which can utilize xylose, also lacks the gene for transaldolase and is presumed to have an alternative PP/glycolytic pathway. Therefore, EC20 might metabolize xylose also by the PP/glycolytic pathway. Two <em>E. mundtii</em> strains (QU 25 and ATCC 882) have two genes of full-length transketolase and all other genes necessary for the xylose metabolism by the two pathways. The results of our genomic analysis coincide with the description on their phenotype about xylose metabolism in Bergey's Manual (p. 599, Table 116).<sup><a reveal-id="DSU003C28" data-open="DSU003C28" class="link link-ref link-reveal xref-bibr">28</a></sup> From these results, remarkable genomic features related to lactic acid fermentation in QU 25 still remain unclear, and the analysis of transcriptional regulation of these genes could help its clarification.</p><p>QU 25 was able to metabolize a mixture of glucose and cellobiose simultaneously without apparent CCR.<sup><a reveal-id="DSU003C1" data-open="DSU003C1" class="link link-ref link-reveal xref-bibr">1</a></sup> In Gram-positive bacteria, the roles of catabolite control protein A (CcpA) and seryl-phosphorylated form of histidine-containing protein (P-Ser-HPr) in CCR have been well studied.<sup><a reveal-id="DSU003C41" data-open="DSU003C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Genes encoding CcpA (EMQU_1943), HPr (EMQU_0954), and HPr kinase/phosphorylase (EMQU_1951) were also found in the QU 25 genome. The mechanism by which CCR is prevented in the presence of glucose is still unknown.</p><p>Strain QU 25 shows a predominant production of <span class="small-caps">l</span>-(+)-lactate when grown at high concentrations of cellobiose and xylose, whereas <span class="small-caps">d</span>-lactic acid was not detected in the culture broth.<sup><a reveal-id="DSU003C1" data-open="DSU003C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU003C2" data-open="DSU003C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, two <span class="small-caps">l</span>-lactate dehydrogenase genes (<span class="small-caps">l</span>-LDH; EMQU_1380 and EMQU_2714) and one <span class="small-caps">d</span>-lactate dehydrogenase gene (<span class="small-caps">d</span>-LDH; EMQU_2453) were identified. Potentially, little or no <span class="small-caps">d</span>-LDH was expressed under these culture conditions. Lactate racemase, another gene involved in <span class="small-caps">d</span>-lactic acid formation,<sup><a reveal-id="DSU003C42" data-open="DSU003C42" class="link link-ref link-reveal xref-bibr">42</a></sup> was not found in the QU 25 genome.</p>                            <h2 scrollto-destination=6663300 id="6663300" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>We have highlighted the phylogenetic relationship of <em>E. mundtii</em> QU 25 with related enterococcal species and characterized mobile genetic elements, including multiple prophages and ISEs, plasmids, and genes for metabolic pathways for lactic acid fermentation in this strain. In addition, we have demonstrated that QU 25 has bacteriocin activity. The complete <em>E. mundtii</em> QU 25 genome sequence described here may be an important resource in the genetic engineering of recombinant strains for optimized production of lactic acid.</p>                            <h2 scrollto-destination=6663302 id="6663302" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6663304 id="6663304" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported by the MEXT-Supported Program for the Strategic Research Foundation at Private Universities, 2008-2012 (S0801025).</p>                            <h2 scrollto-destination=6663306 id="6663306" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Pacific Biosciences, Inc. (Menlo Park, CA, USA) and Tomy Digital Biology Co., Ltd (Tokyo, Japan) for PacBio sequencing and assembly, and Hitachi Solutions, Ltd (Tokyo, Japan) for optical mapping. We also thank Y. Kanesaki for helpful discussions.</p>                            <h2 scrollto-destination=6663308 id="6663308" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU003C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abdel-Rahman</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Tashiro</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zendo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sonomoto</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Isolation and characterisation of lactic acid bacterium for effective fermentation of cellobiose into optically pure homo <span class="small-caps">l</span>-(+)-lactic acid</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">1039</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterisation%20of%20lactic%20acid%20bacterium%20for%20effective%20fermentation%20of%20cellobiose%20into%20optically%20pure%20homo%20l-(%2B)-lactic%20acid&amp;author=M.A.%20Abdel-Rahman&amp;author=Y.%20Tashiro&amp;author=T.%20Zendo&amp;author=K.%20Shibata&amp;author=K.%20Sonomoto&amp;publication_year=2011&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=89&amp;pages=1039-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-010-2986-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-010-2986-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-010-2986-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21061005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abdel-Rahman</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Tashiro</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zendo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hanada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sonomoto</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Efficient homofermentative <span class="small-caps">l</span>-(+)-lactic acid production from xylose by a novel lactic acid bacterium, <em>Enterococcus mundtii</em> QU 25</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2011</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">1892</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20homofermentative%20l-(%2B)-lactic%20acid%20production%20from%20xylose%20by%20a%20novel%20lactic%20acid%20bacterium%2C%20Enterococcus%20mundtii%20QU%2025&amp;author=M.A.%20Abdel-Rahman&amp;author=Y.%20Tashiro&amp;author=T.%20Zendo&amp;author=K.%20Hanada&amp;author=K.%20Shibata&amp;author=K.%20Sonomoto&amp;publication_year=2011&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=77&amp;pages=1892-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02076-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02076-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02076-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21193678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abdel-Rahman</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Tashiro</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zendo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sonomoto</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Improved lactic acid productivity by an open repeated batch fermentation system using <em>Enterococcus mundtii</em> QU 25</div>, <div class="source">RSC Adv.</div> , <div class="year">2013</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">8437</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20lactic%20acid%20productivity%20by%20an%20open%20repeated%20batch%20fermentation%20system%20using%20Enterococcus%20mundtii%20QU%2025&amp;author=M.A.%20Abdel-Rahman&amp;author=Y.%20Tashiro&amp;author=T.%20Zendo&amp;author=K.%20Sonomoto&amp;publication_year=2013&amp;journal=RSC%20Adv.&amp;volume=3&amp;pages=8437-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/c3ra00078h" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fc3ra00078h" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fc3ra00078h"> </span></div> </div></div></div></div></div><div content-id="DSU003C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chin</div> <div class="given-names">C.-S.</div></div>,  <div class="name"><div class="surname">Alexander</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Marks</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data</div>, <div class="source">Nat. Methods</div> , <div class="year">2013</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">563</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonhybrid%2C%20finished%20microbial%20genome%20assemblies%20from%20long-read%20SMRT%20sequencing%20data&amp;author=C.-S.%20Chin&amp;author=D.H.%20Alexander&amp;author=P.%20Marks&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=563-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2474" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2474" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2474"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23644548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Magni</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Espeche</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Repizo</div> <div class="given-names">G.D.</div></div>, et al. </div><div class="article-title">Draft genome sequence of <em>Enterococcus mundtii</em> CRL1656</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> pg. <div class="fpage">550</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Enterococcus%20mundtii%20CRL1656&amp;author=C.%20Magni&amp;author=C.%20Espeche&amp;author=G.D.%20Repizo&amp;publication_year=2012&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=550" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.06415-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.06415-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.06415-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22207752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Machii</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zendo</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Chemically defined media and auxotrophy of the prolific l-lactic acid producer <em>Lactococcus lactis</em> IO-1</div>, <div class="source">J. Biosci. Bioeng.</div> , <div class="year">2013</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">481</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chemically%20defined%20media%20and%20auxotrophy%20of%20the%20prolific%20l-lactic%20acid%20producer%20Lactococcus%20lactis%20IO-1&amp;author=M.%20Machii&amp;author=S.%20Watanabe&amp;author=T.%20Zendo&amp;publication_year=2013&amp;journal=J.%20Biosci.%20Bioeng.&amp;volume=115&amp;pages=481-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jbiosc.2012.11.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jbiosc.2012.11.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jbiosc.2012.11.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23287501" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koren</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schatz</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Walenz</div> <div class="given-names">B.P.</div></div>, et al. </div><div class="article-title">Hybrid error correction and de novo assembly of single-molecule sequencing reads</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">693</div>-<div class="lpage">700</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hybrid%20error%20correction%20and%20de%20novo%20assembly%20of%20single-molecule%20sequencing%20reads&amp;author=S.%20Koren&amp;author=M.C.%20Schatz&amp;author=B.P.%20Walenz&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=693-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22750884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Koren</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Aggressive assembly of pyrosequencing reads with mates</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">2818</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aggressive%20assembly%20of%20pyrosequencing%20reads%20with%20mates&amp;author=J.R.%20Miller&amp;author=A.L.%20Delcher&amp;author=S.%20Koren&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=2818-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn548" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn548" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn548"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18952627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagarajan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Read</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Scaffolding and validation of bacterial genome assemblies using optical restriction maps</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1229</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20and%20validation%20of%20bacterial%20genome%20assemblies%20using%20optical%20restriction%20maps&amp;author=N.%20Nagarajan&amp;author=T.D.%20Read&amp;author=M.%20Pop&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=1229-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18356192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Taniguchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>. </div><div class="article-title">MetaGeneAnnotator: detecting species-specific patterns of ribosomal binding site for precise gene prediction in anonymous prokaryotic and phage genomes</div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">387</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MetaGeneAnnotator%3A%20detecting%20species-specific%20patterns%20of%20ribosomal%20binding%20site%20for%20precise%20gene%20prediction%20in%20anonymous%20prokaryotic%20and%20phage%20genomes&amp;author=H.%20Noguchi&amp;author=T.%20Taniguchi&amp;author=T.%20Itoh&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=387-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18940874" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohyama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Microbial genome annotation pipeline (MiGAP) for diverse users</div>, <div class="year">2009</div><div class="conf-name">The 20th International Conference on Genome Informatics (GIW2009) Poster and Software Demonstrations (Yokohama)</div> <div class="comment">S001-1-2</div></div></div></div></div><div content-id="DSU003C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheeler</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Barrett</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Benson</div> <div class="given-names">D.A.</div></div>, et al. </div><div class="article-title">Database resources of the National Center for Biotechnology Information</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D5</div>-<div class="lpage">D12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology%20Information&amp;author=D.L.%20Wheeler&amp;author=T.%20Barrett&amp;author=D.A.%20Benson&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D5-D12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl1031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl1031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl1031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17170002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zdobnov</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>. </div><div class="article-title">InterProScan—an integration platform for the signature-recognition methods in InterPro</div>, <div class="source">Bioinformatics</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">847</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%E2%80%94an%20integration%20platform%20for%20the%20signature-recognition%20methods%20in%20InterPro&amp;author=E.M.%20Zdobnov&amp;author=R.%20Apweiler&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=847-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.9.847" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.9.847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.9.847"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11590104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moriya</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Okuda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yoshizawa</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">KAAS: an automatic genome annotation and pathway reconstruction server</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W182</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KAAS%3A%20an%20automatic%20genome%20annotation%20and%20pathway%20reconstruction%20server&amp;author=Y.%20Moriya&amp;author=M.%20Itoh&amp;author=S.%20Okuda&amp;author=A.C.%20Yoshizawa&amp;author=M.%20Kanehisa&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W182-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Lynch</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Dennis</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Wishart</div> <div class="given-names">D.S.</div></div>. </div><div class="article-title">PHAST: a fast phage search tool</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">W347</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PHAST%3A%20a%20fast%20phage%20search%20tool&amp;author=Y.%20Zhou&amp;author=Y.%20Liang&amp;author=K.H.%20Lynch&amp;author=J.J.%20Dennis&amp;author=D.S.%20Wishart&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=W347-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21672955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varani</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Siguier</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gourbeyre</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Charneau</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Chandler</div> <div class="given-names">M.</div></div>. </div><div class="article-title">ISsaga is an ensemble of web-based methods for high throughput identification and semi-automatic annotation of insertion sequences in prokaryotic genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">R30</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ISsaga%20is%20an%20ensemble%20of%20web-based%20methods%20for%20high%20throughput%20identification%20and%20semi-automatic%20annotation%20of%20insertion%20sequences%20in%20prokaryotic%20genomes&amp;author=A.M.%20Varani&amp;author=P.%20Siguier&amp;author=E.%20Gourbeyre&amp;author=V.%20Charneau&amp;author=M.%20Chandler&amp;publication_year=2011&amp;journal=Genome%20Biol.&amp;volume=12&amp;pages=R30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-3-r30" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-3-r30" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-3-r30"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21443786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grissa</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Vergnaud</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pourcel</div> <div class="given-names">C.</div></div>. </div><div class="article-title">CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W52</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPRFinder%3A%20a%20web%20tool%20to%20identify%20clustered%20regularly%20interspaced%20short%20palindromic%20repeats&amp;author=I.%20Grissa&amp;author=G.%20Vergnaud&amp;author=C.%20Pourcel&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W52-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm360" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm360" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm360"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17537822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palmer</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Carniol</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Manson</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">High-quality draft genome sequences of 28 <em>Enterococcus</em> sp. isolates</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2010</div>, vol. <div class="volume">192</div> (pg. <div class="fpage">2469</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-quality%20draft%20genome%20sequences%20of%2028%20Enterococcus%20sp.%20isolates&amp;author=K.L.%20Palmer&amp;author=K.%20Carniol&amp;author=J.M.%20Manson&amp;publication_year=2010&amp;journal=J.%20Bacteriol.&amp;volume=192&amp;pages=2469-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00153-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00153-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00153-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20207762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paulsen</div> <div class="given-names">I.T.</div></div>,  <div class="name"><div class="surname">Banerjei</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">G.S.A.</div></div>, et al. </div><div class="article-title">Role of mobile DNA in the evolution of vancomycin-resistant <em>Enterococcus faecalis</em></div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">299</div> (pg. <div class="fpage">2071</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20mobile%20DNA%20in%20the%20evolution%20of%20vancomycin-resistant%20Enterococcus%20faecalis&amp;author=I.T.%20Paulsen&amp;author=L.%20Banerjei&amp;author=G.S.A.%20Myers&amp;publication_year=2003&amp;journal=Science&amp;volume=299&amp;pages=2071-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1080613" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1080613" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1080613"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12663927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lam</div> <div class="given-names">M.M.C.</div></div>,  <div class="name"><div class="surname">Seemann</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bulach</div> <div class="given-names">D.M.</div></div>, et al. </div><div class="article-title">Comparative analysis of the first complete <em>Enterococcus faecium</em> genome</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">2334</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20first%20complete%20Enterococcus%20faecium%20genome&amp;author=M.M.C.%20Lam&amp;author=T.%20Seemann&amp;author=D.M.%20Bulach&amp;publication_year=2012&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=2334-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00259-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00259-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00259-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22366422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qin</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Galloway-Peña</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Sillanpaa</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Enterococcus faecium</em> strain TX16 and comparative genomic analysis of <em>Enterococcus faecium</em> genomes</div>, <div class="source">BMC Microbiol.</div> , <div class="year">2012</div>, vol. <div class="volume">12</div> pg. <div class="fpage">135</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Enterococcus%20faecium%20strain%20TX16%20and%20comparative%20genomic%20analysis%20of%20Enterococcus%20faecium%20genomes&amp;author=X.%20Qin&amp;author=J.R.%20Galloway-Pe%C3%B1a&amp;author=J.%20Sillanpaa&amp;publication_year=2012&amp;journal=BMC%20Microbiol.&amp;volume=12&amp;pages=135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-12-135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-12-135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-12-135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22769602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gaechter</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wunderlin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schmidheini</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Solioz</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genome sequence of <em>Enterococcus hirae</em> (<em>Streptococcus faecalis</em>) ATCC 9790, a model organism for the study of ion transport, bioenergetics, and copper homeostasis</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">5126</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20Enterococcus%20hirae%20(Streptococcus%20faecalis)%20ATCC%209790%2C%20a%20model%20organism%20for%20the%20study%20of%20ion%20transport%2C%20bioenergetics%2C%20and%20copper%20homeostasis&amp;author=T.%20Gaechter&amp;author=C.%20Wunderlin&amp;author=T.%20Schmidheini&amp;author=M.%20Solioz&amp;publication_year=2012&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=5126-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01075-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01075-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01075-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22933757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikeda-Ohtsubo</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GenomeMatcher: a graphical user interface for DNA sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">376</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenomeMatcher%3A%20a%20graphical%20user%20interface%20for%20DNA%20sequence%20comparison&amp;author=Y.%20Ohtsubo&amp;author=W.%20Ikeda-Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18793444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zendo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Eungruttanagorn</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Fujioka</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Identification and production of a bacteriocin from <em>Enterococcus mundtii</em> QU 2 isolated from soybean</div>, <div class="source">J. Appl. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">1181</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20production%20of%20a%20bacteriocin%20from%20Enterococcus%20mundtii%20QU%202%20isolated%20from%20soybean&amp;author=T.%20Zendo&amp;author=N.%20Eungruttanagorn&amp;author=S.%20Fujioka&amp;publication_year=2005&amp;journal=J.%20Appl.%20Microbiol.&amp;volume=99&amp;pages=1181-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jam.2005.99.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjam.2005.99.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjam.2005.99.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16238749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ennahar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Asou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zendo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sonomoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ishizaki</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Biochemical and genetic evidence for production of enterocins A and B by <em>Enterococcus faecium</em> WHE 81</div>, <div class="source">Int. J. Food Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">291</div>-<div class="lpage">301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biochemical%20and%20genetic%20evidence%20for%20production%20of%20enterocins%20A%20and%20B%20by%20Enterococcus%20faecium%20WHE%2081&amp;author=S.%20Ennahar&amp;author=Y.%20Asou&amp;author=T.%20Zendo&amp;author=K.%20Sonomoto&amp;author=A.%20Ishizaki&amp;publication_year=2001&amp;journal=Int.%20J.%20Food%20Microbiol.&amp;volume=70&amp;pages=291-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-1605(01)00565-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-1605(01)00565-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-1605(01)00565-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11764194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shin</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Ahn</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">S.J.</div></div>, et al. </div><div class="article-title">Advantages of single-molecule real-time sequencing in high-GC content genomes</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e68824</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advantages%20of%20single-molecule%20real-time%20sequencing%20in%20high-GC%20content%20genomes&amp;author=S.C.%20Shin&amp;author=D.H.%20Ahn&amp;author=S.J.%20Kim&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e68824" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0068824" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0068824" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0068824"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23894349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Travers</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Chin</div> <div class="given-names">C.-S.</div></div>,  <div class="name"><div class="surname">Rank</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Eid</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">S.W.</div></div>. </div><div class="article-title">A flexible and efficient template format for circular consensus sequencing and SNP detection</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> pg. <div class="fpage">e159</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20flexible%20and%20efficient%20template%20format%20for%20circular%20consensus%20sequencing%20and%20SNP%20detection&amp;author=K.J.%20Travers&amp;author=C.-S.%20Chin&amp;author=D.R.%20Rank&amp;author=J.S.%20Eid&amp;author=S.W.%20Turner&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=e159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq543" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq543" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq543"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20571086" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Whitman</div> <div class="given-names">W.B.</div></div>,  <div class="name"><div class="surname">Goodfellow</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kämpfer</div> <div class="given-names">P.</div></div>, et al. </div>, <div class="source">Bergey's Manual of Systematic Bacteriology</div> , <div class="year">2012</div><div class="edition">2 edition</div><div class="publisher-loc">New York</div><div class="publisher-name">Springer Publishing Company</div></div></div></div></div><div content-id="DSU003C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hendrickson</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lawrence</div> <div class="given-names">J.G.</div></div>. </div><div class="article-title">Mutational bias suggests that replication termination occurs near the dif site, not at Ter Sites</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">42</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20bias%20suggests%20that%20replication%20termination%20occurs%20near%20the%20dif%20site%2C%20not%20at%20Ter%20Sites&amp;author=H.%20Hendrickson&amp;author=J.G.%20Lawrence&amp;publication_year=2007&amp;journal=Mol.%20Microbiol.&amp;volume=64&amp;pages=42-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2007.64.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2007.64.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2007.64.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17376071" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marraffini</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Sontheimer</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">CRISPR interference: RNA-directed adaptive immunity in bacteria and archaea</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">181</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR%20interference%3A%20RNA-directed%20adaptive%20immunity%20in%20bacteria%20and%20archaea&amp;author=L.A.%20Marraffini&amp;author=E.J.%20Sontheimer&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=11&amp;pages=181-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2749" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2749" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2749"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20125085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arias</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Panesso</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">K.V.</div></div>,  <div class="name"><div class="surname">Rice</div> <div class="given-names">L.B.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">B.E.</div></div>. </div><div class="article-title">Cotransfer of antibiotic resistance genes and a hylEfm-containing virulence plasmid in <em>Enterococcus faecium</em></div>, <div class="source">Antimicrob. Agents Chemother.</div> , <div class="year">2009</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">4240</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cotransfer%20of%20antibiotic%20resistance%20genes%20and%20a%20hylEfm-containing%20virulence%20plasmid%20in%20Enterococcus%20faecium&amp;author=C.A.%20Arias&amp;author=D.%20Panesso&amp;author=K.V.%20Singh&amp;author=L.B.%20Rice&amp;author=B.E.%20Murray&amp;publication_year=2009&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;volume=53&amp;pages=4240-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.00242-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.00242-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.00242-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19667280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Migura</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hasman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">L.B.</div></div>. </div><div class="article-title">Presence of pRI1: a small cryptic mobilizable plasmid isolated from <em>Enterococcus faecium</em> of human and animal origin</div>, <div class="source">Curr. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">95</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Presence%20of%20pRI1%3A%20a%20small%20cryptic%20mobilizable%20plasmid%20isolated%20from%20Enterococcus%20faecium%20of%20human%20and%20animal%20origin&amp;author=L.%20Garcia-Migura&amp;author=H.%20Hasman&amp;author=L.B.%20Jensen&amp;publication_year=2009&amp;journal=Curr.%20Microbiol.&amp;volume=58&amp;pages=95-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00284-008-9266-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00284-008-9266-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00284-008-9266-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18830745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Migura</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Liebana</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">L.B.</div></div>. </div><div class="article-title">Transposon characterization of vancomycin-resistant <em>Enterococcus faecium</em> (VREF) and dissemination of resistance associated with transferable plasmids</div>, <div class="source">J. Antimicrob. Chemother.</div> , <div class="year">2007</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">263</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposon%20characterization%20of%20vancomycin-resistant%20Enterococcus%20faecium%20(VREF)%20and%20dissemination%20of%20resistance%20associated%20with%20transferable%20plasmids&amp;author=L.%20Garcia-Migura&amp;author=E.%20Liebana&amp;author=L.B.%20Jensen&amp;publication_year=2007&amp;journal=J.%20Antimicrob.%20Chemother.&amp;volume=60&amp;pages=263-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jac/dkm186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjac%2Fdkm186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjac%2Fdkm186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17561499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hasman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kempf</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Chidaine</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Copper resistance in <em>Enterococcus faecium</em>, mediated by the tcrB gene, is selected by supplementation of pig feed with copper sulfate</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">5784</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copper%20resistance%20in%20Enterococcus%20faecium%2C%20mediated%20by%20the%20tcrB%20gene%2C%20is%20selected%20by%20supplementation%20of%20pig%20feed%20with%20copper%20sulfate&amp;author=H.%20Hasman&amp;author=I.%20Kempf&amp;author=B.%20Chidaine&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=5784-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02979-05" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02979-05" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02979-05"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16957194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moritz</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Hergenrother</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">Toxin-antitoxin systems are ubiquitous and plasmid-encoded in vancomycin-resistant enterococci</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">311</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toxin-antitoxin%20systems%20are%20ubiquitous%20and%20plasmid-encoded%20in%20vancomycin-resistant%20enterococci&amp;author=E.M.%20Moritz&amp;author=P.J.%20Hergenrother&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=311-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0601168104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0601168104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0601168104"> </span></div> </div></div></div></div></div><div content-id="DSU003C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pootoolal</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Neu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wright</div> <div class="given-names">G.D.</div></div>. </div><div class="article-title">Glycopeptide antibiotic resistance</div>, <div class="source">Annu. Rev. Pharmacol. Toxicol.</div> , <div class="year">2002</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">381</div>-<div class="lpage">408</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glycopeptide%20antibiotic%20resistance&amp;author=J.%20Pootoolal&amp;author=J.%20Neu&amp;author=G.D.%20Wright&amp;publication_year=2002&amp;journal=Annu.%20Rev.%20Pharmacol.%20Toxicol.&amp;volume=42&amp;pages=381-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.pharmtox.42.091601.142813" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.pharmtox.42.091601.142813" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.pharmtox.42.091601.142813"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11807177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawamoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shima</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Biochemical and genetic characterization of mundticin KS, an antilisterial peptide produced by <em>Enterococcus mundtii</em> NFRI 7393</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2002</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">3830</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biochemical%20and%20genetic%20characterization%20of%20mundticin%20KS%2C%20an%20antilisterial%20peptide%20produced%20by%20Enterococcus%20mundtii%20NFRI%207393&amp;author=S.%20Kawamoto&amp;author=J.%20Shima&amp;author=R.%20Sato&amp;publication_year=2002&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=68&amp;pages=3830-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.68.8.3830-3840.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.68.8.3830-3840.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.68.8.3830-3840.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12147478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abdel-Rahman</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Tashiro</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sonomoto</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Recent advances in lactic acid production by microbial fermentation processes</div>, <div class="source">Biotechnol. Adv.</div> , <div class="year">2013</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">877</div>-<div class="lpage">902</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20advances%20in%20lactic%20acid%20production%20by%20microbial%20fermentation%20processes&amp;author=M.A.%20Abdel-Rahman&amp;author=Y.%20Tashiro&amp;author=K.%20Sonomoto&amp;publication_year=2013&amp;journal=Biotechnol.%20Adv.&amp;volume=31&amp;pages=877-902" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2013.04.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2013.04.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2013.04.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23624242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Song</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Utilization of <span class="small-caps">d</span>-ribose through <span class="small-caps">d</span>-xylose transporter</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">1998</div>, vol. <div class="volume">163</div> (pg. <div class="fpage">255</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Utilization%20of%20d-ribose%20through%20d-xylose%20transporter&amp;author=S.%20Song&amp;author=C.%20Park&amp;publication_year=1998&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=163&amp;pages=255-61" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9673030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kato</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shiwa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Oshima</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Lactococcus lactis</em> IO-1, a lactic acid bacterium that utilizes xylose and produces high levels of <span class="small-caps">l</span>-lactic acid</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">2102</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Lactococcus%20lactis%20IO-1%2C%20a%20lactic%20acid%20bacterium%20that%20utilizes%20xylose%20and%20produces%20high%20levels%20of%20l-lactic%20acid&amp;author=H.%20Kato&amp;author=Y.%20Shiwa&amp;author=K.%20Oshima&amp;publication_year=2012&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=2102-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00074-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00074-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00074-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22461545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujita</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Carbon catabolite control of the metabolic network in <em>Bacillus subtilis</em></div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2009</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">245</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Carbon%20catabolite%20control%20of%20the%20metabolic%20network%20in%20Bacillus%20subtilis&amp;author=Y.%20Fujita&amp;publication_year=2009&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=73&amp;pages=245-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.80479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.80479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.80479"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19202299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU003C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU003C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goffin</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Deghorain</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mainardi</div> <div class="given-names">J.-L.</div></div>, et al. </div><div class="article-title">Lactate racemization as a rescue pathway for supplying <span class="small-caps">d</span>-lactate to the cell wall biosynthesis machinery in <em>Lactobacillus plantarum</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">6750</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lactate%20racemization%20as%20a%20rescue%20pathway%20for%20supplying%20d-lactate%20to%20the%20cell%20wall%20biosynthesis%20machinery%20in%20Lactobacillus%20plantarum&amp;author=P.%20Goffin&amp;author=M.%20Deghorain&amp;author=J.-L.%20Mainardi&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=6750-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.19.6750-6761.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.19.6750-6761.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.19.6750-6761.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16166538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=445591&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu003-dsu003supp_table5.doc?Expires=2147483647&amp;Signature=mgT~rhho7M8XdRdTjOz4S7O-TwGoHwvoB4FJwjoTfry~tdyQUUlVnfQB-0eKfGAEe~Q02Sg6BaF1RU41z~Wuv-Y1j3mysFtAgYLwKYYOvySKkfNrTtXI6SnB8XD079QEaOOhpNXXopE0NF-ZT-EWFYd9kmsFmWkA4vuiUwot2FqP7pVKU16BF0tS0xQHi~6hIr-XLvDEn8uAf~TK~B9VRucCAYPs0IJWhLlAqdvMdB6xiclZj~QXBA2vp8n6IrSggV6emd39uIbR4IfTIgw0VmW7O5rfPryBtz2xQvyT5P-EL9ZiGWRu2IyaobZ~EZ57F-dhBrQTgjMM~fceYkEtkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu003-dsu003supp_table4.doc?Expires=2147483647&amp;Signature=yOXOe1K-BcFw3ve1U85CP2Pq2-7M39tDkgAs2Z6auu3SEnzp3qo20ZaZMfF65iqO1oqHpz-nN3iJ1vqge1ev4mDGuYkQYz5ayjDcfDlKcp0KwD07C8WqxbJPcPvo82xD45mvGLmytIc~e4ghjYg2KtNRNHNEBLnv6YlqaI0Q1SAat8YUYoWQ7UN3RKHmBqFmcMMSYLyFESSmhXHJqx1oeOLVThij8-93nrwAKOgC-XpDZR7N5jub5INakPt6lw9-P0xuUkR23zWr9okbrpXDsNXUlUTcxZV9KF8ewgjf1UVZ~pPyko7M7oY-vWKXRbdtUf2PH2yg8US6hJT4qVNrkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu003-dsu003supp_table3.doc?Expires=2147483647&amp;Signature=BxAVYiIXPQgQYLOsx-nJVSX48gS2zyaTlnGPEuYnMraTEwQnoFjDZnjd3R4whi4kyij8~YDWwXnFDM1OqRRQp1HG0hVO8vCHS3mqpEI8b2TI-Vn4lfGDpK5Pb~X-xd9sGwKvykIfbaCrPSqPeckXMCSRBDOGSieiYsBfVuZ~BYrGJCKpT5TiRjYlcGnnLE2sjYJgGJ0eGyu2W3tSaLFKsYjBLid5dSr69ZLJnqYXtHbo53yY-ZBVb4wMS~w2nv1e0ZSK4rrb9To08JBeDcIDWbdfYli79NraK54Xq8erDjaRTiTW2hTNiYw9RqoKwA3UMoKuovJ2Bwik39jhJVAM-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu003-dsu003supp_table2.doc?Expires=2147483647&amp;Signature=lfslUiycVo2bAxTJl86~aW0uOim47j1RmXcDPO~zDEDmxpPbeDuqJbWsYZv0ylwMsUKudxlY45LNDwETTiSzauxSXJzYSACZfzSa2Mr5kyXuqaTMqgBShD-LHJg85hQJK7JQrsqyHqguMPoZ83mqPZ0Iy0OAnDeuGM2JT~xLT7vM6nTikcYAWEnPBBkImC2pm8sd9HxlNe8E8vvaopB3FyK~WI2GOaogzRTd-NG3sreQ-QqRrVccH1d8b4VhAvO2u0Wsb~nCoQlITnAJgG1OvHyQ2p~FA83J8imtj3jtc8NXFm3sOGZhagU3RkuXocOmA9sDAqueGfmPgj5lEGvNfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu003/2/dsu003-dsu003supp_table1.doc?Expires=2147483647&amp;Signature=In8MvubpU4Fy30~YP~1YZQKlAFuJUNHkZ3Zkm3IELnqIR5Pr8xN6jw4OyNHuoeWpJxtC07lrrD6w5u6OuAbsZeUWAsj924D2Nkya-iSy2gQDOnIPnErCURBYIXYEK7Kcn87MlW7Oq9ulwEZYQt6iyU8xNTXK~PkKofXN3LKvgitnhFLPmwC4OZeQ8z7Inq6Q3~U3-fgdLoW7E~CuVoKTrf6NG~vSAdstlcg0yaoeKrBYTUz~kADcr6MaAa5veMhAKvzLUd9-pfmAHxAAkjlSz7D~9Nqy6CdyEfVvAfOCoAK9ndWR2GqH3lz1Wc9Uinv1D0SZBKlIfEhTZazfrk9irw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">337</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu003"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="445591"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f4%2f369%2f445591">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344243900003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F4%2F369%2F445591" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/445591'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Complete Genome Sequence of Enterococcus mundtii QU 25, an Efficient l-(+)-Lactic Acid-Producing Bacterium" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu003");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f369%2f445591">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

