

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/3/181/335862","@type":"ScholarlyArticle","name":"Influence of the 3′-UTR-length of mKIAA cDNAs and their Sequence Features to the mRNA Expression Level in the Brain","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi001","keywords":["<italic>mKIAA</italic>","orthology","3′-UTR","expression","brain"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/3/181/335862","author":[{"name":"Okazaki, Noriko","affiliation":"1  Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818","@type":"Person"},{"name":"Imai, Kazuhide","affiliation":"2  Chiba Industry Advancement Center  2-6 Nakase, Mihama-ku, Chiba 261-7126","@type":"Person"},{"name":"Kikuno, Reiko F.","affiliation":"1  Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818","@type":"Person"},{"name":"Misawa, Kazuharu","affiliation":"2  Chiba Industry Advancement Center  2-6 Nakase, Mihama-ku, Chiba 261-7126","@type":"Person"},{"name":"Kawai, Makoto","affiliation":"2  Chiba Industry Advancement Center  2-6 Nakase, Mihama-ku, Chiba 261-7126","@type":"Person"},{"name":"Inamoto, Susumu","affiliation":"2  Chiba Industry Advancement Center  2-6 Nakase, Mihama-ku, Chiba 261-7126    3  Institute of Research and Innovation  1201 Takada, Kashiwa, Chiba 277-0861","@type":"Person"},{"name":"Ohara, Reiko","affiliation":"1  Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818","@type":"Person"},{"name":"Nagase, Takahiro","affiliation":"1  Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818","@type":"Person"},{"name":"Ohara, Osamu","affiliation":"1  Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818    4  RIKEN Yokohama Institute  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama City, Kanagawa, 230-0045, Japan","@type":"Person"},{"name":"Koga, Hisashi","affiliation":"1  Kazusa DNA Research Institute  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818    2  Chiba Industry Advancement Center  2-6 Nakase, Mihama-ku, Chiba 261-7126","@type":"Person"}],"description":"Abstract.  We have previously described the sequence features of ∼1500 mouse KIAA (mKIAA) genes in comparison with those of human KIAA genes (Okazaki, N., Kikun","pageStart":"181","pageEnd":"189","thumbnailURL":"https://academic.oup.com/UI/app/img/generic-cover.png","headline":"Influence of the 3′-UTR-length of mKIAA cDNAs and their Sequence Features to the mRNA Expression Level in the Brain","image":"https://academic.oup.com/UI/app/img/generic-cover.png"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Influence of the 3′-UTR-length of mKIAA cDNAs and their Sequence Features to the mRNA Expression Level in the Brain" />
    <meta property="og:description" content="Abstract.  We have previously described the sequence features of ∼1500 mouse KIAA (mKIAA) genes in comparison with those of human KIAA genes (Okazaki, N., Kikun" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/3/181/335862" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Okazaki, Noriko" /><meta name="citation_author_institution" content="1Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818" /><meta name="citation_author" content="Imai, Kazuhide" /><meta name="citation_author_institution" content="2Chiba Industry Advancement Center2-6 Nakase, Mihama-ku, Chiba 261-7126" /><meta name="citation_author" content="Kikuno, Reiko F." /><meta name="citation_author_institution" content="1Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818" /><meta name="citation_author" content="Misawa, Kazuharu" /><meta name="citation_author_institution" content="2Chiba Industry Advancement Center2-6 Nakase, Mihama-ku, Chiba 261-7126" /><meta name="citation_author" content="Kawai, Makoto" /><meta name="citation_author_institution" content="2Chiba Industry Advancement Center2-6 Nakase, Mihama-ku, Chiba 261-7126" /><meta name="citation_author" content="Inamoto, Susumu" /><meta name="citation_author_institution" content="2Chiba Industry Advancement Center2-6 Nakase, Mihama-ku, Chiba 261-7126" /><meta name="citation_author_institution" content="3Institute of Research and Innovation1201 Takada, Kashiwa, Chiba 277-0861" /><meta name="citation_author" content="Ohara, Reiko" /><meta name="citation_author_institution" content="1Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818" /><meta name="citation_author" content="Nagase, Takahiro" /><meta name="citation_author_institution" content="1Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818" /><meta name="citation_author" content="Ohara, Osamu" /><meta name="citation_author_institution" content="1Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818" /><meta name="citation_author_institution" content="4RIKEN Yokohama Institute1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama City, Kanagawa, 230-0045, Japan" /><meta name="citation_author" content="Koga, Hisashi" /><meta name="citation_author_institution" content="1Kazusa DNA Research Institute2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818" /><meta name="citation_author_institution" content="2Chiba Industry Advancement Center2-6 Nakase, Mihama-ku, Chiba 261-7126" /><meta name="citation_title" content="Influence of the 3′-UTR-length of mKIAA cDNAs and their Sequence Features to the mRNA Expression Level in the Brain" /><meta name="citation_firstpage" content="181" /><meta name="citation_lastpage" content="189" /><meta name="citation_doi" content="10.1093/dnares/dsi001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Prediction of the coding sequences of unidentified human genes. I. The coding sequences of 40 new genes (KIAA0001-KIAA0040) deduced by analysis of randomly sampled cDNA clones from human immature myeloid cell line KG-1; DNA Res.;  citation_year=1994;  citation_volume=1;  citation_pages=27-35; " /><meta name="citation_reference" content="citation_title=Prediction of the coding sequences of mouse homologues of KIAA gene: I. The complete nucleotide sequences of 100 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries; DNA Res.;  citation_year=2002;  citation_volume=9;  citation_pages=179-188; " /><meta name="citation_reference" content="citation_title=Prediction of the coding sequences of mouse homologues of KIAA gene: II. The complete nucleotide sequences of 400 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries; DNA Res.;  citation_year=2003;  citation_volume=10;  citation_pages=35-48; " /><meta name="citation_reference" content="citation_title=Prediction of the coding sequences of mouse homologues of KIAA gene: III. the complete nucleotide sequences of 500 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries; DNA Res.;  citation_year=2003;  citation_volume=10;  citation_pages=167-180; " /><meta name="citation_reference" content="citation_title=Prediction of the coding sequences of mouse homologues of KIAA gene: IV. The complete nucleotide sequences of 500 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=205-218; " /><meta name="citation_reference" content="citation_title=A comprehensive approach for establishment of the platform to analyze functions of KIAA proteins II: public release of inaugural version of InGaP database containing gene/protein expression profiles for 127 mouse KIAA genes/proteins; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=293-304; " /><meta name="citation_reference" content="citation_title=Cytoplasmic regulation of mRNA function: the importance of the 3′ untranslated region; Cell;  citation_year=1993;  citation_volume=74;  citation_pages=9-14; " /><meta name="citation_reference" content="citation_title=Translational regulation in development; Cell;  citation_year=1995;  citation_volume=81;  citation_pages=171-178; " /><meta name="citation_reference" content="citation_title=Translational regulation: versatile mechanisms for metabolic and developmental control; Curr. Opin. Cell Biol.;  citation_year=1995;  citation_volume=7;  citation_pages=393-398; " /><meta name="citation_reference" content="citation_title=Life and death in the cytoplasm: messages from the 3′ end; Curr. Opin. Genet. Dev.;  citation_year=1997;  citation_volume=7;  citation_pages=220-232; " /><meta name="citation_reference" content="citation_title=RNA-binding proteins as regulators of gene expression; Curr. Opin. Genet. Dev.;  citation_year=1997;  citation_volume=7;  citation_pages=345-353; " /><meta name="citation_reference" content="citation_title=Post-transcriptional gene regulatory mechanisms in eukaryotes: an overview; J. Endocrinol.;  citation_year=1998;  citation_volume=157;  citation_pages=361-371; " /><meta name="citation_reference" content="citation_title=RNA-protein interactions in the control of stability and localization of messenger RNA (review); Int. J. Mol. Med.;  citation_year=2000;  citation_volume=5;  citation_pages=111-123; " /><meta name="citation_reference" content="citation_title=Regulation of alpha1(I) collagen messenger RNA decay by interactions with alphaCP at the 3′-untranslated region; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=23822-23829; " /><meta name="citation_reference" content="citation_title=Using the Map Viewer to explore genomes; citation_text=The NCBI Handbook [Internet]. National Library of Medicine (US), National Center for Biotechnology Information, Bethesda (MD);  citation_year=2002; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-410; " /><meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions; Mol. Biol. Evol.;  citation_year=1986;  citation_volume=3;  citation_pages=418-426; " /><meta name="citation_reference" content="citation_title=Human immunodeficiency virus type 1 genetic evolution in children with different rates of development of disease; J. Virol.;  citation_year=1997;  citation_volume=71;  citation_pages=663-677; " /><meta name="citation_reference" content="citation_title=HIV signature and sequence variation analysis; citation_publisher=Computational Analysis of HIV Molecular Sequences, Ch. 4. Kluwer Academic Publishers, Dordrecht, The Netherlands;  citation_year=2000;  citation_pages=55-72; " /><meta name="citation_reference" content="citation_title=Assessing sequence comparison methods with reliable structurally identified distant evolutionary relationships; Proc. Natl Acad. Sci. U.S.A.;  citation_year=1998;  citation_volume=95;  citation_pages=6073-6078; " /><meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=28-33; " /><meta name="citation_reference" content="citation_title=A molecular timescale for vertebrate evolution; Nature;  citation_year=1998;  citation_volume=392;  citation_pages=917-920; " /><meta name="citation_reference" content="citation_title=Evolutionary parameters of the transcribed mammalian genome: an analysis of 2820 orthologous rodent and human sequences; Proc. Natl. Acad. Sci. U.S.A.;  citation_year=1998;  citation_volume=95;  citation_pages=9407-9412; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-425; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of mRNA lengths in Saccharomyces cerevisiae; Genome Biol.;  citation_year=2003;  citation_volume=5;  citation_pages=R2" /><meta name="citation_reference" content="citation_title=Translational control by the 3′-UTR: the ends specify the means; Trends Biochem. Sci.;  citation_year=2003;  citation_volume=28;  citation_pages=91-98; " /><meta name="citation_reference" content="citation_title=Regulatory functions of 3′UTRs; Biochem. Biophys. Res. Commun.;  citation_year=2001;  citation_volume=288;  citation_pages=291-295; " /><meta name="citation_reference" content="citation_title=ARED: human AU-rich element-containing mRNA database reveals an unexpectedly diverse functional repertoire of encoded proteins; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=246-254; " /><meta name="citation_reference" content="citation_title=Suppression of 15-lipoxygenase synthesis by hnRNP E1 is dependent on repetitive nature of LOX mRNA 3′-UTR control element DICE; J. Mol. Biol.;  citation_year=2002;  citation_volume=315;  citation_pages=965-974; " /><meta name="citation_reference" content="citation_title=The 3′ untranslated region of the membrane-bound IL-1R accessory protein mRNA confers tissue-specific destabilization; J. Immunol.;  citation_year=2004;  citation_volume=173;  citation_pages=6248-6258; " /><meta name="citation_reference" content="citation_title=Phylogenetically conserved binding of specific KH domain proteins to the 3′ untranslated region of the vertebrate middle neurofilament mRNA; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=49680-49688; " /><meta name="citation_reference" content="citation_title=A conserved family of elav-like genes in vertebrates; Proc. Natl. Acad. Sci. U.S.A.;  citation_year=1995;  citation_volume=92;  citation_pages=4557-4561; " /><meta name="citation_reference" content="citation_title=Musashi: a translational regulator of cell fate; J. Cell Sci.;  citation_year=2002;  citation_volume=115;  citation_pages=1355-1359; " /><meta name="citation_reference" content="citation_title=Cell-specific RNA-binding proteins in human disease; Trends Cardiovasc. Med.;  citation_year=2003;  citation_volume=13;  citation_pages=188-195; " /><meta name="citation_reference" content="citation_title=Molecular cloning and characterization of a new neuron-specific homologue of rat polypyrimidine tract binding protein; J. Biochem. (Tokyo);  citation_year=2000;  citation_volume=128;  citation_pages=811-821; " /><meta name="citation_reference" content="citation_title=cDNA cloning and characterization of Drb1, a new member of RRM-type neural RNA-binding protein; Biochem. Biophys. Res. Commun.;  citation_year=2002;  citation_volume=297;  citation_pages=96-104; " /><meta name="citation_reference" content="citation_title=Detection of new genes in a bacterial genome using Markov models for three gene classes; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=3554-3562; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of a novel transcriptional repressor of the nicotinic acetylcholine receptor delta-subunit gene; J. Biol. Chem.;  citation_year=1996;  citation_volume=271;  citation_pages=7203-7211; " /><meta name="citation_reference" content="citation_title=Molecular cloning of Rhombex-40 a transmembrane protein from the ventral medullary surface of the rat brain by differential display; Life Sci.;  citation_year=2000;  citation_volume=66;  citation_pages=2183-2191; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/3/181/961138/dsi001.pdf" /><meta name="description" content="Abstract.  We have previously described the sequence features of ∼1500 mouse KIAA (mKIAA) genes in comparison with those of human KIAA genes (Okazaki, N., Kikun" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/3/181/335862" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/3/181/335862" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/3/181/335862");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi001';
    </script>
<title>
	Influence of the 3′-UTR-length of mKIAA cDNAs and their Sequence Features to the mRNA Expression Level in the Brain | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./335862" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdhoPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPa2F6YWtpLCBOb3Jpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkltYWksIEthenVoaWRlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkNoaWJhIEluZHVzdHJ5IEFkdmFuY2VtZW50IENlbnRlcjItNiBOYWthc2UsIE1paGFtYS1rdSwgQ2hpYmEgMjYxLTcxMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaWt1bm8sIFJlaWtvIEYuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXNhd2EsIEthenVoYXJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkNoaWJhIEluZHVzdHJ5IEFkdmFuY2VtZW50IENlbnRlcjItNiBOYWthc2UsIE1paGFtYS1rdSwgQ2hpYmEgMjYxLTcxMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhaSwgTWFrb3RvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkNoaWJhIEluZHVzdHJ5IEFkdmFuY2VtZW50IENlbnRlcjItNiBOYWthc2UsIE1paGFtYS1rdSwgQ2hpYmEgMjYxLTcxMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJbmFtb3RvLCBTdXN1bXUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIyQ2hpYmEgSW5kdXN0cnkgQWR2YW5jZW1lbnQgQ2VudGVyMi02IE5ha2FzZSwgTWloYW1hLWt1LCBDaGliYSAyNjEtNzEyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjNJbnN0aXR1dGUgb2YgUmVzZWFyY2ggYW5kIElubm92YXRpb24xMjAxIFRha2FkYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTA4NjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaGFyYSwgUmVpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5hZ2FzZSwgVGFrYWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9oYXJhLCBPc2FtdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZTItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iNFJJS0VOIFlva29oYW1hIEluc3RpdHV0ZTEtNy0yMiBTdWVoaXJvLWNobywgVHN1cnVtaS1rdSwgWW9rb2hhbWEgQ2l0eSwgS2FuYWdhd2EsIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvZ2EsIEhpc2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJDaGliYSBJbmR1c3RyeSBBZHZhbmNlbWVudCBDZW50ZXIyLTYgTmFrYXNlLCBNaWhhbWEta3UsIENoaWJhIDI2MS03MTI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJJbmZsdWVuY2Ugb2YgdGhlIDPigLItVVRSLWxlbmd0aCBvZiBtS0lBQSBjRE5BcyBhbmQgdGhlaXIgU2VxdWVuY2UgRmVhdHVyZXMgdG8gdGhlIG1STkEgRXhwcmVzc2lvbiBMZXZlbCBpbiB0aGUgQnJhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE4OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzaTAwMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA1LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGlvbiBvZiB0aGUgY29kaW5nIHNlcXVlbmNlcyBvZiB1bmlkZW50aWZpZWQgaHVtYW4gZ2VuZXMuIEkuIFRoZSBjb2Rpbmcgc2VxdWVuY2VzIG9mIDQwIG5ldyBnZW5lcyAoS0lBQTAwMDEtS0lBQTAwNDApIGRlZHVjZWQgYnkgYW5hbHlzaXMgb2YgcmFuZG9tbHkgc2FtcGxlZCBjRE5BIGNsb25lcyBmcm9tIGh1bWFuIGltbWF0dXJlIG15ZWxvaWQgY2VsbCBsaW5lIEtHLTE7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0yNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGlvbiBvZiB0aGUgY29kaW5nIHNlcXVlbmNlcyBvZiBtb3VzZSBob21vbG9ndWVzIG9mIEtJQUEgZ2VuZTogSS4gVGhlIGNvbXBsZXRlIG51Y2xlb3RpZGUgc2VxdWVuY2VzIG9mIDEwMCBtb3VzZSBLSUFBLWhvbW9sb2dvdXMgY0ROQXMgaWRlbnRpZmllZCBieSBzY3JlZW5pbmcgb2YgdGVybWluYWwgc2VxdWVuY2VzIG9mIGNETkEgY2xvbmVzIHJhbmRvbWx5IHNhbXBsZWQgZnJvbSBzaXplLWZyYWN0aW9uYXRlZCBsaWJyYXJpZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xNzktMTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHRoZSBjb2Rpbmcgc2VxdWVuY2VzIG9mIG1vdXNlIGhvbW9sb2d1ZXMgb2YgS0lBQSBnZW5lOiBJSS4gVGhlIGNvbXBsZXRlIG51Y2xlb3RpZGUgc2VxdWVuY2VzIG9mIDQwMCBtb3VzZSBLSUFBLWhvbW9sb2dvdXMgY0ROQXMgaWRlbnRpZmllZCBieSBzY3JlZW5pbmcgb2YgdGVybWluYWwgc2VxdWVuY2VzIG9mIGNETkEgY2xvbmVzIHJhbmRvbWx5IHNhbXBsZWQgZnJvbSBzaXplLWZyYWN0aW9uYXRlZCBsaWJyYXJpZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MzUtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2YgdGhlIGNvZGluZyBzZXF1ZW5jZXMgb2YgbW91c2UgaG9tb2xvZ3VlcyBvZiBLSUFBIGdlbmU6IElJSS4gdGhlIGNvbXBsZXRlIG51Y2xlb3RpZGUgc2VxdWVuY2VzIG9mIDUwMCBtb3VzZSBLSUFBLWhvbW9sb2dvdXMgY0ROQXMgaWRlbnRpZmllZCBieSBzY3JlZW5pbmcgb2YgdGVybWluYWwgc2VxdWVuY2VzIG9mIGNETkEgY2xvbmVzIHJhbmRvbWx5IHNhbXBsZWQgZnJvbSBzaXplLWZyYWN0aW9uYXRlZCBsaWJyYXJpZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTY3LTE4MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGlvbiBvZiB0aGUgY29kaW5nIHNlcXVlbmNlcyBvZiBtb3VzZSBob21vbG9ndWVzIG9mIEtJQUEgZ2VuZTogSVYuIFRoZSBjb21wbGV0ZSBudWNsZW90aWRlIHNlcXVlbmNlcyBvZiA1MDAgbW91c2UgS0lBQS1ob21vbG9nb3VzIGNETkFzIGlkZW50aWZpZWQgYnkgc2NyZWVuaW5nIG9mIHRlcm1pbmFsIHNlcXVlbmNlcyBvZiBjRE5BIGNsb25lcyByYW5kb21seSBzYW1wbGVkIGZyb20gc2l6ZS1mcmFjdGlvbmF0ZWQgbGlicmFyaWVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTIwNS0yMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSBhcHByb2FjaCBmb3IgZXN0YWJsaXNobWVudCBvZiB0aGUgcGxhdGZvcm0gdG8gYW5hbHl6ZSBmdW5jdGlvbnMgb2YgS0lBQSBwcm90ZWlucyBJSTogcHVibGljIHJlbGVhc2Ugb2YgaW5hdWd1cmFsIHZlcnNpb24gb2YgSW5HYVAgZGF0YWJhc2UgY29udGFpbmluZyBnZW5lL3Byb3RlaW4gZXhwcmVzc2lvbiBwcm9maWxlcyBmb3IgMTI3IG1vdXNlIEtJQUEgZ2VuZXMvcHJvdGVpbnM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjkzLTMwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b3BsYXNtaWMgcmVndWxhdGlvbiBvZiBtUk5BIGZ1bmN0aW9uOiB0aGUgaW1wb3J0YW5jZSBvZiB0aGUgM+KAsiB1bnRyYW5zbGF0ZWQgcmVnaW9uOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9OS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCByZWd1bGF0aW9uIGluIGRldmVsb3BtZW50OyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9MTcxLTE3ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCByZWd1bGF0aW9uOiB2ZXJzYXRpbGUgbWVjaGFuaXNtcyBmb3IgbWV0YWJvbGljIGFuZCBkZXZlbG9wbWVudGFsIGNvbnRyb2w7IEN1cnIuIE9waW4uIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTM5My0zOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpZmUgYW5kIGRlYXRoIGluIHRoZSBjeXRvcGxhc206IG1lc3NhZ2VzIGZyb20gdGhlIDPigLIgZW5kOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjIwLTIzMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLWJpbmRpbmcgcHJvdGVpbnMgYXMgcmVndWxhdG9ycyBvZiBnZW5lIGV4cHJlc3Npb247IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zNDUtMzUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3N0LXRyYW5zY3JpcHRpb25hbCBnZW5lIHJlZ3VsYXRvcnkgbWVjaGFuaXNtcyBpbiBldWthcnlvdGVzOiBhbiBvdmVydmlldzsgSi4gRW5kb2NyaW5vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTU3OyAgY2l0YXRpb25fcGFnZXM9MzYxLTM3MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLXByb3RlaW4gaW50ZXJhY3Rpb25zIGluIHRoZSBjb250cm9sIG9mIHN0YWJpbGl0eSBhbmQgbG9jYWxpemF0aW9uIG9mIG1lc3NlbmdlciBSTkEgKHJldmlldyk7IEludC4gSi4gTW9sLiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIGFscGhhMShJKSBjb2xsYWdlbiBtZXNzZW5nZXIgUk5BIGRlY2F5IGJ5IGludGVyYWN0aW9ucyB3aXRoIGFscGhhQ1AgYXQgdGhlIDPigLItdW50cmFuc2xhdGVkIHJlZ2lvbjsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9MjM4MjItMjM4Mjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIHRoZSBNYXAgVmlld2VyIHRvIGV4cGxvcmUgZ2Vub21lczsgY2l0YXRpb25fdGV4dD1UaGUgTkNCSSBIYW5kYm9vayBbSW50ZXJuZXRdLiBOYXRpb25hbCBMaWJyYXJ5IG9mIE1lZGljaW5lIChVUyksIE5hdGlvbmFsIENlbnRlciBmb3IgQmlvdGVjaG5vbG9neSBJbmZvcm1hdGlvbiwgQmV0aGVzZGEgKE1EKTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy00MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBtZXRob2RzIGZvciBlc3RpbWF0aW5nIHRoZSBudW1iZXJzIG9mIHN5bm9ueW1vdXMgYW5kIG5vbnN5bm9ueW1vdXMgbnVjbGVvdGlkZSBzdWJzdGl0dXRpb25zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz00MTgtNDI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiBpbW11bm9kZWZpY2llbmN5IHZpcnVzIHR5cGUgMSBnZW5ldGljIGV2b2x1dGlvbiBpbiBjaGlsZHJlbiB3aXRoIGRpZmZlcmVudCByYXRlcyBvZiBkZXZlbG9wbWVudCBvZiBkaXNlYXNlOyBKLiBWaXJvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz02NjMtNjc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ISVYgc2lnbmF0dXJlIGFuZCBzZXF1ZW5jZSB2YXJpYXRpb24gYW5hbHlzaXM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1Db21wdXRhdGlvbmFsIEFuYWx5c2lzIG9mIEhJViBNb2xlY3VsYXIgU2VxdWVuY2VzLCBDaC4gNC4gS2x1d2VyIEFjYWRlbWljIFB1Ymxpc2hlcnMsIERvcmRyZWNodCwgVGhlIE5ldGhlcmxhbmRzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fcGFnZXM9NTUtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc2luZyBzZXF1ZW5jZSBjb21wYXJpc29uIG1ldGhvZHMgd2l0aCByZWxpYWJsZSBzdHJ1Y3R1cmFsbHkgaWRlbnRpZmllZCBkaXN0YW50IGV2b2x1dGlvbmFyeSByZWxhdGlvbnNoaXBzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVS5TLkEuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9NjA3My02MDc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EYXRhYmFzZSByZXNvdXJjZXMgb2YgdGhlIE5hdGlvbmFsIENlbnRlciBmb3IgQmlvdGVjaG5vbG9neTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjgtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbW9sZWN1bGFyIHRpbWVzY2FsZSBmb3IgdmVydGVicmF0ZSBldm9sdXRpb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zOTI7ICBjaXRhdGlvbl9wYWdlcz05MTctOTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgcGFyYW1ldGVycyBvZiB0aGUgdHJhbnNjcmliZWQgbWFtbWFsaWFuIGdlbm9tZTogYW4gYW5hbHlzaXMgb2YgMjgyMCBvcnRob2xvZ291cyByb2RlbnQgYW5kIGh1bWFuIHNlcXVlbmNlczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLlMuQS47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz05NDA3LTk0MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBuZWlnaGJvci1qb2luaW5nIG1ldGhvZDogYSBuZXcgbWV0aG9kIGZvciByZWNvbnN0cnVjdGluZyBwaHlsb2dlbmV0aWMgdHJlZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi00MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIG1STkEgbGVuZ3RocyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9UjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbmFsIGNvbnRyb2wgYnkgdGhlIDPigLItVVRSOiB0aGUgZW5kcyBzcGVjaWZ5IHRoZSBtZWFuczsgVHJlbmRzIEJpb2NoZW0uIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz05MS05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdG9yeSBmdW5jdGlvbnMgb2YgM+KAslVUUnM7IEJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODg7ICBjaXRhdGlvbl9wYWdlcz0yOTEtMjk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUkVEOiBodW1hbiBBVS1yaWNoIGVsZW1lbnQtY29udGFpbmluZyBtUk5BIGRhdGFiYXNlIHJldmVhbHMgYW4gdW5leHBlY3RlZGx5IGRpdmVyc2UgZnVuY3Rpb25hbCByZXBlcnRvaXJlIG9mIGVuY29kZWQgcHJvdGVpbnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI0Ni0yNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cHByZXNzaW9uIG9mIDE1LWxpcG94eWdlbmFzZSBzeW50aGVzaXMgYnkgaG5STlAgRTEgaXMgZGVwZW5kZW50IG9uIHJlcGV0aXRpdmUgbmF0dXJlIG9mIExPWCBtUk5BIDPigLItVVRSIGNvbnRyb2wgZWxlbWVudCBESUNFOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMxNTsgIGNpdGF0aW9uX3BhZ2VzPTk2NS05NzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSAz4oCyIHVudHJhbnNsYXRlZCByZWdpb24gb2YgdGhlIG1lbWJyYW5lLWJvdW5kIElMLTFSIGFjY2Vzc29yeSBwcm90ZWluIG1STkEgY29uZmVycyB0aXNzdWUtc3BlY2lmaWMgZGVzdGFiaWxpemF0aW9uOyBKLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzM7ICBjaXRhdGlvbl9wYWdlcz02MjQ4LTYyNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpY2FsbHkgY29uc2VydmVkIGJpbmRpbmcgb2Ygc3BlY2lmaWMgS0ggZG9tYWluIHByb3RlaW5zIHRvIHRoZSAz4oCyIHVudHJhbnNsYXRlZCByZWdpb24gb2YgdGhlIHZlcnRlYnJhdGUgbWlkZGxlIG5ldXJvZmlsYW1lbnQgbVJOQTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9NDk2ODAtNDk2ODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29uc2VydmVkIGZhbWlseSBvZiBlbGF2LWxpa2UgZ2VuZXMgaW4gdmVydGVicmF0ZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkEuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9NDU1Ny00NTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXNhc2hpOiBhIHRyYW5zbGF0aW9uYWwgcmVndWxhdG9yIG9mIGNlbGwgZmF0ZTsgSi4gQ2VsbCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTEzNTUtMTM1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbC1zcGVjaWZpYyBSTkEtYmluZGluZyBwcm90ZWlucyBpbiBodW1hbiBkaXNlYXNlOyBUcmVuZHMgQ2FyZGlvdmFzYy4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTE4OC0xOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgbmV3IG5ldXJvbi1zcGVjaWZpYyBob21vbG9ndWUgb2YgcmF0IHBvbHlweXJpbWlkaW5lIHRyYWN0IGJpbmRpbmcgcHJvdGVpbjsgSi4gQmlvY2hlbS4gKFRva3lvKTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz04MTEtODIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1jRE5BIGNsb25pbmcgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgRHJiMSwgYSBuZXcgbWVtYmVyIG9mIFJSTS10eXBlIG5ldXJhbCBSTkEtYmluZGluZyBwcm90ZWluOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk3OyAgY2l0YXRpb25fcGFnZXM9OTYtMTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2YgbmV3IGdlbmVzIGluIGEgYmFjdGVyaWFsIGdlbm9tZSB1c2luZyBNYXJrb3YgbW9kZWxzIGZvciB0aHJlZSBnZW5lIGNsYXNzZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTM1NTQtMzU2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBhIG5vdmVsIHRyYW5zY3JpcHRpb25hbCByZXByZXNzb3Igb2YgdGhlIG5pY290aW5pYyBhY2V0eWxjaG9saW5lIHJlY2VwdG9yIGRlbHRhLXN1YnVuaXQgZ2VuZTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjcxOyAgY2l0YXRpb25fcGFnZXM9NzIwMy03MjExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2xvbmluZyBvZiBSaG9tYmV4LTQwIGEgdHJhbnNtZW1icmFuZSBwcm90ZWluIGZyb20gdGhlIHZlbnRyYWwgbWVkdWxsYXJ5IHN1cmZhY2Ugb2YgdGhlIHJhdCBicmFpbiBieSBkaWZmZXJlbnRpYWwgZGlzcGxheTsgTGlmZSBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9MjE4My0yMTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTIvMy8xODEvOTYxMTM4L2RzaTAwMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgaGF2ZSBwcmV2aW91c2x5IGRlc2NyaWJlZCB0aGUgc2VxdWVuY2UgZmVhdHVyZXMgb2Yg4oi8MTUwMCBtb3VzZSBLSUFBIChtS0lBQSkgZ2VuZXMgaW4gY29tcGFyaXNvbiB3aXRoIHRob3NlIG9mIGh1bWFuIEtJQUEgZ2VuZXMgKE9rYXpha2ksIE4uLCBLaWt1biIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzM1ODYyJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDyd7WSCkihcpaDNl5rJOzPe+ixH" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f3%2f181%2f335862" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f3%2f181%2f335862" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12831" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/3"><img id="issueFallbackImage" class="fb-featured-image" src="//oup.silverchair-cdn.com/UI/app/img/generic-cover.png" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/3">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4674462">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4674462">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4674464">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4674464">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4674468">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4674468">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4674477">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4674477">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4674498">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4674498">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4674505">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4674505">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/12/3/167/337344">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/12/3/191/336418">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Influence of the 3′-UTR-length of mKIAA cDNAs and their Sequence Features to the mRNA Expression Level in the Brain
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Noriko Okazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-438-52-3932, Fax. +81-438-52-3931, E-mail: <a href="mailto:nokazaki@kazusa.or.jp" target="_blank">nokazaki@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Noriko+Okazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okazaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okazaki N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Noriko Okazaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuhide Imai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Chiba Industry Advancement Center</div><div class="addr-line">2-6 Nakase, Mihama-ku, Chiba 261-7126</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuhide+Imai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Imai K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Imai K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuhide Imai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Reiko F. Kikuno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Reiko+F.+Kikuno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kikuno R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kikuno R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Reiko F. Kikuno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuharu Misawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Chiba Industry Advancement Center</div><div class="addr-line">2-6 Nakase, Mihama-ku, Chiba 261-7126</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuharu+Misawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Misawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Misawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuharu Misawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makoto Kawai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Chiba Industry Advancement Center</div><div class="addr-line">2-6 Nakase, Mihama-ku, Chiba 261-7126</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makoto+Kawai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawai M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawai M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Makoto Kawai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Susumu Inamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Chiba Industry Advancement Center</div><div class="addr-line">2-6 Nakase, Mihama-ku, Chiba 261-7126</div></div><div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Institute of Research and Innovation</div><div class="addr-line">1201 Takada, Kashiwa, Chiba 277-0861</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Susumu+Inamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Inamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Inamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Susumu Inamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Reiko Ohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Reiko+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Reiko Ohara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiro Nagase</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiro+Nagase">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagase T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagase T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takahiro Nagase</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Osamu Ohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div></div><div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">RIKEN Yokohama Institute</div><div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama City, Kanagawa, 230-0045, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Osamu+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Osamu Ohara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisashi Koga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Chiba Industry Advancement Center</div><div class="addr-line">2-6 Nakase, Mihama-ku, Chiba 261-7126</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisashi+Koga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koga H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koga H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hisashi Koga</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Michio Oishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 3, 1 January 2005, Pages 181–189, <a href='https://doi.org/10.1093/dnares/dsi001'>https://doi.org/10.1093/dnares/dsi001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 January 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">04 April 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="335862" href="/dnaresearch/article-pdf/12/3/181/961138/dsi001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Noriko Okazaki, Kazuhide Imai, Reiko F. Kikuno, Kazuharu Misawa, Makoto Kawai, Susumu Inamoto, Reiko Ohara, Takahiro Nagase, Osamu Ohara, Hisashi Koga;  Influence of the 3′-UTR-length of mKIAA cDNAs and their Sequence Features to the mRNA Expression Level in the Brain, <em>DNA Research</em>, Volume 12, Issue 3, 1 January 2005, Pages 181–189, <a href='https://doi.org/10.1093/dnares/dsi001'>https://doi.org/10.1093/dnares/dsi001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/335862?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/335862?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/335862?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/335862?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/335862?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/335862?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/335862?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Influence%20of%20the%203%E2%80%B2-UTR-length%20of%20mKIAA%20cDNAs%20and%20their%20Sequence%20Features%20to%20the%20mRNA%20Expression%20Level%20in%20the%20Brain&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=3&amp;author=Okazaki%2C%20Noriko%3B%20Imai%2C%20Kazuhide&amp;startPage=181&amp;endPage=189&amp;contentId=10.1093%2Fdnares%2Fdsi001&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f3%2f181%2f335862" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4674462 id="4674462" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We have previously described the sequence features of ∼1500 mouse KIAA (<em>mKIAA</em>) genes in comparison with those of human <em>KIAA</em> genes (Okazaki, N., Kikuno, R., Inamoto, S., Hara, Y., Nagase, T., Ohara, O., and Koga, H. 2002, <em>DNA Res</em>., <strong>9</strong>, 179–188; Okazaki, N., Kikuno, R., Ohara, R., Inamoto, S., Aizawa, H., Yuasa, S., Nakajima, D., Nagase, T., Ohara, O., and Koga, H. 2003, <em>DNA Res</em>., <strong>10</strong>, 35–48; Okazaki, N., Kikuno, R., Ohara, R., Inamoto, S., Koseki, H., Hiraoka, S., Saga, Y., Nagase, T., Ohara, O., and Koga, H. 2003, <em>DNA Res</em>., <strong>10</strong>, 167–180; and Okazaki, N., F-Kikuno, R., Ohara, R., Inamoto, S., Koseki, H., Hiraoka, S., Saga, Y., Seino, S., Nishimura, M., Kaisho, T., Hoshino, K., Kitamura, H., Nagase, T., Ohara, O., and Koga, H. 2004, <em>DNA Res</em>., <strong>11</strong>, 205–218). To validate the orthologous relationship between <em>mKIAA</em> and <em>KIAA</em> genes in detail, we examined their chromosomal positions and evolutionary rate of synonymous substitutions and confirmed that &gt;93% of the <em>mKIAA</em>/<em>KIAA</em> gene pairs are orthologous. During the sequence analysis of <em>mKIAA</em> genes, we found that 3′-untranslated region (3′-UTR) lengths of <em>mKIAA</em> and <em>KIAA</em> genes are extremely long. In the meanwhile, we have also examined the tissue-specific expression of ∼1700 <em>mKIAA</em> genes using cDNA microarray and verified predominantly their expression in adult brain (Koga, H., Yuasa, S., Nagase, T., Shimada, K., Nagano, M., Imai, K., Ohara, R., Nakajima, D., Murakami, M., Kawai, M., Miki, F., Magae, J., Inamoto, S., Okazaki, N., Ohara, O. 2004, <em>DNA Res</em>., <strong>11</strong>, 293–304). To connect these two evidences, we statistically analysed the relationship between them by using the <em>mKIAA</em> genes. Consequently, a positive correlation was observed between the 3′-UTR lengths and the relative expression intensities in adult brain. Furthermore, we searched sequence elements in the 3′-UTR possibly related with their expression and found some candidates regarding the brain-specific expression.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="mKIAA"><em>mKIAA</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="orthology">orthology</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="3′-UTR">3′-UTR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="expression">expression</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="brain">brain</a></div>                            <h2 scrollto-destination=4674464 id="4674464" class="section-title" >1. Introduction</h2>
<p>After completion of the human genome project, numerous genes have been predicted by bioinformatic approachs. However, library construction still serves as a critical resource of actually existing genes and a platform for comprehensive analysis of the gene functions. With this in mind, we have constructed several cDNA libraries and initiated a human cDNA project to accumulate information regarding the long protein coding sequences (CDSs) of unidentified human genes since 1994.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> We have isolated and entirely sequenced long human cDNA clones (&gt;4 kb) and have already registered &gt;2000 genes (<em>KIAA</em> genes) to the public database to date. However, the function has been identified in some genes, half of the genes are still remained to be elucidated mainly depending on their difficulty for using human materials. Therefore, we decided to collect mouse <em>KIAA</em> cDNAs (<em>mKIAA</em>) for further functional analysis in a model organism.</p><p>We have previously reported the sequences of ∼1500 <em>mKIAA</em> cDNAs and the tissue-specific expression of ∼1700 <em>mKIAA</em> genes using cDNA microarray.<a reveal-id="b2  b6" data-open="b2  b6" class="link link-ref link-reveal xref-bibr">2–6</a> The entire sequence of <em>mKIAA</em> cDNAs revealed that <em>mKIAA</em> (also <em>KIAA</em>) cDNAs have extremely long 3′-untranslated region (3′-UTR) sequences. Furthermore, cDNA microarray analysis elucidated that ∼30% of the genes are predominantly expressed in the brain. We thus assumed some correlations lie between the two observations. The 3′-UTR sequences are critical for determining mRNA stability, mRNA targeting and level of translation.<a reveal-id="b7  b13" data-open="b7  b13" class="link link-ref link-reveal xref-bibr">7–13</a> The sequences are also known to be important for the pathogenesis of some disease through regulation of mRNA stability.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> Although molecular neuroscientists have empirically noted that mRNAs abundant in the brain have long 3′-UTR, statistical or comprehensive approaches on the correlation between the expression levels and the 3′-UTR lengths have not been performed.</p><p>In this paper, we first validate the orthology of <em>mKIAA</em> and <em>KIAA</em> cDNA pairs; subsequently, we report the statistical correlation between the relative expression levels in the brain and the 3′-UTR lengths, and discuss the possible functional sequence elements in the 3′-UTRs found in <em>mKIAA</em> and <em>KIAA</em> cDNAs.</p>                            <h2 scrollto-destination=4674468 id="4674468" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=4674470 id="4674470" class="section-title" >2.1. Nucleotide sequences</h3>
<p>We obtained <em>mKIAA</em> and <em>KIAA</em> cDNA sequences from our ROUGE (<a class="link link-uri" href="http://www.kazusa.or.jp/rouge/index.html" target="_blank"></a>) and HUGE (<a class="link link-uri" href="http://www.kazusa.or.jp/huge/index.html" target="_blank"></a>) databases, respectively.</p>                            <h3 scrollto-destination=4674472 id="4674472" class="section-title" >2.2. Estimation of orthology</h3>
<p>We estimated the evolutionary relationship between <em>KIAA</em> and <em>mKIAA</em> by the following steps: (i) find chromosomal position of <em>KIAA</em> genes using the NCBI Map Viewer (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/mapview/map_search.cgi/" target="_blank"></a>);<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> (ii) find the mouse chromosomal region corresponding to the human chromosomal region encoding the <em>KIAA</em> genes using the NCBI Map Viewer; (iii) find the chromosomal position of <em>mKIAA</em> using BLAST search;<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> and (iv) compare the mouse chromosomal region found in (ii) with the chromosomal position of <em>mKIAA</em> gene found in (iii). The number of synonymous substitutions per site (dS) was estimated by Nei and Gojobori's method<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> using SNAP program.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a></p>                            <h3 scrollto-destination=4674474 id="4674474" class="section-title" >2.3. A search for candidate elements effecting on the specific expression in brain</h3>
<p>We extracted conserved 3′-UTR sequences between 50 <em>mKIAA</em> highly expressing in brain and their orthologous <em>KIAA</em> cDNAs. The MEME program (<a class="link link-uri" href="http://meme.sdsc.edu/meme/website/meme.html" target="_blank"></a>) was used to find 20–25 bp elements that appeared &gt;50 times. The existence of newly identified sequence elements was searched with &gt;60% nucleotide identity against the 3′-UTR sequences of 1031 <em>mKIAA</em> cDNAs, which were verified for the integrity of the 3′-UTR structure. The existence of the elements was also searched against 3′-UTRs of non-<em>mKIAA</em> genes in UTRdb (<a class="link link-uri" href="http://www.ba.itb.cnr.it/BIG/UTRHome/" target="_blank"></a>) entries using BLAST search. Differences of the relative expression levels in brain between <em>mKIAA</em> cDNAs with and without the elements were calculated by the Student's <em>t</em>-test. The tissue specific expression of each gene was based on the descriptions in the literature or our original data (freely available through our database).<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a></p>                            <h2 scrollto-destination=4674477 id="4674477" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=4674478 id="4674478" class="section-title" >3.1. Identification of mouse KIAA-homologous cDNAs</h3>
<p>The mouse homologs of <em>KIAA</em> genes were isolated from size-fractionated cDNA libraries (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). Approximately 55% of <em>KIAA</em> cDNAs (1116 out of 2038 cDNAs) were derived from adult human brain libraries; thus, we initially subjected 23 000 end sequences of adult mouse brain library for the screening of <em>mKIAA</em> genes and identified nearly half of <em>mKIAA</em> cDNAs. The remainder was isolated from other libraries and consequently 1499 <em>mKIAA</em> homologs were successfully isolated and entirely sequenced. By comparing the <em>mKIAA</em> cDNA sequences with human sequences, we designated those 1479 mouse cDNA clones as ‘<em>mKIAA</em>’ and the same four-digit number corresponding to human clones. Although 20 cDNA clones were eventually found not to be orthologous to any <em>KIAA</em> cDNAs, these cDNAs were conventionally designated as ‘<em>mKIAA</em>’ and a four-digit number that has not been allocated for human <em>KIAA</em> genes (<em>mKIAA3000s</em>). The average length of the cDNA inserts was 4.6 kb and deduced gene products was 830 amino acid residues. Multiple CDSs were found in 237 <em>mKIAA</em> cDNAs that are longer than 50 amino acid residues and have high amino acid identity (&gt;50%) to corresponding <em>KIAA</em> by FASTA analysis.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> These multiple CDSs are thought to be produced by spurious CDS splits caused by retained intron(s), alternative splicing or splicing error, reverse transcription error(s) or other cloning artifacts.<a reveal-id="b2  b5" data-open="b2  b5" class="link link-ref link-reveal xref-bibr">2–5</a> We evaluated the spurious CDS splits through the sequence comparison of mouse and human <em>KIAA</em> cDNA pairs and the genome/cDNA structures,<a reveal-id="b2  b5" data-open="b2  b5" class="link link-ref link-reveal xref-bibr">2–5</a> and we found that 144 <em>mKIAA</em> cDNA clones appeared to retain intron(s). Taking the assumption into consideration, alternative prediction of the deduced gene products might increase to 873 amino acid residues.</p>                        <a id="4674480" scrollto-destination="4674480"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>The details of cDNA libraries used in this study.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissue </th><th>Age </th><th>Strain </th><th>Number of clones sequenced from their 5′ or 3′ end </th><th>Number of clones entirely sequenced </th></tr></thead><tbody><tr><td>Brain<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Adult </td><td>BALB/c </td><td>23 290 </td><td>712 </td></tr><tr><td>Tail </td><td>Embryo </td><td>ICR </td><td>31 537 </td><td>404 </td></tr><tr><td>Brain </td><td>Fetus (18 dpc) </td><td>ICR </td><td>20 929 </td><td>152 </td></tr><tr><td>Pancreas </td><td>Adult </td><td>C57BL/6 </td><td>29 334 </td><td>82 </td></tr><tr><td>Thymus </td><td>Adult </td><td>C57BL/6 </td><td>8149 </td><td>54 </td></tr><tr><td>Intestine </td><td>Embryo </td><td>ICR </td><td>12 399 </td><td>52 </td></tr><tr><td>Spleen </td><td>Adult </td><td>ICR </td><td>17 429 </td><td>22 </td></tr><tr><td>Natural killer T cells </td><td> </td><td>C57BL/6 </td><td>11 816 </td><td>19 </td></tr><tr><td>Bone marrow-derived dendritic cells </td><td> </td><td>C57BL/6 </td><td>2113 </td><td>1 </td></tr><tr><td>Splenocytes </td><td>Adult </td><td>ICR </td><td>2917 </td><td>1 </td></tr><tr><td>Total </td><td> </td><td> </td><td>159 913 </td><td>1499 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissue </th><th>Age </th><th>Strain </th><th>Number of clones sequenced from their 5′ or 3′ end </th><th>Number of clones entirely sequenced </th></tr></thead><tbody><tr><td>Brain<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Adult </td><td>BALB/c </td><td>23 290 </td><td>712 </td></tr><tr><td>Tail </td><td>Embryo </td><td>ICR </td><td>31 537 </td><td>404 </td></tr><tr><td>Brain </td><td>Fetus (18 dpc) </td><td>ICR </td><td>20 929 </td><td>152 </td></tr><tr><td>Pancreas </td><td>Adult </td><td>C57BL/6 </td><td>29 334 </td><td>82 </td></tr><tr><td>Thymus </td><td>Adult </td><td>C57BL/6 </td><td>8149 </td><td>54 </td></tr><tr><td>Intestine </td><td>Embryo </td><td>ICR </td><td>12 399 </td><td>52 </td></tr><tr><td>Spleen </td><td>Adult </td><td>ICR </td><td>17 429 </td><td>22 </td></tr><tr><td>Natural killer T cells </td><td> </td><td>C57BL/6 </td><td>11 816 </td><td>19 </td></tr><tr><td>Bone marrow-derived dendritic cells </td><td> </td><td>C57BL/6 </td><td>2113 </td><td>1 </td></tr><tr><td>Splenocytes </td><td>Adult </td><td>ICR </td><td>2917 </td><td>1 </td></tr><tr><td>Total </td><td> </td><td> </td><td>159 913 </td><td>1499 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf1-1"><div class="fn"><p><sup>a</sup>The brain library consists of ∼3 × 10<sup>6</sup> independent clones.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674480" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>The details of cDNA libraries used in this study.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissue </th><th>Age </th><th>Strain </th><th>Number of clones sequenced from their 5′ or 3′ end </th><th>Number of clones entirely sequenced </th></tr></thead><tbody><tr><td>Brain<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Adult </td><td>BALB/c </td><td>23 290 </td><td>712 </td></tr><tr><td>Tail </td><td>Embryo </td><td>ICR </td><td>31 537 </td><td>404 </td></tr><tr><td>Brain </td><td>Fetus (18 dpc) </td><td>ICR </td><td>20 929 </td><td>152 </td></tr><tr><td>Pancreas </td><td>Adult </td><td>C57BL/6 </td><td>29 334 </td><td>82 </td></tr><tr><td>Thymus </td><td>Adult </td><td>C57BL/6 </td><td>8149 </td><td>54 </td></tr><tr><td>Intestine </td><td>Embryo </td><td>ICR </td><td>12 399 </td><td>52 </td></tr><tr><td>Spleen </td><td>Adult </td><td>ICR </td><td>17 429 </td><td>22 </td></tr><tr><td>Natural killer T cells </td><td> </td><td>C57BL/6 </td><td>11 816 </td><td>19 </td></tr><tr><td>Bone marrow-derived dendritic cells </td><td> </td><td>C57BL/6 </td><td>2113 </td><td>1 </td></tr><tr><td>Splenocytes </td><td>Adult </td><td>ICR </td><td>2917 </td><td>1 </td></tr><tr><td>Total </td><td> </td><td> </td><td>159 913 </td><td>1499 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissue </th><th>Age </th><th>Strain </th><th>Number of clones sequenced from their 5′ or 3′ end </th><th>Number of clones entirely sequenced </th></tr></thead><tbody><tr><td>Brain<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Adult </td><td>BALB/c </td><td>23 290 </td><td>712 </td></tr><tr><td>Tail </td><td>Embryo </td><td>ICR </td><td>31 537 </td><td>404 </td></tr><tr><td>Brain </td><td>Fetus (18 dpc) </td><td>ICR </td><td>20 929 </td><td>152 </td></tr><tr><td>Pancreas </td><td>Adult </td><td>C57BL/6 </td><td>29 334 </td><td>82 </td></tr><tr><td>Thymus </td><td>Adult </td><td>C57BL/6 </td><td>8149 </td><td>54 </td></tr><tr><td>Intestine </td><td>Embryo </td><td>ICR </td><td>12 399 </td><td>52 </td></tr><tr><td>Spleen </td><td>Adult </td><td>ICR </td><td>17 429 </td><td>22 </td></tr><tr><td>Natural killer T cells </td><td> </td><td>C57BL/6 </td><td>11 816 </td><td>19 </td></tr><tr><td>Bone marrow-derived dendritic cells </td><td> </td><td>C57BL/6 </td><td>2113 </td><td>1 </td></tr><tr><td>Splenocytes </td><td>Adult </td><td>ICR </td><td>2917 </td><td>1 </td></tr><tr><td>Total </td><td> </td><td> </td><td>159 913 </td><td>1499 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf1-1"><div class="fn"><p><sup>a</sup>The brain library consists of ∼3 × 10<sup>6</sup> independent clones.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674480" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4674481 id="4674481" class="section-title" >3.2. Orthology between mouse and human KIAAs</h3>
<p>We have only defined the cDNA sequences as mouse homologs of <em>KIAA</em> genes when they showed the highest homology against the corresponding human <em>KIAA</em> genes at the time of being sequenced. To support orthologous relationship of <em>mKIAA</em> and <em>KIAA</em> gene pairs, we assigned their chromosomal positions and examined whether they were derived from the common ancestral chromosomal location. We classified the positional relations of the gene pairs into the following three categories using the NCBI Map Viewer:<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> (i) the 896 gene pairs located on the corresponding chromosomal regions between mouse and human; they would be judged to be orthologous; (ii) the 102 gene pairs not located on the corresponding chromosomal regions; they could not be judged to be orthologous; and (iii) the 481 gene pairs whose sequence or data on the chromosome location were not available; also, they could not be judged to be orthologous.</p><p>For further verification of the orthology, we also investigated the evolutionary rate of dS estimated by the Nei and Gojobori's method.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> The rate of human/mouse orthologous pair is approximately constant, since they are diverged concurrently 80 million years ago and accumulated dS during the same period.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> Paralogous gene pairs usually show higher rate of dS than orthologous gene pairs.<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> The dS of 1479 <em>mKIAA</em>/<em>KIAA</em> pairs (0.64 ± 0.22, max. = 4.86, min. = 0.01) is slightly higher but consistent with the previous study that calculated dS of orthologous gene pairs between human and mouse (0.55 ± 0.63).<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> Therefore, most of the <em>mKIAA</em>/<em>KIAA</em> pairs were thought to be orthologous. We then compared the distribution of the dS values among the above-mentioned three categories based on the chromosomal positions (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1a</a>). The distributions of dS values were quite similar, therefore, not many paralogs would be included if they exist in the gene pairs whose correspondence was not supported by the chromosomal locations. Furthermore, to confirm the orthology we examined in detail the orthology of the 100 gene pairs that showed unusually high rate of dS (&gt; mean + 1.31 SD) by the phylogenetic analysis (<a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>, <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1b</a>).<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> Among them 93 <em>mKIAA</em> genes were confirmed to be orthologs, whereas the remaining seven <em>mKIAA</em> genes (<em>mKIAA0345</em>, <em>mKIAA0420</em>, <em>mKIAA0588</em>, <em>mKIAA1032</em>, <em>mKIAA1074</em>, <em>mKIAA1120</em> and <em>mKIAA1811</em>) were determined to be paralogs of the corresponding <em>KIAA</em> genes. These results suggest that most of the <em>KIAA</em>/<em>mKIAA</em> gene pairs have orthologous relationship, and if the paralogous pairs existed it must be &lt;7%.</p>                        <a id="4674485" scrollto-destination="4674485"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/3/10.1093/dnares/dsi001/2/m_dsi001f1.jpeg?Expires=2147483647&Signature=j8QFlPOk8qTW3eLpPgEZIDAj87riWu~GZR-q6BhXK39~wHHcEE5vg5bBtU1ptstsJ5RDjmOgxZnBcRD0UroDWxhyoSVjdHEaIdvVueWom9SxzVz2sjYaJbk0IaCIzJdHTNd5Ca-~FHH6OjcXUVlnROKWGXjRIgRQROQGWx1wDqBbQbeWc2XW655Wph7c4hTqWW0VZmpFA~aOdf0XyCGOHEGACR22-UgNDnI68mkipY-7fNEx9Lh7Ci2rKIidpdpz2E32AqUsqwzk9bbGogX6kVhQOE~WL2l9OUJ1PmDsTQg3bKO72ITXSCsjZ9-SNpvlfVa23ixH~k0n6V9Sya46FQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram of dS and phylogenetic tree for mKIAA/KIAA possible orthologous pairs. (a) Histograms of dS of gene pairs classified into the following three categories by chromosomal localization: (1) the gene pairs located on the corresponding chromosomal regions between human and mouse calculated by Map Viewer; these regions are considered to be derived from the common regions of the ancestral chromosome; (2) the gene pairs whose chromosomal locations were not corresponding to each other; and (3) the gene pairs whose data on the chromosome location or sequence were not available. (b) A representative phylogenetic tree of mKIAA/KIAA protein and 10 highly homologous proteins based on amino acid content. Phylogenetic trees were constructed and analysed by neighbor-joining method.24 The numbers above the branches are confidence values based on Felsenstein's bootstrap method.37 When mKIAA protein belongs to the closest mouse group to KIAA protein, mKIAA protein was estimated to be the ortholog. In this case, mKIAA1032 is defined as the ortholog of NP 006368/AAC19406." path-from-xml="dsi001f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4674485/dsi001f1.jpeg" path-from-xml="dsi001f1.jpeg" target="_blank">View large</a><a section="4674485" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/3/10.1093/dnares/dsi001/2/dsi001f1.jpeg?Expires=2147483647&Signature=fG6qrjj3F7R~6eysCxdtJtkLAptu9Se7SNDgHGpGLx3Vulkwe3md5A-QCbLZii1OnniSv7tvhKIrEPm5LqzdUa5EQuw7kOuG60CvQ79~yL79NnOCWtuq-kbKIWFFFPGhuFOPkc8Gia00tHQr2PWUKQUyio5FGue5qL7jyA71zXmkgRBZ~RLXBMS3fTcEJB-JBE9DgYwsx2k1noqF66XzlZZh2MQ9hp2rtT8ZtMoilikF4L-ossg6VxNYNwc5YugBZGPdcGvteUjwWeft421doJT1pRdzv-uwqjeaS0z~IrZajZzrdiUoPvk71BHrSl1F6Buq175zGtQR64Umr0R2Ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4674485&ar=335862&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi001f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram of dS and phylogenetic tree for <em>mKIAA</em>/<em>KIAA</em> possible orthologous pairs. (a) Histograms of dS of gene pairs classified into the following three categories by chromosomal localization: (1) the gene pairs located on the corresponding chromosomal regions between human and mouse calculated by Map Viewer; these regions are considered to be derived from the common regions of the ancestral chromosome; (2) the gene pairs whose chromosomal locations were not corresponding to each other; and (3) the gene pairs whose data on the chromosome location or sequence were not available. (b) A representative phylogenetic tree of mKIAA/KIAA protein and 10 highly homologous proteins based on amino acid content. Phylogenetic trees were constructed and analysed by neighbor-joining method.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> The numbers above the branches are confidence values based on Felsenstein's bootstrap method.<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> When mKIAA protein belongs to the closest mouse group to KIAA protein, mKIAA protein was estimated to be the ortholog. In this case, <em>mKIAA1032</em> is defined as the ortholog of NP 006368/AAC19406.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/3/10.1093/dnares/dsi001/2/m_dsi001f1.jpeg?Expires=2147483647&Signature=j8QFlPOk8qTW3eLpPgEZIDAj87riWu~GZR-q6BhXK39~wHHcEE5vg5bBtU1ptstsJ5RDjmOgxZnBcRD0UroDWxhyoSVjdHEaIdvVueWom9SxzVz2sjYaJbk0IaCIzJdHTNd5Ca-~FHH6OjcXUVlnROKWGXjRIgRQROQGWx1wDqBbQbeWc2XW655Wph7c4hTqWW0VZmpFA~aOdf0XyCGOHEGACR22-UgNDnI68mkipY-7fNEx9Lh7Ci2rKIidpdpz2E32AqUsqwzk9bbGogX6kVhQOE~WL2l9OUJ1PmDsTQg3bKO72ITXSCsjZ9-SNpvlfVa23ixH~k0n6V9Sya46FQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram of dS and phylogenetic tree for mKIAA/KIAA possible orthologous pairs. (a) Histograms of dS of gene pairs classified into the following three categories by chromosomal localization: (1) the gene pairs located on the corresponding chromosomal regions between human and mouse calculated by Map Viewer; these regions are considered to be derived from the common regions of the ancestral chromosome; (2) the gene pairs whose chromosomal locations were not corresponding to each other; and (3) the gene pairs whose data on the chromosome location or sequence were not available. (b) A representative phylogenetic tree of mKIAA/KIAA protein and 10 highly homologous proteins based on amino acid content. Phylogenetic trees were constructed and analysed by neighbor-joining method.24 The numbers above the branches are confidence values based on Felsenstein's bootstrap method.37 When mKIAA protein belongs to the closest mouse group to KIAA protein, mKIAA protein was estimated to be the ortholog. In this case, mKIAA1032 is defined as the ortholog of NP 006368/AAC19406." path-from-xml="dsi001f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4674485/dsi001f1.jpeg" path-from-xml="dsi001f1.jpeg" target="_blank">View large</a><a section="4674485" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/3/10.1093/dnares/dsi001/2/dsi001f1.jpeg?Expires=2147483647&Signature=fG6qrjj3F7R~6eysCxdtJtkLAptu9Se7SNDgHGpGLx3Vulkwe3md5A-QCbLZii1OnniSv7tvhKIrEPm5LqzdUa5EQuw7kOuG60CvQ79~yL79NnOCWtuq-kbKIWFFFPGhuFOPkc8Gia00tHQr2PWUKQUyio5FGue5qL7jyA71zXmkgRBZ~RLXBMS3fTcEJB-JBE9DgYwsx2k1noqF66XzlZZh2MQ9hp2rtT8ZtMoilikF4L-ossg6VxNYNwc5YugBZGPdcGvteUjwWeft421doJT1pRdzv-uwqjeaS0z~IrZajZzrdiUoPvk71BHrSl1F6Buq175zGtQR64Umr0R2Ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4674485&ar=335862&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi001f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram of dS and phylogenetic tree for <em>mKIAA</em>/<em>KIAA</em> possible orthologous pairs. (a) Histograms of dS of gene pairs classified into the following three categories by chromosomal localization: (1) the gene pairs located on the corresponding chromosomal regions between human and mouse calculated by Map Viewer; these regions are considered to be derived from the common regions of the ancestral chromosome; (2) the gene pairs whose chromosomal locations were not corresponding to each other; and (3) the gene pairs whose data on the chromosome location or sequence were not available. (b) A representative phylogenetic tree of mKIAA/KIAA protein and 10 highly homologous proteins based on amino acid content. Phylogenetic trees were constructed and analysed by neighbor-joining method.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> The numbers above the branches are confidence values based on Felsenstein's bootstrap method.<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> When mKIAA protein belongs to the closest mouse group to KIAA protein, mKIAA protein was estimated to be the ortholog. In this case, <em>mKIAA1032</em> is defined as the ortholog of NP 006368/AAC19406.</p></div></div>                        <a id="4674486" scrollto-destination="4674486"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Orthology between mouse and human <em>KIAAs</em>.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Distance from the mean<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Synonymous substitution<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Chromosomal localization<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">c</a> </th><th>Judgement by phylogenetic tree analysis<a reveal-id="tf2-3" data-open="tf2-3" class="link link-ref link-reveal xref-fn">d</a> </th><th><em>mKIAA</em> </th></tr></thead><tbody><tr><td>&gt;Mean + 4 SD </td><td>NA </td><td>2 </td><td>P </td><td>mKIAA1032 </td></tr><tr><td> </td><td>NA </td><td>3 </td><td>O </td><td>mKIAA1710 </td></tr><tr><td> </td><td>4.86 </td><td>1 </td><td>O </td><td>mKIAA1561 </td></tr><tr><td> </td><td>2.62 </td><td>3 </td><td>O </td><td>mKIAA0961 </td></tr><tr><td> </td><td>1.91 </td><td>2 </td><td>P </td><td>mKIAA0420 </td></tr><tr><td>&gt;Mean + 3 SD </td><td>1.44 </td><td>1 </td><td>P </td><td>mKIAA1811 </td></tr><tr><td> </td><td>1.36 </td><td>1 </td><td>O </td><td>mKIAA0908 </td></tr><tr><td> </td><td>1.36 </td><td>1 </td><td>O </td><td>mKIAA1023 </td></tr><tr><td>&gt;Mean + 2 SD </td><td>1.28 </td><td>3 </td><td>O </td><td>mKIAA1881 </td></tr><tr><td> </td><td>1.28 </td><td>3 </td><td>O </td><td>mKIAA1913 </td></tr><tr><td> </td><td>1.26 </td><td>3 </td><td>O </td><td>mKIAA1273 </td></tr><tr><td> </td><td>1.25 </td><td>2 </td><td>P </td><td>mKIAA1074 </td></tr><tr><td> </td><td>1.23 </td><td>1 </td><td>O </td><td>mKIAA0083 </td></tr><tr><td> </td><td>1.23 </td><td>1 </td><td>O </td><td>mKIAA1162 </td></tr><tr><td> </td><td>1.23 </td><td>3 </td><td>O </td><td>mKIAA1483 </td></tr><tr><td> </td><td>1.22 </td><td>1 </td><td>O </td><td>mKIAA1876 </td></tr><tr><td> </td><td>1.20 </td><td>3 </td><td>O </td><td>mKIAA1600 </td></tr><tr><td> </td><td>1.19 </td><td>1 </td><td>O </td><td>mKIAA1335 </td></tr><tr><td> </td><td>1.18 </td><td>3 </td><td>O </td><td>mKIAA1323 </td></tr><tr><td> </td><td>1.17 </td><td>3 </td><td>O </td><td>mKIAA1466 </td></tr><tr><td> </td><td>1.17 </td><td>3 </td><td>P </td><td>mKIAA0588 </td></tr><tr><td> </td><td>1.16 </td><td>3 </td><td>O </td><td>mKIAA0628 </td></tr><tr><td> </td><td>1.16 </td><td>1 </td><td>O </td><td>mKIAA1878 </td></tr><tr><td> </td><td>1.15 </td><td>2 </td><td>O </td><td>mKIAA1948 </td></tr><tr><td> </td><td>1.14 </td><td>1 </td><td>O </td><td>mKIAA1015 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0213 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA1676 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0826 </td></tr><tr><td> </td><td>1.10 </td><td>3 </td><td>P </td><td>mKIAA1120 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0610 </td></tr><tr><td> </td><td>1.09 </td><td>1 </td><td>O </td><td>mKIAA0439 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1389 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1716 </td></tr><tr><td> </td><td>1.08 </td><td>3 </td><td>O </td><td>mKIAA0365 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1729 </td></tr><tr><td> </td><td>1.08 </td><td>3 </td><td>O </td><td>mKIAA0814 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA0803 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA0207 </td></tr><tr><td> </td><td>1.07 </td><td>3 </td><td>O </td><td>mKIAA0637 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Distance from the mean<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Synonymous substitution<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Chromosomal localization<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">c</a> </th><th>Judgement by phylogenetic tree analysis<a reveal-id="tf2-3" data-open="tf2-3" class="link link-ref link-reveal xref-fn">d</a> </th><th><em>mKIAA</em> </th></tr></thead><tbody><tr><td>&gt;Mean + 4 SD </td><td>NA </td><td>2 </td><td>P </td><td>mKIAA1032 </td></tr><tr><td> </td><td>NA </td><td>3 </td><td>O </td><td>mKIAA1710 </td></tr><tr><td> </td><td>4.86 </td><td>1 </td><td>O </td><td>mKIAA1561 </td></tr><tr><td> </td><td>2.62 </td><td>3 </td><td>O </td><td>mKIAA0961 </td></tr><tr><td> </td><td>1.91 </td><td>2 </td><td>P </td><td>mKIAA0420 </td></tr><tr><td>&gt;Mean + 3 SD </td><td>1.44 </td><td>1 </td><td>P </td><td>mKIAA1811 </td></tr><tr><td> </td><td>1.36 </td><td>1 </td><td>O </td><td>mKIAA0908 </td></tr><tr><td> </td><td>1.36 </td><td>1 </td><td>O </td><td>mKIAA1023 </td></tr><tr><td>&gt;Mean + 2 SD </td><td>1.28 </td><td>3 </td><td>O </td><td>mKIAA1881 </td></tr><tr><td> </td><td>1.28 </td><td>3 </td><td>O </td><td>mKIAA1913 </td></tr><tr><td> </td><td>1.26 </td><td>3 </td><td>O </td><td>mKIAA1273 </td></tr><tr><td> </td><td>1.25 </td><td>2 </td><td>P </td><td>mKIAA1074 </td></tr><tr><td> </td><td>1.23 </td><td>1 </td><td>O </td><td>mKIAA0083 </td></tr><tr><td> </td><td>1.23 </td><td>1 </td><td>O </td><td>mKIAA1162 </td></tr><tr><td> </td><td>1.23 </td><td>3 </td><td>O </td><td>mKIAA1483 </td></tr><tr><td> </td><td>1.22 </td><td>1 </td><td>O </td><td>mKIAA1876 </td></tr><tr><td> </td><td>1.20 </td><td>3 </td><td>O </td><td>mKIAA1600 </td></tr><tr><td> </td><td>1.19 </td><td>1 </td><td>O </td><td>mKIAA1335 </td></tr><tr><td> </td><td>1.18 </td><td>3 </td><td>O </td><td>mKIAA1323 </td></tr><tr><td> </td><td>1.17 </td><td>3 </td><td>O </td><td>mKIAA1466 </td></tr><tr><td> </td><td>1.17 </td><td>3 </td><td>P </td><td>mKIAA0588 </td></tr><tr><td> </td><td>1.16 </td><td>3 </td><td>O </td><td>mKIAA0628 </td></tr><tr><td> </td><td>1.16 </td><td>1 </td><td>O </td><td>mKIAA1878 </td></tr><tr><td> </td><td>1.15 </td><td>2 </td><td>O </td><td>mKIAA1948 </td></tr><tr><td> </td><td>1.14 </td><td>1 </td><td>O </td><td>mKIAA1015 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0213 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA1676 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0826 </td></tr><tr><td> </td><td>1.10 </td><td>3 </td><td>P </td><td>mKIAA1120 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0610 </td></tr><tr><td> </td><td>1.09 </td><td>1 </td><td>O </td><td>mKIAA0439 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1389 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1716 </td></tr><tr><td> </td><td>1.08 </td><td>3 </td><td>O </td><td>mKIAA0365 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1729 </td></tr><tr><td> </td><td>1.08 </td><td>3 </td><td>O </td><td>mKIAA0814 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA0803 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA0207 </td></tr><tr><td> </td><td>1.07 </td><td>3 </td><td>O </td><td>mKIAA0637 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The orthology of top 100 <em>mKIAA</em>/<em>KIAA</em> gene pairs that shows the high rate of dS were investigated in detail by phylogenetic tree analysis.</p></div></div><div content-id="tf2-1"><div class="fn"><p><sup>a</sup>The mean value of dS was 0.635 and the SD was 0.221.</p></div></div><div content-id="tf2-2"><div class="fn"><p><sup>b,c</sup>Value of dS and chromosomal localization were estimated as described in the legend of <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>.</p></div></div><div content-id="tf2-3"><div class="fn"><p><sup>d</sup>Phylogenetic trees of <em>KIAA</em> and <em>mKIAA</em> genes were constructed and analysed as described in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1b</a>. If the pair was judged as orthologous, it is indicated by ‘O’, otherwise, it is indicated by ‘P’. Because of occupancy of too much space the data for &lt; mean + 2 SD were omitted (61 samples). In this range there is a paralog (mKIAA0345; ds = 1.06885; chromosomal localization = no data).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674486" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Orthology between mouse and human <em>KIAAs</em>.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Distance from the mean<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Synonymous substitution<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Chromosomal localization<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">c</a> </th><th>Judgement by phylogenetic tree analysis<a reveal-id="tf2-3" data-open="tf2-3" class="link link-ref link-reveal xref-fn">d</a> </th><th><em>mKIAA</em> </th></tr></thead><tbody><tr><td>&gt;Mean + 4 SD </td><td>NA </td><td>2 </td><td>P </td><td>mKIAA1032 </td></tr><tr><td> </td><td>NA </td><td>3 </td><td>O </td><td>mKIAA1710 </td></tr><tr><td> </td><td>4.86 </td><td>1 </td><td>O </td><td>mKIAA1561 </td></tr><tr><td> </td><td>2.62 </td><td>3 </td><td>O </td><td>mKIAA0961 </td></tr><tr><td> </td><td>1.91 </td><td>2 </td><td>P </td><td>mKIAA0420 </td></tr><tr><td>&gt;Mean + 3 SD </td><td>1.44 </td><td>1 </td><td>P </td><td>mKIAA1811 </td></tr><tr><td> </td><td>1.36 </td><td>1 </td><td>O </td><td>mKIAA0908 </td></tr><tr><td> </td><td>1.36 </td><td>1 </td><td>O </td><td>mKIAA1023 </td></tr><tr><td>&gt;Mean + 2 SD </td><td>1.28 </td><td>3 </td><td>O </td><td>mKIAA1881 </td></tr><tr><td> </td><td>1.28 </td><td>3 </td><td>O </td><td>mKIAA1913 </td></tr><tr><td> </td><td>1.26 </td><td>3 </td><td>O </td><td>mKIAA1273 </td></tr><tr><td> </td><td>1.25 </td><td>2 </td><td>P </td><td>mKIAA1074 </td></tr><tr><td> </td><td>1.23 </td><td>1 </td><td>O </td><td>mKIAA0083 </td></tr><tr><td> </td><td>1.23 </td><td>1 </td><td>O </td><td>mKIAA1162 </td></tr><tr><td> </td><td>1.23 </td><td>3 </td><td>O </td><td>mKIAA1483 </td></tr><tr><td> </td><td>1.22 </td><td>1 </td><td>O </td><td>mKIAA1876 </td></tr><tr><td> </td><td>1.20 </td><td>3 </td><td>O </td><td>mKIAA1600 </td></tr><tr><td> </td><td>1.19 </td><td>1 </td><td>O </td><td>mKIAA1335 </td></tr><tr><td> </td><td>1.18 </td><td>3 </td><td>O </td><td>mKIAA1323 </td></tr><tr><td> </td><td>1.17 </td><td>3 </td><td>O </td><td>mKIAA1466 </td></tr><tr><td> </td><td>1.17 </td><td>3 </td><td>P </td><td>mKIAA0588 </td></tr><tr><td> </td><td>1.16 </td><td>3 </td><td>O </td><td>mKIAA0628 </td></tr><tr><td> </td><td>1.16 </td><td>1 </td><td>O </td><td>mKIAA1878 </td></tr><tr><td> </td><td>1.15 </td><td>2 </td><td>O </td><td>mKIAA1948 </td></tr><tr><td> </td><td>1.14 </td><td>1 </td><td>O </td><td>mKIAA1015 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0213 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA1676 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0826 </td></tr><tr><td> </td><td>1.10 </td><td>3 </td><td>P </td><td>mKIAA1120 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0610 </td></tr><tr><td> </td><td>1.09 </td><td>1 </td><td>O </td><td>mKIAA0439 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1389 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1716 </td></tr><tr><td> </td><td>1.08 </td><td>3 </td><td>O </td><td>mKIAA0365 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1729 </td></tr><tr><td> </td><td>1.08 </td><td>3 </td><td>O </td><td>mKIAA0814 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA0803 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA0207 </td></tr><tr><td> </td><td>1.07 </td><td>3 </td><td>O </td><td>mKIAA0637 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Distance from the mean<a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Synonymous substitution<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Chromosomal localization<a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">c</a> </th><th>Judgement by phylogenetic tree analysis<a reveal-id="tf2-3" data-open="tf2-3" class="link link-ref link-reveal xref-fn">d</a> </th><th><em>mKIAA</em> </th></tr></thead><tbody><tr><td>&gt;Mean + 4 SD </td><td>NA </td><td>2 </td><td>P </td><td>mKIAA1032 </td></tr><tr><td> </td><td>NA </td><td>3 </td><td>O </td><td>mKIAA1710 </td></tr><tr><td> </td><td>4.86 </td><td>1 </td><td>O </td><td>mKIAA1561 </td></tr><tr><td> </td><td>2.62 </td><td>3 </td><td>O </td><td>mKIAA0961 </td></tr><tr><td> </td><td>1.91 </td><td>2 </td><td>P </td><td>mKIAA0420 </td></tr><tr><td>&gt;Mean + 3 SD </td><td>1.44 </td><td>1 </td><td>P </td><td>mKIAA1811 </td></tr><tr><td> </td><td>1.36 </td><td>1 </td><td>O </td><td>mKIAA0908 </td></tr><tr><td> </td><td>1.36 </td><td>1 </td><td>O </td><td>mKIAA1023 </td></tr><tr><td>&gt;Mean + 2 SD </td><td>1.28 </td><td>3 </td><td>O </td><td>mKIAA1881 </td></tr><tr><td> </td><td>1.28 </td><td>3 </td><td>O </td><td>mKIAA1913 </td></tr><tr><td> </td><td>1.26 </td><td>3 </td><td>O </td><td>mKIAA1273 </td></tr><tr><td> </td><td>1.25 </td><td>2 </td><td>P </td><td>mKIAA1074 </td></tr><tr><td> </td><td>1.23 </td><td>1 </td><td>O </td><td>mKIAA0083 </td></tr><tr><td> </td><td>1.23 </td><td>1 </td><td>O </td><td>mKIAA1162 </td></tr><tr><td> </td><td>1.23 </td><td>3 </td><td>O </td><td>mKIAA1483 </td></tr><tr><td> </td><td>1.22 </td><td>1 </td><td>O </td><td>mKIAA1876 </td></tr><tr><td> </td><td>1.20 </td><td>3 </td><td>O </td><td>mKIAA1600 </td></tr><tr><td> </td><td>1.19 </td><td>1 </td><td>O </td><td>mKIAA1335 </td></tr><tr><td> </td><td>1.18 </td><td>3 </td><td>O </td><td>mKIAA1323 </td></tr><tr><td> </td><td>1.17 </td><td>3 </td><td>O </td><td>mKIAA1466 </td></tr><tr><td> </td><td>1.17 </td><td>3 </td><td>P </td><td>mKIAA0588 </td></tr><tr><td> </td><td>1.16 </td><td>3 </td><td>O </td><td>mKIAA0628 </td></tr><tr><td> </td><td>1.16 </td><td>1 </td><td>O </td><td>mKIAA1878 </td></tr><tr><td> </td><td>1.15 </td><td>2 </td><td>O </td><td>mKIAA1948 </td></tr><tr><td> </td><td>1.14 </td><td>1 </td><td>O </td><td>mKIAA1015 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0213 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA1676 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0826 </td></tr><tr><td> </td><td>1.10 </td><td>3 </td><td>P </td><td>mKIAA1120 </td></tr><tr><td> </td><td>1.10 </td><td>1 </td><td>O </td><td>mKIAA0610 </td></tr><tr><td> </td><td>1.09 </td><td>1 </td><td>O </td><td>mKIAA0439 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1389 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1716 </td></tr><tr><td> </td><td>1.08 </td><td>3 </td><td>O </td><td>mKIAA0365 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA1729 </td></tr><tr><td> </td><td>1.08 </td><td>3 </td><td>O </td><td>mKIAA0814 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA0803 </td></tr><tr><td> </td><td>1.08 </td><td>1 </td><td>O </td><td>mKIAA0207 </td></tr><tr><td> </td><td>1.07 </td><td>3 </td><td>O </td><td>mKIAA0637 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The orthology of top 100 <em>mKIAA</em>/<em>KIAA</em> gene pairs that shows the high rate of dS were investigated in detail by phylogenetic tree analysis.</p></div></div><div content-id="tf2-1"><div class="fn"><p><sup>a</sup>The mean value of dS was 0.635 and the SD was 0.221.</p></div></div><div content-id="tf2-2"><div class="fn"><p><sup>b,c</sup>Value of dS and chromosomal localization were estimated as described in the legend of <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>.</p></div></div><div content-id="tf2-3"><div class="fn"><p><sup>d</sup>Phylogenetic trees of <em>KIAA</em> and <em>mKIAA</em> genes were constructed and analysed as described in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1b</a>. If the pair was judged as orthologous, it is indicated by ‘O’, otherwise, it is indicated by ‘P’. Because of occupancy of too much space the data for &lt; mean + 2 SD were omitted (61 samples). In this range there is a paralog (mKIAA0345; ds = 1.06885; chromosomal localization = no data).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674486" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4674487 id="4674487" class="section-title" >3.3. Sequence comparison of mouse and human KIAA cDNA pairs</h3>
<p>In our previous study, we reported the sequence identity of 3′-UTR and CDSs for the 100 mouse and human <em>KIAA</em> cDNA pairs.<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> We extended the same analysis to the 1468 gene pairs (<a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>). Similar to our previous study, the average aligned length of CDS and 3′-UTR of <em>mKIAA</em> and <em>KIAA</em> cDNAs are 2–3 times longer than reported by the other group, while the observed sequence identities of <em>KIAA</em> and <em>mKIAA</em> cDNAs in CDS and 3′-UTR agreed with the values by the other group.<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> In <em>Saccharomyces cerevisiae</em>, the UTR lengths are in a narrow range whereas the CDS lengths widely change in parallel with the mRNA lengths.<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> Although this narrow range of the UTR lengths is applicable to other species, the considerably long 3′-UTR sequences of <em>mKIAA</em>/<em>KIAA</em> cDNAs would be exceptional. Thus, the longer length and high conservation of 3′-UTR sequences of <em>mKIAA</em>/<em>KIAA</em> cDNAs may imply the unverified important roles in the sequences.</p>                        <a id="4674489" scrollto-destination="4674489"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Sequence identities of CDS and 3′-UTR between <em>mKIAA</em>/<em>KIAA</em> cDNA pairs.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Region </th><th>Sample number </th><th>Aligned length ± SD (bp) </th><th>Identity ± SD (%) </th></tr></thead><tbody><tr><td>CDS<a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>1468 </td><td>2335 ± 1232 </td><td>85.6 ± 5.1 </td></tr><tr><td>3′-UTR<a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>1080 </td><td>1435 ± 1092 </td><td>70.6 ± 9.4 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>Region </th><th>Sample number </th><th>Aligned length ± SD (bp) </th><th>Identity ± SD (%) </th></tr></thead><tbody><tr><td>CDS<a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>1468 </td><td>2335 ± 1232 </td><td>85.6 ± 5.1 </td></tr><tr><td>3′-UTR<a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>1080 </td><td>1435 ± 1092 </td><td>70.6 ± 9.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf3-1"><div class="fn"><p><sup>a</sup>We excluded the 31 <em>mKIAA</em> cDNAs for the following reasons: the four <em>mKIAA</em> cDNAs (mKIAA0728, mKIAA1335, mKIAA1466 and mKIAA1968) aligned with only short regions (&lt;50 amino acid residues) of the corresponding human <em>KIAA</em> CDSs; 20 <em>mKIAA</em> cDNAs incidentally identified were not orthologous to any <em>KIAA</em> cDNAs; and seven <em>mKIAA</em> cDNAs were found not to be orthologous to <em>KIAA</em> cDNAs in this study.</p></div></div><div content-id="tf3-2"><div class="fn"><p><sup>b</sup>We only calculated the lengths and the identities for the 3′-UTR when the sequence alignment was available for &gt;50 nt residues between human and mouse. We also excluded the 31 <em>mKIAA</em> cDNAs mentioned in previous footnote.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674489" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Sequence identities of CDS and 3′-UTR between <em>mKIAA</em>/<em>KIAA</em> cDNA pairs.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Region </th><th>Sample number </th><th>Aligned length ± SD (bp) </th><th>Identity ± SD (%) </th></tr></thead><tbody><tr><td>CDS<a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>1468 </td><td>2335 ± 1232 </td><td>85.6 ± 5.1 </td></tr><tr><td>3′-UTR<a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>1080 </td><td>1435 ± 1092 </td><td>70.6 ± 9.4 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>Region </th><th>Sample number </th><th>Aligned length ± SD (bp) </th><th>Identity ± SD (%) </th></tr></thead><tbody><tr><td>CDS<a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>1468 </td><td>2335 ± 1232 </td><td>85.6 ± 5.1 </td></tr><tr><td>3′-UTR<a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>1080 </td><td>1435 ± 1092 </td><td>70.6 ± 9.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf3-1"><div class="fn"><p><sup>a</sup>We excluded the 31 <em>mKIAA</em> cDNAs for the following reasons: the four <em>mKIAA</em> cDNAs (mKIAA0728, mKIAA1335, mKIAA1466 and mKIAA1968) aligned with only short regions (&lt;50 amino acid residues) of the corresponding human <em>KIAA</em> CDSs; 20 <em>mKIAA</em> cDNAs incidentally identified were not orthologous to any <em>KIAA</em> cDNAs; and seven <em>mKIAA</em> cDNAs were found not to be orthologous to <em>KIAA</em> cDNAs in this study.</p></div></div><div content-id="tf3-2"><div class="fn"><p><sup>b</sup>We only calculated the lengths and the identities for the 3′-UTR when the sequence alignment was available for &gt;50 nt residues between human and mouse. We also excluded the 31 <em>mKIAA</em> cDNAs mentioned in previous footnote.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674489" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4674490 id="4674490" class="section-title" >3.4. Positive correlation between the brain-specific expression and the 3′-UTR length of the genes</h3>
<p>Previously, we examined tissue-specific expression of <em>mKIAA</em> genes using <em>mKIAA</em> cDNA microarray.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> On the microarray, 1467 out of 1499 <em>mKIAA</em> cDNAs were spotted in this study and ∼30% of the genes were predominantly expressed in the brain. Therefore, we focused on the long 3′-UTRs to examine the relationship between brain-specific expression and their sequence features. Statistical analysis of ∼1031 clones using the Kendall rank correlation measurement revealed a significant correlation between 3′-UTR lengths and relative expression levels in the brain, exhibiting correlation coefficient (Tau) = 0.16, <em>P</em> = 1.7 × 10<sup>−14</sup>. However, there were no significant correlation between relative expression levels in other tissues and their 3′-UTR lengths (<a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>). To exclude the bias against the sources of cDNA libraries, the clones were subdivided into two groups (brain-derived and other tissues-derived) and statistically analysed using the same formula. Again, we observed significant correlation between 3′-UTR lengths and relative expression levels in both groups [brain, (Tau) = 0.14, <em>P</em> = 7.3 × 10<sup>−6</sup>, other tissues, (Tau) = 0.13, <em>P</em> = 1.3 × 10<sup>−5</sup>]. Furthermore, we examined the relationship between CDS or 5′-UTR lengths and relative expression levels in the brain but could not observe any significant correlation (<a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>). The lower correlation coefficient in these datasets was supposed to occur by a limited number of samples.</p>                        <a id="4674492" scrollto-destination="4674492"></a>
<div content-id="tbl4" class="table-modal"><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Statistical analysis of the relationship between tissue-specific expression and 3′-UTR length of the <em>mKIAA</em> cDNA clones.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissue </th><th>Kendall rank correlation (Tau)<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Probability value<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th></tr></thead><tbody><tr><td>Brain </td><td char=".">0.16 </td><td char=".">1.7 × 10<sup>−14</sup> </td></tr><tr><td>Colon </td><td char=".">−0.029 </td><td char=".">0.16 </td></tr><tr><td>Thymus </td><td char=".">−0.018 </td><td char=".">0.39 </td></tr><tr><td>Testis </td><td char=".">−0.067 </td><td char=".">0.0013 </td></tr><tr><td>Spleen </td><td char=".">−0.068 </td><td char=".">0.0010 </td></tr><tr><td>Heart </td><td char=".">−0.031 </td><td char=".">0.13 </td></tr><tr><td>Lung </td><td char=".">−0.0075 </td><td char=".">0.72 </td></tr><tr><td>Liver </td><td char=".">−0.054 </td><td char=".">0.0094 </td></tr><tr><td>Muscle </td><td char=".">−0.051 </td><td char=".">0.015 </td></tr><tr><td>Pancreas </td><td char=".">−0.012 </td><td char=".">0.57 </td></tr><tr><td>Kidney </td><td char=".">−0.016 </td><td char=".">0.45 </td></tr><tr><td>Stomach </td><td char=".">−0.021 </td><td char=".">0.30 </td></tr><tr><td>Prostate </td><td char=".">−0.0068 </td><td char=".">0.74 </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissue </th><th>Kendall rank correlation (Tau)<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Probability value<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th></tr></thead><tbody><tr><td>Brain </td><td char=".">0.16 </td><td char=".">1.7 × 10<sup>−14</sup> </td></tr><tr><td>Colon </td><td char=".">−0.029 </td><td char=".">0.16 </td></tr><tr><td>Thymus </td><td char=".">−0.018 </td><td char=".">0.39 </td></tr><tr><td>Testis </td><td char=".">−0.067 </td><td char=".">0.0013 </td></tr><tr><td>Spleen </td><td char=".">−0.068 </td><td char=".">0.0010 </td></tr><tr><td>Heart </td><td char=".">−0.031 </td><td char=".">0.13 </td></tr><tr><td>Lung </td><td char=".">−0.0075 </td><td char=".">0.72 </td></tr><tr><td>Liver </td><td char=".">−0.054 </td><td char=".">0.0094 </td></tr><tr><td>Muscle </td><td char=".">−0.051 </td><td char=".">0.015 </td></tr><tr><td>Pancreas </td><td char=".">−0.012 </td><td char=".">0.57 </td></tr><tr><td>Kidney </td><td char=".">−0.016 </td><td char=".">0.45 </td></tr><tr><td>Stomach </td><td char=".">−0.021 </td><td char=".">0.30 </td></tr><tr><td>Prostate </td><td char=".">−0.0068 </td><td char=".">0.74 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf4-1"><div class="fn"><p><sup>a</sup>We selected 1031 <em>mKIAA</em> cDNA clones that were verified for the integrity of the 3′-UTR structure. Our definition of the integrity is as follows: presence of canonical polyadenylation signal (AATAAA) or its single-nucleotide variants in the 35 bp upstream region of the 3′-extreme end; and absence of a stretch of &gt;10 adenines in the sequence adjacent to the corresponding to the 3′-extreme end on genome sequence. Correlation between length of 3′-UTR and relative expression levels in each tissue (the expression value of each tissue/the mean expression value of 13 tissues) were calculated using the Kendall rank correlation measurement.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674492" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Statistical analysis of the relationship between tissue-specific expression and 3′-UTR length of the <em>mKIAA</em> cDNA clones.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissue </th><th>Kendall rank correlation (Tau)<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Probability value<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th></tr></thead><tbody><tr><td>Brain </td><td char=".">0.16 </td><td char=".">1.7 × 10<sup>−14</sup> </td></tr><tr><td>Colon </td><td char=".">−0.029 </td><td char=".">0.16 </td></tr><tr><td>Thymus </td><td char=".">−0.018 </td><td char=".">0.39 </td></tr><tr><td>Testis </td><td char=".">−0.067 </td><td char=".">0.0013 </td></tr><tr><td>Spleen </td><td char=".">−0.068 </td><td char=".">0.0010 </td></tr><tr><td>Heart </td><td char=".">−0.031 </td><td char=".">0.13 </td></tr><tr><td>Lung </td><td char=".">−0.0075 </td><td char=".">0.72 </td></tr><tr><td>Liver </td><td char=".">−0.054 </td><td char=".">0.0094 </td></tr><tr><td>Muscle </td><td char=".">−0.051 </td><td char=".">0.015 </td></tr><tr><td>Pancreas </td><td char=".">−0.012 </td><td char=".">0.57 </td></tr><tr><td>Kidney </td><td char=".">−0.016 </td><td char=".">0.45 </td></tr><tr><td>Stomach </td><td char=".">−0.021 </td><td char=".">0.30 </td></tr><tr><td>Prostate </td><td char=".">−0.0068 </td><td char=".">0.74 </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissue </th><th>Kendall rank correlation (Tau)<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Probability value<a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th></tr></thead><tbody><tr><td>Brain </td><td char=".">0.16 </td><td char=".">1.7 × 10<sup>−14</sup> </td></tr><tr><td>Colon </td><td char=".">−0.029 </td><td char=".">0.16 </td></tr><tr><td>Thymus </td><td char=".">−0.018 </td><td char=".">0.39 </td></tr><tr><td>Testis </td><td char=".">−0.067 </td><td char=".">0.0013 </td></tr><tr><td>Spleen </td><td char=".">−0.068 </td><td char=".">0.0010 </td></tr><tr><td>Heart </td><td char=".">−0.031 </td><td char=".">0.13 </td></tr><tr><td>Lung </td><td char=".">−0.0075 </td><td char=".">0.72 </td></tr><tr><td>Liver </td><td char=".">−0.054 </td><td char=".">0.0094 </td></tr><tr><td>Muscle </td><td char=".">−0.051 </td><td char=".">0.015 </td></tr><tr><td>Pancreas </td><td char=".">−0.012 </td><td char=".">0.57 </td></tr><tr><td>Kidney </td><td char=".">−0.016 </td><td char=".">0.45 </td></tr><tr><td>Stomach </td><td char=".">−0.021 </td><td char=".">0.30 </td></tr><tr><td>Prostate </td><td char=".">−0.0068 </td><td char=".">0.74 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf4-1"><div class="fn"><p><sup>a</sup>We selected 1031 <em>mKIAA</em> cDNA clones that were verified for the integrity of the 3′-UTR structure. Our definition of the integrity is as follows: presence of canonical polyadenylation signal (AATAAA) or its single-nucleotide variants in the 35 bp upstream region of the 3′-extreme end; and absence of a stretch of &gt;10 adenines in the sequence adjacent to the corresponding to the 3′-extreme end on genome sequence. Correlation between length of 3′-UTR and relative expression levels in each tissue (the expression value of each tissue/the mean expression value of 13 tissues) were calculated using the Kendall rank correlation measurement.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674492" target="_blank">View Large</a></div></div>                        <a id="4674494" scrollto-destination="4674494"></a>
<div content-id="tbl5" class="table-modal"><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>The relative level of expression in brain was positively correlated with the length of 3′-UTR but not the length of CDS.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Region<a reveal-id="tf5-1" data-open="tf5-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Tissue<a reveal-id="tf5-2" data-open="tf5-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Selection rule<a reveal-id="tf5-1" data-open="tf5-1" class="link link-ref link-reveal xref-fn">c</a> of clones </th><th>Sample number </th><th>Kendall rank correlation (Tau) </th><th>Probability value </th></tr></thead><tbody><tr><td>3′-UTR length </td><td>Brain </td><td>3′-terminal </td><td>491 </td><td>0.14 </td><td>7.3 × 10<sup>−6</sup> </td></tr><tr><td>3′-UTR length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>0.14 </td><td>0.00048 </td></tr><tr><td>CDS length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>−0.056 </td><td>0.26 </td></tr><tr><td>5′-UTR length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>−0.029 </td><td>0.55 </td></tr><tr><td>Total length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>0.089 </td><td>0.072 </td></tr><tr><td>3′-UTR length </td><td>Other </td><td>3′-terminal </td><td>540 </td><td>0.13 </td><td>1.3 × 10<sup>−5</sup> </td></tr><tr><td>3′-UTR length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>0.12 </td><td>0.0089 </td></tr><tr><td>CDS length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.064 </td><td>0.16 </td></tr><tr><td>5′-UTR length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.029 </td><td>0.53 </td></tr><tr><td>Total length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.031 </td><td>0.49 </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>Region<a reveal-id="tf5-1" data-open="tf5-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Tissue<a reveal-id="tf5-2" data-open="tf5-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Selection rule<a reveal-id="tf5-1" data-open="tf5-1" class="link link-ref link-reveal xref-fn">c</a> of clones </th><th>Sample number </th><th>Kendall rank correlation (Tau) </th><th>Probability value </th></tr></thead><tbody><tr><td>3′-UTR length </td><td>Brain </td><td>3′-terminal </td><td>491 </td><td>0.14 </td><td>7.3 × 10<sup>−6</sup> </td></tr><tr><td>3′-UTR length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>0.14 </td><td>0.00048 </td></tr><tr><td>CDS length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>−0.056 </td><td>0.26 </td></tr><tr><td>5′-UTR length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>−0.029 </td><td>0.55 </td></tr><tr><td>Total length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>0.089 </td><td>0.072 </td></tr><tr><td>3′-UTR length </td><td>Other </td><td>3′-terminal </td><td>540 </td><td>0.13 </td><td>1.3 × 10<sup>−5</sup> </td></tr><tr><td>3′-UTR length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>0.12 </td><td>0.0089 </td></tr><tr><td>CDS length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.064 </td><td>0.16 </td></tr><tr><td>5′-UTR length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.029 </td><td>0.53 </td></tr><tr><td>Total length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.031 </td><td>0.49 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf5-1"><div class="fn"><p><sup>a,c</sup>We further selected 401 clones (184 clones from brain-derived library, 217 clones from other tissues-derived library) from 1031 clones in <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>, which were verified for the integrity of the CDS and the 5′-UTR structures. Our definition of the integrity is as follows, without warning for N-terminal truncation of the coding region by GeneMark analysis.<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> Correlation between lengths of 3′-UTR, CDS and 5′-UTR and relative expression levels in each tissue (the expression value of each tissue/the mean expression value of 13 tissues) were calculated using the Kendall rank correlation measurement.</p></div></div><div content-id="tf5-2"><div class="fn"><p><sup>b</sup>To exclude the bias against the sources of cDNA libraries, the clones were subdivided into two groups (brain-derived and other tissues-derived).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674494" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>The relative level of expression in brain was positively correlated with the length of 3′-UTR but not the length of CDS.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Region<a reveal-id="tf5-1" data-open="tf5-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Tissue<a reveal-id="tf5-2" data-open="tf5-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Selection rule<a reveal-id="tf5-1" data-open="tf5-1" class="link link-ref link-reveal xref-fn">c</a> of clones </th><th>Sample number </th><th>Kendall rank correlation (Tau) </th><th>Probability value </th></tr></thead><tbody><tr><td>3′-UTR length </td><td>Brain </td><td>3′-terminal </td><td>491 </td><td>0.14 </td><td>7.3 × 10<sup>−6</sup> </td></tr><tr><td>3′-UTR length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>0.14 </td><td>0.00048 </td></tr><tr><td>CDS length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>−0.056 </td><td>0.26 </td></tr><tr><td>5′-UTR length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>−0.029 </td><td>0.55 </td></tr><tr><td>Total length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>0.089 </td><td>0.072 </td></tr><tr><td>3′-UTR length </td><td>Other </td><td>3′-terminal </td><td>540 </td><td>0.13 </td><td>1.3 × 10<sup>−5</sup> </td></tr><tr><td>3′-UTR length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>0.12 </td><td>0.0089 </td></tr><tr><td>CDS length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.064 </td><td>0.16 </td></tr><tr><td>5′-UTR length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.029 </td><td>0.53 </td></tr><tr><td>Total length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.031 </td><td>0.49 </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>Region<a reveal-id="tf5-1" data-open="tf5-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Tissue<a reveal-id="tf5-2" data-open="tf5-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Selection rule<a reveal-id="tf5-1" data-open="tf5-1" class="link link-ref link-reveal xref-fn">c</a> of clones </th><th>Sample number </th><th>Kendall rank correlation (Tau) </th><th>Probability value </th></tr></thead><tbody><tr><td>3′-UTR length </td><td>Brain </td><td>3′-terminal </td><td>491 </td><td>0.14 </td><td>7.3 × 10<sup>−6</sup> </td></tr><tr><td>3′-UTR length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>0.14 </td><td>0.00048 </td></tr><tr><td>CDS length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>−0.056 </td><td>0.26 </td></tr><tr><td>5′-UTR length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>−0.029 </td><td>0.55 </td></tr><tr><td>Total length </td><td>Brain </td><td>5′- and 3′-terminal </td><td>184 </td><td>0.089 </td><td>0.072 </td></tr><tr><td>3′-UTR length </td><td>Other </td><td>3′-terminal </td><td>540 </td><td>0.13 </td><td>1.3 × 10<sup>−5</sup> </td></tr><tr><td>3′-UTR length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>0.12 </td><td>0.0089 </td></tr><tr><td>CDS length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.064 </td><td>0.16 </td></tr><tr><td>5′-UTR length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.029 </td><td>0.53 </td></tr><tr><td>Total length </td><td>Other </td><td>5′- and 3′-terminal </td><td>217 </td><td>−0.031 </td><td>0.49 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf5-1"><div class="fn"><p><sup>a,c</sup>We further selected 401 clones (184 clones from brain-derived library, 217 clones from other tissues-derived library) from 1031 clones in <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>, which were verified for the integrity of the CDS and the 5′-UTR structures. Our definition of the integrity is as follows, without warning for N-terminal truncation of the coding region by GeneMark analysis.<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> Correlation between lengths of 3′-UTR, CDS and 5′-UTR and relative expression levels in each tissue (the expression value of each tissue/the mean expression value of 13 tissues) were calculated using the Kendall rank correlation measurement.</p></div></div><div content-id="tf5-2"><div class="fn"><p><sup>b</sup>To exclude the bias against the sources of cDNA libraries, the clones were subdivided into two groups (brain-derived and other tissues-derived).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674494" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4674495 id="4674495" class="section-title" >3.5. Candidates for the cDNA elements effecting on the brain-specific expression of the genes</h3>
<p>To find the novel sequence elements in the 3′-UTRs associating with the high-level expression in brain, we searched highly conserved sequence elements between <em>mKIAA</em> ranked within 50 places and their orthologous <em>KIAA</em> cDNAs using MEME program (<a class="link link-uri" href="http://meme.sdsc.edu/meme/website/" target="_blank"></a>). We calculated the differences of the relative expression levels in brain between <em>mKIAA</em> with and without the elements, and selected 43 statistically significant (<em>P</em> &lt; 10<sup>−3</sup>) elements among 66 elements that were extracted by MEME program (<a reveal-id="tbl6" data-open="tbl6" class="link link-reveal link-table xref-fig">Table 6</a>). We then searched the elements against 3′-UTRs of non-<em>mKIAA</em> genes in UTRdb entries. Consequently, we identified seven elements at the 3′-UTRs in 18 non-<em>mKIAA</em> genes and the elements were conserved among several species (mouse, human, rat, rabbit and bovine). Although there is little information about their expression, the predominant expression in the brain of <em>vms-tm2</em> gene containing an element (TCTTTTGTTTTAAAAGAAGAAATAT) could be found in the previous report. This result was consistent with our result; further accumulation of expression data is necessary for statistical verification of functional significance of the elements.</p>                        <a id="4674497" scrollto-destination="4674497"></a>
<div content-id="tbl6" class="table-modal"><div class="table-wrap"><div id="tbl6" class="table-wrap-title" data-id="tbl6"><div class="label -label"><strong>Table 6</strong></div><div class="caption"><p>Sequence elements found in the 3′-UTR <em>mKIAA</em>/<em>KIAA</em> gene pairs showing relatively high expression in the brain.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Element </th><th>Consensus sequence </th><th colspan="2">Relative level of expression in brain </th><th><em>t</em>-test <em>P</em>-value </th><th>Number of <em>mKIAA</em> cDNA with the motif </th><th>Definition of non-<em>mKIAA</em> gene with the element </th><th>Species </th><th>High expression in brain </th><th>Reference </th></tr><tr><th> </th><th> </th><th colspan="2"><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th> </th><th> </th><th>with Mean ± SD </th><th>without Mean ± SD </th><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td>1 </td><td>CCATCACAAGAGAAGGAGTT </td><td>1.8 ± 1.2 </td><td>1.4 ± 0.9 </td><td>3.7 × 10<sup>−10</sup> </td><td>646 </td><td>FLJ44006 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>2 </td><td>AAGGAAACCATGTAAAGAGAAAGAG </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>2.7 × 10<sup>−10</sup> </td><td>485 </td><td>Thioredoxin domain containing 5 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>3 </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Hypothetical protein </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>KIAA0269 (mKIAA0269 was not yet cloned.) </td><td>Human </td><td>Yes </td><td><a class="link link-uri" href="http://www.kazusa.or.jp" target="_blank"></a> </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>YB2 </td><td>Human </td><td>Ubiquitous </td><td>(<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a>) </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>vms-tm2 single-pass transmembrane protein </td><td>Rat </td><td>Yes </td><td>(<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a>) </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Unknown IMAGE:6530404 </td><td>Rat </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Highly similar to <em>Homo sapiens</em> Sad1 </td><td>Mouse </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Protooncogene A-myb (MYBL2) </td><td>Mouse </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>DKFZp451P2311 hypothetical protein </td><td>Bovine </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>DKFZp566G1424 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Glutathione <em>S</em>-transferase A1 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Hypothetical protein LOC138046 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>4 </td><td>AATGCGCAGATTGAAATGCAATTAA </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>3.3 × 10<sup>−6</sup> </td><td>357 </td><td>Hypothetical protein </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>AATGCGCAGATTGAAATGCAATTAA </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>3.3 × 10<sup>−6</sup> </td><td>357 </td><td>Thioredoxin domain containing 5 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>5 </td><td>AAATATTACTGTGTATCTTA </td><td>1.7 ± 1.2 </td><td>1.4 ± 0.9 </td><td>1.0 × 10<sup>−5</sup> </td><td>821 </td><td>DKFZp686A05136 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>6 </td><td>CAGTCAGAAAGCAGGTGGGC </td><td>1.7 ± 1.2 </td><td>1.5 ± 0.9 </td><td>2.1 × 10<sup>−5</sup> </td><td>740 </td><td>Ring finger protein 130/goliath </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>7 </td><td>CCTTGAATCTCTGTGCAGTG </td><td>1.7 ± 1.1 </td><td>1.4 ± 1.0 </td><td>1.4 × 10<sup>−4</sup> </td><td>823 </td><td>RUSH-1beta </td><td>Rabbit </td><td>? </td><td>No </td></tr></tbody></table></div><div content-id="tbl6" class="table-modal"><table><thead name="thead" align=""><tr><th>Element </th><th>Consensus sequence </th><th colspan="2">Relative level of expression in brain </th><th><em>t</em>-test <em>P</em>-value </th><th>Number of <em>mKIAA</em> cDNA with the motif </th><th>Definition of non-<em>mKIAA</em> gene with the element </th><th>Species </th><th>High expression in brain </th><th>Reference </th></tr><tr><th> </th><th> </th><th colspan="2"><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th> </th><th> </th><th>with Mean ± SD </th><th>without Mean ± SD </th><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td>1 </td><td>CCATCACAAGAGAAGGAGTT </td><td>1.8 ± 1.2 </td><td>1.4 ± 0.9 </td><td>3.7 × 10<sup>−10</sup> </td><td>646 </td><td>FLJ44006 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>2 </td><td>AAGGAAACCATGTAAAGAGAAAGAG </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>2.7 × 10<sup>−10</sup> </td><td>485 </td><td>Thioredoxin domain containing 5 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>3 </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Hypothetical protein </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>KIAA0269 (mKIAA0269 was not yet cloned.) </td><td>Human </td><td>Yes </td><td><a class="link link-uri" href="http://www.kazusa.or.jp" target="_blank"></a> </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>YB2 </td><td>Human </td><td>Ubiquitous </td><td>(<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a>) </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>vms-tm2 single-pass transmembrane protein </td><td>Rat </td><td>Yes </td><td>(<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a>) </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Unknown IMAGE:6530404 </td><td>Rat </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Highly similar to <em>Homo sapiens</em> Sad1 </td><td>Mouse </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Protooncogene A-myb (MYBL2) </td><td>Mouse </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>DKFZp451P2311 hypothetical protein </td><td>Bovine </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>DKFZp566G1424 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Glutathione <em>S</em>-transferase A1 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Hypothetical protein LOC138046 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>4 </td><td>AATGCGCAGATTGAAATGCAATTAA </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>3.3 × 10<sup>−6</sup> </td><td>357 </td><td>Hypothetical protein </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>AATGCGCAGATTGAAATGCAATTAA </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>3.3 × 10<sup>−6</sup> </td><td>357 </td><td>Thioredoxin domain containing 5 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>5 </td><td>AAATATTACTGTGTATCTTA </td><td>1.7 ± 1.2 </td><td>1.4 ± 0.9 </td><td>1.0 × 10<sup>−5</sup> </td><td>821 </td><td>DKFZp686A05136 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>6 </td><td>CAGTCAGAAAGCAGGTGGGC </td><td>1.7 ± 1.2 </td><td>1.5 ± 0.9 </td><td>2.1 × 10<sup>−5</sup> </td><td>740 </td><td>Ring finger protein 130/goliath </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>7 </td><td>CCTTGAATCTCTGTGCAGTG </td><td>1.7 ± 1.1 </td><td>1.4 ± 1.0 </td><td>1.4 × 10<sup>−4</sup> </td><td>823 </td><td>RUSH-1beta </td><td>Rabbit </td><td>? </td><td>No </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674497" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl6" class="table-wrap-title" data-id="tbl6"><div class="label -label"><strong>Table 6</strong></div><div class="caption"><p>Sequence elements found in the 3′-UTR <em>mKIAA</em>/<em>KIAA</em> gene pairs showing relatively high expression in the brain.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Element </th><th>Consensus sequence </th><th colspan="2">Relative level of expression in brain </th><th><em>t</em>-test <em>P</em>-value </th><th>Number of <em>mKIAA</em> cDNA with the motif </th><th>Definition of non-<em>mKIAA</em> gene with the element </th><th>Species </th><th>High expression in brain </th><th>Reference </th></tr><tr><th> </th><th> </th><th colspan="2"><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th> </th><th> </th><th>with Mean ± SD </th><th>without Mean ± SD </th><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td>1 </td><td>CCATCACAAGAGAAGGAGTT </td><td>1.8 ± 1.2 </td><td>1.4 ± 0.9 </td><td>3.7 × 10<sup>−10</sup> </td><td>646 </td><td>FLJ44006 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>2 </td><td>AAGGAAACCATGTAAAGAGAAAGAG </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>2.7 × 10<sup>−10</sup> </td><td>485 </td><td>Thioredoxin domain containing 5 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>3 </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Hypothetical protein </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>KIAA0269 (mKIAA0269 was not yet cloned.) </td><td>Human </td><td>Yes </td><td><a class="link link-uri" href="http://www.kazusa.or.jp" target="_blank"></a> </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>YB2 </td><td>Human </td><td>Ubiquitous </td><td>(<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a>) </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>vms-tm2 single-pass transmembrane protein </td><td>Rat </td><td>Yes </td><td>(<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a>) </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Unknown IMAGE:6530404 </td><td>Rat </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Highly similar to <em>Homo sapiens</em> Sad1 </td><td>Mouse </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Protooncogene A-myb (MYBL2) </td><td>Mouse </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>DKFZp451P2311 hypothetical protein </td><td>Bovine </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>DKFZp566G1424 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Glutathione <em>S</em>-transferase A1 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Hypothetical protein LOC138046 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>4 </td><td>AATGCGCAGATTGAAATGCAATTAA </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>3.3 × 10<sup>−6</sup> </td><td>357 </td><td>Hypothetical protein </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>AATGCGCAGATTGAAATGCAATTAA </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>3.3 × 10<sup>−6</sup> </td><td>357 </td><td>Thioredoxin domain containing 5 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>5 </td><td>AAATATTACTGTGTATCTTA </td><td>1.7 ± 1.2 </td><td>1.4 ± 0.9 </td><td>1.0 × 10<sup>−5</sup> </td><td>821 </td><td>DKFZp686A05136 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>6 </td><td>CAGTCAGAAAGCAGGTGGGC </td><td>1.7 ± 1.2 </td><td>1.5 ± 0.9 </td><td>2.1 × 10<sup>−5</sup> </td><td>740 </td><td>Ring finger protein 130/goliath </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>7 </td><td>CCTTGAATCTCTGTGCAGTG </td><td>1.7 ± 1.1 </td><td>1.4 ± 1.0 </td><td>1.4 × 10<sup>−4</sup> </td><td>823 </td><td>RUSH-1beta </td><td>Rabbit </td><td>? </td><td>No </td></tr></tbody></table></div><div content-id="tbl6" class="table-modal"><table><thead name="thead" align=""><tr><th>Element </th><th>Consensus sequence </th><th colspan="2">Relative level of expression in brain </th><th><em>t</em>-test <em>P</em>-value </th><th>Number of <em>mKIAA</em> cDNA with the motif </th><th>Definition of non-<em>mKIAA</em> gene with the element </th><th>Species </th><th>High expression in brain </th><th>Reference </th></tr><tr><th> </th><th> </th><th colspan="2"><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th> </th><th> </th><th>with Mean ± SD </th><th>without Mean ± SD </th><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td>1 </td><td>CCATCACAAGAGAAGGAGTT </td><td>1.8 ± 1.2 </td><td>1.4 ± 0.9 </td><td>3.7 × 10<sup>−10</sup> </td><td>646 </td><td>FLJ44006 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>2 </td><td>AAGGAAACCATGTAAAGAGAAAGAG </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>2.7 × 10<sup>−10</sup> </td><td>485 </td><td>Thioredoxin domain containing 5 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>3 </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Hypothetical protein </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>KIAA0269 (mKIAA0269 was not yet cloned.) </td><td>Human </td><td>Yes </td><td><a class="link link-uri" href="http://www.kazusa.or.jp" target="_blank"></a> </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>YB2 </td><td>Human </td><td>Ubiquitous </td><td>(<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a>) </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>vms-tm2 single-pass transmembrane protein </td><td>Rat </td><td>Yes </td><td>(<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a>) </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Unknown IMAGE:6530404 </td><td>Rat </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Highly similar to <em>Homo sapiens</em> Sad1 </td><td>Mouse </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Protooncogene A-myb (MYBL2) </td><td>Mouse </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>DKFZp451P2311 hypothetical protein </td><td>Bovine </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>DKFZp566G1424 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Glutathione <em>S</em>-transferase A1 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>TCTTTTGTTTTAAAAGAAGAAATAT </td><td>1.8 ± 1.1 </td><td>1.4 ± 0.9 </td><td>2.9 × 10<sup>−7</sup> </td><td>767 </td><td>Hypothetical protein LOC138046 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>4 </td><td>AATGCGCAGATTGAAATGCAATTAA </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>3.3 × 10<sup>−6</sup> </td><td>357 </td><td>Hypothetical protein </td><td>Human </td><td>? </td><td>No </td></tr><tr><td> </td><td>AATGCGCAGATTGAAATGCAATTAA </td><td>1.9 ± 1.2 </td><td>1.5 ± 1.0 </td><td>3.3 × 10<sup>−6</sup> </td><td>357 </td><td>Thioredoxin domain containing 5 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>5 </td><td>AAATATTACTGTGTATCTTA </td><td>1.7 ± 1.2 </td><td>1.4 ± 0.9 </td><td>1.0 × 10<sup>−5</sup> </td><td>821 </td><td>DKFZp686A05136 </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>6 </td><td>CAGTCAGAAAGCAGGTGGGC </td><td>1.7 ± 1.2 </td><td>1.5 ± 0.9 </td><td>2.1 × 10<sup>−5</sup> </td><td>740 </td><td>Ring finger protein 130/goliath </td><td>Human </td><td>? </td><td>No </td></tr><tr><td>7 </td><td>CCTTGAATCTCTGTGCAGTG </td><td>1.7 ± 1.1 </td><td>1.4 ± 1.0 </td><td>1.4 × 10<sup>−4</sup> </td><td>823 </td><td>RUSH-1beta </td><td>Rabbit </td><td>? </td><td>No </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4674497" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=4674498 id="4674498" class="section-title" >4. Discussion</h2>
<p>To discover novel functional aspect of UTRs, we prepared 1479 <em>mKIAA</em>/<em>KIAA</em> cDNA pairs and verified the orthology from their chromosomal location and the evolutionary rate of dS. Using these mouse cDNAs, we identified a positive correlation between the 3′-UTR length and the relative expression levels in the brain. Paying attention to the function of each gene, for instance, it appears that genes involved in G-protein signaling and vesicle trafficking dominantly expressing in the brain tended to have long 3′-UTR (data not shown). It was already assumed that 3′-UTR length increases with evolutionary age and organism complexity.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> Moreover, molecular neuroscientists have empirically noted that mRNAs, the most abundant in the brain, have long 3′-UTR. Therefore, it seems reasonable to propose that the genes predominantly expressed in mammalian brain have evolved to have long 3′-UTR. Growing evidences of RNA-binding proteins also demonstrates the importance of various 3′-UTR elements in regulation of mRNA turnover at the posttranscriptional level.<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> These evidences and our result motivated us to find novel sequence elements in the 3′-UTR.</p><p>MEME is a web-based tool for discovering elements in a group of related DNA sequences, thus we applied this program to find the highly conserved sequence elements in <em>mKIAA</em>/<em>KIAA</em> cDNAs revealed relatively high expression in the brain. We found 43 statistically significant elements in <em>mKIAA</em>/<em>KIAA</em> cDNAs and identified 7 out of 43 elements in the 3′-UTRs of 18 non-<em>mKIAA</em> genes in UTRdb. Among the several elements in 3′-UTRs that regulate the mRNA levels posttranscriptionally, the novel elements in this study might be one of such kind of elements. Biochemical approaches as well as accumulation of the expression profile of non-<em>mKIAA</em> genes containing the elements may help the verification of the novel elements on the predominant expression in brain. Among those elements, adenylate uridylate-rich elements (AU-rich elements, AREs) are the best characterized elements in 3′-UTR, and posttranscriptionally regulate the cytoplasmic half-life of the mRNAs encoding various proteins that regulate cellular proliferation/differentiation and response to inflammatory and environmental stimuli.<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> Cytidine-rich 15-lipoxygenase differentiation control element (15-LOX DICE) is another well-characterized element in 3′-UTR and is a multifunctional <em>cis</em>-element found in numerous eukaryotic mRNAs.<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> Although there were many AREs and 15-LOX DICEs in <em>mKIAA</em> cDNAs, the numbers of the elements simply correlated with the 3′-UTR length but not obviously with the relative expression level in the brain (data not shown).</p><p>The elements involved in tissue, stage or cell-type specific expression of the genes had also been reported; however, the specific expression of the genes is partly due to the specific expression of certain RNA-binding proteins. For example, mRNA of membrane-bound IL-1R accessory protein expressing in a tissue-specific manner has several elements in the 3′-UTR and the stability is thought to be parallel to the expression of some RNA-binding protein.<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> Neurofilament-M (NF-M) expression is stage-specific and culminates at the most mature stages of axon development. This alteration is partly regulated by the NF-M mRNA stability, parallel with the binding of hnRNP to NF-M 3′-UTRs.<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> Especially in the brain, several RNA-binding proteins expressing in neuronal cell-type or stage-specific manner have been already identified; for instance, three ELAV-like proteins (HuB, HuC and HuD),<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> Musashi,<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> Autoimmune antigens Nova,<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> polypyrimidine tract-binding protein-like protein<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> and Drb1.<a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a> In the brain, posttranscriptional regulation might be more popular than any other organ and multiple RNA-binding proteins might act through the long 3′-UTR of target genes and control them in a complex manner. Since numerous RNA-binding proteins have been reported in particularly matured neuron in which the proteins might play important roles in terminal neuronal differentiation, perpetual neurite outgrowth/retraction and synaptogenesis. Since the long 3′-UTR of <em>mKIAA</em>/<em>KIAA</em> mRNAs potentially have multiple functional elements, the mRNA levels might be regulated by the combination of multiple brain-specific RNA-binding proteins. Accumulating information about sequence elements in mRNA, neuron-specific RNA-binding proteins and their interactions is promising to solve these complicated regulations.</p><p>We thank Nobue Kashima for her technical assistance. This project was supported by grants from the Kazusa DNA Research Institute. This study was also supported by the CREATE Program (Collaboration of Regional Entities for the Advancement of Technological Excellent) from JST (Japan Science and Technology Corporation).</p>                            <h2 scrollto-destination=4674505 id="4674505" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nomura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Miyajima</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sazuka</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Prediction of the coding sequences of unidentified human genes. I. The coding sequences of 40 new genes (KIAA0001-KIAA0040) deduced by analysis of randomly sampled cDNA clones from human immature myeloid cell line KG-1</div>, <div class="source">DNA Res.</div> , <div class="year">1994</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">27</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20the%20coding%20sequences%20of%20unidentified%20human%20genes.%20I.%20The%20coding%20sequences%20of%2040%20new%20genes%20(KIAA0001-KIAA0040)%20deduced%20by%20analysis%20of%20randomly%20sampled%20cDNA%20clones%20from%20human%20immature%20myeloid%20cell%20line%20KG-1&amp;author=N.%20Nomura&amp;author=N.%20Miyajima&amp;author=T.%20Sazuka&amp;publication_year=1994&amp;journal=DNA%20Res.&amp;volume=1&amp;pages=27-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/1.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F1.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F1.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7584026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kikuno</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Prediction of the coding sequences of mouse homologues of KIAA gene: I. The complete nucleotide sequences of 100 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries</div>, <div class="source">DNA Res.</div> , <div class="year">2002</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">179</div>-<div class="lpage">188</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20the%20coding%20sequences%20of%20mouse%20homologues%20of%20KIAA%20gene%3A%20I.%20The%20complete%20nucleotide%20sequences%20of%20100%20mouse%20KIAA-homologous%20cDNAs%20identified%20by%20screening%20of%20terminal%20sequences%20of%20cDNA%20clones%20randomly%20sampled%20from%20size-fractionated%20libraries&amp;author=N.%20Okazaki&amp;author=R.%20Kikuno&amp;author=R.%20Ohara&amp;publication_year=2002&amp;journal=DNA%20Res.&amp;volume=9&amp;pages=179-188" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.5.179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.5.179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.5.179"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12465718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kikuno</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Prediction of the coding sequences of mouse homologues of KIAA gene: II. The complete nucleotide sequences of 400 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries</div>, <div class="source">DNA Res.</div> , <div class="year">2003</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">35</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20the%20coding%20sequences%20of%20mouse%20homologues%20of%20KIAA%20gene%3A%20II.%20The%20complete%20nucleotide%20sequences%20of%20400%20mouse%20KIAA-homologous%20cDNAs%20identified%20by%20screening%20of%20terminal%20sequences%20of%20cDNA%20clones%20randomly%20sampled%20from%20size-fractionated%20libraries&amp;author=N.%20Okazaki&amp;author=R.%20Kikuno&amp;author=R.%20Ohara&amp;publication_year=2003&amp;journal=DNA%20Res.&amp;volume=10&amp;pages=35-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/10.1.35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F10.1.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F10.1.35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12693553" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kikuno</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Prediction of the coding sequences of mouse homologues of KIAA gene: III. the complete nucleotide sequences of 500 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries</div>, <div class="source">DNA Res.</div> , <div class="year">2003</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">167</div>-<div class="lpage">180</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20the%20coding%20sequences%20of%20mouse%20homologues%20of%20KIAA%20gene%3A%20III.%20the%20complete%20nucleotide%20sequences%20of%20500%20mouse%20KIAA-homologous%20cDNAs%20identified%20by%20screening%20of%20terminal%20sequences%20of%20cDNA%20clones%20randomly%20sampled%20from%20size-fractionated%20libraries&amp;author=N.%20Okazaki&amp;author=R.%20Kikuno&amp;author=R.%20Ohara&amp;publication_year=2003&amp;journal=DNA%20Res.&amp;volume=10&amp;pages=167-180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/10.4.167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F10.4.167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F10.4.167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14621295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">F-Kikuno</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Prediction of the coding sequences of mouse homologues of KIAA gene: IV. The complete nucleotide sequences of 500 mouse KIAA-homologous cDNAs identified by screening of terminal sequences of cDNA clones randomly sampled from size-fractionated libraries</div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">205</div>-<div class="lpage">218</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20the%20coding%20sequences%20of%20mouse%20homologues%20of%20KIAA%20gene%3A%20IV.%20The%20complete%20nucleotide%20sequences%20of%20500%20mouse%20KIAA-homologous%20cDNAs%20identified%20by%20screening%20of%20terminal%20sequences%20of%20cDNA%20clones%20randomly%20sampled%20from%20size-fractionated%20libraries&amp;author=N.%20Okazaki&amp;author=R.%20F-Kikuno&amp;author=R.%20Ohara&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=205-218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.3.205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.3.205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.3.205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15368895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koga</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yuasa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nagase</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">A comprehensive approach for establishment of the platform to analyze functions of KIAA proteins II: public release of inaugural version of InGaP database containing gene/protein expression profiles for 127 mouse KIAA genes/proteins</div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">293</div>-<div class="lpage">304</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20approach%20for%20establishment%20of%20the%20platform%20to%20analyze%20functions%20of%20KIAA%20proteins%20II%3A%20public%20release%20of%20inaugural%20version%20of%20InGaP%20database%20containing%20gene%2Fprotein%20expression%20profiles%20for%20127%20mouse%20KIAA%20genes%2Fproteins&amp;author=H.%20Koga&amp;author=S.%20Yuasa&amp;author=T.%20Nagase&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=293-304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.4.293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.4.293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.4.293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15500254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jackson</div> <div class="given-names">R. J.</div></div>. </div><div class="article-title">Cytoplasmic regulation of mRNA function: the importance of the 3′ untranslated region</div>, <div class="source">Cell</div> , <div class="year">1993</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">9</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytoplasmic%20regulation%20of%20mRNA%20function%3A%20the%20importance%20of%20the%203%E2%80%B2%20untranslated%20region&amp;author=R.%20J.%20Jackson&amp;publication_year=1993&amp;journal=Cell&amp;volume=74&amp;pages=9-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(93)90290-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(93)90290-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(93)90290-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7687524" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Curtis</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Lehmann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zamore</div> <div class="given-names">P. D.</div></div>. </div><div class="article-title">Translational regulation in development</div>, <div class="source">Cell</div> , <div class="year">1995</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">171</div>-<div class="lpage">178</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20regulation%20in%20development&amp;author=D.%20Curtis&amp;author=R.%20Lehmann&amp;author=P.%20D.%20Zamore&amp;publication_year=1995&amp;journal=Cell&amp;volume=81&amp;pages=171-178" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(95)90325-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(95)90325-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(95)90325-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7736569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hentze</div> <div class="given-names">M. W.</div></div>. </div><div class="article-title">Translational regulation: versatile mechanisms for metabolic and developmental control</div>, <div class="source">Curr. Opin. Cell Biol.</div> , <div class="year">1995</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">393</div>-<div class="lpage">398</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20regulation%3A%20versatile%20mechanisms%20for%20metabolic%20and%20developmental%20control&amp;author=M.%20W.%20Hentze&amp;publication_year=1995&amp;journal=Curr.%20Opin.%20Cell%20Biol.&amp;volume=7&amp;pages=393-398" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0955-0674(95)80095-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0955-0674(95)80095-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0955-0674(95)80095-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7662370" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wickens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">R. J.</div></div>. </div><div class="article-title">Life and death in the cytoplasm: messages from the 3′ end</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">1997</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">220</div>-<div class="lpage">232</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Life%20and%20death%20in%20the%20cytoplasm%3A%20messages%20from%20the%203%E2%80%B2%20end&amp;author=M.%20Wickens&amp;author=P.%20Anderson&amp;author=R.%20J.%20Jackson&amp;publication_year=1997&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=7&amp;pages=220-232" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(97)80132-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(97)80132-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(97)80132-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9115434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Siomi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Dreyfuss</div> <div class="given-names">G.</div></div>. </div><div class="article-title">RNA-binding proteins as regulators of gene expression</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">1997</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">345</div>-<div class="lpage">353</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-binding%20proteins%20as%20regulators%20of%20gene%20expression&amp;author=H.%20Siomi&amp;author=G.%20Dreyfuss&amp;publication_year=1997&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=7&amp;pages=345-353" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(97)80148-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(97)80148-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(97)80148-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9229110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Day</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Tuite</div> <div class="given-names">M. F.</div></div>. </div><div class="article-title">Post-transcriptional gene regulatory mechanisms in eukaryotes: an overview</div>, <div class="source">J. Endocrinol.</div> , <div class="year">1998</div>, vol. <div class="volume">157</div> (pg. <div class="fpage">361</div>-<div class="lpage">371</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Post-transcriptional%20gene%20regulatory%20mechanisms%20in%20eukaryotes%3A%20an%20overview&amp;author=D.%20A.%20Day&amp;author=M.%20F.%20Tuite&amp;publication_year=1998&amp;journal=J.%20Endocrinol.&amp;volume=157&amp;pages=361-371" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1677/joe.0.1570361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1677%2Fjoe.0.1570361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1677%2Fjoe.0.1570361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9691970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Derrigo</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cestelli</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Savettieri</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Di Liegro</div> <div class="given-names">I.</div></div>. </div><div class="article-title">RNA-protein interactions in the control of stability and localization of messenger RNA (review)</div>, <div class="source">Int. J. Mol. Med.</div> , <div class="year">2000</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">111</div>-<div class="lpage">123</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-protein%20interactions%20in%20the%20control%20of%20stability%20and%20localization%20of%20messenger%20RNA%20(review)&amp;author=M.%20Derrigo&amp;author=A.%20Cestelli&amp;author=G.%20Savettieri&amp;author=I.%20Di%20Liegro&amp;publication_year=2000&amp;journal=Int.%20J.%20Mol.%20Med.&amp;volume=5&amp;pages=111-123" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10639588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lindquist</div> <div class="given-names">J. N.</div></div>,  <div class="name"><div class="surname">Parsons</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Stefanovic</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">D. A.</div></div>. </div><div class="article-title">Regulation of alpha1(I) collagen messenger RNA decay by interactions with alphaCP at the 3′-untranslated region</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">23822</div>-<div class="lpage">23829</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20alpha1(I)%20collagen%20messenger%20RNA%20decay%20by%20interactions%20with%20alphaCP%20at%20the%203%E2%80%B2-untranslated%20region&amp;author=J.%20N.%20Lindquist&amp;author=C.%20J.%20Parsons&amp;author=B.%20Stefanovic&amp;author=D.%20A.%20Brenner&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=23822-23829" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M314060200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M314060200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M314060200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14973140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dombrowski</div> <div class="given-names">S. M.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">McEntyre</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Using the Map Viewer to explore genomes</div>, <div class="year">2002</div> <div class="comment">The NCBI Handbook [Internet]. National Library of Medicine (US), National Center for Biotechnology Information, Bethesda (MD)</div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1986</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">418</div>-<div class="lpage">426</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20methods%20for%20estimating%20the%20numbers%20of%20synonymous%20and%20nonsynonymous%20nucleotide%20substitutions&amp;author=M.%20Nei&amp;author=T.%20Gojobori&amp;publication_year=1986&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=3&amp;pages=418-426" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3444411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ganeshan</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dickover</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Korber</div> <div class="given-names">B. T.</div></div>,  <div class="name"><div class="surname">Bryson</div> <div class="given-names">Y. J.</div></div>,  <div class="name"><div class="surname">Wolinsky</div> <div class="given-names">S. M.</div></div>. </div><div class="article-title">Human immunodeficiency virus type 1 genetic evolution in children with different rates of development of disease</div>, <div class="source">J. Virol.</div> , <div class="year">1997</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">663</div>-<div class="lpage">677</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20immunodeficiency%20virus%20type%201%20genetic%20evolution%20in%20children%20with%20different%20rates%20of%20development%20of%20disease&amp;author=S.%20Ganeshan&amp;author=R.%20E.%20Dickover&amp;author=B.%20T.%20Korber&amp;author=Y.%20J.%20Bryson&amp;author=S.%20M.%20Wolinsky&amp;publication_year=1997&amp;journal=J.%20Virol.&amp;volume=71&amp;pages=663-677" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8985398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korber</div> <div class="given-names">B.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Rodrigo</div> <div class="given-names">A. G.</div></div>,  <div class="name"><div class="surname">Learn</div> <div class="given-names">G. H.</div></div>. </div><div class="article-title">HIV signature and sequence variation analysis</div>, <div class="year">2000</div><div class="publisher-loc">Dordrecht, The Netherlands</div><div class="publisher-name">Computational Analysis of HIV Molecular Sequences, Ch. 4. Kluwer Academic Publishers</div>(pg. <div class="fpage">55</div>-<div class="lpage">72</div>)</div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brenner</div> <div class="given-names">S. E.</div></div>,  <div class="name"><div class="surname">Chothia</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hubbard</div> <div class="given-names">T. J.</div></div>. </div><div class="article-title">Assessing sequence comparison methods with reliable structurally identified distant evolutionary relationships</div>, <div class="source">Proc. Natl Acad. Sci. U.S.A.</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">6073</div>-<div class="lpage">6078</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20sequence%20comparison%20methods%20with%20reliable%20structurally%20identified%20distant%20evolutionary%20relationships&amp;author=S.%20E.%20Brenner&amp;author=C.%20Chothia&amp;author=T.%20J.%20Hubbard&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.S.A.&amp;volume=95&amp;pages=6073-6078" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.11.6073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.11.6073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.11.6073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9600919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheeler</div> <div class="given-names">D. L.</div></div>,  <div class="name"><div class="surname">Church</div> <div class="given-names">D. M.</div></div>,  <div class="name"><div class="surname">Federhen</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Database resources of the National Center for Biotechnology</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">28</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology&amp;author=D.%20L.%20Wheeler&amp;author=D.%20M.%20Church&amp;author=S.%20Federhen&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=28-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519941" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hedges</div> <div class="given-names">S. B.</div></div>. </div><div class="article-title">A molecular timescale for vertebrate evolution</div>, <div class="source">Nature</div> , <div class="year">1998</div>, vol. <div class="volume">392</div> (pg. <div class="fpage">917</div>-<div class="lpage">920</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20timescale%20for%20vertebrate%20evolution&amp;author=S.%20Kumar&amp;author=S.%20B.%20Hedges&amp;publication_year=1998&amp;journal=Nature&amp;volume=392&amp;pages=917-920" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/31927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F31927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F31927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9582070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Makalowski</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Boguski</div> <div class="given-names">M. S.</div></div>. </div><div class="article-title">Evolutionary parameters of the transcribed mammalian genome: an analysis of 2820 orthologous rodent and human sequences</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A.</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">9407</div>-<div class="lpage">9412</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20parameters%20of%20the%20transcribed%20mammalian%20genome%3A%20an%20analysis%20of%202820%20orthologous%20rodent%20and%20human%20sequences&amp;author=W.%20Makalowski&amp;author=M.%20S.%20Boguski&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;volume=95&amp;pages=9407-9412" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.16.9407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.16.9407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.16.9407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9689093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">406</div>-<div class="lpage">425</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-425" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hurowitz</div> <div class="given-names">E. H.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div>. </div><div class="article-title">Genome-wide analysis of mRNA lengths in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Genome Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">5</div> pg. <div class="fpage">R2</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20mRNA%20lengths%20in%20Saccharomyces%20cerevisiae&amp;author=E.%20H.%20Hurowitz&amp;author=P.%20O.%20Brown&amp;publication_year=2003&amp;journal=Genome%20Biol.&amp;volume=5&amp;pages=R2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2003-5-1-r2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2003-5-1-r2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2003-5-1-r2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14709174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mazumder</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Seshadri</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Fox</div> <div class="given-names">P. L.</div></div>. </div><div class="article-title">Translational control by the 3′-UTR: the ends specify the means</div>, <div class="source">Trends Biochem. Sci.</div> , <div class="year">2003</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">91</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20control%20by%20the%203%E2%80%B2-UTR%3A%20the%20ends%20specify%20the%20means&amp;author=B.%20Mazumder&amp;author=V.%20Seshadri&amp;author=P.%20L.%20Fox&amp;publication_year=2003&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=28&amp;pages=91-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(03)00002-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(03)00002-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(03)00002-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12575997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grzybowska</div> <div class="given-names">E. A.</div></div>,  <div class="name"><div class="surname">Wilczynska</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Siedlecki</div> <div class="given-names">J. A.</div></div>. </div><div class="article-title">Regulatory functions of 3′UTRs</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2001</div>, vol. <div class="volume">288</div> (pg. <div class="fpage">291</div>-<div class="lpage">295</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20functions%20of%203%E2%80%B2UTRs&amp;author=E.%20A.%20Grzybowska&amp;author=A.%20Wilczynska&amp;author=J.%20A.%20Siedlecki&amp;publication_year=2001&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=288&amp;pages=291-295" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/bbrc.2001.5738" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fbbrc.2001.5738" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fbbrc.2001.5738"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11606041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bakheet</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Frevel</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">B. R.</div></div>,  <div class="name"><div class="surname">Greer</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Khabar</div> <div class="given-names">K. S.</div></div>. </div><div class="article-title">ARED: human AU-rich element-containing mRNA database reveals an unexpectedly diverse functional repertoire of encoded proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">246</div>-<div class="lpage">254</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ARED%3A%20human%20AU-rich%20element-containing%20mRNA%20database%20reveals%20an%20unexpectedly%20diverse%20functional%20repertoire%20of%20encoded%20proteins&amp;author=T.%20Bakheet&amp;author=M.%20Frevel&amp;author=B.%20R.%20Williams&amp;author=W.%20Greer&amp;author=K.%20S.%20Khabar&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=246-254" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.1.246" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.1.246" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.1.246"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11125104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reimann</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Huth</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Thiele</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Thiele</div> <div class="given-names">B. J.</div></div>. </div><div class="article-title">Suppression of 15-lipoxygenase synthesis by hnRNP E1 is dependent on repetitive nature of LOX mRNA 3′-UTR control element DICE</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">315</div> (pg. <div class="fpage">965</div>-<div class="lpage">974</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Suppression%20of%2015-lipoxygenase%20synthesis%20by%20hnRNP%20E1%20is%20dependent%20on%20repetitive%20nature%20of%20LOX%20mRNA%203%E2%80%B2-UTR%20control%20element%20DICE&amp;author=I.%20Reimann&amp;author=A.%20Huth&amp;author=H.%20Thiele&amp;author=B.%20J.%20Thiele&amp;publication_year=2002&amp;journal=J.%20Mol.%20Biol.&amp;volume=315&amp;pages=965-974" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2001.5315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2001.5315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2001.5315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11827469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jensen</div> <div class="given-names">L. E.</div></div>,  <div class="name"><div class="surname">Whitehead</div> <div class="given-names">A. S.</div></div>. </div><div class="article-title">The 3′ untranslated region of the membrane-bound IL-1R accessory protein mRNA confers tissue-specific destabilization</div>, <div class="source">J. Immunol.</div> , <div class="year">2004</div>, vol. <div class="volume">173</div> (pg. <div class="fpage">6248</div>-<div class="lpage">6258</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%203%E2%80%B2%20untranslated%20region%20of%20the%20membrane-bound%20IL-1R%20accessory%20protein%20mRNA%20confers%20tissue-specific%20destabilization&amp;author=L.%20E.%20Jensen&amp;author=A.%20S.%20Whitehead&amp;publication_year=2004&amp;journal=J.%20Immunol.&amp;volume=173&amp;pages=6248-6258" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4049/jimmunol.173.10.6248" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4049%2Fjimmunol.173.10.6248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4049%2Fjimmunol.173.10.6248"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15528363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thyagarajan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Szaro</div> <div class="given-names">B. G.</div></div>. </div><div class="article-title">Phylogenetically conserved binding of specific KH domain proteins to the 3′ untranslated region of the vertebrate middle neurofilament mRNA</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">49680</div>-<div class="lpage">49688</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetically%20conserved%20binding%20of%20specific%20KH%20domain%20proteins%20to%20the%203%E2%80%B2%20untranslated%20region%20of%20the%20vertebrate%20middle%20neurofilament%20mRNA&amp;author=A.%20Thyagarajan&amp;author=B.%20G.%20Szaro&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=49680-49688" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M408915200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M408915200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M408915200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15364910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Good</div> <div class="given-names">P. J.</div></div>. </div><div class="article-title">A conserved family of elav-like genes in vertebrates</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A.</div> , <div class="year">1995</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">4557</div>-<div class="lpage">4561</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20conserved%20family%20of%20elav-like%20genes%20in%20vertebrates&amp;author=P.%20J.%20Good&amp;publication_year=1995&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;volume=92&amp;pages=4557-4561" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.92.10.4557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.92.10.4557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.92.10.4557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7753842" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okano</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Imai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okabe</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Musashi: a translational regulator of cell fate</div>, <div class="source">J. Cell Sci.</div> , <div class="year">2002</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">1355</div>-<div class="lpage">1359</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Musashi%3A%20a%20translational%20regulator%20of%20cell%20fate&amp;author=H.%20Okano&amp;author=T.%20Imai&amp;author=M.%20Okabe&amp;publication_year=2002&amp;journal=J.%20Cell%20Sci.&amp;volume=115&amp;pages=1355-1359" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11896183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Musunuru</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Cell-specific RNA-binding proteins in human disease</div>, <div class="source">Trends Cardiovasc. Med.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">188</div>-<div class="lpage">195</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell-specific%20RNA-binding%20proteins%20in%20human%20disease&amp;author=K.%20Musunuru&amp;publication_year=2003&amp;journal=Trends%20Cardiovasc.%20Med.&amp;volume=13&amp;pages=188-195" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1050-1738(03)00075-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1050-1738(03)00075-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1050-1738(03)00075-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12837581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kikuchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ichikawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Arai</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Molecular cloning and characterization of a new neuron-specific homologue of rat polypyrimidine tract binding protein</div>, <div class="source">J. Biochem. (Tokyo)</div> , <div class="year">2000</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">811</div>-<div class="lpage">821</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20characterization%20of%20a%20new%20neuron-specific%20homologue%20of%20rat%20polypyrimidine%20tract%20binding%20protein&amp;author=T.%20Kikuchi&amp;author=M.%20Ichikawa&amp;author=J.%20Arai&amp;publication_year=2000&amp;journal=J.%20Biochem.%20(Tokyo)&amp;volume=128&amp;pages=811-821" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a022819" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jbchem.a022819" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jbchem.a022819"> </span></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamada</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sakashita</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Shimazaki</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">cDNA cloning and characterization of Drb1, a new member of RRM-type neural RNA-binding protein</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2002</div>, vol. <div class="volume">297</div> (pg. <div class="fpage">96</div>-<div class="lpage">104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=cDNA%20cloning%20and%20characterization%20of%20Drb1%2C%20a%20new%20member%20of%20RRM-type%20neural%20RNA-binding%20protein&amp;author=H.%20Tamada&amp;author=E.%20Sakashita&amp;author=K.%20Shimazaki&amp;publication_year=2002&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=297&amp;pages=96-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0006-291X(02)02132-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0006-291X(02)02132-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0006-291X(02)02132-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12220514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">McIninch</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Rudd</div> <div class="given-names">K. E.</div></div>,  <div class="name"><div class="surname">Medigue</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Detection of new genes in a bacterial genome using Markov models for three gene classes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">3554</div>-<div class="lpage">3562</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20new%20genes%20in%20a%20bacterial%20genome%20using%20Markov%20models%20for%20three%20gene%20classes&amp;author=M.%20Borodovsky&amp;author=J.%20D.%20McIninch&amp;author=E.%20V.%20Koonin&amp;author=K.%20E.%20Rudd&amp;author=C.%20Medigue&amp;author=A.%20Danchin&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=3554-3562" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.17.3554" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.17.3554" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.17.3554"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7567469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sapru</div> <div class="given-names">M. K.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Walke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Burmeister</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Goldman</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Cloning and characterization of a novel transcriptional repressor of the nicotinic acetylcholine receptor delta-subunit gene</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1996</div>, vol. <div class="volume">271</div> (pg. <div class="fpage">7203</div>-<div class="lpage">7211</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20a%20novel%20transcriptional%20repressor%20of%20the%20nicotinic%20acetylcholine%20receptor%20delta-subunit%20gene&amp;author=M.%20K.%20Sapru&amp;author=J.%20P.%20Gao&amp;author=W.%20Walke&amp;author=M.%20Burmeister&amp;author=D.%20Goldman&amp;publication_year=1996&amp;journal=J.%20Biol.%20Chem.&amp;volume=271&amp;pages=7203-7211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.271.12.7203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.271.12.7203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.271.12.7203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8636158" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shimokawa</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Jingu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Okada</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Miura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Molecular cloning of Rhombex-40 a transmembrane protein from the ventral medullary surface of the rat brain by differential display</div>, <div class="source">Life Sci.</div> , <div class="year">2000</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">2183</div>-<div class="lpage">2191</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20of%20Rhombex-40%20a%20transmembrane%20protein%20from%20the%20ventral%20medullary%20surface%20of%20the%20rat%20brain%20by%20differential%20display&amp;author=N.%20Shimokawa&amp;author=H.%20Jingu&amp;author=J.%20Okada&amp;author=M.%20Miura&amp;publication_year=2000&amp;journal=Life%20Sci.&amp;volume=66&amp;pages=2183-2191" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0024-3205(00)00545-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0024-3205(00)00545-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0024-3205(00)00545-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10834302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2005. Kazusa DNA Research Institute</div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=335862&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">95</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119500003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="335862"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119500003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f3%2f181%2f335862">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242119500003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F3%2F181%2F335862" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/335862'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Influence of the 3′-UTR-length of mKIAA cDNAs and their Sequence Features to the mRNA Expression Level in the Brain" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f3%2f181%2f335862">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

