

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/5/229/400632","@type":"ScholarlyArticle","name":"TISA: Tissue-specific Alternative Splicing in Human and Mouse Genes","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl011","keywords":["alternative splicing","genome-based clustering","splice graph","transcript variants","tissue-specific","gene ontology"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/5/229/400632","author":[{"name":"Noh, Seung-Jae","affiliation":"Bioinformatics Lab. Plant genomics center  KRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea","@type":"Person"},{"name":"Lee, Kyooyeol","affiliation":"Bioinformatics Lab. Plant genomics center  KRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea","@type":"Person"},{"name":"Paik, Hyojung","affiliation":"Bioinformatics Lab. Plant genomics center  KRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea","@type":"Person"},{"name":"Hur, Cheol-Goo","affiliation":"Bioinformatics Lab. Plant genomics center  KRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea","@type":"Person"}],"description":"Abstract.  Alternative splicing (AS) is a mechanism by which multiple transcripts are produced from a single gene and is thought to be an important mechanism fo","pageStart":"229","pageEnd":"243","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/5/0/m_dnaresearch13_5.cover.gif?Expires=2147483647&Signature=mud4uh-30-GlHhdYLUg7ju5UAULSV1q0NE0Wrnu3RkEfHq-bQsyIw-lCORVdItpQPaM1Yqh7Mlu1ZRFevXZzXbVfRNUVlJab8lK-GCnEhdWYhEGef0xZWaCkBQCPrWvFt7U3DQUW~N39pDu-W62zATjEI5H8weIBmZySoS7ld8JJHNoZ~KRFmJwMZ~vay8M9zJZLeiRO8~3Z29cer7cu1n7PWnH2IdAgSUN6NGoLNlAwxp8MJUVjiUx9vdSEaj5zwY9CII2P1JOFiUJHY8pRgIGGQLQdF1uHKlUcELu9skGdb8-~LToWZ5KjifePPIlv7WMkw03h8gvh~iZhhNJb6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"TISA: Tissue-specific Alternative Splicing in Human and Mouse Genes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/5/0/m_dnaresearch13_5.cover.gif?Expires=2147483647&Signature=mud4uh-30-GlHhdYLUg7ju5UAULSV1q0NE0Wrnu3RkEfHq-bQsyIw-lCORVdItpQPaM1Yqh7Mlu1ZRFevXZzXbVfRNUVlJab8lK-GCnEhdWYhEGef0xZWaCkBQCPrWvFt7U3DQUW~N39pDu-W62zATjEI5H8weIBmZySoS7ld8JJHNoZ~KRFmJwMZ~vay8M9zJZLeiRO8~3Z29cer7cu1n7PWnH2IdAgSUN6NGoLNlAwxp8MJUVjiUx9vdSEaj5zwY9CII2P1JOFiUJHY8pRgIGGQLQdF1uHKlUcELu9skGdb8-~LToWZ5KjifePPIlv7WMkw03h8gvh~iZhhNJb6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="TISA: Tissue-specific Alternative Splicing in Human and Mouse Genes" />
    <meta property="og:description" content="Abstract.  Alternative splicing (AS) is a mechanism by which multiple transcripts are produced from a single gene and is thought to be an important mechanism fo" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/5/229/400632" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Noh, Seung-Jae" /><meta name="citation_author_institution" content="Bioinformatics Lab. Plant genomics centerKRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea" /><meta name="citation_author" content="Lee, Kyooyeol" /><meta name="citation_author_institution" content="Bioinformatics Lab. Plant genomics centerKRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea" /><meta name="citation_author" content="Paik, Hyojung" /><meta name="citation_author_institution" content="Bioinformatics Lab. Plant genomics centerKRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea" /><meta name="citation_author" content="Hur, Cheol-Goo" /><meta name="citation_author_institution" content="Bioinformatics Lab. Plant genomics centerKRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea" /><meta name="citation_title" content="TISA: Tissue-specific Alternative Splicing in Human and Mouse Genes" /><meta name="citation_firstpage" content="229" /><meta name="citation_lastpage" content="243" /><meta name="citation_doi" content="10.1093/dnares/dsl011" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A genomic view of AS; Nature Genet.;  citation_year=2002;  citation_volume=30;  citation_pages=13-19; " /><meta name="citation_reference" content="citation_title=Gene structure prediction and AS analysis using genomically aligned ESTs; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=889-900; " /><meta name="citation_reference" content="citation_title=Genome-wide detection of alternative slicing in expressed sequences of human genes; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=2850-2859; " /><meta name="citation_reference" content="citation_title=ESTGenes: AS from ESTs in Ensembl; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=976-987; " /><meta name="citation_reference" content="citation_title=Alternative pre-mRNA splicing and proteome expansion in metazoans; Nature;  citation_year=2002;  citation_volume=418;  citation_pages=236-243; " /><meta name="citation_reference" content="citation_title=Signals and their transduction pathways regulating AS: a new dimension of the human genome; Hum. Mol. Genet.;  citation_year=2002;  citation_volume=11;  citation_pages=2409-2416; " /><meta name="citation_reference" content="citation_title=Multiple links between transcription and splicing; RNA;  citation_year=2004;  citation_volume=10;  citation_pages=1489-1498; " /><meta name="citation_reference" content="citation_title=Evidence for the widespread coupling of AS and nonsense-mediated mRNA decay in humans; Proc. Natl. Acad. Sci.USA;  citation_year=2003;  citation_volume=100;  citation_pages=189-192; " /><meta name="citation_reference" content="citation_title=Listening to silence and understanding nonsense: exonic mutations that affect splicing; Nature Rev.;  citation_year=2002;  citation_volume=3;  citation_pages=285-298; " /><meta name="citation_reference" content="citation_title=The relevance of alternative RNA splicing to pharmacogenomics; Trends Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=346-353; " /><meta name="citation_reference" content="citation_title=AS in disease and therapy; Nature Biotech.;  citation_year=2004;  citation_volume=22;  citation_pages=535-546; " /><meta name="citation_reference" content="citation_title=EST comparison indicates 38% of human mRNAs contain possible alternative splice forms; FEBS Lett.;  citation_year=2000;  citation_volume=474;  citation_pages=83-86; " /><meta name="citation_reference" content="citation_title=ASD: the AS Database; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D64-D69; " /><meta name="citation_reference" content="citation_title=Genome wide identification and classification of AS based on EST data; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=2579-2585; " /><meta name="citation_reference" content="citation_title=ProSplicer: a database of putative AS information derived from protein, mRNA and expressed sequence tag sequence data; Genome Biol.;  citation_year=2003;  citation_volume=4;  citation_pages=R29" /><meta name="citation_reference" content="citation_title=Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays; Science;  citation_year=2003;  citation_volume=302;  citation_pages=2141-2144; " /><meta name="citation_reference" content="citation_title=Revealing global regulatory features of mammalian AS using a quantitative microarray platform; Mol. Cell;  citation_year=2004;  citation_volume=16;  citation_pages=929-941; " /><meta name="citation_reference" content="citation_title=A microarray configuration to quantify expression levels and relative abundance of splice variants; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=e47" /><meta name="citation_reference" content="citation_title=HMM sampling and applications to gene finding and AS; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=ii36-ii41; " /><meta name="citation_reference" content="citation_title=Accurate identification of alternatively spliced exons using support vector machine; Bioinformatics;  citation_year=2004;  citation_volume=21;  citation_pages=897-901; " /><meta name="citation_reference" content="citation_title=RASE: recognition of alternatively spliced exons in C. elegans; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=i369-i377; " /><meta name="citation_reference" content="citation_title=Generating consensus sequences from partial order multiple sequence alignment graphs; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=999-1008; " /><meta name="citation_reference" content="citation_title=The AS Gallery (ASG): bridging the gap between genome and transcriptome; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=3977-3983; " /><meta name="citation_reference" content="citation_title=DEDB: a database of Drosophila melanogaster exons in splicing graph form; BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=189" /><meta name="citation_reference" content="citation_title=ECgene: Genome-based EST clustering and gene modeling for AS; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=566-576; " /><meta name="citation_reference" content="citation_title=The multiassembly problem: Reconstructing multiple transcript isoforms from EST fragment mixtures; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=426-441; " /><meta name="citation_reference" content="citation_title=Common themes in the function of transcription and splicing enhancers; Curr. Opin. Cell Biol.;  citation_year=1997;  citation_volume=9;  citation_pages=350-357; " /><meta name="citation_reference" content="citation_title=Finding signals that regulate AS in the post-genomic era; Genome Biol.;  citation_year=2002;  citation_volume=3;  citation_pages=0008" /><meta name="citation_reference" content="citation_title=Genome-wide detection of tissue-specific AS in the human transcriptome; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=3754-3766; " /><meta name="citation_reference" content="citation_title=GeneKeyDB: A lightweight, gene-centric, relational database to support data mining environments; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=72" /><meta name="citation_reference" content="citation_title=BLAT-The BLAST-Like Alignment Tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=565-664; " /><meta name="citation_reference" content="citation_title=Computer program for aligning a cDNA sequence with a genomic DNA sequence, Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=967-974; " /><meta name="citation_reference" content="citation_title=Selecting for functional alternative splices in ESTs; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1837-1845; " /><meta name="citation_reference" content="citation_title=The significance of digital gene expression profiles; Genome Res.;  citation_year=1997;  citation_volume=7;  citation_pages=986-995; " /><meta name="citation_reference" content="citation_title=The control of the false discovery rate in multiple hypotheses testing under dependency; Annal. Stat.;  citation_year=2001;  citation_volume=29;  citation_pages=1165-1188; " /><meta name="citation_reference" content="citation_title=GoMiner: a resource for biological interpretation of genomic and proteomic data; Genome Biol.;  citation_year=2003;  citation_volume=4;  citation_pages=R28" /><meta name="citation_reference" content="citation_title=A non-EST-based method for exon-skipping prediction; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1617-1623; " /><meta name="citation_reference" content="citation_title=Variation in AS across human tissues; Genome Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=R74" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/5/229/17462553/dsl011.pdf" /><meta name="description" content="Abstract.  Alternative splicing (AS) is a mechanism by which multiple transcripts are produced from a single gene and is thought to be an important mechanism fo" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/5/229/400632" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/5/229/400632" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/5/229/400632");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl011';
    </script>
<title>
	TISA: Tissue-specific Alternative Splicing in Human and Mouse Genes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./400632" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZdNPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOb2gsIFNldW5nLUphZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIExhYi4gUGxhbnQgZ2Vub21pY3MgY2VudGVyS1JJQkIsIDUyIEVvZXVuLWRvbmcsIFl1c2VvbmctZ3UsIERhZWpvbiwgMzA1LTMzMyBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxlZSwgS3lvb3llb2wiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9pbmZvcm1hdGljcyBMYWIuIFBsYW50IGdlbm9taWNzIGNlbnRlcktSSUJCLCA1MiBFb2V1bi1kb25nLCBZdXNlb25nLWd1LCBEYWVqb24sIDMwNS0zMzMgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYWlrLCBIeW9qdW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgTGFiLiBQbGFudCBnZW5vbWljcyBjZW50ZXJLUklCQiwgNTIgRW9ldW4tZG9uZywgWXVzZW9uZy1ndSwgRGFlam9uLCAzMDUtMzMzIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHVyLCBDaGVvbC1Hb28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9pbmZvcm1hdGljcyBMYWIuIFBsYW50IGdlbm9taWNzIGNlbnRlcktSSUJCLCA1MiBFb2V1bi1kb25nLCBZdXNlb25nLWd1LCBEYWVqb24sIDMwNS0zMzMgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlRJU0E6IFRpc3N1ZS1zcGVjaWZpYyBBbHRlcm5hdGl2ZSBTcGxpY2luZyBpbiBIdW1hbiBhbmQgTW91c2UgR2VuZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbDAxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA2LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5vbWljIHZpZXcgb2YgQVM7IE5hdHVyZSBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBzdHJ1Y3R1cmUgcHJlZGljdGlvbiBhbmQgQVMgYW5hbHlzaXMgdXNpbmcgZ2Vub21pY2FsbHkgYWxpZ25lZCBFU1RzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTg4OS05MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGRldGVjdGlvbiBvZiBhbHRlcm5hdGl2ZSBzbGljaW5nIGluIGV4cHJlc3NlZCBzZXF1ZW5jZXMgb2YgaHVtYW4gZ2VuZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI4NTAtMjg1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RVNUR2VuZXM6IEFTIGZyb20gRVNUcyBpbiBFbnNlbWJsOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTk3Ni05ODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHByZS1tUk5BIHNwbGljaW5nIGFuZCBwcm90ZW9tZSBleHBhbnNpb24gaW4gbWV0YXpvYW5zOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE4OyAgY2l0YXRpb25fcGFnZXM9MjM2LTI0MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2lnbmFscyBhbmQgdGhlaXIgdHJhbnNkdWN0aW9uIHBhdGh3YXlzIHJlZ3VsYXRpbmcgQVM6IGEgbmV3IGRpbWVuc2lvbiBvZiB0aGUgaHVtYW4gZ2Vub21lOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjQwOS0yNDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBsaW5rcyBiZXR3ZWVuIHRyYW5zY3JpcHRpb24gYW5kIHNwbGljaW5nOyBSTkE7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNDg5LTE0OTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciB0aGUgd2lkZXNwcmVhZCBjb3VwbGluZyBvZiBBUyBhbmQgbm9uc2Vuc2UtbWVkaWF0ZWQgbVJOQSBkZWNheSBpbiBodW1hbnM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS5VU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MTg5LTE5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlzdGVuaW5nIHRvIHNpbGVuY2UgYW5kIHVuZGVyc3RhbmRpbmcgbm9uc2Vuc2U6IGV4b25pYyBtdXRhdGlvbnMgdGhhdCBhZmZlY3Qgc3BsaWNpbmc7IE5hdHVyZSBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yODUtMjk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVsZXZhbmNlIG9mIGFsdGVybmF0aXZlIFJOQSBzcGxpY2luZyB0byBwaGFybWFjb2dlbm9taWNzOyBUcmVuZHMgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNDYtMzUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUyBpbiBkaXNlYXNlIGFuZCB0aGVyYXB5OyBOYXR1cmUgQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz01MzUtNTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FU1QgY29tcGFyaXNvbiBpbmRpY2F0ZXMgMzglIG9mIGh1bWFuIG1STkFzIGNvbnRhaW4gcG9zc2libGUgYWx0ZXJuYXRpdmUgc3BsaWNlIGZvcm1zOyBGRUJTIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQ3NDsgIGNpdGF0aW9uX3BhZ2VzPTgzLTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BU0Q6IHRoZSBBUyBEYXRhYmFzZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9RDY0LUQ2OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHdpZGUgaWRlbnRpZmljYXRpb24gYW5kIGNsYXNzaWZpY2F0aW9uIG9mIEFTIGJhc2VkIG9uIEVTVCBkYXRhOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI1NzktMjU4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvU3BsaWNlcjogYSBkYXRhYmFzZSBvZiBwdXRhdGl2ZSBBUyBpbmZvcm1hdGlvbiBkZXJpdmVkIGZyb20gcHJvdGVpbiwgbVJOQSBhbmQgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZyBzZXF1ZW5jZSBkYXRhOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPVIyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHN1cnZleSBvZiBodW1hbiBhbHRlcm5hdGl2ZSBwcmUtbVJOQSBzcGxpY2luZyB3aXRoIGV4b24ganVuY3Rpb24gbWljcm9hcnJheXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzAyOyAgY2l0YXRpb25fcGFnZXM9MjE0MS0yMTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXZlYWxpbmcgZ2xvYmFsIHJlZ3VsYXRvcnkgZmVhdHVyZXMgb2YgbWFtbWFsaWFuIEFTIHVzaW5nIGEgcXVhbnRpdGF0aXZlIG1pY3JvYXJyYXkgcGxhdGZvcm07IE1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTkyOS05NDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9hcnJheSBjb25maWd1cmF0aW9uIHRvIHF1YW50aWZ5IGV4cHJlc3Npb24gbGV2ZWxzIGFuZCByZWxhdGl2ZSBhYnVuZGFuY2Ugb2Ygc3BsaWNlIHZhcmlhbnRzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz1lNDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ITU0gc2FtcGxpbmcgYW5kIGFwcGxpY2F0aW9ucyB0byBnZW5lIGZpbmRpbmcgYW5kIEFTOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPWlpMzYtaWk0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgaWRlbnRpZmljYXRpb24gb2YgYWx0ZXJuYXRpdmVseSBzcGxpY2VkIGV4b25zIHVzaW5nIHN1cHBvcnQgdmVjdG9yIG1hY2hpbmU7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9ODk3LTkwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UkFTRTogcmVjb2duaXRpb24gb2YgYWx0ZXJuYXRpdmVseSBzcGxpY2VkIGV4b25zIGluIEMuIGVsZWdhbnM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9aTM2OS1pMzc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmF0aW5nIGNvbnNlbnN1cyBzZXF1ZW5jZXMgZnJvbSBwYXJ0aWFsIG9yZGVyIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBncmFwaHM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9OTk5LTEwMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBUyBHYWxsZXJ5IChBU0cpOiBicmlkZ2luZyB0aGUgZ2FwIGJldHdlZW4gZ2Vub21lIGFuZCB0cmFuc2NyaXB0b21lOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0zOTc3LTM5ODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURFREI6IGEgZGF0YWJhc2Ugb2YgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgZXhvbnMgaW4gc3BsaWNpbmcgZ3JhcGggZm9ybTsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xODkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FQ2dlbmU6IEdlbm9tZS1iYXNlZCBFU1QgY2x1c3RlcmluZyBhbmQgZ2VuZSBtb2RlbGluZyBmb3IgQVM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NTY2LTU3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG11bHRpYXNzZW1ibHkgcHJvYmxlbTogUmVjb25zdHJ1Y3RpbmcgbXVsdGlwbGUgdHJhbnNjcmlwdCBpc29mb3JtcyBmcm9tIEVTVCBmcmFnbWVudCBtaXh0dXJlczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00MjYtNDQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21tb24gdGhlbWVzIGluIHRoZSBmdW5jdGlvbiBvZiB0cmFuc2NyaXB0aW9uIGFuZCBzcGxpY2luZyBlbmhhbmNlcnM7IEN1cnIuIE9waW4uIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM1MC0zNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmRpbmcgc2lnbmFscyB0aGF0IHJlZ3VsYXRlIEFTIGluIHRoZSBwb3N0LWdlbm9taWMgZXJhOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTAwMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBkZXRlY3Rpb24gb2YgdGlzc3VlLXNwZWNpZmljIEFTIGluIHRoZSBodW1hbiB0cmFuc2NyaXB0b21lOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zNzU0LTM3NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVLZXlEQjogQSBsaWdodHdlaWdodCwgZ2VuZS1jZW50cmljLCByZWxhdGlvbmFsIGRhdGFiYXNlIHRvIHN1cHBvcnQgZGF0YSBtaW5pbmcgZW52aXJvbm1lbnRzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkxBVC1UaGUgQkxBU1QtTGlrZSBBbGlnbm1lbnQgVG9vbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz01NjUtNjY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRlciBwcm9ncmFtIGZvciBhbGlnbmluZyBhIGNETkEgc2VxdWVuY2Ugd2l0aCBhIGdlbm9taWMgRE5BIHNlcXVlbmNlLCBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9OTY3LTk3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW5nIGZvciBmdW5jdGlvbmFsIGFsdGVybmF0aXZlIHNwbGljZXMgaW4gRVNUczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xODM3LTE4NDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzaWduaWZpY2FuY2Ugb2YgZGlnaXRhbCBnZW5lIGV4cHJlc3Npb24gcHJvZmlsZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz05ODYtOTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29udHJvbCBvZiB0aGUgZmFsc2UgZGlzY292ZXJ5IHJhdGUgaW4gbXVsdGlwbGUgaHlwb3RoZXNlcyB0ZXN0aW5nIHVuZGVyIGRlcGVuZGVuY3k7IEFubmFsLiBTdGF0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTExNjUtMTE4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R29NaW5lcjogYSByZXNvdXJjZSBmb3IgYmlvbG9naWNhbCBpbnRlcnByZXRhdGlvbiBvZiBnZW5vbWljIGFuZCBwcm90ZW9taWMgZGF0YTsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1SMjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vbi1FU1QtYmFzZWQgbWV0aG9kIGZvciBleG9uLXNraXBwaW5nIHByZWRpY3Rpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTYxNy0xNjIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gaW4gQVMgYWNyb3NzIGh1bWFuIHRpc3N1ZXM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9Ujc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzEzLzUvMjI5LzE3NDYyNTUzL2RzbDAxMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQWx0ZXJuYXRpdmUgc3BsaWNpbmcgKEFTKSBpcyBhIG1lY2hhbmlzbSBieSB3aGljaCBtdWx0aXBsZSB0cmFuc2NyaXB0cyBhcmUgcHJvZHVjZWQgZnJvbSBhIHNpbmdsZSBnZW5lIGFuZCBpcyB0aG91Z2h0IHRvIGJlIGFuIGltcG9ydGFudCBtZWNoYW5pc20gZm8iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQwMDYzMiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQNMnnAa+zQ7Ut1DL9lBTBYJPhUCQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f5%2f229%2f400632" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f5%2f229%2f400632" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15658" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/5/0/m_dnaresearch13_5.cover.gif?Expires=2147483647&amp;Signature=mud4uh-30-GlHhdYLUg7ju5UAULSV1q0NE0Wrnu3RkEfHq-bQsyIw-lCORVdItpQPaM1Yqh7Mlu1ZRFevXZzXbVfRNUVlJab8lK-GCnEhdWYhEGef0xZWaCkBQCPrWvFt7U3DQUW~N39pDu-W62zATjEI5H8weIBmZySoS7ld8JJHNoZ~KRFmJwMZ~vay8M9zJZLeiRO8~3Z29cer7cu1n7PWnH2IdAgSUN6NGoLNlAwxp8MJUVjiUx9vdSEaj5zwY9CII2P1JOFiUJHY8pRgIGGQLQdF1uHKlUcELu9skGdb8-~LToWZ5KjifePPIlv7WMkw03h8gvh~iZhhNJb6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/5">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87194759">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194759">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194762">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194762">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194773">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194773">2 Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194821">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194821">3 Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194859">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194859">REFERENCES</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/13/5/205/401161">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">TISA: Tissue-specific Alternative Splicing in Human and Mouse Genes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seung-Jae Noh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioinformatics Lab. Plant genomics center</div><div class="addr-line">KRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seung-Jae+Noh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Noh S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Noh S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seung-Jae Noh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyooyeol Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioinformatics Lab. Plant genomics center</div><div class="addr-line">KRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyooyeol+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kyooyeol Lee</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hyojung Paik</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioinformatics Lab. Plant genomics center</div><div class="addr-line">KRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hyojung+Paik">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Paik H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Paik H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hyojung Paik</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Cheol-Goo Hur</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioinformatics Lab. Plant genomics center</div><div class="addr-line">KRIBB, 52 Eoeun-dong, Yuseong-gu, Daejon, 305-333 Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel: +82-42-879-8560, Fax: +82-42-879-8569, E-mail: <a href="mailto:hurlee@kribb.re.kr" target="_blank">hurlee@kribb.re.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Cheol-Goo+Hur">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hur C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hur C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Cheol-Goo Hur</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 5, 1 January 2006, Pages 229–243, <a href='https://doi.org/10.1093/dnares/dsl011'>https://doi.org/10.1093/dnares/dsl011</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 July 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">15 October 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="400632" href="/dnaresearch/article-pdf/13/5/229/17462553/dsl011.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Seung-Jae Noh, Kyooyeol Lee, Hyojung Paik, Cheol-Goo Hur;  TISA: Tissue-specific Alternative Splicing in Human and Mouse Genes, <em>DNA Research</em>, Volume 13, Issue 5, 1 January 2006, Pages 229–243, <a href='https://doi.org/10.1093/dnares/dsl011'>https://doi.org/10.1093/dnares/dsl011</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/400632?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/400632?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/400632?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/400632?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/400632?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/400632?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/400632?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=TISA%3A%20Tissue-specific%20Alternative%20Splicing%20in%20Human%20and%20Mouse%20Genes&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=5&amp;author=Noh%2C%20Seung-Jae%3B%20Lee%2C%20Kyooyeol&amp;startPage=229&amp;endPage=243&amp;contentId=10.1093%2Fdnares%2Fdsl011&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f5%2f229%2f400632" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87194759 id="87194759" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alternative splicing (AS) is a mechanism by which multiple transcripts are produced from a single gene and is thought to be an important mechanism for tissue-specific expression of transcript isoforms. Here, we report a novel graphing method for transcript reconstruction and statistical prediction of tissue-specific AS. We applied three selection steps to generate the splice graph and predict the transcript isoforms: (i) a custom scoring rule for exon/intron sets, (ii) binomial statistics for selecting valid alternative splicing with a frequency of at least 1% for the predominant form and (iii) evaluation of transcript structure. We obtained 97 286 and 66 022 valid transcripts from 26 143 human and 27 741 mouse genes, respectively. In addition, we discovered 33 481 AS events for nine types of AS patterns in human. The statistical significance of tissue specificity for each gene, transcript and AS event was assessed based on EST tissue information, followed by a multiple testing correction procedure. In human, 12 711 genes, 16 016 transcripts and 1035 AS events were predicted to be tissue-specific (false discovery rate &lt;0.01). This information on genes, transcript structures, AS events and their tissue specificities in human and mouse are freely accessible on the TISA website (<a class="link link-uri" href="http://tisa.kribb.re.kr/AGC/" target="_blank"></a>).</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome-based clustering&quot;">genome-based clustering</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;splice graph&quot;">splice graph</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcript variants&quot;">transcript variants</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="tissue-specific">tissue-specific</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene ontology&quot;">gene ontology</a></div>                            <h2 scrollto-destination=87194762 id="87194762" class="section-title" >1. Introduction</h2>
<p>Alternative pre-mRNA splicing (AS) is a biological mechanism for producing multiple transcripts from a single gene. Genomic research has revealed that AS occurs in 30–60% of human genes.<a reveal-id="b1  b4" data-open="b1  b4" class="link link-ref link-reveal xref-bibr">1–4</a> Due to AS, higher eukaryotes can generate a number of protein-encoding transcripts from a single gene, thereby increasing the diversity of their proteomes.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> Although proteins from one gene have the same or similar functions, they can have different characteristics, such as their binding properties, intracellular localization, enzymatic activity, regulation and stability.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> Thus, knowledge of AS is important for understanding the complex biological systems in higher eukaryotes. Furthermore, AS in conjunction with transcriptional regulation determines tissue-specific expression of transcript isoforms of a gene.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> One of the effects of AS is to control of RNA isoform stability by nonsense-mediated decay.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> AS is of interest to pharmaceutical research because aberrant AS can lead to various genetic diseases and cancers.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a></p><p>Several different strategies have been applied to AS analysis, including (i) EST mapping against mRNA<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a>, (ii) mRNA/EST/protein mapping to the genome<a reveal-id="b2  b4" data-open="b2  b4" class="link link-ref link-reveal xref-bibr">2–4</a>,<a reveal-id="b13  b15" data-open="b13  b15" class="link link-ref link-reveal xref-bibr">13–15</a>, (iii) splicing microarray analysis<a reveal-id="b16  b18" data-open="b16  b18" class="link link-ref link-reveal xref-bibr">16–18</a> and (iv) <em>ab initio</em> machine learning approaches.<a reveal-id="b19  b21" data-open="b19  b21" class="link link-ref link-reveal xref-bibr">19–21</a> Among these methods, genome-based mRNA/EST alignment has been an important methodology for <em>in silico</em> detection of AS because ESTs and mRNAs are the most abundant resources and are considered direct evidence of the expressed genes. A number of algorithms to analyse AS using genomic alignment of expressed sequences have been reported. Kan et al.<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> classified links between exons within a gene according to four types of connectivity, represented them as a matrix, and then constructed transcripts using matrix analysis to identify the paths that can be connected.</p><p>Recently, several studies have suggested graphical methods to represent gene structure.<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="b22  b25" data-open="b22  b25" class="link link-ref link-reveal xref-bibr">22–25</a> In these methods, exons and introns are verified from structural information obtained by genomic mapping, and splice graphs are constructed in which exons and introns constitute nodes and edges, respectively; however, generating all possible isoforms by graph traversal can produce many false transcripts. Lee's group<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> tried to overcome this problem by using a dynamic programming method. They suggested use of the heaviest bundling algorithm, in which a transcript with only partial alignment information is extended to the maximum length of the path with the maximum likelihood. On the other hand, in ECgene, Kim et al. discriminated the isoforms into three groups by considering the minimum set of sequence data that indicates the reliability of the corresponding transcript.</p><p>AS makes it possible to generate genetic products with different functions from a single gene. This process is controlled by numerous splicing regulatory factors, including serine/arginine-rich proteins that can bind to cognate sequence elements in mRNA precursor such as exonic splicing enhancer, exonic splicing silencer, intron splicing enhancer and intronic splicing silencer sequences, and regulate the splice site selections.<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> The outcome of the AS process can vary depending on several factors, such as the tissue, the developmental stage or the environmental conditions.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> Therefore, global studies of condition-specific AS are required to precisely understand the function of genes. Xu et al.<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> examined tissue-specific AS using Bayesian statistics and EST tissue information, which focuses on only single AS events rather than on the transcript levels. Because each transcript may be the result of many sequential AS events, it is difficult to correlate the tissue specificity of an individual AS event with that of the transcript. In this study, we provide a methodology and results for reconstructing and classifying transcript variants from a splice graph that contains only valid AS sets according to binomial statistics (i.e. an observed frequency of &gt;1% with a 95% confidence level). Moreover, we provide global information on the tissue specificity of transcript isoforms and the distinct AS events according to the organizational information from human and mouse ESTs.</p>                            <h2 scrollto-destination=87194773 id="87194773" class="section-title" >2 Materials and Methods</h2>
                            <h3 scrollto-destination=87194774 id="87194774" class="section-title" >2.1 Data preparation</h3>
<p>Human and mouse mRNAs deposited in GenBank (2006 June version) were retrieved using the following search term: ‘<em>human[Organism] AND biomol mRNA [Properties] NOT gbdiv est[Properties]’</em>. More than 7.5 million human ESTs and ∼4.6 million mouse ESTs were collected from dbEST (2006 June version). Repeat-masked genomic contig sequences of human and mouse were obtained from NCBI (Build 36 for human and mouse). The cDNA library information for human and mouse was downloaded from the Cancer Genome Anatomy Project (CGAP) website (<a class="link link-uri" href="http://cgap.nci.nih.gov/" target="_blank"></a>). Using CGAP library information, we classified all ESTs from 8616 human cDNA libraries into 46 tissues, except for ‘embryonic tissues’, ‘head and neck’, ‘pooled tissue’, ‘uncharacterized tissue’ and ‘whole body’. We also separated the histological status of the ESTs into ‘normal’, ‘cancer’, or ‘unknown’ according to the CGAP histology classification. We used only ESTs from the ‘normal’ library for analysis of tissue specificity. Gene annotation including gene ontology (GO) assignments was performed using GeneKeyDB.<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a></p>                            <h3 scrollto-destination=87194776 id="87194776" class="section-title" >2.2 Pre-processing and genomic mapping</h3>
<p>We generated a non-redundant set of contigs with known chromosomal locations and orientations and used it as a template for genomic alignment. PolyA stretches of mRNA/ESTs were removed, and the EST sequences were trimmed by 20 nt at both ends as previously described.<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> The presence of a polyA tail or polyA addition signal(s) for each query was recorded and used later for the detection of the gene terminus. We prepared a non-redundant set of mRNAs by running the ‘nrdb’ program. Expressed sequences longer than 100 nt were positioned in the genome by homology-based comparison using BLAT<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> and sim4<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a>. After fast alignment using BLAT, top five best matches for a query were refined by sim4 alignment using the genomic region including 10kb flanking regions at both ends of the original BLAT-mapped position. We saved only sequences with &gt;60% coverage and &gt;92% identity with ESTs or &gt;96% identity with mRNAs (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). If several genomic loci satisfied the criteria for a single query, the best match was regarded as the proper alignment.</p>                        <a id="87194780" scrollto-destination="87194780"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f1.jpeg?Expires=2147483647&Signature=nyIGBggr8n-K5ukXpLKTXqMTELE5VE3qAflUdk2CAzYTC46R8YseXaOaPhQUFMRP38gSqp~lgqfnWA4NnnTB2c3hAfLL3BOSrf~c6MBdU54RVhgN0WOW9AmrBiYh12mFHqZVbn-23r48SzLFUI76mUxnprKGvIijOYeFRr7afEGOkQ8FlJGMmldvsclrXHunXgxUdF6mAcy-YISYQXyVRzFCYqNbfmcsCnMJWbHnojoc1EjO-D0~0uHJyijMWxOjCgQai8Tq5-EkINNeD-f0RuF1c3OdWgr775OxLthbn-XIkCl2u32Ip7b09WDKXBX2u02q2aYnrMIC-d-ElGnBmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow chart of the identification of the TISA. The results for each step of the human TISA analysis are shown." path-from-xml="dsl011f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194780/dsl011f1.jpeg" path-from-xml="dsl011f1.jpeg" target="_blank">View large</a><a section="87194780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f1.jpeg?Expires=2147483647&Signature=K82rr9arZKsO~1GNCP1Pdn2s0ICp4wV1N-cF2CQZhPljX16Yae3OwAql2rAQ9mVfl97Kryw7S0ILxeECwlHQ7NjjXyAAvDoWTUW6O2d6QRrfgWIyGXIdA4~yUO-6fJlCxi1Qudn9lHq~JQuwbGlaXUhO8qxpZkxVTvfxpJ7CEzAw7YtnF5CpAlHAyDX4U7DXNhHmq72xF3ChG~zBXL5yBtoEnnHi7DQXqUCtW07KvHJP1yrfehUHsvcTM2F~zEwfP2~0fDfG0wuokqwa-Z4OMDCC4aW0M0zphcEpOzl~cXFpy9kPbLViqFNk8kuIp7Jf7mjVQB9O7r0MexyyE7wkMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194780&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow chart of the identification of the TISA. The results for each step of the human TISA analysis are shown.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f1.jpeg?Expires=2147483647&Signature=nyIGBggr8n-K5ukXpLKTXqMTELE5VE3qAflUdk2CAzYTC46R8YseXaOaPhQUFMRP38gSqp~lgqfnWA4NnnTB2c3hAfLL3BOSrf~c6MBdU54RVhgN0WOW9AmrBiYh12mFHqZVbn-23r48SzLFUI76mUxnprKGvIijOYeFRr7afEGOkQ8FlJGMmldvsclrXHunXgxUdF6mAcy-YISYQXyVRzFCYqNbfmcsCnMJWbHnojoc1EjO-D0~0uHJyijMWxOjCgQai8Tq5-EkINNeD-f0RuF1c3OdWgr775OxLthbn-XIkCl2u32Ip7b09WDKXBX2u02q2aYnrMIC-d-ElGnBmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow chart of the identification of the TISA. The results for each step of the human TISA analysis are shown." path-from-xml="dsl011f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194780/dsl011f1.jpeg" path-from-xml="dsl011f1.jpeg" target="_blank">View large</a><a section="87194780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f1.jpeg?Expires=2147483647&Signature=K82rr9arZKsO~1GNCP1Pdn2s0ICp4wV1N-cF2CQZhPljX16Yae3OwAql2rAQ9mVfl97Kryw7S0ILxeECwlHQ7NjjXyAAvDoWTUW6O2d6QRrfgWIyGXIdA4~yUO-6fJlCxi1Qudn9lHq~JQuwbGlaXUhO8qxpZkxVTvfxpJ7CEzAw7YtnF5CpAlHAyDX4U7DXNhHmq72xF3ChG~zBXL5yBtoEnnHi7DQXqUCtW07KvHJP1yrfehUHsvcTM2F~zEwfP2~0fDfG0wuokqwa-Z4OMDCC4aW0M0zphcEpOzl~cXFpy9kPbLViqFNk8kuIp7Jf7mjVQB9O7r0MexyyE7wkMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194780&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow chart of the identification of the TISA. The results for each step of the human TISA analysis are shown.</p></div></div>                            <h3 scrollto-destination=87194782 id="87194782" class="section-title" >2.3 Genomic clustering</h3>
<p>By scanning each chromosome from the start, we compared the genomic alignment structures of full-length mRNAs whose alignment region overlapped with clustered mRNAs that share at least one common splice site. If the alignment showed a distinct structure, we created a new cluster. From the first cluster, we constructed a preliminary reconstructed structure (RS) that corresponds to a gene unit composed of exon/intron sets and their linkage relationships. During the clustering process, we extended the gene structure whenever new exons or introns were recognized. After clustering the full-length mRNAs, partial mRNAs and ESTs aligned within the preliminary gene boundary were retrieved and compared with the RS structure. We ignored clustering in the genomic region without full-length mRNA mapping, even though many ESTs or partial mRNAs were aligned in that region. We allowed all possible splice sequences as well as canonical splice sequences (CSSs; i.e. GT-AT, GC-AG and AT-AC); however, in the case of non-CSSs, we approved them only if they were confirmed by at least five ESTs. The query orientation along the genome was re-evaluated based on the splice site sequence of the query. For example, if a splice sequence set of a given query was mainly composed of GT-AG, it is obvious that the query orientation along the genome was forward. If a splice sequence set was mainly composed of CT-AC, the query was aligned in a reverse direction. The orientation of a gene was determined as the same as the query. Comparisons of the splice site position and its orientation were used as the main criteria for deciding whether a query should be included in a given cluster.</p>                            <h3 scrollto-destination=87194786 id="87194786" class="section-title" >2.4 Establishment of exon and intron sets</h3>
<p>To measure the confidence of RS exons and introns, we applied a custom scoring scheme. The reliability of RS exons depends on (i) whether the splice sequences of introns linked at both ends of the exon are CSS or non-CSS and (ii) how many queries support the given exon or intron structure. For example, the exons linked with CSS introns at both ends and supported by a number of mRNAs and ESTs have high scores. Because full-length mRNAs provide more reliable evidence than ESTs, we imposed a double score for the exons and introns derived from full-length mRNAs. The group of RS exons/introns whose hard-edge boundaries are within ±6 nt were merged into a single exon or intron and rescored. In the process of RS exon/intron merging, the exon or intron with the best score was selected as a representative. Finally, RS exons with scores higher than the cutoff were saved and used for linkage analysis. The minimal requirements for saving an exon were that it should be supported by at least one EST whose boundaries are linked to CSS introns at both ends. For example, if an internal exon is supported by two ESTs and is linked with non-CSS introns at both ends, its exon score will be 0.4 and 0.4 for the score of non-CSS linked exon boundary is given as 0.2. Because the score of this exon is lower than the cutoff value (for first exon, [−, 1.0]; internal exon, [1.0, 1.0]; last exon, [1.0, −]), it was discarded from the RS exon set. The same rule was applied to the intron scoring and selection (cutoff score = 1.0). We discarded internal exons shorter than 10 bp and introns shorter than 20 bp.</p>                            <h3 scrollto-destination=87194790 id="87194790" class="section-title" >2.5 Evaluation and selection of alternative linkage relationships</h3>
<div class="block-child-p">We evaluated each mutually exclusive alternative linkage relationship using the exact binomial probability as previously reported.<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> Briefly, <em>k</em> is the number of ESTs supporting a particular alternative splice <em>Y</em>, and <em>n</em> is the total number of EST supporting all of mutually exclusive splice patterns including <em>Y</em>. The observed frequency of <em>Y</em> (= <em>k</em>/<em>n</em>) is tested to see if it is higher than a given threshold frequency <em>f</em> with statistical significance. The binomial probability <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011i1.jpeg?Expires=2147483647&amp;Signature=sSUz-G1jMSR6nOnGC~m-geStNCsAforGbvUG6kbHykU5Qbxbkzlx7~38QufTgX~bvrGFqcUIjaf6uwtK9hRw3E-fx63pnc6YY391f0KRN~PudBaUKZNNw8PPXIOsBts0bUNmi0SfOgM7rMkvRIvJR5MsL6K~vAxc9PKeCE9FQKspDJO2MlVtksIlC79Kl60P5mDACwQJ1EVOaUeW1qEavvkgdxx2R7dcF06qltnHVoifz62KPfDqA2~phMsWza3JLl8LupF3C6Yl1Yhv~ZrnqN9dFJUt5QFgkn3OILemguSFDcppsCNzN9J1lWZtBNRs~gk7WwzCycLMk~8seMoGZA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsl011i1.jpeg" /></span> indicates the probability that alternative splice <em>Y</em> occurs at least <em>k</em> times in <em>n</em> trials with a given frequency <em>f</em>. This can be a good estimation of whether the observed frequency of <em>Y</em> exceeds a given threshold frequency relative to all of the alternative splices in mutual relationships. If <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011i2.jpeg?Expires=2147483647&amp;Signature=iOdfhIhsHSUrBT5mOHRkbzWIw8fcsyI5ymUiYeRLvmvi2TXrbrbxC3GacXwTPSYrmo2YJtja7V4tybQ-W61RbLCakR-rTS7cyprSTXdl5q9GZ~nbZxBfZ96pN5kK8ZHcJcwlp06PYGdVM1TSfEfl-Ll~OUOpE~OoqeOlsgsuQQwxdPpKeDQMvCsUNhHPQfOTL5wIKPcJ5cPDo2hDoyOc~8k9j5xDDUYojbHRdD0xpvk775MvF6Z6Ix-pmU~~RQ5tTr5~VyRfcw56R7hA9emMqQaXY0IWLB1va~hE0Cp6Icp7VTJC5btFQ8tIR7aJxPPPcc8unGBDjNJOmn8wcSBXCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsl011i2.jpeg" /></span> is &lt;0.05, then the observed frequency of the alternative splice is expected to exceed the given threshold with high (&gt;95%) confidence. The formula used for the calculation is as follows: <a id="e1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e1"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>P</mi><mrow><mo>(</mo><mrow><mo>|</mo><mi>Y</mi><mo>|</mo><mo>≥</mo><mi>k</mi></mrow><mo>)</mo></mrow><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mi>k</mi></mrow><mi>n</mi></munderover></mstyle><mrow><mi>P</mi><mrow><mo>(</mo><mi>i</mi><mo>)</mo></mrow><mo>;</mo><mi>P</mi><mrow><mo>(</mo><mi>i</mi><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mi>n</mi><mo>!</mo></mrow><mrow><mi>i</mi><mo>!</mo><mrow><mo>(</mo><mrow><mi>n</mi><mo>−</mo><mi>i</mi></mrow><mo>)</mo></mrow><mo>!</mo></mrow></mfrac><msup><mi>f</mi><mi>i</mi></msup></mrow><msup><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>f</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>n</mi><mo>−</mo><mi>i</mi></mrow></msup></mrow></math></div><div class="label">(1)</div></div></div><p>We applied a 1% frequency threshold (<em>f</em> = 0.01) and a 95% CI as cutoff criteria. We allowed only the alternative linkage relationships satisfying the above condition in the splice graph.</p>                            <h3 scrollto-destination=87194800 id="87194800" class="section-title" >2.6 Transcript reconstruction and classification</h3>
<p>After establishing the non-redundant exon/intron sets statistically verified as described above, we found all possible paths by traveling the nodes (i.e. exons) through edges (i.e. introns) in a splice graph starting from the first exon (eTag = 1) and finishing at the last exon (eTag = 3). From each path, we reconstructed a transcript structure in a given gene. For each transcript, we compared its genomic structure one by one with all of query structures clustered in a gene and arranged the list of queries whose structure do not collide with the transcript structure (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Then, we checked the coverage of each transcript structure supported by non-colliding queries and subdivided the transcripts into three classes. In Class I, all of the exons and introns in a transcript were supported by query structure(s) or its combinations. As for Class II, at least one of exons or introns in a transcript was not validated. In Class III, there is no supporting evidence for a transcript. Class I transcripts represent most reliable set of transcripts generated from the splice graph. Class III transcripts are least reliable in which all the exon–exon connections are only theoretically possible in a graph.</p>                        <a id="87194805" scrollto-destination="87194805"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f2.jpeg?Expires=2147483647&Signature=BGkyU-yh9VPHoSR57fPG1XmxsVlYSzs2SZik7OXPWvkwPUC0o8Od4T6SjNtTvOYdwVnli73pH27HAN6l6UrFBcQgiYLdAz~REm9Mv~AG7GJaMcCBgTf~C1PsKWAjRPDOtlXRtdLCP~N~2AC1RggTv81sy1YeBuWzcosXwSApaqg0KXPUriI0CPX23fQ8W37V5tqARBKEybJkc71FdRzn4u1RhjBPoqx8Lvz7TYHPWXaoILcHamEHG6kLTmfvwkiRyBtIYkCKDIamtUlFXe2EvYBn-BLTKws~a~KUSGk2qnxcCpUxeBsnhqGWdk3gieGHD1UKc1HpKGqwX8SFRRPDyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic diagram of the genomic cluster, splice graph and the reconstruction and classification of transcript variants in human SCARF1 gene. In this gene cluster, 13 mRNAs (full-length: dark green, partial: bright green) and 32 ESTs (grey) were clustered as a single gene unit. There are seven AS events in total. Among them, only four AS events are valid and finally remained in the splice graph. (A) The gene structure and alternative linkage relationships were deduced from the genomic alignment structures of 45 queries. Exonic regions are indicated with solid boxes. Alternative splice borders and alternative exonic regions are also represented as dashed lines and dashed boxes. The two purple vertical lines in the last exon represent polyA addition signals (i.e. AATAAA). The ‘A’ at the end of alignments for some of queries represents the presence of a poly A tail in the original untrimmed query. (B) Full splice graph. The boxes with numbers indicate exons and the arrows represent introns. Statistically unverified alternative linkages and corresponding exons were represented as dashed boxes and dashed lines respectively. (C) Reconstruction of transcript variants and their classification. A total of eight transcripts can be generated by all possible traversals from the splice graph. The supporting queries for each transcript isoform are indicated. For all the eight transcript structures 23 queries are common; however, those only cover 5′ part of the gene and the last exon. After examination of the supporting query structures for each isoform, each transcript isoform was classified into Classes I, II or III. In this example, there is no Class III transcript. Exons and introns not supported by clustered queries for a given transcript in Class II are represented as blank boxes and dashed lines, respectively. For example, in transcript isoform 7 and 8, which are supported by only common queries, the internal regions are not supported by the queries. Thus, these are represented by dashed lines and blank boxes, respectively. Query 22, 25, 26 could be not included in any of the isoforms and was therefore treated as singletons." path-from-xml="dsl011f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194805/dsl011f2.jpeg" path-from-xml="dsl011f2.jpeg" target="_blank">View large</a><a section="87194805" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f2.jpeg?Expires=2147483647&Signature=smv8ZMiUiaG4HXCMm2v0Myi-JOfWQxalSfvg8RKNSt4jRgi3iSlwcNpkOa9W2JiyWROqXfDKapytj4L1SorhAKF8yuJPfCwyc~gxgqbEaK6NKEz0Tq-hY-EtT14M3fQGNZOPLh0e7ZU3ZbP92OFxrCcPi8g3g2JnBZnNXEgHrjUV4OCqetjF3iAP6Bux~nHmAKhN4bM4B3~SyL2atb7FOKEBFXNHUwlhBTzVx4XjW3A47vYfjRADh89usMgIQp31GzKpip5M41zE5pGSP6GDmUs6jdqPz-03Kz14nYSES2kIF0~pVNZWf0m232FQGCaS~a0Z2WeIv~20J5R8yk~rDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194805&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram of the genomic cluster, splice graph and the reconstruction and classification of transcript variants in human <em>SCARF1</em> gene. In this gene cluster, 13 mRNAs (full-length: dark green, partial: bright green) and 32 ESTs (grey) were clustered as a single gene unit. There are seven AS events in total. Among them, only four AS events are valid and finally remained in the splice graph. (<strong>A</strong>) The gene structure and alternative linkage relationships were deduced from the genomic alignment structures of 45 queries. Exonic regions are indicated with solid boxes. Alternative splice borders and alternative exonic regions are also represented as dashed lines and dashed boxes. The two purple vertical lines in the last exon represent polyA addition signals (i.e. AATAAA). The ‘A’ at the end of alignments for some of queries represents the presence of a poly A tail in the original untrimmed query. (<strong>B</strong>) Full splice graph. The boxes with numbers indicate exons and the arrows represent introns. Statistically unverified alternative linkages and corresponding exons were represented as dashed boxes and dashed lines respectively. (<strong>C</strong>) Reconstruction of transcript variants and their classification. A total of eight transcripts can be generated by all possible traversals from the splice graph. The supporting queries for each transcript isoform are indicated. For all the eight transcript structures 23 queries are common; however, those only cover 5′ part of the gene and the last exon. After examination of the supporting query structures for each isoform, each transcript isoform was classified into Classes I, II or III. In this example, there is no Class III transcript. Exons and introns not supported by clustered queries for a given transcript in Class II are represented as blank boxes and dashed lines, respectively. For example, in transcript isoform 7 and 8, which are supported by only common queries, the internal regions are not supported by the queries. Thus, these are represented by dashed lines and blank boxes, respectively. Query 22, 25, 26 could be not included in any of the isoforms and was therefore treated as singletons.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f2.jpeg?Expires=2147483647&Signature=BGkyU-yh9VPHoSR57fPG1XmxsVlYSzs2SZik7OXPWvkwPUC0o8Od4T6SjNtTvOYdwVnli73pH27HAN6l6UrFBcQgiYLdAz~REm9Mv~AG7GJaMcCBgTf~C1PsKWAjRPDOtlXRtdLCP~N~2AC1RggTv81sy1YeBuWzcosXwSApaqg0KXPUriI0CPX23fQ8W37V5tqARBKEybJkc71FdRzn4u1RhjBPoqx8Lvz7TYHPWXaoILcHamEHG6kLTmfvwkiRyBtIYkCKDIamtUlFXe2EvYBn-BLTKws~a~KUSGk2qnxcCpUxeBsnhqGWdk3gieGHD1UKc1HpKGqwX8SFRRPDyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic diagram of the genomic cluster, splice graph and the reconstruction and classification of transcript variants in human SCARF1 gene. In this gene cluster, 13 mRNAs (full-length: dark green, partial: bright green) and 32 ESTs (grey) were clustered as a single gene unit. There are seven AS events in total. Among them, only four AS events are valid and finally remained in the splice graph. (A) The gene structure and alternative linkage relationships were deduced from the genomic alignment structures of 45 queries. Exonic regions are indicated with solid boxes. Alternative splice borders and alternative exonic regions are also represented as dashed lines and dashed boxes. The two purple vertical lines in the last exon represent polyA addition signals (i.e. AATAAA). The ‘A’ at the end of alignments for some of queries represents the presence of a poly A tail in the original untrimmed query. (B) Full splice graph. The boxes with numbers indicate exons and the arrows represent introns. Statistically unverified alternative linkages and corresponding exons were represented as dashed boxes and dashed lines respectively. (C) Reconstruction of transcript variants and their classification. A total of eight transcripts can be generated by all possible traversals from the splice graph. The supporting queries for each transcript isoform are indicated. For all the eight transcript structures 23 queries are common; however, those only cover 5′ part of the gene and the last exon. After examination of the supporting query structures for each isoform, each transcript isoform was classified into Classes I, II or III. In this example, there is no Class III transcript. Exons and introns not supported by clustered queries for a given transcript in Class II are represented as blank boxes and dashed lines, respectively. For example, in transcript isoform 7 and 8, which are supported by only common queries, the internal regions are not supported by the queries. Thus, these are represented by dashed lines and blank boxes, respectively. Query 22, 25, 26 could be not included in any of the isoforms and was therefore treated as singletons." path-from-xml="dsl011f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194805/dsl011f2.jpeg" path-from-xml="dsl011f2.jpeg" target="_blank">View large</a><a section="87194805" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f2.jpeg?Expires=2147483647&Signature=smv8ZMiUiaG4HXCMm2v0Myi-JOfWQxalSfvg8RKNSt4jRgi3iSlwcNpkOa9W2JiyWROqXfDKapytj4L1SorhAKF8yuJPfCwyc~gxgqbEaK6NKEz0Tq-hY-EtT14M3fQGNZOPLh0e7ZU3ZbP92OFxrCcPi8g3g2JnBZnNXEgHrjUV4OCqetjF3iAP6Bux~nHmAKhN4bM4B3~SyL2atb7FOKEBFXNHUwlhBTzVx4XjW3A47vYfjRADh89usMgIQp31GzKpip5M41zE5pGSP6GDmUs6jdqPz-03Kz14nYSES2kIF0~pVNZWf0m232FQGCaS~a0Z2WeIv~20J5R8yk~rDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194805&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram of the genomic cluster, splice graph and the reconstruction and classification of transcript variants in human <em>SCARF1</em> gene. In this gene cluster, 13 mRNAs (full-length: dark green, partial: bright green) and 32 ESTs (grey) were clustered as a single gene unit. There are seven AS events in total. Among them, only four AS events are valid and finally remained in the splice graph. (<strong>A</strong>) The gene structure and alternative linkage relationships were deduced from the genomic alignment structures of 45 queries. Exonic regions are indicated with solid boxes. Alternative splice borders and alternative exonic regions are also represented as dashed lines and dashed boxes. The two purple vertical lines in the last exon represent polyA addition signals (i.e. AATAAA). The ‘A’ at the end of alignments for some of queries represents the presence of a poly A tail in the original untrimmed query. (<strong>B</strong>) Full splice graph. The boxes with numbers indicate exons and the arrows represent introns. Statistically unverified alternative linkages and corresponding exons were represented as dashed boxes and dashed lines respectively. (<strong>C</strong>) Reconstruction of transcript variants and their classification. A total of eight transcripts can be generated by all possible traversals from the splice graph. The supporting queries for each transcript isoform are indicated. For all the eight transcript structures 23 queries are common; however, those only cover 5′ part of the gene and the last exon. After examination of the supporting query structures for each isoform, each transcript isoform was classified into Classes I, II or III. In this example, there is no Class III transcript. Exons and introns not supported by clustered queries for a given transcript in Class II are represented as blank boxes and dashed lines, respectively. For example, in transcript isoform 7 and 8, which are supported by only common queries, the internal regions are not supported by the queries. Thus, these are represented by dashed lines and blank boxes, respectively. Query 22, 25, 26 could be not included in any of the isoforms and was therefore treated as singletons.</p></div></div>                            <h3 scrollto-destination=87194810 id="87194810" class="section-title" >2.7 Statistical analyses for tissue specificity</h3>
<p>If a particular gene has a large number of ESTs derived from a specific tissue, we assumed that the gene is likely to be expressed at a higher level in that tissue. We applied the statistical analysis designed by Audic and Claverie<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> to analyse the tissue specificity of all Class I transcripts and genes. We performed Fisher's exact test to assess the tissue specificity of AS events.</p>                            <h4 scrollto-destination=87194814 id="87194814" class="section-title" >2.7.1 Audic's test</h4>
<div class="block-child-p">In Audic's test, the number of successfully clustered ESTs from all genes is <em>N</em>, the number of ESTs from a specific tissue is <em>N<sub>2</sub></em>, the number of ESTs from all other tissues is <em>N<sub>1</sub></em>, the total number of ESTs clustered on a given gene is <em>n</em>, the number of ESTs from a specific tissue in a given gene is <em>y</em>, and the number of ESTs from all other tissues in a given gene is <em>x</em>. Thus, <em>N<sub>1</sub> + N<sub>2</sub> = N</em> and <em>x + y = n</em>. We set the null hypothesis, <em>H<sub>o</sub></em>, i.e. there is no difference between the ratio of the specific tissue count for a gene and the number for ESTs in that tissue (<em>y</em>/<em>N<sub>2</sub></em>) and the ratio of all other tissue counts in a gene and the number of ESTs in all other tissues (<em>x</em>/<em>N<sub>1</sub></em>). Given that <em>x</em> was derived from <em>N<sub>1</sub></em>, the conditional probability that <em>y</em> would be derived from <em>N<sub>2</sub></em> is as follows:<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a><a id="e2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="e2"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>p</mi><mo>(</mo><mi>y</mi><mo>|</mo><mi>x</mi><mo>)</mo><mo>=</mo><msup><mrow><mrow><mo>(</mo><mrow><mfrac><mrow><msub><mi>N</mi><mn>2</mn></msub></mrow><mrow><msub><mi>N</mi><mn>1</mn></msub></mrow></mfrac></mrow><mo>)</mo></mrow></mrow><mi>y</mi></msup><mfrac><mrow><mo>(</mo><mi>x</mi><mo>+</mo><mi>y</mi><mo>)</mo><mo>!</mo></mrow><mrow><mi>x</mi><mo>!</mo><mi>y</mi><mo>!</mo><msup><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><msub><mi>N</mi><mn>2</mn></msub></mrow><mrow><msub><mi>N</mi><mn>1</mn></msub></mrow></mfrac></mrow><mo>)</mo></mrow></mrow><mrow><mrow><mo>(</mo><mrow><mi>x</mi><mo>+</mo><mi>y</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msup></mrow></mfrac></mrow></math></div><div class="label">(2)</div></div> The <em>P</em>-value was calculated using the following equation: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="italic">Pvalue</mi><mo>=</mo><msup><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mi>y</mi><mi>n</mi></munderover></mstyle><mrow><mi>p</mi><mrow><mo>(</mo><mrow><mi>y</mi><mo>|</mo><mi>x</mi></mrow><mo>)</mo></mrow><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mi>y</mi><mi>n</mi></munderover></mstyle><mrow><mrow><mo>(</mo><mrow><mfrac><mrow><msub><mi>N</mi><mn>2</mn></msub></mrow><mrow><msub><mi>N</mi><mn>1</mn></msub></mrow></mfrac></mrow><mo>)</mo></mrow></mrow></mrow></mrow><mi>y</mi></msup><mfrac><mrow><mrow><mo>(</mo><mrow><mi>x</mi><mo>+</mo><mi>y</mi></mrow><mo>)</mo></mrow><mo>!</mo></mrow><mrow><mi>x</mi><mo>!</mo><mi>y</mi><mo>!</mo><msup><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><msub><mi>N</mi><mn>2</mn></msub></mrow><mrow><msub><mi>N</mi><mn>1</mn></msub></mrow></mfrac></mrow><mo>)</mo></mrow></mrow><mrow><mrow><mo>(</mo><mrow><mi>x</mi><mo>+</mo><mi>y</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msup></mrow></mfrac></mrow></math></div><div class="label">(3)</div></div></div><p>If the <em>P-</em>value is &lt;0.01, we deny the null hypothesis with 99% confidence, and the EST count in that tissue for a given gene is significantly different than the EST count in all other tissues, indicating that the gene is expressed at a higher level in that tissue.</p>                            <h4 scrollto-destination=87194817 id="87194817" class="section-title" >2.7.2 Benjamini–Yekutieli method for multiple test correction</h4>
<p>To correct significance estimates for multiple hypothesis testing, we used the false discovery rate (FDR) method of Benjamini and Yekutieli,<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> which calculates a new test statistic that estimates the portion of Type I errors within a group of hypotheses meeting a significance cutoff. The FDR is the expected proportion of erroneously rejected null hypotheses among the rejected ones. The BY procedure is performed as follows. The individual <em>P</em>-values are sorted from smallest to largest: <em>P</em>(1) ≤ <em>P</em>(2) ≤ ⋯ ≤ <em>P</em><sub>(<em>m</em>)</sub>. Starting from the largest <em>P</em>-value (<em>P</em><sub>(<em>m</em>)</sub>), <em>P</em><sub>(<em>i</em>)</sub> is compared with <em>a</em> • <em>i/m</em>/(1/1+1/2+⋯+1/<em>m</em>), where <em>α</em> is a desired level of FDR. This is continued as long as <em>P</em><sub>(<em>i</em>)</sub> &gt; <em>α</em> • <em>i/m</em>/(1/1+1/2+⋯+1/<em>m</em>). Next, <em>k</em> is set as the first time <em>P</em><sub>(<em>i</em>)</sub> ≤ α • <em>i/m</em>/(1/1+1/2+⋯+1/<em>m</em>). The hypotheses corresponding to the smallest <em>k P</em>-values are then declared to be significant. We set 0.01 as the desired value of α and rejected the <em>k</em> hypotheses with <em>P</em>-values less than <em>P</em><sub>(<em>k</em>)</sub> in each evaluation of tissue specificity. The Benjamini–Yekutieli method considers the random dependence between the tests. We reasoned that the tissue specificity of the gene (or transcripts) in a particular tissue can be affected by the tissue specificities in other tissues for a given gene.</p>                            <h3 scrollto-destination=87194819 id="87194819" class="section-title" >2.8 GO analysis</h3>
<p>The full list of GO terms was downloaded from the Gene Ontology Consortium, and the directed acyclic graph structure of GO was rebuilt as similar with AmiGO (<a class="link link-uri" href="http://godatabase.org/" target="_blank"></a>). For each GO term, all associated child GO terms in the downstream branch nodes were gathered recursively by searching the whole GO tree structure for related entries with either an ‘<em>is a</em>’ or a ‘<em>part of</em>’ relationship. A total of 4134 GO terms were found to be associated with 9612 alternatively spliced human genes. Fisher's exact test using a 2 × 2 contingency table was applied to investigate whether there was any bias for a certain AS pattern amongst the nine AS patterns for a given GO term.<a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a> In the table, the two columns are ‘genes with the AS pattern’ and ‘genes without the AS pattern,’ and the two rows are ‘having the GO term’ and ‘not having the GO term.’ Similarly, we carried out tests for the enrichment of specific GO terms in the group of tissue-specific genes or transcripts for each tissue. In these cases, the two columns are ‘genes (or transcripts), tissue-specific in a certain tissue’ and ‘genes (or transcripts), tissue-specific in all other tissues’, and the two rows are ‘having the GO term’ and ‘not having the GO term.’</p>                            <h2 scrollto-destination=87194821 id="87194821" class="section-title" >3 Results and Discussion</h2>
<p>The overall process and the results of each step for human are summarized in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>. The process can be divided into three main steps: (i) construction of a splice graph based on genomic alignment of mRNA/EST; (ii) reconstruction of transcript variants and AS pattern analysis from the splice graph; and (iii) statistical analysis of the tissue specificity of the genes, transcripts and AS events using the tissue categories of the supporting ESTs.</p>                            <h3 scrollto-destination=87194823 id="87194823" class="section-title" >3.1 Splice graph construction from the genomic alignments of mRNAs/ESTs</h3>
<p>Using genome-based mRNA/EST mapping, we successfully localized ∼90% of mRNA/ESTs in the genome. We developed our own clustering program using genomically aligned full-length mRNA as the standard. Because our study focuses on AS analysis and forecasting of tissue-specific AS rather than on gene prediction, we conducted clustering in the genomic region with full-length mRNA mapping. Many partial mRNAs and ESTs aligned with the genome were discarded because a lack of full-length mRNA data. As a result, 178 807 (74%) mRNAs and 5 605 025 (72%) ESTs were finally incorporated into the gene clusters of the human genome. A total of 89 814 full-length mRNAs corresponded to 26 143 genes (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Multi-exon ESTs accounted for 58% (3 261 926) of all clustered ESTs.</p><p>RS is composed of exon/intron sets and exon–exon linkage relationships. Based on the genomic structure of standard mRNA in a cluster, newly added mRNAs or ESTs were included into the RS if their alignment structures shared at least one splice site with the RS. We scored each exon and intron using our scoring rule. The exon score is an index of the confidence of the exon. We controlled the score so that the exon has a higher splice junction score when it has more supporting queries (mRNA/ESTs) and when the introns connected to the end of the exon have CSSs (GT-AG, GC-AG and AT-AC) that are more reliable. We gave differential weights to full-length mRNAs and ESTs, considering the qualitative difference in their sequences. This rule was also applied to intron scoring. Our algorithm also allowed non-CSS introns if they were supported by at least five ESTs. Exons or introns whose boundaries were within ±6 bp at both ends were merged into a single exon or intron, and its score was recalculated. The exon or intron with the best score was chosen as representative. The merging process corrects erroneous splicing that might occur during BLAT mapping or as a result of poor sequence quality of ESTs. Consequently, we were able to obtain reliable non-redundant exon/intron sets by exon/intron merging, rescoring and selection by using a cutoff score. Based on these steps, we constructed the splice graph and conducted the AS analysis.</p>                            <h3 scrollto-destination=87194826 id="87194826" class="section-title" >3.2 Splice graph analysis</h3>
<p>Our splice graph algorithm for detecting AS is essentially the same as previously reported.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> The largest differences are in (i) the application of the selection step for statistically reliable alternative linkage relationships during splice graph construction and (ii) the classification of transcript variants.</p>                            <h4 scrollto-destination=87194828 id="87194828" class="section-title" >3.2.1 Selection of statistically reliable alternative splices</h4>
<p>If all of the alternative splices that can be obtained using erroneous EST data are allowed, one or more alternatively spliced forms could be detected for nearly all genes examined as long as they have sufficient EST coverage. Therefore, a method is needed to discriminate the true isoforms from erroneous splicing resulting from aberrant splicing or misalignment. Kan et al.<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> suggested a statistical method to solve this problem, wherein alternative splices are selected according to the frequency at which they occur with a high confidence level. One example is shown in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>. There are 45 supporting queries (13 mRNAs and 32 ESTs) for the human <em>SCARF1</em> (scavenger receptor Class F, member 1) gene cluster. Among 13 mRNAs queries, 12 are full-length. Seven AS events can be defined from the comparisons of their alignment structures: two alternative first exon events (E1-E3 versus E2-E3, E4-E6 versus E5-E6), complex splice site variation (E7-E13 versus E12-E13), two 3′-splice variations (E8-E10 versus E8-E11, E15-E16 versus E15-E17), intron retention (E8-E10 versus E9) and alternative last exon (E13-E15 versus E14). The seven AS events and the number of sequences supporting each isoform are summarized in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>. Four of them (3, 4, 5 and 7) have sufficient sequence evidence for the minor isoform relative to the major isoform. AS events 1, 2 and 6, however, have only one supporting query for each, whereas the query count for the each counterpart major isoform is much larger, yielding a low observed frequency for the minor isoform. To determine whether the significance of the observed frequency (for example, 1/33 for AS event 1) of the minor isoform is higher than the threshold (frequency of 1%) or not, we calculated <em>P</em>-value adopting binomial statistics. The binomial probability of each AS event was calculated based on the number of supporting queries for the isoform. As shown in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>, the <em>P</em>-value for the minor isoform of AS event 1 exceeding the given threshold (1%) was 0.282. As similar, <em>P</em>-values for AS events 2 and 6 show the value higher than the threshold, whereas the <em>P</em>-values of the other four AS events were under 0.05 (i.e. 95% confidence). Thus, these three events were excluded from the splice graph (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a>).</p>                            <h4 scrollto-destination=87194830 id="87194830" class="section-title" >3.2.2 Reconstruction of transcript isoforms and their classification</h4>
<p>After the selection of a reliable alternative splice set, we built the adjacency list for the directed acyclic graph and constructed all possible combinations of paths through graph traversal as described previously.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> Each path represents a transcript variant. By comparing the alignment structures of each transcript and its supporting queries, we subdivided the transcript isoforms as follows: Class I, all of the exons and introns in the transcript were supported by query structures; Class II, some of exons or introns were not validated; and Class III, none of the exons or introns had supporting evidence (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). We also calculated the total score of each transcript based on the number of supporting queries, considering the exon/intron coverage and sorted transcripts according to their scores. In the example of <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>, a total of eight transcripts were constructed by graph traversal: isoforms 1–5 were Class I; 6–8 were Class II.</p><p><a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> shows the results of the analysis of the AS and gene modelling from the graph algorithm based on genomic clustering data. In the human genome, 13 560 (66%) genes were found to have &gt;1 transcript, and in the mouse genome, 11 752 (56%) genes were estimated to have at least one transcript. A total of 402 995 transcripts were formed, of which 97 286 (24%) in human and 33% (64 255 of 197 361) in mouse were Class I. On average, 6.5 and 5.0 isoforms were generated from one AS gene in human and mouse respectively. The AS patterns and tissue specificities of the transcripts were examined only for Class I transcripts.</p>                        <a id="87194833" scrollto-destination="87194833"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Statistics of the reconstructed genes and transcripts in human and mouse</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><strong>Human</strong> </th><th><strong>Mouse</strong> </th></tr></thead><tbody><tr><td>Genes </td><td>26 143 </td><td>27 841 </td></tr><tr><td>mRNAs </td><td>178 807 </td><td>266,819 </td></tr><tr><td>    Multi-exon </td><td>155 666 </td><td>194 778 </td></tr><tr><td>    Multi-exon, complete cds </td><td>81 780 </td><td>80 634 </td></tr><tr><td>ESTs </td><td>5 605 025 </td><td>3 688 726 </td></tr><tr><td>    Multi-exon </td><td>3 261 926 </td><td>1 645 977 </td></tr><tr><td>Multi-exon genes w / EST support </td><td>20 635 </td><td>20 838 </td></tr><tr><td>Genes w/multiple transcripts </td><td>13 560 (66%)<sup>a</sup> </td><td>11 752 (56%)<sup>a</sup> </td></tr><tr><td>Transcripts </td><td>402 995 </td><td>197 361 </td></tr><tr><td>    Class I </td><td>97 286 </td><td>64 255 </td></tr><tr><td>    Class II </td><td>297 601 </td><td>131 167 </td></tr><tr><td>    Class III </td><td>4952 </td><td>1939 </td></tr><tr><td>Avg. No. Class I transcripts/AS gene </td><td>6.5 </td><td>5 </td></tr><tr><td>Exons<sup>b</sup> </td><td>259 031 (41%) </td><td>247 342 (33%) </td></tr><tr><td>    Single exon (i.e. single exon gene) </td><td>4559 </td><td>6076 </td></tr><tr><td>    First exon </td><td>34 234 </td><td>45 436 </td></tr><tr><td>    Internal exon </td><td>190 341 </td><td>182 125 </td></tr><tr><td>    Last exon </td><td>29 897 </td><td>43 441 </td></tr><tr><td>    Alternative exons<sup>c</sup> </td><td>107 194 </td><td>82 503 </td></tr><tr><td>Introns<sup>b</sup> </td><td>237 215 (46%) </td><td>220 878 (37%) </td></tr><tr><td>    GT‥AG </td><td>230 735 </td><td>215 326 </td></tr><tr><td>    GC‥AG </td><td>2859 </td><td>2600 </td></tr><tr><td>    AT‥AC </td><td>301 </td><td>331 </td></tr><tr><td>    Others </td><td>3320 </td><td>2621 </td></tr><tr><td>    Alternative introns<sup>c</sup> </td><td>108 304 </td><td>81 791 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><strong>Human</strong> </th><th><strong>Mouse</strong> </th></tr></thead><tbody><tr><td>Genes </td><td>26 143 </td><td>27 841 </td></tr><tr><td>mRNAs </td><td>178 807 </td><td>266,819 </td></tr><tr><td>    Multi-exon </td><td>155 666 </td><td>194 778 </td></tr><tr><td>    Multi-exon, complete cds </td><td>81 780 </td><td>80 634 </td></tr><tr><td>ESTs </td><td>5 605 025 </td><td>3 688 726 </td></tr><tr><td>    Multi-exon </td><td>3 261 926 </td><td>1 645 977 </td></tr><tr><td>Multi-exon genes w / EST support </td><td>20 635 </td><td>20 838 </td></tr><tr><td>Genes w/multiple transcripts </td><td>13 560 (66%)<sup>a</sup> </td><td>11 752 (56%)<sup>a</sup> </td></tr><tr><td>Transcripts </td><td>402 995 </td><td>197 361 </td></tr><tr><td>    Class I </td><td>97 286 </td><td>64 255 </td></tr><tr><td>    Class II </td><td>297 601 </td><td>131 167 </td></tr><tr><td>    Class III </td><td>4952 </td><td>1939 </td></tr><tr><td>Avg. No. Class I transcripts/AS gene </td><td>6.5 </td><td>5 </td></tr><tr><td>Exons<sup>b</sup> </td><td>259 031 (41%) </td><td>247 342 (33%) </td></tr><tr><td>    Single exon (i.e. single exon gene) </td><td>4559 </td><td>6076 </td></tr><tr><td>    First exon </td><td>34 234 </td><td>45 436 </td></tr><tr><td>    Internal exon </td><td>190 341 </td><td>182 125 </td></tr><tr><td>    Last exon </td><td>29 897 </td><td>43 441 </td></tr><tr><td>    Alternative exons<sup>c</sup> </td><td>107 194 </td><td>82 503 </td></tr><tr><td>Introns<sup>b</sup> </td><td>237 215 (46%) </td><td>220 878 (37%) </td></tr><tr><td>    GT‥AG </td><td>230 735 </td><td>215 326 </td></tr><tr><td>    GC‥AG </td><td>2859 </td><td>2600 </td></tr><tr><td>    AT‥AC </td><td>301 </td><td>331 </td></tr><tr><td>    Others </td><td>3320 </td><td>2621 </td></tr><tr><td>    Alternative introns<sup>c</sup> </td><td>108 304 </td><td>81 791 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The number in parentheses indicates the percentage of genes having multiple transcripts (Class I only) of the multi-exon genes with EST support.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Only the exons and the introns included in Class I transcripts were analysed.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Alternative exons/introns mean exons or introns that are alternatively included in Class I transcripts of the respective gene.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194833" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Statistics of the reconstructed genes and transcripts in human and mouse</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><strong>Human</strong> </th><th><strong>Mouse</strong> </th></tr></thead><tbody><tr><td>Genes </td><td>26 143 </td><td>27 841 </td></tr><tr><td>mRNAs </td><td>178 807 </td><td>266,819 </td></tr><tr><td>    Multi-exon </td><td>155 666 </td><td>194 778 </td></tr><tr><td>    Multi-exon, complete cds </td><td>81 780 </td><td>80 634 </td></tr><tr><td>ESTs </td><td>5 605 025 </td><td>3 688 726 </td></tr><tr><td>    Multi-exon </td><td>3 261 926 </td><td>1 645 977 </td></tr><tr><td>Multi-exon genes w / EST support </td><td>20 635 </td><td>20 838 </td></tr><tr><td>Genes w/multiple transcripts </td><td>13 560 (66%)<sup>a</sup> </td><td>11 752 (56%)<sup>a</sup> </td></tr><tr><td>Transcripts </td><td>402 995 </td><td>197 361 </td></tr><tr><td>    Class I </td><td>97 286 </td><td>64 255 </td></tr><tr><td>    Class II </td><td>297 601 </td><td>131 167 </td></tr><tr><td>    Class III </td><td>4952 </td><td>1939 </td></tr><tr><td>Avg. No. Class I transcripts/AS gene </td><td>6.5 </td><td>5 </td></tr><tr><td>Exons<sup>b</sup> </td><td>259 031 (41%) </td><td>247 342 (33%) </td></tr><tr><td>    Single exon (i.e. single exon gene) </td><td>4559 </td><td>6076 </td></tr><tr><td>    First exon </td><td>34 234 </td><td>45 436 </td></tr><tr><td>    Internal exon </td><td>190 341 </td><td>182 125 </td></tr><tr><td>    Last exon </td><td>29 897 </td><td>43 441 </td></tr><tr><td>    Alternative exons<sup>c</sup> </td><td>107 194 </td><td>82 503 </td></tr><tr><td>Introns<sup>b</sup> </td><td>237 215 (46%) </td><td>220 878 (37%) </td></tr><tr><td>    GT‥AG </td><td>230 735 </td><td>215 326 </td></tr><tr><td>    GC‥AG </td><td>2859 </td><td>2600 </td></tr><tr><td>    AT‥AC </td><td>301 </td><td>331 </td></tr><tr><td>    Others </td><td>3320 </td><td>2621 </td></tr><tr><td>    Alternative introns<sup>c</sup> </td><td>108 304 </td><td>81 791 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><strong>Human</strong> </th><th><strong>Mouse</strong> </th></tr></thead><tbody><tr><td>Genes </td><td>26 143 </td><td>27 841 </td></tr><tr><td>mRNAs </td><td>178 807 </td><td>266,819 </td></tr><tr><td>    Multi-exon </td><td>155 666 </td><td>194 778 </td></tr><tr><td>    Multi-exon, complete cds </td><td>81 780 </td><td>80 634 </td></tr><tr><td>ESTs </td><td>5 605 025 </td><td>3 688 726 </td></tr><tr><td>    Multi-exon </td><td>3 261 926 </td><td>1 645 977 </td></tr><tr><td>Multi-exon genes w / EST support </td><td>20 635 </td><td>20 838 </td></tr><tr><td>Genes w/multiple transcripts </td><td>13 560 (66%)<sup>a</sup> </td><td>11 752 (56%)<sup>a</sup> </td></tr><tr><td>Transcripts </td><td>402 995 </td><td>197 361 </td></tr><tr><td>    Class I </td><td>97 286 </td><td>64 255 </td></tr><tr><td>    Class II </td><td>297 601 </td><td>131 167 </td></tr><tr><td>    Class III </td><td>4952 </td><td>1939 </td></tr><tr><td>Avg. No. Class I transcripts/AS gene </td><td>6.5 </td><td>5 </td></tr><tr><td>Exons<sup>b</sup> </td><td>259 031 (41%) </td><td>247 342 (33%) </td></tr><tr><td>    Single exon (i.e. single exon gene) </td><td>4559 </td><td>6076 </td></tr><tr><td>    First exon </td><td>34 234 </td><td>45 436 </td></tr><tr><td>    Internal exon </td><td>190 341 </td><td>182 125 </td></tr><tr><td>    Last exon </td><td>29 897 </td><td>43 441 </td></tr><tr><td>    Alternative exons<sup>c</sup> </td><td>107 194 </td><td>82 503 </td></tr><tr><td>Introns<sup>b</sup> </td><td>237 215 (46%) </td><td>220 878 (37%) </td></tr><tr><td>    GT‥AG </td><td>230 735 </td><td>215 326 </td></tr><tr><td>    GC‥AG </td><td>2859 </td><td>2600 </td></tr><tr><td>    AT‥AC </td><td>301 </td><td>331 </td></tr><tr><td>    Others </td><td>3320 </td><td>2621 </td></tr><tr><td>    Alternative introns<sup>c</sup> </td><td>108 304 </td><td>81 791 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The number in parentheses indicates the percentage of genes having multiple transcripts (Class I only) of the multi-exon genes with EST support.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Only the exons and the introns included in Class I transcripts were analysed.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Alternative exons/introns mean exons or introns that are alternatively included in Class I transcripts of the respective gene.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194833" target="_blank">View Large</a></div></div>                        <a id="87194834" scrollto-destination="87194834"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Evaluation of AS events in human <em>SCARF1</em> gene by applying binomial statistics</p></div> </div><div class="table-overflow"><table><tbody><tr><td><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011t1.jpeg?Expires=2147483647&Signature=qf0NwSPwxMxCAOHSiP7w3TjhrRdlB3UvCPZZsHfM0PC7hngS2sRoa1Bfd6Lf7HaMS~iPqk3j6XMU-~0dATk6v~ArjzUYymw2G3sBHotfkOdRr5T0aQGXq7UoE2z07BlOp3Nt4622up~t--BGxxODLR6TRxCflXVw7ZZqKPYFH2rtJ5UUIMKJDsJJRyaSHjuMMRsJiFKmgRQ7x71IoiUSvYD99tF7e2CYTGsWMAdLKPrhprpRyoF-Iz9l0koxFznP3MCIsVCqKSvcF0-bHq4whLSXlmRGIco27LI8MJLQ3qn48j3AVFzdKV-68B~KTdpIvL9MmTik6Qc2sNKeyRq6gQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsl011t1.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011t1.jpeg?Expires=2147483647&Signature=TMVDpCsKWs4bIeNCkvmiTPpixaoz-5xm2wdLbQ1iHWLi8iTcPDOE~2DjL0B1rwB-ey1EAmFDOhdMuHKmRk4jRs-N4SV1v7KI3JiHcS-j--TxinTEooG17d90AvL38o9zhkm34Ij-7RRshXrcxpE9btN5QlzjTyNMVGad38~N8cUxRgo1Xr1SN4pi5BuiyZVXbsZ0j~12O67XMnenUTWRNssF3lWLPlshPirOBAk5v6MBGZLMUIXuez5fowy8dpdosbQU~aI~BQTWQnhUYNTU2hZaEDMrER3zJFdGnOiD9HWDSNXOeqjpsehCPh-ChpOk3xPz93iviC1gQUs8dR2fwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsl011t1.jpeg" /></a></div> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><tbody><tr><td><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011t1.jpeg?Expires=2147483647&Signature=qf0NwSPwxMxCAOHSiP7w3TjhrRdlB3UvCPZZsHfM0PC7hngS2sRoa1Bfd6Lf7HaMS~iPqk3j6XMU-~0dATk6v~ArjzUYymw2G3sBHotfkOdRr5T0aQGXq7UoE2z07BlOp3Nt4622up~t--BGxxODLR6TRxCflXVw7ZZqKPYFH2rtJ5UUIMKJDsJJRyaSHjuMMRsJiFKmgRQ7x71IoiUSvYD99tF7e2CYTGsWMAdLKPrhprpRyoF-Iz9l0koxFznP3MCIsVCqKSvcF0-bHq4whLSXlmRGIco27LI8MJLQ3qn48j3AVFzdKV-68B~KTdpIvL9MmTik6Qc2sNKeyRq6gQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsl011t1.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011t1.jpeg?Expires=2147483647&Signature=TMVDpCsKWs4bIeNCkvmiTPpixaoz-5xm2wdLbQ1iHWLi8iTcPDOE~2DjL0B1rwB-ey1EAmFDOhdMuHKmRk4jRs-N4SV1v7KI3JiHcS-j--TxinTEooG17d90AvL38o9zhkm34Ij-7RRshXrcxpE9btN5QlzjTyNMVGad38~N8cUxRgo1Xr1SN4pi5BuiyZVXbsZ0j~12O67XMnenUTWRNssF3lWLPlshPirOBAk5v6MBGZLMUIXuez5fowy8dpdosbQU~aI~BQTWQnhUYNTU2hZaEDMrER3zJFdGnOiD9HWDSNXOeqjpsehCPh-ChpOk3xPz93iviC1gQUs8dR2fwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsl011t1.jpeg" /></a></div> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>For full-length mRNA queries, each was counted as two as described in Materials and Methods section.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup><em>P</em>-value for the minor form was indicated in the condition of 1% cutoff frequency (freq = 0.01).</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>This complex AS form can not be classified into any of nine AS patterns which we defined at <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>The AS form can be compared to more than one alternative linkage but was only compared to the major linkage with the highest score.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194834" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Evaluation of AS events in human <em>SCARF1</em> gene by applying binomial statistics</p></div> </div><div class="table-overflow"><table><tbody><tr><td><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011t1.jpeg?Expires=2147483647&Signature=qf0NwSPwxMxCAOHSiP7w3TjhrRdlB3UvCPZZsHfM0PC7hngS2sRoa1Bfd6Lf7HaMS~iPqk3j6XMU-~0dATk6v~ArjzUYymw2G3sBHotfkOdRr5T0aQGXq7UoE2z07BlOp3Nt4622up~t--BGxxODLR6TRxCflXVw7ZZqKPYFH2rtJ5UUIMKJDsJJRyaSHjuMMRsJiFKmgRQ7x71IoiUSvYD99tF7e2CYTGsWMAdLKPrhprpRyoF-Iz9l0koxFznP3MCIsVCqKSvcF0-bHq4whLSXlmRGIco27LI8MJLQ3qn48j3AVFzdKV-68B~KTdpIvL9MmTik6Qc2sNKeyRq6gQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsl011t1.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011t1.jpeg?Expires=2147483647&Signature=TMVDpCsKWs4bIeNCkvmiTPpixaoz-5xm2wdLbQ1iHWLi8iTcPDOE~2DjL0B1rwB-ey1EAmFDOhdMuHKmRk4jRs-N4SV1v7KI3JiHcS-j--TxinTEooG17d90AvL38o9zhkm34Ij-7RRshXrcxpE9btN5QlzjTyNMVGad38~N8cUxRgo1Xr1SN4pi5BuiyZVXbsZ0j~12O67XMnenUTWRNssF3lWLPlshPirOBAk5v6MBGZLMUIXuez5fowy8dpdosbQU~aI~BQTWQnhUYNTU2hZaEDMrER3zJFdGnOiD9HWDSNXOeqjpsehCPh-ChpOk3xPz93iviC1gQUs8dR2fwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsl011t1.jpeg" /></a></div> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><tbody><tr><td><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011t1.jpeg?Expires=2147483647&Signature=qf0NwSPwxMxCAOHSiP7w3TjhrRdlB3UvCPZZsHfM0PC7hngS2sRoa1Bfd6Lf7HaMS~iPqk3j6XMU-~0dATk6v~ArjzUYymw2G3sBHotfkOdRr5T0aQGXq7UoE2z07BlOp3Nt4622up~t--BGxxODLR6TRxCflXVw7ZZqKPYFH2rtJ5UUIMKJDsJJRyaSHjuMMRsJiFKmgRQ7x71IoiUSvYD99tF7e2CYTGsWMAdLKPrhprpRyoF-Iz9l0koxFznP3MCIsVCqKSvcF0-bHq4whLSXlmRGIco27LI8MJLQ3qn48j3AVFzdKV-68B~KTdpIvL9MmTik6Qc2sNKeyRq6gQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsl011t1.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011t1.jpeg?Expires=2147483647&Signature=TMVDpCsKWs4bIeNCkvmiTPpixaoz-5xm2wdLbQ1iHWLi8iTcPDOE~2DjL0B1rwB-ey1EAmFDOhdMuHKmRk4jRs-N4SV1v7KI3JiHcS-j--TxinTEooG17d90AvL38o9zhkm34Ij-7RRshXrcxpE9btN5QlzjTyNMVGad38~N8cUxRgo1Xr1SN4pi5BuiyZVXbsZ0j~12O67XMnenUTWRNssF3lWLPlshPirOBAk5v6MBGZLMUIXuez5fowy8dpdosbQU~aI~BQTWQnhUYNTU2hZaEDMrER3zJFdGnOiD9HWDSNXOeqjpsehCPh-ChpOk3xPz93iviC1gQUs8dR2fwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsl011t1.jpeg" /></a></div> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>For full-length mRNA queries, each was counted as two as described in Materials and Methods section.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup><em>P</em>-value for the minor form was indicated in the condition of 1% cutoff frequency (freq = 0.01).</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>This complex AS form can not be classified into any of nine AS patterns which we defined at <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>The AS form can be compared to more than one alternative linkage but was only compared to the major linkage with the highest score.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194834" target="_blank">View Large</a></div></div><p>Given that a transcript is an output of several possible combinations of individual AS events in an individual gene, the prediction and reconstruction of transcript isoforms is another challenging problem. The simplest method is to generate all possible combinations from the graph traversal. As noted above, however, this may produce many false transcripts. Thus, we evaluated its reliability according to EST evidence, and we classified each set of transcripts into three groups. As other good measures to determine the transcript reliability, Kim et al. <a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> classified the transcripts according to the minimum number of sequences required to fully encompass the transcript structure and Xing et al.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> suggested the heaviest bundling algorithm, in which a transcript with only partial alignment information is extended to the maximum length of the path with the maximum likelihood.</p><p>How many genes generate multiple transcripts and how many transcripts exist in humans are very important questions that remain unanswered. Exon-junction microarray analysis shows that ∼75% of human multi-exon genes undergo AS.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> In our case, 52% of all human genes had multiple transcripts, although the percentage of AS genes increases to 66% if we consider only multi-exon genes with EST support. The percentage of AS genes is very similar to the results of other studies based on graph algorithms;<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> however, the number of transcript variants from our study is smaller than other reported values. The low outcome is largely due to the application of strict selection criteria for the AS relationships, namely, a frequency cutoff of 0.01 (1%) with 95% statistical confidence. Without adopting the statistical selection for AS events results in &gt;200 000 Class I transcripts and a total of ∼1 000 000 transcripts for all three classes, which is comparable to the findings of other studies in the ESTGenes, ECgene, ASG and ASD databases (<a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>). In these cases, however, the splice graph will contain many AS events with very low frequency (&lt;0.001) and many transcripts would be generated from the splice graph. We could not guarantee the existence of these AS events in real tissues or organs. Moreover, as mentioned previously,<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> if we allow all of the splice variations based on ESTs, we found that &gt;95% of the multi-exon genes with &gt;300 EST supports undergo AS events. The expectation that almost all genes undergo AS if there are sufficient ESTs does not seem to be reasonable. Thus, we applied a two-step selection strategy as described above. Using our strategy, we found that the percent of AS genes converged to ∼75% even though EST coverage in a gene cluster increased to &gt;1000 (data not shown). Therefore, we suppose the Class I transcripts with small number and AS events defined in our study are the reliable sets confidently selected from all possible combinations.</p>                            <h3 scrollto-destination=87194837 id="87194837" class="section-title" >3.3 Detection of AS events</h3>
<p>For each gene, we defined and examined nine types of AS patterns by exon–exon and exon–intron linkage analysis of Class I transcripts: cassette exon, 5′-splice site variation, 3′-splice site variation, mutually exclusive exon, multiple cassette exon, overlapping exon, composite splice site variation, intron retention and multiple intron retention (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>). In human, cassette exon was the most frequent AS event, followed by intron retention and multiple cassette exon. <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a> summarizes the results of the nine AS patterns. Alternative exons in the defined AS patterns accounted for ∼41% of all alternative exons in our study. Characterization and investigation of more complex AS patterns would allow discovery of additional AS events.</p>                        <a id="87194839" scrollto-destination="87194839"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f3.jpeg?Expires=2147483647&Signature=mKsgOLl7nEn2QhnYWcIKa3I5qX4BYlKqbhKvBvSNn6ev4CIB-e-BfSNK0-Y5a-AQeLw9qDWhomX~sQeBAtSCE3Sh3~TEdg6zC~hqO5tHvAg~-PfwACFXF6nK8rdlm0JV-fDa61uyL1XYVcOHcY4Rk4nmX123SUIHNdgkLvnqjdX5UqH85MoXmW5gfdjPGTkOHXni~CiVnt9JivTBYaqdNGb6HKpNmABDXd9ntTkoVPSuy-xZqY17CgZaYGvbYwcJ8CIVpeOrRBCw7R5-LnsCbVlJ8Jhuap0BsQyoHGX~wbO6h8DMsamdbQFJjUTGx66B7Feozjob~uKqB7ySAaTagA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification of the AS patterns characterized in TISA. The nine types of mutually exclusive AS patterns characterized in this study are represented. Boxes and lines indicate exons and introns, respectively. Alternative exons are indicated as shaded boxes." path-from-xml="dsl011f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194839/dsl011f3.jpeg" path-from-xml="dsl011f3.jpeg" target="_blank">View large</a><a section="87194839" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f3.jpeg?Expires=2147483647&Signature=nyYt-siaqQ6k406f0WdE9NxGaBntlJQ6jTTzU6p9BdofXfAfL6WQStY4UP43OisK-TU1RtmTVGZHHSI--aotwpgaw7qzWUUyOBo1BDmzNWayostFrdADmEkyg259KmY57ZAxX~ZdcvzWqjNm5BCm2Sy-4K3NDIniaYaoBrbnuAqyPmUAGeh5KkaXAthwDRs1yy3ZIaIJSp2j-92-0AoOTp9L-q8SqtaRgtBSxrKkA9CNj-mIDnI6jXf2xc6vj4610nUvLx6dQkvfeAe7mkN6BSc-c1duKWZhXXUzCVw6OZkpf8r-ISs9-7WOuSBky9GsnZF4EnE-P3JnaPhobEiz5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194839&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification of the AS patterns characterized in TISA. The nine types of mutually exclusive AS patterns characterized in this study are represented. Boxes and lines indicate exons and introns, respectively. Alternative exons are indicated as shaded boxes.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f3.jpeg?Expires=2147483647&Signature=mKsgOLl7nEn2QhnYWcIKa3I5qX4BYlKqbhKvBvSNn6ev4CIB-e-BfSNK0-Y5a-AQeLw9qDWhomX~sQeBAtSCE3Sh3~TEdg6zC~hqO5tHvAg~-PfwACFXF6nK8rdlm0JV-fDa61uyL1XYVcOHcY4Rk4nmX123SUIHNdgkLvnqjdX5UqH85MoXmW5gfdjPGTkOHXni~CiVnt9JivTBYaqdNGb6HKpNmABDXd9ntTkoVPSuy-xZqY17CgZaYGvbYwcJ8CIVpeOrRBCw7R5-LnsCbVlJ8Jhuap0BsQyoHGX~wbO6h8DMsamdbQFJjUTGx66B7Feozjob~uKqB7ySAaTagA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification of the AS patterns characterized in TISA. The nine types of mutually exclusive AS patterns characterized in this study are represented. Boxes and lines indicate exons and introns, respectively. Alternative exons are indicated as shaded boxes." path-from-xml="dsl011f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194839/dsl011f3.jpeg" path-from-xml="dsl011f3.jpeg" target="_blank">View large</a><a section="87194839" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f3.jpeg?Expires=2147483647&Signature=nyYt-siaqQ6k406f0WdE9NxGaBntlJQ6jTTzU6p9BdofXfAfL6WQStY4UP43OisK-TU1RtmTVGZHHSI--aotwpgaw7qzWUUyOBo1BDmzNWayostFrdADmEkyg259KmY57ZAxX~ZdcvzWqjNm5BCm2Sy-4K3NDIniaYaoBrbnuAqyPmUAGeh5KkaXAthwDRs1yy3ZIaIJSp2j-92-0AoOTp9L-q8SqtaRgtBSxrKkA9CNj-mIDnI6jXf2xc6vj4610nUvLx6dQkvfeAe7mkN6BSc-c1duKWZhXXUzCVw6OZkpf8r-ISs9-7WOuSBky9GsnZF4EnE-P3JnaPhobEiz5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194839&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification of the AS patterns characterized in TISA. The nine types of mutually exclusive AS patterns characterized in this study are represented. Boxes and lines indicate exons and introns, respectively. Alternative exons are indicated as shaded boxes.</p></div></div>                        <a id="87194840" scrollto-destination="87194840"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>The number of TISA events predicted in human and mouse (<em>P</em> &lt; 0.01)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>AS pattern </th><th colspan="2">Human </th><th colspan="2">Mouse </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>No. AS events (total) </th><th>No. AS events (tissue-specific) [<em>n</em>(%)] </th><th>No. AS events (total) </th><th>No. AS events (tissue-specific) [<em>n</em>(%)] </th></tr></thead><tbody><tr><td>Cassette exon </td><td char=".">13 191 </td><td char=".">551 (4.2) </td><td char=".">6788 </td><td char=".">355 (5.2) </td></tr><tr><td>Multiple cassette exon </td><td char=".">4341 </td><td char=".">102 (2.3) </td><td char=".">1646 </td><td char=".">59 (3.6) </td></tr><tr><td>    5′-splice site variation </td><td char=".">2586 </td><td char=".">97 (3.8) </td><td char=".">1725 </td><td char=".">63 (3.7) </td></tr><tr><td>    3′-splice site variation </td><td char=".">3711 </td><td char=".">106 (2.9) </td><td char=".">2211 </td><td char=".">88 (4.0) </td></tr><tr><td>Mutually exclusive exon </td><td char=".">1232 </td><td char=".">33 (2.7) </td><td char=".">457 </td><td char=".">22 (4.8) </td></tr><tr><td>Composite splice variation </td><td char=".">1356 </td><td char=".">25 (1.8) </td><td char=".">646 </td><td char=".">16 (2.5) </td></tr><tr><td>Overlapping exon </td><td char=".">179 </td><td char=".">2 (1.1) </td><td char=".">51 </td><td char=".">2 (3.9) </td></tr><tr><td>Intron retention </td><td char=".">5551 </td><td char=".">119 (2.1) </td><td char=".">3223 </td><td char=".">91 (2.8) </td></tr><tr><td>Multiple intron retention </td><td char=".">1334 </td><td>- </td><td char=".">659 </td><td> </td></tr><tr><td>Total </td><td char=".">33 481 </td><td char=".">1035 (3.1) </td><td char=".">17 406 </td><td char=".">696 (4.0) </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>AS pattern </th><th colspan="2">Human </th><th colspan="2">Mouse </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>No. AS events (total) </th><th>No. AS events (tissue-specific) [<em>n</em>(%)] </th><th>No. AS events (total) </th><th>No. AS events (tissue-specific) [<em>n</em>(%)] </th></tr></thead><tbody><tr><td>Cassette exon </td><td char=".">13 191 </td><td char=".">551 (4.2) </td><td char=".">6788 </td><td char=".">355 (5.2) </td></tr><tr><td>Multiple cassette exon </td><td char=".">4341 </td><td char=".">102 (2.3) </td><td char=".">1646 </td><td char=".">59 (3.6) </td></tr><tr><td>    5′-splice site variation </td><td char=".">2586 </td><td char=".">97 (3.8) </td><td char=".">1725 </td><td char=".">63 (3.7) </td></tr><tr><td>    3′-splice site variation </td><td char=".">3711 </td><td char=".">106 (2.9) </td><td char=".">2211 </td><td char=".">88 (4.0) </td></tr><tr><td>Mutually exclusive exon </td><td char=".">1232 </td><td char=".">33 (2.7) </td><td char=".">457 </td><td char=".">22 (4.8) </td></tr><tr><td>Composite splice variation </td><td char=".">1356 </td><td char=".">25 (1.8) </td><td char=".">646 </td><td char=".">16 (2.5) </td></tr><tr><td>Overlapping exon </td><td char=".">179 </td><td char=".">2 (1.1) </td><td char=".">51 </td><td char=".">2 (3.9) </td></tr><tr><td>Intron retention </td><td char=".">5551 </td><td char=".">119 (2.1) </td><td char=".">3223 </td><td char=".">91 (2.8) </td></tr><tr><td>Multiple intron retention </td><td char=".">1334 </td><td>- </td><td char=".">659 </td><td> </td></tr><tr><td>Total </td><td char=".">33 481 </td><td char=".">1035 (3.1) </td><td char=".">17 406 </td><td char=".">696 (4.0) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194840" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>The number of TISA events predicted in human and mouse (<em>P</em> &lt; 0.01)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>AS pattern </th><th colspan="2">Human </th><th colspan="2">Mouse </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>No. AS events (total) </th><th>No. AS events (tissue-specific) [<em>n</em>(%)] </th><th>No. AS events (total) </th><th>No. AS events (tissue-specific) [<em>n</em>(%)] </th></tr></thead><tbody><tr><td>Cassette exon </td><td char=".">13 191 </td><td char=".">551 (4.2) </td><td char=".">6788 </td><td char=".">355 (5.2) </td></tr><tr><td>Multiple cassette exon </td><td char=".">4341 </td><td char=".">102 (2.3) </td><td char=".">1646 </td><td char=".">59 (3.6) </td></tr><tr><td>    5′-splice site variation </td><td char=".">2586 </td><td char=".">97 (3.8) </td><td char=".">1725 </td><td char=".">63 (3.7) </td></tr><tr><td>    3′-splice site variation </td><td char=".">3711 </td><td char=".">106 (2.9) </td><td char=".">2211 </td><td char=".">88 (4.0) </td></tr><tr><td>Mutually exclusive exon </td><td char=".">1232 </td><td char=".">33 (2.7) </td><td char=".">457 </td><td char=".">22 (4.8) </td></tr><tr><td>Composite splice variation </td><td char=".">1356 </td><td char=".">25 (1.8) </td><td char=".">646 </td><td char=".">16 (2.5) </td></tr><tr><td>Overlapping exon </td><td char=".">179 </td><td char=".">2 (1.1) </td><td char=".">51 </td><td char=".">2 (3.9) </td></tr><tr><td>Intron retention </td><td char=".">5551 </td><td char=".">119 (2.1) </td><td char=".">3223 </td><td char=".">91 (2.8) </td></tr><tr><td>Multiple intron retention </td><td char=".">1334 </td><td>- </td><td char=".">659 </td><td> </td></tr><tr><td>Total </td><td char=".">33 481 </td><td char=".">1035 (3.1) </td><td char=".">17 406 </td><td char=".">696 (4.0) </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>AS pattern </th><th colspan="2">Human </th><th colspan="2">Mouse </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>No. AS events (total) </th><th>No. AS events (tissue-specific) [<em>n</em>(%)] </th><th>No. AS events (total) </th><th>No. AS events (tissue-specific) [<em>n</em>(%)] </th></tr></thead><tbody><tr><td>Cassette exon </td><td char=".">13 191 </td><td char=".">551 (4.2) </td><td char=".">6788 </td><td char=".">355 (5.2) </td></tr><tr><td>Multiple cassette exon </td><td char=".">4341 </td><td char=".">102 (2.3) </td><td char=".">1646 </td><td char=".">59 (3.6) </td></tr><tr><td>    5′-splice site variation </td><td char=".">2586 </td><td char=".">97 (3.8) </td><td char=".">1725 </td><td char=".">63 (3.7) </td></tr><tr><td>    3′-splice site variation </td><td char=".">3711 </td><td char=".">106 (2.9) </td><td char=".">2211 </td><td char=".">88 (4.0) </td></tr><tr><td>Mutually exclusive exon </td><td char=".">1232 </td><td char=".">33 (2.7) </td><td char=".">457 </td><td char=".">22 (4.8) </td></tr><tr><td>Composite splice variation </td><td char=".">1356 </td><td char=".">25 (1.8) </td><td char=".">646 </td><td char=".">16 (2.5) </td></tr><tr><td>Overlapping exon </td><td char=".">179 </td><td char=".">2 (1.1) </td><td char=".">51 </td><td char=".">2 (3.9) </td></tr><tr><td>Intron retention </td><td char=".">5551 </td><td char=".">119 (2.1) </td><td char=".">3223 </td><td char=".">91 (2.8) </td></tr><tr><td>Multiple intron retention </td><td char=".">1334 </td><td>- </td><td char=".">659 </td><td> </td></tr><tr><td>Total </td><td char=".">33 481 </td><td char=".">1035 (3.1) </td><td char=".">17 406 </td><td char=".">696 (4.0) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194840" target="_blank">View Large</a></div></div>                        <a id="87194841" scrollto-destination="87194841"></a>
<div content-id="tbl4" class="table-modal"><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Comparison of current human AS databases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>No. genes </th><th>No. AS genes [<em>n</em> (%)] </th><th>No. transcripts </th><th>Algorithm </th></tr></thead><tbody><tr><td>ESTGenes </td><td>38 581 </td><td>∼16 590 (43) </td><td>122 247 </td><td>clusterMerge </td></tr><tr><td>ECgene (Part A) </td><td>38 238 </td><td>9 312 (26) </td><td>154 200 </td><td>Graph (Ver. 1.2) </td></tr><tr><td>ASG </td><td>22 127 </td><td>14 405 (65) </td><td>&gt; 1.2 million </td><td>Graph </td></tr><tr><td>ASD </td><td>22 216 </td><td>13 572 (61) </td><td>898 295 </td><td>Graph (Rel. 2) </td></tr><tr><td>TISA (Class I) </td><td>26 143 </td><td>13 560 (66) </td><td>97 286 </td><td>Graph (Ver. 1.0) </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>No. genes </th><th>No. AS genes [<em>n</em> (%)] </th><th>No. transcripts </th><th>Algorithm </th></tr></thead><tbody><tr><td>ESTGenes </td><td>38 581 </td><td>∼16 590 (43) </td><td>122 247 </td><td>clusterMerge </td></tr><tr><td>ECgene (Part A) </td><td>38 238 </td><td>9 312 (26) </td><td>154 200 </td><td>Graph (Ver. 1.2) </td></tr><tr><td>ASG </td><td>22 127 </td><td>14 405 (65) </td><td>&gt; 1.2 million </td><td>Graph </td></tr><tr><td>ASD </td><td>22 216 </td><td>13 572 (61) </td><td>898 295 </td><td>Graph (Rel. 2) </td></tr><tr><td>TISA (Class I) </td><td>26 143 </td><td>13 560 (66) </td><td>97 286 </td><td>Graph (Ver. 1.0) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194841" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Comparison of current human AS databases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>No. genes </th><th>No. AS genes [<em>n</em> (%)] </th><th>No. transcripts </th><th>Algorithm </th></tr></thead><tbody><tr><td>ESTGenes </td><td>38 581 </td><td>∼16 590 (43) </td><td>122 247 </td><td>clusterMerge </td></tr><tr><td>ECgene (Part A) </td><td>38 238 </td><td>9 312 (26) </td><td>154 200 </td><td>Graph (Ver. 1.2) </td></tr><tr><td>ASG </td><td>22 127 </td><td>14 405 (65) </td><td>&gt; 1.2 million </td><td>Graph </td></tr><tr><td>ASD </td><td>22 216 </td><td>13 572 (61) </td><td>898 295 </td><td>Graph (Rel. 2) </td></tr><tr><td>TISA (Class I) </td><td>26 143 </td><td>13 560 (66) </td><td>97 286 </td><td>Graph (Ver. 1.0) </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>No. genes </th><th>No. AS genes [<em>n</em> (%)] </th><th>No. transcripts </th><th>Algorithm </th></tr></thead><tbody><tr><td>ESTGenes </td><td>38 581 </td><td>∼16 590 (43) </td><td>122 247 </td><td>clusterMerge </td></tr><tr><td>ECgene (Part A) </td><td>38 238 </td><td>9 312 (26) </td><td>154 200 </td><td>Graph (Ver. 1.2) </td></tr><tr><td>ASG </td><td>22 127 </td><td>14 405 (65) </td><td>&gt; 1.2 million </td><td>Graph </td></tr><tr><td>ASD </td><td>22 216 </td><td>13 572 (61) </td><td>898 295 </td><td>Graph (Rel. 2) </td></tr><tr><td>TISA (Class I) </td><td>26 143 </td><td>13 560 (66) </td><td>97 286 </td><td>Graph (Ver. 1.0) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194841" target="_blank">View Large</a></div></div><p>Because the method depends on mRNA/EST evidence, it cannot identify novel AS events if there is no sequence evidence. Usually, ESTs are generated by single-pass sequencing of cDNA clones at both ends, resulting in relatively low coverage in the genes' internal region. Although &gt;7.5 million human ESTs are in the public database, we could not determine whether that is sufficient for the AS detection. To overcome this problem, several approaches have been employed, including splice microarrays to detect AS,<a reveal-id="b16  b18" data-open="b16  b18" class="link link-ref link-reveal xref-bibr">16–18</a> machine learning techniques such as support vector machine or hidden Markov models,<a reveal-id="b19  b21" data-open="b19  b21" class="link link-ref link-reveal xref-bibr">19–21</a> and application of comparative genomics.<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> Such methods can be used to detect AS in the internal regions or in the genes with little or no sequence evidence; however, <em>ab initio</em> prediction accompanying the machine learning technique is in a preliminary stage, so the research has so far been conducted on only a few simple AS patterns, such as cassette exon and/or intron retention. As for the splice microarray, its weakness is that the experiment is possible only for already known exon sets, and the detection range is limited to a few of simple AS events, similar to the results obtained so far with the machine learning approach. Therefore, such methods can be possible using the information obtained from EST-based analysis and can enhance the overall sensitivity of AS detection complementarily with the EST-based method.</p>                            <h3 scrollto-destination=87194843 id="87194843" class="section-title" >3.4 Statistical analysis of the tissue specificity of genes and transcript variants</h3>
<p>Using tissue information for ESTs, we have constructed digital expression profiles for each gene and transcript. From 8616 cDNA libraries, there were 7 544 537 ESTs classified into 46 human tissues. Individual ESTs were classified as normal- or cancer-derived according to their histological status. We used only ‘normal’ ESTs. If the proportion of ESTs from a specific tissue was significantly higher than that from other tissues, the gene would have a higher probability of tissue-specific expression in that tissue. In the example shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>, there are two possible isoforms in human TFAP2B (transcription factor AP-2 beta) gene. Among 36 supporting ESTs, 24 are common to both isoforms and 4 are singletons. For the tissue specificity of the gene in retina tissue, a 2 × 2 contingency table was constructed (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4C</a>) and the <em>P</em>-value was calculated as described in the Materials and Methods. In this case, the sample gene is probably retina-specific because its <em>P</em>-value was much &lt;0.01 (99% confidence level). For the other tissues, the same evaluations were performed sequentially. The contingency table of the AS event was constructed in the same way (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4D</a>) as for retina tissue (i.e. cassette exon event in E2 and the <em>P</em>-value calculated using Fisher's exact test). In this case, the cassette exon event is not predicted to be retina-specific in 99% confidence level (however it is significantly retina-specific in 95% confidence level).</p>                        <a id="87194845" scrollto-destination="87194845"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f4.jpeg?Expires=2147483647&Signature=oV4luuC9l~I21ZLwd1yxieWKU3gMXV9I1cmECO0PK9hWlC0b8OhWrEja0KOgysb99Y1uCcG108IFTnFjIpIbiPtdoalKDeKTp-0a5UAxXkh5qBMcbp8Kyy2x3bUObHMxaBcBQHN-zXBRy7qOhQSOxXlh-Q4HOzmUMtXfnUEGfEaqBt-gzhY~Rz-T9SrQpcyV1DzpLD6dTkSNH7hD2mpkbIxYlh0uiAeVvIErxiBhCFeP2Qd8O6kfqL8dsWBZoJmjcbKKN-tv1C3j8VYhFxPVfbQTg9p8hQcpH~-qx3A-s15esgF63lLjHcmT-ColqVw043axZDD9KDSPyFZv0aFnAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of tissue specificity of the gene, transcripts and AS event. (A) The gene cluster of human TFAP2B with two isoforms. There are two transcript isoforms possible from the gene cluster, with 34 supporting ESTs. A cassette exon event in the second exon is marked with a dashed rectangle. In this example, we considered only ESTs from normal libraries. The tissue from which EST was derived is indicated. (B) Tissue distribution table of ESTs for the gene and isoforms. (C) The 2 × 2 contingency table for tissue specificity calculation of the gene in retina tissue. The numbers of total ESTs and retina-specific ESTs were derived from the human clustering data. The P-value calculated by Audic's method is indicated. (D) The contingency table for tissue specificity of the AS event in retina tissue. The P-value calculated by Fisher's exact test is indicated. (E) Tissue distribution table of ESTs for isoform 1. (F) The contingency tables for tissue specificity calculation of the transcript isoforms in retina tissue in Cases 1 and 2 (see the text for details). The P-values calculated by Audic's method are indicated for each case. Singleton ESTs were included at tissue-counting for the gene but were excluded at counting for transcript isoforms." path-from-xml="dsl011f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194845/dsl011f4.jpeg" path-from-xml="dsl011f4.jpeg" target="_blank">View large</a><a section="87194845" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f4.jpeg?Expires=2147483647&Signature=atl4YZ-lnue6skIS2NC60G0QEYNuWfTjuwHKjG886axVYXZ9x2tHz3Y1hMqHpslB32N~0pGYygn~UYkVJ-NPOgRCIrk4Q9ftACMOQ8mUIXBgk3utod4nfWno2JCbIJEtEe~UJz6w~i98u0C3949HvGPyIGJqNCt-u3cN0R6oFnS~Qjns3WR7nZ9cIxva4~7sCpIV8E35lm4~pHIOC89FOWBx6sf9bEZaUDRIpk6dpkQLAua38kNQPVfbVIY2q2aBC6rQ7lfUwqZfmyAHw5tDminvb64~~uPMEQQpmp~WRqY2gAK0MW3xMYCXC5D6wnBwThLGT17x7uMp7WFc8jxIoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194845&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of tissue specificity of the gene, transcripts and AS event. (<strong>A</strong>) The gene cluster of human TFAP2B with two isoforms. There are two transcript isoforms possible from the gene cluster, with 34 supporting ESTs. A cassette exon event in the second exon is marked with a dashed rectangle. In this example, we considered only ESTs from normal libraries. The tissue from which EST was derived is indicated. (<strong>B</strong>) Tissue distribution table of ESTs for the gene and isoforms. (<strong>C</strong>) The 2 × 2 contingency table for tissue specificity calculation of the gene in retina tissue. The numbers of total ESTs and retina-specific ESTs were derived from the human clustering data. The <em>P</em>-value calculated by Audic's method is indicated. (<strong>D</strong>) The contingency table for tissue specificity of the AS event in retina tissue. The <em>P</em>-value calculated by Fisher's exact test is indicated. (<strong>E</strong>) Tissue distribution table of ESTs for isoform 1. (<strong>F</strong>) The contingency tables for tissue specificity calculation of the transcript isoforms in retina tissue in Cases 1 and 2 (see the text for details). The <em>P</em>-values calculated by Audic's method are indicated for each case. Singleton ESTs were included at tissue-counting for the gene but were excluded at counting for transcript isoforms.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f4.jpeg?Expires=2147483647&Signature=oV4luuC9l~I21ZLwd1yxieWKU3gMXV9I1cmECO0PK9hWlC0b8OhWrEja0KOgysb99Y1uCcG108IFTnFjIpIbiPtdoalKDeKTp-0a5UAxXkh5qBMcbp8Kyy2x3bUObHMxaBcBQHN-zXBRy7qOhQSOxXlh-Q4HOzmUMtXfnUEGfEaqBt-gzhY~Rz-T9SrQpcyV1DzpLD6dTkSNH7hD2mpkbIxYlh0uiAeVvIErxiBhCFeP2Qd8O6kfqL8dsWBZoJmjcbKKN-tv1C3j8VYhFxPVfbQTg9p8hQcpH~-qx3A-s15esgF63lLjHcmT-ColqVw043axZDD9KDSPyFZv0aFnAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of tissue specificity of the gene, transcripts and AS event. (A) The gene cluster of human TFAP2B with two isoforms. There are two transcript isoforms possible from the gene cluster, with 34 supporting ESTs. A cassette exon event in the second exon is marked with a dashed rectangle. In this example, we considered only ESTs from normal libraries. The tissue from which EST was derived is indicated. (B) Tissue distribution table of ESTs for the gene and isoforms. (C) The 2 × 2 contingency table for tissue specificity calculation of the gene in retina tissue. The numbers of total ESTs and retina-specific ESTs were derived from the human clustering data. The P-value calculated by Audic's method is indicated. (D) The contingency table for tissue specificity of the AS event in retina tissue. The P-value calculated by Fisher's exact test is indicated. (E) Tissue distribution table of ESTs for isoform 1. (F) The contingency tables for tissue specificity calculation of the transcript isoforms in retina tissue in Cases 1 and 2 (see the text for details). The P-values calculated by Audic's method are indicated for each case. Singleton ESTs were included at tissue-counting for the gene but were excluded at counting for transcript isoforms." path-from-xml="dsl011f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194845/dsl011f4.jpeg" path-from-xml="dsl011f4.jpeg" target="_blank">View large</a><a section="87194845" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f4.jpeg?Expires=2147483647&Signature=atl4YZ-lnue6skIS2NC60G0QEYNuWfTjuwHKjG886axVYXZ9x2tHz3Y1hMqHpslB32N~0pGYygn~UYkVJ-NPOgRCIrk4Q9ftACMOQ8mUIXBgk3utod4nfWno2JCbIJEtEe~UJz6w~i98u0C3949HvGPyIGJqNCt-u3cN0R6oFnS~Qjns3WR7nZ9cIxva4~7sCpIV8E35lm4~pHIOC89FOWBx6sf9bEZaUDRIpk6dpkQLAua38kNQPVfbVIY2q2aBC6rQ7lfUwqZfmyAHw5tDminvb64~~uPMEQQpmp~WRqY2gAK0MW3xMYCXC5D6wnBwThLGT17x7uMp7WFc8jxIoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194845&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of tissue specificity of the gene, transcripts and AS event. (<strong>A</strong>) The gene cluster of human TFAP2B with two isoforms. There are two transcript isoforms possible from the gene cluster, with 34 supporting ESTs. A cassette exon event in the second exon is marked with a dashed rectangle. In this example, we considered only ESTs from normal libraries. The tissue from which EST was derived is indicated. (<strong>B</strong>) Tissue distribution table of ESTs for the gene and isoforms. (<strong>C</strong>) The 2 × 2 contingency table for tissue specificity calculation of the gene in retina tissue. The numbers of total ESTs and retina-specific ESTs were derived from the human clustering data. The <em>P</em>-value calculated by Audic's method is indicated. (<strong>D</strong>) The contingency table for tissue specificity of the AS event in retina tissue. The <em>P</em>-value calculated by Fisher's exact test is indicated. (<strong>E</strong>) Tissue distribution table of ESTs for isoform 1. (<strong>F</strong>) The contingency tables for tissue specificity calculation of the transcript isoforms in retina tissue in Cases 1 and 2 (see the text for details). The <em>P</em>-values calculated by Audic's method are indicated for each case. Singleton ESTs were included at tissue-counting for the gene but were excluded at counting for transcript isoforms.</p></div></div><p>The tissue specificity of the transcript level is determined using essentially the same approach as the tissue specificity of the genes, although one EST could be included in both transcripts. To handle this multiple inclusion problem, we tried two different tissue-EST counting methods. If the multiple inclusion of an EST is ignored, it is possible to obtain a tissue-EST table like that shown for Case 1 in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4E</a>. The default method is to calculate the tissue specificity as for the genes from the table (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4F</a>, Case 1). To improve this method, we introduced a weighting factor that provides a different score for EST tissue counting (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4E</a>, Case 2). If an EST is derived from a certain tissue and is included in <em>n</em> transcript isoforms, its tissue contribution is given as <em>1/n</em>. The tissue count was rounded off to the nearest integer prior to the <em>P</em>-value calculation. In our study, we chose the Case 2 counting method for tissue-specific analysis of transcripts.</p><p>By applying Audic's test and the Benjamini–Yekutieli method, we predicted 12 711 tissue-specific genes and 16 016 tissue-specific transcripts (<a reveal-id="tbl5" data-open="tbl5" class="link link-reveal link-table xref-fig">Table 5</a>) for the condition of FDR &lt;0.01 and a tissue-EST count ≥3. In mouse, 15 895 genes and 19861 transcripts were predicted to be tissue-specific in 44 tissues. It should be noted that a gene or a transcript can be predicted to be tissue-specific in more than one tissue. Among tissue-specific genes and transcripts in human, 7522 genes (59%) and 11 592 transcripts (72%) were specific only to a single tissue. We have checked the conservation of tissue-specificity between orthologous human and mouse gene pair. Orthologous groups were assigned to each gene according to HomoloGene (NCBI Build48.1). Of 4965 orthologous pairs specific to a single tissue, 11.7% (584 genes) were conserved in its tissue-specificity. In particular, testis-specific genes occupied a major portion (58%) among these tissue-conserved genes (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a>).</p>                        <a id="87194848" scrollto-destination="87194848"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f5.jpeg?Expires=2147483647&Signature=D5I6EyzRb5wOyfHB8ROmcs7RxdMaITcDKuBU2vijFRsuQRlLkm3QA2v-9OVjCZpj2Qc7EdZySnTT2rPhta-osxmL0Bj6idEAImHFQDa8NHEeMuQSKd1p6BdbKMY0nV6a9KMVszg5cCP5KcVeYJyKznKeMH0-eedD7shI-YOfYW6XDnZi9a0REogL~TJFTKnBsMOgvRADzxf9jgcI7gqQLhShy1knUxL5JaT4tDKkDdNPp6IoangAlEdgCZA~r9UIyz8ejdx9sUuyXJxibONXtSwROi07Npg4tfzoZP2z0O6jZ~SS1kyA9LpWmIce7x~x2nYHYc0yfU3UvcE-dFpgvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tissue- and GO term-distribution of 584 tissue-specific genes evolutionarily conserved between human and mouse. (A) Tissue distribution. (B) GO term distribution in biological process section." path-from-xml="dsl011f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194848/dsl011f5.jpeg" path-from-xml="dsl011f5.jpeg" target="_blank">View large</a><a section="87194848" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f5.jpeg?Expires=2147483647&Signature=wZF4eYRAmUj-2Rn1rOpDdNCo24EWmnJXQBnCD1nkbH8rzw3pMqNFQM-tBzKH8etnl1WAQxGGmHiwxdVEb6Yhwmm3lURsVkudsp2kdKuACzKu~SDSnhldDRimkD0h2~TCxhEWp4Pta7412Tk7vNV7noD3rYUae5DHGWMeB-RmbKn3FI2XQnycCR0kaK5BJMKezycHvi~kJqyOWbnwnLoS9fWLUW7CtuzkUywq5wGeDmzYBle0SZkmIV~~gjKCcXXKZ3zEV55lUosF4V3uiGJUqIZvIIF46P~krHbUUlHRa45Iyk7eGwWIPaW10b-he3hkKSOrZaXDJNzvkokYwgkYVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194848&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tissue- and GO term-distribution of 584 tissue-specific genes evolutionarily conserved between human and mouse. (<strong>A</strong>) Tissue distribution. (<strong>B</strong>) GO term distribution in biological process section.</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/m_dsl011f5.jpeg?Expires=2147483647&Signature=D5I6EyzRb5wOyfHB8ROmcs7RxdMaITcDKuBU2vijFRsuQRlLkm3QA2v-9OVjCZpj2Qc7EdZySnTT2rPhta-osxmL0Bj6idEAImHFQDa8NHEeMuQSKd1p6BdbKMY0nV6a9KMVszg5cCP5KcVeYJyKznKeMH0-eedD7shI-YOfYW6XDnZi9a0REogL~TJFTKnBsMOgvRADzxf9jgcI7gqQLhShy1knUxL5JaT4tDKkDdNPp6IoangAlEdgCZA~r9UIyz8ejdx9sUuyXJxibONXtSwROi07Npg4tfzoZP2z0O6jZ~SS1kyA9LpWmIce7x~x2nYHYc0yfU3UvcE-dFpgvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Tissue- and GO term-distribution of 584 tissue-specific genes evolutionarily conserved between human and mouse. (A) Tissue distribution. (B) GO term distribution in biological process section." path-from-xml="dsl011f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194848/dsl011f5.jpeg" path-from-xml="dsl011f5.jpeg" target="_blank">View large</a><a section="87194848" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011f5.jpeg?Expires=2147483647&Signature=wZF4eYRAmUj-2Rn1rOpDdNCo24EWmnJXQBnCD1nkbH8rzw3pMqNFQM-tBzKH8etnl1WAQxGGmHiwxdVEb6Yhwmm3lURsVkudsp2kdKuACzKu~SDSnhldDRimkD0h2~TCxhEWp4Pta7412Tk7vNV7noD3rYUae5DHGWMeB-RmbKn3FI2XQnycCR0kaK5BJMKezycHvi~kJqyOWbnwnLoS9fWLUW7CtuzkUywq5wGeDmzYBle0SZkmIV~~gjKCcXXKZ3zEV55lUosF4V3uiGJUqIZvIIF46P~krHbUUlHRa45Iyk7eGwWIPaW10b-he3hkKSOrZaXDJNzvkokYwgkYVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194848&ar=400632&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl011f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tissue- and GO term-distribution of 584 tissue-specific genes evolutionarily conserved between human and mouse. (<strong>A</strong>) Tissue distribution. (<strong>B</strong>) GO term distribution in biological process section.</p></div></div>                        <a id="87194849" scrollto-destination="87194849"></a>
<div content-id="tbl5" class="table-modal"><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>The number of tissue-specific genes and transcripts predicted in human and mouse</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissues </th><th colspan="2">Human </th><th colspan="2">Mouse </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>No. Genes </th><th>No. Transcripts </th><th>No. Genes </th><th>No. Transcripts </th></tr></thead><tbody><tr><td>Bone </td><td char=".">180 </td><td char=".">190 </td><td char=".">602 </td><td char=".">558 </td></tr><tr><td>Bone marrow </td><td char=".">207 </td><td char=".">244 </td><td char=".">1253 </td><td char=".">1563 </td></tr><tr><td>Brain </td><td char=".">1372 </td><td char=".">2066 </td><td char=".">1971 </td><td char=".">1866 </td></tr><tr><td>Cartilage </td><td char=".">206 </td><td char=".">250 </td><td char=".">6 </td><td char=".">2 </td></tr><tr><td>Cerebellum </td><td char=".">809 </td><td char=".">1092 </td><td char=".">1283 </td><td char=".">1137 </td></tr><tr><td>Cerebrum </td><td char=".">915 </td><td char=".">1195 </td><td char=".">1771 </td><td char=".">2426 </td></tr><tr><td>Cervix </td><td char=".">53 </td><td char=".">50 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>Colon </td><td char=".">320 </td><td char=".">319 </td><td char=".">693 </td><td char=".">775 </td></tr><tr><td>Ear </td><td char=".">298 </td><td char=".">316 </td><td char=".">745 </td><td char=".">739 </td></tr><tr><td>Endocrine </td><td char=".">184 </td><td char=".">167 </td><td char=".">165 </td><td char=".">105 </td></tr><tr><td>Eye </td><td char=".">869 </td><td char=".">676 </td><td char=".">842 </td><td char=".">697 </td></tr><tr><td>Gastrointestinal tract </td><td char=".">287 </td><td char=".">291 </td><td char=".">750 </td><td char=".">770 </td></tr><tr><td>Genitourinary </td><td char=".">153 </td><td char=".">120 </td><td char=".">693 </td><td char=".">557 </td></tr><tr><td>Heart </td><td char=".">587 </td><td char=".">588 </td><td char=".">522 </td><td char=".">505 </td></tr><tr><td>Kidney </td><td char=".">931 </td><td char=".">1002 </td><td char=".">960 </td><td char=".">1197 </td></tr><tr><td>Limb </td><td char=".">0 </td><td char=".">0 </td><td char=".">202 </td><td char=".">152 </td></tr><tr><td>Liver </td><td char=".">624 </td><td char=".">821 </td><td char=".">812 </td><td char=".">1115 </td></tr><tr><td>Lung </td><td char=".">628 </td><td char=".">564 </td><td char=".">503 </td><td char=".">390 </td></tr><tr><td>Lymphoreticular </td><td char=".">340 </td><td char=".">425 </td><td char=".">154 </td><td char=".">119 </td></tr><tr><td>Lymph node </td><td char=".">1255 </td><td char=".">1100 </td><td char=".">418 </td><td char=".">240 </td></tr><tr><td>Mammary gland </td><td char=".">559 </td><td char=".">514 </td><td char=".">1112 </td><td char=".">972 </td></tr><tr><td>Muscle </td><td char=".">560 </td><td char=".">807 </td><td char=".">460 </td><td char=".">468 </td></tr><tr><td>Nervous </td><td char=".">1693 </td><td char=".">1731 </td><td char=".">1950 </td><td char=".">2551 </td></tr><tr><td>Ovary </td><td char=".">107 </td><td char=".">61 </td><td char=".">239 </td><td char=".">168 </td></tr><tr><td>Pancreas </td><td char=".">197 </td><td char=".">217 </td><td char=".">751 </td><td char=".">782 </td></tr><tr><td>Pancreatic islet </td><td char=".">809 </td><td char=".">907 </td><td char=".">435 </td><td char=".">457 </td></tr><tr><td>Peripheral nervous system </td><td char=".">95 </td><td char=".">73 </td><td char=".">372 </td><td char=".">314 </td></tr><tr><td>Pineal gland </td><td char=".">164 </td><td char=".">99 </td><td char=".">112 </td><td char=".">72 </td></tr><tr><td>Pituitary gland </td><td char=".">130 </td><td char=".">88 </td><td char=".">166 </td><td char=".">117 </td></tr><tr><td>Placenta </td><td char=".">1238 </td><td char=".">1601 </td><td char=".">527 </td><td char=".">628 </td></tr><tr><td>Prostate </td><td char=".">614 </td><td char=".">596 </td><td char=".">189 </td><td char=".">203 </td></tr><tr><td>Retina </td><td char=".">539 </td><td char=".">427 </td><td char=".">830 </td><td char=".">680 </td></tr><tr><td>Salivary gland </td><td char=".">37 </td><td char=".">35 </td><td char=".">156 </td><td char=".">208 </td></tr><tr><td>Skin </td><td char=".">661 </td><td char=".">530 </td><td char=".">848 </td><td char=".">684 </td></tr><tr><td>Soft tissue </td><td char=".">153 </td><td char=".">111 </td><td char=".">293 </td><td char=".">244 </td></tr><tr><td>Spleen </td><td char=".">502 </td><td char=".">548 </td><td char=".">1236 </td><td char=".">1301 </td></tr><tr><td>Stomach </td><td char=".">364 </td><td char=".">387 </td><td char=".">530 </td><td char=".">424 </td></tr><tr><td>Synovium </td><td char=".">4 </td><td char=".">2 </td><td char=".">431 </td><td char=".">490 </td></tr><tr><td>Testis </td><td char=".">1865 </td><td char=".">1748 </td><td char=".">2146 </td><td char=".">2201 </td></tr><tr><td>Thymus </td><td char=".">94 </td><td char=".">40 </td><td char=".">1 626 </td><td char=".">1375 </td></tr><tr><td>Thyroid </td><td char=".">207 </td><td char=".">152 </td><td char=".">176 </td><td char=".">168 </td></tr><tr><td>Uterus </td><td char=".">533 </td><td char=".">399 </td><td char=".">134 </td><td char=".">84 </td></tr><tr><td>Vascular </td><td char=".">423 </td><td char=".">461 </td><td char=".">577 </td><td char=".">419 </td></tr><tr><td>Total </td><td char=".">12 711 </td><td char=".">16 016 </td><td char=".">15 895 </td><td char=".">19 861 </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissues </th><th colspan="2">Human </th><th colspan="2">Mouse </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>No. Genes </th><th>No. Transcripts </th><th>No. Genes </th><th>No. Transcripts </th></tr></thead><tbody><tr><td>Bone </td><td char=".">180 </td><td char=".">190 </td><td char=".">602 </td><td char=".">558 </td></tr><tr><td>Bone marrow </td><td char=".">207 </td><td char=".">244 </td><td char=".">1253 </td><td char=".">1563 </td></tr><tr><td>Brain </td><td char=".">1372 </td><td char=".">2066 </td><td char=".">1971 </td><td char=".">1866 </td></tr><tr><td>Cartilage </td><td char=".">206 </td><td char=".">250 </td><td char=".">6 </td><td char=".">2 </td></tr><tr><td>Cerebellum </td><td char=".">809 </td><td char=".">1092 </td><td char=".">1283 </td><td char=".">1137 </td></tr><tr><td>Cerebrum </td><td char=".">915 </td><td char=".">1195 </td><td char=".">1771 </td><td char=".">2426 </td></tr><tr><td>Cervix </td><td char=".">53 </td><td char=".">50 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>Colon </td><td char=".">320 </td><td char=".">319 </td><td char=".">693 </td><td char=".">775 </td></tr><tr><td>Ear </td><td char=".">298 </td><td char=".">316 </td><td char=".">745 </td><td char=".">739 </td></tr><tr><td>Endocrine </td><td char=".">184 </td><td char=".">167 </td><td char=".">165 </td><td char=".">105 </td></tr><tr><td>Eye </td><td char=".">869 </td><td char=".">676 </td><td char=".">842 </td><td char=".">697 </td></tr><tr><td>Gastrointestinal tract </td><td char=".">287 </td><td char=".">291 </td><td char=".">750 </td><td char=".">770 </td></tr><tr><td>Genitourinary </td><td char=".">153 </td><td char=".">120 </td><td char=".">693 </td><td char=".">557 </td></tr><tr><td>Heart </td><td char=".">587 </td><td char=".">588 </td><td char=".">522 </td><td char=".">505 </td></tr><tr><td>Kidney </td><td char=".">931 </td><td char=".">1002 </td><td char=".">960 </td><td char=".">1197 </td></tr><tr><td>Limb </td><td char=".">0 </td><td char=".">0 </td><td char=".">202 </td><td char=".">152 </td></tr><tr><td>Liver </td><td char=".">624 </td><td char=".">821 </td><td char=".">812 </td><td char=".">1115 </td></tr><tr><td>Lung </td><td char=".">628 </td><td char=".">564 </td><td char=".">503 </td><td char=".">390 </td></tr><tr><td>Lymphoreticular </td><td char=".">340 </td><td char=".">425 </td><td char=".">154 </td><td char=".">119 </td></tr><tr><td>Lymph node </td><td char=".">1255 </td><td char=".">1100 </td><td char=".">418 </td><td char=".">240 </td></tr><tr><td>Mammary gland </td><td char=".">559 </td><td char=".">514 </td><td char=".">1112 </td><td char=".">972 </td></tr><tr><td>Muscle </td><td char=".">560 </td><td char=".">807 </td><td char=".">460 </td><td char=".">468 </td></tr><tr><td>Nervous </td><td char=".">1693 </td><td char=".">1731 </td><td char=".">1950 </td><td char=".">2551 </td></tr><tr><td>Ovary </td><td char=".">107 </td><td char=".">61 </td><td char=".">239 </td><td char=".">168 </td></tr><tr><td>Pancreas </td><td char=".">197 </td><td char=".">217 </td><td char=".">751 </td><td char=".">782 </td></tr><tr><td>Pancreatic islet </td><td char=".">809 </td><td char=".">907 </td><td char=".">435 </td><td char=".">457 </td></tr><tr><td>Peripheral nervous system </td><td char=".">95 </td><td char=".">73 </td><td char=".">372 </td><td char=".">314 </td></tr><tr><td>Pineal gland </td><td char=".">164 </td><td char=".">99 </td><td char=".">112 </td><td char=".">72 </td></tr><tr><td>Pituitary gland </td><td char=".">130 </td><td char=".">88 </td><td char=".">166 </td><td char=".">117 </td></tr><tr><td>Placenta </td><td char=".">1238 </td><td char=".">1601 </td><td char=".">527 </td><td char=".">628 </td></tr><tr><td>Prostate </td><td char=".">614 </td><td char=".">596 </td><td char=".">189 </td><td char=".">203 </td></tr><tr><td>Retina </td><td char=".">539 </td><td char=".">427 </td><td char=".">830 </td><td char=".">680 </td></tr><tr><td>Salivary gland </td><td char=".">37 </td><td char=".">35 </td><td char=".">156 </td><td char=".">208 </td></tr><tr><td>Skin </td><td char=".">661 </td><td char=".">530 </td><td char=".">848 </td><td char=".">684 </td></tr><tr><td>Soft tissue </td><td char=".">153 </td><td char=".">111 </td><td char=".">293 </td><td char=".">244 </td></tr><tr><td>Spleen </td><td char=".">502 </td><td char=".">548 </td><td char=".">1236 </td><td char=".">1301 </td></tr><tr><td>Stomach </td><td char=".">364 </td><td char=".">387 </td><td char=".">530 </td><td char=".">424 </td></tr><tr><td>Synovium </td><td char=".">4 </td><td char=".">2 </td><td char=".">431 </td><td char=".">490 </td></tr><tr><td>Testis </td><td char=".">1865 </td><td char=".">1748 </td><td char=".">2146 </td><td char=".">2201 </td></tr><tr><td>Thymus </td><td char=".">94 </td><td char=".">40 </td><td char=".">1 626 </td><td char=".">1375 </td></tr><tr><td>Thyroid </td><td char=".">207 </td><td char=".">152 </td><td char=".">176 </td><td char=".">168 </td></tr><tr><td>Uterus </td><td char=".">533 </td><td char=".">399 </td><td char=".">134 </td><td char=".">84 </td></tr><tr><td>Vascular </td><td char=".">423 </td><td char=".">461 </td><td char=".">577 </td><td char=".">419 </td></tr><tr><td>Total </td><td char=".">12 711 </td><td char=".">16 016 </td><td char=".">15 895 </td><td char=".">19 861 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194849" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl5" class="table-wrap-title" data-id="tbl5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>The number of tissue-specific genes and transcripts predicted in human and mouse</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissues </th><th colspan="2">Human </th><th colspan="2">Mouse </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>No. Genes </th><th>No. Transcripts </th><th>No. Genes </th><th>No. Transcripts </th></tr></thead><tbody><tr><td>Bone </td><td char=".">180 </td><td char=".">190 </td><td char=".">602 </td><td char=".">558 </td></tr><tr><td>Bone marrow </td><td char=".">207 </td><td char=".">244 </td><td char=".">1253 </td><td char=".">1563 </td></tr><tr><td>Brain </td><td char=".">1372 </td><td char=".">2066 </td><td char=".">1971 </td><td char=".">1866 </td></tr><tr><td>Cartilage </td><td char=".">206 </td><td char=".">250 </td><td char=".">6 </td><td char=".">2 </td></tr><tr><td>Cerebellum </td><td char=".">809 </td><td char=".">1092 </td><td char=".">1283 </td><td char=".">1137 </td></tr><tr><td>Cerebrum </td><td char=".">915 </td><td char=".">1195 </td><td char=".">1771 </td><td char=".">2426 </td></tr><tr><td>Cervix </td><td char=".">53 </td><td char=".">50 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>Colon </td><td char=".">320 </td><td char=".">319 </td><td char=".">693 </td><td char=".">775 </td></tr><tr><td>Ear </td><td char=".">298 </td><td char=".">316 </td><td char=".">745 </td><td char=".">739 </td></tr><tr><td>Endocrine </td><td char=".">184 </td><td char=".">167 </td><td char=".">165 </td><td char=".">105 </td></tr><tr><td>Eye </td><td char=".">869 </td><td char=".">676 </td><td char=".">842 </td><td char=".">697 </td></tr><tr><td>Gastrointestinal tract </td><td char=".">287 </td><td char=".">291 </td><td char=".">750 </td><td char=".">770 </td></tr><tr><td>Genitourinary </td><td char=".">153 </td><td char=".">120 </td><td char=".">693 </td><td char=".">557 </td></tr><tr><td>Heart </td><td char=".">587 </td><td char=".">588 </td><td char=".">522 </td><td char=".">505 </td></tr><tr><td>Kidney </td><td char=".">931 </td><td char=".">1002 </td><td char=".">960 </td><td char=".">1197 </td></tr><tr><td>Limb </td><td char=".">0 </td><td char=".">0 </td><td char=".">202 </td><td char=".">152 </td></tr><tr><td>Liver </td><td char=".">624 </td><td char=".">821 </td><td char=".">812 </td><td char=".">1115 </td></tr><tr><td>Lung </td><td char=".">628 </td><td char=".">564 </td><td char=".">503 </td><td char=".">390 </td></tr><tr><td>Lymphoreticular </td><td char=".">340 </td><td char=".">425 </td><td char=".">154 </td><td char=".">119 </td></tr><tr><td>Lymph node </td><td char=".">1255 </td><td char=".">1100 </td><td char=".">418 </td><td char=".">240 </td></tr><tr><td>Mammary gland </td><td char=".">559 </td><td char=".">514 </td><td char=".">1112 </td><td char=".">972 </td></tr><tr><td>Muscle </td><td char=".">560 </td><td char=".">807 </td><td char=".">460 </td><td char=".">468 </td></tr><tr><td>Nervous </td><td char=".">1693 </td><td char=".">1731 </td><td char=".">1950 </td><td char=".">2551 </td></tr><tr><td>Ovary </td><td char=".">107 </td><td char=".">61 </td><td char=".">239 </td><td char=".">168 </td></tr><tr><td>Pancreas </td><td char=".">197 </td><td char=".">217 </td><td char=".">751 </td><td char=".">782 </td></tr><tr><td>Pancreatic islet </td><td char=".">809 </td><td char=".">907 </td><td char=".">435 </td><td char=".">457 </td></tr><tr><td>Peripheral nervous system </td><td char=".">95 </td><td char=".">73 </td><td char=".">372 </td><td char=".">314 </td></tr><tr><td>Pineal gland </td><td char=".">164 </td><td char=".">99 </td><td char=".">112 </td><td char=".">72 </td></tr><tr><td>Pituitary gland </td><td char=".">130 </td><td char=".">88 </td><td char=".">166 </td><td char=".">117 </td></tr><tr><td>Placenta </td><td char=".">1238 </td><td char=".">1601 </td><td char=".">527 </td><td char=".">628 </td></tr><tr><td>Prostate </td><td char=".">614 </td><td char=".">596 </td><td char=".">189 </td><td char=".">203 </td></tr><tr><td>Retina </td><td char=".">539 </td><td char=".">427 </td><td char=".">830 </td><td char=".">680 </td></tr><tr><td>Salivary gland </td><td char=".">37 </td><td char=".">35 </td><td char=".">156 </td><td char=".">208 </td></tr><tr><td>Skin </td><td char=".">661 </td><td char=".">530 </td><td char=".">848 </td><td char=".">684 </td></tr><tr><td>Soft tissue </td><td char=".">153 </td><td char=".">111 </td><td char=".">293 </td><td char=".">244 </td></tr><tr><td>Spleen </td><td char=".">502 </td><td char=".">548 </td><td char=".">1236 </td><td char=".">1301 </td></tr><tr><td>Stomach </td><td char=".">364 </td><td char=".">387 </td><td char=".">530 </td><td char=".">424 </td></tr><tr><td>Synovium </td><td char=".">4 </td><td char=".">2 </td><td char=".">431 </td><td char=".">490 </td></tr><tr><td>Testis </td><td char=".">1865 </td><td char=".">1748 </td><td char=".">2146 </td><td char=".">2201 </td></tr><tr><td>Thymus </td><td char=".">94 </td><td char=".">40 </td><td char=".">1 626 </td><td char=".">1375 </td></tr><tr><td>Thyroid </td><td char=".">207 </td><td char=".">152 </td><td char=".">176 </td><td char=".">168 </td></tr><tr><td>Uterus </td><td char=".">533 </td><td char=".">399 </td><td char=".">134 </td><td char=".">84 </td></tr><tr><td>Vascular </td><td char=".">423 </td><td char=".">461 </td><td char=".">577 </td><td char=".">419 </td></tr><tr><td>Total </td><td char=".">12 711 </td><td char=".">16 016 </td><td char=".">15 895 </td><td char=".">19 861 </td></tr></tbody></table></div><div content-id="tbl5" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissues </th><th colspan="2">Human </th><th colspan="2">Mouse </th></tr><tr><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th>No. Genes </th><th>No. Transcripts </th><th>No. Genes </th><th>No. Transcripts </th></tr></thead><tbody><tr><td>Bone </td><td char=".">180 </td><td char=".">190 </td><td char=".">602 </td><td char=".">558 </td></tr><tr><td>Bone marrow </td><td char=".">207 </td><td char=".">244 </td><td char=".">1253 </td><td char=".">1563 </td></tr><tr><td>Brain </td><td char=".">1372 </td><td char=".">2066 </td><td char=".">1971 </td><td char=".">1866 </td></tr><tr><td>Cartilage </td><td char=".">206 </td><td char=".">250 </td><td char=".">6 </td><td char=".">2 </td></tr><tr><td>Cerebellum </td><td char=".">809 </td><td char=".">1092 </td><td char=".">1283 </td><td char=".">1137 </td></tr><tr><td>Cerebrum </td><td char=".">915 </td><td char=".">1195 </td><td char=".">1771 </td><td char=".">2426 </td></tr><tr><td>Cervix </td><td char=".">53 </td><td char=".">50 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>Colon </td><td char=".">320 </td><td char=".">319 </td><td char=".">693 </td><td char=".">775 </td></tr><tr><td>Ear </td><td char=".">298 </td><td char=".">316 </td><td char=".">745 </td><td char=".">739 </td></tr><tr><td>Endocrine </td><td char=".">184 </td><td char=".">167 </td><td char=".">165 </td><td char=".">105 </td></tr><tr><td>Eye </td><td char=".">869 </td><td char=".">676 </td><td char=".">842 </td><td char=".">697 </td></tr><tr><td>Gastrointestinal tract </td><td char=".">287 </td><td char=".">291 </td><td char=".">750 </td><td char=".">770 </td></tr><tr><td>Genitourinary </td><td char=".">153 </td><td char=".">120 </td><td char=".">693 </td><td char=".">557 </td></tr><tr><td>Heart </td><td char=".">587 </td><td char=".">588 </td><td char=".">522 </td><td char=".">505 </td></tr><tr><td>Kidney </td><td char=".">931 </td><td char=".">1002 </td><td char=".">960 </td><td char=".">1197 </td></tr><tr><td>Limb </td><td char=".">0 </td><td char=".">0 </td><td char=".">202 </td><td char=".">152 </td></tr><tr><td>Liver </td><td char=".">624 </td><td char=".">821 </td><td char=".">812 </td><td char=".">1115 </td></tr><tr><td>Lung </td><td char=".">628 </td><td char=".">564 </td><td char=".">503 </td><td char=".">390 </td></tr><tr><td>Lymphoreticular </td><td char=".">340 </td><td char=".">425 </td><td char=".">154 </td><td char=".">119 </td></tr><tr><td>Lymph node </td><td char=".">1255 </td><td char=".">1100 </td><td char=".">418 </td><td char=".">240 </td></tr><tr><td>Mammary gland </td><td char=".">559 </td><td char=".">514 </td><td char=".">1112 </td><td char=".">972 </td></tr><tr><td>Muscle </td><td char=".">560 </td><td char=".">807 </td><td char=".">460 </td><td char=".">468 </td></tr><tr><td>Nervous </td><td char=".">1693 </td><td char=".">1731 </td><td char=".">1950 </td><td char=".">2551 </td></tr><tr><td>Ovary </td><td char=".">107 </td><td char=".">61 </td><td char=".">239 </td><td char=".">168 </td></tr><tr><td>Pancreas </td><td char=".">197 </td><td char=".">217 </td><td char=".">751 </td><td char=".">782 </td></tr><tr><td>Pancreatic islet </td><td char=".">809 </td><td char=".">907 </td><td char=".">435 </td><td char=".">457 </td></tr><tr><td>Peripheral nervous system </td><td char=".">95 </td><td char=".">73 </td><td char=".">372 </td><td char=".">314 </td></tr><tr><td>Pineal gland </td><td char=".">164 </td><td char=".">99 </td><td char=".">112 </td><td char=".">72 </td></tr><tr><td>Pituitary gland </td><td char=".">130 </td><td char=".">88 </td><td char=".">166 </td><td char=".">117 </td></tr><tr><td>Placenta </td><td char=".">1238 </td><td char=".">1601 </td><td char=".">527 </td><td char=".">628 </td></tr><tr><td>Prostate </td><td char=".">614 </td><td char=".">596 </td><td char=".">189 </td><td char=".">203 </td></tr><tr><td>Retina </td><td char=".">539 </td><td char=".">427 </td><td char=".">830 </td><td char=".">680 </td></tr><tr><td>Salivary gland </td><td char=".">37 </td><td char=".">35 </td><td char=".">156 </td><td char=".">208 </td></tr><tr><td>Skin </td><td char=".">661 </td><td char=".">530 </td><td char=".">848 </td><td char=".">684 </td></tr><tr><td>Soft tissue </td><td char=".">153 </td><td char=".">111 </td><td char=".">293 </td><td char=".">244 </td></tr><tr><td>Spleen </td><td char=".">502 </td><td char=".">548 </td><td char=".">1236 </td><td char=".">1301 </td></tr><tr><td>Stomach </td><td char=".">364 </td><td char=".">387 </td><td char=".">530 </td><td char=".">424 </td></tr><tr><td>Synovium </td><td char=".">4 </td><td char=".">2 </td><td char=".">431 </td><td char=".">490 </td></tr><tr><td>Testis </td><td char=".">1865 </td><td char=".">1748 </td><td char=".">2146 </td><td char=".">2201 </td></tr><tr><td>Thymus </td><td char=".">94 </td><td char=".">40 </td><td char=".">1 626 </td><td char=".">1375 </td></tr><tr><td>Thyroid </td><td char=".">207 </td><td char=".">152 </td><td char=".">176 </td><td char=".">168 </td></tr><tr><td>Uterus </td><td char=".">533 </td><td char=".">399 </td><td char=".">134 </td><td char=".">84 </td></tr><tr><td>Vascular </td><td char=".">423 </td><td char=".">461 </td><td char=".">577 </td><td char=".">419 </td></tr><tr><td>Total </td><td char=".">12 711 </td><td char=".">16 016 </td><td char=".">15 895 </td><td char=".">19 861 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194849" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87194850 id="87194850" class="section-title" >3.5 GO analysis</h3>
<p>In human, 14 570 genes were assigned to 4666 distinct GO terms across the three functional categories: biological process, molecular function and cellular components. Among these, 9612 alternatively spliced genes were assigned to 4134 GO terms and 4958 non-alternatively spliced genes were assigned to 2615 GO terms. In mouse, 16 271 genes were assigned to 4613 GO terms and 8519 alternatively spliced genes to 3770 GO terms. GO analysis was carried out to determine whether there was any functional bias of the gene groups for a certain AS pattern. For example, we examined whether the genes with a cassette exon event showed significant enrichment in any of the GO terms. By applying Fisher's exact test, we discovered the differences between the groups of the genes in the different AS types. The selection criteria for the AS pattern and GO term were as follows: GO depth in the tree structure ≥4, number of genes in a given AS pattern for a given GO term ≥10 and <em>P</em>-value ≤0.01. The results are listed at <a reveal-id="tbl6" data-open="tbl6" class="link link-reveal link-table xref-fig">Table 6</a>. Interestingly, genes involved in protein modification including phosphorylation and dephosphorylation had relatively more cassette exon events than other AS events. Also, multiple cassette exon events were substantially enriched with genes involved in apoptosis and mitosis. Mutually exclusive exon events were substantially enriched with ion transport and actin cytoskeleton genes. The genes for chromosome structure formation and modification were coupled with 5′-splice site variation. Genes involved in nucleotide metabolism were the most common in the 3′-splice site variation. For the other AS patterns, there were no significant couplings with any gene group. A previous study also reported the statistical differences in the AS types commonly arising in different tissues.<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> It is interesting that there is an apparent bias in the distributions of genes between different AS patterns and tissues.</p>                        <a id="87194852" scrollto-destination="87194852"></a>
<div content-id="tbl6" class="table-modal"><div class="table-wrap"><div id="tbl6" class="table-wrap-title" data-id="tbl6"><div class="label -label"><strong>Table 6</strong></div><div class="caption"><p>Significant GO terms enriched in genes with each AS pattern in human</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GO term </th><th>GO Category<sup>a</sup> </th><th>GO depth </th><th>No. genes in a given AS type<sup>b</sup> </th><th>Total No. genes<sup>c</sup> </th><th>Relative ratio </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="7">Cassette exon </td></tr><tr><td>    Amino acid metabolism </td><td>BP </td><td char=".">5 </td><td char=".">113 </td><td char=".">243 </td><td char=".">1.21 </td><td char=".">8.69e-3 </td></tr><tr><td>    Protein modification </td><td>BP </td><td char=".">7 </td><td char=".">481 </td><td char=".">1168 </td><td char=".">1.07 </td><td char=".">3.38e-2 </td></tr><tr><td>    Phosphate metabolism </td><td>BP </td><td char=".">6 </td><td char=".">247 </td><td char=".">574 </td><td char=".">1.12 </td><td char=".">1.84e-2 </td></tr><tr><td>    Clathrin-coated vesicle </td><td>CC </td><td char=".">7 </td><td char=".">37 </td><td char=".">69 </td><td char=".">1.40 </td><td char=".">1.24e-2 </td></tr><tr><td>    Peroxisome </td><td>CC </td><td char=".">6 </td><td char=".">31 </td><td char=".">56 </td><td char=".">1.44 </td><td char=".">1.24e-2 </td></tr><tr><td>    Hydrolase activity, acting on glycosyl bonds </td><td>MF </td><td char=".">5 </td><td char=".">44 </td><td char=".">86 </td><td char=".">1.34 </td><td char=".">1.88e-2 </td></tr><tr><td>    Protein serine/threonine kinase activity </td><td>MF </td><td char=".">7 </td><td char=".">164 </td><td char=".">355 </td><td char=".">1.21 </td><td char=".">1.43e-3 </td></tr><tr><td>    Protein tyrosine kinase activity </td><td>MF </td><td char=".">7 </td><td char=".">145 </td><td char=".">324 </td><td char=".">1.17 </td><td char=".">1.63e-2 </td></tr><tr><td>    Transferase activity, transferring phosphorus-containing groups </td><td>MF </td><td char=".">4 </td><td char=".">294 </td><td char=".">671 </td><td char=".">1.14 </td><td char=".">2.24e-3 </td></tr><tr><td colspan="7">Multiple cassette exon </td></tr><tr><td>    Regulation of apoptosis </td><td>BP </td><td char=".">6 </td><td char=".">29 </td><td char=".">202 </td><td char=".">1.52 </td><td char=".">2.08e-2 </td></tr><tr><td>    Positive regulation of programmed cell death </td><td>BP </td><td char=".">6 </td><td char=".">18 </td><td char=".">112 </td><td char=".">1.70 </td><td char=".">2.26e-2 </td></tr><tr><td>    M phase </td><td>BP </td><td char=".">5 </td><td char=".">24 </td><td char=".">162 </td><td char=".">1.56 </td><td char=".">2.89e-2 </td></tr><tr><td>    Ubiquitin cycle </td><td>BP </td><td char=".">8 </td><td char=".">55 </td><td char=".">428 </td><td char=".">1.36 </td><td char=".">1.79e-2 </td></tr><tr><td>    Cysteine-type endopeptidase activity </td><td>MF </td><td char=".">6 </td><td char=".">13 </td><td char=".">79 </td><td char=".">1.74 </td><td char=".">4.99e-2 </td></tr><tr><td>    Protein tyrosine phosphatase activity </td><td>MF </td><td char=".">8 </td><td char=".">28 </td><td char=".">53 </td><td char=".">1.63 </td><td char=".">2.80e-3 </td></tr><tr><td>    Sugar binding </td><td>MF </td><td char=".">4 </td><td char=".">17 </td><td char=".">96 </td><td char=".">1.87 </td><td char=".">1.24e-2 </td></tr><tr><td colspan="7">Mutually exclusive exon </td></tr><tr><td>    Cell–cell signaling </td><td>BP </td><td char=".">4 </td><td char=".">20 </td><td char=".">286 </td><td char=".">1.91 </td><td char=".">5.64e-3 </td></tr><tr><td>    Defense response </td><td>BP </td><td char=".">5 </td><td char=".">30 </td><td char=".">531 </td><td char=".">1.54 </td><td char=".">1.69e-2 </td></tr><tr><td>    Immune response </td><td>BP </td><td char=".">4 </td><td char=".">29 </td><td char=".">489 </td><td char=".">1.62 </td><td char=".">9.18e-3 </td></tr><tr><td>    Ion transport </td><td>BP </td><td char=".">5 </td><td char=".">25 </td><td char=".">447 </td><td char=".">1.53 </td><td char=".">3.75e-2 </td></tr><tr><td>    Metal ion transport </td><td>BP </td><td char=".">7 </td><td char=".">18 </td><td char=".">219 </td><td char=".">2.24 </td><td char=".">1.36e-3 </td></tr><tr><td>    Muscle development </td><td>BP </td><td char=".">5 </td><td char=".">14 </td><td char=".">105 </td><td char=".">3.64 </td><td char=".">1.52e-4 </td></tr><tr><td>    Protein kinase cascade </td><td>BP </td><td char=".">6 </td><td char=".">18 </td><td char=".">197 </td><td char=".">2.50 </td><td char=".">2.99e-4 </td></tr><tr><td>    Secretion </td><td>BP </td><td char=".">5 </td><td char=".">12 </td><td char=".">169 </td><td char=".">1.94 </td><td char=".">3.43e-2 </td></tr><tr><td>    Actin cytoskeleton </td><td>CC </td><td char=".">5 </td><td char=".">18 </td><td char=".">146 </td><td char=".">3.37 </td><td char=".">0.00 </td></tr><tr><td>    Cytoskeleton </td><td>CC </td><td char=".">4 </td><td char=".">31 </td><td char=".">510 </td><td char=".">1.66 </td><td char=".">5.01e-3 </td></tr><tr><td>    Integral to plasma membrane </td><td>CC </td><td char=".">6 </td><td char=".">38 </td><td char=".">717 </td><td char=".">1.45 </td><td char=".">2.22e-2 </td></tr><tr><td>    Actin binding </td><td>MF </td><td char=".">5 </td><td char=".">17 </td><td char=".">158 </td><td char=".">2.94 </td><td char=".">4.02e-5 </td></tr><tr><td>    Cation channel activity </td><td>MF </td><td char=".">5 </td><td char=".">12 </td><td char=".">149 </td><td char=".">2.20 </td><td char=".">1.22e-2 </td></tr><tr><td>    Cytoskeletal protein binding </td><td>MF </td><td char=".">4 </td><td char=".">20 </td><td char=".">235 </td><td char=".">2.32 </td><td char=".">5.08e-4 </td></tr><tr><td>    Ion channel activity </td><td>MF </td><td char=".">5 </td><td char=".">12 </td><td char=".">172 </td><td char=".">2.34 </td><td char=".">5.20e-4 </td></tr><tr><td>    Kinase activity </td><td>MF </td><td char=".">5 </td><td char=".">35 </td><td char=".">585 </td><td char=".">1.63 </td><td char=".">4.15e-3 </td></tr><tr><td>    Phosphotransferase activity, alcohol group as acceptor </td><td>MF </td><td char=".">5 </td><td char=".">30 </td><td char=".">477 </td><td char=".">1.72 </td><td char=".">3.65e-3 </td></tr><tr><td>    Transferase activity, transferring phosphorus-containing groups </td><td>MF </td><td char=".">4 </td><td char=".">42 </td><td char=".">671 </td><td char=".">1.71 </td><td char=".">5.55e-4 </td></tr><tr><td colspan="7">5′-Splice site variation </td></tr><tr><td>    Chromatin remodelling </td><td>BP </td><td char=".">9 </td><td char=".">9 </td><td char=".">25 </td><td char=".">2.73 </td><td char=".">3.34e-3 </td></tr><tr><td>    Chromosome organization and biogenesis </td><td>BP </td><td char=".">6 </td><td char=".">23 </td><td char=".">124 </td><td char=".">1.75 </td><td char=".">7.28e-3 </td></tr><tr><td>    DNA packaging </td><td>BP </td><td char=".">6 </td><td char=".">20 </td><td char=".">103 </td><td char=".">1.83 </td><td char=".">9.57e-3 </td></tr><tr><td>    RNA processing </td><td>BP </td><td char=".">6 </td><td char=".">58 </td><td char=".">272 </td><td char=".">1.61 </td><td char=".">1.25e-4 </td></tr><tr><td>    rRNA metabolism </td><td>BP </td><td char=".">6 </td><td char=".">10 </td><td char=".">32 </td><td char=".">2.37 </td><td char=".">6.44e-3 </td></tr><tr><td>    Exonuclease activity </td><td>MF </td><td char=".">6 </td><td char=".">12 </td><td char=".">41 </td><td char=".">2.22 </td><td char=".">8.29e-3 </td></tr><tr><td>    Methyltransferase activity </td><td>MF </td><td char=".">5 </td><td char=".">23 </td><td char=".">99 </td><td char=".">1.76 </td><td char=".">6.45e-3 </td></tr><tr><td>    Nuclease activity </td><td>MF </td><td char=".">5 </td><td char=".">24 </td><td char=".">98 </td><td char=".">1.85 </td><td char=".">2.31e-3 </td></tr><tr><td>    RNA binding </td><td>MF </td><td char=".">4 </td><td char=".">75 </td><td char=".">407 </td><td char=".">1.40 </td><td char=".">2.63e-3 </td></tr><tr><td colspan="7">3′-Splice site variation </td></tr><tr><td>    Antigen presentation </td><td>BP </td><td char=".">5 </td><td char=".">13 </td><td char=".">36 </td><td char=".">2.09 </td><td char=".">6.34e-3 </td></tr><tr><td>    DNA metabolism </td><td>BP </td><td char=".">5 </td><td char=".">93 </td><td char=".">402 </td><td char=".">1.34 </td><td char=".">2.31e-3 </td></tr><tr><td>    Nucleotide metabolism </td><td>BP </td><td char=".">6 </td><td char=".">37 </td><td char=".">142 </td><td char=".">1.51 </td><td char=".">7.08e-3 </td></tr><tr><td>    Response to stress </td><td>BP </td><td char=".">4 </td><td char=".">140 </td><td char=".">673 </td><td char=".">1.20 </td><td char=".">1.42e-2 </td></tr><tr><td>    RNA metabolism </td><td>BP </td><td char=".">5 </td><td char=".">74 </td><td char=".">329 </td><td char=".">1.30 </td><td char=".">1.41e-2 </td></tr><tr><td>    ATP binding </td><td>MF </td><td char=".">6 </td><td char=".">172 </td><td char=".">851 </td><td char=".">1.17 </td><td char=".">1.93e-2 </td></tr><tr><td colspan="7">Intron retention </td></tr><tr><td>    Cholesterol metabolism </td><td>BP </td><td char=".">7 </td><td char=".">11 </td><td char=".">47 </td><td char=".">2.12 </td><td char=".">1.56e-2 </td></tr><tr><td>    Steroid biosynthesis </td><td>BP </td><td char=".">6 </td><td char=".">14 </td><td char=".">46 </td><td char=".">2.76 </td><td char=".">2.90e-4 </td></tr><tr><td>    Steroid metabolism </td><td>BP </td><td char=".">6 </td><td char=".">21 </td><td char=".">101 </td><td char=".">1.89 </td><td char=".">4.42e-3 </td></tr></tbody></table></div><div content-id="tbl6" class="table-modal"><table><thead name="thead" align=""><tr><th>GO term </th><th>GO Category<sup>a</sup> </th><th>GO depth </th><th>No. genes in a given AS type<sup>b</sup> </th><th>Total No. genes<sup>c</sup> </th><th>Relative ratio </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="7">Cassette exon </td></tr><tr><td>    Amino acid metabolism </td><td>BP </td><td char=".">5 </td><td char=".">113 </td><td char=".">243 </td><td char=".">1.21 </td><td char=".">8.69e-3 </td></tr><tr><td>    Protein modification </td><td>BP </td><td char=".">7 </td><td char=".">481 </td><td char=".">1168 </td><td char=".">1.07 </td><td char=".">3.38e-2 </td></tr><tr><td>    Phosphate metabolism </td><td>BP </td><td char=".">6 </td><td char=".">247 </td><td char=".">574 </td><td char=".">1.12 </td><td char=".">1.84e-2 </td></tr><tr><td>    Clathrin-coated vesicle </td><td>CC </td><td char=".">7 </td><td char=".">37 </td><td char=".">69 </td><td char=".">1.40 </td><td char=".">1.24e-2 </td></tr><tr><td>    Peroxisome </td><td>CC </td><td char=".">6 </td><td char=".">31 </td><td char=".">56 </td><td char=".">1.44 </td><td char=".">1.24e-2 </td></tr><tr><td>    Hydrolase activity, acting on glycosyl bonds </td><td>MF </td><td char=".">5 </td><td char=".">44 </td><td char=".">86 </td><td char=".">1.34 </td><td char=".">1.88e-2 </td></tr><tr><td>    Protein serine/threonine kinase activity </td><td>MF </td><td char=".">7 </td><td char=".">164 </td><td char=".">355 </td><td char=".">1.21 </td><td char=".">1.43e-3 </td></tr><tr><td>    Protein tyrosine kinase activity </td><td>MF </td><td char=".">7 </td><td char=".">145 </td><td char=".">324 </td><td char=".">1.17 </td><td char=".">1.63e-2 </td></tr><tr><td>    Transferase activity, transferring phosphorus-containing groups </td><td>MF </td><td char=".">4 </td><td char=".">294 </td><td char=".">671 </td><td char=".">1.14 </td><td char=".">2.24e-3 </td></tr><tr><td colspan="7">Multiple cassette exon </td></tr><tr><td>    Regulation of apoptosis </td><td>BP </td><td char=".">6 </td><td char=".">29 </td><td char=".">202 </td><td char=".">1.52 </td><td char=".">2.08e-2 </td></tr><tr><td>    Positive regulation of programmed cell death </td><td>BP </td><td char=".">6 </td><td char=".">18 </td><td char=".">112 </td><td char=".">1.70 </td><td char=".">2.26e-2 </td></tr><tr><td>    M phase </td><td>BP </td><td char=".">5 </td><td char=".">24 </td><td char=".">162 </td><td char=".">1.56 </td><td char=".">2.89e-2 </td></tr><tr><td>    Ubiquitin cycle </td><td>BP </td><td char=".">8 </td><td char=".">55 </td><td char=".">428 </td><td char=".">1.36 </td><td char=".">1.79e-2 </td></tr><tr><td>    Cysteine-type endopeptidase activity </td><td>MF </td><td char=".">6 </td><td char=".">13 </td><td char=".">79 </td><td char=".">1.74 </td><td char=".">4.99e-2 </td></tr><tr><td>    Protein tyrosine phosphatase activity </td><td>MF </td><td char=".">8 </td><td char=".">28 </td><td char=".">53 </td><td char=".">1.63 </td><td char=".">2.80e-3 </td></tr><tr><td>    Sugar binding </td><td>MF </td><td char=".">4 </td><td char=".">17 </td><td char=".">96 </td><td char=".">1.87 </td><td char=".">1.24e-2 </td></tr><tr><td colspan="7">Mutually exclusive exon </td></tr><tr><td>    Cell–cell signaling </td><td>BP </td><td char=".">4 </td><td char=".">20 </td><td char=".">286 </td><td char=".">1.91 </td><td char=".">5.64e-3 </td></tr><tr><td>    Defense response </td><td>BP </td><td char=".">5 </td><td char=".">30 </td><td char=".">531 </td><td char=".">1.54 </td><td char=".">1.69e-2 </td></tr><tr><td>    Immune response </td><td>BP </td><td char=".">4 </td><td char=".">29 </td><td char=".">489 </td><td char=".">1.62 </td><td char=".">9.18e-3 </td></tr><tr><td>    Ion transport </td><td>BP </td><td char=".">5 </td><td char=".">25 </td><td char=".">447 </td><td char=".">1.53 </td><td char=".">3.75e-2 </td></tr><tr><td>    Metal ion transport </td><td>BP </td><td char=".">7 </td><td char=".">18 </td><td char=".">219 </td><td char=".">2.24 </td><td char=".">1.36e-3 </td></tr><tr><td>    Muscle development </td><td>BP </td><td char=".">5 </td><td char=".">14 </td><td char=".">105 </td><td char=".">3.64 </td><td char=".">1.52e-4 </td></tr><tr><td>    Protein kinase cascade </td><td>BP </td><td char=".">6 </td><td char=".">18 </td><td char=".">197 </td><td char=".">2.50 </td><td char=".">2.99e-4 </td></tr><tr><td>    Secretion </td><td>BP </td><td char=".">5 </td><td char=".">12 </td><td char=".">169 </td><td char=".">1.94 </td><td char=".">3.43e-2 </td></tr><tr><td>    Actin cytoskeleton </td><td>CC </td><td char=".">5 </td><td char=".">18 </td><td char=".">146 </td><td char=".">3.37 </td><td char=".">0.00 </td></tr><tr><td>    Cytoskeleton </td><td>CC </td><td char=".">4 </td><td char=".">31 </td><td char=".">510 </td><td char=".">1.66 </td><td char=".">5.01e-3 </td></tr><tr><td>    Integral to plasma membrane </td><td>CC </td><td char=".">6 </td><td char=".">38 </td><td char=".">717 </td><td char=".">1.45 </td><td char=".">2.22e-2 </td></tr><tr><td>    Actin binding </td><td>MF </td><td char=".">5 </td><td char=".">17 </td><td char=".">158 </td><td char=".">2.94 </td><td char=".">4.02e-5 </td></tr><tr><td>    Cation channel activity </td><td>MF </td><td char=".">5 </td><td char=".">12 </td><td char=".">149 </td><td char=".">2.20 </td><td char=".">1.22e-2 </td></tr><tr><td>    Cytoskeletal protein binding </td><td>MF </td><td char=".">4 </td><td char=".">20 </td><td char=".">235 </td><td char=".">2.32 </td><td char=".">5.08e-4 </td></tr><tr><td>    Ion channel activity </td><td>MF </td><td char=".">5 </td><td char=".">12 </td><td char=".">172 </td><td char=".">2.34 </td><td char=".">5.20e-4 </td></tr><tr><td>    Kinase activity </td><td>MF </td><td char=".">5 </td><td char=".">35 </td><td char=".">585 </td><td char=".">1.63 </td><td char=".">4.15e-3 </td></tr><tr><td>    Phosphotransferase activity, alcohol group as acceptor </td><td>MF </td><td char=".">5 </td><td char=".">30 </td><td char=".">477 </td><td char=".">1.72 </td><td char=".">3.65e-3 </td></tr><tr><td>    Transferase activity, transferring phosphorus-containing groups </td><td>MF </td><td char=".">4 </td><td char=".">42 </td><td char=".">671 </td><td char=".">1.71 </td><td char=".">5.55e-4 </td></tr><tr><td colspan="7">5′-Splice site variation </td></tr><tr><td>    Chromatin remodelling </td><td>BP </td><td char=".">9 </td><td char=".">9 </td><td char=".">25 </td><td char=".">2.73 </td><td char=".">3.34e-3 </td></tr><tr><td>    Chromosome organization and biogenesis </td><td>BP </td><td char=".">6 </td><td char=".">23 </td><td char=".">124 </td><td char=".">1.75 </td><td char=".">7.28e-3 </td></tr><tr><td>    DNA packaging </td><td>BP </td><td char=".">6 </td><td char=".">20 </td><td char=".">103 </td><td char=".">1.83 </td><td char=".">9.57e-3 </td></tr><tr><td>    RNA processing </td><td>BP </td><td char=".">6 </td><td char=".">58 </td><td char=".">272 </td><td char=".">1.61 </td><td char=".">1.25e-4 </td></tr><tr><td>    rRNA metabolism </td><td>BP </td><td char=".">6 </td><td char=".">10 </td><td char=".">32 </td><td char=".">2.37 </td><td char=".">6.44e-3 </td></tr><tr><td>    Exonuclease activity </td><td>MF </td><td char=".">6 </td><td char=".">12 </td><td char=".">41 </td><td char=".">2.22 </td><td char=".">8.29e-3 </td></tr><tr><td>    Methyltransferase activity </td><td>MF </td><td char=".">5 </td><td char=".">23 </td><td char=".">99 </td><td char=".">1.76 </td><td char=".">6.45e-3 </td></tr><tr><td>    Nuclease activity </td><td>MF </td><td char=".">5 </td><td char=".">24 </td><td char=".">98 </td><td char=".">1.85 </td><td char=".">2.31e-3 </td></tr><tr><td>    RNA binding </td><td>MF </td><td char=".">4 </td><td char=".">75 </td><td char=".">407 </td><td char=".">1.40 </td><td char=".">2.63e-3 </td></tr><tr><td colspan="7">3′-Splice site variation </td></tr><tr><td>    Antigen presentation </td><td>BP </td><td char=".">5 </td><td char=".">13 </td><td char=".">36 </td><td char=".">2.09 </td><td char=".">6.34e-3 </td></tr><tr><td>    DNA metabolism </td><td>BP </td><td char=".">5 </td><td char=".">93 </td><td char=".">402 </td><td char=".">1.34 </td><td char=".">2.31e-3 </td></tr><tr><td>    Nucleotide metabolism </td><td>BP </td><td char=".">6 </td><td char=".">37 </td><td char=".">142 </td><td char=".">1.51 </td><td char=".">7.08e-3 </td></tr><tr><td>    Response to stress </td><td>BP </td><td char=".">4 </td><td char=".">140 </td><td char=".">673 </td><td char=".">1.20 </td><td char=".">1.42e-2 </td></tr><tr><td>    RNA metabolism </td><td>BP </td><td char=".">5 </td><td char=".">74 </td><td char=".">329 </td><td char=".">1.30 </td><td char=".">1.41e-2 </td></tr><tr><td>    ATP binding </td><td>MF </td><td char=".">6 </td><td char=".">172 </td><td char=".">851 </td><td char=".">1.17 </td><td char=".">1.93e-2 </td></tr><tr><td colspan="7">Intron retention </td></tr><tr><td>    Cholesterol metabolism </td><td>BP </td><td char=".">7 </td><td char=".">11 </td><td char=".">47 </td><td char=".">2.12 </td><td char=".">1.56e-2 </td></tr><tr><td>    Steroid biosynthesis </td><td>BP </td><td char=".">6 </td><td char=".">14 </td><td char=".">46 </td><td char=".">2.76 </td><td char=".">2.90e-4 </td></tr><tr><td>    Steroid metabolism </td><td>BP </td><td char=".">6 </td><td char=".">21 </td><td char=".">101 </td><td char=".">1.89 </td><td char=".">4.42e-3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>BP, biological process; CC, cellular component; MF, molecular function.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>‘No. genes in a given AS type’ indicates the number of genes with the specific AS pattern and assigned to a given GO term.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>‘Total No. genes’ indicates the number of genes with any of the AS patterns assigned to a given GO term.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194852" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl6" class="table-wrap-title" data-id="tbl6"><div class="label -label"><strong>Table 6</strong></div><div class="caption"><p>Significant GO terms enriched in genes with each AS pattern in human</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GO term </th><th>GO Category<sup>a</sup> </th><th>GO depth </th><th>No. genes in a given AS type<sup>b</sup> </th><th>Total No. genes<sup>c</sup> </th><th>Relative ratio </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="7">Cassette exon </td></tr><tr><td>    Amino acid metabolism </td><td>BP </td><td char=".">5 </td><td char=".">113 </td><td char=".">243 </td><td char=".">1.21 </td><td char=".">8.69e-3 </td></tr><tr><td>    Protein modification </td><td>BP </td><td char=".">7 </td><td char=".">481 </td><td char=".">1168 </td><td char=".">1.07 </td><td char=".">3.38e-2 </td></tr><tr><td>    Phosphate metabolism </td><td>BP </td><td char=".">6 </td><td char=".">247 </td><td char=".">574 </td><td char=".">1.12 </td><td char=".">1.84e-2 </td></tr><tr><td>    Clathrin-coated vesicle </td><td>CC </td><td char=".">7 </td><td char=".">37 </td><td char=".">69 </td><td char=".">1.40 </td><td char=".">1.24e-2 </td></tr><tr><td>    Peroxisome </td><td>CC </td><td char=".">6 </td><td char=".">31 </td><td char=".">56 </td><td char=".">1.44 </td><td char=".">1.24e-2 </td></tr><tr><td>    Hydrolase activity, acting on glycosyl bonds </td><td>MF </td><td char=".">5 </td><td char=".">44 </td><td char=".">86 </td><td char=".">1.34 </td><td char=".">1.88e-2 </td></tr><tr><td>    Protein serine/threonine kinase activity </td><td>MF </td><td char=".">7 </td><td char=".">164 </td><td char=".">355 </td><td char=".">1.21 </td><td char=".">1.43e-3 </td></tr><tr><td>    Protein tyrosine kinase activity </td><td>MF </td><td char=".">7 </td><td char=".">145 </td><td char=".">324 </td><td char=".">1.17 </td><td char=".">1.63e-2 </td></tr><tr><td>    Transferase activity, transferring phosphorus-containing groups </td><td>MF </td><td char=".">4 </td><td char=".">294 </td><td char=".">671 </td><td char=".">1.14 </td><td char=".">2.24e-3 </td></tr><tr><td colspan="7">Multiple cassette exon </td></tr><tr><td>    Regulation of apoptosis </td><td>BP </td><td char=".">6 </td><td char=".">29 </td><td char=".">202 </td><td char=".">1.52 </td><td char=".">2.08e-2 </td></tr><tr><td>    Positive regulation of programmed cell death </td><td>BP </td><td char=".">6 </td><td char=".">18 </td><td char=".">112 </td><td char=".">1.70 </td><td char=".">2.26e-2 </td></tr><tr><td>    M phase </td><td>BP </td><td char=".">5 </td><td char=".">24 </td><td char=".">162 </td><td char=".">1.56 </td><td char=".">2.89e-2 </td></tr><tr><td>    Ubiquitin cycle </td><td>BP </td><td char=".">8 </td><td char=".">55 </td><td char=".">428 </td><td char=".">1.36 </td><td char=".">1.79e-2 </td></tr><tr><td>    Cysteine-type endopeptidase activity </td><td>MF </td><td char=".">6 </td><td char=".">13 </td><td char=".">79 </td><td char=".">1.74 </td><td char=".">4.99e-2 </td></tr><tr><td>    Protein tyrosine phosphatase activity </td><td>MF </td><td char=".">8 </td><td char=".">28 </td><td char=".">53 </td><td char=".">1.63 </td><td char=".">2.80e-3 </td></tr><tr><td>    Sugar binding </td><td>MF </td><td char=".">4 </td><td char=".">17 </td><td char=".">96 </td><td char=".">1.87 </td><td char=".">1.24e-2 </td></tr><tr><td colspan="7">Mutually exclusive exon </td></tr><tr><td>    Cell–cell signaling </td><td>BP </td><td char=".">4 </td><td char=".">20 </td><td char=".">286 </td><td char=".">1.91 </td><td char=".">5.64e-3 </td></tr><tr><td>    Defense response </td><td>BP </td><td char=".">5 </td><td char=".">30 </td><td char=".">531 </td><td char=".">1.54 </td><td char=".">1.69e-2 </td></tr><tr><td>    Immune response </td><td>BP </td><td char=".">4 </td><td char=".">29 </td><td char=".">489 </td><td char=".">1.62 </td><td char=".">9.18e-3 </td></tr><tr><td>    Ion transport </td><td>BP </td><td char=".">5 </td><td char=".">25 </td><td char=".">447 </td><td char=".">1.53 </td><td char=".">3.75e-2 </td></tr><tr><td>    Metal ion transport </td><td>BP </td><td char=".">7 </td><td char=".">18 </td><td char=".">219 </td><td char=".">2.24 </td><td char=".">1.36e-3 </td></tr><tr><td>    Muscle development </td><td>BP </td><td char=".">5 </td><td char=".">14 </td><td char=".">105 </td><td char=".">3.64 </td><td char=".">1.52e-4 </td></tr><tr><td>    Protein kinase cascade </td><td>BP </td><td char=".">6 </td><td char=".">18 </td><td char=".">197 </td><td char=".">2.50 </td><td char=".">2.99e-4 </td></tr><tr><td>    Secretion </td><td>BP </td><td char=".">5 </td><td char=".">12 </td><td char=".">169 </td><td char=".">1.94 </td><td char=".">3.43e-2 </td></tr><tr><td>    Actin cytoskeleton </td><td>CC </td><td char=".">5 </td><td char=".">18 </td><td char=".">146 </td><td char=".">3.37 </td><td char=".">0.00 </td></tr><tr><td>    Cytoskeleton </td><td>CC </td><td char=".">4 </td><td char=".">31 </td><td char=".">510 </td><td char=".">1.66 </td><td char=".">5.01e-3 </td></tr><tr><td>    Integral to plasma membrane </td><td>CC </td><td char=".">6 </td><td char=".">38 </td><td char=".">717 </td><td char=".">1.45 </td><td char=".">2.22e-2 </td></tr><tr><td>    Actin binding </td><td>MF </td><td char=".">5 </td><td char=".">17 </td><td char=".">158 </td><td char=".">2.94 </td><td char=".">4.02e-5 </td></tr><tr><td>    Cation channel activity </td><td>MF </td><td char=".">5 </td><td char=".">12 </td><td char=".">149 </td><td char=".">2.20 </td><td char=".">1.22e-2 </td></tr><tr><td>    Cytoskeletal protein binding </td><td>MF </td><td char=".">4 </td><td char=".">20 </td><td char=".">235 </td><td char=".">2.32 </td><td char=".">5.08e-4 </td></tr><tr><td>    Ion channel activity </td><td>MF </td><td char=".">5 </td><td char=".">12 </td><td char=".">172 </td><td char=".">2.34 </td><td char=".">5.20e-4 </td></tr><tr><td>    Kinase activity </td><td>MF </td><td char=".">5 </td><td char=".">35 </td><td char=".">585 </td><td char=".">1.63 </td><td char=".">4.15e-3 </td></tr><tr><td>    Phosphotransferase activity, alcohol group as acceptor </td><td>MF </td><td char=".">5 </td><td char=".">30 </td><td char=".">477 </td><td char=".">1.72 </td><td char=".">3.65e-3 </td></tr><tr><td>    Transferase activity, transferring phosphorus-containing groups </td><td>MF </td><td char=".">4 </td><td char=".">42 </td><td char=".">671 </td><td char=".">1.71 </td><td char=".">5.55e-4 </td></tr><tr><td colspan="7">5′-Splice site variation </td></tr><tr><td>    Chromatin remodelling </td><td>BP </td><td char=".">9 </td><td char=".">9 </td><td char=".">25 </td><td char=".">2.73 </td><td char=".">3.34e-3 </td></tr><tr><td>    Chromosome organization and biogenesis </td><td>BP </td><td char=".">6 </td><td char=".">23 </td><td char=".">124 </td><td char=".">1.75 </td><td char=".">7.28e-3 </td></tr><tr><td>    DNA packaging </td><td>BP </td><td char=".">6 </td><td char=".">20 </td><td char=".">103 </td><td char=".">1.83 </td><td char=".">9.57e-3 </td></tr><tr><td>    RNA processing </td><td>BP </td><td char=".">6 </td><td char=".">58 </td><td char=".">272 </td><td char=".">1.61 </td><td char=".">1.25e-4 </td></tr><tr><td>    rRNA metabolism </td><td>BP </td><td char=".">6 </td><td char=".">10 </td><td char=".">32 </td><td char=".">2.37 </td><td char=".">6.44e-3 </td></tr><tr><td>    Exonuclease activity </td><td>MF </td><td char=".">6 </td><td char=".">12 </td><td char=".">41 </td><td char=".">2.22 </td><td char=".">8.29e-3 </td></tr><tr><td>    Methyltransferase activity </td><td>MF </td><td char=".">5 </td><td char=".">23 </td><td char=".">99 </td><td char=".">1.76 </td><td char=".">6.45e-3 </td></tr><tr><td>    Nuclease activity </td><td>MF </td><td char=".">5 </td><td char=".">24 </td><td char=".">98 </td><td char=".">1.85 </td><td char=".">2.31e-3 </td></tr><tr><td>    RNA binding </td><td>MF </td><td char=".">4 </td><td char=".">75 </td><td char=".">407 </td><td char=".">1.40 </td><td char=".">2.63e-3 </td></tr><tr><td colspan="7">3′-Splice site variation </td></tr><tr><td>    Antigen presentation </td><td>BP </td><td char=".">5 </td><td char=".">13 </td><td char=".">36 </td><td char=".">2.09 </td><td char=".">6.34e-3 </td></tr><tr><td>    DNA metabolism </td><td>BP </td><td char=".">5 </td><td char=".">93 </td><td char=".">402 </td><td char=".">1.34 </td><td char=".">2.31e-3 </td></tr><tr><td>    Nucleotide metabolism </td><td>BP </td><td char=".">6 </td><td char=".">37 </td><td char=".">142 </td><td char=".">1.51 </td><td char=".">7.08e-3 </td></tr><tr><td>    Response to stress </td><td>BP </td><td char=".">4 </td><td char=".">140 </td><td char=".">673 </td><td char=".">1.20 </td><td char=".">1.42e-2 </td></tr><tr><td>    RNA metabolism </td><td>BP </td><td char=".">5 </td><td char=".">74 </td><td char=".">329 </td><td char=".">1.30 </td><td char=".">1.41e-2 </td></tr><tr><td>    ATP binding </td><td>MF </td><td char=".">6 </td><td char=".">172 </td><td char=".">851 </td><td char=".">1.17 </td><td char=".">1.93e-2 </td></tr><tr><td colspan="7">Intron retention </td></tr><tr><td>    Cholesterol metabolism </td><td>BP </td><td char=".">7 </td><td char=".">11 </td><td char=".">47 </td><td char=".">2.12 </td><td char=".">1.56e-2 </td></tr><tr><td>    Steroid biosynthesis </td><td>BP </td><td char=".">6 </td><td char=".">14 </td><td char=".">46 </td><td char=".">2.76 </td><td char=".">2.90e-4 </td></tr><tr><td>    Steroid metabolism </td><td>BP </td><td char=".">6 </td><td char=".">21 </td><td char=".">101 </td><td char=".">1.89 </td><td char=".">4.42e-3 </td></tr></tbody></table></div><div content-id="tbl6" class="table-modal"><table><thead name="thead" align=""><tr><th>GO term </th><th>GO Category<sup>a</sup> </th><th>GO depth </th><th>No. genes in a given AS type<sup>b</sup> </th><th>Total No. genes<sup>c</sup> </th><th>Relative ratio </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="7">Cassette exon </td></tr><tr><td>    Amino acid metabolism </td><td>BP </td><td char=".">5 </td><td char=".">113 </td><td char=".">243 </td><td char=".">1.21 </td><td char=".">8.69e-3 </td></tr><tr><td>    Protein modification </td><td>BP </td><td char=".">7 </td><td char=".">481 </td><td char=".">1168 </td><td char=".">1.07 </td><td char=".">3.38e-2 </td></tr><tr><td>    Phosphate metabolism </td><td>BP </td><td char=".">6 </td><td char=".">247 </td><td char=".">574 </td><td char=".">1.12 </td><td char=".">1.84e-2 </td></tr><tr><td>    Clathrin-coated vesicle </td><td>CC </td><td char=".">7 </td><td char=".">37 </td><td char=".">69 </td><td char=".">1.40 </td><td char=".">1.24e-2 </td></tr><tr><td>    Peroxisome </td><td>CC </td><td char=".">6 </td><td char=".">31 </td><td char=".">56 </td><td char=".">1.44 </td><td char=".">1.24e-2 </td></tr><tr><td>    Hydrolase activity, acting on glycosyl bonds </td><td>MF </td><td char=".">5 </td><td char=".">44 </td><td char=".">86 </td><td char=".">1.34 </td><td char=".">1.88e-2 </td></tr><tr><td>    Protein serine/threonine kinase activity </td><td>MF </td><td char=".">7 </td><td char=".">164 </td><td char=".">355 </td><td char=".">1.21 </td><td char=".">1.43e-3 </td></tr><tr><td>    Protein tyrosine kinase activity </td><td>MF </td><td char=".">7 </td><td char=".">145 </td><td char=".">324 </td><td char=".">1.17 </td><td char=".">1.63e-2 </td></tr><tr><td>    Transferase activity, transferring phosphorus-containing groups </td><td>MF </td><td char=".">4 </td><td char=".">294 </td><td char=".">671 </td><td char=".">1.14 </td><td char=".">2.24e-3 </td></tr><tr><td colspan="7">Multiple cassette exon </td></tr><tr><td>    Regulation of apoptosis </td><td>BP </td><td char=".">6 </td><td char=".">29 </td><td char=".">202 </td><td char=".">1.52 </td><td char=".">2.08e-2 </td></tr><tr><td>    Positive regulation of programmed cell death </td><td>BP </td><td char=".">6 </td><td char=".">18 </td><td char=".">112 </td><td char=".">1.70 </td><td char=".">2.26e-2 </td></tr><tr><td>    M phase </td><td>BP </td><td char=".">5 </td><td char=".">24 </td><td char=".">162 </td><td char=".">1.56 </td><td char=".">2.89e-2 </td></tr><tr><td>    Ubiquitin cycle </td><td>BP </td><td char=".">8 </td><td char=".">55 </td><td char=".">428 </td><td char=".">1.36 </td><td char=".">1.79e-2 </td></tr><tr><td>    Cysteine-type endopeptidase activity </td><td>MF </td><td char=".">6 </td><td char=".">13 </td><td char=".">79 </td><td char=".">1.74 </td><td char=".">4.99e-2 </td></tr><tr><td>    Protein tyrosine phosphatase activity </td><td>MF </td><td char=".">8 </td><td char=".">28 </td><td char=".">53 </td><td char=".">1.63 </td><td char=".">2.80e-3 </td></tr><tr><td>    Sugar binding </td><td>MF </td><td char=".">4 </td><td char=".">17 </td><td char=".">96 </td><td char=".">1.87 </td><td char=".">1.24e-2 </td></tr><tr><td colspan="7">Mutually exclusive exon </td></tr><tr><td>    Cell–cell signaling </td><td>BP </td><td char=".">4 </td><td char=".">20 </td><td char=".">286 </td><td char=".">1.91 </td><td char=".">5.64e-3 </td></tr><tr><td>    Defense response </td><td>BP </td><td char=".">5 </td><td char=".">30 </td><td char=".">531 </td><td char=".">1.54 </td><td char=".">1.69e-2 </td></tr><tr><td>    Immune response </td><td>BP </td><td char=".">4 </td><td char=".">29 </td><td char=".">489 </td><td char=".">1.62 </td><td char=".">9.18e-3 </td></tr><tr><td>    Ion transport </td><td>BP </td><td char=".">5 </td><td char=".">25 </td><td char=".">447 </td><td char=".">1.53 </td><td char=".">3.75e-2 </td></tr><tr><td>    Metal ion transport </td><td>BP </td><td char=".">7 </td><td char=".">18 </td><td char=".">219 </td><td char=".">2.24 </td><td char=".">1.36e-3 </td></tr><tr><td>    Muscle development </td><td>BP </td><td char=".">5 </td><td char=".">14 </td><td char=".">105 </td><td char=".">3.64 </td><td char=".">1.52e-4 </td></tr><tr><td>    Protein kinase cascade </td><td>BP </td><td char=".">6 </td><td char=".">18 </td><td char=".">197 </td><td char=".">2.50 </td><td char=".">2.99e-4 </td></tr><tr><td>    Secretion </td><td>BP </td><td char=".">5 </td><td char=".">12 </td><td char=".">169 </td><td char=".">1.94 </td><td char=".">3.43e-2 </td></tr><tr><td>    Actin cytoskeleton </td><td>CC </td><td char=".">5 </td><td char=".">18 </td><td char=".">146 </td><td char=".">3.37 </td><td char=".">0.00 </td></tr><tr><td>    Cytoskeleton </td><td>CC </td><td char=".">4 </td><td char=".">31 </td><td char=".">510 </td><td char=".">1.66 </td><td char=".">5.01e-3 </td></tr><tr><td>    Integral to plasma membrane </td><td>CC </td><td char=".">6 </td><td char=".">38 </td><td char=".">717 </td><td char=".">1.45 </td><td char=".">2.22e-2 </td></tr><tr><td>    Actin binding </td><td>MF </td><td char=".">5 </td><td char=".">17 </td><td char=".">158 </td><td char=".">2.94 </td><td char=".">4.02e-5 </td></tr><tr><td>    Cation channel activity </td><td>MF </td><td char=".">5 </td><td char=".">12 </td><td char=".">149 </td><td char=".">2.20 </td><td char=".">1.22e-2 </td></tr><tr><td>    Cytoskeletal protein binding </td><td>MF </td><td char=".">4 </td><td char=".">20 </td><td char=".">235 </td><td char=".">2.32 </td><td char=".">5.08e-4 </td></tr><tr><td>    Ion channel activity </td><td>MF </td><td char=".">5 </td><td char=".">12 </td><td char=".">172 </td><td char=".">2.34 </td><td char=".">5.20e-4 </td></tr><tr><td>    Kinase activity </td><td>MF </td><td char=".">5 </td><td char=".">35 </td><td char=".">585 </td><td char=".">1.63 </td><td char=".">4.15e-3 </td></tr><tr><td>    Phosphotransferase activity, alcohol group as acceptor </td><td>MF </td><td char=".">5 </td><td char=".">30 </td><td char=".">477 </td><td char=".">1.72 </td><td char=".">3.65e-3 </td></tr><tr><td>    Transferase activity, transferring phosphorus-containing groups </td><td>MF </td><td char=".">4 </td><td char=".">42 </td><td char=".">671 </td><td char=".">1.71 </td><td char=".">5.55e-4 </td></tr><tr><td colspan="7">5′-Splice site variation </td></tr><tr><td>    Chromatin remodelling </td><td>BP </td><td char=".">9 </td><td char=".">9 </td><td char=".">25 </td><td char=".">2.73 </td><td char=".">3.34e-3 </td></tr><tr><td>    Chromosome organization and biogenesis </td><td>BP </td><td char=".">6 </td><td char=".">23 </td><td char=".">124 </td><td char=".">1.75 </td><td char=".">7.28e-3 </td></tr><tr><td>    DNA packaging </td><td>BP </td><td char=".">6 </td><td char=".">20 </td><td char=".">103 </td><td char=".">1.83 </td><td char=".">9.57e-3 </td></tr><tr><td>    RNA processing </td><td>BP </td><td char=".">6 </td><td char=".">58 </td><td char=".">272 </td><td char=".">1.61 </td><td char=".">1.25e-4 </td></tr><tr><td>    rRNA metabolism </td><td>BP </td><td char=".">6 </td><td char=".">10 </td><td char=".">32 </td><td char=".">2.37 </td><td char=".">6.44e-3 </td></tr><tr><td>    Exonuclease activity </td><td>MF </td><td char=".">6 </td><td char=".">12 </td><td char=".">41 </td><td char=".">2.22 </td><td char=".">8.29e-3 </td></tr><tr><td>    Methyltransferase activity </td><td>MF </td><td char=".">5 </td><td char=".">23 </td><td char=".">99 </td><td char=".">1.76 </td><td char=".">6.45e-3 </td></tr><tr><td>    Nuclease activity </td><td>MF </td><td char=".">5 </td><td char=".">24 </td><td char=".">98 </td><td char=".">1.85 </td><td char=".">2.31e-3 </td></tr><tr><td>    RNA binding </td><td>MF </td><td char=".">4 </td><td char=".">75 </td><td char=".">407 </td><td char=".">1.40 </td><td char=".">2.63e-3 </td></tr><tr><td colspan="7">3′-Splice site variation </td></tr><tr><td>    Antigen presentation </td><td>BP </td><td char=".">5 </td><td char=".">13 </td><td char=".">36 </td><td char=".">2.09 </td><td char=".">6.34e-3 </td></tr><tr><td>    DNA metabolism </td><td>BP </td><td char=".">5 </td><td char=".">93 </td><td char=".">402 </td><td char=".">1.34 </td><td char=".">2.31e-3 </td></tr><tr><td>    Nucleotide metabolism </td><td>BP </td><td char=".">6 </td><td char=".">37 </td><td char=".">142 </td><td char=".">1.51 </td><td char=".">7.08e-3 </td></tr><tr><td>    Response to stress </td><td>BP </td><td char=".">4 </td><td char=".">140 </td><td char=".">673 </td><td char=".">1.20 </td><td char=".">1.42e-2 </td></tr><tr><td>    RNA metabolism </td><td>BP </td><td char=".">5 </td><td char=".">74 </td><td char=".">329 </td><td char=".">1.30 </td><td char=".">1.41e-2 </td></tr><tr><td>    ATP binding </td><td>MF </td><td char=".">6 </td><td char=".">172 </td><td char=".">851 </td><td char=".">1.17 </td><td char=".">1.93e-2 </td></tr><tr><td colspan="7">Intron retention </td></tr><tr><td>    Cholesterol metabolism </td><td>BP </td><td char=".">7 </td><td char=".">11 </td><td char=".">47 </td><td char=".">2.12 </td><td char=".">1.56e-2 </td></tr><tr><td>    Steroid biosynthesis </td><td>BP </td><td char=".">6 </td><td char=".">14 </td><td char=".">46 </td><td char=".">2.76 </td><td char=".">2.90e-4 </td></tr><tr><td>    Steroid metabolism </td><td>BP </td><td char=".">6 </td><td char=".">21 </td><td char=".">101 </td><td char=".">1.89 </td><td char=".">4.42e-3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>BP, biological process; CC, cellular component; MF, molecular function.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>‘No. genes in a given AS type’ indicates the number of genes with the specific AS pattern and assigned to a given GO term.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>‘Total No. genes’ indicates the number of genes with any of the AS patterns assigned to a given GO term.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194852" target="_blank">View Large</a></div></div><p>Each tissue has its own characteristics, resulting in a unique biological function, morphology and physiology. These characteristics are due to the differential expressions of genes and/or transcripts. Similar to the analysis of GO terms for different AS patterns, we searched for GO terms significantly enriched within the group of tissue-specific transcripts. Significantly enriched GO terms for the group of tissue-specific transcripts in each tissue are in accordance with our previous knowledge. For example, GO terms closely related to brain functions (i.e. central nervous system development, synaptic vesicle, ion channel activity and neurotransmitter transport) were found to be enriched in brain-specific transcripts (See Supplemental Tables 1 and 2). Similar profiles were obtained for the tissue-specific genes. These results provide indirect evidence of the reliability of our approach for identifying tissue-specific genes and transcripts.</p><p>In conclusion, we have presented a detailed algorithm for genome-based clustering and development of a splice graph to identify AS events and variable transcripts derived from individual genes. Our algorithm reduced the possibility of false transcript formation because we applied a scoring rule and a statistical method for the selection of valid AS sets so that only exon/intron sets and alternative linkages with sufficient evidence were selected and used to build the splice graph. We also carried out additional classification of the final transcript based on the mRNA/EST query alignment information supporting the transcript structure. We then subdivided the transcript into one of the three classes according to its reliability. We also provided statistical applications for identifying tissue-specific genes, transcripts and AS events based on EST tissue information. By adapting very conservative criteria for genomic mapping and allowing statistically significant alternative splice relationships, we identified AS information and transcript variants with fewer false positives. The GO analyses support our methodology and tissue specificity results, suggesting that we correctly identified the set of tissue-specific transcripts.</p><p>Our method constitutes an integrated system and can be easily applied to other organisms for which full genome sequences are available and the number of complete mRNAs is sufficient. In this study, we applied our system to human and mouse sequences. The results obtained from this study might help clarify the complex biological nature of AS and the differential expression of transcript variants in many tissues. This can also provide information about the reliable exon sets, including potential AS exons, which can be employed in other AS detection methods, including the design of a splice microarray and a training set for the machine learning approach.</p><p>This work was supported by grants (nos. FGM0300411 and FGM0300512) from the Korean Science and Engineering Foundation to the international joint research project (M6-0401-00-0178) between the Korean Research Institute of Bioscience and Biotechnology and Weizmann Institute of Science in Israel.</p><p><strong>Supplementary Data:</strong> Supplementary material is available online at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>.</p>                            <h2 scrollto-destination=87194859 id="87194859" class="backreferences-title" >REFERENCES</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">A genomic view of AS</div>, <div class="source">Nature Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">13</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20view%20of%20AS&amp;author=B.%20Modrek&amp;author=C.%20Lee&amp;publication_year=2002&amp;journal=Nature%20Genet.&amp;volume=30&amp;pages=13-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0102-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0102-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0102-13"> </span></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kan</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Rouchka</div> <div class="given-names">E.C.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">and States</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Gene structure prediction and AS analysis using genomically aligned ESTs</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">889</div>-<div class="lpage">900</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20structure%20prediction%20and%20AS%20analysis%20using%20genomically%20aligned%20ESTs&amp;author=Z.%20Kan&amp;author=E.C.%20Rouchka&amp;author=W.%20Gish&amp;author=D.%20and%20States&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=889-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.155001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.155001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.155001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11337482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Resch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Grasso</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">and Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Genome-wide detection of alternative slicing in expressed sequences of human genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">2850</div>-<div class="lpage">2859</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20detection%20of%20alternative%20slicing%20in%20expressed%20sequences%20of%20human%20genes&amp;author=B.%20Modrek&amp;author=A.%20Resch&amp;author=C.%20Grasso&amp;author=C.%20and%20Lee&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=2850-2859" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.13.2850" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.13.2850" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.13.2850"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11433032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eyras</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Caccamo</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Curwen</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">and Clamp</div> <div class="given-names">M.</div></div>. </div><div class="article-title">ESTGenes: AS from ESTs in Ensembl</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">976</div>-<div class="lpage">987</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ESTGenes%3A%20AS%20from%20ESTs%20in%20Ensembl&amp;author=E.%20Eyras&amp;author=M.%20Caccamo&amp;author=V.%20Curwen&amp;author=M.%20and%20Clamp&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=976-987" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1862204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1862204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1862204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15123595" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maniatis</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tasic</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Alternative pre-mRNA splicing and proteome expansion in metazoans</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">418</div> (pg. <div class="fpage">236</div>-<div class="lpage">243</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20pre-mRNA%20splicing%20and%20proteome%20expansion%20in%20metazoans&amp;author=T.%20Maniatis&amp;author=B.%20Tasic&amp;publication_year=2002&amp;journal=Nature&amp;volume=418&amp;pages=236-243" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/418236a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F418236a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F418236a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12110900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamm</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Signals and their transduction pathways regulating AS: a new dimension of the human genome</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">2409</div>-<div class="lpage">2416</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signals%20and%20their%20transduction%20pathways%20regulating%20AS%3A%20a%20new%20dimension%20of%20the%20human%20genome&amp;author=S.%20Stamm&amp;publication_year=2002&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=11&amp;pages=2409-2416" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/11.20.2409" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2F11.20.2409" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2F11.20.2409"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12351576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kornblihtt</div> <div class="given-names">A. R.</div></div>,  <div class="name"><div class="surname">De La Mata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fededa</div> <div class="given-names">J.P.</div></div>, et al. </div><div class="article-title">Multiple links between transcription and splicing</div>, <div class="source">RNA</div> , <div class="year">2004</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1489</div>-<div class="lpage">1498</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20links%20between%20transcription%20and%20splicing&amp;author=A.%20R.%20Kornblihtt&amp;author=M.%20De%20La%20Mata&amp;author=J.P.%20Fededa&amp;publication_year=2004&amp;journal=RNA&amp;volume=10&amp;pages=1489-1498" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.7100104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.7100104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.7100104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15383674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lewis</div> <div class="given-names">B. P.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">and Brenner</div> <div class="given-names">S. E.</div></div>. </div><div class="article-title">Evidence for the widespread coupling of AS and nonsense-mediated mRNA decay in humans</div>, <div class="source">Proc. Natl. Acad. Sci.USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">189</div>-<div class="lpage">192</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20the%20widespread%20coupling%20of%20AS%20and%20nonsense-mediated%20mRNA%20decay%20in%20humans&amp;author=B.%20P.%20Lewis&amp;author=R.E.%20Green&amp;author=S.%20E.%20and%20Brenner&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.USA&amp;volume=100&amp;pages=189-192" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0136770100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0136770100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0136770100"> </span></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cartegni</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chew</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">and Krainer</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Listening to silence and understanding nonsense: exonic mutations that affect splicing</div>, <div class="source">Nature Rev.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">285</div>-<div class="lpage">298</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Listening%20to%20silence%20and%20understanding%20nonsense%3A%20exonic%20mutations%20that%20affect%20splicing&amp;author=L.%20Cartegni&amp;author=S.%20Chew&amp;author=A.%20and%20Krainer&amp;publication_year=2002&amp;journal=Nature%20Rev.&amp;volume=3&amp;pages=285-298" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg775" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg775" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg775"> </span></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bracco</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Kearsey</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The relevance of alternative RNA splicing to pharmacogenomics</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">346</div>-<div class="lpage">353</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relevance%20of%20alternative%20RNA%20splicing%20to%20pharmacogenomics&amp;author=L.%20Bracco&amp;author=J.%20Kearsey&amp;publication_year=2003&amp;journal=Trends%20Biotechnol.&amp;volume=21&amp;pages=346-353" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-7799(03)00146-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-7799(03)00146-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-7799(03)00146-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12902171" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gracia-Blanco</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Baraniak</div> <div class="given-names">A. P.</div></div>,  <div class="name"><div class="surname">and Lasda</div> <div class="given-names">E. L.</div></div>. </div><div class="article-title">AS in disease and therapy</div>, <div class="source">Nature Biotech.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">535</div>-<div class="lpage">546</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AS%20in%20disease%20and%20therapy&amp;author=M.%20A.%20Gracia-Blanco&amp;author=A.%20P.%20Baraniak&amp;author=E.%20L.%20and%20Lasda&amp;publication_year=2004&amp;journal=Nature%20Biotech.&amp;volume=22&amp;pages=535-546" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt964" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt964" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt964"> </span></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brett</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hanke</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lehmann</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">EST comparison indicates 38% of human mRNAs contain possible alternative splice forms</div>, <div class="source">FEBS Lett.</div> , <div class="year">2000</div>, vol. <div class="volume">474</div> (pg. <div class="fpage">83</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST%20comparison%20indicates%2038%25%20of%20human%20mRNAs%20contain%20possible%20alternative%20splice%20forms&amp;author=D.%20Brett&amp;author=J.%20Hanke&amp;author=G.%20Lehmann&amp;publication_year=2000&amp;journal=FEBS%20Lett.&amp;volume=474&amp;pages=83-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(00)01581-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(00)01581-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(00)01581-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10828456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thanaraj</div> <div class="given-names">T. A.</div></div>,  <div class="name"><div class="surname">Stamm</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">ASD: the AS Database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D64</div>-<div class="lpage">D69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ASD%3A%20the%20AS%20Database&amp;author=T.%20A.%20Thanaraj&amp;author=S.%20Stamm&amp;author=F.%20Clark&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D64-D69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zink</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Korn</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Genome wide identification and classification of AS based on EST data</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2579</div>-<div class="lpage">2585</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20wide%20identification%20and%20classification%20of%20AS%20based%20on%20EST%20data&amp;author=S.%20Gupta&amp;author=D.%20Zink&amp;author=B.%20Korn&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=2579-2585" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15117759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">H. D.</div></div>,  <div class="name"><div class="surname">Horng</div> <div class="given-names">J. T.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C. C.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">ProSplicer: a database of putative AS information derived from protein, mRNA and expressed sequence tag sequence data</div>, <div class="source">Genome Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">R29</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ProSplicer%3A%20a%20database%20of%20putative%20AS%20information%20derived%20from%20protein%2C%20mRNA%20and%20expressed%20sequence%20tag%20sequence%20data&amp;author=H.%20D.%20Huang&amp;author=J.%20T.%20Horng&amp;author=C.%20C.%20Lee&amp;author=B.J.%20Liu&amp;publication_year=2003&amp;journal=Genome%20Biol.&amp;volume=4&amp;pages=R29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2003-4-4-r29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2003-4-4-r29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2003-4-4-r29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12702210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Castle</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Garrett-Engele</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">2141</div>-<div class="lpage">2144</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20survey%20of%20human%20alternative%20pre-mRNA%20splicing%20with%20exon%20junction%20microarrays&amp;author=J.%20M.%20Johnson&amp;author=J.%20Castle&amp;author=P.%20Garrett-Engele&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=2141-2144" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1090100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1090100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1090100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14684825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Shai</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Misquitta</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Revealing global regulatory features of mammalian AS using a quantitative microarray platform</div>, <div class="source">Mol. Cell</div> , <div class="year">2004</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">929</div>-<div class="lpage">941</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Revealing%20global%20regulatory%20features%20of%20mammalian%20AS%20using%20a%20quantitative%20microarray%20platform&amp;author=Q.%20Pan&amp;author=O.%20Shai&amp;author=C.%20Misquitta&amp;publication_year=2004&amp;journal=Mol.%20Cell&amp;volume=16&amp;pages=929-941" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2004.12.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2004.12.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2004.12.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15610736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fehlbaum</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Guihal</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bracco</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">and Cochet</div> <div class="given-names">O.</div></div>. </div><div class="article-title">A microarray configuration to quantify expression levels and relative abundance of splice variants</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> pg. <div class="fpage">e47</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microarray%20configuration%20to%20quantify%20expression%20levels%20and%20relative%20abundance%20of%20splice%20variants&amp;author=P.%20Fehlbaum&amp;author=C.%20Guihal&amp;author=L.%20Bracco&amp;author=O.%20and%20Cochet&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=e47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gni047" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgni047" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgni047"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15760843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cawley</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>. </div><div class="article-title">HMM sampling and applications to gene finding and AS</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">ii36</div>-<div class="lpage">ii41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HMM%20sampling%20and%20applications%20to%20gene%20finding%20and%20AS&amp;author=S.%20Cawley&amp;author=L.%20Pachter&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=ii36-ii41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg1057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg1057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg1057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534169" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dror</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Soreck</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">and Shamir</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Accurate identification of alternatively spliced exons using support vector machine</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">897</div>-<div class="lpage">901</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20identification%20of%20alternatively%20spliced%20exons%20using%20support%20vector%20machine&amp;author=G.%20Dror&amp;author=R.%20Soreck&amp;author=R.%20and%20Shamir&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=21&amp;pages=897-901" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti132" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti132" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti132"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15531599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ratsch</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Sonnenburg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">and Scholkopf</div> <div class="given-names">B.</div></div>. </div><div class="article-title">RASE: recognition of alternatively spliced exons in <em>C. elegans</em></div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">i369</div>-<div class="lpage">i377</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RASE%3A%20recognition%20of%20alternatively%20spliced%20exons%20in%20C.%20elegans&amp;author=G.%20Ratsch&amp;author=S.%20Sonnenburg&amp;author=B.%20and%20Scholkopf&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=i369-i377" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Generating consensus sequences from partial order multiple sequence alignment graphs</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">999</div>-<div class="lpage">1008</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generating%20consensus%20sequences%20from%20partial%20order%20multiple%20sequence%20alignment%20graphs&amp;author=C.%20Lee&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=999-1008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12761063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leipzig</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pevzner</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">and Heber</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The AS Gallery (ASG): bridging the gap between genome and transcriptome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">3977</div>-<div class="lpage">3983</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20AS%20Gallery%20(ASG)%3A%20bridging%20the%20gap%20between%20genome%20and%20transcriptome&amp;author=J.%20Leipzig&amp;author=P.%20Pevzner&amp;author=S.%20and%20Heber&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=3977-3983" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh731" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh731" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh731"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15292448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">B. T. K.</div></div>,  <div class="name"><div class="surname">Tan</div> <div class="given-names">T. W.</div></div>,  <div class="name"><div class="surname">Ranganathan</div> <div class="given-names">S.</div></div>. </div><div class="article-title">DEDB: a database of <em>Drosophila melanogaster</em> exons in splicing graph form</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">189</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DEDB%3A%20a%20database%20of%20Drosophila%20melanogaster%20exons%20in%20splicing%20graph%20form&amp;author=B.%20T.%20K.%20Lee&amp;author=T.%20W.%20Tan&amp;author=S.%20Ranganathan&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=189" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-5-189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15581431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Shin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">and Lee</div> <div class="given-names">S.</div></div>. </div><div class="article-title">ECgene: Genome-based EST clustering and gene modeling for AS</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">566</div>-<div class="lpage">576</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ECgene%3A%20Genome-based%20EST%20clustering%20and%20gene%20modeling%20for%20AS&amp;author=N.%20Kim&amp;author=S.%20Shin&amp;author=S.%20and%20Lee&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=566-576" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3030405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3030405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3030405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15805497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xing</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Resch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">and Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">The multiassembly problem: Reconstructing multiple transcript isoforms from EST fragment mixtures</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">426</div>-<div class="lpage">441</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20multiassembly%20problem%3A%20Reconstructing%20multiple%20transcript%20isoforms%20from%20EST%20fragment%20mixtures&amp;author=Y.%20Xing&amp;author=A.%20Resch&amp;author=C.%20and%20Lee&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=426-441" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1304504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1304504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1304504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14962984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hertel</div> <div class="given-names">K. J.</div></div>,  <div class="name"><div class="surname">Lynch</div> <div class="given-names">K. W.</div></div>,  <div class="name"><div class="surname">Maniatis</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Common themes in the function of transcription and splicing enhancers</div>, <div class="source">Curr. Opin. Cell Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">350</div>-<div class="lpage">357</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20themes%20in%20the%20function%20of%20transcription%20and%20splicing%20enhancers&amp;author=K.%20J.%20Hertel&amp;author=K.%20W.%20Lynch&amp;author=T.%20Maniatis&amp;publication_year=1997&amp;journal=Curr.%20Opin.%20Cell%20Biol.&amp;volume=9&amp;pages=350-357" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0955-0674(97)80007-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0955-0674(97)80007-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0955-0674(97)80007-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9159075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ladd</div> <div class="given-names">A.N.</div></div>,  <div class="name"><div class="surname">Cooper</div> <div class="given-names">T.A.</div></div>. </div><div class="article-title">Finding signals that regulate AS in the post-genomic era</div>, <div class="source">Genome Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> pg. <div class="fpage">0008</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Finding%20signals%20that%20regulate%20AS%20in%20the%20post-genomic%20era&amp;author=A.N.%20Ladd&amp;author=T.A.%20Cooper&amp;publication_year=2002&amp;journal=Genome%20Biol.&amp;volume=3&amp;pages=0008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2002-3-11-reviews0008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2002-3-11-reviews0008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2002-3-11-reviews0008"> </span></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Genome-wide detection of tissue-specific AS in the human transcriptome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">3754</div>-<div class="lpage">3766</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20detection%20of%20tissue-specific%20AS%20in%20the%20human%20transcriptome&amp;author=Q.%20Xu&amp;author=B.%20Modrek&amp;author=C.%20Lee&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=3754-3766" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf492" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf492" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf492"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12202761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kirov</div> <div class="given-names">S. A.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Baker</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">GeneKeyDB: A lightweight, gene-centric, relational database to support data mining environments</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">72</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneKeyDB%3A%20A%20lightweight%2C%20gene-centric%2C%20relational%20database%20to%20support%20data%20mining%20environments&amp;author=S.%20A.%20Kirov&amp;author=X.%20Peng&amp;author=E.%20Baker&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-72" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-72"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15790402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT-The BLAST-Like Alignment Tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">565</div>-<div class="lpage">664</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT-The%20BLAST-Like%20Alignment%20Tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=565-664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Florea</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hartzell</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Rubin</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Computer program for aligning a cDNA sequence with a genomic DNA sequence, Genome Res.</div>, <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">967</div>-<div class="lpage">974</div>)</div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kan</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Startes</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Selecting for functional alternative splices in ESTs</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1837</div>-<div class="lpage">1845</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selecting%20for%20functional%20alternative%20splices%20in%20ESTs&amp;author=Z.%20Kan&amp;author=D.%20Startes&amp;author=W.%20Gish&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1837-1845" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.764102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.764102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.764102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Audic</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Claverie</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">The significance of digital gene expression profiles</div>, <div class="source">Genome Res.</div> , <div class="year">1997</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">986</div>-<div class="lpage">995</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20significance%20of%20digital%20gene%20expression%20profiles&amp;author=S.%20Audic&amp;author=J.%20M.%20Claverie&amp;publication_year=1997&amp;journal=Genome%20Res.&amp;volume=7&amp;pages=986-995" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.7.10.986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.7.10.986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.7.10.986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9331369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yekutieli</div> <div class="given-names">D.</div></div>. </div><div class="article-title">The control of the false discovery rate in multiple hypotheses testing under dependency</div>, <div class="source">Annal. Stat.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">1165</div>-<div class="lpage">1188</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20control%20of%20the%20false%20discovery%20rate%20in%20multiple%20hypotheses%20testing%20under%20dependency&amp;author=Y.%20Benjamini&amp;author=D.%20Yekutieli&amp;publication_year=2001&amp;journal=Annal.%20Stat.&amp;volume=29&amp;pages=1165-1188" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1214/aos/1013699998" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1214%2Faos%2F1013699998" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1214%2Faos%2F1013699998"> </span></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeeberg</div> <div class="given-names">B. R.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">GoMiner: a resource for biological interpretation of genomic and proteomic data</div>, <div class="source">Genome Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">R28</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GoMiner%3A%20a%20resource%20for%20biological%20interpretation%20of%20genomic%20and%20proteomic%20data&amp;author=B.%20R.%20Zeeberg&amp;author=W.%20Feng&amp;author=G.%20Wang&amp;publication_year=2003&amp;journal=Genome%20Biol.&amp;volume=4&amp;pages=R28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2003-4-4-r28" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2003-4-4-r28" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2003-4-4-r28"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12702209" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Shemesh</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cohen</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">A non-EST-based method for exon-skipping prediction</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1617</div>-<div class="lpage">1623</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20non-EST-based%20method%20for%20exon-skipping%20prediction&amp;author=R.%20Sorek&amp;author=R.%20Shemesh&amp;author=Y.%20Cohen&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1617-1623" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2572604" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2572604" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2572604"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15289480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yeo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Holste</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kreiman</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">and Burge</div> <div class="given-names">C. B.</div></div>. </div><div class="article-title">Variation in AS across human tissues</div>, <div class="source">Genome Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">R74</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20AS%20across%20human%20tissues&amp;author=G.%20Yeo&amp;author=D.%20Holste&amp;author=G.%20Kreiman&amp;author=C.%20B.%20and%20Burge&amp;publication_year=2004&amp;journal=Genome%20Biol.&amp;volume=5&amp;pages=R74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r74" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-10-r74" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-10-r74"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15461793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=400632&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl011/1/dsl011_Supplementary_Data.zip?Expires=2147483647&amp;Signature=fnquzJ0FtAe07-bW5xCepv4LR3Tr75ZecIcya~JbmVUJ5sfWEb5td4VGpIkUasKGOm-ela8RCQmvz0Yq9XSzwEcLGF4kwHTIxspF-aWXjtMDm5rMCEB9~U88l-NzwkHA2r~8rNACGo6Qr7jpilNabY5IfE1~F7Ytvm3U9to3DPuWya~v5eJXSAbfVMZrmO7hev9lZGZXkUrZqLKE0DJwbM79eaxLz9zGqS-RJemK4c3WTj7Na9FrNEtWQo7DBLn6ZxdLqvP9kEokxvRBVNVvXT~KDA7wdLS50dxdcp6jUUckGcMqMec9GdL5CScZH5mFV5eUETguISNmvwsdiK5VOQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">398</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000243280300004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">18</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl011"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="400632"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000243280300004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f5%2f229%2f400632">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000243280300004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (18)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F5%2F229%2F400632" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/400632'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="TISA: Tissue-specific Alternative Splicing in Human and Mouse Genes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl011");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f5%2f229%2f400632">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

