

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/4/233/501892","@type":"ScholarlyArticle","name":"Learning Biomarkers of Pluripotent Stem Cells in Mouse","datePublished":"2011-07-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2011-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr016","keywords":["pluripotency","machine learning","feature selection","genetic algorithm","support vector machine"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/4/233/501892","author":[{"name":"Scheubert, Lena","affiliation":"1  Institute of Computer Science, University of Osnabrück ,  Albrechtstr. 28, 49076 Osnabrück ,  Germany","@type":"Person"},{"name":"Schmidt, Rainer","affiliation":"2  Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock ,  Ernst-Heydemann-Str. 8, 18057, Rostock ,  Germany","@type":"Person"},{"name":"Repsilber, Dirk","affiliation":"3  Leibniz Institute for Farm Animal Biology (FBN Dummerstorf) ,  Wilhelm-Stahl Allee 2, 18196 Dummerstorf ,  Germany","@type":"Person"},{"name":"Luštrek, Mitja","affiliation":"2  Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock ,  Ernst-Heydemann-Str. 8, 18057, Rostock ,  Germany    4  Department of Intelligent Systems ,  Jožef Stefan Institute ,  Jamova cesta 39, 1000 Ljubljana ,  Slovenia","@type":"Person"},{"name":"Fuellen, Georg","affiliation":"2  Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock ,  Ernst-Heydemann-Str. 8, 18057, Rostock ,  Germany","@type":"Person"}],"description":"Abstract.  Pluripotent stem cells are able to self-renew, and to differentiate into all adult cell types. Many studies report data describing these cells, and c","pageStart":"233","pageEnd":"251","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Learning Biomarkers of Pluripotent Stem Cells in Mouse","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Learning Biomarkers of Pluripotent Stem Cells in Mouse" />
    <meta property="og:description" content="Abstract.  Pluripotent stem cells are able to self-renew, and to differentiate into all adult cell types. Many studies report data describing these cells, and c" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/4/233/501892" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-07-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Scheubert, Lena" /><meta name="citation_author_institution" content="Institute of Computer Science, University of Osnabrück, Albrechtstr. 28, 49076 Osnabrück, Germany" /><meta name="citation_author" content="Schmidt, Rainer" /><meta name="citation_author_institution" content="Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock, Ernst-Heydemann-Str. 8, 18057, Rostock, Germany" /><meta name="citation_author" content="Repsilber, Dirk" /><meta name="citation_author_institution" content="Leibniz Institute for Farm Animal Biology (FBN Dummerstorf), Wilhelm-Stahl Allee 2, 18196 Dummerstorf, Germany" /><meta name="citation_author" content="Luštrek, Mitja" /><meta name="citation_author_institution" content="Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock, Ernst-Heydemann-Str. 8, 18057, Rostock, Germany" /><meta name="citation_author_institution" content="Department of Intelligent Systems, Jožef Stefan Institute, Jamova cesta 39, 1000 Ljubljana, Slovenia" /><meta name="citation_author" content="Fuellen, Georg" /><meta name="citation_author_institution" content="Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock, Ernst-Heydemann-Str. 8, 18057, Rostock, Germany" /><meta name="citation_title" content="Learning Biomarkers of Pluripotent Stem Cells in Mouse" /><meta name="citation_firstpage" content="233" /><meta name="citation_lastpage" content="251" /><meta name="citation_doi" content="10.1093/dnares/dsr016" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2011/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genetics of complex disease: approaches, problems, and solutions; Am. J. Respir. Crit. Care Med.;  citation_year=1997;  citation_volume=156;  citation_pages=S103-9; " /><meta name="citation_reference" content="citation_title=Reverse engineering the genotype-phenotype map with natural genetic variation; Nature;  citation_year=2008;  citation_volume=456;  citation_pages=738-44; " /><meta name="citation_reference" content="citation_title=Regulatory circuits underlying pluripotency and reprogramming; Trends Pharmacol. Sci.;  citation_year=2009;  citation_volume=30;  citation_pages=296-302; " /><meta name="citation_reference" content="citation_title=NCBI GEO: archive for high-throughput functional genomic data; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D885-90; " /><meta name="citation_reference" content="citation_title=Significance analysis of microarrays applied to the ionizing radiation response; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=5116-21; " /><meta name="citation_reference" content="citation_title=Statistical issues in cDNA microarray data analysis; Methods Mol. Biol.;  citation_year=2003;  citation_volume=224;  citation_pages=111-36; " /><meta name="citation_reference" content="citation_title=A structural mixed model for variances in differential gene expression studies; Genet. Res. Camb.;  citation_year=2007;  citation_volume=89;  citation_pages=19-25; " /><meta name="citation_reference" content="citation_title=Simple discriminant functions identify small sets of genes that distinguish cancer phenotype from normal; Genome Inform.;  citation_year=2005;  citation_volume=16;  citation_pages=245-53; " /><meta name="citation_reference" content="citation_title=Many accurate small-discriminatory feature subsets exist in microarray transcript data: biomarker discovery; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=97" /><meta name="citation_reference" content="citation_title=Novel methods to identify biologically relevant genes for leukemia and prostate cancer from gene expression profiles; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=274" /><meta name="citation_reference" content="citation_title=Development and validation of predictive molecular signatures; Curr. Mol. Med.;  citation_year=2010;  citation_volume=10;  citation_pages=173-9; " /><meta name="citation_reference" content="citation_title=Multiclass cancer classification and biomarker discovery using GA-based algorithms; citation_text=doi:10.1093/bioinformatics/bti419; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=2691-7; " /><meta name="citation_reference" content="citation_title=Evolutionary algorithms for finding optimal gene sets in microarray prediction; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=45-52; " /><meta name="citation_reference" content="citation_title=How is pluripotency determined and maintained?; Development;  citation_year=2007;  citation_volume=134;  citation_pages=635-46; " /><meta name="citation_reference" content="citation_title=iPS cell technology in regenerative medicine; Ann. N Y Acad. Sci.;  citation_year=2010;  citation_volume=1192;  citation_pages=38-44; " /><meta name="citation_reference" content="citation_title=Small molecules that modulate embryonic stem cell fate and somatic cell reprogramming; Trends Pharmacol. Sci.;  citation_year=2010;  citation_volume=31;  citation_pages=36-45; " /><meta name="citation_reference" content="citation_title=Reprogramming with defined factors: from induced pluripotency to induced transdifferentiation; citation_text=Epub ahead of print; Mol. Hum. Reprod.;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Resolution of cell fate decisions revealed by single-cell gene expression analysis from zygote to blastocyst; Dev. Cell;  citation_year=2010;  citation_volume=18;  citation_pages=675-85; " /><meta name="citation_reference" content="citation_title=Defining developmental potency and cell lineage trajectories by expression profiling of differentiating mouse embryonic stem cells; citation_text=doi:10.1093/dnares/dsn035; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=73-80; " /><meta name="citation_reference" content="citation_title=Uncovering early response of gene regulatory networks in ESCs by systematic induction of transcription factors; Cell Stem Cell;  citation_year=2009;  citation_volume=5;  citation_pages=420-33; " /><meta name="citation_reference" content="citation_publisher=Addison-Wesley Professional, ; Genetic Algorithms in Search, Optimization, and Machine Learning;  citation_year=1989; " /><meta name="citation_reference" content="citation_title=Summaries of Affymetrix GeneChip probe level data; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=e15" /><meta name="citation_reference" content="citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data; Biostatistics;  citation_year=2003;  citation_volume=4;  citation_pages=249-64; " /><meta name="citation_reference" content="citation_author=Affymetrix; Affymetrix Power Tools (APT) Software Package;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=A comparison of normalization methods for high density oligonucleotide array data based on variance and bias; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=185-93; " /><meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=28-33; " /><meta name="citation_reference" content="citation_title=Statistical significance for genomewide studies; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=9440-5; " /><meta name="citation_reference" content="citation_title=GECCO '07 Proceedings of the 9th annual conference on Genetic and evolutionary computation; citation_text=A platform for the selection of genes in DNA microarraydata using evolutionary algorithms;  citation_year=2007;  citation_pages=415-23; " /><meta name="citation_reference" content="citation_title=The WEKA data mining software: an update; SIGKDD Explorations;  citation_year=2009;  citation_volume=11;  citation_pages=10-8; " /><meta name="citation_reference" content="citation_text=Estimating continuous distributions in Bayesian classifiers, In Proceedings of the Eleventh Conference on Uncertainty in Artifical Intelligence;  citation_year=1995;  citation_pages=338-45; " /><meta name="citation_reference" content="citation_publisher=Morgan Kaufmann Publishers, ; C4.5: Programms for Machine Learning;  citation_year=1993; " /><meta name="citation_reference" content="citation_title=Random Forest; Mach. Learn.;  citation_year=2001;  citation_volume=45;  citation_pages=5-32; " /><meta name="citation_reference" content="citation_title=Instance-based learning algorithms; Mach. Learn.;  citation_year=1991;  citation_volume=6;  citation_pages=37-66; " /><meta name="citation_reference" content="citation_title=Fast training of support vector machines using sequential minimal optimization; citation_publisher=MIT Press, ; Advances in Kernel Methods—Support Vector Learning;  citation_year=1998; " /><meta name="citation_reference" content="LIBSVM – A Library for Support Vector Machines;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=Implementation of Breiman's Random Forest Machine Learning Algorithm, In ECE591Q Machine Learning Conference;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=A GA-based feature selection and parameters optimizationfor support vector machines; Expert Syst. Appl.;  citation_year=2006;  citation_volume=31;  citation_pages=231-40; " /><meta name="citation_reference" content="citation_title=Third International Conference on Genetic Algorithms; citation_text=pp.; citation_publisher=Morgan Kaufmann Publishers, ;  citation_pages=2-9; " /><meta name="citation_reference" content="citation_title=GeneTrail—advanced gene set enrichment analysis; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W186-92; " /><meta name="citation_reference" content="citation_title=The PluriNetWork: an in-silico representation of the network underlying pluripotency in mouse, and its applications; PLoS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e15165" /><meta name="citation_reference" content="citation_title=Systems biology of stem cell fate and cellular reprogramming; Nat. Rev. Mol. Cell Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=672-81; " /><meta name="citation_reference" content="citation_title=Regulatory networks define phenotypic classes of human stem cell lines; Nature;  citation_year=2008;  citation_volume=455;  citation_pages=401-405; " /><meta name="citation_reference" content="citation_title=AutoSOME: a clustering method for identifying gene expression modules without prior knowledge of cluster number; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=117" /><meta name="citation_reference" content="citation_title=More synergetic cooperation of Yamanaka factors in induced pluripotent stem cells than in embryonic stem cells; Cell Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1127-38; " /><meta name="citation_reference" content="citation_title=Signaling pathways in cancer and embryonic stem cells; Stem Cell Rev.;  citation_year=2007;  citation_volume=3;  citation_pages=7-17; " /><meta name="citation_reference" content="citation_title=Molecular mechanisms involved in self-renewal and pluripotency of embryonic stem cells; J. Cell Physiol.;  citation_year=2007;  citation_volume=211;  citation_pages=279-86; " /><meta name="citation_reference" content="citation_title=An explicit description of the reproducing kernel hilbert spaces of Gaussian RBF kernels; IEEE Trans. Info. Theory;  citation_year=2006;  citation_volume=52;  citation_pages=4635-43; " /><meta name="citation_reference" content="A Practical Guide to Support Vector Classification;  citation_year=2003; " /><meta name="citation_reference" content="citation_title=Analysis and computational dissection of molecular signature multiplicity; PLoS Comput. Biol.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000790" /><meta name="citation_reference" content="citation_title=KEGG: kyoto encyclopedia of genes and genomes; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=27-30; " /><meta name="citation_reference" content="citation_title=Mutations in FAM134B, encoding a newly identified Golgi protein, cause severe sensory and autonomic neuropathy; citation_text=doi:10.1038/ng.464; Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=1179-81; " /><meta name="citation_reference" content="citation_title=Alkaline phosphatase in the Golgi substance of intestinal epithelium; citation_text=doi:0014-4827(56)90071-4; Exp. Cell Res.;  citation_year=1956;  citation_volume=10;  citation_pages=63-5; " /><meta name="citation_reference" content="citation_title=Oncogenic properties of a novel gene JK-1 located in chromosome 5p and its overexpression in human esophageal squamous cell carcinoma; Int. J. Mol. Med.;  citation_year=2007;  citation_volume=19;  citation_pages=915-23; " /><meta name="citation_reference" content="citation_title=Analysis of a novel JK-1 gene expression in benign and malignant colorectal tumors; Virchows Arch.;  citation_year=2010;  citation_volume=457;  citation_pages=165" /><meta name="citation_reference" content="citation_title=Genetic and functional differences between multipotent neural and pluripotent embryonic stem cells; citation_text=doi:10.1073/pnas.1834200100; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_issue=Suppl. 1);  citation_pages=11866-72; " /><meta name="citation_reference" content="citation_title=Embryotrophic factor-3 from human oviductal cells affects the messenger RNA expression of mouse blastocyst; Biol. Reprod.;  citation_year=2003;  citation_volume=68;  citation_pages=375-82; " /><meta name="citation_reference" content="citation_title=The puromycin-sensitive aminopeptidase PAM-1 is required for meiotic exit and anteroposterior polarity in the one-cell Caenorhabditis elegans embryo; citation_text=doi:10.1242/dev.02615; Development;  citation_year=2006;  citation_volume=133;  citation_pages=4281-92; " /><meta name="citation_reference" content="citation_title=Ring1B is crucial for the regulation of developmental control genes and PRC1 proteins but not X inactivation in embryonic cells; citation_text=doi:10.1083/jcb.200612127; J. Cell Biol.;  citation_year=2007;  citation_volume=178;  citation_pages=219-29; " /><meta name="citation_reference" content="citation_title=Analysis of human and mouse reprogramming of somatic cells to induced pluripotent stem cells. What is in the plate?; citation_text=doi:10.1371/journal.pone.0012664; PLoS ONE;  citation_year=2010;  citation_volume=5; " /><meta name="citation_reference" content="citation_title=New POU dimer configuration mediates antagonistic control of an osteopontin preimplantation enhancer by Oct-4 and Sox-2; Genes Dev.;  citation_year=1998;  citation_volume=12;  citation_pages=2073-90; " /><meta name="citation_reference" content="citation_title=Dazl functions in maintenance of pluripotency and genetic and epigenetic programs of differentiation in mouse primordial germ cells in vivo and in vitro; citation_text=doi:10.1371/journal.pone.0005654; PLoS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e5654" /><meta name="citation_reference" content="citation_title=A hierarchical network controls protein translation during murine embryonic stem cell self-renewal and differentiation; citation_text=doi:10.1016/j.stem.2008.03.013; Cell Stem Cell;  citation_year=2008;  citation_volume=2;  citation_pages=448-60; " /><meta name="citation_reference" content="citation_title=Alternative splicing regulates mouse embryonic stem cell pluripotency and differentiation; citation_text=doi:10.1073/pnas.0912260107; Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=10514-9; " /><meta name="citation_reference" content="citation_title=A gene regulatory network in mouse embryonic stem cells; citation_text=doi:10.1073/pnas.0701014104; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=16438-43; " /><meta name="citation_reference" content="citation_title=GDF3, a BMP inhibitor, regulates cell fate in stem cells and early embryos; citation_text=doi:10.1242/dev.02192; Development;  citation_year=2006;  citation_volume=133;  citation_pages=209-16; " /><meta name="citation_reference" content="citation_title=Human STELLAR, NANOG, and GDF3 genes are expressed in pluripotent cells and map to chromosome 12p13, a hotspot for teratocarcinoma; citation_text=doi:10.1634/stemcells.22-2-169; Stem Cells;  citation_year=2004;  citation_volume=22;  citation_pages=169-79; " /><meta name="citation_reference" content="citation_title=B-MYB is essential for normal cell cycle progression and chromosomal stability of embryonic stem cells; citation_text=doi:10.1371/journal.pone.0002478; PLoS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e2478" /><meta name="citation_reference" content="citation_title=Regulation of the pluripotency marker Rex-1 by Nanog and Sox2; citation_text=doi:10.1074/jbc.M601811200; J. Biol. Chem.;  citation_year=2006;  citation_volume=281;  citation_pages=23319-25; " /><meta name="citation_reference" content="citation_title=Comparative homology agreement search: an effective combination of homology-search methods; citation_text=doi:10.1073/pnas.0405612101; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=13814-9; " /><meta name="citation_reference" content="citation_title=Sensitive enzymatic quantification of 5-hydroxymethylcytosine in genomic DNA; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=e181" /><meta name="citation_reference" content="citation_title=Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1; citation_text=doi:10.1126/science.1170116; Science;  citation_year=2009;  citation_volume=324;  citation_pages=930-5; " /><meta name="citation_reference" content="citation_title=Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification; citation_text=doi:10.1038/nature09303; Nature;  citation_year=2010;  citation_volume=466;  citation_pages=1129-33; " /><meta name="citation_reference" content="citation_title=Dual functions of Tet1 in transcriptional regulation in mouse embryonic stem cells; citation_text=doi:10.1038/nature09934; Nature;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Dynamic regulation of 5-hydroxymethylcytosine in mouse ES cells and during differentiation; citation_text=doi:10.1038/nature10008; Nature;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Direct targets of Klf5 transcription factor contribute to the maintenance of mouse embryonic stem cell undifferentiated state; citation_text=doi:10.1186/1741-7007-8-128; BMC Biol.;  citation_year=2010;  citation_volume=8;  citation_pages=128" /><meta name="citation_reference" content="citation_title=Smad4-dependent pathways control basement membrane deposition and endodermal cell migration at early stages of mouse development; citation_text=doi:10.1186/1471-213X-9-54; BMC Dev. Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=54" /><meta name="citation_reference" content="citation_title=Expression profiling in transgenic FVB/N embryonic stem cells overexpressing STAT3; citation_text=doi:10.1186/1471-213X-8-57; BMC Dev. Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=57" /><meta name="citation_reference" content="citation_title=Incomplete reactivation of Oct4-related genes in mouse embryos cloned from somatic nuclei; Development;  citation_year=2003;  citation_volume=130;  citation_pages=1673-80; " /><meta name="citation_reference" content="citation_title=Mitochondrial DNA replication during differentiation of murine embryonic stem cells; citation_text=doi:10.1242/jcs.016972; J. Cell Sci.;  citation_year=2007;  citation_volume=120;  citation_pages=4025-34; " /><meta name="citation_reference" content="citation_title=Graded Smad2/3 activation is converted directly into levels of target gene expression in embryonic stem cells; citation_text=doi:10.1371/journal.pone.0004268; PLoS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e4268" /><meta name="citation_reference" content="citation_title=Canopy1, a novel regulator of FGF signaling around the midbrain-hindbrain boundary in zebrafish; citation_text=doi:10.1016/j.cub.2006.01.055; Curr. Biol.;  citation_year=2006;  citation_volume=16;  citation_pages=421-7; " /><meta name="citation_reference" content="citation_title=Evolution of non-coding regulatory sequences involved in the developmental process: reflection of differential employment of paralogous genes as highlighted by Sox2 and group B1 Sox genes; Proc. Jpn Acad. Ser. B Phys. Biol. Sci.;  citation_year=2009;  citation_volume=85;  citation_pages=55-68; " /><meta name="citation_reference" content="citation_title=Mouse germline restriction of Oct4 expression by germ cell nuclear factor; Dev. Cell;  citation_year=2001;  citation_volume=1;  citation_pages=377-87; " /><meta name="citation_reference" content="citation_title=The homeoprotein Nanog is required for maintenance of pluripotency in mouse epiblast and ES cells; Cell;  citation_year=2003;  citation_volume=113;  citation_pages=631-42; " /><meta name="citation_reference" content="citation_title=Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors; Cell;  citation_year=2006;  citation_volume=126;  citation_pages=663-76; " /><meta name="citation_reference" content="citation_title=Stem cells and germ cells: microRNA and gene expression signatures; Histol. Histopathol.;  citation_year=2010;  citation_volume=25;  citation_pages=505-13; " /><meta name="citation_reference" content="citation_title=Gene expression profiles during early differentiation of mouse embryonic stem cells; BMC Dev. Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=5" /><meta name="citation_reference" content="citation_title=Stable isotope labeling by amino acids in cell culture (SILAC) and proteome quantitation of mouse embryonic stem cells to a depth of 5111 proteins; Mol Cell Proteomics;  citation_year=2008;  citation_volume=7;  citation_pages=672-83; " /><meta name="citation_reference" content="citation_title=Proteomic analysis of mouse oocytes reveals 28 candidate factors of the “reprogrammome”; J. Proteome Res.;  citation_year=2011;  citation_volume=10;  citation_pages=2140-53; " /><meta name="citation_reference" content="citation_title=Comment on &quot;'Stemness': transcriptional profiling of embryonic and adult stem cells&quot; and &quot;a stem cell molecular signature&quot;; citation_text=author reply 393, doi:10.1126/science.1086384; Science;  citation_year=2003;  citation_volume=302;  citation_pages=393" /><meta name="citation_reference" content="citation_title=Module map of stem cell genes guides creation of epithelial cancer stem cells; citation_text=doi:10.1016/j.stem.2008.02.009; Cell Stem Cell;  citation_year=2008;  citation_volume=2;  citation_pages=333-44; " /><meta name="citation_reference" content="citation_title=Apparently low reproducibility of true differential expression discoveries in microarray studies; citation_text=doi:10.1093/bioinformatics/btn365; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=2057-63; " /><meta name="citation_reference" content="citation_title=Extracting consistent knowledge from highly inconsistent cancer gene data sources; citation_text=doi:10.1186/1471-2105-11-76; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=76" /><meta name="citation_reference" content="citation_title=Dynamic transcription programs during ES cell differentiation towards mesoderm in serum versus serum-freeBMP4 culture; citation_text=doi:10.1186/1471-2164-8-365; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=365" /><meta name="citation_reference" content="citation_title=Gene profiling for determining pluripotent genes in a time course microarray experiment; citation_text=doi:10.1093/biostatistics/kxn017; Biostatistics;  citation_year=2009;  citation_volume=10;  citation_pages=80-93; " /><meta name="citation_reference" content="citation_title=Functional clustering and lineage markers: insights into cellular differentiation and gene function from large-scale microarray studies of purified primary cell populations; citation_text=doi:10.1016/j.ygeno.2010.03.002; Genomics;  citation_year=2010;  citation_volume=95;  citation_pages=328-38; " /><meta name="citation_reference" content="citation_title=Prediction of hybrid biomass in Arabidopsis thaliana by selected parental SNP and metabolic markers; citation_text=doi:10.1007/s00122-009-1191-2; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=239-47; " /><meta name="citation_reference" content="citation_title=PRICAI'06 Proceedings of the 9th Pacific Rim International Conference on Artificial Intelligence; citation_text=Pattern classification in DNA microarray data of multiple tumor types;  citation_year=2006;  citation_pages=1037-41; " /><meta name="citation_reference" content="citation_title=GECCO '07 Proceedings of the 9th annual Conference on Genetic and Evolutionary Computation; citation_text=Evolutionary selection of minimum number of features for classification of gene expression data using genetic algorithms;  citation_year=2007;  citation_pages=401-6; " /><meta name="citation_reference" content="citation_title=Effective gene selection method using bayesian discriminant based criterion and genetic algorithms; J. Signal Proce. Syst.;  citation_year=2008;  citation_volume=50;  citation_pages=293-304; " /><meta name="citation_reference" content="citation_title=An agent-based hybrid system for microarray data analysis; Intell. Syst.;  citation_year=2009;  citation_volume=24;  citation_pages=53-63; " /><meta name="citation_reference" content="citation_text=A filter-based evolutionary approach for selecting features in high-dimensional micro-array data, In 6th International Conference on Intelligent Information Processing;  citation_year=2010;  citation_pages=297-307; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/4/233/1351041/dsr016.pdf" /><meta name="description" content="Abstract.  Pluripotent stem cells are able to self-renew, and to differentiate into all adult cell types. Many studies report data describing these cells, and c" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/4/233/501892" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/4/233/501892" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/4/233/501892");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr016';
    </script>
<title>
	Learning Biomarkers of Pluripotent Stem Cells in Mouse | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./501892" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BanWATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2NoZXViZXJ0LCBMZW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIENvbXB1dGVyIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgT3NuYWJyw7xjaywgQWxicmVjaHRzdHIuIDI4LCA0OTA3NiBPc25hYnLDvGNrLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2NobWlkdCwgUmFpbmVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIGZvciBCaW9zdGF0aXN0aWNzIGFuZCBJbmZvcm1hdGljcyBpbiBNZWRpY2luZSBhbmQgQWdlaW5nIFJlc2VhcmNoLCBVbml2ZXJzaXR5IG9mIFJvc3RvY2ssIEVybnN0LUhleWRlbWFubi1TdHIuIDgsIDE4MDU3LCBSb3N0b2NrLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmVwc2lsYmVyLCBEaXJrIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGVpYm5peiBJbnN0aXR1dGUgZm9yIEZhcm0gQW5pbWFsIEJpb2xvZ3kgKEZCTiBEdW1tZXJzdG9yZiksIFdpbGhlbG0tU3RhaGwgQWxsZWUgMiwgMTgxOTYgRHVtbWVyc3RvcmYsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMdcWhdHJlaywgTWl0amEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgZm9yIEJpb3N0YXRpc3RpY3MgYW5kIEluZm9ybWF0aWNzIGluIE1lZGljaW5lIGFuZCBBZ2VpbmcgUmVzZWFyY2gsIFVuaXZlcnNpdHkgb2YgUm9zdG9jaywgRXJuc3QtSGV5ZGVtYW5uLVN0ci4gOCwgMTgwNTcsIFJvc3RvY2ssIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEludGVsbGlnZW50IFN5c3RlbXMsIEpvxb5lZiBTdGVmYW4gSW5zdGl0dXRlLCBKYW1vdmEgY2VzdGEgMzksIDEwMDAgTGp1YmxqYW5hLCBTbG92ZW5pYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1ZWxsZW4sIEdlb3JnIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIGZvciBCaW9zdGF0aXN0aWNzIGFuZCBJbmZvcm1hdGljcyBpbiBNZWRpY2luZSBhbmQgQWdlaW5nIFJlc2VhcmNoLCBVbml2ZXJzaXR5IG9mIFJvc3RvY2ssIEVybnN0LUhleWRlbWFubi1TdHIuIDgsIDE4MDU3LCBSb3N0b2NrLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJMZWFybmluZyBCaW9tYXJrZXJzIG9mIFBsdXJpcG90ZW50IFN0ZW0gQ2VsbHMgaW4gTW91c2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY3Mgb2YgY29tcGxleCBkaXNlYXNlOiBhcHByb2FjaGVzLCBwcm9ibGVtcywgYW5kIHNvbHV0aW9uczsgQW0uIEouIFJlc3Bpci4gQ3JpdC4gQ2FyZSBNZWQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPVMxMDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV2ZXJzZSBlbmdpbmVlcmluZyB0aGUgZ2Vub3R5cGUtcGhlbm90eXBlIG1hcCB3aXRoIG5hdHVyYWwgZ2VuZXRpYyB2YXJpYXRpb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTY7ICBjaXRhdGlvbl9wYWdlcz03MzgtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgY2lyY3VpdHMgdW5kZXJseWluZyBwbHVyaXBvdGVuY3kgYW5kIHJlcHJvZ3JhbW1pbmc7IFRyZW5kcyBQaGFybWFjb2wuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yOTYtMzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQ0JJIEdFTzogYXJjaGl2ZSBmb3IgaGlnaC10aHJvdWdocHV0IGZ1bmN0aW9uYWwgZ2Vub21pYyBkYXRhOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1EODg1LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaWduaWZpY2FuY2UgYW5hbHlzaXMgb2YgbWljcm9hcnJheXMgYXBwbGllZCB0byB0aGUgaW9uaXppbmcgcmFkaWF0aW9uIHJlc3BvbnNlOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9NTExNi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGlzdGljYWwgaXNzdWVzIGluIGNETkEgbWljcm9hcnJheSBkYXRhIGFuYWx5c2lzOyBNZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjI0OyAgY2l0YXRpb25fcGFnZXM9MTExLTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN0cnVjdHVyYWwgbWl4ZWQgbW9kZWwgZm9yIHZhcmlhbmNlcyBpbiBkaWZmZXJlbnRpYWwgZ2VuZSBleHByZXNzaW9uIHN0dWRpZXM7IEdlbmV0LiBSZXMuIENhbWIuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9MTktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBkaXNjcmltaW5hbnQgZnVuY3Rpb25zIGlkZW50aWZ5IHNtYWxsIHNldHMgb2YgZ2VuZXMgdGhhdCBkaXN0aW5ndWlzaCBjYW5jZXIgcGhlbm90eXBlIGZyb20gbm9ybWFsOyBHZW5vbWUgSW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI0NS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFueSBhY2N1cmF0ZSBzbWFsbC1kaXNjcmltaW5hdG9yeSBmZWF0dXJlIHN1YnNldHMgZXhpc3QgaW4gbWljcm9hcnJheSB0cmFuc2NyaXB0IGRhdGE6IGJpb21hcmtlciBkaXNjb3Zlcnk7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9OTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBtZXRob2RzIHRvIGlkZW50aWZ5IGJpb2xvZ2ljYWxseSByZWxldmFudCBnZW5lcyBmb3IgbGV1a2VtaWEgYW5kIHByb3N0YXRlIGNhbmNlciBmcm9tIGdlbmUgZXhwcmVzc2lvbiBwcm9maWxlczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9Mjc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIHZhbGlkYXRpb24gb2YgcHJlZGljdGl2ZSBtb2xlY3VsYXIgc2lnbmF0dXJlczsgQ3Vyci4gTW9sLiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTczLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpY2xhc3MgY2FuY2VyIGNsYXNzaWZpY2F0aW9uIGFuZCBiaW9tYXJrZXIgZGlzY292ZXJ5IHVzaW5nIEdBLWJhc2VkIGFsZ29yaXRobXM7IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjEwOTMvYmlvaW5mb3JtYXRpY3MvYnRpNDE5OyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTI2OTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGFsZ29yaXRobXMgZm9yIGZpbmRpbmcgb3B0aW1hbCBnZW5lIHNldHMgaW4gbWljcm9hcnJheSBwcmVkaWN0aW9uOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQ1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgaXMgcGx1cmlwb3RlbmN5IGRldGVybWluZWQgYW5kIG1haW50YWluZWQ/OyBEZXZlbG9wbWVudDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz02MzUtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWlQUyBjZWxsIHRlY2hub2xvZ3kgaW4gcmVnZW5lcmF0aXZlIG1lZGljaW5lOyBBbm4uIE4gWSBBY2FkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOTI7ICBjaXRhdGlvbl9wYWdlcz0zOC00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U21hbGwgbW9sZWN1bGVzIHRoYXQgbW9kdWxhdGUgZW1icnlvbmljIHN0ZW0gY2VsbCBmYXRlIGFuZCBzb21hdGljIGNlbGwgcmVwcm9ncmFtbWluZzsgVHJlbmRzIFBoYXJtYWNvbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM2LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXByb2dyYW1taW5nIHdpdGggZGVmaW5lZCBmYWN0b3JzOiBmcm9tIGluZHVjZWQgcGx1cmlwb3RlbmN5IHRvIGluZHVjZWQgdHJhbnNkaWZmZXJlbnRpYXRpb247IGNpdGF0aW9uX3RleHQ9RXB1YiBhaGVhZCBvZiBwcmludDsgTW9sLiBIdW0uIFJlcHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlc29sdXRpb24gb2YgY2VsbCBmYXRlIGRlY2lzaW9ucyByZXZlYWxlZCBieSBzaW5nbGUtY2VsbCBnZW5lIGV4cHJlc3Npb24gYW5hbHlzaXMgZnJvbSB6eWdvdGUgdG8gYmxhc3RvY3lzdDsgRGV2LiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9Njc1LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWZpbmluZyBkZXZlbG9wbWVudGFsIHBvdGVuY3kgYW5kIGNlbGwgbGluZWFnZSB0cmFqZWN0b3JpZXMgYnkgZXhwcmVzc2lvbiBwcm9maWxpbmcgb2YgZGlmZmVyZW50aWF0aW5nIG1vdXNlIGVtYnJ5b25pYyBzdGVtIGNlbGxzOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMDkzL2RuYXJlcy9kc24wMzU7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NzMtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuY292ZXJpbmcgZWFybHkgcmVzcG9uc2Ugb2YgZ2VuZSByZWd1bGF0b3J5IG5ldHdvcmtzIGluIEVTQ3MgYnkgc3lzdGVtYXRpYyBpbmR1Y3Rpb24gb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3JzOyBDZWxsIFN0ZW0gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDIwLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9QWRkaXNvbi1XZXNsZXkgUHJvZmVzc2lvbmFsLCA7IEdlbmV0aWMgQWxnb3JpdGhtcyBpbiBTZWFyY2gsIE9wdGltaXphdGlvbiwgYW5kIE1hY2hpbmUgTGVhcm5pbmc7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1bW1hcmllcyBvZiBBZmZ5bWV0cml4IEdlbmVDaGlwIHByb2JlIGxldmVsIGRhdGE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPWUxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmF0aW9uLCBub3JtYWxpemF0aW9uLCBhbmQgc3VtbWFyaWVzIG9mIGhpZ2ggZGVuc2l0eSBvbGlnb251Y2xlb3RpZGUgYXJyYXkgcHJvYmUgbGV2ZWwgZGF0YTsgQmlvc3RhdGlzdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjQ5LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9QWZmeW1ldHJpeDsgQWZmeW1ldHJpeCBQb3dlciBUb29scyAoQVBUKSBTb2Z0d2FyZSBQYWNrYWdlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBhcmlzb24gb2Ygbm9ybWFsaXphdGlvbiBtZXRob2RzIGZvciBoaWdoIGRlbnNpdHkgb2xpZ29udWNsZW90aWRlIGFycmF5IGRhdGEgYmFzZWQgb24gdmFyaWFuY2UgYW5kIGJpYXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTg1LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EYXRhYmFzZSByZXNvdXJjZXMgb2YgdGhlIE5hdGlvbmFsIENlbnRlciBmb3IgQmlvdGVjaG5vbG9neTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjgtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsIHNpZ25pZmljYW5jZSBmb3IgZ2Vub21ld2lkZSBzdHVkaWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTk0NDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0VDQ08gJzA3IFByb2NlZWRpbmdzIG9mIHRoZSA5dGggYW5udWFsIGNvbmZlcmVuY2Ugb24gR2VuZXRpYyBhbmQgZXZvbHV0aW9uYXJ5IGNvbXB1dGF0aW9uOyBjaXRhdGlvbl90ZXh0PUEgcGxhdGZvcm0gZm9yIHRoZSBzZWxlY3Rpb24gb2YgZ2VuZXMgaW4gRE5BIG1pY3JvYXJyYXlkYXRhIHVzaW5nIGV2b2x1dGlvbmFyeSBhbGdvcml0aG1zOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fcGFnZXM9NDE1LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgV0VLQSBkYXRhIG1pbmluZyBzb2Z0d2FyZTogYW4gdXBkYXRlOyBTSUdLREQgRXhwbG9yYXRpb25zOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGV4dD1Fc3RpbWF0aW5nIGNvbnRpbnVvdXMgZGlzdHJpYnV0aW9ucyBpbiBCYXllc2lhbiBjbGFzc2lmaWVycywgSW4gUHJvY2VlZGluZ3Mgb2YgdGhlIEVsZXZlbnRoIENvbmZlcmVuY2Ugb24gVW5jZXJ0YWludHkgaW4gQXJ0aWZpY2FsIEludGVsbGlnZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3BhZ2VzPTMzOC00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPU1vcmdhbiBLYXVmbWFubiBQdWJsaXNoZXJzLCA7IEM0LjU6IFByb2dyYW1tcyBmb3IgTWFjaGluZSBMZWFybmluZzsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFuZG9tIEZvcmVzdDsgTWFjaC4gTGVhcm4uOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9NS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zdGFuY2UtYmFzZWQgbGVhcm5pbmcgYWxnb3JpdGhtczsgTWFjaC4gTGVhcm4uOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zNy02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCB0cmFpbmluZyBvZiBzdXBwb3J0IHZlY3RvciBtYWNoaW5lcyB1c2luZyBzZXF1ZW50aWFsIG1pbmltYWwgb3B0aW1pemF0aW9uOyBjaXRhdGlvbl9wdWJsaXNoZXI9TUlUIFByZXNzLCA7IEFkdmFuY2VzIGluIEtlcm5lbCBNZXRob2Rz4oCUU3VwcG9ydCBWZWN0b3IgTGVhcm5pbmc7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkxJQlNWTSDigJMgQSBMaWJyYXJ5IGZvciBTdXBwb3J0IFZlY3RvciBNYWNoaW5lczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wbGVtZW50YXRpb24gb2YgQnJlaW1hbidzIFJhbmRvbSBGb3Jlc3QgTWFjaGluZSBMZWFybmluZyBBbGdvcml0aG0sIEluIEVDRTU5MVEgTWFjaGluZSBMZWFybmluZyBDb25mZXJlbmNlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIEdBLWJhc2VkIGZlYXR1cmUgc2VsZWN0aW9uIGFuZCBwYXJhbWV0ZXJzIG9wdGltaXphdGlvbmZvciBzdXBwb3J0IHZlY3RvciBtYWNoaW5lczsgRXhwZXJ0IFN5c3QuIEFwcGwuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjMxLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGlyZCBJbnRlcm5hdGlvbmFsIENvbmZlcmVuY2Ugb24gR2VuZXRpYyBBbGdvcml0aG1zOyBjaXRhdGlvbl90ZXh0PXBwLjsgY2l0YXRpb25fcHVibGlzaGVyPU1vcmdhbiBLYXVmbWFubiBQdWJsaXNoZXJzLCA7ICBjaXRhdGlvbl9wYWdlcz0yLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVUcmFpbOKAlGFkdmFuY2VkIGdlbmUgc2V0IGVucmljaG1lbnQgYW5hbHlzaXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVcxODYtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQbHVyaU5ldFdvcms6IGFuIGluLXNpbGljbyByZXByZXNlbnRhdGlvbiBvZiB0aGUgbmV0d29yayB1bmRlcmx5aW5nIHBsdXJpcG90ZW5jeSBpbiBtb3VzZSwgYW5kIGl0cyBhcHBsaWNhdGlvbnM7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTUxNjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1zIGJpb2xvZ3kgb2Ygc3RlbSBjZWxsIGZhdGUgYW5kIGNlbGx1bGFyIHJlcHJvZ3JhbW1pbmc7IE5hdC4gUmV2LiBNb2wuIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02NzItODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgbmV0d29ya3MgZGVmaW5lIHBoZW5vdHlwaWMgY2xhc3NlcyBvZiBodW1hbiBzdGVtIGNlbGwgbGluZXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTU7ICBjaXRhdGlvbl9wYWdlcz00MDEtNDA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvU09NRTogYSBjbHVzdGVyaW5nIG1ldGhvZCBmb3IgaWRlbnRpZnlpbmcgZ2VuZSBleHByZXNzaW9uIG1vZHVsZXMgd2l0aG91dCBwcmlvciBrbm93bGVkZ2Ugb2YgY2x1c3RlciBudW1iZXI7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTExNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vcmUgc3luZXJnZXRpYyBjb29wZXJhdGlvbiBvZiBZYW1hbmFrYSBmYWN0b3JzIGluIGluZHVjZWQgcGx1cmlwb3RlbnQgc3RlbSBjZWxscyB0aGFuIGluIGVtYnJ5b25pYyBzdGVtIGNlbGxzOyBDZWxsIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMTI3LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaWduYWxpbmcgcGF0aHdheXMgaW4gY2FuY2VyIGFuZCBlbWJyeW9uaWMgc3RlbSBjZWxsczsgU3RlbSBDZWxsIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTctMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBtZWNoYW5pc21zIGludm9sdmVkIGluIHNlbGYtcmVuZXdhbCBhbmQgcGx1cmlwb3RlbmN5IG9mIGVtYnJ5b25pYyBzdGVtIGNlbGxzOyBKLiBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIxMTsgIGNpdGF0aW9uX3BhZ2VzPTI3OS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXhwbGljaXQgZGVzY3JpcHRpb24gb2YgdGhlIHJlcHJvZHVjaW5nIGtlcm5lbCBoaWxiZXJ0IHNwYWNlcyBvZiBHYXVzc2lhbiBSQkYga2VybmVsczsgSUVFRSBUcmFucy4gSW5mby4gVGhlb3J5OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9NDYzNS00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQSBQcmFjdGljYWwgR3VpZGUgdG8gU3VwcG9ydCBWZWN0b3IgQ2xhc3NpZmljYXRpb247ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIGFuZCBjb21wdXRhdGlvbmFsIGRpc3NlY3Rpb24gb2YgbW9sZWN1bGFyIHNpZ25hdHVyZSBtdWx0aXBsaWNpdHk7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA3OTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LRUdHOiBreW90byBlbmN5Y2xvcGVkaWEgb2YgZ2VuZXMgYW5kIGdlbm9tZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbnMgaW4gRkFNMTM0QiwgZW5jb2RpbmcgYSBuZXdseSBpZGVudGlmaWVkIEdvbGdpIHByb3RlaW4sIGNhdXNlIHNldmVyZSBzZW5zb3J5IGFuZCBhdXRvbm9taWMgbmV1cm9wYXRoeTsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTAzOC9uZy40NjQ7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MTE3OS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxrYWxpbmUgcGhvc3BoYXRhc2UgaW4gdGhlIEdvbGdpIHN1YnN0YW5jZSBvZiBpbnRlc3RpbmFsIGVwaXRoZWxpdW07IGNpdGF0aW9uX3RleHQ9ZG9pOjAwMTQtNDgyNyg1Nik5MDA3MS00OyBFeHAuIENlbGwgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk1NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTYzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uY29nZW5pYyBwcm9wZXJ0aWVzIG9mIGEgbm92ZWwgZ2VuZSBKSy0xIGxvY2F0ZWQgaW4gY2hyb21vc29tZSA1cCBhbmQgaXRzIG92ZXJleHByZXNzaW9uIGluIGh1bWFuIGVzb3BoYWdlYWwgc3F1YW1vdXMgY2VsbCBjYXJjaW5vbWE7IEludC4gSi4gTW9sLiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9OTE1LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBhIG5vdmVsIEpLLTEgZ2VuZSBleHByZXNzaW9uIGluIGJlbmlnbiBhbmQgbWFsaWduYW50IGNvbG9yZWN0YWwgdHVtb3JzOyBWaXJjaG93cyBBcmNoLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NTc7ICBjaXRhdGlvbl9wYWdlcz0xNjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuZCBmdW5jdGlvbmFsIGRpZmZlcmVuY2VzIGJldHdlZW4gbXVsdGlwb3RlbnQgbmV1cmFsIGFuZCBwbHVyaXBvdGVudCBlbWJyeW9uaWMgc3RlbSBjZWxsczsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTA3My9wbmFzLjE4MzQyMDAxMDA7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25faXNzdWU9U3VwcGwuIDEpOyAgY2l0YXRpb25fcGFnZXM9MTE4NjYtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVtYnJ5b3Ryb3BoaWMgZmFjdG9yLTMgZnJvbSBodW1hbiBvdmlkdWN0YWwgY2VsbHMgYWZmZWN0cyB0aGUgbWVzc2VuZ2VyIFJOQSBleHByZXNzaW9uIG9mIG1vdXNlIGJsYXN0b2N5c3Q7IEJpb2wuIFJlcHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0zNzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwdXJvbXljaW4tc2Vuc2l0aXZlIGFtaW5vcGVwdGlkYXNlIFBBTS0xIGlzIHJlcXVpcmVkIGZvciBtZWlvdGljIGV4aXQgYW5kIGFudGVyb3Bvc3RlcmlvciBwb2xhcml0eSBpbiB0aGUgb25lLWNlbGwgQ2Flbm9yaGFiZGl0aXMgZWxlZ2FucyBlbWJyeW87IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjEyNDIvZGV2LjAyNjE1OyBEZXZlbG9wbWVudDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzM7ICBjaXRhdGlvbl9wYWdlcz00MjgxLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SaW5nMUIgaXMgY3J1Y2lhbCBmb3IgdGhlIHJlZ3VsYXRpb24gb2YgZGV2ZWxvcG1lbnRhbCBjb250cm9sIGdlbmVzIGFuZCBQUkMxIHByb3RlaW5zIGJ1dCBub3QgWCBpbmFjdGl2YXRpb24gaW4gZW1icnlvbmljIGNlbGxzOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMDgzL2pjYi4yMDA2MTIxMjc7IEouIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc4OyAgY2l0YXRpb25fcGFnZXM9MjE5LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBodW1hbiBhbmQgbW91c2UgcmVwcm9ncmFtbWluZyBvZiBzb21hdGljIGNlbGxzIHRvIGluZHVjZWQgcGx1cmlwb3RlbnQgc3RlbSBjZWxscy4gV2hhdCBpcyBpbiB0aGUgcGxhdGU/OyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMzcxL2pvdXJuYWwucG9uZS4wMDEyNjY0OyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgUE9VIGRpbWVyIGNvbmZpZ3VyYXRpb24gbWVkaWF0ZXMgYW50YWdvbmlzdGljIGNvbnRyb2wgb2YgYW4gb3N0ZW9wb250aW4gcHJlaW1wbGFudGF0aW9uIGVuaGFuY2VyIGJ5IE9jdC00IGFuZCBTb3gtMjsgR2VuZXMgRGV2LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTIwNzMtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhemwgZnVuY3Rpb25zIGluIG1haW50ZW5hbmNlIG9mIHBsdXJpcG90ZW5jeSBhbmQgZ2VuZXRpYyBhbmQgZXBpZ2VuZXRpYyBwcm9ncmFtcyBvZiBkaWZmZXJlbnRpYXRpb24gaW4gbW91c2UgcHJpbW9yZGlhbCBnZXJtIGNlbGxzIGluIHZpdm8gYW5kIGluIHZpdHJvOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMzcxL2pvdXJuYWwucG9uZS4wMDA1NjU0OyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTU2NTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZXJhcmNoaWNhbCBuZXR3b3JrIGNvbnRyb2xzIHByb3RlaW4gdHJhbnNsYXRpb24gZHVyaW5nIG11cmluZSBlbWJyeW9uaWMgc3RlbSBjZWxsIHNlbGYtcmVuZXdhbCBhbmQgZGlmZmVyZW50aWF0aW9uOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMDE2L2ouc3RlbS4yMDA4LjAzLjAxMzsgQ2VsbCBTdGVtIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTQ0OC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmcgcmVndWxhdGVzIG1vdXNlIGVtYnJ5b25pYyBzdGVtIGNlbGwgcGx1cmlwb3RlbmN5IGFuZCBkaWZmZXJlbnRpYXRpb247IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjEwNzMvcG5hcy4wOTEyMjYwMTA3OyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTEwNTE0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZSByZWd1bGF0b3J5IG5ldHdvcmsgaW4gbW91c2UgZW1icnlvbmljIHN0ZW0gY2VsbHM7IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjEwNzMvcG5hcy4wNzAxMDE0MTA0OyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTE2NDM4LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HREYzLCBhIEJNUCBpbmhpYml0b3IsIHJlZ3VsYXRlcyBjZWxsIGZhdGUgaW4gc3RlbSBjZWxscyBhbmQgZWFybHkgZW1icnlvczsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTI0Mi9kZXYuMDIxOTI7IERldmVsb3BtZW50OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTIwOS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gU1RFTExBUiwgTkFOT0csIGFuZCBHREYzIGdlbmVzIGFyZSBleHByZXNzZWQgaW4gcGx1cmlwb3RlbnQgY2VsbHMgYW5kIG1hcCB0byBjaHJvbW9zb21lIDEycDEzLCBhIGhvdHNwb3QgZm9yIHRlcmF0b2NhcmNpbm9tYTsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTYzNC9zdGVtY2VsbHMuMjItMi0xNjk7IFN0ZW0gQ2VsbHM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjktNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUItTVlCIGlzIGVzc2VudGlhbCBmb3Igbm9ybWFsIGNlbGwgY3ljbGUgcHJvZ3Jlc3Npb24gYW5kIGNocm9tb3NvbWFsIHN0YWJpbGl0eSBvZiBlbWJyeW9uaWMgc3RlbSBjZWxsczsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTM3MS9qb3VybmFsLnBvbmUuMDAwMjQ3ODsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWUyNDc4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiB0aGUgcGx1cmlwb3RlbmN5IG1hcmtlciBSZXgtMSBieSBOYW5vZyBhbmQgU294MjsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTA3NC9qYmMuTTYwMTgxMTIwMDsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjgxOyAgY2l0YXRpb25fcGFnZXM9MjMzMTktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGhvbW9sb2d5IGFncmVlbWVudCBzZWFyY2g6IGFuIGVmZmVjdGl2ZSBjb21iaW5hdGlvbiBvZiBob21vbG9neS1zZWFyY2ggbWV0aG9kczsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTA3My9wbmFzLjA0MDU2MTIxMDE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MTM4MTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2Vuc2l0aXZlIGVuenltYXRpYyBxdWFudGlmaWNhdGlvbiBvZiA1LWh5ZHJveHltZXRoeWxjeXRvc2luZSBpbiBnZW5vbWljIEROQTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9ZTE4MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnZlcnNpb24gb2YgNS1tZXRoeWxjeXRvc2luZSB0byA1LWh5ZHJveHltZXRoeWxjeXRvc2luZSBpbiBtYW1tYWxpYW4gRE5BIGJ5IE1MTCBwYXJ0bmVyIFRFVDE7IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjExMjYvc2NpZW5jZS4xMTcwMTE2OyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTkzMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIFRldCBwcm90ZWlucyBpbiA1bUMgdG8gNWhtQyBjb252ZXJzaW9uLCBFUy1jZWxsIHNlbGYtcmVuZXdhbCBhbmQgaW5uZXIgY2VsbCBtYXNzIHNwZWNpZmljYXRpb247IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjEwMzgvbmF0dXJlMDkzMDM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjY7ICBjaXRhdGlvbl9wYWdlcz0xMTI5LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EdWFsIGZ1bmN0aW9ucyBvZiBUZXQxIGluIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uIGluIG1vdXNlIGVtYnJ5b25pYyBzdGVtIGNlbGxzOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMDM4L25hdHVyZTA5OTM0OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWMgcmVndWxhdGlvbiBvZiA1LWh5ZHJveHltZXRoeWxjeXRvc2luZSBpbiBtb3VzZSBFUyBjZWxscyBhbmQgZHVyaW5nIGRpZmZlcmVudGlhdGlvbjsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTAzOC9uYXR1cmUxMDAwODsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXJlY3QgdGFyZ2V0cyBvZiBLbGY1IHRyYW5zY3JpcHRpb24gZmFjdG9yIGNvbnRyaWJ1dGUgdG8gdGhlIG1haW50ZW5hbmNlIG9mIG1vdXNlIGVtYnJ5b25pYyBzdGVtIGNlbGwgdW5kaWZmZXJlbnRpYXRlZCBzdGF0ZTsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTE4Ni8xNzQxLTcwMDctOC0xMjg7IEJNQyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U21hZDQtZGVwZW5kZW50IHBhdGh3YXlzIGNvbnRyb2wgYmFzZW1lbnQgbWVtYnJhbmUgZGVwb3NpdGlvbiBhbmQgZW5kb2Rlcm1hbCBjZWxsIG1pZ3JhdGlvbiBhdCBlYXJseSBzdGFnZXMgb2YgbW91c2UgZGV2ZWxvcG1lbnQ7IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjExODYvMTQ3MS0yMTNYLTktNTQ7IEJNQyBEZXYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz01NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gcHJvZmlsaW5nIGluIHRyYW5zZ2VuaWMgRlZCL04gZW1icnlvbmljIHN0ZW0gY2VsbHMgb3ZlcmV4cHJlc3NpbmcgU1RBVDM7IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjExODYvMTQ3MS0yMTNYLTgtNTc7IEJNQyBEZXYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz01NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluY29tcGxldGUgcmVhY3RpdmF0aW9uIG9mIE9jdDQtcmVsYXRlZCBnZW5lcyBpbiBtb3VzZSBlbWJyeW9zIGNsb25lZCBmcm9tIHNvbWF0aWMgbnVjbGVpOyBEZXZlbG9wbWVudDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzA7ICBjaXRhdGlvbl9wYWdlcz0xNjczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvY2hvbmRyaWFsIEROQSByZXBsaWNhdGlvbiBkdXJpbmcgZGlmZmVyZW50aWF0aW9uIG9mIG11cmluZSBlbWJyeW9uaWMgc3RlbSBjZWxsczsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTI0Mi9qY3MuMDE2OTcyOyBKLiBDZWxsIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9NDAyNS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3JhZGVkIFNtYWQyLzMgYWN0aXZhdGlvbiBpcyBjb252ZXJ0ZWQgZGlyZWN0bHkgaW50byBsZXZlbHMgb2YgdGFyZ2V0IGdlbmUgZXhwcmVzc2lvbiBpbiBlbWJyeW9uaWMgc3RlbSBjZWxsczsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTM3MS9qb3VybmFsLnBvbmUuMDAwNDI2ODsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU0MjY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2Fub3B5MSwgYSBub3ZlbCByZWd1bGF0b3Igb2YgRkdGIHNpZ25hbGluZyBhcm91bmQgdGhlIG1pZGJyYWluLWhpbmRicmFpbiBib3VuZGFyeSBpbiB6ZWJyYWZpc2g7IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjEwMTYvai5jdWIuMjAwNi4wMS4wNTU7IEN1cnIuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NDIxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBub24tY29kaW5nIHJlZ3VsYXRvcnkgc2VxdWVuY2VzIGludm9sdmVkIGluIHRoZSBkZXZlbG9wbWVudGFsIHByb2Nlc3M6IHJlZmxlY3Rpb24gb2YgZGlmZmVyZW50aWFsIGVtcGxveW1lbnQgb2YgcGFyYWxvZ291cyBnZW5lcyBhcyBoaWdobGlnaHRlZCBieSBTb3gyIGFuZCBncm91cCBCMSBTb3ggZ2VuZXM7IFByb2MuIEpwbiBBY2FkLiBTZXIuIEIgUGh5cy4gQmlvbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTU1LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3VzZSBnZXJtbGluZSByZXN0cmljdGlvbiBvZiBPY3Q0IGV4cHJlc3Npb24gYnkgZ2VybSBjZWxsIG51Y2xlYXIgZmFjdG9yOyBEZXYuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTM3Ny04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGhvbWVvcHJvdGVpbiBOYW5vZyBpcyByZXF1aXJlZCBmb3IgbWFpbnRlbmFuY2Ugb2YgcGx1cmlwb3RlbmN5IGluIG1vdXNlIGVwaWJsYXN0IGFuZCBFUyBjZWxsczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz02MzEtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZHVjdGlvbiBvZiBwbHVyaXBvdGVudCBzdGVtIGNlbGxzIGZyb20gbW91c2UgZW1icnlvbmljIGFuZCBhZHVsdCBmaWJyb2JsYXN0IGN1bHR1cmVzIGJ5IGRlZmluZWQgZmFjdG9yczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz02NjMtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0ZW0gY2VsbHMgYW5kIGdlcm0gY2VsbHM6IG1pY3JvUk5BIGFuZCBnZW5lIGV4cHJlc3Npb24gc2lnbmF0dXJlczsgSGlzdG9sLiBIaXN0b3BhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz01MDUtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXhwcmVzc2lvbiBwcm9maWxlcyBkdXJpbmcgZWFybHkgZGlmZmVyZW50aWF0aW9uIG9mIG1vdXNlIGVtYnJ5b25pYyBzdGVtIGNlbGxzOyBCTUMgRGV2LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YWJsZSBpc290b3BlIGxhYmVsaW5nIGJ5IGFtaW5vIGFjaWRzIGluIGNlbGwgY3VsdHVyZSAoU0lMQUMpIGFuZCBwcm90ZW9tZSBxdWFudGl0YXRpb24gb2YgbW91c2UgZW1icnlvbmljIHN0ZW0gY2VsbHMgdG8gYSBkZXB0aCBvZiA1MTExIHByb3RlaW5zOyBNb2wgQ2VsbCBQcm90ZW9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz02NzItODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3Rlb21pYyBhbmFseXNpcyBvZiBtb3VzZSBvb2N5dGVzIHJldmVhbHMgMjggY2FuZGlkYXRlIGZhY3RvcnMgb2YgdGhlIOKAnHJlcHJvZ3JhbW1vbWXigJ07IEouIFByb3Rlb21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yMTQwLTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21tZW50IG9uICZxdW90OydTdGVtbmVzcyc6IHRyYW5zY3JpcHRpb25hbCBwcm9maWxpbmcgb2YgZW1icnlvbmljIGFuZCBhZHVsdCBzdGVtIGNlbGxzJnF1b3Q7IGFuZCAmcXVvdDthIHN0ZW0gY2VsbCBtb2xlY3VsYXIgc2lnbmF0dXJlJnF1b3Q7OyBjaXRhdGlvbl90ZXh0PWF1dGhvciByZXBseSAzOTMsIGRvaToxMC4xMTI2L3NjaWVuY2UuMTA4NjM4NDsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDI7ICBjaXRhdGlvbl9wYWdlcz0zOTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2R1bGUgbWFwIG9mIHN0ZW0gY2VsbCBnZW5lcyBndWlkZXMgY3JlYXRpb24gb2YgZXBpdGhlbGlhbCBjYW5jZXIgc3RlbSBjZWxsczsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTAxNi9qLnN0ZW0uMjAwOC4wMi4wMDk7IENlbGwgU3RlbSBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zMzMtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGFyZW50bHkgbG93IHJlcHJvZHVjaWJpbGl0eSBvZiB0cnVlIGRpZmZlcmVudGlhbCBleHByZXNzaW9uIGRpc2NvdmVyaWVzIGluIG1pY3JvYXJyYXkgc3R1ZGllczsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTA5My9iaW9pbmZvcm1hdGljcy9idG4zNjU7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjA1Ny02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0cmFjdGluZyBjb25zaXN0ZW50IGtub3dsZWRnZSBmcm9tIGhpZ2hseSBpbmNvbnNpc3RlbnQgY2FuY2VyIGdlbmUgZGF0YSBzb3VyY2VzOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMTg2LzE0NzEtMjEwNS0xMS03NjsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIHRyYW5zY3JpcHRpb24gcHJvZ3JhbXMgZHVyaW5nIEVTIGNlbGwgZGlmZmVyZW50aWF0aW9uIHRvd2FyZHMgbWVzb2Rlcm0gaW4gc2VydW0gdmVyc3VzIHNlcnVtLWZyZWVCTVA0IGN1bHR1cmU7IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjExODYvMTQ3MS0yMTY0LTgtMzY1OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgcHJvZmlsaW5nIGZvciBkZXRlcm1pbmluZyBwbHVyaXBvdGVudCBnZW5lcyBpbiBhIHRpbWUgY291cnNlIG1pY3JvYXJyYXkgZXhwZXJpbWVudDsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTA5My9iaW9zdGF0aXN0aWNzL2t4bjAxNzsgQmlvc3RhdGlzdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTgwLTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGNsdXN0ZXJpbmcgYW5kIGxpbmVhZ2UgbWFya2VyczogaW5zaWdodHMgaW50byBjZWxsdWxhciBkaWZmZXJlbnRpYXRpb24gYW5kIGdlbmUgZnVuY3Rpb24gZnJvbSBsYXJnZS1zY2FsZSBtaWNyb2FycmF5IHN0dWRpZXMgb2YgcHVyaWZpZWQgcHJpbWFyeSBjZWxsIHBvcHVsYXRpb25zOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMDE2L2oueWdlbm8uMjAxMC4wMy4wMDI7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MzI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIGh5YnJpZCBiaW9tYXNzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGJ5IHNlbGVjdGVkIHBhcmVudGFsIFNOUCBhbmQgbWV0YWJvbGljIG1hcmtlcnM7IGNpdGF0aW9uX3RleHQ9ZG9pOjEwLjEwMDcvczAwMTIyLTAwOS0xMTkxLTI7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9MjM5LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QUklDQUknMDYgUHJvY2VlZGluZ3Mgb2YgdGhlIDl0aCBQYWNpZmljIFJpbSBJbnRlcm5hdGlvbmFsIENvbmZlcmVuY2Ugb24gQXJ0aWZpY2lhbCBJbnRlbGxpZ2VuY2U7IGNpdGF0aW9uX3RleHQ9UGF0dGVybiBjbGFzc2lmaWNhdGlvbiBpbiBETkEgbWljcm9hcnJheSBkYXRhIG9mIG11bHRpcGxlIHR1bW9yIHR5cGVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fcGFnZXM9MTAzNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0VDQ08gJzA3IFByb2NlZWRpbmdzIG9mIHRoZSA5dGggYW5udWFsIENvbmZlcmVuY2Ugb24gR2VuZXRpYyBhbmQgRXZvbHV0aW9uYXJ5IENvbXB1dGF0aW9uOyBjaXRhdGlvbl90ZXh0PUV2b2x1dGlvbmFyeSBzZWxlY3Rpb24gb2YgbWluaW11bSBudW1iZXIgb2YgZmVhdHVyZXMgZm9yIGNsYXNzaWZpY2F0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbiBkYXRhIHVzaW5nIGdlbmV0aWMgYWxnb3JpdGhtczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3BhZ2VzPTQwMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3RpdmUgZ2VuZSBzZWxlY3Rpb24gbWV0aG9kIHVzaW5nIGJheWVzaWFuIGRpc2NyaW1pbmFudCBiYXNlZCBjcml0ZXJpb24gYW5kIGdlbmV0aWMgYWxnb3JpdGhtczsgSi4gU2lnbmFsIFByb2NlLiBTeXN0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTI5My0zMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFnZW50LWJhc2VkIGh5YnJpZCBzeXN0ZW0gZm9yIG1pY3JvYXJyYXkgZGF0YSBhbmFseXNpczsgSW50ZWxsLiBTeXN0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTUzLTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90ZXh0PUEgZmlsdGVyLWJhc2VkIGV2b2x1dGlvbmFyeSBhcHByb2FjaCBmb3Igc2VsZWN0aW5nIGZlYXR1cmVzIGluIGhpZ2gtZGltZW5zaW9uYWwgbWljcm8tYXJyYXkgZGF0YSwgSW4gNnRoIEludGVybmF0aW9uYWwgQ29uZmVyZW5jZSBvbiBJbnRlbGxpZ2VudCBJbmZvcm1hdGlvbiBQcm9jZXNzaW5nOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fcGFnZXM9Mjk3LTMwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzQvMjMzLzEzNTEwNDEvZHNyMDE2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBQbHVyaXBvdGVudCBzdGVtIGNlbGxzIGFyZSBhYmxlIHRvIHNlbGYtcmVuZXcsIGFuZCB0byBkaWZmZXJlbnRpYXRlIGludG8gYWxsIGFkdWx0IGNlbGwgdHlwZXMuIE1hbnkgc3R1ZGllcyByZXBvcnQgZGF0YSBkZXNjcmliaW5nIHRoZXNlIGNlbGxzLCBhbmQgYyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTAxODkyJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZNqzEWxgyOJH5GCTrA41wmiIVGQI" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f233%2f501892" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f233%2f501892" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19820" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&amp;Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/4">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7741238">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741238">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741242">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741242">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741251">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741251">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741316">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741316">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741347">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741347">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741370">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741370">5. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741372">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741372">Authors' contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741374">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741374">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741376">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741376">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741378">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741378">Acknowledgement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7741380">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7741380">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/4/221/501057">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/4/253/501301">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Learning Biomarkers of Pluripotent Stem Cells in Mouse
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lena Scheubert</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Computer Science, University of Osnabrück</div>, <div class="addr-line">Albrechtstr. 28, 49076 Osnabrück</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lena+Scheubert">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Scheubert L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Scheubert L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lena Scheubert</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rainer Schmidt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock</div>, <div class="addr-line">Ernst-Heydemann-Str. 8, 18057, Rostock</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rainer+Schmidt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schmidt R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schmidt R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rainer Schmidt</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dirk Repsilber</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Leibniz Institute for Farm Animal Biology (FBN Dummerstorf)</div>, <div class="addr-line">Wilhelm-Stahl Allee 2, 18196 Dummerstorf</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dirk+Repsilber">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Repsilber D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Repsilber D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dirk Repsilber</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitja Luštrek</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock</div>, <div class="addr-line">Ernst-Heydemann-Str. 8, 18057, Rostock</div>, <div class="country">Germany</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Intelligent Systems</div>, <div class="insititution">Jožef Stefan Institute</div>, <div class="addr-line">Jamova cesta 39, 1000 Ljubljana</div>, <div class="country">Slovenia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +386 1-4773380. Fax. +386 1-4773131. Email: <a href="mailto:mitja.lustrek@ijs.si" target="_blank">mitja.lustrek@ijs.si</a> (M.L.); Tel. +49 381-4947360. Fax. +49 381-4947203. Email: <a href="mailto:fuellen@uni-rostock.de" target="_blank">fuellen@uni-rostock.de</a> (G.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitja+Lu%c5%a1trek">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Luštrek M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Luštrek M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mitja Luštrek</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Georg Fuellen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Institute for Biostatistics and Informatics in Medicine and Ageing Research, University of Rostock</div>, <div class="addr-line">Ernst-Heydemann-Str. 8, 18057, Rostock</div>, <div class="country">Germany</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +386 1-4773380. Fax. +386 1-4773131. Email: <a href="mailto:mitja.lustrek@ijs.si" target="_blank">mitja.lustrek@ijs.si</a> (M.L.); Tel. +49 381-4947360. Fax. +49 381-4947203. Email: <a href="mailto:fuellen@uni-rostock.de" target="_blank">fuellen@uni-rostock.de</a> (G.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Georg+Fuellen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fuellen G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fuellen G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Georg Fuellen</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Minoru Ko</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 233–251, <a href='https://doi.org/10.1093/dnares/dsr016'>https://doi.org/10.1093/dnares/dsr016</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 July 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 March 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 May 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="501892" href="/dnaresearch/article-pdf/18/4/233/1351041/dsr016.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Lena Scheubert, Rainer Schmidt, Dirk Repsilber, Mitja Luštrek, Georg Fuellen;  Learning Biomarkers of Pluripotent Stem Cells in Mouse, <em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 233–251, <a href='https://doi.org/10.1093/dnares/dsr016'>https://doi.org/10.1093/dnares/dsr016</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/501892?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/501892?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/501892?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/501892?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/501892?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/501892?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/501892?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Learning%20Biomarkers%20of%20Pluripotent%20Stem%20Cells%20in%20Mouse&amp;publicationDate=2011-07-26&amp;volumeNum=18&amp;issueNum=4&amp;author=Scheubert%2C%20Lena%3B%20Schmidt%2C%20Rainer&amp;startPage=233&amp;endPage=251&amp;contentId=10.1093%2Fdnares%2Fdsr016&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f233%2f501892" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7741238 id="7741238" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Pluripotent stem cells are able to self-renew, and to differentiate into all adult cell types. Many studies report data describing these cells, and characterize them in molecular terms. Machine learning yields classifiers that can accurately identify pluripotent stem cells, but there is a lack of studies yielding minimal sets of best biomarkers (genes/features). We assembled gene expression data of pluripotent stem cells and non-pluripotent cells from the mouse. After normalization and filtering, we applied machine learning, classifying samples into pluripotent and non-pluripotent with high cross-validated accuracy. Furthermore, to identify minimal sets of best biomarkers, we used three methods: information gain, random forests and a wrapper of genetic algorithm and support vector machine (GA/SVM). We demonstrate that the GA/SVM biomarkers work best in combination with each other; pathway and enrichment analyses show that they cover the widest variety of processes implicated in pluripotency. The GA/SVM wrapper yields best biomarkers, no matter which classification method is used. The consensus best biomarker based on the three methods is <em>Tet1</em>, implicated in pluripotency just recently. The best biomarker based on the GA/SVM wrapper approach alone is <em>Fam134b</em>, possibly a missing link between pluripotency and some standard surface markers of unknown function processed by the Golgi apparatus.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="pluripotency">pluripotency</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;machine learning&quot;">machine learning</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;feature selection&quot;">feature selection</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic algorithm&quot;">genetic algorithm</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;support vector machine&quot;">support vector machine</a></div>                            <h2 scrollto-destination=7741242 id="7741242" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Classifying high-level phenotypes based on high-throughput gene-level data<sup><a reveal-id="DSR016C1" data-open="DSR016C1" class="link link-ref link-reveal xref-bibr">1</a></sup> is a fundamental task in bioinformatics, and analysing corresponding sets of important features improves our understanding of the genotype–phenotype map,<sup><a reveal-id="DSR016C2" data-open="DSR016C2" class="link link-ref link-reveal xref-bibr">2</a></sup> delivering basic insights into the biology underlying a particular phenotype. For the cellular phenotype commonly called ‘pluripotent stem cell’<sup><a reveal-id="DSR016C3" data-open="DSR016C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and its more heterogeneous counterpart ‘differentiated’ or ‘non-pluripotent stem’ cell, we set out to collect data in the form of gene expression data (microarrays) from the GEO (Gene Expression Omnibus database<sup><a reveal-id="DSR016C4" data-open="DSR016C4" class="link link-ref link-reveal xref-bibr">4</a></sup>). Gene expression data are among the most abundant molecular data, and they are still very close to the true genotype of the (static) genome; they may inform us about which genes are responsible for the phenotype we wish to understand. This information comes in several ways. It is of interest:
A univariate statistical testing approach (often involving normalization/regularization) together with the inspection of ‘fold change’ is a standard approach to answer <em>question a</em>.<sup><a reveal-id="DSR016C5  DSR016C7" data-open="DSR016C5  DSR016C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> Answers to <em>question b</em> shall yield a comprehensive description of the molecular basis of pluripotency, based on machine learning and feature selection approaches. Answers to <em>question c</em> shall highlight the ‘tips of the iceberg’, and are useful to define small sets of best biomarkers for pluripotency; such ‘minimal-best’ approaches to machine learning have gained popularity in recent years, in particular in search for cancer biomarkers.<sup><a reveal-id="DSR016C8  DSR016C13" data-open="DSR016C8  DSR016C13" class="link link-ref link-reveal xref-bibr">8–13</a></sup></p><ul class="list-simple"><li><p>(a) Which genes are differentially expressed, i.e. expressed more or less strongly in the pluripotent stem cell state, compared to the differentiated one?</p></li><li><p>(b)  Which sets of genes enable the best distinction of the pluripotent stem cell state from the differentiated one, considering their (differential) expression?</p></li><li><p>(c)  Which small sets of genes still enable a good distinction of the two states?</p></li></ul><p>In the past years, the difference between pluripotent stem cells (e.g. embryonic stem/ES cells) and differentiated (e.g. fibroblast) cells has triggered a great deal of interest; it is at the centre of basic research into developmental biology.<sup><a reveal-id="DSR016C3" data-open="DSR016C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR016C14" data-open="DSR016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> At the same time, there are a multitude of potential applications in regenerative medicine and beyond.<sup><a reveal-id="DSR016C15" data-open="DSR016C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Moreover, exciting progress has been made in the <em>in vitro</em> control of basic cellular phenotypes.<sup><a reveal-id="DSR016C16" data-open="DSR016C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR016C17" data-open="DSR016C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Finally, computational methods such as hierarchical clustering and principal component analysis have been used to investigate pluripotency and differentiation in systematic ways, often based on gene expression data.<sup><a reveal-id="DSR016C18  DSR016C20" data-open="DSR016C18  DSR016C20" class="link link-ref link-reveal xref-bibr">18–20</a></sup> At the end of the <em>Discussion</em> section, further related work will be discussed.</p><p>In this paper, we start with a large normalized data set of gene expression experiments obtained from the mouse, using the answer to <em>question a</em> to filter out genes that are not informative. We then tackle <em>question b</em> taking a machine learning approach (including appropriate cross-validation), and we use various machine learning methods resulting in high-accuracy classifiers based on gene expression signatures. Finally, we look into <em>question c</em> by using feature selection methods such as genetic algorithms (GAs)<sup><a reveal-id="DSR016C21" data-open="DSR016C21" class="link link-ref link-reveal xref-bibr">21</a></sup> to obtain classifiers of slightly lower accuracy, working with few features (genes). We evaluate the feature lists obtained to assess their biological plausibility by enrichment analyses. Finally, we discuss the value we may put into the ‘newly discovered’ genes that are supposedly involved in pluripotency.</p>                            <h2 scrollto-destination=7741251 id="7741251" class="section-title" data-legacy-id=s2>2. Methods</h2>
                            <h3 scrollto-destination=7741253 id="7741253" class="section-title" data-legacy-id=s2a>2.1. Gene expression data</h3>
<p>We obtained gene expression data from the GEO database,<sup><a reveal-id="DSR016C4" data-open="DSR016C4" class="link link-ref link-reveal xref-bibr">4</a></sup> taking samples from experiments (data series) related to pluripotency in the mouse, and aiming for a large data set, correctness in class labels and variety in phenotype. As GEO data series usually consist of no more than 5–20 samples, we collected gene expression data from many different GEO series. To ensure straightforward comparability and easy merging of data based on an identical set of probes, we decided to use only data series from the Affymetrix mouse 430.2 oligonucleotide chip (GEO: GPL1261), which is the most popular platform available, containing 45 101 probe sets. To consider a GEO series, it had to contain at least one sample that we could label pluripotent; usually, the GEO series consisted of a mixture of pluripotent and non-pluripotent samples. Based on the sample descriptions, we manually identified samples as pluripotent or non-pluripotent and labelled them as positive (pluripotent) or negative (non-pluripotent). If we were not sure about the label of a sample, we dismissed it rather than risk adding an incorrectly labelled sample. In this paper, pluripotency always refers to stem cells; we do not consider data from cells like zygotes that are pluripotent, but do not have the ability to self-renew. As different microarray platforms have different approaches on how to sample the transcriptome, and about how to represent the concept of ‘genes', the details may differ between different platforms. However, they should converge on the more generic levels of annotation, in particular with respect to most of the UniGene gene symbol annotations of the genes in our analysis; in our analysis, we study mouse genes.</p><p>The positively labelled samples are gene expression data of pluripotent stem cells and the inner cell mass (ICM) of the embryo, whereas the negatively labelled samples arise from all sorts of differentiated cells/tissues. We took care to sample a variety of types of pluripotency, including ‘late stages’ up to embryonic day 3.5, pluripotent germline stem cells and induced pluripotent stem (iPS) cells, and of differentiated phenotypes, including ‘early stages’ from embryoid bodies of day 5 onwards, germline stem cells and partially reprogrammed iPS. More specifically, we included samples described as pluripotent from embryonic stem cells (GSE4309, GSE10806, GSE10871 and others), from the ICM up to embryonic day 3.5 (GSE4309), from germline pluripotent stem cells (GSE11274) and from iPS cells (GSE10806, GSE10871 and others). We included samples described as differentiated from embryoid bodies of day 5 (GSE9563) and day 10 (GSE3653), from partially reprogrammed iPS cells (GSE10871), from germline stem cells (GSE10610, GSE11274) and from tissues including testis, ovary and foetus (GSE9954), but also a wide variety of other organs (GSE9954). The final data set contains 286 samples from 25 GEO data series; 146 labelled as pluripotent and 140 labelled as non-pluripotent (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1)</a>.</p>                            <h3 scrollto-destination=7741260 id="7741260" class="section-title" data-legacy-id=s2b>2.2. Data preprocessing</h3>
<p>To summarize the probe sets from the Affymetrix gene expression arrays, we used the robust multi-array average method<sup><a reveal-id="DSR016C22" data-open="DSR016C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSR016C23" data-open="DSR016C23" class="link link-ref link-reveal xref-bibr">23</a></sup> as implemented in the Affymetrix Power Tools.<sup><a reveal-id="DSR016C24" data-open="DSR016C24" class="link link-ref link-reveal xref-bibr">24</a></sup> First, background adjustment was performed using unmodified perfect match intensities. Then, the intensities were quantile normalized.<sup><a reveal-id="DSR016C25" data-open="DSR016C25" class="link link-ref link-reveal xref-bibr">25</a></sup> As quantiﬁcation method we used median polish.<sup><a reveal-id="DSR016C22" data-open="DSR016C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSR016C23" data-open="DSR016C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The resulting expression values are logarithmized on the log2 scale. We then combined all Affymetrix probe set identifiers that correspond to the same gene symbol from the UniGene record<sup><a reveal-id="DSR016C26" data-open="DSR016C26" class="link link-ref link-reveal xref-bibr">26</a></sup> by calculating the mean value. This way, we obtained the expression values of 20 668 genes. This data set was the starting point for filtering, feature selection and classification, all in the framework of 3-fold cross-validation. Thus, the data set was randomly split into three subsets (folds), to set up a 3-fold cross-validation process. Two-thirds of the data were used to perform filtering, feature selection and finally training of classifiers. The classifiers were then tested on the remaining one-third of the data. This was repeated three times, using different folds for training and testing each time.</p><p>As we wanted to identify genes that affect pluripotency, we preferred those having similar expression values within the pluripotent or the non-pluripotent samples and different expression values between these two groups—i.e. differentially expressed genes. Hence, we filtered the genes by applying a two-sample <em>t</em>-test for samples with unequal variances, testing the difference in mean expression of the genes in the respective training set and corrected resulting <em>P</em>-values based on the concept of false discovery rate.<sup><a reveal-id="DSR016C27" data-open="DSR016C27" class="link link-ref link-reveal xref-bibr">27</a></sup> We dismissed all genes with a <em>q</em>-value (corrected <em>P</em>-value) larger than 0.1. Depending on the training set, we obtained lists of around 16 000 genes. These were sorted by their fold changes determined as log <em>FC<sub>i</sub></em> = |<em>X<sub>i</sub></em> – <em>Y<sub>i</sub></em>|, where <em>X<sub>i</sub></em> is the mean of the gene expression values of gene <em>i</em> for pluripotent samples, and <em>Y<sub>i</sub></em> is the mean of the gene expression values of gene <em>i</em> for non-pluripotent samples, both on the log-scale. The first 5000 and the first 1000 genes of these lists formed our data sets for classification.</p><p>As described, filtering, feature selection and training of classifiers were always performed on two-thirds of the data and testing on the remaining third. Only such strict separation of training and test data (which is unfortunately not standard practice, as noted by Rocha <em>et al</em>.<sup><a reveal-id="DSR016C28" data-open="DSR016C28" class="link link-ref link-reveal xref-bibr">28</a></sup>) can ensure that results are not overly optimistic. On each of the three folds, the GA was run 200 times. However, the whole data set was used for calculating feature importance (Fig. <a reveal-id="DSR016F1" data-open="DSR016F1" class="link link-reveal link-table xref-fig">1</a>), for the list of biomarkers we found (Table <a reveal-id="DSR016TB2" data-open="DSR016TB2" class="link link-reveal link-table xref-fig">2</a>) and for the enrichment analyses (Tables <a reveal-id="DSR016TB3" data-open="DSR016TB3" class="link link-reveal link-table xref-fig">3</a> and <a reveal-id="DSR016TB4" data-open="DSR016TB4" class="link link-reveal link-table xref-fig">4</a>), since these parts of the work did not involve training and testing. In this case, the GA was run 500 times. </p>                        <a id="7741268" scrollto-destination="7741268"></a>
<div id="DSR016F1" data-id="DSR016F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01601.jpeg?Expires=2147483647&Signature=xic5wC2m7l69wHXHbeqFj5wUAub8~S8djP1nyUMdZcdRav6vH5qZMyXlWEuKR3EYRpZT7t7z9nphy3tXWAA6h3kYNwNCyJgtAabakQuCHT9crvTXa2bWmbnfnaKLtuKS5uh7rF1qCCdY5HFJVc0g6bjQF1mkXXWFv7TfFfbn~TvlUfS81P4droXOg6cQe1SSj24vpcgEFaiRfmFGdChZERCZ6AwsioTJJoWrVFrqCCF1pUlRCPDiLj4Pba0Z-t11StVtgNl4497GRp-Q8COdEFeEf4VoR6sW6VoQf-YrSOyiTRZya8zaC3vm39l8ijR96cE6qNc7-VUtjQIRGZSsvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Feature-importance distribution. Each feature (gene) from the 1000-gene data set was assigned an importance score by each of the three feature selection methods. The scores were scaled to the [0, 1] interval. The features were then sorted by their importance, separately for each method, and their importance was plotted." path-from-xml="dsr01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741268/dsr01601.jpeg" path-from-xml="dsr01601.jpeg" target="_blank">View large</a><a section="7741268" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01601.jpeg?Expires=2147483647&Signature=ugX0oEBR0moEzzpKyWq3thIQwKGKyWMKsiBgxjb7JLGPpFAeLGDRG7VyKO7YkZzWUkk9s7J7N7VbYMRN7aAYKUrxciE9Recu5dVlzh0vXDsisdfqOdy-nFh0fss2herI6~0sScy9vmy6O~qrs2cXGnW2YuKxrxQhSVhES2Tk~gwEAKYeeMNzC1s8QQjZ-3gtP3muYv6EWVFPaNSpfUQd1ezMVE1JyY76DAU2mS0I9PEeG~dR06w6LVDrQq5p2cH5MRWUQwz-QiM1WjkEfj64hDwAmp6Ysp2tqlqwQIp~lk6NL5Czffky0zbo8AQRNuyyh6Fc0ivP4jcaI6pDUsLJ8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741268&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Feature-importance distribution. Each feature (gene) from the 1000-gene data set was assigned an importance score by each of the three feature selection methods. The scores were scaled to the [0, 1] interval. The features were then sorted by their importance, separately for each method, and their importance was plotted.</p></div></div><div content-id="DSR016F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01601.jpeg?Expires=2147483647&Signature=xic5wC2m7l69wHXHbeqFj5wUAub8~S8djP1nyUMdZcdRav6vH5qZMyXlWEuKR3EYRpZT7t7z9nphy3tXWAA6h3kYNwNCyJgtAabakQuCHT9crvTXa2bWmbnfnaKLtuKS5uh7rF1qCCdY5HFJVc0g6bjQF1mkXXWFv7TfFfbn~TvlUfS81P4droXOg6cQe1SSj24vpcgEFaiRfmFGdChZERCZ6AwsioTJJoWrVFrqCCF1pUlRCPDiLj4Pba0Z-t11StVtgNl4497GRp-Q8COdEFeEf4VoR6sW6VoQf-YrSOyiTRZya8zaC3vm39l8ijR96cE6qNc7-VUtjQIRGZSsvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Feature-importance distribution. Each feature (gene) from the 1000-gene data set was assigned an importance score by each of the three feature selection methods. The scores were scaled to the [0, 1] interval. The features were then sorted by their importance, separately for each method, and their importance was plotted." path-from-xml="dsr01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741268/dsr01601.jpeg" path-from-xml="dsr01601.jpeg" target="_blank">View large</a><a section="7741268" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01601.jpeg?Expires=2147483647&Signature=ugX0oEBR0moEzzpKyWq3thIQwKGKyWMKsiBgxjb7JLGPpFAeLGDRG7VyKO7YkZzWUkk9s7J7N7VbYMRN7aAYKUrxciE9Recu5dVlzh0vXDsisdfqOdy-nFh0fss2herI6~0sScy9vmy6O~qrs2cXGnW2YuKxrxQhSVhES2Tk~gwEAKYeeMNzC1s8QQjZ-3gtP3muYv6EWVFPaNSpfUQd1ezMVE1JyY76DAU2mS0I9PEeG~dR06w6LVDrQq5p2cH5MRWUQwz-QiM1WjkEfj64hDwAmp6Ysp2tqlqwQIp~lk6NL5Czffky0zbo8AQRNuyyh6Fc0ivP4jcaI6pDUsLJ8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741268&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Feature-importance distribution. Each feature (gene) from the 1000-gene data set was assigned an importance score by each of the three feature selection methods. The scores were scaled to the [0, 1] interval. The features were then sorted by their importance, separately for each method, and their importance was plotted.</p></div></div><p>In <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S2</a>, we provide all lists of input genes we discuss here. Please note that the term ‘gene’ always refers to a gene from the mouse genome (UniGene gene symbol), and its function will be described based on its protein product. Genes are also called ‘features’ in the description of our machine learning methodology. Thus, ‘feature selection’ is synonymous with ‘gene selection’.</p>                            <h3 scrollto-destination=7741271 id="7741271" class="section-title" data-legacy-id=s2c>2.3. Classification</h3>
<p>Classification using both the whole (filtered) data set, and the feature sets selected by the feature selection methods under investigation, was performed with the Weka machine learning suite.<sup><a reveal-id="DSR016C29" data-open="DSR016C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The following machine learning algorithms were used:
Concentrating on the discovery of optimal feature sets (biomarkers), we did not tune parameters to maximize classification performance, and all parameters of the machine learning algorithms were kept at Weka's default values. The only exception was the SVM with Gaussian kernel, for which the default LibSVM<sup><a reveal-id="DSR016C35" data-open="DSR016C35" class="link link-ref link-reveal xref-bibr">35</a></sup> parameter values were used, consistent with the use of LibSVM during feature selection with the GA (see below). We briefly describe two of the machine learning algorithms: the random forest, because of its relevance to feature selection (see below), and the SVM, because it is the algorithm we also used for feature selection with the GA.</p><ul class="bullet"><li><p>Naive Bayes<sup><a reveal-id="DSR016C30" data-open="DSR016C30" class="link link-ref link-reveal xref-bibr">30</a></sup>;</p></li><li><p>C4.5 decision trees, implemented in Weka as J48<sup><a reveal-id="DSR016C31" data-open="DSR016C31" class="link link-ref link-reveal xref-bibr">31</a></sup>;</p></li><li><p>Random forest<sup><a reveal-id="DSR016C32" data-open="DSR016C32" class="link link-ref link-reveal xref-bibr">32</a></sup>;</p></li><li><p>Nearest neighbour, implemented in Weka as IBk<sup><a reveal-id="DSR016C33" data-open="DSR016C33" class="link link-ref link-reveal xref-bibr">33</a></sup>;</p></li><li><p>SVM, implemented in Weka as sequential minimal optimization (SMO).<sup><a reveal-id="DSR016C34" data-open="DSR016C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p></li></ul><p>The random forest<sup><a reveal-id="DSR016C32" data-open="DSR016C32" class="link link-ref link-reveal xref-bibr">32</a></sup> is a machine learning approach working with an ensemble of decision trees. Let <em>N</em> be the size of the training set and <em>M</em> the number of features (in our case gene expression values). To grow a tree, <em>N</em> instances from the training set are selected randomly with replacement (which means that some are selected more than once and some never). Then <em>m</em> features (<em>m</em> &lt;&lt; <em>M</em>) are selected randomly. Out of these, the one that splits the instances into sets purest with respect to the class is assigned to the root of the tree. This procedure is repeated recursively until the leaves of the tree contain only instances of one class (pluripotent or non-pluripotent in our case). The whole forest consists of a number of such decision trees. To classify an instance, it is classified by all the trees in the forest and the final class is selected by majority voting.</p><p>The support vector machine (SVM<sup><a reveal-id="DSR016C34" data-open="DSR016C34" class="link link-ref link-reveal xref-bibr">34</a></sup>) finds a hyperplane across the <em>M</em>-dimensional space occupied by instances that best separates the two classes. In cases where the instances are not linearly separable, their features can be mapped into a higher-dimensional space. Let <em>x<sub>i</sub></em> and <em>x<sub>j</sub></em> be a pair of <em>M</em>-dimensional feature vectors. Let <em>Φ</em>(<em>x<sub>i</sub></em>) and <em>Φ</em>(<em>x<sub>j</sub></em>) be these vectors mapped into a higher-dimensional space. Since computing the hyperplane that separates the instances only involves computing inner products of feature vectors, the mapping can be efficiently accomplished by a kernel function <em>K</em>(<em>x<sub>i</sub></em>, <em>x<sub>j</sub></em>), which returns the inner product of <em>Φ</em>(<em>x<sub>i</sub></em>) and <em>Φ</em>(<em>x<sub>j</sub></em>) without explicitly performing the mapping. The linear kernel function returns the plain dot product: <em>K</em><sub>lin</sub>(<em>x<sub>i</sub></em>, <em>x<sub>j</sub></em>) = <em>x<sub>i</sub></em> · <em>x<sub>j</sub></em>. The Gaussian kernel function is defined as follows: <em>K</em><sub>Gauss</sub>(<em>x<sub>i</sub></em>, <em>x<sub>j</sub></em>) = exp (–<em>γ</em> (<em>x<sub>i</sub></em> − <em>x<sub>j</sub></em>)<sup>2</sup>). The value of <em>γ</em> controls how much the classifier can adapt to an irregular boundary between classes, and must be selected by the experimenter. As mentioned before, we used the default LibSVM value of <em>γ</em> = 1/number_of_features.</p><p>The classification performance was evaluated by 3-fold cross-validation as described in the previous subsection. The performance is presented in terms of the accuracy and the area under the receiver operational characteristics (ROC) curve (AUC). The accuracy is defined as the number of correctly classified instances divided by the total number of instances. It is an appropriate measure when the data set has roughly the same number of instances belonging to each class (as is the case in ours), when misclassifying any class to any other class is equally undesirable and when one is interested in crisp classification. Let one class be considered negative (non-pluripotent in our case) and the other positive (pluripotent) and let the classifier return a continuous value (instead of just one class or the other). The ROC curve is a plot of the true positive rate vs. the false positive rate, obtained by varying the threshold above which the value returned by the classifier is considered to indicate the positive class. The true positive rate is the number of correctly classified positive instances, divided by the number of all positive instances (the probability to recognize pluripotent samples as such). The false positive rate is the number of incorrectly classified negative instances, divided by the number of all negative instances (the probability to mistake a non-pluripotent sample for a pluripotent one). The AUC is an aggregate measure of the performance of the classifier when one considers different thresholds in order to correctly classify more positive instances at the expense of misclassifying negative ones and vice versa. It is appropriate regardless of how many instances belong to each class.</p>                            <h3 scrollto-destination=7741279 id="7741279" class="section-title" data-legacy-id=s2d>2.4. Feature selection with the information gain</h3>
<div class="block-child-p">The information gain of a feature <em>F</em> is a measure of how much information one gains about the class <em>C</em> if one knows the value of <em>F</em>. In other words, it is the reduction in uncertainty about <em>C</em>, which is measured by its entropy <em>H</em>(<em>C</em>). The uncertainty about <em>C</em>, if one knows the value of <em>F</em>, is measured by its conditional entropy <em>H</em>(<em>C</em>|<em>F</em>). The information gain is thus defined as<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr016ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr016ueq1.gif?Expires=2147483647&amp;Signature=ji-yt-HOSfD1~KlNYQcmwMcK4oS-fo0d0UcRJB5Bv5QJI~ikbke9Bf7~AWnj4fNbXg0xnoaTWL8GLlX76iOSYaj9y9RJJW30oaJPHtNGSaTQ7rZU~YjUxvF2dqhL40hZEw04aoIHcdRaU8D1DY1tvvAX1b1-LfmWwPp3LnV1-WpaF4TUMGt87sbYHc04LLWN~xCzLKKjbCBtL-yoivAG9GjMULcQTt1FUym49FED1~OBUh~W3-MBZUcB5JcHUkJtARITD5l8OAXvuSNdBQSecVDAlgWTkg5jC2vQajPQQT6VpxFlGp8PrT2AKQe1ANQ5SbSVoprJAkcrMV2dLzTqDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr016ueq1.gif" /></a></div></div></div> More conveniently, the information gain can be described in terms of joint entropy <em>H</em>(<em>C</em>, <em>F</em>) as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr016ueq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr016ueq2.gif?Expires=2147483647&amp;Signature=OX1hlaRyiTtL5eNe9e~CTy1zFqA220BA~mfVEalIaJua1b0-dl4-jJuPO5b8wR~A3c8OLB937NxgEr8eq10Xj~zmQl3PkQVBEf627~RyAo1rvgP8BM-3yi~OpebMJDoLOvPZtY91tAkk3wM4BPvfFLJ1NoOSfYwg3~WwMWjGT-1Lw4GSjrliaFjgWgCcfMqmMrijbD2eAv3~zvLuooO3TdYJIicNyL1oa5sOh5vxVXvCuPyJM-1gOy0yH4nrT~YQJNZBMBIBmMtmJoUwdU0cXyyNDFFwX7aWiY-hGaJK3Fkrtpzvi~1f07lG2AD4q3KpCatZTMlUjqnTGSmt29Ntjw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr016ueq2.gif" /></a></div></div></div> Let <em>f</em><sub>1</sub>,…, <em>f<sub>k</sub></em> be the possible values of the feature <em>F</em> (if the feature is continuous, as in our case, it is discretized first) and <em>c</em><sub>1</sub>,…, <em>c<sub>l</sub></em> the possible values of the class <em>C</em> (only two in our case). The entropies are computed as follows (the computation of <em>H</em>(<em>F</em>) is analogous to the computation of <em>H</em>(<em>C</em>)):<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr016ueq3.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr016ueq3.gif?Expires=2147483647&amp;Signature=vU7m2xJeLpEs6-gdNuI5PkTWjZMfEloluwME4f65DjOUG3Dg5wzGDvdYItm4SWsWndnkmjMh4ZsdpCYG1wnyEqkvKKEgND7eSW8XAI9PuS8MeRULBSMu-TtQFIRigPKEA4asDGsiiPV9FphEVTxZf8IvgVVrhzlOj1JLkco3q0FEuTpERgiWU7Tbda9rC6MvYbYheCBYxdMRpnyYd-t8YErU-jIECCJRe7HuklVx0tYJCNkXVsc2kMmE~vyPdfwp5X4bwKBHd2kG1PYmqJmmFEW~W21WjmGgDdgM6wMozik9LnQYK8bQ7vBuoa6c~GJ-DG2bdVziCtneAYKRpqt3KA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr016ueq3.gif" /></a></div></div></div><a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr016ueq4.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr016ueq4.gif?Expires=2147483647&amp;Signature=cSFxqNHxLttmvw2yGAgQ3A5h6MB58ubnSbR-4xkQ-1IzfJKsWIPqkMlktqQHZ6N5G73HeBtV7DNK3ZXFv7jq9aaaoLvWkeLX9HmrGzsNRmGQKKBiLsberInB1aL25nwL3DjgdMTQmqwGX4CzLcFzfw7UAyOpIKNPHRFgvf0ZP-u3tmpYkS9ADZCYerIr7c1Z0scMmgR4yKFNa8VIqfIEL8hqgGeTeAuoDE8DYWsPPpUkEKPeZDmhtUI9e0~zNjPuesFyz6IcQgDutxHvi82L6eEMk4JZP2-OhvNfupkGRf~wHqpzTliCisuH3CL0dHrA1YAl8jLqLUibWaV1NIVyzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr016ueq4.gif" /></a></div></div></div> Feature selection with the information gain was performed using Weka machine learning suite.<sup><a reveal-id="DSR016C29" data-open="DSR016C29" class="link link-ref link-reveal xref-bibr">29</a></sup></div>                            <h3 scrollto-destination=7741285 id="7741285" class="section-title" data-legacy-id=s2e>2.5. Feature selection with the random forest</h3>
<p>This subsection describes the procedure to measure feature importance with the random forest<sup><a reveal-id="DSR016C32" data-open="DSR016C32" class="link link-ref link-reveal xref-bibr">32</a></sup>; an overview of how the random forest works is given in the <em>Classification</em> subsection. To grow a tree in a random forest, <em>N</em> instances from the training set are selected randomly with replacement. <em>N</em> is also the size of the training set, but since the instances are selected with replacement, around one-third of the training instances are left out—these are called the out-of-bag instances. The out-of-bag instances are classified with the tree and the number of correct classifications <em>c</em><sub>before</sub> is counted. For each feature <em>F</em>, its values in the out-of-bag instances are randomly permuted, the out-of-bag instances are classified again and the new number of correct classifications <em>c</em><sub>after</sub> is counted. The difference between <em>c</em><sub>before</sub> and <em>c</em><sub>after</sub>, averaged over all the trees in the forest, is a measure of the importance of <em>F</em>.</p><p>Feature selection with the random forest was performed in a custom Weka distribution by Livingston.<sup><a reveal-id="DSR016C36" data-open="DSR016C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The number of trees in a forest was set to 1000 and the other parameters were kept at default values. A random forest was generated three times and the importance of features was averaged over the three runs.</p>                            <h3 scrollto-destination=7741289 id="7741289" class="section-title" data-legacy-id=s2f>2.6. Feature selection with the GA</h3>
<p>We are looking for small sets of genes that enable us to classify a sample as pluripotent or not, which we call minimal sets of best biomarkers. We search for such sets with the GA<sup><a reveal-id="DSR016C21" data-open="DSR016C21" class="link link-ref link-reveal xref-bibr">21</a></sup> guided by a fitness function. The fitness of a gene set is defined by the classification accuracy of the SVM<sup><a reveal-id="DSR016C34" data-open="DSR016C34" class="link link-ref link-reveal xref-bibr">34</a></sup> using that set of genes as features, and the size of the set. Thus, the objective of the GA is to find optimal sets of features for classification. Each set of genes/features is binary encoded in a ‘chromosome’. Each bit of the chromosome represents one gene/feature. If a gene belongs to the set of features encoded by the chromosome, the corresponding bit is set to 1, otherwise it is set to 0. (Apart from the binary representation used here, more compact representations may be used, as was done, for example, by Rocha <em>et al</em>.<sup><a reveal-id="DSR016C28" data-open="DSR016C28" class="link link-ref link-reveal xref-bibr">28</a></sup> A systematic evaluation of such alternative representations is future work.) The initial population of 200 chromosomes is created by setting a random number of bits in each chromosome to 1, and calculating the fitness of the resulting feature set, as follows. </p>                        <a id="7741293" scrollto-destination="7741293"></a>
<div class="table-wrap"><div id="" class="table-wrap-title" data-id=""> </div><div class="table-overflow"><table><tbody><tr><td><strong>Pseudocode: create initial population</strong> </td></tr><tr><td> <strong>for each</strong> chromosome <em>j</em> </td></tr><tr><td>  <strong>for each</strong> bit <em>i</em> of <em>j</em> </td></tr><tr><td>   set <em>i</em> = 0 </td></tr><tr><td>   <strong>do</strong> with a probability of <em>start_chromosome_size/|set_of_all_genes</em>| </td></tr><tr><td>    set <em>i</em> = 1 </td></tr><tr><td>   <strong>end do</strong> </td></tr><tr><td>  <strong>end for</strong> </td></tr><tr><td>  <strong>compute fitness</strong> of the chromosome </td></tr><tr><td><strong>end for</strong> </td></tr></tbody></table></div><div content-id="" class="table-modal"><table><tbody><tr><td><strong>Pseudocode: create initial population</strong> </td></tr><tr><td> <strong>for each</strong> chromosome <em>j</em> </td></tr><tr><td>  <strong>for each</strong> bit <em>i</em> of <em>j</em> </td></tr><tr><td>   set <em>i</em> = 0 </td></tr><tr><td>   <strong>do</strong> with a probability of <em>start_chromosome_size/|set_of_all_genes</em>| </td></tr><tr><td>    set <em>i</em> = 1 </td></tr><tr><td>   <strong>end do</strong> </td></tr><tr><td>  <strong>end for</strong> </td></tr><tr><td>  <strong>compute fitness</strong> of the chromosome </td></tr><tr><td><strong>end for</strong> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741293" target="_blank">View Large</a></div></div><p>The value |<em>set_of_all_genes</em>| is the number of filtered genes we begin with, in our case 1000. The value <em>start_chromosome_size</em> is the mean number of selected genes for each chromosome of the initial population, which in our case is set to 15. That way each chromosome of the initial population consists of 15 genes on average.</p><div class="block-child-p">As mentioned before, the fitness of a chromosome is defined by two criteria: the classification accuracy calculated using the SVM, and the number of features/genes. The smaller the number of features and the better the classification accuracy of the feature set, the fitter the chromosome. Following a multiobjective approach, we combine both criteria into one fitness function, <em>f</em>, to be optimized:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr016ueq5.gif?Expires=2147483647&Signature=yavr8okd-utPO-V38yYDxA5wq7Z9a5L2i6yW5Fy24egM6ERg8oeilElCx4Wsk6r0WmXWf9fQ8b~JTRiLmeJhVenBV~8UAorr~5f8aYyhVT9G2hPh3jPJo3TJ3CWMG9tcPXW8W8EsdWyYHN9d6NhZbcKYRe9GEjvVShNuAZ9zurvZZMQNXnNeCOx8IOP6GruKmvzYuzxHGGgqa3uuJpXmV0BP3lZOJ7KiLtyBoBSPHsewFXKjI0lbZePHWp6RztNettE7Ffw3PqlISw6uR88CT5KiV5SR2aXCjJTOCktps2MlHmu3liLDqCpJ754tNOsIkDsdMULD526nP57M6pGd6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr016ueq5.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr016ueq5.gif?Expires=2147483647&Signature=4sZhm6KaA3DCO55Wvz56CrLFlkNt3jXW7V4nAM3uySNXS2QHCi2AAqNRp4JrLvvQ7KCG9e-PLSYO7GBig~IH9kK4IApBIsskILbUKgvocrkWYrJVsevkdEoXeDcRvlw4xluUM1VEK8um4Jq3wYC7-4JiThXFVMIJgn~7eu6OvN3OYV-f7jnT10BTgB6G5x9TOoumSRg0Bi2KiwFIjnNjEkj-eXl~2A~WLQjCKycN9xkbiK1dGzEcTyKCYKpOVURkZgx4ZPbhAPjvSPiMf4YLJgkdll1MdtC6ZA~LlPQGLR8iw2KbVyCwJ8syo49FQj9TqphCbnLLMjmtZB8phkIY8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr016ueq5.gif" /></a></div></div></div> The value <em>accuracy</em> is the classification accuracy of the SVM with Gaussian kernel (<em>C</em> = 1, <em>γ</em> = 1/start_chromosome_size), using the LibSVM<sup><a reveal-id="DSR016C35" data-open="DSR016C35" class="link link-ref link-reveal xref-bibr">35</a></sup> implementation with 6-fold cross-validation. The value |<em>set_of_genes_on_the_chromosome</em>| is the number of selected features. Because the main objective is finding correct biomarkers, accurate classification capability plays a bigger role than the number of selected features. For this reason we choose <em>W</em> = 0.2, as in reference.<sup><a reveal-id="DSR016C37" data-open="DSR016C37" class="link link-ref link-reveal xref-bibr">37</a></sup></div><p>Based on this initial population, we breed a new generation using recombination and mutation. Recombination is implemented as follows. </p>                        <a id="7741299" scrollto-destination="7741299"></a>
<div class="table-wrap"><div id="" class="table-wrap-title" data-id=""> </div><div class="table-overflow"><table><tbody><tr><td><strong>Pseudocode: recombination</strong> </td></tr><tr><td> <strong>for</strong> 1 <strong>to</strong><em>size_of_population</em><strong>do</strong> </td></tr><tr><td>  <strong>do</strong> with a probability of 0.8 </td></tr><tr><td>   select two chromosomes by <strong>roulette wheel selection</strong> </td></tr><tr><td>   combine the two chromosomes by <strong>uniform cross over</strong> </td></tr><tr><td>  <strong>end do</strong> </td></tr><tr><td>  add new chromosome to population </td></tr><tr><td><strong>end for</strong> </td></tr></tbody></table></div><div content-id="" class="table-modal"><table><tbody><tr><td><strong>Pseudocode: recombination</strong> </td></tr><tr><td> <strong>for</strong> 1 <strong>to</strong><em>size_of_population</em><strong>do</strong> </td></tr><tr><td>  <strong>do</strong> with a probability of 0.8 </td></tr><tr><td>   select two chromosomes by <strong>roulette wheel selection</strong> </td></tr><tr><td>   combine the two chromosomes by <strong>uniform cross over</strong> </td></tr><tr><td>  <strong>end do</strong> </td></tr><tr><td>  add new chromosome to population </td></tr><tr><td><strong>end for</strong> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741299" target="_blank">View Large</a></div></div><p>Due to the recombination rate of 0.8, in each generation, about 160 new chromosomes are generated by recombination using uniform crossover.<sup><a reveal-id="DSR016C38" data-open="DSR016C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Two parent chromosomes are selected using roulette wheel selection, i.e. the probability for a chromosome to be selected is proportional to its fitness. Then, the bits at the same position in both parent chromosomes are compared with each other. If they have the same value, this value is chosen for the child chromosome. If the values are different, the bit in the child chromosome is selected randomly.</p><p>Subsequently to the recombination step, all chromosomes are mutated. On average, we flip 1.5 bits on each chromosome, as follows. </p>                        <a id="7741304" scrollto-destination="7741304"></a>
<div class="table-wrap"><div id="" class="table-wrap-title" data-id=""> </div><div class="table-overflow"><table><tbody><tr><td><strong>Pseudocode: mutation</strong> </td></tr><tr><td> <strong>for each</strong> chromosome <em>j</em> of the population </td></tr><tr><td>  <strong>for each</strong> bit <em>i</em> of <em>j</em> </td></tr><tr><td>   <strong>do</strong> with a probability of 0.1 × <em>start_chomosome_size</em>/|<em>set_of_all_genes</em>| </td></tr><tr><td>    flip bit <em>i</em> </td></tr><tr><td>   <strong>end do</strong> </td></tr><tr><td>   <strong>end for</strong> </td></tr><tr><td>  add new chromosome to population </td></tr><tr><td><strong>end for</strong> </td></tr></tbody></table></div><div content-id="" class="table-modal"><table><tbody><tr><td><strong>Pseudocode: mutation</strong> </td></tr><tr><td> <strong>for each</strong> chromosome <em>j</em> of the population </td></tr><tr><td>  <strong>for each</strong> bit <em>i</em> of <em>j</em> </td></tr><tr><td>   <strong>do</strong> with a probability of 0.1 × <em>start_chomosome_size</em>/|<em>set_of_all_genes</em>| </td></tr><tr><td>    flip bit <em>i</em> </td></tr><tr><td>   <strong>end do</strong> </td></tr><tr><td>   <strong>end for</strong> </td></tr><tr><td>  add new chromosome to population </td></tr><tr><td><strong>end for</strong> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741304" target="_blank">View Large</a></div></div><p>Finally, following an elitist approach, the resulting chromosomes as well as the chromosomes of the initial population are sorted in descending order of fitness. Then, the first 200 chromosomes are selected to form the new population. This new population serves as the initial population for the next generation. Gene sets with very good cross-validation accuracies were already found after 15 to 20 generations. Thus, the whole process was repeated for 25 generations to obtain the final population.</p><p>The best chromosome of the final population contains the potential biomarkers we are looking for. To compare the GA to the other two classification methods, we ran the GA 200 times for each of the three folds. Then, we sorted the genes in decreasing order by frequency of occurrences, generating a ranked list of all genes that is comparable to the lists we obtained with the random forest and the information gain. For those parts of the paper that do not involve training and testing, we ran the GA 500 times, using the whole set of samples.</p>                            <h3 scrollto-destination=7741307 id="7741307" class="section-title" data-legacy-id=s2g>2.7. Top 20 most important genes</h3>
<p>To obtain the list of the top 20 most important genes, an importance score for each gene in the 1000-gene data set was computed by each of the three feature selection methods: the information gain, the random forest and the GA (as described in the previous three subsections). The information gain and the random forest both assign a real-number score to each feature, which we used directly. The GA selects a well-performing set of features that differ from run to run. In our experiments, the size of the set selected by the GA was between 3 and 9 and the features selected varied considerably. Hence we measured the importance of a feature by the number of times it was selected over 500 runs of the GA, using 1000 genes and all the samples in the data set. The genes were ranked by their importance according to each method.</p><p>To compute the overall top 20 most important genes, the ranks for each gene were averaged and the overall top 20 genes were chosen by the average rank. We used ranks instead of numeric importance scores because (Fig. <a reveal-id="DSR016F1" data-open="DSR016F1" class="link link-reveal link-table xref-fig">1</a>) the scores assigned by the information gain are on average much larger than those by the random forest and the GA. As a consequence, using the scores instead of ranks would give the information gain much greater weight in the overall ranking.</p>                            <h3 scrollto-destination=7741311 id="7741311" class="section-title" data-legacy-id=s2h>2.8. Enrichment analysis</h3>
<p>To evaluate the biological relevance of our results, we applied gene set enrichment analysis using the hypergeometric distribution.<sup><a reveal-id="DSR016C39" data-open="DSR016C39" class="link link-ref link-reveal xref-bibr">39</a></sup> We assumed that the genes selected by our feature selection methods are over-represented in gene sets that can be directly associated with the pluripotent status of cells. For this reason, we compared our selected genes with several pluripotency-related networks and pathways<sup><a reveal-id="DSR016C40  DSR016C46" data-open="DSR016C40  DSR016C46" class="link link-ref link-reveal xref-bibr">40–46</a></sup> (<a class="link link-uri" href="http://c-it.mpi-bn.mpg.de/">http://c-it.mpi-bn.mpg.de/</a>, <a class="link link-uri" href="http://www.genome.jp/kegg/pathway.html">http://www.genome.jp/kegg/pathway.html</a>). As the reference set, we used the set of all genes in the Affymetrix array. The over-representation analysis (ORA) determines whether a pluripotency-related gene set based on a network or pathway is over- or under-represented in one of the gene sets selected with our feature selection methods, and estimates how likely this is due to our selection method (as opposed to observing the same over- or under-representation by chance).</p><div class="block-child-p">Let <em>n</em> be the size of an ORA test set (that is, a feature set to be tested for over- or under-representation) and <em>k</em> the number of genes in the set that belong to a pluripotency-related gene set. Furthermore, let <em>m</em> be the number of genes in our reference set and <em>l</em> the number of the genes in the reference set that also belong to the pluripotency-related gene set. The probability for a randomly selected gene from the reference set to belong to the pluripotency-related set is thus <em>l</em>/<em>m</em>, and we expect to find <em>k</em>′ = <em>n l</em>/<em>m</em> genes of the pluripotency-related genes in our test set. If the number of genes actually found (<em>k</em>) is larger than <em>k</em>′, we can say there is an enrichment of pluripotency-related genes, otherwise there is depletion. We then estimate the statistical significance of enrichment by computing the one-tail <em>P</em>-value using the hypergeometric distribution as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr016ueq6.gif?Expires=2147483647&Signature=5O~ew~1IwjdqTnrAfoo3c6Pq-pJBwe14H5jas-j8pfX1aSHyV6LksBC82Uq~GzDKD3Q-4vdO6mdFVTNKCxDm6E~StO1oAoMp9s2kE~quJxHQAT5kviDvM5Y74Icjr4OTbvdI7n60LdvSdVWJImWiJteDmF-2GSTSb40aUvoRnYqLIK6vPJPnGzgrFzJpZbG4zjpD5pNCL-JEIPx4fBpDx5yTj0iMtSE8qkPogQoP3RYHPDVxczIQXJevrppWBvvA~yogbwH3U5YwI75gEqLSRCs8Y79db5tHubHfSFds~OB5zdkxAWyiEyOSzErwWzpJGWJPRM-AJoVSYBGE6EoNag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr016ueq6.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr016ueq6.gif?Expires=2147483647&Signature=HXHALxFk~EbC4vP9a7tOYwrB6kwKI~PKDlwoZxykQPSXoC6QldCMZ6iKylG9bZO6z5f2Ch2693ACuzKSz9feL65zRce0FuCypkSsvCmm-cF9SAIWjgU949GAB54crOMPgT8WWf-UQbLD2mCFmDrCYqSlIncWdR32Xvr5XSfeB4c~kxcLNQCfQnY2e12uG4y7QJ2SjRaZtBX7XX0ijAl~ziy4P8qwXwZoemS8Mv-0Onj5hpfN3YDNcJ-phfMS5uMhqoxrLgen7WYWfC2TM5BwMWvohi8gjMxERAwNtSAGsSzUhSNxF9zMsEkl0BeSx4g2q4vAjpDA9GQndRl3N6rvsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr016ueq6.gif" /></a></div></div></div></div>                            <h2 scrollto-destination=7741316 id="7741316" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7741317 id="7741317" class="section-title" data-legacy-id=s3a>3.1. Classification without feature selection</h3>
<p>Our first objective was to establish how easy it is to classify samples as pluripotent or non-pluripotent with machine learning, and to explore the various machine learning algorithms available for the task. We compared five algorithms chosen to represent different approaches to machine learning: the naive Bayes, the C4.5 decision trees, the random forest (an ensemble of decision trees), the nearest neighbour and the SVM. Two kernels were tried with the SVM: Gaussian and linear. The algorithms were tested on the full set of 20 668 genes as well as on the filtered sets of the 5000 and 1000 most strongly differentially expressed genes. Each sample represented one machine learning instance with gene expression values as features and ‘pluripotent’ or ‘non-pluripotent’ as class labels. The cross-validated classification results are presented in Table <a reveal-id="DSR016TB1" data-open="DSR016TB1" class="link link-reveal link-table xref-fig">1</a> in terms of the accuracy and the AUC. </p>                        <a id="7741320" scrollto-destination="7741320"></a>
<div content-id="DSR016TB1" class="table-modal"><div class="table-wrap"><div id="DSR016TB1" class="table-wrap-title" data-id="DSR016TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of machine learning algorithms and data sets</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01609.gif?Expires=2147483647&Signature=W7GFijmXJGp83g6gY22W-WzFGV6UXx-YBmHhI-bvYA3ewRJbOmDAWsYfdutHHVx9k4VCpj7Dw5P0HEPNNrNEJlUFtf~TaJTiN0Pp9YoP-lpugFppf9aSMMHWU4jIe3TD2RhAom7SedtlFZ8CPqfBmffw-0B3bQm6YAkFQceJql2WgxLgknuKaGKlQ-HS5zsh0Da6mICbh0CWMRMfdwvFVdN4oGqED2O0s3mPY4qZg2sQeVmXyq8OmponsBbDqYU9-VN2E9gkyJ9QeLIuJMTNcFoDUSjCD43kK96RgaT5NY2GH2-UimzD1nY4~rQhQkObaObWkPybtY4TiqXddDqMnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01609.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01609.gif?Expires=2147483647&Signature=W7GFijmXJGp83g6gY22W-WzFGV6UXx-YBmHhI-bvYA3ewRJbOmDAWsYfdutHHVx9k4VCpj7Dw5P0HEPNNrNEJlUFtf~TaJTiN0Pp9YoP-lpugFppf9aSMMHWU4jIe3TD2RhAom7SedtlFZ8CPqfBmffw-0B3bQm6YAkFQceJql2WgxLgknuKaGKlQ-HS5zsh0Da6mICbh0CWMRMfdwvFVdN4oGqED2O0s3mPY4qZg2sQeVmXyq8OmponsBbDqYU9-VN2E9gkyJ9QeLIuJMTNcFoDUSjCD43kK96RgaT5NY2GH2-UimzD1nY4~rQhQkObaObWkPybtY4TiqXddDqMnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01609.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The highest cross-validated accuracy and AUC in each column are shown in bold type. The highest accuracy and AUC in each row are shaded in gray.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741320" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR016TB1" class="table-wrap-title" data-id="DSR016TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of machine learning algorithms and data sets</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01609.gif?Expires=2147483647&Signature=W7GFijmXJGp83g6gY22W-WzFGV6UXx-YBmHhI-bvYA3ewRJbOmDAWsYfdutHHVx9k4VCpj7Dw5P0HEPNNrNEJlUFtf~TaJTiN0Pp9YoP-lpugFppf9aSMMHWU4jIe3TD2RhAom7SedtlFZ8CPqfBmffw-0B3bQm6YAkFQceJql2WgxLgknuKaGKlQ-HS5zsh0Da6mICbh0CWMRMfdwvFVdN4oGqED2O0s3mPY4qZg2sQeVmXyq8OmponsBbDqYU9-VN2E9gkyJ9QeLIuJMTNcFoDUSjCD43kK96RgaT5NY2GH2-UimzD1nY4~rQhQkObaObWkPybtY4TiqXddDqMnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01609.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01609.gif?Expires=2147483647&Signature=W7GFijmXJGp83g6gY22W-WzFGV6UXx-YBmHhI-bvYA3ewRJbOmDAWsYfdutHHVx9k4VCpj7Dw5P0HEPNNrNEJlUFtf~TaJTiN0Pp9YoP-lpugFppf9aSMMHWU4jIe3TD2RhAom7SedtlFZ8CPqfBmffw-0B3bQm6YAkFQceJql2WgxLgknuKaGKlQ-HS5zsh0Da6mICbh0CWMRMfdwvFVdN4oGqED2O0s3mPY4qZg2sQeVmXyq8OmponsBbDqYU9-VN2E9gkyJ9QeLIuJMTNcFoDUSjCD43kK96RgaT5NY2GH2-UimzD1nY4~rQhQkObaObWkPybtY4TiqXddDqMnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01609.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The highest cross-validated accuracy and AUC in each column are shown in bold type. The highest accuracy and AUC in each row are shaded in gray.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741320" target="_blank">View Large</a></div></div><p>The first conclusion we can draw from Table <a reveal-id="DSR016TB1" data-open="DSR016TB1" class="link link-reveal link-table xref-fig">1</a> is that machine learning classifies very well, in many cases perfectly. Furthermore, the SVM + linear kernel is the algorithm best suited to this task. The reason the SVM + Gaussian kernel performs worse than the SVM + linear kernel on the larger two data sets is probably that we have a lot of features compared to the number of instances, and the Gaussian kernel effectively increases the number of features.<sup><a reveal-id="DSR016C47" data-open="DSR016C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSR016C48" data-open="DSR016C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Using a large number of features is problematic because this approach increases the chance of learning spurious relations (overfitting). Based on these findings, our first choice for testing the classification performance of small feature sets (see the next subsections) was the SVM; both linear and Gaussian kernels were considered because the reduction in the number of features by feature selection may favour the Gaussian kernel.</p><p>Table <a reveal-id="DSR016TB1" data-open="DSR016TB1" class="link link-reveal link-table xref-fig">1</a> shows that for the three best performing machine learning algorithms (the SVM with both kernels and the nearest neighbour), classification with 5000 genes is better than with all 20 668 genes, and classification with 1000 genes is better than with 5000 genes. Therefore, searching for pluripotency biomarkers in the 1000-gene data set is sufficient and in fact preferable, because many irrelevant genes are already eliminated. Reducing the number of genes further was left to the feature selection methods we investigated (see the next subsection), for which we did not wish to exclude any genes prematurely. Based on Table <a reveal-id="DSR016TB1" data-open="DSR016TB1" class="link link-reveal link-table xref-fig">1</a>, the list of 1000 genes in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S3</a>, sorted by feature importance, may be called the set of genes that enable the best distinction between pluripotent and non-pluripotent, answering <em>question b</em> of the <em>Introduction</em>.</p>                            <h3 scrollto-destination=7741325 id="7741325" class="section-title" data-legacy-id=s3b>3.2. Feature selection</h3>
<p>Feature selection is a technique used in machine learning to reduce the set of features to the most relevant ones, which often improves classification, and, in our case, also identifies biomarkers with potentially important roles in pluripotency. We used three feature selection methods of increasing complexity. The first method is the ranking of features by information gain. The information gain measures how much information about the class one gains by knowing the value of a feature. It considers each feature on its own. The second method is the ranking by feature importance computed by the random forest machine learning algorithm. The importance is obtained by randomly permuting the values of a feature and measuring the resulting decrease in classification accuracy. While this method still evaluates single features, it measures their importance as a part of a classifier that uses other features as well. The third method is feature selection with the GA. Here, the set of features is optimized by the GA guided by the classification accuracy as the main part of the fitness function, computed using the SVM with Gaussian kernel; the size of the feature set also has a minor influence on the fitness function, since we aimed at small sets of best features. This method evaluates whole sets of features.</p><p>Figure <a reveal-id="DSR016F1" data-open="DSR016F1" class="link link-reveal link-table xref-fig">1</a> shows that the three feature selection methods yield considerably different feature-importance distributions. The importance based on the information gain is distributed fairly evenly. The opposite is the case for the importance based on the GA: there are few genes with a high importance and many with a low importance. The random forest is between the two extremes. The most likely explanation for this stems from the well-known redundancy of biomarkers.<sup><a reveal-id="DSR016C49" data-open="DSR016C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Information gain, which considers each gene on its own, finds many of them predictive of pluripotency. The GA selects only the genes most indispensable for classification many times. From the remaining (redundant) genes, different ones are selected in each run due to the random nature of the algorithm, so none ends up with a high importance.</p><p>In order to evaluate the performance of the three feature selection methods, we compared cross-validated classification accuracy on the feature sets selected by them. The samples were split in three folds and the first 1000 genes of each fold were selected (see the <em>Methods</em> section), the GA was run 200 times on each of them. For each of the three folds, we sorted the features by their importance (as for Fig. <a reveal-id="DSR016F1" data-open="DSR016F1" class="link link-reveal link-table xref-fig">1</a>) and selected the top 50, 40, 30, 20, 10, 5, 3, 2 and 1 features, and employing these, we computed the classification accuracy using the SVM with the Gaussian and linear kernel, and using the random forest. Then, we calculated the average accuracy over all three folds. The SVM was chosen for its high performance shown in the previous subsection. However, since it was already used as a component of the feature selection with the GA, the feature sets selected by the GA might favour classification with the SVM. We therefore used the random forest for balance. If the features selected by the GA (using the SVM internally) turned out best even for classification with the random forest, we would obtain an unbiased recommendation for using the GA for feature selection. The results are shown in Figs <a reveal-id="DSR016F2 DSR016F3  DSR016F4" data-open="DSR016F2 DSR016F3  DSR016F4" class="link link-reveal link-table xref-fig">2–4</a>. </p>                        <a id="7741331" scrollto-destination="7741331"></a>
<div id="DSR016F2" data-id="DSR016F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01602.jpeg?Expires=2147483647&Signature=uLh27p2r52xkWvoLEQQTVksSR1JnhLPNGluyklcpmjZuEQuuslYqRVsF6nKKWNISDF05Wbb-1IV-DUU8TcD35Rms5MNi5Uuk5sCRzNYskzidwaQuouwPLGzyuv0CthONgtdA2VPjQ~zZ5WN2aTDqd33kEOfX6JsF-e06gx9Wkz27w7oKIsCDelkOzt9GffiZc4bURbnAj2XtBcGSkAgmctB3K7AbfwHgq1tFa8XKHqup48vOO5nbyxLOkjqQUam719VpFsrm4XI6tweLywjSms7Ep2O8BajR7aCCqpFoGfaSYKcL6cfJnrJ7OurUCUpRHYsk2n28UQIUSQqHGM61Uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification accuracy measured by the SVM with Gaussian kernel. Feature selection with information gain, random forest and GA was evaluated using incrementally smaller sets of most important features from the 1000-gene data set." path-from-xml="dsr01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741331/dsr01602.jpeg" path-from-xml="dsr01602.jpeg" target="_blank">View large</a><a section="7741331" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01602.jpeg?Expires=2147483647&Signature=1s1B2PIsN697TMWdlQp2mo7c2s0~IenaEZvWtt5lAlmhdJ6HKVMM2h78kosxiCI78CRQkI7xaWW7v6IP6yKNWhp6iILGDZEhgRpsgtHOIXK4kxrq~9O4zq3YhTRxNDv9QbjF28sJKAs6Lv6JfXt0MhhY6f8Th24Q~KdRfuWqgurpu8mT9hSGALgnFJc1cGfJgVspAUiuZEyCEvIaOzLLORTcPWJAW2QHrB0ZpL2gbKH32rTOv9JBELXxx8Tat4hE5w2Ty-HF-X7279obHFSSp4o14ZNmJd47LjYgniV1LmKYsg28BRxWeXC77akG0XEdZ7jCWNAS3mo3-h-vhMdNpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741331&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification accuracy measured by the SVM with Gaussian kernel. Feature selection with information gain, random forest and GA was evaluated using incrementally smaller sets of most important features from the 1000-gene data set.</p></div></div><div content-id="DSR016F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01602.jpeg?Expires=2147483647&Signature=uLh27p2r52xkWvoLEQQTVksSR1JnhLPNGluyklcpmjZuEQuuslYqRVsF6nKKWNISDF05Wbb-1IV-DUU8TcD35Rms5MNi5Uuk5sCRzNYskzidwaQuouwPLGzyuv0CthONgtdA2VPjQ~zZ5WN2aTDqd33kEOfX6JsF-e06gx9Wkz27w7oKIsCDelkOzt9GffiZc4bURbnAj2XtBcGSkAgmctB3K7AbfwHgq1tFa8XKHqup48vOO5nbyxLOkjqQUam719VpFsrm4XI6tweLywjSms7Ep2O8BajR7aCCqpFoGfaSYKcL6cfJnrJ7OurUCUpRHYsk2n28UQIUSQqHGM61Uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification accuracy measured by the SVM with Gaussian kernel. Feature selection with information gain, random forest and GA was evaluated using incrementally smaller sets of most important features from the 1000-gene data set." path-from-xml="dsr01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741331/dsr01602.jpeg" path-from-xml="dsr01602.jpeg" target="_blank">View large</a><a section="7741331" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01602.jpeg?Expires=2147483647&Signature=1s1B2PIsN697TMWdlQp2mo7c2s0~IenaEZvWtt5lAlmhdJ6HKVMM2h78kosxiCI78CRQkI7xaWW7v6IP6yKNWhp6iILGDZEhgRpsgtHOIXK4kxrq~9O4zq3YhTRxNDv9QbjF28sJKAs6Lv6JfXt0MhhY6f8Th24Q~KdRfuWqgurpu8mT9hSGALgnFJc1cGfJgVspAUiuZEyCEvIaOzLLORTcPWJAW2QHrB0ZpL2gbKH32rTOv9JBELXxx8Tat4hE5w2Ty-HF-X7279obHFSSp4o14ZNmJd47LjYgniV1LmKYsg28BRxWeXC77akG0XEdZ7jCWNAS3mo3-h-vhMdNpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741331&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification accuracy measured by the SVM with Gaussian kernel. Feature selection with information gain, random forest and GA was evaluated using incrementally smaller sets of most important features from the 1000-gene data set.</p></div></div>                        <a id="7741333" scrollto-destination="7741333"></a>
<div id="DSR016F3" data-id="DSR016F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01603.jpeg?Expires=2147483647&Signature=mYkTKZzCGp9Xr8wDdMA0GJNwPyi2yB5vAe0v3ADF0zNj7rmC3z1mEGTmqQ8V-3x1zC~XIM6nqrv9rOdVuD~K~jwyhWwGRqHXRXiDA-6idkU7g5CMFf2imAONm1jMuKqXx8oaTAQDYVaJw3eGGI65OivpDyuea1PTTFIP6Rgm8isZWoBfTd~jrOKklDo8LDyHPiV2nGgjHlCBB~G3n2NHN4RTkzQX7MP1EyCfPZD48pP1~KJNKxAZ1TzZZsmpGyTtHRBs9hH21~Bpqx~L4I4q5W0rq3Qn3iHVsncKIo-Dah2SXOgGKKj55qHZxrgAMUU1a3icZIRbE9AqtDOc7MwBoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification accuracy measured by the SVM with linear kernel (Fig. 2)." path-from-xml="dsr01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741333/dsr01603.jpeg" path-from-xml="dsr01603.jpeg" target="_blank">View large</a><a section="7741333" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01603.jpeg?Expires=2147483647&Signature=Wv9rFFF0zFzNsEs4IcmVbZ8rKppP-aWEpNpo2HHrG5L9SxRjDUq6dRaOGZFSqeIf8uqMF87ROu3OzGUp7GuJcfeMSHBtNJTpVR0J3YYqkS3q~Af~bFlxVanbBOBa8HPVmjRjHMGuAt3sI8s5gWBitC2ZMUk39fpO8SoNvNrzyflXwHXVeclksCo-ee1pp2KDaxmkGk4~URA9yeMY1Om5Uw7z6-Le25oE~A1tmcioAPiki-77VfZEvLfK-wzgGJEmD072F7WSOmpM9KaxdSdhSn8GdMQBRj~xlzgFHoaZk34mHibfn5Ff3nBBjX-XcOi1B5FyMBQlZ-5hpAKrSt1e5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741333&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification accuracy measured by the SVM with linear kernel (Fig. <a reveal-id="DSR016F2" data-open="DSR016F2" class="link link-reveal link-table xref-fig">2</a>).</p></div></div><div content-id="DSR016F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01603.jpeg?Expires=2147483647&Signature=mYkTKZzCGp9Xr8wDdMA0GJNwPyi2yB5vAe0v3ADF0zNj7rmC3z1mEGTmqQ8V-3x1zC~XIM6nqrv9rOdVuD~K~jwyhWwGRqHXRXiDA-6idkU7g5CMFf2imAONm1jMuKqXx8oaTAQDYVaJw3eGGI65OivpDyuea1PTTFIP6Rgm8isZWoBfTd~jrOKklDo8LDyHPiV2nGgjHlCBB~G3n2NHN4RTkzQX7MP1EyCfPZD48pP1~KJNKxAZ1TzZZsmpGyTtHRBs9hH21~Bpqx~L4I4q5W0rq3Qn3iHVsncKIo-Dah2SXOgGKKj55qHZxrgAMUU1a3icZIRbE9AqtDOc7MwBoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification accuracy measured by the SVM with linear kernel (Fig. 2)." path-from-xml="dsr01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741333/dsr01603.jpeg" path-from-xml="dsr01603.jpeg" target="_blank">View large</a><a section="7741333" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01603.jpeg?Expires=2147483647&Signature=Wv9rFFF0zFzNsEs4IcmVbZ8rKppP-aWEpNpo2HHrG5L9SxRjDUq6dRaOGZFSqeIf8uqMF87ROu3OzGUp7GuJcfeMSHBtNJTpVR0J3YYqkS3q~Af~bFlxVanbBOBa8HPVmjRjHMGuAt3sI8s5gWBitC2ZMUk39fpO8SoNvNrzyflXwHXVeclksCo-ee1pp2KDaxmkGk4~URA9yeMY1Om5Uw7z6-Le25oE~A1tmcioAPiki-77VfZEvLfK-wzgGJEmD072F7WSOmpM9KaxdSdhSn8GdMQBRj~xlzgFHoaZk34mHibfn5Ff3nBBjX-XcOi1B5FyMBQlZ-5hpAKrSt1e5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741333&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification accuracy measured by the SVM with linear kernel (Fig. <a reveal-id="DSR016F2" data-open="DSR016F2" class="link link-reveal link-table xref-fig">2</a>).</p></div></div>                        <a id="7741334" scrollto-destination="7741334"></a>
<div id="DSR016F4" data-id="DSR016F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01604.jpeg?Expires=2147483647&Signature=Zz8QBhVXyYluHgLI6KOzQRwRNrNv-ObPobCH3AK01PGNfDY1vKlpKz4qw90MwhWuLaHD~vdZUwxo2-77DYKDXhnWQYIKuNq2hMYbAeU8qaf4iboeQHeiRICnUJEUHYuQG3nWbwxgKUSGSKq1iNEQX6VYeLbYrCwMFfysABGOhO38~s0wTs7wP-490K2MGD7I-rDIfFIa8oZMiEr3I6O-e~p5zMEvR2taO0p7P-Y0nW9QNIwLQfFK9IsxdpoBTj64Frl7VQMULimyzCfncKwPj~cq9P3fUJAllcpNp7S8LMcJ7WHJMIy8bF0xeBSC09~tv4Be5Xblb2Dx-qTe2jSiRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification accuracy measured by the random forest (Fig. 2)." path-from-xml="dsr01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741334/dsr01604.jpeg" path-from-xml="dsr01604.jpeg" target="_blank">View large</a><a section="7741334" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01604.jpeg?Expires=2147483647&Signature=MUyzUDACYN0ENDDvblsmoLAuxWL1cAwNb5OmN4A5ZLRPxv15~Iof1Vvkev4yiGB72g-3eXKFJGDX4ciG4ThoL-ECX3Pfn0CQSk5iZtg2bmnAEBxbmY25zSoUyVBp85lNm~vabZcFdtv4U-cK~IXu0~O9UmwgbkhlXe7rLl~~RXDsTUfsIBE4103n0EtsYDphIEMfjYhbXQv8LBka6nJBYfJ9ma5TzbxqNsN5c85E5fPu7bmxERnYiOnMyX9AZUz5~UXYyoBqrglj4vyscwjPWGKlEO7jPf0ezFtrZ736dPNP5FMo2Zq1ToY1MT6ndd653wbuuMChRGskimOOYyxPKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741334&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification accuracy measured by the random forest (Fig. <a reveal-id="DSR016F2" data-open="DSR016F2" class="link link-reveal link-table xref-fig">2</a>).</p></div></div><div content-id="DSR016F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01604.jpeg?Expires=2147483647&Signature=Zz8QBhVXyYluHgLI6KOzQRwRNrNv-ObPobCH3AK01PGNfDY1vKlpKz4qw90MwhWuLaHD~vdZUwxo2-77DYKDXhnWQYIKuNq2hMYbAeU8qaf4iboeQHeiRICnUJEUHYuQG3nWbwxgKUSGSKq1iNEQX6VYeLbYrCwMFfysABGOhO38~s0wTs7wP-490K2MGD7I-rDIfFIa8oZMiEr3I6O-e~p5zMEvR2taO0p7P-Y0nW9QNIwLQfFK9IsxdpoBTj64Frl7VQMULimyzCfncKwPj~cq9P3fUJAllcpNp7S8LMcJ7WHJMIy8bF0xeBSC09~tv4Be5Xblb2Dx-qTe2jSiRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification accuracy measured by the random forest (Fig. 2)." path-from-xml="dsr01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741334/dsr01604.jpeg" path-from-xml="dsr01604.jpeg" target="_blank">View large</a><a section="7741334" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01604.jpeg?Expires=2147483647&Signature=MUyzUDACYN0ENDDvblsmoLAuxWL1cAwNb5OmN4A5ZLRPxv15~Iof1Vvkev4yiGB72g-3eXKFJGDX4ciG4ThoL-ECX3Pfn0CQSk5iZtg2bmnAEBxbmY25zSoUyVBp85lNm~vabZcFdtv4U-cK~IXu0~O9UmwgbkhlXe7rLl~~RXDsTUfsIBE4103n0EtsYDphIEMfjYhbXQv8LBka6nJBYfJ9ma5TzbxqNsN5c85E5fPu7bmxERnYiOnMyX9AZUz5~UXYyoBqrglj4vyscwjPWGKlEO7jPf0ezFtrZ736dPNP5FMo2Zq1ToY1MT6ndd653wbuuMChRGskimOOYyxPKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741334&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification accuracy measured by the random forest (Fig. <a reveal-id="DSR016F2" data-open="DSR016F2" class="link link-reveal link-table xref-fig">2</a>).</p></div></div><p>Inspecting Figs <a reveal-id="DSR016F2 DSR016F3  DSR016F4" data-open="DSR016F2 DSR016F3  DSR016F4" class="link link-reveal link-table xref-fig">2–4</a>, we can see that classification with SVM does not particularly favour features selected by the GA (using the SVM with Gaussian kernel internally). Likewise, the classification with the random forest does not seem to favour features selected by the random forest. We may thus conclude that the genes selected by these methods are tied to pluripotency in general and not to any particular machine learning algorithm.</p><p>Figure <a reveal-id="DSR016F3" data-open="DSR016F3" class="link link-reveal link-table xref-fig">3</a> shows that the classification accuracy of features selected by information gain and random forest is very similar, starting at around 96% with 50 features and dropping to around 92% with a single feature. The situation is similar in Fig. <a reveal-id="DSR016F4" data-open="DSR016F4" class="link link-reveal link-table xref-fig">4</a>, except that the accuracy is overall slightly lower, because the random forest (Fig. <a reveal-id="DSR016F4" data-open="DSR016F4" class="link link-reveal link-table xref-fig">4</a>) does not classify as well as the SVM (Fig. <a reveal-id="DSR016F3" data-open="DSR016F3" class="link link-reveal link-table xref-fig">3</a>). The difference between feature selection with information gain and random forest in Fig. <a reveal-id="DSR016F2" data-open="DSR016F2" class="link link-reveal link-table xref-fig">2</a> is larger, which indicates that the random forest may be better at identifying good features than the information gain; however, since this does not occur in Figs <a reveal-id="DSR016F3" data-open="DSR016F3" class="link link-reveal link-table xref-fig">3</a> and <a reveal-id="DSR016F4" data-open="DSR016F4" class="link link-reveal link-table xref-fig">4</a>, no firm conclusion may be drawn. The behaviour of the features selected by the GA is consistently different from those selected by the other two methods. One feature, two features and in one case three features yield a lower classification accuracy than the same numbers of features selected by the other two methods, but once we have more features, those selected by the GA perform better. This is probably due to the GA selecting sets of features that classify well <em>together</em> but not necessarily individually, whereas the other two methods select features that are good individually.</p><p>Finally, we compared the classification accuracy of features selected by the GA <em>many times</em> over 200 runs in each fold (shown in Fig. <a reveal-id="DSR016F2" data-open="DSR016F2" class="link link-reveal link-table xref-fig">2</a>, for a large number of feature (gene) set sizes) with the accuracy of features selected <em>in single runs</em>. This comparison enables us to investigate the change of accuracy due to aggregation of output (i.e. merging output lists together). The results are shown in Fig. <a reveal-id="DSR016F5" data-open="DSR016F5" class="link link-reveal link-table xref-fig">5</a>. For feature sets of size 7, 6 and 5, features selected most often over 200 runs performed slightly better. For feature sets of size 9 and 8, features selected in single runs were slightly better, but the difference in accuracy was extremely small. This shows that feature sets consisting of features selected most often by the GA offer reasonable performance, while eliminating the variation in features selected in single runs. Only for feature sets of size 4 and 3 did features selected in single runs perform considerably better. This is because the GA selected so few features only when they achieved sufficient classification accuracy by themselves, in exactly this combination; aggregation of output destroys the exact combinations. The features selected most often came from different sets; one could already observe in Figs <a reveal-id="DSR016F2 DSR016F3  DSR016F4" data-open="DSR016F2 DSR016F3  DSR016F4" class="link link-reveal link-table xref-fig">2–4</a> that such features, if they were too few, performed poorly. </p>                        <a id="7741338" scrollto-destination="7741338"></a>
<div id="DSR016F5" data-id="DSR016F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01605.jpeg?Expires=2147483647&Signature=cBRuueATadscz~zVN9RzyuRL2-NjWbX7I7ZZK71cOOJ0iabN3LchW7Dn5nFV61owRnfO~pRdjCFMjE70xnIyRrDyv1girHFSC~pNsy-268oVjAWFTxAdlGLbQyGGssvlCSCotQWU9eTY6XIMKH9lr9Sn5hzr0Csaqrib04NwXcfzIaSb5OE6zBqhoRn2JRkdy277kbLLUi8Aj0dubTgjM5~EPUVQ4SVWydoRgNu9O0JdpL1VzUgsJhpiodh9mrz7~~BMe2eD7d~AG9UomQcbtkevIJMCA6hbbvvOXbx-E6fzBkLrXXdChi26OCqC06jc-2vDouwP64A2tA14eG~yTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification accuracy measured by the SVM with Gaussian kernel, evaluating features selected many times vs. features selected in single runs by the GA. For feature set size s, the classification accuracy of the features selected by the GA in single runs is averaged over all the final feature sets of size s. The features selected many times are the s features most often selected by the GA (classification accuracy is averaged over the three folds)." path-from-xml="dsr01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741338/dsr01605.jpeg" path-from-xml="dsr01605.jpeg" target="_blank">View large</a><a section="7741338" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01605.jpeg?Expires=2147483647&Signature=giAbjaPcY7wnw3ci-iDDR8XlTcVl7gtlbFIlE-d-vIEN-G6tYn0Fp~POmIIkaYs65~uF-AFXKQIRUpCHDz4RWoBT4H3R3DEj8sDctQnX5rfXTQzpE1gJCc2cjMCoK9efbcavTLGp26xG7YB6kpROjNB2FW-pzBy3bjGX3U7iebaf6aLGDT7tXhrKgiHi1ujtmImO6JkvxD3eAdWZAA4zlbqGv55Iyo18qAIjv0M~mKB7shKd1IiiLULBTzkouURJ~bTkjG4tEaowitCg1al7BhFjcmF8h4JmbJqt3GxvdmLOhN4ylrlpvKFnTpiTuVcAi7U7xZRwy6GCmnmy8ouErw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741338&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification accuracy measured by the SVM with Gaussian kernel, evaluating features selected many times vs. features selected in single runs by the GA. For feature set size <em>s</em>, the classification accuracy of the features selected by the GA in single runs is averaged over all the final feature sets of size <em>s</em>. The features selected many times are the <em>s</em> features most often selected by the GA (classification accuracy is averaged over the three folds).</p></div></div><div content-id="DSR016F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01605.jpeg?Expires=2147483647&Signature=cBRuueATadscz~zVN9RzyuRL2-NjWbX7I7ZZK71cOOJ0iabN3LchW7Dn5nFV61owRnfO~pRdjCFMjE70xnIyRrDyv1girHFSC~pNsy-268oVjAWFTxAdlGLbQyGGssvlCSCotQWU9eTY6XIMKH9lr9Sn5hzr0Csaqrib04NwXcfzIaSb5OE6zBqhoRn2JRkdy277kbLLUi8Aj0dubTgjM5~EPUVQ4SVWydoRgNu9O0JdpL1VzUgsJhpiodh9mrz7~~BMe2eD7d~AG9UomQcbtkevIJMCA6hbbvvOXbx-E6fzBkLrXXdChi26OCqC06jc-2vDouwP64A2tA14eG~yTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification accuracy measured by the SVM with Gaussian kernel, evaluating features selected many times vs. features selected in single runs by the GA. For feature set size s, the classification accuracy of the features selected by the GA in single runs is averaged over all the final feature sets of size s. The features selected many times are the s features most often selected by the GA (classification accuracy is averaged over the three folds)." path-from-xml="dsr01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7741338/dsr01605.jpeg" path-from-xml="dsr01605.jpeg" target="_blank">View large</a><a section="7741338" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr01605.jpeg?Expires=2147483647&Signature=giAbjaPcY7wnw3ci-iDDR8XlTcVl7gtlbFIlE-d-vIEN-G6tYn0Fp~POmIIkaYs65~uF-AFXKQIRUpCHDz4RWoBT4H3R3DEj8sDctQnX5rfXTQzpE1gJCc2cjMCoK9efbcavTLGp26xG7YB6kpROjNB2FW-pzBy3bjGX3U7iebaf6aLGDT7tXhrKgiHi1ujtmImO6JkvxD3eAdWZAA4zlbqGv55Iyo18qAIjv0M~mKB7shKd1IiiLULBTzkouURJ~bTkjG4tEaowitCg1al7BhFjcmF8h4JmbJqt3GxvdmLOhN4ylrlpvKFnTpiTuVcAi7U7xZRwy6GCmnmy8ouErw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7741338&ar=501892&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification accuracy measured by the SVM with Gaussian kernel, evaluating features selected many times vs. features selected in single runs by the GA. For feature set size <em>s</em>, the classification accuracy of the features selected by the GA in single runs is averaged over all the final feature sets of size <em>s</em>. The features selected many times are the <em>s</em> features most often selected by the GA (classification accuracy is averaged over the three folds).</p></div></div><p>Table <a reveal-id="DSR016TB2" data-open="DSR016TB2" class="link link-reveal link-table xref-fig">2</a> lists the top 20 most important genes selected from the 1000-gene data set by each of the three feature selection methods. It also lists the overall top 20 genes, which are the ones ranked highest by the three individual methods on average. All the samples in the data set were used and the GA was run 500 times. We can see that the lists by the random forest and the information gain are quite similar, whereas the one by the GA is different. The list of the overall top 20 genes does not contain many of the genes considered important by both the random forest and the information gain, because they were ranked too low by the GA. In the <em>Discussion</em> section, we will describe what is known about the listed genes, with reference to pluripotency. </p>                        <a id="7741340" scrollto-destination="7741340"></a>
<div content-id="DSR016TB2" class="table-modal"><div class="table-wrap"><div id="DSR016TB2" class="table-wrap-title" data-id="DSR016TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Top 20 most important genes</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01610.gif?Expires=2147483647&Signature=sB5q4D49c-B18hcTJ2DcP180toazdTciHgTkg~GdtBxcVzpu7gRb6~B~We1K~7~2M9Ld2lH1Nf6PL7fLFxUhjf1rHdApxsTEYBd-odUhr6JY-ykrbWnM-BAqysujlZBwwcxI80-IFWWaoH7oBUlG4jP8FachuY~pEbextnniyIbBC5fVmY5byb8Z0z4wLdp7m0GdzHJ30kOKsEJR-0BbIck2aOEfLXuVk8G14B~dMn392uai8TMwnRt4GVB4KwMpTqXYfnLL65VfkGwaW6NOeVXZJVAjDmou-CP~duSY~kO8zeVYPYfyja~zK-nfdpGuHln-sWoPHtILs8YNz3u87w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01610.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB2" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01610.gif?Expires=2147483647&Signature=sB5q4D49c-B18hcTJ2DcP180toazdTciHgTkg~GdtBxcVzpu7gRb6~B~We1K~7~2M9Ld2lH1Nf6PL7fLFxUhjf1rHdApxsTEYBd-odUhr6JY-ykrbWnM-BAqysujlZBwwcxI80-IFWWaoH7oBUlG4jP8FachuY~pEbextnniyIbBC5fVmY5byb8Z0z4wLdp7m0GdzHJ30kOKsEJR-0BbIck2aOEfLXuVk8G14B~dMn392uai8TMwnRt4GVB4KwMpTqXYfnLL65VfkGwaW6NOeVXZJVAjDmou-CP~duSY~kO8zeVYPYfyja~zK-nfdpGuHln-sWoPHtILs8YNz3u87w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01610.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The genes that appear in two of the top 20 lists of the individual feature selection methods are shown in italics. The only gene that appears in all three lists (<em>BB001228</em>, also known as <em>Tet1</em>) is underlined. Complete lists are available as <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S2</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741340" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR016TB2" class="table-wrap-title" data-id="DSR016TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Top 20 most important genes</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01610.gif?Expires=2147483647&Signature=sB5q4D49c-B18hcTJ2DcP180toazdTciHgTkg~GdtBxcVzpu7gRb6~B~We1K~7~2M9Ld2lH1Nf6PL7fLFxUhjf1rHdApxsTEYBd-odUhr6JY-ykrbWnM-BAqysujlZBwwcxI80-IFWWaoH7oBUlG4jP8FachuY~pEbextnniyIbBC5fVmY5byb8Z0z4wLdp7m0GdzHJ30kOKsEJR-0BbIck2aOEfLXuVk8G14B~dMn392uai8TMwnRt4GVB4KwMpTqXYfnLL65VfkGwaW6NOeVXZJVAjDmou-CP~duSY~kO8zeVYPYfyja~zK-nfdpGuHln-sWoPHtILs8YNz3u87w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01610.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB2" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01610.gif?Expires=2147483647&Signature=sB5q4D49c-B18hcTJ2DcP180toazdTciHgTkg~GdtBxcVzpu7gRb6~B~We1K~7~2M9Ld2lH1Nf6PL7fLFxUhjf1rHdApxsTEYBd-odUhr6JY-ykrbWnM-BAqysujlZBwwcxI80-IFWWaoH7oBUlG4jP8FachuY~pEbextnniyIbBC5fVmY5byb8Z0z4wLdp7m0GdzHJ30kOKsEJR-0BbIck2aOEfLXuVk8G14B~dMn392uai8TMwnRt4GVB4KwMpTqXYfnLL65VfkGwaW6NOeVXZJVAjDmou-CP~duSY~kO8zeVYPYfyja~zK-nfdpGuHln-sWoPHtILs8YNz3u87w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01610.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The genes that appear in two of the top 20 lists of the individual feature selection methods are shown in italics. The only gene that appears in all three lists (<em>BB001228</em>, also known as <em>Tet1</em>) is underlined. Complete lists are available as <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S2</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741340" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7741341 id="7741341" class="section-title" data-legacy-id=s3c>3.3. Enrichment analysis</h3>
<p>The enrichment analysis was done (as described in the <em>Methods</em> section) for the most important genes identified by feature selection with the GA, information gain and random forest. In the absence of a pre-defined number of most important genes to select, we started with the top 40 genes and increased the number in steps of 20 up to 200 genes. This resulted in 27 gene sets, 9 for each of the three feature selection methods.</p><p>A significant enrichment (<em>P</em>-value &lt; 0.05) of pluripotency-related genes published by Som <em>et al</em>.,<sup><a reveal-id="DSR016C40" data-open="DSR016C40" class="link link-ref link-reveal xref-bibr">40</a></sup> MacArthur <em>et al</em>.<sup><a reveal-id="DSR016C41" data-open="DSR016C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and Muller <em>et al</em>.<sup><a reveal-id="DSR016C42" data-open="DSR016C42" class="link link-ref link-reveal xref-bibr">42</a></sup> can be found for nearly all 27 tested sets, as shown in Table <a reveal-id="DSR016TB3" data-open="DSR016TB3" class="link link-reveal link-table xref-fig">3</a>. For the sets selected by information gain and random forest, we could also observe a consistent significant enrichment of the genes from two other published pluripotency gene lists (PluriUp and PluriPlus<sup><a reveal-id="DSR016C43" data-open="DSR016C43" class="link link-ref link-reveal xref-bibr">43</a></sup>; for calculating enrichments with respect to human genes, we took their mouse orthologs). The analysis was also done with a set of genes which are enriched or depleted in embryonic tissue (abbreviated ‘Tissue+’ and ‘Tissue–' in the table, obtained from <a class="link link-uri" href="http://c-it.mpi-bn.mpg.de/">http://c-it.mpi-bn.mpg.de/</a>). For the genes known to be enriched in embryonic tissue, the lists selected by information gain and random forest show a significant enrichment, whereas no significant enrichment can be found in any of the sets chosen by the GA. For the genes which are depleted in embryonic tissues, there is no enrichment in any of the tested sets. </p>                        <a id="7741344" scrollto-destination="7741344"></a>
<div content-id="DSR016TB3" class="table-modal"><div class="table-wrap"><div id="DSR016TB3" class="table-wrap-title" data-id="DSR016TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Enrichment of genes in pluripotency networks and embryonic tissue</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01606.gif?Expires=2147483647&Signature=FtI~jB9IoBIyLOF8q1AOgWlU8jNKZ~ajHiJqv36TqjFrgfXinwEfJnFrvB-wHC2z0N8NI-xSgp-fm8owcFhlUmjkIyyFCA5wU8uoFm~AVW-SxMFOnKhcJGrY2GAe1KAQUiOeaDbtSgzgJ-i5LII6~~0HGrRM8hEtBU26lYA0CxUQH17w25yRapQQFw0qZMKMgoQ0ODG7TbTbLgrOuInWbYm8OFH-0uGdiSSy2VLgMVDSt2YKYMthr9QKLkE5~qnru2m6OKFgC6ll7KlqIfSgoMHTY~ACS-dPh5LHJb5i4VPoMyluDXRMmE~yMGhNdnXT~IIXm4-vC31xafAAih5xow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01606.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB3" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01606.gif?Expires=2147483647&Signature=FtI~jB9IoBIyLOF8q1AOgWlU8jNKZ~ajHiJqv36TqjFrgfXinwEfJnFrvB-wHC2z0N8NI-xSgp-fm8owcFhlUmjkIyyFCA5wU8uoFm~AVW-SxMFOnKhcJGrY2GAe1KAQUiOeaDbtSgzgJ-i5LII6~~0HGrRM8hEtBU26lYA0CxUQH17w25yRapQQFw0qZMKMgoQ0ODG7TbTbLgrOuInWbYm8OFH-0uGdiSSy2VLgMVDSt2YKYMthr9QKLkE5~qnru2m6OKFgC6ll7KlqIfSgoMHTY~ACS-dPh5LHJb5i4VPoMyluDXRMmE~yMGhNdnXT~IIXm4-vC31xafAAih5xow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01606.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Enrichment and its significance of 27 gene sets found by feature selection, using two colours: enrichment in light grey, significant enrichment (<em>P</em>-value ≤ 0.05) in dark grey. See text for abbreviations. Exact counts and <em>p</em>-values can be found in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S4</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741344" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR016TB3" class="table-wrap-title" data-id="DSR016TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Enrichment of genes in pluripotency networks and embryonic tissue</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01606.gif?Expires=2147483647&Signature=FtI~jB9IoBIyLOF8q1AOgWlU8jNKZ~ajHiJqv36TqjFrgfXinwEfJnFrvB-wHC2z0N8NI-xSgp-fm8owcFhlUmjkIyyFCA5wU8uoFm~AVW-SxMFOnKhcJGrY2GAe1KAQUiOeaDbtSgzgJ-i5LII6~~0HGrRM8hEtBU26lYA0CxUQH17w25yRapQQFw0qZMKMgoQ0ODG7TbTbLgrOuInWbYm8OFH-0uGdiSSy2VLgMVDSt2YKYMthr9QKLkE5~qnru2m6OKFgC6ll7KlqIfSgoMHTY~ACS-dPh5LHJb5i4VPoMyluDXRMmE~yMGhNdnXT~IIXm4-vC31xafAAih5xow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01606.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB3" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01606.gif?Expires=2147483647&Signature=FtI~jB9IoBIyLOF8q1AOgWlU8jNKZ~ajHiJqv36TqjFrgfXinwEfJnFrvB-wHC2z0N8NI-xSgp-fm8owcFhlUmjkIyyFCA5wU8uoFm~AVW-SxMFOnKhcJGrY2GAe1KAQUiOeaDbtSgzgJ-i5LII6~~0HGrRM8hEtBU26lYA0CxUQH17w25yRapQQFw0qZMKMgoQ0ODG7TbTbLgrOuInWbYm8OFH-0uGdiSSy2VLgMVDSt2YKYMthr9QKLkE5~qnru2m6OKFgC6ll7KlqIfSgoMHTY~ACS-dPh5LHJb5i4VPoMyluDXRMmE~yMGhNdnXT~IIXm4-vC31xafAAih5xow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01606.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Enrichment and its significance of 27 gene sets found by feature selection, using two colours: enrichment in light grey, significant enrichment (<em>P</em>-value ≤ 0.05) in dark grey. See text for abbreviations. Exact counts and <em>p</em>-values can be found in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S4</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741344" target="_blank">View Large</a></div></div><p>Enrichment could also be observed in different pluripotency-related pathways.<sup><a reveal-id="DSR016C44  DSR016C46" data-open="DSR016C44  DSR016C46" class="link link-ref link-reveal xref-bibr">44–46</a>,<a reveal-id="DSR016C50" data-open="DSR016C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Here, the gene sets selected by the GA show an enrichment in more pluripotency-related pathways than the gene sets selected by the other two methods (Table <a reveal-id="DSR016TB4" data-open="DSR016TB4" class="link link-reveal link-table xref-fig">4</a>). The larger variety of pathway annotations for the genes selected by the GA reaffirms that this feature selection method selects a broader variety of biomarkers related to different aspects of pluripotency. </p>                        <a id="7741346" scrollto-destination="7741346"></a>
<div content-id="DSR016TB4" class="table-modal"><div class="table-wrap"><div id="DSR016TB4" class="table-wrap-title" data-id="DSR016TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Enrichment of genes in pluripotency pathways (Table <a reveal-id="DSR016TB3" data-open="DSR016TB3" class="link link-reveal link-table xref-fig">3</a>)</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01607.gif?Expires=2147483647&Signature=PXE7IN-PKDaWMFS4tadBsvHtjfoajgrDM9dpfR2Z~-pfM0QfodHkwbO4PL0KVRhuZf-37HOL6zgNyZdbSD6gx6ENLAJBc8ZClooXfqzneES6hgaTl1ZZ3HHtp4kW0ng7ke6MkYUcwQXKmCFwzQ3vIYEU6KyEyO8I~1sXnABkDtFGeS6TwGPAxJs4CgRvknVlTSw47ljVEZ-l3xyfHwiNSZnopjIQakMx2DNeCPYg9U0ZF9D52boGQDcwmXrZ5PJByyBai84eGNzNS1L-I15R3xYpXKlA1UnzR7zn2QPXKRF3oVazxROnN-z59iaPWsWB3MajXwmmPiF3lqoyd2uOZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01607.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB4" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01607.gif?Expires=2147483647&Signature=PXE7IN-PKDaWMFS4tadBsvHtjfoajgrDM9dpfR2Z~-pfM0QfodHkwbO4PL0KVRhuZf-37HOL6zgNyZdbSD6gx6ENLAJBc8ZClooXfqzneES6hgaTl1ZZ3HHtp4kW0ng7ke6MkYUcwQXKmCFwzQ3vIYEU6KyEyO8I~1sXnABkDtFGeS6TwGPAxJs4CgRvknVlTSw47ljVEZ-l3xyfHwiNSZnopjIQakMx2DNeCPYg9U0ZF9D52boGQDcwmXrZ5PJByyBai84eGNzNS1L-I15R3xYpXKlA1UnzR7zn2QPXKRF3oVazxROnN-z59iaPWsWB3MajXwmmPiF3lqoyd2uOZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01607.gif" /> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741346" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR016TB4" class="table-wrap-title" data-id="DSR016TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Enrichment of genes in pluripotency pathways (Table <a reveal-id="DSR016TB3" data-open="DSR016TB3" class="link link-reveal link-table xref-fig">3</a>)</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01607.gif?Expires=2147483647&Signature=PXE7IN-PKDaWMFS4tadBsvHtjfoajgrDM9dpfR2Z~-pfM0QfodHkwbO4PL0KVRhuZf-37HOL6zgNyZdbSD6gx6ENLAJBc8ZClooXfqzneES6hgaTl1ZZ3HHtp4kW0ng7ke6MkYUcwQXKmCFwzQ3vIYEU6KyEyO8I~1sXnABkDtFGeS6TwGPAxJs4CgRvknVlTSw47ljVEZ-l3xyfHwiNSZnopjIQakMx2DNeCPYg9U0ZF9D52boGQDcwmXrZ5PJByyBai84eGNzNS1L-I15R3xYpXKlA1UnzR7zn2QPXKRF3oVazxROnN-z59iaPWsWB3MajXwmmPiF3lqoyd2uOZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01607.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB4" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01607.gif?Expires=2147483647&Signature=PXE7IN-PKDaWMFS4tadBsvHtjfoajgrDM9dpfR2Z~-pfM0QfodHkwbO4PL0KVRhuZf-37HOL6zgNyZdbSD6gx6ENLAJBc8ZClooXfqzneES6hgaTl1ZZ3HHtp4kW0ng7ke6MkYUcwQXKmCFwzQ3vIYEU6KyEyO8I~1sXnABkDtFGeS6TwGPAxJs4CgRvknVlTSw47ljVEZ-l3xyfHwiNSZnopjIQakMx2DNeCPYg9U0ZF9D52boGQDcwmXrZ5PJByyBai84eGNzNS1L-I15R3xYpXKlA1UnzR7zn2QPXKRF3oVazxROnN-z59iaPWsWB3MajXwmmPiF3lqoyd2uOZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01607.gif" /> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741346" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=7741347 id="7741347" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We have shown that on the basis of gene expression data, the distinction between the pluripotent and the differentiated (non-pluripotent stem) cell state can be learned with cross-validated accuracies reaching 100%. We provided evidence that the features (genes) selected by the combination of the GA and the SVM are small sets of features that classify well and that work best in combination (Figs <a reveal-id="DSR016F2 DSR016F3 DSR016F4  DSR016F5" data-open="DSR016F2 DSR016F3 DSR016F4  DSR016F5" class="link link-reveal link-table xref-fig">2–5</a>). We listed these genes as potential biomarkers in Table <a reveal-id="DSR016TB2" data-open="DSR016TB2" class="link link-reveal link-table xref-fig">2</a> together with the features (biomarkers) selected by two other methods (information gain and random forest). Analysing each of the top 10 pluripotency biomarkers in the columns of Table <a reveal-id="DSR016TB2" data-open="DSR016TB2" class="link link-reveal link-table xref-fig">2</a>, we now wish to answer the following questions: </p><ol class="order"><li><p>What can we find out about the selected genes by literature investigations?</p></li><li><p>Why are the genes generally known to be involved in pluripotency under-represented among the selected genes?</p></li><li><p>How much reliability can be assigned to the selection of genes? What kind of relevance do the selected genes have for pluripotency?</p></li></ol>                            <h3 scrollto-destination=7741350 id="7741350" class="section-title" data-legacy-id=s4a>4.1. Literature investigation of the most important pluripotency biomarkers</h3>
<p>The sets of pluripotency markers returned by the GA include the <em>Fam134b</em> gene in 35% of the selected gene sets (174 times out of 500), making it the most important feature by a large margin (the <em>Pam</em> gene is the runner-up selected in 21% of the feature sets—106 times out of 500). In human, mutations of <em>FAM134B</em> cause severe neuropathy, leading to a recent effort to characterize the mouse ortholog.<sup><a reveal-id="DSR016C51" data-open="DSR016C51" class="link link-ref link-reveal xref-bibr">51</a></sup><em>Fam134b</em> expression was found predominantly in ganglia. The protein co-localized with the cis-Golgi marker giantin, and (partially) with the trans-Golgi marker <em>TGN38</em>; relocation after brefeldin A treatment followed a pattern typical for Golgi-resident proteins. shRNA knockdown to levels of 19–27% reduced the size of the cis-Golgi compartment and impaired cell proliferation in N2a cells. Interestingly, isoform 2 of <em>Fam134b</em> is found in testis, but not in neural tissue. In the neural context, Golgi-mediated processing and/or transport of neurotrophin precursors and their receptors may be impaired by a lack of <em>Fam134b</em>. In the ES cell context, Golgi-mediated processing and/or transport of two cell surface markers of unknown function in pluripotency, <em>SSEA1</em> (stage-specific embryonic antigen 1) and <em>AP</em> (alkaline phosphatase) may be impaired; both <em>SSEA1</em> and <em>AP</em><sup><a reveal-id="DSR016C52" data-open="DSR016C52" class="link link-ref link-reveal xref-bibr">52</a></sup> are localized in the Golgi, to be transported to the cell surface. The observed role of <em>Fam134b</em> in cell proliferation in the mouse is reflected in human; <em>FAM134b</em> is implicated in oesophageal carcinoma, and it promotes cell proliferation.<sup><a reveal-id="DSR016C53" data-open="DSR016C53" class="link link-ref link-reveal xref-bibr">53</a></sup> It is also overexpressed in benign tumours (adenomas), but underexpressed in adenocarcinoma.<sup><a reveal-id="DSR016C54" data-open="DSR016C54" class="link link-ref link-reveal xref-bibr">54</a></sup></p><p>In 52 cases, the <em>Fam134b</em> gene was selected together with the <em>F2rl1</em> gene (a.k.a. <em>PAR-2</em>, see below), which is ranked fourth by the GA. By the design of the genetic-algorithm-based feature selection, the most likely ‘connection’ between both genes is that they cover complementary aspects of pluripotency, <em>F2rl1</em> being concerned with the distinction of pluripotency and multipotency,<sup><a reveal-id="DSR016C55" data-open="DSR016C55" class="link link-ref link-reveal xref-bibr">55</a></sup> acting as a G protein-coupled receptor with a putative role in the mouse blastocyst.<sup><a reveal-id="DSR016C56" data-open="DSR016C56" class="link link-ref link-reveal xref-bibr">56</a></sup></p><p>The <em>Pam</em> (peptidylglycine alpha-amidating monooxygenase, a.k.a. <em>Phm</em>) gene is the second-most important feature selected by the GA. The only evidence we could find for its role in pluripotency is provided by Lyczak <em>et al</em>.,<sup><a reveal-id="DSR016C57" data-open="DSR016C57" class="link link-ref link-reveal xref-bibr">57</a></sup> who investigated a putative <em>remote(!)</em> homolog in C. elegans and concluded ‘Our analysis of <em>PAM-1</em> requirements shows that a puromycin-sensitive aminopeptidase is also required for proteolytic regulation of the oocyte to embryo transition’. <em>Dub1</em> (deubiquinating enzyme 1) is ranked third; the gene is also a novel candidate involved in pluripotency. (De-)ubiquination of histones has recently been shown to have an important role in repressing developmental control genes in ES cells, however.<sup><a reveal-id="DSR016C58" data-open="DSR016C58" class="link link-ref link-reveal xref-bibr">58</a></sup> As described, the <em>F2rl1</em> (<em>PAR-2</em>) gene is ranked fourth, and was implicated in the distinction between multipotency and pluripotency.</p><p>Positions 5–10 are assigned to genes already implicated in pluripotency. <em>Gldc</em> is discussed by Boue <em>et al</em>.<sup><a reveal-id="DSR016C59" data-open="DSR016C59" class="link link-ref link-reveal xref-bibr">59</a></sup> and <em>Spp1</em> (a.k.a <em>Osteopontin</em>) is regulated by <em>Pou5f1</em>/<em>Sox2</em>.<sup><a reveal-id="DSR016C60" data-open="DSR016C60" class="link link-ref link-reveal xref-bibr">60</a></sup><em>Dazl</em> functions in the maintenance of pluripotency of mouse primordial germ cells,<sup><a reveal-id="DSR016C61" data-open="DSR016C61" class="link link-ref link-reveal xref-bibr">61</a></sup> and as a translational regulator during ES cell differention.<sup><a reveal-id="DSR016C62" data-open="DSR016C62" class="link link-ref link-reveal xref-bibr">62</a></sup><em>Ccnd2</em> is repressed by <em>Tcf3</em> in embryonic stem cells.<sup><a reveal-id="DSR016C63" data-open="DSR016C63" class="link link-ref link-reveal xref-bibr">63</a></sup><em>100043292</em> (a.k.a. <em>GM4340</em>) is found only in embryonic tissue in cleavage state (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S5</a>) and <em>Otx2</em> is part of the gene regulatory network in mouse ES cells, working with <em>Oct4</em> (<em>Pou5f1</em>), possibly to maintain gene expression in early progenitors of ectodermal lineages.<sup><a reveal-id="DSR016C64" data-open="DSR016C64" class="link link-ref link-reveal xref-bibr">64</a></sup> All the named genes just mentioned are listed by Boue <em>et al</em>.<sup><a reveal-id="DSR016C59" data-open="DSR016C59" class="link link-ref link-reveal xref-bibr">59</a></sup> in the human or mouse networks of their Figures 2 and 3; none of them is a well-known pluripotency gene, however.</p><p>Genes selected as features by the random forest are more well-known. The top 10 are 3 <em>Dppa</em> (development- and pluripotency-associated) genes, 5 other well-known genes, namely <em>Gdf3</em>,<sup><a reveal-id="DSR016C65" data-open="DSR016C65" class="link link-ref link-reveal xref-bibr">65</a>,<a reveal-id="DSR016C66" data-open="DSR016C66" class="link link-ref link-reveal xref-bibr">66</a></sup><em>Mybl2</em> (a.k.a. <em>B-Myb</em>),<sup><a reveal-id="DSR016C67" data-open="DSR016C67" class="link link-ref link-reveal xref-bibr">67</a></sup><em>Rex2</em>,<sup><a reveal-id="DSR016C55" data-open="DSR016C55" class="link link-ref link-reveal xref-bibr">55</a></sup><em>Zfp42</em> (a.k.a. <em>Rex1</em>)<sup><a reveal-id="DSR016C68" data-open="DSR016C68" class="link link-ref link-reveal xref-bibr">68</a></sup> and <em>Tet1</em> (listed by Affymetrix as <em>BB001228</em>, see below) and two unnamed genes (<em>Ottmusg00000010173</em> and <em>2610305d13rik</em>).</p><p>Information gain highlights the largest set of well-known genes, with <em>Esrrb</em> among the top 10, and 6 of the 10 random forest genes just mentioned. Curiously, however, we find <em>Ottmusg00000010173</em> (a.k.a. <em>Gm13051</em>) on the top of the list, so we investigated it further. The gene is transcribed predominantly in embryonic tissue in the blastocyst and organogenesis stage (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S6</a>). Moreover, NextBio.com lists the study ‘Pluripotent stem cells gene expression at different stages of development in <em>Sox2</em>-deficient embryos’ (GSE15358) as the first study for this gene, reporting its downregulation in epiblast-derived pluripotent stem cells vs. ES cells.</p><p>Combining the three lists of features selected by the GA, the random forest and the information gain should highlight those genes which work best as discriminative markers, no matter whether they are used as features in isolation or together with others. The combined list should thus be superior in its general utility and its resistance to outliers, just as combined lists in homology search are.<sup><a reveal-id="DSR016C69" data-open="DSR016C69" class="link link-ref link-reveal xref-bibr">69</a></sup> Three observations are noteworthy. First, the mouse gene <em>BB001228</em> representing the Tet oncogene 1 (T<em>et1</em>) according to the Affymetrix array annotation scores best. <em>Tet1</em> is deemed responsible for the generation of 5-hydroxymethylcytosine (5hmC)<sup><a reveal-id="DSR016C70" data-open="DSR016C70" class="link link-ref link-reveal xref-bibr">70</a></sup> in mouse ES cells under physiological conditions,<sup><a reveal-id="DSR016C71" data-open="DSR016C71" class="link link-ref link-reveal xref-bibr">71</a></sup> and during ES cell differentiation, the amount of Tet1 and 5hmC decreases. Also, Tet1 maintains Nanog expression and its knockdown impairs the self-renewal and maintenance of mouse ES cells<sup><a reveal-id="DSR016C72" data-open="DSR016C72" class="link link-ref link-reveal xref-bibr">72</a></sup> by participating in the upregulation of pluripotency factors<sup><a reveal-id="DSR016C73" data-open="DSR016C73" class="link link-ref link-reveal xref-bibr">73</a>,<a reveal-id="DSR016C74" data-open="DSR016C74" class="link link-ref link-reveal xref-bibr">74</a></sup> and the downregulation of developmental regulators.<sup><a reveal-id="DSR016C73" data-open="DSR016C73" class="link link-ref link-reveal xref-bibr">73</a></sup> Secondly, the <em>E130012a19rik</em> gene is ranked third; it was most recently discussed as a <em>Klf5</em> target with strongest effect (see Figure 2 in Parisi <em>et al</em>.<sup><a reveal-id="DSR016C75" data-open="DSR016C75" class="link link-ref link-reveal xref-bibr">75</a></sup>). Finally, <em>Calcoco2</em> (a.k.a. <em>NDP52l1</em>) and <em>Cnpy1</em> are now among the top 10; the former has been implicated in pluripotency before,<sup><a reveal-id="DSR016C76  DSR016C79" data-open="DSR016C76  DSR016C79" class="link link-ref link-reveal xref-bibr">76–79</a></sup> and the latter has been reported to be a target of <em>Smad2/3</em> signalling in mouse ES cells, noting its expression in the embryo (2-cell-stage,<sup><a reveal-id="DSR016C80" data-open="DSR016C80" class="link link-ref link-reveal xref-bibr">80</a></sup> their <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/cgi/content/full/dsr016/DC1" target="_blank">Table S7</a>), and it is a regulator of FGF signalling in zebrafish, albeit in a neural context.<sup><a reveal-id="DSR016C81" data-open="DSR016C81" class="link link-ref link-reveal xref-bibr">81</a></sup></p>                            <h3 scrollto-destination=7741358 id="7741358" class="section-title" data-legacy-id=s4b>4.2. Underrepresentation of canonical pluripotency genes</h3>
<p>We found some generally known genes such as <em>Esrrb</em>, <em>Gdf3</em>, <em>Mybl2</em>, <em>Zfp42</em>, some <em>Dppa</em> (development- and pluripotency-associated) genes and, as the cumulative top scorer, <em>BB001228</em> (<em>Tet1</em>). This list does not include many generally known genes such as <em>Pou5f1</em> (a.k.a. <em>Oct4</em>), <em>Sox2</em>, <em>Nanog</em> etc., but not finding these by machine learning is a common phenomenon, as discussed in the subsection on <em>Related Work</em> at the end of the <em>Discussion</em>. The genes <em>Pou5f1</em>, <em>Sox2</em>, <em>Nanog</em>, <em>Lin28</em> and <em>Klf4</em> were found at positions 196, 414, 203, 594 and 83 of our list, respectively; the two genes <em>Tcf3</em> and <em>cMyc</em> are not part of the 1000 gene data set, because the fold changes of these genes are too small.</p><p>Our hypothesis is that the latter genes are general indicators of pluripotency (in particular, they are strongly upregulated in pluripotent samples, see Table <a reveal-id="DSR016TB5" data-open="DSR016TB5" class="link link-reveal link-table xref-fig">5</a>), and with certainty they are involved in its mechanistic basis, but they do not provide crisp classification power because (a) they are involved in other processes such as neural stemness (<em>Sox2</em><sup><a reveal-id="DSR016C82" data-open="DSR016C82" class="link link-ref link-reveal xref-bibr">82</a></sup>) and germline maintenance (<em>Oct4</em> (<em>Pou5f1</em>)<sup><a reveal-id="DSR016C83" data-open="DSR016C83" class="link link-ref link-reveal xref-bibr">83</a></sup>) and/or their activity is shut down gradually, and (b) they are redundant in particular in the GA setting: this method may select any one of them for ‘getting the idea’, in combination with more powerful discriminating genes that reflect pluripotency in a complex, context-dependent and non-linear way. With respect to (a), we indeed observed that Sox2 and Oct4 (Pou5f1) are still found in some negative samples, such as GSM272848 or GSM275556. This is not surprising, since, as described, we included pluripotent stages up to embryonal day 3.5 as well as iPS cells as actual positives, and embryonic body stages from day 5 onwards as well as non-pluripotent germline stem cells as actual negatives. With respect to (b), we observe at least one gene of the pluripotency-related network by Som <em>et al</em>.<sup><a reveal-id="DSR016C40" data-open="DSR016C40" class="link link-ref link-reveal xref-bibr">40</a></sup> in 54% of the feature sets selected by the GA. Also, at least one gene of networks published by MacArthur <em>et al</em>. <sup><a reveal-id="DSR016C41" data-open="DSR016C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and Muller <em>et al</em>.<sup><a reveal-id="DSR016C42" data-open="DSR016C42" class="link link-ref link-reveal xref-bibr">42</a></sup> can be found in 33% and 27% of the selected feature sets, respectively. We hypothesize that each feature set includes at least one ‘general indicator’ of pluripotency; some of them are documented in a pluripotency-related network, but some are not known yet. </p>                        <a id="7741361" scrollto-destination="7741361"></a>
<div content-id="DSR016TB5" class="table-modal"><div class="table-wrap"><div id="DSR016TB5" class="table-wrap-title" data-id="DSR016TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Gene expression values of pluripotency genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Generally known pluripotency genes<hr /> </th><th colspan="3">First three genes selected by the genetic algorithm<hr /> </th></tr><tr><th> </th><th>Pou5f1 </th><th>Sox2 </th><th>Nanog </th><th>Fam134b </th><th>Pam </th><th>Dub1 </th></tr></thead><tbody><tr><td>Non-pluripotent </td><td>6.02 </td><td>6.43 </td><td>4.67 </td><td>7.38 </td><td>5.82 </td><td>4.19 </td></tr><tr><td>Pluripotent </td><td>11.71 </td><td>11.59 </td><td>10.91 </td><td>5.58 </td><td>4.39 </td><td>5.69 </td></tr><tr><td>Fold change </td><td>5.69 </td><td>5.16 </td><td>6.24 </td><td>1.79 </td><td>1.43 </td><td>1.50 </td></tr></tbody></table></div><div content-id="DSR016TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Generally known pluripotency genes<hr /> </th><th colspan="3">First three genes selected by the genetic algorithm<hr /> </th></tr><tr><th> </th><th>Pou5f1 </th><th>Sox2 </th><th>Nanog </th><th>Fam134b </th><th>Pam </th><th>Dub1 </th></tr></thead><tbody><tr><td>Non-pluripotent </td><td>6.02 </td><td>6.43 </td><td>4.67 </td><td>7.38 </td><td>5.82 </td><td>4.19 </td></tr><tr><td>Pluripotent </td><td>11.71 </td><td>11.59 </td><td>10.91 </td><td>5.58 </td><td>4.39 </td><td>5.69 </td></tr><tr><td>Fold change </td><td>5.69 </td><td>5.16 </td><td>6.24 </td><td>1.79 </td><td>1.43 </td><td>1.50 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The table shows the mean expression values for the pluripotent samples and the non-pluripotent samples as well as the resulting fold changes for three generally known pluripotency genes. For comparison, the first three genes found by the GA are also listed.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741361" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR016TB5" class="table-wrap-title" data-id="DSR016TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Gene expression values of pluripotency genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Generally known pluripotency genes<hr /> </th><th colspan="3">First three genes selected by the genetic algorithm<hr /> </th></tr><tr><th> </th><th>Pou5f1 </th><th>Sox2 </th><th>Nanog </th><th>Fam134b </th><th>Pam </th><th>Dub1 </th></tr></thead><tbody><tr><td>Non-pluripotent </td><td>6.02 </td><td>6.43 </td><td>4.67 </td><td>7.38 </td><td>5.82 </td><td>4.19 </td></tr><tr><td>Pluripotent </td><td>11.71 </td><td>11.59 </td><td>10.91 </td><td>5.58 </td><td>4.39 </td><td>5.69 </td></tr><tr><td>Fold change </td><td>5.69 </td><td>5.16 </td><td>6.24 </td><td>1.79 </td><td>1.43 </td><td>1.50 </td></tr></tbody></table></div><div content-id="DSR016TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Generally known pluripotency genes<hr /> </th><th colspan="3">First three genes selected by the genetic algorithm<hr /> </th></tr><tr><th> </th><th>Pou5f1 </th><th>Sox2 </th><th>Nanog </th><th>Fam134b </th><th>Pam </th><th>Dub1 </th></tr></thead><tbody><tr><td>Non-pluripotent </td><td>6.02 </td><td>6.43 </td><td>4.67 </td><td>7.38 </td><td>5.82 </td><td>4.19 </td></tr><tr><td>Pluripotent </td><td>11.71 </td><td>11.59 </td><td>10.91 </td><td>5.58 </td><td>4.39 </td><td>5.69 </td></tr><tr><td>Fold change </td><td>5.69 </td><td>5.16 </td><td>6.24 </td><td>1.79 </td><td>1.43 </td><td>1.50 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The table shows the mean expression values for the pluripotent samples and the non-pluripotent samples as well as the resulting fold changes for three generally known pluripotency genes. For comparison, the first three genes found by the GA are also listed.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741361" target="_blank">View Large</a></div></div><p>The underrepresentation of canonical pluripotency genes is also reflected by the lower levels of enrichment of relevant genes in the feature sets based on the GA (when compared with information gain and random forest, see Table <a reveal-id="DSR016TB3" data-open="DSR016TB3" class="link link-reveal link-table xref-fig">3</a>). In turn, the larger variety of relevant pathways enriched in feature sets based on the GA (Table <a reveal-id="DSR016TB4" data-open="DSR016TB4" class="link link-reveal link-table xref-fig">4</a>) may indicate an overrepresentation of non-canonical pluripotency genes that are (at least peripherically) associated with diverse aspects of pluripotency.</p>                            <h3 scrollto-destination=7741363 id="7741363" class="section-title" data-legacy-id=s4c>4.3. Reliability and relevance of the most important pluripotency biomarkers found by the GA</h3>
<p>The GA is a stochastic search, so we repeated it for a second time with the same data set yielding very similar results, as shown in Table <a reveal-id="DSR016TB6" data-open="DSR016TB6" class="link link-reveal link-table xref-fig">6</a>, confirming the reliability of our list. The evidence for relevance is given in several ways: (a) the literature investigation at the start of this section revealed that most genes we found are related to pluripotency, even though most of them are not well-known pluripotency genes. (b) Our enrichment analyses show that many genes that are selected as features are implicated in pluripotency, being included in networks describing pluripotency, and being overexpressed in embryonic tissue (Table <a reveal-id="DSR016TB3" data-open="DSR016TB3" class="link link-reveal link-table xref-fig">3</a>). (c) The enrichment analyses of Table <a reveal-id="DSR016TB4" data-open="DSR016TB4" class="link link-reveal link-table xref-fig">4</a>, together with the observations of Figs <a reveal-id="DSR016F2" data-open="DSR016F2" class="link link-reveal link-table xref-fig">2</a>–4, imply that the genes selected by the GA cover a wide spectrum of many aspects of pluripotency, and they are most informative about pluripotency in combination. </p>                        <a id="7741365" scrollto-destination="7741365"></a>
<div content-id="DSR016TB6" class="table-modal"><div class="table-wrap"><div id="DSR016TB6" class="table-wrap-title" data-id="DSR016TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Similarity of the results of the reference run and the confirmation run of the GA</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01608.gif?Expires=2147483647&Signature=vF7il~1RM4A0N2T5tSUV6HgiECavZfmPaR0vTZAKF0fNaugVOgEkmrp5V2t59BtEbDEoHZiZS7QWtLZd5I-m-WD5w-xJafVJ4mjLbpQ~CC~cKQ1ZpVjvy0TCbwMC-bzBbEy-sKytcQmDyHAp9xh1GLMfTo7yYc0-OIg-rEQWhk5ALRCJYvnzUCzuuLzWVNvL1IpyX1tyJB~6HvCAz56KAJMg7~1-7pRSiojj2vkwTrTnLD4p6jQwJ5c3ujGtk09fuef~TlP9j1bKL6~ICBHO9kRhrGrp1fRN8eQvMQSyWBRGNWYp9pr-sxtZL-zviHxoTz4RQYnheWT1skzXBy97rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01608.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB6" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01608.gif?Expires=2147483647&Signature=vF7il~1RM4A0N2T5tSUV6HgiECavZfmPaR0vTZAKF0fNaugVOgEkmrp5V2t59BtEbDEoHZiZS7QWtLZd5I-m-WD5w-xJafVJ4mjLbpQ~CC~cKQ1ZpVjvy0TCbwMC-bzBbEy-sKytcQmDyHAp9xh1GLMfTo7yYc0-OIg-rEQWhk5ALRCJYvnzUCzuuLzWVNvL1IpyX1tyJB~6HvCAz56KAJMg7~1-7pRSiojj2vkwTrTnLD4p6jQwJ5c3ujGtk09fuef~TlP9j1bKL6~ICBHO9kRhrGrp1fRN8eQvMQSyWBRGNWYp9pr-sxtZL-zviHxoTz4RQYnheWT1skzXBy97rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01608.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The table shows the similarity of the top 20 genes from the confirmation (second) run of the GA, compared to the reference (first) run. The colour code for the first, second, third and fourth quartile of the genes of the reference run is carried over to the confirmation run; the two genes coloured white are found up to position 30 in the reference run.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741365" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR016TB6" class="table-wrap-title" data-id="DSR016TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Similarity of the results of the reference run and the confirmation run of the GA</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01608.gif?Expires=2147483647&Signature=vF7il~1RM4A0N2T5tSUV6HgiECavZfmPaR0vTZAKF0fNaugVOgEkmrp5V2t59BtEbDEoHZiZS7QWtLZd5I-m-WD5w-xJafVJ4mjLbpQ~CC~cKQ1ZpVjvy0TCbwMC-bzBbEy-sKytcQmDyHAp9xh1GLMfTo7yYc0-OIg-rEQWhk5ALRCJYvnzUCzuuLzWVNvL1IpyX1tyJB~6HvCAz56KAJMg7~1-7pRSiojj2vkwTrTnLD4p6jQwJ5c3ujGtk09fuef~TlP9j1bKL6~ICBHO9kRhrGrp1fRN8eQvMQSyWBRGNWYp9pr-sxtZL-zviHxoTz4RQYnheWT1skzXBy97rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01608.gif" /> </td></tr></tbody></table></div><div content-id="DSR016TB6" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/m_dsr01608.gif?Expires=2147483647&Signature=vF7il~1RM4A0N2T5tSUV6HgiECavZfmPaR0vTZAKF0fNaugVOgEkmrp5V2t59BtEbDEoHZiZS7QWtLZd5I-m-WD5w-xJafVJ4mjLbpQ~CC~cKQ1ZpVjvy0TCbwMC-bzBbEy-sKytcQmDyHAp9xh1GLMfTo7yYc0-OIg-rEQWhk5ALRCJYvnzUCzuuLzWVNvL1IpyX1tyJB~6HvCAz56KAJMg7~1-7pRSiojj2vkwTrTnLD4p6jQwJ5c3ujGtk09fuef~TlP9j1bKL6~ICBHO9kRhrGrp1fRN8eQvMQSyWBRGNWYp9pr-sxtZL-zviHxoTz4RQYnheWT1skzXBy97rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsr01608.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The table shows the similarity of the top 20 genes from the confirmation (second) run of the GA, compared to the reference (first) run. The colour code for the first, second, third and fourth quartile of the genes of the reference run is carried over to the confirmation run; the two genes coloured white are found up to position 30 in the reference run.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7741365" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7741366 id="7741366" class="section-title" data-legacy-id=s4d>4.4. Related work</h3>
<p>Bioinformatics approaches have a long history in pluripotency/stem cell research. The method of ‘Digital differential display’ (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/UniGene/help.cgi?item=ddd">http://www.ncbi.nlm.nih.gov/UniGene/help.cgi?item=ddd</a>) may be called a success story, allowing the discovery of the Nanog gene,<sup><a reveal-id="DSR016C84" data-open="DSR016C84" class="link link-ref link-reveal xref-bibr">84</a></sup> which was later recognized to form, together with <em>Oct4</em> (<em>Pou5f1</em>) and <em>Sox2</em>, the so-called ‘core circuit’ behind pluripotency in both the human and mouse. Basically, Nanog emerged by contrasting transcriptomes from mouse ES cells with those from various other sources, identifying differentially expressed genes and experimentally checking for enhancements in self-renewal triggered by these. Moreover, the discovery of the ‘Yamanaka factors’, transcription factors which can reprogram a fibroblast into an ‘induced pluripotent’ stem cell state (iPS),<sup><a reveal-id="DSR016C85" data-open="DSR016C85" class="link link-ref link-reveal xref-bibr">85</a></sup> was based on testing 24 ES cell marker genes known from several sources, partially discovered by gene expression analyses. There are many approaches<sup><a reveal-id="DSR016C86" data-open="DSR016C86" class="link link-ref link-reveal xref-bibr">86</a>,<a reveal-id="DSR016C87" data-open="DSR016C87" class="link link-ref link-reveal xref-bibr">87</a></sup> using gene expression (but also proteomics<sup><a reveal-id="DSR016C88" data-open="DSR016C88" class="link link-ref link-reveal xref-bibr">88</a>,<a reveal-id="DSR016C89" data-open="DSR016C89" class="link link-ref link-reveal xref-bibr">89</a></sup>) data to derive lists of genes involved in pluripotency. In some publications, the intersection of these lists is studied, and it is long known that such intersections feature only a small number of common genes<sup><a reveal-id="DSR016C89" data-open="DSR016C89" class="link link-ref link-reveal xref-bibr">90</a>,<a reveal-id="DSR016C91" data-open="DSR016C91" class="link link-ref link-reveal xref-bibr">91</a></sup>; this problem of ‘small overlap’, ‘missing consensus signature' or ‘missing reproducability’ is not confined to investigations into pluripotency, but it is also found in attempts to define cancer biomarkers.<sup><a reveal-id="DSR016C92" data-open="DSR016C92" class="link link-ref link-reveal xref-bibr">92</a>,<a reveal-id="DSR016C93" data-open="DSR016C93" class="link link-ref link-reveal xref-bibr">93</a></sup> Moreover, gene lists associated with pluripotency often do not feature the obvious suspects such as the ‘core circuit’ of <em>Oct4</em> (<em>Pou5f1</em>), <em>Sox2</em> and <em>Nanog</em>, the other ‘Yamanaka factors’ <em>Klf4</em> and <em>cMyc</em>,<sup><a reveal-id="DSR016C85" data-open="DSR016C85" class="link link-ref link-reveal xref-bibr">85</a></sup> or other genes commonly associated with pluripotency in the literature,<sup><a reveal-id="DSR016C40" data-open="DSR016C40" class="link link-ref link-reveal xref-bibr">40</a></sup> such as <em>Stat3</em> or <em>Esrrb</em>; see references<sup><a reveal-id="DSR016C87" data-open="DSR016C87" class="link link-ref link-reveal xref-bibr">87</a>,<a reveal-id="DSR016C94" data-open="DSR016C94" class="link link-ref link-reveal xref-bibr">94</a></sup> for examples of gene lists including mostly ‘unexpected markers’. Other analyses based on large data sets feature commonly associated genes, but they also include a large proportion of ‘unexpected markers’.<sup><a reveal-id="DSR016C59" data-open="DSR016C59" class="link link-ref link-reveal xref-bibr">59</a>,<a reveal-id="DSR016C95" data-open="DSR016C95" class="link link-ref link-reveal xref-bibr">95</a>,<a reveal-id="DSR016C96" data-open="DSR016C96" class="link link-ref link-reveal xref-bibr">96</a></sup> Literature curation, expression data analysis and machine learning were applied to derive and enhance networks of genes/proteins involved in pluripotency in the mouse<sup><a reveal-id="DSR016C41" data-open="DSR016C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and in human,<sup><a reveal-id="DSR016C42" data-open="DSR016C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSR016C43" data-open="DSR016C43" class="link link-ref link-reveal xref-bibr">43</a></sup> yielding even more novel candidates for marker genes.</p><p>In contrast to previous work, we investigated and compared two complementary learning approaches: (i) comprehensive learning of pluripotency, employing many features (genes) and striving for maximum accuracy, and (ii) ‘minimal-best’ learning of pluripotency, seeking small sets of best features, akin to ‘biomarker signatures’.<sup><a reveal-id="DSR016C11" data-open="DSR016C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSR016C86" data-open="DSR016C86" class="link link-ref link-reveal xref-bibr">86</a></sup> From a bioinformatics perspective, it is known that small sets of features may even improve classification by reducing noise and overfitting, which is often related to the use of megavariate approaches.<sup><a reveal-id="DSR016C97" data-open="DSR016C97" class="link link-ref link-reveal xref-bibr">97</a></sup> For classification, we used a naive Bayes approach, decision trees, random forest, nearest neighbour and SVM. To select features, we employed information gain, random forest and a wrapper consisting of the GA and the SVM. Wrappers are feature selection methods that search the space of feature subsets and evaluate each subset by testing how well a machine learning algorithm can classify the data using that subset.</p><p>Feature selection methods known from machine learning are a standard approach to identifying biomarkers from gene expression data. Wrappers employing the GA to search the space of feature subsets are known to be well suited for biomarker identification, although they were mostly tested on cancer data in the past. Examples include the work of Lin <em>et al</em>.,<sup><a reveal-id="DSR016C98" data-open="DSR016C98" class="link link-ref link-reveal xref-bibr">98</a></sup> Küçükural <em>et al</em>.,<sup><a reveal-id="DSR016C99" data-open="DSR016C99" class="link link-ref link-reveal xref-bibr">99</a></sup> Gan <em>et al</em>.,<sup><a reveal-id="DSR016C100" data-open="DSR016C100" class="link link-ref link-reveal xref-bibr">100</a></sup> Zhang <em>et al</em>.<sup><a reveal-id="DSR016C101" data-open="DSR016C101" class="link link-ref link-reveal xref-bibr">101</a></sup> and Cannas <em>et al</em>.<sup><a reveal-id="DSR016C102" data-open="DSR016C102" class="link link-ref link-reveal xref-bibr">102</a></sup> Each group of researchers proposed some enhancement to the basic wrapper approach. Lin <em>et al</em>.<sup><a reveal-id="DSR016C98" data-open="DSR016C98" class="link link-ref link-reveal xref-bibr">98</a></sup> made the final selection of genes based on the number of times they were selected during multiple runs of the GA, which we adopted. Küçükural <em>et al</em>.<sup><a reveal-id="DSR016C99" data-open="DSR016C99" class="link link-ref link-reveal xref-bibr">99</a></sup> ‘restarted’ the GA after every 10 generations to improve the classification accuracy on training data, which was used as a part of the fitness function. We could already achieve perfect classification on training data without such restarts. Gan <em>et al</em>.<sup><a reveal-id="DSR016C100" data-open="DSR016C100" class="link link-ref link-reveal xref-bibr">100</a></sup> filtered the initial set of features to make the job of the GA easier. They utilized all their data for filtering instead of using cross-validation. We also filtered our data, but we used an appropriate cross-validation in all steps of our work. Zhang <em>et al</em>.<sup><a reveal-id="DSR016C101" data-open="DSR016C101" class="link link-ref link-reveal xref-bibr">101</a></sup> used multiple classifiers to evaluate the feature subsets selected by the GA. Cannas <em>et al</em>.<sup><a reveal-id="DSR016C102" data-open="DSR016C102" class="link link-ref link-reveal xref-bibr">102</a></sup> also employed filtering and used a combination of classification accuracy and size to evaluate feature subsets; we used feature-subset size in a similar fashion.</p>                            <h2 scrollto-destination=7741370 id="7741370" class="section-title" data-legacy-id=s5>5. Conclusions</h2>
<p>We demonstrated that the wrapper approach to feature selection using the GA and SVM, which was previously used to identify cancer biomarkers, works well for pluripotency data. It yields high cross-validated classification accuracy even if the accuracy is not measured by the SVM, but by another method such as the random forest. The biomarkers it identifies are enriched in pluripotency-related genes and pathways, and while many of them are ‘unexpected markers’, a literature review can connect most of them to pluripotency. Ultimately, experimental validation of the new markers is required, and we encourage other researchers to scrutinize our approach and, possibly, to investigate the genes we found in an experimental setting.</p>                            <h2 scrollto-destination=7741372 id="7741372" class="section-title" data-legacy-id=s6>Authors' contributions</h2>
<p>L.S. developed the feature selection based on the GA, performed the enrichment analyses and wrote parts of the manuscript. R.S. carried out classification experiments and feature selection with information gain. M.L. carried out feature selection with the random forest, evaluated feature selection methods and wrote parts of the manuscript. D.R. contributed to the design of the study, regarding cross-validation and gene set enrichment analyses, and improved the manuscript. G.F. coordinated the study and wrote parts of the manuscript, particularly those related to biological relevance. All authors read and approved the final manuscript.</p>                            <h2 scrollto-destination=7741374 id="7741374" class="section-title" data-legacy-id=s7>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7741376 id="7741376" class="section-title" data-legacy-id=s8>Funding</h2>
<p>Funding by the DFG SPP 1356, Pluripotency and Cellular Reprogramming (FU583/2-1) is gratefully acknowledged.</p>                            <h2 scrollto-destination=7741378 id="7741378" class="backacknowledgements-title" >Acknowledgement</h2>
<p>We thank Dr Michele Boiani and Dr Ingo Kurth for discussions on Fam134b.</p>                            <h2 scrollto-destination=7741380 id="7741380" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR016C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schork</div> <div class="given-names">N.J.</div></div>. </div><div class="article-title">Genetics of complex disease: approaches, problems, and solutions</div>, <div class="source">Am. J. Respir. Crit. Care Med.</div> , <div class="year">1997</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">S103</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20complex%20disease%3A%20approaches%2C%20problems%2C%20and%20solutions&amp;author=N.J.%20Schork&amp;publication_year=1997&amp;journal=Am.%20J.%20Respir.%20Crit.%20Care%20Med.&amp;volume=156&amp;pages=S103-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1164/ajrccm.156.4.12-tac-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1164%2Fajrccm.156.4.12-tac-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1164%2Fajrccm.156.4.12-tac-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9351588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rockman</div> <div class="given-names">M.V.</div></div>. </div><div class="article-title">Reverse engineering the genotype-phenotype map with natural genetic variation</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">456</div> (pg. <div class="fpage">738</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reverse%20engineering%20the%20genotype-phenotype%20map%20with%20natural%20genetic%20variation&amp;author=M.V.%20Rockman&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=738-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07633" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07633" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07633"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19079051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Do</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Scholer</div> <div class="given-names">H.R.</div></div>. </div><div class="article-title">Regulatory circuits underlying pluripotency and reprogramming</div>, <div class="source">Trends Pharmacol. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">296</div>-<div class="lpage">302</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20circuits%20underlying%20pluripotency%20and%20reprogramming&amp;author=J.T.%20Do&amp;author=H.R.%20Scholer&amp;publication_year=2009&amp;journal=Trends%20Pharmacol.%20Sci.&amp;volume=30&amp;pages=296-302" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tips.2009.03.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tips.2009.03.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tips.2009.03.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19427042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrett</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Troup</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Wilhite</div> <div class="given-names">S.E.</div></div>, et al. </div><div class="article-title">NCBI GEO: archive for high-throughput functional genomic data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D885</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20GEO%3A%20archive%20for%20high-throughput%20functional%20genomic%20data&amp;author=T.%20Barrett&amp;author=D.B.%20Troup&amp;author=S.E.%20Wilhite&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D885-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn764" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn764" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn764"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18940857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tusher</div> <div class="given-names">V.G.</div></div>,  <div class="name"><div class="surname">Tibshirani</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Chu</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Significance analysis of microarrays applied to the ionizing radiation response</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">5116</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Significance%20analysis%20of%20microarrays%20applied%20to%20the%20ionizing%20radiation%20response&amp;author=V.G.%20Tusher&amp;author=R.%20Tibshirani&amp;author=G.%20Chu&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=5116-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.091062498" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.091062498" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.091062498"> </span></div> </div></div></div></div></div><div content-id="DSR016C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smyth</div> <div class="given-names">G.K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Y.H.</div></div>,  <div class="name"><div class="surname">Speed</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Statistical issues in cDNA microarray data analysis</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">224</div> (pg. <div class="fpage">111</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20issues%20in%20cDNA%20microarray%20data%20analysis&amp;author=G.K.%20Smyth&amp;author=Y.H.%20Yang&amp;author=T.%20Speed&amp;publication_year=2003&amp;journal=Methods%20Mol.%20Biol.&amp;volume=224&amp;pages=111-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12710670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaffrezic</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Marot</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Degrelle</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hue</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Foulley</div> <div class="given-names">J.-L.</div></div>. </div><div class="article-title">A structural mixed model for variances in differential gene expression studies</div>, <div class="source">Genet. Res. Camb.</div> , <div class="year">2007</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">19</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20structural%20mixed%20model%20for%20variances%20in%20differential%20gene%20expression%20studies&amp;author=F.%20Jaffrezic&amp;author=G.%20Marot&amp;author=S.%20Degrelle&amp;author=I.%20Hue&amp;author=J.-L.%20Foulley&amp;publication_year=2007&amp;journal=Genet.%20Res.%20Camb.&amp;volume=89&amp;pages=19-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672307008646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672307008646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672307008646"> </span></div> </div></div></div></div></div><div content-id="DSR016C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dalgin</div> <div class="given-names">G.S.</div></div>,  <div class="name"><div class="surname">DeLisi</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Simple discriminant functions identify small sets of genes that distinguish cancer phenotype from normal</div>, <div class="source">Genome Inform.</div> , <div class="year">2005</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">245</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20discriminant%20functions%20identify%20small%20sets%20of%20genes%20that%20distinguish%20cancer%20phenotype%20from%20normal&amp;author=G.S.%20Dalgin&amp;author=C.%20DeLisi&amp;publication_year=2005&amp;journal=Genome%20Inform.&amp;volume=16&amp;pages=245-53" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16362927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grate</div> <div class="given-names">L.R.</div></div>. </div><div class="article-title">Many accurate small-discriminatory feature subsets exist in microarray transcript data: biomarker discovery</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">97</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Many%20accurate%20small-discriminatory%20feature%20subsets%20exist%20in%20microarray%20transcript%20data%3A%20biomarker%20discovery&amp;author=L.R.%20Grate&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-97" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-97" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-97"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15826317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Tsau</div> <div class="given-names">Y.W.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">C.H.</div></div>. </div><div class="article-title">Novel methods to identify biologically relevant genes for leukemia and prostate cancer from gene expression profiles</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">274</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20methods%20to%20identify%20biologically%20relevant%20genes%20for%20leukemia%20and%20prostate%20cancer%20from%20gene%20expression%20profiles&amp;author=A.H.%20Chen&amp;author=Y.W.%20Tsau&amp;author=C.H.%20Lin&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=274" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-274" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-274" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-274"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20433712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohl</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Development and validation of predictive molecular signatures</div>, <div class="source">Curr. Mol. Med.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">173</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20validation%20of%20predictive%20molecular%20signatures&amp;author=M.%20Kohl&amp;publication_year=2010&amp;journal=Curr.%20Mol.%20Med.&amp;volume=10&amp;pages=173-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2174/156652410790963376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2174%2F156652410790963376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2174%2F156652410790963376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20196729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Cutler</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Multiclass cancer classification and biomarker discovery using GA-based algorithms</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">2691</div>-<div class="lpage">7</div>) <div class="comment">doi:10.1093/bioinformatics/bti419</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiclass%20cancer%20classification%20and%20biomarker%20discovery%20using%20GA-based%20algorithms&amp;author=J.J.%20Liu&amp;author=G.%20Cutler&amp;author=W.%20Li&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2691-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti419"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15814557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deutsch</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Evolutionary algorithms for finding optimal gene sets in microarray prediction</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">45</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20algorithms%20for%20finding%20optimal%20gene%20sets%20in%20microarray%20prediction&amp;author=J.M.%20Deutsch&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=45-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/19.1.45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F19.1.45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F19.1.45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12499292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Niwa</div> <div class="given-names">H.</div></div>. </div><div class="article-title">How is pluripotency determined and maintained?</div>, <div class="source">Development</div> , <div class="year">2007</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">635</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20is%20pluripotency%20determined%20and%20maintained%3F&amp;author=H.%20Niwa&amp;publication_year=2007&amp;journal=Development&amp;volume=134&amp;pages=635-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.02787" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.02787" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.02787"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17215298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lengner</div> <div class="given-names">C.J.</div></div>. </div><div class="article-title">iPS cell technology in regenerative medicine</div>, <div class="source">Ann. N Y Acad. Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">1192</div> (pg. <div class="fpage">38</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=iPS%20cell%20technology%20in%20regenerative%20medicine&amp;author=C.J.%20Lengner&amp;publication_year=2010&amp;journal=Ann.%20N%20Y%20Acad.%20Sci.&amp;volume=1192&amp;pages=38-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1749-6632.2009.05213.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1749-6632.2009.05213.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1749-6632.2009.05213.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20392216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Ding</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Small molecules that modulate embryonic stem cell fate and somatic cell reprogramming</div>, <div class="source">Trends Pharmacol. Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">36</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20molecules%20that%20modulate%20embryonic%20stem%20cell%20fate%20and%20somatic%20cell%20reprogramming&amp;author=W.%20Li&amp;author=S.%20Ding&amp;publication_year=2010&amp;journal=Trends%20Pharmacol.%20Sci.&amp;volume=31&amp;pages=36-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tips.2009.10.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tips.2009.10.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tips.2009.10.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19896224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Masip</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Veiga</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Izpisua</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Simon</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Reprogramming with defined factors: from induced pluripotency to induced transdifferentiation</div>, <div class="source">Mol. Hum. Reprod.</div> , <div class="year">2010</div> <div class="comment">Epub ahead of print</div></div></div></div></div><div content-id="DSR016C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Huss</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tong</div> <div class="given-names">G.Q.</div></div>, et al. </div><div class="article-title">Resolution of cell fate decisions revealed by single-cell gene expression analysis from zygote to blastocyst</div>, <div class="source">Dev. Cell</div> , <div class="year">2010</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">675</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resolution%20of%20cell%20fate%20decisions%20revealed%20by%20single-cell%20gene%20expression%20analysis%20from%20zygote%20to%20blastocyst&amp;author=G.%20Guo&amp;author=M.%20Huss&amp;author=G.Q.%20Tong&amp;publication_year=2010&amp;journal=Dev.%20Cell&amp;volume=18&amp;pages=675-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.devcel.2010.02.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.devcel.2010.02.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.devcel.2010.02.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20412781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aiba</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nedorezov</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Piao</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Defining developmental potency and cell lineage trajectories by expression profiling of differentiating mouse embryonic stem cells</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">73</div>-<div class="lpage">80</div>) <div class="comment">doi:10.1093/dnares/dsn035</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20developmental%20potency%20and%20cell%20lineage%20trajectories%20by%20expression%20profiling%20of%20differentiating%20mouse%20embryonic%20stem%20cells&amp;author=K.%20Aiba&amp;author=T.%20Nedorezov&amp;author=Y.%20Piao&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=73-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19112179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nishiyama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Xin</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Sharov</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Uncovering early response of gene regulatory networks in ESCs by systematic induction of transcription factors</div>, <div class="source">Cell Stem Cell</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">420</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uncovering%20early%20response%20of%20gene%20regulatory%20networks%20in%20ESCs%20by%20systematic%20induction%20of%20transcription%20factors&amp;author=A.%20Nishiyama&amp;author=L.%20Xin&amp;author=A.A.%20Sharov&amp;publication_year=2009&amp;journal=Cell%20Stem%20Cell&amp;volume=5&amp;pages=420-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.stem.2009.07.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.stem.2009.07.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.stem.2009.07.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19796622" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goldberg</div> <div class="given-names">D.E.</div></div>. </div>, <div class="source">Genetic Algorithms in Search, Optimization, and Machine Learning</div> , <div class="year">1989</div><div class="publisher-name">Addison-Wesley Professional</div></div></div></div></div><div content-id="DSR016C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Bolstad</div> <div class="given-names">B.M.</div></div>,  <div class="name"><div class="surname">Collin</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Cope</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Hobbs</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Speed</div> <div class="given-names">T.P.</div></div>. </div><div class="article-title">Summaries of Affymetrix GeneChip probe level data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> pg. <div class="fpage">e15</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Summaries%20of%20Affymetrix%20GeneChip%20probe%20level%20data&amp;author=R.A.%20Irizarry&amp;author=B.M.%20Bolstad&amp;author=F.%20Collin&amp;author=L.M.%20Cope&amp;author=B.%20Hobbs&amp;author=T.P.%20Speed&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=e15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gng015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgng015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgng015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582260" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Hobbs</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Collin</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Exploration, normalization, and summaries of high density oligonucleotide array probe level data</div>, <div class="source">Biostatistics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">249</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploration%2C%20normalization%2C%20and%20summaries%20of%20high%20density%20oligonucleotide%20array%20probe%20level%20data&amp;author=R.A.%20Irizarry&amp;author=B.%20Hobbs&amp;author=F.%20Collin&amp;publication_year=2003&amp;journal=Biostatistics&amp;volume=4&amp;pages=249-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biostatistics/4.2.249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiostatistics%2F4.2.249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiostatistics%2F4.2.249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12925520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="collab">Affymetrix</div>, <div class="source">Affymetrix Power Tools (APT) Software Package</div> , <div class="year">2010</div></div></div></div></div><div content-id="DSR016C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bolstad</div> <div class="given-names">B.M.</div></div>,  <div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Astrand</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Speed</div> <div class="given-names">T.P.</div></div>. </div><div class="article-title">A comparison of normalization methods for high density oligonucleotide array data based on variance and bias</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">185</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20normalization%20methods%20for%20high%20density%20oligonucleotide%20array%20data%20based%20on%20variance%20and%20bias&amp;author=B.M.%20Bolstad&amp;author=R.A.%20Irizarry&amp;author=M.%20Astrand&amp;author=T.P.%20Speed&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=185-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F19.2.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F19.2.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12538238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheeler</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Church</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Edgar</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Database resources of the National Center for Biotechnology</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">28</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology&amp;author=D.L.%20Wheeler&amp;author=D.M.%20Church&amp;author=R.%20Edgar&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=28-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519941" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Storey</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Tibshirani</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Statistical significance for genomewide studies</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">9440</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20significance%20for%20genomewide%20studies&amp;author=J.D.%20Storey&amp;author=R.%20Tibshirani&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=9440-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1530509100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1530509100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1530509100"> </span></div> </div></div></div></div></div><div content-id="DSR016C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mendes</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Maia</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Glez-Peña</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fdez-Riverola</div> <div class="given-names">F.</div></div>. </div><div class="article-title">GECCO '07 Proceedings of the 9th annual conference on Genetic and evolutionary computation</div>, <div class="year">2007</div>(pg. <div class="fpage">415</div>-<div class="lpage">23</div>) <div class="comment">A platform for the selection of genes in DNA microarraydata using evolutionary algorithms</div></div></div></div></div><div content-id="DSR016C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hall</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Frank</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Holmes</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pfahringer</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Reutemann</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Witten</div> <div class="given-names">I.H.</div></div>, et al. </div><div class="article-title">The WEKA data mining software: an update</div>, <div class="source">SIGKDD Explorations</div> , <div class="year">2009</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">10</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20WEKA%20data%20mining%20software%3A%20an%20update&amp;author=M.%20Hall&amp;author=E.%20Frank&amp;author=G.%20Holmes&amp;author=B.%20Pfahringer&amp;author=P.%20Reutemann&amp;author=I.H.%20Witten&amp;publication_year=2009&amp;journal=SIGKDD%20Explorations&amp;volume=11&amp;pages=10-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1145/1656274" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1145%2F1656274" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1145%2F1656274"> </span></div> </div></div></div></div></div><div content-id="DSR016C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">John</div> <div class="given-names">G.H.</div></div>,  <div class="name"><div class="surname">Langley</div> <div class="given-names">P.</div></div>. </div>, <div class="year">1995</div>(pg. <div class="fpage">338</div>-<div class="lpage">45</div>) <div class="comment">Estimating continuous distributions in Bayesian classifiers, In Proceedings of the Eleventh Conference on Uncertainty in Artifical Intelligence</div></div></div></div></div><div content-id="DSR016C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quinlan</div> <div class="given-names">J.R.</div></div>. </div>, <div class="source">C4.5: Programms for Machine Learning</div> , <div class="year">1993</div><div class="publisher-name">Morgan Kaufmann Publishers</div></div></div></div></div><div content-id="DSR016C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Breiman</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Random Forest</div>, <div class="source">Mach. Learn.</div> , <div class="year">2001</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">5</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Random%20Forest&amp;author=L.%20Breiman&amp;publication_year=2001&amp;journal=Mach.%20Learn.&amp;volume=45&amp;pages=5-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1010933404324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1010933404324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1010933404324"> </span></div> </div></div></div></div></div><div content-id="DSR016C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aha</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kibler</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Albert</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Instance-based learning algorithms</div>, <div class="source">Mach. Learn.</div> , <div class="year">1991</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">37</div>-<div class="lpage">66</div>)</div></div></div></div><div content-id="DSR016C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Platt</div> <div class="given-names">J.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Schoelkopf</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Burges</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Smola</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Fast training of support vector machines using sequential minimal optimization</div>, <div class="source">Advances in Kernel Methods—Support Vector Learning</div> , <div class="year">1998</div><div class="publisher-name">MIT Press</div></div></div></div></div><div content-id="DSR016C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chang</div> <div class="given-names">C.-C.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">C.-J.</div></div>. </div>, <div class="source">LIBSVM – A Library for Support Vector Machines</div> , <div class="year">2001</div></div></div></div></div><div content-id="DSR016C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Livingston</div> <div class="given-names">F</div></div>. </div><div class="article-title">Implementation of Breiman's Random Forest Machine Learning Algorithm, In ECE591Q Machine Learning Conference</div>, <div class="year">2005</div></div></div></div></div><div content-id="DSR016C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">C.-L.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">C.-J.</div></div>. </div><div class="article-title">A GA-based feature selection and parameters optimizationfor support vector machines</div>, <div class="source">Expert Syst. Appl.</div> , <div class="year">2006</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">231</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20GA-based%20feature%20selection%20and%20parameters%20optimizationfor%20support%20vector%20machines&amp;author=C.-L.%20Huang&amp;author=C.-J.%20Wang&amp;publication_year=2006&amp;journal=Expert%20Syst.%20Appl.&amp;volume=31&amp;pages=231-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.eswa.2005.09.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.eswa.2005.09.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.eswa.2005.09.024"> </span></div> </div></div></div></div></div><div content-id="DSR016C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sywerda</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Third International Conference on Genetic Algorithms</div><div class="publisher-name">Morgan Kaufmann Publishers</div>(pg. <div class="fpage">2</div>-<div class="lpage">9</div>) <div class="comment">pp.</div></div></div></div></div><div content-id="DSR016C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Backes</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Keller</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kuentzer</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">GeneTrail—advanced gene set enrichment analysis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W186</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneTrail%E2%80%94advanced%20gene%20set%20enrichment%20analysis&amp;author=C.%20Backes&amp;author=A.%20Keller&amp;author=J.%20Kuentzer&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W186-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm323"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Som</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Harder</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Greber</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">The PluriNetWork: an in-silico representation of the network underlying pluripotency in mouse, and its applications</div>, <div class="source">PLoS ONE</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e15165</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20PluriNetWork%3A%20an%20in-silico%20representation%20of%20the%20network%20underlying%20pluripotency%20in%20mouse%2C%20and%20its%20applications&amp;author=A.%20Som&amp;author=C.%20Harder&amp;author=B.%20Greber&amp;publication_year=2010&amp;journal=PLoS%20ONE&amp;volume=5&amp;pages=e15165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0015165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0015165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0015165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21179244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">MacArthur</div> <div class="given-names">B.D.</div></div>,  <div class="name"><div class="surname">Ma'ayan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lemischka</div> <div class="given-names">I.R.</div></div>. </div><div class="article-title">Systems biology of stem cell fate and cellular reprogramming</div>, <div class="source">Nat. Rev. Mol. Cell Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">672</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systems%20biology%20of%20stem%20cell%20fate%20and%20cellular%20reprogramming&amp;author=B.D.%20MacArthur&amp;author=A.%20Ma'ayan&amp;author=I.R.%20Lemischka&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;volume=10&amp;pages=672-81" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19738627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muller</div> <div class="given-names">F.J.</div></div>,  <div class="name"><div class="surname">Laurent</div> <div class="given-names">L.C.</div></div>,  <div class="name"><div class="surname">Kostka</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Regulatory networks define phenotypic classes of human stem cell lines</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">455</div> (pg. <div class="fpage">401</div>-<div class="lpage">405</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20networks%20define%20phenotypic%20classes%20of%20human%20stem%20cell%20lines&amp;author=F.J.%20Muller&amp;author=L.C.%20Laurent&amp;author=D.%20Kostka&amp;publication_year=2008&amp;journal=Nature&amp;volume=455&amp;pages=401-405" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18724358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Newman</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Cooper</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">AutoSOME: a clustering method for identifying gene expression modules without prior knowledge of cluster number</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">117</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AutoSOME%3A%20a%20clustering%20method%20for%20identifying%20gene%20expression%20modules%20without%20prior%20knowledge%20of%20cluster%20number&amp;author=A.M.%20Newman&amp;author=J.B.%20Cooper&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20202218" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">More synergetic cooperation of Yamanaka factors in induced pluripotent stem cells than in embryonic stem cells</div>, <div class="source">Cell Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1127</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=More%20synergetic%20cooperation%20of%20Yamanaka%20factors%20in%20induced%20pluripotent%20stem%20cells%20than%20in%20embryonic%20stem%20cells&amp;author=J.%20Huang&amp;author=T.%20Chen&amp;author=X.%20Liu&amp;publication_year=2009&amp;journal=Cell%20Res.&amp;volume=19&amp;pages=1127-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/cr.2009.106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fcr.2009.106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fcr.2009.106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19736564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dreesen</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Brivanlou</div> <div class="given-names">A.H.</div></div>. </div><div class="article-title">Signaling pathways in cancer and embryonic stem cells</div>, <div class="source">Stem Cell Rev.</div> , <div class="year">2007</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">7</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signaling%20pathways%20in%20cancer%20and%20embryonic%20stem%20cells&amp;author=O.%20Dreesen&amp;author=A.H.%20Brivanlou&amp;publication_year=2007&amp;journal=Stem%20Cell%20Rev.&amp;volume=3&amp;pages=7-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12015-007-0004-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12015-007-0004-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12015-007-0004-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17873377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tian</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Han</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Molecular mechanisms involved in self-renewal and pluripotency of embryonic stem cells</div>, <div class="source">J. Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">211</div> (pg. <div class="fpage">279</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20involved%20in%20self-renewal%20and%20pluripotency%20of%20embryonic%20stem%20cells&amp;author=N.%20Liu&amp;author=M.%20Lu&amp;author=X.%20Tian&amp;author=Z.%20Han&amp;publication_year=2007&amp;journal=J.%20Cell%20Physiol.&amp;volume=211&amp;pages=279-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-4652" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-4652" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-4652"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17195167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steinwart</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Hush</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Scovel</div> <div class="given-names">C.</div></div>. </div><div class="article-title">An explicit description of the reproducing kernel hilbert spaces of Gaussian RBF kernels</div>, <div class="source">IEEE Trans. Info. Theory</div> , <div class="year">2006</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">4635</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20explicit%20description%20of%20the%20reproducing%20kernel%20hilbert%20spaces%20of%20Gaussian%20RBF%20kernels&amp;author=I.%20Steinwart&amp;author=D.%20Hush&amp;author=C.%20Scovel&amp;publication_year=2006&amp;journal=IEEE%20Trans.%20Info.%20Theory&amp;volume=52&amp;pages=4635-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TIT.2006.881713" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTIT.2006.881713" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTIT.2006.881713"> </span></div> </div></div></div></div></div><div content-id="DSR016C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hsu</div> <div class="given-names">C.-W.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">C.-C.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">C.-J.</div></div>. </div>, <div class="source">A Practical Guide to Support Vector Classification</div> , <div class="year">2003</div></div></div></div></div><div content-id="DSR016C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Statnikov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Aliferis</div> <div class="given-names">C.F.</div></div>. </div><div class="article-title">Analysis and computational dissection of molecular signature multiplicity</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e1000790</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20and%20computational%20dissection%20of%20molecular%20signature%20multiplicity&amp;author=A.%20Statnikov&amp;author=C.F.%20Aliferis&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=6&amp;pages=e1000790" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000790" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000790" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000790"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20502670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>. </div><div class="article-title">KEGG: kyoto encyclopedia of genes and genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">27</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=27-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kurth</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Pamminger</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hennings</div> <div class="given-names">J.C.</div></div>, et al. </div><div class="article-title">Mutations in FAM134B, encoding a newly identified Golgi protein, cause severe sensory and autonomic neuropathy</div>, <div class="source">Nat. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">1179</div>-<div class="lpage">81</div>) <div class="comment">doi:10.1038/ng.464</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutations%20in%20FAM134B%2C%20encoding%20a%20newly%20identified%20Golgi%20protein%2C%20cause%20severe%20sensory%20and%20autonomic%20neuropathy&amp;author=I.%20Kurth&amp;author=T.%20Pamminger&amp;author=J.C.%20Hennings&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=1179-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.464" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.464" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.464"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19838196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fredricsson</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Alkaline phosphatase in the Golgi substance of intestinal epithelium</div>, <div class="source">Exp. Cell Res.</div> , <div class="year">1956</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">63</div>-<div class="lpage">5</div>) <div class="comment">doi:0014-4827(56)90071-4</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alkaline%20phosphatase%20in%20the%20Golgi%20substance%20of%20intestinal%20epithelium&amp;author=B.%20Fredricsson&amp;publication_year=1956&amp;journal=Exp.%20Cell%20Res.&amp;volume=10&amp;pages=63-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-4827(56)90071-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-4827(56)90071-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-4827(56)90071-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13294113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">W.K.</div></div>,  <div class="name"><div class="surname">Chui</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Fatima</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Oncogenic properties of a novel gene JK-1 located in chromosome 5p and its overexpression in human esophageal squamous cell carcinoma</div>, <div class="source">Int. J. Mol. Med.</div> , <div class="year">2007</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">915</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oncogenic%20properties%20of%20a%20novel%20gene%20JK-1%20located%20in%20chromosome%205p%20and%20its%20overexpression%20in%20human%20esophageal%20squamous%20cell%20carcinoma&amp;author=W.K.%20Tang&amp;author=C.H.%20Chui&amp;author=S.%20Fatima&amp;publication_year=2007&amp;journal=Int.%20J.%20Mol.%20Med.&amp;volume=19&amp;pages=915-23" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17487424" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kasem</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lam</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Analysis of a novel JK-1 gene expression in benign and malignant colorectal tumors</div>, <div class="source">Virchows Arch.</div> , <div class="year">2010</div>, vol. <div class="volume">457</div> pg. <div class="fpage">165</div> </div></div></div></div><div content-id="DSR016C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">D'Amour</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Gage</div> <div class="given-names">F.H.</div></div>. </div><div class="article-title">Genetic and functional differences between multipotent neural and pluripotent embryonic stem cells</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> <div class="issue">Suppl. 1)</div>(pg. <div class="fpage">11866</div>-<div class="lpage">72</div>) <div class="comment">doi:10.1073/pnas.1834200100</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20functional%20differences%20between%20multipotent%20neural%20and%20pluripotent%20embryonic%20stem%20cells&amp;author=K.A.%20D'Amour&amp;author=F.H.%20Gage&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=11866-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1834200100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1834200100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1834200100"> </span></div> </div></div></div></div></div><div content-id="DSR016C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">Y.L.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">K.F.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">J.S.</div></div>, et al. </div><div class="article-title">Embryotrophic factor-3 from human oviductal cells affects the messenger RNA expression of mouse blastocyst</div>, <div class="source">Biol. Reprod.</div> , <div class="year">2003</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">375</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Embryotrophic%20factor-3%20from%20human%20oviductal%20cells%20affects%20the%20messenger%20RNA%20expression%20of%20mouse%20blastocyst&amp;author=Y.L.%20Lee&amp;author=K.F.%20Lee&amp;author=J.S.%20Xu&amp;publication_year=2003&amp;journal=Biol.%20Reprod.&amp;volume=68&amp;pages=375-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1095/biolreprod.102.007336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1095%2Fbiolreprod.102.007336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1095%2Fbiolreprod.102.007336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12533399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lyczak</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zweier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Group</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">The puromycin-sensitive aminopeptidase PAM-1 is required for meiotic exit and anteroposterior polarity in the one-cell Caenorhabditis elegans embryo</div>, <div class="source">Development</div> , <div class="year">2006</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">4281</div>-<div class="lpage">92</div>) <div class="comment">doi:10.1242/dev.02615</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20puromycin-sensitive%20aminopeptidase%20PAM-1%20is%20required%20for%20meiotic%20exit%20and%20anteroposterior%20polarity%20in%20the%20one-cell%20Caenorhabditis%20elegans%20embryo&amp;author=R.%20Lyczak&amp;author=L.%20Zweier&amp;author=T.%20Group&amp;publication_year=2006&amp;journal=Development&amp;volume=133&amp;pages=4281-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.02615" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.02615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.02615"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17021038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leeb</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wutz</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Ring1B is crucial for the regulation of developmental control genes and PRC1 proteins but not X inactivation in embryonic cells</div>, <div class="source">J. Cell Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">178</div> (pg. <div class="fpage">219</div>-<div class="lpage">29</div>) <div class="comment">doi:10.1083/jcb.200612127</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ring1B%20is%20crucial%20for%20the%20regulation%20of%20developmental%20control%20genes%20and%20PRC1%20proteins%20but%20not%20X%20inactivation%20in%20embryonic%20cells&amp;author=M.%20Leeb&amp;author=A.%20Wutz&amp;publication_year=2007&amp;journal=J.%20Cell%20Biol.&amp;volume=178&amp;pages=219-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.200612127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.200612127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.200612127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17620408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boue</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paramonov</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Barrero</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Izpisua Belmonte</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">Analysis of human and mouse reprogramming of somatic cells to induced pluripotent stem cells. What is in the plate?</div>, <div class="source">PLoS ONE</div> , <div class="year">2010</div>, vol. <div class="volume">5</div>  <div class="comment">doi:10.1371/journal.pone.0012664</div></div></div></div></div><div content-id="DSR016C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Botquin</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Hess</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fuhrmann</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">New POU dimer configuration mediates antagonistic control of an osteopontin preimplantation enhancer by Oct-4 and Sox-2</div>, <div class="source">Genes Dev.</div> , <div class="year">1998</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">2073</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20POU%20dimer%20configuration%20mediates%20antagonistic%20control%20of%20an%20osteopontin%20preimplantation%20enhancer%20by%20Oct-4%20and%20Sox-2&amp;author=V.%20Botquin&amp;author=H.%20Hess&amp;author=G.%20Fuhrmann&amp;publication_year=1998&amp;journal=Genes%20Dev.&amp;volume=12&amp;pages=2073-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.12.13.2073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.12.13.2073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.12.13.2073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9649510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haston</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Tung</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Reijo Pera</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Dazl functions in maintenance of pluripotency and genetic and epigenetic programs of differentiation in mouse primordial germ cells in vivo and in vitro</div>, <div class="source">PLoS ONE</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e5654</div>  <div class="comment">doi:10.1371/journal.pone.0005654</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dazl%20functions%20in%20maintenance%20of%20pluripotency%20and%20genetic%20and%20epigenetic%20programs%20of%20differentiation%20in%20mouse%20primordial%20germ%20cells%20in%20vivo%20and%20in%20vitro&amp;author=K.M.%20Haston&amp;author=J.Y.%20Tung&amp;author=R.A.%20Reijo%20Pera&amp;publication_year=2009&amp;journal=PLoS%20ONE&amp;volume=4&amp;pages=e5654" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0005654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0005654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0005654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19468308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sampath</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Pritchard</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Pabon</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">A hierarchical network controls protein translation during murine embryonic stem cell self-renewal and differentiation</div>, <div class="source">Cell Stem Cell</div> , <div class="year">2008</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">448</div>-<div class="lpage">60</div>) <div class="comment">doi:10.1016/j.stem.2008.03.013</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hierarchical%20network%20controls%20protein%20translation%20during%20murine%20embryonic%20stem%20cell%20self-renewal%20and%20differentiation&amp;author=P.%20Sampath&amp;author=D.K.%20Pritchard&amp;author=L.%20Pabon&amp;publication_year=2008&amp;journal=Cell%20Stem%20Cell&amp;volume=2&amp;pages=448-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.stem.2008.03.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.stem.2008.03.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.stem.2008.03.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18462695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salomonis</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Schlieve</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Pereira</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Alternative splicing regulates mouse embryonic stem cell pluripotency and differentiation</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">10514</div>-<div class="lpage">9</div>) <div class="comment">doi:10.1073/pnas.0912260107</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20regulates%20mouse%20embryonic%20stem%20cell%20pluripotency%20and%20differentiation&amp;author=N.%20Salomonis&amp;author=C.R.%20Schlieve&amp;author=L.%20Pereira&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=10514-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0912260107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0912260107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0912260107"> </span></div> </div></div></div></div></div><div content-id="DSR016C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chipperfield</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Melton</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">A gene regulatory network in mouse embryonic stem cells</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">16438</div>-<div class="lpage">43</div>) <div class="comment">doi:10.1073/pnas.0701014104</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20gene%20regulatory%20network%20in%20mouse%20embryonic%20stem%20cells&amp;author=Q.%20Zhou&amp;author=H.%20Chipperfield&amp;author=D.A.%20Melton&amp;author=W.H.%20Wong&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=16438-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0701014104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0701014104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0701014104"> </span></div> </div></div></div></div></div><div content-id="DSR016C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Levine</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Brivanlou</div> <div class="given-names">A.H.</div></div>. </div><div class="article-title">GDF3, a BMP inhibitor, regulates cell fate in stem cells and early embryos</div>, <div class="source">Development</div> , <div class="year">2006</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">209</div>-<div class="lpage">16</div>) <div class="comment">doi:10.1242/dev.02192</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GDF3%2C%20a%20BMP%20inhibitor%2C%20regulates%20cell%20fate%20in%20stem%20cells%20and%20early%20embryos&amp;author=A.J.%20Levine&amp;author=A.H.%20Brivanlou&amp;publication_year=2006&amp;journal=Development&amp;volume=133&amp;pages=209-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.02192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.02192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.02192"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16339188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clark</div> <div class="given-names">A.T.</div></div>,  <div class="name"><div class="surname">Rodriguez</div> <div class="given-names">R.T.</div></div>,  <div class="name"><div class="surname">Bodnar</div> <div class="given-names">M.S.</div></div>, et al. </div><div class="article-title">Human STELLAR, NANOG, and GDF3 genes are expressed in pluripotent cells and map to chromosome 12p13, a hotspot for teratocarcinoma</div>, <div class="source">Stem Cells</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">169</div>-<div class="lpage">79</div>) <div class="comment">doi:10.1634/stemcells.22-2-169</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20STELLAR%2C%20NANOG%2C%20and%20GDF3%20genes%20are%20expressed%20in%20pluripotent%20cells%20and%20map%20to%20chromosome%2012p13%2C%20a%20hotspot%20for%20teratocarcinoma&amp;author=A.T.%20Clark&amp;author=R.T.%20Rodriguez&amp;author=M.S.%20Bodnar&amp;publication_year=2004&amp;journal=Stem%20Cells&amp;volume=22&amp;pages=169-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1634/stemcells.22-2-169" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1634%2Fstemcells.22-2-169" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1634%2Fstemcells.22-2-169"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14990856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tarasov</div> <div class="given-names">K.V.</div></div>,  <div class="name"><div class="surname">Tarasova</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Tam</div> <div class="given-names">W.L.</div></div>, et al. </div><div class="article-title">B-MYB is essential for normal cell cycle progression and chromosomal stability of embryonic stem cells</div>, <div class="source">PLoS ONE</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e2478</div>  <div class="comment">doi:10.1371/journal.pone.0002478</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=B-MYB%20is%20essential%20for%20normal%20cell%20cycle%20progression%20and%20chromosomal%20stability%20of%20embryonic%20stem%20cells&amp;author=K.V.%20Tarasov&amp;author=Y.S.%20Tarasova&amp;author=W.L.%20Tam&amp;publication_year=2008&amp;journal=PLoS%20ONE&amp;volume=3&amp;pages=e2478" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0002478" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0002478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0002478"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18575582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Geng</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Pei</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Regulation of the pluripotency marker Rex-1 by Nanog and Sox2</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2006</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">23319</div>-<div class="lpage">25</div>) <div class="comment">doi:10.1074/jbc.M601811200</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20the%20pluripotency%20marker%20Rex-1%20by%20Nanog%20and%20Sox2&amp;author=W.%20Shi&amp;author=H.%20Wang&amp;author=G.%20Pan&amp;author=Y.%20Geng&amp;author=Y.%20Guo&amp;author=D.%20Pei&amp;publication_year=2006&amp;journal=J.%20Biol.%20Chem.&amp;volume=281&amp;pages=23319-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M601811200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M601811200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M601811200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16714766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alam</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Dress</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rehmsmeier</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fuellen</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Comparative homology agreement search: an effective combination of homology-search methods</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">13814</div>-<div class="lpage">9</div>) <div class="comment">doi:10.1073/pnas.0405612101</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20homology%20agreement%20search%3A%20an%20effective%20combination%20of%20homology-search%20methods&amp;author=I.%20Alam&amp;author=A.%20Dress&amp;author=M.%20Rehmsmeier&amp;author=G.%20Fuellen&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=13814-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0405612101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0405612101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0405612101"> </span></div> </div></div></div></div></div><div content-id="DSR016C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Szwagierczak</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bultmann</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Spada</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Leonhardt</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Sensitive enzymatic quantification of 5-hydroxymethylcytosine in genomic DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> pg. <div class="fpage">e181</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sensitive%20enzymatic%20quantification%20of%205-hydroxymethylcytosine%20in%20genomic%20DNA&amp;author=A.%20Szwagierczak&amp;author=S.%20Bultmann&amp;author=C.S.%20Schmidt&amp;author=F.%20Spada&amp;author=H.%20Leonhardt&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=e181" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq684"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20685817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tahiliani</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Koh</div> <div class="given-names">K.P.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">324</div> (pg. <div class="fpage">930</div>-<div class="lpage">5</div>) <div class="comment">doi:10.1126/science.1170116</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conversion%20of%205-methylcytosine%20to%205-hydroxymethylcytosine%20in%20mammalian%20DNA%20by%20MLL%20partner%20TET1&amp;author=M.%20Tahiliani&amp;author=K.P.%20Koh&amp;author=Y.%20Shen&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=930-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1170116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1170116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1170116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19372391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">D'Alessio</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Taranova</div> <div class="given-names">O.V.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sowers</div> <div class="given-names">L.C.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">466</div> (pg. <div class="fpage">1129</div>-<div class="lpage">33</div>) <div class="comment">doi:10.1038/nature09303</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20Tet%20proteins%20in%205mC%20to%205hmC%20conversion%2C%20ES-cell%20self-renewal%20and%20inner%20cell%20mass%20specification&amp;author=S.%20Ito&amp;author=A.C.%20D'Alessio&amp;author=O.V.%20Taranova&amp;author=K.%20Hong&amp;author=L.C.%20Sowers&amp;author=Y.%20Zhang&amp;publication_year=2010&amp;journal=Nature&amp;volume=466&amp;pages=1129-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20639862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">D'Alessio</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Ito</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Dual functions of Tet1 in transcriptional regulation in mouse embryonic stem cells</div>, <div class="source">Nature</div> , <div class="year">2011</div> <div class="comment">doi:10.1038/nature09934</div></div></div></div></div><div content-id="DSR016C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ficz</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Branco</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Seisenberger</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Dynamic regulation of 5-hydroxymethylcytosine in mouse ES cells and during differentiation</div>, <div class="source">Nature</div> , <div class="year">2011</div> <div class="comment">doi:10.1038/nature10008</div></div></div></div></div><div content-id="DSR016C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parisi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cozzuto</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Tarantino</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Direct targets of Klf5 transcription factor contribute to the maintenance of mouse embryonic stem cell undifferentiated state</div>, <div class="source">BMC Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">8</div> pg. <div class="fpage">128</div>  <div class="comment">doi:10.1186/1741-7007-8-128</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20targets%20of%20Klf5%20transcription%20factor%20contribute%20to%20the%20maintenance%20of%20mouse%20embryonic%20stem%20cell%20undifferentiated%20state&amp;author=S.%20Parisi&amp;author=L.%20Cozzuto&amp;author=C.%20Tarantino&amp;publication_year=2010&amp;journal=BMC%20Biol.&amp;volume=8&amp;pages=128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1741-7007-8-128" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1741-7007-8-128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1741-7007-8-128"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20875108" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C76" class="ref-content "><div class="label -label">76</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Costello</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Biondi</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Bikoff</div> <div class="given-names">E.K.</div></div>,  <div class="name"><div class="surname">Robertson</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">Smad4-dependent pathways control basement membrane deposition and endodermal cell migration at early stages of mouse development</div>, <div class="source">BMC Dev. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">54</div>  <div class="comment">doi:10.1186/1471-213X-9-54</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Smad4-dependent%20pathways%20control%20basement%20membrane%20deposition%20and%20endodermal%20cell%20migration%20at%20early%20stages%20of%20mouse%20development&amp;author=I.%20Costello&amp;author=C.A.%20Biondi&amp;author=J.M.%20Taylor&amp;author=E.K.%20Bikoff&amp;author=E.J.%20Robertson&amp;publication_year=2009&amp;journal=BMC%20Dev.%20Biol.&amp;volume=9&amp;pages=54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-213X-9-54" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-213X-9-54" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-213X-9-54"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19849841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C77" class="ref-content "><div class="label -label">77</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cinelli</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Casanova</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Uhlig</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Expression profiling in transgenic FVB/N embryonic stem cells overexpressing STAT3</div>, <div class="source">BMC Dev. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">57</div>  <div class="comment">doi:10.1186/1471-213X-8-57</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20profiling%20in%20transgenic%20FVB%2FN%20embryonic%20stem%20cells%20overexpressing%20STAT3&amp;author=P.%20Cinelli&amp;author=E.A.%20Casanova&amp;author=S.%20Uhlig&amp;publication_year=2008&amp;journal=BMC%20Dev.%20Biol.&amp;volume=8&amp;pages=57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-213X-8-57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-213X-8-57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-213X-8-57"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18500982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C78" class="ref-content "><div class="label -label">78</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bortvin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Eggan</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Incomplete reactivation of Oct4-related genes in mouse embryos cloned from somatic nuclei</div>, <div class="source">Development</div> , <div class="year">2003</div>, vol. <div class="volume">130</div> (pg. <div class="fpage">1673</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incomplete%20reactivation%20of%20Oct4-related%20genes%20in%20mouse%20embryos%20cloned%20from%20somatic%20nuclei&amp;author=A.%20Bortvin&amp;author=K.%20Eggan&amp;author=H.%20Skaletsky&amp;publication_year=2003&amp;journal=Development&amp;volume=130&amp;pages=1673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.00366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.00366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.00366"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12620990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C79" class="ref-content "><div class="label -label">79</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Facucho-Oliveira</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Alderson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Spikings</div> <div class="given-names">E.C.</div></div>,  <div class="name"><div class="surname">Egginton</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">St John</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">Mitochondrial DNA replication during differentiation of murine embryonic stem cells</div>, <div class="source">J. Cell Sci.</div> , <div class="year">2007</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">4025</div>-<div class="lpage">34</div>) <div class="comment">doi:10.1242/jcs.016972</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20replication%20during%20differentiation%20of%20murine%20embryonic%20stem%20cells&amp;author=J.M.%20Facucho-Oliveira&amp;author=J.%20Alderson&amp;author=E.C.%20Spikings&amp;author=S.%20Egginton&amp;author=J.C.%20St%20John&amp;publication_year=2007&amp;journal=J.%20Cell%20Sci.&amp;volume=120&amp;pages=4025-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jcs.016972" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjcs.016972" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjcs.016972"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17971411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C80" class="ref-content "><div class="label -label">80</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guzman-Ayala</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Mavrakis</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Goggolidou</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Norris</div> <div class="given-names">D.P.</div></div>,  <div class="name"><div class="surname">Episkopou</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Graded Smad2/3 activation is converted directly into levels of target gene expression in embryonic stem cells</div>, <div class="source">PLoS ONE</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e4268</div>  <div class="comment">doi:10.1371/journal.pone.0004268</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Graded%20Smad2%2F3%20activation%20is%20converted%20directly%20into%20levels%20of%20target%20gene%20expression%20in%20embryonic%20stem%20cells&amp;author=M.%20Guzman-Ayala&amp;author=K.L.%20Lee&amp;author=K.J.%20Mavrakis&amp;author=P.%20Goggolidou&amp;author=D.P.%20Norris&amp;author=V.%20Episkopou&amp;publication_year=2009&amp;journal=PLoS%20ONE&amp;volume=4&amp;pages=e4268" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0004268" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0004268" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0004268"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19172185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C81" class="ref-content "><div class="label -label">81</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirate</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Okamoto</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Canopy1, a novel regulator of FGF signaling around the midbrain-hindbrain boundary in zebrafish</div>, <div class="source">Curr. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">421</div>-<div class="lpage">7</div>) <div class="comment">doi:10.1016/j.cub.2006.01.055</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Canopy1%2C%20a%20novel%20regulator%20of%20FGF%20signaling%20around%20the%20midbrain-hindbrain%20boundary%20in%20zebrafish&amp;author=Y.%20Hirate&amp;author=H.%20Okamoto&amp;publication_year=2006&amp;journal=Curr.%20Biol.&amp;volume=16&amp;pages=421-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2006.01.055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2006.01.055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2006.01.055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16488878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C82" class="ref-content "><div class="label -label">82</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kamachi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Iwafuchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Okuda</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Takemoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Uchikawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kondoh</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Evolution of non-coding regulatory sequences involved in the developmental process: reflection of differential employment of paralogous genes as highlighted by Sox2 and group B1 Sox genes</div>, <div class="source">Proc. Jpn Acad. Ser. B Phys. Biol. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">55</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20non-coding%20regulatory%20sequences%20involved%20in%20the%20developmental%20process%3A%20reflection%20of%20differential%20employment%20of%20paralogous%20genes%20as%20highlighted%20by%20Sox2%20and%20group%20B1%20Sox%20genes&amp;author=Y.%20Kamachi&amp;author=M.%20Iwafuchi&amp;author=Y.%20Okuda&amp;author=T.%20Takemoto&amp;author=M.%20Uchikawa&amp;author=H.%20Kondoh&amp;publication_year=2009&amp;journal=Proc.%20Jpn%20Acad.%20Ser.%20B%20Phys.%20Biol.%20Sci.&amp;volume=85&amp;pages=55-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2183/pjab.85.55" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2183%2Fpjab.85.55" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2183%2Fpjab.85.55"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19212098" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C83" class="ref-content "><div class="label -label">83</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fuhrmann</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Chung</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">K.J.</div></div>, et al. </div><div class="article-title">Mouse germline restriction of Oct4 expression by germ cell nuclear factor</div>, <div class="source">Dev. Cell</div> , <div class="year">2001</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">377</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mouse%20germline%20restriction%20of%20Oct4%20expression%20by%20germ%20cell%20nuclear%20factor&amp;author=G.%20Fuhrmann&amp;author=A.C.%20Chung&amp;author=K.J.%20Jackson&amp;publication_year=2001&amp;journal=Dev.%20Cell&amp;volume=1&amp;pages=377-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1534-5807(01)00038-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1534-5807(01)00038-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1534-5807(01)00038-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11702949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C84" class="ref-content "><div class="label -label">84</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mitsui</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tokuzawa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The homeoprotein Nanog is required for maintenance of pluripotency in mouse epiblast and ES cells</div>, <div class="source">Cell</div> , <div class="year">2003</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">631</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20homeoprotein%20Nanog%20is%20required%20for%20maintenance%20of%20pluripotency%20in%20mouse%20epiblast%20and%20ES%20cells&amp;author=K.%20Mitsui&amp;author=Y.%20Tokuzawa&amp;author=H.%20Itoh&amp;publication_year=2003&amp;journal=Cell&amp;volume=113&amp;pages=631-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(03)00393-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(03)00393-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(03)00393-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12787504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C85" class="ref-content "><div class="label -label">85</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamanaka</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors</div>, <div class="source">Cell</div> , <div class="year">2006</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">663</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Induction%20of%20pluripotent%20stem%20cells%20from%20mouse%20embryonic%20and%20adult%20fibroblast%20cultures%20by%20defined%20factors&amp;author=K.%20Takahashi&amp;author=S.%20Yamanaka&amp;publication_year=2006&amp;journal=Cell&amp;volume=126&amp;pages=663-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.07.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.07.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.07.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16904174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C86" class="ref-content "><div class="label -label">86</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dyce</div> <div class="given-names">P.W.</div></div>,  <div class="name"><div class="surname">Toms</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Stem cells and germ cells: microRNA and gene expression signatures</div>, <div class="source">Histol. Histopathol.</div> , <div class="year">2010</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">505</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stem%20cells%20and%20germ%20cells%3A%20microRNA%20and%20gene%20expression%20signatures&amp;author=P.W.%20Dyce&amp;author=D.%20Toms&amp;author=J.%20Li&amp;publication_year=2010&amp;journal=Histol.%20Histopathol.&amp;volume=25&amp;pages=505-13" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20183803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C87" class="ref-content "><div class="label -label">87</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mansergh</div> <div class="given-names">F.C.</div></div>,  <div class="name"><div class="surname">Daly</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Hurley</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Wride</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Hunter</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Evans</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Gene expression profiles during early differentiation of mouse embryonic stem cells</div>, <div class="source">BMC Dev. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">5</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20profiles%20during%20early%20differentiation%20of%20mouse%20embryonic%20stem%20cells&amp;author=F.C.%20Mansergh&amp;author=C.S.%20Daly&amp;author=A.L.%20Hurley&amp;author=M.A.%20Wride&amp;author=S.M.%20Hunter&amp;author=M.J.%20Evans&amp;publication_year=2009&amp;journal=BMC%20Dev.%20Biol.&amp;volume=9&amp;pages=5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-213X-9-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-213X-9-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-213X-9-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19134196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C88" class="ref-content "><div class="label -label">88</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graumann</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hubner</div> <div class="given-names">N.C.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.B.</div></div>, et al. </div><div class="article-title">Stable isotope labeling by amino acids in cell culture (SILAC) and proteome quantitation of mouse embryonic stem cells to a depth of 5111 proteins</div>, <div class="source">Mol Cell Proteomics</div> , <div class="year">2008</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">672</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stable%20isotope%20labeling%20by%20amino%20acids%20in%20cell%20culture%20(SILAC)%20and%20proteome%20quantitation%20of%20mouse%20embryonic%20stem%20cells%20to%20a%20depth%20of%205111%20proteins&amp;author=J.%20Graumann&amp;author=N.C.%20Hubner&amp;author=J.B.%20Kim&amp;publication_year=2008&amp;journal=Mol%20Cell%20Proteomics&amp;volume=7&amp;pages=672-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/mcp.M700460-MCP200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fmcp.M700460-MCP200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fmcp.M700460-MCP200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18045802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C89" class="ref-content "><div class="label -label">89</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pfeiffer</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Siatkowski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Paudel</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Proteomic analysis of mouse oocytes reveals 28 candidate factors of the “reprogrammome”</div>, <div class="source">J. Proteome Res.</div> , <div class="year">2011</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">2140</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteomic%20analysis%20of%20mouse%20oocytes%20reveals%2028%20candidate%20factors%20of%20the%20%E2%80%9Creprogrammome%E2%80%9D&amp;author=M.J.%20Pfeiffer&amp;author=M.%20Siatkowski&amp;author=Y.%20Paudel&amp;publication_year=2011&amp;journal=J.%20Proteome%20Res.&amp;volume=10&amp;pages=2140-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/pr100706k" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fpr100706k" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fpr100706k"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21344949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C90" class="ref-content "><div class="label -label">90</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fortunel</div> <div class="given-names">N.O.</div></div>,  <div class="name"><div class="surname">Otu</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Ng</div> <div class="given-names">H.H.</div></div>, et al. </div><div class="article-title">Comment on "'Stemness': transcriptional profiling of embryonic and adult stem cells" and "a stem cell molecular signature"</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> pg. <div class="fpage">393</div>  <div class="comment">author reply 393, doi:10.1126/science.1086384</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comment%20on%20%22'Stemness'%3A%20transcriptional%20profiling%20of%20embryonic%20and%20adult%20stem%20cells%22%20and%20%22a%20stem%20cell%20molecular%20signature%22&amp;author=N.O.%20Fortunel&amp;author=H.H.%20Otu&amp;author=H.H.%20Ng&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=393" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1086384" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1086384" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1086384"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14563990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C91" class="ref-content "><div class="label -label">91</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wong</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ridky</div> <div class="given-names">T.W.</div></div>,  <div class="name"><div class="surname">Cassarino</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Segal</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">H.Y.</div></div>. </div><div class="article-title">Module map of stem cell genes guides creation of epithelial cancer stem cells</div>, <div class="source">Cell Stem Cell</div> , <div class="year">2008</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">333</div>-<div class="lpage">44</div>) <div class="comment">doi:10.1016/j.stem.2008.02.009</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Module%20map%20of%20stem%20cell%20genes%20guides%20creation%20of%20epithelial%20cancer%20stem%20cells&amp;author=D.J.%20Wong&amp;author=H.%20Liu&amp;author=T.W.%20Ridky&amp;author=D.%20Cassarino&amp;author=E.%20Segal&amp;author=H.Y.%20Chang&amp;publication_year=2008&amp;journal=Cell%20Stem%20Cell&amp;volume=2&amp;pages=333-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.stem.2008.02.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.stem.2008.02.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.stem.2008.02.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18397753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C92" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C92" class="ref-content "><div class="label -label">92</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Apparently low reproducibility of true differential expression discoveries in microarray studies</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">2057</div>-<div class="lpage">63</div>) <div class="comment">doi:10.1093/bioinformatics/btn365</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Apparently%20low%20reproducibility%20of%20true%20differential%20expression%20discoveries%20in%20microarray%20studies&amp;author=M.%20Zhang&amp;author=C.%20Yao&amp;author=Z.%20Guo&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=2057-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn365" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn365" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn365"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18632747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C93" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C93" class="ref-content "><div class="label -label">93</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gong</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Extracting consistent knowledge from highly inconsistent cancer gene data sources</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">76</div>  <div class="comment">doi:10.1186/1471-2105-11-76</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extracting%20consistent%20knowledge%20from%20highly%20inconsistent%20cancer%20gene%20data%20sources&amp;author=X.%20Gong&amp;author=R.%20Wu&amp;author=Y.%20Zhang&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-76" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-76" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-76"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20137077" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C94" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C94" class="ref-content "><div class="label -label">94</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bruce</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Gardiner</div> <div class="given-names">B.B.</div></div>,  <div class="name"><div class="surname">Burke</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Gongora</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Grimmond</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Perkins</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">Dynamic transcription programs during ES cell differentiation towards mesoderm in serum versus serum-freeBMP4 culture</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">365</div>  <div class="comment">doi:10.1186/1471-2164-8-365</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20transcription%20programs%20during%20ES%20cell%20differentiation%20towards%20mesoderm%20in%20serum%20versus%20serum-freeBMP4%20culture&amp;author=S.J.%20Bruce&amp;author=B.B.%20Gardiner&amp;author=L.J.%20Burke&amp;author=M.M.%20Gongora&amp;author=S.M.%20Grimmond&amp;author=A.C.%20Perkins&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=365" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-365" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-365" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-365"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17925037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C95" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C95" class="ref-content "><div class="label -label">95</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuke</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Glonek</div> <div class="given-names">G.F.</div></div>,  <div class="name"><div class="surname">Solomon</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">Gene profiling for determining pluripotent genes in a time course microarray experiment</div>, <div class="source">Biostatistics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">80</div>-<div class="lpage">93</div>) <div class="comment">doi:10.1093/biostatistics/kxn017</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20profiling%20for%20determining%20pluripotent%20genes%20in%20a%20time%20course%20microarray%20experiment&amp;author=J.%20Tuke&amp;author=G.F.%20Glonek&amp;author=P.J.%20Solomon&amp;publication_year=2009&amp;journal=Biostatistics&amp;volume=10&amp;pages=80-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biostatistics/kxn017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiostatistics%2Fkxn017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiostatistics%2Fkxn017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18562347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C96" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C96" class="ref-content "><div class="label -label">96</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hume</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Summers</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Raza</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Baillie</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Freeman</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Functional clustering and lineage markers: insights into cellular differentiation and gene function from large-scale microarray studies of purified primary cell populations</div>, <div class="source">Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">328</div>-<div class="lpage">38</div>) <div class="comment">doi:10.1016/j.ygeno.2010.03.002</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20clustering%20and%20lineage%20markers%3A%20insights%20into%20cellular%20differentiation%20and%20gene%20function%20from%20large-scale%20microarray%20studies%20of%20purified%20primary%20cell%20populations&amp;author=D.A.%20Hume&amp;author=K.M.%20Summers&amp;author=S.%20Raza&amp;author=J.K.%20Baillie&amp;author=T.C.%20Freeman&amp;publication_year=2010&amp;journal=Genomics&amp;volume=95&amp;pages=328-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2010.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2010.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2010.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20211243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C97" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C97" class="ref-content "><div class="label -label">97</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steinfath</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gärtner</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lisec</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Prediction of hybrid biomass in Arabidopsis thaliana by selected parental SNP and metabolic markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">239</div>-<div class="lpage">47</div>) <div class="comment">doi:10.1007/s00122-009-1191-2</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20hybrid%20biomass%20in%20Arabidopsis%20thaliana%20by%20selected%20parental%20SNP%20and%20metabolic%20markers&amp;author=M.%20Steinfath&amp;author=T.%20G%C3%A4rtner&amp;author=J.%20Lisec&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=239-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1191-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1191-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1191-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19911163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR016C98" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C98" class="ref-content "><div class="label -label">98</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">T.C.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">R.-S.</div></div>,  <div class="name"><div class="surname">Chao</div> <div class="given-names">Y.-T.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">S.-Y.</div></div>. </div><div class="article-title">PRICAI'06 Proceedings of the 9th Pacific Rim International Conference on Artificial Intelligence</div>, <div class="year">2006</div>(pg. <div class="fpage">1037</div>-<div class="lpage">41</div>) <div class="comment">Pattern classification in DNA microarray data of multiple tumor types</div></div></div></div></div><div content-id="DSR016C99" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C99" class="ref-content "><div class="label -label">99</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Küçükural</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yeniterzi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Yeniterzi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sezerman</div> <div class="given-names">O.</div></div>. </div><div class="article-title">GECCO '07 Proceedings of the 9th annual Conference on Genetic and Evolutionary Computation</div>, <div class="year">2007</div>(pg. <div class="fpage">401</div>-<div class="lpage">6</div>) <div class="comment">Evolutionary selection of minimum number of features for classification of gene expression data using genetic algorithms</div></div></div></div></div><div content-id="DSR016C100" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C100" class="ref-content "><div class="label -label">100</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gan</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Chow</div> <div class="given-names">T.W.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Effective gene selection method using bayesian discriminant based criterion and genetic algorithms</div>, <div class="source">J. Signal Proce. Syst.</div> , <div class="year">2008</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">293</div>-<div class="lpage">304</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effective%20gene%20selection%20method%20using%20bayesian%20discriminant%20based%20criterion%20and%20genetic%20algorithms&amp;author=Z.%20Gan&amp;author=T.W.%20Chow&amp;author=D.%20Huang&amp;publication_year=2008&amp;journal=J.%20Signal%20Proce.%20Syst.&amp;volume=50&amp;pages=293-304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11265-007-0120-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11265-007-0120-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11265-007-0120-3"> </span></div> </div></div></div></div></div><div content-id="DSR016C101" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C101" class="ref-content "><div class="label -label">101</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>. </div><div class="article-title">An agent-based hybrid system for microarray data analysis</div>, <div class="source">Intell. Syst.</div> , <div class="year">2009</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">53</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20agent-based%20hybrid%20system%20for%20microarray%20data%20analysis&amp;author=Z.%20Zhang&amp;author=P.%20Yang&amp;author=X.%20Wu&amp;author=C.%20Zhang&amp;publication_year=2009&amp;journal=Intell.%20Syst.&amp;volume=24&amp;pages=53-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/MIS.2009.92" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FMIS.2009.92" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FMIS.2009.92"> </span></div> </div></div></div></div></div><div content-id="DSR016C102" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR016C102" class="ref-content "><div class="label -label">102</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannas</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Dessì</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Pes</div> <div class="given-names">B.</div></div>. </div>, <div class="year">2010</div>(pg. <div class="fpage">297</div>-<div class="lpage">307</div>) <div class="comment">A filter-based evolutionary approach for selecting features in high-dimensional micro-array data, In 6th International Conference on Intelligent Information Processing</div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=501892&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr016-dsr016supp_table3.xls?Expires=2147483647&amp;Signature=DHsONCidiwLFwYgxPxc1LZwqDHQauqbrEiruBFcdt56KTDvmKPvGlEwbIVzqBcZQvBoUe-xusvewwFZCaa1E1G0G3H4bLG-q3cerZfiPy5zlZbYKlanMTDdNKQ1Zm-4NS7LONahh80RWnDUXFrxn-DvSZixg22TWLKmz3gakTcIcNIYnCmQZAftab-yDDJbjn9Buz6YNNM9jxqWMELGvCZhHttFxbwbqMN1jgyBiikvYoLCSsKBvbTltY8Et6v4jw-~7uVlFAfb1szUH87PF8WexjJHMnLZ-1u89z4Vh6Tugs2vRw4sOxHqQLNocSHljB2QgvzOS2jNh8VyfY6JheA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr016-dsr016supp_table2.xls?Expires=2147483647&amp;Signature=q~VEMqXQbFKOsY8hrMKVQOXp1CfJw2oBQim3dijicZKzxX35NTu1dopM3LHU2Jx5UwoZwUJNuvFvcZIzUCU-UiKeFqmr8G8ogXhFMa0wMoMYk5rKuqv9AHRoGS00jXbSPVoLhfa2GgEIJGDhXXAvei3~-9kDUMc31pKxd0aZ0xIGB0IWskPmzVzMeZ8zcDSeHlmK3wfBUV7BZ90o3nDTU2ar4gWr2nIudRGJAQzZtuEm4rgZASXecxPWxNdnouuJ01yV37PieEy2poz~2aqFOOLR93jx8zzy1R3y0E2~0cktkw0M01zkifhKWYbkdqn2XlYfkGGAYEKIzoKOJPvUoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr016-dsr016supp_table1.xls?Expires=2147483647&amp;Signature=X3lV0Cj5MHJ2Bma6sCs40kl9GpOTcQkR-KFveoEdOi~AW-GKG1IqjcAbhVFmQFkmr6bP6PrLIy-nlgcjhoGkfCYM8JT9CIOEnl8OGmzIze-C3DZY4ZVbilwWzPaw-T4b5bfVHDOqAdggMQzoCyCpfmfDH5FuYo7nQasyGFMpW7hF-O5XXf2ayAOhQoia8FmWeQP2RHPBf~cepK4wW6OXj56kdiqi2rRvL2m1VtjfkTNqcwsT43JLGMMqwY82F5eiD53qqs7HgKX5uQTEluGnRa5ULHDJUAYPTDMWUPf3gBTInSoP9A9KQY4brAzKaZwsyQ7y-wIHYzeuz1re4cheYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr016-dsr016supp3.pdf?Expires=2147483647&amp;Signature=4zMAUqKFvYe6syswrAgPBGET68E2ssqxRaP5cnY3yR-xAtT-k32L0l9eJDl~FlltLJY5VeJTguvSeDZlP2QJKEnVwcVJ-lf9LgYynQ124SetW6uLjsNutMRsGLX2yvU6yIR259jZ2raDoDXXHloL0fLBdfiw9BH3GKKZL0BKlJ~YERHbVOVAbo1U-cVQyyDeXllVEo-c-aYQt~f9Mi~dRTqyQn0SOATi~EkqTjnabc5ItL5wrQk-pII9tomVxKflyLbQadlimZg~1v8Q6v5XntTotnf8YzDzkUV7xX7NXw7i9EbOaDaqz6MQ~EY3HZD8IYU0X4mQMQeL7T2uE~~KNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr016-dsr016supp2.pdf?Expires=2147483647&amp;Signature=YPdL5QMh8UcLfSvV03NrOLhm7fgDFCn9i5SIPaNQE5HnC421LQCBJR94ilF0iQBEYYF0ji57dTyqEi85526lceGZv3-OY8dxaa5Bkctszerj~9bvylCb5i-HmTqX2AX9QYKxo0S1MT8Xw8cJI-~~rT4adCSaushyjJESDE7dj7-Zj7X3cIPecY1fZaRiczc0iEQzrmLXAdhjAdHQBKUHwF4ZwG3wlYEmUODQpnaZqVk0o4y8e2mGgABvG3vtAnkjOR~nCgPSNtkSbuMWR0ZbFNZU-wWcPZzUf6VTZ6ajw92eeGKfBieUU9x~Jju9j2VCXPkoV4Q9ASD5suoQgwDgRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr016/2/dsr016-dsr016supp1.pdf?Expires=2147483647&amp;Signature=3zTW5s-cbMYAVy~Cri-UFSv7cNEl1fdWHcU1eUxu0HwmKWzKkKHpC2U7P0BteqcVvynL3A19FMs2PVTQtfvKMtXQJkPlX3VsIjEGmRbR3f60aDYPac2KNHMqBEMQKJ3GFMDbSAn2c8N8d8PZLtEF6VFpYJ13NgjO2KvtewH0J7-HMgwB8Jjz9Swa3RnC3Ac0vgCs3Sn079D-J1NDPgz3Ka0lpzOkocXBs5ojyP3Ghm0wd-wzoIFS8ksfbJ2op~Hm~I9FdwlO9Zilj3L~mhuFwTlpl45TJNJ5wzCX81SeZiMu4UXCFHSr7nR8DMyJTV4o49PtAxE7Fy7TbWUQMfKnEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">285</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr016"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="501892"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f4%2f233%2f501892">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000294936100004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F4%2F233%2F501892" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/501892'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Learning Biomarkers of Pluripotent Stem Cells in Mouse" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr016");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f233%2f501892">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

