

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/6/571/3868562","@type":"ScholarlyArticle","name":"The evolutionary history of  Saccharomyces  species inferred from completed mitochondrial genomes and revision in the ‘yeast mitochondrial genetic code’","datePublished":"2017-06-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2017-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx026","keywords":["mitochondrial DNA","yeasts","genetic code","introns","phylogeny"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/6/571/3868562","author":[{"name":"Sulo, Pavol","affiliation":"Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia","@type":"Person"},{"name":"Szabóová, Dana","affiliation":"Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia","@type":"Person"},{"name":"Bielik, Peter","affiliation":"Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia","@type":"Person"},{"name":"Poláková, Silvia","affiliation":"Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia","@type":"Person"},{"name":"Šoltys, Katarína","affiliation":"Comenius University Science Park, Bratislava 841 04, Slovakia","@type":"Person"},{"name":"Jatzová, Katarína","affiliation":"Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia","@type":"Person"},{"name":"Szemes, Tomáš","affiliation":"Comenius University Science Park, Bratislava 841 04, Slovakia  Department of Molecular Biology, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia  Geneton s.r.o., Galvaniho 7, Bratislava 821 04, Slovakia","@type":"Person"}],"description":"Abstract.  The yeast Saccharomyces are widely used to test ecological and evolutionary hypotheses. A large number of nuclear genomic DNA sequences are available","pageStart":"571","pageEnd":"583","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The evolutionary history of  Saccharomyces  species inferred from completed mitochondrial genomes and revision in the ‘yeast mitochondrial genetic code’","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="evolutionary history of  Saccharomyces  species inferred from completed mitochondrial genomes and revision in the ‘yeast mitochondrial genetic code’" />
    <meta property="og:description" content="Abstract.  The yeast Saccharomyces are widely used to test ecological and evolutionary hypotheses. A large number of nuclear genomic DNA sequences are available" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/6/571/3868562" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-06-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sulo, Pavol" /><meta name="citation_author_institution" content="Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia" /><meta name="citation_author" content="Szabóová, Dana" /><meta name="citation_author_institution" content="Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia" /><meta name="citation_author" content="Bielik, Peter" /><meta name="citation_author_institution" content="Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia" /><meta name="citation_author" content="Poláková, Silvia" /><meta name="citation_author_institution" content="Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia" /><meta name="citation_author" content="Šoltys, Katarína" /><meta name="citation_author_institution" content="Comenius University Science Park, Bratislava 841 04, Slovakia" /><meta name="citation_author" content="Jatzová, Katarína" /><meta name="citation_author_institution" content="Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia" /><meta name="citation_author" content="Szemes, Tomáš" /><meta name="citation_author_institution" content="Comenius University Science Park, Bratislava 841 04, Slovakia" /><meta name="citation_author_institution" content="Department of Molecular Biology, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia" /><meta name="citation_author_institution" content="Geneton s.r.o., Galvaniho 7, Bratislava 821 04, Slovakia" /><meta name="citation_title" content="The evolutionary history of Saccharomyces species inferred from completed mitochondrial genomes and revision in the ‘yeast mitochondrial genetic code’" /><meta name="citation_firstpage" content="571" /><meta name="citation_lastpage" content="583" /><meta name="citation_doi" content="10.1093/dnares/dsx026" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2017/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Elsevier, Amsterdam, NL;  citation_title=The Yeasts, A Taxonomic Study;  citation_year=2011;  citation_pages=733-46; " /><meta name="citation_reference" content="citation_title=Saccharomyces sensu stricto as a model system for evolution and ecology,;  citation_journal_title=Trends Ecol. Evol;  citation_year=2008;  citation_volume=23;  citation_pages=494-501; " /><meta name="citation_reference" content="citation_title=Interspecies hybridization and recombination in Saccharomyces wine yeasts,;  citation_journal_title=FEMS Yeast Res;  citation_year=2008;  citation_volume=8;  citation_pages=996-1007; " /><meta name="citation_reference" content="citation_title=Genomic insights into the Saccharomyces sensu stricto complex,;  citation_journal_title=Genetics;  citation_year=2015;  citation_volume=199;  citation_pages=281-91; " /><meta name="citation_reference" content="citation_title=The genome sequence of Saccharomyces eubayanus and the domestication of lager-brewing yeasts,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=2818-31; " /><meta name="citation_reference" content="citation_title=The complete sequence of the mitochondrial genome of Saccharomyces cerevisiae,;  citation_journal_title=FEBS Lett;  citation_year=1998;  citation_volume=440;  citation_pages=325-31; " /><meta name="citation_reference" content="citation_title=Mitochondrial genome evolution in yeasts: an all-encompassing view,;  citation_journal_title=FEMS Yeast Res;  citation_year=2015;  citation_volume=15;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements,;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=423;  citation_pages=241-54; " /><meta name="citation_reference" content="citation_title=High quality de novo sequencing and assembly of the Saccharomyces arboricolus genome,;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Genomics and the making of yeast biodiversity,;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2015;  citation_volume=35;  citation_pages=100-9; " /><meta name="citation_reference" content="citation_title=A complete sequence of Saccharomyces paradoxus mitochondrial genome that restores the respiration in S. cerevisiae,;  citation_journal_title=FEMS Yeast Res;  citation_year=2012;  citation_volume=12;  citation_pages=819-30; " /><meta name="citation_reference" content="citation_title=Genome sequence of the lager brewing yeast, an interspecies hybrid,;  citation_journal_title=DNA Res;  citation_year=2009;  citation_volume=16;  citation_pages=115-29; " /><meta name="citation_reference" content="citation_title=Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=67-80; " /><meta name="citation_reference" content="citation_title=Comparison of next generation sequencing systems,;  citation_journal_title=J. Biomed. Biotechnol;  citation_year=2012;  citation_volume=2012;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers,;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast,;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=1496-504; " /><meta name="citation_reference" content="citation_title=A high-definition view of functional genetic variation from natural yeast genomes;  citation_journal_title=Mol. Biol. Evol;  citation_year=2014;  citation_volume=31;  citation_pages=872-88; " /><meta name="citation_reference" content="citation_title=Massive programmed translational jumping in mitochondria,;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2014;  citation_volume=111;  citation_pages=5926-31; " /><meta name="citation_reference" content="citation_title=A dynamic mobile DNA family in the yeast mitochondrial genome,;  citation_journal_title=G3 (Bethesda);  citation_year=2015;  citation_volume=5;  citation_pages=1273-82; " /><meta name="citation_reference" content="citation_title=Population structure of mitochondrial genomes in Saccharomyces cerevisiae,;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=The 100-genomes strains, an S. cerevisiae resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen,;  citation_journal_title=Genome Res;  citation_year=2015;  citation_volume=25;  citation_pages=762-74; " /><meta name="citation_reference" content="citation_title=Post-zygotic sterility and cytonuclear compatibility limits in S. cerevisiae xenomitochondrial cybrids;  citation_journal_title=Front. Genet;  citation_year=2015;  citation_volume=5;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=A rapid miniprep method for the preparation of yeast mitochondrial DNA,;  citation_journal_title=Nucleic Acids Res;  citation_year=1991;  citation_volume=19;  citation_pages=185" /><meta name="citation_reference" content="citation_title=Respiratory-competent yeast mitochondrial DNAs generated by deleting intergenic region,;  citation_journal_title=Gene;  citation_year=1989;  citation_volume=81;  citation_pages=165-8; " /><meta name="citation_reference" content="citation_title=Structure and genetic stability of mitochondrial genomes vary among yeasts of the genus Saccharomyces;  citation_journal_title=Int. J. Syst. Bacteriol;  citation_year=1998;  citation_volume=48;  citation_pages=1015-24; " /><meta name="citation_reference" content="citation_title=Microbe domestication and the identification of the wild genetic stock of lager-brewing yeast,;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2011;  citation_volume=108;  citation_pages=14539-44; " /><meta name="citation_reference" content="citation_title=Mitochondrial introns: a critical view,;  citation_journal_title=Trends Genet;  citation_year=2007;  citation_volume=23;  citation_pages=119-25; " /><meta name="citation_reference" content="citation_title=Introns as mobile genetic elements,;  citation_journal_title=Annu. Rev. Biochem;  citation_year=1993;  citation_volume=62;  citation_pages=587-622; " /><meta name="citation_reference" content="citation_title=The primary structure of the mitochondrial genome of Saccharomyces cerevisiae – a review;  citation_journal_title=Gene;  citation_year=1986b;  citation_volume=47;  citation_pages=155-77; " /><meta name="citation_reference" content="citation_title=A comparison of thermodynamic foldings with comparatively derived structures of 16S and 16S-like rRNAs;  citation_journal_title=RNA;  citation_year=1995;  citation_volume=1;  citation_pages=559-74; " /><meta name="citation_reference" content="citation_title=An analysis of large rRNA sequences folded by a thermodynamic method,;  citation_journal_title=Fold. Des;  citation_year=1996;  citation_volume=1;  citation_pages=419-30; " /><meta name="citation_reference" content="citation_title=Mitochondrial RNase P RNAs in ascomycete fungi: lineage-specific variations in RNA secondary structure,;  citation_journal_title=RNA;  citation_year=2003;  citation_volume=9;  citation_pages=1073-83; " /><meta name="citation_reference" content="citation_title=GOBASE – a database of organelle and bacterial genome information;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=D697-99; " /><meta name="citation_reference" content="CLC Genomics Workbench 9.5.2. https://www.qiagenbioinformatics.com/ " /><meta name="citation_reference" content="citation_title=Phylogenies from molecular sequences: inference and reliability,;  citation_journal_title=Annu, Rev. Genet;  citation_year=1988;  citation_volume=22;  citation_pages=521-65; " /><meta name="citation_reference" content="citation_title=Evidence for domesticated and wild populations of Saccharomyces cerevisiae,;  citation_journal_title=PLoS Genet;  citation_year=2005;  citation_volume=1;  citation_pages=66-71; " /><meta name="citation_reference" content="citation_title=Population structure and gene evolution in Saccharomyces cerevisiae,;  citation_journal_title=FEMS Yeast Res;  citation_year=2006;  citation_volume=6;  citation_pages=702-15; " /><meta name="citation_reference" content="citation_title=Application of multi locus sequence typing to the analysis of the biodiversity of indigenous Saccharomyces cerevisiae wine yeasts from Lebanon,;  citation_journal_title=J. Appl. Microbiol;  citation_year=2006;  citation_volume=100;  citation_pages=699-711; " /><meta name="citation_reference" content="citation_title=Surprisingly diverged populations of Saccharomyces cerevisiae in natural environments remote from human activity,;  citation_journal_title=Mol. Ecol;  citation_year=2012;  citation_volume=21;  citation_pages=5404-17; " /><meta name="citation_reference" content="citation_title=The awesome power of yeast evolutionary genetics: new genome sequences and strain resources for the Saccharomyces sensu stricto genus;  citation_journal_title=G3 (Bethesda);  citation_year=2011;  citation_volume=1;  citation_pages=11-25; " /><meta name="citation_reference" content="citation_title=The structure of the small mitochondrial DNA of Kluyveromyces thermotolerans is likely to reflect the ancestral gene order in fungi,;  citation_journal_title=J. Mol. Evol;  citation_year=1994;  citation_volume=38;  citation_pages=593-601; " /><meta name="citation_reference" content="citation_publisher=Springer-Verlag, Berlin, Germany;  citation_title=Organelle Genetics. Evolution of Organelle Genomes and Gene Expression;  citation_year=2012;  citation_pages=431-74; " /><meta name="citation_reference" content="Elzanowski A., Ostell J.  2013 , The genetic codes, NCBI website, 2010, Available at: http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi. " /><meta name="citation_reference" content="citation_publisher=Springer International Publishing, Switzerland;  citation_title=Evolution of the Protein Synthesis Machinery and its Regulation;  citation_year=2016;  citation_pages=109-43; " /><meta name="citation_reference" content="citation_title=Mobile group II introns of yeast mitochondrial DNA are novel site-specific retroelements,;  citation_journal_title=Mol. Cell. Biol;  citation_year=1995;  citation_volume=15;  citation_pages=2828-38; " /><meta name="citation_reference" content="citation_title=Mobile group II introns,;  citation_journal_title=Ann. Rev. Genet;  citation_year=2004;  citation_volume=38;  citation_pages=1-35; " /><meta name="citation_reference" content="citation_title=Phylogenetic circumscription of Saccharomyces, Kluyveromyces and other members of the Saccharomycetaceae, and the proposal of the new genera Lachancea, Nakaseomyces, Naumonia, Vanderwaltozyma and Zygotoruspora,;  citation_journal_title=FEMS Yeast Res;  citation_year=2003;  citation_volume=4;  citation_pages=233-45; " /><meta name="citation_reference" content="citation_title=Sequence analysis of three mitochondrial DNA molecules reveals interesting differences among Saccharomyces yeasts,;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=3081-91; " /><meta name="citation_reference" content="citation_title=In vivo analysis of mutated initiation codons in the mitochondrial COX2 gene of Saccharomyces cerevisiae fused to the reporter gene ARG8m reveals lack of downstream reinitiation,;  citation_journal_title=Mol. Gen. Genet;  citation_year=2000;  citation_volume=262;  citation_pages=1036-46; " /><meta name="citation_reference" content="citation_title=Mitochondrial translation of Saccharomyces cerevisiae COX2 mRNA is controlled by the nucleotide sequence specifying the pre-Cox2p leader peptide,;  citation_journal_title=Mol. Cell. Biol;  citation_year=2001;  citation_volume=21;  citation_pages=2359-72; " /><meta name="citation_reference" content="citation_title=GUG is an efficient initiation codon to translate the human mitochondrial ATP6 gene,;  citation_journal_title=Biochem. Biophys. Res. Commun;  citation_year=2004;  citation_volume=313;  citation_pages=687-93; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of sake yeast Saccharomyces cerevisiae Kyokai no. 7,;  citation_journal_title=DNA Res;  citation_year=2011;  citation_volume=18;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=MeMotif: a database of linear motifs in α-helical transmembrane proteins,;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=38;  citation_pages=D181-9; " /><meta name="citation_reference" content="citation_title=Homing endonucleases: keeping the house in order,;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=3379-88; " /><meta name="citation_reference" content="citation_title=Homing endonucleases: from microbial genetic invaders to reagents for targeted DNA modification,;  citation_journal_title=Structure;  citation_year=2011;  citation_volume=19;  citation_pages=7-15; " /><meta name="citation_reference" content="citation_title=Mne1 is a novel component of the mitochondrial splicing apparatus responsible for processing of a COX1 group I intron in yeast,;  citation_journal_title=J. Biol. Chem;  citation_year=2011;  citation_volume=286;  citation_pages=10137-46; " /><meta name="citation_reference" content="citation_title=The mitochondrial RNA landscape of Saccharomyces cerevisiae,;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=1-21; " /><meta name="citation_reference" content="citation_title=Phylogenomic analysis of the GIY-YIG nuclease superfamily;  citation_journal_title=BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=1-19; " /><meta name="citation_reference" content="citation_title=Multiple acquisitions via horizontal transfer of a group I intron in the mitochondrial cox1 gene during evolution of the Araceae family,;  citation_journal_title=Mol. Biol. Evol;  citation_year=1999;  citation_volume=16;  citation_pages=1155-65; " /><meta name="citation_reference" content="citation_title=Recurrent invasion and extinction of a selfish gene,;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1999;  citation_volume=96;  citation_pages=13880-5; " /><meta name="citation_reference" content="citation_title=Site specific DNA endonuclease and RNA maturase activities of two homologous intron encoded proteins from yeast mitochondria,;  citation_journal_title=Cell;  citation_year=1989;  citation_volume=56;  citation_pages=431-41; " /><meta name="citation_reference" content="citation_title=Inheritance and organisation of the mitochondrial genome differ between two Saccharomyces yeasts,;  citation_journal_title=J. Mol. Biol;  citation_year=2002;  citation_volume=318;  citation_pages=627-36; " /><meta name="citation_reference" content="citation_title=The complete mitochondrial genome of the yeast Kluyveromyces thermotolerans,;  citation_journal_title=FEBS Lett;  citation_year=2005;  citation_volume=579;  citation_pages=30-40; " /><meta name="citation_reference" content="citation_title=The natural history of group I introns,;  citation_journal_title=Trends Genet;  citation_year=2005;  citation_volume=21;  citation_pages=111-9; " /><meta name="citation_reference" content="citation_title=Homing endonuclease structure &amp; function,;  citation_journal_title=Q. Rev. Biophys;  citation_year=2006;  citation_volume=38;  citation_pages=49-95; " /><meta name="citation_reference" content="citation_title=Selfish DNA: homing endonucleases find a home,;  citation_journal_title=Curr. Biol;  citation_year=2009;  citation_volume=19;  citation_pages=R115-7; " /><meta name="citation_reference" content="citation_title=Two for the price of one: a bifunctional intron-encoded DNA endonuclease-RNA maturase,;  citation_journal_title=Genes Dev;  citation_year=2003;  citation_volume=17;  citation_pages=2860-3; " /><meta name="citation_reference" content="citation_title=CDD: a Conserved Domain Database for the functional annotation of proteins,;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=D225-9; " /><meta name="citation_reference" content="citation_title=The Saccharomyces Genome Database Project: Saccharomyces cerevisiae S288C genome annotation: a working hypothesis,;  citation_journal_title=Yeast;  citation_year=2006;  citation_volume=23;  citation_pages=857-65; " /><meta name="citation_reference" content="citation_title=MOTS-c: a novel mitochondrial-derived peptide regulating muscle and fat metabolism;  citation_journal_title=Free Radical Biol. Med;  citation_year=2016;  citation_volume=100;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=The mitochondrial-derived peptide MOTS-c promotes metabolic homeostasis and reduces obesity and insulin resistance,;  citation_journal_title=Cell Metab;  citation_year=2015;  citation_volume=21;  citation_pages=443-54; " /><meta name="citation_reference" content="citation_title=New features of mitochondrial DNA replication system in yeast and man,;  citation_journal_title=Gene;  citation_year=2000;  citation_volume=246;  citation_pages=37-48; " /><meta name="citation_reference" content="citation_title=Putative target sites for mobile G + C rich clusters in yeast mitochondrial DNA: single elements and tandem arrays,;  citation_journal_title=Mol. Gen. Genet;  citation_year=1989;  citation_volume=218;  citation_pages=272-83; " /><meta name="citation_reference" content="citation_title=The GC clusters of the mitochondrial genome of yeast and their evolutionary origin;  citation_journal_title=Gene;  citation_year=1986a;  citation_volume=41;  citation_pages=1-22; " /><meta name="citation_reference" content="citation_title=Transcriptional initiation and processing of the small ribosomal RNA of yeast mitochondria;  citation_journal_title=J. Biol. Chem;  citation_year=1982;  citation_volume=257;  citation_pages=6494-500; " /><meta name="citation_reference" content="citation_title=Initiation of transcription in yeast mitochondria: analysis of origins of replication and of genes coding for a messenger RNA and a transfer RNA;  citation_journal_title=Nucleic Acids Res;  citation_year=1984a;  citation_volume=12;  citation_pages=1889-900; " /><meta name="citation_reference" content="citation_title=RNA maturation in mitochondria of S. cerevisiae and S. pombe,;  citation_journal_title=Gene;  citation_year=2005;  citation_volume=354;  citation_pages=80-5; " /><meta name="citation_reference" content="citation_title=Processing of yeast mitochondrial messenger RNAs at a conserved dodecamer sequence;  citation_journal_title=EMBO J;  citation_year=1984b;  citation_volume=3;  citation_pages=829-34; " /><meta name="citation_reference" content="citation_title=Formation of the 3′ end of yeast mitochondrial mRNAs occurs by site-specific cleavage two bases downstream of a conserved dodecamer sequence,;  citation_journal_title=Yeast;  citation_year=1993;  citation_volume=9;  citation_pages=1319-30; " /><meta name="citation_reference" content="citation_title=Population genomic analysis reveals highly conserved mitochondrial genomes in the yeast species Lachancea thermotolerans,;  citation_journal_title=Genome Biol. Evol;  citation_year=2014;  citation_volume=6;  citation_pages=2586-94; " /><meta name="citation_reference" content="citation_title=Extensive horizontal transfer and homologous recombination generate highly chimeric mitochondrial genomes in yeast,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=2559-70; " /><meta name="citation_reference" content="citation_title=Mitochondrial genomes of yeasts of the Yarrowia clade,;  citation_journal_title=FEMS Yeast Res;  citation_year=2012;  citation_volume=12;  citation_pages=317-31; " /><meta name="citation_reference" content="citation_title=Complete mitochondrial genome sequences of three Nakaseomyces species reveal invasion by palindromic GC clusters and considerable size expansion,;  citation_journal_title=FEMS Yeast Res;  citation_year=2009;  citation_volume=9;  citation_pages=1283-92; " /><meta name="citation_reference" content="citation_title=Mitochondrial genome from the facultative anaerobe and petite-positive yeast Dekkera bruxellensis contains the NADH dehydrogenase subunit genes;  citation_journal_title=FEMS Yeast Res;  citation_year=2010;  citation_volume=10;  citation_pages=545-57; " /><meta name="citation_reference" content="citation_title=Evolution of mitochondrial genomes in fungi,;  citation_journal_title=Int. Rev. Cytol;  citation_year=1992;  citation_volume=141;  citation_pages=89-127; " /><meta name="citation_reference" content="citation_title=Saccharomyces diversity and evolution: a budding model genus,;  citation_journal_title=Trends Genet;  citation_year=2013;  citation_volume=29;  citation_pages=309-17; " /><meta name="citation_reference" content="citation_title=Advances in yeast systematics and phylogeny and their use as predictors of biotechnologically important metabolic pathways,;  citation_journal_title=FEMS Yeast Res;  citation_year=2015;  citation_volume=15;  citation_pages=1-17; " /><meta name="citation_reference" content="citation_title=Complex ancestries of lager-brewing hybrids were shaped by standing variation in the wild yeast Saccharomyces eubayanus,;  citation_journal_title=PLoS Genet;  citation_year=2016;  citation_volume=12;  citation_pages=1-20; " /><meta name="citation_reference" content="citation_title=The inheritance of mtDNA in lager brewing strains,;  citation_journal_title=FEMS Yeast Res;  citation_year=2008;  citation_volume=8;  citation_pages=586-96; " /><meta name="citation_reference" content="citation_title=A Gondwanan imprint on global diversity and domestication of wine and cider yeast Saccharomyces uvarum,;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Population structure and reticulate evolution of Saccharomyces eubayanus and its lager-brewing hybrids;  citation_journal_title=Mol. Ecol;  citation_year=2014;  citation_volume=23;  citation_pages=2031-45; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/6/571/22295732/dsx026.pdf" /><meta name="description" content="Abstract.  The yeast Saccharomyces are widely used to test ecological and evolutionary hypotheses. A large number of nuclear genomic DNA sequences are available" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/6/571/3868562" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/6/571/3868562" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/6/571/3868562");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx026';
    </script>
<title>
	evolutionary history of Saccharomyces species inferred from completed mitochondrial genomes and revision in the ‘yeast mitochondrial genetic code’ | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3868562" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdDKATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3VsbywgUGF2b2wiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSwgRmFjdWx0eSBvZiBOYXR1cmFsIFNjaWVuY2VzLCBDb21lbml1cyBVbml2ZXJzaXR5LCBCcmF0aXNsYXZhIDg0MuKAiTE1LCBTbG92YWtpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN6YWLDs292w6EsIERhbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSwgRmFjdWx0eSBvZiBOYXR1cmFsIFNjaWVuY2VzLCBDb21lbml1cyBVbml2ZXJzaXR5LCBCcmF0aXNsYXZhIDg0MuKAiTE1LCBTbG92YWtpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJpZWxpaywgUGV0ZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSwgRmFjdWx0eSBvZiBOYXR1cmFsIFNjaWVuY2VzLCBDb21lbml1cyBVbml2ZXJzaXR5LCBCcmF0aXNsYXZhIDg0MuKAiTE1LCBTbG92YWtpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBvbMOha292w6EsIFNpbHZpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvY2hlbWlzdHJ5LCBGYWN1bHR5IG9mIE5hdHVyYWwgU2NpZW5jZXMsIENvbWVuaXVzIFVuaXZlcnNpdHksIEJyYXRpc2xhdmEgODQy4oCJMTUsIFNsb3Zha2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0ixaBvbHR5cywgS2F0YXLDrW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29tZW5pdXMgVW5pdmVyc2l0eSBTY2llbmNlIFBhcmssIEJyYXRpc2xhdmEgODQx4oCJMDQsIFNsb3Zha2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmF0em92w6EsIEthdGFyw61uYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvY2hlbWlzdHJ5LCBGYWN1bHR5IG9mIE5hdHVyYWwgU2NpZW5jZXMsIENvbWVuaXVzIFVuaXZlcnNpdHksIEJyYXRpc2xhdmEgODQy4oCJMTUsIFNsb3Zha2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3plbWVzLCBUb23DocWhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29tZW5pdXMgVW5pdmVyc2l0eSBTY2llbmNlIFBhcmssIEJyYXRpc2xhdmEgODQx4oCJMDQsIFNsb3Zha2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgQmlvbG9neSwgRmFjdWx0eSBvZiBOYXR1cmFsIFNjaWVuY2VzLCBDb21lbml1cyBVbml2ZXJzaXR5LCBCcmF0aXNsYXZhIDg0MuKAiTE1LCBTbG92YWtpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdlbmV0b24gcy5yLm8uLCBHYWx2YW5paG8gNywgQnJhdGlzbGF2YSA4MjEgMDQsIFNsb3Zha2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJUaGUgZXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2YgU2FjY2hhcm9teWNlcyBzcGVjaWVzIGluZmVycmVkIGZyb20gY29tcGxldGVkIG1pdG9jaG9uZHJpYWwgZ2Vub21lcyBhbmQgcmV2aXNpb24gaW4gdGhlIOKAmHllYXN0IG1pdG9jaG9uZHJpYWwgZ2VuZXRpYyBjb2Rl4oCZIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1ODMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8xMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1FbHNldmllciwgQW1zdGVyZGFtLCBOTDsgIGNpdGF0aW9uX3RpdGxlPVRoZSBZZWFzdHMsIEEgVGF4b25vbWljIFN0dWR5OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fcGFnZXM9NzMzLTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWNjaGFyb215Y2VzIHNlbnN1IHN0cmljdG8gYXMgYSBtb2RlbCBzeXN0ZW0gZm9yIGV2b2x1dGlvbiBhbmQgZWNvbG9neSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBFY29sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDk0LTUwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJzcGVjaWVzIGh5YnJpZGl6YXRpb24gYW5kIHJlY29tYmluYXRpb24gaW4gU2FjY2hhcm9teWNlcyB3aW5lIHllYXN0cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgWWVhc3QgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz05OTYtMTAwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBpbnNpZ2h0cyBpbnRvIHRoZSBTYWNjaGFyb215Y2VzIHNlbnN1IHN0cmljdG8gY29tcGxleCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE5OTsgIGNpdGF0aW9uX3BhZ2VzPTI4MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBTYWNjaGFyb215Y2VzIGV1YmF5YW51cyBhbmQgdGhlIGRvbWVzdGljYXRpb24gb2YgbGFnZXItYnJld2luZyB5ZWFzdHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0yODE4LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgc2VxdWVuY2Ugb2YgdGhlIG1pdG9jaG9uZHJpYWwgZ2Vub21lIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFQlMgTGV0dDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT00NDA7ICBjaXRhdGlvbl9wYWdlcz0zMjUtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pdG9jaG9uZHJpYWwgZ2Vub21lIGV2b2x1dGlvbiBpbiB5ZWFzdHM6IGFuIGFsbC1lbmNvbXBhc3Npbmcgdmlldyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgWWVhc3QgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIGFuZCBjb21wYXJpc29uIG9mIHllYXN0IHNwZWNpZXMgdG8gaWRlbnRpZnkgZ2VuZXMgYW5kIHJlZ3VsYXRvcnkgZWxlbWVudHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDIzOyAgY2l0YXRpb25fcGFnZXM9MjQxLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIHF1YWxpdHkgZGUgbm92byBzZXF1ZW5jaW5nIGFuZCBhc3NlbWJseSBvZiB0aGUgU2FjY2hhcm9teWNlcyBhcmJvcmljb2x1cyBnZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBhbmQgdGhlIG1ha2luZyBvZiB5ZWFzdCBiaW9kaXZlcnNpdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MTAwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcGxldGUgc2VxdWVuY2Ugb2YgU2FjY2hhcm9teWNlcyBwYXJhZG94dXMgbWl0b2Nob25kcmlhbCBnZW5vbWUgdGhhdCByZXN0b3JlcyB0aGUgcmVzcGlyYXRpb24gaW4gUy4gY2VyZXZpc2lhZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgWWVhc3QgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ODE5LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGxhZ2VyIGJyZXdpbmcgeWVhc3QsIGFuIGludGVyc3BlY2llcyBoeWJyaWQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTE1LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBhbmFseXNpcyBvZiBsYWdlciBicmV3aW5nIHllYXN0IHN0cmFpbnMgcmV2ZWFscyB0aGUgZXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2YgaW50ZXJzcGVjaWVzIGh5YnJpZGl6YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz02Ny04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyBzeXN0ZW1zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbWVkLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIwMTI7ICBjaXRhdGlvbl9wYWdlcz0xLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRhbGUgb2YgdGhyZWUgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNpbmcgcGxhdGZvcm1zOiBjb21wYXJpc29uIG9mIElvbiBUb3JyZW50LCBQYWNpZmljIEJpb3NjaWVuY2VzIGFuZCBJbGx1bWluYSBNaVNlcSBzZXF1ZW5jZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgcGhlbm9taWNzIHJldmVhbHMgaW5zaWdodCBpbnRvIHRoZSBzdHJ1Y3R1cmUgb2YgcGhlbm90eXBpYyBkaXZlcnNpdHkgaW4gYnVkZGluZyB5ZWFzdCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNDk2LTUwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlZmluaXRpb24gdmlldyBvZiBmdW5jdGlvbmFsIGdlbmV0aWMgdmFyaWF0aW9uIGZyb20gbmF0dXJhbCB5ZWFzdCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz04NzItODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hc3NpdmUgcHJvZ3JhbW1lZCB0cmFuc2xhdGlvbmFsIGp1bXBpbmcgaW4gbWl0b2Nob25kcmlhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTU5MjYtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZHluYW1pYyBtb2JpbGUgRE5BIGZhbWlseSBpbiB0aGUgeWVhc3QgbWl0b2Nob25kcmlhbCBnZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMyAoQmV0aGVzZGEpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMjczLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIHN0cnVjdHVyZSBvZiBtaXRvY2hvbmRyaWFsIGdlbm9tZXMgaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIDEwMC1nZW5vbWVzIHN0cmFpbnMsIGFuIFMuIGNlcmV2aXNpYWUgcmVzb3VyY2UgdGhhdCBpbGx1bWluYXRlcyBpdHMgbmF0dXJhbCBwaGVub3R5cGljIGFuZCBnZW5vdHlwaWMgdmFyaWF0aW9uIGFuZCBlbWVyZ2VuY2UgYXMgYW4gb3Bwb3J0dW5pc3RpYyBwYXRob2dlbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz03NjItNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvc3Qtenlnb3RpYyBzdGVyaWxpdHkgYW5kIGN5dG9udWNsZWFyIGNvbXBhdGliaWxpdHkgbGltaXRzIGluIFMuIGNlcmV2aXNpYWUgeGVub21pdG9jaG9uZHJpYWwgY3licmlkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJhcGlkIG1pbmlwcmVwIG1ldGhvZCBmb3IgdGhlIHByZXBhcmF0aW9uIG9mIHllYXN0IG1pdG9jaG9uZHJpYWwgRE5BLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNwaXJhdG9yeS1jb21wZXRlbnQgeWVhc3QgbWl0b2Nob25kcmlhbCBETkFzIGdlbmVyYXRlZCBieSBkZWxldGluZyBpbnRlcmdlbmljIHJlZ2lvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz0xNjUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIGFuZCBnZW5ldGljIHN0YWJpbGl0eSBvZiBtaXRvY2hvbmRyaWFsIGdlbm9tZXMgdmFyeSBhbW9uZyB5ZWFzdHMgb2YgdGhlIGdlbnVzIFNhY2NoYXJvbXljZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gU3lzdC4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MTAxNS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iZSBkb21lc3RpY2F0aW9uIGFuZCB0aGUgaWRlbnRpZmljYXRpb24gb2YgdGhlIHdpbGQgZ2VuZXRpYyBzdG9jayBvZiBsYWdlci1icmV3aW5nIHllYXN0LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTE0NTM5LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvY2hvbmRyaWFsIGludHJvbnM6IGEgY3JpdGljYWwgdmlldyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTExOS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9ucyBhcyBtb2JpbGUgZ2VuZXRpYyBlbGVtZW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gQmlvY2hlbTsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTU4Ny02MjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwcmltYXJ5IHN0cnVjdHVyZSBvZiB0aGUgbWl0b2Nob25kcmlhbCBnZW5vbWUgb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIOKAkyBhIHJldmlldzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MTk4NmI7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0xNTUtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcGFyaXNvbiBvZiB0aGVybW9keW5hbWljIGZvbGRpbmdzIHdpdGggY29tcGFyYXRpdmVseSBkZXJpdmVkIHN0cnVjdHVyZXMgb2YgMTZTIGFuZCAxNlMtbGlrZSByUk5BczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz01NTktNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFuYWx5c2lzIG9mIGxhcmdlIHJSTkEgc2VxdWVuY2VzIGZvbGRlZCBieSBhIHRoZXJtb2R5bmFtaWMgbWV0aG9kLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Rm9sZC4gRGVzOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz00MTktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pdG9jaG9uZHJpYWwgUk5hc2UgUCBSTkFzIGluIGFzY29teWNldGUgZnVuZ2k6IGxpbmVhZ2Utc3BlY2lmaWMgdmFyaWF0aW9ucyBpbiBSTkEgc2Vjb25kYXJ5IHN0cnVjdHVyZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTA3My04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R09CQVNFIOKAkyBhIGRhdGFiYXNlIG9mIG9yZ2FuZWxsZSBhbmQgYmFjdGVyaWFsIGdlbm9tZSBpbmZvcm1hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1ENjk3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJDTEMgR2Vub21pY3MgV29ya2JlbmNoIDkuNS4yLiBodHRwczovL3d3dy5xaWFnZW5iaW9pbmZvcm1hdGljcy5jb20vICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuaWVzIGZyb20gbW9sZWN1bGFyIHNlcXVlbmNlczogaW5mZXJlbmNlIGFuZCByZWxpYWJpbGl0eSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUsIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz01MjEtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciBkb21lc3RpY2F0ZWQgYW5kIHdpbGQgcG9wdWxhdGlvbnMgb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9NjYtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gc3RydWN0dXJlIGFuZCBnZW5lIGV2b2x1dGlvbiBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIFllYXN0IFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NzAyLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHBsaWNhdGlvbiBvZiBtdWx0aSBsb2N1cyBzZXF1ZW5jZSB0eXBpbmcgdG8gdGhlIGFuYWx5c2lzIG9mIHRoZSBiaW9kaXZlcnNpdHkgb2YgaW5kaWdlbm91cyBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgd2luZSB5ZWFzdHMgZnJvbSBMZWJhbm9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQXBwbC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTY5OS03MTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cnByaXNpbmdseSBkaXZlcmdlZCBwb3B1bGF0aW9ucyBvZiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgaW4gbmF0dXJhbCBlbnZpcm9ubWVudHMgcmVtb3RlIGZyb20gaHVtYW4gYWN0aXZpdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz01NDA0LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYXdlc29tZSBwb3dlciBvZiB5ZWFzdCBldm9sdXRpb25hcnkgZ2VuZXRpY3M6IG5ldyBnZW5vbWUgc2VxdWVuY2VzIGFuZCBzdHJhaW4gcmVzb3VyY2VzIGZvciB0aGUgU2FjY2hhcm9teWNlcyBzZW5zdSBzdHJpY3RvIGdlbnVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMyAoQmV0aGVzZGEpOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN0cnVjdHVyZSBvZiB0aGUgc21hbGwgbWl0b2Nob25kcmlhbCBETkEgb2YgS2x1eXZlcm9teWNlcyB0aGVybW90b2xlcmFucyBpcyBsaWtlbHkgdG8gcmVmbGVjdCB0aGUgYW5jZXN0cmFsIGdlbmUgb3JkZXIgaW4gZnVuZ2ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz01OTMtNjAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXItVmVybGFnLCBCZXJsaW4sIEdlcm1hbnk7ICBjaXRhdGlvbl90aXRsZT1PcmdhbmVsbGUgR2VuZXRpY3MuIEV2b2x1dGlvbiBvZiBPcmdhbmVsbGUgR2Vub21lcyBhbmQgR2VuZSBFeHByZXNzaW9uOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fcGFnZXM9NDMxLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJFbHphbm93c2tpIEEuLCBPc3RlbGwgSi4gIDIwMTMgLCBUaGUgZ2VuZXRpYyBjb2RlcywgTkNCSSB3ZWJzaXRlLCAyMDEwLCBBdmFpbGFibGUgYXQ6IGh0dHA6Ly93d3cubmNiaS5ubG0ubmloLmdvdi9UYXhvbm9teS9VdGlscy93cHJpbnRnYy5jZ2kuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciBJbnRlcm5hdGlvbmFsIFB1Ymxpc2hpbmcsIFN3aXR6ZXJsYW5kOyAgY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHRoZSBQcm90ZWluIFN5bnRoZXNpcyBNYWNoaW5lcnkgYW5kIGl0cyBSZWd1bGF0aW9uOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fcGFnZXM9MTA5LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGUgZ3JvdXAgSUkgaW50cm9ucyBvZiB5ZWFzdCBtaXRvY2hvbmRyaWFsIEROQSBhcmUgbm92ZWwgc2l0ZS1zcGVjaWZpYyByZXRyb2VsZW1lbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjgyOC0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9iaWxlIGdyb3VwIElJIGludHJvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0xLTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgY2lyY3Vtc2NyaXB0aW9uIG9mIFNhY2NoYXJvbXljZXMsIEtsdXl2ZXJvbXljZXMgYW5kIG90aGVyIG1lbWJlcnMgb2YgdGhlIFNhY2NoYXJvbXljZXRhY2VhZSwgYW5kIHRoZSBwcm9wb3NhbCBvZiB0aGUgbmV3IGdlbmVyYSBMYWNoYW5jZWEsIE5ha2FzZW9teWNlcywgTmF1bW9uaWEsIFZhbmRlcndhbHRvenltYSBhbmQgWnlnb3RvcnVzcG9yYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgWWVhc3QgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0yMzMtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGFuYWx5c2lzIG9mIHRocmVlIG1pdG9jaG9uZHJpYWwgRE5BIG1vbGVjdWxlcyByZXZlYWxzIGludGVyZXN0aW5nIGRpZmZlcmVuY2VzIGFtb25nIFNhY2NoYXJvbXljZXMgeWVhc3RzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMDgxLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiB2aXZvIGFuYWx5c2lzIG9mIG11dGF0ZWQgaW5pdGlhdGlvbiBjb2RvbnMgaW4gdGhlIG1pdG9jaG9uZHJpYWwgQ09YMiBnZW5lIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBmdXNlZCB0byB0aGUgcmVwb3J0ZXIgZ2VuZSBBUkc4bSByZXZlYWxzIGxhY2sgb2YgZG93bnN0cmVhbSByZWluaXRpYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjYyOyAgY2l0YXRpb25fcGFnZXM9MTAzNi00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWl0b2Nob25kcmlhbCB0cmFuc2xhdGlvbiBvZiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgQ09YMiBtUk5BIGlzIGNvbnRyb2xsZWQgYnkgdGhlIG51Y2xlb3RpZGUgc2VxdWVuY2Ugc3BlY2lmeWluZyB0aGUgcHJlLUNveDJwIGxlYWRlciBwZXB0aWRlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjM1OS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R1VHIGlzIGFuIGVmZmljaWVudCBpbml0aWF0aW9uIGNvZG9uIHRvIHRyYW5zbGF0ZSB0aGUgaHVtYW4gbWl0b2Nob25kcmlhbCBBVFA2IGdlbmUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTM7ICBjaXRhdGlvbl9wYWdlcz02ODctOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nIG9mIHNha2UgeWVhc3QgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIEt5b2thaSBuby4gNyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZU1vdGlmOiBhIGRhdGFiYXNlIG9mIGxpbmVhciBtb3RpZnMgaW4gzrEtaGVsaWNhbCB0cmFuc21lbWJyYW5lIHByb3RlaW5zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1EMTgxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvbWluZyBlbmRvbnVjbGVhc2VzOiBrZWVwaW5nIHRoZSBob3VzZSBpbiBvcmRlciw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM3OS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9taW5nIGVuZG9udWNsZWFzZXM6IGZyb20gbWljcm9iaWFsIGdlbmV0aWMgaW52YWRlcnMgdG8gcmVhZ2VudHMgZm9yIHRhcmdldGVkIEROQSBtb2RpZmljYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TdHJ1Y3R1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz03LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NbmUxIGlzIGEgbm92ZWwgY29tcG9uZW50IG9mIHRoZSBtaXRvY2hvbmRyaWFsIHNwbGljaW5nIGFwcGFyYXR1cyByZXNwb25zaWJsZSBmb3IgcHJvY2Vzc2luZyBvZiBhIENPWDEgZ3JvdXAgSSBpbnRyb24gaW4geWVhc3QsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4NjsgIGNpdGF0aW9uX3BhZ2VzPTEwMTM3LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWl0b2Nob25kcmlhbCBSTkEgbGFuZHNjYXBlIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xLTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbm9taWMgYW5hbHlzaXMgb2YgdGhlIEdJWS1ZSUcgbnVjbGVhc2Ugc3VwZXJmYW1pbHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgYWNxdWlzaXRpb25zIHZpYSBob3Jpem9udGFsIHRyYW5zZmVyIG9mIGEgZ3JvdXAgSSBpbnRyb24gaW4gdGhlIG1pdG9jaG9uZHJpYWwgY294MSBnZW5lIGR1cmluZyBldm9sdXRpb24gb2YgdGhlIEFyYWNlYWUgZmFtaWx5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTE1NS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjdXJyZW50IGludmFzaW9uIGFuZCBleHRpbmN0aW9uIG9mIGEgc2VsZmlzaCBnZW5lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9MTM4ODAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2l0ZSBzcGVjaWZpYyBETkEgZW5kb251Y2xlYXNlIGFuZCBSTkEgbWF0dXJhc2UgYWN0aXZpdGllcyBvZiB0d28gaG9tb2xvZ291cyBpbnRyb24gZW5jb2RlZCBwcm90ZWlucyBmcm9tIHllYXN0IG1pdG9jaG9uZHJpYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz00MzEtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluaGVyaXRhbmNlIGFuZCBvcmdhbmlzYXRpb24gb2YgdGhlIG1pdG9jaG9uZHJpYWwgZ2Vub21lIGRpZmZlciBiZXR3ZWVuIHR3byBTYWNjaGFyb215Y2VzIHllYXN0cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTg7ICBjaXRhdGlvbl9wYWdlcz02MjctMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBtaXRvY2hvbmRyaWFsIGdlbm9tZSBvZiB0aGUgeWVhc3QgS2x1eXZlcm9teWNlcyB0aGVybW90b2xlcmFucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFQlMgTGV0dDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01Nzk7ICBjaXRhdGlvbl9wYWdlcz0zMC00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG5hdHVyYWwgaGlzdG9yeSBvZiBncm91cCBJIGludHJvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9taW5nIGVuZG9udWNsZWFzZSBzdHJ1Y3R1cmUgJmFtcDsgZnVuY3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1RLiBSZXYuIEJpb3BoeXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz00OS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZmlzaCBETkE6IGhvbWluZyBlbmRvbnVjbGVhc2VzIGZpbmQgYSBob21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPVIxMTUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIGZvciB0aGUgcHJpY2Ugb2Ygb25lOiBhIGJpZnVuY3Rpb25hbCBpbnRyb24tZW5jb2RlZCBETkEgZW5kb251Y2xlYXNlLVJOQSBtYXR1cmFzZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTI4NjAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0REOiBhIENvbnNlcnZlZCBEb21haW4gRGF0YWJhc2UgZm9yIHRoZSBmdW5jdGlvbmFsIGFubm90YXRpb24gb2YgcHJvdGVpbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPUQyMjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNhY2NoYXJvbXljZXMgR2Vub21lIERhdGFiYXNlIFByb2plY3Q6IFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBTMjg4QyBnZW5vbWUgYW5ub3RhdGlvbjogYSB3b3JraW5nIGh5cG90aGVzaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ZZWFzdDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTg1Ny02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TU9UUy1jOiBhIG5vdmVsIG1pdG9jaG9uZHJpYWwtZGVyaXZlZCBwZXB0aWRlIHJlZ3VsYXRpbmcgbXVzY2xlIGFuZCBmYXQgbWV0YWJvbGlzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJlZSBSYWRpY2FsIEJpb2wuIE1lZDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0xLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtaXRvY2hvbmRyaWFsLWRlcml2ZWQgcGVwdGlkZSBNT1RTLWMgcHJvbW90ZXMgbWV0YWJvbGljIGhvbWVvc3Rhc2lzIGFuZCByZWR1Y2VzIG9iZXNpdHkgYW5kIGluc3VsaW4gcmVzaXN0YW5jZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgTWV0YWI7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz00NDMtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBmZWF0dXJlcyBvZiBtaXRvY2hvbmRyaWFsIEROQSByZXBsaWNhdGlvbiBzeXN0ZW0gaW4geWVhc3QgYW5kIG1hbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjQ2OyAgY2l0YXRpb25fcGFnZXM9MzctNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB1dGF0aXZlIHRhcmdldCBzaXRlcyBmb3IgbW9iaWxlIEcgKyBDIHJpY2ggY2x1c3RlcnMgaW4geWVhc3QgbWl0b2Nob25kcmlhbCBETkE6IHNpbmdsZSBlbGVtZW50cyBhbmQgdGFuZGVtIGFycmF5cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gR2VuLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTg7ICBjaXRhdGlvbl9wYWdlcz0yNzItODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHQyBjbHVzdGVycyBvZiB0aGUgbWl0b2Nob25kcmlhbCBnZW5vbWUgb2YgeWVhc3QgYW5kIHRoZWlyIGV2b2x1dGlvbmFyeSBvcmlnaW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5ODZhOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbmFsIGluaXRpYXRpb24gYW5kIHByb2Nlc3Npbmcgb2YgdGhlIHNtYWxsIHJpYm9zb21hbCBSTkEgb2YgeWVhc3QgbWl0b2Nob25kcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTI1NzsgIGNpdGF0aW9uX3BhZ2VzPTY0OTQtNTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWF0aW9uIG9mIHRyYW5zY3JpcHRpb24gaW4geWVhc3QgbWl0b2Nob25kcmlhOiBhbmFseXNpcyBvZiBvcmlnaW5zIG9mIHJlcGxpY2F0aW9uIGFuZCBvZiBnZW5lcyBjb2RpbmcgZm9yIGEgbWVzc2VuZ2VyIFJOQSBhbmQgYSB0cmFuc2ZlciBSTkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg0YTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE4ODktOTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgbWF0dXJhdGlvbiBpbiBtaXRvY2hvbmRyaWEgb2YgUy4gY2VyZXZpc2lhZSBhbmQgUy4gcG9tYmUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM1NDsgIGNpdGF0aW9uX3BhZ2VzPTgwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2Nlc3Npbmcgb2YgeWVhc3QgbWl0b2Nob25kcmlhbCBtZXNzZW5nZXIgUk5BcyBhdCBhIGNvbnNlcnZlZCBkb2RlY2FtZXIgc2VxdWVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQk8gSjsgIGNpdGF0aW9uX3llYXI9MTk4NGI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTgyOS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9ybWF0aW9uIG9mIHRoZSAz4oCyIGVuZCBvZiB5ZWFzdCBtaXRvY2hvbmRyaWFsIG1STkFzIG9jY3VycyBieSBzaXRlLXNwZWNpZmljIGNsZWF2YWdlIHR3byBiYXNlcyBkb3duc3RyZWFtIG9mIGEgY29uc2VydmVkIGRvZGVjYW1lciBzZXF1ZW5jZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVllYXN0OyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMzE5LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIGdlbm9taWMgYW5hbHlzaXMgcmV2ZWFscyBoaWdobHkgY29uc2VydmVkIG1pdG9jaG9uZHJpYWwgZ2Vub21lcyBpbiB0aGUgeWVhc3Qgc3BlY2llcyBMYWNoYW5jZWEgdGhlcm1vdG9sZXJhbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjU4Ni05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0ZW5zaXZlIGhvcml6b250YWwgdHJhbnNmZXIgYW5kIGhvbW9sb2dvdXMgcmVjb21iaW5hdGlvbiBnZW5lcmF0ZSBoaWdobHkgY2hpbWVyaWMgbWl0b2Nob25kcmlhbCBnZW5vbWVzIGluIHllYXN0LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MjU1OS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWl0b2Nob25kcmlhbCBnZW5vbWVzIG9mIHllYXN0cyBvZiB0aGUgWWFycm93aWEgY2xhZGUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIFllYXN0IFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMxNy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgbWl0b2Nob25kcmlhbCBnZW5vbWUgc2VxdWVuY2VzIG9mIHRocmVlIE5ha2FzZW9teWNlcyBzcGVjaWVzIHJldmVhbCBpbnZhc2lvbiBieSBwYWxpbmRyb21pYyBHQyBjbHVzdGVycyBhbmQgY29uc2lkZXJhYmxlIHNpemUgZXhwYW5zaW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBZZWFzdCBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEyODMtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pdG9jaG9uZHJpYWwgZ2Vub21lIGZyb20gdGhlIGZhY3VsdGF0aXZlIGFuYWVyb2JlIGFuZCBwZXRpdGUtcG9zaXRpdmUgeWVhc3QgRGVra2VyYSBicnV4ZWxsZW5zaXMgY29udGFpbnMgdGhlIE5BREggZGVoeWRyb2dlbmFzZSBzdWJ1bml0IGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIFllYXN0IFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU0NS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIG1pdG9jaG9uZHJpYWwgZ2Vub21lcyBpbiBmdW5naSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gUmV2LiBDeXRvbDsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz04OS0xMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhY2NoYXJvbXljZXMgZGl2ZXJzaXR5IGFuZCBldm9sdXRpb246IGEgYnVkZGluZyBtb2RlbCBnZW51cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTMwOS0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWR2YW5jZXMgaW4geWVhc3Qgc3lzdGVtYXRpY3MgYW5kIHBoeWxvZ2VueSBhbmQgdGhlaXIgdXNlIGFzIHByZWRpY3RvcnMgb2YgYmlvdGVjaG5vbG9naWNhbGx5IGltcG9ydGFudCBtZXRhYm9saWMgcGF0aHdheXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIFllYXN0IFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXggYW5jZXN0cmllcyBvZiBsYWdlci1icmV3aW5nIGh5YnJpZHMgd2VyZSBzaGFwZWQgYnkgc3RhbmRpbmcgdmFyaWF0aW9uIGluIHRoZSB3aWxkIHllYXN0IFNhY2NoYXJvbXljZXMgZXViYXlhbnVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbmhlcml0YW5jZSBvZiBtdEROQSBpbiBsYWdlciBicmV3aW5nIHN0cmFpbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIFllYXN0IFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NTg2LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIEdvbmR3YW5hbiBpbXByaW50IG9uIGdsb2JhbCBkaXZlcnNpdHkgYW5kIGRvbWVzdGljYXRpb24gb2Ygd2luZSBhbmQgY2lkZXIgeWVhc3QgU2FjY2hhcm9teWNlcyB1dmFydW0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9wdWxhdGlvbiBzdHJ1Y3R1cmUgYW5kIHJldGljdWxhdGUgZXZvbHV0aW9uIG9mIFNhY2NoYXJvbXljZXMgZXViYXlhbnVzIGFuZCBpdHMgbGFnZXItYnJld2luZyBoeWJyaWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMDMxLTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvNi81NzEvMjIyOTU3MzIvZHN4MDI2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgeWVhc3QgU2FjY2hhcm9teWNlcyBhcmUgd2lkZWx5IHVzZWQgdG8gdGVzdCBlY29sb2dpY2FsIGFuZCBldm9sdXRpb25hcnkgaHlwb3RoZXNlcy4gQSBsYXJnZSBudW1iZXIgb2YgbnVjbGVhciBnZW5vbWljIEROQSBzZXF1ZW5jZXMgYXJlIGF2YWlsYWJsZSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mzg2ODU2MiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTE/pIMtKzL3vk13pKWLxDviHYE6Q==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f571%2f3868562" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f571%2f3868562" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=103821" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&amp;Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/6">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="104086139">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086139">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086141">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086141">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086145">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086145">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086154">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086154">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086186">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086186">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086188">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086188">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086196">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086196">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/6/559/3868561">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/6/585/3896153">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The evolutionary history of <em>Saccharomyces</em> species inferred from completed mitochondrial genomes and revision in the ‘yeast mitochondrial genetic code’
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pavol Sulo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx026-cor1" data-open="dsx026-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +421 2 6029 6611. Fax. +421 2 6029 6452. Email: <a href="mailto:sulo@fns.uniba.sk" target="_blank">sulo@fns.uniba.sk</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pavol+Sulo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sulo P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sulo P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Pavol Sulo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dana Szab&#243;ov&#225;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dana+Szab%c3%b3ov%c3%a1">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Szab&#243;ov&#225; D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Szab&#243;ov&#225; D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dana Szabóová</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Peter Bielik</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Peter+Bielik">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bielik P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bielik P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Peter Bielik</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Silvia Pol&#225;kov&#225;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Silvia+Pol%c3%a1kov%c3%a1">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pol&#225;kov&#225; S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pol&#225;kov&#225; S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Silvia Poláková</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katar&#237;na Šoltys</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Comenius University Science Park, Bratislava 841 04, Slovakia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katar%c3%adna+%c5%a0oltys">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Šoltys K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Šoltys K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Katarína Šoltys</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katar&#237;na Jatzov&#225;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biochemistry, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katar%c3%adna+Jatzov%c3%a1">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jatzov&#225; K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jatzov&#225; K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Katarína Jatzová</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tom&#225;š Szemes</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Comenius University Science Park, Bratislava 841 04, Slovakia</div><div class="aff">Department of Molecular Biology, Faculty of Natural Sciences, Comenius University, Bratislava 842 15, Slovakia</div><div class="aff">Geneton s.r.o., Galvaniho 7, Bratislava 821 04, Slovakia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tom%c3%a1%c5%a1+Szemes">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Szemes T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Szemes T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tomáš Szemes</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx026-FM1"><div class="fn"><div class="footnote-content">Present address: Department of Membrane Biochemistry, Centre of Biosciences, Slovak Academy of Sciences, Bratislava 84005, Slovakia</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx026-FM2"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 571–583, <a href='https://doi.org/10.1093/dnares/dsx026'>https://doi.org/10.1093/dnares/dsx026</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 June 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 December 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 May 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3868562" href="/dnaresearch/article-pdf/24/6/571/22295732/dsx026.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Pavol Sulo, Dana Szabóová, Peter Bielik, Silvia Poláková, Katarína Šoltys, Katarína Jatzová, Tomáš Szemes;  The evolutionary history of <em>Saccharomyces</em> species inferred from completed mitochondrial genomes and revision in the ‘yeast mitochondrial genetic code’, <em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 571–583, <a href='https://doi.org/10.1093/dnares/dsx026'>https://doi.org/10.1093/dnares/dsx026</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3868562?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868562?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868562?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868562?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868562?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868562?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868562?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20evolutionary%20history%20of%20%26lt%3Bem%26gt%3BSaccharomyces%26lt%3B%2Fem%26gt%3B%20species%20inferred%20from%20completed%20mitochondrial%20genomes%20and%20revision%20in%20the%20%E2%80%98yeast%20mitochondrial%20genetic%20code%E2%80%99&amp;publicationDate=2017-06-15&amp;volumeNum=24&amp;issueNum=6&amp;author=Sulo%2C%20Pavol%3B%20Szab%C3%B3ov%C3%A1%2C%20Dana&amp;startPage=571&amp;endPage=583&amp;contentId=10.1093%2Fdnares%2Fdsx026&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f571%2f3868562" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=104086139 id="104086139" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The yeast <em>Saccharomyces</em> are widely used to test ecological and evolutionary hypotheses. A large number of nuclear genomic DNA sequences are available, but mitochondrial genomic data are insufficient. We completed mitochondrial DNA (mtDNA) sequencing from Illumina MiSeq reads for all <em>Saccharomyces</em> species. All are circularly mapped molecules decreasing in size with phylogenetic distance from <em>Saccharomyces cerevisiae</em> but with similar gene content including regulatory and selfish elements like origins of replication, introns, free-standing open reading frames or GC clusters. Their most profound feature is species-specific alteration in gene order. The genetic code slightly differs from well-established yeast mitochondrial code as GUG is used rarely as the translation start and CGA and CGC code for arginine. The multilocus phylogeny, inferred from mtDNA, does not correlate with the trees derived from nuclear genes. mtDNA data demonstrate that <em>Saccharomyces cariocanus</em> should be assigned as a separate species and <em>Saccharomyces bayanus</em> CBS 380<sup>T</sup> should not be considered as a distinct species due to mtDNA nearly identical to <em>Saccharomyces uvarum</em> mtDNA. Apparently, comparison of mtDNAs should not be neglected in genomic studies as it is an important tool to understand the origin and evolutionary history of some yeast species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mitochondrial DNA&quot;">mitochondrial DNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="yeasts">yeasts</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic code&quot;">genetic code</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="introns">introns</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogeny">phylogeny</a></div>                            <h2 scrollto-destination=104086141 id="104086141" class="section-title" >1. Introduction</h2>
<p>Because of advanced genetics and simple and easy production of mutants, the yeast <em>Saccharomyces cerevisiae</em> is the most widely used unicellular eukaryotic model organism. It is the best known member of the genus <em>Saccharomyces</em> with accepted species: <em>S. cerevisiae</em>, <em>S. cariocanus</em>, <em>S. paradoxus</em>, <em>S. mikatae</em>, <em>S. kudriavzevii</em>, <em>S. arboricolus</em>, <em>S. bayanus var. bayanus</em>, <em>S. bayanus var. uvarum</em> and <em>S. pastorianus</em>.<a reveal-id="dsx026-B1" data-open="dsx026-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> These yeast species are involved in fermenting sugars and produce enormous amounts of ethanol; however, most of the strains used to manufacture beer, wine or fuel ethanol are interspecific hybrids.<a reveal-id="dsx026-B2 dsx026-B3 dsx026-B4" data-open="dsx026-B2 dsx026-B3 dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>2–4</sup></a> Therefore, the taxonomic classifications of alloploid species <em>S. pastorianus</em> and <em>S. bayanus</em> do not agree with the outcome of whole genome sequencing.<a reveal-id="dsx026-B4" data-open="dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx026-B5" data-open="dsx026-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a></p><p>These discrepancies arose exclusively from the comparison of nuclear chromosomes. The data for much smaller mitochondrial genomes encoding only eight proteins required for oxidative phosphorylation, one ribosomal protein, two ribosomal RNAs (rRNA), the RNA component of RNase P, and a set of 24 tRNA genes were largely overlooked or unavailable.<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx026-B7" data-open="dsx026-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Despite large numbers of nuclear genome sequences now accessible for different <em>Saccharomyces</em> species and isolates<a reveal-id="dsx026-B4" data-open="dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx026-B8" data-open="dsx026-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><sup>–</sup><a reveal-id="dsx026-B10" data-open="dsx026-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> completed mitochondrial genome sequences are known only for <em>S. cerevisiae</em>,<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><em>S. paradoxus</em>,<a reveal-id="dsx026-B11" data-open="dsx026-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><em>S. pastorianus</em><a reveal-id="dsx026-B12" data-open="dsx026-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and <em>Saccharomyces eubayanus</em>.<a reveal-id="dsx026-B5" data-open="dsx026-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Recently, mitochondrial DNA (mtDNA) from <em>S. bayanus</em> and <em>S. pastorianus</em> were sequenced but not annotated.<a reveal-id="dsx026-B13" data-open="dsx026-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The problem is associated with the short DNA reads (100–200 nt) generated by the most next-generation sequencing platforms in genomic studies, especially by Illumina MiSeq.<a reveal-id="dsx026-B14 dsx026-B15 dsx026-B16 dsx026-B17" data-open="dsx026-B14 dsx026-B15 dsx026-B16 dsx026-B17" class="link link-ref link-reveal xref-bibr"><sup>14–17</sup></a> This approach can be used routinely for sequencing mtDNA from different yeast species<a reveal-id="dsx026-B18" data-open="dsx026-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> but not from many species of <em>Saccharomyces</em>.<a reveal-id="dsx026-B9" data-open="dsx026-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx026-B16" data-open="dsx026-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx026-B17" data-open="dsx026-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Their mitochondrial genomes can be assembled only by combination of different gap-filling programmes<a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> or by extensive manual editing.<a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx026-B21" data-open="dsx026-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p><p>The purpose of this work was to obtain complete mitochondrial genomes for all species assigned to <em>Saccharomyces</em> with the aim of comparing their genetic content to that of <em>S. cerevisiae</em>, as many different <em>Saccharomyces</em> mtDNAs can substitute, to some extent, for the original molecule in <em>S. cerevisiae</em>.<a reveal-id="dsx026-B22" data-open="dsx026-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p>                            <h2 scrollto-destination=104086145 id="104086145" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=104086146 id="104086146" class="section-title" >2.1. Yeast strains and mtDNA sequencing</h3>
<p>Sequenced strains are being used as taxonomic standards (type strains)<a reveal-id="dsx026-B1" data-open="dsx026-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> in whole genome studies,<a reveal-id="dsx026-B4" data-open="dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx026-B10" data-open="dsx026-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> or their mtDNA is being transferred to <em>S. cerevisiae</em> in order to prepare xenomitochondrial cybrids.<a reveal-id="dsx026-B22" data-open="dsx026-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Their origin is described in detail in reference.<a reveal-id="dsx026-B22" data-open="dsx026-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><em>Saccharomyces arboricolus</em> NRRL Y-63701 and <em>S. pastorianus</em> NRRL Y-27171<sup>NT</sup> were kindly provided by the Agricultural Research Service Culture Collection, US Department of Agriculture, Peoria, IL, USA. Yeast cells were grown at 28 °C in YPD medium (2% glucose, 1% yeast extract and 1% peptone). The mtDNA was purified mainly by differential centrifugation<a reveal-id="dsx026-B23" data-open="dsx026-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> or alternatively by bisbenzimide/CsCl buoyant density centrifugation.<a reveal-id="dsx026-B24" data-open="dsx026-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsx026-B25" data-open="dsx026-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> mtDNA sequencing libraries were prepared using the Nextera Library Preparation Kit (Illumina, San Diego, USA) and sequenced with average genome coverage between 176 and 3,698× on an Illumina MiSeq sequencing system using a MiSeq Reagent kit v3 (Illumina) with reads 2 × 100 bp in length. The sequencing summary is listed in <a reveal-id="dsx026-T1" data-open="dsx026-T1" class="link link-reveal link-table xref-fig">Table 1</a>. <div content-id="dsx026-T1" class="table-modal"><div class="table-wrap"><div id="dsx026-T1" data-id="dsx026-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Sequencing summary</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain </th><th>Paired reads </th><th>Average length </th><th>Coverage X<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>DNA </th><th>Contigs/nts </th><th>Coverage % </th><th>mtDNA size </th></tr></thead><tbody><tr><td><em>S. cerevisiae</em> NRRL Y-12632<sup>T</sup> </td><td>2,578,141 </td><td>147 </td><td>4,234 </td><td>Df </td><td>5/87,765 </td><td>98 </td><td>89,507 </td></tr><tr><td><em>S. paradoxus</em> CBS 2908 </td><td>1,580,034 </td><td>96 </td><td>2,283 </td><td>Gr </td><td>5/66,147 </td><td>99.6 </td><td>66,436 </td></tr><tr><td><em>S. paradoxus</em> CBS 7400 </td><td>2,242,369 </td><td>96 </td><td>3,014 </td><td>Gr </td><td>2/70,538 </td><td>98.7 </td><td>71,419 </td></tr><tr><td><em>S. cariocanus</em> CBS 7994<sup>T</sup> </td><td>1,897,384 </td><td>96 </td><td>2,354 </td><td>Gr </td><td>10/75,729 </td><td>98 </td><td>77,380 </td></tr><tr><td><em>S. mikatae</em> CBS 8839<sup>T</sup> </td><td>413,405 </td><td>189 </td><td>917 </td><td>Df </td><td>2/85,137 </td><td>99.9 </td><td>85,211 </td></tr><tr><td><em>S. kudriavzevii</em> CBS 8840<sup>T</sup> </td><td>2,259,934 </td><td>96 </td><td>2,692 </td><td>Gr </td><td>3/80,240 </td><td>99.5 </td><td>80,588 </td></tr><tr><td><em>S. arboricolus</em> NRRL Y-63701 </td><td>535,328 </td><td>139 </td><td>1,073 </td><td>Df </td><td>9/70,040 </td><td>101 </td><td>69,363 </td></tr><tr><td><em>S. pastorianus</em> NRRL Y-27171<sup>T</sup> </td><td>555,165 </td><td>158 </td><td>1,271 </td><td>Df </td><td>7/68,800 </td><td>99.7 </td><td>69,019 </td></tr><tr><td><em>S. bayanus</em> CBS 380<sup>T</sup> </td><td>1,221,753 </td><td>85 </td><td>1,604 </td><td>Gr </td><td>1/64,742 </td><td>100 </td><td>64,736 </td></tr><tr><td><em>S. uvarum</em> CBS 395<sup>T</sup><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>25,298,820 </td><td>36 </td><td>14,059 </td><td>– </td><td>34/51,450 </td><td>79.4 </td><td>64,779 </td></tr></tbody></table></div><div content-id="dsx026-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain </th><th>Paired reads </th><th>Average length </th><th>Coverage X<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>DNA </th><th>Contigs/nts </th><th>Coverage % </th><th>mtDNA size </th></tr></thead><tbody><tr><td><em>S. cerevisiae</em> NRRL Y-12632<sup>T</sup> </td><td>2,578,141 </td><td>147 </td><td>4,234 </td><td>Df </td><td>5/87,765 </td><td>98 </td><td>89,507 </td></tr><tr><td><em>S. paradoxus</em> CBS 2908 </td><td>1,580,034 </td><td>96 </td><td>2,283 </td><td>Gr </td><td>5/66,147 </td><td>99.6 </td><td>66,436 </td></tr><tr><td><em>S. paradoxus</em> CBS 7400 </td><td>2,242,369 </td><td>96 </td><td>3,014 </td><td>Gr </td><td>2/70,538 </td><td>98.7 </td><td>71,419 </td></tr><tr><td><em>S. cariocanus</em> CBS 7994<sup>T</sup> </td><td>1,897,384 </td><td>96 </td><td>2,354 </td><td>Gr </td><td>10/75,729 </td><td>98 </td><td>77,380 </td></tr><tr><td><em>S. mikatae</em> CBS 8839<sup>T</sup> </td><td>413,405 </td><td>189 </td><td>917 </td><td>Df </td><td>2/85,137 </td><td>99.9 </td><td>85,211 </td></tr><tr><td><em>S. kudriavzevii</em> CBS 8840<sup>T</sup> </td><td>2,259,934 </td><td>96 </td><td>2,692 </td><td>Gr </td><td>3/80,240 </td><td>99.5 </td><td>80,588 </td></tr><tr><td><em>S. arboricolus</em> NRRL Y-63701 </td><td>535,328 </td><td>139 </td><td>1,073 </td><td>Df </td><td>9/70,040 </td><td>101 </td><td>69,363 </td></tr><tr><td><em>S. pastorianus</em> NRRL Y-27171<sup>T</sup> </td><td>555,165 </td><td>158 </td><td>1,271 </td><td>Df </td><td>7/68,800 </td><td>99.7 </td><td>69,019 </td></tr><tr><td><em>S. bayanus</em> CBS 380<sup>T</sup> </td><td>1,221,753 </td><td>85 </td><td>1,604 </td><td>Gr </td><td>1/64,742 </td><td>100 </td><td>64,736 </td></tr><tr><td><em>S. uvarum</em> CBS 395<sup>T</sup><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>25,298,820 </td><td>36 </td><td>14,059 </td><td>– </td><td>34/51,450 </td><td>79.4 </td><td>64,779 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Coverage calculated as <em>LN</em>/<em>G</em>, where <em>L</em> is the read length, <em>N</em> is the number of reads and <em>G</em> is the mitochondrial genome length.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Assembled from SRR147290 reads downloaded from the Sequence Read Archive, NCBI.<a reveal-id="dsx026-B26" data-open="dsx026-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Gr—DNA from bisbenzimide/CsCl gradient<a reveal-id="dsx026-B25" data-open="dsx026-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>; Df—DNA from differential centrifugation.<a reveal-id="dsx026-B23" data-open="dsx026-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p></div></div></div></div></div><div class="table-wrap"><div id="dsx026-T1" data-id="dsx026-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Sequencing summary</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain </th><th>Paired reads </th><th>Average length </th><th>Coverage X<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>DNA </th><th>Contigs/nts </th><th>Coverage % </th><th>mtDNA size </th></tr></thead><tbody><tr><td><em>S. cerevisiae</em> NRRL Y-12632<sup>T</sup> </td><td>2,578,141 </td><td>147 </td><td>4,234 </td><td>Df </td><td>5/87,765 </td><td>98 </td><td>89,507 </td></tr><tr><td><em>S. paradoxus</em> CBS 2908 </td><td>1,580,034 </td><td>96 </td><td>2,283 </td><td>Gr </td><td>5/66,147 </td><td>99.6 </td><td>66,436 </td></tr><tr><td><em>S. paradoxus</em> CBS 7400 </td><td>2,242,369 </td><td>96 </td><td>3,014 </td><td>Gr </td><td>2/70,538 </td><td>98.7 </td><td>71,419 </td></tr><tr><td><em>S. cariocanus</em> CBS 7994<sup>T</sup> </td><td>1,897,384 </td><td>96 </td><td>2,354 </td><td>Gr </td><td>10/75,729 </td><td>98 </td><td>77,380 </td></tr><tr><td><em>S. mikatae</em> CBS 8839<sup>T</sup> </td><td>413,405 </td><td>189 </td><td>917 </td><td>Df </td><td>2/85,137 </td><td>99.9 </td><td>85,211 </td></tr><tr><td><em>S. kudriavzevii</em> CBS 8840<sup>T</sup> </td><td>2,259,934 </td><td>96 </td><td>2,692 </td><td>Gr </td><td>3/80,240 </td><td>99.5 </td><td>80,588 </td></tr><tr><td><em>S. arboricolus</em> NRRL Y-63701 </td><td>535,328 </td><td>139 </td><td>1,073 </td><td>Df </td><td>9/70,040 </td><td>101 </td><td>69,363 </td></tr><tr><td><em>S. pastorianus</em> NRRL Y-27171<sup>T</sup> </td><td>555,165 </td><td>158 </td><td>1,271 </td><td>Df </td><td>7/68,800 </td><td>99.7 </td><td>69,019 </td></tr><tr><td><em>S. bayanus</em> CBS 380<sup>T</sup> </td><td>1,221,753 </td><td>85 </td><td>1,604 </td><td>Gr </td><td>1/64,742 </td><td>100 </td><td>64,736 </td></tr><tr><td><em>S. uvarum</em> CBS 395<sup>T</sup><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>25,298,820 </td><td>36 </td><td>14,059 </td><td>– </td><td>34/51,450 </td><td>79.4 </td><td>64,779 </td></tr></tbody></table></div><div content-id="dsx026-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain </th><th>Paired reads </th><th>Average length </th><th>Coverage X<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>DNA </th><th>Contigs/nts </th><th>Coverage % </th><th>mtDNA size </th></tr></thead><tbody><tr><td><em>S. cerevisiae</em> NRRL Y-12632<sup>T</sup> </td><td>2,578,141 </td><td>147 </td><td>4,234 </td><td>Df </td><td>5/87,765 </td><td>98 </td><td>89,507 </td></tr><tr><td><em>S. paradoxus</em> CBS 2908 </td><td>1,580,034 </td><td>96 </td><td>2,283 </td><td>Gr </td><td>5/66,147 </td><td>99.6 </td><td>66,436 </td></tr><tr><td><em>S. paradoxus</em> CBS 7400 </td><td>2,242,369 </td><td>96 </td><td>3,014 </td><td>Gr </td><td>2/70,538 </td><td>98.7 </td><td>71,419 </td></tr><tr><td><em>S. cariocanus</em> CBS 7994<sup>T</sup> </td><td>1,897,384 </td><td>96 </td><td>2,354 </td><td>Gr </td><td>10/75,729 </td><td>98 </td><td>77,380 </td></tr><tr><td><em>S. mikatae</em> CBS 8839<sup>T</sup> </td><td>413,405 </td><td>189 </td><td>917 </td><td>Df </td><td>2/85,137 </td><td>99.9 </td><td>85,211 </td></tr><tr><td><em>S. kudriavzevii</em> CBS 8840<sup>T</sup> </td><td>2,259,934 </td><td>96 </td><td>2,692 </td><td>Gr </td><td>3/80,240 </td><td>99.5 </td><td>80,588 </td></tr><tr><td><em>S. arboricolus</em> NRRL Y-63701 </td><td>535,328 </td><td>139 </td><td>1,073 </td><td>Df </td><td>9/70,040 </td><td>101 </td><td>69,363 </td></tr><tr><td><em>S. pastorianus</em> NRRL Y-27171<sup>T</sup> </td><td>555,165 </td><td>158 </td><td>1,271 </td><td>Df </td><td>7/68,800 </td><td>99.7 </td><td>69,019 </td></tr><tr><td><em>S. bayanus</em> CBS 380<sup>T</sup> </td><td>1,221,753 </td><td>85 </td><td>1,604 </td><td>Gr </td><td>1/64,742 </td><td>100 </td><td>64,736 </td></tr><tr><td><em>S. uvarum</em> CBS 395<sup>T</sup><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>25,298,820 </td><td>36 </td><td>14,059 </td><td>– </td><td>34/51,450 </td><td>79.4 </td><td>64,779 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Coverage calculated as <em>LN</em>/<em>G</em>, where <em>L</em> is the read length, <em>N</em> is the number of reads and <em>G</em> is the mitochondrial genome length.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Assembled from SRR147290 reads downloaded from the Sequence Read Archive, NCBI.<a reveal-id="dsx026-B26" data-open="dsx026-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Gr—DNA from bisbenzimide/CsCl gradient<a reveal-id="dsx026-B25" data-open="dsx026-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>; Df—DNA from differential centrifugation.<a reveal-id="dsx026-B23" data-open="dsx026-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p></div></div></div></div></p>                            <h3 scrollto-destination=104086148 id="104086148" class="section-title" >2.2. mtDNA assembly and annotation</h3>
<p>Sequencing provided several hundred megabytes of data in more than one million reads, 90–190 nt long. Paired reads were trimmed, short reads (often with erroneous sequences) removed and assembled into individual contigs using CLC Genomics Workbench 9.5 (Qiagen, Hilden, DE). From thousands of contigs with an average size of ∼1,500 nt, only those containing mtDNA by BLASTN comparison to already known mtDNA from related yeasts (<em>S. cerevisiae</em> NC001224; <em>S. paradoxus</em> JQ862335, <em>S. pastorianus</em> EU852811) were selected. Individual contigs with mtDNA segments were assembled into a single molecule using the Vector NTI v.9.0 (v.10) software package from InforMax, Inc. Overall, 1–10 contigs were obtained covering 98–101% of the entire sequence, depending on the purification method (<a reveal-id="dsx026-T1" data-open="dsx026-T1" class="link link-reveal link-table xref-fig">Table 1</a>). All mtDNAs were interrupted in the redundant intergenic regions, mostly in the origins of replication (<em>ori</em> ∼270 nt) and GC clusters. Contigs were manually edited and linked together from individual reads as previously reported.<a reveal-id="dsx026-B21" data-open="dsx026-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Some gaps were sealed by direct sequencing using fluorescent dye terminator sequencing chemistry. Assembly of <em>S. mikatae</em> CBS 8839 (NRRL Y-27341<sup>T</sup>) mtDNA was confirmed by direct sequencing as previously described.<a reveal-id="dsx026-B11" data-open="dsx026-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> The assembled mitochondrial genomes were evaluated by mapping back the raw reads using CLC Genomics Workbench 9.5. However, a few substitutions, indels, and the occasional GC cluster can be found if our mtDNA is aligned to the mtDNA obtained from the same strains by different groups.<a reveal-id="dsx026-B13" data-open="dsx026-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p><p>Rough gene annotation was carried out with MFannot (<a class="link link-uri" href="http://megasun.bch.umontreal.ca/cgi-bin/mfannot/mfannotInterface.pl" target="_blank">http://megasun.bch.umontreal.ca/cgi-bin/mfannot/mfannotInterface.pl</a>).<a reveal-id="dsx026-B27" data-open="dsx026-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Precise exon and intron positions were corrected according to comparison with annotated sequences from several strains of <em>S. cerevisiae</em> and <em>S. paradoxus</em>.<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx026-B11" data-open="dsx026-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Intron and endonuclease nomenclature was according to<a reveal-id="dsx026-B28" data-open="dsx026-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and the intergenic open reading frames (ORFs) are numbered according to references.<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx026-B29" data-open="dsx026-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The sizes of rRNA were inferred from the model elaborated for their equivalents from <em>S. cerevisiae</em> mitochondria.<a reveal-id="dsx026-B30" data-open="dsx026-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsx026-B31" data-open="dsx026-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> The 5′ and 3′ ends of <em>rnp</em>B RNA were deduced from the secondary structure consensus.<a reveal-id="dsx026-B32" data-open="dsx026-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Gene nomenclature followed the rules described in GOBASE.<a reveal-id="dsx026-B33" data-open="dsx026-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a></p><p><em>Saccharomyces arboricolus</em> CBS 10644 mtDNA was assembled from the sequence CM001579<a reveal-id="dsx026-B9" data-open="dsx026-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> by manual editing from individual reads (EMBL: ERP001702, ERP001703, ERP001704). <em>Saccharomyces uvarum</em> CBS 395<sup>T</sup> was assembled from SRR147290 reads downloaded from the Sequence Read Archive, NCBI.<a reveal-id="dsx026-B26" data-open="dsx026-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>                            <h3 scrollto-destination=104086152 id="104086152" class="section-title" >2.3. Phylogeny</h3>
<p>Phylogenetic relationships were analysed by the Maximum likelihood phylogeny PhyML program included in the CLC Genomics Workbench 9.5 package.<a reveal-id="dsx026-B34" data-open="dsx026-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Phylogenetic trees were constructed from unambiguously aligned portions of the concatenated DNA sequences coding for proteins using the neighbor-joining algorithm and the best model (GTR + G + I) selected by model testing tool (hLRT, BIC, AIC, AICc).<a reveal-id="dsx026-B34" data-open="dsx026-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The stability of individual branches was assessed by the bootstrap method.<a reveal-id="dsx026-B35" data-open="dsx026-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> DNA concatemers for mitochondrial protein coding sequences belonging to other <em>Saccharomyces</em> strains were extracted from GenBank deposits. The nuclear DNA-derived phylogenetic tree was constructed from protein-coding genes used in population studies.<a reveal-id="dsx026-B36 dsx026-B37 dsx026-B38 dsx026-B39" data-open="dsx026-B36 dsx026-B37 dsx026-B38 dsx026-B39" class="link link-ref link-reveal xref-bibr"><sup>36–39</sup></a> Because of the hybrid alloploid nature of <em>S. bayanus</em>, <em>S. uvarum</em> and <em>S. pastorianus</em> we used gene sequences from the species that are the least related to <em>S. cerevisiae</em> variants. Corresponding DNA was obtained from assembled contigs occasionally sealed manually from reads (<em>S. cerevisiae</em> NRRL Y-12632<sup>T</sup>, <em>S. cariocanus</em> CBS 7994<sup>T</sup>, <em>S. arboricolus</em> NRRL Y-63701, <em>S. paradoxus</em> CBS 2908, <em>S. bayanus</em> CBS 380<sup>T</sup>). Other concatemers were obtained from sequences, scaffolds (contigs) deposited in the <em>Saccharomyces</em> genome database (<a class="link link-uri" href="http://www.yeastgenome.org/" target="_blank">http://www.yeastgenome.org/</a>) for <em>S. cerevisiae</em> S288c, <em>S. paradoxus</em> CBS 432,<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> for <em>S. pastorianus</em> Weihenstephan 34/70,<a reveal-id="dsx026-B12" data-open="dsx026-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> for <em>S. kudriavzevii</em> CBS 8840<sup>T</sup> and <em>S. mikatae</em> CBS 8839<a reveal-id="dsx026-B40" data-open="dsx026-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and for <em>S. arboricolus</em> CBS 10644.<a reveal-id="dsx026-B9" data-open="dsx026-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Data for <em>S. bayanus</em> CBS 380 were obtained from contigs assembled from SRR147291 and for <em>S. uvarum</em> CBS 395<sup>T</sup> were obtained from contigs assembled from SRR147290.<a reveal-id="dsx026-B26" data-open="dsx026-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Data for <em>S. pastorianus</em> NRRL Y-27171<sup>NT</sup> (CBS 1538) were compiled from contigs assembled by us and scaffolds BBYX01000001.<a reveal-id="dsx026-B13" data-open="dsx026-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Data for <em>S. eubayanus</em> CBS 12357<sup>T</sup> were obtained from the final genome assembly and annotations in GenBank JMCK00000000.<a reveal-id="dsx026-B5" data-open="dsx026-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Data accessibility. DNA sequences: GenBank accession nos KY095834–KY095933.</p>                            <h2 scrollto-destination=104086154 id="104086154" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=104086155 id="104086155" class="section-title" >3.1. General features of genes of the genetic code and GUG as a translation start</h3>
<p>We have successfully completed mitochondrial genomes for nine <em>Saccharomyces</em> species, and their general features are listed in <a reveal-id="dsx026-T2" data-open="dsx026-T2" class="link link-reveal link-table xref-fig">Table 2</a> (GenBank accession numbers KX657740–KX657750). All mtDNAs are circularly mapped and their size decreases with phylogenetic distance from <em>S. cerevisiae</em> (89.5 kb) to ∼64.7 kb for <em>S. bayanus</em>. GC content varies from 14% for <em>S. paradoxus</em> strains to 19.3% for <em>S. pastorianus</em> and correlates positively with the presence of introns and negatively with the size of AT-rich intergenic spacers. The mtDNAs contain the basic set of genes known in <em>Saccharomyces</em>, coding for the components of cytochrome oxidase (<em>cox1</em>, <em>cox2</em>, <em>cox3</em>), cytochrome b (<em>cob</em>), subunits of ATPase (<em>atp6</em>, <em>atp8</em>, <em>atp9</em>), both rRNA subunits (<em>rnl</em>, <em>rns</em>), the <em>rps3</em> gene for ribosomal protein, the <em>rnpB</em> gene for the RNA subunit of RNase P and the tRNA package<a reveal-id="dsx026-B24" data-open="dsx026-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> (<a reveal-id="dsx026-F1" data-open="dsx026-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). <div content-id="dsx026-T2" class="table-modal"><div class="table-wrap"><div id="dsx026-T2" data-id="dsx026-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>General features of mitochondrial genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Strain </th><th>Size (bp) </th><th>GC (%) </th><th>Genes (%) </th><th>Spacers (%) </th><th>Introns (%) </th><th>Accession number </th></tr></thead><tbody><tr><td><em>S. cerevisiae</em> </td><td>NRRL Y-12632<sup>NT</sup> </td><td>89,507 </td><td>16.8 </td><td>15.6 </td><td>64.1 </td><td>20.3 </td><td>KX657745 </td></tr><tr><td><em>S. paradoxus</em> </td><td>CBS 2908 </td><td>66,436 </td><td>14.0 </td><td>20.3 </td><td>70.7 </td><td>9.0 </td><td>KX657748 </td></tr><tr><td><em>S. paradoxus</em> </td><td>CBS 7400 </td><td>71,419 </td><td>14.5 </td><td>18.9 </td><td>65.6 </td><td>15.6 </td><td>KX657749 </td></tr><tr><td><em>S. cariocanus</em> </td><td>CBS 7994<sup>T</sup> </td><td>77,380 </td><td>15.6 </td><td>17.8 </td><td>69.8 </td><td>12.5 </td><td>KX657744 </td></tr><tr><td><em>S. mikatae</em> </td><td>CBS 8839<sup>T</sup> </td><td>85,211 </td><td>16.1 </td><td>16.1 </td><td>62.5 </td><td>21.4 </td><td>KX657747 </td></tr><tr><td><em>S. kudriavzevii</em> </td><td>CBS 8840<sup>T</sup> </td><td>80,588 </td><td>16.0 </td><td>17.1 </td><td>68.9 </td><td>13.9 </td><td>KX657746 </td></tr><tr><td><em>S. arboricolus</em> </td><td>NRRL Y-63701 </td><td>69,363 </td><td>14.6 </td><td>19.7 </td><td>71.4 </td><td>8.9 </td><td>KX657741 </td></tr><tr><td>S. arboricolus<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>CBS 10644<sup>T</sup> </td><td>71,124 </td><td>14.5 </td><td>19.1 </td><td>69.6 </td><td>11.3 </td><td>KX657740<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td></tr><tr><td><em>S. pastorianus</em> </td><td>NRRL Y-27171<sup>NT</sup> </td><td>69,019 </td><td>19.3 </td><td>19.9 </td><td>65.9 </td><td>14.2 </td><td>KX657750 </td></tr><tr><td><em>S. bayanus</em> </td><td>CBS 380<sup>T</sup> </td><td>64,736 </td><td>16.3 </td><td>21.0 </td><td>63.9 </td><td>15.1 </td><td>KX657743 </td></tr><tr><td><em>S. uvarum</em> </td><td>CBS 395<sup>T</sup> </td><td>64,779 </td><td>16.4 </td><td>21.0 </td><td>63.9 </td><td>15.1 </td><td>KX657742 </td></tr></tbody></table></div><div content-id="dsx026-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Strain </th><th>Size (bp) </th><th>GC (%) </th><th>Genes (%) </th><th>Spacers (%) </th><th>Introns (%) </th><th>Accession number </th></tr></thead><tbody><tr><td><em>S. cerevisiae</em> </td><td>NRRL Y-12632<sup>NT</sup> </td><td>89,507 </td><td>16.8 </td><td>15.6 </td><td>64.1 </td><td>20.3 </td><td>KX657745 </td></tr><tr><td><em>S. paradoxus</em> </td><td>CBS 2908 </td><td>66,436 </td><td>14.0 </td><td>20.3 </td><td>70.7 </td><td>9.0 </td><td>KX657748 </td></tr><tr><td><em>S. paradoxus</em> </td><td>CBS 7400 </td><td>71,419 </td><td>14.5 </td><td>18.9 </td><td>65.6 </td><td>15.6 </td><td>KX657749 </td></tr><tr><td><em>S. cariocanus</em> </td><td>CBS 7994<sup>T</sup> </td><td>77,380 </td><td>15.6 </td><td>17.8 </td><td>69.8 </td><td>12.5 </td><td>KX657744 </td></tr><tr><td><em>S. mikatae</em> </td><td>CBS 8839<sup>T</sup> </td><td>85,211 </td><td>16.1 </td><td>16.1 </td><td>62.5 </td><td>21.4 </td><td>KX657747 </td></tr><tr><td><em>S. kudriavzevii</em> </td><td>CBS 8840<sup>T</sup> </td><td>80,588 </td><td>16.0 </td><td>17.1 </td><td>68.9 </td><td>13.9 </td><td>KX657746 </td></tr><tr><td><em>S. arboricolus</em> </td><td>NRRL Y-63701 </td><td>69,363 </td><td>14.6 </td><td>19.7 </td><td>71.4 </td><td>8.9 </td><td>KX657741 </td></tr><tr><td>S. arboricolus<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>CBS 10644<sup>T</sup> </td><td>71,124 </td><td>14.5 </td><td>19.1 </td><td>69.6 </td><td>11.3 </td><td>KX657740<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td></tr><tr><td><em>S. pastorianus</em> </td><td>NRRL Y-27171<sup>NT</sup> </td><td>69,019 </td><td>19.3 </td><td>19.9 </td><td>65.9 </td><td>14.2 </td><td>KX657750 </td></tr><tr><td><em>S. bayanus</em> </td><td>CBS 380<sup>T</sup> </td><td>64,736 </td><td>16.3 </td><td>21.0 </td><td>63.9 </td><td>15.1 </td><td>KX657743 </td></tr><tr><td><em>S. uvarum</em> </td><td>CBS 395<sup>T</sup> </td><td>64,779 </td><td>16.4 </td><td>21.0 </td><td>63.9 </td><td>15.1 </td><td>KX657742 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Assembled from individual reads (EMBL: ERP001702, ERP001703, ERP001704).<a reveal-id="dsx026-B9" data-open="dsx026-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p></div></div></div></div></div><div class="table-wrap"><div id="dsx026-T2" data-id="dsx026-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>General features of mitochondrial genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Strain </th><th>Size (bp) </th><th>GC (%) </th><th>Genes (%) </th><th>Spacers (%) </th><th>Introns (%) </th><th>Accession number </th></tr></thead><tbody><tr><td><em>S. cerevisiae</em> </td><td>NRRL Y-12632<sup>NT</sup> </td><td>89,507 </td><td>16.8 </td><td>15.6 </td><td>64.1 </td><td>20.3 </td><td>KX657745 </td></tr><tr><td><em>S. paradoxus</em> </td><td>CBS 2908 </td><td>66,436 </td><td>14.0 </td><td>20.3 </td><td>70.7 </td><td>9.0 </td><td>KX657748 </td></tr><tr><td><em>S. paradoxus</em> </td><td>CBS 7400 </td><td>71,419 </td><td>14.5 </td><td>18.9 </td><td>65.6 </td><td>15.6 </td><td>KX657749 </td></tr><tr><td><em>S. cariocanus</em> </td><td>CBS 7994<sup>T</sup> </td><td>77,380 </td><td>15.6 </td><td>17.8 </td><td>69.8 </td><td>12.5 </td><td>KX657744 </td></tr><tr><td><em>S. mikatae</em> </td><td>CBS 8839<sup>T</sup> </td><td>85,211 </td><td>16.1 </td><td>16.1 </td><td>62.5 </td><td>21.4 </td><td>KX657747 </td></tr><tr><td><em>S. kudriavzevii</em> </td><td>CBS 8840<sup>T</sup> </td><td>80,588 </td><td>16.0 </td><td>17.1 </td><td>68.9 </td><td>13.9 </td><td>KX657746 </td></tr><tr><td><em>S. arboricolus</em> </td><td>NRRL Y-63701 </td><td>69,363 </td><td>14.6 </td><td>19.7 </td><td>71.4 </td><td>8.9 </td><td>KX657741 </td></tr><tr><td>S. arboricolus<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>CBS 10644<sup>T</sup> </td><td>71,124 </td><td>14.5 </td><td>19.1 </td><td>69.6 </td><td>11.3 </td><td>KX657740<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td></tr><tr><td><em>S. pastorianus</em> </td><td>NRRL Y-27171<sup>NT</sup> </td><td>69,019 </td><td>19.3 </td><td>19.9 </td><td>65.9 </td><td>14.2 </td><td>KX657750 </td></tr><tr><td><em>S. bayanus</em> </td><td>CBS 380<sup>T</sup> </td><td>64,736 </td><td>16.3 </td><td>21.0 </td><td>63.9 </td><td>15.1 </td><td>KX657743 </td></tr><tr><td><em>S. uvarum</em> </td><td>CBS 395<sup>T</sup> </td><td>64,779 </td><td>16.4 </td><td>21.0 </td><td>63.9 </td><td>15.1 </td><td>KX657742 </td></tr></tbody></table></div><div content-id="dsx026-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Strain </th><th>Size (bp) </th><th>GC (%) </th><th>Genes (%) </th><th>Spacers (%) </th><th>Introns (%) </th><th>Accession number </th></tr></thead><tbody><tr><td><em>S. cerevisiae</em> </td><td>NRRL Y-12632<sup>NT</sup> </td><td>89,507 </td><td>16.8 </td><td>15.6 </td><td>64.1 </td><td>20.3 </td><td>KX657745 </td></tr><tr><td><em>S. paradoxus</em> </td><td>CBS 2908 </td><td>66,436 </td><td>14.0 </td><td>20.3 </td><td>70.7 </td><td>9.0 </td><td>KX657748 </td></tr><tr><td><em>S. paradoxus</em> </td><td>CBS 7400 </td><td>71,419 </td><td>14.5 </td><td>18.9 </td><td>65.6 </td><td>15.6 </td><td>KX657749 </td></tr><tr><td><em>S. cariocanus</em> </td><td>CBS 7994<sup>T</sup> </td><td>77,380 </td><td>15.6 </td><td>17.8 </td><td>69.8 </td><td>12.5 </td><td>KX657744 </td></tr><tr><td><em>S. mikatae</em> </td><td>CBS 8839<sup>T</sup> </td><td>85,211 </td><td>16.1 </td><td>16.1 </td><td>62.5 </td><td>21.4 </td><td>KX657747 </td></tr><tr><td><em>S. kudriavzevii</em> </td><td>CBS 8840<sup>T</sup> </td><td>80,588 </td><td>16.0 </td><td>17.1 </td><td>68.9 </td><td>13.9 </td><td>KX657746 </td></tr><tr><td><em>S. arboricolus</em> </td><td>NRRL Y-63701 </td><td>69,363 </td><td>14.6 </td><td>19.7 </td><td>71.4 </td><td>8.9 </td><td>KX657741 </td></tr><tr><td>S. arboricolus<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>CBS 10644<sup>T</sup> </td><td>71,124 </td><td>14.5 </td><td>19.1 </td><td>69.6 </td><td>11.3 </td><td>KX657740<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td></tr><tr><td><em>S. pastorianus</em> </td><td>NRRL Y-27171<sup>NT</sup> </td><td>69,019 </td><td>19.3 </td><td>19.9 </td><td>65.9 </td><td>14.2 </td><td>KX657750 </td></tr><tr><td><em>S. bayanus</em> </td><td>CBS 380<sup>T</sup> </td><td>64,736 </td><td>16.3 </td><td>21.0 </td><td>63.9 </td><td>15.1 </td><td>KX657743 </td></tr><tr><td><em>S. uvarum</em> </td><td>CBS 395<sup>T</sup> </td><td>64,779 </td><td>16.4 </td><td>21.0 </td><td>63.9 </td><td>15.1 </td><td>KX657742 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Assembled from individual reads (EMBL: ERP001702, ERP001703, ERP001704).<a reveal-id="dsx026-B9" data-open="dsx026-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p></div></div></div></div></p>                        <a id="104086157" scrollto-destination="104086157"></a>
<div id="dsx026-F1" data-id="dsx026-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f1.png?Expires=2147483647&Signature=bd7yWp1Xi57Pm6JZgfj1-hMSYC8BqwXkdt-WRwrjI0~OxKnUn7m5UozP6pnnBlsC5o-AkzQeYea1B-6OUH01n16-WP6DIq8JlNUH7WXbTB4lNzYZZNxABxB-XjeGnlosCLcq2sPf~SK6C478btuQaEjn8GwAKM73HicdUaSlStLqK8J1IzpOv5gT0RN1vj3qpKUUmLjhljvI7n9RKk7pHZI4m~8nIMT0vZlsSWOAGmsfTjqdHTuSONvGcOPKMK9vvaXK-IRaciAGLHtZN~7Uzq9GiuTGJpsh989h4fYvK6i6aFu9A-799ZzOA8WD8vGz8pztqulvgpjv6Sleg6hG2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The genetic organization of Saccharomyces mtDNA. For simpler comparison, the circular genomes, exported from Vector NTI, were aligned at the beginning of the large rRNA subunit (rnl). Protein-coding genes, ribosomal RNA, rnpB are marked as arrows and bar, tRNA genes as black lines, introns with white rectangles, intronic and free-standing ORFs by gray arrows and replication origins with black circles. Gene nomenclature follows the rules described in GOBASE (atp for ATP synthetase subunits, cox for cytochrome oxidase subunits, cob for cytochrome b, rns for small rRNA ribosomal subunit, rnl for large rRNA ribosomal subunit, T2, C, H, etc. for particular tRNA coding genes, rps3 for ribosomal protein and rnpB for the RNA subunit of RNase P). Sizes are given on the bottom line in kbp." path-from-xml="dsx026f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086157/dsx026f1.tif" path-from-xml="dsx026f1.tif" target="_blank">View large</a><a section="104086157" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f1.png?Expires=2147483647&Signature=H3rhJuUt4CZ7o~4oL7T0go3VdtJqEzP90U7oO6qfwteLI12oNkmxioRqQiT-NnI83vg5A~PA--LWm8nt7O5tlwMWSu5yzY7GoVuVSOucjuKSvXp1Ck3P2rlly8QZidt6Bz6Z5Xco~tEKcbwahoheSEdgSXV1zegx2MNExIOE~fr4yJAcGgjlODmak3e75oksxYZy5ukXqt0QzfcONGllM9cKUVOUHi-QRU92EjFW9PdKpWn5Y-IdHRSSEd59of6cTfGcQZ1bXgUgxxdVHguZlDNx1Lvgh~4KAc0RFtP-8oFnuWcc1E41fvmq3Di64MTPXvLJwnis2X1IVspY6mnlcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086157&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The genetic organization of <em>Saccharomyces</em> mtDNA. For simpler comparison, the circular genomes, exported from Vector NTI, were aligned at the beginning of the large rRNA subunit (<em>rnl</em>). Protein-coding genes, ribosomal RNA, <em>rnpB</em> are marked as arrows and bar, tRNA genes as black lines, introns with white rectangles, intronic and free-standing ORFs by gray arrows and replication origins with black circles. Gene nomenclature follows the rules described in GOBASE (<em>atp</em> for ATP synthetase subunits, <em>cox</em> for cytochrome oxidase subunits, <em>cob</em> for cytochrome b, <em>rns</em> for small rRNA ribosomal subunit, <em>rnl</em> for large rRNA ribosomal subunit, T2, C, H, etc. for particular tRNA coding genes, <em>rps3</em> for ribosomal protein and <em>rnpB</em> for the RNA subunit of RNase P). Sizes are given on the bottom line in kbp.</p></div></div><div content-id="dsx026-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f1.png?Expires=2147483647&Signature=bd7yWp1Xi57Pm6JZgfj1-hMSYC8BqwXkdt-WRwrjI0~OxKnUn7m5UozP6pnnBlsC5o-AkzQeYea1B-6OUH01n16-WP6DIq8JlNUH7WXbTB4lNzYZZNxABxB-XjeGnlosCLcq2sPf~SK6C478btuQaEjn8GwAKM73HicdUaSlStLqK8J1IzpOv5gT0RN1vj3qpKUUmLjhljvI7n9RKk7pHZI4m~8nIMT0vZlsSWOAGmsfTjqdHTuSONvGcOPKMK9vvaXK-IRaciAGLHtZN~7Uzq9GiuTGJpsh989h4fYvK6i6aFu9A-799ZzOA8WD8vGz8pztqulvgpjv6Sleg6hG2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The genetic organization of Saccharomyces mtDNA. For simpler comparison, the circular genomes, exported from Vector NTI, were aligned at the beginning of the large rRNA subunit (rnl). Protein-coding genes, ribosomal RNA, rnpB are marked as arrows and bar, tRNA genes as black lines, introns with white rectangles, intronic and free-standing ORFs by gray arrows and replication origins with black circles. Gene nomenclature follows the rules described in GOBASE (atp for ATP synthetase subunits, cox for cytochrome oxidase subunits, cob for cytochrome b, rns for small rRNA ribosomal subunit, rnl for large rRNA ribosomal subunit, T2, C, H, etc. for particular tRNA coding genes, rps3 for ribosomal protein and rnpB for the RNA subunit of RNase P). Sizes are given on the bottom line in kbp." path-from-xml="dsx026f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086157/dsx026f1.tif" path-from-xml="dsx026f1.tif" target="_blank">View large</a><a section="104086157" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f1.png?Expires=2147483647&Signature=H3rhJuUt4CZ7o~4oL7T0go3VdtJqEzP90U7oO6qfwteLI12oNkmxioRqQiT-NnI83vg5A~PA--LWm8nt7O5tlwMWSu5yzY7GoVuVSOucjuKSvXp1Ck3P2rlly8QZidt6Bz6Z5Xco~tEKcbwahoheSEdgSXV1zegx2MNExIOE~fr4yJAcGgjlODmak3e75oksxYZy5ukXqt0QzfcONGllM9cKUVOUHi-QRU92EjFW9PdKpWn5Y-IdHRSSEd59of6cTfGcQZ1bXgUgxxdVHguZlDNx1Lvgh~4KAc0RFtP-8oFnuWcc1E41fvmq3Di64MTPXvLJwnis2X1IVspY6mnlcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086157&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The genetic organization of <em>Saccharomyces</em> mtDNA. For simpler comparison, the circular genomes, exported from Vector NTI, were aligned at the beginning of the large rRNA subunit (<em>rnl</em>). Protein-coding genes, ribosomal RNA, <em>rnpB</em> are marked as arrows and bar, tRNA genes as black lines, introns with white rectangles, intronic and free-standing ORFs by gray arrows and replication origins with black circles. Gene nomenclature follows the rules described in GOBASE (<em>atp</em> for ATP synthetase subunits, <em>cox</em> for cytochrome oxidase subunits, <em>cob</em> for cytochrome b, <em>rns</em> for small rRNA ribosomal subunit, <em>rnl</em> for large rRNA ribosomal subunit, T2, C, H, etc. for particular tRNA coding genes, <em>rps3</em> for ribosomal protein and <em>rnpB</em> for the RNA subunit of RNase P). Sizes are given on the bottom line in kbp.</p></div></div><p>The genetic code in <em>Saccharomyces</em> mtDNA differs from the universal code by TGA being read as tryptophan, CTN as threonine, ATA as methionine and CGN for arginine, with CGA and CGC absent, that is recognized as the ‘yeast mitochondrial code’ (transl_table = 3).<a reveal-id="dsx026-B41 dsx026-B42 dsx026-B43" data-open="dsx026-B41 dsx026-B42 dsx026-B43" class="link link-ref link-reveal xref-bibr"><sup>41–43</sup></a> The <em>trnR2</em> gene (anticodon ACG) was found in all <em>Saccharomyces</em> mtDNAs. Thus, all four CGN codons possess coding potential. However, the CGU codon is present in exons only once (besides <em>S. arboricolus</em>) coding for a non-conserved arginine residue in <em>rps3</em> protein (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s1_dsx026.doc?Expires=2147483647&Signature=1NPSjXvfqnbox4N9CWZa9hD5xZslFbIbvGRK9RiHRCHuTDA0~7c45lMVSPIQVUyE-9pxJ9KouMUJRNuAdcb~s58URh14gec-kzVbmpogVcvMZrFJBiVK3Rx~rPWQcTGZGLMF389vonrMTo0M9cmx8RYAUvlJfKSGPmRGNnegIwtd-rRa62NykzkhGS0tac0Npt78Imuf6obxyN6eGc5xqsXXRNMJZFVqAE-WmIaRjTgKa53kL-BROzLKfIzhpBn4Q02K10aVLSgZx8NdVeM0PA7nU635v56YiLJMbtr4L082m9xGc2mX~xQe1-EyjoFE5ulcE3hfKiiVGTJ6YiTzsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S1). In addition, the CGU codon occurs quite frequently in intron-coded or free-standing ORFs, whereas the CGG codon is present only in GC clusters located inside them. The arginine CGA and CGC codons considered as unassigned are also used sporadically in maturase/homing endonucleases (HE) reading frames coded by introns<a reveal-id="dsx026-B41 dsx026-B42 dsx026-B43 dsx026-B44" data-open="dsx026-B41 dsx026-B42 dsx026-B43 dsx026-B44" class="link link-ref link-reveal xref-bibr"><sup>41–44</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s1_dsx026.doc?Expires=2147483647&Signature=1NPSjXvfqnbox4N9CWZa9hD5xZslFbIbvGRK9RiHRCHuTDA0~7c45lMVSPIQVUyE-9pxJ9KouMUJRNuAdcb~s58URh14gec-kzVbmpogVcvMZrFJBiVK3Rx~rPWQcTGZGLMF389vonrMTo0M9cmx8RYAUvlJfKSGPmRGNnegIwtd-rRa62NykzkhGS0tac0Npt78Imuf6obxyN6eGc5xqsXXRNMJZFVqAE-WmIaRjTgKa53kL-BROzLKfIzhpBn4Q02K10aVLSgZx8NdVeM0PA7nU635v56YiLJMbtr4L082m9xGc2mX~xQe1-EyjoFE5ulcE3hfKiiVGTJ6YiTzsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S1). The most convincing evidence is a CGC codon present in the <em>cox1</em>I2 reading frame in a number of <em>S. cerevisiae</em> strains including type strain NRRL Y-12632 and the well-studied model strain S288c. This reverse transcriptase/maturase protein is required for group II intron splicing from <em>cox1</em> pre-mRNA as well as for intron mobility.<a reveal-id="dsx026-B45" data-open="dsx026-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsx026-B46" data-open="dsx026-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Apparently, tRNA coded by the <em>trnR2</em> gene is required in <em>Saccharomyces</em> pre-dominantly for the expression of mobile elements and is therefore gained and lost in different clades from the <em>Saccharomyce</em>s/<em>Kluyveromyces</em> complex.<a reveal-id="dsx026-B47" data-open="dsx026-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsx026-B48" data-open="dsx026-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Protein-coding genes start with ATG, with some exceptions. The translation start in the <em>S. cariocanus cox3</em> gene is GTG (GUG), reported to be nearly as effective as the original ATG in the <em>S. cerevisiae cox2</em> gene.<a reveal-id="dsx026-B49" data-open="dsx026-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Translation initiation at the nearby downstream AUG is not an option because mutagenesis of the regular initiation codon does not allow translation from this site.<a reveal-id="dsx026-B49" data-open="dsx026-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Instead, a stem loop structure in the mRNA sequence of the first six codons plays a role in the recognition of the translation start.<a reveal-id="dsx026-B50" data-open="dsx026-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> The first 50 nucleotides in the <em>cox3</em> genes of all sequenced <em>Saccharomyces</em> are identical, which emphasizes the reliability of GUG as the initiation start codon (<a reveal-id="dsx026-F2" data-open="dsx026-F2" class="link link-reveal link-table xref-fig">Fig. 2A, B</a>). Atypical start codons are present in many mitochondrial systems, but GUG as a start codon is known only in some birds, invertebrates, plants and protists, but not in yeast and molds.<a reveal-id="dsx026-B43" data-open="dsx026-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsx026-B44" data-open="dsx026-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Also, a maternally inherited and practically homoplasmic mtDNA mutation, 8527 A &gt; G, changing the initiation codon AUG into GUG, was observed in the human <em>atp6</em> gene. The patient harbouring this mutation exhibited clinical symptoms of Leber hereditary optic neuropathy, but his mother was healthy.<a reveal-id="dsx026-B51" data-open="dsx026-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> The second exception is free-standing <em>ORF1</em>s in <em>S. kudriavzevii</em>, <em>S. mikatae</em>, <em>S. bayanus</em>, <em>S. uvarum</em>, <em>S. arboricolus</em> and <em>S. pastorianus</em>, where GUG is used as a translation start site. This gene with unknown function codes for a hypothetical maturase-like protein, the start codon overlaps with the 3′ end of the <em>cox2</em> gene and the reading frame extends 16 bp downstream of the <em>cox2</em> stop codon. Consequently, the ‘yeast mitochondrial code’ should be updated to accept CGA and CGC as arginine codons and GTG as the alternative initiation codon (<a reveal-id="dsx026-F2" data-open="dsx026-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). The preferable termination codon in exons is UAA, and UAG is used only occasionally in <em>cox2</em> of <em>S. kudriavzevii</em> and <em>S. mikatae</em> and more often in intron-coded ORFs. Codon usage in the reading frames shows a strong bias towards codons ending with T or A (in 10:1), as previously observed in the mitochondrial genomes of other yeasts<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx026-B11" data-open="dsx026-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s1_dsx026.doc?Expires=2147483647&Signature=1NPSjXvfqnbox4N9CWZa9hD5xZslFbIbvGRK9RiHRCHuTDA0~7c45lMVSPIQVUyE-9pxJ9KouMUJRNuAdcb~s58URh14gec-kzVbmpogVcvMZrFJBiVK3Rx~rPWQcTGZGLMF389vonrMTo0M9cmx8RYAUvlJfKSGPmRGNnegIwtd-rRa62NykzkhGS0tac0Npt78Imuf6obxyN6eGc5xqsXXRNMJZFVqAE-WmIaRjTgKa53kL-BROzLKfIzhpBn4Q02K10aVLSgZx8NdVeM0PA7nU635v56YiLJMbtr4L082m9xGc2mX~xQe1-EyjoFE5ulcE3hfKiiVGTJ6YiTzsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S1). </p>                        <a id="104086159" scrollto-destination="104086159"></a>
<div id="dsx026-F2" data-id="dsx026-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f2.png?Expires=2147483647&Signature=dMb0OgVTDf8jKPp5~uEx3wNC4TrC~l1M13uror1JTw7dUnj1aav8qKCqtSFXK1SniglLJPpSuu7NkGD9MvSjy-db-keR4dofZDHfLlT104bB4KpXxAbwQPMdZUDZc7diYN~FI1b8K~-AGPC8XCaSEDRfcQAo0fT6~4IuFLdtUhG-IDxV8wTuotuM3xNyJHKeIEm6VjqZl5DUo8dOZPxscKgFB8mWjwXQQ9QLWuqIkBB5rFRseQnNWYZVQkxp4KBlS8Qs0yoW09WgOIBcFa6HfgxAmOZi4ITxXTX9R4eDp91zOH~6WuugfTQu1BQYnbU5xe~snuaVubxvwn6REkJ3Sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GTG as the initiation codon and revised yeast mitochondrial code. (A) Initiation codons in cox3 gene. (B) Initiation codons in ORF1 gene. Sequences overlapping with cox2 3′end are underlined. (C) Revised yeast mitochondrial code. Exceptional codons are grey-shadowed." path-from-xml="dsx026f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086159/dsx026f2.tif" path-from-xml="dsx026f2.tif" target="_blank">View large</a><a section="104086159" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f2.png?Expires=2147483647&Signature=HYWlxjJOOSANAVTj7vu7bio9NZzAZvRsdjz7O6rbTDJdVokVFk2gbQXKLdHRToHw6nJ-GaQnAIR~IEjBZ51SCcDqqzEGAQ~XHLeo4pVeCO2vp2oYS6O3BQcOPaCR4PzKW8vFxhMkhG-VUSbusx1qKfqQGFizr1UTC6MLbUNGetXILO4XvTuWRLodFukb-fTP3qdJtmbzirALPX2uBH2Kp0LWPNRk4QvxV8svJQhjp1CVjucY9F9wwAbytZ93tCXeHG5SYtCKwiu49wxV0w9orBmsQImU0wATcRa9quAU9Im9HTCwkL8PyoXaAlLLbPPzPrnaji9hRR856BJIkIO8gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086159&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GTG as the initiation codon and revised yeast mitochondrial code. (A) Initiation codons in <em>cox3</em> gene. (B) Initiation codons in <em>ORF1</em> gene. Sequences overlapping with <em>cox2</em> 3′end are underlined. (C) Revised yeast mitochondrial code. Exceptional codons are grey-shadowed.</p></div></div><div content-id="dsx026-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f2.png?Expires=2147483647&Signature=dMb0OgVTDf8jKPp5~uEx3wNC4TrC~l1M13uror1JTw7dUnj1aav8qKCqtSFXK1SniglLJPpSuu7NkGD9MvSjy-db-keR4dofZDHfLlT104bB4KpXxAbwQPMdZUDZc7diYN~FI1b8K~-AGPC8XCaSEDRfcQAo0fT6~4IuFLdtUhG-IDxV8wTuotuM3xNyJHKeIEm6VjqZl5DUo8dOZPxscKgFB8mWjwXQQ9QLWuqIkBB5rFRseQnNWYZVQkxp4KBlS8Qs0yoW09WgOIBcFa6HfgxAmOZi4ITxXTX9R4eDp91zOH~6WuugfTQu1BQYnbU5xe~snuaVubxvwn6REkJ3Sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GTG as the initiation codon and revised yeast mitochondrial code. (A) Initiation codons in cox3 gene. (B) Initiation codons in ORF1 gene. Sequences overlapping with cox2 3′end are underlined. (C) Revised yeast mitochondrial code. Exceptional codons are grey-shadowed." path-from-xml="dsx026f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086159/dsx026f2.tif" path-from-xml="dsx026f2.tif" target="_blank">View large</a><a section="104086159" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f2.png?Expires=2147483647&Signature=HYWlxjJOOSANAVTj7vu7bio9NZzAZvRsdjz7O6rbTDJdVokVFk2gbQXKLdHRToHw6nJ-GaQnAIR~IEjBZ51SCcDqqzEGAQ~XHLeo4pVeCO2vp2oYS6O3BQcOPaCR4PzKW8vFxhMkhG-VUSbusx1qKfqQGFizr1UTC6MLbUNGetXILO4XvTuWRLodFukb-fTP3qdJtmbzirALPX2uBH2Kp0LWPNRk4QvxV8svJQhjp1CVjucY9F9wwAbytZ93tCXeHG5SYtCKwiu49wxV0w9orBmsQImU0wATcRa9quAU9Im9HTCwkL8PyoXaAlLLbPPzPrnaji9hRR856BJIkIO8gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086159&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GTG as the initiation codon and revised yeast mitochondrial code. (A) Initiation codons in <em>cox3</em> gene. (B) Initiation codons in <em>ORF1</em> gene. Sequences overlapping with <em>cox2</em> 3′end are underlined. (C) Revised yeast mitochondrial code. Exceptional codons are grey-shadowed.</p></div></div><p>All sequenced mitochondrial genomes exhibit sequence conservation at the nucleotide level within exons (excluding extremely variable <em>rps3</em>), with at least 94% sequence identity between the most unrelated species and a substitution rate of 11%. The substitutions are mostly neutral and have a low impact on the protein sequence, as they generate nearly identical amino acid residues. Therefore, the overall protein identity, even among less-related species, is 97%. Interestingly, the <em>atp9</em> protein sequence is identical in all sequenced species. The same DNA substitution rate (6%) in the other small gene, <em>atp8</em>, is responsible for the three changes at the protein level (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s2_dsx026.doc?Expires=2147483647&Signature=ehiRCfNIbPq~VXTo~lSqSGVcDaeRplhM3a3SmP28ip9fDy-A0-mrPi9Ypmuz-iKfRypxmYDUWqThlez9Ij2pqSukHFxGNq~C1hgWyQeUY80mRthyyr5S3OZtu9EhVRGkc-IMROs8xqIxXBdaLOJzWEsOp-sF9aznVIklnPFogNi2os~WZGWYrW3BN~Ve6dqqU-iG7aN8Y5noLaOgKWpfXVmH3oA~gCeBr7X-RucYUX1AFoDnt~U1q9SZvXZhTT9Nogwqt7QkDO8lJDmp07Crp19EN8W4J0pLxzTcTCzYesW-L8g9Bjeixt39XzGwbLysDskyGehUs2i7qu4cgTSWbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S2).</p>                            <h3 scrollto-destination=104086161 id="104086161" class="section-title" >3.2 Introns and free-standing ORFs</h3>
<p>Introns are present in <em>Saccharomyces</em> only within <em>cox1</em>, <em>cob</em> and <em>rnl</em> genes.<a reveal-id="dsx026-B22" data-open="dsx026-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsx026-B27" data-open="dsx026-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Although the majority are group I introns, four of them belong to group II, and only two of these (<em>cox1</em>I1, <em>cox1</em>I2) code for a protein with the maturase/reverse transcriptase motif. The intervening sequences are inserted into the same sites as has been described in other <em>Saccharomyces</em>, especially more than 100 strains of <em>S. cerevisiae</em><a reveal-id="dsx026-B11" data-open="dsx026-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx026-B21" data-open="dsx026-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsx026-B52" data-open="dsx026-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> (<a reveal-id="dsx026-F3" data-open="dsx026-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). However, a detailed analysis indicates that the insertion site for <em>cox1</em>I3α should be shifted to position 242. Position 243 would generate lysine instead of an asparagine residue in the GGFG<strong><span class="underline">N</span></strong> section in both <em>S. arboricolus</em> molecules. This motif is conserved across the tree of life from yeast to humans. <span class="underline">G-G-F-G-N</span>-[WY]-[FL]-[MV] is a part of the well-recognized water membrane interface motif in cytochrome c and quinol oxidases (MeMotif—<a class="link link-uri" href="http://projects.biotec.tu-dresden.de/memotif" target="_blank">http://projects.biotec.tu-dresden.de/memotif</a>).<a reveal-id="dsx026-B53" data-open="dsx026-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a></p>                        <a id="104086163" scrollto-destination="104086163"></a>
<div id="dsx026-F3" data-id="dsx026-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f3.png?Expires=2147483647&Signature=Mjn-uycGGuHK5F1adffgE7bvvuqcUeBw0BLgOhTO2vdVLvRGU01u7rPXsz7cuX5lCOFXEunRTR30RSKxxgZ4qJSkAXg-aGyn3Bswh~9iufEW6DaaGiThKbRfI6zjcQmZIWMOGcBVDDRZimTxJ2MM9anFanJ3~Y6CwCb2QEMA0WnNHUA7hiEm-Jw1IcYMox5AUhzgEdxzdpxzvedJssOfy7vlpsSxHTCRSyJfQCOul20u8UHU3i4ymjYx18C3HPOkgpWioAP~ndCs8ONnILuM-TaqGRQ16TocsibpkQwf7pE2G1QREkfERQ9G3teike6inMFDZkgebuIUbI43g2pGpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Occurrence of introns and free-standing ORFs. Introns and free-standing ORFs with at least one motif characteristic for the homing endonucleases (LAGLIDADG, GIY-YIG, etc.) are in black; without reading frame are white; with truncated ORF interrupted by stop codon are spotted. Numbers indicate the base preceding the intron insertion site in the CDS. Positions of introns, known as mobile, in S. cerevisiae are marked in bold6; I—group I introns, II—group II introns. ORF1, ORF2 and ORF4 correspond to the nomenclature used in reference29 (their ORF5 is an ORF coded by the rnl ω intron). Only ORFs containing at least one endonuclease/maturase motif were considered." path-from-xml="dsx026f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086163/dsx026f3.tif" path-from-xml="dsx026f3.tif" target="_blank">View large</a><a section="104086163" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f3.png?Expires=2147483647&Signature=oboE7Dhom9p0mtD1MBjd68-vZJUXtPhrK6Z~-b8owBWo~Af5Klv4~a5MXyntZWSbZ~-Ykm-ULJ5qGGdvp4V~FmVNN68w9uTq25Iur4TUMWsExQSrsPWAJCAbqbULdbOTA8LRd52NnyiGXgVJuIJr9Io4EzkX1bHQrMOacgrDSC~DdE5H43oBL~IduQFc-CTMki5Q9EsRpd0NqPd85yrl99PROQgrMQ8HPWK8RStrlsXznz0NCgjbfqWsEHSBef7IU6oj-SkDrctGcJ~mS8K-XVSZlAr4BgfcH-ASAgh6-83DGLdyl4JoMTr4~FBzGtoAQDATRO-sz-DxK3LF6cnIRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086163&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Occurrence of introns and free-standing ORFs. Introns and free-standing ORFs with at least one motif characteristic for the homing endonucleases (LAGLIDADG, GIY-YIG, etc.) are in black; without reading frame are white; with truncated ORF interrupted by stop codon are spotted. Numbers indicate the base preceding the intron insertion site in the CDS. Positions of introns, known as mobile, in <em>S. cerevisiae</em> are marked in bold<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a>; I—group I introns, II—group II introns. <em>ORF1</em>, <em>ORF2</em> and <em>ORF4</em> correspond to the nomenclature used in reference<a reveal-id="dsx026-B29" data-open="dsx026-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> (their <em>ORF5</em> is an ORF coded by the <em>rnl</em> ω intron). Only ORFs containing at least one endonuclease/maturase motif were considered.</p></div></div><div content-id="dsx026-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f3.png?Expires=2147483647&Signature=Mjn-uycGGuHK5F1adffgE7bvvuqcUeBw0BLgOhTO2vdVLvRGU01u7rPXsz7cuX5lCOFXEunRTR30RSKxxgZ4qJSkAXg-aGyn3Bswh~9iufEW6DaaGiThKbRfI6zjcQmZIWMOGcBVDDRZimTxJ2MM9anFanJ3~Y6CwCb2QEMA0WnNHUA7hiEm-Jw1IcYMox5AUhzgEdxzdpxzvedJssOfy7vlpsSxHTCRSyJfQCOul20u8UHU3i4ymjYx18C3HPOkgpWioAP~ndCs8ONnILuM-TaqGRQ16TocsibpkQwf7pE2G1QREkfERQ9G3teike6inMFDZkgebuIUbI43g2pGpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Occurrence of introns and free-standing ORFs. Introns and free-standing ORFs with at least one motif characteristic for the homing endonucleases (LAGLIDADG, GIY-YIG, etc.) are in black; without reading frame are white; with truncated ORF interrupted by stop codon are spotted. Numbers indicate the base preceding the intron insertion site in the CDS. Positions of introns, known as mobile, in S. cerevisiae are marked in bold6; I—group I introns, II—group II introns. ORF1, ORF2 and ORF4 correspond to the nomenclature used in reference29 (their ORF5 is an ORF coded by the rnl ω intron). Only ORFs containing at least one endonuclease/maturase motif were considered." path-from-xml="dsx026f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086163/dsx026f3.tif" path-from-xml="dsx026f3.tif" target="_blank">View large</a><a section="104086163" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f3.png?Expires=2147483647&Signature=oboE7Dhom9p0mtD1MBjd68-vZJUXtPhrK6Z~-b8owBWo~Af5Klv4~a5MXyntZWSbZ~-Ykm-ULJ5qGGdvp4V~FmVNN68w9uTq25Iur4TUMWsExQSrsPWAJCAbqbULdbOTA8LRd52NnyiGXgVJuIJr9Io4EzkX1bHQrMOacgrDSC~DdE5H43oBL~IduQFc-CTMki5Q9EsRpd0NqPd85yrl99PROQgrMQ8HPWK8RStrlsXznz0NCgjbfqWsEHSBef7IU6oj-SkDrctGcJ~mS8K-XVSZlAr4BgfcH-ASAgh6-83DGLdyl4JoMTr4~FBzGtoAQDATRO-sz-DxK3LF6cnIRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086163&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Occurrence of introns and free-standing ORFs. Introns and free-standing ORFs with at least one motif characteristic for the homing endonucleases (LAGLIDADG, GIY-YIG, etc.) are in black; without reading frame are white; with truncated ORF interrupted by stop codon are spotted. Numbers indicate the base preceding the intron insertion site in the CDS. Positions of introns, known as mobile, in <em>S. cerevisiae</em> are marked in bold<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a>; I—group I introns, II—group II introns. <em>ORF1</em>, <em>ORF2</em> and <em>ORF4</em> correspond to the nomenclature used in reference<a reveal-id="dsx026-B29" data-open="dsx026-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> (their <em>ORF5</em> is an ORF coded by the <em>rnl</em> ω intron). Only ORFs containing at least one endonuclease/maturase motif were considered.</p></div></div><p>A number of introns also contain ORFs in phase with the upstream exon, sometimes interrupted by a frame-shift stop codon, often due to the insertion of GC clusters. Only those possessing at least one motif characteristic for the HE (LAGLIDADG, GIY-YIG, etc.) were considered as potentially active<a reveal-id="dsx026-B54" data-open="dsx026-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><a reveal-id="dsx026-B55" data-open="dsx026-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> (<a reveal-id="dsx026-F3" data-open="dsx026-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>).</p><p>However, there are two exceptions: <em>cox1</em>I5β and <em>cob</em>I1α, where an ORF coding for a nuclease/maturase motif is not fused to the upstream exon. <em>cox1</em>I5β is not known as mobile and the splicing requires at least five different nuclei-coded factors and an ORF from <em>cob</em>I3.<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx026-B56" data-open="dsx026-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> The reason is that the 3′ splice site lies unusually far from the catalytic core of the intron; therefore, the ORF 357 amino acids (AA) long containing two LAGLIDADG motifs in the S288c strain was not considered as potentially active.<a reveal-id="dsx026-B56" data-open="dsx026-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> But a recent <em>S. cerevisiae</em> mitochondrial transcriptome study revealed an alternative splice site that puts the <em>cox1</em>I5β ORF in frame with the upstream exon.<a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> We found the same ACTTATTATATATT consensus of the alternative splice site in several other <em>Saccharomyces</em> species (<em>S. mikatae</em>, <em>S. cariocanus</em>, <em>S. bayanus</em>, <em>S. uvarum</em>, <em>S. pastorianus</em> strains and <em>S. eubayanus</em> and <em>S. paradoxus</em> CBS 432; <a reveal-id="dsx026-F4" data-open="dsx026-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). </p>                        <a id="104086166" scrollto-destination="104086166"></a>
<div id="dsx026-F4" data-id="dsx026-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f4.png?Expires=2147483647&Signature=vmm1PFj78hkQKb~6P5sPHYwN0CqGSx36PRjVflGfugxNo3pH08OKMP4~kHoHTW5dtq2BEWZNDm9qRt485u2CfbztIlaUp7m8zSKZepUM0tG9DpmxAGesE5EcVh2UEVmEHwR~N1oILlc3YXYM2~6E0wq8DqHvS76fRclXF6y5KcAx7TZn4akoemGYEnwr0j14N0FsHLT3OF7jrnd7OB8XPWpyYpe7pCkezTLyJzOhX632FOtWoXTumf5k0g8j3qnshtEMNgcAibSTFaCM50DnEqgrhRs5ftYo2~escBOa0RIA6U3CdF7C~8goOwG3C-5X1HBuhn5nfpWmZw986n-TdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alternative splicing of cox1I5β (A) and cobI1α (B). Arrows mark exons–introns boundaries. Consensus of alternative splice site as described in reference57 and hypothetical alternative splice site in cobI1α are underlined." path-from-xml="dsx026f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086166/dsx026f4.tif" path-from-xml="dsx026f4.tif" target="_blank">View large</a><a section="104086166" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f4.png?Expires=2147483647&Signature=hOE-kNQBpzW-cAGN9P7nQGOFV~1jTxCoBdBn~8ggmLEnFsMZDUBn-HmoZAQq77OpoG3M66flWylM2eJRu3KNtzyerv5unhqb7e87ik5gmFzQyIlElSz1TE-qD8EEvNlKCAZy-Rv0p~LY60BBSh3caPXUOhBf0eWGMuq9t7WoB4qplGAX8VvNVHfEyZ71cBX~fnpu6P-54bGBRC7gIv5SAtLO758xO89aG1JFNufDelbDHwOsIX8SKoxMs4CjY9oQq2dvWomc8R-~hxoo80BzQ1ADgx2Ck-tk3nqz7Cm4~kctlL9q8YoyaWxYCfCo8tlkyciGFkr7OzI515~XcWXsXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086166&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alternative splicing of <em>cox1</em>I5β (A) and <em>cob</em>I1α (B). Arrows mark exons–introns boundaries. Consensus of alternative splice site as described in reference<a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and hypothetical alternative splice site in <em>cob</em>I1α are underlined.</p></div></div><div content-id="dsx026-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f4.png?Expires=2147483647&Signature=vmm1PFj78hkQKb~6P5sPHYwN0CqGSx36PRjVflGfugxNo3pH08OKMP4~kHoHTW5dtq2BEWZNDm9qRt485u2CfbztIlaUp7m8zSKZepUM0tG9DpmxAGesE5EcVh2UEVmEHwR~N1oILlc3YXYM2~6E0wq8DqHvS76fRclXF6y5KcAx7TZn4akoemGYEnwr0j14N0FsHLT3OF7jrnd7OB8XPWpyYpe7pCkezTLyJzOhX632FOtWoXTumf5k0g8j3qnshtEMNgcAibSTFaCM50DnEqgrhRs5ftYo2~escBOa0RIA6U3CdF7C~8goOwG3C-5X1HBuhn5nfpWmZw986n-TdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alternative splicing of cox1I5β (A) and cobI1α (B). Arrows mark exons–introns boundaries. Consensus of alternative splice site as described in reference57 and hypothetical alternative splice site in cobI1α are underlined." path-from-xml="dsx026f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086166/dsx026f4.tif" path-from-xml="dsx026f4.tif" target="_blank">View large</a><a section="104086166" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f4.png?Expires=2147483647&Signature=hOE-kNQBpzW-cAGN9P7nQGOFV~1jTxCoBdBn~8ggmLEnFsMZDUBn-HmoZAQq77OpoG3M66flWylM2eJRu3KNtzyerv5unhqb7e87ik5gmFzQyIlElSz1TE-qD8EEvNlKCAZy-Rv0p~LY60BBSh3caPXUOhBf0eWGMuq9t7WoB4qplGAX8VvNVHfEyZ71cBX~fnpu6P-54bGBRC7gIv5SAtLO758xO89aG1JFNufDelbDHwOsIX8SKoxMs4CjY9oQq2dvWomc8R-~hxoo80BzQ1ADgx2Ck-tk3nqz7Cm4~kctlL9q8YoyaWxYCfCo8tlkyciGFkr7OzI515~XcWXsXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086166&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alternative splicing of <em>cox1</em>I5β (A) and <em>cob</em>I1α (B). Arrows mark exons–introns boundaries. Consensus of alternative splice site as described in reference<a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and hypothetical alternative splice site in <em>cob</em>I1α are underlined.</p></div></div><p>Perhaps, a similar mechanism can be used in <em>cob</em>I1α. We found a hypothetical alternative splice site sharing sequence similarity AAATATTAG(G/A)T(A/T)AATTTA allowing the translation of an intron reading frame with an upstream exon (<a reveal-id="dsx026-F4" data-open="dsx026-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). In <em>S. cariocanus</em>, <em>S. pastorianus</em>, <em>S. mikatae</em> and <em>S. eubayanus</em> it contains a 169–297 AA-long reading frame with one GIY-YIG endonuclease maturase motif.<a reveal-id="dsx026-B58" data-open="dsx026-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a></p><p>Introns, especially those containing HEs, represent a class of mobile elements spread horizontally into an intron-less allele by ‘homing’. They can be found in different species in the same position of the same gene in a chain-involved cyclical gain and loss.<a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx026-B59" data-open="dsx026-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a><sup>,</sup><a reveal-id="dsx026-B60" data-open="dsx026-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> Consequently, they are randomly distributed among the different species.<a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> However, there are a few remarkable exceptions. Intron <em>cox1</em>I3β has not been found in <em>S. cerevisiae</em> although more than a hundred strains have been sequenced. Besides both <em>S. arboricolus</em> strains, <em>cox1</em>I3β is present in all other <em>Saccharomyces</em> species.<a reveal-id="dsx026-B19 dsx026-B20 dsx026-B21" data-open="dsx026-B19 dsx026-B20 dsx026-B21" class="link link-ref link-reveal xref-bibr"><sup>19–21</sup></a> Other evidence of uneven <em>cox1</em>I3β distribution comes from <em>S. cerevisiae</em> xenomitochondrial cybrids containing <em>S. paradoxus</em> mtDNA. They exhibit a slower growth rate on a non-fermentable carbon source, decreased respiration capacity and reduced cytochrome <em>aa3</em> content associated with the inefficient splicing of <em>cox1</em>I3β.<a reveal-id="dsx026-B22" data-open="dsx026-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Apparently, the occurrence of <em>cox1</em>I3β is related to Dobzhansky–Muller nuclear incompatibilities in nucleo-mitochondrial communication that may result in the divergence of yeast species.<a reveal-id="dsx026-B22" data-open="dsx026-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p><p>Very rare is the orphan <em>cox1</em>I4γ intron that occurs only in <em>S. bayanus</em> CBS 380 and <em>S. uvarum</em> CBS 395. We were unable to find homologs in spite of extensive search in the literature and databases.<a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Also, <em>cox1</em>I4α, well known as coding for HE I-<em>Scel</em>l with a LAGLIDADG motif,<a reveal-id="dsx026-B61" data-open="dsx026-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> was not identified in any other <em>Saccharomyces</em> but <em>S. cerevisiae</em>. Unlike <em>cox1</em>I4γ it can be found in less related species from the <em>Saccharomyces/Kluyveromyces</em> complex <em>Naumovia castellii</em><a reveal-id="dsx026-B62" data-open="dsx026-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> and <em>Kluyveromyces marxianus</em> species.<a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx026-B63" data-open="dsx026-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a></p><p>Intron homing is catalyzed by HEs<a reveal-id="dsx026-B64" data-open="dsx026-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsx026-B65" data-open="dsx026-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> that have different evolutionary history and can exist alone.<a reveal-id="dsx026-B66" data-open="dsx026-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a><sup>,</sup><a reveal-id="dsx026-B67" data-open="dsx026-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Such free-standing ‘endonucleases’, named ORFs,<a reveal-id="dsx026-B29" data-open="dsx026-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> have already been described in <em>S. cerevisiae</em> and <em>S. pastorianus</em> mtDNA, but their occurrence is strain dependent.<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx026-B29" data-open="dsx026-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><em>ORF1</em> orthologs are most frequent in different <em>Saccharomyces</em> species (<a reveal-id="dsx026-F3" data-open="dsx026-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). They code for one or two LAGLIDADG motifs and a DNA-binding helix-turn-helix domain.<a reveal-id="dsx026-B68" data-open="dsx026-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Because of the number of stop codons or a GC cluster insertion <em>ORF2</em> is present mostly in truncated form.<a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx026-B21" data-open="dsx026-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> A reading frame coding for one LAGLIDADG motif was found only in <em>S. cariocanus</em>. We were unable to detect <em>ORF3</em> in any ‘non-cerevisiae’ <em>Saccharomyces</em>. Apparently this dubious ORF does not encode a functional protein. Full size <em>ORF4</em> can be found in <em>S. cerevisiae</em> and <em>S. arboricolus</em>. Among all previously identified small ORFs, regions with significant homology to <em>ORF5</em><a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> were found only in <em>S. paradoxus</em><a reveal-id="dsx026-B11" data-open="dsx026-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and <em>S. cariocanus</em>. Therefore, <em>ORF5</em> is considered as a probable genomic artifact,<a reveal-id="dsx026-B69" data-open="dsx026-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> although very small proteins coded by mtDNA are expected, since at least two small proteins were reported to be coded by <em>rns</em> in mammalian mitochondria.<a reveal-id="dsx026-B70" data-open="dsx026-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a><sup>,</sup><a reveal-id="dsx026-B71" data-open="dsx026-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a></p>                            <h3 scrollto-destination=104086171 id="104086171" class="section-title" >3.3. Intergenic DNA, GC clusters and origins of replication</h3>
<p>The putative origins of replication in <em>S. cerevisiae</em> and <em>S. paradoxus</em> are ∼270 bp in length and contain GC blocks separated by AT-rich stretches.<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx026-B72" data-open="dsx026-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> According to novel bioinformatics analysis (strain S288c) it is a five-part consensus sequence [<strong>GGGGGAGGGGGTGGGTGAT ∼</strong>200 <strong>A/T-</strong>rich <strong>GGGTCC</strong>C 29 <strong>A/T-</strong>rich <strong>GGGACC</strong>] downstream of the 20-base promoter [DDWDWTAW<strong><span class="underline">AAGT↓A</span></strong>RTADDDD].<a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a><sup>,</sup><em>Saccharomyces cerevisiae</em> mtDNA contains seven or eight <em>ori</em> elements, depending on the strain, but only <em>ori</em> 2, 3 and 5 are active. They contain a transcription initiation site adjacent to the five-part consensus sequence providing a contiguous transcript of &gt;350 nucleotides. The other <em>ori</em> sites are disabled by the insertion of a G-class GC cluster CCCGGTTTCTTACGAAACCGGGACCTCGGAGA(C/A)GT into the promoter site.<a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a><sup>,</sup><a reveal-id="dsx026-B73" data-open="dsx026-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> The <em>ori</em> elements are present in all <em>Saccharomyces</em> species. Their number decreases with the phylogenetic distance from <em>S. cerevisiae</em> as the <em>S. paradoxus</em> species possesses eight, but species from the lager beer species/<em>S. uvarum</em> clade only four (<a reveal-id="dsx026-T3" data-open="dsx026-T3" class="link link-reveal link-table xref-fig">Table 3</a>). All of them can be potentially active as they are adjacent to the hypothetical promoter sequence DDWDWTAW<strong><span class="underline">AAGT↓A</span></strong>RTADDDD<a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s4_dsx026.doc?Expires=2147483647&amp;Signature=rbwTZpfLkW2CL4OMw5zLVsSd9xNAHVsKG97eJfkOyG8gQAIp3jHp-zLajSwyMn7GcUleX5t7mcVeJA7C9Wobila7rNkv8J3xp--8Q~6y~7Kd0MyYKO-MNilvJfpIHbLbJhce8784JuWp0WQIhq21eHoAk4BBV--ncDxCt5-b7RgaV8GtNWzgT4~Ol6TX6PTWZJEGSZkDy0oF6xMi5YXp4Vudg4DGDS15XCRjU5yIBnIw~1Gn2K-cuxfY17l68m1k1PIihDNuDDa3Lqp9ivuYheGL8JN-HCJF2Yt0-pKylP3fseNlBp5QFIgbFnkrBt0-xPh82dKrE2MvvN6QJ5uBbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S4A). <div content-id="dsx026-T3" class="table-modal"><div class="table-wrap"><div id="dsx026-T3" data-id="dsx026-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Replication origins and GC clusters</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"><em>Saccharomyces</em> strain </th><th rowspan="2"><em>ori</em> </th><th colspan="12">Number of GC clusters<hr /> </th></tr><tr><th>G </th><th>V </th><th>M1 </th><th>M1′ </th><th>M2 </th><th>M2′ </th><th>M2″ </th><th>M3 </th><th>M4 </th><th>U </th><th>SpS* </th><th>Classified/total </th></tr></thead><tbody><tr><td><strong><em>S. cerevisiae</em></strong> </td><td rowspan="2">7/2<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="2">5 </td><td rowspan="2">4 </td><td rowspan="2">48 </td><td rowspan="2">7 </td><td rowspan="2">46 </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">11 </td><td rowspan="2">4 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">127/177 </td></tr><tr><td>NRRL Y-12632<sup>T</sup> </td></tr><tr><td><strong><em>S. paradoxus</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">20 </td><td rowspan="2">1 </td><td rowspan="2">12 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">30 </td><td rowspan="2">– </td><td rowspan="2">46/76 </td></tr><tr><td>CBS 2908 </td></tr><tr><td><strong><em>S. paradoxus</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">9 </td><td rowspan="2">19 </td><td rowspan="2">1 </td><td rowspan="2">12 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">32 </td><td rowspan="2">– </td><td rowspan="2">45/77 </td></tr><tr><td>CBS 7400 </td></tr><tr><td><strong><em>S. cariocanus</em></strong> </td><td rowspan="2">7/7 </td><td rowspan="2">– </td><td rowspan="2">37 </td><td rowspan="2">10 </td><td rowspan="2">2 </td><td rowspan="2">23 </td><td rowspan="2">5 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">72 </td><td rowspan="2">– </td><td rowspan="2">77/149 </td></tr><tr><td>CBS 7994<sup>T</sup> </td></tr><tr><td><strong><em>S. mikatae</em></strong> </td><td rowspan="2">7/7 </td><td rowspan="2">– </td><td rowspan="2">13 </td><td rowspan="2">32 </td><td rowspan="2">7 </td><td rowspan="2">30 </td><td rowspan="2">1 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">10 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">90/143 </td></tr><tr><td>CBS 8839<sup>T</sup> </td></tr><tr><td><strong><em>S. kudriavzevii</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">26 </td><td rowspan="2">9 </td><td rowspan="2">37 </td><td rowspan="2">5 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">51 </td><td rowspan="2">– </td><td rowspan="2">85/136 </td></tr><tr><td>CBS 8840<sup>T</sup> </td></tr><tr><td><strong><em>S. arboricolus</em></strong> </td><td rowspan="2">6/6 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">9 </td><td rowspan="2">6 </td><td rowspan="2">5 </td><td rowspan="2">2 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">76 </td><td rowspan="2">7 </td><td rowspan="2">36/112 </td></tr><tr><td>NRRL Y-63701 </td></tr><tr><td><strong><em>S. arboricolus</em></strong> </td><td rowspan="2">6/6 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">7 </td><td rowspan="2">2 </td><td rowspan="2">9 </td><td rowspan="2">5 </td><td rowspan="2">6 </td><td rowspan="2">– </td><td rowspan="2">2 </td><td rowspan="2">63 </td><td rowspan="2">6 </td><td rowspan="2">38/105 </td></tr><tr><td>CBS 10644<sup>T</sup> </td></tr><tr><td><strong><em>S. pastorianus</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">3 </td><td rowspan="2">13 </td><td rowspan="2">– </td><td rowspan="2">44 </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">3 </td><td rowspan="2">1 </td><td rowspan="2">101 </td><td rowspan="2">6 </td><td rowspan="2">82/183 </td></tr><tr><td>NRRL Y-27171<sup>T</sup> </td></tr><tr><td><strong><em>S. bayanus</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">13 </td><td rowspan="2">16 </td><td rowspan="2">– </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">38 </td><td rowspan="2">7 </td><td rowspan="2">57/95 </td></tr><tr><td>CBS 380<sup>T</sup> </td></tr><tr><td><strong><em>S. uvarum</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">13 </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">38 </td><td rowspan="2">7 </td><td rowspan="2">58/96 </td></tr><tr><td>CBS 395<sup>T</sup> </td></tr><tr><td><em>S. cerevisiae</em> </td><td rowspan="2">8/3 </td><td rowspan="2">5 </td><td rowspan="2">4 </td><td rowspan="2">55 </td><td rowspan="2">7 </td><td rowspan="2">37 </td><td rowspan="2">9 </td><td rowspan="2">13 </td><td rowspan="2">3 </td><td rowspan="2">9 </td><td rowspan="2">48 </td><td rowspan="2">– </td><td rowspan="2">127/177 </td></tr><tr><td>S288c </td></tr><tr><td><em>S. eubayanus</em> </td><td rowspan="2">4/4 </td><td rowspan="2">1 </td><td rowspan="2">– </td><td rowspan="2">26 </td><td rowspan="2">– </td><td rowspan="2">33 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">56 </td><td rowspan="2">5 </td><td rowspan="2">64/120 </td></tr><tr><td>CBS 12357<sup><em>T</em></sup> </td></tr><tr><td><em>S. pastorianus</em> </td><td rowspan="2">4/4 </td><td rowspan="2">6 </td><td rowspan="2">3 </td><td rowspan="2">13 </td><td rowspan="2">1 </td><td rowspan="2">50 </td><td rowspan="2">3 </td><td rowspan="2">10 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">85 </td><td rowspan="2">9 </td><td rowspan="2">86/171 </td></tr><tr><td>WS 34/70 </td></tr><tr><td><em>S. paradoxus</em> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">7 </td><td rowspan="2">14 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">32/82 </td></tr><tr><td>CBS 432<sup>T</sup> </td></tr></tbody></table></div><div content-id="dsx026-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"><em>Saccharomyces</em> strain </th><th rowspan="2"><em>ori</em> </th><th colspan="12">Number of GC clusters<hr /> </th></tr><tr><th>G </th><th>V </th><th>M1 </th><th>M1′ </th><th>M2 </th><th>M2′ </th><th>M2″ </th><th>M3 </th><th>M4 </th><th>U </th><th>SpS* </th><th>Classified/total </th></tr></thead><tbody><tr><td><strong><em>S. cerevisiae</em></strong> </td><td rowspan="2">7/2<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="2">5 </td><td rowspan="2">4 </td><td rowspan="2">48 </td><td rowspan="2">7 </td><td rowspan="2">46 </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">11 </td><td rowspan="2">4 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">127/177 </td></tr><tr><td>NRRL Y-12632<sup>T</sup> </td></tr><tr><td><strong><em>S. paradoxus</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">20 </td><td rowspan="2">1 </td><td rowspan="2">12 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">30 </td><td rowspan="2">– </td><td rowspan="2">46/76 </td></tr><tr><td>CBS 2908 </td></tr><tr><td><strong><em>S. paradoxus</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">9 </td><td rowspan="2">19 </td><td rowspan="2">1 </td><td rowspan="2">12 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">32 </td><td rowspan="2">– </td><td rowspan="2">45/77 </td></tr><tr><td>CBS 7400 </td></tr><tr><td><strong><em>S. cariocanus</em></strong> </td><td rowspan="2">7/7 </td><td rowspan="2">– </td><td rowspan="2">37 </td><td rowspan="2">10 </td><td rowspan="2">2 </td><td rowspan="2">23 </td><td rowspan="2">5 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">72 </td><td rowspan="2">– </td><td rowspan="2">77/149 </td></tr><tr><td>CBS 7994<sup>T</sup> </td></tr><tr><td><strong><em>S. mikatae</em></strong> </td><td rowspan="2">7/7 </td><td rowspan="2">– </td><td rowspan="2">13 </td><td rowspan="2">32 </td><td rowspan="2">7 </td><td rowspan="2">30 </td><td rowspan="2">1 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">10 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">90/143 </td></tr><tr><td>CBS 8839<sup>T</sup> </td></tr><tr><td><strong><em>S. kudriavzevii</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">26 </td><td rowspan="2">9 </td><td rowspan="2">37 </td><td rowspan="2">5 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">51 </td><td rowspan="2">– </td><td rowspan="2">85/136 </td></tr><tr><td>CBS 8840<sup>T</sup> </td></tr><tr><td><strong><em>S. arboricolus</em></strong> </td><td rowspan="2">6/6 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">9 </td><td rowspan="2">6 </td><td rowspan="2">5 </td><td rowspan="2">2 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">76 </td><td rowspan="2">7 </td><td rowspan="2">36/112 </td></tr><tr><td>NRRL Y-63701 </td></tr><tr><td><strong><em>S. arboricolus</em></strong> </td><td rowspan="2">6/6 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">7 </td><td rowspan="2">2 </td><td rowspan="2">9 </td><td rowspan="2">5 </td><td rowspan="2">6 </td><td rowspan="2">– </td><td rowspan="2">2 </td><td rowspan="2">63 </td><td rowspan="2">6 </td><td rowspan="2">38/105 </td></tr><tr><td>CBS 10644<sup>T</sup> </td></tr><tr><td><strong><em>S. pastorianus</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">3 </td><td rowspan="2">13 </td><td rowspan="2">– </td><td rowspan="2">44 </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">3 </td><td rowspan="2">1 </td><td rowspan="2">101 </td><td rowspan="2">6 </td><td rowspan="2">82/183 </td></tr><tr><td>NRRL Y-27171<sup>T</sup> </td></tr><tr><td><strong><em>S. bayanus</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">13 </td><td rowspan="2">16 </td><td rowspan="2">– </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">38 </td><td rowspan="2">7 </td><td rowspan="2">57/95 </td></tr><tr><td>CBS 380<sup>T</sup> </td></tr><tr><td><strong><em>S. uvarum</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">13 </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">38 </td><td rowspan="2">7 </td><td rowspan="2">58/96 </td></tr><tr><td>CBS 395<sup>T</sup> </td></tr><tr><td><em>S. cerevisiae</em> </td><td rowspan="2">8/3 </td><td rowspan="2">5 </td><td rowspan="2">4 </td><td rowspan="2">55 </td><td rowspan="2">7 </td><td rowspan="2">37 </td><td rowspan="2">9 </td><td rowspan="2">13 </td><td rowspan="2">3 </td><td rowspan="2">9 </td><td rowspan="2">48 </td><td rowspan="2">– </td><td rowspan="2">127/177 </td></tr><tr><td>S288c </td></tr><tr><td><em>S. eubayanus</em> </td><td rowspan="2">4/4 </td><td rowspan="2">1 </td><td rowspan="2">– </td><td rowspan="2">26 </td><td rowspan="2">– </td><td rowspan="2">33 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">56 </td><td rowspan="2">5 </td><td rowspan="2">64/120 </td></tr><tr><td>CBS 12357<sup><em>T</em></sup> </td></tr><tr><td><em>S. pastorianus</em> </td><td rowspan="2">4/4 </td><td rowspan="2">6 </td><td rowspan="2">3 </td><td rowspan="2">13 </td><td rowspan="2">1 </td><td rowspan="2">50 </td><td rowspan="2">3 </td><td rowspan="2">10 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">85 </td><td rowspan="2">9 </td><td rowspan="2">86/171 </td></tr><tr><td>WS 34/70 </td></tr><tr><td><em>S. paradoxus</em> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">7 </td><td rowspan="2">14 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">32/82 </td></tr><tr><td>CBS 432<sup>T</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p><sup>a</sup><em>ori2</em> considered as active is absent; G, V, M1, M1′, M2, M2′, M2″, M3 and M4 are GC cluster classes identified according to consensus described in detail in reference<a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> with various degrees of degeneration (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s3_dsx026.doc?Expires=2147483647&amp;Signature=TFW~qL1Ee~QMx71bAy6HPjAH51PXFbVQ3W8ZdK8BcWqmkg-AD5o7NZZVvXoM9TE7GopgONuZ2wow9~KnThfmItmx6DxUlJI3OSKAAd65GJfn2My0I182T4E-FfVeanwbM1hV0k7ewhIqe6bdRYzgP151G6E1Whfw5irck3d-kCn6DduHhCMAcJIBauAgTXXGm5icSZya9nvHF2S03vMgQEw3RDamNsgEIxHXplFaBwkBCBSxY6TDp3y7OXjUUlRLYRf5V~JyTJtZ5WyDFDcI3jKBCbaBmF3KphJ~MxlsGp-uVFsHbMmIoEcqrVSNr-oP6vxPfsvirNkugtzx0x0LrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S3C); U—unclassified GC clusters defined as longer than ≥ 20 nt with GC content ≥ 40%; SpS—<strong>sp</strong>ecies-<strong>s</strong>pecific GC clusters found in the <em>S. bayanus</em>, <em>S. uvarum</em>, <em>S. pastorianus</em> clade consensus sequence TCGTNWCGYACCGTCCAATWGGACGGTACG and in <em>S. arboricolus</em> GGGGTCCC N<sub>(28–30)</sub> GGGGTCCC. Strains assembled in this work are marked in bold.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx026-T3" data-id="dsx026-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Replication origins and GC clusters</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"><em>Saccharomyces</em> strain </th><th rowspan="2"><em>ori</em> </th><th colspan="12">Number of GC clusters<hr /> </th></tr><tr><th>G </th><th>V </th><th>M1 </th><th>M1′ </th><th>M2 </th><th>M2′ </th><th>M2″ </th><th>M3 </th><th>M4 </th><th>U </th><th>SpS* </th><th>Classified/total </th></tr></thead><tbody><tr><td><strong><em>S. cerevisiae</em></strong> </td><td rowspan="2">7/2<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="2">5 </td><td rowspan="2">4 </td><td rowspan="2">48 </td><td rowspan="2">7 </td><td rowspan="2">46 </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">11 </td><td rowspan="2">4 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">127/177 </td></tr><tr><td>NRRL Y-12632<sup>T</sup> </td></tr><tr><td><strong><em>S. paradoxus</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">20 </td><td rowspan="2">1 </td><td rowspan="2">12 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">30 </td><td rowspan="2">– </td><td rowspan="2">46/76 </td></tr><tr><td>CBS 2908 </td></tr><tr><td><strong><em>S. paradoxus</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">9 </td><td rowspan="2">19 </td><td rowspan="2">1 </td><td rowspan="2">12 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">32 </td><td rowspan="2">– </td><td rowspan="2">45/77 </td></tr><tr><td>CBS 7400 </td></tr><tr><td><strong><em>S. cariocanus</em></strong> </td><td rowspan="2">7/7 </td><td rowspan="2">– </td><td rowspan="2">37 </td><td rowspan="2">10 </td><td rowspan="2">2 </td><td rowspan="2">23 </td><td rowspan="2">5 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">72 </td><td rowspan="2">– </td><td rowspan="2">77/149 </td></tr><tr><td>CBS 7994<sup>T</sup> </td></tr><tr><td><strong><em>S. mikatae</em></strong> </td><td rowspan="2">7/7 </td><td rowspan="2">– </td><td rowspan="2">13 </td><td rowspan="2">32 </td><td rowspan="2">7 </td><td rowspan="2">30 </td><td rowspan="2">1 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">10 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">90/143 </td></tr><tr><td>CBS 8839<sup>T</sup> </td></tr><tr><td><strong><em>S. kudriavzevii</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">26 </td><td rowspan="2">9 </td><td rowspan="2">37 </td><td rowspan="2">5 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">51 </td><td rowspan="2">– </td><td rowspan="2">85/136 </td></tr><tr><td>CBS 8840<sup>T</sup> </td></tr><tr><td><strong><em>S. arboricolus</em></strong> </td><td rowspan="2">6/6 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">9 </td><td rowspan="2">6 </td><td rowspan="2">5 </td><td rowspan="2">2 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">76 </td><td rowspan="2">7 </td><td rowspan="2">36/112 </td></tr><tr><td>NRRL Y-63701 </td></tr><tr><td><strong><em>S. arboricolus</em></strong> </td><td rowspan="2">6/6 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">7 </td><td rowspan="2">2 </td><td rowspan="2">9 </td><td rowspan="2">5 </td><td rowspan="2">6 </td><td rowspan="2">– </td><td rowspan="2">2 </td><td rowspan="2">63 </td><td rowspan="2">6 </td><td rowspan="2">38/105 </td></tr><tr><td>CBS 10644<sup>T</sup> </td></tr><tr><td><strong><em>S. pastorianus</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">3 </td><td rowspan="2">13 </td><td rowspan="2">– </td><td rowspan="2">44 </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">3 </td><td rowspan="2">1 </td><td rowspan="2">101 </td><td rowspan="2">6 </td><td rowspan="2">82/183 </td></tr><tr><td>NRRL Y-27171<sup>T</sup> </td></tr><tr><td><strong><em>S. bayanus</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">13 </td><td rowspan="2">16 </td><td rowspan="2">– </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">38 </td><td rowspan="2">7 </td><td rowspan="2">57/95 </td></tr><tr><td>CBS 380<sup>T</sup> </td></tr><tr><td><strong><em>S. uvarum</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">13 </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">38 </td><td rowspan="2">7 </td><td rowspan="2">58/96 </td></tr><tr><td>CBS 395<sup>T</sup> </td></tr><tr><td><em>S. cerevisiae</em> </td><td rowspan="2">8/3 </td><td rowspan="2">5 </td><td rowspan="2">4 </td><td rowspan="2">55 </td><td rowspan="2">7 </td><td rowspan="2">37 </td><td rowspan="2">9 </td><td rowspan="2">13 </td><td rowspan="2">3 </td><td rowspan="2">9 </td><td rowspan="2">48 </td><td rowspan="2">– </td><td rowspan="2">127/177 </td></tr><tr><td>S288c </td></tr><tr><td><em>S. eubayanus</em> </td><td rowspan="2">4/4 </td><td rowspan="2">1 </td><td rowspan="2">– </td><td rowspan="2">26 </td><td rowspan="2">– </td><td rowspan="2">33 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">56 </td><td rowspan="2">5 </td><td rowspan="2">64/120 </td></tr><tr><td>CBS 12357<sup><em>T</em></sup> </td></tr><tr><td><em>S. pastorianus</em> </td><td rowspan="2">4/4 </td><td rowspan="2">6 </td><td rowspan="2">3 </td><td rowspan="2">13 </td><td rowspan="2">1 </td><td rowspan="2">50 </td><td rowspan="2">3 </td><td rowspan="2">10 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">85 </td><td rowspan="2">9 </td><td rowspan="2">86/171 </td></tr><tr><td>WS 34/70 </td></tr><tr><td><em>S. paradoxus</em> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">7 </td><td rowspan="2">14 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">32/82 </td></tr><tr><td>CBS 432<sup>T</sup> </td></tr></tbody></table></div><div content-id="dsx026-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"><em>Saccharomyces</em> strain </th><th rowspan="2"><em>ori</em> </th><th colspan="12">Number of GC clusters<hr /> </th></tr><tr><th>G </th><th>V </th><th>M1 </th><th>M1′ </th><th>M2 </th><th>M2′ </th><th>M2″ </th><th>M3 </th><th>M4 </th><th>U </th><th>SpS* </th><th>Classified/total </th></tr></thead><tbody><tr><td><strong><em>S. cerevisiae</em></strong> </td><td rowspan="2">7/2<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="2">5 </td><td rowspan="2">4 </td><td rowspan="2">48 </td><td rowspan="2">7 </td><td rowspan="2">46 </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">11 </td><td rowspan="2">4 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">127/177 </td></tr><tr><td>NRRL Y-12632<sup>T</sup> </td></tr><tr><td><strong><em>S. paradoxus</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">20 </td><td rowspan="2">1 </td><td rowspan="2">12 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">30 </td><td rowspan="2">– </td><td rowspan="2">46/76 </td></tr><tr><td>CBS 2908 </td></tr><tr><td><strong><em>S. paradoxus</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">9 </td><td rowspan="2">19 </td><td rowspan="2">1 </td><td rowspan="2">12 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">32 </td><td rowspan="2">– </td><td rowspan="2">45/77 </td></tr><tr><td>CBS 7400 </td></tr><tr><td><strong><em>S. cariocanus</em></strong> </td><td rowspan="2">7/7 </td><td rowspan="2">– </td><td rowspan="2">37 </td><td rowspan="2">10 </td><td rowspan="2">2 </td><td rowspan="2">23 </td><td rowspan="2">5 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">72 </td><td rowspan="2">– </td><td rowspan="2">77/149 </td></tr><tr><td>CBS 7994<sup>T</sup> </td></tr><tr><td><strong><em>S. mikatae</em></strong> </td><td rowspan="2">7/7 </td><td rowspan="2">– </td><td rowspan="2">13 </td><td rowspan="2">32 </td><td rowspan="2">7 </td><td rowspan="2">30 </td><td rowspan="2">1 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">10 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">90/143 </td></tr><tr><td>CBS 8839<sup>T</sup> </td></tr><tr><td><strong><em>S. kudriavzevii</em></strong> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">26 </td><td rowspan="2">9 </td><td rowspan="2">37 </td><td rowspan="2">5 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">51 </td><td rowspan="2">– </td><td rowspan="2">85/136 </td></tr><tr><td>CBS 8840<sup>T</sup> </td></tr><tr><td><strong><em>S. arboricolus</em></strong> </td><td rowspan="2">6/6 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">9 </td><td rowspan="2">6 </td><td rowspan="2">5 </td><td rowspan="2">2 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">76 </td><td rowspan="2">7 </td><td rowspan="2">36/112 </td></tr><tr><td>NRRL Y-63701 </td></tr><tr><td><strong><em>S. arboricolus</em></strong> </td><td rowspan="2">6/6 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">7 </td><td rowspan="2">2 </td><td rowspan="2">9 </td><td rowspan="2">5 </td><td rowspan="2">6 </td><td rowspan="2">– </td><td rowspan="2">2 </td><td rowspan="2">63 </td><td rowspan="2">6 </td><td rowspan="2">38/105 </td></tr><tr><td>CBS 10644<sup>T</sup> </td></tr><tr><td><strong><em>S. pastorianus</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">3 </td><td rowspan="2">13 </td><td rowspan="2">– </td><td rowspan="2">44 </td><td rowspan="2">1 </td><td rowspan="2">6 </td><td rowspan="2">3 </td><td rowspan="2">1 </td><td rowspan="2">101 </td><td rowspan="2">6 </td><td rowspan="2">82/183 </td></tr><tr><td>NRRL Y-27171<sup>T</sup> </td></tr><tr><td><strong><em>S. bayanus</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">13 </td><td rowspan="2">16 </td><td rowspan="2">– </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">38 </td><td rowspan="2">7 </td><td rowspan="2">57/95 </td></tr><tr><td>CBS 380<sup>T</sup> </td></tr><tr><td><strong><em>S. uvarum</em></strong> </td><td rowspan="2">4/4 </td><td rowspan="2">5 </td><td rowspan="2">13 </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">17 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">38 </td><td rowspan="2">7 </td><td rowspan="2">58/96 </td></tr><tr><td>CBS 395<sup>T</sup> </td></tr><tr><td><em>S. cerevisiae</em> </td><td rowspan="2">8/3 </td><td rowspan="2">5 </td><td rowspan="2">4 </td><td rowspan="2">55 </td><td rowspan="2">7 </td><td rowspan="2">37 </td><td rowspan="2">9 </td><td rowspan="2">13 </td><td rowspan="2">3 </td><td rowspan="2">9 </td><td rowspan="2">48 </td><td rowspan="2">– </td><td rowspan="2">127/177 </td></tr><tr><td>S288c </td></tr><tr><td><em>S. eubayanus</em> </td><td rowspan="2">4/4 </td><td rowspan="2">1 </td><td rowspan="2">– </td><td rowspan="2">26 </td><td rowspan="2">– </td><td rowspan="2">33 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">56 </td><td rowspan="2">5 </td><td rowspan="2">64/120 </td></tr><tr><td>CBS 12357<sup><em>T</em></sup> </td></tr><tr><td><em>S. pastorianus</em> </td><td rowspan="2">4/4 </td><td rowspan="2">6 </td><td rowspan="2">3 </td><td rowspan="2">13 </td><td rowspan="2">1 </td><td rowspan="2">50 </td><td rowspan="2">3 </td><td rowspan="2">10 </td><td rowspan="2">– </td><td rowspan="2">– </td><td rowspan="2">85 </td><td rowspan="2">9 </td><td rowspan="2">86/171 </td></tr><tr><td>WS 34/70 </td></tr><tr><td><em>S. paradoxus</em> </td><td rowspan="2">8/8 </td><td rowspan="2">– </td><td rowspan="2">7 </td><td rowspan="2">14 </td><td rowspan="2">– </td><td rowspan="2">8 </td><td rowspan="2">– </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">1 </td><td rowspan="2">50 </td><td rowspan="2">– </td><td rowspan="2">32/82 </td></tr><tr><td>CBS 432<sup>T</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p><sup>a</sup><em>ori2</em> considered as active is absent; G, V, M1, M1′, M2, M2′, M2″, M3 and M4 are GC cluster classes identified according to consensus described in detail in reference<a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> with various degrees of degeneration (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s3_dsx026.doc?Expires=2147483647&amp;Signature=TFW~qL1Ee~QMx71bAy6HPjAH51PXFbVQ3W8ZdK8BcWqmkg-AD5o7NZZVvXoM9TE7GopgONuZ2wow9~KnThfmItmx6DxUlJI3OSKAAd65GJfn2My0I182T4E-FfVeanwbM1hV0k7ewhIqe6bdRYzgP151G6E1Whfw5irck3d-kCn6DduHhCMAcJIBauAgTXXGm5icSZya9nvHF2S03vMgQEw3RDamNsgEIxHXplFaBwkBCBSxY6TDp3y7OXjUUlRLYRf5V~JyTJtZ5WyDFDcI3jKBCbaBmF3KphJ~MxlsGp-uVFsHbMmIoEcqrVSNr-oP6vxPfsvirNkugtzx0x0LrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S3C); U—unclassified GC clusters defined as longer than ≥ 20 nt with GC content ≥ 40%; SpS—<strong>sp</strong>ecies-<strong>s</strong>pecific GC clusters found in the <em>S. bayanus</em>, <em>S. uvarum</em>, <em>S. pastorianus</em> clade consensus sequence TCGTNWCGYACCGTCCAATWGGACGGTACG and in <em>S. arboricolus</em> GGGGTCCC N<sub>(28–30)</sub> GGGGTCCC. Strains assembled in this work are marked in bold.</p></div></div></div></div></p><p>GC clusters were originally defined as regions 35 bp long on average and 45–62% GC<a reveal-id="dsx026-B74" data-open="dsx026-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> and are in general the major source of noticeable intergenic region polymorphism.<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> According to the characteristic primary structures, they have been assigned to seven families which exhibit varying degrees of homology.<a reveal-id="dsx026-B73" data-open="dsx026-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a><sup>,</sup><a reveal-id="dsx026-B74" data-open="dsx026-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> Recently, the M2″ class consensus was reduced to 14 bases (TCCGGCCGAAGGAG).<a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Therefore, we took into consideration as GC clusters only regions longer than ≥ 20 nt with GC content ≥ 40%. The average number of GC clusters in <em>S. cerevisiae</em> is above 100 and their occurrence decreases in other <em>Saccharomyces</em> species<a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> from 90 to 30 (<a reveal-id="dsx026-T3" data-open="dsx026-T3" class="link link-reveal link-table xref-fig">Table 3</a>). A G-class <em>ori</em>-specific GC cluster has been found in any species from the <em>S. bayanus</em>, <em>S. uvarum</em>, <em>S. pastorianus</em> clade, but remote from <em>ori</em>. The most frequent are M1 and M2 classes, although members of other classes can be found. In addition, a species-specific GC-rich sequence (GGGGTCCC N<sub>(28–30)</sub> GGGGTCCC) was recognized in both <em>S. arboricolus</em> strains and a GC cluster with the consensus TCGTNWCGYACCGTCCAATWGGACGGTACG in species from the lager beer species/<em>S. uvarum</em> clade.</p>                            <h3 scrollto-destination=104086174 id="104086174" class="section-title" >3.4. Transcription units and adjacent motifs</h3>
<p>The highly-conserved sequence motif WTATAAGTA is known as the yeast mitochondrial transcription initiation site.<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx026-B75 dsx026-B76 dsx026-B77" data-open="dsx026-B75 dsx026-B76 dsx026-B77" class="link link-ref link-reveal xref-bibr"><sup>75–77</sup></a> Recently, transcriptome analysis confirmed experimentally the sequence DDWDWTAW<strong><span class="underline">AAGT↓A</span></strong>RTADDDD as the consensus promoter site in <em>S. cerevisiae</em><a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and 19 potential transcription initiation sites<a reveal-id="dsx026-B6" data-open="dsx026-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> were reduced to 11. We found these consensus promoter sites in all sequenced <em>Saccharomyces</em> at approximately the same position as they were reported for <em>S. cerevisiae</em>. Because of the rearrangement of the <em>trnS1</em>-<em>rps3</em> block in the mtDNA of the <em>S. bayanus</em>, <em>S. uvarum</em>, <em>S. pastorianus</em> clade, an additional transcription site was found upstream of the <em>trnS1</em> gene (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s4_dsx026.doc?Expires=2147483647&Signature=rbwTZpfLkW2CL4OMw5zLVsSd9xNAHVsKG97eJfkOyG8gQAIp3jHp-zLajSwyMn7GcUleX5t7mcVeJA7C9Wobila7rNkv8J3xp--8Q~6y~7Kd0MyYKO-MNilvJfpIHbLbJhce8784JuWp0WQIhq21eHoAk4BBV--ncDxCt5-b7RgaV8GtNWzgT4~Ol6TX6PTWZJEGSZkDy0oF6xMi5YXp4Vudg4DGDS15XCRjU5yIBnIw~1Gn2K-cuxfY17l68m1k1PIihDNuDDa3Lqp9ivuYheGL8JN-HCJF2Yt0-pKylP3fseNlBp5QFIgbFnkrBt0-xPh82dKrE2MvvN6QJ5uBbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S4A). When all hypothetical promoter sites are compared, the <em>S. cerevisiae</em> consensus<a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> is slightly degenerated mainly at the 5′ site to D<span class="underline">NN</span>D<span class="underline">N</span>TAW<strong>AAGT</strong>↓<strong>A</strong>RTADDDD that is more related to the original nonanucleotide WTATAAGTA in spite of the phage T7 origin of yeast mitochondrial RNA polymerase. Apparently, the 23 base-long phage T7 consensus promoter is reduced in <em>Saccharomyces</em> mtDNA.</p><p>The 3′ termini of mitochondrial mRNA cleavage sites in budding yeast are recognized according to the postulated dodecamer motif 5′-AAUAA (U/C) AUUCUU-3′ in the 3′ UTR region.<a reveal-id="dsx026-B77 dsx026-B78 dsx026-B79" data-open="dsx026-B77 dsx026-B78 dsx026-B79" class="link link-ref link-reveal xref-bibr"><sup>77–79</sup></a> A processing site was confirmed by transcriptome analysis where an alternative heptakaidecamer 5′AATAATATTCTTAT↓AGTCCGGCC↓CGCCC with part of the M2 GC cluster is recognized instead.<a reveal-id="dsx026-B57" data-open="dsx026-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Variations to the dodecamer have been found in all protein-coding transcription units at the 3′ UTR region of all <em>Saccharomyces</em> species, in most cases up to 200 nucleotides downstream from the termination codon (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s4_dsx026.doc?Expires=2147483647&Signature=rbwTZpfLkW2CL4OMw5zLVsSd9xNAHVsKG97eJfkOyG8gQAIp3jHp-zLajSwyMn7GcUleX5t7mcVeJA7C9Wobila7rNkv8J3xp--8Q~6y~7Kd0MyYKO-MNilvJfpIHbLbJhce8784JuWp0WQIhq21eHoAk4BBV--ncDxCt5-b7RgaV8GtNWzgT4~Ol6TX6PTWZJEGSZkDy0oF6xMi5YXp4Vudg4DGDS15XCRjU5yIBnIw~1Gn2K-cuxfY17l68m1k1PIihDNuDDa3Lqp9ivuYheGL8JN-HCJF2Yt0-pKylP3fseNlBp5QFIgbFnkrBt0-xPh82dKrE2MvvN6QJ5uBbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S4B). A putative <em>atp8</em> transcription termination site exhibits the most degenerated consensus sequence. Interesting features are the multiple cleavage sites for <em>ORF1</em> present in <em>S. mikatae</em>, <em>S. kudriavzevii</em> and <em>S. arboricolus</em>. An alternative heptakaidecamer transcription processing site was found only in <em>S. cerevisiae</em>.</p>                            <h3 scrollto-destination=104086177 id="104086177" class="section-title" >3.5. Gene order and synteny</h3>
<p>The most profound feature is the alteration in gene order that involves mainly <em>trnfM</em>-<em>rnpB</em>-<em>trnP</em>, <em>rns</em>-<em>trnW</em>, <em>trnE</em>-<em>cob</em> and <em>trnS1</em>-<em>rps3</em> gene clusters. <em>S. cerevisiae</em> NRRL Y-12632 mtDNA with a length of almost 90 kb belongs among the largest genomes and has the same gene order as is known in 105 other strains.<a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx026-B21" data-open="dsx026-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><em>S. paradoxus</em> CBS 2908 and CBS 7400 mtDNA differs in size (66 kb versus 71 kb), associated with the intron occurrence in the <em>cob</em> gene. The gene order is the same as in 14 other strains.<a reveal-id="dsx026-B11" data-open="dsx026-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> They differ from <em>S. cerevisiae</em> by the excision of the <em>trnFT1V</em>-<em>cox3</em>-<em>trnfM</em>-<em>rnpB</em>-<em>trnP</em>-<em>rns</em>-<em>trnW</em> segment that is translocated behind the <em>trnS1</em>-<em>rps3</em> gene cluster where <em>rns</em>-<em>trnW</em> is inverted (<a reveal-id="dsx026-F1" data-open="dsx026-F1" class="link link-reveal link-table xref-fig">Figs 1 and 5</a>). Surprisingly, the 77 kb mtDNA of <em>S. cariocanus</em> is syntenic to <em>S. cerevisiae</em>, although these species can be distinguished according to the <em>cox1</em>I3β intron present in <em>S. cariocanus</em> (<a reveal-id="dsx026-F1" data-open="dsx026-F1" class="link link-reveal link-table xref-fig">Figs 1 and 3</a>). <em>S. mikatae</em> mtDNA is &gt;85 kb long and the gene architecture is altered from <em>S. cerevisiae</em> by the movement and inversion of the <em>trnfM</em>-<em>rnpB</em>-<em>trnP</em> gene cluster behind the <em>rns</em>-<em>trnW</em> locus (<a reveal-id="dsx026-F5" data-open="dsx026-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). <em>S. kudriavzevii</em> mtDNA is ∼80 kb long and the gene order differs from <em>S. cerevisiae</em> by the excision of <em>rns</em>-<em>trnW</em> and its transplacement to the complementary strand at the 3′ end of the <em>trnS1</em>-<em>rps3</em> gene cluster (<a reveal-id="dsx026-F5" data-open="dsx026-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Both <em>S. arboricolus</em> mtDNAs are around 70 kb in size and the genome architecture exhibits very extensive reshuffling in relation to <em>S. cerevisiae</em>. The <em>rns</em>-<em>trnW</em> cluster is excised and transferred at the 3′ end of the <em>atp9</em>-<em>trnS1</em>-<em>rps3</em> gene cluster. The remaining <em>trnfM</em>-<em>rnpB</em>-<em>trnP</em>-<em>cox1</em>-<em>atp8</em>-<em>atp6</em> genome segment is translocated at the end of the <em>cob</em> gene (<a reveal-id="dsx026-F1" data-open="dsx026-F1" class="link link-reveal link-table xref-fig">Figs 1 and 5</a>). <em>S. bayanus</em>, <em>S. eubayanus</em>, <em>S. uvarum</em> and <em>S. pastorianus</em> share the same mtDNA architecture. Besides all alterations known for <em>S. arboricolus</em> the characteristic transcription unit <em>atp9</em>-<em>trnS1</em>-<em>rps3</em> is broken and the <em>trnS1</em>-<em>rps3</em> part is inverted at the same location (<a reveal-id="dsx026-F1" data-open="dsx026-F1" class="link link-reveal link-table xref-fig">Figs 1 and 5</a>). Strains from the lager beer species/<em>S. uvarum</em> clade despite their hybrid nature are classified as separate species, although they exhibit strikingly similar mtDNA sequences. Nearly identical are mtDNAs from <em>S. uvarum</em> CBS 395<sup>T</sup> and <em>S. bayanus</em> CBS 380<sup>T</sup>. </p>                        <a id="104086179" scrollto-destination="104086179"></a>
<div id="dsx026-F5" data-id="dsx026-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f5.png?Expires=2147483647&Signature=vEOOlWCovYHnZ1yuybnVZEnFjeQT6fHDFF4MEWIeY9nCYA-1ONTwYVUZo9Cc0ZsCfOSijKWLWQogvxzx~sRZobA38OMzXjAtU88xb5aX~2nJ68jtX6HZmKOjDCQ7m0y03jOThcQihRXuNgpbdyBGLIzIv~MIP5hyTiOAPVl3XjJpE~onhHQq2YZ1Jl12kU8Ynutw3OpKuh5~pErzrLjTOrPYS1fDUzW4Y8RGHCEyABIAOQ3NObYole4Yi7miZkB0ZeH5ywgoVgvm0ZeOfWKTZyVo2FCH7OawU40k4Ic6mvdPu~ZV~bzTtNTuWCe9xSnHQlz8hKq4vG8J4j0CR5SQOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparison of mitochondrial gene order in Saccharomyces. Individual gene clusters are highlighted by different shades of grey. Pentagon symbols indicate the transcription direction." path-from-xml="dsx026f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086179/dsx026f5.tif" path-from-xml="dsx026f5.tif" target="_blank">View large</a><a section="104086179" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f5.png?Expires=2147483647&Signature=XuY3AP5cFepIRKfQ7ssEAEObIw1yPXqQgaw8-TB~7rVWFe9stgXLA72B2jNrJJCIM0IMGW4XCMoIgBo35PRsJFwJOI0MJpjCU~kOgNJG8qnlGT1XbAZJn-oBJEXBA6vqv-7fhaVq5OuQKEnz4IFxgAbMJh5rTetNTQ3aOjnqgO7kMOfLHixXma9kMfaShIhe-us00E95OyftbcuwLZVJuT8EfnE4pfXEf~28r-TfKTrhzhjbnzgeZii08kkzgOW0rwkMs85wFIpHYB8~PtraMH3QsJ4rE08lCfBNtGoZohgNigJnpcJodPW26Yft2JEAOki5OqFwWIUK1QgDEvC7~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086179&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparison of mitochondrial gene order in <em>Saccharomyces</em>. Individual gene clusters are highlighted by different shades of grey. Pentagon symbols indicate the transcription direction.</p></div></div><div content-id="dsx026-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f5.png?Expires=2147483647&Signature=vEOOlWCovYHnZ1yuybnVZEnFjeQT6fHDFF4MEWIeY9nCYA-1ONTwYVUZo9Cc0ZsCfOSijKWLWQogvxzx~sRZobA38OMzXjAtU88xb5aX~2nJ68jtX6HZmKOjDCQ7m0y03jOThcQihRXuNgpbdyBGLIzIv~MIP5hyTiOAPVl3XjJpE~onhHQq2YZ1Jl12kU8Ynutw3OpKuh5~pErzrLjTOrPYS1fDUzW4Y8RGHCEyABIAOQ3NObYole4Yi7miZkB0ZeH5ywgoVgvm0ZeOfWKTZyVo2FCH7OawU40k4Ic6mvdPu~ZV~bzTtNTuWCe9xSnHQlz8hKq4vG8J4j0CR5SQOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparison of mitochondrial gene order in Saccharomyces. Individual gene clusters are highlighted by different shades of grey. Pentagon symbols indicate the transcription direction." path-from-xml="dsx026f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086179/dsx026f5.tif" path-from-xml="dsx026f5.tif" target="_blank">View large</a><a section="104086179" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f5.png?Expires=2147483647&Signature=XuY3AP5cFepIRKfQ7ssEAEObIw1yPXqQgaw8-TB~7rVWFe9stgXLA72B2jNrJJCIM0IMGW4XCMoIgBo35PRsJFwJOI0MJpjCU~kOgNJG8qnlGT1XbAZJn-oBJEXBA6vqv-7fhaVq5OuQKEnz4IFxgAbMJh5rTetNTQ3aOjnqgO7kMOfLHixXma9kMfaShIhe-us00E95OyftbcuwLZVJuT8EfnE4pfXEf~28r-TfKTrhzhjbnzgeZii08kkzgOW0rwkMs85wFIpHYB8~PtraMH3QsJ4rE08lCfBNtGoZohgNigJnpcJodPW26Yft2JEAOki5OqFwWIUK1QgDEvC7~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086179&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparison of mitochondrial gene order in <em>Saccharomyces</em>. Individual gene clusters are highlighted by different shades of grey. Pentagon symbols indicate the transcription direction.</p></div></div><p>The alteration in gene order within yeast genera is not so frequent and is associated with the genome size. The majority of yeast mtDNA is relatively small (&lt; 40 kbp) with the genes located on the same strand. The size of &lt;15% of mtDNA exceeds 50 kb and the genes are scattered in conserved blocks (e.g. <em>cox1</em>-<em>atp8</em>-<em>atp6</em>, <em>rnl</em>-<em>trn</em>) coded in both strands with altered transcription orientation.<a reveal-id="dsx026-B7" data-open="dsx026-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Obviously, larger genomes like in <em>Saccharomyces</em> are more prone to rearrangements. An extremely high degree of gene conservation and synteny is the characteristic feature among and within the species possessing smaller mtDNA of the <em>Lachancea</em>,<a reveal-id="dsx026-B80" data-open="dsx026-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a><sup>,</sup><em>Torulaspora</em><a reveal-id="dsx026-B81" data-open="dsx026-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> and <em>Yarrowia</em> clades.<a reveal-id="dsx026-B82" data-open="dsx026-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> However, significant mitochondrial genome rearrangement was observed in yeasts with larger mtDNA from the <em>Nakaseomyces</em><a reveal-id="dsx026-B83" data-open="dsx026-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a> and <em>Dekkera</em>/<em>Brettanomyces</em> genera.<a reveal-id="dsx026-B84" data-open="dsx026-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> Apparently, alteration in mtDNA gene order depends most of all on the size of intergenic regions and is obvious if their ratio to genes exceeds 60%.<a reveal-id="dsx026-B7" data-open="dsx026-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> The gene order alteration mechanism was deduced exclusively from the DNA sequence, proposing an intermediate with duplicate segments, associated with conserved gene blocks mimicking transcription units.<a reveal-id="dsx026-B11" data-open="dsx026-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx026-B84" data-open="dsx026-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a><sup>,</sup><a reveal-id="dsx026-B85" data-open="dsx026-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> This is not the case in <em>S. bayanus</em>, <em>S. eubayanus</em>, <em>S. uvarum</em> and <em>S. pastorianus</em> where the <em>atp9</em>-<em>trnS1</em>-<em>rps3</em> transcription unit is broken apart and the transposition of <em>trnS1</em>-<em>rps3</em> to the opposite strand requires formation of a <em>de novo</em> promoter.</p>                            <h3 scrollto-destination=104086181 id="104086181" class="section-title" >3.6. Phylogeny and taxonomy</h3>
<p>As mentioned above, currently <em>S. cerevisiae</em>, <em>S. paradoxus</em>, <em>S. mikatae</em>, <em>S. kudriavzevii</em>, <em>S. arboricolus</em>, <em>S. bayanus</em>, <em>S. uvarum</em> and <em>S. pastorianus</em> are the only species accepted by taxonomists as <em>Saccharomyces</em>.<a reveal-id="dsx026-B1" data-open="dsx026-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> However, whole (nuclear) genome studies demonstrated that <em>S. pastorianus</em> is an alloploid of <em>S. cerevisiae</em> and the newly described species <em>S. eubayanus</em> as well as <em>S. bayanus</em> is a hybrid between <em>S. uvarum</em> and <em>S. eubayanus</em> with a small contribution of <em>S. cerevisiae</em>.<a reveal-id="dsx026-B4" data-open="dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx026-B5" data-open="dsx026-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx026-B13" data-open="dsx026-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx026-B26" data-open="dsx026-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx026-B86" data-open="dsx026-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a><sup>,</sup><em>Saccharomyces cariocanus</em> is reproductively isolated by four chromosomal translocations from <em>S. paradoxus</em> but not by sequence and therefore is considered to be a <em>S. paradoxus</em> subspecies.<a reveal-id="dsx026-B4" data-open="dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx026-B5" data-open="dsx026-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx026-B86" data-open="dsx026-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> All these data came exclusively from nuclear genetic information, although mitochondrial genomes are more susceptible to mutations than their nuclear counterparts.</p><p>The accuracy of taxonomic classification as well as the evolution history can be inferred from phylogenetic trees derived from concatenated DNA or protein sequences of mitochondrial or nuclear origin. However, support for many lineages in present phylogenetic trees is often weak and more robust analyses of relationships will require whole genome comparisons.<a reveal-id="dsx026-B87" data-open="dsx026-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> To shed light on these taxonomy–phylogeny pitfalls we constructed phylogenetic trees from unambiguously aligned portions of the concatenated mtDNA sequences coding for proteins (<a reveal-id="dsx026-F6" data-open="dsx026-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>). Because of limited gene availability, we used 10 nuclear genes as a good compromise, especially if they had been used in population studies<a reveal-id="dsx026-B36 dsx026-B37 dsx026-B38 dsx026-B39" data-open="dsx026-B36 dsx026-B37 dsx026-B38 dsx026-B39" class="link link-ref link-reveal xref-bibr"><sup>36–39</sup></a> (<a reveal-id="dsx026-F6" data-open="dsx026-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>). Branching in both trees does not correlate with basic taxonomic classification and the tree derived from the combined sequences of the D1/D2 LSU RNA gene and ITS used in taxonomy<a reveal-id="dsx026-B1" data-open="dsx026-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> (<a reveal-id="dsx026-F6" data-open="dsx026-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Phylogenetic comparison of mitochondrial and nuclear genes clearly demonstrates that <em>S. cariocanus</em> should be assigned as a separate species. It belongs in the nuclei-derived tree to the <em>S. paradoxus</em> clade; however, in the mitochondrial tree it forms a statistically significant separate branch. Considering its mtDNA gene order, which differs from that of <em>S. paradoxus</em>, and the presence of the <em>cox1</em>I3β intron, <em>S. cariocanus</em> should be designated as a separate species (<a reveal-id="dsx026-F6" data-open="dsx026-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). This conclusion, rather than horizontal transfer, supports the low spore viability obtained from crosses with the <em>S. paradoxus</em> tester as well as the absence of any <em>S. cerevisiae</em> genes in the nuclear genome.<a reveal-id="dsx026-B4" data-open="dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx026-B5" data-open="dsx026-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx026-B86" data-open="dsx026-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> Multilocus phylogenies based on conserved mitochondrial genes (except extremely variable <em>rps3</em>) are largely representative; however, they occasionally do not correlate with trees derived from nuclear genes.<a reveal-id="dsx026-B7" data-open="dsx026-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx026-B84" data-open="dsx026-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> The most plausible explanation for this discrepancy is different selection pressures for mitochondrial and nuclear genes, where complex patterns of hybridization and introgression are involved.<a reveal-id="dsx026-B7" data-open="dsx026-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx026-B84" data-open="dsx026-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> However, the <em>S. cariocanus</em> paradox can be also explained as a partially developing divergence, because the genus <em>Saccharomyces</em> is considered as a continuum of taxa differentiating towards speciation.<a reveal-id="dsx026-B1" data-open="dsx026-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a></p>                        <a id="104086184" scrollto-destination="104086184"></a>
<div id="dsx026-F6" data-id="dsx026-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f6.png?Expires=2147483647&Signature=Y3aXoI9b0mhuIok-fXA1MSh7TbR0sIiIuoGP6wVMpwHh9HHLjXpWgQQdB7Pd1BZ~Hj2sjwN9O3XKTnrX~7xWAPikc~AXKqPF-4TdNGRFqn83YBsBolugQesFNoAbugjWspnvKHnXklpbV8wjH3~uE2L4WphY8AM6mIk-mQubiEzvtRZh2km1D4LTARYR7lwMuMC5VZgjRRDtOFltlD42LaBxKM3YnLv3XkRIjEgKy56BT65eBfIS-rqLeGImZ61qoaobBs5qEIJG5VMhVWI-X02nExly6ArTNSZ7hfleyjKs2PaZYAjsYh3Uv42pjvummg42qH8pDvMsl4DUpQq7BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Saccharomyces phylogeny. Both trees were constructed from unambiguously aligned concatenated DNA sequences using the Maximum likelihood phylogeny PhyML program. (A) mtDNA-derived phylogeny from DNA sequences coding for proteins in the order cox1, atp8, atp6, cob, atp9, cox2, cox3. (B) Nuclear DNA-derived phylogenetic tree from CCA1, CYT1, MLS1, RPS5, LAS1, MET4, NUP116, ZDS2, PDR10 and DSN1 protein-coding genes used in population studies.36–39 The branch length is proportional to the nucleotide differences indicated by the bar. The numbers given at the nodes are the frequencies of a given branch appearing in 1,000 bootstrap replications. All are above 50%, indicating good statistical support. Naumovia castellii NRRL Y-12630 was used as an outgroup." path-from-xml="dsx026f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086184/dsx026f6.tif" path-from-xml="dsx026f6.tif" target="_blank">View large</a><a section="104086184" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f6.png?Expires=2147483647&Signature=pxFBmh6Gcpg-UKMmZimwzp-VwjujShKkWw1Amanvbhw2Ka4G~YDV0q2OszUJLcAE5Bwc5J8j2F0w-FV92Q2h~QvaTLH~lWMBUhIGUx3XXPwrTU4NRxvRLbnOwTekEl4sFWGYwmLdFlVpt8~a26BD5HBf3Lnyu530pGLwjNKkqHRDl0SrjhtJvxEy4H737uOoY3kQ0eSlCaWeDNrBvUPj432C2Ev6nbgn7AddQ1Am~95X-tjaJ68OeEyymx4tz01ur7LVFKrKE52MMkUmnPtx22nA06EYwRvyJ7nEqMluWbdaxxaTOgCVYMX2wy47fuK09PU1rhN6BfKze-JjrJ1Qzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086184&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Saccharomyces</em> phylogeny. Both trees were constructed from unambiguously aligned concatenated DNA sequences using the Maximum likelihood phylogeny PhyML program. (A) mtDNA-derived phylogeny from DNA sequences coding for proteins in the order <em>cox1</em>, <em>atp8</em>, <em>atp6</em>, <em>cob</em>, <em>atp9</em>, <em>cox2</em>, <em>cox3</em>. (B) Nuclear DNA-derived phylogenetic tree from <em>CCA1</em>, <em>CYT1</em>, <em>MLS1</em>, <em>RPS5</em>, <em>LAS1</em>, <em>MET4</em>, <em>NUP116</em>, <em>ZDS2</em>, <em>PDR10</em> and <em>DSN1</em> protein-coding genes used in population studies.<a reveal-id="dsx026-B36 dsx026-B37 dsx026-B38 dsx026-B39" data-open="dsx026-B36 dsx026-B37 dsx026-B38 dsx026-B39" class="link link-ref link-reveal xref-bibr"><sup>36–39</sup></a> The branch length is proportional to the nucleotide differences indicated by the bar. The numbers given at the nodes are the frequencies of a given branch appearing in 1,000 bootstrap replications. All are above 50%, indicating good statistical support. <em>Naumovia castellii</em> NRRL Y-12630 was used as an outgroup.</p></div></div><div content-id="dsx026-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/m_dsx026f6.png?Expires=2147483647&Signature=Y3aXoI9b0mhuIok-fXA1MSh7TbR0sIiIuoGP6wVMpwHh9HHLjXpWgQQdB7Pd1BZ~Hj2sjwN9O3XKTnrX~7xWAPikc~AXKqPF-4TdNGRFqn83YBsBolugQesFNoAbugjWspnvKHnXklpbV8wjH3~uE2L4WphY8AM6mIk-mQubiEzvtRZh2km1D4LTARYR7lwMuMC5VZgjRRDtOFltlD42LaBxKM3YnLv3XkRIjEgKy56BT65eBfIS-rqLeGImZ61qoaobBs5qEIJG5VMhVWI-X02nExly6ArTNSZ7hfleyjKs2PaZYAjsYh3Uv42pjvummg42qH8pDvMsl4DUpQq7BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Saccharomyces phylogeny. Both trees were constructed from unambiguously aligned concatenated DNA sequences using the Maximum likelihood phylogeny PhyML program. (A) mtDNA-derived phylogeny from DNA sequences coding for proteins in the order cox1, atp8, atp6, cob, atp9, cox2, cox3. (B) Nuclear DNA-derived phylogenetic tree from CCA1, CYT1, MLS1, RPS5, LAS1, MET4, NUP116, ZDS2, PDR10 and DSN1 protein-coding genes used in population studies.36–39 The branch length is proportional to the nucleotide differences indicated by the bar. The numbers given at the nodes are the frequencies of a given branch appearing in 1,000 bootstrap replications. All are above 50%, indicating good statistical support. Naumovia castellii NRRL Y-12630 was used as an outgroup." path-from-xml="dsx026f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086184/dsx026f6.tif" path-from-xml="dsx026f6.tif" target="_blank">View large</a><a section="104086184" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/dsx026f6.png?Expires=2147483647&Signature=pxFBmh6Gcpg-UKMmZimwzp-VwjujShKkWw1Amanvbhw2Ka4G~YDV0q2OszUJLcAE5Bwc5J8j2F0w-FV92Q2h~QvaTLH~lWMBUhIGUx3XXPwrTU4NRxvRLbnOwTekEl4sFWGYwmLdFlVpt8~a26BD5HBf3Lnyu530pGLwjNKkqHRDl0SrjhtJvxEy4H737uOoY3kQ0eSlCaWeDNrBvUPj432C2Ev6nbgn7AddQ1Am~95X-tjaJ68OeEyymx4tz01ur7LVFKrKE52MMkUmnPtx22nA06EYwRvyJ7nEqMluWbdaxxaTOgCVYMX2wy47fuK09PU1rhN6BfKze-JjrJ1Qzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086184&ar=3868562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx026f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Saccharomyces</em> phylogeny. Both trees were constructed from unambiguously aligned concatenated DNA sequences using the Maximum likelihood phylogeny PhyML program. (A) mtDNA-derived phylogeny from DNA sequences coding for proteins in the order <em>cox1</em>, <em>atp8</em>, <em>atp6</em>, <em>cob</em>, <em>atp9</em>, <em>cox2</em>, <em>cox3</em>. (B) Nuclear DNA-derived phylogenetic tree from <em>CCA1</em>, <em>CYT1</em>, <em>MLS1</em>, <em>RPS5</em>, <em>LAS1</em>, <em>MET4</em>, <em>NUP116</em>, <em>ZDS2</em>, <em>PDR10</em> and <em>DSN1</em> protein-coding genes used in population studies.<a reveal-id="dsx026-B36 dsx026-B37 dsx026-B38 dsx026-B39" data-open="dsx026-B36 dsx026-B37 dsx026-B38 dsx026-B39" class="link link-ref link-reveal xref-bibr"><sup>36–39</sup></a> The branch length is proportional to the nucleotide differences indicated by the bar. The numbers given at the nodes are the frequencies of a given branch appearing in 1,000 bootstrap replications. All are above 50%, indicating good statistical support. <em>Naumovia castellii</em> NRRL Y-12630 was used as an outgroup.</p></div></div><p>Even more complicated relationships are found among the hybrid species from the lager beer species/<em>S. uvarum</em> clade (<a reveal-id="dsx026-F6" data-open="dsx026-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). It consists historically of two species associated with human activities. <em>S. pastorianus</em> is a typical bottom lager beer yeast and <em>S. bayanus</em> (CBS 380) was isolated from turbid beer. Only <em>S. uvarum</em> (CBS 395) was isolated from natural substrate (blackcurrant).<a reveal-id="dsx026-B26" data-open="dsx026-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> The origin of <em>S. pastorianus</em> was studied in detail and has been elucidated by the discovery of a new species <em>S. eubayanus</em>.<a reveal-id="dsx026-B5" data-open="dsx026-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> All modern <em>S. pastorianus</em> are aneuploid descendants of a tetraploid <em>S. eubayanus</em>/<em>S. cerevisiae</em> hybrid (2<em>n</em>, 2<em>n</em>) with limited loss of contribution from either parent.<a reveal-id="dsx026-B4" data-open="dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx026-B5" data-open="dsx026-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx026-B13" data-open="dsx026-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx026-B88" data-open="dsx026-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a> Apparently, the <em>S. pastorianus</em> strain sequenced in this work as well as a number of other strains inherited mtDNA exclusively from <em>S. eubayanus</em> mtDNA.<a reveal-id="dsx026-B13" data-open="dsx026-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx026-B89" data-open="dsx026-B89" class="link link-ref link-reveal xref-bibr"><sup>89</sup></a> The systematics of <em>S. bayanus</em> and <em>S. uvarum</em> has been confusing and controversial for decades. <em>S. uvarum</em> (<em>S. bayanus var. uvarum</em>) strains represent a nearly pure lineage that contains very little genetic input from other <em>Saccharomyces</em> species although type strain CBS 395 does not produce viable spores and sequencing revealed that the strain is aneuploid.<a reveal-id="dsx026-B26" data-open="dsx026-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> However, a number of different strains (including CBS 7001 ‘genomic type strain’) are good sporulants and introgression from other <em>Saccharomyces</em> species is low.<a reveal-id="dsx026-B4" data-open="dsx026-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx026-B90" data-open="dsx026-B90" class="link link-ref link-reveal xref-bibr"><sup>90</sup></a><sup>,</sup><em>S. bayanus</em> is not a ‘species’ in an evolutionary sense and should be rather understood as a product of the artificial brewing environment with no occurrence in nature. Therefore, all known strains of <em>S. bayanus</em> (including the type strain CBS 380) are hybrids of <em>S. eubayanus</em> and <em>S. uvarum</em> that contain contributions from <em>S. cerevisiae</em> in at least some cases.<a reveal-id="dsx026-B26" data-open="dsx026-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Comparison of mtDNA clearly demonstrates its <em>S. uvarum</em> origin in <em>S. bayanus</em> CBS 380. A similar conclusion was reported by<a reveal-id="dsx026-B91" data-open="dsx026-B91" class="link link-ref link-reveal xref-bibr"><sup>91</sup></a> when they compared part of the <em>cox2</em> gene in the collection of <em>S. eubayanus</em> and its interspecies hybrids. A model of <em>S. bayanus</em> formation involves multiple hybridization events of <em>S. pastorianus</em> with wild strains of <em>S. uvarum</em>. In the case of <em>S. bayanus</em> CBS 380 it happened only recently as mtDNA from <em>S. uvarum</em> CBS 395<sup>T</sup> and <em>S. bayanus</em> CBS 380<sup>T</sup> is nearly identical. They only differ in four substitutions and five indels of which one is the insertion of a GC cluster and one a TTATTTAC repeat. Therefore, comparison of mtDNA should not be neglected in genomic studies as it is an important tool to understand the origin and evolutionary history of some yeast species.</p>                            <h2 scrollto-destination=104086186 id="104086186" class="section-title" >4. Conclusions</h2>
<p>We sequenced mtDNA from a variety of <em>Saccharomyces</em> species by Illumina MiSeq. All are circularly mapped molecules decreasing in size with phylogenetic distance from <em>S. cerevisiae</em> but with similar gene content including regulatory and selfish elements like origins of replication, introns, free-standing ORFs or GC clusters. Their most profound feature is species-specific alteration in gene order, apparently accompanying the speciation process in yeasts with larger mtDNA. Conserved mtDNA gene order seems to be a species-specific feature as pointed out by the comparison of more than 100 <em>S. cerevisiae</em> strains<a reveal-id="dsx026-B20" data-open="dsx026-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx026-B21" data-open="dsx026-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and ∼15 <em>S. paradoxus</em> strains<a reveal-id="dsx026-B19" data-open="dsx026-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> as well as 50 strains from <em>Lachancea thermotolerans</em>.<a reveal-id="dsx026-B80" data-open="dsx026-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a> On the other hand, reshuffling of mitochondrial genes evidently accompanies the yeast speciation process if mitochondrial genomes are large enough.<a reveal-id="dsx026-B7" data-open="dsx026-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> The genetic code differs from well-known yeast mitochondrial code as GUG is used as the translation start in the <em>S. cariocanus cox3</em> gene as well as in some free-standing <em>ORF1</em>. Arginine CGA and CGC codons considered as unassigned are present in maturases/HE coded by introns. Because of the alternative splicing of <em>cox1</em>I5β and <em>cob</em>I1α, AUA considered as an initiation codon is absent. The multilocus phylogeny, inferred from mtDNA, does not correlate with the trees derived from nuclear genes. mtDNA data demonstrate that <em>S. cariocanus</em> should be assigned as a separate species and <em>S. bayanus</em> CBS 380 should not be considered as a distinct species due to mtDNA nearly identical to <em>S. uvarum</em> mtDNA. Apparently, comparison of mtDNAs should be included in genomic studies, as it is an important tool to understand the origin and evolutionary history of some yeast species.</p>                            <h2 scrollto-destination=104086188 id="104086188" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank two anonymous reviewers for their valuable comments. The study is a result of implementation of project REVOGENE – Research Center for Molecular Genetics (ITMS 26240220067) supported by the Research &amp; Development Operational Program funded by the ERDF. Part of this work was funded by grants from VEGA 1/0360/12 and 1/0048/16.</p>                            <h3 scrollto-destination=104086190 id="104086190" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=104086192 id="104086192" class="section-title" >Accession numbers</h3>
<p>KX657740, KX657741, KX657742, KX657743, KX657744, KX657745, KX657746, KX657747, KX657748, KX657749, KX657750</p>                            <h3 scrollto-destination=104086194 id="104086194" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s1_dsx026.doc?Expires=2147483647&Signature=1NPSjXvfqnbox4N9CWZa9hD5xZslFbIbvGRK9RiHRCHuTDA0~7c45lMVSPIQVUyE-9pxJ9KouMUJRNuAdcb~s58URh14gec-kzVbmpogVcvMZrFJBiVK3Rx~rPWQcTGZGLMF389vonrMTo0M9cmx8RYAUvlJfKSGPmRGNnegIwtd-rRa62NykzkhGS0tac0Npt78Imuf6obxyN6eGc5xqsXXRNMJZFVqAE-WmIaRjTgKa53kL-BROzLKfIzhpBn4Q02K10aVLSgZx8NdVeM0PA7nU635v56YiLJMbtr4L082m9xGc2mX~xQe1-EyjoFE5ulcE3hfKiiVGTJ6YiTzsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=104086196 id="104086196" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx026-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vaughan-Martini</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Martini</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, <em>Saccharomyces meyen ex reess</em> (1870). In: <div class="person-group"><div class="name string-name"><div class="surname">Kurtzman</div> <div class="given-names">C.P.</div></div></div>, ed. <div class="source">The Yeasts, A Taxonomic Study</div> , 
<div class="publisher-name">Elsevier</div>: 
<div class="publisher-loc">Amsterdam, NL</div>. pp. <div class="fpage">733</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Yeasts%2C%20A%20Taxonomic%20Study&amp;author=A.%20Vaughan-Martini&amp;author=A.%20Martini&amp;author=C.P.%20Kurtzman&amp;publication_year=2011&amp;book=The%20Yeasts%2C%20A%20Taxonomic%20Study" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/B978-0-444-52149-1.00061-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FB978-0-444-52149-1.00061-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FB978-0-444-52149-1.00061-6"> </span></div> </div></div></div></div></div><div content-id="dsx026-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Replansky</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Koufopanou</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Greig</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Bell</div> <div class="given-names">G.</div></div></div> <div class="year">2008</div>, 
<div class="article-title"><em>Saccharomyces sensu stricto</em> as a model system for evolution and ecology,</div> <div class="source">Trends Ecol. Evol</div> ., <div class="volume">23</div>, <div class="fpage">494</div>–<div class="lpage">501</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saccharomyces%20sensu%20stricto%20as%20a%20model%20system%20for%20evolution%20and%20ecology%2C&amp;author=T.%20Replansky&amp;author=V.%20Koufopanou&amp;author=D.%20Greig&amp;author=G.%20Bell&amp;publication_year=2008&amp;journal=Trends%20Ecol.%20Evol&amp;volume=23&amp;pages=494-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2008.05.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2008.05.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2008.05.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18656281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sipiczki</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Interspecies hybridization and recombination in <em>Saccharomyces</em> wine yeasts,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">8</div>, <div class="fpage">996</div>–<div class="lpage">1007</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interspecies%20hybridization%20and%20recombination%20in%20Saccharomyces%20wine%20yeasts%2C&amp;author=M.%20Sipiczki&amp;publication_year=2008&amp;journal=FEMS%20Yeast%20Res&amp;volume=8&amp;pages=996-1007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2008.8.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2008.8.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2008.8.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18355270" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Borneman</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Pretorius</div> <div class="given-names">I.S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genomic insights into the <em>Saccharomyces sensu stricto</em> complex,</div> <div class="source">Genetics</div> , <div class="volume">199</div>, <div class="fpage">281</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20insights%20into%20the%20Saccharomyces%20sensu%20stricto%20complex%2C&amp;author=A.R.%20Borneman&amp;author=I.S.%20Pretorius&amp;publication_year=2015&amp;journal=Genetics&amp;volume=199&amp;pages=281-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.114.173633" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.114.173633" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.114.173633"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25657346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baker</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Bellora</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The genome sequence of <em>Saccharomyces eubayanus</em> and the domestication of lager-brewing yeasts,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">2818</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Saccharomyces%20eubayanus%20and%20the%20domestication%20of%20lager-brewing%20yeasts%2C&amp;author=E.%20Baker&amp;author=B.%20Wang&amp;author=N.%20Bellora&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=2818-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msv168" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsv168" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsv168"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26269586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Foury</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Roganti</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Lecrenier</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Purnelle</div> <div class="given-names">B.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">The complete sequence of the mitochondrial genome of <em>Saccharomyces cerevisiae</em>,</div> <div class="source">FEBS Lett</div> ., <div class="volume">440</div>, <div class="fpage">325</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20sequence%20of%20the%20mitochondrial%20genome%20of%20Saccharomyces%20cerevisiae%2C&amp;author=F.%20Foury&amp;author=T.%20Roganti&amp;author=N.%20Lecrenier&amp;author=B.%20Purnelle&amp;publication_year=1998&amp;journal=FEBS%20Lett&amp;volume=440&amp;pages=325-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(98)01467-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(98)01467-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(98)01467-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9872396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Freel</div> <div class="given-names">K.C.</div></div>, <div class="name string-name"><div class="surname">Friedrich</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Schacherer</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Mitochondrial genome evolution in yeasts: an all-encompassing view,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">15</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20genome%20evolution%20in%20yeasts%3A%20an%20all-encompassing%20view%2C&amp;author=K.C.%20Freel&amp;author=A.%20Friedrich&amp;author=J.%20Schacherer&amp;publication_year=2015&amp;journal=FEMS%20Yeast%20Res&amp;volume=15&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/femsyr/fov023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ffemsyr%2Ffov023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ffemsyr%2Ffov023"> </span></div> </div></div></div></div></div><div content-id="dsx026-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kellis</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Patterson</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Endrizzi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Birren</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Sequencing and comparison of yeast species to identify genes and regulatory elements,</div> <div class="source">Nature</div> , <div class="volume">423</div>, <div class="fpage">241</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20comparison%20of%20yeast%20species%20to%20identify%20genes%20and%20regulatory%20elements%2C&amp;author=M.%20Kellis&amp;author=N.%20Patterson&amp;author=M.%20Endrizzi&amp;author=B.%20Birren&amp;author=E.S.%20Lander&amp;publication_year=2003&amp;journal=Nature&amp;volume=423&amp;pages=241-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12748633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liti</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Ba</div> <div class="given-names">A.N.N.</div></div>, <div class="name string-name"><div class="surname">Blythe</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">High quality de novo sequencing and assembly of the <em>Saccharomyces arboricolus</em> genome,</div> <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20quality%20de%20novo%20sequencing%20and%20assembly%20of%20the%20Saccharomyces%20arboricolus%20genome%2C&amp;author=G.%20Liti&amp;author=A.N.N.%20Ba&amp;author=M.%20Blythe&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-69" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-69" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-69"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23323973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hittinger</div> <div class="given-names">C.T.</div></div>, <div class="name string-name"><div class="surname">Rokas</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bai</div> <div class="given-names">F-Y.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genomics and the making of yeast biodiversity,</div> <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">35</div>, <div class="fpage">100</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20and%20the%20making%20of%20yeast%20biodiversity%2C&amp;author=C.T.%20Hittinger&amp;author=A.%20Rokas&amp;author=F-Y.%20Bai&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=35&amp;pages=100-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2015.10.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2015.10.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2015.10.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26649756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Procházka</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Franko</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Poláková</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sulo</div> <div class="given-names">P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A complete sequence of <em>Saccharomyces paradoxus</em> mitochondrial genome that restores the respiration in <em>S. cerevisiae</em>,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">12</div>, <div class="fpage">819</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20complete%20sequence%20of%20Saccharomyces%20paradoxus%20mitochondrial%20genome%20that%20restores%20the%20respiration%20in%20S.%20cerevisiae%2C&amp;author=E.%20Proch%C3%A1zka&amp;author=F.%20Franko&amp;author=S.%20Pol%C3%A1kov%C3%A1&amp;author=P.%20Sulo&amp;publication_year=2012&amp;journal=FEMS%20Yeast%20Res&amp;volume=12&amp;pages=819-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2012.12.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2012.12.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2012.12.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22830625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nakao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kanamori</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Genome sequence of the lager brewing yeast, an interspecies hybrid,</div> <div class="source">DNA Res</div> ., <div class="volume">16</div>, <div class="fpage">115</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20lager%20brewing%20yeast%2C%20an%20interspecies%20hybrid%2C&amp;author=Y.%20Nakao&amp;author=T.%20Kanamori&amp;author=T.%20Itoh&amp;publication_year=2009&amp;journal=DNA%20Res&amp;volume=16&amp;pages=115-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Okuno</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ryusui</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Morimoto</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization</div>, <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">67</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20analysis%20of%20lager%20brewing%20yeast%20strains%20reveals%20the%20evolutionary%20history%20of%20interspecies%20hybridization&amp;author=M.%20Okuno&amp;author=R.%20Kajitani&amp;author=R.%20Ryusui&amp;author=H.%20Morimoto&amp;author=Y.%20Kodama&amp;author=T.%20Itoh&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=67-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26732986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Comparison of next generation sequencing systems,</div> <div class="source">J. Biomed. Biotechnol</div> ., <div class="volume">2012</div>, <div class="fpage">1</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20next%20generation%20sequencing%20systems%2C&amp;author=L.%20Liu&amp;author=Y.%20Li&amp;author=S.%20Li&amp;publication_year=2012&amp;journal=J.%20Biomed.%20Biotechnol&amp;volume=2012&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21836813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quail</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Coupland</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers,</div> <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20tale%20of%20three%20next%20generation%20sequencing%20platforms%3A%20comparison%20of%20Ion%20Torrent%2C%20Pacific%20Biosciences%20and%20Illumina%20MiSeq%20sequencers%2C&amp;author=M.A.%20Quail&amp;author=M.%20Smith&amp;author=P.%20Coupland&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Skelly</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Merrihew</div> <div class="given-names">G.E.</div></div>, <div class="name string-name"><div class="surname">Riffle</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast,</div> <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">1496</div>–<div class="lpage">504</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20phenomics%20reveals%20insight%20into%20the%20structure%20of%20phenotypic%20diversity%20in%20budding%20yeast%2C&amp;author=D.A.%20Skelly&amp;author=G.E.%20Merrihew&amp;author=M.%20Riffle&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=1496-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.155762.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.155762.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.155762.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23720455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bergstrom</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Simpson</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Salinas</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A high-definition view of functional genetic variation from natural yeast genomes</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">31</div>, <div class="fpage">872</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-definition%20view%20of%20functional%20genetic%20variation%20from%20natural%20yeast%20genomes&amp;author=A.%20Bergstrom&amp;author=J.T.%20Simpson&amp;author=F.%20Salinas&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol&amp;volume=31&amp;pages=872-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24425782" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lang</div> <div class="given-names">B.F.</div></div>, <div class="name string-name"><div class="surname">Jakubkova</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hegedusova</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Massive programmed translational jumping in mitochondria,</div> <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">111</div>, <div class="fpage">5926</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massive%20programmed%20translational%20jumping%20in%20mitochondria%2C&amp;author=B.F.%20Lang&amp;author=M.%20Jakubkova&amp;author=E.%20Hegedusova&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=111&amp;pages=5926-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1322190111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1322190111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1322190111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24711422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Hao</div> <div class="given-names">W.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A dynamic mobile DNA family in the yeast mitochondrial genome,</div> <div class="source">G3 (Bethesda)</div> , <div class="volume">5</div>, <div class="fpage">1273</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dynamic%20mobile%20DNA%20family%20in%20the%20yeast%20mitochondrial%20genome%2C&amp;author=B.%20Wu&amp;author=W.%20Hao&amp;publication_year=2015&amp;journal=G3%20(Bethesda)&amp;volume=5&amp;pages=1273-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.115.017822" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.115.017822" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.115.017822"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25897011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wolters</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Chiu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fiumera</div> <div class="given-names">H.L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Population structure of mitochondrial genomes in <em>Saccharomyces cerevisiae</em>,</div> <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20of%20mitochondrial%20genomes%20in%20Saccharomyces%20cerevisiae%2C&amp;author=J.F.%20Wolters&amp;author=K.%20Chiu&amp;author=H.L.%20Fiumera&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1664-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1664-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1664-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25553907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Strope</div> <div class="given-names">P.K.</div></div>, <div class="name string-name"><div class="surname">Skelly</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Kozmin</div> <div class="given-names">S.G.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The 100-genomes strains, an <em>S. cerevisiae</em> resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen,</div> <div class="source">Genome Res</div> ., <div class="volume">25</div>, <div class="fpage">762</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20100-genomes%20strains%2C%20an%20S.%20cerevisiae%20resource%20that%20illuminates%20its%20natural%20phenotypic%20and%20genotypic%20variation%20and%20emergence%20as%20an%20opportunistic%20pathogen%2C&amp;author=P.K.%20Strope&amp;author=D.A.%20Skelly&amp;author=S.G.%20Kozmin&amp;publication_year=2015&amp;journal=Genome%20Res&amp;volume=25&amp;pages=762-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.185538.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.185538.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.185538.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25840857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Špírek</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Poláková</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jatzová</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sulo</div> <div class="given-names">P.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Post-zygotic sterility and cytonuclear compatibility limits in <em>S. cerevisiae</em> xenomitochondrial cybrids</div>, <div class="source">Front. Genet</div> ., <div class="volume">5</div>, <div class="fpage">1</div>–<div class="lpage">15</div>.</div></div></div></div><div content-id="dsx026-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Defontaine</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lecocq</div> <div class="given-names">F.M.</div></div>, <div class="name string-name"><div class="surname">Hallet</div> <div class="given-names">J.N.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">A rapid miniprep method for the preparation of yeast mitochondrial DNA,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">19</div>, <div class="fpage">185</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20miniprep%20method%20for%20the%20preparation%20of%20yeast%20mitochondrial%20DNA%2C&amp;author=A.%20Defontaine&amp;author=F.M.%20Lecocq&amp;author=J.N.%20Hallet&amp;publication_year=1991&amp;journal=Nucleic%20Acids%20Res&amp;volume=19&amp;pages=185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/19.1.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F19.1.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F19.1.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2011504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Piškur</div> <div class="given-names">J.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">Respiratory-competent yeast mitochondrial DNAs generated by deleting intergenic region,</div> <div class="source">Gene</div> , <div class="volume">81</div>, <div class="fpage">165</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Respiratory-competent%20yeast%20mitochondrial%20DNAs%20generated%20by%20deleting%20intergenic%20region%2C&amp;author=J.%20Pi%C5%A1kur&amp;publication_year=1989&amp;journal=Gene&amp;volume=81&amp;pages=165-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(89)90347-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(89)90347-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(89)90347-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2553538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Piškur</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Smole</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Groth</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Petersen</div> <div class="given-names">R.F.</div></div>, <div class="name string-name"><div class="surname">Pedersen</div> <div class="given-names">M.B.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Structure and genetic stability of mitochondrial genomes vary among yeasts of the genus <em>Saccharomyces</em></div>, <div class="source">Int. J. Syst. Bacteriol</div> ., <div class="volume">48</div>, <div class="fpage">1015</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20genetic%20stability%20of%20mitochondrial%20genomes%20vary%20among%20yeasts%20of%20the%20genus%20Saccharomyces&amp;author=J.%20Pi%C5%A1kur&amp;author=S.%20Smole&amp;author=C.%20Groth&amp;author=R.F.%20Petersen&amp;author=M.B.%20Pedersen&amp;publication_year=1998&amp;journal=Int.%20J.%20Syst.%20Bacteriol&amp;volume=48&amp;pages=1015-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-48-3-1015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-48-3-1015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-48-3-1015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9734058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Libkind</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Hittinger</div> <div class="given-names">C.T.</div></div>, <div class="name string-name"><div class="surname">Valério</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Microbe domestication and the identification of the wild genetic stock of lager-brewing yeast,</div> <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">108</div>, <div class="fpage">14539</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbe%20domestication%20and%20the%20identification%20of%20the%20wild%20genetic%20stock%20of%20lager-brewing%20yeast%2C&amp;author=D.%20Libkind&amp;author=C.T.%20Hittinger&amp;author=E.%20Val%C3%A9rio&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=108&amp;pages=14539-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1105430108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1105430108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1105430108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21873232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lang</div> <div class="given-names">B.F.</div></div>, <div class="name string-name"><div class="surname">Laforest</div> <div class="given-names">M.-J.</div></div>, <div class="name string-name"><div class="surname">Burger</div> <div class="given-names">G.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Mitochondrial introns: a critical view,</div> <div class="source">Trends Genet</div> ., <div class="volume">23</div>, <div class="fpage">119</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20introns%3A%20a%20critical%20view%2C&amp;author=B.F.%20Lang&amp;author=M.-J.%20Laforest&amp;author=G.%20Burger&amp;publication_year=2007&amp;journal=Trends%20Genet&amp;volume=23&amp;pages=119-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.01.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.01.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.01.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17280737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lambowitz</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Belfort</div> <div class="given-names">M.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Introns as mobile genetic elements,</div> <div class="source">Annu. Rev. Biochem</div> ., <div class="volume">62</div>, <div class="fpage">587</div>–<div class="lpage">622</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introns%20as%20mobile%20genetic%20elements%2C&amp;author=A.M.%20Lambowitz&amp;author=M.%20Belfort&amp;publication_year=1993&amp;journal=Annu.%20Rev.%20Biochem&amp;volume=62&amp;pages=587-622" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.bi.62.070193.003103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.bi.62.070193.003103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.bi.62.070193.003103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8352597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Zamaroczy</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div></div> <div class="year">1986b</div>, 
<div class="article-title">The primary structure of the mitochondrial genome of <em>Saccharomyces cerevisiae</em> – a review</div>, <div class="source">Gene</div> , <div class="volume">47</div>, <div class="fpage">155</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20primary%20structure%20of%20the%20mitochondrial%20genome%20of%20Saccharomyces%20cerevisiae%20%E2%80%93%20a%20review&amp;author=M.%20de%20Zamaroczy&amp;author=G.%20Bernardi&amp;publication_year=1986b&amp;journal=Gene&amp;volume=47&amp;pages=155-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(86)90060-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(86)90060-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(86)90060-0"> </span></div> </div></div></div></div></div><div content-id="dsx026-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Konings</div> <div class="given-names">D.A.M.</div></div>, <div class="name string-name"><div class="surname">Gutell</div> <div class="given-names">R.R.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">A comparison of thermodynamic foldings with comparatively derived structures of 16S and 16S-like rRNAs</div>, <div class="source">RNA</div> , <div class="volume">1</div>, <div class="fpage">559</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20thermodynamic%20foldings%20with%20comparatively%20derived%20structures%20of%2016S%20and%2016S-like%20rRNAs&amp;author=D.A.M.%20Konings&amp;author=R.R.%20Gutell&amp;publication_year=1995&amp;journal=RNA&amp;volume=1&amp;pages=559-74" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7489516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fields</div> <div class="given-names">D.S.</div></div>, <div class="name string-name"><div class="surname">Gutell</div> <div class="given-names">R.R.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">An analysis of large rRNA sequences folded by a thermodynamic method,</div> <div class="source">Fold. Des</div> ., <div class="volume">1</div>, <div class="fpage">419</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20analysis%20of%20large%20rRNA%20sequences%20folded%20by%20a%20thermodynamic%20method%2C&amp;author=D.S.%20Fields&amp;author=R.R.%20Gutell&amp;publication_year=1996&amp;journal=Fold.%20Des&amp;volume=1&amp;pages=419-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1359-0278(96)00058-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1359-0278(96)00058-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1359-0278(96)00058-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9080188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seif</div> <div class="given-names">E.R.</div></div>, <div class="name string-name"><div class="surname">Forget</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Lang</div> <div class="given-names">B.F.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Mitochondrial RNase P RNAs in ascomycete fungi: lineage-specific variations in RNA secondary structure,</div> <div class="source">RNA</div> , <div class="volume">9</div>, <div class="fpage">1073</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20RNase%20P%20RNAs%20in%20ascomycete%20fungi%3A%20lineage-specific%20variations%20in%20RNA%20secondary%20structure%2C&amp;author=E.R.%20Seif&amp;author=L.%20Forget&amp;author=N.C.%20Martin&amp;author=B.F.%20Lang&amp;publication_year=2003&amp;journal=RNA&amp;volume=9&amp;pages=1073-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.5880403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.5880403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.5880403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12923256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">O’Brien</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Marie</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Lang</div> <div class="given-names">B.F.</div></div>, <div class="name string-name"><div class="surname">Burger</div> <div class="given-names">G.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">GOBASE – a database of organelle and bacterial genome information</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">D697</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GOBASE%20%E2%80%93%20a%20database%20of%20organelle%20and%20bacterial%20genome%20information&amp;author=E.A.%20O%E2%80%99Brien&amp;author=Y.%20Zhang&amp;author=L.S.%20Yang&amp;author=E.%20Wang&amp;author=V.%20Marie&amp;author=B.F.%20Lang&amp;author=G.%20Burger&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=D697-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">CLC Genomics Workbench 9.5.2. <a class="link link-uri" href="https://www.qiagenbioinformatics.com/" target="_blank">https://www.qiagenbioinformatics.com/</a></div></div></div></div><div content-id="dsx026-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Felsenstein</div> <div class="given-names">J.</div></div></div> <div class="year">1988</div>, 
<div class="article-title">Phylogenies from molecular sequences: inference and reliability,</div> <div class="source">Annu, Rev. Genet</div> ., <div class="volume">22</div>, <div class="fpage">521</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenies%20from%20molecular%20sequences%3A%20inference%20and%20reliability%2C&amp;author=J.%20Felsenstein&amp;publication_year=1988&amp;journal=Annu%2C%20Rev.%20Genet&amp;volume=22&amp;pages=521-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.22.120188.002513" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.22.120188.002513" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.22.120188.002513"> </span></div> </div></div></div></div></div><div content-id="dsx026-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fay</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Benavides</div> <div class="given-names">J.A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Evidence for domesticated and wild populations of <em>Saccharomyces cerevisiae</em>,</div> <div class="source">PLoS Genet</div> ., <div class="volume">1</div>, <div class="fpage">66</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20domesticated%20and%20wild%20populations%20of%20Saccharomyces%20cerevisiae%2C&amp;author=J.C.%20Fay&amp;author=J.A.%20Benavides&amp;publication_year=2005&amp;journal=PLoS%20Genet&amp;volume=1&amp;pages=66-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0010005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0010005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0010005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16103919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aa</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Townsend</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Adams</div> <div class="given-names">R.I.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">J.W.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Population structure and gene evolution in <em>Saccharomyces cerevisiae</em>,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">6</div>, <div class="fpage">702</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20gene%20evolution%20in%20Saccharomyces%20cerevisiae%2C&amp;author=E.%20Aa&amp;author=J.P.%20Townsend&amp;author=R.I.%20Adams&amp;author=K.M.%20Nielsen&amp;author=J.W.%20Taylor&amp;publication_year=2006&amp;journal=FEMS%20Yeast%20Res&amp;volume=6&amp;pages=702-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2006.6.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2006.6.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2006.6.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16879422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ayoub</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Legras</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Saliba</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Gaillardin</div> <div class="given-names">C.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Application of multi locus sequence typing to the analysis of the biodiversity of indigenous <em>Saccharomyces cerevisiae</em> wine yeasts from Lebanon,</div> <div class="source">J. Appl. Microbiol</div> ., <div class="volume">100</div>, <div class="fpage">699</div>–<div class="lpage">711</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20multi%20locus%20sequence%20typing%20to%20the%20analysis%20of%20the%20biodiversity%20of%20indigenous%20Saccharomyces%20cerevisiae%20wine%20yeasts%20from%20Lebanon%2C&amp;author=M.J.%20Ayoub&amp;author=J.L.%20Legras&amp;author=R.%20Saliba&amp;author=C.%20Gaillardin&amp;publication_year=2006&amp;journal=J.%20Appl.%20Microbiol&amp;volume=100&amp;pages=699-711" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jam.2006.100.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjam.2006.100.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjam.2006.100.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16553725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Q.M.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">W.Q.</div></div>, <div class="name string-name"><div class="surname">Liti</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Bai</div> <div class="given-names">F.Y.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Surprisingly diverged populations of <em>Saccharomyces cerevisiae</em> in natural environments remote from human activity,</div> <div class="source">Mol. Ecol</div> ., <div class="volume">21</div>, <div class="fpage">5404</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Surprisingly%20diverged%20populations%20of%20Saccharomyces%20cerevisiae%20in%20natural%20environments%20remote%20from%20human%20activity%2C&amp;author=Q.M.%20Wang&amp;author=W.Q.%20Liu&amp;author=G.%20Liti&amp;author=S.A.%20Wang&amp;author=F.Y.%20Bai&amp;publication_year=2012&amp;journal=Mol.%20Ecol&amp;volume=21&amp;pages=5404-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-294X.2012.05732.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2012.05732.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-294X.2012.05732.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22913817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Scannell</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Zill</div> <div class="given-names">O.A.</div></div></div>, Rokas, et al.   <div class="year">2011</div>, 
<div class="article-title">The awesome power of yeast evolutionary genetics: new genome sequences and strain resources for the <em>Saccharomyces sensu stricto</em> genus</div>, <div class="source">G3 (Bethesda)</div> , <div class="volume">1</div>, <div class="fpage">11</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20awesome%20power%20of%20yeast%20evolutionary%20genetics%3A%20new%20genome%20sequences%20and%20strain%20resources%20for%20the%20Saccharomyces%20sensu%20stricto%20genus&amp;author=D.R.%20Scannell&amp;author=O.A.%20Zill&amp;publication_year=2011&amp;journal=G3%20(Bethesda)&amp;volume=1&amp;pages=11-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.111.000273" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.111.000273" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.111.000273"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clark-Walker</div> <div class="given-names">G.D.</div></div>, <div class="name string-name"><div class="surname">Weiller</div> <div class="given-names">G.F.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">The structure of the small mitochondrial DNA of <em>Kluyveromyces thermotolerans</em> is likely to reflect the ancestral gene order in fungi,</div> <div class="source">J. Mol. Evol</div> ., <div class="volume">38</div>, <div class="fpage">593</div>–<div class="lpage">601</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20the%20small%20mitochondrial%20DNA%20of%20Kluyveromyces%20thermotolerans%20is%20likely%20to%20reflect%20the%20ancestral%20gene%20order%20in%20fungi%2C&amp;author=G.D.%20Clark-Walker&amp;author=G.F.%20Weiller&amp;publication_year=1994&amp;journal=J.%20Mol.%20Evol&amp;volume=38&amp;pages=593-601" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00175879" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00175879" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00175879"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8083884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lang</div> <div class="given-names">B.F.</div></div>, <div class="name string-name"><div class="surname">Lavrov</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Beck</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Steinberg</div> <div class="given-names">S.V.</div></div></div> <div class="year">2012</div>, Mitochondrial tRNA structure, identity, and evolution of the genetic code, In: <div class="person-group"><div class="name string-name"><div class="surname">Bullerwell</div> <div class="given-names">C. E.</div></div></div>, ed., <div class="source">Organelle Genetics. Evolution of Organelle Genomes and Gene Expression</div> , 
<div class="publisher-name">Springer-Verlag</div>: 
<div class="publisher-loc">Berlin, Germany</div>, pp. <div class="fpage">431</div>–<div class="lpage">74</div>.</div></div></div></div><div content-id="dsx026-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elzanowski</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ostell</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, The genetic codes, NCBI website, 2010, Available at: <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi" target="_blank">http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi</a>.</div></div></div></div><div content-id="dsx026-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">García-Guerrero</div> <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">Zamudio-Ochoa</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Camacho-Villasana</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">García-Villegas</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Reyes-Prieto</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Pérez-Martínez</div> <div class="given-names">X.</div></div></div> <div class="year">2016</div>, Evolution of translation in mitochondria, In: <div class="person-group"><div class="name string-name"><div class="surname">Hernández</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Jagus</div> <div class="given-names">R.</div></div></div>, eds., <div class="source">Evolution of the Protein Synthesis Machinery and its Regulation</div> , 
<div class="publisher-name">Springer International Publishing</div>: 
<div class="publisher-loc">Switzerland</div>, pp. <div class="fpage">109</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20Protein%20Synthesis%20Machinery%20and%20its%20Regulation&amp;author=A.E.%20Garc%C3%ADa-Guerrero&amp;author=A.%20Zamudio-Ochoa&amp;author=Y.%20Camacho-Villasana&amp;author=R.%20Garc%C3%ADa-Villegas&amp;author=A.%20Reyes-Prieto&amp;author=X.%20P%C3%A9rez-Mart%C3%ADnez&amp;author=G.%20Hern%C3%A1ndez&amp;author=R.%20Jagus&amp;publication_year=2016&amp;book=Evolution%20of%20the%20Protein%20Synthesis%20Machinery%20and%20its%20Regulation" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-3-319-39468-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-319-39468-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-3-319-39468-8"> </span></div> </div></div></div></div></div><div content-id="dsx026-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moran</div> <div class="given-names">J.V.</div></div>, <div class="name string-name"><div class="surname">Zimmerly</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Eskes</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">1995</div>, 
<div class="article-title">Mobile group II introns of yeast mitochondrial DNA are novel site-specific retroelements,</div> <div class="source">Mol. Cell. Biol</div> ., <div class="volume">15</div>, <div class="fpage">2828</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20group%20II%20introns%20of%20yeast%20mitochondrial%20DNA%20are%20novel%20site-specific%20retroelements%2C&amp;author=J.V.%20Moran&amp;author=S.%20Zimmerly&amp;author=R.%20Eskes&amp;publication_year=1995&amp;journal=Mol.%20Cell.%20Biol&amp;volume=15&amp;pages=2828-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.15.5.2828" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.15.5.2828" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.15.5.2828"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7537853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lambowitz</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Zimmerly</div> <div class="given-names">S.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Mobile group II introns,</div> <div class="source">Ann. Rev. Genet</div> ., <div class="volume">38</div>, <div class="fpage">1</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20group%20II%20introns%2C&amp;author=A.M.%20Lambowitz&amp;author=S.%20Zimmerly&amp;publication_year=2004&amp;journal=Ann.%20Rev.%20Genet&amp;volume=38&amp;pages=1-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.091600" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.38.072902.091600" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.38.072902.091600"> </span></div> </div></div></div></div></div><div content-id="dsx026-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kurtzman</div> <div class="given-names">C.P.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Phylogenetic circumscription of <em>Saccharomyces</em>, <em>Kluyveromyces</em> and other members of the Saccharomycetaceae, and the proposal of the new genera <em>Lachancea</em>, <em>Nakaseomyces</em>, <em>Naumonia</em>, <em>Vanderwaltozyma</em> and <em>Zygotoruspora</em>,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">4</div>, <div class="fpage">233</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20circumscription%20of%20Saccharomyces%2C%20Kluyveromyces%20and%20other%20members%20of%20the%20Saccharomycetaceae%2C%20and%20the%20proposal%20of%20the%20new%20genera%20Lachancea%2C%20Nakaseomyces%2C%20Naumonia%2C%20Vanderwaltozyma%20and%20Zygotoruspora%2C&amp;author=C.P.%20Kurtzman&amp;publication_year=2003&amp;journal=FEMS%20Yeast%20Res&amp;volume=4&amp;pages=233-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1567-1356(03)00175-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1567-1356(03)00175-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1567-1356(03)00175-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14654427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langkjær</div> <div class="given-names">R.B.</div></div>, <div class="name string-name"><div class="surname">Casaregola</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>, <div class="name string-name"><div class="surname">Gaillardin</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Piškur</div> <div class="given-names">J.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Sequence analysis of three mitochondrial DNA molecules reveals interesting differences among <em>Saccharomyces</em> yeasts,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">3081</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20three%20mitochondrial%20DNA%20molecules%20reveals%20interesting%20differences%20among%20Saccharomyces%20yeasts%2C&amp;author=R.B.%20Langkj%C3%A6r&amp;author=S.%20Casaregola&amp;author=D.W.%20Ussery&amp;author=C.%20Gaillardin&amp;author=J.%20Pi%C5%A1kur&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=3081-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg423" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg423" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg423"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12799436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bonnefoy</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Fox</div> <div class="given-names">T.D.</div></div></div> <div class="year">2000</div>, 
<div class="article-title"><em>In vivo</em> analysis of mutated initiation codons in the mitochondrial <em>COX2</em> gene of <em>Saccharomyces cerevisiae</em> fused to the reporter gene <em>ARG8m</em> reveals lack of downstream reinitiation,</div> <div class="source">Mol. Gen. Genet</div> ., <div class="volume">262</div>, <div class="fpage">1036</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20analysis%20of%20mutated%20initiation%20codons%20in%20the%20mitochondrial%20COX2%20gene%20of%20Saccharomyces%20cerevisiae%20fused%20to%20the%20reporter%20gene%20ARG8m%20reveals%20lack%20of%20downstream%20reinitiation%2C&amp;author=N.%20Bonnefoy&amp;author=T.D.%20Fox&amp;publication_year=2000&amp;journal=Mol.%20Gen.%20Genet&amp;volume=262&amp;pages=1036-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00008646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00008646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00008646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10660064" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bonnefoy</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Bsat</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Fox</div> <div class="given-names">T.D.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Mitochondrial translation of <em>Saccharomyces cerevisiae</em> COX2 mRNA is controlled by the nucleotide sequence specifying the pre-Cox2p leader peptide,</div> <div class="source">Mol. Cell. Biol</div> ., <div class="volume">21</div>, <div class="fpage">2359</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20translation%20of%20Saccharomyces%20cerevisiae%20COX2%20mRNA%20is%20controlled%20by%20the%20nucleotide%20sequence%20specifying%20the%20pre-Cox2p%20leader%20peptide%2C&amp;author=N.%20Bonnefoy&amp;author=N.%20Bsat&amp;author=T.D.%20Fox&amp;publication_year=2001&amp;journal=Mol.%20Cell.%20Biol&amp;volume=21&amp;pages=2359-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.21.7.2359-2372.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.21.7.2359-2372.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.21.7.2359-2372.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11259585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dubot</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Godinot</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Dumur</div> <div class="given-names">V.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">GUG is an efficient initiation codon to translate the human mitochondrial ATP6 gene,</div> <div class="source">Biochem. Biophys. Res. Commun</div> ., <div class="volume">313</div>, <div class="fpage">687</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GUG%20is%20an%20efficient%20initiation%20codon%20to%20translate%20the%20human%20mitochondrial%20ATP6%20gene%2C&amp;author=A.%20Dubot&amp;author=C.%20Godinot&amp;author=V.%20Dumur&amp;publication_year=2004&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=313&amp;pages=687-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2003.12.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2003.12.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2003.12.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14697245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akao</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yashiro</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Hosoyama</div> <div class="given-names">A.</div></div></div> et al.   <div class="year">2011</div>, 
<div class="article-title">Whole-genome sequencing of sake yeast <em>Saccharomyces cerevisiae</em> Kyokai no. 7,</div> <div class="source">DNA Res</div> ., <div class="volume">18</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20sake%20yeast%20Saccharomyces%20cerevisiae%20Kyokai%20no.%207%2C&amp;author=T.%20Akao&amp;author=I.%20Yashiro&amp;author=A.%20Hosoyama&amp;publication_year=2011&amp;journal=DNA%20Res&amp;volume=18&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21169340" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marsico</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Scheubert</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tuukkanen</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">MeMotif: a database of linear motifs in α-helical transmembrane proteins,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">38</div>, <div class="fpage">D181</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MeMotif%3A%20a%20database%20of%20linear%20motifs%20in%20%CE%B1-helical%20transmembrane%20proteins%2C&amp;author=A.%20Marsico&amp;author=K.%20Scheubert&amp;author=A.%20Tuukkanen&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=D181-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp1042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp1042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp1042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Belfort</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">R.J.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Homing endonucleases: keeping the house in order,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">3379</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homing%20endonucleases%3A%20keeping%20the%20house%20in%20order%2C&amp;author=M.%20Belfort&amp;author=R.J.%20Roberts&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=3379-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stoddard</div> <div class="given-names">B.L.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Homing endonucleases: from microbial genetic invaders to reagents for targeted DNA modification,</div> <div class="source">Structure</div> , <div class="volume">19</div>, <div class="fpage">7</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homing%20endonucleases%3A%20from%20microbial%20genetic%20invaders%20to%20reagents%20for%20targeted%20DNA%20modification%2C&amp;author=B.L.%20Stoddard&amp;publication_year=2011&amp;journal=Structure&amp;volume=19&amp;pages=7-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.str.2010.12.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.str.2010.12.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.str.2010.12.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21220111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Watts</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Khalimonchuk</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">R.Z.</div></div>, <div class="name string-name"><div class="surname">Turk</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Mohr</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Winge</div> <div class="given-names">D.R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Mne1 is a novel component of the mitochondrial splicing apparatus responsible for processing of a COX1 group I intron in yeast,</div> <div class="source">J. Biol. Chem</div> ., <div class="volume">286</div>, <div class="fpage">10137</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mne1%20is%20a%20novel%20component%20of%20the%20mitochondrial%20splicing%20apparatus%20responsible%20for%20processing%20of%20a%20COX1%20group%20I%20intron%20in%20yeast%2C&amp;author=T.%20Watts&amp;author=O.%20Khalimonchuk&amp;author=R.Z.%20Wolf&amp;author=E.M.%20Turk&amp;author=G.%20Mohr&amp;author=D.R.%20Winge&amp;publication_year=2011&amp;journal=J.%20Biol.%20Chem&amp;volume=286&amp;pages=10137-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M110.205625" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M110.205625" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M110.205625"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21257754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Turk</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Das</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Seibert</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Andrulis</div> <div class="given-names">E.D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The mitochondrial RNA landscape of <em>Saccharomyces cerevisiae</em>,</div> <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">1</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mitochondrial%20RNA%20landscape%20of%20Saccharomyces%20cerevisiae%2C&amp;author=E.M.%20Turk&amp;author=V.%20Das&amp;author=R.D.%20Seibert&amp;author=E.D.%20Andrulis&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=1-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0078105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0078105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0078105"> </span></div> </div></div></div></div></div><div content-id="dsx026-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dunin-Horkawicz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Feder</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bujnicki</div> <div class="given-names">J.M.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Phylogenomic analysis of the GIY-YIG nuclease superfamily</div>, <div class="source">BMC Genomics</div> , <div class="volume">7</div>, <div class="fpage">1</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic%20analysis%20of%20the%20GIY-YIG%20nuclease%20superfamily&amp;author=S.%20Dunin-Horkawicz&amp;author=M.%20Feder&amp;author=J.M.%20Bujnicki&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=1-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-98" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-98" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-98"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16403227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cho</div> <div class="given-names">Y.R.</div></div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">J.D.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Multiple acquisitions via horizontal transfer of a group I intron in the mitochondrial cox1 gene during evolution of the Araceae family,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">16</div>, <div class="fpage">1155</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20acquisitions%20via%20horizontal%20transfer%20of%20a%20group%20I%20intron%20in%20the%20mitochondrial%20cox1%20gene%20during%20evolution%20of%20the%20Araceae%20family%2C&amp;author=Y.R.%20Cho&amp;author=J.D.%20Palmer&amp;publication_year=1999&amp;journal=Mol.%20Biol.%20Evol&amp;volume=16&amp;pages=1155-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10486971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Goddard</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Burt</div> <div class="given-names">A.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Recurrent invasion and extinction of a selfish gene,</div> <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">96</div>, <div class="fpage">13880</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recurrent%20invasion%20and%20extinction%20of%20a%20selfish%20gene%2C&amp;author=M.R.%20Goddard&amp;author=A.%20Burt&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=96&amp;pages=13880-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.24.13880" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.24.13880" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.24.13880"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Delahodde</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Goguel</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Becam</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Creusot</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Banroques</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jacq</div> <div class="given-names">C.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">Site specific DNA endonuclease and RNA maturase activities of two homologous intron encoded proteins from yeast mitochondria,</div> <div class="source">Cell</div> , <div class="volume">56</div>, <div class="fpage">431</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Site%20specific%20DNA%20endonuclease%20and%20RNA%20maturase%20activities%20of%20two%20homologous%20intron%20encoded%20proteins%20from%20yeast%20mitochondria%2C&amp;author=A.%20Delahodde&amp;author=V.%20Goguel&amp;author=A.M.%20Becam&amp;author=F.%20Creusot&amp;author=J.%20Banroques&amp;author=C.%20Jacq&amp;publication_year=1989&amp;journal=Cell&amp;volume=56&amp;pages=431-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(89)90246-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(89)90246-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(89)90246-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2536593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Petersen</div> <div class="given-names">R.F.</div></div>, <div class="name string-name"><div class="surname">Langkjar</div> <div class="given-names">R.B.</div></div>, <div class="name string-name"><div class="surname">Hvidtfeldt</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">Inheritance and organisation of the mitochondrial genome differ between two <em>Saccharomyces</em> yeasts,</div> <div class="source">J. Mol. Biol</div> ., <div class="volume">318</div>, <div class="fpage">627</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inheritance%20and%20organisation%20of%20the%20mitochondrial%20genome%20differ%20between%20two%20Saccharomyces%20yeasts%2C&amp;author=R.F.%20Petersen&amp;author=R.B.%20Langkjar&amp;author=J.%20Hvidtfeldt&amp;publication_year=2002&amp;journal=J.%20Mol.%20Biol&amp;volume=318&amp;pages=627-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(02)00037-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(02)00037-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(02)00037-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12054811" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Talla</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Anthouard</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Bouchier</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Frangeul</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Dujon</div> <div class="given-names">B.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">The complete mitochondrial genome of the yeast <em>Kluyveromyces thermotolerans</em>,</div> <div class="source">FEBS Lett</div> ., <div class="volume">579</div>, <div class="fpage">30</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20mitochondrial%20genome%20of%20the%20yeast%20Kluyveromyces%20thermotolerans%2C&amp;author=E.%20Talla&amp;author=V.%20Anthouard&amp;author=C.%20Bouchier&amp;author=L.%20Frangeul&amp;author=B.%20Dujon&amp;publication_year=2005&amp;journal=FEBS%20Lett&amp;volume=579&amp;pages=30-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2004.10.106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2004.10.106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2004.10.106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15620687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haugen</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Simon</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Bhattacharya</div> <div class="given-names">D.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">The natural history of group I introns,</div> <div class="source">Trends Genet</div> ., <div class="volume">21</div>, <div class="fpage">111</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20natural%20history%20of%20group%20I%20introns%2C&amp;author=P.%20Haugen&amp;author=D.M.%20Simon&amp;author=D.%20Bhattacharya&amp;publication_year=2005&amp;journal=Trends%20Genet&amp;volume=21&amp;pages=111-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.12.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.12.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2004.12.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15661357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stoddard</div> <div class="given-names">B.L.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Homing endonuclease structure &amp; function,</div> <div class="source">Q. Rev. Biophys</div> ., <div class="volume">38</div>, <div class="fpage">49</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homing%20endonuclease%20structure%20%26%20function%2C&amp;author=B.L.%20Stoddard&amp;publication_year=2006&amp;journal=Q.%20Rev.%20Biophys&amp;volume=38&amp;pages=49-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0033583505004063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0033583505004063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0033583505004063"> </span></div> </div></div></div></div></div><div content-id="dsx026-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edgell</div> <div class="given-names">D.R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Selfish DNA: homing endonucleases find a home,</div> <div class="source">Curr. Biol</div> ., <div class="volume">19</div>, <div class="fpage">R115</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selfish%20DNA%3A%20homing%20endonucleases%20find%20a%20home%2C&amp;author=D.R.%20Edgell&amp;publication_year=2009&amp;journal=Curr.%20Biol&amp;volume=19&amp;pages=R115-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2008.12.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2008.12.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2008.12.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19211047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Belfort</div> <div class="given-names">M.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Two for the price of one: a bifunctional intron-encoded DNA endonuclease-RNA maturase,</div> <div class="source">Genes Dev</div> ., <div class="volume">17</div>, <div class="fpage">2860</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20for%20the%20price%20of%20one%3A%20a%20bifunctional%20intron-encoded%20DNA%20endonuclease-RNA%20maturase%2C&amp;author=M.%20Belfort&amp;publication_year=2003&amp;journal=Genes%20Dev&amp;volume=17&amp;pages=2860-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1162503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1162503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1162503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14665667" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marchler-Bauer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">J.B.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">CDD: a Conserved Domain Database for the functional annotation of proteins,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">D225</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CDD%3A%20a%20Conserved%20Domain%20Database%20for%20the%20functional%20annotation%20of%20proteins%2C&amp;author=A.%20Marchler-Bauer&amp;author=S.%20Lu&amp;author=J.B.%20Anderson&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=D225-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21109532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fisk</div> <div class="given-names">D.G.</div></div>, <div class="name string-name"><div class="surname">Ball</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Dolinski</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">The <em>Saccharomyces</em> Genome Database Project: <em>Saccharomyces cerevisiae</em> S288C genome annotation: a working hypothesis,</div> <div class="source">Yeast</div> , <div class="volume">23</div>, <div class="fpage">857</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Saccharomyces%20Genome%20Database%20Project%3A%20Saccharomyces%20cerevisiae%20S288C%20genome%20annotation%3A%20a%20working%20hypothesis%2C&amp;author=D.G.%20Fisk&amp;author=C.A.%20Ball&amp;author=K.%20Dolinski&amp;publication_year=2006&amp;journal=Yeast&amp;volume=23&amp;pages=857-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17001629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Cohen</div> <div class="given-names">P.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">MOTS-c: a novel mitochondrial-derived peptide regulating muscle and fat metabolism</div>, <div class="source">Free Radical Biol. Med</div> ., <div class="volume">100</div>, <div class="fpage">1</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MOTS-c%3A%20a%20novel%20mitochondrial-derived%20peptide%20regulating%20muscle%20and%20fat%20metabolism&amp;author=C.%20Lee&amp;author=K.H.%20Kim&amp;author=P.%20Cohen&amp;publication_year=2016&amp;journal=Free%20Radical%20Biol.%20Med&amp;volume=100&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.freeradbiomed.2016.05.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.freeradbiomed.2016.05.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.freeradbiomed.2016.05.015"> </span></div> </div></div></div></div></div><div content-id="dsx026-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Drew</div> <div class="given-names">B.G.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The mitochondrial-derived peptide MOTS-c promotes metabolic homeostasis and reduces obesity and insulin resistance,</div> <div class="source">Cell Metab</div> ., <div class="volume">21</div>, <div class="fpage">443</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mitochondrial-derived%20peptide%20MOTS-c%20promotes%20metabolic%20homeostasis%20and%20reduces%20obesity%20and%20insulin%20resistance%2C&amp;author=C.%20Lee&amp;author=J.%20Zeng&amp;author=B.G.%20Drew&amp;publication_year=2015&amp;journal=Cell%20Metab&amp;volume=21&amp;pages=443-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cmet.2015.02.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cmet.2015.02.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cmet.2015.02.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25738459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lecrenier</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Foury</div> <div class="given-names">F.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">New features of mitochondrial DNA replication system in yeast and man,</div> <div class="source">Gene</div> , <div class="volume">246</div>, <div class="fpage">37</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20features%20of%20mitochondrial%20DNA%20replication%20system%20in%20yeast%20and%20man%2C&amp;author=N.%20Lecrenier&amp;author=F.%20Foury&amp;publication_year=2000&amp;journal=Gene&amp;volume=246&amp;pages=37-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(00)00107-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(00)00107-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(00)00107-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10767525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weiller</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Schueller</div> <div class="given-names">C.M.E.</div></div>, <div class="name string-name"><div class="surname">Schweyen</div> <div class="given-names">R.J.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">Putative target sites for mobile G + C rich clusters in yeast mitochondrial DNA: single elements and tandem arrays,</div> <div class="source">Mol. Gen. Genet</div> ., <div class="volume">218</div>, <div class="fpage">272</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Putative%20target%20sites%20for%20mobile%20G%20%2B%20C%20rich%20clusters%20in%20yeast%20mitochondrial%20DNA%3A%20single%20elements%20and%20tandem%20arrays%2C&amp;author=G.%20Weiller&amp;author=C.M.E.%20Schueller&amp;author=R.J.%20Schweyen&amp;publication_year=1989&amp;journal=Mol.%20Gen.%20Genet&amp;volume=218&amp;pages=272-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00331278" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00331278" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00331278"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2674655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Zamaroczy</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div></div> <div class="year">1986a</div>, 
<div class="article-title">The GC clusters of the mitochondrial genome of yeast and their evolutionary origin</div>, <div class="source">Gene</div> , <div class="volume">41</div>, <div class="fpage">1</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20GC%20clusters%20of%20the%20mitochondrial%20genome%20of%20yeast%20and%20their%20evolutionary%20origin&amp;author=M.%20de%20Zamaroczy&amp;author=G.%20Bernardi&amp;publication_year=1986a&amp;journal=Gene&amp;volume=41&amp;pages=1-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(86)90262-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(86)90262-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(86)90262-3"> </span></div> </div></div></div></div></div><div content-id="dsx026-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Christianson</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Levens</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Locker</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Rabinowitz</div> <div class="given-names">M.</div></div></div> <div class="year">1982</div>, 
<div class="article-title">Transcriptional initiation and processing of the small ribosomal RNA of yeast mitochondria</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">257</div>, <div class="fpage">6494</div>–<div class="lpage">500</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20initiation%20and%20processing%20of%20the%20small%20ribosomal%20RNA%20of%20yeast%20mitochondria&amp;author=T.%20Christianson&amp;author=J.%20Edwards&amp;author=D.%20Levens&amp;author=J.%20Locker&amp;author=M.%20Rabinowitz&amp;publication_year=1982&amp;journal=J.%20Biol.%20Chem&amp;volume=257&amp;pages=6494-500" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6281273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Osinga</div> <div class="given-names">K.A.</div></div>, <div class="name string-name"><div class="surname">De Vries</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Van der Horst</div> <div class="given-names">G.T.</div></div>, <div class="name string-name"><div class="surname">Tabak</div> <div class="given-names">H.F.</div></div></div> <div class="year">1984a</div>, 
<div class="article-title">Initiation of transcription in yeast mitochondria: analysis of origins of replication and of genes coding for a messenger RNA and a transfer RNA</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">12</div>, <div class="fpage">1889</div>–<div class="lpage">900</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initiation%20of%20transcription%20in%20yeast%20mitochondria%3A%20analysis%20of%20origins%20of%20replication%20and%20of%20genes%20coding%20for%20a%20messenger%20RNA%20and%20a%20transfer%20RNA&amp;author=K.A.%20Osinga&amp;author=E.%20De%20Vries&amp;author=G.T.%20Van%20der%20Horst&amp;author=H.F.%20Tabak&amp;publication_year=1984a&amp;journal=Nucleic%20Acids%20Res&amp;volume=12&amp;pages=1889-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.4.1889" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.4.1889" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.4.1889"> </span></div> </div></div></div></div></div><div content-id="dsx026-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schäfer</div> <div class="given-names">B.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">RNA maturation in mitochondria of <em>S. cerevisiae</em> and <em>S. pombe</em>,</div> <div class="source">Gene</div> , <div class="volume">354</div>, <div class="fpage">80</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20maturation%20in%20mitochondria%20of%20S.%20cerevisiae%20and%20S.%20pombe%2C&amp;author=B.%20Sch%C3%A4fer&amp;publication_year=2005&amp;journal=Gene&amp;volume=354&amp;pages=80-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.03.032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.03.032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.03.032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15913924" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Osinga</div> <div class="given-names">K.A.</div></div>, <div class="name string-name"><div class="surname">De Vries</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Van der Horst</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Tabak</div> <div class="given-names">H.F.</div></div></div> <div class="year">1984b</div>, 
<div class="article-title">Processing of yeast mitochondrial messenger RNAs at a conserved dodecamer sequence</div>, <div class="source">EMBO J</div> ., <div class="volume">3</div>, <div class="fpage">829</div>–<div class="lpage">34</div>.</div></div></div></div><div content-id="dsx026-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hofmann</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Min</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zassenhaus</div> <div class="given-names">H.P.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Formation of the 3′ end of yeast mitochondrial mRNAs occurs by site-specific cleavage two bases downstream of a conserved dodecamer sequence,</div> <div class="source">Yeast</div> , <div class="volume">9</div>, <div class="fpage">1319</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Formation%20of%20the%203%E2%80%B2%20end%20of%20yeast%20mitochondrial%20mRNAs%20occurs%20by%20site-specific%20cleavage%20two%20bases%20downstream%20of%20a%20conserved%20dodecamer%20sequence%2C&amp;author=T.J.%20Hofmann&amp;author=J.%20Min&amp;author=H.P.%20Zassenhaus&amp;publication_year=1993&amp;journal=Yeast&amp;volume=9&amp;pages=1319-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7512302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Freel</div> <div class="given-names">K.C.</div></div>, <div class="name string-name"><div class="surname">Friedrich</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hou</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Schacherer</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Population genomic analysis reveals highly conserved mitochondrial genomes in the yeast species <em>Lachancea thermotolerans</em>,</div> <div class="source">Genome Biol. Evol</div> ., <div class="volume">6</div>, <div class="fpage">2586</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20genomic%20analysis%20reveals%20highly%20conserved%20mitochondrial%20genomes%20in%20the%20yeast%20species%20Lachancea%20thermotolerans%2C&amp;author=K.C.%20Freel&amp;author=A.%20Friedrich&amp;author=J.%20Hou&amp;author=J.%20Schacherer&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol&amp;volume=6&amp;pages=2586-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25212859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Buljic</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hao</div> <div class="given-names">W.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Extensive horizontal transfer and homologous recombination generate highly chimeric mitochondrial genomes in yeast,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">2559</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20horizontal%20transfer%20and%20homologous%20recombination%20generate%20highly%20chimeric%20mitochondrial%20genomes%20in%20yeast%2C&amp;author=B.%20Wu&amp;author=A.%20Buljic&amp;author=W.%20Hao&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=2559-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msv127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsv127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsv127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26018571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gaillardin</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Neuveglise</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kerscher</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nicaud</div> <div class="given-names">J.M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Mitochondrial genomes of yeasts of the <em>Yarrowia</em> clade,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">12</div>, <div class="fpage">317</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20genomes%20of%20yeasts%20of%20the%20Yarrowia%20clade%2C&amp;author=C.%20Gaillardin&amp;author=C.%20Neuveglise&amp;author=S.%20Kerscher&amp;author=J.M.%20Nicaud&amp;publication_year=2012&amp;journal=FEMS%20Yeast%20Res&amp;volume=12&amp;pages=317-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2012.12.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2012.12.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2012.12.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22188421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bouchier</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Creno</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Dujon</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Fairhead</div> <div class="given-names">C.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Complete mitochondrial genome sequences of three <em>Nakaseomyces</em> species reveal invasion by palindromic GC clusters and considerable size expansion,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">9</div>, <div class="fpage">1283</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20mitochondrial%20genome%20sequences%20of%20three%20Nakaseomyces%20species%20reveal%20invasion%20by%20palindromic%20GC%20clusters%20and%20considerable%20size%20expansion%2C&amp;author=C.%20Bouchier&amp;author=L.%20Ma&amp;author=S.%20Creno&amp;author=B.%20Dujon&amp;author=C.%20Fairhead&amp;publication_year=2009&amp;journal=FEMS%20Yeast%20Res&amp;volume=9&amp;pages=1283-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2009.9.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2009.9.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2009.9.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19758332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Procházka</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Poláková</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Piškur</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sulo</div> <div class="given-names">P.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Mitochondrial genome from the facultative anaerobe and petite-positive yeast <em>Dekkera bruxellensis</em> contains the NADH dehydrogenase subunit genes</div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">10</div>, <div class="fpage">545</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20genome%20from%20the%20facultative%20anaerobe%20and%20petite-positive%20yeast%20Dekkera%20bruxellensis%20contains%20the%20NADH%20dehydrogenase%20subunit%20genes&amp;author=E.%20Proch%C3%A1zka&amp;author=S.%20Pol%C3%A1kov%C3%A1&amp;author=J.%20Pi%C5%A1kur&amp;author=P.%20Sulo&amp;publication_year=2010&amp;journal=FEMS%20Yeast%20Res&amp;volume=10&amp;pages=545-57" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20528950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clark-Walker</div> <div class="given-names">G.D.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Evolution of mitochondrial genomes in fungi,</div> <div class="source">Int. Rev. Cytol</div> ., <div class="volume">141</div>, <div class="fpage">89</div>–<div class="lpage">127</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20mitochondrial%20genomes%20in%20fungi%2C&amp;author=G.D.%20Clark-Walker&amp;publication_year=1992&amp;journal=Int.%20Rev.%20Cytol&amp;volume=141&amp;pages=89-127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0074-7696(08)62064-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0074-7696(08)62064-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0074-7696(08)62064-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1452434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hittinger</div> <div class="given-names">C.T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title"><em>Saccharomyces</em> diversity and evolution: a budding model genus,</div> <div class="source">Trends Genet</div> ., <div class="volume">29</div>, <div class="fpage">309</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saccharomyces%20diversity%20and%20evolution%3A%20a%20budding%20model%20genus%2C&amp;author=C.T.%20Hittinger&amp;publication_year=2013&amp;journal=Trends%20Genet&amp;volume=29&amp;pages=309-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2013.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2013.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2013.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23395329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kurtzman</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Mateo</div> <div class="given-names">R.Q.</div></div>, <div class="name string-name"><div class="surname">Kolecka</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Theelen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Robert</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Boekhout</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Advances in yeast systematics and phylogeny and their use as predictors of biotechnologically important metabolic pathways,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">15</div>, <div class="fpage">1</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20yeast%20systematics%20and%20phylogeny%20and%20their%20use%20as%20predictors%20of%20biotechnologically%20important%20metabolic%20pathways%2C&amp;author=C.P.%20Kurtzman&amp;author=R.Q.%20Mateo&amp;author=A.%20Kolecka&amp;author=B.%20Theelen&amp;author=V.%20Robert&amp;author=T.%20Boekhout&amp;publication_year=2015&amp;journal=FEMS%20Yeast%20Res&amp;volume=15&amp;pages=1-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/femsyr/fov050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ffemsyr%2Ffov050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ffemsyr%2Ffov050"> </span></div> </div></div></div></div></div><div content-id="dsx026-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peris</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Langdon</div> <div class="given-names">Q.K.</div></div>, <div class="name string-name"><div class="surname">Moriarty</div> <div class="given-names">R.V.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Complex ancestries of lager-brewing hybrids were shaped by standing variation in the wild yeast <em>Saccharomyces eubayanus</em>,</div> <div class="source">PLoS Genet</div> ., <div class="volume">12</div>, <div class="fpage">1</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20ancestries%20of%20lager-brewing%20hybrids%20were%20shaped%20by%20standing%20variation%20in%20the%20wild%20yeast%20Saccharomyces%20eubayanus%2C&amp;author=D.%20Peris&amp;author=Q.K.%20Langdon&amp;author=R.V.%20Moriarty&amp;publication_year=2016&amp;journal=PLoS%20Genet&amp;volume=12&amp;pages=1-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1006155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1006155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1006155"> </span></div> </div></div></div></div></div><div content-id="dsx026-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B89" class="ref-content "><div class="label -label">89</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rainieri</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nakao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Pulvirenti</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Giudici</div> <div class="given-names">P.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The inheritance of mtDNA in lager brewing strains,</div> <div class="source">FEMS Yeast Res</div> ., <div class="volume">8</div>, <div class="fpage">586</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20inheritance%20of%20mtDNA%20in%20lager%20brewing%20strains%2C&amp;author=S.%20Rainieri&amp;author=Y.%20Kodama&amp;author=Y.%20Nakao&amp;author=A.%20Pulvirenti&amp;author=P.%20Giudici&amp;publication_year=2008&amp;journal=FEMS%20Yeast%20Res&amp;volume=8&amp;pages=586-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2008.8.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2008.8.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2008.8.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18318709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx026-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B90" class="ref-content "><div class="label -label">90</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Almeida</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Goncalves</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Teixeira</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A Gondwanan imprint on global diversity and domestication of wine and cider yeast <em>Saccharomyces uvarum</em>,</div> <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.</div></div></div></div><div content-id="dsx026-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx026-B91" class="ref-content "><div class="label -label">91</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peris</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Sylvester</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Libkind</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gonçalves</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Sampaio</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Alexander</div> <div class="given-names">W.G.</div></div>, <div class="name string-name"><div class="surname">Hittinger</div> <div class="given-names">C.T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Population structure and reticulate evolution of <em>Saccharomyces eubayanus</em> and its lager-brewing hybrids</div>. <div class="source">Mol. Ecol</div> ., <div class="volume">23</div>, <div class="fpage">2031</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20reticulate%20evolution%20of%20Saccharomyces%20eubayanus%20and%20its%20lager-brewing%20hybrids&amp;author=D.%20Peris&amp;author=K.%20Sylvester&amp;author=D.%20Libkind&amp;author=P.%20Gon%C3%A7alves&amp;author=J.P.%20Sampaio&amp;author=W.G.%20Alexander&amp;author=C.T.%20Hittinger&amp;publication_year=2014&amp;journal=Mol.%20Ecol&amp;volume=23&amp;pages=2031-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24612382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3868562&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s1_dsx026.doc?Expires=2147483647&amp;Signature=1NPSjXvfqnbox4N9CWZa9hD5xZslFbIbvGRK9RiHRCHuTDA0~7c45lMVSPIQVUyE-9pxJ9KouMUJRNuAdcb~s58URh14gec-kzVbmpogVcvMZrFJBiVK3Rx~rPWQcTGZGLMF389vonrMTo0M9cmx8RYAUvlJfKSGPmRGNnegIwtd-rRa62NykzkhGS0tac0Npt78Imuf6obxyN6eGc5xqsXXRNMJZFVqAE-WmIaRjTgKa53kL-BROzLKfIzhpBn4Q02K10aVLSgZx8NdVeM0PA7nU635v56YiLJMbtr4L082m9xGc2mX~xQe1-EyjoFE5ulcE3hfKiiVGTJ6YiTzsw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Material S1</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s2_dsx026.doc?Expires=2147483647&amp;Signature=ehiRCfNIbPq~VXTo~lSqSGVcDaeRplhM3a3SmP28ip9fDy-A0-mrPi9Ypmuz-iKfRypxmYDUWqThlez9Ij2pqSukHFxGNq~C1hgWyQeUY80mRthyyr5S3OZtu9EhVRGkc-IMROs8xqIxXBdaLOJzWEsOp-sF9aznVIklnPFogNi2os~WZGWYrW3BN~Ve6dqqU-iG7aN8Y5noLaOgKWpfXVmH3oA~gCeBr7X-RucYUX1AFoDnt~U1q9SZvXZhTT9Nogwqt7QkDO8lJDmp07Crp19EN8W4J0pLxzTcTCzYesW-L8g9Bjeixt39XzGwbLysDskyGehUs2i7qu4cgTSWbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Material S2</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s3_dsx026.doc?Expires=2147483647&amp;Signature=TFW~qL1Ee~QMx71bAy6HPjAH51PXFbVQ3W8ZdK8BcWqmkg-AD5o7NZZVvXoM9TE7GopgONuZ2wow9~KnThfmItmx6DxUlJI3OSKAAd65GJfn2My0I182T4E-FfVeanwbM1hV0k7ewhIqe6bdRYzgP151G6E1Whfw5irck3d-kCn6DduHhCMAcJIBauAgTXXGm5icSZya9nvHF2S03vMgQEw3RDamNsgEIxHXplFaBwkBCBSxY6TDp3y7OXjUUlRLYRf5V~JyTJtZ5WyDFDcI3jKBCbaBmF3KphJ~MxlsGp-uVFsHbMmIoEcqrVSNr-oP6vxPfsvirNkugtzx0x0LrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Material S3</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx026/1/s4_dsx026.doc?Expires=2147483647&amp;Signature=rbwTZpfLkW2CL4OMw5zLVsSd9xNAHVsKG97eJfkOyG8gQAIp3jHp-zLajSwyMn7GcUleX5t7mcVeJA7C9Wobila7rNkv8J3xp--8Q~6y~7Kd0MyYKO-MNilvJfpIHbLbJhce8784JuWp0WQIhq21eHoAk4BBV--ncDxCt5-b7RgaV8GtNWzgT4~Ol6TX6PTWZJEGSZkDy0oF6xMi5YXp4Vudg4DGDS15XCRjU5yIBnIw~1Gn2K-cuxfY17l68m1k1PIihDNuDDa3Lqp9ivuYheGL8JN-HCJF2Yt0-pKylP3fseNlBp5QFIgbFnkrBt0-xPh82dKrE2MvvN6QJ5uBbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Material S4</div></a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">907</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx026"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3868562"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f6%2f571%2f3868562">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000418420400003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F6%2F571%2F3868562" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3868562'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="evolutionary history of Saccharomyces species inferred from completed mitochondrial genomes and revision in the ‘yeast mitochondrial genetic code’" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx026");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f571%2f3868562">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

