

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/3/289/2960056","@type":"ScholarlyArticle","name":"Integrated high-density consensus genetic map of  Pyrus  and anchoring of the ‘Bartlett’ v1.0 ( Pyrus communis ) genome","datePublished":"2017-01-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2017-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw063","keywords":["pear","genetic map","genome assembly","SNP","SSR"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/3/289/2960056","author":[{"name":"Li, Leiting","affiliation":"1 State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China","@type":"Person"},{"name":"Deng, Cecilia H.","affiliation":"2 The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand","@type":"Person"},{"name":"Knäbel, Mareike","affiliation":"2 The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand","@type":"Person"},{"name":"Chagné, David","affiliation":"2 The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand","@type":"Person"},{"name":"Kumar, Satish","affiliation":"2 The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand","@type":"Person"},{"name":"Sun, Jiangmei","affiliation":"1 State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China","@type":"Person"},{"name":"Zhang, Shaoling","affiliation":"1 State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China","@type":"Person"},{"name":"Wu, Jun","affiliation":"1 State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China","@type":"Person"}],"description":"Abstract.  Genetic maps are essential tools for pear genetics and genomics research. In this study, we first constructed an integrated simple sequence repeat (S","pageStart":"289","pageEnd":"301","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Integrated high-density consensus genetic map of  Pyrus  and anchoring of the ‘Bartlett’ v1.0 ( Pyrus communis ) genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Integrated high-density consensus genetic map of  Pyrus  and anchoring of the ‘Bartlett’ v1.0 ( Pyrus communis ) genome" />
    <meta property="og:description" content="Abstract.  Genetic maps are essential tools for pear genetics and genomics research. In this study, we first constructed an integrated simple sequence repeat (S" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/3/289/2960056" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-01-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Leiting" /><meta name="citation_author_institution" content="State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China" /><meta name="citation_author" content="Deng, Cecilia H." /><meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand" /><meta name="citation_author" content="Knäbel, Mareike" /><meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand" /><meta name="citation_author" content="Chagné, David" /><meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand" /><meta name="citation_author" content="Kumar, Satish" /><meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand" /><meta name="citation_author" content="Sun, Jiangmei" /><meta name="citation_author_institution" content="State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China" /><meta name="citation_author" content="Zhang, Shaoling" /><meta name="citation_author_institution" content="State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China" /><meta name="citation_author" content="Wu, Jun" /><meta name="citation_author_institution" content="State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China" /><meta name="citation_title" content="Integrated high-density consensus genetic map of Pyrus and anchoring of the ‘Bartlett’ v1.0 (Pyrus communis) genome" /><meta name="citation_firstpage" content="289" /><meta name="citation_lastpage" content="301" /><meta name="citation_doi" content="10.1093/dnares/dsw063" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2017/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Crop breeding for salt tolerance in the era of molecular markers and marker-assisted selection;  citation_journal_title=Plant Breed;  citation_year=2013;  citation_volume=132;  citation_pages=10-20; " /><meta name="citation_reference" content="citation_title=Genetic diversity and population structure of pear (Pyrus spp.) collections revealed by a set of core genome-wide SSR markers;  citation_journal_title=Tree Genet. Genomes;  citation_year=2015;  citation_volume=11; " /><meta name="citation_reference" content="citation_title=Identifying genetic diversity and a preliminary core collection of Pyrus pyrifolia cultivars by a genome-wide set of SSR markers;  citation_journal_title=Sci. Horticult;  citation_year=2014;  citation_volume=167;  citation_pages=5-16; " /><meta name="citation_reference" content="citation_title=A novel set of EST-derived SSR markers for pear and cross-species transferability in Rosaceae;  citation_journal_title=Plant Mol. Biol. Rep;  citation_year=2014;  citation_volume=32;  citation_pages=290-302; " /><meta name="citation_reference" content="citation_title=Genetic control of pear rootstock-induced dwarfing and precocity is linked to a chromosomal region syntenic to the apple Dw1 loci;  citation_journal_title=BMC Plant Biol;  citation_year=2015;  citation_volume=15;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Genetic mapping of Cacopsylla pyri resistance in an interspecific pear (Pyrus spp.) population;  citation_journal_title=Tree Genet. Genomes;  citation_year=2015;  citation_volume=11;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Genome mapping of postzygotic hybrid necrosis in an interspecific pear population;  citation_journal_title=Hortic. Res;  citation_year=2016;  citation_volume=3;  citation_pages=15064" /><meta name="citation_reference" content="citation_title=Genetic, metabolite and developmental determinism of fruit friction discolouration in pear;  citation_journal_title=BMC Plant Biol;  citation_year=2014;  citation_volume=14;  citation_pages=241" /><meta name="citation_reference" content="citation_title=Genetic mapping of polygenic scab (Venturia pirina) resistance in an interspecific pear family;  citation_journal_title=Mol. Breed;  citation_year=2014;  citation_volume=34;  citation_pages=2179-2189; " /><meta name="citation_reference" content="citation_title=High-density genetic linkage map construction and identification of fruit-related QTLs in pear using SNP and SSR markers;  citation_journal_title=J Exp. Bot;  citation_year=2014;  citation_volume=65;  citation_pages=5771-5781; " /><meta name="citation_reference" content="citation_title=An AFLP, SRAP, and SSR genetic linkage map and identification of QTLs for fruit traits in pear (Pyrus L.);  citation_journal_title=Plant Mol. Biol. Rep;  citation_year=2013;  citation_volume=31;  citation_pages=678-687; " /><meta name="citation_reference" content="citation_title=Fine mapping of the gene for susceptibility to black spot disease in Japanese pear (Pyrus pyrifolia Nakai);  citation_journal_title=Breed. Sci;  citation_year=2016;  citation_volume=66;  citation_pages=271" /><meta name="citation_reference" content="citation_title=Mapping of disease-related genes in japanese pear using a molecular linkage map with RAPD markers;  citation_journal_title=Breed. Sci;  citation_year=2001;  citation_volume=51;  citation_pages=179-184; " /><meta name="citation_reference" content="citation_title=Genetic linkage maps constructed by using an interspecific cross between Japanese and European pears;  citation_journal_title=Theor. Appl. Genet;  citation_year=2002;  citation_volume=106;  citation_pages=9-18; " /><meta name="citation_reference" content="citation_title=Identifying QTLs for fire-blight resistance via a European pear (Pyrus communis L.) genetic linkage map;  citation_journal_title=Mol. Breed;  citation_year=2005;  citation_volume=14;  citation_pages=407-418; " /><meta name="citation_reference" content="citation_title=Integrated reference genetic linkage maps of pear based on SSR and AFLP markers;  citation_journal_title=Breed. Sci;  citation_year=2007;  citation_volume=57;  citation_pages=321-329; " /><meta name="citation_reference" content="citation_title=Genetic linkage map of the Japanese pear ‘Housui’ identifying three homozygous genomic regions;  citation_journal_title=J. Jpn. Soc. Hortic. Sci;  citation_year=2009;  citation_volume=78;  citation_pages=417-424; " /><meta name="citation_reference" content="citation_title=Transferability of newly developed pear SSR markers to other Rosaceae species;  citation_journal_title=Plant Mol. Biol. Rep;  citation_year=2013;  citation_volume=31;  citation_pages=1271-1282; " /><meta name="citation_reference" content="citation_title=Construction of a high-density simple sequence repeat consensus genetic map for pear (Pyrus spp.);  citation_journal_title=Plant Mol. Biol. Rep;  citation_year=2015;  citation_volume=33;  citation_pages=316-325; " /><meta name="citation_reference" content="citation_title=DNA markers developed from genome sequencing analysis in Japanese pear (Pyrus pyrifolia);  citation_journal_title=Acta Hortic;  citation_year=2013;  citation_volume=976;  citation_pages=477-483; " /><meta name="citation_reference" content="citation_title=Identification of QTLs controlling harvest time and fruit skin color in Japanese pear (Pyrus pyrifolia Nakai);  citation_journal_title=Breed. Sci;  citation_year=2014;  citation_volume=64;  citation_pages=351-361; " /><meta name="citation_reference" content="citation_title=The genome of the pear (Pyrus bretschneideri Rehd.);  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=396-408; " /><meta name="citation_reference" content="citation_title=Identification of Pyrus single nucleotide polymorphisms (SNPs) and evaluation for genetic mapping in European pear and interspecific Pyrus hybrids;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e77022" /><meta name="citation_reference" content="citation_title=Transcriptome-based single nucleotide polymorphism markers for genome mapping in Japanese pear (Pyrus pyrifolia Nakai);  citation_journal_title=Tree Genet. Genomes;  citation_year=2014;  citation_volume=10;  citation_pages=853-863; " /><meta name="citation_reference" content="citation_title=ALLMAPS: robust scaffold ordering based on multiple maps;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=3" /><meta name="citation_reference" content="citation_title=The draft genome sequence of European pear (Pyrus communis L. ‘Bartlett’);  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e92644" /><meta name="citation_reference" content="citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e19379" /><meta name="citation_reference" content="citation_title=Genomic plant DNA preparation from fresh tissue-CTAB method;  citation_journal_title=Phytochem. Bull;  citation_year=1987;  citation_volume=19;  citation_pages=11-15; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis reveals novel features of the molecular events occurring in the laticifers of Hevea brasiliensis (para rubber tree);  citation_journal_title=Plant Mol. Biol;  citation_year=2003;  citation_volume=53;  citation_pages=479-492; " /><meta name="citation_reference" content="citation_title=FastQC, Available: http://sourceforge.net/projects/picard/files/picard-tools/, 31 December 2016, date last accessed;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=TASSEL: software for association mapping of complex traits in diverse samples;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2633-2635; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-359; " /><meta name="citation_reference" content="citation_title=Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species;  citation_journal_title=Theor. Popul. Biol;  citation_year=2002;  citation_volume=61;  citation_pages=349-363; " /><meta name="citation_reference" content="citation_title=OneMap: software for genetic mapping in outcrossing species;  citation_journal_title=Hereditas;  citation_year=2007;  citation_volume=144;  citation_pages=78-79; " /><meta name="citation_reference" content="citation_title=Accurate construction of consensus genetic maps via integer linear programming;  citation_journal_title=IEEE/ACM Trans. Comput. Biol;  citation_year=2011;  citation_volume=8;  citation_pages=381-394; " /><meta name="citation_reference" content="citation_title=Comparative visualization of genetic and physical maps with Strudel;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=1307-1308; " /><meta name="citation_reference" content="citation_title=BLAT – the BLAST-like alignment tool;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=656-664; " /><meta name="citation_reference" content="citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e49" /><meta name="citation_reference" content="citation_title=Integrated consensus map of cultivated peanut and wild relatives reveals structures of the A and B genomes of Arachis and divergence of the legume genomes;  citation_journal_title=DNA Res;  citation_year=2013;  citation_volume=20;  citation_pages=173-184; " /><meta name="citation_reference" content="citation_title=The genome of the domesticated apple (Malus × domestica Borkh.);  citation_journal_title=Nat. Genet;  citation_year=2010;  citation_volume=42;  citation_pages=833-839; " /><meta name="citation_reference" content="citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers;  citation_journal_title=PLoS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e58700" /><meta name="citation_reference" content="citation_title=A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7; " /><meta name="citation_reference" content="citation_title=Hierarchical scaffolding with Bambus;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=149-159; " /><meta name="citation_reference" content="citation_title=Progress, challenges and the future of crop genomes;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2015;  citation_volume=24;  citation_pages=71-81; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/3/289/17743946/dsw063.pdf" /><meta name="description" content="Abstract.  Genetic maps are essential tools for pear genetics and genomics research. In this study, we first constructed an integrated simple sequence repeat (S" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/3/289/2960056" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/3/289/2960056" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/3/289/2960056");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw063';
    </script>
<title>
	Integrated high-density consensus genetic map of Pyrus and anchoring of the ‘Bartlett’ v1.0 (Pyrus communis) genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2960056" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbdxPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaSwgTGVpdGluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpY3MgYW5kIEdlcm1wbGFzbSBFbmhhbmNlbWVudCwgQ2VudHJlIG9mIFBlYXIgRW5naW5lZXJpbmcgVGVjaG5vbG9neSBSZXNlYXJjaCwgTmFuamluZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgTmFuamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEZW5nLCBDZWNpbGlhIEguIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGhlIE5ldyBaZWFsYW5kIEluc3RpdHV0ZSBmb3IgUGxhbnQgJmFtcDsgRm9vZCBSZXNlYXJjaCBMaW1pdGVkIChQbGFudCAmYW1wOyBGb29kIFJlc2VhcmNoKSwgTmV3IFplYWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLbsOkYmVsLCBNYXJlaWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGhlIE5ldyBaZWFsYW5kIEluc3RpdHV0ZSBmb3IgUGxhbnQgJmFtcDsgRm9vZCBSZXNlYXJjaCBMaW1pdGVkIChQbGFudCAmYW1wOyBGb29kIFJlc2VhcmNoKSwgTmV3IFplYWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGFnbsOpLCBEYXZpZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBOZXcgWmVhbGFuZCBJbnN0aXR1dGUgZm9yIFBsYW50ICZhbXA7IEZvb2QgUmVzZWFyY2ggTGltaXRlZCAoUGxhbnQgJmFtcDsgRm9vZCBSZXNlYXJjaCksIE5ldyBaZWFsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VtYXIsIFNhdGlzaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBOZXcgWmVhbGFuZCBJbnN0aXR1dGUgZm9yIFBsYW50ICZhbXA7IEZvb2QgUmVzZWFyY2ggTGltaXRlZCAoUGxhbnQgJmFtcDsgRm9vZCBSZXNlYXJjaCksIE5ldyBaZWFsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3VuLCBKaWFuZ21laSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpY3MgYW5kIEdlcm1wbGFzbSBFbmhhbmNlbWVudCwgQ2VudHJlIG9mIFBlYXIgRW5naW5lZXJpbmcgVGVjaG5vbG9neSBSZXNlYXJjaCwgTmFuamluZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgTmFuamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgU2hhb2xpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBDcm9wIEdlbmV0aWNzIGFuZCBHZXJtcGxhc20gRW5oYW5jZW1lbnQsIENlbnRyZSBvZiBQZWFyIEVuZ2luZWVyaW5nIFRlY2hub2xvZ3kgUmVzZWFyY2gsIE5hbmppbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIE5hbmppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV3UsIEp1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpY3MgYW5kIEdlcm1wbGFzbSBFbmhhbmNlbWVudCwgQ2VudHJlIG9mIFBlYXIgRW5naW5lZXJpbmcgVGVjaG5vbG9neSBSZXNlYXJjaCwgTmFuamluZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgTmFuamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkludGVncmF0ZWQgaGlnaC1kZW5zaXR5IGNvbnNlbnN1cyBnZW5ldGljIG1hcCBvZiBQeXJ1cyBhbmQgYW5jaG9yaW5nIG9mIHRoZSDigJhCYXJ0bGV0dOKAmSB2MS4wIChQeXJ1cyBjb21tdW5pcykgZ2Vub21lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjg5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyb3AgYnJlZWRpbmcgZm9yIHNhbHQgdG9sZXJhbmNlIGluIHRoZSBlcmEgb2YgbW9sZWN1bGFyIG1hcmtlcnMgYW5kIG1hcmtlci1hc3Npc3RlZCBzZWxlY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJyZWVkOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTEwLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBhbmQgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgb2YgcGVhciAoUHlydXMgc3BwLikgY29sbGVjdGlvbnMgcmV2ZWFsZWQgYnkgYSBzZXQgb2YgY29yZSBnZW5vbWUtd2lkZSBTU1IgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgZ2VuZXRpYyBkaXZlcnNpdHkgYW5kIGEgcHJlbGltaW5hcnkgY29yZSBjb2xsZWN0aW9uIG9mIFB5cnVzIHB5cmlmb2xpYSBjdWx0aXZhcnMgYnkgYSBnZW5vbWUtd2lkZSBzZXQgb2YgU1NSIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gSG9ydGljdWx0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE2NzsgIGNpdGF0aW9uX3BhZ2VzPTUtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgc2V0IG9mIEVTVC1kZXJpdmVkIFNTUiBtYXJrZXJzIGZvciBwZWFyIGFuZCBjcm9zcy1zcGVjaWVzIHRyYW5zZmVyYWJpbGl0eSBpbiBSb3NhY2VhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0yOTAtMzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGNvbnRyb2wgb2YgcGVhciByb290c3RvY2staW5kdWNlZCBkd2FyZmluZyBhbmQgcHJlY29jaXR5IGlzIGxpbmtlZCB0byBhIGNocm9tb3NvbWFsIHJlZ2lvbiBzeW50ZW5pYyB0byB0aGUgYXBwbGUgRHcxIGxvY2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWFwcGluZyBvZiBDYWNvcHN5bGxhIHB5cmkgcmVzaXN0YW5jZSBpbiBhbiBpbnRlcnNwZWNpZmljIHBlYXIgKFB5cnVzIHNwcC4pIHBvcHVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgbWFwcGluZyBvZiBwb3N0enlnb3RpYyBoeWJyaWQgbmVjcm9zaXMgaW4gYW4gaW50ZXJzcGVjaWZpYyBwZWFyIHBvcHVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhvcnRpYy4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xNTA2NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMsIG1ldGFib2xpdGUgYW5kIGRldmVsb3BtZW50YWwgZGV0ZXJtaW5pc20gb2YgZnJ1aXQgZnJpY3Rpb24gZGlzY29sb3VyYXRpb24gaW4gcGVhcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIG1hcHBpbmcgb2YgcG9seWdlbmljIHNjYWIgKFZlbnR1cmlhIHBpcmluYSkgcmVzaXN0YW5jZSBpbiBhbiBpbnRlcnNwZWNpZmljIHBlYXIgZmFtaWx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MjE3OS0yMTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLWRlbnNpdHkgZ2VuZXRpYyBsaW5rYWdlIG1hcCBjb25zdHJ1Y3Rpb24gYW5kIGlkZW50aWZpY2F0aW9uIG9mIGZydWl0LXJlbGF0ZWQgUVRMcyBpbiBwZWFyIHVzaW5nIFNOUCBhbmQgU1NSIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUogRXhwLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz01NzcxLTU3ODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIEFGTFAsIFNSQVAsIGFuZCBTU1IgZ2VuZXRpYyBsaW5rYWdlIG1hcCBhbmQgaWRlbnRpZmljYXRpb24gb2YgUVRMcyBmb3IgZnJ1aXQgdHJhaXRzIGluIHBlYXIgKFB5cnVzIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz02NzgtNjg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5lIG1hcHBpbmcgb2YgdGhlIGdlbmUgZm9yIHN1c2NlcHRpYmlsaXR5IHRvIGJsYWNrIHNwb3QgZGlzZWFzZSBpbiBKYXBhbmVzZSBwZWFyIChQeXJ1cyBweXJpZm9saWEgTmFrYWkpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmVlZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9MjcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBvZiBkaXNlYXNlLXJlbGF0ZWQgZ2VuZXMgaW4gamFwYW5lc2UgcGVhciB1c2luZyBhIG1vbGVjdWxhciBsaW5rYWdlIG1hcCB3aXRoIFJBUEQgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJlZWQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTE3OS0xODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbGlua2FnZSBtYXBzIGNvbnN0cnVjdGVkIGJ5IHVzaW5nIGFuIGludGVyc3BlY2lmaWMgY3Jvc3MgYmV0d2VlbiBKYXBhbmVzZSBhbmQgRXVyb3BlYW4gcGVhcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz05LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmeWluZyBRVExzIGZvciBmaXJlLWJsaWdodCByZXNpc3RhbmNlIHZpYSBhIEV1cm9wZWFuIHBlYXIgKFB5cnVzIGNvbW11bmlzIEwuKSBnZW5ldGljIGxpbmthZ2UgbWFwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NDA3LTQxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRlZCByZWZlcmVuY2UgZ2VuZXRpYyBsaW5rYWdlIG1hcHMgb2YgcGVhciBiYXNlZCBvbiBTU1IgYW5kIEFGTFAgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJlZWQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01NzsgIGNpdGF0aW9uX3BhZ2VzPTMyMS0zMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbGlua2FnZSBtYXAgb2YgdGhlIEphcGFuZXNlIHBlYXIg4oCYSG91c3Vp4oCZIGlkZW50aWZ5aW5nIHRocmVlIGhvbW96eWdvdXMgZ2Vub21pYyByZWdpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBKcG4uIFNvYy4gSG9ydGljLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz00MTctNDI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2ZlcmFiaWxpdHkgb2YgbmV3bHkgZGV2ZWxvcGVkIHBlYXIgU1NSIG1hcmtlcnMgdG8gb3RoZXIgUm9zYWNlYWUgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xMjcxLTEyODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGhpZ2gtZGVuc2l0eSBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0IGNvbnNlbnN1cyBnZW5ldGljIG1hcCBmb3IgcGVhciAoUHlydXMgc3BwLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbC4gQmlvbC4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MzE2LTMyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1hcmtlcnMgZGV2ZWxvcGVkIGZyb20gZ2Vub21lIHNlcXVlbmNpbmcgYW5hbHlzaXMgaW4gSmFwYW5lc2UgcGVhciAoUHlydXMgcHlyaWZvbGlhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWN0YSBIb3J0aWM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTc2OyAgY2l0YXRpb25fcGFnZXM9NDc3LTQ4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgUVRMcyBjb250cm9sbGluZyBoYXJ2ZXN0IHRpbWUgYW5kIGZydWl0IHNraW4gY29sb3IgaW4gSmFwYW5lc2UgcGVhciAoUHlydXMgcHlyaWZvbGlhIE5ha2FpKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJlZWQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTM1MS0zNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIHBlYXIgKFB5cnVzIGJyZXRzY2huZWlkZXJpIFJlaGQuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTM5Ni00MDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFB5cnVzIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgKFNOUHMpIGFuZCBldmFsdWF0aW9uIGZvciBnZW5ldGljIG1hcHBpbmcgaW4gRXVyb3BlYW4gcGVhciBhbmQgaW50ZXJzcGVjaWZpYyBQeXJ1cyBoeWJyaWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTc3MDIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZS1iYXNlZCBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gbWFya2VycyBmb3IgZ2Vub21lIG1hcHBpbmcgaW4gSmFwYW5lc2UgcGVhciAoUHlydXMgcHlyaWZvbGlhIE5ha2FpKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTg1My04NjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFMTE1BUFM6IHJvYnVzdCBzY2FmZm9sZCBvcmRlcmluZyBiYXNlZCBvbiBtdWx0aXBsZSBtYXBzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHJhZnQgZ2Vub21lIHNlcXVlbmNlIG9mIEV1cm9wZWFuIHBlYXIgKFB5cnVzIGNvbW11bmlzIEwuIOKAmEJhcnRsZXR04oCZKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWU5MjY0NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcm9idXN0LCBzaW1wbGUgZ2Vub3R5cGluZy1ieS1zZXF1ZW5jaW5nIChHQlMpIGFwcHJvYWNoIGZvciBoaWdoIGRpdmVyc2l0eSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE5Mzc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBwbGFudCBETkEgcHJlcGFyYXRpb24gZnJvbSBmcmVzaCB0aXNzdWUtQ1RBQiBtZXRob2Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXRvY2hlbS4gQnVsbDsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTExLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuYWx5c2lzIHJldmVhbHMgbm92ZWwgZmVhdHVyZXMgb2YgdGhlIG1vbGVjdWxhciBldmVudHMgb2NjdXJyaW5nIGluIHRoZSBsYXRpY2lmZXJzIG9mIEhldmVhIGJyYXNpbGllbnNpcyAocGFyYSBydWJiZXIgdHJlZSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTQ3OS00OTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3RRQywgQXZhaWxhYmxlOiBodHRwOi8vc291cmNlZm9yZ2UubmV0L3Byb2plY3RzL3BpY2FyZC9maWxlcy9waWNhcmQtdG9vbHMvLCAzMSBEZWNlbWJlciAyMDE2LCBkYXRlIGxhc3QgYWNjZXNzZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRBU1NFTDogc29mdHdhcmUgZm9yIGFzc29jaWF0aW9uIG1hcHBpbmcgb2YgY29tcGxleCB0cmFpdHMgaW4gZGl2ZXJzZSBzYW1wbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI2MzMtMjYzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBnYXBwZWQtcmVhZCBhbGlnbm1lbnQgd2l0aCBCb3d0aWUgMjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctMzU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW11bHRhbmVvdXMgbWF4aW11bSBsaWtlbGlob29kIGVzdGltYXRpb24gb2YgbGlua2FnZSBhbmQgbGlua2FnZSBwaGFzZXMgaW4gb3V0Y3Jvc3Npbmcgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIFBvcHVsLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MzQ5LTM2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T25lTWFwOiBzb2Z0d2FyZSBmb3IgZ2VuZXRpYyBtYXBwaW5nIGluIG91dGNyb3NzaW5nIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhlcmVkaXRhczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDQ7ICBjaXRhdGlvbl9wYWdlcz03OC03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgY29uc3RydWN0aW9uIG9mIGNvbnNlbnN1cyBnZW5ldGljIG1hcHMgdmlhIGludGVnZXIgbGluZWFyIHByb2dyYW1taW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JRUVFL0FDTSBUcmFucy4gQ29tcHV0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zODEtMzk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSB2aXN1YWxpemF0aW9uIG9mIGdlbmV0aWMgYW5kIHBoeXNpY2FsIG1hcHMgd2l0aCBTdHJ1ZGVsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTEzMDctMTMwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkxBVCDigJMgdGhlIEJMQVNULWxpa2UgYWxpZ25tZW50IHRvb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NTYtNjY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQ1NjYW5YOiBhIHRvb2xraXQgZm9yIGRldGVjdGlvbiBhbmQgZXZvbHV0aW9uYXJ5IGFuYWx5c2lzIG9mIGdlbmUgc3ludGVueSBhbmQgY29sbGluZWFyaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPWU0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0ZWQgY29uc2Vuc3VzIG1hcCBvZiBjdWx0aXZhdGVkIHBlYW51dCBhbmQgd2lsZCByZWxhdGl2ZXMgcmV2ZWFscyBzdHJ1Y3R1cmVzIG9mIHRoZSBBIGFuZCBCIGdlbm9tZXMgb2YgQXJhY2hpcyBhbmQgZGl2ZXJnZW5jZSBvZiB0aGUgbGVndW1lIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNzMtMTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBkb21lc3RpY2F0ZWQgYXBwbGUgKE1hbHVz4oCJw5figIlkb21lc3RpY2EgQm9ya2guKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTgzMy04Mzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbmV0aWMgbWFwcGluZyB1c2luZyBzZXF1ZW5jZWQgUkFEIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNMQUYtc2VxOiBhbiBlZmZpY2llbnQgbWV0aG9kIG9mIGxhcmdlLXNjYWxlIGRlIG5vdm8gU05QIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyB1c2luZyBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU1ODcwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbXVsdGktcG9wdWxhdGlvbiBjb25zZW5zdXMgZ2VuZXRpYyBtYXAgcmV2ZWFscyBpbmNvbnNpc3RlbnQgbWFya2VyIG9yZGVyIGFtb25nIG1hcHMgbGlrZWx5IGF0dHJpYnV0ZWQgdG8gc3RydWN0dXJhbCB2YXJpYXRpb25zIGluIHRoZSBhcHBsZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZXJhcmNoaWNhbCBzY2FmZm9sZGluZyB3aXRoIEJhbWJ1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE0OS0xNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2dyZXNzLCBjaGFsbGVuZ2VzIGFuZCB0aGUgZnV0dXJlIG9mIGNyb3AgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTcxLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvMy8yODkvMTc3NDM5NDYvZHN3MDYzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBHZW5ldGljIG1hcHMgYXJlIGVzc2VudGlhbCB0b29scyBmb3IgcGVhciBnZW5ldGljcyBhbmQgZ2Vub21pY3MgcmVzZWFyY2guIEluIHRoaXMgc3R1ZHksIHdlIGZpcnN0IGNvbnN0cnVjdGVkIGFuIGludGVncmF0ZWQgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCAoUyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mjk2MDA1NiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQ63Hw3Fh1mKBBwimvGYUs7+1SI7w==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f289%2f2960056" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f289%2f2960056" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&amp;Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/3">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="88473398">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473398">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473400">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473400">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473406">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473406">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473420">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473420">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473436">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473436">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473441">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473441">Accession number</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473443">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473443">Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473445">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473445">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473447">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473447">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473449">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473449">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473451">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473451">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/3/279/2972857">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/3/303/2972686">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Integrated high-density consensus genetic map of <em>Pyrus</em> and anchoring of the ‘Bartlett’ v1.0 (<em>Pyrus communis</em>) genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Leiting Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Leiting+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Leiting Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Cecilia H. Deng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Cecilia+H.+Deng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Deng C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Deng C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Cecilia H. Deng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mareike Kn&#228;bel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mareike+Kn%c3%a4bel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kn&#228;bel M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kn&#228;bel M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mareike Knäbel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David Chagn&#233;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=David+Chagn%c3%a9">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chagn&#233; D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chagn&#233; D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">David Chagné</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satish Kumar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>The New Zealand Institute for Plant &amp; Food Research Limited (Plant &amp; Food Research), New Zealand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satish+Kumar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumar S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kumar S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satish Kumar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiangmei Sun</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jiangmei+Sun">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sun J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sun J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jiangmei Sun</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shaoling Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shaoling+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shaoling Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Crop Genetics and Germplasm Enhancement, Centre of Pear Engineering Technology Research, Nanjing Agricultural University, Nanjing, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw063-cor1" data-open="dsw063-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 25 84396485, Fax. +86 25 84396485. Email: <a href="mailto:wujun@njau.edu.cn" target="_blank">wujun@njau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jun Wu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw063-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Sachiko Isobe</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 289–301, <a href='https://doi.org/10.1093/dnares/dsw063'>https://doi.org/10.1093/dnares/dsw063</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 January 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 August 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">16 December 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2960056" href="/dnaresearch/article-pdf/24/3/289/17743946/dsw063.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Leiting Li, Cecilia H. Deng, Mareike Knäbel, David Chagné, Satish Kumar, Jiangmei Sun, Shaoling Zhang, Jun Wu;  Integrated high-density consensus genetic map of <em>Pyrus</em> and anchoring of the ‘Bartlett’ v1.0 (<em>Pyrus communis</em>) genome, <em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 289–301, <a href='https://doi.org/10.1093/dnares/dsw063'>https://doi.org/10.1093/dnares/dsw063</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2960056?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960056?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960056?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960056?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960056?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960056?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960056?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Integrated%20high-density%20consensus%20genetic%20map%20of%20%26lt%3Bem%26gt%3BPyrus%26lt%3B%2Fem%26gt%3B%20and%20anchoring%20of%20the%20%E2%80%98Bartlett%E2%80%99%20v1.0%20(%26lt%3Bem%26gt%3BPyrus%20communis%26lt%3B%2Fem%26gt%3B)%20genome&amp;publicationDate=2017-01-27&amp;volumeNum=24&amp;issueNum=3&amp;author=Li%2C%20Leiting%3B%20Deng%2C%20Cecilia%20H.&amp;startPage=289&amp;endPage=301&amp;contentId=10.1093%2Fdnares%2Fdsw063&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f289%2f2960056" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=88473398 id="88473398" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Genetic maps are essential tools for pear genetics and genomics research. In this study, we first constructed an integrated simple sequence repeat (SSR) and single nucleotide polymorphism (SNP)-based consensus genetic map for pear based on common SSR markers between nine published maps. A total of 5,085 markers, including 1,232 SSRs and 3,853 SNPs, were localized on a consensus map spanning 3,266.0 cM in total, with an average marker interval of 0.64 cM, which represents the highest density consensus map of pear to date. Using three sets of high-density SNP-based genetic maps with European pear genetic backgrounds, we anchored a total of 291.5 Mb of the ‘Bartlett’ v1.0 (<em>Pyrus communis</em> L.) genome scaffolds into 17 pseudo-chromosomes. This accounted for 50.5% of the genome assembly, which was a great improvement on the 29.7% achieved originally. Intra-genome and inter-genome synteny analyses of the new ‘Bartlett’ v1.1 genome assembly with the Asian pear ‘Dangshansuli’ (<em>Pyrus bretschneideri</em> Rehd.) and apple (<em>Malus</em> × <em>domestica</em> Borkh.) genomes uncovered four new segmental duplication regions. The integrated high-density SSR and SNP-based consensus genetic map provided new insights into the genetic structure patterns of pear and assisted in the genome assembly of ‘Bartlett’ through further exploration of different pear genetic maps.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="pear">pear</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic map&quot;">genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome assembly&quot;">genome assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SSR">SSR</a></div>                            <h2 scrollto-destination=88473400 id="88473400" class="section-title" >1. Introduction</h2>
<p>Pear (<em>Pyrus</em> spp.) is an important and popular rosaceous fruit crop cultivated in temperate regions for its economic and nutritional value. In 2014, the world production of pear was 25.8 million tonnes, from a harvest area of 1.6 million ha (<a class="link link-uri" href="http://faostat3.fao.org" target="_blank">http://faostat3.fao.org</a>, 31 December 2016, date last accessed), making it the second most important fruit crop in the Maloideae, after apple. Improving pear germplasm is necessary for fruit breeders to meet consumer preferences and adapt to variable cultivation conditions. Increasingly, traditional breeding approaches are complemented by molecular breeding to take advantage of its efficiency in seedling selection and breeding for complex traits<a reveal-id="dsw063-B1" data-open="dsw063-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> using genomic selection. Essential molecular breeding tools include molecular markers, genetic maps, and whole genome sequences. These tools have many applications in pear, including genetic diversity analyses<a reveal-id="dsw063-B2 dsw063-B3 dsw063-B4" data-open="dsw063-B2 dsw063-B3 dsw063-B4" class="link link-ref link-reveal xref-bibr"><sup>2–4</sup></a> and quantitative trait loci (QTL) mapping for production traits and fruit quality.<a reveal-id="dsw063-B5 dsw063-B6 dsw063-B7 dsw063-B8 dsw063-B9 dsw063-B10 dsw063-B11 dsw063-B12" data-open="dsw063-B5 dsw063-B6 dsw063-B7 dsw063-B8 dsw063-B9 dsw063-B10 dsw063-B11 dsw063-B12" class="link link-ref link-reveal xref-bibr"><sup>5–12</sup></a></p><p>Prototypes of pear genetic maps used marker techniques, such as random amplified polymorphic DNA (RAPD),<a reveal-id="dsw063-B13" data-open="dsw063-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> amplified fragment length polymorphism (AFLP) and simple sequence repeat (SSR).<a reveal-id="dsw063-B14 dsw063-B15 dsw063-B16 dsw063-B17" data-open="dsw063-B14 dsw063-B15 dsw063-B16 dsw063-B17" class="link link-ref link-reveal xref-bibr"><sup>14–17</sup></a> Among different types of markers, SSR markers have been widely used in constructing pear genetic maps because of their co-dominant and transferable nature,<a reveal-id="dsw063-B18" data-open="dsw063-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> thus can be used as common markers to compare and construct consensus maps. From 2007, various pear genetic maps were constructed based on SSR markers.<a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw063-B17" data-open="dsw063-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw063-B19 dsw063-B20 dsw063-B21" data-open="dsw063-B19 dsw063-B20 dsw063-B21" class="link link-ref link-reveal xref-bibr"><sup>19–21</sup></a> However, the map with the highest density<a reveal-id="dsw063-B19" data-open="dsw063-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> consisted of only 734 markers and covered 1,661.4 cM, which makes it difficult to facilitate map-based gene cloning. The construction of these maps were limited to the available molecular markers, either developed from pear expressed sequence tags (ESTs) or transferred from related species, both of which are labour- and time-consuming.</p><p>When the cost of next-generation sequencing (NGS) decreased, pear whole genome sequences were released,<a reveal-id="dsw063-B22" data-open="dsw063-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and single nucleotide polymorphism (SNP) markers were applied rapidly for genetic map construction.<a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw063-B20" data-open="dsw063-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsw063-B24" data-open="dsw063-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The first SNP-based pear genetic map had 857 markers,<a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> but subsequently a ‘Bayuehong (BYH)’ × ‘Dangshansuli’ map with 3,143 SNP markers was developed with the restriction site-associated DNA sequencing (RADseq) technology and 98 SSR markers.<a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> A ‘Hosui’ (<em>Pyrus</em><em>pyrifolia</em>) map of 609 SNP markers and 252 SSR markers was developed from transcriptome data.<a reveal-id="dsw063-B24" data-open="dsw063-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Compared with previous genetic maps of pear, SNP-based maps significantly increased the number of genetic markers covering the whole genome.</p><p>High-quality genetic maps are not only valuable tools for QTL mapping, but can also serve to anchor genome assembly scaffolds into pseudo-chromosomes.<a reveal-id="dsw063-B25" data-open="dsw063-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> The draft genome of ‘Dangshansuli’ (<em>Pyrus</em><em>bretschneideri</em> Rehd.)<a reveal-id="dsw063-B22" data-open="dsw063-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> was reported as highly heterozygous, and assembled with a ×194 coverage of BAC-by-BAC (bacterial artificial chromosome) and NGS. The assembly consisted of 512.0 Mb sequences and 2,103 scaffolds, corresponding to 97.1% of the estimated genome size. From the assembly, 386.7 Mb (or 75.5%), was anchored into 17 pseudo-chromosomes corresponding to the basic chromosome number of pear (<em>x</em> = 17), using a genetic map of 2,005 SNP markers. Concurrently, the draft genome of the European pear ‘Bartlett’ (<em>Pyrus</em><em>communis</em> L.) was assembled with ×11.4 coverage NGS data and consisted of 142,089 scaffolds covering 577.3 Mb in the assembled genome,<a reveal-id="dsw063-B26" data-open="dsw063-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> accounting for most of the expected 600 Mb genome size. However, using 2,279 genetically mapped loci (1,391 apple and 888 pear SNPs), only 171.4 Mb (29.7%) of these scaffolds were anchored to linkage groups (LGs), due to the relatively fragmented scaffold sequences and an insufficient genetic map used to anchor the scaffolds. Therefore, it is important and necessary to develop higher quality genetic maps to help improve the scaffolds anchoring ratio of the ‘Bartlett’ v1.0 genome assembly.</p><p>For this study, we collected nine published pear genetic maps to build an integrated SSR and SNP-based pear consensus genetic map, which was used for map comparisons. Additionally, a set of three high-quality SNP-based pear genetic maps with European pear genetic backgrounds was used for anchoring the ‘Bartlett’ v1.0 genome scaffolds. The new ‘Bartlett’ v1.1 genome assembly was further used for synteny analyses.</p>                            <h2 scrollto-destination=88473406 id="88473406" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=88473407 id="88473407" class="section-title" >2.1. Construction of a new genetic map for ‘Old Home (OH)’ × ‘Louise Bon Jersey (LBJ)’</h3>
<p>A new OHxLBJ genetic map was constructed using SNPs derived from genotyping-by-sequencing (GBS).<a reveal-id="dsw063-B27" data-open="dsw063-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The two European pear cultivars ‘OH’ and ‘LBJ’ were used to create a segregation population of 297 progenies at The New Zealand Institute for Plant &amp; Food Research (PFR).<a reveal-id="dsw063-B5" data-open="dsw063-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Total DNA was extracted from young leaves of sixty F1 trees in their third year of growth after grafting using the CTAB method.<a reveal-id="dsw063-B28" data-open="dsw063-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> The GBS libraries were prepared using restriction enzyme <em>Bam</em>HI following a modified version of the protocol developed by Elshire et al.<a reveal-id="dsw063-B27" data-open="dsw063-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> with the following modifications: one microgram of DNA was used for the restriction digestion; adapter annealing was done following the protocol of Ko et al.<a reveal-id="dsw063-B29" data-open="dsw063-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a>; the adapter ligation step was performed following digestion without drying out the DNA/adaptor mixture; high fidelity enzyme (AccuPrime Taq DNA polymerase High Fidelity, Life Technologies) was used for amplification; the libraries were amplified and cleaned up prior to pooling. The pooled GBS libraries were sequenced on an Illumina HiSeq2000 platform in parallel in two lanes in single-end mode with reads length of 101 bases. The average read count per library was 1.85 million and 1.86 million for the two lanes. The reads were trimmed from 3ʹ and 64 bases long fragments were kept after de-multiplexing. The Phred score for bases in the trimmed reads were higher than 20, so no read was discarded after trimming. The sequencing quality was checked with FastQC version 0.11.2.<a reveal-id="dsw063-B30" data-open="dsw063-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Genotyping was carried out using TASSEL/GBS v5 pipeline,<a reveal-id="dsw063-B31" data-open="dsw063-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> using the ‘Bartlett’ v1.0 genome assembly as a reference genome. For SNP discovery, we requested at least eight supporting reads per site. Mapping of tags (64-base long unique genomics sequences) to the reference genome was performed using Bowtie2 v2.2.5<a reveal-id="dsw063-B32" data-open="dsw063-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> in—end-to-end and—very-sensitive modes. To get the flanking sequences, 250 nucleotides at either side of an SNP site were extracted from the reference genome. Additionally, SNP markers were considered of good quality when they segregated in a Mendelian manner: ABxAA (1:1 ratio), ABxAB (1:2:1 ratio) and AAxAB (1:1 ratio).<a reveal-id="dsw063-B33" data-open="dsw063-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> All SNPs with missing data for &gt;10 individuals were removed from the analysis. A merged genetic map for both parents was constructed using the package Onemap<a reveal-id="dsw063-B34" data-open="dsw063-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> version 2.0-4 in R-studio (R Core Team, 2016). A LOD (logarithm of odds) score of five or higher was used for grouping and the genetic maps were calculated using the Kosambi function. LG numbers were assigned by comparing markers in the new OHxLBJ with the SNP array-based OHxLBJ map published by Montanari et al.<a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p>                            <h3 scrollto-destination=88473409 id="88473409" class="section-title" >2.2. Collection of pear genetic maps and genome resources</h3>
<p>Nine previously published genetic maps were selected to construct an integrated pear consensus genetic (I-PCG) map using common SSR markers among maps as shown in <a reveal-id="dsw063-F1" data-open="dsw063-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>.<a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw063-B17" data-open="dsw063-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw063-B19 dsw063-B20 dsw063-B21" data-open="dsw063-B19 dsw063-B20 dsw063-B21" class="link link-ref link-reveal xref-bibr"><sup>19–21</sup></a><sup>,</sup><a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsw063-B24" data-open="dsw063-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> These maps were generated from four pear populations, including ‘BYH’ × ‘Dangshansuli’ (two maps), ‘Bartlett’ × ‘Hosui’ (four maps), ‘Shinsei’ × 282-12 (‘Hosui’ × ‘La France’) (one map), and ‘Akiakari’ × ‘Taihaku’ (two maps). ‘BYH’ is a descendant of European pear ‘Clapp’s Favorite’ and Chinese pear ‘Zaosuli’. ‘Dangshansuli’ is a native Chinese pear cultivar. The convention of SSR marker names was not exactly the same, especially for those SSRs indicating multiple loci. For instance, TsuGNH235-1 and TsuGNH235_m1 should be the same marker. Thus, we scanned all SSR marker names and renamed them if necessary. The map IDs were named based on their cultivar abbreviations and publication abbreviations (detail information is listed in <a reveal-id="dsw063-T1" data-open="dsw063-T1" class="link link-reveal link-table xref-fig">Table 1</a>). <div content-id="dsw063-T1" class="table-modal"><div class="table-wrap"><div id="dsw063-T1" data-id="dsw063-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Pear genetic maps used to create the integrated SSR and SNP-based pear consensus genetic (I-PCG) map using common SSR markers as bridging markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Map ID </th><th>Parents </th><th>Number of markers </th><th>Number of SSRs </th><th>Number of SNPs </th><th>Number of linkage groups </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>References code </th></tr></thead><tbody><tr><td>BYHxDS-JXB<sup>a</sup> </td><td>‘Bayuehong’ × ‘Dangshansuli’ </td><td char=".">3,241 </td><td char=".">98 </td><td char=".">3,143 </td><td char=".">17 </td><td char=".">2,243.4 </td><td char=".">0.70 </td><td char="."><a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr">10</a> </td></tr><tr><td>HS-TGG<sup>b</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">922 </td><td char=".">252 </td><td char=".">670 </td><td char=".">22 </td><td char=".">1,341.9 </td><td char=".">1.49 </td><td char="."><a reveal-id="dsw063-B24" data-open="dsw063-B24" class="link link-ref link-reveal xref-bibr">24</a> </td></tr><tr><td>BYHxDS-PMBR<sup>c</sup> </td><td>‘Bayuehong’ × ‘Dangshansuli’ </td><td char=".">734 </td><td char=".">734 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,661.4 </td><td char=".">2.32 </td><td char="."><a reveal-id="dsw063-B19" data-open="dsw063-B19" class="link link-ref link-reveal xref-bibr">19</a> </td></tr><tr><td>BL-AH<sup>d</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">467 </td><td char=".">400 </td><td char=".">67 </td><td char=".">17 </td><td char=".">965.0 </td><td char=".">2.14 </td><td char="."><a reveal-id="dsw063-B20" data-open="dsw063-B20" class="link link-ref link-reveal xref-bibr">20</a> </td></tr><tr><td>TH-BS<sup>e</sup> </td><td>‘Akiakari’ × ‘Taihaku’ </td><td char=".">275 </td><td char=".">275 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,039.0 </td><td char=".">4.03 </td><td char="."><a reveal-id="dsw063-B21" data-open="dsw063-B21" class="link link-ref link-reveal xref-bibr">21</a> </td></tr><tr><td>AK-BS<sup>f</sup> </td><td>‘Akiakari’ × ‘Taihaku’ </td><td char=".">208 </td><td char=".">208 </td><td char=".">0 </td><td char=".">18 </td><td char=".">799.0 </td><td char=".">4.20 </td><td char="."><a reveal-id="dsw063-B21" data-open="dsw063-B21" class="link link-ref link-reveal xref-bibr">21</a> </td></tr><tr><td>LF-BS<sup>g</sup> </td><td>‘Shinsei’ × 282-12 (‘Hosui’ × ‘La France’) </td><td char=".">134 </td><td char=".">134 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,156.0 </td><td char=".">9.88 </td><td char="."><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr">16</a> </td></tr><tr><td>BL-BS<sup>h</sup> </td><td>‘Hosui’ × ‘Bartlett’ </td><td char=".">118 </td><td char=".">118 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,000.0 </td><td char=".">9.90 </td><td char="."><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr">16</a> </td></tr><tr><td>HS-JJSHS<sup>i</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">105 </td><td char=".">105 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,174.0 </td><td char=".">13.34 </td><td char="."><a reveal-id="dsw063-B17" data-open="dsw063-B17" class="link link-ref link-reveal xref-bibr">17</a> </td></tr></tbody></table></div><div content-id="dsw063-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Map ID </th><th>Parents </th><th>Number of markers </th><th>Number of SSRs </th><th>Number of SNPs </th><th>Number of linkage groups </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>References code </th></tr></thead><tbody><tr><td>BYHxDS-JXB<sup>a</sup> </td><td>‘Bayuehong’ × ‘Dangshansuli’ </td><td char=".">3,241 </td><td char=".">98 </td><td char=".">3,143 </td><td char=".">17 </td><td char=".">2,243.4 </td><td char=".">0.70 </td><td char="."><a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr">10</a> </td></tr><tr><td>HS-TGG<sup>b</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">922 </td><td char=".">252 </td><td char=".">670 </td><td char=".">22 </td><td char=".">1,341.9 </td><td char=".">1.49 </td><td char="."><a reveal-id="dsw063-B24" data-open="dsw063-B24" class="link link-ref link-reveal xref-bibr">24</a> </td></tr><tr><td>BYHxDS-PMBR<sup>c</sup> </td><td>‘Bayuehong’ × ‘Dangshansuli’ </td><td char=".">734 </td><td char=".">734 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,661.4 </td><td char=".">2.32 </td><td char="."><a reveal-id="dsw063-B19" data-open="dsw063-B19" class="link link-ref link-reveal xref-bibr">19</a> </td></tr><tr><td>BL-AH<sup>d</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">467 </td><td char=".">400 </td><td char=".">67 </td><td char=".">17 </td><td char=".">965.0 </td><td char=".">2.14 </td><td char="."><a reveal-id="dsw063-B20" data-open="dsw063-B20" class="link link-ref link-reveal xref-bibr">20</a> </td></tr><tr><td>TH-BS<sup>e</sup> </td><td>‘Akiakari’ × ‘Taihaku’ </td><td char=".">275 </td><td char=".">275 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,039.0 </td><td char=".">4.03 </td><td char="."><a reveal-id="dsw063-B21" data-open="dsw063-B21" class="link link-ref link-reveal xref-bibr">21</a> </td></tr><tr><td>AK-BS<sup>f</sup> </td><td>‘Akiakari’ × ‘Taihaku’ </td><td char=".">208 </td><td char=".">208 </td><td char=".">0 </td><td char=".">18 </td><td char=".">799.0 </td><td char=".">4.20 </td><td char="."><a reveal-id="dsw063-B21" data-open="dsw063-B21" class="link link-ref link-reveal xref-bibr">21</a> </td></tr><tr><td>LF-BS<sup>g</sup> </td><td>‘Shinsei’ × 282-12 (‘Hosui’ × ‘La France’) </td><td char=".">134 </td><td char=".">134 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,156.0 </td><td char=".">9.88 </td><td char="."><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr">16</a> </td></tr><tr><td>BL-BS<sup>h</sup> </td><td>‘Hosui’ × ‘Bartlett’ </td><td char=".">118 </td><td char=".">118 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,000.0 </td><td char=".">9.90 </td><td char="."><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr">16</a> </td></tr><tr><td>HS-JJSHS<sup>i</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">105 </td><td char=".">105 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,174.0 </td><td char=".">13.34 </td><td char="."><a reveal-id="dsw063-B17" data-open="dsw063-B17" class="link link-ref link-reveal xref-bibr">17</a> </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw063-T1" data-id="dsw063-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Pear genetic maps used to create the integrated SSR and SNP-based pear consensus genetic (I-PCG) map using common SSR markers as bridging markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Map ID </th><th>Parents </th><th>Number of markers </th><th>Number of SSRs </th><th>Number of SNPs </th><th>Number of linkage groups </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>References code </th></tr></thead><tbody><tr><td>BYHxDS-JXB<sup>a</sup> </td><td>‘Bayuehong’ × ‘Dangshansuli’ </td><td char=".">3,241 </td><td char=".">98 </td><td char=".">3,143 </td><td char=".">17 </td><td char=".">2,243.4 </td><td char=".">0.70 </td><td char="."><a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr">10</a> </td></tr><tr><td>HS-TGG<sup>b</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">922 </td><td char=".">252 </td><td char=".">670 </td><td char=".">22 </td><td char=".">1,341.9 </td><td char=".">1.49 </td><td char="."><a reveal-id="dsw063-B24" data-open="dsw063-B24" class="link link-ref link-reveal xref-bibr">24</a> </td></tr><tr><td>BYHxDS-PMBR<sup>c</sup> </td><td>‘Bayuehong’ × ‘Dangshansuli’ </td><td char=".">734 </td><td char=".">734 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,661.4 </td><td char=".">2.32 </td><td char="."><a reveal-id="dsw063-B19" data-open="dsw063-B19" class="link link-ref link-reveal xref-bibr">19</a> </td></tr><tr><td>BL-AH<sup>d</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">467 </td><td char=".">400 </td><td char=".">67 </td><td char=".">17 </td><td char=".">965.0 </td><td char=".">2.14 </td><td char="."><a reveal-id="dsw063-B20" data-open="dsw063-B20" class="link link-ref link-reveal xref-bibr">20</a> </td></tr><tr><td>TH-BS<sup>e</sup> </td><td>‘Akiakari’ × ‘Taihaku’ </td><td char=".">275 </td><td char=".">275 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,039.0 </td><td char=".">4.03 </td><td char="."><a reveal-id="dsw063-B21" data-open="dsw063-B21" class="link link-ref link-reveal xref-bibr">21</a> </td></tr><tr><td>AK-BS<sup>f</sup> </td><td>‘Akiakari’ × ‘Taihaku’ </td><td char=".">208 </td><td char=".">208 </td><td char=".">0 </td><td char=".">18 </td><td char=".">799.0 </td><td char=".">4.20 </td><td char="."><a reveal-id="dsw063-B21" data-open="dsw063-B21" class="link link-ref link-reveal xref-bibr">21</a> </td></tr><tr><td>LF-BS<sup>g</sup> </td><td>‘Shinsei’ × 282-12 (‘Hosui’ × ‘La France’) </td><td char=".">134 </td><td char=".">134 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,156.0 </td><td char=".">9.88 </td><td char="."><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr">16</a> </td></tr><tr><td>BL-BS<sup>h</sup> </td><td>‘Hosui’ × ‘Bartlett’ </td><td char=".">118 </td><td char=".">118 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,000.0 </td><td char=".">9.90 </td><td char="."><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr">16</a> </td></tr><tr><td>HS-JJSHS<sup>i</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">105 </td><td char=".">105 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,174.0 </td><td char=".">13.34 </td><td char="."><a reveal-id="dsw063-B17" data-open="dsw063-B17" class="link link-ref link-reveal xref-bibr">17</a> </td></tr></tbody></table></div><div content-id="dsw063-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Map ID </th><th>Parents </th><th>Number of markers </th><th>Number of SSRs </th><th>Number of SNPs </th><th>Number of linkage groups </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>References code </th></tr></thead><tbody><tr><td>BYHxDS-JXB<sup>a</sup> </td><td>‘Bayuehong’ × ‘Dangshansuli’ </td><td char=".">3,241 </td><td char=".">98 </td><td char=".">3,143 </td><td char=".">17 </td><td char=".">2,243.4 </td><td char=".">0.70 </td><td char="."><a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr">10</a> </td></tr><tr><td>HS-TGG<sup>b</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">922 </td><td char=".">252 </td><td char=".">670 </td><td char=".">22 </td><td char=".">1,341.9 </td><td char=".">1.49 </td><td char="."><a reveal-id="dsw063-B24" data-open="dsw063-B24" class="link link-ref link-reveal xref-bibr">24</a> </td></tr><tr><td>BYHxDS-PMBR<sup>c</sup> </td><td>‘Bayuehong’ × ‘Dangshansuli’ </td><td char=".">734 </td><td char=".">734 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,661.4 </td><td char=".">2.32 </td><td char="."><a reveal-id="dsw063-B19" data-open="dsw063-B19" class="link link-ref link-reveal xref-bibr">19</a> </td></tr><tr><td>BL-AH<sup>d</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">467 </td><td char=".">400 </td><td char=".">67 </td><td char=".">17 </td><td char=".">965.0 </td><td char=".">2.14 </td><td char="."><a reveal-id="dsw063-B20" data-open="dsw063-B20" class="link link-ref link-reveal xref-bibr">20</a> </td></tr><tr><td>TH-BS<sup>e</sup> </td><td>‘Akiakari’ × ‘Taihaku’ </td><td char=".">275 </td><td char=".">275 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,039.0 </td><td char=".">4.03 </td><td char="."><a reveal-id="dsw063-B21" data-open="dsw063-B21" class="link link-ref link-reveal xref-bibr">21</a> </td></tr><tr><td>AK-BS<sup>f</sup> </td><td>‘Akiakari’ × ‘Taihaku’ </td><td char=".">208 </td><td char=".">208 </td><td char=".">0 </td><td char=".">18 </td><td char=".">799.0 </td><td char=".">4.20 </td><td char="."><a reveal-id="dsw063-B21" data-open="dsw063-B21" class="link link-ref link-reveal xref-bibr">21</a> </td></tr><tr><td>LF-BS<sup>g</sup> </td><td>‘Shinsei’ × 282-12 (‘Hosui’ × ‘La France’) </td><td char=".">134 </td><td char=".">134 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,156.0 </td><td char=".">9.88 </td><td char="."><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr">16</a> </td></tr><tr><td>BL-BS<sup>h</sup> </td><td>‘Hosui’ × ‘Bartlett’ </td><td char=".">118 </td><td char=".">118 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,000.0 </td><td char=".">9.90 </td><td char="."><a reveal-id="dsw063-B16" data-open="dsw063-B16" class="link link-ref link-reveal xref-bibr">16</a> </td></tr><tr><td>HS-JJSHS<sup>i</sup> </td><td>‘Bartlett’ × ‘Hosui’ </td><td char=".">105 </td><td char=".">105 </td><td char=".">0 </td><td char=".">17 </td><td char=".">1,174.0 </td><td char=".">13.34 </td><td char="."><a reveal-id="dsw063-B17" data-open="dsw063-B17" class="link link-ref link-reveal xref-bibr">17</a> </td></tr></tbody></table></div></div></p>                        <a id="88473411" scrollto-destination="88473411"></a>
<div id="dsw063-F1" data-id="dsw063-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f1.png?Expires=2147483647&Signature=Y6k1ssnZyVYNDxRqdxST2YDjzoJJKUcwiEm4raElX6Vi3MFXVQS7FiM6mQFHpqswQXAMV9lW05vY7S6HXUpYvsikPUFIBTMTxWCocVMRDcWKC58J-yVZXgnWC2KezdOpkngNTZVSzsXPyoq7QIL6VWJXCnRKPU8GSR42BSdRE8UdmTgEEF8-oPmTX5rOOIvbsPcqQa2bKne8ElRnWWu17qh89f6vYYz1WF0kzHsLgPGoYhXs9yEu~teTnbHyLVwPwdJVz42TQdRrTCAhrIGyiGMGcJ6aB7sisFVMUTREHvrfJ7mLvIkxV0JgV8XPu2GAFcSIHR7peMNR-lgAOWUPBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow for the integrated SSR and SNP-based pear consensus genetic (I-PCG) map construction and ‘Bartlett’ v1.0 genome scaffolds anchoring." path-from-xml="dsw063f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473411/dsw063f1.tif" path-from-xml="dsw063f1.tif" target="_blank">View large</a><a section="88473411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f1.png?Expires=2147483647&Signature=BTF-ae-8PozL4ZUi20WxpBufNpqSNygavq7gZ0ZrSulA7A0nm4zz758y2fMq39NNL2iCM08Cul0ky9K0dVldC3fLXT~efxFkuv2SNhNB1Ha7XdGqdyhV~34o2JBrWlYkI8~18QXuD6EvozM4Bk8IFENeI6nLai8S60HTQIJLJyqBWNcFsftlwjh2Tnk2-uoXVySAqSumwute082kdrMe1qtOwy8ZgiHZVjpTuyEt6sgAj75ePOoA8SRa0sTnEiaKuf-DoEdXJ4asjXoQs2IacD1gDG2bQ0ulT~ekvq-MJNEzclyxXR6Ao1oa48xyDGuSDPNIrtgdUPPHKNLnzFcYpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473411&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow for the integrated SSR and SNP-based pear consensus genetic (I-PCG) map construction and ‘Bartlett’ v1.0 genome scaffolds anchoring.</p></div></div><div content-id="dsw063-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f1.png?Expires=2147483647&Signature=Y6k1ssnZyVYNDxRqdxST2YDjzoJJKUcwiEm4raElX6Vi3MFXVQS7FiM6mQFHpqswQXAMV9lW05vY7S6HXUpYvsikPUFIBTMTxWCocVMRDcWKC58J-yVZXgnWC2KezdOpkngNTZVSzsXPyoq7QIL6VWJXCnRKPU8GSR42BSdRE8UdmTgEEF8-oPmTX5rOOIvbsPcqQa2bKne8ElRnWWu17qh89f6vYYz1WF0kzHsLgPGoYhXs9yEu~teTnbHyLVwPwdJVz42TQdRrTCAhrIGyiGMGcJ6aB7sisFVMUTREHvrfJ7mLvIkxV0JgV8XPu2GAFcSIHR7peMNR-lgAOWUPBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow for the integrated SSR and SNP-based pear consensus genetic (I-PCG) map construction and ‘Bartlett’ v1.0 genome scaffolds anchoring." path-from-xml="dsw063f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473411/dsw063f1.tif" path-from-xml="dsw063f1.tif" target="_blank">View large</a><a section="88473411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f1.png?Expires=2147483647&Signature=BTF-ae-8PozL4ZUi20WxpBufNpqSNygavq7gZ0ZrSulA7A0nm4zz758y2fMq39NNL2iCM08Cul0ky9K0dVldC3fLXT~efxFkuv2SNhNB1Ha7XdGqdyhV~34o2JBrWlYkI8~18QXuD6EvozM4Bk8IFENeI6nLai8S60HTQIJLJyqBWNcFsftlwjh2Tnk2-uoXVySAqSumwute082kdrMe1qtOwy8ZgiHZVjpTuyEt6sgAj75ePOoA8SRa0sTnEiaKuf-DoEdXJ4asjXoQs2IacD1gDG2bQ0ulT~ekvq-MJNEzclyxXR6Ao1oa48xyDGuSDPNIrtgdUPPHKNLnzFcYpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473411&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow for the integrated SSR and SNP-based pear consensus genetic (I-PCG) map construction and ‘Bartlett’ v1.0 genome scaffolds anchoring.</p></div></div><p>Three high-density SNP-based genetic maps (<a reveal-id="dsw063-F1" data-open="dsw063-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>) with European pear genetic backgrounds were used for anchoring the ‘Bartlett’ v1.0 genome scaffolds, while other genetic maps with Asian pear (<em>P. bretschneideri</em> or <em>P. pyrifolia</em>) genetic backgrounds were not involved in the anchoring because of the large differences in SNPs and their low efficiency of transferability among different species. These three maps included the newly created genetic map OHxLBJ, a ‘BYH’ map extracted from the ‘BYH’ × ‘Dangshansuli’ consensus map (named BYHxDS-JXB),<a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> and a European pear and interspecific Asian pear hybrid consensus genetic (PH-CG) map attained from the 10 genetic maps published by Montanari et al.<a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> derived from five populations, including OHxLBJ, PremP003 × ‘Moonglow’ (T003xM), PEAR1 × PEAR2 (T042xT081),<a reveal-id="dsw063-B9" data-open="dsw063-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> POP369 (T052xT003), and POP356 (T052xT064).<a reveal-id="dsw063-B8" data-open="dsw063-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> The BYH map was a subset of BYHxDS-JXB<a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> with marker names start with Pyb and Pybd, representing SNPs derived from ‘BYH’ and both parents, respectively.</p><p>The I-PCG map was not used for anchoring the ‘Bartlett’ v1.0 genome scaffolds (<a reveal-id="dsw063-F1" data-open="dsw063-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) because the consensus genetic map had a mixed genetic background of <em>P. bretschneideri</em>, <em>P. pyrifolia</em> and <em>P. communis</em>, which could cause more unpredictable conflicts owing to genome variation. The OHxLBJ and PH-CG maps were not merged into the I-PCG map (<a reveal-id="dsw063-F1" data-open="dsw063-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) because they were SNP-based maps, and markers in these maps were all unique and had no common markers with the I-PCG map for anchoring the corresponding LGs.</p><p>Pseudo-chromosome sequences of ‘Dangshansuli’ genome v1.0 were retrieved from the Pear Genome Project website (<a class="link link-uri" href="http://peargenome.njau.edu.cn/default.asp?d=4&amp;m=2" target="_blank">http://peargenome.njau.edu.cn/default.asp?d=4&amp;m=2</a>, 31 December 2016, date last accessed); scaffold sequences of the ‘Bartlett’ v1.0 genome (<a class="link link-uri" href="https://www.rosaceae.org/species/pyrus/pyrus_communis/genome_v1.0" target="_blank">https://www.rosaceae.org/species/pyrus/pyrus_communis/genome_v1.0</a>, 31 December 2016, date last accessed); and pseudo-chromosome sequences of the ‘Golden Delicious’ genome version 1.0 primary (<a class="link link-uri" href="https://www.rosaceae.org/species/malus/malus_x_domestica/genome_v1.0p" target="_blank">https://www.rosaceae.org/species/malus/malus_x_domestica/genome_v1.0p</a>, 31 December 2016, date last accessed) were retrieved from the Genome Database of Rosaceae (<a class="link link-uri" href="https://www.rosaceae.org/" target="_blank">https://www.rosaceae.org/</a>).</p>                            <h3 scrollto-destination=88473415 id="88473415" class="section-title" >2.3. Construction and comparison of pear consensus genetic maps</h3>
<p>MergeMap,<a reveal-id="dsw063-B35" data-open="dsw063-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> a software that uses common markers from different maps as bridging markers to merge maps, was used to create the I-PCG map from nine published maps (<a reveal-id="dsw063-F1" data-open="dsw063-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>) and the PH-CG map (<a reveal-id="dsw063-F1" data-open="dsw063-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). To prepare input data for Mergemap, we firstly removed conflict markers that presented in inconsistent LGs. Secondly, each individual LG in the input genetic maps for MergeMap was filtered using three criteria: number of markers in a LG not &lt;3, LG length not &lt;5 cM, and maximum common markers of a LG with other maps not &lt;3. Thirdly, for the I-PCG map, each individual map was set as equal weights in the map merging process, while for the PH-CG map, individual maps were set based on the genetic background of European pear, weight = 4 if an individual map was <em>P. communis</em>, weight = 2 if one of its parents was <em>P. communis</em>, and weight = 1 if one of its grandparents was <em>P. communis</em>. Lastly, Mergemap tried to resolve conflicts in map positions among the individual maps by deleting a minimum set of marker occurrences.<a reveal-id="dsw063-B35" data-open="dsw063-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Genetic map comparison between the I-PCG and individual maps were visualized by the Strudel<a reveal-id="dsw063-B36" data-open="dsw063-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> software.</p><p>Most maps for constructing the I-PCG map consisted of 17 LGs, which was the same as the basic chromosome number of pear (<em>x</em> = 17), except for two maps (HS-TGG and AK-BS) (<a reveal-id="dsw063-T1" data-open="dsw063-T1" class="link link-reveal link-table xref-fig">Table 1</a>). To confirm whether these maps were appropriate for merging into one consensus map, we firstly calculated the number of common SSR markers between these maps (<a reveal-id="dsw063-T2" data-open="dsw063-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Each map was found to have at least 26 SSR markers in common with any other map. For instance, the number of common SSR markers for AK-BS vs. TH-BS, AK-BS vs. BL-AH and TH-BS vs. BL-AH was 115, 105 and 139, respectively. <div content-id="dsw063-T2" class="table-modal"><div class="table-wrap"><div id="dsw063-T2" data-id="dsw063-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Number of pairwise common SSR markers for nine pear genetic maps used for creating the integrated SSR and SNP-based pear consensus genetic (I-PCG) map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Map ID </th><th>BYHxDS-JXB </th><th>HS-TGG </th><th>BYHxDS-PMBR </th><th>BL-AH </th><th>TH-BS </th><th>AK-BS </th><th>LF-BS </th><th>BL-BS </th></tr></thead><tbody><tr><td>HS-TGG </td><td char=".">42 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BYHxDS-PMBR </td><td char=".">63 </td><td char=".">45 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BL-AH </td><td char=".">52 </td><td char=".">153 </td><td char=".">65 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>TH-BS </td><td char=".">31 </td><td char=".">80 </td><td char=".">38 </td><td char=".">139 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>AK-BS </td><td char=".">26 </td><td char=".">87 </td><td char=".">34 </td><td char=".">105 </td><td char=".">115 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>LF-BS </td><td char=".">43 </td><td char=".">56 </td><td char=".">59 </td><td char=".">95 </td><td char=".">62 </td><td char=".">46 </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BL-BS </td><td char=".">44 </td><td char=".">53 </td><td char=".">45 </td><td char=".">112 </td><td char=".">54 </td><td char=".">39 </td><td char=".">95 </td><td char=".">– </td></tr><tr><td>HS-JJSHS </td><td char=".">36 </td><td char=".">94 </td><td char=".">36 </td><td char=".">61 </td><td char=".">39 </td><td char=".">36 </td><td char=".">55 </td><td char=".">53 </td></tr></tbody></table></div><div content-id="dsw063-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Map ID </th><th>BYHxDS-JXB </th><th>HS-TGG </th><th>BYHxDS-PMBR </th><th>BL-AH </th><th>TH-BS </th><th>AK-BS </th><th>LF-BS </th><th>BL-BS </th></tr></thead><tbody><tr><td>HS-TGG </td><td char=".">42 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BYHxDS-PMBR </td><td char=".">63 </td><td char=".">45 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BL-AH </td><td char=".">52 </td><td char=".">153 </td><td char=".">65 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>TH-BS </td><td char=".">31 </td><td char=".">80 </td><td char=".">38 </td><td char=".">139 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>AK-BS </td><td char=".">26 </td><td char=".">87 </td><td char=".">34 </td><td char=".">105 </td><td char=".">115 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>LF-BS </td><td char=".">43 </td><td char=".">56 </td><td char=".">59 </td><td char=".">95 </td><td char=".">62 </td><td char=".">46 </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BL-BS </td><td char=".">44 </td><td char=".">53 </td><td char=".">45 </td><td char=".">112 </td><td char=".">54 </td><td char=".">39 </td><td char=".">95 </td><td char=".">– </td></tr><tr><td>HS-JJSHS </td><td char=".">36 </td><td char=".">94 </td><td char=".">36 </td><td char=".">61 </td><td char=".">39 </td><td char=".">36 </td><td char=".">55 </td><td char=".">53 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw063-T2" data-id="dsw063-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Number of pairwise common SSR markers for nine pear genetic maps used for creating the integrated SSR and SNP-based pear consensus genetic (I-PCG) map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Map ID </th><th>BYHxDS-JXB </th><th>HS-TGG </th><th>BYHxDS-PMBR </th><th>BL-AH </th><th>TH-BS </th><th>AK-BS </th><th>LF-BS </th><th>BL-BS </th></tr></thead><tbody><tr><td>HS-TGG </td><td char=".">42 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BYHxDS-PMBR </td><td char=".">63 </td><td char=".">45 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BL-AH </td><td char=".">52 </td><td char=".">153 </td><td char=".">65 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>TH-BS </td><td char=".">31 </td><td char=".">80 </td><td char=".">38 </td><td char=".">139 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>AK-BS </td><td char=".">26 </td><td char=".">87 </td><td char=".">34 </td><td char=".">105 </td><td char=".">115 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>LF-BS </td><td char=".">43 </td><td char=".">56 </td><td char=".">59 </td><td char=".">95 </td><td char=".">62 </td><td char=".">46 </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BL-BS </td><td char=".">44 </td><td char=".">53 </td><td char=".">45 </td><td char=".">112 </td><td char=".">54 </td><td char=".">39 </td><td char=".">95 </td><td char=".">– </td></tr><tr><td>HS-JJSHS </td><td char=".">36 </td><td char=".">94 </td><td char=".">36 </td><td char=".">61 </td><td char=".">39 </td><td char=".">36 </td><td char=".">55 </td><td char=".">53 </td></tr></tbody></table></div><div content-id="dsw063-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Map ID </th><th>BYHxDS-JXB </th><th>HS-TGG </th><th>BYHxDS-PMBR </th><th>BL-AH </th><th>TH-BS </th><th>AK-BS </th><th>LF-BS </th><th>BL-BS </th></tr></thead><tbody><tr><td>HS-TGG </td><td char=".">42 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BYHxDS-PMBR </td><td char=".">63 </td><td char=".">45 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BL-AH </td><td char=".">52 </td><td char=".">153 </td><td char=".">65 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>TH-BS </td><td char=".">31 </td><td char=".">80 </td><td char=".">38 </td><td char=".">139 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>AK-BS </td><td char=".">26 </td><td char=".">87 </td><td char=".">34 </td><td char=".">105 </td><td char=".">115 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td></tr><tr><td>LF-BS </td><td char=".">43 </td><td char=".">56 </td><td char=".">59 </td><td char=".">95 </td><td char=".">62 </td><td char=".">46 </td><td char=".">– </td><td char=".">– </td></tr><tr><td>BL-BS </td><td char=".">44 </td><td char=".">53 </td><td char=".">45 </td><td char=".">112 </td><td char=".">54 </td><td char=".">39 </td><td char=".">95 </td><td char=".">– </td></tr><tr><td>HS-JJSHS </td><td char=".">36 </td><td char=".">94 </td><td char=".">36 </td><td char=".">61 </td><td char=".">39 </td><td char=".">36 </td><td char=".">55 </td><td char=".">53 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=88473418 id="88473418" class="section-title" >2.4. Anchoring ‘Bartlett’ scaffolds and synteny analyses of intra- or inter-genomes</h3>
<p>SSR primer sequences and SNP flanking sequences were mapped onto ‘Bartlett’ scaffolds using isPcr and blat programs, respectively.<a reveal-id="dsw063-B37" data-open="dsw063-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> In either condition, only the best match of each marker was kept, and markers were removed if there were two equally good best matches. The ALLMAPS software, with the strength of computing a scaffold ordering that maximizes collinearity across a collection of maps,<a reveal-id="dsw063-B25" data-open="dsw063-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> was used for anchoring the ‘Bartlett’ genome with the guidance of three high-density SNP-based genetic maps with European pear genetic backgrounds as mentioned above. The multiple collinearity scan toolkit (MCScanX)<a reveal-id="dsw063-B38" data-open="dsw063-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> was used to perform intra-genome synteny analyses for the ‘Bartlett’ genome, and inter-genome analyses for the ‘Bartlett’ versus ‘Dangshansuli’ genomes, and ‘Bartlett’ genome versus the genome of apple cultivar ‘Golden Delicious’.</p>                            <h2 scrollto-destination=88473420 id="88473420" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=88473421 id="88473421" class="section-title" >3.1. Merging multiple genetic maps leads to a high quality and density integrated SSR and SNP-based consensus map of pear</h3>
<p>Nine published pear genetic maps (<a reveal-id="dsw063-T1" data-open="dsw063-T1" class="link link-reveal link-table xref-fig">Table 1</a>) constructed mainly using SSR and SNP markers were integrated into a consensus genetic map I-PCG, using SSR markers in common and the software MergeMap. The number of markers for these maps ranged from 105 for HS-JJSHS<a reveal-id="dsw063-B17" data-open="dsw063-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> to 3,241 for BYHxDS-JXB.<a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Among these, BYHxDS-JXB had the most SNP markers (3,143) and BYHxDS-PMBR<a reveal-id="dsw063-B19" data-open="dsw063-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> had the most SSR markers (734). The integrated SSR and SNP-based pear consensus genetic map (I-PCG map) consisted of 17 LGs made up of 5,085 markers in total (1,232 SSRs, and 3,853 SNPs), spanning 3,266 cM, with an average marker interval of 0.64 cM (<a reveal-id="dsw063-T3" data-open="dsw063-T3" class="link link-reveal link-table xref-fig">Table 3</a>; markers listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). As shown in <a reveal-id="dsw063-F2" data-open="dsw063-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, a large proportion (75.8%) of the markers in the map were SNPs, from a minimum of 63.3% on LG 16 (abbreviated as LG16, same as below), to 90.2% on LG5. SSR markers were interspersed with SNP markers in almost all parts of the 17 LGs, except about 75.5 cM at the lower end of LG5, which contained only one SSR marker, and the top 56.5 cM of LG12, which contained no SNP markers. The LG length varied from the shortest 87.7 cM (LG4) to the longest 269.3 cM (LG15). Among the 17 LGs, only LG4 and LG7 (96.6cM) were shorter than 100cM, and eight LGs (LG3, LG9, LG10, LG11, LG12, LG14, LG15, and LG17) were longer than 200cM. The total genetic length of 3,266 cM makes I-PCG the longest pear consensus genetic map being constructed to date. On the other hand, the large number of markers in this genetic map resulted in the smallest average marker interval of 0.64 cM in pear. In addition, locus clusters were observed in all LGs (<a reveal-id="dsw063-T3" data-open="dsw063-T3" class="link link-reveal link-table xref-fig">Table 3</a>), which were defined as 5 cM regions with not &lt;10 markers.<a reveal-id="dsw063-B39" data-open="dsw063-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> In total, 3,776 markers in 237 clusters accounting for 1,185 cM length were identified in the I-PCG map (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), representing 74.3% of markers and varying from 32 markers (3 clusters) on LG1 to 471 markers (27 clusters) on LG15. <div content-id="dsw063-T3" class="table-modal"><div class="table-wrap"><div id="dsw063-T3" data-id="dsw063-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Summary of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Number of markers </th><th>Number of SSRs </th><th>Number of SNPs </th><th>Number of common SSR markers </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>Length of the locus cluster </th><th>Marker number in the locus clusters </th></tr></thead><tbody><tr><td>1 </td><td char=".">124 </td><td char=".">38 </td><td char=".">86 </td><td char=".">20 </td><td char=".">147.9 </td><td char=".">1.20 </td><td char=".">15 </td><td char=".">32 </td></tr><tr><td>2 </td><td char=".">367 </td><td char=".">90 </td><td char=".">277 </td><td char=".">29 </td><td char=".">198.6 </td><td char=".">0.54 </td><td char=".">100 </td><td char=".">324 </td></tr><tr><td>3 </td><td char=".">374 </td><td char=".">82 </td><td char=".">292 </td><td char=".">29 </td><td char=".">247.6 </td><td char=".">0.66 </td><td char=".">90 </td><td char=".">278 </td></tr><tr><td>4 </td><td char=".">169 </td><td char=".">51 </td><td char=".">118 </td><td char=".">15 </td><td char=".">87.7 </td><td char=".">0.52 </td><td char=".">55 </td><td char=".">146 </td></tr><tr><td>5 </td><td char=".">325 </td><td char=".">32 </td><td char=".">293 </td><td char=".">19 </td><td char=".">185.5 </td><td char=".">0.57 </td><td char=".">65 </td><td char=".">248 </td></tr><tr><td>6 </td><td char=".">318 </td><td char=".">69 </td><td char=".">249 </td><td char=".">19 </td><td char=".">158.3 </td><td char=".">0.50 </td><td char=".">75 </td><td char=".">257 </td></tr><tr><td>7 </td><td char=".">206 </td><td char=".">37 </td><td char=".">169 </td><td char=".">20 </td><td char=".">96.6 </td><td char=".">0.47 </td><td char=".">55 </td><td char=".">178 </td></tr><tr><td>8 </td><td char=".">273 </td><td char=".">56 </td><td char=".">217 </td><td char=".">20 </td><td char=".">168.0 </td><td char=".">0.62 </td><td char=".">65 </td><td char=".">212 </td></tr><tr><td>9 </td><td char=".">265 </td><td char=".">66 </td><td char=".">199 </td><td char=".">32 </td><td char=".">224.2 </td><td char=".">0.85 </td><td char=".">55 </td><td char=".">173 </td></tr><tr><td>10 </td><td char=".">373 </td><td char=".">96 </td><td char=".">277 </td><td char=".">34 </td><td char=".">244.3 </td><td char=".">0.66 </td><td char=".">80 </td><td char=".">274 </td></tr><tr><td>11 </td><td char=".">401 </td><td char=".">86 </td><td char=".">315 </td><td char=".">25 </td><td char=".">230.1 </td><td char=".">0.58 </td><td char=".">100 </td><td char=".">326 </td></tr><tr><td>12 </td><td char=".">247 </td><td char=".">86 </td><td char=".">161 </td><td char=".">19 </td><td char=".">206.4 </td><td char=".">0.84 </td><td char=".">45 </td><td char=".">146 </td></tr><tr><td>13 </td><td char=".">255 </td><td char=".">58 </td><td char=".">197 </td><td char=".">21 </td><td char=".">165.3 </td><td char=".">0.65 </td><td char=".">50 </td><td char=".">161 </td></tr><tr><td>14 </td><td char=".">285 </td><td char=".">84 </td><td char=".">201 </td><td char=".">33 </td><td char=".">211.4 </td><td char=".">0.74 </td><td char=".">75 </td><td char=".">198 </td></tr><tr><td>15 </td><td char=".">562 </td><td char=".">131 </td><td char=".">431 </td><td char=".">37 </td><td char=".">269.3 </td><td char=".">0.48 </td><td char=".">135 </td><td char=".">471 </td></tr><tr><td>16 </td><td char=".">207 </td><td char=".">76 </td><td char=".">131 </td><td char=".">21 </td><td char=".">180.3 </td><td char=".">0.88 </td><td char=".">50 </td><td char=".">130 </td></tr><tr><td>17 </td><td char=".">334 </td><td char=".">94 </td><td char=".">240 </td><td char=".">32 </td><td char=".">244.5 </td><td char=".">0.73 </td><td char=".">75 </td><td char=".">222 </td></tr><tr><td><strong>Total</strong> </td><td char=".">5,085 </td><td char=".">1,232 </td><td char=".">3,853 </td><td char=".">425 </td><td>3,266 </td><td char=".">0.64 </td><td char=".">1,185 </td><td char=".">3,776 </td></tr></tbody></table></div><div content-id="dsw063-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Number of markers </th><th>Number of SSRs </th><th>Number of SNPs </th><th>Number of common SSR markers </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>Length of the locus cluster </th><th>Marker number in the locus clusters </th></tr></thead><tbody><tr><td>1 </td><td char=".">124 </td><td char=".">38 </td><td char=".">86 </td><td char=".">20 </td><td char=".">147.9 </td><td char=".">1.20 </td><td char=".">15 </td><td char=".">32 </td></tr><tr><td>2 </td><td char=".">367 </td><td char=".">90 </td><td char=".">277 </td><td char=".">29 </td><td char=".">198.6 </td><td char=".">0.54 </td><td char=".">100 </td><td char=".">324 </td></tr><tr><td>3 </td><td char=".">374 </td><td char=".">82 </td><td char=".">292 </td><td char=".">29 </td><td char=".">247.6 </td><td char=".">0.66 </td><td char=".">90 </td><td char=".">278 </td></tr><tr><td>4 </td><td char=".">169 </td><td char=".">51 </td><td char=".">118 </td><td char=".">15 </td><td char=".">87.7 </td><td char=".">0.52 </td><td char=".">55 </td><td char=".">146 </td></tr><tr><td>5 </td><td char=".">325 </td><td char=".">32 </td><td char=".">293 </td><td char=".">19 </td><td char=".">185.5 </td><td char=".">0.57 </td><td char=".">65 </td><td char=".">248 </td></tr><tr><td>6 </td><td char=".">318 </td><td char=".">69 </td><td char=".">249 </td><td char=".">19 </td><td char=".">158.3 </td><td char=".">0.50 </td><td char=".">75 </td><td char=".">257 </td></tr><tr><td>7 </td><td char=".">206 </td><td char=".">37 </td><td char=".">169 </td><td char=".">20 </td><td char=".">96.6 </td><td char=".">0.47 </td><td char=".">55 </td><td char=".">178 </td></tr><tr><td>8 </td><td char=".">273 </td><td char=".">56 </td><td char=".">217 </td><td char=".">20 </td><td char=".">168.0 </td><td char=".">0.62 </td><td char=".">65 </td><td char=".">212 </td></tr><tr><td>9 </td><td char=".">265 </td><td char=".">66 </td><td char=".">199 </td><td char=".">32 </td><td char=".">224.2 </td><td char=".">0.85 </td><td char=".">55 </td><td char=".">173 </td></tr><tr><td>10 </td><td char=".">373 </td><td char=".">96 </td><td char=".">277 </td><td char=".">34 </td><td char=".">244.3 </td><td char=".">0.66 </td><td char=".">80 </td><td char=".">274 </td></tr><tr><td>11 </td><td char=".">401 </td><td char=".">86 </td><td char=".">315 </td><td char=".">25 </td><td char=".">230.1 </td><td char=".">0.58 </td><td char=".">100 </td><td char=".">326 </td></tr><tr><td>12 </td><td char=".">247 </td><td char=".">86 </td><td char=".">161 </td><td char=".">19 </td><td char=".">206.4 </td><td char=".">0.84 </td><td char=".">45 </td><td char=".">146 </td></tr><tr><td>13 </td><td char=".">255 </td><td char=".">58 </td><td char=".">197 </td><td char=".">21 </td><td char=".">165.3 </td><td char=".">0.65 </td><td char=".">50 </td><td char=".">161 </td></tr><tr><td>14 </td><td char=".">285 </td><td char=".">84 </td><td char=".">201 </td><td char=".">33 </td><td char=".">211.4 </td><td char=".">0.74 </td><td char=".">75 </td><td char=".">198 </td></tr><tr><td>15 </td><td char=".">562 </td><td char=".">131 </td><td char=".">431 </td><td char=".">37 </td><td char=".">269.3 </td><td char=".">0.48 </td><td char=".">135 </td><td char=".">471 </td></tr><tr><td>16 </td><td char=".">207 </td><td char=".">76 </td><td char=".">131 </td><td char=".">21 </td><td char=".">180.3 </td><td char=".">0.88 </td><td char=".">50 </td><td char=".">130 </td></tr><tr><td>17 </td><td char=".">334 </td><td char=".">94 </td><td char=".">240 </td><td char=".">32 </td><td char=".">244.5 </td><td char=".">0.73 </td><td char=".">75 </td><td char=".">222 </td></tr><tr><td><strong>Total</strong> </td><td char=".">5,085 </td><td char=".">1,232 </td><td char=".">3,853 </td><td char=".">425 </td><td>3,266 </td><td char=".">0.64 </td><td char=".">1,185 </td><td char=".">3,776 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw063-T3" data-id="dsw063-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Summary of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Number of markers </th><th>Number of SSRs </th><th>Number of SNPs </th><th>Number of common SSR markers </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>Length of the locus cluster </th><th>Marker number in the locus clusters </th></tr></thead><tbody><tr><td>1 </td><td char=".">124 </td><td char=".">38 </td><td char=".">86 </td><td char=".">20 </td><td char=".">147.9 </td><td char=".">1.20 </td><td char=".">15 </td><td char=".">32 </td></tr><tr><td>2 </td><td char=".">367 </td><td char=".">90 </td><td char=".">277 </td><td char=".">29 </td><td char=".">198.6 </td><td char=".">0.54 </td><td char=".">100 </td><td char=".">324 </td></tr><tr><td>3 </td><td char=".">374 </td><td char=".">82 </td><td char=".">292 </td><td char=".">29 </td><td char=".">247.6 </td><td char=".">0.66 </td><td char=".">90 </td><td char=".">278 </td></tr><tr><td>4 </td><td char=".">169 </td><td char=".">51 </td><td char=".">118 </td><td char=".">15 </td><td char=".">87.7 </td><td char=".">0.52 </td><td char=".">55 </td><td char=".">146 </td></tr><tr><td>5 </td><td char=".">325 </td><td char=".">32 </td><td char=".">293 </td><td char=".">19 </td><td char=".">185.5 </td><td char=".">0.57 </td><td char=".">65 </td><td char=".">248 </td></tr><tr><td>6 </td><td char=".">318 </td><td char=".">69 </td><td char=".">249 </td><td char=".">19 </td><td char=".">158.3 </td><td char=".">0.50 </td><td char=".">75 </td><td char=".">257 </td></tr><tr><td>7 </td><td char=".">206 </td><td char=".">37 </td><td char=".">169 </td><td char=".">20 </td><td char=".">96.6 </td><td char=".">0.47 </td><td char=".">55 </td><td char=".">178 </td></tr><tr><td>8 </td><td char=".">273 </td><td char=".">56 </td><td char=".">217 </td><td char=".">20 </td><td char=".">168.0 </td><td char=".">0.62 </td><td char=".">65 </td><td char=".">212 </td></tr><tr><td>9 </td><td char=".">265 </td><td char=".">66 </td><td char=".">199 </td><td char=".">32 </td><td char=".">224.2 </td><td char=".">0.85 </td><td char=".">55 </td><td char=".">173 </td></tr><tr><td>10 </td><td char=".">373 </td><td char=".">96 </td><td char=".">277 </td><td char=".">34 </td><td char=".">244.3 </td><td char=".">0.66 </td><td char=".">80 </td><td char=".">274 </td></tr><tr><td>11 </td><td char=".">401 </td><td char=".">86 </td><td char=".">315 </td><td char=".">25 </td><td char=".">230.1 </td><td char=".">0.58 </td><td char=".">100 </td><td char=".">326 </td></tr><tr><td>12 </td><td char=".">247 </td><td char=".">86 </td><td char=".">161 </td><td char=".">19 </td><td char=".">206.4 </td><td char=".">0.84 </td><td char=".">45 </td><td char=".">146 </td></tr><tr><td>13 </td><td char=".">255 </td><td char=".">58 </td><td char=".">197 </td><td char=".">21 </td><td char=".">165.3 </td><td char=".">0.65 </td><td char=".">50 </td><td char=".">161 </td></tr><tr><td>14 </td><td char=".">285 </td><td char=".">84 </td><td char=".">201 </td><td char=".">33 </td><td char=".">211.4 </td><td char=".">0.74 </td><td char=".">75 </td><td char=".">198 </td></tr><tr><td>15 </td><td char=".">562 </td><td char=".">131 </td><td char=".">431 </td><td char=".">37 </td><td char=".">269.3 </td><td char=".">0.48 </td><td char=".">135 </td><td char=".">471 </td></tr><tr><td>16 </td><td char=".">207 </td><td char=".">76 </td><td char=".">131 </td><td char=".">21 </td><td char=".">180.3 </td><td char=".">0.88 </td><td char=".">50 </td><td char=".">130 </td></tr><tr><td>17 </td><td char=".">334 </td><td char=".">94 </td><td char=".">240 </td><td char=".">32 </td><td char=".">244.5 </td><td char=".">0.73 </td><td char=".">75 </td><td char=".">222 </td></tr><tr><td><strong>Total</strong> </td><td char=".">5,085 </td><td char=".">1,232 </td><td char=".">3,853 </td><td char=".">425 </td><td>3,266 </td><td char=".">0.64 </td><td char=".">1,185 </td><td char=".">3,776 </td></tr></tbody></table></div><div content-id="dsw063-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Number of markers </th><th>Number of SSRs </th><th>Number of SNPs </th><th>Number of common SSR markers </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>Length of the locus cluster </th><th>Marker number in the locus clusters </th></tr></thead><tbody><tr><td>1 </td><td char=".">124 </td><td char=".">38 </td><td char=".">86 </td><td char=".">20 </td><td char=".">147.9 </td><td char=".">1.20 </td><td char=".">15 </td><td char=".">32 </td></tr><tr><td>2 </td><td char=".">367 </td><td char=".">90 </td><td char=".">277 </td><td char=".">29 </td><td char=".">198.6 </td><td char=".">0.54 </td><td char=".">100 </td><td char=".">324 </td></tr><tr><td>3 </td><td char=".">374 </td><td char=".">82 </td><td char=".">292 </td><td char=".">29 </td><td char=".">247.6 </td><td char=".">0.66 </td><td char=".">90 </td><td char=".">278 </td></tr><tr><td>4 </td><td char=".">169 </td><td char=".">51 </td><td char=".">118 </td><td char=".">15 </td><td char=".">87.7 </td><td char=".">0.52 </td><td char=".">55 </td><td char=".">146 </td></tr><tr><td>5 </td><td char=".">325 </td><td char=".">32 </td><td char=".">293 </td><td char=".">19 </td><td char=".">185.5 </td><td char=".">0.57 </td><td char=".">65 </td><td char=".">248 </td></tr><tr><td>6 </td><td char=".">318 </td><td char=".">69 </td><td char=".">249 </td><td char=".">19 </td><td char=".">158.3 </td><td char=".">0.50 </td><td char=".">75 </td><td char=".">257 </td></tr><tr><td>7 </td><td char=".">206 </td><td char=".">37 </td><td char=".">169 </td><td char=".">20 </td><td char=".">96.6 </td><td char=".">0.47 </td><td char=".">55 </td><td char=".">178 </td></tr><tr><td>8 </td><td char=".">273 </td><td char=".">56 </td><td char=".">217 </td><td char=".">20 </td><td char=".">168.0 </td><td char=".">0.62 </td><td char=".">65 </td><td char=".">212 </td></tr><tr><td>9 </td><td char=".">265 </td><td char=".">66 </td><td char=".">199 </td><td char=".">32 </td><td char=".">224.2 </td><td char=".">0.85 </td><td char=".">55 </td><td char=".">173 </td></tr><tr><td>10 </td><td char=".">373 </td><td char=".">96 </td><td char=".">277 </td><td char=".">34 </td><td char=".">244.3 </td><td char=".">0.66 </td><td char=".">80 </td><td char=".">274 </td></tr><tr><td>11 </td><td char=".">401 </td><td char=".">86 </td><td char=".">315 </td><td char=".">25 </td><td char=".">230.1 </td><td char=".">0.58 </td><td char=".">100 </td><td char=".">326 </td></tr><tr><td>12 </td><td char=".">247 </td><td char=".">86 </td><td char=".">161 </td><td char=".">19 </td><td char=".">206.4 </td><td char=".">0.84 </td><td char=".">45 </td><td char=".">146 </td></tr><tr><td>13 </td><td char=".">255 </td><td char=".">58 </td><td char=".">197 </td><td char=".">21 </td><td char=".">165.3 </td><td char=".">0.65 </td><td char=".">50 </td><td char=".">161 </td></tr><tr><td>14 </td><td char=".">285 </td><td char=".">84 </td><td char=".">201 </td><td char=".">33 </td><td char=".">211.4 </td><td char=".">0.74 </td><td char=".">75 </td><td char=".">198 </td></tr><tr><td>15 </td><td char=".">562 </td><td char=".">131 </td><td char=".">431 </td><td char=".">37 </td><td char=".">269.3 </td><td char=".">0.48 </td><td char=".">135 </td><td char=".">471 </td></tr><tr><td>16 </td><td char=".">207 </td><td char=".">76 </td><td char=".">131 </td><td char=".">21 </td><td char=".">180.3 </td><td char=".">0.88 </td><td char=".">50 </td><td char=".">130 </td></tr><tr><td>17 </td><td char=".">334 </td><td char=".">94 </td><td char=".">240 </td><td char=".">32 </td><td char=".">244.5 </td><td char=".">0.73 </td><td char=".">75 </td><td char=".">222 </td></tr><tr><td><strong>Total</strong> </td><td char=".">5,085 </td><td char=".">1,232 </td><td char=".">3,853 </td><td char=".">425 </td><td>3,266 </td><td char=".">0.64 </td><td char=".">1,185 </td><td char=".">3,776 </td></tr></tbody></table></div></div></p>                        <a id="88473423" scrollto-destination="88473423"></a>
<div id="dsw063-F2" data-id="dsw063-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f2.png?Expires=2147483647&Signature=YNzq0E2exF1~1JAJmlfBFTH77GN-Oru9kdY9lYKJM3zKKb3UxqXczaaNc2DqPBr~OcPbodl0ZsiHJPaa0STd9mrek4Yf-nfLS0d-s86t5SzzM6LzlLyy616SYra0605vkk14E12et6EtjjTxlgHT4VyFf8sJTcAeBLalcCBKxItN-1iKDnanZ6L7QWgcstYcutMFu5VaP5sorPtMXEgsF49i1gPRyTOqqvBC~4PFcOB~peth0ocJKmZGQZXK-QEpVcf1yTFxd5kYkJDcF5DeGlLlTVltUzBu-0J-8~PQFoJR2sFzvyM7xhhIRTrPJzkYgJ03HsYO-1LA2EKoXu3XuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of SNP and SSR markers on 17 linkage groups (LGs) of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map. Red bars indicate SSR markers, and black bars indicate SNP markers. X-axis indicates LG number, and Y-axis LG length in centiMorgans (cM). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473423/dsw063f2.tif" path-from-xml="dsw063f2.tif" target="_blank">View large</a><a section="88473423" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f2.png?Expires=2147483647&Signature=duj4k9IInQfUWcj0HUfitduYdsBhxO5bMJgThiMLnGiZzBbKq2iJr-wBgv-Xd0MJFHvBJm2M5IPxwRGu8czx4h47JMuE50zzjEMO7UgVhhTHEG7I3i0YMCcdemEsxjwPyoDUQA7IctlIBxTMYmv7ElwRk352cbduJdlpf8hp2DqRsxjHbPHEQoEQSJfCuFe-N3XBOh7kpEWkKVlIkDKUMgeLtHAaayAVlURAxRtXflVNIJ6UmExaVtUUNCb8oqaBCydew~ntlE1U4nob-PoCh17YKaV3bRYCQnAyj5YY3CHv71Us1d94eZ09GAgMNNko5Q6o79yG~bDy2m4Ocq-9xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473423&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of SNP and SSR markers on 17 linkage groups (LGs) of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map. Red bars indicate SSR markers, and black bars indicate SNP markers. <em>X</em>-axis indicates LG number, and <em>Y</em>-axis LG length in centiMorgans (cM). This figure is available in black and white in print and in colour at DNA Research online.</p></div></div><div content-id="dsw063-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f2.png?Expires=2147483647&Signature=YNzq0E2exF1~1JAJmlfBFTH77GN-Oru9kdY9lYKJM3zKKb3UxqXczaaNc2DqPBr~OcPbodl0ZsiHJPaa0STd9mrek4Yf-nfLS0d-s86t5SzzM6LzlLyy616SYra0605vkk14E12et6EtjjTxlgHT4VyFf8sJTcAeBLalcCBKxItN-1iKDnanZ6L7QWgcstYcutMFu5VaP5sorPtMXEgsF49i1gPRyTOqqvBC~4PFcOB~peth0ocJKmZGQZXK-QEpVcf1yTFxd5kYkJDcF5DeGlLlTVltUzBu-0J-8~PQFoJR2sFzvyM7xhhIRTrPJzkYgJ03HsYO-1LA2EKoXu3XuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of SNP and SSR markers on 17 linkage groups (LGs) of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map. Red bars indicate SSR markers, and black bars indicate SNP markers. X-axis indicates LG number, and Y-axis LG length in centiMorgans (cM). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473423/dsw063f2.tif" path-from-xml="dsw063f2.tif" target="_blank">View large</a><a section="88473423" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f2.png?Expires=2147483647&Signature=duj4k9IInQfUWcj0HUfitduYdsBhxO5bMJgThiMLnGiZzBbKq2iJr-wBgv-Xd0MJFHvBJm2M5IPxwRGu8czx4h47JMuE50zzjEMO7UgVhhTHEG7I3i0YMCcdemEsxjwPyoDUQA7IctlIBxTMYmv7ElwRk352cbduJdlpf8hp2DqRsxjHbPHEQoEQSJfCuFe-N3XBOh7kpEWkKVlIkDKUMgeLtHAaayAVlURAxRtXflVNIJ6UmExaVtUUNCb8oqaBCydew~ntlE1U4nob-PoCh17YKaV3bRYCQnAyj5YY3CHv71Us1d94eZ09GAgMNNko5Q6o79yG~bDy2m4Ocq-9xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473423&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of SNP and SSR markers on 17 linkage groups (LGs) of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map. Red bars indicate SSR markers, and black bars indicate SNP markers. <em>X</em>-axis indicates LG number, and <em>Y</em>-axis LG length in centiMorgans (cM). This figure is available in black and white in print and in colour at DNA Research online.</p></div></div><p>In total, 425 common SSR markers were detected on the nine individual maps, as mentioned above, to construct the I-PCG map (<a reveal-id="dsw063-T3" data-open="dsw063-T3" class="link link-reveal link-table xref-fig">Table 3</a>, <a reveal-id="dsw063-F3" data-open="dsw063-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Of these, 200 (47.1%) were derived from apple, 148 (34.8%) from pear genomic sequences, and 77 (18.1%) from pear EST sequences. There was a large proportion of markers from apple because few SSR markers were developed from pear before the whole genome sequences were released. Common markers on the 17 LGs varied from 15 on LG4 to 37 on LG15. Of these markers, 70% were common between two or three maps, and five markers were common on all nine individual maps: NB113a (LG3), CH03g12-2 (LG3), CH02c11 (LG10), CH02b03b (LG10), and CH04h02 (LG11).</p>                        <a id="88473425" scrollto-destination="88473425"></a>
<div id="dsw063-F3" data-id="dsw063-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f3a.png?Expires=2147483647&amp;Signature=HOM0ei4s3TybX-AsKyMW2GWp75ZbgvcK-pCWYQDZzKtlhS5e~8dktg~hu-eM0FO1X-GbpspFdG-zMTRSMeMV5S-Axq46dyRcK6aA6nD293ujuUH3v4btnKSZ~sX6u8GKUWz3n5cINcJBfNpxy-BHV4xNOs9pwsOOm5vFcxRPm7R10Q9NE9T5pKhqCw90sFWKtNECwrgRjygNnNDN99oN4vs-SOmN1s70X70ci3EnU2uJwUVWTVPC9LkhXC0UbziKKFVVdHv1N0akY0XqPrZQHTGxtg7fTmGVAery2YI4TKMQFqsHceDX0TrT~Q2KKXLgh5sxDr17IWkSiT8HlzX-~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map and the nine individual maps. Red bars on the left denote linkage groups (LGs) of the I-PCG map, and blue/green bars on the right represent the corresponding LGs for individual maps. Blue and green colours indicate forward and reverse orientation, respectively. Purple lines connecting the I-PCG map and individual maps indicate common markers (present in at least two individual maps), and grey lines indicate non-common markers. LG length is relative to total length. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f3a.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473425/dsw063f3a.tif" path-from-xml="dsw063f3a.tif" target="_blank">View large</a><a section="88473425" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f3a.png?Expires=2147483647&amp;Signature=taLRXGkcorPwqhIC132QX2zQKIDeBKSnL3NhW1TnmG7T6xcbeSkwbrAq0F2AWuDgqU1snk3JQJi4-eCJV8j4ES2Os9ghpTcPXMMcOXwMOnTjUBZnl4A~L7LpYdYp3RGkce~mkS8rgtu1mvVRGI7ifaukoC88RO2PbWMchwiu5KfwmCarwXueqWUX1DUw~pZxNkwRurMKF572dC3VR~C2nNI-8I1eruN2Yc3ZelG8xNec~hxrFcCY2HZWX3ocKmh6k2It1nYiLPVOAoUDavF6sYAwCmy3tYbuStVgG-cZTSy66eH1DelCf-AdEUZfxIBOa2NfrPmPgZ7-HVnnVUXZHg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473425&amp;ar=2960056&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw063f3a.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f3b.png?Expires=2147483647&amp;Signature=nAh7C2ce9IddOyDAUsf34rz-GDX7Wq7UglrfuKbru3-FOcReMBiBwsLIW6yyfrLFKXH~heM4Rwr6R7C2PguT8MuUFYyyzPM3D5Iu~Ld~4udFlvFmEzMMux9xyf8XKkO4LfN-ckjuDr9h01yhtsuyfMrnDqVHprKkjww4fW7X0h7uOR5QXOgvQEyggKU5SQkAA2o1Y03cViBGsIC0YiBuaVqVI3pBwX0zf1D2AM-610VPL5JBxB5~6mSZJ3aQVEfnGbzYsO61WbLfk~lO6nGhw8i4iNNbRpzvV0Gp1GN4zGSuvgCTEu-n~4TaPmcEPtp1wVpnqCB5PodbHN26OSx6Ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map and the nine individual maps. Red bars on the left denote linkage groups (LGs) of the I-PCG map, and blue/green bars on the right represent the corresponding LGs for individual maps. Blue and green colours indicate forward and reverse orientation, respectively. Purple lines connecting the I-PCG map and individual maps indicate common markers (present in at least two individual maps), and grey lines indicate non-common markers. LG length is relative to total length. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f3b.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473425/dsw063f3b.tif" path-from-xml="dsw063f3b.tif" target="_blank">View large</a><a section="88473425" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f3b.png?Expires=2147483647&amp;Signature=MrA6H1hBndpWrwxVT8fuHGzlDXyzBn39Vbwwa~TGHv-s2uuYGuIlogVjV8EJph7yUBhGu0kzzbLANLJC20oqIpEl~D4rH1yQg4BgMbKiWilXueg8Y682dYmzwOlmukskv2VXTgxSM5DxCw5BA1ElkCzdIUJrn~o3WOT6c-BmUTetqwiK17oBwt-odg~jtqLXbcoieHNEIB1OFoPvu5LbXFBRl9CcXk0WwXH~Tx-DW3r9JwWUXEa6RyeBJRZm6WoXfEWVCPty2fSyZ27XMf1zBsYsa88ISrtAINtkhtLtgu8KQP9-8NbY-Kjf6AdQJqKuNhTHGnn9zn9zsajk3-pZbA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473425&amp;ar=2960056&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw063f3b.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map and the nine individual maps. Red bars on the left denote linkage groups (LGs) of the I-PCG map, and blue/green bars on the right represent the corresponding LGs for individual maps. Blue and green colours indicate forward and reverse orientation, respectively. Purple lines connecting the I-PCG map and individual maps indicate common markers (present in at least two individual maps), and grey lines indicate non-common markers. LG length is relative to total length. This figure is available in black and white in print and in colour at DNA Research online.</p></div></div><div content-id="dsw063-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f3a.png?Expires=2147483647&amp;Signature=HOM0ei4s3TybX-AsKyMW2GWp75ZbgvcK-pCWYQDZzKtlhS5e~8dktg~hu-eM0FO1X-GbpspFdG-zMTRSMeMV5S-Axq46dyRcK6aA6nD293ujuUH3v4btnKSZ~sX6u8GKUWz3n5cINcJBfNpxy-BHV4xNOs9pwsOOm5vFcxRPm7R10Q9NE9T5pKhqCw90sFWKtNECwrgRjygNnNDN99oN4vs-SOmN1s70X70ci3EnU2uJwUVWTVPC9LkhXC0UbziKKFVVdHv1N0akY0XqPrZQHTGxtg7fTmGVAery2YI4TKMQFqsHceDX0TrT~Q2KKXLgh5sxDr17IWkSiT8HlzX-~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map and the nine individual maps. Red bars on the left denote linkage groups (LGs) of the I-PCG map, and blue/green bars on the right represent the corresponding LGs for individual maps. Blue and green colours indicate forward and reverse orientation, respectively. Purple lines connecting the I-PCG map and individual maps indicate common markers (present in at least two individual maps), and grey lines indicate non-common markers. LG length is relative to total length. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f3a.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473425/dsw063f3a.tif" path-from-xml="dsw063f3a.tif" target="_blank">View large</a><a section="88473425" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f3a.png?Expires=2147483647&amp;Signature=taLRXGkcorPwqhIC132QX2zQKIDeBKSnL3NhW1TnmG7T6xcbeSkwbrAq0F2AWuDgqU1snk3JQJi4-eCJV8j4ES2Os9ghpTcPXMMcOXwMOnTjUBZnl4A~L7LpYdYp3RGkce~mkS8rgtu1mvVRGI7ifaukoC88RO2PbWMchwiu5KfwmCarwXueqWUX1DUw~pZxNkwRurMKF572dC3VR~C2nNI-8I1eruN2Yc3ZelG8xNec~hxrFcCY2HZWX3ocKmh6k2It1nYiLPVOAoUDavF6sYAwCmy3tYbuStVgG-cZTSy66eH1DelCf-AdEUZfxIBOa2NfrPmPgZ7-HVnnVUXZHg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473425&amp;ar=2960056&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw063f3a.tif" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f3b.png?Expires=2147483647&amp;Signature=nAh7C2ce9IddOyDAUsf34rz-GDX7Wq7UglrfuKbru3-FOcReMBiBwsLIW6yyfrLFKXH~heM4Rwr6R7C2PguT8MuUFYyyzPM3D5Iu~Ld~4udFlvFmEzMMux9xyf8XKkO4LfN-ckjuDr9h01yhtsuyfMrnDqVHprKkjww4fW7X0h7uOR5QXOgvQEyggKU5SQkAA2o1Y03cViBGsIC0YiBuaVqVI3pBwX0zf1D2AM-610VPL5JBxB5~6mSZJ3aQVEfnGbzYsO61WbLfk~lO6nGhw8i4iNNbRpzvV0Gp1GN4zGSuvgCTEu-n~4TaPmcEPtp1wVpnqCB5PodbHN26OSx6Ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map and the nine individual maps. Red bars on the left denote linkage groups (LGs) of the I-PCG map, and blue/green bars on the right represent the corresponding LGs for individual maps. Blue and green colours indicate forward and reverse orientation, respectively. Purple lines connecting the I-PCG map and individual maps indicate common markers (present in at least two individual maps), and grey lines indicate non-common markers. LG length is relative to total length. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f3b.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473425/dsw063f3b.tif" path-from-xml="dsw063f3b.tif" target="_blank">View large</a><a section="88473425" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f3b.png?Expires=2147483647&amp;Signature=MrA6H1hBndpWrwxVT8fuHGzlDXyzBn39Vbwwa~TGHv-s2uuYGuIlogVjV8EJph7yUBhGu0kzzbLANLJC20oqIpEl~D4rH1yQg4BgMbKiWilXueg8Y682dYmzwOlmukskv2VXTgxSM5DxCw5BA1ElkCzdIUJrn~o3WOT6c-BmUTetqwiK17oBwt-odg~jtqLXbcoieHNEIB1OFoPvu5LbXFBRl9CcXk0WwXH~Tx-DW3r9JwWUXEa6RyeBJRZm6WoXfEWVCPty2fSyZ27XMf1zBsYsa88ISrtAINtkhtLtgu8KQP9-8NbY-Kjf6AdQJqKuNhTHGnn9zn9zsajk3-pZbA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473425&amp;ar=2960056&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw063f3b.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map and the nine individual maps. Red bars on the left denote linkage groups (LGs) of the I-PCG map, and blue/green bars on the right represent the corresponding LGs for individual maps. Blue and green colours indicate forward and reverse orientation, respectively. Purple lines connecting the I-PCG map and individual maps indicate common markers (present in at least two individual maps), and grey lines indicate non-common markers. LG length is relative to total length. This figure is available in black and white in print and in colour at DNA Research online.</p></div></div>                            <h3 scrollto-destination=88473426 id="88473426" class="section-title" >3.2. Construction of a new GBS SNP map for the ‘OH’ × ‘LBJ’ population</h3>
<p>The new OHxLBJ map, constructed using SNPs derived from the GBS approach, consists of 9,151 markers and spans 1,664.6 cM with an average of 0.18 cM distance between markers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span> for a summary and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span> for a detailed list of markers). The SNPs are located on 1,719 ‘Bartlett’ assembly scaffolds. The information on the LG assignment from the OHxLBJ SNP array map<a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> enabled the assignment of LGs on the new GBS-based OHxLBJ pear genetic map in this study. The lengths of the LGs showed a wide range, from 12.3 cM (LG8) to 255.0 cM (LG17). In total, five LGs were &gt;100 cM, and another three LGs were &lt;50 cM, indicating that some LGs were saturated, while other LGs were partially covered. In addition, we noticed high-level marker co-segregation in this map; for instance, 403 SNP markers on LG1 were located in 24 unique genetic positions, suggesting 16.8 markers per unique genetic position for LG1. Overall, there were 428 unique genetic positions on this GBS-based OHxLBJ map, and on average 21.4 markers per unique genetic position, with an average interval of 4.05 cM.</p>                            <h3 scrollto-destination=88473428 id="88473428" class="section-title" >3.3. Improvement of ‘Bartlett’ genome assembly anchoring by multiple high-density maps</h3>
<p>Since genetic maps were commonly used for anchoring scaffolds into pseudo-chromosomes, we evaluated the potential application of the I-PCG map and three high-density SNP-based pear genetic maps with European pear background to improve the ‘Bartlett’ v1.0 genome assembly. The PH-CG map<a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> consisted of 1,740 markers and spanned 2,773.3 cM (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> for a summary, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> for a detailed list), while the new OHxLBJ consisted of 9,151 markers and spanned 1,664.6 cM (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span> for a summary and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span> for a detailed list). A brief overview of the maps is shown in <a reveal-id="dsw063-T4" data-open="dsw063-T4" class="link link-reveal link-table xref-fig">Table 4</a>. Using these maps, a total of 291.5 Mb ‘Bartlett’ v1.1 genome assembly was obtained in 17 pseudo-chromosomes, increasing the assembly ratio to 50.5% (<a reveal-id="dsw063-T5" data-open="dsw063-T5" class="link link-reveal link-table xref-fig">Table 5</a> for a summary and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span> for a list of scaffold ordering). Pseudo-chromosome 1 (abbreviated as Chr1, same as below) was the shortest (10.3 Mb) and Chr15 was the longest (28.2 Mb), which were the same with the ‘Dangshansuli’ genome: the shortest of 10.7 Mb for Chr1 and the longest of 43.6 Mb for Chr15. In apple, the longest pseudo-chromosome was also Chr15 (55.8 Mb), but the shortest one was Chr16 (23.5 Mb) (<a reveal-id="dsw063-F4" data-open="dsw063-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). <div content-id="dsw063-T4" class="table-modal"><div class="table-wrap"><div id="dsw063-T4" data-id="dsw063-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p>High-density SNP-based pear genetic maps with European pear genetic background used for anchoring the ‘Bartlett’ v1.0 genome scaffolds</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Map ID </th><th>Number of SNPs </th><th>Number of linkage groups </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>References </th></tr></thead><tbody><tr><td>OHxLBJ<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">9,151 </td><td char=".">17 </td><td char=".">1,664.6 </td><td char=".">0.18 </td><td char=".">This study </td></tr><tr><td>BYH<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">2,863 </td><td char=".">17 </td><td char=".">2,080.0 </td><td char=".">0.73 </td><td char="."><a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr">10</a> </td></tr><tr><td>PH-CG<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td char=".">1,740 </td><td char=".">17 </td><td char=".">2,773.3 </td><td char=".">1.61 </td><td char="."><a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr">23</a> </td></tr></tbody></table></div><div content-id="dsw063-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>Map ID </th><th>Number of SNPs </th><th>Number of linkage groups </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>References </th></tr></thead><tbody><tr><td>OHxLBJ<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">9,151 </td><td char=".">17 </td><td char=".">1,664.6 </td><td char=".">0.18 </td><td char=".">This study </td></tr><tr><td>BYH<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">2,863 </td><td char=".">17 </td><td char=".">2,080.0 </td><td char=".">0.73 </td><td char="."><a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr">10</a> </td></tr><tr><td>PH-CG<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td char=".">1,740 </td><td char=".">17 </td><td char=".">2,773.3 </td><td char=".">1.61 </td><td char="."><a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr">23</a> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Population: <sup>a</sup>‘Old Home’ × ‘Louise Bon Jersey’.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>‘Bayuehong’ × ‘Dangshansuli’.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>Consensus genetic map derived from five populations: ‘Old Home’ × ‘Louise Bon Jersey’, PremP003 × ‘Moonglow’ (T003xM), PEAR1 × PEAR2 (T042xT081),<a reveal-id="dsw063-B9" data-open="dsw063-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> POP369 (T052xT003), and POP356 (T052xT064).<a reveal-id="dsw063-B8" data-open="dsw063-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p></div></div></div></div></div><div class="table-wrap"><div id="dsw063-T4" data-id="dsw063-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p>High-density SNP-based pear genetic maps with European pear genetic background used for anchoring the ‘Bartlett’ v1.0 genome scaffolds</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Map ID </th><th>Number of SNPs </th><th>Number of linkage groups </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>References </th></tr></thead><tbody><tr><td>OHxLBJ<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">9,151 </td><td char=".">17 </td><td char=".">1,664.6 </td><td char=".">0.18 </td><td char=".">This study </td></tr><tr><td>BYH<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">2,863 </td><td char=".">17 </td><td char=".">2,080.0 </td><td char=".">0.73 </td><td char="."><a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr">10</a> </td></tr><tr><td>PH-CG<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td char=".">1,740 </td><td char=".">17 </td><td char=".">2,773.3 </td><td char=".">1.61 </td><td char="."><a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr">23</a> </td></tr></tbody></table></div><div content-id="dsw063-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>Map ID </th><th>Number of SNPs </th><th>Number of linkage groups </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>References </th></tr></thead><tbody><tr><td>OHxLBJ<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">9,151 </td><td char=".">17 </td><td char=".">1,664.6 </td><td char=".">0.18 </td><td char=".">This study </td></tr><tr><td>BYH<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td char=".">2,863 </td><td char=".">17 </td><td char=".">2,080.0 </td><td char=".">0.73 </td><td char="."><a reveal-id="dsw063-B10" data-open="dsw063-B10" class="link link-ref link-reveal xref-bibr">10</a> </td></tr><tr><td>PH-CG<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td char=".">1,740 </td><td char=".">17 </td><td char=".">2,773.3 </td><td char=".">1.61 </td><td char="."><a reveal-id="dsw063-B23" data-open="dsw063-B23" class="link link-ref link-reveal xref-bibr">23</a> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Population: <sup>a</sup>‘Old Home’ × ‘Louise Bon Jersey’.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>‘Bayuehong’ × ‘Dangshansuli’.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>Consensus genetic map derived from five populations: ‘Old Home’ × ‘Louise Bon Jersey’, PremP003 × ‘Moonglow’ (T003xM), PEAR1 × PEAR2 (T042xT081),<a reveal-id="dsw063-B9" data-open="dsw063-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> POP369 (T052xT003), and POP356 (T052xT064).<a reveal-id="dsw063-B8" data-open="dsw063-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p></div></div></div></div><div content-id="dsw063-T5" class="table-modal"><div class="table-wrap"><div id="dsw063-T5" data-id="dsw063-T5" class="table-wrap-title"><div class="label -label">Table 5</div><div class="caption"><p>Summary of the new Bartlett v1.1 genome assembly constructed using the ALLMAPS software</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pseudo- chromosome </th><th>Length of pseudo- chromosome (bp) </th><th>Number of scaffolds </th><th>Length of scaffolds (bp) </th><th>Number of markers </th><th>Markers from BYH </th><th>Markers from PH-CG </th><th>Markers from OHxLBJ </th></tr></thead><tbody><tr><td>Chr1 </td><td char=".">10,263,956 </td><td char=".">110 </td><td char=".">10,253,056 </td><td char=".">408 </td><td char=".">2 </td><td char=".">65 </td><td char=".">341 </td></tr><tr><td>Chr2 </td><td char=".">17,147,276 </td><td char=".">141 </td><td char=".">17,133,276 </td><td char=".">618 </td><td char=".">212 </td><td char=".">107 </td><td char=".">299 </td></tr><tr><td>Chr3 </td><td char=".">18,593,297 </td><td char=".">184 </td><td char=".">18,574,997 </td><td char=".">996 </td><td char=".">188 </td><td char=".">89 </td><td char=".">719 </td></tr><tr><td>Chr4 </td><td char=".">14,885,393 </td><td char=".">156 </td><td char=".">14,869,893 </td><td char=".">703 </td><td char=".">95 </td><td char=".">60 </td><td char=".">548 </td></tr><tr><td>Chr5 </td><td char=".">20,478,923 </td><td char=".">210 </td><td char=".">20,458,023 </td><td char=".">847 </td><td char=".">241 </td><td char=".">97 </td><td char=".">509 </td></tr><tr><td>Chr6 </td><td char=".">15,384,309 </td><td char=".">167 </td><td char=".">15,367,709 </td><td char=".">784 </td><td char=".">174 </td><td char=".">79 </td><td char=".">531 </td></tr><tr><td>Chr7 </td><td char=".">16,091,674 </td><td char=".">154 </td><td char=".">16,076,374 </td><td char=".">684 </td><td char=".">110 </td><td char=".">71 </td><td char=".">503 </td></tr><tr><td>Chr8 </td><td char=".">16,073,589 </td><td char=".">130 </td><td char=".">16,060,689 </td><td char=".">650 </td><td char=".">150 </td><td char=".">58 </td><td char=".">442 </td></tr><tr><td>Chr9 </td><td char=".">14,040,893 </td><td char=".">127 </td><td char=".">14,028,293 </td><td char=".">353 </td><td char=".">127 </td><td char=".">82 </td><td char=".">144 </td></tr><tr><td>Chr10 </td><td char=".">20,488,728 </td><td char=".">212 </td><td char=".">20,467,628 </td><td char=".">1,019 </td><td char=".">196 </td><td char=".">105 </td><td char=".">718 </td></tr><tr><td>Chr11 </td><td char=".">19,300,883 </td><td char=".">200 </td><td char=".">19,280,983 </td><td char=".">859 </td><td char=".">209 </td><td char=".">102 </td><td char=".">548 </td></tr><tr><td>Chr12 </td><td char=".">16,230,967 </td><td char=".">154 </td><td char=".">16,215,667 </td><td char=".">766 </td><td char=".">144 </td><td char=".">80 </td><td char=".">542 </td></tr><tr><td>Chr13 </td><td char=".">15,861,480 </td><td char=".">113 </td><td char=".">15,850,280 </td><td char=".">619 </td><td char=".">126 </td><td char=".">87 </td><td char=".">406 </td></tr><tr><td>Chr14 </td><td char=".">14,746,025 </td><td char=".">131 </td><td char=".">14,733,025 </td><td char=".">553 </td><td char=".">146 </td><td char=".">53 </td><td char=".">354 </td></tr><tr><td>Chr15 </td><td char=".">28,200,744 </td><td char=".">212 </td><td char=".">28,179,644 </td><td char=".">1,075 </td><td char=".">316 </td><td char=".">127 </td><td char=".">632 </td></tr><tr><td>Chr16 </td><td char=".">15,009,554 </td><td char=".">146 </td><td char=".">14,995,054 </td><td char=".">554 </td><td char=".">82 </td><td char=".">61 </td><td char=".">411 </td></tr><tr><td>Chr17 </td><td char=".">19,000,797 </td><td char=".">173 </td><td char=".">18,983,597 </td><td char=".">765 </td><td char=".">150 </td><td char=".">81 </td><td char=".">534 </td></tr><tr><td>Sum </td><td char=".">291,798,488 </td><td char=".">2,720 </td><td char=".">291,528,188 (50.5%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">12,253 </td><td char=".">2,668 </td><td char=".">1,404 </td><td char=".">8,181 </td></tr></tbody></table></div><div content-id="dsw063-T5" class="table-modal"><table><thead name="thead" align=""><tr><th>Pseudo- chromosome </th><th>Length of pseudo- chromosome (bp) </th><th>Number of scaffolds </th><th>Length of scaffolds (bp) </th><th>Number of markers </th><th>Markers from BYH </th><th>Markers from PH-CG </th><th>Markers from OHxLBJ </th></tr></thead><tbody><tr><td>Chr1 </td><td char=".">10,263,956 </td><td char=".">110 </td><td char=".">10,253,056 </td><td char=".">408 </td><td char=".">2 </td><td char=".">65 </td><td char=".">341 </td></tr><tr><td>Chr2 </td><td char=".">17,147,276 </td><td char=".">141 </td><td char=".">17,133,276 </td><td char=".">618 </td><td char=".">212 </td><td char=".">107 </td><td char=".">299 </td></tr><tr><td>Chr3 </td><td char=".">18,593,297 </td><td char=".">184 </td><td char=".">18,574,997 </td><td char=".">996 </td><td char=".">188 </td><td char=".">89 </td><td char=".">719 </td></tr><tr><td>Chr4 </td><td char=".">14,885,393 </td><td char=".">156 </td><td char=".">14,869,893 </td><td char=".">703 </td><td char=".">95 </td><td char=".">60 </td><td char=".">548 </td></tr><tr><td>Chr5 </td><td char=".">20,478,923 </td><td char=".">210 </td><td char=".">20,458,023 </td><td char=".">847 </td><td char=".">241 </td><td char=".">97 </td><td char=".">509 </td></tr><tr><td>Chr6 </td><td char=".">15,384,309 </td><td char=".">167 </td><td char=".">15,367,709 </td><td char=".">784 </td><td char=".">174 </td><td char=".">79 </td><td char=".">531 </td></tr><tr><td>Chr7 </td><td char=".">16,091,674 </td><td char=".">154 </td><td char=".">16,076,374 </td><td char=".">684 </td><td char=".">110 </td><td char=".">71 </td><td char=".">503 </td></tr><tr><td>Chr8 </td><td char=".">16,073,589 </td><td char=".">130 </td><td char=".">16,060,689 </td><td char=".">650 </td><td char=".">150 </td><td char=".">58 </td><td char=".">442 </td></tr><tr><td>Chr9 </td><td char=".">14,040,893 </td><td char=".">127 </td><td char=".">14,028,293 </td><td char=".">353 </td><td char=".">127 </td><td char=".">82 </td><td char=".">144 </td></tr><tr><td>Chr10 </td><td char=".">20,488,728 </td><td char=".">212 </td><td char=".">20,467,628 </td><td char=".">1,019 </td><td char=".">196 </td><td char=".">105 </td><td char=".">718 </td></tr><tr><td>Chr11 </td><td char=".">19,300,883 </td><td char=".">200 </td><td char=".">19,280,983 </td><td char=".">859 </td><td char=".">209 </td><td char=".">102 </td><td char=".">548 </td></tr><tr><td>Chr12 </td><td char=".">16,230,967 </td><td char=".">154 </td><td char=".">16,215,667 </td><td char=".">766 </td><td char=".">144 </td><td char=".">80 </td><td char=".">542 </td></tr><tr><td>Chr13 </td><td char=".">15,861,480 </td><td char=".">113 </td><td char=".">15,850,280 </td><td char=".">619 </td><td char=".">126 </td><td char=".">87 </td><td char=".">406 </td></tr><tr><td>Chr14 </td><td char=".">14,746,025 </td><td char=".">131 </td><td char=".">14,733,025 </td><td char=".">553 </td><td char=".">146 </td><td char=".">53 </td><td char=".">354 </td></tr><tr><td>Chr15 </td><td char=".">28,200,744 </td><td char=".">212 </td><td char=".">28,179,644 </td><td char=".">1,075 </td><td char=".">316 </td><td char=".">127 </td><td char=".">632 </td></tr><tr><td>Chr16 </td><td char=".">15,009,554 </td><td char=".">146 </td><td char=".">14,995,054 </td><td char=".">554 </td><td char=".">82 </td><td char=".">61 </td><td char=".">411 </td></tr><tr><td>Chr17 </td><td char=".">19,000,797 </td><td char=".">173 </td><td char=".">18,983,597 </td><td char=".">765 </td><td char=".">150 </td><td char=".">81 </td><td char=".">534 </td></tr><tr><td>Sum </td><td char=".">291,798,488 </td><td char=".">2,720 </td><td char=".">291,528,188 (50.5%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">12,253 </td><td char=".">2,668 </td><td char=".">1,404 </td><td char=".">8,181 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p><sup>a</sup>The total size of ‘Bartlett’ v1.0 (<em>Pyrus communis</em>) scaffold sequences was 577,335,413 bp.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw063-T5" data-id="dsw063-T5" class="table-wrap-title"><div class="label -label">Table 5</div><div class="caption"><p>Summary of the new Bartlett v1.1 genome assembly constructed using the ALLMAPS software</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pseudo- chromosome </th><th>Length of pseudo- chromosome (bp) </th><th>Number of scaffolds </th><th>Length of scaffolds (bp) </th><th>Number of markers </th><th>Markers from BYH </th><th>Markers from PH-CG </th><th>Markers from OHxLBJ </th></tr></thead><tbody><tr><td>Chr1 </td><td char=".">10,263,956 </td><td char=".">110 </td><td char=".">10,253,056 </td><td char=".">408 </td><td char=".">2 </td><td char=".">65 </td><td char=".">341 </td></tr><tr><td>Chr2 </td><td char=".">17,147,276 </td><td char=".">141 </td><td char=".">17,133,276 </td><td char=".">618 </td><td char=".">212 </td><td char=".">107 </td><td char=".">299 </td></tr><tr><td>Chr3 </td><td char=".">18,593,297 </td><td char=".">184 </td><td char=".">18,574,997 </td><td char=".">996 </td><td char=".">188 </td><td char=".">89 </td><td char=".">719 </td></tr><tr><td>Chr4 </td><td char=".">14,885,393 </td><td char=".">156 </td><td char=".">14,869,893 </td><td char=".">703 </td><td char=".">95 </td><td char=".">60 </td><td char=".">548 </td></tr><tr><td>Chr5 </td><td char=".">20,478,923 </td><td char=".">210 </td><td char=".">20,458,023 </td><td char=".">847 </td><td char=".">241 </td><td char=".">97 </td><td char=".">509 </td></tr><tr><td>Chr6 </td><td char=".">15,384,309 </td><td char=".">167 </td><td char=".">15,367,709 </td><td char=".">784 </td><td char=".">174 </td><td char=".">79 </td><td char=".">531 </td></tr><tr><td>Chr7 </td><td char=".">16,091,674 </td><td char=".">154 </td><td char=".">16,076,374 </td><td char=".">684 </td><td char=".">110 </td><td char=".">71 </td><td char=".">503 </td></tr><tr><td>Chr8 </td><td char=".">16,073,589 </td><td char=".">130 </td><td char=".">16,060,689 </td><td char=".">650 </td><td char=".">150 </td><td char=".">58 </td><td char=".">442 </td></tr><tr><td>Chr9 </td><td char=".">14,040,893 </td><td char=".">127 </td><td char=".">14,028,293 </td><td char=".">353 </td><td char=".">127 </td><td char=".">82 </td><td char=".">144 </td></tr><tr><td>Chr10 </td><td char=".">20,488,728 </td><td char=".">212 </td><td char=".">20,467,628 </td><td char=".">1,019 </td><td char=".">196 </td><td char=".">105 </td><td char=".">718 </td></tr><tr><td>Chr11 </td><td char=".">19,300,883 </td><td char=".">200 </td><td char=".">19,280,983 </td><td char=".">859 </td><td char=".">209 </td><td char=".">102 </td><td char=".">548 </td></tr><tr><td>Chr12 </td><td char=".">16,230,967 </td><td char=".">154 </td><td char=".">16,215,667 </td><td char=".">766 </td><td char=".">144 </td><td char=".">80 </td><td char=".">542 </td></tr><tr><td>Chr13 </td><td char=".">15,861,480 </td><td char=".">113 </td><td char=".">15,850,280 </td><td char=".">619 </td><td char=".">126 </td><td char=".">87 </td><td char=".">406 </td></tr><tr><td>Chr14 </td><td char=".">14,746,025 </td><td char=".">131 </td><td char=".">14,733,025 </td><td char=".">553 </td><td char=".">146 </td><td char=".">53 </td><td char=".">354 </td></tr><tr><td>Chr15 </td><td char=".">28,200,744 </td><td char=".">212 </td><td char=".">28,179,644 </td><td char=".">1,075 </td><td char=".">316 </td><td char=".">127 </td><td char=".">632 </td></tr><tr><td>Chr16 </td><td char=".">15,009,554 </td><td char=".">146 </td><td char=".">14,995,054 </td><td char=".">554 </td><td char=".">82 </td><td char=".">61 </td><td char=".">411 </td></tr><tr><td>Chr17 </td><td char=".">19,000,797 </td><td char=".">173 </td><td char=".">18,983,597 </td><td char=".">765 </td><td char=".">150 </td><td char=".">81 </td><td char=".">534 </td></tr><tr><td>Sum </td><td char=".">291,798,488 </td><td char=".">2,720 </td><td char=".">291,528,188 (50.5%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">12,253 </td><td char=".">2,668 </td><td char=".">1,404 </td><td char=".">8,181 </td></tr></tbody></table></div><div content-id="dsw063-T5" class="table-modal"><table><thead name="thead" align=""><tr><th>Pseudo- chromosome </th><th>Length of pseudo- chromosome (bp) </th><th>Number of scaffolds </th><th>Length of scaffolds (bp) </th><th>Number of markers </th><th>Markers from BYH </th><th>Markers from PH-CG </th><th>Markers from OHxLBJ </th></tr></thead><tbody><tr><td>Chr1 </td><td char=".">10,263,956 </td><td char=".">110 </td><td char=".">10,253,056 </td><td char=".">408 </td><td char=".">2 </td><td char=".">65 </td><td char=".">341 </td></tr><tr><td>Chr2 </td><td char=".">17,147,276 </td><td char=".">141 </td><td char=".">17,133,276 </td><td char=".">618 </td><td char=".">212 </td><td char=".">107 </td><td char=".">299 </td></tr><tr><td>Chr3 </td><td char=".">18,593,297 </td><td char=".">184 </td><td char=".">18,574,997 </td><td char=".">996 </td><td char=".">188 </td><td char=".">89 </td><td char=".">719 </td></tr><tr><td>Chr4 </td><td char=".">14,885,393 </td><td char=".">156 </td><td char=".">14,869,893 </td><td char=".">703 </td><td char=".">95 </td><td char=".">60 </td><td char=".">548 </td></tr><tr><td>Chr5 </td><td char=".">20,478,923 </td><td char=".">210 </td><td char=".">20,458,023 </td><td char=".">847 </td><td char=".">241 </td><td char=".">97 </td><td char=".">509 </td></tr><tr><td>Chr6 </td><td char=".">15,384,309 </td><td char=".">167 </td><td char=".">15,367,709 </td><td char=".">784 </td><td char=".">174 </td><td char=".">79 </td><td char=".">531 </td></tr><tr><td>Chr7 </td><td char=".">16,091,674 </td><td char=".">154 </td><td char=".">16,076,374 </td><td char=".">684 </td><td char=".">110 </td><td char=".">71 </td><td char=".">503 </td></tr><tr><td>Chr8 </td><td char=".">16,073,589 </td><td char=".">130 </td><td char=".">16,060,689 </td><td char=".">650 </td><td char=".">150 </td><td char=".">58 </td><td char=".">442 </td></tr><tr><td>Chr9 </td><td char=".">14,040,893 </td><td char=".">127 </td><td char=".">14,028,293 </td><td char=".">353 </td><td char=".">127 </td><td char=".">82 </td><td char=".">144 </td></tr><tr><td>Chr10 </td><td char=".">20,488,728 </td><td char=".">212 </td><td char=".">20,467,628 </td><td char=".">1,019 </td><td char=".">196 </td><td char=".">105 </td><td char=".">718 </td></tr><tr><td>Chr11 </td><td char=".">19,300,883 </td><td char=".">200 </td><td char=".">19,280,983 </td><td char=".">859 </td><td char=".">209 </td><td char=".">102 </td><td char=".">548 </td></tr><tr><td>Chr12 </td><td char=".">16,230,967 </td><td char=".">154 </td><td char=".">16,215,667 </td><td char=".">766 </td><td char=".">144 </td><td char=".">80 </td><td char=".">542 </td></tr><tr><td>Chr13 </td><td char=".">15,861,480 </td><td char=".">113 </td><td char=".">15,850,280 </td><td char=".">619 </td><td char=".">126 </td><td char=".">87 </td><td char=".">406 </td></tr><tr><td>Chr14 </td><td char=".">14,746,025 </td><td char=".">131 </td><td char=".">14,733,025 </td><td char=".">553 </td><td char=".">146 </td><td char=".">53 </td><td char=".">354 </td></tr><tr><td>Chr15 </td><td char=".">28,200,744 </td><td char=".">212 </td><td char=".">28,179,644 </td><td char=".">1,075 </td><td char=".">316 </td><td char=".">127 </td><td char=".">632 </td></tr><tr><td>Chr16 </td><td char=".">15,009,554 </td><td char=".">146 </td><td char=".">14,995,054 </td><td char=".">554 </td><td char=".">82 </td><td char=".">61 </td><td char=".">411 </td></tr><tr><td>Chr17 </td><td char=".">19,000,797 </td><td char=".">173 </td><td char=".">18,983,597 </td><td char=".">765 </td><td char=".">150 </td><td char=".">81 </td><td char=".">534 </td></tr><tr><td>Sum </td><td char=".">291,798,488 </td><td char=".">2,720 </td><td char=".">291,528,188 (50.5%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">12,253 </td><td char=".">2,668 </td><td char=".">1,404 </td><td char=".">8,181 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p><sup>a</sup>The total size of ‘Bartlett’ v1.0 (<em>Pyrus communis</em>) scaffold sequences was 577,335,413 bp.</p></div></div></div></div></p>                        <a id="88473430" scrollto-destination="88473430"></a>
<div id="dsw063-F4" data-id="dsw063-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f4.png?Expires=2147483647&Signature=yE~GjV2ZyWF267u8eN2NHrCIy02dquJd5mRjTUad1LeZFo6rK6YGatOtda93pjlXIH36fuR89yIV69ZtKJrFZPtb17EsafYSp07GKNraNriTh61AWf8wJZQ1EfoRntMNEhv8kLqiEJDLXO2sINqLeQkEShoQiANy3lH~LZS9YX6ALOEjh-oZNbgaPbnkucMQo-VQoV3ULpX~Nqnw1AC~7R9HbmiD1vtSjgxRODLFT5og4KbxJTtdD4RPaZVZsp1~VHMteDcwyjPhHFoz80n~FDSbV7e08KvcJmgZBd5Pzt-xjRlBbP3qKQS5QNN3rnjflB8-w3EFiHvKnn8xmwA3FA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of pseudo-chromosome length of ‘Bartlett’ (P. communis), ‘Dangshansuli’ (P. bretschneideri), and ‘Golden Delicious’ (Malus × domestica) genome assemblies. X-axis means pseudo-chromosome size, million bases (Mb) as unit, and Y-axis means chromosome number." path-from-xml="dsw063f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473430/dsw063f4.tif" path-from-xml="dsw063f4.tif" target="_blank">View large</a><a section="88473430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f4.png?Expires=2147483647&Signature=VyPRtFXuDMhB~S-kYmP9LngYgm-gQcdqkHVPph~hf42IxCB7yIl7EWGZYacuWLTcj8jFxQGoMWPp2kJBEcLEm2S992iCamnsyTc0AmkdoiRbi1z6FItZlzx3VPUfj5s1RjubR2g6NJlg3~IGebQadjE8Cz9VX0YMBA5H~8G9MVnexxYG0J1H6F8KIArgP8v76t-jOfek06eOMVPQw9uC4L5uU0KyxCTZx5wfDASzsTDT6t9si8eMLPu5jgVPvpd~77e7GvldT0jzJWThAQztn80DdT~DNTN42Vt6-VSEor-G3ORmWt9LVUmHMMvYYbZ9ss6wQVoE~V-8BtO4KrLeWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473430&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of pseudo-chromosome length of ‘Bartlett’ (<em>P. communis</em>), ‘Dangshansuli’ (<em>P. bretschneideri</em>), and ‘Golden Delicious’ (<em>Malus</em> × <em>domestica</em>) genome assemblies. <em>X</em>-axis means pseudo-chromosome size, million bases (Mb) as unit, and <em>Y</em>-axis means chromosome number.</p></div></div><div content-id="dsw063-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f4.png?Expires=2147483647&Signature=yE~GjV2ZyWF267u8eN2NHrCIy02dquJd5mRjTUad1LeZFo6rK6YGatOtda93pjlXIH36fuR89yIV69ZtKJrFZPtb17EsafYSp07GKNraNriTh61AWf8wJZQ1EfoRntMNEhv8kLqiEJDLXO2sINqLeQkEShoQiANy3lH~LZS9YX6ALOEjh-oZNbgaPbnkucMQo-VQoV3ULpX~Nqnw1AC~7R9HbmiD1vtSjgxRODLFT5og4KbxJTtdD4RPaZVZsp1~VHMteDcwyjPhHFoz80n~FDSbV7e08KvcJmgZBd5Pzt-xjRlBbP3qKQS5QNN3rnjflB8-w3EFiHvKnn8xmwA3FA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of pseudo-chromosome length of ‘Bartlett’ (P. communis), ‘Dangshansuli’ (P. bretschneideri), and ‘Golden Delicious’ (Malus × domestica) genome assemblies. X-axis means pseudo-chromosome size, million bases (Mb) as unit, and Y-axis means chromosome number." path-from-xml="dsw063f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473430/dsw063f4.tif" path-from-xml="dsw063f4.tif" target="_blank">View large</a><a section="88473430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f4.png?Expires=2147483647&Signature=VyPRtFXuDMhB~S-kYmP9LngYgm-gQcdqkHVPph~hf42IxCB7yIl7EWGZYacuWLTcj8jFxQGoMWPp2kJBEcLEm2S992iCamnsyTc0AmkdoiRbi1z6FItZlzx3VPUfj5s1RjubR2g6NJlg3~IGebQadjE8Cz9VX0YMBA5H~8G9MVnexxYG0J1H6F8KIArgP8v76t-jOfek06eOMVPQw9uC4L5uU0KyxCTZx5wfDASzsTDT6t9si8eMLPu5jgVPvpd~77e7GvldT0jzJWThAQztn80DdT~DNTN42Vt6-VSEor-G3ORmWt9LVUmHMMvYYbZ9ss6wQVoE~V-8BtO4KrLeWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473430&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of pseudo-chromosome length of ‘Bartlett’ (<em>P. communis</em>), ‘Dangshansuli’ (<em>P. bretschneideri</em>), and ‘Golden Delicious’ (<em>Malus</em> × <em>domestica</em>) genome assemblies. <em>X</em>-axis means pseudo-chromosome size, million bases (Mb) as unit, and <em>Y</em>-axis means chromosome number.</p></div></div><p>Synteny analyses for the new ‘Bartlett’ v1.1 genome assembly against ‘Dangshansuli’ and ‘Golden Delicious’ genomes were performed (<a reveal-id="dsw063-F5" data-open="dsw063-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Intra-genome synteny analyses of ‘Bartlett’ genome (<a reveal-id="dsw063-F5" data-open="dsw063-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>) identified homologous pseudo-chromosome pairs as per previous reports<a reveal-id="dsw063-B22" data-open="dsw063-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw063-B40" data-open="dsw063-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a>: 1–7, 2–7, 2–15, 3–11, 4–12, 5–10, 6–14, 8–15, 9–17, 12–14 and 13–16. Additionally, we identified four segmental duplication regions for pseudo-chromosome pairs 6–13, 9–13, 14–16 and 14–17, which were also confirmed by inter-genome synteny analyses between the ‘Bartlett’ pear and ‘Golden Delicious’ apple genomes. Inter-genome synteny analyses between both pear genomes of ‘Bartlett’ and ‘Dangshansuli’ (<a reveal-id="dsw063-F5" data-open="dsw063-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>) showed good collinearity. The near-linear arrangement of homologous regions for all 17 corresponding pseudo-chromosomes was observed. Nonetheless, the arrangements of Chr1, Chr2, Chr3 and Chr5 were not as well aligned as other pseudo-chromosomes. Interestingly, inter-genome synteny analyses between ‘Bartlett’ and the apple genome of ‘Golden Delicious’ (<a reveal-id="dsw063-F5" data-open="dsw063-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>), both in the subfamily Maloideae, showed not only good collinearity, but also homologous relationships, similar to the ‘Bartlett’ intra-genome synteny analyses. This phenomenon was also observed when inter-genome synteny analysis was performed between ‘Dangshansuli’ and ‘Golden Delicious’ (data not shown), indicating the unique genome structure for apple.</p>                        <a id="88473432" scrollto-destination="88473432"></a>
<div id="dsw063-F5" data-id="dsw063-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f5.png?Expires=2147483647&Signature=t267bW~S-2~oIInuFKMTk8X8Off63Nqc9vv4C~EzSynNBZ7hxEmh-O-LJblJ-iVEvlJPMuwI7MSSNu40edVrceFp83zawSb8EetBSBxXFexdqDqcN2neDQ--TkUqYA~Twc9qwnxqM~ashMHxaGKry-kTLYIUvOrg2nULMpZQDkfQdgIankpCsEuQ6uZRuTjzDCzJT9IQEc-L7VX8x69YjWYVjKsV~wnec5xgaVmee-a4NYMPi6Weo-tyCGSNacUjRZYBa3Lkg3asZg3GGKoEwcea1A-J6X75JmvSXnKp1HYgeQ-xOYi7TV7y7cllosk1dtegWkAbhXgUyqLI~-vBzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Intra-genome synteny for the ‘Bartlett’ (P. communis) v1.1 pseudo-chromosomes; (B) Inter-genome synteny for the ‘Bartlett’ v1.1 and ‘Dangshansuli’ (P. bretschneideri) pseudo-chromosomes; (C) Inter-genome synteny for the ‘Bartlett’ v1,1 and ‘Golden Delicious’ (Malus × domestica) pseudo-chromosomes. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473432/dsw063f5.tif" path-from-xml="dsw063f5.tif" target="_blank">View large</a><a section="88473432" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f5.png?Expires=2147483647&Signature=zVaWULgpRP40OEVGomA5liNuwAEo--ZyihL6CJqP8lxHJRnkm7hrDtcNCr1UeN9QpIjRdA8zWL5gNnMGie04au-~k~voLoz0DncgpBthvCJLkPayiHwx~ZHZW0OhibpGpfOejzr-1z8QeRoejp1Oeah2m~oDvy3ySJkOG3P9gwIVZiCSc8amh-unmQrBvULaOgJydhf4AUNMDVCliJ7lcqqkDPurJ4gJsHdCAP~DAcdCNbXNa~heN2DbdWJwrCX7AdOZgAo73WWz-YUo3wBvrSiZkIyL-YRYuPwrrhP8KDvWl6YPhrnrj~njvnlESSzjNo2o5j8~VcsImC6Jr01BwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473432&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Intra-genome synteny for the ‘Bartlett’ (<em>P. communis</em>) v1.1 pseudo-chromosomes; (B) Inter-genome synteny for the ‘Bartlett’ v1.1 and ‘Dangshansuli’ (<em>P. bretschneideri</em>) pseudo-chromosomes; (C) Inter-genome synteny for the ‘Bartlett’ v1,1 and ‘Golden Delicious’ (<em>Malus</em> × <em>domestica</em>) pseudo-chromosomes. This figure is available in black and white in print and in colour at DNA Research online.</p></div></div><div content-id="dsw063-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f5.png?Expires=2147483647&Signature=t267bW~S-2~oIInuFKMTk8X8Off63Nqc9vv4C~EzSynNBZ7hxEmh-O-LJblJ-iVEvlJPMuwI7MSSNu40edVrceFp83zawSb8EetBSBxXFexdqDqcN2neDQ--TkUqYA~Twc9qwnxqM~ashMHxaGKry-kTLYIUvOrg2nULMpZQDkfQdgIankpCsEuQ6uZRuTjzDCzJT9IQEc-L7VX8x69YjWYVjKsV~wnec5xgaVmee-a4NYMPi6Weo-tyCGSNacUjRZYBa3Lkg3asZg3GGKoEwcea1A-J6X75JmvSXnKp1HYgeQ-xOYi7TV7y7cllosk1dtegWkAbhXgUyqLI~-vBzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Intra-genome synteny for the ‘Bartlett’ (P. communis) v1.1 pseudo-chromosomes; (B) Inter-genome synteny for the ‘Bartlett’ v1.1 and ‘Dangshansuli’ (P. bretschneideri) pseudo-chromosomes; (C) Inter-genome synteny for the ‘Bartlett’ v1,1 and ‘Golden Delicious’ (Malus × domestica) pseudo-chromosomes. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473432/dsw063f5.tif" path-from-xml="dsw063f5.tif" target="_blank">View large</a><a section="88473432" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f5.png?Expires=2147483647&Signature=zVaWULgpRP40OEVGomA5liNuwAEo--ZyihL6CJqP8lxHJRnkm7hrDtcNCr1UeN9QpIjRdA8zWL5gNnMGie04au-~k~voLoz0DncgpBthvCJLkPayiHwx~ZHZW0OhibpGpfOejzr-1z8QeRoejp1Oeah2m~oDvy3ySJkOG3P9gwIVZiCSc8amh-unmQrBvULaOgJydhf4AUNMDVCliJ7lcqqkDPurJ4gJsHdCAP~DAcdCNbXNa~heN2DbdWJwrCX7AdOZgAo73WWz-YUo3wBvrSiZkIyL-YRYuPwrrhP8KDvWl6YPhrnrj~njvnlESSzjNo2o5j8~VcsImC6Jr01BwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473432&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Intra-genome synteny for the ‘Bartlett’ (<em>P. communis</em>) v1.1 pseudo-chromosomes; (B) Inter-genome synteny for the ‘Bartlett’ v1.1 and ‘Dangshansuli’ (<em>P. bretschneideri</em>) pseudo-chromosomes; (C) Inter-genome synteny for the ‘Bartlett’ v1,1 and ‘Golden Delicious’ (<em>Malus</em> × <em>domestica</em>) pseudo-chromosomes. This figure is available in black and white in print and in colour at DNA Research online.</p></div></div>                            <h3 scrollto-destination=88473433 id="88473433" class="section-title" >3.4. Comparison of the I-PCG map with genome assemblies of pear and apple revealed both collinearity and misalignments</h3>
<p>To further evaluate the collinearity between the pear genetic and physical maps, we compared the I-PCG map with the new ‘Bartlett’ v 1.1 (<a reveal-id="dsw063-F6" data-open="dsw063-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>), ‘Dangshansuli’ and apple genome assemblies (<a reveal-id="dsw063-F6" data-open="dsw063-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>). In total, 4,036, 4,157 and 3,707 markers were mapped onto the ‘Bartlett’ ‘Dangshansuli’ and ‘Golden Delicious’ genome assemblies, and among these, 3,622, 4,069 and 3,707 markers were mapped on pseudo-chromosomes, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). As shown in <a reveal-id="dsw063-F6" data-open="dsw063-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>, the I-PCG map and the new ‘Bartlett’ v1.1 genome assembly displayed good collinearity. The uniformity of the genetic map with the physical maps could also be observed in the ‘Dangshansuli’ and ‘Golden Delicious’ (<a reveal-id="dsw063-F6" data-open="dsw063-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>), but with more misalignments (markers were not located on corresponding pseudo-chromosomes). For ‘Bartlett’, 3,427 markers (94.6% of markers mapped on pseudo-chromosomes) were located in the corresponding pseudo-chromosomes, 3,649 markers (89.7%) for ‘Dangshansuli’, and 2,117 markers (57.1%) for ‘Golden Delicious’. Among these, 3,414 markers were located in corresponding pseudo-chromosomes in at least two of the above three genome assemblies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Furthermore, misaligned markers between the I-PCG map and ‘Bartlett’ genome assembly were enriched in five LG/Chr pairs (at least five markers for each LG/Chr pair): 1–7, 9–17, 3–11, 13–16 and 6–14. For the ‘Dangshansuli’ genome, they were enriched in six LG/Chr pairs (at least 17 markers for each LG/Chr pair): 1–7, 11–15, 8–15, 3–11, 5–11 and 13–16. For ‘Golden Delicious’, they were enriched in nine LG/Chr pairs (at least 41 markers for each LG/Chr pair): 5–10, 3–11, 1–7, 9–17, 13–16, 8–15, 2–15, 6–14 and 1–15. Overall, comparing the I-PCG map with the ‘Dangshansuli’, ‘Bartlett’ and ‘Golden Delicious’ assemblies confirmed the reliability of the consensus genetic map.</p>                        <a id="88473435" scrollto-destination="88473435"></a>
<div id="dsw063-F6" data-id="dsw063-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f6.png?Expires=2147483647&Signature=ukuvZdW63L4YlWbVCH5wgpYRYB15jh4gFWCWoP7pYoU8IWKlcT8th100R8zhw3AcZKL12Z84UFiBjPMzcSE2ED~WOSfpmylk7nENOP9eVCXWsLhr~jPqFE37YSYjJWeEhrLmsGuhHkBhdMwe-Flxe8lfNppSfIYgYBvkF91aMzLW8q8x-j0BJAIDMRWGZoucr8p8VapduDYmYmSD9jBxU-vilu5RJbqJzfy09UIQRj7WKkY~dDHHbfrL5GNO2B661QhxZFqLHQe9CPP82ezJxmy-IEejSgpR9ro9YuAxqq0KqZ8i67RqbnhHcbKmBnNBnVndNRfWKdiPd4Ygv90n6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map with (A) ‘Bartlett’ pseudo-chromosomes (Pc), (B) ‘Dangshansuli’ (Pb) and apple (Md) pseudo-chromosomes. Red bars indicate linkage groups (LGs) of the I-PCG map. Blue and green bars indicate forward and reverse orientation LGs, respectively. LG length is relative to total length. The yellow lines indicate markers located in the same linkage group/pseudo-chromosome number and the grey lines indicate markers in different linkage group/pseudo-chromosome number. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473435/dsw063f6.tif" path-from-xml="dsw063f6.tif" target="_blank">View large</a><a section="88473435" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f6.png?Expires=2147483647&Signature=4C0G566ZSeJf8b3d~uFYqyuoIvmg8GL~vkVdSwgAFFIYrFQG4c4OUhSUs50~YRQLHLyRlE~Zn2xy6bGlLBLcI5K8vDUh76ZgzzPerIZ8deWhjCEyj~W3Ddg9FsAtTks3dWjIuvD2uDwVVGz~Vc4Gqezi6~jNKvy70zFUgTrjYK0Mj92UeJds024W4p~xiRiuNIknRVCJP3mxkexoFtjw~pMUUnJWqfximJwT30nuX2385BlOuq4hoGc~LNkCOIHEe1OwsjJmEpljik5kRCoYj4AcCJ4ON-wGEegs32QFTK8klLrca2T8GHvjfh8iAPOsxQDJN7lCwLv2CGYECpJ2rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473435&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map with (A) ‘Bartlett’ pseudo-chromosomes (Pc), (B) ‘Dangshansuli’ (Pb) and apple (Md) pseudo-chromosomes. Red bars indicate linkage groups (LGs) of the I-PCG map. Blue and green bars indicate forward and reverse orientation LGs, respectively. LG length is relative to total length. The yellow lines indicate markers located in the same linkage group/pseudo-chromosome number and the grey lines indicate markers in different linkage group/pseudo-chromosome number. This figure is available in black and white in print and in colour at DNA Research online.</p></div></div><div content-id="dsw063-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/m_dsw063f6.png?Expires=2147483647&Signature=ukuvZdW63L4YlWbVCH5wgpYRYB15jh4gFWCWoP7pYoU8IWKlcT8th100R8zhw3AcZKL12Z84UFiBjPMzcSE2ED~WOSfpmylk7nENOP9eVCXWsLhr~jPqFE37YSYjJWeEhrLmsGuhHkBhdMwe-Flxe8lfNppSfIYgYBvkF91aMzLW8q8x-j0BJAIDMRWGZoucr8p8VapduDYmYmSD9jBxU-vilu5RJbqJzfy09UIQRj7WKkY~dDHHbfrL5GNO2B661QhxZFqLHQe9CPP82ezJxmy-IEejSgpR9ro9YuAxqq0KqZ8i67RqbnhHcbKmBnNBnVndNRfWKdiPd4Ygv90n6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map with (A) ‘Bartlett’ pseudo-chromosomes (Pc), (B) ‘Dangshansuli’ (Pb) and apple (Md) pseudo-chromosomes. Red bars indicate linkage groups (LGs) of the I-PCG map. Blue and green bars indicate forward and reverse orientation LGs, respectively. LG length is relative to total length. The yellow lines indicate markers located in the same linkage group/pseudo-chromosome number and the grey lines indicate markers in different linkage group/pseudo-chromosome number. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw063f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473435/dsw063f6.tif" path-from-xml="dsw063f6.tif" target="_blank">View large</a><a section="88473435" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063f6.png?Expires=2147483647&Signature=4C0G566ZSeJf8b3d~uFYqyuoIvmg8GL~vkVdSwgAFFIYrFQG4c4OUhSUs50~YRQLHLyRlE~Zn2xy6bGlLBLcI5K8vDUh76ZgzzPerIZ8deWhjCEyj~W3Ddg9FsAtTks3dWjIuvD2uDwVVGz~Vc4Gqezi6~jNKvy70zFUgTrjYK0Mj92UeJds024W4p~xiRiuNIknRVCJP3mxkexoFtjw~pMUUnJWqfximJwT30nuX2385BlOuq4hoGc~LNkCOIHEe1OwsjJmEpljik5kRCoYj4AcCJ4ON-wGEegs32QFTK8klLrca2T8GHvjfh8iAPOsxQDJN7lCwLv2CGYECpJ2rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473435&ar=2960056&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw063f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of the integrated SSR and SNP-based pear consensus genetic (I-PCG) map with (A) ‘Bartlett’ pseudo-chromosomes (Pc), (B) ‘Dangshansuli’ (Pb) and apple (Md) pseudo-chromosomes. Red bars indicate linkage groups (LGs) of the I-PCG map. Blue and green bars indicate forward and reverse orientation LGs, respectively. LG length is relative to total length. The yellow lines indicate markers located in the same linkage group/pseudo-chromosome number and the grey lines indicate markers in different linkage group/pseudo-chromosome number. This figure is available in black and white in print and in colour at DNA Research online.</p></div></div>                            <h2 scrollto-destination=88473436 id="88473436" class="section-title" >4. Discussion</h2>
<p>Common methods to build high-density genetic maps with NGS usually start with genotyping individuals in a hybrid population through reduced-genome complexity technologies, such as RADseq,<a reveal-id="dsw063-B41" data-open="dsw063-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> GBS,<a reveal-id="dsw063-B27" data-open="dsw063-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and SLAF-Seq.<a reveal-id="dsw063-B42" data-open="dsw063-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> In this study, we merged published pear genetic maps by treating common SSR markers presented in maps from multiple sources as bridging markers and constructed an integrated SSR and SNP-based pear consensus genetic map (the I-PCG map). The advantages of merging genetic maps include integrating multiple population maps, obtaining larger genetic maps, discovering the genome and genetic variation, and resolving conflicts between different maps. Nevertheless, the length of the consensus genetic map could be inflated (&gt;3,000 cM in this study) due to the method used to develop the consensus map, i.e. artifact of the merging process, not true recombination. Thus, the consensus genetic map may be more relevant to the relative positioning of markers than the absolute distances between them.<a reveal-id="dsw063-B43" data-open="dsw063-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a></p><p>Anchoring scaffold sequences into pseudo-chromosomes requires both high-quality scaffold sequences and high-quality genetic maps.<a reveal-id="dsw063-B25" data-open="dsw063-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw063-B44" data-open="dsw063-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> In this study, we selected three high-quality maps with European pear genetic backgrounds for re-anchoring the ‘Bartlett’ scaffolds. The constructed pseudo-chromosomes captured 291.5 Mb of the ‘Bartlett’ genome sequence and dramatically increased the anchored ‘Bartlett’ genome from 29.7% in the original assembly to 50.5% in version 1.1. The improvement of a larger ratio for ‘Bartlett’ genome assembly was due to the higher density genetic maps used compared with the original assembly, which contained 1,391 <em>Malus</em> SNPs, and 888 <em>P. communis</em> SNPs.<a reveal-id="dsw063-B26" data-open="dsw063-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> In this study, we used a 5.4-fold number of SNP markers, including 2,668 from the BYH map, 1,404 from the PH-CG map, and 8,181 from the OHxLBJ map (<a reveal-id="dsw063-T5" data-open="dsw063-T5" class="link link-reveal link-table xref-fig">Table 5</a>), ensuring the capture of more ‘Bartlett’ scaffolds. Synteny analyses of the new ‘Bartlett’ v1.1 assembly indicated an extensive conserved genome collinearity with <em>P. bretschneideri</em> ‘Dangshansuli’ and <em>Malus</em> × <em>domestica</em> ‘Golden Delicious’, indicating that the new ‘Bartlett’ v1.1 is consistent with its closely related species. Furthermore, the homologous pseudo-chromosome pairing relationships demonstrated by intra-genome synteny analyses were similar with previous studies in ‘Dangshansuli’<a reveal-id="dsw063-B22" data-open="dsw063-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and ‘Golden Delicious’,<a reveal-id="dsw063-B40" data-open="dsw063-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> indicating that the genome structure of the new ‘Bartlett’ v1.1 genome assembly was correct. However, we noticed that four pseudo-chromosomes of ‘Bartlett’ did not have such good collinearity with ‘Dangshansuli’ and ‘Golden Delicious’ as other pseudo-chromosomes, namely Chr1, Chr2, Chr3, and Chr5, and that there were a number of dispersed collinear regions. These were possibly due to differences in the genome structure between ‘Bartlett’ and the other two genomes. On the other hand, this assembly ratio was still lower than that for ‘Dangshansuli’, which accounted for 75.5% of the genome. This may be the result of the genetic maps we used not capturing more scaffold sequences, although we used 5.4 times more SNP markers than for the original ‘Bartlett’ genome assembly. Also, the scaffold N50 length of the ‘Bartlett’ genome assembly was 88 kb compared with 540.8 kb of the ‘Dangshansuli’ one, which made it difficult to capture more scaffold sequences.<a reveal-id="dsw063-B25" data-open="dsw063-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Overall, we significantly improved the ‘Bartlett’ genome assembly, but more work is required for further improvement, especially a better quality scaffold assembly, which is the target of a recent initiative by an international European pear genome consortium.</p><p>The integrated I-PCG map was compared with the improved ‘Bartlett’ v1.1, ‘Dangshansuli’<a reveal-id="dsw063-B22" data-open="dsw063-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and ‘Golden Delicious’<a reveal-id="dsw063-B40" data-open="dsw063-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> genome assemblies. Both collinearity and misalignment of markers were observed for this comparison, indicating that the I-PCG map we constructed was reliable, but both the pear genetic and physical map are far from being complete. The reason for the misalignment may be repeat content or genome-wide duplication in the pear and apple genomes.<a reveal-id="dsw063-B45" data-open="dsw063-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> The results demonstrated that most misaligned markers were coordinated with homologous pairs identified in the pear and apple genomes. Furthermore, there were 42.9% misaligned markers for the apple genome of ‘Golden Delicious’, much higher than the 5.4% and 10.3% for the ‘Bartlett’ and ‘Dangshansuli’ genomes, respectively, indicating the much difference between pear and apple. In addition, similar homologous pseudo-chromosome pairing relationships were revealed by both comparisons of the consensus map and synteny analyses of the genome assembly of pear, which was the first step for further integration of genetic and physical maps in the future.</p><p>In conclusion, we demonstrated that a set of different pear genetic maps is a valuable resource for map integration and genetic comparison, which has been exploited in anchoring genome scaffolds of European pear ‘Bartlett’ (<em>P. communis</em>). It was proved that the abundant common markers could be merged into a consensus map, resulting in an integrated SSR and SNP-based pear consensus genetic map with the highest marker density to date. It consists of 5,085 markers and spans 3,266.0 cM, with an average marker interval of 0.64 cM. Meanwhile, the multiple high-density SNP-based maps enabled a significant improvement (291.5 Mb) in scaffold anchoring to pseudo-chromosomes, which accounted for 50.5% of the ‘Bartlett’ genome. The genetic and physical map comparison further revealed synteny and variation between European pear and Chinese white pear or apple.</p>                            <h2 scrollto-destination=88473441 id="88473441" class="section-title" >Accession number</h2>
<p>PRJNA349767.</p>                            <h2 scrollto-destination=88473443 id="88473443" class="section-title" >Data availability</h2>
<p>The sequencing data for the OHxLBJ GBS libraries were deposited at NCBI Sequence Read Archive (SRA) database under BioProject accession number of PRJNA349767. The pseudo-chromosome sequences for the European pear (<em>P</em><em>.</em><em>communis</em>) ‘Bartlett’ v1.1 genome were deposited at the Pear Genome Project website (<a class="link link-uri" href="http://peargenome.njau.edu.cn/default.asp?d=4&amp;m=2" target="_blank">http://peargenome.njau.edu.cn/default.asp?d=4&amp;m=2</a>).</p>                            <h2 scrollto-destination=88473445 id="88473445" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=88473447 id="88473447" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=88473449 id="88473449" class="section-title" >Funding</h2>
<p>This work was financially supported by the National High-tech R&amp;D Program (863) of China (2013AA102606-02), the National Natural Science Foundation of China (31372045), the Earmarked Fund for China Agriculture Research System (CARS-29), the Science Foundation for Distinguished Young Scientists in Jiangsu Province (BK20150025) and the Six Talent Peaks Project in Jiangsu Province (2014-NY-025).</p>                            <h2 scrollto-destination=88473451 id="88473451" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw063-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ashraf</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Foolad</div> <div class="given-names">M. R.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Crop breeding for salt tolerance in the era of molecular markers and marker-assisted selection</div>. <div class="source">Plant Breed</div> ., <div class="volume">132</div>, <div class="fpage">10</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crop%20breeding%20for%20salt%20tolerance%20in%20the%20era%20of%20molecular%20markers%20and%20marker-assisted%20selection&amp;author=M.%20Ashraf&amp;author=M.%20R.%20Foolad&amp;publication_year=2013&amp;journal=Plant%20Breed&amp;volume=132&amp;pages=10-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2013.132.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2013.132.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2013.132.issue-1"> </span></div> </div></div></div></div></div><div content-id="dsw063-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q. W.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genetic diversity and population structure of pear (<em>Pyrus</em> spp.) collections revealed by a set of core genome-wide SSR markers</div>. <div class="source">Tree Genet. Genomes</div> , <div class="volume">11</div>, 128.</div></div></div></div><div content-id="dsw063-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Song</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Identifying genetic diversity and a preliminary core collection of <em>Pyrus pyrifolia</em> cultivars by a genome-wide set of SSR markers</div>. <div class="source">Sci. Horticult</div> ., <div class="volume">167</div>, <div class="fpage">5</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20genetic%20diversity%20and%20a%20preliminary%20core%20collection%20of%20Pyrus%20pyrifolia%20cultivars%20by%20a%20genome-wide%20set%20of%20SSR%20markers&amp;author=Y.%20Song&amp;author=L.%20Fan&amp;author=H.%20Chen&amp;publication_year=2014&amp;journal=Sci.%20Horticult&amp;volume=167&amp;pages=5-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scienta.2013.12.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scienta.2013.12.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scienta.2013.12.005"> </span></div> </div></div></div></div></div><div content-id="dsw063-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">M. Y.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q. Z.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A novel set of EST-derived SSR markers for pear and cross-species transferability in Rosaceae</div>. <div class="source">Plant Mol. Biol. Rep</div> ., <div class="volume">32</div>, <div class="fpage">290</div>–<div class="lpage">302</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20set%20of%20EST-derived%20SSR%20markers%20for%20pear%20and%20cross-species%20transferability%20in%20Rosaceae&amp;author=M.%20Y.%20Zhang&amp;author=L.%20Fan&amp;author=Q.%20Z.%20Liu&amp;publication_year=2014&amp;journal=Plant%20Mol.%20Biol.%20Rep&amp;volume=32&amp;pages=290-302" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-013-0638-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-013-0638-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-013-0638-4"> </span></div> </div></div></div></div></div><div content-id="dsw063-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Knäbel</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Friend</div> <div class="given-names">A. P.</div></div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">J. W.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genetic control of pear rootstock-induced dwarfing and precocity is linked to a chromosomal region syntenic to the apple <em>Dw1</em> loci</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">15</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20pear%20rootstock-induced%20dwarfing%20and%20precocity%20is%20linked%20to%20a%20chromosomal%20region%20syntenic%20to%20the%20apple%20Dw1%20loci&amp;author=M.%20Kn%C3%A4bel&amp;author=A.%20P.%20Friend&amp;author=J.%20W.%20Palmer&amp;publication_year=2015&amp;journal=BMC%20Plant%20Biol&amp;volume=15&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-015-0620-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-015-0620-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-015-0620-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25592487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Montanari</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Guérif</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Ravon</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genetic mapping of <em>Cacopsylla pyri</em> resistance in an interspecific pear (<em>Pyrus</em> spp.) population</div>. <div class="source">Tree Genet. Genomes</div> , <div class="volume">11</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20Cacopsylla%20pyri%20resistance%20in%20an%20interspecific%20pear%20(Pyrus%20spp.)%20population&amp;author=S.%20Montanari&amp;author=P.%20Gu%C3%A9rif&amp;author=E.%20Ravon&amp;publication_year=2015&amp;journal=Tree%20Genet.%20Genomes&amp;volume=11&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-015-0901-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-015-0901-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-015-0901-y"> </span></div> </div></div></div></div></div><div content-id="dsw063-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Montanari</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Brewer</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Lamberts</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Genome mapping of postzygotic hybrid necrosis in an interspecific pear population</div>. <div class="source">Hortic. Res</div> ., <div class="volume">3</div>, <div class="fpage">15064</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20mapping%20of%20postzygotic%20hybrid%20necrosis%20in%20an%20interspecific%20pear%20population&amp;author=S.%20Montanari&amp;author=L.%20Brewer&amp;author=R.%20Lamberts&amp;publication_year=2016&amp;journal=Hortic.%20Res&amp;volume=3&amp;pages=15064" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hortres.2015.64" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhortres.2015.64" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhortres.2015.64"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26770810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saeed</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Brewer</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Johnston</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genetic, metabolite and developmental determinism of fruit friction discolouration in pear</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">14</div>, <div class="fpage">241</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%2C%20metabolite%20and%20developmental%20determinism%20of%20fruit%20friction%20discolouration%20in%20pear&amp;author=M.%20Saeed&amp;author=L.%20Brewer&amp;author=J.%20Johnston&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=241" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-014-0241-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-014-0241-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-014-0241-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25224302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Won</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Bastiaanse</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">Y. K.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genetic mapping of polygenic scab (<em>Venturia pirina</em>) resistance in an interspecific pear family</div>. <div class="source">Mol. Breed</div> ., <div class="volume">34</div>, <div class="fpage">2179</div>–<div class="lpage">2189</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20polygenic%20scab%20(Venturia%20pirina)%20resistance%20in%20an%20interspecific%20pear%20family&amp;author=K.%20Won&amp;author=H.%20Bastiaanse&amp;author=Y.%20K.%20Kim&amp;publication_year=2014&amp;journal=Mol.%20Breed&amp;volume=34&amp;pages=2179-2189" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-014-0172-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-014-0172-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-014-0172-6"> </span></div> </div></div></div></div></div><div content-id="dsw063-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L. T.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">High-density genetic linkage map construction and identification of fruit-related QTLs in pear using SNP and SSR markers</div>. <div class="source">J Exp. Bot</div> ., <div class="volume">65</div>, <div class="fpage">5771</div>–<div class="lpage">5781</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20genetic%20linkage%20map%20construction%20and%20identification%20of%20fruit-related%20QTLs%20in%20pear%20using%20SNP%20and%20SSR%20markers&amp;author=J.%20Wu&amp;author=L.%20T.%20Li&amp;author=M.%20Li&amp;publication_year=2014&amp;journal=J%20Exp.%20Bot&amp;volume=65&amp;pages=5771-5781" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/eru311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Feru311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Feru311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25129128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">R.-P.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.-G.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">An AFLP, SRAP, and SSR genetic linkage map and identification of QTLs for fruit traits in pear (<em>Pyrus</em> L.)</div>. <div class="source">Plant Mol. Biol. Rep</div> ., <div class="volume">31</div>, <div class="fpage">678</div>–<div class="lpage">687</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20AFLP%2C%20SRAP%2C%20and%20SSR%20genetic%20linkage%20map%20and%20identification%20of%20QTLs%20for%20fruit%20traits%20in%20pear%20(Pyrus%20L.)&amp;author=R.-P.%20Zhang&amp;author=J.%20Wu&amp;author=X.-G.%20Li&amp;publication_year=2013&amp;journal=Plant%20Mol.%20Biol.%20Rep&amp;volume=31&amp;pages=678-687" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-012-0544-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-012-0544-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-012-0544-1"> </span></div> </div></div></div></div></div><div content-id="dsw063-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Terakami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Moriya</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Adachi</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Fine mapping of the gene for susceptibility to black spot disease in Japanese pear (<em>Pyrus pyrifolia</em> Nakai)</div>. <div class="source">Breed. Sci</div> ., <div class="volume">66</div>, <div class="fpage">271</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20mapping%20of%20the%20gene%20for%20susceptibility%20to%20black%20spot%20disease%20in%20Japanese%20pear%20(Pyrus%20pyrifolia%20Nakai)&amp;author=S.%20Terakami&amp;author=S.%20Moriya&amp;author=Y.%20Adachi&amp;publication_year=2016&amp;journal=Breed.%20Sci&amp;volume=66&amp;pages=271" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.66.271" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.66.271" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.66.271"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27162498" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Abe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2001</div>, 
<div class="article-title">Mapping of disease-related genes in japanese pear using a molecular linkage map with RAPD markers</div>. <div class="source">Breed. Sci</div> ., <div class="volume">51</div>, <div class="fpage">179</div>–<div class="lpage">184</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20disease-related%20genes%20in%20japanese%20pear%20using%20a%20molecular%20linkage%20map%20with%20RAPD%20markers&amp;author=H.%20Iketani&amp;author=K.%20Abe&amp;author=T.%20Yamamoto&amp;publication_year=2001&amp;journal=Breed.%20Sci&amp;volume=51&amp;pages=179-184" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.51.179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.51.179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.51.179"> </span></div> </div></div></div></div></div><div content-id="dsw063-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kimura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shoda</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">Genetic linkage maps constructed by using an interspecific cross between Japanese and European pears</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">106</div>, <div class="fpage">9</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20maps%20constructed%20by%20using%20an%20interspecific%20cross%20between%20Japanese%20and%20European%20pears&amp;author=T.%20Yamamoto&amp;author=T.%20Kimura&amp;author=M.%20Shoda&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet&amp;volume=106&amp;pages=9-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-0966-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-0966-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-0966-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dondini</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Pierantoni</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gaiotti</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Identifying QTLs for fire-blight resistance via a European pear (<em>Pyrus communis</em> L.) genetic linkage map</div>. <div class="source">Mol. Breed</div> ., <div class="volume">14</div>, <div class="fpage">407</div>–<div class="lpage">418</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20QTLs%20for%20fire-blight%20resistance%20via%20a%20European%20pear%20(Pyrus%20communis%20L.)%20genetic%20linkage%20map&amp;author=L.%20Dondini&amp;author=L.%20Pierantoni&amp;author=F.%20Gaiotti&amp;publication_year=2005&amp;journal=Mol.%20Breed&amp;volume=14&amp;pages=407-418" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-005-0505-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-005-0505-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-005-0505-6"> </span></div> </div></div></div></div></div><div content-id="dsw063-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kimura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Terakami</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Integrated reference genetic linkage maps of pear based on SSR and AFLP markers</div>. <div class="source">Breed. Sci</div> ., <div class="volume">57</div>, <div class="fpage">321</div>–<div class="lpage">329</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20reference%20genetic%20linkage%20maps%20of%20pear%20based%20on%20SSR%20and%20AFLP%20markers&amp;author=T.%20Yamamoto&amp;author=T.%20Kimura&amp;author=S.%20Terakami&amp;publication_year=2007&amp;journal=Breed.%20Sci&amp;volume=57&amp;pages=321-329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.57.321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.57.321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.57.321"> </span></div> </div></div></div></div></div><div content-id="dsw063-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Terakami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kimura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Nishitani</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Genetic linkage map of the Japanese pear ‘Housui’ identifying three homozygous genomic regions</div>. <div class="source">J. Jpn. Soc. Hortic. Sci</div> ., <div class="volume">78</div>, <div class="fpage">417</div>–<div class="lpage">424</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20map%20of%20the%20Japanese%20pear%20%E2%80%98Housui%E2%80%99%20identifying%20three%20homozygous%20genomic%20regions&amp;author=S.%20Terakami&amp;author=T.%20Kimura&amp;author=C.%20Nishitani&amp;publication_year=2009&amp;journal=J.%20Jpn.%20Soc.%20Hortic.%20Sci&amp;volume=78&amp;pages=417-424" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.78.417" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.78.417" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.78.417"> </span></div> </div></div></div></div></div><div content-id="dsw063-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fan</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">M. Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q. Z.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Transferability of newly developed pear SSR markers to other Rosaceae species</div>. <div class="source">Plant Mol. Biol. Rep</div> ., <div class="volume">31</div>, <div class="fpage">1271</div>–<div class="lpage">1282</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transferability%20of%20newly%20developed%20pear%20SSR%20markers%20to%20other%20Rosaceae%20species&amp;author=L.%20Fan&amp;author=M.%20Y.%20Zhang&amp;author=Q.%20Z.%20Liu&amp;publication_year=2013&amp;journal=Plant%20Mol.%20Biol.%20Rep&amp;volume=31&amp;pages=1271-1282" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-013-0586-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-013-0586-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-013-0586-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24415844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L. T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Construction of a high-density simple sequence repeat consensus genetic map for pear (<em>Pyrus</em> spp.)</div>. <div class="source">Plant Mol. Biol. Rep</div> ., <div class="volume">33</div>, <div class="fpage">316</div>–<div class="lpage">325</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20simple%20sequence%20repeat%20consensus%20genetic%20map%20for%20pear%20(Pyrus%20spp.)&amp;author=H.%20Chen&amp;author=Y.%20Song&amp;author=L.%20T.%20Li&amp;publication_year=2015&amp;journal=Plant%20Mol.%20Biol.%20Rep&amp;volume=33&amp;pages=316-325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-014-0745-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-014-0745-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-014-0745-x"> </span></div> </div></div></div></div></div><div content-id="dsw063-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Terakami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Moriya</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">DNA markers developed from genome sequencing analysis in Japanese pear (<em>Pyrus pyrifolia</em>)</div>. <div class="source">Acta Hortic</div> ., <div class="volume">976</div>, <div class="fpage">477</div>–<div class="lpage">483</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20markers%20developed%20from%20genome%20sequencing%20analysis%20in%20Japanese%20pear%20(Pyrus%20pyrifolia)&amp;author=T.%20Yamamoto&amp;author=S.%20Terakami&amp;author=S.%20Moriya&amp;publication_year=2013&amp;journal=Acta%20Hortic&amp;volume=976&amp;pages=477-483" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.17660/ActaHortic.2013.976.67" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.17660%2FActaHortic.2013.976.67" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.17660%2FActaHortic.2013.976.67"> </span></div> </div></div></div></div></div><div content-id="dsw063-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Terakami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Takada</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Identification of QTLs controlling harvest time and fruit skin color in Japanese pear (<em>Pyrus pyrifolia</em> Nakai)</div>. <div class="source">Breed. Sci</div> ., <div class="volume">64</div>, <div class="fpage">351</div>–<div class="lpage">361</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20QTLs%20controlling%20harvest%20time%20and%20fruit%20skin%20color%20in%20Japanese%20pear%20(Pyrus%20pyrifolia%20Nakai)&amp;author=T.%20Yamamoto&amp;author=S.%20Terakami&amp;author=N.%20Takada&amp;publication_year=2014&amp;journal=Breed.%20Sci&amp;volume=64&amp;pages=351-361" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.64.351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.64.351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.64.351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25914590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The genome of the pear (<em>Pyrus bretschneideri</em> Rehd.)</div>. <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">396</div>–<div class="lpage">408</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20pear%20(Pyrus%20bretschneideri%20Rehd.)&amp;author=J.%20Wu&amp;author=Z.%20Wang&amp;author=Z.%20Shi&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=396-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.144311.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.144311.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.144311.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23149293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Montanari</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Saeed</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Knabel</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Identification of Pyrus single nucleotide polymorphisms (SNPs) and evaluation for genetic mapping in European pear and interspecific Pyrus hybrids</div>. <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e77022</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20Pyrus%20single%20nucleotide%20polymorphisms%20(SNPs)%20and%20evaluation%20for%20genetic%20mapping%20in%20European%20pear%20and%20interspecific%20Pyrus%20hybrids&amp;author=S.%20Montanari&amp;author=M.%20Saeed&amp;author=M.%20Knabel&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e77022" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0077022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0077022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0077022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24155917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Terakami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nishitani</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kunihisa</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Transcriptome-based single nucleotide polymorphism markers for genome mapping in Japanese pear (<em>Pyrus pyrifolia</em> Nakai)</div>. <div class="source">Tree Genet. Genomes</div> , <div class="volume">10</div>, <div class="fpage">853</div>–<div class="lpage">863</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome-based%20single%20nucleotide%20polymorphism%20markers%20for%20genome%20mapping%20in%20Japanese%20pear%20(Pyrus%20pyrifolia%20Nakai)&amp;author=S.%20Terakami&amp;author=C.%20Nishitani&amp;author=M.%20Kunihisa&amp;publication_year=2014&amp;journal=Tree%20Genet.%20Genomes&amp;volume=10&amp;pages=853-863" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-014-0726-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-014-0726-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-014-0726-0"> </span></div> </div></div></div></div></div><div content-id="dsw063-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Miao</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">ALLMAPS: robust scaffold ordering based on multiple maps</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ALLMAPS%3A%20robust%20scaffold%20ordering%20based%20on%20multiple%20maps&amp;author=H.%20Tang&amp;author=X.%20Zhang&amp;author=C.%20Miao&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0573-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0573-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0573-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25583564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chagné</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Crowhurst</div> <div class="given-names">R. N.</div></div>, <div class="name string-name"><div class="surname">Pindo</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The draft genome sequence of European pear (<em>Pyrus communis</em> L. ‘Bartlett’)</div>. <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e92644</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20sequence%20of%20European%20pear%20(Pyrus%20communis%20L.%20%E2%80%98Bartlett%E2%80%99)&amp;author=D.%20Chagn%C3%A9&amp;author=R.%20N.%20Crowhurst&amp;author=M.%20Pindo&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e92644" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0092644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0092644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0092644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24699266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elshire</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Glaubitz</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</div>. <div class="source">PLoS ONE</div> , <div class="volume">6</div>, <div class="fpage">e19379</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20(GBS)%20approach%20for%20high%20diversity%20species&amp;author=R.%20J.%20Elshire&amp;author=J.%20C.%20Glaubitz&amp;author=Q.%20Sun&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e19379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21573248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J. L.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">Genomic plant DNA preparation from fresh tissue-CTAB method</div>. <div class="source">Phytochem. Bull</div> ., <div class="volume">19</div>, <div class="fpage">11</div>–<div class="lpage">15</div>.</div></div></div></div><div content-id="dsw063-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ko</div> <div class="given-names">J. H.</div></div>, <div class="name string-name"><div class="surname">Chow</div> <div class="given-names">K. S.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">K. H.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Transcriptome analysis reveals novel features of the molecular events occurring in the laticifers of <em>Hevea brasiliensis</em> (para rubber tree)</div>. <div class="source">Plant Mol. Biol</div> ., 2003, <div class="volume">53</div>, <div class="fpage">479</div>–<div class="lpage">492</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20reveals%20novel%20features%20of%20the%20molecular%20events%20occurring%20in%20the%20laticifers%20of%20Hevea%20brasiliensis%20(para%20rubber%20tree)&amp;author=J.%20H.%20Ko&amp;author=K.%20S.%20Chow&amp;author=K.%20H.%20Han&amp;publication_year=2003&amp;journal=Plant%20Mol.%20Biol&amp;volume=53&amp;pages=479-492" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:PLAN.0000019119.66643.5d" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:PLAN.0000019119.66643.5d" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:PLAN.0000019119.66643.5d"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15010614" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Andrews</div> <div class="given-names">S.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">FastQC, Available: http://sourceforge.net/projects/picard/files/picard-tools/, 31 December 2016, date last accessed</div>.</div></div></div></div><div content-id="dsw063-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bradbury</div> <div class="given-names">P. J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Kroon</div> <div class="given-names">D. E.</div></div>, <div class="name string-name"><div class="surname">Casstevens</div> <div class="given-names">T. M.</div></div>, <div class="name string-name"><div class="surname">Ramdoss</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Buckler</div> <div class="given-names">E. S.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">TASSEL: software for association mapping of complex traits in diverse samples</div>. <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2633</div>–<div class="lpage">2635</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.%20J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.%20E.%20Kroon&amp;author=T.%20M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=E.%20S.%20Buckler&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2633-2635" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>. <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">359</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-359" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">C. X.</div></div>, <div class="name string-name"><div class="surname">Painter</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">Z. B.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species</div>. <div class="source">Theor. Popul. Biol</div> ., <div class="volume">61</div>: <div class="fpage">349</div>–<div class="lpage">363</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simultaneous%20maximum%20likelihood%20estimation%20of%20linkage%20and%20linkage%20phases%20in%20outcrossing%20species&amp;author=R.%20Wu&amp;author=C.%20X.%20Ma&amp;author=I.%20Painter&amp;author=Z.%20B.%20Zeng&amp;publication_year=2002&amp;journal=Theor.%20Popul.%20Biol&amp;volume=61&amp;pages=349-363" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/tpbi.2002.1577" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Ftpbi.2002.1577" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Ftpbi.2002.1577"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12027621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Margarido</div> <div class="given-names">G. R. A.</div></div>, <div class="name string-name"><div class="surname">Souza</div> <div class="given-names">A. P.</div></div>, <div class="name string-name"><div class="surname">Garcia</div> <div class="given-names">A. A. F.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">OneMap: software for genetic mapping in outcrossing species</div>. <div class="source">Hereditas</div> , <div class="volume">144</div>, <div class="fpage">78</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OneMap%3A%20software%20for%20genetic%20mapping%20in%20outcrossing%20species&amp;author=G.%20R.%20A.%20Margarido&amp;author=A.%20P.%20Souza&amp;author=A.%20A.%20F.%20Garcia&amp;publication_year=2007&amp;journal=Hereditas&amp;volume=144&amp;pages=78-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.2007.0018-0661.02000.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.2007.0018-0661.02000.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.2007.0018-0661.02000.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17663699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">Y. H.</div></div>, <div class="name string-name"><div class="surname">Close</div> <div class="given-names">T. J.</div></div>, <div class="name string-name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Accurate construction of consensus genetic maps via integer linear programming</div>. <div class="source">IEEE/ACM Trans. Comput. Biol</div> ., <div class="volume">8</div>, <div class="fpage">381</div>–<div class="lpage">394</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20construction%20of%20consensus%20genetic%20maps%20via%20integer%20linear%20programming&amp;author=Y.%20H.%20Wu&amp;author=T.%20J.%20Close&amp;author=S.%20Lonardi&amp;publication_year=2011&amp;journal=IEEE%2FACM%20Trans.%20Comput.%20Biol&amp;volume=8&amp;pages=381-394" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TCBB.2010.35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTCBB.2010.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTCBB.2010.35"> </span></div> </div></div></div></div></div><div content-id="dsw063-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bayer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Milne</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Stephen</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Comparative visualization of genetic and physical maps with Strudel</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">1307</div>–<div class="lpage">1308</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20visualization%20of%20genetic%20and%20physical%20maps%20with%20Strudel&amp;author=M.%20Bayer&amp;author=I.%20Milne&amp;author=G.%20Stephen&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=1307-1308" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21372085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W. J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT – the BLAST-like alignment tool</div>. <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">664</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%20%E2%80%93%20the%20BLAST-like%20alignment%20tool&amp;author=W.%20J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=656-664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Debarry</div> <div class="given-names">J. D.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.%20D.%20Debarry&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22217600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Bertioli</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Varshney</div> <div class="given-names">R. K.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Integrated consensus map of cultivated peanut and wild relatives reveals structures of the A and B genomes of Arachis and divergence of the legume genomes</div>. <div class="source">DNA Res</div> ., <div class="volume">20</div>, <div class="fpage">173</div>–<div class="lpage">184</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20consensus%20map%20of%20cultivated%20peanut%20and%20wild%20relatives%20reveals%20structures%20of%20the%20A%20and%20B%20genomes%20of%20Arachis%20and%20divergence%20of%20the%20legume%20genomes&amp;author=K.%20Shirasawa&amp;author=D.%20J.%20Bertioli&amp;author=R.%20K.%20Varshney&amp;publication_year=2013&amp;journal=DNA%20Res&amp;volume=20&amp;pages=173-184" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23315685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Velasco</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Zharkikh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Affourtit</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">The genome of the domesticated apple (<em>Malus</em> × <em>domestica</em> Borkh.)</div>. <div class="source">Nat. Genet</div> ., <div class="volume">42</div>, <div class="fpage">833</div>–<div class="lpage">839</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20domesticated%20apple%20(Malus%E2%80%89%C3%97%E2%80%89domestica%20Borkh.)&amp;author=R.%20Velasco&amp;author=A.%20Zharkikh&amp;author=J.%20Affourtit&amp;publication_year=2010&amp;journal=Nat.%20Genet&amp;volume=42&amp;pages=833-839" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20802477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baird</div> <div class="given-names">N. A.</div></div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Atwood</div> <div class="given-names">T. S.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</div>. <div class="source">PLoS ONE</div> , <div class="volume">3</div>, <div class="fpage">e3376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20discovery%20and%20genetic%20mapping%20using%20sequenced%20RAD%20markers&amp;author=N.%20A.%20Baird&amp;author=P.%20D.%20Etter&amp;author=T.%20S.%20Atwood&amp;publication_year=2008&amp;journal=PLoS%20ONE&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">SLAF-seq: an efficient method of large-scale <em>de novo</em> SNP discovery and genotyping using high-throughput sequencing</div>. <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e58700</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SLAF-seq%3A%20an%20efficient%20method%20of%20large-scale%20de%20novo%20SNP%20discovery%20and%20genotyping%20using%20high-throughput%20sequencing&amp;author=X.%20Sun&amp;author=D.%20Liu&amp;author=X.%20Zhang&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e58700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0058700" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0058700" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0058700"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23527008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khan</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">Y. P.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Y. F.</div></div>, <div class="name string-name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Korban</div> <div class="given-names">S. S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome</div>. <div class="source">PLoS ONE</div> , <div class="volume">7</div>, e47864.</div></div></div></div><div content-id="dsw063-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kosack</div> <div class="given-names">D. S.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Hierarchical scaffolding with Bambus</div>. <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">149</div>–<div class="lpage">159</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hierarchical%20scaffolding%20with%20Bambus&amp;author=M.%20Pop&amp;author=D.%20S.%20Kosack&amp;author=S.%20L.%20Salzberg&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=149-159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1536204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1536204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1536204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14707177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw063-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw063-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Michael</div> <div class="given-names">T. P.</div></div>, <div class="name string-name"><div class="surname">VanBuren</div> <div class="given-names">R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Progress, challenges and the future of crop genomes</div>. <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">24</div>, <div class="fpage">71</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progress%2C%20challenges%20and%20the%20future%20of%20crop%20genomes&amp;author=T.%20P.%20Michael&amp;author=R.%20VanBuren&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=24&amp;pages=71-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2015.02.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2015.02.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2015.02.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25703261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2960056&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw063/1/dsw063_Supp.xlsx?Expires=2147483647&amp;Signature=BRQ-vWiBijZwSVGciBQEcya9bday7i~G2s0l5rL2KNnEROAetrwpn1afPo7ftRWPjnNhwSFHCqlOEN1lA~U3lI--XB4X4erVNzM3eXkyKf6Nx0AVSK3nyzJEPbiWRYA4z4VHGVLaul2jI0xRwbxUpfgZLhLMAwTXi3WLfJitsSiI9ebgitkfrBqgMJ7DA8gSNyoe4VarEeQGw71HD-hkkUVstapvSXjtZ9soFHJw8v-K0JZ-~s51djfkWpW-qmh15pCpFYDHGmSHFlsI59D9KKj8U~uloFz4BVtLFH76SNZvS1~W33AsGDVa33tX0BbZccsBbRKsjFDHRKCZ9NTFAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">912</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw063"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2960056"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f3%2f289%2f2960056">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000404439600007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F3%2F289%2F2960056" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2960056'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Integrated high-density consensus genetic map of Pyrus and anchoring of the ‘Bartlett’ v1.0 (Pyrus communis) genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw063");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f289%2f2960056">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

