

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/1/25/2449750","@type":"ScholarlyArticle","name":"Unveiling DNA structural features of promoters associated with various types of TSSs in prokaryotic transcriptomes and their role in gene expression","datePublished":"2016-10-31","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw045","keywords":["promoter prediction","DNA structural properties","transcriptome","gene expression"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/1/25/2449750","author":[{"name":"Kumar, Aditya","@type":"Person"},{"name":"Bansal, Manju","@type":"Person"}],"description":"Abstract.  Next-generation sequencing studies have revealed that a variety of transcripts are present in the prokaryotic transcriptome and a significant fractio","pageStart":"25","pageEnd":"35","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Unveiling DNA structural features of promoters associated with various types of TSSs in prokaryotic transcriptomes and their role in gene expression","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Unveiling DNA structural features of promoters associated with various types of TSSs in prokaryotic transcriptomes and their role in gene expression" />
    <meta property="og:description" content="Abstract.  Next-generation sequencing studies have revealed that a variety of transcripts are present in the prokaryotic transcriptome and a significant fractio" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/1/25/2449750" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-10-31" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kumar, Aditya" /><meta name="citation_author" content="Bansal, Manju" /><meta name="citation_title" content="Unveiling DNA structural features of promoters associated with various types of TSSs in prokaryotic transcriptomes and their role in gene expression" /><meta name="citation_firstpage" content="25" /><meta name="citation_lastpage" content="35" /><meta name="citation_doi" content="10.1093/dnares/dsw045" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2017/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome-wide transcription factor binding: beyond direct target regulation;  citation_journal_title=Trends Genet;  citation_year=2011;  citation_volume=27;  citation_pages=141-8; " /><meta name="citation_reference" content="citation_title=Cofactor binding evokes latent differences in DNA binding specificity between Hox proteins;  citation_journal_title=Cell;  citation_year=2011;  citation_volume=147;  citation_pages=1270-82; " /><meta name="citation_reference" content="citation_title=Genomic regions flanking E-box binding sites influence DNA binding specificity of bHLH transcription factors through DNA shape;  citation_journal_title=Cell Rep;  citation_year=2013;  citation_volume=3;  citation_pages=1093-104; " /><meta name="citation_reference" content="citation_title=In pursuit of design principles of regulatory sequences;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=453-468; " /><meta name="citation_reference" content="citation_title=DNA structural properties in the classification of genomic transcription regulation elements;  citation_journal_title=Bioinform. Biol. Insights;  citation_year=2012;  citation_volume=6;  citation_pages=155-68; " /><meta name="citation_reference" content="citation_title=Improved predictions of transcription factor binding sites using physicochemical features of DNA;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e175" /><meta name="citation_reference" content="citation_title=Quantitative modeling of transcription factor binding specificities using DNA shape;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=4654-9; " /><meta name="citation_reference" content="citation_title=Unraveling determinants of transcription factor binding outside the core binding site;  citation_journal_title=Genome Res;  citation_year=2015;  citation_pages=185033-114; " /><meta name="citation_reference" content="citation_title=GBshape: a genome browser database for DNA shape annotations;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D103-9; " /><meta name="citation_reference" content="citation_title=A statistical thermodynamic model for investigating the stability of DNA sequences from oligonucleotides to genomes;  citation_journal_title=Biophys J;  citation_year=2014;  citation_volume=106;  citation_pages=2465-73; " /><meta name="citation_reference" content="citation_title=DNA-water interactions distinguish messenger RNA genes from transfer RNA genes;  citation_journal_title=J. Am. Chem. Soc;  citation_year=2012;  citation_volume=134;  citation_pages=8814-6; " /><meta name="citation_reference" content="citation_title=DNA-energetics-based analyses suggest additional genes in prokaryotes;  citation_journal_title=J. Biosci;  citation_year=2012;  citation_volume=37;  citation_pages=433-44; " /><meta name="citation_reference" content="citation_title=Large-scale structural analysis of the core promoter in mammalian and plant genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=4255-64; " /><meta name="citation_reference" content="citation_title=Structural properties of promoters: similarities and differences between prokaryotes and eukaryotes;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=3165-3175; " /><meta name="citation_reference" content="citation_title=Structural properties of prokaryotic promoter regions correlate with functional features;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e88717" /><meta name="citation_reference" content="citation_title=Role of DNA sequence based structural features of promoters in transcription initiation and gene expression;  citation_journal_title=Curr. Opin. Struct. Biol;  citation_year=2014;  citation_volume=25;  citation_pages=77-85; " /><meta name="citation_reference" content="citation_title=Relative stability of DNA as a generic criterion for promoter prediction: whole genome annotation of microbial genomes with varying nucleotide base composition;  citation_journal_title=Mol. Biosyst;  citation_year=2009;  citation_volume=5;  citation_pages=1758-1769; " /><meta name="citation_reference" content="citation_title=High-quality annotation of promoter regions for 913 bacterial genomes;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=3043-3050; " /><meta name="citation_reference" content="citation_title=PromBase: a web resource for various genomic features and predicted promoters in prokaryotic genomes;  citation_journal_title=BMC Res. Notes;  citation_year=2011;  citation_volume=4;  citation_pages=257" /><meta name="citation_reference" content="citation_title=Characterization of structural and free energy properties of promoters associated with Primary and Operon TSS in Helicobacter pylori genome and their orthologs;  citation_journal_title=J. Biosci;  citation_year=2012;  citation_volume=37;  citation_pages=423-31; " /><meta name="citation_reference" content="citation_title=The genome-wide distribution of non-B DNA motifs is shaped by operon structure and suggests the transcriptional importance of non-B DNA structures in Escherichia coli;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=5965-77; " /><meta name="citation_reference" content="citation_title=Distinguishing between productive and abortive promoters using a random forest classifier in Mycoplasma pneumoniae;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=3442-53; " /><meta name="citation_reference" content="citation_title=Pervasive transcription: illuminating the dark matter of bacterial transcriptomes;  citation_journal_title=Nat. Rev. Microbiol;  citation_year=2014;  citation_volume=12;  citation_pages=647-653; " /><meta name="citation_reference" content="citation_title=Regulatory RNAs in photosynthetic cyanobacteria;  citation_journal_title=FEMS Microbiol. Rev;  citation_year=2015;  citation_volume=39;  citation_pages=301-15; " /><meta name="citation_reference" content="citation_title=Transcriptional interference–a crash course;  citation_journal_title=Trends Genet;  citation_year=2005;  citation_volume=21;  citation_pages=339-45; " /><meta name="citation_reference" content="citation_title=Antisense transcription as a tool to tune gene expression;  citation_journal_title=Mol. Syst. Biol;  citation_year=2016;  citation_volume=12;  citation_pages=854" /><meta name="citation_reference" content="citation_title=Non-coding RNA: a new frontier in regulatory biology;  citation_journal_title=Natl. Sci. Rev;  citation_year=2014;  citation_volume=1;  citation_pages=190-204; " /><meta name="citation_reference" content="citation_title=Promoter strength prediction based on occurrence frequencies of consensus patterns;  citation_journal_title=J. Theor. Biol;  citation_year=1994;  citation_volume=171;  citation_pages=355-9; " /><meta name="citation_reference" content="citation_title=Predicting strength and function for promoters of the Escherichia coli alternative sigma factor, sigmaE;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=2854-59; " /><meta name="citation_reference" content="citation_title=Predicting the strength of UP-elements and full-length E. coli sigmaE promoters;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=2907-24; " /><meta name="citation_reference" content="citation_title=Gene order and chromosome dynamics coordinate spatiotemporal gene expression during the bacterial growth cycle;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=E42-50; " /><meta name="citation_reference" content="citation_title=DNA thermodynamic stability and supercoil dynamics determine the gene expression program during the bacterial growth cycle;  citation_journal_title=Mol. Biosyst;  citation_year=2013;  citation_volume=9;  citation_pages=1643-51; " /><meta name="citation_reference" content="citation_title=The primary transcriptome of the major human pathogen Helicobacter pylori;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=250-5; " /><meta name="citation_reference" content="citation_title=Dynamics of transcriptional start site selection during nitrogen stress-induced cell differentiation in Anabaena sp. PCC7120;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=20130-5; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the primary transcriptome of Synechocystis sp. PCC 6803;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=527-39; " /><meta name="citation_reference" content="citation_title=Comparative analysis of regulatory elements between Escherichia coli and Klebsiella pneumoniae by genome-wide transcription start site profiling;  citation_journal_title=PLoS Genet;  citation_year=2012;  citation_volume=8;  citation_pages=e1002867" /><meta name="citation_reference" content="citation_title=The transcriptional landscape and small RNAs of Salmonella enterica serovar Typhimurium;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=E1277-86; " /><meta name="citation_reference" content="citation_title=A unified view of polymer, dumbbell, and oligonucleotide DNA nearest-neighbor thermodynamics;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=1460-5; " /><meta name="citation_reference" content="citation_publisher=Springer, Netherlands;  citation_title=Systems and Synthetic Biology;  citation_year=2015;  citation_pages=241-4; " /><meta name="citation_reference" content="citation_title=Trinucleotide models for DNA bending propensity: comparison of models based on DNaseI digestion and nucleosome packaging data;  citation_journal_title=J. Biomol. Struct. Dyn;  citation_year=1995;  citation_volume=13;  citation_pages=309-17; " /><meta name="citation_reference" content="citation_title=Sequence periodicities in chicken nucleosome core DNA;  citation_journal_title=J. Mol. Biol;  citation_year=1986;  citation_volume=191;  citation_pages=659-75; " /><meta name="citation_reference" content="citation_title=An assessment of three dinucleotide parameters to predict DNA curvature by quantitative comparison with experimental data;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=2647-2658; " /><meta name="citation_reference" content="citation_title=Curved DNA without A-A: experimental estimation of all 16 DNA wedge angles;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=2312-6; " /><meta name="citation_reference" content="citation_title=NUPARM and NUCGEN: software for analysis and generation of sequence dependent nucleic acid structures;  citation_journal_title=Comput. Appl. Biosci;  citation_year=1995;  citation_volume=11;  citation_pages=281-7; " /><meta name="citation_reference" content="citation_title=NCBI GEO: archive for functional genomics data sets–update;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D991-5; " /><meta name="citation_reference" content="citation_title=Differential expression analysis for sequence count data;  citation_journal_title=Genome Biol;  citation_year=2010;  citation_volume=11;  citation_pages=R106" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/1/25/10731790/dsw045.pdf" /><meta name="description" content="Abstract.  Next-generation sequencing studies have revealed that a variety of transcripts are present in the prokaryotic transcriptome and a significant fractio" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/1/25/2449750" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/1/25/2449750" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/1/25/2449750");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw045';
    </script>
<title>
	Unveiling DNA structural features of promoters associated with various types of TSSs in prokaryotic transcriptomes and their role in gene expression | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2449750" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeZmPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdW1hciwgQWRpdHlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmFuc2FsLCBNYW5qdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVW52ZWlsaW5nIEROQSBzdHJ1Y3R1cmFsIGZlYXR1cmVzIG9mIHByb21vdGVycyBhc3NvY2lhdGVkIHdpdGggdmFyaW91cyB0eXBlcyBvZiBUU1NzIGluIHByb2thcnlvdGljIHRyYW5zY3JpcHRvbWVzIGFuZCB0aGVpciByb2xlIGluIGdlbmUgZXhwcmVzc2lvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzA0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgdHJhbnNjcmlwdGlvbiBmYWN0b3IgYmluZGluZzogYmV5b25kIGRpcmVjdCB0YXJnZXQgcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTQxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZmFjdG9yIGJpbmRpbmcgZXZva2VzIGxhdGVudCBkaWZmZXJlbmNlcyBpbiBETkEgYmluZGluZyBzcGVjaWZpY2l0eSBiZXR3ZWVuIEhveCBwcm90ZWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDc7ICBjaXRhdGlvbl9wYWdlcz0xMjcwLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHJlZ2lvbnMgZmxhbmtpbmcgRS1ib3ggYmluZGluZyBzaXRlcyBpbmZsdWVuY2UgRE5BIGJpbmRpbmcgc3BlY2lmaWNpdHkgb2YgYkhMSCB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgdGhyb3VnaCBETkEgc2hhcGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVwOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMDkzLTEwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gcHVyc3VpdCBvZiBkZXNpZ24gcHJpbmNpcGxlcyBvZiByZWd1bGF0b3J5IHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NDUzLTQ2ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHN0cnVjdHVyYWwgcHJvcGVydGllcyBpbiB0aGUgY2xhc3NpZmljYXRpb24gb2YgZ2Vub21pYyB0cmFuc2NyaXB0aW9uIHJlZ3VsYXRpb24gZWxlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybS4gQmlvbC4gSW5zaWdodHM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE1NS02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZWQgcHJlZGljdGlvbnMgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3IgYmluZGluZyBzaXRlcyB1c2luZyBwaHlzaWNvY2hlbWljYWwgZmVhdHVyZXMgb2YgRE5BOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPWUxNzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgbW9kZWxpbmcgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3IgYmluZGluZyBzcGVjaWZpY2l0aWVzIHVzaW5nIEROQSBzaGFwZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9NDY1NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbnJhdmVsaW5nIGRldGVybWluYW50cyBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kaW5nIG91dHNpZGUgdGhlIGNvcmUgYmluZGluZyBzaXRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fcGFnZXM9MTg1MDMzLTExNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0JzaGFwZTogYSBnZW5vbWUgYnJvd3NlciBkYXRhYmFzZSBmb3IgRE5BIHNoYXBlIGFubm90YXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPUQxMDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdGF0aXN0aWNhbCB0aGVybW9keW5hbWljIG1vZGVsIGZvciBpbnZlc3RpZ2F0aW5nIHRoZSBzdGFiaWxpdHkgb2YgRE5BIHNlcXVlbmNlcyBmcm9tIG9saWdvbnVjbGVvdGlkZXMgdG8gZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvcGh5cyBKOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTI0NjUtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQS13YXRlciBpbnRlcmFjdGlvbnMgZGlzdGluZ3Vpc2ggbWVzc2VuZ2VyIFJOQSBnZW5lcyBmcm9tIHRyYW5zZmVyIFJOQSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW0uIENoZW0uIFNvYzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz04ODE0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQS1lbmVyZ2V0aWNzLWJhc2VkIGFuYWx5c2VzIHN1Z2dlc3QgYWRkaXRpb25hbCBnZW5lcyBpbiBwcm9rYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvc2NpOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9NDMzLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBzdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIHRoZSBjb3JlIHByb21vdGVyIGluIG1hbW1hbGlhbiBhbmQgcGxhbnQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz00MjU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIHByb3BlcnRpZXMgb2YgcHJvbW90ZXJzOiBzaW1pbGFyaXRpZXMgYW5kIGRpZmZlcmVuY2VzIGJldHdlZW4gcHJva2FyeW90ZXMgYW5kIGV1a2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MzE2NS0zMTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIHByb3BlcnRpZXMgb2YgcHJva2FyeW90aWMgcHJvbW90ZXIgcmVnaW9ucyBjb3JyZWxhdGUgd2l0aCBmdW5jdGlvbmFsIGZlYXR1cmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTg4NzE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZSBvZiBETkEgc2VxdWVuY2UgYmFzZWQgc3RydWN0dXJhbCBmZWF0dXJlcyBvZiBwcm9tb3RlcnMgaW4gdHJhbnNjcmlwdGlvbiBpbml0aWF0aW9uIGFuZCBnZW5lIGV4cHJlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIFN0cnVjdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWxhdGl2ZSBzdGFiaWxpdHkgb2YgRE5BIGFzIGEgZ2VuZXJpYyBjcml0ZXJpb24gZm9yIHByb21vdGVyIHByZWRpY3Rpb246IHdob2xlIGdlbm9tZSBhbm5vdGF0aW9uIG9mIG1pY3JvYmlhbCBnZW5vbWVzIHdpdGggdmFyeWluZyBudWNsZW90aWRlIGJhc2UgY29tcG9zaXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvc3lzdDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTc1OC0xNzY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXF1YWxpdHkgYW5ub3RhdGlvbiBvZiBwcm9tb3RlciByZWdpb25zIGZvciA5MTMgYmFjdGVyaWFsIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MzA0My0zMDUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9tQmFzZTogYSB3ZWIgcmVzb3VyY2UgZm9yIHZhcmlvdXMgZ2Vub21pYyBmZWF0dXJlcyBhbmQgcHJlZGljdGVkIHByb21vdGVycyBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUmVzLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBzdHJ1Y3R1cmFsIGFuZCBmcmVlIGVuZXJneSBwcm9wZXJ0aWVzIG9mIHByb21vdGVycyBhc3NvY2lhdGVkIHdpdGggUHJpbWFyeSBhbmQgT3Blcm9uIFRTUyBpbiBIZWxpY29iYWN0ZXIgcHlsb3JpIGdlbm9tZSBhbmQgdGhlaXIgb3J0aG9sb2dzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9zY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz00MjMtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUtd2lkZSBkaXN0cmlidXRpb24gb2Ygbm9uLUIgRE5BIG1vdGlmcyBpcyBzaGFwZWQgYnkgb3Blcm9uIHN0cnVjdHVyZSBhbmQgc3VnZ2VzdHMgdGhlIHRyYW5zY3JpcHRpb25hbCBpbXBvcnRhbmNlIG9mIG5vbi1CIEROQSBzdHJ1Y3R1cmVzIGluIEVzY2hlcmljaGlhIGNvbGk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9NTk2NS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdGluZ3Vpc2hpbmcgYmV0d2VlbiBwcm9kdWN0aXZlIGFuZCBhYm9ydGl2ZSBwcm9tb3RlcnMgdXNpbmcgYSByYW5kb20gZm9yZXN0IGNsYXNzaWZpZXIgaW4gTXljb3BsYXNtYSBwbmV1bW9uaWFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTM0NDItNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcnZhc2l2ZSB0cmFuc2NyaXB0aW9uOiBpbGx1bWluYXRpbmcgdGhlIGRhcmsgbWF0dGVyIG9mIGJhY3RlcmlhbCB0cmFuc2NyaXB0b21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY0Ny02NTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgUk5BcyBpbiBwaG90b3N5bnRoZXRpYyBjeWFub2JhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gUmV2OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MzAxLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgaW50ZXJmZXJlbmNl4oCTYSBjcmFzaCBjb3Vyc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTMzOS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aXNlbnNlIHRyYW5zY3JpcHRpb24gYXMgYSB0b29sIHRvIHR1bmUgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFN5c3QuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz04NTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob24tY29kaW5nIFJOQTogYSBuZXcgZnJvbnRpZXIgaW4gcmVndWxhdG9yeSBiaW9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXRsLiBTY2kuIFJldjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTkwLTIwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvbW90ZXIgc3RyZW5ndGggcHJlZGljdGlvbiBiYXNlZCBvbiBvY2N1cnJlbmNlIGZyZXF1ZW5jaWVzIG9mIGNvbnNlbnN1cyBwYXR0ZXJuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gVGhlb3IuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTcxOyAgY2l0YXRpb25fcGFnZXM9MzU1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpbmcgc3RyZW5ndGggYW5kIGZ1bmN0aW9uIGZvciBwcm9tb3RlcnMgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgYWx0ZXJuYXRpdmUgc2lnbWEgZmFjdG9yLCBzaWdtYUU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTI4NTQtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3RpbmcgdGhlIHN0cmVuZ3RoIG9mIFVQLWVsZW1lbnRzIGFuZCBmdWxsLWxlbmd0aCBFLiBjb2xpIHNpZ21hRSBwcm9tb3RlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MjkwNy0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBvcmRlciBhbmQgY2hyb21vc29tZSBkeW5hbWljcyBjb29yZGluYXRlIHNwYXRpb3RlbXBvcmFsIGdlbmUgZXhwcmVzc2lvbiBkdXJpbmcgdGhlIGJhY3RlcmlhbCBncm93dGggY3ljbGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPUU0Mi01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHRoZXJtb2R5bmFtaWMgc3RhYmlsaXR5IGFuZCBzdXBlcmNvaWwgZHluYW1pY3MgZGV0ZXJtaW5lIHRoZSBnZW5lIGV4cHJlc3Npb24gcHJvZ3JhbSBkdXJpbmcgdGhlIGJhY3RlcmlhbCBncm93dGggY3ljbGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvc3lzdDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTY0My01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHByaW1hcnkgdHJhbnNjcmlwdG9tZSBvZiB0aGUgbWFqb3IgaHVtYW4gcGF0aG9nZW4gSGVsaWNvYmFjdGVyIHB5bG9yaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NDsgIGNpdGF0aW9uX3BhZ2VzPTI1MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljcyBvZiB0cmFuc2NyaXB0aW9uYWwgc3RhcnQgc2l0ZSBzZWxlY3Rpb24gZHVyaW5nIG5pdHJvZ2VuIHN0cmVzcy1pbmR1Y2VkIGNlbGwgZGlmZmVyZW50aWF0aW9uIGluIEFuYWJhZW5hIHNwLiBQQ0M3MTIwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0yMDEzMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgcHJpbWFyeSB0cmFuc2NyaXB0b21lIG9mIFN5bmVjaG9jeXN0aXMgc3AuIFBDQyA2ODAzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTI3LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiByZWd1bGF0b3J5IGVsZW1lbnRzIGJldHdlZW4gRXNjaGVyaWNoaWEgY29saSBhbmQgS2xlYnNpZWxsYSBwbmV1bW9uaWFlIGJ5IGdlbm9tZS13aWRlIHRyYW5zY3JpcHRpb24gc3RhcnQgc2l0ZSBwcm9maWxpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyODY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRpb25hbCBsYW5kc2NhcGUgYW5kIHNtYWxsIFJOQXMgb2YgU2FsbW9uZWxsYSBlbnRlcmljYSBzZXJvdmFyIFR5cGhpbXVyaXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz1FMTI3Ny04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB1bmlmaWVkIHZpZXcgb2YgcG9seW1lciwgZHVtYmJlbGwsIGFuZCBvbGlnb251Y2xlb3RpZGUgRE5BIG5lYXJlc3QtbmVpZ2hib3IgdGhlcm1vZHluYW1pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MTQ2MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldGhlcmxhbmRzOyAgY2l0YXRpb25fdGl0bGU9U3lzdGVtcyBhbmQgU3ludGhldGljIEJpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl9wYWdlcz0yNDEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpbnVjbGVvdGlkZSBtb2RlbHMgZm9yIEROQSBiZW5kaW5nIHByb3BlbnNpdHk6IGNvbXBhcmlzb24gb2YgbW9kZWxzIGJhc2VkIG9uIEROYXNlSSBkaWdlc3Rpb24gYW5kIG51Y2xlb3NvbWUgcGFja2FnaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb21vbC4gU3RydWN0LiBEeW47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zMDktMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIHBlcmlvZGljaXRpZXMgaW4gY2hpY2tlbiBudWNsZW9zb21lIGNvcmUgRE5BOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MTkxOyAgY2l0YXRpb25fcGFnZXM9NjU5LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhc3Nlc3NtZW50IG9mIHRocmVlIGRpbnVjbGVvdGlkZSBwYXJhbWV0ZXJzIHRvIHByZWRpY3QgRE5BIGN1cnZhdHVyZSBieSBxdWFudGl0YXRpdmUgY29tcGFyaXNvbiB3aXRoIGV4cGVyaW1lbnRhbCBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI2NDctMjY1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3VydmVkIEROQSB3aXRob3V0IEEtQTogZXhwZXJpbWVudGFsIGVzdGltYXRpb24gb2YgYWxsIDE2IEROQSB3ZWRnZSBhbmdsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTg4OyAgY2l0YXRpb25fcGFnZXM9MjMxMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OVVBBUk0gYW5kIE5VQ0dFTjogc29mdHdhcmUgZm9yIGFuYWx5c2lzIGFuZCBnZW5lcmF0aW9uIG9mIHNlcXVlbmNlIGRlcGVuZGVudCBudWNsZWljIGFjaWQgc3RydWN0dXJlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcHV0LiBBcHBsLiBCaW9zY2k7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yODEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkNCSSBHRU86IGFyY2hpdmUgZm9yIGZ1bmN0aW9uYWwgZ2Vub21pY3MgZGF0YSBzZXRz4oCTdXBkYXRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPUQ5OTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGV4cHJlc3Npb24gYW5hbHlzaXMgZm9yIHNlcXVlbmNlIGNvdW50IGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9UjEwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC8xLzI1LzEwNzMxNzkwL2RzdzA0NS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgc3R1ZGllcyBoYXZlIHJldmVhbGVkIHRoYXQgYSB2YXJpZXR5IG9mIHRyYW5zY3JpcHRzIGFyZSBwcmVzZW50IGluIHRoZSBwcm9rYXJ5b3RpYyB0cmFuc2NyaXB0b21lIGFuZCBhIHNpZ25pZmljYW50IGZyYWN0aW8iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI0NDk3NTAmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkHRbn+otjjAj/j+1b+Ghc1hFDb9Y=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f25%2f2449750" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f25%2f2449750" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=96271" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&amp;Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/1">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="61462952">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462952">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462954">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462954">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462957">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462957">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462982">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462982">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462998">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462998">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463000">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463000">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463006">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463006">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/1/11/2670108">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/1/37/2670112">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Unveiling DNA structural features of promoters associated with various types of TSSs in prokaryotic transcriptomes and their role in gene expression
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aditya Kumar</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aditya+Kumar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumar A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kumar A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aditya Kumar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manju Bansal</div>
    </div>


                <div class="info-card-note">
                    <a reveal-id="dsw045-cor1" data-open="dsw045-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +91-8022932534. Fax: +91-8023600535. Email: <a href="mailto:mb@mbu.iisc.ernet.in" target="_blank">mb@mbu.iisc.ernet.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manju+Bansal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bansal M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bansal M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Manju Bansal</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw045-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 25–35, <a href='https://doi.org/10.1093/dnares/dsw045'>https://doi.org/10.1093/dnares/dsw045</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">31 October 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 May 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 September 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2449750" href="/dnaresearch/article-pdf/24/1/25/10731790/dsw045.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Aditya Kumar, Manju Bansal;  Unveiling DNA structural features of promoters associated with various types of TSSs in prokaryotic transcriptomes and their role in gene expression, <em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 25–35, <a href='https://doi.org/10.1093/dnares/dsw045'>https://doi.org/10.1093/dnares/dsw045</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2449750?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2449750?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2449750?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2449750?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2449750?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2449750?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2449750?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Unveiling%20DNA%20structural%20features%20of%20promoters%20associated%20with%20various%20types%20of%20TSSs%20in%20prokaryotic%20transcriptomes%20and%20their%20role%20in%20gene%20expression&amp;publicationDate=2016-10-31&amp;volumeNum=24&amp;issueNum=1&amp;author=Kumar%2C%20Aditya%3B%20Bansal%2C%20Manju&amp;startPage=25&amp;endPage=35&amp;contentId=10.1093%2Fdnares%2Fdsw045&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f25%2f2449750" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=61462952 id="61462952" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Next-generation sequencing studies have revealed that a variety of transcripts are present in the prokaryotic transcriptome and a significant fraction of them are functional, being involved in various regulatory activities apart from coding for proteins. Identification of promoters associated with different transcripts is necessary for characterization of the transcriptome. Promoter regions have been shown to have unique structural features as compared with their flanking region, in organisms covering all domains of life. Here we report an <em>in silico</em> analysis of DNA sequence dependent structural properties like stability, bendability and curvature in the promoter region of six different prokaryotic transcriptomes. Using these structural features, we predicted promoters associated with different categories of transcripts (mRNA, internal, antisense and non-coding), which constitute the transcriptome. Promoter annotation using structural features is fairly accurate and reliable with about 50% of the primary promoters being characterized by all three structural properties while at least one property identifies 95%. We also studied the relative differences of these structural features in terms of gene expression and found that the features, viz. lower stability, lesser bendability and higher curvature are more prominent in the promoter regions which are associated with high gene expression as compared with low expression genes. Hence, promoters, which are associated with higher gene expression, get annotated well using DNA structural features as compared with those, which are linked to lower gene expression.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;promoter prediction&quot;">promoter prediction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA structural properties&quot;">DNA structural properties</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene expression&quot;">gene expression</a></div>                            <h2 scrollto-destination=61462954 id="61462954" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Advances in genome sequencing technology have resulted in a large amount of raw data in the form of whole genome sequences of organisms. This sequence data needs to be annotated, viz. identification of coding regions, non-coding regions and regulatory elements. Computational tools are the only viable option for fast and fairly reliable annotation of many genome sequences. Promoter prediction is an important step in the genome annotation process; not only for the validation of the predicted genes but also for the identification of novel genes, especially those associated with non-coding RNA, which are often missed by gene prediction programs. Identification of transcription factor-binding sites (TFBSs) and promoters is essential for understanding transcription and regulation of the genes. Regulatory regions that affect the transcription initiation process are beyond the target DNA sequence motifs.<a reveal-id="dsw045-B1 dsw045-B2 dsw045-B3 dsw045-B4" data-open="dsw045-B1 dsw045-B2 dsw045-B3 dsw045-B4" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a> There are studies in which DNA structural features are integrated with motif search algorithms to identify TFBSs<a reveal-id="dsw045-B5" data-open="dsw045-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw045-B6" data-open="dsw045-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a>. DNA shape determined by four distinct features—Minor groove width, Propeller twist, Roll and Helix twist has been found to be an important determinant in the identification of TFBSs and transcription start sites (TSSs).<a reveal-id="dsw045-B7 dsw045-B8 dsw045-B9" data-open="dsw045-B7 dsw045-B8 dsw045-B9" class="link link-ref link-reveal xref-bibr"><sup>7–9</sup></a> Physico-chemical properties of DNA double helix such as hydrogen bonding, stacking energy etc. show DNA sequence functional density signatures.<a reveal-id="dsw045-B10 dsw045-B11 dsw045-B12" data-open="dsw045-B10 dsw045-B11 dsw045-B12" class="link link-ref link-reveal xref-bibr"><sup>10–12</sup></a> Promoter regions have been found to be associated with some unique structural features (like low stability, lesser bendability and more curvature etc.) across organisms.<a reveal-id="dsw045-B13 dsw045-B14 dsw045-B15 dsw045-B16" data-open="dsw045-B13 dsw045-B14 dsw045-B15 dsw045-B16" class="link link-ref link-reveal xref-bibr"><sup>13–16</sup></a> Promoter annotation based on the relative stability of DNA duplex has been found to be more reliable and accurate for a wide range of prokaryotes, compared with sequence based approaches.<a reveal-id="dsw045-B17" data-open="dsw045-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw045-B18" data-open="dsw045-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> PromBase is a comprehensive database which provides promoter predictions, based on relative stability, for whole genome sequences of bacteria and archaea with respect to their Translation Start Sites.<a reveal-id="dsw045-B19" data-open="dsw045-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Profiles of some other structural properties, such as bendability and curvature are also displayed in addition to stability. A previous study has shown that different categories of promoters (primary, secondary and internal promoters) display a gradation in their sequence dependent structural properties and which is mostly conserved for orthologous genes in ten strains of the bacterium <em>Helicobacter pylori</em><em>.</em><a reveal-id="dsw045-B20" data-open="dsw045-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Non-B DNA structural motifs are preferentially located in the regulatory regions of operons in <em>E. coli</em>.<a reveal-id="dsw045-B21" data-open="dsw045-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a>The recent study in <em>Mycoplasma pneumonia</em> shows that free energy of promoter region is an important determinant in the identification of true promoter from abortive and non promoters, apart from sequence elements such as −10, extended −10 and −35 box.<a reveal-id="dsw045-B22" data-open="dsw045-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p><p>Whole genome transcriptome studies reveal the pervasive nature of transcription and suggest that transcription is not restricted to regions upstream of the annotated coding sequences (CDSs) but can initiate from almost any genomic location.<a reveal-id="dsw045-B23" data-open="dsw045-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The wide variety of transcripts like mRNA, internal, antisense and non-coding RNA have been found to exist in the prokaryotic genome. Large fractions of these non-coding RNAs are found to be functional and are involved in various regulatory activities in the cell.<a reveal-id="dsw045-B24" data-open="dsw045-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Antisense transcripts play an important role in gene regulatory networks and constitute a secondary layer of regulatory switch, which has great potential in genetic engineering.<a reveal-id="dsw045-B25" data-open="dsw045-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw045-B26" data-open="dsw045-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Hence, annotation of non-coding transcripts is crucial for a complete understanding of functional genomics and gene regulatory circuits.<a reveal-id="dsw045-B26" data-open="dsw045-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsw045-B27" data-open="dsw045-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> To address these issues, we have studied structural properties of the promoter regions in different categories of transcripts characterized in the transcriptome of six different model organisms, viz. <em>H. pylori</em>, <em>Anabaena</em>, <em>Synechocystis</em>, <em>E. coli</em>, <em>Salmonella</em> and <em>Klebsiella</em>. The genomic GC content for these organisms varies from 39% for <em>H</em>. <em>pylori</em> to 57% for <em>Klebsiella</em> (<a reveal-id="dsw045-T1" data-open="dsw045-T1" class="link link-reveal link-table xref-fig">Table 1</a>). We used the differences in these properties between promoter regions and the flanking sequences to predict promoters associated with the different categories of transcripts such as primary, internal, antisense and non-coding RNA. Ability to predict the strength of a promoter is the next challenge in promoter annotation process. Sequence motif based approaches have been reported for <em>E. coli</em>, which predict the strength of the promoter moderately well in an organism specific manner.<a reveal-id="dsw045-B28 dsw045-B29 dsw045-B30" data-open="dsw045-B28 dsw045-B29 dsw045-B30" class="link link-ref link-reveal xref-bibr"><sup>28–30</sup></a> DNA thermodynamics and supercoiling dynamics have been found to be associated with gene expression during different bacterial growth cycle.<a reveal-id="dsw045-B31" data-open="dsw045-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw045-B32" data-open="dsw045-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Following this up, we have analysed the relationship between DNA structural features and the strength of the promoter, in terms of gene expression in six different model organisms. <div content-id="dsw045-T1" class="table-modal"><div class="table-wrap"><div id="dsw045-T1" data-id="dsw045-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p> Genomic features and number of TSSs of various categories in six different model organisms used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organisms </th><th>Genome size (in Mb) </th><th>Genome GC (in %) </th><th>Protein genes </th><th>RNA genes </th><th>gTSS </th><th>iTSS </th><th>aTSS </th><th>nTSS </th></tr></thead><tbody><tr><td><strong><em>H. pylori<sup>33</sup></em></strong> </td><td char=".">1.7 </td><td char=".">38.9 </td><td char=".">1,469 </td><td char=".">43 </td><td char=".">714 </td><td char=".">426 </td><td char=".">1,018 </td><td char=".">NA </td></tr><tr><td><strong><em>Anabaena<sup>34</sup></em></strong> </td><td char=".">6.4 </td><td char=".">41.4 </td><td char=".">5,365 </td><td char=".">66 </td><td char=".">2,517 </td><td char=".">1,878 </td><td char=".">2,196 </td><td char=".">1,266 </td></tr><tr><td><strong><em>Synechocystis<sup>35</sup></em></strong> </td><td char=".">3.6 </td><td char=".">47.7 </td><td char=".">3,179 </td><td char=".">50 </td><td char=".">1,639 </td><td char=".">125 </td><td char=".">1,356 </td><td char=".">170 </td></tr><tr><td><strong><em>E. coli<sup>36</sup></em></strong> </td><td char=".">4.6 </td><td char=".">50.8 </td><td char=".">4,145 </td><td char=".">175 </td><td char=".">1,333 </td><td char=".">1,184 </td><td char=".">NA </td><td char=".">NA </td></tr><tr><td><strong><em>Salmonella<sup>37</sup></em></strong> </td><td char=".">4.9 </td><td char=".">52.2 </td><td char=".">4,446 </td><td char=".">109 </td><td char=".">981 </td><td char=".">238 </td><td char=".">171 </td><td char=".">NA </td></tr><tr><td><strong><em>Klebsiella<sup>36</sup></em></strong> </td><td char=".">5.3 </td><td char=".">57.5 </td><td char=".">4,776 </td><td char=".">111 </td><td char=".">1,329 </td><td char=".">837 </td><td char=".">NA </td><td char=".">NA </td></tr></tbody></table></div><div content-id="dsw045-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Organisms </th><th>Genome size (in Mb) </th><th>Genome GC (in %) </th><th>Protein genes </th><th>RNA genes </th><th>gTSS </th><th>iTSS </th><th>aTSS </th><th>nTSS </th></tr></thead><tbody><tr><td><strong><em>H. pylori<sup>33</sup></em></strong> </td><td char=".">1.7 </td><td char=".">38.9 </td><td char=".">1,469 </td><td char=".">43 </td><td char=".">714 </td><td char=".">426 </td><td char=".">1,018 </td><td char=".">NA </td></tr><tr><td><strong><em>Anabaena<sup>34</sup></em></strong> </td><td char=".">6.4 </td><td char=".">41.4 </td><td char=".">5,365 </td><td char=".">66 </td><td char=".">2,517 </td><td char=".">1,878 </td><td char=".">2,196 </td><td char=".">1,266 </td></tr><tr><td><strong><em>Synechocystis<sup>35</sup></em></strong> </td><td char=".">3.6 </td><td char=".">47.7 </td><td char=".">3,179 </td><td char=".">50 </td><td char=".">1,639 </td><td char=".">125 </td><td char=".">1,356 </td><td char=".">170 </td></tr><tr><td><strong><em>E. coli<sup>36</sup></em></strong> </td><td char=".">4.6 </td><td char=".">50.8 </td><td char=".">4,145 </td><td char=".">175 </td><td char=".">1,333 </td><td char=".">1,184 </td><td char=".">NA </td><td char=".">NA </td></tr><tr><td><strong><em>Salmonella<sup>37</sup></em></strong> </td><td char=".">4.9 </td><td char=".">52.2 </td><td char=".">4,446 </td><td char=".">109 </td><td char=".">981 </td><td char=".">238 </td><td char=".">171 </td><td char=".">NA </td></tr><tr><td><strong><em>Klebsiella<sup>36</sup></em></strong> </td><td char=".">5.3 </td><td char=".">57.5 </td><td char=".">4,776 </td><td char=".">111 </td><td char=".">1,329 </td><td char=".">837 </td><td char=".">NA </td><td char=".">NA </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Abbreviations used: gTSS, primary TSS; iTSS, internal TSS; aTSS, antisense TSS; nTSS, non-coding TSS.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw045-T1" data-id="dsw045-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p> Genomic features and number of TSSs of various categories in six different model organisms used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organisms </th><th>Genome size (in Mb) </th><th>Genome GC (in %) </th><th>Protein genes </th><th>RNA genes </th><th>gTSS </th><th>iTSS </th><th>aTSS </th><th>nTSS </th></tr></thead><tbody><tr><td><strong><em>H. pylori<sup>33</sup></em></strong> </td><td char=".">1.7 </td><td char=".">38.9 </td><td char=".">1,469 </td><td char=".">43 </td><td char=".">714 </td><td char=".">426 </td><td char=".">1,018 </td><td char=".">NA </td></tr><tr><td><strong><em>Anabaena<sup>34</sup></em></strong> </td><td char=".">6.4 </td><td char=".">41.4 </td><td char=".">5,365 </td><td char=".">66 </td><td char=".">2,517 </td><td char=".">1,878 </td><td char=".">2,196 </td><td char=".">1,266 </td></tr><tr><td><strong><em>Synechocystis<sup>35</sup></em></strong> </td><td char=".">3.6 </td><td char=".">47.7 </td><td char=".">3,179 </td><td char=".">50 </td><td char=".">1,639 </td><td char=".">125 </td><td char=".">1,356 </td><td char=".">170 </td></tr><tr><td><strong><em>E. coli<sup>36</sup></em></strong> </td><td char=".">4.6 </td><td char=".">50.8 </td><td char=".">4,145 </td><td char=".">175 </td><td char=".">1,333 </td><td char=".">1,184 </td><td char=".">NA </td><td char=".">NA </td></tr><tr><td><strong><em>Salmonella<sup>37</sup></em></strong> </td><td char=".">4.9 </td><td char=".">52.2 </td><td char=".">4,446 </td><td char=".">109 </td><td char=".">981 </td><td char=".">238 </td><td char=".">171 </td><td char=".">NA </td></tr><tr><td><strong><em>Klebsiella<sup>36</sup></em></strong> </td><td char=".">5.3 </td><td char=".">57.5 </td><td char=".">4,776 </td><td char=".">111 </td><td char=".">1,329 </td><td char=".">837 </td><td char=".">NA </td><td char=".">NA </td></tr></tbody></table></div><div content-id="dsw045-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Organisms </th><th>Genome size (in Mb) </th><th>Genome GC (in %) </th><th>Protein genes </th><th>RNA genes </th><th>gTSS </th><th>iTSS </th><th>aTSS </th><th>nTSS </th></tr></thead><tbody><tr><td><strong><em>H. pylori<sup>33</sup></em></strong> </td><td char=".">1.7 </td><td char=".">38.9 </td><td char=".">1,469 </td><td char=".">43 </td><td char=".">714 </td><td char=".">426 </td><td char=".">1,018 </td><td char=".">NA </td></tr><tr><td><strong><em>Anabaena<sup>34</sup></em></strong> </td><td char=".">6.4 </td><td char=".">41.4 </td><td char=".">5,365 </td><td char=".">66 </td><td char=".">2,517 </td><td char=".">1,878 </td><td char=".">2,196 </td><td char=".">1,266 </td></tr><tr><td><strong><em>Synechocystis<sup>35</sup></em></strong> </td><td char=".">3.6 </td><td char=".">47.7 </td><td char=".">3,179 </td><td char=".">50 </td><td char=".">1,639 </td><td char=".">125 </td><td char=".">1,356 </td><td char=".">170 </td></tr><tr><td><strong><em>E. coli<sup>36</sup></em></strong> </td><td char=".">4.6 </td><td char=".">50.8 </td><td char=".">4,145 </td><td char=".">175 </td><td char=".">1,333 </td><td char=".">1,184 </td><td char=".">NA </td><td char=".">NA </td></tr><tr><td><strong><em>Salmonella<sup>37</sup></em></strong> </td><td char=".">4.9 </td><td char=".">52.2 </td><td char=".">4,446 </td><td char=".">109 </td><td char=".">981 </td><td char=".">238 </td><td char=".">171 </td><td char=".">NA </td></tr><tr><td><strong><em>Klebsiella<sup>36</sup></em></strong> </td><td char=".">5.3 </td><td char=".">57.5 </td><td char=".">4,776 </td><td char=".">111 </td><td char=".">1,329 </td><td char=".">837 </td><td char=".">NA </td><td char=".">NA </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Abbreviations used: gTSS, primary TSS; iTSS, internal TSS; aTSS, antisense TSS; nTSS, non-coding TSS.</p></div></div></div></div></p>                            <h2 scrollto-destination=61462957 id="61462957" class="section-title" >2. Material and methods</h2>
                            <h3 scrollto-destination=61462958 id="61462958" class="section-title" >2.1. Promoter sequence dataset preparation</h3>
<p>TSS information of <em>H</em><em>.</em><em>pylori</em> strain 26695<a reveal-id="dsw045-B33" data-open="dsw045-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> (<em>H</em><em>.</em><em>pylori</em>), <em>Anabaena</em> sp. PCC7120<a reveal-id="dsw045-B34" data-open="dsw045-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> (<em>Anabaena</em>), <em>Synechocystis</em> sp. PCC 6803<a reveal-id="dsw045-B35" data-open="dsw045-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> (<em>Synechocystis</em>), <em>E</em><em>.</em><em>coli</em> K-12 MG1655<a reveal-id="dsw045-B36" data-open="dsw045-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> (<em>E. coli</em>), <em>Salmonella enterica</em> serovar Typhimurium<a reveal-id="dsw045-B37" data-open="dsw045-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> (<em>Salmonella</em>) and <em>Klebsiella pneumoniae</em><a reveal-id="dsw045-B36" data-open="dsw045-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> (<em>Klebsiella</em>) was obtained from the relevant published data. For each gene (protein/RNA), TSS with highest number of reads was selected for creating a unique dataset of primary/gTSS (transcribing mRNA), iTSS (TSSs present inside the gene), aTSS (corresponding to antisense transcripts, encompassed within the gene), whereas all available nTSS (non-coding transcripts) were taken (<a reveal-id="dsw045-T1" data-open="dsw045-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Only those categories of TSSs, that had more than 100 TSSs, were selected for the study. Whole genome sequence and translation start site (TLS) information were downloaded from ftp site (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genomes/Bacteria/" target="_blank">ftp://ftp.ncbi.nih.gov/genomes/Bacteria/</a>) of NCBI. Genome sequences of 1001 nucleotides length (spanning 500 nucleotides upstream and downstream of the TSS positioned at 0) were extracted from the whole genome sequence using TSS information as provided in published data.</p><p>Non-promoter sequence or negative dataset was created, by taking the CDS of the genes associated with gTSS (primary TSS).</p>                            <h3 scrollto-destination=61462961 id="61462961" class="section-title" >2.2. Estimation of structural properties of promoter sequences</h3>
<p>We have studied three sequence dependent structural properties stability, bendability and curvature. Stability of a piece of double stranded DNA sequence molecule can be calculated in terms of the free energy of its constituent dinucleotides. The free energy values of the 10 unique dinucleotides were taken from data based on melting studies of oligo and polynucleotides.<a reveal-id="dsw045-B38" data-open="dsw045-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Stability profile of 1001 nucleotides long promoter sequence was calculated by dividing each sequence in sliding windows of 15 base pairs each (or 14 dinucleotide steps) using the free energy values for the constituent dinucleotides. Average free energy (AFE) (in kcal/mol) was calculated at each nucleotide position after aligning all TSS at 0 position.<a reveal-id="dsw045-B39" data-open="dsw045-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a></p><p>Bendability of promoter sequences was calculated by using two different trinucleotide models, DNase I sensitivity<a reveal-id="dsw045-B40" data-open="dsw045-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (DNase I) and nucleosomal positioning preference (NPP).<a reveal-id="dsw045-B41" data-open="dsw045-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Bendability profile was calculated using a 30 nucleotides window size.</p><p>Curvature of promoter sequence was estimated by the calculated value of d/lmax (the ratio of minimum end to end distance ‘d’ to the length of a DNA fragment ‘lmax’)<a reveal-id="dsw045-B42" data-open="dsw045-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> for the models generated using a set of dinucleotide values obtained from gel retardation mobility assay of multiple polymeric DNA sequences (BMHT).<a reveal-id="dsw045-B43" data-open="dsw045-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The DNA models were generated using 75 nucleotides window size and d/lmax vale was calculated by using in-house program NUCGEN.<a reveal-id="dsw045-B44" data-open="dsw045-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Values of d/lmax ranges from 0 (completely closed circle indicating high curvature) to 1 (perfectly linear DNA).</p>                            <h3 scrollto-destination=61462965 id="61462965" class="section-title" >2.3. Promoter prediction methodology</h3>
<p>Promoter sequences of 1001 nucleotide length from six different bacteria were categorized on the basis of their GC composition (at 5% GC intervals). Sequence information was translated to numerical information in terms of structural properties mentioned above. The training set of promoter sequences were selected from the primary TSS promoters, associated with gTSS. A 5-fold analysis was performed, by selecting 80% as training dataset and it was applied on full dataset. Two independent cycles of cutoff derivation and promoter predictions were performed using 75 and 100 nucleotide long windows, respectively, for all given structural features.</p>                            <h4 scrollto-destination=61462967 id="61462967" class="section-title" >2.3.1. Cutoff derivations</h4>
<p>The more prominent low stability peak was observed in the vicinity of TSS across all bacterium species. Hence, the region −55 to + 20 (with respect to TSS) was selected to derive one of the cutoff values (STB1) for the first cycle while −80 to + 20 region was taken for the second cycle. Promoters were predicted by comparing structural properties of neighboring regions; hence a 75 nucleotides window from +200 to + 275 with respect to TSSs and 100 nucleotides window from +200 to + 300 with respect to TSSs were chosen to derive second cutoff value (STB2) for stability for cycle one and cycle two respectively. Less bendability and high curvature span were observed in the upstream regions of TSSs across all the organisms. Hence −100 to −25 and −125 to −25 nucleotides with respect to TSSs were considered for derivation of first cutoff values for bendability (both models BDC1 and BNC1) and curvature (CBC1), for the first and second cycle of predictions respectively. Similarly, the second set of cutoff values for bendability (BDC2 and BNC2) and curvature (CBC2) were derived from +200 to + 275 and +200 to + 300 nucleotides with respect to TSSs for the first and second cycle of predictions, respectively.</p>                            <h4 scrollto-destination=61462969 id="61462969" class="section-title" >2.3.2. Promoter prediction</h4>
<p>The cutoff values derived at 5%GC interval for all DNA sequence dependent structural properties were applied within −200 to + 100 nucleotide region (R1) and +200 to + 500 nucleotide region (R2) with respect to TSS in the full dataset. A given sequence was identified as a promoter sequence if a particular sequence dependent structural property was found to be present and satisfies the following set of conditions. <ul class="list-simple"><li><p>R1 ≥ STB1 &amp; R2 ≤ STB2 for stability</p></li><li><p>R1≤ BDC1 &amp; R2 ≥ BDC2 for bendability (DNase I)</p></li><li><p>R1 ≥ BNC1 &amp; R2 ≤ BNC2 for bendability (NPP)</p></li><li><p>R1 ≤ CBC1 &amp; R2 ≥ CBC2 for curvature</p></li></ul></p><p>Sequences satisfying the above conditions were assigned as true positive (TP) for the representative feature, else they were classified as false negative (FN). In the negative dataset, +200 to + 500 nucleotide region was scanned for conditions mentioned earlier. If a sequence from negative dataset was found to satisfy above criteria then it was designated as false positive (FP) for the respective property, else it was tagged as true negative (TN).</p><a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mtext>Sensitivity</mtext><mo>=</mo><mfrac><mrow><mn>100</mn><mo>×</mo><mtext>TP</mtext></mrow><mrow><mtext>TP</mtext><mo>+</mo><mtext>FN</mtext></mrow></mfrac></mrow></math></div></div></div><a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mtext>Specificity</mtext><mo>=</mo><mfrac><mrow><mn>100</mn><mo>×</mo><mtext>TN</mtext></mrow><mrow><mtext>TN</mtext><mo>+</mo><mtext>FP</mtext></mrow></mfrac></mrow></math></div></div></div><a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mtext>Precision</mtext><mo>=</mo><mfrac><mrow><mn>100</mn><mo>×</mo><mtext>TP</mtext></mrow><mrow><mtext>TP</mtext><mo>+</mo><mtext>FP</mtext></mrow></mfrac></mrow></math></div></div></div><a id="E4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mtext>Balanced</mtext><mo>−</mo><mtext>Accuracy</mtext><mo>=</mo><mfrac><mrow><mtext>Sensitivity×Specificity</mtext></mrow><mn>2</mn></mfrac></mrow></math></div></div></div><a id="E5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>F</mi><mo>−</mo><mtext>score</mtext><mo>=</mo><mfrac><mrow><mn>2</mn><mo>×</mo><mtext>Sensitivity</mtext><mo>=</mo><mtext>Precision</mtext></mrow><mrow><mtext>Sensitivity</mtext><mo>+</mo><mtext>Precision</mtext></mrow></mfrac></mrow></math></div></div></div><p>TP and TN rates were defined by calculating sensitivity (also known as recall) and specificity respectively, to measure the performance of binary classification test. Fidelity of the search was determined by calculating precision. Evaluation of classifier was done, by calculating balanced–accuracy and <em>F</em>-score, which are arithmetic and harmonic means of sensitivity and precision respectively.</p>                            <h3 scrollto-destination=61462978 id="61462978" class="section-title" >2.4. Gene expression estimation</h3>
<p>Microarray expression profiling data (RNA sample from wild type) <em>H. pylori</em> (GSM931753), <em>Synechocystis</em> (GSM1316725), <em>E. coli</em> (GSM1374996), <em>Salmonella</em> (GSM1102757) and <em>Klebsiella</em> (GSM877524) was downloaded from GEO (Gene Expression Omnibus), NCBI.<a reveal-id="dsw045-B45" data-open="dsw045-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a>. Differential expression was calculated by using DESeq (R/Bioconductor package)<a reveal-id="dsw045-B46" data-open="dsw045-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> to measure the change in the expression fold in the absence of Nitrogen from transcriptome data.<a reveal-id="dsw045-B34" data-open="dsw045-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Primary TSSs (gTSS) were sorted in descending order (in the terms of their expression level) and top 10 percentile were considered as high expression genes expected to be associated with strong promoters. The bottom 10 percentile was taken as a dataset for low expression genes with weak promoters. Structural features were calculated for the sequences extracted using TSS location and strand information from the whole genome sequence.</p>                            <h3 scrollto-destination=61462980 id="61462980" class="section-title" >2.5. Statistical methods</h3>
<p>Statistical significance of structural properties enrichment in the promoter regions of highly expressed genes as compared with lowly expressed genes were analyzed using the Kolmogorov-Smirnov test (KS test). One-sided two-sample KS test was performed to test the significance of the difference in feature value between the two classes (here null hypothesis was that the both datasets have similar CDF distribution). Null hypothesis was rejected at the significance level of 1%. MATLAB was used for statistical analysis.</p>                            <h2 scrollto-destination=61462982 id="61462982" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=61462983 id="61462983" class="section-title" >3.1 Promoter regions corresponding to prokaryotic transcriptomes show distinct structural properties</h3>
<div class="block-child-p">We have characterized DNA duplex stability (AFE), bendability using two different models DNase I sensitivity (DNase I) and NPP and intrinsic curvature in the promoter region of different categories of TSSs in six different model organisms. DNase I sensitivity model gives the bending propensity of each trinucleotide in terms of bending towards major groove while NPP model gives the rotational preference of each trinucleotide towards histone core. The DNA structural profiles show that promoter (primary) regions are in general less stable, less bendable and more curved as compared to neighboring regions (<a reveal-id="dsw045-F1" data-open="dsw045-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). DNA duplex stability, expressed in the terms of AFE (kcal/mol) profile, shows a low stability peak in the vicinity of the TSS across organisms with varying genomic GC content. With increase in genomic GC% from <em>H. pylori</em> with 39% to <em>Klebsiella</em> with 57% the baseline shifts to higher stability (lower AFE) values, while the low stability peak splits. The secondary low stability peak is likely to be associated with −35 element. Bendability profiles as calculated using two different trinucleotide models (DNase I and NPP), show similar low bendability peaks in the region upstream of the TSS. Curvature profile shows that the promoter region is more intrinsically curved as compared with flanking regions. <div data-id="dsw045-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f1.png?Expires=2147483647&Signature=ysTi0Gd79aTBYNj9~z9OEwm11rucn6Jbymk0fuG5jSlmYeH9Tih1CecCkKKQPl1AWiEAhooarvHCeJrDBa0Y7WXF0IL7MMfHBwP~MJ~aTx3880rIr-o4GO9UGsbpZQpy5yZqPEBH4huO-eqSm7-1lHNnkoZWKaiCS-3KLXZzKnpvKAviIX04kBzRz4dwyghZKS~Upuld4N5LIFL-AwsR5o1oWzrekH6CYrVdmJBlbbs9RDEDFDzDCVGBg4h7xCjd7Kc1klGslTc5v~XPtoMJqjQFplWUz-Bnk1RoXttb47MNc09ORfsqLUSxZdHw~vA~aOph5wN4kxYJa1qijBEImg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA sequence dependent structural properties: AFE, DNase I sensitivity, NPP and curvature profiles for 1001 nucleotides long primary (gTSS) promoter sequences (extending from −500 to + 500 nucleotides with respect to TSS at 0). The black colour plots correspond to profiles of structural feature for shuffled sequences (upstream −500 to − 100, promoter region −80 to + 20 and downstream +100 to + 500 nucleotide position with respect to TSS at 0). The number of promoter sequences and whole genome GC% is mentioned along with the organism name, on the left of each row." path-from-xml="dsw045f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462984/dsw045f1.tif" path-from-xml="dsw045f1.tif" target="_blank">View large</a><a section="61462984" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f1.png?Expires=2147483647&Signature=ajyn55Y849WgLevCVfDUJ7dc6J0AzkyFwmK3Zh9uYtnqHXNsyv-iBPjgwbZZojquTlPbr8zgDRRT3jywQuiyRi-lS4y4V4HReNeD4nuH8x8siaMIRF6HXX9JZBUWkD13~5kMNVMY-J0SPjkxda0WLnZH-boA74rX3vxO1ksRbEFHJgCqi9ZmCk7STc6TxhMivmQimYKmTmdAMPgu5UlgGdgf2ERZ1Bp9QliE95Yw72Q8aN58CnC5SCT2EetKVlVHINyUjLzdj0RQlw221Tk9vP1pUmL8OUY5qMbtC1ZL7oQqG5gX5HNTdrhK6ni-PbdG5lQ~sJ9SukDeWDeVjcBj4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462984&ar=2449750&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw045f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA sequence dependent structural properties: AFE, DNase I sensitivity, NPP and curvature profiles for 1001 nucleotides long primary (gTSS) promoter sequences (extending from −500 to + 500 nucleotides with respect to TSS at 0). The black colour plots correspond to profiles of structural feature for shuffled sequences (upstream −500 to − 100, promoter region −80 to + 20 and downstream +100 to + 500 nucleotide position with respect to TSS at 0). The number of promoter sequences and whole genome GC% is mentioned along with the organism name, on the left of each row.</p></div></div><div content-id="dsw045-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f1.png?Expires=2147483647&Signature=ysTi0Gd79aTBYNj9~z9OEwm11rucn6Jbymk0fuG5jSlmYeH9Tih1CecCkKKQPl1AWiEAhooarvHCeJrDBa0Y7WXF0IL7MMfHBwP~MJ~aTx3880rIr-o4GO9UGsbpZQpy5yZqPEBH4huO-eqSm7-1lHNnkoZWKaiCS-3KLXZzKnpvKAviIX04kBzRz4dwyghZKS~Upuld4N5LIFL-AwsR5o1oWzrekH6CYrVdmJBlbbs9RDEDFDzDCVGBg4h7xCjd7Kc1klGslTc5v~XPtoMJqjQFplWUz-Bnk1RoXttb47MNc09ORfsqLUSxZdHw~vA~aOph5wN4kxYJa1qijBEImg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA sequence dependent structural properties: AFE, DNase I sensitivity, NPP and curvature profiles for 1001 nucleotides long primary (gTSS) promoter sequences (extending from −500 to + 500 nucleotides with respect to TSS at 0). The black colour plots correspond to profiles of structural feature for shuffled sequences (upstream −500 to − 100, promoter region −80 to + 20 and downstream +100 to + 500 nucleotide position with respect to TSS at 0). The number of promoter sequences and whole genome GC% is mentioned along with the organism name, on the left of each row." path-from-xml="dsw045f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462984/dsw045f1.tif" path-from-xml="dsw045f1.tif" target="_blank">View large</a><a section="61462984" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f1.png?Expires=2147483647&Signature=ajyn55Y849WgLevCVfDUJ7dc6J0AzkyFwmK3Zh9uYtnqHXNsyv-iBPjgwbZZojquTlPbr8zgDRRT3jywQuiyRi-lS4y4V4HReNeD4nuH8x8siaMIRF6HXX9JZBUWkD13~5kMNVMY-J0SPjkxda0WLnZH-boA74rX3vxO1ksRbEFHJgCqi9ZmCk7STc6TxhMivmQimYKmTmdAMPgu5UlgGdgf2ERZ1Bp9QliE95Yw72Q8aN58CnC5SCT2EetKVlVHINyUjLzdj0RQlw221Tk9vP1pUmL8OUY5qMbtC1ZL7oQqG5gX5HNTdrhK6ni-PbdG5lQ~sJ9SukDeWDeVjcBj4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462984&ar=2449750&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw045f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA sequence dependent structural properties: AFE, DNase I sensitivity, NPP and curvature profiles for 1001 nucleotides long primary (gTSS) promoter sequences (extending from −500 to + 500 nucleotides with respect to TSS at 0). The black colour plots correspond to profiles of structural feature for shuffled sequences (upstream −500 to − 100, promoter region −80 to + 20 and downstream +100 to + 500 nucleotide position with respect to TSS at 0). The number of promoter sequences and whole genome GC% is mentioned along with the organism name, on the left of each row.</p></div></div></div><p>The shuffled promoter sequences obtained by the randomization of native promoter sequences do not show these properties, even when they were estimated separately for three different regions: upstream (−500 to −100 nucleotide), promoter region (−80 to +20 nucleotide) and downstream region (+100 to +500) nucleotide with respect to TSS at 0 position in order to retain the GC composition of these regions. Interestingly genomic sequences in the promoter, as well as the non-promoter region, are overall less bendable and more curved than the shuffled sequences. On the other hand, while genomic promoter sequences are less stable than their shuffled counterparts, the flanking genomic sequences are more stable. The loss of special structural features in shuffled sequences suggests that these features arise due to specific base sequence patterns.</p><div class="block-child-p">Similar structural profiles were observed for internal promoters, antisense promoters and non-coding RNA gene promoters (<a reveal-id="dsw045-F2" data-open="dsw045-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Stability profiles show sharp low stability peaks in the case of internal, antisense and non-coding RNA promoters as compared with primary promoters across all organisms. Bendability and curvature profiles also show distinct features in their promoter regions, but they are less prominent for these categories of promoters as compared with primary promoters. Base composition analysis shows that primary category of promoters (−100 to 0 nucleotide with respect to TSS at 0) show preference for several AT rich tetramers as compared with background region (−500 to +500 nucleotide with respect to TSS at 0) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&amp;Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Internal and antisense categories of promoters show less preference for AT rich tetramers and TATA sequence motifs as compared with primary and non-coding RNA promoters. Core promoter region (−35 and −10 element) shows poor sequence conservation even for promoters associated with primary category of TSSs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&amp;Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) and becomes poorer for internal and antisense categories of promoters as compared with the primary category across the six different model organisms with varying genomic GC content. Interestingly no sequence motif signature was observed in their -35 region, while a secondary low stability peak was observed in the same region for primary category of TSSs across the organisms. The weak preference for AT rich tetramers and poor sequence motif conservation in core promoter elements (−35 and −10 elements) makes it difficult to predict these promoters by sequence motif based approaches. <div data-id="dsw045-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f2.png?Expires=2147483647&amp;Signature=aiqQH3ts66v3iuKwZyvDpG588WGIcy61TEmM0e5w5H32vE-FzTxLAoxO97acjtoy6B1YXoR5nGrDUv97OI1hC5gW3GPxP9zHIPGuIeVo1z0EyV65x5t6pntb1y5HAqVimQ8yHBdGxr-RdZrJk1ZqLU41Mhaj~btDnhC9OV6GrKI7qPlLWWEREn4LjwSm6np6GnNwXo0W0-ODK-Y-HYz4uJgqm9Cdn3X0QLSxzZWt~nqduZK4Ap61B1q3cn-a8jb1Sgpxw9BdmiG0GBidNqvpMzKmGhvPSPM0T6QbJSw5ZJVQgpHcvgo-RSIDEEOcV3pmxrf3TJBZXdO~J0sAxXP9pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA sequence dependent structural properties: AFE, DNase I sensitivity, NPP and curvature profiles for 1001 nucleotides long internal (rows 1-6), antisense (rows 7-10) and Non-coding (rows 11-12) promoter sequences (extending from −500 to + 500 nucleotides with respect to TSS at 0). The black colour plots correspond to profiles of structural feature for shuffled sequences (upstream −500 to − 100, promoter region −80 to + 20 and downstream +100 to + 500 nucleotide position with respect to TSS at 0). The number of promoter sequences and whole genome GC% is mentioned along with the organism name, on the left of each row." path-from-xml="dsw045f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462986/dsw045f2.tif" path-from-xml="dsw045f2.tif" target="_blank">View large</a><a section="61462986" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f2.png?Expires=2147483647&amp;Signature=Z-A4Fmbdj~nXbl-eZfwNN-GUDR0P5GvcGxekpD7qWFkWytgnnXGGzaslcWO2vjoJLBbqZ~aPGNy3l3DSaXQdMHNJoDg26KA8CiZveoodFn7i0tBCB0B-xBeLKUbaQp9indhSphfWNK0~XNQtLTG8Vsn8CbUIpR78gDs8OysJahwGPYsa6rkNAVGvYZFF-wTE9pVwf8IAuG4EDzJfPJNOyUxWnG1Et0rcPLdScGnKPdNwTL979xwJM4QdeW3G0IkYlXTw0gwwqRfjN6cpzysX4FW4acjSKWWQoDEAPXCXul6mQfWnwcgPgB9jyM5etxujhjpVRs8ioXqwbTejo1mqnw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462986&amp;ar=2449750&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw045f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA sequence dependent structural properties: AFE, DNase I sensitivity, NPP and curvature profiles for 1001 nucleotides long internal (rows 1-6), antisense (rows 7-10) and Non-coding (rows 11-12) promoter sequences (extending from −500 to + 500 nucleotides with respect to TSS at 0). The black colour plots correspond to profiles of structural feature for shuffled sequences (upstream −500 to − 100, promoter region −80 to + 20 and downstream +100 to + 500 nucleotide position with respect to TSS at 0). The number of promoter sequences and whole genome GC% is mentioned along with the organism name, on the left of each row.</p></div></div><div content-id="dsw045-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f2.png?Expires=2147483647&amp;Signature=aiqQH3ts66v3iuKwZyvDpG588WGIcy61TEmM0e5w5H32vE-FzTxLAoxO97acjtoy6B1YXoR5nGrDUv97OI1hC5gW3GPxP9zHIPGuIeVo1z0EyV65x5t6pntb1y5HAqVimQ8yHBdGxr-RdZrJk1ZqLU41Mhaj~btDnhC9OV6GrKI7qPlLWWEREn4LjwSm6np6GnNwXo0W0-ODK-Y-HYz4uJgqm9Cdn3X0QLSxzZWt~nqduZK4Ap61B1q3cn-a8jb1Sgpxw9BdmiG0GBidNqvpMzKmGhvPSPM0T6QbJSw5ZJVQgpHcvgo-RSIDEEOcV3pmxrf3TJBZXdO~J0sAxXP9pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA sequence dependent structural properties: AFE, DNase I sensitivity, NPP and curvature profiles for 1001 nucleotides long internal (rows 1-6), antisense (rows 7-10) and Non-coding (rows 11-12) promoter sequences (extending from −500 to + 500 nucleotides with respect to TSS at 0). The black colour plots correspond to profiles of structural feature for shuffled sequences (upstream −500 to − 100, promoter region −80 to + 20 and downstream +100 to + 500 nucleotide position with respect to TSS at 0). The number of promoter sequences and whole genome GC% is mentioned along with the organism name, on the left of each row." path-from-xml="dsw045f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462986/dsw045f2.tif" path-from-xml="dsw045f2.tif" target="_blank">View large</a><a section="61462986" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f2.png?Expires=2147483647&amp;Signature=Z-A4Fmbdj~nXbl-eZfwNN-GUDR0P5GvcGxekpD7qWFkWytgnnXGGzaslcWO2vjoJLBbqZ~aPGNy3l3DSaXQdMHNJoDg26KA8CiZveoodFn7i0tBCB0B-xBeLKUbaQp9indhSphfWNK0~XNQtLTG8Vsn8CbUIpR78gDs8OysJahwGPYsa6rkNAVGvYZFF-wTE9pVwf8IAuG4EDzJfPJNOyUxWnG1Et0rcPLdScGnKPdNwTL979xwJM4QdeW3G0IkYlXTw0gwwqRfjN6cpzysX4FW4acjSKWWQoDEAPXCXul6mQfWnwcgPgB9jyM5etxujhjpVRs8ioXqwbTejo1mqnw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462986&amp;ar=2449750&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw045f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA sequence dependent structural properties: AFE, DNase I sensitivity, NPP and curvature profiles for 1001 nucleotides long internal (rows 1-6), antisense (rows 7-10) and Non-coding (rows 11-12) promoter sequences (extending from −500 to + 500 nucleotides with respect to TSS at 0). The black colour plots correspond to profiles of structural feature for shuffled sequences (upstream −500 to − 100, promoter region −80 to + 20 and downstream +100 to + 500 nucleotide position with respect to TSS at 0). The number of promoter sequences and whole genome GC% is mentioned along with the organism name, on the left of each row.</p></div></div></div>                            <h3 scrollto-destination=61462987 id="61462987" class="section-title" >3.2 Predicting promoters associated with different categories of TSSs using structural features</h3>
<div class="block-child-p">Promoters were predicted in the full set of 1001 nucleotides long sequences, flanking the annotated TSSs of all categories (primary, internal, antisense and non-coding). Two cycles of predictions were performed using 75 and 100 nucleotides window size (using two different sets of cut-off values derived using 75 and 100 nucleotides fragment length for cycle one and two, respectively). The sensitivity and specificity achieved in each cycle, using the four different properties are shown in <a reveal-id="dsw045-F3" data-open="dsw045-F3" class="link link-reveal link-table xref-fig">Figure 3</a> for all six organisms. Higher specificity was achieved at the cost of sensitivity in the second cycle of prediction, which uses larger fragment length of DNA sequence and most importantly, it was found to be valid for all structural features in primary category of promoters. Bendability (DNase I sensitivity model) and curvature were found to be equally sensitive but less specific in nature as compared with stability in all six model organisms. <div data-id="dsw045-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f3.png?Expires=2147483647&Signature=T3jFn1IWwWGXH2eLFer3~YgFmWf2L5L6RX0LALiCl77HSyrRvyhDpTnAl~nOGiyY7Urb5vE2oTwCfbSy5BTwyep3c~G4Bx9CZ5VxomVzevLyN6l4hrh--Xc~ZYuG1nPPCjcB0hiqj1mQGIqsM9cvJ1ZhF~MR4R8AU7FyNnlpBPNvx3coAntsE6s4gnycZmnognEC8XjOLD6h75FNw8euZHb8iM~Dhw2Jp6QNUBHBcTM4NlUvKqAV6seSh7t9jAgSw1695dWX8C8Hd4MSQYCqCbky3JHMUHVD8pAPyVrQvuYOY71YAZFl0ZDOzuAo1Fpbw~ddmh2TY1MqHRoKYHRE1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction statistics using sequence dependent structural properties (STB, stability; BDC, bendability using DNase I sensitivity; BNC, bendability using NPP model; CBC, curvature). (A) The performance of the classifier for primary promoters. Markers in Square and Diamond shape represent the first and second cycle of prediction, respectively. (B) Sensitivity values obtained for internal, antisense and Non-coding promoters, top and bottom row correspond to the first and second cycle of prediction respectively. Error bars represent the SD obtained from the 5-fold analysis." path-from-xml="dsw045f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462988/dsw045f3.tif" path-from-xml="dsw045f3.tif" target="_blank">View large</a><a section="61462988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f3.png?Expires=2147483647&Signature=2uHzR6SluBQuZl~NsI8aVgNI6ephv-r1xkgrbiQrBTWwx6s7gB-nD9Ga7nfH1oLtzLznwu-8FgIN8XxOXwHlpTBwWfaiksDUtJwvDfdkVYhZZ10B3WJRMstr6~aubEMxogNJjS4BUb4RAZ6NK38lkuAJ6SAHGWUeh9Q15fPJtn~fdsLe11LSr7k9YgZdlQU4xEIusWPPmQqYnwFypRhnfm715mwuTEJ6L6nk2VwKuRiqs4y8X5HG~eDQ4GgOUoDP-hUBMNXIJXmC7d1VPC4GUMqJcuiduydVF5RbMBghwZeaE9GDiTkDyx0PAyJb4CNF5P5rSdGAw9ouLmBRJPRc5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462988&ar=2449750&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw045f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction statistics using sequence dependent structural properties (STB, stability; BDC, bendability using DNase I sensitivity; BNC, bendability using NPP model; CBC, curvature). <strong>(A)</strong> The performance of the classifier for primary promoters. Markers in Square and Diamond shape represent the first and second cycle of prediction, respectively. <strong>(B)</strong> Sensitivity values obtained for internal, antisense and Non-coding promoters, top and bottom row correspond to the first and second cycle of prediction respectively. Error bars represent the SD obtained from the 5-fold analysis.</p></div></div><div content-id="dsw045-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f3.png?Expires=2147483647&Signature=T3jFn1IWwWGXH2eLFer3~YgFmWf2L5L6RX0LALiCl77HSyrRvyhDpTnAl~nOGiyY7Urb5vE2oTwCfbSy5BTwyep3c~G4Bx9CZ5VxomVzevLyN6l4hrh--Xc~ZYuG1nPPCjcB0hiqj1mQGIqsM9cvJ1ZhF~MR4R8AU7FyNnlpBPNvx3coAntsE6s4gnycZmnognEC8XjOLD6h75FNw8euZHb8iM~Dhw2Jp6QNUBHBcTM4NlUvKqAV6seSh7t9jAgSw1695dWX8C8Hd4MSQYCqCbky3JHMUHVD8pAPyVrQvuYOY71YAZFl0ZDOzuAo1Fpbw~ddmh2TY1MqHRoKYHRE1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction statistics using sequence dependent structural properties (STB, stability; BDC, bendability using DNase I sensitivity; BNC, bendability using NPP model; CBC, curvature). (A) The performance of the classifier for primary promoters. Markers in Square and Diamond shape represent the first and second cycle of prediction, respectively. (B) Sensitivity values obtained for internal, antisense and Non-coding promoters, top and bottom row correspond to the first and second cycle of prediction respectively. Error bars represent the SD obtained from the 5-fold analysis." path-from-xml="dsw045f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462988/dsw045f3.tif" path-from-xml="dsw045f3.tif" target="_blank">View large</a><a section="61462988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f3.png?Expires=2147483647&Signature=2uHzR6SluBQuZl~NsI8aVgNI6ephv-r1xkgrbiQrBTWwx6s7gB-nD9Ga7nfH1oLtzLznwu-8FgIN8XxOXwHlpTBwWfaiksDUtJwvDfdkVYhZZ10B3WJRMstr6~aubEMxogNJjS4BUb4RAZ6NK38lkuAJ6SAHGWUeh9Q15fPJtn~fdsLe11LSr7k9YgZdlQU4xEIusWPPmQqYnwFypRhnfm715mwuTEJ6L6nk2VwKuRiqs4y8X5HG~eDQ4GgOUoDP-hUBMNXIJXmC7d1VPC4GUMqJcuiduydVF5RbMBghwZeaE9GDiTkDyx0PAyJb4CNF5P5rSdGAw9ouLmBRJPRc5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462988&ar=2449750&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw045f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction statistics using sequence dependent structural properties (STB, stability; BDC, bendability using DNase I sensitivity; BNC, bendability using NPP model; CBC, curvature). <strong>(A)</strong> The performance of the classifier for primary promoters. Markers in Square and Diamond shape represent the first and second cycle of prediction, respectively. <strong>(B)</strong> Sensitivity values obtained for internal, antisense and Non-coding promoters, top and bottom row correspond to the first and second cycle of prediction respectively. Error bars represent the SD obtained from the 5-fold analysis.</p></div></div></div><p>Stability was found to be the most characteristic structural feature in all organisms and achieved higher precision values in the second cycle of prediction compared with cycle one (<a reveal-id="dsw045-T2" data-open="dsw045-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Similarly, bendability (both models DNase I sensitivity and NPP) and curvature features were found to give high precision values in cycle two as compared with cycle one. Both prediction cycles were found to give an almost similar balanced-accuracy (BA), which is the mean of TP rate (sensitivity) and TN rate (specificity). Overall cycle one was found to be a better predictor, with better <em>F</em>-score (harmonic mean of sensitivity and precision) values across all the model organisms for all structural features. <div content-id="dsw045-T2" class="table-modal"><div class="table-wrap"><div id="dsw045-T2" data-id="dsw045-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p> Assessment of DNA structural features for their ability to differentiate between promoter and non-promoter sequences in six different model organisms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Organism </th><th rowspan="2">Number of sequences </th><th colspan="3">First cycle<hr /> </th><th colspan="3">Second cycle<hr /> </th><th rowspan="2">Property </th></tr><tr><th>Prec. </th><th>BA </th><th>F-score </th><th>Prec. </th><th>BA </th><th>F-score </th></tr></thead><tbody><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">77 </td><td char=".">75 </td><td char=".">75 </td><td char=".">80 </td><td char=".">75 </td><td char=".">72 </td><td rowspan="6"><strong>STB</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">72 </td><td char=".">71 </td><td char=".">70 </td><td char=".">76 </td><td char=".">71 </td><td char=".">68 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">69 </td><td char=".">69 </td><td char=".">69 </td><td char=".">74 </td><td char=".">69 </td><td char=".">65 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">74 </td><td char=".">75 </td><td char=".">76 </td><td char=".">80 </td><td char=".">77 </td><td char=".">76 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">75 </td><td char=".">76 </td><td char=".">77 </td><td char=".">79 </td><td char=".">77 </td><td char=".">76 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">80 </td><td char=".">80 </td><td char=".">80 </td><td char=".">84 </td><td char=".">81 </td><td char=".">80 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">54 </td><td char=".">56 </td><td char=".">63 </td><td char=".">56 </td><td char=".">58 </td><td char=".">63 </td><td rowspan="6"><strong>BDC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">52 </td><td char=".">54 </td><td char=".">62 </td><td char=".">53 </td><td char=".">53 </td><td char=".">58 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">52 </td><td char=".">53 </td><td char=".">61 </td><td char=".">53 </td><td char=".">53 </td><td char=".">57 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">54 </td><td char=".">56 </td><td char=".">63 </td><td char=".">55 </td><td char=".">57 </td><td char=".">61 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">55 </td><td char=".">57 </td><td char=".">65 </td><td char=".">56 </td><td char=".">58 </td><td char=".">62 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">56 </td><td char=".">59 </td><td char=".">66 </td><td char=".">59 </td><td char=".">61 </td><td char=".">65 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">54 </td><td char=".">54 </td><td char=".">55 </td><td char=".">58 </td><td char=".">56 </td><td char=".">51 </td><td rowspan="6"><strong>BNC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">55 </td><td char=".">57 </td><td char=".">64 </td><td char=".">57 </td><td char=".">58 </td><td char=".">61 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">52 </td><td char=".">53 </td><td char=".">62 </td><td char=".">53 </td><td char=".">54 </td><td char=".">59 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">52 </td><td char=".">54 </td><td char=".">63 </td><td char=".">52 </td><td char=".">53 </td><td char=".">59 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">52 </td><td char=".">53 </td><td char=".">62 </td><td char=".">52 </td><td char=".">53 </td><td char=".">59 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">51 </td><td char=".">52 </td><td char=".">63 </td><td char=".">51 </td><td char=".">51 </td><td char=".">59 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">53 </td><td char=".">55 </td><td char=".">62 </td><td char=".">54 </td><td char=".">55 </td><td char=".">59 </td><td rowspan="6"><strong>CBC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">53 </td><td char=".">54 </td><td char=".">60 </td><td char=".">55 </td><td char=".">55 </td><td char=".">58 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">53 </td><td char=".">54 </td><td char=".">61 </td><td char=".">53 </td><td char=".">54 </td><td char=".">58 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">57 </td><td char=".">58 </td><td char=".">63 </td><td char=".">58 </td><td char=".">59 </td><td char=".">60 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">56 </td><td char=".">58 </td><td char=".">64 </td><td char=".">59 </td><td char=".">60 </td><td char=".">63 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">58 </td><td char=".">60 </td><td char=".">66 </td><td char=".">59 </td><td char=".">60 </td><td char=".">64 </td></tr></tbody></table></div><div content-id="dsw045-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Organism </th><th rowspan="2">Number of sequences </th><th colspan="3">First cycle<hr /> </th><th colspan="3">Second cycle<hr /> </th><th rowspan="2">Property </th></tr><tr><th>Prec. </th><th>BA </th><th>F-score </th><th>Prec. </th><th>BA </th><th>F-score </th></tr></thead><tbody><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">77 </td><td char=".">75 </td><td char=".">75 </td><td char=".">80 </td><td char=".">75 </td><td char=".">72 </td><td rowspan="6"><strong>STB</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">72 </td><td char=".">71 </td><td char=".">70 </td><td char=".">76 </td><td char=".">71 </td><td char=".">68 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">69 </td><td char=".">69 </td><td char=".">69 </td><td char=".">74 </td><td char=".">69 </td><td char=".">65 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">74 </td><td char=".">75 </td><td char=".">76 </td><td char=".">80 </td><td char=".">77 </td><td char=".">76 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">75 </td><td char=".">76 </td><td char=".">77 </td><td char=".">79 </td><td char=".">77 </td><td char=".">76 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">80 </td><td char=".">80 </td><td char=".">80 </td><td char=".">84 </td><td char=".">81 </td><td char=".">80 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">54 </td><td char=".">56 </td><td char=".">63 </td><td char=".">56 </td><td char=".">58 </td><td char=".">63 </td><td rowspan="6"><strong>BDC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">52 </td><td char=".">54 </td><td char=".">62 </td><td char=".">53 </td><td char=".">53 </td><td char=".">58 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">52 </td><td char=".">53 </td><td char=".">61 </td><td char=".">53 </td><td char=".">53 </td><td char=".">57 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">54 </td><td char=".">56 </td><td char=".">63 </td><td char=".">55 </td><td char=".">57 </td><td char=".">61 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">55 </td><td char=".">57 </td><td char=".">65 </td><td char=".">56 </td><td char=".">58 </td><td char=".">62 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">56 </td><td char=".">59 </td><td char=".">66 </td><td char=".">59 </td><td char=".">61 </td><td char=".">65 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">54 </td><td char=".">54 </td><td char=".">55 </td><td char=".">58 </td><td char=".">56 </td><td char=".">51 </td><td rowspan="6"><strong>BNC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">55 </td><td char=".">57 </td><td char=".">64 </td><td char=".">57 </td><td char=".">58 </td><td char=".">61 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">52 </td><td char=".">53 </td><td char=".">62 </td><td char=".">53 </td><td char=".">54 </td><td char=".">59 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">52 </td><td char=".">54 </td><td char=".">63 </td><td char=".">52 </td><td char=".">53 </td><td char=".">59 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">52 </td><td char=".">53 </td><td char=".">62 </td><td char=".">52 </td><td char=".">53 </td><td char=".">59 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">51 </td><td char=".">52 </td><td char=".">63 </td><td char=".">51 </td><td char=".">51 </td><td char=".">59 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">53 </td><td char=".">55 </td><td char=".">62 </td><td char=".">54 </td><td char=".">55 </td><td char=".">59 </td><td rowspan="6"><strong>CBC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">53 </td><td char=".">54 </td><td char=".">60 </td><td char=".">55 </td><td char=".">55 </td><td char=".">58 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">53 </td><td char=".">54 </td><td char=".">61 </td><td char=".">53 </td><td char=".">54 </td><td char=".">58 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">57 </td><td char=".">58 </td><td char=".">63 </td><td char=".">58 </td><td char=".">59 </td><td char=".">60 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">56 </td><td char=".">58 </td><td char=".">64 </td><td char=".">59 </td><td char=".">60 </td><td char=".">63 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">58 </td><td char=".">60 </td><td char=".">66 </td><td char=".">59 </td><td char=".">60 </td><td char=".">64 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Out of the total test dataset (promoters associated with the primary category of TSS), 1001 nucleotide long sequences with 30–65% GC content were considered. The sequences with promoter predictions falling in the 300 nucleotide region spanning −200 to +100 with respect to TSS at 0 were considered as TP while sequences with predictions falling in the coding region +200 to + 500 with respect to TSS were labelled as FP. Evaluation parameters precision (Prec.), BA and <em>F</em>-score were calculated using formulas explained in the ‘Material and methods’ section.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw045-T2" data-id="dsw045-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p> Assessment of DNA structural features for their ability to differentiate between promoter and non-promoter sequences in six different model organisms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Organism </th><th rowspan="2">Number of sequences </th><th colspan="3">First cycle<hr /> </th><th colspan="3">Second cycle<hr /> </th><th rowspan="2">Property </th></tr><tr><th>Prec. </th><th>BA </th><th>F-score </th><th>Prec. </th><th>BA </th><th>F-score </th></tr></thead><tbody><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">77 </td><td char=".">75 </td><td char=".">75 </td><td char=".">80 </td><td char=".">75 </td><td char=".">72 </td><td rowspan="6"><strong>STB</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">72 </td><td char=".">71 </td><td char=".">70 </td><td char=".">76 </td><td char=".">71 </td><td char=".">68 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">69 </td><td char=".">69 </td><td char=".">69 </td><td char=".">74 </td><td char=".">69 </td><td char=".">65 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">74 </td><td char=".">75 </td><td char=".">76 </td><td char=".">80 </td><td char=".">77 </td><td char=".">76 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">75 </td><td char=".">76 </td><td char=".">77 </td><td char=".">79 </td><td char=".">77 </td><td char=".">76 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">80 </td><td char=".">80 </td><td char=".">80 </td><td char=".">84 </td><td char=".">81 </td><td char=".">80 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">54 </td><td char=".">56 </td><td char=".">63 </td><td char=".">56 </td><td char=".">58 </td><td char=".">63 </td><td rowspan="6"><strong>BDC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">52 </td><td char=".">54 </td><td char=".">62 </td><td char=".">53 </td><td char=".">53 </td><td char=".">58 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">52 </td><td char=".">53 </td><td char=".">61 </td><td char=".">53 </td><td char=".">53 </td><td char=".">57 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">54 </td><td char=".">56 </td><td char=".">63 </td><td char=".">55 </td><td char=".">57 </td><td char=".">61 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">55 </td><td char=".">57 </td><td char=".">65 </td><td char=".">56 </td><td char=".">58 </td><td char=".">62 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">56 </td><td char=".">59 </td><td char=".">66 </td><td char=".">59 </td><td char=".">61 </td><td char=".">65 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">54 </td><td char=".">54 </td><td char=".">55 </td><td char=".">58 </td><td char=".">56 </td><td char=".">51 </td><td rowspan="6"><strong>BNC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">55 </td><td char=".">57 </td><td char=".">64 </td><td char=".">57 </td><td char=".">58 </td><td char=".">61 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">52 </td><td char=".">53 </td><td char=".">62 </td><td char=".">53 </td><td char=".">54 </td><td char=".">59 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">52 </td><td char=".">54 </td><td char=".">63 </td><td char=".">52 </td><td char=".">53 </td><td char=".">59 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">52 </td><td char=".">53 </td><td char=".">62 </td><td char=".">52 </td><td char=".">53 </td><td char=".">59 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">51 </td><td char=".">52 </td><td char=".">63 </td><td char=".">51 </td><td char=".">51 </td><td char=".">59 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">53 </td><td char=".">55 </td><td char=".">62 </td><td char=".">54 </td><td char=".">55 </td><td char=".">59 </td><td rowspan="6"><strong>CBC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">53 </td><td char=".">54 </td><td char=".">60 </td><td char=".">55 </td><td char=".">55 </td><td char=".">58 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">53 </td><td char=".">54 </td><td char=".">61 </td><td char=".">53 </td><td char=".">54 </td><td char=".">58 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">57 </td><td char=".">58 </td><td char=".">63 </td><td char=".">58 </td><td char=".">59 </td><td char=".">60 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">56 </td><td char=".">58 </td><td char=".">64 </td><td char=".">59 </td><td char=".">60 </td><td char=".">63 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">58 </td><td char=".">60 </td><td char=".">66 </td><td char=".">59 </td><td char=".">60 </td><td char=".">64 </td></tr></tbody></table></div><div content-id="dsw045-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Organism </th><th rowspan="2">Number of sequences </th><th colspan="3">First cycle<hr /> </th><th colspan="3">Second cycle<hr /> </th><th rowspan="2">Property </th></tr><tr><th>Prec. </th><th>BA </th><th>F-score </th><th>Prec. </th><th>BA </th><th>F-score </th></tr></thead><tbody><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">77 </td><td char=".">75 </td><td char=".">75 </td><td char=".">80 </td><td char=".">75 </td><td char=".">72 </td><td rowspan="6"><strong>STB</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">72 </td><td char=".">71 </td><td char=".">70 </td><td char=".">76 </td><td char=".">71 </td><td char=".">68 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">69 </td><td char=".">69 </td><td char=".">69 </td><td char=".">74 </td><td char=".">69 </td><td char=".">65 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">74 </td><td char=".">75 </td><td char=".">76 </td><td char=".">80 </td><td char=".">77 </td><td char=".">76 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">75 </td><td char=".">76 </td><td char=".">77 </td><td char=".">79 </td><td char=".">77 </td><td char=".">76 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">80 </td><td char=".">80 </td><td char=".">80 </td><td char=".">84 </td><td char=".">81 </td><td char=".">80 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">54 </td><td char=".">56 </td><td char=".">63 </td><td char=".">56 </td><td char=".">58 </td><td char=".">63 </td><td rowspan="6"><strong>BDC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">52 </td><td char=".">54 </td><td char=".">62 </td><td char=".">53 </td><td char=".">53 </td><td char=".">58 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">52 </td><td char=".">53 </td><td char=".">61 </td><td char=".">53 </td><td char=".">53 </td><td char=".">57 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">54 </td><td char=".">56 </td><td char=".">63 </td><td char=".">55 </td><td char=".">57 </td><td char=".">61 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">55 </td><td char=".">57 </td><td char=".">65 </td><td char=".">56 </td><td char=".">58 </td><td char=".">62 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">56 </td><td char=".">59 </td><td char=".">66 </td><td char=".">59 </td><td char=".">61 </td><td char=".">65 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">54 </td><td char=".">54 </td><td char=".">55 </td><td char=".">58 </td><td char=".">56 </td><td char=".">51 </td><td rowspan="6"><strong>BNC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">55 </td><td char=".">57 </td><td char=".">64 </td><td char=".">57 </td><td char=".">58 </td><td char=".">61 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">52 </td><td char=".">53 </td><td char=".">62 </td><td char=".">53 </td><td char=".">54 </td><td char=".">59 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">52 </td><td char=".">54 </td><td char=".">63 </td><td char=".">52 </td><td char=".">53 </td><td char=".">59 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">52 </td><td char=".">53 </td><td char=".">62 </td><td char=".">52 </td><td char=".">53 </td><td char=".">59 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">51 </td><td char=".">52 </td><td char=".">63 </td><td char=".">51 </td><td char=".">51 </td><td char=".">59 </td></tr><tr><td><strong><em>H. pylori</em></strong> </td><td char=".">706 </td><td char=".">53 </td><td char=".">55 </td><td char=".">62 </td><td char=".">54 </td><td char=".">55 </td><td char=".">59 </td><td rowspan="6"><strong>CBC</strong> </td></tr><tr><td><strong><em>Anabaena</em></strong> </td><td char=".">2513 </td><td char=".">53 </td><td char=".">54 </td><td char=".">60 </td><td char=".">55 </td><td char=".">55 </td><td char=".">58 </td></tr><tr><td><strong><em>Synechocystis</em></strong> </td><td char=".">1639 </td><td char=".">53 </td><td char=".">54 </td><td char=".">61 </td><td char=".">53 </td><td char=".">54 </td><td char=".">58 </td></tr><tr><td><strong><em>E. coli</em></strong> </td><td char=".">1331 </td><td char=".">57 </td><td char=".">58 </td><td char=".">63 </td><td char=".">58 </td><td char=".">59 </td><td char=".">60 </td></tr><tr><td><strong><em>Salmonella</em></strong> </td><td char=".">981 </td><td char=".">56 </td><td char=".">58 </td><td char=".">64 </td><td char=".">59 </td><td char=".">60 </td><td char=".">63 </td></tr><tr><td><strong><em>Klebsiella</em></strong> </td><td char=".">1324 </td><td char=".">58 </td><td char=".">60 </td><td char=".">66 </td><td char=".">59 </td><td char=".">60 </td><td char=".">64 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Out of the total test dataset (promoters associated with the primary category of TSS), 1001 nucleotide long sequences with 30–65% GC content were considered. The sequences with promoter predictions falling in the 300 nucleotide region spanning −200 to +100 with respect to TSS at 0 were considered as TP while sequences with predictions falling in the coding region +200 to + 500 with respect to TSS were labelled as FP. Evaluation parameters precision (Prec.), BA and <em>F</em>-score were calculated using formulas explained in the ‘Material and methods’ section.</p></div></div></div></div></p><p>We categorized the predicted promoters based on the function performed by the associated gene, according to the Cluster of Orthologous Groups information obtained from NCBI (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Promoters associated with genes involved in information storage and processing function (transcription, translation and replication), which are constitutively expressed in the cell are annotated well across the six different organisms. However, sensitivity values for an individual structural feature vary among the organisms, e.g. promoters associated with genes which have a role in replication and repair exhibit sensitivity values (using stability) ranging from a low of 38% for <em>Synechocystis</em> to a high of 72% for <em>Klebsiella</em>. Promoters of genes involved in cellular processing and cell signaling are relatively less well annotated. Genes involved in metabolism function show variation in the sensitivity values for different gene families. General function genes (which are based on prediction only) also get annotated reliably in all six prokaryotic organisms. Genes which are not characterized or with unknown function are found to be relatively less annotated in all organisms. Hence, prediction results suggest that structural features in the promoter regions are related to the function associated with the gene.</p><p>Internal, antisense and non-coding RNA promoters were also predicted using the cut-off values derived from primary promoter sequences. Higher sensitivity values were obtained in the first cycle of prediction as compared with the second cycle. Bendability (both by DNase I sensitivity and NPP model) and curvature were found to be more sensitive for these categories of promoters, as compared with stability. The reduction in the sensitivity values for stability feature for internal and antisense promoters was due to sharp and narrow low stability peak in the AFE profile (<a reveal-id="dsw045-F2" data-open="dsw045-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The narrower peaks can be attributed to these categories of promoters being present within the coding region while primary promoters are mostly present in the relatively AT rich intergenic region upstream of TSS.</p>                            <h3 scrollto-destination=61462992 id="61462992" class="section-title" >3.3 Promoter annotation using combination of structural features</h3>
<div class="block-child-p">Although stability was found to be the most distinctive feature, differentiating the promoter from the flanking region for a given genomic sequence, it was able to achieve maximum 81% sensitivity for primary category of promoters in <em>Klebsiella</em>, in cycle one prediction (<a reveal-id="dsw045-F3" data-open="dsw045-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). In order to assess the number of promoter sequences that do not get identified by any structural feature, we applied a combinatorial approach. It was found that ≈95% primary category of promoter sequences get identified by at least one structural feature. Similar sensitivity values are obtained from a combinatorial approach using DNase I sensitivity and NPP models for bendability (as seen in <a reveal-id="dsw045-F4" data-open="dsw045-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Likewise internal and antisense categories of promoters achieve ≈85 and ≈89% sensitivity, respectively. Further analysis of promoter prediction in the different categories of transcripts revealed several interesting facts. For example, ≈50% of the primary category of promoter sequences are characterized by all three structural features, viz. stability, bendability and curvature, while remaining promoter sequences lack either one or two structural features (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&amp;Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Interestingly, majority of internal and antisense categories of promoters are characterized by bendability and curvature while promoters associated with non-coding RNA genes are similar to primary category of promoters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&amp;Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). This suggests that the promoter sequences which occur in the coding region, such as internal and antisense promoters, are distinguishable by different structural features, when compared with the primary and non-coding RNA promoter sequences, which are present in the intergenic region. <div data-id="dsw045-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f4.png?Expires=2147483647&amp;Signature=TeAUxt~9gkn1gK~HhFgI6hnIAeQkQyaSTKVY-Imq0vvUNovwl4T0~saKPcB~glqO1A1IXKI6PpnV2Ntdn4fDX0xLfV8QZlH4ED6uXk5PQtYBxkfif2QxucVL9n4Utaom9Zxf~rb3u~2gL57Q-fon166NUOLDCDwT~EZ9gMo4JEZRxEPz6eSpUAgRtWtKIrXRdIFNx5eiBjk0HNT-W23Crte2a0IrcPCVSfjiC2yaUa--fYCbZn4WEGygpN~RAKVQixNPaT9bVb0mdAQ3xxjjOI-gVv8hFW6vxH3lj8-Wcx578H-dRBCE96QVG50pwAQ0A8I9j~ZRKt2WGC11IisSFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bar plots showing sensitivity values obtained from cycle I predictions, using one or more structural features for different categories of TSSs. Top row correspond to stability, bendability using DNase I sensitivity and curvature while bottom row corresponds to stability, bendability using NPP and curvature." path-from-xml="dsw045f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462993/dsw045f4.tif" path-from-xml="dsw045f4.tif" target="_blank">View large</a><a section="61462993" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f4.png?Expires=2147483647&amp;Signature=P1PoYXHtikXVF0EjP~CJ8XcGz1j7FtS5bWB5epCQZw0mQIWzirfDuzDr5lYO8sLn-JeGOsw3iR85aSU4cFvfQCys0rRHXH~KWddrrUj-Y4dtMLK~YgTpARcBhMl-lxmSk58K7jzm4b7fNKWL1rcbGGLD2WbZG7SOR3GbHO3lvuzuZ53VykSCnm-xj0Ays3yT557jFe5fdP1EHe~4W9eG1dMYfwFAItCybIBdqxcD1lWOR3AJ1swXBUwBkALkILikxJ6ArH2eBFwoc5eIlebUVUfCMJ2tFjFiPs87IZE8nV21uRTA3pIYErVeTAzOihpj7dUKHgQMQYTnQFoJ5H~VCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462993&amp;ar=2449750&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw045f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bar plots showing sensitivity values obtained from cycle I predictions, using one or more structural features for different categories of TSSs. Top row correspond to stability, bendability using DNase I sensitivity and curvature while bottom row corresponds to stability, bendability using NPP and curvature.</p></div></div><div content-id="dsw045-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f4.png?Expires=2147483647&amp;Signature=TeAUxt~9gkn1gK~HhFgI6hnIAeQkQyaSTKVY-Imq0vvUNovwl4T0~saKPcB~glqO1A1IXKI6PpnV2Ntdn4fDX0xLfV8QZlH4ED6uXk5PQtYBxkfif2QxucVL9n4Utaom9Zxf~rb3u~2gL57Q-fon166NUOLDCDwT~EZ9gMo4JEZRxEPz6eSpUAgRtWtKIrXRdIFNx5eiBjk0HNT-W23Crte2a0IrcPCVSfjiC2yaUa--fYCbZn4WEGygpN~RAKVQixNPaT9bVb0mdAQ3xxjjOI-gVv8hFW6vxH3lj8-Wcx578H-dRBCE96QVG50pwAQ0A8I9j~ZRKt2WGC11IisSFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bar plots showing sensitivity values obtained from cycle I predictions, using one or more structural features for different categories of TSSs. Top row correspond to stability, bendability using DNase I sensitivity and curvature while bottom row corresponds to stability, bendability using NPP and curvature." path-from-xml="dsw045f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462993/dsw045f4.tif" path-from-xml="dsw045f4.tif" target="_blank">View large</a><a section="61462993" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f4.png?Expires=2147483647&amp;Signature=P1PoYXHtikXVF0EjP~CJ8XcGz1j7FtS5bWB5epCQZw0mQIWzirfDuzDr5lYO8sLn-JeGOsw3iR85aSU4cFvfQCys0rRHXH~KWddrrUj-Y4dtMLK~YgTpARcBhMl-lxmSk58K7jzm4b7fNKWL1rcbGGLD2WbZG7SOR3GbHO3lvuzuZ53VykSCnm-xj0Ays3yT557jFe5fdP1EHe~4W9eG1dMYfwFAItCybIBdqxcD1lWOR3AJ1swXBUwBkALkILikxJ6ArH2eBFwoc5eIlebUVUfCMJ2tFjFiPs87IZE8nV21uRTA3pIYErVeTAzOihpj7dUKHgQMQYTnQFoJ5H~VCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462993&amp;ar=2449750&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw045f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bar plots showing sensitivity values obtained from cycle I predictions, using one or more structural features for different categories of TSSs. Top row correspond to stability, bendability using DNase I sensitivity and curvature while bottom row corresponds to stability, bendability using NPP and curvature.</p></div></div></div>                            <h3 scrollto-destination=61462994 id="61462994" class="section-title" >3.4 DNA structural features of the promoter region are correlated with the gene expression</h3>
<div class="block-child-p">Sequence dependent structural properties of double stranded DNA are associated with the physiology of the promoter. For example DNA duplex has to be less stable to facilitate easier melting in the vicinity of the TSS during transcription initiation. Hence, in order to study the promoter architecture in terms of structural features, we characterized these features in two classes of promoters, viz. those associated with genes exhibiting high and low gene expression. Gene expression data was obtained from GEO, NCBI expression microarray profiling. The structural properties (stability, bendability and curvature) in the promoter regions associated with high gene expression are found to be more pronounced as compared with low gene expression. A violin plot drawn from the values of DNA structural features underscores the difference in the distribution of probability densities between the promoters associated with high and low expression categories of genes (<a reveal-id="dsw045-F5" data-open="dsw045-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The promoters of highly expressed genes were less stable, less bendable and more curved as compared with promoters of lowly expressed genes. One-sided two sample KS test was performed to check if the difference in feature value between the two classes is significant enough for them to be compared (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&amp;Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Interestingly curvature was found to be the only consistent property demarcating promoters from high and low gene expression classes, across all organisms, while other properties show one or more exceptions in case of <em>Synechocystis</em>, <em>Salmonella</em> and <em>Klebsiella</em>. <div data-id="dsw045-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f5.png?Expires=2147483647&amp;Signature=UlJqD805apR9dTQtqklL1raH9IT6CVGY4Z3gW2EkKwO9O81biKNwyhsAoxXQ1Dm6A19J9n3CLE1NMAg28nqa1hazpT2j3qN8HrlnfL-K~eIAdTUwU4vRJv2oGH1JR87tUh6AYAz35vICUlmN6RaC1UZyFpTbdfYIOAe-2nW~qQRe4HlrPU-gWhth85WzfNBrxfCLbLqRIteH66ld-5~7v4qesNmMtdckvqMMd2GOD0S7qixtVyBFIRxrC5nv25Wrr4xTS5n6jwMXaBdGf1MovbstHUP100kzn9MgJDkiuUwKcFDHnXlE1xjnwGA9FLuCQJyGqAmfIQuoYwiV5HnAWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Violin plot of DNA structural property values in the promoter regions (−100 to 0 nucleotide with respect to TSS at 0) associated with high and low gene expression. The x-axis shows the probability density while y-axis represents the DNA structural features value. Plots with shaded background indicate the cases which failed to reject the null hypothesis using two sample KS test at the level of significance of P = 0.01." path-from-xml="dsw045f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462995/dsw045f5.tif" path-from-xml="dsw045f5.tif" target="_blank">View large</a><a section="61462995" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f5.png?Expires=2147483647&amp;Signature=BwG6Rf-YjB10cUG9-dHdngNSV353bQTzMxCWG6M13iapd5Ai85ADX9z29-RcB4jWqZ0HLGfpaFh3-EyCKx5KIewV8nHZMhY~9mLwjWMuC5p4D2q8up5LgCvtQWAkeHK0I3M6Rdo8fPU~k78s7HYgWmwt-AO8cdQMxTjkOsnpjFQnxvi~nvfkf6dYsidlMDmilYiEShZ1EkzadZTcWXQt6tII-M0nQqauAya3SPNRV-7RcLMKC~tyV1FOOqAdsfeYPjYlr~fPbEF1m9cyISuAdXfQWI3dxXaUk6Wgp78Q9Ldbty4gGo8uh249MTDo5IVxvQtZmM-PGN6G70yn4ksFkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462995&amp;ar=2449750&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw045f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Violin plot of DNA structural property values in the promoter regions (−100 to 0 nucleotide with respect to TSS at 0) associated with high and low gene expression. The x-axis shows the probability density while y-axis represents the DNA structural features value. Plots with shaded background indicate the cases which failed to reject the null hypothesis using two sample KS test at the level of significance of <em>P</em> = 0.01.</p></div></div><div content-id="dsw045-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f5.png?Expires=2147483647&amp;Signature=UlJqD805apR9dTQtqklL1raH9IT6CVGY4Z3gW2EkKwO9O81biKNwyhsAoxXQ1Dm6A19J9n3CLE1NMAg28nqa1hazpT2j3qN8HrlnfL-K~eIAdTUwU4vRJv2oGH1JR87tUh6AYAz35vICUlmN6RaC1UZyFpTbdfYIOAe-2nW~qQRe4HlrPU-gWhth85WzfNBrxfCLbLqRIteH66ld-5~7v4qesNmMtdckvqMMd2GOD0S7qixtVyBFIRxrC5nv25Wrr4xTS5n6jwMXaBdGf1MovbstHUP100kzn9MgJDkiuUwKcFDHnXlE1xjnwGA9FLuCQJyGqAmfIQuoYwiV5HnAWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Violin plot of DNA structural property values in the promoter regions (−100 to 0 nucleotide with respect to TSS at 0) associated with high and low gene expression. The x-axis shows the probability density while y-axis represents the DNA structural features value. Plots with shaded background indicate the cases which failed to reject the null hypothesis using two sample KS test at the level of significance of P = 0.01." path-from-xml="dsw045f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462995/dsw045f5.tif" path-from-xml="dsw045f5.tif" target="_blank">View large</a><a section="61462995" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f5.png?Expires=2147483647&amp;Signature=BwG6Rf-YjB10cUG9-dHdngNSV353bQTzMxCWG6M13iapd5Ai85ADX9z29-RcB4jWqZ0HLGfpaFh3-EyCKx5KIewV8nHZMhY~9mLwjWMuC5p4D2q8up5LgCvtQWAkeHK0I3M6Rdo8fPU~k78s7HYgWmwt-AO8cdQMxTjkOsnpjFQnxvi~nvfkf6dYsidlMDmilYiEShZ1EkzadZTcWXQt6tII-M0nQqauAya3SPNRV-7RcLMKC~tyV1FOOqAdsfeYPjYlr~fPbEF1m9cyISuAdXfQWI3dxXaUk6Wgp78Q9Ldbty4gGo8uh249MTDo5IVxvQtZmM-PGN6G70yn4ksFkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462995&amp;ar=2449750&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw045f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Violin plot of DNA structural property values in the promoter regions (−100 to 0 nucleotide with respect to TSS at 0) associated with high and low gene expression. The x-axis shows the probability density while y-axis represents the DNA structural features value. Plots with shaded background indicate the cases which failed to reject the null hypothesis using two sample KS test at the level of significance of <em>P</em> = 0.01.</p></div></div></div><p>Trinucleotide preference analysis of the whole promoter region (UP elements and core promoter region) was performed in order to understand the underlying difference in terms of oligonucleotide composition. It showed that overall high expression genes favor rigid and minor groove facing histone preferring trimers, such as AAA/TTT and AAT/ATT while flexible or major groove facing histone preferring trinucleotides like GCC/GGC are less preferred (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Preference for AAA/TTT reduces gradually as genomic GC% increases.</p><div class="block-child-p">Promoter prediction using DNA structural features shows that sensitivity for promoters associated with high gene expression is higher as compared with the low gene expression (<a reveal-id="dsw045-F6" data-open="dsw045-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Interestingly it was found that promoters associated with higher gene expression are better characterized (higher sensitivity) by a combination of all structural features (stability, bendability and curvature) as compared with those associated with low gene expression. Correspondingly, promoters for low gene expression category are less likely to be predicted as compared with the high gene expression category. Moreover, this trend is present across organisms with varying genomic GC content. <div data-id="dsw045-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f6.png?Expires=2147483647&Signature=0EFNklyMzogqaP~5W08UKwjl1XeMIi~OSPynvZz1xjLwREfx019~EV9YTJ90V5pwe-F3VXw6ACTl0kcv-WSmIbXIEjUetoCmx5PzeLGKdAlgOo-OiRyr1jLHMt14JNPaY7rV41ZgEgOxqwaY~1wGFiC7TIq0JDqOBS-P068g68ShX-UHUVdOTQ1lBcTTA28q7LJwEhlxMVMFMI1t5x~tClOHPGYPDp844iH~nnqssk~IIb7GXvYjLYEGB-2qyyxMKZE3k~Mruvokiw8X2rwr-7eZpSVZdMHLfPmn5SCOw21Z4Qz3miF5mRS7A-Z-v8c44EUXCA2ciuL7JwzZZWhE2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bar plot of sensitivity values obtained from cycle I predictions using all three structural features (predicted) or none (unpredicted) (stability, bendability using DNase I sensitivity model, bendability using NPP model and curvature). Left and right plots correspond to the DNase I sensitivity and NPP models of bendability." path-from-xml="dsw045f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462997/dsw045f6.tif" path-from-xml="dsw045f6.tif" target="_blank">View large</a><a section="61462997" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f6.png?Expires=2147483647&Signature=uyABtIBXP~UszJpBeHSfmySOtJcXWl4cN7Nl-oo8nB~TPuhiyjpy06sEMPV6rbtKg5yPpC3Kjd8UvjtZSIvLa6P2ENX0lPy3zjLo7P06lQR1lYBmcF306QReEqqdmpXrIWf8nNUWAoD-EPNGvbgAqMju668fByY7c0WgMUiw3Eo2sR~cQlkQ7qWu65vlsVODF9EVPiGX762CFoM6slC7AbNtG0E~K5L4r~y3z6fAaKI9TFRUw~O5JuZQgTcBlIaHwq3munkKoKrrDBjgitmwpH3Ap4YN4-RVfmzePUxjsYoWrY-790UW9mLWZKGYLo1zFIKNxqa~dXf4pcPdi9B3Nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462997&ar=2449750&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw045f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bar plot of sensitivity values obtained from cycle I predictions using all three structural features (predicted) or none (unpredicted) (stability, bendability using DNase I sensitivity model, bendability using NPP model and curvature). Left and right plots correspond to the DNase I sensitivity and NPP models of bendability.</p></div></div><div content-id="dsw045-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/m_dsw045f6.png?Expires=2147483647&Signature=0EFNklyMzogqaP~5W08UKwjl1XeMIi~OSPynvZz1xjLwREfx019~EV9YTJ90V5pwe-F3VXw6ACTl0kcv-WSmIbXIEjUetoCmx5PzeLGKdAlgOo-OiRyr1jLHMt14JNPaY7rV41ZgEgOxqwaY~1wGFiC7TIq0JDqOBS-P068g68ShX-UHUVdOTQ1lBcTTA28q7LJwEhlxMVMFMI1t5x~tClOHPGYPDp844iH~nnqssk~IIb7GXvYjLYEGB-2qyyxMKZE3k~Mruvokiw8X2rwr-7eZpSVZdMHLfPmn5SCOw21Z4Qz3miF5mRS7A-Z-v8c44EUXCA2ciuL7JwzZZWhE2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bar plot of sensitivity values obtained from cycle I predictions using all three structural features (predicted) or none (unpredicted) (stability, bendability using DNase I sensitivity model, bendability using NPP model and curvature). Left and right plots correspond to the DNase I sensitivity and NPP models of bendability." path-from-xml="dsw045f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462997/dsw045f6.tif" path-from-xml="dsw045f6.tif" target="_blank">View large</a><a section="61462997" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045f6.png?Expires=2147483647&Signature=uyABtIBXP~UszJpBeHSfmySOtJcXWl4cN7Nl-oo8nB~TPuhiyjpy06sEMPV6rbtKg5yPpC3Kjd8UvjtZSIvLa6P2ENX0lPy3zjLo7P06lQR1lYBmcF306QReEqqdmpXrIWf8nNUWAoD-EPNGvbgAqMju668fByY7c0WgMUiw3Eo2sR~cQlkQ7qWu65vlsVODF9EVPiGX762CFoM6slC7AbNtG0E~K5L4r~y3z6fAaKI9TFRUw~O5JuZQgTcBlIaHwq3munkKoKrrDBjgitmwpH3Ap4YN4-RVfmzePUxjsYoWrY-790UW9mLWZKGYLo1zFIKNxqa~dXf4pcPdi9B3Nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462997&ar=2449750&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw045f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bar plot of sensitivity values obtained from cycle I predictions using all three structural features (predicted) or none (unpredicted) (stability, bendability using DNase I sensitivity model, bendability using NPP model and curvature). Left and right plots correspond to the DNase I sensitivity and NPP models of bendability.</p></div></div></div>                            <h2 scrollto-destination=61462998 id="61462998" class="section-title" >4. Conclusion</h2>
<p>DNA sequence dependent structural properties like duplex stability, protein induced bendability and intrinsic curvature are well studied and physiologically relevant properties for the promoter region. Since there is very weak sequence (tetramer) preference and poor sequence motif conservation in the internal and antisense promoters as compared with primary promoters, it is very difficult to identify them using sequence based approaches. Characteristic structural features, on the other hand, are present in the promoter regions of all categories of promoters identified in prokaryotic transcriptomes. Significantly, these properties are not organism specific in nature. However, promoters associated with primary, internal, antisense and non-coding RNA category of transcripts show differences in the profiles of their structural properties. In particular, internal and antisense categories of promoters show distinctly different structural feature profiles as compared with the primary category of promoters. Since a single feature does not predict all promoters, transcriptome can be better annotated using a combinatorial approach. Annotation and analysis of promoters corresponding to the different transcripts present in the transcriptome can provide a better understanding of the complexity involved in gene regulation process. Finally, the promoter DNA structural features show good correlation with the level of expression of the associated gene. Hence by examining structural features associated with promoter regions, one can possibly estimate the level of gene expression.</p>                            <h2 scrollto-destination=61463000 id="61463000" class="backacknowledgements-title" >Acknowledgements</h2>
<p>M.B. is the recipient of J. C. Bose National Fellowship of DST, India.</p>                            <h3 scrollto-destination=61463002 id="61463002" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=61463004 id="61463004" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=61463006 id="61463006" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw045-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">MacQuarrie</div> <div class="given-names">K. L.</div></div><div class="name string-name"><div class="surname">Fong</div> <div class="given-names">A. P.</div></div><div class="name string-name"><div class="surname">Morse</div> <div class="given-names">R. H.</div></div><div class="name string-name"><div class="surname">Tapscott</div> <div class="given-names">S. J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genome-wide transcription factor binding: beyond direct target regulation</div>. <div class="source">Trends Genet</div> ., <div class="volume">27</div>, <div class="fpage">141</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20transcription%20factor%20binding%3A%20beyond%20direct%20target%20regulation&amp;author=K.%20L.%20MacQuarrie&amp;author=A.%20P.%20Fong&amp;author=R.%20H.%20Morse&amp;author=S.%20J.%20Tapscott&amp;publication_year=2011&amp;journal=Trends%20Genet&amp;volume=27&amp;pages=141-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2011.01.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2011.01.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2011.01.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21295369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Slattery</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Riley</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Cofactor binding evokes latent differences in DNA binding specificity between Hox proteins</div>. <div class="source">Cell</div> , <div class="volume">147</div>, <div class="fpage">1270</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cofactor%20binding%20evokes%20latent%20differences%20in%20DNA%20binding%20specificity%20between%20Hox%20proteins&amp;author=M.%20Slattery&amp;author=T.%20Riley&amp;author=P.%20Liu&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=1270-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.10.053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.10.053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.10.053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22153072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gordan</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Shen</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Dror</div> <div class="given-names">I.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Genomic regions flanking E-box binding sites influence DNA binding specificity of bHLH transcription factors through DNA shape</div>. <div class="source">Cell Rep</div> ., <div class="volume">3</div>, <div class="fpage">1093</div>–<div class="lpage">104</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20regions%20flanking%20E-box%20binding%20sites%20influence%20DNA%20binding%20specificity%20of%20bHLH%20transcription%20factors%20through%20DNA%20shape&amp;author=R.%20Gordan&amp;author=N.%20Shen&amp;author=I.%20Dror&amp;publication_year=2013&amp;journal=Cell%20Rep&amp;volume=3&amp;pages=1093-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2013.03.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2013.03.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2013.03.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23562153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Levo</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Segal</div> <div class="given-names">E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">In pursuit of design principles of regulatory sequences</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="fpage">453</div>–<div class="lpage">468</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20pursuit%20of%20design%20principles%20of%20regulatory%20sequences&amp;author=M.%20Levo&amp;author=E.%20Segal&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=453-468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3684"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24913666" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meysman</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Marchal</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Engelen</div> <div class="given-names">K.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">DNA structural properties in the classification of genomic transcription regulation elements</div>. <div class="source">Bioinform. Biol. Insights</div> , <div class="volume">6</div>, <div class="fpage">155</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20structural%20properties%20in%20the%20classification%20of%20genomic%20transcription%20regulation%20elements&amp;author=P.%20Meysman&amp;author=K.%20Marchal&amp;author=K.%20Engelen&amp;publication_year=2012&amp;journal=Bioinform.%20Biol.%20Insights&amp;volume=6&amp;pages=155-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4137/BBI" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4137%2FBBI" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4137%2FBBI"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22837642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maienschein-Cline</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Dinner</div> <div class="given-names">A. R.</div></div><div class="name string-name"><div class="surname">Hlavacek</div> <div class="given-names">W. S.</div></div><div class="name string-name"><div class="surname">Mu</div> <div class="given-names">F.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Improved predictions of transcription factor binding sites using physicochemical features of DNA</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e175</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20predictions%20of%20transcription%20factor%20binding%20sites%20using%20physicochemical%20features%20of%20DNA&amp;author=M.%20Maienschein-Cline&amp;author=A.%20R.%20Dinner&amp;author=W.%20S.%20Hlavacek&amp;author=F.%20Mu&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks771" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks771" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks771"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22923524" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Shen</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Quantitative modeling of transcription factor binding specificities using DNA shape</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">4654</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20modeling%20of%20transcription%20factor%20binding%20specificities%20using%20DNA%20shape&amp;author=T.%20Zhou&amp;author=N.%20Shen&amp;author=L.%20Yang&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=4654-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1422023112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1422023112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1422023112"> </span></div> </div></div></div></div></div><div content-id="dsw045-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Levo</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Zalckvar</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Sharon</div> <div class="given-names">E.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Unraveling determinants of transcription factor binding outside the core binding site</div>. <div class="source">Genome Res</div> ., <div class="fpage">185033</div>–<div class="lpage">114</div>.</div></div></div></div><div content-id="dsw045-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chiu</div> <div class="given-names">T. P.</div></div><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">GBshape: a genome browser database for DNA shape annotations</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D103</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GBshape%3A%20a%20genome%20browser%20database%20for%20DNA%20shape%20annotations&amp;author=T.%20P.%20Chiu&amp;author=L.%20Yang&amp;author=T.%20Zhou&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D103-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku977" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku977" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku977"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25326329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khandelwal</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">R. A.</div></div><div class="name string-name"><div class="surname">Jayaram</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Beveridge</div> <div class="given-names">D. L.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A statistical thermodynamic model for investigating the stability of DNA sequences from oligonucleotides to genomes</div>. <div class="source">Biophys J</div> ., <div class="volume">106</div>, <div class="fpage">2465</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20thermodynamic%20model%20for%20investigating%20the%20stability%20of%20DNA%20sequences%20from%20oligonucleotides%20to%20genomes&amp;author=G.%20Khandelwal&amp;author=R.%20A.%20Lee&amp;author=B.%20Jayaram&amp;author=D.%20L.%20Beveridge&amp;publication_year=2014&amp;journal=Biophys%20J&amp;volume=106&amp;pages=2465-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bpj.2014.04.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bpj.2014.04.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bpj.2014.04.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24896126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khandelwal</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Jayaram</div> <div class="given-names">B.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">DNA-water interactions distinguish messenger RNA genes from transfer RNA genes</div>. <div class="source">J. Am. Chem. Soc</div> ., <div class="volume">134</div>, <div class="fpage">8814</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-water%20interactions%20distinguish%20messenger%20RNA%20genes%20from%20transfer%20RNA%20genes&amp;author=G.%20Khandelwal&amp;author=B.%20Jayaram&amp;publication_year=2012&amp;journal=J.%20Am.%20Chem.%20Soc&amp;volume=134&amp;pages=8814-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/ja3020956" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fja3020956" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fja3020956"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22551381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khandelwal</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Jayaram</div> <div class="given-names">B.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">DNA-energetics-based analyses suggest additional genes in prokaryotes</div>. <div class="source">J. Biosci</div> ., <div class="volume">37</div>, <div class="fpage">433</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-energetics-based%20analyses%20suggest%20additional%20genes%20in%20prokaryotes&amp;author=G.%20Khandelwal&amp;author=J.%20Gupta&amp;author=B.%20Jayaram&amp;publication_year=2012&amp;journal=J.%20Biosci&amp;volume=37&amp;pages=433-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12038-012-9221-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12038-012-9221-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12038-012-9221-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22750981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Florquin</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Saeys</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Degroeve</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Rouze</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Large-scale structural analysis of the core promoter in mammalian and plant genomes</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">4255</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20structural%20analysis%20of%20the%20core%20promoter%20in%20mammalian%20and%20plant%20genomes&amp;author=K.%20Florquin&amp;author=Y.%20Saeys&amp;author=S.%20Degroeve&amp;author=P.%20Rouze&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=4255-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki737" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki737" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki737"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16049029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kanhere</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Structural properties of promoters: similarities and differences between prokaryotes and eukaryotes</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">3165</div>–<div class="lpage">3175</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20properties%20of%20promoters%3A%20similarities%20and%20differences%20between%20prokaryotes%20and%20eukaryotes&amp;author=A.%20Kanhere&amp;author=M.%20Bansal&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=3165-3175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki627" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15939933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meysman</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Collado-Vides</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Morett</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Viola</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Engelen</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Laukens</div> <div class="given-names">K.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Structural properties of prokaryotic promoter regions correlate with functional features</div>. <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e88717</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20properties%20of%20prokaryotic%20promoter%20regions%20correlate%20with%20functional%20features&amp;author=P.%20Meysman&amp;author=J.%20Collado-Vides&amp;author=E.%20Morett&amp;author=R.%20Viola&amp;author=K.%20Engelen&amp;author=K.%20Laukens&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e88717" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0088717" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0088717" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0088717"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24516674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Yella</div> <div class="given-names">V. R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Role of DNA sequence based structural features of promoters in transcription initiation and gene expression</div>. <div class="source">Curr. Opin. Struct. Biol</div> ., <div class="volume">25</div>, <div class="fpage">77</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20DNA%20sequence%20based%20structural%20features%20of%20promoters%20in%20transcription%20initiation%20and%20gene%20expression&amp;author=M.%20Bansal&amp;author=A.%20Kumar&amp;author=V.%20R.%20Yella&amp;publication_year=2014&amp;journal=Curr.%20Opin.%20Struct.%20Biol&amp;volume=25&amp;pages=77-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.sbi.2014.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.sbi.2014.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.sbi.2014.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24503515" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rangannan</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Relative stability of DNA as a generic criterion for promoter prediction: whole genome annotation of microbial genomes with varying nucleotide base composition</div>. <div class="source">Mol. Biosyst</div> ., <div class="volume">5</div>, <div class="fpage">1758</div>–<div class="lpage">1769</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relative%20stability%20of%20DNA%20as%20a%20generic%20criterion%20for%20promoter%20prediction%3A%20whole%20genome%20annotation%20of%20microbial%20genomes%20with%20varying%20nucleotide%20base%20composition&amp;author=V.%20Rangannan&amp;author=M.%20Bansal&amp;publication_year=2009&amp;journal=Mol.%20Biosyst&amp;volume=5&amp;pages=1758-1769" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/b906535k" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fb906535k" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fb906535k"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19593472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rangannan</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">High-quality annotation of promoter regions for 913 bacterial genomes</div>. <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">3043</div>–<div class="lpage">3050</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-quality%20annotation%20of%20promoter%20regions%20for%20913%20bacterial%20genomes&amp;author=V.%20Rangannan&amp;author=M.%20Bansal&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=3043-3050" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq577" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq577" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq577"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20956245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rangannan</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">PromBase: a web resource for various genomic features and predicted promoters in prokaryotic genomes</div>. <div class="source">BMC Res. Notes</div> , <div class="volume">4</div>, <div class="fpage">257</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PromBase%3A%20a%20web%20resource%20for%20various%20genomic%20features%20and%20predicted%20promoters%20in%20prokaryotic%20genomes&amp;author=V.%20Rangannan&amp;author=M.%20Bansal&amp;publication_year=2011&amp;journal=BMC%20Res.%20Notes&amp;volume=4&amp;pages=257" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-4-257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-4-257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-4-257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21781326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Characterization of structural and free energy properties of promoters associated with Primary and Operon TSS in <em>Helicobacter pylori</em> genome and their orthologs</div>. <div class="source">J. Biosci</div> ., <div class="volume">37</div>, <div class="fpage">423</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20structural%20and%20free%20energy%20properties%20of%20promoters%20associated%20with%20Primary%20and%20Operon%20TSS%20in%20Helicobacter%20pylori%20genome%20and%20their%20orthologs&amp;author=A.%20Kumar&amp;author=M.%20Bansal&amp;publication_year=2012&amp;journal=J.%20Biosci&amp;volume=37&amp;pages=423-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12038-012-9214-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12038-012-9214-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12038-012-9214-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22750980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Du</div> <div class="given-names">X.</div></div><div class="name string-name"><div class="surname">Wojtowicz</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Bowers</div> <div class="given-names">A. A.</div></div><div class="name string-name"><div class="surname">Levens</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Benham</div> <div class="given-names">C. J.</div></div><div class="name string-name"><div class="surname">Przytycka</div> <div class="given-names">T. M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The genome-wide distribution of non-B DNA motifs is shaped by operon structure and suggests the transcriptional importance of non-B DNA structures in <em>Escherichia coli</em></div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">5965</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome-wide%20distribution%20of%20non-B%20DNA%20motifs%20is%20shaped%20by%20operon%20structure%20and%20suggests%20the%20transcriptional%20importance%20of%20non-B%20DNA%20structures%20in%20Escherichia%20coli&amp;author=X.%20Du&amp;author=D.%20Wojtowicz&amp;author=A.%20A.%20Bowers&amp;author=D.%20Levens&amp;author=C.%20J.%20Benham&amp;author=T.%20M.%20Przytycka&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=5965-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23620297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Llorens-Rico</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Lluch-Senar</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Serrano</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Distinguishing between productive and abortive promoters using a random forest classifier in <em>Mycoplasma pneumoniae</em></div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">3442</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinguishing%20between%20productive%20and%20abortive%20promoters%20using%20a%20random%20forest%20classifier%20in%20Mycoplasma%20pneumoniae&amp;author=V.%20Llorens-Rico&amp;author=M.%20Lluch-Senar&amp;author=L.%20Serrano&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=3442-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25779052" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wade</div> <div class="given-names">J. T.</div></div><div class="name string-name"><div class="surname">Grainger</div> <div class="given-names">D. C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Pervasive transcription: illuminating the dark matter of bacterial transcriptomes</div>. <div class="source">Nat. Rev. Microbiol</div> ., <div class="volume">12</div>, <div class="fpage">647</div>–<div class="lpage">653</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pervasive%20transcription%3A%20illuminating%20the%20dark%20matter%20of%20bacterial%20transcriptomes&amp;author=J.%20T.%20Wade&amp;author=D.%20C.%20Grainger&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Microbiol&amp;volume=12&amp;pages=647-653" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro3316" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro3316" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro3316"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25069631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kopf</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W. R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Regulatory RNAs in photosynthetic cyanobacteria</div>. <div class="source">FEMS Microbiol. Rev</div> ., <div class="volume">39</div>, <div class="fpage">301</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20RNAs%20in%20photosynthetic%20cyanobacteria&amp;author=M.%20Kopf&amp;author=W.%20R.%20Hess&amp;publication_year=2015&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=39&amp;pages=301-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/femsre/fuv017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ffemsre%2Ffuv017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ffemsre%2Ffuv017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25934122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shearwin</div> <div class="given-names">K. E.</div></div><div class="name string-name"><div class="surname">Callen</div> <div class="given-names">B. P.</div></div><div class="name string-name"><div class="surname">Egan</div> <div class="given-names">J. B.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Transcriptional interference–a crash course</div>. <div class="source">Trends Genet</div> ., <div class="volume">21</div>, <div class="fpage">339</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20interference%E2%80%93a%20crash%20course&amp;author=K.%20E.%20Shearwin&amp;author=B.%20P.%20Callen&amp;author=J.%20B.%20Egan&amp;publication_year=2005&amp;journal=Trends%20Genet&amp;volume=21&amp;pages=339-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.04.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.04.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.04.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15922833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brophy</div> <div class="given-names">J. A.</div></div><div class="name string-name"><div class="surname">Voigt</div> <div class="given-names">C. A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Antisense transcription as a tool to tune gene expression</div>. <div class="source">Mol. Syst. Biol</div> ., <div class="volume">12</div>, <div class="fpage">854</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antisense%20transcription%20as%20a%20tool%20to%20tune%20gene%20expression&amp;author=J.%20A.%20Brophy&amp;author=C.%20A.%20Voigt&amp;publication_year=2016&amp;journal=Mol.%20Syst.%20Biol&amp;volume=12&amp;pages=854" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.15252/msb.20156540" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.15252%2Fmsb.20156540" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.15252%2Fmsb.20156540"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26769567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">X.-D.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Non-coding RNA: a new frontier in regulatory biology</div>. <div class="source">Natl. Sci. Rev</div> ., <div class="volume">1</div>, <div class="fpage">190</div>–<div class="lpage">204</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-coding%20RNA%3A%20a%20new%20frontier%20in%20regulatory%20biology&amp;author=X.-D.%20Fu&amp;publication_year=2014&amp;journal=Natl.%20Sci.%20Rev&amp;volume=1&amp;pages=190-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nsr/nwu008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnsr%2Fnwu008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnsr%2Fnwu008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25821635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weller</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Recknagel</div> <div class="given-names">R. D.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Promoter strength prediction based on occurrence frequencies of consensus patterns</div>. <div class="source">J. Theor. Biol</div> ., <div class="volume">171</div>, <div class="fpage">355</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoter%20strength%20prediction%20based%20on%20occurrence%20frequencies%20of%20consensus%20patterns&amp;author=K.%20Weller&amp;author=R.%20D.%20Recknagel&amp;publication_year=1994&amp;journal=J.%20Theor.%20Biol&amp;volume=171&amp;pages=355-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jtbi.1994.1239" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.1994.1239" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjtbi.1994.1239"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7869735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rhodius</div> <div class="given-names">V. A.</div></div><div class="name string-name"><div class="surname">Mutalik</div> <div class="given-names">V. K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Predicting strength and function for promoters of the <em>Escherichia coli</em> alternative sigma factor, sigmaE</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">2854</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20strength%20and%20function%20for%20promoters%20of%20the%20Escherichia%20coli%20alternative%20sigma%20factor%2C%20sigmaE&amp;author=V.%20A.%20Rhodius&amp;author=V.%20K.%20Mutalik&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=2854-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0915066107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0915066107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0915066107"> </span></div> </div></div></div></div></div><div content-id="dsw045-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rhodius</div> <div class="given-names">V. A.</div></div><div class="name string-name"><div class="surname">Mutalik</div> <div class="given-names">V. K.</div></div><div class="name string-name"><div class="surname">Gross</div> <div class="given-names">C. A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Predicting the strength of UP-elements and full-length <em>E. coli</em> sigmaE promoters</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">2907</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20the%20strength%20of%20UP-elements%20and%20full-length%20E.%20coli%20sigmaE%20promoters&amp;author=V.%20A.%20Rhodius&amp;author=V.%20K.%20Mutalik&amp;author=C.%20A.%20Gross&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=2907-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1190" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1190" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1190"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22156164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sobetzko</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Travers</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Muskhelishvili</div> <div class="given-names">G.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Gene order and chromosome dynamics coordinate spatiotemporal gene expression during the bacterial growth cycle</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">E42</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20order%20and%20chromosome%20dynamics%20coordinate%20spatiotemporal%20gene%20expression%20during%20the%20bacterial%20growth%20cycle&amp;author=P.%20Sobetzko&amp;author=A.%20Travers&amp;author=G.%20Muskhelishvili&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=E42-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1108229109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1108229109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1108229109"> </span></div> </div></div></div></div></div><div content-id="dsw045-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sobetzko</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Glinkowska</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Travers</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Muskhelishvili</div> <div class="given-names">G.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">DNA thermodynamic stability and supercoil dynamics determine the gene expression program during the bacterial growth cycle</div>. <div class="source">Mol. Biosyst</div> ., <div class="volume">9</div>, <div class="fpage">1643</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20thermodynamic%20stability%20and%20supercoil%20dynamics%20determine%20the%20gene%20expression%20program%20during%20the%20bacterial%20growth%20cycle&amp;author=P.%20Sobetzko&amp;author=M.%20Glinkowska&amp;author=A.%20Travers&amp;author=G.%20Muskhelishvili&amp;publication_year=2013&amp;journal=Mol.%20Biosyst&amp;volume=9&amp;pages=1643-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/c3mb25515h" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fc3mb25515h" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fc3mb25515h"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23493878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">C. M.</div></div><div class="name string-name"><div class="surname">Hoffmann</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Darfeuille</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">The primary transcriptome of the major human pathogen Helicobacter pylori</div>. <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">250</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20primary%20transcriptome%20of%20the%20major%20human%20pathogen%20Helicobacter%20pylori&amp;author=C.%20M.%20Sharma&amp;author=S.%20Hoffmann&amp;author=F.%20Darfeuille&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=250-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20164839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mitschke</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Vioque</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W. R.</div></div><div class="name string-name"><div class="surname">Muro-Pastor</div> <div class="given-names">A. M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Dynamics of transcriptional start site selection during nitrogen stress-induced cell differentiation in Anabaena sp. PCC7120</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">20130</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamics%20of%20transcriptional%20start%20site%20selection%20during%20nitrogen%20stress-induced%20cell%20differentiation%20in%20Anabaena%20sp.%20PCC7120&amp;author=J.%20Mitschke&amp;author=A.%20Vioque&amp;author=F.%20Haas&amp;author=W.%20R.%20Hess&amp;author=A.%20M.%20Muro-Pastor&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=20130-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1112724108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1112724108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1112724108"> </span></div> </div></div></div></div></div><div content-id="dsw045-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kopf</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Klahn</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Scholz</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Matthiessen</div> <div class="given-names">J. K.</div></div><div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W. R.</div></div><div class="name string-name"><div class="surname">Voss</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Comparative analysis of the primary transcriptome of Synechocystis sp. PCC 6803</div>. <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">527</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20primary%20transcriptome%20of%20Synechocystis%20sp.%20PCC%206803&amp;author=M.%20Kopf&amp;author=S.%20Klahn&amp;author=I.%20Scholz&amp;author=J.%20K.%20Matthiessen&amp;author=W.%20R.%20Hess&amp;author=B.%20Voss&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=527-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24935866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Hong</div> <div class="given-names">J. S.</div></div><div class="name string-name"><div class="surname">Qiu</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Comparative analysis of regulatory elements between Escherichia coli and Klebsiella pneumoniae by genome-wide transcription start site profiling</div>. <div class="source">PLoS Genet</div> ., <div class="volume">8</div>, <div class="fpage">e1002867</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20regulatory%20elements%20between%20Escherichia%20coli%20and%20Klebsiella%20pneumoniae%20by%20genome-wide%20transcription%20start%20site%20profiling&amp;author=D.%20Kim&amp;author=J.%20S.%20Hong&amp;author=Y.%20Qiu&amp;publication_year=2012&amp;journal=PLoS%20Genet&amp;volume=8&amp;pages=e1002867" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002867" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002867" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002867"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22912590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kroger</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Dillon</div> <div class="given-names">S. C.</div></div><div class="name string-name"><div class="surname">Cameron</div> <div class="given-names">A. D.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">The transcriptional landscape and small RNAs of Salmonella enterica serovar Typhimurium</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">E1277</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20landscape%20and%20small%20RNAs%20of%20Salmonella%20enterica%20serovar%20Typhimurium&amp;author=C.%20Kroger&amp;author=S.%20C.%20Dillon&amp;author=A.%20D.%20Cameron&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=E1277-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1201061109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1201061109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1201061109"> </span></div> </div></div></div></div></div><div content-id="dsw045-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">SantaLucia</div> <div class="given-names">J.</div>Jr.</div></div> <div class="year">1998</div>, 
<div class="article-title">A unified view of polymer, dumbbell, and oligonucleotide DNA nearest-neighbor thermodynamics</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">95</div>, <div class="fpage">1460</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20view%20of%20polymer%2C%20dumbbell%2C%20and%20oligonucleotide%20DNA%20nearest-neighbor%20thermodynamics&amp;author=J.%20SantaLucia&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=1460-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.4.1460" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.4.1460" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.4.1460"> </span></div> </div></div></div></div></div><div content-id="dsw045-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yella</div> <div class="given-names">V. R.</div></div><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, DNA Structure and Promoter Engineering. In: <div class="person-group"><div class="name string-name"><div class="surname">Singh</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Dhar</div> <div class="given-names">P. K.</div></div></div> (eds), <div class="source">Systems and Synthetic Biology</div> , 
<div class="publisher-name">Springer</div>, 
<div class="publisher-loc">Netherlands</div>, pp. <div class="fpage">241</div>–<div class="lpage">4</div>.</div></div></div></div><div content-id="dsw045-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brukner</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Sanchez</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Suck</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Pongor</div> <div class="given-names">S.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Trinucleotide models for DNA bending propensity: comparison of models based on DNaseI digestion and nucleosome packaging data</div>. <div class="source">J. Biomol. Struct. Dyn</div> ., <div class="volume">13</div>, <div class="fpage">309</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trinucleotide%20models%20for%20DNA%20bending%20propensity%3A%20comparison%20of%20models%20based%20on%20DNaseI%20digestion%20and%20nucleosome%20packaging%20data&amp;author=I.%20Brukner&amp;author=R.%20Sanchez&amp;author=D.%20Suck&amp;author=S.%20Pongor&amp;publication_year=1995&amp;journal=J.%20Biomol.%20Struct.%20Dyn&amp;volume=13&amp;pages=309-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.1995.10508842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.1995.10508842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.1995.10508842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8579790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Satchwell</div> <div class="given-names">S.C.</div></div><div class="name string-name"><div class="surname">Drew</div> <div class="given-names">H.R.</div></div><div class="name string-name"><div class="surname">Travers</div> <div class="given-names">A.A.</div></div></div> <div class="year">1986</div>, 
<div class="article-title">Sequence periodicities in chicken nucleosome core DNA</div>. <div class="source">J. Mol. Biol</div> ., <div class="volume">191</div>, <div class="fpage">659</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20periodicities%20in%20chicken%20nucleosome%20core%20DNA&amp;author=S.C.%20Satchwell&amp;author=H.R.%20Drew&amp;author=A.A.%20Travers&amp;publication_year=1986&amp;journal=J.%20Mol.%20Biol&amp;volume=191&amp;pages=659-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(86)90452-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(86)90452-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(86)90452-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3806678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kanhere</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">An assessment of three dinucleotide parameters to predict DNA curvature by quantitative comparison with experimental data</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">2647</div>–<div class="lpage">2658</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20assessment%20of%20three%20dinucleotide%20parameters%20to%20predict%20DNA%20curvature%20by%20quantitative%20comparison%20with%20experimental%20data&amp;author=A.%20Kanhere&amp;author=M.%20Bansal&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=2647-2658" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg362" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12736315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolshoy</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">McNamara</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Harrington</div> <div class="given-names">R. E.</div></div><div class="name string-name"><div class="surname">Trifonov</div> <div class="given-names">E. N.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Curved DNA without A-A: experimental estimation of all 16 DNA wedge angles</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">88</div>, <div class="fpage">2312</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Curved%20DNA%20without%20A-A%3A%20experimental%20estimation%20of%20all%2016%20DNA%20wedge%20angles&amp;author=A.%20Bolshoy&amp;author=P.%20McNamara&amp;author=R.%20E.%20Harrington&amp;author=E.%20N.%20Trifonov&amp;publication_year=1991&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=2312-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.6.2312" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.6.2312" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.6.2312"> </span></div> </div></div></div></div></div><div content-id="dsw045-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Bhattacharyya</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Ravi</div> <div class="given-names">B.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">NUPARM and NUCGEN: software for analysis and generation of sequence dependent nucleic acid structures</div>. <div class="source">Comput. Appl. Biosci</div> ., <div class="volume">11</div>, <div class="fpage">281</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NUPARM%20and%20NUCGEN%3A%20software%20for%20analysis%20and%20generation%20of%20sequence%20dependent%20nucleic%20acid%20structures&amp;author=M.%20Bansal&amp;author=D.%20Bhattacharyya&amp;author=B.%20Ravi&amp;publication_year=1995&amp;journal=Comput.%20Appl.%20Biosci&amp;volume=11&amp;pages=281-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7583696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barrett</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Wilhite</div> <div class="given-names">S. E.</div></div><div class="name string-name"><div class="surname">Ledoux</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">NCBI GEO: archive for functional genomics data sets–update</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D991</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20GEO%3A%20archive%20for%20functional%20genomics%20data%20sets%E2%80%93update&amp;author=T.%20Barrett&amp;author=S.%20E.%20Wilhite&amp;author=P.%20Ledoux&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D991-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1193" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1193" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1193"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw045-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw045-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anders</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Huber</div> <div class="given-names">W.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Differential expression analysis for sequence count data</div>. <div class="source">Genome Biol</div> ., <div class="volume">11</div>, <div class="fpage">R106</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20analysis%20for%20sequence%20count%20data&amp;author=S.%20Anders&amp;author=W.%20Huber&amp;publication_year=2010&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-10-r106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-10-r106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-10-r106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute. </div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2449750&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw045/1/dsw045_Supp.doc?Expires=2147483647&amp;Signature=e-YCdaEmtepFQ8HEp6zUpaPOGvxVeVK4yXkfgFWqXykHExzqhNhGa5SkJ2LrCZnkCDoOsYD6CUTcLFs2qEWniF77g4yfYKuZarKAW~5JxRUtZRo0hTALIWvCZjtrJ~0GnwLVQj8FPDV7Vp98bp5EH33n7VsNUmSlbPV2z9EieB0~E261qJlt4b4g-C-bFP7IBd9FuODuqyhqDEYfwQQ83HnqWxk27HXJpuDNMIqjt5U6fdfnTVKNsGdgGhiRIIsK0OcNRjjT60KOWvxJfi4-qOTph6fYkQKy25HRH~eYeyyvdnFacYcOsrQtYYxvWT0Zql4JPcrLI~3qpj39j9bL-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">523</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000397051500003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw045"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2449750"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000397051500003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f1%2f25%2f2449750">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000397051500003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F1%2F25%2F2449750" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2449750'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Unveiling DNA structural features of promoters associated with various types of TSSs in prokaryotic transcriptomes and their role in gene expression" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw045");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f25%2f2449750">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

