

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/4/355/353793","@type":"ScholarlyArticle","name":"Functionally Relevant Microsatellite Markers From Chickpea Transcription Factor Genes for Efficient Genotyping Applications and Trait Association Mapping","datePublished":"2013-04-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2013-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst015","keywords":["association mapping","chickpea","microsatellite","SNP","transcription factor"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/4/355/353793","author":[{"name":"Kujur, Alice","affiliation":"1  National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Bajaj, Deepak","affiliation":"1  National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Saxena, Maneesha S.","affiliation":"1  National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Tripathi, Shailesh","affiliation":"2  Division of Genetics ,  Indian Agricultural Research Institute (IARI) ,  New Delhi 110012 ,  India","@type":"Person"},{"name":"Upadhyaya, Hari D.","affiliation":"3  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324, Andhra Pradesh ,  India","@type":"Person"},{"name":"Gowda, C.L.L.","affiliation":"3  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324, Andhra Pradesh ,  India","@type":"Person"},{"name":"Singh, Sube","affiliation":"3  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324, Andhra Pradesh ,  India","@type":"Person"},{"name":"Jain, Mukesh","affiliation":"1  National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Tyagi, Akhilesh K.","affiliation":"1  National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Parida, Swarup K.","affiliation":"1  National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110067 ,  India","@type":"Person"}],"description":"Abstract.  We developed 1108 transcription factor gene-derived microsatellite (TFGMS) and 161 transcription factor functional domain-associated microsatellite (","pageStart":"355","pageEnd":"374","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Functionally Relevant Microsatellite Markers From Chickpea Transcription Factor Genes for Efficient Genotyping Applications and Trait Association Mapping","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Functionally Relevant Microsatellite Markers From Chickpea Transcription Factor Genes for Efficient Genotyping Applications and Trait Association Mapping" />
    <meta property="og:description" content="Abstract.  We developed 1108 transcription factor gene-derived microsatellite (TFGMS) and 161 transcription factor functional domain-associated microsatellite (" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/4/355/353793" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-04-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kujur, Alice" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Bajaj, Deepak" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Saxena, Maneesha S." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Tripathi, Shailesh" /><meta name="citation_author_institution" content="Division of Genetics, Indian Agricultural Research Institute (IARI), New Delhi 110012, India" /><meta name="citation_author" content="Upadhyaya, Hari D." /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Andhra Pradesh, India" /><meta name="citation_author" content="Gowda, C.L.L." /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Andhra Pradesh, India" /><meta name="citation_author" content="Singh, Sube" /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Andhra Pradesh, India" /><meta name="citation_author" content="Jain, Mukesh" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Tyagi, Akhilesh K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Parida, Swarup K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_title" content="Functionally Relevant Microsatellite Markers From Chickpea Transcription Factor Genes for Efficient Genotyping Applications and Trait Association Mapping" /><meta name="citation_firstpage" content="355" /><meta name="citation_lastpage" content="374" /><meta name="citation_doi" content="10.1093/dnares/dst015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2013/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Microsatellites within genes: structure, function, and evolution; Mol. Biol. Evol.;  citation_year=2004;  citation_volume=21;  citation_pages=991-1007; " /><meta name="citation_reference" content="citation_title=Genic microsatellite markers in plants: features and applications; Trends Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=48-55; " /><meta name="citation_reference" content="citation_title=Genic non-coding microsatellites in the rice genome: characterization, marker design and use in assessing genetic and evolutionary relationships among domesticated groups; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=140" /><meta name="citation_reference" content="citation_title=Unigene derived microsatellite markers for the cereal genomes; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=808-17; " /><meta name="citation_reference" content="citation_title=Functionally relevant microsatellites in sugarcane unigenes; BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=251" /><meta name="citation_reference" content="citation_title=Integration of novel SSR and gene-based SNP marker loci in the chickpea genetic map and establishment of new anchor points with Medicago truncatula genome; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=1415-41; " /><meta name="citation_reference" content="citation_title=Development and use of genic molecular markers (GMMs) for construction of a transcript map of chickpea (Cicer arietinum L.); Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=1577-89; " /><meta name="citation_reference" content="citation_title=EST-derived genic molecular markers: development and utilization for generating an advanced transcript map of chickpea; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=1449-62; " /><meta name="citation_reference" content="citation_title=Orphan legume crops enter the genomics era; Curr. Opin. Plant Biol.;  citation_year=2009;  citation_volume=12;  citation_pages=202-10; " /><meta name="citation_reference" content="citation_title=Development of chickpea EST-SSR markers and analysis of allelic variation across related species; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=591-608; " /><meta name="citation_reference" content="citation_title=De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=53-63; " /><meta name="citation_reference" content="citation_title=Gene discovery and tissue-specific transcriptome analysis in chickpea with massively parallel pyrosequencing and web resource development; Plant Physiol.;  citation_year=2011b;  citation_volume=156;  citation_pages=1661-78; " /><meta name="citation_reference" content="citation_title=Large-scale transcriptome analysis in chickpea (Cicer arietinum L.), an orphan legume crop of the semi-arid tropics of Asia and Africa; Plant Biotechnol. J.;  citation_year=2011;  citation_volume=9;  citation_pages=922-31; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of wild chickpea as a rich resource for marker development; Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=690-702; " /><meta name="citation_reference" content="citation_title=Regulating the regulators: the future prospects for transcription-factor-based agricultural biotechnology products; Plant Physiol.;  citation_year=2008;  citation_volume=147;  citation_pages=20-9; " /><meta name="citation_reference" content="citation_title=Legume transcription factor genes: what makes legumes so special?; Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=991-1001; " /><meta name="citation_reference" content="citation_title=Legume transcription factors: global regulators of plant development and response to the environment; Plant Physiol.;  citation_year=2007;  citation_volume=144;  citation_pages=538-49; " /><meta name="citation_reference" content="citation_title=Transcription factors regulating the progression of monocot and dicot seed development; Bioessays;  citation_year=2011;  citation_volume=33;  citation_pages=189-202; " /><meta name="citation_reference" content="citation_title=Functional domain marker (FDM): an in silico demonstration in Solanaceae using simple sequence repeats (SSRs); Plant Mol. Biol. Rep.;  citation_year=2010;  citation_volume=28;  citation_pages=352-6; " /><meta name="citation_reference" content="citation_title=Computational and experimental analysis of microsatellites in rice (Oryza sativa L.): frequency, length-variation, transposon associations and genetic marker potential; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1441-52; " /><meta name="citation_reference" content="citation_title=AGRIS: the Arabidopsis gene regulatory information server, an update; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=D1118-22; " /><meta name="citation_reference" content="citation_title=LegumeTFDB: an integrative database of Glycine max, Lotus japonicus and Medicago truncatula transcription factors; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=290-1; " /><meta name="citation_reference" content="citation_title=The first draft of the pigeonpea genome sequence; J. Plant Biochem. Biotechnol.;  citation_year=2011;  citation_volume=21;  citation_pages=98-112; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of chickpea (Cicer arietinum) provides a resource for trait improvement; Nat Biotechnol.;  citation_year=2013;  citation_volume=31;  citation_pages=240-46; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=PowerMarker: an integrated analysis environment for genetic marker analysis; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=2128-9; " /><meta name="citation_reference" content="citation_title=Characterization and mapping of sequence-tagged microsatellite sites in the chickpea (Cicer arietinum L.) genome; Mol. Gen. Genet.;  citation_year=1999;  citation_volume=262;  citation_pages=90-101; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=945-59; " /><meta name="citation_reference" content="citation_title=TASSEL: software for association mapping of complex traits in diverse samples; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2633-5; " /><meta name="citation_reference" content="citation_title=Spagedi: a versatile computer program to analyze spatial genetic structure at the individual or population levels; Mol. Ecol. Notes;  citation_year=2002;  citation_volume=2;  citation_pages=618-20; " /><meta name="citation_reference" content="citation_publisher=B. V. Kyazma Wageningen, Netherlands; MapQTL 6: Software for the Mapping of Quantitative Trait Loci in Experimental Populations of Diploid Species;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Validation of internal control genes for quantitative gene expression studies in chickpea (Cicer arietinum L.); Biochem. Biophys. Res. Commun.;  citation_year=2010;  citation_volume=396;  citation_pages=283-8; " /><meta name="citation_reference" content="citation_title=Stereochemistry of polypeptide chain configurations; J. Mol. Biol.;  citation_year=1963;  citation_volume=7;  citation_pages=95-9; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1451-2; " /><meta name="citation_reference" content="citation_publisher=Columbia University Press, New York; Molecular Evolutionary Genetics;  citation_year=1987; " /><meta name="citation_reference" content="citation_title=On the number of segregating sites in genetical models without recombination; Theor. Popul. Biol.;  citation_year=1975;  citation_volume=7;  citation_pages=256-76; " /><meta name="citation_reference" content="citation_title=Statistical-method for testing the neutral mutation hypothesis by DNA polymorphism; Genetics;  citation_year=1989;  citation_volume=123;  citation_pages=585-95; " /><meta name="citation_reference" content="citation_title=Extent of linkage disequilibrium in rice (Oryza sativa L.); Genetics;  citation_year=2007;  citation_volume=177;  citation_pages=2223-32; " /><meta name="citation_reference" content="citation_title=Association mapping of stigma and spikelet characteristics in rice (Oryza sativa L.); Mol. Breed.;  citation_year=2009;  citation_volume=24;  citation_pages=277-92; " /><meta name="citation_reference" content="citation_title=Development of microsatellite markers and analysis of intra-specific genetic variability in chickpea (Cicer arietinum L.); Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=1416-28; " /><meta name="citation_reference" content="citation_title=Advancing the STMS genomic resources for defining new locations on the intra-specific genetic linkage map of chickpea (Cicer arietinum L.); BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=117" /><meta name="citation_reference" content="citation_title=Molecular diversity and phylogeny in geographical collection of chickpea (Cicer sp.) accessions; J. Genet.;  citation_year=2011;  citation_volume=90;  citation_pages=e94-e100; " /><meta name="citation_reference" content="citation_title=Microsatellites in Brassica unigenes: relative abundance, marker design and use in comparative physical mapping and genome analysis; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=55-67; " /><meta name="citation_reference" content="citation_title=Diversity of microsatellites derived from genomic libraries and GenBank sequences in rice (Oryza sativa L.); Theor. Appl. Genet.;  citation_year=2000;  citation_volume=100;  citation_pages=713-22; " /><meta name="citation_reference" content="citation_title=A single domestication for maize shown by multilocus microsatellite genotyping; Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=6080-4; " /><meta name="citation_reference" content="citation_title=High mutation rate and mutational bias at (TAA)n microsatellite loci in chickpea (Cicer arietinum L.); Mol. Genet. Genomics;  citation_year=2001;  citation_volume=265;  citation_pages=1097-103; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis in the genus Cicer and cultivated chickpea using RAPD and ISSR markers; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=643-51; " /><meta name="citation_reference" content="citation_title=Genetic structure, diversity, and allelic richness in composite collection and reference set in chickpea (Cicer arietinum L.); BMC Plant Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=106" /><meta name="citation_reference" content="citation_title=Morphological variation in wild annual Cicer species in comparison to the cultigens; Euphytica;  citation_year=1997;  citation_volume=95;  citation_pages=309-19; " /><meta name="citation_reference" content="citation_title=A comparative study of chromosome morphology among the nine annual species of Cicer L; Cytobios;  citation_year=2000;  citation_volume=101;  citation_pages=37-53; " /><meta name="citation_reference" content="citation_title=Allozyme variation and phylogeny in annual species of Cicer (Leguminosae); Plant Syst. Evol.;  citation_year=1991;  citation_volume=175;  citation_pages=11-21; " /><meta name="citation_reference" content="citation_title=A note on the evolution of kabuli chickpeas as shown by induced mutations in Cicer reticulatum Ladizinsky; Genet. Resour. Crop Evol.;  citation_year=2009;  citation_volume=56;  citation_pages=7-12; " /><meta name="citation_reference" content="citation_title=Evolution of cultivated chickpea: four bottlenecks limit diversity and constrain adaptation; Funct. Plant Biol.;  citation_year=2003;  citation_volume=30;  citation_pages=1081-7; " /><meta name="citation_reference" content="citation_title=The cradle of agriculture; Science;  citation_year=2000;  citation_volume=288;  citation_pages=1602-3; " /><meta name="citation_reference" content="citation_title=Ecogeography of annual wild Cicer species: the poor state of the world collection; Crop Sci.;  citation_year=2003;  citation_volume=43;  citation_pages=1076-90; " /><meta name="citation_reference" content="citation_title=The origins of cultivation of Cicer arietinum L. and Vicia faba L.: early ﬁnds from Tell el-Kerkh, north-west Syria, late 10th millennium B.P; Veg. Hist. Archaeobot.;  citation_year=2006;  citation_volume=15;  citation_pages=197-204; " /><meta name="citation_reference" content="citation_title=Conservation and variability of sequence-tagged microsatellites (STMSs) from chickpea (Cicer aerietinum L.) within the genus Cicer; Theor. Appl. Genet.;  citation_year=2000;  citation_volume=101;  citation_pages=269-78; " /><meta name="citation_reference" content="citation_title=Genetics of flowering time in chickpea and its bearing on productivity in semiarid environments; Adv. Agron.;  citation_year=2001;  citation_volume=72;  citation_pages=107-38; " /><meta name="citation_reference" content="citation_title=Evolution in the genus Cicer vernalisation response and low temperature pod set in chickpea (C. arietinum L.) and its annual wild relatives; Aust. J. Agric. Res.;  citation_year=2005;  citation_volume=56;  citation_pages=1191-200; " /><meta name="citation_reference" content="citation_title=Natural variation in GS5 plays an important role in regulating grain size and yield in rice; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=1266-70; " /><meta name="citation_reference" content="citation_title=Patterns of linkage disequilibrium and association mapping in diploid alfalfa (M. sativa L.); Theor. Appl. Genet.;  citation_year=2012;  citation_volume=125;  citation_pages=577-90; " /><meta name="citation_reference" content="citation_title=Genetic structure and linkage disequilibrium pattern of a rapeseed (Brassica napus L.) association mapping panel revealed by microsatellites; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=125;  citation_pages=437-47; " /><meta name="citation_reference" content="citation_title=Association analysis of vegetable soybean quality traits with SSR markers; Plant Breed.;  citation_year=2011;  citation_volume=113;  citation_pages=413-506; " /><meta name="citation_reference" content="citation_title=Polymorphisms in monolignol biosynthetic genes are associated with biomass yield and agronomic traits in European maize (Zea mays L.); BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=12-22; " /><meta name="citation_reference" content="citation_title=SNP in starch biosynthesis genes associated with nutritional and functional properties of rice; Sci. Rep.;  citation_year=2012;  citation_volume=2;  citation_pages=557" /><meta name="citation_reference" content="citation_title=Linking differential domain functions of the GS3 protein to natural variation of grain size in rice; Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=19579-84; " /><meta name="citation_reference" content="citation_title=Global identification of targets of the Arabidopsis MADS domain protein AGAMOUS-Like15; Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=2563-77; " /><meta name="citation_reference" content="citation_title=Global analysis of gene activity during Arabidopsis seed development and identification of seed-specific transcription factors; Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=8063-70; " /><meta name="citation_reference" content="citation_title=A linkage map of chickpea (Cicer arietinum L.) genome based on recombinant inbred lines from a C. arietinum × C. reticulatum cross: localization of resistance genes for Fusarium wilt races 4 and 5; Theor. Appl. Genet.;  citation_year=2000;  citation_volume=101;  citation_pages=1155-63; " /><meta name="citation_reference" content="citation_title=Microsatellites in starch-synthesizing genes in relation to starch physicochemical properties in waxy rice (Oryza sativa L.); Theor. Appl. Genet.;  citation_year=2002;  citation_volume=105;  citation_pages=898-905; " /><meta name="citation_reference" content="citation_title=Novel ribosomal genes from maize are differentially expressed in the zygotic and somatic cell cycles; Mol. Gen. Genet.;  citation_year=1999;  citation_volume=261;  citation_pages=416-27; " /><meta name="citation_reference" content="citation_title=Conservation of noncoding microsatellites in plants: implication for gene regulation; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=323" /><meta name="citation_reference" content="citation_title=A novel zinc-binding motif revealed by solution structures of DNA-binding domains of Arabidopsis SBP-family transcription factors; J. Mol. Biol.;  citation_year=2004;  citation_volume=337;  citation_pages=49-63; " /><meta name="citation_reference" content="citation_title=Functional characterization of OsMADS18, a member of the AP1/SQUA subfamily of MADS box genes; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=2207-19; " /><meta name="citation_reference" content="citation_title=Flower and fruit development in Arabidopsis thaliana; Int. J. Dev. Biol.;  citation_year=2005;  citation_volume=49;  citation_pages=633-43; " /><meta name="citation_reference" content="citation_title=SPL8, a local regulator in a subset of gibberellin-mediated developmental processes in Arabidopsis; Plant Mol. Biol.;  citation_year=2006;  citation_volume=63;  citation_pages=429-39; " /><meta name="citation_reference" content="citation_title=Genomic organization, differential expression, and interaction of SQUAMOSA promoter-binding-like transcription factors and micro-RNA156 in rice; Plant Physiol.;  citation_year=2006;  citation_volume=142;  citation_pages=280-93; " /><meta name="citation_reference" content="citation_title=Genome-wide identiﬁcation and evolutionary analysis of the plant specific SBP-box transcription factor family; Gene;  citation_year=2008;  citation_volume=418;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=A naturally occurring epigenetic mutation in a gene encoding an SBP-box transcription factor inhibits tomato fruit ripening; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=948-52; " /><meta name="citation_reference" content="citation_title=Structure of linkage disequilibrium in plants; Annu. Rev. Plant Biol.;  citation_year=2003;  citation_volume=54;  citation_pages=357-74; " /><meta name="citation_reference" content="citation_title=Linkage disequilibrium and association studies in higher plants: present status and future prospects; Plant Mol. Biol.;  citation_year=2005;  citation_volume=57;  citation_pages=461-85; " /><meta name="citation_reference" content="citation_title=Candidate genes versus genome-wide associations: which are better for detecting genetic susceptibility to infectious disease?; Proc. R Soc. B;  citation_year=2011;  citation_volume=278;  citation_pages=1183-8; " /><meta name="citation_reference" content="citation_title=An SNP caused loss of seed shattering during rice domestication; Science;  citation_year=2006;  citation_volume=312;  citation_pages=1392-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/4/355/1011928/dst015.pdf" /><meta name="description" content="Abstract.  We developed 1108 transcription factor gene-derived microsatellite (TFGMS) and 161 transcription factor functional domain-associated microsatellite (" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/4/355/353793" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/4/355/353793" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/4/355/353793");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst015';
    </script>
<title>
	Functionally Relevant Microsatellite Markers From Chickpea Transcription Factor Genes for Efficient Genotyping Applications and Trait Association Mapping | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353793" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bfy4ATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VqdXIsIEFsaWNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJhamFqLCBEZWVwYWsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F4ZW5hLCBNYW5lZXNoYSBTLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUcmlwYXRoaSwgU2hhaWxlc2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBHZW5ldGljcywgSW5kaWFuIEFncmljdWx0dXJhbCBSZXNlYXJjaCBJbnN0aXR1dGUgKElBUkkpLCBOZXcgRGVsaGkgMTEwMDEyLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVwYWRoeWF5YSwgSGFyaSBELiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkludGVybmF0aW9uYWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIGZvciB0aGUgU2VtaS1BcmlkIFRyb3BpY3MgKElDUklTQVQpLCBQYXRhbmNoZXJ1IDUwMjMyNCwgQW5kaHJhIFByYWRlc2gsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR293ZGEsIEMuTC5MLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkludGVybmF0aW9uYWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIGZvciB0aGUgU2VtaS1BcmlkIFRyb3BpY3MgKElDUklTQVQpLCBQYXRhbmNoZXJ1IDUwMjMyNCwgQW5kaHJhIFByYWRlc2gsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2luZ2gsIFN1YmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnRlcm5hdGlvbmFsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBmb3IgdGhlIFNlbWktQXJpZCBUcm9waWNzIChJQ1JJU0FUKSwgUGF0YW5jaGVydSA1MDIzMjQsIEFuZGhyYSBQcmFkZXNoLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkphaW4sIE11a2VzaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUeWFnaSwgQWtoaWxlc2ggSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFyaWRhLCBTd2FydXAgSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJGdW5jdGlvbmFsbHkgUmVsZXZhbnQgTWljcm9zYXRlbGxpdGUgTWFya2VycyBGcm9tIENoaWNrcGVhIFRyYW5zY3JpcHRpb24gRmFjdG9yIEdlbmVzIGZvciBFZmZpY2llbnQgR2Vub3R5cGluZyBBcHBsaWNhdGlvbnMgYW5kIFRyYWl0IEFzc29jaWF0aW9uIE1hcHBpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM3NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzIHdpdGhpbiBnZW5lczogc3RydWN0dXJlLCBmdW5jdGlvbiwgYW5kIGV2b2x1dGlvbjsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTk5MS0xMDA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5pYyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGluIHBsYW50czogZmVhdHVyZXMgYW5kIGFwcGxpY2F0aW9uczsgVHJlbmRzIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDgtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmljIG5vbi1jb2RpbmcgbWljcm9zYXRlbGxpdGVzIGluIHRoZSByaWNlIGdlbm9tZTogY2hhcmFjdGVyaXphdGlvbiwgbWFya2VyIGRlc2lnbiBhbmQgdXNlIGluIGFzc2Vzc2luZyBnZW5ldGljIGFuZCBldm9sdXRpb25hcnkgcmVsYXRpb25zaGlwcyBhbW9uZyBkb21lc3RpY2F0ZWQgZ3JvdXBzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmlnZW5lIGRlcml2ZWQgbWljcm9zYXRlbGxpdGUgbWFya2VycyBmb3IgdGhlIGNlcmVhbCBnZW5vbWVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTgwOC0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbGx5IHJlbGV2YW50IG1pY3Jvc2F0ZWxsaXRlcyBpbiBzdWdhcmNhbmUgdW5pZ2VuZXM7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTI1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aW9uIG9mIG5vdmVsIFNTUiBhbmQgZ2VuZS1iYXNlZCBTTlAgbWFya2VyIGxvY2kgaW4gdGhlIGNoaWNrcGVhIGdlbmV0aWMgbWFwIGFuZCBlc3RhYmxpc2htZW50IG9mIG5ldyBhbmNob3IgcG9pbnRzIHdpdGggTWVkaWNhZ28gdHJ1bmNhdHVsYSBnZW5vbWU7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9MTQxNS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIHVzZSBvZiBnZW5pYyBtb2xlY3VsYXIgbWFya2VycyAoR01NcykgZm9yIGNvbnN0cnVjdGlvbiBvZiBhIHRyYW5zY3JpcHQgbWFwIG9mIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyMjsgIGNpdGF0aW9uX3BhZ2VzPTE1NzctODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVTVC1kZXJpdmVkIGdlbmljIG1vbGVjdWxhciBtYXJrZXJzOiBkZXZlbG9wbWVudCBhbmQgdXRpbGl6YXRpb24gZm9yIGdlbmVyYXRpbmcgYW4gYWR2YW5jZWQgdHJhbnNjcmlwdCBtYXAgb2YgY2hpY2twZWE7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9MTQ0OS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JwaGFuIGxlZ3VtZSBjcm9wcyBlbnRlciB0aGUgZ2Vub21pY3MgZXJhOyBDdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTIwMi0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgY2hpY2twZWEgRVNULVNTUiBtYXJrZXJzIGFuZCBhbmFseXNpcyBvZiBhbGxlbGljIHZhcmlhdGlvbiBhY3Jvc3MgcmVsYXRlZCBzcGVjaWVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTU5MS02MDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgY2hpY2twZWEgdHJhbnNjcmlwdG9tZSB1c2luZyBzaG9ydCByZWFkcyBmb3IgZ2VuZSBkaXNjb3ZlcnkgYW5kIG1hcmtlciBpZGVudGlmaWNhdGlvbjsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz01My02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBkaXNjb3ZlcnkgYW5kIHRpc3N1ZS1zcGVjaWZpYyB0cmFuc2NyaXB0b21lIGFuYWx5c2lzIGluIGNoaWNrcGVhIHdpdGggbWFzc2l2ZWx5IHBhcmFsbGVsIHB5cm9zZXF1ZW5jaW5nIGFuZCB3ZWIgcmVzb3VyY2UgZGV2ZWxvcG1lbnQ7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExYjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0xNjYxLTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSB0cmFuc2NyaXB0b21lIGFuYWx5c2lzIGluIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pLCBhbiBvcnBoYW4gbGVndW1lIGNyb3Agb2YgdGhlIHNlbWktYXJpZCB0cm9waWNzIG9mIEFzaWEgYW5kIEFmcmljYTsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTkyMi0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIG9mIHdpbGQgY2hpY2twZWEgYXMgYSByaWNoIHJlc291cmNlIGZvciBtYXJrZXIgZGV2ZWxvcG1lbnQ7IFBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NjkwLTcwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGluZyB0aGUgcmVndWxhdG9yczogdGhlIGZ1dHVyZSBwcm9zcGVjdHMgZm9yIHRyYW5zY3JpcHRpb24tZmFjdG9yLWJhc2VkIGFncmljdWx0dXJhbCBiaW90ZWNobm9sb2d5IHByb2R1Y3RzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDc7ICBjaXRhdGlvbl9wYWdlcz0yMC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZWd1bWUgdHJhbnNjcmlwdGlvbiBmYWN0b3IgZ2VuZXM6IHdoYXQgbWFrZXMgbGVndW1lcyBzbyBzcGVjaWFsPzsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9OTkxLTEwMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlZ3VtZSB0cmFuc2NyaXB0aW9uIGZhY3RvcnM6IGdsb2JhbCByZWd1bGF0b3JzIG9mIHBsYW50IGRldmVsb3BtZW50IGFuZCByZXNwb25zZSB0byB0aGUgZW52aXJvbm1lbnQ7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0NDsgIGNpdGF0aW9uX3BhZ2VzPTUzOC00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBmYWN0b3JzIHJlZ3VsYXRpbmcgdGhlIHByb2dyZXNzaW9uIG9mIG1vbm9jb3QgYW5kIGRpY290IHNlZWQgZGV2ZWxvcG1lbnQ7IEJpb2Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTE4OS0yMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgZG9tYWluIG1hcmtlciAoRkRNKTogYW4gaW4gc2lsaWNvIGRlbW9uc3RyYXRpb24gaW4gU29sYW5hY2VhZSB1c2luZyBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyAoU1NScyk7IFBsYW50IE1vbC4gQmlvbC4gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTM1Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGFuZCBleHBlcmltZW50YWwgYW5hbHlzaXMgb2YgbWljcm9zYXRlbGxpdGVzIGluIHJpY2UgKE9yeXphIHNhdGl2YSBMLik6IGZyZXF1ZW5jeSwgbGVuZ3RoLXZhcmlhdGlvbiwgdHJhbnNwb3NvbiBhc3NvY2lhdGlvbnMgYW5kIGdlbmV0aWMgbWFya2VyIHBvdGVudGlhbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNDQxLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BR1JJUzogdGhlIEFyYWJpZG9wc2lzIGdlbmUgcmVndWxhdG9yeSBpbmZvcm1hdGlvbiBzZXJ2ZXIsIGFuIHVwZGF0ZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9RDExMTgtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlZ3VtZVRGREI6IGFuIGludGVncmF0aXZlIGRhdGFiYXNlIG9mIEdseWNpbmUgbWF4LCBMb3R1cyBqYXBvbmljdXMgYW5kIE1lZGljYWdvIHRydW5jYXR1bGEgdHJhbnNjcmlwdGlvbiBmYWN0b3JzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTI5MC0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZmlyc3QgZHJhZnQgb2YgdGhlIHBpZ2VvbnBlYSBnZW5vbWUgc2VxdWVuY2U7IEouIFBsYW50IEJpb2NoZW0uIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9OTgtMTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSkgcHJvdmlkZXMgYSByZXNvdXJjZSBmb3IgdHJhaXQgaW1wcm92ZW1lbnQ7IE5hdCBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI0MC00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwQ2hhcnQ6IHNvZnR3YXJlIGZvciB0aGUgZ3JhcGhpY2FsIHByZXNlbnRhdGlvbiBvZiBsaW5rYWdlIG1hcHMgYW5kIFFUTHM7IEouIEhlcmVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvd2VyTWFya2VyOiBhbiBpbnRlZ3JhdGVkIGFuYWx5c2lzIGVudmlyb25tZW50IGZvciBnZW5ldGljIG1hcmtlciBhbmFseXNpczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yMTI4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gYW5kIG1hcHBpbmcgb2Ygc2VxdWVuY2UtdGFnZ2VkIG1pY3Jvc2F0ZWxsaXRlIHNpdGVzIGluIHRoZSBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKSBnZW5vbWU7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjYyOyAgY2l0YXRpb25fcGFnZXM9OTAtMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcmVuY2Ugb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgdXNpbmcgbXVsdGlsb2N1cyBnZW5vdHlwZSBkYXRhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTU7ICBjaXRhdGlvbl9wYWdlcz05NDUtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRBU1NFTDogc29mdHdhcmUgZm9yIGFzc29jaWF0aW9uIG1hcHBpbmcgb2YgY29tcGxleCB0cmFpdHMgaW4gZGl2ZXJzZSBzYW1wbGVzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI2MzMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BhZ2VkaTogYSB2ZXJzYXRpbGUgY29tcHV0ZXIgcHJvZ3JhbSB0byBhbmFseXplIHNwYXRpYWwgZ2VuZXRpYyBzdHJ1Y3R1cmUgYXQgdGhlIGluZGl2aWR1YWwgb3IgcG9wdWxhdGlvbiBsZXZlbHM7IE1vbC4gRWNvbC4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTYxOC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUIuIFYuIEt5YXptYSBXYWdlbmluZ2VuLCBOZXRoZXJsYW5kczsgTWFwUVRMIDY6IFNvZnR3YXJlIGZvciB0aGUgTWFwcGluZyBvZiBRdWFudGl0YXRpdmUgVHJhaXQgTG9jaSBpbiBFeHBlcmltZW50YWwgUG9wdWxhdGlvbnMgb2YgRGlwbG9pZCBTcGVjaWVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYWxpZGF0aW9uIG9mIGludGVybmFsIGNvbnRyb2wgZ2VuZXMgZm9yIHF1YW50aXRhdGl2ZSBnZW5lIGV4cHJlc3Npb24gc3R1ZGllcyBpbiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM5NjsgIGNpdGF0aW9uX3BhZ2VzPTI4My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGVyZW9jaGVtaXN0cnkgb2YgcG9seXBlcHRpZGUgY2hhaW4gY29uZmlndXJhdGlvbnM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5NjM7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTk1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E1OiBtb2xlY3VsYXIgZXZvbHV0aW9uYXJ5IGdlbmV0aWNzIGFuYWx5c2lzIHVzaW5nIG1heGltdW0gbGlrZWxpaG9vZCwgZXZvbHV0aW9uYXJ5IGRpc3RhbmNlLCBhbmQgbWF4aW11bSBwYXJzaW1vbnkgbWV0aG9kczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI3MzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG5hU1AgdjU6IGEgc29mdHdhcmUgZm9yIGNvbXByZWhlbnNpdmUgYW5hbHlzaXMgb2YgRE5BIHBvbHltb3JwaGlzbSBkYXRhOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE0NTEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNvbHVtYmlhIFVuaXZlcnNpdHkgUHJlc3MsIE5ldyBZb3JrOyBNb2xlY3VsYXIgRXZvbHV0aW9uYXJ5IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgbnVtYmVyIG9mIHNlZ3JlZ2F0aW5nIHNpdGVzIGluIGdlbmV0aWNhbCBtb2RlbHMgd2l0aG91dCByZWNvbWJpbmF0aW9uOyBUaGVvci4gUG9wdWwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTc1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yNTYtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsLW1ldGhvZCBmb3IgdGVzdGluZyB0aGUgbmV1dHJhbCBtdXRhdGlvbiBoeXBvdGhlc2lzIGJ5IEROQSBwb2x5bW9ycGhpc207IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTU4NS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0ZW50IG9mIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc3OyAgY2l0YXRpb25fcGFnZXM9MjIyMy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gbWFwcGluZyBvZiBzdGlnbWEgYW5kIHNwaWtlbGV0IGNoYXJhY3RlcmlzdGljcyBpbiByaWNlIChPcnl6YSBzYXRpdmEgTC4pOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI3Ny05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgbWljcm9zYXRlbGxpdGUgbWFya2VycyBhbmQgYW5hbHlzaXMgb2YgaW50cmEtc3BlY2lmaWMgZ2VuZXRpYyB2YXJpYWJpbGl0eSBpbiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz0xNDE2LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNpbmcgdGhlIFNUTVMgZ2Vub21pYyByZXNvdXJjZXMgZm9yIGRlZmluaW5nIG5ldyBsb2NhdGlvbnMgb24gdGhlIGludHJhLXNwZWNpZmljIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLik7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTExNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBkaXZlcnNpdHkgYW5kIHBoeWxvZ2VueSBpbiBnZW9ncmFwaGljYWwgY29sbGVjdGlvbiBvZiBjaGlja3BlYSAoQ2ljZXIgc3AuKSBhY2Nlc3Npb25zOyBKLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz1lOTQtZTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzIGluIEJyYXNzaWNhIHVuaWdlbmVzOiByZWxhdGl2ZSBhYnVuZGFuY2UsIG1hcmtlciBkZXNpZ24gYW5kIHVzZSBpbiBjb21wYXJhdGl2ZSBwaHlzaWNhbCBtYXBwaW5nIGFuZCBnZW5vbWUgYW5hbHlzaXM7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTU1LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgb2YgbWljcm9zYXRlbGxpdGVzIGRlcml2ZWQgZnJvbSBnZW5vbWljIGxpYnJhcmllcyBhbmQgR2VuQmFuayBzZXF1ZW5jZXMgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz03MTMtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2luZ2xlIGRvbWVzdGljYXRpb24gZm9yIG1haXplIHNob3duIGJ5IG11bHRpbG9jdXMgbWljcm9zYXRlbGxpdGUgZ2Vub3R5cGluZzsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz02MDgwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggbXV0YXRpb24gcmF0ZSBhbmQgbXV0YXRpb25hbCBiaWFzIGF0IChUQUEpbiBtaWNyb3NhdGVsbGl0ZSBsb2NpIGluIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyBNb2wuIEdlbmV0LiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjU7ICBjaXRhdGlvbl9wYWdlcz0xMDk3LTEwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIGFuYWx5c2lzIGluIHRoZSBnZW51cyBDaWNlciBhbmQgY3VsdGl2YXRlZCBjaGlja3BlYSB1c2luZyBSQVBEIGFuZCBJU1NSIG1hcmtlcnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9NjQzLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHN0cnVjdHVyZSwgZGl2ZXJzaXR5LCBhbmQgYWxsZWxpYyByaWNobmVzcyBpbiBjb21wb3NpdGUgY29sbGVjdGlvbiBhbmQgcmVmZXJlbmNlIHNldCBpbiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3JwaG9sb2dpY2FsIHZhcmlhdGlvbiBpbiB3aWxkIGFubnVhbCBDaWNlciBzcGVjaWVzIGluIGNvbXBhcmlzb24gdG8gdGhlIGN1bHRpZ2VuczsgRXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MzA5LTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBhcmF0aXZlIHN0dWR5IG9mIGNocm9tb3NvbWUgbW9ycGhvbG9neSBhbW9uZyB0aGUgbmluZSBhbm51YWwgc3BlY2llcyBvZiBDaWNlciBMOyBDeXRvYmlvczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0zNy01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxsb3p5bWUgdmFyaWF0aW9uIGFuZCBwaHlsb2dlbnkgaW4gYW5udWFsIHNwZWNpZXMgb2YgQ2ljZXIgKExlZ3VtaW5vc2FlKTsgUGxhbnQgU3lzdC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTc1OyAgY2l0YXRpb25fcGFnZXM9MTEtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm90ZSBvbiB0aGUgZXZvbHV0aW9uIG9mIGthYnVsaSBjaGlja3BlYXMgYXMgc2hvd24gYnkgaW5kdWNlZCBtdXRhdGlvbnMgaW4gQ2ljZXIgcmV0aWN1bGF0dW0gTGFkaXppbnNreTsgR2VuZXQuIFJlc291ci4gQ3JvcCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTctMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBjdWx0aXZhdGVkIGNoaWNrcGVhOiBmb3VyIGJvdHRsZW5lY2tzIGxpbWl0IGRpdmVyc2l0eSBhbmQgY29uc3RyYWluIGFkYXB0YXRpb247IEZ1bmN0LiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEwODEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNyYWRsZSBvZiBhZ3JpY3VsdHVyZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODg7ICBjaXRhdGlvbl9wYWdlcz0xNjAyLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVjb2dlb2dyYXBoeSBvZiBhbm51YWwgd2lsZCBDaWNlciBzcGVjaWVzOiB0aGUgcG9vciBzdGF0ZSBvZiB0aGUgd29ybGQgY29sbGVjdGlvbjsgQ3JvcCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MTA3Ni05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG9yaWdpbnMgb2YgY3VsdGl2YXRpb24gb2YgQ2ljZXIgYXJpZXRpbnVtIEwuIGFuZCBWaWNpYSBmYWJhIEwuOiBlYXJseSDvrIFuZHMgZnJvbSBUZWxsIGVsLUtlcmtoLCBub3J0aC13ZXN0IFN5cmlhLCBsYXRlIDEwdGggbWlsbGVubml1bSBCLlA7IFZlZy4gSGlzdC4gQXJjaGFlb2JvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gYW5kIHZhcmlhYmlsaXR5IG9mIHNlcXVlbmNlLXRhZ2dlZCBtaWNyb3NhdGVsbGl0ZXMgKFNUTVNzKSBmcm9tIGNoaWNrcGVhIChDaWNlciBhZXJpZXRpbnVtIEwuKSB3aXRoaW4gdGhlIGdlbnVzIENpY2VyOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTI2OS03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY3Mgb2YgZmxvd2VyaW5nIHRpbWUgaW4gY2hpY2twZWEgYW5kIGl0cyBiZWFyaW5nIG9uIHByb2R1Y3Rpdml0eSBpbiBzZW1pYXJpZCBlbnZpcm9ubWVudHM7IEFkdi4gQWdyb24uOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9MTA3LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gaW4gdGhlIGdlbnVzIENpY2VyIHZlcm5hbGlzYXRpb24gcmVzcG9uc2UgYW5kIGxvdyB0ZW1wZXJhdHVyZSBwb2Qgc2V0IGluIGNoaWNrcGVhIChDLiBhcmlldGludW0gTC4pIGFuZCBpdHMgYW5udWFsIHdpbGQgcmVsYXRpdmVzOyBBdXN0LiBKLiBBZ3JpYy4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTExOTEtMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsIHZhcmlhdGlvbiBpbiBHUzUgcGxheXMgYW4gaW1wb3J0YW50IHJvbGUgaW4gcmVndWxhdGluZyBncmFpbiBzaXplIGFuZCB5aWVsZCBpbiByaWNlOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTEyNjYtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm5zIG9mIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gYW5kIGFzc29jaWF0aW9uIG1hcHBpbmcgaW4gZGlwbG9pZCBhbGZhbGZhIChNLiBzYXRpdmEgTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyNTsgIGNpdGF0aW9uX3BhZ2VzPTU3Ny05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBzdHJ1Y3R1cmUgYW5kIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gcGF0dGVybiBvZiBhIHJhcGVzZWVkIChCcmFzc2ljYSBuYXB1cyBMLikgYXNzb2NpYXRpb24gbWFwcGluZyBwYW5lbCByZXZlYWxlZCBieSBtaWNyb3NhdGVsbGl0ZXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI1OyAgY2l0YXRpb25fcGFnZXM9NDM3LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbiBhbmFseXNpcyBvZiB2ZWdldGFibGUgc295YmVhbiBxdWFsaXR5IHRyYWl0cyB3aXRoIFNTUiBtYXJrZXJzOyBQbGFudCBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9NDEzLTUwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seW1vcnBoaXNtcyBpbiBtb25vbGlnbm9sIGJpb3N5bnRoZXRpYyBnZW5lcyBhcmUgYXNzb2NpYXRlZCB3aXRoIGJpb21hc3MgeWllbGQgYW5kIGFncm9ub21pYyB0cmFpdHMgaW4gRXVyb3BlYW4gbWFpemUgKFplYSBtYXlzIEwuKTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTItMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBpbiBzdGFyY2ggYmlvc3ludGhlc2lzIGdlbmVzIGFzc29jaWF0ZWQgd2l0aCBudXRyaXRpb25hbCBhbmQgZnVuY3Rpb25hbCBwcm9wZXJ0aWVzIG9mIHJpY2U7IFNjaS4gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NTU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlua2luZyBkaWZmZXJlbnRpYWwgZG9tYWluIGZ1bmN0aW9ucyBvZiB0aGUgR1MzIHByb3RlaW4gdG8gbmF0dXJhbCB2YXJpYXRpb24gb2YgZ3JhaW4gc2l6ZSBpbiByaWNlOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0xOTU3OS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGlkZW50aWZpY2F0aW9uIG9mIHRhcmdldHMgb2YgdGhlIEFyYWJpZG9wc2lzIE1BRFMgZG9tYWluIHByb3RlaW4gQUdBTU9VUy1MaWtlMTU7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yNTYzLTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgYW5hbHlzaXMgb2YgZ2VuZSBhY3Rpdml0eSBkdXJpbmcgQXJhYmlkb3BzaXMgc2VlZCBkZXZlbG9wbWVudCBhbmQgaWRlbnRpZmljYXRpb24gb2Ygc2VlZC1zcGVjaWZpYyB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTgwNjMtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbGlua2FnZSBtYXAgb2YgY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLikgZ2Vub21lIGJhc2VkIG9uIHJlY29tYmluYW50IGluYnJlZCBsaW5lcyBmcm9tIGEgQy4gYXJpZXRpbnVtIMOXIEMuIHJldGljdWxhdHVtIGNyb3NzOiBsb2NhbGl6YXRpb24gb2YgcmVzaXN0YW5jZSBnZW5lcyBmb3IgRnVzYXJpdW0gd2lsdCByYWNlcyA0IGFuZCA1OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTExNTUtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlcyBpbiBzdGFyY2gtc3ludGhlc2l6aW5nIGdlbmVzIGluIHJlbGF0aW9uIHRvIHN0YXJjaCBwaHlzaWNvY2hlbWljYWwgcHJvcGVydGllcyBpbiB3YXh5IHJpY2UgKE9yeXphIHNhdGl2YSBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9ODk4LTkwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgcmlib3NvbWFsIGdlbmVzIGZyb20gbWFpemUgYXJlIGRpZmZlcmVudGlhbGx5IGV4cHJlc3NlZCBpbiB0aGUgenlnb3RpYyBhbmQgc29tYXRpYyBjZWxsIGN5Y2xlczsgTW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjE7ICBjaXRhdGlvbl9wYWdlcz00MTYtMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZhdGlvbiBvZiBub25jb2RpbmcgbWljcm9zYXRlbGxpdGVzIGluIHBsYW50czogaW1wbGljYXRpb24gZm9yIGdlbmUgcmVndWxhdGlvbjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIHppbmMtYmluZGluZyBtb3RpZiByZXZlYWxlZCBieSBzb2x1dGlvbiBzdHJ1Y3R1cmVzIG9mIEROQS1iaW5kaW5nIGRvbWFpbnMgb2YgQXJhYmlkb3BzaXMgU0JQLWZhbWlseSB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzM3OyAgY2l0YXRpb25fcGFnZXM9NDktNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgY2hhcmFjdGVyaXphdGlvbiBvZiBPc01BRFMxOCwgYSBtZW1iZXIgb2YgdGhlIEFQMS9TUVVBIHN1YmZhbWlseSBvZiBNQURTIGJveCBnZW5lczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTM1OyAgY2l0YXRpb25fcGFnZXM9MjIwNy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rmxvd2VyIGFuZCBmcnVpdCBkZXZlbG9wbWVudCBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgSW50LiBKLiBEZXYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9NjMzLTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TUEw4LCBhIGxvY2FsIHJlZ3VsYXRvciBpbiBhIHN1YnNldCBvZiBnaWJiZXJlbGxpbi1tZWRpYXRlZCBkZXZlbG9wbWVudGFsIHByb2Nlc3NlcyBpbiBBcmFiaWRvcHNpczsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTQyOS0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBvcmdhbml6YXRpb24sIGRpZmZlcmVudGlhbCBleHByZXNzaW9uLCBhbmQgaW50ZXJhY3Rpb24gb2YgU1FVQU1PU0EgcHJvbW90ZXItYmluZGluZy1saWtlIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBhbmQgbWljcm8tUk5BMTU2IGluIHJpY2U7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0MjsgIGNpdGF0aW9uX3BhZ2VzPTI4MC05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgaWRlbnRp76yBY2F0aW9uIGFuZCBldm9sdXRpb25hcnkgYW5hbHlzaXMgb2YgdGhlIHBsYW50IHNwZWNpZmljIFNCUC1ib3ggdHJhbnNjcmlwdGlvbiBmYWN0b3IgZmFtaWx5OyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQxODsgIGNpdGF0aW9uX3BhZ2VzPTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuYXR1cmFsbHkgb2NjdXJyaW5nIGVwaWdlbmV0aWMgbXV0YXRpb24gaW4gYSBnZW5lIGVuY29kaW5nIGFuIFNCUC1ib3ggdHJhbnNjcmlwdGlvbiBmYWN0b3IgaW5oaWJpdHMgdG9tYXRvIGZydWl0IHJpcGVuaW5nOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTk0OC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIG9mIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaW4gcGxhbnRzOyBBbm51LiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MzU3LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGFuZCBhc3NvY2lhdGlvbiBzdHVkaWVzIGluIGhpZ2hlciBwbGFudHM6IHByZXNlbnQgc3RhdHVzIGFuZCBmdXR1cmUgcHJvc3BlY3RzOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9NDYxLTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYW5kaWRhdGUgZ2VuZXMgdmVyc3VzIGdlbm9tZS13aWRlIGFzc29jaWF0aW9uczogd2hpY2ggYXJlIGJldHRlciBmb3IgZGV0ZWN0aW5nIGdlbmV0aWMgc3VzY2VwdGliaWxpdHkgdG8gaW5mZWN0aW91cyBkaXNlYXNlPzsgUHJvYy4gUiBTb2MuIEI7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc4OyAgY2l0YXRpb25fcGFnZXM9MTE4My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBTTlAgY2F1c2VkIGxvc3Mgb2Ygc2VlZCBzaGF0dGVyaW5nIGR1cmluZyByaWNlIGRvbWVzdGljYXRpb247IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzEyOyAgY2l0YXRpb25fcGFnZXM9MTM5Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvNC8zNTUvMTAxMTkyOC9kc3QwMTUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdlIGRldmVsb3BlZCAxMTA4IHRyYW5zY3JpcHRpb24gZmFjdG9yIGdlbmUtZGVyaXZlZCBtaWNyb3NhdGVsbGl0ZSAoVEZHTVMpIGFuZCAxNjEgdHJhbnNjcmlwdGlvbiBmYWN0b3IgZnVuY3Rpb25hbCBkb21haW4tYXNzb2NpYXRlZCBtaWNyb3NhdGVsbGl0ZSAoIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTM3OTMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkoDC+ynmlEOaCz9SnSd9R7FHee1o=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f355%2f353793" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f355%2f353793" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13681" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&amp;Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/4">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5039551">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5039551">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5039553">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5039553">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5039557">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5039557">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5039578">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5039578">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5039695">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5039695">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5039703">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5039703">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5039712">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5039712">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5039720">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5039720">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/4/339/353499">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/4/375/354164">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Functionally Relevant Microsatellite Markers From Chickpea Transcription Factor Genes for Efficient Genotyping Applications and Trait Association Mapping
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alice Kujur</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alice+Kujur">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kujur A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kujur A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alice Kujur</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deepak Bajaj</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Deepak+Bajaj">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bajaj D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bajaj D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Deepak Bajaj</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maneesha S. Saxena</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maneesha+S.+Saxena">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saxena M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saxena M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Maneesha S. Saxena</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shailesh Tripathi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Division of Genetics</div>, <div class="insititution">Indian Agricultural Research Institute (IARI)</div>, <div class="addr-line">New Delhi 110012</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shailesh+Tripathi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tripathi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tripathi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shailesh Tripathi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hari D. Upadhyaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324, Andhra Pradesh</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hari+D.+Upadhyaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Upadhyaya H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Upadhyaya H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hari D. Upadhyaya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">C.L.L. Gowda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324, Andhra Pradesh</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=C.L.L.+Gowda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gowda C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gowda C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">C.L.L. Gowda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sube Singh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324, Andhra Pradesh</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sube+Singh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Singh S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Singh S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sube Singh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mukesh Jain</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mukesh+Jain">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jain M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jain M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mukesh Jain</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akhilesh K. Tyagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akhilesh+K.+Tyagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tyagi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tyagi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akhilesh K. Tyagi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Swarup K. Parida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91-11-26735228. Fax. +91-11-26741658. E-mail: <a href="mailto:swarup@nipgr.ac.in" target="_blank">swarup@nipgr.ac.in</a>; <a href="mailto:swarupdbt@gmail.com" target="_blank">swarupdbt@gmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Swarup+K.+Parida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parida S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Parida S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Swarup K. Parida</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">The three authors have contributed equally to the work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 4, 1 August 2013, Pages 355–374, <a href='https://doi.org/10.1093/dnares/dst015'>https://doi.org/10.1093/dnares/dst015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 April 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 November 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 April 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353793" href="/dnaresearch/article-pdf/20/4/355/1011928/dst015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Alice Kujur, Deepak Bajaj, Maneesha S. Saxena, Shailesh Tripathi, Hari D. Upadhyaya, C.L.L. Gowda, Sube Singh, Mukesh Jain, Akhilesh K. Tyagi, Swarup K. Parida;  Functionally Relevant Microsatellite Markers From Chickpea Transcription Factor Genes for Efficient Genotyping Applications and Trait Association Mapping, <em>DNA Research</em>, Volume 20, Issue 4, 1 August 2013, Pages 355–374, <a href='https://doi.org/10.1093/dnares/dst015'>https://doi.org/10.1093/dnares/dst015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353793?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353793?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353793?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353793?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353793?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353793?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353793?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Functionally%20Relevant%20Microsatellite%20Markers%20From%20Chickpea%20Transcription%20Factor%20Genes%20for%20Efficient%20Genotyping%20Applications%20and%20Trait%20Association%20Mapping&amp;publicationDate=2013-04-29&amp;volumeNum=20&amp;issueNum=4&amp;author=Kujur%2C%20Alice%3B%20Bajaj%2C%20Deepak&amp;startPage=355&amp;endPage=374&amp;contentId=10.1093%2Fdnares%2Fdst015&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f355%2f353793" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5039551 id="5039551" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We developed 1108 transcription factor gene-derived microsatellite (TFGMS) and 161 transcription factor functional domain-associated microsatellite (TFFDMS) markers from 707 TFs of chickpea. The robust amplification efficiency (96.5%) and high intra-specific polymorphic potential (34%) detected by markers suggest their immense utilities in efficient large-scale genotyping applications, including construction of both physical and functional transcript maps and understanding population structure. Candidate gene-based association analysis revealed strong genetic association of TFFDMS markers with three major seed and pod traits. Further, TFGMS markers in the 5′ untranslated regions of TF genes showing differential expression during seed development had higher trait association potential. The significance of TFFDMS markers was demonstrated by correlating their allelic variation with amino acid sequence expansion/contraction in the functional domain and alteration of secondary protein structure encoded by genes. The seed weight-associated markers were validated through traditional bi-parental genetic mapping. The determination of gene-specific linkage disequilibrium (LD) patterns in <em>desi</em> and <em>kabuli</em> based on single nucleotide polymorphism-microsatellite marker haplotypes revealed extended LD decay, enhanced LD resolution and trait association potential of genes. The evolutionary history of a strong seed-size/weight-associated TF based on natural variation and haplotype sharing among <em>desi</em>, <em>kabuli</em> and wild unravelled useful information having implication for seed-size trait evolution during chickpea domestication.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;association mapping&quot;">association mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="chickpea">chickpea</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="microsatellite">microsatellite</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factor&quot;">transcription factor</a></div>                            <h2 scrollto-destination=5039553 id="5039553" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Microsatellite markers are preferred in plant genetic and genome analyses owing to their desirable genetic attributes, including hypervariability, multi-allelic nature, reproducibility, co-dominant inheritance, chromosome-specific location and wide genomic distribution. The significance of such markers particularly those derived from the protein-coding sequences and non-coding untranslated regions (UTRs) of genes has reportedly been attributed to their inherent affinity to act as ‘functional genetic markers’ for rapidly establishing marker–trait linkages and to identify genes/quantitative trait loci (QTLs) for traits of agricultural importance in crop plants.<sup><a reveal-id="DST015C1  DST015C3" data-open="DST015C1  DST015C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> Moreover, these genic markers have proven to be advantageous over anonymous markers for functional genetic diversity estimation, construction of transcript map, marker-assisted selection, trait association analysis and comparative mapping as well as in evolutionary studies as anchor markers in plant species, including chickpea.<sup><a reveal-id="DST015C2  DST015C8" data-open="DST015C2  DST015C8" class="link link-ref link-reveal xref-bibr">2–8</a></sup> In chickpea, a total of ∼11 000 genic microsatellite markers have been developed so far <em>in silico</em> from the expressed sequence tags (ESTs)<sup><a reveal-id="DST015C7  DST015C10" data-open="DST015C7  DST015C10" class="link link-ref link-reveal xref-bibr">7–10</a></sup> and next-generation <em>de novo</em> transcript sequences of diverse contrasting genotypes.<sup><a reveal-id="DST015C11  DST015C14" data-open="DST015C11  DST015C14" class="link link-ref link-reveal xref-bibr">11–14</a></sup> However, only a smaller proportion (∼5–10%) of these markers have been validated experimentally and used for various genotyping applications in chickpea. The remaining (∼90–95%) larger sets of <em>in silico</em> designed genic markers have not even been verified for their amplification of target microsatellite sequence in chickpea genotypes from which the markers were designed. Nevertheless, the lack of requisite intra- and inter-specific genetic polymorphisms detected by such genic microsatellite markers,<sup><a reveal-id="DST015C6" data-open="DST015C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST015C7" data-open="DST015C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST015C9" data-open="DST015C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST015C10" data-open="DST015C10" class="link link-ref link-reveal xref-bibr">10</a></sup> which are derived from the conserved expressed component of the genome, has limited their use in identification and fine mapping of genes/QTLs for important agronomic traits in chickpea. Additionally, it is also difficult to select useful and informative genic markers from such largely available <em>in silico</em> marker datasets (∼11 000 markers) for validation and efficient genotyping applications in chickpea. Therefore, the construction of a smaller database comprising of experimentally validated and functionally relevant genic microsatellite markers with relatively high polymorphic potential is essential for various applications of genetics, genomics and breeding in chickpea.</p><p>The recent advancement of novel array-based sequencing technologies, high-throughput genotyping and modern genomic tools have accelerated the generation of transcript sequences encoding the known/candidate genes globally and their corresponding protein-coding and UTR sequences in diverse contrasting chickpea genotypes at a genome-wide scale.<sup><a reveal-id="DST015C8" data-open="DST015C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST015C11  DST015C14" data-open="DST015C11  DST015C14" class="link link-ref link-reveal xref-bibr">11–14</a></sup> It includes identification of 1773 transcription factor (TF)-encoding genes from very high-depth next-generation <em>de novo</em> transcript sequences of chickpea <em>desi</em> genotype ICC 4958 (Chickpea Transcriptome Database, CTDB release 1.0, <a class="link link-uri" href="http://www.nipgr.res.in/ctdb.html">http://www.nipgr.res.in/ctdb.html</a>).<sup><a reveal-id="DST015C12" data-open="DST015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> There have been substantial evidences on the role of functional domains in the TF gene-encoding proteins as master regulators for controlling diverse and complex cellular processes,<sup><a reveal-id="DST015C15" data-open="DST015C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST015C16" data-open="DST015C16" class="link link-ref link-reveal xref-bibr">16</a></sup> such as development and stress responses and domestication in monocot and dicot species including legumes.<sup><a reveal-id="DST015C16  DST015C18" data-open="DST015C16  DST015C18" class="link link-ref link-reveal xref-bibr">16–18</a></sup> The use of functional domains of TF genes for developing informative genic microsatellite markers in the case of two Solanaceous plant species, such as tomato and pepper, has been well documented.<sup><a reveal-id="DST015C19" data-open="DST015C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Hence, the TFs with well-characterized functional domains will serve as excellent candidates and valuable transcriptomic resource for novel sequence-based genic functional microsatellite marker development in chickpea. Besides, the large-scale validation and high-throughput genotyping of such designed TF gene-derived microsatellite (TFGMS) and TF functional domain-associated microsatellite (TFFDMS) markers in a diverse set of <em>desi</em>, <em>kabuli</em> and wild genotypes would have profound utility in various applications of structural, functional and comparative genomics, including marker-assisted genetic improvement in chickpea.</p><p>In view of the above, the present study was undertaken to develop informative TFGMS and TFFDMS markers from the TF genes and to evaluate their potential for large-scale genotyping applications in chickpea. The functional significance of such TF gene-based microsatellite markers was assessed by integrating trait association analysis with genetic mapping, differential expression profiling, protein modelling and linkage disequilibrium (LD)-based haplotype gene evolution in chickpea.</p>                            <h2 scrollto-destination=5039557 id="5039557" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5039558 id="5039558" class="section-title" data-legacy-id=s2a>2.1. Development and physical mapping of genic TFGMS and TFFDMS markers</h3>
<p>The FASTA sequences of 1773 unique TF encoding transcripts of chickpea genotype ICC4958<sup><a reveal-id="DST015C11" data-open="DST015C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DST015C12" data-open="DST015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> were batch acquired from Chickpea Transcriptome Database (CTDB version 1.0, <a class="link link-uri" href="http://www.nipgr.res.in/ctdb.html">http://www.nipgr.res.in/ctdb.html</a>). These transcript sequences were searched individually for microsatellites using MISA (Microsatellite, <a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa">http://pgrc.ipk-gatersleben.de/misa</a>). The identified microsatellites were characterized as perfect (mononucleotides to hexanucleotides) and hypervariable Class I (≥20 nucleotides) and Class II (12–20 nucleotides) based on the length of the repeat motifs.<sup><a reveal-id="DST015C4" data-open="DST015C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST015C20" data-open="DST015C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The microsatellite containing TF-encoding transcripts were analysed in the NCBI open reading frame (ORF) Finder (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/projects/gorf">http://www.ncbi.nlm.nih.gov/projects/gorf</a>) to predict the longest ORF within the transcripts. The amino acid sequences encoded by the predicted ORF of microsatellite containing transcripts were analysed using Pfam and further BLAST searched against the NCBI conserved domain and the nr protein database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi">http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi</a>) to determine the presence of functional domains within the ORF. Further, these amino acid sequences were sequence homology (BLAST) searched against 1770 AGRIS TF genes<sup><a reveal-id="DST015C21" data-open="DST015C21" class="link link-ref link-reveal xref-bibr">21</a></sup> of <em>Arabidopsis thaliana</em> (TAIR 10 release, <a class="link link-uri" href="http://www.arabidopsis.org">www.arabidopsis.org</a>), TF database<sup><a reveal-id="DST015C22" data-open="DST015C22" class="link link-ref link-reveal xref-bibr">22</a></sup> of legumes (1467 TFs of <em>Medicago truncatula</em>, 5035 of <em>Glycine max</em> and 1626 of <em>Lotus japonicas</em>) and 1470 TFs of <em>Cajanus cajan</em>.<sup><a reveal-id="DST015C23" data-open="DST015C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Based on above analyses, a set of microsatellite containing TF gene orthologues of chickpea with conserved functional domains were screened at a high bit score of ≥500 and <em>E-</em>value of ≤1e−41. Finally, the localization of functional domains in the ORF encoding amino acid sequences was correlated with the positions of microsatellite repeat motifs in their corresponding transcripts. The unique forward and reverse primers with the amplification product size of 200–400 bp targeting both functional domain and microsatellite repeat-motif regions were designed (BatchPrimer3, <a class="link link-uri" href="http://probes.pw.usda.gov/batchprimer3">http://probes.pw.usda.gov/batchprimer3</a>) based on two criteria (Fig. <a reveal-id="DST015F1" data-open="DST015F1" class="link link-reveal link-table xref-fig">1</a>), and two distinct groups of Type I and Type II TFFDMS markers in chickpea were developed. The unique primer pairs designed exclusively from the ORF and 5′ and 3′ UTRs of TF gene/transcript sequences flanking the microsatellite repeat motifs (without inclusion of the functional domain regions) were developed as TFGMS markers (Fig. <a reveal-id="DST015F1" data-open="DST015F1" class="link link-reveal link-table xref-fig">1</a>) in chickpea. </p>                        <a id="5039560" scrollto-destination="5039560"></a>
<div id="DST015F1" data-id="DST015F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/m_dst01501.jpeg?Expires=2147483647&Signature=niHfKuLM~552qMarwimzThgOYSUl8ShtzN4GUeUkQWVfjobLeU~frrwHKEJUmUWISGtp-jG2pvy-B5Djz9sU3lE9OcbPsqUkGqvJQOYA7YTyJ04Nmn3NEaWoKJGozCLa0DVPN7Y91vvVoXyFIVBE3XgLQlVP0HXUPbjyE24dirlQcjcNadDMn2yDKswJzOhskKPvwB5iXVtaZ8UNGxLdiADsBTGJ8VC6VsfBvDP0Id3qU~MW958DFZaIZOauZZO~6AkhEhGAxvaZxQUkB9lG4QB8GZB4h0G-WS4uSQCq8thapWvjSglfBvJtCGkUtmWkGUsxSabIocWqOqD9hjc9Kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Development of TFGMS and TFFDMS markers from different UTR and ORF sequence components of TF genes in chickpea. The forward and reverse primers designed from the transcript sequences flanking the microsatellite repeat motifs that are present within and/or adjacent to the functional domain regions of TF genes were used to develop Type I (A) and Type II (B) TFFDMS markers, respectively. Primers designed from the flanking sequences of the microsatellite repeat motifs that are present either in the ORF (A) (other than Type I and Type II TFFDMS) or 5′ and 3′ UTRs (B) of TF genes were used to develop TFGMS markers." path-from-xml="dst01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5039560/dst01501.jpeg" path-from-xml="dst01501.jpeg" target="_blank">View large</a><a section="5039560" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst01501.jpeg?Expires=2147483647&Signature=do3mnsxWhB1OxLj~1wTYdIHTiQHSWzmpZ6OBJOK7BB2gt6OH0m8NriHqbsUs3hvSZfyBsmZLoMdDVrB-rNcdvvkVnsM10YUf-N1dIfZ0GJNKfhsWj4R1IqWthxuKEqrsPye5xHH5XbsJIiX8hs0k8GI0npUqKp~~W6AzmGeRaoA1H5n3rAWmcbEP~KJ9WoYEayElWSNvoMrZwu6iEKIPuXZJD7mf7DHWOYKjOvbMB8-4qQSm4dWgXMD7L4Bl4j2N9~4~x905S1Gxd-bhLd5fbSEq6wPSK2X9d-pUPdtS~ap8lC11kzQeH8bLfo9JYKMDeqjEkkTJqOYJDl9GE81LlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5039560&ar=353793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Development of TFGMS and TFFDMS markers from different UTR and ORF sequence components of TF genes in chickpea. The forward and reverse primers designed from the transcript sequences flanking the microsatellite repeat motifs that are present within and/or adjacent to the functional domain regions of TF genes were used to develop Type I (A) and Type II (B) TFFDMS markers, respectively. Primers designed from the flanking sequences of the microsatellite repeat motifs that are present either in the ORF (A) (other than Type I and Type II TFFDMS) or 5′ and 3′ UTRs (B) of TF genes were used to develop TFGMS markers.</p></div></div><div content-id="DST015F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/m_dst01501.jpeg?Expires=2147483647&Signature=niHfKuLM~552qMarwimzThgOYSUl8ShtzN4GUeUkQWVfjobLeU~frrwHKEJUmUWISGtp-jG2pvy-B5Djz9sU3lE9OcbPsqUkGqvJQOYA7YTyJ04Nmn3NEaWoKJGozCLa0DVPN7Y91vvVoXyFIVBE3XgLQlVP0HXUPbjyE24dirlQcjcNadDMn2yDKswJzOhskKPvwB5iXVtaZ8UNGxLdiADsBTGJ8VC6VsfBvDP0Id3qU~MW958DFZaIZOauZZO~6AkhEhGAxvaZxQUkB9lG4QB8GZB4h0G-WS4uSQCq8thapWvjSglfBvJtCGkUtmWkGUsxSabIocWqOqD9hjc9Kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Development of TFGMS and TFFDMS markers from different UTR and ORF sequence components of TF genes in chickpea. The forward and reverse primers designed from the transcript sequences flanking the microsatellite repeat motifs that are present within and/or adjacent to the functional domain regions of TF genes were used to develop Type I (A) and Type II (B) TFFDMS markers, respectively. Primers designed from the flanking sequences of the microsatellite repeat motifs that are present either in the ORF (A) (other than Type I and Type II TFFDMS) or 5′ and 3′ UTRs (B) of TF genes were used to develop TFGMS markers." path-from-xml="dst01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5039560/dst01501.jpeg" path-from-xml="dst01501.jpeg" target="_blank">View large</a><a section="5039560" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst01501.jpeg?Expires=2147483647&Signature=do3mnsxWhB1OxLj~1wTYdIHTiQHSWzmpZ6OBJOK7BB2gt6OH0m8NriHqbsUs3hvSZfyBsmZLoMdDVrB-rNcdvvkVnsM10YUf-N1dIfZ0GJNKfhsWj4R1IqWthxuKEqrsPye5xHH5XbsJIiX8hs0k8GI0npUqKp~~W6AzmGeRaoA1H5n3rAWmcbEP~KJ9WoYEayElWSNvoMrZwu6iEKIPuXZJD7mf7DHWOYKjOvbMB8-4qQSm4dWgXMD7L4Bl4j2N9~4~x905S1Gxd-bhLd5fbSEq6wPSK2X9d-pUPdtS~ap8lC11kzQeH8bLfo9JYKMDeqjEkkTJqOYJDl9GE81LlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5039560&ar=353793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Development of TFGMS and TFFDMS markers from different UTR and ORF sequence components of TF genes in chickpea. The forward and reverse primers designed from the transcript sequences flanking the microsatellite repeat motifs that are present within and/or adjacent to the functional domain regions of TF genes were used to develop Type I (A) and Type II (B) TFFDMS markers, respectively. Primers designed from the flanking sequences of the microsatellite repeat motifs that are present either in the ORF (A) (other than Type I and Type II TFFDMS) or 5′ and 3′ UTRs (B) of TF genes were used to develop TFGMS markers.</p></div></div><p>The transcript sequences flanking the TFGMS and TFFDMS repeat motifs were BLAST searched (<em>E-</em>value = 0) against recently released draft genome pseudomolecules (version 1.1)<sup><a reveal-id="DST015C24" data-open="DST015C24" class="link link-ref link-reveal xref-bibr">24</a></sup> of chickpea to determine the physical positions (bp) of the designed genic markers on the eight chromosomes. The microsatellite markers with unique BLAST hits on the pseudomolecules were allocated individually across the chickpea chromosomes based on their ascending order of physical position (bp) and visualized using the MapChart software (version 2.2).<sup><a reveal-id="DST015C25" data-open="DST015C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p>                            <h3 scrollto-destination=5039562 id="5039562" class="section-title" data-legacy-id=s2b>2.2. Evaluation of amplification efficiency and polymorphic potential</h3>
<p>To evaluate the amplification efficiency of <em>in silico</em> designed genic microsatellite markers, primers for a total of 1000 perfect microsatellite markers, including 148 Class I and 691 Class II TFGMS and 161 TFFDMS markers, were synthesized and amplification carried out from genomic DNA of chickpea genotype ICC4958, the source of sequence for designing primers. The TFGMS and TFFDMS markers exhibiting clear and reproducible amplicons in the gel-based assay were further amplified using the genomic DNA isolated from 92 <em>desi</em> and <em>kabuli</em> chickpea genotypes and 4 accession of wild species (<em>Cicer reticulatum</em>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) for accessing their polymorphic potential. The standard polymerase chain reaction (PCR) constituents and optimum cyclic conditions except for annealing temperature (55–60°C, depending on the primers used) were used for PCR amplification. The amplified PCR products were resolved in 3.5% metaphor agarose gel and their band-size was determined against a 50-bp DNA ladder. A selected set of TFGMS and TFFDMS markers showing monomorphic as well as polymorphic amplification in metaphor agarose gel among 96 genotypes were further resolved using the automated fragment analyzer. For automated fragment analysis, the PCR products amplified from fluorescent dye-labelled primers were mixed with Hi-Di™ formamide loading buffer and GeneScan-600LIZ size standard (Applied Biosystems, IL, USA), denatured, cooled and resolved in automated 96 capillary ABI 3730xl DNA Analyzer (Applied Biosystems). The electrophoregram containing trace files were analysed, and automated allele calling was carried out using the GeneMapper version 4.0 (Applied Biosystems). Based on these analyses, the actual allele size (bp) and fragment length polymorphism (bp) detected by TFGMS and TFFDMS markers in 96 genotypes were determined. The genotyping data of all TFGMS and TFFDMS markers validated both in gel-based assay and automated fragment analysis were used to estimate the frequency and distribution of amplified alleles, average polymorphic alleles per marker, per cent polymorphism and polymorphism information content (PIC) employing the PowerMarker version 3.51.<sup><a reveal-id="DST015C26" data-open="DST015C26" class="link link-ref link-reveal xref-bibr">26</a></sup> To confirm that the TFGMS and TFFDMS markers did amplify the expected microsatellite repeat motifs in different genotypes, the amplified PCR products of size variant amplicons generated from different polymorphic TFGMS and TFFDMS markers were purified, cloned in pGEM-T Easy Vector (Promega, USA) and sequenced in both forward and reverse directions twice on a capillary-based Automated DNA Sequencer (Applied Biosystems, ABI 3730xl DNA Analyzer) using the BigDye Terminator v3.1 sequencing kit and M13 forward and reverse primers. The trace files were base called, checked for quality and then assembled into contigs. The high-quality consensus sequences thus obtained for each markers were aligned and compared among genotypes. The presence of predicted microsatellite repeat motifs and correspondence of fragment length polymorphism with repeat length expansion/contraction for each polymorphic markers were inferred among chickpea genotypes.</p>                            <h3 scrollto-destination=5039564 id="5039564" class="section-title" data-legacy-id=s2c>2.3. Assessment of population genetic structure and functional molecular diversity</h3>
<p>The validated polymorphic TFGMS and TFFDMS markers and 96 previously reported<sup><a reveal-id="DST015C27" data-open="DST015C27" class="link link-ref link-reveal xref-bibr">27</a></sup> genomic microsatellite markers distributed over eight chromosomes/linkage groups (LGs) of chickpea were used to determine population structure and functional molecular diversity and to establish genetic relationships among 96 genotypes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The marker genotyping data were used in a model-based programme, STRUCTURE,<sup><a reveal-id="DST015C28" data-open="DST015C28" class="link link-ref link-reveal xref-bibr">28</a></sup> for the determination of population structure using the admixture and correlated allele frequency with a burn-in of 50 000 iterations and run length of 100 000. Analysis of population genetic structure among genotypes was carried out using the Bayesian clustering algorithm of STRUCTURE with varying levels of K (number of population) = 2–10. The average of Ln P(D) against each <em>K</em> across 20 independent replications was estimated and plotted. The <em>K</em> at which Ln P(D) plateaus with <em>α</em>-value of &lt;0.2 was considered as optimum K. Using this optimum K, the population structure model representing better relationships among 96 genotypes was constructed. Various population genetic parameters, including the efficiency of TFGMS and TFFDMS marker loci, for detecting genetic variability (<em>F</em><sub>ST</sub>) and the degree of admixture within and between population groups were estimated.</p>                            <h3 scrollto-destination=5039566 id="5039566" class="section-title" data-legacy-id=s2d>2.4. Trait association analysis</h3>
<p>To evaluate the efficiency of TFGMS and TFFDMS markers for seed and pod trait-specific association in chickpea, the candidate gene-based association analysis was performed in 96 genotypes (representing &gt;85% diversity of total core germplasm lines), which were selected from 417 global core germplasm collections of chickpea based on their phenotypic and genotypic diversity estimate measures. The constituted association panel (representing 16 countries of the world) included 92 diverse small and large seed-size <em>desi</em> and <em>kabuli</em> germplasm lines, released varieties and breeding lines and four wild accessions of <em>C. reticulatum</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The genotyping data of validated polymorphic TFGMS and TFFDMS markers and 96 genomic microsatellite markers<sup><a reveal-id="DST015C27" data-open="DST015C27" class="link link-ref link-reveal xref-bibr">27</a></sup> distributed over eight chromosomes/LGs of chickpea and phenotypic data of three major seed and pod traits (100-seed weight, number of pods per plant and number of seeds per plant) on 96 association mapping panel were analysed in the TASSEL (<a class="link link-uri" href="http://www.maizegenetics.net">http://www.maizegenetics.net</a><sup><a reveal-id="DST015C29" data-open="DST015C29" class="link link-ref link-reveal xref-bibr">29</a></sup>). A general linear model (GLM) and mixed linear model (MLM) considering the multiple levels of ancestry coefficient data (<em>Q</em> matrix) as obtained in the population genetic structure at an optimal population number and relative kinship (<em>K</em>) matrix estimated from SPAGeDi 1.2<sup><a reveal-id="DST015C30" data-open="DST015C30" class="link link-ref link-reveal xref-bibr">30</a></sup> were used to measure the two important parameters of trait association namely, P_adj_marker (significant association of TFGMS and TFFDMS markers with traits) and marker R_square (magnitude of association/correlation, i.e. proportion of phenotypic variance explained by the markers with traits). The GLM trait association model was permuted 1000 times to optimize the threshold significance level for association analysis. Finally, based on the combined results of GLM and MLM analyses, the TFGMS and TFFDMS markers showing association with three major seed and pod traits in chickpea genotypes at significant cut-off <em>P</em>_adj value of ≤0.05 (with 95% confidence) and <em>R</em><sup>2</sup>-value of ≥0.10 were selected for further analyses.</p>                            <h3 scrollto-destination=5039568 id="5039568" class="section-title" data-legacy-id=s2e>2.5. Transcript map construction and genetic mapping</h3>
<p>The genotyping data of TFGMS and TFFDMS markers and genomic microsatellite markers showing polymorphism between two parental small <em>desi</em> (ICCX-810800 with 100-seed weight 11 g) and large <em>kabuli</em> (ICC20268 with 100-seed weight 47 g) genotypes and 94 individuals of F<sub>3</sub> mapping population derived from intra-specific crosses between above <em>desi</em> × <em>kabuli</em> genotypes were analysed using JoinMap 4.1 (<a class="link link-uri" href="http://www.kyazma.nl/index.php/mc.JoinMap">http://www.kyazma.nl/index.php/mc.JoinMap</a>) at higher logarithm of odds (LOD) threshold with Kosambi function. Finally, the genic microsatellite markers one in each TFs and genomic microsatellite markers were allocated into defined LGs, and the genetic map was constructed using the MapChart version 2.2.<sup><a reveal-id="DST015C25" data-open="DST015C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>Genetic/QTL mapping was performed using the genotyping data of 96 parental polymorphic TFGMS and TFFDMS markers and 32 genomic microsatellite markers, 100-seed weight/size trait-specific precise phenotyping data of 94 F<sub>3</sub> segregating individuals and 2 parental genotypes and genetic linkage map information of markers mapped on the eight LGs of chickpea. The segregating individuals along with parental genotypes were phenotyped in the field in randomized block design with at least two replications for two consecutive years, and 100-seed weight (g) was measured by taking the average weight of 100 seeds in each mapping individuals. The frequency distribution and analysis of variance (ANOVA) of 100-seed weight trait-specific phenotypic data in the 96 segregating mapping population were analysed using SPSS 17.0 (<a class="link link-uri" href="http://www.spss.com/statistics">http://www.spss.com/statistics</a>, IBM SPSS Inc. for window, Chicago, USA). The single marker analysis, interval mapping and composite interval mapping function of MapQTL 6<sup><a reveal-id="DST015C31" data-open="DST015C31" class="link link-ref link-reveal xref-bibr">31</a></sup> with an LOD threshold score of &gt;3.0 at 1000 permutations were significantly considered (5% level of significance) to identify and to map the major TF genes/QTLs for 100-seed weight/size on the chickpea LGs.</p>                            <h3 scrollto-destination=5039571 id="5039571" class="section-title" data-legacy-id=s2f>2.6. Differential expression profiling</h3>
<p>To access the utility of genic markers for regulatory gene functions, the differential expression analyses of seed and pod trait-specific TFGMS and TFFDMS-associated TF genes were performed in two chickpea genotypes. The total RNA was isolated (Trizol, Invitrogen, USA) from two different seed developmental stages (early cell division phase during 10–20 days after podding (DAP) and late maturation phase during 21–30 DAP) of small (<em>desi</em> ICCX-810800) and large (<em>kabuli</em> ICC20268) seed-size genotypes considering leaf at vegetative stage as control. The isolated total RNA was quantified and diluted to equal concentration and used for first-strand cDNA synthesis by the ImProm-II™ Reverese Transcription system (Promega). The synthesized cDNA was further amplified with above-said TF gene-based markers using the semi-quantitative and SYBR green chemistry-based real-time quantitative RT-PCR. The house-keeping gene elongation factor 1-alpha (<em>EF1α</em>) suitable for consistent expression across various organs and developmental stages of chickpea<sup><a reveal-id="DST015C32" data-open="DST015C32" class="link link-ref link-reveal xref-bibr">32</a></sup> was used as internal control in RT-PCR assays. Three biological replicates of each sample and three technical replicates of each biological replicate, with no template and primer as control, were analysed for individual marker in quantitative RT-PCR. The expression level of trait-associated genic markers obtained in diverse seed developmental stages of two contrasting genotypes was compared with each other and with control. Based on this, a set of trait-associated TF genes showing differential expression during seed development in chickpea genotypes was identified. To confirm the presence of TFGMS and TFFDMS repeat motifs in the transcripts and further to derive the correspondence of differential expression profiling with fragment length polymorphism based on variable microsatellite repeat units, the cDNA and genomic DNA fragments amplified from seed and pod trait-associated TF gene-based markers in two contrasting chickpea genotypes were cloned and sequenced as mentioned above.</p>                            <h3 scrollto-destination=5039573 id="5039573" class="section-title" data-legacy-id=s2g>2.7. Functional relevance of TFFDMS markers <em>in silico</em></h3>
<p>To infer <em>in silico</em>, the functional significance of seed and pod trait-associated TFFDMS markers showing fragment length polymorphism in both small and large seed-size chickpea genotypes, the amino acid sequences of the functional domains carrying microsatellites in five TF genes was analysed using Modeller 9v8 (<a class="link link-uri" href="http://salilab.org/modeller">http://salilab.org/modeller</a>). Three-dimensional homology modelling of these TFFDMS containing TF gene-encoding proteins were performed by selecting a suitable template in each case where 10 models of the target proteins were generated. These predicted models were energy minimized (1000 steps) and their overall stereochemical quality and accuracy were evaluated employing Ramchandran plot analysis.<sup><a reveal-id="DST015C33" data-open="DST015C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The best model was selected having &gt;90% of its residues lying in the most favoured region of Ramchandran plot. Finally, the best models of each TFFDMS carrying TF gene-encoding proteins were visualized and analysed using a highly interactive and extensive molecular graphic programme of CHIMERA (<a class="link link-uri" href="http://www.cgl.ucsj.edu/chimera">www.cgl.ucsj.edu/chimera</a>).</p>                            <h3 scrollto-destination=5039575 id="5039575" class="section-title" data-legacy-id=s2h>2.8. Estimation of gene-specific LD patterns</h3>
<p>To determine the pattern of LD within the genes, the seed and pod trait-specific TFGMS and TFFDMS markers-associated TF genes were cloned and sequenced. The high-quality sequences generated for each gene were aligned among 96 chickpea genotypes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) using the CLUSTALW multiple sequence alignment tool in MEGA 4.0<sup><a reveal-id="DST015C34" data-open="DST015C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and mined the single nucleotide polymorphism (SNP) loci. The results were exported in meg format, and the meg files were analysed further using DNaSp version 5.0<sup><a reveal-id="DST015C35" data-open="DST015C35" class="link link-ref link-reveal xref-bibr">35</a></sup> to estimate the level of nucleotide diversity (<em>π</em>),<sup><a reveal-id="DST015C36" data-open="DST015C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Watterson's <em>θ</em> level of SNPs for the population mutation rate<sup><a reveal-id="DST015C37" data-open="DST015C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and Tajima's <em>D</em> statistic<sup><a reveal-id="DST015C38" data-open="DST015C38" class="link link-ref link-reveal xref-bibr">38</a></sup> across the genotypes. Finally, the SNP (minor allele frequency ≥0.05), TFGMS and TFFDMS marker genotyping information generated in the TF genes were used for the constitution of haplotypes and the determination of haplotype diversity and LD patterns within these genes among <em>desi</em> and <em>kabuli</em>. The genic SNP and microsatellite marker-based haplotype genotyping information were further correlated with three major seed and pod trait-specific phenotypic information of 96 association mapping panel for trait association analysis using the TASSEL as mentioned earlier. In the TF genes, the LD decay was estimated by combining the <em>r</em><sup>2</sup>-values of marker pairs present in each 100 bp equal interval sequenced regions between 0 and 1 kb. The graph was plotted between pooled <em>r</em><sup>2</sup> and physical distance based on the non-linear regression model;<sup><a reveal-id="DST015C39" data-open="DST015C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DST015C40" data-open="DST015C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and ultimately, the trend of LD decay was estimated in <em>desi</em> and <em>kabuli</em>.</p>                            <h2 scrollto-destination=5039578 id="5039578" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5039582 id="5039582" class="section-title" data-legacy-id=s3a>3.1. Development of genic TFGMS and TFFDMS markers and their PCR amplification efficiency</h3>
<p>A total of 1773 TF gene-encoding transcript sequences of chickpea ICC4958 were mined for microsatellite repeat motifs and used to design both the genic TFGMS and TFFDMS markers for large-scale genotyping applications. Forward and reverse primer pairs could be designed from the transcript sequences flanking 1269 perfect microsatellite repeat motifs present in 707 TF genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The analysis led to the development of 1108, including 148 (13.4%) Class I and 960 (86.6%) Class II TFGMS markers in 564 TF genes of chickpea. A set of 567 (51.2%) and 541 (48.8%) TFGMS markers were designed from the ORF and 5′ and 3′ UTRs of the TF genes, respectively. The density of TFGMS markers in the ORF of TF genes was higher (1 per 1.1 kb) than that estimated in the UTRs (1 per 1.4 kb). The CAA-rich trinucleotide (82.2%) and non-triplet repeat motifs (57.7%) carrying TFGMS markers were predominant in the ORF and UTRs of TF genes, respectively. It could be due to more selection against the frameshift mutations in the ORF than the UTRs, resulting in limited expansion of non-triplet microsatellite motifs in the ORF of TF genes in chickpea.<sup><a reveal-id="DST015C2  DST015C4" data-open="DST015C2  DST015C4" class="link link-ref link-reveal xref-bibr">2–4</a>,<a reveal-id="DST015C11" data-open="DST015C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DST015C12" data-open="DST015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> According to localization of the microsatellite repeat motifs adjacent/within the functional domains of TF genes (Fig. <a reveal-id="DST015F1" data-open="DST015F1" class="link link-reveal link-table xref-fig">1</a>), a set of 161 including 20 (12.4%) Class I TFFDMS markers in the 143 TF genes was designed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). It comprises 74 Type I and 87 Type II TFFDMS markers. The trinucleotide repeat motifs (87.7%) carrying TFFDMS markers were abundant in the TF genes. Remarkably, we identified 27 such genes in which multiple TFGMS and TFFDMS markers could be designed from individual genes. It thus resulted in the development of 64 markers, including 37 TFGMS and 27 TFFDMS, from 27 TF genes. All the designed 1108 TFGMS and 161 TFFDMS markers in the 707 TF genes were submitted to publicly available NCBI Probe database (Accession numbers PUID16584974 to PUID16586242) for unrestricted use (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Our marker dataset was compared between earlier reported 657 TF gene-derived and 279 EST-based microsatellite markers in chickpea,<sup><a reveal-id="DST015C6" data-open="DST015C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST015C7" data-open="DST015C7" class="link link-ref link-reveal xref-bibr">7</a></sup> where notably two genic microsatellite markers were found to be common in both. This confirms TF gene-based microsatellite markers developed in our study for chickpea genome to be novel and not reported earlier.</p><p>To evaluate the amplification efficiency of designed genic microsatellite markers, a selected set of 1000 including 148 Class I and 691 Class II perfect TFGMS markers and 161 TFFDMS markers were PCR amplified (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). A total of 801 (95.5%) TFGMS and 157 (97.5%) TFFDMS markers of these produced clear and reproducible amplicons with fragments of expected product size in metaphor agarose gel. The remaining (3–5%) did not give amplification in any of the 96 chickpea genotypes including the ICC4958 from which the primers were designed. To verify the amplicon for expected microsatellite repeat motifs and also to exclude the possibility of cross-amplification particularly for the TFFDMS markers, the amplified PCR products obtained from each of 24 TFGMS and TFFDMS markers in selected genotypes were cloned and sequenced. In about 97% of markers, the existence of target TFGMS and TFFDMS repeat motifs as well as their corresponding flanking transcript sequences were observed (Fig. <a reveal-id="DST015F2" data-open="DST015F2" class="link link-reveal link-table xref-fig">2</a>A). A high (average 96.5%) amplification success rate of TFGMS and TFFDMS markers and their subsequent validation through large-scale cloned amplicon sequencing suggested the utility of TF gene-encoding transcript sequences in designing the informative sequence-based robust genic markers for their immense use in various large-scale genotyping applications in chickpea. </p>                        <a id="5039593" scrollto-destination="5039593"></a>
<div id="DST015F2" data-id="DST015F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/m_dst01502.jpeg?Expires=2147483647&Signature=V96eBZhhYcSVSxypAEY97WGhQA6Tzl52sKrrRPHFApa~SnNxaV5RLUYW3CylXEcqjNwqOLJ6tUxmSZDJyTZw9~D2XWzXtY3c-B-sZiNbvmnw9us~q0ht05vFKS2cVxJr1cj~kdN04i3YU1w0Ixydnd43rFDt8Hi8-ovDUBO0fcPjXpMqrKpOnR3~~2xS211KpjEiRdO0o9R00cAkDXRLAen6qLi8Lxf-aqRkObo2z5yAS7y0KLxcyGzPkvOsPCnF6wgsL-pkC9ejfTlxKwA1wgnp4w1UhaP6~muQmacBdpGAjvOdPSTvyI~Q9MJa5-sQZ0zDf6GX4SiAKaP8LL27IA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Allelic variation detected among a representative set of desi and kabuli genotypes using normal unlabelled and fluorescent dye-labelled TFGMS and TFFDMS markers in gel-based assay (A) and automated fragment analyzer (B), respectively. A maximum of 8 and 10 polymorphic alleles were amplified by markers among 96 chickpea genotypes using the gel-based assay (A) and automated fragment analyzer (B), respectively. The fragment sizes (bp) for all the amplified polymorphic alleles are indicated. The sequencing of cloned amplicons from different TFGMS and TFFDMS markers showing fragment length polymorphism validated the presence of expected microsatellite repeat motifs and further corresponded with their variations and step-wise multiples of the number of repeat units." path-from-xml="dst01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5039593/dst01502.jpeg" path-from-xml="dst01502.jpeg" target="_blank">View large</a><a section="5039593" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst01502.jpeg?Expires=2147483647&Signature=Hq-zZYlm~sW7TC8tGBQ1eOFzKsHcCaocDIodJK8H3LxyOK~3mafT~PJD6GkcSyBKL2GfLz7KMLJdzJb5wXaDw4Y137tuekw3fzS3DS5Fstl5UI76PfNcyGJtRqU1~7obwRemtuvrZdwDiAEXfH7RdXZtArZIlsE6tCe02-Q08qnCmUEIEvzH8tX8MNRUl-EgxvqSnaJGlDilB4iYZ7bVx-OXj06OOmXdOo-UfXCQH8DuztyLM52JNJ0n-hGV-lx3JnqQa~z9j6ZoGAu2PZ~7s3aRRwSx5Mt4trLdJRV2vSGM2FlQ9p2tfFUXrc4fXlB~bpBFJ6FrralW8skkNRkm8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5039593&ar=353793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Allelic variation detected among a representative set of <em>desi</em> and <em>kabuli</em> genotypes using normal unlabelled and fluorescent dye-labelled TFGMS and TFFDMS markers in gel-based assay (A) and automated fragment analyzer (B), respectively. A maximum of 8 and 10 polymorphic alleles were amplified by markers among 96 chickpea genotypes using the gel-based assay (A) and automated fragment analyzer (B), respectively. The fragment sizes (bp) for all the amplified polymorphic alleles are indicated. The sequencing of cloned amplicons from different TFGMS and TFFDMS markers showing fragment length polymorphism validated the presence of expected microsatellite repeat motifs and further corresponded with their variations and step-wise multiples of the number of repeat units.</p></div></div><div content-id="DST015F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/m_dst01502.jpeg?Expires=2147483647&Signature=V96eBZhhYcSVSxypAEY97WGhQA6Tzl52sKrrRPHFApa~SnNxaV5RLUYW3CylXEcqjNwqOLJ6tUxmSZDJyTZw9~D2XWzXtY3c-B-sZiNbvmnw9us~q0ht05vFKS2cVxJr1cj~kdN04i3YU1w0Ixydnd43rFDt8Hi8-ovDUBO0fcPjXpMqrKpOnR3~~2xS211KpjEiRdO0o9R00cAkDXRLAen6qLi8Lxf-aqRkObo2z5yAS7y0KLxcyGzPkvOsPCnF6wgsL-pkC9ejfTlxKwA1wgnp4w1UhaP6~muQmacBdpGAjvOdPSTvyI~Q9MJa5-sQZ0zDf6GX4SiAKaP8LL27IA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Allelic variation detected among a representative set of desi and kabuli genotypes using normal unlabelled and fluorescent dye-labelled TFGMS and TFFDMS markers in gel-based assay (A) and automated fragment analyzer (B), respectively. A maximum of 8 and 10 polymorphic alleles were amplified by markers among 96 chickpea genotypes using the gel-based assay (A) and automated fragment analyzer (B), respectively. The fragment sizes (bp) for all the amplified polymorphic alleles are indicated. The sequencing of cloned amplicons from different TFGMS and TFFDMS markers showing fragment length polymorphism validated the presence of expected microsatellite repeat motifs and further corresponded with their variations and step-wise multiples of the number of repeat units." path-from-xml="dst01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5039593/dst01502.jpeg" path-from-xml="dst01502.jpeg" target="_blank">View large</a><a section="5039593" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst01502.jpeg?Expires=2147483647&Signature=Hq-zZYlm~sW7TC8tGBQ1eOFzKsHcCaocDIodJK8H3LxyOK~3mafT~PJD6GkcSyBKL2GfLz7KMLJdzJb5wXaDw4Y137tuekw3fzS3DS5Fstl5UI76PfNcyGJtRqU1~7obwRemtuvrZdwDiAEXfH7RdXZtArZIlsE6tCe02-Q08qnCmUEIEvzH8tX8MNRUl-EgxvqSnaJGlDilB4iYZ7bVx-OXj06OOmXdOo-UfXCQH8DuztyLM52JNJ0n-hGV-lx3JnqQa~z9j6ZoGAu2PZ~7s3aRRwSx5Mt4trLdJRV2vSGM2FlQ9p2tfFUXrc4fXlB~bpBFJ6FrralW8skkNRkm8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5039593&ar=353793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Allelic variation detected among a representative set of <em>desi</em> and <em>kabuli</em> genotypes using normal unlabelled and fluorescent dye-labelled TFGMS and TFFDMS markers in gel-based assay (A) and automated fragment analyzer (B), respectively. A maximum of 8 and 10 polymorphic alleles were amplified by markers among 96 chickpea genotypes using the gel-based assay (A) and automated fragment analyzer (B), respectively. The fragment sizes (bp) for all the amplified polymorphic alleles are indicated. The sequencing of cloned amplicons from different TFGMS and TFFDMS markers showing fragment length polymorphism validated the presence of expected microsatellite repeat motifs and further corresponded with their variations and step-wise multiples of the number of repeat units.</p></div></div>                            <h3 scrollto-destination=5039599 id="5039599" class="section-title" data-legacy-id=s3b>3.2. Polymorphic potential of genic TFGMS and TFFDMS markers and molecular basis of their fragment length polymorphism</h3>
<p>The 801 TFGMS (137 Class I and 664 Class II) and 157 TFFDMS markers showing successful amplification were used to study polymorphic potential among a set of 92 <em>desi</em> and <em>kabuli</em> genotypes and 4 accessions of a wild species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Two hundred and fifty-eight (32.2%) of the 801 TFGMS markers in the 203 TF genes were polymorphic (with average PIC of 0.44) among 96 genotypes in 3.5% metaphor agarose gel (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). It included 54 of the 137 (39.4%, with an average PIC of 0.51) Class I and 204 of the 664 (30.7%, 0.40) Class II TFGMS markers. The TFGMS markers designed specifically from the 5′ UTR (36.2%) of TF genes revealed more polymorphism than that from the conserved ORF sequences (30.7%). Two hundred and thirty-one (28.8%) of the 801 TFGMS markers in the 193 TF genes showed polymorphism (1–5 alleles and mean PIC of 0.41) among the 52 <em>desi</em> genotypes, whereas 192 (24%) markers in the 147 genes detected polymorphism (1–4 alleles and mean PIC of 0.36) in the 40 <em>kabuli</em> genotypes. The extent of polymorphism detected by the genic markers between <em>desi</em> and <em>kabuli</em> genotypes (31%) was higher than within the <em>desi</em> and <em>kabuli</em> genotypes (26%). The inter-specific polymorphism between cultivated and wild species (33.4%) was higher in contrast to that of intra-specific polymorphism in the <em>desi</em> and <em>kabuli</em> genotypes (28%). A total of 897 alleles were amplified by 258 polymorphic TFGMS markers in 96 genotypes. The number of alleles amplified by the polymorphic TFGMS markers in these genotypes varied from 1 to 8 with an average of 3.5 alleles per marker (Fig. <a reveal-id="DST015F2" data-open="DST015F2" class="link link-reveal link-table xref-fig">2</a>A).</p><p>The efficiency of 157 TFFDMS markers in the 141 TF genes was compared with that of 801 TFGMS markers to detect polymorphism in the same set of 96 genotypes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The TFFDMS markers in contrast to TFGMS markers detected the low level of polymorphism (29.3%, 46 markers) with 1–4 alleles per locus giving an average PIC of 0.39 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The intra- (26%) and inter- (31%) specific polymorphism detected by TFGMS and TFFDMS markers among the 96 genotypes gel-based assay are comparable with that estimated earlier using the normal unlabelled EST-derived genic microsatellite markers (25–30%),<sup><a reveal-id="DST015C8" data-open="DST015C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST015C10" data-open="DST015C10" class="link link-ref link-reveal xref-bibr">10</a></sup> but lower than for unlabelled genomic (40–65%)<sup><a reveal-id="DST015C41  DST015C43" data-open="DST015C41  DST015C43" class="link link-ref link-reveal xref-bibr">41–43</a></sup> microsatellite markers. The higher polymorphic potential of Class I TFGMS and TFFDMS markers (39.4%) when compared with the Class II markers (29.4%) reflected the correspondence between the length of microsatellite repeats and level of polymorphism as observed earlier in many plant species.<sup><a reveal-id="DST015C2" data-open="DST015C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST015C5" data-open="DST015C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST015C44" data-open="DST015C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The microsatellites with longer repeat motifs have a tendency to be more polymorphic due to their high length-dependent replication slippage.<sup><a reveal-id="DST015C20" data-open="DST015C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Besides, high polymorphic potential of TFGMS markers derived from the UTRs than that from the conserved ORF, which are constrained by more purifying selection,<sup><a reveal-id="DST015C3" data-open="DST015C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST015C5" data-open="DST015C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST015C44" data-open="DST015C44" class="link link-ref link-reveal xref-bibr">44</a></sup> suggested the utility of TF genes having such repeat motifs as a source of polymorphic microsatellite markers in chickpea. Higher polymorphic potential of UTR-derived TFGMS markers is expected due to an abundance of non-triplet microsatellite repeat motifs (57.7%), i.e. polymorphic class of di- and tetra-nucleotide motifs in the UTRs that are more prone to replication slippage. Overall, the hypervariable Class I and UTR-derived microsatellite markers designed in this study from the TF genes provided a strategy for the rapid selection of more polymorphic markers from the huge marker database and thus would have greater utility for large-scale validation and efficient genotyping applications in chickpea.</p><p>The level of polymorphism detected by automated fragment analysis using 50 of each fluorescent dye-labelled TFGMS (39.7% polymorphism, PIC 0.53 and 1–8 alleles) and TFFDMS markers (34.3%, 0.43 and 1–6) was much higher than based on metaphor agarose gel with the normal unlabelled markers. However, the trend of polymorphism observed in Classes I and II fluorescent dye-labelled TFGMS and TFFDMS markers and <em>desi</em> and <em>kabuli</em> genotypes using the automated fragment analysis remained intact as obtained with the gel electrophoresis. The fluorescent dye-labelled TFGMS and TFFDMS markers overall detected 1–10 alleles (average 5 alleles) per marker locus (37.8% polymorphic, PIC 0.47), with a total of 196 alleles in 96 genotypes (Fig. <a reveal-id="DST015F2" data-open="DST015F2" class="link link-reveal link-table xref-fig">2</a>B). The level of intra-specific polymorphism estimated above in <em>desi</em> and <em>kabuli</em> is comparable with the level reported previously with labelled EST-derived microsatellite markers (37%).<sup><a reveal-id="DST015C7" data-open="DST015C7" class="link link-ref link-reveal xref-bibr">7</a></sup> With the use of automated fragment analyzer, all the allelic variants of TFGMS and TFFDMS markers could be captured efficiently that enabled to generate precise allele size-based genotyping information for 96 genotypes. The acquired genotypic information thus would enable multiplexing of genic TFGMS and TFFDMS markers and their utility in many large-scale and high-throughput genotyping applications in chickpea. Genetic analysis has been hindered in chickpea due to the lack of sufficient usable and robust sequence-based informative molecular markers and low intra-specific polymorphism specifically among <em>desi</em> and <em>kabuli</em> genotypes. Therefore, a large number of experimentally validated informative genic microsatellite markers with relatively high intra-specific polymorphic potential developed in our study would be useful in chickpea genetics, genomics and breeding. Moreover, the TF genes have utility in controlling diverse cellular processes, including plant growth, differentiation and developmental life cycles and environmental abiotic and biotic stress responses as well as domestication in plant species and legumes.<sup><a reveal-id="DST015C16  DST015C18" data-open="DST015C16  DST015C18" class="link link-ref link-reveal xref-bibr">16–18</a></sup> These designed microsatellite markers, particularly being derived from the different functionally relevant sequence components of TF genes, thus could be useful as functional markers for rapidly establishing marker–trait linkages and identifying genes for many qualitative and quantitative traits of agricultural importance in chickpea.</p><p>For determining the molecular basis of TFGMS and TFFDMS fragment length polymorphism, the size variant amplicons of 24 each TFGMS and TFFDMS markers from the selected genotypes were cloned and sequenced. The high-quality sequence analysis revealed the presence of expected microsatellite repeat-motif sequences with conserved primer binding sites, but variable numbers of repeat units in the size variant amplicons. However, the fragment size variation of sequenced alleles corresponded exactly to the differences in the number of repeat units and the step-wise multiples of the number of repeat units as well (Fig. <a reveal-id="DST015F2" data-open="DST015F2" class="link link-reveal link-table xref-fig">2</a>A). Such fragment length polymorphism due to expansion/contraction of microsatellite repeat units have been observed previously in rice,<sup><a reveal-id="DST015C3" data-open="DST015C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST015C45" data-open="DST015C45" class="link link-ref link-reveal xref-bibr">45</a></sup> maize,<sup><a reveal-id="DST015C46" data-open="DST015C46" class="link link-ref link-reveal xref-bibr">46</a></sup> chickpea,<sup><a reveal-id="DST015C41" data-open="DST015C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DST015C47" data-open="DST015C47" class="link link-ref link-reveal xref-bibr">47</a></sup> sugarcane<sup><a reveal-id="DST015C3" data-open="DST015C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST015C5" data-open="DST015C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and <em>Brassica.</em><sup><a reveal-id="DST015C44" data-open="DST015C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Specifically, in a selected set of 12 Type I TFFDMS markers, the fragment length polymorphism among the genotypes was explained by variation in the number of microsatellite repeat units and corresponded exactly to the expansion/contraction and step-wise multiples of the number of microsatellite repeats within the functional domains of TF genes (Fig. <a reveal-id="DST015F2" data-open="DST015F2" class="link link-reveal link-table xref-fig">2</a>A). The association of genic microsatellite markers with many traits, including diseases like neuronal disorders and cancers in humans, based on the expansion/contraction of the repeated tracts of microsatellite encoding amino acid residues in the functional domain of proteins, has been well demonstrated.<sup><a reveal-id="DST015C1" data-open="DST015C1" class="link link-ref link-reveal xref-bibr">1</a></sup> It would be thus interesting to understand the functional relevance of such designed TFFDMS markers based on their repeat expansion/contraction in the functional domain regions of TF genes for trait association analysis in chickpea.</p>                            <h3 scrollto-destination=5039618 id="5039618" class="section-title" data-legacy-id=s3c>3.3. Physical mapping and genome coverage of TFGMS and TFFDMS markers in chickpea</h3>
<p>One thousand one hundred and forty-three (90.1%) of the 1243 TFGMS and TFFDMS markers developed from the TF genes showing an unique significant hit were physically mapped on the eight chickpea chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>), with average map density of 303.8 kb. Maximum markers were physically mapped on the chromosome 4 (174 markers, 15.2%), followed by the chromosomes 1 (165, 14.4%), 5 (164, 14.3%) and 7 (163, 14.2%) and least on the chromosome 8 (84, 7.3%). The average marker density was maximum in the chromosome 8 (84 markers, 196.2 kb), followed by the chromosomes 3 (148 markers, 270.2 kb) and 4 (174 markers, 282.7 kb) and minimum in the chromosome 2 (91 markers, 402.5 kb). A high-density 1143 TFGMS and TFFDMS marker-based physical map constructed in our study could be used as a reference for the rapid selection of genic microsatellite markers at genome-wide scale/chromosomal level. It would be thus useful for rapid targeted mapping of genes/QTLs for useful traits, mapping the whole genome and sequencing of gene rich regions of the genome and eventually for comparative genome mapping across legumes.</p>                            <h3 scrollto-destination=5039624 id="5039624" class="section-title" data-legacy-id=s3d>3.4. Understanding the population genetic structure and functional molecular diversity in chickpea</h3>
<p>The population genetic structure among the 96 genotypes based on 310 TFGMS and TFFDMS markers and 96 genomic microsatellite markers<sup><a reveal-id="DST015C27" data-open="DST015C27" class="link link-ref link-reveal xref-bibr">27</a></sup> distributed over eight chickpea chromosomes/LGs using STRUCTURE with varying levels of <em>K</em> (<em>K</em> = 2–10) with 20 replications was determined (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). It revealed that, at <em>K</em> = 5, all the genotypes were classified into five distinct sub-populations grouped under three major model-based genetically distinct wild (I), <em>desi</em> (II) and <em>kabuli</em> (III) population (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>B). At this <em>K</em>-value (<em>K</em> = 5), the best replicate gave maximum log-likelihood values with a sharp peak. The genotypes were grouped into five sub-populations at <em>K</em> = 5 with a high-resolution population structure that corresponded well with their expected pedigree relationships and parentage. Based on the population genetic structure analysis, 96 genotypes used in the study were classified into five distinct sub-populations (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>B and Supplementary Table S1): Groups I (four wild genotypes and two <em>kabuli</em> landraces), IIa (23 <em>desi</em> varieties/genetic stocks/breeding lines and five <em>desi</em> landraces), IIb (18 <em>desi</em> landraces and six <em>desi</em> varieties/genetic stocks/breeding lines), IIIa (15 <em>kabuli</em> varieties/genetic stocks/breeding lines) and IIIb (15 <em>kabuli</em> landraces, three <em>kabuli</em> varieties/genetic stocks/breeding lines and five <em>desi</em> landraces). The genetic diversity and phylogenetic relationships established by the genic markers in most of the selected <em>desi</em>, <em>kabuli</em> and wild genotypes were in accordance with the expectation based on pedigree relationships. This is also comparable with earlier documentation using the random<sup><a reveal-id="DST015C48" data-open="DST015C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and microsatellite<sup><a reveal-id="DST015C43" data-open="DST015C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DST015C49" data-open="DST015C49" class="link link-ref link-reveal xref-bibr">49</a></sup> markers and more recently employing a larger set of genome-wide SNP and InDel markers.<sup><a reveal-id="DST015C24" data-open="DST015C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The close evolutionary relationships of domesticated <em>desi</em> and <em>kabuli</em> with its wild progenitor <em>C. reticulatum</em> have already been reported by different morphological,<sup><a reveal-id="DST015C50" data-open="DST015C50" class="link link-ref link-reveal xref-bibr">50</a></sup> cytological,<sup><a reveal-id="DST015C51" data-open="DST015C51" class="link link-ref link-reveal xref-bibr">51</a></sup> biochemical<sup><a reveal-id="DST015C52" data-open="DST015C52" class="link link-ref link-reveal xref-bibr">52</a></sup> and molecular studies.<sup><a reveal-id="DST015C41" data-open="DST015C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DST015C43" data-open="DST015C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DST015C49  DST015C56" data-open="DST015C49  DST015C56" class="link link-ref link-reveal xref-bibr">49–56</a></sup> Therefore, the microsatellite markers developed from the TF genes might assay more relevant transcribed sequence components of the chickpea genome for establishing distinctness and assaying functional diversity among the genotypes and thus, would be useful in chickpea breeding, including variety identification and understanding the diversity pattern.</p><p>Population structure is also an indicator of genetic differentiation among population groups comprising of many genotypes. The molecular genetic variation among and within the five sub-populations (three <em>desi</em>, <em>kabuli</em> and wild populations) based on 310 polymorphic TFGMS and TFFDMS markers and 96 genomic microsatellite markers revealed a wider level of quantitative genetic differentiation (<em>F</em><sub>ST</sub> varied from 0.03 to 0.82 with an average of 0.53) among these population groups. This is higher than the previously detected level with the EST-derived (0.03–0.49<sup><a reveal-id="DST015C10" data-open="DST015C10" class="link link-ref link-reveal xref-bibr">10</a></sup>) and genomic (0.37–0.80<sup><a reveal-id="DST015C41" data-open="DST015C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and 0.32–0.80<sup><a reveal-id="DST015C42" data-open="DST015C42" class="link link-ref link-reveal xref-bibr">42</a></sup>) microsatellite markers. Among three population groups, the divergence was maximum between <em>kabuli</em> and wild (<em>F</em><sub>ST</sub> = 0.61) and minimum between <em>desi</em> and <em>kabuli</em> (0.29). The proportion of <em>F</em><sub>ST</sub> and thus diversity between population groups (53%) were higher when compared with that estimated within the population (36%), which is expected in a self-pollinated crop species like chickpea. Higher population genetic differentiation within <em>desi</em> (<em>F</em><sub>ST</sub> 0.39), in contrast to <em>kabuli</em>, (<em>F</em><sub>ST</sub> 0.30) agrees well with the earlier observations.<sup><a reveal-id="DST015C43" data-open="DST015C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DST015C49" data-open="DST015C49" class="link link-ref link-reveal xref-bibr">49</a></sup> The TFGMS and TFFDMS markers developed from the TF genes assaying variation largely in the expressed component of the genome might be directly correlated with phenotypic variation and thus would be of much relevance in the selection of desirable plant types in breeding programme. Henceforth, a wider genetic base and functional molecular diversity detected by the genic markers among <em>desi</em> and <em>kabuli</em> genotypes would be of significance in the perspective of varietal improvement in chickpea.</p><p>All the 96 genotypes clearly belonged to a single population in which ∼91.7% of their inferred ancestry was derived from one of the model-based population and remaining ∼8.3% contained admixed ancestry (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>B). The existence of admix ancestry among five sub-populations (three population groups) reflected their complex breeding history involving inter-crossing and introgression among genotypes coupled with different strong adaptive selection pressure during domestication. This is expected because most of the <em>desi</em> and <em>kabuli</em> genotypes analysed in this study were developed through multiple-crosses, mostly three-way crosses among diverse selected germplasm accessions by the joint effort of International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) and National Agricultural Research systems (NARS) for widening the genetic base and improving the stress tolerance. For instance, ICC12968/ICCV2, a short-duration <em>kabuli</em> chickpea variety showing an admixture of <em>desi</em> population groups has been developed from a multiple inter-cross involving five parental <em>desi</em> and <em>kabuli</em> genotypes [(K850 × GW-5/7) × P458] × (L550 × Guamuchil) for <em>Fusarium</em> wilt resistance and adaptation to short-duration environment of Southern and Central India.</p><p>Maximum admixtures obtained between <em>desi</em> and <em>kabuli</em> population groups (5%) supported well with the commonly accepted presumption that the large seed-size domesticated <em>kabuli</em> types originated from the small seed-size domesticated <em>desi</em> chickpea.<sup><a reveal-id="DST015C53" data-open="DST015C53" class="link link-ref link-reveal xref-bibr">53</a></sup> About 3% admixed ancestry between wild <em>C. reticulatum</em> and <em>desi</em> population groups and their evolutionary closeness is thus expected due to the resemblance of seed morphology between <em>desi</em> and wild progenitor (<em>C. reticulatum</em>) of domesticated chickpea at the archaeological sites of South Eastern Turkey nearly ∼10 000 years ago.<sup><a reveal-id="DST015C54  DST015C56" data-open="DST015C54  DST015C56" class="link link-ref link-reveal xref-bibr">54–56</a></sup> An admixed ancestry of ∼0.3% between <em>kabuli</em> and wild population groups indicated that domesticated <em>kabuli</em> possibly originated directly from the spontaneous mutants of wild <em>C. reticulatum</em> in ancient South Eastern Turkey with a separate independent evolutionary pathway as suggested by the earlier archeobotanical,<sup><a reveal-id="DST015C57" data-open="DST015C57" class="link link-ref link-reveal xref-bibr">57</a></sup> mutant<sup><a reveal-id="DST015C53" data-open="DST015C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and microsatellite marker-based<sup><a reveal-id="DST015C58" data-open="DST015C58" class="link link-ref link-reveal xref-bibr">58</a></sup> evolutionary studies involving wild and <em>kabuli</em> genotypes. A higher admixture between the <em>desi</em> and wild population when compared with that between the <em>desi</em> and <em>kabuli</em> population groups might be attributed to maximum adaptation-based selection pressure during the multiple domestication of pre-adapted <em>desi</em> and wild <em>C. reticulatum</em> followed by the punctuation of a series of four sequential evolutionary bottlenecks that have narrowed down the genetic base in cultivated chickpea.<sup><a reveal-id="DST015C53" data-open="DST015C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DST015C54" data-open="DST015C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DST015C59" data-open="DST015C59" class="link link-ref link-reveal xref-bibr">59</a>,<a reveal-id="DST015C60" data-open="DST015C60" class="link link-ref link-reveal xref-bibr">60</a></sup> However, complete understanding about the genetic structure, domestication and evolutionary history of <em>desi</em>, <em>kabuli</em> and wild chickpea populations would require analysis of a greater number of contrasting genotypes belonging to each of these three population groups using high-throughput marker genotyping information.</p>                            <h3 scrollto-destination=5039639 id="5039639" class="section-title" data-legacy-id=s3e>3.5. Genetic association potential of TFGMS and TFFDMS markers</h3>
<p>The microsatellite marker-based trait association mapping has proven to be an effective approach for dissecting the complex quantitative traits in many plant species, including legumes.<sup><a reveal-id="DST015C61" data-open="DST015C61" class="link link-ref link-reveal xref-bibr">61</a>–<a reveal-id="DST015C64" data-open="DST015C64" class="link link-ref link-reveal xref-bibr">64</a></sup> Genetic association analysis was performed by correlating the genotyping data of 310 informative TFGMS and TFFDMS markers and 96 genomic microsatellite markers with the phenotypic information of three major seed and pod traits on the 96 association mapping panel. Higher phenotypic variation was observed in the case of 100-seed weight (5–70 g), pod number per plant (20–130) and seed number per plant (32–198) based on the multi-location replicated field data. The phenotypically and genotypically (based on population structure) diverse association mapping panels selected in our study is a structured population, thus having significance for seed and pod trait-specific association analysis in chickpea. The combined results of GLM and MLM in TASSEL identified 23 TFFDMS and 21 TFGMS markers in the 34 TF genes, showing a significant association with 100-seed weight, pod number per plant and seed number per plant at <em>P</em>-value of ≤0.05 and <em>R</em><sup>2</sup> from 0.18 to 0.37 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). However, none of the 96 genomic microsatellite markers distributed over eight chickpea LGs/chromosomes (used as controls) exhibited any significant seed and pod trait-specific association. It reflects the robustness, including reliability and utility of genic microsatellite markers, identified in the TF genes for trait association analysis in chickpea. Ten TFFDMS and nine TFGMS markers in the 12 TF genes revealed association with all the three seed and pod traits, whereas 13 TFFDMS and 12 TFGMS markers in the 22 genes had association with a single trait only. The significant association of multiple microsatellite markers in more than one TF genes distributed over eight chickpea chromosomes with multiple seed and pod traits provides clues for the complex quantitative genetic inheritance pattern of traits<sup><a reveal-id="DST015C65" data-open="DST015C65" class="link link-ref link-reveal xref-bibr">65</a></sup> under study. In spite of low polymorphic potential, particularly 23 TFFDMS markers in the 22 TF genes showed higher seed and pod trait-specific association potential (significant <em>P</em> ≤ 0.001 and <em>R</em><sup>2</sup> 0.23–0.38) in chickpea genotypes when compared with that obtained using the 21 TFGMS markers (<em>P</em> 0.01–0.05 and <em>R</em><sup>2</sup> 0.18–0.31) in the 12 genes. Based on these observations, it is possible that the varied trait association potential estimated for TFGMS and TFFDMS markers are due to their presence in different TF genes. Henceforth, for better understanding and comparing the potential of these two genic marker types in trait association analysis in chickpea, the five seed and pod trait-associated TF genes carrying both polymorphic nine TFGMS and five TFFDMS markers were analysed in detail (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). The above similar trend depicting the higher efficiency of four TFFDMS markers in contrast to eight TFGMS markers in four TF genes for the seed and pod trait association in chickpea genotypes was observed (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). Interestingly, the remaining one TFGMS marker (<em>P</em> &lt; 0.0024 and <em>R</em><sup>2</sup> 0.31) in domain of unknown function (DUF)1635 TF gene showed higher seed and pod trait association potential in contrast to one TFFDMS marker (<em>P</em> &lt; 0.047 and <em>R</em><sup>2</sup> 0.23) in the functional domain of similar gene (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). This ascertains a strategy to select the target sequence components of genes that are functionally more relevant for developing genic microsatellite markers for rapid trait association analysis in chickpea. Several recent reports have documented the diverse trait association potential of functional marker loci developed from different coding and UTR sequence components of a gene like starch biosynthesis<sup><a reveal-id="DST015C66" data-open="DST015C66" class="link link-ref link-reveal xref-bibr">66</a></sup> and grain size (<em>GS3</em><sup><a reveal-id="DST015C67" data-open="DST015C67" class="link link-ref link-reveal xref-bibr">67</a></sup> and <em>GS5</em><sup><a reveal-id="DST015C61" data-open="DST015C61" class="link link-ref link-reveal xref-bibr">61</a></sup>) genes in rice. Therefore, the selections of suitable marker loci from the different sequence components of target genes are always crucial for evaluating the definite potential of genes for trait association mapping. </p>                        <a id="5039645" scrollto-destination="5039645"></a>
<div content-id="DST015TB1" class="table-modal"><div class="table-wrap"><div id="DST015TB1" class="table-wrap-title" data-id="DST015TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Functional significance of TFGMS and TFFDMS markers designed from the five TF genes showed association with seed and pod traits in chickpea</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TFs </th><th>Functional domains </th><th>TFGMS markers (NCBI PUIDs) </th><th>TFGMS repeat motifs </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) and differentially expressed (DE)/not differentially expressed (NDE) during seed development </th><th>TFFDMS markers (NCBI PUIDs) </th><th>TFFDMS repeat motifs </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) and differentially expressed (DE)/not differentially expressed (NDE) during seed development </th><th>SNP loci discovered </th><th>Sequenced gene amplicon size (bp) </th><th>Frequency (SNPs/bp) </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) </th><th>Associated seed and pod traits </th><th>Genetic position (cM) on the LGs </th></tr></thead><tbody><tr><td>Teosinte branched 1/cycloidea/proliferating cell factors (TCP) </td><td>TCP </td><td>TFGMS5<br /> (16 584 978) </td><td>(TCTCAT)<sub>4</sub> </td><td>&lt;0.046<br /> 0.19<br /> NDE </td><td>TFFDMS15-Type-I<br /> (16 586 183) </td><td>(TCT)<sub>5</sub> </td><td>&lt;0.0034<br /> 0.31<br /> NDE </td><td>A/T, G/A,<br /> T/A, C/A,<br /> G/T, T/C </td><td>1850 </td><td>1/308.3 </td><td>&lt;0.0026<br /> 0.40 </td><td>Pod number per plant </td><td>LG1<br /> 55.4 </td></tr><tr><td>Squamosa-promoter-binding protein (SBP) </td><td>SBP </td><td>TFGMS21<br /> (16 584 994) </td><td>(CTC)<sub>4</sub> </td><td>&lt;0.014<br /> 0.25<br /> NDE </td><td>TFFDMS21-Type-I<sup>a</sup><br /> (16 586 189) </td><td>(CAC)<sub>8</sub> </td><td>&lt;0.0019<br /> 0.38<br /> NDE </td><td>C/T, T/A,<br /> C/T, G/C,<br /> C/T, A/T,<br /> G/A, G/A,<br /> C/T, A/C,<br /> G/A </td><td>3000 </td><td>1/272.7 </td><td>&lt;0.0012<br /> 0.45 </td><td>Seed size/100-seed weight </td><td>LG1<br /> 74.5 </td></tr><tr><td rowspan="3">No apical meristem-arabidopsis transcription activation factor-cup shaped cotyledon (NAC) </td><td rowspan="3">NAM </td><td>TFGMS12<br /> (16 584 985) </td><td>(AATCCA)<sub>5</sub> </td><td>&lt;0.037<br /> 0.22<br /> NDE </td><td rowspan="3">TFFDMS18-Type-I<sup>a</sup><br /> 16 586 186 </td><td rowspan="3">(GAT)<sub>6</sub> </td><td rowspan="3">&lt;0.0059<br /> 0.33<br /> NDE </td><td rowspan="3">A/G, T/C,<br /> G/C, T/A,<br /> G/A </td><td rowspan="3">1400 </td><td rowspan="3">1/280 </td><td rowspan="3">&lt;0.0054<br /> 0.38 </td><td rowspan="3">Seed size/100-seed weight<br /> Pod number per plant<br /> Seed number per plant </td><td rowspan="3">LG1<br /> 73.9 </td></tr><tr><td>TFGMS17<br /> (16 584 990) </td><td>(CAA)<sub>5</sub> </td><td>&lt;0.032<br /> 0.25<br /> NDE </td></tr><tr><td>TFGMS3<br /> (16 584 976) </td><td>(TAA)<sub>5</sub> </td><td>&lt;0.049<br /> 0.18<br /> NDE </td></tr><tr><td rowspan="3">Domain of unknown function (DUF1635) </td><td rowspan="3">DUF1635 </td><td>TFGMS18<sup>a</sup><br /> (16 584 991) </td><td>(CTA)<sub>6</sub> </td><td>&lt;0.0024<br /> 0.31<br /> DE </td><td rowspan="3">TFFDMS16-Type-I<br /> (16 586 184) </td><td rowspan="3">(CAA)<sub>6</sub> </td><td rowspan="3">&lt;0.047<br /> 0.23<br /> NDE </td><td rowspan="3">A/G, C/T,<br /> C/G, A/T,<br /> C/T, T/G </td><td rowspan="3">2000 </td><td rowspan="3">1/333.3 </td><td rowspan="3">&lt;0.0046<br /> 0.43 </td><td rowspan="3">Seed size/100-seed weight </td><td rowspan="3">LG1<br /> 73.5 </td></tr><tr><td>TFGMS1<br /> (16 584 974) </td><td>(GAA)<sub>6</sub> </td><td>&lt;0.043<br /> 0.18<br /> NDE </td></tr><tr><td>TFGMS4<br /> (16 584 977) </td><td>(AGA)<sub>8</sub> </td><td>&lt;0.041<br /> 0.19<br /> NDE </td></tr><tr><td>Domain of unknown function (DUF702) </td><td>DUF702 </td><td>TFGMS13<br /> (16 584 986) </td><td>(ACA)<sub>4</sub> </td><td>&lt;0.028<br /> 0.23<br /> NDE </td><td>TFFDMS20-Type-I<sup>a</sup><br /> (16 586 188) </td><td>(ATA)<sub>5</sub> </td><td>&lt;0.0062<br /> 0.35<br /> NDE </td><td>C/T, G/A,<br /> A/C, G/C </td><td>1100 </td><td>1/275 </td><td>&lt;0.0058<br /> 0.41 </td><td>Seed size/100-seed weight<br /> Seed number per plant </td><td>LG1<br /> 75.2 </td></tr></tbody></table></div><div content-id="DST015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>TFs </th><th>Functional domains </th><th>TFGMS markers (NCBI PUIDs) </th><th>TFGMS repeat motifs </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) and differentially expressed (DE)/not differentially expressed (NDE) during seed development </th><th>TFFDMS markers (NCBI PUIDs) </th><th>TFFDMS repeat motifs </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) and differentially expressed (DE)/not differentially expressed (NDE) during seed development </th><th>SNP loci discovered </th><th>Sequenced gene amplicon size (bp) </th><th>Frequency (SNPs/bp) </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) </th><th>Associated seed and pod traits </th><th>Genetic position (cM) on the LGs </th></tr></thead><tbody><tr><td>Teosinte branched 1/cycloidea/proliferating cell factors (TCP) </td><td>TCP </td><td>TFGMS5<br /> (16 584 978) </td><td>(TCTCAT)<sub>4</sub> </td><td>&lt;0.046<br /> 0.19<br /> NDE </td><td>TFFDMS15-Type-I<br /> (16 586 183) </td><td>(TCT)<sub>5</sub> </td><td>&lt;0.0034<br /> 0.31<br /> NDE </td><td>A/T, G/A,<br /> T/A, C/A,<br /> G/T, T/C </td><td>1850 </td><td>1/308.3 </td><td>&lt;0.0026<br /> 0.40 </td><td>Pod number per plant </td><td>LG1<br /> 55.4 </td></tr><tr><td>Squamosa-promoter-binding protein (SBP) </td><td>SBP </td><td>TFGMS21<br /> (16 584 994) </td><td>(CTC)<sub>4</sub> </td><td>&lt;0.014<br /> 0.25<br /> NDE </td><td>TFFDMS21-Type-I<sup>a</sup><br /> (16 586 189) </td><td>(CAC)<sub>8</sub> </td><td>&lt;0.0019<br /> 0.38<br /> NDE </td><td>C/T, T/A,<br /> C/T, G/C,<br /> C/T, A/T,<br /> G/A, G/A,<br /> C/T, A/C,<br /> G/A </td><td>3000 </td><td>1/272.7 </td><td>&lt;0.0012<br /> 0.45 </td><td>Seed size/100-seed weight </td><td>LG1<br /> 74.5 </td></tr><tr><td rowspan="3">No apical meristem-arabidopsis transcription activation factor-cup shaped cotyledon (NAC) </td><td rowspan="3">NAM </td><td>TFGMS12<br /> (16 584 985) </td><td>(AATCCA)<sub>5</sub> </td><td>&lt;0.037<br /> 0.22<br /> NDE </td><td rowspan="3">TFFDMS18-Type-I<sup>a</sup><br /> 16 586 186 </td><td rowspan="3">(GAT)<sub>6</sub> </td><td rowspan="3">&lt;0.0059<br /> 0.33<br /> NDE </td><td rowspan="3">A/G, T/C,<br /> G/C, T/A,<br /> G/A </td><td rowspan="3">1400 </td><td rowspan="3">1/280 </td><td rowspan="3">&lt;0.0054<br /> 0.38 </td><td rowspan="3">Seed size/100-seed weight<br /> Pod number per plant<br /> Seed number per plant </td><td rowspan="3">LG1<br /> 73.9 </td></tr><tr><td>TFGMS17<br /> (16 584 990) </td><td>(CAA)<sub>5</sub> </td><td>&lt;0.032<br /> 0.25<br /> NDE </td></tr><tr><td>TFGMS3<br /> (16 584 976) </td><td>(TAA)<sub>5</sub> </td><td>&lt;0.049<br /> 0.18<br /> NDE </td></tr><tr><td rowspan="3">Domain of unknown function (DUF1635) </td><td rowspan="3">DUF1635 </td><td>TFGMS18<sup>a</sup><br /> (16 584 991) </td><td>(CTA)<sub>6</sub> </td><td>&lt;0.0024<br /> 0.31<br /> DE </td><td rowspan="3">TFFDMS16-Type-I<br /> (16 586 184) </td><td rowspan="3">(CAA)<sub>6</sub> </td><td rowspan="3">&lt;0.047<br /> 0.23<br /> NDE </td><td rowspan="3">A/G, C/T,<br /> C/G, A/T,<br /> C/T, T/G </td><td rowspan="3">2000 </td><td rowspan="3">1/333.3 </td><td rowspan="3">&lt;0.0046<br /> 0.43 </td><td rowspan="3">Seed size/100-seed weight </td><td rowspan="3">LG1<br /> 73.5 </td></tr><tr><td>TFGMS1<br /> (16 584 974) </td><td>(GAA)<sub>6</sub> </td><td>&lt;0.043<br /> 0.18<br /> NDE </td></tr><tr><td>TFGMS4<br /> (16 584 977) </td><td>(AGA)<sub>8</sub> </td><td>&lt;0.041<br /> 0.19<br /> NDE </td></tr><tr><td>Domain of unknown function (DUF702) </td><td>DUF702 </td><td>TFGMS13<br /> (16 584 986) </td><td>(ACA)<sub>4</sub> </td><td>&lt;0.028<br /> 0.23<br /> NDE </td><td>TFFDMS20-Type-I<sup>a</sup><br /> (16 586 188) </td><td>(ATA)<sub>5</sub> </td><td>&lt;0.0062<br /> 0.35<br /> NDE </td><td>C/T, G/A,<br /> A/C, G/C </td><td>1100 </td><td>1/275 </td><td>&lt;0.0058<br /> 0.41 </td><td>Seed size/100-seed weight<br /> Seed number per plant </td><td>LG1<br /> 75.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Validated through traditional bi-parental genetic mapping.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5039645" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST015TB1" class="table-wrap-title" data-id="DST015TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Functional significance of TFGMS and TFFDMS markers designed from the five TF genes showed association with seed and pod traits in chickpea</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TFs </th><th>Functional domains </th><th>TFGMS markers (NCBI PUIDs) </th><th>TFGMS repeat motifs </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) and differentially expressed (DE)/not differentially expressed (NDE) during seed development </th><th>TFFDMS markers (NCBI PUIDs) </th><th>TFFDMS repeat motifs </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) and differentially expressed (DE)/not differentially expressed (NDE) during seed development </th><th>SNP loci discovered </th><th>Sequenced gene amplicon size (bp) </th><th>Frequency (SNPs/bp) </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) </th><th>Associated seed and pod traits </th><th>Genetic position (cM) on the LGs </th></tr></thead><tbody><tr><td>Teosinte branched 1/cycloidea/proliferating cell factors (TCP) </td><td>TCP </td><td>TFGMS5<br /> (16 584 978) </td><td>(TCTCAT)<sub>4</sub> </td><td>&lt;0.046<br /> 0.19<br /> NDE </td><td>TFFDMS15-Type-I<br /> (16 586 183) </td><td>(TCT)<sub>5</sub> </td><td>&lt;0.0034<br /> 0.31<br /> NDE </td><td>A/T, G/A,<br /> T/A, C/A,<br /> G/T, T/C </td><td>1850 </td><td>1/308.3 </td><td>&lt;0.0026<br /> 0.40 </td><td>Pod number per plant </td><td>LG1<br /> 55.4 </td></tr><tr><td>Squamosa-promoter-binding protein (SBP) </td><td>SBP </td><td>TFGMS21<br /> (16 584 994) </td><td>(CTC)<sub>4</sub> </td><td>&lt;0.014<br /> 0.25<br /> NDE </td><td>TFFDMS21-Type-I<sup>a</sup><br /> (16 586 189) </td><td>(CAC)<sub>8</sub> </td><td>&lt;0.0019<br /> 0.38<br /> NDE </td><td>C/T, T/A,<br /> C/T, G/C,<br /> C/T, A/T,<br /> G/A, G/A,<br /> C/T, A/C,<br /> G/A </td><td>3000 </td><td>1/272.7 </td><td>&lt;0.0012<br /> 0.45 </td><td>Seed size/100-seed weight </td><td>LG1<br /> 74.5 </td></tr><tr><td rowspan="3">No apical meristem-arabidopsis transcription activation factor-cup shaped cotyledon (NAC) </td><td rowspan="3">NAM </td><td>TFGMS12<br /> (16 584 985) </td><td>(AATCCA)<sub>5</sub> </td><td>&lt;0.037<br /> 0.22<br /> NDE </td><td rowspan="3">TFFDMS18-Type-I<sup>a</sup><br /> 16 586 186 </td><td rowspan="3">(GAT)<sub>6</sub> </td><td rowspan="3">&lt;0.0059<br /> 0.33<br /> NDE </td><td rowspan="3">A/G, T/C,<br /> G/C, T/A,<br /> G/A </td><td rowspan="3">1400 </td><td rowspan="3">1/280 </td><td rowspan="3">&lt;0.0054<br /> 0.38 </td><td rowspan="3">Seed size/100-seed weight<br /> Pod number per plant<br /> Seed number per plant </td><td rowspan="3">LG1<br /> 73.9 </td></tr><tr><td>TFGMS17<br /> (16 584 990) </td><td>(CAA)<sub>5</sub> </td><td>&lt;0.032<br /> 0.25<br /> NDE </td></tr><tr><td>TFGMS3<br /> (16 584 976) </td><td>(TAA)<sub>5</sub> </td><td>&lt;0.049<br /> 0.18<br /> NDE </td></tr><tr><td rowspan="3">Domain of unknown function (DUF1635) </td><td rowspan="3">DUF1635 </td><td>TFGMS18<sup>a</sup><br /> (16 584 991) </td><td>(CTA)<sub>6</sub> </td><td>&lt;0.0024<br /> 0.31<br /> DE </td><td rowspan="3">TFFDMS16-Type-I<br /> (16 586 184) </td><td rowspan="3">(CAA)<sub>6</sub> </td><td rowspan="3">&lt;0.047<br /> 0.23<br /> NDE </td><td rowspan="3">A/G, C/T,<br /> C/G, A/T,<br /> C/T, T/G </td><td rowspan="3">2000 </td><td rowspan="3">1/333.3 </td><td rowspan="3">&lt;0.0046<br /> 0.43 </td><td rowspan="3">Seed size/100-seed weight </td><td rowspan="3">LG1<br /> 73.5 </td></tr><tr><td>TFGMS1<br /> (16 584 974) </td><td>(GAA)<sub>6</sub> </td><td>&lt;0.043<br /> 0.18<br /> NDE </td></tr><tr><td>TFGMS4<br /> (16 584 977) </td><td>(AGA)<sub>8</sub> </td><td>&lt;0.041<br /> 0.19<br /> NDE </td></tr><tr><td>Domain of unknown function (DUF702) </td><td>DUF702 </td><td>TFGMS13<br /> (16 584 986) </td><td>(ACA)<sub>4</sub> </td><td>&lt;0.028<br /> 0.23<br /> NDE </td><td>TFFDMS20-Type-I<sup>a</sup><br /> (16 586 188) </td><td>(ATA)<sub>5</sub> </td><td>&lt;0.0062<br /> 0.35<br /> NDE </td><td>C/T, G/A,<br /> A/C, G/C </td><td>1100 </td><td>1/275 </td><td>&lt;0.0058<br /> 0.41 </td><td>Seed size/100-seed weight<br /> Seed number per plant </td><td>LG1<br /> 75.2 </td></tr></tbody></table></div><div content-id="DST015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>TFs </th><th>Functional domains </th><th>TFGMS markers (NCBI PUIDs) </th><th>TFGMS repeat motifs </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) and differentially expressed (DE)/not differentially expressed (NDE) during seed development </th><th>TFFDMS markers (NCBI PUIDs) </th><th>TFFDMS repeat motifs </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) and differentially expressed (DE)/not differentially expressed (NDE) during seed development </th><th>SNP loci discovered </th><th>Sequenced gene amplicon size (bp) </th><th>Frequency (SNPs/bp) </th><th>Association (<em>P</em> and <em>R</em><sup>2</sup> value) </th><th>Associated seed and pod traits </th><th>Genetic position (cM) on the LGs </th></tr></thead><tbody><tr><td>Teosinte branched 1/cycloidea/proliferating cell factors (TCP) </td><td>TCP </td><td>TFGMS5<br /> (16 584 978) </td><td>(TCTCAT)<sub>4</sub> </td><td>&lt;0.046<br /> 0.19<br /> NDE </td><td>TFFDMS15-Type-I<br /> (16 586 183) </td><td>(TCT)<sub>5</sub> </td><td>&lt;0.0034<br /> 0.31<br /> NDE </td><td>A/T, G/A,<br /> T/A, C/A,<br /> G/T, T/C </td><td>1850 </td><td>1/308.3 </td><td>&lt;0.0026<br /> 0.40 </td><td>Pod number per plant </td><td>LG1<br /> 55.4 </td></tr><tr><td>Squamosa-promoter-binding protein (SBP) </td><td>SBP </td><td>TFGMS21<br /> (16 584 994) </td><td>(CTC)<sub>4</sub> </td><td>&lt;0.014<br /> 0.25<br /> NDE </td><td>TFFDMS21-Type-I<sup>a</sup><br /> (16 586 189) </td><td>(CAC)<sub>8</sub> </td><td>&lt;0.0019<br /> 0.38<br /> NDE </td><td>C/T, T/A,<br /> C/T, G/C,<br /> C/T, A/T,<br /> G/A, G/A,<br /> C/T, A/C,<br /> G/A </td><td>3000 </td><td>1/272.7 </td><td>&lt;0.0012<br /> 0.45 </td><td>Seed size/100-seed weight </td><td>LG1<br /> 74.5 </td></tr><tr><td rowspan="3">No apical meristem-arabidopsis transcription activation factor-cup shaped cotyledon (NAC) </td><td rowspan="3">NAM </td><td>TFGMS12<br /> (16 584 985) </td><td>(AATCCA)<sub>5</sub> </td><td>&lt;0.037<br /> 0.22<br /> NDE </td><td rowspan="3">TFFDMS18-Type-I<sup>a</sup><br /> 16 586 186 </td><td rowspan="3">(GAT)<sub>6</sub> </td><td rowspan="3">&lt;0.0059<br /> 0.33<br /> NDE </td><td rowspan="3">A/G, T/C,<br /> G/C, T/A,<br /> G/A </td><td rowspan="3">1400 </td><td rowspan="3">1/280 </td><td rowspan="3">&lt;0.0054<br /> 0.38 </td><td rowspan="3">Seed size/100-seed weight<br /> Pod number per plant<br /> Seed number per plant </td><td rowspan="3">LG1<br /> 73.9 </td></tr><tr><td>TFGMS17<br /> (16 584 990) </td><td>(CAA)<sub>5</sub> </td><td>&lt;0.032<br /> 0.25<br /> NDE </td></tr><tr><td>TFGMS3<br /> (16 584 976) </td><td>(TAA)<sub>5</sub> </td><td>&lt;0.049<br /> 0.18<br /> NDE </td></tr><tr><td rowspan="3">Domain of unknown function (DUF1635) </td><td rowspan="3">DUF1635 </td><td>TFGMS18<sup>a</sup><br /> (16 584 991) </td><td>(CTA)<sub>6</sub> </td><td>&lt;0.0024<br /> 0.31<br /> DE </td><td rowspan="3">TFFDMS16-Type-I<br /> (16 586 184) </td><td rowspan="3">(CAA)<sub>6</sub> </td><td rowspan="3">&lt;0.047<br /> 0.23<br /> NDE </td><td rowspan="3">A/G, C/T,<br /> C/G, A/T,<br /> C/T, T/G </td><td rowspan="3">2000 </td><td rowspan="3">1/333.3 </td><td rowspan="3">&lt;0.0046<br /> 0.43 </td><td rowspan="3">Seed size/100-seed weight </td><td rowspan="3">LG1<br /> 73.5 </td></tr><tr><td>TFGMS1<br /> (16 584 974) </td><td>(GAA)<sub>6</sub> </td><td>&lt;0.043<br /> 0.18<br /> NDE </td></tr><tr><td>TFGMS4<br /> (16 584 977) </td><td>(AGA)<sub>8</sub> </td><td>&lt;0.041<br /> 0.19<br /> NDE </td></tr><tr><td>Domain of unknown function (DUF702) </td><td>DUF702 </td><td>TFGMS13<br /> (16 584 986) </td><td>(ACA)<sub>4</sub> </td><td>&lt;0.028<br /> 0.23<br /> NDE </td><td>TFFDMS20-Type-I<sup>a</sup><br /> (16 586 188) </td><td>(ATA)<sub>5</sub> </td><td>&lt;0.0062<br /> 0.35<br /> NDE </td><td>C/T, G/A,<br /> A/C, G/C </td><td>1100 </td><td>1/275 </td><td>&lt;0.0058<br /> 0.41 </td><td>Seed size/100-seed weight<br /> Seed number per plant </td><td>LG1<br /> 75.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Validated through traditional bi-parental genetic mapping.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5039645" target="_blank">View Large</a></div></div><p>The detail structural and functional annotation of five seed and pod trait-associated TFs were performed to identify their downstream structural gene targets in chickpea. Using the AGRIS regulatory network database (AtRegNet, <a class="link link-uri" href="http://arabidopsis.med.ohio-state.edu">http://arabidopsis.med.ohio-state.edu</a>), the direct interactions of three seed and pod trait-associated chickpea TF genes [Teosinte branched 1/cycloidea/proliferating cell factors (TCP) squamosa-promoter-binding protein (SBP) and DUF702] with MCM1-agamous-deficiens-serum response factor (MADS) TFs, including SEPALLATA3 and AGL15, that are conserved in <em>Arabidopsis</em> were clearly evident. Among all MADS TF genes, the role of AGL15 has already been experimentally validated in controlling the downstream targets (<em>LEAFY COTYLEDON2</em>, <em>FUSCA3</em> and <em>ABA INSENSITIVE3</em>) encoding B3 domain TFs, which are the key regulators of embryogenesis during seed development in <em>Arabidopsis</em>.<sup><a reveal-id="DST015C18" data-open="DST015C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DST015C68" data-open="DST015C68" class="link link-ref link-reveal xref-bibr">68</a>,<a reveal-id="DST015C69" data-open="DST015C69" class="link link-ref link-reveal xref-bibr">69</a></sup></p>                            <h3 scrollto-destination=5039650 id="5039650" class="section-title" data-legacy-id=s3f>3.6. Intra-specific functional transcript map of chickpea</h3>
<p>For the construction of intra-specific genetic linkage map (transcript map), 74 TFMS and 22 FDMS markers one in each 96 TF genes and 32 previously reported genomic microsatellite markers<sup><a reveal-id="DST015C27" data-open="DST015C27" class="link link-ref link-reveal xref-bibr">27</a></sup> showing parental polymorphism between <em>desi</em> (ICCX-810800) and <em>kabuli</em> (ICC20268) were genotyped among 94 individuals of a F<sub>3</sub> mapping population. The linkage analysis of marker genotyping data mapped a total of 96 TFMS and FDMS marker loci carrying TF genes and 32 genomic microsatellite markers onto the eight LGs (CaLG1 to CaLG8, haploid chromosome numbers) of chickpea (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). The LGs were designated and numbered accordingly with Arabic numerals (LG1 to LG8) based on the common parental polymorphic 32 genomic microsatellite marker positions and groupings that shared between corresponding eight LGs as reported by previous studies.<sup><a reveal-id="DST015C6  DST015C8" data-open="DST015C6  DST015C8" class="link link-ref link-reveal xref-bibr">6–8</a>,<a reveal-id="DST015C27" data-open="DST015C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DST015C42" data-open="DST015C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DST015C70" data-open="DST015C70" class="link link-ref link-reveal xref-bibr">70</a></sup> The transcript map constructed for eight LGs spanned a total map length of 782.8 cM with an average inter-marker distance of 6.12 cM (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Based on genetic distance, the chromosome 8 showed shortest LG spanning of 78.7 cM, while the chromosome 1 had the longest LG with map length of 121.7 cM. The average inter-marker distance was maximum for chromosome 7 (6.75 cM), followed by chromosome 3 (6.67 cM), and minimum for chromosome 1 (5.07 cM). The average inter-marker distance estimated for the transcript map was comparable with that reported for previously constructed (1.77–8.01 cM) intra- and inter-specific genetic linkage maps of chickpea using a large number of genic and genomic microsatellite markers and a smaller set of SNP markers.<sup><a reveal-id="DST015C6  DST015C8" data-open="DST015C6  DST015C8" class="link link-ref link-reveal xref-bibr">6–8</a>,<a reveal-id="DST015C42" data-open="DST015C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DST015C70" data-open="DST015C70" class="link link-ref link-reveal xref-bibr">70</a></sup> The TFMS and FDMS markers carrying TF genes mapped genetically on the intra-specific transcript map would expedite large-scale genotyping applications, including the construction of high-density comparative genetic linkage (transcript) map and identification/mapping of genes/QTLs for important agronomic traits in chickpea.</p>                            <h3 scrollto-destination=5039655 id="5039655" class="section-title" data-legacy-id=s3g>3.7. Validation of trait-associated TF genes through genetic mapping</h3>
<p>The traditional bi-parental genetic mapping in a 96 F<sub>3</sub> mapping population (ICCX-810800 × ICC20268) contrasting for seed weight (100-seed weight varied from 5.4 to 49.6 g) using the 96 informative TFGMS and TFFDMS markers and 32 genomic microsatellite markers was performed. An ANOVA revealed a significant difference of 100-seed weight (5.4–49.6 g) between the parental <em>desi</em> and <em>kabuli</em> genotypes and among segregating individuals. The frequency distribution of 100-seed weight (g) among 96 segregating individuals of a F<sub>3</sub> mapping population depicted a goodness of fit to the normal distribution (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>), and the phenotypic trait values of two parental genotypes also lied within the range that estimated for the entire mapping population. It thus indicated the quantitative inheritance pattern of seed traits under study in the developed mapping population and its utility in seed weight trait-specific bi-parental linkage mapping. The genetic mapping identified and mapped one major and significant (LOD threshold score &gt;3.7) QTL region (phenotypic effect variance/<em>R</em><sup>2</sup> = 30.7%) on the chickpea LG1 governing the 100-seed weight/size (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). The target 1.7-cM QTL region (73.5–75.2 cM) identified for seed weight based on both interval mapping and composite interval mapping spanned with three TFFDMS and one TFGMS markers, thus having high seed weight trait-specific association potential in <em>desi</em> and <em>kabuli</em> based on genetic association mapping. Therefore, the four seed weight trait-specific TFFDMS and TFGMS markers in the ORF and 5′ UTRs of TF genes validated both by traditional bi-parental linkage mapping and association mapping were selected further as the target candidate genes for controlling seed weight/size in <em>desi</em> and <em>kabuli</em>.</p>                            <h3 scrollto-destination=5039660 id="5039660" class="section-title" data-legacy-id=s3h>3.8. Differential expression profiling and functional significance of TFGMS and TFFDMS markers</h3>
<p>To derive correlation between association analysis and differential expression profiling, the seed and pod trait-specific 23 TFFDMS and 21 TFGMS markers associated 34 TF genes were validated in two different seed developmental stages of two contrasting small and large seed-size chickpea genotypes namely, ICCX-810800 and ICC20268 through semi-quantitative and quantitative RT-PCR. None of the TFFDMS and seven (33.3%) of the 21 TFGMS markers associated seven TF genes revealed differential expression during seed development in two contrasting chickpea genotypes by both semi-quantitative and quantitative RT-PCR. When we compared the differential expression pattern of both seed and pod trait-specific five TFFDMS and nine TFGMS markers associated five TF genes, none of the TFFDMS markers in these genes showed differential expression in seed developmental stages of two contrasting genotypes (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). In contrast, differential expression for one of the TFGMS marker associated TF gene during seed development in two contrasting small and large seed-size genotypes was observed (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). For instance, the TFFDMS marker containing (CAC)<sub>8</sub> repeat motif in the SBP functional domain of a high 100-seed weight-associated (<em>R</em><sup>2</sup> 0.38 and <em>P</em> &lt; 0.0019) TF gene namely, SBP, did not show differential regulation in seed developmental stages of ICCX-810800 and ICC20268 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>A). In contrast, the TFGMS marker containing (CTA)<sub>6</sub> repeat motif in the 5′ UTR of seed weight/size-associated (<em>R</em><sup>2</sup> 0.31 and <em>P</em> &lt; 0.0024) DUF1635 TF gene revealed down-regulated expression (at least 3.5 times) specifically in seed developmental stages of a large seed-size <em>kabuli</em> genotype ICC20268 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>B and D). While in two seed developmental stages of a small seed-size <em>desi</em> genotype ICCX-810800, such TFGMS marker associated TF gene showed almost an equal level of expression. The higher association potential of TFGMS marker in the 5′ UTR of the TF gene and their further correlation with differential down-regulated pattern of gene expression during seed development in two contrasting small and large seed-size genotypes suggested the functional relevance of such identified microsatellite markers in the TF genes for trait association in chickpea.</p><p>The sequencing of cloned cDNA and genomic DNA amplicons from the seed and pod trait-associated five TF genes in ICCX-810800 and ICC20268 revealed the presence of expected microsatellite repeat-motif sequences. Interestingly, the amplicon sequences from the five TFFDMS markers associated five TF genes that did not show differential expression during seed development contained the variable number of microsatellite repeat units in the functional domains of these genes between ICCX-810800 and ICC20268. The amplicon sequences from the TFGMS markers of TF genes that did not show differential expression during seed development, including the one which revealed differential expression, contained varied microsatellite repeats between ICCX-810800 and ICC20268. The sequencing of amplicons from the TFFDMS marker of 100-seed weight-associated SBP TF gene contained the desired (CAC)<sub>8</sub> repeat motif in the transcripts of ICCX-810800 as in ICC4958 from which the primers were designed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>A). Interestingly, the cDNA sequence alignment of such TFFDMS marker associated SBP TF gene between ICCX-810800 and ICC20268 showed the presence of variable number of repeat units like (CAC)<sub>8</sub> and (CAC)<sub>10</sub> in the SBP functional domain, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>A). The comparison between cDNA and genomic DNA sequences from the 100-seed weight associated and differentially expressed TFGMS marker associated DUF1635 TF gene revealed the variable number of microsatellite repeats (CTA)<sub>6</sub> and (CTA)<sub>8</sub> between ICCX-810800 and ICC20268, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). The expansion and contraction of microsatellite repeats in the 5′ UTRs of genes have significance in regulating gene expression for many traits including amylose content in rice,<sup><a reveal-id="DST015C71" data-open="DST015C71" class="link link-ref link-reveal xref-bibr">71</a></sup> quality protein in maize<sup><a reveal-id="DST015C72" data-open="DST015C72" class="link link-ref link-reveal xref-bibr">72</a></sup> and light and salicylic responses in <em>Brassica</em>.<sup><a reveal-id="DST015C73" data-open="DST015C73" class="link link-ref link-reveal xref-bibr">73</a></sup> These findings thus inferred the significant correlation between the number of microsatellite repeat units in the different sequence components (functional domain and 5′ UTR) of seed weight/size trait-specific TFFDMS and TFGMS associated TF genes and their differential expression in the genes preferentially with regard to seed development in contrasting chickpea genotypes. Understanding the adaptive and evolutionary significance of such TFFDMS and TFGMS repeat expansion/contraction in the functional domain and 5′ UTRs of TF genes, respectively, is of relevance that needs further systematic experimentation by assaying a large number of contrasting chickpea genotypes of different tissues and/or stages (embryogenesis to maturation) of seed development.</p><p>To further understand the functional significance of TFFDMS markers associated TF genes that are not showing differential expression during seed development, the high seed and pod trait-associated four TFFDMS markers showing fragment length polymorphism in the functional domains of four TF genes were selected (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). The alteration of secondary protein structure and functional domain sites between the variant and native forms due to the presence of variable number of TFFDMS repeat units encoding the amino acid residues in the functional domains of four seed and pod trait-associated TF genes was observed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>B). For instance, the expansion of TFFDMS repeat motif from (CAC)<sub>8</sub> in <em>desi</em> variety ICCX-810800 to (CAC)<sub>10</sub> in <em>kabuli</em> variety ICC20268 encoding histidine amino acid residues in the DNA-binding functional domain of SBP TF gene created an altered secondary structure of protein (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>B) and functional domain region that possibly affects the DNA binding and transcriptional activity of the target gene during developmental stages in chickpea. The altered transcriptional activity of SBP TF gene could be due to its differential sequence-specific DNA binding of all eight conserved cysteine/histidine amino acid residues (formed two separate novel zinc-binding sites) present in the DNA-binding domain with the recognition domains of promoter(s) of target gene(s).<sup><a reveal-id="DST015C74" data-open="DST015C74" class="link link-ref link-reveal xref-bibr">74</a></sup> Differential DNA-binding selectivity of SBP TF towards consensus box in the target gene promoter due to alteration of conserved cysteine/histidine amino acid residues within the DNA-binding functional domain and their correlation with transcriptional activity during developmental process have been previously reported in many plant species, including <em>Antirrhinum</em>,<sup><a reveal-id="DST015C75" data-open="DST015C75" class="link link-ref link-reveal xref-bibr">75</a>,<a reveal-id="DST015C76" data-open="DST015C76" class="link link-ref link-reveal xref-bibr">76</a></sup><em>Arabidopsis</em>,<sup><a reveal-id="DST015C77" data-open="DST015C77" class="link link-ref link-reveal xref-bibr">77</a></sup> rice<sup><a reveal-id="DST015C78" data-open="DST015C78" class="link link-ref link-reveal xref-bibr">78</a>,<a reveal-id="DST015C79" data-open="DST015C79" class="link link-ref link-reveal xref-bibr">79</a></sup> and tomato.<sup><a reveal-id="DST015C80" data-open="DST015C80" class="link link-ref link-reveal xref-bibr">80</a></sup> Overall, the correspondence of higher trait association potential of TFFDMS markers with fragment length polymorphism due to the presence of variable number of TFFDMS repeat units encoding amino acid residues in the functional domain of TF genes and alteration of secondary structure of proteins and functional domain sites suggested the utility of these functional genetic markers for rapidly establishing marker–trait linkages and identifying genes/QTLs for many useful agronomic traits in chickpea.</p>                            <h3 scrollto-destination=5039669 id="5039669" class="section-title" data-legacy-id=s3i>3.9. Gene-specific LD patterns in chickpea</h3>
<p>Understanding the gene-specific LD pattern is of significance to determine the marker density required for association mapping of complex traits in a target plant population under study.<sup><a reveal-id="DST015C81" data-open="DST015C81" class="link link-ref link-reveal xref-bibr">81</a>,<a reveal-id="DST015C82" data-open="DST015C82" class="link link-ref link-reveal xref-bibr">82</a></sup> For determining gene-specific LD estimates and extent of LD decay, the 32 SNP loci were identified in five seed and pod trait-associated TF genes among 96 chickpea genotypes based on cloned amplicon sequencing with an average SNP frequency of 1/297 bp. The nucleotide diversity (mean <em>π</em> = 4.7 × 10<sup>−3</sup> bp) and frequency of Watterson's <em>θ</em> level (mean <em>θ</em> = 2.3 × 10<sup>−3</sup>) estimated for 32 SNP loci in the TF genes varied widely across the genotypes. Higher nucleotide diversity of the SNP loci within <em>desi</em> (−0.44) when compared with that of <em>kabuli</em> (−0.39) was evident from the test of Tajima's <em>D</em> neutrality pattern of nucleotide polymorphism. The haplotype analysis using the SNPs, TFGMS and TFFDMS marker genotyping data identified 10–21 haplotypes per gene amplicon resulting in higher haplotype-based PIC values among the genotypes that varied from 0.53 to 0.97 with an average of 0.74. The candidate gene-based association analysis by the inclusion of SNP and microsatellite marker-based haplotypes identified in the five TF genes revealed a strong association of these genes with the seed and pod traits studied at significant <em>P</em> ≤ 0.001 and <em>R</em><sup>2</sup> from 0.38 to 0.45 (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). The genetic association potential of TF genes estimated combining the SNP, TFGMS and TFFDMS marker-based haplotypes was much higher than that obtained using the individual TFGMS, TFFDMS and SNP markers (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). The LD estimates (<em>r</em><sup>2</sup>) and extent of LD decay using the all possible pair-combinations of marker haplotypes were determined within the seed and pod trait-associated five TF genes. In <em>desi</em> and <em>kabuli</em> population groups, an LD did not decay below <em>r</em><sup>2</sup> = 0.1 within the 700-bp sequenced region of all the five TF genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). However, an overall decreasing trend of LD decay in the genes with an increase in the sequenced region (bp) was observed. It indicated the direct correlation of LD patterning with physical linkage of markers on chromosomes and marker density required in these genes. The diverse <em>desi</em> population groups showed faster LD decay than that of <em>kabuli</em> in all the seed and pod trait-associated five TF genes. In spite of inclusion of higher number of combined SNP and microsatellite haplotypes, the varied LD decay observed in other four TF genes (TCP, NAC, DUF1635 and DUF702) indicates the effect of factors other than marker density like genetic diversity, population genetic structure, population admixture and adaptive selection on shaping the LD patterns in these genes. Remarkably, the overall trend of LD decay was directly correlated with association potential of five TF genes for seed and pod traits in chickpea. The genes showing a low degree of association potential decayed LD much faster/earlier with an increase in the sequenced region (bp) when compared with the genes having higher association potential (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). These observations broadly suggested that, with the inclusion of additional SNP loci identified in the TF genes and their corresponding two TFGMS and three TFFDMS marker alleles, a higher number of haplotype pairs could be produced which in turn enhanced the LD resolution and trait association potential of the genes.</p><p>The SBP TF gene showing a strong genetic association with 100-seed weight/size did not decay its LD even across the 1000-bp sequenced region of <em>desi</em> and <em>kabuli</em> population. In 3000-bp sequenced amplicon of this TF gene (covering the ORF region), we identified the combinations of 11 synonymous SNP loci, 2 different alleles of TFGMS and 3 different alleles of TFFDMS polymorphic marker loci (Fig. <a reveal-id="DST015F3" data-open="DST015F3" class="link link-reveal link-table xref-fig">3</a>A) across 96 chickpea genotypes that enabled to constitute a maximum of 21 haplotypes in this gene (Fig. <a reveal-id="DST015F3" data-open="DST015F3" class="link link-reveal link-table xref-fig">3</a>B). The haplotype pair-based LD estimation displayed a significant LD, which persists at a high level (<em>r</em><sup>2</sup> &gt; 0.15 and <em>P</em> &lt; 0.0001) across the entire 3000-bp sequenced region of this gene (Fig. <a reveal-id="DST015F3" data-open="DST015F3" class="link link-reveal link-table xref-fig">3</a>C). The feature of high LD estimates covering the complete ORF of such TF gene thus increased its overall potential for seed trait (100-seed weight/size) association (<em>R</em><sup>2</sup> 0.45 and <em>P</em> &lt; 0.0012) in chickpea (Table <a reveal-id="DST015TB1" data-open="DST015TB1" class="link link-reveal link-table xref-fig">1</a>). Such microsatellite–SNP marker haplotype-based trait association analysis has already been implemented successfully for identification of genes/QTLs associated with infectious diseases in humans<sup><a reveal-id="DST015C83" data-open="DST015C83" class="link link-ref link-reveal xref-bibr">83</a></sup> and seed shattering during rice domestication.<sup><a reveal-id="DST015C84" data-open="DST015C84" class="link link-ref link-reveal xref-bibr">84</a></sup> Overall, these results suggested that the population genetic structure and marker density in the genes play a crucial role in LD patterning of TFs at the gene level in domesticated <em>desi</em> and <em>kabuli</em>. </p>                        <a id="5039677" scrollto-destination="5039677"></a>
<div id="DST015F3" data-id="DST015F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/m_dst01503.jpeg?Expires=2147483647&Signature=bxvtRIWC3YRc~bnPqpUSU0HVeP5SNM-j9yzWpCA9KwBmbfCwhEb~pWFRcp0UKIsWQWnJBYg0uhydUJdqaP2HAnmzK7IVLdMRsRaQS~FEDWYrHXPRM35Oe6mapmI59mZ~B5pTZS6d2IZD2BTTmcZTH4-qZ~EI6~m5SOSxTbbgXR5JbzvonMDLaimLHIG26Otuy~Z2wgQb~pX3ZBEhLpvRWRFC6k-Oy1WgtyzyMdmbzbgj-ilncoABcaGpu8NCXVuAxAvnbL~NMz3F2X4UEMO2jxXPbv1crAt~7CAK4rXfR6m1AmmMiAHAJ9Qn7OOlEcLk3VtfADA8sSw-SX5h2VjIrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The genotyping of 13 polymorphic marker loci [11 and 1 single nucleotide polymorphisms (SNPs) of each TFFDMS and TFGMS markers] identified in different ORF and UTR sequence components of a high seed-size/weight-associated SBP TF gene (A) among 96 association mapping panel constituted 2 haplotypes in wild, 13 haplotypes in desi and 6 haplotypes in kabuli (B). The 21 haplotype marker-based genotyping information produced high LD estimates (r2 &gt; 0.15 and P &lt; 0.0001) covering the complete 3000-bp sequenced region of SBP TF gene (C) and thus enhanced its overall LD resolution. The specific haplotypes showing differentiation between small (ICCX-810800) and large (ICC20268) seed-size contrasting chickpea genotypes (B) were represented more in the 96 association mapping panel, which reemphasizes the strong potential of SBP TF for seed-size/weight-specific trait association." path-from-xml="dst01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5039677/dst01503.jpeg" path-from-xml="dst01503.jpeg" target="_blank">View large</a><a section="5039677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst01503.jpeg?Expires=2147483647&Signature=D3tcdcGtaXc6vAtxPbwws1CAwV-AvVK-MPTNIUQoNJIBc4FSV6GWCHFIcdaTK35UsYb~1w-vxnGEF8p7lPY9zyRT5D46mt-qURKpXFKn6-Z92-2J2fyPiOoOLsAB1KCuwPsU1W96P6Cgl0~Ym4MO3CZrAhIEbVHj8VOi6fdtuCAlQlAnrN0h1hCTOV1mMbOCjtFAhUMqNXkUZTouLMRI27YLVC4BhB7X4FWzvJNXHBuD-pNlXNs453fhk6gc9rsgBj0AF2klO3--S93Vw4O8SOnNKUheoDv2BVqbtsnShMSN-1hIojywply20hSC8fmgSRZgXgvDsuzfsXBcwlAegQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5039677&ar=353793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The genotyping of 13 polymorphic marker loci [11 and 1 single nucleotide polymorphisms (SNPs) of each TFFDMS and TFGMS markers] identified in different ORF and UTR sequence components of a high seed-size/weight-associated SBP TF gene (A) among 96 association mapping panel constituted 2 haplotypes in wild, 13 haplotypes in <em>desi</em> and 6 haplotypes in <em>kabuli</em> (B). The 21 haplotype marker-based genotyping information produced high LD estimates (<em>r</em><sup>2</sup> &gt; 0.15 and <em>P</em> &lt; 0.0001) covering the complete 3000-bp sequenced region of SBP TF gene (C) and thus enhanced its overall LD resolution. The specific haplotypes showing differentiation between small (ICCX-810800) and large (ICC20268) seed-size contrasting chickpea genotypes (B) were represented more in the 96 association mapping panel, which reemphasizes the strong potential of SBP TF for seed-size/weight-specific trait association.</p></div></div><div content-id="DST015F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/m_dst01503.jpeg?Expires=2147483647&Signature=bxvtRIWC3YRc~bnPqpUSU0HVeP5SNM-j9yzWpCA9KwBmbfCwhEb~pWFRcp0UKIsWQWnJBYg0uhydUJdqaP2HAnmzK7IVLdMRsRaQS~FEDWYrHXPRM35Oe6mapmI59mZ~B5pTZS6d2IZD2BTTmcZTH4-qZ~EI6~m5SOSxTbbgXR5JbzvonMDLaimLHIG26Otuy~Z2wgQb~pX3ZBEhLpvRWRFC6k-Oy1WgtyzyMdmbzbgj-ilncoABcaGpu8NCXVuAxAvnbL~NMz3F2X4UEMO2jxXPbv1crAt~7CAK4rXfR6m1AmmMiAHAJ9Qn7OOlEcLk3VtfADA8sSw-SX5h2VjIrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The genotyping of 13 polymorphic marker loci [11 and 1 single nucleotide polymorphisms (SNPs) of each TFFDMS and TFGMS markers] identified in different ORF and UTR sequence components of a high seed-size/weight-associated SBP TF gene (A) among 96 association mapping panel constituted 2 haplotypes in wild, 13 haplotypes in desi and 6 haplotypes in kabuli (B). The 21 haplotype marker-based genotyping information produced high LD estimates (r2 &gt; 0.15 and P &lt; 0.0001) covering the complete 3000-bp sequenced region of SBP TF gene (C) and thus enhanced its overall LD resolution. The specific haplotypes showing differentiation between small (ICCX-810800) and large (ICC20268) seed-size contrasting chickpea genotypes (B) were represented more in the 96 association mapping panel, which reemphasizes the strong potential of SBP TF for seed-size/weight-specific trait association." path-from-xml="dst01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5039677/dst01503.jpeg" path-from-xml="dst01503.jpeg" target="_blank">View large</a><a section="5039677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst01503.jpeg?Expires=2147483647&Signature=D3tcdcGtaXc6vAtxPbwws1CAwV-AvVK-MPTNIUQoNJIBc4FSV6GWCHFIcdaTK35UsYb~1w-vxnGEF8p7lPY9zyRT5D46mt-qURKpXFKn6-Z92-2J2fyPiOoOLsAB1KCuwPsU1W96P6Cgl0~Ym4MO3CZrAhIEbVHj8VOi6fdtuCAlQlAnrN0h1hCTOV1mMbOCjtFAhUMqNXkUZTouLMRI27YLVC4BhB7X4FWzvJNXHBuD-pNlXNs453fhk6gc9rsgBj0AF2klO3--S93Vw4O8SOnNKUheoDv2BVqbtsnShMSN-1hIojywply20hSC8fmgSRZgXgvDsuzfsXBcwlAegQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5039677&ar=353793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The genotyping of 13 polymorphic marker loci [11 and 1 single nucleotide polymorphisms (SNPs) of each TFFDMS and TFGMS markers] identified in different ORF and UTR sequence components of a high seed-size/weight-associated SBP TF gene (A) among 96 association mapping panel constituted 2 haplotypes in wild, 13 haplotypes in <em>desi</em> and 6 haplotypes in <em>kabuli</em> (B). The 21 haplotype marker-based genotyping information produced high LD estimates (<em>r</em><sup>2</sup> &gt; 0.15 and <em>P</em> &lt; 0.0001) covering the complete 3000-bp sequenced region of SBP TF gene (C) and thus enhanced its overall LD resolution. The specific haplotypes showing differentiation between small (ICCX-810800) and large (ICC20268) seed-size contrasting chickpea genotypes (B) were represented more in the 96 association mapping panel, which reemphasizes the strong potential of SBP TF for seed-size/weight-specific trait association.</p></div></div>                            <h3 scrollto-destination=5039680 id="5039680" class="section-title" data-legacy-id=s3j>3.10. Possible haplotype-based evolutionary history of SBP TF genes during chickpea domestication</h3>
<p>The determination of step-wise haplotype sharing/changes in the genes among cultivated and wild species will have significance in understanding the evolutionary process of domestication of those target genes from ancestors to cultivated species. A set of 21 TFGMS–TFFDMS–SNP marker haplotypes identified in a high seed weight-associated SBP TF gene among <em>desi</em>, <em>kabuli</em> and wild population groups gave clues to study the hypothetical evolutionary history of SBP gene during chickpea domestication. It implied that three SNPs (mutations) and two microsatellite repeat expansion (constituting two haplotypes) in wild, two SNPs, three microsatellite repeat expansion/contraction and three recombination events among nine SNPs (13 haplotypes) in <em>desi</em>, and four SNPs and two microsatellite repeat expansion (six haplotypes) in <em>kabuli</em>, were adequate to explain the natural variation of SBP TF gene in chickpea (Fig. <a reveal-id="DST015F3" data-open="DST015F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>). It also implied that the seed weight/size-associated TFFDMS alleles might be fixed in the SBP gene haplotypes of domesticated <em>desi</em> and <em>kabuli</em> chickpea as pre-existing natural variation from wild <em>C. reticulatum</em>. The seed weight/size-associated haplotypes, including the TFFDMS alleles, that differentiated the contrasting small seed-size <em>desi</em> from large seed-size <em>kabuli</em> based on the variation of TFFDMS repeats in the functional domain of SBP gene was represented mostly (89) among 96 genotypes under study. This ascertained the higher association potential of SBP TF gene for seed weight/size in chickpea. Maximum sharing of <em>desi</em> particularly the seed weight-associated contrasting small seed-size ICCX-810800-specific haplotypes (21) and TFFDMS alleles (16) with wild <em>C. reticulatum</em> was evident (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>). It is supported well with the earlier reports on the resemblance and evolutionary closeness of small seed-size morphology between <em>desi</em> and wild <em>C. reticulatum</em>. In contrast, the sharing of <em>kabuli</em> particularly the seed weight/size-associated contrasting large seed-size ICC20268-specific haplotypes (16) and TFFDMS alleles (14) was more with <em>desi</em> when compared to that with wild. It gave clues that the divergence between <em>kabuli</em> and <em>desi</em> is much recent when compared with that between <em>desi</em> and wild progenitor <em>C. reticulatum</em>.</p><p>The occurrence of nucleotide polymorphisms, seed weight/size-associated TFFDMS repeat expansion/contraction and recombination events in the haplotypes of SBP gene among <em>desi</em>, <em>kabuli</em> and wild reflected that the strong artificial selection has played a major role during domestication of this gene towards assortment of more preferential seed weight/size traits in chickpea. This is in agreement with earlier archeological, phenotypic and molecular phylogeny studies<sup><a reveal-id="DST015C53" data-open="DST015C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DST015C54" data-open="DST015C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DST015C59" data-open="DST015C59" class="link link-ref link-reveal xref-bibr">59</a>,<a reveal-id="DST015C60" data-open="DST015C60" class="link link-ref link-reveal xref-bibr">60</a></sup> that describe that the chickpea domestication (∼10 000 years) might have led through population bottlenecks during their evolutionary transition from wild <em>C. reticulatum</em> by the occurrence of spontaneous mutation, recombination and fixation. Such newly occurring changes would have become fixed through propagation in cultivated chickpea, which is self-pollinated in nature. The higher microsatellite expansion/contraction of seed weight/size-associated TFFDMS alleles and recombination among SNPs within the haplotypes of SBP TF gene attributed to more seed weight/size-specific haplotype diversity in <em>desi</em> in contrast to <em>kabuli</em>. It could have resulted from artificial selection and modern breeding efforts (inter-specific hybridization/introgression of small seed-size <em>desi</em> with large seed-size <em>kabuli</em>) continuously practiced during the genetic improvement of <em>desi</em> for different seed-size characteristics particularly for a large seed-size trait, which is of a high consumer preference and trade value. The molecular marker-based haplotyping in the SBP TF gene overall gave clues to understand the possible evolutionary pathway of the target gene in domesticated <em>desi</em>, <em>kabuli</em> and wild chickpea and the consequence of their natural variations on seed-size/weight-specific trait evolution during chickpea domestication. Therefore, the seed-size/weight supposed to represent an important component of domestication trait in chickpea. However, the complete understanding of evolutionary history of such seed-size/weight-associated SBP TF gene during chickpea domestication needs further systematic experimentation by assaying high-throughput marker genotyping information in a larger set of contrasting genotypes at the genome and gene level.</p>                            <h2 scrollto-destination=5039695 id="5039695" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>A larger set of experimentally validated TFGMS and TFFDMS markers with relatively high intra-specific polymorphic potential developed from the TF genes will have immense use in large-scale genotyping applications, including construction of high-density physical and functional transcript map as well as understanding the population genetic structure and functional molecular diversity in chickpea. The functional relevance of these informative genic markers was demonstrated by integrating seed and pod trait association analysis with genetic mapping, differential expression profiling, protein modelling and haplotype gene evolution study in relation to selected five TF genes, thereby suggesting the utility of markers in many aspects of chickpea breeding and genetic analysis.</p>                            <h2 scrollto-destination=5039703 id="5039703" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5039712 id="5039712" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors gratefully acknowledge the financial support by the Department of Biotechnology (DBT), Government of India. We thank the DNA Sequencing Facility, NIPGR for automated fragment analysis and sequencing. Ms Alice Kujur acknowledges the Council of Scientific and Industrial Research (CSIR) for the award of Junior Research Fellowship (JRF).</p>                            <h2 scrollto-destination=5039720 id="5039720" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST015C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.C.</div></div>,  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Fahima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Microsatellites within genes: structure, function, and evolution</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">991</div>-<div class="lpage">1007</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msh073" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msh073</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20within%20genes%3A%20structure%2C%20function%2C%20and%20evolution&amp;author=Y.C.%20Li&amp;author=A.B.%20Korol&amp;author=T.%20Fahima&amp;author=E.%20Nevo&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=21&amp;pages=991-1007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14963101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genic microsatellite markers in plants: features and applications</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">48</div>-<div class="lpage">55</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tibtech.2004.11.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tibtech.2004.11.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20microsatellite%20markers%20in%20plants%3A%20features%20and%20applications&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Biotechnol.&amp;volume=23&amp;pages=48-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.11.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.11.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Dalal</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Genic non-coding microsatellites in the rice genome: characterization, marker design and use in assessing genetic and evolutionary relationships among domesticated groups</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">140</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-10-140" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-10-140</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20non-coding%20microsatellites%20in%20the%20rice%20genome%3A%20characterization%2C%20marker%20design%20and%20use%20in%20assessing%20genetic%20and%20evolutionary%20relationships%20among%20domesticated%20groups&amp;author=S.K.%20Parida&amp;author=V.%20Dalal&amp;author=N.K.%20Singh&amp;author=T.%20Mohapatra&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=140" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19335879" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Raj Kumar</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Dalal</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Unigene derived microsatellite markers for the cereal genomes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">808</div>-<div class="lpage">17</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-005-0182-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-005-0182-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unigene%20derived%20microsatellite%20markers%20for%20the%20cereal%20genomes&amp;author=S.K.%20Parida&amp;author=K.A.%20Raj%20Kumar&amp;author=V.%20Dalal&amp;author=N.K.%20Singh&amp;author=T.%20Mohapatra&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=808-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0182-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0182-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0182-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16429310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Pandit</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gaikwad</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Functionally relevant microsatellites in sugarcane unigenes</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">251</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-251" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-10-251</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functionally%20relevant%20microsatellites%20in%20sugarcane%20unigenes&amp;author=S.K.%20Parida&amp;author=A.%20Pandit&amp;author=K.%20Gaikwad&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=251" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21083898" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nayak</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Varghese</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Integration of novel SSR and gene-based SNP marker loci in the chickpea genetic map and establishment of new anchor points with <em>Medicago truncatula</em> genome</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">1415</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-010-1265-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-010-1265-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20novel%20SSR%20and%20gene-based%20SNP%20marker%20loci%20in%20the%20chickpea%20genetic%20map%20and%20establishment%20of%20new%20anchor%20points%20with%20Medicago%20truncatula%20genome&amp;author=S.N.%20Nayak&amp;author=H.%20Zhu&amp;author=N.%20Varghese&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=1415-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1265-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1265-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1265-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20098978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gujaria</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dauthal</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Development and use of genic molecular markers (GMMs) for construction of a transcript map of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">1577</div>-<div class="lpage">89</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-011-1556-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-011-1556-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20use%20of%20genic%20molecular%20markers%20(GMMs)%20for%20construction%20of%20a%20transcript%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=N.%20Gujaria&amp;author=A.%20Kumar&amp;author=P.%20Dauthal&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=1577-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1556-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1556-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1556-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21384113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gaur</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>. </div><div class="article-title">EST-derived genic molecular markers: development and utilization for generating an advanced transcript map of chickpea</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">1449</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-012-1800-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-012-1800-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST-derived%20genic%20molecular%20markers%3A%20development%20and%20utilization%20for%20generating%20an%20advanced%20transcript%20map%20of%20chickpea&amp;author=S.%20Choudhary&amp;author=R.%20Gaur&amp;author=S.%20Gupta&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=1449-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1800-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1800-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1800-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22301907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Cook</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">Orphan legume crops enter the genomics era</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">202</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.pbi.2008.12.004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.pbi.2008.12.004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Orphan%20legume%20crops%20enter%20the%20genomics%20era&amp;author=R.K.%20Varshney&amp;author=T.J.%20Close&amp;author=N.K.%20Singh&amp;author=D.A.%20Hoisington&amp;author=D.R.%20Cook&amp;publication_year=2009&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=12&amp;pages=202-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2008.12.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2008.12.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2008.12.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19157958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of chickpea EST-SSR markers and analysis of allelic variation across related species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">591</div>-<div class="lpage">608</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-008-0923-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-008-0923-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20chickpea%20EST-SSR%20markers%20and%20analysis%20of%20allelic%20variation%20across%20related%20species&amp;author=S.%20Choudhary&amp;author=N.K.%20Sethy&amp;author=B.%20Shokeen&amp;author=S.%20Bhatia&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=591-608" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0923-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0923-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0923-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19020854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>De novo</em> assembly of chickpea transcriptome using short reads for gene discovery and marker identification</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">53</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsq028" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsq028</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20chickpea%20transcriptome%20using%20short%20reads%20for%20gene%20discovery%20and%20marker%20identification&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=53-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Jhanwar</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Gene discovery and tissue-specific transcriptome analysis in chickpea with massively parallel pyrosequencing and web resource development</div>, <div class="source">Plant Physiol.</div> , <div class="year">2011b</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1661</div>-<div class="lpage">78</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.111.178616" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.111.178616</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20discovery%20and%20tissue-specific%20transcriptome%20analysis%20in%20chickpea%20with%20massively%20parallel%20pyrosequencing%20and%20web%20resource%20development&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=S.%20Jhanwar&amp;publication_year=2011b&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=1661-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.178616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.178616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.178616"> </span></div> </div></div></div></div></div><div content-id="DST015C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiremath</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Farmer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>, et al. </div><div class="article-title">Large-scale transcriptome analysis in chickpea (<em>Cicer arietinum</em> L.), an orphan legume crop of the semi-arid tropics of Asia and Africa</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2011</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">922</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1467-7652.2011.00625.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1467-7652.2011.00625.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20transcriptome%20analysis%20in%20chickpea%20(Cicer%20arietinum%20L.)%2C%20an%20orphan%20legume%20crop%20of%20the%20semi-arid%20tropics%20of%20Asia%20and%20Africa&amp;author=P.J.%20Hiremath&amp;author=A.%20Farmer&amp;author=S.B.%20Cannon&amp;publication_year=2011&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=9&amp;pages=922-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2011.00625.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2011.00625.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2011.00625.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21615673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jhanwar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Priya</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Transcriptome sequencing of wild chickpea as a rich resource for marker development</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2012</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">690</div>-<div class="lpage">702</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1467-7652.2012.00712.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1467-7652.2012.00712.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20wild%20chickpea%20as%20a%20rich%20resource%20for%20marker%20development&amp;author=S.%20Jhanwar&amp;author=P.%20Priya&amp;author=R.%20Garg&amp;author=S.K.%20Parida&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=690-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00712.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00712.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00712.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22672127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Century</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Reuber</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Ratcliffe</div> <div class="given-names">O.J.</div></div>. </div><div class="article-title">Regulating the regulators: the future prospects for transcription-factor-based agricultural biotechnology products</div>, <div class="source">Plant Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">20</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.108.117887" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.108.117887</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulating%20the%20regulators%3A%20the%20future%20prospects%20for%20transcription-factor-based%20agricultural%20biotechnology%20products&amp;author=K.%20Century&amp;author=T.L.%20Reuber&amp;author=O.J.%20Ratcliffe&amp;publication_year=2008&amp;journal=Plant%20Physiol.&amp;volume=147&amp;pages=20-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.117887" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.117887" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.117887"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18443103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Libault</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Joshi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Benedito</div> <div class="given-names">V.A.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Udvardi</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Stacey</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Legume transcription factor genes: what makes legumes so special?</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">991</div>-<div class="lpage">1001</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.109.144105" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.109.144105</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legume%20transcription%20factor%20genes%3A%20what%20makes%20legumes%20so%20special%3F&amp;author=M.%20Libault&amp;author=T.%20Joshi&amp;author=V.A.%20Benedito&amp;author=D.%20Xu&amp;author=M.K.%20Udvardi&amp;author=G.%20Stacey&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=991-1001" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.144105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.144105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.144105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19726573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Udvardi</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Kakar</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wandrey</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Legume transcription factors: global regulators of plant development and response to the environment</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">538</div>-<div class="lpage">49</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.107.098061" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.107.098061</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legume%20transcription%20factors%3A%20global%20regulators%20of%20plant%20development%20and%20response%20to%20the%20environment&amp;author=M.K.%20Udvardi&amp;author=K.%20Kakar&amp;author=M.%20Wandrey&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=144&amp;pages=538-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.098061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.098061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.098061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17556517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Agarwal</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kapoor</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Transcription factors regulating the progression of monocot and dicot seed development</div>, <div class="source">Bioessays</div> , <div class="year">2011</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">189</div>-<div class="lpage">202</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/bies.201000107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/bies.201000107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factors%20regulating%20the%20progression%20of%20monocot%20and%20dicot%20seed%20development&amp;author=P.%20Agarwal&amp;author=S.%20Kapoor&amp;author=A.K.%20Tyagi&amp;publication_year=2011&amp;journal=Bioessays&amp;volume=33&amp;pages=189-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bies.201000107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbies.201000107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbies.201000107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21319185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Paik</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Han</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Choe</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Hur</div> <div class="given-names">C.G.</div></div>. </div><div class="article-title">Functional domain marker (FDM): an <em>in silico</em> demonstration in Solanaceae using simple sequence repeats (SSRs)</div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="year">2010</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">352</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11105-009-0154-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11105-009-0154-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20domain%20marker%20(FDM)%3A%20an%20in%20silico%20demonstration%20in%20Solanaceae%20using%20simple%20sequence%20repeats%20(SSRs)&amp;author=J.K.%20Yu&amp;author=H.%20Paik&amp;author=J.P.%20Choi&amp;author=J.H.%20Han&amp;author=J.K.%20Choe&amp;author=C.G.%20Hur&amp;publication_year=2010&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=28&amp;pages=352-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-009-0154-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-009-0154-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-009-0154-8"> </span></div> </div></div></div></div></div><div content-id="DST015C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">DeClerck</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lukashova</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lipovich</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cartinhour</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Computational and experimental analysis of microsatellites in rice (<em>Oryza sativa</em> L.): frequency, length-variation, transposon associations and genetic marker potential</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1441</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.184001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.184001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20analysis%20of%20microsatellites%20in%20rice%20(Oryza%20sativa%20L.)%3A%20frequency%2C%20length-variation%2C%20transposon%20associations%20and%20genetic%20marker%20potential&amp;author=S.%20Temnykh&amp;author=G.%20DeClerck&amp;author=A.%20Lukashova&amp;author=L.%20Lipovich&amp;author=S.%20Cartinhour&amp;author=S.%20McCouch&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1441-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.184001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.184001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.184001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yilmaz</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mejia-Guerra</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Kurz</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Welch</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Grotewold</div> <div class="given-names">E.</div></div>. </div><div class="article-title">AGRIS: the <em>Arabidopsis</em> gene regulatory information server, an update</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">D1118</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkq1120" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkq1120</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AGRIS%3A%20the%20Arabidopsis%20gene%20regulatory%20information%20server%2C%20an%20update&amp;author=A.%20Yilmaz&amp;author=M.K.%20Mejia-Guerra&amp;author=K.%20Kurz&amp;author=X.%20Liang&amp;author=L.%20Welch&amp;author=E.%20Grotewold&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=D1118-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21059685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamaguchi-Shinozaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tran</div> <div class="given-names">L.S.</div></div>. </div><div class="article-title">LegumeTFDB: an integrative database of <em>Glycine max</em>, <em>Lotus japonicus</em> and <em>Medicago truncatula</em> transcription factors</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">290</div>-<div class="lpage">1</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btp645" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btp645</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LegumeTFDB%3A%20an%20integrative%20database%20of%20Glycine%20max%2C%20Lotus%20japonicus%20and%20Medicago%20truncatula%20transcription%20factors&amp;author=K.%20Mochida&amp;author=T.%20Yoshida&amp;author=T.%20Sakurai&amp;author=K.%20Yamaguchi-Shinozaki&amp;author=K.%20Shinozaki&amp;author=L.S.%20Tran&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=290-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp645" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp645" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp645"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19933159" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Jayaswal</div> <div class="given-names">P.K.</div></div>, et al. </div><div class="article-title">The first draft of the pigeonpea genome sequence</div>, <div class="source">J. Plant Biochem. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">98</div>-<div class="lpage">112</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s13562-011-0088-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s13562-011-0088-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20draft%20of%20the%20pigeonpea%20genome%20sequence&amp;author=N.K.%20Singh&amp;author=D.K.%20Gupta&amp;author=P.K.%20Jayaswal&amp;publication_year=2011&amp;journal=J.%20Plant%20Biochem.%20Biotechnol.&amp;volume=21&amp;pages=98-112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13562-011-0088-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13562-011-0088-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13562-011-0088-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24431589" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>, et al. </div><div class="article-title">Draft genome sequence of chickpea (<em>Cicer arietinum</em>) provides a resource for trait improvement</div>, <div class="source">Nat Biotechnol.</div> , <div class="year">2013</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">240</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20chickpea%20(Cicer%20arietinum)%20provides%20a%20resource%20for%20trait%20improvement&amp;author=R.K.%20Varshney&amp;author=C.%20Song&amp;author=R.K.%20Saxena&amp;publication_year=2013&amp;journal=Nat%20Biotechnol.&amp;volume=31&amp;pages=240-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2491"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23354103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">77</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jhered/93.1.77" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jhered/93.1.77</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Muse</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">PowerMarker: an integrated analysis environment for genetic marker analysis</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">2128</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bti282" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bti282</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PowerMarker%3A%20an%20integrated%20analysis%20environment%20for%20genetic%20marker%20analysis&amp;author=K.%20Liu&amp;author=S.V.%20Muse&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2128-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15705655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Pfaff</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Udupa</div> <div class="given-names">S.M.</div></div>, et al. </div><div class="article-title">Characterization and mapping of sequence-tagged microsatellite sites in the chickpea (<em>Cicer arietinum</em> L.) genome</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">262</div> (pg. <div class="fpage">90</div>-<div class="lpage">101</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s004380051063" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s004380051063</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20mapping%20of%20sequence-tagged%20microsatellite%20sites%20in%20the%20chickpea%20(Cicer%20arietinum%20L.)%20genome&amp;author=P.%20Winter&amp;author=T.%20Pfaff&amp;author=S.M.%20Udupa&amp;publication_year=1999&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=262&amp;pages=90-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380051063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380051063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380051063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10503540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Inference of population structure using multilocus genotype data</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">945</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=945-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Kroon</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Casstevens</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Ramdoss</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">TASSEL: software for association mapping of complex traits in diverse samples</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2633</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btm308" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btm308</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.E.%20Kroon&amp;author=T.M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=E.S.%20Buckler&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2633-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hardy</div> <div class="given-names">O.J.</div></div>,  <div class="name"><div class="surname">Vekemans</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Spagedi: a versatile computer program to analyze spatial genetic structure at the individual or population levels</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2002</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">618</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1471-8286.2002.00305.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1471-8286.2002.00305.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spagedi%3A%20a%20versatile%20computer%20program%20to%20analyze%20spatial%20genetic%20structure%20at%20the%20individual%20or%20population%20levels&amp;author=O.J.%20Hardy&amp;author=X.%20Vekemans&amp;publication_year=2002&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=2&amp;pages=618-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2002.00305.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2002.00305.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2002.00305.x"> </span></div> </div></div></div></div></div><div content-id="DST015C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>. </div>, <div class="source">MapQTL 6: Software for the Mapping of Quantitative Trait Loci in Experimental Populations of Diploid Species</div> , <div class="year">2009</div><div class="publisher-loc">Netherlands</div><div class="publisher-name">B. V. Kyazma Wageningen</div></div></div></div></div><div content-id="DST015C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sahoo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Validation of internal control genes for quantitative gene expression studies in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2010</div>, vol. <div class="volume">396</div> (pg. <div class="fpage">283</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbrc.2010.04.079" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bbrc.2010.04.079</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Validation%20of%20internal%20control%20genes%20for%20quantitative%20gene%20expression%20studies%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Garg&amp;author=A.%20Sahoo&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2010&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=396&amp;pages=283-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2010.04.079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2010.04.079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2010.04.079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20399753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramachandran</div> <div class="given-names">G.N.</div></div>,  <div class="name"><div class="surname">Ramakrishnan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Sasisekharan</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Stereochemistry of polypeptide chain configurations</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1963</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">95</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0022-2836(63)80023-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0022-2836(63)80023-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stereochemistry%20of%20polypeptide%20chain%20configurations&amp;author=G.N.%20Ramachandran&amp;author=C.%20Ramakrishnan&amp;author=V.%20Sasisekharan&amp;publication_year=1963&amp;journal=J.%20Mol.%20Biol.&amp;volume=7&amp;pages=95-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(63)80023-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(63)80023-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(63)80023-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13990617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msr121" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msr121</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Librado</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rozas</div> <div class="given-names">J.</div></div>. </div><div class="article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1451</div>-<div class="lpage">2</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btp187" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btp187</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaSP%20v5%3A%20a%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data&amp;author=P.%20Librado&amp;author=J.%20Rozas&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1451-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19346325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Molecular Evolutionary Genetics</div> , <div class="year">1987</div><div class="publisher-loc">New York</div><div class="publisher-name">Columbia University Press</div></div></div></div></div><div content-id="DST015C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watterson</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">On the number of segregating sites in genetical models without recombination</div>, <div class="source">Theor. Popul. Biol.</div> , <div class="year">1975</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">256</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0040-5809(75)90020-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0040-5809(75)90020-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20number%20of%20segregating%20sites%20in%20genetical%20models%20without%20recombination&amp;author=G.A.%20Watterson&amp;publication_year=1975&amp;journal=Theor.%20Popul.%20Biol.&amp;volume=7&amp;pages=256-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0040-5809(75)90020-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0040-5809(75)90020-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0040-5809(75)90020-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1145509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tajima</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Statistical-method for testing the neutral mutation hypothesis by DNA polymorphism</div>, <div class="source">Genetics</div> , <div class="year">1989</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">585</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical-method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism&amp;author=F.%20Tajima&amp;publication_year=1989&amp;journal=Genetics&amp;volume=123&amp;pages=585-95" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mather</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Caicedo</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Polato</div> <div class="given-names">N.R.</div></div>,  <div class="name"><div class="surname">Olsen</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Purugganan</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">Extent of linkage disequilibrium in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Genetics</div> , <div class="year">2007</div>, vol. <div class="volume">177</div> (pg. <div class="fpage">2223</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.107.079616" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.107.079616</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extent%20of%20linkage%20disequilibrium%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=K.A.%20Mather&amp;author=A.L.%20Caicedo&amp;author=N.R.%20Polato&amp;author=K.M.%20Olsen&amp;author=S.%20McCouch&amp;author=M.D.%20Purugganan&amp;publication_year=2007&amp;journal=Genetics&amp;volume=177&amp;pages=2223-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.079616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.079616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.079616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17947413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yan</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Agrama</div> <div class="given-names">H.A.</div></div>, et al. </div><div class="article-title">Association mapping of stigma and spikelet characteristics in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">277</div>-<div class="lpage">92</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11032-009-9290-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11032-009-9290-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20of%20stigma%20and%20spikelet%20characteristics%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=W.G.%20Yan&amp;author=Y.%20Li&amp;author=H.A.%20Agrama&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=24&amp;pages=277-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9290-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9290-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9290-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20234878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of microsatellite markers and analysis of intra-specific genetic variability in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">1416</div>-<div class="lpage">28</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-006-0243-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-006-0243-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20microsatellite%20markers%20and%20analysis%20of%20intra-specific%20genetic%20variability%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=N.K.%20Sethy&amp;author=B.%20Shokeen&amp;author=K.J.%20Edwards&amp;author=S.%20Bhatia&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=1416-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0243-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0243-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0243-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16534564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gaur</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sethy</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Choudhary</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shokeen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Advancing the STMS genomic resources for defining new locations on the intra-specific genetic linkage map of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">117</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-12-117" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-12-117</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advancing%20the%20STMS%20genomic%20resources%20for%20defining%20new%20locations%20on%20the%20intra-specific%20genetic%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Gaur&amp;author=N.K.%20Sethy&amp;author=S.%20Choudhary&amp;author=B.%20Shokeen&amp;author=V.%20Gupta&amp;author=S.%20Bhatia&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21329497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bharadwaj</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Srivastava</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Chauhan</div> <div class="given-names">S.K.</div></div>, et al. </div><div class="article-title">Molecular diversity and phylogeny in geographical collection of chickpea (<em>Cicer</em> sp.) accessions</div>, <div class="source">J. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">e94</div>-<div class="lpage">e100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20diversity%20and%20phylogeny%20in%20geographical%20collection%20of%20chickpea%20(Cicer%20sp.)%20accessions&amp;author=C.%20Bharadwaj&amp;author=R.%20Srivastava&amp;author=S.K.%20Chauhan&amp;publication_year=2011&amp;journal=J.%20Genet.&amp;volume=90&amp;pages=e94-e100" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22232200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Yadava</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Microsatellites in <em>Brassica</em> unigenes: relative abundance, marker design and use in comparative physical mapping and genome analysis</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">55</div>-<div class="lpage">67</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G09-084" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G09-084</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20in%20Brassica%20unigenes%3A%20relative%20abundance%2C%20marker%20design%20and%20use%20in%20comparative%20physical%20mapping%20and%20genome%20analysis&amp;author=S.K.%20Parida&amp;author=D.K.%20Yadava&amp;author=T.%20Mohapatra&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=55-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20130749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cho</div> <div class="given-names">Y.G.</div></div>,  <div class="name"><div class="surname">Ishii</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Diversity of microsatellites derived from genomic libraries and GenBank sequences in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">713</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s001220051343" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s001220051343</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20microsatellites%20derived%20from%20genomic%20libraries%20and%20GenBank%20sequences%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=Y.G.%20Cho&amp;author=T.%20Ishii&amp;author=S.%20Temnykh&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=100&amp;pages=713-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051343" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051343"> </span></div> </div></div></div></div></div><div content-id="DST015C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Vigouroux</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Goodman</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Sanchez</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Doebley</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A single domestication for maize shown by multilocus microsatellite genotyping</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">6080</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.052125199" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.052125199</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20single%20domestication%20for%20maize%20shown%20by%20multilocus%20microsatellite%20genotyping&amp;author=Y.%20Matsuoka&amp;author=Y.%20Vigouroux&amp;author=M.M.%20Goodman&amp;author=G.J.%20Sanchez&amp;author=E.%20Buckler&amp;author=J.%20Doebley&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=6080-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.052125199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.052125199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.052125199"> </span></div> </div></div></div></div></div><div content-id="DST015C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Udupa</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Baum</div> <div class="given-names">M.</div></div>. </div><div class="article-title">High mutation rate and mutational bias at (TAA)<sub>n</sub> microsatellite loci in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2001</div>, vol. <div class="volume">265</div> (pg. <div class="fpage">1097</div>-<div class="lpage">103</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s004380100508" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s004380100508</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20mutation%20rate%20and%20mutational%20bias%20at%20(TAA)n%20microsatellite%20loci%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=S.M.%20Udupa&amp;author=M.%20Baum&amp;publication_year=2001&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=265&amp;pages=1097-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380100508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380100508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380100508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11523782" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iruela</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rubio</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cubero</div> <div class="given-names">J.I.</div></div>,  <div class="name"><div class="surname">Gil</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Millan</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Phylogenetic analysis in the genus <em>Cicer</em> and cultivated chickpea using RAPD and ISSR markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">643</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s001220100751" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s001220100751</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20in%20the%20genus%20Cicer%20and%20cultivated%20chickpea%20using%20RAPD%20and%20ISSR%20markers&amp;author=M.%20Iruela&amp;author=J.%20Rubio&amp;author=J.I.%20Cubero&amp;author=J.%20Gil&amp;author=T.%20Millan&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=643-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220100751" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220100751" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220100751"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Upadhyaya</div> <div class="given-names">H.D.</div></div>,  <div class="name"><div class="surname">Dwivedi</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Baum</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genetic structure, diversity, and allelic richness in composite collection and reference set in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">106</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-8-106" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-8-106</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%2C%20diversity%2C%20and%20allelic%20richness%20in%20composite%20collection%20and%20reference%20set%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=H.D.%20Upadhyaya&amp;author=S.L.%20Dwivedi&amp;author=M.%20Baum&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol.&amp;volume=8&amp;pages=106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18922189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robertson</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Ocampo</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">K.B.</div></div>. </div><div class="article-title">Morphological variation in wild annual <em>Cicer</em> species in comparison to the cultigens</div>, <div class="source">Euphytica</div> , <div class="year">1997</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">309</div>-<div class="lpage">19</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1003004516921" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/A:1003004516921</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Morphological%20variation%20in%20wild%20annual%20Cicer%20species%20in%20comparison%20to%20the%20cultigens&amp;author=L.D.%20Robertson&amp;author=B.%20Ocampo&amp;author=K.B.%20Singh&amp;publication_year=1997&amp;journal=Euphytica&amp;volume=95&amp;pages=309-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1003004516921" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1003004516921" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1003004516921"> </span></div> </div></div></div></div></div><div content-id="DST015C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ahmad</div> <div class="given-names">F.</div></div>. </div><div class="article-title">A comparative study of chromosome morphology among the nine annual species of <em>Cicer</em> L</div>, <div class="source">Cytobios</div> , <div class="year">2000</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">37</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20study%20of%20chromosome%20morphology%20among%20the%20nine%20annual%20species%20of%20Cicer%20L&amp;author=F.%20Ahmad&amp;publication_year=2000&amp;journal=Cytobios&amp;volume=101&amp;pages=37-53" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10697744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kazan</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Muehlbauer</div> <div class="given-names">F.J.</div></div>. </div><div class="article-title">Allozyme variation and phylogeny in annual species of <em>Cicer</em> (Leguminosae)</div>, <div class="source">Plant Syst. Evol.</div> , <div class="year">1991</div>, vol. <div class="volume">175</div> (pg. <div class="fpage">11</div>-<div class="lpage">21</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF00942142" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF00942142</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allozyme%20variation%20and%20phylogeny%20in%20annual%20species%20of%20Cicer%20(Leguminosae)&amp;author=K.%20Kazan&amp;author=F.J.%20Muehlbauer&amp;publication_year=1991&amp;journal=Plant%20Syst.%20Evol.&amp;volume=175&amp;pages=11-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00942142" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00942142" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00942142"> </span></div> </div></div></div></div></div><div content-id="DST015C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Toker</div> <div class="given-names">C.</div></div>. </div><div class="article-title">A note on the evolution of <em>kabuli</em> chickpeas as shown by induced mutations in <em>Cicer reticulatum</em> Ladizinsky</div>, <div class="source">Genet. Resour. Crop Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">7</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10722-008-9336-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10722-008-9336-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20note%20on%20the%20evolution%20of%20kabuli%20chickpeas%20as%20shown%20by%20induced%20mutations%20in%20Cicer%20reticulatum%20Ladizinsky&amp;author=C.%20Toker&amp;publication_year=2009&amp;journal=Genet.%20Resour.%20Crop%20Evol.&amp;volume=56&amp;pages=7-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-008-9336-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-008-9336-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-008-9336-8"> </span></div> </div></div></div></div></div><div content-id="DST015C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Berger</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">N.C.</div></div>. </div><div class="article-title">Evolution of cultivated chickpea: four bottlenecks limit diversity and constrain adaptation</div>, <div class="source">Funct. Plant Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1081</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1071/FP03084" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1071/FP03084</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20cultivated%20chickpea%3A%20four%20bottlenecks%20limit%20diversity%20and%20constrain%20adaptation&amp;author=S.%20Abbo&amp;author=J.%20Berger&amp;author=N.C.%20Turner&amp;publication_year=2003&amp;journal=Funct.%20Plant%20Biol.&amp;volume=30&amp;pages=1081-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/FP03084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FFP03084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FFP03084"> </span></div> </div></div></div></div></div><div content-id="DST015C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lev-Yadun</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gopher</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The cradle of agriculture</div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">288</div> (pg. <div class="fpage">1602</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.288.5471.1602" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.288.5471.1602</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cradle%20of%20agriculture&amp;author=S.%20Lev-Yadun&amp;author=A.%20Gopher&amp;author=S.%20Abbo&amp;publication_year=2000&amp;journal=Science&amp;volume=288&amp;pages=1602-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.288.5471.1602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.288.5471.1602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.288.5471.1602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10858140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berger</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">N.C.</div></div>. </div><div class="article-title">Ecogeography of annual wild <em>Cicer</em> species: the poor state of the world collection</div>, <div class="source">Crop Sci.</div> , <div class="year">2003</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">1076</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2135/cropsci2003.1076" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2135/cropsci2003.1076</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ecogeography%20of%20annual%20wild%20Cicer%20species%3A%20the%20poor%20state%20of%20the%20world%20collection&amp;author=J.%20Berger&amp;author=S.%20Abbo&amp;author=N.C.%20Turner&amp;publication_year=2003&amp;journal=Crop%20Sci.&amp;volume=43&amp;pages=1076-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2003.1076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2003.1076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2003.1076"> </span></div> </div></div></div></div></div><div content-id="DST015C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanno</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Willcox</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The origins of cultivation of <em>Cicer arietinum</em> L. and <em>Vicia faba</em> L.: early ﬁnds from Tell el-Kerkh, north-west Syria, late 10th millennium B.P</div>, <div class="source">Veg. Hist. Archaeobot.</div> , <div class="year">2006</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">197</div>-<div class="lpage">204</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00334-005-0027-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00334-005-0027-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origins%20of%20cultivation%20of%20Cicer%20arietinum%20L.%20and%20Vicia%20faba%20L.%3A%20early%20%EF%AC%81nds%20from%20Tell%20el-Kerkh%2C%20north-west%20Syria%2C%20late%2010th%20millennium%20B.P&amp;author=K.%20Tanno&amp;author=G.%20Willcox&amp;publication_year=2006&amp;journal=Veg.%20Hist.%20Archaeobot.&amp;volume=15&amp;pages=197-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00334-005-0027-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00334-005-0027-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00334-005-0027-5"> </span></div> </div></div></div></div></div><div content-id="DST015C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choumane</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Weigand</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kahl</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Conservation and variability of sequence-tagged microsatellites (STMSs) from chickpea (<em>Cicer aerietinum</em> L.) within the genus <em>Cicer</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">269</div>-<div class="lpage">78</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s001220051479" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s001220051479</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20variability%20of%20sequence-tagged%20microsatellites%20(STMSs)%20from%20chickpea%20(Cicer%20aerietinum%20L.)%20within%20the%20genus%20Cicer&amp;author=W.%20Choumane&amp;author=P.%20Winter&amp;author=F.%20Weigand&amp;author=G.%20Kahl&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=101&amp;pages=269-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051479"> </span></div> </div></div></div></div></div><div content-id="DST015C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Genetics of flowering time in chickpea and its bearing on productivity in semiarid environments</div>, <div class="source">Adv. Agron.</div> , <div class="year">2001</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">107</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0065-2113(01)72012-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0065-2113(01)72012-3</a></div></div></div></div></div><div content-id="DST015C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berger</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Buck</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Henzell</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">N.C.</div></div>. </div><div class="article-title">Evolution in the genus <em>Cicer</em> vernalisation response and low temperature pod set in chickpea (<em>C. arietinum</em> L.) and its annual wild relatives</div>, <div class="source">Aust. J. Agric. Res.</div> , <div class="year">2005</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">1191</div>-<div class="lpage">200</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1071/AR05089" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1071/AR05089</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20in%20the%20genus%20Cicer%20vernalisation%20response%20and%20low%20temperature%20pod%20set%20in%20chickpea%20(C.%20arietinum%20L.)%20and%20its%20annual%20wild%20relatives&amp;author=J.D.%20Berger&amp;author=R.%20Buck&amp;author=J.M.%20Henzell&amp;author=N.C.%20Turner&amp;publication_year=2005&amp;journal=Aust.%20J.%20Agric.%20Res.&amp;volume=56&amp;pages=1191-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/AR05089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FAR05089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FAR05089"> </span></div> </div></div></div></div></div><div content-id="DST015C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Xing</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Natural variation in <em>GS5</em> plays an important role in regulating grain size and yield in rice</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">1266</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng.977" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng.977</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20variation%20in%20GS5%20plays%20an%20important%20role%20in%20regulating%20grain%20size%20and%20yield%20in%20rice&amp;author=Y.%20Li&amp;author=C.%20Fan&amp;author=Y.%20Xing&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=1266-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.977" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.977" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.977"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22019783" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakiroglu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sherman-Broyles</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Story</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Patterns of linkage disequilibrium and association mapping in diploid alfalfa (<em>M. sativa</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">577</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-012-1854-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-012-1854-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20linkage%20disequilibrium%20and%20association%20mapping%20in%20diploid%20alfalfa%20(M.%20sativa%20L.)&amp;author=M.%20Sakiroglu&amp;author=S.%20Sherman-Broyles&amp;author=A.%20Story&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=125&amp;pages=577-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1854-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1854-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1854-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22476875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xiao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Genetic structure and linkage disequilibrium pattern of a rapeseed (<em>Brassica napus</em> L.) association mapping panel revealed by microsatellites</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">437</div>-<div class="lpage">47</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-012-1843-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-012-1843-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20linkage%20disequilibrium%20pattern%20of%20a%20rapeseed%20(Brassica%20napus%20L.)%20association%20mapping%20panel%20revealed%20by%20microsatellites&amp;author=Y.%20Xiao&amp;author=D.%20Cai&amp;author=W.%20Yang&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=125&amp;pages=437-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1843-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1843-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1843-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22437490" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hou</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Association analysis of vegetable soybean quality traits with SSR markers</div>, <div class="source">Plant Breed.</div> , <div class="year">2011</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">413</div>-<div class="lpage">506</div>)</div></div></div></div><div content-id="DST015C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zein</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">E.A.</div></div>, et al. </div><div class="article-title">Polymorphisms in monolignol biosynthetic genes are associated with biomass yield and agronomic traits in European maize (<em>Zea mays</em> L.)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">12</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-12" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-10-12</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymorphisms%20in%20monolignol%20biosynthetic%20genes%20are%20associated%20with%20biomass%20yield%20and%20agronomic%20traits%20in%20European%20maize%20(Zea%20mays%20L.)&amp;author=Y.%20Chen&amp;author=I.%20Zein&amp;author=E.A.%20Brenner&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=12-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20078869" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kharabian-Masouleh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Waters</div> <div class="given-names">D.L.E.</div></div>,  <div class="name"><div class="surname">Reinke</div> <div class="given-names">R.F.</div></div>,  <div class="name"><div class="surname">Ward</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Henry</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">SNP in starch biosynthesis genes associated with nutritional and functional properties of rice</div>, <div class="source">Sci. Rep.</div> , <div class="year">2012</div>, vol. <div class="volume">2</div> pg. <div class="fpage">557</div> <div class="pub-id-doi"><a href="https://doi.org/10.1038/srep00557" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/srep00557</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20in%20starch%20biosynthesis%20genes%20associated%20with%20nutritional%20and%20functional%20properties%20of%20rice&amp;author=A.%20Kharabian-Masouleh&amp;author=D.L.E.%20Waters&amp;author=R.F.%20Reinke&amp;author=R.%20Ward&amp;author=R.J.%20Henry&amp;publication_year=2012&amp;journal=Sci.%20Rep.&amp;volume=2&amp;pages=557" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep00557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep00557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep00557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22870386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mao</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Linking differential domain functions of the GS3 protein to natural variation of grain size in rice</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">19579</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.1014419107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.1014419107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linking%20differential%20domain%20functions%20of%20the%20GS3%20protein%20to%20natural%20variation%20of%20grain%20size%20in%20rice&amp;author=H.%20Mao&amp;author=S.%20Sun&amp;author=J.%20Yao&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=19579-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1014419107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1014419107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1014419107"> </span></div> </div></div></div></div></div><div content-id="DST015C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zheng</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Stromberg</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Perry</div> <div class="given-names">S.E.</div></div>. </div><div class="article-title">Global identification of targets of the <em>Arabidopsis</em> MADS domain protein AGAMOUS-Like15</div>, <div class="source">Plant Cell</div> , <div class="year">2009</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">2563</div>-<div class="lpage">77</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.109.068890" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.109.068890</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20identification%20of%20targets%20of%20the%20Arabidopsis%20MADS%20domain%20protein%20AGAMOUS-Like15&amp;author=Y.%20Zheng&amp;author=N.%20Ren&amp;author=H.%20Wang&amp;author=A.J.%20Stromberg&amp;author=S.E.%20Perry&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=2563-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.068890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.068890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.068890"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19767455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Le</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bui</div> <div class="given-names">A.Q.</div></div>, et al. </div><div class="article-title">Global analysis of gene activity during <em>Arabidopsis</em> seed development and identification of seed-specific transcription factors</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">8063</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.1003530107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.1003530107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20gene%20activity%20during%20Arabidopsis%20seed%20development%20and%20identification%20of%20seed-specific%20transcription%20factors&amp;author=B.H.%20Le&amp;author=C.%20Cheng&amp;author=A.Q.%20Bui&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=8063-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1003530107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1003530107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1003530107"> </span></div> </div></div></div></div></div><div content-id="DST015C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Benko-Iseppon</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Huttel</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">A linkage map of chickpea (<em>Cicer arietinum</em> L.) genome based on recombinant inbred lines from a <em>C. arietinum</em> × <em>C. reticulatum</em> cross: localization of resistance genes for <em>Fusarium</em> wilt races 4 and 5</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">1155</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s001220051592" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s001220051592</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)%20genome%20based%20on%20recombinant%20inbred%20lines%20from%20a%20C.%20arietinum%20%C3%97%20C.%20reticulatum%20cross%3A%20localization%20of%20resistance%20genes%20for%20Fusarium%20wilt%20races%204%20and%205&amp;author=P.%20Winter&amp;author=A.M.%20Benko-Iseppon&amp;author=B.%20Huttel&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=101&amp;pages=1155-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051592" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051592" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051592"> </span></div> </div></div></div></div></div><div content-id="DST015C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Corke</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Microsatellites in starch-synthesizing genes in relation to starch physicochemical properties in waxy rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">898</div>-<div class="lpage">905</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-002-1049-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-002-1049-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20in%20starch-synthesizing%20genes%20in%20relation%20to%20starch%20physicochemical%20properties%20in%20waxy%20rice%20(Oryza%20sativa%20L.)&amp;author=J.S.%20Bao&amp;author=H.%20Corke&amp;author=M.%20Sun&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=105&amp;pages=898-905" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-1049-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-1049-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-1049-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582915" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dresselhaus</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Cordts</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Heuer</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Novel ribosomal genes from maize are differentially expressed in the zygotic and somatic cell cycles</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">416</div>-<div class="lpage">27</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s004380050983" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s004380050983</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20ribosomal%20genes%20from%20maize%20are%20differentially%20expressed%20in%20the%20zygotic%20and%20somatic%20cell%20cycles&amp;author=T.%20Dresselhaus&amp;author=S.%20Cordts&amp;author=S.%20Heuer&amp;publication_year=1999&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=261&amp;pages=416-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380050983" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380050983" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380050983"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10102378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zuo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Conservation of noncoding microsatellites in plants: implication for gene regulation</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">323</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-7-323" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-7-323</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20noncoding%20microsatellites%20in%20plants%3A%20implication%20for%20gene%20regulation&amp;author=L.%20Zhang&amp;author=K.%20Zuo&amp;author=F.%20Zhang&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-323"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17187690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamasaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kigawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Inoue</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A novel zinc-binding motif revealed by solution structures of DNA-binding domains of <em>Arabidopsis</em> SBP-family transcription factors</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">337</div> (pg. <div class="fpage">49</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.jmb.2004.01.015" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.jmb.2004.01.015</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20zinc-binding%20motif%20revealed%20by%20solution%20structures%20of%20DNA-binding%20domains%20of%20Arabidopsis%20SBP-family%20transcription%20factors&amp;author=K.%20Yamasaki&amp;author=T.%20Kigawa&amp;author=M.%20Inoue&amp;publication_year=2004&amp;journal=J.%20Mol.%20Biol.&amp;volume=337&amp;pages=49-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2004.01.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2004.01.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2004.01.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15001351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fornara</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Parenicova</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Falasca</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Functional characterization of <em>OsMADS18</em>, a member of the <em>AP1/SQUA</em> subfamily of MADS box genes</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">2207</div>-<div class="lpage">19</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.045039" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.104.045039</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20characterization%20of%20OsMADS18%2C%20a%20member%20of%20the%20AP1%2FSQUA%20subfamily%20of%20MADS%20box%20genes&amp;author=F.%20Fornara&amp;author=L.%20Parenicova&amp;author=G.%20Falasca&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=2207-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.045039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.045039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.045039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15299121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C76" class="ref-content "><div class="label -label">76</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robles</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Pelaz</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Flower and fruit development in <em>Arabidopsis thaliana</em></div>, <div class="source">Int. J. Dev. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">633</div>-<div class="lpage">43</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1387/ijdb.052020pr" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1387/ijdb.052020pr</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flower%20and%20fruit%20development%20in%20Arabidopsis%20thaliana&amp;author=P.%20Robles&amp;author=S.%20Pelaz&amp;publication_year=2005&amp;journal=Int.%20J.%20Dev.%20Biol.&amp;volume=49&amp;pages=633-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1387/ijdb.052020pr" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1387%2Fijdb.052020pr" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1387%2Fijdb.052020pr"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16096970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C77" class="ref-content "><div class="label -label">77</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Schwarz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Saedler</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Huijser</div> <div class="given-names">P.</div></div>. </div><div class="article-title">SPL8, a local regulator in a subset of gibberellin-mediated developmental processes in Arabidopsis</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">429</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-006-9099-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11103-006-9099-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPL8%2C%20a%20local%20regulator%20in%20a%20subset%20of%20gibberellin-mediated%20developmental%20processes%20in%20Arabidopsis&amp;author=Y.%20Zhang&amp;author=S.%20Schwarz&amp;author=H.%20Saedler&amp;author=P.%20Huijser&amp;publication_year=2006&amp;journal=Plant%20Mol.%20Biol.&amp;volume=63&amp;pages=429-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-006-9099-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-006-9099-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-006-9099-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17093870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C78" class="ref-content "><div class="label -label">78</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Xiong</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Genomic organization, differential expression, and interaction of SQUAMOSA promoter-binding-like transcription factors and micro-RNA156 in rice</div>, <div class="source">Plant Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">142</div> (pg. <div class="fpage">280</div>-<div class="lpage">93</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.106.084475" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.106.084475</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20organization%2C%20differential%20expression%2C%20and%20interaction%20of%20SQUAMOSA%20promoter-binding-like%20transcription%20factors%20and%20micro-RNA156%20in%20rice&amp;author=K.%20Xie&amp;author=C.%20Wu&amp;author=L.%20Xiong&amp;publication_year=2006&amp;journal=Plant%20Physiol.&amp;volume=142&amp;pages=280-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.084475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.084475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.084475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16861571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C79" class="ref-content "><div class="label -label">79</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">A.Y.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Q.H.</div></div>,  <div class="name"><div class="surname">Gu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Genome-wide identiﬁcation and evolutionary analysis of the plant specific SBP-box transcription factor family</div>, <div class="source">Gene</div> , <div class="year">2008</div>, vol. <div class="volume">418</div> (pg. <div class="fpage">1</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gene.2008.03.016" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gene.2008.03.016</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identi%EF%AC%81cation%20and%20evolutionary%20analysis%20of%20the%20plant%20specific%20SBP-box%20transcription%20factor%20family&amp;author=A.Y.%20Guo&amp;author=Q.H.%20Zhu&amp;author=X.%20Gu&amp;publication_year=2008&amp;journal=Gene&amp;volume=418&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.03.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.03.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.03.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18495384" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C80" class="ref-content "><div class="label -label">80</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manning</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tor</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Poole</div> <div class="given-names">M.Y.</div></div>, et al. </div><div class="article-title">A naturally occurring epigenetic mutation in a gene encoding an SBP-box transcription factor inhibits tomato fruit ripening</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">948</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng1841" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng1841</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20naturally%20occurring%20epigenetic%20mutation%20in%20a%20gene%20encoding%20an%20SBP-box%20transcription%20factor%20inhibits%20tomato%20fruit%20ripening&amp;author=K.%20Manning&amp;author=M.%20Tor&amp;author=M.Y.%20Poole&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=948-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16832354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C81" class="ref-content "><div class="label -label">81</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flint-Garcia</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Thornsberry</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">Structure of linkage disequilibrium in plants</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">357</div>-<div class="lpage">74</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.arplant.54.031902.134907" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.arplant.54.031902.134907</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20linkage%20disequilibrium%20in%20plants&amp;author=S.A.%20Flint-Garcia&amp;author=J.M.%20Thornsberry&amp;author=E.S.%20Buckler&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=54&amp;pages=357-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.54.031902.134907" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.54.031902.134907" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.54.031902.134907"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14502995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C82" class="ref-content "><div class="label -label">82</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">P.K.</div></div>,  <div class="name"><div class="surname">Rustgi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kulwal</div> <div class="given-names">P.L.</div></div>. </div><div class="article-title">Linkage disequilibrium and association studies in higher plants: present status and future prospects</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">461</div>-<div class="lpage">85</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-005-0257-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11103-005-0257-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20disequilibrium%20and%20association%20studies%20in%20higher%20plants%3A%20present%20status%20and%20future%20prospects&amp;author=P.K.%20Gupta&amp;author=S.%20Rustgi&amp;author=P.L.%20Kulwal&amp;publication_year=2005&amp;journal=Plant%20Mol.%20Biol.&amp;volume=57&amp;pages=461-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-005-0257-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-005-0257-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-005-0257-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15821975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST015C83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C83" class="ref-content "><div class="label -label">83</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amos</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Driscol</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Hoffman</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">Candidate genes versus genome-wide associations: which are better for detecting genetic susceptibility to infectious disease?</div>, <div class="source">Proc. R Soc. B</div> , <div class="year">2011</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">1183</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1098/rspb.2010.1920" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1098/rspb.2010.1920</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Candidate%20genes%20versus%20genome-wide%20associations%3A%20which%20are%20better%20for%20detecting%20genetic%20susceptibility%20to%20infectious%20disease%3F&amp;author=W.%20Amos&amp;author=E.%20Driscol&amp;author=J.I.%20Hoffman&amp;publication_year=2011&amp;journal=Proc.%20R%20Soc.%20B&amp;volume=278&amp;pages=1183-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.2010.1920" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2010.1920" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frspb.2010.1920"> </span></div> </div></div></div></div></div><div content-id="DST015C84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST015C84" class="ref-content "><div class="label -label">84</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Konishi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Izawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">S.Y.</div></div>, et al. </div><div class="article-title">An SNP caused loss of seed shattering during rice domestication</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">1392</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1126410" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1126410</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20SNP%20caused%20loss%20of%20seed%20shattering%20during%20rice%20domestication&amp;author=S.%20Konishi&amp;author=T.%20Izawa&amp;author=S.Y.%20Lin&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=1392-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1126410" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1126410" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1126410"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16614172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353793&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_table4.pdf?Expires=2147483647&amp;Signature=ibNPPSCSSIswkdQuKlJ0zP56K3f5niFhYhkgwzJUZxGv~ohUw5U5Gy8HM4gI5jBKZ28lBUplH3K0~~h68qKx0bZuBAmQx9UvHXQR1Jkmsiem6HvZonHmK3DuSdH8a6o7av~SQe5vXD5WGoAzbTS1xmvylpKaMUGH1srF4q6aWEmYEKkFN9RwEZhD-AIQ0LFHc1t4Hg-HDW8ba~wdRY0-rsxO5AaljFoQX9QEsvnt86E06OwniE56QWZspPsyh71VqkigaeIqcX2Xx0fBHUsc2794RwW-wEZLN9mo8r5Xi3-qUmBVwiMGHqmI7z-A6FfqXBHBqnA0XUxvAUtWthMqEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-14</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_table3.pdf?Expires=2147483647&amp;Signature=AfBusI5IHe963--vNFTtcFdPjhuEB5Fduu-IMbb~x8NesFdL3OJ4Q0rz7x6hebT80T6tWLjDyBYaedTu269Ps8tSazHGI-bXn3VDuoC~e-eIQe3HUbx3sFujV-SDzwNEh7o~rgBxquxga77sLG8NF9ffEyP0JdALhwq7T7-tawKVPuoXQ68j7yoaAjjMylyF9lJgiaIVBG3P1s7DZQKOVOTVAbUVveidp~LL2QrClUksU4sN4YgNU3v9a9ej0bt~dQSIQBfJgrHxDjhIqQ0uBP-KExUXh6hMNUtU0DOa43z4SRhcGi-NNnGyGufM5YUyagZ5qCfqDH06KGAMZWWncw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_table2.pdf?Expires=2147483647&amp;Signature=pEGcWsO836NE4tE5vSJtaHhQ178XC~vDLAsBfzUKgOrPZO44Go~0fgO3qMav4UQhI~c~4zb11gTSfJ2xCztTb223ESa0zxt5yZExrJqH~gZbh3Sc-2nPMVWBWf6XX8-SVgrzeCoHrmdkf89QvPXDF-QEDTp2oSH-7FeXUdOn7fuA~eUfo4T1RKINwwAzNTIWuwjaQFAVCcGi3-0VNjLwK-1jkXLMxJXo8WcEl8Py~2IBNiEGwPW2u26IGqpK0iOYeO-op0tyaEDQDb1UWAi05i0quYZxLmrsqrEr-HPgSbV1Tv0cAcI3p9t3AxhXTP7WhxkR4nuFzeN8UDDvZrAJhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_table1.pdf?Expires=2147483647&amp;Signature=ghZsJw1IRhFjei~ijruc8cxPuh1tDjM1L8bGjWYBDP8wnxX5M7iRUyroQUrW6m8cC95PMDuiDeTIoiDPOJhFMu3GKsPCfRIs42Cd6GoS~vDzyg6en7FVVfAzTgcRuQ19NdF4zLUaygpoFb9JNQWSQNQlLIQ-X-YvdiRpOy0m6ZBwI6bXP~7BX~BdYlpqi1HP9x~UtTQxS03EcoqntMyhh2-fTxFUE9qPR3mCN8zIz8cLdWl-XQ7XgE0Vjf~puwUuy1JEcpgic9S1N4WemEN3Vx8yEk2p1GFZZ7Oge5tcvw-DLiD755T~ybnsZYBlPcMhqDUdKb6lKWNQxkOGd28Uag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig9.png?Expires=2147483647&amp;Signature=uQu5YHBf22pTiJwXvGTWdEL5DtpzSFGI2ipOVel2PSGGJbFwwkOD1FohUckUlw~SpAfmYKz7tWlZycWm3HQofu-SppjkzAkI7biaJohdTTibLP3~eVxpUslfj13~MKFTuPnl2EULunFUEdRg34NWq7xoU3Ew5sc4rJJm2znQCgZBwvDpOvIxRZ7bLudvDnYaVYE-IUAtNkuWLtsGldV36bjawOzYIbIBcaWpCzlQyZ0POAiT5CwB2l8KLQBqMvawwnxLUpNXzUh0DT6w5MpvuSByOtCuc~m8Z7qeoQwU4p2vmwltB8ci6humaE7Jtl9OloF1DiAwKoXJXRiaiVxUVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig8.png?Expires=2147483647&amp;Signature=sts-qIWUni0n-9E1mKOSG3Dx3xYmmoCbnnTf4D8HRe56YvtpzMT~8sVJQS5fEWErnySfpQIilHO-CXGsoPjZSxNETRmP~U7k7iwj-vOehOrcGRqG0NRT1CDconQBeMEYaxUXesEuyhx9Y~E-ZW7V-Sk7iyTnVpoK-XPnNz5XzaEJWfh9P-3DqIL4BO82XfdPB3acNjLLm03X5jOle8UKgiQ7~4BQCu9vdzaMeZWig~2DGCM41kgW24HY9LmS7auT3pFmQ4eGjHWtCR-zigLMJJtDv3V-Pkm7i51gLp~-gHnLDmBu8WeRElImmzlCD6YI4tk7hJYLor94GfdW5Zq59A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig7.png?Expires=2147483647&amp;Signature=CCJXWfMHa4ult8yy6Uv6lCy13J-TezStgPQnTsDUYiF65SZdBwvzf2yNV1xsOqxu319~lWTnD4cs3Hq92D3gXf8HbF9FwmyZLdjXBbTFPNgIBGhoT0DaRnzhn~UqJcmSkthgv3vMGuAJzUz01CNAKS-1yNSo5t06EgNb-ff54YEmmaOfKvvvvTWABVlyrFkfmPVzUbZ1P0udF3dac24nWI8lo0kKE9sA0faQjA6wIPOuOQSL72pb4xPyjVnT7gXebQfdM0CHtf3zggffFaitmlNO31G4Jnx9nX8HiJjpiW9OS6bqfznf0CkWVf1BeoP0ysVXsMDvadEkYvygj-IVOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig6.png?Expires=2147483647&amp;Signature=42AizLR9zdiMoQgtKNTmJKMG7lqYXDDCgkspwD5cUSs1cSDdM2BJkO2j1Ll2ZAkqKb9tQTL8bZWpPh~8vTegxfbsEPlUH-0RD5j3b50JcKXGHbiYpYmwnSxZpPX~s35ViDC3rm8X-nL0dUy5MAkyJIPBWLmP03hFXpVFH-geceFpvCqFsbs5UEfgqJ6eV-bjlhm3rZQJNP3kmntxkEPignmZU-RMqmOg~pCLSA4qDj38Fc~9eE1KStiUvM6nJMQnFS~KauE~7l-AJNpxi26bLRbsK9IakiBbcFDl78PNTPecGSQWzopGgBdHcUEPbRDGT0fdDtb3249fn7juDmxFBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig5.png?Expires=2147483647&amp;Signature=ibBFvZkKPvpu1VGiF87VpM9IcPIoAA-24djU4UkVswxJFkCnPeAgMCvLYiA1IdU4oyxydR9IpmFJtFmpbqjJ0F2DCiGaccRiLUQ5JVQjYBfyFGnIh6dSwVUVPR-ruO9WpH0Xbujl2K4AgIHEUuM6MkubE2MhSAIbbaJcDX-gnNkxzk4tHnXi0xmWCvHdwjYyJd-iCIny0s0Oj8AnWsi2osv235d5~gTB9HzwmJA6xZw7WfibZRf9xqphMI-ZbBzhC5shDHmSXm8bxU1XZ9Wws~EK-sDqLSj2kWCuXyp5bDWSUmqC1nPHsUH~uLqwqEm0CT09ABOJGgTplU39D95aIw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig4.png?Expires=2147483647&amp;Signature=MX~RDhNDukSG~MIovc7ZqdKCfFmfLSlZhmW23IfZ1DgLueZ6ky-KjMkQFUgsAwK4HrzvYkm~9Pnh~3IjdjKrC1kRrqd6BE4TIQVTuvkOvnMN4PgKkiMwsjPB9VeH9Tx-lSmhtq7H7QmbSoM4C2izX6-mZVhdZ~nfJ7g0nCIFCDRRADaCB4qanxPweT8NrxYMz2Q97KF0rf5JfVmwuI9JPqBDPeEjoMjJj~y2M8Q2UixjCXWgTX2TKSG6~17jLxND9~4xQGtl3Fd7P5t7nVrbGCjZfov~i85nNVlBFjyFYfqY4rIEXLBVZhfsgcE8m5IZvhWk9S4uNUFPAM7~ltOpKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig3.png?Expires=2147483647&amp;Signature=FjLXnIhHfKRTDUseaXqsp4hTsYBrxKy~iqYQWFl~ANJIslqwFGhLF0D094n7EZSali8qWgUvevR~1m0I4GZy30KqvPqs~KdLTsVhUAW5aTym3y00U~xa9YPVuKpdDxj3yMKQH17bgmqrqIBm~4huhb5LNcikqDYMT81~HwxTi~vhVXwNnkY-0VN3ESJh-hmsxQX6qHe4G2FDq~xkVz~L~kKOODU6MKYBg5VJj7vPsAYeu1gvLr7HfydYobF3D0pl1Q5vMHqPyHY2YVGqU8lfV5ReqKFWvkEGXXs0frbxgJ-Y4O2rqiHJ4sL1ygyd3VEuFSdOtGkICXEfe7bogLwp2w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig2.png?Expires=2147483647&amp;Signature=mlHetFrcuEHy7KL687CCUqmGM1Zvsidec7nfzJJMAIBIrsKkPMrYbLKBtYJueTytkT5-gr~hkpIhXIQmq0Wzaq27Yeqqs2y5j6kfQO0mEI~Mag-nE0nrD6W4cDWfCzj6FEtOO0a4yUc-OTV-faM~K~6VA3c9m2bxhStEYy-VbSe8aBjNcSyvMnag~vdhdPQN8Dimdzkoh0zaZPYMPxX9bFzDigt5smO1nfPp2ebBWYw3Q0uXLdvFH7BbXLp5kHIFihEEtnmFi0rA7te-sijc8WsLwsrx8V0sBPKe0cZPR0~0uE5~wF1qgncO3Rxlv-5mjCFAlMY5PtafHRIFXHUwnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig10.png?Expires=2147483647&amp;Signature=4rlsAHnZ2UZRQzaH580tM2aCeJuLTLMSENxiUzEXbWda5QzUK8BOazu42Ip3k7Wbmsg~kMjNIAE-tukivIW2dgLaAR1-asgNzklvrGGgDi1O18QiPmXX2qMsCoX7aHTCUO1~qWreyu5r4foqjpYMGWc9nGzTjOVE0h-Y-jKCl5lsMbR-C8xcEYHUmqrC~fCv-k9rUpIo1eP0vS8SHGy2PPG72U6aBIgo9uYJS8lsjHCrvad2a9lTJ2niSyRghuWtQPE4aHHOIGxOXJ5hjvB30UlY94nUH9RcOjSvYXmfKVsIX8zJuHgsD-F2COy98RYzgu~sQh5~UQgTUY6Y9eqPwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst015/2/dst015-dst015supp_fig1.png?Expires=2147483647&amp;Signature=shEcoNx958za~Psn4G56klWpWakmI0zskBUCuRnVuc41ubkwEE8HNlSDREdzuHID~KjYoXfB8oDLwnbKZ2zFq~hXiEI0K-d1xDISNitgeUokMngV9qsGPQH7aoKy6JlK9uBHjI3V-clIVElcLI3opd6XjXf2q~0pfGvD~mMZrey60Qri6U3dK9QKDzrtpSr4P0TgCjH34A~MXOomyenPB7G3DDAjMOdyPcsSO4vpdyWwbgXrHvexPCdvdKcb5zWSDT91l0yZU0Sspqy9Bor83LM08EC0YUfpqs9j9IPuByPibq-E0JkCFIshiuyz2RkmusVMoAo6ewqFqqNN8ljCXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - png file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">348</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000324223800004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">63</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353793"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000324223800004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f4%2f355%2f353793">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000324223800004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (63)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F4%2F355%2F353793" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353793'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Functionally Relevant Microsatellite Markers From Chickpea Transcription Factor Genes for Efficient Genotyping Applications and Trait Association Mapping" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f355%2f353793">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

