

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":2148,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/3/169/353612","@type":"ScholarlyArticle","name":"Characterization of Transcription Start Sites of Putative Non-coding RNAs by Multifaceted Use of Massively Paralleled Sequencer","datePublished":"2010-04-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2010-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq007","keywords":["non-coding RNA","integrated transcriptome analysis","transcriptional start site cluster (TSC)","intergenic transcript","antisense transcript"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/3/169/353612","author":[{"name":"Sathira, Nuankanya","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568 ,  Japan","@type":"Person"},{"name":"Yamashita, Riu","affiliation":"2  Human Genome Center, Institute of Medical Science ,  The University of Tokyo ,  4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639 ,  Japan","@type":"Person"},{"name":"Tanimoto, Kousuke","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568 ,  Japan","@type":"Person"},{"name":"Kanai, Akinori","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568 ,  Japan","@type":"Person"},{"name":"Arauchi, Takako","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568 ,  Japan","@type":"Person"},{"name":"Kanematsu, Soutaro","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568 ,  Japan","@type":"Person"},{"name":"Nakai, Kenta","affiliation":"2  Human Genome Center, Institute of Medical Science ,  The University of Tokyo ,  4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639 ,  Japan","@type":"Person"},{"name":"Suzuki, Yutaka","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568 ,  Japan","@type":"Person"},{"name":"Sugano, Sumio","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568 ,  Japan","@type":"Person"}],"description":"Abstract.  On the basis of integrated transcriptome analysis, we show that not all transcriptional start site clusters (TSCs) in the intergenic regions (iTSCs) ","pageStart":"169","pageEnd":"183","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Characterization of Transcription Start Sites of Putative Non-coding RNAs by Multifaceted Use of Massively Paralleled Sequencer","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Characterization of Transcription Start Sites of Putative Non-coding RNAs by Multifaceted Use of Massively Paralleled Sequencer" />
    <meta property="og:description" content="Abstract.  On the basis of integrated transcriptome analysis, we show that not all transcriptional start site clusters (TSCs) in the intergenic regions (iTSCs) " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/3/169/353612" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-04-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sathira, Nuankanya" /><meta name="citation_author_institution" content="Department of Medical Genome Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568, Japan" /><meta name="citation_author" content="Yamashita, Riu" /><meta name="citation_author_institution" content="Human Genome Center, Institute of Medical Science, The University of Tokyo, 4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639, Japan" /><meta name="citation_author" content="Tanimoto, Kousuke" /><meta name="citation_author_institution" content="Department of Medical Genome Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568, Japan" /><meta name="citation_author" content="Kanai, Akinori" /><meta name="citation_author_institution" content="Department of Medical Genome Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568, Japan" /><meta name="citation_author" content="Arauchi, Takako" /><meta name="citation_author_institution" content="Department of Medical Genome Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568, Japan" /><meta name="citation_author" content="Kanematsu, Soutaro" /><meta name="citation_author_institution" content="Department of Medical Genome Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568, Japan" /><meta name="citation_author" content="Nakai, Kenta" /><meta name="citation_author_institution" content="Human Genome Center, Institute of Medical Science, The University of Tokyo, 4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639, Japan" /><meta name="citation_author" content="Suzuki, Yutaka" /><meta name="citation_author_institution" content="Department of Medical Genome Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568, Japan" /><meta name="citation_author" content="Sugano, Sumio" /><meta name="citation_author_institution" content="Department of Medical Genome Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568, Japan" /><meta name="citation_title" content="Characterization of Transcription Start Sites of Putative Non-coding RNAs by Multifaceted Use of Massively Paralleled Sequencer" /><meta name="citation_firstpage" content="169" /><meta name="citation_lastpage" content="183" /><meta name="citation_doi" content="10.1093/dnares/dsq007" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2010/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=5′-end SAGE for the analysis of transcriptional start sites; Nat. Biotechnol.;  citation_year=2004;  citation_volume=22;  citation_pages=1146-9; " /><meta name="citation_reference" content="citation_title=The transcriptional landscape of the mammalian genome; Science;  citation_year=2005;  citation_volume=309;  citation_pages=1559-63; " /><meta name="citation_reference" content="citation_title=Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project; Nature;  citation_year=2007;  citation_volume=447;  citation_pages=799-816; " /><meta name="citation_reference" content="citation_title=Global identification of human transcribed sequences with genome tiling arrays; Science;  citation_year=2004;  citation_volume=306;  citation_pages=2242-6; " /><meta name="citation_reference" content="citation_title=Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals; Nature;  citation_year=2009;  citation_volume=458;  citation_pages=223-7; " /><meta name="citation_reference" content="citation_title=A systematic search for new mammalian noncoding RNAs indicates little conserved intergenic transcription; BMC Genomics;  citation_year=2005;  citation_volume=6;  citation_pages=104" /><meta name="citation_reference" content="citation_title=The transcriptional activity of human chromosome 22; Genes Dev.;  citation_year=2003;  citation_volume=17;  citation_pages=529-40; " /><meta name="citation_reference" content="citation_title=Widespread occurrence of antisense transcription in the human genome; Nat. Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=379-86; " /><meta name="citation_reference" content="citation_title=Systematic identification of sense-antisense transcripts in mammalian cells; Nat. Biotechnol.;  citation_year=2004;  citation_volume=22;  citation_pages=104-8; " /><meta name="citation_reference" content="citation_title=Functionality of intergenic transcription: an evolutionary comparison; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=e171" /><meta name="citation_reference" content="citation_title=Genome mapping and expression analyses of human intronic noncoding RNAs reveal tissue-specific patterns and enrichment in genes related to regulation of transcription; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R43" /><meta name="citation_reference" content="citation_title=Identification of putative noncoding RNAs among the RIKEN mouse full-length cDNA collection; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=1301-6; " /><meta name="citation_reference" content="citation_title=Experimental validation of the regulated expression of large numbers of non-coding RNAs from the mouse genome; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=11-9; " /><meta name="citation_reference" content="citation_title=The enigmatic world of mRNA-like ncRNAs: their role in human evolution and in human diseases; Semin. Cancer Biol.;  citation_year=2008;  citation_volume=18;  citation_pages=141-8; " /><meta name="citation_reference" content="citation_title=Long, abundantly expressed non-coding transcripts are altered in cancer; Hum. Mol. Genet.;  citation_year=2008;  citation_volume=17;  citation_pages=642-55; " /><meta name="citation_reference" content="citation_title=Specific expression of long noncoding RNAs in the mouse brain; Proc. Natl Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=716-21; " /><meta name="citation_reference" content="citation_title=Evolution and functions of long noncoding RNAs; Cell;  citation_year=2009;  citation_volume=136;  citation_pages=629-41; " /><meta name="citation_reference" content="citation_title=Functionality or transcriptional noise? Evidence for selection within long noncoding RNAs; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=556-65; " /><meta name="citation_reference" content="citation_title=Filtering transcriptional noise during development: concepts and mechanisms; Nat. Rev. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=34-44; " /><meta name="citation_reference" content="citation_title=Transcriptional noise and the fidelity of initiation by RNA polymerase II; Nat. Struct. Mol. Biol.;  citation_year=2007;  citation_volume=14;  citation_pages=103-5; " /><meta name="citation_reference" content="citation_title=Waste not, want not–transcript excess in multicellular eukaryotes; Trends Genet.;  citation_year=2005;  citation_volume=21;  citation_pages=287-8; " /><meta name="citation_reference" content="citation_title=Non-coding RNAs: hope or hype?; Trends Genet.;  citation_year=2005;  citation_volume=21;  citation_pages=289-97; " /><meta name="citation_reference" content="citation_title=Characterization of evolutionary rates and constraints in three mammalian genomes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=539-48; " /><meta name="citation_reference" content="citation_title=Distribution and intensity of constraint in mammalian genomic sequence; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=901-13; " /><meta name="citation_reference" content="citation_title=Mouse transcriptome: neutral evolution of ‘non-coding’ complementary DNAs; citation_text=1 p following 757; discussion following 757; Nature;  citation_year=2004;  citation_volume=431; " /><meta name="citation_reference" content="citation_title=DBTSS: database of transcription start sites progress report; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D97-101; " /><meta name="citation_reference" content="citation_title=Construction of a full-length enriched and a 5′-end enriched cDNA library using the oligo-capping method; Methods Mol. Biol.;  citation_year=2003;  citation_volume=221;  citation_pages=73-91; " /><meta name="citation_reference" content="citation_title=Complete sequencing and characterization of 21,243 full-length human cDNAs; Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=40-5; " /><meta name="citation_reference" content="citation_title=Whole-genome re-sequencing; Curr. Opin. Genet. Dev.;  citation_year=2006;  citation_volume=16;  citation_pages=545-52; " /><meta name="citation_reference" content="citation_title=Massive transcriptional start site analysis of human genes in hypoxia cells; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=2249-63; " /><meta name="citation_reference" content="citation_title=Dynamic regulation of nucleosome positioning in the human genome; Cell;  citation_year=2008;  citation_volume=132;  citation_pages=887-98; " /><meta name="citation_reference" content="citation_title=Translational and rotational settings of H2A.Z nucleosomes across the Saccharomyces cerevisiae genome; Nature;  citation_year=2007;  citation_volume=446;  citation_pages=572-6; " /><meta name="citation_reference" content="citation_title=Genome-wide mapping of in vivo protein-DNA interactions; Science;  citation_year=2007;  citation_volume=316;  citation_pages=1497-502; " /><meta name="citation_reference" content="citation_title=Divergent transcription from active promoters; Science;  citation_year=2008;  citation_volume=322;  citation_pages=1849-51; " /><meta name="citation_reference" content="citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=621-8; " /><meta name="citation_reference" content="citation_title=Ripples from neighbouring transcription; Nat. Cell Biol.;  citation_year=2008;  citation_volume=10;  citation_pages=1106-13; " /><meta name="citation_reference" content="citation_title=Predicting human nucleosome occupancy from primary sequence; PLoS Comput. Biol.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000134" /><meta name="citation_reference" content="citation_title=Nucleosome positioning and gene regulation: advances through genomics; Nat. Rev. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=161-72; " /><meta name="citation_reference" content="citation_title=High-throughput mapping of the chromatin structure of human promoters; Nat. Biotechnol.;  citation_year=2007;  citation_volume=25;  citation_pages=244-8; " /><meta name="citation_reference" content="citation_title=Diverse transcriptional initiation revealed by fine, large-scale mapping of mRNA start sites; EMBO Rep.;  citation_year=2001;  citation_volume=2;  citation_pages=388-93; " /><meta name="citation_reference" content="citation_title=miRBase: tools for microRNA genomics; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D154-8; " /><meta name="citation_reference" content="citation_title=Sno/scaRNAbase: a curated database for small nucleolar RNAs and cajal body-specific RNAs; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D183-7; " /><meta name="citation_reference" content="citation_title=The H19 non-coding RNA is essential for human tumor growth; PLoS One;  citation_year=2007;  citation_volume=2;  citation_pages=e845" /><meta name="citation_reference" content="citation_title=The imprinted H19 noncoding RNA is a primary microRNA precursor; RNA;  citation_year=2007;  citation_volume=13;  citation_pages=313-6; " /><meta name="citation_reference" content="citation_title=The gene ontology (GO) project in 2006; citation_author=GeneOntologyConsortium; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D322-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/3/169/1011285/dsq007.pdf" /><meta name="description" content="Abstract.  On the basis of integrated transcriptome analysis, we show that not all transcriptional start site clusters (TSCs) in the intergenic regions (iTSCs) " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/3/169/353612" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/3/169/353612" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/3/169/353612");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq007';
    </script>
<title>
	Characterization of Transcription Start Sites of Putative Non-coding RNAs by Multifaceted Use of Massively Paralleled Sequencer | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353612" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BchmPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRoaXJhLCBOdWFua2FueWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1lZGljYWwgR2Vub21lIFNjaWVuY2VzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCA1LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YS1zaGksIENoaWJhIDI3Ny04NTY4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFzaGl0YSwgUml1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSHVtYW4gR2Vub21lIENlbnRlciwgSW5zdGl0dXRlIG9mIE1lZGljYWwgU2NpZW5jZSwgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDQtNi0xIFNoaXJva2FuZWRhaSwgTWluYXRvLWt1LCBUb2t5byAxMDgtODYzOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYW5pbW90bywgS291c3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01IEthc2hpd2Fub2hhLCBLYXNoaXdhLXNoaSwgQ2hpYmEgMjc3LTg1NjgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FuYWksIEFraW5vcmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1lZGljYWwgR2Vub21lIFNjaWVuY2VzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCA1LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YS1zaGksIENoaWJhIDI3Ny04NTY4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFyYXVjaGksIFRha2FrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01IEthc2hpd2Fub2hhLCBLYXNoaXdhLXNoaSwgQ2hpYmEgMjc3LTg1NjgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FuZW1hdHN1LCBTb3V0YXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIEdlbm9tZSBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2Etc2hpLCBDaGliYSAyNzctODU2OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthaSwgS2VudGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJIdW1hbiBHZW5vbWUgQ2VudGVyLCBJbnN0aXR1dGUgb2YgTWVkaWNhbCBTY2llbmNlLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNC02LTEgU2hpcm9rYW5lZGFpLCBNaW5hdG8ta3UsIFRva3lvIDEwOC04NjM5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1enVraSwgWXV0YWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIEdlbm9tZSBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2Etc2hpLCBDaGliYSAyNzctODU2OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdWdhbm8sIFN1bWlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIEdlbm9tZSBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2Etc2hpLCBDaGliYSAyNzctODU2OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNoYXJhY3Rlcml6YXRpb24gb2YgVHJhbnNjcmlwdGlvbiBTdGFydCBTaXRlcyBvZiBQdXRhdGl2ZSBOb24tY29kaW5nIFJOQXMgYnkgTXVsdGlmYWNldGVkIFVzZSBvZiBNYXNzaXZlbHkgUGFyYWxsZWxlZCBTZXF1ZW5jZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxNjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE4MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcTAwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEwLzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9NeKAsi1lbmQgU0FHRSBmb3IgdGhlIGFuYWx5c2lzIG9mIHRyYW5zY3JpcHRpb25hbCBzdGFydCBzaXRlczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTExNDYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRpb25hbCBsYW5kc2NhcGUgb2YgdGhlIG1hbW1hbGlhbiBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA5OyAgY2l0YXRpb25fcGFnZXM9MTU1OS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGFuYWx5c2lzIG9mIGZ1bmN0aW9uYWwgZWxlbWVudHMgaW4gMSUgb2YgdGhlIGh1bWFuIGdlbm9tZSBieSB0aGUgRU5DT0RFIHBpbG90IHByb2plY3Q7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDc7ICBjaXRhdGlvbl9wYWdlcz03OTktODE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgaWRlbnRpZmljYXRpb24gb2YgaHVtYW4gdHJhbnNjcmliZWQgc2VxdWVuY2VzIHdpdGggZ2Vub21lIHRpbGluZyBhcnJheXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA2OyAgY2l0YXRpb25fcGFnZXM9MjI0Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbWF0aW4gc2lnbmF0dXJlIHJldmVhbHMgb3ZlciBhIHRob3VzYW5kIGhpZ2hseSBjb25zZXJ2ZWQgbGFyZ2Ugbm9uLWNvZGluZyBSTkFzIGluIG1hbW1hbHM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NTg7ICBjaXRhdGlvbl9wYWdlcz0yMjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzeXN0ZW1hdGljIHNlYXJjaCBmb3IgbmV3IG1hbW1hbGlhbiBub25jb2RpbmcgUk5BcyBpbmRpY2F0ZXMgbGl0dGxlIGNvbnNlcnZlZCBpbnRlcmdlbmljIHRyYW5zY3JpcHRpb247IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRpb25hbCBhY3Rpdml0eSBvZiBodW1hbiBjaHJvbW9zb21lIDIyOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NTI5LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlc3ByZWFkIG9jY3VycmVuY2Ugb2YgYW50aXNlbnNlIHRyYW5zY3JpcHRpb24gaW4gdGhlIGh1bWFuIGdlbm9tZTsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM3OS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBpZGVudGlmaWNhdGlvbiBvZiBzZW5zZS1hbnRpc2Vuc2UgdHJhbnNjcmlwdHMgaW4gbWFtbWFsaWFuIGNlbGxzOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTA0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWxpdHkgb2YgaW50ZXJnZW5pYyB0cmFuc2NyaXB0aW9uOiBhbiBldm9sdXRpb25hcnkgY29tcGFyaXNvbjsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUxNzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgbWFwcGluZyBhbmQgZXhwcmVzc2lvbiBhbmFseXNlcyBvZiBodW1hbiBpbnRyb25pYyBub25jb2RpbmcgUk5BcyByZXZlYWwgdGlzc3VlLXNwZWNpZmljIHBhdHRlcm5zIGFuZCBlbnJpY2htZW50IGluIGdlbmVzIHJlbGF0ZWQgdG8gcmVndWxhdGlvbiBvZiB0cmFuc2NyaXB0aW9uOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPVI0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHB1dGF0aXZlIG5vbmNvZGluZyBSTkFzIGFtb25nIHRoZSBSSUtFTiBtb3VzZSBmdWxsLWxlbmd0aCBjRE5BIGNvbGxlY3Rpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTMwMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBlcmltZW50YWwgdmFsaWRhdGlvbiBvZiB0aGUgcmVndWxhdGVkIGV4cHJlc3Npb24gb2YgbGFyZ2UgbnVtYmVycyBvZiBub24tY29kaW5nIFJOQXMgZnJvbSB0aGUgbW91c2UgZ2Vub21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTExLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlbmlnbWF0aWMgd29ybGQgb2YgbVJOQS1saWtlIG5jUk5BczogdGhlaXIgcm9sZSBpbiBodW1hbiBldm9sdXRpb24gYW5kIGluIGh1bWFuIGRpc2Vhc2VzOyBTZW1pbi4gQ2FuY2VyIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTQxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmcsIGFidW5kYW50bHkgZXhwcmVzc2VkIG5vbi1jb2RpbmcgdHJhbnNjcmlwdHMgYXJlIGFsdGVyZWQgaW4gY2FuY2VyOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NjQyLTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGVjaWZpYyBleHByZXNzaW9uIG9mIGxvbmcgbm9uY29kaW5nIFJOQXMgaW4gdGhlIG1vdXNlIGJyYWluOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTcxNi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIGFuZCBmdW5jdGlvbnMgb2YgbG9uZyBub25jb2RpbmcgUk5BczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzY7ICBjaXRhdGlvbl9wYWdlcz02MjktNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWxpdHkgb3IgdHJhbnNjcmlwdGlvbmFsIG5vaXNlPyBFdmlkZW5jZSBmb3Igc2VsZWN0aW9uIHdpdGhpbiBsb25nIG5vbmNvZGluZyBSTkFzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTU1Ni02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmlsdGVyaW5nIHRyYW5zY3JpcHRpb25hbCBub2lzZSBkdXJpbmcgZGV2ZWxvcG1lbnQ6IGNvbmNlcHRzIGFuZCBtZWNoYW5pc21zOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zNC00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbmFsIG5vaXNlIGFuZCB0aGUgZmlkZWxpdHkgb2YgaW5pdGlhdGlvbiBieSBSTkEgcG9seW1lcmFzZSBJSTsgTmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMDMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2FzdGUgbm90LCB3YW50IG5vdOKAk3RyYW5zY3JpcHQgZXhjZXNzIGluIG11bHRpY2VsbHVsYXIgZXVrYXJ5b3RlczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTI4Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob24tY29kaW5nIFJOQXM6IGhvcGUgb3IgaHlwZT87IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yODktOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgZXZvbHV0aW9uYXJ5IHJhdGVzIGFuZCBjb25zdHJhaW50cyBpbiB0aHJlZSBtYW1tYWxpYW4gZ2Vub21lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01MzktNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3RyaWJ1dGlvbiBhbmQgaW50ZW5zaXR5IG9mIGNvbnN0cmFpbnQgaW4gbWFtbWFsaWFuIGdlbm9taWMgc2VxdWVuY2U7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9OTAxLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3VzZSB0cmFuc2NyaXB0b21lOiBuZXV0cmFsIGV2b2x1dGlvbiBvZiDigJhub24tY29kaW5n4oCZIGNvbXBsZW1lbnRhcnkgRE5BczsgY2l0YXRpb25fdGV4dD0xIHAgZm9sbG93aW5nIDc1NzsgZGlzY3Vzc2lvbiBmb2xsb3dpbmcgNzU3OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EQlRTUzogZGF0YWJhc2Ugb2YgdHJhbnNjcmlwdGlvbiBzdGFydCBzaXRlcyBwcm9ncmVzcyByZXBvcnQ7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQ5Ny0xMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGZ1bGwtbGVuZ3RoIGVucmljaGVkIGFuZCBhIDXigLItZW5kIGVucmljaGVkIGNETkEgbGlicmFyeSB1c2luZyB0aGUgb2xpZ28tY2FwcGluZyBtZXRob2Q7IE1ldGhvZHMgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjE7ICBjaXRhdGlvbl9wYWdlcz03My05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgc2VxdWVuY2luZyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiAyMSwyNDMgZnVsbC1sZW5ndGggaHVtYW4gY0ROQXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIHJlLXNlcXVlbmNpbmc7IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTQ1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXNzaXZlIHRyYW5zY3JpcHRpb25hbCBzdGFydCBzaXRlIGFuYWx5c2lzIG9mIGh1bWFuIGdlbmVzIGluIGh5cG94aWEgY2VsbHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTIyNDktNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWMgcmVndWxhdGlvbiBvZiBudWNsZW9zb21lIHBvc2l0aW9uaW5nIGluIHRoZSBodW1hbiBnZW5vbWU7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9ODg3LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbmFsIGFuZCByb3RhdGlvbmFsIHNldHRpbmdzIG9mIEgyQS5aIG51Y2xlb3NvbWVzIGFjcm9zcyB0aGUgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0NjsgIGNpdGF0aW9uX3BhZ2VzPTU3Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBtYXBwaW5nIG9mIGluIHZpdm8gcHJvdGVpbi1ETkEgaW50ZXJhY3Rpb25zOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxNjsgIGNpdGF0aW9uX3BhZ2VzPTE0OTctNTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcmdlbnQgdHJhbnNjcmlwdGlvbiBmcm9tIGFjdGl2ZSBwcm9tb3RlcnM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzIyOyAgY2l0YXRpb25fcGFnZXM9MTg0OS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBhbmQgcXVhbnRpZnlpbmcgbWFtbWFsaWFuIHRyYW5zY3JpcHRvbWVzIGJ5IFJOQS1TZXE7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NjIxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpcHBsZXMgZnJvbSBuZWlnaGJvdXJpbmcgdHJhbnNjcmlwdGlvbjsgTmF0LiBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTEwNi0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyBodW1hbiBudWNsZW9zb21lIG9jY3VwYW5jeSBmcm9tIHByaW1hcnkgc2VxdWVuY2U7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAxMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW9zb21lIHBvc2l0aW9uaW5nIGFuZCBnZW5lIHJlZ3VsYXRpb246IGFkdmFuY2VzIHRocm91Z2ggZ2Vub21pY3M7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNjEtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBtYXBwaW5nIG9mIHRoZSBjaHJvbWF0aW4gc3RydWN0dXJlIG9mIGh1bWFuIHByb21vdGVyczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI0NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNlIHRyYW5zY3JpcHRpb25hbCBpbml0aWF0aW9uIHJldmVhbGVkIGJ5IGZpbmUsIGxhcmdlLXNjYWxlIG1hcHBpbmcgb2YgbVJOQSBzdGFydCBzaXRlczsgRU1CTyBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zODgtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW1pUkJhc2U6IHRvb2xzIGZvciBtaWNyb1JOQSBnZW5vbWljczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDE1NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tbm8vc2NhUk5BYmFzZTogYSBjdXJhdGVkIGRhdGFiYXNlIGZvciBzbWFsbCBudWNsZW9sYXIgUk5BcyBhbmQgY2FqYWwgYm9keS1zcGVjaWZpYyBSTkFzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1EMTgzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBIMTkgbm9uLWNvZGluZyBSTkEgaXMgZXNzZW50aWFsIGZvciBodW1hbiB0dW1vciBncm93dGg7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lODQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGltcHJpbnRlZCBIMTkgbm9uY29kaW5nIFJOQSBpcyBhIHByaW1hcnkgbWljcm9STkEgcHJlY3Vyc29yOyBSTkE7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zMTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbmUgb250b2xvZ3kgKEdPKSBwcm9qZWN0IGluIDIwMDY7IGNpdGF0aW9uX2F1dGhvcj1HZW5lT250b2xvZ3lDb25zb3J0aXVtOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1EMzIyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNy8zLzE2OS8xMDExMjg1L2RzcTAwNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgT24gdGhlIGJhc2lzIG9mIGludGVncmF0ZWQgdHJhbnNjcmlwdG9tZSBhbmFseXNpcywgd2Ugc2hvdyB0aGF0IG5vdCBhbGwgdHJhbnNjcmlwdGlvbmFsIHN0YXJ0IHNpdGUgY2x1c3RlcnMgKFRTQ3MpIGluIHRoZSBpbnRlcmdlbmljIHJlZ2lvbnMgKGlUU0NzKSAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1MzYxMiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRQNvyCsCml4RoIRkhgBxZt/Pcwbw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f169%2f353612" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f169%2f353612" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13687" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&amp;Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/3">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5033966">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5033966">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5033968">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5033968">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5033975">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5033975">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034003">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034003">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034081">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034081">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034084">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034084">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034086">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034086">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5034088">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5034088">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/3/155/353741">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/3/185/354288">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Characterization of Transcription Start Sites of Putative Non-coding RNAs by Multifaceted Use of Massively Paralleled Sequencer
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nuankanya Sathira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Genome Sciences, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nuankanya+Sathira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sathira N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sathira N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nuankanya Sathira</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Riu Yamashita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Human Genome Center, Institute of Medical Science</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Riu+Yamashita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamashita R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamashita R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Riu Yamashita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kousuke Tanimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Genome Sciences, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kousuke+Tanimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanimoto K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanimoto K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kousuke Tanimoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akinori Kanai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Genome Sciences, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akinori+Kanai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanai A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanai A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akinori Kanai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takako Arauchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Genome Sciences, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takako+Arauchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arauchi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arauchi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takako Arauchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Soutaro Kanematsu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Genome Sciences, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Soutaro+Kanematsu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanematsu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanematsu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Soutaro Kanematsu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Nakai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Human Genome Center, Institute of Medical Science</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Nakai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakai K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakai K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenta Nakai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Genome Sciences, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 4-7136-3607. Fax. <div class="fax">+81 4-7136-3607</div>. E-mail: <a href="mailto:ysuzuki@hgc.jp" target="_blank">ysuzuki@hgc.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yutaka Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sumio Sugano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Genome Sciences, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa-shi, Chiba 277-8568</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sumio+Sugano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugano S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugano S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sumio Sugano</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 3, 1 June 2010, Pages 169–183, <a href='https://doi.org/10.1093/dnares/dsq007'>https://doi.org/10.1093/dnares/dsq007</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 April 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 December 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 February 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353612" href="/dnaresearch/article-pdf/17/3/169/1011285/dsq007.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Nuankanya Sathira, Riu Yamashita, Kousuke Tanimoto, Akinori Kanai, Takako Arauchi, Soutaro Kanematsu, Kenta Nakai, Yutaka Suzuki, Sumio Sugano;  Characterization of Transcription Start Sites of Putative Non-coding RNAs by Multifaceted Use of Massively Paralleled Sequencer, <em>DNA Research</em>, Volume 17, Issue 3, 1 June 2010, Pages 169–183, <a href='https://doi.org/10.1093/dnares/dsq007'>https://doi.org/10.1093/dnares/dsq007</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353612?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353612?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353612?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353612?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353612?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353612?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353612?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Characterization%20of%20Transcription%20Start%20Sites%20of%20Putative%20Non-coding%20RNAs%20by%20Multifaceted%20Use%20of%20Massively%20Paralleled%20Sequencer&amp;publicationDate=2010-04-17&amp;volumeNum=17&amp;issueNum=3&amp;author=Sathira%2C%20Nuankanya%3B%20Yamashita%2C%20Riu&amp;startPage=169&amp;endPage=183&amp;contentId=10.1093%2Fdnares%2Fdsq007&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f169%2f353612" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5033966 id="5033966" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>On the basis of integrated transcriptome analysis, we show that not all transcriptional start site clusters (TSCs) in the intergenic regions (iTSCs) have the same properties; thus, it is possible to discriminate the iTSCs that are likely to have biological relevance from the other noise-level iTSCs. We used a total of 251 933 381 short-read sequence tags generated from various types of transcriptome analyses in order to characterize 6039 iTSCs, which have significant expression levels. We analyzed and found that 23% of these iTSCs were located in the proximal regions of the RefSeq genes. These RefSeq-linked iTSCs showed similar expression patterns with the neighboring RefSeq genes, had widely fluctuating transcription start sites and lacked ordered nucleosome positioning. These iTSCs seemed not to form independent transcriptional units, simply representing the by-products of the neighboring RefSeq genes, in spite of their significant expression levels. Similar features were also observed for the TSCs located in the antisense regions of the RefSeq genes. Furthermore, for the remaining iTSCs that were not associated with any RefSeq genes, we demonstrate that integrative interpretation of the transcriptome data provides essential information to specify their biological functions in the hypoxic responses of the cells.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;non-coding RNA&quot;">non-coding RNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;integrated transcriptome analysis&quot;">integrated transcriptome analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptional start site cluster (TSC)&quot;">transcriptional start site cluster (TSC)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;intergenic transcript&quot;">intergenic transcript</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;antisense transcript&quot;">antisense transcript</a></div>                            <h2 scrollto-destination=5033968 id="5033968" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Since human genome and several other organisms' genomes are completely sequenced, gene regulation is realized to be more immense and more complex than expected. The functional properties are known not only on protein-coding sequences but also on non-coding or untranslated sequences. Recent cDNA studies<sup><a reveal-id="DSQ007C1" data-open="DSQ007C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ007C2" data-open="DSQ007C2" class="link link-ref link-reveal xref-bibr">2</a></sup> suggested that transcription start sites (TSSs) are widespread throughout the human genomes,<sup><a reveal-id="DSQ007C3" data-open="DSQ007C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSQ007C4" data-open="DSQ007C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and ENCODE consortium reported that major part of such TSSs should produce long non-coding RNAs (lncRNAs),<sup><a reveal-id="DSQ007C3" data-open="DSQ007C3" class="link link-ref link-reveal xref-bibr">3</a></sup> especially from intergenic regions.<sup><a reveal-id="DSQ007C4  DSQ007C7" data-open="DSQ007C4  DSQ007C7" class="link link-ref link-reveal xref-bibr">4–7</a></sup> Furthermore, a large number of putatively lncRNAs were discovered from the antisense regions of the protein-coding genes.<sup><a reveal-id="DSQ007C8" data-open="DSQ007C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ007C9" data-open="DSQ007C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Although tens of thousands of non-coding RNAs (ncRNAs) have been identified in mammalian cells,<sup><a reveal-id="DSQ007C10  DSQ007C13" data-open="DSQ007C10  DSQ007C13" class="link link-ref link-reveal xref-bibr">10–13</a></sup> their identification, regulation mechanisms and functional characterization are still incomplete.<sup><a reveal-id="DSQ007C14  DSQ007C17" data-open="DSQ007C14  DSQ007C17" class="link link-ref link-reveal xref-bibr">14–17</a></sup></p><p>With the growing number of the uncharacterized lncRNAs, concerns have been raised that they may be mere transcriptional noise of mammalian cells.<sup><a reveal-id="DSQ007C18  DSQ007C22" data-open="DSQ007C18  DSQ007C22" class="link link-ref link-reveal xref-bibr">18–22</a></sup> A large number of lncRNAs are not evolutionarily conserved, and their expression levels seem too low to imply any biological functions, frequently below the limit of the northern blotting analysis.<sup><a reveal-id="DSQ007C23  DSQ007C25" data-open="DSQ007C23  DSQ007C25" class="link link-ref link-reveal xref-bibr">23–25</a></sup> Nonetheless, a recent study demonstrated that the lncRNAs with potential biological significance could be selected by examining chromatin signatures,<sup><a reveal-id="DSQ007C5" data-open="DSQ007C5" class="link link-ref link-reveal xref-bibr">5</a></sup> proving that at least some of the lncRNAs do have biological relevance.</p><p>Although information about millions of TSSs has become available,<sup><a reveal-id="DSQ007C1" data-open="DSQ007C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ007C26" data-open="DSQ007C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSQ007C27" data-open="DSQ007C27" class="link link-ref link-reveal xref-bibr">27</a></sup> it is still insufficient to represent comprehensive overview of intergenic TSSs in a quantitative manner. In order to understand the nature of intergenic TSSs of unknown functions, further in-depth and wider variety of biological data, focusing on a particular cell type or a cellular response in a more quantitative manner, are essential.</p><p>We have been collecting and analyzing full-length cDNAs of human genes by utilizing our cap selection method, oligo-capping.<sup><a reveal-id="DSQ007C26  DSQ007C28" data-open="DSQ007C26  DSQ007C28" class="link link-ref link-reveal xref-bibr">26–28</a></sup> Recently, to increase the throughput of the cDNA sequencing, we combined oligo-capping with the massively paralleled sequencing technology, Illumina GA.<sup><a reveal-id="DSQ007C29" data-open="DSQ007C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In this method, named TSS Seq, sequence adaptor, which is necessary for Illumina GA sequencing, is directly introduced to the cap site of the mRNA (TSS tag).<sup><a reveal-id="DSQ007C30" data-open="DSQ007C30" class="link link-ref link-reveal xref-bibr">30</a></sup> In addition to TSS Seq, various high-throughput analyses have been enabled by using the massively paralleled sequencer. It is now possible to perform genome-wide analyses to examine nucleosome structure (Nucleosome Seq),<sup><a reveal-id="DSQ007C31" data-open="DSQ007C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSQ007C32" data-open="DSQ007C32" class="link link-ref link-reveal xref-bibr">32</a></sup> binding of a transcription factor (ChIP Seq),<sup><a reveal-id="DSQ007C33" data-open="DSQ007C33" class="link link-ref link-reveal xref-bibr">33</a></sup> complete cDNA sequences (cDNA shotgun), identification and characterization of RNAs in a particular subcellular locations or an experimental condition (RNA Seq)<sup><a reveal-id="DSQ007C34" data-open="DSQ007C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSQ007C35" data-open="DSQ007C35" class="link link-ref link-reveal xref-bibr">35</a></sup> with feasible cost and efforts.</p><p>In our previous study, we identified 371 849 intergenic TSS clusters (iTSCs) that were identified by the TSS Seq analysis of 12 human cell lines and tissues. We found that, among these TSS clusters, only 6039 are having significant expression levels (R. Yamashita <em>et al</em>., submitted). In this paper, we report our first integrative transcriptome analyses of the intergenic TSS using a total of 254 076 723 short-read sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). We show that not all intergenic TSSs have the same properties; thus, it is possible to classify them and discriminate intergenic TSSs which are likely to have biological relevance from the other noise-level TSSs (Fig. <a reveal-id="DSQ007F1" data-open="DSQ007F1" class="link link-reveal link-table xref-fig">1</a>).</p>                        <a id="5033974" scrollto-destination="5033974"></a>
<div id="DSQ007F1" data-id="DSQ007F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00701.jpeg?Expires=2147483647&Signature=GfKBbDdjrXWKN~qGXDcz8TEHZlBqSo1PQ3dnlmWFINX-UKpLoZSpf9OqizRWJfy9ES07nS3pyM7DXEqSJGkEPdE9wo8aeDfkKX5Q06wIKRKxgznES6T6LhL5SfnNC563jdsOC-RGcBk0wjEi3I1LRcitCrl-nV-Wcqj1RzxZ4sFPKW4A9xkRIXXDgF53twVSuMPZeg81d5unyWTNgQ2CNttLpsMUiiSZLfxT0CAYqh8KMfTbOGFkPCjz5NodSqgUEg85U12ntAPOgFVZQenkVD~m1qgO~ZfWihW2g8Iv2HxKMrrkdRei1NdAgy7V8v4CUesutYWX0Mi7s4muEe57fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification of the iTSCs. Schematic representation of the classification of the iTSCs is shown. Number of the iTSCs belonging to each category is shown in parenthesis. Characteristic features of each category of the iTSCs are shown in the margin. Analysis methods to identify them are also shown in the double-line box and the number of the sequence tags used is shown in the parenthesis." path-from-xml="dsq00701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5033974/dsq00701.jpeg" path-from-xml="dsq00701.jpeg" target="_blank">View large</a><a section="5033974" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00701.jpeg?Expires=2147483647&Signature=qC6TknYVFQNIE~jMXGE1fGxhs~0vCAylFySCZDeIAeyo1b75FWibqMs4rSgvuprDlPn-MbMjfo5mHeAZn9T-Z-SArSRa4PA8mj1SoidKfDZ3hskOhTmwFqhOBsY1rRcGFxDgu2nyI-W0qFNurU6WFwwMLf614FBZXQRB0LUxI0QZPSamptgDqK5zYiiUtwzS1EhpmWwg0T57~1r26HQr7WEeCIk6dNf2Y5IdeaWmsTar1nIsUYzgI1gH2QObsKg2UguyMxW2BBmA4evyEFF-u2f9qRgJfhvMpcsE26LOCoKFibAv8N9vemJAeiVC0P0atsF4o8xJxOGOUtYQeS00pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5033974&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification of the iTSCs. Schematic representation of the classification of the iTSCs is shown. Number of the iTSCs belonging to each category is shown in parenthesis. Characteristic features of each category of the iTSCs are shown in the margin. Analysis methods to identify them are also shown in the double-line box and the number of the sequence tags used is shown in the parenthesis.</p></div></div><div content-id="DSQ007F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00701.jpeg?Expires=2147483647&Signature=GfKBbDdjrXWKN~qGXDcz8TEHZlBqSo1PQ3dnlmWFINX-UKpLoZSpf9OqizRWJfy9ES07nS3pyM7DXEqSJGkEPdE9wo8aeDfkKX5Q06wIKRKxgznES6T6LhL5SfnNC563jdsOC-RGcBk0wjEi3I1LRcitCrl-nV-Wcqj1RzxZ4sFPKW4A9xkRIXXDgF53twVSuMPZeg81d5unyWTNgQ2CNttLpsMUiiSZLfxT0CAYqh8KMfTbOGFkPCjz5NodSqgUEg85U12ntAPOgFVZQenkVD~m1qgO~ZfWihW2g8Iv2HxKMrrkdRei1NdAgy7V8v4CUesutYWX0Mi7s4muEe57fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification of the iTSCs. Schematic representation of the classification of the iTSCs is shown. Number of the iTSCs belonging to each category is shown in parenthesis. Characteristic features of each category of the iTSCs are shown in the margin. Analysis methods to identify them are also shown in the double-line box and the number of the sequence tags used is shown in the parenthesis." path-from-xml="dsq00701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5033974/dsq00701.jpeg" path-from-xml="dsq00701.jpeg" target="_blank">View large</a><a section="5033974" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00701.jpeg?Expires=2147483647&Signature=qC6TknYVFQNIE~jMXGE1fGxhs~0vCAylFySCZDeIAeyo1b75FWibqMs4rSgvuprDlPn-MbMjfo5mHeAZn9T-Z-SArSRa4PA8mj1SoidKfDZ3hskOhTmwFqhOBsY1rRcGFxDgu2nyI-W0qFNurU6WFwwMLf614FBZXQRB0LUxI0QZPSamptgDqK5zYiiUtwzS1EhpmWwg0T57~1r26HQr7WEeCIk6dNf2Y5IdeaWmsTar1nIsUYzgI1gH2QObsKg2UguyMxW2BBmA4evyEFF-u2f9qRgJfhvMpcsE26LOCoKFibAv8N9vemJAeiVC0P0atsF4o8xJxOGOUtYQeS00pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5033974&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification of the iTSCs. Schematic representation of the classification of the iTSCs is shown. Number of the iTSCs belonging to each category is shown in parenthesis. Characteristic features of each category of the iTSCs are shown in the margin. Analysis methods to identify them are also shown in the double-line box and the number of the sequence tags used is shown in the parenthesis.</p></div></div>                            <h2 scrollto-destination=5033975 id="5033975" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5033976 id="5033976" class="section-title" data-legacy-id=s2a>2.1. Cell culture and tissues</h3>
<p>Human cell lines, DLD-1 cells (ATCC number: CCL-221), were maintained in Dulbecco's modified Eagle's medium (DMEM; Invitrogen) supplemented with 10% fetal calf serum, 4.5 g/l glucose and antibiotics. HEK293, MCF7, TIG3, BEAS2B and Ramos cells (ATCC number: CRL-1573, ATCC number: HTB-22 and Japan Cell Resource Bank number: JCRB0506) were cultured in standard conditions. For hypoxic induction, DLD-1, HEK 293, MCF7 and TIG3 cells were incubated in 1% O<sub>2</sub> and 5% CO<sub>2</sub> for 24 h. In order to get the RNA, six million cells were cultured and harvested for RNA extraction using RNeasy (Qiagen). The quality and quantity of the obtained total RNAs were assessed, using BioAnalyzer (Agilent). Human tissue RNAs (brain, heart, kidney, fetal brain, fetal heart, fetal kidney) were purchased from Clontech (catalog numbers: 636526, 636529, 636530, 636532, 636583 and 636584).</p>                            <h3 scrollto-destination=5033978 id="5033978" class="section-title" data-legacy-id=s2b>2.2. Construction of the TSS Seq libraries and analysis of the TSS tags</h3>
<p>Two hundred micrograms of the obtained total RNA were subjected to oligo-capping; after the successive treatment of the RNA with 2.5 U BAP (TaKaRa) at 37°C for 1 h and 40 U TAP (Ambion) at 37°C for 1 h, the BAP-TAP-treated RNAs were ligated with 1.2 μg of RNA oligo(5′-AAUGAUACGGCGACCACCGAGAUCUACACUCUUUCCCUACACGACGCUCUUCCGAUCUGG-3′) using 250 U T4 RNA ligase (TaKaRa) at 20°C for 3 h. After the DNase I treatment (TaKaRa), polyA+ RNA was selected using oligo-dT powder (Collaborative). First-strand cDNA was synthesized from 10 pmol of random hexamer primer (5′-CAAGCAGAAGACGGCATACGANNNNNNC-3′) using Super Script II (Invitrogen) by incubating at 12°C for 1 h and at 42°C overnight. Template RNA was degraded by alkaline treatment. For polymerase chain reaction (PCR), one-fifth of the first-strand cDNAs were used as the PCR template. Gene Amp PCR kits (PerkinElmer) were used with the PCR primers 5′-AATGATACGGCGACCACCGAG-3′ and 5′-CAAGCAGAAGACGGCATACGA-3′ under the following reaction conditions: 15 cycles of 94°C for 1 min, 56°C for 1 min and 72°C for 2 min. The PCR fragments were size fractionated by 12% polyacrylamide gel electrophoresis and the fraction of 150–250 bp was recovered. The quality and quantity of the obtained single-stranded first-strand cDNAs were assessed, using BioAnalyzer (Agilent). One nanogram of the size-fractionated cDNA was used for the sequencing reactions with the Illumina GA. The sequencing reactions were performed according to the manufacturer's instructions.</p><p>Thirty-six-base pair-read TSS tags were mapped onto the human genome sequence (hg18 as of UCSC Genome Browser) using ELAND. Uniquely and completely (without any mismatch) mapped TSS tags were used for the analysis. TSSs located from the 3′-end boundaries to −50 kb of the 5′-end boundaries of the adjoining RefSeq genes were selected. Then, the clustering of the intergenic TSS tags into 500-bp bin was performed. TSS clusters were removed when all the belonging TSS tags were located at the internal exonic region of the corresponding RefSeq genes. The RefSeq information as to the genomic coordinate, position of the protein-coding region and so on is as of hg18 (<a class="link link-uri" href="http://hgdownload.cse.ucsc.edu/goldenPath/hg18/">http://hgdownload.cse.ucsc.edu/goldenPath/hg18/</a>). Gene ontology terms (as of 1 June 2009) were correlated with RefSeq using loc2go GO (as of 1 June 2009) using NCBI Entrez Gene database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=gene">http://www.ncbi.nlm.nih.gov/sites/entrez?db=gene</a>).</p>                            <h3 scrollto-destination=5033981 id="5033981" class="section-title" data-legacy-id=s2c>2.3. Construction of the Nucleosome Seq library and analysis of the nucleosome tags</h3>
<p>DLD-1 cells were cultured at 80% confluency and were treated with micrococcus nuclease to generate mononucleosomes using ChIP-IT Express Enzymatic (Active motif). The cells were formaldehyde-cross-linked before isolating nucleosomes. Cross-linking was achieved with fixation solution (DMEM, 0.01% formaldehyde) for 10 min at room temperature. The cells were then washed with PBS, and cross-linking was stopped with Glycine Stop-Fix Solution (PBS, Glycine buffer) for 5 min at room temperature. After washing with PBS, ice-cold cell scraping solution was added to the dish and cells were harvested. The cells were re-suspended in ice-cold lysis buffer, were incubate on ice for 30 min and homogenized with 15–20 strokes with a pestle. The nuclei were suspended in digestion buffer and pre-warm this solution for 5 min at 37°C. Enzymatic cocktail dissolved (200 U/ml) was added to the sample and the digestion reaction was incubated at 37°C for 15 min. The reaction was stopped with EDTA, the nuclei were pelleted by centrifugation and the supernatants were collected. Formaldehyde cross-linking was reversed by adding 5 M NaCl, RNase and incubation at 65°C for over 4 h. Proteinase K was added and incubated at 42°C for 1.5 h. DNA was purified by phenol/chloroform extractions and ethanol precipitation. Using the extracted DNA, sample preparation for Illumina GA was made according to the manufacturer's instructions.</p>                            <h3 scrollto-destination=5033983 id="5033983" class="section-title" data-legacy-id=s2d>2.4. Shotgun sequencing of the cDNA</h3>
<p>Colony PCR was performed for the cDNA clones using the common PCR primers of 5′-TCAGTGGATGTTGCCTTTAC-3′ and 5′-TGTGGGAGGTTTTTTCTCTA-3′. Obtained PCR products were nebulized so that the average fragments size is 200–500 bp. Sample preparation for Illumina GA was made according to the manufacturer's instructions. Using the generated 36-bp sequence tags, cDNA sequences were assembled based on the mapping information to the human genome sequence. Details of the computational procedure for the assembling and their quality are described elsewhere. Since the quality of the base-call varied, only the boundary information of the exons was extracted and actual nucleotide sequences were replaced with those of the reference human genome.</p>                            <h3 scrollto-destination=5033985 id="5033985" class="section-title" data-legacy-id=s2e>2.5. Subcellular fractionation</h3>
<p>A total of 1 × 10<sup>8</sup> DLD-1 cells were incubated with the medium containing 0.1 mg/ml cycloheximide for 5 min at 37°C and washed with PBS (containing 0.1 mg/ml cycloheximide). Cell pellets were re-suspended in lysis buffer [20 mM Tris–HCl (pH 7.5), 150 mM NaCl, 15 mM MgCl<sub>2</sub>, 1% Triton X-100, 0.1 mg/ml cycloheximide, 0.1 mM dithiothreitol, RNase inhibitor, Complete Protease Inhibitor Cocktail (Roche)] and lysed on ice for 10 min. Lysate was separated into cytoplasmic fraction (supernatant) and nuclear fraction (pellet) by centrifugation. Nuclear pellet was re-suspended in lysis buffer, homogenized and lysed on ice for 10 min. A portion of cytoplasmic fraction was layered on the top of a 10-ml 15–50% (w/v) sucrose gradient and centrifuged at 100 000 g in Beckman SW41Ti rotor for 2 h 15 min at 4°C. Cytoplasmic and nuclear fractions were treated with 200 µg/ml proteinase K, and RNA was extracted by TRIzol LS (Invitrogen). Obtained RNA concentration was analyzed by 2100 Bioanalyzer (Agilent Technologies). For validation of the precise separation of the subcellular fractions, see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>.</p>                            <h3 scrollto-destination=5033987 id="5033987" class="section-title" data-legacy-id=s2f>2.6. Quantitative reverse transcription–PCR analysis</h3>
<p>Twenty kinds of normal human tissues were selected for expression analysis, both fetal and adult major organs, included adrenal gland, bone marrow, brain (whole), fetal brain, fetal liver, heart, kidney, liver, lung (whole), placenta, prostate, salivary gland, skeletal muscle, testis, thymus, thyroid gland, trachea, uterus, colon w/mucosa and spinal cord. Total RNAs were purchased from Clontech (Human total RNA Master Panel II). The qualitative assessment of all total RNA was done utilizing the Agilent 2100 Bioanalyzer. Total RNA (8 µg) from each sample was reverse transcribed using an oligo(dT) and the Superscript II reverse transcriptase kit (Invitrogen). Negative control samples (no first-strand synthesis) were prepared by performing reverse transcription reactions in the absence of reverse transcriptase. Expression profile was performed using the quantitative reverse transcription–PCR (qRT–PCR). Gene-specific primer pairs were designed using Primer3 software, with an optimal primer size of 20 bp, amplification size of 100–500 bp and annealing temperature of 55°C. The primers were purchased from Operon. Quantitative real-time PCR was carried out with 100 pg of total RNA per test well using the Power SYBR Green PCR Master Mix (Applied Biosystem). The PCR reactions were performed using an ABI 7900HT Fast Real-Time System (Applied Biosystems) using the following cycling protocols: 40 cycles of 15 s at 95°C and 60 s at 60°C. The threshold cycle (Ct) value was calculated from amplification plots, in which the fluorescence signal detected was plotted against the PCR cycle. We defined express sequence when the threshold cycle values (Ct) &lt;37 in the presence of reverse transcriptase, but not when reverse transcriptase was absent. The product size was checked by 2% agarose gel electrophoresis, and melting curves were analyzed to monitor the specificity of the PCR.</p>                            <h3 scrollto-destination=5033989 id="5033989" class="section-title" data-legacy-id=s2g>2.7. Computational procedures</h3>
<p>Overlap between the iTSCs and miRNA and snoRNA, from miRBase (<a class="link link-uri" href="http://microrna.sanger.ac.uk/sequences/index.shtml">http://microrna.sanger.ac.uk/sequences/index.shtml</a>) and snoRNABase (<a class="link link-uri" href="http://www-snorna.biotoul.fr/index.php">http://www-snorna.biotoul.fr/index.php</a>), respectively, were examined. Statistical significance of the difference in the sequence features around the iTSCs was evaluated by the indicated methods.</p><div class="block-child-p">Occupancy of the nucleosome was calculated according to the reference.<sup><a reveal-id="DSQ007C32" data-open="DSQ007C32" class="link link-ref link-reveal xref-bibr">32</a></sup> First, we defined the positions of nucleosome centers <em>i</em> by adding 75 bp from the 5′-end of each mapped nucleosome tag. The counts of nucleosome centers <em>c</em>(<em>i</em>) were converted to the nucleosome signals <em>s</em>(<em>p</em><sub><em>j</em></sub>) throughout the whole genome using the logarithm of a weighted average as follow:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq007ueq1.gif?Expires=2147483647&Signature=lwhrPsOoNu6ucBkI8Eh-zidW7Xw3iZeljWeWVaU~AMGQBnBZYw93asV0I2-lIDSAsFpDiEEjEi1kqr3Vgbog5zaVeEJB6XklJwsPL-l0w62dqcsVezgkVc6PR4tNjbAX~bCDM0KjYWQ8Lh0~8Sm6EG9KjHU3IgyMRMVlpM9qPdkEWtFndGuw~Ly3EsJWBTJgTYMGHUBFzCI7wWam0HerpiBz57YzlJkbX2A~P1MiumlqaYlM7JSl5GDAHh24q2mVG9d1pNaDGyRHcxVFenxqVjC0rl~txLg5Ni3fyLDuxdkQBL4GtO3dFKUXeQqtdnCQSMfzwjijuAND~5F9fipPbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsq007ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq007ueq1.gif?Expires=2147483647&Signature=C7eTxhFMiFcujcer4CqEa0oA-3PY6PkuVvw9wtIxu-iADgrGYGKQttnu7FCopfBvSWrBg0aRRiIgbkrpuFjwZtAH3vqf-mFWHDNoA21VemzuqvVs2xbkkDxg2NNJ5AxwYsgmWmveGtgIdWXJH~hLg1RI9Omqf~rhxZaTPjRIGzwsxHvwQvbesb2cES3j9He-90l4kZci8PgLdlc0dLxC4eVw~pq57ieIp-W7a5t0YeL1NvU5v5tpT9dsQraA4eHc8a8yPSFQK21yXfVrgc49BWuNJCvFAXoNoHDaaYv~q5sOiohCYEIYwqjvv7XYDOgKWPlGHoKkeWEyr8R8YerDug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq007ueq1.gif" /></a></div></div></div> where <em>p</em><sub><em>j</em></sub> = 5 + 10 <em>j</em> (<em>j</em> = 0, 1, 2, …) and <em>w</em>(<em>i</em>) is the Gaussian distribution with mean <em>p</em><sub><em>j</em></sub> and standard deviation 20.</div><div class="block-child-p">Enrichment of the RNA Seq tags in the respective subcellular fractions was similarly evaluated using the Poisson distribution:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq007ueq2.gif?Expires=2147483647&Signature=hYSZnVkYiG-Lmq6GYXDCWJvlb6j7SB9bIyfhHXMK57P7dtXi9iFAnZ8pojvyt80~UaVo7Lnjkl1kq6ldfDj29CtlIbfVFBVpj6UXM610GKpVBBmXV-LtZW6ZvBLHySyzcqw5hZlDwmHkhStMjSVeopGJQ6pxvJ-e6f9tWHqFM3M726LTeVRDK-TbBBXtSpSb6SzuPItbaOPV2seRqD4swxj2AhKJFIV35PugzyDDFx5C2mY7zPiQNfzes8s157-RJyq~0v7U1JaDaL2R3Zq2d-2uJj-dEM5ybwgIu7Jh1oaYdUT-mt~CsnnKwo3b7nnUV8Tfz9g5NmDAlI-Ki9IBmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsq007ueq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq007ueq2.gif?Expires=2147483647&Signature=Sl8Pfrrz31d0MmekaLP3jkxvw27zUp6RglT~V1FojB12a~nebKyavJZqXFe9DhqA7GuYCxv58cv0LoR5m0rHE8a7fiZpEnkFY9~NCFKp09r6FkqCsfrlSt0-jTQBVWEaOeVzXhsWigaIXMkTtojOM5aRKR9SYYI5hfLQaerb~qeWxMCHZzyy71L53YVymD0GEtR4QNtOAIR2dCWJ0YTPFEIjhnquqObs5Hslo4BZOZVYaTxP8VJbkHLbe7ZVLLD6ZNYTuJlC77QHZJsOhrYTE67kffitu09WzItf3abYtjonf1L46wFrjJtMu3lT9SsQx2MMS~K8jYmfeucIIVAArg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq007ueq2.gif" /></a></div></div></div> where <em>p</em>(<em>x</em>, λ) is the probability of enrichment, λ the expected tag number in the cytoplasmic fraction for each gene based on the tag number in the total fraction, and <em>x</em> the observed tag number in the cytoplasmic fraction for each gene.</div>                            <h2 scrollto-destination=5034003 id="5034003" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5034005 id="5034005" class="section-title" data-legacy-id=s3a>3.1. Identification and initial characterization of iTSCs</h3>
<p>Schematic representation of the TSS Seq and statistics of the 139 446 730 thirty-six-base pair TSS tags, which were collected from 12 cell lines and tissues of humans and used in this study, are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> (GenBank accession number for each of the TSS tag data set is also shown there). All of the TSS information is publicly available from our database (<a class="link link-uri" href="http://dbtss.hgc.jp">http://dbtss.hgc.jp</a>). The TSS tags that were mapped to the intergenic regions, from 3′-end boundaries to −50 kb of the 5′-end boundaries of the adjoining RefSeq genes (Fig. <a reveal-id="DSQ007F2" data-open="DSQ007F2" class="link link-reveal link-table xref-fig">2</a>), were selected. The 5′-end of the RefSeq genes was mixed with 50-kb upstream regions to remove the possible alternative promoters of the RefSeq genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1E</a>). Then, the selected TSSs were clustered into 500 bp bins (iTSCs) in every cell type. The clustering analysis revealed that the numbers of iTSCs with &gt;5 ppm TSS tags were only around 500–2000 depending on cell types. Taken all TSS data from each cell types together, there were 371 849 iTSCs; however, the number of iTSCs with &gt;5 ppm TSS tags at least in one cell type was only 6039. We tentatively selected iTSCs of &gt;5 ppm, since 5 ppm is supported to be roughly corresponding to 5 copies/cell, assuming that each cell has 1 million mRNA copies. We considered that such a transcript level should be necessary to robustly realize biological functions (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>, for further detailed discussion).</p>                        <a id="5034008" scrollto-destination="5034008"></a>
<div id="DSQ007F2" data-id="DSQ007F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00702.jpeg?Expires=2147483647&Signature=JSTdYOrs0JzdN3sDJ0IGYaYoYze2DVIyx-yU3ZsRIKXssSKe2Csx8qqxQcnqXe6NxNsQpHV6vQyd1Q9CtrK-HCd4uLbG7QvKpWH~np6fA7GxayOyC-4zlLZj2F8XlG5mLsF7~BOow9ymyoijXzYcQG744NcJjKJFPTMHL499ogc3lEmggeSaWdaM8HrimLDa2CUc3myjG-toK6o~DCufJ4IPdXWuz5hU1MhSEBKmYB0a3Xrxs2SfJrNjF1grVTQDz3w4bd93z-GrseNI5VKmbyExlN6XxsD3S2bMF1pvrPf2KtWjB1A~zr5YHUdB9jlrqs82gSWHEQBCQ2fal2uhyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intergenic region. Intergenic regions were defined as regions from 3′-end boundaries to the −50 kb of the 5′-end boundaries of the adjoining RefSeq genes; ‘50 kb margins’ was adopted at the 5′-ends of the RefSeq genes, considering that some of the iTSCs located within these regions might be alternative promoters of downstream RefSeq genes." path-from-xml="dsq00702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034008/dsq00702.jpeg" path-from-xml="dsq00702.jpeg" target="_blank">View large</a><a section="5034008" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00702.jpeg?Expires=2147483647&Signature=q2kmy-y7pPgOsMhghaj4t71MvrdlCnq8JGEzVUR~pwUDfyy2~SFh4aAVoOnFxLTKjyli9KyVP8JtQW91XDJ5qXDZpR835iGrXF3~-J35eoqOlwrLK6VTZS8HRSj63tU9n1r4aF~NFAwvOHB7Kp8F0xodkBN6KzlreoIH2ZGnZIRZGoVgNR4wWrzsMv9-Udbx5y8289L4NcDp0~-6njECVVYV4~H9Wwcv7sk~sCnzR5OH1GowbCxtz5RYYTu0Hm34blbazVLpDZRpm-i2XnNtiUVQG~MdaaqswdNkho1KxczpdAg6xMBo4yMbKxkeKHB~NpHT9vohSA~CbgyHb0bmgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034008&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intergenic region. Intergenic regions were defined as regions from 3′-end boundaries to the −50 kb of the 5′-end boundaries of the adjoining RefSeq genes; ‘50 kb margins’ was adopted at the 5′-ends of the RefSeq genes, considering that some of the iTSCs located within these regions might be alternative promoters of downstream RefSeq genes.</p></div></div><div content-id="DSQ007F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00702.jpeg?Expires=2147483647&Signature=JSTdYOrs0JzdN3sDJ0IGYaYoYze2DVIyx-yU3ZsRIKXssSKe2Csx8qqxQcnqXe6NxNsQpHV6vQyd1Q9CtrK-HCd4uLbG7QvKpWH~np6fA7GxayOyC-4zlLZj2F8XlG5mLsF7~BOow9ymyoijXzYcQG744NcJjKJFPTMHL499ogc3lEmggeSaWdaM8HrimLDa2CUc3myjG-toK6o~DCufJ4IPdXWuz5hU1MhSEBKmYB0a3Xrxs2SfJrNjF1grVTQDz3w4bd93z-GrseNI5VKmbyExlN6XxsD3S2bMF1pvrPf2KtWjB1A~zr5YHUdB9jlrqs82gSWHEQBCQ2fal2uhyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intergenic region. Intergenic regions were defined as regions from 3′-end boundaries to the −50 kb of the 5′-end boundaries of the adjoining RefSeq genes; ‘50 kb margins’ was adopted at the 5′-ends of the RefSeq genes, considering that some of the iTSCs located within these regions might be alternative promoters of downstream RefSeq genes." path-from-xml="dsq00702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034008/dsq00702.jpeg" path-from-xml="dsq00702.jpeg" target="_blank">View large</a><a section="5034008" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00702.jpeg?Expires=2147483647&Signature=q2kmy-y7pPgOsMhghaj4t71MvrdlCnq8JGEzVUR~pwUDfyy2~SFh4aAVoOnFxLTKjyli9KyVP8JtQW91XDJ5qXDZpR835iGrXF3~-J35eoqOlwrLK6VTZS8HRSj63tU9n1r4aF~NFAwvOHB7Kp8F0xodkBN6KzlreoIH2ZGnZIRZGoVgNR4wWrzsMv9-Udbx5y8289L4NcDp0~-6njECVVYV4~H9Wwcv7sk~sCnzR5OH1GowbCxtz5RYYTu0Hm34blbazVLpDZRpm-i2XnNtiUVQG~MdaaqswdNkho1KxczpdAg6xMBo4yMbKxkeKHB~NpHT9vohSA~CbgyHb0bmgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034008&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intergenic region. Intergenic regions were defined as regions from 3′-end boundaries to the −50 kb of the 5′-end boundaries of the adjoining RefSeq genes; ‘50 kb margins’ was adopted at the 5′-ends of the RefSeq genes, considering that some of the iTSCs located within these regions might be alternative promoters of downstream RefSeq genes.</p></div></div>                            <h3 scrollto-destination=5034012 id="5034012" class="section-title" data-legacy-id=s3b>3.2. Complete sequencing of the transcripts of the iTSCs</h3>
<p>For both iTSCs of &gt;5 ppm and those of &lt;5 ppm, we characterized what transcripts were transcribed from the identified iTSCs. We selected completely sequenced full-length DNA clones which overlapped the iTSCs from MGC and FLJ cDNA collections. Among the total 371 849 iTSCs, there were 395 and 1617 iTSCs which overlapped the 5′-ends of the MGC and FLJ cDNAs, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3A</a>). We examined the cDNA sequences and found that about 60% of the cDNAs contained open reading frames (ORFs) of no more than 100 amino acids. When including the cDNAs which were possible targets for the nonsense mRNA decay or had 5′-untranslated region of &gt;750 bases, 85% of them had features which may hamper efficient translation and, thus, were likely to be ncRNAs.</p><p>In order to exclude the possible selection bias of the cDNA clones in the respective cDNA projects (note these cDNA projects aimed to select protein-coding transcripts), we selected and determined the complete sequences of the cDNA from our cDNA collection anew. In order to expedite the sequencing, we employed shotgun sequencing of the cDNA using Illumina GA. Details of the computational procedure for the assembling of the cDNA sequences are described in the ‘Materials and methods’ section. As a result, 361 and 103 cDNAs which correspond to the iTSCs of &gt;5 ppm and those of &lt;5 ppm were successfully assembled without any gap using a total of 3 382 901 short-read sequence tags. We analyzed the assembled sequences and found that the 80% of them contained no ORF longer than 100 amino acids and 91% of them had either of the above translational caveats (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3B</a>). These results strongly suggested that, either &gt;5 ppm or &lt;5 ppm, the iTSCs identified here should represent TSSs of ncRNAs.</p>                            <h3 scrollto-destination=5034018 id="5034018" class="section-title" data-legacy-id=s3c>3.3. Expression profiles of the iTSCs of &gt;5 ppm</h3>
<p>In our previous study, we found that the iTSCs of very low expression levels are generally: (i) located in AT rich regions, rarely associated with CpG island; (ii) evolutionarily poorly conserved, often associated with repetitive sequences (also see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4A</a>). We also found that, unlike the cases in significantly expressed iTSCs, iTSCs of very low expression levels are rarely associated with RNA polymerase II binding sites based on the results of the ChIP-Seq analysis (also see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S4C and S5</a>). In summary, the iTSCs of significant expression levels have clearly distinctive features, which are rather similar to the TSSs of protein-coding transcripts. On the other hand, sequence features observed for iTSCs of low expression levels seemed consistent with the hypothesis that very minor iTSCs occasionally occur from universal genomic sequences as biological noise, thus evolutionarily unstable since no selection pressure is being extorted. Further details of this analysis are described in the reference (Yamashita <em>et al</em>., submitted).</p><p>In order to further characterize the iTSCs of &gt;5 ppm, we examined their expression profiles. For this, we first used digital TSS Seq tag counts in 12 cell types. Among the 6039 iTSCs of &gt;5 ppm, we counted number of cell types which gave TSS tag counts of &gt;5 ppm. We found that 3739 (62%) of them were expressed at &gt;5 ppm only in one cell type, suggesting that many of them are expressed in a tissue-preferred manner (Fig. <a reveal-id="DSQ007F3" data-open="DSQ007F3" class="link link-reveal link-table xref-fig">3</a>A). Among these, fetal brain was the richest source of the iTSCs of &gt;5 ppm. In all of 1812 iTSCs expressed at &gt;5 ppm in fetal brain, 722 (40%) were expressed only in fetal brain and additionally 166 (9%) iTSCs were expressed in fetal brain and brain (Fig. <a reveal-id="DSQ007F3" data-open="DSQ007F3" class="link link-reveal link-table xref-fig">3</a>B). On the other hand, the poorest source of the tissue-specific iTSCs was HEK293 [nevertheless, still 66 (25%) of the iTSCs showed specific expressions in HEK293; Fig. <a reveal-id="DSQ007F3" data-open="DSQ007F3" class="link link-reveal link-table xref-fig">3</a>C]. Generally, tissue-specific expressions of the iTSCs were observed more frequently in normal tissues than cultured cells, possibly reflecting the reduced complexity of the gene functions in cultured cell lines.</p>                        <a id="5034025" scrollto-destination="5034025"></a>
<div id="DSQ007F3" data-id="DSQ007F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00703.jpeg?Expires=2147483647&Signature=ht3Cbat1OEYxDcCVySR4Rxk2yarwwepBxGqEAz~m8vwtb33SeOAwR37JPxrQuT4TUG2CLqWEbnEwhDxK2j8M4b1M5FfhX~rgS4MDpQ5BBdGSrKqbsvkB0v6IGnt9n~Y-haqDiEmS23Dsm3WHAhCLgwsurRpVsiW2c1IPfDwTAyxuD5-QYWKNC6h3S-3G6E~uDvL1PIgolrRO5A3K4ouArLG7p1zqBi17fxbs6QKzSL5KsBH52TjUefnfp08wkvfKJGlhbQRkp2HUuM~Rna~L7NgBocASU2o-oFAierPr3FiscywZTAs2eEy2gS4vri2X5v-RdSsKMQzI4Nc6znyNfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profile of the iTSCs of &gt;5 ppm. (A) Tissue specificity of the iTSCs of &gt;5 ppm. The number of the cell types in which iTSCs of &gt;5 ppm were observed is indicated in x-axis. (B and C) As for iTSCs which were observed in fetal brain (B) and HEK293 cells (C), whether they were also observed in different cell types was examined. Populations of the iTSCs whose expressions were observed in the indicated number of the cell types are shown." path-from-xml="dsq00703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034025/dsq00703.jpeg" path-from-xml="dsq00703.jpeg" target="_blank">View large</a><a section="5034025" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00703.jpeg?Expires=2147483647&Signature=bKrJ3VFJDdvG0miWTLjNVh7DsWGifmh7ixiBQYEiQBFMxeZW69J3dOerXmptvASSDZIa~6ue2EvCqF9p0zZnGbvrxqB9EkJihda1ZDOdI6v2Vs4JbPfGGdUcljiSykLraSlIYARfjTYHqR0JAR0g4VxcnP0XL-nPixzvY8x3BdjkbcHQZYfa-es24Wfe8FIe3tTsVQoOJU1pBkW-ZJMMUBpuNPt61P9k7kyi9MWQm4hnFw7rIj8G6k4dfwYV29whJKzRUgKTfgLEo3rOL4hw-9WW4VLrPmpFvMKH-ttGJsypMXeOTgSLtbuA09pES3M~Ko~2oucNZosynJgymqVMHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034025&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profile of the iTSCs of &gt;5 ppm. (<strong>A</strong>) Tissue specificity of the iTSCs of &gt;5 ppm. The number of the cell types in which iTSCs of &gt;5 ppm were observed is indicated in <em>x</em>-axis. (<strong>B</strong> and <strong>C</strong>) As for iTSCs which were observed in fetal brain (B) and HEK293 cells (C), whether they were also observed in different cell types was examined. Populations of the iTSCs whose expressions were observed in the indicated number of the cell types are shown.</p></div></div><div content-id="DSQ007F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00703.jpeg?Expires=2147483647&Signature=ht3Cbat1OEYxDcCVySR4Rxk2yarwwepBxGqEAz~m8vwtb33SeOAwR37JPxrQuT4TUG2CLqWEbnEwhDxK2j8M4b1M5FfhX~rgS4MDpQ5BBdGSrKqbsvkB0v6IGnt9n~Y-haqDiEmS23Dsm3WHAhCLgwsurRpVsiW2c1IPfDwTAyxuD5-QYWKNC6h3S-3G6E~uDvL1PIgolrRO5A3K4ouArLG7p1zqBi17fxbs6QKzSL5KsBH52TjUefnfp08wkvfKJGlhbQRkp2HUuM~Rna~L7NgBocASU2o-oFAierPr3FiscywZTAs2eEy2gS4vri2X5v-RdSsKMQzI4Nc6znyNfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profile of the iTSCs of &gt;5 ppm. (A) Tissue specificity of the iTSCs of &gt;5 ppm. The number of the cell types in which iTSCs of &gt;5 ppm were observed is indicated in x-axis. (B and C) As for iTSCs which were observed in fetal brain (B) and HEK293 cells (C), whether they were also observed in different cell types was examined. Populations of the iTSCs whose expressions were observed in the indicated number of the cell types are shown." path-from-xml="dsq00703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034025/dsq00703.jpeg" path-from-xml="dsq00703.jpeg" target="_blank">View large</a><a section="5034025" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00703.jpeg?Expires=2147483647&Signature=bKrJ3VFJDdvG0miWTLjNVh7DsWGifmh7ixiBQYEiQBFMxeZW69J3dOerXmptvASSDZIa~6ue2EvCqF9p0zZnGbvrxqB9EkJihda1ZDOdI6v2Vs4JbPfGGdUcljiSykLraSlIYARfjTYHqR0JAR0g4VxcnP0XL-nPixzvY8x3BdjkbcHQZYfa-es24Wfe8FIe3tTsVQoOJU1pBkW-ZJMMUBpuNPt61P9k7kyi9MWQm4hnFw7rIj8G6k4dfwYV29whJKzRUgKTfgLEo3rOL4hw-9WW4VLrPmpFvMKH-ttGJsypMXeOTgSLtbuA09pES3M~Ko~2oucNZosynJgymqVMHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034025&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profile of the iTSCs of &gt;5 ppm. (<strong>A</strong>) Tissue specificity of the iTSCs of &gt;5 ppm. The number of the cell types in which iTSCs of &gt;5 ppm were observed is indicated in <em>x</em>-axis. (<strong>B</strong> and <strong>C</strong>) As for iTSCs which were observed in fetal brain (B) and HEK293 cells (C), whether they were also observed in different cell types was examined. Populations of the iTSCs whose expressions were observed in the indicated number of the cell types are shown.</p></div></div><p>In order to observe the expression patterns of the iTSCs in wider cell types, we performed qRT–PCR assays in additional 20 kinds of human normal tissues using the PCR primers designed based on the complete cDNA sequences. For 44 out of 67 iTSCs of &gt;5 ppm, we observed clear signals at least in one tissue. Again, we observed that the expression of the iTSCs of &gt;5 ppm generally had tissue preference (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). We also examined the expression of the 93 iTSCs of &lt;5 ppm by qRT–PCR. In 60 out of 93 iTSCs of &lt;5 ppm, weak but clear signals were obtained. Of these expressed iTSCs, 16 iTSCs (27%) showed clear tissue specificity. Although the presence of the transcripts for those iTSCs was confirmed, their expression levels determined by qRT–PCR were lower than iTSCs of &gt;5 ppm across the tissues examined.</p>                            <h3 scrollto-destination=5034029 id="5034029" class="section-title" data-legacy-id=s3d>3.4. Correlation between a group of iTSCs of &gt;5 ppm and RefSeq genes</h3>
<p>We further scrutinized the iTSCs of &gt;5 ppm and found that iTSCs of &gt;5 ppm are not a uniform group. To precisely examine the genomic position of the iTSCs, 10 kb windows of the genome position were defined relatively to the RefSeq gene positions. We found that a number of iTSCs of &gt;5 ppm were located in the proximal regions of RefSeq genes (Fig. <a reveal-id="DSQ007F4" data-open="DSQ007F4" class="link link-reveal link-table xref-fig">4</a>). Although 23% (1382 cases) of the iTSCs of &gt;5 ppm were located within 10 kb downstream of the RefSeq gene in the same strand (RefSeq-linked iTSCs), the frequency of such iTSCs in the iTSC group of &lt;5 ppm was only 7% (24 486 cases). We compared the expression patterns of the iTSCs of &gt;5 ppm with their neighboring RefSeq genes by calculating Pearson's correlation of the respective TSS tag counts. We found that the expressions of the RefSeq-linked iTSCs were generally resembled to the expressions of their nearest RefSeq genes (Fig. <a reveal-id="DSQ007F5" data-open="DSQ007F5" class="link link-reveal link-table xref-fig">5</a>A). Such correlation was not detected in the iTSCs located further than 10 kb from the nearest RefSeq genes (non-RefSeq-linked iTSCs; Fig. <a reveal-id="DSQ007F5" data-open="DSQ007F5" class="link link-reveal link-table xref-fig">5</a>B). In order to observe the expression correlation in wider cell types, we performed qRT–PCR assays in 20 normal tissues using 12 representative cases. Again, we observed general correlation of the expression patterns between the RefSeq-linked iTSCs of &gt;5 ppm and the nearby RefSeq genes (exemplified in Fig. <a reveal-id="DSQ007F5" data-open="DSQ007F5" class="link link-reveal link-table xref-fig">5</a>C), and no correlation was observed for the non-RefSeq-linked iTSCs of &gt;5 ppm (exemplified in Fig. <a reveal-id="DSQ007F5" data-open="DSQ007F5" class="link link-reveal link-table xref-fig">5</a>D).</p>                        <a id="5034032" scrollto-destination="5034032"></a>
<div id="DSQ007F4" data-id="DSQ007F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00704.jpeg?Expires=2147483647&Signature=t~fHMhDym7OWbHjZbx4-78ateCyfCVmTi03muF7eAVYX~1In3TmiOJhFtVjrlCYFRee~Ep~BqZR~NRHTnWSgRXKEK8Q8eEsiMNrxf4IJ5awwNYI2vzunvnmt99Wxtd1M-bstJgaSsxYa2mJVvlwh0C4PWd~xLupmITduYiikQhqB9VjdrvVa6Ck~UAhzblLfosf9kPD~5Dqkm4CdMgAYHIjhnoBtt5KCpVnrT2ooodhLycyxPp6ctGcn-l10tlZf5ymzonjqts3vlNxBia9fTWTC0DqHsTls1Sn~WvAGinBdy-N5nmiXUa0UuYx1QUMuBkAYzzU~ORg~9l-wTM604g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distance from the iTSCs to the nearest RefSeq genes. Distribution of the distance between the iTSCs and the nearest RefSeq genes is shown in the case of the iTSCs of &gt;5 ppm (A) and the those of &lt;5 ppm (B). According to this positioning observation, the iTSCs were separated into two groups: (C) RefSeq-linked iTSCs (iTSCs which lie within 10 kb of RefSeq genes) and non-RefSeq-linked iTSCs (iTSCs which locate further than 10 kb from the nearest RefSeq genes)." path-from-xml="dsq00704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034032/dsq00704.jpeg" path-from-xml="dsq00704.jpeg" target="_blank">View large</a><a section="5034032" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00704.jpeg?Expires=2147483647&Signature=Aw0~6sFPNix4gjpebfRf9hyW8pK~AS~dWsyMciHoIcOWdoFSBChsH71m5eaw37BCx7nf7WLpuew8GPRx5qCrnSBOCGDFgNmwYm1zUBScOipITwBDY-vTZE8kyFsKvzhpsd46rhBxfWLn0E5LzU6AKRGvDKklp2jPyBWUuJ-NsXDIeOEhDsZ3rcS3JJ54IRbNrocRvoxtmI5~eOHEWRpjA7Cw8dbv9VNywga8kHMEnupqtLS35-tw1BL4UjrA7T2JhlvLmBPIXC2MUiomztnx0DKCkvQ95YLNIHY5zUrhqwioh5vYRKH5XQJXOYvc9pKWwCL-OgHaQkyLWnZSYFhrAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034032&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distance from the iTSCs to the nearest RefSeq genes. Distribution of the distance between the iTSCs and the nearest RefSeq genes is shown in the case of the iTSCs of &gt;5 ppm (<strong>A</strong>) and the those of &lt;5 ppm (<strong>B</strong>). According to this positioning observation, the iTSCs were separated into two groups: (<strong>C</strong>) RefSeq-linked iTSCs (iTSCs which lie within 10 kb of RefSeq genes) and non-RefSeq-linked iTSCs (iTSCs which locate further than 10 kb from the nearest RefSeq genes).</p></div></div><div content-id="DSQ007F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00704.jpeg?Expires=2147483647&Signature=t~fHMhDym7OWbHjZbx4-78ateCyfCVmTi03muF7eAVYX~1In3TmiOJhFtVjrlCYFRee~Ep~BqZR~NRHTnWSgRXKEK8Q8eEsiMNrxf4IJ5awwNYI2vzunvnmt99Wxtd1M-bstJgaSsxYa2mJVvlwh0C4PWd~xLupmITduYiikQhqB9VjdrvVa6Ck~UAhzblLfosf9kPD~5Dqkm4CdMgAYHIjhnoBtt5KCpVnrT2ooodhLycyxPp6ctGcn-l10tlZf5ymzonjqts3vlNxBia9fTWTC0DqHsTls1Sn~WvAGinBdy-N5nmiXUa0UuYx1QUMuBkAYzzU~ORg~9l-wTM604g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distance from the iTSCs to the nearest RefSeq genes. Distribution of the distance between the iTSCs and the nearest RefSeq genes is shown in the case of the iTSCs of &gt;5 ppm (A) and the those of &lt;5 ppm (B). According to this positioning observation, the iTSCs were separated into two groups: (C) RefSeq-linked iTSCs (iTSCs which lie within 10 kb of RefSeq genes) and non-RefSeq-linked iTSCs (iTSCs which locate further than 10 kb from the nearest RefSeq genes)." path-from-xml="dsq00704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034032/dsq00704.jpeg" path-from-xml="dsq00704.jpeg" target="_blank">View large</a><a section="5034032" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00704.jpeg?Expires=2147483647&Signature=Aw0~6sFPNix4gjpebfRf9hyW8pK~AS~dWsyMciHoIcOWdoFSBChsH71m5eaw37BCx7nf7WLpuew8GPRx5qCrnSBOCGDFgNmwYm1zUBScOipITwBDY-vTZE8kyFsKvzhpsd46rhBxfWLn0E5LzU6AKRGvDKklp2jPyBWUuJ-NsXDIeOEhDsZ3rcS3JJ54IRbNrocRvoxtmI5~eOHEWRpjA7Cw8dbv9VNywga8kHMEnupqtLS35-tw1BL4UjrA7T2JhlvLmBPIXC2MUiomztnx0DKCkvQ95YLNIHY5zUrhqwioh5vYRKH5XQJXOYvc9pKWwCL-OgHaQkyLWnZSYFhrAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034032&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distance from the iTSCs to the nearest RefSeq genes. Distribution of the distance between the iTSCs and the nearest RefSeq genes is shown in the case of the iTSCs of &gt;5 ppm (<strong>A</strong>) and the those of &lt;5 ppm (<strong>B</strong>). According to this positioning observation, the iTSCs were separated into two groups: (<strong>C</strong>) RefSeq-linked iTSCs (iTSCs which lie within 10 kb of RefSeq genes) and non-RefSeq-linked iTSCs (iTSCs which locate further than 10 kb from the nearest RefSeq genes).</p></div></div>                        <a id="5034034" scrollto-destination="5034034"></a>
<div id="DSQ007F5" data-id="DSQ007F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00705.jpeg?Expires=2147483647&Signature=bKHwgDMuwlD5r-VAeSicwAUO7AwWoxtCdSIYWnX5J7grV8~Dr8gV3A6azeesW6u3S6IlpsR7a~xprZ3zi6b2MDKR1rdYKMK1K7uNwJNtCcGyZEIBfbQy-aza-35TNZukUFr5DS2f9UUSFjbegXOhUIYMgc9UIfKJTirs4mQH1sGW0N4p9rp2-Xm6CRU4lpEhWUF4q3yJKtvOPA2iCDDIBAHcd~hzpEvd-1P5Vf5~ZJQoeSHE8U2Pi3FoO1zOu7qSv6UKin-fViyLj5SwrTP96n53okD9TTIJHoCZfQ4Y3HwFHKEUic5p9~3AfC3BgZVXD3~FK9Yyb8fj4U1Irkw37w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the iTSCs and the nearest RefSeq genes. Distribution of Pearson's correlation of the TSS tag counts between the iTSCs located within 10 kb of the RefSeq genes (A) and those located outside 10 kb of the nearest RefSeq genes (B). (C) Correlation of the expression patterns monitored by quantitative RT-PCR in 20 normal human tissues (AD, adrenal gland; BM, bone marrow; BW, brain (whole); FB, fetal brain; FL, fetal liver; HE, heart; KI, kidney; LI, liver; LU, lung; PL, placenta; PR, prostate; SG, salivary gland; SM, skeletal muscle; TE, testis; TH, thymus; TG, thyroid gland; TR, trachea; UT, uterus; CO, colon w/mucosa; SC, spinal cord) for a RefSeq-linked iTSC and the linked RefSeq (NM_005575; 244 bp apart from the iTSC). (D) Result from the similar analysis as (C) using a non-RefSeq-linked iTSC and the nearest RefSeq gene (NM_203390; 11 581 bp apart from the iTSC). Typical examples for the RefSeq-linked iTSC (E) and non-RefSeq-linked iTSC (F). Completely sequenced cDNA and distributions of the TSS Seq tags and ChIP-Seq tags (pol II) in the surrounding region are shown." path-from-xml="dsq00705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034034/dsq00705.jpeg" path-from-xml="dsq00705.jpeg" target="_blank">View large</a><a section="5034034" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00705.jpeg?Expires=2147483647&Signature=A5XQMcbUAWEWomBx3TS-L3q3CXT6IPjEcIsyTMJFcoDytcee9j5iUXX-~q2O8-1XaxbuucUk8Ayyb2ssfbkjWgQD7G4fvHf1J8xvBEdrvMmL9gwg0AFYBmxaLkC9hI7jhic1M3qHYaysU-mvktCSQEyfaBC6uwmEsUnJ6PdPd1B7RDDSV5uMdwxi~ApfJSQ-N9a3sZhEdvr0OZXBUVs30YP0iTwn9WV0~0o6Fg97rtrtc-jqxAINwpZIVV1~A0ra1BeswwcUVNjZpEieq9eeg-vuoNH-l5j8zhKrEjqNdN0WUBPK~wYyKxuPi9q4IbHKdEdmPpC7Fgxpt5x2tJDfoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034034&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the iTSCs and the nearest RefSeq genes. Distribution of Pearson's correlation of the TSS tag counts between the iTSCs located within 10 kb of the RefSeq genes (<strong>A</strong>) and those located outside 10 kb of the nearest RefSeq genes (<strong>B</strong>). (<strong>C</strong>) Correlation of the expression patterns monitored by quantitative RT-PCR in 20 normal human tissues (AD, adrenal gland; BM, bone marrow; BW, brain (whole); FB, fetal brain; FL, fetal liver; HE, heart; KI, kidney; LI, liver; LU, lung; PL, placenta; PR, prostate; SG, salivary gland; SM, skeletal muscle; TE, testis; TH, thymus; TG, thyroid gland; TR, trachea; UT, uterus; CO, colon w/mucosa; SC, spinal cord) for a RefSeq-linked iTSC and the linked RefSeq (NM_005575; 244 bp apart from the iTSC). (<strong>D</strong>) Result from the similar analysis as (C) using a non-RefSeq-linked iTSC and the nearest RefSeq gene (NM_203390; 11 581 bp apart from the iTSC). Typical examples for the RefSeq-linked iTSC (<strong>E</strong>) and non-RefSeq-linked iTSC (<strong>F</strong>). Completely sequenced cDNA and distributions of the TSS Seq tags and ChIP-Seq tags (pol II) in the surrounding region are shown.</p></div></div><div content-id="DSQ007F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00705.jpeg?Expires=2147483647&Signature=bKHwgDMuwlD5r-VAeSicwAUO7AwWoxtCdSIYWnX5J7grV8~Dr8gV3A6azeesW6u3S6IlpsR7a~xprZ3zi6b2MDKR1rdYKMK1K7uNwJNtCcGyZEIBfbQy-aza-35TNZukUFr5DS2f9UUSFjbegXOhUIYMgc9UIfKJTirs4mQH1sGW0N4p9rp2-Xm6CRU4lpEhWUF4q3yJKtvOPA2iCDDIBAHcd~hzpEvd-1P5Vf5~ZJQoeSHE8U2Pi3FoO1zOu7qSv6UKin-fViyLj5SwrTP96n53okD9TTIJHoCZfQ4Y3HwFHKEUic5p9~3AfC3BgZVXD3~FK9Yyb8fj4U1Irkw37w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the iTSCs and the nearest RefSeq genes. Distribution of Pearson's correlation of the TSS tag counts between the iTSCs located within 10 kb of the RefSeq genes (A) and those located outside 10 kb of the nearest RefSeq genes (B). (C) Correlation of the expression patterns monitored by quantitative RT-PCR in 20 normal human tissues (AD, adrenal gland; BM, bone marrow; BW, brain (whole); FB, fetal brain; FL, fetal liver; HE, heart; KI, kidney; LI, liver; LU, lung; PL, placenta; PR, prostate; SG, salivary gland; SM, skeletal muscle; TE, testis; TH, thymus; TG, thyroid gland; TR, trachea; UT, uterus; CO, colon w/mucosa; SC, spinal cord) for a RefSeq-linked iTSC and the linked RefSeq (NM_005575; 244 bp apart from the iTSC). (D) Result from the similar analysis as (C) using a non-RefSeq-linked iTSC and the nearest RefSeq gene (NM_203390; 11 581 bp apart from the iTSC). Typical examples for the RefSeq-linked iTSC (E) and non-RefSeq-linked iTSC (F). Completely sequenced cDNA and distributions of the TSS Seq tags and ChIP-Seq tags (pol II) in the surrounding region are shown." path-from-xml="dsq00705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034034/dsq00705.jpeg" path-from-xml="dsq00705.jpeg" target="_blank">View large</a><a section="5034034" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00705.jpeg?Expires=2147483647&Signature=A5XQMcbUAWEWomBx3TS-L3q3CXT6IPjEcIsyTMJFcoDytcee9j5iUXX-~q2O8-1XaxbuucUk8Ayyb2ssfbkjWgQD7G4fvHf1J8xvBEdrvMmL9gwg0AFYBmxaLkC9hI7jhic1M3qHYaysU-mvktCSQEyfaBC6uwmEsUnJ6PdPd1B7RDDSV5uMdwxi~ApfJSQ-N9a3sZhEdvr0OZXBUVs30YP0iTwn9WV0~0o6Fg97rtrtc-jqxAINwpZIVV1~A0ra1BeswwcUVNjZpEieq9eeg-vuoNH-l5j8zhKrEjqNdN0WUBPK~wYyKxuPi9q4IbHKdEdmPpC7Fgxpt5x2tJDfoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034034&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the iTSCs and the nearest RefSeq genes. Distribution of Pearson's correlation of the TSS tag counts between the iTSCs located within 10 kb of the RefSeq genes (<strong>A</strong>) and those located outside 10 kb of the nearest RefSeq genes (<strong>B</strong>). (<strong>C</strong>) Correlation of the expression patterns monitored by quantitative RT-PCR in 20 normal human tissues (AD, adrenal gland; BM, bone marrow; BW, brain (whole); FB, fetal brain; FL, fetal liver; HE, heart; KI, kidney; LI, liver; LU, lung; PL, placenta; PR, prostate; SG, salivary gland; SM, skeletal muscle; TE, testis; TH, thymus; TG, thyroid gland; TR, trachea; UT, uterus; CO, colon w/mucosa; SC, spinal cord) for a RefSeq-linked iTSC and the linked RefSeq (NM_005575; 244 bp apart from the iTSC). (<strong>D</strong>) Result from the similar analysis as (C) using a non-RefSeq-linked iTSC and the nearest RefSeq gene (NM_203390; 11 581 bp apart from the iTSC). Typical examples for the RefSeq-linked iTSC (<strong>E</strong>) and non-RefSeq-linked iTSC (<strong>F</strong>). Completely sequenced cDNA and distributions of the TSS Seq tags and ChIP-Seq tags (pol II) in the surrounding region are shown.</p></div></div><p>It was unlikely that these RefSeq-linked iTSCs of &gt;5 ppm were derived from the 5′-end truncated products of the RefSeq gene transcripts due to the errors in the cap selection procedure.<sup><a reveal-id="DSQ007C27" data-open="DSQ007C27" class="link link-ref link-reveal xref-bibr">27</a></sup> At least in 186 out of 356 (52%) RefSeq-linked iTSCs which were expressed at &gt;5 ppm in DLD-1 cells, their presence were also clearly supported by bindings of RNA polymerase II by ChIP-Seq analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4C</a>). Of these, 50 RefSeq-linked iTSCs were further supported by cDNA clones, which did not overlap the RefSeq transcripts (Fig. <a reveal-id="DSQ007F5" data-open="DSQ007F5" class="link link-reveal link-table xref-fig">5</a>E and F). It was rather likely that these RefSeq-linked iTSCs might be activated simply because of the generally loose chromatin structure which was formed to facilitate the transcription of the upstream RefSeq gene (mentioned as ‘transcriptional rippling noise’ in a recent paper<sup><a reveal-id="DSQ007C36" data-open="DSQ007C36" class="link link-ref link-reveal xref-bibr">36</a></sup>).</p><p>In order to examine this hypothesis, we analyzed the nucleosome structure in their surrounding regions by Nucleosome Seq using DLD-1 cells. A total of 19 570 149 sequence tags were collected and occupancy of the nucleosome was calculated (see the ‘Materials and methods’ section, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a> and the references<sup><a reveal-id="DSQ007C37" data-open="DSQ007C37" class="link link-ref link-reveal xref-bibr">37</a>–<a reveal-id="DSQ007C39" data-open="DSQ007C39" class="link link-ref link-reveal xref-bibr">39</a></sup>). Orderly nucleosome structure was not observed for the RefSeq-linked iTSCs of &gt;5 ppm (Fig. <a reveal-id="DSQ007F6" data-open="DSQ007F6" class="link link-reveal link-table xref-fig">6</a>B) and those of &lt;5 ppm (Fig. <a reveal-id="DSQ007F6" data-open="DSQ007F6" class="link link-reveal link-table xref-fig">6</a>C). On the contrary, for the non-RefSeq-linked TSCs of &gt;5 ppm, we observed the formation of the nucleosome-free regions just upstream of the TSSs and periodic positioning of the nucleosome immediately downstream of the TSS (Fig. <a reveal-id="DSQ007F6" data-open="DSQ007F6" class="link link-reveal link-table xref-fig">6</a>A). These results may reflect that transcriptions of RefSeq-linked iTSCs of &gt;5 ppm seem not to form independent transcriptional ‘regulatory’ units, in spite of their relatively high expression levels, whereas non-RefSeq-linked iTSCs of &gt;5 ppm are subjected to strict independent regulations. Consistently, we further examined and found that the TSSs were more widely fluctuating (also see Suzuki <em>et al</em>.<sup><a reveal-id="DSQ007C40" data-open="DSQ007C40" class="link link-ref link-reveal xref-bibr">40</a></sup>) in RefSeq-linked iTSCs of &gt;5 ppm than non-RefSeq-linked iTSCs of &gt;5 ppm (7e−16; Wilcoxon test). The average sizes of the regions which covered 80% of the TSSs within a particular TSC were 264 and 122 bp, respectively.</p>                        <a id="5034038" scrollto-destination="5034038"></a>
<div id="DSQ007F6" data-id="DSQ007F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00706.jpeg?Expires=2147483647&Signature=VbfTlnYtnzIpSKRMQydtXL4gMSjKrtOJmBJtcSDXkpy30daFC7oY0atGBOC-72D8VqkzG0lHcW0WnzhlxttQKnatPZZzqldVAIGRQIQRppUKJtM7C6tv3uOSilY1OAKST2Cb-pvVZkJKM1A0-OP9SDsPBlsYEe1blsY65QDCInZZ1OY1fHl8CezXy9sp0u-c6wfnotC8KvPhe2Uo5nbgKmTz1G3S1awgjYJTMPhRwL2DOmWkYnhMm~J~fRo6cWP7uG2sdDOUi5kMWLtY5OfPtfkT3qQsZ4oIOZSubhAdJAfvoTeXHBR4hbKj9r2fJcc-dDhFpTRVdcBpWnuue7TJiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nucleosome structure of the surrounding regions of the iTSCs. Nucleosome structure in the surrounding regions of the iTSCs of &gt;5 ppm, located outside of 10 kb from the nearest RefSeq genes on the same strand (A), those of &gt;5 ppm, located within 10 kb from the nearest RefSeq genes on the same strand (B), and those of &lt;5 ppm (C). (D) Nucleosome structure in the surrounding regions of RefSeq genes. Six hundred and fifteen (total number of the RefSeq-linked iTSCs plus non-RefSeq-linked RNAs) RefSeq genes were randomly selected so that the distribution of their expression levels should be similar to that of iTSCs. For each group, nucleosome occupancy scores (y-axis) were calculated for the indicated genomic position (x-axis) relative to the TSS, according to the standard method shown in the reference." path-from-xml="dsq00706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034038/dsq00706.jpeg" path-from-xml="dsq00706.jpeg" target="_blank">View large</a><a section="5034038" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00706.jpeg?Expires=2147483647&Signature=xrhF3XjB1ixTF26qDYnkTmFVulRCpwGUJ7ijxp2WDjEJZFejmnzF6qUnXVuFrAe16c4bobHQcr-QL0QlfhbOsuIKMtbFvhwiLXzSCsZZDYjWzUAirCOuuKRwdp9dAdWbY6Ss8Op53Fb0YGYTDqA6FnYppifkcnlphPgLe1-N9Ylp09ylsLRMCVD-FEFZBcSeRjR7Ob7xxy3CzM12tBynOaDjdGhQS9N7zdDgB5K0gWOjIHFXV-aPkvawKvPKYrphic0E0qeoTTzudjT2hGZdf47mitnqGY~d2Mnatk3sXwvX-v2pqB6JjhP4pSpb7k0NvOR1z8kMT8TtH-CaS2a7cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034038&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nucleosome structure of the surrounding regions of the iTSCs. Nucleosome structure in the surrounding regions of the iTSCs of &gt;5 ppm, located outside of 10 kb from the nearest RefSeq genes on the same strand (<strong>A</strong>), those of &gt;5 ppm, located within 10 kb from the nearest RefSeq genes on the same strand (<strong>B</strong>), and those of &lt;5 ppm (<strong>C</strong>). (<strong>D</strong>) Nucleosome structure in the surrounding regions of RefSeq genes. Six hundred and fifteen (total number of the RefSeq-linked iTSCs plus non-RefSeq-linked RNAs) RefSeq genes were randomly selected so that the distribution of their expression levels should be similar to that of iTSCs. For each group, nucleosome occupancy scores (<em>y</em>-axis) were calculated for the indicated genomic position (<em>x</em>-axis) relative to the TSS, according to the standard method shown in the reference.</p></div></div><div content-id="DSQ007F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00706.jpeg?Expires=2147483647&Signature=VbfTlnYtnzIpSKRMQydtXL4gMSjKrtOJmBJtcSDXkpy30daFC7oY0atGBOC-72D8VqkzG0lHcW0WnzhlxttQKnatPZZzqldVAIGRQIQRppUKJtM7C6tv3uOSilY1OAKST2Cb-pvVZkJKM1A0-OP9SDsPBlsYEe1blsY65QDCInZZ1OY1fHl8CezXy9sp0u-c6wfnotC8KvPhe2Uo5nbgKmTz1G3S1awgjYJTMPhRwL2DOmWkYnhMm~J~fRo6cWP7uG2sdDOUi5kMWLtY5OfPtfkT3qQsZ4oIOZSubhAdJAfvoTeXHBR4hbKj9r2fJcc-dDhFpTRVdcBpWnuue7TJiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nucleosome structure of the surrounding regions of the iTSCs. Nucleosome structure in the surrounding regions of the iTSCs of &gt;5 ppm, located outside of 10 kb from the nearest RefSeq genes on the same strand (A), those of &gt;5 ppm, located within 10 kb from the nearest RefSeq genes on the same strand (B), and those of &lt;5 ppm (C). (D) Nucleosome structure in the surrounding regions of RefSeq genes. Six hundred and fifteen (total number of the RefSeq-linked iTSCs plus non-RefSeq-linked RNAs) RefSeq genes were randomly selected so that the distribution of their expression levels should be similar to that of iTSCs. For each group, nucleosome occupancy scores (y-axis) were calculated for the indicated genomic position (x-axis) relative to the TSS, according to the standard method shown in the reference." path-from-xml="dsq00706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034038/dsq00706.jpeg" path-from-xml="dsq00706.jpeg" target="_blank">View large</a><a section="5034038" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00706.jpeg?Expires=2147483647&Signature=xrhF3XjB1ixTF26qDYnkTmFVulRCpwGUJ7ijxp2WDjEJZFejmnzF6qUnXVuFrAe16c4bobHQcr-QL0QlfhbOsuIKMtbFvhwiLXzSCsZZDYjWzUAirCOuuKRwdp9dAdWbY6Ss8Op53Fb0YGYTDqA6FnYppifkcnlphPgLe1-N9Ylp09ylsLRMCVD-FEFZBcSeRjR7Ob7xxy3CzM12tBynOaDjdGhQS9N7zdDgB5K0gWOjIHFXV-aPkvawKvPKYrphic0E0qeoTTzudjT2hGZdf47mitnqGY~d2Mnatk3sXwvX-v2pqB6JjhP4pSpb7k0NvOR1z8kMT8TtH-CaS2a7cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034038&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nucleosome structure of the surrounding regions of the iTSCs. Nucleosome structure in the surrounding regions of the iTSCs of &gt;5 ppm, located outside of 10 kb from the nearest RefSeq genes on the same strand (<strong>A</strong>), those of &gt;5 ppm, located within 10 kb from the nearest RefSeq genes on the same strand (<strong>B</strong>), and those of &lt;5 ppm (<strong>C</strong>). (<strong>D</strong>) Nucleosome structure in the surrounding regions of RefSeq genes. Six hundred and fifteen (total number of the RefSeq-linked iTSCs plus non-RefSeq-linked RNAs) RefSeq genes were randomly selected so that the distribution of their expression levels should be similar to that of iTSCs. For each group, nucleosome occupancy scores (<em>y</em>-axis) were calculated for the indicated genomic position (<em>x</em>-axis) relative to the TSS, according to the standard method shown in the reference.</p></div></div>                            <h3 scrollto-destination=5034040 id="5034040" class="section-title" data-legacy-id=s3e>3.5. Characterization of the non-RefSeq-linked iTSCs</h3>
<p>For the remaining 3651 non-RefSeq-linked iTSCs of &gt;5 ppm, we considered that some of them might participate in transcription of the precursors of miRNAs or other small RNAs, since the structures of the primary transcripts of them were still mostly unknown. We examined overlap of the iTSCs with previously reported miRNAs and snoRNAs registered in miRBase<sup><a reveal-id="DSQ007C41" data-open="DSQ007C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and snoBase,<sup><a reveal-id="DSQ007C42" data-open="DSQ007C42" class="link link-ref link-reveal xref-bibr">42</a></sup> respectively. We found that 23 miRNAs and 123 snoRNAs were located within 50 kb downstream of the identified iTSCs (note 50 kb downstream sequences were extended from identified iTSCs, representing transcription regions of iTSC). Particularly, for 22 miRNAs and 11 snoRNAs, complete cDNA sequences directly showed that the mature forms of the small RNAs were actually located within the transcribed regions following the iTSCs (exemplified in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>).</p><p>In order to further identify the iTSCs which transcribe the primary transcripts of miRNAs, we performed small RNA Seq analysis using DLD-1 cells. In total, 604 141 sequences of size-selected RNAs of 21-25 bp were determined. Obtained short-read sequences were clustered, and 21-bp sequences which had expression levels of &gt;50 ppm were selected (in this case, the ppm value was calculated against the total number of the sequenced small RNAs, whose gross expression levels was estimated 1/10–1/100 compared with mRNAs/lncRNAs. Thus, 50 ppm is expected to be at the similar level with 5 ppm in the case of mRNAs/lncRNAs). Among the identified 1266 putative miRNAs (49 overlapped the previously reported miRNAs), 24 miRNAs were located within 50 kb of the non-RefSeq-linked iTSCs of &gt;5 ppm in DLD-1 cells (4% of the total 575 non-RefSeq-linked iTSCs). Of these putative miRNAs, only two were directly supported by cDNA clones. Having observed that most of the iTSCs were not correlated with the miRNAs, the majority of non-RefSeq-linked iTSCs are not used for transcribing primary transcripts of miRNAs.</p><p>We examined subcellular localization of their transcripts by RNA Seq of RNAs extracted from nuclear and cytoplasmic fractions using DLD-1 cells. As shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>, precise separation of the cytoplasmic fraction from the nuclear fraction was confirmed by qRT–PCR, using nuclear scaRNAs and other snoRNAs, and western blotting analysis using nuclear lamin A/C proteins and cytoplasmic GAPDH protein. A total of 20 094 475 and 14 879 174 tags were collected from the nuclear and cytoplasmic RNAs, respectively. RNA Seq tags were associated with the iTSCs when their mapped genomic positions overlapped. Among the 575 non-RefSeq-linked iTSCs with expression levels of &gt;5 ppm in DLD-1 cells, 337 iTSCs (59%) overlapped at least three RNA Seq tags from the cytoplasmic fraction. Of these, 99 iTSCs (17%) were particularly enriched in cytoplasm (<em>P</em> &lt; 0.01). These results showed that a large number of iTSC transcripts are actually exported into cytoplasm without losing the surrounding sequences of the TSSs, possibly in their native forms as lncRNAs.</p>                            <h3 scrollto-destination=5034047 id="5034047" class="section-title" data-legacy-id=s3f>3.6. Integration of the transcriptome data for inferring biological functions of the non-RefSeq-linked iTSCs</h3>
<p>We wished to demonstrate that specific functions of the iTSCs underlying particular biological events could be inferred from the integrative interpretation of hereby described various types of transcriptome data. We focused the analysis on the iTSCs whose functions may be associated with hypoxic response of the cell. For this, we generated additional 36 761 810 thirty-six-base pair TSS tags in DLD-1, MCF7, HEK293 and TIG3 cells which were cultured in 1% O<sub>2</sub> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). As shown in Table <a reveal-id="DSQ007TB1" data-open="DSQ007TB1" class="link link-reveal link-table xref-fig">1</a>, 508 and 365 iTSCs of &gt;5 ppm showed expression induction and repression by more than 5-fold in response to the hypoxic shock in at least one cell type, respectively. Among the 508 hypoxia-induced iTSCs, 232 were RefSeq-linked iTSCs and 276 were non-RefSeq-linked iTSCs. Among the 276 non-RefSeq-linked hypoxia-induced iTSCs, iTSC of an lncRNA, H19, whose induction in response to hypoxia has been recently reported to be essential in the tumorigenecity of the cell, was included.<sup><a reveal-id="DSQ007C43" data-open="DSQ007C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p>                        <a id="5034051" scrollto-destination="5034051"></a>
<div content-id="DSQ007TB1" class="table-modal"><div class="table-wrap"><div id="DSQ007TB1" class="table-wrap-title" data-id="DSQ007TB1"><div class="label -label">Table 1</div><div class="caption"><p>Statistics of the TSS Seq analysis monitoring hypoxic responses of the cultured cells</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Cell type </th><th>DLD-1 1% O<sub>2</sub> </th><th>HEK293_1% O<sub>2</sub> </th><th>MCF-7 1% O<sub>2</sub> </th><th>TIG-3 1% O<sub>2</sub> </th><th>Total </th></tr></thead><tbody><tr><td rowspan="3">Fold (1%/21% O<sub>2</sub>) &lt;0.2 </td><td>Total </td><td>127 </td><td>99 </td><td>99 </td><td>69 </td><td>365 </td></tr><tr><td>Non-RefSeq-linked iTSC </td><td>79 </td><td>63 </td><td>56 </td><td>27 </td><td>213 </td></tr><tr><td>RefSeq-linked iTSC </td><td>48 </td><td>36 </td><td>43 </td><td>42 </td><td>152 </td></tr><tr><td rowspan="3">Fold (1%/21% O<sub>2</sub>) &gt;5 </td><td>Total </td><td>264 </td><td>111 </td><td>105 </td><td>59 </td><td>508 </td></tr><tr><td>Non-RefSeq-linked iTSC </td><td>169 </td><td>45 </td><td>56 </td><td>19 </td><td>276 </td></tr><tr><td>RefSeq-linked iTSC </td><td>95 </td><td>66 </td><td>49 </td><td>40 </td><td>232 </td></tr></tbody></table></div><div content-id="DSQ007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Cell type </th><th>DLD-1 1% O<sub>2</sub> </th><th>HEK293_1% O<sub>2</sub> </th><th>MCF-7 1% O<sub>2</sub> </th><th>TIG-3 1% O<sub>2</sub> </th><th>Total </th></tr></thead><tbody><tr><td rowspan="3">Fold (1%/21% O<sub>2</sub>) &lt;0.2 </td><td>Total </td><td>127 </td><td>99 </td><td>99 </td><td>69 </td><td>365 </td></tr><tr><td>Non-RefSeq-linked iTSC </td><td>79 </td><td>63 </td><td>56 </td><td>27 </td><td>213 </td></tr><tr><td>RefSeq-linked iTSC </td><td>48 </td><td>36 </td><td>43 </td><td>42 </td><td>152 </td></tr><tr><td rowspan="3">Fold (1%/21% O<sub>2</sub>) &gt;5 </td><td>Total </td><td>264 </td><td>111 </td><td>105 </td><td>59 </td><td>508 </td></tr><tr><td>Non-RefSeq-linked iTSC </td><td>169 </td><td>45 </td><td>56 </td><td>19 </td><td>276 </td></tr><tr><td>RefSeq-linked iTSC </td><td>95 </td><td>66 </td><td>49 </td><td>40 </td><td>232 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034051" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ007TB1" class="table-wrap-title" data-id="DSQ007TB1"><div class="label -label">Table 1</div><div class="caption"><p>Statistics of the TSS Seq analysis monitoring hypoxic responses of the cultured cells</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Cell type </th><th>DLD-1 1% O<sub>2</sub> </th><th>HEK293_1% O<sub>2</sub> </th><th>MCF-7 1% O<sub>2</sub> </th><th>TIG-3 1% O<sub>2</sub> </th><th>Total </th></tr></thead><tbody><tr><td rowspan="3">Fold (1%/21% O<sub>2</sub>) &lt;0.2 </td><td>Total </td><td>127 </td><td>99 </td><td>99 </td><td>69 </td><td>365 </td></tr><tr><td>Non-RefSeq-linked iTSC </td><td>79 </td><td>63 </td><td>56 </td><td>27 </td><td>213 </td></tr><tr><td>RefSeq-linked iTSC </td><td>48 </td><td>36 </td><td>43 </td><td>42 </td><td>152 </td></tr><tr><td rowspan="3">Fold (1%/21% O<sub>2</sub>) &gt;5 </td><td>Total </td><td>264 </td><td>111 </td><td>105 </td><td>59 </td><td>508 </td></tr><tr><td>Non-RefSeq-linked iTSC </td><td>169 </td><td>45 </td><td>56 </td><td>19 </td><td>276 </td></tr><tr><td>RefSeq-linked iTSC </td><td>95 </td><td>66 </td><td>49 </td><td>40 </td><td>232 </td></tr></tbody></table></div><div content-id="DSQ007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Cell type </th><th>DLD-1 1% O<sub>2</sub> </th><th>HEK293_1% O<sub>2</sub> </th><th>MCF-7 1% O<sub>2</sub> </th><th>TIG-3 1% O<sub>2</sub> </th><th>Total </th></tr></thead><tbody><tr><td rowspan="3">Fold (1%/21% O<sub>2</sub>) &lt;0.2 </td><td>Total </td><td>127 </td><td>99 </td><td>99 </td><td>69 </td><td>365 </td></tr><tr><td>Non-RefSeq-linked iTSC </td><td>79 </td><td>63 </td><td>56 </td><td>27 </td><td>213 </td></tr><tr><td>RefSeq-linked iTSC </td><td>48 </td><td>36 </td><td>43 </td><td>42 </td><td>152 </td></tr><tr><td rowspan="3">Fold (1%/21% O<sub>2</sub>) &gt;5 </td><td>Total </td><td>264 </td><td>111 </td><td>105 </td><td>59 </td><td>508 </td></tr><tr><td>Non-RefSeq-linked iTSC </td><td>169 </td><td>45 </td><td>56 </td><td>19 </td><td>276 </td></tr><tr><td>RefSeq-linked iTSC </td><td>95 </td><td>66 </td><td>49 </td><td>40 </td><td>232 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034051" target="_blank">View Large</a></div></div><p>On the basis of the collected TSS Seq data, we examined whether expression patterns of the iTSCs were correlated with any groups of the RefSeq genes. We associated the RefSeq genes with the iTSC when Pearson's correlation coefficient was &gt;0.8 (Pearson's correlation coefficients between the TSS tag counts of an lncRNA, H19, and those of p57kip2 and angiogenin, which are reported direct effectors of H19 in the hypoxic responses,<sup><a reveal-id="DSQ007C30" data-open="DSQ007C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSQ007C43" data-open="DSQ007C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSQ007C44" data-open="DSQ007C44" class="link link-ref link-reveal xref-bibr">44</a></sup> were 0.98 and 0.78, respectively). Then, we examined whether particular GO terms<sup><a reveal-id="DSQ007C45" data-open="DSQ007C45" class="link link-ref link-reveal xref-bibr">45</a></sup> were enriched (<em>P</em> &lt; 1e−6; hypergeometric distribution) in the group of RefSeq genes which were associated with the iTSCs.</p><p>As exemplified in Fig. <a reveal-id="DSQ007F7" data-open="DSQ007F7" class="link link-reveal link-table xref-fig">7</a>A, a non-RefSeq-linked iTSC, which was 9.6-fold induced in MCF-7 cells by hypoxia, was correlated with the 34 RefSeq genes having GO terms, ‘hemophilic cell adhesion’, in common (<em>P</em> = 7e−6). Subcellular localization of the RNA products of this iTSC identified by the RNA Seq analysis was ‘cytoplasm’ (<em>P</em> = 0.0). The biological function of this lncRNA might be to interact directly with cell adhesion protein molecules in modulating cellular mobility. Full list of the iTSCs for which biological functions were inferred by the GO terms is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. Interestingly, although various GO terms were associated, the GO term ‘hemophilic cell adhesion’ appeared far more frequently than the other GO terms (<em>P</em> &lt; 4e−13). In total, 53 iTSCs were correlated with this GO term. One of the major biological roles of iTSCs in response to hypoxia might be to control cell–cell interactions.</p>                        <a id="5034057" scrollto-destination="5034057"></a>
<div id="DSQ007F7" data-id="DSQ007F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00707.jpeg?Expires=2147483647&Signature=N4GjFvyEGIsxBYJirdy9hpy4KrqhxyRxb1WyqQTATKz9cdmbZ4Y1GlUi7wNHweWFjqdoGvqQD33BP1NPFy8BJHsejDiw4QwI6pZoZA-ywy7Fo4bmPoo29ZE8YusC~O9kDabA4UOmIquzq3pnF5n4GlerQARjg5w8Ax9ByN6lOtinaTjpY3IuUVAHlP1OE~myfNLoVsGiDEylHZDlhaVgCMFv3VKKo02AiPd7WHd5SldYsEXTZ2VQV-YL6hn9gLUtTglViVbHBnd7eKSQPITPZpUrE0cbbmTlKXOcYpHWYsOmmC3oPZ6klCSfI15LkwgzJlKw9QAmQoHTfiVjMlGuJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Integration of the transcriptome data obtained from various viewpoints. Transcriptome information collected by the indicated analyses is shown for a non-RefSeq-linked iTSC (A) and a RefSeq-linked iTSC (B)." path-from-xml="dsq00707.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034057/dsq00707.jpeg" path-from-xml="dsq00707.jpeg" target="_blank">View large</a><a section="5034057" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00707.jpeg?Expires=2147483647&Signature=3B0bxMq2c0nmO03IkDi5uOmkPWbH4AwWW7LR6CjwHEm1cdcLwTT7gGLVcWA6r2GFkuXoAPtRePoKGdiA17Hsbh4HWWZVqh6M90mlpLZaDBoUcazAqkMQ~qi-tmo5khS0QuCIt2ZtfXtslcExIDEki78eiwIWwTfnl9Nx7vhx6seXtUOVdwLZc5y~pahVXYhQ4vYyE-jhh-cGtjDXJPv-8z9Ysl8jxdbSyGWJHiw~i5oVI7J0sFMzuuFlhyWiXIspoaFqKuaPy3OCtRqOrLbQ96r70vqs3NIUR75MBCQJoK-BZLXgazwKqA7wNJ21ORJmbeOYfdsv9cI4ZHFiCksu-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034057&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00707.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Integration of the transcriptome data obtained from various viewpoints. Transcriptome information collected by the indicated analyses is shown for a non-RefSeq-linked iTSC (<strong>A</strong>) and a RefSeq-linked iTSC (<strong>B</strong>).</p></div></div><div content-id="DSQ007F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00707.jpeg?Expires=2147483647&Signature=N4GjFvyEGIsxBYJirdy9hpy4KrqhxyRxb1WyqQTATKz9cdmbZ4Y1GlUi7wNHweWFjqdoGvqQD33BP1NPFy8BJHsejDiw4QwI6pZoZA-ywy7Fo4bmPoo29ZE8YusC~O9kDabA4UOmIquzq3pnF5n4GlerQARjg5w8Ax9ByN6lOtinaTjpY3IuUVAHlP1OE~myfNLoVsGiDEylHZDlhaVgCMFv3VKKo02AiPd7WHd5SldYsEXTZ2VQV-YL6hn9gLUtTglViVbHBnd7eKSQPITPZpUrE0cbbmTlKXOcYpHWYsOmmC3oPZ6klCSfI15LkwgzJlKw9QAmQoHTfiVjMlGuJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Integration of the transcriptome data obtained from various viewpoints. Transcriptome information collected by the indicated analyses is shown for a non-RefSeq-linked iTSC (A) and a RefSeq-linked iTSC (B)." path-from-xml="dsq00707.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034057/dsq00707.jpeg" path-from-xml="dsq00707.jpeg" target="_blank">View large</a><a section="5034057" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00707.jpeg?Expires=2147483647&Signature=3B0bxMq2c0nmO03IkDi5uOmkPWbH4AwWW7LR6CjwHEm1cdcLwTT7gGLVcWA6r2GFkuXoAPtRePoKGdiA17Hsbh4HWWZVqh6M90mlpLZaDBoUcazAqkMQ~qi-tmo5khS0QuCIt2ZtfXtslcExIDEki78eiwIWwTfnl9Nx7vhx6seXtUOVdwLZc5y~pahVXYhQ4vYyE-jhh-cGtjDXJPv-8z9Ysl8jxdbSyGWJHiw~i5oVI7J0sFMzuuFlhyWiXIspoaFqKuaPy3OCtRqOrLbQ96r70vqs3NIUR75MBCQJoK-BZLXgazwKqA7wNJ21ORJmbeOYfdsv9cI4ZHFiCksu-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034057&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00707.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Integration of the transcriptome data obtained from various viewpoints. Transcriptome information collected by the indicated analyses is shown for a non-RefSeq-linked iTSC (<strong>A</strong>) and a RefSeq-linked iTSC (<strong>B</strong>).</p></div></div><p>On the other hand, expressions of a significant number of RefSeq-linked iTSCs were also changed by hypoxia, as exemplified in Fig. <a reveal-id="DSQ007F7" data-open="DSQ007F7" class="link link-reveal link-table xref-fig">7</a>B. However, in this group of iTSCs, generally fewer RefSeq genes other than their neighboring RefSeq genes had correlated expression patterns with those iTSCs and fewer GO terms were associated. These results may reflect the fact that inductions of the RefSeq-linked iTSCs are not well coordinated in the transcriptional regulatory network of human genes.</p>                            <h3 scrollto-destination=5034062 id="5034062" class="section-title" data-legacy-id=s3g>3.7. Identification and characterization of TSCs in the antisense of the RefSeq genes</h3>
<p>About 1–5% of the TSS tags were mapped to the antisense of the RefSeq genes, depending on cell types (Table <a reveal-id="DSQ007TB2" data-open="DSQ007TB2" class="link link-reveal link-table xref-fig">2</a>). Similar to the case in the iTSCs, we clustered and analyzed those TSS tags (antisense TSCs; aTSCs). Again, we observed that the aTSCs having significant expression levels are only minor part. Taken all cell types together, although there were 127 036 aTSCs, aTCSs of &gt;5 ppm were only 3301 (Table <a reveal-id="DSQ007TB3" data-open="DSQ007TB3" class="link link-reveal link-table xref-fig">3</a>; also see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). We examined the characteristic features of the aTSCs and found that they showed somewhat similar features with iTSCs of &gt;5 ppm; located in evolutionarily conserved GC-rich regions and rarely associated with repetitive elements (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4B</a>). We also confirmed that the presence of the aTSCs of &gt;5 ppm is clearly supported by pol II binding in some cases (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4C</a>). Interestingly, those aTSCs of &gt;5 ppm were concentrated around the 5′- and 3′-ends of their harboring genes (Fig. <a reveal-id="DSQ007F8" data-open="DSQ007F8" class="link link-reveal link-table xref-fig">8</a>A). Nevertheless, when we further examined their expression patterns with the overlapping RefSeq genes by calculating Pearson's correlation based on the digital TSS tag counts, we found generally correlation between the expression patterns of aTSCs and those of the antisense RefSeq genes (Fig. <a reveal-id="DSQ007F8" data-open="DSQ007F8" class="link link-reveal link-table xref-fig">8</a>B). Also, we observed that general lack of the ordered nucleosome positioning in the surrounding regions (Fig. <a reveal-id="DSQ007F8" data-open="DSQ007F8" class="link link-reveal link-table xref-fig">8</a>C and D), perhaps reflecting the lack of strict transcriptional regulations. These results may imply that, similar to the case in the RefSeq-linked iTSCs, major part of the aTCSs, if not all, are also ‘transcriptional rippling’ products of the transcriptions of the RefSeq genes and have no biological consequences on their own. Distal regions of the genes might be particularly susceptible to such transcriptions, although the possibility that at least a part of those aTSCs are involved in transcriptional regulations of RefSeq genes has not been completely excluded. </p>                        <a id="5034067" scrollto-destination="5034067"></a>
<div content-id="DSQ007TB2" class="table-modal"><div class="table-wrap"><div id="DSQ007TB2" class="table-wrap-title" data-id="DSQ007TB2"><div class="label -label">Table 2</div><div class="caption"><p>Statistics of the sequence tags categorized to the indicated criteria</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Cell type </th><th>DLD-1 </th><th>Ramos </th><th>BEAS2B </th><th>HEK293 </th><th>MCF7 </th><th>TIG3 </th><th>Brain </th><th>Kidney </th><th>Heart </th><th>Fetal brain </th><th>Fetal kidney </th><th>Fetal heart </th><th>Total </th></tr></thead><tbody><tr><td>#TSS tag in RefSeq region </td><td>7 679 112 (87%) </td><td>13 875 727 (83%) </td><td>20 106 118 (89%) </td><td>8 880 335 (91%) </td><td>7 476 053 (89%) </td><td>8 883 872 (90%) </td><td>7 761 770 (66%) </td><td>7 285 131 (64%) </td><td>5 379 867 (57%) </td><td>9 607 689 (82%) </td><td>6 783 071 (77%) </td><td>7 982 779 (77%) </td><td>111 70 1 524 (80%) </td></tr><tr><td>#TSS tag in intergenic region </td><td>957 301 (11%) </td><td>2 355 840 (14%) </td><td>1 744 854 (8%) </td><td>736 616 (8%) </td><td>796 086 (10%) </td><td>783 112 (8%) </td><td>3 472 374 (29%) </td><td>3 458 501 (31%) </td><td>3 653 984 (38%) </td><td>1 542 402 (13%) </td><td>1 592 092 (18%) </td><td>1 817 819 (18%) </td><td>22 910 981 (16%) </td></tr><tr><td>#TSS tag in antisense of RefSeq region </td><td>213 119 (2%) </td><td>408 592 (3%) </td><td>623 077 (3%) </td><td>117 363 (1%) </td><td>82 131 (1%) </td><td>217 908 (2%) </td><td>602 851 (5%) </td><td>579 412 (5%) </td><td>441 998 (5%) </td><td>559 169 (5%) </td><td>487 960 (5%) </td><td>500 645 (5%) </td><td>4 834 225 (4%) </td></tr><tr><td>Total </td><td>8 849 532 </td><td>16 640 159 </td><td>22 474 049 </td><td>9 734 314 </td><td>8 354 270 </td><td>9 884 892 </td><td>11 836 995 </td><td>11 323 044 </td><td>9 475 849 </td><td>11 709 260 </td><td>8 863 123 </td><td>10 301 243 </td><td>139 446 730 </td></tr><tr><td>#iTSC (total) </td><td>47 347 </td><td>58 755 </td><td>59 529 </td><td>27 959 </td><td>23 914 </td><td>18 450 </td><td>91 915 </td><td>41 251 </td><td>53 204 </td><td>67 897 </td><td>50 950 </td><td>64 144 </td><td>371 849 </td></tr><tr><td>#iTSC (&gt;5 ppm) </td><td>931 </td><td>1066 </td><td>1004 </td><td>833 </td><td>1088 </td><td>861 </td><td>1207 </td><td>1013 </td><td>688 </td><td>1812 </td><td>1434 </td><td>1323 </td><td>6039 </td></tr></tbody></table></div><div content-id="DSQ007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Cell type </th><th>DLD-1 </th><th>Ramos </th><th>BEAS2B </th><th>HEK293 </th><th>MCF7 </th><th>TIG3 </th><th>Brain </th><th>Kidney </th><th>Heart </th><th>Fetal brain </th><th>Fetal kidney </th><th>Fetal heart </th><th>Total </th></tr></thead><tbody><tr><td>#TSS tag in RefSeq region </td><td>7 679 112 (87%) </td><td>13 875 727 (83%) </td><td>20 106 118 (89%) </td><td>8 880 335 (91%) </td><td>7 476 053 (89%) </td><td>8 883 872 (90%) </td><td>7 761 770 (66%) </td><td>7 285 131 (64%) </td><td>5 379 867 (57%) </td><td>9 607 689 (82%) </td><td>6 783 071 (77%) </td><td>7 982 779 (77%) </td><td>111 70 1 524 (80%) </td></tr><tr><td>#TSS tag in intergenic region </td><td>957 301 (11%) </td><td>2 355 840 (14%) </td><td>1 744 854 (8%) </td><td>736 616 (8%) </td><td>796 086 (10%) </td><td>783 112 (8%) </td><td>3 472 374 (29%) </td><td>3 458 501 (31%) </td><td>3 653 984 (38%) </td><td>1 542 402 (13%) </td><td>1 592 092 (18%) </td><td>1 817 819 (18%) </td><td>22 910 981 (16%) </td></tr><tr><td>#TSS tag in antisense of RefSeq region </td><td>213 119 (2%) </td><td>408 592 (3%) </td><td>623 077 (3%) </td><td>117 363 (1%) </td><td>82 131 (1%) </td><td>217 908 (2%) </td><td>602 851 (5%) </td><td>579 412 (5%) </td><td>441 998 (5%) </td><td>559 169 (5%) </td><td>487 960 (5%) </td><td>500 645 (5%) </td><td>4 834 225 (4%) </td></tr><tr><td>Total </td><td>8 849 532 </td><td>16 640 159 </td><td>22 474 049 </td><td>9 734 314 </td><td>8 354 270 </td><td>9 884 892 </td><td>11 836 995 </td><td>11 323 044 </td><td>9 475 849 </td><td>11 709 260 </td><td>8 863 123 </td><td>10 301 243 </td><td>139 446 730 </td></tr><tr><td>#iTSC (total) </td><td>47 347 </td><td>58 755 </td><td>59 529 </td><td>27 959 </td><td>23 914 </td><td>18 450 </td><td>91 915 </td><td>41 251 </td><td>53 204 </td><td>67 897 </td><td>50 950 </td><td>64 144 </td><td>371 849 </td></tr><tr><td>#iTSC (&gt;5 ppm) </td><td>931 </td><td>1066 </td><td>1004 </td><td>833 </td><td>1088 </td><td>861 </td><td>1207 </td><td>1013 </td><td>688 </td><td>1812 </td><td>1434 </td><td>1323 </td><td>6039 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034067" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ007TB2" class="table-wrap-title" data-id="DSQ007TB2"><div class="label -label">Table 2</div><div class="caption"><p>Statistics of the sequence tags categorized to the indicated criteria</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Cell type </th><th>DLD-1 </th><th>Ramos </th><th>BEAS2B </th><th>HEK293 </th><th>MCF7 </th><th>TIG3 </th><th>Brain </th><th>Kidney </th><th>Heart </th><th>Fetal brain </th><th>Fetal kidney </th><th>Fetal heart </th><th>Total </th></tr></thead><tbody><tr><td>#TSS tag in RefSeq region </td><td>7 679 112 (87%) </td><td>13 875 727 (83%) </td><td>20 106 118 (89%) </td><td>8 880 335 (91%) </td><td>7 476 053 (89%) </td><td>8 883 872 (90%) </td><td>7 761 770 (66%) </td><td>7 285 131 (64%) </td><td>5 379 867 (57%) </td><td>9 607 689 (82%) </td><td>6 783 071 (77%) </td><td>7 982 779 (77%) </td><td>111 70 1 524 (80%) </td></tr><tr><td>#TSS tag in intergenic region </td><td>957 301 (11%) </td><td>2 355 840 (14%) </td><td>1 744 854 (8%) </td><td>736 616 (8%) </td><td>796 086 (10%) </td><td>783 112 (8%) </td><td>3 472 374 (29%) </td><td>3 458 501 (31%) </td><td>3 653 984 (38%) </td><td>1 542 402 (13%) </td><td>1 592 092 (18%) </td><td>1 817 819 (18%) </td><td>22 910 981 (16%) </td></tr><tr><td>#TSS tag in antisense of RefSeq region </td><td>213 119 (2%) </td><td>408 592 (3%) </td><td>623 077 (3%) </td><td>117 363 (1%) </td><td>82 131 (1%) </td><td>217 908 (2%) </td><td>602 851 (5%) </td><td>579 412 (5%) </td><td>441 998 (5%) </td><td>559 169 (5%) </td><td>487 960 (5%) </td><td>500 645 (5%) </td><td>4 834 225 (4%) </td></tr><tr><td>Total </td><td>8 849 532 </td><td>16 640 159 </td><td>22 474 049 </td><td>9 734 314 </td><td>8 354 270 </td><td>9 884 892 </td><td>11 836 995 </td><td>11 323 044 </td><td>9 475 849 </td><td>11 709 260 </td><td>8 863 123 </td><td>10 301 243 </td><td>139 446 730 </td></tr><tr><td>#iTSC (total) </td><td>47 347 </td><td>58 755 </td><td>59 529 </td><td>27 959 </td><td>23 914 </td><td>18 450 </td><td>91 915 </td><td>41 251 </td><td>53 204 </td><td>67 897 </td><td>50 950 </td><td>64 144 </td><td>371 849 </td></tr><tr><td>#iTSC (&gt;5 ppm) </td><td>931 </td><td>1066 </td><td>1004 </td><td>833 </td><td>1088 </td><td>861 </td><td>1207 </td><td>1013 </td><td>688 </td><td>1812 </td><td>1434 </td><td>1323 </td><td>6039 </td></tr></tbody></table></div><div content-id="DSQ007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Cell type </th><th>DLD-1 </th><th>Ramos </th><th>BEAS2B </th><th>HEK293 </th><th>MCF7 </th><th>TIG3 </th><th>Brain </th><th>Kidney </th><th>Heart </th><th>Fetal brain </th><th>Fetal kidney </th><th>Fetal heart </th><th>Total </th></tr></thead><tbody><tr><td>#TSS tag in RefSeq region </td><td>7 679 112 (87%) </td><td>13 875 727 (83%) </td><td>20 106 118 (89%) </td><td>8 880 335 (91%) </td><td>7 476 053 (89%) </td><td>8 883 872 (90%) </td><td>7 761 770 (66%) </td><td>7 285 131 (64%) </td><td>5 379 867 (57%) </td><td>9 607 689 (82%) </td><td>6 783 071 (77%) </td><td>7 982 779 (77%) </td><td>111 70 1 524 (80%) </td></tr><tr><td>#TSS tag in intergenic region </td><td>957 301 (11%) </td><td>2 355 840 (14%) </td><td>1 744 854 (8%) </td><td>736 616 (8%) </td><td>796 086 (10%) </td><td>783 112 (8%) </td><td>3 472 374 (29%) </td><td>3 458 501 (31%) </td><td>3 653 984 (38%) </td><td>1 542 402 (13%) </td><td>1 592 092 (18%) </td><td>1 817 819 (18%) </td><td>22 910 981 (16%) </td></tr><tr><td>#TSS tag in antisense of RefSeq region </td><td>213 119 (2%) </td><td>408 592 (3%) </td><td>623 077 (3%) </td><td>117 363 (1%) </td><td>82 131 (1%) </td><td>217 908 (2%) </td><td>602 851 (5%) </td><td>579 412 (5%) </td><td>441 998 (5%) </td><td>559 169 (5%) </td><td>487 960 (5%) </td><td>500 645 (5%) </td><td>4 834 225 (4%) </td></tr><tr><td>Total </td><td>8 849 532 </td><td>16 640 159 </td><td>22 474 049 </td><td>9 734 314 </td><td>8 354 270 </td><td>9 884 892 </td><td>11 836 995 </td><td>11 323 044 </td><td>9 475 849 </td><td>11 709 260 </td><td>8 863 123 </td><td>10 301 243 </td><td>139 446 730 </td></tr><tr><td>#iTSC (total) </td><td>47 347 </td><td>58 755 </td><td>59 529 </td><td>27 959 </td><td>23 914 </td><td>18 450 </td><td>91 915 </td><td>41 251 </td><td>53 204 </td><td>67 897 </td><td>50 950 </td><td>64 144 </td><td>371 849 </td></tr><tr><td>#iTSC (&gt;5 ppm) </td><td>931 </td><td>1066 </td><td>1004 </td><td>833 </td><td>1088 </td><td>861 </td><td>1207 </td><td>1013 </td><td>688 </td><td>1812 </td><td>1434 </td><td>1323 </td><td>6039 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034067" target="_blank">View Large</a></div></div>                        <a id="5034069" scrollto-destination="5034069"></a>
<div content-id="DSQ007TB3" class="table-modal"><div class="table-wrap"><div id="DSQ007TB3" class="table-wrap-title" data-id="DSQ007TB3"><div class="label -label">Table 3</div><div class="caption"><p>Statistics of the aTSCs of &gt;5 ppm and those of &lt;5 ppm</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Cell type </th><th>DLD-1 </th><th>Ramos </th><th>BEAS2B </th><th>HEK293 </th><th>MCF7 </th><th>TIG3 </th><th>Brain </th><th>Kidney </th><th>Heart </th><th>Fetal brain </th><th>Fetal kidney </th><th>Fetal heart </th><th>Total </th></tr></thead><tbody><tr><td># aTSC (total) </td><td>17 516 </td><td>19 757 </td><td>23 836 </td><td>8914 </td><td>6491 </td><td>7294 </td><td>28 972 </td><td>20 174 </td><td>25 522 </td><td>20 284 </td><td>23 870 </td><td>23 815 </td><td>127 036 </td></tr><tr><td># aTSC (&gt;5 ppm) </td><td>575 </td><td>453 </td><td>519 </td><td>203 </td><td>197 </td><td>457 </td><td>765 </td><td>897 </td><td>608 </td><td>945 </td><td>1331 </td><td>856 </td><td>3301 </td></tr></tbody></table></div><div content-id="DSQ007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Cell type </th><th>DLD-1 </th><th>Ramos </th><th>BEAS2B </th><th>HEK293 </th><th>MCF7 </th><th>TIG3 </th><th>Brain </th><th>Kidney </th><th>Heart </th><th>Fetal brain </th><th>Fetal kidney </th><th>Fetal heart </th><th>Total </th></tr></thead><tbody><tr><td># aTSC (total) </td><td>17 516 </td><td>19 757 </td><td>23 836 </td><td>8914 </td><td>6491 </td><td>7294 </td><td>28 972 </td><td>20 174 </td><td>25 522 </td><td>20 284 </td><td>23 870 </td><td>23 815 </td><td>127 036 </td></tr><tr><td># aTSC (&gt;5 ppm) </td><td>575 </td><td>453 </td><td>519 </td><td>203 </td><td>197 </td><td>457 </td><td>765 </td><td>897 </td><td>608 </td><td>945 </td><td>1331 </td><td>856 </td><td>3301 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034069" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ007TB3" class="table-wrap-title" data-id="DSQ007TB3"><div class="label -label">Table 3</div><div class="caption"><p>Statistics of the aTSCs of &gt;5 ppm and those of &lt;5 ppm</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Cell type </th><th>DLD-1 </th><th>Ramos </th><th>BEAS2B </th><th>HEK293 </th><th>MCF7 </th><th>TIG3 </th><th>Brain </th><th>Kidney </th><th>Heart </th><th>Fetal brain </th><th>Fetal kidney </th><th>Fetal heart </th><th>Total </th></tr></thead><tbody><tr><td># aTSC (total) </td><td>17 516 </td><td>19 757 </td><td>23 836 </td><td>8914 </td><td>6491 </td><td>7294 </td><td>28 972 </td><td>20 174 </td><td>25 522 </td><td>20 284 </td><td>23 870 </td><td>23 815 </td><td>127 036 </td></tr><tr><td># aTSC (&gt;5 ppm) </td><td>575 </td><td>453 </td><td>519 </td><td>203 </td><td>197 </td><td>457 </td><td>765 </td><td>897 </td><td>608 </td><td>945 </td><td>1331 </td><td>856 </td><td>3301 </td></tr></tbody></table></div><div content-id="DSQ007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Cell type </th><th>DLD-1 </th><th>Ramos </th><th>BEAS2B </th><th>HEK293 </th><th>MCF7 </th><th>TIG3 </th><th>Brain </th><th>Kidney </th><th>Heart </th><th>Fetal brain </th><th>Fetal kidney </th><th>Fetal heart </th><th>Total </th></tr></thead><tbody><tr><td># aTSC (total) </td><td>17 516 </td><td>19 757 </td><td>23 836 </td><td>8914 </td><td>6491 </td><td>7294 </td><td>28 972 </td><td>20 174 </td><td>25 522 </td><td>20 284 </td><td>23 870 </td><td>23 815 </td><td>127 036 </td></tr><tr><td># aTSC (&gt;5 ppm) </td><td>575 </td><td>453 </td><td>519 </td><td>203 </td><td>197 </td><td>457 </td><td>765 </td><td>897 </td><td>608 </td><td>945 </td><td>1331 </td><td>856 </td><td>3301 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5034069" target="_blank">View Large</a></div></div>                        <a id="5034071" scrollto-destination="5034071"></a>
<div id="DSQ007F8" data-id="DSQ007F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00708.jpeg?Expires=2147483647&Signature=mj2PGA8uANrXU-ceKfVnodGwrTXM8mcJopsDGx9eU~yFbLyNMoydzAJIUemDMZUBdxQfaQsA92ETFnU6FmvgChwrr21uWn7xS7H7Y4YemDytXow1YKhf-wqB-kQPz7ZehB1duyzMKNJHuUPXUwK5wWC1oPUyz9l~0CTYzmiYW3bajKCja8tLSvVt3xKcrOSnvu5HY3KZVEt6YQVgkEswxtHl7RwVupCSmreSlybj7fv6~MQU0sbp4kr5eG8VMO0C16s7YHOrC9SNucNa-ZNAFYnH4J37YfFXRKbhfb5vmitspY-Uci3MMZ7bIG711lgewbi5AvHzGGewwRNjudGV9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="aTSCs of the RefSeq genes. In addition, we performed integrated analysis of aTSCs of the RefSeq genes. Interestingly, aTSCs of &gt;5 ppm were concentrated around the 5′- and 3′-ends of their harboring genes (A). Distribution of Pearson's correlation of the TSS tag counts between aTSCs and their harboring genes revealed that there are generally correlation between the expression patterns of aTSCs and those of the antisense RefSeq genes (B). By analyzing the nucleosome structure, we observed a general lack of the ordered nucleosome positioning in the surrounding regions of both aTSCs of &gt;5 ppm (C) and those of &lt;5 ppm (D)." path-from-xml="dsq00708.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034071/dsq00708.jpeg" path-from-xml="dsq00708.jpeg" target="_blank">View large</a><a section="5034071" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00708.jpeg?Expires=2147483647&Signature=NeeUBki~0xQdq1q7FyJf2YLGb8YXDybpippVS6s94vFy3KUbGQ5cEjJlSkaDQ~GBVKnrIbwRHQw9rmteMxjQjJ0A~N-rugXneVa3D-WJ~~oje0YoLZbg6fshNVq8tMppfRM2V6x0c~0Zgk2qvmcyxR8xN~SOLQeLs6cUA3Qpc-oyqc9fEOnK1V0zN81Cr3M3G-N06ZXQGZjYPYPL8mAHykeTiueKOaHlkoszsc7AIhjrD8xfcLKegCHKGyJKmsXVRN4HiI6S4YNtad9JBiAvSvLtQlJmjUVmwXfbmYWAPF2m8qT0rJ6R46AVQST8vA5wN09LWLDrxAbJVYVTm6taTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034071&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00708.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>aTSCs of the RefSeq genes. In addition, we performed integrated analysis of aTSCs of the RefSeq genes. Interestingly, aTSCs of &gt;5 ppm were concentrated around the 5′- and 3′-ends of their harboring genes (<strong>A</strong>). Distribution of Pearson's correlation of the TSS tag counts between aTSCs and their harboring genes revealed that there are generally correlation between the expression patterns of aTSCs and those of the antisense RefSeq genes (<strong>B</strong>). By analyzing the nucleosome structure, we observed a general lack of the ordered nucleosome positioning in the surrounding regions of both aTSCs of &gt;5 ppm (<strong>C</strong>) and those of &lt;5 ppm (<strong>D</strong>).</p></div></div><div content-id="DSQ007F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/m_dsq00708.jpeg?Expires=2147483647&Signature=mj2PGA8uANrXU-ceKfVnodGwrTXM8mcJopsDGx9eU~yFbLyNMoydzAJIUemDMZUBdxQfaQsA92ETFnU6FmvgChwrr21uWn7xS7H7Y4YemDytXow1YKhf-wqB-kQPz7ZehB1duyzMKNJHuUPXUwK5wWC1oPUyz9l~0CTYzmiYW3bajKCja8tLSvVt3xKcrOSnvu5HY3KZVEt6YQVgkEswxtHl7RwVupCSmreSlybj7fv6~MQU0sbp4kr5eG8VMO0C16s7YHOrC9SNucNa-ZNAFYnH4J37YfFXRKbhfb5vmitspY-Uci3MMZ7bIG711lgewbi5AvHzGGewwRNjudGV9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="aTSCs of the RefSeq genes. In addition, we performed integrated analysis of aTSCs of the RefSeq genes. Interestingly, aTSCs of &gt;5 ppm were concentrated around the 5′- and 3′-ends of their harboring genes (A). Distribution of Pearson's correlation of the TSS tag counts between aTSCs and their harboring genes revealed that there are generally correlation between the expression patterns of aTSCs and those of the antisense RefSeq genes (B). By analyzing the nucleosome structure, we observed a general lack of the ordered nucleosome positioning in the surrounding regions of both aTSCs of &gt;5 ppm (C) and those of &lt;5 ppm (D)." path-from-xml="dsq00708.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5034071/dsq00708.jpeg" path-from-xml="dsq00708.jpeg" target="_blank">View large</a><a section="5034071" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq00708.jpeg?Expires=2147483647&Signature=NeeUBki~0xQdq1q7FyJf2YLGb8YXDybpippVS6s94vFy3KUbGQ5cEjJlSkaDQ~GBVKnrIbwRHQw9rmteMxjQjJ0A~N-rugXneVa3D-WJ~~oje0YoLZbg6fshNVq8tMppfRM2V6x0c~0Zgk2qvmcyxR8xN~SOLQeLs6cUA3Qpc-oyqc9fEOnK1V0zN81Cr3M3G-N06ZXQGZjYPYPL8mAHykeTiueKOaHlkoszsc7AIhjrD8xfcLKegCHKGyJKmsXVRN4HiI6S4YNtad9JBiAvSvLtQlJmjUVmwXfbmYWAPF2m8qT0rJ6R46AVQST8vA5wN09LWLDrxAbJVYVTm6taTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5034071&ar=353612&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00708.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>aTSCs of the RefSeq genes. In addition, we performed integrated analysis of aTSCs of the RefSeq genes. Interestingly, aTSCs of &gt;5 ppm were concentrated around the 5′- and 3′-ends of their harboring genes (<strong>A</strong>). Distribution of Pearson's correlation of the TSS tag counts between aTSCs and their harboring genes revealed that there are generally correlation between the expression patterns of aTSCs and those of the antisense RefSeq genes (<strong>B</strong>). By analyzing the nucleosome structure, we observed a general lack of the ordered nucleosome positioning in the surrounding regions of both aTSCs of &gt;5 ppm (<strong>C</strong>) and those of &lt;5 ppm (<strong>D</strong>).</p></div></div>                            <h3 scrollto-destination=5034076 id="5034076" class="section-title" data-legacy-id=s3h>3.8. Conclusions</h3>
<p>In this paper, we described our first attempt to characterize the iTSCs by multifaceted use of massively paralleled sequencer. Integrated analysis of various types of transcriptome data revealed that different types of iTSCs have different properties. Particularly, we showed that non-RefSeq-linked iTSCs of &gt;5 ppm have several features which should be essential to realize biologically relevant transcriptions. And they could be clearly discriminated from other type of iTSCs, many of which might represent noise products of the transcriptions inherent to the long and complex human genome. We also showed that major parts of the aTSCs have the similar properties with the iTSCs which seemed not to form independent transcriptional units. It is true that we used the conservative computational filters placed on the sequencing results. For example, we considered only TSCs of &gt;5 ppm in the limited cell types. We also removed TSCs located within 50-kb upstream regions of the RefSeq genes and defined the iTSCs located within 10-kb downstream regions of the RefSeq genes as ‘RefSeq-linked’ iTSCs. Therefore, classification using further optimized parameters is necessary to re-defining the characteristics of the respective iTSCs group. However, it should be significant that we could observe not all the iTSCs have the same features even as the first approximation.</p><p>We also demonstrated that integrative transcriptome data gives useful starting point for further investigation of the biological roles of the iTSCs, specifically in the case of the hypoxic response of the cells. Indeed, it is important to consider various types of transcriptome data. Importantly, as shown in this study, the number of iTSCs/aTSCs whose biological relevance seems somewhat questionable is far larger than the iTSCs/aTSCs which have presumed biological functions. Without any biological information in addition to cataloged TSSs/cDNAs, it is practically impossible to select and conduct meaningful functional assays for the increasing number of iTSCs/aTSCs. In our opinion, the biggest advantage of the massively paralleled sequencer is that various approaches of genome-wide analysis are simultaneously enabled on a common platform. Further intensive use of the massively paralleled sequencer will shed brighter light on dynamically interacting worlds of ncRNAs and proteins, which collectively realize complex human gene network. With such knowledge, we will finally understand the biological meaning of every transcription initiation event in the human genome.</p>                            <h2 scrollto-destination=5034081 id="5034081" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at DNA Research online.</a></p>                            <h2 scrollto-destination=5034084 id="5034084" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by Ministry of Economy, Trade and Industry (METI) of Japan [the Japan Key Technology Center project]; Ministry of Health, Labour and Welfare [Third-Term Comprehensive 10-Year Strategy for Cancer Control, Grants-in-Aid for Cancer Research]; Ministry of Education, Culture, Sports, Science and Technology-Japan [Grant-in-Aid for Scientific Research on Priority Areas].</p>                            <h2 scrollto-destination=5034086 id="5034086" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to K. Abe, E. Sekimori and K. Imamura for technical supports. We are also thankful to Dr Sierro, Dr M. Frith and Mr F. Sathira for critical reading of the manuscript.</p>                            <h2 scrollto-destination=5034088 id="5034088" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ007C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hashimoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kasai</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">5′-end SAGE for the analysis of transcriptional start sites</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1146</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt998" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt998</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=5%E2%80%B2-end%20SAGE%20for%20the%20analysis%20of%20transcriptional%20start%20sites&amp;author=S.%20Hashimoto&amp;author=Y.%20Suzuki&amp;author=Y.%20Kasai&amp;publication_year=2004&amp;journal=Nat.%20Biotechnol.&amp;volume=22&amp;pages=1146-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt998" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt998" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt998"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15300261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kasukawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Katayama</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The transcriptional landscape of the mammalian genome</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">1559</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20landscape%20of%20the%20mammalian%20genome&amp;author=P.%20Carninci&amp;author=T.%20Kasukawa&amp;author=S.%20Katayama&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=1559-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1112014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1112014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1112014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16141072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Stamatoyannopoulos</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">447</div> (pg. <div class="fpage">799</div>-<div class="lpage">816</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature05874" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature05874</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20analysis%20of%20functional%20elements%20in%201%25%20of%20the%20human%20genome%20by%20the%20ENCODE%20pilot%20project&amp;author=E.%20Birney&amp;author=J.A.%20Stamatoyannopoulos&amp;author=A.%20Dutta&amp;publication_year=2007&amp;journal=Nature&amp;volume=447&amp;pages=799-816" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05874" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05874" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05874"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17571346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bertone</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Stolc</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Royce</div> <div class="given-names">T.E.</div></div>, et al. </div><div class="article-title">Global identification of human transcribed sequences with genome tiling arrays</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">306</div> (pg. <div class="fpage">2242</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1103388" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1103388</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20identification%20of%20human%20transcribed%20sequences%20with%20genome%20tiling%20arrays&amp;author=P.%20Bertone&amp;author=V.%20Stolc&amp;author=T.E.%20Royce&amp;publication_year=2004&amp;journal=Science&amp;volume=306&amp;pages=2242-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1103388" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1103388" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1103388"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15539566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guttman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Amit</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Garber</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">458</div> (pg. <div class="fpage">223</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature07672" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature07672</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20signature%20reveals%20over%20a%20thousand%20highly%20conserved%20large%20non-coding%20RNAs%20in%20mammals&amp;author=M.%20Guttman&amp;author=I.%20Amit&amp;author=M.%20Garber&amp;publication_year=2009&amp;journal=Nature&amp;volume=458&amp;pages=223-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07672" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07672" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07672"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19182780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Babak</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">T.R.</div></div>. </div><div class="article-title">A systematic search for new mammalian noncoding RNAs indicates little conserved intergenic transcription</div>, <div class="source">BMC Genomics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">104</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-6-104" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-6-104</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20systematic%20search%20for%20new%20mammalian%20noncoding%20RNAs%20indicates%20little%20conserved%20intergenic%20transcription&amp;author=T.%20Babak&amp;author=B.J.%20Blencowe&amp;author=T.R.%20Hughes&amp;publication_year=2005&amp;journal=BMC%20Genomics&amp;volume=6&amp;pages=104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-6-104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-6-104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-6-104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16083503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rinn</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Euskirchen</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Bertone</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The transcriptional activity of human chromosome 22</div>, <div class="source">Genes Dev.</div> , <div class="year">2003</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">529</div>-<div class="lpage">40</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gad.1055203" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gad.1055203</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20activity%20of%20human%20chromosome%2022&amp;author=J.L.%20Rinn&amp;author=G.%20Euskirchen&amp;author=P.%20Bertone&amp;publication_year=2003&amp;journal=Genes%20Dev.&amp;volume=17&amp;pages=529-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1055203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1055203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1055203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12600945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yelin</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Dahary</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Widespread occurrence of antisense transcription in the human genome</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">379</div>-<div class="lpage">86</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt808" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt808</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20occurrence%20of%20antisense%20transcription%20in%20the%20human%20genome&amp;author=R.%20Yelin&amp;author=D.%20Dahary&amp;author=R.%20Sorek&amp;publication_year=2003&amp;journal=Nat.%20Biotechnol.&amp;volume=21&amp;pages=379-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12640466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosok</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Sioud</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Systematic identification of sense-antisense transcripts in mammalian cells</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">104</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt925" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt925</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20identification%20of%20sense-antisense%20transcripts%20in%20mammalian%20cells&amp;author=O.%20Rosok&amp;author=M.%20Sioud&amp;publication_year=2004&amp;journal=Nat.%20Biotechnol.&amp;volume=22&amp;pages=104-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt925" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt925" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt925"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Khaitovich</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kelso</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Franz</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Functionality of intergenic transcription: an evolutionary comparison</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e171</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pgen.0020171" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pgen.0020171</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functionality%20of%20intergenic%20transcription%3A%20an%20evolutionary%20comparison&amp;author=P.%20Khaitovich&amp;author=J.%20Kelso&amp;author=H.%20Franz&amp;publication_year=2006&amp;journal=PLoS%20Genet.&amp;volume=2&amp;pages=e171" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020171" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020171" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020171"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17040132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakaya</div> <div class="given-names">H.I.</div></div>,  <div class="name"><div class="surname">Amaral</div> <div class="given-names">P.P.</div></div>,  <div class="name"><div class="surname">Louro</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Genome mapping and expression analyses of human intronic noncoding RNAs reveal tissue-specific patterns and enrichment in genes related to regulation of transcription</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R43</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2007-8-3-r43" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2007-8-3-r43</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20mapping%20and%20expression%20analyses%20of%20human%20intronic%20noncoding%20RNAs%20reveal%20tissue-specific%20patterns%20and%20enrichment%20in%20genes%20related%20to%20regulation%20of%20transcription&amp;author=H.I.%20Nakaya&amp;author=P.P.%20Amaral&amp;author=R.%20Louro&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-3-r43" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-3-r43" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-3-r43"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17386095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Numata</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kanai</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Identification of putative noncoding RNAs among the RIKEN mouse full-length cDNA collection</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1301</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.1011603" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.1011603</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20putative%20noncoding%20RNAs%20among%20the%20RIKEN%20mouse%20full-length%20cDNA%20collection&amp;author=K.%20Numata&amp;author=A.%20Kanai&amp;author=R.%20Saito&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=1301-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1011603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1011603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1011603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12819127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ravasi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Pang</div> <div class="given-names">K.C.</div></div>, et al. </div><div class="article-title">Experimental validation of the regulated expression of large numbers of non-coding RNAs from the mouse genome</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">11</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.4200206" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.4200206</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20validation%20of%20the%20regulated%20expression%20of%20large%20numbers%20of%20non-coding%20RNAs%20from%20the%20mouse%20genome&amp;author=T.%20Ravasi&amp;author=H.%20Suzuki&amp;author=K.C.%20Pang&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=11-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4200206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4200206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4200206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16344565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Szell</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bata-Csorgo</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Kemeny</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The enigmatic world of mRNA-like ncRNAs: their role in human evolution and in human diseases</div>, <div class="source">Semin. Cancer Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">141</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.semcancer.2008.01.007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.semcancer.2008.01.007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20enigmatic%20world%20of%20mRNA-like%20ncRNAs%3A%20their%20role%20in%20human%20evolution%20and%20in%20human%20diseases&amp;author=M.%20Szell&amp;author=Z.%20Bata-Csorgo&amp;author=L.%20Kemeny&amp;publication_year=2008&amp;journal=Semin.%20Cancer%20Biol.&amp;volume=18&amp;pages=141-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.semcancer.2008.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.semcancer.2008.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.semcancer.2008.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18282717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perez</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Hoage</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Pritchett</div> <div class="given-names">J.R.</div></div>, et al. </div><div class="article-title">Long, abundantly expressed non-coding transcripts are altered in cancer</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">642</div>-<div class="lpage">55</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/hmg/ddm336" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/hmg/ddm336</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%2C%20abundantly%20expressed%20non-coding%20transcripts%20are%20altered%20in%20cancer&amp;author=D.S.%20Perez&amp;author=T.R.%20Hoage&amp;author=J.R.%20Pritchett&amp;publication_year=2008&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=17&amp;pages=642-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddm336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddm336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddm336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18006640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mercer</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Dinger</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Sunkin</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Mehler</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Mattick</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">Specific expression of long noncoding RNAs in the mouse brain</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">716</div>-<div class="lpage">21</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0706729105" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0706729105</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specific%20expression%20of%20long%20noncoding%20RNAs%20in%20the%20mouse%20brain&amp;author=T.R.%20Mercer&amp;author=M.E.%20Dinger&amp;author=S.M.%20Sunkin&amp;author=M.F.%20Mehler&amp;author=J.S.%20Mattick&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=716-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0706729105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0706729105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0706729105"> </span></div> </div></div></div></div></div><div content-id="DSQ007C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ponting</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">P.L.</div></div>,  <div class="name"><div class="surname">Reik</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Evolution and functions of long noncoding RNAs</div>, <div class="source">Cell</div> , <div class="year">2009</div>, vol. <div class="volume">136</div> (pg. <div class="fpage">629</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2009.02.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2009.02.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20and%20functions%20of%20long%20noncoding%20RNAs&amp;author=C.P.%20Ponting&amp;author=P.L.%20Oliver&amp;author=W.%20Reik&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=629-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.02.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.02.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.02.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19239885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ponjavic</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ponting</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">Lunter</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Functionality or transcriptional noise? Evidence for selection within long noncoding RNAs</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">556</div>-<div class="lpage">65</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.6036807" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.6036807</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functionality%20or%20transcriptional%20noise%3F%20Evidence%20for%20selection%20within%20long%20noncoding%20RNAs&amp;author=J.%20Ponjavic&amp;author=C.P.%20Ponting&amp;author=G.%20Lunter&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=556-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6036807" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6036807" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6036807"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17387145" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arias</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Hayward</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Filtering transcriptional noise during development: concepts and mechanisms</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">34</div>-<div class="lpage">44</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg1750" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg1750</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Filtering%20transcriptional%20noise%20during%20development%3A%20concepts%20and%20mechanisms&amp;author=A.M.%20Arias&amp;author=P.%20Hayward&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=7&amp;pages=34-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1750" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1750" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1750"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16369570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Struhl</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Transcriptional noise and the fidelity of initiation by RNA polymerase II</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">103</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nsmb0207-103" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nsmb0207-103</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20noise%20and%20the%20fidelity%20of%20initiation%20by%20RNA%20polymerase%20II&amp;author=K.%20Struhl&amp;publication_year=2007&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=14&amp;pages=103-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb0207-103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb0207-103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb0207-103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17277804" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brosius</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Waste not, want not–transcript excess in multicellular eukaryotes</div>, <div class="source">Trends Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">287</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2005.02.014" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2005.02.014</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Waste%20not%2C%20want%20not%E2%80%93transcript%20excess%20in%20multicellular%20eukaryotes&amp;author=J.%20Brosius&amp;publication_year=2005&amp;journal=Trends%20Genet.&amp;volume=21&amp;pages=287-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.02.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.02.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.02.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15851065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huttenhofer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schattner</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Polacek</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Non-coding RNAs: hope or hype?</div>, <div class="source">Trends Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">289</div>-<div class="lpage">97</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2005.03.007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2005.03.007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-coding%20RNAs%3A%20hope%20or%20hype%3F&amp;author=A.%20Huttenhofer&amp;author=P.%20Schattner&amp;author=N.%20Polacek&amp;publication_year=2005&amp;journal=Trends%20Genet.&amp;volume=21&amp;pages=289-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.03.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.03.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.03.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15851066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cooper</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Brudno</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Stone</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Dubchak</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Batzoglou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sidow</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Characterization of evolutionary rates and constraints in three mammalian genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">539</div>-<div class="lpage">48</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2034704" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2034704</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20evolutionary%20rates%20and%20constraints%20in%20three%20mammalian%20genomes&amp;author=G.M.%20Cooper&amp;author=M.%20Brudno&amp;author=E.A.%20Stone&amp;author=I.%20Dubchak&amp;author=S.%20Batzoglou&amp;author=A.%20Sidow&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=539-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2034704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2034704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2034704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15059994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cooper</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Stone</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Asimenos</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Batzoglou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sidow</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Distribution and intensity of constraint in mammalian genomic sequence</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">901</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.3577405" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.3577405</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20and%20intensity%20of%20constraint%20in%20mammalian%20genomic%20sequence&amp;author=G.M.%20Cooper&amp;author=E.A.%20Stone&amp;author=G.%20Asimenos&amp;author=E.D.%20Green&amp;author=S.%20Batzoglou&amp;author=A.%20Sidow&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=901-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3577405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3577405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3577405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15965027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Mouse transcriptome: neutral evolution of ‘non-coding’ complementary DNAs</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">431</div>  <div class="comment">1 p following 757; discussion following 757</div></div></div></div></div><div content-id="DSQ007C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wakaguri</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yamashita</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>. </div><div class="article-title">DBTSS: database of transcription start sites progress report</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D97</div>-<div class="lpage">101</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm901" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm901</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DBTSS%3A%20database%20of%20transcription%20start%20sites%20progress%20report&amp;author=H.%20Wakaguri&amp;author=R.%20Yamashita&amp;author=Y.%20Suzuki&amp;author=S.%20Sugano&amp;author=K.%20Nakai&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D97-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm901" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm901" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm901"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17942421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Construction of a full-length enriched and a 5′-end enriched cDNA library using the oligo-capping method</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">221</div> (pg. <div class="fpage">73</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20full-length%20enriched%20and%20a%205%E2%80%B2-end%20enriched%20cDNA%20library%20using%20the%20oligo-capping%20method&amp;author=Y.%20Suzuki&amp;author=S.%20Sugano&amp;publication_year=2003&amp;journal=Methods%20Mol.%20Biol.&amp;volume=221&amp;pages=73-91" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12703735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ota</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Complete sequencing and characterization of 21,243 full-length human cDNAs</div>, <div class="source">Nat. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">40</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng1285" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng1285</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20sequencing%20and%20characterization%20of%2021%2C243%20full-length%20human%20cDNAs&amp;author=T.%20Ota&amp;author=Y.%20Suzuki&amp;author=T.%20Nishikawa&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=40-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14702039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bentley</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">Whole-genome re-sequencing</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">545</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gde.2006.10.009" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gde.2006.10.009</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20re-sequencing&amp;author=D.R.%20Bentley&amp;publication_year=2006&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=16&amp;pages=545-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2006.10.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2006.10.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2006.10.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17055251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsuchihara</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wakaguri</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Massive transcriptional start site analysis of human genes in hypoxia cells</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">2249</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkp066" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkp066</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massive%20transcriptional%20start%20site%20analysis%20of%20human%20genes%20in%20hypoxia%20cells&amp;author=K.%20Tsuchihara&amp;author=Y.%20Suzuki&amp;author=H.%20Wakaguri&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=2249-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp066"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19237398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schones</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Cuddapah</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Dynamic regulation of nucleosome positioning in the human genome</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">887</div>-<div class="lpage">98</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2008.02.022" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2008.02.022</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20regulation%20of%20nucleosome%20positioning%20in%20the%20human%20genome&amp;author=D.E.%20Schones&amp;author=K.%20Cui&amp;author=S.%20Cuddapah&amp;publication_year=2008&amp;journal=Cell&amp;volume=132&amp;pages=887-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.02.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.02.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.02.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18329373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Albert</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Mavrich</div> <div class="given-names">T.N.</div></div>,  <div class="name"><div class="surname">Tomsho</div> <div class="given-names">L.P.</div></div>, et al. </div><div class="article-title">Translational and rotational settings of H2A.Z nucleosomes across the <em>Saccharomyces cerevisiae</em> genome</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">446</div> (pg. <div class="fpage">572</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature05632" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature05632</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20and%20rotational%20settings%20of%20H2A.Z%20nucleosomes%20across%20the%20Saccharomyces%20cerevisiae%20genome&amp;author=I.%20Albert&amp;author=T.N.%20Mavrich&amp;author=L.P.%20Tomsho&amp;publication_year=2007&amp;journal=Nature&amp;volume=446&amp;pages=572-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05632" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05632" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05632"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17392789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Wold</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Genome-wide mapping of in vivo protein-DNA interactions</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">316</div> (pg. <div class="fpage">1497</div>-<div class="lpage">502</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1141319" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1141319</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20mapping%20of%20in%20vivo%20protein-DNA%20interactions&amp;author=D.S.%20Johnson&amp;author=A.%20Mortazavi&amp;author=R.M.%20Myers&amp;author=B.%20Wold&amp;publication_year=2007&amp;journal=Science&amp;volume=316&amp;pages=1497-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1141319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1141319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1141319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seila</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Calabrese</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Levine</div> <div class="given-names">S.S.</div></div>, et al. </div><div class="article-title">Divergent transcription from active promoters</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">322</div> (pg. <div class="fpage">1849</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1162253" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1162253</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergent%20transcription%20from%20active%20promoters&amp;author=A.C.%20Seila&amp;author=J.M.%20Calabrese&amp;author=S.S.%20Levine&amp;publication_year=2008&amp;journal=Science&amp;volume=322&amp;pages=1849-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1162253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1162253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1162253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19056940" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">McCue</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Schaeffer</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wold</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">621</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1226" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1226</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20quantifying%20mammalian%20transcriptomes%20by%20RNA-Seq&amp;author=A.%20Mortazavi&amp;author=B.A.%20Williams&amp;author=K.%20McCue&amp;author=L.%20Schaeffer&amp;author=B.%20Wold&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=621-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ebisuya</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakajima</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishida</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Ripples from neighbouring transcription</div>, <div class="source">Nat. Cell Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1106</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ncb1771" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ncb1771</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ripples%20from%20neighbouring%20transcription&amp;author=M.%20Ebisuya&amp;author=T.%20Yamamoto&amp;author=M.%20Nakajima&amp;author=E.%20Nishida&amp;publication_year=2008&amp;journal=Nat.%20Cell%20Biol.&amp;volume=10&amp;pages=1106-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncb1771" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncb1771" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncb1771"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19160492" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dennis</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Thurman</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Kingston</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Stamatoyannopoulos</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Noble</div> <div class="given-names">W.S.</div></div>. </div><div class="article-title">Predicting human nucleosome occupancy from primary sequence</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000134</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pcbi.1000134" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pcbi.1000134</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20human%20nucleosome%20occupancy%20from%20primary%20sequence&amp;author=S.%20Gupta&amp;author=J.%20Dennis&amp;author=R.E.%20Thurman&amp;author=R.%20Kingston&amp;author=J.A.%20Stamatoyannopoulos&amp;author=W.S.%20Noble&amp;publication_year=2008&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=4&amp;pages=e1000134" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000134" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000134" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000134"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18725940" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pugh</div> <div class="given-names">B.F.</div></div>. </div><div class="article-title">Nucleosome positioning and gene regulation: advances through genomics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">161</div>-<div class="lpage">72</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg2522" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg2522</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosome%20positioning%20and%20gene%20regulation%3A%20advances%20through%20genomics&amp;author=C.%20Jiang&amp;author=B.F.%20Pugh&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=10&amp;pages=161-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2522" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2522" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2522"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19204718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ozsolak</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.S.</div></div>,  <div class="name"><div class="surname">Fisher</div> <div class="given-names">D.E.</div></div>. </div><div class="article-title">High-throughput mapping of the chromatin structure of human promoters</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">244</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt1279" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt1279</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20mapping%20of%20the%20chromatin%20structure%20of%20human%20promoters&amp;author=F.%20Ozsolak&amp;author=J.S.%20Song&amp;author=X.S.%20Liu&amp;author=D.E.%20Fisher&amp;publication_year=2007&amp;journal=Nat.%20Biotechnol.&amp;volume=25&amp;pages=244-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1279"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17220878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Taira</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tsunoda</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Diverse transcriptional initiation revealed by fine, large-scale mapping of mRNA start sites</div>, <div class="source">EMBO Rep.</div> , <div class="year">2001</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">388</div>-<div class="lpage">93</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11375929" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diverse%20transcriptional%20initiation%20revealed%20by%20fine%2C%20large-scale%20mapping%20of%20mRNA%20start%20sites&amp;author=Y.%20Suzuki&amp;author=H.%20Taira&amp;author=T.%20Tsunoda&amp;publication_year=2001&amp;journal=EMBO%20Rep.&amp;volume=2&amp;pages=388-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/embo-reports/kve085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fembo-reports%2Fkve085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fembo-reports%2Fkve085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11375929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Griffiths-Jones</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Saini</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">van Dongen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Enright</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">miRBase: tools for microRNA genomics</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D154</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm952" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm952</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=miRBase%3A%20tools%20for%20microRNA%20genomics&amp;author=S.%20Griffiths-Jones&amp;author=H.K.%20Saini&amp;author=S.%20van%20Dongen&amp;author=A.J.%20Enright&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D154-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm952" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm952" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm952"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17991681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hua</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Sno/scaRNAbase: a curated database for small nucleolar RNAs and cajal body-specific RNAs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D183</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl873" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl873</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sno%2FscaRNAbase%3A%20a%20curated%20database%20for%20small%20nucleolar%20RNAs%20and%20cajal%20body-specific%20RNAs&amp;author=J.%20Xie&amp;author=M.%20Zhang&amp;author=T.%20Zhou&amp;author=X.%20Hua&amp;author=L.%20Tang&amp;author=W.%20Wu&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D183-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl873" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl873" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl873"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17099227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matouk</div> <div class="given-names">I.J.</div></div>,  <div class="name"><div class="surname">DeGroot</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Mezan</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The H19 non-coding RNA is essential for human tumor growth</div>, <div class="source">PLoS One</div> , <div class="year">2007</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e845</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0000845" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0000845</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20H19%20non-coding%20RNA%20is%20essential%20for%20human%20tumor%20growth&amp;author=I.J.%20Matouk&amp;author=N.%20DeGroot&amp;author=S.%20Mezan&amp;publication_year=2007&amp;journal=PLoS%20One&amp;volume=2&amp;pages=e845" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0000845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0000845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0000845"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17786216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cai</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Cullen</div> <div class="given-names">B.R.</div></div>. </div><div class="article-title">The imprinted H19 noncoding RNA is a primary microRNA precursor</div>, <div class="source">RNA</div> , <div class="year">2007</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">313</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1261/rna.351707" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1261/rna.351707</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20imprinted%20H19%20noncoding%20RNA%20is%20a%20primary%20microRNA%20precursor&amp;author=X.%20Cai&amp;author=B.R.%20Cullen&amp;publication_year=2007&amp;journal=RNA&amp;volume=13&amp;pages=313-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.351707" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.351707" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.351707"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ007C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ007C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="collab">GeneOntologyConsortium</div><div class="article-title">The gene ontology (GO) project in 2006</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D322</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkj021" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkj021</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353612&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq007-dsq007_supp_table_2.xls?Expires=2147483647&amp;Signature=uacI5cKNO5J~D6tdidsNnQwv5070E971csUAny0Pi~wGsPDhY0V9-h9d5GSAoOmWlzO3G-PhrjzYUoNiNiPuj6lyhtgO-l5hQeFqO70SopTYzYv79JVX8n3Yh1M4EaTEclCzDQVlppMO9TvZkMDcQHuRrRkH~QsjeKk2N-TjltQhEZBTltzPeyVyyy8~ZnJSQeJyPwlU620TCf1xEJIr0Vkzu3LFY8opzfgp6RaONJY5SrgXMkKw0rMWhgf3BfbHpkEJInxDlGmDMqMD6roG-tZpL3te3t9dF8NGcOyUGRVni0K2IdYCnDIlQ8DY3lOzVM6j7yfqaoyvIz~o-4YN1Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq007-dsq007_supp_table_1.xls?Expires=2147483647&amp;Signature=u1-sQ~y6scPhmcMu5cUNpMEQZOjsJsG6k5~Or5WFB99VIWlcxQ2E-D3a7JIYaMlenRKu6wXsKpgx8gSXW2a-jgIJrAN34qatKYZnp7KwB23f4Hqcl10WypUaXHepin6svfkQX6Qlobw3XoivRBxEzCDXKDPge6RDfZhrrF8-FKgREULUdRPqP7M7V-ezfOA2bdWdag7ksqTXfw6zbJhFBcTy132~~a2tnTeyCG7vkKLUCiElUsCSjFBUJXA~-SCrr2OVZPA6sCCBkkRpCw52IWX8130nJQ~1Tcqu80OK8ORB0QBzjlt5n8xwM9GotAT6PIIhWIUM7CzGm9LfRH5rqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq007/2/dsq007-dsq007supp.pdf?Expires=2147483647&amp;Signature=qiIJ2LZin07QA150V-Wj888AYTpM~6m6uPQseg4TPSkJ1ZDRFT0HR00-8ZMI-Ky8rlL0JO5sgvQS8queUVOF5chlFRmpQqWX1KWXPBPCUM2mrFCcFLMxA9mkeoi4YVyO98rUkjdhreh41lozMwzo7B~ruq0BCnlM-muoOvbGCYQUnqCoCSOjJvclXk7cDUph21NHXl3~q7h--nNEvjeLF-1Ic99gSb3TNyDyabDvYN~ImklETQzT-1R9lk-dOFxu7yPrchyG8NVGtSawdqvXflbSoNHbWnHhWLdAlW79WDvx3ZkRUOMR1YLFSuk5PdZIpMIoSmzVr-hCndZRHTokzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">139</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000279411300004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq007"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353612"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000279411300004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f3%2f169%2f353612">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000279411300004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F3%2F169%2F353612" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353612'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Characterization of Transcription Start Sites of Putative Non-coding RNAs by Multifaceted Use of Massively Paralleled Sequencer" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq007");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f169%2f353612">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

